<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/3/653" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/3/653.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/3/653.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/3/653.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/3/653/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/3/653.atom" />
<meta name="HW.pisa" content="ggg;6/3/653" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Combinatorial Cis-regulation in Saccharomyces Species" />
<meta name="DC.Identifier" content="10.1534/g3.115.024331" />
<meta name="DC.Date" content="2016-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2016 Spivak and Stormo. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Transcriptional control of gene expression requires interactions between the cis -regulatory elements (CREs) controlling gene promoters. We developed a sensitive computational method to identify CRE combinations with conserved spacing that does not require genome alignments. When applied to seven sensu stricto and sensu lato Saccharomyces species, 80% of the predicted interactions displayed some evidence of combinatorial transcriptional behavior in several existing datasets including: (1) chromatin immunoprecipitation data for colocalization of transcription factors, (2) gene expression data for coexpression of predicted regulatory targets, and (3) gene ontology databases for common pathway membership of predicted regulatory targets. We tested several predicted CRE interactions with chromatin immunoprecipitation experiments in a wild-type strain and strains in which a predicted cofactor was deleted. Our experiments confirmed that transcription factor (TF) occupancy at the promoters of the CRE combination target genes depends on the predicted cofactor while occupancy of other promoters is independent of the predicted cofactor. Our method has the additional advantage of identifying regulatory differences between species. By analyzing the S. cerevisiae and S . bayanus genomes, we identified differences in combinatorial cis -regulation between the species and showed that the predicted changes in gene regulation explain several of the species-specific differences seen in gene expression datasets. In some instances, the same CRE combinations appear to regulate genes involved in distinct biological processes in the two different species. The results of this research demonstrate that (1) combinatorial cis -regulation can be inferred by multi-genome analysis and (2) combinatorial cis -regulation can explain differences in gene expression between species." />
<meta name="DC.Contributor" content="Aaron T. Spivak" />
<meta name="DC.Contributor" content="Gary D. Stormo" />
<meta name="article:published_time" content="2016-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Combinatorial Cis-regulation in Saccharomyces Species" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Transcriptional control of gene expression requires interactions between the &lt;i&gt;cis&lt;/i&gt;-regulatory elements (CREs) controlling gene promoters. We developed a sensitive computational method to identify CRE combinations with conserved spacing that does not require genome alignments. When applied to seven &lt;i&gt;sensu stricto&lt;/i&gt; and &lt;i&gt;sensu lato Saccharomyces&lt;/i&gt; species, 80% of the predicted interactions displayed some evidence of combinatorial transcriptional behavior in several existing datasets including: (1) chromatin immunoprecipitation data for colocalization of transcription factors, (2) gene expression data for coexpression of predicted regulatory targets, and (3) gene ontology databases for common pathway membership of predicted regulatory targets. We tested several predicted CRE interactions with chromatin immunoprecipitation experiments in a wild-type strain and strains in which a predicted cofactor was deleted. Our experiments confirmed that transcription factor (TF) occupancy at the promoters of the CRE combination target genes depends on the predicted cofactor while occupancy of other promoters is independent of the predicted cofactor. Our method has the additional advantage of identifying regulatory differences between species. By analyzing the &lt;i&gt;S. cerevisiae&lt;/i&gt; and &lt;i&gt;S&lt;/i&gt;&lt;i&gt;. bayanus&lt;/i&gt; genomes, we identified differences in combinatorial &lt;i&gt;cis&lt;/i&gt;-regulation between the species and showed that the predicted changes in gene regulation explain several of the species-specific differences seen in gene expression datasets. In some instances, the same CRE combinations appear to regulate genes involved in distinct biological processes in the two different species. The results of this research demonstrate that (1) combinatorial &lt;i&gt;cis&lt;/i&gt;-regulation can be inferred by multi-genome analysis and (2) combinatorial &lt;i&gt;cis&lt;/i&gt;-regulation can explain differences in gene expression between species.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/03/01" />
<meta name="citation_mjid" content="ggg;6/3/653" />
<meta name="citation_id" content="6/3/653" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/3/653" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/3/653.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/3/653.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/3/653.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.024331" />
<meta name="citation_pmid" content="26772747" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="15" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="653" />
<meta name="citation_lastpage" content="667" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Aaron T. Spivak" />
<meta name="citation_author_institution" content="Department of Genetics, Center for Genome Sciences and Systems Biology, Washington University School of Medicine, St Louis, Missouri 63108" />
<meta name="citation_author" content="Gary D. Stormo" />
<meta name="citation_author_institution" content="Department of Genetics, Center for Genome Sciences and Systems Biology, Washington University School of Medicine, St Louis, Missouri 63108" />
<meta name="citation_author_email" content="stormo@wustl.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6896-1850" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=D. Aguilar;citation_author=B. Oliva;citation_title=Topological comparison of methods for predicting transcriptional cooperativity in yeast.;citation_pages=137-137;citation_volume=9;citation_year=2008;citation_pmid=18366726;citation_doi=10.1186/1471-2164-9-137" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=U. Alon;citation_title=Network motifs: theory and experimental approaches.;citation_pages=450-461;citation_volume=8;citation_year=2007;citation_issue=6;citation_pmid=17510665;citation_doi=10.1038/nrg2102" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. N. Arbeitman;citation_author=E. E. Furlong;citation_author=F. Imam;citation_author=E. Johnson;citation_author=B. H. Null;citation_title=Gene Expression During the Life Cycle of Drosophila melanogaster;citation_pages=2270-2275;citation_volume=297;citation_year=2002;citation_issue=5590;citation_pmid=12351791;citation_doi=10.1126/science.1072152" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. Ashburner;citation_author=C. A. Ball;citation_author=J. A. Blake;citation_author=D. Botstein;citation_author=H. Butler;citation_title=Gene ontology: tool for the unification of biology. The Gene Ontology Consortium.;citation_pages=25-29;citation_volume=25;citation_year=2000;citation_issue=1;citation_pmid=10802651;citation_doi=10.1038/75556" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=G. Badis;citation_author=M. F. Berger;citation_author=A. A. Philippakis;citation_author=S. Talukder;citation_author=A. R. Gehrke;citation_title=Diversity and complexity in DNA recognition by transcription factors.;citation_volume=324;citation_year=2009;citation_pmid=19443739;citation_doi=10.1126/science.1162327" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. Balaji;citation_author=M. M. Babu;citation_author=L. M. Iyer;citation_author=N. M. Luscombe;citation_author=L. Aravind;citation_title=Comprehensive analysis of combinatorial regulation using the transcriptional regulatory network of yeast.;citation_pages=213-227;citation_volume=360;citation_year=2006;citation_issue=1;citation_pmid=16762362;citation_doi=10.1016/j.jmb.2006.04.029" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=N. Banerjee;citation_author=M. Q. Zhang;citation_title=Identifying cooperativity among transcription factors controlling the cell cycle in yeast;citation_pages=7024-7031;citation_volume=31;citation_year=2003;citation_issue=23;citation_pmid=14627835;citation_doi=10.1093/nar/gkg894" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=M. A. Beer;citation_author=S. Tavazoie;citation_title=Predicting gene expression from sequence.;citation_pages=185-198;citation_volume=117;citation_year=2004;citation_issue=2;citation_pmid=15084257;citation_doi=10.1016/S0092-8674(04)00304-6" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=A. P. Boyle;citation_author=E. L. Hong;citation_author=M. Hariharan;citation_author=Y. Cheng;citation_author=M. A. Schaub;citation_title=Annotation of functional variation in personal genomes using RegulomeDB;citation_pages=1790-1797;citation_volume=22;citation_year=2012;citation_issue=9;citation_pmid=22955989;citation_doi=10.1101/gr.137323.112" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=B. J. Breitkreutz;citation_author=C. Stark;citation_author=T. Reguly;citation_author=L. Boucher;citation_author=A. Breitkreutz;citation_title=The BioGRID Interaction Database: 2008 update.;citation_volume=36;citation_year=2008;citation_pmid=18000002;citation_doi=10.1093/nar/gkm1001" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. H. Bullard;citation_author=Y. Mostovoy;citation_author=S. Dudoit;citation_author=R. B. Brem;citation_title=Polygenic and directional regulatory evolution across pathways in Saccharomyces.;citation_volume=107;citation_year=2010;citation_pmid=20194736;citation_doi=10.1073/pnas.0912959107" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=R. A. Butow;citation_author=N. G. Avadhani;citation_title=Mitochondrial signaling: the retrograde response.;citation_pages=1-15;citation_volume=14;citation_year=2004;citation_issue=1;citation_pmid=15068799;citation_doi=10.1016/S1097-2765(04)00179-0" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. M. Cherry;citation_author=E. L. Hong;citation_author=C. Amundsen;citation_author=R. Balakrishnan;citation_author=G. Binkley;citation_title=Saccharomyces Genome Database: the genomics resource of budding yeast.;citation_volume=40;citation_year=2012;citation_pmid=22110037;citation_doi=10.1093/nar/gkr1029" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=D. Y. Chiang;citation_author=A. M. Moses;citation_author=M. Kellis;citation_author=E. S. Lander;citation_author=M. B. Eisen;citation_title=Phylogenetically and spatially conserved word pairs associated with gene-expression changes in yeasts.;citation_pages=R43-R43;citation_volume=4;citation_year=2003;citation_issue=7;citation_pmid=12844359;citation_doi=10.1186/gb-2003-4-7-r43" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biological Chemistry;citation_journal_abbrev=J Biol Chem;citation_author=R. S. Conlan;citation_author=N. Gounalaki;citation_author=P. Hatzis;citation_author=D. Tzamarias;citation_title=The Tup1-Cyc8 Protein Complex Can Shift from a Transcriptional Co-repressor to a Transcriptional Co-activator;citation_pages=205-210;citation_volume=274;citation_year=1999;citation_issue=1;citation_pmid=9867831;citation_doi=10.1074/jbc.274.1.205" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. Das;citation_author=N. Banerjee;citation_author=M. Q. Zhang;citation_title=Interacting models of cooperative gene regulation.;citation_volume=101;citation_year=2004;citation_pmid=15534222;citation_doi=10.1073/pnas.0407365101" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=C. G. de Boer;citation_author=T. R. Hughes;citation_title=YeTFaSCo: a database of evaluated yeast transcription factor sequence specificities.;citation_volume=40;citation_year=2012;citation_pmid=22102575;citation_doi=10.1093/nar/gkr993" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=C. Devlin;citation_author=K. Tice-Baldwin;citation_author=D. Shore;citation_author=K. T. Arndt;citation_title=RAP1 is required for BAS1/BAS2- and GCN4-dependent transcription of the yeast HIS4 gene.;citation_pages=3642-3651;citation_volume=11;citation_year=1991;citation_issue=7;citation_pmid=1904543;citation_doi=10.1128/MCB.11.7.3642" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=S. W. Doniger;citation_author=J. C. Fay;citation_title=Frequent gain and loss of functional transcription factor binding sites.;citation_pages=e99-e99;citation_volume=3;citation_year=2007;citation_issue=5;citation_pmid=17530920;citation_doi=10.1371/journal.pcbi.0030099" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=C. M. Drazinic;citation_author=J. B. Smerage;citation_author=M. C. Lopez;citation_author=H. V. Baker;citation_title=Activation mechanism of the multifunctional transcription factor repressor-activator protein 1 (Rap1p);citation_pages=3187-3196;citation_volume=16;citation_year=1996;citation_issue=6;citation_pmid=8649429;citation_doi=10.1128/MCB.16.6.3187" />
<meta name="citation_reference" content="citation_journal_title=Molecular &amp; general genetics :  MGG;citation_journal_abbrev=Mol Gen Genet;citation_author=E. Dubois;citation_author=J. Bercy;citation_author=F. Messenguy;citation_title=Characterization of two genes, ARGRI and ARGRIII required for specific regulation of arginine metabolism in yeast.;citation_pages=142-148;citation_volume=207;citation_year=1987;citation_issue=1;citation_pmid=3298999;citation_doi=10.1007/BF00331501" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=P. M. Fordyce;citation_author=D. Gerber;citation_author=D. Tran;citation_author=J. Zheng;citation_author=H. Li;citation_title=De novo identification and biophysical characterization of transcription-factor binding sites with microfluidic affinity analysis.;citation_pages=970-975;citation_volume=28;citation_year=2010;citation_issue=9;citation_pmid=20802496;citation_doi=10.1038/nbt.1675" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology of the Cell;citation_journal_abbrev=Mol. Biol. Cell;citation_author=A. P. Gasch;citation_author=P. T. Spellman;citation_author=C. M. Kao;citation_author=O. Carmel-Harel;citation_author=M. B. Eisen;citation_title=Genomic Expression Programs in the Response of Yeast Cells to Environmental Changes;citation_pages=4241-4257;citation_volume=11;citation_year=2000;citation_issue=12;citation_pmid=11102521;citation_doi=10.1091/mbc.11.12.4241" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. Gerke;citation_author=K. Lorenz;citation_author=B. Cohen;citation_title=Genetic Interactions Between Transcription Factors Cause Natural Variation in Yeast;citation_pages=498-501;citation_volume=323;citation_year=2009;citation_issue=5913;citation_pmid=19164747;citation_doi=10.1126/science.1166426" />
<meta name="citation_reference" content="citation_journal_title=Mol. Syst. Biol.;citation_author=J. Gertz;citation_author=B. A. Cohen;citation_title=Environment-specific combinatorial cis-regulation in synthetic promoters.;citation_pages=244;citation_volume=5;citation_year=2009;citation_pmid=19225457" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Giaever;citation_author=A. M. Chu;citation_author=L. Ni;citation_author=C. Connelly;citation_author=L. Riles;citation_title=Functional profiling of the Saccharomyces cerevisiae genome.;citation_pages=387-391;citation_volume=418;citation_year=2002;citation_issue=6896;citation_pmid=12140549;citation_doi=10.1038/nature00935" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=H. Z. Girgis;citation_author=I. Ovcharenko;citation_title=Predicting tissue specific cis-regulatory modules in the human genome using pairs of co-occurring motifs.;citation_pages=25-25;citation_volume=13;citation_year=2012;citation_pmid=22313678;citation_doi=10.1186/1471-2105-13-25" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=Y. Guan;citation_author=M. Dunham;citation_author=A. Caudy;citation_author=O. Troyanskaya;citation_title=Systematic planning of genome-scale experiments in poorly studied species.;citation_pages=e1000698-e1000698;citation_volume=6;citation_year=2010;citation_issue=3;citation_pmid=20221257;citation_doi=10.1371/journal.pcbi.1000698" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=Y. Guan;citation_author=M. J. Dunham;citation_author=O. G. Troyanskaya;citation_author=A. A. Caudy;citation_title=Comparative gene expression between two yeast species.;citation_pages=33;citation_volume=14;citation_year=2013;citation_pmid=23324262;citation_doi=10.1186/1471-2164-14-33" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=D. GuhaThakurta;citation_author=G. D. Stormo;citation_title=Identifying target sites for cooperatively binding factors;citation_pages=608-621;citation_volume=17;citation_year=2001;citation_issue=7;citation_pmid=11448879;citation_doi=10.1093/bioinformatics/17.7.608" />
<meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci.;citation_author=H. Guturu;citation_author=A. C. Doxey;citation_author=A. M. Wenger;citation_author=G. Bejerano;citation_title=Structure-aided prediction of mammalian transcription factor complexes in conserved non-coding elements.;citation_volume=368;citation_year=2013;citation_pmid=24218641;citation_doi=10.1098/rstb.2013.0029" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=N. Ha;citation_author=M. Polychronidou;citation_author=I. Lohmann;citation_title=COPS: detecting co-occurrence and spatial arrangement of transcription factor binding motifs in genome-wide datasets.;citation_pages=e52055;citation_volume=7;citation_year=2012;citation_pmid=23272209;citation_doi=10.1371/journal.pone.0052055" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. T. Harbison;citation_author=D. B. Gordon;citation_author=T. I. Lee;citation_author=N. J. Rinaldi;citation_author=K. D. Macisaac;citation_title=Transcriptional regulatory code of a eukaryotic genome.;citation_pages=99-104;citation_volume=431;citation_year=2004;citation_issue=7004;citation_pmid=15343339;citation_doi=10.1038/nature02800" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=X. He;citation_author=C. C. Chen;citation_author=F. Hong;citation_author=F. Fang;citation_author=S. Sinha;citation_title=A biophysical model for analysis of transcription factor interaction and binding site arrangement from genome-wide binding data.;citation_pages=e8155-e8155;citation_volume=4;citation_year=2009;citation_issue=12;citation_pmid=19956545;citation_doi=10.1371/journal.pone.0008155" />
<meta name="citation_reference" content="citation_journal_title=Mol. Syst. Biol.;citation_author=S. D. Hooper;citation_author=S. Boue;citation_author=R. Krause;citation_author=L. J. Jensen;citation_author=C. E. Mason;citation_title=Identification of tightly regulated groups of genes during Drosophila melanogaster embryogenesis.;citation_pages=72;citation_volume=3;citation_year=2007;citation_pmid=17224916" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=Z. Hu;citation_author=B. Hu;citation_author=J. F. Collins;citation_title=Prediction of synergistic transcription factors by function conservation.;citation_pages=R257-R257;citation_volume=8;citation_year=2007;citation_issue=12;citation_pmid=18053230;citation_doi=10.1186/gb-2007-8-12-r257" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. R. Hughes;citation_author=C. G. de Boer;citation_title=Mapping Yeast Transcriptional Networks;citation_pages=9-36;citation_volume=195;citation_year=2013;citation_issue=1;citation_pmid=24018767;citation_doi=10.1534/genetics.113.153262" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=T. R. Hughes;citation_author=M. J. Marton;citation_author=A. R. Jones;citation_author=C. J. Roberts;citation_author=R. Stoughton;citation_title=Functional discovery via a compendium of expression profiles.;citation_pages=109-126;citation_volume=102;citation_year=2000;citation_issue=1;citation_pmid=10929718;citation_doi=10.1016/S0092-8674(00)00015-5" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=P. Jiang;citation_author=M. Singh;citation_title=CCAT: Combinatorial Code Analysis Tool for transcriptional regulation.;citation_volume=42;citation_year=2014;citation_pmid=24366875;citation_doi=10.1093/nar/gkt1302" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=M. Kanehisa;citation_author=S. Goto;citation_title=KEGG: Kyoto Encyclopedia of Genes and Genomes;citation_pages=27-30;citation_volume=28;citation_year=2000;citation_issue=1;citation_pmid=10592173;citation_doi=10.1093/nar/28.1.27" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=M. Kato;citation_author=N. Hata;citation_author=N. Banerjee;citation_author=B. Futcher;citation_author=M. Q. Zhang;citation_title=Identifying combinatorial regulation of transcription factors and binding motifs.;citation_pages=R56-R56;citation_volume=5;citation_year=2004;citation_issue=8;citation_pmid=15287978;citation_doi=10.1186/gb-2004-5-8-r56" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. Kazemian;citation_author=H. Pham;citation_author=S. A. Wolfe;citation_author=M. H. Brodsky;citation_author=S. Sinha;citation_title=Widespread evidence of cooperative DNA binding by transcription factors in Drosophila development.;citation_volume=41;citation_year=2013;citation_pmid=23847101;citation_doi=10.1093/nar/gkt598" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Kellis;citation_author=N. Patterson;citation_author=M. Endrizzi;citation_author=B. Birren;citation_author=E. S. Lander;citation_title=Sequencing and comparison of yeast species to identify genes and regulatory elements.;citation_pages=241-254;citation_volume=423;citation_year=2003;citation_issue=6937;citation_pmid=12748633;citation_doi=10.1038/nature01644" />
<meta name="citation_reference" content="citation_journal_title=Biochemical and biophysical research communications;citation_journal_abbrev=Biochem Biophys Res Commun;citation_author=M. Kim;citation_author=C. H. Park;citation_author=M. S. Lee;citation_author=B. A. Carlson;citation_author=D. L. Hatfield;citation_title=A novel TBP-interacting zinc finger protein represses transcription by inhibiting the recruitment of TFIIA and TFIIB.;citation_pages=231-238;citation_volume=306;citation_year=2003;citation_issue=1;citation_pmid=12788093;citation_doi=10.1016/S0006-291X(03)00939-2" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=C. Koch;citation_author=T. Moll;citation_author=M. Neuberg;citation_author=H. Ahorn;citation_author=K. Nasmyth;citation_title=A role for the transcription factors Mbp1 and Swi4 in progression from G1 to S phase;citation_pages=1551-1557;citation_volume=261;citation_year=1993;citation_issue=5128;citation_pmid=8372350;citation_doi=10.1126/science.8372350" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=N. J. Krogan;citation_author=G. Cagney;citation_author=H. Yu;citation_author=G. Zhong;citation_author=X. Guo;citation_title=Global landscape of protein complexes in the yeast Saccharomyces cerevisiae.;citation_pages=637-643;citation_volume=440;citation_year=2006;citation_issue=7084;citation_pmid=16554755;citation_doi=10.1038/nature04670" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=P. Lefrancois;citation_author=G. M. Euskirchen;citation_author=R. K. Auerbach;citation_author=J. Rozowsky;citation_author=T. Gibson;citation_title=Efficient yeast ChIP-Seq using multiplex short-read DNA sequencing.;citation_pages=37-37;citation_volume=10;citation_year=2009;citation_pmid=19159457;citation_doi=10.1186/1471-2164-10-37" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=D. Liko;citation_author=M. G. Slattery;citation_author=W. Heideman;citation_title=Stb3 binds to ribosomal RNA processing element motifs that control transcriptional responses to growth in Saccharomyces cerevisiae.;citation_volume=282;citation_year=2007;citation_pmid=17616518;citation_doi=10.1074/jbc.M704762200" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. Loewith;citation_author=M. N. Hall;citation_title=Target of Rapamycin (TOR) in Nutrient Signaling and Growth Control;citation_pages=1177-1201;citation_volume=189;citation_year=2011;citation_issue=4;citation_pmid=22174183;citation_doi=10.1534/genetics.111.133363" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. Mertin;citation_author=S. G. McDowall;citation_author=V. R. Harley;citation_title=The DNA-binding specificity of SOX9 and other SOX proteins.;citation_volume=27;citation_year=1999;citation_pmid=9973626;citation_doi=10.1093/nar/27.5.1359" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. Nandi;citation_author=A. Blais;citation_author=I. Ioshikhes;citation_title=Identification of cis-regulatory modules in promoters of human genes exploiting mutual positioning of transcription factors.;citation_volume=41;citation_year=2013;citation_pmid=23913413;citation_doi=10.1093/nar/gkt578" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=K. Natarajan;citation_author=M. R. Meyer;citation_author=B. M. Jackson;citation_author=D. Slade;citation_author=C. Roberts;citation_title=Transcriptional Profiling Shows that Gcn4p Is a Master Regulator of Gene Expression during Amino Acid Starvation in Yeast;citation_pages=4347-4368;citation_volume=21;citation_year=2001;citation_issue=13;citation_pmid=11390663;citation_doi=10.1128/MCB.21.13.4347-4368.2001" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=H. H. Ng;citation_author=F. Robert;citation_author=R. A. Young;citation_author=K. Struhl;citation_title=Genome-wide location and regulated recruitment of the RSC nucleosome-remodeling complex;citation_pages=806-819;citation_volume=16;citation_year=2002;citation_issue=7;citation_pmid=11937489;citation_doi=10.1101/gad.978902" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=Y. Pilpel;citation_author=P. Sudarsanam;citation_author=G. M. Church;citation_title=Identifying regulatory networks by combinatorial analysis of promoter elements.;citation_pages=153-159;citation_volume=29;citation_year=2001;citation_issue=2;citation_pmid=11547334;citation_doi=10.1038/ng724" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=T. Pramila;citation_author=S. Miles;citation_author=D. GuhaThakurta;citation_author=D. Jemiolo;citation_author=L. L. Breeden;citation_title=Conserved homeodomain proteins interact with MADS box protein Mcm1 to restrict ECB-dependent transcription to the M/G1 phase of the cell cycle;citation_pages=3034-3045;citation_volume=16;citation_year=2002;citation_issue=23;citation_pmid=12464633;citation_doi=10.1101/gad.1034302" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=T. Pramila;citation_author=W. Wu;citation_author=S. Miles;citation_author=W. S. Noble;citation_author=L. L. Breeden;citation_title=The Forkhead transcription factor Hcm1 regulates chromosome segregation genes and fills the S-phase gap in the transcriptional circuitry of the cell cycle.;citation_pages=2266-2278;citation_volume=20;citation_year=2006;citation_issue=16;citation_pmid=16912276;citation_doi=10.1101/gad.1450606" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=M. G. Pray-Grant;citation_author=D. Schieltz;citation_author=S. J. McMahon;citation_author=J. M. Wood;citation_author=E. L. Kennedy;citation_title=The Novel SLIK Histone Acetyltransferase Complex Functions in the Yeast Retrograde Response Pathway;citation_pages=8774-8786;citation_volume=22;citation_year=2002;citation_issue=24;citation_pmid=12446794;citation_doi=10.1128/MCB.22.24.8774-8786.2002" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=B. Raught;citation_author=A. C. Gingras;citation_author=N. Sonenberg;citation_title=The target of rapamycin (TOR) proteins;citation_pages=7037-7044;citation_volume=98;citation_year=2001;citation_issue=13;citation_pmid=11416184;citation_doi=10.1073/pnas.121145898" />
<meta name="citation_reference" content="citation_journal_title=Mol. Cell;citation_author=J. S. Reece-Hoyes;citation_author=C. Pons;citation_author=A. Diallo;citation_author=A. Mori;citation_author=S. Shrestha;citation_title=Extensive rewiring and complex evolutionary dynamics in a C. elegans multiparameter transcription factor network.;citation_pages=116-127;citation_volume=51;citation_year=2013;citation_pmid=23791784;citation_doi=10.1016/j.molcel.2013.05.018" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. Reimand;citation_author=J. M. Vaquerizas;citation_author=A. E. Todd;citation_author=J. Vilo;citation_author=N. M. Luscombe;citation_title=Comprehensive reanalysis of transcription factor knockout expression data in Saccharomyces cerevisiae reveals many new targets.;citation_volume=38;citation_year=2010;citation_pmid=20385592;citation_doi=10.1093/nar/gkq232" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=S. Sarafova;citation_author=G. Siu;citation_title=Precise arrangement of factor-binding sites is required for murine CD4 promoter function;citation_pages=2664-2671;citation_volume=28;citation_year=2000;citation_issue=14;citation_pmid=10908321;citation_doi=10.1093/nar/28.14.2664" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=M. A. Shea;citation_author=G. K. Ackers;citation_title=The OR control system of bacteriophage lambda. A physical-chemical model for gene regulation.;citation_pages=211-230;citation_volume=181;citation_year=1985;citation_issue=2;citation_pmid=3157005;citation_doi=10.1016/0022-2836(85)90086-5" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=Y. Shibata;citation_author=N. C. Sheffield;citation_author=O. Fedrigo;citation_author=C. C. Babbitt;citation_author=M. Wortham;citation_title=Extensive evolutionary changes in regulatory element activity during human origins are associated with altered gene expression and positive selection.;citation_pages=e1002789-e1002789;citation_volume=8;citation_year=2012;citation_issue=6;citation_pmid=22761590;citation_doi=10.1371/journal.pgen.1002789" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=N. Sierro;citation_author=Y. Makita;citation_author=M. de Hoon;citation_author=K. Nakai;citation_title=DBTBS: a database of transcriptional regulation in Bacillus subtilis containing upstream intergenic conservation information.;citation_volume=36;citation_year=2008;citation_pmid=17962296;citation_doi=10.1093/nar/gkm910" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. T. Spivak;citation_author=G. D. Stormo;citation_title=ScerTF: a comprehensive database of benchmarked position weight matrices for Saccharomyces species.;citation_volume=40;citation_year=2012;citation_pmid=22140105;citation_doi=10.1093/nar/gkr1180" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=P. Sudarsanam;citation_author=Y. Pilpel;citation_author=G. M. Church;citation_title=Genome-wide Co-occurrence of Promoter Elements Reveals a cis-Regulatory Cassette of rRNA Transcription Motifs in Saccharomyces cerevisiae;citation_pages=1723-1731;citation_volume=12;citation_year=2002;citation_issue=11;citation_pmid=12421759;citation_doi=10.1101/gr.301202" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=C. Tachibana;citation_author=J. Y. Yoo;citation_author=J. B. Tagne;citation_author=N. Kacherovsky;citation_author=T. I. Lee;citation_title=Combined Global Localization Analysis and Transcriptome Data Identify Genes That Are Directly Coregulated by Adr1 and Cat8;citation_pages=2138-2146;citation_volume=25;citation_year=2005;citation_issue=6;citation_pmid=15743812;citation_doi=10.1128/MCB.25.6.2138-2146.2005" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. Tanay;citation_title=Extensive low-affinity transcriptional interactions in the yeast genome.;citation_volume=16;citation_year=2006;citation_pmid=16809671;citation_doi=10.1101/gr.5113606" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. Tavazoie;citation_author=J. D. Hughes;citation_author=M. J. Campbell;citation_author=R. J. Cho;citation_author=G. M. Church;citation_title=Systematic determination of genetic network architecture.;citation_pages=281-285;citation_volume=22;citation_year=1999;citation_issue=3;citation_pmid=10391217;citation_doi=10.1038/10343" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. E. Thurman;citation_author=E. Rynes;citation_author=R. Humbert;citation_author=J. Vierstra;citation_author=M. T. Maurano;citation_title=The accessible chromatin landscape of the human genome.;citation_pages=75-82;citation_volume=489;citation_year=2012;citation_issue=7414;citation_pmid=22955617;citation_doi=10.1038/nature11232" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=I. Tirosh;citation_author=N. Barkai;citation_title=Comparative analysis indicates regulatory neofunctionalization of yeast duplicates.;citation_pages=R50-R50;citation_volume=8;citation_year=2007;citation_issue=4;citation_pmid=17411427;citation_doi=10.1186/gb-2007-8-4-r50" />
<meta name="citation_reference" content="citation_journal_title=Mol. Syst. Biol.;citation_author=I. Tirosh;citation_author=A. Weinberger;citation_author=D. Bezalel;citation_author=M. Kaganovich;citation_author=N. Barkai;citation_title=On the relation between promoter divergence and gene expression evolution.;citation_pages=159;citation_volume=4;citation_year=2008;citation_pmid=18197176" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=I. Tirosh;citation_author=S. Reikhav;citation_author=A. A. Levy;citation_author=N. Barkai;citation_title=A Yeast Hybrid Provides Insight into the Evolution of Gene Expression Regulation;citation_pages=659-662;citation_volume=324;citation_year=2009;citation_issue=5927;citation_pmid=19407207;citation_doi=10.1126/science.1169766" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=B. B. Tuch;citation_author=D. J. Galgoczy;citation_author=A. D. Hernday;citation_author=H. Li;citation_author=A. D. Johnson;citation_title=The evolution of combinatorial gene regulation in fungi.;citation_pages=e38-e38;citation_volume=6;citation_year=2008;citation_issue=2;citation_pmid=18303948;citation_doi=10.1371/journal.pbio.0060038" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=B. B. Tuch;citation_author=H. Li;citation_author=A. D. Johnson;citation_title=Evolution of Eukaryotic Transcription Circuits;citation_pages=1797-1799;citation_volume=319;citation_year=2008;citation_issue=5871;citation_pmid=18369141;citation_doi=10.1126/science.1152398" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=B. J. Venters;citation_author=S. Wachi;citation_author=T. N. Mavrich;citation_author=B. E. Andersen;citation_author=P. Jena;citation_title=A comprehensive genomic binding map of gene and chromatin regulatory proteins in Saccharomyces.;citation_pages=480-492;citation_volume=41;citation_year=2011;citation_issue=4;citation_pmid=21329885;citation_doi=10.1016/j.molcel.2011.01.015" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=L. Wang;citation_author=W. Zheng;citation_author=H. Zhao;citation_author=M. Deng;citation_title=Statistical analysis reveals co-expression patterns of many pairs of genes in yeast are jointly regulated by interacting loci.;citation_pages=e1003414;citation_volume=9;citation_year=2013;citation_pmid=23555313;citation_doi=10.1371/journal.pgen.1003414" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=M. T. Weirauch;citation_author=T. R. Hughes;citation_title=Conserved expression without conserved regulatory sequence: the more things change, the more they stay the same.;citation_pages=66-74;citation_volume=26;citation_year=2010;citation_issue=2;citation_pmid=20083321;citation_doi=10.1016/j.tig.2009.12.002" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. Xie;citation_author=J. Cai;citation_author=N. Y. Chia;citation_author=H. H. Ng;citation_author=S. Zhong;citation_title=Cross-species de novo identification of cis-regulatory modules with GibbsModule: application to gene regulation in embryonic stem cells.;citation_volume=18;citation_year=2008;citation_pmid=18490265;citation_doi=10.1101/gr.072769.107" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. Xie;citation_author=C. C. Chen;citation_author=L. M. Ptaszek;citation_author=S. Xiao;citation_author=X. Cao;citation_title=Rewirable gene regulatory networks in the preimplantation embryonic development of three mammalian species.;citation_volume=20;citation_year=2010;citation_pmid=20219939;citation_doi=10.1101/gr.100594.109" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=Z. Zhang;citation_author=J. Gu;citation_author=X. Gu;citation_title=How much expression divergence after yeast gene duplication could be explained by regulatory motif evolution?;citation_pages=403-407;citation_volume=20;citation_year=2004;citation_issue=9;citation_pmid=15313547;citation_doi=10.1016/j.tig.2004.07.006" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genomics and human genetics;citation_journal_abbrev=Annu Rev Genomics Hum Genet;citation_author=W. Zheng;citation_author=T. A. Gianoulis;citation_author=K. J. Karczewski;citation_author=H. Zhao;citation_author=M. Snyder;citation_title=Regulatory variation within and between species.;citation_pages=327-346;citation_volume=12;citation_year=2011;citation_pmid=21721942;citation_doi=10.1146/annurev-genom-082908-150139" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=C. Zhu;citation_author=K. J. Byers;citation_author=R. P. McCord;citation_author=Z. Shi;citation_author=M. F. Berger;citation_title=High-resolution DNA-binding specificity analysis of yeast transcription factors.;citation_volume=19;citation_year=2009;citation_pmid=19158363;citation_doi=10.1101/gr.090233.108" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Combinatorial Cis-regulation in Saccharomyces Species" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3.cover-source.jpg" />
<meta name="twitter:description" content="Transcriptional control of gene expression requires interactions between the cis -regulatory elements (CREs) controlling gene promoters. We developed a sensitive computational method to identify CRE combinations with conserved spacing that does not require genome alignments. When applied to seven sensu stricto and sensu lato Saccharomyces species, 80% of the predicted interactions displayed some evidence of combinatorial transcriptional behavior in several existing datasets including: (1) chromatin immunoprecipitation data for colocalization of transcription factors, (2) gene expression data for coexpression of predicted regulatory targets, and (3) gene ontology databases for common pathway membership of predicted regulatory targets. We tested several predicted CRE interactions with chromatin immunoprecipitation experiments in a wild-type strain and strains in which a predicted cofactor was deleted. Our experiments confirmed that transcription factor (TF) occupancy at the promoters of the CRE combination target genes depends on the predicted cofactor while occupancy of other promoters is independent of the predicted cofactor. Our method has the additional advantage of identifying regulatory differences between species. By analyzing the S. cerevisiae and S . bayanus genomes, we identified differences in combinatorial cis -regulation between the species and showed that the predicted changes in gene regulation explain several of the species-specific differences seen in gene expression datasets. In some instances, the same CRE combinations appear to regulate genes involved in distinct biological processes in the two different species. The results of this research demonstrate that (1) combinatorial cis -regulation can be inferred by multi-genome analysis and (2) combinatorial cis -regulation can explain differences in gene expression between species." />
<meta name="og-title" property="og:title" content="Combinatorial Cis-regulation in Saccharomyces Species" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/3/653" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Transcriptional control of gene expression requires interactions between the cis -regulatory elements (CREs) controlling gene promoters. We developed a sensitive computational method to identify CRE combinations with conserved spacing that does not require genome alignments. When applied to seven sensu stricto and sensu lato Saccharomyces species, 80% of the predicted interactions displayed some evidence of combinatorial transcriptional behavior in several existing datasets including: (1) chromatin immunoprecipitation data for colocalization of transcription factors, (2) gene expression data for coexpression of predicted regulatory targets, and (3) gene ontology databases for common pathway membership of predicted regulatory targets. We tested several predicted CRE interactions with chromatin immunoprecipitation experiments in a wild-type strain and strains in which a predicted cofactor was deleted. Our experiments confirmed that transcription factor (TF) occupancy at the promoters of the CRE combination target genes depends on the predicted cofactor while occupancy of other promoters is independent of the predicted cofactor. Our method has the additional advantage of identifying regulatory differences between species. By analyzing the S. cerevisiae and S . bayanus genomes, we identified differences in combinatorial cis -regulation between the species and showed that the predicted changes in gene regulation explain several of the species-specific differences seen in gene expression datasets. In some instances, the same CRE combinations appear to regulate genes involved in distinct biological processes in the two different species. The results of this research demonstrate that (1) combinatorial cis -regulation can be inferred by multi-genome analysis and (2) combinatorial cis -regulation can explain differences in gene expression between species." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3.cover-source.jpg" />
    <title>Combinatorial Cis-regulation in Saccharomyces Species | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"oA-GFiqaVsjbm9_XZhH6kW-xHxeEZl2Ir55tHm5lyyM","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460080","apath":"\/ggg\/6\/3\/653.atom","pisa":"ggg;6\/3\/653","ac":{"\/ggg\/6\/3\/653.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/3\/653.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/3\/653"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/3\/653"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/3\/653":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022XjKoVteRFa417qHU0hQ\\\\\\\/3fu+M2hgC84dT8OoxQllNCzM1ZggEm5vlUbCRyfZi6IOLQP8SdSNfax+ojjKc5V3EOOupaRct4uwAE0oRujb5VPjJhFoIqZEVNkac4rNl5BZ0r5HV5T40cgq36ShJ5I9kNAnN4DquHUO0LcVTrUnA2rVivwn1Ublo0UnnLAYZixjKRcBEekHbmcmL10m93dzimo0Wa38FaHm2lf0b3kX4Fk9Nsvio5obf\\\\\\\/e4hKq5qptKkOl3K1h0nlfWpMmg19BuzZMUtixj4i9RxXKJ7bR+vQXYxS0VLeVImTWLcyJpZKwA6zW9H3+MGxi9JiECf672A04D1oSABSou7ITM7pY2EqH4hx\\\\\\\/iUaetBvbT2qYXD7n\\\\\\\/hDifFKlnycWFHc9trdxI1UWMXg1p9lK9lnQzwqsxaX0Lgc+0dH197R\\\\\\\/gLTKJ6TLADGF94iaEzjM6xhcHBYM4LOo29ICFqE6UEgKkyQA0ZHiIvl\\\\\\\/kJIfSG8pqkQK3zmZowS2SAQNmESewrDGG\\\\\\\/qUnOjJPWuPeez6gyVF5MPwpNElRtLN2b9YiFs+Wp0olqt0rjiN0KZhGfgwLmDQFt2Vy1dQtu7m2b5+4pN++15qRC1m0Wc3rHVx7MrIidjQaxWYjBd1hfw1QWsZEatBc+DOnFUg+ZIrxGphUXFypv31NPtMDk5ze9hO5iY3stKSmQgy8xXhQI01XNZzgcP\\\\\\\/3AS1rM76MzoDK\\\\\\\/QwcBly\\\\\\\/7zBETFPLZy4DsC6NHHjcZ7uPOX0oynCJ6VcHBZ45Y8xiMVzn+IwyxrD1\\\\\\\/ZlRqWJlATHTixT+DXqU3rq+Bs8YbGvKqaFV6S+cqR6d4uKpmQskAvl1Ttch1XDdA3yp7M8yuvLQSGFFRY0ytV82xRx0MTTOV7P\\\\\\\/SJJ7l9VBqgiGDx0ZHk2wddEpfJrMG0VCmSNB5w6KWyJmCfQIyVblmF37kJ5X9ZAQRVqEoW\\\\\\\/mXS9bnTeeV9W+rzmvNrtAjuwBpn0oiFS5T+TpVMCtzUuRSzxuKkuTWBcyjyS6qP16LEG2\\\\\\\/BPyT+DGowIpAn5DHKe\\\\\\\/OnD4\\\\\\\/TOuqtD4+sf4DpqLa4C1xEvTX38fYox9ZUZS7+GI0nF3F2HgHkDSrNy45z20ZqdS4SnqyUionH4isErwvoOPTsliAIWjn4zvP0eTNdXHnHboZxV3Gv5cStbwyqD2WCcz4GglVlsEe2UXgfJmeyMwealrqa8nxyYexLk7qg\\\\\\\/tM+jqbFpsncRweBq+8VKUwF+mbl4ddOkgyO69K9wGjm4wUmuFJEvJ\\\u0022,\\\u0022iv\\\u0022:\\\u0022SmhXFlRioOpSB72j+lme+g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002224b27d6ca525c6406e5072d148e8dace404644befa18074c640b596aff2ec980\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022WcljnhOgZj\\\\\\\/AUxUCwJasHHcDvWAf+t9zaEqJoKpfvG8oxTsEVmtaZ3SWWKJdidSWD2QtESrpPy4HMLfSlUuwoxzakdC9u7iKVYP4vc0IUJYjlXEs9ezGAC89Ry2RYV\\\\\\\/CCmoxeNtFwCimBKndAhoW7gK+vN4dxkh+In2vKs4CG8QdDSZJEXFNNERtarAFB7xt2\\\\\\\/nN43hWkyU83sM\\\\\\\/5\\\\\\\/YyQ5rYBct3FTa44G5FJnsf8c2LBTbyMw1ILSLyc23DTA6QaJq4rnFS\\\\\\\/84cJIQofp2wcyt4rg+9pUflM49V7D\\\\\\\/4izrG8adUb\\\\\\\/W7YtGgWSIE1xogmO5yw57iCQY\\\\\\\/nhrFAUMQlbowGvxROKI\\\\\\\/XQHPO2goil7UhYP8pBJ+KFGQP9Ft6WvKL0bHgx9s4gUPCrcAXAvOTDHOIUnflP8tG8y9rPPWig\\\\\\\/EpGKQKYfbeywESiMaT1neCoSUQ\\\\\\\/+IdB1Uxs29gUHK6oecTSbpU3t+I2Td\\\\\\\/GX6Y0P8Zg9lX6GZdv4rRTmLJg8x\\\\\\\/2aE9NTi\\\\\\\/IkJWhWQgKXyaK5HCdaqZQhw1JFy2F95Ip3B9nxIN9IEXt38vTCakxdGuB3dne+BKCxbdVBg3jBVotvCMwYIaULwtoYyk8pE2fcx6IBfZSZCr740\\\\\\\/1zGsSn1lSS6FSxmpA9G3DPlHom0aMON+O62p\\\\\\\/XgEvfDLGyK4jpHq1VYLcMllysYRia6QPFOxvFbCUE3MGsITydhBo3uJpZzzTslK+Aj3JU\\\\\\\/ueZBtGAen9sBo28SqYg4gAj5KEUNDW9qQw4nFCIbc4r4\\\\\\\/esvks0DgguXLewtay8du3txmdjFQXdUOgcvBQguHvaZ9T8CKmIZsY9ueIXUO8R7okLIS6RQc3KJnl53kM1CR+E47zoc3tQ1IuM0KE4LdIoB3bS+H420\\\\\\\/vGbBi0ahVVzqHT3Coogr2uQ+tWQ9w87mj4JPeBOhek0tpMGh+u\\\\\\\/Y+W8wq4XLy6qQ9QWscbw3jCYdFW4dYx7RFwUPXf3VLxiG4vN9zghmOF3pspRQic8TBTs7sAKZ1hdyfuNzqNHS4asReECCZOqfcMQvoGoXYbMknjBXw5\\\\\\\/55pLxyqfDQ5nHhmaUevRTnJ89bZBhu2KeZO0VwC+eL8rbgm4Mo700LvmT9Oy7JjmxXQ3CP7VcymlXtDWwiMdS2SPwOZ8SHJEwlZ9vBvYlmHMP+hPo3+BP4sw8C\\\\\\\/BZenLSDJK6LIFGazDFEP1zpwgB9N+J4R1kxW0Er\\\\\\\/\\\\\\\/Rv2MNFk056Io1uiczS76E\\\\\\\/6VN7AyNwyLs3Lf\\\\\\\/qQdAUpfM6tpzTCa6ESH60hclddeoCX21wYF5TgEuclEptaHZiI+guxDB\\\\\\\/B2wpxzjEWFAO2XQ+tBh4e6R9rg7gz35LI95b1bujiX23NkaA4AAP\\\\\\\/IHJKJwM+GmggY\\\\\\\/yJB6LueY3tVlmNNlv9ZQqDSfcaiSxPQwIjQrFo+ox0pV4322Rb8Ui86cBwaZyYtxHFh2cL697pRZzkoFHfrey87nx1H0EYd1RpV01m8cwXyK3o3Y7Qa0CfZwASpgY8RkkjsG6Nqr\\\\\\\/qjxScZSFPPS4s+tA+7DDpJkbe6jHgISWvIoGTkcoAJPIvLkpCqEO9ly4lgH6aDln9yxaL6S3qNH6wWti+c8tde9KwQiyKnMJq0DwtxDimibx0t92fbHvCCPIWo5Fwhk3P\\\\\\\/r2LvWOgVQKTUmh5AQrdeBS1F5FxhbWSZ9rno3ndfx9188otAjSAmctInKE\\\\\\\/JuphsUQATzXAwdni4wxfaWP+vDjCy\\\\\\\/95oBqz8BghlaHRa03DEzGxBr2ryzq52hNPMwnQR8suA5qZA+sa6ptCWBmgP+KXBbu8Cb3byPXiC5ggcOb9s4no0DimHndzDJB+3lAeDprS0f1fxIBYtKHy0e+qNuKjHvIgwDVJffPysYvyQMhhc2dDlHZgFE041jrlqDyXXDgr2H7VLQepFYr3TnUQCsH1wHWLm1QuGBoQahDscbdkk+iNYW8fC+Ier66Asm6o8xuRmJhJZf20IuJnvUqiaxADpmVo2IzAmqe1ZTL0fhu+uJNJVnO4P+VB5kd+4S9EE6iI\\\\\\\/IyO31FOwWdEcnbugTrWWvCKZHSVMS0cCEwvKt5uBOD6n\\\\\\\/3tfsKBurmjooPlQzn0KmoZWjP6ga8JLwHCEZ4E\\\\\\\/BMuTWX8hACRassmzKaik0Lqwzxu7MYpCueDEWkGazrbrVw6rGET72YlHoj3pbeM86YtJkNFsLAd8is9miuToGmrEtOziWgNhHFEthRmq0emCrDGsiI8qkcvriVKbumaNz6OXGgV9MKZC4VhqPjwLrAYe42EWZmGDwFgGbjqywDGjjlBOjf0y8FOGyQDJPSrkdTTX5MxzWpSIlpoZcUXrNLQZzoHdLvjHZOTE5W5kaFrTlJZGq4TXRMjKCu\\\\\\\/z\\\\\\\/m89jM6xLp0v9rqPFa3evPuhF8pkQDqoIukCIsvLV19CyPhj6eEhu6o321zZsFRpeorxMDXMHMANnngGPRrWfaewo+Shf0AUuUYECS3rebWmcmNNIDIm7ugSzbMIt6yn8KL5+SUlwRYo\\\\\\\/hfgR6EbFU1piDMZF40EgEUag5cEghG+Ofen78w5Oxxn3Adp0xcUElneOKJ81H4t0wpq7km9yhAJMPmomxRImUeIXK0bOH3ajSdmvymxAI1lXhVQJMTD0kUm58TKSfdR6nG7HoOupGZ5uqtCXsFELHSqYQkCoFKEqfqF6UAaA3\\\\\\\/FxpcKXQsHIUOCSW\\\\\\\/DmeFsZFvo41a6zqqJTs8+yiF41CAvDldz2xUiK87r+1Y2NO4chonipuTG2VAw9AHIFdd5iiqbshTKhVpmIIv5rXyvl2\\\\\\\/U2VO9\\\\\\\/OIfIl\\\\\\\/kjV\\\\\\\/dOqanenZdhv0LM6qYZDwHwpYfycHkkSLGQGG0JHwFcaFf121Cho8tRKGUyiMzOGZLMucXLZdCWaeIw7\\\\\\\/F1EZUTAmgLnHSTiaoeetEv2XxqNciTQBD7fDrjQ3pnjLNFfJTfTYvCfwRYIk5NvBC7MISP2Htv7RVQY1CgxHj3LbxFKKdJ0YGQG7ZvZL9MGKJ+brIkM3rrU3eZ9uyUY5eH5EI15rl008nk7WDy+aJ+U\\\\\\\/DmLgAKwPHqG4K2+EOJzrkhf6PUSbBH2bGIpT1Gg4DQSy+e3IKIn8CqxXLfIqhkqunFUSDthCPNRBg5EeIvaVCT2t99GR1I+9NxsS\\\\\\\/lO1zDQvMvvbhTppLRyQgRc+GYY25glelsxmj0dw0uhOlbz9hPMCvurhAxQwYY0TsNF5cEobdIZFwLVpIJ7NcuZsfn6cmS98rv+IkIMI33xYyWl\\\\\\\/GXKPqMuZLtEz9h8xG29Lxum2mKENGb8oHErCgU93CxrlprqMPkFegsT0Ic1I419XTSb9f8mEIK7rQBRMyH\\\\\\\/HW+z9r3EMmNy1x3MPXgX33bo3CWGRs9iElB2NzORKe5aGcdREOmmDmFMuuOueSpn1PrdYicv0ii\\\\\\\/LLfr3v5XGcNbkKHpCGfhrgsBn1k8JwLJQlM2uO8xnssiiYc0Bkmjjlx7SLKV3uLlucJG6xoTO1omjKLA1zdpPIcUZJkzqn+FZodnl5Y0W4yPFt083rlo+Wm98Aw6KfNTkmCsKFoailjilOkME1xt9iLZCRZ7\\\\\\\/Q9ZExYEkdKM+I5UYltOwW5WlevyBrc+VvR4SZ11JViLYp1lCFmgrkOxMNHgE2zBmPO9sqyuQ+zbe+2mQv16QcuTU4EsmvnYQRnMxvMImVCUz1y9w+MuyomD8b9YuDkTQZa7kbPTFL7o8g0yM2fEESV2RcWKeLNvismUNXbEokiG99JrsI24hwTnMV7+zOTJ6cOZkPLBd1Juf7Ft0Mdn+ET4dBGCzO\\\\\\\/azn01p7hLjALiWZIL4Kw4zARmcArYMrEB\\\\\\\/lVr1vXMRBG42Hdf3NRiBPGl9HwlQxBKTcCpBj+gn1PSHzM7O3wW0o8QzGnMbAMxy9RfWYqq4Sr6dvq5u3JZJSfE6fFSGjAYUkxHS8j9Q+Mh3LQtS5sTXXlbpCaARftQdFGGi2ZJQbgvHgLbothN87JhZfS+Lbs2qTtD+ZB6V76QWq9VV3vmLEBiYqEP60LAeYyrjCYTlfV3rWAuX3sxyYm9sX84gR2r08OYYv9t57rL5TzmXAprqkkx2uxYJN9Lej2\\\\\\\/jzKm2IWmKmFeGJyVQqooPZNBCcBeYDpAkx5+XTxEDso69Cgf\\\\\\\/7sLi3gNZAFw7ihDhG2YALhqGmIMLEqaXWk08k1Kdt8fTGS9tD6qOqcQyZljwJUuxAM5FNJF\\\\\\\/0UnkrVMmCl9NE+EriN96bQ7xcZ9+NcO9hocFZczsnfBcDPNPfdA4XLIoQa5C6FtICCF0vRUS7cyQ33+yD\\\\\\\/5GViYNZssUu0TWm+Wwk\\\\\\\/EymCn7awNFQXZXMd9t0YzqecU7ByzkRoB1NlfzoDKZKM56DPE1AfyrOR37c508mga5IxOzNk+0EEUJ8OLpqgqRj5gubt8BnQNCPsHaGigpkzJUm9wvisH6jx5cwHCZBKjX29CtFyKBI5C2yIJcZHBv3qVOgErQSMfciLN0eUoGQ1OWfQDmLm89YJsx1ZYrCGvvSNY2f7AHo2V\\\\\\\/I0KbGyFeIOKga9UDUcJxR1gVbHLcpiKOugXwJfCsTKZoMHWllQKCbz3YJieE98c\\\\\\\/RzVwuakPADAgYniO9z4R7YQ9e7esf5q8UFkD+prkHE8Tq+kbjKHW9HZZVbxVbtKahYmV3fzenQTYmZE41Q5bVhnTrU5HNft6Wj9G8oZamTdTpXN2OnE8ljLG69djbD+ckJbZtZovDq\\\\\\\/ofM7\\\\\\\/jAGObwOMGDUqFV8vbgwVJMt9R4EF3Xs1y+jVlCrex+27dAg6r1+Ljij9zK0DnfJrRceEshMuaF5w+IwLO4i4cLBNNKiVDZxHtMN3XoYpLNOG2jvAGNMIz9ociAqsXHRcny5wt8w8LEmvFkdSI6QuK+gpKqOM1FEaM+Gg5ZNP7urtEyRXkiYBonTeh4HJZsmOrZt53tpfMUrUvM4Abi4U\\\\\\\/D\\\\\\\/\\\\\\\/06yBG+ZyVHmnBYil+b\\\\\\\/7ElELrLxr9XGdKa5KoKOcQa6zylMqd6EzJjh9fN3nYrVtOG9hllTHXRe4TMmLYkYsQxrFUJKP1Pv+oIQi91tSAq1jFPvWqgpfHoDOx9xQ3zLFsyCShlnxM4eR1odCZbr+QXkAZPkrc6Pf7KK1iNdfuMEYj\\\\\\\/rcj84kr545Lql92JkmY9S3JOgpHy4B0aPHyuktNbpPrN2PiyKcvqrfgN9y2j2bDLQPcfrmstEylaZcsOwP1bBHrlcfthjorh2NjiExURcLWCcvaG0vNUI6WB637uPHDLxm+g0zosYp1Duae91G5RCPsX4ML8Yx6ZNlsYQVUxtKIYBFDh56htPEK3yDxCpjrogd3Tw7k5YzKJzTKNMUHEH0+YowS85E6ObwJ8hpcB2oMA8VIId7yFeu+SutWsCU9tftpj2f44vMa8EsCEdKWvRdLzex8TYsf2BuozP8xsOcC7yWpdAFU+XOsEl3+nup2tvpBKXgtPuFZyHAxL+Ml6mJxS11ivNGpGn+KBfFddnrzo\\\\\\\/ZkuMluZxKl038s2kOliJeEvwsTwaJvvOKFujht9iIQIMfmF9XHmYxNUSOvyyRcwfyJNh9Gy6xMkitQ6pMFb4HWE9Wg9H+ySWEn\\\\\\\/VBbs2tnFSNtqTVxp45nuxEhPWF1V6G5yz8UMQuAqW4McvfVrP7UYChrKfzLoBJLbNizAlnzVQEKpki7KkiYk6DAswX0jFePU+4Nmo2jZ1fXdNTMnL3CRwbRdbti\\\\\\\/MjX74eV5JFmguJnTCkCKnsyHp80HPFNBbMv13dj2lce3aWgN4O51gQQes8q0fIRo1LfupOX1FhGGCYzcIeviY+9HG\\\\\\\/sWSTsMX\\\\\\\/FQiv\\\\\\\/rivYP2tT5VHkWHuSCTvCxxtGfwoBO69o4K37zyNq1DTqzYb6OkuoV6gQ2d1OFMm0RsU4uRTfx1iUxJaAmGO\\\\\\\/BQfBRwud+fu6m1\\\\\\\/klcocV273MpWDfVcFdiwr+Q2fhYRVtTpY6iH3EAXwVMDRMd2YnqYc3SqUBzB0mb62Gjg+OsTGvVRs\\\\\\\/Vk5PXeNMGOKHBzhfuuY4NnzPhCaGu0xxeMe25AdxhBm3+WMEb3trgOKen8tHUSlAzeo9l4OEfCQRPAZAWrKNwfQ3yB3\\\\\\\/G+GBwRsApJvgo8+EGJ4+5FKYCiiWMhsPC5oYZbRKHDxlP2IPe9liaCPhQKwoO1PDfdyiOR+PdEV21u\\\\\\\/uz1LW+oNHuxqoWvs3eMqHtUnjR8DxBFBU3VNx0I2ThKUZryhgpqAXoE86fxUmpmqeFFdi+yc+sY8zaaNDjZKG3zWG72mKf+aZVXw\\\\\\\/9\\\\\\\/eh8aWS7eZML7PTxXw9FyJrXSK19O1iwuJme0G5AK2GS5e2+oCXq0SuV8dwGk3QNfrd3cs04UC43PBSHUz+f34sLn6EWEqoIMgLO6GICTmVAd1v3YgkrUskXMXvqqbad13H1NioZ0BJFsjZLo004NAPmXlEjElm44Byc65WgLkdE8u3YYKrvakQykQBhtZTQhPqdY9UHR4v6tgvyGyPrGwgTaZ5+C7U5PcLCLjZdeUpByV0kwPUHsKb9UHtbJvBF0bxW4TpA9OCNsAUpwjq\\\\\\\/8oDdXsxIjQO1m+r3nwxhChPxUU3Qdn2hh086LzD4aBHEbp+au2sYQ\\\\\\\/bEuAQjNXFCz5yiiFF843jWJgbZrSHre2Pj6iwdAoIfWbxaqPHcVigtAMiyJjLuNRjTNfry2tlRirmI0pRyksisblqEt+MCf2dPumdl+V6UPmy45n4tKmWkMt7XzByzvZrKP8+B0mGHqot6piAG9ZPXWU7y0vJD6qjC9Ogg9bqOll2Pzv7uRJ2wJ0g5Pu0mC1FUAHaqWfaZLcPACR1w1JKmNjuaH80CFs+gtndlECQtHtdNGTzqj\\\\\\\/Car9zTV3qdL\\\\\\\/AnIwk2GejSi4Ff+B6kV6i6CAJpN4VWuQfHfmneUjUowU+XppZ8mqxFsRBev3DUn\\\\\\\/JSRxXXCcc9ErvFIF0fV2KP7WGDfqQssJLp8+AkSMqFz95w0hCtBKslzw3ghkxPyMxZvCR8quq+6gOOHu\\\\\\\/4EYsGGdYFH+aMEQYIk9+F4+gr1uU1P\\\\\\\/u\\\\\\\/fo+R62m+TGChbn8vFN0esM6q33xBcNye06naV0TUl\\\\\\\/mJG8x3EyzlYGQ0V1fvvGKRCwaGkEj6tcWJCb7MENBm03RAIoU478MsrSWWdjdAimaZW4ts5ukEuZ1+TxdBLu5J5dJwB\\\\\\\/EFXfasQyFGzl1+Rk4Cetpdq9NqENOsZ4V0JuxOXC5\\\\\\\/GH8hfB0BIphu1rw6KXsTtZIGsdNrIyrf269pXC7HeKWMMorOVNmn3xq05Bv66+\\\\\\\/k3FQeUDJ8jguokMNlmrF+m+S4BVnp\\\\\\\/JFIiSYvjegPOlOnv5ANNCfln3HUA4mEupQBOoPWs7d946VkksLNBk7rqLElKqx7fTW9hjzrFeKpjgcjBk2gBWtY1nip4KBZwZVzOtJFRO3xNMnwyHhlnqaDgKvO0E3Ojs2wBszCTLV50Kc0bZIVgk90HyPoiBMWWCkx76140h1rLJFwE4BAsBldexftiWM3THjwUGi4elAVXkwTJlwR8lHhl8a6E8lOYXWdy\\\\\\\/5rPaSmOOmQByKs7mFnRV4VQrNjAcpq3Zog2DEWVERo4kJeC\\\\\\\/hn3XZcM4\\\\\\\/bXDA1jkbM5qKxhhYeC0VYqBjCxQZt7ClZtUn4wnrQdYOD95i9sfZ9jWQ8BjigWbU9I4pAvzPALMuMLfLYJddt9i4dSiYd77tI+aqSJWgeEUqbSHGEc\\\\\\\/PXXmoE9Y87yw2rbow6Fc885fCeRiRI96svNaMC7i8mfiDl1zHMqmZlXFBUYjMUPg1Wz3XmOTuMd41NJBm1oVYMVmd+EhycDP5b2QvBX9bgXLNo7Gf4zMkmVluKfr92Sw\\\\\\\/20VrUkn\\\\\\\/4vKxg+QYBk\\\\\\\/uW4Qk9BuR8v98MJ3njVVqac9U\\\\\\\/l6pHmBkNdofdbiTaaAuE2xgr89T3q9xCEABTrSPEFllVvuSRrp3vyMbxAnnBrUIwduo9qyAh4D3lsUjrX2ztuVYB\\\\\\\/doLrd2qrTsrjeeXK27roRj2wz3jFSNjq62D4ly2TcdGvUyp3Hz6FnBBRU+IQrmqP\\\\\\\/o\\\\\\\/zw8qxzg6TRqIKu+d9VioS2O0lBUphEN1xhlsN7n\\\\\\\/Vcz1m3hgNwoUnVB3d0ThVgG0ZKNdS9i9wE0akxaOmg0c7h5jT97pVhCFGg4VwpPk1PonvEeemQ3hkrLSiniGkGg4rnsfBqUXy7MgiMH2ggLz2efcA30e9TBoPJCBXhZOPNKZAgtL0FlkXuOVisVYx\\\\\\\/WmCK0zeHyagGYagSF\\\\\\\/UKaS\\\\\\\/ndudx7eji6pQymnhqqojeqQB\\\\\\\/kvuvoeewlKDa5nBeWaYdg5DOmzL3zIlnTkypUqx33oIsnv8kk05rZVAj4V3m2nfdfHMUhfN4EhMd3Ul3\\\\\\\/1hAgMSISZS0IKkjlTW\\\\\\\/XAdl5u2N1azUNukYCdqzUvkSpec8h8Yje5+wJ\\\\\\\/0YS83bx4D0hOQpP1NRoXtCe2OUVNgtkkSScDFc2jGZLK2iQmt3mmxeAuOgHjR65m+bxFvtnhYA\\\\\\\/dJ35IDBdMPWcXBWGC7oZnJkGHKLnDUfqFP1pudZiuwdjCmMOmExMIWeGxry\\\\\\\/EM2nrfiXqPVnQM6kI9XVk+PqsLTnxQI69u38xVdOgLJMkmExSNjXmTi8zrbfoRkS8owtGqMH3F6WVnz\\\\\\\/6ssUo1awRZltBbIU5fMB+RL7KXkgc4ebyk9pvb0SoIG+6ZH7f53gxNoCnLEdS4aU59qdy+Oqs2umMMN2j9xli+iqDX6iiZZMZqenSDeLCZFgx\\\\\\\/zOzLQQD\\\\\\\/WyfDxdS\\\\\\\/Yq7PFoYM52BT3aGaDhaOdminEy\\\\\\\/SO3\\\\\\\/8njXXzFqAp7ykBhAKWZFzom0eK4dWXG5YajFN1j6787gRHadeuBqL4C7yCikCD7lysXAxFdiM5a+9izx8c8LLu74trm0rb6aegH+ctFd\\\\\\\/e4S1SUttTGjk6vACjc5lnY1NADlVbEhKKibmLoXKGDrAFeR3esSSjoHasQEOgZ1pv0aPS\\\\\\\/T7Z7ih23FR6W4UDdwIPtAl3dQSVzKKHaa+u81ws6xZ\\\\\\\/Yld5RSdgSh3+u+AGPQQzKg5fpilrYVFbIOpHLK9CbaaQOCdOORaNqMbgCDcgKtog9641XyZLg7anSjKlNv64F0tUQqAdrP9Xws80ocejKBcSGZaWHQYjS0Y2BFvrYQ1ddwIN2B6Um\\\\\\\/45+RE6YjFpdS0c6RRnsTK8Kxs1uC2+vUTSpPBxVizHd3oGd3kHhs8Jl7b2xlPwTZ8egrnqcn7d1IGZeSaK1sdgq1fRnZ4cH2vZJDLGgG+N8SEVqyc8VxGZAGpiBWUtgsvDlq5AZsUQipcGHHWxKcBeicS6x90D0arz8RotetOY9jEzANRGPpIHdBq9h28sChWnbr6gPPynBHI2stugojOrJZEuGFIuV\\\\\\\/RG1YcH3xqpOAZN77leWd87pt6uTtxzFCok0aj7ccmLPplliGnlHoBIYbCJ8ebMPylQcbVpNVGzH7ElrETXElvQNXOA62Qm1CV27cske5f4NG2nTUMO4GW3l50lPwo3ZX1dbDMBNodIJM8PQTQeunKs3AoTsK\\\\\\\/22RFsQbO20gZ09aU8UtsVkhwXZQNPH\\\\\\\/zHNlIBnRyBKJNsKHNeoz6O226nh5dN8239lb0KYrh04yL8lPlMme0U4LnScUXEpe9fJex2FFfUhNXCQa5uCsQfMPsYYt2ZZlvC7SnfHCs10UPhMEeEYcnKs8opT\\\\\\\/EVD3blMzEjZLUeT01GBr3RfwgNQm35B+3LLvbTbKe5CZ3LuV2gy9cSlsrJ52KqrTjmz4+sPVTK02yTtHlGS3f0msOpb\\\\\\\/hKxlmfWXUzI85YBsI+5RG4fLuQazCpvNe67fP+ukpFq\\\\\\\/FzCAKpNcD8UaF6K0jwYFt5tdU7ShTFWLNRfQQiny6zp\\\\\\\/T\\\\\\\/SMz4oHdECXOKbv4JLNvzF8D+9GaHkn60o6gr0ec52uKSRTPYqXdWr0gQEcHs2Zf5K++ecsJcbQcE5RshTupkmh5jQY0SjBdiEDELrYEDaybJhrIEJtoNfpjteE3npjJ6Z8bi+Q6mLLea13xtgx4HBZs4CJCr\\\\\\\/6emCHeM3wStKgWiP8ryKFixX+fO448vETX6vKPgiVzBosiw20pP\\\\\\\/SkWQJpTGkD+\\\\\\\/Ko7UN+DEbYoLanL0yyqC8s9qCaZA73mJDEr5l7UbZTd1nzgl0Uw0\\\\\\\/pmSAwXEvfjw67u1GCNBSpY9eri11MZVpVVqvO+SCARTkwu64Kq0pIMOJSira+H32DrRz7Rd2KBLgHXX0Pa5dXw+YqWfMOdH+\\\\\\\/ZWOi\\\\\\\/uBiaN6YabmPZ3qb9DE3J+\\\\\\\/D+tclWbGR1UD9laX2eqK6s66canH0epL4Paxa3seEA6MNoGGUb3ft60YrcIdaala45qGZ3SfLayjcthunvhKcPhaYcmlAO5H7BFsn9ioineYuYJaippNk\\\\\\\/GGrUI0FCVOR4qqhZYfFj282MrOuOo3YBY8Y4FfMzA8UHwEP\\\\\\\/9XdpWxDNghlMZ8gh32bQlJrbJxwlspE09kUGwYTZJYVlqv+S3eUhQbNI+4YeWbCYdY2wZwkZzC6faC3Dd7c9D4KSeD2IDDYEe81LTOtBEOlWeeopiZFBvdWEgFb+F7knBdMCejRPFCUtiWU4Ws0In8FF\\\\\\\/ueWrEiB1auH\\\\\\\/WFjgOPGLH\\\\\\\/ctWDZlzolO2PomXwiJRNjcFk7wWWH7pfYAnWog24WcUGFMBAJS4KM3mDQQlf60HLj+gA2V5hmIz2xHJSEJ1AwERAtJ+N01rWCq4LqotIqlcQmzkzWUHT\\\\\\\/o8kfOZ+GAmE3WsCunmBbzAsNGUyA\\\\\\\/fxplt8+xN1ubdoGvM0g0wSzpyi6eHVx8e3VfjJQY5NCkBZW22jNUZXBPysPmfvnuP+huk3fke\\\\\\\/y3LMfi9juXRUmMdgBA9wuxlLKvYeFli73sZEPvL8xfsnFibsWhhK07piej7svhxCR5kTS0BXdaq14DneF71oIHgyUiTg93ul8iVMPeLhTPd9Ke2oh6p400gpT8IbL+8gN6YHjfRFoEX97xY5ASr\\\\\\\/OAYJatcNAl2IFwtJSbG2CLzdI7xNRAv2yiwbBxMNE0BEGRXQ4B+UJk4SAGBIPR\\\\\\\/+cOyNrt3HfS4uiHdQIvNbSe2nuMW19nqRJYUMte99Kv3OGzvcR9vcZD2e6LBqJNt2959UKA18BxwTF6zQx3KR3LsqJvn1f86RZL8\\\\\\\/pPYoSKjZN3GfzcvvpVA4EVSR5bLrycAdD1mwzENYowMqz8xYPXKLv8qYU6PcMOJ6R4Two0e2coBWNDOXWQnovhH1s0+61CftYnSDdh\\\\\\\/mTyEflJ70HHm+lX5j\\\\\\\/3WlKJs8owWlpviKco5qmMBI7qJOGHFJfMqB5y9xE\\\\\\\/Zd4gCDUdaJm\\\\\\\/MPF3WfnPp2Xg7zu4JOhVXb5G5M47EV+ykrNZad821uKRkkbxZCvZBBwWqtMl8f57TETrOpSMfJrKWCy9fuiI470dGeC4DStsNfq6XhevA2IWCGgNAF0JpbrnQHXsKY9OQhMEarSQk50dhFL327KDe7KWoWLR\\\\\\\/LXmUIk51o5gCIZY4Og5eG65Vbgq1BkjBfeSGEzy7ndtTYz1gs8\\\\\\\/hT9FG0\\\\\\\/lkpW8lKvofbB0NDvmQIijkq+1gbh9aLGFCyOVO5ETBHRiHlaNPngHnxL4eoHOA6kYjKErpm86R8\\\\\\\/vIvDX+oOE62ILTX3FRV22o1WTWmXY2WuoE5dK\\\\\\\/qXwTXpJ4IdqkrbfptpnhxZnJbyxXy4ync9ww6qdmbQjdZz5g6iFz2iQKCKQGVU1EBERiGQkr1QJNDH+aewSW4ma3hpLH5gYwE5ze2L9XigDY0WHvQyuuYL+x8leO2MI7RxltVlgm9eZyzh08x+Zx8Gbz\\\\\\\/k32UlfQ20px5MT4g3pHkE1kFFdarZQrykLHiNYBwTJmdvJDrb45k\\\\\\\/xld2CuxGwbM8HHPzZwRZphUyhF4Zk4QaYId7+yDDDc9\\\\\\\/Jr0minCliLExkEELecrSAdnjTW72KWZtNrlObO8BNt6gFcMQm9wtMT5l9RtiJ7HkrDYSdoZQFnDoCy3wdPYPqda+gHHUGwy8JT5IHT9S5UUg5brPjoOO+Kvzin94kUtUJJEnDYAjbcAKukyjrjmhdzzJN8Y6TgLw+fxtuZePtLUgYMliNUdB061\\\\\\\/FuOCuNMtGoS4dAOTtEu6SQxeDyjYsYwR9HAbAiLMXip6jNg0GJno23m\\\\\\\/7CZr7RVyMjBEz4hUNOwO3OMxJrunkPJOZ235N5402E1bpCJOYAabYYm5WiNEFx9ijQCv5vaHYSLJx9Ob4ywsB0raTg3lL2+kwgkNST6JP0d3qlO4a69Gq3pbtDB3k0qedmNY4qD6V+ZOCx8n5Goi350Z9coRwcOo2ArzoIy5St\\\\\\\/lsxLVs+Laov180\\\\\\\/DYeh+DVazFuTKKw71CU01RbXGeTZAf\\\\\\\/D8Pg91V\\\\\\\/8JaC\\\\\\\/KXxWdyNlE9Udkm3wDFHqCzAq1QaL8Gg8kdrk3KT30zv96NTFgnOPnoeVr6560r0eqoclJ+QxCED\\\\\\\/mlNH7DoWWOl53xRrJ1ft58Zg2qyrNqUSr8r6tCJdiz5cMx51exSEE+b+tWFDWR3\\\\\\\/QkHlP4HnJk6PKt7KBprvA2NTjr\\\\\\\/UsAma5xJqYuVuH+te4eyuAAzTQir+CuC6SAxcgjjEzwG7\\\\\\\/yW+mnuQYP1YIKrOq4EVtPlLOSir2ZfhL\\\\\\\/a0FCQQN87U\\\\\\\/+b4skBB73UZnuhbu7eaA6v4lCfyCYrtr1dR0ZLbsCk\\\\\\\/Yqh44Qz2fTwj++58yxAbAq9ltfVsG04FohZw7Xv1iyReOXM94twqvCuOGgbU9kj1AW+t5L5AWVRMikjplJWHDQqLNQeXFfmhyeUFSi5tzM2yu\\\\\\\/C6FI4ruKwdyBIqZy78kY2Iy1E61MdJ+YfJbM4iQbX5K283RjakZpOcCObDX12lX2t9hTVUylXQVMtshE8sh1C5AUXjXMrKJrZI+9jU4lhr\\\\\\\/B3g7ZGz+SRc4J16MMCNIezsa3Ncj2KSPCXl7k4IPrMFAfvz377+ikgKgRT4dutElwIYhTTNaa7CUHL4ZG26CrsnRiMfXCxBfYtvolmWXFtXaoAVtxDgiVVXLM7BhwdWGQKNJyrQFq2h886MZbPX\\\\\\\/lcmmHNpBrGJi8EkWVciofO66ziJKiplKvr1nlWBYpA5cM9MiMQinYOYTCZ5HHpJE+stx45hYR+7VgU\\\\\\\/Rlk2Z7qTer4FosbmJ88pTddk02+l3F5VD1knA2e9MMk7pLeiQjJPqafcb5EIcIGRMG5eJqrOa+j6aM9Nf++vqq9Uz73C6gI1lTv5POQibrAL1EHxBhdKvBxszSkYVmMCHXpmUXepkqQIzLwx0V5fJwT7xG+fGFyGCyKVn73QlcHQDC309PwiFDijasMLvaEGsMKOVwGwFYwmAyFiU3RIyNVehBbI+JgfQzPoQ1ZvlcwpyNMBkQNME91YC5i6EvGQZQXV1+8emtA1DEiIZcIOVGvgl2sj0dq9yXTFTxMl6cdCreqMcnJqy2xyYv6djC9F0ua01BjrzETXfSC68Glz6BNxYWLed\\\\\\\/aKbdX4A3\\\\\\\/5UvejAnLCeNvL3z+l5yuTFG4oXJxjbkikrnBCna+NwGl8SqI1nS9E5kls2SKVSWGC4I+Se68yyjzIrp3EjK0PIxnaqewUYxIbyOYZcVd\\\\\\\/NIZObp2jqt\\\\\\\/OW4R6rNicr9NEEhNo7o72N4L0GbhjupA8M6I8431YVvpdCionVrKnNMlgsTt0ZRopOqAZSLSUo1G3vBE75kfmFcYcwXTn3LdumTTgujKamf2O1Kf+L\\\\\\\/8sirQnN0SBKH83hhK0VK64bsQdWKhvE2nvzP6ijD+Pq4TEzs38Ip2a\\\\\\\/QW83YkS\\\\\\\/Fp96MyXTvo\\\\\\\/d09+wZ7SpBHmfiTo1H1s3fgAIVmAPcn0JOMtxwc96Xg8loFfGEpG2PFYRqAJTfpc4kD9cz1\\\\\\\/LFmhKZ6ictlP9d5QAy8pJrTo8ehaDq4jqWRRC75tJAjt08mKmm9dDITHNtftIXMco+bAoaHyEh3Pkv\\\\\\\/UcSH+CbssqBhJoNq4UdrYv\\\\\\\/34Q4Qf8G5BoF9weab0mKDNVltSgzkbFAnUgRX40E5jd5lMu4FojckNtoq5+UOfR6ec9jaLZr9R+qRIzqZCuV16q0IvChZuTnEgcM2E4JHJpzY1XGf+sXcBQVZg1tBqw8VJo+yPd0b1ZV8y\\\\\\\/HGR5KQonRTAj54yYEgbNWE\\\\\\\/m9t2kvIYhw5gTK9m1ZERPrl0QkWkUg56Ag1OEMYeWrOFclev++jpa5lCSItKK1c48QcNCVU65VrpCQueRnt5cX2giMYkXQdM2hBd9lGz6iBHG91zavf9pYHCi2y9tq7e9qoeqez38ZJRHOSOhvpPqOOQCf0oz3xZEtXGshGQztmtgZGNltQnMoAjKUi+rDiUjXP9QOM1\\\\\\\/vMKhLy8iZwUQoG7NXVVMIoTOLlqHmF0tLKc35Low1ft9Vn+iCeS3FM5ableBXtyxxtSu52EUdA87Nu6DJ4ZwFbHzazwXYwgGMVaPPHH9rSt8Ge9wlx2S+lt35amS\\\\\\\/LSj45o0gPnrScWvhxXiLZHR\\\\\\\/h0kt1bbK3AIDvDfoPN41FffkVA\\\\\\\/ly3izMQJ+goBNXTwqNoL7856Ap2p38Ud\\\\\\\/x+IKSWhgpBwbnf8RppyP1qF7HgjVJC57XM8P\\\\\\\/U99l8WsUaWldQ6ggUfVbF71pdHgYR3JQ7ZbJpqCVd\\\\\\\/jIfoh0RjIFd8RmpgaRiwhrG\\\\\\\/2OVi0xRgSAH3MyoQ0lSmiHG+\\\\\\\/HKniwfo7XDTjKWDdoP4efGRpXm+nRKoRkExfI3EsTI+ZOTyABPdR6AeEjSlPqxolpN9qNNFFyoCTCYVU\\\\\\\/eQZqzuVejwQoJVVZ9tPGnOXj53bT1CpgHwOyXxjYpf4YmTkilBAHK3D58XKA6mMUFQCDZjTgpE621eV7YtDlVwJq8ctc7UN\\\\\\\/PHZ96fJGG5y03vdxJotizELRTuZDxEV69SmCaC4904m5xS4dwZvy9BP2HDT16lIXreMX3f71BN9fbJh\\\\\\\/6TYtN+AK6OZAy5mh7csCnCCN5SRQlXMrJhY37NcjAABPvDu8rL5xMW6MxsiEwDi60XMBnhbPdRAzBrrmCg5VhYy2xl3CUWx8TPf\\\\\\\/c6KyQlYrmQHzLpaOitpoZBUSCRZn7a4JkkFccwuP6wT3Gpd5bEpsLq7uheLSEUtlk\\\\\\\/YqEQUJIe2AtoyPpj+4\\\\\\\/jH2sUZC9QJoDuv324MC+9zbrcToPraQzpT1GzhR8vkcmp41Yq2lbwk2UTzGS0n+Zp9OzXhWkCdicZn5zamELzC1cMZ+EOjKWw7Cdw5+ri+fGRNpVwRumqHaRLxjoTqGIP79EsTO0lyzbOL\\\\\\\/M9d4vt\\\\\\\/dd+vxqTJJHwAluY7Yqxo9iHM9WFlIQigARm8c4kY\\\\\\\/MSeOWpQ2EqZXT8QZYfxZ4C+o9qzSUrEdglgjqji87w6lj+24Hh9LcXVpPI4p7qTU5Hg2luE0m7RSbwf\\\\\\\/seTIfswWlJ+zvml6bAUmkVhC\\\\\\\/Q1q5xYW7rr2RdHERVQjQFoX36nQvfuGNCjRXs321APf4wZrk1CavpbhONMPCh5C4HeWg\\\\\\\/gU7uyMRi4Dx\\\\\\\/QGAlMUNO6KLOBLUNjXRtYT0RP+plKKC7vUjxx8FYeQ8Mq2tgX3vQQ2tX2SFFFQyVorW91sSRxrZgPCu1kPTN6lWfIeML\\\\\\\/VeVHTcJfP2b4Q7pCWwKzOjQIv6eRBT0CO9sQMBvPLLif5HZLMyj4Z5ceMUN9nwLvywW4JoULpIya6XSc\\\\\\\/TDKyQKZ3oSOvrw8VyKiWfFTeLygLy7WTPg9NTuR\\\\\\\/wTKcj98v\\\\\\\/4B\\\\\\\/h+Hs8wtOz19JcgZhEF9TxDHheurVKzlujN6HVttnxhm\\\\\\\/j6FDYQ13z\\\\\\\/XqkqDFp7gPfgmSCScWfiVpNsW8ADOSFbQZlb2Pia5HfAqh1nY0HCUPRSg4H8rAVdjhFeg7ye4+DoM6v9FIkoO5sVL8rtH8BYpFgS9OuhrHs3OaDJD0MG\\\\\\\/K\\\\\\\/uVJlg0pqcJALne0S33o\\\\\\\/+FHK9wns++o0BvmeW6ZIYdTCggQhIKr\\\\\\\/wpqqy0CKN+Q0VWjiNWuaQMM0t59PZIJyFA6Icl2hFBQgFkOBToebUYMdlbaF8oPkkcMa34DKL6kylQA\\\\\\\/XuT+q6C3mjy8reqd5+nkQcEqL0gCJ\\\\\\\/IvEPVeLdOlHSr8qXLxEm0oYDzSFd1sknGqFcyZxOw0IOEbWDY\\\\\\\/HWq0Utuv6+cBSBiKl+vcBVHa83+R8th9RZgmxJVaC6o+oB9GF5CvlQML6OHC8aoZzdrbhCJ1gvZLmFUlKtSerEMjqmJUYP7SEeqI45+SmkUIWumqbg9D+aIrqrLnZgk2fX1BuN5M2Oe5v8\\\\\\\/j1AxhLlXhgFrKbzdhtQIypaxLXMvk\\\\\\\/kxzKPpYO9oscMjrp4EdNrv6HXjGVpyhHSkwYx+RJuxgOTaMY+oJdM0PidnnehRJFSsxgIxCYdmXbgWB88VqqBtOUQEQio0p0g4r\\\\\\\/UBvGdiKd3ZwkLOQ+EDcoN+8gy8gvsj3WHsonTlVHDE1nmUOProi3xdPc9cKIZak0I+EnU+S93wJ9EKf0p6GErEf0CHMxvxotAKMYI0R04ZBC7A+8v\\\\\\\/kJpUk44uY663dpk9kG5JG4QpTi385rjeI9DpsFn\\\\\\\/ItsPi3icS2CxDnwZCOKauFnClRcD0hwQDzVfT1fHxXTU2b30pZ0GZxncqCo4hQk3sx1pVAUBl9pV0XnlyO6I85H\\\\\\\/09wxjkv1vtp4QHV86fp3lGgbTCuVpOX+XZeOhdmuk93RZ6D05jd3pr5ntCPa7YcGiEC\\\\\\\/fkW\\\\\\\/Z6If0cQf\\\\\\\/OotZwjnoPyhPGvVK7Mq\\\\\\\/0ZVkvJpvnDTTDArkbtz19EcQHSRySlm1ZpcQK2G1JqB7C6uicpOT1AKS0Fs33T1X4ES8hSqUNkeaz\\\\\\\/edSpaYOZ39blXPvAda2mU2P\\\\\\\/0pgQj7Xfd1DlC0trVcYHk3CbNn\\\\\\\/R8lV+YbKSQ1QO16FUbGNRLbEE8X+p6nLgChmd2PQ\\\\\\\/DDHTVZyy1ArAt9cWKskX0dGsdmVuarwEOhjlMVJxXHWtZSa90MaU4sdp\\\\\\\/ekEGdx8g+ec1yBqtq2hzG7rn\\\\\\\/ELZ\\\\\\\/kmQ\\\\\\\/7hWCXzFM5kEgNBjOX0cQYMDs0tpCIXrgChMqYQCzm9jzmSIuqnwPqzEQ47dqCTR6ErOzbBSKYp\\\\\\\/RFPdxUVyghQitgoEhIzCHNFP2Xdv2IFvmpys8RvEyF4Cvyn2KQSKkeMh3BcX0v4fLSf0aUwqwpT0796UVErltj2+rFalNWFvEZ675cf7ccgzK5QObrtDIDmV4N\\\\\\\/Wn8V3F9Iw+OxhfxpfXfs3CXxPI4LY+kAud+gwQQ2j7sH0Rutbw3Cu68E88vzDEhMnrLCtjUumqF+vflzJaI7W1Y6mLoAFs8K1cLu9JM03bMEKxHYaifWVv6OotCaqcQugETnnKGLsccuNctyWEpyWV7mav1EUenFNP+ndizAo5XYluebN+eMNjgesWqWsQRaLLmnuOnPISh2IXRBy+InZu\\\\\\\/atCxBJES4\\\\\\\/HXSHVK4g25WcsJrP8mhvtGCkQxKQ3uik3udqRh+VAwK9GQ1jRrw2Eg8B1xtqDTe2acYpNUCNKfJv01IlDd+LJ80VKPd+OGC0AzmcvLoBWMxf+EO\\\\\\\/lU7y0MbfpMp9W6zVMA3mJ\\\\\\\/Uu2rV5vKhG76Hlgtyet97KhMjDbxYzDn1qMn+kny5DbZ2N7nC6WqflO6pMU8T7zgY5Z+6psQapAW7\\\\\\\/CisF\\\\\\\/oNNFBMZnq9yxQ4jrAj+rxTrYaUxsU1EYqkiR52uOJpZ1fm1NKz4c13QGST7EBSV6esLeLqlQQ7AzbAnlLlZLPIDKWndqJI1d0LpXn08eEvJXzngV866g8gFraNXJ7tvBfR+og\\\\\\\/vJ\\\\\\\/bDGeoZR02qRW598vtqxh4kBsFlaMwxxwcCUjuxBu+JVMK84yetMl1ZffK7CW06gXgvR7OFMyTQl14n9UyEHwbi\\\\\\\/baFZR1BJduqtDoggIN8Vqci5ivUZFDLpbDcqQ6obMW70T\\\\\\\/7SSrMgHzaI3oH8e75iVozHUr6o4WuQUEpx+JN0W5OpA7PJNtnwiOdeMVbEovDt0h4tX4K4au+nfcecenN81TxMIdyRsw1C352CIvfGauU8Epl3qGGsaYB0IWkvncuY\\\\\\\/V\\\\\\\/eFo1jyTpxQvMwh6\\\\\\\/8A1+okgif6+5vSPLaCX7OzR2WerL7IX6gsAEmqLWNoepuNbOBb1o77R3QzCyMmJrqRU8byUzengOKWwECJl8QGnxkhDwdBPbG93248N0\\\\\\\/klBT3np\\\\\\\/IqHj9+1m0bdXUjABchokD5ieKDUcZdA9geKaaEAlPegF+vm\\\\\\\/PT5OSJFtCJvO1zax1lSWtjsY5E3hdmUH5fLxi9Ame81V4+w4FKk7G0aekg3w+9EmQfyy0e2EV4+\\\\\\\/kUzoDxgGS1xmue\\\\\\\/3ewSyFGAbW2qfzU5VfhnVgYPr7iTN+Yznr98lKT8Fn30Y8hccslwTcABL\\\\\\\/ldD2NCBO4Tb8UBjWOEQTS60IopY3VkP\\\\\\\/Ngk2Ayy9Ffi5jndDETAcoMDwliTMlpV+A6uH23X76kej6E+vEgBG\\\\\\\/QUUxbSebwoQNtBQ23rFe3iufWpcN1lIFmPxup9ZT3CjcjawmEfO0HbJwTeQK6sL7tscdqNKix9Saxl9rXQYRYQBtGQBkD0FB2hyE7mqUl+dGZU\\\\\\\/fWQ44ekHNagNyrD+vtHPfWTGYer0Wi0kmn1XOg+3V+UNGlZMPooPVvR\\\\\\\/e90qXc3cu4fZxvXeYLtJTldY7zif5PWkpdIjM5QpH4HDebUgq+fS87u2bHttwurElWGbnXdCOAOJaOmXvmPJJL+4mffWWA5\\\\\\\/5UMKze6ewIVB\\\\\\\/T0I34z0zrqtWaX9l7493UqbAuHi0PZax1e3DhS+G9AFkLKMqgfPyUS3ax2ojwh+mPOD+3tiP5arAYyd2ziJOyC1AsBDbqwPjPv9aOHkM1oxDmMer2M8uCwdU\\\\\\\/MtVaFgMJlEWszbqxN4evaHjbG0vbWA4xfOwBy2STIyqwYn5zntAMJLhRf+g4MXVPkPlTb\\\\\\\/9CgKjj38P4briD4PibmUmAgxAADgwPhLnPHXOrX6GOu5rjNlLj6\\\\\\\/9gxy+MOhqjJJQoek96Cht+q0pYq5fZOVB7ldORTeQThvUl8g0afjsnyWplnl0hfkqvdNuSoAQHLZvjLKJv\\\\\\\/5+z2HcCUQZqxgyRBFX0bqr+hqn3f7KxlqSQMer3Wl2TIDDuANGdDLQpwVgj9xB1PaImECAWhWUPoaPIFcEvx7prqGR60AfaIavPmRAFYkdIdGNUFtWHh+mbSStNEehgqx+Vd0ccDEWOBPLknR9IsQDO1dTEYJCvDtX8WYJCq0KMG6eoZEg8fpfUGkq+SeKk5o2kgTParwMMfsEivkTiJSw6nwlovXqapQhRym1jDTxJxuOXW+hA4DGOBqVygw6ADw50zu0Za8FJSnMMPpqcaAaQ5vr+T6zDjut4aw6\\\\\\\/5zh56OZ9bLdAoHWj\\\\\\\/p\\\\\\\/eFb8J6IVoZmIVI1KSdCKoKOGIgMjirUOrUkXUpzYCQtDadYF\\\\\\\/ukaq8WgI14mP\\\\\\\/wWV0bg3zryKeqgeUNsXIbp69KLn2VwswwHw335KBmnAAtAo4eM6x\\\\\\\/LtQQkNHO6yLLeE+vuybg1SU9GZZ761iiZZVeuYXeVlMWwlRcjqFYdhafMgOD4IEPNrvu1fxa38KuO4jsGgIUSN8X9aiIbZ8wRE8Qk1IgarUFgeoQBhlN1U7+X5zrxta9izkwxrmLGuFLj1IIqwbtgTYG\\\\\\\/\\\\\\\/vSbQbAK5ZKNyQNPR7t1kQaBnByP5TeKagXx8kERnEfZmF2KdgMr\\\\\\\/83tquhf8Uvs+9obHWcO4XZMmyNeR+gSiDV4Nz4x\\\\\\\/y\\\\\\\/x094hO1MtU7G4bpSFsRg3El2+GRg9efg1+aFFiLpI4M\\\\\\\/r5yblSvsNWE6ADvrdtu5r6\\\\\\\/XEQaHo2UrgQuUedvD26XmPHxcnvDZjZsluJDtUqwYmRwDPWq3Ftgzr91ZIOAi7emRdwITtrK9rk7x6byD0yrrIOWRDFmFa+Ysifmbb5hG4lwXelNFYEHyxTtnBtVuhGe65+k2uBtxAIL3wWEqRTvlI7nyX200VfU1JcCMWkfI1\\\\\\\/lDkQ76Pi45xNbMRNPrCDbr6EHwZmFrI8rJJACH+GEv\\\\\\\/VBTNBWjltXfNVck2KGFD4BLLnSRclTLl\\\\\\\/VarVp13uRExTme3654uLfBIdq2mvsNCmGw2PnKp2nfk4uvadXEMAVUJQKDzdklc6Di+lpzrANHMNJngh+dpNJ5sAKp886AX2M220dgznFO5dc+0qMreoG3pX1zeQ\\\\\\\/HR3rdRjqHRXsEkM35hsCxrIZ8ztcRDY3z1Rn94cbb78Wz4FvIvCaozrbng+1imPjf0rKI3kCeYyhsmWS73fwEG\\\\\\\/8r+vZ54sTCRCDIh4m0Z+zu15FBjnFEQy6mV1jsiMVujA4o1bal2E4C477F\\\\\\\/3tdQD4jmDO55WGZ1ask0cDP+KeHOjuG0OTZhBipWFBihw\\\\\\\/HYonfxRBEFeodtuvAKwkjQJa2RHPbkbaCALkOOwK+OQ\\\\\\\/14JkxY2cxXUiSmPGUsShPrSicsOl7zL5PY+EPQArSyaWnyCNRWw41agtJaqANgyMDgQFcj8vsCVXbo73WPW066lATtct3Vzpz0suofYpiCazfPpLkl1COKOLh8lpHb\\\\\\\/2ta8QBsghPpAq2C6tBvkax0MEisBbu2gQZGgRWmrz9\\\\\\\/u3qPxDcognlKGg23TijNp1Gft38gtJPwx362kspVrqK9tVOSoBMCN5sEbdJzOsAxrRv00lrpy2u3kNeRynXlSiDPlpcArLUquQlvbhf0leDj2TV8\\\\\\\/m1SPNKBCY2Nr0OAlV4uyoHDybR68Y3bp8WmzHd0K9f65pTpW\\\\\\\/vgHZFZjt8KA7orQ3yP0YnoYBF2GQY0SojqxPre5V\\\\\\\/v3D3biMvlSJXWPK575679\\\\\\\/T4NSxe+C7XnxvMbX4DLggrKnipBOu8Dbbqfo\\\\\\\/U6LTDzIrIAybyq4UbpJmW4ZTgmjyFanDhbnSsGUZM0Rx0w4neLpTola2tov1lIdVastj7yOi+XH2noyRHTLVCSvyR98ow01TtLmwqTzLDVAl6xsI\\\\\\\/g02HBt5xj4V8zW9e2zH8g\\\\\\\/VYm30XGQFW478sKexY\\\\\\\/iqiEY9HaiFNjfyi7nXNdV5VT1maY9RN\\\\\\\/7T\\\\\\\/Hjrzhb00apRwSq5KU1O02rbMIEza3cpmqrCMax8OdP3IUgU0Ih8D6ZiB9zYOcbYvR3ImJTBIG7H13t63ToNJ+JDD1t97wmTa\\\\\\\/6F1jTmyXieXFNI3xt7qTxf6t5UQhxs\\\\\\\/lF+LRbZTVbV70Ad0b5S5m+wf1SOYhwQceXYlUOnrSOmwr+WKkfkOzeOqUNTVwhAXfmm1LPdhYhY\\\\\\\/PKOGOibGerUp3olp6GZ92R7G17o5uIv2EDcOgvGBzCuQY\\\\\\\/mjhO4xlX9ZbT372LOUTsjjWU8hCXKJi1C6ivWQkZgvw61\\\\\\\/WW\\\\\\\/EYqba4mV2Orcuw0XjpiPolrvx+uZv8+bnC7ROkfBcGa4ieGTaMeldt0CxdSZLxc9duzvL4sZzqJDrmMId4SIBufbSelmaOSJ0RUJBVzXFu7F\\\\\\\/POub8RFQIyVNMz+snnDCL6yHTceoWKVbXnVFtOTIfaWg\\\\\\\/1T4uosyu4DNze+b6GyFJwzoit8qKW3+Vv\\\\\\\/MsRuJN7RKLXdv0lCr3N62HYtfhpv4JHqH3nPPJeBzp1qDPgF9BBguNB3g640\\\\\\\/yzpT9UBu2Npi7QnSS\\\\\\\/NTGJjUoRDPKak5h0C850t19uAGBa35Dmb06wS2hztRcV0BA7JpymYEDZQYcgvBNFxeuXBOvFwNJC6p57qzSBOek9\\\\\\\/\\\\\\\/y9mkCQTLhtu2tLNpbN9rj1mtDVj6afsibj2gSXVgcmXPFBMp4Ku8LdvxNB4+Cot5q3MUMR4ML2Si4Vy3CARNODxynPlM5oKRFo0CB8eZboRFtz4LS6PDIv5ZIwxZqavBPqutt9WziqVrWFmK8DYIxE7hNssgizrbXcPO2n94dQLqGILf3+qwybfTEurIv\\\\\\\/6w6zkJnbBPKEgKMsOiTqrCOTfW+UTumCWKrjNGB7OHPGWOCxCqNcQ72u8UMbyghHmgRgGb4UuLp61VUc4VgGceNHjqYQPNQh4S3jMfMp0D8J5phDskE4cksWga6r3Ng7WFX8ZbOiVandChiN6lapysD1WxEd7Ls6+q\\\\\\\/Xpf7ZO1YxJ\\\\\\\/J5eeJqUMBBtBD85I9L87jEsmx+t6kTleCJ+4XrSnggLBRVf5V853lmW\\\\\\\/H6IXIkXiADtzO08Z9\\\\\\\/Zhwru9sDOfoe0r\\\\\\\/isrVpCJCOfi5WNgB236UgI\\\\\\\/TS9yyAU86d2o0UVdaqIaNJBCUntcRe1Kv67OG5woWlcRO40h\\\\\\\/T7LMhF2gMRA\\\\\\\/5VCIcyGrRg+YdJ0jZ8LtlU3KmJbbyl1WVu6sO3D\\\\\\\/2VXIElrRYOgMEJ\\\\\\\/9USbxvTdmeHNfJsV0uPO3UuPmZp8VjMw04d6kVFGsyTSWMuPROW7xcl367m0FYDjVQV4oKLHKb9Hu4GMmS6u1RNkdOM8ANU1qaJ6oe0d4ivc1mrovKdVQGxsWHcXUcR\\\\\\\/jomApNT3X9BQQL7Rg9jK44gsKVLq66NwB3O6pIupuTt2CP6gKy76Bcnm3LwMH3ZCHhQ82rFCfLgGhFeMhayT9nmsBJG4koOOQ2IyVX88fqXdT5uOQi6B52fn1vgAUvZ0HOVAJTGyNSvs0YSZU\\\\\\\/c0wZ9nSGJ5a6rO0wEyYqQ68wsHrAm1gylmTnBgIUD3ZuBJWBxWgrgJRK6GD+jGnuix3\\\\\\\/j0y3GrVhJfqpjE5Ps+JbwDKQMxTpbkpk087iLYhDg6wJlpAFDvQY35pbP5vBTuhrHoKqikL8gmcN+bWG1LDC8ErCsO6+FxdlLWSw10AiuzRsD+dB0iy0yoWJIbAU+KsdnEu\\\\\\\/MAfS\\\\\\\/XhZ6GNeP4UbKTWzF771mntuFcivKpTpNj0iWMA\\\\\\\/jDY7RrFS\\\\\\\/9djJvIIXBxnMJIoM8OMOD1\\\\\\\/6EBmySzAuDCUWvfRyuPngajeLCatZblTeoRafqsmyqJWJGR\\\\\\\/9H2d2n2CiEWrZqUjLuvwJ2wJH7VpqfESmzv7nu2In9pqWEenCm0fZf27PTomWpqWU26L5xXor+NyZs0QiXHuHIXDBqo0CKUCOg4Wny2Y8DBHVTqGpTj8eZWZr76okNOyknJc001rAE6hRF1Lqxg8jrQ3OaxElVqkNG0vCBY+Lh61QBBkahloADx1a+RvGdSoPtKELAzuSINE6zAkYHc8AfS5SXBnERj4Hias7W9JB4L+LnXcq4KliAxDNvesxfMo2AoJLZrm\\\\\\\/WAsEe\\\\\\\/Eewps2bEkYadZ3YXUWTyUWkoU6swNsPavK3vSH8n3B20j2VPDA5kxhDv7br1jnTq7Hqblb812KgWW8gWKgxhU5mWijYidd+2k0+g1l1dX4MbV1Q9b5gbx0fwMCmBkbD\\\\\\\/udGh3ULSv6bpQ0dXXsW2m0chgeo5YaGCaE0UtahsF+j6E77zMKerSdeet47kr53zVGZ+7g678EgbipKRln4raiL8d+2MqXvoDs3TwVesnYtUWAgLeukea8R445NyQwvsGo4ucMyG7DA48SmhKMoSpw\\\\\\\/TER5z8DRmtlt7x6nIUPAh39uwKwtq2yGgqTjkoHTv64HDaZjySMRC0dGXvzvDWgepLwcAfo7brFUmvGv9NVzbgaDe5rNZ2Wnn4LA5itCWDLu7xwstyIlm5Y\\\\\\\/yQiR9HH9snfH8uprK0zVIElAFPTFMqNnOGj8wELwKpPzzh6Clcm4vuCvxl7d92k309iH\\\\\\\/HowmiNyAAWQ4L+MA+WI+Abk686UObO2YB7wfcH1v74fwL94KUNRxZoqSKVxd4zXI\\\\\\\/yP8vUqEhLp24yJwA7smD0fjyMleIbQGXSyxngSpOwyaI46hpGE6+82iicPW16pwMMwexz2Hz21t+4yJNIFWe4+SnfF5w7LjCY7I2bAs45Dd4O0owYKFdu6EDsKlv3IyGaAzqsCaNNubn7GJcjWE+sFEEymnegXe3WxDGRNT6bJx00S\\\\\\\/uydZhrwqaEv733bw0l6rKr1eZYQgdM8L7nQOMhBcyp\\\\\\\/PUArDiSeSPP\\\\\\\/Rw2WF7v\\\\\\\/vvDFYBV\\\\\\\/A8bi6+jtIPZeTmlZozcTfnYgEJh6hitR6RUmBbtG2\\\\\\\/uXenwaag3wm9gpR6iZXMbRL+9hRQO6bsoUkdjqYrnvGcnP0THrzpscOg8l\\\\\\\/zp0MLq86s5H4nCVlfWv+ZljIcmvFUAc3K2583c+eu7uFpJJDvMYGLaIHqSVp2VMzqVR5QcXIttmVuoV3GefSkEMOel21Mb638YIZnnR7wjC0OJyfYaWtAL\\\\\\\/xBsl12E\\\\\\\/kdTn6Z9FUnwfDkZXhv\\\\\\\/zraJYUu0TFH5tTyguF5pz0whoACl8RSbuaFmin7LWSj\\\\\\\/JUHwkvAECBu3lBJMBxTnK5c4OlrOKQqcXxarrWnQuezxlPQUbX66ZUQXEC04LSzEdFkFs2IDeaUhS+EAniFhMX36BeJGgnia3I8tIokmUuzM7ACd2ogQmrJYK8lSi8pEIuvYeFjh+YB\\\\\\\/n5hy4DpqA4msbatOS1kqFaZqjUxwU9fMsbneRL+FhEDubHg9zmfZpmL2I5jGkSH++Ndr8C8E6uoMTh6\\\\\\\/unTB9uejUyGVJyE0B60SJV6Dun9g4r7SwNwh8b4BEYZtoCwzk35aUy8e4Txn+EepIyVsaERQ5+WGBOHBDlJYXxwgWms4kFRDX697tZNo2RY4xnRCJHq2ZR3jB97e4cfvGD50GRifwZ0alJ2lQGMURv1\\\\\\\/sROACS9tLbD8MKK88E3\\\\\\\/Gf\\\\\\\/NOOCtfRPIQyeh4fGc8vizWgAThz6Ru4J80CsZxyYapolalZaMdsmyE7UeQ8v1aMY\\\\\\\/gYte+sMRFzQuj0G2X9yTvcmCJFZpkiBCU\\\\\\\/6Aol\\\\\\\/dqmkalcZdKmVY9eX6EI67VersFUA4PA5PTfr0nIWRMZn44moXD68RMGPd0vbIlOs94T9H2mRZjYG1IzNT0Noz96X4HaGAOObMr5CNcKhDff\\\\\\\/8AITdmWF68V0VwNmX2lQfgfNhx2PMvsN\\\\\\\/1ds9FrWhg8Nx5hH58vD\\\\\\\/R65MCVaBJUTe4g8PMMFnru3fwEqq0sy4XF\\\\\\\/5t472hBxLQylnRga+zyk38kuCDmoJJGVvRTNxTObJQeO3QjcskFsC4E7slDV9GKxJYTgYpfH1I5RImxIJbqeQH04BEkdssa+wBAHir\\\\\\\/NSbIE3EjJeVo9f9+D2T\\\\\\\/OV0LYRK2XrY59rnk+QiW7sEiL9gcGHnVcHWsVdIqL2oLK3q5BhRZhSvnfZvuC9wVGImHi3JSK2TbQqsKKfH2ijUaU1bvIaE\\\\\\\/SgzSFqMjTpfyWx\\\\\\\/xgBqcyT+76kz\\\\\\\/Ad9PYXcSLIWX\\\\\\\/+a6GQi6V1ciXi4Wa9rqVM6\\\\\\\/qvA7v\\\\\\\/gru9nCg\\\\\\\/D5z\\\\\\\/LOSGu86mDHC1YwHpqkqJFONmFGfQBsEl0aWP0VsG0vaj\\\\\\\/KK+EyOuGDDnDJaK0EBv6YC+xKXF5\\\\\\\/vgqll3sHHB0iiHxki3lX8smv1HNLameODUFRSsXogQFEj\\\\\\\/+jVThgB4OWFsb2ylLdhkwoLCyDeoqvDrP8dRAySEE4kdAhenO2zuZuiObGjXr98++0MfFNZUB2+kBrCEIt5hJEQYoAwk\\\\\\\/O5JI2fu7hJ\\\\\\\/otWVb95s\\\\\\\/MKHTBiySLUGFgxh+IAPud0Ck77Yyzj6Bty\\\\\\\/J16z\\\\\\\/bnI3JnHr3\\\\\\\/46AH2eiwjRGY6lqe0Fq+rcHvDU0gjyjxSKMADRDAaYDrvUcJU9gRGUyniRPB5bWs5njleWGYI5\\\\\\\/tmYKof\\\\\\\/2Bzx4\\\\\\\/ZljkocuZRq4\\\\\\\/drkuw3mYORYU5al1gyhT+dB3kfuIreikZIuSzEJTMqmyuC\\\\\\\/ERt0XTVblHIBFh2ttMcGLxHaGmGQ9a8wTwh4LPGjSxy+atFEC\\\\\\\/IsKkNclqtSH3Ed1vTQ2h9TCDKgpOWYgRBAAUUCHxcEdak2JFCMESr+\\\\\\\/83U6IQriFpfqhAieP+8Yib9hHoJZNDa0IlMqrwua4SnRty\\\\\\\/\\\\\\\/YJMzdOf7\\\\\\\/TrHeC6cQbQ+H5PJxP845o\\\\\\\/NMP7EBymJbqwW1GwbCVqZizRPmBBwlsxh1YyWT4VZMqLuW5ojNotJjE+wQRi1YXlKvdfjRcfHP21mGgw9SXwBFDW+d61srSius1IsbKoB9h\\\\\\\/oZpSS6sLc9fANI21AdC435MnutL+O\\\\\\\/hHgszGDaSBKUbjHg7LBEvH25aYi2XQv9Sxm5Iiy+oHfWwgnI+xKh3xFpMuv8k06AvIQz9+hQSPXXnu5JhCsKVRTBFPwufwKplGb3j7gMJK848K6Aw8pKQoDZfxpbkvwxUBZzrbLofa918t7ZaFqUrdBVmgn8dUHnagY8p35hs3yL6ff9zhBK0b0b4q++RmS1UycRUQIk5wdD2deiv2\\\\\\\/ULdU7HqXCkQviiCPKCNwLHh91PKYsz4++UGk1afKGtJ7xXZYEeZNI8Crsi57K6hkHC8H03XFjDnnHrP\\\\\\\/YYo1\\\\\\\/l4Ip8EP\\\\\\\/FpMw0saHn5jy8JMKAq+M5jjJM5v1KRviauAXf5KNBFhGESFnBpJd7Wp9kcfecbvG6bvmzK6VKQ7nFWwtVtITGs+20NoSXq\\\\\\\/pCqEBDShys7Fq7F5RWckd7+3DtXnPHfMXruNCCztyXqL7NjxTOyVvFowy7uFAhpGgxsp4yRHJyk3yO5+hSph2f4TDz4vuxPvPwaAB5Cr2UDV5zsH4ggelwbxXGwPUDUF6lrrnzjd\\\\\\\/N6F17YGRGguu\\\\\\\/B\\\\\\\/lnHQb8a7pBXd9hUGpaeZPAL3RvHSKRKUsqgUaY8TmqpGuS5lyRLPjiDmttEv6N9t+aT5hk6GaIBfBRKS8whdqJ9fnNZQbF7PaFNmZh9fYjci5zVVvySkGuJ51s6xCajUnuU3eiaJRye6y1i6JOcsAMMQO+vlQJ6zC6pxUBb4zWbcfELVwlHvzpk3SBPJoRWDvvKyd+QrKsOh8s9HlP1hrkT8+aBqJ9l7Qf0Mn5ty4CDsDsMoqyRJP6CuUXOi5eJ1NjeJ\\\\\\\/9tXJqgcN8EGQPzVUuLBRlDQWxSCIptq+VAJUMLwtLqSF+786CKyXuRfqBqyoC2VP6FGXjVLabJtIWemgfwzhChSsecsQCXkhjMU4lg2Oo+PdQExEbgHsqs4k0NnCX61hl5UhSNavi90+lI6Hi3tldz2YFoa9cXoBY\\\\\\\/3\\\\\\\/Ve6AFj2COwkChNzpX44DDYRmEWLlA3rx1TA3YB\\\\\\\/wu+Li57EBAo+9isW882GYCL09BJLJ+bdgMVgFxsGrKCTqdXfmlWSaiQjbz0w28MVxLTzZJjS0kKyKQifIIV407J9diOu01zWOHqudaVFPYol\\\\\\\/PtznMtIpa9U389QOq6Qrj1LyhUivpiBTo9jxuXUGCkfvr17OFbronCw5rN4Gi7GX+X5\\\\\\\/Gz4LRpZrLAtkHszPe8P+IYACf36cP0FSfGw7etRPnPljZau4SFLfe4jwgfceVeHG4G3sFN7AQkMPuT3LA9PDpmgbUft9kA7EcgPkThIBG86YKMBUksIiCtcmJ9inIKH9FsnWh0rxTb9uqzK4SQ6atwEqRcxkUq9P+uWkHxygeEYdzZwzF12aMs0uEp0G8yAfrfF0xP3uC7YLiDeEhNveq4fYKdEWekEJDfjf9qGsDDhlGXZGrenPKMxcB0SngDepS1yrHz1tQVsFPpSK+WQYMMZs\\\\\\\/l5Ys3vfI11PbbSzAfd4HQvHvwTVmseEl\\\\\\\/GBS0ROFCbZvCUzCVDGswf42ndZW2JFv2\\\\\\\/O1NjJviehdiUy7ouuWjXyz\\\\\\\/eeoEPOIyNh3kXUuU7w3RD7sqjXZvoHCk82hFwL0fwc0z7efgPgGdQRyQ3FvMp30MFb85YX8l4UnxEMw5I6JzqV3dMdcvgAwUUzJiych4iUoLNvNhctIxqYBusq3VQZc3ebFHTKkr82LrsoEWnzYzjDt029zXCZsrP0oCaG4OlZT68sH539pNdOGsqNeeju62vLJQnYRLWle7JojgGWveriLIe9LW1RbymCl3D4E2Ja+9k1GNglmkGNLQJXUAa+gMx2Dt8u1RFIoyQUEBG7dPqe8kEi52hX8FSSp9s40aDzxaFDNtDpRuNjGubx\\\\\\\/eUwENp7rp\\\\\\\/J5mcBhOv\\\\\\\/KD5C1uX\\\\\\\/clTLN8CK\\\\\\\/oCqKG0o5yF5vPPm9efC7loLNrBVY+yf0ierfXwOcdSHaNb+CvETTR\\\\\\\/1k38wm8xb9qWGxC9OTyBW7gRdLh25pPibZueoNr4IuI\\\u0022,\\\u0022iv\\\u0022:\\\u0022fVN9j7lTDcMmDsWknMQeRQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022efb885534f7be3c8dffb464e3254bd1aa6bdfe26cdc21e16051201cd2efa9be4\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022+WQtZW+xTmY7jVxG36peMjQV63xeUykl6hhczQ1H1bmVRJ8Jpb5PVOVhUqMgRiUKCDs+BnCi1XnPcD13pUt0WvuA7KCrKlLurDuDUJYG3k1Aisu+D1y9HKKuerLqt7zou2tVs6+Ll8uVRRnYzebo3nrepRlgUmFROiP5On62ez0dK\\\\\\\/Piyfr72DGpCBM\\\\\\\/Aw0KGDANzISk8xB5bo6Fm1WAmhXtbpNj+EzN5xGIYIVTfAaypuc1THLVUdkuR9jyYlao5JNldJx8oHFRn20EKSmC97EDmu0kRzWwC446hB+2f93qlGTQ6KUHIOCLQKMKC8OEzj5I4WQ8dwo74FEQw\\\\\\\/eDUqbi8tdoNbOkUFcuT0Lap+R+jW4PVhbx6tkDkvi00R55hJlf\\\\\\\/pa3VmlmeITxAWlktcf0D1X9SP56JMxaqSVSAqH7+7qpNO3xcsTNbv7Fgj0u1wL\\\\\\\/3hDpI1qpZ8s8rcwk54iCsA+uKoKm5NBKlTnDPLbPH4TTW8m3HqGfT51E6O+BQQnYnxP0UQUSVCegEZtpcxCsW\\\\\\\/2VRMKCSGq1WeK8hj3rEJ5x27Hg4QRrOtDJvsbLYHTbrIgAwK4\\\\\\\/eYtF3ZmjqzqJzpOYsJSkZZgfKjBl6hqvxitAool+Q73GPzSlQyyntjhZ8jbQ7e4lbF\\\\\\\/vVxxZYxsO24vdAQ1nuCjRRq\\\\\\\/X8GTM47wLFs5m1TtCzp6mV\\\\\\\/cV7tsfTv2j40+TingxctJ66bVLBTKBoz4P7OOVGQh197qshbqU3Z02FUs6AGjiBX+LFix+3KprfSFlzF1DsIgKm8IeJHJQ7iaFrcuOHWzBCUqVzkm82rgUHe0JQyQ\\\\\\\/1f35G2YkyAGScxlEEziTPcA2H53kkt4noOBQMvp5\\\\\\\/dh5J6ffxAryDQ1d1uAZuanw5Xv74d491WGV7RyouO9cgpnthCSrQWOYdY2etkxgLcoYS1B9liRMJdl9eyXWrZNK0QCSu2u+XeG+N4PNvEuelsX+M8CZh3\\\\\\\/c45fngYPOKRmecYvmz4cUu7FhfLE\\\\\\\/CMpZjXqqQnVz\\\\\\\/F69a\\\\\\\/KFYB23nnDdbsYCDOqeHB14waTvkCf1NeKcXRfp4j5CGO8JMzixsyU63n\\\\\\\/DtDlfJu1CQDYVhkHpT51V\\\\\\\/aupXS7NdwcnZG8\\\\\\\/1Cj+MvEzFtLgDG8vaBQwfCx87qrLM996KOHl1q5c2lgZZBvGLMUbt4bY2Na9cGLDNbOF6PeXJs4cc+rKW674F0uuy0diiuf7xNTSbY2m9Rg5rbderoh1pzuAhVcDgGKhnpeq\\\\\\\/BsFdzfgHFT5K433xhQr\\\\\\\/hJpCFk9HGgLH2EGN7jffc10teA+bt1eH56mG7rxereiyf5yXfk3iYkGcQwibgFgsFMLq6cleHvzfJmZVNHRKk0aEF2cngacawiMUaGZgzP5x+2MruNMeZB7RaNxn\\\\\\\/MYf929ug1gO5hi71VfwMb4u7yWK2SqB9lsWkODGTNhHLr8L+UyYPfWeraPmexhITMN5UvyccKtFzh4\\\\\\\/BRxxzB7lmjn4zlB1g9ku3V4iQtA2y+mGv0W7jhPpYwKoNm0TmGetKIRC6crEpOwFe8\\\\\\\/xu5vqhcd9zrQUcu81BCums\\\\\\\/5+fAtUAtxerb5PvAtLsJpS3uF2T\\\\\\\/6KUDQ1UL26y0F6j1fw1EDm\\\\\\\/5db6lQiZuOavKUSZeEzm2lIIya6VpPtRuzNGpqrkGcD0FVLUsdO7BJ0HhrTuxV1jayMmxqqrP8yTrukMOgkJhuwkyhirThsX2N4IIAoE3QiQY3JtLoSmAVEw0QlY4sq43w3JdgurPCOdw4PDG7anob3QjGZA9Igky+AYVvWOwbrIn4n\\\\\\\/LkH8NCfZTf8hWLTyxkHwc4FLNlfA2m8JbXIWHochnkIxySf\\\\\\\/rJR3bWuZLJsfJUFqimPe8eDSmif0blYDxN1iLNovfCtqwyjEeHPppfzYMyBPfx7mtQKDLU2\\\\\\\/Pqdq9z2+FX+UOS3adSr65YhfkHBXBRMKvE00lFSfvS4KPD96s1bq+LrN0zu7gdYqP\\\\\\\/bVgBLWKf7mkLy2H2pR3t01pcPCPQZhAsuvSEYPc=\\\u0022,\\\u0022iv\\\u0022:\\\u0022ePLkab0z\\\\\\\/gK94RMYS7WQEw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002217407e227868bec65dc7ef7584fd2d8c1986be968b2c05a3bd83c4adf5d30e48\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460080 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/3/653" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-8vBgCAYJ6ON-TKBnWT1qfkSQ3xV3uMhDbFVUGwwoMjE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/3/653" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YrW-e40td_Tyfkow26lmGwtHNtJ9xctAvflMhVGmQQ4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/3/641" title="Vesicular Trafficking Systems Impact TORC1-Controlled Transcriptional Programs in Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/3/669" title="A Computational Approach to Estimating Nondisjunction Frequency in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460080" id="node460080" data-pisa="ggg;6/3/653" data-pisa-master="ggg;g3.115.024331" data-apath="/ggg/6/3/653.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Combinatorial <em>Cis</em>-regulation in <em>Saccharomyces</em> Species</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Aaron T. Spivak</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-6896-1850" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gary D. Stormo</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">653-667;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.024331 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460080" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Aaron T. Spivak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Center for Genome Sciences and Systems Biology, Washington University School of Medicine, St Louis, Missouri 63108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BT.%2BSpivak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Spivak%20AT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAaron%2BT.%2BSpivak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gary D. Stormo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Center for Genome Sciences and Systems Biology, Washington University School of Medicine, St Louis, Missouri 63108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BD.%2BStormo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stormo%20GD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGary%2BD.%2BStormo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6896-1850" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gary D. Stormo</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:stormo@wustl.edu" class="" data-icon-position="" data-hide-link-title="0">stormo@wustl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/3/653" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460080" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460080/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/3/653.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460080" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460080/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/3/653.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460080" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460080/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/3/653.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460080" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460080/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/3/653.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460080" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460080/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Transcriptional control of gene expression requires interactions between the <em>cis</em>-regulatory elements (CREs) controlling gene promoters. We developed a sensitive computational method to identify CRE combinations with conserved spacing that does not require genome alignments. When applied to seven <em>sensu stricto</em> and <em>sensu lato Saccharomyces</em> species, 80% of the predicted interactions displayed some evidence of combinatorial transcriptional behavior in several existing datasets including: (1) chromatin immunoprecipitation data for colocalization of transcription factors, (2) gene expression data for coexpression of predicted regulatory targets, and (3) gene ontology databases for common pathway membership of predicted regulatory targets. We tested several predicted CRE interactions with chromatin immunoprecipitation experiments in a wild-type strain and strains in which a predicted cofactor was deleted. Our experiments confirmed that transcription factor (TF) occupancy at the promoters of the CRE combination target genes depends on the predicted cofactor while occupancy of other promoters is independent of the predicted cofactor. Our method has the additional advantage of identifying regulatory differences between species. By analyzing the <em>S. cerevisiae</em> and <em>S</em><em>. bayanus</em> genomes, we identified differences in combinatorial <em>cis</em>-regulation between the species and showed that the predicted changes in gene regulation explain several of the species-specific differences seen in gene expression datasets. In some instances, the same CRE combinations appear to regulate genes involved in distinct biological processes in the two different species. The results of this research demonstrate that (1) combinatorial <em>cis</em>-regulation can be inferred by multi-genome analysis and (2) combinatorial <em>cis</em>-regulation can explain differences in gene expression between species.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/gene-regulation" class="hw-term hw-article-keyword hw-article-keyword-gene-regulation" rel="nofollow">gene regulation</a></li><li class="kwd"><a href="/keyword/transcription-factors" class="hw-term hw-article-keyword hw-article-keyword-transcription-factors" rel="nofollow">transcription factors</a></li><li class="kwd"><a href="/keyword/combinatorial-regulation" class="hw-term hw-article-keyword hw-article-keyword-combinatorial-regulation" rel="nofollow">combinatorial regulation</a></li><li class="kwd"><a href="/keyword/transcriptional-control" class="hw-term hw-article-keyword hw-article-keyword-transcriptional-control" rel="nofollow">transcriptional control</a></li></ul><p id="p-3">The combination of <em>cis</em>-regulatory elements (CREs) in a promoter is an important determinant of gene expression patterns (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Pilpel <em>et al.</em> 2001</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Balaji <em>et al.</em> 2006</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gertz and Cohen 2009</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Kazemian <em>et al.</em> 2013</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Nandi <em>et al.</em> 2013</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Wang <em>et al.</em> 2013</a>), but we have only a limited understanding of how interactions between regulatory elements affect gene expression. There is clear evidence that certain combinations of CREs produce nonadditive effects on gene expression (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Pramila <em>et al.</em> 2002</a>), but it remains very challenging to discover which CREs interact on a genome scale (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Balaji <em>et al.</em> 2006</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aguilar and Oliva 2008</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">He <em>et al.</em> 2009</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Girgis and Ovcharenko 2012</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Ha <em>et al.</em> 2012</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Kazemian <em>et al.</em> 2013</a>; <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Nandi <em>et al.</em> 2013</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Jiang and Singh 2014</a>). Understanding eukaryotic gene expression requires identifying the CRE combinations that interact to produce nonadditive effects on gene expression.</p><p id="p-4">Gene regulation studies using synthetic promoters made from random combinations of CREs have been successful in discovering new synergistic combinations (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Gertz and Cohen 2009</a>). However, the number of possible CRE combinations that could interact to regulate gene expression is too large to explore comprehensively with existing experimental techniques. There are approximately 200 transcription factors (TFs) in <em>Saccharomyces cerevisiae</em> for which the DNA-binding specificity is known (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">de Boer and Hughes 2012</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Spivak and Stormo 2012</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Hughes and de Boer 2013</a>). If only pairwise interactions between CREs are considered, there are nearly 20,000 possible CRE combinations to evaluate. There is a clear need to efficiently and sensitively identify CRE combinations with nonadditive influence over gene expression.</p><p id="p-5">To address this need, several computational methods have been developed to identify pairs of interacting CREs. When CREs interact to control gene expression, previous evidence indicates that the CREs will cluster near each other in the genome (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Pilpel <em>et al.</em> 2001</a>; <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Pramila <em>et al.</em> 2002</a>). Efforts to identify combinatorial CRE pairs have exploited this feature by scanning the genome for CRE co-occurrences (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">GuhaThakurta and Stormo 2001</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Chiang <em>et al.</em> 2003</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Beer and Tavazoie 2004</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Das <em>et al.</em> 2004</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Kato <em>et al.</em> 2004</a>; <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Balaji <em>et al.</em> 2006</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Krogan <em>et al.</em> 2006</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Hu <em>et al.</em> 2007</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Girgis and Ovcharenko 2012</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Ha <em>et al.</em> 2012</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Guturu <em>et al.</em> 2013</a>; <a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Kazemian <em>et al.</em> 2013</a>; <a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Nandi <em>et al.</em> 2013</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Jiang and Singh 2014</a>) or by examining ChIP data for TF colocalization (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Aguilar and Oliva 2008</a>). However, TF colocalization alone is only a weak indicator of combinatorial regulation (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Badis <em>et al.</em> 2009</a>) and chance cooccurrence of CREs confound analyzes of single genome sequences. Separate methods have been developed that reduce the number of chance co-occurrences between CREs and enrich for functional CRE interactions by limiting the search space to conserved regions in multiple-species alignments (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Chiang <em>et al.</em> 2003</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Kellis <em>et al.</em> 2003</a>; <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Xie <em>et al.</em> 2008</a>; <a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Jiang and Singh 2014</a>). However, aligning promoter sequences from multiple species can eliminate functional binding sites if regulation is not conserved between species or if regulation is conserved but there is turnover of individual sites. This is not a trivial caveat, as comparative genomic studies have revealed extensive gain and loss of CREs between <em>Saccharomyces</em> species (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Doniger and Fay 2007</a>).</p><p id="p-6">Although individual CREs are often not conserved between species (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Hooper <em>et al.</em> 2007</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Xie <em>et al.</em> 2010</a>; <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Zheng <em>et al.</em> 2011</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Shibata <em>et al.</em> 2012</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Reece-Hoyes <em>et al.</em> 2013</a>), functional interactions between CREs are often conserved among distantly related species (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Tuch <em>et al.</em> 2008a</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Gerke <em>et al.</em> 2009</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Cherry <em>et al.</em> 2012</a>; <a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Jiang and Singh 2014</a>). Furthermore, studies of gene regulatory evolution have found that interactions between transcription factors are conserved even if the TFs regulate different sets of genes between species (<a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">Tuch <em>et al.</em> 2008a</a>). Therefore, cooccurrence of <em>cis</em>-regulatory elements in multiple unaligned genomes can be used to identify interacting CREs. Incorporating this feature into a prediction method avoids many of the limitations inherent to previous strategies.</p><p id="p-7">We have developed a computational method for identifying coregulatory CREs and provide strong evidence that conservation of a spacing bias between CREs, that is observed in multiple species, indicates combinatorial gene regulation. We use this observation to identify many new instances of significantly co-occurring CREs and to predict combinatorial <em>cis</em>-regulation in the yeast genome. We tested the accuracy of our predictions using ChIP-Seq to assay DNA occupancy genome-wide for a few TFs predicted to interact in our computational screen. We made knockout strains of the predicted cofactor and assayed TF occupancy in this cofactor deletion strain. These experiments show that TF occupancy is dependent on the predicted cofactor at specific promoters, but not genome-wide. Finally, we examined the role of CRE combinations in predicting regulatory differences between species. Attempts to predict regulatory divergence genome-wide have generally found little correlation between CRE gain/loss and gene expression (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Zhang <em>et al.</em> 2004</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Tirosh <em>et al.</em> 2008</a>). However, gain/loss of CRE combinations can better explain species-specific differences observed in gene expression data.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Multi-species analysis of CRE co-occurrence</h3><p id="p-8">Position weight matrices (PWMs), curated from 11 different literature sources that describe the DNA-binding specificity of 196 <em>S. cerevisiae</em> transcription factors (TFs), were obtained from the ScerTF database (<a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Spivak and Stormo 2012</a>). PWMs were adjusted to account for the genome composition for each species. We then predicted binding sites (CREs) within the genomes of <em>S. cerevisiae</em> and six other <em>sensu stricto</em> and <em>sensu lato Saccharomyces</em> species: <em>S. bayanus</em>, <em>S. castellii</em>, <em>S</em>. <em>kluyveri</em>, <em>S. kudriavzevii</em>, <em>S. mikatae</em>, and <em>S. paradoxus</em>. The use of multiple species increases the sample size and therefore makes it easier to detect co-occurring CREs. This method is similar to that of <a id="xref-ref-15-3" class="xref-bibr" href="#ref-15">Chiang <em>et al.</em> (2003)</a> but differs in two important ways. They used word (hexamer) pairs whereas we use PWMs which should increase the sensitivity by better modeling the specificities of TFs. In addition we do not use alignments between species and are not requiring that the occurrences are orthologous, although we expect that many of them are and that such occurrences increase the signal-to-noise and allow us to find CRE pairs that differ between species. CREs were predicted as DNA sites within 25-fold of the consensus sequence predicted affinity, based on the PWM. This is a conservative cutoff that will miss some functional sites (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Tanay 2006</a>) but it reduces the false positive rate compared to more relaxed cutoffs. For each pair of PWMs, we calculate the co-occurrence of CREs within each promoter region, defined to be 600 bp upstream of each coding region, in each genome. The observed pattern of CRE co-occurrence in a genome is recorded as a distribution of spacings between CRE pairs in every promoter.</p><p id="p-9">If two CREs interact, evidence indicates that the distribution of spacings between CREs will be skewed toward shorter distances (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Drazinic <em>et al.</em> 1996</a>; <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Krogan <em>et al.</em> 2006</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Tirosh and Barkai 2007</a>). In order to take advantage of this observation, we developed a genome simulation method to determine the expected distribution of CRE spacings while maintaining the total occurrence for each CRE and the spatial localization of CREs within each promoter because those are not randomly distributed (functional sites are more common near the promoter than far away) (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Sarafova and Siu 2000</a>). We do this by using permutations that shuffle the CRE annotation associated with each predicted binding site to maintain the number of binding sites associated with each TF and the number and locations of binding sites for every promoter. This shuffling procedure is conducted 1000 times, and the resultant distributions are combined to produce an average expectation.</p><p id="p-10">As expected, the number of co-occurrences in the simulations with a motif spacing of <em>d</em> is closely approximated (see Supporting Information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024331/-/DC1/FigureS1.pdf">Figure S1</a>) by:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/653/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/653/embed/mml-math-1.gif"/></noscript></span></span>where <em>N</em> is the observed number of CRE co-occurrences in the genome and <em>B</em> is maximum possible distance between regulatory elements on a promoter (here defined as 600 bp, minus the combined length of the two CREs being evaluated). Having shown that the observed data are well modeled by the formula, we directly test the likelihood of the observed data under a Poisson model with mean and variance parameterized by <em>S(d)</em> to determine if the number of observed co-occurrences for a CRE pair is significantly more than expected. We use 25 bp as the limit between CREs to consider them to be interacting. In this step, we apply a Bonferroni correction to account for the multiple hypotheses tested. CRE combinations that do not significantly co-occur (<em>P</em> &gt; 0.01) in this step are removed from the analysis.</p><p id="p-11">The subset of CRE pairs that were found to significantly co-occur in multiple <em>Saccharomyces</em> genomes when compared against the genome-wide null model were then compared against a null model derived from promoter-by-promoter simulations of CRE co-occurrence. These simulations are conducted similarly to the genome simulations described above, with the exception that these simulations permute the predicted binding sites at each promoter independently. In this way the number of co-occurrences of each CRE pair within all of the promoters is constant between the simulations and the observed data, and we can examine explicitly the intermotif spacing distribution between CREs in greater detail. Although the individual promoter simulations can be time-intensive, most of the possible CRE combinations are removed in the first step of the analysis, which dramatically reduced the search space. The promoter simulations are necessary to distinguish CREs that co-occur near each other from CREs that regulate a common set of genes but are independently distributed at those genes. We compare the observed distribution to the expected distribution derived from 1000 simulation experiments using a chi-square test. The genome-wide analysis examines both CRE spacing and co-occurrence, while the promoter simulations only examine the spacing between CREs. The promoter simulations correct for a source of bias inherent in the genome-wide analysis. Therefore, a multiple hypothesis correction was not applied in this step.</p></div><div id="sec-3" class="subsection"><h3>Corroborating evidence</h3><div id="sec-4" class="subsection"><h4>ChIP-chip analysis:</h4><p id="p-12">That two predicted CREs occur near one another more frequently than expected does not necessarily mean that they interact to affect gene expression. Immunoprecipitation experiments provide corroborating evidence that TFs are actually bound to the predicted CREs. If two TFs coordinately regulate a set of genes, then both factors need to bind the promoters of those genes. A notable compilation of experiments was conducted by <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Harbison <em>et al.</em> (2004)</a> who collected data for over 100 yeast TFs under several different growth conditions. We also analyzed an earlier ChIP-chip dataset (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Arbeitman <em>et al.</em> 2002</a>) and a more recent ChIP-chip dataset (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Venters <em>et al.</em> 2011</a>). A hyper-geometric test was used to determine if a significant number of probes are bound by both TFs for a candidate CRE pair. ChIP occupancy data provides evidence that two TFs both bind to the same promoters in the same environmental growth condition.</p></div><div id="sec-5" class="subsection"><h4>Target gene expression analysis:</h4><p id="p-13">CRE combinations that functionally interact to coordinately regulate target gene expression should generate similar expression profiles among the genes they regulate (<a id="xref-ref-55-3" class="xref-bibr" href="#ref-55">Pilpel <em>et al.</em> 2001</a>). We calculate the similarity of expression profiles for predicted coregulated genes and for genes predicted to be regulated by only one of the CREs in a pair to assess a functional consequence from the CRE co-occurrences.</p><p id="p-14">Three expression datasets were used to determine if predicted target genes of both CREs were coexpressed across multiple cell cycle time points (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Pramila <em>et al.</em> 2006</a>), environmental conditions (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Gasch <em>et al.</em> 2000</a>) or gene deletion conditions (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Hughes <em>et al.</em> 2000</a>). For each dataset, a Pearsonâs correlation coefficient (PCC) was calculated between gene expression profiles for all pairs of predicted target genes, which produced a distribution of PCC values describing the expression profile similarities of the target genes. This distribution of PCC values for predicted targets of the CRE combination was compared with the distribution of PCC values calculated for expression profiles of the targets in which each CRE was predicted to act in isolation. The distribution of PCC values for predicted target genes of both CREs is compared to the two distributions of PCC values for predicted targets of only one CRE using a one-sided MannâWhitneyâWilcoxin test.</p><p id="p-15">An alternative way to employ gene expression data to identify relationships between TFs is to ask whether a similar set of target genes is significantly up/downregulated in deletion mutants for each of the TFs. <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Reimand <em>et al.</em> (2010)</a> undertook an analysis to identify differentially expressed genes in TF perturbation experiments (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alon 2007</a>). This provides data of potential regulatory target genes for each TF in our analysis. For each CRE combination with a conserved spacing bias, we determined whether there was a significant overlap between target gene sets in TF perturbation experiments using a hyper-geometric test.</p></div><div id="sec-6" class="subsection"><h4>Target gene pathway analysis:</h4><p id="p-16">Previous analyzes of coexpressed genes have found that when a set of genes is coregulated by a combination of TFs, the genes are often involved in a common process or even share a common protein complex (<a id="xref-ref-55-4" class="xref-bibr" href="#ref-55">Pilpel <em>et al.</em> 2001</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Breitkreutz <em>et al.</em> 2008</a>). Therefore, one way to corroborate a CRE pair identified from the co-occurrence screen is to determine if the combination regulates a set of genes with a common biological process. The target genes identified in the co-occurrence screen are used here to define a set of genes with binding sites for both TFs in a potential CRE pair. These predicted target genes are examined to determine whether the genes share a common biological pathway. The GO process (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Ashburner <em>et al.</em> 2000</a>) and KEGG pathway (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Kanehisa and Goto 2000</a>) databases are queried with the target gene set to retrieve all the processes associated with each target gene. A hyper-geometric test is used to determine whether the target genes share a common pathway or process. Target genes may have multiple annotations, so a Bonferroni correction is applied for all of the annotations associated with the target gene set.</p></div></div><div id="sec-7" class="subsection"><h3>Experimental tests of interactions</h3><div id="sec-8" class="subsection"><h4>Yeast strains and growth conditions:</h4><p id="p-17">Yeast strains with <em>c-myc</em> epitopes fused to the C-terminus of the TFs assayed in this study were obtained from the Young Lab (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Harbison <em>et al.</em> 2004</a>). For each of the assayed TFs, a knockout strain was generated in which the predicted cofactor was replaced with a kanamycin resistance marker obtained from the yeast deletion collection (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Giaever <em>et al.</em> 2002</a>). Alleles in the knockout strain were replaced using the yeast gene deletion collection strains as a template with the PCR-based recombination strategy detailed by Giaever <em>et al.</em> (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Giaever <em>et al.</em> 2002</a>).</p><p id="p-18"><a href="http://www.yeastgenome.org/locus/S000002450/overview">NRG1</a>-myc, <a href="http://www.yeastgenome.org/locus/S000003130/overview">SUT1</a>-myc, and <a href="http://www.yeastgenome.org/locus/S000000913/overview">SWI4</a>-myc strains were grown at 30Â° in yeast, peptone, dextrose(YPD)-rich media to exponential midlog phase (OD<sub>600</sub> â¼0.8). GCN4-myc and RTG3-myc strains were grown in YPD-rich media to OD<sub>600</sub> â¼0.7, after which rapamycin was added to the media to a final concentration of 100 nM and the cultures were harvested after 20 min (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Harbison <em>et al.</em> 2004</a>). Strains were grown in 1 l volumes and subsequently split into three equal volumes for chromatin immunoprecipitation. These strains, which harbor both an epitope-tagged transcription factor and a deletion mutant, were assayed in biological triplicate and grown separately in 330 ml culture volumes.</p></div><div id="sec-9" class="subsection"><h4>Chromatin immunoprecipitation:</h4><p id="p-19">Chromatin immunoprecipitations were performed essentially as described previously in the literature (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Aparicio <em>et al.</em> 2004</a>). However, slight modifications were made to the existing protocol to improve yield and reproducibility. Cell cultures were grown to midlog phase (OD<sub>600</sub> â¼0.8) and cross-linked in a final concentration of 1% formaldehyde for 15 min. The reaction was quenched with 150 ml 2.5 M glycine (50 ml for the 330 ml cultures) and incubated at room temperature for 10 min. The cell cultures were centrifuged at 2000 Ã g at 4Â° for 10 min in a Sorvall RC58 centrifuge. This pellet was washed twice with deionized, distilled H<sub>2</sub>O and recentrifuged. The final pellet was frozen at â80Â° overnight.</p><p id="p-20">A cell extract was prepared by first adding lysis buffer (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Tachibana <em>et al.</em> 2005</a>) with protease inhibitor to the frozen pellet and transferring the cell suspension to a 2-ml flat-bottomed screw-cap tube. Zirconia beads (0.5 mm diameter) were added to each tube, and cells were lysed in a beadbeater (BioSpec) set to maximum power for 6 Ã 5 min cycles with a 2 min rest on ice between cycles. This lysate was transferred to a 15 ml conical tube using the hot-needle transfer method (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Aparicio <em>et al.</em> 2004</a>) and the volume of the lysate was increased to 5 ml with lysis buffer. The lysate was then sonicated with a Branson Sonifier 250 tip sonicator set to maximum output for 8 Ã 30 sec cycles with 2 min rest in an ice/ethanol bath between cycles. The lysate was preclarified by centrifugation for 3 min at 3000 Ã g, and then transferred to microcentrifuge tubes and clarified by centrifugation for 7 min at 10,000 Ã g. The supernatant was collected (approximately 4 ml) and used for immunoprecipitation. At this step, a 250 Î¼l sample was removed and labeled as the INPUT sample.</p><p id="p-21">Immunoprecipitations (IPs) were performed using anti-c-myc resin (anti-Myc EZiew affinity gel; Sigma-Aldrich). Each 4 ml sample was split into 4 Ã 1.7 ml microcentrifuge tubes and 50 Î¼l resin was prepared for each microcentrifuge tube (200 Î¼l total for each IP). Resin was washed three times with lysis buffer before use. Samples were incubated by inverting for 14â16 hr at 4Â°. Samples were then centrifuged for 30 sec at 400 Ã g. Each sample was washed six times with 1 ml of the following buffers: one lysis buffer wash, one high-salt buffer wash, two wash buffer washes, and two TE (pH 8.0) washes. Finally, samples were eluted by adding 250 Î¼l elution buffer and incubating at 70Â° for 15 min. From this sample, 200 Î¼l was removed and an additional 100 Î¼l elution buffer was added to the resin. The samples were incubated at 70Â° for an additional 15 min and 100 Î¼l was removed from the sample and pooled with the first elution. Eluates from the four microcentrifuge tubes per sample were pooled for a final volume of 1.2 ml ChIP elution. This was labeled as the IP sample and incubated overnight at 70Â° to reverse cross-links. For the INPUT sample, 250 Î¼l elution buffer was added to the aliquot saved earlier and this sample was also incubated overnight at 70Â°.</p><p id="p-22">After cross-link reversal, the IP sample was concentrated to approximately 500 Î¼l with a vacuum microcentrifuge. Both the IP and INPUT samples were RNAse treated by adding 1 Î¼l 20 mg/ml RNAse and incubating at 40Â° for 30 min. DNA was then isolated by phenol: chloroform extraction. This DNA was precipitated with 1 ml 100% isopropanol and stored overnight at 4Â°. The samples were then centrifuged for 1 hr at max at 4Â°, washed with 75% ethanol, and then recentrifuged for an additional hour at max 4Â°. The supernatant was discarded and the pellet was resuspended in H<sub>2</sub>O.</p><p id="p-23">These samples were then prepared as libraries for Illumina sequencing (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Lefrancois <em>et al.</em> 2009</a>). After an end-repair reaction, an adenosine nucleotide was added to the 3â² end of each strand and sequencing adapters were ligated to the DNA fragments. Fragments were size selected (200â600 bp) and amplified with 15 cycles of PCR. Libraries were sequenced using the Illumina HiSeq-2000 in 42 bp single-read runs (data available in NCBI GEO database: GSE60281).</p></div><div id="sec-10" class="subsection"><h4>Chip-Seq peak analysis:</h4><p id="p-24">The multiplexed sequencing data were then deconvoluted using the indexing barcode and aligned to the yeast genome with Novoalign (Novocraft Technologies). If a sequenced fragment did not uniquely align to the genome it was discarded. Gene promoters were defined as the 600 bp immediately upstream of the translational start site of each gene defined in the <em>Saccharomyces</em> Genome Database. The number of fragments that aligned to these annotated promoters was recorded for each INPUT and IP sample. This converted the data from read alignments to a table of read counts per promoter.</p><p id="p-25">Transcription factor regulatory targets were determined from the wild-type ChIP-Seq experiments. Regulatory targets were determined separately for each of the biological triplicates using the MACS peak-finding algorithm (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Thurman <em>et al.</em> 2012</a>). MACS uses a simple sliding window strategy to compare INPUT and IP samples at each position along a chromosome. The algorithm assumes that the number of reads aligned to any particular window is a Poisson process, so the null hypothesis is that the number of reads that align to the current window in the IP sample can be modeled by a Poisson distribution parameterized using the number of reads that align to that same window in the INPUT sample. Regions with a significantly greater number of reads than expected from the INPUT sample are called âpeaksâ and denote regions of the genome that are bound by the assayed DNA-binding protein. The peaks identified by MACS were used to annotate target genes of the assayed transcription factor; if the MACS peak overlapped with the promoter of a gene, that gene is assumed to be a target of the assayed transcription factor.</p><p id="p-26">Although peak identification was conducted separately for each replicate, annotation of target genes relied on consistency between replicates. Target genes were sorted by support from the peak-finding results for the individual replicates; genes with support from at least two replicates were used to identify joint targets of the TF combination. Gene promoters that were significantly bound in both wild-type strains for a CRE pair were defined as TF combination target genes. In the differential occupancy analysis described below, the statistical test employed is sensitive to sample size. Therefore, the target gene sets defined for each TF binding DNA in isolation were restricted to be the size of the combination target gene set. As such, only the most significant independent target genes from the peak-finding analysis were used to define the genes included in the single TF target gene sets. This analysis of wild-type ChIP data analysis yields, for each TF pair, three equal size gene sets: TF1+TF2 targets, TF1 only targets, and TF2 only targets.</p><p id="p-27">Once the target gene sets were defined for the TF pair and each TF acting in isolation, we examined the difference in occupancy between the wild-type and cofactor deletion strains for each of the three different target gene sets. For each target promoter, we calculated the number of reads that uniquely aligned to that promoter in the INPUT and IP samples and normalized these sums by the total number of million reads in each sample. This calculation transforms the raw read counts to reads per promoter per million mapped. The enrichment ratio for each gene in each IP sample is expressed as the ratio of the IP reads per million mapped divided by the INPUT reads per million mapped. For each gene, we averaged the enrichment ratio across replicates. The cofactor deletion mutant can be considered a âtreatmentâ applied to the target genes for each of the three different gene sets. We would like to determine whether the treatment has an effect on the enrichment ratio (IP/INPUT) for genes within the three different target gene sets. We used a paired T-test to compare the enrichment ratios between wild-type and deletion strains for each gene set. If occupancy of the assayed transcription factor depends on the presence of the predicted cofactor, then the enrichment ratios should be significantly different between the wild-type and deletion strains for the joint targets of the TF combination. If deletion of the cofactor has a more universal effect on the ability of the assayed TF to bind its target promoters, then the enrichment ratios would also be significantly different between treatments for the gene set in which the assayed factor binds promoters without the predicted cofactor.</p></div></div><div id="sec-11" class="subsection"><h3>Identification of species-specific coregulated genes</h3><p id="p-28">Directly comparing gene expression profiles between different species has proven to be a difficult task (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Badis <em>et al.</em> 2009</a>). Therefore, we took an alternative approach to identify regulatory differences between species. We selected the subset of the significant CRE combinations that predict coherent gene expression patterns in both <em>S. cerevisiae</em> and <em>S. bayanus</em> to determine if the CRE combinations regulate different sets of genes between the two <em>Saccharomyces</em> species.</p><p id="p-29">For each candidate CRE combination, the method identifies potential regulatory targets by scanning each genome separately for instances of the CRE combination within 25 bp of each other using the previously described PWMs (<a id="xref-ref-66-3" class="xref-bibr" href="#ref-66">Spivak and Stormo 2012</a>). This produces two sets of potential target genes for each CRE combination; one set contains the predicted targets in <em>S. cerevisiae</em> while the other set contains predicted targets in <em>S. bayanus</em>. In general, there is substantial overlap between these two sets; predicted target genes in <em>S. cerevisiae</em> often have orthologs in <em>S. bayanus</em> that are also predicted to be target genes of the CRE combination using the ortholog mapping from Kellis <em>et al.</em> (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Kellis <em>et al.</em> 2003</a>; <a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Harbison <em>et al.</em> 2004</a>) to assign a unique ortholog to each gene. The overlap in predicted targets produces three sets of genes: target genes predicted in <em>S. cerevisiae</em> but not <em>S. bayanus</em>; target genes predicted for <em>S. bayanus</em> but not <em>S. cerevisiae</em>; and target genes predicted to be regulated by the CRE combination in both species.</p><div id="sec-12" class="subsection"><h4>Initial target gene expression analysis within each species:</h4><p id="p-30">In order to assess regulatory rewiring between the two species, we first test whether the CRE combination can be associated with a coherent gene expression pattern within each species. Therefore, as an initial verification that the CRE combination is functional in each of the species, we calculate the similarity of expression profiles for predicted coregulated genes and for genes regulated by only one of the CREs in a pair to infer a functional consequence from the CRE co-occurrences. If a CRE combination regulates a coherent set of genes in one species but not the other, it is possible that this combination is only functional in one of the species. Alternatively, it is possible that the CRE combination is functional in both species but the appropriate conditions were not assayed in one of the two expression datasets.</p><p id="p-31">A CRE combination may not actively regulate gene expression in all or even most of the conditions assayed by the two datasets considered in this study (<a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Gasch <em>et al.</em> 2000</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Guan <em>et al.</em> 2010</a>). Incorporating irrelevant growth conditions into the initial assessment of a CRE combination will obfuscate corroboration of the CRE combination and complicate downstream analyzes. Therefore, it is important to compare the expression profiles of CRE combination target genes to genes regulated by only one of the CREs in a pair using only the appropriate conditions in which the CREs are most likely to be active. We identify the relevant growth conditions for each TF in a regulatory pair by analyzing a collection of expression profiles published for <em>S. cerevisiae</em> (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Gasch <em>et al.</em> 2000</a>) and <em>S. bayanus</em> (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Guan <em>et al.</em> 2010</a>). For each CRE, the relevant growth conditions are identified by determining whether the CRE target genes are significantly differentially expressed in a condition. A CRE target gene is defined as a gene with a PWM match for that TF above specified cutoff. In each condition, the expression ratios reported for every gene are converted to Z-scores, and we use a Z-test to determine whether the CRE target genes are significantly differentially regulated compared to the expression of all genes in that condition.</p><p id="p-32">Once the relevant conditions were selected, we could evaluate each CRE combination in both <em>S. cerevisiae</em> and <em>S. bayanus</em>. For each dataset, a PCC was calculated between gene expression profiles for all pairs of predicted target genes in each species, which produced a distribution of PCC values describing the expression profile similarities of the target genes. This distribution of PCC values for predicted targets of the CRE combination was compared with the distribution of PCC values calculated for expression profiles of the targets in which each CRE was predicted to act in isolation. The distribution of PCC values for predicted target genes of both CREs is compared to the two distributions of PCC values for predicted targets of only one CRE using a one-sided MannâWhitneyâWilcoxin test.</p></div><div id="sec-13" class="subsection"><h4>Comparison of species-specific gene expression profiles:</h4><p id="p-33">The search for transcriptional rewiring is a search to identify species-specific gene regulation. In the first part of the analysis, we identified CRE combinations that coordinately regulate target gene expression in each species separately and then predicted species-specific targets of the CRE combination. Afterward, the algorithm tests the hypothesis that the predicted species-specific target genes are coherently expressed in the appropriate species while the orthologs of these targets without the CRE combination are not coherently expressed in the partner species.</p><p id="p-34">This procedure generates three sets of genes predicted to be regulated by the CRE combination under consideration: <em>S. cerevisiae</em>-specific target genes (set A), <em>S. bayanus</em>-specific target genes (set C), and species-independent target genes (set B) (see <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> for a graphical description). The species-independent target genes should be regulated by the CRE combination in both <em>S. cerevisiae</em> and <em>S. bayanus</em>. This set of genes provides a benchmark against which we can evaluate the species-specific target genes for <em>S. cerevisiae</em> and <em>S. bayanus</em>. Within <em>S. cerevisiae</em>, the <em>S. cerevisiae</em>-specific target genes should have a similar expression pattern to the shared target genes, while the <em>S. bayanus</em>-specific target genes should not have a similar expression pattern because those genes are not predicted to be regulated by the CRE combination in <em>S. cerevisiae</em>. The opposite pattern should emerge when analyzing the <em>S. bayanus</em>-specific target genes using the <em>S. bayanus</em> gene expression dataset.</p><p id="p-35">The null hypothesis is that there is no transcriptional rewiring between <em>S. cerevisiae</em> and <em>S. bayanus</em>, in which case there should not be any species-specific target genes regulated by this CRE combination. If the null hypothesis is true, then the gene sets designated as <em>A</em> and <em>C</em> are simply false predictions. Either the âspecies-specificâ target genes are not regulated by the CRE combination or they are actually shared target genes regulated by the CRE combination in both species. If either scenario is true, then within each species, the expression profiles of both <em>A</em> and <em>C</em> will be equally similar to the expression profile of <em>B</em>. If the expression profiles of <em>A</em> and <em>C</em> are equally similar to the expression profile observed for <em>B</em>, then we can combine <em>A</em> and <em>C</em> and randomly sample from this pool to generate simulated âspecies-specificâ gene sets, <em>A*</em> and <em>C*</em>. The simulated gene sets <em>A*</em> and <em>C*</em> are each the same size as <em>A</em> and <em>C</em>, respectively; only the composition of the gene sets has been shuffled. We compare the expression profiles of the simulated gene set to <em>B</em> by calculating PCC between each gene in the simulated set and every gene in <em>B</em>. This generates a distribution of PCC that compares the simulated gene set to <em>B</em>. If the null hypothesis is true, then the correlation of <em>A*</em> <em>vs.</em> <em>B</em> should be similar to the correlation of <em>A</em> <em>vs.</em> <em>B</em> in <em>S. cerevisiae</em>, and the correlation of <em>C*</em> <em>vs.</em> <em>B</em> should be similar to the correlation of <em>C</em> <em>vs.</em> <em>B</em> in <em>S. bayanus</em>.</p><p id="p-36">If there is support for regulatory rewiring between <em>S. bayanus</em> and <em>S. cerevisiae</em>, then the species-specific designations are meaningful. As a consequence, the profile comparison between <em>A</em> and <em>B</em> in <em>S. cerevisiae</em> should have a higher mean than the profile comparison between A* and <em>B</em>. Similarly, in <em>S. bayanus</em>, set <em>C</em> should be more similar to <em>B</em> than set <em>C*</em>.</p><p id="p-37">We conduct this simulation 1,000 times to estimate the probability that the predicted species-specific gene sets have the observed expression coherence with the shared target genes by chance. A simulation is counted as successful if the correlation of <em>A*</em> <em>vs.</em> <em>B</em> is greater than the correlation of <em>A</em> <em>vs.</em> <em>B</em> using the <em>S. cerevisiae</em> expression dataset and the correlation of <em>C*</em> <em>vs.</em> <em>B</em> is greater than the correlation of <em>C</em> <em>vs.</em> <em>B</em> in the <em>S. bayanus</em> expression dataset. We estimate the probability that the observed expression profile similarities occurred by chance as the number of successful simulations divided by the total number of simulations.</p></div><div id="sec-14" class="subsection"><h4>Biological pathway analysis:</h4><p id="p-38">A CRE combination that regulates different sets of genes between related species might regulate different biological processes. To investigate this possibility, we assessed the biological pathway enrichment of the three different sets of target genes defined for each CRE combination. An overrepresentation of genes associated with a specific pathway in <em>A</em> but not <em>B</em> and <em>C</em> suggests that the CRE combination regulates that pathway only in <em>S. cerevisiae</em>. Similarly, pathway enrichment apparent in <em>C</em> but not <em>A</em> and <em>B</em> indicates <em>S. bayanus</em>-specific regulation of that pathway.</p><p id="p-39">We determined pathway enrichment for each gene set using the Gene Ontology database of biological processes (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Ashburner <em>et al.</em> 2000</a>) and KEGG database of biological pathways (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Kanehisa and Goto 2000</a>) to retrieve all the processes associated with each target gene. A hyper-geometric test is used to determine whether the target genes share a common pathway or process. Target genes may have multiple annotations, so a Bonferroni correction is applied for all of the annotations associated with the target gene set.</p></div></div><div id="sec-15" class="subsection"><h3>Data availability</h3><p id="p-40">ChIP-seq data accessible from NCBI GEO database: GSE60281.</p></div></div><div class="section results-discussion" id="sec-16"><h2 class="">Results and Discussion</h2><div id="sec-17" class="subsection"><h3>Multiple-species spacing bias predicts combinatorial function of CRE pairs</h3><p id="p-41">PWMs curated from the literature (<a id="xref-ref-66-4" class="xref-bibr" href="#ref-66">Spivak and Stormo 2012</a>) were used to identify potential binding sites for 196 TFs in the genomes of <em>S. cerevisiae</em>, <em>S. bayanus</em>, <em>S. castelli</em>, <em>S. kluyveri</em>, <em>S. kudriavzevii</em>, <em>S. mikatae</em>, and <em>S. paradoxus</em>. For each pair of PWMs, we calculated the distribution of nucleotide spacings between the predicted binding sites and identified pairs for which the observed distribution deviated significantly from random expectation (see <em>Materials and Methods</em>). The co-occurrence screen identified 1399 CRE combinations, 7.3% of the 19,110 possible, that exhibit a conserved spacing bias across multiple <em>Saccharomyces</em> genomes (<em>P</em> &lt; 0.01 after correction for multiple tests). This collection includes many known examples of combinatorial <em>cis</em>-regulation, demonstrating that the screen can successfully identify genuinely functional TF interactions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024331/-/DC1/TableS1.zip">Table S1</a>). Furthermore, although the screen did not require that the CRE combinations occur in every species, in almost all cases they are observed in each of the seven species and usually with similar frequencies (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024331/-/DC1/TableS2.zip">Table S2</a>).</p><p id="p-42">One example of a known case is the highly significant interaction identified between PAC and RRPE elements in ribosomal genes, which are recognized by the TFs <a href="http://www.yeastgenome.org/locus/S000000890/overview">Pbf2</a> and <a href="http://www.yeastgenome.org/locus/S000002576/overview">Stb3</a> (<a id="xref-ref-55-5" class="xref-bibr" href="#ref-55">Pilpel <em>et al.</em> 2001</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Liko <em>et al.</em> 2007</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Zhu <em>et al.</em> 2009</a>). Our analysis also found that the <a href="http://www.yeastgenome.org/locus/S000002576/overview">STB3</a> motif significantly co-occurs with several other motifs involved in the cell cycle (<a href="http://www.yeastgenome.org/locus/S000002214/overview">MBP1</a>, <a href="http://www.yeastgenome.org/locus/S000000913/overview">SWI4</a>), metabolism (<a href="http://www.yeastgenome.org/locus/S000000735/overview">GCN4</a>), and stress response (<a href="http://www.yeastgenome.org/locus/S000001363/overview">XBP1</a>). Previous studies have identified a role for <a href="http://www.yeastgenome.org/locus/S000002576/overview">STB3</a> in the transcriptional regulation of both cell cycle (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Tavazoie <em>et al.</em> 1999</a>) and stress response (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Gasch <em>et al.</em> 2000</a>) genes, indicating that the co-occurrence screen has likely identified functionally relevant CRE interactions.</p></div><div id="sec-18" class="subsection"><h3>Corroborating evidence</h3><div id="sec-19" class="subsection"><h4>Eighty per cent of predictions have corroborating experimental support:</h4><p id="p-43">There are three main features that distinguish CRE combinations from independent CREs. First, when TFs coordinately regulate a set of genes, both factors bind the promoters of those genes (<a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">Harbison <em>et al.</em> 2004</a>). Second, interactions between CREs often produce nonadditive changes in gene expression (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Shea and Ackers 1985</a>; <a id="xref-ref-55-6" class="xref-bibr" href="#ref-55">Pilpel <em>et al.</em> 2001</a>). Third, genes that are coordinately regulated by a particular combination of TFs often share a common biological process (<a id="xref-ref-55-7" class="xref-bibr" href="#ref-55">Pilpel <em>et al.</em> 2001</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Banerjee and Zhang 2003</a>). We analyzed existing ChIP-chip (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Arbeitman <em>et al.</em> 2002</a>; <a id="xref-ref-34-6" class="xref-bibr" href="#ref-34">Harbison <em>et al.</em> 2004</a>; <a id="xref-ref-77-2" class="xref-bibr" href="#ref-77">Venters <em>et al.</em> 2011</a>), gene expression (<a id="xref-ref-24-5" class="xref-bibr" href="#ref-24">Gasch <em>et al.</em> 2000</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Hughes <em>et al.</em> 2000</a>; <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Pramila <em>et al.</em> 2006</a>; <a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Reimand <em>et al.</em> 2010</a>), and biological pathway data (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Ashburner <em>et al.</em> 2000</a>; <a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Kanehisa and Goto 2000</a>) to identify corroborating experimental evidence supporting the computational predictions from our phylogenetic analysis (see <em>Materials and Methods</em>). Of the 1399 pairs of CREs that co-occur in multiple yeast species, 1121 CRE pairs, representing approximately 80% of the computational predictions, have at least one type of experimental evidence supporting the prediction (<em>P</em> &lt; 0.01 in at least one corroborative analysis). Approximately 36% of the predictions are supported by at least two different types of experimental evidence and 8% of the predictions are corroborated by all three experimental methods (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024331/-/DC1/TableS1.zip">Table S1</a>). The number of examples for each type of supporting experimental data depends on the specific thresholds used. But given those sets we can ask whether the specific combinations are significantly overrepresented. In fact all of the combinations are significant at <em>P</em> &lt; 0.05, and for the combination of Chip-chip colocalization and pathway enrichment, and for the combination of all three types of data, the significance is <em>P</em> &lt; 10<sup>â3</sup>. <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> shows four examples with gene expression corroborating evidence. In those examples the genes that have both of the CREs show much more coherent expression than genes that have either of the two CREs alone.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/653/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Expression profiles of predicted CRE combination target genes are more correlated than predicted target genes of either CRE acting alone. The yellow line in each graph depicts the distribution of correlation coefficients calculated between gene expression profiles for each pair of target genes predicted to be regulated by the CRE combination indicated. The black and purple lines relate the distribution of correlations for target genes predicted to be regulated by each CRE acting alone. &#x201C;Opposite&#x201D; refers to a correlation of &#x2212;1 and &#x201C;Identical&#x201D; refers to a correlation of +1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1648113993" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Expression profiles of predicted CRE combination target genes are more correlated than predicted target genes of either CRE acting alone. The yellow line in each graph depicts the distribution of correlation coefficients calculated between gene expression profiles for each pair of target genes predicted to be regulated by the CRE combination indicated. The black and purple lines relate the distribution of correlations for target genes predicted to be regulated by each CRE acting alone. &#x201C;Opposite&#x201D; refers to a correlation of &#x2212;1 and &#x201C;Identical&#x201D; refers to a correlation of +1.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/653/F1.medium.gif" width="440" height="320"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/3/653/F1.medium.gif" width="440" height="320"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/653/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/653/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467676" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-44" class="first-child">Expression profiles of predicted CRE combination target genes are more correlated than predicted target genes of either CRE acting alone. The yellow line in each graph depicts the distribution of correlation coefficients calculated between gene expression profiles for each pair of target genes predicted to be regulated by the CRE combination indicated. The black and purple lines relate the distribution of correlations for target genes predicted to be regulated by each CRE acting alone. âOppositeâ refers to a correlation of â1 and âIdenticalâ refers to a correlation of +1.</p><div class="sb-div caption-clear"></div></div></div><p id="p-45">Most known examples of combinatorial <em>cis</em>-regulation come from a handful of thoroughly studied biological processes (<em>e.g.</em>, cell cycle, starvation, <em>etc</em>.) or have been inferred from high-throughput genetic screens (<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Krogan <em>et al.</em> 2006</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Fordyce <em>et al.</em> 2010</a>). To identify condition-specific CRE pairs, we analyzed target gene expression coherence in three different gene expression datasets: a cell cycle time course experiment (<a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Pramila <em>et al.</em> 2006</a>), a series of growth experiments in multiple environmental stress conditions (<a id="xref-ref-24-6" class="xref-bibr" href="#ref-24">Gasch <em>et al.</em> 2000</a>), and a compendium of gene deletion mutants (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Hughes <em>et al.</em> 2000</a>). Many of the most significant interactions discovered from the cell cycle time course experiments are well known interactions involving the cell cycle regulators <a href="http://www.yeastgenome.org/locus/S000000913/overview">SWI4</a>, <a href="http://www.yeastgenome.org/locus/S000004172/overview">SWI6</a>, and <a href="http://www.yeastgenome.org/locus/S000002214/overview">MBP1</a> and are listed in the <em>Saccharomyces</em> Genome Database (SGD) (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Boyle <em>et al.</em> 2012</a>) (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Our analysis of the environmental and genetic perturbation data, however, mostly identified unknown CRE combinations whose target genes are significantly coexpressed across conditions (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). That most of these interactions have not been documented previously suggests that, unlike the well-studied cell cycle transcriptional network, gene regulation in response to environmental changes remains largely open for new discoveries.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467589/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467589/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Most significant CRE combinations from cell cycle time course expression coherence analysis</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467786/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467786/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Most significant CRE combinations from environmental stress expression coherence analysis</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-20" class="subsection"><h4>Physical constraints of CRE combinations:</h4><p id="p-46">TFs that bind cooperatively to DNA sometimes exhibit a strong bias in the relative position and orientation of their binding sites (<a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Pramila <em>et al.</em> 2002</a>). Previous studies have found that positional constraints on CREs can be important determinants of gene expression patterns (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Sudarsanam <em>et al.</em> 2002</a>). Therefore, a CRE combination with a conserved pattern of binding site arrangements may indicate that the orientation or order of these binding sites influences gene regulation. After identifying CRE combinations that co-occur more than expected by chance, we further analyzed the results from our co-occurrence screen to detect biases in the physical arrangement of CREs for co-occurring CRE pairs. Specifically, we looked for CRE combinations in which one particular orientation or order of binding sites occurred more than expectation and then compared the expression coherence between regulatory targets with the preferred arrangement of binding sites and targets with a different binding site arrangement.</p><p id="p-47"><a href="http://www.yeastgenome.org/locus/S000001592/overview">HAP4</a> and <a href="http://www.yeastgenome.org/locus/S000003130/overview">SUT1</a> is an example of previously undescribed interaction with a preferred orientation (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). Interestingly, this CRE combination may only be active in certain environmental conditions. There is no significant overlap between ChIP-chip experiments for <a href="http://www.yeastgenome.org/locus/S000001592/overview">HAP4</a> and <a href="http://www.yeastgenome.org/locus/S000003130/overview">SUT1</a> when cultured in optimal growth conditions (<a id="xref-ref-34-7" class="xref-bibr" href="#ref-34">Harbison <em>et al.</em> 2004</a>). However, both the expression coherence analysis and the biological pathway analysis support a functional role for the predicted interaction between <a href="http://www.yeastgenome.org/locus/S000001592/overview">HAP4</a> and <a href="http://www.yeastgenome.org/locus/S000003130/overview">SUT1</a>. Additionally, the target genes of the <a href="http://www.yeastgenome.org/locus/S000001592/overview">HAP4</a> Ã <a href="http://www.yeastgenome.org/locus/S000003130/overview">SUT1</a> combination are significantly differentially expressed in oxidative stress conditions and growth on suboptimal carbon sources. After identifying this combination from the co-occurrence screen, we further divided the co-occurrences into each of the four possible orientations and found a significant overrepresentation of one particular orientation among the <em>Saccharomyces</em> genomes. We partitioned the target genes of the <a href="http://www.yeastgenome.org/locus/S000001592/overview">HAP4</a> Ã <a href="http://www.yeastgenome.org/locus/S000003130/overview">SUT1</a> combination into a set with the overrepresented orientation and a set with the three remaining orientations and determined that the set of target genes with the preferred <a href="http://www.yeastgenome.org/locus/S000001592/overview">HAP4</a> Ã <a href="http://www.yeastgenome.org/locus/S000003130/overview">SUT1</a> orientation were significantly more coherently expressed across environmental conditions than the set of target genes without the preferred binding site arrangement.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/653/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Orientation biases for CRE combinations. The pattern of occurrences in multiple species for HAP4 and SUT1 CREs (A) and for FKH2 and SWI4 CREs (B) indicates an overrepresentation of Orientation 1 (depicted in orange). The horizontal line crossing the bar graph represents the expected number of occurrences for each orientation if orientation is random. The expression profile plot depicts the distribution of correlation coefficients calculated between gene expression profiles for target genes with the overrepresented orientation (orange) and all other orientations (black)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1648113993" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Orientation biases for CRE combinations. The pattern of occurrences in multiple species for HAP4 and SUT1 CREs (A) and for FKH2 and SWI4 CREs (B) indicates an overrepresentation of Orientation 1 (depicted in orange). The horizontal line crossing the bar graph represents the expected number of occurrences for each orientation if orientation is random. The expression profile plot depicts the distribution of correlation coefficients calculated between gene expression profiles for target genes with the overrepresented orientation (orange) and all other orientations (black).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/653/F2.medium.gif" width="336" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/3/653/F2.medium.gif" width="336" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/653/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/653/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467736" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-48" class="first-child">Orientation biases for CRE combinations. The pattern of occurrences in multiple species for HAP4 and SUT1 CREs (A) and for FKH2 and SWI4 CREs (B) indicates an overrepresentation of Orientation 1 (depicted in orange). The horizontal line crossing the bar graph represents the expected number of occurrences for each orientation if orientation is random. The expression profile plot depicts the distribution of correlation coefficients calculated between gene expression profiles for target genes with the overrepresented orientation (orange) and all other orientations (black).</p><div class="sb-div caption-clear"></div></div></div><p id="p-49">Several known examples of combinations of CREs with known positional constraints were identified by our method, including <a href="http://www.yeastgenome.org/locus/S000004646/overview">MCM1</a> Ã <a href="http://www.yeastgenome.org/locus/S000004489/overview">YOX1</a> (<a id="xref-ref-56-4" class="xref-bibr" href="#ref-56">Pramila <em>et al.</em> 2002</a>), <a href="http://www.yeastgenome.org/locus/S000004646/overview">MCM1</a> Ã <a href="http://www.yeastgenome.org/locus/S000005012/overview">FKH2</a> (<a id="xref-ref-57-4" class="xref-bibr" href="#ref-57">Pramila <em>et al.</em> 2006</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Tuch <em>et al.</em> 2008b</a>), and <a href="http://www.yeastgenome.org/locus/S000000890/overview">PBF2</a> Ã <a href="http://www.yeastgenome.org/locus/S000002576/overview">STB3</a> (<a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Sudarsanam <em>et al.</em> 2002</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Liko <em>et al.</em> 2007</a>). We also identified the pair <a href="http://www.yeastgenome.org/locus/S000005012/overview">FKH2</a> Ã <a href="http://www.yeastgenome.org/locus/S000000913/overview">SWI4</a> which was previously reported to interact to control expression of S phase genes in the cell cycle (<a id="xref-ref-67-3" class="xref-bibr" href="#ref-67">Sudarsanam <em>et al.</em> 2002</a>) but for which a positional bias had not been reported. <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a> shows that one orientation is much more common than the other three and also that the genes with that position bias are expressed much more coherently than the genes with alternative orientations.</p></div></div><div id="sec-21" class="subsection"><h3>Experimental tests of interactions</h3><div id="sec-22" class="subsection"><h4>ChIP-Seq reveals asymmetry in TF combinatorial interactions:</h4><p id="p-50">As a preliminary assessment of the experimental strategy we immunoprecipitated <a href="http://www.yeastgenome.org/locus/S000004172/overview">Swi6p</a> in a wild-type and <a href="http://www.yeastgenome.org/locus/S000000913/overview">swi4</a>Î strain. <a href="http://www.yeastgenome.org/locus/S000000913/overview">Swi4p</a> and <a href="http://www.yeastgenome.org/locus/S000004172/overview">Swi6p</a> are the two components of the SBF regulatory complex that control G1 to S phase transition during the cell cycle (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Koch <em>et al.</em> 1993</a>). <a href="http://www.yeastgenome.org/locus/S000004172/overview">Swi6p</a> is not believed to have the ability to bind DNA directly, and its association with DNA is mediated by its various cofactors, which include <a href="http://www.yeastgenome.org/locus/S000000913/overview">Swi4p</a>, <a href="http://www.yeastgenome.org/locus/S000002214/overview">Mbp1p</a>, and <a href="http://www.yeastgenome.org/locus/S000005253/overview">Stb1p</a> (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Koch <em>et al.</em> 1993</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Conlan <em>et al.</em> 1999</a>). Therefore, we assayed the <a href="http://www.yeastgenome.org/locus/S000000913/overview">SWI4</a> Ã <a href="http://www.yeastgenome.org/locus/S000004172/overview">SWI6</a> combination to determine if ChIP-Seq can be used to quantitatively measure occupancy differences between wild-type and cofactor deletion strains. Using the wild-type <a href="http://www.yeastgenome.org/locus/S000004172/overview">SWI6</a>::myc18 and <a href="http://www.yeastgenome.org/locus/S000000913/overview">SWI4</a>::myc18 strains, we could define the combinatorial targets and independent targets for the TF pair. In the <a href="http://www.yeastgenome.org/locus/S000004172/overview">SWI6</a>::myc18/<a href="http://www.yeastgenome.org/locus/S000000913/overview">swi4</a>Î strain, <a href="http://www.yeastgenome.org/locus/S000004172/overview">Swi6p</a> should not be able to bind the combinatorial targets of <a href="http://www.yeastgenome.org/locus/S000000913/overview">SWI4</a> Ã <a href="http://www.yeastgenome.org/locus/S000004172/overview">SWI6</a> because the interaction between <a href="http://www.yeastgenome.org/locus/S000000913/overview">Swi4p</a> and <a href="http://www.yeastgenome.org/locus/S000004172/overview">Swi6p</a> has been disrupted. However, the individual targets defined for <a href="http://www.yeastgenome.org/locus/S000004172/overview">Swi6p</a>, in which a ChIP-Seq peak for <a href="http://www.yeastgenome.org/locus/S000004172/overview">Swi6p</a> did not overlap with any ChIP-Seq peaks for <a href="http://www.yeastgenome.org/locus/S000000913/overview">Swi4p</a>, should be relatively unaffected by the loss of <a href="http://www.yeastgenome.org/locus/S000000913/overview">SWI4</a>. If <a href="http://www.yeastgenome.org/locus/S000004172/overview">Swi6p</a> occupancy of these target genes is significantly affected by the deletion of <a href="http://www.yeastgenome.org/locus/S000000913/overview">SWI4</a>, then there is a genetic interaction between <a href="http://www.yeastgenome.org/locus/S000004172/overview">SWI6</a> and <a href="http://www.yeastgenome.org/locus/S000000913/overview">SWI4</a> in which <a href="http://www.yeastgenome.org/locus/S000000913/overview">SWI4</a> globally affects the activity of <a href="http://www.yeastgenome.org/locus/S000004172/overview">SWI6</a>. In this scenario, a physical dependency cannot be inferred because the results do not divorce physical interactions from genetic interactions.</p><p id="p-51">The results of this initial experiment confirm the utility of ChIP-Seq as a method capable of quantitatively measuring the dependence between two TFs at combinatorial target genes and Swi6p-only target genes. In the wild-type <a href="http://www.yeastgenome.org/locus/S000004172/overview">SWI6</a>::myc18 experiments, the IP samples are highly enriched for combinatorial target genes with greater than eightfold enrichment observed for some target genes. However, in the <a href="http://www.yeastgenome.org/locus/S000004172/overview">SWI6</a>::myc18/<a href="http://www.yeastgenome.org/locus/S000000913/overview">swi4</a>Î strain, these target genes are no longer enriched in the IP sample, indicating that <a href="http://www.yeastgenome.org/locus/S000004172/overview">Swi6p</a> occupancy of these target genes is dependent on <a href="http://www.yeastgenome.org/locus/S000000913/overview">SWI4</a> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024331/-/DC1/FigureS2.pdf">Figure S2</a>). A paired-sample Wilcoxin Signed Rank Test comparing the wild-type and deletion experiments reports a significant difference in <a href="http://www.yeastgenome.org/locus/S000004172/overview">Swi6p</a> occupancy between the conditions (<em>P</em> &lt; 10<sup>â5</sup>). In contrast, the difference between the wild-type and <a href="http://www.yeastgenome.org/locus/S000000913/overview">swi4</a>Î strains was not significant when examining the Swi6p-only targets reported by MACS (<em>P</em> = 0.27; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024331/-/DC1/FigureS2.pdf">Figure S2</a>). It is interesting to note that although most genes do not appear different between the two conditions, <a href="http://www.yeastgenome.org/locus/S000004172/overview">Swi6p</a> occupancy of some of these target genes does appear to change between conditions. Of these nine target genes with an occupancy difference between the wild-type and <a href="http://www.yeastgenome.org/locus/S000000913/overview">swi4</a>Î strains, six have a match to the <a href="http://www.yeastgenome.org/locus/S000000913/overview">SWI4</a> binding site in their promoter sequences. One possibility is that these six genes are, in fact, combinatorial genes but were not categorized as such by the MACS peak-finding algorithm. We also examined differences in Swi4p occupancy between wild-type <a href="http://www.yeastgenome.org/locus/S000000913/overview">SWI4</a>::myc18 and <a href="http://www.yeastgenome.org/locus/S000000913/overview">SWI4</a>::myc18/<a href="http://www.yeastgenome.org/locus/S000004172/overview">swi6</a>Î strains. In general there are only small differences between wild-type and deletion conditions for the combinatorial target genes and the Swi4p-only target genes, as expected (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024331/-/DC1/FigureS2.pdf">Figure S2</a>).</p></div><div id="sec-23" class="subsection"><h4>Interaction between NRG1 and SUT1:</h4><p id="p-52">The co-occurrence screen identified a significant spacing bias between <a href="http://www.yeastgenome.org/locus/S000002450/overview">NRG1</a> and <a href="http://www.yeastgenome.org/locus/S000003130/overview">SUT1</a> CRE<em>s</em> (<em>P</em> &lt; 10<sup>â5</sup>). Additionally, predicted target genes of the <a href="http://www.yeastgenome.org/locus/S000002450/overview">NRG1</a> Ã <a href="http://www.yeastgenome.org/locus/S000003130/overview">SUT1</a> CRE combination were significantly more coherently expressed than expected by chance (<em>P</em> &lt; 10<sup>â27</sup>). Based on this data, we investigated the interaction between <a href="http://www.yeastgenome.org/locus/S000002450/overview">NRG1</a> and <a href="http://www.yeastgenome.org/locus/S000003130/overview">SUT1</a> using ChIP-Seq to measure occupancy of <a href="http://www.yeastgenome.org/locus/S000002450/overview">Nrg1p</a> in wild-type and <a href="http://www.yeastgenome.org/locus/S000003130/overview">sut1</a>Î strains as well as the reciprocal experiment for <a href="http://www.yeastgenome.org/locus/S000003130/overview">Sut1p</a>. As depicted in <a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Balaji <em>et al.</em> (2006)</a>, <a id="xref-ref-43-4" class="xref-bibr" href="#ref-43">Kazemian <em>et al.</em> (2013</a>), and <a id="xref-ref-52-4" class="xref-bibr" href="#ref-52">Nandi <em>et al.</em> (2013</a>), <a href="http://www.yeastgenome.org/locus/S000002450/overview">Nrg1p</a> occupancy of <a href="http://www.yeastgenome.org/locus/S000002450/overview">NRG1</a> Ã <a href="http://www.yeastgenome.org/locus/S000003130/overview">SUT1</a> combinatorial targets depends on the presence of <a href="http://www.yeastgenome.org/locus/S000003130/overview">SUT1</a> (<em>P</em> &lt; 10<sup>â6</sup>), while <a href="http://www.yeastgenome.org/locus/S000002450/overview">Nrg1p</a> occupancy of Nrg1p-only targets is much less dependent on <a href="http://www.yeastgenome.org/locus/S000003130/overview">SUT1</a> (<em>P</em> = 0.011). There is an observable difference in occupancy between the wild-type and <a href="http://www.yeastgenome.org/locus/S000003130/overview">sut1</a>Î strains for approximately five genes in the <a href="http://www.yeastgenome.org/locus/S000002450/overview">Nrg1p</a>-only target set; interestingly two of those five genes, <a href="http://www.yeastgenome.org/locus/S000006454/overview">snR63</a> and <a href="http://www.yeastgenome.org/locus/S000002446/overview">YDR039C</a>, have a match to the <a href="http://www.yeastgenome.org/locus/S000003130/overview">SUT1</a> binding site but were not identified as bound regions by MACS in the <a href="http://www.yeastgenome.org/locus/S000003130/overview">Sut1p</a> ChIP-Seq. Removing these two genes from the <a href="http://www.yeastgenome.org/locus/S000002450/overview">Nrg1p</a>-only target set increases the <em>P</em>-value for the comparison between wild-type and <a href="http://www.yeastgenome.org/locus/S000003130/overview">sut1</a>Î strains from 0.011 to 0.032.</p><p id="p-53">In contrast to the results for the <a href="http://www.yeastgenome.org/locus/S000002450/overview">Nrg1</a>, the ChIP-Seq data for <a href="http://www.yeastgenome.org/locus/S000003130/overview">Sut1p</a> shows that <a href="http://www.yeastgenome.org/locus/S000003130/overview">Sut1p</a> occupancy increases in the <a href="http://www.yeastgenome.org/locus/S000002450/overview">nrg1</a>Î strain (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3B</a>). This trend is significant for both the combinatorial and Sut1p-only target gene sets (<em>P</em> &lt; 10<sup>â6</sup> and &lt; 10<sup>â5</sup>, respectively). In this case, it is impossible to determine if <a href="http://www.yeastgenome.org/locus/S000003130/overview">Sut1p</a> physically depends on <a href="http://www.yeastgenome.org/locus/S000002450/overview">Nrg1p</a> for promoter occupancy because there is a genetic interaction between <a href="http://www.yeastgenome.org/locus/S000003130/overview">SUT1</a> and <a href="http://www.yeastgenome.org/locus/S000002450/overview">NRG1</a> in which deletion of <a href="http://www.yeastgenome.org/locus/S000002450/overview">NRG1</a> increases the DNA-binding activity of <a href="http://www.yeastgenome.org/locus/S000003130/overview">Sut1p</a>. It is unclear how deletion of <a href="http://www.yeastgenome.org/locus/S000002450/overview">NRG1</a> exerts a global effect on <a href="http://www.yeastgenome.org/locus/S000003130/overview">Sut1p</a> activity. In both this study and previous studies, <a href="http://www.yeastgenome.org/locus/S000002450/overview">Nrg1p</a> does not appear to associate with the promoter of <a href="http://www.yeastgenome.org/locus/S000003130/overview">SUT1</a> under the conditions of our experiment (<a id="xref-ref-34-8" class="xref-bibr" href="#ref-34">Harbison <em>et al.</em> 2004</a>), and deletion of <a href="http://www.yeastgenome.org/locus/S000002450/overview">NRG1</a> does not significantly affect the expression of <a href="http://www.yeastgenome.org/locus/S000003130/overview">SUT1</a> (<a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Reimand <em>et al.</em> 2010</a>). However, the <a href="http://www.yeastgenome.org/locus/S000003130/overview">SUT1</a> promoter is significantly bound by <a href="http://www.yeastgenome.org/locus/S000002624/overview">Adr1p</a> in cell cultures shifted to low glucose conditions (<a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Tachibana <em>et al.</em> 2005</a>). <a href="http://www.yeastgenome.org/locus/S000002624/overview">ADR1</a> activates expression of genes required for nonoptimal carbon source metabolism in response to glucose starvation (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Kim <em>et al.</em> 2003</a>). Similarly, <a href="http://www.yeastgenome.org/locus/S000002450/overview">NRG1</a> negatively regulates genes required for nonoptimal carbon source metabolism when glucose is present in the growth media (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Mertin <em>et al.</em> 1999</a>). One possibility is that direct regulation of <a href="http://www.yeastgenome.org/locus/S000003130/overview">SUT1</a> by <a href="http://www.yeastgenome.org/locus/S000002624/overview">ADR1</a> indirectly links <a href="http://www.yeastgenome.org/locus/S000003130/overview">SUT1</a> and <a href="http://www.yeastgenome.org/locus/S000002450/overview">NRG1</a> through the glucose sensing network.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/653/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Enrichment ratios from ChIP-seq experiments. Target genes for each genotype (see Materials and Methods for details of peak identification). Each of individual experiments from the triplicates is shown (labeled with #). (A) Nrg1 ChIP-seq in NRG1::myc strain (top) and NRG1::myc/sut1&#x394; strain (bottom). (B) Sut1 ChIP-seq in SUT1::myc strain (top) and SUT1::myc/nrg1&#x394; strain (bottom). Gene names are provided in Table S3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1648113993" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Enrichment ratios from ChIP-seq experiments. Target genes for each genotype (see &lt;em&gt;Materials and Methods&lt;/em&gt; for details of peak identification). Each of individual experiments from the triplicates is shown (labeled with #). (A) Nrg1 ChIP-seq in &lt;em&gt;NRG1&lt;/em&gt;::&lt;em&gt;myc&lt;/em&gt; strain (top) and &lt;em&gt;NRG1&lt;/em&gt;::&lt;em&gt;myc/sut1&lt;/em&gt;&#x394; strain (bottom). (B) Sut1 ChIP-seq in &lt;em&gt;SUT1&lt;/em&gt;::&lt;em&gt;myc&lt;/em&gt; strain (top) and &lt;em&gt;SUT1&lt;/em&gt;::&lt;em&gt;myc/nrg1&lt;/em&gt;&#x394; strain (bottom). Gene names are provided in Table S3.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/653/F3.medium.gif" width="285" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/3/653/F3.medium.gif" width="285" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/653/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/653/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467650" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-54" class="first-child">Enrichment ratios from ChIP-seq experiments. Target genes for each genotype (see <em>Materials and Methods</em> for details of peak identification). Each of individual experiments from the triplicates is shown (labeled with #). (A) Nrg1 ChIP-seq in <em>NRG1</em>::<em>myc</em> strain (top) and <em>NRG1</em>::<em>myc/sut1</em>Î strain (bottom). (B) Sut1 ChIP-seq in <em>SUT1</em>::<em>myc</em> strain (top) and <em>SUT1</em>::<em>myc/nrg1</em>Î strain (bottom). Gene names are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024331/-/DC1/TableS3.pdf">Table S3</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-24" class="subsection"><h4>Interaction between GCN4 and RTG3:</h4><p id="p-55">Several of the CRE combinations identified in the co-occurrence screen integrate distinct physiological processes of the cell. For one such CRE combination, <a href="http://www.yeastgenome.org/locus/S000000735/overview">GCN4</a> Ã <a href="http://www.yeastgenome.org/locus/S000000199/overview">RTG3</a>, we used the differential ChIP-Seq assay to investigate dependencies between the TFs involved in regulation. <a href="http://www.yeastgenome.org/locus/S000000735/overview">GCN4</a> is a transcriptional activator that induces expression of amino acid biosynthesis genes in response to nutrient starvation (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Natarajan <em>et al.</em> 2001</a>). <a href="http://www.yeastgenome.org/locus/S000000199/overview">RTG3</a> serves to activate expression of genes involved in the retrograde and TOR (Target Of Rapamycin) pathways (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Butow and Avadhani 2004</a>). The retrograde response signals mitochondrial dysfunction to the nucleus and induces changes in carbohydrate and nitrogen metabolism. The TOR pathway couples nutrient sensing to protein synthesis/degradation (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Raught <em>et al.</em> 2001</a>). Thus, <a href="http://www.yeastgenome.org/locus/S000000735/overview">GCN4</a> and <a href="http://www.yeastgenome.org/locus/S000000199/overview">RTG3</a> regulation should converge in nutrient starvation growth conditions. Indeed, analysis of data from a previous ChIP-chip study (<a id="xref-ref-34-9" class="xref-bibr" href="#ref-34">Harbison <em>et al.</em> 2004</a>) reveals that the regulatory targets bound by <a href="http://www.yeastgenome.org/locus/S000000735/overview">Gcn4p</a> and <a href="http://www.yeastgenome.org/locus/S000000199/overview">Rtg3p</a> significantly overlap (hyper-geometric test, <em>P</em> &lt; 10<sup>â12</sup>) upon treatment with rapamycin. Rapamycin is an antifungal drug that inactivates TOR signaling in <em>S. cerevisiae</em>, which elicits a nutrient starvation response (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Loewith and Hall 2011</a>).</p><p id="p-56">Following the method of Harbison <em>et al.</em> (<a id="xref-ref-34-10" class="xref-bibr" href="#ref-34">Harbison <em>et al.</em> 2004</a>) we treated cell cultures with rapamycin and measured <a href="http://www.yeastgenome.org/locus/S000000735/overview">Gcn4p</a> and <a href="http://www.yeastgenome.org/locus/S000000199/overview">Rtg3p</a> occupancy in wild-type and cofactor deletion strains. The differential ChIP-Seq experiments show that <a href="http://www.yeastgenome.org/locus/S000000735/overview">Gcn4p</a> occupancy of combinatorial target genes is significantly greater in the wild-type yeast strain compared to the <a href="http://www.yeastgenome.org/locus/S000000735/overview">GCN4</a>::myc9/<a href="http://www.yeastgenome.org/locus/S000000199/overview">rtg3</a>Î strain (<em>P</em> = 0.001) (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). The occupancy difference observed for the combinatorial targets is not due to global changes in <a href="http://www.yeastgenome.org/locus/S000000735/overview">Gcn4p</a> activity; occupancy of <a href="http://www.yeastgenome.org/locus/S000000735/overview">Gcn4p</a>-only targets was not significantly different between the two strains (<em>P</em> = 0.6). In contrast, ChIP-Seq analysis of <a href="http://www.yeastgenome.org/locus/S000000199/overview">Rtg3p</a> indicates that <a href="http://www.yeastgenome.org/locus/S000000199/overview">Rtg3p</a> binding is independent of <a href="http://www.yeastgenome.org/locus/S000000735/overview">Gcn4p</a> (<em>P</em> = 0.46) (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a>). These data suggest that <a href="http://www.yeastgenome.org/locus/S000000735/overview">Gcn4p</a> depends on <a href="http://www.yeastgenome.org/locus/S000000199/overview">Rtg3p</a> for occupancy of the <a href="http://www.yeastgenome.org/locus/S000000735/overview">GCN4</a> Ã <a href="http://www.yeastgenome.org/locus/S000000199/overview">RTG3</a> combinatorial target promoters, but <a href="http://www.yeastgenome.org/locus/S000000199/overview">Rtg3p</a> binding is independent of <a href="http://www.yeastgenome.org/locus/S000000735/overview">Gcn4p</a>. Similar results have been observed previously for GCN4-mediated gene regulation (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Devlin <em>et al.</em> 1991</a>). <a href="http://www.yeastgenome.org/locus/S000005160/overview">Rap1p</a> binds the <a href="http://www.yeastgenome.org/locus/S000000535/overview">HIS4</a> promoter independently of <a href="http://www.yeastgenome.org/locus/S000000735/overview">Gcn4p</a>, but <a href="http://www.yeastgenome.org/locus/S000005160/overview">Rap1p</a> binding is required for <a href="http://www.yeastgenome.org/locus/S000000735/overview">Gcn4p</a> activation of <a href="http://www.yeastgenome.org/locus/S000000535/overview">HIS4</a> (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Devlin <em>et al.</em> 1991</a>). In a later study, it was concluded that <a href="http://www.yeastgenome.org/locus/S000005160/overview">Rap1p</a> overcomes a repressive chromatin structure at the <a href="http://www.yeastgenome.org/locus/S000000535/overview">HIS4</a> promoter and increases promoter accessibility for <a href="http://www.yeastgenome.org/locus/S000000735/overview">Gcn4p</a> (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Sierro <em>et al.</em> 2008</a>). <a href="http://www.yeastgenome.org/locus/S000000199/overview">RTG3</a> may act in a similar fashion; although <a href="http://www.yeastgenome.org/locus/S000000199/overview">Rtg3p</a> can act as a transcriptional activator, components of the SAGA chromatin remodeling complex, <a href="http://www.yeastgenome.org/locus/S000002856/overview">Ada2p</a> and <a href="http://www.yeastgenome.org/locus/S000003484/overview">Gcn5p</a>, are required for <a href="http://www.yeastgenome.org/locus/S000000199/overview">Rtg3p</a> activity (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Pray-Grant <em>et al.</em> 2002</a>). <a href="http://www.yeastgenome.org/locus/S000000199/overview">Rtg3p</a> is also known to physically interact with other chromatin remodeling complexes including SLIK (<a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Pray-Grant <em>et al.</em> 2002</a>) and the <a href="http://www.yeastgenome.org/locus/S000000680/overview">Tup1</a>-Cyc8 repressor complex (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Conlan <em>et al.</em> 1999</a>). Additionally, <a href="http://www.yeastgenome.org/locus/S000000199/overview">Rtg3p</a> may recruit the RSC nucleosome-remodeling complex (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Ng <em>et al.</em> 2002</a>). One possible model that accounts for the observed results and is consistent with previous studies involves <a href="http://www.yeastgenome.org/locus/S000000199/overview">Rtg3p</a> altering the chromatin state of the CRE combination target genes to permit <a href="http://www.yeastgenome.org/locus/S000000735/overview">GCN4</a> occupancy.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/653/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Enrichment ratios from ChIP-seq experiments. (A)Gcn4 ChIP-seq in GCN4::myc strain (top) and GCN4::myc/rtg3&#x394; strain (bottom). (B) Rtg3 ChIP-seq in RTG3::myc (top) and RTG3::myc/gcn4&#x394; (bottom). Gene names are provided in Table S4." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1648113993" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Enrichment ratios from ChIP-seq experiments. (A)Gcn4 ChIP-seq in &lt;em&gt;GCN4&lt;/em&gt;::&lt;em&gt;myc&lt;/em&gt; strain (top) and &lt;em&gt;GCN4&lt;/em&gt;::&lt;em&gt;myc/rtg3&lt;/em&gt;&#x394; strain (bottom). (B) Rtg3 ChIP-seq in &lt;em&gt;RTG3&lt;/em&gt;::&lt;em&gt;myc&lt;/em&gt; (top) and &lt;em&gt;RTG3&lt;/em&gt;::&lt;em&gt;myc/gcn4&lt;/em&gt;&#x394; (bottom). Gene names are provided in Table S4.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/653/F4.medium.gif" width="291" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/3/653/F4.medium.gif" width="291" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/653/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/653/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467692" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-57" class="first-child">Enrichment ratios from ChIP-seq experiments. (A)Gcn4 ChIP-seq in <em>GCN4</em>::<em>myc</em> strain (top) and <em>GCN4</em>::<em>myc/rtg3</em>Î strain (bottom). (B) Rtg3 ChIP-seq in <em>RTG3</em>::<em>myc</em> (top) and <em>RTG3</em>::<em>myc/gcn4</em>Î (bottom). Gene names are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024331/-/DC1/TableS4.pdf">Table S4</a>.</p><div class="sb-div caption-clear"></div></div></div></div></div><div id="sec-25" class="subsection"><h3>CRE combinations can identify species-specific gene expression patterns</h3><p id="p-58">Studies of interspecies <em>Saccharomyces</em> hybrids indicate that expression divergence between species is largely a consequence of differences in <em>cis</em>-regulation (<a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Tirosh <em>et al.</em> 2009</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Bullard <em>et al.</em> 2010</a>). The term ârewiringâ refers to differences in gene regulatory connections between species that result from variations in <em>cis</em>-regulatory content (<a id="xref-ref-75-3" class="xref-bibr" href="#ref-75">Tuch <em>et al.</em> 2008a</a>; <a id="xref-ref-81-2" class="xref-bibr" href="#ref-81">Xie <em>et al.</em> 2010</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Reece-Hoyes <em>et al.</em> 2013</a>). However, despite divergence in promoter sequences between species, orthologous genes often display relatively conserved expression patterns (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Weirauch and Hughes 2010</a>). Similarly, gain and loss of CREs between species is only poorly correlated with expression divergence (<a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Tirosh <em>et al.</em> 2008</a>). However, most previous studies have only focused on individual CREs.</p><div id="sec-26" class="subsection"><h4>Expression coherence corresponds to co-occurrences of CREs, not individual CREs:</h4><p id="p-59">Using the CRE combinations identified in our co-occurrence screen, we searched for examples of rewiring between both <em>S. cerevisiae</em> and <em>S. bayanus</em> using two criteria. First, there had to be sets of genes containing the predicted CRE combination in both species, and also sets of genes with the predicted CRE combination that were unique to each species; we are specifically looking for gain and loss of genes regulated by the CRE combination in both species. Second, there must be conditions for which gene expression assays demonstrate that the genes with the CRE combination are coherently expressed in both species. Using expression data from several different environmental conditions for both <em>S. cerevisiae</em> (<a id="xref-ref-24-7" class="xref-bibr" href="#ref-24">Gasch <em>et al.</em> 2000</a>) and <em>S. bayanus</em> (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Guan <em>et al.</em> 2010</a><span class="xref-sep">,</span><a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">2013</a>), 275 CRE combinations met both criteria. We then measured the expression coherence in three sets of genes defined by the occurrence of the CRE combination: set A are the genes with the CRE combination only in <em>S. cerevisiae</em>; set B are the genes with the CRE combination in both <em>S. cerevisiae</em> and <em>S. bayanus</em>; set C are the genes with the CRE combination only in <em>S. bayanus</em> (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/653/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Graphical representation of expression analysis between S. cerevisiae and S. bayanus. In section I CRE combination target genes are predicted in each species and the overlap defines three target gene sets (A, B, C). Section II shows the average correlation coefficient observed when comparing expression profiles of each gene set with gene set B for the two different gene expression datasets. Section III shows a decrease in the average correlation between target gene sets and gene set B when genes are randomly assigned to either set A or C in simulation experiments." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1648113993" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Graphical representation of expression analysis between &lt;em&gt;S. cerevisiae&lt;/em&gt; and &lt;em&gt;S. bayanus&lt;/em&gt;. In section I CRE combination target genes are predicted in each species and the overlap defines three target gene sets (A, B, C). Section II shows the average correlation coefficient observed when comparing expression profiles of each gene set with gene set &lt;em&gt;B&lt;/em&gt; for the two different gene expression datasets. Section III shows a decrease in the average correlation between target gene sets and gene set &lt;em&gt;B&lt;/em&gt; when genes are randomly assigned to either set &lt;em&gt;A&lt;/em&gt; or &lt;em&gt;C&lt;/em&gt; in simulation experiments.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/653/F5.medium.gif" width="440" height="329"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/3/653/F5.medium.gif" width="440" height="329"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/653/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/653/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467583" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-60" class="first-child">Graphical representation of expression analysis between <em>S. cerevisiae</em> and <em>S. bayanus</em>. In section I CRE combination target genes are predicted in each species and the overlap defines three target gene sets (A, B, C). Section II shows the average correlation coefficient observed when comparing expression profiles of each gene set with gene set <em>B</em> for the two different gene expression datasets. Section III shows a decrease in the average correlation between target gene sets and gene set <em>B</em> when genes are randomly assigned to either set <em>A</em> or <em>C</em> in simulation experiments.</p><div class="sb-div caption-clear"></div></div></div><p id="p-61">Of the 275 CRE combinations considered, we identified 81 CRE pairs (<em>P</em> &lt; 0.05; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024331/-/DC1/TableS5.pdf">Table S5</a>) for which the expression profiles of <em>A</em> and <em>B</em>, but not <em>C</em>, were significantly similar in <em>S. cerevisiae</em> while <em>C</em> and <em>B</em>, but not <em>A</em>, were significantly similar in <em>S. bayanus</em>. This result indicates that the CRE combinations we identified have species-specific regulatory targets in both <em>S. cerevisiae</em> and <em>S. bayanus</em>. These regulatory targets have similar expression profiles to the species-independent target genes in the appropriate species, while the remaining genes which lack the CRE combination do not display a similar expression profile. In these cases, gain and loss of a combination of CREs between species accurately predicts gain and loss of expression coherence.</p><p id="p-62"><a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> shows the results for the CRE combination <a href="http://www.yeastgenome.org/locus/S000002214/overview">MBP1</a> Ã <a href="http://www.yeastgenome.org/locus/S000002576/overview">STB3</a>. In <em>S. cerevisiae</em> there is much higher expression coherence between the genes in sets A and B than in C, whereas in <em>S. bayanus</em> the much higher expression coherence is between genes in sets B and C rather than A.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/653/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Expression profile similarity between gene sets for each species. Predicted genes regulated by MBP1 and STB3 CREs (top) for both species and all three gene sets (A, B, C), for MBP1-only predicted genes (middle) and for STB3-only predicted genes (bottom)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1648113993" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Expression profile similarity between gene sets for each species. Predicted genes regulated by MBP1 and STB3 CREs (top) for both species and all three gene sets (A, B, C), for MBP1-only predicted genes (middle) and for STB3-only predicted genes (bottom).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/653/F6.medium.gif" width="256" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/3/653/F6.medium.gif" width="256" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/653/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/653/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467742" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-63" class="first-child">Expression profile similarity between gene sets for each species. Predicted genes regulated by MBP1 and STB3 CREs (top) for both species and all three gene sets (A, B, C), for MBP1-only predicted genes (middle) and for STB3-only predicted genes (bottom).</p><div class="sb-div caption-clear"></div></div></div><p id="p-64">A comparable analysis using individual CREs to predict species-specific gene regulation in <em>S. bayanus</em> and <em>S. cerevisiae</em> to determine if gain/loss of individual CREs can predict differences in expression patterns between species shows no such difference in expression coherence (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). These results mirror the findings of previous attempts to predict genome-wide transcriptional rewiring between species using individual CREs (<a id="xref-ref-82-2" class="xref-bibr" href="#ref-82">Zhang <em>et al.</em> 2004</a>; <a id="xref-ref-73-3" class="xref-bibr" href="#ref-73">Tirosh <em>et al.</em> 2008</a>). <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024331/-/DC1/FigureS3.pdf">Figure S3</a> shows an additional set of 18 pairs that show significant rewiring between <em>S. cerevisiae</em> and <em>S. bayanus</em>.</p></div><div id="sec-27" class="subsection"><h4>Species-specific target gene pathway enrichment:</h4><p id="p-65">An interesting hypothesis is that the CRE combinations with species-specific targets are responsible for regulating different biological processes within each species. The alternative is that there is no enrichment among the different gene sets for distinct biological processes and the species-specific targets have been acquired at random. We examined the different gene sets defined for each CRE combination to determine if any of the gene sets exhibited enrichment for a particular biological process that was exclusive to that gene set. In fact, for several of the CRE combinations with species-specific regulatory targets, at least one of the gene sets (A, B or C) is significantly enriched for a biological pathway not associated with any of the genes in the other two gene sets (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024331/-/DC1/TableS6.pdf">Table S6</a>).</p><p id="p-66">For several of the combinations, more than one of the gene sets has an exclusive biological pathway enrichment. As an example, the CRE combination <a href="http://www.yeastgenome.org/locus/S000004645/overview">ARG80</a> Ã <a href="http://www.yeastgenome.org/locus/S000000735/overview">GCN4</a> regulates genes associated with arginine biosynthesis (GO term 6526) in both <em>S. cerevisiae</em> and <em>S. bayanus</em> (<em>P</em> &lt; 10<sup>â4</sup>), but in <em>S. bayanus</em> the combination is also associated with regulation of lysine metabolism (GO term 9085; <em>P</em> &lt; 10<sup>â3</sup>). <a href="http://www.yeastgenome.org/locus/S000000735/overview">GCN4</a> is a master regulator of amino acid biosynthesis (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Natarajan <em>et al.</em> 2001</a>) and <a href="http://www.yeastgenome.org/locus/S000004645/overview">ARG80</a> is responsible for arginine biosynthesis (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Dubois <em>et al.</em> 1987</a>), so the association with the GO category for arginine biosynthesis is not surprising. However <a href="http://www.yeastgenome.org/locus/S000004645/overview">ARG80</a> is not known to be associated with lysine biosynthesis, so this <em>S. bayanus</em>-specific pathway association could indicate that the regulatory role of <a href="http://www.yeastgenome.org/locus/S000004645/overview">ARG80</a> has expanded in <em>S. bayanus</em>.</p></div></div><div id="sec-28" class="subsection"><h3>Conclusions</h3><p id="p-67">The combination of CREs in a promoter is an important determinant of gene expression patterns but we have only a limited understanding of which TFs interact. We have developed a computational approach to determine if a conserved pattern of CRE spacing in multiple, unaligned genomes can predict combinatorial regulation. The ability of this method to recover known CRE combinations indicates that conserved patterns of CRE clustering can be used to infer modular regulatory function, and extensive supporting evidence also indicates the reliability of the method. Experimental tests of two new interacting TF pairs verified the predictions but also showed asymmetry in the binding requirements. Previous methods to infer combinatorial regulation from CRE proximity often only considered a single genome or use multiple-species alignments as a filter to reduce the size of the genome before assessing CRE co-occurrence. By using multiple species but not requiring aligned orthologous promoters we use more extensive data to identify co-occurring TF pairs and can include examples of rewiring of the regulatory network. TF motif degeneracy complicates the detection of functional <em>cis</em>-regulatory modules for all methods because many nonfunctional CRE co-occurrences will be observed by chance. This effect could be reduced by using DNA accessibility information, but that is often not available. However, if two CREs cluster together in the genome to coordinately regulate gene expression, these observations occur in addition to the random co-occurrences of any two CREs. With enough observations, a nonrandom pattern of CRE clustering can be more easily distinguished from a random pattern. Differences in the gene sets containing significant TF pairs can indicate evolutionary rewiring events, something that is often difficult to predict reliably using only single TF binding site predictions.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-68">We thank all members of the Stormo lab for helpful discussions and suggestions about this work. We thank Drs. Mark Johnston and Barak Cohen and members of his laboratory, especially Dr. Michael White, for useful advice at many stages of the work. We thank the Richard Young laboratory for providing strains. This work was supported by two grants from the National Institutes of Health: GM078222 and HG000249.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-1"><p id="p-69">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024331/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024331/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-70">ChIP-seq data accessible from NCBI GEO database: GSE60281</p></li><li class="fn" id="fn-3"><p id="p-71"><em>Communicating editor: A. Gasch</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-09-23"><span class="received-label">Received </span>September 23, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-01-03"><span class="accepted-label">Accepted </span>January 3, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2016 Spivak and Stormo</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.1" data-doi="10.1186/1471-2164-9-137"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliva</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Topological comparison of methods for predicting transcriptional cooperativity in yeast.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DAguilar%26rft.auinit1%253DD.%26rft.volume%253D9%26rft.spage%253D137%26rft.epage%253D137%26rft.atitle%253DTopological%2Bcomparison%2Bof%2Bmethods%2Bfor%2Bpredicting%2Btranscriptional%2Bcooperativity%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-137%26rft_id%253Dinfo%253Apmid%252F18366726%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-137&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18366726&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.2" data-doi="10.1038/nrg2102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alon</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Network motifs: theory and experimental approaches.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">450</span>â<span class="cit-lpage">461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DAlon%26rft.auinit1%253DU.%26rft.volume%253D8%26rft.issue%253D6%26rft.spage%253D450%26rft.epage%253D461%26rft.atitle%253DNetwork%2Bmotifs%253A%2Btheory%2Band%2Bexperimental%2Bapproaches.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2102%26rft_id%253Dinfo%253Apmid%252F17510665%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2102&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17510665&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246603400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-book" id="cit-6.3.653.3"><div class="cit-metadata unstructured">Aparicio, O., J. V. Geisberg, and K. Struhl, 2004âChromatin immunoprecipitation for determining the association of proteins with specific genomic sequences in vivo. Curr. Protoc. Cell Biol. Sep; Chapter 17: Unit 17.17.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.4" data-doi="10.1126/science.1072152"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arbeitman</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Furlong</span>  <span class="cit-name-given-names">E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Imam</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Null</span>  <span class="cit-name-given-names">B. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Gene expression during the life cycle of Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">297</span>: <span class="cit-fpage">2270</span>â<span class="cit-lpage">2275</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DArbeitman%26rft.auinit1%253DM.%2BN.%26rft.volume%253D297%26rft.issue%253D5590%26rft.spage%253D2270%26rft.epage%253D2275%26rft.atitle%253DGene%2BExpression%2BDuring%2Bthe%2BLife%2BCycle%2Bof%2BDrosophila%2Bmelanogaster%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1072152%26rft_id%253Dinfo%253Apmid%252F12351791%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTcvNTU5MC8yMjcwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.5" data-doi="10.1038/75556"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashburner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ball</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blake</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Gene ontology: tool for the unification of biology. The Gene Ontology Consortium.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">25</span>â<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DAshburner%26rft.auinit1%253DM.%26rft.volume%253D25%26rft.issue%253D1%26rft.spage%253D25%26rft.epage%253D29%26rft.atitle%253DGene%2Bontology%253A%2Btool%2Bfor%2Bthe%2Bunification%2Bof%2Bbiology.%2BThe%2BGene%2BOntology%2BConsortium.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F75556%26rft_id%253Dinfo%253Apmid%252F10802651%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/75556&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10802651&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086884000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.6" data-doi="10.1126/science.1162327"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Badis</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berger</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Philippakis</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Talukder</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gehrke</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Diversity and complexity in DNA recognition by transcription factors.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">324</span>: <span class="cit-fpage">1720</span>â<span class="cit-lpage">1723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1162327%26rft_id%253Dinfo%253Apmid%252F19443739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjQvNTkzNS8xNzIwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.7" data-doi="10.1016/j.jmb.2006.04.029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Balaji</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babu</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iyer</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luscombe</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aravind</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Comprehensive analysis of combinatorial regulation using the transcriptional regulatory network of yeast.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">360</span>: <span class="cit-fpage">213</span>â<span class="cit-lpage">227</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DBalaji%26rft.auinit1%253DS.%26rft.volume%253D360%26rft.issue%253D1%26rft.spage%253D213%26rft.epage%253D227%26rft.atitle%253DComprehensive%2Banalysis%2Bof%2Bcombinatorial%2Bregulation%2Busing%2Bthe%2Btranscriptional%2Bregulatory%2Bnetwork%2Bof%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2006.04.029%26rft_id%253Dinfo%253Apmid%252F16762362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2006.04.029&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16762362&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238844500020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.8" data-doi="10.1093/nar/gkg894"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Banerjee</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">M. Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Identifying cooperativity among transcription factors controlling the cell cycle in yeast.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">7024</span>â<span class="cit-lpage">7031</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DBanerjee%26rft.auinit1%253DN.%26rft.volume%253D31%26rft.issue%253D23%26rft.spage%253D7024%26rft.epage%253D7031%26rft.atitle%253DIdentifying%2Bcooperativity%2Bamong%2Btranscription%2Bfactors%2Bcontrolling%2Bthe%2Bcell%2Bcycle%2Bin%2Byeast%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkg894%26rft_id%253Dinfo%253Apmid%252F14627835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMS8yMy83MDI0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.9" data-doi="10.1016/S0092-8674(04)00304-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beer</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tavazoie</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Predicting gene expression from sequence.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">185</span>â<span class="cit-lpage">198</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBeer%26rft.auinit1%253DM.%2BA.%26rft.volume%253D117%26rft.issue%253D2%26rft.spage%253D185%26rft.epage%253D198%26rft.atitle%253DPredicting%2Bgene%2Bexpression%2Bfrom%2Bsequence.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252804%252900304-6%26rft_id%253Dinfo%253Apmid%252F15084257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(04)00304-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15084257&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220913400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.10" data-doi="10.1101/gr.137323.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boyle</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hariharan</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaub</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Annotation of functional variation in personal genomes using RegulomeDB.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1790</span>â<span class="cit-lpage">1797</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DBoyle%26rft.auinit1%253DA.%2BP.%26rft.volume%253D22%26rft.issue%253D9%26rft.spage%253D1790%26rft.epage%253D1797%26rft.atitle%253DAnnotation%2Bof%2Bfunctional%2Bvariation%2Bin%2Bpersonal%2Bgenomes%2Busing%2BRegulomeDB%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.137323.112%26rft_id%253Dinfo%253Apmid%252F22955989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIyLzkvMTc5MCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMy82NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.11" data-doi="10.1093/nar/gkm1001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breitkreutz</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stark</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reguly</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boucher</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breitkreutz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">The BioGRID Interaction Database: 2008 update.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">D637</span>â<span class="cit-lpage">D640</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm1001%26rft_id%253Dinfo%253Apmid%252F18000002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzNi9zdXBwbF8xL0Q2MzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.12" data-doi="10.1073/pnas.0912959107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bullard</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mostovoy</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dudoit</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brem</span>  <span class="cit-name-given-names">R. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Polygenic and directional regulatory evolution across pathways in Saccharomyces.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">5058</span>â<span class="cit-lpage">5063</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0912959107%26rft_id%253Dinfo%253Apmid%252F20194736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA3LzExLzUwNTgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.13" data-doi="10.1016/S1097-2765(04)00179-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butow</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Avadhani</span>  <span class="cit-name-given-names">N. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Mitochondrial signaling: the retrograde response.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1</span>â<span class="cit-lpage">15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DButow%26rft.auinit1%253DR.%2BA.%26rft.volume%253D14%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D15%26rft.atitle%253DMitochondrial%2Bsignaling%253A%2Bthe%2Bretrograde%2Bresponse.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1097-2765%252804%252900179-0%26rft_id%253Dinfo%253Apmid%252F15068799%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1097-2765(04)00179-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15068799&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220829800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.14" data-doi="10.1093/nar/gkr1029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cherry</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amundsen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balakrishnan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Binkley</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Saccharomyces Genome Database: the genomics resource of budding yeast.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D700</span>â<span class="cit-lpage">D705</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr1029%26rft_id%253Dinfo%253Apmid%252F22110037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MC9EMS9ENzAwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.15" data-doi="10.1186/gb-2003-4-7-r43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chiang</span>  <span class="cit-name-given-names">D. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moses</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kellis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eisen</span>  <span class="cit-name-given-names">M. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Phylogenetically and spatially conserved word pairs associated with gene-expression changes in yeasts.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">R43</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DChiang%26rft.auinit1%253DD.%2BY.%26rft.volume%253D4%26rft.issue%253D7%26rft.spage%253DR43%26rft.epage%253DR43%26rft.atitle%253DPhylogenetically%2Band%2Bspatially%2Bconserved%2Bword%2Bpairs%2Bassociated%2Bwith%2Bgene-expression%2Bchanges%2Bin%2Byeasts.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2003-4-7-r43%26rft_id%253Dinfo%253Apmid%252F12844359%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2003-4-7-r43&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12844359&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.16" data-doi="10.1074/jbc.274.1.205"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conlan</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gounalaki</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hatzis</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tzamarias</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span>â<span class="cit-article-title">The Tup1-Cyc8 protein complex can shift from a transcriptional co-repressor to a transcriptional co-activator.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">274</span>: <span class="cit-fpage">205</span>â<span class="cit-lpage">210</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiological%2BChemistry%26rft.stitle%253DJ%2BBiol%2BChem%26rft.aulast%253DConlan%26rft.auinit1%253DR.%2BS.%26rft.volume%253D274%26rft.issue%253D1%26rft.spage%253D205%26rft.epage%253D210%26rft.atitle%253DThe%2BTup1-Cyc8%2BProtein%2BComplex%2BCan%2BShift%2Bfrom%2Ba%2BTranscriptional%2BCo-repressor%2Bto%2Ba%2BTranscriptional%2BCo-activator%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.274.1.205%26rft_id%253Dinfo%253Apmid%252F9867831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjk6IjI3NC8xLzIwNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMy82NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.17" data-doi="10.1073/pnas.0407365101"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Das</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banerjee</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">M. Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Interacting models of cooperative gene regulation.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">16234</span>â<span class="cit-lpage">16239</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0407365101%26rft_id%253Dinfo%253Apmid%252F15534222%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAxLzQ2LzE2MjM0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.18" data-doi="10.1093/nar/gkr993"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Boer</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">T. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">YeTFaSCo: a database of evaluated yeast transcription factor sequence specificities.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D169</span>â<span class="cit-lpage">D179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr993%26rft_id%253Dinfo%253Apmid%252F22102575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MC9EMS9EMTY5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.19" data-doi="10.1128/MCB.11.7.3642"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Devlin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tice-Baldwin</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shore</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arndt</span>  <span class="cit-name-given-names">K. T.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span>â<span class="cit-article-title">RAP1 is required for BAS1/BAS2- and GCN4-dependent transcription of the yeast HIS4 gene.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">3642</span>â<span class="cit-lpage">3651</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DDevlin%26rft.auinit1%253DC%26rft.volume%253D11%26rft.issue%253D7%26rft.spage%253D3642%26rft.epage%253D3651%26rft.atitle%253DRAP1%2Bis%2Brequired%2Bfor%2BBAS1%252FBAS2-%2Band%2BGCN4-dependent%2Btranscription%2Bof%2Bthe%2Byeast%2BHIS4%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.11.7.3642%26rft_id%253Dinfo%253Apmid%252F1904543%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjExLzcvMzY0MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMy82NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.20" data-doi="10.1371/journal.pcbi.0030099"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doniger</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Frequent gain and loss of functional transcription factor binding sites.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e99</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DDoniger%26rft.auinit1%253DS.%2BW.%26rft.volume%253D3%26rft.issue%253D5%26rft.spage%253De99%26rft.epage%253De99%26rft.atitle%253DFrequent%2Bgain%2Band%2Bloss%2Bof%2Bfunctional%2Btranscription%2Bfactor%2Bbinding%2Bsites.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.0030099%26rft_id%253Dinfo%253Apmid%252F17530920%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.0030099&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17530920&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.21" data-doi="10.1128/MCB.16.6.3187"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drazinic</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smerage</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopez</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">H. V.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span>â<span class="cit-article-title">Activation mechanism of the multifunctional transcription factor repressor-activator protein 1 (Rap1p).</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">3187</span>â<span class="cit-lpage">3196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DDrazinic%26rft.auinit1%253DC.%2BM.%26rft.volume%253D16%26rft.issue%253D6%26rft.spage%253D3187%26rft.epage%253D3196%26rft.atitle%253DActivation%2Bmechanism%2Bof%2Bthe%2Bmultifunctional%2Btranscription%2Bfactor%2Brepressor-activator%2Bprotein%2B1%2B%2528Rap1p%2529%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.16.6.3187%26rft_id%253Dinfo%253Apmid%252F8649429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjE2LzYvMzE4NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMy82NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.22" data-doi="10.1007/BF00331501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dubois</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bercy</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Messenguy</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span>â<span class="cit-article-title">Characterization of two genes, ARGRI and ARGRIII required for specific regulation of arginine metabolism in yeast.</span> <abbr class="cit-jnl-abbrev">Mol. Gen. Genet.</abbr> <span class="cit-vol">207</span>: <span class="cit-fpage">142</span>â<span class="cit-lpage">148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2B%2526%2Bgeneral%2Bgenetics%2B%253A%2B%2BMGG%26rft.stitle%253DMol%2BGen%2BGenet%26rft.aulast%253DDubois%26rft.auinit1%253DE.%26rft.volume%253D207%26rft.issue%253D1%26rft.spage%253D142%26rft.epage%253D148%26rft.atitle%253DCharacterization%2Bof%2Btwo%2Bgenes%252C%2BARGRI%2Band%2BARGRIII%2Brequired%2Bfor%2Bspecific%2Bregulation%2Bof%2Barginine%2Bmetabolism%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00331501%26rft_id%253Dinfo%253Apmid%252F3298999%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00331501&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3298999&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1987G966500019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.23" data-doi="10.1038/nbt.1675"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fordyce</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gerber</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tran</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">De novo identification and biophysical characterization of transcription-factor binding sites with microfluidic affinity analysis.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">970</span>â<span class="cit-lpage">975</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DFordyce%26rft.auinit1%253DP.%2BM.%26rft.volume%253D28%26rft.issue%253D9%26rft.spage%253D970%26rft.epage%253D975%26rft.atitle%253DDe%2Bnovo%2Bidentification%2Band%2Bbiophysical%2Bcharacterization%2Bof%2Btranscription-factor%2Bbinding%2Bsites%2Bwith%2Bmicrofluidic%2Baffinity%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1675%26rft_id%253Dinfo%253Apmid%252F20802496%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1675&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20802496&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000281719100024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.24" data-doi="10.1091/mbc.11.12.4241"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gasch</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spellman</span>  <span class="cit-name-given-names">P. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carmel-Harel</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eisen</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Genomic expression programs in the response of yeast cells to environmental changes.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Cell</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">4241</span>â<span class="cit-lpage">4257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Bof%2Bthe%2BCell%26rft.stitle%253DMol.%2BBiol.%2BCell%26rft.aulast%253DGasch%26rft.auinit1%253DA.%2BP.%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253D4241%26rft.epage%253D4257%26rft.atitle%253DGenomic%2BExpression%2BPrograms%2Bin%2Bthe%2BResponse%2Bof%2BYeast%2BCells%2Bto%2BEnvironmental%2BChanges%26rft_id%253Dinfo%253Adoi%252F10.1091%252Fmbc.11.12.4241%26rft_id%253Dinfo%253Apmid%252F11102521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIxMS8xMi80MjQxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.25" data-doi="10.1126/science.1166426"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerke</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lorenz</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cohen</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Genetic interactions between transcription factors cause natural variation in yeast.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">323</span>: <span class="cit-fpage">498</span>â<span class="cit-lpage">501</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DGerke%26rft.auinit1%253DJ.%26rft.volume%253D323%26rft.issue%253D5913%26rft.spage%253D498%26rft.epage%253D501%26rft.atitle%253DGenetic%2BInteractions%2BBetween%2BTranscription%2BFactors%2BCause%2BNatural%2BVariation%2Bin%2BYeast%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1166426%26rft_id%253Dinfo%253Apmid%252F19164747%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjMvNTkxMy80OTgiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gertz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cohen</span>  <span class="cit-name-given-names">B. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Environment-specific combinatorial cis-regulation in synthetic promoters.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">244</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BSyst.%2BBiol.%26rft.volume%253D5%26rft.spage%253D244%26rft_id%253Dinfo%253Apmid%252F19225457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19225457&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.27" data-doi="10.1038/nature00935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Giaever</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chu</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ni</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connelly</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riles</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Functional profiling of the Saccharomyces cerevisiae genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">418</span>: <span class="cit-fpage">387</span>â<span class="cit-lpage">391</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGiaever%26rft.auinit1%253DG.%26rft.volume%253D418%26rft.issue%253D6896%26rft.spage%253D387%26rft.epage%253D391%26rft.atitle%253DFunctional%2Bprofiling%2Bof%2Bthe%2BSaccharomyces%2Bcerevisiae%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature00935%26rft_id%253Dinfo%253Apmid%252F12140549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature00935&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12140549&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177009700029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.28" data-doi="10.1186/1471-2105-13-25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Girgis</span>  <span class="cit-name-given-names">H. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ovcharenko</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Predicting tissue specific cis-regulatory modules in the human genome using pairs of co-occurring motifs.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DGirgis%26rft.auinit1%253DH.%2BZ.%26rft.volume%253D13%26rft.spage%253D25%26rft.epage%253D25%26rft.atitle%253DPredicting%2Btissue%2Bspecific%2Bcis-regulatory%2Bmodules%2Bin%2Bthe%2Bhuman%2Bgenome%2Busing%2Bpairs%2Bof%2Bco-occurring%2Bmotifs.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-13-25%26rft_id%253Dinfo%253Apmid%252F22313678%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-13-25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22313678&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.29" data-doi="10.1371/journal.pcbi.1000698"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caudy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troyanskaya</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Systematic planning of genome-scale experiments in poorly studied species.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e1000698</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DGuan%26rft.auinit1%253DY.%26rft.volume%253D6%26rft.issue%253D3%26rft.spage%253De1000698%26rft.epage%253De1000698%26rft.atitle%253DSystematic%2Bplanning%2Bof%2Bgenome-scale%2Bexperiments%2Bin%2Bpoorly%2Bstudied%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1000698%26rft_id%253Dinfo%253Apmid%252F20221257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1000698&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20221257&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.30" data-doi="10.1186/1471-2164-14-33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guan</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunham</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Troyanskaya</span>  <span class="cit-name-given-names">O. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Caudy</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Comparative gene expression between two yeast species.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D33%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-33%26rft_id%253Dinfo%253Apmid%252F23324262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-33&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23324262&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.31" data-doi="10.1093/bioinformatics/17.7.608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">GuhaThakurta</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stormo</span>  <span class="cit-name-given-names">G. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">Identifying target sites for cooperatively binding factors.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">608</span>â<span class="cit-lpage">621</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DGuhaThakurta%26rft.auinit1%253DD.%26rft.volume%253D17%26rft.issue%253D7%26rft.spage%253D608%26rft.epage%253D621%26rft.atitle%253DIdentifying%2Btarget%2Bsites%2Bfor%2Bcooperatively%2Bbinding%2Bfactors%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252F17.7.608%26rft_id%253Dinfo%253Apmid%252F11448879%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIxNy83LzYwOCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMy82NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.32" data-doi="10.1098/rstb.2013.0029"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guturu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doxey</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wenger</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bejerano</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Structure-aided prediction of mammalian transcription factor complexes in conserved non-coding elements.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">368</span>: <span class="cit-fpage">20130029</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos.%2BTrans.%2BR.%2BSoc.%2BLond.%2BB%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2013.0029%26rft_id%253Dinfo%253Apmid%252F24218641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjE3OiIzNjgvMTYzMi8yMDEzMDAyOSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMy82NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.33" data-doi="10.1371/journal.pone.0052055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ha</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Polychronidou</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohmann</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">COPS: detecting co-occurrence and spatial arrangement of transcription factor binding motifs in genome-wide datasets.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e52055</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D7%26rft.spage%253De52055%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0052055%26rft_id%253Dinfo%253Apmid%252F23272209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0052055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23272209&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.34" data-doi="10.1038/nature02800"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harbison</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordon</span>  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">T. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rinaldi</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macisaac</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Transcriptional regulatory code of a eukaryotic genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">431</span>: <span class="cit-fpage">99</span>â<span class="cit-lpage">104</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DHarbison%26rft.auinit1%253DC.%2BT.%26rft.volume%253D431%26rft.issue%253D7004%26rft.spage%253D99%26rft.epage%253D104%26rft.atitle%253DTranscriptional%2Bregulatory%2Bcode%2Bof%2Ba%2Beukaryotic%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature02800%26rft_id%253Dinfo%253Apmid%252F15343339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature02800&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15343339&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223641500050&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.35" data-doi="10.1371/journal.pone.0008155"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hong</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">A biophysical model for analysis of transcription factor interaction and binding site arrangement from genome-wide binding data.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e8155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DHe%26rft.auinit1%253DX.%26rft.volume%253D4%26rft.issue%253D12%26rft.spage%253De8155%26rft.epage%253De8155%26rft.atitle%253DA%2Bbiophysical%2Bmodel%2Bfor%2Banalysis%2Bof%2Btranscription%2Bfactor%2Binteraction%2Band%2Bbinding%2Bsite%2Barrangement%2Bfrom%2Bgenome-wide%2Bbinding%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0008155%26rft_id%253Dinfo%253Apmid%252F19956545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0008155&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19956545&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hooper</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boue</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krause</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">L. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mason</span>  <span class="cit-name-given-names">C. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Identification of tightly regulated groups of genes during Drosophila melanogaster embryogenesis.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">72</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BSyst.%2BBiol.%26rft.volume%253D3%26rft.spage%253D72%26rft_id%253Dinfo%253Apmid%252F17224916%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17224916&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.37" data-doi="10.1186/gb-2007-8-12-r257"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Collins</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Prediction of synergistic transcription factors by function conservation.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">R257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DHu%26rft.auinit1%253DZ.%26rft.volume%253D8%26rft.issue%253D12%26rft.spage%253DR257%26rft.epage%253DR257%26rft.atitle%253DPrediction%2Bof%2Bsynergistic%2Btranscription%2Bfactors%2Bby%2Bfunction%2Bconservation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2007-8-12-r257%26rft_id%253Dinfo%253Apmid%252F18053230%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2007-8-12-r257&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18053230&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.38" data-doi="10.1534/genetics.113.153262"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Boer</span>  <span class="cit-name-given-names">C. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Mapping yeast transcriptional networks.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">9</span>â<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHughes%26rft.auinit1%253DT.%2BR.%26rft.volume%253D195%26rft.issue%253D1%26rft.spage%253D9%26rft.epage%253D36%26rft.atitle%253DMapping%2BYeast%2BTranscriptional%2BNetworks%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.153262%26rft_id%253Dinfo%253Apmid%252F24018767%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6NzoiMTk1LzEvOSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMy82NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.39" data-doi="10.1016/S0092-8674(00)00015-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">T. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marton</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoughton</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Functional discovery via a compendium of expression profiles.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">109</span>â<span class="cit-lpage">126</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DHughes%26rft.auinit1%253DT.%2BR.%26rft.volume%253D102%26rft.issue%253D1%26rft.spage%253D109%26rft.epage%253D126%26rft.atitle%253DFunctional%2Bdiscovery%2Bvia%2Ba%2Bcompendium%2Bof%2Bexpression%2Bprofiles.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252900015-5%26rft_id%253Dinfo%253Apmid%252F10929718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)00015-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10929718&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000088091700013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.40" data-doi="10.1093/nar/gkt1302"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">CCAT: Combinatorial Code Analysis Tool for transcriptional regulation.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">2833</span>â<span class="cit-lpage">2847</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1302%26rft_id%253Dinfo%253Apmid%252F24366875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjQyLzUvMjgzMyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMy82NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.41" data-doi="10.1093/nar/28.1.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kanehisa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goto</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">KEGG: Kyoto encyclopedia of genes and genomes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">27</span>â<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DKanehisa%26rft.auinit1%253DM.%26rft.volume%253D28%26rft.issue%253D1%26rft.spage%253D27%26rft.epage%253D30%26rft.atitle%253DKEGG%253A%2BKyoto%2BEncyclopedia%2Bof%2BGenes%2Band%2BGenomes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F28.1.27%26rft_id%253Dinfo%253Apmid%252F10592173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjc6IjI4LzEvMjciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.42" data-doi="10.1186/gb-2004-5-8-r56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kato</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hata</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banerjee</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Futcher</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">M. Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">Identifying combinatorial regulation of transcription factors and binding motifs.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">R56</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DKato%26rft.auinit1%253DM.%26rft.volume%253D5%26rft.issue%253D8%26rft.spage%253DR56%26rft.epage%253DR56%26rft.atitle%253DIdentifying%2Bcombinatorial%2Bregulation%2Bof%2Btranscription%2Bfactors%2Band%2Bbinding%2Bmotifs.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2004-5-8-r56%26rft_id%253Dinfo%253Apmid%252F15287978%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2004-5-8-r56&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15287978&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.43" data-doi="10.1093/nar/gkt598"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kazemian</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pham</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wolfe</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brodsky</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinha</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Widespread evidence of cooperative DNA binding by transcription factors in Drosophila development.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">8237</span>â<span class="cit-lpage">8252</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt598%26rft_id%253Dinfo%253Apmid%252F23847101%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MS8xNy84MjM3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.44" data-doi="10.1038/nature01644"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kellis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Endrizzi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birren</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lander</span>  <span class="cit-name-given-names">E. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Sequencing and comparison of yeast species to identify genes and regulatory elements.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">423</span>: <span class="cit-fpage">241</span>â<span class="cit-lpage">254</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKellis%26rft.auinit1%253DM.%26rft.volume%253D423%26rft.issue%253D6937%26rft.spage%253D241%26rft.epage%253D254%26rft.atitle%253DSequencing%2Band%2Bcomparison%2Bof%2Byeast%2Bspecies%2Bto%2Bidentify%2Bgenes%2Band%2Bregulatory%2Belements.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01644%26rft_id%253Dinfo%253Apmid%252F12748633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01644&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12748633&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182853100033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.45" data-doi="10.1016/S0006-291X(03)00939-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hatfield</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">A novel TBP-interacting zinc finger protein represses transcription by inhibiting the recruitment of TFIIA and TFIIB.</span> <abbr class="cit-jnl-abbrev">Biochem. Biophys. Res. Commun.</abbr> <span class="cit-vol">306</span>: <span class="cit-fpage">231</span>â<span class="cit-lpage">238</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2Band%2Bbiophysical%2Bresearch%2Bcommunications%26rft.stitle%253DBiochem%2BBiophys%2BRes%2BCommun%26rft.aulast%253DKim%26rft.auinit1%253DM.%26rft.volume%253D306%26rft.issue%253D1%26rft.spage%253D231%26rft.epage%253D238%26rft.atitle%253DA%2Bnovel%2BTBP-interacting%2Bzinc%2Bfinger%2Bprotein%2Brepresses%2Btranscription%2Bby%2Binhibiting%2Bthe%2Brecruitment%2Bof%2BTFIIA%2Band%2BTFIIB.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0006-291X%252803%252900939-2%26rft_id%253Dinfo%253Apmid%252F12788093%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0006-291X(03)00939-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12788093&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.46" data-doi="10.1126/science.8372350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koch</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moll</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neuberg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ahorn</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nasmyth</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span>â<span class="cit-article-title">A role for the transcription factors Mbp1 and Swi4 in progression from G1 to S phase.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">261</span>: <span class="cit-fpage">1551</span>â<span class="cit-lpage">1557</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKoch%26rft.auinit1%253DC%26rft.volume%253D261%26rft.issue%253D5128%26rft.spage%253D1551%26rft.epage%253D1557%26rft.atitle%253DA%2Brole%2Bfor%2Bthe%2Btranscription%2Bfactors%2BMbp1%2Band%2BSwi4%2Bin%2Bprogression%2Bfrom%2BG1%2Bto%2BS%2Bphase%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.8372350%26rft_id%253Dinfo%253Apmid%252F8372350%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNjEvNTEyOC8xNTUxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.47" data-doi="10.1038/nature04670"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krogan</span>  <span class="cit-name-given-names">N. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cagney</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Global landscape of protein complexes in the yeast Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">440</span>: <span class="cit-fpage">637</span>â<span class="cit-lpage">643</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKrogan%26rft.auinit1%253DN.%2BJ.%26rft.volume%253D440%26rft.issue%253D7084%26rft.spage%253D637%26rft.epage%253D643%26rft.atitle%253DGlobal%2Blandscape%2Bof%2Bprotein%2Bcomplexes%2Bin%2Bthe%2Byeast%2BSaccharomyces%2Bcerevisiae.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature04670%26rft_id%253Dinfo%253Apmid%252F16554755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature04670&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16554755&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236350400036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.48" data-doi="10.1186/1471-2164-10-37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lefrancois</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Euskirchen</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auerbach</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rozowsky</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Efficient yeast ChIP-Seq using multiplex short-read DNA sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DLefrancois%26rft.auinit1%253DP.%26rft.volume%253D10%26rft.spage%253D37%26rft.epage%253D37%26rft.atitle%253DEfficient%2Byeast%2BChIP-Seq%2Busing%2Bmultiplex%2Bshort-read%2BDNA%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-37%26rft_id%253Dinfo%253Apmid%252F19159457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-37&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19159457&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.49" data-doi="10.1074/jbc.M704762200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liko</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slattery</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heideman</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Stb3 binds to ribosomal RNA processing element motifs that control transcriptional responses to growth in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">282</span>: <span class="cit-fpage">26623</span>â<span class="cit-lpage">26628</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M704762200%26rft_id%253Dinfo%253Apmid%252F17616518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjEyOiIyODIvMzYvMjY2MjMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.50" data-doi="10.1534/genetics.111.133363"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loewith</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hall</span>  <span class="cit-name-given-names">M. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Target of rapamycin (TOR) in nutrient signaling and growth control.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">1177</span>â<span class="cit-lpage">1201</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLoewith%26rft.auinit1%253DR.%26rft.volume%253D189%26rft.issue%253D4%26rft.spage%253D1177%26rft.epage%253D1201%26rft.atitle%253DTarget%2Bof%2BRapamycin%2B%2528TOR%2529%2Bin%2BNutrient%2BSignaling%2Band%2BGrowth%2BControl%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.133363%26rft_id%253Dinfo%253Apmid%252F22174183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4OS80LzExNzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.51" data-doi="10.1093/nar/27.5.1359"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mertin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDowall</span>  <span class="cit-name-given-names">S. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harley</span>  <span class="cit-name-given-names">V. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span>â<span class="cit-article-title">The DNA-binding specificity of SOX9 and other SOX proteins.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1359</span>â<span class="cit-lpage">1364</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F27.5.1359%26rft_id%253Dinfo%253Apmid%252F9973626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjI3LzUvMTM1OSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMy82NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.52" data-doi="10.1093/nar/gkt578"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nandi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blais</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ioshikhes</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Identification of cis-regulatory modules in promoters of human genes exploiting mutual positioning of transcription factors.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">8822</span>â<span class="cit-lpage">8841</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt578%26rft_id%253Dinfo%253Apmid%252F23913413%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MS8xOS84ODIyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.53" data-doi="10.1128/MCB.21.13.4347-4368.2001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Natarajan</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Slade</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">Transcriptional profiling shows that Gcn4p is a master regulator of gene expression during amino acid starvation in yeast.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">4347</span>â<span class="cit-lpage">4368</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DNatarajan%26rft.auinit1%253DK.%26rft.volume%253D21%26rft.issue%253D13%26rft.spage%253D4347%26rft.epage%253D4368%26rft.atitle%253DTranscriptional%2BProfiling%2BShows%2Bthat%2BGcn4p%2BIs%2Ba%2BMaster%2BRegulator%2Bof%2BGene%2BExpression%2Bduring%2BAmino%2BAcid%2BStarvation%2Bin%2BYeast%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.21.13.4347-4368.2001%26rft_id%253Dinfo%253Apmid%252F11390663%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyMS8xMy80MzQ3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.54" data-doi="10.1101/gad.978902"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robert</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Struhl</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Genome-wide location and regulated recruitment of the RSC nucleosome-remodeling complex.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">806</span>â<span class="cit-lpage">819</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DNg%26rft.auinit1%253DH.%2BH.%26rft.volume%253D16%26rft.issue%253D7%26rft.spage%253D806%26rft.epage%253D819%26rft.atitle%253DGenome-wide%2Blocation%2Band%2Bregulated%2Brecruitment%2Bof%2Bthe%2BRSC%2Bnucleosome-remodeling%2Bcomplex%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.978902%26rft_id%253Dinfo%253Apmid%252F11937489%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6ODoiMTYvNy84MDYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.55" data-doi="10.1038/ng724"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pilpel</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sudarsanam</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">G. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">Identifying regulatory networks by combinatorial analysis of promoter elements.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">153</span>â<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DPilpel%26rft.auinit1%253DY.%26rft.volume%253D29%26rft.issue%253D2%26rft.spage%253D153%26rft.epage%253D159%26rft.atitle%253DIdentifying%2Bregulatory%2Bnetworks%2Bby%2Bcombinatorial%2Banalysis%2Bof%2Bpromoter%2Belements.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng724%26rft_id%253Dinfo%253Apmid%252F11547334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng724&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11547334&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171374100020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.56" data-doi="10.1101/gad.1034302"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pramila</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miles</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">GuhaThakurta</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jemiolo</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breeden</span>  <span class="cit-name-given-names">L. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Conserved homeodomain proteins interact with MADS box protein Mcm1 to restrict ECB-dependent transcription to the M/G1 phase of the cell cycle.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">3034</span>â<span class="cit-lpage">3045</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DPramila%26rft.auinit1%253DT.%26rft.volume%253D16%26rft.issue%253D23%26rft.spage%253D3034%26rft.epage%253D3045%26rft.atitle%253DConserved%2Bhomeodomain%2Bproteins%2Binteract%2Bwith%2BMADS%2Bbox%2Bprotein%2BMcm1%2Bto%2Brestrict%2BECB-dependent%2Btranscription%2Bto%2Bthe%2BM%252FG1%2Bphase%2Bof%2Bthe%2Bcell%2Bcycle%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1034302%26rft_id%253Dinfo%253Apmid%252F12464633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE2LzIzLzMwMzQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.57" data-doi="10.1101/gad.1450606"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pramila</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miles</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noble</span>  <span class="cit-name-given-names">W. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Breeden</span>  <span class="cit-name-given-names">L. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">The Forkhead transcription factor Hcm1 regulates chromosome segregation genes and fills the S-phase gap in the transcriptional circuitry of the cell cycle.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">2266</span>â<span class="cit-lpage">2278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DPramila%26rft.auinit1%253DT.%26rft.volume%253D20%26rft.issue%253D16%26rft.spage%253D2266%26rft.epage%253D2278%26rft.atitle%253DThe%2BForkhead%2Btranscription%2Bfactor%2BHcm1%2Bregulates%2Bchromosome%2Bsegregation%2Bgenes%2Band%2Bfills%2Bthe%2BS-phase%2Bgap%2Bin%2Bthe%2Btranscriptional%2Bcircuitry%2Bof%2Bthe%2Bcell%2Bcycle.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1450606%26rft_id%253Dinfo%253Apmid%252F16912276%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjIwLzE2LzIyNjYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.58" data-doi="10.1128/MCB.22.24.8774-8786.2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pray-Grant</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schieltz</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McMahon</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wood</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kennedy</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">The novel SLIK histone acetyltransferase complex functions in the yeast retrograde response pathway.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">8774</span>â<span class="cit-lpage">8786</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DPray-Grant%26rft.auinit1%253DM.%2BG.%26rft.volume%253D22%26rft.issue%253D24%26rft.spage%253D8774%26rft.epage%253D8786%26rft.atitle%253DThe%2BNovel%2BSLIK%2BHistone%2BAcetyltransferase%2BComplex%2BFunctions%2Bin%2Bthe%2BYeast%2BRetrograde%2BResponse%2BPathway%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.22.24.8774-8786.2002%26rft_id%253Dinfo%253Apmid%252F12446794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8yNC84Nzc0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.59" data-doi="10.1073/pnas.121145898"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raught</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gingras</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sonenberg</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">The target of rapamycin (TOR) proteins.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">7037</span>â<span class="cit-lpage">7044</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DRaught%26rft.auinit1%253DB.%26rft.volume%253D98%26rft.issue%253D13%26rft.spage%253D7037%26rft.epage%253D7044%26rft.atitle%253DThe%2Btarget%2Bof%2Brapamycin%2B%2528TOR%2529%2Bproteins%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.121145898%26rft_id%253Dinfo%253Apmid%252F11416184%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTgvMTMvNzAzNyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMy82NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.60" data-doi="10.1016/j.molcel.2013.05.018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reece-Hoyes</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pons</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diallo</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mori</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shrestha</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Extensive rewiring and complex evolutionary dynamics in a C. elegans multiparameter transcription factor network.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">116</span>â<span class="cit-lpage">127</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BCell%26rft.volume%253D51%26rft.spage%253D116%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2013.05.018%26rft_id%253Dinfo%253Apmid%252F23791784%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2013.05.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23791784&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321794100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.61" data-doi="10.1093/nar/gkq232"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reimand</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vaquerizas</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Todd</span>  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vilo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luscombe</span>  <span class="cit-name-given-names">N. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Comprehensive reanalysis of transcription factor knockout expression data in Saccharomyces cerevisiae reveals many new targets.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">4768</span>â<span class="cit-lpage">4777</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq232%26rft_id%253Dinfo%253Apmid%252F20385592%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzOC8xNC80NzY4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.62" data-doi="10.1093/nar/28.14.2664"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sarafova</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siu</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Precise arrangement of factor-binding sites is required for murine CD4 promoter function.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2664</span>â<span class="cit-lpage">2671</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSarafova%26rft.auinit1%253DS.%26rft.volume%253D28%26rft.issue%253D14%26rft.spage%253D2664%26rft.epage%253D2671%26rft.atitle%253DPrecise%2Barrangement%2Bof%2Bfactor-binding%2Bsites%2Bis%2Brequired%2Bfor%2Bmurine%2BCD4%2Bpromoter%2Bfunction%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F28.14.2664%26rft_id%253Dinfo%253Apmid%252F10908321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIyOC8xNC8yNjY0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.63" data-doi="10.1016/0022-2836(85)90086-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shea</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ackers</span>  <span class="cit-name-given-names">G. K.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span>â<span class="cit-article-title">The OR control system of bacteriophage lambda. A physical-chemical model for gene regulation.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">181</span>: <span class="cit-fpage">211</span>â<span class="cit-lpage">230</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DShea%26rft.auinit1%253DM.%2BA.%26rft.volume%253D181%26rft.issue%253D2%26rft.spage%253D211%26rft.epage%253D230%26rft.atitle%253DThe%2BOR%2Bcontrol%2Bsystem%2Bof%2Bbacteriophage%2Blambda.%2BA%2Bphysical-chemical%2Bmodel%2Bfor%2Bgene%2Bregulation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252885%252990086-5%26rft_id%253Dinfo%253Apmid%252F3157005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(85)90086-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3157005&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1985ACV2900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.64" data-doi="10.1371/journal.pgen.1002789"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shibata</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sheffield</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fedrigo</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Babbitt</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wortham</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Extensive evolutionary changes in regulatory element activity during human origins are associated with altered gene expression and positive selection.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002789</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DShibata%26rft.auinit1%253DY.%26rft.volume%253D8%26rft.issue%253D6%26rft.spage%253De1002789%26rft.epage%253De1002789%26rft.atitle%253DExtensive%2Bevolutionary%2Bchanges%2Bin%2Bregulatory%2Belement%2Bactivity%2Bduring%2Bhuman%2Borigins%2Bare%2Bassociated%2Bwith%2Baltered%2Bgene%2Bexpression%2Band%2Bpositive%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002789%26rft_id%253Dinfo%253Apmid%252F22761590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002789&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22761590&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.65" data-doi="10.1093/nar/gkm910"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sierro</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Makita</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Hoon</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakai</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">DBTBS: a database of transcriptional regulation in Bacillus subtilis containing upstream intergenic conservation information.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">D93</span>â<span class="cit-lpage">D96</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm910%26rft_id%253Dinfo%253Apmid%252F17962296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE0OiIzNi9zdXBwbF8xL0Q5MyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMy82NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.66" data-doi="10.1093/nar/gkr1180"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spivak</span>  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stormo</span>  <span class="cit-name-given-names">G. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">ScerTF: a comprehensive database of benchmarked position weight matrices for Saccharomyces species.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D162</span>â<span class="cit-lpage">D168</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr1180%26rft_id%253Dinfo%253Apmid%252F22140105%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiI0MC9EMS9EMTYyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.67" data-doi="10.1101/gr.301202"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sudarsanam</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pilpel</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">G. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Genome-wide co-occurrence of promoter elements reveals a cis-regulatory cassette of rRNA transcription motifs in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1723</span>â<span class="cit-lpage">1731</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DSudarsanam%26rft.auinit1%253DP.%26rft.volume%253D12%26rft.issue%253D11%26rft.spage%253D1723%26rft.epage%253D1731%26rft.atitle%253DGenome-wide%2BCo-occurrence%2Bof%2BPromoter%2BElements%2BReveals%2Ba%2Bcis-Regulatory%2BCassette%2Bof%2BrRNA%2BTranscription%2BMotifs%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.301202%26rft_id%253Dinfo%253Apmid%252F12421759%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxMi8xMS8xNzIzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.68" data-doi="10.1128/MCB.25.6.2138-2146.2005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tachibana</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoo</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tagne</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kacherovsky</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">T. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Combined global localization analysis and transcriptome data identify genes that are directly coregulated by Adr1 and Cat8.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2138</span>â<span class="cit-lpage">2146</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DTachibana%26rft.auinit1%253DC.%26rft.volume%253D25%26rft.issue%253D6%26rft.spage%253D2138%26rft.epage%253D2146%26rft.atitle%253DCombined%2BGlobal%2BLocalization%2BAnalysis%2Band%2BTranscriptome%2BData%2BIdentify%2BGenes%2BThat%2BAre%2BDirectly%2BCoregulated%2Bby%2BAdr1%2Band%2BCat8%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.25.6.2138-2146.2005%26rft_id%253Dinfo%253Apmid%252F15743812%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjI1LzYvMjEzOCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMy82NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.69" data-doi="10.1101/gr.5113606"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tanay</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Extensive low-affinity transcriptional interactions in the yeast genome.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">962</span>â<span class="cit-lpage">972</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5113606%26rft_id%253Dinfo%253Apmid%252F16809671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE2LzgvOTYyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.70" data-doi="10.1038/10343"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tavazoie</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cho</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Church</span>  <span class="cit-name-given-names">G. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span>â<span class="cit-article-title">Systematic determination of genetic network architecture.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">281</span>â<span class="cit-lpage">285</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTavazoie%26rft.auinit1%253DS.%26rft.volume%253D22%26rft.issue%253D3%26rft.spage%253D281%26rft.epage%253D285%26rft.atitle%253DSystematic%2Bdetermination%2Bof%2Bgenetic%2Bnetwork%2Barchitecture.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F10343%26rft_id%253Dinfo%253Apmid%252F10391217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/10343&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10391217&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081125900022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.71" data-doi="10.1038/nature11232"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thurman</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rynes</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Humbert</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vierstra</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maurano</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">The accessible chromatin landscape of the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">489</span>: <span class="cit-fpage">75</span>â<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DThurman%26rft.auinit1%253DR.%2BE.%26rft.volume%253D489%26rft.issue%253D7414%26rft.spage%253D75%26rft.epage%253D82%26rft.atitle%253DThe%2Baccessible%2Bchromatin%2Blandscape%2Bof%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11232%26rft_id%253Dinfo%253Apmid%252F22955617%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11232&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22955617&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308347000040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.72" data-doi="10.1186/gb-2007-8-4-r50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tirosh</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barkai</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Comparative analysis indicates regulatory neofunctionalization of yeast duplicates.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">R50</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DTirosh%26rft.auinit1%253DI.%26rft.volume%253D8%26rft.issue%253D4%26rft.spage%253DR50%26rft.epage%253DR50%26rft.atitle%253DComparative%2Banalysis%2Bindicates%2Bregulatory%2Bneofunctionalization%2Bof%2Byeast%2Bduplicates.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2007-8-4-r50%26rft_id%253Dinfo%253Apmid%252F17411427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2007-8-4-r50&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17411427&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.73"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tirosh</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinberger</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bezalel</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaganovich</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barkai</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">On the relation between promoter divergence and gene expression evolution.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BSyst.%2BBiol.%26rft.volume%253D4%26rft.spage%253D159%26rft_id%253Dinfo%253Apmid%252F18197176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18197176&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.74" data-doi="10.1126/science.1169766"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tirosh</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reikhav</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levy</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barkai</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">A yeast hybrid provides insight into the evolution of gene expression regulation.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">324</span>: <span class="cit-fpage">659</span>â<span class="cit-lpage">662</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTirosh%26rft.auinit1%253DI.%26rft.volume%253D324%26rft.issue%253D5927%26rft.spage%253D659%26rft.epage%253D662%26rft.atitle%253DA%2BYeast%2BHybrid%2BProvides%2BInsight%2Binto%2Bthe%2BEvolution%2Bof%2BGene%2BExpression%2BRegulation%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1169766%26rft_id%253Dinfo%253Apmid%252F19407207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjQvNTkyNy82NTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjUzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.75" data-doi="10.1371/journal.pbio.0060038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tuch</span>  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Galgoczy</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernday</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span><span class="cit-comment">a</span>â<span class="cit-article-title">The evolution of combinatorial gene regulation in fungi.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e38</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DTuch%26rft.auinit1%253DB.%2BB.%26rft.volume%253D6%26rft.issue%253D2%26rft.spage%253De38%26rft.epage%253De38%26rft.atitle%253DThe%2Bevolution%2Bof%2Bcombinatorial%2Bgene%2Bregulation%2Bin%2Bfungi.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0060038%26rft_id%253Dinfo%253Apmid%252F18303948%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0060038&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18303948&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.76" data-doi="10.1126/science.1152398"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tuch</span>  <span class="cit-name-given-names">B. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span><span class="cit-comment">b</span>â<span class="cit-article-title">Evolution of eukaryotic transcription circuits.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">319</span>: <span class="cit-fpage">1797</span>â<span class="cit-lpage">1799</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTuch%26rft.auinit1%253DB.%2BB.%26rft.volume%253D319%26rft.issue%253D5871%26rft.spage%253D1797%26rft.epage%253D1799%26rft.atitle%253DEvolution%2Bof%2BEukaryotic%2BTranscription%2BCircuits%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1152398%26rft_id%253Dinfo%253Apmid%252F18369141%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTkvNTg3MS8xNzk3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.77" data-doi="10.1016/j.molcel.2011.01.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Venters</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wachi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mavrich</span>  <span class="cit-name-given-names">T. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Andersen</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jena</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">A comprehensive genomic binding map of gene and chromatin regulatory proteins in Saccharomyces.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">480</span>â<span class="cit-lpage">492</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DVenters%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D41%26rft.issue%253D4%26rft.spage%253D480%26rft.epage%253D492%26rft.atitle%253DA%2Bcomprehensive%2Bgenomic%2Bbinding%2Bmap%2Bof%2Bgene%2Band%2Bchromatin%2Bregulatory%2Bproteins%2Bin%2BSaccharomyces.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2011.01.015%26rft_id%253Dinfo%253Apmid%252F21329885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2011.01.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21329885&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000287773300012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.78" data-doi="10.1371/journal.pgen.1003414"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Statistical analysis reveals co-expression patterns of many pairs of genes in yeast are jointly regulated by interacting loci.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003414</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003414%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003414%26rft_id%253Dinfo%253Apmid%252F23555313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003414&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23555313&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.79" data-doi="10.1016/j.tig.2009.12.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weirauch</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hughes</span>  <span class="cit-name-given-names">T. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Conserved expression without conserved regulatory sequence: the more things change, the more they stay the same.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">66</span>â<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DWeirauch%26rft.auinit1%253DM.%2BT.%26rft.volume%253D26%26rft.issue%253D2%26rft.spage%253D66%26rft.epage%253D74%26rft.atitle%253DConserved%2Bexpression%2Bwithout%2Bconserved%2Bregulatory%2Bsequence%253A%2Bthe%2Bmore%2Bthings%2Bchange%252C%2Bthe%2Bmore%2Bthey%2Bstay%2Bthe%2Bsame.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2009.12.002%26rft_id%253Dinfo%253Apmid%252F20083321%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2009.12.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20083321&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274987400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.80" data-doi="10.1101/gr.072769.107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cai</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chia</span>  <span class="cit-name-given-names">N. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Cross-species de novo identification of cis-regulatory modules with GibbsModule: application to gene regulation in embryonic stem cells.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1325</span>â<span class="cit-lpage">1335</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.072769.107%26rft_id%253Dinfo%253Apmid%252F18490265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE4LzgvMTMyNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMy82NTMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.81" data-doi="10.1101/gr.100594.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ptaszek</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Rewirable gene regulatory networks in the preimplantation embryonic development of three mammalian species.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">804</span>â<span class="cit-lpage">815</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.100594.109%26rft_id%253Dinfo%253Apmid%252F20219939%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIwLzYvODA0IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.82" data-doi="10.1016/j.tig.2004.07.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">How much expression divergence after yeast gene duplication could be explained by regulatory motif evolution?</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">403</span>â<span class="cit-lpage">407</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DZhang%26rft.auinit1%253DZ.%26rft.volume%253D20%26rft.issue%253D9%26rft.spage%253D403%26rft.epage%253D407%26rft.atitle%253DHow%2Bmuch%2Bexpression%2Bdivergence%2Bafter%2Byeast%2Bgene%2Bduplication%2Bcould%2Bbe%2Bexplained%2Bby%2Bregulatory%2Bmotif%2Bevolution%253F%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2004.07.006%26rft_id%253Dinfo%253Apmid%252F15313547%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2004.07.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15313547&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223726600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.83" data-doi="10.1146/annurev-genom-082908-150139"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gianoulis</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karczewski</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snyder</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Regulatory variation within and between species.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">327</span>â<span class="cit-lpage">346</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenomics%2Band%2Bhuman%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenomics%2BHum%2BGenet%26rft.aulast%253DZheng%26rft.auinit1%253DW.%26rft.volume%253D12%26rft.spage%253D327%26rft.epage%253D346%26rft.atitle%253DRegulatory%2Bvariation%2Bwithin%2Band%2Bbetween%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genom-082908-150139%26rft_id%253Dinfo%253Apmid%252F21721942%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genom-082908-150139&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21721942&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F653.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295819900014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.653.84" data-doi="10.1101/gr.090233.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byers</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCord</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berger</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">High-resolution DNA-binding specificity analysis of yeast transcription factors.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">556</span>â<span class="cit-lpage">566</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.090233.108%26rft_id%253Dinfo%253Apmid%252F19158363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzQvNTU2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY1My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/3/653.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/3/641" title="Vesicular Trafficking Systems Impact TORC1-Controlled Transcriptional Programs in Saccharomyces cerevisiae" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/3/669" title="A Computational Approach to Estimating Nondisjunction Frequency in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/3"><span>Volume 6 Issue 3, March 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/3.cover-source.jpg?itok=8EDC4x5t" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/3/653" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-ygcQ613tNZgsxzRHFib04-bfFR3spHdRjsfbUn5CDb4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/3/653" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/3/653" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/3/653" class="active" data-icon-position="" data-hide-link-title="0">Combinatorial Cis-regulation in Saccharomyces Species</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460080" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-7BdHFVPrHeMf_EjZx7V6MlwFIJlVmdks4wLPbRwKN_o" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/3/653.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/3/653" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-DacuQzu_SklkgSgqZdNl19rQgeWrNdw4s0J4NdtA0KU" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/3/653" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/3/653.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460080" id="node460080--2" data-pisa="ggg;6/3/653" data-pisa-master="ggg;g3.115.024331" data-apath="/ggg/6/3/653.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Combinatorial <em>Cis</em>-regulation in <em>Saccharomyces</em> Species</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Aaron T. Spivak</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-6896-1850" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gary D. Stormo</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">653-667;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.024331 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460080--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Aaron T. Spivak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Center for Genome Sciences and Systems Biology, Washington University School of Medicine, St Louis, Missouri 63108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BT.%2BSpivak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Spivak%20AT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAaron%2BT.%2BSpivak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gary D. Stormo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Center for Genome Sciences and Systems Biology, Washington University School of Medicine, St Louis, Missouri 63108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BD.%2BStormo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stormo%20GD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGary%2BD.%2BStormo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6896-1850" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gary D. Stormo</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:stormo@wustl.edu" class="" data-icon-position="" data-hide-link-title="0">stormo@wustl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/3/653" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/3/653&amp;title=Combinatorial%20Cis-regulation%20in%20Saccharomyces%20Species" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/3/653&amp;title=Combinatorial%20Cis-regulation%20in%20Saccharomyces%20Species" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/3/653&amp;title=Combinatorial%20Cis-regulation%20in%20Saccharomyces%20Species" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/3/653&amp;text=Combinatorial%20Cis-regulation%20in%20Saccharomyces%20Species" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/3/653&amp;title=Combinatorial%20Cis-regulation%20in%20Saccharomyces%20Species" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/3/653&amp;t=Combinatorial%20Cis-regulation%20in%20Saccharomyces%20Species" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/3/653&amp;title=Combinatorial%20Cis-regulation%20in%20Saccharomyces%20Species" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/3/653&amp;title=Combinatorial%20Cis-regulation%20in%20Saccharomyces%20Species" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460080/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460080" id="node460080--3" data-pisa="ggg;6/3/653" data-pisa-master="ggg;g3.115.024331" data-apath="/ggg/6/3/653.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Combinatorial <em>Cis</em>-regulation in <em>Saccharomyces</em> Species</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Aaron T. Spivak</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-6896-1850" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Gary D. Stormo</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">653-667;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.024331 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460080--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Aaron T. Spivak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Center for Genome Sciences and Systems Biology, Washington University School of Medicine, St Louis, Missouri 63108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aaron%2BT.%2BSpivak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Spivak%20AT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAaron%2BT.%2BSpivak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gary D. Stormo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Center for Genome Sciences and Systems Biology, Washington University School of Medicine, St Louis, Missouri 63108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BD.%2BStormo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stormo%20GD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGary%2BD.%2BStormo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-6896-1850" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Gary D. Stormo</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:stormo@wustl.edu" class="" data-icon-position="" data-hide-link-title="0">stormo@wustl.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=653&amp;rft.epage=667&amp;rft.atitle=Combinatorial%20Cis-regulation%20in%20Saccharomyces%20Species&amp;rft.volume=6&amp;rft.issue=3&amp;rft.date=2016-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Spivak%2C+Aaron+T.&amp;rft.au=Stormo%2C+Gary+D."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460080/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460080/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460080/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460080/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460080/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460080/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460080/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460080/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460080/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460080/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460080/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460080/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/3/653" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/3/653.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/3/653.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/3/653.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/3/653.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
