<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/3/559" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/3/559.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/3/559.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/3/559.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/3/559.atom" />
<meta name="HW.pisa" content="ggg;6/3/559" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Novel Heterotypic Rox Sites for Combinatorial Dre Recombination Strategies" />
<meta name="DC.Identifier" content="10.1534/g3.115.025841" />
<meta name="DC.Date" content="2016-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2016 Chuang et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Site-specific recombinases (SSRs) such as Cre are widely used in gene targeting and genetic approaches for cell labeling and manipulation. They mediate DNA strand exchange between two DNA molecules at dedicated recognition sites. Precise understanding of the Cre recombination mechanism, including the role of individual base pairs in its loxP target site, guided the generation of mutant lox sites that specifically recombine with themselves but not with the wild type loxP. This has led to the development of a variety of combinatorial Cre-dependent genetic strategies, such as multicolor reporters, irreversible inversions, or recombination-mediated cassette exchange. Dre, a Cre-related phage integrase that recognizes roxP sites, does not cross-react with the Cre-loxP system, but has similar recombination efficiency. We have previously described intersectional genetic strategies combining Dre and Cre. We now report a mutagenesis screen aimed at identifying roxP base pairs critical for self-recognition. We describe several rox variant sites that are incompatible with roxP, but are able to efficiently recombine with themselves in either purified systems or bacterial and eukaryotic tissue culture systems. These newly identified rox sites are not recognized by Cre, thus enabling potential combinatorial strategies involving Cre, Dre, and target loci including multiple loxP and roxP variants." />
<meta name="DC.Contributor" content="Katherine Chuang" />
<meta name="DC.Contributor" content="Eileen Nguyen" />
<meta name="DC.Contributor" content="Yuri Sergeev" />
<meta name="DC.Contributor" content="Tudor C. Badea" />
<meta name="article:published_time" content="2016-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Novel Heterotypic Rox Sites for Combinatorial Dre Recombination Strategies" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Site-specific recombinases (SSRs) such as Cre are widely used in gene targeting and genetic approaches for cell labeling and manipulation. They mediate DNA strand exchange between two DNA molecules at dedicated recognition sites. Precise understanding of the Cre recombination mechanism, including the role of individual base pairs in its loxP target site, guided the generation of mutant lox sites that specifically recombine with themselves but not with the wild type loxP. This has led to the development of a variety of combinatorial Cre-dependent genetic strategies, such as multicolor reporters, irreversible inversions, or recombination-mediated cassette exchange. Dre, a Cre-related phage integrase that recognizes roxP sites, does not cross-react with the Cre-loxP system, but has similar recombination efficiency. We have previously described intersectional genetic strategies combining Dre and Cre. We now report a mutagenesis screen aimed at identifying roxP base pairs critical for self-recognition. We describe several rox variant sites that are incompatible with roxP, but are able to efficiently recombine with themselves in either purified systems or bacterial and eukaryotic tissue culture systems. These newly identified rox sites are not recognized by Cre, thus enabling potential combinatorial strategies involving Cre, Dre, and target loci including multiple loxP and roxP variants.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/03/01" />
<meta name="citation_mjid" content="ggg;6/3/559" />
<meta name="citation_id" content="6/3/559" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/3/559" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/3/559.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/3/559.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/3/559.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.025841" />
<meta name="citation_pmid" content="26715092" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="559" />
<meta name="citation_lastpage" content="571" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Katherine Chuang" />
<meta name="citation_author_institution" content="Retinal Circuits Development and Genetics Unit, National Eye Institute, National Institutes of Health, Bethesda, Maryland 20892" />
<meta name="citation_author" content="Eileen Nguyen" />
<meta name="citation_author_institution" content="Retinal Circuits Development and Genetics Unit, National Eye Institute, National Institutes of Health, Bethesda, Maryland 20892" />
<meta name="citation_author" content="Yuri Sergeev" />
<meta name="citation_author_institution" content="Retinal Circuits Development and Genetics Unit, National Eye Institute, National Institutes of Health, Bethesda, Maryland 20892" />
<meta name="citation_author" content="Tudor C. Badea" />
<meta name="citation_author_institution" content="Retinal Circuits Development and Genetics Unit, National Eye Institute, National Institutes of Health, Bethesda, Maryland 20892" />
<meta name="citation_author_email" content="badeatc@mail.nih.gov" />
<meta name="citation_reference" content="citation_journal_title=Dis. Model. Mech.;citation_author=K. Anastassiadis;citation_author=J. Fu;citation_author=C. Patsch;citation_author=S. Hu;citation_author=S. Weidlich;citation_title=Dre recombinase, like Cre, is a highly efficient site-specific recombinase in E. coli, mammalian cells and mice.;citation_volume=2;citation_year=2009;citation_pmid=19692579;citation_doi=10.1242/dmm.003087" />
<meta name="citation_reference" content="citation_journal_title=Journal of Neuroscience;citation_journal_abbrev=J. Neurosci.;citation_author=D. Atasoy;citation_author=Y. Aponte;citation_author=H. H. Su;citation_author=S. M. Sternson;citation_title=A FLEX Switch Targets Channelrhodopsin-2 to Multiple Cell Types for Imaging and Long-Range Circuit Mapping;citation_pages=7025-7030;citation_volume=28;citation_year=2008;citation_issue=28;citation_pmid=18614669;citation_doi=10.1523/JNEUROSCI.1954-08.2008" />
<meta name="citation_reference" content="citation_journal_title=Neuron;citation_author=T. C. Badea;citation_author=H. Cahill;citation_author=J. Ecker;citation_author=S. Hattar;citation_author=J. Nathans;citation_title=Distinct roles of transcription factors brn3a and brn3b in controlling the development, morphology, and function of retinal ganglion cells.;citation_pages=852-864;citation_volume=61;citation_year=2009;citation_pmid=19323995;citation_doi=10.1016/j.neuron.2009.01.020" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=M.-C. Birling;citation_author=F. Gofflot;citation_author=X. Warot;citation_title=Site-specific recombinases for manipulation of the mouse genome.;citation_pages=245-263;citation_volume=561;citation_year=2009;citation_pmid=19504076;citation_doi=10.1007/978-1-60327-019-9_16" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=J. Bischof;citation_author=K. Basler;citation_title=Recombinases and their use in gene activation, gene inactivation, and transgenesis.;citation_pages=175-195;citation_volume=420;citation_year=2008;citation_pmid=18641947;citation_doi=10.1007/978-1-59745-583-1_10" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=G. Churchward;citation_author=D. Belin;citation_author=Y. Nagamine;citation_title=A pSC101-derived plasmid which shows no sequence homology to other commonly used cloning vectors.;citation_pages=165-171;citation_volume=31;citation_year=1984;citation_issue=1-3;citation_pmid=6098521;citation_doi=10.1016/0378-1119(84)90207-5" />
<meta name="citation_reference" content="citation_journal_title=Neuron;citation_journal_abbrev=Neuron;citation_author=A. F. Farago;citation_author=R. B. Awatramani;citation_author=S. M. Dymecki;citation_title=Assembly of the brainstem cochlear nuclear complex is revealed by intersectional and subtractive genetic fate maps.;citation_pages=205-218;citation_volume=50;citation_year=2006;citation_issue=2;citation_pmid=16630833;citation_doi=10.1016/j.neuron.2006.03.014" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=L. E. Fenno;citation_author=J. Mattis;citation_author=C. Ramakrishnan;citation_author=M. Hyun;citation_author=S. Y. Lee;citation_title=Targeting cells with single vectors using multiple-feature Boolean logic.;citation_pages=763-772;citation_volume=11;citation_year=2014;citation_pmid=24908100;citation_doi=10.1038/nmeth.2996" />
<meta name="citation_reference" content="citation_journal_title=Methods (San Diego, Calif.);citation_journal_abbrev=Methods;citation_author=K. Ghosh;citation_author=G. D. Van Duyne;citation_title=Cre-loxP biochemistry.;citation_pages=374-383;citation_volume=28;citation_year=2002;citation_issue=3;citation_pmid=12431441;citation_doi=10.1016/S1046-2023(02)00244-X" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in structural biology;citation_journal_abbrev=Curr Opin Struct Biol;citation_author=D. N. Gopaul;citation_author=G. D. Van Duyne;citation_title=Structure and mechanism in site-specific recombination.;citation_pages=14-20;citation_volume=9;citation_year=1999;citation_issue=1;citation_pmid=10047575;citation_doi=10.1016/S0959-440X(99)80003-7" />
<meta name="citation_reference" content="citation_journal_title=Annual review of biochemistry;citation_journal_abbrev=Annu Rev Biochem;citation_author=N. D. F. Grindley;citation_author=K. L. Whiteson;citation_author=P. A. Rice;citation_title=Mechanisms of site-specific recombination.;citation_pages=567-605;citation_volume=75;citation_year=2006;citation_pmid=16756503;citation_doi=10.1146/annurev.biochem.73.011303.073908" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=F. Guo;citation_author=D. N. Gopaul;citation_author=G. D. Van Duyne;citation_title=Structure of Cre recombinase complexed with DNA in a site-specific recombination synapse.;citation_pages=40-46;citation_volume=389;citation_year=1997;citation_issue=6646;citation_pmid=9288963;citation_doi=10.1038/37925" />
<meta name="citation_reference" content="citation_journal_title=Genesis;citation_author=M. R. Hirsch;citation_author=F. dâAutreaux;citation_author=S. M. Dymecki;citation_author=J. F. Brunet;citation_author=C. Goridis;citation_title=A Phox2b:FLPo transgenic mouse line suitable for intersectional genetics.;citation_pages=506-514;citation_volume=51;citation_year=2013;citation_pmid=23592597;citation_doi=10.1002/dvg.22393" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=R. H. Hoess;citation_author=K. Abremski;citation_title=Interaction of the bacteriophage P1 recombinase Cre with the recombining site loxP.;citation_pages=1026-1029;citation_volume=81;citation_year=1984;citation_issue=4;citation_pmid=6230671;citation_doi=10.1073/pnas.81.4.1026" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=R. H. Hoess;citation_author=K. Abremski;citation_title=Mechanism of strand cleavage and exchange in the Cre-lox site-specific recombination system.;citation_pages=351-362;citation_volume=181;citation_year=1985;citation_issue=3;citation_pmid=3856690;citation_doi=10.1016/0022-2836(85)90224-4" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. H. Hoess;citation_author=A. Wierzbicki;citation_author=K. Abremski;citation_title=The role of the loxP spacer region in PI site-specific recombination;citation_pages=2287-2300;citation_volume=14;citation_year=1986;citation_issue=5;citation_pmid=3457367;citation_doi=10.1093/nar/14.5.2287" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=R. Hoess;citation_author=K. Abremski;citation_author=S. Irwin;citation_author=M. Kendall;citation_author=A. Mack;citation_title=DNA specificity of the Cre recombinase resides in the 25 kDa carboxyl domain of the protein.;citation_pages=873-882;citation_volume=216;citation_year=1990;citation_issue=4;citation_pmid=2266559;citation_doi=10.1016/S0022-2836(99)80007-2" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in neurobiology;citation_journal_abbrev=Curr Opin Neurobiol;citation_author=G. S. X. E. Jefferis;citation_author=J. Livet;citation_title=Sparse and combinatorial neuron labelling.;citation_pages=101-110;citation_volume=22;citation_year=2012;citation_issue=1;citation_pmid=22030345;citation_doi=10.1016/j.conb.2011.09.010" />
<meta name="citation_reference" content="citation_journal_title=Methods Mol. Biol.;citation_author=P. Jensen;citation_author=S. M. Dymecki;citation_title=Essentials of recombinase-based genetic fate mapping in mice.;citation_pages=437-454;citation_volume=1092;citation_year=2014;citation_pmid=24318835;citation_doi=10.1007/978-1-60327-292-6_26" />
<meta name="citation_reference" content="citation_journal_title=Biochemical and biophysical research communications;citation_journal_abbrev=Biochem Biophys Res Commun;citation_author=U. J. Jung;citation_author=S. Park;citation_author=G. Lee;citation_author=H. J. Shin;citation_author=M. H. Kwon;citation_title=Analysis of spacer regions derived from intramolecular recombination between heterologous loxP sites.;citation_pages=183-189;citation_volume=363;citation_year=2007;citation_issue=1;citation_pmid=17826735;citation_doi=10.1016/j.bbrc.2007.08.145" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=M. Karimova;citation_author=J. Abi-Ghanem;citation_author=N. Berger;citation_author=V. Surendranath;citation_author=M. T. Pisabarro;citation_title=Vika/vox, a novel efficient and specific Cre/loxP-like site-specific recombination system.;citation_volume=41;citation_year=2013;citation_pmid=23143104;citation_doi=10.1093/nar/gks1037" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=S. J. Langer;citation_author=A. P. Ghafoori;citation_author=M. Byrd;citation_author=L. Leinwand;citation_title=A genetic screen identifies novel non-compatible loxP sites;citation_pages=3067-3077;citation_volume=30;citation_year=2002;citation_issue=14;citation_pmid=12136089;citation_doi=10.1093/nar/gkf421" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_journal_abbrev=Gene;citation_author=G. Lee;citation_author=I. Saito;citation_title=Role of nucleotide sequences of loxP spacer region in Cre-mediated recombination.;citation_pages=55-65;citation_volume=216;citation_year=1998;citation_issue=1;citation_pmid=9714735;citation_doi=10.1016/S0378-1119(98)00325-4" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. Livet;citation_author=T. A. Weissman;citation_author=H. Kang;citation_author=R. W. Draft;citation_author=J. Lu;citation_title=Transgenic strategies for combinatorial expression of fluorescent proteins in the nervous system.;citation_pages=56-62;citation_volume=450;citation_year=2007;citation_issue=7166;citation_pmid=17972876;citation_doi=10.1038/nature06293" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=C. G. Lobe;citation_author=K. E. Koop;citation_author=W. Kreppner;citation_author=H. Lomeli;citation_author=M. Gertsenstein;citation_title=Z/AP, a double reporter for cre-mediated recombination.;citation_pages=281-292;citation_volume=208;citation_year=1999;citation_issue=2;citation_pmid=10191045;citation_doi=10.1006/dbio.1999.9209" />
<meta name="citation_reference" content="citation_journal_title=Neuron;citation_author=L. Madisen;citation_author=A. R. Garner;citation_author=D. Shimaoka;citation_author=A. S. Chuong;citation_author=N. C. Klapoetke;citation_title=Transgenic mice for intersectional targeting of neural sensors and effectors with high specificity and performance.;citation_pages=942-958;citation_volume=85;citation_year=2015;citation_pmid=25741722;citation_doi=10.1016/j.neuron.2015.02.022" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=P. I. Missirlis;citation_author=D. E. Smailus;citation_author=R. A. Holt;citation_title=A high-throughput screen identifying sequence and promiscuity characteristics of the loxP spacer region in Cre-mediated recombination.;citation_pages=73-73;citation_volume=7;citation_year=2006;citation_pmid=16595017;citation_doi=10.1186/1471-2164-7-73" />
<meta name="citation_reference" content="citation_journal_title=Methods in molecular biology (Clifton, N.J.);citation_journal_abbrev=Methods Mol Biol;citation_author=A. Nagy;citation_author=L. Mar;citation_author=G. Watts;citation_title=Creation and use of a cre recombinase transgenic database.;citation_pages=365-378;citation_volume=530;citation_year=2009;citation_pmid=19266338;citation_doi=10.1007/978-1-59745-471-1_19" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Nern;citation_author=B. D. Pfeiffer;citation_author=K. Svoboda;citation_author=G. M. Rubin;citation_title=Multiple new site-specific recombinases for use in manipulating animal genomes.;citation_volume=108;citation_year=2011;citation_pmid=21831835;citation_doi=10.1073/pnas.1111704108" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. T. Park;citation_author=S. D. Leach;citation_title=TAILOR: transgene activation and inactivation using lox and rox in zebrafish.;citation_pages=e85218;citation_volume=8;citation_year=2013;citation_pmid=24391998;citation_doi=10.1371/journal.pone.0085218" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=S. Sajgo;citation_author=M. G. Ghinia;citation_author=M. Shi;citation_author=P. Liu;citation_author=L. Dong;citation_title=Dre - Cre sequential recombination provides new tools for retinal ganglion cell labeling and manipulation in mice.;citation_pages=e91435;citation_volume=9;citation_year=2014;citation_pmid=24608965;citation_doi=10.1371/journal.pone.0091435" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=B. Sauer;citation_author=N. Henderson;citation_title=Site-specific DNA recombination in mammalian cells by the Cre recombinase of bacteriophage P1.;citation_pages=5166-5170;citation_volume=85;citation_year=1988;citation_issue=14;citation_pmid=2839833;citation_doi=10.1073/pnas.85.14.5166" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=B. Sauer;citation_author=J. McDermott;citation_title=DNA recombination with a heterospecific Cre homolog identified from comparison of the pac-c1 regions of P1-related phages;citation_pages=6086-6095;citation_volume=32;citation_year=2004;citation_issue=20;citation_pmid=15550568;citation_doi=10.1093/nar/gkh941" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=F. Schnutgen;citation_author=N. Doerflinger;citation_author=C. Calleja;citation_author=O. Wendling;citation_author=P. Chambon;citation_title=A directional strategy for monitoring Cre-mediated recombination at the cellular level in the mouse.;citation_pages=562-565;citation_volume=21;citation_year=2003;citation_issue=5;citation_pmid=12665802;citation_doi=10.1038/nbt811" />
<meta name="citation_reference" content="citation_journal_title=FEBS Letters;citation_journal_abbrev=FEBS Letters;citation_author=R. W. Siegel;citation_author=R. Jain;citation_author=A. Bradbury;citation_title=Using an in vivo phagemid system to identify non-compatible loxP sequences.;citation_pages=147-153;citation_volume=499;citation_year=2001;citation_issue=1-2;citation_pmid=11418130;citation_doi=10.1016/S0014-5793(01)02541-8" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=E. Suzuki;citation_author=M. Nakayama;citation_title=VCre/VloxP and SCre/SloxP: new site-specific recombination systems for genome engineering.;citation_volume=39;citation_year=2011;citation_pmid=21288882;citation_doi=10.1093/nar/gkq1280" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_author=S. Turan;citation_author=C. Zehe;citation_author=J. Kuehle;citation_author=J. Qiao;citation_author=J. Bode;citation_title=Recombinase-mediated cassette exchange (RMCE) - a rapidly-expanding toolbox for targeted genomic modifications.;citation_pages=1-27;citation_volume=515;citation_year=2013;citation_pmid=23201421;citation_doi=10.1016/j.gene.2012.11.016" />
<meta name="citation_reference" content="citation_journal_title=Applied Microbiology and Biotechnology;citation_journal_abbrev=Applied Microbiology and Biotechnology;citation_author=J. E. Vick;citation_author=E. T. Johnson;citation_author=S. Choudhary;citation_author=S. E. Bloch;citation_author=F. Lopez-Gallego;citation_title=Optimized compatible set of BioBrickTM vectors for metabolic pathway engineering.;citation_pages=1275-1286;citation_volume=92;citation_year=2011;citation_issue=6;citation_pmid=22033566;citation_doi=10.1007/s00253-011-3633-4" />
<meta name="citation_reference" content="citation_journal_title=Journal of Virology;citation_journal_abbrev=J. Virol.;citation_author=R. Zufferey;citation_author=J. E. Donello;citation_author=D. Trono;citation_author=T. J. Hope;citation_title=Woodchuck Hepatitis Virus Posttranscriptional Regulatory Element Enhances Expression of Transgenes Delivered by Retroviral Vectors;citation_pages=2886-2892;citation_volume=73;citation_year=1999;citation_issue=4;citation_pmid=10074136;citation_doi=10.1016/S0960-9822(98)70255-6" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Novel Heterotypic Rox Sites for Combinatorial Dre Recombination Strategies" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3.cover-source.jpg" />
<meta name="twitter:description" content="Site-specific recombinases (SSRs) such as Cre are widely used in gene targeting and genetic approaches for cell labeling and manipulation. They mediate DNA strand exchange between two DNA molecules at dedicated recognition sites. Precise understanding of the Cre recombination mechanism, including the role of individual base pairs in its loxP target site, guided the generation of mutant lox sites that specifically recombine with themselves but not with the wild type loxP. This has led to the development of a variety of combinatorial Cre-dependent genetic strategies, such as multicolor reporters, irreversible inversions, or recombination-mediated cassette exchange. Dre, a Cre-related phage integrase that recognizes roxP sites, does not cross-react with the Cre-loxP system, but has similar recombination efficiency. We have previously described intersectional genetic strategies combining Dre and Cre. We now report a mutagenesis screen aimed at identifying roxP base pairs critical for self-recognition. We describe several rox variant sites that are incompatible with roxP, but are able to efficiently recombine with themselves in either purified systems or bacterial and eukaryotic tissue culture systems. These newly identified rox sites are not recognized by Cre, thus enabling potential combinatorial strategies involving Cre, Dre, and target loci including multiple loxP and roxP variants." />
<meta name="og-title" property="og:title" content="Novel Heterotypic Rox Sites for Combinatorial Dre Recombination Strategies" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/3/559" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Site-specific recombinases (SSRs) such as Cre are widely used in gene targeting and genetic approaches for cell labeling and manipulation. They mediate DNA strand exchange between two DNA molecules at dedicated recognition sites. Precise understanding of the Cre recombination mechanism, including the role of individual base pairs in its loxP target site, guided the generation of mutant lox sites that specifically recombine with themselves but not with the wild type loxP. This has led to the development of a variety of combinatorial Cre-dependent genetic strategies, such as multicolor reporters, irreversible inversions, or recombination-mediated cassette exchange. Dre, a Cre-related phage integrase that recognizes roxP sites, does not cross-react with the Cre-loxP system, but has similar recombination efficiency. We have previously described intersectional genetic strategies combining Dre and Cre. We now report a mutagenesis screen aimed at identifying roxP base pairs critical for self-recognition. We describe several rox variant sites that are incompatible with roxP, but are able to efficiently recombine with themselves in either purified systems or bacterial and eukaryotic tissue culture systems. These newly identified rox sites are not recognized by Cre, thus enabling potential combinatorial strategies involving Cre, Dre, and target loci including multiple loxP and roxP variants." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3.cover-source.jpg" />
    <title>Novel Heterotypic Rox Sites for Combinatorial Dre Recombination Strategies | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"yzTHqLwmf06VnOC0_7icJXBdPbQlBD4UQkeO0iW5oiM","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460078","apath":"\/ggg\/6\/3\/559.atom","pisa":"ggg;6\/3\/559","ac":{"\/ggg\/6\/3\/559.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/3\/559.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/3\/559"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/3\/559"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/3\/559.full":true,"\/content\/6\/3\/559":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/P2Dca2LA5fX9OjYsAU8w+IDEq7m+LOifiikYfAy4waQiThqNH\\\\\\\/udQDt6MDHYtzqKP0qFVVPyZUXlG1EPO0ZQaE+OeFo+TI2nYlRq1bqM72WaKVXVTCzTQJ21U60PyL7+VBGyHo0gCUDSfygffB3+QWQdzVuLQtA\\\\\\\/PugSDV+z2xyvIdw8YV\\\\\\\/b+K7d6ZQRxBN876rniwjMfKftgG+11coEuaZHwNrUc7Lmes5HQppptNAFcNLfH1Nok+Qlx8FB1ReahQmhXJOT+BM54CZPwSnUNNGUlx69fnewT6YSujdvk2Zb3yr2v7v9bjwgQZLCB+6stG\\\\\\\/vPROBSjGvf6FHSI0OQx7RKcJ5xG0i9xCYsaYvCVozfwbuWnM2sya7wxQirPAGZiaZ73UKavu5b9OKCE1Qw+x8L0Gy7CpaNZG2e9GBTLPkXtcNKkp8r2mO51558i8NAI0vMvonN0yau7t8bI2F12iykZTDoaCVbr+DEpjBRCCJDCy7mkaqcxJGnuG6QtFXLzD1rVmxdjI5Ij7IPGLmhPA11X4NhP\\\\\\\/PSqJJ\\\\\\\/vD\\\\\\\/mLOYKAjMN3OJHxYYvaCowtuuNO0sZxkk4Y\\\\\\\/UCT8c2WuIemizesiPbGbeFtOj1n0somxVqdExoXXcn7QCcPVTcQhPK3vrllBgcuYGxYTudvQqmDVG5RNxdIqTbVIPNk8RyPVZsEXQZtTAEjF\\\\\\\/\\\\\\\/k0Rq63XR7f6nY6YvajpCqQNQv2NLJeoM+L4ge\\\\\\\/hnf6tQJnkWULkznKyKg4\\\\\\\/rBlKFDDrOotfqjZseRYEkaM8s2jQM0kcIblmICGYyzs\\\\\\\/sDz20kt3yCiO4alL\\\\\\\/NaSS91MSciJij3mqmkUEUSGG8iPHYWsIR17FyPDk7HsyASCYqnt7\\\\\\\/DMucAnBviZl9dEJZ1nlJDaZKfxrhYAnfH6PfXFtI2LJ3tw3jFQxYO5GLQu3\\\\\\\/7VH3xnNAe8GXWCwxsl773HDFuRu3TLHwkJr0Qiw3VfukhmDQTJrsKz3F5\\\\\\\/TE5aw46o\\\\\\\/AnRYMT6odiDp7dl85e8fu74oiw0upQLzTQIdjdVsRhyPtaz9OeUtRZurlaYfJU6AnRTHt11ZGOSjmHUaPuPVW8r04lB1ZBVw7hrmymrMQPwOa9BVOlUe35o4GVKxkkErKI0F9TNbCdvXc4IHl0N\\\\\\\/0K6R\\\\\\\/KLEhUa3fL\\\\\\\/Ldns0R7rFdp3vmonIFBXscHF1fIswABbIYnBShTbmuRXkz3p7Hlkq9E2p8nXugXXO6xaMecVRujUMzXxKWMxdVYYcC145F6KA4J1ELLHuYoqk4ZhMTVLE7C\\\u0022,\\\u0022iv\\\u0022:\\\u00229CqcfFGc131JmGObtkcjMg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002214f0b6bdeeaa12728b2fbaba82dbe7b5f979559d4b85fa706188980d5058d4b4\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022qkse\\\\\\\/dzm6MIf0q\\\\\\\/gWz2TsxqRzygSjDUissilZPsMHde0ak91HsZ+QSQi7Ar24rXRL\\\\\\\/lQ1aW7l5VIb1DmqOvWO1k02t6W9S+y1tkxFaDK5UqnTvuZO8Y0vEruuZcBd7TnEMfG5CCzp5SF9tGJ9KeIT3zOOvLWNkSIeRDF7Lw2A9i8A7ppqLDqJ9ra27wZQx01F6F+8+CeFQl8\\\\\\\/ZJtLiuS9T6Qxg\\\\\\\/qH+s+RD1WA7nrQpYgHwwK1hqqwzFB9z9DSrb7bP5gURNQ94H3ICeu07M+KY2Y93cQdyxJWDSjovZUPSJa7Bj+ZBwLr9kCHgn\\\\\\\/uH4xQkxBF\\\\\\\/\\\\\\\/g5nzPB9L81QZ+p5X5edLoFO3EJlqpO+f8Uu+nX8jzj42i6EDEuMLFBRRDZdr4LoVrChr6pZA8vmIQwsmAyPkC8gQa6PQIXdibcZrjbsU0s1p6ex13FKNpklhzTscFzBG2MvUBoB0zgcAlnl0O4eRIeld\\\\\\\/sorcyjLzDhVmJ6fHUFAWJY+thgLA9V8VqoWB8EHWSsCfESIAo8fJQccQgihSp9FfKtV8a3TWz5udvWv+D9xnyqIgzx0jW2VvXVms28Z+1Ry3bunKKHrFuzPs8J9soqlNtjtCAo7LqA7sGS8RQdadwq6WZFb0twv4MMnkQZ0OlnTXyKQqwbPaRaqBgj6l+vVzt0IpY0fYEgqoIwFZq50yuiKO2tlHJQBhjoia35E01rkDCbKN4xksmtXVWpXCQ9TXZH++hljmpoj0CeLTMSP+\\\\\\\/NmUnqsRz5QHAShqFcKinEVKAWrrv7DeEIfcqpVI2jJ29tBKr8DD99KDzWMwEES+z8grbDiB9y5woprKkj4aTH1H8J9NH4yVMZBdIesplPQ3gaTmoZlwAm3ZONg3sVOU2AjlCu\\\\\\\/S0tRgKXQgfp1yYOkpmbuBmhdYO2glEfKLCXwM8QMlj2gdHH1jOV5E8IftgIV5EmBn7Wo29lw4wjqKKaTzaUBebE99ml2hLiTZP7+wCF02UONKywXmOvtvdM0apP1vrixnPyGVdFEpaFcz4LFHn996TMB+FWym\\\\\\\/H8jaopfYs6GqS9vcOslZ6+fw1HagvbLGYdEAKSVZGDsAm+vT2yBocFYHqF+8Vc5iXaTU15DKyFXbcPCq\\\\\\\/udzvaCjOwFdhDo2tckOcl\\\\\\\/8bd3eqKZYtGMkbqY0RLddAL7UeYkjKuwIvSaxFUuAcnPCg1zwRX68Zs7txJ64AAP9TcwWHWbT5USCErIdglaH5FntYB7VKUIkg6J3Vu710HWQz288WItCKGsuDmaty6+5hUXOeRvJ7rv9dPE5OmI8uZgcN0c6Wxfd8GE44nh92jU5AP+rxZwy+u9JXQB3cTDdKdNHYNPLafo3+dxbDtLu2S6fFbjgYZCqwvScYbj8vK4u5uAJgpDBKJoG+0ot4CsvMiG1j1v0YugA0dZd288eF285gNQVX96aQygf7bv5hvAfzl\\\\\\\/IyrpFiAYVouv\\\\\\\/k9wxUO9kFXknSdHt2I9bWrsXtHYVhJ2y1KZcE69j6nE9JcCSj362H9H\\\\\\\/EBQWyVWs8OqGxjlxgN83Aoa2USUdqNwPfhccItarIFWpsq46+CwaDkoN49y9xqvrFmDZ4FuT7nJ2\\\\\\\/b6viUsGmPML4dd8dDJpT+MveUwwdhT7OZWgtY\\\\\\\/7tuhEqnPcXZ0WIGvjOCefBLw5kgUn3q0yy\\\\\\\/ReMm1mppH3pULdFMpY7HEBv12N1J6NLytxxsZTxMW\\\\\\\/lbiI07HLMnkEozsnocorPtqyQ6O8L5vsz0fZbm5CAie5+pIs9J2GTxiAHM8nko\\\\\\\/KBPjtyOy\\\\\\\/tUa4aQzqChrBywO+oliFe5HKL9EcfCpPYyvn7Gcf3jCFeoJie+++aX6m\\\\\\\/2c9Uabw19QBack3bm3smyEdEpkoVNQ0wTAlMgJB+REcOu8ApP3AofQA5xByx9z7fDWEiN0K0xqs9y4\\\\\\\/I1IdxGQAoFnUfcR6rKNESxWWTipj\\\\\\\/DLlOOYosr7opoLdq0rLBdm8a8PgY2BQF96jRYgeetlfBFBqunrdGtD7ZYGuq86cM7PB2PzAaqkG0KcD7zI8z1\\\\\\\/sgk17hGkXi2N8Ht4+ImcIM4AaSQGNVmFoSQYmeeiUmg\\\\\\\/9TEzLEuiDq5YYW4aBRgM2Le6PERkwiAQqTIvXdJN5A0gPfW24oRdX2vm9Xy1V1y9qBOFL2dPlAPOQniez8BEC8djyCJVZEHjpDbJ1oyDYdnQomWMK5tAKe3R++9NihxscMENzJpc8sQJ5XstaNlRfYRhK97sYvI6pi5WtBd0U1utsgepf7vDCueaCkmXBGoxlT5IUPphphct4kRE9UTrvC4qZE840s1wGCCnR6wGXJsKVUeamanai\\\\\\\/0rzvpFw+WOzfbx487UoaiGVaU3K8ag9L0Lx+hmGrCJjxEJYhP8sJKlxEsGTeG6APCSxmYgpqbV0ldC5o824hZ3ZX2D8v1bu\\\\\\\/9RcMxFpDCoS9sPssSnwnk3UK5m5u41S\\\\\\\/xmKCLAzjPOU6vmeLm656jroaIzcjbgHxpaQp1WKzkXlv0PxMRknuhy0eY2H4q+tnIng2tDSH24KmOwLAqZDn10w7NRdTbmhU4u+4bMWd2vrZvv3VFT9G0Wv4P56L7pYyhGARciyxAfFdzcV3dCIxA3phjeRiqrj10ulZphxnSESmsB8ri+FBrdiA0VDCo5VwtwWZLr+gS4yfBlBYThyZmokvFWd5tji04hjiRYe5S4Uz5z4UgrYbM74s+gyjD2t+YYzcrO3RGGRjHH8dkginyJSVufmdh7c4O2oSFuuHSX+2FkOyL7q17G5mF8ugXP5kaQpk25bJUZW7a0CVBSmAOYEEgIv2i5NG4OrReHYNqi31w4NE+FY9Yjj3D3W5h9kjgFgNJQe0upZqTNXc3FZLMrDCn6t9YrV0ExzgKGG82SnnakctM2D6D9+HxhCEf+uTA5eVXIPK3COBA61ZNTgYSl6njE3W1eAo6ycSVvUbWO7xcthuu2MydUmIwnukwhSdtcwKzSvQCGVxRDWZm0zLTfbCsnwmbdN7YNGLibFiNYIDGeh8jsNXt2QyUiEM+23x1M5508xMq+\\\\\\\/nY9xQRTTFNGmmfTUWPWcVmzEbPCAol3NSxPjNMcqDI8khSpiyhQCefhOPmNdsjtK+CTe2M2CFos6oPR03ZDaNkQJOm1R+my7u65Ae8J6D0zasHwHQfI\\\\\\\/QZz0GnNxb51PiqMVQM0jFF6ddwldW14AnBlT07XP3GXa0NBDiLlHsxckkLNrSEWsgGm8Nc9QW+gN8Svjq3C4Y9pMagAghiaUiOS4hx+a7PylvLXpycGtBBTkkKFnwg0E3lTwNPUtn8mjwt3NRUYTM2nCYAE4S3qOmuAVADT03zhQZl\\\\\\\/xKf92WyEXYaEvrrykblAfQCa9vBX4xfs+l2LC73Z264hx\\\\\\\/feF\\\\\\\/hRkEmX4BrRvYqAYlATEA6\\\\\\\/FkOpu4IV4hfYCKzHgyYX8gQcJFbTQWpEnQYNqNxV0lnL\\\\\\\/ZL5G89BPVj9qfPdvivEeIoygU3ww3FN+d1JtUZgPqKpZEbceJgMX0gZS1N\\\\\\\/IzY8bg0UH8bzX2bvMk5Vbb7iFcZhR7JdVNscH77yUJ4VyeXtKXSY27aga0440Kylj4WqKCEh1\\\\\\\/HVRUi9iM6V0eXpJHhX\\\\\\\/EKFEnOkOGBYsPD9DtIFyNBwXn0dorVQXhE2uixUiZpDmFPB6RhlZ50zntQ3hUf7gTemSkaYYwQmmnmrE5HXzRoxGh0Y5bykGGJlae05o5SWgL0VXE0wl2z0M0lDEsn5E4lNp3NVRpTYSvTz4rf1bOTjdDWPjH18sIFACE38L8gEaYpXCU2vr0VYPFnYi0VZm+2sZcoXFnnvZs3Vf8+CZaincCPQ6TDPhanjZbLOP8NlY2s6bMXCByHPL9qh9QDJqlLN3zneCH0VHmurB0yHPdfVcoze1F3OH0+drPausGtMHiCWlFImjjruqghim24dqgH3CZUyKRLY9idp5Js8Z2GMOCEjobr8rsiH+ghYtQMFLMrSKd3npsCrFeXNyrsTqGuzz7itLV8\\\\\\\/Ndz5SB8jJNrU9BD34Vyel1HW0tjHzKMM9TNZEUWZ\\\\\\\/9Lctv6Tmv5SKWZQiwT8AwxieYXocof6CzS31EJtqDyhCa\\\\\\\/4R9NAj\\\\\\\/+\\\\\\\/rFtVa8e8MTLxk1T2slWyah+re5w81n1cyNo856p\\\\\\\/Bs9CgzEWchxwW5\\\\\\\/y+qRgEdBobHE4joPpiIVCWhXxQC7HRflkEZ9rmZZdwbhMXKi95GVDQMLJgLrCfryixezz6zqlrBFPshQ\\\\\\\/ZbsPyikX6l8gOq0nE976PoFhk+7QzyJaG8QR5+OM45sxDPQN65GgEGRBFkrA7m1XzfIMtUIlh1eAHgpgic7XkA3spY\\\\\\\/\\\\\\\/Sq4Vq4QrIQiIbYdVMq15WLSVfItqgSNpVZjoTmd1SHn11QBKNHaoDD5lRYABajN93z+Ey+IhSgS90iqcEk8G4B0z7O845MVDR4sFn9MymnKSb\\\\\\\/cbHR7GaaPRHs96jeXmDWRUBc0W0ZQI3tQ1\\\\\\\/J+KMNwBcJDoWOx8C9g05KIKLb+7MyN9RjHWU23AO5+GUWSPeMstp3DwTd6ULZ+N9CL73LJ1465zmTLTXuphDAyOgExGUdMwKOP9Ir5aEiJzfHD147h5eHMiYmHyDJs28r8O3S0VlLYVcVVr6dO8pQje0gQCls0i2FWQ+nUMJdRmVPPf9J3mHk\\\\\\\/fiHGTcvL7i12+09qJ19rqy1prlLs6Ix6+wv8Weknp2X35m70gPvN5mslzsIY9VDJ\\\\\\\/QUaTBI2e2dCVYc6p2rZVFTbCx5xC7JrRPSzjtgsV6vd875v9\\\\\\\/6M3nT2uSn7pz4mJzKboYYWRTnBfnZLHBzQeqxGyt1hOQ7dug3sqlbmMS8BgrWpzbnKjJWzRflBF1Bn32DGadeQVS5UgJAIatEgZF6H1GoNNoK4qfMQXIfPzQB7V9nzwR+b0qoud3XsFmeLhLIPsnh5nHyce9omAXJa60Rx+H2sYC50+uhUAC4kClvupp4DPv8b0QxohBI0KCxS7AQfKvTvDi0PTbM3NO5Vtd+xrrCHt1QOAlJX7HJYc8J8Vm2rzg5z+YlfZYwCed9o8PT9bYvMQ9rbMPwpPJZ9EhHLBe29JLX7RgQiFRN0i7GGyzbC9tvC++8YYrLhUMT6vLv3SyeFe9zbL4FrRYXV6mlyZKBbz7pcB5cVQt+7A1XwBQ4Cro72tRd6h8eP+AA6jAbY8n9ItbZ\\\\\\\/ryC4aZCH4YrfCjUxXM3+yMZmy51kXh0klkhBO8aPcuPckAFpJowOR4Hxp\\\\\\\/91WP9Ue\\\\\\\/\\\\\\\/V+yUriEZ\\\\\\\/H6x9PpL7wPovzpNBXyqNg2vH3FJLANmoA2Yi7+Fhm5GMp2mVnVoGiu\\\\\\\/4plLLNGZhntEosf0fkipapwQYCToCSUU5qW4iuBh3LLe7SYUcoa4D8sJk1iErOvzB43Tt77SVOnrIV39awC1Xl\\\\\\\/sfW4icXUerYDqRKMIt6GNNsJMkgKvpoGZAuCgE9QtZF+kA2IjkHFQoNeTiEYssvTNZyzQl9vaDtLPGeA8WLUyJkeflNcsFNPYpSKT9T7h8RSbw+9EhhnShFkLUlX6OY2wV+cyZRVKwAMM617oO7QRlll5XBebOeO10oMUbS2MiDcF3Uq9V5Fqe8RkN\\\\\\\/D+zqgdwjTFM1mdzz4KxQR0Z8tGkSi34ujdYzqQd9b5Y6ntD84c5apkpoXhjD9C5mL+84D6euQenTGLLfu9iDHcKfE5TUKupuhcHURB0TIh\\\\\\\/JyHbh35HnCouhbELdui1qDhhpaUzkKBrSrrSwoMnaksvavXtlFFTeU8qStk16hJPRWKZ6hq1N0OjAD30qOUUoAKRsnOX1+u+uoTlV2PSx7byueB66bdhYQzNJmk9jop0g4hnnQPP0y4dt1l238gnQLvTJaY1ouZm3PVqPfR6HdvkJ3yTM3qdxin86Uv5HFEe1XHdQJJMpkHB3Bru0zRR\\\\\\\/Gwd1WCpGb7xEIsOAqUmByHXqq\\\\\\\/f5Y\\\\\\\/PqIIreW5w7dB78VkIx0vHD+OnXWfyxon7VMgAHTXJnRu4+BGlYg4JWvUU+H2j9vDciCzqxBhmOEq0Y7fUjuOUV32bD0MG3p9ugPWTio8tloU2DErV9L+\\\\\\\/JoPeHLXlzLRvM3po9OatucAVYz98pTRlIErmWsZGVkkBZ74g6w0NG96mzC3ystIhss6ZSkIGDRhvpz849ovpiQUDOIwg5mtd9KDxz5ZXt7p7aCKdEk7f1c4zIhd1sY8XZ7dLJrh8VYa67xVcL80FzMh0IpuMgVIF4t5+aUpPGvISDeABVZ5IZ4imsKyaflo90HDwh2MXzfo01To9iHuBeJWTHE1x\\\\\\\/2vW5YUMn+1QT2tuufA7D5S30\\\\\\\/qgPti9Qwq76Vtq9cqmFhiZt++DHNduG5zLk4UJIxnMMkoOvznGHlN0sKBY2crf14NZ3pPOyXTDj5lzNX7YO8i05i4Y4Gru1lry128eKmFYGFb4KQ9nwfaJ0pxeebbdjtwMxAFgjML7bnY9jMWIPTEfVKXPb38jrsNu9QXv4g26\\\\\\\/U3xYzJqNcOyCQ+7taxOpyG39HSDzbjN\\\\\\\/+FLgPRDKv8083LnW\\\\\\\/OHm4Jx7WlY7kXE\\\\\\\/ld7k7+0MXlwMWWoiEyeUl+mTQ0lWkZ0hwKygNijcCAYY5klyctomaaVjJpvJp9am9nDin5pTZ01AVKmpeK15aj2Dl81JHxEys4YQZ6L7i0J4CagVA\\\\\\\/sDaEAN2WK8EBGQRmrKpS3NcgFf0JN\\\\\\\/5CYFTCu9sA5GPpTUZmNRklVI+alXZOgwzso6nKsJZhg2Z9v8xBW44Qosvdl3I\\\\\\\/kdx+5YADnVm2SkulWbvS3DpKiucMU1bE9gJzYCGKtLJr1o9TRyxsX6HLVoCLO2tARv0qL2D4B\\\\\\\/0x\\\\\\\/HB5QJmR+1Ct8ICkf2IleRwZWa9PoOZhKBgDkH2D7TFI4Np0xsz1PhCseGOVloQwgdrscfLeCzC3PfrR18DuWEk22+614jlKbyNwQ79EsxevL2EZtMQw4YrBZjuWASV4lM+sjORuFp9OxnRFnfgtzpncXsu\\\\\\\/qZ81JTcfUuvolGha+KpXcDP9wD1q3GuA0kJRnPa+sJQkFgSSsnuPnv5WRUX\\\\\\\/GsE7\\\\\\\/tYKmHquSrqJopIz\\\\\\\/oIj8WYA2oeM7yvRMEUdP4gN\\\\\\\/XVylsRYwOhW3Pfza\\\\\\\/TFMn7FlGvIRMNKqKKdnGAjIhwXEe55OQgH+tLWzJwL6rkkiWJvSwxl5t0ZmeO4Qq+F86J+KRo8wS0wr8PRG8\\\\\\\/m9M+5fnyDDPl509wGd4yAByTdxl2qvd75sPxnE+O23CSYB2WDGf6kjjKfzVOk7Egx9lYA8+axH9JTk8C+USIUh1g4BKBL16E5nf7TuKr5HORosk7UcStiCEL\\\\\\\/LWdaIk7o6gGOmpXWqMvkRSaupfglCCuBFndUgtZDINc37DVJyZFQYTmiHRbpr+2T7pHnphaJDodC7qg2FC1Psib3NYGQWd1h0TBq8nTF\\\\\\\/IRfYFiS4jPB3D9Kaje8TnSv2b04WgrxVZfZfD3FU8sQ1ylFOkOacww+m6df7\\\\\\\/MeidmvtP3w4BJ7csgNUH\\\\\\\/i5vyJkuW2m83hos+G8gD6K2Vj1kj+2kFXMUKDHsF5wlpLfGVYbRubLIApJ+IjsLJCUj7FPLfiSRhVHAZdVtJ+kWIyBcYg3c1N2FyPGjEnjT47f5fQ83QHNMe+haN9FQYvDoV+kUzuf\\\\\\\/AdvXB75zgGQRx2qRnYasz7jtUez\\\\\\\/XuSEgkipFQQl2WaLszSGTzSw\\\\\\\/ONPCgB7IxCQvaVdKqflMMMlFYLUUgkEmSgjIW1ZB0HEQG360Vi0ZxtraSozVKi4BLqpLJspXEO32lgmCnzHJ12xY09Rdq43o9HDv4\\\\\\\/6IFb4Lmjg9f7QyA5sY3TUHyu8qCEUPBMTuFD+vt2krtNKESyV1TDAhU\\\\\\\/+QXGGZlz2\\\\\\\/ZaoRr1XMHIQWIQ34k5yLKleeNUK2f+5r8O6vpKTwKmZVky8TyBJ5+LNIsVjj3JscCB4n4rL\\\\\\\/r02xoutyroTdzNyEfJrVeNsH80pH61LYbXJUcxb7xPpG9MJKSQOkB7RDD\\\\\\\/9cIjzJH1x5fJew2rOp4qLLtQHnGDdL80p6oZ8m\\\\\\\/hK5cD1THxRe33hXfFtioHVk\\\\\\\/TPPI5UjVKQT5CC1yVw7WN9p7Hy\\\\\\\/Mn+oL3p+BSlNSQqSB1uhYF9c55arCymqEXfG3GYredAhh3n29X1LgGKb47q3HLSL2Ewt43jYOtadQi969hhHYtwrXdzD56doY2rHaK3omvgnRjiEh1oZATmsZTTmOxkjC2NPMsGAqOhcrpUQ4My5ghY99K8DvL7seIgxhoKPzEasv08r3+0c04N\\\\\\\/e9ZXUZHB3k8IMnDpSQV23n11yW4QLHUJHbE\\\\\\\/B+k5jzdGkjI7hLAvgHtvQPe6tC7UjsJWBN+JHAI1slz+C2TX0sxqQQhEPeV0bxglqz4DkgAZIxYcwbVsNOkHUqn7omdfx5FlmVFKROwkRQJE1mgdWvRMyLCwiZ4DwF4P02IkjiMxqrnB2TC+8NPMvpJSGpJlLZ\\\\\\\/8VixFhk2QXTlh+oBcVwjfX20o9UG7Os5aQZMHxqxWDyIutcwmV76k1M8QO1s3U7Fb6uZrxhXMjwxjL5G7gjEKCG8Ejic5wOk5sd\\\\\\\/og9Tdg91G0\\\\\\\/8xi7FyXIRvctFxVMiOHIVdkJVQjWmfdk8g1BYDCwoMMtHiBlJrdpIzYE9oUi8N0AaFFf8Gi5ET50aoSQh3VO2O5yu3EYG3Js+pATK0UTf4j7daWecfnGCO+Ye8zT+WxgvKmuLGNhCHuXsmIZPnlw+nmzw2D\\\\\\\/e8MGkHU5\\\\\\\/kR9zy1fqtiEDLpFocJyVG\\\\\\\/rv0g+fRaNwaTwWuMjad+MEGWxx5+X23dtXidQ2QjnS6PJ7z90F1B5jTJZ4FO4u7rlXTN6rVMWwO\\\\\\\/3UslmSnQGc2y6Hx0y+s+EQq90w2BUIGeZLrt7FGcn0rrx\\\\\\\/n+LhcQ4iWWoxYmDm4RXB48xYVipY2slBQP26yWeFGqB4DuSWc\\\\\\\/yJuqp2sAGCyv+drXDXv43b2L9gb3Pr7vWxXu+WM5jhaDGfn1tnIgSdSJcrTjYLAXcX1SXD8l1V\\\\\\\/w4Hla+rIiPX+Pm3OhFSqR+LYDQ1NDzqfGOTdOsEBH0isRXkfTw9L0kbiwdHjZ97PtuQbD8aVAy+MOwUfjRPtkKxqxwju77X4iIh9TWcIe+mZ3zNd\\\\\\\/ehXwke4vfIsfJn2YS13TbAE2qnel0\\\\\\\/6HhnU4mMLFOXEKVEeCF2tiNKFg+H3inzNlmRKNFs30tTnM6xiXH0CJV8vSnolaK2sCg4ixD\\\\\\\/F1oFZpqECVEDKUs0PYfaacs385fsRSTHLgpBimJBOL+EPPRW2zPT8aUhFKxIPdXN97o4P7bfujK6UrWznV57GkF76O8ays1DvnEmKUuWmr5YYd8UR1gOjEMvtZeDKu35tOw6XI50VPrSlabdgXgGi4BCIb2dIrdUgvlO4+6HBLz23J3bfSf4oqO7U8TeJDZ890mEcXhZxhTu\\\\\\\/HW5B8cmrxhnfQHsLQ+vZshbS15gEQ2B64d7fKSpMuRuMQqMZW\\\\\\\/pQ10j\\\\\\\/vkPmJc5uWnatRDt5DTPYg3u7guy70mF1iGrerx7v4IYKUfM0AMIVl0l5vM\\\\\\\/p7+yG3nhoXF6mgAhygEWuOhgebxfnt9ZVhzFogbCUDpG6hjW2I1flrm2CxNnjKmThGgozV1nI+tlsd4LRovo6y22PGOhmhge2MjsNDtAQbpCHHaujbvSF9SgDWiAjYxN\\\\\\\/CEN8oQ4L1L0l9I2XaaxtsdxzkIZYuwT8UYSLhPSlZWPE0EKex0mVMbi2gdcK0iHPx70mE4CDHcKrPKhO2Z5jjgyI9nrKsQhXhQOTleomzEhiWBQyUydPCJ97jITC6rmEGtRxtshB9PjvTZWht3QpITPO4oAoDlhweNc2frtbCzgoHB6+EGZf0bmKcusViIMKqJMaYpxB\\\\\\\/6QQMSkXbjajlwKRS7KeY5IfRX3kkfRVfg5u2c\\\\\\\/JNR6rsUkYKAN1l4JDgQyZNbYCqT9NRrLZDiMNE5V18lZ\\\\\\\/sQYBwpfW2KsD6b5M09wtUM30fIDgho0boXIDiGa7d76mrRYY5LxWzNLFWpireWcIYIScDNZMs1aUGOpyRGOrZ8YKlxIn1S9onCvMgytkvI00GtOMSpZy3FUCTkWH\\\\\\\/Y\\\\\\\/y1PTV4h9TzGgPrDJWNixtcYct4ddLfQlz5JExZW2DB54aUTDu2yj07a71TOutd9gWFZgnCqOrc8ygBjReNQNilxjPGCEX5p2p+2HcFxE5p92wZSLtSe3jTRgZe5jnpT47URJAUuChCxsou0g5rXPKtWf22OrVaFsaXQo0le4WeggcaIMnmPMbruG+G1b5So6pUFdug2jVNnO8MAefq5\\\\\\\/Q1c7vTD1jBlJNj9u8f\\\\\\\/yf8HcfzLCUF0mqZ6TagF1Z9pLYQilWyVmYcu2MpuFoGbU0o8gH\\\\\\\/sJH4VbNPGTX\\\\\\\/QN1PqKzil5nBg5VSCgTlTxVzUxAWVu8Qb0e8thw4ocmCNDRPzzLRMS9eT8YxxivQ+T1uD\\\\\\\/fZM1G\\\\\\\/6wr6adTniEE006N517aCZzRhkMQ05BAONXBaII\\\\\\\/5z7SGKBCVu3lSH494hbnL2p1xnp8pVj6h6qOyArf40u6CvLOY+2XAASExmH3IUWIszSFd85keENhjLyB+RDF7Ghp3WfsBBzeN3hBoDaz76AETCulb+QlQ5XjSPZHzYRBoBObKwjvdoNzuHssq+sdE6XL\\\\\\\/v0Mktw\\\\\\\/roc0AnCYet2riOxwkqSznpmb9GemFCHl1nhfRmL\\\\\\\/ZpLPPnwmcP5YyG7G0sz500WniXmPl8e8vQRdSbt52QhVQZjaIFA7szjnd4Bdvq\\\\\\\/Js7JUfJqcvHWnb1PEgYA2jZsJcN+g6XtJ09zhfqb+jHfAdIQ7ig7WnlAzVmVwkmNnsxzd+isCzBrQCKjIZ3IeIHZb3ht7\\\\\\\/ExEOWQhLTBVyKhpLSrOOAaM+qK25Stu08HnPFk5YHqcXIqIWrWvA07tpw4XhKrnBLdPh6NQU8w3JvqaR7DNOv37lGqCaC5jDPKORhLygpSCPQDhbNLJwpCNcg1YNbWNtpzeh0UDaGIdv3d8nIOqkFp3l6v4OwJgV+7eSUyzybCZbD+aR8X6W67gqCil7vGbe14GyuJMXx5l6LLlpuhgxWMWBtMSEi092wtohONRdG1ppOJiRBXFQlcWBXPK5euNeJI9zQF91tkDfLe6inedKxxpmPjioGjIOZnze4kUo7MJQIfO640Q6rAQaBlIJlSwow7xnuaVQDRwWyDkwTVsxcahK05uMo7gURmLn1QGAvaLhSTvoKDBFNxytDwujfGEoQaXToDa42sz3Q74I\\\\\\\/XiXYH6p6GujhPEgMOpsZRYe4+AAPOYMiQzEAITW\\\\\\\/xx\\\\\\\/DAIPtPxUN3UWvfIJdIO+jcAKc5BbGeG5yz96AQzBcbtGQR6JxEf21BQUvrZNIwGHHW1WKnWZh5w7bAYpq1C6qxqzZnw5x+20vpAx84g\\\\\\\/GLSP72prnFxbZxarnWgTbkpp30Fh5ADGpz7Muv\\\\\\\/GBUW9dGnNIhXrTkPY4I8klLld7xjH3Jm3zgwpvQJfLleZO0WRkXu4wuWzRUZkAgk2fUoh9gw3xSKH2V\\\\\\\/f0HjROgZGXCUD6\\\\\\\/yz3I02iCuljwNolFLXnKRtkSWUFpehNCBM9RfG9kWokzELEyuSqtc2yjGmKzFXXaEidmy83eqzDD596L5kdyllkm+FIQzztL8z6g1XLYVZvDBa\\\\\\\/+i2vv4vDFZ14BmjR9gIJIrOaLIKyRYjbf7\\\\\\\/b1CxBRAwUADQFMYk0ROBywQCJk2CHci3\\\\\\\/+CcZjfNFkVw0yzmDiV5iP7\\\\\\\/kWCZaCgL+aCa1WiEtRLfEdFklIe0QfgeEhuFhBry9Cz8+UcGuxT7Grs96NiVHYta2DnnelTILH2iziCmRJXrYbspzBtFzEg0IRqdKHus2jIwI7l\\\\\\\/wJ\\\\\\\/KfJMhPmbdoghaQEuRRiVeaydRI5xZpYWMYbosKvu9gPsc2tI24DpHLTAC39TvE+j7FfuvxiOqjQFgAN6tJ9CFpg9rMQn9kP4Wvbu7xnmxjzJYEsI8ZHSgsRtJeG4IuQVDOVaAb4XZ5Y8Xx8iS0cAENap4BLpJqlCl6BL+7QOmwfVy3REUoI0ZfCi7BIpjrW4uuqXc6+2eNb9UHkd2iXP8toWdLB4NF2M18YkQlOwl+GeZAJsojAdsU49zjhni+wZEBfi1JriL9gAPUZzKW59Rt07uqo9xuYcZrQ7MGeIZmtROZsxtmF68X3Wic3o0cI8HBJPlzWcjQIP4sq+BAf2Hbmh\\\\\\\/121aqCXvRfQ1RcYxcXZshf0RYI4YPgQP4EG3W5nglCFAe+OOvCOzcQADzkmUsZu4S9kVTT4Lv7B4AfkbKLzxhBAfOK8Q6w5soiO2A3rQPFOH4Lzz0qYYKRoBJM\\\\\\\/i2HBy3CFhRZgD147mec6t73lCdFdgOzhtEgBfFLFEcrSG3pewkf4xIU3GYMXT8POUCSUKbD24wQcJWcey9ZyW3kkLLY47Ca+1UWLq6AIsoeC4zvvi14psFE6A7cEWeu3Gsd0fTwhBVJNlKGXGDpYzp7ErGugK4eZNwDx5YMp3xeDNapyKzLuMLLBEvkuUp0DC4NhwP1MzO1cjWJ+v5IFqWQgZGtMpGyfS1WduRYwUQQUwu2VrDiDWTHy9lPh2CxJ3pwLZPhDlAswVwztep1vBsQkRVcV1GAUu7Y2m6vWCDQpmWKLeEeMUs7oI5h2ZMncXEUW1609uJdIn\\\\\\\/\\\\\\\/2donPqTmYDQtpEabXZv8qgw2dm3X7\\\\\\\/RDPEROQUT4fZcLSiwL3hbeXDbsZoPDQ6TrnkcuftCutiASUUl5N0yZs5iAXM\\\\\\\/N0nk+b6HVrCiiEBvQEDTBAWiLACaQZ2rOgNgkdJ480QIjiER31cjaASlY7SKkoUxRsIHuWQidzmzIZ1MGEPYL4Zh56OVWiZTMYflQdsQNFUsIb+M0RfjsbkMHaH7bhUm9hUGhBMmWvgrTakDe5JXbBD0gQ9J6dEVWSG5kIsClvqHt+F5zH9lbFidcgd+elffkJLPTgYCFpD4pCHN5qUlNvUNDpcDJmnQ8vw4SXWPiSqPoDbES9Bfi94HfN0Ac81pI0B\\\\\\\/wbCHOCkXzfSFaq0zyCbzsqG5paMT7rYM5ewOHfGnLDQTZA0RiNyvaGosc215go3c3qNSLgn5dPx4Gz5ugoNxAlwPGVawnG51FEIT0rnrIhE72cYdlem+Zui51a\\\\\\\/g87SeIxqoE\\\\\\\/46AWlOIkuB6boQbYzS5ZYIIMPQn675nvFUqgOpz2QCx8OHKOdPBpFQsdznQXGNWILng1d3+ZeA89wpdvUZpZJGQPbGEDe5g6tUhSHb3WtFyqyqoqda85Ti744cZWDg6RZNvcsBeTPoN076t\\\\\\\/RtnKOTsVLJ+h+Mkrbz0eF7EyrUn6SxuybhAjMacLPlgxHbladUvQ6dUkEtRfKAyw5UXwW97e\\\\\\\/R586EaOebYBHKb+PYoUACo5FpR+lUS24h1tddLh7lLVKWFaILdx2DEw8UozlD\\\\\\\/V6WwwRDyiO8VVZ1+mGz1gLC0Nzivq4BDGIJ7R1Y4nUfT5veOzXP8+\\\\\\\/NInYPyJDrTFFD+ouPGBQLbpNzZ\\\\\\\/VKsGTTht02DZ8NDIRYDqKbuzFEhm\\\\\\\/RrZ66ZE9PNjpuGgEybJVIi1nqLYsItTvH8SWz9Mj3pXP+3Gbi6ehar6CG+FhF++vYC3xW5mwV+FOy7OlSUxe+vUi8oXJbrUjg3qaDZNzsBY51eNbuKv0i55CdeeZqhTg1WIPOt4hDik5pGUOFaZ6d0hhMGNf0wb7T8FkaWa5nYEhTej570lOivPAiEGByz1V9ojch8L7vG+kVZIiZszCUYFQ\\\\\\\/wFCcw1RO+BeJFRGfqDKMt7D7+gIjB3aTGhbIh6waGmmC3p99myfFnQWdq9XYswdpvm6nBT9UOfkGWnAHICkO7A9RP067XDR5WVr3XcJWtFgEekYUV6\\\\\\\/O+Bje1ab+JmHYow\\\\\\\/KKochUxYsTz0BiHbdyfYXwLQIsq5T9Ln0awMb45Bto\\\\\\\/OW1mxSvchXM9vXo3AcOKAeTjBh0KwZdzHecbPtlrUkvv0qMRxh+JsAKVnIhp6gD6QMZkAnVnFvPMOQTbleY72Tkiv8ROisBh\\\\\\\/lpeELjN3W1bLK3006KPLq4fWUtfTUBOyLt3aTJFMeUzMQKL839gAZwo9nQVuOGdKhKiv4f8cHD12q4iugIh6mYgefBcny3w3HxiMPQW7kH8jyC2O4Tc9h\\\\\\\/80H4wsyZm7RWgGlpQB5QV7J5HWWizd9278zI1ywlifl2yS43KYjVsZp1J71OfHrphGeeutmKztccDX1ec4fsqUVot\\\\\\\/tWVh5GOTJkAs5Wu5yzEKRwCg6LGzHbDP4Au2N896w4ptjouo5vGu9TjpsCutUvp9ZD+ykYgDC4DWrT5LC\\\\\\\/Sf72\\\\\\\/bTheYPhpKzzFwgqE0xgJm7ZxJFcYOZ8XQGcUk28hPBvVv9Z6VNzQF0q1KEoE9VDmZRaB41HQWtm1vfPcUJYpEu2uPtL8AWyaH9U0XqxnyA2b0JRn6DUZbK2SNtW6zihm8r8KLf6Yzu5E3ggP+yOgRITWc5QfWfBR3JMHGHglcF\\\\\\\/\\\\\\\/jonPYIv4SV4yo56XSk1i12BDshyOiRX08w7TSl7p5njUobOFzDr1PgVU0EbTqyxykkp0gMxveKWj7IbFlHW4q3LS7Du8PtkYMhzS3ecFmkOskk61GPZDTkVw9Yr7NM6XlIJSnA3qS13QGvZq51KF\\\\\\\/U6FXgpxVF2GxWyAdkYu9cowOtgLpw\\\\\\\/fvE11LI8WtOALXNcD6G0z0NZL+hsh3dBDxW8viCu1EaTyeeVIWzr16yg\\\\\\\/zKrAbmytBaKufHp2tu0qhDul1epKsChLAbW4UVfb8\\\\\\\/JvfOoEDKjqHkVeLkp54SgyxuvdGB2e3gJ3euohou8Cj41ZaExsIsdZ8qKghwGn3d+hHG9ipWyd2CjaE6ysM6dDdflWMqf+lMQW7eqi3cQbuExXn22ljpXTjRdtPWmc+Sqhpnj9PF3noDgRRCOq3lMtU2uL5I5nXlK1WBeWE10VCgMU9u\\\\\\\/yL11+waXO4A1hekbEzGL6pCZ7x836u1gXqQWuK1JCedrRh3JCAUVdGuQksOH9JRMcrQBDu8ES7uo9yQ570UlIJIaANJSOFL+cD15HoAIgtp3O\\\\\\\/TnGLERAKDagKeDG9oIWJAzMnQTuXMUN7X3+m4uXexQMbQjgevXLJxS8S8UBWlaP2nJhL+vMx3FF7\\\\\\\/FQ9pyVhmYBJcoNs7HFWR31xEXaxgpmDDtxndFxikBCKUfCbInPjVuALameSZFlOaGeQvqoc7e0JJM82i+DJpiIVSR7U9XDx\\\\\\\/bXvirDV35gUA7\\\\\\\/tvWCI6LWC5uQAVkJgKejmB0WfOwArKRAyJqvGEjC9yZrtXMmDWq\\\\\\\/mlWhcVMr3Q8QvasyZinexM2bB5xJEnr6E4VKF2C6T5Ox4GzY7jeZUa0CvSLfMr2hvQVq\\\\\\\/aGAwu5bKn4xXKXmvnu1Ygu9zaATIpSeIK2JvEmSHCXgeAUl05Jd5SYrSjJBhNFnC7jFmFKrQqeK4l7hxIxju28npvdPTLj0I0dYIh9IhTDHNlUKvGTEHJWWd1CSsd1gYud+016GuJs0HtjLqv3yb0GWUxTRJPrGP6u7cdFYuIevdUhYlOE9wLStPmQLlPd2K1pe7nOcs6NM1x53NUinmKxI\\\\\\\/nW6pUl0m8h9UfUj43NB7gn7U+HiWuPFwGmNa8susoYazpQ+iiLpmCmOw5En3CnTApD\\\\\\\/hfYMOPU9LdClp547\\\\\\\/82Cqgo\\\\\\\/skmVAFg2bIlnAKxwLsX\\\\\\\/MDJLm9foP2a4tkDCUP++U71wd28iS+7ZBJSyNjVjYvaP1I1dQOWPxOoiF2nrlVW\\\\\\\/hZ\\\\\\\/ymM7ZsIJoRcvOFKGkAeji43sVB19SzO2SJRdCwSDpuNOigkjmpIdUz99NYtBXi2Mm8vt9hyuZ4tbCRBiFJZcZ9AF8A7fStbtmtU8ZQFIOsmrPQe4GviIhl8vswodWK6Vuc\\\\\\\/QfF1rp8r1iCdNI9T0Ou0aOvhi9TRAmBU+3+98oPvEgyBC21JgJ+RAk8HO2W+iNWFim3Vw0yxIIAg57uAebCGIZBw9gotkvtyUm\\\\\\\/7ljjWH9uld+d0udx+zw\\\\\\\/zxpd1smIfDyTRMC2hRULef9iNyAP3td3\\\\\\\/D9gwNUaUL8JSseCJvs1QUMgk7DIiG6LMScVNr\\\\\\\/a766vHrxcn1G2Tv+48ESOeH9ye2YCTB4EhCp5EvVcNDqHiBzD8lPMQaU\\\\\\\/4\\\\\\\/T0U4OvWeybqUTue\\\\\\\/Kt+F9nJl7Sl+DBzYlpwMfLbYLMdlmizhcslso87faoO+ADtdAFjbyU5fKLxUAkbCi7yqRVZgO6fThjeocQCUp5s7IjN+QrRsENMJ1ot8g4LpjiCNSv1t2Bpe6nWE7oCHzao2Mfu66MJjqORfjv4Edl+YuKkOZM5uvt7hmNQ8qFARaaHCr\\\\\\\/1r\\\\\\\/rMgEB4RLoeuy5MVb049NVZk4mG+lqMtwbM5NV8vMvA7b9iB8ejrqkP3v6nHpHCfoJqBIDOu5LZ2QcxeN4fTRcidRvk4MJlRIYAsXakyDeCRZEaXQlAX18gp18nFezPqZPhJeOe4omZNkU976b7udSvNOLedh1gV4dZc0w5Bf8UuLRBi3PN8bM24elTssffAlXyqD5IKBR0GGQoW+OevuePYBPnktvWBL4NV9ZNuEmpor2Xb8Fc97uGWlGXj1sApnBQD2ewldhd+A7axp7iSV9jiLTPRDrw+rT3qJbMxeCo9X7ZBjzL0UjwqWSZs2XOhXOMdYOqBWyaKfVYX3HJYH53PX83BkYmasC6fkcnU6imdrRwI\\\\\\\/xgzySrxSFPGyD7DAeFA7AYSJDi1AwKzbshhmtDQ+WIwiUX97eKZ8xeWSpVyyrFDfHrs8rLTTnNIqBKEQy8aUUB\\\\\\\/aROQ\\\\\\\/oJtv1YoPXhW6wSG4y43dSrZ4DleBZmt7C0XdENfi+5o2M\\\\\\\/cKXw757IxxNZ21gvbszhGtWpwCwnMPg1v5MQICmesjXP5PqILeO35dtYBbGDEcNTwflPEJIqGJ+OG4u\\\\\\\/zXmuCicAflJSOuKdZ1LRnyEIadmTTXe8x6eQuIvVUc6vx2iwLHSQSQF0cD0B3X9FFmCyNfbJYcFmAu7DEjZIcHUszTrx+sDV7mwMmr8GvSGO4jWuo9LWI1w6Ak1Hi7mTekEgNcStstf79uPCW4YexpL4+cqlR9ZzV2J+gCUAEEdjuMZW4sSiNydSYTxH2ZdZGP5NIWH3cj8e6\\\\\\\/aZX5flKEMef1o7aOQtJJnAMOK98uj2gdhTGJEl2XkMRQUHfCD\\\\\\\/xcBYUaAvSjw0zUyGM4EXugseCCfKTV\\\\\\\/maJhPDDHOqvAkr\\\\\\\/g1K9X3DjyD87BX2zMfUK2K8N2lM3c1zdiof4BWq42B+xINELR94RWEKARFKc6KJJRQWSqIRnXkvYTJNoRQKY9HX1deoO12VErjBaeIfmtLiJChxEcE3BZWMoVp56q3f9Y22ho23bt\\\\\\\/m4tew58qVDJVe2+Lp91rcyFXU431ZCm7G0EwkJRkiG4Z1b9jeVNR3RJQIdnbP9nra9iY2lZnKvvBLoIFkgg7t+utdE1cAZJj8Or0F+hZgGqoHZqt5xvhiv6sCQ+ccUN8arQswsnM54tTxD\\\\\\\/Gw\\\\\\\/4Sedlakq\\\\\\\/bnSEEyVr+Hlu4M5ioWLxgfZEfX\\\\\\\/ke3x8YGX76u+oT\\\\\\\/T0jbYipzfrGsKBaWMMlnNQLjrqx8DhM\\\\\\\/zrR1Jr86OTVqsHEJIT1wAP+Hr1P0BY4\\\\\\\/UUIkOQ3gMlEECA2IBWuFFXXlLPSOSCc4Q2wEgXsX+W5HmZRu+7WAJ8HkHGCnBJtrU2WlrPekFeo7DrElCCFO0mnoNhGo\\\\\\\/nBKclD2VuWj09P3a26HWlbFABkpx2ZNiC+YhNeOroAX6bTiS\\\\\\\/S\\\\\\\/pvEGJExQa82I\\\\\\\/ZcP1xxQ1XfCA8qbHHNoPaun+lcajMqgXvuB8aGRUTY7yVY0K1Iy3IBkVq5wtwUIW9qyK1xSnNraRxFuT7pzRbTKzzGQIdAlofbX+SneQo\\\\\\\/IjD42jUih\\\\\\\/Hu+qFUI6wTVO0ST3KPVyejXcGRvEXy5ndRXcNICJO+6RX3\\\\\\\/40QZ+UMV8cp53hnm7o4U5sao7E3LgzZEB1njWIn4pyCzUGmH8wclIrlwg2H+BuYIb\\\\\\\/52Z9YhOSFhdQWfu65XG3Z6iVVlHIwP92FX6kHug9Y5qJsdlII7bytjLJuAKGUiAuu8Pf+vM8NQAVZG2inFQ1SmJsYuq6Q5S9cd3igKzdMKbe06533RzdddsBSx2DACHE+QoJkHEL1BHT4f6gsPVitt+n7\\\\\\\/XZO\\\\\\\/Me2J0eqFRs\\\\\\\/4iwgaR\\\\\\\/7gqbjrukk0NU0V4WcpJ51YGXhc\\\\\\\/Gc4YJzI33RVbcPWPUvBtzGFqfLumz634AKKqc6Vbdno1iw2dWRLohpp58iexhwyQ8sTnOn1nLxH8A1SZDGS9g9S4nWdbGFVFiCtKZrtKgpgODbilY0qEOptXvlcS3mSs+QXWzS2quWOcAoKPaxlgHf2yVxZAvP4y085ppwZ9e0R3Ad+L911prZctKR6GSjk9nJJO4EnCkUPWIBYx+Yukn101m3Hz4+B7GGoih6UCzU2kds3lxJh6+xKL7DlfWuEbfF8GW\\\\\\\/AgBLHVsVwyV9VkaA9TJUYOuwnaSXzXvZzgjuHp4D+S9Cv+R+\\\\\\\/RRDITN0PiNDZSMytArWwTMQNg2d\\\\\\\/Ldr7ny3CyOGk5nFaAJrE\\\\\\\/mfjCmahazJHq5zaKbhpASTkf8ZTH9HbPbJtwP4Uekc3tY1ZtpFCHfbDD7SxSNPkHptBsHdorIAgxpNReEAqbfMwelxPaxR0EMKB4WLVQwXFwsoG5df42ZydkKoHjVjRpdBizihmJpsxpgcf1w0XQZ8QnAWh\\\\\\\/w6Hn4lZTFKfyx8j\\\\\\\/MdgF4qgYk\\\\\\\/oZ3BXelf23\\\\\\\/lMEQEoUhLRFLTtkRGun4vyOW6UKcvJtAaTzUXj58y0ECQ7Jd3j5t4QDGvd3s+0\\\\\\\/OPMDWeG\\\\\\\/3dvV2pdDLPEyH0RZar6voW2AHzFVRcGhCqAIkMAhEX6Zc78yedPfG5en0Ky89+cwLC1\\\\\\\/aR\\\\\\\/q+9piEWiHPHtHrD3OHNJwEhI\\\\\\\/6B1V4pz7W0Az+HLaYzQAbKi51r4fCJW6tYF283uA7Aq5wPVqGbXg6mQffv8lDvHjZRqMRkZIXW0J\\\\\\\/JvlZi3JLylAKsfgA7oCg5Olv7ked3HYeytGAnWLQ8aRym5oA93MGjFsOn+l68UioLplv3jlFk3HaKb4hdk\\\\\\\/sLYOr5r9qd5VVtDYzbCijQr7BA7oSqfbgGEG6M7uFLRCA6hYliislDd5\\\\\\\/GrW3XuO5yjS5C5o+pG4kSKekbn8QJOgx2a\\\\\\\/AI6OhD2\\\\\\\/vzLNCwCoaLmQ5dQngDaU4Lef4Enr9dxLEi5yxC2TjtcOcyZq7nzao10tgglDHsZ1OkGud7D\\\\\\\/BmTg+P0DD\\\\\\\/LYzkvtSwxFwPUMbz7c9a60hzQEWPnnt9DZ4GcuXS5TOcKPp0GOfluiiPK\\\\\\\/NhmeD\\\\\\\/TkwMeEBfQ9bvu+LMNPKzknV67jNSK\\\\\\\/mGYrcfj\\\\\\\/bYYOl74Tl0YK+kyi0MA+R6fEiL54xb0VmMdPwYklaF5qFyD3VAhFFomgbO5dlhNaxDk8S94gDTNMZhvKl3f7JfA\\\\\\\/onr34mx3DAO7WpolBF+gO4z1+mBAPvCJINFKMpoMysuuRS2+GIHJn8raEa6\\\\\\\/FOGMnmOPMA+BwpOBkDXiVW9zIsaArd77CPSJ3zPeXMBIwd2g1TdtwTpziLGOaG6HYbS1Od6y6y7+Sp6gZRfiwMQVWeMppRNAoB35CVvEsAzjCtj\\\\\\\/Hv7\\\\\\\/SNT9JJaCXqUQf37rbJ9u6t7PbUz8edn7HhNM+TWqmfb5ELPLHUkktag1jsbLLgbo8Chl7AljSoGp5pZq4gfwB\\\\\\\/fqzR3VUqlcFsSKe63TtMhhDnLCOV4p9MeJn0mfd8dS5hr9gZtquOI3EezNiYSFS58sgSLNBjtHd4DiuNDn\\\\\\\/umqI2RGCb3g+quJXCxtpSpu628p3Xxv43vpqrxfRG8AsXnGyzln6jCUV6lWKrpSM9WEsa8nFSzF5ekGGAtWGYPpvMpry6DXPSIk0NNujxWXwj\\\\\\\/vxJQhQw8ldLJ\\\\\\\/iaJlS\\\\\\\/WfL28QRxaksfGMa4+b1GPpyvA7jMIyNtyoHlZGZe\\\\\\\/X6Zk7HB0A\\\\\\\/SldQAiWrd8L3lih26oO8PTh0R4BZBdTxv++ihPBFwvKc8QTiFhbVEEjqvaZILUBgyx4u2RF9yRTWjX2ZqZ\\\\\\\/LeRRkTrtRwFIOkggRrS4Xs+4oEA0j\\\\\\\/kR4sVn3zF25C0UGpkbi49vweCeuLCnutzJtsAS+u0IEVuc00nz+Q5Sxx7M9+W\\\\\\\/PU\\\\\\\/1+FxNPt3bRMl7BwMWP4xMQRY5s2OsuNSRN4DCoQ9jsmiXC262U52Y7USYcfYHfDocYMlYOHIrWdsx+H7ZPdpqvkkFBiZNpDKQtPEOI\\\\\\\/YfZR\\\\\\\/qj5mlQkcPNAg5Dl3N2WLRm4oB4VI6Isq91UIiOZeQjiUQlyPjKltX7WipXvuue1Y+okxA0pew6guwlRER3qEebmCh3\\\\\\\/UZNhQHWNpK1Zh78Vu2rVFA3x5BAXPuhXjevsZvqtLvfPQoj9TkM9x+heHtYKuF\\\\\\\/zxVhqEFdmfiB2QO961n0M2vM0JIqYnRdCRr93vH1UJTdrrxBPSMqMXJbJUo5ofxIKt5t9G8jUCwXM1plRlPHgUX5cEwb+yaB8lt7BmSoVMkmFSD1xBYOeRB6Gjyi5rFIw8maf\\\\\\\/krzBH8ds300sfHZoVhldTd5wpNAS5K7HMJcAQuspOfSITezdUXYe1t0hssEG6wx9Qwyn8s62CBvbMArQhZwxSLR+IZ3t09ZbwFqj6U39ywcAN9vz1gPSFet162RIeZWv12s+UJFqQtT3kN4c\\\\\\\/1RgvU3bKT+FCqu7gkvluRZ41jBHoSvjwgja\\\\\\\/o9Z0m9WAaTGRiEZNFq5QAHee1mM1TZRHRdZwrM\\\\\\\/h+TP4z0F9g4jeckgnDkaxe6eCSAbAUIfoVAAbSlXDuNnjeloRO8Y5uRoXtDo8eKnMhdBQyFFVoLIKk0dqXcGt\\\\\\\/7DC4E2qKl0ot25y2LmU0fgj3Y+sMeSREAmsEClHeZ6\\\\\\\/IrA\\\\\\\/x0k0vdjJXy2tiKInP8G00ncD8vIuLjX0zBp9LiY15EGz+FV5iVv0RQo+BfuZL0fKhc8vXYslYRqMBKRTjlNuNcFxnVEMqmpszf1kvtPSrfsb9jgrJvmODe2qsXYbU+RWN9JEndySxbcaJBLzi99XFlXcuVaXbluTM4DkhiC16nY85vXKO5X\\\\\\\/yUR2jDIWhSJkIruTApgd9DtjIl7hrQEB3oCWUFer9WDm46LWbaxEdpmdMUGB6rVes8ZCR\\\\\\\/HPnftJUu\\\\\\\/z5bZOdo\\\\\\\/5Ygynyb9MAppAU117hLuUyF\\\\\\\/3povg69b\\\\\\\/Hm1959gSYFqYPHjo8pL4J2bfTXD5YHH\\\\\\\/k9bqs\\\\\\\/YbQC+uxWd77iIGrh1RQirDKGqKTCuXHCiMQxGSv+YV8iQBfzIbA3Z75CVk7lq3xovAz+lr\\\\\\\/k2mpUEd3zfmRJaOaWEdovUZaEJWN0O8MI\\\\\\\/yXAk85HjAyEpAAgq8u5DTyha83B+C0EeaY\\\\\\\/v9hexsH2BpXBHJGP3oa+Ai7qaiwv80gpqkhjdeOO1Cr5cxX4ln7KtR0EA4JBPs1I4XZEqzbUnUQCQbMT5y7JuTMTyp2i3hB4DoqZMvpqZzqfldCY9Y731QNy1tYCz23sl6RVxSGLiKWmYSYy83YdFaXp9GkgD6RLDsB+5f\\\\\\\/7CiBdsXjlLpXtXlpMoQV8v8F75OmWM\\\\\\\/+w7SA951CGgLU4+mgX\\\\\\\/3CFYhdjC2MhGLJBBuC6SCtvcIHD\\\\\\\/STYw2XyejzYOGsIGBbRSIWPQZT9ihLq7Kd+lQFJtyaTHJJGXD9HEXZR32YP9aqEVhPsT1TIJk3RxItZASYDp9fGlgUgCt4cCwX6bmWo\\\\\\\/NUrFvkVo1Fi0LMjshWeQ6v\\\\\\\/mZr5L5djUNkzeaV1bXPW11cjzW49S0G2zENFpGtKz9aNf\\\\\\\/HcbUU0TC3n12pZs77lHFhlKCiJ6IPGX8R7Ueods7nxqBHMSHVL5rn8BYavSCUeJ703gLsS5LI\\\\\\\/WETstdUlGcWu3hdx53x8WhEKwCnmjcAkZO6oRml2EK1CUeFPiiOmpFo5pRW1gNpFi1NOHW68nu\\\\\\\/auo\\\\\\\/ij1saSTpJ7mea25wLrK5O6lwehzBZQ1OV+tLh6+wqCUDkBTlMSX6sm9RfUHv5gkHI1CMS25Ux4kWv844JEpyiJa\\\\\\\/6TGB6Neao0sJIwOPPgTFlXlbRGCjF24HfwOjiQ3ONz8aPSkXm2\\\\\\\/\\\\\\\/8nhxjQylRzFwn5osabwD\\\\\\\/93RoxNDaDnMKN7uzrdeRQogfOHqs4AJFnzAYcwF60jkHUAroqiBbt10dtGJa8s3OuMG3RywIpp5bAbc0+4BmrnKcd3mF4tcUKjEZbgOodgVpYX+fz7YO6ZewaB235EWr3xmj2ZDtH0wV6F\\\\\\\/y\\\\\\\/riwIO1MLYWifaMvZ3SPcjcPJEMtBqitniyazoo50aWum24rsGWkX5bMQ6UJUSLtm3c8VZTc4K0xh1+MC5YEEImtMqrlqu8phabh7mt6LvABZfeuyWVx1eIH5lFQWQI8PiVJPpDPLsLUGdsDjrZbxg9Qx71w2FgSpom9k48UuAcaX2iBBtCco0ZLnN71zHwSpiFitmS4qpOg1kXq+EwoIWp\\\\\\\/\\\\\\\/FS2\\\\\\\/hyXA4AAiE0QkVh1C0dy7kylnRhof9HBDlKQavMQmUnedLBr80+z2kujTr2nr5nzDbiN1WuUlRlnKd5ygrX3UnhjXQTpzUL3Mw5cRzRSKE45U5ySz+1Tmmd2bmh4MLdppmzHIp+9e5gWIKCSSMz5IQHhuYQ3Axu1kZ3U1IndtchPGalSWmOdCGt+vyglOS9mcTdKuuxjGlhKWYkjF9CZckpKjnHmPE3gADHyEO7BUKcpprRWlEGotYjuZriU9r\\\\\\\/HwaxOOdQ0CV5z98nfSdwnLEXGa8xKb8yJU7nlDmYVBR3TxK8Ag6c83+k86zWsuIwmtOAEuuRVUMUnMtYNMFNEpYK0ZYHtr8MKcDTuXe3Oe58ZNl989O9\\\\\\\/vw\\\\\\\/2aMeslq+6My8ANvX+yy8oq48wvrEyhKqlMdQrKqbpOyrhHZBh9KwhzHWrlgUKdgRYVmSOV2xR4citCJxdXpFQi5lafDLMpe03xe0no3\\\\\\\/WQzt39e1CJbSMohpLxdXgJoYJLUOI1yik4F6IoCGu7vY7E9HzEltEyFIzLkOayBwEr42hIp5HYCvLAhfKU7NhueDfFcIJ\\\\\\\/Dqir2qFDhN3PMe+SfceMsZ8s\\\\\\\/U8x7iMPZqOnpAYIRtm9vAd\\\\\\\/vnRZRSc6Ogt3SgKQDAMWyzLcG18pU4KbBYnt\\\\\\\/G+dZNdkkzk8rZNZ4PU08Lt0xZk08hmrk9et00DfouTjvsNteLJMfQB321iLuqdEgaape1hRnW2+TMbcoOy14hd4aqSBTrxuskg3aXYFteG7LbjSrmeipbPjCFxc795U8z4tuKrrmGj5P9mte41a+KroYWtrLmchjOscCPPj3FtLSkXXJ\\\\\\\/3lMTrlekAOOldQtIzuxXMR8VdsluAY40QaMnT8Jx1FwsWXtnjju6WrvKdrvLVmhuoLn8cSEbRUlGh+EHeHhMZt+SDZ1wNUtHosjvkFezc8MewhTMbE5zueNKMjI0AtyeICq24+6UNbeuScEJiU9\\\\\\\/5QsML+\\\\\\\/mGPi0AbZt4KxbRczUPtMSLHziZd4\\\\\\\/PnqLiKTIh3NA9yYjjsWqHB5bRkI+FNmDb4vCpUIumYnP3UoImVv9dfK7bnvoqqb4qcKX8ZoXME8xGucM3V38EKgT10RL5M3eLQqJJlobRMXDt5X8TjiXKJc3YiACYI8YWNwXNeRgW53Xlx2t7OdWWKV74RWX7toUwWQEJue1yLWMJqqhgNpqCGKXxWtCuzit0GVLW35Hd+QVa3iwLdRcX2Bi42B9PQNkMjo+jzRiyOwSRDwDhTTUBj1uRfY40XhqNmjOekJsmmJuOVxlUwLw+5G538KdPg8XhpmAHsdSFi6gw4Q1KyTsXB6HHK+J1iqNj14MXN45PWXOq8pbuibhuat7cuRNpC\\\\\\\/jKwAfb38hZmBnBn03B48Wn06J3f0GkSJGTtj0nSDMlyCyA8niEbGM1+Zwztw0Knut9sbIKzr0qdfuZe2TNNEedK4QNlcdk2x+4LPlKpmBs7HvgULSKZG046c50YHtyxVXK56WUyMXNXygvl36dSis\\\\\\\/u8J1eEBh371684b6hHa7ffaApH1CKtBK2zs\\\\\\\/shmRvonOM5bJGufG1dNzCfNLcL0IO4NQ+P\\\\\\\/ybkGeH64Bj2NaKZ5x5o321cznWQ1T+XgqhWgC+VrFCpEfSXeWUxqgW6FUQnv7tCWezP49rDwzYH2hgjqd0WKP1vB0If8WBoRGft1fWPb7qE2rXxhE0vqdXFdRndvGvkCXfEj+Y4g4+rfQ3blEGEOFL+Vv30wmx0OGg\\\\\\\/MLCZdK\\\\\\\/NTqWPZSpPmjAsIIoZM+4\\\\\\\/2hQX9iQN4xbDcH9CZifJORlmzIF8dfrF7k5y9tVyHFuNHWci7KiOcoe7BmYFNj3LAcObbeR\\\\\\\/kFBeKN70jHTH30Re0BrjXPkCX5HmQQGhca58158CSXN8qZpWZbjJFFIJi7BDF90sJh4pcKtU7X7rjMqsyYasTYN1z2L2BWpduTQWsbGPRCAF0LKG0W7WsHf4J\\\\\\\/YWOnY\\\\\\\/i4\\\\\\\/tSLsXySOPcZOn0ac47dbZeRotK3yzQbDMiA2bMSv3S5Y30E2uEx6eyT7UruqEu5a626M6\\\\\\\/SGPPg4jXmtF13O3WsK+coPqWFbCqgYq4b5NjIsjj0WrloMYuPvrGFPNbVhRVyTW2FVDabpLlZ7+KwStZpl+BHYVb+o4tCnMTqqSZxCkSPIwMopE97JYHmhWWXkwxqvWWNxnZPvh8+mdsfXrPGsE6cmuzhjgb4kvdjmLLi37mm08\\\\\\\/YyGpIWJApVlmfEvhoirrwbtscrhEZA9I4WQEhJYGKnRIaEhj75XP9Lu1\\\\\\\/c4HbxW8esjeveyLhB3lEzTkSTxn87\\\\\\\/4qyXcMG\\\\\\\/LyT28CQYYr3xtGzG89EM7LhcWX5x0G\\\\\\\/xMsXaZ35\\\\\\\/myW0WoQ2zC+IAMQtZpPfMxc+VodTtSaIqjpnls0iFYc7EmisiUxDAqX5r1h96QjUbb8LWi7N6FP8ZdtCFJvnC0UcSFzpyhXFeYVmxB0pCeEeHLgyE1fy4xReccwDACWK\\\\\\\/+N\\\\\\\/VhBJFbw+f4ojyf3nJ02j5NRDtuM2Ylm+xRUQzC1wNBSD4nbj6GferZMw+8u\\\\\\\/yIQCOI6p7FGyMRaX01jiY0NYwz7u8BF74B0yZuvXHnByau+gOTnQQ22NP5QytMwMgOCAJZRmxlVXWoEh+BaBt0Fsrn35\\\\\\\/KDnlAm5JSaQ7H04gKtYKcV\\\\\\\/j8oVwYXYVlmbYNNV6o0Y3Ho2i5EiH9nCSbvuu3n1eW1lSyGA7Wreu9U7cnziYzTTEaFaxd7f8k3iI4\\\\\\\/726ioOasdo+lfu4p2DCVmQQrJ3fsgXJ0\\\\\\\/sLz1QOnAk++Vti2hO21QDQBPjcWuu\\\\\\\/lLW63hZmZd7ORRAxV3aEcmsXr4RcTvpML0p172IoqsTcRdjixhCTuR32saT4xFZH1p8qwc7LNvS1D\\\\\\\/rZ6olxW915LCWIV4YdgRkt4N98Tb4qlb2mEZeTVFKkcyPZWkVQHLxB8aOTQnPhWninQJOJ7Q3epSU3mQPfyKRQV0tsB\\\\\\\/vxC0p2DfYJWwBgxcnCiJpkPdpgvjHinJyWpOQnNyAVBaxdeh0wMz9vincF6A9\\\\\\\/21zm674V5Cz8Wq+g\\\\\\\/X+BiQpWhAzKFszbbwsaVJmfj6\\\\\\\/VMIiRyFRN9y9o5P6BfQ+JZO9s3bBUuv+4Edt64XNZo+ALEYUKJPJ9e18yAP6vEY86uer8Q7\\\\\\\/kzH7wlLROcSsCTTD7FPzxKgjYlMuo1JD9a0FmqlIfp0zbSb3sYxR2W135fxGosqRwfhaTDJ02qbPsKrHtxv8BzcwhOP8ZemTN84V+HI5tI89xsqakerbpJrlVUGmdvTQH0sYS4J0KRwTVWU2IT9DqDRYT9zrMp8jDyJCgMsQ\\\\\\\/rcueBN+J619OxtYNYULFaQHbcz+uAyke+HsmDKxEdFB68xzx1tvUbQGuDuPQ1NTXRQMASppA\\\\\\\/FiLBqs9nFjCh5kjkZrv4Zr12thzL1bPpzG8yfuUcxC61Zi\\\\\\\/92hrt\\\\\\\/fNnUWEJDtO9dalqc19R4LdheBHTJLjjpVrHoQ+MkBeyn6bBoVuZ8Vw2kFHSTPDm0HIGDxNbf3wj6cAdw6eSMWqE8XauTus8JCjXjXmDsCKRXvu+ujCDQW6FSKR20PMmdjrkSjJ4DxWFoemoQuTCq3TiUoX2NOp34Ukw7VdjcGC6WTAFkJ+I3JvcIyXUbJ07cjODdjFWLsYtLqQphwoeYJsQNnQUt8q1YEeTd0zvF0MQ8x7V6fKk\\\\\\\/77Kms2NvdxaQ+4wP4DzW0cLht50t8rpHm0loqo9D9f42scwjKq\\\\\\\/0hOFVSONmw0UghmZM7\\\\\\\/qI0zzkKhUGub2KhpP1fEZ0+AWtUkwCKDwnGZh8C8eJ38sMXDAR\\\\\\\/Pr4dWptR4t5WTbcORFAbvPN5dUqnWsy3yaQOxMO+gGZSSAARVNhsPO4qYPAbTXGOBgoEPhntpXrdvA9s3zzXenMdfeeUDdY6J7X\\\\\\\/eRiwtwZoeIZ5CUKzop8rMpNyHlyZBq4jbrsWlvCAMIBMXEw3WS2t2Yj1wikU5NZyS9eejwuJz5pFE+0QzpKaUG45FWik5pJoJdkoVZJ2wPoMy9+aoVT\\\\\\\/Q1brO\\\\\\\/F6H6K82Zl\\\\\\\/CEh0R8O0VPFBC+Zxusi5a0ly+8uUUOlYtpsM\\\\\\\/U31oc2lRRWze\\\\\\\/+Vj8tBalqL36qXphdIV\\\\\\\/KYuwz9yI4jXDRRHtM9B\\\\\\\/GwroGZbV520+Xpds66qNP0W1Cv8Enx13xUQ7e1LE4e4xsQkXz1fqfgq0DV5Sna\\\\\\\/LgQ6G63DL1dkK+jUUUnQzUrIEBgAKtjgTKkG8pgifexiLZ9i1t1Y6Bpx6ss6bSK753bszJqRMd7PgDckbSPGTNZ5rpBFuHjrk1wYIhTvDUWEC7dw0JwKPuQJNjyvEbfvxKrt2tTzTqaBXdnxy5RpVdU7PtOdK\\\\\\\/f9RDiBBPjMnX+lX7J\\\\\\\/\\\\\\\/jBoy9FB+OIlMH+zD\\\\\\\/NBZCyigZvx5W5vPlUPOJtu2OOMuRTTBROzRLC7Ww6eUMF+z374wwt+h0M9PrSoirguHwPQtG0rRkzRk\\\\\\\/eh9z47I4nMUCTYXS8JfRfFFnPYIO\\\\\\\/bhKCog7woqsSZHYa\\\\\\\/ihSoMMH\\\\\\\/B5CNfK\\\\\\\/ixU2pj\\\\\\\/9hH8GYKG+C02vONKGeJmCCjCWUwIy6rwS0F7cM1x4ONlX5D3qMYqUUKzY\\\\\\\/D10MWBc5QoPPx2gBB4wGDTQ1i6sxP8cGP6trzJkgxp3gx7J+8c+9yb7sZQ\\\\\\\/EOGoSDBDNgK0CTvWbuRWS3yotYeS+PtTvoyUVlxpUA\\\\\\\/5yJTyx+oMfO7P8bAKq\\\\\\\/dq8zB746iW8ozQ3JWHbZQMqwG2XH1DJCyAZtpI4wtIx1OA4d1IsepUm5URuGHm0G9jtQSnLfNS9BCvLHO70nDm3sXLHWoFnOahUH2jB4p25ab8EGrY9DM54ikvW9XAu1DWSKO4N0XBbb3e3T+ncuY4J4lLOBAIB+fms3o+Rm4rNaCPnukT\\\\\\\/3+vcq42bGmU6W9AXt6BHjfNWk7Eqtk\\\\\\\/Exx\\\\\\\/0P\\\\\\\/4DeH3IswFJ92oxP+Fsr6dr0fJ5Dq1d3kHQn6NiHeCmnSm4lop4kTPCy4sbudLyd9JYDPqAbP3teizDLyiGz6lh94fHAGobj2RiqhZlBQiikza0XLZjXDGUk6YJviyWyVq+0+YS2xm4gxvhLgc62rEnQy9klPPwXTp3RjL11lyhch42yZpRrWcyMA3AMKVnik1WQnDV5Ifah6xWzf\\\\\\\/Nh2F3t0V5FKN6rA5sz\\\\\\\/4zqCY3Slpv4S5iy7z3pcDGozwcz732\\\\\\\/Z+SMGIT37dX3KTmLS33Qc43Loxhr0DT9AKjII8jA6V2AD8tbU\\\\\\\/tApS\\\\\\\/Rd4Z3zYcGL29D3eM6rPx2tIXpeTY3xw1C7JS0sKOBHh91WWWBvcXLMIp02pFQeycoPVw8Nc5KNI2zto61v6pns4M1z061GTqgIrXA2nNljEUvJeRNF4BeXAA+Dy5l\\\\\\\/ujRYbLWMIHboAJPl67S8RdCPJErwRXxWVTvViXvrv2MQl+OESqd5KPR0khOsNcHGykzj3eAH\\\\\\\/ABg+6V\\\\\\\/MJKLmkdPHd4fb2TKLHSpjhoA2nviWmqjAF141PrOQsBes9zHdhN8ryOb\\\\\\\/UtKrLF0p5BPr4LVIvsx42qpLatxoptUTZrgeU=\\\u0022,\\\u0022iv\\\u0022:\\\u0022ugOk2i0Kk3PzVBebsNSryA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223f484f746286e43790fba0effec7d37d4609a4eec12f94a0781df812617643f4\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022YQFdFwVob6bNF\\\\\\\/9\\\\\\\/m0PHmyJq3nnFWh76p7g1LvnWhorV7AHC0l2indi+dnceL7cMWmzclmBe2bTozzOxps5ME\\\\\\\/juC9d8ju\\\\\\\/KdiBEteXKUN3w8+Gzq\\\\\\\/wZICBx7GaENN+BC4\\\\\\\/\\\\\\\/9DR8aL2cA6uk7TtOSN6ctf12SAfF32YL7c4yE\\\\\\\/rd8p8mvgewa9Q4OzcwUE8cvaiPSxG2scuyg\\\\\\\/\\\\\\\/7xOdiPSg1BuQDdUOdXoAVWuQAj6M1O0zn21UQalnUw0EUn0dyuMx7c1CuDt1Vgdg5sm3BXnX6SyRoD9iYsgEv4aEJZ0m1cn4\\\\\\\/dVmbKFWP5IhFMaPKYSLn7\\\\\\\/poAkpZLSRDMuQ6cWG65LOxil5HyGXbYRBFnhY1hxFvMuKLtKl7BHF+B60vW4sgO7\\\\\\\/+PQQYORqvRwFrPUFvAT31tqbcbCwWfXg6+75deli2xPMgRL48Tn4LcbWLW+kllQBFiXDm2TWS\\\\\\\/UZlpyn22ZBqSHwJM9Aa9OIYQ721evthPkLKcaRDWpLKgB5xVX9Mjdy2UYFy7ZPP5mV9gHI\\\\\\\/KN0oMX8QrgmQL6OR23Ey\\\\\\\/jdUSjpFXO8ynX2F6OsyBUK6NtsPJk5\\\\\\\/X5xHALdq8E8winJKGaAwv6g561XmiYbp4jp7jAAK60yNnRXu+Qa0A9go3kmf\\\\\\\/z5UmZdVkR29JRsVeaHma2+Qo6uEtlEiOU4Glyc5nV1YNgIvqTOxj+FQz5Qm9zRU2YsbOu4jdH0upgMAuqmplvGHW8+lO1ehz+BbsJhHGgAYivTX1l2lnc2nCDPEiApfedA9hv4gpKsBcthMcpbyFUnsbHzz57DQYfEF78KerObCJv6mdYyirTpj3CAuGVSOlPZlq5sa5LcV9NkulJIUkwXdiiohl9y2aVeZ3RLXh0nH5Q\\\\\\\/Oct4k0lhU4TR0DpEPCJ9M75O9outjAID2Mn+JA2vZReRBKqVpQC+3K5Udv1WFSrnN2ZjzLY02unSf4fAR53glhp13PLl24yMgCXK+M\\\\\\\/UgustB2hwQMebengKb\\\\\\\/RyIdoXBP8j033LHyjAEL0RxYrDwT4VJ54QSGAuHcozDRRkIxA89XW6CcDNMOAULJ6lIXbY1y4Ic5Sb697KO7ZE7W+dxVwWwmBbdcZIJJYIn1J8dvLGdG9WNDUaYksDmAh5kLxrOqHVSpuKtPps8THOBIZvZlmSuTaQhxJO66fTHlOgFjl40vdjK3qqg3Yex7HGW9nle6hVaRJPcPucv2aS18nI5CsB7hZiLosIwGVKbMHGkng4n3roo25DfN50CZMUivaq4zD3FZAy4UAvf9H8XsmqRtAQNoRpiA5jLu7zzV+5kUmgO5y0H1fVcqBrg8yV3d3aHk98Di\\\\\\\/jF9LB4WcQbTXVfnx6CZpcHtQQ1CFYFwK+Rx5D+0l7PgNqPiDr5QN\\\\\\\/XxiR2PUamT\\\\\\\/RX4\\\\\\\/NWd8XfVzC5eWv6Y0oSU0WM8\\\\\\\/X0eoWV9mh1QQ45Cj0VdiqLULHTiwZEubP\\\\\\\/+l6eCVqckkk6gstVMG8itaHjMl1aHDbet7+0G1RI1pPSBYhOcVWY7XMubL1fo30TIOo+kKN2LpOFRXYp56\\\\\\\/A+tXCeO3e9ETOjjZboYpd4HVDKIK0E72DyvEUxBt8azb\\\\\\\/KCTYt0BoI7WkjKD3QOoagDSQcl2ovxK2Xv07VNQwQWmINpgjgqC+QCvfqWNYiR6QkMtmFRaGQHaHivEtnH4HnCKwAcvVdlbYyUrh9cyaOMB6PukXPxDp07q8CFhZfPUq8J3ppQ97FlJ4OxtkllRvJMueK6BzC6wk9IBMRplH6SuFL4BYQnnQT3n4ZS6sbLAMvSfDHHj\\\\\\\/7rFWLeRTM9CdmgMl\\\\\\\/a0mOq7VA2FfI44Zd0R7NmLIU+JEkhk+KLzOKTNAPMCCBQhZNPvkPiQEZg1I5ksNhnOSdfd\\\\\\\/Sk7loHDkjuvhvFT3oEk2NK1+jL4KXFaL4dxC6Lo5WgXCktpgo2zvhkgrvxB0Sy1bIwDAdMnDi9dwzk1c02qqeDQzgvTIN1dyXzlO6h6fG4YBy6Nux\\\\\\\/LtGDvD6M\\\\\\\/n30wQZK9IJB3LT3ngFwz527la+wyW36qhYqaY8va18KeEOn9+FFW06EpUPaKNx6Wxc5623WenUxxQDvt\\\\\\\/pbrhYFBiS09QGqZdH4\\\\\\\/5NSdKLA2fqsvON2L2CwURezF9cA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Kb7nUsDujiTxpNkJ9e80wA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221f858c186e027066b58ce1cc4159c84d266a13bdd401e3e47836d7bb25c4f440\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460078 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/3/559" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-UbmhgKsxMvJfgsTVdrokdJwut0P6roCp2ebiSIX6mtQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/3/559" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-8Ur2ExZHnWpSlkGLWVS8scu8-yrnjR_m29_7qPvb0_I" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/3/551" title="Isolation of a Hypomorphic skn-1 Allele That Does Not Require a Balancer for Maintenance" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/3/573" title="A Novel Recombinant DNA System for High Efficiency Affinity Purification of Proteins in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460078" id="node460078" data-pisa="ggg;6/3/559" data-pisa-master="ggg;g3.115.025841" data-apath="/ggg/6/3/559.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Novel Heterotypic Rox Sites for Combinatorial Dre Recombination Strategies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Katherine Chuang</span>, <span class="highwire-citation-author" data-delta="1">Eileen Nguyen</span>, <span class="highwire-citation-author" data-delta="2">Yuri Sergeev</span> and <span class="highwire-citation-author" data-delta="3">Tudor C. Badea</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">559-571;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.025841 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460078" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Katherine Chuang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Retinal Circuits Development and Genetics Unit, National Eye Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katherine%2BChuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chuang%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKatherine%2BChuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Eileen Nguyen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Retinal Circuits Development and Genetics Unit, National Eye Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eileen%2BNguyen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nguyen%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEileen%2BNguyen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yuri Sergeev </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Retinal Circuits Development and Genetics Unit, National Eye Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yuri%2BSergeev%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sergeev%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYuri%2BSergeev%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Tudor C. Badea </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Retinal Circuits Development and Genetics Unit, National Eye Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tudor%2BC.%2BBadea%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Badea%20TC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATudor%2BC.%2BBadea%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:badeatc@mail.nih.gov" class="" data-icon-position="" data-hide-link-title="0">badeatc@mail.nih.gov</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/3/559" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460078" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460078/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/3/559.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460078" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460078/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/3/559.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460078" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460078/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/3/559.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460078" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460078/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/3/559.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460078" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460078/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Site-specific recombinases (SSRs) such as Cre are widely used in gene targeting and genetic approaches for cell labeling and manipulation. They mediate DNA strand exchange between two DNA molecules at dedicated recognition sites. Precise understanding of the Cre recombination mechanism, including the role of individual base pairs in its loxP target site, guided the generation of mutant lox sites that specifically recombine with themselves but not with the wild type loxP. This has led to the development of a variety of combinatorial Cre-dependent genetic strategies, such as multicolor reporters, irreversible inversions, or recombination-mediated cassette exchange. Dre, a Cre-related phage integrase that recognizes roxP sites, does not cross-react with the Cre-loxP system, but has similar recombination efficiency. We have previously described intersectional genetic strategies combining Dre and Cre. We now report a mutagenesis screen aimed at identifying roxP base pairs critical for self-recognition. We describe several rox variant sites that are incompatible with roxP, but are able to efficiently recombine with themselves in either purified systems or bacterial and eukaryotic tissue culture systems. These newly identified rox sites are not recognized by Cre, thus enabling potential combinatorial strategies involving Cre, Dre, and target loci including multiple loxP and roxP variants.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/site-specific-recombination-0" class="hw-term hw-article-keyword hw-article-keyword-site-specific-recombination" rel="nofollow">site specific recombination</a></li><li class="kwd"><a href="/keyword/cre-recombination" class="hw-term hw-article-keyword hw-article-keyword-cre-recombination" rel="nofollow">Cre recombination</a></li><li class="kwd"><a href="/keyword/dre-recombinase" class="hw-term hw-article-keyword hw-article-keyword-dre-recombinase" rel="nofollow">Dre recombinase</a></li><li class="kwd"><a href="/keyword/gene-targeting" class="hw-term hw-article-keyword hw-article-keyword-gene-targeting" rel="nofollow">gene targeting</a></li></ul><p id="p-3">SSRs such as Cre, Flp, and phiC31 are used extensively in genetic model organisms to direct tissue and/or time-dependent genetic manipulations (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Sauer and Henderson 1988</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Grindley <em>et al.</em> 2006</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bischof and Basler 2008</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Birling <em>et al.</em> 2009</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Jefferis and Livet 2012</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Turan <em>et al.</em> 2013</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Jensen and Dymecki 2014</a>). Typical application involves the expression of a recombinase (<em>e.g.</em>, Cre) in a particular subset of cells, inducing a recombination event at its target sites (<em>e.g.</em>, loxP sites) resulting in gene ablation or induction of a reporter or effector gene. As the resolution of addressed biological questions increases, so does the requirement for restricting our genetic manipulations to specific cell types and time windows. Spatially and/or temporally restricted manipulation is typically achieved by intersecting the expression patterns of existent or engineered gene regulatory elements that control the expression of the recombinase and its target. The target will be recombined in all tissues expressing the Cre, however, only the subset that also expresses the targeted gene or reporter will exhibit a phenotype, cell labeling, or other desired manipulation. Modern gene expression analyses reveal broadly overlapping gene expression profiles, and in only a few instances can unique cell types be suitably approached by intersecting the expression domains of two genes. This issue is particularly critical in the nervous system, where a multitude of cell types are present (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Jefferis and Livet 2012</a>). Therefore, intersectional strategies using multiple recombinases, genetic loci, and/or local vector delivery are of great interest (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Fenno <em>et al.</em> 2014</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Madisen <em>et al.</em> 2015</a>).</p><p id="p-4">The Cre-loxP recombination system is by far the most developed of the genetic manipulation toolbox, benefitting from a variety of mouse lines expressing Cre or Cre-dependent targets (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Nagy <em>et al.</em> 2009</a>) and flexibility of strategies used to generate tightly regulated or combinatorially recombined target sites (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Schnutgen <em>et al.</em> 2003</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Livet <em>et al.</em> 2007</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Atasoy <em>et al.</em> 2008</a>). These targeting methodologies have been made possible by our extensive knowledge of the Cre-loxP recombination mechanism (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Gopaul and Van Duyne 1999</a>). The loxP target site consists of two 13 bp inverted repeats, flanking an asymmetric 8 bp spacer (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>, horizontal arrows) (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hoess and Abremski 1984</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hoess <em>et al.</em> 1986</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Guo <em>et al.</em> 1997</a>). Cre recombination involves the formation of a complex containing two loxP sites and four Cre proteins. During recombination, each inverted repeat of the two lox sites is bound specifically by one Cre protein. The spacer regions undergo single strand breaks, followed by strand exchange, strand migration and formation of a Holiday junction, a second cleavage and strand exchange, and finally the resolving of the Cre-loxP complex (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Guo <em>et al.</em> 1997</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Gopaul and Van Duyne 1999</a>; <a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Grindley <em>et al.</em> 2006</a>). The asymmetric spacer base pairs are involved in the catalytic reaction, and ensure the unidirectionality of the Cre-loxP reaction. If the two loxP sites present on the same DNA segment have identical orientations, Cre recombination results in an excision reaction, while loxP sites with opposing orientation produce inversion reactions. loxP strand breaks occur between spacer positions 1â2 on the top strand and 7â8 on the bottom strand (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1A</a>, vertical arrows) (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Hoess and Abremski 1985</a>; <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Guo <em>et al.</em> 1997</a>). Several screens have been performed to explore the influence of the 8 bp spacer sequence on recombination efficiency and specificity (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lee and Saito 1998</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Siegel <em>et al.</em> 2001</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Langer <em>et al.</em> 2002</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Missirlis <em>et al.</em> 2006</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Jung <em>et al.</em> 2007</a>). Systematic base substitution analysis showed that single mutations of the loxP spacer at position 7, or double mutants involving position 7 plus any of positions 2â5 (<em>e.g.</em>, lox2272), yield mutant spacers that do not recombine with the wild type (WT), but do recombine efficiently with themselves. It is proposed that mutations of positions 6 and 7 (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1A</a>) interfere with the initial strand exchange and therefore recombination initiation, while mutations at positions 2 through 5 arrest the loxP recombination in the intermediate states (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Lee and Saito 1998</a>; <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Gopaul and Van Duyne 1999</a>). The two alternative lox site configurations (direct or reversed) combined with lox sites (lox511 and lox 2272) that do not recombine with loxP, but self-recombine efficiently (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Lee and Saito 1998</a>; <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Siegel <em>et al.</em> 2001</a>) are the base for the design of combined inversionâexcision or alternative excision strategies, such as FLEX and Brainbow (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Jefferis and Livet 2012</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/559/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Random nucleotide libraries targeted at the roxP spacer region. (A) Comparison of roxP (top) and loxP (bottom) wild type sites. Horizontal arrows mark the inverted repeats for each target site and the black horizontal bars label identical base pairs. Regions highlighted in red correspond to the spacer region of the loxP site and the equivalent base pairs on the roxP site (numbered 1&#x2013;8). The vertical arrowheads on the loxP sequence mark the sites of catalytic attack and strand break during Cre recombination. Mutations at positions 2 and 7 in the loxP spacer, highlighted in green, are used in the lox2272 mutant. (B) Libraries carrying random nucleotides indicated by N&#x2019;s inserted at positions 3&#x2013;6 (rox4R), 2&#x2013;7 (rox6R) and 2, 3, 6, and 7 (rox2N) in the spacer region of roxP. (C) Plasmids and recombination test used in this paper. pGB2-Dre contains a replication cassette consisting of the repA protein and SC101 origin of replication (ensuring 1&#x2013;5 copies/cell), a spectinomycin resistance gene (Spcr), and a Dre open reading frame driven by a lac promoter (LP). proxA-Kanr-roxB contains a pMB1 origin (generating &gt;75 copies/cell), an ampicillin resistance gene, and a kanamycin (Kanr) resistance cassette, flanked by two rox sites arranged in direct orientation, roxA and roxB. If both rox sites are occupied by wild type roxP sequences (pPKanP vector), Dre recombination results in deletion of the intervening fragment containing the Kanr element. In the unrecombined plasmid, PvuI + PstI digestion results in 1.8 kb and 1.4 kb fragments, while after Dre recombination and Kanr removal the 1.8 kb fragment is reduced to 0.8 kb, while the 1.4 kb fragment (containing the pMB1-Ori) is unaffected. (D) Wild type roxP sites were inserted at the roxB position, while the mutant rox sites carrying the random nucleotide libraries shown in B were cloned into the roxA position, generating the p4RKanP, p6RKanP, and p2NKanP vectors." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-30435511" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Random nucleotide libraries targeted at the roxP spacer region. (A) Comparison of roxP (top) and loxP (bottom) wild type sites. Horizontal arrows mark the inverted repeats for each target site and the black horizontal bars label identical base pairs. Regions highlighted in red correspond to the spacer region of the loxP site and the equivalent base pairs on the roxP site (numbered 1&#x2013;8). The vertical arrowheads on the loxP sequence mark the sites of catalytic attack and strand break during Cre recombination. Mutations at positions 2 and 7 in the loxP spacer, highlighted in green, are used in the lox2272 mutant. (B) Libraries carrying random nucleotides indicated by N&#x2019;s inserted at positions 3&#x2013;6 (rox4R), 2&#x2013;7 (rox6R) and 2, 3, 6, and 7 (rox2N) in the spacer region of roxP. (C) Plasmids and recombination test used in this paper. pGB2-Dre contains a replication cassette consisting of the repA protein and SC101 origin of replication (ensuring 1&#x2013;5 copies/cell), a spectinomycin resistance gene (Spc&lt;sup&gt;r&lt;/sup&gt;), and a Dre open reading frame driven by a lac promoter (LP). proxA-Kan&lt;sup&gt;r&lt;/sup&gt;-roxB contains a pMB1 origin (generating &gt;75 copies/cell), an ampicillin resistance gene, and a kanamycin (Kan&lt;sup&gt;r&lt;/sup&gt;) resistance cassette, flanked by two rox sites arranged in direct orientation, roxA and roxB. If both rox sites are occupied by wild type roxP sequences (pPKanP vector), Dre recombination results in deletion of the intervening fragment containing the Kan&lt;sup&gt;r&lt;/sup&gt; element. In the unrecombined plasmid, &lt;em&gt;Pvu&lt;/em&gt;I + &lt;em&gt;Pst&lt;/em&gt;I digestion results in 1.8 kb and 1.4 kb fragments, while after Dre recombination and Kan&lt;sup&gt;r&lt;/sup&gt; removal the 1.8 kb fragment is reduced to 0.8 kb, while the 1.4 kb fragment (containing the pMB1-Ori) is unaffected. (D) Wild type roxP sites were inserted at the roxB position, while the mutant rox sites carrying the random nucleotide libraries shown in B were cloned into the roxA position, generating the p4RKanP, p6RKanP, and p2NKanP vectors.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/559/F1.medium.gif" width="383" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/3/559/F1.medium.gif" width="383" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/559/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/559/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467788" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-5" class="first-child">Random nucleotide libraries targeted at the roxP spacer region. (A) Comparison of roxP (top) and loxP (bottom) wild type sites. Horizontal arrows mark the inverted repeats for each target site and the black horizontal bars label identical base pairs. Regions highlighted in red correspond to the spacer region of the loxP site and the equivalent base pairs on the roxP site (numbered 1â8). The vertical arrowheads on the loxP sequence mark the sites of catalytic attack and strand break during Cre recombination. Mutations at positions 2 and 7 in the loxP spacer, highlighted in green, are used in the lox2272 mutant. (B) Libraries carrying random nucleotides indicated by Nâs inserted at positions 3â6 (rox4R), 2â7 (rox6R) and 2, 3, 6, and 7 (rox2N) in the spacer region of roxP. (C) Plasmids and recombination test used in this paper. pGB2-Dre contains a replication cassette consisting of the repA protein and SC101 origin of replication (ensuring 1â5 copies/cell), a spectinomycin resistance gene (Spc<sup>r</sup>), and a Dre open reading frame driven by a lac promoter (LP). proxA-Kan<sup>r</sup>-roxB contains a pMB1 origin (generating &gt;75 copies/cell), an ampicillin resistance gene, and a kanamycin (Kan<sup>r</sup>) resistance cassette, flanked by two rox sites arranged in direct orientation, roxA and roxB. If both rox sites are occupied by wild type roxP sequences (pPKanP vector), Dre recombination results in deletion of the intervening fragment containing the Kan<sup>r</sup> element. In the unrecombined plasmid, <em>Pvu</em>I + <em>Pst</em>I digestion results in 1.8 kb and 1.4 kb fragments, while after Dre recombination and Kan<sup>r</sup> removal the 1.8 kb fragment is reduced to 0.8 kb, while the 1.4 kb fragment (containing the pMB1-Ori) is unaffected. (D) Wild type roxP sites were inserted at the roxB position, while the mutant rox sites carrying the random nucleotide libraries shown in B were cloned into the roxA position, generating the p4RKanP, p6RKanP, and p2NKanP vectors.</p><div class="sb-div caption-clear"></div></div></div><p id="p-6">Among many recombinases recently added to the toolbox of genetic manipulations, Dre has several features that make it particularly interesting (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Sauer and McDermott 2004</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Anastassiadis <em>et al.</em> 2009</a>). It is a close homolog of Cre, but has similar recombination efficiency and specificity for its own target sites, while exhibiting no cross-reactivity with the other well established recombinases, Cre and Flp. Several mouse and zebrafish models have been developed and validated demonstrating both the feasibility of Dre recombination in genetic manipulations and its use in combinatorial approaches with other recombinases (<a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Anastassiadis <em>et al.</em> 2009</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Park and Leach 2013</a>; <a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Fenno <em>et al.</em> 2014</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Sajgo <em>et al.</em> 2014</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Madisen <em>et al.</em> 2015</a>). Some of these studies have reported a moderate level of cross-talk between Dre and Cre, especially in the context of viral delivery of the recombinases and targets. Cre and Dre protein sequences are 40% homologous and all Cre active site amino acids (Arg173, His289, Arg292, Trp315, and Tyr324) are conserved in Dre (Arg175, His290, Arg293, Trp316, and Tyr325). Furthermore, there is partial conservation of amino acids and base pairs involved in the interface between Cre and the loxP inverted repeat (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Hoess and Abremski 1984</a>, <a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">1985</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Hoess <em>et al.</em> 1990</a>; <a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Guo <em>et al.</em> 1997</a>; <a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Sauer and McDermott 2004</a>). We therefore speculated that the mechanisms of Dre and Cre might be similar and that the pattern of protein to DNA interaction over the inverted repeat and spacer area are conserved. However, the inverted repeats of the roxP site are 14 bp long, while the asymmetric internal spacer is 4 bp long (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1A</a>), (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Sauer and McDermott 2004</a>). This precludes a simple alignment with the loxP site. Moreover, the regions of identity between the two sites are mostly confined to the inverted repeats, and reach into the first 3 bp of the loxP spacer on only one side (base pairs 1â3, ATG, <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1A</a>). Thus, it is not clear which base pairs of the roxP site might be the functional equivalents of the critical base pairs 2 and 7 in loxP.</p><p id="p-7">We therefore designed a mutation selection screen to identify roxP variants that are not compatible with the WT roxP site but preserve their ability to self-recombine. We have identified several such mutant spacers, and demonstrate their ability to function in a specific and efficient manner in prokaryotic and eukaryotic cell contexts, and analyzed their self-recombination efficiency in a purified system. This recombination specificity and efficiency allows for the development of a tight inversionâexcision strategy, which we call FREX.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Vector design and construction</h3><p id="p-8">Bacterial Dre expression and target library selection constructs were built in vectors with distinct replication origins, allowing for cotransformation in bacteria. pGB2-Dre was generated by inserting a <em>lac</em> promoter â Dre cassette into pGB2 between <em>Eco</em>RI and <em>Hin</em>dIII sites (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1C</a>) (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Churchward <em>et al.</em> 1984</a>). <em>Dre dependent roxP recombination target plasmids and derivatives:</em> a kanamycin resistance cassette (Kan<sup>r</sup>) was PCR amplified from pBBRBB-eGFP (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Vick <em>et al.</em> 2011</a>) and subcloned between <em>Xba</em>I and <em>Not</em>I into pUCBB-eGFP (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Vick <em>et al.</em> 2011</a>), replacing the eGFP expression cassette. A WT roxP site (5â²-TAACTTTAAATAATGCCAATTATTTAAAGTTA-3â²) was inserted between <em>Not</em>I and <em>Xho</em>I, downstream of the Kan<sup>r</sup> gene (pKan-roxP). Next, a WT roxP site was inserted between <em>Eco</em>RI and <em>Nhe</em>I/<em>Xba</em>I sites upstream of the Kan<sup>r</sup> cassette, to generate pPKanP (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1C</a>). Random oligonucleotide insertion libraries (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1B</a>): rox4R (5â²-TAACTTTAAATAATNNNNATTATTTAAAGTTA-3â²), rox6R (5â²-TAACTTTAAATAANNNNNNTTATTTAAAGTTA-3â²), rox2N (5â²-TAACTTTAAATAANNCCNNTTATTTAAAGTTA-3â²) were custom ordered to be commercially built with hand-mixed bases (IDT) and amplified by PCR, adapting <em>Eco</em>RI and <em>Nhe</em>I sites at the ends. They were inserted in pKan-roxP upstream of the kanamycin promoter, between <em>Eco</em>RI and <em>Xba</em>I generating the p4RKanP, p6RKanP, or p2NKanP vector libraries (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1D</a>). Diversity of the libraries was confirmed by sequencing 12 individual clones and comparing the base pair distributions at the mutated sites. For p2NKanP, a total of 24 minipreps were sequenced, and the results are summarized in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2D</a>, bottom panel. Plasmids p7KanP, p8KanP, p9KanP, p12KanP, p34KanP, p61KanP, and p85KanP (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2E</a> and <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>) are clones identified during the screen. Vectors p7Kan7, p8Kan8, p12Kan12, p61Kan61, and p85Kan85 (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>) were generated by replacing the WT roxP downstream of the Kan<sup>r</sup> gene in p7KanP, p8KanP, p12KanP, p61KanP, and p85KanP, respectively, with rox sites carrying the mutated spacer sequences rox7 (AgGCCAgT), rox8 (AcGCCtcT), rox12 (AgGCCtgT), rox61 (AgGCCcgT), and rox85 (AgGCCggT), between <em>Not</em>I and <em>Xho</em>I. <em>Eukaryotic roxP recombination targets</em>: An eGFP-bGH-PA (bovine growth hormone polyadenylation signal) cassette was inserted in pcDNA3.1(-) (Invitrogen) between <em>Eco</em>RI-<em>Hin</em>dIII sites. The Kan<sup>r</sup> cassette in vectors pPKanP, p12Kan12, p12KanP, p85Kan85, and p85KanP was replaced with a mCherry-3xSTOP (3 Ã SV40 polyA transcription termination site) inserted between <em>Not</em>I and <em>Pvu</em>II restriction sites, thus being flanked by the respective roxA and roxB sites of each vector. The orientation of the mCherry-3xSTOP is reversed compared to the orientation of the Kan<sup>r</sup> cassette. Then, the rox-mCherry-3xSTOP-rox cassettes from each of the five vectors: pPCherryP, p12Cherry12, p12CherryP, p85Cherry85, and p85CherryP, were subcloned into the pcDNA3.1(-)-EGFP-bGH-PA vector between <em>Xho</em>I and <em>Eco</em>RI sites, resulting in the final eukaryotic target vectors: proxP-R-roxP-G, prox12-R-rox12-G, proxP-R-rox12-G, prox85-R-roxP-85, and proxP-R-rox85-G (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/559/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Screen strategy and results for identification of mutant rox sites. (A) Strategy used for recombination test (for p4RKanP, p6RKanP, or p2NKanP libraries) and for library screening (p2NKanP library). For details, see also Material and Methods. (B) PvuI + PstI restriction digest analysis of pooled target vectors before (&#x2212;) and after (+) exposure to Dre recombination. Linearized pGB2-Dre can be seen in the (+) lanes (5.3 kb). Target vectors show the unrecombined restriction pattern (1.4 kb + 1.8 kb) before pGB2-Dre cotransformation (&#x2212; lanes). After cotransformation (+ lanes), various degrees of recombination (0.8 kb band) can be seen. (C) Densitometric analysis of recombination efficiency of libraries and wild type control. (D) Nucleotide distributions at positions 1&#x2013;8 in 82 individual p2NKanP minipreps sequenced after cotransformation with pGB2-Dre, and Kan selection (top) and 24 control p2NKanP minipreps from the unrecombined library (bottom). Original roxP sequence marked as wild type (WT) is indicated at the top. Positions 1, 4, 5, and 8 were not mutated in the rox2N spacer library. Numbers in the colored bars represent number of spacer sequences having the indicated nucleotide at the respective position. Nucleotide distributions were not significantly different between Dre-treated and untreated libraries at positions 3 (&#x3C7;2 = 0.73, P = 0.87) and 6 (&#x3C7;2 = 3.84, P = 0.28). However, after Dre selection, position 2 shows a highly significant bias toward G and C (&#x3C7;2 = 18.44, P = 0.000357), and position 7 a significant bias toward G (&#x3C7;2 = 9.99, P = 0.0186). (E) Most frequent spacers recovered from the screen. Nucleotides differing from consensus are shown in black. Occ. Nr. represents the number of times that specific sequence was observed. Clone number is the representative clone used for further studies." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-30435511" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Screen strategy and results for identification of mutant rox sites. (A) Strategy used for recombination test (for p4RKanP, p6RKanP, or p2NKanP libraries) and for library screening (p2NKanP library). For details, see also &lt;em&gt;Material and Methods&lt;/em&gt;. (B) &lt;em&gt;Pvu&lt;/em&gt;I + &lt;em&gt;Pst&lt;/em&gt;I restriction digest analysis of pooled target vectors before (&#x2212;) and after (+) exposure to Dre recombination. Linearized pGB2-Dre can be seen in the (+) lanes (5.3 kb). Target vectors show the unrecombined restriction pattern (1.4 kb + 1.8 kb) before pGB2-Dre cotransformation (&#x2212; lanes). After cotransformation (+ lanes), various degrees of recombination (0.8 kb band) can be seen. (C) Densitometric analysis of recombination efficiency of libraries and wild type control. (D) Nucleotide distributions at positions 1&#x2013;8 in 82 individual p2NKanP minipreps sequenced after cotransformation with pGB2-Dre, and Kan selection (top) and 24 control p2NKanP minipreps from the unrecombined library (bottom). Original roxP sequence marked as wild type (WT) is indicated at the top. Positions 1, 4, 5, and 8 were not mutated in the rox2N spacer library. Numbers in the colored bars represent number of spacer sequences having the indicated nucleotide at the respective position. Nucleotide distributions were not significantly different between Dre-treated and untreated libraries at positions 3 (&#x3C7;&lt;sup&gt;2&lt;/sup&gt; = 0.73, &lt;em&gt;P&lt;/em&gt; = 0.87) and 6 (&#x3C7;&lt;sup&gt;2&lt;/sup&gt; = 3.84, &lt;em&gt;P&lt;/em&gt; = 0.28). However, after Dre selection, position 2 shows a highly significant bias toward G and C (&#x3C7;&lt;sup&gt;2&lt;/sup&gt; = 18.44, &lt;em&gt;P&lt;/em&gt; = 0.000357), and position 7 a significant bias toward G (&#x3C7;&lt;sup&gt;2&lt;/sup&gt; = 9.99, &lt;em&gt;P&lt;/em&gt; = 0.0186). (E) Most frequent spacers recovered from the screen. Nucleotides differing from consensus are shown in black. Occ. Nr. represents the number of times that specific sequence was observed. Clone number is the representative clone used for further studies.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/559/F2.medium.gif" width="440" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/3/559/F2.medium.gif" width="440" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/559/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/559/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467836" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-9" class="first-child">Screen strategy and results for identification of mutant rox sites. (A) Strategy used for recombination test (for p4RKanP, p6RKanP, or p2NKanP libraries) and for library screening (p2NKanP library). For details, see also <em>Material and Methods</em>. (B) <em>Pvu</em>I + <em>Pst</em>I restriction digest analysis of pooled target vectors before (â) and after (+) exposure to Dre recombination. Linearized pGB2-Dre can be seen in the (+) lanes (5.3 kb). Target vectors show the unrecombined restriction pattern (1.4 kb + 1.8 kb) before pGB2-Dre cotransformation (â lanes). After cotransformation (+ lanes), various degrees of recombination (0.8 kb band) can be seen. (C) Densitometric analysis of recombination efficiency of libraries and wild type control. (D) Nucleotide distributions at positions 1â8 in 82 individual p2NKanP minipreps sequenced after cotransformation with pGB2-Dre, and Kan selection (top) and 24 control p2NKanP minipreps from the unrecombined library (bottom). Original roxP sequence marked as wild type (WT) is indicated at the top. Positions 1, 4, 5, and 8 were not mutated in the rox2N spacer library. Numbers in the colored bars represent number of spacer sequences having the indicated nucleotide at the respective position. Nucleotide distributions were not significantly different between Dre-treated and untreated libraries at positions 3 (Ï<sup>2</sup> = 0.73, <em>P</em> = 0.87) and 6 (Ï<sup>2</sup> = 3.84, <em>P</em> = 0.28). However, after Dre selection, position 2 shows a highly significant bias toward G and C (Ï<sup>2</sup> = 18.44, <em>P</em> = 0.000357), and position 7 a significant bias toward G (Ï<sup>2</sup> = 9.99, <em>P</em> = 0.0186). (E) Most frequent spacers recovered from the screen. Nucleotides differing from consensus are shown in black. Occ. Nr. represents the number of times that specific sequence was observed. Clone number is the representative clone used for further studies.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/559/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Identified rox mutants are indeed unable to recombine with the wild type. Seven of the identified mutant spacers (A) were tested for the ability to recombine with the wild type roxP site, by cotransformation with pGB2-Dre in DH5&#x3B1;. Transformations were plated on ampicillin+ spectinomycin+, and two minipreps for each clone were amplified and diagnosed by restriction digest with PvuI + PstI (B, C). In addition, both clones were retransformed into DH5&#x3B1; plated on ampicillin plates, and replica plated using a velvet replicator onto kanamycin plates (D). (A) Schematic of the generic recombination target construct (top), and naming convention for the individual mutant constructs (bottom table). The roxB site is always roxP. The roxA site is either a wild type rox site (pPKanP) or one of the seven mutant spacers (Figure 2E). (B) Digest with PvuI + PstI. The 5.3 kb band represents the linearized pGB2-Dre. Both isolates of all seven mutant clones failed to recombine (1.8 + 1.4 kb bands), while the wild type isolates fully recombined (1.4 + 0.8 kb fragments). (C) Densitometric analysis confirms minimal recombination for mutants 7&#x2013;9, and essentially no recombination for mutants 12&#x2013;85. (D) Replica plating experiments for one of the two isolates for each clone, as well as the wild type control. For each clone, the number is indicated in the bottom left corner. Top rows are initial ampicillin plates, and bottom rows replicates onto kanamycin plates. The ratio of colonies surviving on kanamycin vs. ampicillin plates is shown as percent in A, bottom table. Images of full plates are provided in Figure S2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-30435511" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Identified rox mutants are indeed unable to recombine with the wild type. Seven of the identified mutant spacers (A) were tested for the ability to recombine with the wild type roxP site, by cotransformation with pGB2-Dre in DH5&#x3B1;. Transformations were plated on ampicillin&lt;sup&gt;+&lt;/sup&gt; spectinomycin&lt;sup&gt;+&lt;/sup&gt;, and two minipreps for each clone were amplified and diagnosed by restriction digest with &lt;em&gt;Pvu&lt;/em&gt;I + &lt;em&gt;Pst&lt;/em&gt;I (B, C). In addition, both clones were retransformed into DH5&#x3B1; plated on ampicillin plates, and replica plated using a velvet replicator onto kanamycin plates (D). (A) Schematic of the generic recombination target construct (top), and naming convention for the individual mutant constructs (bottom table). The roxB site is always roxP. The roxA site is either a wild type rox site (pPKanP) or one of the seven mutant spacers (Figure 2E). (B) Digest with &lt;em&gt;Pvu&lt;/em&gt;I + &lt;em&gt;Pst&lt;/em&gt;I. The 5.3 kb band represents the linearized pGB2-Dre. Both isolates of all seven mutant clones failed to recombine (1.8 + 1.4 kb bands), while the wild type isolates fully recombined (1.4 + 0.8 kb fragments). (C) Densitometric analysis confirms minimal recombination for mutants 7&#x2013;9, and essentially no recombination for mutants 12&#x2013;85. (D) Replica plating experiments for one of the two isolates for each clone, as well as the wild type control. For each clone, the number is indicated in the bottom left corner. Top rows are initial ampicillin plates, and bottom rows replicates onto kanamycin plates. The ratio of colonies surviving on kanamycin &lt;em&gt;vs.&lt;/em&gt; ampicillin plates is shown as percent in A, bottom table. Images of full plates are provided in Figure S2.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/559/F3.medium.gif" width="440" height="356"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/3/559/F3.medium.gif" width="440" height="356"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/559/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/559/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467754" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-10" class="first-child">Identified rox mutants are indeed unable to recombine with the wild type. Seven of the identified mutant spacers (A) were tested for the ability to recombine with the wild type roxP site, by cotransformation with pGB2-Dre in DH5Î±. Transformations were plated on ampicillin<sup>+</sup> spectinomycin<sup>+</sup>, and two minipreps for each clone were amplified and diagnosed by restriction digest with <em>Pvu</em>I + <em>Pst</em>I (B, C). In addition, both clones were retransformed into DH5Î± plated on ampicillin plates, and replica plated using a velvet replicator onto kanamycin plates (D). (A) Schematic of the generic recombination target construct (top), and naming convention for the individual mutant constructs (bottom table). The roxB site is always roxP. The roxA site is either a wild type rox site (pPKanP) or one of the seven mutant spacers (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2E</a>). (B) Digest with <em>Pvu</em>I + <em>Pst</em>I. The 5.3 kb band represents the linearized pGB2-Dre. Both isolates of all seven mutant clones failed to recombine (1.8 + 1.4 kb bands), while the wild type isolates fully recombined (1.4 + 0.8 kb fragments). (C) Densitometric analysis confirms minimal recombination for mutants 7â9, and essentially no recombination for mutants 12â85. (D) Replica plating experiments for one of the two isolates for each clone, as well as the wild type control. For each clone, the number is indicated in the bottom left corner. Top rows are initial ampicillin plates, and bottom rows replicates onto kanamycin plates. The ratio of colonies surviving on kanamycin <em>vs.</em> ampicillin plates is shown as percent in A, bottom table. Images of full plates are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025841/-/DC1/FigureS2.pdf">Figure S2</a>.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/559/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Rox mutants rox7, rox8, rox12, rox61, and rox85 recombine with themselves in the presence of Dre but not Cre recombinase. (A) Rox sites with spacers identical to clones 7, 8, 12, 61, and 85 (Figure 2E) were cloned at both the roxA and roxB positions in direct orientation, generating the p7Kan7, p8Kan8, p12Kan12, p61Kan61, and p85Kan85 constructs. Each construct, as well as the pPKanP positive control, were then exposed to Dre recombinase by the same cotransformation protocol outlined in Figure 3. Two separate isolates for each clone were then tested by PvuI + PstI restriction digestion (B, C) and replica plating (Figure S3, A, B, C, and F). All rox sites tested showed complete recombination (1.4 + 0.8 kb fragments) when exposed to Dre recombinase, and essentially no surviving colonies when replica plated from ampicillin to kanamycin plates (Figure S3A, and table in A, right hand column). Quantitations of B are shown in C. To test the sensitivity of the wild type roxP and mutants to Cre recombination, pPKanP and p7Kan7, p8Kan8, p12Kan12, p61Kan61, and p85Kan85 mutants were transformed in the BS1365 strain, constitutively expressing Cre. Transformations were plated on kanamycin and ampicillin, and two individual colonies from each clone were amplified and analyzed by PvuI + PstI restriction digestion (D, E). In addition, DNA from the two colonies was retransformed in DH5&#x3B1;, plated on ampicillin, and colonies replica plated onto kanamycin plates using a velvet replicator (Figure S3B). All tested constructs show the 1.8 and 1.4 kb fragments characteristic for lack of recombination (D) and resistance to kanamycin in the replica platting assay (Figure S3B and table in A, right hand column). Note that the BS1365 strain carries the Cre recombinase on a F&#x2019; element that also contains a kanamycin resistance gene. Therefore, kanamycin does not select against rox target vector recombinants in the initial BS1365 cotransformation step." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-30435511" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Rox mutants rox7, rox8, rox12, rox61, and rox85 recombine with themselves in the presence of Dre but not Cre recombinase. (A) Rox sites with spacers identical to clones 7, 8, 12, 61, and 85 (Figure 2E) were cloned at both the roxA and roxB positions in direct orientation, generating the p7Kan7, p8Kan8, p12Kan12, p61Kan61, and p85Kan85 constructs. Each construct, as well as the pPKanP positive control, were then exposed to Dre recombinase by the same cotransformation protocol outlined in Figure 3. Two separate isolates for each clone were then tested by &lt;em&gt;Pvu&lt;/em&gt;I + &lt;em&gt;Pst&lt;/em&gt;I restriction digestion (B, C) and replica plating (Figure S3, A, B, C, and F). All rox sites tested showed complete recombination (1.4 + 0.8 kb fragments) when exposed to Dre recombinase, and essentially no surviving colonies when replica plated from ampicillin to kanamycin plates (Figure S3A, and table in A, right hand column). Quantitations of B are shown in C. To test the sensitivity of the wild type roxP and mutants to Cre recombination, pPKanP and p7Kan7, p8Kan8, p12Kan12, p61Kan61, and p85Kan85 mutants were transformed in the BS1365 strain, constitutively expressing Cre. Transformations were plated on kanamycin and ampicillin, and two individual colonies from each clone were amplified and analyzed by &lt;em&gt;Pvu&lt;/em&gt;I + &lt;em&gt;Pst&lt;/em&gt;I restriction digestion (D, E). In addition, DNA from the two colonies was retransformed in DH5&#x3B1;, plated on ampicillin, and colonies replica plated onto kanamycin plates using a velvet replicator (Figure S3B). All tested constructs show the 1.8 and 1.4 kb fragments characteristic for lack of recombination (D) and resistance to kanamycin in the replica platting assay (Figure S3B and table in A, right hand column). Note that the BS1365 strain carries the Cre recombinase on a F&#x2019; element that also contains a kanamycin resistance gene. Therefore, kanamycin does not select against rox target vector recombinants in the initial BS1365 cotransformation step.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/559/F4.medium.gif" width="440" height="408"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/3/559/F4.medium.gif" width="440" height="408"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/559/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/559/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467778" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-11" class="first-child">Rox mutants rox7, rox8, rox12, rox61, and rox85 recombine with themselves in the presence of Dre but not Cre recombinase. (A) Rox sites with spacers identical to clones 7, 8, 12, 61, and 85 (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2E</a>) were cloned at both the roxA and roxB positions in direct orientation, generating the p7Kan7, p8Kan8, p12Kan12, p61Kan61, and p85Kan85 constructs. Each construct, as well as the pPKanP positive control, were then exposed to Dre recombinase by the same cotransformation protocol outlined in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>. Two separate isolates for each clone were then tested by <em>Pvu</em>I + <em>Pst</em>I restriction digestion (B, C) and replica plating (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025841/-/DC1/FigureS3.pdf">Figure S3, A, B, C, and F</a>). All rox sites tested showed complete recombination (1.4 + 0.8 kb fragments) when exposed to Dre recombinase, and essentially no surviving colonies when replica plated from ampicillin to kanamycin plates (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025841/-/DC1/FigureS3.pdf">Figure S3A</a>, and table in A, right hand column). Quantitations of B are shown in C. To test the sensitivity of the wild type roxP and mutants to Cre recombination, pPKanP and p7Kan7, p8Kan8, p12Kan12, p61Kan61, and p85Kan85 mutants were transformed in the BS1365 strain, constitutively expressing Cre. Transformations were plated on kanamycin and ampicillin, and two individual colonies from each clone were amplified and analyzed by <em>Pvu</em>I + <em>Pst</em>I restriction digestion (D, E). In addition, DNA from the two colonies was retransformed in DH5Î±, plated on ampicillin, and colonies replica plated onto kanamycin plates using a velvet replicator (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025841/-/DC1/FigureS3.pdf">Figure S3B</a>). All tested constructs show the 1.8 and 1.4 kb fragments characteristic for lack of recombination (D) and resistance to kanamycin in the replica platting assay (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025841/-/DC1/FigureS3.pdf">Figure S3B</a> and table in A, right hand column). Note that the BS1365 strain carries the Cre recombinase on a Fâ element that also contains a kanamycin resistance gene. Therefore, kanamycin does not select against rox target vector recombinants in the initial BS1365 cotransformation step.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/559/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Self-recombination efficiencies of roxP and novel sites tested in purified system. (A) Diagram of in vitro recombination reaction. proxA-Kanr-roxB DNA is exposed to affinity-purified Dre, resulting in a recombined plasmid and a circle. Digestion with PvuI + PstI + MscI results in a common PstI-PvuI 1458 bp fragment spanning the pMB1 origin and part of the Ampr and two alternative, recombination-dependent sets of fragments. Without recombination, a MscI&#x2013;PstI 713 bp fragment spanning the roxB site and a PvuI&#x2013;MscI 1061 bp fragment spanning the roxA can be detected (see for instance the 0 min timepoint in B). After recombination, a PstI&#x2013;PvuI fragment of 789 bp spanning the roxX site is observed in the remaining vector, and the 985 recombination circle is linearized by MscI. Without MscI digestion, the recombination circle migrates at about 600 bp (not shown). (B) Representative gels for the 0, 20, and 120 min time points reveal gradually accumulating recombination products. Lanes represent either marker (MW), or the prox-Kanr-rox vectors carrying either wild type (P) or spacers 7, 8, 12, 61, and 85 at both roxA and roxB sites (Figure 4A). Note that samples for the 20 and 120 min time points are taken from the same gel. (C) Densitometric analysis revealing percent recombination over the 120 min timecourse for wild type and all five mutant spacers. Solid lines represent medians, and dotted lines 25th and 75th quartiles for each set of samples (n = 6 data points/spacer and time point). (D) Box plots for the 60 min time point. Recombination efficiencies for each mutant spacer are shown as ratios to the wild type (P) from the same experiment (n = 10 data points/spacer). (Kolmogorov&#x2013;Smirnov two sided test, significance levels ** P &lt; 0.005, *** P &lt; 0.001.)" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-30435511" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Self-recombination efficiencies of roxP and novel sites tested in purified system. (A) Diagram of &lt;em&gt;in vitro&lt;/em&gt; recombination reaction. proxA-Kan&lt;sup&gt;r&lt;/sup&gt;-roxB DNA is exposed to affinity-purified Dre, resulting in a recombined plasmid and a circle. Digestion with &lt;em&gt;Pvu&lt;/em&gt;I + &lt;em&gt;Pst&lt;/em&gt;I + &lt;em&gt;Msc&lt;/em&gt;I results in a common &lt;em&gt;Pst&lt;/em&gt;I-&lt;em&gt;Pvu&lt;/em&gt;I 1458 bp fragment spanning the pMB1 origin and part of the Amp&lt;sup&gt;r&lt;/sup&gt; and two alternative, recombination-dependent sets of fragments. Without recombination, a &lt;em&gt;Msc&lt;/em&gt;I&#x2013;&lt;em&gt;Pst&lt;/em&gt;I 713 bp fragment spanning the roxB site and a &lt;em&gt;Pvu&lt;/em&gt;I&#x2013;&lt;em&gt;Msc&lt;/em&gt;I 1061 bp fragment spanning the roxA can be detected (see for instance the 0 min timepoint in B). After recombination, a &lt;em&gt;Pst&lt;/em&gt;I&#x2013;&lt;em&gt;Pvu&lt;/em&gt;I fragment of 789 bp spanning the roxX site is observed in the remaining vector, and the 985 recombination circle is linearized by &lt;em&gt;Msc&lt;/em&gt;I. Without &lt;em&gt;Msc&lt;/em&gt;I digestion, the recombination circle migrates at about 600 bp (not shown). (B) Representative gels for the 0, 20, and 120 min time points reveal gradually accumulating recombination products. Lanes represent either marker (MW), or the prox-Kan&lt;sup&gt;r&lt;/sup&gt;-rox vectors carrying either wild type (P) or spacers 7, 8, 12, 61, and 85 at both roxA and roxB sites (Figure 4A). Note that samples for the 20 and 120 min time points are taken from the same gel. (C) Densitometric analysis revealing percent recombination over the 120 min timecourse for wild type and all five mutant spacers. Solid lines represent medians, and dotted lines 25&lt;sup&gt;th&lt;/sup&gt; and 75&lt;sup&gt;th&lt;/sup&gt; quartiles for each set of samples (n = 6 data points/spacer and time point). (D) Box plots for the 60 min time point. Recombination efficiencies for each mutant spacer are shown as ratios to the wild type (P) from the same experiment (n = 10 data points/spacer). (Kolmogorov&#x2013;Smirnov two sided test, significance levels ** &lt;em&gt;P&lt;/em&gt; &lt; 0.005, *** &lt;em&gt;P&lt;/em&gt; &lt; 0.001.)&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/559/F5.medium.gif" width="395" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/3/559/F5.medium.gif" width="395" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/559/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/559/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467719" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-12" class="first-child">Self-recombination efficiencies of roxP and novel sites tested in purified system. (A) Diagram of <em>in vitro</em> recombination reaction. proxA-Kan<sup>r</sup>-roxB DNA is exposed to affinity-purified Dre, resulting in a recombined plasmid and a circle. Digestion with <em>Pvu</em>I + <em>Pst</em>I + <em>Msc</em>I results in a common <em>Pst</em>I-<em>Pvu</em>I 1458 bp fragment spanning the pMB1 origin and part of the Amp<sup>r</sup> and two alternative, recombination-dependent sets of fragments. Without recombination, a <em>Msc</em>Iâ<em>Pst</em>I 713 bp fragment spanning the roxB site and a <em>Pvu</em>Iâ<em>Msc</em>I 1061 bp fragment spanning the roxA can be detected (see for instance the 0 min timepoint in B). After recombination, a <em>Pst</em>Iâ<em>Pvu</em>I fragment of 789 bp spanning the roxX site is observed in the remaining vector, and the 985 recombination circle is linearized by <em>Msc</em>I. Without <em>Msc</em>I digestion, the recombination circle migrates at about 600 bp (not shown). (B) Representative gels for the 0, 20, and 120 min time points reveal gradually accumulating recombination products. Lanes represent either marker (MW), or the prox-Kan<sup>r</sup>-rox vectors carrying either wild type (P) or spacers 7, 8, 12, 61, and 85 at both roxA and roxB sites (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>). Note that samples for the 20 and 120 min time points are taken from the same gel. (C) Densitometric analysis revealing percent recombination over the 120 min timecourse for wild type and all five mutant spacers. Solid lines represent medians, and dotted lines 25<sup>th</sup> and 75<sup>th</sup> quartiles for each set of samples (n = 6 data points/spacer and time point). (D) Box plots for the 60 min time point. Recombination efficiencies for each mutant spacer are shown as ratios to the wild type (P) from the same experiment (n = 10 data points/spacer). (KolmogorovâSmirnov two sided test, significance levels ** <em>P</em> &lt; 0.005, *** <em>P</em> &lt; 0.001.)</p><div class="sb-div caption-clear"></div></div></div><p id="p-13">FREX cassette vector (pAAVFREX-eGFp-mCherry): Synthetic oligos containing the rox12 and roxP WT arranged in tandem in direct and reverse orientation surrounding a multicloning site were assembled in pBSKS (pBSKS-FREX). The FREX cassette was then cloned using a <em>Bam</em>HI to <em>Eco</em>RI into an AAV expression vector containing AAV2 ITRs, a ubiquitously expressing chicken Î²-actin enhancer CMV promoter fusion (CAG) (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Atasoy <em>et al.</em> 2008</a>), followed by a WPRE (a posttranscriptional regulatory element derived from a woodchuck hepatitis virus (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Zufferey <em>et al.</em> 1999</a>) and bovine Growth Hormone polyadenylation signal, resulting in the pAAVCAGFREX vector. An eGFP cDNA in direct orientation, and a mCherry-SV40 polyA transcription STOP cassette in reverse orientation relative to the promoter, were cloned into the multicloning site (pAAV-FREX-eGFP-mCherry, <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). The SV40 polyA sequence contains transcriptional stop and polyadenylation signals in both orientations, and thus will also serve as a transcriptional terminator after inversion. The control vector pAAVPTPY contains a PSD95-TFP-P2A-GAP43-eYFP expression cassette cloned in inverted orientation in the pAAVFLEX vector containing inverted tandem copies of loxP and lox2272 (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Atasoy <em>et al.</em> 2008</a>) (B. Wu and T. C. Badea, unpublished results).</p></div><div id="sec-3" class="subsection"><h3>Bacterial recombination assay and library screening</h3><p id="p-14">Bacterial strains (<em>Escherichia coli</em>) were DH5Î± for Dre recombination tests and cloning, and BS1365 (FâKan, recA1, endA1, Î» imm434 x1-Cre), a generous gift from Dr. Brian Sauer, for testing CreâroxP interactions. For the <em>Library Dre recombination test</em>, pGB2-Dre (Spc<sup>r</sup>) was cotransformed into DH5Î± bacteria with either the WT control plasmid pPKanP or one of the proxA-Kan<sup>r</sup>-roxB (Kan<sup>r</sup>, Amp<sup>r</sup>) target libraries, and transformants were plated onto spectinomycin<sup>+</sup> ampicillin<sup>+</sup> selection plates. Colonies from 10â20 plates were scraped off the plates, allowed to grow under selection, and then submitted to large scale DNA extraction. An aliquot of the resulting DNA was digested with <em>Pvu</em>I + <em>Pst</em>I. <em>Library screening</em> for the rox2N library was performed essentially as described for the recombination assay, except that the cotransformation was plated on spectinomycin + kanamycin, and pooled colonies were further amplified on kanamycin, to ensure stringent selection for unrecombined target plasmids. Preliminary tests showed that Spc<sup>r</sup> does not convey kanamycin resistance nor does Kan<sup>r</sup> spectinomycin resistance (data not shown). About 50 ng of plasmid DNA was then retransformed in DH5Î± and selected on kanamycin, and plasmid DNA was isolated from individual colonies and tested for recombination by restriction digest with <em>Pvu</em>I + <em>Pst</em>I. Mutated rox sites were sequenced for 92 colonies. Ten of the sequenced clones had missing or mutated direct inverted repeats, outside the spacer region, and were discarded from further analysis. For recombination assays for clone confirmation and testing mutant self-recombination, the target plasmids were cotransformed with pGB2-Dre onto spectinomycin<sup>+</sup> ampicillin<sup>+</sup> selection. For each target, two isolated colonies were miniprepped, and DNA digested with <em>Pvu</em>I and <em>Pst</em>I as above, or retransformed on ampicillin plates. Colonies from the ampicillin plates were replicated onto kanamycin plates using sterilized velvet replicators.</p></div><div id="sec-4" class="subsection"><h3>Recombination in purified system</h3><p id="p-15">The DNA substrates were the prox-Kan<sup>r</sup>-rox vectors series used for checking homotypic recognition, p7Kan7, p8Kan8, p12Kan12, p61Kan61, and p85Kan85, along with the pPKanP control vector. The Dre protein was produced as follows. The Dre sequence was subcloned into the pET-28a vector, which carries an N-terminal histidine tag. Cultures growing in log phase were subjected to IPTG induction for 2 hr before cells were harvested. The pellet was resuspended in 1 mM PMSF in PBS and sonicated for 12 cycles of 10 sec on, 5 sec off at 30 W. Expression of the His-tagged Dre protein (MW = 42,343) was tested by colloidal blue (Simply Blue, Invitrogen) staining of SDS-PAGE. The Dre protein was purified using a HiTrap Nickel column, followed by dialysis against storage buffer overnight at 4Â°. Dre protein yield and purity were determined by quantifying the total protein concentration using a BCA Assay (Pierce BCA protein assay) combined with densitometric evaluation of SDS-PAGE gels stained with coloidal blue. The purified protein was stored at â80Â° in 10 mM Tris-HCl, 150 mM NaCl, and 0.5 mM EDTA. For the recombination reaction, the protein was kept in storage buffer containing 40% glycerol at â20Â°. The buffer optimized for the recombination reaction contained: 50 mM Tris, 100 Î¼g/ml BSA, 10 mM MgCl, 50 mM NaCl, and 1 mM DTT. Vectors were linearized with <em>Pst</em>I for 1 hr at 37Â°, followed by heat inactivation (20 min at 70Â°), and then DNA (â¼0.25 mmol) was incubated with Dre protein (â¼2 mmol) at 37Â° and the recombination reactions were terminated at given time points by heat-inactivation of the Dre protein at 70Â°. The DNA was then further digested with <em>Pvu</em>I and <em>Msc</em>I to generate diagnostic fragments allowing the differentiation of unrecombined from recombined products. Diagnostic digests were then resolved on a 1.5% agarose gel stained with ethidium bromide, and band densitometric analysis was performed using ImageJ (Wayne Rasband, NIH, <a href="http://imagej.nih.gov/ij">http://imagej.nih.gov/ij</a>).</p></div><div id="sec-5" class="subsection"><h3>HEK293 recombination assay</h3><p id="p-16">HEK293 lines for eukaryotic recombination tests were: a previously generated HEK293-Cre stable transfected line (neomycin resistant â Neo<sup>r</sup>) and a newly generated HEK293-Dre (hygromycin resistant â Hygro<sup>r</sup>) line, built by Flp-mediated insertion of a pcDNA5/FRT-Dre expression vector into the HEK293-FlpIn cell line, using manufacturer recommended procedures (Flp-In system, Invitrogen). In this strategy, the Dre cDNA is inserted as a single copy into the HEK293 genome, downstream of a CMV promoter, together with the Hygro<sup>r</sup> cassette. Several colonies were selected under hygromycin, and tested for Dre recombination with the WT proxP-R-roxP-G vector described above. Clone 4 was used in the studies described below. The eukaryotic expression vectors containing combinations of mutant and WT roxP sites were transfected onto duplicate poly-D-lysine (Sigma-Aldrich, P6407, 0.5 Âµg/Âµl in water) coated coverslips of either HEK293-Dre (clone 4) or HEK293-Cre cells, using a branched polyethylenimine transfection reagent (Sigma-Aldrich, cat # 408727, 1 Âµg/Âµl) (generated and optimized by Jung-Woong Kim, N-NRL/NEI). After 48 hr, cells were fixed for 10 min in 4% paraformaldehyde, counterstained with DAPI, and mounted in Fluoromount-G (Southern Biotech) onto slides. Cells were imaged with a 20 Ã objective using a Zeiss Axioimager.Z2 fitted with an Apotome, and a MRM Axiocam. For each construct and HEK293 line, six 0.15 mm<sup>2</sup> (full camera field) images were taken using the same set of fluorescence intensities and exposure time settings across the entire experiment. All images were collected with identical settings for fluorescence intensity and exposure times. However, red fluorescence signals in <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6 B</a> columns 1, 2, and 5, were much dimmer then in all other conditions, so the maximum value of the red lookup table was adjusted to one-fifth of its initial value for these panels. Images were imported and analyzed in ImageJ, using the LOCI bioformats plugin (Open Microscopy Environment, <a href="http://www.openmicroscopy.org/site/products/bio-formats">http://www.openmicroscopy.org/site/products/bio-formats</a>), and red, green, and yellow (double positive) cells were counted using the cell counter plugin (Kurt De Vos, University of Sheffield, Academic Neurology, kurt.devos@iop.kcl.ac.uk) over six individual 20 Ã fields derived from two distinct transfected coverslips for each of the experimental conditions in <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6, B and C</a>, and <a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7, C and D</a>. For prox-R-rox G experiments, green and yellow (double positive) cells were considered as recombined and normalized to the total number of transfected cells (sum of red, green, and yellow cells).</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/559/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Rox mutants rox12 and rox85 recombine with themselves but not with roxP in eukaryotic cells. (A) In the prox-R-rox-G eukaryotic expression vector, a CMV promoter is driving transcription of mCherry before Dre recombination between the roxA and roxB sites, and of eGFP after recombination. rox12 and rox85 mutants or roxP wild type control were placed at both roxA and roxB sites to generate prox12-R-rox12-G, prox85-R-roxP-85 and proxP-R-roxP-G vectors, suitable for testing the ability of each rox site to recombine with itself. In addition, rox12 or rox85 mutants were cloned at the roxB site in combination with a roxP at the roxA site, to generate proxP-R-rox12-G and proxP-R-rox85-G, to test the compatibility between rox mutants and roxP wild type. (B, C) HEK293 cells stably expressing either the Dre (B) or Cre (C) recombinase were transfected with the constructs described in A. (D, E) Numbers in pie charts indicate the sums for red (mCherry only), green (eGFP only), and yellow (double positive) cells over six individual 20 &#xD7; fields derived from two distinct transfected coverslips for each of the experimental conditions in B and C. No eGFP positive cells were seen in HEK293Dre cells transfected with proxP-R-rox12-G and proxP-R-rox85-G (B, D, columns 3 and 5) or HEK293Cre cells transfected with either one of the five constructs (C, E). (F) Box-whisker plots quantitating the percent of recombination positive cells in the HEK293Dre experiments. The percentage represents 100 &#xD7; (eGFP+ + eGFP+mCherry+)/(eGFP+ + eGFP+mCherry+ + mCherry+). Statistical significance was determined by student t-test and Kolmogorov&#x2013;Smirnov test (n.s. = P &gt; 0.05, * = P &lt; 0.05, ** = P &lt; 0.005). Scale bar in C = 50 &#xB5;m." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-30435511" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Rox mutants rox12 and rox85 recombine with themselves but not with roxP in eukaryotic cells. (A) In the prox-R-rox-G eukaryotic expression vector, a CMV promoter is driving transcription of mCherry before Dre recombination between the roxA and roxB sites, and of eGFP after recombination. rox12 and rox85 mutants or roxP wild type control were placed at both roxA and roxB sites to generate prox12-R-rox12-G, prox85-R-roxP-85 and proxP-R-roxP-G vectors, suitable for testing the ability of each rox site to recombine with itself. In addition, rox12 or rox85 mutants were cloned at the roxB site in combination with a roxP at the roxA site, to generate proxP-R-rox12-G and proxP-R-rox85-G, to test the compatibility between rox mutants and roxP wild type. (B, C) HEK293 cells stably expressing either the Dre (B) or Cre (C) recombinase were transfected with the constructs described in A. (D, E) Numbers in pie charts indicate the sums for red (mCherry only), green (eGFP only), and yellow (double positive) cells over six individual 20 &#xD7; fields derived from two distinct transfected coverslips for each of the experimental conditions in B and C. No eGFP positive cells were seen in HEK293Dre cells transfected with proxP-R-rox12-G and proxP-R-rox85-G (B, D, columns 3 and 5) or HEK293Cre cells transfected with either one of the five constructs (C, E). (F) Box-whisker plots quantitating the percent of recombination positive cells in the HEK293Dre experiments. The percentage represents 100 &#xD7; (eGFP&lt;sup&gt;+&lt;/sup&gt; + eGFP&lt;sup&gt;+&lt;/sup&gt;mCherry&lt;sup&gt;+&lt;/sup&gt;)/(eGFP&lt;sup&gt;+&lt;/sup&gt; + eGFP&lt;sup&gt;+&lt;/sup&gt;mCherry&lt;sup&gt;+&lt;/sup&gt; + mCherry&lt;sup&gt;+&lt;/sup&gt;). Statistical significance was determined by student &lt;em&gt;t&lt;/em&gt;-test and Kolmogorov&#x2013;Smirnov test (n.s. = &lt;em&gt;P&lt;/em&gt; &gt; 0.05, * = &lt;em&gt;P&lt;/em&gt; &lt; 0.05, ** = &lt;em&gt;P&lt;/em&gt; &lt; 0.005). Scale bar in C = 50 &#xB5;m.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/559/F6.medium.gif" width="292" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/3/559/F6.medium.gif" width="292" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/559/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/559/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467847" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-17" class="first-child">Rox mutants rox12 and rox85 recombine with themselves but not with roxP in eukaryotic cells. (A) In the prox-R-rox-G eukaryotic expression vector, a CMV promoter is driving transcription of mCherry before Dre recombination between the roxA and roxB sites, and of eGFP after recombination. rox12 and rox85 mutants or roxP wild type control were placed at both roxA and roxB sites to generate prox12-R-rox12-G, prox85-R-roxP-85 and proxP-R-roxP-G vectors, suitable for testing the ability of each rox site to recombine with itself. In addition, rox12 or rox85 mutants were cloned at the roxB site in combination with a roxP at the roxA site, to generate proxP-R-rox12-G and proxP-R-rox85-G, to test the compatibility between rox mutants and roxP wild type. (B, C) HEK293 cells stably expressing either the Dre (B) or Cre (C) recombinase were transfected with the constructs described in A. (D, E) Numbers in pie charts indicate the sums for red (mCherry only), green (eGFP only), and yellow (double positive) cells over six individual 20 Ã fields derived from two distinct transfected coverslips for each of the experimental conditions in B and C. No eGFP positive cells were seen in HEK293Dre cells transfected with proxP-R-rox12-G and proxP-R-rox85-G (B, D, columns 3 and 5) or HEK293Cre cells transfected with either one of the five constructs (C, E). (F) Box-whisker plots quantitating the percent of recombination positive cells in the HEK293Dre experiments. The percentage represents 100 Ã (eGFP<sup>+</sup> + eGFP<sup>+</sup>mCherry<sup>+</sup>)/(eGFP<sup>+</sup> + eGFP<sup>+</sup>mCherry<sup>+</sup> + mCherry<sup>+</sup>). Statistical significance was determined by student <em>t</em>-test and KolmogorovâSmirnov test (n.s. = <em>P</em> &gt; 0.05, * = <em>P</em> &lt; 0.05, ** = <em>P</em> &lt; 0.005). Scale bar in C = 50 Âµm.</p><div class="sb-div caption-clear"></div></div></div><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/559/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Efficient Dre recombination using an inversion&#x2013;excision cassette (FREX) based on rox12 and roxP. (A) The pAAV-FREX-eGFP-mCherry eukaryotic viral expression vector contains a CAG promoter driving an expression cassette flanked by tandem rox12-roxP sites. The cassette contains eGFP and mCherry cDNAs placed in reverse orientation to each other, and separated by a triple repeat of a bidirectional SV40 transcription stop/polyadenylation signal. WPRE and bGH elements ensure mRNA stability and processing and AAV2 ITRs (not shown) allow for viral packaging. (B) Dre induces two alternative inversion reactions (double arrows) followed by irreversible excision (single arrow) reactions. The CAG promoter drives transcription of eGFP before, and of mCherry after, Dre recombination. HEK293-Dre (C) or HEK293-Cre (D) cells were transfected with FREX (column 1), prox-R-rox-G (column 2), or pAAVPTPY, a FLEX-based Cre reporter plasmid (column 3). (E, F) Numbers in pie charts for columns 1 and 2 indicate the sums for red (mCherry only), green (eGFP only), and yellow (double positive) cells. Results from cells transfected with pAAVPTPY (cyan) are shown in column 3 (E, F), comparing recombined cells to the total number of cells (DAPI, blue). Only Hek293Cre cells recombined the PTPY construct. (G) Box-whisker plots quantitating the percent of recombination positive cells. The percentage represents 100 &#xD7; (mCherry+ + eGFP+mCherry+)/(eGFP+ + eGFP+mCherry+ + mCherry+) for FREX transfections or 100 &#xD7; (eGFP+ + eGFP+mCherry+)/(eGFP+ + eGFP+mCherry+ + mCherry+) for pRoxP-R-roxP-G transfections. Recombination of PTPY is represented by 100 &#xD7; Cyan+/DAPI+. Recombination efficiency was not statistically significant between the FREX and pRoxP-R-roxP-G in Dre-expressing cells. Statistical significance was determined with student t-test and Kolmogorov&#x2013;Smirnov test (n.s. = P &gt; 0.05, ** = P &lt; 0.005). Scale bar in C = 50 &#x3BC;m." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-30435511" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Efficient Dre recombination using an inversion&#x2013;excision cassette (FREX) based on rox12 and roxP. (A) The pAAV-FREX-eGFP-mCherry eukaryotic viral expression vector contains a CAG promoter driving an expression cassette flanked by tandem rox12-roxP sites. The cassette contains eGFP and mCherry cDNAs placed in reverse orientation to each other, and separated by a triple repeat of a bidirectional SV40 transcription stop/polyadenylation signal. WPRE and bGH elements ensure mRNA stability and processing and AAV2 ITRs (not shown) allow for viral packaging. (B) Dre induces two alternative inversion reactions (double arrows) followed by irreversible excision (single arrow) reactions. The CAG promoter drives transcription of eGFP before, and of mCherry after, Dre recombination. HEK293-Dre (C) or HEK293-Cre (D) cells were transfected with FREX (column 1), prox-R-rox-G (column 2), or pAAVPTPY, a FLEX-based Cre reporter plasmid (column 3). (E, F) Numbers in pie charts for columns 1 and 2 indicate the sums for red (mCherry only), green (eGFP only), and yellow (double positive) cells. Results from cells transfected with pAAVPTPY (cyan) are shown in column 3 (E, F), comparing recombined cells to the total number of cells (DAPI, blue). Only Hek293Cre cells recombined the PTPY construct. (G) Box-whisker plots quantitating the percent of recombination positive cells. The percentage represents 100 &#xD7; (mCherry&lt;sup&gt;+&lt;/sup&gt; + eGFP&lt;sup&gt;+&lt;/sup&gt;mCherry&lt;sup&gt;+&lt;/sup&gt;)/(eGFP&lt;sup&gt;+&lt;/sup&gt; + eGFP&lt;sup&gt;+&lt;/sup&gt;mCherry&lt;sup&gt;+&lt;/sup&gt; + mCherry&lt;sup&gt;+&lt;/sup&gt;) for FREX transfections or 100 &#xD7; (eGFP&lt;sup&gt;+&lt;/sup&gt; + eGFP&lt;sup&gt;+&lt;/sup&gt;mCherry&lt;sup&gt;+&lt;/sup&gt;)/(eGFP&lt;sup&gt;+&lt;/sup&gt; + eGFP&lt;sup&gt;+&lt;/sup&gt;mCherry&lt;sup&gt;+&lt;/sup&gt; + mCherry&lt;sup&gt;+&lt;/sup&gt;) for pRoxP-R-roxP-G transfections. Recombination of PTPY is represented by 100 &#xD7; Cyan&lt;sup&gt;+&lt;/sup&gt;/DAPI&lt;sup&gt;+&lt;/sup&gt;. Recombination efficiency was not statistically significant between the FREX and pRoxP-R-roxP-G in Dre-expressing cells. Statistical significance was determined with student &lt;em&gt;t&lt;/em&gt;-test and Kolmogorov&#x2013;Smirnov test (n.s. = &lt;em&gt;P&lt;/em&gt; &gt; 0.05, ** = &lt;em&gt;P&lt;/em&gt; &lt; 0.005). Scale bar in C = 50 &#x3BC;m.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/559/F7.medium.gif" width="440" height="343"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/6/3/559/F7.medium.gif" width="440" height="343"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/559/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/559/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467741" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-18" class="first-child">Efficient Dre recombination using an inversionâexcision cassette (FREX) based on rox12 and roxP. (A) The pAAV-FREX-eGFP-mCherry eukaryotic viral expression vector contains a CAG promoter driving an expression cassette flanked by tandem rox12-roxP sites. The cassette contains eGFP and mCherry cDNAs placed in reverse orientation to each other, and separated by a triple repeat of a bidirectional SV40 transcription stop/polyadenylation signal. WPRE and bGH elements ensure mRNA stability and processing and AAV2 ITRs (not shown) allow for viral packaging. (B) Dre induces two alternative inversion reactions (double arrows) followed by irreversible excision (single arrow) reactions. The CAG promoter drives transcription of eGFP before, and of mCherry after, Dre recombination. HEK293-Dre (C) or HEK293-Cre (D) cells were transfected with FREX (column 1), prox-R-rox-G (column 2), or pAAVPTPY, a FLEX-based Cre reporter plasmid (column 3). (E, F) Numbers in pie charts for columns 1 and 2 indicate the sums for red (mCherry only), green (eGFP only), and yellow (double positive) cells. Results from cells transfected with pAAVPTPY (cyan) are shown in column 3 (E, F), comparing recombined cells to the total number of cells (DAPI, blue). Only Hek293Cre cells recombined the PTPY construct. (G) Box-whisker plots quantitating the percent of recombination positive cells. The percentage represents 100 Ã (mCherry<sup>+</sup> + eGFP<sup>+</sup>mCherry<sup>+</sup>)/(eGFP<sup>+</sup> + eGFP<sup>+</sup>mCherry<sup>+</sup> + mCherry<sup>+</sup>) for FREX transfections or 100 Ã (eGFP<sup>+</sup> + eGFP<sup>+</sup>mCherry<sup>+</sup>)/(eGFP<sup>+</sup> + eGFP<sup>+</sup>mCherry<sup>+</sup> + mCherry<sup>+</sup>) for pRoxP-R-roxP-G transfections. Recombination of PTPY is represented by 100 Ã Cyan<sup>+</sup>/DAPI<sup>+</sup>. Recombination efficiency was not statistically significant between the FREX and pRoxP-R-roxP-G in Dre-expressing cells. Statistical significance was determined with student <em>t</em>-test and KolmogorovâSmirnov test (n.s. = <em>P</em> &gt; 0.05, ** = <em>P</em> &lt; 0.005). Scale bar in C = 50 Î¼m.</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">For pAAVFREX experiments, red and yellow (double positive) cells were considered as recombined and normalized to the total number of transfected cells (sum of red, green, and yellow cells). For the pAAVPTPY construct, TFP and YFP positive cells were considered recombined and normalized to the number of DAPI cells in the field (pAAVPTPY does not express any reporter in the absence of recombination).</p></div><div id="sec-6" class="subsection"><h3>Statistical analysis</h3><p id="p-20">Comparison of nucleotide frequency distribution in the p2NKanP vector before and after Dre recombination (see legend for <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>) was performed using Ï<sup>2</sup> statistics with a custom written Matlab (Mathworks, Inc.) procedure, as follows. Actual A, G, C, and T frequencies for each of the four mutated positions (2, 3, 6, and 7) were tabulated for the 82 Dre-treated minipreps and 24 untreated controls. Expected frequencies for each position (sample), treatment and nucleotide (category) were calculated, and then the Ï<sup>2</sup> statistics and significance value for each position were derived, using built-in Matlab functions, given three degrees of freedom [(two treatmentsâ1) Ã (four categoriesâ1)]. The null hypothesis was defined as the two sets of frequencies for each position being derived from the same distribution. The Ï<sup>2</sup> tables are reported in Supporting Information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025841/-/DC1/TableS1.pdf">Table S1</a>, and the Matlab script is provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025841/-/DC1/FileS1.pdf">File S1</a>.</p><p id="p-21">Box-whisker plots and statistical analyses were performed in Matlab. <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025841/-/DC1/TableS2.pdf">Table S2</a> shows medians, means, number of replicates, images taken, and cells counted in each experiment, as well as outcomes of Student T tests and KolmogorovâSmirnov tests for comparisons drawn in <a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7</a>. Explanation of Box Whisker plots: the tops and bottoms of each ââBoxââ are the 25<sup>th</sup> and 75<sup>th</sup> percentiles of the samples, respectively. The distances between the tops and bottoms are the interquartile ranges. The line in the middle of each box is the sample median. Whiskers are drawn from the ends of the interquartile ranges to the furthest observations within the whisker length (the adjacent values).</p></div><div id="sec-7" class="subsection"><h3>Data availability</h3><p id="p-22">All reagents generated and described in this work are available upon request from the corresponding author.</p></div></div><div class="section results" id="sec-8"><h2 class="">Results</h2><div id="sec-9" class="subsection"><h3>Library and vector design for Dre recombination assay</h3><p id="p-23">We used the existing information on the loxP site to guide our analysis of the roxP spacer region. Since the equivalence of the spacer positions on the two target sites was not immediately obvious, we designed two random nucleotide libraries, carrying random base pairs in the roxP spacer and surrounding bases (<a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1A</a> [top] and <a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1B</a>). The rox4R library (spacer positions 3â6) would contain 4<sup>4</sup> = 256 possible combinations. The rox6R library (positions 2â7) covers the roxP spacer and the first adjacent base of the inverted repeat on either side, which would allow us to test the prediction that positions 2 and 7 are the functional equivalents of those in the loxP spacer, and would contain 4<sup>6</sup> = 4096 variants. Given that this number of combinations would be difficult to cover in a screen for the desired mutants, we also generated a library that covers two bases at the junction between each inverted repeat and the spacer (rox2N library, positions 2, 3, 6, and 7, <a id="xref-fig-1-12" class="xref-fig" href="#F1">Figure 1A</a> [top], and <a id="xref-fig-1-13" class="xref-fig" href="#F1">Figure 1B</a>), but expected to have a smaller search space (256 variants). The three libraries, and a WT positive control roxP were inserted at the roxA site of the proxA-Kan<sup>r</sup>-roxB recombination target vectors (<a id="xref-fig-1-14" class="xref-fig" href="#F1">Figure 1, C and D</a>). Upon successful Dre-mediated recombination, the kanamycin resistance gene is eliminated, resulting in a vector containing just the ampicillin resistance cassette and the origin of replication (<a id="xref-fig-1-15" class="xref-fig" href="#F1">Figure 1C</a>, bottom right). The circular recombination product including the Kan<sup>r</sup> is eliminated in subsequent bacterial cell divisions since it does not have a replication origin.</p></div><div id="sec-10" class="subsection"><h3>Recombination test strategy</h3><p id="p-24">The p4RKanP, p6RKanP, and p2NKanP libraries, and the pPKanP WT control, were assayed for Dre recombination by cotransformation with pGB2-Dre (Spc<sup>r</sup>) into DH5Î±, and plated on spectinomycin<sup>+</sup> ampicillin<sup>+</sup> plates, ensuring selection for both Dre and target vectors, without selecting against recombination (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2A</a>). Since the pMB1 origin of the target vectors replicates at more than 75 copies per bacteria, we reasoned that unrecombined targets might be mixed with recombined ones in the same colony, and therefore did not test individual colonies from this step. Rather, we pooled all colonies from the plates (around 2000 colonies for p4RKanP and p2NKanP, and 6000 colonies for p6RKanP), allowed them to amplify, extracted DNA from the common mix, and digested an aliquot of this pooled DNA with <em>Pvu</em>I + <em>Pst</em>I (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2, B and C</a>). Whereas the pPKanP WT roxP control and the p4RkanP library showed a nearly complete recombination pattern (â¼0.9), the p6RKanP and p2NkanP libraries showed significantly less recombination (â¼0.3 and 0.1, respectively, quantitated in <a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2C</a>). These results suggest that the two positions flanking the spacer (positions 2 and 7, <a id="xref-fig-1-16" class="xref-fig" href="#F1">Figure 1A</a>) have a much stronger influence on the ability of the mutant to recombine with the WT roxP, while substitutions in the central 4bp of the roxP site affect compatibility with the WT roxP in a less significant manner. Thus, it appears that the desired mutants might be present in the p6RKanP and p2NKanP libraries, and that the crossover region of the roxP site might extend beyond the 4 bp central asymmetric motif (positions 3â6).</p><p id="p-25">In order to select mutant spacer sequences incompatible with the WT, the p2NKanP library was cotransformed with pGB2-Dre (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2A</a>) and selected on spectinomycin<sup>+</sup> kanamycin<sup>+</sup> plates. Purified DNA from the pooled colonies was retransformed and individual colonies were amplified under kanamycin selection. Colonies were analyzed by restriction digestion, sequenced, and compared to sequences from the input, unrecombined p2NKanP library (<a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2D</a>). The nucleotide distributions at spacer positions 2 and 7 were significantly skewed in the spacers derived from the screen when compared to the WT library, while distributions at positions 3 and 6 were relatively unaffected by the selection process (<a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2D</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025841/-/DC1/TableS1.pdf">Table S1</a>; statistical significance assessed for each spacer position by Ï<sup>2</sup> test, see <em>Materials and Methods</em>). Position 2 had a strong bias toward G (41/82 = 50% after, compared to 8/24 = 33.3% before recombination) and C (33/82 = 40.24% after, compared to 5/24 = 20.83% before recombination). Position 7 had a strong bias toward G (40/82 = 48.8% after, compared to 5/24 = 20.83% before recombination). Thus a mutation consensus could be derived for the roxP spacer as 1â8 = Ag/cGCCggT, keeping in mind that, in this screen, positions 1 (A), 4 (C), 5 (C), and 8 (T) were kept WT. Of the four mutated positions, the WT nucleotide was selected against only at position 2 (T, 7/82 = 8.53% after, compared to 4/24 = 16.67% before recombination). Since we were interested in the spacers that are least likely to interact with the WT, we identified the variant spacers that occurred more than once (tabulated in <a id="xref-fig-2-12" class="xref-fig" href="#F2">Figure 2E</a>). Five mutant spacer sequences occurred three times in our screen, while only one spacer sequence occurred four times (AgGCCtgT). Thirteen spacer sequences occurred twice, among them the mutant consensus, AgGCCggT. This indicates a level of enrichment for certain spacers, as the expected probability of any given spacer occurring four or three times, respectively, in 82 draws would be (82/256)^4 = 0.01, and (82/256)^3 = 0.03. Among the seven spacers selected for further analysis (<a id="xref-fig-2-13" class="xref-fig" href="#F2">Figure 2E</a>), only rox7 (AgGCCAgT) was also found (once) in the unrecombined control sequences. Thus, although the sequencing depth for both the control and selected libraries were not sufficient to cover the expected diversity, it allowed us to identify a reasonable number of candidates for roxP incompatible sites, and confirmed our prediction that mutations at positions 2 and 7 are deleterious for roxP compatibility.</p></div><div id="sec-11" class="subsection"><h3>Incompatible mutant spacer validation</h3><p id="p-26">We picked representative clones for each of the enriched mutant spacers, as well as the mutation consensus (<a id="xref-fig-2-14" class="xref-fig" href="#F2">Figure 2E</a>, last row), and retested them in our recombination assay. Henceforth, the mutant spacer sequences will be identified by the representative clone number, <em>e.g.</em>, AgGCCtgT, present in clone number 12 is rox12 (<a id="xref-fig-2-15" class="xref-fig" href="#F2">Figure 2E</a>). The plasmids carrying the seven rox mutant spacers rox7ârox85 upstream and a WT roxP site downstream of the Kan<sup>r</sup> cassette (p7KanP â p85KanP, <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3A</a>) were subjected to the Dre recombination assay (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>). The diagnostic restriction digest revealed a nearly complete lack of recombination for all tested mutant rox spacers, while the positive control pPKanP vector showed full recombination (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3, B and C</a>), despite the persistence of the Dre expression vector in the bacterial isolate (5.3 kb band, <a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3B</a>). In addition, purified DNA from the two independent colonies was transformed on ampicillin<sup>+</sup> plates, and then replica plated onto kanamycin<sup>+</sup> plates (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3D</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025841/-/DC1/FigureS2.pdf">Figure S2</a>). As expected, no Kan<sup>r</sup> colonies were observed for the control pPKanP plasmid, while most Amp<sup>r</sup> colonies from the mutant spacer constructs (p7KanPâp85KanP) were successfully replicated onto the kanamycin<sup>+</sup> plates (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3D</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025841/-/DC1/FigureS2.pdf">Figure S2</a>, and quantitation in last column of <a id="xref-fig-3-9" class="xref-fig" href="#F3">Figure 3A</a>).</p></div><div id="sec-12" class="subsection"><h3>Testing self-recombination efficiency of identified roxP spacers</h3><p id="p-27">To test the ability of the identified mutant spacers to engage in Dre-mediated recombination, we generated constructs in which the roxA and roxB sites, encompassing the Kan<sup>r</sup> gene, were occupied by the same rox mutant spacer. We conducted this test for spacers rox7, rox8, rox12, rox61, and rox85, which had the highest degree of identity with our derived consensus, and compared them to the roxP WT control (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4A</a>). After cotransformation with pGB2-Dre and selection on spectinomycin<sup>+</sup> ampicillin<sup>+</sup>, two individual colonies were picked and analyzed as described in the previous section (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4, B and C</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025841/-/DC1/FigureS3.pdf">Figure S3A</a>). The complete loss of the unrecombined (1.8 kb) band in the diagnostic restriction digest, and essentially complete loss of kanamycin resistance in replica plating tests, indicated that the five chosen mutant spacers are able to recombine with themselves with efficiencies comparable to the WT control in the prokaryotic environment. Together with results presented in the previous section, this suggests that the lack of recombination with the WT roxP is a result of spacer incompatibility and not of lack of intrinsic ability to participate in the recombination reaction.</p></div><div id="sec-13" class="subsection"><h3>Testing cross-talk between novel rox variants and Cre recombinase</h3><p id="p-28">To address the possibility of cross-talk between the Cre recombinase and the newly generated mutant sites, we transformed our five rox mutant plasmids and the WT control in a bacterial strain expressing the Cre recombinase (BS1365), which we and others have routinely used to test Cre-loxP recombination in target vectors (<em>e.g.</em>, <a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Siegel <em>et al.</em> 2001</a>, and data not shown). As seen in <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4, D and E</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025841/-/DC1/FigureS3.pdf">Figure S3B</a>, no visible recombination of either mutant or WT rox target sites was detected by either diagnostic digest or replica plating. This confirms the many previous reports that there is little reactivity between the Dre and Cre recombination systems, and that our newly identified spacers did not acquire Cre sensitivity by virtue of the substituted bases.</p></div><div id="sec-14" class="subsection"><h3>Testing recombination efficiencies of mutant rox sites in a purified system</h3><p id="p-29">Bacterial recombination assays were powerful for the screening and evaluation of identified mutants. However, saturation effects could overestimate recombination efficiencies seen for the identified mutant spacers. We therefore used a purified system to directly compare self-recombination efficiencies of the identified mutants and WT roxP sequence. The same set of vectors used in the prokaryotic self-recombination test (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4A</a>) were subjected to Dre recombination using bacterially expressed and affinity purified Dre protein, and reaction conditions similar to those used for equivalent <em>in vitro</em> assays for the Cre-loxP purified system (<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Lee and Saito 1998</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Ghosh and Van Duyne 2002</a>). The vectors were linearized, and then subjected to recombination for 5, 10, 15, 20, 30, 60, and 120 min, with around 10-fold molar excess Dre. For diagnostic and quantitative purposes, the DNA was further digested with <em>Pvu</em>I and <em>Msc</em>I, loaded on agarose gels and quantitated by gel densitometry. The characteristic recombination bands (see <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, legend) were normalized to the recombination-indifferent internal control band, and reported in percent.</p><p id="p-30">We find that <em>in vitro</em> self-recombination efficiencies for the five analyzed mutants vary significantly, with spacer rox8 exhibiting about 0.7 and spacers rox7, rox12, and rox61 about 0.4â0.5 of WT recombination efficiency (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5, BâD</a>).</p></div><div id="sec-15" class="subsection"><h3>Testing mutant rox sites in eukaryotic cells</h3><p id="p-31">We next established an assay to test the recombination specificity and efficiency of our rox mutants in eukaryotic cells. A Dre-expressing HEK-293 stable cell line was established using a single copy integration system based on Flp recombination (see <em>Materials and Methods</em>). In addition, a roxA-mCherry-STOP-roxB-eGFP cassette was inserted downstream of a CMV promoter in the pcDNA3.1 eukaryotic expression vector (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6A</a>, prox-R-rox-G). Using this strategy, we tested two mutants, rox12 (most frequently recovered spacer) and rox85 (consensus noncompatible sequence), for the ability to self-recombine and the lack of recombination with the WT locus. The vectors (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6A</a>, right table) were individually transfected in either the HEK293-Dre cell line (<a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6, B, D, and F</a>), or a previously generated HEK293-Cre line (<a id="xref-fig-6-7" class="xref-fig" href="#F6">Figure 6, C and E</a>), and analyzed by direct fluorescence 48 hr after transfection. We found that rox12 and rox85 mutant rox sites were capable of self-recombination at efficiencies comparable to the WT roxP (<a id="xref-fig-6-8" class="xref-fig" href="#F6">Figure 6, B, D, and F</a>, columns 1, 2, and 4). HEK293-Dre cells transfected with the proxP-R-roxP-G, prox12-R-rox12-G, and prox85-R-rox85-G vectors did not show a complete conversion from red to green fluorescence, with more than half the cells exhibiting both. However, whereas images were taken with identical fluorescence intensity and exposure time settings across all experiments, the red fluorescence intensity was much lower for images in <a id="xref-fig-6-9" class="xref-fig" href="#F6">Figure 6B</a>, columns 1, 2, and 4, compared to the rest of the images and had to be digitally enhanced for counting or visualizing transfected cells. This suggests that mCherry levels were very low in HEK293-Dre cells transfected with the three vectors capable of self-recombination, perhaps as a result of residual expression of mCherry from target plasmid before recombination. Nevertheless, 5% of prox12-R-rox12-G and 8.7% of prox85-R-rox85-G transfectants were GFP negative (<a id="xref-fig-6-10" class="xref-fig" href="#F6">Figure 6B</a>, quantitated in <a id="xref-fig-6-11" class="xref-fig" href="#F6">Figure 6, D and F</a>) while less than 1% of proxP-R-roxP-G transfected HEK293-Dre cells had not undergone any detectable recombination. Thus, recombination mediated by these two newly identified rox spacers is slightly less efficient than that for the WT. However, neither rox12 nor rox85 showed any detectable recombination with the WT roxP (<a id="xref-fig-6-12" class="xref-fig" href="#F6">Figure 6, B, D, and F</a>, columns 3 and 5). In addition, none of the WTâmutant or mutantâmutant combinations resulted in detectable recombination when transfected into the HEK293-Cre line (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5, C and E</a>). In contrast, a control FLEX cassette type vector (using loxp and lox2272 sites) expressing membrane-bound eYFP and PSD95-TFP in a Cre-dependent manner (B. Wu and T. C. Badea, unpublished results) showed fluorescently labeled cells when transfected in HEK293-Cre but not HEK293-Dre-expressing cells (data not shown and <a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7, CâG</a>). Thus, the two novel rox sites identified in the bacterial recombination screen exhibit specific, efficient self-recombination when exposed to the Dre recombinase, and no cross-reactivity with the Cre recombinase when tested in eukaryotic cell culture.</p><p id="p-32">Since the rox12 self-recombination efficiency more closely matched the WT control, we used the rox12 and roxP sites to generate a sequential inversionâexcision Dre recombination cassette, analogous to the FLEX cassettes used for the loxP system. cDNAs for mCherry and eGFP arranged in reversed configuration and separated by a bidirectional SV40 polyA transcription STOP signal were placed between two sets of rox12-roxP sites, arranged in inverted configuration with respect to each other (<a id="xref-fig-7-5" class="xref-fig" href="#F7">Figure 7A</a>). This cassette is designed to express eGFP before and mCherry after Dre recombination. Two alternative, reversible inversions, followed by irreversible excisions, result in an irreversible inversion of the fluorescent protein cassette and mCherry expression (<a id="xref-fig-7-6" class="xref-fig" href="#F7">Figure 7B</a>). When tested in our HEK293-Dre line, the FREX cassette showed more than 95% recombination efficiencies, at levels similar to the WT roxP-mCherry-STOP-roxP-eGFP configuration (<a id="xref-fig-7-7" class="xref-fig" href="#F7">Figure 7, C, E, and G</a>). However, no recombination was detected in HEK293-Cre cells (<a id="xref-fig-7-8" class="xref-fig" href="#F7">Figure 7, D, F, and G</a>). In contrast, the Cre-dependent FLEX vector, pAAVPTPY, resulted in recombination in HEK293-Cre but not HEK293-Dre cells (<a id="xref-fig-7-9" class="xref-fig" href="#F7">Figure 7, CâG</a>).</p></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><p id="p-33">We report here the identification of several roxP variants that are incompatible with the WT Dre recombinase roxP target site, but recombine efficiently with themselves. We used a random nucleotide insertion strategy to map the involvement of base pair positions in and around the roxP asymmetric spacer in Dre-mediated recombination specificity. We find that, whereas mutations covering the central 4 bp have a mild effect on WT compatibility, mutations involving the bases adjacent to the roxP spacer, specifically in the first base of the inverted repeat, drastically reduce rox mutant-roxP WT recombination. Most of the identified mutants show three or more base substitutions compared to the WT roxP consensus. WT incompatible sites tend to have strong preferences for G/C and G, respectively, in the bases flanking the unidirectional spacer.</p><div id="sec-17" class="subsection"><h3>Implications for the Dre-roxP mechanism</h3><p id="p-34">Despite the good conservation between Dre and Cre proteins and between roxP and loxP sites (<a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Guo <em>et al.</em> 1997</a>; <a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Sauer and McDermott 2004</a>), the task of mapping the previously identified base pair functions of the loxP site onto the roxP site is more difficult than expected. The major challenge consists in the smaller asymmetric spacer of the roxP site (4 bp <em>vs.</em> 8 bp in loxP). The loxP spacerâs compatibility with the WT is significantly affected by substitutions at the G-C base pair 5â² to the catalytic attack site (<a id="xref-fig-1-17" class="xref-fig" href="#F1">Figure 1A</a>, position 7), less perturbed by substitutions at positions 3â6, and essentially unaffected by mutations at positions 1 and 8 (<a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">Lee and Saito 1998</a>). The backbone phosphate bonds between nucleotides 7 and 8 on the bottom strand, and subsequently nucleotides 1 and 2 on the top strand, are the catalytic attack sites that form transient covalent bonds with tyrosine Tyr324 of Cre and electrostatic bonds with the other four bases of the catalytic pocket (Arg173, His289, Arg292, and Trp315) (<a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">Guo <em>et al.</em> 1997</a>). If the bases responsible for spacer compatibility are recognized by virtue of their asymmetry, one might infer that the equivalent positions for the Dre-roxP system lie within the four central base pairs of the roxP site (positions 3â6 in <a id="xref-fig-1-18" class="xref-fig" href="#F1">Figure 1A</a>). However, if the effects on spacer compatibility are dictated by their positions relative to the cleavage site, and can be extrapolated to the Dre-roxP complex, our data argue that the roxP crossover region, and hence the cleavage site, reaches at least into the first base of the inverted repeat, since mutations including spacer bases 2,3 and 6,7 (<a id="xref-fig-1-19" class="xref-fig" href="#F1">Figure 1A</a>) had dramatically more impact than mutations restricted to positions 3 through 6. This may suggest that the roxP crossover region is actually closer in length to loxP. These predictions are consistent with our atomic structure model of the Dre-roxP interaction based on a previously published crystal structure of the Cre-loxP complex (<a id="xref-ref-12-7" class="xref-bibr" href="#ref-12">Guo <em>et al.</em> 1997</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025841/-/DC1/FigureS1.pdf">Figure S1</a>) using homology modeling. These findings lend support to the hypothesis that functionally the roxP and loxP crossover regions are similar in length, and that strand cleavage happens at equivalent positions (1â2 top and 7â8 bottom) in both (<a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Sauer and McDermott 2004</a>). Since the mapping of the roxP spacer was only accessory to our main goal, we have not further dissected these questions. Clearly, more mutational analysis and other complementary approaches, like the use of suicide substrates and crystalography, are needed to definitively identify the Dre catalytic target onto the roxP spacer. However, our analysis offers a useful starting point.</p></div><div id="sec-18" class="subsection"><h3>Specificity and recombination efficiency</h3><p id="p-35">Out of the identified roxP incompatible variant spacers, the seven most frequently encountered sequences showed essentially no cross-reactivity with the WT roxP site upon retesting. Six variants had guanine nucleotides at positions 2 and 7 in the spacer (<a id="xref-fig-2-16" class="xref-fig" href="#F2">Figure 2D</a>), while a seventh had cytosine nucleotides on either side (rox8), in keeping with the noncompatible consensus derived from our screen (Ag/cGCCggT, <a id="xref-fig-2-17" class="xref-fig" href="#F2">Figure 2D</a>). In fact, similar observations have been reported in screens for loxP-incompatible lox sites (<a id="xref-ref-23-6" class="xref-bibr" href="#ref-23">Lee and Saito 1998</a>; <a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Siegel <em>et al.</em> 2001</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Langer <em>et al.</em> 2002</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Missirlis <em>et al.</em> 2006</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Jung <em>et al.</em> 2007</a>), in which self-compatible, efficiently recombining sequences tend to prefer G or C at positions 2 and 7 (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Langer <em>et al.</em> 2002</a>; <a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Missirlis <em>et al.</em> 2006</a>), and the general recombination efficiency is associated with the presence of G bases (<a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Missirlis <em>et al.</em> 2006</a>). However, it should be pointed out that in the saturation screen performed in <a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">Missirlis <em>et al.</em> (2006</a>), the spacers exhibiting the highest frequency of self-recombination also had a high degree of promiscuity <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a> in (<a id="xref-ref-27-6" class="xref-bibr" href="#ref-27">Missirlis <em>et al.</em> 2006</a>), but that most of that promiscuity may be with other G containing spacers, since sequences containing Gs at positions 2 and 7 were less likely to recombine with spacers containing other nucleotides at those positions, <a id="xref-fig-6-13" class="xref-fig" href="#F6">Figure 6</a> in (<a id="xref-ref-27-7" class="xref-bibr" href="#ref-27">Missirlis <em>et al.</em> 2006</a>). In this context, it would be interesting to explore the mutual recombination compatibility of our novel rox sites. If differences at positions 2 and 7 are a predictor of spacer compatibility, six of the further characterized spacers that carry Gs at both positions are very likely to cross-react. Since mutant rox8 (AcGCCtcT) carries Cs in positons 2 and 7, it is the most likely to be incompatible with the other newly identified rox variants. The five rox variants tested for self-recognition in the bacterial system (rox7, rox8, rox12, rox61, and rox85) showed nearly complete recombination patterns. However, the time course analysis performed with purified components reveals that all identified spacers have reduced self-recombination efficiency when compared to the WT, with rox8 exhibiting the highest self-recombination efficiency among the novel mutant sites (â¼70%), while three others (rox7, rox12, and rox65) are around 40%, consistent with the findings in the Cre-loxP system, in which substitutions at positions 2 and 7 had reduced self-recombination properties (<a id="xref-ref-23-7" class="xref-bibr" href="#ref-23">Lee and Saito 1998</a>). For the two spacers assayed in eukaryotic tissue culture, rox12 (the most frequently recovered rox variant from the screen) and rox85 (representing the consensus site), recombination specificity was high (no cross-talk with the WT) but the efficiency of self-recombination was somewhat reduced compared to the WT control, as judged by both the number of remaining unrecombined cells and the degree of complete conversion from the unrecombined to recombined forms.</p><p id="p-36">The assay-dependent differences in self-recombination efficiency may be a measure of relative amounts of Dre protein to roxP targets available and the distinct time courses employed in the purified, bacterial and eukaryotic systems. Whereas the purified component, biochemistry experiment seems to get close to the maximal levels at around 30 min for roxP and most of its variants, the bacterial recombination assay has 24â48 hr and the eukaryotic assay 48 hr of potential interaction between less well understood amounts of Dre and target sites. Although the <em>in vitro</em> 10-fold molar excess sounds like a large margin, in the bacterial experiment, the number of targets/bacterium should be equivalent to the maximum number of plasmid copies, (â¼75 for pMB1), and we have not assayed the amount of Dre generated from the pSC101 plasmid (1â5 plasmid copies/bacteria, and lac promoter). Moreover, the small circular product, devoid of replication origin, will be gradually diluted during cell divisions, favoring the excision reaction and complete recombination. However, recombination assays using roxP WT target vectors but coexpressing Dre from a high copy number vector (300â500 copies/bacteria) resulted in abnormally recombined products (data not shown), suggesting that the amount of Dre protein delivered can be critical not only to the efficiency but also to the specificity of the reaction. In the eukaryotic system, the HEK293-Dre line expresses Dre from a single copy knock-in locus using a CAG promoter, and hence protein levels are expected to be smaller than those for transient transfection or infection conditions. In addition, the fluorescent protein reflecting the unrecombined state (<em>e.g.</em>, mCherry) could be expressed and accumulated in transfected cells before recombination and conversion to eGFP. However, it is clear that the identified spacer mutations have an impact on the recombination efficiency, consistent with findings for lox2272 and lox511 (lox51), the most widely used loxP substitutes (<a id="xref-ref-23-8" class="xref-bibr" href="#ref-23">Lee and Saito 1998</a>; <a id="xref-ref-35-5" class="xref-bibr" href="#ref-35">Siegel <em>et al.</em> 2001</a>). Although rox8 was the most efficient variant in the <em>in vitro</em> self-recombination assay, it also exhibited a small amount of roxP WT cross-talk in the cotransformation assay, and hence we opted for rox12 (and the least efficient rox85 as a control) as our further candidate in eukaryotic tests.</p></div><div id="sec-19" class="subsection"><h3>Is there cross-talk between the Dre and Cre systems?</h3><p id="p-37">We do not find any evidence for cross-talk between Cre recombinase and the newly identified roxP target sites or the WT control. This is in agreement with observations from many labs that have reported no cross-activation between the Dre and Cre systems in <em>E. coli</em>, HEK293 cells, zebrafish, mouse embryonic stem cells, or mouse lines carrying either Dre- or Cre-expressing transgenes and suitable reporters (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Sauer and Henderson 1988</a>; <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Anastassiadis <em>et al.</em> 2009</a>; <a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Park and Leach 2013</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Sajgo <em>et al.</em> 2014</a>). However, recently two studies have reported cross-activation between Dre and Cre systems, especially when the Dre or Cre recombinase and/or the reporter constructs are delivered to HEK293 cells or hippocampal or cortical neurons via AAV vectors (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Fenno <em>et al.</em> 2014</a>; <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Madisen <em>et al.</em> 2015</a>). In fact, Madisen <em>et al.</em> found little cross-talk between the two systems when the same genetically encoded reporter was tested by crossing with single copy targeted Dre and Cre drivers (compare <a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Madisen <em>et al.</em> 2015</a>, Supplementary Figure 3B to Supplementary Figure 4B). In addition, this could be due in part to the choice of mutations in the target roxP sites, <a id="xref-fig-1-20" class="xref-fig" href="#F1">Figure 1</a> in (<a id="xref-ref-8-4" class="xref-bibr" href="#ref-8">Fenno <em>et al.</em> 2014</a>), which bring up the homology between Dre and Cre left hemisites to 13/16 nucleotides. In our hands, nonspecific recombination, even in the absence of cognate target sites, can be achieved when large amounts of recombinase are expressed (see above paragraph). We therefore think that the relative amounts of recombinases and target vectors expressed could be the reason for the nonspecific effects seen in the viral delivery experiments, but not in full embryo genetic manipulations.</p></div><div id="sec-20" class="subsection"><h3>Potential applications in genetic manipulations</h3><p id="p-38">The use of SSRs in molecular genetic manipulations is rendered tremendously powerful by its ability to generate combinatorial expression patterns restricting gene manipulations to increasingly narrower domains (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Jensen and Dymecki 2014</a>). Dre could function as one of the members of the combinatorial recombination code. In many instances, it is desirable to use two distinct recombination target sites in order to be able to intersect three gene expression domains and single out one cell population. This type of manipulation has been implemented at genetic loci for the Dre, Flp and Cre recombinases in mice and zebrafish (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Farago <em>et al.</em> 2006</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hirsch <em>et al.</em> 2013</a>; <a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Park and Leach 2013</a>; <a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Sajgo <em>et al.</em> 2014</a>; <a id="xref-ref-26-5" class="xref-bibr" href="#ref-26">Madisen <em>et al.</em> 2015</a>), and for several new recombinases in AAV-based strategies (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Nern <em>et al.</em> 2011</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Suzuki and Nakayama 2011</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Karimova <em>et al.</em> 2013</a>; <a id="xref-ref-8-5" class="xref-bibr" href="#ref-8">Fenno <em>et al.</em> 2014</a>). The strategy may be arranged âin parallelâ, by having the two recombinases expressed from two different genetic elements, with a third locus expressing a dual reporter, which sequentially switches color in response to either one or both recombinases (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Farago <em>et al.</em> 2006</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Hirsch <em>et al.</em> 2013</a>; <a id="xref-ref-30-4" class="xref-bibr" href="#ref-30">Park and Leach 2013</a>; <a id="xref-ref-26-6" class="xref-bibr" href="#ref-26">Madisen <em>et al.</em> 2015</a>). It could also be arranged in a âserialâ fashion, involving a first genetic locus driving one of the recombinases which then induces the expression of the second recombinase from a distinct locus, followed by the final activation of a reporter dependent on a third gene (<a id="xref-ref-31-4" class="xref-bibr" href="#ref-31">Sajgo <em>et al.</em> 2014</a>).</p><p id="p-39">Constructs in which expression of a reporter is activated by a recombinase have been arranged in variants of two basic designs. The âPromoter â loxP â STOP â loxP â Reporterâ design (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lobe <em>et al.</em> 1999</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Badea <em>et al.</em> 2009</a>), as seen for instance in the vectors in <a id="xref-fig-6-14" class="xref-fig" href="#F6">Figure 6</a>, will prevent expression of the reporter by transcriptional termination at the STOP signal until Cre recombination occurs. Although successfully used in many occasions, it is often flawed by incomplete transcription termination at the STOP, resulting in some reporter being made even in the absence of Cre (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Badea <em>et al.</em> 2009</a>; <a id="xref-ref-31-5" class="xref-bibr" href="#ref-31">Sajgo <em>et al.</em> 2014</a>). This inconvenience can be overcome by the FLEX design, in which two incompatible lox sites (lox2272 and loxP) are arranged in tandem and in reverse orientation on either side of an inverted (inv) reporter [Promoter â lox2272 â loxP â inv(Reporter) â inv(lox2272) â inv(loxP)] (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Schnutgen <em>et al.</em> 2003</a>; <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Atasoy <em>et al.</em> 2008</a>). Cre induces two alternative inversions followed by excisions resulting in the same final configuration: Promoter â lox2272 â Reporter â inv(loxP). This strategy, first used in a mouse model, and extremely popular for Cre-dependent AAV reporters, relies heavily on the incompatibility between the WT loxP site and the mutant lox2272 site. Our novel rox sites can be used for the same type of application in combination with roxP for the Dre recombinase. We propose the analogous FREX cassette for the Dre to roxP system as demonstrated for the roxP-rox12 combination, or extended to the rox8 or other mutants identified in the screen. Since these mutants had zero recombination with the WT and were not targeted by the Cre recombinase, they may represent an improvement over a recently published alternative rox site (<a id="xref-ref-8-6" class="xref-bibr" href="#ref-8">Fenno <em>et al.</em> 2014</a>), and would be ideal in future combinatorial genetic manipulations involving Dre. Regardless of the specific configuration of the adopted strategy and/or the specificity and efficiency of the employed recombinase and specific target sites, our results add to the many previously published biochemical data that highlight the importance of the level of expression of the recombinase and the abundance of the target sites in the outcome. These aspects are particularly critical in genetic applications, where driver loci have intrinsic cell type specificities, time courses and levels of expression, while target loci have distinct tolerance for recombination, all properties which are somewhat predetermined and have to be carefully matched in order to achieve the desired level and specificity of recombination.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-40">The authors acknowledge Michael Cashel for helpful discussions and suggestions, and providing the pGB2 vector; Beverly Wu for constructing and testing the pAAVPTPY vector; and Edward Giniger, Harold Burgess, Kathryn Tabor, and Chi-Hon Lee for critical reading of the manuscript and insightful comments. Funding provided to T.B. and Y.S. by intramural funds of the National Eye Institute (National Institutes of Health, Bethesda). </p><p id="p-41">Author contributions: K.C., E.N., and T.B. designed and performed all experiments. Y.S. and T.B. designed and performed molecular modeling of the Dre-roxP complex. K.C., E.N., Y.S., and T.B. generated the figures and wrote the manuscript. The authors declare no conflicts of interest.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-1"><p id="p-42">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025841/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.025841/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-43"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-08-27"><span class="received-label">Received </span>August 27, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-12-21"><span class="accepted-label">Accepted </span>December 21, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2016 Chuang <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.1" data-doi="10.1242/dmm.003087"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anastassiadis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patsch</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weidlich</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Dre recombinase, like Cre, is a highly efficient site-specific recombinase in E. coli, mammalian cells and mice.</span> <abbr class="cit-jnl-abbrev">Dis. Model. Mech.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">508</span>â<span class="cit-lpage">515</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDis.%2BModel.%2BMech.%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdmm.003087%26rft_id%253Dinfo%253Apmid%252F19692579%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZG1tIjtzOjU6InJlc2lkIjtzOjEwOiIyLzktMTAvNTA4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzU1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.2" data-doi="10.1523/JNEUROSCI.1954-08.2008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Atasoy</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aponte</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Su</span>  <span class="cit-name-given-names">H. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sternson</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">A FLEX switch targets Channelrhodopsin-2 to multiple cell types for imaging and long-range circuit mapping.</span> <abbr class="cit-jnl-abbrev">J. Neurosci.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">7025</span>â<span class="cit-lpage">7030</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BNeuroscience%26rft.stitle%253DJ.%2BNeurosci.%26rft.aulast%253DAtasoy%26rft.auinit1%253DD.%26rft.volume%253D28%26rft.issue%253D28%26rft.spage%253D7025%26rft.epage%253D7030%26rft.atitle%253DA%2BFLEX%2BSwitch%2BTargets%2BChannelrhodopsin-2%2Bto%2BMultiple%2BCell%2BTypes%2Bfor%2BImaging%2Band%2BLong-Range%2BCircuit%2BMapping%26rft_id%253Dinfo%253Adoi%252F10.1523%252FJNEUROSCI.1954-08.2008%26rft_id%253Dinfo%253Apmid%252F18614669%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6Njoiam5ldXJvIjtzOjU6InJlc2lkIjtzOjEwOiIyOC8yOC83MDI1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzU1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.3" data-doi="10.1016/j.neuron.2009.01.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Badea</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cahill</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ecker</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hattar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nathans</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Distinct roles of transcription factors brn3a and brn3b in controlling the development, morphology, and function of retinal ganglion cells.</span> <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">852</span>â<span class="cit-lpage">864</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D61%26rft.spage%253D852%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2009.01.020%26rft_id%253Dinfo%253Apmid%252F19323995%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2009.01.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19323995&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000264741400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.4" data-doi="10.1007/978-1-60327-019-9_16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Birling</span>  <span class="cit-name-given-names">M.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gofflot</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warot</span>  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Site-specific recombinases for manipulation of the mouse genome.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">561</span>: <span class="cit-fpage">245</span>â<span class="cit-lpage">263</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DBirling%26rft.auinit1%253DM.%2BC.%26rft.volume%253D561%26rft.spage%253D245%26rft.epage%253D263%26rft.atitle%253DSite-specific%2Brecombinases%2Bfor%2Bmanipulation%2Bof%2Bthe%2Bmouse%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-60327-019-9_16%26rft_id%253Dinfo%253Apmid%252F19504076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-60327-019-9_16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19504076&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.5" data-doi="10.1007/978-1-59745-583-1_10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bischof</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Basler</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Recombinases and their use in gene activation, gene inactivation, and transgenesis.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">420</span>: <span class="cit-fpage">175</span>â<span class="cit-lpage">195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DBischof%26rft.auinit1%253DJ.%26rft.volume%253D420%26rft.spage%253D175%26rft.epage%253D195%26rft.atitle%253DRecombinases%2Band%2Btheir%2Buse%2Bin%2Bgene%2Bactivation%252C%2Bgene%2Binactivation%252C%2Band%2Btransgenesis.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-59745-583-1_10%26rft_id%253Dinfo%253Apmid%252F18641947%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-59745-583-1_10&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18641947&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.6" data-doi="10.1016/0378-1119(84)90207-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchward</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belin</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagamine</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span>â<span class="cit-article-title">A pSC101-derived plasmid which shows no sequence homology to other commonly used cloning vectors.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">165</span>â<span class="cit-lpage">171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DChurchward%26rft.auinit1%253DG.%26rft.volume%253D31%26rft.issue%253D1-3%26rft.spage%253D165%26rft.epage%253D171%26rft.atitle%253DA%2BpSC101-derived%2Bplasmid%2Bwhich%2Bshows%2Bno%2Bsequence%2Bhomology%2Bto%2Bother%2Bcommonly%2Bused%2Bcloning%2Bvectors.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0378-1119%252884%252990207-5%26rft_id%253Dinfo%253Apmid%252F6098521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0378-1119(84)90207-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6098521&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1984TU90200020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.7" data-doi="10.1016/j.neuron.2006.03.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Farago</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Awatramani</span>  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dymecki</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Assembly of the brainstem cochlear nuclear complex is revealed by intersectional and subtractive genetic fate maps.</span> <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">205</span>â<span class="cit-lpage">218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.stitle%253DNeuron%26rft.aulast%253DFarago%26rft.auinit1%253DA.%2BF.%26rft.volume%253D50%26rft.issue%253D2%26rft.spage%253D205%26rft.epage%253D218%26rft.atitle%253DAssembly%2Bof%2Bthe%2Bbrainstem%2Bcochlear%2Bnuclear%2Bcomplex%2Bis%2Brevealed%2Bby%2Bintersectional%2Band%2Bsubtractive%2Bgenetic%2Bfate%2Bmaps.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2006.03.014%26rft_id%253Dinfo%253Apmid%252F16630833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2006.03.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16630833&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237176700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.8" data-doi="10.1038/nmeth.2996"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fenno</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mattis</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramakrishnan</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyun</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Targeting cells with single vectors using multiple-feature Boolean logic.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">763</span>â<span class="cit-lpage">772</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D11%26rft.spage%253D763%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2996%26rft_id%253Dinfo%253Apmid%252F24908100%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2996&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24908100&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.9" data-doi="10.1016/S1046-2023(02)00244-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ghosh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Duyne</span>  <span class="cit-name-given-names">G. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">Cre-loxP biochemistry.</span> <abbr class="cit-jnl-abbrev">Methods</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">374</span>â<span class="cit-lpage">383</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2B%2528San%2BDiego%252C%2BCalif.%2529%26rft.stitle%253DMethods%26rft.aulast%253DGhosh%26rft.auinit1%253DK.%26rft.volume%253D28%26rft.issue%253D3%26rft.spage%253D374%26rft.epage%253D383%26rft.atitle%253DCre-loxP%2Bbiochemistry.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1046-2023%252802%252900244-X%26rft_id%253Dinfo%253Apmid%252F12431441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1046-2023(02)00244-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12431441&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000179487300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.10" data-doi="10.1016/S0959-440X(99)80003-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gopaul</span>  <span class="cit-name-given-names">D. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Duyne</span>  <span class="cit-name-given-names">G. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span>â<span class="cit-article-title">Structure and mechanism in site-specific recombination.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Struct. Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">14</span>â<span class="cit-lpage">20</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bstructural%2Bbiology%26rft.stitle%253DCurr%2BOpin%2BStruct%2BBiol%26rft.aulast%253DGopaul%26rft.auinit1%253DD.%2BN.%26rft.volume%253D9%26rft.issue%253D1%26rft.spage%253D14%26rft.epage%253D20%26rft.atitle%253DStructure%2Band%2Bmechanism%2Bin%2Bsite-specific%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0959-440X%252899%252980003-7%26rft_id%253Dinfo%253Apmid%252F10047575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0959-440X(99)80003-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10047575&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.11" data-doi="10.1146/annurev.biochem.73.011303.073908"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grindley</span>  <span class="cit-name-given-names">N. D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whiteson</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rice</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Mechanisms of site-specific recombination.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Biochem.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">567</span>â<span class="cit-lpage">605</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bbiochemistry%26rft.stitle%253DAnnu%2BRev%2BBiochem%26rft.aulast%253DGrindley%26rft.auinit1%253DN.%2BD.%26rft.volume%253D75%26rft.spage%253D567%26rft.epage%253D605%26rft.atitle%253DMechanisms%2Bof%2Bsite-specific%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.biochem.73.011303.073908%26rft_id%253Dinfo%253Apmid%252F16756503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.biochem.73.011303.073908&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16756503&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239807600022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.12" data-doi="10.1038/37925"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gopaul</span>  <span class="cit-name-given-names">D. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Duyne</span>  <span class="cit-name-given-names">G. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-article-title">Structure of Cre recombinase complexed with DNA in a site-specific recombination synapse.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">389</span>: <span class="cit-fpage">40</span>â<span class="cit-lpage">46</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DGuo%26rft.auinit1%253DF.%26rft.volume%253D389%26rft.issue%253D6646%26rft.spage%253D40%26rft.epage%253D46%26rft.atitle%253DStructure%2Bof%2BCre%2Brecombinase%2Bcomplexed%2Bwith%2BDNA%2Bin%2Ba%2Bsite-specific%2Brecombination%2Bsynapse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F37925%26rft_id%253Dinfo%253Apmid%252F9288963%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/37925&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9288963&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XU59600037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.13" data-doi="10.1002/dvg.22393"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">dâAutreaux</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dymecki</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brunet</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goridis</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">A Phox2b:FLPo transgenic mouse line suitable for intersectional genetics.</span> <abbr class="cit-jnl-abbrev">Genesis</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">506</span>â<span class="cit-lpage">514</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenesis%26rft.volume%253D51%26rft.spage%253D506%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fdvg.22393%26rft_id%253Dinfo%253Apmid%252F23592597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/dvg.22393&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23592597&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321982400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.14" data-doi="10.1073/pnas.81.4.1026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoess</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abremski</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span>â<span class="cit-article-title">Interaction of the bacteriophage P1 recombinase Cre with the recombining site loxP.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">1026</span>â<span class="cit-lpage">1029</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DHoess%26rft.auinit1%253DR.%2BH.%26rft.volume%253D81%26rft.issue%253D4%26rft.spage%253D1026%26rft.epage%253D1029%26rft.atitle%253DInteraction%2Bof%2Bthe%2Bbacteriophage%2BP1%2Brecombinase%2BCre%2Bwith%2Bthe%2Brecombining%2Bsite%2BloxP.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.81.4.1026%26rft_id%253Dinfo%253Apmid%252F6230671%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI4MS80LzEwMjYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNTU5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.15" data-doi="10.1016/0022-2836(85)90224-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoess</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abremski</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span>â<span class="cit-article-title">Mechanism of strand cleavage and exchange in the Cre-lox site-specific recombination system.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">181</span>: <span class="cit-fpage">351</span>â<span class="cit-lpage">362</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DHoess%26rft.auinit1%253DR.%2BH.%26rft.volume%253D181%26rft.issue%253D3%26rft.spage%253D351%26rft.epage%253D362%26rft.atitle%253DMechanism%2Bof%2Bstrand%2Bcleavage%2Band%2Bexchange%2Bin%2Bthe%2BCre-lox%2Bsite-specific%2Brecombination%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0022-2836%252885%252990224-4%26rft_id%253Dinfo%253Apmid%252F3856690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0022-2836(85)90224-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3856690&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1985ADP4200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.16" data-doi="10.1093/nar/14.5.2287"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoess</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wierzbicki</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abremski</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span>â<span class="cit-article-title">The role of the loxP spacer region in P1 site-specific recombination.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">2287</span>â<span class="cit-lpage">2300</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DHoess%26rft.auinit1%253DR.%2BH.%26rft.volume%253D14%26rft.issue%253D5%26rft.spage%253D2287%26rft.epage%253D2300%26rft.atitle%253DThe%2Brole%2Bof%2Bthe%2BloxP%2Bspacer%2Bregion%2Bin%2BPI%2Bsite-specific%2Brecombination%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F14.5.2287%26rft_id%253Dinfo%253Apmid%252F3457367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjE0LzUvMjI4NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMy81NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.17" data-doi="10.1016/S0022-2836(99)80007-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoess</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abremski</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Irwin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kendall</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mack</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span>â<span class="cit-article-title">DNA specificity of the Cre recombinase resides in the 25 kDa carboxyl domain of the protein.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">216</span>: <span class="cit-fpage">873</span>â<span class="cit-lpage">882</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DHoess%26rft.auinit1%253DR.%26rft.volume%253D216%26rft.issue%253D4%26rft.spage%253D873%26rft.epage%253D882%26rft.atitle%253DDNA%2Bspecificity%2Bof%2Bthe%2BCre%2Brecombinase%2Bresides%2Bin%2Bthe%2B25%2BkDa%2Bcarboxyl%2Bdomain%2Bof%2Bthe%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0022-2836%252899%252980007-2%26rft_id%253Dinfo%253Apmid%252F2266559%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0022-2836(99)80007-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2266559&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ER95200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.18" data-doi="10.1016/j.conb.2011.09.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jefferis</span>  <span class="cit-name-given-names">G. S. X. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Livet</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Sparse and combinatorial neuron labelling.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Neurobiol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">101</span>â<span class="cit-lpage">110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bneurobiology%26rft.stitle%253DCurr%2BOpin%2BNeurobiol%26rft.aulast%253DJefferis%26rft.auinit1%253DG.%2BS.%26rft.volume%253D22%26rft.issue%253D1%26rft.spage%253D101%26rft.epage%253D110%26rft.atitle%253DSparse%2Band%2Bcombinatorial%2Bneuron%2Blabelling.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.conb.2011.09.010%26rft_id%253Dinfo%253Apmid%252F22030345%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.conb.2011.09.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22030345&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.19" data-doi="10.1007/978-1-60327-292-6_26"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jensen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dymecki</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Essentials of recombinase-based genetic fate mapping in mice.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">1092</span>: <span class="cit-fpage">437</span>â<span class="cit-lpage">454</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BMol.%2BBiol.%26rft.volume%253D1092%26rft.spage%253D437%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-60327-292-6_26%26rft_id%253Dinfo%253Apmid%252F24318835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-60327-292-6_26&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24318835&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.20" data-doi="10.1016/j.bbrc.2007.08.145"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">U. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shin</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwon</span>  <span class="cit-name-given-names">M. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Analysis of spacer regions derived from intramolecular recombination between heterologous loxP sites.</span> <abbr class="cit-jnl-abbrev">Biochem. Biophys. Res. Commun.</abbr> <span class="cit-vol">363</span>: <span class="cit-fpage">183</span>â<span class="cit-lpage">189</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochemical%2Band%2Bbiophysical%2Bresearch%2Bcommunications%26rft.stitle%253DBiochem%2BBiophys%2BRes%2BCommun%26rft.aulast%253DJung%26rft.auinit1%253DU.%2BJ.%26rft.volume%253D363%26rft.issue%253D1%26rft.spage%253D183%26rft.epage%253D189%26rft.atitle%253DAnalysis%2Bof%2Bspacer%2Bregions%2Bderived%2Bfrom%2Bintramolecular%2Brecombination%2Bbetween%2Bheterologous%2BloxP%2Bsites.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbrc.2007.08.145%26rft_id%253Dinfo%253Apmid%252F17826735%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbrc.2007.08.145&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17826735&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.21" data-doi="10.1093/nar/gks1037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karimova</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abi-Ghanem</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Berger</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Surendranath</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pisabarro</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Vika/vox, a novel efficient and specific Cre/loxP-like site-specific recombination system.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">e37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgks1037%26rft_id%253Dinfo%253Apmid%252F23143104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjQxLzIvZTM3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzU1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.22" data-doi="10.1093/nar/gkf421"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langer</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghafoori</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Byrd</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leinwand</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span>â<span class="cit-article-title">A genetic screen identifies novel non-compatible loxP sites.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">3067</span>â<span class="cit-lpage">3077</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DLanger%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D30%26rft.issue%253D14%26rft.spage%253D3067%26rft.epage%253D3077%26rft.atitle%253DA%2Bgenetic%2Bscreen%2Bidentifies%2Bnovel%2Bnon-compatible%2BloxP%2Bsites%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkf421%26rft_id%253Dinfo%253Apmid%252F12136089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMC8xNC8zMDY3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzU1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.23" data-doi="10.1016/S0378-1119(98)00325-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saito</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span>â<span class="cit-article-title">Role of nucleotide sequences of loxP spacer region in Cre-mediated recombination.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">216</span>: <span class="cit-fpage">55</span>â<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.stitle%253DGene%26rft.aulast%253DLee%26rft.auinit1%253DG.%26rft.volume%253D216%26rft.issue%253D1%26rft.spage%253D55%26rft.epage%253D65%26rft.atitle%253DRole%2Bof%2Bnucleotide%2Bsequences%2Bof%2BloxP%2Bspacer%2Bregion%2Bin%2BCre-mediated%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0378-1119%252898%252900325-4%26rft_id%253Dinfo%253Apmid%252F9714735%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0378-1119(98)00325-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9714735&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000075606600007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.24" data-doi="10.1038/nature06293"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Livet</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weissman</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Draft</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Transgenic strategies for combinatorial expression of fluorescent proteins in the nervous system.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">450</span>: <span class="cit-fpage">56</span>â<span class="cit-lpage">62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DLivet%26rft.auinit1%253DJ.%26rft.volume%253D450%26rft.issue%253D7166%26rft.spage%253D56%26rft.epage%253D62%26rft.atitle%253DTransgenic%2Bstrategies%2Bfor%2Bcombinatorial%2Bexpression%2Bof%2Bfluorescent%2Bproteins%2Bin%2Bthe%2Bnervous%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06293%26rft_id%253Dinfo%253Apmid%252F17972876%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06293&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17972876&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250585800035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.25" data-doi="10.1006/dbio.1999.9209"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lobe</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koop</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kreppner</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lomeli</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gertsenstein</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span>â<span class="cit-article-title">Z/AP, a double reporter for Cre-mediated recombination.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">208</span>: <span class="cit-fpage">281</span>â<span class="cit-lpage">292</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DLobe%26rft.auinit1%253DC.%2BG.%26rft.volume%253D208%26rft.issue%253D2%26rft.spage%253D281%26rft.epage%253D292%26rft.atitle%253DZ%252FAP%252C%2Ba%2Bdouble%2Breporter%2Bfor%2Bcre-mediated%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fdbio.1999.9209%26rft_id%253Dinfo%253Apmid%252F10191045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/dbio.1999.9209&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10191045&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000079923200004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.26" data-doi="10.1016/j.neuron.2015.02.022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Madisen</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garner</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shimaoka</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chuong</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klapoetke</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Transgenic mice for intersectional targeting of neural sensors and effectors with high specificity and performance.</span> <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">942</span>â<span class="cit-lpage">958</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.volume%253D85%26rft.spage%253D942%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.neuron.2015.02.022%26rft_id%253Dinfo%253Apmid%252F25741722%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.neuron.2015.02.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25741722&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.27" data-doi="10.1186/1471-2164-7-73"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Missirlis</span>  <span class="cit-name-given-names">P. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smailus</span>  <span class="cit-name-given-names">D. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">A high-throughput screen identifying sequence and promiscuity characteristics of the loxP spacer region in Cre-mediated recombination.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">73</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMissirlis%26rft.auinit1%253DP.%2BI.%26rft.volume%253D7%26rft.spage%253D73%26rft.epage%253D73%26rft.atitle%253DA%2Bhigh-throughput%2Bscreen%2Bidentifying%2Bsequence%2Band%2Bpromiscuity%2Bcharacteristics%2Bof%2Bthe%2BloxP%2Bspacer%2Bregion%2Bin%2BCre-mediated%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-7-73%26rft_id%253Dinfo%253Apmid%252F16595017%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-7-73&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16595017&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.28" data-doi="10.1007/978-1-59745-471-1_19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagy</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mar</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watts</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Creation and use of a Cre recombinase transgenic database.</span> <abbr class="cit-jnl-abbrev">Methods Mol. Biol.</abbr> <span class="cit-vol">530</span>: <span class="cit-fpage">365</span>â<span class="cit-lpage">378</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Bmolecular%2Bbiology%2B%2528Clifton%252C%2BN.J.%2529%26rft.stitle%253DMethods%2BMol%2BBiol%26rft.aulast%253DNagy%26rft.auinit1%253DA.%26rft.volume%253D530%26rft.spage%253D365%26rft.epage%253D378%26rft.atitle%253DCreation%2Band%2Buse%2Bof%2Ba%2Bcre%2Brecombinase%2Btransgenic%2Bdatabase.%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-59745-471-1_19%26rft_id%253Dinfo%253Apmid%252F19266338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-59745-471-1_19&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19266338&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.29" data-doi="10.1073/pnas.1111704108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nern</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pfeiffer</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Svoboda</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">G. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Multiple new site-specific recombinases for use in manipulating animal genomes.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">14198</span>â<span class="cit-lpage">14203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1111704108%26rft_id%253Dinfo%253Apmid%252F21831835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzM0LzE0MTk4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzU1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.30" data-doi="10.1371/journal.pone.0085218"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leach</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">TAILOR: transgene activation and inactivation using lox and rox in zebrafish.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e85218</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De85218%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0085218%26rft_id%253Dinfo%253Apmid%252F24391998%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0085218&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24391998&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.31" data-doi="10.1371/journal.pone.0091435"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sajgo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ghinia</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Dre - Cre sequential recombination provides new tools for retinal ganglion cell labeling and manipulation in mice.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e91435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De91435%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0091435%26rft_id%253Dinfo%253Apmid%252F24608965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0091435&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24608965&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.32" data-doi="10.1073/pnas.85.14.5166"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sauer</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span>â<span class="cit-article-title">Site-specific DNA recombination in mammalian cells by the Cre recombinase of bacteriophage P1.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">5166</span>â<span class="cit-lpage">5170</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DSauer%26rft.auinit1%253DB.%26rft.volume%253D85%26rft.issue%253D14%26rft.spage%253D5166%26rft.epage%253D5170%26rft.atitle%253DSite-specific%2BDNA%2Brecombination%2Bin%2Bmammalian%2Bcells%2Bby%2Bthe%2BCre%2Brecombinase%2Bof%2Bbacteriophage%2BP1.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.85.14.5166%26rft_id%253Dinfo%253Apmid%252F2839833%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODUvMTQvNTE2NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMy81NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.33" data-doi="10.1093/nar/gkh941"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sauer</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McDermott</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span>â<span class="cit-article-title">DNA recombination with a heterospecific Cre homolog identified from comparison of the pac-c1 regions of P1-related phages.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">6086</span>â<span class="cit-lpage">6095</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DSauer%26rft.auinit1%253DB.%26rft.volume%253D32%26rft.issue%253D20%26rft.spage%253D6086%26rft.epage%253D6095%26rft.atitle%253DDNA%2Brecombination%2Bwith%2Ba%2Bheterospecific%2BCre%2Bhomolog%2Bidentified%2Bfrom%2Bcomparison%2Bof%2Bthe%2Bpac-c1%2Bregions%2Bof%2BP1-related%2Bphages%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh941%26rft_id%253Dinfo%253Apmid%252F15550568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjEwOiIzMi8yMC82MDg2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzU1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.34" data-doi="10.1038/nbt811"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schnutgen</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doerflinger</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calleja</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wendling</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chambon</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">A directional strategy for monitoring Cre-mediated recombination at the cellular level in the mouse.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">562</span>â<span class="cit-lpage">565</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DSchn%25C3%25BCtgen%26rft.auinit1%253DF.%26rft.volume%253D21%26rft.issue%253D5%26rft.spage%253D562%26rft.epage%253D565%26rft.atitle%253DA%2Bdirectional%2Bstrategy%2Bfor%2Bmonitoring%2BCre-mediated%2Brecombination%2Bat%2Bthe%2Bcellular%2Blevel%2Bin%2Bthe%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt811%26rft_id%253Dinfo%253Apmid%252F12665802%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt811&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12665802&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182660300035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.35" data-doi="10.1016/S0014-5793(01)02541-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Siegel</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jain</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span>â<span class="cit-article-title">Using an in vivo phagemid system to identify non-compatible loxP sequences.</span> <abbr class="cit-jnl-abbrev">FEBS Lett.</abbr> <span class="cit-vol">499</span>: <span class="cit-fpage">147</span>â<span class="cit-lpage">153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BLetters%26rft.stitle%253DFEBS%2BLetters%26rft.aulast%253DSiegel%26rft.auinit1%253DR.%2BW.%26rft.volume%253D499%26rft.issue%253D1-2%26rft.spage%253D147%26rft.epage%253D153%26rft.atitle%253DUsing%2Ban%2Bin%2Bvivo%2Bphagemid%2Bsystem%2Bto%2Bidentify%2Bnon-compatible%2BloxP%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0014-5793%252801%252902541-8%26rft_id%253Dinfo%253Apmid%252F11418130%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0014-5793(01)02541-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11418130&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169549100031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.36" data-doi="10.1093/nar/gkq1280"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Suzuki</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakayama</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">VCre/VloxP and SCre/SloxP: new site-specific recombination systems for genome engineering.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">e49</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq1280%26rft_id%253Dinfo%253Apmid%252F21288882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjM5LzgvZTQ5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzU1OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.37" data-doi="10.1016/j.gene.2012.11.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turan</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zehe</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuehle</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qiao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bode</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Recombinase-mediated cassette exchange (RMCE) - a rapidly-expanding toolbox for targeted genomic modifications.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">515</span>: <span class="cit-fpage">1</span>â<span class="cit-lpage">27</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.volume%253D515%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gene.2012.11.016%26rft_id%253Dinfo%253Apmid%252F23201421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gene.2012.11.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23201421&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.38" data-doi="10.1007/s00253-011-3633-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vick</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">E. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choudhary</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bloch</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopez-Gallego</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Optimized compatible set of BioBrick vectors for metabolic pathway engineering.</span> <abbr class="cit-jnl-abbrev">Appl. Microbiol. Biotechnol.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">1275</span>â<span class="cit-lpage">1286</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DApplied%2BMicrobiology%2Band%2BBiotechnology%26rft.stitle%253DApplied%2BMicrobiology%2Band%2BBiotechnology%26rft.aulast%253DVick%26rft.auinit1%253DJ.%2BE.%26rft.volume%253D92%26rft.issue%253D6%26rft.spage%253D1275%26rft.epage%253D1286%26rft.atitle%253DOptimized%2Bcompatible%2Bset%2Bof%2BBioBrickTM%2Bvectors%2Bfor%2Bmetabolic%2Bpathway%2Bengineering.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00253-011-3633-4%26rft_id%253Dinfo%253Apmid%252F22033566%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00253-011-3633-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22033566&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F559.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">âµ</a><div class="cit ref-cit ref-journal" id="cit-6.3.559.39" data-doi="10.1016/S0960-9822(98)70255-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zufferey</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donello</span>  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trono</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hope</span>  <span class="cit-name-given-names">T. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span>â<span class="cit-article-title">Woodchuck hepatitis virus posttranscriptional regulatory element enhances expression of transgenes delivered by retroviral vectors.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">73</span>: <span class="cit-fpage">2886</span>â<span class="cit-lpage">2892</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BVirology%26rft.stitle%253DJ.%2BVirol.%26rft.aulast%253DZufferey%26rft.auinit1%253DR.%26rft.volume%253D73%26rft.issue%253D4%26rft.spage%253D2886%26rft.epage%253D2892%26rft.atitle%253DWoodchuck%2BHepatitis%2BVirus%2BPosttranscriptional%2BRegulatory%2BElement%2BEnhances%2BExpression%2Bof%2BTransgenes%2BDelivered%2Bby%2BRetroviral%2BVectors%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0960-9822%252898%252970255-6%26rft_id%253Dinfo%253Apmid%252F10074136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjk6IjczLzQvMjg4NiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMy81NTkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/3/559.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/3/551" title="Isolation of a Hypomorphic skn-1 Allele That Does Not Require a Balancer for Maintenance" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/3/573" title="A Novel Recombinant DNA System for High Efficiency Affinity Purification of Proteins in Saccharomyces cerevisiae" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/3"><span>Volume 6 Issue 3, March 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/3.cover-source.jpg?itok=8EDC4x5t" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/3/559.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-9J9BUtKd3LLSDKlRSHb5nAu3sCiodrJDIeEAw47WKQE" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/3/559" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/3/559" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/3/559" class="active" data-icon-position="" data-hide-link-title="0">Novel Heterotypic Rox Sites for Combinatorial Dre Recombination Strategies</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460078" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-cq3f8Wk0pSKPEpUcddtGxApCdWUcK9dstj8L0e4B41c" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/3/559.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/3/559" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Vl8cadO1XCallw8hbSJPrdrDjTcgk2L0A-dtz6MCdYk" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/3/559" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/3/559.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460078" id="node460078--2" data-pisa="ggg;6/3/559" data-pisa-master="ggg;g3.115.025841" data-apath="/ggg/6/3/559.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Novel Heterotypic Rox Sites for Combinatorial Dre Recombination Strategies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Katherine Chuang</span>, <span class="highwire-citation-author" data-delta="1">Eileen Nguyen</span>, <span class="highwire-citation-author" data-delta="2">Yuri Sergeev</span> and <span class="highwire-citation-author" data-delta="3">Tudor C. Badea</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">559-571;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.025841 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460078--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Katherine Chuang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Retinal Circuits Development and Genetics Unit, National Eye Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katherine%2BChuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chuang%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKatherine%2BChuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Eileen Nguyen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Retinal Circuits Development and Genetics Unit, National Eye Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eileen%2BNguyen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nguyen%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEileen%2BNguyen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yuri Sergeev </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Retinal Circuits Development and Genetics Unit, National Eye Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yuri%2BSergeev%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sergeev%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYuri%2BSergeev%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Tudor C. Badea </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Retinal Circuits Development and Genetics Unit, National Eye Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tudor%2BC.%2BBadea%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Badea%20TC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATudor%2BC.%2BBadea%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:badeatc@mail.nih.gov" class="" data-icon-position="" data-hide-link-title="0">badeatc@mail.nih.gov</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/3/559" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/3/559&amp;title=Novel%20Heterotypic%20Rox%20Sites%20for%20Combinatorial%20Dre%20Recombination%20Strategies" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/3/559&amp;title=Novel%20Heterotypic%20Rox%20Sites%20for%20Combinatorial%20Dre%20Recombination%20Strategies" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/3/559&amp;title=Novel%20Heterotypic%20Rox%20Sites%20for%20Combinatorial%20Dre%20Recombination%20Strategies" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/3/559&amp;text=Novel%20Heterotypic%20Rox%20Sites%20for%20Combinatorial%20Dre%20Recombination%20Strategies" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/3/559&amp;title=Novel%20Heterotypic%20Rox%20Sites%20for%20Combinatorial%20Dre%20Recombination%20Strategies" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/3/559&amp;t=Novel%20Heterotypic%20Rox%20Sites%20for%20Combinatorial%20Dre%20Recombination%20Strategies" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/3/559&amp;title=Novel%20Heterotypic%20Rox%20Sites%20for%20Combinatorial%20Dre%20Recombination%20Strategies" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/3/559&amp;title=Novel%20Heterotypic%20Rox%20Sites%20for%20Combinatorial%20Dre%20Recombination%20Strategies" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460078/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460078" id="node460078--3" data-pisa="ggg;6/3/559" data-pisa-master="ggg;g3.115.025841" data-apath="/ggg/6/3/559.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Novel Heterotypic Rox Sites for Combinatorial Dre Recombination Strategies</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Katherine Chuang</span>, <span class="highwire-citation-author" data-delta="1">Eileen Nguyen</span>, <span class="highwire-citation-author" data-delta="2">Yuri Sergeev</span> and <span class="highwire-citation-author" data-delta="3">Tudor C. Badea</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">559-571;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.025841 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460078--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Katherine Chuang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Retinal Circuits Development and Genetics Unit, National Eye Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katherine%2BChuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chuang%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKatherine%2BChuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Eileen Nguyen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Retinal Circuits Development and Genetics Unit, National Eye Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Eileen%2BNguyen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nguyen%20E&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AEileen%2BNguyen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yuri Sergeev </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Retinal Circuits Development and Genetics Unit, National Eye Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yuri%2BSergeev%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sergeev%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYuri%2BSergeev%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Tudor C. Badea </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Retinal Circuits Development and Genetics Unit, National Eye Institute, National Institutes of Health, Bethesda, Maryland 20892</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tudor%2BC.%2BBadea%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Badea%20TC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ATudor%2BC.%2BBadea%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:badeatc@mail.nih.gov" class="" data-icon-position="" data-hide-link-title="0">badeatc@mail.nih.gov</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=559&amp;rft.epage=571&amp;rft.atitle=Novel%20Heterotypic%20Rox%20Sites%20for%20Combinatorial%20Dre%20Recombination%20Strategies&amp;rft.volume=6&amp;rft.issue=3&amp;rft.date=2016-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Chuang%2C+Katherine&amp;rft.au=Nguyen%2C+Eileen&amp;rft.au=Sergeev%2C+Yuri&amp;rft.au=Badea%2C+Tudor+C."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460078/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460078/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460078/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460078/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460078/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460078/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460078/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460078/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460078/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460078/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460078/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460078/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/3/559" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-8" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/3/559.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/3/559.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/3/559.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/3/559.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
