<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/3/509" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/3/509.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/3/509.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/3/509.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/3/509.atom" />
<meta name="HW.pisa" content="ggg;6/3/509" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Exploring a Nonmodel Teleost Genome Through RAD Sequencing—Linkage Mapping in Common Pandora, Pagellus erythrinus and Comparative Genomic Analysis" />
<meta name="DC.Identifier" content="10.1534/g3.115.023432" />
<meta name="DC.Date" content="2016-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Manousaki et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Common pandora ( Pagellus erythrinus ) is a benthopelagic marine fish belonging to the teleost family Sparidae, and a newly recruited species in Mediterranean aquaculture. The paucity of genetic information relating to sparids, despite their growing economic value for aquaculture, provides the impetus for exploring the genomics of this fish group. Genomic tool development, such as genetic linkage maps provision, lays the groundwork for linking genotype to phenotype, allowing fine-mapping of loci responsible for beneficial traits. In this study, we applied ddRAD methodology to identify polymorphic markers in a full-sib family of common pandora. Employing the Illumina MiSeq platform, we sampled and sequenced a size-selected genomic fraction of 99 individuals, which led to the identification of 920 polymorphic loci. Downstream mapping analysis resulted in the construction of 24 robust linkage groups, corresponding to the karyotype of the species. The common pandora linkage map showed varying degrees of conserved synteny with four other teleost genomes, namely the European seabass ( Dicentrarchus labrax ), Nile tilapia ( Oreochromis niloticus ), stickleback ( Gasterosteus aculeatus ), and medaka ( Oryzias latipes ), suggesting a conserved genomic evolution in Sparidae. Our work exploits the possibilities of genotyping by sequencing to gain novel insights into genome structure and evolution. Such information will boost the study of cultured species and will set the foundation for a deeper understanding of the complex evolutionary history of teleosts." />
<meta name="DC.Contributor" content="Tereza Manousaki" />
<meta name="DC.Contributor" content="Alexandros Tsakogiannis" />
<meta name="DC.Contributor" content="John B. Taggart" />
<meta name="DC.Contributor" content="Christos Palaiokostas" />
<meta name="DC.Contributor" content="Dimitris Tsaparis" />
<meta name="DC.Contributor" content="Jacques Lagnel" />
<meta name="DC.Contributor" content="Dimitrios Chatziplis" />
<meta name="DC.Contributor" content="Antonios Magoulas" />
<meta name="DC.Contributor" content="Nikos  Papandroulakis" />
<meta name="DC.Contributor" content="Constantinos C. Mylonas" />
<meta name="DC.Contributor" content="Costas S. Tsigenopoulos" />
<meta name="article:published_time" content="2016-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Exploring a Nonmodel Teleost Genome Through RAD Sequencing—Linkage Mapping in Common Pandora, Pagellus erythrinus and Comparative Genomic Analysis" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Common pandora (&lt;i&gt;Pagellus erythrinus&lt;/i&gt;) is a benthopelagic marine fish belonging to the teleost family Sparidae, and a newly recruited species in Mediterranean aquaculture. The paucity of genetic information relating to sparids, despite their growing economic value for aquaculture, provides the impetus for exploring the genomics of this fish group. Genomic tool development, such as genetic linkage maps provision, lays the groundwork for linking genotype to phenotype, allowing fine-mapping of loci responsible for beneficial traits. In this study, we applied ddRAD methodology to identify polymorphic markers in a full-sib family of common pandora. Employing the Illumina MiSeq platform, we sampled and sequenced a size-selected genomic fraction of 99 individuals, which led to the identification of 920 polymorphic loci. Downstream mapping analysis resulted in the construction of 24 robust linkage groups, corresponding to the karyotype of the species. The common pandora linkage map showed varying degrees of conserved synteny with four other teleost genomes, namely the European seabass (&lt;i&gt;Dicentrarchus labrax&lt;/i&gt;), Nile tilapia (&lt;i&gt;Oreochromis niloticus&lt;/i&gt;), stickleback (&lt;i&gt;Gasterosteus aculeatus&lt;/i&gt;), and medaka (&lt;i&gt;Oryzias latipes&lt;/i&gt;), suggesting a conserved genomic evolution in Sparidae. Our work exploits the possibilities of genotyping by sequencing to gain novel insights into genome structure and evolution. Such information will boost the study of cultured species and will set the foundation for a deeper understanding of the complex evolutionary history of teleosts.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/03/01" />
<meta name="citation_mjid" content="ggg;6/3/509" />
<meta name="citation_id" content="6/3/509" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/3/509" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/3/509.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/3/509.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/3/509.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.023432" />
<meta name="citation_pmid" content="26715088" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="509" />
<meta name="citation_lastpage" content="519" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Tereza Manousaki" />
<meta name="citation_author_institution" content="Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece" />
<meta name="citation_author" content="Alexandros Tsakogiannis" />
<meta name="citation_author_institution" content="Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece" />
<meta name="citation_author_institution" content="Department of Biology, University of Crete, Heraklion, 70013, Greece" />
<meta name="citation_author" content="John B. Taggart" />
<meta name="citation_author_institution" content="Institute of Aquaculture, School of Natural Sciences, University of Stirling, Scotland, FK9 4LA, United Kingdom" />
<meta name="citation_author" content="Christos Palaiokostas" />
<meta name="citation_author_institution" content="Institute of Aquaculture, School of Natural Sciences, University of Stirling, Scotland, FK9 4LA, United Kingdom" />
<meta name="citation_author" content="Dimitris Tsaparis" />
<meta name="citation_author_institution" content="Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece" />
<meta name="citation_author" content="Jacques Lagnel" />
<meta name="citation_author_institution" content="Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece" />
<meta name="citation_author" content="Dimitrios Chatziplis" />
<meta name="citation_author_institution" content="Laboratory of Agrobiotechnology and Inspection of Agricultural Products, Department of Agricultural Technology, School of Agricultural Technology Food Technology and Nutrition, Alexander Technological Educational Institute of Thessaloniki, Sindos, 57400, Greece" />
<meta name="citation_author" content="Antonios Magoulas" />
<meta name="citation_author_institution" content="Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece" />
<meta name="citation_author" content="Nikos  Papandroulakis" />
<meta name="citation_author_institution" content="Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece" />
<meta name="citation_author" content="Constantinos C. Mylonas" />
<meta name="citation_author_institution" content="Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece" />
<meta name="citation_author" content="Costas S. Tsigenopoulos" />
<meta name="citation_author_institution" content="Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece" />
<meta name="citation_author_email" content="tsigeno@hcmr.gr" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. Amores;citation_author=J. Catchen;citation_author=A. Ferrara;citation_author=Q. Fontenot;citation_author=J. H. Postlethwait;citation_title=Genome Evolution and Meiotic Maps by Massively Parallel DNA Sequencing: Spotted Gar, an Outgroup for the Teleost Genome Duplication;citation_pages=799-808;citation_volume=188;citation_year=2011;citation_issue=4;citation_pmid=21828280;citation_doi=10.1534/genetics.111.127324" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Amores;citation_author=J. Catchen;citation_author=I. Nanda;citation_author=W. Warren;citation_author=R. Walter;citation_title=A RAD-tag genetic map for the platyfish (Xiphophorus maculatus) reveals mechanisms of karyotype evolution among teleost fish.;citation_volume=197;citation_year=2014;citation_pmid=24700104;citation_doi=10.1534/genetics.114.164293" />
<meta name="citation_reference" content="citation_journal_title=Animal Behaviour;citation_journal_abbrev=Animal Behaviour;citation_title=Guidelines for the treatment of animals in behavioural research and teaching.;citation_pages=271-275;citation_volume=61;citation_year=2001;citation_issue=1;citation_pmid=11170716;citation_doi=10.1006/anbe.2000.1652" />
<meta name="citation_reference" content="citation_journal_title=Biologia;citation_author=A. Apostolidis;citation_author=K. Moutou;citation_author=C. Stamatis;citation_author=Z. Mamuris;citation_title=Genetic structure of three marine fishes from the Gulf of Pagasitikos (Greece) based on allozymes, RAPD, and mtDNA RFLP markers.;citation_pages=1005-1010;citation_volume=64;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=N. A. Baird;citation_author=P. D. Etter;citation_author=T. S. Atwood;citation_author=M. C. Currey;citation_author=A. L. Shiver;citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers.;citation_pages=e3376-e3376;citation_volume=3;citation_year=2008;citation_issue=10;citation_pmid=18852878;citation_doi=10.1371/journal.pone.0003376" />
<meta name="citation_reference" content="citation_title=Sparidae: Biology and Aquaculture of Gilthead Sea Bream and Other Species,;citation_title=Current status of Sparidae aquaculture,;citation_author=B. Basurco;citation_author=A. Lovatelli;citation_author=B. Garcia;citation_pages=1-50;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=PLoS Curr.;citation_title=The tree of life and a new classification of bony fishes.;citation_author=R. R. Betancur;citation_author=R. E. Broughton;citation_author=E. O. Wiley;citation_author=K. Carpenter;citation_author=J. A. Lopez;citation_volume=5;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Dev. Evol.;citation_author=I. Braasch;citation_author=S. M. Peterson;citation_author=T. Desvignes;citation_author=B. M. McCluskey;citation_author=P. Batzel;citation_title=A new model army: Emerging fish models to study the genomics of vertebrate Evo-Devo. J. Exp. Zool. B Mol.;citation_pages=316-341;citation_volume=324;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=K. Boulton;citation_author=C. Massault;citation_author=R. D. Houston;citation_author=D. J. de Koning;citation_author=C. S. Haley;citation_title=QTL affecting morphometric traits and stress response in the gilthead seabream (Sparus aurata).;citation_pages=58-66;citation_volume=319;citation_year=2011;citation_doi=10.1016/j.aquaculture.2011.06.044" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=N. R. Campbell;citation_author=S. A. Harmon;citation_author=S. R. Narum;citation_title=Genotyping-in-Thousands by sequencing (GT-seq): a cost effective SNP genotyping method based on custom amplicon sequencing.;citation_pages=855-867;citation_volume=15;citation_year=2015;citation_pmid=25476721;citation_doi=10.1111/1755-0998.12357" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=J. Castresana;citation_title=Selection of Conserved Blocks from Multiple Alignments for Their Use in Phylogenetic Analysis;citation_pages=540-552;citation_volume=17;citation_year=2000;citation_issue=4;citation_pmid=10742046;citation_doi=10.1093/oxfordjournals.molbev.a026334" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=S. Cataudella;citation_author=P. P. Riz;citation_author=L. Sola;citation_title=A chromosome study of eight Mediterranean species of Sparidae (Pisces, Perciformes).;citation_pages=155-159;citation_volume=54;citation_year=1980;citation_doi=10.1007/BF00055985" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=J. M. Catchen;citation_author=A. Amores;citation_author=P. Hohenlohe;citation_author=W. Cresko;citation_author=J. H. Postlethwait;citation_title=Stacks: Building and Genotyping Loci De Novo From Short-Read Sequences;citation_pages=171-182;citation_volume=1;citation_year=2011;citation_issue=3;citation_pmid=22384329;citation_doi=10.1534/g3.111.000240" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=D. Darriba;citation_author=G. L. Taboada;citation_author=R. Doallo;citation_author=D. Posada;citation_title=jModelTest 2: more models, new heuristics and parallel computing.;citation_pages=772-772;citation_volume=9;citation_year=2012;citation_issue=8;citation_pmid=22847109;citation_doi=10.1038/nmeth.2109" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Integr. Comp. Biol.;citation_author=B. E. Erisman;citation_author=C. W. Petersen;citation_author=P. A. Hastings;citation_author=R. R. Warner;citation_title=Phylogenetic perspectives on the evolution of functional hermaphroditism in teleost fishes.;citation_volume=53;citation_year=2013;citation_pmid=23817661;citation_doi=10.1093/icb/ict077" />
<meta name="citation_reference" content="citation_journal_title=Ichthyol. Res.;citation_author=C. Fassatoui;citation_author=E. Mdelgi;citation_author=M. Romdhane;citation_title=A preliminary investigation of allozyme genetic variation and population structure in common pandora (Pagellus erythrinus, Sparidae) from Tunisian and Libyan coasts.;citation_pages=301-307;citation_volume=56;citation_year=2009;citation_doi=10.1007/s10228-008-0094-6" />
<meta name="citation_reference" content="citation_journal_title=Mar. Ecol. Prog. Ser.;citation_author=C. Fassatoui;citation_author=A. Chenuil;citation_author=M. S. Romdhane;citation_title=Relationships between heterozygosity, growth parameters and age in the common pandora Pagellus erythrinus (Sparidae) in the Gabes Gulf (Tunisia).;citation_pages=251-261;citation_volume=445;citation_year=2012;citation_doi=10.3354/meps09531" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=R. Franch;citation_author=B. Louro;citation_author=M. Tsalavouta;citation_author=D. Chatziplis;citation_author=C. S. Tsigenopoulos;citation_title=A genetic linkage map of the hermaphrodite teleost fish Sparus aurata L.;citation_volume=174;citation_year=2006;citation_pmid=16951080;citation_doi=10.1534/genetics.106.059014" />
<meta name="citation_reference" content="citation_journal_title=Hydrobiologia;citation_author=P. M. Galetti;citation_author=C. T. Aguilar;citation_author=W. F. Molina;citation_title=An overview of marine fish cytogenetics.;citation_pages=55-62;citation_volume=420;citation_year=2000;citation_doi=10.1023/A:1003977418900" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=S. Guindon;citation_author=O. Gascuel;citation_title=A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood.;citation_volume=52;citation_year=2003;citation_pmid=14530136;citation_doi=10.1080/10635150390235520" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=R. Guyon;citation_author=M. Rakotomanga;citation_author=N. Azzouzi;citation_author=J. P. Coutanceau;citation_author=C. Bonillo;citation_title=A high-resolution map of the Nile tilapia genome: a resource for studying cichlids and other percomorphs.;citation_pages=222-222;citation_volume=13;citation_year=2012;citation_pmid=22672252;citation_doi=10.1186/1471-2164-13-222" />
<meta name="citation_reference" content="citation_title=Sparidae: Biology and Aquaculture of Gilthead Sea Bream and Other Species,;citation_title=Phylogeny, evolution and taxonomy of sparids with some notes on their ecology and biology,;citation_author=R. Hanel;citation_author=C. S. Tsigenopoulos;citation_pages=51-73;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=W. Kai;citation_author=K. Nomura;citation_author=A. Fujiwara;citation_author=Y. Nakamura;citation_author=M. Yasuike;citation_title=A ddRAD-based genetic map and its integration with the genome assembly of Japanese eel (Anguilla japonica) provides insights into genome evolution after the teleost-specific genome duplication.;citation_pages=233;citation_volume=15;citation_year=2014;citation_pmid=24669946;citation_doi=10.1186/1471-2164-15-233" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=R. Kakioka;citation_author=T. Kokita;citation_author=H. Kumada;citation_author=K. Watanabe;citation_author=N. Okuda;citation_title=A RAD-based linkage map and comparative genomics in the gudgeons (genus Gnathopogon, Cyprinidae).;citation_pages=32;citation_volume=14;citation_year=2013;citation_pmid=23324215;citation_doi=10.1186/1471-2164-14-32" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Katoh;citation_author=D. M. Standley;citation_title=MAFFT Multiple sequence alignment software version 7: Improvements in performance and usability.;citation_volume=30;citation_year=2013;citation_pmid=23329690;citation_doi=10.1093/molbev/mst010" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=M. Krzywinski;citation_author=J. Schein;citation_author=I. Birol;citation_author=J. Connors;citation_author=R. Gascoyne;citation_title=Circos: an information aesthetic for comparative genomics.;citation_volume=19;citation_year=2009;citation_pmid=19541911;citation_doi=10.1101/gr.092759.109" />
<meta name="citation_reference" content="citation_journal_abbrev=J Biomed Biotechnol;citation_author=H. Kuhl;citation_author=E. Sarropoulou;citation_author=M. Tine;citation_author=G. Kotoulas;citation_author=A. Magoulas;citation_title=A Comparative BAC map for the gilthead sea bream (Sparus aurata L.).;citation_pages=329025-329025;citation_volume=2011;citation_year=2011;citation_pmid=21049003" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=D. Loukovitis;citation_author=E. Sarropoulou;citation_author=C. S. Tsigenopoulos;citation_author=C. Batargias;citation_author=A. Magoulas;citation_title=Quantitative trait loci involved in sex determination and body growth in the gilthead sea bream (Sparus aurata L.) through targeted genome scan.;citation_pages=e16599-e16599;citation_volume=6;citation_year=2011;citation_issue=1;citation_pmid=21304996;citation_doi=10.1371/journal.pone.0016599" />
<meta name="citation_reference" content="citation_journal_title=Animal genetics;citation_journal_abbrev=Anim Genet;citation_author=D. Loukovitis;citation_author=E. Sarropoulou;citation_author=C. Batargias;citation_author=A. P. Apostolidis;citation_author=G. Kotoulas;citation_title=Quantitative trait loci for body growth and sex determination in the hermaphrodite teleost fish Sparus aurata L.;citation_pages=753-759;citation_volume=43;citation_year=2012;citation_issue=6;citation_pmid=22497460;citation_doi=10.1111/j.1365-2052.2012.02346.x" />
<meta name="citation_reference" content="citation_journal_title=Anim. Genet.;citation_author=D. Loukovitis;citation_author=C. Batargias;citation_author=E. Sarropoulou;citation_author=A. P. Apostolidis;citation_author=G. Kotoulas;citation_title=Quantitative trait loci affecting morphology traits in gilthead seabream (Sparus aurata L.).;citation_pages=480-483;citation_volume=44;citation_year=2013;citation_pmid=23418819;citation_doi=10.1111/age.12027" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=T. Manousaki;citation_author=A. Tsakogiannis;citation_author=J. Lagnel;citation_author=E. Sarropoulou;citation_author=J. Z. Xiang;citation_title=The sex-specific transcriptome of the hermaphrodite sparid sharpsnout seabream (Diplodus puntazzo).;citation_pages=655;citation_volume=15;citation_year=2014;citation_pmid=25099474;citation_doi=10.1186/1471-2164-15-655" />
<meta name="citation_reference" content="citation_journal_title=Hereditas;citation_journal_abbrev=Hereditas;citation_author=G. R. Margarido;citation_author=A. P. Souza;citation_author=A. A. Garcia;citation_title=OneMap: software for genetic mapping in outcrossing species.;citation_pages=78-79;citation_volume=144;citation_year=2007;citation_issue=3;citation_pmid=17663699;citation_doi=10.1111/j.2007.0018-0661.02000.x" />
<meta name="citation_reference" content="citation_journal_title=Anim. Genet.;citation_author=C. Massault;citation_author=R. Franch;citation_author=C. Haley;citation_author=D. J. de Koning;citation_author=H. Bovenhuis;citation_title=Quantitative trait loci for resistance to fish pasteurellosis in gilthead sea bream (Sparus aurata).;citation_pages=191-203;citation_volume=42;citation_year=2011;citation_pmid=20946317;citation_doi=10.1111/j.1365-2052.2010.02110.x" />
<meta name="citation_reference" content="citation_title=Sparidae: Biology and Aquaculture of Gilthead Sea Bream and Related Species;citation_title=Reproduction and broodstock management;citation_author=C. C. Mylonas;citation_author=Y. Zohar;citation_author=N. W. Pankhurst;citation_author=H. Kagawa;citation_pages=95-131;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=K. Naruse;citation_author=M. Tanaka;citation_author=K. Mita;citation_author=A. Shima;citation_author=J. Postlethwait;citation_title=A medaka gene map: the trace of ancestral vertebrate proto-chromosomes revealed by comparative gene mapping.;citation_volume=14;citation_year=2004;citation_pmid=15078856;citation_doi=10.1101/gr.2004004" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=K. E. O’Quin;citation_author=M. Yoshizawa;citation_author=P. Doshi;citation_author=W. R. Jeffery;citation_title=Quantitative genetic analysis of retinal degeneration in the blind cavefish Astyanax mexicanus.;citation_pages=e57281;citation_volume=8;citation_year=2013;citation_pmid=23437360;citation_doi=10.1371/journal.pone.0057281" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=C. Palaiokostas;citation_author=M. Bekaert;citation_author=A. Davie;citation_author=M. E. Cowan;citation_author=M. Oral;citation_title=Mapping the sex determination locus in the Atlantic halibut (Hippoglossus hippoglossus) using RAD sequencing.;citation_pages=566;citation_volume=14;citation_year=2013;citation_pmid=23957753;citation_doi=10.1186/1471-2164-14-566" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=C. Palaiokostas;citation_author=M. Bekaert;citation_author=M. G. Khan;citation_author=J. B. Taggart;citation_author=K. Gharbi;citation_title=Mapping and validation of the major sex-determining region in Nile tilapia (Oreochromis niloticus L.) using RAD Sequencing.;citation_pages=e68389;citation_volume=8;citation_year=2013;citation_pmid=23874606;citation_doi=10.1371/journal.pone.0068389" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=C. Palaiokostas;citation_author=M. Bekaert;citation_author=M. G. Khan;citation_author=J. B. Taggart;citation_author=K. Gharbi;citation_title=A novel sex-determining QTL in Nile tilapia (Oreochromis niloticus).;citation_pages=171;citation_volume=16;citation_year=2015;citation_pmid=25888226;citation_doi=10.1186/s12864-015-1383-x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. Parra;citation_author=K. Bradnam;citation_author=I. Korf;citation_title=CEGMA: a pipeline to accurately annotate core genes in eukaryotic genomes.;citation_volume=23;citation_year=2007;citation_pmid=17332020;citation_doi=10.1093/bioinformatics/btm071" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=B. K. Peterson;citation_author=J. N. Weber;citation_author=E. H. Kay;citation_author=H. S. Fisher;citation_author=H. E. Hoekstra;citation_title=Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species.;citation_pages=e37135-e37135;citation_volume=7;citation_year=2012;citation_issue=5;citation_pmid=22675423;citation_doi=10.1371/journal.pone.0037135" />
<meta name="citation_reference" content="citation_journal_title=G3&amp;#58; Genes|Genomes|Genetics;citation_journal_abbrev=G3;citation_author=H. Recknagel;citation_author=K. R. Elmer;citation_author=A. Meyer;citation_title=A Hybrid Genetic Linkage Map of Two Ecologically and Morphologically Divergent Midas Cichlid Fishes (Amphilophus spp.) Obtained by Massively Parallel DNA Sequencing (ddRADSeq);citation_pages=65-74;citation_volume=3;citation_year=2013;citation_issue=1;citation_pmid=23316439;citation_doi=10.1534/g3.112.003897" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=E. Sarropoulou;citation_author=R. Franch;citation_author=B. Louro;citation_author=D. M. Power;citation_author=L. Bargelloni;citation_title=A gene-based radiation hybrid map of the gilthead sea bream Sparus aurata refines and exploits conserved synteny with Tetraodon nigroviridis.;citation_pages=44-44;citation_volume=8;citation_year=2007;citation_pmid=17286862;citation_doi=10.1186/1471-2164-8-44" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=F. Senger;citation_author=C. Priat;citation_author=C. Hitte;citation_author=E. Sarropoulou;citation_author=R. Franch;citation_title=The first radiation hybrid map of a perch-like fish: the gilthead seabream (Sparus aurata L).;citation_pages=793-800;citation_volume=87;citation_year=2006;citation_issue=6;citation_pmid=16413167;citation_doi=10.1016/j.ygeno.2005.11.019" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=X. Sun;citation_author=D. Liu;citation_author=X. Zhang;citation_author=W. Li;citation_author=H. Liu;citation_title=SLAF-seq: an efficient method of large-scale de novo SNP discovery and genotyping using high-throughput sequencing.;citation_pages=e58700;citation_volume=8;citation_year=2013;citation_pmid=23527008;citation_doi=10.1371/journal.pone.0058700" />
<meta name="citation_reference" content="citation_journal_title=Aquaculture;citation_author=J. F. Taylor;citation_title=Implementation and accuracy of genomic selection.;citation_pages=S8-S14;citation_volume=420–421;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=M. Tine;citation_author=H. Kuhl;citation_author=P. A. Gagnaire;citation_author=B. Louro;citation_author=E. Desmarais;citation_title=European sea bass genome and its variation provide insights into adaptation to euryhalinity and speciation.;citation_pages=5770;citation_volume=5;citation_year=2014;citation_pmid=25534655;citation_doi=10.1038/ncomms6770" />
<meta name="citation_reference" content="citation_journal_title=PeerJ;citation_author=R. J. Toonen;citation_author=J. B. Puritz;citation_author=Z. H. Forsman;citation_author=J. L. Whitney;citation_author=I. Fernandez-Silva;citation_title=ezRAD: a simplified method for genomic genotyping in non-model organisms.;citation_pages=e203;citation_volume=1;citation_year=2013;citation_pmid=24282669;citation_doi=10.7717/peerj.203" />
<meta name="citation_reference" content="citation_journal_title=Mar. Genomics;citation_author=C. S. Tsigenopoulos;citation_author=B. Louro;citation_author=D. Chatziplis;citation_author=J. Lagnel;citation_author=E. Vogiatzi;citation_title=Second generation genetic linkage map for the gilthead sea bream Sparus aurata L.;citation_pages=77-82;citation_volume=18;citation_year=2014;citation_pmid=25307204;citation_doi=10.1016/j.margen.2014.09.008" />
<meta name="citation_reference" content="citation_journal_title=Cytogenetic and genome research;citation_journal_abbrev=Cytogenet Genome Res;citation_author=J. R. Urton;citation_author=S. R. McCann;citation_author=C. L. Peichel;citation_title=Karyotype differentiation between two stickleback species (Gasterosteidae).;citation_pages=150-159;citation_volume=135;citation_year=2011;citation_issue=2;citation_pmid=21921583;citation_doi=10.1159/000331232" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=R. E. Voorrips;citation_title=MapChart: Software for the Graphical Presentation of Linkage Maps and QTLs;citation_pages=77-78;citation_volume=93;citation_year=2002;citation_issue=1;citation_pmid=12011185;citation_doi=10.1093/jhered/93.1.77" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=S. Wang;citation_author=E. Meyer;citation_author=J. K. McKay;citation_author=M. V. Matz;citation_title=2b-RAD: a simple and flexible method for genome-wide genotyping.;citation_pages=808-810;citation_volume=9;citation_year=2012;citation_issue=8;citation_pmid=22609625;citation_doi=10.1038/nmeth.2023" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=R. Wu;citation_author=C. X. Ma;citation_author=I. Painter;citation_author=Z. B. Zeng;citation_title=Simultaneous maximum likelihood estimation of linkage and linkage phases in outcrossing species.;citation_pages=349-363;citation_volume=61;citation_year=2002;citation_issue=3;citation_pmid=12027621;citation_doi=10.1006/tpbi.2002.1577" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=X. You;citation_author=L. Shu;citation_author=S. Li;citation_author=J. Chen;citation_author=J. Luo;citation_title=Construction of high-density genetic linkage maps for orange-spotted grouper Epinephelus coioides using multiplexed shotgun genotyping.;citation_pages=113;citation_volume=14;citation_year=2013;citation_pmid=24289265" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Exploring a Nonmodel Teleost Genome Through RAD Sequencing—Linkage Mapping in Common Pandora, Pagellus erythrinus and Comparative Genomic Analysis" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3.cover-source.jpg" />
<meta name="twitter:description" content="Common pandora ( Pagellus erythrinus ) is a benthopelagic marine fish belonging to the teleost family Sparidae, and a newly recruited species in Mediterranean aquaculture. The paucity of genetic information relating to sparids, despite their growing economic value for aquaculture, provides the impetus for exploring the genomics of this fish group. Genomic tool development, such as genetic linkage maps provision, lays the groundwork for linking genotype to phenotype, allowing fine-mapping of loci responsible for beneficial traits. In this study, we applied ddRAD methodology to identify polymorphic markers in a full-sib family of common pandora. Employing the Illumina MiSeq platform, we sampled and sequenced a size-selected genomic fraction of 99 individuals, which led to the identification of 920 polymorphic loci. Downstream mapping analysis resulted in the construction of 24 robust linkage groups, corresponding to the karyotype of the species. The common pandora linkage map showed varying degrees of conserved synteny with four other teleost genomes, namely the European seabass ( Dicentrarchus labrax ), Nile tilapia ( Oreochromis niloticus ), stickleback ( Gasterosteus aculeatus ), and medaka ( Oryzias latipes ), suggesting a conserved genomic evolution in Sparidae. Our work exploits the possibilities of genotyping by sequencing to gain novel insights into genome structure and evolution. Such information will boost the study of cultured species and will set the foundation for a deeper understanding of the complex evolutionary history of teleosts." />
<meta name="og-title" property="og:title" content="Exploring a Nonmodel Teleost Genome Through RAD Sequencing—Linkage Mapping in Common Pandora, Pagellus erythrinus and Comparative Genomic Analysis" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/3/509" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Common pandora ( Pagellus erythrinus ) is a benthopelagic marine fish belonging to the teleost family Sparidae, and a newly recruited species in Mediterranean aquaculture. The paucity of genetic information relating to sparids, despite their growing economic value for aquaculture, provides the impetus for exploring the genomics of this fish group. Genomic tool development, such as genetic linkage maps provision, lays the groundwork for linking genotype to phenotype, allowing fine-mapping of loci responsible for beneficial traits. In this study, we applied ddRAD methodology to identify polymorphic markers in a full-sib family of common pandora. Employing the Illumina MiSeq platform, we sampled and sequenced a size-selected genomic fraction of 99 individuals, which led to the identification of 920 polymorphic loci. Downstream mapping analysis resulted in the construction of 24 robust linkage groups, corresponding to the karyotype of the species. The common pandora linkage map showed varying degrees of conserved synteny with four other teleost genomes, namely the European seabass ( Dicentrarchus labrax ), Nile tilapia ( Oreochromis niloticus ), stickleback ( Gasterosteus aculeatus ), and medaka ( Oryzias latipes ), suggesting a conserved genomic evolution in Sparidae. Our work exploits the possibilities of genotyping by sequencing to gain novel insights into genome structure and evolution. Such information will boost the study of cultured species and will set the foundation for a deeper understanding of the complex evolutionary history of teleosts." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3.cover-source.jpg" />
    <title>Exploring a Nonmodel Teleost Genome Through RAD Sequencing—Linkage Mapping in Common Pandora, Pagellus erythrinus and Comparative Genomic Analysis | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__DbqSGiVjr90FgBIuPlGEAiimgNNAH8S6qbiYFsi8JyQ__E2bjoC6LAI1j30ywBL_JgYRwIhvNl2n0yRyt_snD9IM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"ZdBD59C56kRFitKajQvJdekgVl8uDhOQmfdtatmm-_Y","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460082","apath":"\/ggg\/6\/3\/509.atom","pisa":"ggg;6\/3\/509","ac":{"\/ggg\/6\/3\/509.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/3\/509.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/3\/509"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/3\/509"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/3\/509":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022F8vc5+vERiQWvLN7+V\\\\\\\/m5OsfDCpIa6Y0cond6Qm2kt4hHRKuledkExOOI2VOVewNbp\\\\\\\/HIf574cAQgb6dYGqazmFXpRBhkP7IdSyW2EjdMqAZ5TscpAX2dbRkLtjkxQlrIDUSIqGArqixXtLNTGcJtjXGsmT+k7N58THOVhv0f9iQkkyxlVUImc71EJTlt02WYcYVPw6XBylApeNhFb6Xw2fW1G3TNZhOh2PUK3PuPsgPQI3zbusc3JBocFJVaxbfqrntKluG6qqJAw4pjPHlNXNMqshdvg5X5FgHrYcN5QEUMD4px1av0zfYl7C\\\\\\\/4aGqVGPykA8WAJPOQJY9THLw6B+wsmQ8ZBlhGUoqiNNGe1Ag\\\\\\\/BiRz60Q1q3Ryi6k\\\\\\\/dE2qKaxMBRjCbT\\\\\\\/8bLiq6foh2sze9l\\\\\\\/ZEYQ9OwQHrUU7pAPQJmmqbF3LVCOM2PCala1I3zsPbFAiK409pkC4yeKYgPt+LqBhrlXdvMqsQDE0dp9gamXLwrkDNFJdOtRdiOGojAaYE+wkyAQ+LH\\\\\\\/tszNcxBOSxOTBMzxmOEWKRgLEoAxn0C90sHjpHGLFTvow\\\\\\\/3sScDbEsq+6qOsG50R3f3HqaC02Xg3qj7+4+8DYR2OW3zXgJRvU9p9R4neEXg9NEN3E2uN\\\\\\\/av3JWmc7Q5fYccCbpYtEDOQiaHOnZenFccSD7l6CQIrICgx9AIkacGbDbMAGGO1nHLWGCwlKyGAzDImRj8IJ\\\\\\\/YXj5F9kNkx2\\\\\\\/Gyas7gQ8Tj7ZfLTCe8lX9b7Rs\\\\\\\/xSiTmpi9MzRkVICcNoQRqUsN1g5EaOfFguozxvU9lwWZxqSag+BzEH\\\\\\\/nTiECd9cFXGJYCzM4tAljH7WG2wG10A5ya6t\\\\\\\/WHy\\\\\\\/5i7ESsyMg4kZpHLAf9wE87xc9Deal2m7EzMoLn+vYbw+pDCxW2Wm45HVpzotl+oUmugo0qlIazzNs0ot\\\\\\\/KoQojKOLpJZ3A01cwa32sap3EHGR7+\\\\\\\/AWChDE9xDcp4Blb7t\\\\\\\/1EtCSsAzP9znbvz6eS5rpH4z4SRFpQ7jYF2XNOpFfzEipZqimZuKpm4CXqeieyxhEOgN0bHkbx5GkCgHMDE4S1jZfEN8\\\\\\\/esw11XAb74S\\\\\\\/suglccM9YolyQaaQaWq3Z5nZFIVyp6C3zeOtgT\\\\\\\/\\\\\\\/jm5\\\\\\\/PQmYllCfgvAqk8EMJbvVII\\\\\\\/Z4NXagMpsQFVeOc+NmPtsoSfsFPhBHFIiNQM4TJS1Q3nzi9IrsTNISPuKSWru3CopeRfmnfq\\\\\\\/aiXCzt5d8R5sOTLxtibCpv6D4zfBtvi6Zb1bH5DRX\\\u0022,\\\u0022iv\\\u0022:\\\u0022rZO4sRF17m6+gSVAGbQB4Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228c3ff0e4abfa7b4fd3806625f35ea4656a9500f6e95c044161d91c245e8b6230\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022yB9nZp8mTHagL4jBrVXB9zyfwx2izZjMPeo9V4cBB7Pii9xtUmv\\\\\\\/oUT2K+FIGQOlgnBJTSfsZNxkVc\\\\\\\/A\\\\\\\/ifqJOYH1ZzZ36j+VqeeNukGEU+iWxJkwU91yn4SuNmffn71mz6mH4I6fL9dP6DtmTB\\\\\\\/3vWICNuMxxpD46ApxrkCyufFDP5IdFwpMoywwFBQgBgLtIA2jEy7+ZN4LJGFytF3Mj8KeBeZUU0Hoc5Xnxf1MTCZATPwO7X3K8Qwbnp4JoX71fDlnQR4e8U\\\\\\\/S8ht5T5vVPIGYC1v+4xdxnpf94oDMktjYdOGk21nUbd2s9K9nuXqVm1h8q3ieYFgmj+7\\\\\\\/HFCEZLEllPPG1NxmbZWfK+idKXtNgD4jB5+3NHQIgpLn38aNTN+6AGPW3+Yg65zuLnjCnv9S\\\\\\\/qbw+e8h\\\\\\\/AGcgVBtzU0YXXgaiomrGU66LPZJGBy0\\\\\\\/W+3W7oizN6h8+G3qpMRwdc7aSipa65PAJYBPrudaNlw4utAbpjU4SHYx3m21GlF+pF0sqC6EW\\\\\\\/xg1\\\\\\\/VCerVoI+qw+Snb2475FfQ9J\\\\\\\/hxDGZyc49dHWaPGqj7Z\\\\\\\/\\\\\\\/1mz7xsZZYEm0WClSV9VRt2q\\\\\\\/03aluZAouAVGh6PXfObrl416vYMM\\\\\\\/qf7JPT3mm3OpBKH2bUsFYpdAmTYC42o\\\\\\\/NWOfc6Bep+8qfxJPkLdDKI79mD0i5rk5MopdMfKOlvd817EpsCRFInvQV3YdbSOsxHU7F7mAr4bv0UQmsF7ptdPHuvsaAr0HpY7sCqh4nzNL127Fr4G23c7l+6DdunecHu4igzNjPTx6EgdUJFgjKYawjK27cAscJ874wuF27TsOBvXS\\\\\\\/P8jVEman4S9xU6JH9FY3kwhxrsubvWqj7ECFkP2UDPHyo6PJ8AiCKy2uQ6K5YLmoPG+FPCyLkJaG9lc3yhKFbO0skqqpDVG7b2kF86t13V0F71ZJqoC6OnB9mNEZmJAI7N07dr5UfOPeSEvDNpDDgM\\\\\\\/UrdCc6mVnfAgNvS3KVUfSaf3dw9jxqJlHI0POps3esYAmt8J1HOFfJHgnfyr4e9SMMiZQsbOfbmNR6E+3rIlzwOiAG6LZtL\\\\\\\/OxuHVcvvAHMgMHXUR+1moZTeB2Bk68HfO4tftbExnyu2eNmfkOVAheAn0VFi93zSSQzvWdUNjwJ\\\\\\\/kJqH3LsukOMgua9sm\\\\\\\/fQlzuzgu\\\\\\\/gkBOYzzUU1ydLeuS16yNWLF8T9915s5n35WuttAOVzO1XIkPvUGR7jLINeaXtkwSRfefCeipSpnV5MjA4MFiGSF5m1mJAD62kzs1a6rvS6NtXHcJZYht9kOsWv3N4kGZfoStKwg3n2BUl+7XreGwxNq4Eb3Y5dhEo8K7ka5ACzXGJQUXj1o8EMdXA6QpEOdtN5+0NaX4E62bXYwwEZTzDKBNZqSgvONdIvvKIUpIEECNFAAgO\\\\\\\/H2N7TygNFCLO8NekhH5Dsh7H19VFU2eQWt9erXpUHknUKb7pnphW7ot4KDpPz+ilz552S+m5JN6J87\\\\\\\/ihxLja2r9\\\\\\\/lGt8Id7Aw4IQUKVAJoNmsIO5gGL79DT7dXKkSEqbo53FDNBbgopHiY6lWVtIryZqjdCFYZEcRxLy5l+zTM+w1wzt3QzT9hy11UV3KlBfXDgmy7ULOZYrihxuAQqw2UuKlczOTuk54whJphG7ssk4psqe9YC3dOLSfrPwNW3Ev9N5lx5Iv7WJpC9V1VIbsH9rNGFIuEHAL+tadkDgRGUn7KzY7DuFdRyRRSppnIxcAtmh6ivnfVc4hWcmiVhJ99cYMReoMKoGscQoIVln8Ppyq4NHfNidMyZ4znoYpa0An6m0X0wQnehEj+yUReG6BLTOc1jPXKk1wg0RQy2Tmn+Vc9tKX952vZHjSNdm4LFReXDT\\\\\\\/jD69BYTOjN38ZmU49L5u0x14FRaOo\\\\\\\/DI\\\\\\\/r8IKUT8iToXu4SQaD8cvqV0T5t\\\\\\\/YcDtFcVmP\\\\\\\/sxCkxjKxCCFt6zg+iV3L53mljX1xe+rNrPYcUH4\\\\\\\/iVxEZqSijGneWdzE\\\\\\\/nvZQ+SlvsZOP3fhR7Cylad5dinGc6RCFWFquBTEyAhEuvlJRLQX\\\\\\\/g88k\\\\\\\/qEJG97KJB03Je7lphwlkbfR4QX4greH8pIvWheb9HkRt55TtoRHTYQLYN1nB1L4fWVGpYuPkRipigE3L17v0PlQr4GHH22UPVemOufQZ7isDCZEUF6G6fHQoHenOCTCqK4LHkTZtqAwLgY9QfeBoPr6CgoXvs6B0ORzKWdHM1yVBEfJq4GMGG1mRpWiUDTkyIpDT9yvnetZRQhA6zJxxE7gDC8UN\\\\\\\/jlRZ4K44E88cnBx4ZmlMYTuFVCU5iJfMUby436BQClBOwkEDtApOiuy+TWA\\\\\\\/pIrEZJctj8LME\\\\\\\/O5WMbjdRB38njqqEhLB+6JkDKzkJUkxZhdASylw84adclGvL5CruOTweF0zTQecA6R5VJ7bL0J30Fwdrw\\\\\\\/kbo6gVLbntEfYksxXOjL6U9\\\\\\\/Mb\\\\\\\/7eUcNz9couJN1l\\\\\\\/wUfvILOXWFcX1o4lCVP2YhuHmtLUxVQN9xvV05VkSJgcw3KVGQVg4o+6\\\\\\\/Xh82wVG7fCeOKugmB3Bi8ipJdTfVqJhYD0dAA861HMZGgFh6pNQo612UkXzSxulvq0JnA92INO9dJtY+RiLXV\\\\\\\/7rMPf9qGDoOHBUNah4RyIujc5lc+awOp7Jh7LMcpNMbPOjRIOoomHhQDSsgOZuhxgvJMLO+j9baJcYvse7jL+yl\\\\\\\/HbdTe\\\\\\\/HQ1bB\\\\\\\/isOPCyMPhUpLg1w9ECvTmvohDFjR+b2ZQ+eMBHSE\\\\\\\/5hkdlFOcFSJ2dpYeS0Z\\\\\\\/nX6HBfOp6hchmvmVIPykrq3hLdPcZ9ucrmmh8hwuIu5f3FtTgyH0\\\\\\\/oij7CZrzJEpa48WkSCIvFNsuC1D5l5xWwY7VuJzyOLtNDrLz5BYiluWKdXYYvvLdnnPJgir1ovbu20dww2KMTZ9norx\\\\\\\/1Di2yw9hZTemW1zjChVpwi1aEFMAECzqDeOUtlODA7lA61TSScMNS4rksGS3Zvweg18SOcnb9uCe0iDFkMCRO18Dcq6Cw+EBlSmsXRgSTOLPJEkXH2zuIg5aq7Qny1eliYKfUv7VyZ8yoZRegQxzLN4WqvTmcFns9eSx5joCYn1vFOKpO5eVUwwRmDsClUwr1eDPunpEEX\\\\\\\/fy8i5IPqMZ8BI9boH+v2bbvEdWTvaSQ6Q7uYYmO3Z\\\\\\\/7lI8Lzdo9ymGCnP2VW219yblh92igTsgNK\\\\\\\/HXyZWz\\\\\\\/O8WB6EicZbI4bIm0w9K\\\\\\\/Mg14sa64sXbO1mM\\\\\\\/0OCmNahknJzkKU4F4K9CTZN8b5AAp6bZlMdjfqQZ3njO2w2f6P2qEhj9Ze02L4mpggh6Ufxe16FJXDeC2Xq41FA0I5aYcJHQ93bcp5MWbD6PgIdLQLx4OmzbM8IGJfwUR6ZX9tyJIvoRehJsMEc6MLZWx+iA0nRtlICzKqBmTichshlo5uVGt42vlxN66daCa1OPhHa\\\\\\\/ivlF7z1gFN2DoUOADV91C2hhsbW2TpLSSGMturuoT6\\\\\\\/RslZu3Gps0gGNLat4JEPt2QPVgYYmmTvlPMEn0BpUPomROWC4RM1FEuldOTvQh475Qz63TnodH+bgzGpdBwkXN5jTkZfPXw9oGwDcTEVMh+3Bl1X6jEY8cMpmkxRFLNJrrHPDQQqnm9tj\\\\\\\/\\\\\\\/NN0GTv0n18NngBxkVf7tYrwmjPcP5Qrq60+8HUQhRP8VPG5ywb5o0CcaiBbAnz6WuoZuTxrL4t4I9Dz2ReRze4Qjs\\\\\\\/kDYFQi1K7nlXLM\\\\\\\/7otcZ5h5zRPuu3aIjtpWEM8az5WaVImPwAwAmMOQldlBNX4NnGJFgwW56+l+IDuL8K4\\\\\\\/RHee\\\\\\\/RRbeLj7CVAhr0e8DoXDcI83sewJhcnRCKO\\\\\\\/L3BvhVCweIAwFdaUrMoe+H8lM\\\\\\\/cLuR\\\\\\\/Z+fscaIAKzJkb258uFv+535kFnyeP0u+gMX6wzJPjDnSTEm2bcgvjfq53\\\\\\\/2pMa1qsX59GEJ8Rlc2ia6rSDMbcdbaUJSro1rBB3LG69WY61u239GpzJxkr9DxPBgBOUIy+GDbvEip9aE9KCtpeifiW8oIUkI9bjYVRqXDqnI1vND+hHxtz\\\\\\\/MhD0NQ2qIhGI02f6dnOk2xbPwum9nrCev\\\\\\\/gj0b\\\\\\\/HxfJcL70o1SvXe3qR4KX26NpDuvDYvcyVNDGyXfKsNobnyEAJ8z0MkFciOCtdQpxjdVus1NUIWTnskQKnMr64q7cs+MgC8Kt\\\\\\\/NKhS1bqtbCjEm6ZzHz20nU0Z1DXUaKq7fV7yi\\\\\\\/+El5uSfXGcwTGPcecfBJl3gJc6BEaPFROj3ZCH+OMJmjEgfFZhcpy09pAOv1Ev2eSyGkRmJoWhx3O4gF8uUEjpxG1LsGHmcFbLXQ3M8LAP6ZJ0UlCToZ\\\\\\\/NWdCoPDeecOLZ9\\\\\\\/R3iDVLhoDJNixVilVEwmMAzudfO26Ic1y1nsuVBikRvnqLpHmwemdSCCzwyNgNFwhc+5OkghP2C1woXlWWU+UeUcJuFHiGCIGBHbvMBCp78JAnkbIssiYgee703cqY\\\\\\\/\\\\\\\/v6\\\\\\\/QS0uco3sAAQnc6Um\\\\\\\/7u\\\\\\\/hju3rJg0Na+bwxsckz5pJEPR+\\\\\\\/q05M+NvjD5c\\\\\\\/5BQ1dYwG\\\\\\\/qBVNDUa\\\\\\\/4liU4dq9LLYB2JMtI3Yjvn+cODg050ybOGKsVgvc1W7iSukEHyvkBOCeqGwPg9rEVNZTIaqdAxgbeh0wpBffpW8mHIPfflSGDe5ji+jG5divUhP7pcj2UlUG+A672GexNPQUeWkwTjz0cc83g33SCHn8hqY6huip9FzfqAR69oc4eLRxiSSQaQrGCwF94SypB15yYDSogQtSUquRTt09J2ytB6MPFAx7eXqW5UK0UGBpuF66J9QJ4GED50k7Un6MM42okaS1UzteJe7J4R7eu6+dpjvyYL+AsgivbQjZ66eZmD+YTg8o+d82eGGEaf\\\\\\\/mfpEH3Sjv+0cE4JO3OTz8k2CNSAVUXi6U4yQF4lEoJdQ1WTyBEHOxlP\\\\\\\/WKymhPhhlRpdcto1bPY+khYh8\\\\\\\/fv3KIgAXGu9wsWkS7XQriSBIeM5HkhXuaaWlCZto3wGJbCKfvl+CMjP7cKUob\\\\\\\/RnQc6mtGrDthdTv1A2yU+mipXkAefmDNI5FFem8xDcya28kerxCzEhzHZISCeFdr77rkAxiUNyxkySywtWqz\\\\\\\/uRvX2w1AX5SGwjm\\\\\\\/nNbtmSqEpUDPi5w4IiSB81Y5\\\\\\\/vwO71jRB9goJgAlqTh3o8HznmkDjYiaXn4H\\\\\\\/VwEmGVNuoxBoLQoPQCXJRUn9pO7nmzxodp6vd3mNifrTbxuf3H9\\\\\\\/msPSLgKAysfhXBEQMQFjMVgYKol37HCktFoX1Oa\\\\\\\/GSZZsSf0J\\\\\\\/wh402KcNdwW32cukE7RRKz5EtBf6knDjEDqYF6LZOrV79dC3vj1\\\\\\\/WWHHYsVEBJ8LqPf\\\\\\\/+RsuMtLMIyoY3Qjaoi3J7P7c+w3l0BDbfHw8eT8vMOKJefmXGMmHCKoQoj34x8\\\\\\\/JL9\\\\\\\/OEW6e\\\\\\\/s5ybmxNjq9yW3YPX4JHaPdKz\\\\\\\/7p0uKpSoBjKyzoZ3Ah6iTXeQv31Rch\\\\\\\/OiL6EeeEKiY42uBXbDzlds6FQ7k+HCqyaeCRsXbdwOo3asJqN7fLQfoes3j54BRr+VLkvHdseiYMgZxMK4EtmbbBetOsR3QDPfZF8XbBmvCr7HXlT29s5OIELdvN5s8+mVaoKyO528GSbyiDnURIUhGVveafKU\\\\\\\/gQyqv3MFRLjvJLurxwqFNMm+dFPTnD\\\\\\\/fPLt8Fusrpe5ZhR\\\\\\\/N5DgVqtLoz50OIG2urr08ka22i5soBYvSTiXVDSpR46Ma0kF+GWAJB6A39fOs0blqA0SFqH7qqnHcRi5qG15y98NnW+0lV1PKYlC6i5TfnQjM6h4ttWK6xlURuckALwRxneghFUqVEzilLPxt\\\\\\\/IszuCvzlYwEJerBhlSROr9znf4DMKaon40byVIx3Jz6CKAD8+k7UxBEyq+3w5d2SFQXKBXrV6XVNexiFNvzYE+JL7etg7Udi+mfqF3IoXUCsW4DR7qmBt9Q00FaUmv5dBtl7\\\\\\\/yLO0KR5KUlLSbwSy9X0wkZQwILczcE5K67y7XVielQZPFGRlDdDcb\\\\\\\/vDXvVY7y4lyMDNQkk6bSDK\\\\\\\/4wKUdWSTS9kvDJforzeLfSxuOorG474WF60JGjDqpffoSTi+oOF2bHACexuP7ulS11W1h3w5zl9od7bXEbF9fhsa+GvSABt3meueVyxT6uy6DKdJ46kPFJBXU6QR7FZhV1ZRqGsqldV\\\\\\\/bjzMSkbvWlmZMUGIfjSuZ949qLGVpKSzEticbR2S\\\\\\\/DvdBCiOX+7XI5TmT6AGvAiuc\\\\\\\/psMx+Pai2fYOtMXxfm\\\\\\\/lX5dWQYNC5VS4fxNZaQV5akjIhXxpv97naZNGQMcYMWmdbnB7LiyQZAsShvSla3YhuVMKzEvYCqcgfwLlypzHtUyGSUGNb5saFFlCce1f2M4EgP3SRlmbZ7v0K39\\\\\\\/l0XAlVcEvLRAmr10lJC59QX6vcbLpofEooM2NZQOl7VM\\\\\\\/DLIlp0iB66q9cXVtSb\\\\\\\/m4IMAicJeN59sw35DcWx87UjzWfhxabUCk+HuLR2mV\\\\\\\/zHyy1NnwpH5DngagcVBV2srajpCp743Z9h4M1xHw7W0APxLvVowCBtzJxQDW9QxmBayJSH23uTjQrMEf2wSjcMWMmtP5bagWceVq52PaOEQEwh2C\\\\\\\/6nO3\\\\\\\/oRW2VopYRL8j+u0BMi5neqs+foWqGYo3VrRTpe9q3aV9Bn9ah4+FCgnumx7vkelUCQE+IsS1Qfr7j8L4Nu7nZ5NMxtw9cQnxIYWEJSXwftcShk+LwJ3edC8KtBGVAlHIZBy3+AsDOlWmREwAiVk+PN3uInmUJGptJfMUbUY1SPPS7NivEyjQB0h7YhPRqQ\\\\\\\/N2o+v\\\\\\\/N1ITXKkWAyp\\\\\\\/F78Ofc2suq2fqbkrwmeaKjZwQZwfB3hClNqvNeqXDACI1zGmyMWZLn8F+yxlvDcPgIoLbyiDfKrE0AiwGx2hdVSPBGH7hrBuj5iqOmKkZsaNiYsNu+Sg02aIXwq+KZu4Ucq80fKDCY7+Kn41kiy5wRJ6ZJt6fCQ03coRqDYcYTQuXtdrddVTve14O1m414nKiHgGKifXYw1Up4xFIHquooLsIQoM9503kplePSFXpJJjiS+mMhljYEWiyBn59Ntx7iolkxqNSsbsB2BZvH7R9kbIo4EwyrAuw2Ilmz55VhFjhAhiLXcC5qmewn8RiX7nUC2JYJnTHYTL\\\\\\\/2xnNrazR5W4HgdhEkdymIwnzLd4qizZdST4fs4TUmKAda99qooXAX6\\\\\\\/vR\\\\\\\/XdFOo1rszyRqqjiSBEV7i99591b6cfwwoIXuiVT6Hf3Pfe6kudNs5NsULgIHAaPUqOqduKcOSZaR\\\\\\\/q5PY2nTnJ8EXqQd4A3Cgvdfs+GQvaQ8ARfscLWOrwHN+DXF+RcedhaqxKk++uCBDcxNCFXScCGQ4FXmL2sxmqe4nGRQINVKIS5qC6ASEITl9H1BjTMi6nBoxo\\\\\\\/tlg2r66lp+kpolRxDKcqAVjAUoXOnZNcee4OJsPX5MiWWWjy5DR4DkMfujNL4QctsjWMD2C50AwDgxzse3f6GRRdPJw5+WEvAqOCQ1frQGDRMF8YqQBwIJqnDr+UBy\\\\\\\/OonWxZBgoKjs7FUdG57sxKF4uNW4TyRSl9MLpiNBIw0Mg2Oc1QTr6nBhDqsEP3lKdeFGuponwOZLykkO4k3CUx8eTjmIVHRg8AsDyPhdwQSiIkyj4fYNqyareAWyU8OAAAbFv2S4giIfqi4ZknscUozefyZ4Ox1Ko+H521GuDhEuUTHR8uoOKWAg+ducjPCoia7X9Githx2XSbJBc8pLsByfPJsSLsBK2J5gmGkMK4cMGq1sw9Or7hmwJwqL\\\\\\\/AJ6Z2C3U52Svd6MLtw+w4QxqlcsrQE7QnrK35iBVq+4DwT9y4bUv23fl16CP\\\\\\\/b8bPBG\\\\\\\/y7FAap96X0Hav0W3c3ccoJccSbyjnPZWrsacdPe+ud6HLKL+3oISJSDCYIWc3JJpVcw4XtL+gtIYp4iQNxcKdVgjZGQ8bQjWAoLFyEU6DZXwaEmsvzGKwHvBm0RePWWsClvTg9hKGRn+yfX85Wj0a8TSVHcC4Sm16g97oHXcQS5S8faB+nQjSiI2EoxnUDIVqWAP90hmcsx4slcI8bqKj094fcuBlTMNIOsVhbcJ8QqkZ4w\\\\\\\/\\\\\\\/iJJsPhS6x6ZWIc3xErQ2qKvxEmBG+aHyImnjLYcv\\\\\\\/BSRgoMxf0haeNXfJvOrFt+Yq8gLG4fAeyOB7OqRRoLblUGY73n0xRb+ERqYSywTacImKpSHkditwsRowWAj46tOpB0M5Othf\\\\\\\/1u69zdfJ8x\\\\\\\/RLz8i2WxseF+W3Lj8Vb1ZNAkg6dyBVcM3o0+rvCYn5v628Zck+I6DYqq\\\\\\\/rUjycIkY5dl2Q5gm3pY1UyB+QirDwr+6QbrFh4WDGbsidPyFyuPmltYalGGKL0M7sFzRSW2+ZGQGwGJNSsImESGXv0viG1L3BeB9sjQQHCNldnIeEa3z\\\\\\\/MmhIngnoDeEStr9lg0NJbanR6gMuuIuCdAbKb3St9f1yAA8OQdYK2ZJpfzvHDqFa+I8bhrYyH5\\\\\\\/qghuEe7wypMepgG6veA9f5a517nSICSouk0VFAm32t\\\\\\\/NC45xXTLf9b8cm3PVR\\\\\\\/w7nb25bS+SrBAHEMcdd\\\\\\\/hbp5kkRFHZ7bVG4xk7bx\\\\\\\/jbe5O\\\\\\\/yhyZSanJUK142EICBNg\\\\\\\/qbNP43t95+5er0orc3alUc+QXcsVuC4rUS4KUYyoho8pB13vX+OXpoK1TvAYhHAG1yiTUy4TQbMuWX1YUP5wwEjIjWzrNNN6V+eKwEGu64nHLSFbxerRVz4vjJ1C3GSHRzXXI7PfnAdq6HDIo\\\\\\\/hVoBg8GQ7f4wsDDbrfg+9anTTRUtzmVWoYIEk9oBneLCBeHcIt+ZwH+uCLhls4VKaRWu19rh6kCRhVTuLnxWxW7T2hwnvAct\\\\\\\/TB7Z7hX70ks3q\\\\\\\/OpEhgRygXaETWyRgBU55+5PEKll0NKH5ZAlPyyU4\\\\\\\/aUcU4\\\\\\\/mpBl6SI0RxtQEd8P51WLlS4ePmVV0IJcTavIbEsUOuqAtwyddR5VLga7WeTeROeXch1vlsQTUabuuPtJcBsvVChINK49VZxKeJAMdEkZ0i08iJWQiIK2kaGf+9i5dkmRHArklbSWHjC+nkWc\\\\\\\/85S5x3kKVSTrV7zq0twxxP3fYOSp4HkhQZEgkTqswym6SlAKRreJYej7nlW0RYEws0ICC2wQ7qmSss1SmtuSJJfOsy2bBYQL2rPhj+VC1+slak0JBDte2BMIGs5F9WEtSU0ysRzzcxE+BI3WVnljmwYCyHhcFGoc2nxBkYOXfiC3Vgmlkf4tkerj46eaAkMDSW1t9EHzl8ZVHmE5tY0NsN3kk8RIpuyBFKNsTTi27Lu5mLUdT0fMFZpd8zxr+lGSE6dAlh+X7i8uzbl4bripvL8idsz1rFAL9E3c71662PGz30egMDavNba0NT\\\\\\\/nl5Jw8Y\\\\\\\/FbdgaTuoc+kjiFna9Yh7nx8DbNNhxhfU\\\\\\\/Gfsc0WFMwJPr\\\\\\\/a+5G89w8vD67vDuTIdiMZ5nNzkODaKy2JmM5lGjKj7wJl0AiTJbd1W\\\\\\\/bGSSDil01hyxyrsLZbeGU4jSGNQ709WFG7Uj6BSoBiBsDIpdd6y9FXqb5hP2irJlHWr1QQfWLYgTgkdXLE8JfCkN1uqw\\\\\\\/VttcjROhlZ9pKC+sK7AasQututIJl\\\\\\\/nK4MSdnDVTI7ukICSubdAlfov964obWlI3HFBjAyJNV4\\\\\\\/fiIzCSYwBgQcDMBCpr8U5SaOtziUpBoRWZicDaavG4\\\\\\\/wNn0D+IcP3pLIikiKfAy+XZQUBzN4wWOyPwHZVcuKt0C8hbWFIzcFb\\\\\\\/WEu4NXDRQFwPG77ahMKJfcugVevfpdUbHLqef3PPYc\\\\\\\/dM9e84CycmW0AF6vjCeBKjpeKlUk+q8zXXI2pnklLAMEYnufvbrpiQRnsRfG2xTuNaYmcFgY3zpklz7mG+jb3pis4To3dMRxWLYYicXgtUWGcKnrL\\\\\\\/okT\\\\\\\/jKT6W4dYsVg6kPmrTVbW4o3Q\\\\\\\/YpfRWZV4gbwgr3RvtS+hDij+M8SkriEKT8X4fGELBu38tJkaDm+s4Ar8abeKAkQ6QYQVYq6tUCX8clg+sNkYPuWQ36qsW9QSuzDo2OmSm1kRdbJEsB0FiyxJEwiSw63dcD3ExLw3VB+OipcIsRTe\\\\\\\/ums5IllmOcsaze6EqXtDfSZaJYtV3sro5Trphu5xfDwm2u6UTqdFe36GfewRQhOXFeFP6ZpOa1WLodmo5OxADGQDaK0d6j62UdESm6GBcROAY37gxL32LIO3Bmm5g3BzWrl3oiAEgaB2D+BgLMQi4XBXdSV8r+jhKp1xWFY+CFOK2101ezxsGVK+leSFZ+YOlY36fDuKEfgclTDa7sRgv23PpMBvOnUR+1VxwbbP8uNGjZQNxfxaACgf7ytvLUD3Jzc2mS7K\\\\\\\/l38cPg\\\\\\\/rR678teL76jKeNL3kbM2G\\\\\\\/L\\\\\\\/vqWj0tLTaT5jSq4IJc5XXIATvQ6FpC7eZ5cGfKXFXSMolrTijkTf4\\\\\\\/+WbHzZL\\\\\\\/oDKpgluGPYmBDQQGXw6bAnYyZqpUQWJ7pQdGN9baj50H5UBvCXf7AYVAwhWaJKKGwkKwhF2\\\\\\\/hlrTEkkIQHOwz76wt4A0XGv87QDwvO\\\\\\\/g\\\\\\\/5ULxQiH+cs9hZxpT+4pnVsLS1SxNdfS1T\\\\\\\/ft46vLnlJOVr465rF4kyLeRez7VYsAX83F2EL3bwIVnyQ19JCoKmsbAEsFoI0AMMBo9tDzfhogTDabVkFit2Szm3gUNHPuuzlPa8ORfwDLLFeJCY+x\\\\\\\/UwW8xGVtn+89XNx5Rp9+b4F6ONgC0CrSFliULkGtNX7Fks6pxGFGZYthpsrYpeApOhjUb07agvXe2sBXI2KxbEsafSj78ElvO2BQaJlyjY5SJna++OBX3+YvmFeKhHprn5\\\\\\\/ElYu0+rEu9E80+Ochwd3SzOryXZh6lTyMtJPpfkfVnILxmCkXgT62FYDM3AuRebsFzszur5pxxU30kAe58YzEeAtj\\\\\\\/Cu5VWbdgyn4ryGhrAjEC5nHg0Qh7aK6369y7xw5u9FhlCHJqf6yK9WaNcMuEb2gXVTQ6HxD8ji3F2opSPQjJd+NTWPzxMnwxau+nGr3m+XLD4OZaAt0nqNrISDUl9+UwGBiPEoZoutjRrPI85W39YG6n2vaq1lBUqpNKmgvheMhX+sRnMArJNqL\\\\\\\/xiH9OHPhcMRR34CGLc+zZflz27rW2KF\\\\\\\/vVeUPhoC7S5ybdc13FB9RO\\\\\\\/peCbi4pfQXTFYo\\\\\\\/oNmkChJfZZA0lalGfzM48aCM4m3phBG3e9VOwwYnBidc335ee28dWTNPFVnV9J7rR93XLV2biO6f9+tDqEE\\\\\\\/5NFsFq\\\\\\\/EeGNuqycE0XLotNYbILbZvFPCSQ5a5RruZYdfjW40FeHAArBZZBsYS\\\\\\\/m\\\\\\\/+k2sN5FMlSgi8UZ8dz0QBo8R2gVyLi+SIGeJqbCOAq7V6XHDDIkVpWdrH\\\\\\\/OqG4x31iUzE5gn2wJmds\\\\\\\/Xd8Fh2Wi5+eP3BDTRPaXrdqWvJU57FwhvgLAjjHChtf3+lvzNZpmEPODXFmYGWqoEsHmSzs5++ahhUQhntDq2pANwcfGoCeUin0B4iQSJU2pUN4huyKqgSSuwXLCU+jVJ\\\\\\\/GkHrqvXi5XTOyPwN1sGrtC\\\\\\\/DjBoUly6AguM5UKoWjGA9aljGq8cjscSAlqBWEp8VhSlREN3xsYzIcczwjrer+LNmtlvRiWJgVOxcwR9ooUhFPVaV0a+pgJEsa4s7onAvl34GVGFjQBxBgk\\\\\\\/bpEKDDpGk2ALfWec3ISCoecw2w77RHjzaBvYXaWfEAjl5Tz4n6QlW5Sta22UP96V2v6zQ8NsI7qImtLwSTenZjRafv91eqzsFJXEicjGeI1UnF7WjvfM9zZVvl3bcoQssEBeLiG4NwGpLtkeUfnPE455ElVAruKxANT0wKBffqqUfQimPYyIQeE4NmqLViZrgzTHD3kLuL15EiQek9QKAb4Q7ISSDMPp\\\\\\\/pX3XZkysveMuxErfMI9sdC6ARG5j8PYFid000GER9rBnkqZUwkDL5oU8i47F1IS\\\\\\\/UHZTmuKz\\\\\\\/6Mwore\\\\\\\/glHU0O6fg9gShco12E2YlxFxVYS6XcKmvtqx3+7keVtpr+4VB2vErvRKJ4v9uxt9iGVMMkiY3czTp7s2ODRKiApJFk\\\\\\\/8EMrhki0r8v8LC5E2FXqweAP32a+LxGDb077ZJyEtB35hduwfum1qU+ZAyqGqVSi7lVEB9gSnJt\\\\\\\/UDk95xTqOAsRe3lkMASdW93I+1jtQq3kcW+iz7wFPMCVNsjl5QhTInkqkqYPEG1z+5YjJBjmumXSQ9wonhQHidaihKzSkrCvpsIQUtFIC9S\\\\\\\/9H79Q5bWjh7KzSMgQB92HPo2AFybq7uHZmVgXY+c8LTrjECyd3lugA8CdJtI77JCikWcRDMW4dR8YxKL3Fb7nkdOr1JF\\\\\\\/UHg7pvH7ph0Lg7Ycrtstm5l6qtTQZ5RHormvPcqDl0imskZgsJt+xkqfouE9KalcwYQ5xqdLBaRnIaXo6ypNVZDURHepl\\\\\\\/LBVu0U\\\\\\\/LT3+4D4Ao+o5MUy7J9NMr4nAIa+HUTvTkzp0eGipk++sVZbz1NJESV3Pmxr+25Dbq07RrbHS0Alqc0c\\\\\\\/KbaQUNMRLDntmQ82Rcjml6P34\\\\\\\/18b13PiM0\\\\\\\/62uT13yPmdSKz+C81JxU93eGuIjRnzv37aJAa49vc0iN9mqDydprXxvXc0boW1aHwdx1h41oUF4lK493szaHC0+ysahjejLLijatal8GEbBE9lbTXWJLkmWi\\\\\\\/TKKyFqrTqN2K4oDAsGCrhppUI9i3ZIEd72qacHIQlLeP5sEXBaYeU10hg4nfTBszMHI2BxvKWixXMhKOx735X1xRhKtk\\\\\\\/0tWvOzvv2XiJg77M2rVYVghziQ4qH1ZKAjGticMwTYJrrpey72T3AO6kKLSRe9vg42gzRoS+J\\\\\\\/2OppgAdq6Cww+Rq6D0v5O6gPoRYIlcKgKAvcCrQRO0ZL4nz6kK6M5th6GyXVYrMnzcLxNJvnA5ExMZTi+YIjhXc\\\\\\\/SeTXcdhGVsHhnkrbajnhvGcm\\\\\\\/LA84P+vxKm04iDLHsWtOqnh+P0UcHrzK0J858z05iHftKr9cNO2ZN3ViSeMQf8RZ2TTO6ZelBbFA+8Zqo6bEC\\\\\\\/muE3TRgrsz6tn21oW1aEJhurciH+I9JQ7xRQLD4iC1O42EEKgGSDAIh+9nZIvXmvoGakiBjflcOaPrXih\\\\\\\/SBGfJy+8UmiH0RCMSecNk1aahcCGUpOdSEsn35NE4igVTRyMzKxb4Ppmuem611MRIIiDnPrbqOv5bOcMJG5RLUJmZmk7clm7sE66RpITK0ze2K2Bs+9LE7fOKbwtsPSC+39dHMIdDbNAet7VEaGe1Ok77l8LNGWJJFOl6ERRIdIPOeeZB3K+l3nkWgFZDcz75vTFQ6V914gE01Ro\\\\\\\/LeEBVjAeSVeCB4sv8br+rivAcGg3Vvd7Z68Cknt48QhdEtZvgiURsuhwAgHBYZt3wTlOlWG\\\\\\\/UpLnWsXUEioXVzjnqTBrD7dLinmvBUBzu7djk7jIJiaj5zFe+WF46GFQCJ4c8F8PTz2fXAFdScVRtY4zR3qQffHCah3NmQlgIHBH4TYcOhGbnC5gaQqjteYqMNdbUvItMkM8QJ7Df8uhSfUVMeQ4+gafaANz0nrxjNF+pkRo7NOfyOxo8L9xijNKw3QoWroHZ8hEWy7aJHL9PdkX1nCUoTPzbbkATZqRqPAN1lk14ti78BDZ3\\\\\\\/7AIUnuxTKdQvM+iv41+b9CUtE20TmQnjNYIVKXUEOWoBtJB25jvper71R7xXvje5hHwIH8N7xc2yx3w4AdE+NYjQULyCK2BTrzrx8i35PY7IqmI\\\\\\\/Ks2r\\\\\\\/hU5PtlIGv2Jpdr2FtPHEcrb4DXhGhbiIIIO4QigE53mdYCQ6DkX0FkFMYmvtcKftvYCQc3odm7xMuTRwAmF0vNMDWDmfYDGAGDs5k5zTgvEpjtE2Ma+6PafgP258emDbWT8+spXUbvt7c2pXBTv7ib7IN7d4dOqRF+05yNkYWtP7HK5Bxia5kGWqcW9rMBEHn6ZKdp73GsZ4eEeJLLh3jqf7vWFpHleII\\\\\\\/vb2q2Kx8P8tLpurEFcKqMoAWnNkxUMYnvgCkizfDYjfp3pLAbvkk6+NJuWknu5214ET1mRm2773YWNVGaqU87Rtag3L8nfnOf7y+ETrgC3ZXmDmyv2LZu1Hq0ao7P7R6m+qXnzTBP5Yaef1vb9LQYkraEBayRiaqc8KNy5lv8SMJCCh5ziXNFy8sB5peGzXFsDoNbXgRZzOQkWvO6I+Y92UuZ6T1f9xCDRqMEHM2vOjYoeBBY5X4LtKXs9cdArTd26gmKPVDZ\\\\\\\/dauzUSrc9yiTb5NrYTvqEJ41U6b1m0YFR2OpTWFQBoyfaRZTuVLTpESjkXN1nTrPyWhHIya+klM+SFiAyEZdlQt3VaTceKMYH+xu+OT55g9YuGVxidA0qp6p\\\\\\\/0N\\\\\\\/RrTCC2kcK5H3W60DsR4f9gjDXGA+HMw\\\\\\\/uaB1b2qxIkMS4OdpaxFHsjRKdWkj\\\\\\\/3i3FMdWaHJoiDNfJhSeP6H\\\\\\\/w0ISReI2\\\\\\\/l3lxN1QnGn3p3p3n4spR3HTMsJAJeUOtou6p5i5+gvV6WBu4acMGZmHa7L+1o8IK8Wga9IMXZW9Kb2LOdsgOj8GV\\\\\\\/0ZjqwuO1BogPYbY\\\\\\\/AA\\\\\\\/N9RfzqTqEjdnRDSPIGq4cXkY+tJImR7sLGesYZm7lHoTZpd5QmlYr0MXSVGu+uBDmL1gha+uVeOzm6Bc3w6quOA15FaS0Zatm7h\\\\\\\/ZdkCN+9nwRK87zsGwDGLnyuTihBoSXG+iI+x\\\\\\\/LrigmmR7lm+UaDowo0RSluhumxiHCi7JlBSZFPQcWbjp7OAFOv4fujFutXkopeAk6J3nPKmfFvXOWGKtlxq9M8\\\\\\\/GtUDEU1evTJJQpMb6GbWw\\\\\\\/A4un0mUqz1cl0IppXr39o+sh4XaQ0pLDp16xDVS0048vq804tF3e3imgEZnSW6Yyw+11MZncgcpVBZzcH8\\\\\\\/k3r1SOY30HYW+DeCsT+ggq5dMWWIJLRR+MJMIeH2C8ah9\\\\\\\/TYo\\\\\\\/ucRfD2yjOA7Gks+4yZuWivG9zPx42fqba9vQM83vyMCwHEm+TVq7BrU9eZ0W+jbNDqxfQ7IzYRS3Bpj3jNQebVsMhqcQ+O86Ujkpu3TxzbrwJIK0LHQ6f+KfygROgeJLlEqnwigjI\\\\\\\/lR+AAV31AcGwh1KKqQMd\\\\\\\/WsqAztIte+0QNuJk81Y2\\\\\\\/\\\\\\\/YTsDzyrLXVxdFzt6BXLOUY55\\\\\\\/HlsSwHNdseObjflTQdflcWTewETAJDyZiGNe+Klja8NDoX5MdT6QMVpG2J8r4jcpaFL1gCJGgshQN9Yn+YA1aKjbwsH322MP6UK\\\\\\\/TczbjvnafifGmACHCkY1Dv5sesEItlriPmnn1VWyJvR62QDpDSPWHFpKn1UivYjk+eH0YaJm\\\\\\\/Xes7jjtQVjXzIf2i34nCfcd1u0pH7\\\\\\\/iFegnpbWQLmU34D7okCwDTn37jhUHs5SGCi+fxuIwSzXdu6MNgn6z3kEW\\\\\\\/kylqBzg5vrfBeQM1I8oPpS41oewP6aflEQ4E72Gxsfvb9VIZFmXBNWivDj0TR4rg\\\\\\\/qK2VksOIRbqXuInpHIAta6EUUNsNuZCVQhAu0szl4sj3F9qnA4fMTOckALkDvguPgKhDEIx7aZWUu7NvLrYrohGrGcMuR+Borxb6fHG6Wsgfay3AXnVuEsAYmsTMfjt85SgGbT56wy73+l1Q5JoJZjM3keyc4LA3RKXlH1NRDmAoU75QWsEHASETgShwcXlKhE7sPUp3WVGCl81SCDbFmZJTR+yCQNtqaazUpgJuUPDsIVv4asi6l5wM0Tag+TPY0xyJ5SS35IcysMYLNJfw30Iw62uTfyUFYRxIePh5bra9nIDhmG8MPj7WCRDeE0K4iBITKIKlGTkxvJJxAEBIEqz6w6ddxpnEIxwGQGtSql4h74oCTPzgRZ2CF2xYrLIqvU3T5VJUagStxHCkpGb9pfiUD\\\\\\\/WlQQaowepkeTj84xqsl4adMMC4iJDHqvhhjLDiMrXgeir699HsGWqjGu9cRY\\\\\\\/DRcDDmRe7Z8nRtYnl42ye0pByBWpP6u8yNw6Vu22wJ8cySdScPqvZTA0x3Cza7mJZLiJWUqchUZs0q0llTU5Bb04EBirV8ODNCK9KXSQVsJHlktjFxmviAm32Fvm0mtG\\\\\\\/iOMgzV73cLakLPB9GRDGWToSa9YkacCSB2MHaf2Qh9iIDnBx361S5cMHi8dTzSmmn+HJYSXvwBrr8E0c3KggsgkAitzACaH2T7wrS6PRTILcZDtcwwDwJbB3PDKgDzrLZMXfDnfjiPFheqIpHExvYzCb4+WbdmELAj9iiaSgrhtEuwjqqHdskDKP4UElgps9LcghGkgTUqgQgayahtabrdJKB1LN6cvp2zIB8M96gO11amXyT88hdTh\\\\\\\/cKngMgEywkawq3xy2SqVR7O6THgnFeNSiJvIgyTh79Og13OzNmg1Np5NCb9Y+rXLmmIKf+JgGJg2mZywT3BtnErjpd2DpwW9Y6S6wT3IXzdhREamim98lO+\\\\\\\/TzvXjPpy1yV7rmxZ8HPZamGoXQoeemStVsbYo7ZF+a\\\\\\\/zEXfv4MgZ1U5mNHL1mxMjMx5lSzrrXetmRxJatY9k72yz6QtKLO4+1G1vBaPQrihLEKkzY4G5KACjSlcB6huF4dCetXhRuM8kvMyoSaZO1oZqFbjX87B8QGfDuNNtqFIvOZFUFuSYYyULXeBeOgcCvWjiAdtkifXmhyc8xf7sFCsBhKodxK0GlxWapa8pQi5wSTh2xZVQnx\\\\\\\/bvRH2An2UQobLZRWiG+L+Sh6X8m7sBRPyH4npUHz1uuqe4OfHL0jAJJu+1M8Iin3cN2U+WOqcqHkMKmMLS1L9oe6sIiH7TAiAbOxFeAGRoaiflY1SUtUgAwslkRrG+qt8lSA2ZUcL3c2oGyRuOw3rIF0JX5WTRXeTx8iwSIj74Hs4bgSJNV00dI41czQOmKWah6TSCSeXAq0TIdftOJGaop1WXjHyBvRLfisQIZlf1w3dC4dnjDV4t6IwLatDBEbPqhPD47DmN1V93lG7nmKz96JlH6GFRbNRZYD845\\\\\\\/dSrPMkGyFc8LdaADoQ1\\\\\\\/qRBTmhHyVA4431SmPuMuPxgQLKZL1upOF9Aj41cHus6HIHOGQAL0WGszJbBuckNnnZpcPwDq50WgXQa4\\\\\\\/KBVsSfX7boNrd6ZVLXuf13izp5mKdzVOoGS6Ph3WBOqf3fB\\\\\\\/W5wg5oiza3UVE90ZauFKp0dQny0S\\\\\\\/BKkDUd3ChBlGkNo0x25G8yF937gZhNRNOw9eIe9cvPl\\\\\\\/nqwz8HL2\\\\\\\/GZAV0oEuYJHsIHE+YBQLbJvNkrvAB3yQDgYm7ImTRcI3CJ3vZmBN+lAdEfTF\\\\\\\/Pg9U6ombQiLUSl7ZKEaKTGNs1o5wRx\\\\\\\/Bi06kIYiSVVD4WmdKbC31eToAF+qWNxoXndkggmUl16wK30GcH1IMkeLNTQXbXmH2HWY45Lo7Jgezu9MbBZpCWRUi8qD9yhMnbUJf74qMhq\\\\\\\/jap+c3ct1n3TAjOyTfmN4n9XS+JCJkynUn9rGOtzhNkYcF40+GuiYUYqaJW+d1IjNiGiczRflfsmmBGQRfntAgfP8kcc65BtjO1ugdl6TuReW\\\\\\\/Upo4R\\\\\\\/EI9UPEWEvRzCxbrubqL6uwDwmJp7auGhui6M3NFwl8+uw8tFu\\\\\\\/hKKXLIowkfygRaIXwIrAQGZ0EJhQ6tTWT5yR8THP8pUDcNhWh1Pw74TbyC+SFv03lEqHwV+yTPGmHLivWmPy7uyPQYCDkZOXw1JwMfnAmfsfMIUgaU1RmXEbolCZuzVxABmM4qFvWk6RBwQ2SJMpcSJiyULzpRy3NnXQ1+3hH4iUIogn1pkVERBQkxOMStP8\\\\\\\/uS39o1hDbe6Ahkqh+IYToNjBxV4y\\\\\\\/rEVq5oHNBrmVdocDxIeRewUky6RdXC4k4iThAq98R5I47k1+F7WJp1KRpsl3\\\\\\\/CEkL7NIDL08la9yDHGS+jERpFQ8aD2nS3XlsxwRaMIhJ8jx\\\\\\\/mWfe7gT\\\\\\\/sEfgAGYhq\\\\\\\/cuquNOD8qRK2bhGJZ+87sHA099yF2o3KIW7rmxcZUIhjujQMfEs8bs0\\\\\\\/C\\\\\\\/MQGXw98KEMcY+Rr1e3nI3YmEm4bU1iWH9KAKNpyQIJyyCqb7iYEzCsazmFUtXulArVM5FHVkpqdg5WY7beliYiv5fojTMAZwWMFXGGu2Bhy9iby8aha6w7X+o7fNvIwW1r3W8CW3bPfOncv0\\\\\\\/HZAepASOrXAcfLROnMpF5H28zfeBv\\\\\\\/Tjt8CmrIEOe0OFQMm3rBBckS+LqI1MfzdH3bP15BhrHiqQ+FliYKE5G9XZmyIOx2Xak7miLwqaanWG6JPpUMAI6i90tKP7fuaQ68P7WL3AuhBrtYjZ6cyXy9sTy\\\\\\\/5ho19b6mi6roshB0XyK6LQdLOXoOlQ9UlPSaZCeQties9WyNLHzJezZrxbrRPDUQSt1dG9+gmtN78ApN\\\\\\\/ZCsKVEK6yd23vVoLrSnCDRBCa2uQ8W\\\\\\\/kL+13d92ujwdJP9Moq96jtL7xwWcumbf7gEv3N4Lwy7sPOh6meYEdXstTcWu4sQAhRrmdoxcp1mKU+C\\\\\\\/YYgUo+X79dOOo7BIl9Ri+cZJebT1V2Z\\\\\\\/IgjdIA76wAbCiCbGX5SBwRLEe8OguSKrm3NZs\\\\\\\/KlcDza52dexfr56RWk3YZ6w9gXJi0584yUqeQDDng9ViSu6WxnXwdWtIT0uBEmdidegECIZ0ILLjNYmOJN0O2Md1Xl+z71jabriUNehoXpMYq7KIEEU5hksGkGuO9lnIr\\\\\\\/9ObFd2+kDBxqZf5cfzhmI0NZX2REAh773f79pA8ZhHBp\\\\\\\/Caxaas\\\\\\\/lcNW6P7+sQQzeqj2efAz6pW+ZGXncxS7F854rHoQTAzt6NGh8gNX\\\\\\\/xyKDuHcLradiSertNIbiZUWyq3xTHCJPq1DuO7q6sZn8PCxx0HU6siYOhXecAESMAHbnVvy1lyVm\\\\\\\/QDcM4QW0B\\\\\\\/aTPfNiOgDUb3TJXGEu24WakW+9Mkpf55m8dt+Dlpp\\\\\\\/CsQ2Ls4scVH0u7JWM+YZxZXFXekfzHs5rjVUkMkVSZoCixmVbfIxz2oIvPHKOEhRPE70PmJOKHoOAoGpwgBd+ul7\\\\\\\/xYy70Zu6wonh7z4RDQNSqoBZ+o2xP+QOcoKfi4VfmSfMLvT9MBHXDuL0bY4odRi75y9EUmfWAOxFMo7Iv8qP5Qvk2a8g+piTQ7USB1eXjrxCucKz\\\\\\\/lbDxc0wMSS9d7eEI4PWNuSy4JN2lI6etmO5BBuUoL8R5tXNyQWuTlhoCCzjAJnIsqiPPiN8m3zKyuUE3xyYPhLGnhw7RjfysB9ur1j2Ki8xUW3X5OqOMI1RUIDIOa74pHdaPgJF4c8KZWrtBBdb7TW\\\\\\\/wEvY3jgMVXcoUVNxx\\\\\\\/tEZmXUZT3TNLPQJKrPG8azEmo+VWk\\\\\\\/Qx31Lh\\\\\\\/jMgfhm9KRYVWhj29WTk8OPMmx9HvwsH5ZcwdV3Qwr15mO\\\\\\\/izKJ\\\\\\\/ZzqBnSqSVpRfcVxIKZA8SaIrT\\\\\\\/8eqUBgkfYFiqqzLfloIn\\\\\\\/6U\\\\\\\/Uh5UjbkXVQF5k4ds4fI7iBdmw5Gazk5KQ4cMVQlD1mi\\\\\\\/AjghTe\\\\\\\/Jzl87iywGgT8Rpzt2NHY6YmdcBio7+aWrsjnGJstEOXZbDWUVoOsQiAKjtEGc45dLF9\\\\\\\/WpnxbFNd5vQKv6WcHddJA2p5rzy3xs6nXkMaxWS+PV1DhtDRPeMsVIqK7jFhnzFqaQ0IMy43iKYdxqmHknMNiu2ybgwguxfw3lLhay1eDWhJRcwHw8U52s0t\\\\\\\/bsMj\\\\\\\/SzyOym2KpXutGO0LftvuHmRl0f8Cbtp1USU6b8MJw9Enq84A54IPcGt3nJTKRBVSK5hYnuIgdr\\\\\\\/6OrYq8EhAgFhQ8xXJ34nWtxCB6GUnc+zbzy04JUPgRukXX0khncsg1r11TP7fzvAsNfJs6l0dXwfa4V5+LTvUJyeDNHfVm6ofHfzFHwUpsWWLCBlkvwWnAvSeYZOtjxS2cndBja9sIDpjlDeL9li58lJportGFOx+Al0vFgHF3cMrqswVEOigFQrrIk0TvnkyduUs\\\\\\\/GwzDuObUc+m5Td2Dvw6Wfd3Xl1N6RoHaiGGnN0sKGVDr0kM7Q1wEK7\\\\\\\/qyEJZvM\\\\\\\/oe3xxRaGCMDYRUKobkN\\\\\\\/PLIMxvyfh85EV51fb8A26SA2U0y63OF3NGbM\\\\\\\/jXY0ChjCJpALVHX3gI\\\\\\\/56Xwdox5P4QNNJzattQjyWfb0ST4R8bVA5XzJXCj20XfHQSdFa8IPiFbn\\\\\\\/JNRGo174BhB\\\\\\\/jWs3etsoDeLlObHEyvVUQ6QK7Z1iutc13mJ9jNzJBHIBGQ1wTK6cGtDSowiuQrzvYZEsXZBzDfYb+iwp0PGqkAZKmcNDFsTmcbmB9kxHVa5++VuvO2vjnCNUJJ+uPQ1UQ6JTb4dU7V4KA2bXy1Y\\\\\\\/ddrWrZujEl7ENU0XVq3SRct77U9C548qRtk161GoNdmLCk+1t8aPSjS5Ro\\\\\\\/WvY6CigBUJb6+NcvFfeuOwgxRiDQC5AKO2KK0RYB8Uz7ih5UHldDWaEL0bl013uvgHHtOXuU6BcdiXXcTgRqSioh5Xm11PcrND\\\\\\\/scmfVfe9POYLscVf18\\\\\\\/0eGVAXYAABI9KRvXx4t+fuVWWiSYNQw8+VQpT3j5xKebeXdG7tuAaBsGtl+x7\\\\\\\/g0CXwodEFzKYut1kuKkjOpDn4uFLUswIgeFRmWV7Kv05t\\\\\\\/IFGhzQk\\\\\\\/rnwsOVk8H2OsLUkCzVqFN48\\\\\\\/vZDxyLguz3QBCRgaPRGHC49VmFAcqsNQGlsGFwCuYOa8F5EG3FVx2rG9yNss1\\\\\\\/sxHs5BMotNPwqXXyrGgCi0R55Qt4NpOUnl34Hv28uzDQR2L3l47tb7b41HY4Rjm2Nc7OKB7HEsCCQ7dLPYQShOunFopzqRKgfiy3R23Fp+Cu858eKFoyGd06yoEBzb3n93RlspWbl3qjpjJzBnmjfDay5hipzOS7eWd1QD5gpL6CaNNdNkxXERx1uA2deoaWaPdy2xcG9ivSyLYMuWbQciLl\\\\\\\/KD37W3Ljg7ThQ272k1h3s2flBIggijJS2thZFInbpfYHI0FLcGqGJIMTWeWXqSv7tqRFlLE2vpH5atcEoyY7PyB5z+\\\\\\\/fuV+v+hhSyTH6VriKlfU4\\\\\\\/\\\\\\\/Wn8POxGplojdK1esedH0YE9kfZ1wmtp+ePcQUdorSKiK9vG3sWLKiLPVq1lwsavH93dN\\\\\\\/9CBMD9HC6aUJEIT\\\\\\\/Btve\\\\\\\/uloYlNBqrxmLsVOcHiIujRRuv9RdUMb1j\\\\\\\/\\\\\\\/x9czSwgHUCzkZ4IeGToEJ\\\\\\\/NU+3DQyZbdepLsJTq6mp5CMrRSimHS81+cBssgZRsWAvkhscnrfxb9qapr3POW2NDDdwpBXEeOxAST288iuxBewJosJTfqnvi5GKKfMNXS9MRnaGwd\\\\\\\/eCQsFhpfD2dIXgFwEC6w08yUI1ZgkR7ydu0MD0lnE1sDETnxqWuMlwxQfsgHWm1EIfXg\\\\\\\/Qj4Xqpdxx3zmnMt+P2TqFjNQgHiTtWkuByecjIs3Z9wuHGeKi94KBjv9sQB5GR1PIwGk\\\\\\\/6gTihICJW+fj6idSepC\\\\\\\/ie\\\\\\\/sim7g8zC3mnS2yJzdUaUSIY5AIPDD6VMEMfCZP8CvSxOhxAoyb0hp+YTi2I\\\\\\\/C9Q124FOq9cUfznmnFfzpOtj6cqsv1wZamqBBwNwd8c51Kn3vEXz0fldnJuJFjIetpraqrp+wXNCvnRz9UZQYCnoIUNsw3+2y2ld3\\\\\\\/3v3BBL\\\\\\\/9y\\\\\\\/qvstUuUgXCJF+hjM8nDdvrf0M1XaMI7qEz1OdNk8vDLxyIZ\\\\\\\/gTxe+ZSVrZwRhtwiaO\\\\\\\/L+EGKq\\\\\\\/KPgI95oV9409oLe9mrg4Gn0oCMSVorvSeZKKhhkNKBBcab3n\\\\\\\/079JANcCelg43YwkVw1KYpIlsayjM2c5nU7vuuXCy6n+FqhYl5cvIh+orapnBDGpvf\\\\\\\/+\\\\\\\/eJ6Z6c\\\\\\\/8sSzf9usjzyKWJTaE\\\\\\\/vu5\\\\\\\/bAuqK\\\\\\\/GdemWJx\\\\\\\/cnI6G57+ke6cudSr\\\\\\\/5rnxNRBkpDhiMu0r47nOfwdLGeGVWwri572fPjqf97UmdrwhnmzSyVovU\\\\\\\/AzOLQjskumcwS5VZh\\\\\\\/bTtyE4DOHDHxzD4mX734xVfQh9XgqzUnC5PejgFArTDOq3POeqd3AaYsTT6bfD0XCKTKCs72M+9l8lJYuJ4SWPsBYM49RKBd66bONm2bxpLt9hr78kqPw2M95L3k6xws1VPiHXKtblwemXg2oP8vmI1JZHtPi23RHSYoXfKuy51exfga0daxeTAKcELQb+yhrMLJm8Y5Y+ZGDeeDUO0MU\\\\\\\/z7MS681ruih4Oj2t\\\\\\\/URFsCN7ERYqulrmi3kCDqncMJ+AgX0wNQ2\\\\\\\/6hwr2XJvdg4WK3OtjXaUoLojUmrBKyeRM6Ov\\\\\\\/1P7eu6ZjcO8GXkNMf5CejdBdAerdjOWlOM343OAALa9kfpxGf0xPG6xm6njbfqK+HI+nwv78l0chWuJCrrenc\\\\\\\/xnEj8auLGv01ORtuN\\\\\\\/tPaZNffu1QZGR2eG3sCgo4A3KtbWPw6Vk7d50+ALr9wUwnnc\\\\\\\/gz24mL2m6BgNeF28tFRvCYCVVsannG3z7FmoF5NZ5MuOnj82ZoUZqf5g5X7Bo1DBuqlPo+6XwK0XgYwR2xQW4ZTkNArxFTCtOa0eF\\\\\\\/xiGzZITUsXG6DNxzPFelMEZctw0GTG9GG1JkgSZqaLUQCe4a6EpNAeNCbaiWiZuP\\\\\\\/SXWZnJfPIbXhQCP7NBeEIR0Aap3GR5lz\\\\\\\/gc6Xo\\\\\\\/OTZveNiE4VxEITr69F8Vo8DDyK4bvRSx9oMiLJWJQPiaKIielfDepRWCxYRuweDGTV5oIo8zjdX13Rh9mxHnz0zqNcrWqKpG8XJB7FeHwBTcV9GVG2V64K4DGrr2UgH+O7lb3O6o+WX0brEmdvongJqky6RB3t0KFmIw0DZeAVCLXxj3oS\\\\\\\/McYUmzjiCP5JUvKSUb\\\\\\\/6ym4a4uwngIT60+yXmRVs2csLOxXgUmUkH2Z\\\\\\\/JVsn4QJZF1K3yblbR7Tqbk3aBpRrI5llKQ8uyLsMChy75HH2SJRjk1x53SS9ovaOr8SC\\\\\\\/aiQPvnUEU4RpQGwdlNxF\\\\\\\/Vs6T4kxL6rZS6oDBoMkqnpmw\\\\\\\/zIZ2cSo+pu0xHn47b9RPuuYE9mvYqzbMgzvAl3CZgZ1qqyll\\\\\\\/L2k7Ylu0C+zFe1xm\\\\\\\/1CbSQB50Kv4KqY10jjl6V95OhCK\\\\\\\/uA0VrXP+LLV0MKQkqS2OUkgaRptZ+f7FUXPbyLf9M91y\\\\\\\/rW2IiBhqpdTXhfaMP8803hNHs+zzZLq1jGW3jN7iM+Qgd6E8MzP1eVB9Ubp\\\\\\\/KIkMWX220SEpY5GRn4ldtAYwRVvukD\\\\\\\/QPaKDJID7sjHOx7QpTz\\\\\\\/2jaypv8jEiOltiF8lHyMKpr\\\\\\\/ml3+BvLeOsn8DDCUU6mhMpEh4mbUzIr4oNcS7CvaBxZSbMvSe8DoASb3SWaw+l4Alzpoev4PLtXIwAuS0uFkeaeFCu1H0Vtyzy9Jz6Xr9j1TBgWeZhsJxsG315gMw8RDtEjQxhzjX6lwq1hEjcDTdkn+NLNtW8of6WIoqIY+IugcuTDV4uw1cC5+xLLlrTX42AqB0+3t1RgF\\\\\\\/xUBZvyOxnqjd8wGLybUYAhc5DFLDI\\\\\\\/Q3rkn46OpF\\\\\\\/U8YzYUYkDD8xqDVeMRb3y++ye+Vm+sLtVcw1GHc0wCUFFNUGb821CJuzwBF03j\\\\\\\/sWX+xln7jGcprCarlMjzZejMg9dOTOpD58rgvPD4Izmt+\\\\\\\/4rodWWybACL6WMRlrWiG5WKlOFlH71eTib7hjtGmE4nP+VjL1G8YpoYeNCdSN1DPcI1GZlh3X86HweauB83XQMCet4a\\\\\\\/iPogOhsLUQlR0hZ4956yN20Zn674xI\\\\\\\/+MYzSyG5HawnBv81COWOJhM3sHdnjpkf2Ywom5tzkUHmtaCGcKnHuWUYx7rEQP2zpVTiPyRzZKno7Eh60OgBccTqYMHezlM2uZCqANDoqK95kUxzWsQLKuZ9PLxKlisOIttwGMQAiqh+R3SZRmH83J96KyncxFMNa1whT3Y\\\\\\\/+r6wI0FIhPaBHtBItTwFajsc3vkExnBB5i\\\\\\\/lyys31PMYl\\\\\\\/ZqR\\\\\\\/G46WLxxh5gVzSfhhAOe2af9xt7tUox4ScgGoK3tc0CMEbW0Ulxa0vkRVXmI5w4+8t5idjT9j3yFCurMUQprNb61xhy4vM40szJuK5dg6JOdF6VALXtnaZz0tGW2WhUWBoYOPpVroWED+b0nmeL6C7+qO\\\\\\\/o5wzJhHb9Qz01iDhCBWgSiHNKHy9xtYrE4ODh9xt2lb9qoNkE1tWbfmxfxNGbczrKTLuWPwq\\\\\\\/twpwOq98miGdrDWYIqRRsSZL\\\\\\\/zZgXF\\\\\\\/9SSeQUlw9ls+hI6BHXCYUDmTy5jttil1KoAjc+A4HoIzQfXi3b6cn2A5unTp4yPBIKvXQRfFZ5flb7C4+CJtYpP7SQJs\\\\\\\/35bYb6d7g97+T4rxhUNdLPBtlKr5eRGxoBEsWxh8ngZ6EFW3eYBv\\\\\\\/wPxeO33D33lxFmlcPJ6\\\\\\\/79CNMfd0eWENorHqI0frXr2HGm3EfJF97ntpYatB6cN\\\\\\\/ID7ct0lOXw1QSSwR53hBHzuk6ef3H7ljiAdZtvbFJktSmdFpsfWcllzZ1nyd7uTrVofCuJgiyVZLOMSMt+uYUUL7xdC7zWmzpbBKWYIUXJE9zyZHvaE+u20OwAUqkcONjSFJ+QSdLP5Runvbg5qBYLi1uN52Zfwl1i3Br7HkkWIlX0wzjnQgJtrhyFcDYchiwXw8vstwM0Fs1phpV3mVvD9CExihGLnXlKsLSV0pG4aDXAT4ofynhEL\\\\\\\/9ZeK6TH4TWeBP4w6hf3HvyviZsZ6yhqUhTNA72Oo69svdhVD60QnNxF3TwthbTgU\\\\\\\/YVHtPtZjl64Q0onq6L9E\\\\\\\/3+smLUBfGrzpuXJtx0ez03+NOTFOuRxhJviVP\\\\\\\/y3K3HPiKbrAMOx8+HSq7ngwBIM6Gnug3Ez2g\\\\\\\/E8rTG3uf4MEM2tyhwmtOVuQW+6RGXO2ejSJl0k2d22h+3C8hZ6dvyJU5ZMmY23kIAa9TpXzXaW6s5ex\\\\\\\/YAPfPNDTBhJnjZK4TooP8kcgq61pyzGMtFG4Uh9+aN+LqNYHhU42tSyTYuOCoC0C4BRuYbMWbztuF4bdSMelJxIGxConS9CTx+OBY\\\\\\\/OW3c9\\\\\\\/I3K\\\\\\\/pKbzRW52+RgQvhJg8UyRQ4ru1izQZRFc1mVR2tKDcNFQswlJs5z64LoubLqmzNBTCOpXWu3tM\\\\\\\/ogP5q6FRM85ud04kTzHQK3ixSKWzgPs02xVn1sQ42Xk7ovLUebBhYtr+tJQ7o1zolsXlVMpVxgvlL\\\\\\\/dGkm6um+I1zvIYyX3vt3b6Emchaz9dt+gzJRFpD5gF7JzQ6JH5NulGTlKWoCsOraO7Ybt8iG0+\\\\\\\/UAt7kx1EO19Qi0icDt\\\\\\\/KiPnTcRWCd7Qs1t1JRL2CiGsYf9AOC79uwuo30AUBfEL5rvMDLE2laW722oBc2Zhvi0dg38qGA5ILwKwcJqVWSGhDbLwNQNshJKFCwIs46+wNMTlNMvnOIS2noW2ac6a3uyRYRq+NYh6G1wKZHgUmmH1ULIv+2S2dL+28\\\\\\\/3GcvZXIwQV2Q7wx1KZ8gk9rV07MtqQEM7PgUxmhV3TzJ8aTYymgwRJiQCatnByF0PBhlZ5jgA7F+AQCS\\\\\\\/+KWtJ1cYb\\\\\\\/7iye6q7xHJ+314Ma7hKWqqu5\\\\\\\/d5v1fTu+AfSU8EHbr0\\\\\\\/ueJreOuXfUNP3qAfWt2n2J6Mm1SUdRTGKDfAFtJ\\\\\\\/YYSYJlLiXgRGgFCyjm3rXvC3Yk6n9JfglHUIHcTuDwfnDgksD4hZ\\\\\\\/YmgbGJq1QYTvat0m5muJNpyp1+eCIhvHI3atoSHfky9mnGQTqpJ\\\\\\\/qY3oZ7VJ+SjIoViJBSB5gXBMoGfc8gqbWuGrakyRDnB0S05Ueg4iMOpWr+E8C6gWZMaCwM8kcFL8m3AOMJ9Ie1WI+Yi4HyU2lwBvTgcKCotf7SEBqVUSTEy5jcc4aKnIvcec07UOQ46YwPHuv8cijcbkZolVvPypi9yat3bTuSjVTn\\\\\\\/Z4yZm6zjf7jAeUDmb0kNz+LkNgF+l1PK5kjqpX6m2fEOtzI8nZJPsQ2AevgdszhxwIFg\\\\\\\/zxyrM8r4dpJiM4kWUX15vXvnky7bdmKcTAQIAKGitgKyPoqwz4DKttmJcgccqQ7nFNG1W47tesi0DeWq9pcwk8vtHQGWk6lNVz68WGN\\\\\\\/eVJyQaAdb+7UTsSGlPNABEtOhH3qm2o+CWjcw83LH6D5DIbpSCWGxu+8Luw4wSETHlb9\\\\\\\/o7\\\\\\\/Kd7G+tzFQXvTuEwmz0Vaycg13kBPQ47GTdC6M9FwTofdIh35YGt0fgWm6awngsaPdBy2Sq+FxVO8Ch9\\\\\\\/mactZQj9JHM2MddZU7gfX914f7Sj6BSzevhN4iIh8MkAIBqnKeeY9dUz\\\\\\\/TAWb\\\\\\\/+YKHD7IeuHVQyp9aVPZ4o1UcxExO6FKrGB\\\\\\\/\\\\\\\/sTaNzLAhTvWRYzoS2q0L24xPsLcCLJyJuV0YA4GlZb0WSE0qOtYu1zwZcSMqP6Jyco4NXQ6DQbjlijh+eCI5kbUEhPmbyrt6TLj3nHPQYuwmr4aLs45a1Rn38b5aXDsSlxa5sw\\\\\\\/\\\\\\\/hQrIVR+KzgZQIa3wefgkf\\\\\\\/70IFr5m6djBNQJxmu04J4v\\\\\\\/CD7SYNqqRW+\\\\\\\/RkGj7Kv7j\\\\\\\/HHZcLj52rYTS3HncvuXErzsJQodjGblKiTRV9Rruc1y6hUNnWteicNPCXBAUV\\\\\\\/t8xtWYS29Xjrg5UEwaJ0LghjyY7usC+5ZHZ3jW+68S9knJ0u3D9MwIS58AP1fQKXZmW0Pr6NJIChL+RWjCS27r+ZR\\\\\\\/L1gz+onhmHI7KRxSfz8khcfbbHJ36CSqHknMkI+fyw2CEwJ\\\\\\\/I1ReF1ttvL+ajEWkvnGmpjPNkvROVSztVSIEyYeBG7jWyj5b\\\\\\\/K1jt7eYnok3RnEniqOofpQ45BCsEesq8NQrlay2M8mqs4Xxs20hf12nwhVs5o4X3kpwvjgj1HjKYoHcU2zsLTp2rfUcKAzuHchFIEAmK1ySsfL3mFzbKsDq8sWAaq8iLVUqvr6xxFaHesFxmlFzukiw4XOjtmZhMhGAaFvyUFz2HbcQbEtO44ZFtJyORalPpHA84WLA\\\\\\\/uWfPpBI3qq95j95lbW1UzCw6lvSUZyi+qNtinewjyuHeDB\\\\\\\/5LDfPag1vD\\\\\\\/uD5hYHy5aEWB5ARR1fnPzROleDPi5estXa\\\\\\\/C3RVG8PJDhcI7NCgUoBMVx8TB9dAHF2LabL6PbNT6PBR8uUENahOGvdymh2zWs+Dc1IMz4XrNvLGVANnzT8A9GHKmzzq1T9HsCqkeE68HNUjyAVpCXEevr6AAFjgx49HPNYVarIMcNJXnjqGlhvZ1+vaoDOKdPgtxHQ6BuIYyv\\\\\\\/LQ8aO6PBZTNYCVcLfcvK6q5tXiDwjjEf\\\\\\\/CBZDbwy3CUdNdvQu9spvI65oit1bv92WHpEeck92d2NIbjCcWXh75qKwBXlSaQFdzhKwolBfMNdCt0NV80MyHnyN0QDfl81gOeia0b8SnmTB3TOnjdoV9\\\\\\\/T0NsuntuYi9z+mK+fCFipkJIGpaS0SDV0Qt9VetvKrutHTqg4vlP9AKQZhbh28oVWQdVk1+ws+4JNA4ZWsgVFNk3U87iamLnduyZeHQ2GCvf5BEjm4PHM91JvJRpRiXX8CslxMVgrmqs\\\\\\\/ajqwj1rkKKXpeACh9jroNtahYZ7xzp79qgeSbYtrPISVfRjGsgXil\\\\\\\/3RLCcncsIYurFdvdIIoW9Yb\\\\\\\/6eDamPeo3feLj8ZpTIvp31Mscy4Vdykm9FHt1h4Fo7AOGwBuCI\\\\\\\/iwkiMQtG1k+O6rBcrKqu7WC8WIh9SJMqUiLUYyorzkY9X33G\\\\\\\/\\\\\\\/XEAnbj4uYNcP4toZz4SwqRTmiGVJK8aNOSHRbPKK2n7kBsdfAx4B3CntCcyhN4oQjKHFJVWysLubMeLUNeOkO\\\\\\\/wpuSsQnkWMSVp6i0U4rhHrt3ZqW1YaXI1NrZ\\\\\\\/oTYNQGMURjLkQZg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022BOZX4Pol\\\\\\\/M6rLERzqZCP3g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022eda29f91a32e903a098ec2943c1cd34255ff56982e9d31502795c9985fc552df\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Tin29zeU4VzMgSW2M35bkw1Nq9FnK1flySm4sggTNAPCzzczdmBTwzYxEasWSG5Z6MaaFX4hXrO+7eZpurAFFImbbXL4wfVwLSL+U2gaxn+0I1naPB8tBBwdRO0jqrxrwRi+X3VCQXxumaeEi1AxFPnKhW9dBn9fRWprRDVeL9eGG49HEqDCuD7QQE+CBRKMAM43NzVuw9Kkxyp2D2sO7PIcoZQdBWcjI1DIaWK6AbHcFTT6i3hWUnJnzzVzJBfoL6KotMSSRyOCizUbuitUe9hxAvY3bWjYhhVw4pOE+mU2mq\\\\\\\/MqRKxxxumQmxpMkoX4tpERVsFwilw6TnToxux9l+5\\\\\\\/kNgbQ6d06gaRfrtaLW2Zld4kGAjFxAyDaDGZli+A73mW\\\\\\\/ME086phRM\\\\\\\/JsZQbFojSyCrD9SbJSs8i7ruPqWu5xTuuqLgt27yN4EoKFItduizeH2JDe6fbr9U7LoxYFDzPSebGvD7IUpvlymh3wJs3ScLFfsD2DjGFWu6\\\\\\\/fRiwqkrHUy2UPOuKbS6ImuEzDa0CAROLE22M0WfyuYbP\\\\\\\/FGiSzi6Gll8HRa+023k4WhPYIMADfq50DZ4Jav6o5j\\\\\\\/d+QYpORRhvLAs7jVFMbxGHucGhX0EKLpCfDZfjICN9hreKv+AxyS\\\\\\\/QSFamyeMZsmAVCboaruBwSriaDME5se7feBKPcgp0dohb\\\\\\\/izdO7vZjBVja7CCqrlfcwrB27kFUOLwW5xHTZibfIlY+U2C6moP2c\\\\\\\/y7yygdQRt2ZS\\\\\\\/jh\\\\\\\/8jCd4C7ZN6owQCvTeMHaeFFCHAF5PNOIBtSeh1QzuzX7AbXbNXIA6wap7hErnEohpMGpUSFwkgXISisM0oLp3uOMMKLEmKbabem6udcvbpAcBtJek4oX2h2IKXQWxu8v3TNcegnTWPUEngmy2XUka5YIEDjLMk63NyyhsLFqMXGGiSTbXehH7A0ZWI1x7MV9Z0ZDvgA5VKw9aU46srW+RP770HuHzD3of44646BBppkb96gDai1mcH6VQNttDCWpiezVqLM5TsnPssMME7PFAlOGH6mv+Kgeg4Crz0zFADhpyLBqdiJfypyIyjwxpvvPH0IvI9w8MKLjl823jehcG3sPUic7qUKeIq7JzP4tQXkz1n5GKcpk9lzYtjlqf8VVX5QnPunXunBvBQEuXfnj3qEV\\\\\\\/1lBXoxIweSvVbIKSGx2NK2MXsDamkCQwDsGFuk3IpHVfH44kiR7rpQHBwScg\\\\\\\/3S2jokEZPrtykMfTGgespfOo\\\\\\\/vvYHZUrXAg4JMxIShSBUbdM3Yu2h4lkWamkmflJFG8ElOI9aLeGgLFLd9eUA9UyGel6pd85zblwdRgrZog\\\\\\\/DCOJuTMPvNzmBLGZFoWIbUvmRs0CtGw+kYwg5fpVrG4t0eiK4LhBh7kInVAcxAVhB3QAcYqSfmV87mTi2nx+6bQyrFD8HuhVEcO2TD0r00rmqo1haMCNxKGqgz2mQ3G+YqYNBSBCoSegy17LUIlnDKaHA3\\\\\\\/pdyMYHuGD1JM1ST1139B18hhsfmKZPfOL8GR+0ft\\\\\\\/kJrN+2Q7j91UaPsDsmILppLMmB0wls\\\\\\\/h2g+MSKxvuKkQliedYj\\\\\\\/AQWLlrw5mTDx\\\\\\\/T5YUrOz3DKPE1PohkHe+y9t9aMJmjV0s+pDrtpFXc4dQLSuonm3GV+8Ab+MrjDHt7nvT7yLeRnEOlxGFHXQYMI\\\\\\\/TG3ZZKIi6YGGc5sphZWOfAhAEnEbv8ZOzXnlspN4tiZ+D+Sn6hGs5fa31v05avbSXP5lisnCjdk8EHiUsf6j\\\\\\\/0lo8LaNKT3M+EAIUqxxJ+3lBpxt6\\\\\\\/E1QfETmeB5Yw0U0bKT+mu4E8pl\\\\\\\/Uo8YQ5QJ7YyN4bpeLt63\\\\\\\/kmYWPxYf3xwYoG274MG09IzYXJIBSA+DQUcgQe+HdNK44\\\\\\\/RdocrSbemgfCx\\\\\\\/l4i3HnCLAwRkp\\\\\\\/gwpHfC7BV2Y7yyAMeF633Ry0nOJscBjs87YeSSCjvT\\\\\\\/km79I9uBWUB7l3dhU07OHDSP83h1BvccWavw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022HxLdwAbeeFGpHu25vCkB2A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226dab5f137385db55bf3cf11ccb898c7adb757528a6be1d7e550f26b15883cb6f\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460082 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/3/509" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-1eiksE3G8Z9ejg7u8CFe1YzkPkB0EU8OzHg5Nxk_1Xk" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/3/509" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-_8thloIw9XWLYgf16BAanxiuK4khaDvsmXpEodPKh5Y" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/3/499" title="Partial Dominance, Overdominance, Epistasis and QTL by Environment Interactions Contribute to Heterosis in Two Upland Cotton Hybrids" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/3/521" title="Whole-Genome Sequencing of a Canine Family Trio Reveals a FAM83G Variant Associated with Hereditary Footpad Hyperkeratosis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460082" id="node460082" data-pisa="ggg;6/3/509" data-pisa-master="ggg;g3.115.023432" data-apath="/ggg/6/3/509.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Exploring a Nonmodel Teleost Genome Through RAD Sequencing—Linkage Mapping in Common Pandora, <em>Pagellus erythrinus</em> and Comparative Genomic Analysis</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Tereza Manousaki</span>, <span class="highwire-citation-author" data-delta="1">Alexandros Tsakogiannis</span>, <span class="highwire-citation-author" data-delta="2">John B. Taggart</span>, <span class="highwire-citation-author" data-delta="3">Christos Palaiokostas</span>, <span class="highwire-citation-author" data-delta="4">Dimitris Tsaparis</span>, <span class="highwire-citation-author" data-delta="5">Jacques Lagnel</span>, <span class="highwire-citation-author" data-delta="6">Dimitrios Chatziplis</span>, <span class="highwire-citation-author" data-delta="7">Antonios Magoulas</span>, <span class="highwire-citation-author" data-delta="8">Nikos  Papandroulakis</span>, <span class="highwire-citation-author" data-delta="9">Constantinos C. Mylonas</span> and <span class="highwire-citation-author" data-delta="10">Costas S. Tsigenopoulos</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">509-519;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.023432 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460082" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Tereza Manousaki </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tereza%2BManousaki%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Manousaki%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATereza%2BManousaki%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alexandros Tsakogiannis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, University of Crete, Heraklion, 70013, Greece</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexandros%2BTsakogiannis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tsakogiannis%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexandros%2BTsakogiannis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">John B. Taggart </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Aquaculture, School of Natural Sciences, University of Stirling, Scotland, FK9 4LA, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BB.%2BTaggart%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Taggart%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BB.%2BTaggart%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Christos Palaiokostas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Aquaculture, School of Natural Sciences, University of Stirling, Scotland, FK9 4LA, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christos%2BPalaiokostas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palaiokostas%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristos%2BPalaiokostas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Dimitris Tsaparis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dimitris%2BTsaparis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tsaparis%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADimitris%2BTsaparis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jacques Lagnel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacques%2BLagnel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lagnel%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJacques%2BLagnel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Dimitrios Chatziplis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Laboratory of Agrobiotechnology and Inspection of Agricultural Products, Department of Agricultural Technology, School of Agricultural Technology Food Technology and Nutrition, Alexander Technological Educational Institute of Thessaloniki, Sindos, 57400, Greece</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dimitrios%2BChatziplis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chatziplis%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADimitrios%2BChatziplis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Antonios Magoulas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonios%2BMagoulas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Magoulas%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAntonios%2BMagoulas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Nikos  Papandroulakis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nikos%2B%2BPapandroulakis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Papandroulakis%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANikos%2B%2BPapandroulakis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Constantinos C. Mylonas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Constantinos%2BC.%2BMylonas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mylonas%20CC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AConstantinos%2BC.%2BMylonas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Costas S. Tsigenopoulos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Costas%2BS.%2BTsigenopoulos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tsigenopoulos%20CS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACostas%2BS.%2BTsigenopoulos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:tsigeno@hcmr.gr" class="" data-icon-position="" data-hide-link-title="0">tsigeno@hcmr.gr</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/3/509" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460082" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460082/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/3/509.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460082" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460082/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/3/509.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460082" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460082/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/3/509.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460082" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460082/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/3/509.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460082" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460082/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Common pandora (<em>Pagellus erythrinus</em>) is a benthopelagic marine fish belonging to the teleost family Sparidae, and a newly recruited species in Mediterranean aquaculture. The paucity of genetic information relating to sparids, despite their growing economic value for aquaculture, provides the impetus for exploring the genomics of this fish group. Genomic tool development, such as genetic linkage maps provision, lays the groundwork for linking genotype to phenotype, allowing fine-mapping of loci responsible for beneficial traits. In this study, we applied ddRAD methodology to identify polymorphic markers in a full-sib family of common pandora. Employing the Illumina MiSeq platform, we sampled and sequenced a size-selected genomic fraction of 99 individuals, which led to the identification of 920 polymorphic loci. Downstream mapping analysis resulted in the construction of 24 robust linkage groups, corresponding to the karyotype of the species. The common pandora linkage map showed varying degrees of conserved synteny with four other teleost genomes, namely the European seabass (<em>Dicentrarchus labrax</em>), Nile tilapia (<em>Oreochromis niloticus</em>), stickleback (<em>Gasterosteus aculeatus</em>), and medaka (<em>Oryzias latipes</em>), suggesting a conserved genomic evolution in Sparidae. Our work exploits the possibilities of genotyping by sequencing to gain novel insights into genome structure and evolution. Such information will boost the study of cultured species and will set the foundation for a deeper understanding of the complex evolutionary history of teleosts.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/sparidae" class="hw-term hw-article-keyword hw-article-keyword-sparidae" rel="nofollow">Sparidae</a></li><li class="kwd"><a href="/keyword/rad-sequencing" class="hw-term hw-article-keyword hw-article-keyword-rad-sequencing" rel="nofollow">RAD sequencing</a></li><li class="kwd"><a href="/keyword/ddrad" class="hw-term hw-article-keyword hw-article-keyword-ddrad" rel="nofollow">ddRAD</a></li><li class="kwd"><a href="/keyword/synteny" class="hw-term hw-article-keyword hw-article-keyword-synteny" rel="nofollow">synteny</a></li><li class="kwd"><a href="/keyword/aquaculture" class="hw-term hw-article-keyword hw-article-keyword-aquaculture" rel="nofollow">aquaculture</a></li></ul><p id="p-3">The Sparidae is a species-rich family of marine teleosts belonging to the Percomorphs group. Sparids inhabit tropical and temperate coastal waters. They are of considerable economic importance, particularly around the Mediterranean area, with many species being targeted by capture fisheries, and a lesser number also being cultured commercially (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Basurco <em>et al.</em> 2011</a>). Currently, the dominating species in Mediterranean aquaculture is gilthead seabream (<em>Sparus aurata</em>), the most intensively studied sparid. However, driven by the need for diversification within the aquaculture industry, other sparid species have been gaining in commercial and scientific interest. This group exhibits extensive variability in reproductive modes, such as alternative types of hermaphroditism and gonochorism (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Mylonas <em>et al.</em> 2011</a>), with contrasting mechanisms being found even among closely related species (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Erisman <em>et al.</em> 2013</a>). As such, they are gaining their own “niche” in the new model fish army (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Braasch <em>et al.</em> 2014</a>) for studying the evolution of hermaphroditism.</p><p id="p-4">One of the baselines for efficient genetic selection programs in any species is the availability of genetic linkage maps. Linkage maps allow for mapping phenotypic traits of interest and provide a backbone for further genetic studies up to whole genome sequencing. Until recently, building a map generally required the genotyping of hundreds of microsatellite markers across a genome. The newly developed genotyping by sequencing technologies, which allow the <em>de novo</em> discovery and simultaneous scoring of hundreds to thousands of single nucleotide polymorphism (SNP) markers from a single sequencing run for dozens of individuals, provide a new means to rapidly characterize the genomes of nonmodel species. Various adaptations of these genome-reduction screening techniques provide alternative approaches for different applications. RAD-Seq (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Baird <em>et al.</em> 2008</a>) is one of the earliest described methodologies that allows the routine identification of many thousands of SNPs, but requires considerable sequencing effort per individual. Variations of this method [<em>e.g.</em>, Genotyping-by-Sequencing (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Elshire <em>et al.</em> 2011</a>); ddRAD (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Peterson <em>et al.</em> 2012</a>); 2bRAD (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Wang <em>et al.</em> 2012</a>); ezRAD (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Toonen <em>et al.</em> 2013</a>); SLAF-Seq (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Sun <em>et al.</em> 2013</a>); and GT-Seq (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Campbell <em>et al.</em> 2015</a>)] can be employed to limit the extent of marker discovery to a lesser, but adequate, degree, thereby allowing sequencing of a greater number of individuals for fewer markers for the same sequencing effort. Linkage maps using genotyping by sequencing approaches have already been produced for numerous fishes, such as the spotted gar (<em>Lepisosteus oculatus</em>) (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Amores <em>et al.</em> 2011</a>), Midas cichlid (<em>Amphilophus spp</em>.) (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Recknagel <em>et al.</em> 2013</a>), gudgeon (genus <em>Gnathopogon</em>) (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kakioka <em>et al.</em> 2013</a>), blind cavefish (<em>Astyanax mexicanus</em>) (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">O’Quin <em>et al.</em> 2013</a>), Nile tilapia (<em>Oreochromis niloticus</em>) (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Palaiokostas <em>et al.</em> 2013a</a>), Atlantic halibut (<em>Hippoglossus hippoglossus</em>) (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Palaiokostas <em>et al.</em> 2013b</a>), orange-spotted grouper (<em>Epinephelus coioides</em>) (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">You <em>et al.</em> 2013</a>), Japanese eel (<em>Anguilla japonica</em>) (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kai <em>et al.</em> 2014</a>), and platyfish (<em>Xiphophorus maculatus</em>) (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Amores <em>et al.</em> 2014</a>), among others. The construction of relatively dense linkage maps without prior knowledge of genetic marker panels sets a new standard for studying nonmodel species.</p><p id="p-5">Most of the Mediterranean sparids have 24 haploid chromosomes (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Cataudella <em>et al.</em> 1980</a>), in line with the majority of teleosts (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Naruse <em>et al.</em> 2004</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Galetti <em>et al.</em> 2000</a>). The family member that has been most rigorously studied in terms of its genetics is gilthead seabream. Previous efforts have produced two radiation hybrid maps (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Senger <em>et al.</em> 2006</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Sarropoulou <em>et al.</em> 2007</a>), one BACmap (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Kuhl <em>et al.</em> 2011</a>), and two genetic maps based on microsatellites (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Franch <em>et al.</em> 2006</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Tsigenopoulos <em>et al.</em> 2014</a>). Linkage mapping efforts have allowed also for quantitative trait locus (QTL) mapping (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Boulton <em>et al.</em> 2011</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Loukovitis <em>et al.</em> 2011</a>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">2012</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">2013</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Massault <em>et al.</em> 2011</a>) and are expected to facilitate the ongoing genome sequencing project of gilthead seabream.</p><p id="p-6">Until recently, with the exception of gilthead seabream, relatively limited effort has been invested in exploring the genomic background of sparids. To our knowledge, recent work on the sex-specific transcriptomic profiling for the rudimentary hermaphrodite sharpsnout seabream (<em>Diplodus puntazzo</em>) (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Manousaki <em>et al.</em> 2014</a>) is the only report on another sparid. Here, we focus on the protogynous common pandora (<em>Pagellus erythrinus</em>), a benthopelagic sparid found across the Mediterranean and North East Atlantic. Although only recently farmed commercially, production is increasing year by year, and its potential as a significant aquaculture species is recognized (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Basurco <em>et al.</em> 2011</a>). Comparatively little is known about the genetics of common pandora, with only a few studies reporting on the use of genetic markers to study population structure within the species (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Apostolidis <em>et al.</em> 2009</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Fassatoui <em>et al.</em> 2009</a>, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">2012</a>). In the present study, we employed double digest restriction associated DNA (ddRAD) sequencing to construct the first genetic linkage map for common pandora. We identified nearly 1000 polymorphic ddRAD loci, and built a genetic linkage map comprising 24 linkage groups (LGs). Furthermore, comparative analyses uncovered homologies between common pandora and four other ‘model’ fish species, namely the European seabass (<em>Dicentrarchus labrax</em>), Nile tilapia (<em>Oreochromis niloticus</em>), stickleback (<em>Gasterosteus aculeatus</em>), and medaka (<em>Oryzias latipes</em>). Finally, through a phylogenetic analysis we provided insights into the phylogenetic relationships among these species.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Ethics statement</h3><p id="p-7">All experiments were performed in accordance with the “Guidelines for the treatment of animals in behavioral research and teaching” (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Animal Behavior Society 2001</a>).</p></div><div id="sec-3" class="subsection"><h3>Selection of linkage panel</h3><p id="p-8">A wild-caught common pandora broodstock was maintained in culture conditions at the AQUALABS facility of the Hellenic Centre for Marine Research (HCMR), Crete, Greece. The fish were exposed to simulated ambient photo-thermal conditions and were allowed to spawn spontaneously. A male and a female fish were kept separately, and spawned at the end of June 2013. The floating eggs were collected and reared in a mesocosm using commercial larval rearing methods for sparid fishes. Fin clips were sampled from the two parents, and 2 months later from 97 of their offspring (average weight approximately 1.5 g). Samples were kept at –20° until DNA extraction. DNA was extracted by a modified salt-based extraction protocol using SSTNE extraction buffer (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Blanquer 1990</a>), and treated with RNase to remove residual RNA. Genomic DNA was eluted in 5 mmol Tris, pH 8.5 and stored at 4°. Each sample was quantified by spectrophotometry (Nanodrop 1000, Thermo Fisher Scientific) and quality assessed by 0.7% agarose gel electrophoresis. Parents, together with 97 full-sibs, were used for the ddRAD library construction (99 fish in total).</p></div><div id="sec-4" class="subsection"><h3>ddRAD library preparation and sequencing</h3><p id="p-9">The ddRAD library preparation protocol was based on the methodology originally reported by <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Peterson <em>et al.</em> (2012)</a>. The modified protocol used here is essentially that described in <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Palaiokostas <em>et al.</em> (2015)</a>, with additional refinements being flagged below. Briefly, each of 103 separate DNA samples (both parents in triplicates and 97 offspring; 20 ng DNA per sample) was simultaneously digested by two high fidelity restriction enzymes (RE): <em>SbfI</em> (CCTGCA|GG recognition site), and <em>SphI</em> (GCATG|C recognition site), both sourced from New England Biolabs, (NEB) UK. Digestions were incubated at 37° for 50 min, using 10 U of each enzyme per microgram DNA in 1× CutSmart Buffer (NEB), in a 6 μl total reaction volume. Deviating from the methodology described in <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Palaiokostas <em>et al.</em> (2015)</a>, the reactions were not heat-inactivated as this was deemed to be unnecessary, and possibly detrimental, given the high temperature (80°) recommended by the RE supplier. Barcoded adapters were designed such that adapter–genomic DNA ligations did not reconstitute RE sites, while residual RE activity limited concatemerization of genomic fragments. After cooling the reactions to room temperature, 3 μl of a premade adapter mix was added to the digested DNA, and incubated at room temperature for 10 min. This adapter mix comprised individual-specific combinations of P1 (<em>Sbf</em>I-compatible) and P2 (<em>Sph</em>I-compatible) adapters at 6 nM and 72 nM concentrations respectively, in 1× reaction buffer 2 (NEB). The ratio of P1 to P2 adapter (1:12) was different than that described previously (1:4; <a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">Palaiokostas <em>et al.</em> 2015</a>), as this was expected to more accurately reflect the relative abundance of <em>Sbf</em>I and <em>Sph</em>I cut sites present. P1 and P2 adapter included an inline five- or seven-base barcode for sample identification. Ligation was performed over 3 hr at 22° by addition of a further 3 µl of a ligation mix comprising 4 mM rATP (Promega, UK), and 2000 cohesive-end units of T4 ligase (NEB) in 1× CutSmart buffer (NEB). The ligated samples were then heat-denatured at 65° for 20 min, cooled, and combined into a single pool. The pooled sample was column-purified (MinElute PCR Purification Kit, Qiagen, UK), and eluted in 100 μl EB buffer (Qiagen, UK). Size selection of fragments, ranging from approximately 300 bp to 600 bp, was performed by agarose gel separation. Following gel purification (MinElute Gel Extraction Kit, Qiagen, UK), the eluted size-selected template DNA (60 μl in EB buffer) was PCR amplified (14 cycles PCR; 36 separate 12.5-μl reactions, each with 1 µl template DNA) using a high fidelity <em>Taq</em> polymerase (Q5 Hot Start High-Fidelity DNA Polymerase, NEB). The PCR reactions were combined (450 μl total), and column-purified (MinElute PCR Purification Kit). The 55-μl eluate, in EB buffer, was then subjected to a further size-selection clean-up using an equal volume of AMPure magnetic beads (Perkin-Elmer, UK), to maximize removal of small fragments (less than <em>ca</em>. 200 bp). The final library was eluted in 22 μl EB buffer. Finally, the ddRAD library was sequenced at the Institute of Marine Biology, Biotechnology and Aquaculture (IMBBC) of HCMR in Crete using two runs of an Illumina MiSeq (v2 chemistry, 300 cycle kit, 162 bp paired end reads).</p></div><div id="sec-5" class="subsection"><h3>Building RAD loci</h3><p id="p-10">Raw reads were analyzed in Stacks 1.19 (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Catchen <em>et al.</em> 2011</a>). Quality control, filtering for ambiguous barcodes and restriction sites, and demultiplexing took place using the script <em>process_radtags</em> (options -c -q -r). Due to the use of barcodes with different length, and limitations in this Stacks version to accommodate this design in a single step, <em>process_radtags</em> took place in four independent steps. First, the samples with seven-base-long barcodes in both ends were demultiplexed, second those with five at one end and seven bases at the other, third those with seven at one end and five at the other, and finally those with five-base barcodes at both ends. After each demultiplexing step, the unassigned reads (including the reads with the barcode combinations other than that in the ongoing step) were reconstructed in pairs using the script <em>fastqCombinePairedEnd.pl</em> written by Eric Normandeau (<a href="https://github.com/enormandeau/Scripts/blob/master/fastqCombinePairedEnd.py">https://github.com/enormandeau/Scripts/blob/master/fastqCombinePairedEnd.py</a>), and used for the subsequent round of demultiplexing.</p><p id="p-11">The files containing the paired forward and reverse reads of each sample were then concatenated and reads were trimmed to a length of 100 bases. Reads less than 100 bases long were discarded. Trimming and filtering took place with FASTX_toolkit (<a href="http://hannonlab.cshl.edu/fastx_toolkit/">http://hannonlab.cshl.edu/fastx_toolkit/</a>). Stacks were built for each individual with the wrapper script <em>denovo_map.pl</em> included in Stacks using the default parameters (a minimum of three reads to form a stack, and a minimum of two mismatches allowed between loci when processing a single individual). Secondary reads were not used for genotype calling to reduce possible genotypic errors (option -H).</p></div><div id="sec-6" class="subsection"><h3>Linkage map construction</h3><p id="p-12">For linkage mapping, we exported the haplotypes of each individual for each RAD locus using the Stacks script <em>genotypes</em> in the OneMap format (-r 80 -t CP -o onemap -c -s). Only parental loci mapped in more than 80 out of 97 progeny were kept. Finally, loci showing significant segregation distortion (<em>chi</em>-square <em>p</em> value &lt; 0.01) were excluded. The resultant genotypic data were input in OneMap (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Margarido <em>et al.</em> 2007</a>)—a software for genetic mapping in outbred populations. OneMap applies the methodology proposed by <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Wu <em>et al.</em> (2002)</a>, and leads to the construction of a linkage map combining information from markers showing different segregation patterns in both parents. Recombination fraction between all pairs of markers was estimated with two-point tests (function <em>rf.2pts</em>). Then, markers were grouped to linkage group with the function <em>group</em>. To select the appropriate LOD score, we used a range of LOD scores incrementing by one and starting from the value three up to ten (Supporting Information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023432/-/DC1/FigureS1.eps">Figure S1</a>) with a maximum recombination fraction of 0.3. The final LOD score for marker grouping was selected based on whether the number of LGs matched the number of chromosomes of common pandora. Then, markers within each linkage group were ordered using the <em>order.seq</em> function within OneMap (n.init = 5, THRES = 4), which conducts an exhaustive search for the five most informative markers, and then adds one marker at a time with a minimum LOD score 4. With the <em>order.seq</em> function, markers that were not uniquely mapped were mapped on the most likely position (the one with the largest value of log-likelihood). Following marker ordering, alternative orders were checked with the function <em>ripple.seq</em>. Map distances were calculated using the Kosambi map function (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023432/-/DC1/FileS1.R">File S1</a> for the R script including the commands implemented in OneMap). LGs were numbered based on the homology with European seabass, the species with the highest similarity as shown by the comparative genomic analyses presented in the <em>Results</em> section. Finally, the map was visualized with MapChart (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Voorrips 2002</a>).</p></div><div id="sec-7" class="subsection"><h3>Comparative genomics</h3><p id="p-13">The mapped RAD loci in common pandora were used in a comparative analysis with genomes from the following relatively closely related teleosts: medaka (<em>Oryzias latipes</em>, Ensembl 73), stickleback (<em>Gasterosteus aculeatus</em>, Ensembl 73), Nile tilapia (<em>Oreochromis niloticus</em>, ncbi GCA_000188235.2) and European seabass (<em>Dicentrarchus labrax</em>, dicLab v1.0c <a href="http://seabass.mpipz.de">http://seabass.mpipz.de</a>). Common pandora RAD loci sequences were extracted from the MySQL database built through Stacks pipeline, and were used in BLASTN sequence similarity searches against the genomes of the four other species (<em>e</em>-value threshold 10<sup>−9</sup>). Loci with more than 10 hits or more than 10 HSPs (high-scoring segment pairs) within the first hit, were excluded to eliminate repetitive sequences. The top hit per sequence was retained and considered homologous to the RAD locus. The chromosome of the reference species that corresponded to each common pandora LG was inferred based on the similarity search. If most loci of a common pandora LG were homologous to loci from a single chromosome in a reference species, we considered them homologous chromosomes. All links between common pandora LGs and the reference species chromosomes were displayed with Circos software (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Krzywinski <em>et al.</em> 2009</a>). To investigate which loci are located within protein coding sequences, we conducted a second round of similarity searches against each of the four species cDNA sequence datasets, downloaded from <a href="http://seabass.mpipz.de">http://seabass.mpipz.de</a> for European seabass with corresponding annotations, and Ensembl 73 for the other three, and ran with the same parameters as above. The numbers of shared hits found in the four species were represented by Venn diagrams constructed using the online tool available at <a href="http://bioinformatics.psb.ugent.be/webtools/Venn/">http://bioinformatics.psb.ugent.be/webtools/Venn/</a>.</p><p id="p-14">Finally, to check whether the number of hits found in each of the four species was due to variable genome sequencing completeness, we implemented the CEGMA pipeline (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Parra <em>et al.</em> 2007</a>), and assessed the completeness based on the percentage of 248 conserved genes present even partially in each genome.</p></div><div id="sec-8" class="subsection"><h3>ddRAD-based phylogenetic analysis</h3><p id="p-15">From the comparative genomic analysis we selected the loci showing hits in all four species after excluding those whose hits overlapped even partially in any of the four genomes. The homologous loci sequences from each species were extracted with a custom perl script. Then, all sequences were aligned for each locus independently using mafft v7.050b (–auto option) (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Katoh and Standley 2013</a>). The individual alignments were concatenated to a matrix with the perl script catfasta2phyml.pl (available at <a href="https://www.abc.se/~nylander/catfasta2phyml/">https://www.abc.se/~nylander/catfasta2phyml/</a>). The matrix was filtered to eliminate divergent and ambiguously aligned regions with GBlocks v 0.91b (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Castresana 2000</a>) (default parameters apart from setting the maximum number of contiguous nonconserved positions to five). The resultant filtered alignment was used for phylogenetic analysis. First, the model of nucleotide substitution was chosen with jModelTest 2.1.7 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Guindon and Gascuel 2003</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Darriba <em>et al.</em> 2012</a>). Then, the best model returned through Akaike <em>information criterion</em> (AIC) and <em>Bayesian information criterion</em> (BIC) criteria was used to estimate the tree topology in PhyML 3.1 (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Guindon and Gascuel 2003</a>). Branch support was based on 100 bootstrap (BS) datasets.</p></div><div id="sec-9" class="subsection"><h3>Data availability</h3><p id="p-16">Raw reads are deposited in N.C.B.I. sequence read archive under the BioProject ID PRJNA302241.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>ddRAD data analysis</h3><p id="p-17">Following initial quality filtering, demultiplexing, and length filtering of the two combined MiSeq runs, a total of 40,662,844 high quality reads were assigned to the 99 individuals (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023432/-/DC1/TableS1.xls">Table S1</a>). As planned, a higher coverage was obtained for the two parents (in excess of 2 million reads each), while read numbers for progeny ranged from ∼89,000 to 1.05 million reads (mean 366,861; SD 185,182). Stacks analysis identified 10,821 and 11,657 RAD loci in sire and dam respectively (mean coverage per locus in excess of ×180), with 2647 and 2591 potential SNPs being identified. Progeny contained on average 8211 stacks (SD 4106) with an average coverage of ×36 per locus. The number of potential SNPs identified in the progeny ranged from 1267 to 2328 (mean 1949; SD 223). Finally, out of 2947 parental loci cataloged that contained one or two SNPs, 1032 were genotyped in at least 80 progeny. Loci that exhibited significant segregation distortion were discarded, resulting in 920 RAD loci containing 1181 SNPs that were used in the subsequent linkage analysis (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023432/-/DC1/TableS2.xls">Table S2</a>).</p></div><div id="sec-12" class="subsection"><h3>Common pandora linkage map</h3><p id="p-18">The 920 informative RAD loci comprised the following segregation patterns: aa/ab 323, ab/ab 99, ab/aa 352, and ab/ac 146. For the linkage mapping, we conducted a thorough exploration of LOD scores to recover a linkage map that approaches the species karyotype (see <em>Materials and Methods</em>). We selected the LOD value six, which resolved 24 linkage groups (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023432/-/DC1/FigureS1.eps">Figure S1</a>), and corresponds to the haploid chromosome number of common pandora (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Cataudella <em>et al.</em> 1980</a>). The constructed LGs, incorporating 917 of the 920 identified markers, contained from 16 to 71 RAD loci spanning 56.63 to 132.54 cM in length (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023432/-/DC1/TableS2.xls">Table S2</a>). The total length of the map was 2201.78 cM. After conflating markers that occurred at same position at the linkage map, 686 unique mapping positions were identified, with a mean distance between them of 3.98 cM.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/509/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Linkage map of common pandora Pagellus erythrinus produced from 917 ddRAD loci. Map distances are calculated using the Kosambi function. The black lines indicate the location of the markers on the linkage groups. Numbering follows this in European seabass based on synteny analyses." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1658270713" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Linkage map of common pandora &lt;em&gt;Pagellus erythrinus&lt;/em&gt; produced from 917 ddRAD loci. Map distances are calculated using the Kosambi function. The black lines indicate the location of the markers on the linkage groups. Numbering follows this in European seabass based on synteny analyses.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/509/F1.medium.gif" width="440" height="295"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/3/509/F1.medium.gif" width="440" height="295"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/509/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/509/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467746" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-19" class="first-child">Linkage map of common pandora <em>Pagellus erythrinus</em> produced from 917 ddRAD loci. Map distances are calculated using the Kosambi function. The black lines indicate the location of the markers on the linkage groups. Numbering follows this in European seabass based on synteny analyses.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Comparative genomic analysis</h3><p id="p-20">To validate the constructed linkage map and compare the genomic architecture of common pandora to that of other teleosts, we identified the most similar homologous regions of common pandora RAD loci present in the genomes of medaka, stickleback, Nile tilapia and European seabass (<em>e</em>-value threshold 10<sup>−9</sup>) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023432/-/DC1/TableS2.xls">Table S2</a>). This search revealed numerous presumed homologous loci in each of those four species (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> and <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). The majority of the ‘homologous’ loci identified were positioned on chromosomes and many of them fell within coding sequences (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023432/-/DC1/TableS2.xls">Table S2</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023432/-/DC1/FigureS3.eps">Figure S3</a>). The remaining loci were located in unlinked scaffolds or contigs, and were excluded from downstream analyses. To test whether the difference in the number of homologous loci in the four model species might be a reflection of the genomic coverage, we identified and compared the CEGMA core genes within their genomes. The results showed that Nile tilapia genome contains, at least partially, 98.39% of CEGMA core genes, stickleback genome contains 97.50%, medaka contains 97.98% and European seabass contains 97.98%, suggesting a similar coverage over all these genomes.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/509/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The number of homologous loci for each of common pandora linkage group with European seabass, Nile tilapia, stickleback, and medaka." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1658270713" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The number of homologous loci for each of common pandora linkage group with European seabass, Nile tilapia, stickleback, and medaka.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/509/F2.medium.gif" width="440" height="273"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/3/509/F2.medium.gif" width="440" height="273"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/509/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/509/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467686" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-21" class="first-child">The number of homologous loci for each of common pandora linkage group with European seabass, Nile tilapia, stickleback, and medaka.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467725/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467725/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Summary of the genetic linkage map of common pandora, and comparative analysis with the chromosomes of medaka, stickleback, Nile tilapia, and European seabass</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467772/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467772/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Summary of common pandora linkage map comparative analysis<sup><strong><em><sup>a</sup></em></strong></sup></span><div class="sb-div caption-clear"></div></div></div><p id="p-23">The RAD loci identified as having significant sequence similarities within the chromosomes/linkage groups of medaka, stickleback, Nile tilapia, and European seabass were used in the comparative analysis (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Inferring the genomic location of the homologous loci in all four species revealed extensive conservation of synteny (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023432/-/DC1/FigureS2.eps">Figure S2</a>). The highest degree of similarity was detected with European seabass, with which 43% of common pandora RAD loci were matched, fewer in Nile tilapia (20%) and stickleback (17%), and least in medaka (10%).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/509/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparative view of common pandora linkage groups with European seabass. Common pandora linkage groups are named based on their homologous European seabass chromosome. At the periphery, from inner to outside, the homologous loci of common pandora with Nile tilapia, stickleback, and medaka are shown. Bands show the loci position, and their color represents the different common pandora linkage groups." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1658270713" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparative view of common pandora linkage groups with European seabass. Common pandora linkage groups are named based on their homologous European seabass chromosome. At the periphery, from inner to outside, the homologous loci of common pandora with Nile tilapia, stickleback, and medaka are shown. Bands show the loci position, and their color represents the different common pandora linkage groups.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/509/F3.medium.gif" width="387" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/3/509/F3.medium.gif" width="387" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/509/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/509/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467591" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-24" class="first-child">Comparative view of common pandora linkage groups with European seabass. Common pandora linkage groups are named based on their homologous European seabass chromosome. At the periphery, from inner to outside, the homologous loci of common pandora with Nile tilapia, stickleback, and medaka are shown. Bands show the loci position, and their color represents the different common pandora linkage groups.</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">At the linkage group level, the comparison with medaka (<em>n</em> = 24) revealed one-to-one homology with the great majority of common pandora LGs (peLG) (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Where homology with medaka chromosomes (olChr) was found, up to eight ‘homologous’ loci per peLG were identified. Although all medaka chromosomes had hits with common pandora loci, peLG21 showed no homology with any medaka chromosome. Further, common pandora peLG16 appeared to have partial homology with four medaka chromosomes, making assignment of homology unclear. The remainder of peLGs resolved to a potentially homologous single medaka chromosome (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>), though this can only be stated with limited confidence due to the relatively low number of homologous loci found between the two species.</p><p id="p-26">The comparison with stickleback (<em>n</em> = 21) also revealed one-to-one homology for the great majority of peLGs (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>), with 1–15 RAD loci being associated per peLG. In particular, synteny was detected with all 21 groups of stickleback (gaGroup). In three cases, two common pandora LGs were linked to a stickleback LG, <em>i.e.</em>, peLG13 and peLG24 to gaGroupI, peLG2 and peLG23 to gaGroupIV, and last peLG14 and peLG3 to gaGroupVII. In the latter case, peLG3 had only a single hit to gaGroupVII. Overall, conservation of synteny was more apparent in stickleback compared to medaka.</p><p id="p-27">For Nile tilapia (<em>n</em> = 22), we also identified one-to-one homology for almost all peLGs with Nile tilapia LGs (onLG), sharing 1–19 homologous loci per LG (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>). In particular, 15 out of 24 peLGs were homologous to a single onLG. It is worth noting that both peLG6 and peLG19 showed full homology to onLG7. Further, peLG24 showed no homology to any onLG. Overall, common pandora seemed to share higher homology with Nile tilapia than with stickleback.</p><p id="p-28">Finally, in a comparison with the recently published European seabass genome (<em>n</em> = 24), we observed the greatest homology with common pandora RAD loci. Between five and 39 RAD loci were identified in each European seabass chromosome (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). European seabass had 1-to-1 homology with common pandora for all of its 24 chromosomes, with two European seabass chromosomes (dlLG) containing multiple homologous loci from 2 peLGs (<a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>). First, dlLG1B showed 100% agreement to peLG21 (<em>i.e.</em>, peLG21 had no homology with any other dlLG) and to parts of peLG16. Second, dlLG14 showed 100% consistency to peLG14 and to parts of peLG3. Note that both peLG14 and peLG3 were also assigned to stickleback gaGroupVII. To understand further the syntenic relationships among peLG14, peLG3, gaGroupVII, and dlLG14 and dlLGB1, we plotted their homology pattern in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>. The pattern observed shows that while peLG14 is clearly homologous to dlLG14 and gaGroupVII, peLG3 has a weaker signal. However, it is linked with both dlLG14 and dlLG3 in European seabass, and only with gaGroupVII in stickleback.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/509/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Detailed comparative view of common pandora peLG14 and peLG3 in comparison to stickleback gaGroupVII and European seabass dlLG3 and dlLG14. Links show homology between pairs of loci. Link colors follow the same pattern as in Figure 3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1658270713" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Detailed comparative view of common pandora peLG14 and peLG3 in comparison to stickleback gaGroupVII and European seabass dlLG3 and dlLG14. Links show homology between pairs of loci. Link colors follow the same pattern as in Figure 3.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/509/F4.medium.gif" width="440" height="431"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/3/509/F4.medium.gif" width="440" height="431"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/509/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/509/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467739" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-29" class="first-child">Detailed comparative view of common pandora peLG14 and peLG3 in comparison to stickleback gaGroupVII and European seabass dlLG3 and dlLG14. Links show homology between pairs of loci. Link colors follow the same pattern as in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Phylogenetic reconstruction</h3><p id="p-30">Tree reconstruction took place after aligning the 50 RAD loci that had a hit in all four species used for the comparative analysis. From those loci, 41 were coding while the rest were noncoding. The concatenated alignment consisted of 5015 nucleotide sites, where 3752 were well-aligned and used for the phylogenetic reconstruction employing the model HKY + Γ<sub>4</sub>. The resultant unrooted phylogenetic tree suggested a close relationship of common pandora to stickleback (BS = 90) (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). European seabass clusters with the common pandora/stickleback clade (BS = 100) and Nile tilapia is closer to medaka.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/509/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="RAD-based maximum likelihood phylogenetic tree of common pandora and the species used in the comparative analysis. The tree was built using HKY+ &#x393;4 model on the concatenated alignment of 50 loci shared among all five species. Bootstrap values are shown as branch labels." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1658270713" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;RAD-based maximum likelihood phylogenetic tree of common pandora and the species used in the comparative analysis. The tree was built using HKY+ &#x393;&lt;sub&gt;4&lt;/sub&gt; model on the concatenated alignment of 50 loci shared among all five species. Bootstrap values are shown as branch labels.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/509/F5.medium.gif" width="398" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/3/509/F5.medium.gif" width="398" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/509/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/509/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467841" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-31" class="first-child">RAD-based maximum likelihood phylogenetic tree of common pandora and the species used in the comparative analysis. The tree was built using HKY+ Γ<sub>4</sub> model on the concatenated alignment of 50 loci shared among all five species. Bootstrap values are shown as branch labels.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-32">The employed modified ddRAD methodology allowed the discovery of nearly 1000 polymorphic loci in the unexplored genome of common pandora. Based on 97 progeny of one full-sib family, we built a linkage map of 24 linkage groups, potentially corresponding to the 24 chromosomes of the species, and compared them with those of medaka, stickleback, Nile tilapia, and European seabass, revealing various levels of genome conservation.</p><div id="sec-16" class="subsection"><h3>ddRAD sequencing as a linkage mapping tool</h3><p id="p-33">The ddRAD methodology applied is an adaptation of the first ddRAD protocol (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Peterson <em>et al.</em> 2012</a>), slightly modified from <a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">Palaiokostas <em>et al.</em> (2015)</a>. Rather than processing each sample separately or in small groups with final pooling after amplification, the strategy used was to pool all samples after ligation of barcoded adapters, such that only a single gel size-selection step was required. Library production was thus much less labor-intensive than the original ddRAD protocol, and produced a common set of fragment sizes for all samples. Although read depth per individual varied considerably (possibly due to unevenness in initial DNA quality and quantification), the data produced were more than adequate to infer a robust linkage map for the screened pedigree. A major advantage of linkage mapping by genotyping by sequencing <em>cf</em>. individual SNP or microsatellite assays is the opportunity it provides to use the sequence data generated to explore similarities to other genomes, as shown in previous studies (<em>e.g.</em> <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Amores <em>et al.</em> 2011</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Recknagel <em>et al.</em> 2013</a>; <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">You <em>et al.</em> 2013</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Kakioka <em>et al.</em> 2013</a>).</p></div><div id="sec-17" class="subsection"><h3>Common pandora linkage map</h3><p id="p-34">The constructed linkage map included 24 LGs matching the expected number of chromosomes given that the haploid genome of common pandora—and of all other already karyotyped sparids—is organized in 24 chromosomes (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Cataudella <em>et al.</em> 1980</a>). This prior knowledge guided our LOD score choice for the linkage computations assisting our data analysis design. The methodology applied for the linkage map construction is based on the average between sexes, taking advantage of the informative markers detected for both parents, but without taking into account the differences in recombination between the two.</p><p id="p-35">Given the importance of integrating the knowledge provided by previous QTL mapping efforts, we indirectly identified the candidate LGs of common pandora that are likely to accommodate QTL found in Gilthead seabream based on the linkage map of <a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Tsigenopoulos <em>et al.</em> (2014)</a>. As most studies have used stickleback for comparative genomic analyses, we used our comparative genomic analysis with stickleback (see below) as means to bridge the LGs built in each study with the present map. Based on the findings presented in each study regarding the homology of built Gilthead seabream linkage groups (SpLG) with stickleback genome, we can assume the location of several QTL in the linkage map of common pandora (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023432/-/DC1/TableS3.xls">Table S3</a>). This linkage map is expected to guide future QTL mapping experiments and selected breeding programs.</p></div><div id="sec-18" class="subsection"><h3>Comparing the genome of common pandora with other teleosts</h3><p id="p-36">We analyzed the homology of the generated common pandora linkage map with other teleosts based on presumed homologous RAD loci. The linkage map contained numerous polymorphic markers that exhibited significant sequence similarity with other teleost genomes. Notably, our comparative analysis revealed large syntenic regions within the other four species examined (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023432/-/DC1/FigureS2.eps">Figure S2</a>). The built peLGs tended to have one-to-one homology with chromosomes of European seabass, Nile tilapia, medaka, and stickleback. The extensive synteny and chromosome homology of common pandora with these species suggest a relatively conserved genomic structure among the groups, and provide an independent line of evidence confirming the robustness of the employed linkage mapping approach.</p><p id="p-37">Medaka, a species with 24 chromosomes, showed the least similarity to common pandora, indicative of a more distant evolutionary relationship between the two species. However, most common pandora LGs were matched to a single main homologous chromosome in medaka genome, except peLG21, where no homology was observed. Note that peLG21 has the least homology in all comparisons, possibly due to the low number of included loci. Only peLG16 seems homologous to a combination of different medaka chromosomes. The rest exhibit strong homology with one medaka chromosome, apart from two LGs containing only a single homologous locus in medaka. Overall, although the amount of similarity is limited, common pandora LGs have apparent synteny with medaka chromosomes.</p><p id="p-38">Stickleback (<em>n</em> = 21) has the least number of chromosomes of the examined species and is known for its rapid divergence from other teleosts in terms of chromosome number and morphology (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Urton <em>et al.</em> 2011</a>). The observation in this study that six common pandora LGs exhibited homology to three stickleback LGs in a pair-wise manner is consistent with expectations, reflecting the fusions that have taken place in the ancestral stickleback genome. This has been similarly observed in other synteny studies involving sticklebacks, <em>e.g.</em>, in the comparison of platyfish linkage map to stickleback (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Amores <em>et al.</em> 2014</a>), and in the comparative analysis of the European seabass genome with stickleback (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Tine <em>et al.</em> 2014</a>). Stickleback chromosomes GroupI, GroupIV, and GroupVII are formed by the fusion of ancestral chromosomes that share homology to peLG13-peLG24, peLG2-peLG23, and peLG14-peLG3, respectively. The same three stickleback chromosomes seem to be homologous to the European seabass pairs dlLG13-dlLG24, dlLG2-dlLGX, and dlLG14-dlLG3 in the analysis of <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Tine <em>et al.</em> (2014)</a>. The agreement of our results with <a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Tine <em>et al.</em> (2014)</a> and the homology of common pandora LGs with single chromosomes in all medaka, Nile tilapia, and European seabass confirm the independence of common pandora LGs. Conservation of synteny between sparids and stickleback had been revealed by previous efforts to conduct comparative mapping in gilthead seabream (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Sarropoulou <em>et al.</em> 2007</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Kuhl <em>et al.</em> 2011</a>; <a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Tsigenopoulos <em>et al.</em> 2014</a>).</p><p id="p-39">Comparison to Nile tilapia revealed a high degree of conservation of synteny. Nile tilapia is known to have 22 chromosomes—two less than common pandora. Thus, assuming that the ancestral teleost had 24 chromosomes, one would expect to observe two fusion events in Nile tilapia compared to common pandora. Indeed, peLG6 and peLG19 both link to onLG7, a known fused chromosome in tilapia (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Guyon <em>et al.</em> 2012</a>). Interestingly, the second fusion event could not be detected in our comparative analyses. A denser linkage map is needed to more clearly resolve the syntenic relationship between common pandora and Nile tilapia.</p><p id="p-40">European seabass shared the highest sequence homology with common pandora. The majority of the 24 European seabass linkage groups shared homology with a single common pandora LG. Exceptions were dlLG1B and dlLG14, which showed homology to two peLGs. Particularly for dlLG14, we observe homology with peLG14 and peLG3 (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). Surprisingly, those two LGs correspond to the fused stickleback gaGroupVII. However, peLG3 shares homology also with European seabass dlLG3. Thus, we can hypothesize that there is a translocation of a chromosomal part of seabass from dlLG3 to dlLG14. Note that dlLG3 is one of the smallest chromosomes of European seabass genome. To independently confirm this observation, we tracked down the homology of European seabass dlLG3 and dlLG14 with stickleback in the deep synteny analysis conducted by <a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">Tine <em>et al.</em> (2014)</a>. In this screening, we confirmed the link between dlLG3-dlLG14 and that of two more European seabass LG pairs observed to be linked in stickleback genome in our analysis (dlLG13-dlLG24, dlLG2-dlLGX) as well. Thus, if we assume that Nile tilapia, medaka, and common pandora maintained the ancestral structure of linkage groups peLG3 and peLG14, we can hypothesize a translocation in European seabass toward the direction of the fusion observed in stickleback.</p><p id="p-41">Overall, from our comparative genomic analysis, we found no common pandora LG that shares homology with more than one chromosome in all the compared species, confirming the linkage mapping conducted.</p><p id="p-42">Finally, the percentage of common pandora RAD loci identified as being homologous to coding regions of the reference species was relatively high (more than one-third on average, <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). This possibly reflects the higher chance of finding sequence similarity in the functionally constrained protein coding moiety of the genome compared to noncoding regions, and/or could be due to the ddRAD strategy employed, where restriction enzymes with GC-rich recognition sites were employed in library construction.</p></div><div id="sec-19" class="subsection"><h3>Insights into Sparidae phylogenetic position</h3><p id="p-43">The phylogenetic relationship of Sparidae to other teleost families is still controversial (reviewed in <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hanel and Tsigenopoulos 2011</a>). The similarity search of RAD loci <em>vs.</em> Nile tilapia, stickleback, medaka, and European seabass indicated that common pandora, and probably Sparidae as a whole, was more similar to European seabass than any of the other model species (sharing twice as many RAD loci as the second most similar species—Nile tilapia). This finding does not appear to be unduly biased by differing levels of genome completeness among the four model species, with the CEGMA pipeline analysis confirming similar coverage across all. However, our phylogenetic analysis suggests a closer relationship of Sparids with stickleback rather than European seabass or Nile tilapia with relatively high support (BS = 90). Given this analysis outcome, one would expect a higher sequence similarity of common pandora with stickleback, which was not observed. The fact that common pandora has higher number of similar sequences throughout the genome with European seabass and Nile tilapia compared to stickleback might reflect the longer branch, and thus the higher sequence divergence, observed in the latter (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). Further, the phylogenetic analysis puts forward that European seabass is closer to stickleback <em>cf</em>. Nile tilapia. This is in line with the findings of <a id="xref-ref-49-5" class="xref-bibr" href="#ref-49">Tine <em>et al.</em> (2014)</a> who resolved the phylogenetic relationship of European seabass with model teleost species through a rigorous phylogenomic analysis of 621 genes. In the phylogenetic tree constructed by <a id="xref-ref-49-6" class="xref-bibr" href="#ref-49">Tine <em>et al.</em> (2014)</a> stickleback exhibited a profoundly longer branch compared to European seabass. Finally, medaka and tilapia clustered together agreeing with their known phylogeny (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Betancur <em>et al.</em> 2013</a>). However, the clustering of common pandora with stickleback in the ddRAD markers phylogenetic analysis suggests that the translocation observed in European seabass LG14-LG3 (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>) is independent from the chromosomal fusion that led to stickleback GroupVII, regardless of the involvement of the same linkage groups in common pandora. To answer this question, and unambiguously resolve the position of sparids in the tree of teleosts, a thorough phylogenomic analysis with broad taxon sampling and inclusion of multiple informative outgroups has to be employed.</p></div><div id="sec-20" class="subsection"><h3>Conclusions</h3><p id="p-44">Here, we built the first linkage map for common pandora and present the first application of RAD-Sequencing in the Sparidae family. This linkage map should provide the basis for future marker-assisted selection and QTL mapping of important traits on the species, boosting its aquaculture production through genetic selection programs. Moreover, the extensive similarity of common pandora genome compared to European seabass could be an indication that the prediction of Direct Genomic Value (DGV) in common pandora broodstock might not be as ineffective (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Taylor 2014</a>), when Genomic Estimated Breeding Values (GEBV) are available in a training population of a more commercially exploited aquacultured species (<em>e.g.</em>, European seabass, Gilthead seabream).</p><p id="p-45">On top of the importance of our effort for aquaculture, we use common pandora as a starting point to understand in more depth the genomic evolution of sparids. Comparative genomic analyses revealed an extensive conservation of the genome evolution of common pandora mostly compared to European seabass, Nile tilapia, and to a lesser extent to stickleback and medaka. Interestingly, our phylogenetic analysis suggested that the genomic sequence similarity observed does not reflect phylogenetic proximity as common pandora seems phylogenetically closer to stickleback rather than to European seabass or Nile tilapia.</p><p id="p-46">Finally, the addition of genome-wide information from new nonmodel species from the enormously big tree of teleost fish will shed light upon the evolution of this diverse group of vertebrates. The inclusion of more and more nonmodel species to the genomics arena is the great opportunity in the postgenomic era, leading to a spectacular increase of the available knowledge on teleosts.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-47">We thank Dr. Jon Kristoffersen for assistance with sequencing and fruitful discussions. We are thankful to four anonymous reviewers for providing valuable comments and input that greatly improved the article. Financial support was provided by the Ministry of Education and Religious Affairs, under the call “ARISTEIA I” of the National Strategic Reference Framework 2007–2013 (SPARCOMP, #36), co-funded by the European Union and the Hellenic Republic through the European Social Fund. J.B.T. was supported by the Marine Alliance for Science and Technology for Scotland (MASTS) pooling initiative in the completion of this study. MASTS is funded by the Scottish Funding Council (grant reference HR09011) and contributing institutions. The authors declare that they have no competing interests.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-2"><p id="p-48">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023432/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.023432/-/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-49">Data archiving: Raw reads are deposited in N.C.B.I. sequence read archive under the BioProject ID PRJNA302241.</p></li><li class="fn" id="fn-4"><p id="p-50"><em>Communicating editor: W. S. Davidson</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-10-06"><span class="received-label">Received </span>October 6, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-12-16"><span class="accepted-label">Accepted </span>December 16, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Manousaki <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.1" data-doi="10.1534/genetics.111.127324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferrara</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fontenot</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Postlethwait</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome evolution and meiotic maps by massively parallel DNA sequencing: spotted gar, an outgroup for the teleost genome duplication.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">799</span>–<span class="cit-lpage">808</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAmores%26rft.auinit1%253DA.%26rft.volume%253D188%26rft.issue%253D4%26rft.spage%253D799%26rft.epage%253D808%26rft.atitle%253DGenome%2BEvolution%2Band%2BMeiotic%2BMaps%2Bby%2BMassively%2BParallel%2BDNA%2BSequencing%253A%2BSpotted%2BGar%252C%2Ban%2BOutgroup%2Bfor%2Bthe%2BTeleost%2BGenome%2BDuplication%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.127324%26rft_id%253Dinfo%253Apmid%252F21828280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzQvNzk5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzUwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.2" data-doi="10.1534/genetics.114.164293"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nanda</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warren</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walter</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A RAD-tag genetic map for the platyfish (<em>Xiphophorus maculatus</em>) reveals mechanisms of karyotype evolution among teleost fish.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">625</span>–<span class="cit-lpage">641</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164293%26rft_id%253Dinfo%253Apmid%252F24700104%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk3LzIvNjI1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzUwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.3" data-doi="10.1006/anbe.2000.1652"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Animal Behaviour Society</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Guidelines for the treatment of animals in behavioural research and teaching.</span> <abbr class="cit-jnl-abbrev">Anim. Behav.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">271</span>–<span class="cit-lpage">275</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2BBehaviour%26rft.stitle%253DAnimal%2BBehaviour%26rft.volume%253D61%26rft.issue%253D1%26rft.spage%253D271%26rft.epage%253D275%26rft.atitle%253DGuidelines%2Bfor%2Bthe%2Btreatment%2Bof%2Banimals%2Bin%2Bbehavioural%2Bresearch%2Band%2Bteaching.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fanbe.2000.1652%26rft_id%253Dinfo%253Apmid%252F11170716%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/anbe.2000.1652&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11170716&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167452600026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Apostolidis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moutou</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stamatis</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mamuris</span>  <span class="cit-name-given-names">Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genetic structure of three marine fishes from the Gulf of Pagasitikos (Greece) based on allozymes, RAPD, and mtDNA RFLP markers.</span> <abbr class="cit-jnl-abbrev">Biologia</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">1005</span>–<span class="cit-lpage">1010</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiologia%26rft.volume%253D64%26rft.spage%253D1005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.5" data-doi="10.1371/journal.pone.0003376"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baird</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etter</span>  <span class="cit-name-given-names">P. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atwood</span>  <span class="cit-name-given-names">T. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currey</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shiver</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">e3376</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBaird%26rft.auinit1%253DN.%2BA.%26rft.volume%253D3%26rft.issue%253D10%26rft.spage%253De3376%26rft.epage%253De3376%26rft.atitle%253DRapid%2BSNP%2Bdiscovery%2Band%2Bgenetic%2Bmapping%2Busing%2Bsequenced%2BRAD%2Bmarkers.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0003376%26rft_id%253Dinfo%253Apmid%252F18852878%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0003376&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18852878&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-book" id="cit-6.3.509.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Basurco</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lovatelli</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Current status of Sparidae aquaculture,</span> pp. <span class="cit-fpage">1</span>–<span class="cit-lpage">50</span> in <span class="cit-source"><em>Sparidae: Biology and Aquaculture of Gilthead Sea Bream and Other Species</em>,</span> edited by <span class="cit-ed"><span class="cit-name-surname">Pavlidis</span>  <span class="cit-name-given-names">M.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Mylonas</span>  <span class="cit-name-given-names">C.C.</span></span>. <span class="cit-publ-name">Wiley-Blackwell</span>, <span class="cit-publ-loc">Oxford</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Betancur</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Broughton</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiley</span>  <span class="cit-name-given-names">E. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carpenter</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lopez</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The tree of life and a new classification of bony fishes.</span> <abbr class="cit-jnl-abbrev">PLoS Curr.</abbr> <span class="cit-vol">5</span>: pii: ecurrents.tol.53ba26640df0ccaee75bb165c8c26288.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-thesis" id="cit-6.3.509.8"><div class="cit-metadata unstructured">Blanquer, A., 1990 Phylogeographie intraspecifique d’un poisson marin, le flet <em>Platichthys flesus L</em>. (Heterosomata). Polymorphisme des marqueurs nucleaires et mitochondriaux. Ph.D. Thesis, University of Montpellier.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Braasch</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desvignes</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCluskey</span>  <span class="cit-name-given-names">B. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batzel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A new model army: Emerging fish models to study the genomics of vertebrate Evo-Devo. J. Exp. Zool. B Mol.</span> <abbr class="cit-jnl-abbrev">Dev. Evol.</abbr> <span class="cit-vol">324</span>: <span class="cit-fpage">316</span>–<span class="cit-lpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDev.%2BEvol.%26rft.volume%253D324%26rft.spage%253D316%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.10" data-doi="10.1016/j.aquaculture.2011.06.044"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boulton</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Massault</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Houston</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Koning</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">QTL affecting morphometric traits and stress response in the gilthead seabream (<em>Sparus aurata</em>).</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">319</span>: <span class="cit-fpage">58</span>–<span class="cit-lpage">66</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D319%26rft.spage%253D58%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.aquaculture.2011.06.044%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.aquaculture.2011.06.044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.11" data-doi="10.1111/1755-0998.12357"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harmon</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Narum</span>  <span class="cit-name-given-names">S. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genotyping-in-Thousands by sequencing (GT-seq): a cost effective SNP genotyping method based on custom amplicon sequencing.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">855</span>–<span class="cit-lpage">867</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D15%26rft.spage%253D855%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12357%26rft_id%253Dinfo%253Apmid%252F25476721%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12357&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25476721&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.12" data-doi="10.1093/oxfordjournals.molbev.a026334"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Castresana</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">540</span>–<span class="cit-lpage">552</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DCastresana%26rft.auinit1%253DJ.%26rft.volume%253D17%26rft.issue%253D4%26rft.spage%253D540%26rft.epage%253D552%26rft.atitle%253DSelection%2Bof%2BConserved%2BBlocks%2Bfrom%2BMultiple%2BAlignments%2Bfor%2BTheir%2BUse%2Bin%2BPhylogenetic%2BAnalysis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a026334%26rft_id%253Dinfo%253Apmid%252F10742046%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjE3LzQvNTQwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzUwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.13" data-doi="10.1007/BF00055985"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cataudella</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riz</span>  <span class="cit-name-given-names">P. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sola</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">A chromosome study of eight Mediterranean species of Sparidae (Pisces, Perciformes).</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">155</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D54%26rft.spage%253D155%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00055985%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00055985&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.14" data-doi="10.1534/g3.111.000240"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Catchen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amores</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hohenlohe</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cresko</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Postlethwait</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Stacks: building and genotyping loci de novo from short-read sequences.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">171</span>–<span class="cit-lpage">182</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DCatchen%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D1%26rft.issue%253D3%26rft.spage%253D171%26rft.epage%253D182%26rft.atitle%253DStacks%253A%2BBuilding%2Band%2BGenotyping%2BLoci%2BDe%2BNovo%2BFrom%2BShort-Read%2BSequences%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.111.000240%26rft_id%253Dinfo%253Apmid%252F22384329%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjEvMy8xNzEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNTA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.15" data-doi="10.1038/nmeth.2109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Darriba</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taboada</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doallo</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Posada</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">jModelTest 2: more models, new heuristics and parallel computing.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">772</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DPatel%26rft.auinit1%253DV.%2BR.%26rft.volume%253D9%26rft.issue%253D8%26rft.spage%253D772%26rft.epage%253D772%26rft.atitle%253DjModelTest%2B2%253A%2Bmore%2Bmodels%252C%2Bnew%2Bheuristics%2Band%2Bparallel%2Bcomputing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2109%26rft_id%253Dinfo%253Apmid%252F22847109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22847109&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.16" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kawamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e19379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.17" data-doi="10.1093/icb/ict077"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Erisman</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petersen</span>  <span class="cit-name-given-names">C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hastings</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Warner</span>  <span class="cit-name-given-names">R. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Phylogenetic perspectives on the evolution of functional hermaphroditism in teleost fishes.</span> <abbr class="cit-jnl-abbrev">Integr. Comp. Biol.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">736</span>–<span class="cit-lpage">754</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIntegr.%2BComp.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Ficb%252Fict077%26rft_id%253Dinfo%253Apmid%252F23817661%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiaWNiaW9sIjtzOjU6InJlc2lkIjtzOjg6IjUzLzQvNzM2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzUwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.18" data-doi="10.1007/s10228-008-0094-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fassatoui</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mdelgi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romdhane</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A preliminary investigation of allozyme genetic variation and population structure in common pandora (<em>Pagellus erythrinus</em>, Sparidae) from Tunisian and Libyan coasts.</span> <abbr class="cit-jnl-abbrev">Ichthyol. Res.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">301</span>–<span class="cit-lpage">307</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIchthyol.%2BRes.%26rft.volume%253D56%26rft.spage%253D301%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10228-008-0094-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10228-008-0094-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.19" data-doi="10.3354/meps09531"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fassatoui</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chenuil</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Romdhane</span>  <span class="cit-name-given-names">M. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Relationships between heterozygosity, growth parameters and age in the common pandora <em>Pagellus erythrinus</em> (Sparidae) in the Gabes Gulf (Tunisia).</span> <abbr class="cit-jnl-abbrev">Mar. Ecol. Prog. Ser.</abbr> <span class="cit-vol">445</span>: <span class="cit-fpage">251</span>–<span class="cit-lpage">261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMar.%2BEcol.%2BProg.%2BSer.%26rft.volume%253D445%26rft.spage%253D251%26rft_id%253Dinfo%253Adoi%252F10.3354%252Fmeps09531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3354/meps09531&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.20" data-doi="10.1534/genetics.106.059014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Franch</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louro</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsalavouta</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chatziplis</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsigenopoulos</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A genetic linkage map of the hermaphrodite teleost fish <em>Sparus aurata L</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">851</span>–<span class="cit-lpage">861</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.059014%26rft_id%253Dinfo%253Apmid%252F16951080%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc0LzIvODUxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzUwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.21" data-doi="10.1023/A:1003977418900"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galetti</span>  <span class="cit-name-given-names">P. M</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>  <span class="cit-name-given-names">C. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Molina</span>  <span class="cit-name-given-names">W. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">An overview of marine fish cytogenetics.</span> <abbr class="cit-jnl-abbrev">Hydrobiologia</abbr> <span class="cit-vol">420</span>: <span class="cit-fpage">55</span>–<span class="cit-lpage">62</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHydrobiologia%26rft.volume%253D420%26rft.spage%253D55%26rft_id%253Dinfo%253Adoi%252F10.1023%252FA%253A1003977418900%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/A:1003977418900&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.22" data-doi="10.1080/10635150390235520"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guindon</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gascuel</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood.</span> <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">696</span>–<span class="cit-lpage">704</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150390235520%26rft_id%253Dinfo%253Apmid%252F14530136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoic3lzYmlvIjtzOjU6InJlc2lkIjtzOjg6IjUyLzUvNjk2IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzUwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.23" data-doi="10.1186/1471-2164-13-222"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guyon</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rakotomanga</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Azzouzi</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coutanceau</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonillo</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A high-resolution map of the Nile tilapia genome: a resource for studying cichlids and other percomorphs.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DGuyon%26rft.auinit1%253DR.%26rft.volume%253D13%26rft.spage%253D222%26rft.epage%253D222%26rft.atitle%253DA%2Bhigh-resolution%2Bmap%2Bof%2Bthe%2BNile%2Btilapia%2Bgenome%253A%2Ba%2Bresource%2Bfor%2Bstudying%2Bcichlids%2Band%2Bother%2Bpercomorphs.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-222%26rft_id%253Dinfo%253Apmid%252F22672252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-222&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22672252&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-book" id="cit-6.3.509.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hanel</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsigenopoulos</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Phylogeny, evolution and taxonomy of sparids with some notes on their ecology and biology,</span> pp. <span class="cit-fpage">51</span>–<span class="cit-lpage">73</span> in <span class="cit-source"><em>Sparidae: Biology and Aquaculture of Gilthead Sea Bream and Other Species</em>,</span> edited by <span class="cit-ed"><span class="cit-name-surname">Pavlidis</span>  <span class="cit-name-given-names">M.</span></span>, and <span class="cit-ed"><span class="cit-name-surname">Mylonas</span>  <span class="cit-name-given-names">C.C.</span></span>. <span class="cit-publ-name">Wiley-Blackwell</span>, <span class="cit-publ-loc">Oxford</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.25" data-doi="10.1186/1471-2164-15-233"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kai</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nomura</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fujiwara</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakamura</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yasuike</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A ddRAD-based genetic map and its integration with the genome assembly of Japanese eel (<em>Anguilla japonica</em>) provides insights into genome evolution after the teleost-specific genome duplication.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">233</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D233%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-233%26rft_id%253Dinfo%253Apmid%252F24669946%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-233&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24669946&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.26" data-doi="10.1186/1471-2164-14-32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kakioka</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kokita</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumada</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Okuda</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A RAD-based linkage map and comparative genomics in the gudgeons (genus <em>Gnathopogon</em>, Cyprinidae).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D32%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-32%26rft_id%253Dinfo%253Apmid%252F23324215%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-32&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23324215&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.27" data-doi="10.1093/molbev/mst010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katoh</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Standley</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">MAFFT Multiple sequence alignment software version 7: Improvements in performance and usability.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">772</span>–<span class="cit-lpage">780</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmst010%26rft_id%253Dinfo%253Apmid%252F23329690%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjMwLzQvNzcyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzUwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.28" data-doi="10.1101/gr.092759.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krzywinski</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schein</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birol</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Connors</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gascoyne</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Circos: an information aesthetic for comparative genomics.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1639</span>–<span class="cit-lpage">1645</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.092759.109%26rft_id%253Dinfo%253Apmid%252F19541911%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzkvMTYzOSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMy81MDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kuhl</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarropoulou</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tine</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kotoulas</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magoulas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A Comparative BAC map for the gilthead sea bream (<em>Sparus aurata L</em>.).</span> <abbr class="cit-jnl-abbrev">J. Biomed. Biotechnol.</abbr> <span class="cit-vol">2011</span>: <span class="cit-fpage">329025</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BBiomed%2BBiotechnol%26rft.aulast%253DKuhl%26rft.auinit1%253DH.%26rft.volume%253D2011%26rft.spage%253D329025%26rft.epage%253D329025%26rft.atitle%253DA%2BComparative%2BBAC%2Bmap%2Bfor%2Bthe%2Bgilthead%2Bsea%2Bbream%2B%2528Sparus%2Baurata%2BL.%2529.%26rft_id%253Dinfo%253Apmid%252F21049003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=21049003&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.30" data-doi="10.1371/journal.pone.0016599"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loukovitis</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarropoulou</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsigenopoulos</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batargias</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magoulas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Quantitative trait loci involved in sex determination and body growth in the Gilthead sea bream (<em>Sparus aurata L</em>.) through targeted genome scan.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e16599</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DLoukovitis%26rft.auinit1%253DD.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253De16599%26rft.epage%253De16599%26rft.atitle%253DQuantitative%2Btrait%2Bloci%2Binvolved%2Bin%2Bsex%2Bdetermination%2Band%2Bbody%2Bgrowth%2Bin%2Bthe%2Bgilthead%2Bsea%2Bbream%2B%2528Sparus%2Baurata%2BL.%2529%2Bthrough%2Btargeted%2Bgenome%2Bscan.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0016599%26rft_id%253Dinfo%253Apmid%252F21304996%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0016599&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21304996&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.31" data-doi="10.1111/j.1365-2052.2012.02346.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loukovitis</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarropoulou</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batargias</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Apostolidis</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kotoulas</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Quantitative trait loci for body growth and sex determination in the hermaphrodite teleost fish <em>Sparus aurata L</em>.</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">753</span>–<span class="cit-lpage">759</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnimal%2Bgenetics%26rft.stitle%253DAnim%2BGenet%26rft.aulast%253DLoukovitis%26rft.auinit1%253DD.%26rft.volume%253D43%26rft.issue%253D6%26rft.spage%253D753%26rft.epage%253D759%26rft.atitle%253DQuantitative%2Btrait%2Bloci%2Bfor%2Bbody%2Bgrowth%2Band%2Bsex%2Bdetermination%2Bin%2Bthe%2Bhermaphrodite%2Bteleost%2Bfish%2BSparus%2Baurata%2BL.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2052.2012.02346.x%26rft_id%253Dinfo%253Apmid%252F22497460%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2052.2012.02346.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22497460&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.32" data-doi="10.1111/age.12027"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loukovitis</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batargias</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarropoulou</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Apostolidis</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kotoulas</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Quantitative trait loci affecting morphology traits in gilthead seabream (<em>Sparus aurata L</em>.).</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">480</span>–<span class="cit-lpage">483</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BGenet.%26rft.volume%253D44%26rft.spage%253D480%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fage.12027%26rft_id%253Dinfo%253Apmid%252F23418819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/age.12027&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23418819&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.33" data-doi="10.1186/1471-2164-15-655"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Manousaki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsakogiannis</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lagnel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarropoulou</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xiang</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The sex-specific transcriptome of the hermaphrodite sparid sharpsnout seabream (<em>Diplodus puntazzo</em>).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">655</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D655%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-655%26rft_id%253Dinfo%253Apmid%252F25099474%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-655&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25099474&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.34" data-doi="10.1111/j.2007.0018-0661.02000.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Margarido</span>  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Souza</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">OneMap: software for genetic mapping in outcrossing species.</span> <abbr class="cit-jnl-abbrev">Hereditas</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">78</span>–<span class="cit-lpage">79</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHereditas%26rft.stitle%253DHereditas%26rft.aulast%253DMargarido%26rft.auinit1%253DG.%2BR.%26rft.volume%253D144%26rft.issue%253D3%26rft.spage%253D78%26rft.epage%253D79%26rft.atitle%253DOneMap%253A%2Bsoftware%2Bfor%2Bgenetic%2Bmapping%2Bin%2Boutcrossing%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.2007.0018-0661.02000.x%26rft_id%253Dinfo%253Apmid%252F17663699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.2007.0018-0661.02000.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17663699&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248257800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.35" data-doi="10.1111/j.1365-2052.2010.02110.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Massault</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franch</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haley</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Koning</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bovenhuis</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Quantitative trait loci for resistance to fish pasteurellosis in gilthead sea bream (<em>Sparus aurata</em>).</span> <abbr class="cit-jnl-abbrev">Anim. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">191</span>–<span class="cit-lpage">203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnim.%2BGenet.%26rft.volume%253D42%26rft.spage%253D191%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2052.2010.02110.x%26rft_id%253Dinfo%253Apmid%252F20946317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2052.2010.02110.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20946317&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-book" id="cit-6.3.509.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mylonas</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zohar</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pankhurst</span>  <span class="cit-name-given-names">N. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kagawa</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Reproduction and broodstock management</span>, pp. <span class="cit-fpage">95</span>–<span class="cit-lpage">131</span> in <span class="cit-source">Sparidae: Biology and Aquaculture of Gilthead Sea Bream and Related Species</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Pavlidis</span>  <span class="cit-name-given-names">M.</span></span>, <span class="cit-ed"><span class="cit-name-surname">Mylonas</span>  <span class="cit-name-given-names">C. C.</span></span>. <span class="cit-publ-name">Wiley-Blackwell</span>, <span class="cit-publ-loc">Oxford</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.37" data-doi="10.1101/gr.2004004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Naruse</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanaka</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mita</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shima</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Postlethwait</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A medaka gene map: the trace of ancestral vertebrate proto-chromosomes revealed by comparative gene mapping.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">820</span>–<span class="cit-lpage">828</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.2004004%26rft_id%253Dinfo%253Apmid%252F15078856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE0LzUvODIwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzUwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.38" data-doi="10.1371/journal.pone.0057281"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">O’Quin</span>  <span class="cit-name-given-names">K. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoshizawa</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doshi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeffery</span>  <span class="cit-name-given-names">W. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Quantitative genetic analysis of retinal degeneration in the blind cavefish <em>Astyanax mexicanus</em>.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e57281</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De57281%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0057281%26rft_id%253Dinfo%253Apmid%252F23437360%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0057281&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23437360&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.39" data-doi="10.1186/1471-2164-14-566"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palaiokostas</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bekaert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davie</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cowan</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oral</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">a</span> <span class="cit-article-title">Mapping the sex determination locus in the Atlantic halibut (<em>Hippoglossus hippoglossus</em>) using RAD sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">566</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D566%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-566%26rft_id%253Dinfo%253Apmid%252F23957753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-566&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23957753&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.40" data-doi="10.1371/journal.pone.0068389"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palaiokostas</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bekaert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khan</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taggart</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span><span class="cit-comment">b</span> <span class="cit-article-title">Mapping and validation of the major sex-determining region in Nile tilapia (<em>Oreochromis niloticus L</em>.) using RAD Sequencing.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e68389</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De68389%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0068389%26rft_id%253Dinfo%253Apmid%252F23874606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0068389&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23874606&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.41" data-doi="10.1186/s12864-015-1383-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Palaiokostas</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bekaert</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khan</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taggart</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gharbi</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A novel sex-determining QTL in Nile tilapia (<em>Oreochromis niloticus</em>).</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">171</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D171%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1383-x%26rft_id%253Dinfo%253Apmid%252F25888226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1383-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25888226&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.42" data-doi="10.1093/bioinformatics/btm071"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Parra</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradnam</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korf</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">CEGMA: a pipeline to accurately annotate core genes in eukaryotic genomes.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1061</span>–<span class="cit-lpage">1067</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm071%26rft_id%253Dinfo%253Apmid%252F17332020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyMy85LzEwNjEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNTA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.43" data-doi="10.1371/journal.pone.0037135"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kay</span>  <span class="cit-name-given-names">E. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fisher</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Double digest RADseq: an inexpensive method for de novo SNP discovery and genotyping in model and non-model species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e37135</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPeterson%26rft.auinit1%253DB.%2BK.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253De37135%26rft.epage%253De37135%26rft.atitle%253DDouble%2Bdigest%2BRADseq%253A%2Ban%2Binexpensive%2Bmethod%2Bfor%2Bde%2Bnovo%2BSNP%2Bdiscovery%2Band%2Bgenotyping%2Bin%2Bmodel%2Band%2Bnon-model%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0037135%26rft_id%253Dinfo%253Apmid%252F22675423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0037135&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22675423&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.44" data-doi="10.1534/g3.112.003897"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Recknagel</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elmer</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A hybrid genetic linkage map of two ecologically and morphologically divergent Midas cichlid fishes (<em>Amphilophus spp</em>.) obtained by massively parallel DNA sequencing (ddRADSeq).</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2526%252358%253B%2BGenes%257CGenomes%257CGenetics%26rft.stitle%253DG3%26rft.aulast%253DRecknagel%26rft.auinit1%253DH.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253D65%26rft.epage%253D74%26rft.atitle%253DA%2BHybrid%2BGenetic%2BLinkage%2BMap%2Bof%2BTwo%2BEcologically%2Band%2BMorphologically%2BDivergent%2BMidas%2BCichlid%2BFishes%2B%2528Amphilophus%2Bspp.%2529%2BObtained%2Bby%2BMassively%2BParallel%2BDNA%2BSequencing%2B%2528ddRADSeq%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.112.003897%26rft_id%253Dinfo%253Apmid%252F23316439%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjY6IjMvMS82NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMy81MDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.45" data-doi="10.1186/1471-2164-8-44"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sarropoulou</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franch</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louro</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Power</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bargelloni</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A gene-based radiation hybrid map of the gilthead sea bream <em>Sparus aurata</em> refines and exploits conserved synteny with <em>Tetraodon nigroviridis</em>.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">44</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DSarropoulou%26rft.auinit1%253DE.%26rft.volume%253D8%26rft.spage%253D44%26rft.epage%253D44%26rft.atitle%253DA%2Bgene-based%2Bradiation%2Bhybrid%2Bmap%2Bof%2Bthe%2Bgilthead%2Bsea%2Bbream%2BSparus%2Baurata%2Brefines%2Band%2Bexploits%2Bconserved%2Bsynteny%2Bwith%2BTetraodon%2Bnigroviridis.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-8-44%26rft_id%253Dinfo%253Apmid%252F17286862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-8-44&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17286862&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.46" data-doi="10.1016/j.ygeno.2005.11.019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Senger</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Priat</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hitte</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarropoulou</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Franch</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The first radiation hybrid map of a perch-like fish: the gilthead seabream (<em>Sparus aurata L</em>).</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">793</span>–<span class="cit-lpage">800</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DSenger%26rft.auinit1%253DF.%26rft.volume%253D87%26rft.issue%253D6%26rft.spage%253D793%26rft.epage%253D800%26rft.atitle%253DThe%2Bfirst%2Bradiation%2Bhybrid%2Bmap%2Bof%2Ba%2Bperch-like%2Bfish%253A%2Bthe%2Bgilthead%2Bseabream%2B%2528Sparus%2Baurata%2BL%2529.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygeno.2005.11.019%26rft_id%253Dinfo%253Apmid%252F16413167%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ygeno.2005.11.019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16413167&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238168400012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.47" data-doi="10.1371/journal.pone.0058700"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">SLAF-seq: an efficient method of large-scale de novo SNP discovery and genotyping using high-throughput sequencing.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e58700</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De58700%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0058700%26rft_id%253Dinfo%253Apmid%252F23527008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0058700&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23527008&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Implementation and accuracy of genomic selection.</span> <abbr class="cit-jnl-abbrev">Aquaculture</abbr> <span class="cit-vol">420–421</span>: <span class="cit-fpage">S8</span>–<span class="cit-lpage">S14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAquaculture%26rft.volume%253D420421%26rft.spage%253DS8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.49" data-doi="10.1038/ncomms6770"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tine</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhl</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gagnaire</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louro</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Desmarais</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">European sea bass genome and its variation provide insights into adaptation to euryhalinity and speciation.</span> <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">5770</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D5770%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms6770%26rft_id%253Dinfo%253Apmid%252F25534655%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms6770&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25534655&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.50" data-doi="10.7717/peerj.203"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Toonen</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Puritz</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forsman</span>  <span class="cit-name-given-names">Z. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitney</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernandez-Silva</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">ezRAD: a simplified method for genomic genotyping in non-model organisms.</span> <abbr class="cit-jnl-abbrev">PeerJ</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">e203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPeerJ%26rft.volume%253D1%26rft.spage%253De203%26rft_id%253Dinfo%253Adoi%252F10.7717%252Fpeerj.203%26rft_id%253Dinfo%253Apmid%252F24282669%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.7717/peerj.203&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24282669&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.51" data-doi="10.1016/j.margen.2014.09.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsigenopoulos</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louro</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chatziplis</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lagnel</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vogiatzi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Second generation genetic linkage map for the gilthead sea bream <em>Sparus aurata L.</em></span> <abbr class="cit-jnl-abbrev">Mar. Genomics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMar.%2BGenomics%26rft.volume%253D18%26rft.spage%253D77%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.margen.2014.09.008%26rft_id%253Dinfo%253Apmid%252F25307204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.margen.2014.09.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25307204&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.52" data-doi="10.1159/000331232"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Urton</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCann</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peichel</span>  <span class="cit-name-given-names">C. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Karyotype differentiation between two stickleback species (Gasterosteidae).</span> <abbr class="cit-jnl-abbrev">Cytogenet. Genome Res.</abbr> <span class="cit-vol">135</span>: <span class="cit-fpage">150</span>–<span class="cit-lpage">159</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenetic%2Band%2Bgenome%2Bresearch%26rft.stitle%253DCytogenet%2BGenome%2BRes%26rft.aulast%253DUrton%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D135%26rft.issue%253D2%26rft.spage%253D150%26rft.epage%253D159%26rft.atitle%253DKaryotype%2Bdifferentiation%2Bbetween%2Btwo%2Bstickleback%2Bspecies%2B%2528Gasterosteidae%2529.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000331232%26rft_id%253Dinfo%253Apmid%252F21921583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000331232&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21921583&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.53" data-doi="10.1093/jhered/93.1.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">MapChart: software for the graphical presentation of linkage maps and QTLs.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DVoorrips%26rft.auinit1%253DR.%2BE.%26rft.volume%253D93%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D78%26rft.atitle%253DMapChart%253A%2BSoftware%2Bfor%2Bthe%2BGraphical%2BPresentation%2Bof%2BLinkage%2BMaps%2Band%2BQTLs%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F93.1.77%26rft_id%253Dinfo%253Apmid%252F12011185%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjc6IjkzLzEvNzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNTA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.54" data-doi="10.1038/nmeth.2023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meyer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McKay</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Matz</span>  <span class="cit-name-given-names">M. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">2b-RAD: a simple and flexible method for genome-wide genotyping.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">808</span>–<span class="cit-lpage">810</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DWang%26rft.auinit1%253DS.%26rft.volume%253D9%26rft.issue%253D8%26rft.spage%253D808%26rft.epage%253D810%26rft.atitle%253D2b-RAD%253A%2Ba%2Bsimple%2Band%2Bflexible%2Bmethod%2Bfor%2Bgenome-wide%2Bgenotyping.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2023%26rft_id%253Dinfo%253Apmid%252F22609625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22609625&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000307015700021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.55" data-doi="10.1006/tpbi.2002.1577"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">C. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Painter</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">Z. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Simultaneous maximum likelihood estimation of linkage and linkage phases in outcrossing species.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">363</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DWu%26rft.auinit1%253DR.%26rft.volume%253D61%26rft.issue%253D3%26rft.spage%253D349%26rft.epage%253D363%26rft.atitle%253DSimultaneous%2Bmaximum%2Blikelihood%2Bestimation%2Bof%2Blinkage%2Band%2Blinkage%2Bphases%2Bin%2Boutcrossing%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Ftpbi.2002.1577%26rft_id%253Dinfo%253Apmid%252F12027621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/tpbi.2002.1577&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12027621&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176540100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.509.56"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">You</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Construction of high-density genetic linkage maps for orange-spotted grouper <em>Epinephelus coioides</em> using multiplexed shotgun genotyping.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">113</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D14%26rft.spage%253D113%26rft_id%253Dinfo%253Apmid%252F24289265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24289265&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F509.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/3/509.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/3/499" title="Partial Dominance, Overdominance, Epistasis and QTL by Environment Interactions Contribute to Heterosis in Two Upland Cotton Hybrids" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/3/521" title="Whole-Genome Sequencing of a Canine Family Trio Reveals a FAM83G Variant Associated with Hereditary Footpad Hyperkeratosis" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/3"><span>Volume 6 Issue 3, March 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/3.cover-source.jpg?itok=8EDC4x5t" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/3/509" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-jeVRkp3xVYxGEfuMFp0FgWsFY0bxCxEqODaG8TT_4GM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/3/509" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/3/509" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/3/509" class="active" data-icon-position="" data-hide-link-title="0">Exploring a Nonmodel Teleost Genome Through RAD Sequencing—Linkage Mapping in Common Pandora, Pagellus erythrinus and Comparative Genomic Analysis</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460082" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-MoMQ__K0Fj29RN8NxLBd_Eb2snpKQ3CzYm0_nBxNjOw" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/3/509.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/3/509" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Rz-KyXogStSoqksFHM7oO4xMFHm0tAuj4opH5lIWw0Y" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/3/509" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/3/509.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460082" id="node460082--2" data-pisa="ggg;6/3/509" data-pisa-master="ggg;g3.115.023432" data-apath="/ggg/6/3/509.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Exploring a Nonmodel Teleost Genome Through RAD Sequencing—Linkage Mapping in Common Pandora, <em>Pagellus erythrinus</em> and Comparative Genomic Analysis</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Tereza Manousaki</span>, <span class="highwire-citation-author" data-delta="1">Alexandros Tsakogiannis</span>, <span class="highwire-citation-author" data-delta="2">John B. Taggart</span>, <span class="highwire-citation-author" data-delta="3">Christos Palaiokostas</span>, <span class="highwire-citation-author" data-delta="4">Dimitris Tsaparis</span>, <span class="highwire-citation-author" data-delta="5">Jacques Lagnel</span>, <span class="highwire-citation-author" data-delta="6">Dimitrios Chatziplis</span>, <span class="highwire-citation-author" data-delta="7">Antonios Magoulas</span>, <span class="highwire-citation-author" data-delta="8">Nikos  Papandroulakis</span>, <span class="highwire-citation-author" data-delta="9">Constantinos C. Mylonas</span> and <span class="highwire-citation-author" data-delta="10">Costas S. Tsigenopoulos</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">509-519;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.023432 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460082--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Tereza Manousaki </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tereza%2BManousaki%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Manousaki%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATereza%2BManousaki%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alexandros Tsakogiannis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, University of Crete, Heraklion, 70013, Greece</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexandros%2BTsakogiannis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tsakogiannis%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexandros%2BTsakogiannis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">John B. Taggart </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Aquaculture, School of Natural Sciences, University of Stirling, Scotland, FK9 4LA, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BB.%2BTaggart%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Taggart%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BB.%2BTaggart%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Christos Palaiokostas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Aquaculture, School of Natural Sciences, University of Stirling, Scotland, FK9 4LA, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christos%2BPalaiokostas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palaiokostas%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristos%2BPalaiokostas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Dimitris Tsaparis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dimitris%2BTsaparis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tsaparis%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADimitris%2BTsaparis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jacques Lagnel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacques%2BLagnel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lagnel%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJacques%2BLagnel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Dimitrios Chatziplis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Laboratory of Agrobiotechnology and Inspection of Agricultural Products, Department of Agricultural Technology, School of Agricultural Technology Food Technology and Nutrition, Alexander Technological Educational Institute of Thessaloniki, Sindos, 57400, Greece</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dimitrios%2BChatziplis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chatziplis%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADimitrios%2BChatziplis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Antonios Magoulas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonios%2BMagoulas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Magoulas%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAntonios%2BMagoulas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Nikos  Papandroulakis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nikos%2B%2BPapandroulakis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Papandroulakis%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANikos%2B%2BPapandroulakis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Constantinos C. Mylonas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Constantinos%2BC.%2BMylonas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mylonas%20CC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AConstantinos%2BC.%2BMylonas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Costas S. Tsigenopoulos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Costas%2BS.%2BTsigenopoulos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tsigenopoulos%20CS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACostas%2BS.%2BTsigenopoulos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:tsigeno@hcmr.gr" class="" data-icon-position="" data-hide-link-title="0">tsigeno@hcmr.gr</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/3/509" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/3/509&amp;title=Exploring%20a%20Nonmodel%20Teleost%20Genome%20Through%20RAD%20Sequencing%E2%80%94Linkage%20Mapping%20in%20Common%20Pandora%2C%20Pagellus%20erythrinus%20and%20Comparative%20Genomic%20Analysis" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/3/509&amp;title=Exploring%20a%20Nonmodel%20Teleost%20Genome%20Through%20RAD%20Sequencing%E2%80%94Linkage%20Mapping%20in%20Common%20Pandora%2C%20Pagellus%20erythrinus%20and%20Comparative%20Genomic%20Analysis" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/3/509&amp;title=Exploring%20a%20Nonmodel%20Teleost%20Genome%20Through%20RAD%20Sequencing%E2%80%94Linkage%20Mapping%20in%20Common%20Pandora%2C%20Pagellus%20erythrinus%20and%20Comparative%20Genomic%20Analysis" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/3/509&amp;text=Exploring%20a%20Nonmodel%20Teleost%20Genome%20Through%20RAD%20Sequencing%E2%80%94Linkage%20Mapping%20in%20Common%20Pandora%2C%20Pagellus%20erythrinus%20and%20Comparative%20Genomic%20Analysis" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/3/509&amp;title=Exploring%20a%20Nonmodel%20Teleost%20Genome%20Through%20RAD%20Sequencing%E2%80%94Linkage%20Mapping%20in%20Common%20Pandora%2C%20Pagellus%20erythrinus%20and%20Comparative%20Genomic%20Analysis" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/3/509&amp;t=Exploring%20a%20Nonmodel%20Teleost%20Genome%20Through%20RAD%20Sequencing%E2%80%94Linkage%20Mapping%20in%20Common%20Pandora%2C%20Pagellus%20erythrinus%20and%20Comparative%20Genomic%20Analysis" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/3/509&amp;title=Exploring%20a%20Nonmodel%20Teleost%20Genome%20Through%20RAD%20Sequencing%E2%80%94Linkage%20Mapping%20in%20Common%20Pandora%2C%20Pagellus%20erythrinus%20and%20Comparative%20Genomic%20Analysis" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/3/509&amp;title=Exploring%20a%20Nonmodel%20Teleost%20Genome%20Through%20RAD%20Sequencing%E2%80%94Linkage%20Mapping%20in%20Common%20Pandora%2C%20Pagellus%20erythrinus%20and%20Comparative%20Genomic%20Analysis" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460082/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460082" id="node460082--3" data-pisa="ggg;6/3/509" data-pisa-master="ggg;g3.115.023432" data-apath="/ggg/6/3/509.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Exploring a Nonmodel Teleost Genome Through RAD Sequencing—Linkage Mapping in Common Pandora, <em>Pagellus erythrinus</em> and Comparative Genomic Analysis</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Tereza Manousaki</span>, <span class="highwire-citation-author" data-delta="1">Alexandros Tsakogiannis</span>, <span class="highwire-citation-author" data-delta="2">John B. Taggart</span>, <span class="highwire-citation-author" data-delta="3">Christos Palaiokostas</span>, <span class="highwire-citation-author" data-delta="4">Dimitris Tsaparis</span>, <span class="highwire-citation-author" data-delta="5">Jacques Lagnel</span>, <span class="highwire-citation-author" data-delta="6">Dimitrios Chatziplis</span>, <span class="highwire-citation-author" data-delta="7">Antonios Magoulas</span>, <span class="highwire-citation-author" data-delta="8">Nikos  Papandroulakis</span>, <span class="highwire-citation-author" data-delta="9">Constantinos C. Mylonas</span> and <span class="highwire-citation-author" data-delta="10">Costas S. Tsigenopoulos</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">509-519;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.023432 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460082--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Tereza Manousaki </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tereza%2BManousaki%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Manousaki%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATereza%2BManousaki%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alexandros Tsakogiannis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Biology, University of Crete, Heraklion, 70013, Greece</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alexandros%2BTsakogiannis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tsakogiannis%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlexandros%2BTsakogiannis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">John B. Taggart </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Aquaculture, School of Natural Sciences, University of Stirling, Scotland, FK9 4LA, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BB.%2BTaggart%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Taggart%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BB.%2BTaggart%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Christos Palaiokostas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institute of Aquaculture, School of Natural Sciences, University of Stirling, Scotland, FK9 4LA, United Kingdom</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christos%2BPalaiokostas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palaiokostas%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristos%2BPalaiokostas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Dimitris Tsaparis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dimitris%2BTsaparis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tsaparis%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADimitris%2BTsaparis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Jacques Lagnel </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jacques%2BLagnel%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lagnel%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJacques%2BLagnel%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Dimitrios Chatziplis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Laboratory of Agrobiotechnology and Inspection of Agricultural Products, Department of Agricultural Technology, School of Agricultural Technology Food Technology and Nutrition, Alexander Technological Educational Institute of Thessaloniki, Sindos, 57400, Greece</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dimitrios%2BChatziplis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chatziplis%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADimitrios%2BChatziplis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Antonios Magoulas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonios%2BMagoulas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Magoulas%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAntonios%2BMagoulas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Nikos  Papandroulakis </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nikos%2B%2BPapandroulakis%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Papandroulakis%20N&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ANikos%2B%2BPapandroulakis%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Constantinos C. Mylonas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Constantinos%2BC.%2BMylonas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mylonas%20CC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AConstantinos%2BC.%2BMylonas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Costas S. Tsigenopoulos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Marine Biology, Biotechnology and Aquaculture, Hellenic Centre for Marine Research, Heraklion, 71500, Greece</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Costas%2BS.%2BTsigenopoulos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tsigenopoulos%20CS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACostas%2BS.%2BTsigenopoulos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:tsigeno@hcmr.gr" class="" data-icon-position="" data-hide-link-title="0">tsigeno@hcmr.gr</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=509&amp;rft.epage=519&amp;rft.atitle=Exploring%20a%20Nonmodel%20Teleost%20Genome%20Through%20RAD%20Sequencing%E2%80%94Linkage%20Mapping%20in%20Common%20Pandora%2C%20Pagellus%20erythrinus%20and%20Comparative%20Genomic%20Analysis&amp;rft.volume=6&amp;rft.issue=3&amp;rft.date=2016-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Manousaki%2C+Tereza&amp;rft.au=Tsakogiannis%2C+Alexandros&amp;rft.au=Taggart%2C+John+B.&amp;rft.au=Palaiokostas%2C+Christos&amp;rft.au=Tsaparis%2C+Dimitris&amp;rft.au=Lagnel%2C+Jacques&amp;rft.au=Chatziplis%2C+Dimitrios&amp;rft.au=Magoulas%2C+Antonios&amp;rft.au=Papandroulakis%2C+Nikos+&amp;rft.au=Mylonas%2C+Constantinos+C.&amp;rft.au=Tsigenopoulos%2C+Costas+S."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460082/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460082/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460082/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460082/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460082/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460082/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460082/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460082/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460082/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460082/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460082/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460082/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/3/509" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/3/509.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/3/509.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/3/509.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/3/509.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
