<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/3/669" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/3/669.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/3/669.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/3/669.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/3/669/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/3/669.atom" />
<meta name="HW.pisa" content="ggg;6/3/669" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A Computational Approach to Estimating Nondisjunction Frequency in Saccharomyces cerevisiae" />
<meta name="DC.Identifier" content="10.1534/g3.115.024380" />
<meta name="DC.Date" content="2016-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Chu and Burgess. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Errors segregating homologous chromosomes during meiosis result in aneuploid gametes and are the largest contributing factor to birth defects and spontaneous abortions in humans. Saccharomyces cerevisiae has long served as a model organism for studying the gene network supporting normal chromosome segregation. Measuring homolog nondisjunction frequencies is laborious, and involves dissecting thousands of tetrads to detect missegregation of individually marked chromosomes. Here we describe a computational method (TetFit) to estimate the relative contributions of meiosis I nondisjunction and random-spore death to spore inviability in wild type and mutant strains. These values are based on finding the best-fit distribution of 4, 3, 2, 1, and 0 viable-spore tetrads to an observed distribution. Using TetFit, we found that meiosis I nondisjunction is an intrinsic component of spore inviability in wild-type strains. We show proof-of-principle that the calculated average meiosis I nondisjunction frequency determined by TetFit closely matches empirically determined values in mutant strains. Using these published data sets, TetFit uncovered two classes of mutants: Class A mutants skew toward increased nondisjunction death, and include those with known defects in establishing pairing, recombination, and/or synapsis of homologous chromosomes. Class B mutants skew toward random spore death, and include those with defects in sister-chromatid cohesion and centromere function. Epistasis analysis using TetFit is facilitated by the low numbers of tetrads (as few as 200) required to compare the contributions to spore death in different mutant backgrounds. TetFit analysis does not require any special strain construction, and can be applied to previously observed tetrad distributions." />
<meta name="DC.Contributor" content="Daniel B. Chu" />
<meta name="DC.Contributor" content="Sean M. Burgess" />
<meta name="article:published_time" content="2016-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A Computational Approach to Estimating Nondisjunction Frequency in Saccharomyces cerevisiae" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Errors segregating homologous chromosomes during meiosis result in aneuploid gametes and are the largest contributing factor to birth defects and spontaneous abortions in humans. &lt;i&gt;Saccharomyces cerevisiae&lt;/i&gt; has long served as a model organism for studying the gene network supporting normal chromosome segregation. Measuring homolog nondisjunction frequencies is laborious, and involves dissecting thousands of tetrads to detect missegregation of individually marked chromosomes. Here we describe a computational method (TetFit) to estimate the relative contributions of meiosis&amp;nbsp;I nondisjunction and random-spore death to spore inviability in wild type and mutant strains. These values are based on finding the best-fit distribution of 4, 3, 2, 1, and 0 viable-spore tetrads to an observed distribution. Using TetFit, we found that meiosis&amp;nbsp;I nondisjunction is an intrinsic component of spore inviability in wild-type strains. We show proof-of-principle that the calculated average meiosis I nondisjunction frequency determined by TetFit closely matches empirically determined values in mutant strains. Using these published data sets, TetFit uncovered two classes of mutants: Class A mutants skew toward increased nondisjunction death, and include those with known defects in establishing pairing, recombination, and/or synapsis of homologous chromosomes. Class B mutants skew toward random spore death, and include those with defects in sister-chromatid cohesion and centromere function. Epistasis analysis using TetFit is facilitated by the low numbers of tetrads (as few as 200) required to compare the contributions to spore death in different mutant backgrounds. TetFit analysis does not require any special strain construction, and can be applied to previously observed tetrad distributions.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/03/01" />
<meta name="citation_mjid" content="ggg;6/3/669" />
<meta name="citation_id" content="6/3/669" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/3/669" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/3/669.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/3/669.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/3/669.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.024380" />
<meta name="citation_pmid" content="26747203" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="669" />
<meta name="citation_lastpage" content="682" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Daniel B. Chu" />
<meta name="citation_author_institution" content="Department of Molecular and Cellular Biology, University of California, Davis, California 95616" />
<meta name="citation_author" content="Sean M. Burgess" />
<meta name="citation_author_institution" content="Department of Molecular and Cellular Biology, University of California, Davis, California 95616" />
<meta name="citation_author_email" content="smburgess@ucdavis.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0511-0084" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=R. E. Baker;citation_author=D. C. Masison;citation_title=Isolation of the gene encoding the Saccharomyces cerevisiae centromere-binding protein CP1.;citation_pages=2458-2467;citation_volume=10;citation_year=1990;citation_issue=6;citation_pmid=2188087;citation_doi=10.1128/MCB.10.6.2458" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=G. V. Borner;citation_author=N. Kleckner;citation_author=N. Hunter;citation_title=Crossover/noncrossover differentiation, synaptonemal complex formation, and regulatory surveillance at the leptotene/zygotene transition of meiosis.;citation_pages=29-45;citation_volume=117;citation_year=2004;citation_issue=1;citation_pmid=15066280;citation_doi=10.1016/S0092-8674(04)00292-2" />
<meta name="citation_reference" content="citation_journal_title=Current genetics;citation_journal_abbrev=Curr Genet;citation_author=D. A. Campbell;citation_author=M. M. Doolittle;citation_title=Coincident chromosomal disomy in meiotic dyads from triploid yeast.;citation_pages=569-576;citation_volume=12;citation_year=1987;citation_issue=8;citation_pmid=3332249;citation_doi=10.1007/BF00368058" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. Chan;citation_author=R. H. Borts;citation_author=E. Hoffmann;citation_title=Temperature-dependent modulation of chromosome segregation in msh4 mutants of budding yeast.;citation_pages=e7284-e7284;citation_volume=4;citation_year=2009;citation_issue=10;citation_pmid=19816584;citation_doi=10.1371/journal.pone.0007284" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=P. S. Cheslock;citation_author=B. J. Kemp;citation_author=R. M. Boumil;citation_author=D. S. Dawson;citation_title=The roles of MAD1, MAD2 and MAD3 in meiotic progression and the segregation of nonexchange chromosomes.;citation_pages=756-760;citation_volume=37;citation_year=2005;citation_issue=7;citation_pmid=15951820;citation_doi=10.1038/ng1588" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=P. R. Chua;citation_author=G. S. Roeder;citation_title=Tam1, a telomere-associated meiotic protein, functions in chromosome synapsis and crossover interference.;citation_pages=1786-1800;citation_volume=11;citation_year=1997;citation_issue=14;citation_pmid=9242487;citation_doi=10.1101/gad.11.14.1786" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. N. Conrad;citation_author=A. M. Dominguez;citation_author=M. E. Dresser;citation_title=Ndj1p, a Meiotic Telomere Protein Required for Normal Chromosome Synapsis and Segregation in Yeast;citation_pages=1252-1255;citation_volume=276;citation_year=1997;citation_issue=5316;citation_pmid=9157883;citation_doi=10.1126/science.276.5316.1252" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=M. S. Esposito;citation_author=R. E. Esposito;citation_author=M. Arnaud;citation_author=H. Halvorson;citation_title=Conditional Mutants of Meiosis in Yeast;citation_pages=202-210;citation_volume=104;citation_year=1970;citation_issue=1;citation_pmid=5473889" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. Fernius;citation_author=A. L. Marston;citation_title=Establishment of cohesion at the pericentromere by the Ctf19 kinetochore subcomplex and the replication fork-associated factor, Csm3.;citation_pages=e1000629-e1000629;citation_volume=5;citation_year=2009;citation_issue=9;citation_pmid=19730685;citation_doi=10.1371/journal.pgen.1000629" />
<meta name="citation_reference" content="citation_journal_title=Current genetics;citation_journal_abbrev=Curr Genet;citation_author=S. K. Ghosh;citation_author=S. Sau;citation_author=S. Lahiri;citation_author=A. Lohia;citation_author=P. Sinha;citation_title=The Iml3 protein of the budding yeast is required for the prevention of precocious sister chromatid separation in meiosis I and for sister chromatid disjunction in meiosis II.;citation_pages=82-91;citation_volume=46;citation_year=2004;citation_issue=2;citation_pmid=15241623" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=M. N. Gladstone;citation_author=D. Obeso;citation_author=H. Chuong;citation_author=D. S. Dawson;citation_title=The synaptonemal complex protein Zip1 promotes bi-orientation of centromeres at meiosis I.;citation_pages=e1000771-e1000771;citation_volume=5;citation_year=2009;citation_issue=12;citation_pmid=20011112;citation_doi=10.1371/journal.pgen.1000771" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=T. Hassold;citation_author=P. Hunt;citation_title=To err (meiotically) is human: the genesis of human aneuploidy.;citation_pages=280-291;citation_volume=2;citation_year=2001;citation_issue=4;citation_pmid=11283700;citation_doi=10.1038/35066065" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. A. Heck;citation_author=D. Gresham;citation_author=D. Botstein;citation_author=E. Alani;citation_title=Accumulation of recessive lethal mutations in Saccharomyces cerevisiae mlh1 mismatch repair mutants is not associated with gross chromosomal rearrangements.;citation_volume=174;citation_year=2006;citation_pmid=16816424;citation_doi=10.1534/genetics.106.059311" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=A. Hochwagen;citation_author=A. Amon;citation_title=Checking your breaks: surveillance mechanisms of meiotic recombination.;citation_pages=R217-R228;citation_volume=16;citation_year=2006;citation_issue=6;citation_pmid=16546077;citation_doi=10.1016/j.cub.2006.03.009" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=E. R. Hoffmann;citation_author=P. V. Shcherbakova;citation_author=T. A. Kunkel;citation_author=R. H. Borts;citation_title=MLH1 Mutations Differentially Affect Meiotic Functions in Saccharomyces cerevisiae;citation_pages=515-526;citation_volume=163;citation_year=2003;citation_issue=2;citation_pmid=12618391" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=N. M. Hollingsworth;citation_author=L. Ponte;citation_author=C. Halsey;citation_title=MSH5, a novel MutS homolog, facilitates meiotic reciprocal recombination between homologs in Saccharomyces cerevisiae but not mismatch repair.;citation_pages=1728-1739;citation_volume=9;citation_year=1995;citation_issue=14;citation_pmid=7622037;citation_doi=10.1101/gad.9.14.1728" />
<meta name="citation_reference" content="citation_journal_title=Scand. J. Stat.;citation_author=S. Holm;citation_title=A SIMPLE SEQUENTIALLY REJECTIVE MULTIPLE TEST PROCEDURE.;citation_pages=65-70;citation_volume=6;citation_year=1979" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=N. Hunter;citation_author=R. H. Borts;citation_title=Mlh1 is unique among mismatch repair proteins in its ability to promote crossing-over during meiosis.;citation_pages=1573-1582;citation_volume=11;citation_year=1997;citation_issue=12;citation_pmid=9203583;citation_doi=10.1101/gad.11.12.1573" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=L. Jessop;citation_author=B. Rockmill;citation_author=G. S. Roeder;citation_author=M. Lichten;citation_title=Meiotic chromosome synapsis-promoting proteins antagonize the anti-crossover activity of sgs1.;citation_pages=e155-e155;citation_volume=2;citation_year=2006;citation_issue=9;citation_pmid=17002499;citation_doi=10.1371/journal.pgen.0020155" />
<meta name="citation_reference" content="citation_journal_title=DNA Repair (Amst.);citation_author=R. E. Keelagher;citation_author=V. E. Cotton;citation_author=A. S. Goldman;citation_author=R. H. Borts;citation_title=Separable roles for exonuclease I in meiotic DNA double-strand break repair.;citation_pages=126-137;citation_volume=10;citation_year=2011;citation_pmid=21044871;citation_doi=10.1016/j.dnarep.2010.09.024" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=B. Kemp;citation_author=R. M. Boumil;citation_author=M. N. Stewart;citation_author=D. S. Dawson;citation_title=A role for centromere pairing in meiotic chromosome segregation.;citation_volume=18;citation_year=2004;citation_pmid=15289462;citation_doi=10.1101/gad.1227304" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. T. Kirkpatrick;citation_author=J. R. Ferguson;citation_author=T. D. Petes;citation_author=L. S. Symington;citation_title=Decreased Meiotic Intergenic Recombination and Increased Meiosis I Nondisjunction in exo1 Mutants of Saccharomyces cerevisiae;citation_pages=1549-1557;citation_volume=156;citation_year=2000;citation_issue=4;citation_pmid=11102356" />
<meta name="citation_reference" content="citation_journal_title=FEBS J.;citation_title=Centromere pairing–tethering partner chromosomes in meiosis I.;citation_author=E. L. Kurdzo;citation_author=D. S. Dawson;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. Lacefield;citation_author=A. W. Murray;citation_title=The spindle checkpoint rescues the meiotic segregation of chromosomes whose crossovers are far from the centromere.;citation_pages=1273-1277;citation_volume=39;citation_year=2007;citation_issue=10;citation_pmid=17828265;citation_doi=10.1038/ng2120" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=C. Y. Lee;citation_author=M. N. Conrad;citation_author=M. E. Dresser;citation_title=Meiotic chromosome pairing is promoted by telomere-led chromosome movements independent of bouquet formation.;citation_pages=e1002730-e1002730;citation_volume=8;citation_year=2012;citation_issue=5;citation_pmid=22654677;citation_doi=10.1371/journal.pgen.1002730" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=A. L. Marston;citation_author=W. H. Tham;citation_author=H. Shah;citation_author=A. Amon;citation_title=A genome-wide screen identifies genes required for centromeric cohesion.;citation_volume=303;citation_year=2004;citation_pmid=14752166;citation_doi=10.1126/science.1094220" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=E. Martini;citation_author=R. L. Diaz;citation_author=N. Hunter;citation_author=S. Keeney;citation_title=Crossover homeostasis in yeast meiosis.;citation_pages=285-295;citation_volume=126;citation_year=2006;citation_issue=2;citation_pmid=16873061;citation_doi=10.1016/j.cell.2006.05.044" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. C. Masison;citation_author=R. E. Baker;citation_title=Meiosis in Saccharomyces cerevisiae mutants lacking the centromere-binding protein CP1.;citation_pages=43-53;citation_volume=131;citation_year=1992;citation_issue=1;citation_pmid=1592241" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=C. Michaelis;citation_author=R. Ciosk;citation_author=K. Nasmyth;citation_title=Cohesins: chromosomal proteins that prevent premature separation of sister chromatids.;citation_pages=35-45;citation_volume=91;citation_year=1997;citation_issue=1;citation_pmid=9335333;citation_doi=10.1016/S0092-8674(01)80007-6" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=A. W. Murray;citation_author=N. P. Schultes;citation_author=J. W. Szostak;citation_title=Chromosome length controls mitotic chromosome segregation in yeast.;citation_pages=529-536;citation_volume=45;citation_year=1986;citation_issue=4;citation_pmid=3518944;citation_doi=10.1016/0092-8674(86)90284-9" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=S. I. Nagaoka;citation_author=T. J. Hassold;citation_author=P. A. Hunt;citation_title=Human aneuploidy: mechanisms and new insights into an age-old problem.;citation_pages=493-504;citation_volume=13;citation_year=2012;citation_issue=7;citation_pmid=22705668;citation_doi=10.1038/nrg3245" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=A. M. Neiman;citation_title=Sporulation in the Budding Yeast Saccharomyces cerevisiae;citation_pages=737-765;citation_volume=189;citation_year=2011;citation_issue=3;citation_pmid=22084423;citation_doi=10.1534/genetics.111.127126" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. Newnham;citation_author=P. Jordan;citation_author=B. Rockmill;citation_author=G. S. Roeder;citation_author=E. Hoffmann;citation_title=The synaptonemal complex protein, Zip1, promotes the segregation of nonexchange chromosomes at meiosis I.;citation_volume=107;citation_year=2010;citation_pmid=20080752;citation_doi=10.1073/pnas.0913435107" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=M. Petronczki;citation_author=M. F. Siomos;citation_author=K. Nasmyth;citation_title=Un menage a quatre: the molecular biology of chromosome segregation in meiosis.;citation_pages=423-440;citation_volume=112;citation_year=2003;citation_issue=4;citation_pmid=12600308;citation_doi=10.1016/S0092-8674(03)00083-7" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. Reenan;citation_author=R. D. Kolodner;citation_title=Characterization of insertion mutations in the Saccharomyces cerevisiae MSH1 and MSH2 genes: evidence for separate mitochondrial and nuclear functions.;citation_pages=975-985;citation_volume=132;citation_year=1992;citation_issue=4;citation_pmid=1334021" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=B. Rockmill;citation_author=K. Voelkel-Meiman;citation_author=G. S. Roeder;citation_title=Centromere-proximal crossovers are associated with precocious separation of sister chromatids during meiosis in Saccharomyces cerevisiae.;citation_volume=174;citation_year=2006;citation_pmid=17028345;citation_doi=10.1534/genetics.106.058933" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=G. S. Roeder;citation_title=Sex and the single cell: meiosis in yeast.;citation_pages=10450-10456;citation_volume=92;citation_year=1995;citation_issue=23;citation_pmid=7479818;citation_doi=10.1073/pnas.92.23.10450" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=P. Ross-Macdonald;citation_author=G. S. Roeder;citation_title=Mutation of a meiosis-specific MutS homolog decreases crossing over but not mismatch correction.;citation_pages=1069-1080;citation_volume=79;citation_year=1994;citation_issue=6;citation_pmid=8001134;citation_doi=10.1016/0092-8674(94)90037-X" />
<meta name="citation_reference" content="citation_journal_title=Methods in enzymology;citation_journal_abbrev=Methods Enzymol;citation_author=F. Sherman;citation_title=Getting started with yeast.;citation_pages=3-21;citation_volume=194;citation_year=1991;citation_pmid=2005794;citation_doi=10.1016/S1046-2023(05)80158-6" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. A. Shonn;citation_author=R. McCarroll;citation_author=A. W. Murray;citation_title=Requirement of the Spindle Checkpoint for Proper Chromosome Segregation in Budding Yeast Meiosis;citation_pages=300-303;citation_volume=289;citation_year=2000;citation_issue=5477;citation_pmid=10894778;citation_doi=10.1126/science.289.5477.300" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. St Charles;citation_author=M. L. Hamilton;citation_author=T. D. Petes;citation_title=Meiotic chromosome segregation in triploid strains of Saccharomyces cerevisiae.;citation_volume=186;citation_year=2010;citation_pmid=20697121;citation_doi=10.1534/genetics.110.121533" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=A. F. Straight;citation_author=A. S. Belmont;citation_author=C. C. Robinett;citation_author=A. W. Murray;citation_title=GFP tagging of budding yeast chromosomes reveals that protein-protein interactions can mediate sister chromatid cohesion.;citation_pages=1599-1608;citation_volume=6;citation_year=1996;citation_issue=12;citation_pmid=8994824;citation_doi=10.1016/S0960-9822(02)70783-5" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=M. Sym;citation_author=G. Roeder;citation_title=Crossover interference is abolished in the absence of a synaptonemal complex protein.;citation_pages=283-292;citation_volume=79;citation_year=1994;citation_issue=2;citation_pmid=7954796;citation_doi=10.1016/0092-8674(94)90197-X" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Thacker;citation_author=I. Lam;citation_author=M. Knop;citation_author=S. Keeney;citation_title=Exploiting spore-autonomous fluorescent protein expression to quantify meiotic chromosome behaviors in Saccharomyces cerevisiae.;citation_volume=189;citation_year=2011;citation_pmid=21840861;citation_doi=10.1534/genetics.111.131326" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. M. Torres;citation_author=T. Sokolsky;citation_author=C. M. Tucker;citation_author=L. Y. Chan;citation_author=M. Boselli;citation_title=Effects of Aneuploidy on Cellular Physiology and Cell Division in Haploid Yeast;citation_pages=916-924;citation_volume=317;citation_year=2007;citation_issue=5840;citation_pmid=17702937;citation_doi=10.1126/science.1142210" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. J. Wanat;citation_author=K. P. Kim;citation_author=R. Koszul;citation_author=S. Zanders;citation_author=B. Weiner;citation_title=Csm4, in collaboration with Ndj1, mediates telomere-led chromosome dynamics and recombination during yeast meiosis.;citation_pages=e1000188-e1000188;citation_volume=4;citation_year=2008;citation_issue=9;citation_pmid=18818741;citation_doi=10.1371/journal.pgen.1000188" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=T. F. Wang;citation_author=N. Kleckner;citation_author=N. Hunter;citation_title=Functional specificity of MutL homologs in yeast: Evidence for three Mlh1-based heterocomplexes with distinct roles during meiosis in recombination and mismatch correction;citation_pages=13914-13919;citation_volume=96;citation_year=1999;citation_issue=24;citation_pmid=10570173;citation_doi=10.1073/pnas.96.24.13914" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=P. M. Watt;citation_author=E. J. Louis;citation_author=R. H. Borts;citation_author=I. D. Hickson;citation_title=Sgs1: a eukaryotic homolog of E. coli RecQ that interacts with topoisomerase II in vivo and is required for faithful chromosome segregation.;citation_pages=253-260;citation_volume=81;citation_year=1995;citation_issue=2;citation_pmid=7736577;citation_doi=10.1016/0092-8674(95)90335-6" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=S. Zanders;citation_author=E. Alani;citation_title=The pch2Delta mutation in baker's yeast alters meiotic crossover levels and confers a defect in crossover interference.;citation_pages=e1000571-e1000571;citation_volume=5;citation_year=2009;citation_issue=7;citation_pmid=19629178;citation_doi=10.1371/journal.pgen.1000571" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A Computational Approach to Estimating Nondisjunction Frequency in Saccharomyces cerevisiae" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3.cover-source.jpg" />
<meta name="twitter:description" content="Errors segregating homologous chromosomes during meiosis result in aneuploid gametes and are the largest contributing factor to birth defects and spontaneous abortions in humans. Saccharomyces cerevisiae has long served as a model organism for studying the gene network supporting normal chromosome segregation. Measuring homolog nondisjunction frequencies is laborious, and involves dissecting thousands of tetrads to detect missegregation of individually marked chromosomes. Here we describe a computational method (TetFit) to estimate the relative contributions of meiosis I nondisjunction and random-spore death to spore inviability in wild type and mutant strains. These values are based on finding the best-fit distribution of 4, 3, 2, 1, and 0 viable-spore tetrads to an observed distribution. Using TetFit, we found that meiosis I nondisjunction is an intrinsic component of spore inviability in wild-type strains. We show proof-of-principle that the calculated average meiosis I nondisjunction frequency determined by TetFit closely matches empirically determined values in mutant strains. Using these published data sets, TetFit uncovered two classes of mutants: Class A mutants skew toward increased nondisjunction death, and include those with known defects in establishing pairing, recombination, and/or synapsis of homologous chromosomes. Class B mutants skew toward random spore death, and include those with defects in sister-chromatid cohesion and centromere function. Epistasis analysis using TetFit is facilitated by the low numbers of tetrads (as few as 200) required to compare the contributions to spore death in different mutant backgrounds. TetFit analysis does not require any special strain construction, and can be applied to previously observed tetrad distributions." />
<meta name="og-title" property="og:title" content="A Computational Approach to Estimating Nondisjunction Frequency in Saccharomyces cerevisiae" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/3/669" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Errors segregating homologous chromosomes during meiosis result in aneuploid gametes and are the largest contributing factor to birth defects and spontaneous abortions in humans. Saccharomyces cerevisiae has long served as a model organism for studying the gene network supporting normal chromosome segregation. Measuring homolog nondisjunction frequencies is laborious, and involves dissecting thousands of tetrads to detect missegregation of individually marked chromosomes. Here we describe a computational method (TetFit) to estimate the relative contributions of meiosis I nondisjunction and random-spore death to spore inviability in wild type and mutant strains. These values are based on finding the best-fit distribution of 4, 3, 2, 1, and 0 viable-spore tetrads to an observed distribution. Using TetFit, we found that meiosis I nondisjunction is an intrinsic component of spore inviability in wild-type strains. We show proof-of-principle that the calculated average meiosis I nondisjunction frequency determined by TetFit closely matches empirically determined values in mutant strains. Using these published data sets, TetFit uncovered two classes of mutants: Class A mutants skew toward increased nondisjunction death, and include those with known defects in establishing pairing, recombination, and/or synapsis of homologous chromosomes. Class B mutants skew toward random spore death, and include those with defects in sister-chromatid cohesion and centromere function. Epistasis analysis using TetFit is facilitated by the low numbers of tetrads (as few as 200) required to compare the contributions to spore death in different mutant backgrounds. TetFit analysis does not require any special strain construction, and can be applied to previously observed tetrad distributions." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3.cover-source.jpg" />
    <title>A Computational Approach to Estimating Nondisjunction Frequency in Saccharomyces cerevisiae | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"htmBWbnHqq0Nsr7RGJdFeJRapZJWYDoj4kq3IKkIab8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460077","apath":"\/ggg\/6\/3\/669.atom","pisa":"ggg;6\/3\/669","ac":{"\/ggg\/6\/3\/669.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/3\/669.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/3\/669"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/3\/669"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/3\/669.full":true,"\/content\/6\/3\/669":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022PtoHCKfmFudkD8lDuvQAdCsWGE5+mpX1JZNp42BSUKrAsVl1tW2OofEtS4FfjvT4gLRvpmJGBBT4kvgkuyK9i9r5+t56Ys5AgMwrCtNGEofYiL5TZcR3zidWPLppV7DluOZsnITXe+T68ifnIC12YsaVB9XiGN4PQ4WFcLQFKFRYVuRAxieBIyoN5kQ\\\\\\\/gjPdWCx0xd5\\\\\\\/n\\\\\\\/8mPy5BuJVwWemKrUaGCNgLURdlETXuqGJhiYkw3XSMh6UjQGyVPVGSrsNlYzeDdenX7WdESkRsZ0UqSOuCv\\\\\\\/Z4qE2\\\\\\\/G\\\\\\\/xVqpsvySKYgfjFID1QBNwzI3PASTTl6pHKe\\\\\\\/sVbAEWHpWrKlGZzGZLxD9Iybgp79jK64UPQEOkP2BJ6yO6P0iEcGFNt+vkYja3WXLQ9dvoGKenU2HNK4utFcCr598MedpFkESYhO9Z2D9A2RFCsjbuj69f+lMaRDjFOfB5T\\\\\\\/f8+NLErry9GW6rA06CX6b8WvVHV6VTtk1OwshXivFSHgL0Z2Gl\\\\\\\/lq3WO\\\\\\\/2CuV2WSOjGpez+44l9QHSCaOYjnZG7HxMidDV67Mev+8iLVhU9+ujQXOlLjtBXWBp9QAX7LMyTUuWCik8obqoI3z0QTdn92\\\\\\\/p8YaEzd7DdUroidnbFNAwGbGh69y\\\\\\\/4qmDgDN4Vfi4OcQqqGVZPGsRaLelZXCwc+B6ZsaBpUbl+nbrj8Rr21c6niRG7sAl8z0oNzwKgwgPZe2YJhRPUNFhoBsXg5cZhHCbnzdmHSO\\\\\\\/kmEwtbi0jLzHwtZIUVLvB5\\\\\\\/GZpsdVnrqlQ5bfAE1uOuRuQcz19N+6wkBjwAOs6smS\\\\\\\/LFm98dhaiSz7BdMPK\\\\\\\/wRx1ZDVE9fyC5wx49\\\\\\\/gdmT8nNdP9ScqO2G8qqy0ju7mTtvo8JIO3QXTR+BYnbTXuSt7BCCuQcMAw7m\\\\\\\/YwVw4\\\\\\\/K1pYP\\\\\\\/cc96jMxKLL95p7gEfoJ2D6eXkxlECJ2tUN8mfgnCSy38e+spyCwjmYikH5tjzWz9TgEc2PiPCYIMpELDYlNN\\\\\\\/5aVLFH7ntVPDhzALhJX1n4FjWmNP4CJ7qH316P+qchztN9J+bNyCEw3nY3mRDRIBufKm9uQy7rsKB6ZItT6o3MoCvlBg\\\\\\\/mjKZ6RkfVXimV8+kVO5sI7JatJmamPROThVqdxfIe5JjEKSt7fwzqlagZdjD06WGP1dkuG2b6V26zdcDR3S+3d7fk2w2NnGbGYKHWO70cIGrT41RwHLoul\\\\\\\/8zu\\\\\\\/VqM5LcG8KbTkpCVsvWDi5rqWBbrcmaiqZBIQrZRWYaWLLF3It53v\\\u0022,\\\u0022iv\\\u0022:\\\u0022FBseRiNy1eB7u8JWmilEwQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221159d220e5d4241c2f9c30fb74ce9dd46f8f3498f8bd8cc13c2f1d3b01a894d6\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/YT4A6X32zY2NUsGNUPTJ3wDCHG6NZ5Si4ZYXHzQ2TCImQJgETgujQi\\\\\\\/2Ml4T5zNgxKNu1k64Kp\\\\\\\/PK1Pi0DJUxi1oCRhH09fl8Lx770IlsPSoP41mWQcYVqaH\\\\\\\/1FGKu2Y6B\\\\\\\/VznD042WyYBN\\\\\\\/9rDkoVc0JdGoNW0VBJtMl\\\\\\\/BETq1JC0F1FT5UurEDgx4Eoh6AoMDO8QbXISx4IuXXiZYM4gjGGPbQSfs62uMmUVcE4AfNcZng1wjsMJ1oLMlsVxgl0pS27YU60eLSJi\\\\\\\/Y6f+RQdl+YgwohjMlwh\\\\\\\/VIeFWwB7g5x6FV4LbYSAlx13X1oZf3o+bXNzQfTHTrPWNpXTud0kR2Sx9pdIl9lwHf38D0sbqNtNjBSzYuolFXdz7+jd7bpJxCmMtfXcZiVZy6+gNdFkfNiBFMpD6ODAiZ+vs+SB88I5roDHgOXu78u2LWyTln6yWQ10Jz\\\\\\\/iGcOmCuHoVYvnv1+5IfTYkEAKNV3fcLwgcIHlpWjViX1ooSq1OeO3tHaYL3YQpoo2846jxQLkO9XYBv2pjiXgphzZPDk2h9klFx2b+\\\\\\\/\\\\\\\/E9w\\\\\\\/0ZpOPMA1f2ZIJLm50uE7Oa2XYbeuPWXRjIfcYLPLC1FxdTdhDM3g5sKZz4GuiM4J+KmkbgHkaG2751mCPP84adE4t4FBsp2e06NNFLHnHRzafIR6kwLXavis+uYj38+wf\\\\\\\/GLGzmQ86HY1QK91\\\\\\\/4kwiOlRsWYGIgeXIEvJHYypxylzGm8jaRUtDt6hPj6fL2Stc688AOKUh4IV5Jj4DXdCycRpxNyGZoIUPmYZsPOMf5WFGBuKP6IHKNtDWoxOl8GyG4l0NUmrb\\\\\\\/PZE+41GRgCXJJA7I3WKn2nwBwSNcFD172Sv0lbOyUvXu3CJZtuBnxaOz5JT4yk1fJwRLrR6nCe1mT2T\\\\\\\/sXk3xmpFdYReUbUevqnAClY1NE7h2iQ\\\\\\\/xiXiH0FL+l35+T3lHOSoox27nserie9D\\\\\\\/KC2PuXnHLK9xRt2L1ihpozcohjK0tZxd+p3Ll4FsOdgqhiGRbkOMeLy\\\\\\\/iEbLEfWVay7Z1kwVF13AriL6Lx5qkq38iofCh5b+aUXtnIo8VFjIbH3o1bPEhkL+5RBG82J1gDkjjYmZDwCHgx5lHK1coTXhGMoCzSb6Ds3ZBUFHLx5O4kiF3nMqcAfdwvZuOxopUI5sOSkUXaCwgbReV49demEweyLjgRmhDQP1\\\\\\\/3ZOkrngtcXXL0o50tKopZnKIlhs\\\\\\\/GzQMQx2B89j\\\\\\\/if9YQWKkQC9Dp1MUwvk+\\\\\\\/K9bMZ2asZ6zT9X5+qBMfFZ3qFEyvYAby\\\\\\\/b0kP0O4pYbjDdKXJOplGf8O1vLzGC3TzSoM2P+kqQ4lJw0UP6mF1qyvDeDurTYnM5Cgim3n18u5MkkLBrjWT1ZbfK2YP4+K48sP3rCi4LWoC8PO03nQxq\\\\\\\/c2QZ12yH3CSa5Cp8WElFkIrT+aZHOhSJM1rSNIj8+NiGnYqcuh8d3DAihCiNoNjnORyekf3iNlqj3jpHZGNh8H+KmQoSeO2V207qK6Qur0JEXkc4MXqCbecYA3yUuQyaQZrGEZ+ENtD85GdeowaM6YHTlBoKjZPWu8M2+DpvtjsqLJ4gyrxHXohBY4oeCJBA\\\\\\\/tBRjs53AUFlIxqExNUF\\\\\\\/mJv\\\\\\\/vN93oJE4a1UXgoRJSIiSr7ucq60a8LZX2byWf\\\\\\\/7Ym0AKZRnsJnF0ZWhWDHGKSRq\\\\\\\/\\\\\\\/XggT7536VCx1UH6NDFzPiL8QbpWaCFTWbu9dFH\\\\\\\/GQFobQkM03bL\\\\\\\/NIYG8MEtNPGMHqMaGXLteBnm\\\\\\\/6CUUpcWnFVT73onyynjqYyQ042+wcdRQBjfPOUqzve0G3XehyC6FOGjEpkpzg428ADncAr9ycwxjRvCMlNZIek0Id0I5d1A7VR1DBsOGkaipb+hQ3+CP4JC\\\\\\\/WZDCShf+aKO+xDOMBF57y0KOkaXRcpEw0FnIJINKptmNqgwN28+eP9UfJ3V34HULyQDgfDNdPhttICfEL2G6tMxCAZcoIHNSsgpuPnNXg91ATTc21oaE\\\\\\\/Bvm1g6LIvFNEo3GuF76kgm+Tqelgr+tGY6QOIoOXdWpzWNAiSYvJoMrY1aB1InnjI6iYnT0c5mXIx6gklrKzXHewhwPzUJACFR1dZ83zteb21og0CrhPr6QyykqJskM8kWNDBnJ\\\\\\\/dV\\\\\\\/ZfDuPnkBk6fLapuFqVghfzVg0stfuCUDshhYBYuHSdWQnTFk3vI9SFb9rB7Gtnsras+5iX8+XpLW4Hef09qLv7PIxZiPz99heD7fWOa+13eXLEYQuz5GN1NQJde5VxV36ng0jAKkobVPLUAAx7ZRGW8vQHyQNq\\\\\\\/S8AWlJBJUv8D+1k1QCOYPaj6DRh2ANf70V+7nupDYZ9rY\\\\\\\/yb7nKDnT5OTjJ4paEHDl6SbVH6LDDIoyvTc6sPqpGJSXCNdYMiZOWeXb8M6yML9snLyt45OP\\\\\\\/HEskhyDgkBh4pbbXVlRajH30gmK7GDo+uuLsrawsw2i3A\\\\\\\/uKIT+nHzGSA9t3S\\\\\\\/M9hKIdErEa\\\\\\\/Kk3T6Ms6I0yhyV1OpH4uQa4Ldo5UqgA5XnJh2BCbG+PQ0MMPUDm8E\\\\\\\/SNbnKFnqh5YwPU8odrd3WtVvxO1AVKbNMZN4yTsliXd8hC3ELIOAtqG7ljj9uOmZCUMgwuN1CsT+kCb47gm7rI2gXaM8HLIE+g8qfB3nVBOwfT1r0mUCrgwy6UwlOX7IvUKHe6weEuFZw7x\\\\\\\/+A9kGXSG+RUdr8RsnDEBO1GWH1il\\\\\\\/Uwa0SkgGmZELC31PZ7sIu5TeNfgvF+swo+Zidvzjp3gDx4tTCSM3Hl78naMcFJLdM+pguWRgEDCCctM6qnYcsqFjmPQLV\\\\\\\/sUJyo9yKPrlyE1TEr6UQEuL1Cfdso1pW7X70JbzK8NlGAVNYeQuf54PML1qw4+egzASRUI51EC3kVGfQCQfeqKtUaUzoH4cDibEHDMpzV6QdSGPpCrGVgJOul0o1iORrldt22OXmNMbmOEIRwloId6cnVReQY7\\\\\\\/oIr0RRmrSQuzwfQY6edGllqtj3tkb6pCu+x8bNM8InD4xjW0+7F+yTPKAz0KDJG+HFQPHZ11JOLV4LnJhxnH9xNZpqVnu8wfFY+GeCB0NSSjp4nIjdyzLhMacyxIUzWeEbCETYzd1QsxA\\\\\\\/6wo2m+Kuv1Ats+zIFBWPFAsvNWTIDX7+9TE1q7HJlHTyq3s0j5Ftn2pf2z2oHsFJwq94\\\\\\\/xc5UWb9lM95NG3TygT+sWXgYmXe5X3lfa0T4Hu8lGdjfzTOd5S3pXQ8deEQYugfSlCUNbbbhVYVnKryBMw8Hfirq9WDDvN\\\\\\\/fAlPVCv05GMrrvWElCNYQqS3yEcmYC66kMA6w79yj+ZMoER2WP7tyWNOCcXTbmStD3k1CbLwgwZGuOeQRIruslDj21e4DqhvWVTxFeKDZ+CeJlPSmCCO7VNWHXa9egTzGmUy4rI0wURSMsBKpXCHLi\\\\\\\/OFtrna4ri0S8H\\\\\\\/REZ+ctoOg9msc\\\\\\\/M1kgQ+DlQ5bFMGe99JpzB3mG6+1NC3m+F\\\\\\\/4ITHNnjDCCw6oSMt9Nt2s0Xx53alf2ucQHLkKZ8RFqaqOjHSPyBB3\\\\\\\/VTqlHX4FrLMm9n\\\\\\\/J2Ap6GL6mFYIxIqZz71XsLkKsBWpKEzIMVWiRUWzyt1QFOx6JCvWWPVxgoB7HrEihTo1YhnpgRfaxj7sB4t\\\\\\\/b6YoFrwYuoNQIDtoGBU0ej0fUC1SsDhUjY+yDUzK2LdDLFG9lb+fGRZRnyagoOEFzfH9VTIGuF+ccxPXgR+7hHfL9Qputla7rYGz5NN\\\\\\\/Dbv3iScnWLVD2rpDvhN2sO00rvGJKLdThJhbNoLtP+GkXZ8I8f3atQ+tdXkUSK08PuFv\\\\\\\/mBowELnHv3G2goI5HvaBqaYQALjP1Ce5tvBJJBioWDYVHHvgng35vfrLIHqEAEaOPe4Q\\\\\\\/D6gh2IC9umLTCZHS\\\\\\\/f819T3i0hXEQyZgXCms8PwYZh2rzCKO6+RUTBRyRa17vg2RqkBZltBjwUKGeZF0qKemIgjOYAelJmrLVOR5Jg9KDWZp9hvv8AgOVwo1urU+i4luAHt\\\\\\\/R2TA+pnyxHsk1aceZ5QekOVRGX0GWFLG7NK2FkOdRNiQIntkR0k+i7WWGy4l5pznJL+RdECck645Nr5KI+b3IjpdW9P5eVAkA3vPbuyeHx+pUyg+Uqul00NvkLpkxlpFIckdrvzkVbnlx\\\\\\\/kxVjE\\\\\\\/TYr0LiSsO7MAeY6bp9I1DiXYg3zTMQPJFVKv0w5q81G3acRX+jZRlhITlsoqhLUzCJo0+0eRuquQ3cOlsTI5J7NdDwcE1Ki7XeiZfhYeGEvSDlqxrwRFF4QcQLSj8AXLrA+UBngsgD1w5VdyIb6TynYSf+fGd4\\\\\\\/PLUfMZpFGBPubaZWCoP\\\\\\\/RctXhMnheb4\\\\\\\/kfr5fc0xWPHxf62eaIIfhqG2SJCV7XQV5YteFP3GlA6tvYMKDaQzcPxBcOXLy921cxHP4oQ9\\\\\\\/tVDNKH\\\\\\\/EIuK6ILtjTMWZVYPU7sav1Y+LT3hoTir92o3kxMJdDZMGYCdE3gIaLrLSsI7c87J7vmWPAxLkt0n0OEmB3VyIEPN34fbAcbyUlbakc9lMl7AWS5lQ9X\\\\\\\/zQWodJF1S0eNBrnO5UKLY90RxmbqhDcdgEVJr\\\\\\\/64OmVeuUUK2+8I44+aA\\\\\\\/ygNmvbYOAOg5Y1QciVX7VFyLG7sLwP8HypKy8S88IFCkAI0ecqC0rwdhUgw7+H\\\\\\\/EAD4Oh+D3vHnZAUw4\\\\\\\/9zEXvnm4fyg6xUFfc+A0KQO\\\\\\\/PFbvFjQe9zcf1rQn1UgAAjDnjYtShPuQxSzEOovpX0nhBJkMRS9dzpp7TK+GSnQJ36B7riSZwCFxkR1bY1iunLOA3zTna7LtuAuokdZTpGPLVSGapdyDVegIxOqoXx2iRsfS4psQNFEf+\\\\\\\/iYxhwog4ws6a03p4vQCbtnJzXXVZI8X6P0JwS+EDbbOt9fh9nYaKLRrwXWUrmiG+tdnEGhfK1j7gzHJQJIfpwgM+PRQOUeva1Rt43RnfcqhLhQ6MmUpbQUB\\\\\\\/YrmdpFWtvqdC5YqkRlKHUsmdtGEwY5e6siH\\\\\\\/1ldWmPENzqHKW9PzbNY10ECBQavd92qq5fSESWywwaN1c6nYSAjAlB2k6WCAUvRBPxxXCY\\\\\\\/IlMcdLceSrj+eiZEiQdSKiHVeRTAZxx1MjpMwtmsR89k7Jb5v+hPtPOhmVjbHEekE8kqu8VcVN0AIAn4z4raBRn1\\\\\\\/gRpFjIeBnCgN6UbM+CUVkKxPLeed0NcQnOppWSGHY3L8p9c9sf+02cxD6tL4mIE1\\\\\\\/3Y9\\\\\\\/kCFRAnkRxNkwxfo8F9nH+041isW\\\\\\\/2O8BnH4+YW66Y23nILEV2\\\\\\\/wrjxCotLe0E1vNAttYV1KkaV3ys0TwEmi3NtS5WpZ7PPm\\\\\\\/lXmbuVN1Jqy77NhE+oxuCeMZC62ybTaq2W0okjw2L8HsQPbchZiEayH0aAfBUJCzIsWhPJRcAPaSw70J520X7qBJEH8ruFaP9BPtBy7zS3GrEkP1oUGw\\\\\\\/V2SamUj4M3x5Y7+kHJhyofYJlFi78TGmm1A3\\\\\\\/8U34MSy01tzOezPkaMGuHdma2R5oKP+nYYI6KQzf8dVmecf512qGvuFNuzhyJx0P\\\\\\\/X+Lfg9drrkVOx3G8aFKjeGa5BCMB+\\\\\\\/N0NckfNR8F5wmi97c9UYNIbbsGPKAPDuB9RGgTZKS58paO9ULi6XpNljhIFFfwp6diRcvWoKMdvxMriL8cLtd3QFHlH9LLs\\\\\\\/lp\\\\\\\/vjfnByJmCK9IGvD3aEeyAPHq64wwEPgICj0YOgko5CueDJB8kQUYzv2dYzBgtFP9QHwRVln14LhBmblmLLNdIRVpg\\\\\\\/ilvXg0t9Nnk\\\\\\\/kutAQq1eGJnhdX50yQpscTMYtuuKFuCxa+AJiGsDJREGewVN\\\\\\\/N9tY\\\\\\\/oJt0zJyv8FD+tdi7yKOP5i6Ma1H13UrFyYIt+QH04NDbQMGl9nDgArr8VZJPQZU2yFQ8MTtR22gH69+71NEpG+4lQlEzSyvhrrvakDpaRWzAM0Ny6OK2qggwxHIKGg7gERWP+2npQ47XADO39ZTF8MvkGAGAnJmHoi3G7FfIS0tdjpUhSY8OIT3KTlqGkj1Ktoe+BaiZHlRhyxQX3MA2zNH+\\\\\\\/J9AT\\\\\\\/Sv\\\\\\\/zi3hz8pWVAdtYr81uqk\\\\\\\/F2ChxI1J6c0\\\\\\\/TmSgU3ttzythrwFqeTOeKI84\\\\\\\/cgVgLoKHC9TxbSMDMwGDmXEHE8LeuLnezbUqGGgdnQ86sqHTFKiphfo7h3PHVQQcnf69TGfqS37wZEtGyW+99u6dg5uW7oreQHGVfWebTKSFEz08fwCyQ7FUnnbfftyewfTWeZ4GqQnbtZJ6rAQ1qv\\\\\\\/36941CvKhp+Rn0UDeEzprQ0XlbQJmUxn0Ba7oqyY+nAy1\\\\\\\/Zo\\\\\\\/2ISRrT+ZiFuZBVF0UxIZFS3VEL5qkinY45X2jWiGuhyEArUwzrXUIQq+9UxBjDsmOcvy49DhIxYOLvFXZ7Wawj\\\\\\\/oVHXEE6\\\\\\\/VpAeCWZovPoRYkw\\\\\\\/T0xPfpAj8FikVCBREUDTvpwkx\\\\\\\/Q0e9Vj9A72h2wRo8dQ+3Mp5LsMvhIjFwCHQBmO1RHl2b9LHqBjBQMUxe82aioRDqMPysxSpTJCifgNoD4AZwpVdc1sBG87F8pHTLTuD+4PeezRASvbHFcKGKD5YEwIlA2QearL16ihZV1izM3g907KyTtJhhZOjTuDcktfOyE5hVj6\\\\\\\/dxT77GMlf9CRzGbRERYdah6IsJlzTpe1edXS0fJcfXoKHeZyijTND5qc9S+sqropVsDQzeb9r0CM4HQTmF5xJzOvp0D5H4cTu7dbuugDuB\\\\\\\/+XvNepBM0q6Xs3E+E7K39QGTWDPPNQMVp1mQ\\\\\\\/nDayUsdrCZbmi0Be3wdQpIlWRWXuU0rhfiUYAbDxfrGDZh\\\\\\\/JRe2kKfW74MY+0hyhQ9lP0oRti2bd\\\\\\\/dBn36vqr1mRpPfcLv61osJ8S+mm6vWIGNQwNkmIihgm1h6NYLhV9cdcig16WtHfVTxMkdyzX0CLpoyicy+Lj+mFZA2qtvoBMgaXdIS6xCVjdsgllOdu3BzQSsm64Ex\\\\\\\/hGt4plyIW9ZfqKP4G5LhRew4U+TO6T17c2NfmtS7Yu3pQ7J\\\\\\\/CNNOLStbPtPskbKin4SElg2jPrMZ5FEnMoh4qqkPjB543KwhfrMH\\\\\\\/mzxY8TnhIUcRQQTQQs8DiZFkFdriTn\\\\\\\/enLcopx9N4s8capVJuCFt3vUdFlRKT9CWjV2ZVpe96C1ZPAkfUZl9NoVSc9tnFLgrCEoKzgw75CcWIiUUNOmxjQYd5qOfuWCggGPijLQPyFLEQUjWImEmxq7KZdMh4AwobOd1dflU5y0UkCQoeWLhfAtBXv0Cul0we2I6eEZzWgsgKvki27J9h4t4a82hfgaVpSVE0IXhAnqEwOcJqwd\\\\\\\/ExTi6xVfcsSfZr\\\\\\\/xugkStJhzpTyDNBX85eweFKLS7yWTTa1zy7kNIHEb9+YBsEk3NqrevXPZKZ4ZfZ+ZfHz1tgoJhiLRDUxPB1HBFLwEgzbPRWZQX6Lp3hh1WjU0OfVV7NW0x5masFzFO4\\\\\\\/WF9gxmH17mPM2AkiiP4vWh\\\\\\\/7OIL29V582aU8yX0thdoVenqO2tLiZsaTMe5vhAHiaFHgA1I+VmwhdNPc8D5+\\\\\\\/xl2uQG94wn3YLHMyl8MJLqz2T2XgMjPBwDkX502sPj13Z1RjLCctfXcUQ5Y0UdsWj3UK89SuI0x+S6IGZPvILRPJ1F7mVYkMIu1l9Hq6WHdNzt15q2P\\\\\\\/1eAqSK0f6VNAWABtn9VPczLDXSt2w1cNAAdDZGFYHRywyxSDbHDsooprMNnFWWJALOM8zoilM6YN7geCZC1gUJCpfB4hPKAxIk\\\\\\\/XupBpi0y07ompKZtST85RFcu69Taq3zeIDPZ4RTnb\\\\\\\/GPm5gI1R71KEdoY+8rYHlUl0GOKorIeObYZBxBocCXxWKSuc1+Pcc9nqj1eHPRKmfnUq0Zppgy060XQMEUSQoNrUD63B8zvb9pPOkWvoNO4i7\\\\\\\/GEBS0TLMtD6th9cjFchKbGcOAsDgflL9ZUt9bh0Bh6Qkur2DiXobMv5xbLJAQMs9fpg6kK45dCrMpdw8L1t8nu1h1i11bD8Zy1w59g4sdD\\\\\\\/Omx\\\\\\\/2lzm5bTphtEGYz7MdS4xm8bxlMuTf8rig54nUsiSE0hofCpTdWa8VMMsqSpVF707zkxBYl\\\\\\\/C\\\\\\\/uAlbn4rGOvhqHGKdFisrGCKUUhKOeCqMTUZRTG\\\\\\\/R3PzvReeEIXN97++aOWQ1bIvfKC9t34exBxbfkrRADphutjn2R3vKojDTyZwIalD+RnEExzSQGvV21CP90NcPbCxUPKbA8aDK0f5DFXYFT594FoJwvWBDqDbSJaK2XVWAFiwceq5kB6YvSIhrYtLYCfygtFLutbhYvKMv2oHL20mAuPXaT047pxCOLUTfFvGxAa9Uk6OdT3EUi8JqcUQCkmOixNo\\\\\\\/T0xF4x778qVrAXsoYG8vjToYz2XqKv0HRtZc9mTXfSKpiVLKlvk9xk8Xfz6oOLJHmqZrz8qoJAZLpTo4rxN3Yfspefq\\\\\\\/gI1IxXBGc0AH7ifnhXzMyh2aYtP\\\\\\\/rE\\\\\\\/yHAvY5FjgXE2d3C31wdOX0bTC\\\\\\\/lsG1wg6GhNiB2zPcBYs9mdguS5GI2i\\\\\\\/UYuucmT9FIwAJAOHZ+TAZa2vUWC2T95lsyBa7jc4USxHGbKsTafeSpkFvvSzz+dDnuExal\\\\\\\/mKLL3hUoPjl2MRpcPR9f1aVsYuum4JCHlUEJB79rzbizTdTmbIIvgh4D9KsZlNbIa3S98+4KOcmnzrLod0NZcALRSObAU7oux8nkQx6mJFyO0YlQmcYD5rItlZKDl7siu\\\\\\\/mOVutESfugqpLsT+3FfO3fj+\\\\\\\/DAAwMABFYNNScHV3X7kCodj1rMVjdbeOwZJmcNRlkV5UP07tc9\\\\\\\/gCRU6fIAPkw\\\\\\\/HDMick6r90PdfmUt2p6eBZDS9NvfFX4aoBqb+urQKtaeNxAzIhT5B+ztWsy92\\\\\\\/nAW\\\\\\\/vKcJIuz9X9g8LVBDVNrscfRejzlDLg8N3ajWi12KizeHj7G3XnndvlZlFhCuJIHIBc5zwQCJUudCCuQLY4Zi0p6QuI5DcOG3Nok94TNXtn54ok3MY4HPKxSvgHNXls\\\\\\\/Ptc6cOHqOZCT+92OlNvt4LnwI6n+wiMadi0IC+klfXkIQ1Um6KrZvHJfyxtM0g2sqGeqhYg1hkKqUmKnD5oz4a7gXn3sElkeG5mqMz4dLkHyokUE5xuON2mqk\\\\\\\/d4qXZ0We5DjdYPq4AKYUJ7a1xVAUAgmknSKGwzqyu73iv0OEdN9LCrRa\\\\\\\/m7Iu+2JyKr7fUln3gNGCXj1xQGyc1L3KUK9lRysG0uxnU4TkieIc0YiSItlvNYr0SuK6abXNlepQk2yb4n2O1+giuzS3NJlxLKOyaDzcIq3YCs5YX11N6s+Ob1CTBHnW5JfeJ1gAeMletzss9vjshP2yR+D5x3NTv3htqF5+wo32rP5D5Algwxy4lhV6eA8NQM1j2PoMJwl7ldJ8qtCp4RtuGRwW6tLlvb\\\\\\\/d7tV7jQr8HgM+Ji15u4UkKN792KL7g+6pM0xZHbvLzbhfsoQLIUYZiY0GDzFu0vAvN13dbD2uEG1ZllhI03pZ8e7QuOdUvfcPdLcNdFo4\\\\\\\/Iwrv5EDz5YSbNo0mvXWswKfj1V+XqHEtUicKzCCsaJGuPeoLEnrErxrWS+85QsTqHrGWJyVNgj68DC9aQZYcV33XKBkWVSBP2yMEj+QXWGY4U80vaWS+OHcrqAt1KQEqmZwPnJG2G\\\\\\\/oyBRjDMncOmNI+k0JWKrPhJSZFoWzLoMtmdiqVZ2SqZhp2QTmJRWng6sM7vlPqbkIyIkPCM0z+VNPmdGv1gjomGzwIM9R3bJM4DhmM5Ulx1hBbVHnCRqXYHLruF+S0OfI\\\\\\\/FUCoNI7wS4FKf5lxbNhtuNIkAZ6vvdXqK4ZlYRuHE\\\\\\\/4LikpvHJ7RlVbSb3zxOcQglkaq5zWaeSAFhuT4LBJP17pYP09HHAuz0vz5bdKUrMeZopiuSWnEnKGKxykicx\\\\\\\/Qrrhw7DGNMSVzZDbJLYHLN6IBmqodpsXWLOfkvLxv5FnD9soKvd21YmzFCRlDut7idNHJSgebqLgJveaWoUfLuyuf5IriDSBbLallpqgtiRtRR\\\\\\\/JgS+5yNlw7Xg1SV\\\\\\\/7HPvFcOUh40LFNAFMV8XQ3g9AnCofbIw\\\\\\\/02C0zDjd7XMJykeurZNBLfue+qtNjnLDj9z0ZZ14CHzYTNSYIDKYe6LNuFXeUIviPitxuHGTQ4oocNxMG1935jP1ak6aWx21ufzFtEE2X59CJGJ97\\\\\\\/URjwiXEQ5BhrR68VfSxt1G31iXeCVHq67SbGbOdkk1SvwXa9Vdv3HptyHChmw+sm7EOxt\\\\\\\/tYUTWURe+U+62BNmQGubl5OQdWjYCm7aL5H9Zh2ta8MzoaimCb9XL9LTySwktC1dtdUX7Ju\\\\\\\/9U5F\\\\\\\/8EUzvdqfEYTYUWtSNYE21ShSpkAfvnG\\\\\\\/MMy7Iua+Ajfw+ajZzlzxGzEh8wDXI8MBv\\\\\\\/qK4FdvuCF23cJGDy88va1uv\\\\\\\/zN\\\\\\\/x2ScSEOp7vFjQS7gQ8Xq6I6LwcaXs2eQF668Wh6hdc1+8vF+QNTRxvPgpxQDkVORQhjpBW94nKh7o8o4GanHEYcTkb\\\\\\\/gkVehgDqXTqgq3GsWy0g92zTaDIuU2RGCltpxVSHxBFJ82v4gEa7CNiCPWTZn6Fwfv7b2tFBJ85oNbKehzLXhhWGUmjPq3ESF72umvp7tpR+8q+DkoCxxp0Kmbjn69YiLxofXeoPUZ4uVD6mgDWQ3dPlZrLZ2qwigUvWTZzBVoNT4SbXW\\\\\\\/hEHoJgQHJqtLHA\\\\\\\/RTbb3Hv0xo8LK2SqySmWcPPdWrYWasAaIf09YY0VFjoDCZrPJ8QJQqGY3X1BD9r\\\\\\\/xAbJeigG0TFVPQNzLSLwU2cbeM0wl95USq0TCIP+P3BCEA\\\\\\\/da0OVCjwILflcBtG0O2G9V3aeOqPpnWb6xIXXgprSVJ87Oxf7MHioc4F+PvYd0UoyXpqhv2Urs+NIFx9UyKzceHBMdRyqkTf3a\\\\\\\/lMUATLJJbbazjGdANwJtg5EkfuRpWK80w0GrQRTAljXiberBwNwcVncWg+jx00Kxt1MvrOTnHsOltyS9BBsD+XwCRYIHZeVxi9kSRcp1PsA1udPdK8GpiTFnHMfb1CvmEhbzZGsv8VVcNbO7fumeALejjTf43xBs7Rqx\\\\\\\/ATaK0M57KPzirWnxr+R3Hv7+80iL3vTaMK+4SCIgRMTnlD09zU9nLeXwvys8o\\\\\\\/\\\\\\\/iRRoPl0NoyyzEnVaR3nEe98h7em69mkUTYByeQqViLx8ekUgImRt4rROEhsRRcZspyqek5he4ioxTzwjw7rXaCZNB7nXu+u6v+Xs7++i3YoUtBCJxmKm8T5WuNQXOteo\\\\\\\/8UvbV3yIsIlha2r5tLGgVL1c1oDGMVIPzxNmliLeQCBWuQ0m8SEWkOLmFUhjitYP8z88RqsA5GvkkoL8AaCb3335XAUB3v5aJY9\\\\\\\/ewWgtRmGsxxItjUo9zYNXl1LSmzZ0dayt6YvwZOCy56RG4yIISSrNLQLtybd9aMxNmmBSYc3TjADfobpn8kCGiBwkZHwDD4OGtAA2wlh7jpZtqGM3PD+DmxNJ\\\\\\\/L8JGMB6vZdRIX14SQrPeYWz+j0TPuoD6jm+A8\\\\\\\/nNeP8fyMLnYFgF1f8vPry31s2VWtikqLfSE+MssKSehiICMBqnKLdzqVENZ90Sw43uy5o2QsJZm2DZpjt+qI7X3xjDRWDP8SBwrDDxyXIU483+K2KsUGOE9UJ2n00efCxD6SUnXIPEMXyrq85Ou6ogvt\\\\\\\/5ukq9xsPT\\\\\\\/0hp\\\\\\\/MLBIp\\\\\\\/NBGxhQeZokdMiOOzIHhF\\\\\\\/od4E60Szxi9q920bXrfH+xNQG53v0UPDhqREiD5LYb5ZcE8t\\\\\\\/IYj2DQwpOPJEqKserquxScmAUGWfPGX12d7XJ6piWaWNIEB+zqzW4Ht6zdAFevF6YLh9uUI7yqxgiIGQ82iG3pYqxg9Rd0SYIW4dJBnbjVKDjhswDAQjDMD4955ignMU+H7hn7QvKSd0CXUkRQfbz5Lxa7Zpz9qYtnB3ar\\\\\\\/KExedkio9NQ0COPLf5gRry6EBdPXNogbSL7KppltJ\\\\\\\/o8gfd44ZHKjqLKsIqSUPDfZlVDpVHVgV5fA90VlKn\\\\\\\/PDzEWMRzEnxYlndrdLNsEi0u9wWBvwKTaOBJ0iKCvlW7cGxtu8gRpEiLUgxqUvDvRveyZNomow31\\\\\\\/DmVRKnPJf1oLYw+7ffvbmss9lDYbrhpPy7+hN7nogaZMfGmISY41rqB2ThOJq2Vs7SVO1mS62IEQZ\\\\\\\/8USzGsc9N36Wg17W7FkCE7OCRLjIIDs8imzhVjBiMmoxm7rCg7jRg0zS0RpieXEZ31hWe7eMGyYl22ke66E5BT0fMD+Pe1TdRHuu8FZykwcENEKfUCzTgAj97Me92B1toX5Ec1yweDBLRY8TjYnu0wu+HCW76rlhfuS\\\\\\\/jgB5GFOOH5zYbDCPgwpqUhQ0vJDeXjQ2cz+xlsRDuRmhcNccobtvG4QrG5dmqyc3vWMLX0tyTBX+\\\\\\\/fuZWD7AUgQbatoTKwUb+jvqxksdEYJzS+bRADKBesmbXJN3Twj76+tPI2yKqkPlsKWLw1EZyQN9MbbJngVli1vczI+Wxk65YeUzNCzX1yt9SVG0VZ4qPIpo4CsQXyREi32gahwTIPsOXM7fnvpF4IEvYsmmCt28COaSqyxf2WlpeZnP4uLu\\\\\\\/tKh0297V0bLA09M4zMJesjnak7TJVHJjgZxo32HSW0Hr+RkltPwH8TwtfOIL71lurdIZ\\\\\\\/dGcpsYuil9J8+66rv9rFIFSMy35UvWkDk69uyv7zqB7mxoiMvZgRcDKfPhY+BAq8IwwU9QOuk8bARjzWQCvOCrOEktHaT8KqBxyoGW3Qnw+MXjBe90pdEECzqhPXystsCSmnDaBps38np6fC1gj91nkEq+H\\\\\\\/F4PRIC\\\\\\\/HuHVV+D\\\\\\\/uW71zfHGAzO7hCeHISaEqEEdVZiNzmiHYS3E1rLRzAa30okhI3\\\\\\\/90AKaWjcWVJN0+ATtybS4hhKL6mB1zbvP4\\\\\\\/mB9IePuk7OnTdLMub70kO+N8GN5Uief+\\\\\\\/jqMAf9eKArGlsic7a24B\\\\\\\/JAIEbLNaKdG8UMk+WBctZBB2rkCmtYOMBR8NshClwz+BLxYkqQvFSN2nPDrU8V2TeyD7WINiW6OH45drH0vAcywPV5mgWQroRc9DzDj2HEIIejoTJWupUelAvOCH\\\\\\\/T58ZmRFwdWqv4x\\\\\\\/bueQZ4TMwSvkP3x8tIlZruqbSEDlE\\\\\\\/AJSKNkt6cAteeTOmZIEB7eds8o72g69Ineh0M1Bm9yCEGN4F5gKHBoydeOB8V+EM0d2VimTB\\\\\\\/WMWVx80+INQUjrodvPhIGBCLZCYV7qOaj12pFCJV1ac8Jvr3sBbq1LnKZweWXwskpBgS18\\\\\\\/ck4oAATKL+ihlRrzTQTJcNC8ghZ3oBvmS\\\\\\\/gIlBe+sXzIRHLXq8cvU1S+XgMocjmA5SBx8OiKglgC7OcrgqQN4koUpN0peELcMkKJkDICAUiUWfZDmqaVz5OD3EpNq5xFOYVZIVPqy3J+vpzgECs9StBUeVqkunWq0FZdqniBP5AwDlTtRZfQTtNffWxCBS7NsXmndpD7NpR0f9vWcXUQgiTllH1T58pgdppq\\\\\\\/XtXUad70Wx31rDUwhw6sPhLCpeYdig0kA6KbD\\\\\\\/996MAYj02KbAtS3\\\\\\\/YlGVfstLY5iUbRNtvdjt+IgQoEzVl9fEjdQBZI05pJWoa\\\\\\\/xsQavC1UCgzs14HFd0Z\\\\\\\/nKXF1JdT9NQwBEiBncrUGrqdvBo+PPvak1BJRmPnTlFS+prJCgAcDRLVfQOrXG9AD8YJ+oc6BiPZJs5t4\\\\\\\/XRsRK0r+1pwW4kXlB8YsGiD2d13uisoIUxorH98u54l3Dsz5LGJE6aj+GdnUAakPeMw4mcbMNEAIMwrHSxT8A3IIuIBvLRWPdcMdxDEtu568igAmQrkLueDPDVR3ck3rDoTx97rEJr5ZiPl2UBI9oBNLsoTZPLgZiiaHfWiNd6gzLn2fKlUmMLp0FrHQkh7mCJSc6NqzwN9iSwp5tygG+uC72cemziWRXglgorL6lseFGjKU97X10XCCsUNsCog3v89GITUbOQCSWj3NErtnn\\\\\\\/\\\\\\\/p8FloFWHENEpsmHSnjZHL\\\\\\\/Dx2uEeiARlfttyQ6UUsobopOZKV35rRVJ2PI+46neZLWu6WxOOXJfndu6qsXha0uftAlYI3Ohtsi57xUM6Dp3BdLergmks5XYO2J\\\\\\\/IfNIHGqquxa8\\\\\\\/CQiW5\\\\\\\/gOOEMFnhOzIfyYZAlu1913HanuREP0BExqmEH1Bi6kAnr51ZkAVERuhv5w0SCVxesnbXKB9WGKlGB\\\\\\\/zPS4TaDJfltSfRdPt3lZ9bkCsgjJOAeNSfUcHjArYKKJWcbSGX4Aw9sBwJMPwX9qUztrhTRyx8767BJbK\\\\\\\/be+gllYmFGs0qrpMz8FgJyLr2wziyG5g0vqk+BP4ICtgV9aeIZjSW+\\\\\\\/2HcLgYy3huvP77GCrAqXwqCSgSHam\\\\\\\/umk1N1itRthDBNS\\\\\\\/IZtQadRlHwOafPNr+YPsAko0+j5EpIfdA0AKxFqXvw0duDQo7IDG0MPzlSWen1LZv2FyqCby4CTkkwoMNJj6VcFAia6QMUUOmMWvKCIyS9cOub2p88Fkx1P4EVkmObxQAlVlM+AX+3doAttbkCPGK7gOblqRs+eGmYZq3P3MMkDoXsqrYHAf+hBY+ccw5I9e9wk943DRA4wCs6Zwpd\\\\\\\/G7EdzZUILEVK2N+pxvscZ17Bl5+iRnHSN9tFcZtnNVuZU3l9M5M05kEYN2\\\\\\\/UfB5yzKNcQcmNnaYIFsZZs3NVdMzpdiXjP26n7LZ4n2HJARc1t0RakVQc8APouBVXWQN6o9CRJAm74lOppEPB8mz21yDi1k8uFCtWb5ao8YGvBuZZVpejVJ\\\\\\\/jZqLb07igpxxGTpMctRzmZ1oagcFKXyDICdDocMPcE8rtgm8UuNxljiz5vhKC5t2sSlRghwUjBtxSDPQzSlY8Au7ubz3b6vi4qkw1ScGq4Bdw2HX4OW7DhBfimiS47TmmofPVn\\\\\\\/KgTnHOsATiq4ZuaCi4F5Q0wxfv0XmvV3BFX+cQp0hY9HrXSJHl8uUVq6JeMEwXKP1qobGHHMh77IR7ESNaMPi32kmuQgyzdZp+4GDcjiitX8SHpd3Vas3fSl5msr3xaUQSm7t6JgU1n7207dzHrsfaFtciKc34WkC76uvyMz2Z8bff4CzTMnWfJt68dY7yMMIdPki7ulCR9FADv8Z3M9A6QKe+S594lLfq9NASQVwTAqBbZEYqCzNZh7dRI9FP238pWnSJE9I1OMJo+Salgn1UH5cb3TWFe3DERHx8VWIvBM83qaDon28oq2Iu4WRbwxo8NhZJN\\\\\\\/jN7+dU8EKhD\\\\\\\/3xUfNeuqtVev8JnZcj4uiFRtD0oBQLxOCY0O\\\\\\\/Gmgu89KF5\\\\\\\/s0IFHRTj4mf1VOYurjshOsGyncrI49QyzWZfVpyjuLnQ4l91d+4llXj7lh\\\\\\\/v2VP\\\\\\\/HSUaV9SgjSwn1ZlRznlzZh+djXNzKdgj\\\\\\\/EJKVJ3h2Lb8O3fNmtelFIL5\\\\\\\/oAqRoE6AIl\\\\\\\/4QLqXgmj+EEsKwPQ1Ic9VK9bVZWktxOfxhUW1M6P7XJk2ajjJ1EuWixIIVjK\\\\\\\/v+6Xe8nfb6XuYPt67HRicvAXwhU7jcJX1tHX0e0w8p2vXfkowk1vaeJ7jqullKE4Z6WdUTec2\\\\\\\/+WyaUKEa72TtPO3pjwGnPp6xMKU2wPwX02tzJPFl5xEL46Iv9gRSEymFDsHPoPZX0q9FFraol53qXb1iZSIxV5UCgSzqOTHd8GtXz+pMb9tH1SGu2\\\\\\\/Jfqs4bDAxwlS+GAhwf5DMNRiXgmmt1Sc2uwvyeH0\\\\\\\/m6Mf+ygxCcamsfROzbF5UhsXG7KNYkI+4yvgdjp1B39ly1s\\\\\\\/S7qgyR0o6StGPfDGqbTN5gXsbFDkuiKmGKA43H2SPrtD39oDkJHyeAnRbHPlXBG8TBl1mURZZ5OLTZkS1mKccQkJgf9TjIOUUGtqLt\\\\\\\/MZh8HsDq5PlwBjVI8PmsGvONO7wcy5c7HJH6EyOS\\\\\\\/Pw4f6zWjjiL7HiPeHqU2ngZurGjQEJc\\\\\\\/OvctqcKIF3hbHuS3HMfIkyf6aW6ujk0bEoO7CnN4C0E8LzWkX57ysluvLRUT5Yp5MhTbZ1MIa+CZ+S2VS45OZlY+ubu\\\\\\\/3oUJASArp46cWzSiI5e9v9ci\\\\\\\/cGSUg3xjy1sEc3rkk8F+bXYL0JTL1LZI0ItsvwILHJKOIJvdLjgEQE9OC25T8avu1klXqyu3MjF03TemxkPyM1IzhN+flVb7JIPs\\\\\\\/zoqJpn7MxYgH7dlz8Fn71+Fe0qbqBCJsngR5VmDEJeGBmfBQKNzxG2oNag1dKx1Lo5NHMg2Xgq\\\\\\\/z9CAqlXP9oesVI+6ZQ\\\\\\\/cE8OazchniJ7umQrpNraF5Kka2FLYlosN6tak0I9qFRR3TBXQ2zPxibrsGcr+NDb\\\\\\\/WHBFvvWpTVqIkdzxXASnS082y0a06H\\\\\\\/hXCbslyGmrOQ1pSI6IRJ6VyzEAH1hAzZfM6PCrL\\\\\\\/6nMnp1IUveYqCMQMAyWspxenwUEc3Mv+u\\\\\\\/Cb6Nv0EFBtHAI9w5nfleMhi3YG\\\\\\\/eJMIo1\\\\\\\/StSBn5X7K4WDxm\\\\\\\/Xr5F0WXru7+EgWO9f0INKVyElgDyY0BY\\\\\\\/JohneE2Zh1SPMHXAFnqckQGqiu9MSG8QtkrmGYYpjTuA3DOTtkBiRGA+Ua6Oketqz2vjEBDb6cURdIuCFnL5itzGDV\\\\\\\/9eGGVnho7dYot3rb0ynf5Vrz6vGg54AAnaYQaWAhyouG6kTnyXeE+rdOvNiA4Hf7mqhI5D9l5EducXAtTALL94pz5J4+CuyGRI7mugKOfn3L2IBB4+dan+KnBlghBdhD00zTHQA1sdFnM1Tbq0shkGOTKlHOB4xqPpfj0xippOjZYUH4qeJ1RM\\\\\\\/qPfBYy\\\\\\\/VM+VscPa7VoV3u6VDRWJ\\\\\\\/iw\\\\\\\/D+7CTXi0iCLMoKfTHROxxg9aucGIBZXVOk9b17KraEAyCuV3Ey00rqr6NM8LpUBC+79AbhlGkrmLgqe1na0WtfI2nNm\\\\\\\/Kezl8VwMQaZxo96Z\\\\\\\/2N\\\\\\\/LgIhCdLdN9yPkREOWbCBSj27wqsZa3Boj7csjZKYl2UaZmVC7UBu3F11PWCOWXFAkrSPHYfFUmbRdtj8jJQSt690ACoK7oPa4r9Z6Hz5lp17egmJjj1o+3j\\\\\\\/gJqafNNtsKmu9ZGWUMBendy\\\\\\\/LNgx75QUvyXC9+QMeewC3Qf1gZKMeYmXVCk7xwxeYxn9kqBqQ2GOG3QQYYmspI3RUMCYSXco4x4gESJ0UH+XhpOYYcW7bh2gM7MfTPltUlzMjiOqby1hKVureksh5d9+t1fsK6EgDgWoSKS6zGlApjr5mB5j9yHNMD7epkPtEehUeTra1AeS1rYqPnrw51TGrpixxSu4Kx4TANFs0NI8HYBWI8l1lSs1IREf7PhQ6+ndKml8Wgjzx2u4Y1Zuvnre3TeVHMXlu\\\\\\\/sphh8tfrMSVuvRyT27ZfcR1GfCW4zlibJtcZLiw63t5HDbU+Q0vsyztIcQ\\\\\\\/aDYgPBw1Yb5w8t0JRz3bCWS2tX4ajo8yfAvGutcPBoU5Tk+t5+auzi8MV99R2JgwcqRFTHwXkve948xFlS66mEGiSnYHO8WHMhmnqNNfRlG\\\\\\\/FjBr04x+Sp3qlQhEh1sINYb8zVGRvKW8v33zt5t7\\\\\\\/38SQDolf5ACl39X\\\\\\\/7CWtzxoYNEDeZjrW\\\\\\\/8Ft0DEZAtoXmETkiNOBLhYGTgfmatfOd8DG4nu\\\\\\\/vI\\\\\\\/5+5cMYrXh8\\\\\\\/\\\\\\\/fPHqWVC9USJiNEhKAnvrB5el1DT4VY1CT+8G8UmhB\\\\\\\/eVR\\\\\\\/X6lvZQ3dcNo6s88LR7aWRQCD\\\\\\\/XrxERxLnFuJ8uI2BsVQxL87QlQzK9hhDMn2Jc4zjfg4xB30Ay241O2e1oG5mKTqPZDWYPYK2hcKTe\\\\\\\/Hbs6Yv7RrwStSJNalAHrHivlQEf879IyaPu\\\\\\\/72Siwn3Mu4JxInOtzttGklGnYoMhZu49MrcxRi3OjSHQVSLjMppD+9RZ2sfDwCVjLmkg7ky5GgkMnJi6Ahl5uvOxgZA9FbyIQA51DlJRR5wp\\\\\\\/Bujwkpaljduukb34pjg3xYvEvRi4JzAudK63MmxBtZMRvzSkwPTvyts2DWPc5B7Q3tp4o5QmLfeFJTpb68EXbjklFcX58gpsZGc9+4g2+G0LQ3aZKoJX83bwij7U26vvBnjWtoWguHZmWyQEv+0BtJ9GvKZwTIGVtXlwjB58C83\\\\\\\/jeoXKYZHeUto6ZMMoRqKTXIelQSloygKaCDIM7C7H3IMRUUMuVRNqyljesXxJ6LQqGn\\\\\\\/\\\\\\\/1ITjqLM0bxlnvJ9i0aCM16S5kyB+lA8p6Sx6PRCjrII6JyVq5vLSmNWxSkpU+uG1FjkOTbh96Fj05nKRm+5+kJ7FSPt2n+nTBE+bVE9\\\\\\\/tvDyxH5rtl5kEpZoeBu6+i8Gdx1kMUi1ScUjurUsnMMFSHcsbjIOC7yXuuxhazmFBm6HoUowufAiCt3VMu4hLBC1v\\\\\\\/Fks5jOeckF0ZkHsqFDuJc8smfUNazZBligrPPnw\\\\\\\/whovZyix4tYYwjQzL7vff6fq56rCkMKJZJNBq6J34UBG9p1UttuvxAsSWKmtUGeMFfIvK4uwBL0VcWHw7UrSXqmqbe7c6oKaC0UB66f4HJk0YAb\\\\\\\/IjJeckTyPDu56kfgTsbyE2NS+erA2Myr7JoU9h4NWZBIAWu1FoYNkFM2bQ3J0Q8\\\\\\\/x1yyTaK6ZjcJaSUmpw6YM0FiiPjZFjW3PCKEii\\\\\\\/oHGt+DAI8vyq193QlASWLiWkBd3PKeRt+iLuHF5K4orDJ\\\\\\\/Y76vgdCXBtLlwliwzzUEn7EhyXJNkSOhkb53lzl1jhgFH4jkO\\\\\\\/hpC9H21NJrcZ+bm\\\\\\\/2cUL7SS5o\\\\\\\/T6+gwYev1ZSgC6HPYUcwuOskwzFrp4y5g3ufJzGHn8E3XsKImu93tbUuLlwlbnB2hYA95jjmGyoWnhBr3IG2Jr8hRTsscX4UMuulO5tqQ4ngIgbDQ9uckoenTOTx9CDrCwjXjgy5Vm5L0eRG+SvKkADP9wSq9gEbVkJeJvGsssvS\\\\\\\/DXSz8\\\\\\\/ol0X3QzDMSNetl5+pxmwq7oQMxYPNRhEyD+\\\\\\\/JsNYK337y6gsx4AzNvPR2WR0y4O0QN9OEBo6qmPPL4GlB0DKp\\\\\\\/vYFxmZPPBITgNhopm7q1nKqWSeK9FhKSSg0RPizIO9X2SwJn1mjETLYoXj+LXmAIIF5Oqzt6e6CYHZvlPpWqY10DpUKIukpoy0NSHqV4vC4upV6u1kW5J431aKXKi+44zpTQNvRNjyUup7dT9eIi3v+d\\\\\\\/QRqohQVNDq1mxuV6Kzy4KhnOXxv2T\\\\\\\/bNvZ1k4XHfo4XSN\\\\\\\/HfzseA4NJJ\\\\\\\/VVj8xh1eAETkcY2KZvfQ7kd0lUUd7Ep+LwE1BH\\\\\\\/v6kmxzVsjFe7+n1n9WZjiFXb1v2Zfuhh7xaZceYENJTO1YxqQncEwmZFWcr4P7ufgHdpG15apiwgcGGw0rfcN3uSm9g+7EA91r4aggIZ5FNcsy9YRKLIr3wGHvgrIVQrJDvXfosQLEW+KoAmwwoxkpr4SxI0e\\\\\\\/2WmQ88QNO01VqLWniJ06mokRlW6m4iJNz6dKS0KJkp4Atl\\\\\\\/NlZTRpuPoGEiJ4GOzZHRf84hHp9fLib3mFrawSup6PHUHM3ROgt\\\\\\\/h9s+ZsO043Vthdg+nfs5eaNrkQl4VkuNUMykAa1hWpkNdkjh74FAPGUg3Ztvijb6sAoackYVBbcwcxeL32rf1LJcrF7ibKpP9lihsLWQSBo+sZGGYqmzW7hfU63tRkfAaX6\\\\\\\/qTDg4qF001vPGQsPJGmTDBDVDCqDZxc0cXkKSEnVhtm+0GHq2ScLi3Q9rI8XpO\\\\\\\/XBHSJvUA9bHU0Jb7n9stMfgZ0mrwTvAWw0vJi\\\\\\\/Th0tJJ4yg1kjWmuK\\\\\\\/88sbZe\\\\\\\/GlOWKWVZbLoiCp8MagNIN3EXndU98SoaATGi7DZCiSos\\\\\\\/pHO9SgQ\\\\\\\/KmU6jOfbd\\\\\\\/wGCd\\\\\\\/u1CtJJwi9lYUlzhzdJmygvXP52vuga8QMY7mj5X3bGsSL6G41sZ\\\\\\\/kwa0BpkdssFCPnPcvWr7bH61SRRktDdf8Ez9EGP84isXdorHmnUrEhNiu3ZF05I+fsPc5WR0l3+IESg1F6gPsqWzypcmGI3q7d75g036gDICzET7vk2WPjF5HO3RtBcqU\\\\\\\/pPOMjLosRfoLfljZWTrctX9PdF7RPsHmhVcTVejQyUkzQ11NEJSRMWgqLLdWF8CWrvNNjytmU26OOIuCpzmLw3HdmXe9iQHi2o\\\\\\\/E8hLPk8OPiAoyIz0D\\\\\\\/0La+igWmvG9ijIPkXpQkmVGGdxQHLMG9+KKg7sLXtyi2yYXge7223HBNxglAQqpzYQQzGMU87ckkn9gLWqZApnln9kPkskzyu0PQj05wrB0j4uhMUkHN0C4pFdPbYFqQ7o\\\\\\\/rtXdbw\\\\\\\/awDspzAtRgZ999hD8YVCgr7FSBe\\\\\\\/EMhRDAXQnjDUb9vNTFhJUHX7Hbt33Yf0\\\\\\\/d4lDVYRRvYQ3RLlCyg2acKnEaLZVFmGbdOJXf8T9FvX+oRKyNlSy1cHlQ2RQ5teVchIn653iZLWZBhqropj1nVvrHnoY87\\\\\\\/6\\\\\\\/1tenz5HCPa\\\\\\\/gf+x2X7nk7iKZBsacoTKypBSBONtnHxLLnDmMCoM0vU3QtSmR3XJEhnjkdhVAAEd4yNxFWEgDOmfK9aViBgh98+ToBUCCs59TMfY5PoHzWdJDI1x3iW12v2vzpf22cRvAhyK2U6N0WwIOYMNhxC9zXuCIC0KQ6kjNur1jR8\\\\\\\/v\\\\\\\/UBJ1Yd+GA0Mgg3w2bE2i1T2AT0S6PhtxFHVK5ArqNzy5HlOt2qczn1ns8\\\\\\\/S4dY6kwlkucjoDC2e9B\\\\\\\/2k46gHlgl9Hizm8gSayKJV8\\\\\\\/2TVk3uo38rd8ec4uNbihLktdM4Ab042KmyLhmLCl0CuH47UnYfE4hywFKIrcTI8JT7xoU1JoU7otrcbNQeY2iUdcX3jlALRbMYKNhdQfv6AEgtK292NHTvrKH7tXOtGyk5p640zr\\\\\\\/CzuzNDdUsRZ7d\\\\\\\/qjomzxOft6TuHCiayNATbbg1F0p\\\\\\\/wPVK\\\\\\\/2dpX32U\\\\\\\/H8nbEgQVKyWHfx+xU+PbhQw\\\\\\\/zr7d81EJ+Okz6Bc9ljrPFmUFQzx006+w12W21f8jd5bEv6eZohKvEqlG5xlixjikcsAXqVoPSyK7\\\\\\\/GjF2ZYuBz14ep7IgEbqr30rRdjDlqV03WooFvjtNubxdP2ccBz8Almsh6nK+P27aitMhO\\\\\\\/6Kum07Hx6KE4gODrSfdNMa0QMtTMrigwnHrkC9vIWGCRpOKRVYutlRsx2GCzfC77nOAhxlHWliP6pBFyI0BZa47tco+Up3g+ORacYhvbMZsBCsj4oILUxbPT4\\\\\\\/QAaJoVcg5oBnIePVkiATI4SS1y7xkgbMfvi+RKn3VjIXnbnr0p03uj7++RSE5yCH9nyimMLGcKbLHIyB6T0EwjRPA6NSc\\\\\\\/5vhFm8cafsjvVTnlUoKtA\\\\\\\/6kckk5YR6LopPckCcu+Wps8M+TDV1LTzceA64Ca89ScgphJtpduR4LzN0eqRHT\\\\\\\/8jeHaNzSXnTv8dDGpJuWpVarm\\\\\\\/cyb786fhcoDnu9oJ3xlhfF9fRkogZEZg6fAScSk6h5Bp\\\\\\\/oQS0w1DrnkMndQlAZ0rh0Bw5HehBmElsS6vff8YSxzR4OBmWGnllhYmGdxjiNOuNQop518B+bmaW\\\\\\\/0FmQhgkSvdVM5BMgt4pNdOJmedGGyZqPQMccTi2DQ3+B6lDCdI1a6+KN2LCw3qqT+uX7PMl6D7pkFYZIykpFmO5+HfAUh15\\\\\\\/gDaQfEi24drYu4PPYUIKiTT9ZIPtzcXA2C5w1aK8At7S4JsBtedr7iBGNYQtYYMHQmdxslaFSMLFcWfko6qu3KomfGuau9ByKtPyDi1aWmsxUBLGYYYq20ZpT3S0w+cOZTKOzsFsQ8PgPn4eMGafwRqevcCNflhGdxQTt9S6NqL9JwaPInaFoHgJW7kSDnm1oVz5XEu0rH7+\\\\\\\/EI4JtF9aJ+BEmPIb\\\\\\\/SRQui5U8ymG5GnvwMbi+6zq2bvxV0U1d\\\\\\\/8+U2vJOkwuI+ImMazMMIY+9ehbusT5Eoh8XM9WDfSjD2lPSk52oyyuePxASSn5fp3Ivw+qr7ncyg96OvGLd1PmuJArfVE8zK4k4QqSUH7sp8FP6ymiOgovH9QkXf9LJxsy\\\\\\\/xBOke36oa8AkPmFobkahHXxC5DuwLpYolBDiZMII0Ih3o2TaU7oOfWohuOSlU4N6Kr6K6ruJVz1m2m5tiVkZNfgAVZymvNAsjj8mPmFaUPLSLX8I900\\\\\\\/tigCmYx+XMxfpcPSHVggjGKSzJr+uUKTbynqakcany1voWRQ2r0rvJySvT0g2yuVaHjHHXDJRAk\\\\\\\/dz0YsY3NiqzJiNwolr+LVEI420fSBF7E2vCZlWw14pBQnygZMyCK6cZizAudcsxQwLXyO3XJ37iwxGoDmBgTZubLI00E6WHV3U8JYPAB03NXzmWRIg\\\\\\\/QwuKnHbygJteHzNY4Ly7tTrkF9JMO9SM1T1hhwaVu07GDZmLjwEUiMbd\\\\\\\/yVki3VJmDUpCE6KhCIt2J9FXgwnThbYyHsrKnb23alBxyyunadpSTzArSHtQog\\\\\\\/Q1h3TdkxNS4eo7JE431Cypw6POogY96nnkDeD6CC1V\\\\\\\/wjueZBppEwJU7X5XOfLRqC5KlpCey7bxteeMP6poywh6217N6K7w3Vu4T8ZnkU0QV6CMRPRA90f40tuDyBdQCKdz3zIkFxArXxq8tABk9WJql9XOSQlDKv3uKh6qpuosU8aeVc2uzf\\\\\\\/GtJvsnFj+jAjbMCNyAdqEAgYtqjEbfKQFWHO4a5ynw9I5COJZiSU7srobf776+NEE7JmNA533DRhhizJl\\\\\\\/1iNatmTj7vlUU1zzK4bEL8\\\\\\\/\\\\\\\/a4tAG8osEBSGHF7mccsKFO2sljbhKUeRlYeQb4dILtfX9u+8I0TMzjkqn8FtS5JEXa9W2VdX5rMRYXx1bWRjWmaNgOvAxS02YIIVb+lpJtxzW43YcvZE7dLgzmevmBoy3BHDdX\\\\\\\/DjUMhTr72tKOm+4emxuJI9DUpVtV2KPz9NjDxqJm78JlVt7zt74gpMOY+kdhAL8oRWj6vmPO3Vj8URLQrKOdICn7JB9\\\\\\\/tS6Jw3OkjzAoy3rEXWD7AvYWzkfBDwbhtiqyvOAiKubV947iKAVHuTtwnSW2YgQUNJXyPGh5+rGmAASNJncpF7nwrBqNVlF+bjauBOqbZvhl+nKWVTVvwh+XvF9JXDsGkGKn1O3emM58CO3I4YrnOPUb5BY4UiOR4RFeivERoml6GuUe6O5irYFVrRWsiARqBhdrxZGwEWCMLXoCFvn7ci35aWw3JF16U+eyUyQHPHB0egzM+gvPpDzDBo6kruKBPSYVbj5\\\\\\\/BrGg5ixoFwkJCXVHNmF0Ps4JRedi2i3M3KW9ptc\\\\\\\/pk76eZqFFWR3GFskMiRiP4JlMp48cAcQCoKVyuaQZEfN6EHOYltJr0g7Nmyv\\\\\\\/mOuGy1GIpZ980nGQknLnfzDYn159ZUjPTceClli1wjmC7KA9jFwXaoe2NP8hcrfPoLMLyo71oqwM1u0N+cQF\\\\\\\/+x2JqwmNFBe+EJXCq0X\\\\\\\/vnX5JAi+Bqfd1q4E69MPCc7WdKUnOh\\\\\\\/E1xdSLKakPfOJNpKg2Q84wtVujPFnu3QdsN1XYhEY+W9+eTaQ1SmbWMfJUGCu6mc3HBYMx0vNvc9Y1EdNWWti2YVK9bxKG\\\\\\\/KCbkEE4haNhb2X4nPtoyVsfH5VkcTs0PDGWlxhFo8k5+Fe6SGCZrRauLyuPF+npVAVN3sHhBJHrKurlgQEmqa+6LPxYmM1L8Pml+owZSpTsRQj9GdTy0PHde8n0bNPZK3abSxkRLhjyPCyCBQ18WaNba2ScRLf9ZhQ1Wl13IlPwleZ7mnDVwkzvyezAy23Ru3r3fdxd5Pa3VLTt+j7ggAEO7+kO1ZYCnzHiAWrOwrpnPSIDTfu8nkRqvdS96jKIRmd5RhMwO+Q2WSm7hgm+PKPcVkRVK\\\\\\\/6X0lEPs4j9rlPdi6vmbN\\\\\\\/v0Hxztxl\\\\\\\/sjbkoL3aKiUhAzRZ21Lev5xWQ+DTnkhm3TusytjKESZxyY53pQCZxT\\\\\\\/W1G3aIE8E7lkx30jS4iUXBzCshT7ThAdKhCP5RWTanEcyGVFupTaN23EsDl7XQcVsZ8RVbr3Cm2cYH3rhAGpBu+vFm2u0eLxSpfXDkFw64glhWJ3qK+HyqHp8+lAbZ65riS6JSNI+4DINT8n3v\\\\\\\/E3Us2+PRMSmY9u5u6MrsUy1kjfRW\\\\\\\/iTN8Owp7ph6Ns3wIup5w\\\\\\\/qhJ6RCM\\\\\\\/ABZIzNdDjKJPXms682SDjiajQ3uLsCDVPN8fQ8SBDT5FLqixIlpbUVg2ZrWEZrG3FrplLtgqtiGMByyWUgcr2ynBRbIPliwzKNzKEZoQp7LHw66aGk6bzHvOfUTmVfGsKVKr5sqPqlK+7urx1S6zpnAQ4xwWbJeA4WYjIrO\\\\\\\/GmlNke0cFq1v8njzMTItdZG4xAOA0XumQJglADUm0W0mPlNYh4gNtZ6dYnEO0600Df84tl38XVr8TDqGfrTfagIUBBoE9S2km32W2NBi+6iwvCUIyQG+18SxIQWZlvfMdZ5H7uilGEz6bNca\\\\\\\/iDp0yIp86SN7zA5rGbk8pIJ4ab\\\\\\\/ASwrAP\\\\\\\/CRbRim\\\\\\\/mOB0vd04LfXOOJWXe70sH\\\\\\\/BcAgHoWHQQ07qVEeFKNiVPflergtpMhxq0rYrIfD0tVi8+pfINqAxHgqsOTi5DMVIpDlYn2OK0yvBEzeOQUjSMOqW9CslPCLOfeem6Fr6O3Y1ls+AJ1lgN9zVHgBQUB\\\\\\\/D9IhDpiG1DMrQykfVvbmD+xp2gizEHa82JV8Td\\\\\\\/8lYrQ7Gbo4xIVWAwXA61xF5DqErM1EYxYJySN7nfekT659pH7V9aEoZ4Xcy2YzjowV3SHUbAqk0hhIl2\\\\\\\/+SLTqfCzViMkaD2ppb4DnVpOEeRDXy45YMfVsffg64HyoF9JfO8XQ\\\\\\\/Us\\\\\\\/AEcmByeuwsqp8MpW2QolidE7NTq9odfXAlXH0GRW+V5r54xg2p9rTFTl7wWCyUKZCoGVWNXVO3p3xvxz2YNMZbyZmwhZ6hjo7wiz9PaDxbLD9JF2XwoWNoCLZ02ZishwPGe7OUGuts+kUNNbvp27Vd2xRCZWBAAjmPshbR+sWtYlbUlgZmCy3cKeSb7ER7nw+SAEjcYv08LCKlIYTnScT3BXc9PoIT2IvR\\\\\\\/qJ+LitO\\\\\\\/m4v6PiD8bm35mtSa+rTAmh2Ttauv3ZSmQrYqwWmtMjyxQLvGWWSnV5xDZgnDhYVBBUc19eXBg6q45EzlAJ3Mlws49lOeI9KI\\\\\\\/DE1TM\\\\\\\/88Vn41DHd3k4X9fYUDa+DMUoIUR8h+ojBAm0RcjYZUeOTLBVea1lf6+XQEN+PdZsB8jqN\\\\\\\/9PWuo8K\\\\\\\/sOH0KBNjdyuXXrM3SziTPB1oO5j6RwaR7S5BblyvuBI7NqswZeI5l1ujMoN91cIp5fYbUhAdxmNasGFfn3nJTZ5E3Kl35dR0i8DrtN7bBahQ+luYm903QplxXgc7ReS8PQl5bYnrgIKPe8rSmZXmBqRiqILOnlROJeeMQ2idQA0R+7TK9ZgXQ1HrL4fxyS9xdcsHZXiPc+4ECI+7ONlE2+92s5JUMabUaqZA\\\\\\\/\\\\\\\/uYqncH9FZKeHBQeQjb99Ef+yJol0tQV1qmsUh9Ftz+UWMxEsv5QCW\\\\\\\/HCIWNy16qxwGaiRsSiZhXt+bTsDq+qfWiDQFyX6hkf\\\\\\\/ex4gpvr1LmQZZgzqTddLNBCTQBtb5F+kcXi2wnuCuC57fLacrIjSKwYt82EnRuMOgqvAdLi7YdVJJpaG9IqCJUQmHOoPLCzjShdmHOb2gAZ+lxzNa8eJnO5WRRx\\\\\\\/oukjduELk+nQmOsBio1DxC\\\\\\\/d3boLmPTjyT0wEy6BCOQfTndijH+qffGRzO5FoWtnSkxR6IM\\\\\\\/0SyWIFhcFh9hRm1TcogUJYD7+OUR5HW7YaT6Hqh1T2nrLODSu9rtF80z+bA5j0a6Ec7DOPKz1p8MmSLSxipD20x8rxsEFDiPafVE\\\\\\\/+3avNzdnExv32Gfq1eWf8S\\\\\\\/sk1fse5AMcejpf73xBOdQWs2MNtV8TD6xPTm\\\\\\\/upLxpmdoyPzICkY6piVBqsGkBtVUxRKrWNDQ+7HOvXiYbdO40qko+\\\\\\\/4kPHj+80AuWuuF6WzYwysriSr8qDMl5uXTn942NZumSlGO3En6zVHFXEI00NB5lueqwG0PhwfjrjxWZqpGYfI6z57lg0R+E0AvEs2FDjGZ1+TqtzQf1t6vL6w6Jgjfp2yNsBj1l7qCsln8aFzmd4vuBYXZmVxpzZjx+KS7zXrNWl4Pgl3+D8iMwIWthlEyKCTJEUP\\\\\\\/4CB2xnrbxUiKV2rnaA1L4QNAhqJLbDCje5H77tJXrR8HRCBP9ZjivRPwd5bwBzNTA\\\\\\\/hoh1+OdiLQDqo9eS8cWq6WhlmV\\\\\\\/O6Tg1P\\\\\\\/6+3fTJ6rVZSaWqOlClM9muzBC\\\\\\\/u7XpRByNfaj8yC57oiaa0gEyLFvFmvFSph+gsFCDXcOzMZ4lQzTN4wL1YG+sLsu9s9BGVrc2ODpJ+ww1KDcCW1fXXw0ZssWtBRBwHkGZ6eu+sKDX6urlXVbVYnubBrTVPr0H9TZLANOs+Dr8VsgjC4cuh5PQWcjwIk0\\\\\\\/wjuqvQ43QPcZZFkjDFIp\\\\\\\/c2AaIJL1dZxgm4uKhnJP+UeFy4QI9g+G\\\\\\\/8Xac2jZtwb2W7ROqVfLwXXOB0+ToBD7RrvyCWBFZEHMB4gHYr8D5dAcddb\\\\\\\/v0ME7zP8yx2tOMygQQMcdamKj+J1LYtmLDHtlifCLAuAMTZelT1nKahf2uCoTzSJweHq8EG49T0OV2tdMA5Ys7VNMtRMJDBU7Z9xQYLnaWewPBUZSg6FsOfBkwihJF9X5LiFziy5pOvitOu+WvWxw7J3YBNuU5rYVxSE3HPgbs+vKzO8rj+hdBdw7WNsDKOijkZ1adY\\\\\\\/vPeQg7Z\\\\\\\/i2Qlx149mhvSWELBD8pbF+FORw1AhHwK8Y3C\\\\\\\/cfhWOwTFrT2Wq4b6Re7+zIR\\\\\\\/9CPi8HtZrMiBNL7u5CYV8lCNkPUcsShJzJS7CSvbjxkCdt6JuVUJpFLC2Ou1ZPZbTQad76NWQ6udwQTDnBMHwFCOfUK543S2kegzDB5t82B++u3AOtLqz7K\\\\\\\/bf+xcNbY4ILW3USBufdauHyIh74hkUAO66oQZq4qU7j0+ta01ihxLJU7CpfPejsHty7wUAELxXNoomPkYUBR7Mpq7qNMRUbG2ESSo2BNEom33BINxtjSsn3R2u7QsaR7xRhVYCha3dJ\\\\\\\/qFvcaim6hJfOYijfszc3oa9piQnXnhhfgqnwzqPzq0lAMyqCVmP2O\\\\\\\/bOTGU8FQvW1BPN5Kvw7RuyBr63uYG06\\\\\\\/d58+h+FUS0EB9\\\\\\\/NyrFbPLvFHwTu\\\\\\\/ZEMwW4BE8erWYEB4XxHBVQwee+ah7KogunCImCmRhmf++sXMpmXDxtCGQ1Hb8tyuK0qcLv6oIEJ0nBb4YfDeLtGjcPotZnGKGk6XApho67YN\\\\\\\/i22kXiZZQ3Joa4\\\\\\\/pIe2Y\\\\\\\/sEb7f7edsgunOw16l3ksK3J9+hTPP+hX0tsfRR3U+XfkrAPUjBsXBoOTabxbSH5nYA\\\\\\\/dUyNo9EiyRUoI+vDee8qhmUYKIgtCGs\\\\\\\/C5aGtJnsmk8GvTxuM3762BhFcvdor+dtmXangkPiVjqrUWYzuJccJfZJ8S8Gu2lvnsoA6qNZvMZF2LipMQ\\\\\\\/96klg+WmahApj8K7rbxlmRbySD39+8+0pQQvFtlgW6p4RMOgADAuwxm7od+NWOL50a2R1KUwa0vjGzoFNRXyaR0vQS\\\\\\\/hy3n5etaWODJzcM1p0wPbIIGPHnkA3v08Sy1fX+OjPvBQom2DY4KGfR6bC64ilsCLfDpvUsWM+AyGotBbcwuzcWNewoGy4P0fVg29C\\\\\\\/zvfG4lqp5tSv0rcg7i1AKdXXD6GQBolZ8LeoQNToJCLb5aK3fAtTLMYjjluGxiUpiGEjhWhiNXhtZ1\\\\\\\/bSfZlOVDopECHD8jCc+yhAvejdOyPA8era6lnNIdug1i7O\\\\\\\/xsLXIU+Q\\\u0022,\\\u0022iv\\\u0022:\\\u0022tbZBI\\\\\\\/1CAOirwMU6B6LQzg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223247323bbf18807d910ae9e8ffdf3a3ae2b9429b3c41dddf45ebe8314546e421\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LRsa\\\\\\\/ULQqASmQjiOc24WNF1DvZIrK82wuII8ubQWDQos3qscFJiNfEV\\\\\\\/3jEMXLFIJnKmWqGRO7xW4gjguYhFe54yhBIVvcE1y8OetEDfgVhp3iyJtWGaaYgSpEMm6PQMXtkg3BFStwFL2vbCn1n0BmO962stRqb5tbGgKdMSV+uGm87eG7svNFaLyz8o3X6yJm0sexBuy1Ct2HzzKmxquhQFXT+z41TTR\\\\\\\/17mhzcnRRfuE+\\\\\\\/gcH1fk+1vbWxBUAknH6H+CcJCKOJneoqJTxORsPoqv4idVHpjvTVuQVpTLc0Ese9V22mpLp2hFG9x2of3kJoRLhi+xH\\\\\\\/wG3tgufmyffwTp9jbIyenQLDkurd6uUUlbrv4isJnDgyX8ZyjRP7DbJxW8blsob4Fnsb\\\\\\\/6cuU\\\\\\\/NChdds2bQL6JQb7oLRWjvfhmgiWH4PxXePs8r0Ve00xYpSKdWLzohjsxJNDZyU1ZwevNI\\\\\\\/7jfXPJGBkZyKvuxwjyGid9AaMe+wgNajzhF5uemMh3HWzIENEpFYFKLsjRDm636jFBUfD6Pheh4xVPQ9dYPZOyVEP2Fl40LAzouRvyYQ524T7yIIKVMvx7lzK\\\\\\\/ZeDh4hgAlf5645ElwIT3Ve+IRtA5HXtjFd8I9A+Dxlzy8qDMc0xss+lnfWZIH5OqUdNYf7PaAOrn9ZgevTtDUKBcPXknv7LYdoHQjcafRRhFZN7p1\\\\\\\/\\\\\\\/pnyibGpL5VnGFAYW4Gwm\\\\\\\/EYbuD7RdJodPmX4pasRTYfPMxOQItvMX0eKOMmrRlRsjQiNa3ZvKxDlo\\\\\\\/rZn04\\\\\\\/WQg7UC0ZZ4jGqPd9UKGcKdlZcw0RMKAa0bSRKPrqfjvCllFv9QgZqz7ErvSRs3iRijZLpniYHFqmk67yA8y1h9dSbKJ0NhjGshpGvXz08oZCDFbcKY3TYcZH3xSGGkT90X\\\\\\\/tHciHfOApDv3wJZHddR95RP+inkYu\\\\\\\/nrRrpLN1E6Iktji5iWVG9p5UupocSQ7UDQyYLMXwC\\\\\\\/bOS70c4UPTPNCiaVez2xdp4jzgZ+2SkgfeAaCv2jSCgT3qJ+qZr3k3X9tq+B6taUEoObjjxsuwhnfMEkv6pa8F\\\\\\\/lTT172Olvs1PACsvPHezh0PAq1Vw8V6i46OkGhDS6kNlSvuyMaiRMk0MOiQSzh8tZ4EuIU5JNqtYFJmzKfpE97ZcRjTBJ9lHOE8m++0fYqlQIiftd793vCT+Umlgx+ui09T\\\\\\\/ncXvrxXiEWI0lHCLkJ6UpDUKusGDLTlGmhtjoPjhJ3kV8tYNy\\\\\\\/0OMLGXZq4CjnTluSylhel8Whk4\\\\\\\/q5mFPfBivq4wmBLwu2+OGa8qJomL5gy8uIH0VFbb7JubKTjV1rUd9sxggAoBE3qxUcd7kr47MuMfZQKvW9AX0+97QYTwb44Sl5+lBfQaQGQs3ikMHxIfIpxbjImmu4VGJ7\\\\\\\/6uTSECyOBgwkCo55SFhJfwo+8egCm9VVM3h6DCdZ2IhFJnIVsZSG+RRgmqeEPP02w2nGdGuM6V+b6NdXA5Oj726IFCUlkEYckXd8E8cA1Bco\\\\\\\/PPWut+Z8UBj8eN\\\\\\\/oCJt6R3KhYXOvYXjXSAAwoF4GITu43ek0uBPLMfBWN4W6o6sC5CXyLptLOKDzkLvNvEnmURmB2momfMS7tSyUyG\\\\\\\/Sroh42is\\\\\\\/CZIlKw6xJsIDHO6ewrW9C9xN8W7cC8iCGH3zayQPuaseP4yO6lgRisbbO5uxZwk6sFVHTYsBIPTy3IQjmkFqErWhAU6AGejYoks94\\\\\\\/vkqUn4Gp8tRNRMLMpfbnZUs6YoYN8fAKJqIS1AvpCi9+wLoCUK7nbrhbZX61vCw7zPSAgWy3uTvv+V8Ps2K9SzqkI0lODJCPPTHDBJefVKM90qlEno6fYWm3Qp92uv5bIWy9Vx+30598gg5wLnd+3qKTp+uRZu61RZCrGPGX+EnZR+9cw1bvFYglByITwMalURPy4BHuy9RBuMOvKodYg4t+DKlWAFt4FF8Y+IjMU57QI1uEfpNRNNXBk5yIGM5DH7JIuJH4fBG+3LByOGGf4=\\\u0022,\\\u0022iv\\\u0022:\\\u0022trQ4Y4UaL1rByU1clCcs7A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227efe729f7ecae562464f9c08991f93a56dae3e69f2af76fefdba4aa68e223c73\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460077 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/3/669" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-VTfANIOZcZsppMirTGtGysxqPtSSJFvtV3Nzc1di6Ls" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/3/669" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-V5ZcRbUwnZZQuvRH5VkwGjuYCK-Gryg8Np-vDdm_VFo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/3/653" title="Combinatorial Cis-regulation in Saccharomyces Species" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/3/683" title="Diverse Hormone Response Networks in 41 Independent Drosophila Cell Lines" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460077" id="node460077" data-pisa="ggg;6/3/669" data-pisa-master="ggg;g3.115.024380" data-apath="/ggg/6/3/669.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Computational Approach to Estimating Nondisjunction Frequency in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Daniel B. Chu</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-0511-0084" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sean M. Burgess</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">669-682;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.024380 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460077" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel B. Chu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Molecular and Cellular Biology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BB.%2BChu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chu%20DB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BB.%2BChu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sean M. Burgess </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Molecular and Cellular Biology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sean%2BM.%2BBurgess%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgess%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASean%2BM.%2BBurgess%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0511-0084" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sean M. Burgess</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:smburgess@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">smburgess@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/3/669" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460077" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460077/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/3/669.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460077" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460077/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/3/669.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460077" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460077/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/3/669.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460077" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460077/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/3/669.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460077" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460077/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Errors segregating homologous chromosomes during meiosis result in aneuploid gametes and are the largest contributing factor to birth defects and spontaneous abortions in humans. <em>Saccharomyces cerevisiae</em> has long served as a model organism for studying the gene network supporting normal chromosome segregation. Measuring homolog nondisjunction frequencies is laborious, and involves dissecting thousands of tetrads to detect missegregation of individually marked chromosomes. Here we describe a computational method (TetFit) to estimate the relative contributions of meiosis I nondisjunction and random-spore death to spore inviability in wild type and mutant strains. These values are based on finding the best-fit distribution of 4, 3, 2, 1, and 0 viable-spore tetrads to an observed distribution. Using TetFit, we found that meiosis I nondisjunction is an intrinsic component of spore inviability in wild-type strains. We show proof-of-principle that the calculated average meiosis I nondisjunction frequency determined by TetFit closely matches empirically determined values in mutant strains. Using these published data sets, TetFit uncovered two classes of mutants: Class A mutants skew toward increased nondisjunction death, and include those with known defects in establishing pairing, recombination, and/or synapsis of homologous chromosomes. Class B mutants skew toward random spore death, and include those with defects in sister-chromatid cohesion and centromere function. Epistasis analysis using TetFit is facilitated by the low numbers of tetrads (as few as 200) required to compare the contributions to spore death in different mutant backgrounds. TetFit analysis does not require any special strain construction, and can be applied to previously observed tetrad distributions.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/meiosis" class="hw-term hw-article-keyword hw-article-keyword-meiosis" rel="nofollow">meiosis</a></li><li class="kwd"><a href="/keyword/nondisjunction" class="hw-term hw-article-keyword hw-article-keyword-nondisjunction" rel="nofollow">nondisjunction</a></li><li class="kwd"><a href="/keyword/precocious-sister-chromatid-separation-0" class="hw-term hw-article-keyword hw-article-keyword-precocious-sister-chromatid-separation" rel="nofollow">precocious sister chromatid separation</a></li><li class="kwd"><a href="/keyword/budding-yeast" class="hw-term hw-article-keyword hw-article-keyword-budding-yeast" rel="nofollow">budding yeast</a></li><li class="kwd"><a href="/keyword/tetrad-analysis" class="hw-term hw-article-keyword hw-article-keyword-tetrad-analysis" rel="nofollow">tetrad analysis</a></li></ul><p id="p-3">Meiosis is an integral developmental program required for sexual reproduction in eukaryotes (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Petronczki <em>et al.</em> 2003</a>). Through two rounds of chromosome segregation, the DNA content of parent diploid cells (2<em>n</em>) is reduced to form haploid gametes (1<em>n</em>). Homologous chromosomes separate from one another in the first meiotic division that follows meiosis I prophase (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Proper separation requires a series of dynamic chromosome events that physically tether homologous chromosomes together to form a bivalent. Errors in chromosome segregation are the leading genetic cause of birth defects in humans (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Hassold and Hunt 2001</a>). Many instances of nondisjunction (ND) can be traced to defects occurring during meiosis I prophase. Failure to properly separate homologs during meiosis I anaphase can result in chromosome aneuploidy in the fertilized zygote, often resulting in miscarriage or still birth (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Hassold and Hunt 2001</a>). Notable examples of viable aneuploidy are trisomy 21, which is the cause of Down syndrome, and XXY, which is the cause of Klinefelter syndrome.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/669/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Schematic of different forms of nondisjunction. Newly replicated chromosomes (right) proceed to two sequential rounds of chromosome separation. The cell has two pairs of homologs, a long pair (red and blue), and a short pair (green and purple). Each homolog contains two sister chromatids represented by single lines. Every chromosome is essential, and the absence of either a long or short chromosome results in a dead spore. A spore with a single long and short chromosome is considered live and normal. A spore with either two long chromosomes and one short chromosome, or one long chromosome and two short chromosomes is considered a live disomes. The live disomes have a high probability of living but may die at a low frequency due to aneuploidy induced death (ANID, see Materials and Methods). (A, Normal meiosis) The long and short homologous chromosomes each undergo crossing over to ensure their proper segregation at MI. Following MII, each spore receives a long and a short chromosome resulting in four live spores. (B, Top; single MI nondisjunction) The long pair of chromosomes fails to form a crossover, resulting in an MI nondisjunction event where both chromosomes segregate to the same pole. Following MII, two spores are disomic for the long chromosome, and, in most cases produce live spores. The two spores missing a copy of the long chromosome die. (Bottom; MI double-nondisjunction) Both the long and short chromosomes fail to form a crossover, leading to a double nondisjunction where each pair of homologs separate to opposite poles. The result is four dead spores with two spores missing a long chromosome, and two spores missing a short chromosome. (C, Precocious sister chromatid separation) A loss of sister chromatid cohesion on a pair of short sister chromatids causes a sister chromatid to fail to properly disjoin during MI segregation. The sister chromatid segregates randomly during MI segregation, resulting in two live spores, one live but disomic spore with an extra short chromosome, and one dead spore missing a short chromosome. (D, Single MII nondisjunction) Meiotic prophase and MI segregation are normal. Before MII segregation there is loss of sister chromatid cohesion on a pair of short sister chromatids causing them to segregate to the same pole. There are two live spores, one live but disomic spore with an extra short chromosome, and one dead spore missing a short chromosome. (E, MI nondisjunction and precocious sister chromatid separation) The long pair of chromosomes fail to form a crossover, and there is a loss of sister chromatid cohesion between a pair of short sister chromatids. Only one spore is live but disomes and the rest of the spores are dead because they are missing either a long or short chromosome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2083027092" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Schematic of different forms of nondisjunction. Newly replicated chromosomes (right) proceed to two sequential rounds of chromosome separation. The cell has two pairs of homologs, a long pair (red and blue), and a short pair (green and purple). Each homolog contains two sister chromatids represented by single lines. Every chromosome is essential, and the absence of either a long or short chromosome results in a dead spore. A spore with a single long and short chromosome is considered live and normal. A spore with either two long chromosomes and one short chromosome, or one long chromosome and two short chromosomes is considered a live disomes. The live disomes have a high probability of living but may die at a low frequency due to aneuploidy induced death (ANID, see &lt;em&gt;Materials and Methods&lt;/em&gt;). (A, Normal meiosis) The long and short homologous chromosomes each undergo crossing over to ensure their proper segregation at MI. Following MII, each spore receives a long and a short chromosome resulting in four live spores. (B, Top; single MI nondisjunction) The long pair of chromosomes fails to form a crossover, resulting in an MI nondisjunction event where both chromosomes segregate to the same pole. Following MII, two spores are disomic for the long chromosome, and, in most cases produce live spores. The two spores missing a copy of the long chromosome die. (Bottom; MI double-nondisjunction) Both the long and short chromosomes fail to form a crossover, leading to a double nondisjunction where each pair of homologs separate to opposite poles. The result is four dead spores with two spores missing a long chromosome, and two spores missing a short chromosome. (C, Precocious sister chromatid separation) A loss of sister chromatid cohesion on a pair of short sister chromatids causes a sister chromatid to fail to properly disjoin during MI segregation. The sister chromatid segregates randomly during MI segregation, resulting in two live spores, one live but disomic spore with an extra short chromosome, and one dead spore missing a short chromosome. (D, Single MII nondisjunction) Meiotic prophase and MI segregation are normal. Before MII segregation there is loss of sister chromatid cohesion on a pair of short sister chromatids causing them to segregate to the same pole. There are two live spores, one live but disomic spore with an extra short chromosome, and one dead spore missing a short chromosome. (E, MI nondisjunction and precocious sister chromatid separation) The long pair of chromosomes fail to form a crossover, and there is a loss of sister chromatid cohesion between a pair of short sister chromatids. Only one spore is live but disomes and the rest of the spores are dead because they are missing either a long or short chromosome.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/669/F1.medium.gif" width="302" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/3/669/F1.medium.gif" width="302" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/669/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/669/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467805" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-4" class="first-child">Schematic of different forms of nondisjunction. Newly replicated chromosomes (right) proceed to two sequential rounds of chromosome separation. The cell has two pairs of homologs, a long pair (red and blue), and a short pair (green and purple). Each homolog contains two sister chromatids represented by single lines. Every chromosome is essential, and the absence of either a long or short chromosome results in a dead spore. A spore with a single long and short chromosome is considered live and normal. A spore with either two long chromosomes and one short chromosome, or one long chromosome and two short chromosomes is considered a live disomes. The live disomes have a high probability of living but may die at a low frequency due to aneuploidy induced death (ANID, see <em>Materials and Methods</em>). (A, Normal meiosis) The long and short homologous chromosomes each undergo crossing over to ensure their proper segregation at MI. Following MII, each spore receives a long and a short chromosome resulting in four live spores. (B, Top; single MI nondisjunction) The long pair of chromosomes fails to form a crossover, resulting in an MI nondisjunction event where both chromosomes segregate to the same pole. Following MII, two spores are disomic for the long chromosome, and, in most cases produce live spores. The two spores missing a copy of the long chromosome die. (Bottom; MI double-nondisjunction) Both the long and short chromosomes fail to form a crossover, leading to a double nondisjunction where each pair of homologs separate to opposite poles. The result is four dead spores with two spores missing a long chromosome, and two spores missing a short chromosome. (C, Precocious sister chromatid separation) A loss of sister chromatid cohesion on a pair of short sister chromatids causes a sister chromatid to fail to properly disjoin during MI segregation. The sister chromatid segregates randomly during MI segregation, resulting in two live spores, one live but disomic spore with an extra short chromosome, and one dead spore missing a short chromosome. (D, Single MII nondisjunction) Meiotic prophase and MI segregation are normal. Before MII segregation there is loss of sister chromatid cohesion on a pair of short sister chromatids causing them to segregate to the same pole. There are two live spores, one live but disomic spore with an extra short chromosome, and one dead spore missing a short chromosome. (E, MI nondisjunction and precocious sister chromatid separation) The long pair of chromosomes fail to form a crossover, and there is a loss of sister chromatid cohesion between a pair of short sister chromatids. Only one spore is live but disomes and the rest of the spores are dead because they are missing either a long or short chromosome.</p><div class="sb-div caption-clear"></div></div></div><p id="p-5">In female mammals, meiosis I prophase occurs in ovaries of the fetus, yet cells are blocked from completing anaphase until sexual maturity (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Nagaoka <em>et al.</em> 2012</a>). Thus, these contacts must be robust enough to last through decades spanning a reproductive lifespan. A maternal age effect increases the incidence of meiosis I (MI) chromosome missegregation in these gametes with compromised homolog attachments. The increased incidence of miscarriages in older women is likely due to the increased levels of MI nondisjunction (MI-ND; <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Hassold and Hunt 2001</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Nagaoka <em>et al.</em> 2012</a>).</p><p id="p-6">Budding yeast has long served as an excellent model organism for studying the chromosome events of MI prophase. Forward genetic screens have identified dozens of conserved genes involved in key events such as homolog pairing, crossing over by homologous recombination, rapid chromosome motion, and the formation of the monopolar attachment of sister chromatid pairs, which ensures proper homolog disjunction (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Esposito <em>et al.</em> 1970</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Roeder 1995</a>). The ability to characterize these events in mutant strains is facilitated by the ability to separate the four haploid spore products of meiosis from a single tetrad.</p><p id="p-7">Mutants defective for processes related to the pairing, recombination, or synapsis of homologous chromosomes show decreased levels of spore viability due to MI-ND (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>). While the presence of an extra chromosome (<em>n</em> + 1) generally supports viability, the loss of any one chromosome will result in spore death (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Campbell and Doolittle 1987</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Torres <em>et al.</em> 2007</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">St Charles <em>et al.</em> 2010</a>). Thus, the outcome of a single MI-ND event generally results in a tetrad containing two live disomic spores, and two nullosomic dead spores (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Ross-Macdonald and Roeder 1994</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Sym and Roeder 1994</a>). Two MI-ND events will result in either a 0:4 live:dead tetrad if the homolog pairs segregate away from each other, or a 2:2 live:dead tetrad if the two homolog pairs segregate to the same pole.</p><p id="p-8">Other types of meiotic errors causing spore inviability will appear as random spore death (RSD), which is the frequency any given spore will die, and that the viability of that spore is independent of the viability of any other spore. One example of RSD would be the nondisjunction of a single sister chromatid due to precocious sister chromatid separation (PSCS) or meiosis II nondisjunction (MII-ND); either event would result in a 3:1 live:dead tetrad (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1, C and D</a>). Inviable spores could also arise in cases unrelated to chromosome ND. These include defects in prospore membrane or spore wall formation, improper partitioning of organelles or other essential cytoplasmic components, or when defects in essential cellular processes lead to germination defects (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Neiman 2011</a>). In addition, defects in meiotic double-strand break repair can result in lethal lesions independent of segregation defects (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Hochwagen and Amon 2006</a>). Combinations of MI-ND and RSD can result tetrads containing 3, 2, 1, or 0 viable spores (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1, B and E</a>).</p><p id="p-9">In previous studies, a high incidence of 2:2 and 0:4 tetrads has been interpreted as one indicator of increased MI-ND (<em>e.g.</em>, in mutants showing decreased levels of meiotic recombination; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Ross-Macdonald and Roeder 1994</a>; <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Sym and Roeder 1994</a>). A quantitative description of this qualitative observation is lacking, and there is little mechanistic insight that can be made from this observation alone. For example, one shortcoming of relying on these events as an indicator of MI-ND is that increases in MI-ND may be difficult to observe in some strain backgrounds if significant, independent contributions to spore death are also at play (above).</p><p id="p-10">Confirmation of increased MI-ND can be made by analyzing MI-ND frequencies of individual chromosomes (see references in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). These measurements are typically carried out on a chromosome-by-chromosome basis, and can require dissecting up to thousands of tetrads to generate accurate MI-ND frequencies. Such assays typically involve the detection of segregating heterozygous codominant genetic markers among the viable spore clones. Examples of this approach include cosegregation of <em><a href="http://www.yeastgenome.org/locus/S000124955/overview">MATa</a></em>/<em><a href="http://www.yeastgenome.org/locus/S000029699/overview">MATα</a></em> to form nonmating spore clones for chromosome III, and the use of <em>CEN</em>-linked genetic markers (<em>e.g.</em>, <em><a href="http://www.yeastgenome.org/locus/S000000747/overview">URA3</a></em>/<em><a href="http://www.yeastgenome.org/locus/S000002414/overview">TRP1</a>)</em> for other chromosomes (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Wanat <em>et al.</em> 2008</a>). Alternatively, fluorescent methods can be used to detect MI-ND in intact tetrads. One method is to tag specific chromosomes with large repetitive arrays of <em>tetO</em> or <em>lacO</em> DNA sequence (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Straight <em>et al.</em> 1996</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Michaelis <em>et al.</em> 1997</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Marston <em>et al.</em> 2004</a>). The chromosome can then be tracked by expressing either TetR or LacI fused to a fluorophore, [<em>e.g.</em>, green fluorescent protein (GFP)], which will bind to the arrays, allowing the presence of the tagged chromosomes to be visualized. Alternatively, a homolog pair can be engineered to express two different fluorophores from each homolog (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Thacker <em>et al.</em> 2011</a>). In the case of MI-ND, some spores will harbor multiple fluorophores, while others will have none. Both fluorescence methods have the advantage that many tetrads can be assayed without dissection; however, substantial strain construction carrying all relevant markers is first required.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467717/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467717/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Measured MI-ND frequencies and computationally generated Avg-ND frequencies</span><div class="sb-div caption-clear"></div></div></div><p id="p-12">A chromosome-by-chromosome approach also has limitations. First, if all chromosomes were to have an equal chance of undergoing MI-ND, and only 1 out of 16 chromosomes is analyzed, then only ∼6% of an already rare event can be detected. Accordingly, several thousands of tetrads generally need to be dissected to detect a few rare MI-ND events for only a subset of chromosomes. Second, MI-ND frequencies will be underestimated in mutants where there are high levels of 0:4 live:dead tetrads. Third, it is possible that MI-ND is nonrandomly distributed across the 16 chromosomes for any number of reasons (<em>e.g.</em>, chromosome size effect). For example, the <em><a href="http://www.yeastgenome.org/locus/S000003567/overview">mad2</a></em>Δ mutation, which abolishes the spindle-assembly checkpoint, causes large chromosomes to nondisjoin at a higher frequency than small chromosomes (<a id="xref-ref-40-12" class="xref-bibr" href="#ref-40">Shonn <em>et al.</em> 2000</a>). Therefore, unless three or more chromosomes of different sizes are assayed, any negative results could be misleading.</p><p id="p-13">Here, we offer a computational method (TetFit) to quantify the independent contributions of MI-ND and RSD to spore inviability based on finding the best-fit distribution of 4:0, 3:1, 2:2, 1:3, and 0:4 live:dead spores to an observed distribution for any given genotype. This holistic approach overcomes some of the limitations of measuring MI-ND frequencies of individual chromosomes. TetFit can be applied to any set of tetrad data to determine a calculated average MI-ND frequency (Avg-ND) for any given strain, and even other yeast species. Data mining of published data sets can also uncover heretofore unknown contributions to the fidelity of MI chromosome segregation. We validated TetFit using published data of WT, and mutant strains of budding yeast sets, and uncovered two classes of mutants that skew to either to MI-ND or RSD as the main contributors to spore inviability. We also show that as few as 200 tetrads can be dissected to distinguish MI-ND frequencies for two related mutants.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Calculating the distribution of tetrad types based on random spore death</h3><p id="p-14">TetSim is an R-script that simulates the expected distribution of tetrads giving 4, 3, 2, 1, or 0 viable spores due to RSD (Supporting Information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024380/-/DC1/FileS1.zip">File S1</a>). Here we define RSD as the frequency any given spore will die and that the viability of that spore is independent of the viability of any other spore. We simulated the distributions rather than using a binomial expansion to calculate the distributions so that we could account for data sets with low numbers of tetrads. Moreover, it allowed us to visualize how observed live:dead tetrad frequencies differed from the simulated distributions. In brief, the TetSim generates a matrix with the dimensions of 4 × number of tetrads set by the user that is then filled with computer-generated random fractional numbers of a uniform distribution between 0 and 1. The random numbers are then converted to 1 (live), if the number is less than the spore viability set by the user (<em>e.g.</em>, the random number 0.7 will be converted to 1 if the spore viability is 0.85), or 0 (dead), if the number is greater than the spore viability set by the user. Each set of four spores generated are then converted to a hypothetical tetrad giving 4, 3, 2, 1, or 0 viable spores, and the frequency of each tetrad class is calculated and recorded. This process is then repeated for the specified number of simulations, and the distributions can be plotted using the Beeswarm package of R. For this paper, 50,000 simulations were recorded each time TetSim was used.</p></div><div id="sec-3" class="subsection"><h3>Modeling the effects of RSD and MI-ND on live:dead tetrad distributions</h3><p id="p-15">We created TetFit (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024380/-/DC1/FileS2.zip">File S2</a>), an R-Script, which generates a matrix of live:dead tetrad distributions representing possible pairwise combinations of RSD and MI-ND frequencies using the user-set ranges (the defaults are 0 &lt; RSD &lt; 0.8 and 0 &lt; MI-ND &lt; 0.017) and the user-set intervals (the defaults are 500 for both RSD and MI-ND). The computationally-generated live:dead tetrad frequencies are then compared to the observed live:dead tetrad frequencies to find the RSD and MI-ND frequencies that best fit the observed data, based on the highest <em>P</em>-value generated by a chi-squared test.</p><p id="p-16">To generate the live:dead tetrad frequencies, TetFit first calculates the effects of each RSD generated above for each number of dead spores (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-1.gif"/></noscript></span></span>, 0–4). A binomial expansion is used to determine the frequency of each <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-2.gif"/></noscript></span></span> solely from the contributions of RSD (<span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-3.gif"/></noscript></span></span>) using the equation:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-4.gif"/></noscript></span><span class="disp-formula-label">(Equation 1)</span></span>TetFit next calculates the effects of MI-ND on live:dead tetrad frequencies. TetFit takes into account several features of chromosome aneuploidy. First, MI-ND is defined in the classical sense in which two homologs fail to properly disjoin and segregate to the same pole (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1B</a>). We assumed that every chromosome is essential. As most disomies are well tolerated in yeast, most tetrads that have undergone a single MI-ND event will give two live and two dead spores (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Torres <em>et al.</em> 2007</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">St Charles <em>et al.</em> 2010</a>).</p><p id="p-17">For this model, to simplify the calculations, we assumed that the MI-ND frequency for each chromosome is equal. Since we cannot predict <em>a priori</em> how any given chromosome may be affected in any given mutant, we assume the null hypothesis to be true. We compiled 10 different published studies where individual chromosomes from wild type (WT) strains were assayed for MI-ND frequency (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). We found a very modest trend toward increased MI-ND with increasing chromosome size (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). The <em><a href="http://www.yeastgenome.org/locus/S000003567/overview">mad2</a></em>Δ mutant is the only example that we know of where a chromosome size effect has been measured for enough chromosomes to show a significant trend (<a id="xref-ref-40-13" class="xref-bibr" href="#ref-40">Shonn <em>et al.</em> 2000</a>). We expanded the results from <a id="xref-ref-40-14" class="xref-bibr" href="#ref-40">Shonn <em>et al.</em> (2000)</a> to include data from three other studies, and plotted MI-ND frequencies for individual chromosomes based on size (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a> and <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2, B and C</a>). A trend line fit to the <em><a href="http://www.yeastgenome.org/locus/S000003567/overview">mad2</a></em>Δ data showed a positive correlation (<em>r</em><sup>2</sup> = 0.56). Anecdotal evidence suggests that the opposite is true for mutants affecting meiotic recombination. For the <em><a href="http://www.yeastgenome.org/locus/S000004777/overview">mlh1</a></em>Δ and <em><a href="http://www.yeastgenome.org/locus/S000002313/overview">msh5</a></em>Δ mutants, the data showed a weak negative correlation (<em>r</em><sup>2</sup> = 0.20 and 0.42, respectively; <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a> and <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>). The slopes and correlations for both <em><a href="http://www.yeastgenome.org/locus/S000004777/overview">mlh1</a></em>Δ and <em><a href="http://www.yeastgenome.org/locus/S000002313/overview">msh5</a></em>Δ appeared to be dependent on a single outlier point (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2B</a>). We next combined all of the MI-ND data for all of the mutants we could find data for to determine if an overall mutant chromosome size effect trend could be detected (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a> and <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2C</a>). There was no correlation between chromosome size and MI-ND in the combined data set (<em>r</em><sup>2</sup> = 0.00; <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2C</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/669/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Chromosome size effect. Scatter plots of previously published MI-ND frequencies compared to chromosome size. References for all data points can be found in Table 1. (A) Scatter plot and best fit lines for WT (red) and mad2&#x394; (blue). (B) Scatter plot and best fit lines for WT (red), msh5&#x394; (green), and mlh1&#x394; (orange). (C) Scatter plot and best fit line for mad2&#x394; (blue), msh5&#x394; (green), mlh1&#x394; (orange), and all other mutant MI-ND data from Table 1, excluding spo11-HA, which does not have a MI-ND defect (purple). The best fit line was generated from all data points plotted. The chromosomes analyzed for MI-ND include the shortest yeast chromosome, Chr I (230&#xA0;kbp), and the longest yeast chromosome, Chr IV (1532&#xA0;kbp). Other yeast chromosomes analyzed include Chr III (317&#xA0;kbp), Chr V (577&#xA0;kbp), Chr VII (1091&#xA0;kbp), Chr VIII (563&#xA0;kbp), Chr X (746&#xA0;kbp), and ChrXV (1091&#xA0;kbp)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2083027092" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Chromosome size effect. Scatter plots of previously published MI-ND frequencies compared to chromosome size. References for all data points can be found in Table 1. (A) Scatter plot and best fit lines for WT (red) and &lt;em&gt;mad2&lt;/em&gt;&#x394; (blue). (B) Scatter plot and best fit lines for WT (red), &lt;em&gt;msh5&lt;/em&gt;&#x394; (green), and &lt;em&gt;mlh1&lt;/em&gt;&#x394; (orange). (C) Scatter plot and best fit line for &lt;em&gt;mad2&lt;/em&gt;&#x394; (blue), &lt;em&gt;msh5&lt;/em&gt;&#x394; (green), &lt;em&gt;mlh1&lt;/em&gt;&#x394; (orange), and all other mutant MI-ND data from Table 1, excluding spo11-HA, which does not have a MI-ND defect (purple). The best fit line was generated from all data points plotted. The chromosomes analyzed for MI-ND include the shortest yeast chromosome, Chr I (230&#xA0;kbp), and the longest yeast chromosome, Chr IV (1532&#xA0;kbp). Other yeast chromosomes analyzed include Chr III (317&#xA0;kbp), Chr V (577&#xA0;kbp), Chr VII (1091&#xA0;kbp), Chr VIII (563&#xA0;kbp), Chr X (746&#xA0;kbp), and ChrXV (1091&#xA0;kbp).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/669/F2.medium.gif" width="387" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/3/669/F2.medium.gif" width="387" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/669/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/669/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467753" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-18" class="first-child">Chromosome size effect. Scatter plots of previously published MI-ND frequencies compared to chromosome size. References for all data points can be found in <a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>. (A) Scatter plot and best fit lines for WT (red) and <em>mad2</em>Δ (blue). (B) Scatter plot and best fit lines for WT (red), <em>msh5</em>Δ (green), and <em>mlh1</em>Δ (orange). (C) Scatter plot and best fit line for <em>mad2</em>Δ (blue), <em>msh5</em>Δ (green), <em>mlh1</em>Δ (orange), and all other mutant MI-ND data from <a id="xref-table-wrap-1-7" class="xref-table" href="#T1">Table 1</a>, excluding spo11-HA, which does not have a MI-ND defect (purple). The best fit line was generated from all data points plotted. The chromosomes analyzed for MI-ND include the shortest yeast chromosome, Chr I (230 kbp), and the longest yeast chromosome, Chr IV (1532 kbp). Other yeast chromosomes analyzed include Chr III (317 kbp), Chr V (577 kbp), Chr VII (1091 kbp), Chr VIII (563 kbp), Chr X (746 kbp), and ChrXV (1091 kbp).</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">The incidence of multiple MI-ND events do not follow a Poisson distribution (<a id="xref-ref-40-15" class="xref-bibr" href="#ref-40">Shonn <em>et al.</em> 2000</a>); that is, there are more multi-chromosome MI-ND events than expected from a random distribution. This is likely due to the distributive or achiasmate segregation mechanism that segregates nonexchange chromosomes to opposite poles (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kurdzo and Dawson 2015</a>). This segregation is directed by homology independent pairing of centromeres (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Kemp <em>et al.</em> 2004</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Gladstone <em>et al.</em> 2009</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Newnham <em>et al.</em> 2010</a>). In the case of a single achiasmate homolog pair, this distributive mechanism can direct proper segregation; however, in the case of multiple achiasmate chromosomes, the distributive mechanism will be overwhelmed. To account for the nonrandom distribution of MI-ND events, we included an MI-ND multiplier to increase the number of multi-chromosome MI-ND events.</p><p id="p-20">For each number of <span class="underline">n</span>ondisjunction <span class="underline">c</span>hromosomes (<span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-5.gif"/></noscript></span></span>, 0–16), TetFit uses the MI-ND frequency (<span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-6.gif"/></noscript></span></span>) to calculate the <span class="underline">f</span>requency of <span class="underline">n</span>on<span class="underline">d</span>is<span class="underline">j</span>unction for NC chromosomes per meiosis (<span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-7.gif"/></noscript></span></span>). We used of a binomial expansion to calculate <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-8.gif"/></noscript></span></span> since it was a simple and logical starting place compared to other possible distributions (<em>e.g.</em>, a Poisson distribution). A weakness of the Poisson distribution is that the tail-end of the distribution will include nondisjunction chromosome values greater than the number of yeast chromosomes. The binomial distribution has the advantage that yeasts with fewer than 16 chromosomes (<em>e.g.</em>, <em>Schizosaccharomyces pombe</em>, <em>n</em> = 3) can be easily accommodated by the model, which would not apply to a Poisson distribution. Moreover, we go to great lengths in the paper to show that it models MI-ND quite well. As is, we feel the current solution is sufficient for its intended use. To calculate <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-9.gif"/></noscript></span></span> TetFit uses the equation:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-10.gif"/></noscript></span><span class="disp-formula-label">(Equation 2)</span></span>When <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-11.gif"/></noscript></span></span> &gt; 1, <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-12.gif"/></noscript></span></span> is multiplied by the MI-ND multiplier to account for the nonrandom distribution of MI-ND events. An empirically determined MI-ND multiplier of 10 gave good best-fit live:dead tetrad distributions for the combined Shonn, Martini, and Wanat data sets as a test case.</p><p id="p-21">Without the MI-ND multiplier, Equation 2 will generate a probability distribution under all conditions. Equation 2 will also calculate a probability distribution that sums to 1 when the MI-ND multiplier is set at the default (10), and MI-ND is less than 0.017. Any increase in the frequency of MI-ND due to the MI-ND multiplier is offset by a decrease in the frequency of cells with no MI-ND. Under the default parameters, all of the individual frequencies will be positive and sum to 1. TetFit will generate negative frequencies when the increase from the MI-ND multiplier is greater than the frequency of cells with no MI-ND; this will only occur under extremely high levels of MI-ND (<em>e.g.</em>, when spore inviability due to MI-ND &gt; 75%). Using the default settings will prevent this.</p><p id="p-22">In the case of multiple MI-ND events (<span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-13.gif"/></noscript></span></span> &gt; 1), homolog pairs segregate randomly to each pole. The fraction of cells for which all nondisjunction homolog pairs segregate to a <span class="underline">s</span>ingle <span class="underline">p</span>ole for each <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-14.gif"/></noscript></span></span> (<span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-15.gif"/></noscript></span></span><em>)</em> is given by the equation:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-16.gif"/></noscript></span><span class="disp-formula-label">(Equation 3)</span></span>Though disomy is generally tolerated and viable, disomic strains have growth defects and reduced germination rates (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Campbell and Doolittle 1987</a>; <a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Torres <em>et al.</em> 2007</a>; <a id="xref-ref-41-3" class="xref-bibr" href="#ref-41">St Charles <em>et al.</em> 2010</a>). The effects of multiple disomies appear to be relatively low since spore viability from triploid meiosis, where nearly every spore would contain multiple disomic chromosomes, can be as high as 50–83% (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Campbell and Doolittle 1987</a>; <a id="xref-ref-41-4" class="xref-bibr" href="#ref-41">St Charles <em>et al.</em> 2010</a>). To incorporate the negative effects of an additional chromosome, we added concept of <span class="underline">an</span>euploidy-<span class="underline">i</span>nduced <span class="underline">d</span>eath for a single chromosome aneuploidy (<span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-17.gif"/></noscript></span></span>). To account for the negative effects of multiple aneuploidies, we use a model where aneuploidy-induced death increases with the number of nondisjunction chromosomes (<span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-18.gif"/></noscript></span></span>).</p><p id="p-23">We empirically determined that an <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-19.gif"/></noscript></span></span> of 0.035 was able to produce good fits for the Shonn, Martini, and Wanat data sets. To calculate <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-20.gif"/></noscript></span></span> TetFit uses the equation:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-21.gif"/></noscript></span><span class="disp-formula-label">(Equation 4)</span></span>We assumed that effects of aneuploidy on the two surviving spores were independent, such that the viability or inviability of one spore did not affect the other spore. TetFit calculates the frequency of aneuploidy-induced death of 0, 1, and 2 spores per meiosis (<span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-22.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-23.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-24.gif"/></noscript></span></span>, respectively) for each <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-25.gif"/></noscript></span></span> using the following equations:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-26.gif"/></noscript></span><span class="disp-formula-label">(Equation 5)</span></span><span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-27.gif"/></noscript></span><span class="disp-formula-label">(Equation 6)</span></span><span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-28.gif"/></noscript></span><span class="disp-formula-label">(Equation 7)</span></span>TetFit calculates the frequency of 2, 3, or 4 <span class="underline">s</span>pores <span class="underline">d</span>ead (<span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-29.gif"/></noscript></span></span>, <em><span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-30.gif"/></noscript></span></span></em>, and <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-31.gif"/></noscript></span></span>, respectively) due to MI-ND from <span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-32.gif"/></noscript></span></span> nondisjunction chromosomes, and <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-33.gif"/></noscript></span></span> using the following equations:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-34.gif"/></noscript></span><span class="disp-formula-label">(Equation 8)</span></span><span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-35.gif"/></noscript></span><span class="disp-formula-label">(Equation 9)</span></span><span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-36.gif"/></noscript></span><span class="disp-formula-label">(Equation 10)</span></span>TetFit next calculates the total frequencies of 2, 3, or 4 spores dead from all MI-ND events (<span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-37.gif"/></noscript></span></span>, <em><span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-38.gif"/></noscript></span></span></em>, and <em><span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-39.gif"/></noscript></span></span></em>) by summing all of the <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-40.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-41.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-42.gif"/></noscript></span></span>, respectively, using the following equations:<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-43.gif"/></noscript></span><span class="disp-formula-label">(Equation 11)</span></span><span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-44.gif"/></noscript></span><span class="disp-formula-label">(Equation 12)</span></span><span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-45.gif"/></noscript></span><span class="disp-formula-label">(Equation 13)</span></span>Finally, to calculate the final frequencies of each live:dead tetrad class, we used the tetrad distributions generated from the RSD modeled distribution, and then applied the spore death due to MI-ND to find the final frequencies of tetrads with 4, 3, 2, 1, and 0 live spores (<span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-46.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-47.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-48.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-49.gif"/></noscript></span></span>, and <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-50.gif"/></noscript></span></span>, respectively). We assumed that inviability due to MI-ND and RSD occur completely independently of each other, such that a spore dying from MI-ND does not affect its chance of dying from RSD. Moreover, a spore may be “killed twice” by both death from MI-ND and RSD. Equations 14–18 were derived as follows:</p><p id="p-24">First, the frequencies of 4:0 and 3:1 tetrads are simply the fraction of tetrads with 0 or 1 spores dying due solely to RSD (<span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-51.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-52.gif"/></noscript></span></span> from Equation 1), respectively.<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-53.gif"/></noscript></span><span class="disp-formula-label">(Equation 14)</span></span><span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-54.gif"/></noscript></span><span class="disp-formula-label">(Equation 15)</span></span>Second, to calculate the final frequency of 2:2 live:dead tetrads (<span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-55.gif"/></noscript></span></span> we considered all combinations of RSD and MI-ND that could give this distribution. For the simplest cases, 2:2 tetrads can arise when two spores die due to only to RSD <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-56.gif"/></noscript></span></span>, or when two spores die due to only a single, MI-ND event <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-57.gif"/></noscript></span></span>. Another contributing class includes the tetrads where the two dead spores have been killed twice by RSD and MI-ND. This frequency, 1/6, is analogous to the one out of six chances that the segregation of two unlinked genetic markers results in a nonparental ditype (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Sherman 1991</a>). For the fraction of tetrads that had either 0 or 1 spore die due to RSD, the inclusion of an MI-ND event will result in a 2:2 live:dead tetrad one-half of the time<span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-58.gif"/></noscript></span></span> while the other half will give 1:3 live:dead tetrads (see Equation 17). Thus, TetFit calculates the final frequency of 2:2 live:dead tetrads using Equation 16:<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-59.gif"/></noscript></span><span class="disp-formula-label">(Equation 16)</span></span>We used a similar logic to calculate the final frequency of 1:3 live:dead tetrads <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-60.gif"/></noscript></span></span> by considering the fraction of three spores dying from RSD alone <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-61.gif"/></noscript></span></span>) and the combinations of RSD and MI-ND events using Equation 17.<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-62.gif"/></noscript></span><span class="disp-formula-label">(Equation 17)</span></span>To calculate the final frequency of tetrads with 0 live spores, TetFit subtracts the frequency of 4, 3, 2, and 1 live spore tetrads using the following equation:<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-63.gif"/></noscript></span><span class="disp-formula-label">(Equation 18)</span></span>To calculate average frequency of MI-ND per chromosome per cell (<span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-64.gif"/></noscript></span></span>), TetFit uses the equation:<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-65.gif"/></noscript></span><span class="disp-formula-label">(Equation 19)</span></span>To calculate fraction of spores that are inviable from MI-ND due to MI-<span class="underline">ND</span> <span class="underline">d</span>eath (<span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-66.gif"/></noscript></span></span>) TetFit uses the equation:<span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-67.gif"/></noscript></span><span class="disp-formula-label">(Equation 20)</span></span>Although RSD and <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-68.gif"/></noscript></span></span> can be directly compared to assess their relative contributions to spore inviability, they cannot directly be added together to determine spore viability because of RSD and <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3/669/embed/mml-math-69.gif"/></noscript></span></span> double-counted dead tetrads.</p></div><div id="sec-4" class="subsection"><h3>Determining the best fit RSD and MI-ND</h3><p id="p-25">Once all of the live:dead tetrad distributions are calculated for each RSD and ND combination, each computationally generated live:dead tetrad distribution is compared to the observed distributions for each genotype using a chi-squared test. The live:dead tetrad distribution with the lowest chi-squared statistic is recorded as the best fitting tetrad for each genotype. When performing multiple comparisons, <em>P</em> values were adjusted using the Holm method (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Holm 1979</a>).</p></div></div><div class="section results" id="sec-5"><h2 class="">Results</h2><div id="sec-6" class="subsection"><h3>RSD alone does not account for the observed distribution of live:dead tetrads from WT strains</h3><p id="p-26">WT strains of budding yeast typically give ∼98% spore viability among tetrads (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Rockmill <em>et al.</em> (2006</a>) showed that chromosome III disomic spores could be recovered as a consequence of PSCS events, which would appear as RSD in this analysis. We wondered whether the 2% spore inviability in WT cells could be attributed solely to RSD, or if other factors were at play. To test this, we created TetSim to model the expected distribution of live:dead tetrads due to RSD, which includes PSCS and MII-ND, based on empirically measured spore viability frequencies. These calculated distributions were then compared to a set of six empirically measured data from WT strains, as reported in six previously published papers (<a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Masison and Baker 1992</a>; <a id="xref-ref-40-16" class="xref-bibr" href="#ref-40">Shonn <em>et al.</em> 2000</a>; <a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Jessop <em>et al.</em> 2006</a>; <a id="xref-ref-27-4" class="xref-bibr" href="#ref-27">Martini <em>et al.</em> 2006</a>; <a id="xref-ref-46-12" class="xref-bibr" href="#ref-46">Wanat <em>et al.</em> 2008</a>; <a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Keelagher <em>et al.</em> 2011</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/669/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Simulation of random spore death in WT strains. Representative simulation of tetrad distributions using TetSim are based on the observed spore viability and number of tetrads dissected for the indicated WT data sets. A distribution of dissected tetrad simulations is shown [in gray (4:0), green (3:1), dark blue (2:2), light blue (1:3), and purple (0:4)] with the upper and lower dashed lines represent 99% and 1% percentiles, respectively. The observed frequencies of live:dead tetrad distributions for individual strains are shown as solid lines. (A)&#x2013;(F) Comparison of simulated tetrad distributions with WT data from previously published data sets: (A) Shonn et al. 2000; (B) Martini et al. 2006; (C) Wanat et al. 2008; (D) Keelagher et al. 2011; (E) Masison et al. 1992; and (F) Jessop et al. 2006." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2083027092" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Simulation of random spore death in WT strains. Representative simulation of tetrad distributions using TetSim are based on the observed spore viability and number of tetrads dissected for the indicated WT data sets. A distribution of dissected tetrad simulations is shown [in gray (4:0), green (3:1), dark blue (2:2), light blue (1:3), and purple (0:4)] with the upper and lower dashed lines represent 99% and 1% percentiles, respectively. The observed frequencies of live:dead tetrad distributions for individual strains are shown as solid lines. (A)&#x2013;(F) Comparison of simulated tetrad distributions with WT data from previously published data sets: (A) Shonn &lt;em&gt;et al.&lt;/em&gt; 2000; (B) Martini &lt;em&gt;et al.&lt;/em&gt; 2006; (C) Wanat &lt;em&gt;et al.&lt;/em&gt; 2008; (D) Keelagher &lt;em&gt;et al.&lt;/em&gt; 2011; (E) Masison &lt;em&gt;et al.&lt;/em&gt; 1992; and (F) Jessop &lt;em&gt;et al.&lt;/em&gt; 2006.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/669/F3.medium.gif" width="315" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/3/669/F3.medium.gif" width="315" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/669/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/669/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467710" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">Simulation of random spore death in WT strains. Representative simulation of tetrad distributions using TetSim are based on the observed spore viability and number of tetrads dissected for the indicated WT data sets. A distribution of dissected tetrad simulations is shown [in gray (4:0), green (3:1), dark blue (2:2), light blue (1:3), and purple (0:4)] with the upper and lower dashed lines represent 99% and 1% percentiles, respectively. The observed frequencies of live:dead tetrad distributions for individual strains are shown as solid lines. (A)–(F) Comparison of simulated tetrad distributions with WT data from previously published data sets: (A) <a id="xref-ref-40-17" class="xref-bibr" href="#ref-40">Shonn <em>et al.</em> 2000</a>; (B) <a id="xref-ref-27-5" class="xref-bibr" href="#ref-27">Martini <em>et al.</em> 2006</a>; (C) <a id="xref-ref-46-13" class="xref-bibr" href="#ref-46">Wanat <em>et al.</em> 2008</a>; (D) <a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">Keelagher <em>et al.</em> 2011</a>; (E) <a id="xref-ref-28-5" class="xref-bibr" href="#ref-28">Masison <em>et al.</em> 1992</a>; and (F) <a id="xref-ref-19-5" class="xref-bibr" href="#ref-19">Jessop <em>et al.</em> 2006</a>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">If the spore inviability in the six WT strains was due solely to RSD, then the observed distribution of live:dead tetrads should resemble the output of TetSim. We found, however, that the observed distributions reported for all WT strains fell outside of the 1–99<sup>th</sup> percentiles of simulated data for at least one of the live:dead tetrad classes (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). Specifically, the published data showed an overrepresentation of 2:2 live:dead tetrads, and underrepresentation of 3:1 live:dead tetrad distributions compared to the simulation (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). The observed pattern could not be modeled by simply increasing or decreasing the simulated spore viability. That is, increasing the simulated spore viability would consequently decrease the frequency of 2:2 live:dead tetrads. Conversely, decreasing the simulated spore viability increased the frequency of 3:1 live:dead tetrads. These results suggest that spore inviability in WT strains is not due solely to RSD or PSCS. Instead, the increased incidence of 2:2 live:dead tetrads observed for WT compared to the simulation suggests that spore death may (in part) be due to an intrinsic level of MI-ND.</p></div><div id="sec-7" class="subsection"><h3>RSD does not account for spore inviability in mutants defective for meiotic chromosome segregation</h3><p id="p-29">Using the same approach as for WT (above), we selected mutants with known meiotic chromosome segregation defects and compared the observed distributions of 4, 3, 2, 1 and 0 viable spore tetrads to distributions based on RSD. For this analysis, we included <em><a href="http://www.yeastgenome.org/locus/S000003054/overview">mad1</a></em>Δ and <em><a href="http://www.yeastgenome.org/locus/S000003567/overview">mad2</a></em>Δ mutants that have a defective spindle-assembly checkpoint (<a id="xref-ref-40-18" class="xref-bibr" href="#ref-40">Shonn <em>et al.</em> 2000</a>), the hypomorphic <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">SPO11</a></em> mutants that reduce double-strand break levels (<em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a>-yf</em> and <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a>-df</em>; <a id="xref-ref-27-6" class="xref-bibr" href="#ref-27">Martini <em>et al.</em> 2006</a>), <em><a href="http://www.yeastgenome.org/locus/S000005464/overview">ndj1</a></em>Δ and <em><a href="http://www.yeastgenome.org/locus/S000006121/overview">csm4</a></em>Δ mutants defective for telomere-led motion (<a id="xref-ref-46-14" class="xref-bibr" href="#ref-46">Wanat <em>et al.</em> 2008</a>), and <em><a href="http://www.yeastgenome.org/locus/S000002313/overview">msh5</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000004777/overview">mlh1</a></em>Δ, and <em><a href="http://www.yeastgenome.org/locus/S000005559/overview">exo1</a></em>Δ mutants with defects in homologous recombination (<a id="xref-ref-46-15" class="xref-bibr" href="#ref-46">Wanat <em>et al.</em> 2008</a>; <a id="xref-ref-20-6" class="xref-bibr" href="#ref-20">Keelagher <em>et al.</em> 2011</a>). We also included in our analysis <em><a href="http://www.yeastgenome.org/locus/S000003821/overview">cbf1</a></em>Δ and <em><a href="http://www.yeastgenome.org/locus/S000000311/overview">iml3</a></em>Δ mutants, which have defective centromeres (<a id="xref-ref-28-6" class="xref-bibr" href="#ref-28">Masison and Baker 1992</a>; <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Ghosh <em>et al.</em> 2004</a>), and <em><a href="http://www.yeastgenome.org/locus/S000004802/overview">sgs1</a></em>Δ and <em>sgs1</em>Δ<em>795</em> mutants that display increased PSCS due to increased crossover formation near centromeres (<a id="xref-ref-19-6" class="xref-bibr" href="#ref-19">Jessop <em>et al.</em> 2006</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Rockmill <em>et al.</em> 2006</a>).</p><p id="p-30">All mutants gave one or more tetrad classes outside the 1 through 99<sup>th</sup> percentiles based solely on RSD (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>). These results point to factors other than RSD (or PSCS or MII-ND) contributing to spore death. Poor fits to the RSD model would be expected for mutants with increased MI-ND (<a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a>). For <em><a href="http://www.yeastgenome.org/locus/S000004802/overview">sgs1</a></em> mutants, where PSCS has been shown to contribute to spore inviability (<a id="xref-ref-19-7" class="xref-bibr" href="#ref-19">Jessop <em>et al.</em> 2006</a>; <a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Rockmill <em>et al.</em> 2006</a>), the poor fit to TetSim suggests that MI-ND may also contribute to spore inviability.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/669/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Simulation of random spore death in mutant strains. Representative simulation of tetrad distributions using TetSim based observed spore viability and number of tetrads dissected for the indicated mutant data sets. A distribution of dissected tetrad simulations is shown (in red, green, dark blue, light blue, and purple) with the upper and lower dashed lines represent 99% and 1% percentiles, respectively. The observed frequencies of live:dead tetrad distributions for individual strains are shown as solid lines. Comparison of simulated tetrad distributions with mutant data from previously published data sets generated by Shonn et al. (2000), Martini et al. (2006), Wanat et al. (2008), Keelagher et al. (2011), Masison et al. (1992), and Jessop et al. (2006)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2083027092" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Simulation of random spore death in mutant strains. Representative simulation of tetrad distributions using TetSim based observed spore viability and number of tetrads dissected for the indicated mutant data sets. A distribution of dissected tetrad simulations is shown (in red, green, dark blue, light blue, and purple) with the upper and lower dashed lines represent 99% and 1% percentiles, respectively. The observed frequencies of live:dead tetrad distributions for individual strains are shown as solid lines. Comparison of simulated tetrad distributions with mutant data from previously published data sets generated by Shonn &lt;em&gt;et al.&lt;/em&gt; (2000), Martini &lt;em&gt;et al.&lt;/em&gt; (2006), Wanat &lt;em&gt;et al.&lt;/em&gt; (2008), Keelagher &lt;em&gt;et al.&lt;/em&gt; (2011), Masison &lt;em&gt;et al.&lt;/em&gt; (1992), and Jessop &lt;em&gt;et al.&lt;/em&gt; (2006).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/669/F4.medium.gif" width="360" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/3/669/F4.medium.gif" width="360" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/669/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/669/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467667" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-31" class="first-child">Simulation of random spore death in mutant strains. Representative simulation of tetrad distributions using TetSim based observed spore viability and number of tetrads dissected for the indicated mutant data sets. A distribution of dissected tetrad simulations is shown (in red, green, dark blue, light blue, and purple) with the upper and lower dashed lines represent 99% and 1% percentiles, respectively. The observed frequencies of live:dead tetrad distributions for individual strains are shown as solid lines. Comparison of simulated tetrad distributions with mutant data from previously published data sets generated by <a id="xref-ref-40-19" class="xref-bibr" href="#ref-40">Shonn <em>et al.</em> (2000)</a>, <a id="xref-ref-27-7" class="xref-bibr" href="#ref-27">Martini <em>et al.</em> (2006)</a>, <a id="xref-ref-46-16" class="xref-bibr" href="#ref-46">Wanat <em>et al.</em> (2008)</a>, <a id="xref-ref-20-7" class="xref-bibr" href="#ref-20">Keelagher <em>et al.</em> (2011)</a>, <a id="xref-ref-28-7" class="xref-bibr" href="#ref-28">Masison <em>et al.</em> (1992)</a>, and <a id="xref-ref-19-8" class="xref-bibr" href="#ref-19">Jessop <em>et al.</em> (2006)</a>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>Modeling MI-ND</h3><p id="p-32">We next tested if the differences in observed and expected RSD distributions for WT and mutant strains could be explained by MI-ND. For this we developed TetFit to model the relative contributions of RSD and MI-ND events that give the best fit to the observed data for WT and mutant strains. TetFit also generates the average MI-ND (Avg-ND) frequency, which is the frequency of any given chromosome undergoing a MI-ND event per meiosis.</p><p id="p-33">The assumptions used to model MI-ND are described in detail in <em>Materials and Methods</em>. Briefly, MI-ND is defined in the classical sense in which the two homologs fail to properly disjoin and segregate to the same pole (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1B</a>). We assumed that every chromosome is essential, and that the MI-ND frequency for each chromosome is equal. As most disomies are well tolerated in yeast, most tetrads that have undergone a MI-ND event will give two live and two dead spores (<a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Torres <em>et al.</em> 2007</a>; <a id="xref-ref-41-5" class="xref-bibr" href="#ref-41">St Charles <em>et al.</em> 2010</a>). In the case of multiple MI-ND events, we assumed that the homologs would segregate randomly to each pole. We also accounted for the negative effect of multiple disomies on spore germination (<a id="xref-ref-3-4" class="xref-bibr" href="#ref-3">Campbell and Doolittle 1987</a>; <a id="xref-ref-45-5" class="xref-bibr" href="#ref-45">Torres <em>et al.</em> 2007</a>; <a id="xref-ref-41-6" class="xref-bibr" href="#ref-41">St Charles <em>et al.</em> 2010</a>). Finally, we considered that multi-chromosome MI-ND events occur at higher frequencies than expected from a Poisson distribution (<a id="xref-ref-40-20" class="xref-bibr" href="#ref-40">Shonn <em>et al.</em> 2000</a>).</p></div><div id="sec-9" class="subsection"><h3>Modeled tetrad distributions closely match observed tetrad distributions in WT and mutant strains</h3><p id="p-34">TetFit-A finds the expected (E) RSD and MI-ND frequencies that give the best fit to the observed (O) tetrad distributions for each mutant (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). In 11 out of 15 strains described above, TetFit found combinations of MI-ND and RSD values giving best-fit distributions that were statistically indistinguishable from the observed distributions (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>). These included mutants with known defects in chromosome segregation including <em><a href="http://www.yeastgenome.org/locus/S000003054/overview">mad1</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000003567/overview">mad2</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a>-yf</em>, <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a>-da</em>, <em><a href="http://www.yeastgenome.org/locus/S000005559/overview">exo1</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000006121/overview">csm4</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000005464/overview">ndj1</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000006121/overview">csm4</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000005464/overview">ndj1</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000000311/overview">iml3</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000004802/overview">sgs1</a></em>Δ, and <em><a href="http://www.yeastgenome.org/locus/S000004802/overview">sgs1</a>-795</em>. For a subset of mutants, however, TetFit-A could not find a best-fit distribution. This result points to factors other than MI-ND and RSD that impact spore death in these strains. These cases are discussed in more detail below. In all cases where each mutant strain’s respective intrinsic WT MI-ND is used (TetFit-B), no distribution giving a good fit to the observed distribution could be generated. These findings are consistent with the interpretation that increased levels of MI-ND contribute to spore inviability in WT cells. The <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a>-HA</em> mutant was the only exception since the WT distribution was not significantly different; this would be expected since <em><a href="http://www.yeastgenome.org/locus/S000001014/overview">spo11</a>-HA</em> does not display a spore viability defect (<a id="xref-ref-27-8" class="xref-bibr" href="#ref-27">Martini <em>et al.</em> 2006</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/669/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of observed and expected tetrad distributions. The observed tetrad distributions of genotype (O, middle bar), the expected tetrad distributions with the best fitting MI-ND and RSD (E, right bar), and the expected tetrad distributions using WT MI-ND and the best fitting RSD (W, left bar). Expected live:dead tetrad frequencies were generated for all genotypes using TetFit with the following conditions: the number of nondisjunction intervals (ndint) was set to 3000, the number of random spore death intervals (rsdint) was set to 3000, the number of chromosomes (chr) was set to 16, ANID was set to 0.035, and the MI-ND multiplier (ndm) was set to 10. (A) &#x2013; (H) Comparison of simulated tetrad distributions with WT and mutant data from previously published data sets: (A) Shonn et al. (2000) data set; (B) Martini et al. (2006) data set; (C) Keelagher et al. (2011) data set; (D) Wanat et al. (2008) data set; (E) Ghosh et al. (2004) data set; (F) Masison et al. (1992) data set; (G) Jessop et al. (2006) data set; and (H) Jessop et al. (2006) data set. In these strains, the endogenous SGS1 has been deleted but SGS1 or sgs1&#x394;795 have been inserted at TRP1 in the WT and sgs1&#x394;795 strains, respectively. Significance is noted as **&#xA0;P&#xA0;" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2083027092" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of observed and expected tetrad distributions. The observed tetrad distributions of genotype (O, middle bar), the expected tetrad distributions with the best fitting MI-ND and RSD (E, right bar), and the expected tetrad distributions using WT MI-ND and the best fitting RSD (W, left bar). Expected live:dead tetrad frequencies were generated for all genotypes using TetFit with the following conditions: the number of nondisjunction intervals (ndint) was set to 3000, the number of random spore death intervals (rsdint) was set to 3000, the number of chromosomes (chr) was set to 16, ANID was set to 0.035, and the MI-ND multiplier (ndm) was set to 10. (A) &#x2013; (H) Comparison of simulated tetrad distributions with WT and mutant data from previously published data sets: (A) Shonn &lt;em&gt;et al.&lt;/em&gt; (2000) data set; (B) Martini &lt;em&gt;et al.&lt;/em&gt; (2006) data set; (C) Keelagher &lt;em&gt;et al.&lt;/em&gt; (2011) data set; (D) Wanat &lt;em&gt;et al.&lt;/em&gt; (2008) data set; (E) Ghosh &lt;em&gt;et al.&lt;/em&gt; (2004) data set; (F) Masison &lt;em&gt;et al.&lt;/em&gt; (1992) data set; (G) Jessop &lt;em&gt;et al.&lt;/em&gt; (2006) data set; and (H) Jessop &lt;em&gt;et al.&lt;/em&gt; (2006) data set. In these strains, the endogenous SGS1 has been deleted but &lt;em&gt;SGS1&lt;/em&gt; or &lt;em&gt;sgs1&lt;/em&gt;&#x394;&lt;em&gt;795&lt;/em&gt; have been inserted at &lt;em&gt;TRP1&lt;/em&gt; in the WT and &lt;em&gt;sgs1&lt;/em&gt;&#x394;&lt;em&gt;795&lt;/em&gt; strains, respectively. Significance is noted as **&#xA0;&lt;em&gt;P&lt;/em&gt;&#xA0;&lt;&#xA0;0.005; ***&#xA0;&lt;em&gt;P&lt;/em&gt;&#xA0;&lt;&#xA0;0.0001 (chi-squared, Holm corrected).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/669/F5.medium.gif" width="315" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/3/669/F5.medium.gif" width="315" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/669/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/669/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467831" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-35" class="first-child">Comparison of observed and expected tetrad distributions. The observed tetrad distributions of genotype (O, middle bar), the expected tetrad distributions with the best fitting MI-ND and RSD (E, right bar), and the expected tetrad distributions using WT MI-ND and the best fitting RSD (W, left bar). Expected live:dead tetrad frequencies were generated for all genotypes using TetFit with the following conditions: the number of <span class="underline">n</span>on<span class="underline">d</span>isjunction <span class="underline">int</span>ervals (ndint) was set to 3000, the number of <span class="underline">r</span>andom <span class="underline">s</span>pore <span class="underline">d</span>eath <span class="underline">int</span>ervals (rsdint) was set to 3000, the number of chromosomes (chr) was set to 16, ANID was set to 0.035, and the MI-<span class="underline">ND</span> <span class="underline">m</span>ultiplier (ndm) was set to 10. (A) – (H) Comparison of simulated tetrad distributions with WT and mutant data from previously published data sets: (A) <a id="xref-ref-40-21" class="xref-bibr" href="#ref-40">Shonn <em>et al.</em> (2000)</a> data set; (B) <a id="xref-ref-27-9" class="xref-bibr" href="#ref-27">Martini <em>et al.</em> (2006)</a> data set; (C) <a id="xref-ref-20-8" class="xref-bibr" href="#ref-20">Keelagher <em>et al.</em> (2011)</a> data set; (D) <a id="xref-ref-46-17" class="xref-bibr" href="#ref-46">Wanat <em>et al.</em> (2008)</a> data set; (E) <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Ghosh <em>et al.</em> (2004)</a> data set; (F) <a id="xref-ref-28-8" class="xref-bibr" href="#ref-28">Masison <em>et al.</em> (1992)</a> data set; (G) <a id="xref-ref-19-9" class="xref-bibr" href="#ref-19">Jessop <em>et al.</em> (2006)</a> data set; and (H) <a id="xref-ref-19-10" class="xref-bibr" href="#ref-19">Jessop <em>et al.</em> (2006)</a> data set. In these strains, the endogenous SGS1 has been deleted but <em>SGS1</em> or <em>sgs1</em>Δ<em>795</em> have been inserted at <em>TRP1</em> in the WT and <em>sgs1</em>Δ<em>795</em> strains, respectively. Significance is noted as ** <em>P</em> &lt; 0.005; *** <em>P</em> &lt; 0.0001 (chi-squared, Holm corrected).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Factors in addition to RSD and MI-ND contribute to spore inviability</h3><p id="p-36">While TetFit generated good fits for most mutants, exceptions were found for <em><a href="http://www.yeastgenome.org/locus/S000003821/overview">cbf1</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000004777/overview">mlh1</a></em>Δ, and <em><a href="http://www.yeastgenome.org/locus/S000002313/overview">msh5</a></em>Δ. That is, no combination of MI-ND and RSD values could be generated to fit the observed distribution. Nonetheless, the distributions with the lowest <em>P</em>-values were qualitatively much more similar to the observed distribution than the distribution for RSD alone or with WT MI-ND values. Certainly, there could be other nonrandom factors contributing to spore inviability that are not accounted for by TetFit. We suggest that mutations or aneuploidies generated during mitotic growth (<em><a href="http://www.yeastgenome.org/locus/S000003821/overview">cbf1</a></em>Δ, <em><a href="http://www.yeastgenome.org/locus/S000004777/overview">mlh1</a></em>Δ) or poor sporulation efficiency (<em><a href="http://www.yeastgenome.org/locus/S000002313/overview">msh5</a></em>Δ) could impart spore death independent of MI-ND or RSD.</p><p id="p-37">Loss of <em><a href="http://www.yeastgenome.org/locus/S000003821/overview">cbf1</a></em>Δ is known to increase nondisjunction during mitotic growth, which may cause a population of the dissected tetrads to be aneuploid (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Baker and Masison 1990</a>; <a id="xref-ref-28-9" class="xref-bibr" href="#ref-28">Masison and Baker 1992</a>). In the case of a single monosomy in diploid cells, there would be an increase the 2:2 live dead frequencies, because two spores would be nullosomic after meiosis. Similarly, the mutL homolog, <a href="http://www.yeastgenome.org/locus/S000004777/overview">Mlh1</a>, plays a critical role in mismatch repair during mitotic growth, and loss of <a href="http://www.yeastgenome.org/locus/S000004777/overview">Mlh1</a> results in greatly increased mutation rates (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Heck <em>et al.</em> 2006</a>). Indeed, the cell divisions required for a single diploid <em><a href="http://www.yeastgenome.org/locus/S000004777/overview">mlh1</a></em>Δ cell to form a colony can lead to the accumulation of large numbers of spontaneous recessive lethal mutations, leading to a sizeable loss of spore viability and increased incidence of 2:2 tetrads (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Reenan and Kolodner 1992</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Heck <em>et al.</em> 2006</a>). The <em><a href="http://www.yeastgenome.org/locus/S000004777/overview">mlh1</a></em>Δ dissection data we used was generated using a zero growth mating protocol, in which the two haploid strains are mated and then immediately induced to sporulate (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Reenan and Kolodner 1992</a>; <a id="xref-ref-46-18" class="xref-bibr" href="#ref-46">Wanat <em>et al.</em> 2008</a>). Despite this, it is likely that the haploids used to mate and sporulate would have accumulated nonlethal mutations during their initial growth. Some of these mutations would likely affect mitotic growth and germination, leading to the inability to form a visible colony. Moreover, the random segregation of mutations from each parent may result in synthetic lethal mutation combinations leading to spore inviability.</p><p id="p-38">The <em><a href="http://www.yeastgenome.org/locus/S000002313/overview">msh5</a></em>Δ mutation is not known to cause mitotic defects that could lead to altered live:dead tetrad frequencies. One possible explanation for the poor fit is that unrepaired meiotic recombination intermediates in these strains could introduce a variable that is not accounted for by TetFit. Another possibility of the poor fit for <em><a href="http://www.yeastgenome.org/locus/S000002313/overview">msh5</a></em>Δ is that its low sporulation efficiency may lead to biases in the tetrad population dissected (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Borner <em>et al.</em> 2004</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chan <em>et al.</em> 2009</a>). The meiotic defects caused by <em><a href="http://www.yeastgenome.org/locus/S000002313/overview">msh5</a></em>Δ are reduced at lower temperatures, leading to increased sporulation efficiency of <em><a href="http://www.yeastgenome.org/locus/S000002313/overview">msh5</a></em>Δ cells at lower temperatures (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Borner <em>et al.</em> 2004</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Chan <em>et al.</em> 2009</a>). The <em><a href="http://www.yeastgenome.org/locus/S000002313/overview">msh5</a></em>Δ cells from the Wanat dataset were sporulated at 30°, a temperature at which sporulation efficiency is reduced compared to lower temperatures (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Borner <em>et al.</em> 2004</a>; <a id="xref-ref-46-19" class="xref-bibr" href="#ref-46">Wanat <em>et al.</em> 2008</a>; <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Chan <em>et al.</em> 2009</a>). To test if decreased sporulation efficiency was the cause of the poor fits generated by TetFit, we examined a different dataset of <em><a href="http://www.yeastgenome.org/locus/S000002313/overview">msh5</a></em>Δ sporulated at 23° or 33° with sporulation efficiencies of 68% and 21%, respectively (<a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Chan <em>et al.</em> 2009</a>). We found that TetFit generated better fitting distributions to the observed distributions for the 23° dissection than the 33° dissection (<em>P</em> = 0.105 and <em>P</em> = 0.039, respectively; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024380/-/DC1/FigureS1.pdf">Figure S1</a>), indicating that a biased tetrad population may be the cause of TetFit poor fits to the <em><a href="http://www.yeastgenome.org/locus/S000002313/overview">msh5</a></em>Δ strains.</p><p id="p-39">These results do not necessarily detract from the utility of the computational method to identify mutants exhibiting increased MI-ND. Instead, these examples indicate that genotypes with high levels of spontaneous mutations or reduced sporulation efficiency can have altered spore inviability compared to that expected from RSD and MI-ND.</p></div><div id="sec-11" class="subsection"><h3>Comparison of Avg-ND to observed MI-ND for individual chromosomes</h3><p id="p-40">The calculated Avg-ND for the WT strains described above ranged from 0.1% to 0.6% with an average of 0.3% (<a id="xref-table-wrap-1-9" class="xref-table" href="#T1">Table 1</a>). These values are similar to empirically determined values for MI-ND frequencies for a limited set of individual chromosomes measured using different assays, which ranged from &lt; 0.1–1.5% (<a id="xref-table-wrap-1-10" class="xref-table" href="#T1">Table 1</a>). These results indicate that MI-ND contributes to spore inviability in WT strains.</p><p id="p-41">Measurements of MI-ND frequencies in mutant strains can vary widely depending on the assay and chromosome analyzed (<a id="xref-table-wrap-1-11" class="xref-table" href="#T1">Table 1</a>). For example, for the <em><a href="http://www.yeastgenome.org/locus/S000003567/overview">mad2</a></em>Δ mutant, measurements using the same assay and the same strain background gave MI-ND frequencies ranging from 2% to 18% for different chromosomes suggesting a chromosome size effect, with longer chromosomes more likely to missegregate than shorter chromosomes (<a id="xref-table-wrap-1-12" class="xref-table" href="#T1">Table 1</a>; <a id="xref-ref-40-22" class="xref-bibr" href="#ref-40">Shonn <em>et al.</em> 2000</a>). On the other hand, different assays measuring MI-ND of the same chromosome can be quite different. For example, Chr III measured in <em><a href="http://www.yeastgenome.org/locus/S000003567/overview">mad2</a></em>Δ using the <em>lacO</em> or <em>tetO</em> array markers gave MI-ND frequencies of 2% and 15%, respectively (<a id="xref-table-wrap-1-13" class="xref-table" href="#T1">Table 1</a>). Moreover, using CEN markers on Chr III in the <em><a href="http://www.yeastgenome.org/locus/S000005464/overview">ndj1</a></em>Δ mutant gave MI-ND frequencies from 0.3% to 1.8%, while using a <em>tetO</em> array gave a MI-ND frequency of 6% (<a id="xref-table-wrap-1-14" class="xref-table" href="#T1">Table 1</a>). TetFit calculates average MI-ND frequency for all chromosomes, thus eliminating the risk of picking a chromosome with much higher or lower level of MI-ND compared to other chromosomes. By this reasoning, we expected that the calculated Avg-ND frequency for WT and the mutants affecting MI-ND would fall within the range of observed frequencies for individual chromosomes.</p><p id="p-42">For the mutants shown previously to increase MI-ND, the Avg-ND was within the range of measured values, if not slightly higher (<a id="xref-table-wrap-1-15" class="xref-table" href="#T1">Table 1</a>). By contrast, for the mutants shown previously to increase PSCS, the calculated Avg-ND tended to be slightly lower (<a id="xref-table-wrap-1-16" class="xref-table" href="#T1">Table 1</a>). This could be due to the fact that a subset of PSCS events may appear as MI-ND events in the given assays, which would not be included in the Avg-ND value. These results indicate that Avg-ND gives an approximate value of MI-ND, thus allowing direct comparisons between mutant strains.</p></div><div id="sec-12" class="subsection"><h3>The predictive power of a computational approach to measuring nondisjunction frequency</h3><p id="p-43">MI-ND and RSD (including PSCS and MII-ND events) contribute independently to spore inviability (above). To directly compare the contributions of death from MI-ND and RSD, we computationally converted the expression of MI-ND frequency to MI-<span class="underline">ND</span> <span class="underline">d</span>eath (NDD). For WT strains, the contribution of NDD and RSD were roughly equal (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> and <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Mutant strains appeared to fall into two classes: The first class (Class A) gave NDD/RSD &gt; 3.3, while the second class (Class B) gave NDD/RSD &lt; 0.8 (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>). Not surprisingly, Class A genes include those with roles in homolog engagement and separation, while Class B genes include those with roles in sister chromatid cohesion and centromere function.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/669/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of random spore death and nondisjunction death. A scatter plot of the calculated best fits of RSD and NDD for all of the WT and mutant data sets analyzed. All WT strains are shown as unlabeled spots on the plot. Data points are colored based on the reported number of tetrads dissected. The solid line represents a NDD/RSD slope of 1, which indicates an equal contribution of NDD and RSD on spore inviability. The green shaded area covers the Class&#xA0;A mutants that have an NDD/RSD ratio above 3.3, and the pink shaded area covers Class&#xA0;B mutants that have an NDD/RSD ratio below 0.8." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2083027092" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of random spore death and nondisjunction death. A scatter plot of the calculated best fits of RSD and NDD for all of the WT and mutant data sets analyzed. All WT strains are shown as unlabeled spots on the plot. Data points are colored based on the reported number of tetrads dissected. The solid line represents a NDD/RSD slope of 1, which indicates an equal contribution of NDD and RSD on spore inviability. The green shaded area covers the Class&#xA0;A mutants that have an NDD/RSD ratio above 3.3, and the pink shaded area covers Class&#xA0;B mutants that have an NDD/RSD ratio below 0.8.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/669/F6.medium.gif" width="405" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/3/669/F6.medium.gif" width="405" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/669/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/669/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467634" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-44" class="first-child">Comparison of random spore death and nondisjunction death. A scatter plot of the calculated best fits of RSD and NDD for all of the WT and mutant data sets analyzed. All WT strains are shown as unlabeled spots on the plot. Data points are colored based on the reported number of tetrads dissected. The solid line represents a NDD/RSD slope of 1, which indicates an equal contribution of NDD and RSD on spore inviability. The green shaded area covers the Class A mutants that have an NDD/RSD ratio above 3.3, and the pink shaded area covers Class B mutants that have an NDD/RSD ratio below 0.8.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467757/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467757/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Computationally derived random spore death and nondisjunction death</span><div class="sb-div caption-clear"></div></div></div><p id="p-46">We next tested the robustness of TetFit by running simulations based on the calculated NDD and RSD values determined for <em><a href="http://www.yeastgenome.org/locus/S000006121/overview">csm4</a></em>Δ and <em><a href="http://www.yeastgenome.org/locus/S000005464/overview">ndj1</a></em>Δ using different numbers of tetrads (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>). For these two mutants, with fairly similar defects in spore viability, we found that 200 tetrads is a reasonable number to distinguish their effects on NDD and RSD.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/669/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of variability from a varying number of tetrads analyzed. Simulations were performed using 100&#x2013;1000 simulated dissected tetrads using the best fit RSD and NDD for csm4&#x394; and ndj1&#x394; from the Wanat et al. (2008) dataset. For each simulation 50 sets of tetrad dissections were simulated for csm4&#x394; and ndj1&#x394;. Best fits for csm4&#x394; and ndj1&#x394; are shown with black borders in yellow and blue, respectively. Simulations for csm4&#x394; and ndj1&#x394; are shown in green and purple, respectively. The standard deviations of each simulation for csm4&#x394; RSD (CR), csm4&#x394; NDD (CN), ndj1&#x394; RSD (NR), and ndj1&#x394; NDD (NN) are given." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2083027092" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Analysis of variability from a varying number of tetrads analyzed. Simulations were performed using 100&#x2013;1000 simulated dissected tetrads using the best fit RSD and NDD for &lt;em&gt;csm4&lt;/em&gt;&#x394; and &lt;em&gt;ndj1&lt;/em&gt;&#x394; from the Wanat &lt;em&gt;et al.&lt;/em&gt; (2008) dataset. For each simulation 50 sets of tetrad dissections were simulated for &lt;em&gt;csm4&lt;/em&gt;&#x394; and &lt;em&gt;ndj1&lt;/em&gt;&#x394;. Best fits for &lt;em&gt;csm4&lt;/em&gt;&#x394; and &lt;em&gt;ndj1&lt;/em&gt;&#x394; are shown with black borders in yellow and blue, respectively. Simulations for &lt;em&gt;csm4&lt;/em&gt;&#x394; and &lt;em&gt;ndj1&lt;/em&gt;&#x394; are shown in green and purple, respectively. The standard deviations of each simulation for &lt;em&gt;csm4&lt;/em&gt;&#x394; RSD (CR), &lt;em&gt;csm4&lt;/em&gt;&#x394; NDD (CN), &lt;em&gt;ndj1&lt;/em&gt;&#x394; RSD (NR), and &lt;em&gt;ndj1&lt;/em&gt;&#x394; NDD (NN) are given.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/669/F7.medium.gif" width="358" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/6/3/669/F7.medium.gif" width="358" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/669/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/669/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467770" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-47" class="first-child">Analysis of variability from a varying number of tetrads analyzed. Simulations were performed using 100–1000 simulated dissected tetrads using the best fit RSD and NDD for <em>csm4</em>Δ and <em>ndj1</em>Δ from the <a id="xref-ref-46-28" class="xref-bibr" href="#ref-46">Wanat <em>et al.</em> (2008)</a> dataset. For each simulation 50 sets of tetrad dissections were simulated for <em>csm4</em>Δ and <em>ndj1</em>Δ. Best fits for <em>csm4</em>Δ and <em>ndj1</em>Δ are shown with black borders in yellow and blue, respectively. Simulations for <em>csm4</em>Δ and <em>ndj1</em>Δ are shown in green and purple, respectively. The standard deviations of each simulation for <em>csm4</em>Δ RSD (CR), <em>csm4</em>Δ NDD (CN), <em>ndj1</em>Δ RSD (NR), and <em>ndj1</em>Δ NDD (NN) are given.</p><div class="sb-div caption-clear"></div></div></div><p id="p-48">The ability to separate the relative contributions of RSD and NDD is also a useful tool for genetic epistasis analysis. Both the <em><a href="http://www.yeastgenome.org/locus/S000006121/overview">csm4</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000002313/overview">msh5</a></em>Δ and <em><a href="http://www.yeastgenome.org/locus/S000006121/overview">csm4</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000004777/overview">mlh1</a></em>Δ double mutants give increased levels of RSD and NDD compared to their respective single mutants (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8A</a>). The NDD/RSD ratio for <em><a href="http://www.yeastgenome.org/locus/S000006121/overview">csm4</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000002313/overview">msh5</a></em>Δ is intermediate to those for each single mutant, suggesting that <em><a href="http://www.yeastgenome.org/locus/S000006121/overview">csm4</a></em>Δ and <em><a href="http://www.yeastgenome.org/locus/S000002313/overview">msh5</a></em>Δ both increase NDD and RSD through independent pathways.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/669/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic analysis of double mutants. (A) Bar plot of the NDD/RSD ratios from the Wanat et al. (2008) data set. (B) Scatter plot of RSD and NDD using the Wanat et al. (2008) data set. Single mutants and WT are shown in red and double mutants are shown in blue. Solid lines connect the double mutants to their respective single mutants, and dashed lines connect the single mutants together. A solid line with a NDD/RSD ratio of 1 is shown." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2083027092" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genetic analysis of double mutants. (A) Bar plot of the NDD/RSD ratios from the Wanat &lt;em&gt;et al.&lt;/em&gt; (2008) data set. (B) Scatter plot of RSD and NDD using the Wanat &lt;em&gt;et al.&lt;/em&gt; (2008) data set. Single mutants and WT are shown in red and double mutants are shown in blue. Solid lines connect the double mutants to their respective single mutants, and dashed lines connect the single mutants together. A solid line with a NDD/RSD ratio of 1 is shown.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/669/F8.medium.gif" width="440" height="181"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/6/3/669/F8.medium.gif" width="440" height="181"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/669/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/669/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467718" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-49" class="first-child">Genetic analysis of double mutants. (A) Bar plot of the NDD/RSD ratios from the <a id="xref-ref-46-29" class="xref-bibr" href="#ref-46">Wanat <em>et al.</em> (2008)</a> data set. (B) Scatter plot of RSD and NDD using the <a id="xref-ref-46-30" class="xref-bibr" href="#ref-46">Wanat <em>et al.</em> (2008)</a> data set. Single mutants and WT are shown in red and double mutants are shown in blue. Solid lines connect the double mutants to their respective single mutants, and dashed lines connect the single mutants together. A solid line with a NDD/RSD ratio of 1 is shown.</p><div class="sb-div caption-clear"></div></div></div><p id="p-50">By contrast, the NDD/RSD ratio for <em><a href="http://www.yeastgenome.org/locus/S000006121/overview">csm4</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000004777/overview">mlh1</a></em>Δ is lower than for each single mutant. By comparing the absolute NDD and RSD values, it appears that <em><a href="http://www.yeastgenome.org/locus/S000006121/overview">csm4</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000004777/overview">mlh1</a></em>Δ exhibits less NDD and more RSD than expected from the single-mutant phenotypes, suggesting a more complex relationship between the two mutations than previously observed (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8B</a>; <a id="xref-ref-46-31" class="xref-bibr" href="#ref-46">Wanat <em>et al.</em> 2008</a>). That is, the two mutations act together to reduce NDD, perhaps at the expense of increased RSD.</p><p id="p-51">Both RSD and NDD are reduced in the <em><a href="http://www.yeastgenome.org/locus/S000005464/overview">ndj1</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000006121/overview">csm4</a></em>Δ double mutant compared to each single mutant (<a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8B</a>). The <em><a href="http://www.yeastgenome.org/locus/S000005464/overview">ndj1</a></em>Δ <em><a href="http://www.yeastgenome.org/locus/S000006121/overview">csm4</a></em>Δ double mutant has been shown previously to interact genetically in a partial-reciprocal epistastic manner, in which the phenotype of the double mutant is weaker than either of the single mutants (<a id="xref-ref-46-32" class="xref-bibr" href="#ref-46">Wanat <em>et al.</em> 2008</a>). Our results are consistent with such a relationship since it appears that is that <em><a href="http://www.yeastgenome.org/locus/S000006121/overview">csm4</a></em>Δ suppresses the increased RSD phenotype of the <em><a href="http://www.yeastgenome.org/locus/S000005464/overview">ndj1</a></em>Δ mutant (presumably due to PSCS), and that <em><a href="http://www.yeastgenome.org/locus/S000005464/overview">ndj1</a></em>Δ suppresses the NDD phenotype of <em><a href="http://www.yeastgenome.org/locus/S000006121/overview">csm4</a></em>Δ.</p></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-52">Here, we describe a computational approach (TetFit) to estimate the contributions of meiosis I nondisjunction (MI-ND) and random spore death (RSD) to spore viability in WT and mutant strains. We show that low levels of spore inviability observed for WT strains can be explained by a combination of both RSD and MI-ND, where RSD may be due to precocious sister chromatid separation or meiosis II nondisjunction events. We also find combinations of MI-ND and RSD that can account for spore inviability in a battery of mutant strains defective for meiotic chromosome events. In addition, TetFit can uncover additional contributions to spore death not accounted for by MI-ND or RSD. Thus, TetFit is an effective tool for yeast researchers to uncover the molecular basis for spore inviability in uncharacterized strains.</p><p id="p-53">TetFit will also calculate the average frequency of MI-ND per chromosome (Avg-ND), which has some advantages to measuring MI-ND for individual chromosomes. To date, no comprehensive analysis of the MI-ND frequency of all 16 chromosomes of budding yeast has been performed in either WT or mutant strains. As such, the full range of MI-ND frequencies for each chromosome is not known. <a id="xref-ref-40-26" class="xref-bibr" href="#ref-40">Shonn <em>et al.</em> (2000)</a> examined four chromosomes of differing lengths and found no size dependence on MI-ND in WT cells, but increasing levels of MI-ND for longer chromosomes in a <em><a href="http://www.yeastgenome.org/locus/S000003567/overview">mad2</a></em>Δ background (<a id="xref-table-wrap-1-17" class="xref-table" href="#T1">Table 1</a>). For most strains analyzed, MI-ND frequencies of only one to three chromosomes were determined (<a id="xref-table-wrap-1-18" class="xref-table" href="#T1">Table 1</a>). Even so, MI-ND frequencies for the same chromosome can vary across platforms (<a id="xref-table-wrap-1-19" class="xref-table" href="#T1">Table 1</a>). For example, the MI-ND frequency of chromosome III in the <em><a href="http://www.yeastgenome.org/locus/S000006121/overview">csm4</a></em>Δ mutant has been reported in three different publications, and ranges from 1.9%, 7.8%, to 12% (<a id="xref-table-wrap-1-20" class="xref-table" href="#T1">Table 1</a>). Analysis of artificial chromosome constructs has shown that shorter chromosomes are more likely to missegregate during mitosis than longer chromosomes (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Murray <em>et al.</em> 1986</a>). Nonetheless, it is not possible, <em>a priori</em>, to predict a given mutation may or may not exhibit chromosome-specific effects. The method described here has an increased chance of detecting an MI-ND phenotype.</p><p id="p-54">In mutant backgrounds with a moderate level of MI-ND, analyzing MI-ND frequencies through dominant selectable markers can require dissecting a very large number of tetrads, far more than needed to generate reliable genetic maps. For example, <a id="xref-ref-46-33" class="xref-bibr" href="#ref-46">Wanat <em>et al.</em> (2008</a>) dissected 1164 and 1400 tetrads of <em><a href="http://www.yeastgenome.org/locus/S000006121/overview">csm4</a></em>Δ and <em><a href="http://www.yeastgenome.org/locus/S000004777/overview">mlh1</a></em>Δ mutants, respectively, and compared their MI-ND frequency to WT (1199 tetrads). While these numbers were sufficient for measuring map distances, only three and five Ch XV MI-ND events were detected for <em><a href="http://www.yeastgenome.org/locus/S000006121/overview">csm4</a></em>Δ and <em><a href="http://www.yeastgenome.org/locus/S000004777/overview">mlh1</a></em>Δ, respectively (<a id="xref-ref-46-34" class="xref-bibr" href="#ref-46">Wanat <em>et al.</em> 2008</a>). A Fisher’s exact test comparing WT to <em><a href="http://www.yeastgenome.org/locus/S000006121/overview">csm4</a></em>Δ and <em><a href="http://www.yeastgenome.org/locus/S000004777/overview">mlh1</a></em>Δ, assuming WT had no MI-ND events, gives <em>P</em> values of 0.12 and 0.13 (not adjusted for multiple comparisons), respectively. In these cases, loss of <a href="http://www.yeastgenome.org/locus/S000006121/overview">Csm4</a> or <a href="http://www.yeastgenome.org/locus/S000004777/overview">Mlh1</a> does not appear to significantly increase MI-ND. For strains with lower levels of MI-ND, the work required to detect increases in MI-ND frequencies would be prohibitive.</p><p id="p-55">Using TetFit described here requires the analysis of as few as 200 dissected tetrads to generate RSD and MI-ND data for analysis. The low numbers of tetrads required for each strain also facilitates epistasis analysis. Here we show proof-of-principle that phenotypes of single and double mutant strains can be quantitatively compared. Our analysis also provides additional insight into the nature of spore death that occurs in the double mutants <em>vs.</em> single mutants. We have also allowed for variable chromosome complements so that the TetFit can be easily customized for species other than <em>S. cerevisiae</em>. Importantly, the application of the R-Scripts does not require any special strain construction, and can be applied to previously observed tetrad distributions.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-56">We would like to thank JoAnne Engebrecht, Anne Britt, and An Nguyen for their insightful review of this manuscript. We also thank Eric Alani for sharing tetrad data. The authors have declared that no conflicting interests exist. The work was funded by NIH R01 GM075119 award to S.M.B.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-3"><p id="p-57">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024380/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024380/-/DC1</a></p></li><li class="fn" id="fn-4"><p id="p-58"><em>Communicating editor: C. L. Myers</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-11-02"><span class="received-label">Received </span>November 2, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-01-05"><span class="accepted-label">Accepted </span>January 5, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Chu and Burgess</li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.1" data-doi="10.1128/MCB.10.6.2458"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Masison</span>  <span class="cit-name-given-names">D. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Isolation of the gene encoding the <em>Saccharomyces cerevisiae</em> centromere-binding protein CP1.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">2458</span>–<span class="cit-lpage">2467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DBaker%26rft.auinit1%253DR%2BE%26rft.volume%253D10%26rft.issue%253D6%26rft.spage%253D2458%26rft.epage%253D2467%26rft.atitle%253DIsolation%2Bof%2Bthe%2Bgene%2Bencoding%2Bthe%2BSaccharomyces%2Bcerevisiae%2Bcentromere-binding%2Bprotein%2BCP1.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.10.6.2458%26rft_id%253Dinfo%253Apmid%252F2188087%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjEwLzYvMjQ1OCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMy82NjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.2" data-doi="10.1016/S0092-8674(04)00292-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Borner</span>  <span class="cit-name-given-names">G. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Crossover/noncrossover differentiation, synaptonemal complex formation, and regulatory surveillance at the leptotene/zygotene transition of meiosis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DB%25C3%25B6rner%26rft.auinit1%253DG.%2BV.%26rft.volume%253D117%26rft.issue%253D1%26rft.spage%253D29%26rft.epage%253D45%26rft.atitle%253DCrossover%252Fnoncrossover%2Bdifferentiation%252C%2Bsynaptonemal%2Bcomplex%2Bformation%252C%2Band%2Bregulatory%2Bsurveillance%2Bat%2Bthe%2Bleptotene%252Fzygotene%2Btransition%2Bof%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252804%252900292-2%26rft_id%253Dinfo%253Apmid%252F15066280%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(04)00292-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15066280&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000221500300006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.3" data-doi="10.1007/BF00368058"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doolittle</span>  <span class="cit-name-given-names">M. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Coincident chromosomal disomy in meiotic dyads from triploid yeast.</span> <abbr class="cit-jnl-abbrev">Curr. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">569</span>–<span class="cit-lpage">576</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bgenetics%26rft.stitle%253DCurr%2BGenet%26rft.aulast%253DCampbell%26rft.auinit1%253DD.%2BA.%26rft.volume%253D12%26rft.issue%253D8%26rft.spage%253D569%26rft.epage%253D576%26rft.atitle%253DCoincident%2Bchromosomal%2Bdisomy%2Bin%2Bmeiotic%2Bdyads%2Bfrom%2Btriploid%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00368058%26rft_id%253Dinfo%253Apmid%252F3332249%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00368058&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3332249&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.4" data-doi="10.1371/journal.pone.0007284"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borts</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Temperature-dependent modulation of chromosome segregation in msh4 mutants of budding yeast.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e7284</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DChan%26rft.auinit1%253DA.%2BC.%26rft.volume%253D4%26rft.issue%253D10%26rft.spage%253De7284%26rft.epage%253De7284%26rft.atitle%253DTemperature-dependent%2Bmodulation%2Bof%2Bchromosome%2Bsegregation%2Bin%2Bmsh4%2Bmutants%2Bof%2Bbudding%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0007284%26rft_id%253Dinfo%253Apmid%252F19816584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0007284&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19816584&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.3.669.5" data-doi="10.1038/ng1588"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheslock</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kemp</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boumil</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The roles of MAD1, MAD2 and MAD3 in meiotic progression and the segregation of nonexchange chromosomes.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">756</span>–<span class="cit-lpage">760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DCheslock%26rft.auinit1%253DP.%2BS.%26rft.volume%253D37%26rft.issue%253D7%26rft.spage%253D756%26rft.epage%253D760%26rft.atitle%253DThe%2Broles%2Bof%2BMAD1%252C%2BMAD2%2Band%2BMAD3%2Bin%2Bmeiotic%2Bprogression%2Band%2Bthe%2Bsegregation%2Bof%2Bnonexchange%2Bchromosomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1588%26rft_id%253Dinfo%253Apmid%252F15951820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1588&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15951820&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230196400024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.3.669.6" data-doi="10.1101/gad.11.14.1786"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chua</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Tam1, a telomere-associated meiotic protein, functions in chromosome synapsis and crossover interference.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1786</span>–<span class="cit-lpage">1800</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DChua%26rft.auinit1%253DP%2BR%26rft.volume%253D11%26rft.issue%253D14%26rft.spage%253D1786%26rft.epage%253D1800%26rft.atitle%253DTam1%252C%2Ba%2Btelomere-associated%2Bmeiotic%2Bprotein%252C%2Bfunctions%2Bin%2Bchromosome%2Bsynapsis%2Band%2Bcrossover%2Binterference.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.11.14.1786%26rft_id%253Dinfo%253Apmid%252F9242487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjExLzE0LzE3ODYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.3.669.7" data-doi="10.1126/science.276.5316.1252"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conrad</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dominguez</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dresser</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Ndj1p, a meiotic telomere protein required for normal chromosome synapsis and segregation in yeast.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">276</span>: <span class="cit-fpage">1252</span>–<span class="cit-lpage">1255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DConrad%26rft.auinit1%253DM.%2BN.%26rft.volume%253D276%26rft.issue%253D5316%26rft.spage%253D1252%26rft.epage%253D1255%26rft.atitle%253DNdj1p%252C%2Ba%2BMeiotic%2BTelomere%2BProtein%2BRequired%2Bfor%2BNormal%2BChromosome%2BSynapsis%2Band%2BSegregation%2Bin%2BYeast%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.276.5316.1252%26rft_id%253Dinfo%253Apmid%252F9157883%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyNzYvNTMxNi8xMjUyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Esposito</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Esposito</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arnaud</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halvorson</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-article-title">Conditional mutants of meiosis in yeast.</span> <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">202</span>–<span class="cit-lpage">210</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DEsposito%26rft.auinit1%253DM.%2BS.%26rft.volume%253D104%26rft.issue%253D1%26rft.spage%253D202%26rft.epage%253D210%26rft.atitle%253DConditional%2BMutants%2Bof%2BMeiosis%2Bin%2BYeast%26rft_id%253Dinfo%253Apmid%252F5473889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6OToiMTA0LzEvMjAyIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.3.669.9" data-doi="10.1371/journal.pgen.1000629"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fernius</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marston</span>  <span class="cit-name-given-names">A. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Establishment of cohesion at the pericentromere by the Ctf19 kinetochore subcomplex and the replication fork-associated factor, Csm3.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000629</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DFernius%26rft.auinit1%253DJ.%26rft.volume%253D5%26rft.issue%253D9%26rft.spage%253De1000629%26rft.epage%253De1000629%26rft.atitle%253DEstablishment%2Bof%2Bcohesion%2Bat%2Bthe%2Bpericentromere%2Bby%2Bthe%2BCtf19%2Bkinetochore%2Bsubcomplex%2Band%2Bthe%2Breplication%2Bfork-associated%2Bfactor%252C%2BCsm3.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000629%26rft_id%253Dinfo%253Apmid%252F19730685%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000629&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19730685&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-3" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ghosh</span>  <span class="cit-name-given-names">S. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sau</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lahiri</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lohia</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinha</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The Iml3 protein of the budding yeast is required for the prevention of precocious sister chromatid separation in meiosis I and for sister chromatid disjunction in meiosis II.</span> <abbr class="cit-jnl-abbrev">Curr. Genet.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">82</span>–<span class="cit-lpage">91</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bgenetics%26rft.stitle%253DCurr%2BGenet%26rft.aulast%253DGhosh%26rft.auinit1%253DS.%2BK.%26rft.volume%253D46%26rft.issue%253D2%26rft.spage%253D82%26rft.epage%253D91%26rft.atitle%253DThe%2BIml3%2Bprotein%2Bof%2Bthe%2Bbudding%2Byeast%2Bis%2Brequired%2Bfor%2Bthe%2Bprevention%2Bof%2Bprecocious%2Bsister%2Bchromatid%2Bseparation%2Bin%2Bmeiosis%2BI%2Band%2Bfor%2Bsister%2Bchromatid%2Bdisjunction%2Bin%2Bmeiosis%2BII.%26rft_id%253Dinfo%253Apmid%252F15241623%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=15241623&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223016200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.11" data-doi="10.1371/journal.pgen.1000771"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gladstone</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Obeso</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chuong</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The synaptonemal complex protein Zip1 promotes bi-orientation of centromeres at meiosis I.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000771</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGladstone%26rft.auinit1%253DM.%2BN.%26rft.volume%253D5%26rft.issue%253D12%26rft.spage%253De1000771%26rft.epage%253De1000771%26rft.atitle%253DThe%2Bsynaptonemal%2Bcomplex%2Bprotein%2BZip1%2Bpromotes%2Bbi-orientation%2Bof%2Bcentromeres%2Bat%2Bmeiosis%2BI.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000771%26rft_id%253Dinfo%253Apmid%252F20011112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000771&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20011112&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.12" data-doi="10.1038/35066065"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hassold</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunt</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">To err (meiotically) is human: the genesis of human aneuploidy.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">280</span>–<span class="cit-lpage">291</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DHassold%26rft.auinit1%253DT.%26rft.volume%253D2%26rft.issue%253D4%26rft.spage%253D280%26rft.epage%253D291%26rft.atitle%253DTo%2Berr%2B%2528meiotically%2529%2Bis%2Bhuman%253A%2Bthe%2Bgenesis%2Bof%2Bhuman%2Baneuploidy.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35066065%26rft_id%253Dinfo%253Apmid%252F11283700%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35066065&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11283700&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167837900013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.13" data-doi="10.1534/genetics.106.059311"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heck</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gresham</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botstein</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alani</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Accumulation of recessive lethal mutations in Saccharomyces cerevisiae mlh1 mismatch repair mutants is not associated with gross chromosomal rearrangements.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">519</span>–<span class="cit-lpage">523</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.059311%26rft_id%253Dinfo%253Apmid%252F16816424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc0LzEvNTE5IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.14" data-doi="10.1016/j.cub.2006.03.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hochwagen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amon</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Checking your breaks: surveillance mechanisms of meiotic recombination.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">R217</span>–<span class="cit-lpage">R228</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DHochwagen%26rft.auinit1%253DA.%26rft.volume%253D16%26rft.issue%253D6%26rft.spage%253DR217%26rft.epage%253DR228%26rft.atitle%253DChecking%2Byour%2Bbreaks%253A%2Bsurveillance%2Bmechanisms%2Bof%2Bmeiotic%2Brecombination.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2006.03.009%26rft_id%253Dinfo%253Apmid%252F16546077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2006.03.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16546077&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236480400019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.3.669.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">E. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shcherbakova</span>  <span class="cit-name-given-names">P. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kunkel</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borts</span>  <span class="cit-name-given-names">R. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">MLH1 mutations differentially affect meiotic functions in Saccharomyces cerevisiae.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">515</span>–<span class="cit-lpage">526</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHoffmann%26rft.auinit1%253DE.%2BR.%26rft.volume%253D163%26rft.issue%253D2%26rft.spage%253D515%26rft.epage%253D526%26rft.atitle%253DMLH1%2BMutations%2BDifferentially%2BAffect%2BMeiotic%2BFunctions%2Bin%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Apmid%252F12618391%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYzLzIvNTE1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.3.669.16" data-doi="10.1101/gad.9.14.1728"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hollingsworth</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ponte</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Halsey</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">MSH5, a novel MutS homolog, facilitates meiotic reciprocal recombination between homologs in <em>Saccharomyces cerevisiae</em> but not mismatch repair.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1728</span>–<span class="cit-lpage">1739</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DHollingsworth%26rft.auinit1%253DN%2BM%26rft.volume%253D9%26rft.issue%253D14%26rft.spage%253D1728%26rft.epage%253D1739%26rft.atitle%253DMSH5%252C%2Ba%2Bnovel%2BMutS%2Bhomolog%252C%2Bfacilitates%2Bmeiotic%2Breciprocal%2Brecombination%2Bbetween%2Bhomologs%2Bin%2BSaccharomyces%2Bcerevisiae%2Bbut%2Bnot%2Bmismatch%2Brepair.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.9.14.1728%26rft_id%253Dinfo%253Apmid%252F7622037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6OToiOS8xNC8xNzI4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holm</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">A simple sequentially rejective multiple test procedure.</span> <abbr class="cit-jnl-abbrev">Scand. J. Stat.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">65</span>–<span class="cit-lpage">70</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScand.%2BJ.%2BStat.%26rft.volume%253D6%26rft.spage%253D65%26rft.atitle%253DA%2BSIMPLE%2BSEQUENTIALLY%2BREJECTIVE%2BMULTIPLE%2BTEST%2BPROCEDURE.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=A1979JY78700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.3.669.18" data-doi="10.1101/gad.11.12.1573"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borts</span>  <span class="cit-name-given-names">R. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Mlh1 is unique among mismatch repair proteins in its ability to promote crossing-over during meiosis.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1573</span>–<span class="cit-lpage">1582</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DHunter%26rft.auinit1%253DN%26rft.volume%253D11%26rft.issue%253D12%26rft.spage%253D1573%26rft.epage%253D1582%26rft.atitle%253DMlh1%2Bis%2Bunique%2Bamong%2Bmismatch%2Brepair%2Bproteins%2Bin%2Bits%2Bability%2Bto%2Bpromote%2Bcrossing-over%2Bduring%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.11.12.1573%26rft_id%253Dinfo%253Apmid%252F9203583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjExLzEyLzE1NzMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-4" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.19" data-doi="10.1371/journal.pgen.0020155"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jessop</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockmill</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichten</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Meiotic chromosome synapsis-promoting proteins antagonize the anti-crossover activity of sgs1.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DJessop%26rft.auinit1%253DL.%26rft.volume%253D2%26rft.issue%253D9%26rft.spage%253De155%26rft.epage%253De155%26rft.atitle%253DMeiotic%2Bchromosome%2Bsynapsis-promoting%2Bproteins%2Bantagonize%2Bthe%2Banti-crossover%2Bactivity%2Bof%2Bsgs1.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020155%26rft_id%253Dinfo%253Apmid%252F17002499%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020155&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17002499&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-4" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.20" data-doi="10.1016/j.dnarep.2010.09.024"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keelagher</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cotton</span>  <span class="cit-name-given-names">V. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goldman</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borts</span>  <span class="cit-name-given-names">R. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Separable roles for exonuclease I in meiotic DNA double-strand break repair.</span> <abbr class="cit-jnl-abbrev">DNA Repair (Amst.)</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">126</span>–<span class="cit-lpage">137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRepair%2B%2528Amst.%2529%26rft.volume%253D10%26rft.spage%253D126%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.dnarep.2010.09.024%26rft_id%253Dinfo%253Apmid%252F21044871%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.dnarep.2010.09.024&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21044871&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.21" data-doi="10.1101/gad.1227304"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kemp</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boumil</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A role for centromere pairing in meiotic chromosome segregation.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1946</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1227304%26rft_id%253Dinfo%253Apmid%252F15289462%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE4LzE2LzE5NDYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.3.669.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kirkpatrick</span>  <span class="cit-name-given-names">D. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferguson</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petes</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Symington</span>  <span class="cit-name-given-names">L. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Decreased meiotic intergenic recombination and increased meiosis I nondisjunction in exo1 mutants of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">1549</span>–<span class="cit-lpage">1557</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DKirkpatrick%26rft.auinit1%253DD.%2BT.%26rft.volume%253D156%26rft.issue%253D4%26rft.spage%253D1549%26rft.epage%253D1557%26rft.atitle%253DDecreased%2BMeiotic%2BIntergenic%2BRecombination%2Band%2BIncreased%2BMeiosis%2BI%2BNondisjunction%2Bin%2Bexo1%2BMutants%2Bof%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Apmid%252F11102356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ni80LzE1NDkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kurdzo</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dawson</span>  <span class="cit-name-given-names">D. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Centromere pairing–tethering partner chromosomes in meiosis I.</span> <abbr class="cit-jnl-abbrev">FEBS J.</abbr> 282: 2458–2470</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.3.669.24" data-doi="10.1038/ng2120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lacefield</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">A. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The spindle checkpoint rescues the meiotic segregation of chromosomes whose crossovers are far from the centromere.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1273</span>–<span class="cit-lpage">1277</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DLacefield%26rft.auinit1%253DS.%26rft.volume%253D39%26rft.issue%253D10%26rft.spage%253D1273%26rft.epage%253D1277%26rft.atitle%253DThe%2Bspindle%2Bcheckpoint%2Brescues%2Bthe%2Bmeiotic%2Bsegregation%2Bof%2Bchromosomes%2Bwhose%2Bcrossovers%2Bare%2Bfar%2Bfrom%2Bthe%2Bcentromere.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2120%26rft_id%253Dinfo%253Apmid%252F17828265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2120&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17828265&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249737400025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.3.669.25" data-doi="10.1371/journal.pgen.1002730"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">C. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conrad</span>  <span class="cit-name-given-names">M. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dresser</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Meiotic chromosome pairing is promoted by telomere-led chromosome movements independent of bouquet formation.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e1002730</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DLee%26rft.auinit1%253DC.%2BY.%26rft.volume%253D8%26rft.issue%253D5%26rft.spage%253De1002730%26rft.epage%253De1002730%26rft.atitle%253DMeiotic%2Bchromosome%2Bpairing%2Bis%2Bpromoted%2Bby%2Btelomere-led%2Bchromosome%2Bmovements%2Bindependent%2Bof%2Bbouquet%2Bformation.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002730%26rft_id%253Dinfo%253Apmid%252F22654677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002730&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22654677&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.26" data-doi="10.1126/science.1094220"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marston</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tham</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shah</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Amon</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A genome-wide screen identifies genes required for centromeric cohesion.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">303</span>: <span class="cit-fpage">1367</span>–<span class="cit-lpage">1370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1094220%26rft_id%253Dinfo%253Apmid%252F14752166%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDMvNTY2Mi8xMzY3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-4" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.27" data-doi="10.1016/j.cell.2006.05.044"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martini</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Diaz</span>  <span class="cit-name-given-names">R. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keeney</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Crossover homeostasis in yeast meiosis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">295</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMartini%26rft.auinit1%253DE.%26rft.volume%253D126%26rft.issue%253D2%26rft.spage%253D285%26rft.epage%253D295%26rft.atitle%253DCrossover%2Bhomeostasis%2Bin%2Byeast%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2006.05.044%26rft_id%253Dinfo%253Apmid%252F16873061%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2006.05.044&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16873061&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239552600015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-4" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Masison</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Meiosis in <em>Saccharomyces cerevisiae</em> mutants lacking the centromere-binding protein CP1.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">53</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMasison%26rft.auinit1%253DD.%2BC.%26rft.volume%253D131%26rft.issue%253D1%26rft.spage%253D43%26rft.epage%253D53%26rft.atitle%253DMeiosis%2Bin%2BSaccharomyces%2Bcerevisiae%2Bmutants%2Blacking%2Bthe%2Bcentromere-binding%2Bprotein%2BCP1.%26rft_id%253Dinfo%253Apmid%252F1592241%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTMxLzEvNDMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.29" data-doi="10.1016/S0092-8674(01)80007-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Michaelis</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ciosk</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nasmyth</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Cohesins: chromosomal proteins that prevent premature separation of sister chromatids.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">35</span>–<span class="cit-lpage">45</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMichaelis%26rft.auinit1%253DC.%26rft.volume%253D91%26rft.issue%253D1%26rft.spage%253D35%26rft.epage%253D45%26rft.atitle%253DCohesins%253A%2Bchromosomal%2Bproteins%2Bthat%2Bprevent%2Bpremature%2Bseparation%2Bof%2Bsister%2Bchromatids.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252801%252980007-6%26rft_id%253Dinfo%253Apmid%252F9335333%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(01)80007-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9335333&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997XZ80900007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.30" data-doi="10.1016/0092-8674(86)90284-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">A. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schultes</span>  <span class="cit-name-given-names">N. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szostak</span>  <span class="cit-name-given-names">J. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1986</span> <span class="cit-article-title">Chromosome length controls mitotic chromosome segregation in yeast.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">529</span>–<span class="cit-lpage">536</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMurray%26rft.auinit1%253DA.%2BW.%26rft.volume%253D45%26rft.issue%253D4%26rft.spage%253D529%26rft.epage%253D536%26rft.atitle%253DChromosome%2Blength%2Bcontrols%2Bmitotic%2Bchromosome%2Bsegregation%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252886%252990284-9%26rft_id%253Dinfo%253Apmid%252F3518944%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(86)90284-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3518944&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1986C554500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.31" data-doi="10.1038/nrg3245"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagaoka</span>  <span class="cit-name-given-names">S. I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hassold</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunt</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Human aneuploidy: mechanisms and new insights into an age-old problem.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">493</span>–<span class="cit-lpage">504</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DNagaoka%26rft.auinit1%253DS.%2BI.%26rft.volume%253D13%26rft.issue%253D7%26rft.spage%253D493%26rft.epage%253D504%26rft.atitle%253DHuman%2Baneuploidy%253A%2Bmechanisms%2Band%2Bnew%2Binsights%2Binto%2Ban%2Bage-old%2Bproblem.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3245%26rft_id%253Dinfo%253Apmid%252F22705668%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3245&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22705668&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.32" data-doi="10.1534/genetics.111.127126"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neiman</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Sporulation in the budding yeast <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">737</span>–<span class="cit-lpage">765</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DNeiman%26rft.auinit1%253DA.%2BM.%26rft.volume%253D189%26rft.issue%253D3%26rft.spage%253D737%26rft.epage%253D765%26rft.atitle%253DSporulation%2Bin%2Bthe%2BBudding%2BYeast%2BSaccharomyces%2Bcerevisiae%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.127126%26rft_id%253Dinfo%253Apmid%252F22084423%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg5LzMvNzM3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.33" data-doi="10.1073/pnas.0913435107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Newnham</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jordan</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rockmill</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoffmann</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The synaptonemal complex protein, Zip1, promotes the segregation of nonexchange chromosomes at meiosis I.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">781</span>–<span class="cit-lpage">785</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0913435107%26rft_id%253Dinfo%253Apmid%252F20080752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMDcvMi83ODEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.34" data-doi="10.1016/S0092-8674(03)00083-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petronczki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siomos</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nasmyth</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Un menage a quatre: the molecular biology of chromosome segregation in meiosis.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">440</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DPetronczki%26rft.auinit1%253DM.%26rft.volume%253D112%26rft.issue%253D4%26rft.spage%253D423%26rft.epage%253D440%26rft.atitle%253DUn%2Bmenage%2Ba%2Bquatre%253A%2Bthe%2Bmolecular%2Bbiology%2Bof%2Bchromosome%2Bsegregation%2Bin%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252803%252900083-7%26rft_id%253Dinfo%253Apmid%252F12600308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(03)00083-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12600308&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181252600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reenan</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kolodner</span>  <span class="cit-name-given-names">R. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Characterization of insertion mutations in the Saccharomyces cerevisiae MSH1 and MSH2 genes: evidence for separate mitochondrial and nuclear functions.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">975</span>–<span class="cit-lpage">985</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DReenan%26rft.auinit1%253DR.%2BA.%26rft.volume%253D132%26rft.issue%253D4%26rft.spage%253D975%26rft.epage%253D985%26rft.atitle%253DCharacterization%2Bof%2Binsertion%2Bmutations%2Bin%2Bthe%2BSaccharomyces%2Bcerevisiae%2BMSH1%2Band%2BMSH2%2Bgenes%253A%2Bevidence%2Bfor%2Bseparate%2Bmitochondrial%2Band%2Bnuclear%2Bfunctions.%26rft_id%253Dinfo%253Apmid%252F1334021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTMyLzQvOTc1IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.36" data-doi="10.1534/genetics.106.058933"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rockmill</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Voelkel-Meiman</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Centromere-proximal crossovers are associated with precocious separation of sister chromatids during meiosis in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">174</span>: <span class="cit-fpage">1745</span>–<span class="cit-lpage">1754</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.058933%26rft_id%253Dinfo%253Apmid%252F17028345%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3NC80LzE3NDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.37" data-doi="10.1073/pnas.92.23.10450"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Sex and the single cell: meiosis in yeast.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">10450</span>–<span class="cit-lpage">10456</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DRoeder%26rft.auinit1%253DG.%2BS.%26rft.volume%253D92%26rft.issue%253D23%26rft.spage%253D10450%26rft.epage%253D10456%26rft.atitle%253DSex%2Band%2Bthe%2Bsingle%2Bcell%253A%2Bmeiosis%2Bin%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.92.23.10450%26rft_id%253Dinfo%253Apmid%252F7479818%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTIvMjMvMTA0NTAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.38" data-doi="10.1016/0092-8674(94)90037-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ross-Macdonald</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G. S.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Mutation of a meiosis-specific MutS homolog decreases crossing over but not mismatch correction.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">1069</span>–<span class="cit-lpage">1080</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DRoss-Macdonald%26rft.auinit1%253DP.%26rft.volume%253D79%26rft.issue%253D6%26rft.spage%253D1069%26rft.epage%253D1080%26rft.atitle%253DMutation%2Bof%2Ba%2Bmeiosis-specific%2BMutS%2Bhomolog%2Bdecreases%2Bcrossing%2Bover%2Bbut%2Bnot%2Bmismatch%2Bcorrection.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252894%252990037-X%26rft_id%253Dinfo%253Apmid%252F8001134%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(94)90037-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8001134&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PY08600016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.39" data-doi="10.1016/S1046-2023(05)80158-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Getting started with yeast.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">3</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2Bin%2Benzymology%26rft.stitle%253DMethods%2BEnzymol%26rft.aulast%253DSherman%26rft.auinit1%253DF.%26rft.volume%253D194%26rft.spage%253D3%26rft.epage%253D21%26rft.atitle%253DGetting%2Bstarted%2Bwith%2Byeast.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1046-2023%252805%252980158-6%26rft_id%253Dinfo%253Apmid%252F2005794%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1046-2023(05)80158-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2005794&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1991FN84200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-12" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.40" data-doi="10.1126/science.289.5477.300"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shonn</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarroll</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">A. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Requirement of the spindle checkpoint for proper chromosome segregation in budding yeast meiosis.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">289</span>: <span class="cit-fpage">300</span>–<span class="cit-lpage">303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DShonn%26rft.auinit1%253DM.%2BA.%26rft.volume%253D289%26rft.issue%253D5477%26rft.spage%253D300%26rft.epage%253D303%26rft.atitle%253DRequirement%2Bof%2Bthe%2BSpindle%2BCheckpoint%2Bfor%2BProper%2BChromosome%2BSegregation%2Bin%2BBudding%2BYeast%2BMeiosis%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.289.5477.300%26rft_id%253Dinfo%253Apmid%252F10894778%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODkvNTQ3Ny8zMDAiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.41" data-doi="10.1534/genetics.110.121533"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">St Charles</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamilton</span>  <span class="cit-name-given-names">M. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petes</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Meiotic chromosome segregation in triploid strains of <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">537</span>–<span class="cit-lpage">550</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.121533%26rft_id%253Dinfo%253Apmid%252F20697121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNTM3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.42" data-doi="10.1016/S0960-9822(02)70783-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Straight</span>  <span class="cit-name-given-names">A. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Belmont</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robinett</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murray</span>  <span class="cit-name-given-names">A. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">GFP tagging of budding yeast chromosomes reveals that protein–protein interactions can mediate sister chromatid cohesion.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">1599</span>–<span class="cit-lpage">1608</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DStraight%26rft.auinit1%253DA.%2BF.%26rft.volume%253D6%26rft.issue%253D12%26rft.spage%253D1599%26rft.epage%253D1608%26rft.atitle%253DGFP%2Btagging%2Bof%2Bbudding%2Byeast%2Bchromosomes%2Breveals%2Bthat%2Bprotein-protein%2Binteractions%2Bcan%2Bmediate%2Bsister%2Bchromatid%2Bcohesion.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0960-9822%252802%252970783-5%26rft_id%253Dinfo%253Apmid%252F8994824%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0960-9822(02)70783-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8994824&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VX48200023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.43" data-doi="10.1016/0092-8674(94)90197-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sym</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roeder</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Crossover interference is abolished in the absence of a synaptonemal complex protein.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">283</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DSym%26rft.auinit1%253DM.%26rft.volume%253D79%26rft.issue%253D2%26rft.spage%253D283%26rft.epage%253D292%26rft.atitle%253DCrossover%2Binterference%2Bis%2Babolished%2Bin%2Bthe%2Babsence%2Bof%2Ba%2Bsynaptonemal%2Bcomplex%2Bprotein.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252894%252990197-X%26rft_id%253Dinfo%253Apmid%252F7954796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(94)90197-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7954796&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994PN63000012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.44" data-doi="10.1534/genetics.111.131326"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thacker</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knop</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keeney</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Exploiting spore-autonomous fluorescent protein expression to quantify meiotic chromosome behaviors in <em>Saccharomyces cerevisiae</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">423</span>–<span class="cit-lpage">439</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.131326%26rft_id%253Dinfo%253Apmid%252F21840861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg5LzIvNDIzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzY2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.45" data-doi="10.1126/science.1142210"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Torres</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sokolsky</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tucker</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">L. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boselli</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Effects of aneuploidy on cellular physiology and cell division in haploid yeast.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">317</span>: <span class="cit-fpage">916</span>–<span class="cit-lpage">924</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTorres%26rft.auinit1%253DE.%2BM.%26rft.volume%253D317%26rft.issue%253D5840%26rft.spage%253D916%26rft.epage%253D924%26rft.atitle%253DEffects%2Bof%2BAneuploidy%2Bon%2BCellular%2BPhysiology%2Band%2BCell%2BDivision%2Bin%2BHaploid%2BYeast%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1142210%26rft_id%253Dinfo%253Apmid%252F17702937%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTcvNTg0MC85MTYiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.669.46" data-doi="10.1371/journal.pgen.1000188"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wanat</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koszul</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zanders</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weiner</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Csm4, in collaboration with Ndj1, mediates telomere-led chromosome dynamics and recombination during yeast meiosis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e1000188</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DWanat%26rft.auinit1%253DJ.%2BJ.%26rft.volume%253D4%26rft.issue%253D9%26rft.spage%253De1000188%26rft.epage%253De1000188%26rft.atitle%253DCsm4%252C%2Bin%2Bcollaboration%2Bwith%2BNdj1%252C%2Bmediates%2Btelomere-led%2Bchromosome%2Bdynamics%2Band%2Brecombination%2Bduring%2Byeast%2Bmeiosis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000188%26rft_id%253Dinfo%253Apmid%252F18818741%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000188&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18818741&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.3.669.47" data-doi="10.1073/pnas.96.24.13914"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">T. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kleckner</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hunter</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Functional specificity of MutL homologs in yeast: evidence for three Mlh1-based heterocomplexes with distinct roles during meiosis in recombination and mismatch correction.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">13914</span>–<span class="cit-lpage">13919</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DWang%26rft.auinit1%253DT.-F.%26rft.volume%253D96%26rft.issue%253D24%26rft.spage%253D13914%26rft.epage%253D13919%26rft.atitle%253DFunctional%2Bspecificity%2Bof%2BMutL%2Bhomologs%2Bin%2Byeast%253A%2BEvidence%2Bfor%2Bthree%2BMlh1-based%2Bheterocomplexes%2Bwith%2Bdistinct%2Broles%2Bduring%2Bmeiosis%2Bin%2Brecombination%2Band%2Bmismatch%2Bcorrection%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.96.24.13914%26rft_id%253Dinfo%253Apmid%252F10570173%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTYvMjQvMTM5MTQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.3.669.48" data-doi="10.1016/0092-8674(95)90335-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watt</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Louis</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borts</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hickson</span>  <span class="cit-name-given-names">I. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Sgs1: a eukaryotic homolog of <em>E. coli</em> RecQ that interacts with topoisomerase II in vivo and is required for faithful chromosome segregation.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">253</span>–<span class="cit-lpage">260</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DWatt%26rft.auinit1%253DP.%2BM.%26rft.volume%253D81%26rft.issue%253D2%26rft.spage%253D253%26rft.epage%253D260%26rft.atitle%253DSgs1%253A%2Ba%2Beukaryotic%2Bhomolog%2Bof%2BE.%2Bcoli%2BRecQ%2Bthat%2Binteracts%2Bwith%2Btopoisomerase%2BII%2Bin%2Bvivo%2Band%2Bis%2Brequired%2Bfor%2Bfaithful%2Bchromosome%2Bsegregation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252895%252990335-6%26rft_id%253Dinfo%253Apmid%252F7736577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(95)90335-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7736577&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995QV41000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.3.669.49" data-doi="10.1371/journal.pgen.1000571"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zanders</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alani</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The pch2Delta mutation in baker’s yeast alters meiotic crossover levels and confers a defect in crossover interference.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000571</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DZanders%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.issue%253D7%26rft.spage%253De1000571%26rft.epage%253De1000571%26rft.atitle%253DThe%2Bpch2Delta%2Bmutation%2Bin%2Bbaker%2527s%2Byeast%2Balters%2Bmeiotic%2Bcrossover%2Blevels%2Band%2Bconfers%2Ba%2Bdefect%2Bin%2Bcrossover%2Binterference.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000571%26rft_id%253Dinfo%253Apmid%252F19629178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000571&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19629178&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F669.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/3/669.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/3/653" title="Combinatorial Cis-regulation in Saccharomyces Species" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/3/683" title="Diverse Hormone Response Networks in 41 Independent Drosophila Cell Lines" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/3"><span>Volume 6 Issue 3, March 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/3.cover-source.jpg?itok=8EDC4x5t" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/3/669.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YO-1cWYREPMlazfS_axYJbpnNsQD6OYxHC0Qm8A7Tr4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/3/669" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/3/669" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/3/669" class="active" data-icon-position="" data-hide-link-title="0">A Computational Approach to Estimating Nondisjunction Frequency in Saccharomyces cerevisiae</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460077" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-z2Y03UlzPFDC1VLpFhbvZFbmb5GpWjt0g29QcrasVMM" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/3/669.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/3/669" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-e9mFG0HLbdV30ZeYJwhMQAAWYXEyphXv_yb5sjN9tDo" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/3/669" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/3/669.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460077" id="node460077--2" data-pisa="ggg;6/3/669" data-pisa-master="ggg;g3.115.024380" data-apath="/ggg/6/3/669.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Computational Approach to Estimating Nondisjunction Frequency in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Daniel B. Chu</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-0511-0084" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sean M. Burgess</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">669-682;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.024380 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460077--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel B. Chu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Molecular and Cellular Biology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BB.%2BChu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chu%20DB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BB.%2BChu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sean M. Burgess </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Molecular and Cellular Biology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sean%2BM.%2BBurgess%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgess%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASean%2BM.%2BBurgess%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0511-0084" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sean M. Burgess</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:smburgess@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">smburgess@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/3/669" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/3/669&amp;title=A%20Computational%20Approach%20to%20Estimating%20Nondisjunction%20Frequency%20in%20Saccharomyces%20cerevisiae" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/3/669&amp;title=A%20Computational%20Approach%20to%20Estimating%20Nondisjunction%20Frequency%20in%20Saccharomyces%20cerevisiae" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/3/669&amp;title=A%20Computational%20Approach%20to%20Estimating%20Nondisjunction%20Frequency%20in%20Saccharomyces%20cerevisiae" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/3/669&amp;text=A%20Computational%20Approach%20to%20Estimating%20Nondisjunction%20Frequency%20in%20Saccharomyces%20cerevisiae" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/3/669&amp;title=A%20Computational%20Approach%20to%20Estimating%20Nondisjunction%20Frequency%20in%20Saccharomyces%20cerevisiae" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/3/669&amp;t=A%20Computational%20Approach%20to%20Estimating%20Nondisjunction%20Frequency%20in%20Saccharomyces%20cerevisiae" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/3/669&amp;title=A%20Computational%20Approach%20to%20Estimating%20Nondisjunction%20Frequency%20in%20Saccharomyces%20cerevisiae" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/3/669&amp;title=A%20Computational%20Approach%20to%20Estimating%20Nondisjunction%20Frequency%20in%20Saccharomyces%20cerevisiae" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460077/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460077" id="node460077--3" data-pisa="ggg;6/3/669" data-pisa-master="ggg;g3.115.024380" data-apath="/ggg/6/3/669.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A Computational Approach to Estimating Nondisjunction Frequency in <em>Saccharomyces cerevisiae</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Daniel B. Chu</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-0511-0084" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Sean M. Burgess</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">669-682;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.024380 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460077--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel B. Chu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Molecular and Cellular Biology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BB.%2BChu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chu%20DB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BB.%2BChu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Sean M. Burgess </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Molecular and Cellular Biology, University of California, Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sean%2BM.%2BBurgess%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Burgess%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ASean%2BM.%2BBurgess%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-0511-0084" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Sean M. Burgess</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:smburgess@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">smburgess@ucdavis.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=669&amp;rft.epage=682&amp;rft.atitle=A%20Computational%20Approach%20to%20Estimating%20Nondisjunction%20Frequency%20in%20Saccharomyces%20cerevisiae&amp;rft.volume=6&amp;rft.issue=3&amp;rft.date=2016-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Chu%2C+Daniel+B.&amp;rft.au=Burgess%2C+Sean+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460077/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460077/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460077/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460077/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460077/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460077/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460077/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460077/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460077/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460077/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460077/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460077/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/3/669" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-5" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/3/669.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/3/669.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/3/669.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/3/669.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
