<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/3/609" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/3/609.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/3/609.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/3/609.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/3/609.atom" />
<meta name="HW.pisa" content="ggg;6/3/609" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="The Effects of Both Recent and Long-Term Selection and Genetic Drift Are Readily Evident in North American Barley Breeding Populations" />
<meta name="DC.Identifier" content="10.1534/g3.115.024349" />
<meta name="DC.Date" content="2016-03-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Poets et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Barley was introduced to North America ∼400 yr ago but adaptation to modern production environments is more recent. Comparisons of allele frequencies among growth habits and spike (inflorescence) types in North America indicate that significant genetic differentiation has accumulated in a relatively short evolutionary time span. Allele frequency differentiation is greatest among barley with two-row vs. six-row spikes, followed by spring vs. winter growth habit. Large changes in allele frequency among breeding programs suggest a major contribution of genetic drift and linked selection on genetic variation. Despite this, comparisons of 3613 modern North American cultivated barley breeding lines that differ for spike-type and growth habit permit the discovery of 142 single nucleotide polymorphism (SNP) outliers putatively linked to targets of selection. For example, SNPs within the Cbf4 , Ppd-H1 , and Vrn-H1 loci, which have previously been associated with agronomically adaptive phenotypes, are identified as outliers. Analysis of extended haplotype sharing identifies genomic regions shared within and among breeding populations, suggestive of a number of genomic regions subject to recent selection. Finally, we are able to identify recent bouts of gene flow between breeding populations that could point to the sharing of agronomically adaptive variation. These results are supported by pedigrees and breeders’ understanding of germplasm sharing." />
<meta name="DC.Contributor" content="Ana M. Poets" />
<meta name="DC.Contributor" content="Mohsen Mohammadi" />
<meta name="DC.Contributor" content="Kiran Seth" />
<meta name="DC.Contributor" content="Hongyun Wang" />
<meta name="DC.Contributor" content="Thomas J. Y. Kono" />
<meta name="DC.Contributor" content="Zhou Fang" />
<meta name="DC.Contributor" content="Gary J. Muehlbauer" />
<meta name="DC.Contributor" content="Kevin P. Smith" />
<meta name="DC.Contributor" content="Peter L. Morrell" />
<meta name="article:published_time" content="2016-03-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="The Effects of Both Recent and Long-Term Selection and Genetic Drift Are Readily Evident in North American Barley Breeding Populations" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Barley was introduced to North America ∼400 yr ago but adaptation to modern production environments is more recent. Comparisons of allele frequencies among growth habits and spike (inflorescence) types in North America indicate that significant genetic differentiation has accumulated in a relatively short evolutionary time span. Allele frequency differentiation is greatest among barley with two-row &lt;i&gt;vs.&lt;/i&gt; six-row spikes, followed by spring &lt;i&gt;vs.&lt;/i&gt; winter growth habit. Large changes in allele frequency among breeding programs suggest a major contribution of genetic drift and linked selection on genetic variation. Despite this, comparisons of 3613 modern North American cultivated barley breeding lines that differ for spike-type and growth habit permit the discovery of 142 single nucleotide polymorphism (SNP) outliers putatively linked to targets of selection. For example, SNPs within the &lt;i&gt;Cbf4&lt;/i&gt;, &lt;i&gt;Ppd-H1&lt;/i&gt;, and &lt;i&gt;Vrn-H1&lt;/i&gt; loci, which have previously been associated with agronomically adaptive phenotypes, are identified as outliers. Analysis of extended haplotype sharing identifies genomic regions shared within and among breeding populations, suggestive of a number of genomic regions subject to recent selection. Finally, we are able to identify recent bouts of gene flow between breeding populations that could point to the sharing of agronomically adaptive variation. These results are supported by pedigrees and breeders’ understanding of germplasm sharing.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/03/01" />
<meta name="citation_mjid" content="ggg;6/3/609" />
<meta name="citation_id" content="6/3/609" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/3/609" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/3/609.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/3/609.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/3/609.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.115.024349" />
<meta name="citation_pmid" content="26715093" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="3" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="609" />
<meta name="citation_lastpage" content="622" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Ana M. Poets" />
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108" />
<meta name="citation_author_email" content="agonzale@umn.edu" />
<meta name="citation_author_email" content="pmorrell@umn.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4332-5986" />
<meta name="citation_author" content="Mohsen Mohammadi" />
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108" />
<meta name="citation_author" content="Kiran Seth" />
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108" />
<meta name="citation_author" content="Hongyun Wang" />
<meta name="citation_author_institution" content="DuPont Pioneer, Mankato, Minnesota 56001" />
<meta name="citation_author" content="Thomas J. Y. Kono" />
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108" />
<meta name="citation_author" content="Zhou Fang" />
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108" />
<meta name="citation_author" content="Gary J. Muehlbauer" />
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108" />
<meta name="citation_author" content="Kevin P. Smith" />
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108" />
<meta name="citation_author" content="Peter L. Morrell" />
<meta name="citation_author_institution" content="Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108" />
<meta name="citation_author_email" content="agonzale@umn.edu" />
<meta name="citation_author_email" content="pmorrell@umn.edu" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=M. A. Beaumont;citation_author=D. J. Balding;citation_title=Identifying adaptive genetic divergence among populations from genome scans.;citation_pages=969-980;citation_volume=13;citation_year=2004;citation_issue=4;citation_pmid=15012769;citation_doi=10.1111/j.1365-294X.2004.02125.x" />
<meta name="citation_reference" content="citation_journal_title=Crop Pasture Sci.;citation_author=W. J. R. Boyd;citation_author=C. D. Li;citation_author=C. R. Grime;citation_author=M. Cakir;citation_author=S. Potipibool;citation_title=Conventional and molecular genetic analysis of factors contributing to variation in the timing of heading among spring barley (Hordeum vulgare L.) genotypes grown over a mild winter growing season.;citation_pages=1277-1301;citation_volume=54;citation_year=2003;citation_issue=12;citation_doi=10.1071/AR03014" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=D. Briscoe;citation_author=J. C. Stephens;citation_author=S. J. O’brien;citation_title=Linkage Disequilibrium in Admixed Populations: Applications in Gene Mapping;citation_pages=59-63;citation_volume=85;citation_year=1994;citation_issue=1;citation_pmid=8120361" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=A. L. Caicedo;citation_author=S. H. Williamson;citation_author=R. D. Hernandez;citation_author=A. Boyko;citation_author=A. Fledel-Alon;citation_title=Genome-wide patterns of nucleotide polymorphism in domesticated rice.;citation_pages=1745-1756;citation_volume=3;citation_year=2007;citation_issue=9;citation_pmid=17907810" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=L. L. Cavalli-Sforza;citation_title=Population Structure and Human Evolution;citation_pages=362-379;citation_volume=164;citation_year=1966;citation_issue=995;citation_pmid=4379525;citation_doi=10.1098/rspb.1966.0038" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. R. Cavanagh;citation_author=S. Chao;citation_author=S. Wang;citation_author=B. E. Huang;citation_author=S. Stephen;citation_title=Genome-wide comparative diversity uncovers multiple targets of selection for improvement in hexaploid wheat landraces and cultivars.;citation_volume=110;citation_year=2013;citation_issue=20;citation_pmid=23630259;citation_doi=10.1073/pnas.1217133110" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=R. Chakraborty;citation_author=P. E. Smouse;citation_title=Recombination of haplotypes leads to biased estimates of admixture proportions in human populations.;citation_pages=3071-3074;citation_volume=85;citation_year=1988;citation_issue=9;citation_pmid=3362862;citation_doi=10.1073/pnas.85.9.3071" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=R. Chakraborty;citation_author=K. M. Weiss;citation_title=Admixture as a tool for finding linked genes and detecting that difference from allelic association between loci.;citation_pages=9119-9123;citation_volume=85;citation_year=1988;citation_issue=23;citation_pmid=3194414;citation_doi=10.1073/pnas.85.23.9119" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=C. C. Chang;citation_author=C. C. Chow;citation_author=L. C. Tellier;citation_author=S. Vattikuti;citation_author=S. M. Purcell;citation_title=Second-generation PLINK: rising to the challenge of larger and richer datasets.;citation_pages=7;citation_volume=4;citation_year=2015;citation_pmid=25722852;citation_doi=10.1186/s13742-015-0047-8" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=T. J. Close;citation_author=P. R. Bhat;citation_author=S. Lonardi;citation_author=Y. Wu;citation_author=N. Rostoks;citation_title=Development and implementation of high-throughput SNP genotyping in barley.;citation_pages=582-582;citation_volume=10;citation_year=2009;citation_pmid=19961604;citation_doi=10.1186/1471-2164-10-582" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. Comadran;citation_author=J. R. Russell;citation_author=A. Booth;citation_author=A. Pswarayi;citation_author=S. Ceccarelli;citation_title=Mixed model association scans of multi-environmental trial data reveal major loci controlling yield and yield related traits in Hordeum vulgare in Mediterranean environments.;citation_pages=1363-1373;citation_volume=122;citation_year=2011;citation_issue=7;citation_pmid=21279625;citation_doi=10.1007/s00122-011-1537-4" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Comadran;citation_author=B. Kilian;citation_author=J. Russell;citation_author=L. Ramsay;citation_author=N. Stein;citation_title=Natural variation in a homolog of Antirrhinum CENTRORADIALIS contributed to spring growth habit and environmental adaptation in cultivated barley.;citation_pages=1388-1392;citation_volume=44;citation_year=2012;citation_issue=12;citation_pmid=23160098;citation_doi=10.1038/ng.2447" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=F. Condón;citation_author=C. Gustus;citation_author=D. C. Rasmusson;citation_author=K. P. Smith;citation_title=Effect of Advanced Cycle Breeding on Genetic Diversity in Barley Breeding Germplasm;citation_pages=1027-1036;citation_volume=48;citation_year=2008;citation_issue=3;citation_doi=10.2135/cropsci2007.07.0415" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=A. Cuesta-Marcos;citation_author=P. Szucs;citation_author=T. J. Close;citation_author=T. Filichkin;citation_author=G. J. Muehlbauer;citation_title=Genome-wide SNPs and re-sequencing of growth habit and inflorescence genes in barley: implications for association mapping in germplasm arrays varying in size and structure.;citation_pages=707-707;citation_volume=11;citation_year=2010;citation_pmid=21159198;citation_doi=10.1186/1471-2164-11-707." />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=Z. Fang;citation_author=A. Eule-Nashoba;citation_author=C. Powers;citation_author=T. Y. Kono;citation_author=S. Takuno;citation_title=Comparative analyses identify the contributions of exotic donors to disease resistance in a barley experimental population;citation_volume=3;citation_year=2013;citation_issue=11;citation_pmid=24048643;citation_doi=10.1534/g3.113.007294" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=Z. Fang;citation_author=A. M. Gonzales;citation_author=M. T. Clegg;citation_author=K. P. Smith;citation_author=G. J. Muehlbauer;citation_title=Two genomic regions contribute disproportionately to geographic differentiation in wild barley.;citation_volume=4;citation_year=2014;citation_issue=7;citation_pmid=24760390;citation_doi=10.1534/g3.114.010561" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. C. Fay;citation_author=C. I. Wu;citation_title=Hitchhiking Under Positive Darwinian Selection;citation_pages=1405-1413;citation_volume=155;citation_year=2000;citation_issue=3;citation_pmid=10880498" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. Felsenstein;citation_title=THE EVOLUTIONARY ADVANTAGE OF RECOMBINATION;citation_pages=737-756;citation_volume=78;citation_year=1974;citation_issue=2;citation_pmid=4448362" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Gerke;citation_author=J. Edwards;citation_author=G. Ke;citation_author=J. Ross-Ibarra;citation_author=M. D. Mcmullen;citation_title=The genomic impacts of drift and selection for hybrid performance in maize.;citation_volume=201;citation_year=2015;citation_issue=3;citation_pmid=26385980;citation_doi=10.1534/genetics.115.182410" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Notes;citation_author=J. Goudet;citation_title=Hierfstat, a package for R to compute and test hierarchical F statistics.;citation_pages=184-186;citation_volume=5;citation_year=2005;citation_issue=1;citation_doi=10.1111/j.1471-8286.2004.00828.x" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. Gusev;citation_author=J. K. Lowe;citation_author=M. Stoffel;citation_author=M. J. Daly;citation_author=D. Altshuler;citation_title=Whole population, genome-wide mapping of hidden relatedness.;citation_volume=19;citation_year=2009;citation_issue=2;citation_pmid=18971310;citation_doi=10.1101/gr.081398.108" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. Gusev;citation_author=P. F. Palamara;citation_author=G. Aponte;citation_author=Z. Zhuang;citation_author=A. Darvasi;citation_title=The architecture of long-range haplotypes shared within and across populations.;citation_volume=29;citation_year=2012;citation_issue=2;citation_pmid=21984068;citation_doi=10.1093/molbev/msr133" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=V. Haake;citation_author=D. Cook;citation_author=J. L. Riechmann;citation_author=O. Pineda;citation_author=M. F. Thomashow;citation_title=Transcription factor CBF4 is a regulator of drought adaptation in Arabidopsis.;citation_volume=130;citation_year=2002;citation_issue=2;citation_pmid=12376631;citation_doi=10.1104/pp.006478" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=M. T. Hamblin;citation_author=T. J. Close;citation_author=P. R. Bhat;citation_author=S. Chao;citation_author=J. G. Kling;citation_title=Population Structure and Linkage Disequilibrium in U.S. Barley Germplasm: Implications for Association Mapping;citation_pages=556-566;citation_volume=50;citation_year=2010;citation_issue=2;citation_doi=10.2135/cropsci2009.04.0198" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. P. Hanrahan;citation_author=E. J. Eisen;citation_author=J. E. Lagates;citation_title=EFFECTS OF POPULATION SIZE AND SELECTION INTENSITY ON SHORT-TERM RESPONSE TO SELECTION FOR POSTWEANING GAIN IN MICE;citation_pages=513-530;citation_volume=73;citation_year=1973;citation_issue=3;citation_pmid=4700062" />
<meta name="citation_reference" content="citation_journal_title=FEBS Journal;citation_journal_abbrev=FEBS J;citation_author=A. Herbik;citation_author=G. Koch;citation_author=H. P. Mock;citation_author=D. Dushkov;citation_author=A. Czihal;citation_title=Isolation, characterization and cDNA cloning of nicotianamine synthase from barley. A key enzyme for iron homeostasis in plants.;citation_pages=231-239;citation_volume=265;citation_year=1999;citation_issue=1;citation_pmid=10491178;citation_doi=10.1046/j.1432-1327.1999.00717.x" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=W. G. Hill;citation_author=A. Robertson;citation_title=The effect of linkage on limits to artificial selection.;citation_pages=269-294;citation_volume=8;citation_year=1966;citation_issue=3;citation_pmid=5980116;citation_doi=10.1017/S0016672300010156" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. W. Horton;citation_author=A. M. Hancock;citation_author=Y. S. Huang;citation_author=C. Toomajian;citation_author=S. Atwell;citation_title=Genome-wide patterns of genetic variation in worldwide Arabidopsis thaliana accessions from the RegMap panel.;citation_pages=212-216;citation_volume=44;citation_year=2012;citation_issue=2;citation_pmid=22231484;citation_doi=10.1038/ng.1042" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. R. Hudson;citation_author=K. Bailey;citation_author=D. Skarecky;citation_author=J. Kwiatowski;citation_author=F. J. Ayala;citation_title=Evidence for positive selection in the superoxide dismutase (Sod) region of Drosophila melanogaster.;citation_pages=1329-1340;citation_volume=136;citation_year=1994;citation_issue=4;citation_pmid=8013910" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. B. Hufford;citation_author=X. Xu;citation_author=J. Van Heerwaarden;citation_author=T. Pyhäjärvi;citation_author=J. M. Chia;citation_title=Comparative population genomics of maize domestication and improvement.;citation_pages=808-811;citation_volume=44;citation_year=2012;citation_issue=7;citation_pmid=22660546;citation_doi=10.1038/ng.2309" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=M. B. Hufford;citation_author=P. Lubinksy;citation_author=T. Pyhajarvi;citation_author=M. T. Devengenzo;citation_author=N. C. Ellstrand;citation_title=The genomic signature of crop-wild introgression in maize.;citation_pages=e1003477;citation_volume=9;citation_year=2013;citation_issue=5;citation_pmid=23671421;citation_doi=10.1371/journal.pgen.1003477" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. Innan;citation_author=K. Zhang;citation_author=P. Marjoram;citation_author=S. Tavaré;citation_author=N. A. Rosenberg;citation_title=Statistical tests of the coalescent model based on the haplotype frequency distribution and the number of segregating sites.;citation_volume=169;citation_year=2005;citation_issue=3;citation_pmid=15654103;citation_doi=10.1534/genetics.104.032219" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=H. Jones;citation_author=F. J. Leigh;citation_author=I. Mackay;citation_author=M. A. Bower;citation_author=L. M. J. Smith;citation_title=Population-based resequencing reveals that the flowering time adaptation of cultivated barley originated east of the Fertile Crescent.;citation_volume=25;citation_year=2008;citation_issue=10;citation_pmid=18669581;citation_doi=10.1093/molbev/msn167" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=I. Karsai;citation_author=P. Szucs;citation_author=B. Koszegi;citation_author=P. M. Hayes;citation_author=A. Casas;citation_title=Effects of photo and thermo cycles on flowering time in barley: a genetical phenomics approach.;citation_volume=59;citation_year=2008;citation_issue=10;citation_pmid=18550600;citation_doi=10.1093/jxb/ern131" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Kearse;citation_author=R. Moir;citation_author=A. Wilson;citation_author=S. Stones-Havas;citation_author=M. Cheung;citation_title=Geneious Basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data.;citation_volume=28;citation_year=2012;citation_issue=12;citation_pmid=22543367;citation_doi=10.1093/bioinformatics/bts199" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. Keinan;citation_author=J. C. Mullikin;citation_author=N. Patterson;citation_author=D. Reich;citation_title=Measurement of the human allele frequency spectrum demonstrates greater genetic drift in East Asians than in Europeans.;citation_pages=1251-1255;citation_volume=39;citation_year=2007;citation_issue=10;citation_pmid=17828266;citation_doi=10.1038/ng2116" />
<meta name="citation_reference" content="citation_title=The Neutral Theory of Molecular Evolution;citation_author=M. Kimura;citation_year=1983" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=T. Komatsuda;citation_author=Y. Mano;citation_title=Molecular mapping of the intermedium spike-c ( int-c) and non-brittle rachis 1 ( btr1) loci in barley ( Hordeum vulgare L.).;citation_pages=85-90;citation_volume=105;citation_year=2002;citation_issue=1;citation_pmid=12582565;citation_doi=10.1007/s00122-001-0858-0" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=T. Komatsuda;citation_author=M. Pourkheirandish;citation_author=C. He;citation_author=P. Azhaguvel;citation_author=H. Kanamori;citation_title=Six-rowed barley originated from a mutation in a homeodomain-leucine zipper I-class homeobox gene.;citation_volume=104;citation_year=2007;citation_issue=4;citation_pmid=17220272;citation_doi=10.1073/pnas.0608580104" />
<meta name="citation_reference" content="citation_journal_title=Mol. Ecol. Resour.;citation_author=T. J. Y. Kono;citation_author=K. Seth;citation_author=J. A. Poland;citation_title=SNPMeta: SNP annotation and SNP Metadata collection without a reference genome.;citation_pages=419-425;citation_volume=14;citation_year=2014;citation_issue=2;citation_pmid=24237904;citation_doi=10.1111/1755-0998.12183" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=K. Kristiansson;citation_author=J. Naukkarinen;citation_author=L. Peltonen;citation_title=Isolated populations and complex disease gene identification.;citation_pages=109-109;citation_volume=9;citation_year=2008;citation_issue=8;citation_pmid=18771588;citation_doi=10.1186/gb-2008-9-8-109" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=D. A. Laurie;citation_author=N. Pratchett;citation_author=J. W. Snape;citation_author=J. H. Bezant;citation_title=RFLP mapping of five major genes and eight quantitative trait loci controlling flowering time in a winter x spring barley (Hordeum vulgare L.) cross.;citation_pages=575-585;citation_volume=38;citation_year=1995;citation_issue=3;citation_pmid=18470191" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=R. C. Lewontin;citation_author=J. Krakauer;citation_title=DISTRIBUTION OF GENE FREQUENCY AS A TEST OF THE THEORY OF THE SELECTIVE NEUTRALITY OF POLYMORPHISMS;citation_pages=175-195;citation_volume=74;citation_year=1973;citation_issue=1;citation_pmid=4711903" />
<meta name="citation_reference" content="citation_journal_title=J. Archaeol. Sci.;citation_author=D. L. Lister;citation_author=S. Thaw;citation_author=M. A. Bower;citation_author=H. Jones;citation_author=M. P. Charles;citation_title=Latitudinal variation in a photoperiod response gene in European barley: insight into the dynamics of agricultural spread from ‘historic’ specimens.;citation_pages=1092-1098;citation_volume=36;citation_year=2009;citation_issue=4;citation_doi=10.1016/j.jas.2008.12.012" />
<meta name="citation_reference" content="citation_title=Statistical and Population Genetics;citation_author=J. Marchini;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=J. M. Martin;citation_author=T. K. Blake;citation_author=E. A. Hockett;citation_title=Diversity among North American Spring Barley Cultivars Based on Coefficients of Parentage;citation_pages=1131-1137;citation_volume=31;citation_year=1991;citation_issue=5;citation_doi=10.2135/cropsci1991.0011183X003100050009x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. F. Mayer;citation_author=R. Waugh;citation_author=P. Langridge;citation_author=T. J. Close;citation_author=R. P. Wise;citation_title=A physical, genetic and functional sequence assembly of the barley genome.;citation_pages=711-716;citation_volume=491;citation_year=2012;citation_issue=7456;citation_pmid=23075845" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=P. L. Morrell;citation_author=E. S. Buckler;citation_author=J. Ross-Ibarra;citation_title=Crop genomics: advances and applications.;citation_pages=85-96;citation_volume=13;citation_year=2012;citation_issue=2" />
<meta name="citation_reference" content="citation_journal_title=The Plant Genome Journal;citation_author=M. Muñoz-Amatriaín;citation_author=M. J. Moscou;citation_author=P. R. Bhat;citation_author=J. T. Svensson;citation_author=J. Bartoš;citation_title=An improved consensus linkage map of barley based on flow-sorted chromosomes and single nucleotide polymorphism markers.;citation_pages=238;citation_volume=4;citation_year=2011;citation_issue=3;citation_doi=10.3835/plantgenome2011.08.0023" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. Series B Stat. Methodol.;citation_author=G. Nicholson;citation_author=A. V. Smith;citation_author=F. Jónsson;citation_author=Ó. Gústafsson;citation_author=K. Stefánsson;citation_title=Assessing population differentiation and isolation from single‐nucleotide polymorphism data.;citation_pages=695-715;citation_volume=64;citation_year=2002;citation_issue=4;citation_doi=10.1111/1467-9868.00357" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=R. Nielsen;citation_author=I. Hellmann;citation_author=M. Hubisz;citation_author=C. Bustamante;citation_author=A. G. Clark;citation_title=Recent and ongoing selection in the human genome.;citation_pages=857-868;citation_volume=8;citation_year=2007;citation_issue=11;citation_pmid=17943193" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=M. Nordborg;citation_author=S. Tavaré;citation_title=Linkage disequilibrium: what history has to tell us.;citation_pages=83-90;citation_volume=18;citation_year=2002;citation_issue=2;citation_pmid=11818140;citation_doi=10.1016/S0168-9525(02)02557-X" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=N. Patterson;citation_author=A. L. Price;citation_author=D. Reich;citation_title=Population structure and eigenanalysis.;citation_pages=e190-e190;citation_volume=2;citation_year=2006;citation_issue=12;citation_pmid=17194218;citation_doi=10.1371/journal.pgen.0020190" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. K. Pickrell;citation_author=J. K. Pritchard;citation_title=Inference of population splits and mixtures from genome-wide allele frequency data.;citation_pages=e1002967-e1002967;citation_volume=8;citation_year=2012;citation_issue=11;citation_pmid=23166502;citation_doi=10.1371/journal.pgen.1002967" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=A. M. Poets;citation_author=Z. Fang;citation_author=M. T. Clegg;citation_author=P. Morrell;citation_title=Barley landraces are characterized by geographically heterogeneous genomic origins.;citation_pages=173;citation_volume=16;citation_year=2015;citation_issue=1;citation_pmid=26293830;citation_doi=10.1186/s13059-015-0712-3" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=J. K. Pritchard;citation_author=J. K. Pickrell;citation_author=G. Coop;citation_title=The genetics of human adaptation: hard sweeps, soft sweeps, and polygenic adaptation.;citation_pages=R208-R215;citation_volume=20;citation_year=2010;citation_issue=4;citation_pmid=20178769;citation_doi=10.1016/j.cub.2009.11.055" />
<meta name="citation_reference" content="citation_title=R: A language and environment for statistical computing;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Ralph;citation_author=G. Coop;citation_title=Parallel adaptation: one or many waves of advance of an advantageous allele?;citation_volume=186;citation_year=2010;citation_issue=2;citation_pmid=20660645;citation_doi=10.1534/genetics.110.119594" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=L. Ramsay;citation_author=J. Comadran;citation_author=A. Druka;citation_author=D. F. Marshall;citation_author=W. T. Thomas;citation_title=INTERMEDIUM-C, a modifier of lateral spikelet fertility in barley, is an ortholog of the maize domestication gene TEOSINTE BRANCHED 1.;citation_pages=169-172;citation_volume=43;citation_year=2011;citation_issue=2;citation_pmid=21217754;citation_doi=10.1038/ng.745" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. Reich;citation_author=K. Thangaraj;citation_author=N. Patterson;citation_author=A. L. Price;citation_author=L. Singh;citation_title=Reconstructing Indian population history.;citation_pages=489-494;citation_volume=461;citation_year=2009;citation_issue=7263;citation_pmid=19779445;citation_doi=10.1038/nature08365" />
<meta name="citation_reference" content="citation_journal_title=Proceedings of the Royal Society B: Biological Sciences;citation_journal_abbrev=Proc R Soc B;citation_author=A. Robertson;citation_title=A Theory of Limits in Artificial Selection;citation_pages=234-249;citation_volume=153;citation_year=1960;citation_issue=951;citation_doi=10.1098/rspb.1960.0099" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=E. Rodgers-Melnick;citation_author=P. J. Bradbury;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=C. B. Acharya;citation_title=Recombination in diverse maize is stable, predictable, and associated with genetic load.;citation_volume=112;citation_year=2015;citation_issue=12;citation_pmid=25775595;citation_doi=10.1073/pnas.1413864112" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. Ross-Ibarra;citation_author=P. L. Morrell;citation_author=B. S. Gaut;citation_title=Plant domestication, a unique opportunity to identify the genetic basis of adaptation.;citation_volume=104;citation_year=2007;citation_pmid=17494757;citation_doi=10.1073/pnas.0700643104" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=P. Scheet;citation_author=M. Stephens;citation_title=A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase.;citation_pages=629-644;citation_volume=78;citation_year=2006;citation_issue=4;citation_pmid=16532393;citation_doi=10.1086/502802" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=J. S. Skinner;citation_author=J. Von Zitzewitz;citation_author=P. Szucs;citation_author=L. Marquez-Cedillo;citation_author=T. Filichkin;citation_title=Structural, functional, and phylogenetic characterization of a large CBF gene family in barley.;citation_pages=533-551;citation_volume=59;citation_year=2005;citation_issue=4;citation_pmid=16244905;citation_doi=10.1007/s11103-005-2498-2" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Slatkin;citation_title=Gene flow and the geographic structure of natural populations;citation_pages=787-792;citation_volume=236;citation_year=1987;citation_issue=4803;citation_pmid=3576198;citation_doi=10.1126/science.3576198" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=T. Taji;citation_author=C. Ohsumi;citation_author=S. Iuchi;citation_author=M. Seki;citation_author=M. Kasuga;citation_title=Important roles of drought- and cold-inducible genes for galactinol synthase in stress tolerance in Arabidopsis thaliana.;citation_pages=417-426;citation_volume=29;citation_year=2002;citation_issue=4;citation_pmid=11846875;citation_doi=10.1046/j.0960-7412.2001.01227.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=K. Thornton;citation_title=libsequence: a C++ class library for evolutionary genetic analysis;citation_pages=2325-2327;citation_volume=19;citation_year=2003;citation_issue=17;citation_pmid=14630667;citation_doi=10.1093/bioinformatics/btg316" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=K. R. Thornton;citation_author=A. J. Foran;citation_author=A. D. Long;citation_title=Properties and modeling of GWAS when complex disease risk is due to non-complementing, deleterious mutations in genes of large effect.;citation_pages=e1003258;citation_volume=9;citation_year=2013;citation_issue=2;citation_pmid=23437004;citation_doi=10.1371/journal.pgen.1003258" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=C. Toomajian;citation_author=T. T. Hu;citation_author=M. J. Aranzana;citation_author=C. Lister;citation_author=C. Tang;citation_title=A nonparametric test reveals selection for rapid flowering in the Arabidopsis genome.;citation_pages=e137-e137;citation_volume=4;citation_year=2006;citation_issue=5;citation_pmid=16623598;citation_doi=10.1371/journal.pbio.0040137" />
<meta name="citation_reference" content="citation_journal_title=Annual review of plant biology;citation_journal_abbrev=Annu Rev Plant Biol;citation_author=F. Turck;citation_author=F. Fornara;citation_author=G. Coupland;citation_title=Regulation and identity of florigen: FLOWERING LOCUS T moves center stage.;citation_pages=573-594;citation_volume=59;citation_year=2008;citation_pmid=18444908;citation_doi=10.1146/annurev.arplant.59.032607.092755" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Turner;citation_author=J. Beales;citation_author=S. Faure;citation_author=R. P. Dunford;citation_author=D. A. Laurie;citation_title=The Pseudo-Response Regulator Ppd-H1 Provides Adaptation to Photoperiod in Barley;citation_pages=1031-1034;citation_volume=310;citation_year=2005;citation_issue=5750;citation_pmid=16284181;citation_doi=10.1126/science.1117619" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. van Heerwaarden;citation_author=M. B. Hufford;citation_author=J. Ross-Ibarra;citation_title=Historical genomics of North American maize.;citation_volume=109;citation_year=2012;citation_issue=31;citation_pmid=22802642;citation_doi=10.1073/pnas.1209275109" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=J. von Zitzewitz;citation_author=P. Szucs;citation_author=J. Dubcovsky;citation_author=L. Yan;citation_author=E. Francia;citation_title=Molecular and structural characterization of barley vernalization genes.;citation_pages=449-467;citation_volume=59;citation_year=2005;citation_issue=3;citation_pmid=16235110;citation_doi=10.1007/s11103-005-0351-2" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=H. Wang;citation_author=K. P. Smith;citation_author=E. Combs;citation_author=T. Blake;citation_author=R. D. Horsley;citation_title=Effect of population size and unbalanced data sets on QTL detection using genome-wide association mapping in barley breeding germplasm.;citation_pages=111-124;citation_volume=124;citation_year=2012;citation_issue=1;citation_pmid=21898052;citation_doi=10.1007/s00122-011-1691-8" />
<meta name="citation_reference" content="citation_journal_title=Geogr. Rev.;citation_author=J. C. Weaver;citation_title=Barley in the United States: a historical sketch.;citation_pages=56-73;citation_volume=33;citation_year=1943;citation_issue=1;citation_doi=10.2307/210618" />
<meta name="citation_reference" content="citation_title=American Barley Production: A Study in Agricultural Geography;citation_author=J. C. Weaver;citation_year=1950" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=B. S. Weir;citation_author=C. C. Cockerham;citation_title=Estimating F-statistics for the analysis of population structure.;citation_pages=1358-1370;citation_volume=38;citation_year=1984;citation_issue=6;citation_doi=10.2307/2408641" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. Yan;citation_author=D. Fu;citation_author=C. Li;citation_author=A. Blechl;citation_author=G. Tranquilli;citation_title=The wheat and barley vernalization gene VRN3 is an orthologue of FT.;citation_volume=103;citation_year=2006;citation_issue=51;citation_pmid=17158798;citation_doi=10.1073/pnas.0607142103" />
<meta name="citation_reference" content="citation_journal_title=Genet. Resour. Crop Evol.;citation_author=H. M. Youssef;citation_author=R. Koppolu;citation_author=T. Schnurbusch;citation_title=Re-sequencing of vrs1 and int-c loci shows that labile barleys (Hordeum vulgare convar. labile) have a six-rowed genetic background.;citation_pages=1319-1328;citation_volume=59;citation_year=2012;citation_issue=7;citation_doi=10.1007/s10722-011-9759-5" />
<meta name="citation_reference" content="citation_journal_abbrev=J Zhejiang Univ Sci B;citation_author=H. Zhou;citation_author=G. Muehlbauer;citation_author=B. N. Steffenson;citation_title=Population structure and linkage disequilibrium in elite barley breeding germplasm from the United States.;citation_pages=438-451;citation_volume=13;citation_year=2012;citation_issue=6;citation_pmid=22661207;citation_doi=10.1631/jzus.B1200003" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="The Effects of Both Recent and Long-Term Selection and Genetic Drift Are Readily Evident in North American Barley Breeding Populations" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3.cover-source.jpg" />
<meta name="twitter:description" content="Barley was introduced to North America ∼400 yr ago but adaptation to modern production environments is more recent. Comparisons of allele frequencies among growth habits and spike (inflorescence) types in North America indicate that significant genetic differentiation has accumulated in a relatively short evolutionary time span. Allele frequency differentiation is greatest among barley with two-row vs. six-row spikes, followed by spring vs. winter growth habit. Large changes in allele frequency among breeding programs suggest a major contribution of genetic drift and linked selection on genetic variation. Despite this, comparisons of 3613 modern North American cultivated barley breeding lines that differ for spike-type and growth habit permit the discovery of 142 single nucleotide polymorphism (SNP) outliers putatively linked to targets of selection. For example, SNPs within the Cbf4 , Ppd-H1 , and Vrn-H1 loci, which have previously been associated with agronomically adaptive phenotypes, are identified as outliers. Analysis of extended haplotype sharing identifies genomic regions shared within and among breeding populations, suggestive of a number of genomic regions subject to recent selection. Finally, we are able to identify recent bouts of gene flow between breeding populations that could point to the sharing of agronomically adaptive variation. These results are supported by pedigrees and breeders’ understanding of germplasm sharing." />
<meta name="og-title" property="og:title" content="The Effects of Both Recent and Long-Term Selection and Genetic Drift Are Readily Evident in North American Barley Breeding Populations" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/3/609" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Barley was introduced to North America ∼400 yr ago but adaptation to modern production environments is more recent. Comparisons of allele frequencies among growth habits and spike (inflorescence) types in North America indicate that significant genetic differentiation has accumulated in a relatively short evolutionary time span. Allele frequency differentiation is greatest among barley with two-row vs. six-row spikes, followed by spring vs. winter growth habit. Large changes in allele frequency among breeding programs suggest a major contribution of genetic drift and linked selection on genetic variation. Despite this, comparisons of 3613 modern North American cultivated barley breeding lines that differ for spike-type and growth habit permit the discovery of 142 single nucleotide polymorphism (SNP) outliers putatively linked to targets of selection. For example, SNPs within the Cbf4 , Ppd-H1 , and Vrn-H1 loci, which have previously been associated with agronomically adaptive phenotypes, are identified as outliers. Analysis of extended haplotype sharing identifies genomic regions shared within and among breeding populations, suggestive of a number of genomic regions subject to recent selection. Finally, we are able to identify recent bouts of gene flow between breeding populations that could point to the sharing of agronomically adaptive variation. These results are supported by pedigrees and breeders’ understanding of germplasm sharing." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/3.cover-source.jpg" />
    <title>The Effects of Both Recent and Long-Term Selection and Genetic Drift Are Readily Evident in North American Barley Breeding Populations | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"1_Ik2QBBG_PKXPI8Z51ROlucLhvlv2C2KRloe8IyJ1o","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460099","apath":"\/ggg\/6\/3\/609.atom","pisa":"ggg;6\/3\/609","ac":{"\/ggg\/6\/3\/609.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/3\/609.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/3\/609"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/3\/609"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/3\/609":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022tsHFjaZUaPD\\\\\\\/RSgnsZmtrU8jMVwv9n0tWGVDILsGlWpFSY\\\\\\\/f2dcHGWx6U4HdtpY919rfhMXudoQ8adaZxyvxCVe6Tv3EOCyb\\\\\\\/isNSkRHhIWsxh+AiGFTu3VKFBCwX\\\\\\\/HNkDe1p91Dw14hGFBtDq2qXqcNukJYWuk9zP9Oi8Sf\\\\\\\/GKQZczu22ka13fVSQm97gBtC4YADiDkeTO0lpgbBNA+1KbIm4jQr9TmxeIAN+bkH5PgryNMpmhnwOiIwDyJzeLmKUJqUK+vkJNuGgICxdFqc6XZ6t\\\\\\\/hRsyu0YFy8WQLdqOBT26xXrJYubMuK4ykKQRerNBjd3pAI0MBzvoSN4WlFuCor+3gm6spdJz+kRPv1JenoUgzNIvSwzF9oC0ql7Ks+db6ucaEwR5xo7egO6y\\\\\\\/8eegv8RGJE06D1akt9eHWyu\\\\\\\/PF92vfIanLrtv7yG95s+qC64QEjv+hNELh60X6vubSZds5griwjDyLTV+JSu4Se2C+YeZ0ikbFXALU3\\\\\\\/Jq35thSG3hL5eAO+cIyzBbWY2k3N5\\\\\\\/VPm83lXDm8B\\\\\\\/2VDunbMke04VhLpXyyAumHTfFRamoiPJJcwkqmNJJb21Gubrn8G\\\\\\\/B6jCgrojAKeUrbZ6mGh5LgNtaoFS9S84vNw7u\\\\\\\/5fwzUyq\\\\\\\/QJNffFqZKMsgyx7EkYwfK+0lJ6a9\\\\\\\/4qEK4DkRGTtCSxFiGtkQqewanxi5yU6xFW1A2S0msBapLNOCCuk6RW9BJPbThvt8Kgg3jrbH8NGw7Hpmg1NTXALXhPIwdZUkHQZSf2hMuvu8xV+CMXWHfEIiy1eY7Sdsyli9tP+HAxGgBWJShJM79a9uUPnGnoOT6GwT6Gx39Y0eifbuDzyNxTf79fsoPa8QCDbJKmpC\\\\\\\/iNgKjSxaAuBiGHgY+o9jQ9zhJHj0bL7Yh\\\\\\\/JEUWc+UMyQFBLOYuXA7XavejKNFsDTN2aLOrxY9orMgTWgcD9hctdj0+6+3mUk5WWIht\\\\\\\/9Ug4zDkBGGShrL3I7JBJOTLCUBdGG3ICthusnlrDl2ilweJ0NpVURzP8Kh9ItDA4K98posv7uEBstmdJlkM4rw6HIvCWvesZ\\\\\\\/\\\\\\\/AZ6AOkORuKzthyOoJVmeGepOVanHf4xwAct1uXTqWtoYEDHPnUf\\\\\\\/T8Mh5SDd9fM8X20LVbO2mxQZtG8m52jPVYvPlm\\\\\\\/IALbuSKNTDXNBaoUKLxO9MKCBdO3FZJM95hYILOmWSzHGCB9cs1VzrzxGP5vm8WTuJu+1pbY8tWQGx8TNTJwL+KS8asRMi3EgKBEaixBVLUJ+W\\\u0022,\\\u0022iv\\\u0022:\\\u0022YGuD73Ko6sLwZHi1qpelLQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002231055416b09f3228f0c098d9aa83311d4d8a7f94c5a6d3a7eb18bba1f79cf695\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LqkBeStoaNIT6hG18+gFRypxrSXGdOozNzcSGgNwk8W+i\\\\\\\/lsqIuyKyVoJByrMZSwecWFX3ExmMJlhzRCUDijMXAIiZA6I3ge7B4V1iHV2Ev+mhYR1cJX3fNDw8PJV5EgnbZ3ETedp+dfw1\\\\\\\/JIcX4SOwBE6VrtdNzHtfgxleE8bwywg27TeHAsVZ+FgpMac+WMhWC4WMFj5R8bZVbONSEqUf4WnhFJ8XCw7mYF5wdrQboqoso4BvwMXAymOCfnaIsEfOTEOoIBVmMCdLOQDzxr2RGmGOfj0jR8Z7bpvnmXK\\\\\\\/Hoh0AFzt99h+Dcv3rRK5BqSltPHKkqDjKlKquybXMZ1l3SZvR8JLoTXfWtID7w+r0tOEQpQhWlzYhvZXX03zz2V1WpccqLHSamW3eOoLW0HlNdpGn55zNe0ubmhMLyBk5O6mCxAwqnVh9HkKH76IVgAKP3mPklnKnpt\\\\\\\/YhxeRKa1ThCeeABk9hu7+O\\\\\\\/xDJfVWqzsxX+q00t2GL2sTKJTdjTTyMQLLj643IWIX367bFiHZj7dnNciR2vOVjsBnQsir+aTQBXe+EWOPt5edX4EcXjeidQFPK9x\\\\\\\/KD6cIoDXtc4zt4ukAtq1+mAdjfrnD22bFwIMB+53gYPxs6QEh83rwF09nkLTmjjDT4WU6sSAW\\\\\\\/KAXUK54xibPMiO7y\\\\\\\/mbf7ZhcUh3nJ3RzjDvxJQzGadkzGbmd\\\\\\\/oKYmwv6PWih+O27oeNqQF+e9pQJC8vOupt+25N+xU7DyUfUHvwbTkSZg8rjcyF1VqCPi0afHrz\\\\\\\/Ei2ZFUHuljg91EzWKw80U0wjD23txLb6igH3PFD3M817oORlBMPlUuVM7Gw5L8O+6Sn3lMNgFiXspFmlNrehIT1vdoIdpcJBVCVciRtevpvLzVuOjmMHooSWgv4lXyJoEHMfAR\\\\\\\/nXV0PYrGYw1dzRScLjT0cLOmTAovIBQSmxAYI+zCdhkX5V8wDtwRzuVaw2jQx6pCsSdDcwtgW4eClvN\\\\\\\/nZPnSrOVVqgoCw6HGjfnoo44uWM7EsmtG7mnoL4dzRDqK3C8CkNFVUs1CZ\\\\\\\/lrJCUxtKnT7O60HEdubybi+76WSNNDTdjSx9QlqKIx6WyA5pEG4Y1o2txSkQVdgc8zAvDl7a7oIyN6PAw4UjVWCFhAzODf4Z24IIlE\\\\\\\/RMo6AZRwzCbxTwF6N+KM5HsS84Ip1n1rvTWPzunzgY0Vc4i6DUK02m11Eb8KkjBGB+hyP6rXDIq9TFwvXnFQwSPP\\\\\\\/z3XeIOiXUozKO6kNx4CUBhwejr94F9lrQvekTZnOzRvACeiCMV6+uTPLVTkm+txptN1v7ILN+jCipdTCkeEw6yBau4HQ0elYYwGkMjD4+aaPX4m2OsD5RUOgxAINwCoi4XXN6l7n7bw+fe3xyUJ8peObGxXXGvSBQIN13LcrDINuQYsfH0bk3WZgIIq2ZSAIP27Xk2ktSn5DhvvfQ7iUmDlmPLGablNI+t99BUMjHMYDA5lfhDDMndP4YduDLrRYk1PoK7qHN172r7zYLYyU3z4RLEpoMbl0ndzEqcxKI8XSvbOeqxWcEsYCUeK3etl27wAgc23Vmwc2LqrDI2UeWy8UkGdJBr3ER7yWYk+gq7AAt+6AkxojqldYDIavyK9v1XHSrtvWCXWkxhbC\\\\\\\/cv7jv1gDh6ZXK7\\\\\\\/Fzg44GcysjquQv8td+CQkat9rqWE42cgkst8wWETb+ou0aaQjbZmmJiuc0yT1rBSEvHT\\\\\\\/p+yLIrCpbUnNplW4bjjBRoQsU0bg8vB\\\\\\\/lqGEzdfIdg0Ms8ZHPjLtRAx0HNuhhoag9kKwmX7CxjtbIwd7S5kR5bP9cbxTdMDpMp+UPs6XO7f86tqR2uW01y7X+XUQyDW9EaPh16DeCr6C+es2dWG5\\\\\\\/aQg+y0ReMSNcFQwz3oE1UrQScmabidcfBIcVmd\\\\\\\/ERdjFLeJYAnrmQ0x4Ua18zmGQjbq5EO5EqWYUCSG9OZjOO6QkYPrQHGpIop0YOe4MpFeJcaGagm7TDt4JKApx7zaC1CrZb\\\\\\\/h\\\\\\\/oPQD5+EqrmMBPUxjRLYSqoYPjkEI4RRcx080se\\\\\\\/VP69\\\\\\\/4UrMZ32\\\\\\\/8J9\\\\\\\/BECFxLfIX9GlF8ZEcPeh0MIUtoI31+nsQ6DWhaCcYSpbdecWwRPcqkpj9V2C2EbzNceUapnV5oRO61SoycdWdCmKlE3pnq1O6ceGVhPVzUEBbNZqEbrFmLV56lH6VBDAkPeJ5MmWfSjnRKCG5VFfddesQPxAe+7SAWnzEwcLNdgh6YChhoDX3uW0AsZHM7Zh86GMG\\\\\\\/aGjNjZT7tW3\\\\\\\/qgGOFPFbpPkBIhW\\\\\\\/txZ9yhrqeJigMveBF9WvI\\\\\\\/2wCfapCBn5GFcpQHHkxHUZ\\\\\\\/GY75i3nQYn5b5eekicK\\\\\\\/OdsUOdZYCvb1AJgNiIZ3uRduYIp1vlGNVjCDvtV6Jr3HOHPmDpCh8c\\\\\\\/Gkupmla7\\\\\\\/AkWFVWayZzvL5YrrFG9kaO0+UB4rcxBiAhnwZc7HAkKZCxVOGbfE7xt482hI3B8r\\\\\\\/deESSHMoZGVBPVk+poBKbPcj\\\\\\\/9s33LPzsu2fDqxaIdKmh8r9EgQNhFpF\\\\\\\/Iney0j6mxG3BFhZ7ShBaS9YL6u\\\\\\\/yqkwVBIkdMddEiZU98257M0X1nI6gWAPpwO\\\\\\\/ghOCfqtXCk85uKAfayvUCNUHUwO6oqmpKi7iSABkyvtM+BLYCnigKVd17pZQH8JaMvlGYVMYIzl4NOukewDfU3iNBee8P+mXrq5zVOe+fsKabXpT6X+oNGLNsO5op6VcGzEjfDtqlH6rdVBJKI5bd0ksCamYT55osAMaQIA7V7u5V2JZl7GpUIj8Pm29+NWIhOpeVouvMq7Zud7IhFCcKtcsWoi8PTq6xK8V4qZY\\\\\\\/WLi2aYehSlLAzHoeqcTekRuPFK0VyQ1jO6njUhYn47MDycAVNX7qLTXi89C4ousyPgeWysjLh\\\\\\\/6eKwegOP1Ogr2Vb0MCqoGlNhOZCH58wfR8z6ZyCagBV4SvLZwseDZCBqmXtR+1GicH+jdjFttYGHSQfuIRKpgs\\\\\\\/pwO4GKm+Q\\\\\\\/qxKMnXcO5wpWrG\\\\\\\/A0jDWw1RT09P6L0L38Rf\\\\\\\/h7X3rRI04v6+niqc2VWxQEMvULbaqoJmMRY31coqq6MXH8gVVjtXz0HyIMg5OSkZvlGU\\\\\\\/bHouo9Fs\\\\\\\/lVf2MLVAWmCXBmsBgP7P4jqYBBo1VKKdQ7xATIrNTvQbktFKqYBUpD9DW6mvf1wcNFX\\\\\\\/vJO6rK3QoMJgqqE58EKF0ylTNPuIYlFOVKOeNKZwwp8ab1KPFujRmrPdeLl2KZ6f7mAGYwhh18a2QUkVqcWPyahlhSpjsMABnN0yAzLWWLWiJnbK1a5yG729QQUpRyEuy6FDkRwlUWg9TdlzXD0\\\\\\\/YoPze4mfx8XhieJFyY6vbmMgtMwgSgVttygkDWgCC1JRI1W\\\\\\\/7Tgz2\\\\\\\/lRaISm79GvlK2PC4p3L2Pxqhi3vRnfwWrnIRLOhhfqH6WZbNBsBvUVw75akt4GeJAdqcB5C6vM0ObYAlVy5\\\\\\\/aQ6QGORG4zsmFYL0Er2k+qEpAGjvHvoitbRLWXIM48gKpzeb\\\\\\\/ZWRptRGSOEG9LxUL\\\\\\\/YXn6fBnQTQ2xbGJj+8VpYX0LCWLNWbDVwFkfEL95re1PqgP9OLeRUXy9aAQFH6h3+d4a+OZvEeR+FhLuceHTFarMBWO2oGNm3CKGts\\\\\\\/l09riW4cGlulD2+GApjl9JNBqCJNmU8HWAbFsBnNFFLpOWiBn9RjR2630MlB9WU\\\\\\\/Jz3P+sIx0GNNsMPiLCZMkbKApCG\\\\\\\/OHFTwWbGnfT+Unk63TJgczPUzp+NOy00BdMpfhoWe7SBAH2a3Qb7y9syancdKNvtJBDngpJx6prVPPbnMqKtzmw29GEfQJtbt0fO2L0swD7zuCFiYXSvDN68ZBxAXa6U5ebdXGBj4yfYRaW+xpE8tFmOsFENZ1RRYlXEjHRgxfbJCo8tC6e7DCQRTZaWQ5hxKKIoKGbp8IGzQ66jfLknYo6drEtlaBsZYT69GaY6ic8V\\\\\\\/invQSOT8DgE036dyatFRxxk3FVmd1c33JivA2EkPprIUGaxtrOtennwPgkAXil0tlDw2YIAOVwlIZ+M\\\\\\\/8RTQMzxnpaf9L0rkDGcI0UEN9bTSeyProKSMWIeOU9BUpERT81atQhvRSHAyZd6jepVsUnY5FgSBKlJ3i8cm4bbkvifagF+SC75eDntj4iiSdx8iA65S++rNne0878a5Hw\\\\\\\/HoqUEVDB9cnxQz1Bv+E2ACp4naT12aVIM2lEkPoLpHp4W0+3K3YzIf2if1acqYnRq\\\\\\\/JMNpLgRBzvzfsAezyjKgdWG3nlVLXJ4KPgbZ3YEXoZ39d3A0\\\\\\\/jIkAWQHKcLv8Xd6UGaHMg0HrE1eMMdoeqeQfIQ54lmOy70\\\\\\\/FS+EBnAhO8JPxmE8\\\\\\\/EuPPkVnICnvUODwUlaMOMzX8RP6rN5uHBNm8X7UnHCp6vGOd5+3rQLF\\\\\\\/1eWzL8OiTWnc9XxNHXGf9HnFPInTaq\\\\\\\/\\\\\\\/ErOhNcJPdbn6\\\\\\\/\\\\\\\/13Nk288dIoYwVWEePPX3EF25LxW6TszsC86Z6aJ9lRGTAKl2BBFnwM+qAZ1iNh7gTOLZsUFNFZNtSW17QpCKDev8ttM1lBrZt\\\\\\\/gjeTyGDRr7QNb1apKdHYiGNDaFMLFbWl4Dte9KlSFgPa3UB0XQ6g+pFcaoawiTPJr1\\\\\\\/Vgt+9xdYQaO\\\\\\\/cHH8y4F5JQrnh\\\\\\\/X7wcAAEqenNcT3eiXDtUKwjAmRYv6pBVBNFle3kNfHXH3PXR0vvuwoEchr3YpcTCLqRNBq3US0U7f76Qw\\\\\\\/tSM7wNGHGHC8G6C8FrSA1sObtc6bFCWnr2zThRZfkysJXBxskWlSsl0m7\\\\\\\/UK8SkJBHsWMgomtD3VMFfcv53gPyHZqzUlXRtEpn45UxcR6YdhhkEuJ2\\\\\\\/s\\\\\\\/haIB7Oj5\\\\\\\/8QtF\\\\\\\/nf1tNZk5XkkGBoQ5OvRZTOfG\\\\\\\/MPZMBk1d0YnpX2yllttEIdHyBH6uXA5gFODK4cr7RB0aY0BgYUxc6YFWbG64kt5Hvs22QwS60yPpwO+OsWN0X3wPtXH2++UPqDWCpQzlhU\\\\\\\/FeD2V0EMagAtBglEQH\\\\\\\/arxCV99zrA\\\\\\\/gRyZ6a4OFIN97pFCulKsTw\\\\\\\/RzZmy30UZZHn748keE\\\\\\\/vMX+GXyd3\\\\\\\/WZJz5XwNNISZmge1zTKvKtsd+j69luKuN+CZyqrJBq7TpBJ6KWl2SWbyGNB++vp2hgT5MOjA7aIiTZ9SnZHL8rI0o4O100hOli4jS+lPkK+Vj0usQHR4QHZvxvVvRLB7uaEZsDChQLFZAcuOiu3m5+v9NXrSFQOotIPlDQfji\\\\\\\/cXaHFzNABtcO+k6zEZj\\\\\\\/pboHdqFSomU8UMcDzmxQY5C3Sui9qX7n+3LH0CncyEqXbAL\\\\\\\/poKfK+g3MvmqHPwmGXhdOF8nK\\\\\\\/45az5Tnwfj\\\\\\\/ptVFhxisUUJcOmp9ZAcmMYAAQvqD+JPE3RBeASm0qsSp44MonG+SAiU1dgnyQxBoEuzf4V4n4PZfSR8shnLBKoVgEECR1htEx8YcLVkRDBqZjHZFTSbWsxu79gQDuOpQuKmA8B9ETPBDWA\\\\\\\/wGyNir6G2U8Mid3CiorIv\\\\\\\/g+5rwW\\\\\\\/LFtfFoPZ3v8vdTtMbX2qggg2tLhRHiYNbb1Oa3AiIpJs7QC2vy8sbIbHWDchtUdsOH1w2W33r9EYBEIY8+qF8SOA8qa7vcvYG7qUnMFImM2mnpmdQibHICwPZcxkK8iNJOM61Qdw7YrN+he0MPon9EfCkDxJveSLtht2wHg3YXv0thprD0HAoFIXpMsMaNLzj2SvQ\\\\\\\/5qzlJDAxbkD3zLVrlyAS788XImPq5PcFocMKVhsA1VVpuJIIbGURCqginvn8oV3q8w8uVN1AdIzZL6qf7SQoDALlzAe\\\\\\\/\\\\\\\/qbolTI1r66omf31B5kNichn8Ea8JQdYXlx6wJKyrdZtXqEYdutgtogqHWH4+YsG0jyFZeUO6zVaZQNxQ6oOXyX0gil4qfRfk7NAor88gK01kPSOlaTLBWLZudh7yy7MIV7F+bknH727YdSmnn3XL4uMihmJ3NKuGm+hQBksrKx8ut35zfMJll0uCRh0ZdUhtXkeOURt3uWj6nHbNv0t8ur6SmzlQmvUI0HP4X\\\\\\\/CMBO3eOENEPoXstHIskwsqo8lIrNotmF7nN6HNQLzJH6uwNvv17ez4QELfVtmN5PkohOfkcheLLIL+IIjbd33JEs\\\\\\\/D6L0\\\\\\\/CLoHnjCvYmT+\\\\\\\/k5Qwv9swcAWjA6MCBOCxP\\\\\\\/D0FAOU5Y3DJr2mAKR+YkO8QGQ6KPob4+EmbD\\\\\\\/jw59RjWZnSfPDb49Xb7ajOq3XqtmZOlVpL99TqBPgDsr5EWhOBtu+60T4u\\\\\\\/UkLxxFwWxLCeSNof7gizXuyvZftNfeqTOHpL9EROGs35eY6BXDJKIvoUX4BaN9wMTcwTG1Vqkxl0tIYtwEVR8V+dXZ7x8kFu5YAC4C4JR+A+1q1cW2cj6gKYPJfLOiuv3MgGYjYTcmd+iV0NBDhUeevUTNzUFgo+475LqEiMiqVLrypoY1ZnunHPZQ6Q4uSCooVOfn\\\\\\\/nafBtP+Q63FFYgiqmiXs1G2qgN9R2utpltcHfWmz+95h+CM7mg1AqUH\\\\\\\/xmcHKhXq3ehQIxlId5nMXlv9sxfFpSDPpLePNvhUroLs\\\\\\\/ksRYHa9Dvt0\\\\\\\/vIb\\\\\\\/l0CFDDRlmR45ZMpjbZTLdEGfgAI7flrojSdiQRlhLU\\\\\\\/fOVvK7JGylePiENMldCAt6qQrNFdByn46\\\\\\\/iXKYBAxPyG90QJ0ZW2F9uaL\\\\\\\/ypgxXE5pbKrMFp7rrPj6eMtAt2Tko4U5OMt31nAfaaCoWlfqugiPfKvqt40TeRXbkCrghn746h4ot+qNU+T1hOobp9F3gATDv7t0AMeZwIgsd0QGE\\\\\\\/LFiPODFkivTalcQh3hWajdEI4pumzJTs+b8oH5uYRoQS1ZVusbKf6QPS0IF2fB+ARdygnPax8mgISNaLcRYb\\\\\\\/Vn8E9rcD17B+UR87tQ2oUQTxOhKr5Gf2HJuAIznXLwNJqmF4Ao4jwwYXQhL0lYGC3XKvNJje060+CqX55nARwXI1dTJi5FIjzQsyr8OcEI4QquxFM4oPWwuDM7mNWrbO5N+1qwx\\\\\\\/9SLHF1U7UT9eEjqCppWKd+6vHBuF\\\\\\\/Gj2Z0IfpJLhIavlsNdi7grDLaHMk4sihJ7mU1XAB9ywbZJ6+U1x8nfE6ANsudxGDbyzH59uxjEYQ7yORlieJ6T73cTSmVvuO5ahSD7E9p+702KmZynEqy2dKXDoxmq3U\\\\\\\/A8LCXNBnzLMN764+X3is\\\\\\\/bYobQwPK1zAA8A8CxCEsrj2TK8pjzGd0sUw6FfySuX7l2LPr0K2qO6EECwGeJXmYJAb0KhgRV20IPCZ36qTBkHRYj9gOlUAbne\\\\\\\/DbMVlgMYZs880xy\\\\\\\/uSMFmTaeyBpfn5XBTSQeT5AfCL5u87IE\\\\\\\/fATs5laUiyCun7LX\\\\\\\/I6viZJw6qZ63sveMVnfbzHINtC9OpW3wYDll8aF\\\\\\\/euLIyzoQ9kF19E8w6gSNY\\\\\\\/q0Bvhq05XyHdz\\\\\\\/IWPb7Bvvw1AduKqCPol1hcTLqRgCYqRwBN0WHcBTDGQktinYNt1JsShW8fqat7+UzrimggPAMF+oEBSNEweIZ8QbMtaca8B0GuR9IiPvQ4avkyjpEVZPfY4fNP+MxAuzI44r592lTySv52j7sm1Z4D8lXrATJ5T9nGoFz4EcH3J1wujp9DeDAdy1cRs50GfoHADUcrTawoOPBovgNS4jWrkE12VBriz+mLko9LD\\\\\\\/abYw+I3Wqy47qjS1HGDtDoBRiX2fMGlS5x5odMS1myiuemXHDuBsHpVOUYwQO9QRkmLK++d1MrHMkytBopGokgQeJ\\\\\\\/AfQOSrSlZmGZRwacSiM\\\\\\\/FRo3wlJQ9Q9puMYvmYyP2TZeorjo33OQ96FOLb5icbWVIwWXPktCBOv5z\\\\\\\/xheRiO4fxlx\\\\\\\/RUYS+UkHf1mOQXBe24IvWWGiiT9+9r1Ec1nyle1j89GZXf14GcRE+r1emaXzYHYBT0dnWZ\\\\\\\/t1Yi7U3+oLogTKKYzxxHV2+p7oUNNv3IyHyFD\\\\\\\/pc0T\\\\\\\/cP6oyjgyQTnELXOaz43SK1\\\\\\\/HCttUshsPyZRdMfd4o6Pii\\\\\\\/H8NefrguT7uK6FJ4hCmmBsiezZcRwpZR+9gRtl1SLRCmQ+gCaJCaqt\\\\\\\/a0tQ7OsOoqnoOyH2AiBqMK6ZEItzknf4s5KKuHaSmhWlio91TACxkDe+rvHQeZpKPAnrOyDEUnMAqwwiAmqLG2O4cWv8Ur4myaug5xwzPWF0mplPav7Ea1z3kOKOtWl\\\\\\\/4fmG1h2NzDZyhQMtcILz7jYKGuoEMGBUucVvJuWG+DBggrb8HN8ODneFzCRv2rvAL\\\\\\\/\\\\\\\/ic+Hol8y4zEgMlOv0+TcVIqrITrxZ4BMH1IiVOkgrqT8O0SqC\\\\\\\/oVxMJx8mu72\\\\\\\/ZUe\\\\\\\/9K1u3zuSX+jp4qGYGsgbpBJ1NpIkp57FKpwppSv55UqeSw3oqwXxqSAzy5tTrB681FgfXL0zHIAwgaL5v0cvGdJb4mD73M4faQe7SU9Re4CCC7jkh37aB2Z+MB5jjz6azolAiBN1KiwtH1ChDZzI6BqE7dUVT5vljZ00vHfbOMWsN+YGuwShUzOz\\\\\\\/Pu6vmyXvjY41ciLhFkjYVF+lEtPXDQTtSefcZRr8m5LHwonfIr6ZbWTGMefJ8PBC\\\\\\\/LdndfvfnKhoDU5lah2cUwqmbXNhba0i\\\\\\\/7dDQplyudp9Wv3fzVP9baj7p+Tg0NV6Im8lskyyLQWtYLMzNTneKtjrpm4nLvs+B1VmCoe9V4NYgbXTXQ\\\\\\\/JlRQe64uexfmBjA8AVJQRjwcyL9owKx6yPtK+Mty2q3um+hbsEsqh\\\\\\\/wH6BKVjJ2FtzVpU9mEEe\\\\\\\/hBH6XwYZqZHZI8wNDty0DpbZ1tDuTKbYGn7nyB6RTpMVM7Tv5mxbOK\\\\\\\/5CT0+91bH8Kr64wbeXQjAKutJUa440UdBSxfqdIJ+c4uuMT1vgos4HOaKPL1to+lHcnyUl3AYZIAnhaP2q5NzONsTnwYmD2Xm7NTdafA637e+4UAzyKllAyw1nWCQaq0CJ8qoa70\\\\\\\/LK39gEjVjawhYF4Sgamt5A\\\\\\\/rKBdoJivb+goP11d7GFQN\\\\\\\/EpT8SqtsqNx4o0ouK8dwtjWEqk2DdM68TZrNw4agDcJGsqCncsAXYq2P53JbAaoQnRgNyciBLhHV0jmHBOwX1oo4MkEcZ9C0S0nB444hEWjJGLQRJcTQxjkzsWmOUMdQvALjXGjcTn7fqVn6UC1b3Nv1CmX04jGDXBxBmb8NsOipsiakC9cIPozro9gMYLk0VAF3TgtpIJaTQfqo9n0RIIjRTqTEFZRzoIiMqhQIpzDM4+fNfrNdnnbmsB6isYXdwSBFo1p8cvHijqHqb+ENGCr+1qDmZ84oqiI9vwIO2TYguFRKF62zEsmho3CH62Fvkal4+sLo42QGz1rYkkBOWOcl2NctDd3ifjVws+y5C\\\\\\\/qcw+kernVV2XP78Qn6On\\\\\\\/aZWrq7fOeWYD+XVfvA+L10hDvdZEmWjHYCOlJuLhrSjF7oWFqDLX05kI7HR9Mf4sm7+H4RInvvWdBxc\\\\\\\/iUOF2v8sotgs7PXLn+DKjjRAiGPxsxZ7ZdhrAL80eBegt1teYQvJoI2DkWkO21EeFrM0K2QZJQ3p6VOIXsIps11lJR2p76XwVqrmAdFxaFG\\\\\\\/BO04qqzucxfd8XfxpOOFjULoXk9iLArvxqfpzvAjWgawr3bnyZWSlGa3nmaujlaafRKGx3ZTk2vKt+gxlBITVVcgcwmgZGUK0vEN+KgMmn\\\\\\\/LdFVKLt8y6Y4+wvZQcDDeK4MOlcsRoA4GvQGKFlWJUW4yASUcn1TFtmSZK9coWTyCnW2HkPoFBi3BOro5KdjzkLsH9k0JnMtsts70269cr1MqDohnwTeflj6\\\\\\\/FAsCQf\\\\\\\/a20HF7uK63jsGswYo+Ss1S08kbYdYSAcvfwMDA7va1uDuH9RN3Ahm2r5rFPD+6f\\\\\\\/Au5aHVZJaXq9reISHVfQRU1gkatZUt94yc9EJlhlgYyHdqAlv79qImrxNKOg2R7zpviKQAjXM5WsKV\\\\\\\/GKtXAhezsp+TAIEMj3kuE6aYCD\\\\\\\/WK0qOYlrc6iFn+fzx8AgkYFdV4zwj+kZ4qv2ib3JdKSUGf5hvOiyGfXgkEMrGr6jh9id8UR\\\\\\\/x5OB\\\\\\\/TjrkPUXF3+hK1jxdmUtdiX0A+kiiLDk8NOTnJ2z7hDD5m\\\\\\\/99\\\\\\\/L16YHFzm1TQZiv4dqX5GKhkEVbZnqwJ68mOzh7GXRBila3abQoZRiv7hd1xZjRpZKwh8j3ltqLHRBwKlJn0+bajYJXLLjNz0KQcj3D6Ue98FIDQ4IMxSiyQ41yZf8KygyTDQihZxgpjgaPXLQz3LP4xBpVlbx2AhIMVHiu\\\\\\\/WTfebnhNPXna6WOgoVzcXz\\\\\\\/+wDI9MwaSmW\\\\\\\/Cfgr\\\\\\\/81Ppw5No5WgEV4weIANJq3AcPFmj5\\\\\\\/dTRaf5zczPdUsrLDjMAE0MrkQc9hXcYCU+dFiZxUl4wH9CfTiFZ9vwwSdFvn41nV3oba+mlbRwPxI09GsBzEeoyDXGi7d1+6hbSmtjR2Pb6Q2HnS1noy6K43OYV0txlTm4qmVezPCx9cdFhOEJrmAkhC045S38wHAPPglaV0HLCafSMfAZ1t51EQerifieDzCIt67JeBqEy+7Hf3PMTJvX1c9sQHJ1R28ARnspdr9Xo5dFvXuA4DhsKQaql9x\\\\\\\/s7L8+cfqS+FhTMUq5HI6BR6IO4hrYKdpM7gSvq+0P1t9ZDRaUaD0gw9uFjCoz+i8o1ew\\\\\\\/dfuNuih0vw7vauNSsEjfLorjB5vM6Ovq1fdDxBz44t8WtYmtFNE00laOXGlSscC40EdrZqQonYm2qFMfg3yYk9ZkgUINc2pUnMlLMnvUbI7L6mGug27HejCuYsFGhA6grV2abVyaOK2M47wjYelofBh+XIl\\\\\\\/mUdB\\\\\\\/0J0NWG+DBkrcF9jZTJnPKJQhsQbPr8UhyDsw0UUQRldKmuivTaI\\\\\\\/jfMxLDiALiqsAWVDIA8Z7lvxXTeWfos345rs3lB7gJeYYUlIF0fr8N7ZRx1URs28X+LmGFUUQoDZxSCsk0zv\\\\\\\/+KFNWo2AX+XHchGw15h+R7GpmVbQtyTloM4ht+frLjUHrHTY8xXLAA9mlAUvsEqfwkWzezT65IN1vxJabZPBRDQwzf5\\\\\\\/fwn58USsWqHj6NiihEFmiclm9OIKf9LUperxArtsIh66u1T2bEcLywQNXVkCHioi+btk4elOlko9ZKfxJm2EyQRRq5xVZecsamRJUN+9yVP9PzS6zmHxxO54obfLXxix0LJUnREIKsRLPafjHcx4BJ4sm4VOLjVS9oDmVqLKMJITT\\\\\\\/zlrsuJeRIcL7n8YuFAGpnBE2j3jfnPk\\\\\\\/HJjm72aD5KwdHUoEGannmcAyVuRq\\\\\\\/69tgCB1wdZmpQjM+Y5A7E9mZRES4DY9rdh9IfcNt2gHk6ciDlw+e5UJ57Q\\\\\\\/mZO\\\\\\\/G301E7XnydQYYZar7V18sPUc3uokbDQVo0IIc3wn3mf+nSDhy3M7RzuMbmF948zZTGkGRQSTE5G0mHLYaYjAaH5ETq2mFjCsajqjnC\\\\\\\/NUg\\\\\\\/NXmS2QK9HtqqCHfag3tcIPZB4t2vz5Vus0rVcBmqhOq0g5luYk0wy5TV9XBJ0q3CxZD1wNhXqpGr8xXwHz1r5bnNv0j19x2nZum\\\\\\\/oYgDYfrkV+iQVnBRct9t\\\\\\\/T0aFK2ABWI4nT5\\\\\\\/NS5kUdGsd7olOhT0SwjYTtYQ1WPwKTUzOg7vy2Zo2LZ5QRcyYjcJLC8O29BsmTXY+Qhsu5kgHQxky+innuGWbYd3XfBufmofCwmGNR1QlMgtTknaPVwVTF7Pa2dl9LdhvEge8g8Ua5thWnSgGTfqcXic33n3lPZvLOUjybmniN2yyY91VFUqYboEtTgzivrGNkNmD\\\\\\\/Z8ytwIZX+iXCpPVHF6zcFnvzhbtR\\\\\\\/uuuAWA8l8J0ntLOcEKz5bBrhjQf49ASL26M8A21G7BU4VY5imKM4Xp8VQ\\\\\\\/VUewOz7Vr7tjaM\\\\\\\/8C7yeoeVglUOKklXy8KF\\\\\\\/fJ\\\\\\\/1sQuxiocL7g9HCITjF4pDexB1lqYG3EQ5L0HSLUm46DmsNMRQCD3na+jI6+b1pbBzVPnxtiMsRxpGUctArqioWuaFe5aT0Oz\\\\\\\/osdJj\\\\\\\/DcqGf24IbGTSimqLQtvYS1tbnq7NovywsyNCCW4\\\\\\\/YSZgJgm6AHRT39+OVuDEoP2xOSQcKpeVHFrfmIJZEoLWc8DbqTAOCkdQ5v9nU9ckxjhs2dbFaq6zY0C0iduxT+V9wQPvxYmzQQy7CSbH1LqbHnUM8OUDt6C9+gDjkyZLag8qijbsXMABWTYANdo7Q9vFp3csWVpMkOOmPFdmCLZWpEhRXw8m5VGpHzmX8dHAZZVnGHt+9sv1FTiAZsrbWPDShCJtgHPn6EqAbtAIk+DkZhrJagRq+3idiitxKi3Hh4th8qddgkI4VFxs1DM+dLOrGuLoqs8Whp+UDWv7NW\\\\\\\/0KpBrkFYII511z81VGSrABysg1CcesWkayM0Q8dHoIA58UD9+\\\\\\\/zleQyd81j4mIRTrmu9mZ0c7s7IDSiqfwNnGARHbnC33ZHLkfrPCi6LOUwZALBjrVw8SQgYCJXU\\\\\\\/hfqopXjL3ANYdZi++jjU\\\\\\\/4koVX+AcQB6yRBy53M9xkiGsRDgFrW3xVgUYCTSvoKXd2tYryx4o8xaaQHrsPr2VZPvfinadFkTa9X4faO5xhNu52denVVyXw2DhgxX+BREKtmnAEqiUW9fA4E0074pXQJ5y9dMnG52wGb7jR7LcYPn5NKsZZdcF9FuJfGROCXMn9iN3+tuikzwxcM2h3YCAak9X3CktdK7taKrYCC2AKntGkkmW0NSgfOvVAAFX9HesS9bIms+eTo7dMZ+QKsUJlkpej17GrzkBsEw36xlDQpT0PDVLBRWnXqo\\\\\\\/ZXQgWLF9nMLpMA9TirlbSXL3BEJT8NJXAExsjCtNA10WVGMfLIsqJ2FdZt4HXStinCh2pHz42pA2NTQo48v47Pw7sdsROp7nvQDIjfJabl48VXndGSkoh2mX6y3rf5SjDr8Q+EAMxxfXGTm\\\\\\\/U9TdIr5OejxgjTVGzexTs0Us3d2LqGvnTGfp93pdpCx8WqZdWd9bnEzqL5vXGHEO5pUr9WaG6lI8Qyi7KM62qT7ZqVNFGqz7cu5MoV2mEAoJE3VHKTFutbTZuC1nqY7t\\\\\\\/YvrAM\\\\\\\/mXr9B7AnCEHFuv6ecKI4zywDOmJ1R0ozJzUQbYMwby3dffEcW+gTIdlo+4LQJUZVmaOts\\\\\\\/LWRIGCeck2UMHXnK115C+ZudRLlKnShJbqpv8L6xlF67augCPSv\\\\\\\/54XA3AJkC2n\\\\\\\/4k1ixyM+GXzo0XNxQQ9VpAyuhZ1+wnMdPZlb00AG9arUaORi3qWyFGUqMfTq0gv0L672ubOialqCdXkND9hCPaSuwZpGKOrM1HK3urUTMqU3xFKZd+9H6+kPGZ\\\\\\\/n4lqeJofsbMHUHzEPtxl8ir\\\\\\\/6PDw61K1t2pdKYkCM+V+6uuJI5tEdzklRFixgCHWsTdn9pFxfwKhj\\\\\\\/d5gj3uWviBINlP2XHChBukZo4uBgSRXI38aIYUBTlVY+mjnGtEaZG8Jme2qXdw+PTG62802cJSligezOCS\\\\\\\/3xrcjMYeORmhibkVPROO\\\\\\\/TQCJcBPJEar4D85sxLpVIuVq9GnsSPJ4cFbT9YeSAiZcNqdkN+r3ceuX4HkixG9oRNNPxznwOGf2YVLywZFoK0U9DhUQdHGN+pRo0d13PfVRW+XQmo7I2ZZ6Y3GIEREC3pC\\\\\\\/UeK2xwf1TY19zKsKJ3tS4TOYZxPbiefJI\\\\\\\/Xc4CroIoCtrmFsXc1V3tGQQrAXoliSLIn3VzQOdm4T4QivSsDWmw4xaEtZt2t5yNzBx6dDgSYcAmynG5RRr4iYg\\\\\\\/UeMM+dOSmLqqQqu5tTgLSghkp8JGnh7ddAeKlm+SSeqLHAU5OcSU4EuE2fyyYp05VUSDyng3ojlB1B6uFIwjNVpHm49HsULwNjlW\\\\\\\/rqbD\\\\\\\/cr4mOLPsFyQLr23pcNZeMmvJ3uk5hqt21xzKNvCTGFg4TXDypvPlvjzjBbWvbW2cUiwdvpYjeDRaCWHUKePKP4WECAvNJQlXwN3aq5VT1c\\\\\\\/arnRQiCeYfC1rCS8KD1jH3wsUqAm8LZP23PQ+S1dVIQcswien5eyj5hcQR4XP5ZTAu++lOyDDF4bf6tbgzx6+0J0AK7gdkFTk7W2synADiaSgM0VWOhNSQRgR6lY4VL9Z+vpzRMQsmg+IxnSOia+XR6D2h1HI3dsxRWw1SS6IID52CGF0KBwaZxW8vY4LZQvU2JTQchcanBBvUJxyEaN2rQIJs\\\\\\\/nsrFqp+94yUrxLPtSqHPwD95Z2k4ZpW0h1wSkVrc3GuK\\\\\\\/pirD5CEWJ6qVU1O4hDB9eeguiViSXamYMh1Fg26H26pM1I7diD7bDt8H47S0KoJoK1RtyjPzoLiPtMBfk\\\\\\\/Tx72YZ0df38Fb6x1X2TiJiOrgGjxMJTV3pevkUmzYIFfAJdATmNTHMKz0C+ZKH5UBTnmVPm2WhTBLqgz0oYYMyyFkoFypw9jvkQtcwdIwzzj1Ok7x6cEDURpqRO1sPHCgKL8l4xrrTO+VO3nUATMQfb2JJErLeUMTPrgHE\\\\\\\/vspVtC9GASbgwPM4Jh1FJHeMGDD\\\\\\\/0LaMuWk9zlcp+WoN4GdB+tT5OJi3kbctFHKCxt4mnZNuGilQrZZJQ9lqbkKJISw24rLiqhB2G4uJHZGLhPsbVWlcVSWNYAm\\\\\\\/KpGc4qr5Ghx+ymlv0+bsazZYm1aIgPH84usDc9UELahxNBujKjhIV+\\\\\\\/yS3nBIwyX2Hgkf6nkJe8MOs2QGpCPT19E6KJR2AzvpHjpV9DZ9FH9M6as7ZoD4ho9Mblh0PFV8XOhnT4u\\\\\\\/0147N1V\\\\\\\/TREx1ljHhh0zltnTlU8ZTq2tuZhfovGOljMJ3\\\\\\\/A42Wo1iqEUuIJPQsDYWx8BiGC1TZnIq529LrpoMABldHHNscApDH7ntRsdtssZ7\\\\\\\/orXvGcLy89gr1EoJIopO5uLUawfxM8ippdKPUnTRMMaUrYkMH2fa5P3yrcrcgcLDyyP+qkVqzk81VGfCSje1Cnm9Lak0frgA+CULr9IEuPjiAXH5W2pfTkk7tfQrIetG8C4qw+17AJXqqJ+HGGxFenQzOEMBaOo76eGf\\\\\\\/F8x4rlA1aGpTnTK75Rz5e+53+6HddFwpr4BvOXSV19CuJtSb0yhVMpnglYpo4UIXhCFPCQpoUQQlbOA7N\\\\\\\/p6w\\\\\\\/rMJR8VTmLVbJ+MMeNaa4BS3f4hI5Bf0tEXN2mfHISQgvqYdOXdgLEJ2ddwlQ640XHN\\\\\\\/tdY\\\\\\\/Yu1olHVwZ0CmiQCkUhbhJCU7wsOpENOaYVrz9j4H4Ta1WlUaTPSTgyduvsKVkXPGUXrevJUqVIvwQlg25TuMBCTTa\\\\\\\/N9dlK3M6njHW3WBYpdsLLhOP0ql3ZpqlMMR7WxeRuBGUc0DQi9lxyUprIJAzfqNoIybsgMpcBPMwdDFB4P4EG3+JxU2COWXl6CahIQCDyxxyoA8cHSGLCYOvH4QjDZMajNDRHBNoeBObCrcwAE\\\\\\\/ouJuHvX44Hfm98r55tWllSwraY\\\\\\\/xv\\\\\\\/15095SJtEK9pl\\\\\\\/pUXcxa4o+WWf1BWSqccSvLIiYNG+VcV\\\\\\\/NW596rWoTol4i2UreGQDMZDCOO96scPgmphRzrW9mlGOPajn+t7Stg+o4CkfpoEt8OTpdAMFCLxLpoNP0julh6TWo8IznmZInAQtlwuhs76YYJQKfZgr1SkDR1S\\\\\\\/ltyIAU\\\\\\\/+IG6bBsF8fZHAUebG\\\\\\\/DYB+xR1MQdulPBKCBnnJvQtxDQGYVWEJnnDXHmAZjIcr33aglIbXa83qI5sDalmsAHRAneVVu\\\\\\\/UJZf7NyE\\\\\\\/bw29u6UzKahKUflc2QkiKPov3q51Tr0vtLT8x2Ag9dZS8pSEGIFeQi\\\\\\\/3FrdYQKtYvJhlWM7y2\\\\\\\/uWFuNxXsaIFLR90VuUMYJTPq2UtDfxIV6qMbKt9YW1wuEpxDDEFvtTkE2WTQyLy5tB12FQVGs\\\\\\\/yCNaw+7Z+NWsYargvUyvd8immpNthQqjjpQ2O92LNpy1ETY2JE7jFsPqHos9q5k5C3cRFVHQDpWsH\\\\\\\/gbXSDf\\\\\\\/Ai0pDjSXxVLFqC2DGcpYSMCQF8UGkAIxbiR\\\\\\\/J+wLn4t5vHfYw3IkxJdqWjpTc4wVrNhXPNfWYGCoN6Z5IHuAd4O5Qhzag\\\\\\\/4FnjuN3JtnR7m0yYDCVkAK3LvFHeJl6LeNtBDoNniSVdMQr1EgyfsDEAxibdNR5UYRhfFdITWt9uhhy+r\\\\\\\/ub51DQ\\\\\\\/kypDDxEOiMBNucSyDxs5V4Fqz4Ma6FweT+XF1jw4zKgES6XyUFooyZtTsxQaOAxrCk4vizLxdMSbBmgFurgcyC\\\\\\\/rw+9AdpFKNmD7wFiuvnUXTDXnWvh82Mwo+BJA8zN6a4IqZfEYPWpGCgr1T64LwHCqDHf26acDLJnZm5\\\\\\\/H1GPA3\\\\\\\/DUtZaTW1ayeocmoqzvyShCEfkMS6imrKZOdlXsZvLA3B9HlHLtQVoAZBo1bAMvCg19sJ30aSahH5Z2IHvBK1nNOHvQH270e9RSjOCD7uDd+eLkM7t+RVQBEJs4VH7pPLMaJUPU0TdQKTTPBfsB5cOPnMlESdjGwRaj9Hg22RT4Z\\\\\\\/cGga3VqOCDd2b3WWLOenHTKsK9kQpuKSwkcPAhnD2ZDxitdPHUEuP5pJPJUxeSp6Ai+IoQo2DVyAK45u3chMlvWzXZq\\\\\\\/7sJ5OmEFldLOk5FIB7H5O1wifFWzGtkAwIZnulinoYDjrzQgHmTnr\\\\\\\/aJC2UpTEDKByNgYZRZMDLgNxgX3eJ35gYdq7vT9rmo3nxIMkGpKBkfkchXLfHc3Fx+qUcAPfAP6K38v0duMgCcUW7jR6wg+po\\\\\\\/n+E+Ki0u6UF+qmsCv7zWexx7LolNVYL9FBLEec0V1VPYZ5zcmIgJApe8jWEu3KS8wBePD7\\\\\\\/4rLpsmMn\\\\\\\/Ja7+ehewWQa9GtReOIn56n6OUj9ZEiMB3jhVcWYENOYuYmj9eFtN7aB835Dd9v52bMWbz7QIYZui8zP3RVX4\\\\\\\/9NgOQqmUUj1ZhERbtaL4PDx6QWubX+6oqbMWbfuLRi59u86XYPvvA4hwO3q\\\\\\\/tkwK5THccEuF+uom2srBPkK9VDrj4OjfNaJt+Onwr\\\\\\\/foGGwyH8oir7XIuaHzCHpj925wNcD0UCrPQwybqmSI4yC0lKZt2dJX4tXHdx19\\\\\\\/yNQrpF1RvDNn+lmqYc7npQmrCr6ki59sRUgQM60ePBNbQii228cQKZvkwAfOF8AbOyZ0gOfIT0+tEZQshUYtxFGm4zsYSeyXkh1y8Ui4Hd06rmxGQ3bSOINpxCVkjzjqhC7CfF0k8eqph0YvYVpIKPVXLn69rZXBsSt6o48iH4MdRnOzfOzPCm55mZAYtLMle51KznXcWd0v4lc0XuekC\\\\\\\/9wp+a3E+U1Hh8ka7yRgnAj6Gp35pISkgLqTBpNRzWupXNknXTxLN33s6awbk8bAG1toibZeaH6YxS6uCplU0aUBGpyfpd84TLvUcu+1EVDg2K2pSoUkXb7gOGM2knkS51PXYO9\\\\\\\/sZX7jHd+K5Xh1BkFy5RtITr2rltGo4jMUXwlz5\\\\\\\/nd3xYOvJf3NQp0hqZQYL6oy0QL5cQ+on9QmyCXoZfi8PatV7Z8QHtMQTL2w0EDerI6U2Mj5hemM1V5GBB+Z2t9o+7IOGWzM7IE2JUBlwPRv1\\\\\\\/H0wf917UzdmD1burmx+zg7e15Eq99XZfwOXpKaHM2oaqMjP6cjMB2+9baApgj6hCNlM1ng+5BNFIcAn1RURzRhvuo7FDWvY14iGvCwfQCpimjQPhJO9PPXZgKAphJfGELBvZw4fJ8z+g+EXJ6PmHbNO8IroKfKJFQ8hf0Nu5lunV7agSHMghQUsN3nj92rUSlTemmft+jshDYWRft7STfoARMjJrx4wlf9w7nLK8TPwbJNU2S9pEYTtDdifcTr7UPsqXeUX5ZL5WfxbFbyf9Vf0T7eWNw1VZkVIxFa3mkPYaX+V6RU4A3kR7dGBruePeCdXrAHDruVaLg9ZIyWBPptCdFeQFFfCqBedMFHj5g5+632RUqG+fXbyF87BjV3ZQ3tJfjrclZr6mGEhPBKGNUm\\\\\\\/PvM0l0LdqFOv4eTmOaESCs1zKERl9vhjzVJ2bOUN4gNIlbKEJoXCMiR31L9kQIFHY5m2fy+z1YFSWQGMgSlWWOXagquk5HeIoIsaUh6v7+QU8dMR57uikRXiCZ1oLRLosm8FNrqdm\\\\\\\/L8MkEJAxwiZJoMu89Uzus\\\\\\\/RQ50xa+K9qotjVnA+2ShyvX0QfeqTmL+GyrLu64Wm7bDlWSbXjCQS5WS0zudWrhY0t2kD9KzjhhGbjGUrVIkNpLeiTN\\\\\\\/PuZWk8D+CmgcNugHoqsdpPwWBZHaAUIHMzSi\\\\\\\/78qELA\\\\\\\/LRwHGe1WpuquICLatFXwzSlXBZ4aBaGqjKuxAXl3XqNK2tMjVXAhsl\\\\\\\/gwMI4TjhfphnX5o719vnN3RubRTcU4Jtovsq9EKHCDxXgD0r1ThI3d2gb02UqxD46JQMbZRAZGzjwG+N168yGIPAqkyl88nfhmYCioaLCFSaB3E1hDa07xqxMW6vn8pMyDY6NWaWJhrBFqkC6E5RLXI9VpWgrUIPRoh63Zm366zvO3HBmewxb3y4KjHud7n3gDKm2l9Om5TZGzZkqUW5pDtkOFyH0tuafnUH4cSgLWYj3JpckXG+jNZsKVXbd6bGhnNzYXr3UgK4g3rmWEq+j\\\\\\\/iLi1hzGLvHkx5WHZ845EghhjOb8N1lW8UzN0dQPiThFC\\\\\\\/XIx8vDBnzCPFYPmXGqeobxvacGtYhpdx42Iqg1WL3+jKJNYo8\\\\\\\/fVLd0c8W+k5sutchRR7iBnFQOiy3RVh5djSL8uktAXC67OhNi33UCnPG\\\\\\\/QfUitgdLHZ\\\\\\\/pHOL3wxuqMTdpy4osQTq4f9V5mJcDQ0A3oFUmfcyKmN47Tib0XHaI09dDZTlxnNxDeDxMhwiJRj835\\\\\\\/lW4+IIgQtICN2TkEwZMAwKu34roM5eSeb8\\\\\\\/R5x1wwa0a8LVS7EYu+GVAY9czOxYq00pldx+6c7Pvrrdi1wPw0M5DlyrrV8ErEsaFFKvPHRE9DHm4zWoHheWQ7OwGEAQ+mMFthWoIf3lN8UvA+sxfi7Z2G6jO7X0lA\\\\\\\/3clIDAAMCglLjcbHnqn20y0Dwa4N2jX5640LKCKrDR3iq3OZA\\\\\\\/HtbMDuYQolZQ1h8lBCL5pxQFc4daD2M4bop7hD8ZnMGvE1i6S+vHtGgtJehxhAd1O7WECiJ842hXZ376gxyXuY+Tx7Hld2qs\\\\\\\/YM5F5dYG9cRHDqihgxeHZ8nSgkCfwBbYLJtvVBdeBSm29HeIBhvsJp4UTuC0utC4cl3JaVT1Ix1jBU9KGT\\\\\\\/to+Ba1LiwPexbBNKSVsf9avMCfS6zZRCyT\\\\\\\/J9QTETv3G1mrR0KJMKYkA79iXbmd+PCUWlpzKr43cOHpLUBHEt5jawGjKjAauRNOSfWum\\\\\\\/VRpRnVslGKtI6qzj3Lk2WBMY7ecXtgpkNKOuh+FcORCKSizjWhJPu9v17AEkiuLc5hYAYcc3l5Y\\\\\\\/Y52h7UoeGBmF\\\\\\\/TAdo4s6e+OyG2I33s8v0D9QGSqcMqNpmogdRkyMT3qZV8nYgh9CFRAq+e9BurprCJ1h9TislQQ7abGczi2wpSfyPWafWKjgekp2WCwx2WjqDolDUKaxuX8aD1bevzjQ5xdLXyyj5dwKmkyHMSl+ITtreolxT9RpGFDY5R0vUHor4x8q3tTpXvBG3m5YMLqLmxb\\\\\\\/UgEky2pl7cA0f5+sXfXs\\\\\\\/3BtC5kmoAjQ4gOHSoQSet5DHksrEl88tTIixFDGIhUIr1JMgkb2GgF777h0KR0TrZ4l1lgoH2p5G+GZH3XTmZBgDvO6imGgHZlOJWyiZfr1SQ2Byf\\\\\\\/M4+RdeKNwPLUe6pyGTpiJyMZIECWUgEvzim1ou8c6+tiQmkTVvIg09q7kSjIEu49qgskP5f+JmULntVgEa2oWoi+jOorE1QYkMEXUREYpLuFTKhw0LXE7kKENAxkPP9uDU4N7GO5Bf1RvQl8WTkn2X+aJuS0cSUnV6ygIjpNTh64ACZ1WmGJ4SDR7vyAyJHVA\\\\\\\/y8bIuhXhkwP9YJ5yrl\\\\\\\/OeCFfgqbOb5OCsu2QXy3\\\\\\\/ANbJyKIJnp7FyESPQj5Trf6ceiNXUboNewxwKRPTQ2Vx+lJ7qwFQ7CBmGhknq4tHFK6Fr4ZAvkmE5Gxm0yerFGqVmeP7d+EbZjbd7HqfB836q1EpBMSmDmEDH4asuxrpiv24iNs1UTRPXaCpDmU7bwK+AzW3oN0pHk\\\\\\\/efO\\\\\\\/RRNUurpcYO3FTrzErITrs3TVTnLjmUt6ti936e1pTNuDlBO6zSESeeNSGB3ZWHyFJXLa9Dn2OpRpaD\\\\\\\/FX9MVVmxJd1+xmZDpx8Ka\\\\\\\/u2zok8RwHpaFKXdt9+ndOtUOCqTWrWzpTWkxTjNB2BjE\\\\\\\/e9lxAXKKt4NFRCZ+lg5euOx61VUAuNyQhLF2S+WEXiKmO9AbhRfPg1s0mLPxP1veOFj4p4n8vGBsVmeUIGgfHd0Udwi6CWmwk10mQy42f3SCDM8xdUDrw27rxJY2THF+U9QQ96iIcfxulBODFkCCM2+agbQwT8jpdaFdm4nCGO2V\\\\\\\/FsZ1E2PXkjfti6\\\\\\\/RJYD+zZPfi9Pmv+xthIodIlcxvkOGLVFb7UQAfOzKy0E+DpoQ0JgDzQ9fxG2kJJ84PzB2wXD85VoIV48X6PD1uUxI8ObCIN5vh7+nIW85JnPewoMMTP6Co\\\\\\\/\\\\\\\/ygXI63OBeF7x3GMOkQBtNkx9Od36IBk1Y\\\\\\\/gH4JM9rbzaq\\\\\\\/Cdey3iZljsmMoPEpbKXyFJCE2WNpl2l+mYl69jX0EydL+4Ura\\\\\\\/fcHe3WrlvMdzJNjucc7ZpmBG\\\\\\\/bMliVg717UZZfOAG2DlnToM10\\\\\\\/ScUu4YjblxfGBtKtKaONsTCqxmsxg5VHsk6I+bpAGL1RX3OSssrnU6NlJ9kSAHxBXGs+mQwFNmskAB8zGZa2Yk8C3nHWlsFxFaDsLqE43yAo10rR9\\\\\\\/DnibEibME2rEyFtPJ4ZlSCOer\\\\\\\/u2N4EUsPHR7eMPly7Z+4owlWHjQSbpsi+ZGH6lgPG0SvwhyBWkdo9jD+j6R+mZcL1rGBPI+0pPGUV\\\\\\\/oW4guK5oeC5OGOJSzyQzV8ttD7MIP3wSCsxJhyoWhBfsFqcX55tNVKXdrSPico5IG5lJ5On39g+YlZ07pVYxvBTLAqwqgpID3HCNbA6+Tm0WLhtPiZdQNb99iE1wcH1V9OqH+C7GuGue6pJY2bsKzx6AN0f3DAo7tXtrgwraVnsL8cms2HHSi6UF9SoiS58H3qUrEwFeAZMaCpPiv8H3f8ysRB3X2cT5HbpC20ShAGV6mwFVH5SSydMZ751m2ht9HIdFuOleQ5\\\\\\\/4j7pQBpZripxTyl7U2yc69rgaMsBvqt84IJNnd6+SpMI39TT85b9VIDQIv6HxyeWusirmK2CD4\\\\\\\/sBSOyUwGftyQ5qB4NOS\\\\\\\/11lerPDktLws416s6Zg569b\\\\\\\/8pupEa\\\\\\\/VLLMTzLRMlEc9coroVB+CSt5VkbZPf6dW+b+mfdQXquPvaJjnKJZzWjHyJv9F8\\\\\\\/G\\\\\\\/e6TtA3sTe1ExT1nW8kF4bYJLRC\\\\\\\/2w2FR9EBLXBRvKBCdQqMNHp34MAWryBlywtH+oZrHUc\\\\\\\/51KV1yaKDGtdUTytP8lpDrXBk6WkRUuLCKnXBuems7BJAX3w1ZqohGLuMHiM4MhoCbtNvQqAED7Wbw5KgBE0fwXvKneY5B\\\\\\\/BoypI8ye8eaAZP1kkCpBw4pG9YewTZWC\\\\\\\/3rcJaWiVqOu+Zl2m6XY7JxnBc\\\\\\\/rnbzsWaUnhyHCpmFisvzj4UBlOM6ZUVEOe1IG3kC6DtP\\\\\\\/X1mpqmKTlxy9lsfUts8xhbFIJEd+bnRwP0c\\\\\\\/KIX3KCeu9iv2M1j5ViMswU65phHbAvAXl+qjSDupGKmpPU\\\\\\\/odClJ3Ja56l8PU0u15LH4rjkq+q5fz\\\\\\\/uzx35jbB+vWv6iwPEkNckr79JtK758esbhFkcWz4Q0f4hMa+IqqPokzXH+QUsF0cp4w80pgwcSk2M5ws7ATvuW4CifdRq34EL0\\\\\\\/MtuucA1lYcggMWs5jd+rB3ngNYEJefpn\\\\\\\/73T8MK82CBPYQV9fQ8oAmwQqpxPYKaBiUHEDuwF1J6iDFuoGKSSCTPw6rObfxihKacaFxzGZA6AO4udC3EcRQQKVz8iatjFLfoXUeuw+aIQCCiWlJfcD6CowGghGwES2C5G0RByEu8iBAM1vD6u9wsosDslKGIVj6d3nVKSNj6hn92+siTplQkgGfs7AS2Fku4OHnLccBbAAnYKCg3FujilFhia5ETz+NaFUOSAfglV1T5\\\\\\\/g\\\\\\\/H7Do8xJOivxqGVQszbGVIuvMVy916nA6vo33WXwjoUemGt4tUAP6QncphiFKWibxK26qL8mq3n5K\\\\\\\/EA69gHUcSS21+ubDFdDboCxX7T6ksmSYbV2+4B8jU2AT+pbM8B5xu6zc1fuFm+98ci9vAittXcyWbb5cKBPps0eFFWQ1FOJtGkSICzM0JFlB68bob4OxHr4Tp2NQF4yQobw6vEmfzJPnwRFFPXeS70mIsA31h2vMBHlnAEQ1BhRFcpx7opYgryri0olmlLY5wb09dtfsPcs1AdXAy64beqFqE\\\\\\\/fjkUkr\\\\\\\/Ivc1ZU2iaC0zMVLhFxKd0r1SOhsD86Iv7xL6HHiiVwavR16zpw4k76sUILbpIzd\\\\\\\/HFpb3BF3tsVN2kRNLbYpqcwPbEcIc2WmMEK7Vg6KQLJBxZrGVtr\\\\\\\/2hqnAZcsFyIr\\\\\\\/UKT\\\\\\\/rNTB7ozXRa6O35hZimJnvZanyrZ7e5zknidSlgouC14S5MXNr5LdSlmPK6rxodw\\\\\\\/059Vcpq7PmVqeGghtcJbFtUVZjGKTNVW6eIR5bioxs0uKryMCZTleFFJFPTsvG643zip5j1gBPvhhDgC2SY\\\\\\\/vR8Zv9PVdROVqdfKkGlkd5qa673FLyUxXTqtSzDBpzTOCkaOXoEscdPKpSPBR2P42NJ9uKYLZSeeKZj9tGJxqhg3QSiKbzlMkrhogAO5PfEVNCQ\\\\\\\/bCW4PInlBxFHC8a2YcirwrcFMj2J0UQplDM+LJYE1meqIXXpPXl4g\\\\\\\/BHrzz3aWLKODqfOcjVJ9YdVoNnUIl2ErJOSXDKmAxVva+5DOAHXUsHbeoKTKYBjQS5diNvxSSBisGGQhTFKLBXv7UBawizrlmmh\\\\\\\/veI\\\\\\\/w8C5rYZhDfdBA2jxDeXuAmeF+Yc289PEmIUxsYdJnnCujsE\\\\\\\/u38Lwobgy\\\\\\\/tYnjoHEpWSmQs8xqgz4Zt2DB59NacOePvqXlOduJzQ1wROuJIpAizA7POAgsjNSFB9ZuJUAJbPoJNDjYWM33sEUqqpxBojURzpC6\\\\\\\/JbMk59C8mKuzv35zCHOCpYqNpBHIp8LOiDvQL4N1GNpnxQS5CkIiM4MpqPuqRnHik5AXhtPWQrOLT\\\\\\\/yVJS8WSP8\\\\\\\/BgUHpUMdHF51uWP2b2sqH\\\\\\\/tsxh50\\\\\\\/FpOdzQZo7ikNV76R9flBkQjXK88hIDJ\\\\\\\/vBgWc5JMwyfGUByP9rqWgUw4Q4iOjpbF2bD0rq2CKJKtnVXe812jKdaw8pVwF0yEiSMJezIYs+mYdIRUnbZyfMXTR5+h3F67JRupYrS6PdzspamtJfflRlxCktW1x4V01Z3MUBWNBkON2dlT5UYsgEv0ymrWXahBVn03av8VUe1tFlD5DaPHZ598BGciiCE4Yz0Ak6YJu1PuDKzwugE\\\\\\\/EFSqgC7gYd0Q8GDUP3le7Hz7ia6ejW0S9h6bw9Goa\\\\\\\/uNgzTVyO7qydozG6IuQnAAzeVPFDxysm2AEOTLrD8NLDGUoWeLW\\\\\\\/VPNGz4n2WcQ9fmc5ANrXh6d1jGyBIgn3kD52YJd0js17zvwhCnRHvdglpygafr2FceBAZ4DaqoJEPDyI6vLlT8oejIkmvMZvfL4kQl8bT3DgrGt4Ylmir3BfGAgIy1Ftr9JmUBH3bOAtyLDyRZ7iDK1ZWyNX1NAoaPibDmxGu43ldt07ZRWctrnDoy7ayfCddYEY9OJDmporB087BIytcOhJN6wvkemvydt7P809mtl+8TgRoF2eESXqMiknPurBUDwidWtWjjm2Nf7h22r8\\\\\\\/ygCHyo52pN2NsbICTXf8DlegzyCnPPpF4tCRW0jvI5gI\\\\\\\/+X7BvpZYuuNiDG\\\\\\\/PghbPAcUSnxBXiGxk+caAezafmqtejqTZQh3Y\\\\\\\/Ucu+eDiWQS9Zr\\\\\\\/VeLXGpmsX9mLzCGfXSnp\\\\\\\/hcc1xTNuBqjyaBrMgQ3YbwQTwsBdAUETI6El\\\\\\\/WfxB94UdT+niFH\\\\\\\/Syo9ImC812siBHkz8lmsMYuV62Ewr3x2LzHXgsquDfMHM2qRhfLKdbPLSo9al9icx24mv86SWFwSLqUUhZIzAZg8YGbtEB6eZxjnFDk5sy6MB0tHohxQ305r2+yxTkZfCKNvb+u2tvfo11flQOvKrTo9S1LYJ5puzKlhsWe1owYmMEAimMAl0lZiz7yltFY9C2PQWEh47WX90b6OqPMVYepbuAk9T8JlWQ1f0WVtYpovAXN5qOpbuU7Bgo2PBvPvDTdDKmwfNf0ycu0GJxPixHRua\\\\\\\/0md6tTIMQbx8O7L+i9w6IqB9YFzsTkk0cEJgIwuhm+hGHubywqxyEDnuKEC2PycCfX7Kvuc33a66K2M50R\\\\\\\/BvBKp\\\\\\\/s5xRtNueXxM51idQOR+PG5DkVgK7zbYhTt4jdndhwsrjmg35KPxzA\\\\\\\/GnOcd2z9APObrjfh4V1o8hi00NAOLPYE9hQPyNQ1BrdfXg6gks5v+TJjCALooQbYrX2k67EX7\\\\\\\/gDJKRwiNee7aZ6Ue45I0GqXyLXuyhDg1bt4yoKtsBFsKFGMVPyT8J8KCsNp2gPTxLUidBqt0SXXub5hTmlj+NmbAo53iC2tw+xmP1kv1qYHyRsjm4pBcPtoA23XBDswvKPnCnEmCAPq5zfGN9xkCbWPlvXrCXfGnQu4zCjRF7CSNH\\\\\\\/9ex8uSqelSz6Z4fzCBWnPkv53Q1UAX0D2DcRn9gkgHvjYDuwnA0j3uUptl2HOTZUaVby+oUyR55WlUl09druOUi\\\\\\\/aL51O4KGwtfI8JxicUDlB+cjFaqK9K\\\\\\\/yiPujSLlHAY+0CdzINklBPhRaC4mncMbfQCdrzaZbFc1mFF8aK4a+uuKNnkfVQAIRBB11vIlyS2rVpx49v2ffidugzkZsp93fqM9ho4v55+GlOA\\\\\\\/nxDdKw+mWxJRq93yChQ6wgfr23Dkhb+o35fWc7Dk+s5el\\\\\\\/6iZrPGf9uxCB2BkrvdCT472LyJu85R+xFjGJtJ6xxoBcLQ8Zpq4x8raqr0GIBNy9fZuYJwYP70N3oVA4jbRNHggj1BMtHXrJyoy0sve3Ym5NC\\\\\\\/LOyo2GJW+3C5JuHfiOf3FwZShZ71y+OX2dTy\\\\\\\/S8CEPtOw5sKXzO5klDFjhoDm7Uj0y4W\\\\\\\/X1OlAFe1gxGtV87wJ1Zh5nHBpi+AWAiBj+J9\\\\\\\/AKrmtmC2RtNtbwHlK7FIKc+m3UkvunEqgw1cCQJguj8cp5Dbg6WjAU6oe7+03wshN6yy2kLTVdGMX5I2O8VOF8lBpar7CVQLCCyjC2U1nfzy4nxz6NwzHHf7Kj+0qq4AtIJaR7+XIEeirVro7Nu6Ot3P7QefmusyN5yL0OvlpbwpEZ4yG6Ol\\\\\\\/wsn7yCnFODuWjnCA\\\\\\\/jgEuBgpGGLha7hZwOEIWEYo2glqpfGumhsKXjStpALx6ipihXrIhFkWNLuYHDZAiGhbIVeNpi3EEj5H9N6+tvD4kSx2ehcoww4Dr8bqSHB8538JiVuHyNR1eSqZY7gIurU3xFOzG5tVAoEeL1vaazyOOZ0MOAqiMyZZSpzfh+6e7Om9wJWXorU8+rD\\\\\\\/\\\\\\\/hozUFWfpCehuOrCR2Z19G5mntoKzAM\\\\\\\/eWt5cv71I7euiRuROYC4t7gCvmMEi+FlPqY8eLI0yEuuvW1aww9ZBcNWosd\\\\\\\/OJKZxliwHycx33X\\\\\\\/9yY8FQUyevQ9k64hd7ePryDafeM7UFOqwpvdiXOJjPKDTYlgyodEpo9qw7Sn3a0HYmk4T5jtt4lYBz2VT+ziUtvYjjIkVVST\\\\\\\/v0zHiGEql5LKZk215v9H25JBmHhwItgFXtz0OT\\\\\\\/XZWs9E5WhAAYS5ulaFeogUCIrKRbTiwPGJ4JIPkZlZybUpbDzp1eW8F4WLrtuHgJWhU4+fIZXsM4izrb3im0xkiqVXjH96tJkflVOX+bK9U8V66fQBvfhNzJM\\\\\\\/kBn64JWdlLKKfdG3gmC6jGVMgLa9PwlalSlrb2CUUguNtwUbX2u+80huH+gMyhI+fk3GssjtubdcMiITUm308lO+K6np9eBY7Xf5XptjfmvLlrjB\\\\\\\/DPDsblz+khcEx9Jw8FWaf17dGxONZHzQWf4FDrHc\\\\\\\/vaY2thLdRCxUU5WrGkeFVVFihSZSZc+iHx4fEX1o+0BsybR2I05pqm7VMICuCmVWgjKWI\\\\\\\/WxxhEPJqOdvzt3Bba4z\\\\\\\/TB5TUnIlJmpwt2ecBPebNrjXJblwpQ+WO0aaBmEOJHUhS5wKAGsmBgCMqTAlsok6AMiT2I4L4BHBgvTgMVJrDwL23bF6dV9XRpNG6GMkjiLTkdXjy++e\\\\\\\/SheexyhIbD6a4Hg2kdybXgGm8dwhM3Uj9d2ciHAF0SVfuwxuW8zNKGZN+sVcFotnkHA1KbU6UFUaNkfOracytsut9qxh5TShANCFnQc6yPvf+B2khdW7enq3y9\\\\\\\/mZbzg8wzPoD5mYnSbEGYelVBYhZgcqlOaUgssq9V1hC8ckbS6YuX8y\\\\\\\/bjNr8dEgKMPp+HJ\\\\\\\/LGDr6246fVfEK+m01BxorImVSSA0pqPYnUJWwwAKufKGcoOPLGB6vdYbbPlLrv9W4PRVoUO0qi++8iZfSlmhoMmmDWN95CKjqOqBefzACjLeOqEQC071BAhxIgDf9mz6IN4boouoSa4Xwn0aeVH9e7L5VnsCadYkuDK4QtnHe8vy3p6tbBD6OblVkEtXPyfT+90sjl4whOygM5VH4j+4UXhl49VUfXi+iwAm1fnqOPftfn+Hx2J7VlnzNHoRRGQcOiO2nxbJp2u41BbnrYA+qwOjcJTnT\\\\\\\/I0UcziClDYwfVtvs0z240TTm3ju0V+dNuwwUmnl73ltvIVUbq2FnlHC7Sn3L5BDozPdqdpHO8lFkxgOn5QtVu7FLb+Me2YPZVQFaLGOZxZXX3jDHTlBrNOque3DTOREugtCuphPeyx1bUxmeE6HevWVqSiiPb7b86FmzO4rkJ0C1UJh4lw3ya0eSk5QUbmk9TnJ2oHkVaoSKIXlfhKzRxwqVoWN0EkwUvTML94bCUtOvyYBhTJ7OnfrNVRWLll4Z5RLN\\\\\\\/noIJDxm3Rh6g6MqCRBlRtU55VhQu1aDBx+Px5GF3NgPWgcHkI5GCwEK5121kBfEfyOmEpA+cOSMSBv+zSnq65tB1xWhoOk13OHmOUBbD8CIYC6SFC5hrOKdPdJRNVKbP7xIZsD26z17UM9r+TM1eHZLUtQ5y+DwsKOiV7e2IznfvfJwbSwvBxhlrtBhm8NndGnXUyxGar4eXUSBevcNHd77p\\\\\\\/RPAzpPdfEYUAyLrG+aN7N59FIdvxToDcvUVVkCyXQ800fE1AM03JgUTUsPJsrxZcxky265b21u+4c6lLfY0uDrUkqPHbQWCgbUNfc7tAa2KxYI\\\\\\\/FiyS+vGqRJp\\\\\\\/znV2NqqvxwNtND4yEpNlfuzpzKCp1O8WcFQfTIei8I\\\\\\\/IQMILoZZDtSMBg7QZsTe7GxzUvaZSgrEfyb3eWAOotpnIXU6LoPxBWD3vwEG16DP\\\\\\\/ptgt30YhQnLM5mx\\\\\\\/cCLImaKm4ltTbe6jkrixlPI\\\\\\\/ntI3ObGrHTH2yN1S29VKugPRPotPGfHJS\\\\\\\/lmSA6Qi2\\\\\\\/X6iPyQ98ctvzGFJEqsNZO7BaVu9kSzEUf4+FctyNcoUtqG\\\\\\\/L8t3F1wEIaDZhEOkvRDCeUxxd5e72GccyMWcYlFPDSPqmxbIWnCQCDpfcy0Co+iQdhxlrEwbQHp6+3aDfiHcs67CYmTgCPyKV4ioT\\\\\\\/G7YjeKITZ3tw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Q8vMkWixUiVAZ0r3aurOEw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b4658e3f3521541075994b6d854a56332cbe19730a1bb5c2b612a1b0d14d17a6\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022rZwQeBsy\\\\\\\/94t+fzW9e7ndHpxZHXIvcnTbkdSdnvzkpK2KU71h7HUDUo\\\\\\\/LCgBDHbvfOEonXKneJFgAhxK3gd+e+5O\\\\\\\/PPld1d4x+XkGsD6NOZG0b9t3Kh4w4XfvEu+D\\\\\\\/ZJH2PnMoubY94kjeVO2RfyrznQbzuPPVl3uuiZW\\\\\\\/mJJRUulROPZRGvgN8xcxUCagJRva1ZH5Y7oBYMfV0avBSAvv5oIGobeB3jhHTh7+kI0AhFDB5SxMDoQT0oOSIGP9tHXOlYTO5mIL31sWQI\\\\\\\/CTWBQWoeko5d7INfGy1UbDhxypBIqJsY4WFSUBmjPj57PB1hZj21QWh6t7Y1gndaxJ9C7b8OfSYhRqw5quuxrnpbsAJNXDKFH2TUOdHdtZLy\\\\\\\/0fyip33\\\\\\\/yEKsPgHZnQyMxPICjCPJ7J+s9k2EqeNyzzzdNOiWwJEFDCEPmZgOpH6\\\\\\\/gYk1fkVTHFkbGiggjlWGHnmPbyzev3SUykf4Mz\\\\\\\/8l2swzqJ64b6M+N9OhAXqsolxMvdEsIdQRCUKcdGl1KL3vK\\\\\\\/ly0oUtaMEFv06uZW3ZdfEMjlpVBqc506WR68Lo3++QCMeWV3odWG5iZhCB02HX3z2fR17wHQO1yLKCMn3LENFql5IwognmdZTU67TwGfiMQUfSzdSx8Qupf9MzvWxnx9TH8TZYrtkPp8pR10iFR06xSDOxPxnrD3X5uWeJ6Ic+Q1lbbWTKtFyv5IjYhqKWXnlcgp9wWQGq1Od4HGPaaaHk7Myg3A9A1Q3XODu9prLnnZKH6+Bhu06sYF0\\\\\\\/P8kyalYEI5jlpvSOhOh1QRUVNtmQukDmsxzz3wiScCwkj8cU9XAP\\\\\\\/17LomuR6gBXsG1dSJvoph1hgWFy9QyaBAD1ZZlWdLI7vSH85AnCSNXuuWRtmuVGq8gsMJ3kDgVjA8CUNeXTS2heFZk5RfnzFCu7eZBhtUgqZNR\\\\\\\/dcUdYixOeBZrPgHCnv9C2hcnkmlklL2rgafEuktsRSUGbM4GE5l1qOsFlkKPrme1W4EYf\\\\\\\/uUYx1NISSV9WV4mLvGLBoaueHSKaPBlrYd8DsZocVycH7cID3wktsY7uE77lpDlA93SAQkJTArOEvoiQIzW6BplJHBle9W6L\\\\\\\/tx\\\\\\\/MDxewGDzicBbW3oEoNA38MVHFTSW5rYFqi78PrEtFMYqePFLxi+w6qJcv9QHwlU1p2yfsHj7zyLRNWC+J5uEzFru0r93tom4c3TXr2pm\\\\\\\/TZZvIooRisOUVXQ0oorZxYQEmKdlt\\\\\\\/JipMft1mveu8EcPj2DSKF2+Wx4xgDq74S+zgFz8cZ4YlPSKdoyy5hvUMQDi4aMFBTTyXBuHJFDK4punwed5WwD5+P\\\\\\\/k\\\\\\\/rVcqPS7F1c7TqoIMtiPuozp8orNyPL2JxwhKLOh9qwu3EOvEAzlzL5LSAdbpIuvy0T4wQ4XRaGyQ+7\\\\\\\/UkbcFQDPG4eUrfWjFa7Pc1RBtkoOt+Z9maxy+iVKnkKgVe01DzwMx4DPraikDs5p2Z2qs3vFmq7x2PgD5EHzxqHWpcWHjiIFU0iABc7RrTfmaO\\\\\\\/NL\\\\\\\/0TUibmabMccORyfdrH\\\\\\\/0dLZKGX8po9pPQ5fLqj\\\\\\\/cm+yxSI21+S4e9UFogVZ9HlJaw4jmx+H9FCmaX1QZtc\\\\\\\/3d95iT9xHglNdq2IOeRc2UztqS5QaYzg8+CB\\\\\\\/gKH41Mr8urk+fNjOR6ohABHqtVxOrRL1XkTEe5ELFbaEkHpxiMbi2KLzmSv9+BklnGA8z0DVHHHhgPaYJ++mbtFXHYSLAXrMAAFfvTAV+UPUpelnviqia3cA7JLcBEMHaHA1fTaNmuQs0TWb\\\\\\\/JUfFJ7BDqlwBsVIx63gRKcvm0OKGxhA2NioDCQJv2yVHcEKStg5w38jhrvJk3RszY4dgXg1clj4z5o4MXO3c9RMhxQXHYuf2EZkJsmAXCQVyvU1WcBhEt0nX39VdVsAFsC5juTJDM4qnGhCtglb8xm1BH40FMP2MFF5Bj8J0z\\\\\\\/mbnn9xkQ4OzdgAav0NcbdoIKRNjh2A73dQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022QNsz\\\\\\\/sJ2AtUzo8QyQzSsLQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220bb5e3ddefcb2fe5e63ce9fe0e5d324583c0f8041fb93bc5892387eaf349dcd3\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460099 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/3/609" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Uohw3t3bL7MpBhOrYbiCrR12aZ7yF2JRo18ag29NBak" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/3/609" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-_6S-2zBkyp2RtGpUp4G3OmDFHQYzc1cEaIIrgzE6IzI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/3/599" title="Genetic Correlates of Individual Differences in Sleep Behavior of Free-Living Great Tits (Parus major)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/3/623" title="Suppression of F1 Male-Specific Lethality in Caenorhabditis Hybrids by cbr-him-8" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460099" id="node460099" data-pisa="ggg;6/3/609" data-pisa-master="ggg;g3.115.024349" data-apath="/ggg/6/3/609.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Effects of Both Recent and Long-Term Selection and Genetic Drift Are Readily Evident in North American Barley Breeding Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4332-5986" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ana M. Poets</span>, <span class="highwire-citation-author" data-delta="1">Mohsen Mohammadi</span>, <span class="highwire-citation-author" data-delta="2">Kiran Seth</span>, <span class="highwire-citation-author" data-delta="3">Hongyun Wang</span>, <span class="highwire-citation-author" data-delta="4">Thomas J. Y. Kono</span>, <span class="highwire-citation-author" data-delta="5">Zhou Fang</span>, <span class="highwire-citation-author" data-delta="6">Gary J. Muehlbauer</span>, <span class="highwire-citation-author" data-delta="7">Kevin P. Smith</span> and <span class="highwire-citation-author" data-delta="8">Peter L. Morrell</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">609-622;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.024349 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460099" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ana M. Poets </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ana%2BM.%2BPoets%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poets%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAna%2BM.%2BPoets%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4332-5986" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ana M. Poets</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:agonzale@umn.edu" class="" data-icon-position="" data-hide-link-title="0">agonzale@umn.edu</a>
<a href="mailto:pmorrell@umn.edu" class="" data-icon-position="" data-hide-link-title="0">pmorrell@umn.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mohsen Mohammadi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mohsen%2BMohammadi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mohammadi%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMohsen%2BMohammadi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kiran Seth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kiran%2BSeth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Seth%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKiran%2BSeth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Hongyun Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>DuPont Pioneer, Mankato, Minnesota 56001</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hongyun%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHongyun%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Thomas J. Y. Kono </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BJ.%2BY.%2BKono%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kono%20TJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BJ.%2BY.%2BKono%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Zhou Fang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhou%2BFang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhou%2BFang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Gary J. Muehlbauer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BJ.%2BMuehlbauer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Muehlbauer%20GJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGary%2BJ.%2BMuehlbauer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kevin P. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BP.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20KP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKevin%2BP.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Peter L. Morrell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BL.%2BMorrell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morrell%20PL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BL.%2BMorrell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:agonzale@umn.edu" class="" data-icon-position="" data-hide-link-title="0">agonzale@umn.edu</a>
<a href="mailto:pmorrell@umn.edu" class="" data-icon-position="" data-hide-link-title="0">pmorrell@umn.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/3/609" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460099" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460099/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/3/609.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460099" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460099/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/3/609.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460099" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460099/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/3/609.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460099" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460099/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/3/609.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460099" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460099/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Barley was introduced to North America ∼400 yr ago but adaptation to modern production environments is more recent. Comparisons of allele frequencies among growth habits and spike (inflorescence) types in North America indicate that significant genetic differentiation has accumulated in a relatively short evolutionary time span. Allele frequency differentiation is greatest among barley with two-row <em>vs.</em> six-row spikes, followed by spring <em>vs.</em> winter growth habit. Large changes in allele frequency among breeding programs suggest a major contribution of genetic drift and linked selection on genetic variation. Despite this, comparisons of 3613 modern North American cultivated barley breeding lines that differ for spike-type and growth habit permit the discovery of 142 single nucleotide polymorphism (SNP) outliers putatively linked to targets of selection. For example, SNPs within the <em>Cbf4</em>, <em>Ppd-H1</em>, and <em>Vrn-H1</em> loci, which have previously been associated with agronomically adaptive phenotypes, are identified as outliers. Analysis of extended haplotype sharing identifies genomic regions shared within and among breeding populations, suggestive of a number of genomic regions subject to recent selection. Finally, we are able to identify recent bouts of gene flow between breeding populations that could point to the sharing of agronomically adaptive variation. These results are supported by pedigrees and breeders’ understanding of germplasm sharing.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/introgression" class="hw-term hw-article-keyword hw-article-keyword-introgression" rel="nofollow">introgression</a></li><li class="kwd"><a href="/keyword/breeding" class="hw-term hw-article-keyword hw-article-keyword-breeding" rel="nofollow">breeding</a></li><li class="kwd"><a href="/keyword/evolution-crops" class="hw-term hw-article-keyword hw-article-keyword-evolution-of-crops" rel="nofollow">evolution of crops</a></li><li class="kwd"><a href="/keyword/genetics" class="hw-term hw-article-keyword hw-article-keyword-genetics" rel="nofollow">genetics</a></li><li class="kwd"><a href="/keyword/identity-state-0" class="hw-term hw-article-keyword hw-article-keyword-identity-by-state" rel="nofollow">identity by state</a></li></ul><p id="p-5">The evolution of breeding populations encompasses processes that reasonably mimic the evolution of natural populations, but in accelerated time (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Ross-Ibarra <em>et al.</em> 2007</a>). Comparative population genetic approaches can be used for the identification of genes underlying adaptive variation and to understand the effects of demographic patterns on diversity without specific phenotypic information (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Nielsen <em>et al.</em> 2007</a>; <a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Ross-Ibarra <em>et al.</em> 2007</a>).</p><p id="p-6">Breeding populations are subject to demographic effects, such as founder events associated with their initiation (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Martin <em>et al.</em> 1991</a>), and potentially bottlenecks, and gene flow from other breeding populations or exotic sources. Although the primary goal of breeding programs is the selection of progeny with improved agronomic adaptation, these demographic effects also contribute to differences in allele frequency among breeding populations. Thus the differences among populations are a combination of genetic drift and selection for both agronomic traits and for improved local adaptation. This selection also involves some degree of concomitant linked selection. Understanding the selection and demographic history, including the effects of genetic drift and migration in breeding populations, can accelerate crop improvement (<a id="xref-ref-64-3" class="xref-bibr" href="#ref-64">Ross-Ibarra <em>et al.</em> 2007</a>), for example through the identification of loci involved in domestication and improvement (<em>e.g.</em>, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Cavanagh <em>et al.</em> 2013</a>; <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">van Heerwaarden <em>et al.</em> 2012</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hufford <em>et al.</em> 2012</a>), identification of introgression between domesticates and wild relatives (<em>e.g.</em>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Hufford <em>et al.</em> 2013</a>), and the determination of specific donor individuals contributing to disease resistance variants (<em>e.g.</em>, <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Fang <em>et al.</em> 2013</a>).</p><p id="p-7">The history of population differentiation due to selection can be investigated by the identification of loci with large differences in allele frequency (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Cavalli-Sforza 1966</a>). Selection at different timescales can be detected by complementary techniques. For older and longer-term selective events we can use fixation indices or <em>F</em>-statistics (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Nielsen <em>et al.</em> 2007</a>) to measure allele frequency differences between populations. Variants with large differences in allele frequency among subpopulations, as measured by <em>F</em><sub>ST</sub>, may have been subject to selection (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Lewontin and Krakauer 1973</a>). A high degree of haplotype sharing between individuals can be indicative of more recent selection (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Pritchard <em>et al.</em> 2010</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Horton <em>et al.</em> 2012</a>). Haplotype sharing based analyses can identify specific haplotypes (and their underlying sequence variants) subject to selection based on their relative frequencies in a population (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Hudson <em>et al.</em> 1994</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Innan <em>et al.</em> 2005</a>).</p><p id="p-8">Migration between populations can contribute to adaptive variation (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Slatkin 1987</a>). Identity by state (IBS) analysis is a sensitive approach for the identification of specific genomic regions involved in migration between distinct populations. IBS analyses have been used in studies of barley and maize to identify specific genomic regions involved in adaptation to new environments (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Hufford <em>et al.</em> 2013</a>) and disease resistance genes derived from exotic germplasm (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Fang <em>et al.</em> 2013</a>).</p><p id="p-9">In the present study we use SNP genotype data from the Barley Coordinated Agricultural Project to investigate the breeding history of North American barley breeding populations. Barley (<em>Hordeum vulgare</em> subsp. <em>vulgare</em>) was introduced to North America by European colonists as early as 1602 as a crop essential for beer production (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Weaver 1950</a>). Early successes in barley production in North America involved introduction of cultivars adapted to similar environmental conditions. In the eastern growing region of the United States barley was introduced from northern Europe, including English malting varieties; and in the western growing region from Mediterranean feed cultivars (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Weaver 1943</a>). Notable exceptions from outside northern Europe included “Manchuria” from northeastern China, which was well adapted to the Upper Midwest growing environment, “Stavropol” from Russia, which was the most important cultivar in the Lower Midwest (especially Kansas), and “Trebi” from the southern shores of the Black Sea in Turkey, which was found to be particularly productive in western regions under irrigated conditions (<a id="xref-ref-77-2" class="xref-bibr" href="#ref-77">Weaver 1943</a>; <a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Wiebe and Reid 1961</a>). Despite the use of germplasm from multiple Old World sources, the set of founder cultivars was relatively narrow, which likely contributed to reduced diversity in modern North American cultivars (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Martin <em>et al.</em> 1991</a>).</p><p id="p-10">Barley production is frequently divided into spring and winter growth habit. Spring barley dominates in North America, while winter barley is grown in more southerly latitudes and moderate coastal climates (<a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Weaver 1950</a>). Spring and winter barley have been bred separately by breeding programs located in distinct geographic regions. Further trait requirements by end-use markets (<em>i.e.</em>, malting, feed, and food) and the establishment of breeding programs for two- and six-rowed spike-type contribute to highly structured barley populations (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Cuesta-Marcos <em>et al.</em> 2010</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hamblin <em>et al.</em> 2010</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Wang <em>et al.</em> 2012</a>; <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Zhou <em>et al.</em> 2012</a>).</p><p id="p-11">Our analysis focuses on four major questions. First, which of the factors, including breeding populations, growth habit, and spike-type, contribute most directly to genetic differentiation among our samples? Second, to what extent are loci identified as major contributors to phenotypic variance in Old World barley germplasm contributing to allele frequency differences in North American breeding populations? Third, can we identify evidence of recent or long-term selection acting on breeding populations and what loci are involved? Fourth, how have patterns of shared ancestry and migration contributed to diversity and relatedness in current breeding populations?</p><p id="p-12">Allele frequency-based analyses of the North American breeding programs indicate a large contribution of genetic drift and linked selection to genetic variation. Patterns of allele frequencies observed in loci known to be involved in adaptive traits in the Old World are recapitulated in North American breeding programs. Additionally we were able to identify populations and specific genomic segments involved in more recent gene flow.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant materials</h3><p id="p-13">Genotypic data from a total of 3971 barley accessions including cultivars, advanced lines, and genetic stocks from barley breeding programs that participated in the Barley Coordinated Agricultural Project (CAP) were downloaded from The Triticeae Toolbox (T3) (<a href="http://triticeaetoolbox.org/">http://triticeaetoolbox.org/</a>). Barley samples are representatives of 4 yr of germplasm enhancement (2006–2009) from 10 breeding programs. These breeding programs include most barley growing regions and market end-uses in the United States. For more information about the programs see <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Hamblin <em>et al.</em> (2010)</a> and <a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">Wang <em>et al.</em> (2012</a>).</p><p id="p-14">The breeding of two- and six-rowed barley within a single program often involves different objectives; thus we consider these types as independent populations resulting in 16 breeding populations (see <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Hereafter, the following notation was used when referring to each breeding population: “breeding program’s name abbreviation” followed by 2 or 6 for two- and six-rowed spikes, respectively, as described in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>. The Busch Agricultural Resources international program referred to here as BAI2, was separated from the North American two-rowed lines, referred to here as BA2. There were 10 six-rowed accessions from Oregon mislabeled in T3 as two-rowed (P. Hayes, personal communication); we used the corrected spike-type (see Supporting Information, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/FileS1.pdf">File S1</a> for more details). The current sample is divided hierarchically at the highest level into spring and winter growth habits and then within each growth habit by breeding population (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/467665/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/467665/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Descriptive statistics by breeding populations</span><div class="sb-div caption-clear"></div></div></div><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/609/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Breeding programs. University of Idaho in Aberdeen (AB), Busch Agricultural Resources, Inc. (BA), Busch Agricultural Resources, Inc. International lines (BAI), Oregon State University (OR), Utah State University (UT), Washington State University (WA), Montana State University (MT), Virginia Polytechnic Institute and State University (VT), North Dakota State University two-rowed (N2), North Dakota State University six-rowed (N6), and University of Minnesota (MN)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1800241352" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Breeding programs. University of Idaho in Aberdeen (AB), Busch Agricultural Resources, Inc. (BA), Busch Agricultural Resources, Inc. International lines (BAI), Oregon State University (OR), Utah State University (UT), Washington State University (WA), Montana State University (MT), Virginia Polytechnic Institute and State University (VT), North Dakota State University two-rowed (N2), North Dakota State University six-rowed (N6), and University of Minnesota (MN).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/609/F1.medium.gif" width="440" height="240"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/3/609/F1.medium.gif" width="440" height="240"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/609/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/609/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467677" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-15" class="first-child">Breeding programs. University of Idaho in Aberdeen (AB), Busch Agricultural Resources, Inc. (BA), Busch Agricultural Resources, Inc. International lines (BAI), Oregon State University (OR), Utah State University (UT), Washington State University (WA), Montana State University (MT), Virginia Polytechnic Institute and State University (VT), North Dakota State University two-rowed (N2), North Dakota State University six-rowed (N6), and University of Minnesota (MN).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Genotyping</h3><p id="p-16">All 3971 accessions were genotyped with 2882 Barley Oligo Pooled Assay Single Nucleotide Polymorphisms, referred to as BOPA SNPs (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Close <em>et al.</em> 2009</a>) using Illumina GoldenGate Technology (Illumina, San Diego, CA). BOPA SNPs were identified primarily from resequencing of expressed sequenced tags (ESTs) (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Close <em>et al.</em> 2009</a>). SNP order along each linkage group is based on the consensus genetic map of <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Muñoz-Amatriaín <em>et al.</em> (2011)</a>. Sampled accessions were self-fertilized to at least the F<sub>4</sub> generation before genotyping (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Hamblin <em>et al.</em> 2010</a>; <a id="xref-ref-76-3" class="xref-bibr" href="#ref-76">Wang <em>et al.</em> 2012</a>), resulting in average expected heterozygosity of 6.25%. Genotyping information was downloaded from (<a href="http://triticeaetoolbox.org/">http://triticeaetoolbox.org/</a>) using only the OPA 2011 markers. The options to download the dataset were set to minimum MAF ≥0% and to remove markers and accessions with &gt;99.9% missing data.</p><p id="p-17">SNP annotations and metadata information, including gene names and whether SNPs occur in genic or nongenic regions, were obtained using SNPMeta (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Kono <em>et al.</em> 2014</a>).</p></div><div id="sec-4" class="subsection"><h3>Quality control</h3><p id="p-18">The dataset was filtered for monomorphic SNPs, and SNPs or accessions with more than 25% missing data. Additionally, we removed accessions with incomplete sample information (<em>i.e.</em>, spike-type or growth habit) and accessions where single lines represented a population. We also removed accessions with heterozygosity that exceeded the expected 6.25%, a conservative threshold intended to filter out accessions that might have genotyping errors. Finally, we removed genetic stocks or near-isogenic lines such that one accession from each near-isogenic line set was retained in the dataset.</p></div><div id="sec-5" class="subsection"><h3>Summary statistics</h3><p id="p-19">Descriptive statistics were calculated for each of the 16 breeding populations (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). The degree of inbreeding was estimated by the inbreeding coefficient <em>F</em><sub>IS</sub> (1 – H<sub>o</sub>/H<sub>e</sub>) using a custom Python script. The similarity between samples was estimated by the percent pairwise diversity calculated using the <em>compute</em> program from the libsequence library (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Thornton 2003</a>). Monomorphic SNPs in each population were excluded and heterozygous and ambiguous calls were treated as missing data. The SharedPoly program from libsequence (<a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Thornton 2003</a>) was used to count the number of private SNPs in each breeding population.</p></div><div id="sec-6" class="subsection"><h3>Derived site frequency spectrum (SFS)</h3><p id="p-20">Derived site frequency spectra (<em>i.e.</em>, the distribution of the derived allele frequencies) were calculated and plotted for each of the 16 populations and for the complete dataset. To infer ancestral state, SNP states from <em>Hordeum bulbosum</em> as reported for BOPA SNPs were used (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Fang <em>et al.</em> 2014</a>). This involved alignment of RNAseq data from one diploid accession of <em>H. bulbosum</em> (Cb2920/4) to the Morex draft assembly (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Mayer <em>et al.</em> 2012</a>), and calling the <em>H. bulbosum</em> nucleotide at the BOPA SNP positions. Ambiguous nucleotide calls, <em>trans</em>-specific polymorphisms, and sites at which <em>H. bulbosum</em> segregates for different nucleotides than barley (<em>H. vulgare</em> subsp. <em>vulgare)</em> were treated as missing data.</p></div><div id="sec-7" class="subsection"><h3>Joint derived SFS</h3><p id="p-21">To compare the change in frequency of derived alleles between partitions of the data we used the joint derived SFS. Following the same procedure as for the derived SFS for each breeding population, we calculated the derived SFS within winter or spring accessions and within two- or six-rowed accessions using a custom R script. The derived SFS was compared between growth habit and between spike-types. The joint derived SFSs were plotted using the R package grDevices (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">R Core Team 2014</a>).</p></div><div id="sec-8" class="subsection"><h3>Population structure</h3><p id="p-22">The degree of differentiation among individuals from all breeding programs was estimated by Principal Component Analysis (PCA). The analysis was performed using the SmartPCA program from the EIGENSOFT package (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Patterson <em>et al.</em> 2006</a>). SmartPCA permits PCA analysis with SNPs that include missing data.</p></div><div id="sec-9" class="subsection"><h3>Maximum likelihood tree of relatedness and migration</h3><p id="p-23">The population relatedness and patterns of gene flow between breeding populations was inferred using a maximum likelihood approach implemented in TreeMix (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Pickrell and Pritchard 2012</a>). To polarize the divergence among populations we used genotyping data for 803 landrace accessions from the Old World (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Poets <em>et al.</em> 2015</a>) (see <a href="https://github.com/AnaPoets/BarleyLandraces">https://github.com/AnaPoets/BarleyLandraces</a>). The 2021 SNPs shared between datasets were used to build a population tree using landraces as the outgroup. We ran 25 replicates of the tree, bootstrapping with 75 SNP windows. We used the replicate with the lowest standard error for the residuals as the base tree topology and inferred the likelihood of having between one and five migration events among breeding populations. The plot of the residuals was used to evaluate which tree best fit the data. Candidates for admixture can be identified by those population pairs with residuals above zero standard error, which represent populations that are more closely related to each other in the data than in the best-fit tree (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Pickrell and Pritchard 2012</a>).</p></div><div id="sec-10" class="subsection"><h3>Changes in allele frequency</h3><p id="p-24">To identify putative targets of long-term selection involved in the spike-type and growth habit differentiation among breeding programs we used the <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Weir and Cockerham (1984)</a> measure of <em>F</em><sub>ST</sub> referred to as θ, as implemented in the R package hierfstat (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Goudet 2005</a>). <em>F</em><sub>ST</sub> was calculated for the following partitions of the data: (1) spring <em>vs.</em> winter, (2) two- <em>vs.</em> six-rowed, and (3) among breeding populations. Owing to the high level of inbreeding in the dataset, a haploid model for <em>F</em><sub>ST</sub> estimation was used as this better represented the sample size in our study. Heterozygous SNPs were treated as missing data. An empirical genome-wide threshold for the top 2.5% of <em>F</em><sub>ST</sub> values was used to identify SNPs with large differences in frequency relative to the genome-wide average. To identify the degree of differentiation that each breeding population has with respect to other populations we report <em>F</em><sub>ST</sub> for all pairwise comparisons. To determine the degree to which our <em>F</em><sub>ST</sub> results are robust to either the specific set of SNPs or individuals used here, we used bootstrapping to remove either 20% of SNPs or samples per iteration. We ran 100 iterations in each case.</p><p id="p-25">To characterize average allele frequency divergence for breeding populations we used the <em>F</em><sub>ST</sub>-like measure of <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Nicholson <em>et al.</em> (2002)</a>, reported here as “<em>c</em>.” This estimate differs from other <em>F</em><sub>ST</sub>-like measures in that it attempts to estimate the degree of divergence of a population from ancestral allele frequencies. For this analysis, the dataset was divided into two groups: spring six-rowed and spring two-rowed. We do not report <em>c</em> values for winter barleys owing to limited sampling. For this analysis, monomorphic SNPs were removed from each group. The <em>c</em> value was calculated using the popdiv program from the popgen package in R (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Marchini 2013</a>). We used a burn-in period of 1000 iterations followed by a run length of 10,000 iterations with the scale parameter of a Dirichlet distribution used to update global allele frequencies <em>m</em> = 10 (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/FileS1.pdf">File S1</a> for more details).</p></div><div id="sec-11" class="subsection"><h3>Analysis of resequencing data for known genes contributing to phenotypic differentiation</h3><p id="p-26">Resequencing data for 10 accessions for vernalization sensitivity loci (<em>Vrn-H3</em>, <em>Sgh3</em>) (<a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">von Zitzewitz <em>et al.</em> 2005</a>) and 96 accessions for the <em>Vrs1</em> gene controlling spike-type differentiation (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Komatsuda <em>et al.</em> 2007</a>) were obtained from NCBI Popsets [UID #157652625 (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Karsai <em>et al.</em> 2008</a>) and 219664771 (unpublished data)]. Contextual sequences for SNPs known to occur in these genes were downloaded from T3. To determine the position of SNPs within these genes and their correlation with growth habit and spike-type differentiation, SNP contextual sequence for individual SNPs were aligned to each resequencing dataset in Geneious v.7.1.9 (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Kearse <em>et al.</em> 2012</a>).</p></div><div id="sec-12" class="subsection"><h3>Identity by state</h3><p id="p-27">We used an IBS analysis to identify shared genomic segments among individuals between breeding populations potentially indicative of recent introgression. The analysis used PLINK v.1.90 (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chang <em>et al.</em> 2015</a>) with window sizes of 50 and 100 SNPs, allowing for up to 10% mismatch. The frequency of shared segments between two populations was estimated for each SNP window. This analysis made use of phased genotyping data, with phase inferred using fastPHASE v1.2 (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Scheet and Stephens 2006</a>). Missing genotypic state was treated as missing (<em>i.e.</em>, genotypic state inferred during phasing was ignored). The phased data were only used for IBS and pairwise haplotype sharing analyses.</p></div><div id="sec-13" class="subsection"><h3>Pairwise haplotype sharing</h3><p id="p-28">To explore recent events of selection within populations, we used the pairwise haplotype sharing (PHS) approach (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Toomajian <em>et al.</em> 2006</a>). A shared haplotype is defined as a genomic segment that extends out from a focal SNP, and is shared among individuals in a population. PHS is a form of IBS analysis that compares the extent of shared haplotypes among individuals normalized by genome-wide sharing. A PHS score depends on the length of the shared haplotype and its frequency in the population. Extended shared haplotypes are potentially suggestive of recent or ongoing selection, owing to limited potential for recombination to break down genomic regions subject to recent selection (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Horton <em>et al.</em> 2012</a>). PHS was calculated within each breeding population using a customized Perl script (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Cavanagh <em>et al.</em> 2013</a>). An empirical threshold of the upper 2.5% of PHS values and a minimum focal SNP frequency of 10% within each population were used to identify outliers in the distribution of PHS.</p></div><div id="sec-14" class="subsection"><h3>Four-population test for gene flow detection</h3><p id="p-29">The robustness of patterns of migration inferred using TreeMix (see <em>Results</em> for details) was assessed using the four-population test (<em>f<sub>4</sub></em>-test) (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Keinan <em>et al.</em> 2007</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Reich <em>et al.</em> 2009</a>). The <em>f<sub>4</sub></em>-test is designed to distinguish introgression from incomplete lineage sorting. The test evaluates trees of relatedness among populations and measures genetic drift along lineages quantitatively based on the variance in allele frequencies. Significant deviations from zero in three possible tree topologies (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/FileS1.pdf">File S1</a> for details) indicate that the tree evaluated does not fit the data, suggesting the presence of gene flow.</p><p id="p-30">We used the topology inferred in TreeMix as our hypothesized relationship between populations. We inferred that a tree of relatedness with three migration events provides a better fit to the data than topologies that include either a lesser or greater number of migration events (see <em>Results</em> for more details). Following the [(A, B),(C,D)] notation for a tree topology with four populations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/FigureS1.tiff">Figure S1</a>), we assessed migration among the following sets of populations: [(N2, X),(Y<sub>1</sub>, Y<sub>2</sub>)], [(UT6, UT2), (Y<sub>1</sub>, Y<sub>2</sub>)] and [(OR2, X), (OR6,VT6)], where X was replaced iteratively for any two-rowed barley and Y<sub>1</sub> and Y<sub>2</sub> were replaced by a pair of six-rowed barley populations. The populations UT6, OR2, and N2 were chosen because of an indication of gene flow based on TreeMix results. Pairs of populations from the same branch of the TreeMix topology were considered on the inference that these populations are more similar to each other due to shared ancestral polymorphisms. The second pair of populations was selected from the branch containing the population putatively involved in the migration event (populations connected by the arrows in <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). Since the population putatively involved in migration was not clearly defined, we ran the analysis iteratively between each possible pair of populations on the branch. We used the fourpop option in the TreeMix software (<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Pickrell and Pritchard 2012</a>) to estimate the <em>f<sub>4</sub>-value</em> for each configuration. Significance of <em>f<sub>4</sub>-values</em> was determined at <em>P</em> &lt; 0.05. We infer that migration has occurred when the three possible trees had a significant nonzero <em>f<sub>4</sub>-value</em>.</p></div><div id="sec-15" class="subsection"><h3>Data availability</h3><p id="p-31"><a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/TableS1.xls">Table S1</a> contains information about the samples and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/TableS2.xls">Table S2</a> presents SNP ID numbers, genetic positions, and annotation information. The computer code used for all analyses (unless otherwise specified) and plots are available at <a href="https://github.com/MorrellLAB/NorthAmerica_Fst">https://github.com/MorrellLAB/NorthAmerica_Fst</a>.</p></div></div><div class="section results" id="sec-16"><h2 class="">Results</h2><p id="p-32">The original dataset included 3971 accessions genotyped with 2882 SNPs representative of 10 breeding programs across the United States. We removed 340 SNPs and 241 accessions with ≥25% missing data, 22 accessions with heterozygosity &gt;6.25% across SNPs, 13 accessions with missing growth habit (spring <em>vs.</em> winter) or spike-type (two- <em>vs.</em> six-row spikes) information, 79 near-isogenic lines, and three accessions with single accessions representing a population. After quality control our dataset consisted of 3613 barley accessions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/TableS1.xls">Table S1</a>) and 2542 SNPs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/TableS2.xls">Table S2</a>).</p><div id="sec-17" class="subsection"><h3>Characterization of North American breeding populations</h3><p id="p-33">Analysis of the structure of the North American populations using PCA (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/FigureS2.tiff">Figure S2</a>) revealed that the primary population structure (PC1 = 19.6% variance) is explained by differences in spike-type, which corresponds to an average <em>F</em><sub>ST</sub> of 0.23 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/FigureS3.tiff">Figure S3</a>). PC2 indicates that 9.1% of the variance among lines is explained by differentiation in growth habit, with an average <em>F</em><sub>ST</sub> of 0.17. These results are congruent with earlier analyses on a subset of these populations (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Cuesta-Marcos <em>et al.</em> 2010</a>; <a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Hamblin <em>et al.</em> 2010</a>; <a id="xref-ref-76-4" class="xref-bibr" href="#ref-76">Wang <em>et al.</em> 2012</a>) and on a comparable set of samples analyzed by <a id="xref-ref-83-2" class="xref-bibr" href="#ref-83">Zhou <em>et al.</em> (2012)</a>.</p><p id="p-34">The 16 breeding populations (<em>i.e.</em>, breeding programs separated by spike-type) were represented by an average sample size of 225 lines with a minimum of 30 lines (from UT2), and a maximum sample size of 386 for MN6 (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a> and <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). On average, only two SNPs were private to each of the breeding populations, with a maximum of 11 private SNPs in OR6. Winter populations had 16 private SNPs with respect to spring populations, which in turn had 72 private SNPs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/TableS3.xlsx">Table S3</a>). The average inbreeding coefficient (<em>F</em><sub>IS</sub>) across populations was 0.98 as expected after four generations of self-fertilization. Percent pairwise diversity ranged from 0.15 to 0.36 and averaged 0.25 across breeding populations (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a> and <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/609/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genetic diversity in breeding populations. (A) Average pairwise diversity in each breeding population. (B) Genome-wide pairwise diversity for the most and least diverse populations according to average diversity, Utah State University (two-rowed) and University of Minnesota (six-rowed), respectively. Diversity values were averaged in 10-SNP sliding windows with a step of five SNPs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1800241352" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Genetic diversity in breeding populations. (A) Average pairwise diversity in each breeding population. (B) Genome-wide pairwise diversity for the most and least diverse populations according to average diversity, Utah State University (two-rowed) and University of Minnesota (six-rowed), respectively. Diversity values were averaged in 10-SNP sliding windows with a step of five SNPs.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/609/F2.medium.gif" width="440" height="202"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/3/609/F2.medium.gif" width="440" height="202"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/609/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/609/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467793" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-35" class="first-child">Genetic diversity in breeding populations. (A) Average pairwise diversity in each breeding population. (B) Genome-wide pairwise diversity for the most and least diverse populations according to average diversity, Utah State University (two-rowed) and University of Minnesota (six-rowed), respectively. Diversity values were averaged in 10-SNP sliding windows with a step of five SNPs.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">The joint unfolded SFS showed that there are slightly more rare variants segregating in spring than winter programs and slightly more rare variants segregating in six-rowed than in two-rowed programs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/FigureS4.tiff">Figure S4</a>), reflecting minimal impact of ascertainment bias in each of these partitions. In individual breeding populations there is an excess of rare and high-frequency variants relative to neutral expectations based on Kimura’s model of a population at equilibrium (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Kimura 1983</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/FigureS5.tiff">Figure S5</a>). OR2, OR6, UT6, and UT2 have a higher proportion of midfrequency variants than other populations. When the breeding populations are considered jointly, the derived SFS (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/FigureS6.tiff">Figure S6</a>) displays an elevated number of midfrequency variants, consistent with retention of variants segregating at an average minor allele frequency of 24% in the discovery panel (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Close <em>et al.</em> 2009</a>).</p></div><div id="sec-18" class="subsection"><h3>Genome-wide scan for evidence of selection in North American breeding programs</h3><p id="p-37">The distribution of <em>F</em><sub>ST</sub> statistics for comparisons of growth habit, spike-type, and breeding populations showed that among the three classifications, the greatest differentiation in allele frequencies is found among breeding populations with an average <em>F</em><sub>ST</sub> of 0.37, <em>vs.</em> 0.23 for spike-type and 0.17 for growth habit (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/FigureS3.tiff">Figure S3</a>). For each of the three partitions, we identified 61 SNPs in the upper ≥0.975 of <em>F</em><sub>ST</sub> values (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/TableS4.xlsx">Table S4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/TableS5.xlsx">Table S5</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/TableS6.xlsx">Table S6</a>). Of these, 34 SNPs were outliers for both the spike-type and breeding population comparisons, seven were common between breeding populations and growth habit comparisons, and there were no common outlier SNPs between growth habit and spike-type comparisons.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/609/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Allele frequency differentiation between comparisons of partitions of the data. (A) Among breeding populations; (B) between spring and winter types; and (C) between six- and two-rowed spike-types. Triangles indicate the genetic position of centromeres (Mu&#xF1;oz-Amatria&#xED;n et al. 2011). Dotted line corresponds to the 97.5th percentile of the FST values distribution. Linkage groups are shown with different colors. In black font are genes corresponding to outlier FST SNPs. In red font are either the SNP linked to a predicted gene involved in the phenotype or the gene itself. The arrows below the threshold line indicate SNPs located in genes known to be involved in the phenotype." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1800241352" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Allele frequency differentiation between comparisons of partitions of the data. (A) Among breeding populations; (B) between spring and winter types; and (C) between six- and two-rowed spike-types. Triangles indicate the genetic position of centromeres (Mu&#xF1;oz-Amatria&#xED;n &lt;em&gt;et al.&lt;/em&gt; 2011). Dotted line corresponds to the 97.5th percentile of the &lt;em&gt;F&lt;/em&gt;&lt;sub&gt;ST&lt;/sub&gt; values distribution. Linkage groups are shown with different colors. In black font are genes corresponding to outlier &lt;em&gt;F&lt;/em&gt;&lt;sub&gt;ST&lt;/sub&gt; SNPs. In red font are either the SNP linked to a predicted gene involved in the phenotype or the gene itself. The arrows below the threshold line indicate SNPs located in genes known to be involved in the phenotype.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/609/F3.medium.gif" width="339" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/3/609/F3.medium.gif" width="339" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/609/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/609/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467632" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-38" class="first-child">Allele frequency differentiation between comparisons of partitions of the data. (A) Among breeding populations; (B) between spring and winter types; and (C) between six- and two-rowed spike-types. Triangles indicate the genetic position of centromeres (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Muñoz-Amatriaín <em>et al.</em> 2011</a>). Dotted line corresponds to the 97.5th percentile of the <em>F</em><sub>ST</sub> values distribution. Linkage groups are shown with different colors. In black font are genes corresponding to outlier <em>F</em><sub>ST</sub> SNPs. In red font are either the SNP linked to a predicted gene involved in the phenotype or the gene itself. The arrows below the threshold line indicate SNPs located in genes known to be involved in the phenotype.</p><div class="sb-div caption-clear"></div></div></div><p id="p-39">Bootstrapping, by removing 20% of individuals in each partition of our sample or 20% of SNPs, was used to determine whether <em>F</em><sub>ST</sub> outlier detection was dependent on the particular individual’s sampled or set of markers used. Across samples we recovered an average of 95% of SNPs outliers in any given comparison. Outliers include SNPs near or in genes mentioned below. Across SNPs we recovered an average of 87% of the genetic regions where SNPs were identified as outliers in the full dataset. This average excludes linkage group 1H where the large number of outliers precludes identification of outliers plausibly attributable to selection</p></div><div id="sec-19" class="subsection"><h3>Known genes contributing to phenotypic differentiation</h3><p id="p-40">We identified a SNP outlier (12_30883, linkage group 5H) in the <em>F</em><sub>ST</sub> comparison for growth habit located in the vernalization sensitivity locus (<em>Vrn-H1</em>, <em>Sgh2</em>) (<a id="xref-ref-75-2" class="xref-bibr" href="#ref-75">von Zitzewitz <em>et al.</em> 2005</a>) known to be involved in the growth habit differentiation. SNPs within two additional well-characterized genes including photoperiod response-1H (<em>Ppd-H1</em>, <em>Eam1</em>) (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Turner <em>et al.</em> 2005</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Jones <em>et al.</em> 2008</a>) and c-repeat binding factor 4 (<em>Cbf4</em>) (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Haake <em>et al.</em> 2002</a>) were also found to be <em>F</em><sub>ST</sub> outliers (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). Both of these genes have reported functions related to growth habit differentiation. <em>Ppd-H1</em> alters the flowering time response under long-day conditions, thus making it possible to avoid unfavorable seasonal conditions (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Lister <em>et al.</em> 2009</a>) whereas <em>Cbf4</em> contributes to cold acclimation (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Skinner <em>et al.</em> 2005</a>).</p><p id="p-41">While vernalization sensitivity and photoperiod response are important determinants that contribute to growth habit through an environmental response, there are also loci that contribute to flowering time independent of environmental cues. This class of genes in barley is referred to as Earliness <em>per se</em> (EPS) loci; one characterized example is the early maturity 6 locus (<em>Eam6</em>, <em>eps2S</em>, <em>mat-c</em>) (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Boyd <em>et al.</em> 2003</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Comadran <em>et al.</em> 2012</a>). A linkage mapping study placed early maturity 6 locus (<em>Eam6</em>) (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Laurie <em>et al.</em> 1995</a>) near the centromere of linkage group 2H (cM 67.8), a region where we identify six SNPs with elevated <em>F</em><sub>ST</sub> (average <em>F</em><sub>ST</sub>= 0.87) (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/TableS5.xlsx">Table S5</a> and <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). The two SNPs (11_20438 and 11_20366) most strongly associated with flowering time in a recent GWAS study (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Comadran <em>et al.</em> 2011</a>) that resulted in the cloning of the barley <em>Centroradialis</em> gene (HvCEN) responsible for flowering time variation (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Comadran <em>et al.</em> 2012</a>) were not identified as outliers in our <em>F</em><sub>ST</sub> comparison of growth habit. However, both SNPs 11_20438 and 11_20366 have an above average <em>F</em><sub>ST</sub> of 0.52 and 0.25, respectively.</p><p id="p-42">In an association mapping (AM) study <a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Cuesta-Marcos <em>et al.</em> (2010)</a> identified two significant SNPs in linkage groups 1H and 2H (12_31319 and 11_10213, respectively) associated with spike-type differentiation. In our analysis, 12_31319 has an outlier <em>F</em><sub>ST</sub> value of 0.86, while 11_10213 has an above average <em>F</em><sub>ST</sub> = 0.47 but is not defined as an outlier (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a>). Additionally, there are two SNPs (11_20422 and 11_20606) in linkage group 4H that have been identified near the <em>Intermedium-C</em> gene (<em>Int-c</em>) (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Ramsay <em>et al.</em> 2011</a>), which is a modifier of lateral spikelet development in barley. After quality control, only 11_20422 remained in our dataset. This SNP has an <em>F</em><sub>ST</sub> value of 0.81 and is thus considered an outlier in the comparisons between spike-types (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3C</a>).</p><p id="p-43">Other SNPs occurring in well-characterized genes did not appear as outliers despite the previously reported contribution to function. There were three SNPs in our dataset (12_30893, 12_30894, and 12_30895 on linkage group 7H) occurring within the vernalization sensitive locus 3 (<em>Vrn-H3)</em> (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Yan <em>et al.</em> 2006</a>) that were not outliers in the <em>F</em><sub>ST</sub> comparison between spring and winter types (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3</a>). This gene is an ortholog of the <em>Arabidopsis thaliana</em> flowering locus T (<em>FT</em>) that promotes flowering time under long days (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Turck <em>et al.</em> 2008</a>). In barley, nine linked polymorphisms in the first intron have been predicted to be responsible for the variation in flowering time at this locus (<a id="xref-ref-81-2" class="xref-bibr" href="#ref-81">Yan <em>et al.</em> 2006</a>). Our alignment of the three outlier SNPs to resequencing data of this gene from 10 barley accessions (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Karsai <em>et al.</em> 2008</a>) identified three of the SNPs segregating in the first intron of <em>Vrn-H3</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/FigureS7.tiff">Figure S7A</a>). In a study by <a id="xref-ref-81-3" class="xref-bibr" href="#ref-81">Yan <em>et al.</em> (2006)</a> nucleotide states “A” and “G” in 12_30894 and 12_30895, respectively, were associated with spring barley types (two accessions), while “T” and “C” were associated with winter types (eight accessions). The resequencing data in part support this association having two out of four spring barleys with the inferred haplotype while five out of six winter barleys carried the correct inferred haplotype. However, in our much larger dataset of spring and winter accessions these SNPs are segregating at an average allele frequency of 50% in both spring and winter growth habits (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/FigureS7.tiff">Figure S7B</a>), showing no association between these SNPs and spring <em>vs.</em> winter growth habit (maximum <em>F</em><sub>ST</sub> = 0.04).</p><p id="p-44">The <em>Vrs1</em> gene, a well-characterized contributor to spike-type (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Komatsuda <em>et al.</em> 2007</a>) was not identified in the <em>F</em><sub>ST</sub> comparison between two- and six-rowed accessions. Our SNP panel included five SNPs (12_30896, 12_30897, 12_30899, 12_30900, and 12_30901, linkage group 2H) in <em>Vrs1</em>. However, none of the SNPs reached the empirical cutoff for <em>F</em><sub>ST</sub> in this comparison, with a maximum observed <em>F</em><sub>ST</sub> of 0.45. The “G” nucleotide state at SNP 12_30900 results in an amino acid substitution associated with the six-rowed phenotype (<em>vrs1.a3</em> allele); however, the alternative nucleotide state “C” can be found in either six- or two-rowed accessions (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Komatsuda <em>et al.</em> 2007</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Youssef <em>et al.</em> 2012</a>). In a panel of 96 European accessions of cultivated barley (Popset ID 219664771) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/FigureS7.tiff">Figure S7B</a>) the “G” state always resulted in a six-rowed phenotype. In our sample of North American breeding programs 2% of individuals that carry this variant state were reported as two-rowed barleys, which is similar to previous results that found this SNP significant for spike-type differentiation segregating in two-rowed accessions with an allele frequency of 1% (<a id="xref-ref-14-4" class="xref-bibr" href="#ref-14">Cuesta-Marcos <em>et al.</em> 2010</a>). The “C” state segregated at 50% frequency in each of the row-type partitions.</p></div><div id="sec-20" class="subsection"><h3>Haplotype sharing and evidence for recent selection</h3><p id="p-45">The PHS analysis permits the identification of genomic regions that are putatively involved in more recent selection. PHS analysis within individual breeding populations identified a total of 775 SNPs in the upper ≥0.975 of the PHS distribution (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/TableS7.xlsx">Table S7</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/FigureS8.tiff">Figure S8</a>). In a small number of cases, focal SNPs in the PHS analysis were identified as outliers in more than one breeding population. Sharing of PHS outliers is greatest for OR6 and UT6 (22), BA6 and WA6 (17), and AB2 and OR2 (14) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/TableS8.xlsx">Table S8</a>). The number of focal SNPs that were PHS outliers shared among populations were an average of three SNPs shared within two-rowed populations and four SNPs within six-rowed populations. An average of two focal SNPs were shared between two- and six-rowed populations. Out of the 775 SNPs, 77 were in genes with known function. The haplotypes for these significant PHS values varied in length from 19.7 cM to 139.6 cM with mean 55.9 cM across breeding populations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/FigureS9.tiff">Figure S9A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/TableS9.xlsx">Table S9</a>). The frequency of the SNP state with significant PHS ranged from 10% (the minimum value we consider) to 61% with mean 24% (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/FigureS9.tiff">Figure S9B</a>).</p><p id="p-46">Within BAI2, BA6, MN6, UT2, UT6, WA6, and OR2 we observed long runs of haplotype sharing (average length 112.45 cM) at an average frequency of 24%, significantly exceeding genome-wide similarities. These regions were putatively subject to recent selection. Among outliers for PHS, BA2 showed significant PHS surrounding three SNPs (12_30893, 12_30894, 12_30895, in linkage group 7H) with all three SNPs occurring within <em>Vrn-H3</em>. AB2, MT2, OR2, and OR6 had an outlier PHS value for SNP 12_30901 (linkage group 2H) in the <em>Vrs1</em> gene. The SNPs in <em>Vrn-H3</em> and <em>Vrs1</em> were at an average frequency of 22.25% in each of these populations, with an average length of shared haplotype of 66.98 cM (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/TableS7.xlsx">Table S7</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/TableS9.xlsx">Table S9</a>). There are three SNPs (12_20368, 12_20593, and 12_21049 in linkage group 2H) with significant PHS value in OR2 (frequency 11% and haplotype length 138.9 cM). SNPMeta annotations identified SNP 12_20593 within the nicotianamine synthase 2 (<em>nashor2</em>) gene in barley, which is a key enzyme for iron homeostasis (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Herbik <em>et al.</em> 1999</a>), and SNP 12_20368 within a gene with sequence similarity to galactinol synthase 2 gene in wheat (<em>TaGolS</em> 2), which in turn is orthologous to the characterized gene <em>TaGolS</em> 2 in <em>A. thaliana</em> (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Taji <em>et al.</em> 2002</a>). In <em>A. thaliana</em>, <em>TaGolS</em> 2 has been identified as playing an important role in drought-stress tolerance (<a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">Taji <em>et al.</em> 2002</a>).</p></div><div id="sec-21" class="subsection"><h3>Divergence from ancestral allele frequencies</h3><p id="p-47">Quantification of allele frequency divergence based on <em>c</em> (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Nicholson <em>et al.</em> 2002</a>) showed that six-rowed breeding populations have experienced more divergence than two-rowed populations, with mean <em>c</em> of 23.1% and 17.1%, respectively (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/TableS10.xlsx">Table S10</a>). Among the two-rowed populations, Utah two-rowed has diverged the most while Idaho two-rowed resembles ancestral allele frequencies, suggesting reduced effects of drift or linked selection in the latter population. Among the six-rowed populations, Utah six-rowed was the closest to ancestral allele frequencies while Minnesota six-rowed has experienced the most divergence. Utah two-rowed diverged more from ancestral allele frequencies than its six-rowed counterpart (contrasting the pattern observed in other spike-type pairs of populations). This is expected because the two-rowed individuals came from crosses between two-rowed and six-rowed varieties (data not shown).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/609/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Marginal posterior density plots obtained for the c estimator for spring barley populations. Density distribution after 10,000 iterations. Plot shows the amount of drift estimated from ancestral allele frequencies (0,0 coordinates). (A) Two-rowed barley populations; (B) six-rowed barley populations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1800241352" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Marginal posterior density plots obtained for the &lt;em&gt;c&lt;/em&gt; estimator for spring barley populations. Density distribution after 10,000 iterations. Plot shows the amount of drift estimated from ancestral allele frequencies (0,0 coordinates). (A) Two-rowed barley populations; (B) six-rowed barley populations.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/609/F4.medium.gif" width="440" height="210"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/3/609/F4.medium.gif" width="440" height="210"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/609/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/609/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467587" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-48" class="first-child">Marginal posterior density plots obtained for the <em>c</em> estimator for spring barley populations. Density distribution after 10,000 iterations. Plot shows the amount of drift estimated from ancestral allele frequencies (0,0 coordinates). (A) Two-rowed barley populations; (B) six-rowed barley populations.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-22" class="subsection"><h3>Gene flow between breeding populations</h3><p id="p-49">To identify genomic segments subject to recent introgression, we tested for regions with a high degree of IBS between breeding populations. Window sizes of 50 (0.63–44.96 cM) and 100 (13.0–62.25 cM) SNPs were used, and we permitted up to 10% mismatch among haplotypes. As expected, populations within growth habit and spike-type had a higher degree of haplotype sharing than between these partitions. The degree of haplotype sharing was lower within spike-type than within growth habit (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/TableS11.xlsx">Table S11</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/TableS12.xlsx">Table S12</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/TableS13.xls">Table S13</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/FigureS10.tiff">Figure S10</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/FigureS11.tiff">Figure S11</a>). There was a high frequency of shared haplotypes among two-row populations at 50 SNP windows, but this was reduced when 100 SNP windows were considered. At 100 SNP windows BAI2 and N2 presented the lowest degree of shared haplotypes with other two-rowed populations, while BA2 shared the most haplotypes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/FigureS11.tiff">Figure S11, C, E, and H</a>). For both 50 and 100 SNP windows, high-frequency shared haplotypes were common for all six-rowed populations – see for example <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>, except for the Utah populations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/FigureS10.tiff">Figure S10</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/FigureS11.tiff">Figure S11</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/TableS14.xlsx">Table S14</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/TableS15.xlsx">Table S15</a>). This contrasted with VT6, which showed a very low degree of haplotype sharing, particularly with the various spring breeding populations (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). Haplotype sharing for VT6 occurred primarily with the OR2 and OR6 populations, <em>i.e.</em>, with the only other populations with similar growth habit. However, OR2 and OR6 showed higher levels of allele frequency similarity with other breeding populations than VT6 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/FigureS10.tiff">Figure S10, J, K, and N</a>). These results are consistent with differentiation in allele frequency observed in <em>F</em><sub>ST</sub> comparisons between breeding populations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/TableS16.xls">Table S16</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/609/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Frequency of IBS haplotypes in breeding populations from (A) University of Minnesota and (B) Virginia Polytechnic Institute and State University. Using 100 SNP windows. Linkage groups are in the X-axis. The Y-axis is the frequency of the haplotype in the population indicated in the Y-axis label (&#x201C;base population&#x201D;). The color of the bars in the plot represents the distinct barley breeding populations that share the same segment with the &#x201C;base population&#x201D;. The width of each bar corresponds to the frequency of that haplotype in the compared populations. The depiction shows that there is a high frequency of shared haplotypes among six-rowed spring barleys. This contrasts with the Virginia population (winter six-rowed) which shares a few chromosome segments only with Oregon six-rowed and to a lesser extent Oregon two-rowed." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1800241352" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Frequency of IBS haplotypes in breeding populations from (A) University of Minnesota and (B) Virginia Polytechnic Institute and State University. Using 100 SNP windows. Linkage groups are in the X-axis. The Y-axis is the frequency of the haplotype in the population indicated in the Y-axis label (&#x201C;base population&#x201D;). The color of the bars in the plot represents the distinct barley breeding populations that share the same segment with the &#x201C;base population&#x201D;. The width of each bar corresponds to the frequency of that haplotype in the compared populations. The depiction shows that there is a high frequency of shared haplotypes among six-rowed spring barleys. This contrasts with the Virginia population (winter six-rowed) which shares a few chromosome segments only with Oregon six-rowed and to a lesser extent Oregon two-rowed.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/609/F5.medium.gif" width="312" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/3/609/F5.medium.gif" width="312" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/609/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/609/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467735" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-50" class="first-child">Frequency of IBS haplotypes in breeding populations from (A) University of Minnesota and (B) Virginia Polytechnic Institute and State University. Using 100 SNP windows. Linkage groups are in the X-axis. The Y-axis is the frequency of the haplotype in the population indicated in the Y-axis label (“base population”). The color of the bars in the plot represents the distinct barley breeding populations that share the same segment with the “base population”. The width of each bar corresponds to the frequency of that haplotype in the compared populations. The depiction shows that there is a high frequency of shared haplotypes among six-rowed spring barleys. This contrasts with the Virginia population (winter six-rowed) which shares a few chromosome segments only with Oregon six-rowed and to a lesser extent Oregon two-rowed.</p><div class="sb-div caption-clear"></div></div></div><p id="p-51">For 50 SNP windows, there were a high number of IBS segments shared between spike-types. However, when windows of 100 SNPs were considered, the majority of the IBS haplotypes were at low frequency (&lt;20%) in one of the two populations compared (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/TableS13.xls">Table S13</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/TableS14.xlsx">Table S14</a>). An exception to this was the comparison between OR2 and OR6, where shared haplotypes were quite common. Sharing occurred at every 100 SNP window, with shared haplotypes at frequencies as high as 90% in the two populations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/FigureS11.tiff">Figure S11K</a>) with an average frequency within populations of 0.46 and 0.58 (for OR2 and OR6, respectively) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/TableS15.xlsx">Table S15</a>). IBS with 50 SNP windows for N2 and N6 spanned 94% of all windows, with shared haplotypes occurring at average frequencies of 22% and 50%, respectively (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/FigureS10.tiff">Figure S10</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/TableS14.xlsx">Table S14</a>). Increasing the window size to 100 SNPs identified many fewer shared haplotypes between N2 and N6, leaving 70% of the genome shared at an average frequency of 8% in N2 and 35% in N6 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/FigureS11.tiff">Figure S11</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/TableS15.xlsx">Table S15</a>).</p></div><div id="sec-23" class="subsection"><h3>Maximum likelihood tree of relatedness and migration</h3><p id="p-52">We determined that the tree topology that best describes the 16 populations separates the populations first by spike-type followed by growth habit (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>), consistent with the <em>F</em><sub>ST</sub> and PCA results, with the exception of UT2 and UT6 that are not separated by spike-type. In the TreeMix analysis, the two Utah and winter populations were more similar to ancestral allele frequencies, while MN6 was the most diverged.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/3/609/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Tree of relatedness among North American breeding populations. (A) Plotted is the structure of the graph inferred by TreeMix for the 16 North American breeding populations, allowing three migration events. The arrows (migration events) are colored according to their weight. Horizontal branch length is proportional to the amount of genetic drift in the branch. The scale bar on the left shows 10 times the average standard error of the entries in the sample covariance matrix. (B) Residual fit of inferred tree of relatedness. Residuals above zero represent populations that are more closely related to each other in the data than in the best-fit tree, thus are considered candidates for admixture events. Spike-type for each population is shown in parentheses." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1800241352" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Tree of relatedness among North American breeding populations. (A) Plotted is the structure of the graph inferred by TreeMix for the 16 North American breeding populations, allowing three migration events. The arrows (migration events) are colored according to their weight. Horizontal branch length is proportional to the amount of genetic drift in the branch. The scale bar on the left shows 10 times the average standard error of the entries in the sample covariance matrix. (B) Residual fit of inferred tree of relatedness. Residuals above zero represent populations that are more closely related to each other in the data than in the best-fit tree, thus are considered candidates for admixture events. Spike-type for each population is shown in parentheses.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/3/609/F6.medium.gif" width="273" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/3/609/F6.medium.gif" width="273" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/3/609/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/3/609/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/467810" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-53" class="first-child">Tree of relatedness among North American breeding populations. (A) Plotted is the structure of the graph inferred by TreeMix for the 16 North American breeding populations, allowing three migration events. The arrows (migration events) are colored according to their weight. Horizontal branch length is proportional to the amount of genetic drift in the branch. The scale bar on the left shows 10 times the average standard error of the entries in the sample covariance matrix. (B) Residual fit of inferred tree of relatedness. Residuals above zero represent populations that are more closely related to each other in the data than in the best-fit tree, thus are considered candidates for admixture events. Spike-type for each population is shown in parentheses.</p><div class="sb-div caption-clear"></div></div></div><p id="p-54">Adding three migration events resulted in the lowest residuals and standard errors relative to trees with no migration, or one, two, four, or five migrations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/FigureS12.tiff">Figure S12</a>). With three migrations, we infer exchange between spring six-rowed populations and N2; VT6/OR6, and OR2, and spring six-rowed populations and UT6 (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6</a>).</p><p id="p-55">Based on the <em>f<sub>4</sub></em>-test, six different topologies yielded significant results suggesting gene flow between UT6 and spring six-rowed populations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/TableS17.xlsx">Table S17</a>), with z-scores significantly different from zero (significance at <em>P</em> &lt; 0.05). The <em>f<sub>4</sub></em>-test for introgression between N2 and any spring six-rowed population resulted in values significantly different from zero, consistent with gene flow. The <em>f<sub>4</sub></em>-test did not support introgression between OR2 and VT6 and/or OR6 since none of the possible tree topologies involving these populations were significant.</p></div></div><div class="section discussion" id="sec-24"><h2 class="">Discussion</h2><p id="p-56">Comparative analyses of allele frequency differentiation and extended haplotype sharing in a sample of 3613 barley accessions representing 16 barley breeding populations result in five primary conclusions: (i) barley breeding programs in North America have strongly differentiated allele frequencies among breeding populations; across programs, inflorescence type followed by growth habit account for the greatest proportion of variance in allele frequency; (ii) a number of loci previously identified as major contributors to growth habit adaptation in barley (primarily in Europe) are readily identifiable as outliers in allele frequency in our <em>F</em><sub>ST</sub> analyses of North American breeding populations; (iii) we identify putative signals of recent and long-term selection similar in magnitude to those at previously isolated genes of known function; (iv) the average per SNP allele frequency divergence at the population level appears consistent with the dominant action of genetic drift and linked selection; and (v) we identify populations that are recently subject to exchange of genetic material. There are low levels of genetic exchange across spike-type boundaries.</p><div id="sec-25" class="subsection"><h3>Identification of loci putatively subject to recent and long-term selection</h3><p id="p-57">Allele frequency differentiation sufficient to be detected in an <em>F</em><sub>ST</sub> outlier analysis is generally the result of long-term directional selection (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Beaumont and Balding 2004</a>). The SNPs identified as outliers for spring <em>vs.</em> winter growth habit include previously isolated genes of large effect, for example <em>Vrn-H1</em> and <em>Ppd-H1</em>, congruent with a previous <em>F</em><sub>ST</sub> comparison in European barleys (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Comadran <em>et al.</em> 2012</a>). Allele frequency differentiation between spike-type, which explains a larger portion of the allele frequency divergence in our sample, identified two SNPs in the genomic region of genes contributing to spike-type; however, SNPs within the well-characterized <em>Vrs1</em> locus were not identified as <em>F</em><sub>ST</sub> outliers (see below in the <em>Limitations of allele frequency comparisons</em> section). Many additional <em>F</em><sub>ST</sub> outliers are newly identified as putative targets of selection.</p><p id="p-58">Despite a relatively recent introduction to North America, multiple genes found to contribute to agronomic traits in Eurasia appear to recapitulate allele frequencies in current, highly structured breeding programs. This appears to be particularly true when the phenotype is conferred by a single allele as opposed to an allelic series, as occurs at <em>Vrs1</em> where the six-rowed phenotype is conferred by multiple alleles (see <a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Komatsuda <em>et al.</em> 2007</a>). Perhaps more surprising is that many of the same allele frequency outliers are identified in European and North American samples despite very different demographic and breeding histories.</p><p id="p-59">PHS analysis in the North American breeding programs detected a series of loci that are putatively targets of recent selection within barley breeding populations, comparable to findings in wheat (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Cavanagh <em>et al.</em> 2013</a>). The length of the haplotypes shared (average 81.38 cM) and their frequencies (average 25%) are consistent within recent generations, limiting the potential for recombination to break down the haplotypes. This analysis indicates that selection and linked selection are altering patterns of haplotype diversity across the genome. However, there are some chromosomes that have little or no evidence of selection in recent generations (<em>e.g.</em>, linkage group 2H in BA2 and WA2; see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/FigureS8.tiff">Figure S8</a>).</p></div><div id="sec-26" class="subsection"><h3>Limitations of allele frequency comparisons</h3><p id="p-60">Despite having five SNPs positioned in the <em>Vrs1</em> gene (controlling the fertility of lateral spikelet) (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Komatsuda and Mano 2002</a>), including one SNP (12_30900) variant cosegregating with the six-rowed phenotype, the comparison of allele frequencies differentiation between spike-type fails to identify outliers in <em>Vrs1</em>. One important contributing factor may be that the six-rowed phenotype can result from multiple disruptions of the <em>Vrs1</em> gene (<a id="xref-ref-39-5" class="xref-bibr" href="#ref-39">Komatsuda <em>et al.</em> 2007</a>). The phenomenon of multiple mutational paths to the same phenotype has been identified as “functionally equivalent mutations” (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Ralph and Coop 2010</a>) and can result in multiple variants at modest frequency controlling a phenotype. The maintenance of alleles of large effect at modest frequency in the population reduces the power to detect phenotypic associations, in part because any given variant explains a small portion of phenotypic variation (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Thornton <em>et al.</em> 2013</a>).</p><p id="p-61">The BOPA SNP platform used here is largely derived from variants identified from cDNA libraries, from cultivated accessions (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Close <em>et al.</em> 2009</a>), and thus is highly enriched for SNPs in genic rich regions (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Kono <em>et al.</em> 2014</a>). The BOPA SNPs are 89% synonymous or noncoding and 11% are nonsynonymous (data not shown). However, SNPs genotyped within a locus can have limited correlation with a phenotype, depending on the haplotype on which they occur (see <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Nordborg and Tavaré (2002</a>) for discussion of this issue). This may be the case for the previously cloned <em>Vrn-H3</em> where we see limited correlation between SNPs in this gene and allele frequency difference in the growth habit comparison. It should also be noted that the resequencing panel in which <em>Vrn-H3</em> alleles were found to be associated with the phenotype included only five spring and eight winter individuals (<a id="xref-ref-81-4" class="xref-bibr" href="#ref-81">Yan <em>et al.</em> 2006</a>). It is perhaps not surprising that sampling error (<em>i.e.</em>, small sample sizes) accounts for inconsistencies in the phenotype–genotype associations reported in previous studies and our study using a panel of &gt;3000 accessions.</p></div><div id="sec-27" class="subsection"><h3>The effects of linked selection and drift in breeding populations</h3><p id="p-62">The derived SFS in individual breeding populations identifies an excess of rare and high-frequency derived variants (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/FigureS5.tiff">Figure S5</a>) with respect to neutral model of a population at equilibrium. During selection, alleles linked to the target variant can be carried to high frequencies. Therefore, an excess of rare and high-frequency derived variants can be associated with the effects of linked selection (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Fay and Wu 2000</a>) as has been observed in a resequencing study of <em>Oryza sativa</em> (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Caicedo <em>et al.</em> 2007</a>). The PHS results presented here suggest considerable potential to detect the effect of linked selection, particularly when the selection was recent, impacting large genomic regions. One of the largest genomic regions detected as an outlier in the PHS analysis is found in OR2; this haplotype involves 138.9 cM on linkage group 2H, observed at 11% frequency. Individuals carrying this haplotype were derived from a two- by six-rowed cross (Merlin × Strider) and were selected to recover the two-rowed phenotype (P. Hayes, personal communication). This resulted in a long shared haplotype centered on the <em>Vrs1</em> gene contributing to spike-type (identified by SNP 12_30901). Two of the SNPs that permitted the identification of the outlier haplotype occur within an iron homeostasis (<em>nashor 2</em>) and drought-stress tolerance genes (<em>TaGolS 2</em>).</p><p id="p-63">Our comparisons of allele frequency differences among individual barley breeding populations (see <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/TableS16.xls">Table S16</a>) suggest that genetic drift (and likely linked selection) plays a major role in differentiation among these populations. One major factor that can reduce effective population size, and thus increase the role of drift in breeding populations, is the frequent reuse of popular cultivars as parents in breeding programs. Historically, this included founders such as Betzes, Hanna, Lion, Manchuria, and Trebi (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Martin <em>et al.</em> 1991</a>) but more recently lines such as Bonanza, Cheri, and Chevron occur frequently in pedigrees (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Condón <em>et al.</em> 2008</a>).</p><p id="p-64">The cumulative effects of selection and genetic drift over several generations of breeding may result in reduced response to selection (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Hanrahan <em>et al.</em> 1973</a>), with larger effects of drift when the population is small and/or highly inbred (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Robertson 1960</a>). The magnitude of drift suggests that the majority of barley breeding programs have small effective population sizes. <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gerke <em>et al.</em> (2015</a>) noted that genetic drift played a major role in changes in allele frequency over the history of maize breeding in North America. Although, in the larger partitions of barley populations, signals of directional selection are clearly evident at loci of large effect, drift may be extremely important to the loss of variation for variants of small effect (or for phenotypes determined by multiple genes), especially when the selective pressure is sporadic (<em>e.g.</em>, drought tolerance).</p></div><div id="sec-28" class="subsection"><h3>Gene flow among North American breeding populations</h3><p id="p-65">Gene flow between populations can be detected by the presence of large chromosomal regions in (admixture) linkage disequilibrium (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chakraborty and Smouse 1988</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chakraborty and Weiss 1988</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Briscoe <em>et al.</em> 1994</a>) or extended genomic regions of shared ancestry (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Gusev <em>et al.</em> 2009</a>, <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">2012</a>). Shorter shared haplotypes indicate shared ancestry a larger number of generations before present (more generations of recombination). Considering the high frequency of IBS segments (and associated low <em>F</em><sub>ST</sub>) across the genome between populations of similar inflorescence type, we speculate that these segments could reflect a history of shared ancestry among these populations, possibly predating the separation of breeding programs (<a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">Martin <em>et al.</em> 1991</a>).</p><p id="p-66">Gene flow that involves adaptive variation can result in differential retention of genomic segments in the recipient population, and can also be evident as shared genomic regions with reduced diversity (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Hufford <em>et al.</em> 2013</a>). For example, MN6 demonstrates a high degree of IBS with other six-rowed populations (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6A</a>), and these genomic regions have lower average pairwise diversity than other regions when the six-rowed populations are analyzed together (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/FigureS13.tiff">Figure S13</a>).</p><p id="p-67">With regard to IBS, VT6 is the most clearly differentiated from other North American populations, showing similarity only to the other winter populations, OR2 and OR6 (for 50 SNP windows, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/FigureS10.tiff">Figure S10</a>). Pairwise <em>F</em><sub>ST</sub> with other breeding populations and genetic differentiation as measured by PCA (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1/FigureS2.tiff">Figure S2B</a>) suggest that this isolation has been maintained over many generations. Large IBS segments (100 SNP windows) shared between VT6 and OR2 and OR6 (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>) indicate that winter populations have recently exchanged genetic material. The demographic history inferred in the TreeMix analysis supports this relationship (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6</a>), particularly when invoking three migration events, the topology best supported by the data. However, the formal examination of migration using the <em>f<sub>4</sub></em>-test failed to support gene flow among two- and six-rowed winter populations, thus suggesting that haplotype similarities across spike-types are due to ancestral history rather than recent introgression. In addition to the postulated shared ancestry, genetic similarity is expected between these populations under the premise that both Virginia and Oregon breeding populations have been subject to relative recent introgression from leaf rust resistance lines from the International Maize and Wheat Improvement Center (CIMMYT) (C. Griffey and P. Hayes, personal communication). Isolated populations, like VT6, can carry adaptive variants absent in other populations (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Kristiansson <em>et al.</em> 2008</a>). Therefore, VT6 could be used for the identification of new variants for yield or disease resistance, as well as a source of genetic variation to increase diversity in other breeding populations.</p><p id="p-68">The large number of IBS segments shared between the Utah populations and most of the six-rowed populations yielded a significant signal of introgression based on the <em>f<sub>4</sub></em>-test. It is possible that the signal detected here is due to the ongoing genetic exchange between UT2 and UT6 (as documented in pedigrees) since 2001 (D. Hole, personal communication) and the genetic history shared between six-rowed barleys. This may also account for the elevated pairwise diversity in the Utah populations. Additionally, there is evidence of introgression between N2 and two six-rowed populations (WA6 and AB6), but not with N6. Introgression between the six-rowed populations and N2 are uncommon (J. Franckowiak, personal communication), thus this result needs more exploration.</p><p id="p-69">Using the tree of relatedness as our hypothetical relationship, a necessary component of the <em>f<sub>4</sub></em>-test, forces populations of the same spike-type to be assumed as related by shared history. Therefore, gene flow within spike-types is not investigated. Although crosses across spike-type are not common in barley breeding programs due to concerns of recombining desirable alleles with less favorable ones, introgression across spike-type could increase diversity (<a id="xref-ref-46-5" class="xref-bibr" href="#ref-46">Martin <em>et al.</em> 1991</a>). We detect two instances of introgression across spike-type that result in higher levels of genetic diversity.</p></div><div id="sec-29" class="subsection"><h3>Caveats of the analysis</h3><p id="p-70">It is important to note that the results presented here are dependent on the samples submitted for genotyping (<a href="https://triticeaetoolbox.org">https://triticeaetoolbox.org</a>) by each breeding program. Given that one of the goals of the Barley Coordinated Agricultural Project (<a href="http://www.barleycap.org">http://www.barleycap.org</a>) was to evaluate the diversity in the North American barley breeding program, each program was encouraged to submit representative lines. We assume that the data represent the diversity within each breeding program and that the sampling scheme across breeding programs is comparable. Deviations from these assumptions could influence the results in four primary ways: (i) underestimation of the diversity within breeding populations; (ii) overestimation of the role of drift in breeding populations due to reduced representation of the parental lines used in the breeding programs; (iii) over- or underestimation of allele frequency differentiation between partitions of the data; and (iv) excess of shared haplotypes due to accessions highly related by pedigree (<em>e.g.</em>, sibs and half-sibs). With these caveats in mind, we made the best use of the dataset to uncover the underlying genetic response to breeder’s efforts. We encourage readers of this paper and barley breeders to evaluate the results with an understanding of the limitations of the sampling.</p></div><div id="sec-30" class="subsection"><h3>Implications</h3><p id="p-71">Comparative population genetic approaches have the potential to uncover breeding history at a level of detail not previously possible. Our applications of allele frequency differentiation analyses in barley breeding populations are able to identify candidate genes or linked markers controlling major traits. Though the nature of the target of selection identified by individual SNPs is not always readily apparent, the identification of <em>F</em><sub>ST</sub> outliers reported here provides a reasonable first step toward the discovery of genes underlying agronomic adaptation or potential markers linked to these genes.</p><p id="p-72"><em>F</em><sub>ST</sub> analyses also reveal the effects of linked selection and drift on breeding populations. Linkage has the potential to reduce the efficacy of selection by forcing selection to act on the net effect of linked variants (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Hill and Robertson 1966</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Robertson 1970</a>). Furthermore, beneficial variants could be in linkage with deleterious mutations (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Felsenstein 1974</a>). Gains from selection in breeding programs will depend on the disassociation of unfavorable linkage (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Morrell <em>et al.</em> 2012</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Rodgers-Melnick <em>et al.</em> 2015</a>). This can be achieved by increasing the effective amount of recombination. <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Hill and Robertson (1966</a>) suggested that a generation of relaxed selection with a larger number of contributing parents between each generation of selection could increase the amount of recombination in the population, increasing the potential to disassociate these variants.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-73">The authors thank Michael Kantar, Ron Okagaki, and Paul Hoffman for helpful comments on an earlier version of the manuscript. We thank Kate Crosby, Justin Gerke, and Timothy Beissinger for thoughtful reviews that improved the final text. We thank Yaniv Brandvain for helpful suggestions on data analysis. We thank Dr. David Hole, Dr. Patrick Hayes, and Dr. Jerome Franckowiak for sharing information about their breeding programs. This work was performed using computing resources at the University of Minnesota Supercomputing Institute. We acknowledge funding from the US Department of Agriculture National Institute for Food and Agriculture USDA NIFA 2011- 68002-30029 (to P.L.M.), Barley Coordinated Agricultural Project: Leveraging Genomics, Genetics, and Breeding for Gene Discovery and Barley Improvement USDA-CSREES-NRI grant 2006-55606-16722 and USDA-NIFA grant 2009-85606-0570. We also acknowledge funding from University of Minnesota Doctoral Dissertation Fellowship (to A.M.P., T.J.Y.K., and Z.F.).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn-supplementary-material" id="fn-3"><p id="p-74">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.115.024349/-/DC1</a></p></li><li class="fn" id="fn-4"><p id="p-75"><em>Communicating editor: J. Ross-Ibarra</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2015-10-30"><span class="received-label">Received </span>October 30, 2015.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2015-12-25"><span class="accepted-label">Accepted </span>December 25, 2015.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Poets <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.1" data-doi="10.1111/j.1365-294X.2004.02125.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beaumont</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balding</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Identifying adaptive genetic divergence among populations from genome scans.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">969</span>–<span class="cit-lpage">980</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DBeaumont%26rft.auinit1%253DM.%2BA.%26rft.volume%253D13%26rft.issue%253D4%26rft.spage%253D969%26rft.epage%253D980%26rft.atitle%253DIdentifying%2Badaptive%2Bgenetic%2Bdivergence%2Bamong%2Bpopulations%2Bfrom%2Bgenome%2Bscans.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2004.02125.x%26rft_id%253Dinfo%253Apmid%252F15012769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2004.02125.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15012769&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220153000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.2" data-doi="10.1071/AR03014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boyd</span>  <span class="cit-name-given-names">W. J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grime</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cakir</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Potipibool</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Conventional and molecular genetic analysis of factors contributing to variation in the timing of heading among spring barley (<em>Hordeum vulgare L</em>.) genotypes grown over a mild winter growing season.</span> <abbr class="cit-jnl-abbrev">Crop Pasture Sci.</abbr> <span class="cit-vol">54</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">1277</span>–<span class="cit-lpage">1301</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BPasture%2BSci.%26rft.volume%253D54%26rft.spage%253D1277%26rft_id%253Dinfo%253Adoi%252F10.1071%252FAR03014%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/AR03014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Briscoe</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’brien</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Linkage disequilibrium in admixed populations: applications in gene mapping.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">85</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">59</span>–<span class="cit-lpage">63</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DBriscoe%26rft.auinit1%253DD.%26rft.volume%253D85%26rft.issue%253D1%26rft.spage%253D59%26rft.epage%253D63%26rft.atitle%253DLinkage%2BDisequilibrium%2Bin%2BAdmixed%2BPopulations%253A%2BApplications%2Bin%2BGene%2BMapping%26rft_id%253Dinfo%253Apmid%252F8120361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjc6Ijg1LzEvNTkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Caicedo</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williamson</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hernandez</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fledel-Alon</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genome-wide patterns of nucleotide polymorphism in domesticated rice.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">1745</span>–<span class="cit-lpage">1756</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DCaicedo%26rft.auinit1%253DA.%2BL.%26rft.volume%253D3%26rft.issue%253D9%26rft.spage%253D1745%26rft.epage%253D1756%26rft.atitle%253DGenome-wide%2Bpatterns%2Bof%2Bnucleotide%2Bpolymorphism%2Bin%2Bdomesticated%2Brice.%26rft_id%253Dinfo%253Apmid%252F17907810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17907810&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249767800017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.5" data-doi="10.1098/rspb.1966.0038"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cavalli-Sforza</span>  <span class="cit-name-given-names">L. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">Population structure and human evolution.</span> <abbr class="cit-jnl-abbrev">Proc. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">164</span>(995): <span class="cit-fpage">362</span>–<span class="cit-lpage">379</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DCavalli-Sforza%26rft.auinit1%253DL.%2BL.%26rft.volume%253D164%26rft.issue%253D995%26rft.spage%253D362%26rft.epage%253D379%26rft.atitle%253DPopulation%2BStructure%2Band%2BHuman%2BEvolution%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.1966.0038%26rft_id%253Dinfo%253Apmid%252F4379525%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo3OiJyb3lwcnNiIjtzOjU6InJlc2lkIjtzOjExOiIxNjQvOTk1LzM2MiI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMy82MDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.6" data-doi="10.1073/pnas.1217133110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cavanagh</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome-wide comparative diversity uncovers multiple targets of selection for improvement in hexaploid wheat landraces and cultivars.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>(<span class="cit-issue">20</span>): <span class="cit-fpage">8057</span>–<span class="cit-lpage">8062</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1217133110%26rft_id%253Dinfo%253Apmid%252F23630259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEwLzIwLzgwNTciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.7" data-doi="10.1073/pnas.85.9.3071"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chakraborty</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smouse</span>  <span class="cit-name-given-names">P. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Recombination of haplotypes leads to biased estimates of admixture proportions in human populations.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">85</span>(<span class="cit-issue">9</span>): <span class="cit-fpage">3071</span>–<span class="cit-lpage">3074</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DChakraborty%26rft.auinit1%253DR.%26rft.volume%253D85%26rft.issue%253D9%26rft.spage%253D3071%26rft.epage%253D3074%26rft.atitle%253DRecombination%2Bof%2Bhaplotypes%2Bleads%2Bto%2Bbiased%2Bestimates%2Bof%2Badmixture%2Bproportions%2Bin%2Bhuman%2Bpopulations.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.85.9.3071%26rft_id%253Dinfo%253Apmid%252F3362862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI4NS85LzMwNzEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.8" data-doi="10.1073/pnas.85.23.9119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chakraborty</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weiss</span>  <span class="cit-name-given-names">K. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Admixture as a tool for finding linked genes and detecting that difference from allelic association between loci.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">85</span>(<span class="cit-issue">23</span>): <span class="cit-fpage">9119</span>–<span class="cit-lpage">9123</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DChakraborty%26rft.auinit1%253DR.%26rft.volume%253D85%26rft.issue%253D23%26rft.spage%253D9119%26rft.epage%253D9123%26rft.atitle%253DAdmixture%2Bas%2Ba%2Btool%2Bfor%2Bfinding%2Blinked%2Bgenes%2Band%2Bdetecting%2Bthat%2Bdifference%2Bfrom%2Ballelic%2Bassociation%2Bbetween%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.85.23.9119%26rft_id%253Dinfo%253Apmid%252F3194414%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiODUvMjMvOTExOSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMy82MDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.9" data-doi="10.1186/s13742-015-0047-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chow</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tellier</span>  <span class="cit-name-given-names">L. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vattikuti</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Second-generation PLINK: rising to the challenge of larger and richer datasets.</span> <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1186/s13742-015-0047-8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D4%26rft.spage%253D7%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13742-015-0047-8%26rft_id%253Dinfo%253Apmid%252F25722852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13742-015-0047-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25722852&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.10" data-doi="10.1186/1471-2164-10-582"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Close</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhat</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lonardi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rostoks</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Development and implementation of high-throughput SNP genotyping in barley.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">582</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1186/1471-2164-10-582.</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DClose%26rft.auinit1%253DT.%2BJ.%26rft.volume%253D10%26rft.spage%253D582%26rft.epage%253D582%26rft.atitle%253DDevelopment%2Band%2Bimplementation%2Bof%2Bhigh-throughput%2BSNP%2Bgenotyping%2Bin%2Bbarley.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-10-582%26rft_id%253Dinfo%253Apmid%252F19961604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-10-582&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19961604&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.11" data-doi="10.1007/s00122-011-1537-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comadran</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Booth</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pswarayi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ceccarelli</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mixed model association scans of multi-environmental trial data reveal major loci controlling yield and yield related traits in <em>Hordeum vulgare</em> in Mediterranean environments.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">122</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">1363</span>–<span class="cit-lpage">1373</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DComadran%26rft.auinit1%253DJ.%26rft.volume%253D122%26rft.issue%253D7%26rft.spage%253D1363%26rft.epage%253D1373%26rft.atitle%253DMixed%2Bmodel%2Bassociation%2Bscans%2Bof%2Bmulti-environmental%2Btrial%2Bdata%2Breveal%2Bmajor%2Bloci%2Bcontrolling%2Byield%2Band%2Byield%2Brelated%2Btraits%2Bin%2BHordeum%2Bvulgare%2Bin%2BMediterranean%2Benvironments.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1537-4%26rft_id%253Dinfo%253Apmid%252F21279625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1537-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21279625&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.12" data-doi="10.1038/ng.2447"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Comadran</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kilian</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramsay</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stein</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Natural variation in a homolog of <em>Antirrhinum</em> CENTRORADIALIS contributed to spring growth habit and environmental adaptation in cultivated barley.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">1388</span>–<span class="cit-lpage">1392</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DComadran%26rft.auinit1%253DJ.%26rft.volume%253D44%26rft.issue%253D12%26rft.spage%253D1388%26rft.epage%253D1392%26rft.atitle%253DNatural%2Bvariation%2Bin%2Ba%2Bhomolog%2Bof%2BAntirrhinum%2BCENTRORADIALIS%2Bcontributed%2Bto%2Bspring%2Bgrowth%2Bhabit%2Band%2Benvironmental%2Badaptation%2Bin%2Bcultivated%2Bbarley.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2447%26rft_id%253Dinfo%253Apmid%252F23160098%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2447&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23160098&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.13" data-doi="10.2135/cropsci2007.07.0415"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Condón</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gustus</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rasmusson</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Effect of advanced cycle breeding on genetic diversity in barley breeding germplasm.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">48</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1027</span>–<span class="cit-lpage">1036</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DCondon%26rft.auinit1%253DF.%26rft.volume%253D48%26rft.issue%253D3%26rft.spage%253D1027%26rft.epage%253D1036%26rft.atitle%253DEffect%2Bof%2BAdvanced%2BCycle%2BBreeding%2Bon%2BGenetic%2BDiversity%2Bin%2BBarley%2BBreeding%2BGermplasm%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2007.07.0415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2007.07.0415&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000256635400020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.14" data-doi="10.1186/1471-2164-11-707."><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cuesta-Marcos</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szucs</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Close</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Filichkin</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muehlbauer</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genome-wide SNPs and re-sequencing of growth habit and inflorescence genes in barley: implications for association mapping in germplasm arrays varying in size and structure.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">707</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id-scheme">DOI: </span><span class="cit-pub-id cit-pub-id-doi">10.1186/1471-2164-11-707.</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DCuesta-Marcos%26rft.auinit1%253DA.%26rft.volume%253D11%26rft.spage%253D707%26rft.epage%253D707%26rft.atitle%253DGenome-wide%2BSNPs%2Band%2Bre-sequencing%2Bof%2Bgrowth%2Bhabit%2Band%2Binflorescence%2Bgenes%2Bin%2Bbarley%253A%2Bimplications%2Bfor%2Bassociation%2Bmapping%2Bin%2Bgermplasm%2Barrays%2Bvarying%2Bin%2Bsize%2Band%2Bstructure.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-11-707.%26rft_id%253Dinfo%253Apmid%252F21159198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-11-707.&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21159198&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.15" data-doi="10.1534/g3.113.007294"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eule-Nashoba</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Powers</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kono</span>  <span class="cit-name-given-names">T. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takuno</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comparative analyses identify the contributions of exotic donors to disease resistance in a barley experimental population</span>. <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">1945</span>–<span class="cit-lpage">1953</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.007294%26rft_id%253Dinfo%253Apmid%252F24048643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjMvMTEvMTk0NSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMy82MDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.16" data-doi="10.1534/g3.114.010561"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gonzales</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clegg</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muehlbauer</span>  <span class="cit-name-given-names">G. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Two genomic regions contribute disproportionately to geographic differentiation in wild barley.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">1193</span>–<span class="cit-lpage">1203</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.010561%26rft_id%253Dinfo%253Apmid%252F24760390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjQvNy8xMTkzIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzYwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fay</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">C. I.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Hitchhiking under positive Darwinian selection.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1405</span>–<span class="cit-lpage">1413</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFay%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D155%26rft.issue%253D3%26rft.spage%253D1405%26rft.epage%253D1413%26rft.atitle%253DHitchhiking%2BUnder%2BPositive%2BDarwinian%2BSelection%26rft_id%253Dinfo%253Apmid%252F10880498%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1NS8zLzE0MDUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">The evolutionary advantage of recombination.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">78</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">737</span>–<span class="cit-lpage">756</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFelsenstein%26rft.auinit1%253DJ.%26rft.volume%253D78%26rft.issue%253D2%26rft.spage%253D737%26rft.epage%253D756%26rft.atitle%253DTHE%2BEVOLUTIONARY%2BADVANTAGE%2BOF%2BRECOMBINATION%26rft_id%253Dinfo%253Apmid%252F4448362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiNzgvMi83MzciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.19" data-doi="10.1534/genetics.115.182410"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerke</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edwards</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ke</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross-Ibarra</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mcmullen</span>  <span class="cit-name-given-names">M. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The genomic impacts of drift and selection for hybrid performance in maize.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">201</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1201</span>–<span class="cit-lpage">1211</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.182410%26rft_id%253Dinfo%253Apmid%252F26385980%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMS8zLzEyMDEiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.20" data-doi="10.1111/j.1471-8286.2004.00828.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goudet</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Hierfstat, a package for R to compute and test hierarchical F statistics.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Notes</abbr> <span class="cit-vol">5</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">184</span>–<span class="cit-lpage">186</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BNotes%26rft.volume%253D5%26rft.spage%253D184%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2004.00828.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1471-8286.2004.00828.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000227598500059&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.21" data-doi="10.1101/gr.081398.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gusev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lowe</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stoffel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altshuler</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Whole population, genome-wide mapping of hidden relatedness.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">318</span>–<span class="cit-lpage">326</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.081398.108%26rft_id%253Dinfo%253Apmid%252F18971310%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzIvMzE4IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzYwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.22" data-doi="10.1093/molbev/msr133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gusev</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palamara</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aponte</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhuang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Darvasi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The architecture of long-range haplotypes shared within and across populations.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">29</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">473</span>–<span class="cit-lpage">486</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr133%26rft_id%253Dinfo%253Apmid%252F21984068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjI5LzIvNDczIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzYwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.23" data-doi="10.1104/pp.006478"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haake</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Riechmann</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pineda</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomashow</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Transcription factor <em>CBF4</em> is a regulator of drought adaptation in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">130</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">639</span>–<span class="cit-lpage">648</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.006478%26rft_id%253Dinfo%253Apmid%252F12376631%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxMzAvMi82MzkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.24" data-doi="10.2135/cropsci2009.04.0198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hamblin</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Close</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhat</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chao</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kling</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Population structure and linkage disequilibrium in US barley germplasm: implications for association mapping.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">50</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">556</span>–<span class="cit-lpage">566</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHamblin%26rft.auinit1%253DM.%2BT.%26rft.volume%253D50%26rft.issue%253D2%26rft.spage%253D556%26rft.epage%253D566%26rft.atitle%253DPopulation%2BStructure%2Band%2BLinkage%2BDisequilibrium%2Bin%2BU.S.%2BBarley%2BGermplasm%253A%2BImplications%2Bfor%2BAssociation%2BMapping%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.04.0198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.04.0198&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000275564500013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hanrahan</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eisen</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lagates</span>  <span class="cit-name-given-names">J. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Effects of population size and selection intensity of short-term response to selection for postweaning gain in mice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">73</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">513</span>–<span class="cit-lpage">530</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHanrahan%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D73%26rft.issue%253D3%26rft.spage%253D513%26rft.epage%253D530%26rft.atitle%253DEFFECTS%2BOF%2BPOPULATION%2BSIZE%2BAND%2BSELECTION%2BINTENSITY%2BON%2BSHORT-TERM%2BRESPONSE%2BTO%2BSELECTION%2BFOR%2BPOSTWEANING%2BGAIN%2BIN%2BMICE%26rft_id%253Dinfo%253Apmid%252F4700062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiNzMvMy81MTMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.26" data-doi="10.1046/j.1432-1327.1999.00717.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Herbik</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koch</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mock</span>  <span class="cit-name-given-names">H. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dushkov</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Czihal</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Isolation, characterization and cDNA cloning of nicotianamine synthase from barley. A key enzyme for iron homeostasis in plants.</span> <abbr class="cit-jnl-abbrev">Eur. J. Biochem.</abbr> <span class="cit-vol">265</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">231</span>–<span class="cit-lpage">239</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFEBS%2BJournal%26rft.stitle%253DFEBS%2BJ%26rft.aulast%253DHerbik%26rft.auinit1%253DA.%26rft.volume%253D265%26rft.issue%253D1%26rft.spage%253D231%26rft.epage%253D239%26rft.atitle%253DIsolation%252C%2Bcharacterization%2Band%2BcDNA%2Bcloning%2Bof%2Bnicotianamine%2Bsynthase%2Bfrom%2Bbarley.%2BA%2Bkey%2Benzyme%2Bfor%2Biron%2Bhomeostasis%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1432-1327.1999.00717.x%26rft_id%253Dinfo%253Apmid%252F10491178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1432-1327.1999.00717.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10491178&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000082893000027&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.27" data-doi="10.1017/S0016672300010156"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hill</span>  <span class="cit-name-given-names">W. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1966</span> <span class="cit-article-title">The effect of linkage on limits to artificial selection.</span> <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">269</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DHill%26rft.auinit1%253DW.%2BG.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D269%26rft.epage%253D294%26rft.atitle%253DThe%2Beffect%2Bof%2Blinkage%2Bon%2Blimits%2Bto%2Bartificial%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672300010156%26rft_id%253Dinfo%253Apmid%252F5980116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672300010156&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=5980116&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A19668750300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.28" data-doi="10.1038/ng.1042"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Horton</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hancock</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toomajian</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atwell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide patterns of genetic variation in worldwide Arabidopsis thaliana accessions from the RegMap panel.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">212</span>–<span class="cit-lpage">216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHorton%26rft.auinit1%253DM.%2BW.%26rft.volume%253D44%26rft.issue%253D2%26rft.spage%253D212%26rft.epage%253D216%26rft.atitle%253DGenome-wide%2Bpatterns%2Bof%2Bgenetic%2Bvariation%2Bin%2Bworldwide%2BArabidopsis%2Bthaliana%2Baccessions%2Bfrom%2Bthe%2BRegMap%2Bpanel.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.1042%26rft_id%253Dinfo%253Apmid%252F22231484%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.1042&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22231484&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skarecky</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwiatowski</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayala</span>  <span class="cit-name-given-names">F. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Evidence for positive selection in the superoxide dismutase (<em>Sod</em>) region of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">136</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1329</span>–<span class="cit-lpage">1340</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHudson%26rft.auinit1%253DR.%2BR.%26rft.volume%253D136%26rft.issue%253D4%26rft.spage%253D1329%26rft.epage%253D1340%26rft.atitle%253DEvidence%2Bfor%2Bpositive%2Bselection%2Bin%2Bthe%2Bsuperoxide%2Bdismutase%2B%2528Sod%2529%2Bregion%2Bof%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Apmid%252F8013910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzNi80LzEzMjkiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.30" data-doi="10.1038/ng.2309"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hufford</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van Heerwaarden</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pyhäjärvi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chia</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Comparative population genomics of maize domestication and improvement.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">44</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">808</span>–<span class="cit-lpage">811</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHufford%26rft.auinit1%253DM.%2BB.%26rft.volume%253D44%26rft.issue%253D7%26rft.spage%253D808%26rft.epage%253D811%26rft.atitle%253DComparative%2Bpopulation%2Bgenomics%2Bof%2Bmaize%2Bdomestication%2Band%2Bimprovement.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.2309%26rft_id%253Dinfo%253Apmid%252F22660546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.2309&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22660546&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.31" data-doi="10.1371/journal.pgen.1003477"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hufford</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lubinksy</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pyhajarvi</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Devengenzo</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ellstrand</span>  <span class="cit-name-given-names">N. C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The genomic signature of crop-wild introgression in maize.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">e1003477</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003477%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003477%26rft_id%253Dinfo%253Apmid%252F23671421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003477&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23671421&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.32" data-doi="10.1534/genetics.104.032219"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Innan</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marjoram</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tavaré</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rosenberg</span>  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Statistical tests of the coalescent model based on the haplotype frequency distribution and the number of segregating sites.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">169</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">1763</span>–<span class="cit-lpage">1777</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.032219%26rft_id%253Dinfo%253Apmid%252F15654103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2OS8zLzE3NjMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.33" data-doi="10.1093/molbev/msn167"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leigh</span>  <span class="cit-name-given-names">F. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bower</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">L. M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Population-based resequencing reveals that the flowering time adaptation of cultivated barley originated east of the Fertile Crescent.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">25</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">2211</span>–<span class="cit-lpage">2219</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn167%26rft_id%253Dinfo%253Apmid%252F18669581%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjEwOiIyNS8xMC8yMjExIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzYwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.34" data-doi="10.1093/jxb/ern131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karsai</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szucs</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koszegi</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayes</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Effects of photo and thermo cycles on flowering time in barley: a genetical phenomics approach.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">59</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">2707</span>–<span class="cit-lpage">2715</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Fern131%26rft_id%253Dinfo%253Apmid%252F18550600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4Ym90IjtzOjU6InJlc2lkIjtzOjEwOiI1OS8xMC8yNzA3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzYwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.35" data-doi="10.1093/bioinformatics/bts199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kearse</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moir</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stones-Havas</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheung</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Geneious Basic: an integrated and extendable desktop software platform for the organization and analysis of sequence data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">1647</span>–<span class="cit-lpage">1649</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts199%26rft_id%253Dinfo%253Apmid%252F22543367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjgvMTIvMTY0NyI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMy82MDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.36" data-doi="10.1038/ng2116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keinan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mullikin</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Measurement of the human allele frequency spectrum demonstrates greater genetic drift in East Asians than in Europeans.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>(<span class="cit-issue">10</span>): <span class="cit-fpage">1251</span>–<span class="cit-lpage">1255</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKeinan%26rft.auinit1%253DA.%26rft.volume%253D39%26rft.issue%253D10%26rft.spage%253D1251%26rft.epage%253D1255%26rft.atitle%253DMeasurement%2Bof%2Bthe%2Bhuman%2Ballele%2Bfrequency%2Bspectrum%2Bdemonstrates%2Bgreater%2Bgenetic%2Bdrift%2Bin%2BEast%2BAsians%2Bthan%2Bin%2BEuropeans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2116%26rft_id%253Dinfo%253Apmid%252F17828266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2116&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17828266&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249737400021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-book" id="cit-6.3.609.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-source">The Neutral Theory of Molecular Evolution</span>. <span class="cit-publ-name">Cambridge University Press</span>, <span class="cit-publ-loc">Cambridge, United Kingdom</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.38" data-doi="10.1007/s00122-001-0858-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Komatsuda</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mano</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Molecular mapping of the intermedium spike-c (<em>int-c</em>) and non-brittle rachis 1 (<em>btr1</em>) loci in barley (<em>Hordeum vulgare L</em>.).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">105</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">85</span>–<span class="cit-lpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DKomatsuda%26rft.auinit1%253DT.%26rft.volume%253D105%26rft.issue%253D1%26rft.spage%253D85%26rft.epage%253D90%26rft.atitle%253DMolecular%2Bmapping%2Bof%2Bthe%2Bintermedium%2Bspike-c%2B%2528%2Bint-c%2529%2Band%2Bnon-brittle%2Brachis%2B1%2B%2528%2Bbtr1%2529%2Bloci%2Bin%2Bbarley%2B%2528%2BHordeum%2Bvulgare%2BL.%2529.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-001-0858-0%26rft_id%253Dinfo%253Apmid%252F12582565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-001-0858-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12582565&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177178800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.39" data-doi="10.1073/pnas.0608580104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Komatsuda</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pourkheirandish</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Azhaguvel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kanamori</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Six-rowed barley originated from a mutation in a homeodomain-leucine zipper I-class homeobox gene.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1424</span>–<span class="cit-lpage">1429</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0608580104%26rft_id%253Dinfo%253Apmid%252F17220272%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA0LzQvMTQyNCI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMy82MDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.40" data-doi="10.1111/1755-0998.12183"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kono</span>  <span class="cit-name-given-names">T. J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seth</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poland</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">SNPMeta: SNP annotation and SNP Metadata collection without a reference genome.</span> <abbr class="cit-jnl-abbrev">Mol. Ecol. Resour.</abbr> <span class="cit-vol">14</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">419</span>–<span class="cit-lpage">425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BEcol.%2BResour.%26rft.volume%253D14%26rft.spage%253D419%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1755-0998.12183%26rft_id%253Dinfo%253Apmid%252F24237904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1755-0998.12183&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24237904&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.41" data-doi="10.1186/gb-2008-9-8-109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kristiansson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Naukkarinen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peltonen</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Isolated populations and complex disease gene identification.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">8</span>): <span class="cit-fpage">109</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DKristiansson%26rft.auinit1%253DK.%26rft.volume%253D9%26rft.issue%253D8%26rft.spage%253D109%26rft.epage%253D109%26rft.atitle%253DIsolated%2Bpopulations%2Band%2Bcomplex%2Bdisease%2Bgene%2Bidentification.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2008-9-8-109%26rft_id%253Dinfo%253Apmid%252F18771588%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2008-9-8-109&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18771588&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Laurie</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pratchett</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Snape</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bezant</span>  <span class="cit-name-given-names">J. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">RFLP mapping of five major genes and eight quantitative trait loci controlling flowering time in a winter× spring barley (<em>Hordeum vulgare L</em>.) cross.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">38</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">575</span>–<span class="cit-lpage">585</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DLaurie%26rft.auinit1%253DD.%2BA.%26rft.volume%253D38%26rft.issue%253D3%26rft.spage%253D575%26rft.epage%253D585%26rft.atitle%253DRFLP%2Bmapping%2Bof%2Bfive%2Bmajor%2Bgenes%2Band%2Beight%2Bquantitative%2Btrait%2Bloci%2Bcontrolling%2Bflowering%2Btime%2Bin%2Ba%2Bwinter%2Bx%2Bspring%2Bbarley%2B%2528Hordeum%2Bvulgare%2BL.%2529%2Bcross.%26rft_id%253Dinfo%253Apmid%252F18470191%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18470191&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lewontin</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krakauer</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1973</span> <span class="cit-article-title">Distribution of gene frequency as a test of the theory of the selective neutrality of polymorphisms.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">74</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">175</span>–<span class="cit-lpage">195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DLewontin%26rft.auinit1%253DR.%2BC.%26rft.volume%253D74%26rft.issue%253D1%26rft.spage%253D175%26rft.epage%253D195%26rft.atitle%253DDISTRIBUTION%2BOF%2BGENE%2BFREQUENCY%2BAS%2BA%2BTEST%2BOF%2BTHE%2BTHEORY%2BOF%2BTHE%2BSELECTIVE%2BNEUTRALITY%2BOF%2BPOLYMORPHISMS%26rft_id%253Dinfo%253Apmid%252F4711903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiNzQvMS8xNzUiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.44" data-doi="10.1016/j.jas.2008.12.012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lister</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thaw</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bower</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Charles</span>  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Latitudinal variation in a photoperiod response gene in European barley: insight into the dynamics of agricultural spread from ‘historic’ specimens.</span> <abbr class="cit-jnl-abbrev">J. Archaeol. Sci.</abbr> <span class="cit-vol">36</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">1092</span>–<span class="cit-lpage">1098</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BArchaeol.%2BSci.%26rft.volume%253D36%26rft.spage%253D1092%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jas.2008.12.012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jas.2008.12.012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000264081100016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-book" id="cit-6.3.609.45"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marchini</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-source"><em>Statistical and Population Genetics</em></span>. <span class="cit-publ-name">CRAN</span>. Available at: <a href="http://www.stats.ox.ac.uk/~marchini/software.html">http://www.stats.ox.ac.uk/~marchini/software.html</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.46" data-doi="10.2135/cropsci1991.0011183X003100050009x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blake</span>  <span class="cit-name-given-names">T. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hockett</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Diversity among North American spring barley cultivars based on coefficients of parentage.</span> <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">31</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">1131</span>–<span class="cit-lpage">1137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DMartin%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D31%26rft.issue%253D5%26rft.spage%253D1131%26rft.epage%253D1137%26rft.atitle%253DDiversity%2Bamong%2BNorth%2BAmerican%2BSpring%2BBarley%2BCultivars%2BBased%2Bon%2BCoefficients%2Bof%2BParentage%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci1991.0011183X003100050009x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci1991.0011183X003100050009x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1991GH12300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mayer</span>  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Waugh</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langridge</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Close</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wise</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A physical, genetic and functional sequence assembly of the barley genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">491</span>(<span class="cit-issue">7456</span>): <span class="cit-fpage">711</span>–<span class="cit-lpage">716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMayer%26rft.auinit1%253DK.%2BF.%26rft.volume%253D491%26rft.issue%253D7426%26rft.spage%253D711%26rft.epage%253D716%26rft.atitle%253DA%2Bphysical%252C%2Bgenetic%2Band%2Bfunctional%2Bsequence%2Bassembly%2Bof%2Bthe%2Bbarley%2Bgenome.%26rft_id%253Dinfo%253Apmid%252F23075845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23075845&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311606000035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morrell</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buckler</span>  <span class="cit-name-given-names">E. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross-Ibarra</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Crop genomics: advances and applications.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">85</span>–<span class="cit-lpage">96</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D13%26rft.spage%253D85%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.49" data-doi="10.3835/plantgenome2011.08.0023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muñoz-Amatriaín</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moscou</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bhat</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Svensson</span>  <span class="cit-name-given-names">J. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartoš</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">An improved consensus linkage map of barley based on flow-sorted chromosomes and single nucleotide polymorphism markers.</span> <abbr class="cit-jnl-abbrev">The Plant Genome Journal</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">238</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BGenome%2BJournal%26rft.volume%253D4%26rft.spage%253D238%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.08.0023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.08.0023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.50" data-doi="10.1111/1467-9868.00357"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nicholson</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jónsson</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gústafsson</span>  <span class="cit-name-given-names">Ó.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stefánsson</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Assessing population differentiation and isolation from single‐nucleotide polymorphism data.</span> <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. Series B Stat. Methodol.</abbr> <span class="cit-vol">64</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">695</span>–<span class="cit-lpage">715</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BSeries%2BB%2BStat.%2BMethodol.%26rft.volume%253D64%26rft.spage%253D695%26rft_id%253Dinfo%253Adoi%252F10.1111%252F1467-9868.00357%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/1467-9868.00357&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hellmann</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hubisz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustamante</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Recent and ongoing selection in the human genome.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">857</span>–<span class="cit-lpage">868</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DNielsen%26rft.auinit1%253DR.%26rft.volume%253D8%26rft.issue%253D11%26rft.spage%253D857%26rft.epage%253D868%26rft.atitle%253DRecent%2Band%2Bongoing%2Bselection%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Apmid%252F17943193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17943193&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.52" data-doi="10.1016/S0168-9525(02)02557-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tavaré</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Linkage disequilibrium: what history has to tell us.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">18</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">83</span>–<span class="cit-lpage">90</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DNordborg%26rft.auinit1%253DM.%26rft.volume%253D18%26rft.issue%253D2%26rft.spage%253D83%26rft.epage%253D90%26rft.atitle%253DLinkage%2Bdisequilibrium%253A%2Bwhat%2Bhistory%2Bhas%2Bto%2Btell%2Bus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252802%252902557-X%26rft_id%253Dinfo%253Apmid%252F11818140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(02)02557-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11818140&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000173596300009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.53" data-doi="10.1371/journal.pgen.0020190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Population structure and eigenanalysis.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">e190</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DPatterson%26rft.auinit1%253DN.%26rft.volume%253D2%26rft.issue%253D12%26rft.spage%253De190%26rft.epage%253De190%26rft.atitle%253DPopulation%2Bstructure%2Band%2Beigenanalysis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020190%26rft_id%253Dinfo%253Apmid%252F17194218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020190&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17194218&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.54" data-doi="10.1371/journal.pgen.1002967"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pickrell</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Inference of population splits and mixtures from genome-wide allele frequency data.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>(<span class="cit-issue">11</span>): <span class="cit-fpage">e1002967</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DPickrell%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D8%26rft.issue%253D11%26rft.spage%253De1002967%26rft.epage%253De1002967%26rft.atitle%253DInference%2Bof%2Bpopulation%2Bsplits%2Band%2Bmixtures%2Bfrom%2Bgenome-wide%2Ballele%2Bfrequency%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002967%26rft_id%253Dinfo%253Apmid%252F23166502%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002967&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23166502&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.55" data-doi="10.1186/s13059-015-0712-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Poets</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clegg</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morrell</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Barley landraces are characterized by geographically heterogeneous genomic origins.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">16</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">173</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D16%26rft.spage%253D173%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13059-015-0712-3%26rft_id%253Dinfo%253Apmid%252F26293830%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13059-015-0712-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26293830&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.56" data-doi="10.1016/j.cub.2009.11.055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pickrell</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genetics of human adaptation: hard sweeps, soft sweeps, and polygenic adaptation.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">20</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">R208</span>–<span class="cit-lpage">R215</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D20%26rft.issue%253D4%26rft.spage%253DR208%26rft.epage%253DR215%26rft.atitle%253DThe%2Bgenetics%2Bof%2Bhuman%2Badaptation%253A%2Bhard%2Bsweeps%252C%2Bsoft%2Bsweeps%252C%2Band%2Bpolygenic%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2009.11.055%26rft_id%253Dinfo%253Apmid%252F20178769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2009.11.055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20178769&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275149200026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-book" id="cit-6.3.609.57"><div class="cit-metadata"><ol class="cit-ed-list"><li><span class="cit-ed cit-collab">R Core Team</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-source">R: A language and environment for statistical computing</span>. <span class="cit-publ-name">R Foundation for Statistical Computing</span>, <span class="cit-publ-loc">Vienna, Austria</span>. Available at: <a href="http://www.R-project.org/">http://www.R-project.org/</a>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.58" data-doi="10.1534/genetics.110.119594"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ralph</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coop</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Parallel adaptation: one or many waves of advance of an advantageous allele?</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">647</span>–<span class="cit-lpage">668</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.119594%26rft_id%253Dinfo%253Apmid%252F20660645%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNjQ3IjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzYwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.59" data-doi="10.1038/ng.745"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramsay</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Comadran</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Druka</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marshall</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">W. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title"><em>INTERMEDIUM-C</em>, a modifier of lateral spikelet fertility in barley, is an ortholog of the maize domestication gene <em>TEOSINTE BRANCHED 1</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">169</span>–<span class="cit-lpage">172</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DRamsay%26rft.auinit1%253DL.%26rft.volume%253D43%26rft.issue%253D2%26rft.spage%253D169%26rft.epage%253D172%26rft.atitle%253DINTERMEDIUM-C%252C%2Ba%2Bmodifier%2Bof%2Blateral%2Bspikelet%2Bfertility%2Bin%2Bbarley%252C%2Bis%2Ban%2Bortholog%2Bof%2Bthe%2Bmaize%2Bdomestication%2Bgene%2BTEOSINTE%2BBRANCHED%2B1.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.745%26rft_id%253Dinfo%253Apmid%252F21217754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.745&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21217754&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286623800018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.60" data-doi="10.1038/nature08365"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Reich</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thangaraj</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Reconstructing Indian population history.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">461</span>(<span class="cit-issue">7263</span>): <span class="cit-fpage">489</span>–<span class="cit-lpage">494</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DReich%26rft.auinit1%253DD.%26rft.volume%253D461%26rft.issue%253D7263%26rft.spage%253D489%26rft.epage%253D494%26rft.atitle%253DReconstructing%2BIndian%2Bpopulation%2Bhistory.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08365%26rft_id%253Dinfo%253Apmid%252F19779445%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08365&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19779445&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270082900032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.61" data-doi="10.1098/rspb.1960.0099"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robertson</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1960</span> <span class="cit-article-title">A theory of limits in artificial selection.</span> <abbr class="cit-jnl-abbrev">Proc. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">153</span>(<span class="cit-issue">951</span>): <span class="cit-fpage">234</span>–<span class="cit-lpage">249</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProceedings%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DProc%2BR%2BSoc%2BB%26rft.aulast%253DRobertson%26rft.auinit1%253DA.%26rft.volume%253D153%26rft.issue%253D951%26rft.spage%253D234%26rft.epage%253D249%26rft.atitle%253DA%2BTheory%2Bof%2BLimits%2Bin%2BArtificial%2BSelection%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frspb.1960.0099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Nzoicm95cHJzYiI7czo1OiJyZXNpZCI7czoxMToiMTUzLzk1MS8yMzQiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-book" id="cit-6.3.609.62"><div class="cit-metadata unstructured">Robertson, A., 1970 A theory of limits in artificial selection with many linked loci, pp. 246–288 in <em>Mathematical Topics in Population Genetics</em>, edited by K.- Kojima. Springer, Berlin-New York.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.63" data-doi="10.1073/pnas.1413864112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rodgers-Melnick</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acharya</span>  <span class="cit-name-given-names">C. B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Recombination in diverse maize is stable, predictable, and associated with genetic load.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>(<span class="cit-issue">12</span>): <span class="cit-fpage">3823</span>–<span class="cit-lpage">3828</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1413864112%26rft_id%253Dinfo%253Apmid%252F25775595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzEyLzM4MjMiO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.64" data-doi="10.1073/pnas.0700643104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ross-Ibarra</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morrell</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gaut</span>  <span class="cit-name-given-names">B. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Plant domestication, a unique opportunity to identify the genetic basis of adaptation.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">8641</span>–<span class="cit-lpage">8648</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0700643104%26rft_id%253Dinfo%253Apmid%252F17494757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxNjoiMTA0L3N1cHBsXzEvODY0MSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMy82MDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.65" data-doi="10.1086/502802"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Scheet</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stephens</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A fast and flexible statistical model for large-scale population genotype data: applications to inferring missing genotypes and haplotypic phase.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">78</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">629</span>–<span class="cit-lpage">644</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DScheet%26rft.auinit1%253DP.%26rft.volume%253D78%26rft.issue%253D4%26rft.spage%253D629%26rft.epage%253D644%26rft.atitle%253DA%2Bfast%2Band%2Bflexible%2Bstatistical%2Bmodel%2Bfor%2Blarge-scale%2Bpopulation%2Bgenotype%2Bdata%253A%2Bapplications%2Bto%2Binferring%2Bmissing%2Bgenotypes%2Band%2Bhaplotypic%2Bphase.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F502802%26rft_id%253Dinfo%253Apmid%252F16532393%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/502802&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16532393&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236755900009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.66" data-doi="10.1007/s11103-005-2498-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Skinner</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Von Zitzewitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szucs</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marquez-Cedillo</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Filichkin</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Structural, functional, and phylogenetic characterization of a large CBF gene family in barley.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">59</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">533</span>–<span class="cit-lpage">551</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DSkinner%26rft.auinit1%253DJ.%2BS.%26rft.volume%253D59%26rft.issue%253D4%26rft.spage%253D533%26rft.epage%253D551%26rft.atitle%253DStructural%252C%2Bfunctional%252C%2Band%2Bphylogenetic%2Bcharacterization%2Bof%2Ba%2Blarge%2BCBF%2Bgene%2Bfamily%2Bin%2Bbarley.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11103-005-2498-2%26rft_id%253Dinfo%253Apmid%252F16244905%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11103-005-2498-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16244905&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232795800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.67" data-doi="10.1126/science.3576198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slatkin</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Gene flow and the geographic structure of natural populations.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">236</span>(<span class="cit-issue">4803</span>): <span class="cit-fpage">787</span>–<span class="cit-lpage">792</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSlatkin%26rft.auinit1%253DM%26rft.volume%253D236%26rft.issue%253D4803%26rft.spage%253D787%26rft.epage%253D792%26rft.atitle%253DGene%2Bflow%2Band%2Bthe%2Bgeographic%2Bstructure%2Bof%2Bnatural%2Bpopulations%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.3576198%26rft_id%253Dinfo%253Apmid%252F3576198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyMzYvNDgwMy83ODciO3M6NDoiYXRvbSI7czoxNzoiL2dnZy82LzMvNjA5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.68" data-doi="10.1046/j.0960-7412.2001.01227.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taji</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohsumi</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Iuchi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seki</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasuga</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Important roles of drought- and cold-inducible genes for galactinol synthase in stress tolerance in Arabidopsis thaliana.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">29</span>(<span class="cit-issue">4</span>): <span class="cit-fpage">417</span>–<span class="cit-lpage">426</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DTaji%26rft.auinit1%253DT.%26rft.volume%253D29%26rft.issue%253D4%26rft.spage%253D417%26rft.epage%253D426%26rft.atitle%253DImportant%2Broles%2Bof%2Bdrought-%2Band%2Bcold-inducible%2Bgenes%2Bfor%2Bgalactinol%2Bsynthase%2Bin%2Bstress%2Btolerance%2Bin%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.0960-7412.2001.01227.x%26rft_id%253Dinfo%253Apmid%252F11846875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.0960-7412.2001.01227.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11846875&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174219300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.69" data-doi="10.1093/bioinformatics/btg316"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">libsequence: a C++ class library for evolutionary genetic analysis.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>(<span class="cit-issue">17</span>): <span class="cit-fpage">2325</span>–<span class="cit-lpage">2327</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DThornton%26rft.auinit1%253DK.%26rft.volume%253D19%26rft.issue%253D17%26rft.spage%253D2325%26rft.epage%253D2327%26rft.atitle%253Dlibsequence%253A%2Ba%2BC%252B%252B%2Bclass%2Blibrary%2Bfor%2Bevolutionary%2Bgenetic%2Banalysis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg316%26rft_id%253Dinfo%253Apmid%252F14630667%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMTkvMTcvMjMyNSI7czo0OiJhdG9tIjtzOjE3OiIvZ2dnLzYvMy82MDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.70" data-doi="10.1371/journal.pgen.1003258"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foran</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Long</span>  <span class="cit-name-given-names">A. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Properties and modeling of GWAS when complex disease risk is due to non-complementing, deleterious mutations in genes of large effect.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>(<span class="cit-issue">2</span>): <span class="cit-fpage">e1003258</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003258%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003258%26rft_id%253Dinfo%253Apmid%252F23437004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003258&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23437004&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.71" data-doi="10.1371/journal.pbio.0040137"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Toomajian</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aranzana</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lister</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A nonparametric test reveals selection for rapid flowering in the <em>Arabidopsis</em> genome.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">4</span>(<span class="cit-issue">5</span>): <span class="cit-fpage">e137</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DToomajian%26rft.auinit1%253DC.%26rft.volume%253D4%26rft.issue%253D5%26rft.spage%253De137%26rft.epage%253De137%26rft.atitle%253DA%2Bnonparametric%2Btest%2Breveals%2Bselection%2Bfor%2Brapid%2Bflowering%2Bin%2Bthe%2BArabidopsis%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0040137%26rft_id%253Dinfo%253Apmid%252F16623598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0040137&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16623598&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.72" data-doi="10.1146/annurev.arplant.59.032607.092755"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turck</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fornara</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coupland</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Regulation and identity of florigen: FLOWERING LOCUS T moves center stage.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Plant Biol.</abbr> <span class="cit-vol">59</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">594</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bplant%2Bbiology%26rft.stitle%253DAnnu%2BRev%2BPlant%2BBiol%26rft.aulast%253DTurck%26rft.auinit1%253DF.%26rft.volume%253D59%26rft.spage%253D573%26rft.epage%253D594%26rft.atitle%253DRegulation%2Band%2Bidentity%2Bof%2Bflorigen%253A%2BFLOWERING%2BLOCUS%2BT%2Bmoves%2Bcenter%2Bstage.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.arplant.59.032607.092755%26rft_id%253Dinfo%253Apmid%252F18444908%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.arplant.59.032607.092755&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18444908&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256593200023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.73" data-doi="10.1126/science.1117619"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Turner</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beales</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Faure</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dunford</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laurie</span>  <span class="cit-name-given-names">D. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">The pseudo-response regulator <em>Ppd-H1</em> provides adaptation to photoperiod in barley.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">310</span>(<span class="cit-issue">5750</span>): <span class="cit-fpage">1031</span>–<span class="cit-lpage">1034</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTurner%26rft.auinit1%253DA.%26rft.volume%253D310%26rft.issue%253D5750%26rft.spage%253D1031%26rft.epage%253D1034%26rft.atitle%253DThe%2BPseudo-Response%2BRegulator%2BPpd-H1%2BProvides%2BAdaptation%2Bto%2BPhotoperiod%2Bin%2BBarley%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1117619%26rft_id%253Dinfo%253Apmid%252F16284181%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTAvNTc1MC8xMDMxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzYwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.74" data-doi="10.1073/pnas.1209275109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Heerwaarden</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hufford</span>  <span class="cit-name-given-names">M. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross-Ibarra</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Historical genomics of North American maize.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>(<span class="cit-issue">31</span>): <span class="cit-fpage">12420</span>–<span class="cit-lpage">12425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1209275109%26rft_id%253Dinfo%253Apmid%252F22802642%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA5LzMxLzEyNDIwIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzYwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.75" data-doi="10.1007/s11103-005-0351-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">von Zitzewitz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szucs</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dubcovsky</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Francia</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Molecular and structural characterization of barley vernalization genes.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">59</span>(<span class="cit-issue">3</span>): <span class="cit-fpage">449</span>–<span class="cit-lpage">467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253Dvon%2BZitzewitz%26rft.auinit1%253DJ.%26rft.volume%253D59%26rft.issue%253D3%26rft.spage%253D449%26rft.epage%253D467%26rft.atitle%253DMolecular%2Band%2Bstructural%2Bcharacterization%2Bof%2Bbarley%2Bvernalization%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11103-005-0351-2%26rft_id%253Dinfo%253Apmid%252F16235110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11103-005-0351-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16235110&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232694400005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.76" data-doi="10.1007/s00122-011-1691-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">K. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Combs</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blake</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horsley</span>  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Effect of population size and unbalanced data sets on QTL detection using genome-wide association mapping in barley breeding germplasm.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">124</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">111</span>–<span class="cit-lpage">124</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DWang%26rft.auinit1%253DH.%26rft.volume%253D124%26rft.issue%253D1%26rft.spage%253D111%26rft.epage%253D124%26rft.atitle%253DEffect%2Bof%2Bpopulation%2Bsize%2Band%2Bunbalanced%2Bdata%2Bsets%2Bon%2BQTL%2Bdetection%2Busing%2Bgenome-wide%2Bassociation%2Bmapping%2Bin%2Bbarley%2Bbreeding%2Bgermplasm.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-011-1691-8%26rft_id%253Dinfo%253Apmid%252F21898052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-011-1691-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21898052&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.77" data-doi="10.2307/210618"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weaver</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1943</span> <span class="cit-article-title">Barley in the United States: a historical sketch.</span> <abbr class="cit-jnl-abbrev">Geogr. Rev.</abbr> <span class="cit-vol">33</span>(<span class="cit-issue">1</span>): <span class="cit-fpage">56</span>–<span class="cit-lpage">73</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGeogr.%2BRev.%26rft.volume%253D33%26rft.spage%253D56%26rft_id%253Dinfo%253Adoi%252F10.2307%252F210618%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/210618&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-book" id="cit-6.3.609.78"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weaver</span>  <span class="cit-name-given-names">J. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1950</span> <span class="cit-source">American Barley Production: A Study in Agricultural Geography</span>. <span class="cit-publ-name">Burgess Pub. Co.</span>, <span class="cit-publ-loc">Minneapolis, MN</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.79" data-doi="10.2307/2408641"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cockerham</span>  <span class="cit-name-given-names">C. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Estimating F-statistics for the analysis of population structure.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">38</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">1358</span>–<span class="cit-lpage">1370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D38%26rft.spage%253D1358%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2408641%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2408641&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1984TY40400017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-other" id="cit-6.3.609.80"><div class="cit-metadata unstructured">Wiebe, G. A., and D. A. Reid, 1961 Classification of barley varieties growing in the United States and Canada in 1958. US Department of Agriculture 210.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.81" data-doi="10.1073/pnas.0607142103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yan</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blechl</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tranquilli</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The wheat and barley vernalization gene <em>VRN3</em> is an orthologue of <em>FT</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>(<span class="cit-issue">51</span>): <span class="cit-fpage">19581</span>–<span class="cit-lpage">19586</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0607142103%26rft_id%253Dinfo%253Apmid%252F17158798%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzUxLzE5NTgxIjtzOjQ6ImF0b20iO3M6MTc6Ii9nZ2cvNi8zLzYwOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.82" data-doi="10.1007/s10722-011-9759-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Youssef</span>  <span class="cit-name-given-names">H. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koppolu</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schnurbusch</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Re-sequencing of vrs1 and int-c loci shows that labile barleys (<em>Hordeum vulgare convar. labile</em>) have a six-rowed genetic background.</span> <abbr class="cit-jnl-abbrev">Genet. Resour. Crop Evol.</abbr> <span class="cit-vol">59</span>(<span class="cit-issue">7</span>): <span class="cit-fpage">1319</span>–<span class="cit-lpage">1328</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BResour.%2BCrop%2BEvol.%26rft.volume%253D59%26rft.spage%253D1319%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10722-011-9759-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10722-011-9759-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-6.3.609.83" data-doi="10.1631/jzus.B1200003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Muehlbauer</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steffenson</span>  <span class="cit-name-given-names">B. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Population structure and linkage disequilibrium in elite barley breeding germplasm from the United States.</span> <abbr class="cit-jnl-abbrev">J. Zhejiang Univ. Sci. B</abbr> <span class="cit-vol">13</span>(<span class="cit-issue">6</span>): <span class="cit-fpage">438</span>–<span class="cit-lpage">451</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BZhejiang%2BUniv%2BSci%2BB%26rft.aulast%253DZhou%26rft.auinit1%253DH.%26rft.volume%253D13%26rft.issue%253D6%26rft.spage%253D438%26rft.epage%253D451%26rft.atitle%253DPopulation%2Bstructure%2Band%2Blinkage%2Bdisequilibrium%2Bin%2Belite%2Bbarley%2Bbreeding%2Bgermplasm%2Bfrom%2Bthe%2BUnited%2BStates.%26rft_id%253Dinfo%253Adoi%252F10.1631%252Fjzus.B1200003%26rft_id%253Dinfo%253Apmid%252F22661207%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1631/jzus.B1200003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22661207&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F3%2F609.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/3/609.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/3/599" title="Genetic Correlates of Individual Differences in Sleep Behavior of Free-Living Great Tits (Parus major)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/3/623" title="Suppression of F1 Male-Specific Lethality in Caenorhabditis Hybrids by cbr-him-8" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/3"><span>Volume 6 Issue 3, March 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/3.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/3.cover-source.jpg?itok=8EDC4x5t" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (3)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/3/609" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-RUQKyVE2VU4j1EWUo_r2D5Jrrgoemjsuyxn7RgfSy3M" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/3/609" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/3/609" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/3/609" class="active" data-icon-position="" data-hide-link-title="0">The Effects of Both Recent and Long-Term Selection and Genetic Drift Are Readily Evident in North American Barley Breeding Populations</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460099" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-89bk5PTjAuFfCSJstICNG_bkfo0eIdMho3q6Z-b_xMc" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/3/609.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/3/609" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-hZwAFVS2AqmQc9-oYznbFHScPvxaAHslgzaCTsMp3AE" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/3/609" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/3/609.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460099" id="node460099--2" data-pisa="ggg;6/3/609" data-pisa-master="ggg;g3.115.024349" data-apath="/ggg/6/3/609.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Effects of Both Recent and Long-Term Selection and Genetic Drift Are Readily Evident in North American Barley Breeding Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4332-5986" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ana M. Poets</span>, <span class="highwire-citation-author" data-delta="1">Mohsen Mohammadi</span>, <span class="highwire-citation-author" data-delta="2">Kiran Seth</span>, <span class="highwire-citation-author" data-delta="3">Hongyun Wang</span>, <span class="highwire-citation-author" data-delta="4">Thomas J. Y. Kono</span>, <span class="highwire-citation-author" data-delta="5">Zhou Fang</span>, <span class="highwire-citation-author" data-delta="6">Gary J. Muehlbauer</span>, <span class="highwire-citation-author" data-delta="7">Kevin P. Smith</span> and <span class="highwire-citation-author" data-delta="8">Peter L. Morrell</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">609-622;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.024349 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460099--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ana M. Poets </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ana%2BM.%2BPoets%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poets%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAna%2BM.%2BPoets%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4332-5986" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ana M. Poets</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:agonzale@umn.edu" class="" data-icon-position="" data-hide-link-title="0">agonzale@umn.edu</a>
<a href="mailto:pmorrell@umn.edu" class="" data-icon-position="" data-hide-link-title="0">pmorrell@umn.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mohsen Mohammadi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mohsen%2BMohammadi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mohammadi%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMohsen%2BMohammadi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kiran Seth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kiran%2BSeth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Seth%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKiran%2BSeth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Hongyun Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>DuPont Pioneer, Mankato, Minnesota 56001</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hongyun%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHongyun%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Thomas J. Y. Kono </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BJ.%2BY.%2BKono%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kono%20TJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BJ.%2BY.%2BKono%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Zhou Fang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhou%2BFang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhou%2BFang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Gary J. Muehlbauer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BJ.%2BMuehlbauer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Muehlbauer%20GJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGary%2BJ.%2BMuehlbauer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kevin P. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BP.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20KP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKevin%2BP.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Peter L. Morrell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BL.%2BMorrell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morrell%20PL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BL.%2BMorrell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:agonzale@umn.edu" class="" data-icon-position="" data-hide-link-title="0">agonzale@umn.edu</a>
<a href="mailto:pmorrell@umn.edu" class="" data-icon-position="" data-hide-link-title="0">pmorrell@umn.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/3/609" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/3/609&amp;title=The%20Effects%20of%20Both%20Recent%20and%20Long-Term%20Selection%20and%20Genetic%20Drift%20Are%20Readily%20Evident%20in%20North%20American%20Barley%20Breeding%20Populations" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/3/609&amp;title=The%20Effects%20of%20Both%20Recent%20and%20Long-Term%20Selection%20and%20Genetic%20Drift%20Are%20Readily%20Evident%20in%20North%20American%20Barley%20Breeding%20Populations" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/3/609&amp;title=The%20Effects%20of%20Both%20Recent%20and%20Long-Term%20Selection%20and%20Genetic%20Drift%20Are%20Readily%20Evident%20in%20North%20American%20Barley%20Breeding%20Populations" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/3/609&amp;text=The%20Effects%20of%20Both%20Recent%20and%20Long-Term%20Selection%20and%20Genetic%20Drift%20Are%20Readily%20Evident%20in%20North%20American%20Barley%20Breeding%20Populations" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/3/609&amp;title=The%20Effects%20of%20Both%20Recent%20and%20Long-Term%20Selection%20and%20Genetic%20Drift%20Are%20Readily%20Evident%20in%20North%20American%20Barley%20Breeding%20Populations" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/3/609&amp;t=The%20Effects%20of%20Both%20Recent%20and%20Long-Term%20Selection%20and%20Genetic%20Drift%20Are%20Readily%20Evident%20in%20North%20American%20Barley%20Breeding%20Populations" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/3/609&amp;title=The%20Effects%20of%20Both%20Recent%20and%20Long-Term%20Selection%20and%20Genetic%20Drift%20Are%20Readily%20Evident%20in%20North%20American%20Barley%20Breeding%20Populations" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/3/609&amp;title=The%20Effects%20of%20Both%20Recent%20and%20Long-Term%20Selection%20and%20Genetic%20Drift%20Are%20Readily%20Evident%20in%20North%20American%20Barley%20Breeding%20Populations" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460099/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460099" id="node460099--3" data-pisa="ggg;6/3/609" data-pisa-master="ggg;g3.115.024349" data-apath="/ggg/6/3/609.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">The Effects of Both Recent and Long-Term Selection and Genetic Drift Are Readily Evident in North American Barley Breeding Populations</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-4332-5986" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ana M. Poets</span>, <span class="highwire-citation-author" data-delta="1">Mohsen Mohammadi</span>, <span class="highwire-citation-author" data-delta="2">Kiran Seth</span>, <span class="highwire-citation-author" data-delta="3">Hongyun Wang</span>, <span class="highwire-citation-author" data-delta="4">Thomas J. Y. Kono</span>, <span class="highwire-citation-author" data-delta="5">Zhou Fang</span>, <span class="highwire-citation-author" data-delta="6">Gary J. Muehlbauer</span>, <span class="highwire-citation-author" data-delta="7">Kevin P. Smith</span> and <span class="highwire-citation-author" data-delta="8">Peter L. Morrell</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">March 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 3 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">609-622;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.115.024349 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460099--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Ana M. Poets </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ana%2BM.%2BPoets%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poets%20AM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAna%2BM.%2BPoets%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-4332-5986" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ana M. Poets</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:agonzale@umn.edu" class="" data-icon-position="" data-hide-link-title="0">agonzale@umn.edu</a>
<a href="mailto:pmorrell@umn.edu" class="" data-icon-position="" data-hide-link-title="0">pmorrell@umn.edu</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mohsen Mohammadi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mohsen%2BMohammadi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mohammadi%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMohsen%2BMohammadi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Kiran Seth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kiran%2BSeth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Seth%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKiran%2BSeth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Hongyun Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>DuPont Pioneer, Mankato, Minnesota 56001</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hongyun%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHongyun%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Thomas J. Y. Kono </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thomas%2BJ.%2BY.%2BKono%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kono%20TJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThomas%2BJ.%2BY.%2BKono%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Zhou Fang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhou%2BFang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fang%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZhou%2BFang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Gary J. Muehlbauer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BJ.%2BMuehlbauer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Muehlbauer%20GJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGary%2BJ.%2BMuehlbauer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Kevin P. Smith </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Kevin%2BP.%2BSmith%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smith%20KP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AKevin%2BP.%2BSmith%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Peter L. Morrell </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Agronomy and Plant Genetics, University of Minnesota, Saint Paul, Minnesota 55108</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Peter%2BL.%2BMorrell%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morrell%20PL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APeter%2BL.%2BMorrell%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:agonzale@umn.edu" class="" data-icon-position="" data-hide-link-title="0">agonzale@umn.edu</a>
<a href="mailto:pmorrell@umn.edu" class="" data-icon-position="" data-hide-link-title="0">pmorrell@umn.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=609&amp;rft.epage=622&amp;rft.atitle=The%20Effects%20of%20Both%20Recent%20and%20Long-Term%20Selection%20and%20Genetic%20Drift%20Are%20Readily%20Evident%20in%20North%20American%20Barley%20Breeding%20Populations&amp;rft.volume=6&amp;rft.issue=3&amp;rft.date=2016-03-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Poets%2C+Ana+M.&amp;rft.au=Mohammadi%2C+Mohsen&amp;rft.au=Seth%2C+Kiran&amp;rft.au=Wang%2C+Hongyun&amp;rft.au=Kono%2C+Thomas+J.+Y.&amp;rft.au=Fang%2C+Zhou&amp;rft.au=Muehlbauer%2C+Gary+J.&amp;rft.au=Smith%2C+Kevin+P.&amp;rft.au=Morrell%2C+Peter+L."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460099/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460099/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460099/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460099/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460099/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460099/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460099/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460099/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460099/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460099/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460099/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460099/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/3/609" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-24" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/3/609.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/3/609.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/3/609.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/3/609.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
