<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/8/2405" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/8/2405.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/8/2405.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/8/2405.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/8/2405.atom" />
<meta name="HW.pisa" content="ggg;9/8/2405" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Population Genome Sequencing of the Scab Fungal Species Venturia inaequalis, Venturia pirina, Venturia aucupariae and Venturia asperata" />
<meta name="DC.Identifier" content="10.1534/g3.119.400047" />
<meta name="DC.Date" content="2019-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Le Cam et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The Venturia genus comprises fungal species that are pathogens on Rosaceae host plants, including V. inaequalis and V. asperata on apple, V. aucupariae on sorbus and V. pirina on pear. Although the genetic structure of V. inaequalis populations has been investigated in detail, genomic features underlying these subdivisions remain poorly understood. Here, we report whole genome sequencing of 87 Venturia strains that represent each species and each population within V. inaequalis . We present a PacBio genome assembly for the V. inaequalis EU-B04 reference isolate. The size of selected genomes was determined by flow cytometry, and varied from 45 to 93 Mb. Genome assemblies of V. inaequalis and V. aucupariae contain a high content of transposable elements (TEs), most of which belong to the Gypsy or Copia LTR superfamilies and have been inactivated by Repeat-Induced Point mutations. The reference assembly of V. inaequalis presents a mosaic structure of GC-equilibrated regions that mainly contain predicted genes and AT-rich regions, mainly composed of TEs. Six pairs of strains were identified as clones. Single-Nucleotide Polymorphism (SNP) analysis between these clones revealed a high number of SNPs that are mostly located in AT-rich regions due to misalignments and allowed determining a false discovery rate. The availability of these genome sequences is expected to stimulate genetics and population genomics research of Venturia pathogens. Especially, it will help understanding the evolutionary history of Venturia species that are pathogenic on different hosts, a history that has probably been substantially influenced by TEs." />
<meta name="DC.Contributor" content="Bruno Le Cam" />
<meta name="DC.Contributor" content="Dan Sargent" />
<meta name="DC.Contributor" content="Jérôme Gouzy" />
<meta name="DC.Contributor" content="Joëlle Amselem" />
<meta name="DC.Contributor" content="Marie-Noëlle Bellanger" />
<meta name="DC.Contributor" content="Olivier Bouchez" />
<meta name="DC.Contributor" content="Spencer Brown" />
<meta name="DC.Contributor" content="Valérie Caffier" />
<meta name="DC.Contributor" content="Marie De Gracia" />
<meta name="DC.Contributor" content="Robert Debuchy" />
<meta name="DC.Contributor" content="Ludovic Duvaux" />
<meta name="DC.Contributor" content="Thibaut Payen" />
<meta name="DC.Contributor" content="Mélanie Sannier" />
<meta name="DC.Contributor" content="Jason Shiller" />
<meta name="DC.Contributor" content="Jérôme Collemare" />
<meta name="DC.Contributor" content="Christophe Lemaire" />
<meta name="article:published_time" content="2019-08-01" />
<meta name="article:section" content="Genome Report" />
<meta name="citation_title" content="Population Genome Sequencing of the Scab Fungal Species Venturia inaequalis, Venturia pirina, Venturia aucupariae and Venturia asperata" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The &lt;i&gt;Venturia&lt;/i&gt; genus comprises fungal species that are pathogens on &lt;i&gt;Rosaceae&lt;/i&gt; host plants, including &lt;i&gt;V. inaequalis&lt;/i&gt; and &lt;i&gt;V. asperata&lt;/i&gt; on apple, &lt;i&gt;V. aucupariae&lt;/i&gt; on sorbus and &lt;i&gt;V. pirina&lt;/i&gt; on pear. Although the genetic structure of &lt;i&gt;V. inaequalis&lt;/i&gt; populations has been investigated in detail, genomic features underlying these subdivisions remain poorly understood. Here, we report whole genome sequencing of 87 &lt;i&gt;Venturia&lt;/i&gt; strains that represent each species and each population within &lt;i&gt;V. inaequalis&lt;/i&gt;. We present a PacBio genome assembly for the &lt;i&gt;V. inaequalis&lt;/i&gt; EU-B04 reference isolate. The size of selected genomes was determined by flow cytometry, and varied from 45 to 93 Mb. Genome assemblies of &lt;i&gt;V. inaequalis&lt;/i&gt; and &lt;i&gt;V. aucupariae&lt;/i&gt; contain a high content of transposable elements (TEs), most of which belong to the Gypsy or Copia LTR superfamilies and have been inactivated by Repeat-Induced Point mutations. The reference assembly of &lt;i&gt;V. inaequalis&lt;/i&gt; presents a mosaic structure of GC-equilibrated regions that mainly contain predicted genes and AT-rich regions, mainly composed of TEs. Six pairs of strains were identified as clones. Single-Nucleotide Polymorphism (SNP) analysis between these clones revealed a high number of SNPs that are mostly located in AT-rich regions due to misalignments and allowed determining a false discovery rate. The availability of these genome sequences is expected to stimulate genetics and population genomics research of &lt;i&gt;Venturia&lt;/i&gt; pathogens. Especially, it will help understanding the evolutionary history of &lt;i&gt;Venturia&lt;/i&gt; species that are pathogenic on different hosts, a history that has probably been substantially influenced by TEs.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/08/01" />
<meta name="citation_mjid" content="ggg;9/8/2405" />
<meta name="citation_id" content="9/8/2405" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/8/2405" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/8/2405.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/8/2405.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/8/2405.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400047" />
<meta name="citation_pmid" content="31253647" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genome Report" />
<meta name="citation_firstpage" content="2405" />
<meta name="citation_lastpage" content="2414" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Bruno Le Cam" />
<meta name="citation_author_institution" content="UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France," />
<meta name="citation_author_email" content="bruno.le-cam@inra.fr" />
<meta name="citation_author" content="Dan Sargent" />
<meta name="citation_author_institution" content="Research and Innovation Centre, Fondazione Edmund Mach, via Mach 1, 38010, San Michele All’adige, TN, Italy," />
<meta name="citation_author" content="Jérôme Gouzy" />
<meta name="citation_author_institution" content="Laboratoire Interactions Plantes Micro-organismes [LIPM], Université de Toulouse, INRA, CNRS, Castanet-Tolosan, France," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5695-4557" />
<meta name="citation_author" content="Joëlle Amselem" />
<meta name="citation_author_institution" content="Unité de Recherche Génomique Informatique [URGI], INRA, Université Paris-Saclay, 78026, Versailles, France," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7124-3454" />
<meta name="citation_author" content="Marie-Noëlle Bellanger" />
<meta name="citation_author_institution" content="UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France," />
<meta name="citation_author" content="Olivier Bouchez" />
<meta name="citation_author_institution" content="US1426, INRA, GeT-PlaGe, Genotoul, Castanet-Tolosan, France," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2328-5769" />
<meta name="citation_author" content="Spencer Brown" />
<meta name="citation_author_institution" content="UMR 8298, Institute for Integrative Biology of the Cell (I2BC), CEA, CNRS," />
<meta name="citation_author" content="Valérie Caffier" />
<meta name="citation_author_institution" content="UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France," />
<meta name="citation_author" content="Marie De Gracia" />
<meta name="citation_author_institution" content="UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France," />
<meta name="citation_author" content="Robert Debuchy" />
<meta name="citation_author_institution" content="UMR 8621, Institut de Génétique et Microbiologie CNRS, Université Paris-Sud, 91405 Orsay, France, and" />
<meta name="citation_author" content="Ludovic Duvaux" />
<meta name="citation_author_institution" content="UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0960-0312" />
<meta name="citation_author" content="Thibaut Payen" />
<meta name="citation_author_institution" content="UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France," />
<meta name="citation_author" content="Mélanie Sannier" />
<meta name="citation_author_institution" content="UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France," />
<meta name="citation_author" content="Jason Shiller" />
<meta name="citation_author_institution" content="UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France," />
<meta name="citation_author" content="Jérôme Collemare" />
<meta name="citation_author_institution" content="UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France," />
<meta name="citation_author_institution" content="Westerdijk Fungal Biodiversity Institute, 3584 CT, Utrecht, The Netherlands" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8152-6642" />
<meta name="citation_author" content="Christophe Lemaire" />
<meta name="citation_author_institution" content="UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France," />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=D. Aird;citation_author=M. G. Ross;citation_author=W. S. Chen;citation_author=M. Denielsson;citation_author=T. Fennell;citation_title=Analyzing and minimizing PCR amplification bias in Illumina sequencing libraries.;citation_pages=R18-R18;citation_volume=12;citation_year=2011;citation_issue=2;citation_pmid=21338519;citation_doi=10.1186/gb-2011-12-2-r18" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=Z. Bao;citation_author=S. Eddy;citation_title=Automated de novo identification of repeat sequence families in sequenced genomes;citation_volume=12;citation_year=2002;citation_pmid=12176934;citation_doi=10.1101/gr.88502" />
<meta name="citation_reference" content="citation_journal_title=Mycol. Prog.;citation_author=A. Beck;citation_author=A. Ritschel;citation_author=K. Schubert;citation_author=U. Braun;citation_author=D. Triebel;citation_title=Phylogenetic relationships of the anamorphic genus Fusicladium s. lat. as inferred by ITS nrDNA data;citation_pages=111-116;citation_volume=4;citation_year=2005;citation_doi=10.1007/s11557-006-0114-8" />
<meta name="citation_reference" content="citation_journal_title=Phytopathology;citation_journal_abbrev=Phytopathology;citation_author=G. Bénaouf;citation_author=L. Parisi;citation_title=Genetics of Host-Pathogen Relationships Between Venturia inaequalis Races 6 and 7 and Malus Species.;citation_pages=236-242;citation_volume=90;citation_year=2000;citation_issue=3;citation_pmid=18944614;citation_doi=10.1094/PHYTO.2000.90.3.236" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=K. Berlin;citation_author=S. Koren;citation_author=C. S. Chin;citation_author=J. P. Drake;citation_author=J. M. Landolin;citation_title=Assembling large genomes with single-molecule sequencing and locality-sensitive hashing;citation_pages=623-630;citation_volume=33;citation_year=2015;citation_pmid=26006009;citation_doi=10.1038/nbt.3238" />
<meta name="citation_reference" content="citation_journal_title=Stand. Genomic Sci.;citation_author=C. H. Bock;citation_author=C. Chen;citation_author=F. Yu;citation_author=K. L. Stevenson;citation_author=B. W. Wood;citation_title=Draft genome sequence of Fusicladium effusum, cause of pecan scab;citation_pages=36;citation_volume=11;citation_year=2016;citation_doi=10.1186/s40793-016-0161-y" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. M. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data;citation_volume=30;citation_year=2014;citation_pmid=24695404;citation_doi=10.1093/bioinformatics/btu170" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Plant Pathol;citation_author=J. K. Bowen;citation_author=C. H. Mesarich;citation_author=J. Rees-George;citation_author=W. Cui;citation_author=A. Fitzgerald;citation_title=Candidate effector gene identification in the ascomycete fungal phytopathogen Venturia inaequalis by expressed sequence tag analysis.;citation_pages=431-448;citation_volume=10;citation_year=2009;citation_issue=3;citation_pmid=19400844;citation_doi=10.1111/j.1364-3703.2009.00543.x" />
<meta name="citation_reference" content="citation_journal_abbrev=Mol Plant Pathol;citation_author=J. K. Bowen;citation_author=C. H. Mesarich;citation_author=V. G. Bus;citation_author=R. M. Beresford;citation_author=K. M. Plummer;citation_title=Venturia inaequalis: the causal agent of apple scab.;citation_pages=105-122;citation_volume=12;citation_year=2011;citation_issue=2;citation_pmid=21199562;citation_doi=10.1111/j.1364-3703.2010.00656.x" />
<meta name="citation_reference" content="citation_journal_title=Plant Pathol.;citation_author=V. Caffier;citation_author=B. Le Cam;citation_author=P. Expert;citation_author=M. Devaux;citation_author=M. Giraud;citation_title=A new scab-like disease on apple caused by the formerly saprotroph fungus Venturia asperata;citation_pages=915-924;citation_volume=61;citation_year=2012;citation_doi=10.1111/j.1365-3059.2011.02583.x" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=J. Castresana;citation_title=Selection of Conserved Blocks from Multiple Alignments for Their Use in Phylogenetic Analysis;citation_pages=540-552;citation_volume=17;citation_year=2000;citation_issue=4;citation_pmid=10742046;citation_doi=10.1093/oxfordjournals.molbev.a026334" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=K. Cheeseman;citation_author=J. Ropars;citation_author=P. Renault;citation_author=J. Dupont;citation_author=J. Gouzy;citation_title=Multiple recent horizontal transfers of a large genomic region in cheese making fungi;citation_pages=2876;citation_volume=5;citation_year=2014;citation_pmid=24407037;citation_doi=10.1038/ncomms3876" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=C. S. Chin;citation_author=D. H. Alexander;citation_author=P. Marks;citation_author=A. A. Klammer;citation_author=J. Drake;citation_title=Nonhybrid, finished microbial genome assemblies from long-read SMRT sequencing data;citation_pages=563-569;citation_volume=10;citation_year=2013;citation_pmid=23644548;citation_doi=10.1038/nmeth.2474" />
<meta name="citation_reference" content="citation_journal_title=J. Proteome Res.;citation_author=I. R. Cooke;citation_author=D. Jones;citation_author=J. K. Bowen;citation_author=C. Deng;citation_author=P. Faou;citation_title=Proteogenomic analysis of the Venturia pirina (pear scab fungus) secretome reveals potential effectors;citation_pages=3635-3644;citation_volume=13;citation_year=2014;citation_doi=10.1021/pr500176c" />
<meta name="citation_reference" content="citation_journal_title=Studies in Mycology;citation_journal_abbrev=SIM;citation_author=P. W. Crous;citation_author=K. Schubert;citation_author=U. Braun;citation_author=G. S. de Hoog;citation_author=A. D. Hocking;citation_title=Opportunistic, human-pathogenic species in the Herpotrichiellaceae are phenotypically similar to saprobic or phytopathogenic species in the Venturiaceae;citation_pages=185-217;citation_volume=58;citation_year=2007;citation_issue=1;citation_pmid=18491000;citation_doi=10.3114/sim.2007.58.07" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=C. H. Deng;citation_author=K. M. Plummer;citation_author=D. A. B. Jones;citation_author=C. H. Mesarich;citation_author=J. Shiller;citation_title=Comparative analysis of the predicted secretomes of Rosaceae scab pathogens Venturia inaequalis and V. pirina reveals expanded effector families and putative determinants of host range;citation_pages=339;citation_volume=18;citation_year=2017;citation_doi=10.1186/s12864-017-3699-1" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. A. DePristo;citation_author=E. Banks;citation_author=R. Poplin;citation_author=K. V. Garimella;citation_author=J. R. Maguire;citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data.;citation_pages=491-498;citation_volume=43;citation_year=2011;citation_issue=5;citation_pmid=21478889;citation_doi=10.1038/ng.806" />
<meta name="citation_reference" content="citation_journal_title=Cytometry. Part A :  the journal of the International Society for Analytical Cytology;citation_journal_abbrev=Cytometry A;citation_author=J. Dolezel;citation_author=J. Bartos;citation_author=H. Voglmayr;citation_author=J. Greilhuber;citation_title=Nuclear DNA content and genome size of trout and human.;citation_pages=127-128;citation_volume=51;citation_year=2003;citation_issue=2;citation_pmid=12541287;citation_doi=10.1002/cyto.a.10013" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=L. Ebrahimi;citation_author=K.-B. Fotuhifar;citation_author=M. Javan Nikkhah;citation_author=M. R. Naghavi;citation_author=N. Baisakh;citation_title=Population genetic structure of apple scab (Venturia inaequalis (Cooke) G. Winter) in Iran;citation_pages=e0160737;citation_volume=11;citation_year= 2016;citation_doi=10.1371/journal.pone.0160737" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. C. Edgar;citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput;citation_pages=1792-1797;citation_volume=32;citation_year=2004;citation_issue=5;citation_pmid=15034147;citation_doi=10.1093/nar/gkh340" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=R. Edgar;citation_author=E. Myers;citation_title=PILER: identification and classification of genomic repeats;citation_pages=i152-i158;citation_volume=21;citation_year=2005;citation_issue=suppl_1;citation_pmid=15961452;citation_doi=10.1093/bioinformatics/bti1003" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=B. Eisenhaber;citation_author=G. Schneider;citation_author=M. Wildpaner;citation_author=F. Eisenhaber;citation_title=A sensitive predictor for potential GPI lipid modification sites in fungal protein sequences and its application to genome-wide studies for Aspergillus nidulans, Candida albicans, Neurospora crassa, Saccharomyces cerevisiae and Schizosaccharomyces pombe.;citation_pages=243-253;citation_volume=337;citation_year=2004;citation_issue=2;citation_pmid=15003443;citation_doi=10.1016/j.jmb.2004.01.025" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. D. Finn;citation_author=A. Bateman;citation_author=J. Clements;citation_author=P. Coggill;citation_author=R. Y. Eberhardt;citation_title=Pfam: the protein families database;citation_volume=42;citation_year=2014;citation_pmid=24288371;citation_doi=10.1093/nar/gkt1223" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=T. Flutre;citation_author=E. Duprat;citation_author=C. Feuillet;citation_author=H. Quesneville;citation_title=Considering transposable element diversification in de novo annotation approaches.;citation_pages=e16526-e16526;citation_volume=6;citation_year=2011;citation_issue=1;citation_pmid=21304975;citation_doi=10.1371/journal.pone.0016526" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=P. Gladieux;citation_author=X. G. Zhang;citation_author=D. Afoufa-Bastien;citation_author=R. M. Valdebenito Sanhueza;citation_author=M. Sbaghi;citation_title=On the origin and spread of the Scab disease of apple: out of central Asia.;citation_pages=e1455-e1455;citation_volume=3;citation_year=2008;citation_issue=1;citation_pmid=18197265;citation_doi=10.1371/journal.pone.0001455" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=P. Gladieux;citation_author=V. Caffier;citation_author=M. Devaux;citation_author=B. Le Cam;citation_title=Host-specific differentiation among populations of Venturia inaequalis causing scab on apple, pyracantha and loquat.;citation_pages=511-521;citation_volume=47;citation_year=2010;citation_issue=6;citation_pmid=20060485;citation_doi=10.1016/j.fgb.2009.12.007" />
<meta name="citation_reference" content="citation_journal_title=Molecular Ecology (Print);citation_journal_abbrev=Molecular Ecology (Print);citation_author=P. Gladieux;citation_author=X. G. Zhang;citation_author=I. Roldan Ruiz;citation_author=V. Caffier;citation_author=T. Leroy;citation_title=Evolution of the population structure of Venturia inaequalis, the apple scab fungus, associated with the domestication of its host.;citation_pages=658-674;citation_volume=19;citation_year=2010;citation_issue=4;citation_pmid=20088887;citation_doi=10.1111/j.1365-294X.2009.04498.x" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=F. Guérin;citation_author=P. Gladieux;citation_author=B. Le Cam;citation_title=Origin and colonization history of newly virulent strains of the phytopathogenic fungus Venturia inaequalis.;citation_pages=284-292;citation_volume=44;citation_year=2007;citation_issue=4;citation_pmid=17166752;citation_doi=10.1016/j.fgb.2006.10.005" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Gurevich;citation_author=V. Saveliev;citation_author=N. Vyahhi;citation_author=G. Tesler;citation_title=QUAST: quality assessment tool for genome assemblies;citation_volume=29;citation_year=2013;citation_pmid=23422339;citation_doi=10.1093/bioinformatics/btt086" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=C. Hoede;citation_author=S. Arnoux;citation_author=M. Moisset;citation_author=T. Chaumier;citation_author=O. Inizan;citation_title=PASTEC: An Automatic Transposable Element Classification Tool;citation_pages=e91929;citation_volume=9;citation_year=2014;citation_pmid=24786468;citation_doi=10.1371/journal.pone.0091929" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=K. J. Hoff;citation_author=S. Lange;citation_author=A. Lomsadze;citation_author=M. Borodovsky;citation_author=M. Stanke;citation_title=BRAKER1: Unsupervised RNA-Seq-Based Genome Annotation with GeneMark-ET and AUGUSTUS;citation_volume=32;citation_year=2016;citation_pmid=26559507;citation_doi=10.1093/bioinformatics/btv661" />
<meta name="citation_reference" content="citation_journal_title=Comput. Appl. Biosci. CABIOS.;citation_author=X. Huang;citation_title=On global sequence alignment;citation_pages=227-235;citation_volume=10;citation_year=1994" />
<meta name="citation_reference" content="citation_title=Compendium of apple and pear diseases;citation_year=1990" />
<meta name="citation_reference" content="citation_journal_title=Cytogenetic and genome research;citation_journal_abbrev=Cytogenet Genome Res;citation_author=J. Jurka;citation_author=V. Kapitonov;citation_author=A. Pavlicek;citation_author=P. Klonowski;citation_author=O. Kohany;citation_title=Repbase Update, a database of eukaryotic repetitive elements.;citation_pages=462-467;citation_volume=110;citation_year=2005;citation_issue=1-4;citation_pmid=16093699;citation_doi=10.1159/000084979" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=A. Krogh;citation_author=B. Larsson;citation_author=G. von Heijne;citation_author=E. L. Sonnhammer;citation_title=Predicting transmembrane protein topology with a hidden Markov model: application to complete genomes.;citation_pages=567-580;citation_volume=305;citation_year=2001;citation_issue=3;citation_pmid=11152613;citation_doi=10.1006/jmbi.2000.4315" />
<meta name="citation_reference" content="citation_journal_title=Fungal genetics and biology :  FG &amp; B;citation_journal_abbrev=Fungal Genet Biol;citation_author=N. Kucheryava;citation_author=J. K. Bowen;citation_author=P. W. Sutherland;citation_author=J. J. Conolly;citation_author=C. H. Mesarich;citation_title=Two novel Venturia inaequalis genes induced upon morphogenetic differentiation during infection and in vitro growth on cellophane.;citation_pages=1329-1339;citation_volume=45;citation_year=2008;citation_issue=10;citation_pmid=18692586;citation_doi=10.1016/j.fgb.2008.07.010" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Phytopathology;citation_journal_abbrev=Phytopathology;citation_author=B. Le Cam;citation_author=L. Parisi;citation_author=L. Arène;citation_title=Evidence of Two Formae Speciales in Venturia inaequalis, Responsible for Apple and Pyracantha Scab.;citation_pages=314-320;citation_volume=92;citation_year=2002;citation_issue=3;citation_pmid=18944005;citation_doi=10.1094/PHYTO.2002.92.3.314" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=C. Lemaire;citation_author=M. De Gracia;citation_author=T. Leroy;citation_author=M. Michalecka;citation_author=H. Lindhard-Pedersen;citation_title=Emergence of new virulent populations of apple scab from non-agricultural disease reservoirs;citation_pages=1220-1229;citation_volume=209;citation_year=2016;citation_pmid=26428268;citation_doi=10.1111/nph.13658" />
<meta name="citation_reference" content="citation_journal_title=BMC Evol. Biol.;citation_author=T. Leroy;citation_author=C. Lemaire;citation_author=F. Dunemann;citation_author=B. Le Cam;citation_title=The genetic structure of a Venturia inaequalis population in a heterogeneous host population composed of different Malusspecies;citation_pages=64;citation_volume=13;citation_year=2013;citation_doi=10.1186/1471-2148-13-64" />
<meta name="citation_reference" content="citation_journal_title=New Phytol.;citation_author=T. Leroy;citation_author=V. Caffier;citation_author=J. M. Celton;citation_author=N. Anger;citation_author=C. E. Durel;citation_title=When virulence originates from nonagricultural hosts: evolutionary and epidemiological consequences of introgressions following secondary contacts in Venturia inaequalis;citation_pages=1443-1452;citation_volume=210;citation_year=2016;citation_doi=10.1111/nph.13873" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Plant Biol.;citation_author=L. Lo Presti;citation_author=D. Lanver;citation_author=G. Schweizer;citation_author=S. Tanaka;citation_author=L. Liang;citation_title=Fungal effectors and plant susceptibility;citation_pages=513-545;citation_volume=66;citation_year=2015;citation_pmid=25923844;citation_doi=10.1146/annurev-arplant-043014-114623" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=G. Lunter;citation_author=M. Goodson;citation_title=Stampy: a statistical algorithm for sensitive and fast mapping of Illumina sequence reads;citation_volume=21;citation_year=2011;citation_pmid=20980556;citation_doi=10.1101/gr.111120.110" />
<meta name="citation_reference" content="citation_journal_title=Gigascience;citation_author=R. Luo;citation_author=B. Liu;citation_author=Y. Xie;citation_author=Z. Li;citation_author=W. Huang;citation_title=SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler;citation_pages=18;citation_volume=1;citation_year=2012;citation_pmid=23587118;citation_doi=10.1186/2047-217X-1-18" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Biology of the cell / under the auspices of the European Cell Biology Organization;citation_journal_abbrev=Biol Cell;citation_author=D. Marie;citation_author=S. C. Brown;citation_title=A cytometric exercise in plant DNA histograms, with 2C values for 70 species.;citation_pages=41-51;citation_volume=78;citation_year=1993;citation_issue=1-2;citation_pmid=8220226;citation_doi=10.1016/0248-4900(93)90113-S" />
<meta name="citation_reference" content="citation_journal_title=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_journal_abbrev=Biochimica et Biophysica Acta. Protein Structure and Molecular Enzymology;citation_author=C. H. Mesarich;citation_author=M. Schmitz;citation_author=P. Tremouilhac;citation_author=D. J. McGillivray;citation_author=M. D. Templeton;citation_title=Structure, dynamics and domain organization of the repeat protein Cin1 from the apple scab fungus.;citation_pages=1118-1128;citation_volume=1824;citation_year=2012;citation_issue=10;citation_pmid=22771296;citation_doi=10.1016/j.bbapap.2012.06.015" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=E. M. Möller;citation_author=G. Bahnweg;citation_author=H. Sandermann;citation_author=H. H. Geiger;citation_title=A simple and efficient protocol for isolation of high molecular weight DNA from filamentous fungi, fruit bodies, and infected plant tissues;citation_pages=6115-6116;citation_volume=20;citation_year=1992;citation_issue=22;citation_pmid=1461751;citation_doi=10.1093/nar/20.22.6115" />
<meta name="citation_reference" content="citation_author=C. Monat;citation_author=C. Tranchant-Dubreuil;citation_author=A. Kougbeadjo;citation_author=C. Farcy;citation_author=E. Ortega-Abboud;citation_pages=374;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12859-015-0795-6" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=S. Oyola;citation_author=T. Otto;citation_author=Y. Gu;citation_author=G. Maslen;citation_author=M. Manske;citation_title=Optimizing Illumina next-generation sequencing library preparation for extremely AT-biased genomes.;citation_pages=1-1;citation_volume=13;citation_year=2012;citation_pmid=22214261;citation_doi=10.1186/1471-2164-13-1" />
<meta name="citation_reference" content="citation_journal_title=Microbiology Resource Announcements;citation_journal_abbrev=Microbiol Resour Announc;citation_author=T. Passey;citation_author=A. D. Armitage;citation_author=X. Xu;citation_title=Annotated Draft Genome Sequence of the Apple Scab Pathogen Venturia inaequalis;citation_pages=e01062-18-e01062-18;citation_volume=7;citation_year=2018;citation_issue=12;citation_doi=10.1128/MRA.01062-18" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=T. M. Petersen;citation_author=S. Brunak;citation_author=G. von Heijne;citation_author=H. Nielsen;citation_title=SignalP 4.0: discriminating signal peptides from transmembrane regions.;citation_pages=785-786;citation_volume=8;citation_year=2011;citation_issue=10;citation_pmid=21959131;citation_doi=10.1038/nmeth.1701" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=A. Pierleoni;citation_author=P. L. Martelli;citation_author=R. Casadio;citation_title=PredGPI: a GPI-anchor predictor.;citation_pages=392-392;citation_volume=9;citation_year=2008;citation_pmid=18811934;citation_doi=10.1186/1471-2105-9-392" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=B. Purcell;citation_author=K. Neale;citation_author=L. Todd-Brown;citation_author=M. A. R. Thomas;citation_author=D. Ferreira;citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses.;citation_pages=559-575;citation_volume=81;citation_year=2007;citation_issue=3;citation_pmid=17701901;citation_doi=10.1086/519795" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=M. A. Quail;citation_author=M. Smith;citation_author=P. Coupland;citation_author=T. D. Otto;citation_author=S. R. Harris;citation_title=A tale of three next generation sequencing platforms: comparison of Ion Torrent, Pacific Biosciences and Illumina MiSeq sequencers.;citation_pages=341-341;citation_volume=13;citation_year=2012;citation_pmid=22827831;citation_doi=10.1186/1471-2164-13-341" />
<meta name="citation_reference" content="citation_journal_title=J. Mol. Evol.;citation_author=H. Quesneville;citation_author=D. Nouaud;citation_author=D. Anxolabéhère;citation_title=Detection of new transposable element families in Drosophila melanogaster and Anopheles gambiae Genomes;citation_pages=S50-S59;citation_volume=57;citation_year=2003;citation_pmid=15008403;citation_doi=10.1007/s00239-003-0007-2" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Comput Biol;citation_author=H. Quesneville;citation_author=C. Bergman;citation_author=O. Andrieu;citation_author=D. Autard;citation_author=D. Nouaud;citation_title=Combined evidence annotation of transposable elements in genome sequences.;citation_pages=166-175;citation_volume=1;citation_year=2005;citation_issue=2;citation_pmid=16110336;citation_doi=10.1371/journal.pcbi.0010022" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Commun;citation_author=T. Rouxel;citation_author=J. Grandaubert;citation_author=J. K. Hane;citation_author=C. Hoede;citation_author=A. P. van de Wouw;citation_title=Effector diversification within compartments of the Leptosphaeria maculans genome affected by Repeat-Induced Point mutations.;citation_pages=202-202;citation_volume=2;citation_year=2011;citation_pmid=21326234;citation_doi=10.1038/ncomms1189" />
<meta name="citation_reference" content="citation_journal_title=N. Z. J. Bot.;citation_author=G. J. Samuels;citation_author=A. Sivanesan;citation_title=Venturia asperata sp. nov. and its Fusicladium state on apple leaves;citation_pages=645-652;citation_volume=13;citation_year=1975;citation_doi=10.1080/0028825X.1975.10430352" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=J. Shiller;citation_author=A. P. Van de Wouw;citation_author=A. P. Taranto;citation_author=J. K. Bowen;citation_author=D. Dubois;citation_title=A large family of AvrLm6-like genes in the apple and pear scab pathogens, Venturia inaequalis and Venturia pirina;citation_pages=980;citation_volume=6;citation_year=2015;citation_doi=10.3389/fpls.2015.00980" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=F. A. Simão;citation_author=R. M. Waterhouse;citation_author=P. Ioannidis;citation_author=E. V. Kriventseva;citation_author=E. M. Zdobnov;citation_title=BUSCO:assessing genome assembly and annotation completeness with single-copy orthologs;citation_volume=31;citation_year=2015;citation_pmid=26059717;citation_doi=10.1093/bioinformatics/btv351" />
<meta name="citation_reference" content="citation_journal_title=Z. Naturforsch. C;citation_author=P. Singh;citation_author=M. Piotrowski;citation_author=A. E. Gau;citation_title=Purification and partial characterization of an extracellular melanoprotein from the fungus Venturia inaequalis;citation_pages=109-115;citation_volume=60;citation_year=2005;citation_doi=10.1515/znc-2005-1-220" />
<meta name="citation_reference" content="citation_title=The Taxonomy and Pathology of Venturia Species;citation_year=1977" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. Stanke;citation_author=M. Diekhans;citation_author=R. Baertsch;citation_author=D. Haussler;citation_title=Using native and syntenically mapped cDNA alignments to improve de novo gene finding;citation_volume=24;citation_year=2008;citation_pmid=18218656;citation_doi=10.1093/bioinformatics/btn013" />
<meta name="citation_reference" content="citation_journal_title=Phytopathology;citation_journal_abbrev=Phytopathology;citation_author=U. Steiner;citation_author=E. C. Oerke;citation_title=Localized Melanization of Appressoria Is Required for Pathogenicity of Venturia inaequalis.;citation_pages=1222-1230;citation_volume=97;citation_year=2007;citation_issue=10;citation_pmid=18943680;citation_doi=10.1094/PHYTO-97-10-1222" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. Tamura;citation_author=D. Peterson;citation_author=N. Peterson;citation_author=G. Stecher;citation_author=M. Nei;citation_title=MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods;citation_volume=28;citation_year=2011;citation_pmid=21546353;citation_doi=10.1093/molbev/msr121" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=A. C. Testa;citation_author=R. P. Oliver;citation_author=J. K. Hane;citation_title=OcculterCut: A comprehensive survey of AT-rich regions in fungal genomes;citation_volume=8;citation_year=2016;citation_pmid=27289099;citation_doi=10.1093/gbe/evw121" />
<meta name="citation_reference" content="citation_journal_title=Curr. Protoc. Bioinformatics;citation_author=G. A. Van der Auwera;citation_author=M. O. Carneiro;citation_author=C. Hartl;citation_author=R. Poplin;citation_author=G. Del Angel;citation_title=From FastQ data to high confidence variant calls: the Genome Analysis Toolkit best practices pipeline;citation_pages=11-33;citation_volume=43;citation_year=2013;citation_doi=doi:10.1002/0471250953.bi1110s43" />
<meta name="citation_reference" content="citation_journal_title=Microbial Genetics Bulletin;citation_author=H. J. Vogel;citation_title=A convenient growth medium for Neurospora crassa;citation_pages=42-47;citation_volume=13;citation_year=1956" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. K. Watters;citation_author=T. A. Randall;citation_author=B. S. Margolin;citation_author=E. U. Selker;citation_author=D. R. Stadier;citation_title=Action of Repeat-Induced Point Mutation on Both Strands of a Duplex and on Tandem Duplications of Various Sizes in Neurospora;citation_pages=705-714;citation_volume=153;citation_year=1999;citation_issue=2;citation_pmid=10511550" />
<meta name="citation_reference" content="citation_journal_title=Phytopathology;citation_author=C. A. Young;citation_author=C. H. Bock;citation_author=N. D. Charlton;citation_author=C. Mattupalli;citation_author=N. Krom;citation_title=Evidence for Sexual Reproduction: Identification, Frequency, and Spatial Distribution of Venturia effusa (Pecan Scab) Mating Type Idiomorphs;citation_pages=837-846;citation_volume=108;citation_year=2018;citation_doi=10.1094/PHYTO-07-17-0233-R" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. R. Zerbino;citation_author=E. Birney;citation_title=Velvet: algorithms for de novo short read assembly using de Bruijn graphs;citation_volume=18;citation_year=2008;citation_pmid=18349386;citation_doi=10.1101/gr.074492.107" />
<meta name="citation_reference" content="citation_journal_abbrev=Fungal Divers;citation_author=Y. Zhang;citation_author=P. W. Crous;citation_author=C. L. Schoch;citation_author=A. H. Bahkali;citation_author=L. D. Guo;citation_title=A molecular, morphological and ecological re-appraisal of Venturiales-a new order of Dothideomycetes.;citation_pages=249-277;citation_volume=51;citation_year=2011;citation_issue=1;citation_pmid=22368534;citation_doi=10.1007/s13225-011-0141-x" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Population Genome Sequencing of the Scab Fungal Species Venturia inaequalis, Venturia pirina, Venturia aucupariae and Venturia asperata" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8.cover-source.jpg" />
<meta name="twitter:description" content="The Venturia genus comprises fungal species that are pathogens on Rosaceae host plants, including V. inaequalis and V. asperata on apple, V. aucupariae on sorbus and V. pirina on pear. Although the genetic structure of V. inaequalis populations has been investigated in detail, genomic features underlying these subdivisions remain poorly understood. Here, we report whole genome sequencing of 87 Venturia strains that represent each species and each population within V. inaequalis . We present a PacBio genome assembly for the V. inaequalis EU-B04 reference isolate. The size of selected genomes was determined by flow cytometry, and varied from 45 to 93 Mb. Genome assemblies of V. inaequalis and V. aucupariae contain a high content of transposable elements (TEs), most of which belong to the Gypsy or Copia LTR superfamilies and have been inactivated by Repeat-Induced Point mutations. The reference assembly of V. inaequalis presents a mosaic structure of GC-equilibrated regions that mainly contain predicted genes and AT-rich regions, mainly composed of TEs. Six pairs of strains were identified as clones. Single-Nucleotide Polymorphism (SNP) analysis between these clones revealed a high number of SNPs that are mostly located in AT-rich regions due to misalignments and allowed determining a false discovery rate. The availability of these genome sequences is expected to stimulate genetics and population genomics research of Venturia pathogens. Especially, it will help understanding the evolutionary history of Venturia species that are pathogenic on different hosts, a history that has probably been substantially influenced by TEs." />
<meta name="og-title" property="og:title" content="Population Genome Sequencing of the Scab Fungal Species Venturia inaequalis, Venturia pirina, Venturia aucupariae and Venturia asperata" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/8/2405" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The Venturia genus comprises fungal species that are pathogens on Rosaceae host plants, including V. inaequalis and V. asperata on apple, V. aucupariae on sorbus and V. pirina on pear. Although the genetic structure of V. inaequalis populations has been investigated in detail, genomic features underlying these subdivisions remain poorly understood. Here, we report whole genome sequencing of 87 Venturia strains that represent each species and each population within V. inaequalis . We present a PacBio genome assembly for the V. inaequalis EU-B04 reference isolate. The size of selected genomes was determined by flow cytometry, and varied from 45 to 93 Mb. Genome assemblies of V. inaequalis and V. aucupariae contain a high content of transposable elements (TEs), most of which belong to the Gypsy or Copia LTR superfamilies and have been inactivated by Repeat-Induced Point mutations. The reference assembly of V. inaequalis presents a mosaic structure of GC-equilibrated regions that mainly contain predicted genes and AT-rich regions, mainly composed of TEs. Six pairs of strains were identified as clones. Single-Nucleotide Polymorphism (SNP) analysis between these clones revealed a high number of SNPs that are mostly located in AT-rich regions due to misalignments and allowed determining a false discovery rate. The availability of these genome sequences is expected to stimulate genetics and population genomics research of Venturia pathogens. Especially, it will help understanding the evolutionary history of Venturia species that are pathogenic on different hosts, a history that has probably been substantially influenced by TEs." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8.cover-source.jpg" />
    <title>Population Genome Sequencing of the Scab Fungal Species Venturia inaequalis, Venturia pirina, Venturia aucupariae and Venturia asperata | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"C9ON3MYfzlgB8elvNP6be70syyIA-j92OqqRjEgiUwI","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"493769","apath":"\/ggg\/9\/8\/2405.atom","pisa":"ggg;9\/8\/2405","ac":{"\/ggg\/9\/8\/2405.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/8\/2405.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/8\/2405"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/8\/2405"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/8\/2405":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022l95XdPoVTF1zoM\\\\\\\/7bK3Nc0jeh6k0LzRk412GApAmG6TWwmPBJx85SXVo9HoCbT5h98Aw\\\\\\\/+l\\\\\\\/\\\\\\\/j26ecmO6\\\\\\\/9\\\\\\\/9gz9KacNqEDgrJgJ79wSagoFAQFqcLkofbgRtbKUOnGYeWa+GqQ2bMVS0RwHFmMpJbgrRb\\\\\\\/ScRxMhUFEo5nMjm+A2ycBDwnpCz8dpXeyV3vRAfHfVCAdl2yBEJeial3EIKGRGf29c9Sa\\\\\\\/JeHOf+o8up+2w0TYI5yZdXVfUhMv43bxMDwu1FHl1J5S1tOO1iqksRDO3937Zqgdizo9ZtojeZwIpNK+zCvoiBaUuRmFJHXTasxwTr6pNTBNfcwvAh1WPyQJERteriF1WRVzC0zCRYXJzF8wkqfQJTkBoXCY+dXxak\\\\\\\/EAveT0eGtyo+Ti+LzYMmxvEcmKDM1HkZdqzrarYiLNl12kmTM+jb1GJYRfbACBm+JSzkCMTJBSR+ighs1s3D+171CYxmRC25BkJmfxrQw2HxQsmVj5ei59vEhb0AKqA6WtjHEEHT8jnxc0ug3FcMnfwHPhK5Uo16XwMOTFcPP43C4ttYNntnwKyQqIclJaP4mYCMPe1x+mNayw3et\\\\\\\/VFJmqOpj59u6lat\\\\\\\/Sevr9+gldzqVpW3Yr2AwxB\\\\\\\/Gjen6LoBGcJzf5qTED1G1x2kCWGQI1I7mwen0iNrqx\\\\\\\/VkkNo2tpKYAVi905DMp0Wx6nZnDtNJIhSqDjUXgxpid4yJevFK0v8BxVrbjQoyc5gsGWDXvb3dySNaagsciBxgwoXWIiBSECNEqzqiKZxQ6mzUB+MEyuiD5YduZlQAtTV35v36eYOBNdveAT2FqUkFaVOt+BkiGhn3Z3TGC8OcY0PCPJt5LWpUIlKPfxGksFiBdPlqN4vKqv0ttHYhigSG5FeqjB2vh\\\\\\\/Jzq6MfGT1TLVGjfcRe9fZYHhGVS4x\\\\\\\/EMhyEJKIbmbAtKq05fruYOO1qM0nxDcFB2WvKprz9kHgtMHu62dN0+Vq0bc49dNqZOS34cxG6Gi6O092vm1Ixdp2rMcuFgarynJoVCbztkDql1kh1n\\\\\\\/y+JGBfr42m\\\\\\\/7DlahRU3aOsl13uqyHPDMglXlsMSnI3E3H0Me3Euh2qZ8UY8WKxuwvIvSo45GSlkWlUkmR3JMcC1w86e7HdlE\\\\\\\/B0CvRXucDLYKO5aGhhAATVYr+AQwReUH767gWOBrTPiM+38mSns36JTrl0GM8mkpul3B1THzTWm99um654sq5kZGt3ckCRlYkUjXjUNWGIuk3JEiPHfbpBA0hZuX\\\\\\\/YRvyOj2h8\\\u0022,\\\u0022iv\\\u0022:\\\u0022QvMX+hL7waNhomIcRXus4Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225879fa593d1e0f2626f2de4f775818c5eeea8090d709143d1b9a8c44041cffc8\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00224gwqwqfXRNHhtUQ4mc\\\\\\\/0fDVhdtyaWecQqk534igq9NnATx13pCYknnoXFORJRmSxo0Kp1sctMsM80FX\\\\\\\/ShXfYUIqipzlYoss+xBbXgznqEzIIG4DADJmuC9\\\\\\\/873YHo2Ie0KlDFG76qcnX8wauNK1AWbQNR03U1m7ESkqNUMh6UyJ6GYcBD\\\\\\\/N0mSSefCE63Zrqfbe312es+8NRvOB6L9+mibyhejelh6YEqBpl4yrR33WqP8w2ER\\\\\\\/XYS4xBf63UlSJut8JxEoDw45\\\\\\\/XRrlNwncvIMODyVQbLnow8sivGQ\\\\\\\/k5825xU00ifcc2Jp68IRlkYtHa39+d4bJroXTa5l+6Vq9sjsXR4FL5NvKrdBD4\\\\\\\/pxmalfeOiXOvd6scfPBOxYk9yMPne66MiE99SmJXbFyNtotFGDeG4QAUmCfOQKD1PvygV6bKuKyXG8dK22vDWr5WVoXQf2FEgvYcxGdbQ28tvUhNLCOTbbJJR4QZ4XG+nsX6mRrL281tWbvWnsJu8vPxggkrqqpFUlyGNwg0m\\\\\\\/QnVIhUxKQ5BcyJYVrEQvOByc5VTWwcy3PZwHu0iFQHPVAw8Z\\\\\\\/6eVIaRLYzvsx1iygoNmBTfV+yZRQyA2XNKZj38Bz6bxxL89COJzWvNwRQJm063Qj08pZX0\\\\\\\/zjayxw60lHC82mfnO846WPkHUGFA3UgMTrmLumWrISuo5T0cbcdM7EHyu0\\\\\\\/iv3ME45fJanDx6oaCUw6KFkuy45Mutn20ZKXGyakdN2nCyzp7c18fb6tw8yvw4s3bfn7sfY8TN8PO7GuonyssKm4VAA\\\\\\\/6XbYSlHuxWZgZrBi4itAhILF079ESZNVWwUAmPpK9a5HT\\\\\\\/RgUnsUnUC6novHPxhgmv+medpGUF9Z+rGsXmN3myGnvG84pDmcevRVOdEoqS6lGd2nXrIg2VXgjWoAUSB88aEoe4iqvL9cxFrnjoeW6IjqYvYaguw3nnPZltMsydANdlkkXMPRXBaD1bscdG\\\\\\\/rjkBj+w5QENFYcf2Qe7UQMfWWnVAGDN4w5YPpto4KbRGjEH1xCUTN+o0TMiC7bfxnZnHyk8Uab1NwNaDvbXbde9WgxKyk6avtLDoSEQnc1WHWUZy5H9GbI1AlM4YbVOp2RvdTe1xkCJJhC3G4ysO6BS+XNZljVJDhSq0P1qigDGig5TugpOeZr9wYig4+ybFQ+uEkDSWWYzi0e2qp0eqgvVeoyQTZ3nx26Ok6Cc65WV\\\\\\\/o305kjzZGMgu5LeMZLWiiDxvHWLe6QerDAFQC0ZybAsx0a6Ar4v7kfDzb8+gbzSGmXqcp2Dqk7YTxXvAjRqNe3m4N750gDcMbW3RtWjozeM17H94PJOrVJnpgeefTbB5+YWwosOEB1mEI+\\\\\\\/16y5qXFXyh4P9fqmxK\\\\\\\/f6Uh2KrZynzC7cae\\\\\\\/7rPnp\\\\\\\/b4SFmy7dRD\\\\\\\/0hJsoW2rHhlCyadHun4EaADgwaEpK+a1Nv47s9BTbrEozKSK\\\\\\\/MjRiUjuXwWDIwQIMys6BYqW1rpavaZTj+aTXpky\\\\\\\/nz2vo4\\\\\\\/C8beXilP61XJmcg\\\\\\\/cslOpa6S+1AP0x4Wa42lGzu6xTQObzBqfU9iGIBSwVCQ8ulYR8GCm5lmvaIbek0E0swRX5gUO9o0eaSySxRR3nw2ximzFj+AW3m81OMU3krrq4Xb+eUfJSMrbridNCFdcPzUs9ZziK75krmq57PuLOhzjiKkhh5wQ9Ql1mL6aRxhy4adodGfG3xpWFXcJBzSiOHjr7qf4DIZI9G9nsCra1zLfrnTpj9Q2yLWcMpcZFAA5jpsEpPmeg4k5iAKuAOhI61nYScvNU36M2pbAgtsu5\\\\\\\/7enhtb99paJ75j1qip\\\\\\\/bXLIeF4ht9TCvmoOiNnIijJNUzRu1BYqIogzzvWrm+W3+iJT\\\\\\\/IYjDERt2jYXlMitBq2kwkhDW4CaNYv+oLLdX2647+P+NcAvYuAFF6f0kE\\\\\\\/\\\\\\\/lY8dA1FFpCYO6d0bKnJudwcROITHoxgiIZP67P9Y\\\\\\\/qlboI+kBlRSlSE947rX1JtEePvgGbDwXsXDjJ3Lb26MQ+mqWfCP5OGK3HvBauGkPeRJNtHwY4zuuWhWb\\\\\\\/vXKsyupleTErlmAps4e2P4tof6GWhM2gfuKW6eOuLg8uJJUp94IrDq3Nz0UyB6RO7d6iwe5YoCbjT1Z8ww1GCfdmuF6oGJtfLTFu76GC3W9ZlXP0bVb4cvDMrJCGvMth5uLJL1as+AQx7w0ooBgmrj+tq4GF4YSa63r5ESD6A15Bn6Oa5Dj1bysszoq8q7CkTIS4rUmJbihO0DgEWHD11lAowyKFZaj4IriYWxiAfBIUI27MV4f9D7fSW3hxvasCbgj7nJxf47WaDACL16o5FUuYxZRO3PSai7+p95CY+aPRJE+bE47UFv0FkYZ642ionnTNJ0stQS9M+JnOiZMQm\\\\\\\/dlN2o3H025MNLtx6QIjV5e5753YNKDn0F5SSqBQ+RdRw9i33LMPFtr\\\\\\\/8HU72o9fL8xX\\\\\\\/FsQR53cHFlrDfuj0Ih9ZpHIsb2Qjx7XGdxSHn9dPKnMWPFxyyOsPvDIDxggYCt7NZGmT7mgNu5hwCZQ5WvSXIsqkqAYIJvMduFr8av+9Jv2n\\\\\\\/NO0g80YT30PYo\\\\\\\/ulWrtITIAj48\\\\\\\/EfEXRslTDeJQJPr2WK2sqSVj7JNG8k\\\\\\\/NfEgwXSdKZ7myhEnq9PTYOGo5txngH9oRn0Nih0Psu4jIKxztzNR9fPTZbCeQhQx0QOYeDLQztKddciD\\\\\\\/3vddio\\\\\\\/zQhMslAArSTRF9YQ6ocTIYFd8nviyVGn9geHXLUim93Rx6bpmd1CsvXhchOzWwavqC1ZemvBV6f17MyzhlcKqI0FPGcyEsDLNgvLmeOUm0BOzfEENviHOr7OScVWcISxpAqbXzLK+huVjgEUCVdynkkbSwyZVJsR1w8T\\\\\\\/9S5SpQww\\\\\\\/E8cnODpOinS8SQaN5EW7Dpp3zgdmfVelMZSZ1CdQUfTM+Iuvyv0zu4bAdUkdHQMH2PLiZjRRjl7zPSXg7y6YS5Vaz3MrM6pphvHScPkM\\\\\\\/gtWbcaLeC8YEg+QmVqhTSr92\\\\\\\/42JE6flmatDmK5mtci6+yQoGVipZ1ij4C4BVMP+dhVff\\\\\\\/ORuJ5PcafpEIWLtWgAvG5AwuDBBko92X9R0w7520Zfaafux8D\\\\\\\/5lZariVsugz2IwBglrYuSg+UU6O+XOcgFuFSuFsRG3yIqNMiKg8w+TDwK92UPC75nJ9TogoKPOzV8u+H73nvcR9u5Hx\\\\\\\/qCRIR6kzpV6RH4szKd31FMgN+AKxf0i4pwCV8lfTnZcyDKkr9TMR0jvEZRG5tnC\\\\\\\/TJD6KtpLiKfNsO42edKMK3VL1fFdAVMJ84doD5g58ogIwA4eSKxp4N3nTBOnAJNFReHRKv9wLAcpmyCE4Ja6qbphNAtqMa8Ya7m30mIy73vczr5S5jge7Uf6VJhWYqgcvuz7IZctNdOGkWfWa+hArY7m\\\\\\\/RCwt1VFDqhN7uWc+Y\\\\\\\/P\\\\\\\/AjJQgSNu5fMJt1Ps5pHX9aii7zgtZ9yu1DynkpaODaPTOmhSsyasKum3rq7kd9xifZhmyAI43WuYojYqv4MKkkzf\\\\\\\/0rBnhWvctgddY4+0K4LFVKO0lHIxcgkKaKGPZPkPjPsuHeSx4lZuxmgBPX9e45qSZNdZrqxkVRXX3\\\\\\\/11HqckiFozELS1FC3Crq5\\\\\\\/GKcFM6k7oImkZiGgLNKcaxHrMD6\\\\\\\/6h+5SOT2wnBzRgU24zNQM6r0c2XMbNQCRsFM6kwQPKqOmVHXcZYTObo2O08vpRU3yvn4T2JnuTv8yutS3n5jHCw93Wp9lma0qlQi5ZQ6DAMZro62LX8bNTPy5PrPUD8uo1h1rjwxYTfXQqCPYim59eilMOlcys72BCTaRpyA6kBxMPbWX4qXva4MI5wseJTiwop+z43pZhEK79uOsSpcYmsO7TescBK4lEqmWqp8le2zqpbn6op9itfOyAoo+Mc5zPkXEyzgA+BASBDb+oxQIB+U+S4AJXUOx+TW1AoYMK40cqVfYV1kt7DxTiG+Id1hFRr+QxvpdT4oKRrh0gUwwSqYkiho\\\\\\\/XhmPObeQU\\\\\\\/StqO+oF2Wxjzq8sbb0L9hnmAdlQcUXwEqTOwzC\\\\\\\/h2oNLxsppPPF1QCtbKh0L8wUw\\\\\\\/MY\\\\\\\/PZYg+sq81rF7orYXPJmiJfZunP+XZ+gFk8z0sTkzpAv5OoYTjbAB04a68hSiQK9yGNrm5qXpqaAemzSek65EqOqijrTfXHgTAF1G+FZarIM8MHEPuoQX3Y7Mzxc+IJ9r7ucf5ZIvEvFbnaRD9tcHA3Z\\\\\\\/6vn3HQiBkTOFoQItGli\\\\\\\/OgCd3TLy7sfyIbAKKmV9pWs5DOXhRcnCZgIR1NSam6pZefYdteeDBu\\\\\\\/jBqrzPhX8AE3bE6Cs8rDYtu4MX2v9wYyGlAofEpPT9pNk\\\\\\\/Q7J\\\\\\\/eZDYOFpZJ1DGbs30kAh+oMiIdEgPEHZmUHwlKTA\\\\\\\/jfKdOMLDaKhsr28JkdbcaKbGJfBFZOs3kKDPbAG5xU0cpZPRGvMcVO0A7T0y+TigNgYcEyCkikmMaLt3FK7ti3PAYwFz88MtY63SS9vgf5BbXKx6BBT8eRY\\\\\\\/9sRidsFTtNrPz9lIYtEd3Ple+P0eiSEMO0H75S2ac+J6cZGqs0P80hk+69a4hMbyuaX+wTSxVXT40rRuX4tk5yiX\\\\\\\/BbaCMKnUOqrnWmX3QMtOHsh84feZ1peJ8WmRABof5y+L7XSbqmApExxXbCWY66A5P7W1M7kjuNDDIhSde3M1KavJYc8Ycw6xbsWvTL1J9LHd5O\\\\\\\/aUByugUdSKvgjKd8KphkBVsREaNLdpQTGDqD8PBrO8NThYStQK79BSNGaZ9NIUGvdtcnSq1U848l6ihwBG18KINMH6PNVYdeI8KSeRZXxzCiAljVz1FXv5O\\\\\\\/pCZuomMkuzrr\\\\\\\/hP+XMOQwniKkCi0yVcyRGh32GEKrF65Kb4OvTMt22va\\\\\\\/CfL8ArBzhc38uH7PUvvm7fDOEJ41byf781Bmh5HUMOnNOc6zFoFulJpbVWiDCDjSBJ\\\\\\\/2MrFyH77HnJJm5eiQQ3fiNQIX3R6SA6xZoqrzwNiR9GOGfixbpbllrG0fPINkbk2kNSDR8SKRPBYXC0PGy0ZDcJDmKGM604WgIf3hByQB2sUW3mdDJazLtZNMVZVfeyRNpMu1JyCieAetdcioQ7z3gtCNRIwC0D8RQnlveJLYOTFez4B6omkrfq8Djfg14T62kRoss4eXGGsRX2crC\\\\\\\/vdHzwYQ4YHxZY5vqYxX2udRLgHdA5nfTdp1PkAcCb82jEILS0knymHrwj95Wbx96HKafBCKExdagPXxBNZOSiWOYsh\\\\\\\/Df3FkFbW3LfFj3Mv9azvQrJF7IXDynCiGIKImdHo9eRNjI8xMcs0pVVEx6E0Ym6\\\\\\\/BM+yVafYKNG4ggvHgHFxhYnLAUnBAcOKZOgFapUB5ucFtlSvevXKfWsOJUZGpBx1YWMfzG3G2CW5DwSu+z2v2Rai1Ojr8NWiuxdj5vBHEoBU0m9jlndDTfn19CMq8DSZs\\\\\\\/BLkVKvCrUUnQgzKGzbAqg4+xqxh9+QL\\\\\\\/DLKvN0YrTGpwQuH5IltZ3pAkK1NzYrLXHmpCpS5H2KLEnq93jQW8DHUHCW+PJMhqBtZmP3iBEcacHmAAs9kywbiYhFV\\\\\\\/+cTMtFtFewOW4KJFI8u+npKHD4CkyK9Z1BLateqGHYr4BlxrFsI9yjMQJBTE6fP75qtWmmF3cVcPfVHpYSZSIT3jbv1TVpGGYJavPc8CAdSIcuneu\\\\\\\/D+6McSJAHSgTcXZVZp1q+vcpvMsfpeKel35KCwHe\\\\\\\/kN6de\\\\\\\/TCDkNY5PWhPqQkThsmo+0+fVOPy2CALX8p7lN1pJiAG939z6JGM0NjmBKHecBB39QpXQCqbnL01MCd\\\\\\\/T4F0Ia5HdgVNPxx2NZUEOtdlWqcxOyBxNoG+liN5lv6piQpADdaVPXYvRx52WVuxmlcMxB8rbFdy0ovqk+tt6fO+83pDrHUQBxNOvgxtUzY+g9xp2OjOXt6m5MG1BG5Icuz1Q\\\\\\\/k81nC14GJe\\\\\\\/OTRAWFwQPRO1NddpdfvrhJpZCrbjH3YaXAAgGKovI8hD1i\\\\\\\/VB\\\\\\\/uR2RHtQ220TAkIgttPRAYjTFpHh57hdARZvzF7S7LTuhJau8M\\\\\\\/iBuc03KY2k1\\\\\\\/JnMPwwXfUcU55fEBSLrkTzVcxOtEH\\\\\\\/fJKnrqPPnlCKbQonfSGJP9N7Lp1DXdn+TZPt+qzeajQazWrksjdt5OxAKJW\\\\\\\/vOhc9w8hCq2NgL6YfH2uONiWMjR9dyymzjjquBKOy0EFEXRFb7aF43q3rxwqbtVfziTXzLASOSlhS3imd\\\\\\\/7HR7ZlIK3r17Uu0uTVHLJCLh6ERbxhXRf\\\\\\\/9DEOMhNKhOeh4TbH2lVjw+AXr\\\\\\\/iVj\\\\\\\/6dL2Q+xFHr\\\\\\\/TZrC8NH9G9tQE0MUyDAgsvA\\\\\\\/giIGYTrOttZ8oPRRxWni4kwXWwNlp0OhS+dvSM6V+Dun\\\\\\\/KV\\\\\\\/XflPZiRv+2HtqX5B2YnmQMJd4ouzwDn+p7FEy0GXD\\\\\\\/hAUd\\\\\\\/sOrZEumkzVFvlZ0c1CcW3ShRMify0xqEIIGZVP1PxRM35TeUM7L+nVgG55+SmOpBx68aMiWmqqemuXmYJnDKlqb4QfQd0FoqFrLpwRaTuqdwVjBa+qfRw2fB0zNmM3ORCM7973HyroCcmUdCqJvmTuMLizbA7nvjiwhgmwI364aelHx\\\\\\\/IEp7uoHeLyjIwQh2jtVIci6itO\\\\\\\/YSLs+jTPboph1NKb5R7Wn3TSeiJI5atr7xpjqohYbR2ybt335kP1qvWPJ3N553Z6kguyWjeHiv2WhJm1r6a4M5HUNhS7EMO9feUoUQYlIclxYWbiEum0L7aie5EdgTlUIe24oziad5+WPwi+1sn\\\\\\\/IDXTxvKLFgSklTifPiC\\\\\\\/TKrFdkkE9oeDGnztTnI600iyDz08cQ+h\\\\\\\/nDszoIQTEEAYdnQU6Q6A1pLqiuVqU2RetqaKLwG4abS\\\\\\\/UaScfb4alnypWMtuoSeKK1fs66qvNBIItt4wr3WcKp5mfOEgQ\\\\\\\/g2Ma0G9XLcNl4S607YxVVFN+M\\\\\\\/FXs7DgxiWfDqqhPPhJGAwVBNMQi8WYHL+RL8HMUf32kSbDzEkvnAuH3bkLCVvBKUIvFE5Kx8iuLlDJEss2eRXBpovjO7vR\\\\\\\/vYwe8y7A0KNFq+ZXh6r2GKs63+fTPQLd4iXxLqn7pepOxunZ7yt3JzthGFTuyt2+s2utXtBZsJNNrJ3tGcTmYHj4Gnxqt97XMaBZ\\\\\\\/l6m2GzglXIsJAXhY54YTQTjXlRHI2nApj8duZVzj1sMdcKGFZjHLFLTQlRDjZPtfmm+JqOtucoBPDzRAHtVI1+7LZN256XEsOYvxp2Ni5FLX7gDIOsl+H+8xtDpPE4sBNXviPfC0Z2DzyBmqBeOCGYc+ZGpWu\\\\\\\/HlKO9EsaR50vlKw8vSyKNDshMyCuD0LRq2pNt4\\\\\\\/78ZUTAH52SMwozmU8pPjLIeknINVsGTvjOat3XZtwgcQxCR7T1ojViGco+XmGitT4WvIj8c7LDtsHjHJPwhUG1epLxzTN14iFsmRIffrWJBBE6JZJ2V3E04ibBjbkGcpImKyKJB0L\\\\\\\/XZThbX8ti8prYEOfkB40w\\\\\\\/fdFw3RqZJ1LAfrfjVWvk3ZPv5ydxs9XRC+T\\\\\\\/wf\\\\\\\/eYHkA1uGOZBtLKWv6Pv2jbXiBxxXp9\\\\\\\/Reze5IjEaeUxBcTp8vdhMyfG31taaUH5Hy7bTBz43hAXjl7z3QDi2rKcgha32mqcg8mTUx12Qnaut8Zs67SgLIZW6odWElPUIeuIaMHcsLVpbLQVcIA4Cwyj+HtE2DFBXTZkZswmKGP9eysh3FThpkefVHnyA0O9n2tcEHfN\\\\\\\/i8DzU7uX901pPCmaLFPldfF7oMwEPMd29mHlpGoxFjvjD0\\\\\\\/DuZrWzTpaVbgMtaPWvgQMhoBlA1o9SqdbZUbSJT+fgkfkr9jz1w+mWYh1Wt49BlQzrHH+6\\\\\\\/wKAqVC\\\\\\\/UUrd7z2HJqD5Rza7lI2DKkJsulaQiulTnmIA3eACeeco+Gu\\\\\\\/eZCpxOh53ZV2X6pZhCfVASIMmV74z0ymlJpNETctdJ3zrzWfK3L83WVOi7WuxCS6gbjcdMErEHZd\\\\\\\/cXE2BIEic8W8iVnvnvpFLphE0YJcz1PK+l9VhpeL1zI7280UCi7xJ\\\\\\\/WTOqGKzhx1C+6juGQwGHho80eyPeGRYahMmMMWKWVQqbdIGIylO7zbAvAWY8Fc39jLalfE4L1V5f1JB17PExNKIneQgZaWipgHSaNOpDvt0HMR2mSJZdODh2wJHaHH2kXX3ltMJYdVyVRUu\\\\\\\/9oxnNvZxHBgxGMHht7UW80J5qBZcwAmGehdCa5EPxrvsbHdS1hGQNR1Csv\\\\\\\/Tz5raypM++E36VvXgmSv4MlSyhepOD6p9c3Fqdhe2GpkDGza\\\\\\\/sOMJuET3g7D7go+rdCnznxPfBY2QJIyPAxLEiUhUUNDGupgOYZx0l0Zmc\\\\\\\/WAvMsYvOa1nNody0JyLAAL2QntqwCw5B\\\\\\\/cg3i60qdjj8WoYh6Er2bFd2nCYb\\\\\\\/yFT19rDUyPCqVsZ\\\\\\\/t749PvHK1ShdiOYSco56ICVto47ZDqtVvc9WcMimeD\\\\\\\/Chq4Qv146D9w4m\\\\\\\/jLz+yVg\\\\\\\/csEVdpzDbUjCu4mgasZCg4IDg+Ot6I0IP3V6Il\\\\\\\/Pdy0a7KLK75PCj1YNDkGmaIctbQPRZ9bgWRZAPAkppuucwJvjUAkE5yM3GzUQrTYTMrpXDZ8bYNqoCOeOUO5STjo7QZ1iG6YSRqYzARRaELzLcTfDMRYy+V5cZ87XD97UhPclYBD5IYTpNsGzTh18PfkkkzIDXk1c8Fft\\\\\\\/NgW3qn97d0FzFBE3GyqxP+LFEM8SPwhmZvzYRWyAd570+RbGtPzuvHMbCV1tZTgIT4qLObs5mkFiH+R\\\\\\\/iWSNUVeWxwzAPNLjGaRy8EDuhx+Cafmz2JWWH9Rg6wUUeZf2wfw++swebgmpY4m\\\\\\\/2k\\\\\\\/sghbv3egXRbgONx6xramgtydkthQLlmcVXIvDwsGweSjVJ7Z1t5TuTx8efB0ZfJrsEEP7v\\\\\\\/TudycXgigCHxmXEFkllxaTSze7uJ54fQWQWjqOl3b1VFaSCxVlsI09Tn3G\\\\\\\/4\\\\\\\/H8ln6tFyRRaqPHxFy4vg7orSZXwKZxvdf4+sJHkOahgt+VciAivZl2jP3nT1t4tj98Sw0Dtax7hyK7vdSSjEe1jvNYuzP9osFj2ijvHM6KxyeEBXhJrsRuotbfUQ2UqQx6z1qtsWG3GPnE1u0SPHDezTHNrSLPmR7YHqSTC1qdCPn9vZI7alnmitXe0SJk8VeU0gx2166bHrX+Kr5wlw7dY9sw6oUDANvJBc80RS4fmYL2IOHumG9w2RWGEIevinjHVEGLt\\\\\\\/Cgiy6tq1QXsB76gPEFnyhBeyVwF0Jr4o+AoHp7Gf8iaStZwq87bS4GIdH65sG3cVnBaj0nmJbh2NMDgTBZITceucogCdVBS0g\\\\\\\/GwYRyZWkuLUWDbl7zaeUFimqixcu1vRJ1S8HuPFQ1M3P3ASyg\\\\\\\/XXWQ+Fw1gGX2WWBZDJvotY4UDXeETdfpYEbEqVGKFc143T1K5N8hFWoR3W2n1U064RZhXTDrHhErEgt1MSIzdudXcYpuumliAN374tmDrUPRj7fQdPgwx9KVRYfOZJry2s\\\\\\\/p5BezEoxDLuyfUXNyCYCKbJdjENHinAk6NnJqjZKYbawfe0k8FP2+heCWlcnQ1hc5T97H7wWQcEi92FLACkb2vhOnFkA6s6wBVlGkMuEypjB8mKp5PPswb8sR300Gmhsn+03Oq1+ZAdQ+eidMTH5TJT6RP8SVESPeSFlfJhC16ceMTCDWFy4fSwPhDF1A5OYwHAQw8OL8kWdM54CLKgXqq546QNGhdMBELrZucAOqShoNhGYhHNvQehdJ3lPlDvLjJHs7sl1oLq2Qr\\\\\\\/4vumguCdcTb78kVuIeVbA6\\\\\\\/PtO0CS1AjSZjIv90VcS95gYznUXkYRRnnuFVz7cBNbpTpOeKGtCaF4wOFnLrdhLzpv34QRtx8maVyq+v1WSVWyzrxdglGA6uvD7FGfA0t1JpjdVHlgA4HiblDINnCdl3tV85BC97qdQQ9wvPnNo8Iz2WVfF5938RY7wfKx+R0IEYN1ihT8ZQveD0OsryK4GKN96EXIDMN8VZKVjBR2ImQHh+\\\\\\\/+PJT3RoEu9siNrLoSftqXBDcObtpoPu7eTB6NEiE78m1lkCVKTH\\\\\\\/9OdBC2AqfYfDMbfDmUjQjz2+t4mJ4HH7csuduczxXSho0JtHDMGey0QBMFcg4SW9dMX4I1ETqHEYtKyeog\\\\\\\/gMx1C0tGsO7fzdmu628q8fF+OAlXZX1ev+YhR7gkdWFSB0ObfV0E2ueO9gBXqSFMUoRLcjCV61i9E7fZX4prLuBpr47qhtUxnV0757liStGnwwpxIT0sMwY6xje2VUe\\\\\\\/9\\\\\\\/Sxs8B7Xm5WzfBkjW8XWiFGgOsFDWYRfd\\\\\\\/1IhmyMFX362Cpzb0v2HSdjlSclz6xNIX96ZT8M23mblbcKzsnzv656lOxN92O4D09mV1zI551suPNYPwHpD4xoXZW51GHjOxPdUeZhG3yiu+sSg+82Clo\\\\\\\/XOYIDKwas1sSrl4rX95MT0WMfn1mVKT\\\\\\\/7oC1Eegp+p0NermM5vMCEfaxdBYD5r\\\\\\\/pg8wxXxEZwHZ7t1CBPJY8wOQ+gZwTeI2kWTLyH5J2J8xvBS\\\\\\\/syLsHgm2TnnGGdL+41gieMA\\\\\\\/NkD71aRQpvevcUFUgoDCyY7ypJO5eAAK2M\\\\\\\/WqibgPTJK4Ylwu\\\\\\\/cw964iWgfwLaToey2GTHMNj8x6H0jnfcGwQbdq31YcyqGsTpAMg+99RxsIJrUhw3jd52+VTj49Zjh2trbR7IouqEVK\\\\\\\/+CP2\\\\\\\/22EYvD96Bqn1FP6upYUeVmjEL246xxFV+lfxPyYH4VRxw1Y0jCs8MNVSmzzZc5PvmcfQZbivjCkV7Fq8uANOWRlLZhjxC2JdDvfsPgbxnqGcx7ruScisfk1JmxtuRFBxIej80zOqSlSuAIaATB3pj7Dv0tkib1mAZZm1t9fgfava31ROJCcRWxr4qyfaQmk0B72NLkzaEiuWrCf7r2kPG0SwsISJLPqpcbvvDVnf68bNI6e\\\\\\\/HL1GYJLeZOTSEfXS0puf77kL1UNIPG9gpj9DZK0QVIuKtKgibQKsDgLG7aTljsrl4UJuDGTLabno02yqL61IbZFy1pDJvKDn1IvbCVn3YFBrbyYdZlLzPyTCrWlubT+b1MREUSMnoTC1oNpToOQ9T+IbCqW86KU8na5beXB3sDIC5xjJFppGqxmzthr6xOXKKvkTmP2jsHxWqc6kyuyqZieNmi+oryDRF\\\\\\\/rBDo0SmRB2ednWItGxuSiB9Mt\\\\\\\/xn5+wHCm6i1+dw849ZiH04op8NxpqOdDWO9nOaUerCVtZWD3+YlR6r46vyRxdl4ENHgK4vmWuWzQPRlV5plm8CLLrv99k1b++FAk6Gu3mJpHuDYYC60nkMwxGf1HSI9UiLU\\\\\\\/K1pIwyuF9sabpHCVm0X8W7\\\\\\\/SyYGg+lLX5RRM6v5Ma+a8\\\\\\\/geip56r7xgTQvLomeOK8DkrxFAHwTkFfHPVSC9aJtkV2tx1oV9pwTG17cUbbTkG4HMk8l0bPc0FRCTXZETQnBVLFm+FfJw1Ib3A9uIYh4ZfzXJaWfgnzrWE2hMACXdw9axlN+r0fTE2x3aacVSY2GnrJKABFbS5002GlwYzuRabE83OP3bf6uqCjKEsB2IUMbtO+V1UmtNPvYRrr7M+KDhM9MH\\\\\\\/dWJrFZ0wwoPh\\\\\\\/jntRVCHmI\\\\\\\/eOSSHGghlKeXSF4FfTiQumyyCUhiegUFYc1LQn7YrsgyaHJ+KrT\\\\\\\/Omf\\\\\\\/nCwyJVPzBo7Rg2R9bAFu1eCem3pi+G5dJx7jsBcCqS8qisLJftQ+doDgSg4jOlydkHF6RqTnbJuaXLqZHiQ29hRF2XX9mXkJRPSMedGiJVN+9oMFVg0nAuMuHukyYA+uRtxrKj2mzjOwSEw\\\\\\\/cWYapuzSRX+\\\\\\\/bT9y2bjQMScBu5lT7xhiAGihOCkm\\\\\\\/igLw\\\\\\\/88haA7D7YFxRmTrPH2vI+l8boYvK+w9C2XtBfj6ltyKfjfKEBCXg6Q4MGiy+oQ4lNov6qzrfP5iQXmze4pMhjHcaNvn1RLeJ5+R7KeMmIEkpzq7kLiCrnTgFpGZE9\\\\\\\/rwO2v4u8sutlMW0gkxCwnsW4Mkkmpj75Eosy6K4rc66qYD3GPGnU5LcJQ6dWJ8rea\\\\\\\/ZdxOeQS1hqNnwIMnEa952J8omj01SGLJmrH3Gd20DmQFdGDkY+7JtWWWpxP6FCkG8kTwJLzom1YI9pIVAJO+2PXgJZRIFHBbqkzG9J1QlK4vqPknSic2pOeH6w7\\\\\\\/\\\\\\\/KjfJohd8Y2NyYZTmeiP5ELf0gLk4nrJdIuuTn46LUlBwBwLbWjQBwS0xGDizND0uNnGi79sFUoPnUhP1917sZU9KFijU4wqxt0uv4Mj4NdUBns+OIQKin3ZgPv78ZDnBKvlSpGq9ro28Vq0da6zUUavz2f3FA7+88TMulfMxLj6sD+0UKhUoQdVKppWHHt6MX6j46CsIDqLlhmeV3iRrxzOl\\\\\\\/aVkSASPD6ZwARRZZKWBxN33cpFY82p7DfXBE1MQvxFmv1CbqGAudRpc8sjexFnjIsugAhPPaJr1EJPhkijB9r8\\\\\\\/\\\\\\\/uFtXHAGpIFbIgIikBDVZYXu\\\\\\\/KQ3yvPO\\\\\\\/qI3Q5avJmsLh\\\\\\\/\\\\\\\/OdgT6iX3fZCZpSfebi61UU5oVTh8F57nzUYyJIYuhJEkBKm4jXKa5Bvp9qKB\\\\\\\/fMrjtjBhJSQ71geIQz\\\\\\\/iq5RlilmQYL8w3QeeBBnGBJTO7VEYYBuEYpNcoeRV\\\\\\\/I5vcTfWYsEVTAU02cdjlfI\\\\\\\/87oZyVzIqGBVGPQ50UYVAYzh7WZEI75pKvfHlMCzNG+4YLjjWU71yMUyZjTS9h6MHRlgQ5b5Cx\\\\\\\/isht3sDPCpfheBcc1knDV3gnvu+pFXg0Vw4BKvjkOSxD8Ci8cXMQRGcSP2\\\\\\\/BqfWyHwIiiHgme\\\\\\\/AC3p4T3kVa\\\\\\\/I5UsYttZZ\\\\\\\/x9VkAX+ZQRHMN4B873Px\\\\\\\/OuKkVEf5YBhkmrlA43bBpXnJ2g0lhlSdyaIuV4zXfD5KnpuPWBYccRlzkZMII27HYG5z86+hauNi4pb5oj3qMpCu+fmWe4RrsFRdqeyRjfR4UspmjQ6QydwQPE45UOkxhtw50f+QVqPZknaWOFJ0pR6a7fatIiW56d6w+JKRCmw0tIvBv4yb1wo59HElU70CxX69BZnwphRPX1zqgPZP0H4pXm5Ght\\\\\\\/Aysxz5q8BLusgGB+HR4lpVRZpuICLJ3E\\\\\\\/5tCMt7g5R1c5BmSgBYzm24go1gk7ZFx1P\\\\\\\/TNjqk1Y7Q+XmwtavzLRVjjk7IZdSmINAoC1dQBXbl11U3Z9BXtnz2\\\\\\\/If25E4mTH\\\\\\\/vSKa5FjcRBCDMb2Y5017Gx0RyQhUb3t15AaM\\\\\\\/TBWTldCyL2VGQ1WjUckhJNk9xFZ51Xy8irYlDjhXvgAs1Tw6pWst8WA8rf52RWrR90y1HhiVyM2rA3ia49e2VWV6qQidFpUqEi4Ko3zq4ZqmBX1XGZWErv9EH5zcgm8NvWr6rq471cH2dmtne8QxUkMA6dZv0CQi852xVxuU7jvUXn0Cba0M9qc\\\\\\\/46o+\\\\\\\/bU3mGdxDwGIKonepzj\\\\\\\/7fZStJaxfGG0mhB0FY3y7wUJvG3hFd962mMiAFQsb9ZMmzeNa7Lfi5BRDoFA8Vf+Alf+mnmaRBzZs2elYBrAn\\\\\\\/wUxbtq\\\\\\\/NIVCalhPOqTn34lp2erQKGQI\\\\\\\/URJvPTaig+Xn76qtcEDSIVe2IqOLGK1PlNFHoi93+1833KlRHOZLLm9zbmUfVG9LPTtACKs9yGvgNTEakJEJ8qdyD7wpv0FMoQHUIBmD25VY6X+kyxIcEashjYnSurPFu1w\\\\\\\/JciiqPg9nlhONyMDv+FYD40nmJ9I3DMNBGYv6BrC7m7pFEhlhduFUdI6nnIcHOHOISNGwXYL29Qyy3v\\\\\\\/xo2iw8t2ey8oG8uPPe1niAZZrV1\\\\\\\/ogWlULdmAnW5\\\\\\\/gbo6k0cdjy5pqEkSYVWPmrFneBDNQzN5CqDLCPYd9aaEmay\\\\\\\/1RQFqhjUMDAlYi+Q9eWrcASB2UefOHixI9fO3yYO7hMr8uQ\\\\\\\/T8npvlDBJoRVgrPhxU+ocPWYsVb7VNDZq1hhJUQg1EVLiVeEFiEUBc7iF0sYnLy9ygD21AN0Sxg1I8nxAbogtSSkOvq3iBs4ddnH5ynPoUf5pDsn8kjknGvt5ZrU64YPar79NnKGRiTe+Tk+12geYJBK4cqp86LzJx4oRGzp9jhtd6fZ1\\\\\\\/fHcZq8Hq4qA39RCUOyjShj2M4Tsw6Fwc0FYw6cgYo8LSpzixlO43EzKbZnBRMtAmEqmMWvlDPFG9mybOtL\\\\\\\/YoAQ6bc7oXHvDvorZd08b9p5yJZfP3zNOQntEPGP4kuqaqU7X\\\\\\\/rP0VevyxEyjJNpgL6IH3N6sNEKPscoyIKudYRrIZ144unh\\\\\\\/Kb0KsTR+zagH\\\\\\\/\\\\\\\/CfdoGg64532GoaOze+M5NIR9xLPazCstQe3emJvDVcOQ3k8kIr0HKf36cYn5+CboQLXweb9wXafmCB4d57uxYL00n5z9CdcWFijIN9xo\\\\\\\/LFYu8PlmFW3NbH5AKptRCRLKq\\\\\\\/kwhVs9cmNXMcBfm9b0wLoQNo6cDNapy1VW0H7o+oHFwyEe6skQ\\\\\\\/5GZb1XmbeQ\\\\\\\/2dxBogKm3DA6IHIOUMsV85ykgXvqaAWlDirhRdp0o0Pf2MTln\\\\\\\/tO4q21pne0rmlxn2wehc3lruu+CJORbV2HcV0idYAbPWfxAh0TD4psEywZMMbzxldISKMLBiR1kX0cO+jPXVpN1L6WMAGjODhpmoI6vnDURS8ik9e\\\\\\\/RgA0YtlSEPsgA+aR20OA\\\\\\\/TUHNlXToNf8qPCXf\\\\\\\/w3P\\\\\\\/NYPzXZ6g9D2pNGNdGFYFJXaHn3CszLDkFBrCB5aqcc7MNAoTVobVvA9IQXFyQx+Kluceh92hrU0GKl6nie2phnMGQbKgVdOSZj\\\\\\\/Ai20YonTAi5wNRFCymfR3GXCLWFCy8Bq+WDCgeb\\\\\\\/WXQ5QOdhQhPhr+3DowIlpzuq2LVM2JgwPZc9JRTxMNsM0j\\\\\\\/ONLvoZBcHzsyDrp9xx\\\\\\\/7GH2MTdZdmkcWKmt7fGpKd61akDmfDzWiigNzpKqXlq9wfQY5Wn3r6b6fXW4fUEFk8ltrgTXeUd2wVwwC1qEEKiW38H3QF6sgAi9kbNsGXgANK7AGGFukZFFWDIMn\\\\\\\/GzElucNR9hnqhCKzDwifpxUBER6vSCiKbUdwo\\\\\\\/GwBpuiZOfcWlE8SNuJQ6F7m6xNyF3adTUiHDinXUXZHwal1pM9TUOi3Ja6Cd2Q7SbP2K0vm84cMb2KOdzE\\\\\\\/dqRGVTOi6g9q4f7hEnr4\\\\\\\/1l1\\\\\\\/PrfL9zgdcPiDAFcxo85KiUSJ+HOPrSboceBqPk0I4BPMQz\\\\\\\/bMPnApQeakrzE5OBCOhCkyiClYlCtXNqqUIkI5o21zCibSmH3KFKYOmUoaMbbvHzjopLKnSSrtYAvmEedUNm2Oi5lchzH6R4MhfJKJHvVLcLOOU0NxsSeAp5sI00ULYSOJVZ7ysNsTF8U6n7RG55kZkEmXd+fcZgj26HqnB4msCFg\\\\\\\/ZRCh\\\\\\\/HOLeyXR\\\\\\\/ubTeV8ZPQ8w\\\\\\\/Y5domCwcUvYAhwALMDHwNORpO0dtGfVEQgqZ8eYJnM8r7HyDzql8vNybwMs\\\\\\\/uQbi2PgztyPe3c3WdURYW8KX+y4\\\\\\\/WhExV8fJpP+Z4DjdBAn5R3kxasfbxxzYhSztoeBB8sn79RnKaftsTPNfnr4fBWX3tPgLccucFGY6YSVlx2wZyNt8HMA4Zd8KQnXkayqNnqOO71wCYl96veRmCBKVaMkEOUme9R1C61C1fE2Ll0kIAJ1tK5b92wDENKGa+I5Ec4OHth\\\\\\\/HByEmqupzqeLDwggsc1m4AiueoeO0nb2W0JfZmHyZ61UqUrZQ2zAKj\\\\\\\/fnWIpYK41Ui2n5PWcu99aBnAfY21+9GEfpf\\\\\\\/JtovBwcqsVHsKd7poxESr6ujQUj0nyLtRH2Nupt9t2D3lgosFpwS3J2mAQdT0WKHd7M9Yv3iXyMWjUigCLZZq\\\\\\\/SIhtai6WGNNJcOsj1DZHJp91ncvqZnqt4AwJMDo2bRVbN\\\\\\\/mga72iN\\\\\\\/6buaZ1hkXjRqgPUc6TLBuArFn8NKlq0zJsObt2T0jyDV9xvBfiZkMMI6BER9FpCghzbTjzvflNZOESbWrPdioNDoIi\\\\\\\/Wd4NX2+2A3HCoizjkoycqyq8\\\\\\\/DT0EhmdYe8UC33I+zx1WBHiwrJI6k5L+cd+kl5e22btss0tpkiuTdvkl2KJgnRQhq9qG5yUtbhcE4YUPzmecAGJ2aDLOuwbiY49yPGB9ep0RafaUI1NG8WPzkLGfhE8uhM+G46L5rGw6e2DO8cI8YAXwTyPKOd1xkFrbYzcWXn0FQVAVrj5ZhBGpjx63ooHKVFnZWoTVQTFFad5Z7SeKBBw\\\\\\\/M8xzWDUpw5GIKMx\\\\\\\/gtiwi35pvORGuuF436WMB0nmAsG9ATMbkm19Pd6Bjm7GbisB06MMTntgVgzwAWAmmN2mcrM+I+Uf2FFGEhtmj6GsmxG3bLcN6f\\\\\\\/VzlB0Wk41+\\\\\\\/CzKlPJ6Cla6a9mDwC3HXkl+90cdBXihkmbBfa0NzN032xdbPfOg\\\\\\\/xteyLf5J\\\\\\\/pUAzUMssbua9NZpGrVKWdJFlGFeMwkU3zC+X2ckeVIGpzkOiOqaW7N0l\\\\\\\/rhb\\\\\\\/KjohU4fvyFpZLM01PhW2U9APUi6kFhAx+3yRsj21JDr1lyCniB6cmjDCGveXWzsykuYn7K2rpzfrC4GF97E0TrZK9bdbLRWYcfp0pMnUsFSSAdUDgvM3gIAVDVXhtcOjVTNNDm68IVLEccgrjcw1PdO2fZC7+\\\\\\\/gzEztd8Fj0JIye7Tjgw03wg\\\\\\\/fj+8UgDtW9Y1pCqqT9Al1xFKyziBhS87TgzdQH2qotb5QojUVzLLnOOtxCe8j342P3yuEb9xcI61AjdKKnhGfg9m85oauvA3mFV2ZwtfFfUzZR8t14EaUSiqjciFlTA6Nx4mwqbZoLmZ1iwypXIiVZMFxGzLd+re8jMh2ZTdDNbmHIgbFfDKaBZzpXM6pDuhV+SlEcm16+\\\\\\\/waj4vX+qlyiJ39K\\\\\\\/uRWOeWlEwWsrkCW2sb6fsqBOqXDyz4XJX\\\\\\\/LajxD54wTcBBqfHDdP\\\\\\\/U8tdA\\\\\\\/kq0C0f45QaDn1Q7iA6u9rrEGxEN9Bsgcl9VkNJ7i4rZtub3\\\\\\\/54u5zXn6CBHUdU0R9AJvlxPX8HXn4OgLUvXVqqlHZMDS\\\\\\\/xXKDdCOdNu1mpOirmjyZuXHmDAXK1leJrekl4A6KLZVgRyt3\\\\\\\/AytempxtrfTMxDqPXN6o0n2tbQiw1LKI4gvM6L7KwsKGr\\\\\\\/3QMPGtWfxSxL82K64V8Nk53sBWTPqceVYd2eSFyx1mZIX3N8z7Bzih3yDpgjWn+jgKN5A8Ue1lQsWYyKQxkDSQzaqSNs0QthtbLyyXd9HhydJycLN17UpRmBoJ86YE+LDxNnHfigvrP7g6BPerSa3tzowE8h8T0wRylHg\\\\\\\/dpD7NX4KZ2pa3OU8rN2G\\\\\\\/SFkkznVmQ6rRLcjpKZFCbrfb9fDWj7rpHdOJaelNgmd3sPQRCVV9V6HfXa29F\\\\\\\/SOoItutumKlVSlLVl3jfbAkq8GeXwoeLhcWrOiTNiP4imZOKXwPhz2aKAfen5icWOLjLpHR11G5eunh7\\\\\\\/E0\\\\\\\/t3DnOro\\\\\\\/TlfkRou480EnRZxZH1GZuboBZ0xc8Rj3PThqiZhOjzDK82tvSOjcpO5E5UfyQE2dS+HitGl4eeTf2NdZwnqBEr2TD0KBSYm7kIeF3emxezYN9TCUAS7Kqt+cPg03uN+jANjXd9OaJNnl0p6Ta2i0+nIAfHn5Gz2wPipjQGH7\\\\\\\/uWNrTxqa5fyQBaQOnNwzABx6m6\\\\\\\/gwFHs4\\\\\\\/Psbmx966uJMZ7O4yjYpBKZ11TcuIC+z+YALqVqxGBGtQC+PC\\\\\\\/dfLrT5H9lZEWuCn+sWO0N4i5tIVGk1GeHtcxtChPXUvSKqqdxlRq5gA1a\\\\\\\/Ef4R6pPpS2UT+wuTkeWtxkcqldK1P91zPICX7Jb62qZQeThI3Eqw6aNOckxQc5V7wcLy6knPplXm5D8k\\\\\\\/5dGGhZJKGYyEey5A+oMPlBukspwZuCL4vOOeDHciNoIQb\\\\\\\/5tZ6paGzDEndCm312Qu7SpcD6u4s0fGB5mZZ4z\\\\\\\/PvXsv1pWCJLg\\\\\\\/TwBLHOebJUQNFv\\\\\\\/MqJEsg3Y1Cz\\\\\\\/Q13Dl5obur\\\\\\\/oshXWtXHvdAlXuhePMGBD07A9CGo1Z+z6UKPhQ+lee4q8o2XDEvZaL+WjNkt8UdQD1gkkc8b9jQjFEK9btraboi1O4SuIbpc1oUiwCIxXeOBG30TXo7PkBUd9PhItJeffXl7RjFAT0EExAzE2ZoshJfOQDJveQQA\\\\\\\/v1xntS2fn1XRqVjNtySQ454WClBjiHkZmHJlILI7UJqChFmhaYBUVYgsfTs2eTcZGU\\\\\\\/SrLd2DbAurEvU0WvgqyFDeEQxmgoxSHuKqpBh1VCfgSin0WeOTdPFzpCnFIP43Ls\\\\\\\/glXJZ\\\\\\\/idCA\\\\\\\/\\\\\\\/dEMumJxS8wtkrZUzpraKrGi+ZXXCyOATN6g+EbZwNKEGK4igKia7eeN2p3v3LnmmXgIojwTRnlCmv2ImA5Mmb6Ee\\\\\\\/u\\\\\\\/qJOhpy3C8QrkzKxvZXJ9tRN3M3RdO3AQ5m3RiCx40svimuHaYjr6KmssMfu2ohdwmPyDvRTXOPWiq8FrUqqpDpfQdjWIpTzm9maU5YoFeCHE2WGdSGpHMiKDV612RDA9a9QrCQVlDWZiMjaM6Zc9v1gkzN7voWVSFtvIm\\\\\\\/ZYEerrocq8m8DjwX3NB4KMAwhyqV3AboPJ\\\\\\\/GRgOqxsAmpPAwh+1t1fw3RzVRsMxhyqdl9nFwJnEMJy5dib14R5e8LtVUDKNJAsKK13MUDLYV8xHVWspuc6JHWP94gKwS6aJ\\\\\\\/6BSfIbISjFSGMF34wNsMQeziTO0RXpprR7zjRXp\\\\\\\/jKR449ht+Qpun41cpOX5C\\\\\\\/9uhns9DV5pcrAcbapoVU0Mg2mo1\\\\\\\/uslO6YgMBCDAtdhsc949syljiW\\\\\\\/lfzvl8CUhHNl03jpIwDXXbfi0engHFDnBBlm+\\\\\\\/SZhWuCu4UuWVy2E9GEAswSZdRvrJfFadhHasBWGVZmYBH6pW1BLNRl0T9eDB3PaCqk9TszmSeuWtlBeSPDhpkINV1fZn9ugC6TadYZeq6UEX7p777+XUWacZbUYiSEL2YaNBquRMDrKdMmixN\\\\\\\/F8bGhQdsjGbazTTAPC5z9fGtWn2+Ydeb7cQUZ7oCdwGz+H\\\\\\\/nxRFc0odDAF5JJMbvmjEGeC8YjhLs+tVOCTVt2cwy5fNqX6PVRU448jyK3vN8CRRj3gDNEFtvgK4tlnkG\\\\\\\/oPZNIaDqFfys58sIM2tVq0s2royG1CWnJ\\\\\\\/uvAFwmczBvHp2k\\\\\\\/LCSjxCX8cn0WUzwqxdkLvq5n65z+vhDFq4WSsb+pqzuu3o2hzO7fnETfu5ocFLCG63vtfRCVjtb3zAmX6nAUVQTpHd4w0InV2graaWr4cr3sowiMr3YYDqMLfDxYOSAfulIOw6Aqwqp5\\\\\\\/TjPrdLP1ygnmat2k3MsNn9SIQNmWPpw12RKXnJoird63y6HEXxPkaDp0E6lv4j1xVKXv28wmhMZlmNvxZ+UJegVRJTs5oWzVKp9dz1fzPBffjGQZa9d32GmLz+8TFaBmXRJAzZ05sRRHqLhIse8q0HnqTzDu\\\\\\\/i7QaCNcJhOpqbmZniBMLJCD+NQtKQAw8GFo8INyQwTeImybdC3l2laydZTz+8Kb3xddf1ui55rmJfFjATVihyF0l+E\\\\\\\/Dz6VLxUBHEvUK5LRX2bXx\\\\\\\/mW5wR0R81KiVmJXJtEaoApb+7NsG57u5mwGEcgiI4+TTo\\\\\\\/AfEmMiOt2yxiQBoO6KavsG+IoRQkyosvRQSZjNFD2xcMhP1oo1FB2KQvWxb0wiqTRWk5SK73WQhHr0h5tlDxl1IXg8TqSsgWYjathML2YXi2QeKobQC7VhxmsY3JRE\\\\\\\/Ve4qEYj+b8\\\\\\\/wtlFhqNrwnP7mlYqWaC5CB5QSjVj0SOiS80SCFQ4SzifGMxRRiv8JocseQLJ++rsiMk8h2k5cCSIi8joT+k1IyMFzH6C68OZoW1frXF1dz8HVFnbwWnH\\\\\\\/FkHvMC5zcSYShHMLPtesR2sD6XMMEDC+vUKYpVSJqNXVqGR13z+QdN5TE4bQmptLP\\\\\\\/FI8D2OJX0\\\\\\\/5pPTBnJ5HsGCQI0mvMgZP6ytM\\\\\\\/PDlO21k3wkAZ7Sl1gyLQK6CN2COVeRQfZniIs+crZVo9TK7UAHqF9LSQsPoCZSmKZlUCIOMXWqG3yuzBGxmOpPKGvleMIC6vkBOUPwIvypb1MpvY53mABlp6Q3jx9pGVB\\\\\\\/g7gwyFgvYKHue11lLbI5YZvFNv4J7qHIvEEntA4xX9vUbDJf6EPNFh2c5oF+\\\\\\\/cVSlS7B4jJB24y4fm8pT3Ti\\\\\\\/I8qACHqqTHJF34mBu9zK1L5z\\\\\\\/FgL0O\\\\\\\/zv+U3dUU1D\\\\\\\/kZXNpVEq\\\\\\\/MfZ7fwnCfr+VgVB5uu1iPoU7belTB9nLiksV\\\\\\\/qnkldFOq3NZYHIx9o92yy7z+FFfPSLDTPqshgLj4YJ8QnITgRs8jMCKKrFJFNp92s0Sp8lbOIqzHo1myz94Hf3hfj9eC1y9BYQcLO0A9SETRZaZnWWGKcEFfGL+twVzjhehzoqV\\\\\\\/8Ns1bHNjZglwQH8Wz3G5DYgt3cejQa7sW6jcI5qclHmlzwMolg\\\\\\\/JGOEgoSt\\\\\\\/B9w4GlBv\\\\\\\/yCqZkwPB4lnozQOP+EHzXvKzgK7Kr89lO8mzFFzqjzQ1Je4XKvrC\\\\\\\/JZTIV8NyeQmNKBoqmlLXEBUatqmngUTS7SIkkU8gYXis+ykMxsD\\\\\\\/L\\\\\\\/YW45EWXyPdj52\\\\\\\/2TUWYoToSfJ6NumFqYsCZSwNO+Sse4ngbAt9C+\\\\\\\/gUNuhikr9RV\\\\\\\/EYgpEYyD1Nd7U9gI8CJSNfSpKP9xnA2MXMV9MF7+zgeXCwF8Y0lzE1td8gZu1BJPcnXlP5CYkl0XTvFumkCMatXScqus1RlZlxOvqlPEAd+i1Be6L1KGugD\\\\\\\/R\\\\\\\/3Y+Ofa9E+3YPut6cf3Y6GBERyICeN9jEu2y\\\\\\\/SOuGyh6hZEBKYNVnh+CuO5XGC8tS7xGpzUXMqeI6ypgSoNf0EeUXvYaVmJeRRhc4T8pZdBd2bQlP40ykTRVf\\\\\\\/73koM8LNtJMOMtmObWt\\\\\\\/6wyJF6vA10xBOfCF91DvolNbfFu+FlUofd38QXD8p\\\\\\\/UaMCkyg9N+5EHJvEIQQn9Ds3RaUIvyknLUufhlRldjvuhwTHEkKPoSOsACjiidhcaQWxbQWbrtqm0z+oo3vtyB4KcvcXHFuY4NY075sL5n0A+sxL1W5yyRRJPt5COYDbJWAISx+lE2DFpiVUIJqDhXW7EHmBSZYM4WgnjgFJnbc8Fns4YSidxJhplldhSVaTOtzlkUAolMiXf2BzMaaJzI50r2WA2cJ0uiMK9wWgR2+0ibFE\\\\\\\/\\\\\\\/dqsu2RIecFOeeKzmx8GhQoYuFlP7PLJvf+XMHap5i2tnCfM5F7px1x9oUOX1z\\\\\\\/CAQ17GorbtAYO+Y2T60Y0CJYLz2qXP\\\\\\\/OtlwUA+5DwkniSGL3HUF2VL85s4u+7MXNqRZGBJ+FipmIaVcE9R3sVNYOZ7jvDdxg4zqSrEKl+v5CsO1vDSwrrzzq0uX9F78qZ\\\\\\\/0Iy8mYXhVObBq9Uu2DYJjCFXLQJZCCCEA1TdOSDHROvc67gVOE0VmrX2WvBuTaa6KmUOdKGFlDf9AlgwD\\\\\\\/pEjNfAfqNG\\\\\\\/d6tkSEOQqcQYOAFeDo1u7Np8elLDh+rHffqj\\\\\\\/W6PkheK6Muvd1GN06pG2p0Dtygt6zmuTOQRQv5+vDIC9V9SvMCtcW+KmtfnjxXjIaHw7SWDArMk0SfdSfPbqaDJTn85Zt9zAkVVbGqYqDxI8EfWxt7VrEuW5cfjjhegz72T4WM5j9JuaRWGEr3+DI+FOdrS65nvlDE+pu7z0z6OuH3LIbifnJ29zP\\\\\\\/GmDzNrc8r\\\\\\\/O+ZImOZcQCc5ZPTOInu2TTFeLB43D0zkSZvZ3JsFkXIhc2MnIpCxbsmkOhFtrcv3LHK0mMYPL+AOBlwqZXTr2tugMtdVrMS95R9D6+tDT8sqtPBFA1eAn6xvKafzGvVbwDZWlbNRhbYwiYMeEImfalG7CQGDE7iHh1YsSjFA\\\\\\\/ehcb9EEEKnI+Io0Mfn0MPpGigHIBzcwx8ZRShchEgpmHJl4noHc6351GeqLsltROxFICNDRt4IXRJ\\\\\\\/1+mNUp8Qr9WoXNENax2zeTQ7vzQF+mp6UrecCW4L6auWUByjjq+KPmVd36ZkG2cv7QvF60tdLjdXvmf1tHlMY3D\\\\\\\/52DOl3OfrkmZA7ib7Y5D2D4+UbrX8xvxJWl1n7ta9hqIUYVF855dpzJJrsVAQQQH1NkuuuOLrdChlrSxjkMFxKuymmzog7ebVUB9Ss8FMdjt1v0kJHS5ZY82I91ZwjOsQ\\\\\\\/WLfvFtZAxOjog06lfZgHD93FeHnx9iZZIxFLk01u4uaDTqyPUndnwTvogsSk5Ed6GEn\\\\\\\/Rs3eNl10aB5OHBISbMeVkp1ljlTYkZPQ54JIpVoiOJo39IC4P4LS8UMfasaTe\\\\\\\/hg4eJ\\\\\\\/0C8ettuTfYSIbdCDr0j20+L3ZFiEG9kYxqZbL1piVzrVFoR3yYHdg0ymp2ZVQzsefIiJRDMG2cSNs7DAAGFiwpFoHtkU9AHvbnc9IIOHDxpLJiSl7ut39mibgneEAivPkjytjdLAVPgl80RBI2UbZbc7SM4BuUGiYpD5zkhLiBgWbvTH6c\\\\\\\/2ZDrBkB33YYIekmPcaNSsRZTOUwCnNNjs1ZlQD79mkaluDVHIZ1vnenwozazkh2Q91dYnzPnqJcrCb5gqQ7RMGyR3vFXj6D7ETvJLSkL0BJibEURpibvjWlS6kawLc\\\\\\\/yL8gPDL8zqyicDM8wjhIH+etivbhPLAL0v+xPSWZXHB4G9BKeSYcqAgdnd5EvlkpkTUj0NzC\\\\\\\/sLNAvb4b7gF4gi3xBaFjJnB+hDGCKHoFqFSQBcoNw5kUYhVrQjDM4fgJEkaeF4oyZ7mpzM+j\\\\\\\/0hgaWZ\\\\\\\/yLOffKWzaGUxmyUztUTSBtIEraDFNYj0hyVf+1DQJS\\\\\\\/dH8IjpQdAR0To1HLBlAKww2e\\\\\\\/L1sdOX4FQBx4xcYk4KfX5ymAtfYjDQSKWrBoQiWrs1EmCiO442CUgfbStUtt\\\\\\\/y4wYYtLuPWAblSpIae\\\\\\\/I7hM++s3C2PQnGdoDIr9rCGlxtIYOzwP6MF90fPOU55VtEg8MiuznbEVJCCK6ga5OcGtXf954JqEE589m\\\\\\\/lclx7WAkaUUVpsrW4AKe9J6qTNBka7YEDxOI5vWu5PQeQj3ooxucgIgr8jyqxAmuGW4yfXCjYdI5DE21zRryZZBtVnO9tllU++caEl5v5J3SZ0DnRcJjWvnovArGg5wAtIlSWKGFTr4NTn072pws6ZkvwJok05dyQGomNdkBGn+wpkoYatUCRr1dEPQFuoaDFj7YQoDFpQyBwql+6zUfWLbV53vUzFKNHrgYgHonoLuslmI4xgmxwimU8ABc+3hcJyTElzdfIhX\\\\\\\/JoGUt9YPLv3jqmWke2lK1Oc27znfEoSzHy8Gz3762PqeiKrmynyL\\\\\\\/fsLaQtx+UAT4VixoBcyiQel3efybJEFbOfPxvuNqP5Fxyc34hzxC2Od9ibX+errRw9UzoIt6tMHVuQ8AOKAT5COYMsAn1Ty2V\\\\\\\/XfzogPAQ8mlWKl9SH4OivxW9wGE2wJSR7ZCd44YNGs6bs9ggxEt3TXx\\\\\\\/q2g1d9fzovKJ8SF99HyARmeZMBZn0x0L4bvkrvSr\\\\\\\/D2AMJlvvzuSMzYbN6NtV4ztUNYq0xZq9BzM3zEmLERymuXv0B+BFy\\\\\\\/Pv\\\\\\\/8SzFfm\\\\\\\/eaiwvkqqH1lKyMtox+DEGo2ssBI+MTp\\\\\\\/MKN9tYsjr2ZeXYlbdFeo+VsrHnMrzELzYUhUTbdD2Tb6By9bfZnfLh6ci09ASfDO\\\\\\\/NEjLYlHPhvuOxDW9Aa0Uy5aar\\\\\\\/Kv7KspykKWIxVMPWLs+4HY1kCbMtMWB9VZkoYT5JwJ8esKKq0FET5k71AQ2Dog15hXGnxPGVeOpdRSv4+0ghrbp7ZSZOb3hmcV9Fs1cmOP+tr0EiyNLVpoWNEPqArppPxi6rl6g\\\\\\\/KouzTGk46Z0BKdtWSNsjp4AvGA8rKexkuf\\\\\\\/rBLFeSljs7KJAgU0eI8bxzzPji2QqCLK9aatdYjaXD\\\\\\\/KH2WpPBhuXGUxo2SIOZdvsUcWo08uEyuGdAD5AK5Wx2HKRsaYTZ1Zh+Wd9aORMQzvvKBHoV6vEE6nRl01+yxse9vloBId8Clf0BGt6Wi6LWii1ube\\\\\\\/JTRCJDSnc8YKWXsPoWPGFCmQDCWQrsPihhXxmXrhf2GTDbLKweDQSz6k4Pr8kbmI+xhb9MZmqIuxy8qIvNDNzigfzhzZE3fHg5i11AffKLL902erdSjATS25qBZvkY8++HIUrRFoy5jxDYizQTOdTahkzf5+5JeifZAootwPYPkojFx7kdN4bn3gzOVo4qX62xxUCPaFOBZRwc+8U9IccumDC6VGkwqMt4bTmWyrgzOMBhaXYiZKYi40WWlrJPusoLNk+Kq6ajbVUkNYYQDcBHV\\\\\\\/6uqGxME7VodL1y5X9cm5ZV\\\\\\\/qb3ImsgnGJIorn6imc7Nk10H7CAi1C\\\\\\\/bB7pc0dbMlpjlQ4xbIYkTiFseL1phviNeAlgGlmYKYpRWhs3fUpb+llcZ73P+o4OBFPIPQkhtgLSIj1Vtfvfpt+YLHLYn8weUJyHVlGrslb2sxsbPYAd+YVzEbz+pZ1nTj2puqSj\\\\\\\/NZ9LaYp5b6RGHeMwb4d6LX5inUBWCgf9z0Is3oZMQwxFGzO1pdmg6pnSk6n45Obb83NHRx28srKUj1vqCSrE+jrj+igoSlte0Qm5IhLd5AqRlhI85C4n4HZK6rSaEq9FdZOmYdD\\\\\\\/kgyatq2OPkWQ0h5oSy3RIcNJWpAkh5mnJnzAJRbR3IpZVD+xaCblnc\\\\\\\/prIYK9702\\\\\\\/+F0Xue6OL99OZuWSXRQbK3acxyc7aZY8z4AQi+KA1twUYshhukF77\\\\\\\/GUad4kG+h8nDa\\\\\\\/G7Wh+SgbfUI7DJsSJgiUU2+4WPobxlS+XYmsfu6yaPeWae6d+8QHZB1EXUI92oJyGmzhJuyMIcjOQAhRfx6iaWg1u0P8abTWkOh0iEb1LBUCs2tO8m6odRV9nmI\\\\\\\/hskrk7UySkzV+ioC9pdfG6V3NamwL4Mr0DWp9KCTOnXMS0xhd82CLrR8Iwn66n+6L00qCmd0vySfLqb9Erfy57tFbNh6Vir6ZgOcZW0dEox5RQotcenUfSn1TIBXC2nmGEPHaZxepKRzfEExI7ktl1uu6AEmi0ZGTOR4zGOy\\\\\\\/3kprPvdcqzyab+P5\\\\\\\/HRnvOziaHYQK8gJyrNn+EaIqSR8Iemo0iRTJUhLskr9bVO+sJew+YjjH8FZ0XFieHnPTXa00GVj1HG7jCjSUqfklNazcjIn8wzGLwhkTt2LIomSZ3Wwr4zomkLWoawdNHJPb1QIE+Ti37jN4jotMTl\\\\\\\/1AMJ+\\\\\\\/YFKYvd2Akbuj\\\\\\\/OseJNnNMAMaaT5ss9LULvoYnXkWPKmiqPyRI2k9ASelBk7kqtDyqiL0kIPXwDKU\\\\\\\/OgiqZrCBMAmSaxI\\\\\\\/Q\\\\\\\/BqmRUs6uzeY1L\\\\\\\/G0wdoYpm4ltJ8mOR8zOAqcKUq0VJd3C+RholSdrPStstPsSgaCkRgFWu7B1xR2KRFdxwLgLCuTQxOF1+5f\\\\\\\/M+u\\\\\\\/aNCEvnMJI2Jt2zDCUb+tT1SLjLNgUgLhZ0dGjWQiAhvBluxN+TsPkhsuHnLcBpAFWlOcGfVv8z5pOh1gUN70tY4Z0AhqQrh\\\\\\\/k4kPGfhL0+7MXyT2ubG6LApl\\\\\\\/IiPEYlXpeX7GLgmW94KyI7gE13KomWXva0N9fYKl5s28a\\\\\\\/\\\\\\\/yWwuoHUzYDcgeXn+Wo+PbhDJtaUR+dO0wt9tL1RClf36zmj4BeOk3Djn8mVKUHhGA7mwM+XSCTOxApXj70b01AA5o0+VOEdCIkwSCIjQzwE4j2Gz6WBWlrVaTzOu5796iy2+CYnnNzxzlgZ0Jt10ACgQ9QrwZDDnM\\\\\\\/sufPKlZJ49doK+BZbNeOCOpk1Fa1yPdbPpp37zM4Wga91wHf2yB+Kt34V4OyH93w7YqF5GgQfcUTNxR9+H\\\\\\\/fxgDVNriJxBZLQuUJnK0mF\\\\\\\/7V7sY5J5hdG617Bu8+k0G1ro27YoP+QJoT+5YjESOhuAGDkypbZO6KzPi2HkL3DFth3txrBymlRblwLDdN7X4NizhCbmSWYhD0Nauki7w6k\\\\\\\/R2YGHjD+8\\\\\\\/2PXvIakXM8O\\\\\\\/e1S9SWO\\\\\\\/5ZSLWh539tft17HmrNZXlZP3mSICo8o83xqMLSsKkhv1Jp7mbZQra5Fo4lNeWHO7j6gPFvH8A0KZp0AMjko2V6JsOg74DyH052a6qhfJ6JBYDssSgvJcTSdDUkZvZw48bcGVACU\\\\\\\/uPGJLAHeT84OTa6oN6foPdnvT2HpNW\\\\\\\/XLgI0GtyDV1Yd99qJTAVLekivjfhtOGL\\\\\\\/+nr\\\\\\\/NMvGD0u+YhkOrUNa49Re+gfnblG\\\\\\\/avxGBVUeuYgeK0mzcp96Nm5mknX5zzMPvaKYJwZeeAeapwNJQ\\\\\\\/RSeOurcXt69eBxppkVcri9VwpOKficOBAxF3gWs1WupXIBM55w0NVoBy2H7wUadhyGkoy7UmfoBIS59\\\\\\\/ospf7BuaksYnSMuYHXmJYQgFxKcaJ02+cujIcJ+qeklN8i+M2uBo\\\\\\\/FRgKnuUpMb5aw3Zt7HNNCI\\\\\\\/sOFbxkMR8A9w1Zm0K171Yijg9XO0+91Vuzy5Bf6VsazD34e4O0NoEBtHbpqyMgBTwDC1bnmKmrB6qMDld2LhaMiqGsnv7KJf3ktPl++2SnShazzANzYsgQRQj3lY1rXOmptug0c0feVsLbOSwfsokGpZ+hYXu63L\\\\\\\/HmOk43kyneb\\\\\\\/J+GT\\\\\\\/lQeeVUNPdUqagjpT54yYw5cYv1y3EQU+LzO9a7Uu4uB79X1Sfv7phVvocbdnzLrTkNRljJOaenpbI3ewMBhLtMXqxUXIuhBUr\\\\\\\/jrlP\\\\\\\/NeGLVd4uQCtvmO\\\\\\\/3XGPwf8yd9JG2hbTwNsKjKwJ8uqya+j+nUoQcaw+mP6gvpBRHcCpxgDe4vj9ONHNxusyjqfIfHiCCzIBUlWdhDosPiuRqmwyMx6GOLuOLh+j\\\\\\\/U286m\\\\\\\/TvVU7V5KPrpDbXIx4y0eb83sU7t4MvUNwI\\\\\\\/TEB8V\\\\\\\/uFVZ1TIcRMNVa8k6BwDoMMXDNaxCduuGNzR3UlRkCtnRfd1+Bp0eUQQMDco+1U+bT6MtCSk6fGgDOWpIW0tFU8HnPpPICc\\\\\\\/fucDyBPyFa+l+5+F7iA4rtfJ+ce6DMiwSJXCydPQOjwpi6ozVEaWlz6OpeiiGIoq4p55O7Cc7GowP7gjeUGsZbTulQClqHRecoj\\\\\\\/l7zW\\\\\\\/dQkCB7aZjTcPWTfP12nER1dWhjwTs96rsCr6Tb19HvNUEBt1k+8T40QVlHIXGkUkejPkVr6qNX4vrJEED5pU4Sr+v9dwFMom53toADPzzjGG+iDN\\\\\\\/Z4hlutqPY\\\\\\\/1wrigwBWEKnfHEV85fcpDpMPPBL+0gV\\\\\\\/9\\\\\\\/YnT9nvv7+kBs4Jg\\\\\\\/B+2Bt3STJc65ie4iZoKZ4Lp1K+H7kdFXTnsPpZU7hOkSpuwJX7MdkCFx2qjaBOLvw0iNMn\\\\\\\/CPRJkfAZKd2+PcjBFOBpNkeSUB1l6zziie7NFcIr07+RPG3\\\\\\\/nfPuxl1fi379HOErK0TfZWgSoYLYXPVIEKQHlTXE8\\\\\\\/DzgCRp7VdQst+9whANz97iZg58SRqXoMCAejVbH6Rgy52SpRdyMyxZIkUfMJjJn1BDTiHW7TYYGjKmxXTTNrNfLhfZ+rTPvLSYVlbAfHooM9OazHA71jyCPOUSm2XVSCv6CONwBBajJ4Yh5QRV7FqmP5JkCVFCL2yY07WtHJHasWvUxIdxie+TAisnOg8o5cIu32IKvbIxuKeHX8F\\\\\\\/1jkF8b5\\\\\\\/8+2ohm8gaLotIrQNtuCa9wnAX5TnHiHHJnCDbn1yDzpW\\\\\\\/vvDO7eEZ1fOtmyAuHSayoNCENUtgNiVCDPM0xYduoBo5aBtt18UFOTBWWs3bw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022qbI92zJjS\\\\\\\/3+0edsAgSJjQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002273e86562c6b176f0be1e4a8dc4d7a83235e0559ec14e63cf2cccbbd868d5d4cd\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022x2kDnlLp80QChdzLIT4CNhJyrj0XutGZbmXTH0sexq65o\\\\\\\/MlatuC81YVR+IuzVj0AxT0rOTBOQVabzBkR2cMbLpC3k0wTT0CEgDyq8YZ0og62VJLoaQDaduV6Xn+fDGDulxvrWuffMIQF91H0gVKwoCYJs78IgXGJdLFK\\\\\\\/\\\\\\\/bpxMNMGPOZrdzj+yfo\\\\\\\/DRVANB8yPXevDuC7SNrMNCaFmRksCW7UK\\\\\\\/NFlHolVEeL4qncJ7bXKKTJ1xHAFcu9EP\\\\\\\/9dvAh1y2gmUmYr6NnU5LQP4u4rNMqbKDMZE84VsEQRykhR8y3bLw1Ys+CzSZSW3+nwTMZ8Jbq++KkYfWiYUmxM4UanqH4QlRoJBmDZY9SmQqycMVn1Q\\\\\\\/6PR5jhmAIsohr\\\\\\\/1jVQ826E\\\\\\\/MKh5wUwbS1\\\\\\\/ej918QJOO3tfq6t+T3LnCt+j9x8hH+vZv46B21SlI8XrKDdqoZbS1KBEzXFHALT5f90yaKZt19EIP6x8dFkoWQ3fqNSK2XSaeQHCJE4XXvGRRKtPQV3CGE+2TVO6AmVabliPMRGoYwDnI8PEtxhR5psDc365SfBhLe9hyynzJpP7emch9CD9s7pf3ADTVekFcWUALM\\\\\\\/9aJMBKJiYoapIbxDDXseJj\\\\\\\/cBdUaFdGG5SQxx01tiT84R2QFRo\\\\\\\/FFMYksvsjwkQKGu0WLT57xbD4Up84R6Av18Zz5lJ37EqJqEuZ6M5uSUf7QVRRYL7\\\\\\\/ezgGjzW46993ZAjWIavs02SEj7xy7olSwerTM3+WtCawfSdfhnu4iB2UuDwbsJfipE9RRnGZyoUOcYXNfEfxPY7ztV8Y9G57KLG9VYDouz4e9CPRtJp3EosE+pYpDcnPlPs1LzNVQfWSp4PeiHB+AfIjNU9WB8W0Moyo\\\\\\\/fUG8p4x9av3uC\\\\\\\/mJjU+K7hyXs8FyB0UDwwhO3eHPWosDi+fFLEB0Y8O589RuDO0N3VmszjzfTqGtOwxfgIynuVGwryRSOtzG9qw8wRrC58XqEBlKSc3QPnJEzVsqH5bYnmzjnqcYMqXEt1QkE\\\\\\\/px1NL09eLaZiRzlcq6Lbj13+YMauM9Mhpru00ZtfGAA5pbBExgjqQlDjsSgXL1uj03Sh7Nqu\\\\\\\/9X\\\\\\\/IjAnTqKTYJdNTt33Wbnl9gdgLYLepbPCFil6BugIb762UEg1r16q0O7d3lP63boIgHMs9470X83mebGleGxbUYSfzxwPXXLPGJiH1ac7+LsHSbkPKjRiSh5tqRmVOp0e0gCZZoYQ3SdmNBb6vhG3cPtuP1R4AiQUa9abnxbeRy4Wze5kKm6pnca4v0cEUJR4Q0+Y9LvHwulx7U7vCjR+drCULL2JOaOyDq3YMaTj6A7CjBAXSmxMP9VFV66ONrV8+avr\\\\\\\/OXcF8cPOuYEJXB3EkAikOpttHIuIZl5eFET8Chpie0XSfeN6LNb4LhlpIusbpIXEZ6dmO\\\\\\\/lznYxIyJqvD2+nPo85kN4+ZoZyfXg4\\\\\\\/Wacxl4n1SGkmtnSasxIQYzK9hKR4WeRJuH0ZkrZQm1wb5McqTY++2PAIC5kz+TmM7SQA+RhfJPEpk34htFrAyF8OVy0HCr4GNf7D1P7uXkVJRYYkiKJK0fZwmDBFZ+CfSS05namX2DaGslRAujZ9nxQrtr8LJmbL5nhETbqOWVwE0JvRh4h1w\\\\\\\/0X54gzSXGKdqea+AptZZkAcOc5A7FDT8yO9sqT\\\\\\\/FmvOpClEtCBlQ4FGmmMM5QYGynPxix+fLuB1Kg5veNvhl9BQBcuwIEw5Zx+J7h6cpqSQxJ7UOk2lOJ93VdXFBE2p2MY\\\\\\\/zO0C7FZ4XFwS5rjhpq62YZMgSIJN2efol6n8j6ZA0fWXywOURu\\\\\\\/HcLe+Lgn6RyuWVI+47jAHhb85fUU1c0vKG02zi1LHXQXQCVihkcAQJb4YNdm3Tstp71xQhbuXRrfDOVTrfT3oiZ+sVyEsZ6Qxa0dELDcKRQmDYOQNDMG+HjV56F2ghMxITBUYvv2VaStdp3rJxCsgd\\\\\\\/GzLk5ngfLX6zWYKl6MroJCCQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022vpGTdIEOcMNsxU46z8ufHw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a85873a971a712e71b3f2cff4b4bcc9a148bcb0f148a90e1e700e0482b0b7002\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-493769 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genome-report">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/8/2405" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-m9gxnTRPtpISRXmke3SVuzP96UAPe5rEGgVryiIHj5c" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/8/2405" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-z9WnxKoLfpJYdux8oXtpRNmbkThfGfGKwYzJ287WJxg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/8/2395" title="High Quality Draft Genome of Arogyapacha (Trichopus zeylanicus), an Important Medicinal Plant Endemic to Western Ghats of India" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/8/2415" title="Forward Genetic Screen for Caenorhabditis elegans Mutants with a Shortened Locomotor Healthspan" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493769" id="node493769" data-pisa="ggg;9/8/2405" data-pisa-master="ggg;g3.119.400047" data-apath="/ggg/9/8/2405.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Population Genome Sequencing of the Scab Fungal Species <em>Venturia inaequalis</em>, <em>Venturia pirina</em>, <em>Venturia aucupariae</em> and <em>Venturia asperata</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Bruno Le Cam</span>, <span class="highwire-citation-author" data-delta="1">Dan Sargent</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-5695-4557" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jérôme Gouzy</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-7124-3454" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Joëlle Amselem</span>, <span class="highwire-citation-author" data-delta="4">Marie-Noëlle Bellanger</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-2328-5769" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Olivier Bouchez</span>, <span class="highwire-citation-author" data-delta="6">Spencer Brown</span>, <span class="highwire-citation-author" data-delta="7">Valérie Caffier</span>, <span class="highwire-citation-author" data-delta="8">Marie De Gracia</span>, <span class="highwire-citation-author" data-delta="9">Robert Debuchy</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0003-0960-0312" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ludovic Duvaux</span>, <span class="highwire-citation-author" data-delta="11">Thibaut Payen</span>, <span class="highwire-citation-author" data-delta="12">Mélanie Sannier</span>, <span class="highwire-citation-author" data-delta="13">Jason Shiller</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="14"><a href="http://orcid.org/0000-0001-8152-6642" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jérôme Collemare</span> and <span class="highwire-citation-author" data-delta="15">Christophe Lemaire</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2405-2414;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400047 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493769" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bruno Le Cam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bruno%2BLe%2BCam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Le%20Cam%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABruno%2BLe%2BCam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bruno.le-cam@inra.fr" class="" data-icon-position="" data-hide-link-title="0">bruno.le-cam@inra.fr</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dan Sargent </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Research and Innovation Centre, Fondazione Edmund Mach, via Mach 1, 38010, San Michele All’adige, TN, Italy,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dan%2BSargent%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sargent%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADan%2BSargent%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jérôme Gouzy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratoire Interactions Plantes Micro-organismes [LIPM], Université de Toulouse, INRA, CNRS, Castanet-Tolosan, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%A9r%C3%B4me%2BGouzy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gouzy%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJ%25C3%25A9r%25C3%25B4me%2BGouzy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5695-4557" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jérôme Gouzy</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Joëlle Amselem </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Unité de Recherche Génomique Informatique [URGI], INRA, Université Paris-Saclay, 78026, Versailles, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jo%C3%ABlle%2BAmselem%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Amselem%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJo%25C3%25ABlle%2BAmselem%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7124-3454" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Joëlle Amselem</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Marie-Noëlle Bellanger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie-No%C3%ABlle%2BBellanger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bellanger%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarie-No%25C3%25ABlle%2BBellanger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Olivier Bouchez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>US1426, INRA, GeT-PlaGe, Genotoul, Castanet-Tolosan, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Olivier%2BBouchez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bouchez%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOlivier%2BBouchez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2328-5769" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Olivier Bouchez</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Spencer Brown </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>UMR 8298, Institute for Integrative Biology of the Cell (I2BC), CEA, CNRS,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Spencer%2BBrown%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brown%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASpencer%2BBrown%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Valérie Caffier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Val%C3%A9rie%2BCaffier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Caffier%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVal%25C3%25A9rie%2BCaffier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Marie De Gracia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie%2BDe%2BGracia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=De%20Gracia%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarie%2BDe%2BGracia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Robert Debuchy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>UMR 8621, Institut de Génétique et Microbiologie CNRS, Université Paris-Sud, 91405 Orsay, France, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BDebuchy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Debuchy%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BDebuchy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Ludovic Duvaux </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ludovic%2BDuvaux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Duvaux%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALudovic%2BDuvaux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0960-0312" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ludovic Duvaux</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Thibaut Payen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thibaut%2BPayen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Payen%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThibaut%2BPayen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Mélanie Sannier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M%C3%A9lanie%2BSannier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sannier%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM%25C3%25A9lanie%2BSannier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Jason Shiller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BShiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shiller%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJason%2BShiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Jérôme Collemare </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France,</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>Westerdijk Fungal Biodiversity Institute, 3584 CT, Utrecht, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%A9r%C3%B4me%2BCollemare%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Collemare%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJ%25C3%25A9r%25C3%25B4me%2BCollemare%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8152-6642" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jérôme Collemare</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Christophe Lemaire </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christophe%2BLemaire%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lemaire%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristophe%2BLemaire%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/8/2405" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:493769" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:493769/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/8/2405.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:493769" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:493769/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/8/2405.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:493769" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:493769/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/8/2405.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:493769" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:493769/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The <em>Venturia</em> genus comprises fungal species that are pathogens on <em>Rosaceae</em> host plants, including <em>V. inaequalis</em> and <em>V. asperata</em> on apple, <em>V. aucupariae</em> on sorbus and <em>V. pirina</em> on pear. Although the genetic structure of <em>V. inaequalis</em> populations has been investigated in detail, genomic features underlying these subdivisions remain poorly understood. Here, we report whole genome sequencing of 87 <em>Venturia</em> strains that represent each species and each population within <em>V. inaequalis</em>. We present a PacBio genome assembly for the <em>V. inaequalis</em> EU-B04 reference isolate. The size of selected genomes was determined by flow cytometry, and varied from 45 to 93 Mb. Genome assemblies of <em>V. inaequalis</em> and <em>V. aucupariae</em> contain a high content of transposable elements (TEs), most of which belong to the Gypsy or Copia LTR superfamilies and have been inactivated by Repeat-Induced Point mutations. The reference assembly of <em>V. inaequalis</em> presents a mosaic structure of GC-equilibrated regions that mainly contain predicted genes and AT-rich regions, mainly composed of TEs. Six pairs of strains were identified as clones. Single-Nucleotide Polymorphism (SNP) analysis between these clones revealed a high number of SNPs that are mostly located in AT-rich regions due to misalignments and allowed determining a false discovery rate. The availability of these genome sequences is expected to stimulate genetics and population genomics research of <em>Venturia</em> pathogens. Especially, it will help understanding the evolutionary history of <em>Venturia</em> species that are pathogenic on different hosts, a history that has probably been substantially influenced by TEs.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/venturia" class="hw-term hw-article-keyword hw-article-keyword-venturia" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Venturia</em></a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/fusicladium" class="hw-term hw-article-keyword hw-article-keyword-fusicladium" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Fusicladium</em></a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/formae-specialis" class="hw-term hw-article-keyword hw-article-keyword-formae-specialis" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">formae specialis</em></a></li><li class="kwd"><a href="/keyword/transposable-elements" class="hw-term hw-article-keyword hw-article-keyword-transposable-elements" rel="nofollow">transposable elements</a></li><li class="kwd"><a href="/keyword/effectors" class="hw-term hw-article-keyword hw-article-keyword-effectors" rel="nofollow">effectors</a></li><li class="kwd"><a href="/keyword/apple-scab" class="hw-term hw-article-keyword hw-article-keyword-apple-scab" rel="nofollow">apple scab</a></li><li class="kwd"><a href="/keyword/apple" class="hw-term hw-article-keyword hw-article-keyword-apple" rel="nofollow">apple</a></li><li class="kwd"><a href="/keyword/pear" class="hw-term hw-article-keyword hw-article-keyword-pear" rel="nofollow">pear</a></li></ul><p id="p-3">Fungal species from the <em>Venturia</em> Sacc. genus (anamorph is <em>Fusicladium</em> Bonord.; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Beck <em>et al.</em> 2005</a>) are pathogenic on important <em>Rosaceae</em> host crops including apple and pear (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Sivanesan, 1977</a>). They are responsible for scab diseases which dramatically reduce the economic value of fruits by causing black lesions on the skin (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bowen <em>et al.</em> 2011</a>). Taxonomically, scab fungi belong to the <em>Dothideomycetes</em> class and to the order <em>Venturiales</em> within the <em>Venturiaceae</em> family (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Zhang <em>et al.</em> 2011</a>).</p><p id="p-4">Scab caused by <em>Venturia inaequalis</em> Cooke (Wint.) is the main fungal disease on apple, control of which may require over 20 chemical treatments each year. <em>V. inaequalis</em> has been extensively studied as a model for pathogens on woody hosts, particularly because it is amenable to <em>in vitro</em> crossing (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bénaouf and Parisi, 2000</a>). <em>V. inaequalis</em> has been described on several genera of Rosaceous including <em>Malus</em>, <em>Pyracantha</em>, <em>Eriobotrya</em> and <em>Sorbus</em> (<a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Sivanesan 1977</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Jones and Aldwinckle 1990</a>). In some cases, host specificity occurs; for instance strains from <em>Pyracantha</em> are not able to infect <em>Malus</em> and vice-versa, which is referred to as <em>formae specialis</em> f. sp <em>pyracantha</em> and f.sp <em>pomi</em>, respectively (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Le Cam <em>et al.</em> 2002</a>). Worldwide sampling of <em>V. inaequalis</em> strains has identified divergent populations, mostly reflecting their hosts: wild apple (<em>Malus sieversii</em>, <em>M. orientalis</em>, <em>M sylvestris</em>, <em>M. floribunda</em>) and domesticated apple (<em>Malus x domestica</em>) trees (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Gladieux <em>et al.</em> 2008</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Gladieux <em>et al.</em> 2010a</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Gladieux <em>et al.</em> 2010b</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Leroy <em>et al.</em> 2013</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Ebrahimi <em>et al.</em> 2016</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Leroy <em>et al.</em> 2016</a>). Certain populations have experienced a secondary contact with subsequent gene flow, resulting in the introduction from a wild population into the agricultural compartment of strains that are virulent on resistant cultivars carrying the <em>Rvi6</em> resistance gene (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Guérin <em>et al.</em> 2007</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Lemaire <em>et al.</em> 2016</a>). On apple, scab can also be caused by another <em>Venturia</em> sp., <em>V. asperata</em> initially described as a saprophyte (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Samuels and Sivanesan, 1975</a>), and recently reported as a pathogen on apple trees (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Caffier <em>et al.</em> 2012</a>). Other <em>Venturia</em> species cause scab damages on their respective <em>Rosaceae</em> hosts, <em>V. pirina</em> (Aderh.) on European pear (<em>Pyrus communis</em>), <em>V. nashicola</em> on Asian pear (<em>Pyrus pyrifolia</em>), and <em>V. carpophila</em> on apricot (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Crous <em>et al.</em> 2007</a>; <a id="xref-ref-63-3" class="xref-bibr" href="#ref-63">Sivanesan 1977</a>). Other scab fungi are pathogenic on various tree hosts, including <em>Fusicladium effusum</em> on pecan, <em>F. oleaginum</em> on olive, <em>V. populina</em> on poplar, <em>V. saliciperda</em> on willow and <em>V. fraxini</em> on ash, among many others (<a id="xref-ref-63-4" class="xref-bibr" href="#ref-63">Sivanesan 1977</a>; <a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Zhang <em>et al.</em> 2011</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Young <em>et al.</em> 2018</a>). Despite being economically important pathogens, the genetic basis of the interactions between scab fungi and their respective hosts remains poorly understood (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Singh <em>et al.</em> 2005</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Steiner and Oerke 2007</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Kucheryava <em>et al.</em> 2008</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bowen <em>et al.</em> 2009</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Mesarich <em>et al.</em> 2012</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Cooke <em>et al.</em> 2014</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Shiller <em>et al.</em> 2015</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Deng <em>et al.</em> 2017</a>).</p><p id="p-5">Genome sequences are already available for <em>V. inaequalis f. sp. pomi</em>, <em>V. inaequalis f. sp. eriobotrya</em>, <em>V. pirina</em> and <em>F. effusum</em>, and they revealed that <em>V. inaequalis</em> strains tend to have a higher repeat content compared to other species, with up to 47% of their genome sequence covered by repetitive elements (<a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Shiller <em>et al.</em> 2015</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bock <em>et al.</em> 2016</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Deng <em>et al.</em> 2017</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Passey <em>et al.</em> 2018</a>). These studies highlighted in <em>Venturia</em> genomes the expansion of putative effector gene families, genes that are known to determine plant-fungus interactions (<a id="xref-ref-60-3" class="xref-bibr" href="#ref-60">Shiller <em>et al.</em> 2015</a>; <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Deng <em>et al.</em> 2017</a>). The high repeat content and the employed Illumina sequencing strategy resulted in assemblies with between 1,012 and 1,680 scaffolds (<a id="xref-ref-60-4" class="xref-bibr" href="#ref-60">Shiller <em>et al.</em> 2015</a>; <a id="xref-ref-16-4" class="xref-bibr" href="#ref-16">Deng <em>et al.</em> 2017</a>) and up to 231 contigs using the PacBio technology (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Passey <em>et al.</em> 2018</a>). Genetic studies need higher quality genome assemblies and genomic population studies require re-sequencing of many strains from diverse populations.</p><p id="p-6">Here, we report the PacBio genome assembly of the <em>V. inaequalis</em> EU-B04 reference strain and re-sequencing of 79 strains from two <em>formae specialis</em> and from diverse populations within <em>V. inaequalis f. sp. pomi</em>. We also present re-sequencing of <em>V. pirina</em> strains and whole genome sequencing of <em>V. aucupariae</em> and <em>V. asperata</em>. Availability of these genome sequences will greatly facilitate genetic and genomic population studies to better understand the evolution of <em>Venturia</em> pathogens and their interactions with different hosts.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Origin of Venturia strains</h3><p id="p-7">All 87 <em>Venturia</em> strains used in this study are described in Table S1. Forty strains of <em>V. inaequalis</em> sampled in Kazakhstan on wild apple <em>M. sieversii</em> were used, of which half belong to the wild CAM (Central Asian Mountains) population and the other half to the domestic CAP (Central Asian Plains) population as described in <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Gladieux <em>et al.</em> (2010b)</a>. Other <em>V. inaequalis</em> strains were isolated from <em>M. x domestica</em> cultivars that do not carry the <em>Rvi6</em> resistance gene (6 domEU and 2 domASIA), from <em>M. x domestica</em> cultivars carrying the <em>Rvi6</em> resistance gene (5 dom<em>Rvi6</em>), from ornamental <em>M. floribunda</em> (4 FLO), from wild <em>M. sylvestris</em> (6 SYL) and <em>M. orientalis</em> (6 ORI), from <em>Pyracantha</em> (5 PYR), from <em>E. japonica</em> (5 LOQ) and from <em>Sorbus aucuparia</em> (1). In addition, strains of <em>V. pirina</em> (3), <em>V. aucupariae</em> (1) and <em>V. asperata</em> (3) were isolated from <em>Pyrus communis</em> (pear), <em>S. aucuparia</em> and <em>M. x domestica</em>, respectively (Table S1).</p><p id="p-8">All strains were grown at 18° on cellophane membranes displayed on PDA-YE (Potato-Dextrose Agar (BD Difco) supplemented with 0.3% Yeast Extract (Sigma-Aldrich)) plates with a 12h/12h light/night regime. They are stored at -20° on dried cellophane membranes in the <em>Venturia</em> collection of the Institut de Recherche en Horticulture et Semences (IRHS, Beaucouzé, France).</p></div><div id="sec-3" class="subsection"><h3>Genomic DNA extraction, library construction and DNA sequencing</h3><p id="p-9">High molecular weight genomic DNA from <em>V. inaequalis</em> EU-B04 strain was extracted from mycelium grown on PDA-YE covered with cellophane membrane using the method described in <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cheeseman <em>et al.</em> (2014)</a> (extraction with CTAB followed by purification on an isopycnic gradient), except that ethidium bromide was used instead of DAPI for the purification of genomic DNA on a cesium chloride gradient. Ethidum bromide was preferred because it resulted in a single band of genomic DNA, while DAPI resulted in the dispersion of genomic DNA in several bands of different AT contents.</p><p id="p-10">Genomic DNA quantity and quality were validated at the GeT-PlaGe core facility (Toulouse, France) using Qubit (Thermofisher), Nanodrop (Thermofischer), and Fragment Analyzer (Advanced Analytical) before purification on 0.5X AMPure PB beads and shearing at 40 kb using the Megaruptor system (Diagenode). Libraries were prepared according to the “20-kb Template Preparation Using BluePippin (Sage Science) Size-Selection System (15-kb size cutoff)” PACBIO protocol using 10 µg of sheared genomic DNA and the SMRTBell template Prep Kit. DNA fragments from 10 kb to 90 kb were selected. The sequencing run was performed on a PacBio RSII sequencer with the OneCellPerWell Protocol (OCPW) on C4 P6 chemistry and 360 min movies. Another PacBio sequencing was performed at the Cold Spring Harbor core genomics facility (CSHL, NY, USA); genomic DNA was sheared and purified, and a long-insert library was prepared following the protocol described in <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Quail <em>et al.</em> (2012)</a>. The library was sequenced on four SMRT cells using the C3 P6 chemistry using 2x 60 min movies.</p><p id="p-11">Genomic DNA of all other <em>Venturia</em> strains was extracted from mycelium grown on PDA-YE plates using Möller’s protocol (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Möller <em>et al.</em> 1992</a>). Libraries for CAM and CAP strains were prepared using SPRIworks- Fragment Library System I (Beckman Coulter) and sequenced at GATC Biotech (Konstanz, Germany) on HiSeq 2000 using TruSeq Multiplex Sequencing Primer and TruSeq PE (2x50bp) (Illumina, San Diego, CA). Libraries for the remaining strains were prepared using the TruSeq Nano kit (Illumina, San Diego, CA) and sequenced at the GeT-PlaGe core facility (Toulouse, France) using SBS v3 (2x100 pb) or v4 (2x125 pb) reagent kits and using HiSeq 2000 or HiSeq 2500. For library construction, genomic DNAs were fragmented by sonication and sequencing adaptors were ligated. Libraries were amplified with eight PCR cycles; their quality was assessed using a Bioanalyzer (Agilent) and quantity measured by quantitative PCR using the Kapa Library Quantification Kit (KapaBiosystems).</p></div><div id="sec-4" class="subsection"><h3>Genome assembly</h3><p id="p-12">For the PacBio genome, the PBcR wgs8.3rc2 assembly pipeline (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Berlin <em>et al.</em> 2015</a>) was used to perform read correction and assemble the corrected reads. The high sequencing depth and quality of raw data allowed modifying several parameters to increase stringency: i) only raw reads longer than 5 kb were considered; ii) 50x of corrected reads were collected (assembleCoverage = 50); iii) the runCA part of the pipeline was run with a minimum overlap length of 1 kb (ovlMinLen = 1000); and iv) the unitigger bogart was tuned to consider overlaps with a very low error rate only (utgErrorRate = 0.0075 and utgMergeErrorRate = 0.04). Spurious contigs that were fully included in larger contigs were removed using a script based on ncbi-megablast to perform the all-<em>vs</em>-all analysis, combined with additional filters (minimum identity of 93%, minimum overlap of 500 nt and maximum overhang length of 500 nt). Bacterial contaminant contigs were identified using ncbi-blastx search against the NR database (downloaded in May 2016) and manual inspection. A total of 46 contigs were removed (maximum length of 113 kb) based on best hits with bacterial proteins confirmed with a GC percent analysis. Finally, the genome was polished with quiver (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Chin <em>et al.</em> 2013</a>) using stringent alignment cutoffs for pbalign (–minLength 3000–maxHits 1), ensuring a mapping as specific as possible. Assemblies of other genomes were performed by two algorithms: SOAPdeNovo2 (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Zerbino and Birney 2008</a>) and Velvet (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Luo <em>et al.</em> 2012</a>) adapted for <em>de novo</em> genome assembly using next-generation sequencing (NGS) short reads.</p><p id="p-13">Genome statistics were determined using Quast 2.3 (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Gurevich <em>et al.</em> 2013</a>). The GC content and distribution in each genome was analyzed using OcculterCut (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Testa <em>et al.</em> 2016</a>). Completeness of genome assemblies was assessed using BUSCO v1.1b1 (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Simão <em>et al.</em> 2015</a>) and the fungal BUSCO profile (busco.ezlab.org/v1/files/fungi_buscos.tar.gz). Fungal BUSCO genes are expected to be core genes, <em>i.e.</em>, they are present in most fungal species. Thus, the absence of BUSCO genes in a genome assembly is indicative of an assembly that is likely not complete. The program was run in “genome assembly” mode using the metaparameters of <em>Fusarium graminearum</em> for AUGUSTUS gene prediction.</p></div><div id="sec-5" class="subsection"><h3>SNP discovery, false discovery analysis and population structure of Venturia spp.</h3><p id="p-14">For all strains, genome sequencing reads were cleaned using Trimmomatic 0.36 (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bolger <em>et al.</em> 2014</a>) before mapping on the EU-B04 PacBio reference genome using BWA-MEM (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Li and Durbin 2009</a>). For species other than <em>V. inaequalis</em>, read mapping was refined with Stampy 1.0.29 (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Lunter and Goodson 2011</a>) using the parameter “–bamkeepgoodreads” and the parameter “–substitutionrate” was set to 0.15, 0.075, and 0.07 for <em>V. asperata</em>, <em>V. pirina</em> and <em>V. aucupariae</em>, respectively. Multisample SNP discovery was performed using the Unified Genotyper module of GATK 3.6 (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">McKenna <em>et al.</em> 2010</a>), applying indel realignment and duplicate removal according to GATK Best Practices recommendations (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">DePristo <em>et al.</em> 2011</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Van der Auwera <em>et al.</em> 2013</a>). Instead of applying any posterior quality score recalibration and standard hard filtering, we performed a SNP false discovery rate (FDR) analysis making use of clones present in our dataset (Table S1). The FDR was estimated by simply counting the number of SNPs for each pair of clones. All steps but the Trimmomatic, Stampy and FDR analysis were integrated in a pipeline set using the TOGGLE toolbox (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Monat <em>et al.</em> 2015</a>). In order to assess the discrimination power of our resulting true positive SNP dataset, we computed a PCA of all the non-clonal <em>Venturia inaequalis</em> strains using PLINK v1.90b3.44 (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Purcell <em>et al.</em> 2007</a>).</p></div><div id="sec-6" class="subsection"><h3>RNA extraction and sequencing</h3><p id="p-15"><em>V. inaequalis</em> EU-B04, <em>V. pirina</em> VP36 and <em>V. asperata</em> 2349 strains were grown on seven different agar media: PDA-YE, apple medium (10% organic apple sauce in water), and Vogel media (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Vogel, 1956</a>) without carbon or nitrogen source, or supplemented with 1% glucose or 1% saccharose (with and without 80 µM zebularin). Total RNAs were extracted using the kit Nucleospin RNA Plant (Macherey-Nagel, Düren, Germany), according to the manufacturer’s recommendations. Total RNAs from complex media and from defined media were combined in two respective pools. Paired-end 2x150 bp libraries were prepared using the Illumina TruSeq Stranded mRNA kit (Illumina, San Diego, CA) and sequenced at the GeT-PlaGe core facility (Toulouse, France) using a paired-end read length of 2x150 pb with the Illumina HiSeq3000 sequencing kits and using HiSeq 3000. Libraries were prepared according to Illumina’s protocols on a Tecan EVO200 liquid handler. Messenger RNAs were selected using poly-T beads, fragmented to generate double stranded cDNA; and then sequencing adaptors were ligated. Libraries were amplified with ten PCR cycles. Their quality was assessed using a Fragment Analyzer (Advanced Analytical) and quantity measured by quantitative PCR using the Kapa Library Quantification Kit (KapaBiosystems).</p></div><div id="sec-7" class="subsection"><h3>Flow cytometry</h3><p id="p-16">Analyses were performed according to <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Marie and Brown (1993)</a>. About 1 cm<sup>2</sup> of mycelium grown on PDA-YE plate was scraped and resuspended in 500 µL of cold nuclear buffer (45 mM MgCl<sub>2</sub>, 30 mM sodium citrate, 20 mM MOPS pH 7, 0.02% Triton X-100, 1% (w/v) cold polyvinyl pyrrolidone, 5 mM sodium metabisulfite, 0.05 mg/mL RNase (Roche, Basel, Switzerland)). A ligule of <em>Arabidopsis thaliana</em> cv. Columbia-0 was added as an internal standard. Samples were cut with a razor blade, filtered with 30 µm CellTrics paper (Sysmex Partec, Kobe, Japan) and stained with 50 µg/mL propidium iodide (Fluka). After 5 min incubation at 4°, cytometric analysis was performed on 20,000 nuclei using a CyFlo3 cytometer (Sysmex Partec, Kobe, Japan). Intensity ratio between 1C fungal nuclei and 2C (= 0.33 pg, corresponding to 326 Mbp) <em>A. thaliana</em> nuclei was determined to measure the size of fungal genomes (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Dolezel <em>et al.</em> 2003</a>).</p></div><div id="sec-8" class="subsection"><h3>Gene and transposable element prediction and annotation</h3><p id="p-17">Genes of the EU-B04 reference genome were predicted using the Braker1 pipeline (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Hoff <em>et al.</em> 2016</a>). RNA-alignments to the EU-B04 genome from all the <em>in vitro</em> conditions described above were used to provide extrinsic hints for gene prediction in this pipeline. Genes in all other genomes were predicted using AUGUSTUS version 3.2.3 (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Stanke <em>et al.</em> 2008</a>) utilizing the species training parameter files created in the Braker1 pipeline above.</p><p id="p-18">Secreted proteins in the predicted proteome were annotated with SignalP version 4.1 (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Petersen <em>et al.</em> 2011</a>) using default parameters. Small-secreted proteins (SSPs) were identified as secreted proteins with a maximum of 300 amino acids, containing no Glycosylphosphatidylinositol (GPI) anchor motif according to Big-PI (default parameters; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Eisenhaber <em>et al.</em> 2004</a>) and PredGPI (False-Positive rate lower than 1e-3; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Pierleoni <em>et al.</em> 2008</a>), and no transmembrane domain in the mature form according to TMHMM (default parameters; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Krogh <em>et al.</em> 2001</a>).</p><p id="p-19">The REPET pipeline (urgi.versailles.inra.fr/Tools/REPET) was used for the detection, classification (TEdenovo; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Flutre <em>et al.</em> 2011</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hoede <em>et al.</em> 2014</a>) and annotation (TEannot; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Quesneville <em>et al.</em> 2005</a>) of transposable elements (TEs). The TEdenovo pipeline was used to search for repeats in <em>V. inaequalis</em> EU-B04 genome sequence. The first step used Blaster with the following parameters: identity &gt; 90%, HSP (high-scoring segment pairs) length &gt;100 bp and &lt;20 kb, e-value ≤ 1e-300. HSPs were clustered using three different methods: Piler (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Edgar and Myers, 2005</a>), Grouper (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Quesneville <em>et al.</em> 2003</a>) and Recon (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Bao and Eddy, 2002</a>). The MAP software (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Huang, 1994</a>) was used to make multiple alignments of the 20 longest members of each cluster containing at least 3 members and build a consensus. Consensus sequences were then classified on the basis of their structure and similarities relative to Repbase Update (v17.11) (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Jurka <em>et al.</em> 2005</a>) and PFAM domain library v2 (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Finn <em>et al.</em> 2014</a>), before redundancy removal with Blaster and Matcher as performed in the TEdenovo pipeline. Consensus sequences with no known structure or similarity were classified as “unknown”.</p></div><div id="sec-9" class="subsection"><h3>Phylogenetic species tree</h3><p id="p-20">All strains of <em>V. pirina</em>, <em>V. asperata</em> and <em>V. aucupariae</em>, and one isolate from each population of <em>V. inaequalis</em> were used to build a phylogenetic tree. The <em>Dothideomycetes Zymoseptoria tritici</em> IPO323 and <em>Alternaria alternata</em> SRC1lrK2f (genome.jgi.doe.gov) were used as outgroups. The DNA sequence of β-tubulin <em>TUB1</em> (KOG1375), eukaryotic initiation factor 3, subunit b <em>eIF3b</em> (KOG2314), phosphoadenylphosphosulfate reductase <em>PAP</em> (KOG0189) and ribosomal protein S9 <em>RPS9</em> (KOG1753) genes were retrieved in all genomes and each were aligned using MUSCLE (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Edgar, 2004</a>). Regions with low quality alignment were removed using Gblocks (default parameters; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Castresana, 2000</a>) prior to concatenation of the four alignments. A maximum-likelihood tree was built with MEGA6 (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Tamura <em>et al.</em> 2011</a>) using the best-fit Tamura-Nei substitution model with 5 gamma distribution categories and 500 bootstrap replicates.</p></div><div id="sec-10" class="subsection"><h3>Data availability</h3><p id="p-21">All genome sequencing data generated in this study are accessible (assembly fasta files) in GenBank under the BioProject number PRJNA407103. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.7346714">https://doi.org/10.25387/g3.7346714</a>.</p></div></div><div class="section results-discussion" id="sec-11"><h2 class="">Results and Discussion</h2><div id="sec-12" class="subsection"><h3>Phylogeny of Venturia species</h3><p id="p-22">Phylogenetic relationships between the different <em>Venturia</em> species were determined using the <em>Dothideomycete</em> species <em>Zymoseptoria tritici</em> and <em>Alternaria alternata</em> as outgroup to the <em>Venturiales</em> order (<a id="xref-ref-73-3" class="xref-bibr" href="#ref-73">Zhang <em>et al.</em> 2011</a>; <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Bock <em>et al.</em> 2016</a>). The maximum likelihood tree shows that <em>V. asperata</em> is basal to the other <em>Venturia</em> species (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). <em>V. aucupariae</em> has diverged more recently from <em>V. inaequalis</em> than <em>V. pirina</em>. Phylogenetic relationships among <em>V. inaequalis</em> strains from different populations cannot be accurately resolved, probably because of gene flow between populations (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Gladieux <em>et al.</em> 2010b</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Lemaire <em>et al.</em> 2016</a>; <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Leroy <em>et al.</em> 2016</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2405/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Phylogeny of Venturia species and genome characteristics. A) A four-gene maximum-likelihood phylogenetic tree was built using the Dothideomycete species Alternaria alternata and Zymoseptoria tritici as outgroup. Numbers on the branches indicate bootstrap values. The name of the isolates indicates the species (Vina: V. inaequalis; Vauc: V. aucupariae; Vpir: V. pirina; Vasp: V. asperata), name in the collection and, for V. inaequalis isolates, the population of origin (see Table S1). B) Assembly size is shown for isolates used in the phylogenetic analysis. The size of the assembly covered by transposable elements was determined using the REPET pipeline (urgi.versailles.inra.fr/Tools/REPET). Red lines indicate actual genome sizes as determined by flow cytometry. Stars indicate that the genome size value is the average of other isolates from a given population. C) The GC composition in each genome was determined using OcculterCut (Testa et al. 2016). D) Gene models were predicted using Braker1 (Hoff et al. 2016). Small-secreted proteins (SSPs) contain a predicted leader signal peptide, do not contain a transmembrane domain or GPI-anchor signal, and are shorter than 300 amino acids. Numbers on the right indicate the fraction of the predicted proteome that is represented by SSPs." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1754084616" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Phylogeny of &lt;em&gt;Venturia&lt;/em&gt; species and genome characteristics. A) A four-gene maximum-likelihood phylogenetic tree was built using the &lt;em&gt;Dothideomycete&lt;/em&gt; species &lt;em&gt;Alternaria alternata&lt;/em&gt; and &lt;em&gt;Zymoseptoria tritici&lt;/em&gt; as outgroup. Numbers on the branches indicate bootstrap values. The name of the isolates indicates the species (Vina: &lt;em&gt;V. inaequalis&lt;/em&gt;; Vauc: &lt;em&gt;V. aucupariae&lt;/em&gt;; Vpir: &lt;em&gt;V. pirina&lt;/em&gt;; Vasp: &lt;em&gt;V. asperata&lt;/em&gt;), name in the collection and, for &lt;em&gt;V. inaequalis&lt;/em&gt; isolates, the population of origin (see Table S1). B) Assembly size is shown for isolates used in the phylogenetic analysis. The size of the assembly covered by transposable elements was determined using the REPET pipeline (urgi.versailles.inra.fr/Tools/REPET). Red lines indicate actual genome sizes as determined by flow cytometry. Stars indicate that the genome size value is the average of other isolates from a given population. C) The GC composition in each genome was determined using OcculterCut (Testa &lt;em&gt;et al.&lt;/em&gt; 2016). D) Gene models were predicted using Braker1 (Hoff &lt;em&gt;et al.&lt;/em&gt; 2016). Small-secreted proteins (SSPs) contain a predicted leader signal peptide, do not contain a transmembrane domain or GPI-anchor signal, and are shorter than 300 amino acids. Numbers on the right indicate the fraction of the predicted proteome that is represented by SSPs.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2405/F1.medium.gif" width="440" height="230"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/8/2405/F1.medium.gif" width="440" height="230"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2405/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2405/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493942" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-23" class="first-child">Phylogeny of <em>Venturia</em> species and genome characteristics. A) A four-gene maximum-likelihood phylogenetic tree was built using the <em>Dothideomycete</em> species <em>Alternaria alternata</em> and <em>Zymoseptoria tritici</em> as outgroup. Numbers on the branches indicate bootstrap values. The name of the isolates indicates the species (Vina: <em>V. inaequalis</em>; Vauc: <em>V. aucupariae</em>; Vpir: <em>V. pirina</em>; Vasp: <em>V. asperata</em>), name in the collection and, for <em>V. inaequalis</em> isolates, the population of origin (see Table S1). B) Assembly size is shown for isolates used in the phylogenetic analysis. The size of the assembly covered by transposable elements was determined using the REPET pipeline (urgi.versailles.inra.fr/Tools/REPET). Red lines indicate actual genome sizes as determined by flow cytometry. Stars indicate that the genome size value is the average of other isolates from a given population. C) The GC composition in each genome was determined using OcculterCut (<a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Testa <em>et al.</em> 2016</a>). D) Gene models were predicted using Braker1 (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Hoff <em>et al.</em> 2016</a>). Small-secreted proteins (SSPs) contain a predicted leader signal peptide, do not contain a transmembrane domain or GPI-anchor signal, and are shorter than 300 amino acids. Numbers on the right indicate the fraction of the predicted proteome that is represented by SSPs.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Genome assemblies</h3><p id="p-24">The assembly size of the different <em>V. inaequalis</em> strains varies greatly from 30 to 73 Mb, with a median of 7,435 scaffolds (from 66 to 31,301), an N50 median of 55.7 kb and an estimated sequencing coverage ranging from 8.8x to 150x (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a> and Table S1), consistent with previous sequencing of <em>V. inaequalis</em> strains (<a id="xref-ref-60-5" class="xref-bibr" href="#ref-60">Shiller <em>et al.</em> 2015</a>; <a id="xref-ref-16-5" class="xref-bibr" href="#ref-16">Deng <em>et al.</em> 2017</a>; <a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Passey <em>et al.</em> 2018</a>). As expected, the best assembly result was obtained for the EU-B04 reference strain with PacBio sequencing (66 scaffolds, N50 = 2,978 kb). Genomic DNA library construction likely explains the variability in assembly size as certain protocols might have excluded long repetitive regions. It is especially the case of CAM and CAP strains that both exhibit a genome size half the size of genomes of strains found on apple trees. In this case, library construction kits included PCR amplification that might have filtered out the major part of AT-rich regions (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aird <em>et al.</em> 2011</a>), a bias already reported in other fungi harboring an AT-rich genome (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Oyola <em>et al.</em> 2012</a>). The actual genome size of <em>V. inaequalis</em> strains is over 80 Mb as measured using flow cytometry (average size of 85.1 ± 5.0 Mb; <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1B</a> and Table S1). The measured genome sizes suggest that scaffolds in the PacBio assembly are separated by extremely long stretches of repetitive elements. The average assembly size for <em>V. aucupariae</em>, <em>V. pirina</em> and <em>V. asperata</em> strains is 52, 38 and 34 Mb, respectively (<a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1B</a> and Table S1). The genome of these species appeared to really contain fewer repeats as library construction and sequencing strategy for these genomes was similar to those of <em>V. inaequalis</em> genomes for which repetitive elements were sequenced. However, flow cytometry determined genome sizes of 63.6 ± 1.6 Mb, 70.6 ± 2.7 Mb and 46.1 ± 1.2 Mb for <em>V. aucupariae</em>, <em>V. pirina</em> and <em>V. asperata</em>, respectively (<a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1B</a> and Table S1). The former two species appear to contain a significant amount of repetitive elements that could not be deduced from their genome assemblies likely because Illumina sequencing tends to collapse repeat regions (<a id="xref-ref-60-6" class="xref-bibr" href="#ref-60">Shiller <em>et al.</em> 2015</a>; <a id="xref-ref-16-6" class="xref-bibr" href="#ref-16">Deng <em>et al.</em> 2017</a>).</p><p id="p-25">The genome structure of <em>V. inaequalis</em> is also a mosaic of GC-equilibrated and AT-rich regions as shown for the EU-B04 reference isolate (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Indeed, the GC-content distribution in all genomes exhibits a bimodal pattern typical of genomes with large proportions of AT-rich regions (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1C</a>; <a id="xref-ref-67-3" class="xref-bibr" href="#ref-67">Testa <em>et al.</em> 2016</a>). These regions cover on average 45.3% of <em>V. inaequalis</em> genome assemblies with a size over 60 Mb, while they cover on average 32.5%, 6.6% and 9.9% of <em>V. aucupariae</em>, <em>V. pirina</em> and <em>V. asperata</em> assembled genomes, respectively (Table S1). Considering the measured genome sizes, these values are likely underestimated and AT-rich regions may represent from 30% in <em>V. asperata</em> to 50% in <em>V. pirina</em>. The GC content of AT-rich regions in <em>Venturia</em> species ranges from 33.5 to 41.3% (Table S1), which is similar to the 33.8% reported for <em>Leptosphaeria maculans</em> (<a id="xref-ref-67-4" class="xref-bibr" href="#ref-67">Testa <em>et al.</em> 2016</a>), another species with a mosaic genome structure of GC-equilibrated and AT-rich regions (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Rouxel <em>et al.</em> 2011</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2405/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Genome features of the Venturia inaequalis EU-B04 reference strain. Twenty-nine out of 66 scaffolds are shown, representing 95% of the total assembly. Scaffold number is indicated on the outer circle. Tracks show (A) Single-Nucleotide Polymorphism (SNP) density between isolates (black) and clones (yellow); (B) GC content (%), the orange line indicate 50%; (C) Gene density according to Braker1 gene prediction, magenta circles indicate the localization of small secreted proteins (SSPs). (D) transposable element density according to REPET analysis. (E) RIP (Repeat Induced Point mutation) index). (F) close-up on scaffold 1 showing SNP densities and GC content as described in (A) and (B) tracks, respectively, the dotted blue line indicating the GC content threshold used by OcculterCut (Testa et al. 2016) to separate GC-quilibrated and AT-rich regions; the latter are highlighted with gray shading. (G) Difference in di-nucleotide frequency between GC-equilibrated and AT-rich regions in genomes of all isolates as determined by OcculterCut. Negative and positive values indicate over-representation in GC-equilibrated and AT-rich regions, respectively. Values for the EU-B04 strain reference assembly are indicated in yellow." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1754084616" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Genome features of the &lt;em&gt;Venturia inaequalis&lt;/em&gt; EU-B04 reference strain. Twenty-nine out of 66 scaffolds are shown, representing 95% of the total assembly. Scaffold number is indicated on the outer circle. Tracks show (A) Single-Nucleotide Polymorphism (SNP) density between isolates (black) and clones (yellow); (B) GC content (%), the orange line indicate 50%; (C) Gene density according to Braker1 gene prediction, magenta circles indicate the localization of small secreted proteins (SSPs). (D) transposable element density according to REPET analysis. (E) RIP (Repeat Induced Point mutation) index). (F) close-up on scaffold 1 showing SNP densities and GC content as described in (A) and (B) tracks, respectively, the dotted blue line indicating the GC content threshold used by OcculterCut (Testa &lt;em&gt;et al.&lt;/em&gt; 2016) to separate GC-quilibrated and AT-rich regions; the latter are highlighted with gray shading. (G) Difference in di-nucleotide frequency between GC-equilibrated and AT-rich regions in genomes of all isolates as determined by OcculterCut. Negative and positive values indicate over-representation in GC-equilibrated and AT-rich regions, respectively. Values for the EU-B04 strain reference assembly are indicated in yellow.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2405/F2.medium.gif" width="440" height="356"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/8/2405/F2.medium.gif" width="440" height="356"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2405/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2405/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493817" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-26" class="first-child">Genome features of the <em>Venturia inaequalis</em> EU-B04 reference strain. Twenty-nine out of 66 scaffolds are shown, representing 95% of the total assembly. Scaffold number is indicated on the outer circle. Tracks show (A) Single-Nucleotide Polymorphism (SNP) density between isolates (black) and clones (yellow); (B) GC content (%), the orange line indicate 50%; (C) Gene density according to Braker1 gene prediction, magenta circles indicate the localization of small secreted proteins (SSPs). (D) transposable element density according to REPET analysis. (E) RIP (Repeat Induced Point mutation) index). (F) close-up on scaffold 1 showing SNP densities and GC content as described in (A) and (B) tracks, respectively, the dotted blue line indicating the GC content threshold used by OcculterCut (<a id="xref-ref-67-5" class="xref-bibr" href="#ref-67">Testa <em>et al.</em> 2016</a>) to separate GC-quilibrated and AT-rich regions; the latter are highlighted with gray shading. (G) Difference in di-nucleotide frequency between GC-equilibrated and AT-rich regions in genomes of all isolates as determined by OcculterCut. Negative and positive values indicate over-representation in GC-equilibrated and AT-rich regions, respectively. Values for the EU-B04 strain reference assembly are indicated in yellow.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">Despite the difference in assembly size, the BUSCO assessment of gene content indicates a similar and very high level of completeness for most strains (median of 98% complete single copy genes). This result also indicates that most of the predicted genes are located in GC-equilibrated regions, which represent the most well sequenced and assembled in our dataset. Six strains show a retrieval rate of BUSCO genes lower than 90% (Table S1), meaning that genome sequencing and/or assembly quality is very low and these genomes should be used with caution in further analyses.</p></div><div id="sec-14" class="subsection"><h3>SNP discovery and population structure of Venturia inaequalis</h3><p id="p-28">Using multiple sequence alignments of the identified BUSCO genes, we computed genetic distances between isolates and identified six pairs of strains and one triplet with very low divergence, suggesting they were clones (2225/2446, 2226/2450, 2447/2458, 2474/2475, 2478/2480, 186/2269 and 2503/2504/2263). Consistently, fewer SNPs were discovered between these pairs compared to other strains from the same populations (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). We could use this information to perform a SNP false discovery analysis because we expect <em>de novo</em> mutations to be rare between clones of <em>V. inaequalis</em> as it reproduces sexually each year (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Bowen <em>et al.</em> 2011</a>). Our data suggests that 25–50% of SNPs identified without masking genomes are false positives (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). About 90% of SNPs identified between clones are located throughout the genome within AT-rich regions (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>, <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>, <a id="xref-fig-2-3" class="xref-fig" href="#F2">2D</a> and <a id="xref-fig-2-4" class="xref-fig" href="#F2">2F</a>). Reads that originate from repetitive elements sharing high sequence similarity are likely to be inaccurately mapped on the reference genome, which could explain the vast majority of SNPs observed between clones. In contrast, GC-equilibrated regions appear to contain a higher proportion of SNPs discovered between non-clonal strains, which are enriched on certain scaffolds (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>, <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2A</a> and <a id="xref-fig-2-6" class="xref-fig" href="#F2">2F</a>). Masking AT-rich regions as determined by OcculterCut eliminated between 88.7% and 96.6% of false positive SNPs for a saving of 16.5–48.4% of the initial SNPs between non-clonal strains (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). The proportion of retained SNPs reached 52% in the global dataset of 12.7 million SNPs identified between all strains. The fact that GC-equilibrated regions still contain 3.4–11.3% of false positive SNPs is likely explained by the expansion of gene families through gene duplication in the genome of <em>Venturia</em> species (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>; <a id="xref-ref-60-7" class="xref-bibr" href="#ref-60">Shiller <em>et al.</em> 2015</a>; <a id="xref-ref-16-7" class="xref-bibr" href="#ref-16">Deng <em>et al.</em> 2017</a>). This observation suggests that masking AT-rich regions or transposable elements alone often may be not enough to reach an acceptable FDR. As reported here, sequencing two clones or twice the same individuals easily allows detecting regions with high level of FDRs and is of great help to define hard filtering and/or SNP recalibration settings, especially for highly repetitive or polyploid genomes of sexual organisms.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493888/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493888/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Single Nucleotide Polymorphism (SNP) false discovery analysis in <em>Venturia inaequalis</em> populations. Isolates originate from populations isolated on loquat (LOQ), firethorn (PYR), <em>Malus sylvestris</em> (SYL) and on <em>Malus sieversii</em> in Central Asian Mountains (CAM) and Central Asian Plains (CAP)</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">Once cleared of the false positive SNPs, our dataset allowed a clear discrimination of the <em>V. inaequalis</em> strains according to their host of origin as shown by a principle component analysis (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). This result is highly congruent with our phylogenetic analysis (<a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1A</a>) and with previous studies (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Gladieux <em>et al.</em> 2010a</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2405/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Population structure of Venturia inaequalis. Principle components analysis of 71 V. inaequalis strains sampled on Malus sp. (green), Eriobotrya japonica (in red), Pyracantha sp. (blue), and Sorbus aucuparia (purple). The first axis differentiates strains sampled on apple trees. The second axis differentiates strains sampled on Pyracantha sp. from those sampled on E. japonica and S. aucuparia." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1754084616" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Population structure of &lt;em&gt;Venturia inaequalis&lt;/em&gt;. Principle components analysis of 71 &lt;em&gt;V. inaequalis&lt;/em&gt; strains sampled on &lt;em&gt;Malus&lt;/em&gt; sp. (green), &lt;em&gt;Eriobotrya japonica&lt;/em&gt; (in red), &lt;em&gt;Pyracantha&lt;/em&gt; sp. (blue), and &lt;em&gt;Sorbus aucuparia&lt;/em&gt; (purple). The first axis differentiates strains sampled on apple trees. The second axis differentiates strains sampled on &lt;em&gt;Pyracantha&lt;/em&gt; sp. from those sampled on &lt;em&gt;E. japonica&lt;/em&gt; and &lt;em&gt;S. aucuparia&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2405/F3.medium.gif" width="440" height="330"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/8/2405/F3.medium.gif" width="440" height="330"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2405/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2405/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493784" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-33" class="first-child">Population structure of <em>Venturia inaequalis</em>. Principle components analysis of 71 <em>V. inaequalis</em> strains sampled on <em>Malus</em> sp. (green), <em>Eriobotrya japonica</em> (in red), <em>Pyracantha</em> sp. (blue), and <em>Sorbus aucuparia</em> (purple). The first axis differentiates strains sampled on apple trees. The second axis differentiates strains sampled on <em>Pyracantha</em> sp. from those sampled on <em>E. japonica</em> and <em>S. aucuparia</em>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Gene predictions</h3><p id="p-34">Most of <em>V. inaequalis</em>, <em>V. pirina</em> and <em>V. aucupariae</em> strains contain around 11,600 predicted genes, while <em>V. asperata</em> strains contain fewer predicted gene models, around 10,400 (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figure 1D</a> and Table S1). Virulence of plant pathogens relies on the secretion of small-secreted proteins (SSPs) that are often rich in cysteine residues (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Lo Presti <em>et al.</em> 2015</a>). The number of predicted SSPs of less than 300 amino acids ranges between 527 and 756 in <em>V. inaequalis</em> and <em>V. pirina</em> strains, representing between 4.8% and 6.1% of the predicted proteome (<a id="xref-fig-1-9" class="xref-fig" href="#F1">Figure 1D</a> and Table S1). The PacBio assembly of the <em>V. inaequalis</em> EU-B04 isolate certainly explains the much higher number of predicted SSPs and percentage of the proteome. Despite a similar total number of predicted genes, the <em>V. aucupariae</em> isolate contains fewer SSPs (461; <a id="xref-fig-1-10" class="xref-fig" href="#F1">Figure 1D</a>). In addition to a slightly reduced gene complement, <em>V. asperata</em> strains contain fewer predicted SSPs that represent less than 4% of the predicted proteome (between 395 and 427; <a id="xref-fig-1-11" class="xref-fig" href="#F1">Figure 1D</a> and Table S1). Detailed analysis of the <em>V. inaequalis</em> and <em>V. pirina</em> predicted secretome identified specific SSPs putatively involved in host specificity and recognition by apple resistant genes (<a id="xref-ref-16-8" class="xref-bibr" href="#ref-16">Deng <em>et al.</em> 2017</a>). Similarly, in our report, differences in gene complement might explain the host range of the different species, especially the host specialization of <em>V. inaequalis formae specialis</em> and the low virulence of <em>V. asperata</em> on apple (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Caffier <em>et al.</em> 2012</a>). In the PacBio assembly, genes tend to be located within GC-equilibrated regions (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2B</a> and <a id="xref-fig-2-8" class="xref-fig" href="#F2">2C</a>). Although SSPs of <em>V. inaequalis</em> were reported closer to repetitive elements compared to other genes (<a id="xref-ref-16-9" class="xref-bibr" href="#ref-16">Deng <em>et al.</em> 2017</a>), SSPs do not tend to be located within long AT-rich regions as observed in <em>L. maculans</em> (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2C</a> and <a id="xref-fig-2-10" class="xref-fig" href="#F2">2D</a>; <a id="xref-ref-58-2" class="xref-bibr" href="#ref-58">Rouxel <em>et al.</em> 2011</a>). Further detailed analyses of specific gene families, especially SSPs, carbohydrate degrading enzymes and secondary metabolite gene clusters will provide new insights about the pathogenicity and host specificity of <em>Venturia</em> species.</p></div><div id="sec-16" class="subsection"><h3>Invasion of transposable elements</h3><p id="p-35">The repetitive element content across all genomes was annotated using the REPET pipeline (<a id="xref-fig-1-12" class="xref-fig" href="#F1">Figure 1B</a>; Table S1). The transposable element (TE) content covers 55.7% of the reference EU-B04 genome assembly, of which 66.4% are Class I LTR element, mainly assigned to Gypsy or Copia superfamilies (37.5% and 26.3%, respectively); and 9.5% are Class II elements, mostly belonging to the TIR superfamily (8.2%) (Figure S1). Consistent with the exclusion of repeat content from CAM and CAP populations DNA libraries, only a median of 8.5% of the assemblies are covered with TEs (<a id="xref-fig-1-13" class="xref-fig" href="#F1">Figure 1B</a> and Table S1). In other <em>V. inaequalis</em> genomes, the median coverage of TEs is 41.7% (<a id="xref-fig-1-14" class="xref-fig" href="#F1">Figure 1B</a> and Table S1). TE coverage in the <em>V. aucupariae</em> assembly is similar, with 17.4% (<a id="xref-fig-1-15" class="xref-fig" href="#F1">Figure 1B</a> and Table S1). In contrast, TE coverage is very limited in <em>V. pirina</em> (4.1% on average) and <em>V. asperata</em> (3.4% on average), consistent with the exclusion of repeat content from genomic assemblies of the former and the limited number of TEs in the latter (<a id="xref-fig-1-16" class="xref-fig" href="#F1">Figures 1B</a> and Table S1). The repetitive content of <em>V. asperata</em> strains differs from others as it does not contain Helitron and LINE elements, and is almost devoid of Copia elements (Figure S1 and Table S2). Gypsy and Copia TEs are predominant in all other <em>Venturia</em> strains. Notably Copia elements and class II TEs, especially TIR elements, appear to have greatly expanded after the divergence with <em>V. asperata</em> (Table S2 and Figure S1). Hierarchical clustering of the coverage of TE models annotated y REPET suggests recent expansion of 32 Copia and Gypsy models in all <em>Venturia</em> species (Table S3 and Figure S2); particularly two models (RLG-comp_VeinUTG-B-G309-Map3 and noCat_VeinUTG-B-R23-Map20) exhibits high coverage and are likely still actively transposing (Table S3 and Figure S2). A second cluster of 31 models might represent ancestral expansion of currently inactive TEs and a third cluster contains models with limited coverage (Figure S2). A few elements appear to be actively multiplying in certain genomes. For example, one Gypsy element (RLG-comp_VeinUTG-B-G1005-Map14) exhibits higher coverage in <em>V. asperata</em>, <em>V. aucupariae</em> and, to a lesser extent, in <em>V. pirina</em> compared to <em>V. inaequalis</em> strains (Table S3 and Figure S2), while another Gypsy element (RLG-comp_VeinUTG-B-R34-Map20) and a LARD element (RXX-LARD_VeinUTG-B-G646-Map11) show highest coverage in <em>V. inaequalis</em> 2199 and <em>V. inaequalis</em> B04 strains, respectively (Table S3 and Figure S2). In the <em>V. inaequalis</em> B04 reference genome, TEs are mostly located within AT-rich regions (<a id="xref-fig-2-11" class="xref-fig" href="#F2">Figure 2B</a> and <a id="xref-fig-2-12" class="xref-fig" href="#F2">2D</a>), but many appear to be located in close vicinity to genes as reported for SSPs (<a id="xref-ref-16-10" class="xref-bibr" href="#ref-16">Deng <em>et al.</em> 2017</a>). In fungi, TE activity can be suppressed by Repeat-Induced Point mutation (RIP), a process that induces in repetitive DNA cytosine to thymine transition mutations during heterokaryon formation prior to meiosis (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Watters <em>et al.</em> 1999</a>). As such, RIP is a mechanism that produces AT-rich regions. Evaluation of RIP in <em>Venturia</em> strains showed that most TEs have been mutated through RIP (<a id="xref-fig-2-13" class="xref-fig" href="#F2">Figure 2E</a> and Table S1). AT-rich regions in all strains are enriched in TpA di-nucleotides (<a id="xref-fig-2-14" class="xref-fig" href="#F2">Figure 2G</a>), a feature typical of RIP in non-coding regions (<a id="xref-ref-67-7" class="xref-bibr" href="#ref-67">Testa <em>et al.</em> 2016</a>). Detailed analysis of the dynamics of TEs will determine their impact on the different interactions between <em>Venturia</em> species and their diverse hosts.</p></div><div id="sec-17" class="subsection"><h3>Conclusions</h3><p id="p-36">Strains of the <em>Venturia</em> species complex are responsible for important diseases on Rosaceae fruit trees, as such they constitute an environmental and public health concern due to the massive use of fungicides necessary for their control. Our study provides valuable genomic resources for those interested in identification of molecular basis of <em>Venturia</em> species adaptation to their host and in mapping traits of interest (virulence, aggressiveness, resistance to fungicides etc…). In addition, these genomes will be used for comparative genomic studies to develop diagnostic molecular tools and to understand the evolution of genomes in relation to their hosts and in particular during the domestication of the apple tree.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-37">The project was funded Plant Healt Department of INRA (ESCAPADES Project) and by the Agence Nationale pour la Recherche (Bioadapt ANR Gandalf ANR-12-ADAP-0009 and Fungisochore ANR-09-GENM-028 projects) and by the French Region Pays de la Loire (ROAD MOVIE project). JS and TP research was conducted in the framework of the regional program “Objectif Végétal, Research, Education and Innovation in Pays de la Loire“, supported by the French Region Pays de la Loire, Angers Loire Métropole and the European Regional Development Fund. LD was supported by the University of Angers, MDG was supported by Plant health Department of INRA and Region Pays de La Loire.We thank Dr. P. Gladieux (INRA, Montpellier, France), Dr. XG Zhang (Shandong University, Taian, China), Dr. H Ishii (Kobe University, Kobe, Japan), Dr. Tim Roberts (East Malling Research, East Malling, England), Dr. A Kollar (JKI, Dossenheim, Germany) and Dr. E Dapena (SERIDA, Villaviciosia, Spain), Dr. F. Laurens (INRA, Angers, France) and Catherine Peix for collecting and providing strains or infected apple leaves.</p><p id="p-38">This work was performed in collaboration with the GeT core facility (Toulouse, France; get.genotoul.fr), and was supported by France Génomique National infrastructure, funded as part of “Investissement d’avenir” program managed by Agence Nationale pour la Recherche (contract ANR-10-INBS-09).</p><p id="p-39">The present work has benefited from the Cytometry facility of Imagerie - Gif, (<a href="http://www.i2bc.paris-saclay.fr">www.i2bc.paris-saclay.fr</a>), member of IBiSA (<a href="http://www.ibisa.net">www.ibisa.net</a>), supported by “France-BioImaging” (ANR-10-INBS-04-01), and the Labex “Saclay Plant Science”(ANR-11-IDEX-0003-02).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-4"><p id="p-40">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.7346714">https://doi.org/10.25387/g3.7346714</a>.</p></li><li class="fn" id="fn-5"><p id="p-41"><em>Communicating editor: D. Baltrus</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-01-31"><span class="received-label">Received </span>January 31, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-06-18"><span class="accepted-label">Accepted </span>June 18, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Le Cam <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.1" data-doi="10.1186/gb-2011-12-2-r18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aird</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. G.</span> <span class="cit-name-surname">Ross</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. S.</span> <span class="cit-name-surname">Chen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Denielsson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Fennell</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Analyzing and minimizing PCR amplification bias in Illumina sequencing libraries</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">R18</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2011–12–2-r18</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DAird%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.issue%253D2%26rft.spage%253DR18%26rft.epage%253DR18%26rft.atitle%253DAnalyzing%2Band%2Bminimizing%2BPCR%2Bamplification%2Bbias%2Bin%2BIllumina%2Bsequencing%2Blibraries.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2011-12-2-r18%26rft_id%253Dinfo%253Apmid%252F21338519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2011-12-2-r18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21338519&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.2" data-doi="10.1101/gr.88502"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bao</span>,  <span class="cit-name-given-names">Z.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Eddy</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Automated de novo identification of repeat sequence families in sequenced genomes</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1269</span>–<span class="cit-lpage">1276</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.88502</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.88502%26rft_id%253Dinfo%253Apmid%252F12176934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjEyLzgvMTI2OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOC8yNDA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.3" data-doi="10.1007/s11557-006-0114-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beck</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ritschel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Schubert</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Braun</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Triebel</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Phylogenetic relationships of the anamorphic genus <em>Fusicladium s. lat</em>. as inferred by ITS nrDNA data</span>. <abbr class="cit-jnl-abbrev">Mycol. Prog.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">116</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11557-006-0114-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMycol.%2BProg.%26rft.volume%253D4%26rft.spage%253D111%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11557-006-0114-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11557-006-0114-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.4" data-doi="10.1094/PHYTO.2000.90.3.236"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bénaouf</span>,  <span class="cit-name-given-names">G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Parisi</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Genetics of host-pathogen relationships between <em>Venturia inaequalis</em> races 6 and 7 and <em>Malus</em> species</span>. <abbr class="cit-jnl-abbrev">Phytopathology</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">236</span>–<span class="cit-lpage">242</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1094/PHYTO.2000.90.3.236</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytopathology%26rft.stitle%253DPhytopathology%26rft.aulast%253DBenaouf%26rft.auinit1%253DG.%26rft.volume%253D90%26rft.issue%253D3%26rft.spage%253D236%26rft.epage%253D242%26rft.atitle%253DGenetics%2Bof%2BHost-Pathogen%2BRelationships%2BBetween%2BVenturia%2Binaequalis%2BRaces%2B6%2Band%2B7%2Band%2BMalus%2BSpecies.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPHYTO.2000.90.3.236%26rft_id%253Dinfo%253Apmid%252F18944614%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PHYTO.2000.90.3.236&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18944614&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.5" data-doi="10.1038/nbt.3238"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berlin</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Koren</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. S.</span> <span class="cit-name-surname">Chin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Drake</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Landolin</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Assembling large genomes with single-molecule sequencing and locality-sensitive hashing</span>. <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">623</span>–<span class="cit-lpage">630</span>. Erratum: 1109.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D623%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3238%26rft_id%253Dinfo%253Apmid%252F26006009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3238&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26006009&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.6" data-doi="10.1186/s40793-016-0161-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bock</span>,  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Chen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Yu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. L.</span> <span class="cit-name-surname">Stevenson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. W.</span> <span class="cit-name-surname">Wood</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Draft genome sequence of <em>Fusicladium effusum</em>, cause of pecan scab</span>. <abbr class="cit-jnl-abbrev">Stand. Genomic Sci.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">36</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s40793-016-0161-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStand.%2BGenomic%2BSci.%26rft.volume%253D11%26rft.spage%253D36%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs40793-016-0161-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s40793-016-0161-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.7" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>,  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lohse</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Usadel</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu170</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24695404&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340049100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.8" data-doi="10.1111/j.1364-3703.2009.00543.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bowen</span>,  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. H.</span> <span class="cit-name-surname">Mesarich</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Rees-George</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Cui</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Fitzgerald</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Candidate effector gene identification in the ascomycete fungal phytopathogen <em>Venturia inaequalis</em> by expressed sequence tag analysis</span>. <abbr class="cit-jnl-abbrev">Mol. Plant Pathol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">431</span>–<span class="cit-lpage">448</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1364-3703.2009.00543.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BPlant%2BPathol%26rft.aulast%253DBowen%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253D431%26rft.epage%253D448%26rft.atitle%253DCandidate%2Beffector%2Bgene%2Bidentification%2Bin%2Bthe%2Bascomycete%2Bfungal%2Bphytopathogen%2BVenturia%2Binaequalis%2Bby%2Bexpressed%2Bsequence%2Btag%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1364-3703.2009.00543.x%26rft_id%253Dinfo%253Apmid%252F19400844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1364-3703.2009.00543.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19400844&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.9" data-doi="10.1111/j.1364-3703.2010.00656.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bowen</span>,  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. H.</span> <span class="cit-name-surname">Mesarich</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V. G.</span> <span class="cit-name-surname">Bus</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Beresford</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Plummer</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title"><em>Venturia inaequalis</em>: the causal agent of apple scab</span>. <abbr class="cit-jnl-abbrev">Mol. Plant Pathol.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">105</span>–<span class="cit-lpage">122</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1364-3703.2010.00656.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DMol%2BPlant%2BPathol%26rft.aulast%253DBowen%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D12%26rft.issue%253D2%26rft.spage%253D105%26rft.epage%253D122%26rft.atitle%253DVenturia%2Binaequalis%253A%2Bthe%2Bcausal%2Bagent%2Bof%2Bapple%2Bscab.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1364-3703.2010.00656.x%26rft_id%253Dinfo%253Apmid%252F21199562%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1364-3703.2010.00656.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21199562&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285880800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.10" data-doi="10.1111/j.1365-3059.2011.02583.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Caffier</span>,  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Le Cam</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Expert</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Devaux</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Giraud</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A new scab-like disease on apple caused by the formerly saprotroph fungus <em>Venturia asperata</em></span>. <abbr class="cit-jnl-abbrev">Plant Pathol.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">915</span>–<span class="cit-lpage">924</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-3059.2011.02583.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPathol.%26rft.volume%253D61%26rft.spage%253D915%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-3059.2011.02583.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-3059.2011.02583.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.11" data-doi="10.1093/oxfordjournals.molbev.a026334"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Castresana</span>,  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">540</span>–<span class="cit-lpage">552</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/oxfordjournals.molbev.a026334</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DCastresana%26rft.auinit1%253DJ.%26rft.volume%253D17%26rft.issue%253D4%26rft.spage%253D540%26rft.epage%253D552%26rft.atitle%253DSelection%2Bof%2BConserved%2BBlocks%2Bfrom%2BMultiple%2BAlignments%2Bfor%2BTheir%2BUse%2Bin%2BPhylogenetic%2BAnalysis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Foxfordjournals.molbev.a026334%26rft_id%253Dinfo%253Apmid%252F10742046%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/oxfordjournals.molbev.a026334&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10742046&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086337700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.12" data-doi="10.1038/ncomms3876"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheeseman</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ropars</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Renault</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Dupont</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Gouzy</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Multiple recent horizontal transfers of a large genomic region in cheese making fungi</span>. <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">2876</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms3876</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D5%26rft.spage%253D2876%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms3876%26rft_id%253Dinfo%253Apmid%252F24407037%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms3876&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24407037&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.13" data-doi="10.1038/nmeth.2474"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chin</span>,  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. H.</span> <span class="cit-name-surname">Alexander</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Marks</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Klammer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Drake</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Nonhybrid, finished microbial genome assemblies from long-read SMRT sequencing data</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">563</span>–<span class="cit-lpage">569</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.2474</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D10%26rft.spage%253D563%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.2474%26rft_id%253Dinfo%253Apmid%252F23644548%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.2474&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23644548&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000319668700028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.14" data-doi="10.1021/pr500176c"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cooke</span>,  <span class="cit-name-given-names">I. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Jones</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Bowen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Deng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Faou</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Proteogenomic analysis of the <em>Venturia pirina</em> (pear scab fungus) secretome reveals potential effectors</span>. <abbr class="cit-jnl-abbrev">J. Proteome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">3635</span>–<span class="cit-lpage">3644</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1021/pr500176c</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BProteome%2BRes.%26rft.volume%253D13%26rft.spage%253D3635%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fpr500176c%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/pr500176c&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.15" data-doi="10.3114/sim.2007.58.07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crous</span>,  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Schubert</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Braun</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. S.</span> <span class="cit-name-surname">de Hoog</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Hocking</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Opportunistic, human-pathogenic species in the Herpotrichiellaceae are phenotypically similar to saprobic or phytopathogenic species in the Venturiaceae</span>. <abbr class="cit-jnl-abbrev">Stud. Mycol.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">217</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3114/sim.2007.58.07</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStudies%2Bin%2BMycology%26rft.stitle%253DSIM%26rft.aulast%253DCrous%26rft.auinit1%253DP.W.%26rft.volume%253D58%26rft.issue%253D1%26rft.spage%253D185%26rft.epage%253D217%26rft.atitle%253DOpportunistic%252C%2Bhuman-pathogenic%2Bspecies%2Bin%2Bthe%2BHerpotrichiellaceae%2Bare%2Bphenotypically%2Bsimilar%2Bto%2Bsaprobic%2Bor%2Bphytopathogenic%2Bspecies%2Bin%2Bthe%2BVenturiaceae%26rft_id%253Dinfo%253Adoi%252F10.3114%252Fsim.2007.58.07%26rft_id%253Dinfo%253Apmid%252F18491000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3114/sim.2007.58.07&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18491000&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.16" data-doi="10.1186/s12864-017-3699-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Deng</span>,  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Plummer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A. B.</span> <span class="cit-name-surname">Jones</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. H.</span> <span class="cit-name-surname">Mesarich</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Shiller</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Comparative analysis of the predicted secretomes of <em>Rosaceae</em> scab pathogens <em>Venturia inaequalis</em> and <em>V. pirina</em> reveals expanded effector families and putative determinants of host range</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">339</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-017-3699-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D18%26rft.spage%253D339%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-017-3699-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-017-3699-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.17" data-doi="10.1038/ng.806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>,  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Banks</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Poplin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. V.</span> <span class="cit-name-surname">Garimella</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Maguire</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">498</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.806</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDepristo%26rft.auinit1%253DM.%2BA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D491%26rft.epage%253D498%26rft.atitle%253DA%2Bframework%2Bfor%2Bvariation%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2BDNA%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.806%26rft_id%253Dinfo%253Apmid%252F21478889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478889&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289972600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.18" data-doi="10.1002/cyto.a.10013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dolezel</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bartos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Voglmayr</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Greilhuber</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Nuclear DNA content and genome size of trout and human</span>. <abbr class="cit-jnl-abbrev">Cytometry A</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">127</span>–<span class="cit-lpage">128</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytometry.%2BPart%2BA%2B%253A%2B%2Bthe%2Bjournal%2Bof%2Bthe%2BInternational%2BSociety%2Bfor%2BAnalytical%2BCytology%26rft.stitle%253DCytometry%2BA%26rft.aulast%253DDolezel%26rft.auinit1%253DJ.%26rft.volume%253D51%26rft.issue%253D2%26rft.spage%253D127%26rft.epage%253D128%26rft.atitle%253DNuclear%2BDNA%2Bcontent%2Band%2Bgenome%2Bsize%2Bof%2Btrout%2Band%2Bhuman.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fcyto.a.10013%26rft_id%253Dinfo%253Apmid%252F12541287%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/cyto.a.10013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12541287&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.19" data-doi="10.1371/journal.pone.0160737"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ebrahimi</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.-B.</span> <span class="cit-name-surname">Fotuhifar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Javan Nikkhah</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Naghavi</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Baisakh</span></span></li></ol><cite>, <span class="cit-pub-date"> 2016</span> <span class="cit-article-title">Population genetic structure of apple scab (<em>Venturia inaequalis</em> (Cooke) G. Winter) in Iran</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e0160737</span>. Erratum: e0167415.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0160737</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De0160737%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0160737%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0160737&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.20" data-doi="10.1093/nar/gkh340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>,  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1792</span>–<span class="cit-lpage">1797</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkh340</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkh340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15034147&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220487200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.21" data-doi="10.1093/bioinformatics/bti1003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>,  <span class="cit-name-given-names">R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Myers</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">PILER: identification and classification of genomic repeats</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">i152</span>–<span class="cit-lpage">i158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bti1003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D21%26rft.issue%253Dsuppl_1%26rft.spage%253Di152%26rft.epage%253Di158%26rft.atitle%253DPILER%253A%2Bidentification%2Band%2Bclassification%2Bof%2Bgenomic%2Brepeats%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti1003%26rft_id%253Dinfo%253Apmid%252F15961452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bti1003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15961452&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230273000017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.22" data-doi="10.1016/j.jmb.2004.01.025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eisenhaber</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Schneider</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Wildpaner</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Eisenhaber</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A sensitive predictor for potential GPI lipid modification sites in fungal protein sequences and its application to genome-wide studies for <em>Aspergillus nidulans</em>, <em>Candida albicans</em>, <em>Neurospora crassa</em>, <em>Saccharomyces cerevisiae</em>, and <em>Schizosaccharomyces pombe</em></span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">337</span>: <span class="cit-fpage">243</span>–<span class="cit-lpage">253</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.jmb.2004.01.025</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DEisenhaber%26rft.auinit1%253DB.%26rft.volume%253D337%26rft.issue%253D2%26rft.spage%253D243%26rft.epage%253D253%26rft.atitle%253DA%2Bsensitive%2Bpredictor%2Bfor%2Bpotential%2BGPI%2Blipid%2Bmodification%2Bsites%2Bin%2Bfungal%2Bprotein%2Bsequences%2Band%2Bits%2Bapplication%2Bto%2Bgenome-wide%2Bstudies%2Bfor%2BAspergillus%2Bnidulans%252C%2BCandida%2Balbicans%252C%2BNeurospora%2Bcrassa%252C%2BSaccharomyces%2Bcerevisiae%2Band%2BSchizosaccharomyces%2Bpombe.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jmb.2004.01.025%26rft_id%253Dinfo%253Apmid%252F15003443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jmb.2004.01.025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15003443&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220236200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.23" data-doi="10.1093/nar/gkt1223"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Finn</span>,  <span class="cit-name-given-names">R. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bateman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Clements</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Coggill</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. Y.</span> <span class="cit-name-surname">Eberhardt</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Pfam: the protein families database</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">D222</span>–<span class="cit-lpage">D230</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkt1223</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkt1223%26rft_id%253Dinfo%253Apmid%252F24288371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkt1223&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24288371&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000331139800034&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.24" data-doi="10.1371/journal.pone.0016526"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flutre</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Duprat</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Feuillet</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Quesneville</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Considering transposable element diversification in de novo annotation approaches</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-elocation-id">e16526</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0016526</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DFlutre%26rft.auinit1%253DT.%26rft.volume%253D6%26rft.issue%253D1%26rft.spage%253De16526%26rft.epage%253De16526%26rft.atitle%253DConsidering%2Btransposable%2Belement%2Bdiversification%2Bin%2Bde%2Bnovo%2Bannotation%2Bapproaches.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0016526%26rft_id%253Dinfo%253Apmid%252F21304975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0016526&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21304975&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.25" data-doi="10.1371/journal.pone.0001455"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gladieux</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X. G.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Afoufa-Bastien</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Valdebenito Sanhueza</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Sbaghi</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">On the origin and spread of the scab disease of apple: out of central Asia</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">3</span>: <span class="cit-elocation-id">e1455</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0001455</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DGladieux%26rft.auinit1%253DP.%26rft.volume%253D3%26rft.issue%253D1%26rft.spage%253De1455%26rft.epage%253De1455%26rft.atitle%253DOn%2Bthe%2Borigin%2Band%2Bspread%2Bof%2Bthe%2BScab%2Bdisease%2Bof%2Bapple%253A%2Bout%2Bof%2Bcentral%2BAsia.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0001455%26rft_id%253Dinfo%253Apmid%252F18197265%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0001455&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18197265&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.26" data-doi="10.1016/j.fgb.2009.12.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gladieux</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Caffier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Devaux</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Le Cam</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span><span class="cit-comment">a</span> <span class="cit-article-title">Host-specific differentiation among populations of <em>Venturia inaequalis</em> causing scab on apple, pyracantha and loquat</span>. <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">511</span>–<span class="cit-lpage">521</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fgb.2009.12.007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DGladieux%26rft.auinit1%253DP.%26rft.volume%253D47%26rft.issue%253D6%26rft.spage%253D511%26rft.epage%253D521%26rft.atitle%253DHost-specific%2Bdifferentiation%2Bamong%2Bpopulations%2Bof%2BVenturia%2Binaequalis%2Bcausing%2Bscab%2Bon%2Bapple%252C%2Bpyracantha%2Band%2Bloquat.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2009.12.007%26rft_id%253Dinfo%253Apmid%252F20060485%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2009.12.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20060485&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000278170800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.27" data-doi="10.1111/j.1365-294X.2009.04498.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gladieux</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X. G.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Roldan Ruiz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Caffier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Leroy</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span><span class="cit-comment">b</span> <span class="cit-article-title">Evolution of the population structure of <em>Venturia inaequalis</em>, the apple scab fungus, associated with the domestication of its host</span>. <abbr class="cit-jnl-abbrev">Mol. Ecol.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">658</span>–<span class="cit-lpage">674</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-294X.2009.04498.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.stitle%253DMolecular%2BEcology%2B%2528Print%2529%26rft.aulast%253DGladieux%26rft.auinit1%253DP.%26rft.volume%253D19%26rft.issue%253D4%26rft.spage%253D658%26rft.epage%253D674%26rft.atitle%253DEvolution%2Bof%2Bthe%2Bpopulation%2Bstructure%2Bof%2BVenturia%2Binaequalis%252C%2Bthe%2Bapple%2Bscab%2Bfungus%252C%2Bassociated%2Bwith%2Bthe%2Bdomestication%2Bof%2Bits%2Bhost.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-294X.2009.04498.x%26rft_id%253Dinfo%253Apmid%252F20088887%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-294X.2009.04498.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20088887&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.28" data-doi="10.1016/j.fgb.2006.10.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guérin</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Gladieux</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Le Cam</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Origin and colonization history of newly virulent strains of the phytopathogenic fungus <em>Venturia inaequalis</em></span>. <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">284</span>–<span class="cit-lpage">292</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fgb.2006.10.005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DGuerin%26rft.auinit1%253DF.%26rft.volume%253D44%26rft.issue%253D4%26rft.spage%253D284%26rft.epage%253D292%26rft.atitle%253DOrigin%2Band%2Bcolonization%2Bhistory%2Bof%2Bnewly%2Bvirulent%2Bstrains%2Bof%2Bthe%2Bphytopathogenic%2Bfungus%2BVenturia%2Binaequalis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2006.10.005%26rft_id%253Dinfo%253Apmid%252F17166752%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2006.10.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17166752&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244902500006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.29" data-doi="10.1093/bioinformatics/btt086"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gurevich</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Saveliev</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Vyahhi</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Tesler</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">QUAST: quality assessment tool for genome assemblies</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1072</span>–<span class="cit-lpage">1075</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btt086</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt086%26rft_id%253Dinfo%253Apmid%252F23422339%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt086&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23422339&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000318109300015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.30" data-doi="10.1371/journal.pone.0091929"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoede</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Arnoux</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Moisset</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Chaumier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Inizan</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">PASTEC: An Automatic Transposable Element Classification Tool</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e91929</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0091929</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De91929%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0091929%26rft_id%253Dinfo%253Apmid%252F24786468%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0091929&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24786468&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.31" data-doi="10.1093/bioinformatics/btv661"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoff</span>,  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Lange</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Lomsadze</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Borodovsky</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Stanke</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">BRAKER1: Unsupervised RNA-Seq-Based Genome Annotation with GeneMark-ET and AUGUSTUS</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">767</span>–<span class="cit-lpage">769</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btv661</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv661%26rft_id%253Dinfo%253Apmid%252F26559507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv661&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26559507&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>,  <span class="cit-name-given-names">X.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">On global sequence alignment</span>. <abbr class="cit-jnl-abbrev">Comput. Appl. Biosci. CABIOS.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">227</span>–<span class="cit-lpage">235</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BAppl.%2BBiosci.%2BCABIOS.%26rft.volume%253D10%26rft.spage%253D227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-book" id="cit-9.8.2405.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>,  <span class="cit-name-given-names">A. L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. S.</span> <span class="cit-name-surname">Aldwinckle</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-source">Compendium of apple and pear diseases</span>, <span class="cit-publ-name">APS press</span>, <span class="cit-publ-loc">St Paul, Minn.</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.34" data-doi="10.1159/000084979"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jurka</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Kapitonov</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Pavlicek</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Klonowski</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Kohany</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Repbase Update, a database of eukaryotic repetitive elements</span>. <abbr class="cit-jnl-abbrev">Cytogenet. Genome Res.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">462</span>–<span class="cit-lpage">467</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1159/000084979</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenetic%2Band%2Bgenome%2Bresearch%26rft.stitle%253DCytogenet%2BGenome%2BRes%26rft.aulast%253DJurka%26rft.auinit1%253DJ.%26rft.volume%253D110%26rft.issue%253D1-4%26rft.spage%253D462%26rft.epage%253D467%26rft.atitle%253DRepbase%2BUpdate%252C%2Ba%2Bdatabase%2Bof%2Beukaryotic%2Brepetitive%2Belements.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000084979%26rft_id%253Dinfo%253Apmid%252F16093699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000084979&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16093699&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000231064600047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.35" data-doi="10.1006/jmbi.2000.4315"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krogh</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Larsson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">von Heijne</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. L.</span> <span class="cit-name-surname">Sonnhammer</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Predicting transmembrane protein topology with a Hidden Markov Model: application to complete genomes</span>. <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">305</span>: <span class="cit-fpage">567</span>–<span class="cit-lpage">580</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1006/jmbi.2000.4315</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DKrogh%26rft.auinit1%253DA.%26rft.volume%253D305%26rft.issue%253D3%26rft.spage%253D567%26rft.epage%253D580%26rft.atitle%253DPredicting%2Btransmembrane%2Bprotein%2Btopology%2Bwith%2Ba%2Bhidden%2BMarkov%2Bmodel%253A%2Bapplication%2Bto%2Bcomplete%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.2000.4315%26rft_id%253Dinfo%253Apmid%252F11152613%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.2000.4315&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11152613&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000167760800017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.36" data-doi="10.1016/j.fgb.2008.07.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kucheryava</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Bowen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. W.</span> <span class="cit-name-surname">Sutherland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Conolly</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. H.</span> <span class="cit-name-surname">Mesarich</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Two novel <em>Venturia inaequalis</em> genes induced upon morphogenetic differentiation during infection and <em>in vitro</em> growth on cellophane</span>. <abbr class="cit-jnl-abbrev">Fungal Genet. Biol.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">1329</span>–<span class="cit-lpage">1339</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fgb.2008.07.010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFungal%2Bgenetics%2Band%2Bbiology%2B%253A%2B%2BFG%2B%2526%2BB%26rft.stitle%253DFungal%2BGenet%2BBiol%26rft.aulast%253DKucheryava%26rft.auinit1%253DN.%26rft.volume%253D45%26rft.issue%253D10%26rft.spage%253D1329%26rft.epage%253D1339%26rft.atitle%253DTwo%2Bnovel%2BVenturia%2Binaequalis%2Bgenes%2Binduced%2Bupon%2Bmorphogenetic%2Bdifferentiation%2Bduring%2Binfection%2Band%2Bin%2Bvitro%2Bgrowth%2Bon%2Bcellophane.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fgb.2008.07.010%26rft_id%253Dinfo%253Apmid%252F18692586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fgb.2008.07.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18692586&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.37" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Durbin</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp324</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.38" data-doi="10.1094/PHYTO.2002.92.3.314"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Le Cam</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Parisi</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Arène</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Evidence of two formae speciales in <em>Venturia inaequalis</em>, responsible for apple and pyracantha scab</span>. <abbr class="cit-jnl-abbrev">Phytopathology</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">314</span>–<span class="cit-lpage">320</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1094/PHYTO.2002.92.3.314</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytopathology%26rft.stitle%253DPhytopathology%26rft.aulast%253DLe%2BCam%26rft.auinit1%253DB.%26rft.volume%253D92%26rft.issue%253D3%26rft.spage%253D314%26rft.epage%253D320%26rft.atitle%253DEvidence%2Bof%2BTwo%2BFormae%2BSpeciales%2Bin%2BVenturia%2Binaequalis%252C%2BResponsible%2Bfor%2BApple%2Band%2BPyracantha%2BScab.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPHYTO.2002.92.3.314%26rft_id%253Dinfo%253Apmid%252F18944005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PHYTO.2002.92.3.314&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18944005&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.39" data-doi="10.1111/nph.13658"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lemaire</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">De Gracia</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Leroy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Michalecka</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Lindhard-Pedersen</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Emergence of new virulent populations of apple scab from non-agricultural disease reservoirs</span>. <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">209</span>: <span class="cit-fpage">1220</span>–<span class="cit-lpage">1229</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.13658</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D209%26rft.spage%253D1220%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.13658%26rft_id%253Dinfo%253Apmid%252F26428268%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.13658&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26428268&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.40" data-doi="10.1186/1471-2148-13-64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leroy</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Lemaire</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Dunemann</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Le Cam</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The genetic structure of a <em>Venturia inaequalis</em> population in a heterogeneous host population composed of different Malusspecies</span>. <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">64</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2148-13-64</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvol.%2BBiol.%26rft.volume%253D13%26rft.spage%253D64%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-13-64%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-13-64&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.41" data-doi="10.1111/nph.13873"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leroy</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Caffier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Celton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Anger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. E.</span> <span class="cit-name-surname">Durel</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">When virulence originates from nonagricultural hosts: evolutionary and epidemiological consequences of introgressions following secondary contacts in <em>Venturia inaequalis</em></span>. <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">210</span>: <span class="cit-fpage">1443</span>–<span class="cit-lpage">1452</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/nph.13873</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytol.%26rft.volume%253D210%26rft.spage%253D1443%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fnph.13873%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/nph.13873&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.42" data-doi="10.1146/annurev-arplant-043014-114623"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lo Presti</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Lanver</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Schweizer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Tanaka</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Liang</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Fungal effectors and plant susceptibility</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Plant Biol.</abbr> <span class="cit-vol">66</span>: <span class="cit-fpage">513</span>–<span class="cit-lpage">545</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-arplant-043014-114623</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BPlant%2BBiol.%26rft.volume%253D66%26rft.spage%253D513%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-arplant-043014-114623%26rft_id%253Dinfo%253Apmid%252F25923844%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-arplant-043014-114623&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25923844&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.43" data-doi="10.1101/gr.111120.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lunter</span>,  <span class="cit-name-given-names">G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Goodson</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Stampy: a statistical algorithm for sensitive and fast mapping of Illumina sequence reads</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">936</span>–<span class="cit-lpage">939</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.111120.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.111120.110%26rft_id%253Dinfo%253Apmid%252F20980556%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzYvOTM2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.44" data-doi="10.1186/2047-217X-1-18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Xie</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Huang</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler</span>. Erratum: 4: 30. <abbr class="cit-jnl-abbrev">Gigascience</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">18</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/2047-217X-1-18</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigascience%26rft.volume%253D1%26rft.spage%253D18%26rft_id%253Dinfo%253Adoi%252F10.1186%252F2047-217X-1-18%26rft_id%253Dinfo%253Apmid%252F23587118%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/2047-217X-1-18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23587118&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.45" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hanna</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Banks</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sivachenko</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Cibulskis</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.107524.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOC8yNDA1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.46" data-doi="10.1016/0248-4900(93)90113-S"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marie</span>,  <span class="cit-name-given-names">D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. C.</span> <span class="cit-name-surname">Brown</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">A cytometric exercise in plant DNA histograms, with 2C values for 70 species</span>. <abbr class="cit-jnl-abbrev">Biol. Cell</abbr> <span class="cit-vol">78</span>: <span class="cit-fpage">41</span>–<span class="cit-lpage">51</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0248-4900(93)90113-S</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiology%2Bof%2Bthe%2Bcell%2B%252F%2Bunder%2Bthe%2Bauspices%2Bof%2Bthe%2BEuropean%2BCell%2BBiology%2BOrganization%26rft.stitle%253DBiol%2BCell%26rft.aulast%253DMarie%26rft.auinit1%253DD.%26rft.volume%253D78%26rft.issue%253D1-2%26rft.spage%253D41%26rft.epage%253D51%26rft.atitle%253DA%2Bcytometric%2Bexercise%2Bin%2Bplant%2BDNA%2Bhistograms%252C%2Bwith%2B2C%2Bvalues%2Bfor%2B70%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0248-4900%252893%252990113-S%26rft_id%253Dinfo%253Apmid%252F8220226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0248-4900(93)90113-S&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8220226&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993MA01500005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.47" data-doi="10.1016/j.bbapap.2012.06.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mesarich</span>,  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Schmitz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Tremouilhac</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">McGillivray</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Templeton</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Structure, dynamics and domain organization of the repeat protein Cin1 from the apple scab fungus</span>. <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1824</span>: <span class="cit-fpage">1118</span>–<span class="cit-lpage">1128</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.bbapap.2012.06.015</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.stitle%253DBiochimica%2Bet%2BBiophysica%2BActa.%2BProtein%2BStructure%2Band%2BMolecular%2BEnzymology%26rft.aulast%253DMesarich%26rft.auinit1%253DC.%2BH.%26rft.volume%253D1824%26rft.issue%253D10%26rft.spage%253D1118%26rft.epage%253D1128%26rft.atitle%253DStructure%252C%2Bdynamics%2Band%2Bdomain%2Borganization%2Bof%2Bthe%2Brepeat%2Bprotein%2BCin1%2Bfrom%2Bthe%2Bapple%2Bscab%2Bfungus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbapap.2012.06.015%26rft_id%253Dinfo%253Apmid%252F22771296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbapap.2012.06.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22771296&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.48" data-doi="10.1093/nar/20.22.6115"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Möller</span>,  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Bahnweg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Sandermann</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. H.</span> <span class="cit-name-surname">Geiger</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">A simple and efficient protocol for isolation of high molecular weight DNA from filamentous fungi, fruit bodies, and infected plant tissues</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">6115</span>–<span class="cit-lpage">6116</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/20.22.6115</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DMoller%26rft.auinit1%253DE.M.%26rft.volume%253D20%26rft.issue%253D22%26rft.spage%253D6115%26rft.epage%253D6116%26rft.atitle%253DA%2Bsimple%2Band%2Befficient%2Bprotocol%2Bfor%2Bisolation%2Bof%2Bhigh%2Bmolecular%2Bweight%2BDNA%2Bfrom%2Bfilamentous%2Bfungi%252C%2Bfruit%2Bbodies%252C%2Band%2Binfected%2Bplant%2Btissues%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F20.22.6115%26rft_id%253Dinfo%253Apmid%252F1461751%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/20.22.6115&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1461751&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992KB04900040&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.49" data-doi="10.1186/s12859-015-0795-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Monat</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Tranchant-Dubreuil</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kougbeadjo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Farcy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Ortega-Abboud</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> TOGGLE: toolbox for generic NGS analyses. BMC Bioinformatics 9:16:374.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12859-015-0795-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.volume%253D16%26rft.spage%253D374%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-015-0795-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-015-0795-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.50" data-doi="10.1186/1471-2164-13-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oyola</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Otto</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Gu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Maslen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Manske</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Optimizing illumina next-generation sequencing library preparation for extremely AT-biased genomes</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-13-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DOyola%26rft.auinit1%253DS.%2BO.%26rft.volume%253D13%26rft.spage%253D1%26rft.epage%253D1%26rft.atitle%253DOptimizing%2BIllumina%2Bnext-generation%2Bsequencing%2Blibrary%2Bpreparation%2Bfor%2Bextremely%2BAT-biased%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-1%26rft_id%253Dinfo%253Apmid%252F22214261%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22214261&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.51" data-doi="10.1128/MRA.01062-18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Passey</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Armitage</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Xu</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Annotated Draft Genome Sequence of the Apple Scab Pathogen <em>Venturia inaequalis</em></span>. <abbr class="cit-jnl-abbrev">Microbiol Resour Announc</abbr> <span class="cit-vol">7</span>. pii: e01062-18.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/MRA.01062-18</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2BResource%2BAnnouncements%26rft.stitle%253DMicrobiol%2BResour%2BAnnounc%26rft.aulast%253DPassey%26rft.auinit1%253DT.%2BA.%2BJ.%26rft.volume%253D7%26rft.issue%253D12%26rft.spage%253De01062-18%26rft.epage%253De01062-18%26rft.atitle%253DAnnotated%2BDraft%2BGenome%2BSequence%2Bof%2Bthe%2BApple%2BScab%2BPathogen%2BVenturia%2Binaequalis%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMRA.01062-18%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiZ2EiO3M6NToicmVzaWQiO3M6MTQ6IjcvMTIvZTAxMDYyLTE4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.52" data-doi="10.1038/nmeth.1701"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Petersen</span>,  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Brunak</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">von Heijne</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Nielsen</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">SignalP 4.0: discriminating signal peptides from transmembrane regions</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">785</span>–<span class="cit-lpage">786</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1701</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DPetersen%26rft.auinit1%253DT.%2BN.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253D785%26rft.epage%253D786%26rft.atitle%253DSignalP%2B4.0%253A%2Bdiscriminating%2Bsignal%2Bpeptides%2Bfrom%2Btransmembrane%2Bregions.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1701%26rft_id%253Dinfo%253Apmid%252F21959131%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1701&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21959131&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295358000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.53" data-doi="10.1186/1471-2105-9-392"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pierleoni</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. L.</span> <span class="cit-name-surname">Martelli</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Casadio</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">PredGPI: a GPI anchor predictor</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">392</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-9-392</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DPierleoni%26rft.auinit1%253DA.%26rft.volume%253D9%26rft.spage%253D392%26rft.epage%253D392%26rft.atitle%253DPredGPI%253A%2Ba%2BGPI-anchor%2Bpredictor.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-9-392%26rft_id%253Dinfo%253Apmid%252F18811934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-9-392&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18811934&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.54" data-doi="10.1086/519795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Neale</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Todd-Brown</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A. R.</span> <span class="cit-name-surname">Thomas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Ferreira</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">PLINK: A tool set for whole-genome association and population-based linkage analyses</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">559</span>–<span class="cit-lpage">575</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/519795</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D559%26rft.epage%253D575%26rft.atitle%253DPLINK%253A%2Ba%2Btool%2Bset%2Bfor%2Bwhole-genome%2Bassociation%2Band%2Bpopulation-based%2Blinkage%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F519795%26rft_id%253Dinfo%253Apmid%252F17701901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/519795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17701901&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.55" data-doi="10.1186/1471-2164-13-341"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quail</span>,  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Smith</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Coupland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. D.</span> <span class="cit-name-surname">Otto</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Harris</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A tale of three next generation sequencing platforms: comparison of Ion Torrent. Pacific Biosciences and Illumina MiSeq sequencers</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">341</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-13-341</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DQuail%26rft.auinit1%253DM.%26rft.volume%253D13%26rft.spage%253D341%26rft.epage%253D341%26rft.atitle%253DA%2Btale%2Bof%2Bthree%2Bnext%2Bgeneration%2Bsequencing%2Bplatforms%253A%2Bcomparison%2Bof%2BIon%2BTorrent%252C%2BPacific%2BBiosciences%2Band%2BIllumina%2BMiSeq%2Bsequencers.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-341%26rft_id%253Dinfo%253Apmid%252F22827831%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-341&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22827831&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.56" data-doi="10.1007/s00239-003-0007-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quesneville</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Nouaud</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Anxolabéhère</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Detection of new transposable element families in <em>Drosophila melanogaster</em> and <em>Anopheles gambiae</em> Genomes</span>. <abbr class="cit-jnl-abbrev">J. Mol. Evol.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">S50</span>–<span class="cit-lpage">S59</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00239-003-0007-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMol.%2BEvol.%26rft.volume%253D57%26rft.spage%253DS50%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00239-003-0007-2%26rft_id%253Dinfo%253Apmid%252F15008403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00239-003-0007-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15008403&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186396000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.57" data-doi="10.1371/journal.pcbi.0010022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quesneville</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Bergman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Andrieu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Autard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Nouaud</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Combined evidence annotation of transposable elements in genome sequences</span>. <abbr class="cit-jnl-abbrev">PLoS Comput. Biol.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">166</span>–<span class="cit-lpage">175</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pcbi.0010022</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BComput%2BBiol%26rft.aulast%253DQuesneville%26rft.auinit1%253DH.%26rft.volume%253D1%26rft.issue%253D2%26rft.spage%253D166%26rft.epage%253D175%26rft.atitle%253DCombined%2Bevidence%2Bannotation%2Bof%2Btransposable%2Belements%2Bin%2Bgenome%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.0010022%26rft_id%253Dinfo%253Apmid%252F16110336%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.0010022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16110336&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000234712600008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.58" data-doi="10.1038/ncomms1189"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rouxel</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Grandaubert</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Hane</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Hoede</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">van de Wouw</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Effector diversification within compartments of the <em>Leptosphaeria maculans</em> genome affected by Repeat-Induced Point mutations</span>. <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">202</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms1189</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BCommun%26rft.aulast%253DRouxel%26rft.auinit1%253DT.%26rft.volume%253D2%26rft.spage%253D202%26rft.epage%253D202%26rft.atitle%253DEffector%2Bdiversification%2Bwithin%2Bcompartments%2Bof%2Bthe%2BLeptosphaeria%2Bmaculans%2Bgenome%2Baffected%2Bby%2BRepeat-Induced%2BPoint%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms1189%26rft_id%253Dinfo%253Apmid%252F21326234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms1189&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21326234&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.59" data-doi="10.1080/0028825X.1975.10430352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Samuels</span>,  <span class="cit-name-given-names">G. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sivanesan</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title"><em>Venturia asperata sp. nov</em>. and its <em>Fusicladium</em> state on apple leaves</span>. <abbr class="cit-jnl-abbrev">N. Z. J. Bot.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">645</span>–<span class="cit-lpage">652</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/0028825X.1975.10430352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DN.%2BZ.%2BJ.%2BBot.%26rft.volume%253D13%26rft.spage%253D645%26rft_id%253Dinfo%253Adoi%252F10.1080%252F0028825X.1975.10430352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/0028825X.1975.10430352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.60" data-doi="10.3389/fpls.2015.00980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shiller</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">Van de Wouw</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">Taranto</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Bowen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Dubois</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A large family of <em>AvrLm6</em>-like genes in the apple and pear scab pathogens, <em>Venturia inaequalis</em> and <em>Venturia pirina</em></span>. <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">980</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2015.00980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D6%26rft.spage%253D980%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2015.00980%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2015.00980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.61" data-doi="10.1093/bioinformatics/btv351"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simão</span>,  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Waterhouse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Ioannidis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. V.</span> <span class="cit-name-surname">Kriventseva</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. M.</span> <span class="cit-name-surname">Zdobnov</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">BUSCO:assessing genome assembly and annotation completeness with single-copy orthologs</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">3210</span>–<span class="cit-lpage">3212</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btv351</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv351%26rft_id%253Dinfo%253Apmid%252F26059717%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv351&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26059717&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.62" data-doi="10.1515/znc-2005-1-220"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Piotrowski</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Gau</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Purification and partial characterization of an extracellular melanoprotein from the fungus <em>Venturia inaequalis</em></span>. <abbr class="cit-jnl-abbrev">Z. Naturforsch. C</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">109</span>–<span class="cit-lpage">115</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1515/znc-2005-1-220</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DZ.%2BNaturforsch.%2BC%26rft.volume%253D60%26rft.spage%253D109%26rft_id%253Dinfo%253Adoi%252F10.1515%252Fznc-2005-1-220%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1515/znc-2005-1-220&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-book" id="cit-9.8.2405.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sivanesan</span>,  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-source">The Taxonomy and Pathology of Venturia Species</span>, <span class="cit-publ-name">Bibliotheca Mycologica</span>, <span class="cit-publ-loc">Vaduz, Liechtenstein</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.64" data-doi="10.1093/bioinformatics/btn013"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stanke</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Diekhans</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Baertsch</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Haussler</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Using native and syntenically mapped cDNA alignments to improve <em>de novo</em> gene finding</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">637</span>–<span class="cit-lpage">644</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btn013</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn013%26rft_id%253Dinfo%253Apmid%252F18218656%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btn013&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18218656&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253746400006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.65" data-doi="10.1094/PHYTO-97-10-1222"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steiner</span>,  <span class="cit-name-given-names">U.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. C.</span> <span class="cit-name-surname">Oerke</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Localized melanization of appressoria is required for pathogenicity of <em>Venturia inaequalis</em></span>. <abbr class="cit-jnl-abbrev">Phytopathology</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">1222</span>–<span class="cit-lpage">1230</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1094/PHYTO-97-10-1222</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytopathology%26rft.stitle%253DPhytopathology%26rft.aulast%253DSteiner%26rft.auinit1%253DU.%26rft.volume%253D97%26rft.issue%253D10%26rft.spage%253D1222%26rft.epage%253D1230%26rft.atitle%253DLocalized%2BMelanization%2Bof%2BAppressoria%2BIs%2BRequired%2Bfor%2BPathogenicity%2Bof%2BVenturia%2Binaequalis.%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPHYTO-97-10-1222%26rft_id%253Dinfo%253Apmid%252F18943680%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PHYTO-97-10-1222&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18943680&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249634800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.66" data-doi="10.1093/molbev/msr121"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tamura</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Peterson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Peterson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Stecher</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Nei</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2731</span>–<span class="cit-lpage">2739</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msr121</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsr121%26rft_id%253Dinfo%253Apmid%252F21546353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msr121&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21546353&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295184200003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.67" data-doi="10.1093/gbe/evw121"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Testa</span>,  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. P.</span> <span class="cit-name-surname">Oliver</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Hane</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">OcculterCut: A comprehensive survey of AT-rich regions in fungal genomes</span>. <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">2044</span>–<span class="cit-lpage">2064</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/gbe/evw121</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevw121%26rft_id%253Dinfo%253Apmid%252F27289099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/gbe/evw121&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27289099&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.68" data-doi="doi:10.1002/0471250953.bi1110s43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van der Auwera</span>,  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. O.</span> <span class="cit-name-surname">Carneiro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Hartl</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Poplin</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Del Angel</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">From FastQ data to high confidence variant calls: the Genome Analysis Toolkit best practices pipeline</span>. <abbr class="cit-jnl-abbrev">Curr. Protoc. Bioinformatics</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BProtoc.%2BBioinformatics%26rft.volume%253D43%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252Fdoi%253A10.1002%252F0471250953.bi1110s43%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=doi:10.1002/0471250953.bi1110s43&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>,  <span class="cit-name-given-names">H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1956</span> <span class="cit-article-title">A convenient growth medium for <em>Neurospora crassa</em></span>. <abbr class="cit-jnl-abbrev">Microbial Genetics Bulletin</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">42</span>–<span class="cit-lpage">47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobial%2BGenetics%2BBulletin%26rft.volume%253D13%26rft.spage%253D42%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.70"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Watters</span>,  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. A.</span> <span class="cit-name-surname">Randall</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. S.</span> <span class="cit-name-surname">Margolin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. U.</span> <span class="cit-name-surname">Selker</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Stadier</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Action of repeat-induced point mutation on both strands of a duplex and on tandem duplications of various sizes in <em>Neurospora</em></span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">153</span>: <span class="cit-fpage">705</span>–<span class="cit-lpage">714</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWatters%26rft.auinit1%253DM.%2BK.%26rft.volume%253D153%26rft.issue%253D2%26rft.spage%253D705%26rft.epage%253D714%26rft.atitle%253DAction%2Bof%2BRepeat-Induced%2BPoint%2BMutation%2Bon%2BBoth%2BStrands%2Bof%2Ba%2BDuplex%2Band%2Bon%2BTandem%2BDuplications%2Bof%2BVarious%2BSizes%2Bin%2BNeurospora%26rft_id%253Dinfo%253Apmid%252F10511550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTUzLzIvNzA1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.71" data-doi="10.1094/PHYTO-07-17-0233-R"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Young</span>,  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. H.</span> <span class="cit-name-surname">Bock</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. D.</span> <span class="cit-name-surname">Charlton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Mattupalli</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Krom</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Evidence for Sexual Reproduction: Identification, Frequency, and Spatial Distribution of <em>Venturia effusa</em> (Pecan Scab) Mating Type Idiomorphs</span>. <abbr class="cit-jnl-abbrev">Phytopathology</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">837</span>–<span class="cit-lpage">846</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1094/PHYTO-07-17-0233-R</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhytopathology%26rft.volume%253D108%26rft.spage%253D837%26rft_id%253Dinfo%253Adoi%252F10.1094%252FPHYTO-07-17-0233-R%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1094/PHYTO-07-17-0233-R&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.72" data-doi="10.1101/gr.074492.107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zerbino</span>,  <span class="cit-name-given-names">D. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Birney</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Velvet: algorithms for de novo short read assembly using de Bruijn graphs</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">821</span>–<span class="cit-lpage">829</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.074492.107</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.074492.107%26rft_id%253Dinfo%253Apmid%252F18349386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE4LzUvODIxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI0MDUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2405.73" data-doi="10.1007/s13225-011-0141-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. W.</span> <span class="cit-name-surname">Crous</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. L.</span> <span class="cit-name-surname">Schoch</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. H.</span> <span class="cit-name-surname">Bahkali</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. D.</span> <span class="cit-name-surname">Guo</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A molecular, morphological and ecological re-appraisal of <em>Venturia</em>les -a new order of <em>Dothideomycetes</em></span>. <abbr class="cit-jnl-abbrev">Fungal Divers.</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">277</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s13225-011-0141-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DFungal%2BDivers%26rft.aulast%253DZhang%26rft.auinit1%253DY.%26rft.volume%253D51%26rft.issue%253D1%26rft.spage%253D249%26rft.epage%253D277%26rft.atitle%253DA%2Bmolecular%252C%2Bmorphological%2Band%2Becological%2Bre-appraisal%2Bof%2BVenturiales-a%2Bnew%2Border%2Bof%2BDothideomycetes.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs13225-011-0141-x%26rft_id%253Dinfo%253Apmid%252F22368534%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s13225-011-0141-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22368534&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2405.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/8/2405.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/8/2395" title="High Quality Draft Genome of Arogyapacha (Trichopus zeylanicus), an Important Medicinal Plant Endemic to Western Ghats of India" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/8/2415" title="Forward Genetic Screen for Caenorhabditis elegans Mutants with a Shortened Locomotor Healthspan" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/8"><span>Volume 9 Issue 8, August 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/8.cover-source.jpg?itok=cSNYBBSf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/8/2405" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-qbMnzHW9uBojATHdyZavNAKG-QpeO6_Aztfmd0PTZDo" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genome-report" class="" data-icon-position="" data-hide-link-title="0">Genome Report</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/8/2405" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/8/2405" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/8/2405" class="active" data-icon-position="" data-hide-link-title="0">Population Genome Sequencing of the Scab Fungal Species Venturia inaequalis, Venturia pirina, Venturia aucupariae and Venturia asperata</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/493769" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-E9r7wFvPEfnvWBXZy3sKE0TSfK1k4F84cr7c_0Feo-I" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/8/2405.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/8/2405" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-IYkJq4QnHEFLrvyGGenqEn9tTWCI0Kw3co8tROHEKqk" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/8/2405" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/8/2405.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493769" id="node493769--2" data-pisa="ggg;9/8/2405" data-pisa-master="ggg;g3.119.400047" data-apath="/ggg/9/8/2405.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Population Genome Sequencing of the Scab Fungal Species <em>Venturia inaequalis</em>, <em>Venturia pirina</em>, <em>Venturia aucupariae</em> and <em>Venturia asperata</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Bruno Le Cam</span>, <span class="highwire-citation-author" data-delta="1">Dan Sargent</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-5695-4557" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jérôme Gouzy</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-7124-3454" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Joëlle Amselem</span>, <span class="highwire-citation-author" data-delta="4">Marie-Noëlle Bellanger</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-2328-5769" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Olivier Bouchez</span>, <span class="highwire-citation-author" data-delta="6">Spencer Brown</span>, <span class="highwire-citation-author" data-delta="7">Valérie Caffier</span>, <span class="highwire-citation-author" data-delta="8">Marie De Gracia</span>, <span class="highwire-citation-author" data-delta="9">Robert Debuchy</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0003-0960-0312" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ludovic Duvaux</span>, <span class="highwire-citation-author" data-delta="11">Thibaut Payen</span>, <span class="highwire-citation-author" data-delta="12">Mélanie Sannier</span>, <span class="highwire-citation-author" data-delta="13">Jason Shiller</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="14"><a href="http://orcid.org/0000-0001-8152-6642" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jérôme Collemare</span> and <span class="highwire-citation-author" data-delta="15">Christophe Lemaire</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2405-2414;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400047 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493769--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bruno Le Cam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bruno%2BLe%2BCam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Le%20Cam%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABruno%2BLe%2BCam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bruno.le-cam@inra.fr" class="" data-icon-position="" data-hide-link-title="0">bruno.le-cam@inra.fr</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dan Sargent </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Research and Innovation Centre, Fondazione Edmund Mach, via Mach 1, 38010, San Michele All’adige, TN, Italy,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dan%2BSargent%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sargent%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADan%2BSargent%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jérôme Gouzy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratoire Interactions Plantes Micro-organismes [LIPM], Université de Toulouse, INRA, CNRS, Castanet-Tolosan, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%A9r%C3%B4me%2BGouzy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gouzy%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJ%25C3%25A9r%25C3%25B4me%2BGouzy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5695-4557" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jérôme Gouzy</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Joëlle Amselem </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Unité de Recherche Génomique Informatique [URGI], INRA, Université Paris-Saclay, 78026, Versailles, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jo%C3%ABlle%2BAmselem%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Amselem%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJo%25C3%25ABlle%2BAmselem%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7124-3454" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Joëlle Amselem</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Marie-Noëlle Bellanger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie-No%C3%ABlle%2BBellanger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bellanger%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarie-No%25C3%25ABlle%2BBellanger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Olivier Bouchez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>US1426, INRA, GeT-PlaGe, Genotoul, Castanet-Tolosan, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Olivier%2BBouchez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bouchez%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOlivier%2BBouchez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2328-5769" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Olivier Bouchez</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Spencer Brown </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>UMR 8298, Institute for Integrative Biology of the Cell (I2BC), CEA, CNRS,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Spencer%2BBrown%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brown%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASpencer%2BBrown%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Valérie Caffier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Val%C3%A9rie%2BCaffier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Caffier%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVal%25C3%25A9rie%2BCaffier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Marie De Gracia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie%2BDe%2BGracia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=De%20Gracia%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarie%2BDe%2BGracia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Robert Debuchy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>UMR 8621, Institut de Génétique et Microbiologie CNRS, Université Paris-Sud, 91405 Orsay, France, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BDebuchy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Debuchy%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BDebuchy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Ludovic Duvaux </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ludovic%2BDuvaux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Duvaux%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALudovic%2BDuvaux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0960-0312" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ludovic Duvaux</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Thibaut Payen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thibaut%2BPayen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Payen%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThibaut%2BPayen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Mélanie Sannier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M%C3%A9lanie%2BSannier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sannier%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM%25C3%25A9lanie%2BSannier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Jason Shiller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BShiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shiller%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJason%2BShiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Jérôme Collemare </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France,</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>Westerdijk Fungal Biodiversity Institute, 3584 CT, Utrecht, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%A9r%C3%B4me%2BCollemare%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Collemare%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJ%25C3%25A9r%25C3%25B4me%2BCollemare%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8152-6642" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jérôme Collemare</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Christophe Lemaire </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christophe%2BLemaire%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lemaire%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristophe%2BLemaire%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/8/2405" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/8/2405&amp;title=Population%20Genome%20Sequencing%20of%20the%20Scab%20Fungal%20Species%20Venturia%20inaequalis%2C%20Venturia%20pirina%2C%20Venturia%20aucupariae%20and%20Venturia%20asperata" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/8/2405&amp;title=Population%20Genome%20Sequencing%20of%20the%20Scab%20Fungal%20Species%20Venturia%20inaequalis%2C%20Venturia%20pirina%2C%20Venturia%20aucupariae%20and%20Venturia%20asperata" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/8/2405&amp;title=Population%20Genome%20Sequencing%20of%20the%20Scab%20Fungal%20Species%20Venturia%20inaequalis%2C%20Venturia%20pirina%2C%20Venturia%20aucupariae%20and%20Venturia%20asperata" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/8/2405&amp;text=Population%20Genome%20Sequencing%20of%20the%20Scab%20Fungal%20Species%20Venturia%20inaequalis%2C%20Venturia%20pirina%2C%20Venturia%20aucupariae%20and%20Venturia%20asperata" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/8/2405&amp;title=Population%20Genome%20Sequencing%20of%20the%20Scab%20Fungal%20Species%20Venturia%20inaequalis%2C%20Venturia%20pirina%2C%20Venturia%20aucupariae%20and%20Venturia%20asperata" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/8/2405&amp;t=Population%20Genome%20Sequencing%20of%20the%20Scab%20Fungal%20Species%20Venturia%20inaequalis%2C%20Venturia%20pirina%2C%20Venturia%20aucupariae%20and%20Venturia%20asperata" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/8/2405&amp;title=Population%20Genome%20Sequencing%20of%20the%20Scab%20Fungal%20Species%20Venturia%20inaequalis%2C%20Venturia%20pirina%2C%20Venturia%20aucupariae%20and%20Venturia%20asperata" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/8/2405&amp;title=Population%20Genome%20Sequencing%20of%20the%20Scab%20Fungal%20Species%20Venturia%20inaequalis%2C%20Venturia%20pirina%2C%20Venturia%20aucupariae%20and%20Venturia%20asperata" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/493769/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493769" id="node493769--3" data-pisa="ggg;9/8/2405" data-pisa-master="ggg;g3.119.400047" data-apath="/ggg/9/8/2405.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Population Genome Sequencing of the Scab Fungal Species <em>Venturia inaequalis</em>, <em>Venturia pirina</em>, <em>Venturia aucupariae</em> and <em>Venturia asperata</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Bruno Le Cam</span>, <span class="highwire-citation-author" data-delta="1">Dan Sargent</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-5695-4557" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jérôme Gouzy</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0001-7124-3454" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Joëlle Amselem</span>, <span class="highwire-citation-author" data-delta="4">Marie-Noëlle Bellanger</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-2328-5769" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Olivier Bouchez</span>, <span class="highwire-citation-author" data-delta="6">Spencer Brown</span>, <span class="highwire-citation-author" data-delta="7">Valérie Caffier</span>, <span class="highwire-citation-author" data-delta="8">Marie De Gracia</span>, <span class="highwire-citation-author" data-delta="9">Robert Debuchy</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0003-0960-0312" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Ludovic Duvaux</span>, <span class="highwire-citation-author" data-delta="11">Thibaut Payen</span>, <span class="highwire-citation-author" data-delta="12">Mélanie Sannier</span>, <span class="highwire-citation-author" data-delta="13">Jason Shiller</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="14"><a href="http://orcid.org/0000-0001-8152-6642" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Jérôme Collemare</span> and <span class="highwire-citation-author" data-delta="15">Christophe Lemaire</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2405-2414;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400047 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493769--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Bruno Le Cam </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bruno%2BLe%2BCam%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Le%20Cam%20B&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABruno%2BLe%2BCam%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:bruno.le-cam@inra.fr" class="" data-icon-position="" data-hide-link-title="0">bruno.le-cam@inra.fr</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dan Sargent </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Research and Innovation Centre, Fondazione Edmund Mach, via Mach 1, 38010, San Michele All’adige, TN, Italy,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dan%2BSargent%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sargent%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADan%2BSargent%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jérôme Gouzy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratoire Interactions Plantes Micro-organismes [LIPM], Université de Toulouse, INRA, CNRS, Castanet-Tolosan, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%A9r%C3%B4me%2BGouzy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gouzy%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJ%25C3%25A9r%25C3%25B4me%2BGouzy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5695-4557" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jérôme Gouzy</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Joëlle Amselem </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Unité de Recherche Génomique Informatique [URGI], INRA, Université Paris-Saclay, 78026, Versailles, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jo%C3%ABlle%2BAmselem%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Amselem%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJo%25C3%25ABlle%2BAmselem%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-7124-3454" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Joëlle Amselem</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Marie-Noëlle Bellanger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie-No%C3%ABlle%2BBellanger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bellanger%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarie-No%25C3%25ABlle%2BBellanger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Olivier Bouchez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>US1426, INRA, GeT-PlaGe, Genotoul, Castanet-Tolosan, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Olivier%2BBouchez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bouchez%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AOlivier%2BBouchez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-2328-5769" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Olivier Bouchez</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Spencer Brown </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>UMR 8298, Institute for Integrative Biology of the Cell (I2BC), CEA, CNRS,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Spencer%2BBrown%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Brown%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASpencer%2BBrown%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Valérie Caffier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Val%C3%A9rie%2BCaffier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Caffier%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVal%25C3%25A9rie%2BCaffier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Marie De Gracia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marie%2BDe%2BGracia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=De%20Gracia%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarie%2BDe%2BGracia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Robert Debuchy </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡‡</span>UMR 8621, Institut de Génétique et Microbiologie CNRS, Université Paris-Sud, 91405 Orsay, France, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Robert%2BDebuchy%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Debuchy%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARobert%2BDebuchy%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Ludovic Duvaux </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ludovic%2BDuvaux%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Duvaux%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALudovic%2BDuvaux%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0960-0312" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Ludovic Duvaux</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Thibaut Payen </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Thibaut%2BPayen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Payen%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AThibaut%2BPayen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Mélanie Sannier </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=M%C3%A9lanie%2BSannier%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sannier%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AM%25C3%25A9lanie%2BSannier%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Jason Shiller </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jason%2BShiller%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shiller%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJason%2BShiller%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-14"><div class="author-tooltip-name">Jérôme Collemare </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France,</div><div class='author-affiliation'><span class='nlm-sup'>§§</span>Westerdijk Fungal Biodiversity Institute, 3584 CT, Utrecht, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J%C3%A9r%C3%B4me%2BCollemare%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Collemare%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJ%25C3%25A9r%25C3%25B4me%2BCollemare%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8152-6642" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Jérôme Collemare</a></li></ul></div><div class="author-tooltip-15"><div class="author-tooltip-name">Christophe Lemaire </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>UMR 1345 Institut de Recherche en Horticulture et Semences (IRHS), INRA, Université d’Angers, Agrocampus-Ouest, SFR 4207 QuaSaV, 49071, Beaucouzé, France,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Christophe%2BLemaire%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lemaire%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChristophe%2BLemaire%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2405&amp;rft.epage=2414&amp;rft.atitle=Population%20Genome%20Sequencing%20of%20the%20Scab%20Fungal%20Species%20Venturia%20inaequalis%2C%20Venturia%20pirina%2C%20Venturia%20aucupariae%20and%20Venturia%20asperata&amp;rft.volume=9&amp;rft.issue=8&amp;rft.date=2019-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Le+Cam%2C+Bruno&amp;rft.au=Sargent%2C+Dan&amp;rft.au=Gouzy%2C+J%C3%A9r%C3%B4me&amp;rft.au=Amselem%2C+Jo%C3%ABlle&amp;rft.au=Bellanger%2C+Marie-No%C3%ABlle&amp;rft.au=Bouchez%2C+Olivier&amp;rft.au=Brown%2C+Spencer&amp;rft.au=Caffier%2C+Val%C3%A9rie&amp;rft.au=De+Gracia%2C+Marie&amp;rft.au=Debuchy%2C+Robert&amp;rft.au=Duvaux%2C+Ludovic&amp;rft.au=Payen%2C+Thibaut&amp;rft.au=Sannier%2C+M%C3%A9lanie&amp;rft.au=Shiller%2C+Jason&amp;rft.au=Collemare%2C+J%C3%A9r%C3%B4me&amp;rft.au=Lemaire%2C+Christophe"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/493769/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/493769/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/493769/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/493769/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/493769/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/493769/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/493769/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/493769/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/493769/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/493769/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/493769/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/493769/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496044" id="node496044" data-pisa="ggg;g3.119.400871v1" data-pisa-master="ggg;g3.119.400871" data-apath="/ggg/early/2020/03/17/g3.119.400871.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/17/g3.119.400871" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the <em>indica</em> Rice Variety IR64 Using Linked-Read Sequencing and Nanopore Sequencing</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496040" id="node496040" data-pisa="ggg;g3.119.400893v1" data-pisa-master="ggg;g3.119.400893" data-apath="/ggg/early/2020/03/12/g3.119.400893.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/12/g3.119.400893" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Understanding the Genetic Domestication History of the Jianchang Duck by Genotyping and Sequencing of Genomic Genes Under Selection</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="496037" id="node496037" data-pisa="ggg;g3.120.401071v1" data-pisa-master="ggg;g3.120.401071" data-apath="/ggg/early/2020/03/11/g3.120.401071.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/early/2020/03/11/g3.120.401071" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title"><em>De Novo</em> Genome Assembly of the Meadow Brown Butterfly, <em>Maniola jurtina</em></span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genome%20Report" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genome Report</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/8/2405" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/8/2405.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/8/2405.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/8/2405.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
