<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/8/2511" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/8/2511.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/8/2511.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/8/2511.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/8/2511/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/8/2511.atom" />
<meta name="HW.pisa" content="ggg;9/8/2511" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Molecular Traits of Long Non-protein Coding RNAs from Diverse Plant Species Show Little Evidence of Phylogenetic Relationships" />
<meta name="DC.Identifier" content="10.1534/g3.119.400201" />
<meta name="DC.Date" content="2019-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Simopoulos et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Long non-coding RNAs (lncRNAs) represent a diverse class of regulatory loci with roles in development and stress responses throughout all kingdoms of life. LncRNAs, however, remain under-studied in plants compared to animal systems. To address this deficiency, we applied a machine learning prediction tool, Classifying RNA by Ensemble Machine learning Algorithm (CREMA), to analyze RNAseq data from 11 plant species chosen to represent a wide range of evolutionary histories. Transcript sequences of all expressed and/or annotated loci from plants grown in unstressed (control) conditions were assembled and input into CREMA for comparative analyses. On average, 6.4% of the plant transcripts were identified by CREMA as encoding lncRNAs. Gene annotation associated with the transcripts showed that up to 99% of all predicted lncRNAs for Solanum tuberosum and Amborella trichopoda were missing from their reference annotations whereas the reference annotation for the genetic model plant Arabidopsis thaliana contains 96% of all predicted lncRNAs for this species. Thus a reliance on reference annotations for use in lncRNA research in less well-studied plants can be impeded by the near absence of annotations associated with these regulatory transcripts. Moreover, our work using phylogenetic signal analyses suggests that molecular traits of plant lncRNAs display different evolutionary patterns than all other transcripts in plants and have molecular traits that do not follow a classic evolutionary pattern. Specifically, GC content was the only tested trait of lncRNAs with consistently significant and high phylogenetic signal, contrary to high signal in all tested molecular traits for the other transcripts in our tested plant species." />
<meta name="DC.Contributor" content="Caitlin M. A. Simopoulos" />
<meta name="DC.Contributor" content="Elizabeth A. Weretilnyk" />
<meta name="DC.Contributor" content="G. Brian Golding" />
<meta name="article:published_time" content="2019-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Molecular Traits of Long Non-protein Coding RNAs from Diverse Plant Species Show Little Evidence of Phylogenetic Relationships" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Long non-coding RNAs (lncRNAs) represent a diverse class of regulatory loci with roles in development and stress responses throughout all kingdoms of life. LncRNAs, however, remain under-studied in plants compared to animal systems. To address this deficiency, we applied a machine learning prediction tool, Classifying RNA by Ensemble Machine learning Algorithm (CREMA), to analyze RNAseq data from 11 plant species chosen to represent a wide range of evolutionary histories. Transcript sequences of all expressed and/or annotated loci from plants grown in unstressed (control) conditions were assembled and input into CREMA for comparative analyses. On average, 6.4% of the plant transcripts were identified by CREMA as encoding lncRNAs. Gene annotation associated with the transcripts showed that up to 99% of all predicted lncRNAs for &lt;i&gt;Solanum tuberosum&lt;/i&gt; and &lt;i&gt;Amborella trichopoda&lt;/i&gt; were missing from their reference annotations whereas the reference annotation for the genetic model plant &lt;i&gt;Arabidopsis thaliana&lt;/i&gt; contains 96% of all predicted lncRNAs for this species. Thus a reliance on reference annotations for use in lncRNA research in less well-studied plants can be impeded by the near absence of annotations associated with these regulatory transcripts. Moreover, our work using phylogenetic signal analyses suggests that molecular traits of plant lncRNAs display different evolutionary patterns than all other transcripts in plants and have molecular traits that do not follow a classic evolutionary pattern. Specifically, GC content was the only tested trait of lncRNAs with consistently significant and high phylogenetic signal, contrary to high signal in all tested molecular traits for the other transcripts in our tested plant species.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/08/01" />
<meta name="citation_mjid" content="ggg;9/8/2511" />
<meta name="citation_id" content="9/8/2511" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/8/2511" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/8/2511.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/8/2511.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/8/2511.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400201" />
<meta name="citation_pmid" content="31235560" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2511" />
<meta name="citation_lastpage" content="2520" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Caitlin M. A. Simopoulos" />
<meta name="citation_author_institution" content="Department of Biology, McMaster University, Hamilton, Ontario, Canada, L8S 4K1" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9832-7797" />
<meta name="citation_author" content="Elizabeth A. Weretilnyk" />
<meta name="citation_author_institution" content="Department of Biology, McMaster University, Hamilton, Ontario, Canada, L8S 4K1" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0399-6232" />
<meta name="citation_author" content="G. Brian Golding" />
<meta name="citation_author_institution" content="Department of Biology, McMaster University, Hamilton, Ontario, Canada, L8S 4K1" />
<meta name="citation_author_email" content="golding@mcmaster.ca" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7575-0282" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_title=The Amborella Genome and the Evolution of Flowering Plants;citation_pages=1241089-1241089;citation_volume=342;citation_year=2013;citation_issue=6165;citation_pmid=24357323;citation_doi=10.1126/science.1241089" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=J. Banks;citation_author=T. Nishiyama;citation_author=M. Hasebe;citation_author=J. Bowman;citation_author=M. Gribskov;citation_title=The Selaginella genome identifies genetic changes associated with the evolution of vascular plants;citation_volume=332;citation_year=2011;citation_pmid=21551031;citation_doi=10.1126/science.1203810" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=S. Blomberg;citation_author=T. Garland;citation_author=A. R. Ives;citation_title=Testing for phylogenetic signal in comparative data: behavioral traits are more labile.;citation_pages=717-745;citation_volume=57;citation_year=2003;citation_issue=4;citation_pmid=12778543;citation_doi=10.1554/0014-3820(2003)057[0717:TFPSIC]2.0.CO;2" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data;citation_volume=30;citation_year=2014;citation_pmid=24695404;citation_doi=10.1093/bioinformatics/btu170" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=M. Bolger;citation_author=B. Arsova;citation_author=B. Usadel;citation_title=Plant genome and transcriptome annotations: from misconceptions to simple solutions;citation_pages=437-449;citation_volume=19;citation_year=2018" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=P. Cárdenas;citation_author=P. Sonawane;citation_author=J. Pollier;citation_author=R. VandenBossche;citation_author=V. Dewangan;citation_title=GAME9 regulates the biosynthesis of steroidal alkaloids and upstream isoprenoids in the plant mevalonate pathway;citation_pages=10654;citation_volume=7;citation_year=2016;citation_pmid=26876023;citation_doi=10.1038/ncomms10654" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=M. Champigny;citation_author=W. Sung;citation_author=V. Catana;citation_author=R. Salwan;citation_author=P. Summers;citation_title=RNA-Seq effectively monitors gene expression in Eutrema salsugineum plants growing in an extreme natural habitat and in controlled growth cabinet conditions;citation_pages=578;citation_volume=14;citation_year=2013;citation_doi=10.1186/1471-2164-14-578" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=C. Cheng;citation_author=V. Krishnakumar;citation_author=A. Chan;citation_author=F. Thibaud-Nissen;citation_author=S. Schobel;citation_title=Araport11: a complete reannotation of the Arabidopsis thaliana reference genome;citation_pages=789-804;citation_volume=89;citation_year=2017;citation_pmid=27862469;citation_doi=10.1111/tpj.13415" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=T. Derrien;citation_author=R. Johnson;citation_author=G. Bussotti;citation_author=A. Tanzer;citation_author=S. Djebali;citation_title=The GENCODE v7 catalog of human long noncoding RNAs: Analysis of their gene structure, evolution, and expression;citation_pages=1775-1789;citation_volume=22;citation_year=2012;citation_issue=9;citation_pmid=22955988;citation_doi=10.1101/gr.132159.111" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=J. Diniz-Filho;citation_title=Phylogenetic autocorrelation under distinct evolutionary processes.;citation_pages=1104-1109;citation_volume=55;citation_year=2001;citation_issue=6;citation_pmid=11475046;citation_doi=10.1111/j.0014-3820.2001.tb00630.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. Dobin;citation_author=C. Davis;citation_author=F. Schlesinger;citation_author=J. Drenkow;citation_author=C. Zaleski;citation_title=STAR: ultrafast universal RNA-seq aligner;citation_volume=29;citation_year=2013;citation_pmid=23104886;citation_doi=10.1093/bioinformatics/bts635" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. Fang;citation_author=L. Zhang;citation_author=J. Guo;citation_author=Y. Niu;citation_author=Y. Wu;citation_title=NONCODEV5: a comprehensive annotation database for long non-coding RNAs;citation_pages=D308-D314;citation_volume=46;citation_year=2017;citation_doi=10.1093/nar/gkx1107" />
<meta name="citation_reference" content="citation_title=PHYLIP (phylogeny inference package), version 3.5 c;citation_year=1993" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=M. Frank;citation_author=M. Scanlon;citation_title=Transcriptomic evidence for the evolution of shoot meristem function in sporophyte-dominant land plants through concerted selection of ancestral gametophytic and sporophytic genetic programs;citation_volume=32;citation_year=2015;citation_pmid=25371433;citation_doi=10.1093/molbev/msu303" />
<meta name="citation_reference" content="citation_journal_title=Syst. Zool.;citation_author=J. L. Gittleman;citation_author=M. Kot;citation_title=Adaptation: statistics and a null model for estimating phylogenetic effects;citation_volume=39;citation_year=1990;citation_doi=10.2307/2992183" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=Q. Gong;citation_author=P. Li;citation_author=S. Ma;citation_author=S. Indu Rupassara;citation_author=H. J. Bohnert;citation_title=Salinity stress adaptation competence in the extremophile Thellungiella halophila in comparison with its relative Arabidopsis thaliana.;citation_pages=826-839;citation_volume=44;citation_year= 2005;citation_issue=5;citation_pmid=16297073;citation_doi=10.1111/j.1365-313X.2005.02587.x" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=E. Gonzalez-Munoz;citation_author=A. Avendano-Vazquez;citation_author=R. Montes;citation_author=S. deFolter;citation_author=L. Andres-Hernandez;citation_title=The maize (Zea mays ssp. mays var. B73) genome encodes 33 members of the purple acid phosphatase family;citation_pages=341;citation_volume=6;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Plant, Cell, and Environment (Print);citation_journal_abbrev=Plant, Cell, and Environment (Print);citation_author=M. Griffith;citation_author=M. Timonin;citation_author=A. Wong;citation_author=G. Gray;citation_author=S. Akhter;citation_title=Thellungiella: an Arabidopsis-related model plant adapted to cold temperatures.;citation_pages=529-538;citation_volume=30;citation_year=2007;citation_issue=5;citation_pmid=17407531;citation_doi=10.1111/j.1365-3040.2007.01653.x" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=W. Haerty;citation_author=C. Ponting;citation_title=Unexpected selection to retain high GC content and splicing enhancers within exons of multiexonic lncRNA loci;citation_volume=21;citation_year=2015;citation_pmid=25589248;citation_doi=10.1261/rna.047324.114" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_author=T. Hansen;citation_title=Stabilizing selection and the comparative analysis of adaptation;citation_pages=1341-1351;citation_volume=51;citation_year=1997;citation_pmid=28568616;citation_doi=10.2307/2411186" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=L. Harmon;citation_author=J. Weir;citation_author=C. Brock;citation_author=R. Glor;citation_author=W. Challenger;citation_title=Geiger: investigating evolutionary radiations;citation_volume=24;citation_year=2008;citation_pmid=18006550;citation_doi=10.1093/bioinformatics/btm538" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=J. Harrow;citation_author=A. Frankish;citation_author=J. M. Gonzalez;citation_author=E. Tapanari;citation_author=M. Diekhans;citation_title=GENCODE: The reference human genome annotation for The ENCODE Project;citation_pages=1760-1774;citation_volume=22;citation_year=2012;citation_issue=9;citation_pmid=22955987;citation_doi=10.1101/gr.135350.111" />
<meta name="citation_reference" content="citation_journal_title=Cell Reports;citation_author=H. Hezroni;citation_author=D. Koppstein;citation_author=M. Schwartz;citation_author=A. Avrutin;citation_author=D. Bartel;citation_title=Principles of long noncoding RNA evolution derived from direct comparison of transcriptomes in 17 species;citation_pages=1110-1122;citation_volume=11;citation_year=2015;citation_pmid=25959816;citation_doi=10.1016/j.celrep.2015.04.023" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=R. Jackson;citation_author=L. Kroehling;citation_author=A. Khitun;citation_author=W. Bailis;citation_author=A. Jarret;citation_title=The translation of non-canonical open reading frames controls mucosal immunity;citation_volume=564;citation_year=2018;citation_doi=10.1038/s41586-018-0794-7" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=A. James;citation_author=A. Jayasena;citation_author=J. Zhang;citation_author=O. Berkowitz;citation_author=D. Secco;citation_title=Evidence for Ancient Origins of Bowman-Birk Inhibitors from Selaginella moellendorffii;citation_volume=29;citation_year=2017;citation_pmid=28298518;citation_doi=10.1105/tpc.16.00831" />
<meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci.;citation_author=J. Kamilar;citation_author=N. Cooper;citation_title=Phylogenetic signal in primate behaviour, ecology and life history;citation_volume=368;citation_year=2013;citation_pmid=23569289;citation_doi=10.1098/rstb.2012.0341" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=P. Kapranov;citation_author=J. Cheng;citation_author=S. Dike;citation_author=D. A. Nix;citation_author=R. Duttagupta;citation_title=RNA maps reveal new RNA classes and a possible function for pervasive transcription;citation_volume=316;citation_year=2007;citation_pmid=17510325;citation_doi=10.1126/science.1138341" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=A. Kapusta;citation_author=C. Feschotte;citation_title=Volatile evolution of long noncoding RNA repertoires: mechanisms and biological implications;citation_pages=439-452;citation_volume=30;citation_year=2014;citation_pmid=25218058;citation_doi=10.1016/j.tig.2014.08.004" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=K. Katoh;citation_author=K. Misawa;citation_author=K. Kuma;citation_author=T. Miyata;citation_title=MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform;citation_pages=3059-3066;citation_volume=30;citation_year=2002;citation_issue=14;citation_pmid=12136088;citation_doi=10.1093/nar/gkf436" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=Y. Kazachkova;citation_author=G. Eshel;citation_author=P. Pantha;citation_author=J. Cheeseman;citation_author=M. Dassanayake;citation_title=Halophytism: What Have We Learnt From Arabidopsis thaliana Relative Model Systems?;citation_volume=178;citation_year=2018;citation_pmid=30237204;citation_doi=10.1104/pp.18.00863" />
<meta name="citation_reference" content="citation_journal_title=Ecol. Evol.;citation_author=F. Keck;citation_author=F. Rimet;citation_author=A. Bouchez;citation_author=A. Franc;citation_title=phylosignal: an R package to measure, test, and explore the phylogenetic signal;citation_pages=2774-2780;citation_volume=6;citation_year=2016;citation_doi=10.1002/ece3.2051" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=D. Lang;citation_author=K. Ullrich;citation_author=F. Murat;citation_author=J. Fuchs;citation_author=J. Jenkins;citation_title=The Physcomitrella patens chromosome-scale assembly reveals moss genome structure and evolution;citation_pages=515-533;citation_volume=93;citation_year=2018;citation_doi=10.1111/tpj.13801" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=W. Liu;citation_author=C. Cheng;citation_author=Y. Lin;citation_author=X. XuHan;citation_author=Z. Lai;citation_title=Genome-wide identification and characterization of mRNAs and lncRNAs involved in cold stress in the wild banana (Musa itinerans);citation_pages=e0200002;citation_volume=13;citation_year= 2018;citation_doi=10.1371/journal.pone.0200002" />
<meta name="citation_reference" content="citation_journal_title=Physiol. Plant.;citation_author=M. J. MacLeod;citation_author=J. Dedrick;citation_author=C. Ashton;citation_author=W. W. Sung;citation_author=M. J. Champigny;citation_title=Exposure of two Eutrema salsugineum (Thellungiella salsuginea) accessions to water deficits reveals different coping strategies in response to drought;citation_volume=155;citation_year=2015;citation_doi=10.1111/ppl.12316" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Merchant;citation_author=S. Prochnik;citation_author=O. Vallon;citation_author=E. Harris;citation_author=S. Karpowicz;citation_title=The Chlamydomonas Genome Reveals the Evolution of Key Animal and Plant Functions;citation_pages=245-250;citation_volume=318;citation_year=2007;citation_issue=5848;citation_pmid=17932292;citation_doi=10.1126/science.1143609" />
<meta name="citation_reference" content="citation_journal_title=BMC Plant Biol.;citation_author=S. Mohammadin;citation_author=P. Edger;citation_author=J. Pires;citation_author=M. Schranz;citation_title=Positionally-conserved but sequence-diverged: identification of long non-coding RNAs in the Brassicaceae and Cleomaceae;citation_pages=217;citation_volume=15;citation_year=2015;citation_pmid=26362138;citation_doi=10.1186/s12870-015-0603-5" />
<meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B;citation_author=P. A. Moran;citation_title=The interpretation of statistical maps;citation_pages=243-251;citation_volume=10;citation_year=1948" />
<meta name="citation_reference" content="citation_journal_title=Crit. Rev. Biotechnol.;citation_author=N. Nejat;citation_author=N. Mantri;citation_title=Emerging roles of long non-coding RNAs in plant response to biotic and abiotic stresses;citation_pages=93-105;citation_volume=38;citation_year=2018;citation_doi=10.1080/07388551.2017.1312270" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=A. Nelson;citation_author=E. Forsythe;citation_author=U. Devisetty;citation_author=D. Clausen;citation_author=A. Haug-Batzell;citation_title=A Genomic Analysis of Factors Driving lincRNA Diversification: Lessons from Plants;citation_volume=6;citation_year=2016;citation_pmid=27440919;citation_doi=10.1534/g3.116.030338" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=F. Niazi;citation_author=S. Valadkhan;citation_title=Computational analysis of functional long noncoding RNAs reveals lack of peptide-coding capacity and parallels with 3′ UTRs;citation_volume=18;citation_year=2012;citation_pmid=22361292;citation_doi=10.1261/rna.029520.111" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. Ouyang;citation_author=W. Zhu;citation_author=J. Hamilton;citation_author=H. Lin;citation_author=M. Campbell;citation_title=The TIGR Rice Genome Annotation Resource: improvements and new features;citation_volume=35;citation_year=2007;citation_pmid=17145706;citation_doi=10.1093/nar/gkl976" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Pagel;citation_title=Inferring the historical patterns of biological evolution.;citation_pages=877-884;citation_volume=401;citation_year=1999;citation_issue=6756;citation_pmid=10553904;citation_doi=10.1038/44766" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=N. Panchy;citation_author=G. Wu;citation_author=L. Newton;citation_author=C. Tsai;citation_author=J. Chen;citation_title=Prevalence, evolution, and cis-regulation of diel transcription in Chlamydomonas reinhardtii;citation_volume=4;citation_year=2014;citation_pmid=25354782;citation_doi=10.1534/g3.114.015032" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=E. Paradis;citation_author=K. Schliep;citation_title=ape 5.0: an environment for modern phylogenetics and evolutionary analyses in R;citation_volume=35;citation_year=2019;citation_pmid=27016728;citation_doi=10.1093/bioinformatics/bty633" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=M. Pertea;citation_author=G. Pertea;citation_author=C. Antonescu;citation_author=T. Chang;citation_author=J. Mendell;citation_title=StringTie enables improved reconstruction of a transcriptome from RNA-seq reads;citation_pages=290-295;citation_volume=33;citation_year=2015;citation_pmid=25690850;citation_doi=10.1038/nbt.3122" />
<meta name="citation_reference" content="citation_journal_title=Syst. Biol.;citation_author=L. Revell;citation_author=L. Harmon;citation_author=D. Collar;citation_title=Phylogenetic signal, evolutionary process, and rate;citation_volume=57;citation_year=2008;citation_pmid=18709597;citation_doi=10.1080/10635150802302427" />
<meta name="citation_reference" content="citation_journal_title=Nat. Methods;citation_author=E. Rivas;citation_author=J. Clements;citation_author=S. Eddy;citation_title=A statistical test for conserved RNA structure shows lack of evidence for structure in lncRNAs;citation_pages=45-48;citation_volume=14;citation_year=2017;citation_doi=10.1038/nmeth.4066" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=H. Saini;citation_author=A. Enright;citation_author=S. Griffiths-Jones;citation_title=Annotation of mammalian primary microRNAs.;citation_pages=564-564;citation_volume=9;citation_year=2008;citation_pmid=19038026;citation_doi=10.1186/1471-2164-9-564" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. Schnable;citation_author=D. Ware;citation_author=R. Fulton;citation_author=J. Stein;citation_author=F. Wei;citation_title=The B73 Maize Genome: Complexity, Diversity, and Dynamics;citation_pages=1112-1115;citation_volume=326;citation_year=2009;citation_issue=5956;citation_pmid=19965430;citation_doi=10.1126/science.1178534" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=S. Sharma;citation_author=D. Bolser;citation_author=J. deBoer;citation_author=M. Sonderkaer;citation_author=W. Amoros;citation_title=Construction of reference chromosome-scale pseudomolecules for potato: integrating the potato genome with genetic and physical maps;citation_volume=3;citation_year=2013;citation_pmid=24062527;citation_doi=10.1534/g3.113.007153" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=P. Shuai;citation_author=D. Liang;citation_author=S. Tang;citation_author=Z. Zhang;citation_author=C. Ye;citation_title=Genome-wide identification and functional prediction of novel and drought-responsive lincRNAs in Populus trichocarpa;citation_volume=65;citation_year=2014;citation_pmid=24948679;citation_doi=10.1093/jxb/eru256" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=C. Simopoulos;citation_author=E. Weretilnyk;citation_author=G. Golding;citation_title=Prediction of plant lncRNA by ensemble machine learning classifiers;citation_pages=316;citation_volume=19;citation_year=2018;citation_doi=10.1186/s12864-018-4665-2" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell Environ.;citation_author=H. Sprenger;citation_author=C. Kurowsky;citation_author=R. Horn;citation_author=A. Erban;citation_author=S. Seddig;citation_title=The drought response of potato reference cultivars with contrasting tolerance;citation_pages=2370-2389;citation_volume=39;citation_year=2016;citation_doi=10.1111/pce.12780" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=T. Taji;citation_author=M. Seki;citation_author=M. Satou;citation_author=T. Sakurai;citation_author=M. Kobayashi;citation_title=Comparative genomics in salt tolerance between Arabidopsis and Arabidopsis-related halophyte salt cress using Arabidopsis microarray;citation_volume=135;citation_year=2004;citation_pmid=15247402;citation_doi=10.1104/pp.104.039909" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=The tomato genome sequence provides insights into fleshy fruit evolution.;citation_pages=635-641;citation_volume=485;citation_year=2012;citation_issue=7400;citation_pmid=22660326;citation_doi=10.1038/nature11119" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell Environ.;citation_author=V. Velasco;citation_author=J. Mansbridge;citation_author=S. Bremner;citation_author=K. Carruthers;citation_author=P. Summers;citation_title=Acclimation of the crucifer Eutrema salsugineum to phosphate limitation is associated with constitutively high expression of phosphate-starvation genes;citation_pages=1818-1834;citation_volume=39;citation_year=2016;citation_doi=10.1111/pce.12750" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=D. Wang;citation_author=Z. Qu;citation_author=L. Yang;citation_author=Q. Zhang;citation_author=Z. Liu;citation_title=Transposable elements (TEs) contribute to stress-related long intergenic noncoding RNAs in plants;citation_volume=90;citation_year=2017;citation_doi=10.1111/tpj.13481" />
<meta name="citation_reference" content="citation_title=Long Non Coding RNA Biology;citation_author=H. V. Wang;citation_author=J. A. Cheksnova;citation_title=Long Noncoding RNAs in Plants;citation_year=2017;citation_doi=10.1007/978-981-10-5203-3_5" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=O. Wilkins;citation_author=C. Hafemeister;citation_author=A. Plessis;citation_author=M. Holloway-Phillips;citation_author=G. Pham;citation_title=EGRINs (Environmental Gene Regulatory Influence Networks) in Rice That Function in the Response to Water Deficit, High Temperature, and Agricultural Environments;citation_volume=28;citation_year=2016;citation_pmid=27655842;citation_doi=10.1105/tpc.16.00158" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=C. Wong;citation_author=Y. Li;citation_author=A. Labbe;citation_author=D. Guevara;citation_author=P. Nuin;citation_title=Transcriptional profiling implicates novel interactions between abiotic stress and hormonal responses in Thellungiella, a close relative of Arabidopsis;citation_volume=140;citation_year=2006;citation_pmid=16500996;citation_doi=10.1104/pp.105.070508" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=H. Woo;citation_author=H. Koo;citation_author=J. Kim;citation_author=H. Jeong;citation_author=J. Yang;citation_title=Programming of Plant Leaf Senescence with Temporal and Inter-Organellar Coordination of Transcriptome in Arabidopsis;citation_volume=171;citation_year=2016;citation_pmid=26966169;citation_doi=10.1104/pp.15.01929" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=L. Xiao;citation_author=G. Yang;citation_author=L. Zhang;citation_author=X. Yang;citation_author=S. Zhao;citation_title=The resurrection genome of Boea hygrometrica: A blueprint for survival of dehydration;citation_volume=112;citation_year=2015;citation_pmid=25902549;citation_doi=10.1073/pnas.1505811112" />
<meta name="citation_reference" content="citation_journal_title=BMC Plant Biol.;citation_author=Q. Xu;citation_author=Z. Song;citation_author=C. Zhu;citation_author=C. Tao;citation_author=L. Kang;citation_title=Systematic comparison of lncRNAs with protein coding mRNAs in population expression and their response to environmental change;citation_pages=42;citation_volume=17;citation_year=2017;citation_doi=10.1186/s12870-017-0984-8" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=R. Yang;citation_author=D. Jarvis;citation_author=H. Chen;citation_author=M. Beilstein;citation_author=J. Grimwood;citation_title=The Reference Genome of the Halophytic Plant Eutrema salsugineum;citation_pages=46;citation_volume=4;citation_year=2013;citation_pmid=23518688;citation_doi=10.3389/fpls.2013.00046" />
<meta name="citation_reference" content="citation_journal_title=Plant Direct;citation_author=J. Yin;citation_author=M. J. Gosney;citation_author=B. P. Dilkes;citation_author=M. V. Mickelbart;citation_title=Dark period transcriptomic and metabolic profiling of two diverse Eutrema salsugineum accessions;citation_pages=00032e;citation_volume=2;citation_year=2018;citation_doi=10.1002/pld3.32" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=X. Zhao;citation_author=J. Li;citation_author=B. Lian;citation_author=H. Gu;citation_author=Y. Li;citation_title=Global identification of Arabidopsis lncRNAs reveals the regulation of MAF4 by a natural antisense RNA;citation_pages=5056;citation_volume=9;citation_year=2018;citation_doi=10.1038/s41467-018-07500-7" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell Physiol.;citation_author=Y. Zhu;citation_author=B. Wang;citation_author=J. Phillips;citation_author=Z. Zhang;citation_author=H. Du;citation_title=Global Transcriptome Analysis Reveals Acclimation-Primed Processes Involved in the Acquisition of Desiccation Tolerance in Boea hygrometrica;citation_volume=56;citation_year=2015;citation_pmid=25907569;citation_doi=10.1093/pcp/pcv059" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Molecular Traits of Long Non-protein Coding RNAs from Diverse Plant Species Show Little Evidence of Phylogenetic Relationships" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8.cover-source.jpg" />
<meta name="twitter:description" content="Long non-coding RNAs (lncRNAs) represent a diverse class of regulatory loci with roles in development and stress responses throughout all kingdoms of life. LncRNAs, however, remain under-studied in plants compared to animal systems. To address this deficiency, we applied a machine learning prediction tool, Classifying RNA by Ensemble Machine learning Algorithm (CREMA), to analyze RNAseq data from 11 plant species chosen to represent a wide range of evolutionary histories. Transcript sequences of all expressed and/or annotated loci from plants grown in unstressed (control) conditions were assembled and input into CREMA for comparative analyses. On average, 6.4% of the plant transcripts were identified by CREMA as encoding lncRNAs. Gene annotation associated with the transcripts showed that up to 99% of all predicted lncRNAs for Solanum tuberosum and Amborella trichopoda were missing from their reference annotations whereas the reference annotation for the genetic model plant Arabidopsis thaliana contains 96% of all predicted lncRNAs for this species. Thus a reliance on reference annotations for use in lncRNA research in less well-studied plants can be impeded by the near absence of annotations associated with these regulatory transcripts. Moreover, our work using phylogenetic signal analyses suggests that molecular traits of plant lncRNAs display different evolutionary patterns than all other transcripts in plants and have molecular traits that do not follow a classic evolutionary pattern. Specifically, GC content was the only tested trait of lncRNAs with consistently significant and high phylogenetic signal, contrary to high signal in all tested molecular traits for the other transcripts in our tested plant species." />
<meta name="og-title" property="og:title" content="Molecular Traits of Long Non-protein Coding RNAs from Diverse Plant Species Show Little Evidence of Phylogenetic Relationships" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/8/2511" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Long non-coding RNAs (lncRNAs) represent a diverse class of regulatory loci with roles in development and stress responses throughout all kingdoms of life. LncRNAs, however, remain under-studied in plants compared to animal systems. To address this deficiency, we applied a machine learning prediction tool, Classifying RNA by Ensemble Machine learning Algorithm (CREMA), to analyze RNAseq data from 11 plant species chosen to represent a wide range of evolutionary histories. Transcript sequences of all expressed and/or annotated loci from plants grown in unstressed (control) conditions were assembled and input into CREMA for comparative analyses. On average, 6.4% of the plant transcripts were identified by CREMA as encoding lncRNAs. Gene annotation associated with the transcripts showed that up to 99% of all predicted lncRNAs for Solanum tuberosum and Amborella trichopoda were missing from their reference annotations whereas the reference annotation for the genetic model plant Arabidopsis thaliana contains 96% of all predicted lncRNAs for this species. Thus a reliance on reference annotations for use in lncRNA research in less well-studied plants can be impeded by the near absence of annotations associated with these regulatory transcripts. Moreover, our work using phylogenetic signal analyses suggests that molecular traits of plant lncRNAs display different evolutionary patterns than all other transcripts in plants and have molecular traits that do not follow a classic evolutionary pattern. Specifically, GC content was the only tested trait of lncRNAs with consistently significant and high phylogenetic signal, contrary to high signal in all tested molecular traits for the other transcripts in our tested plant species." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8.cover-source.jpg" />
    <title>Molecular Traits of Long Non-protein Coding RNAs from Diverse Plant Species Show Little Evidence of Phylogenetic Relationships | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"xNpSAwpmGFZgoohBwguwxPlOET-GeQKx9MOtEOoKwjs","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"493746","apath":"\/ggg\/9\/8\/2511.atom","pisa":"ggg;9\/8\/2511","ac":{"\/ggg\/9\/8\/2511.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/8\/2511.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/8\/2511"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/8\/2511"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/8\/2511":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022er9qjFibm72vI0coo5QmeP1y1doJ+glqG5sy7C5zmlJIhlRTdTGNMsHuvv3kM05mf30nolSMkLOMXH1jdGEt1903itnqmbYHJlAlM8onSQxk+4fiKpkQV1h6llcuuqNC8SLBYjb871gPm+6+qrxDD+bmY69tCXXZ5NXMhalSJkuOFnMSm3GGbyYbRE7qD9hdRCv5g+kto2E74x9UjNYCAoOE9SFX98BaHaoFbnKbqhTY3\\\\\\\/I2TokBI+OrGzN+Fc+XL90xTUCvVaasrOwei6mHEYRo8isd3M+r7DUmtNRl+qXL8+mq0bZD4RAdyKxeHPx56puMFCqbRdCKfEG2pXIcmEv3V2y4bPY2ptSsQzlAOHabWe7hpRCPiAxW5QdzI94UATatZarG+q1oXrtRq45No3zrQ6T4TUfcYGCyOh5QMgF3ksyMk\\\\\\\/6BeNPmjLnDmyxQqX\\\\\\\/tXlUdjYIgR55GFzv\\\\\\\/GagqTbgHjlrEw4IHlDiOPcQ0G0HLt\\\\\\\/9yPIUOxUUFPGhLKfDsYftNOIxJiSbDxj\\\\\\\/WPpnnk5tYhdzTY12R+ZrAYT5m7rKocoaNmB8UKuHpzsvR7xAzWtFHGdYzNX2SHET1FangpTfIDI\\\\\\\/xsopCaBq0hKQha32UubG8xn0QW1wDNZg87yN5FPTVH9QWIPnjmr6tQxval49gloviAI+rs6X3Fqqk\\\\\\\/UDe9AC\\\\\\\/joZKy8uENsGAskxmyDc0KGAZuRAx3poF8iczLHhVBi+en9V7i1Rw1diVOmhSkfMH61tIHugUZc0Wq\\\\\\\/ZtlaDfw3GT0FS4vcx3BAKfx3hMs4kOoWpwlEIowVaskDyILLFUYzD4lLWRvJZYKzbvyFHOCUxZxChAF6e1H5qz5q1mrCsdZevXTmElvZlAwK4Ya4p+J00Ns7tyUI\\\\\\\/TNj1Ai5mhpW3YQwPHefnnqo0zR4vOD4kaJ21O5PO035DZ+gK\\\\\\\/dNL8auavVhYi1BiXmSmPORKRkUIc0ZTRajLKUCft5fUWC4Oc1TYO4smODLL8UaBG\\\\\\\/ejAyJ55rtaikFkDG1Kam7m83zzqCbPdXqXt\\\\\\\/VKhVgVtRwEbtA+r+3CbtB8DDA5NDxlK\\\\\\\/\\\\\\\/G2cq1mpiWCyre87DpI3CS0Z\\\\\\\/FlE8TfnmIu3PsxGNr9K2qORzLdofN5L8FaDVdE1LfubsrZZcXCLKMFRTP28vU0g3kAoimdpbQ2Apcod9ffcmB8\\\\\\\/JRyWHKn4UZNrthsho8SfgQZxUgm41J0USPmN2E2j4Q3oGNjU8kox7hS+eIIx2669a9nv79DPZi2Qre6dEsHIs5JPJIU\\\u0022,\\\u0022iv\\\u0022:\\\u00221DdiX5IXphcGw2rlofHD\\\\\\\/w==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00220fb740a5ba3c2232c3e0c6c6a0b4e66cd96ac9bc0fcddc93f3ec897dd49a81af\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LUcNmcdtCKeP3dUXGGYYnp19p5eg35QT6WX5D0ZqB8kk+H5vQOU+o78NvLH1QcZXgOiHNKS9Muw3lG\\\\\\\/hGcnS3VbwI+NUHJKV2DyF+l3E6vwfSH7eyS24VMjl0n\\\\\\\/wv5mOvK5oUcsVaLse5ONNxvxH4ofH5pUb8IatkMa0UMIQppILcsAk7SZ9v7SP85r0I2KJiSa5dmvzqjbSqzLbgwfz3iWrQKgMMc77O4onPBb9NEJ7TZQMgV1G4kBGK1iFi1DY\\\\\\\/QKfpChAMpgcZz6HHc8iXtl66WtKRkaAYj53bkNFADM4knqbxJqLOjs\\\\\\\/UVaDpk1QCcOii88rJkyQJK8WfmVp+dxTxANwG+WZ5VVTPz+Fv+KHUwAo\\\\\\\/ycAe5H6axIsnv6dsmPrG3f8UeldHyYFsYQbEIBBMAdXAntqkzPSMOm\\\\\\\/Eg9JX7vw6RQNOtTIXMmU5v4wFQLD3dfOUqxHAHFaTMtAzWJI4hCLQYHNNH0QqN7A6bgB5hZLG\\\\\\\/a6sS8v0CP+tts\\\\\\\/xhp15DrDaqlaf+LZw7fgfsGz2kt++7dOBSSh0+S+ZKst6DnHoytPUbD1w6R+pym+0cqZqZ33Dq57Q90C7PCPo3FjAJxC4TOw1OYqlgiuI4iV2JMFlIwo0pVmzxDNpOQRsup9V252G2Tsqn8xiyYoaNTpzHAYEckLdkLEJrCGkrEu5f3lL9Zg8NaXOdcELm\\\\\\\/dgtm4gLJuG8g3eSuVigKc4sWGlD04OizwJNXQeXWX0NoJhwf53PpkAVtfPbbsV3X059aCel+kzR7hOd88\\\\\\\/HCfWgNAJdvrLgImPVffU4tftP5cOIhiD3qRblFqot4KmpHM3Hqw3L5PuEXeoqZrNP8CiLKqbelKsEKDUn0JOP9HOsmBhErLJOCtKIOnNPMTdMxm7qb49g3r6+ZjWgQRkoyCXFNYC61I0JP2qLfs8pmemhrJKJm0AAlxK9HlzA32wPSLMJZxV+pWES4KfOiis9Pve5yFayFkCpwTOxiSvCx190l7RaNbovVOztVpyINAfRQMXxploZLDN4udjsoS4dnm0nQnamRduHevHTnI1wNAG7JF+VmeyOeI38t2fXCrY3DkyemNJptJnEsX8gbWBUPUzsB1LQFm6snBou1y1hAPgZEbSTEG3HM6tZUdCX+6A\\\\\\\/UD7cF4Fk\\\\\\\/To5UdH+PED\\\\\\\/ziYVtWimDC5CJZtNS++yss\\\\\\\/gbJDoYMQg4OESy9Onst+VODIXJABmH0GXXvHhdsQZJ4iTCuhWL1YdUsFJDL8kEzuaiyN31wc91O4Jj+YYg0\\\\\\\/niz1ehVydBXGmESYJyE+5HHuVhVYWZx3u1\\\\\\\/Qu2SHCG2L99OZ\\\\\\\/DGzVMBH7Q72WL102Dfq3jhg3+WU6oTgRjj7sqB1BUg9VnuO3h5lactfKjkenbg+IoimBgQGcE2nI\\\\\\\/epdmDUT77zGBCuMcEiEYQGmZgIiDaZHdWalGjmZoW00gwDXGVDpCpT8GMElWjvjaetHLZ7oWqzYX8NdUOqUOBmjnDesPud\\\\\\\/FuO1SThtQCFUnkKDSh1Kz74W+6uI6+4xzIuPHuQZ11pDvooVaezJXWOmev7iZYabjOr\\\\\\\/QHLpFq4gZs0nMYUUbxWwlract\\\\\\\/Lst7ulK9pjHHNIKIApZp+tGyr87zJyaKG3rvYdmzI1TLljO82O+7a+BAlya3fgk30T6BDnL8pUQNwOt\\\\\\\/SUVFfTCwdM8wVrDQs+rydlq62JmpeZVNFFUil+Dumjlib+xbsDRuzFkMyxeuZqsTrHPX22mon9WtvMXqj5seQ1HbR\\\\\\\/1KPYTKb0zOuUTnVGi15wBs\\\\\\\/1IVMln\\\\\\\/6Ex+1Q+56lFABk9KTC+Ht+RyN9K7oyjucXG6cAMqZod8lgnmAjpGwdD1CSr9wkUcldhnuB2QIute2EgOglpx5bEmKR5TqWWlOERcpUle4IKg1+s\\\\\\\/H9CcHtHSXpqNmZr3Rwm1249GcoHH\\\\\\\/zRzU\\\\\\\/su5IZXWFy82orqovsQz+AqJMSosO7Sn6DQWegddWK1MWDZHxcEuPoDwdyZCxYA65wOhXxkiXdpUCsB6hWAY0v0KFSzL2nDwy4LNsotdAyjNwNkg42xzI0XMSxkJPLKj3mJ7V5tmIZegXVRpxljwArhognqAjjWdtlLFEzVvKfJk9GYxhUKHSe4\\\\\\\/6OZMSc9LuIJcdhV+UvbTuqy+9VRlcSVaLRx8j7sch4kKpWpsOzo6nZkf5TX9uC1QhM4OIAdFZ3JtG\\\\\\\/h5KKIcHv2IxlYHgDVGOQNHpu9TF0Xz6Mos29fFvf06eku1jQN+zpAjfzCXjZR3UlBCVwjWFH7P7G8+LaGEUGHa8GEdWlMWx+LFKKmSPFWNiykn66BgXLrAQ6tDl5vAnpjbV2IMcuQWGnTTxaAWWjl0zR\\\\\\\/9Su4BkIbp53+M+eNP3mgOmUCUmOtJd+y+lpwilRrb\\\\\\\/PqyWNE7omck2wUJoqurmyCPVVjB7CqOUW8M468btFirAIseAE++4YRaZPgZuf9RKhgMVzFZGn9kmr64ulPXctKqRye9NaEtVX9Wz8+vWm\\\\\\\/LqYu1ETBhP4nS9s8+p0vDBbeJnaGciYS4vitqTxNOwUOTgllZggdRCHFqXSoVwaaQtG1G7RDjVSkNS2nFdVaM20Mov6+5VkLbW\\\\\\\/PpZy+0fQCilobxS5jo5L1\\\\\\\/yKoI1e+FBr2nKWgRG13EQ6ZEsP6vgq7p5agJvG4uKWU5tunaLI4obcmMj7el6th8UvSamyLHAXoFNyd5ea8\\\\\\\/otDcolP15i1N8lKn\\\\\\\/E5LzSWmazQB+nqr6+NLTQ\\\\\\\/pXvjgrFO2ruoBfH4N7vCb5yilStq8Wro\\\\\\\/jw17\\\\\\\/iSWZP34LUq2q6tsGxugtCYLPUBgUizlU5CQlny568sebS4fB1KUrUFpYCpZlx2L7fiIVGt4ATtAUnU8J74raMV7XA03h4IisyudBUTwrOscXkYGIPoqJt8Znd1AkQQVS0Y0hzwsGzQo5xRN081VVNTiEw\\\\\\\/tnBGU4XXs8oBZPGzBRmZTeC9rUIpiEG5pFRXNv8VxoZugf6O1VFhr9\\\\\\\/tKza0TWOiqo\\\\\\\/5W2\\\\\\\/468\\\\\\\/XdhvYJyBM069ZxpAaBa+d9MDkKxVibcyMzt\\\\\\\/c4b+kxD7wnpSIGkm\\\\\\\/n4fJx9FUvgF5mFY89chBSFebNpZIWAQxVj5kre8cdvveAvZgM41BJNahaV+sHV9WRk0WqeYasjxI4LnqhRTRau1rIwKyrMEebM211l0Ztp0sECh0b\\\\\\\/oi1mdFU0Ldb1rR+\\\\\\\/e\\\\\\\/ChXLYIYH1EtwhJehiDW1vMk6dEOdUgq373NfgF7hjGs0kMZ\\\\\\\/Zjzo70r7i76KtPlb3RqEbIO7A\\\\\\\/zBPV+7T0UyEbXPMahA\\\\\\\/Yx6\\\\\\\/qloHQE7lWAagJqKpwVnnaouyJKknAiiJRmhAPJXCMxeRujZNlIjHFb4HSwcG\\\\\\\/TWxI2oucPhe3RnlOZEgSU7Dhj\\\\\\\/pVdixUwf6X8swUPfKhUhosgvcB+8NZd3YZjBz\\\\\\\/Pq5H5D6+3K8EmULNuhG3fMj\\\\\\\/HCd8ZOe7C49bHkFtXv3ZVWxt9S6j6Kmo76YAomAEd+lcjQ9mCDsUN8BMS\\\\\\\/mRJx7zY7JwHQMppf8t9aYiYwjbBoH3wuxVEAk8M4sB6qc9JjkboJu3XOee6n1Hf1rbTyOQCEIotwDZVtB3sqwlAJNTgzR5xp+iF8n4Ce6blZbH\\\\\\\/BRkSQgAF8HiD1yORtQ5jOpFRv9CnqkRqJf\\\\\\\/z9WBuhVtRiDwUcSuEYrT3QU79WWUaf+Dw9nYa3t64m0T\\\\\\\/4EgDYbS2x9NR+CmDlZbP\\\\\\\/JsIhG+YbXjk3IOAHlVm7x\\\\\\\/0N4\\\\\\\/y8d3kfI65Hyj2HswxCoGt7UXAwJCe\\\\\\\/5Bf+IWEj242ebE3Nb25B8EdShNvreYkutEKVM5iLhC94zbrt2\\\\\\\/\\\\\\\/rw93xCl3\\\\\\\/ufWJ0hFQ+xo8nKONbIIBaXlOwTYOX\\\\\\\/96y8k28nBi7Z0Fl8helaaU+EQuka4ohO65cHCb3c0ex3bTA2tK9zoyqwfbcgonpa7tKU2A10Q7rUH3EWgnQJGbD1mSTkrNWyW9DTqsukJ9tqHPlQYGJPI\\\\\\\/VQVGEEoxpLU0CRtzfIacwA0VivKFWAqy+SlOpF\\\\\\\/X3g5wK7GjBcVxY3hOl6qDEbkh03jGITHcTX1idoizo0RXX4TqQgfn7VE12YNivpUR7goB1Ek5STO\\\\\\\/8yJopkh067AicYwFIgwpaipLTraKsd\\\\\\\/whnnuopGUOCaT\\\\\\\/dJuQ0V5xgJ4Frur952xqgSPMw\\\\\\\/zQw7Z+HohfnxEUltgiaxsD\\\\\\\/7z4fCQYDt3b8+nSyv+VIKx+7v\\\\\\\/H3qImLo82TQ\\\\\\\/0asbGOitSmBbK60Ave5YNXtftSfpsJHWLfIg3RHZO4H2aHwHAL4bLt9m2bo4APBwNEpmEn3jaOTDJj+57wjNte3elhGg6o1HlVfBibzrfnlUhMfJhU6hgUA4n1AfBrwyzQMZ\\\\\\\/HMpr4zjNwG7u+GGPlWZ5eIeoNnwwPPADPx+v+PhIDDPMlpAIiETxo5vfyB6nMzR0t\\\\\\\/IwUtjBp80LMNsTAhmwpKYhy41BDEZhg0bC7kTFkJgUUbHHGWKB32GylUDDgp8rb5\\\\\\\/rxsiFB5\\\\\\\/nePtI2XKknUnvHtBSFgY4K4JaJffQpdlT1dFGqddunvH+Mllo8SscWEtRrVvC8BxFPLFfKS4Y9eKiisJzkhyIdwanJKSpHqhx0cYHTo5LyHZLjkO11v2QkZEz2W+y7xfdf8qdMAv0axYaAUjzIZUEp6s76g2S4IZMqGmBXZK\\\\\\\/nXmnaXgCcvqPNXo3tGTRNFX8mArULwEYCZEqdA2hEUOIQXtRPFqSAd+JAhaJ0pWo8jXFr\\\\\\\/D8+5moKVIedG0BRvkBJapCu0U2T\\\\\\\/C\\\\\\\/GdMBmEfADNtofkCeDaxcNnFH4thPUq3F80xHB2O7\\\\\\\/dFCMWe6di5H5Umv0bWVl9Bcq6yMb4oiQORLMWmk6TigNxCjsn018RbjDwVrsr0ytJ0AYWi5CMhEfSMoxsQ97EMfGa3O9yaat0ocnonh7paYjJpqUEEm54b0Zl\\\\\\\/hBKdC\\\\\\\/VPXiRd3r+WIlugW1FrtJRpVQxuNXBADKmJpavBEHtAKDikcgj1IEJ3qinOy7JSB\\\\\\\/UG3pKh9NB8hjsOxbUh2i568f0c60B3Nus3TMGHhOVGujIs2YFhmihFOOFAzhjvs2te7FqX0ycFV4qSCuo3yZaIsBtKDmEAtRTeLQzdKa1sW+2y41tINRWGhhu\\\\\\\/bqL4vdPm86ms4WFujzX0GtjPdLyEafNkNo5bGOrKtDO1zyzzEA2ZG8MVDuMK6tVWM6V8+G\\\\\\\/yWAE604mbjbSXB5KYi5wQRAIvbeSr8OResGALD3hVPFCx38tlUtl6dATQ\\\\\\\/l8f8uQbwAfL30KVMCz9a81EWWPCv4g6LZ189y\\\\\\\/8DlFnkTsKDfzpVsvNzfGELQo4wfV2nZPvzBjrZiqw22XDn\\\\\\\/fx1mP\\\\\\\/DyA+QK1mI8rDLvRtcghSfRslZetARYvsu8r1RQa\\\\\\\/IzkZUUmabH0vXrNpwgtFWejASG6JLQLuemylyVV47hPqOW+MLOVz6m+2RRBAe8mzw8XEOKIQ4kS7aMT8EuMob\\\\\\\/oHaJcKMTNdAy2HkcnzYmFEczWFd4W5eJZTPOqdjyHIdgJN9R56qFx57itOmproWBnpbB9CDOmYnP10b2SUDG0bnlE9HKris2AYaORUcnZEjy3\\\\\\\/ni7J\\\\\\\/YdhA5qWzLJSzTrJP13TC9Vk7JJmmdoGngvOL+E2pz4l\\\\\\\/0u3+6PxPT\\\\\\\/+H0ylit6+oZJofe5Ana42z+FOqa29t7vvvZ6Dzt1E4WM2u6hwFb38SyMBP\\\\\\\/6CrXmAmUiFiJvBNcatgadp72fpiQ6pnLg8yMUEjx2wVmUe0Mi7fuUcuvuNh4vEO+TA95mBfjoKJgntE95plrAWMe7goDSwN7+T54CQByB9T4vAwHvp3YJjt6p1hnoJEYNFE6k6JWXTEgRrB1WagHVzoGStFuNNLpM7M\\\\\\\/i6uz34Npgyt4SMbK2DG4++JGb04YKg4L\\\\\\\/uV0IPwk4Ga9vAvhJjObaj08eb+kTcKjYPxfRWxIhCIAdRoo7gi6q314oHUVm7PYw97PhRNc148EcFTsXvOwpuYZBHJ5Em+sC+YPHrz9TWtJXYmld+1FpmLe2gVEq24bJZDoA1rNxVHGXAw+4acKD8\\\\\\\/S4h2O9n1DKzFANuyzKz9hwmPyukdwz4DvujjpONZ0NnxCpBTwjX80p\\\\\\\/WXdVZMnaHhfyr29J7k9YOoETLUq5KUvR8CdC7pfEhX+S\\\\\\\/HjjGXjpxoJgcBBNUKwl\\\\\\\/4G0QHmfMjSp2SHbvSm5GR+SKQTVWGDkpOz0uKyw12bXuuXvN0eZxjPq6lbzJtwOG3jiYX6emLsDKAug2FAcOKf+PzIJ3swxBOYADKHV1oncRqo8gqlsYlsN3vm4sK83KDFXH0NbSoZbRf2S07k3BuwtpjNNQQgFB8q9yULqfiBe4xnDSPNNQTDeEqw4EX9WoBD8\\\\\\\/59\\\\\\\/e6GZaFVEIUrQSUJp0Xh57uNHHi\\\\\\\/ZQuuOgMNWbeFmsALw2\\\\\\\/tIC6cF4jUGxcIF2f\\\\\\\/3Dnpv5\\\\\\\/vONOFVqe7VUABb6PJkPXGJdV81j1gntkQJnlSh1s2WKpjX+bnPg7mXtzntto\\\\\\\/XtxbbR9JyVEcYosH8TTfwZExlsDem486grzHBsPXczhRvBJTdpIfqcXYb5dAmtBrfUBpkAyJdDR9eqnoL0ZAgXgIVFgRd3hyR1f1YVTPWf4RQz7V6ljlB8elCq+uEaRqfmLYtaiZTAP7DaADGO4H4dLgqbRT2K2Lfa2pHBUluBTxGMAj+rN+Pxv6Ep\\\\\\\/Ek3ei+AoBFXyGT2BIECrZRnqraEkj6ojTyktw5Lq7d8hPorJoTWvDlk14UhEeJiX6j6RTdo95gEknYZestI5okY0956a2JhbewQtoVZRMWuHEjmnjQ1WPygitMr6r8dNAURRJI9OX0VMeaJYtk6FrRhikscFozKLhT6+hOsSODzQlN+xZHZ6ED7duwWd9P6Dwkey3gcu0iPwxolgRwwzE5oysjKLmLue\\\\\\\/UsOUZretQaCI4v0eEOHiaQ9dDn2v3Z9RNLHQfC5EbDY39YRe\\\\\\\/cDBZAa3cw4Ba\\\\\\\/4GI6s27lkx\\\\\\\/j\\\\\\\/VatqMFcxX+PHyIT8kabmQ5\\\\\\\/QCU\\\\\\\/luHBiwPVN\\\\\\\/jbSQInx5qDixAG60uoyJDOIr\\\\\\\/Ivv2qmtc2tJTGmscsxh\\\\\\\/6SGWsLknjkwxgdr2+ByRQHrA6XlBip1VVnFEgkvoAiuCDYnmu7HldYAIYHOaU81fQFx49KA\\\\\\\/Aj0M5RA0Dmo8e6Gndaup\\\\\\\/LyAUkg4WSrDIW6d07cUQjyS2HAgfPeASrBF1XlCgI9mRzDmuAxOZRN2X5g5dnoZslnINLXB2C2yKRjA1Ktwc5HPZrN0ptPkKhuc9J7kZd33Ab+jDqpspXrOuBTqfawX\\\\\\\/RttYA8+Vr6wmy6nDwXZk7O52xRERVBYBJvsypMBTGYXPL9A3JyWVT0WfTHnw2xqF7y3KmpaPZK\\\\\\\/rWVVFlZU06BRXbSfdmGHgcJF9TvQUfJlIQsDZNfL1ApjhogGaB51xL8fjgQYB8T3gu1MNv4mSF5hz2KqyL+xEGMz8VF68q\\\\\\\/wBr0WIXHx4+7JXDvr8xma0JytmKAe\\\\\\\/i9aU0s70dZBJ91c3UVL1Ntttbt5V+PZyWNf1Tdf\\\\\\\/x3\\\\\\\/wty06NA4SMPIL80+knrBBOSk4EGfrS9mmoOme6xUoJikiJfPJsTKCSzgKa2S4WOd5+jklE6\\\\\\\/PUtYheAdyLLL\\\\\\\/3aGXEeSgsklsW+IhhIUyNhckQTL5KaYxSUph6w\\\\\\\/aApBk4oeDVCvHt0GCOaZDToBoJb+iFoqMUgmtZdYZoOPj7I7YJIdw8Dx63ghrjO20TTxpmWVzJYbnDW5W26U1SDhyr5QVRacGuf2Ik6Fysj9qyiJ6BgUwon\\\\\\\/66h2LHtHl9RxKZLIJ+O\\\\\\\/B36LtW9EmKRlW91yoIsnFVxUot9rKSV+8zKvB+mM8eanB7XdMFnHnk+px3dQIKdFPf+wRdnO37G+FDCSrRq2MjqlM0LsMgN8fFsHu8RZbY6EGRxU0KppgIOtY0XQfkaQzb8x4P4OHh418rQKKujEeKvDW8GBO\\\\\\\/GSBgpTeNAJH3bEqc2OxGK+f\\\\\\\/UYn5CKxrJlwWkKL4Y4vXb1GRHipenVTo57JYu1ypn0Zu9KtEsWwNRoiyWYwh3J35p3bHEeGuG17GvgVCNah1od6TNKLFH0yBpQkn\\\\\\\/Ajhmv2aQ7qh2PipolqDXvgviuShDmzhs\\\\\\\/RGyMuPKgqeNkUOqKBQqdui++F8rxi++BtAdpMGWmBUx+KkodKxarpv2fuZ+EMW1MWDo\\\\\\\/eSOoc9yhDwsWcwYrYEcnvzW3yXkJ8QyMaL3w1JLuSCQOa\\\\\\\/E\\\\\\\/CTHv7fxqil8J7w7Kma\\\\\\\/ulC5zwrQchJ\\\\\\\/hgJ+XrZZStLk8T+jX1\\\\\\\/4zefgCTv+zIVOmoE5K9nFMonhZEpsQr2ZQYEQ6k99i70bOpSFM5Qr5N4tD7nCuzUBFmF0PcaYhKZhDQ1DH7GRz97A0wv4WoATAsMOAOysabmbG7IJhmAYa99EF+Hz1qhHp3G67yoE+py+lZvoXFZ3h+ybmnHGOZGBljyhe0iHzi3wB2H2Rrlz+jdst9L7kE8EyDk2zo7r6MFasoHo7Fl3YKWylKpgiFV4cabK8FXILiQi83dIXFw\\\\\\\/C7t49g6K+YtE6PG44zhOHtEYwVDlFBV4exXjM+ECdBM3l\\\\\\\/9rLDiJ5ZNr9lZ64Q0TLTjINZVVcGUQXYEzzu3pWV3cBGURNAGff7\\\\\\\/B0uINC9VkZy8lnSbCv0pXwNdY\\\\\\\/8Qbb9JWHaRYDHIyhy56h6VcOqMJpDrsqqq5ef4aQ0tYwIFpRgNmqYdtRvOWsGWvQ0mA6uFH4SaiskTUm3f9S2ts8q5H9KHzU3fo5C9zn3eOxOSdCDy1A125hPgD30pFB8A0Uvs1Sw5DQr9VjPFbZqnPQQJ17BKqCNW95K+9aymhaUBG8vCSxdE\\\\\\\/46Fy0GgRtxdUqLID0\\\\\\\/2WKHdTLwOVSqXDGmmQ0IF8tEM4ICyQOdbqHLEd6aMn4r38dqMQb7qKwmLN8NIo+I53NU12\\\\\\\/J6RS9WnkecAbNwWXpr0SMd8WuP5zbfMYHDQxdvsxvp5s\\\\\\\/UL7sxz+XE1Ud4+4FwUTo7q6AJbVaXneEEkNZA4wLIC1W6ykLzB3Xu5uu7Z3JP+SqVtZuNn7PbHsU6GPdvk1Feut3VKLO8vSw10FQSnAxGhen7r+i1jy2ynHJirb67Kb\\\\\\\/nZhsXX2M5osBHpx5m+CpDzlm3tf3zGK7EyhbQKq6ySPMBtsLQb4asoZa1mi8cntoR4pUPhZBgtQD0AxaTh4u+YIfbFAvP2SLUkWHTIZpuj+a5xy2aPmaQsAFGpdmDUHluD0TNuRFnFLCFWyQed4pfxwoIXX7FPbvbSP9G0Wa4cFwxONiaNe\\\\\\\/dQieIqepqcDVdQFAAsc2kljLEukrfZ4fmq2FYkUZfFHhD3L0+1pztRx7okUGDXu42yg8H8R9fEhpCJtqSJqTUuXFnHhGcgOUKO5rDj0oYfkR8DAdf7lPIWytWeRABzcWtIcsMkBTVwKS2Dvf7PwEGIO\\\\\\\/GYZ2VcD75zhk0IMiHdySBzW9iYadl+qUAcc6BJZC5Rx4qfbYqo+xDxCdmptZdiTY79ByjgfparBgF35rgY2IBWrBgi3uTcvk\\\\\\\/N5I\\\\\\\/2BJgoozovxNXbQmS3G8YmD+d6Xh5WZTnUPt\\\\\\\/a7nuyxpjh5wRGqP9bFoImf8rWrVw6SwH8sMVYA2ZwGJVUR75T5q6lOxYDeMK96wkQYRsP4Qqr1WEot0cyNWlqwKB7l+TcIIbt3rC2JJHQXbvTe5H2H050FJ57NoF4EJj58pf9bdGFbcJu5Vlv\\\\\\\/ikyaOnZtuiZtx4vZpjuilIs1On+rD5WKuMb82Qo4O7DFL3AZP16trzLiXEjhsKvFSMqvqslzjAObNQ72Z\\\\\\\/G\\\\\\\/ke3F4BwEUV0cTAFmY+SrlsoPeT7WuGWqTYakWQO8YGUDaEydfUQyXBD\\\\\\\/DXhDGMh3ZWzTrF4bu8Qk18MWosQmHL8ORW+0Tcy1liP6mvuVFv\\\\\\\/Vd7+LLlfxcuJBIXsQeQgu\\\\\\\/u75CqLTj8\\\\\\\/5tCXJZTHTNAdlvkpHbwx\\\\\\\/Q5ipRx41WrCX1gsCB9XheGm+vMyY4YzBpJmIhDpfPClWHM3WCc7Rx0aGUlZ7m6WiaNLwIw7VzFOh\\\\\\\/BxJG\\\\\\\/7VJuTFoDqdYBSPQMdDg86TapnqlMsN7BH+EEHLF8PSruPq6gfDm6a+rpnn37WiLvwXSque06UAlm+X1zw\\\\\\\/fFM33z+0SDO9jJKLhoHSTXlrxNEeiDJlkyvru0yqZd0iuSOY6NxZ\\\\\\\/b4GJd\\\\\\\/NkPR+XN\\\\\\\/JXfdnrnaXc2o9LLslsFZm8UWPPsnjwfptJ4iBb52Y+29bDyDgm4wO9YoMCPQxCQoyULBgqx0UJPG4gjzP8Bn\\\\\\\/yrF5mLxEFf6zO\\\\\\\/9z4VngHMR8Y7ITGEPiPCuvVjkmN8Zfmx2E2zpAEKj7l1LrpW3mRx3yNtAwEl4ncstlR7SmonsJlqE0mgyjwiWn6Ugu2DuEBTuoJMbvDTHkQzu8YmZk1gbzu9W9VeFet3ZeIUyTxTFlz1EJP5F5+iH6wrksQeeUcbkZ4DbDbUHieZk1pXKclbc2XkSmGtnBru33q5gOSiBTRm\\\\\\\/g36oR1kEhgFUwvB7h9UMvhXM5ySZ+0dRhl4rLjD5UTn\\\\\\\/kNsjcNsqPOhZHpaVRSdP1ttcnw2pOQteAssom4Ij8BVI+02nmNklE1sxJKR7tJGv6fi1b5W5Pso750ibiQ08X6T0SAi\\\\\\\/8Ws\\\\\\\/Gy0khWpQINucTE0UUjPR5eFvLHJj6UqY14fnPBsbbdWa3mUpvDXWPOCJ4LcsosY1cB0ZTDieFuuLtyIb8xG5ht3tsl9pJML0IjijEcZfA7cmT9Y2vzvV4rJNP5iFEOA4OobnFcYjjDCBPjkMLa5em5E2nGF8\\\\\\\/EXEWlnVdQdmXSjzTilEeRrIC9H86vAQDOB0XYPCYF5e3lQ6mObwx5n2CDkaAm31Mcz3hXAZF\\\\\\\/QPDnxiIK9xdvE42iQcTJdPpazwBZ1JNpmtUlfAPInurhfkqhg+UXI4NwHA0GiifIHHNq1g5LnyiV8tdC9Za3vE15aCWozcQm6maPeAXHQUV9v\\\\\\\/JrSXOcpUwgzgxe96qynagZZe+5zWw3ZhAGW4ul3oHYe0XivQlldO+1YIK9XpK5YFp+\\\\\\\/ETXYqBFNPmD0d9ixuLmJUQDQFxWZYm39XjRis7DIvPqpjqqPNhF+UK1Ft6Nf+X+DaR1zTvxLHQFSJUirE5aoHHG+DEVP1e9yxR+\\\\\\\/KEcSb34DBXSeni8UKwcKso+8Qatf+WJAmOcTbVcQy5x9HF2cwuci\\\\\\\/CkezJG+SZblH7g1H9\\\\\\\/cU6R+58NqwTIQKaa32DGUrn5ERk+VGRTOBb7sv7t55QAdi1aCfEYYGSySIwgXCau\\\\\\\/rE9fHmfa8bdIxE+X29iJtQtcQNxpDEvIEvOC\\\\\\\/AsHLXqnr3AvuivkO5vSP6apEbF6C8ysm8z7gVBKavuDasF1FeSSZhC3k\\\\\\\/RxZHYY2uNAfmuUxo+FyAOqoOuxVzneAnJgbHZFEIxOgYEXS1VMhCwgy1qMjTaxpeOnu7hXr8VpJkoyMYA6LsJr6X81Utu0qTK9w2hxAEzFeOtbTXJVDGU4QWSv71oeY9BZVwiQJpLgE\\\\\\\/kYyjEZPqd00lHMQwor2rg98LzeUnGUftk2DLX\\\\\\\/yHnQ7t4Qov8205yhGUlF0lyVL8aK2RRYwKysHQ5xRxFWoZa8wpSSHsWZZ3ooVz8Gd65i6wg0L5odKwDAsk68OtTpuiTlbRdjhkzcpvc59w\\\\\\\/I533Ep2CqrQg9haLl+GVobLfMftr0OpNt2yCmW1IK6xaSqcatWuv8pf4CZJqO7Qfb2d4MJgVtsM\\\\\\\/zrqYgc00q3S3xYMRJ7ZWuQ6zKMVFKmzhcHnb5Tu2N7hhy5e2uPvdCYmFY0pPn\\\\\\\/SfSHbKg9KhxzNau8Mtz6JYCQrxR9E7tlxsnpssj8MZTb9N\\\\\\\/q\\\\\\\/uJIDznFapNrKRjnNpnZuTBJx0CzuXAnwK6HIRCpfrq0kVQau1dhQY+KTC+UvNsc\\\\\\\/loOBuG0fOU\\\\\\\/il5htUR5erjPWQHQTRRz\\\\\\\/EdEbu8CBsLcQHDmQlioaBkeJWJE6OLjCpbRhclb4snLAYtBQ7FqJE+la7rZFSzZg\\\\\\\/DQV8u7FX4hUHOKSF7IyluJVPBPSsW2UMQqJ1GAZkqoFQ9BOUACNlJkZgPZwZ+s02mBFNRVhRRHiMA\\\\\\\/lOwuVvgydrslYe07gsBMEFLldM\\\\\\\/4UME+iTqb1O153lwP2RbIyXm7Yzlk8HYyJlwRNWN9xAZaPM1lK5IRcxuUEBQXnms2cAlOjyoNpuLYNure9ZQRFrUwkZ90VFGHbCsrBGvytm00rZ+wsfaBMVXaHCJ8+jU++nEMDNcWYUXL93JaH3yfeQVLolU2AkXiG9jXIP+h9VVLKcbW48UPFS4ksYfNl8iIwj6prNMx9jLXbP4ex4fxvXxwsqlE0dCsvMWF5urlWbNaf24IBYY71USxstQOK7au28BG6ZxXdHxiFKD0rVuu3xahiBmMAijGGRoEnw\\\\\\\/NfquNJc\\\\\\\/wPAA2jSTGm+waCSuCjQnP+5\\\\\\\/hi1EiqZizXuVYf1QwKmTX4rI5yM9VBH1wmnWmPS3mfMP\\\\\\\/FFXF8XzHAg2TvFS3KeO8GQ9iZjRaOyNiAMW7y\\\\\\\/rUXH+pjeCC27fE5PD4K1y9qC4LLzywuUgRTFF5l28OtZM875ms059N5qykgUO54kjUNyzKbJLL+m1sxj59Go68ScyXkT7vICh8pDB5X8CumAt1ZyZrn\\\\\\\/wV59wGnd27lZJRCXHBy+72UvdYXMi39gjnL5PUQGkMbTnyS8CgA6EHyEn1G9cmJDSOwM1HfsbOQJrbeIcSghI4InBxft5nB18zE2smCpvdh4sNTaTGaaaPw0sjju\\\\\\\/\\\\\\\/KpkVqO9X03+CZb\\\\\\\/uGZjwxAebhdTG0t73IjwOyZNSSnK0N0RkJyzy4qXpiN5VhYwAwuSoKiTCQ4yqMPTPgCQKSMSKbMtNOntEcxga++RBav0wJ7N9oOk97HSlssjQOHg\\\\\\\/MXKK93LWp8+TvGZzZJaxjeefE8XZDI3exoC+FDY2ng0vom8Yl+MsNGd7FULiuK08g4DV8BQgMjxw5MWZyDRwX3Op23QeieRpl57ovKra0MVK3TGuyKJ7VxcWMjOtszSr0Agtjtvdt8AGW1a7SIxZNvOHDb9pdQ9vINSDNHqZstIyrs3IIuVvgjWtb6tgX9R4kjSg6Mpd49063VyxLs75ffVAJpFrQliBnVf6pwJaMUiowUhlratsQNAMlD3mLabj6t7cOLl17D1wtCf4e\\\\\\\/++fPDrYknDwUrZgu7NlLygDyw92tIBT4baWiqQKXGc1ubDqeihHqmwqlmRtOtL0M2hX9DhlbYYQU5xuQ\\\\\\\/cT1qreebH+dqZxVf2ALH6zGLpQ0nYKXV4LU0PxHuuzpb+QK0w4YU6gNmHgX4iUr884lH+AjAn9ou+KRPNLJpBV1gEElhzVXyVobmfDPJr86wXNuDpwzo+A1Mkdl3SpjhSEts+BzAXviIC7QMeezqt2xmXDQ15aLfjxfWG5pd+U5zF8PTguY3UkHKbjpqMG9zGXYGL3HVTbIkMmuKqVW5x0VvzPp9QU\\\\\\\/J6ywP6JSv0nsjn1iG3rCjLZTDlwW3IZiVuYGy9owz2FO4n7vSJZgHa6QM\\\\\\\/mwKw3fBta1ycT92bqrMR9cWilfkC7eUew5GWpxYriWd59aSp2SChYIg7JlbXx8\\\\\\\/rwc8cEwT1M3tG4zmT2XiFwzlRXXLmPYd8phh7P\\\\\\\/wEbvlzPwhPk0YSgFg+y3VospQBD0kSYsnB5qQ+6zCsSyGay3HvlGPEOHhSIyYLMb+rx7Odp6HaQz9E1hT7RBCjSqE76HxqKk+byR3oQ3NjABv7RIp7Om+Z3W9BZhNLU+AjnWTs2JQgfmVECtOkTZCO\\\\\\\/Hs2R9x70LAOydEPlVzbYSX+WBVr2tgbXdYaaNCpZPGVnwq8E3+gmxw4JBavWOATRt4XuuNJ3kpIhPoOLv77pcpFDFabB1OrgdpcDW5PsXN\\\\\\\/SbPRWY2dCV2OzV7qeeHUSuNhDt6fDoAD+nDqkZ4jAz4oUsuLpvoRgmnuQL3Gg\\\\\\\/qrApA2tz13dOB2hSrC8vsDBp5\\\\\\\/8\\\\\\\/7WCyLpg47\\\\\\\/JSBrJVLJ\\\\\\\/ULUt7I\\\\\\\/cjLi8SuGUOqIb7u1dkH82oZtpm37TojmQ8E5tkxnpxXX8ddI4PJKFFfGPpG2AjDuwVuA6tYyrY0OgVLNP6FdhZuR74ot0hH3hyosOEcsjlr\\\\\\\/tUNqeOpHJP0CWSbC5McbNcy1BJb\\\\\\\/TVLBV\\\\\\\/shIVRhmDwkQnjzKHekXFdwzP4JxCnHWJJs2BT\\\\\\\/pvkitmt8OkljWs1RoCIGoXeh7yW8F8SV+qOVsm0uVPoadbreMcfk2yC5Wul6LVvrBd3PhvwG1HxX3Szzp588BWnqzUsXpj2y8yx4MpULIe1AI3C7+ZUEEaKXaXw0jhgDiutumMxRicnGAIVrToKbHfiISz\\\\\\\/tsmFhva4MEflD8YcCON8F4So1+qvR6j5qo5kjrVq34U0n\\\\\\\/s5pV12hwfyR8q1\\\\\\\/oHvzIl46v6JO106NqLRInZoDYt2mkY+SMG3MvcCKcvecV9P8xMoka+hU8chSkMcaIZHWFwjO4IqKaWFH1eHoNYIFhJJxpvs+igCy78f+VGODk352m+ZhbfzQpNgc4qdxJ7Q3QeBuc7u8bSxBi5Bc1iPWhwRFp09ZgRVyBo4leylzfkMqnf4FtK2\\\\\\\/QUHcE0yfLc4hFqbJ9lT6nAOk0cRFI1NKKuoE6ivHUaaoCQfdlK8s6Twlk\\\\\\\/u6B1ZwfEdLmqoVVlapTMvUbKpM6SZqdwAV4ZoK+TheOxG6R5hoprHWn\\\\\\\/zqdbUVWJiqzxzehHSi+MQJud++cufadZft7q9\\\\\\\/6Lbu3jPcFZGkt+MmPNZp0hTJ\\\\\\\/SpWrH2VozZIO2ljbgnHiAw6eWde8UbEQj9rQHjXgjw4NfVYl+n42sd7uKARDffVT7SxLPbgM\\\\\\\/QPeTjUuLd7LDeJeNsKWINwpW0iTF3xbuiYh9NZGet\\\\\\\/DkrICQt7LlufUexj21SgYStjB9fD00TiHFxbHo8KFo8UwW5vhPmBfijjl3ykSeZ7GxBSNVtaIwyRdw5EO\\\\\\\/HXtzodTKl5E+ur17KGMd7RmkZlFxz\\\\\\\/dliaqOhyRLOA2IjOHObZJwHY4zD87Z1DrZGp08TsMHkd2uieMkwV2BLtOj3T\\\\\\\/G04yJVr3xVlPolM6qndvT0HvvDO7y7mar\\\\\\\/mYbV0d\\\\\\\/CCieeBJNW69Tq4Bkq08Onq0r5ZE+DMr2k0nDB24bqB2Ovm1rCedKOo4JqNvMYr2UA0HQ6Wk03s9C+Ff6HhiJGBTHvW\\\\\\\/KR35BkBAqg8rO93g0TCx6n\\\\\\\/A4j78\\\\\\\/+pT2bJLkn355AiR\\\\\\\/IImkT9YvJe3KxTVnMIW46iNDMAn9DVPtG2OqaVeOtE++rLLsx3uuxDMyY3Q6\\\\\\\/t3RGxwdTbcb4h4a1tJ30R6woB0iJOs6YU5O95lTTOhji\\\\\\\/YEqxrRYxnO35r3JkW6UC\\\\\\\/NLEF4baQvyZ0HsVGSj4AcDf0ritnUO3ocJTIOiexNsX9Ah640ZQZB7bxAiH9CHtbFRirpa2Lyke0zpENV3TdMx2tsWjxX+VeWm7BYcZ1hKQFhF4DvLrg8XFs2k6AmFuZ5mEu5iRbxIgX8XGGgGeFiII1amKfrYfk6s6rBZvQkNWLX8gVxgsxCgdgnS9lmuHPu33oFpz8ml4auqAchmcobaiKNsENriwVweSLNpX9yMc9lEhhWfJDqF+Bls\\\\\\\/11AKzxiKiRET7THmd2Wlu7oFiqZPymR0YbXfbtCzt5G9WEK4esv2c\\\\\\\/EtdoD1np9toGftS4iQOxc+UrBEsgJv2l0Jj3ptgCcC04u4W2XCNtwrqVUdI9x1Akfzy6a+qw44Nf3tviRzwZQ3VfSBZG4DENEaLCfIl2mVKDjLYaxJ855QJe57CLrVI8Pbo575rBWUPf9LVoK2nnHDzs3SvondntB75EDs6+CyR+F09m2HL7We\\\\\\\/QPUeq25VUbXTV75bs7gEEZ3Uq3EAa6eS+NqV4NNqyXV26kmDDR3UkMSEeC9lMYbTUD3DKTW1u+LDkbVVO3aVV89UtqfwounxZz3e4FQROeWTodxT7Bk64JUVMp641a2+uKS4PP1NHN5FNe9v9lOs0JbW9eDlvfcZArwX5k\\\\\\\/ZiyNjzK5PqbAlEn8keNqiNzDDZnA9XNxyo0nwME\\\\\\\/tREZhF+EixKzOy1QgkzTkW4j+\\\\\\\/5B4RspuYyq9xY7eWYpd\\\\\\\/4V6NjR9Vn8+1a\\\\\\\/VPE34VWqoC2F8sYbOQBUMx579PMK5Q93oJD0Ps8wY9JjoFmtLgShHGdyHAR\\\\\\\/KZ4J4K5+VNz\\\\\\\/KIf+52NoxewFFwZb64myIZmeHnmRXWCitU7nP9Njtw2k\\\\\\\/uUkamV1s46sevg\\\\\\\/LenNn\\\\\\\/shr3Gt6tMv4HmHug5T13wX90K2sKluboF1+DHf+A8cA1gbcrgaAr8TZSXCBmJrqAgOpgOWE8DDyY1z4tujJWPCC+sd+ilcVZ\\\\\\\/sTdvo\\\\\\\/eym\\\\\\\/4DCI+0Yhy+hQc0FsxeubaA7Ip4uReTVac8cS8kLqad3d3OK\\\\\\\/8iPIxWIY1IruGhz\\\\\\\/TiDV9G11L1b9j5PbGawkgHSWPS7VMXlUxHRY4ht2pB4ETTbzAnOfkA5clfvYZ45e0QDw4kFVieWvMHao2zbd0lvn2M1t594+abk8P4PhWi5XjD931lpmpRhRQtX8KymboDheGloWM0QjVj2fiEvw7nWk\\\\\\\/7J5VO2nKTy0Rr80XcSEnSYPjHMwuwBQZsmmPA4KgZGI4Rhx36Ro3+0BrwXVfp+7SGMrVJmL1uSh1kwYpkCrTtQdx7CHON7pWORIUVD63h2re8Bvv0FAqgrluizrQJuux+gJ3qfGVLJHnidIAzlyGBEWncv37gB5qEjg4VTnbxqMskMjxDThRZ7u+RIt4J9YtmWBBj9uCNCZFaBpV6Tp1A8f+U5RDXZHBe0KV4CefNxbUczI9gE4cC\\\\\\\/6PqTTrQiGgB5RiyuWGfAMzoyjt\\\\\\\/oVwaXUN02WSeckYj\\\\\\\/wzW0+akrnBWh7XhoWYZEajp0LTkNcaSSDjglHwphcWAHSiMGRvIoQBqOoPkDRkfpY9bH0rqMLF7y\\\\\\\/MnqcO22InXw1tcm+T0T\\\\\\\/cyadYqPnaw40I\\\\\\\/+urBvX9KYGnKuJS15KZNIfThmVqpPeDQxiRwDxwz0ALFDKDPYS8c8jWv6kZZBcj\\\\\\\/MzEDdOL2Nw+kADvg0+CZYf44x2NuHqYPTJzsz5w5E\\\\\\\/QqI\\\\\\\/2LZNNshaGXNMFC0JeS8SkpkMfjPRM\\\\\\\/sTUISYYYzy8CXbh6gXEVGonOb4hZDcsIRoA1dwnFn5L0JFyhM7qCZOGacvkt6olUzArhtSA0L3RTE1Z\\\\\\\/j3eA2rm3YI0cUQQgIzhyDLxoLo+qHD2M9d\\\\\\\/DBk1KL6shJ25ZUCacq6qr93vflD1BSrDpaSDiLXYjAtIRx42oLCXNZF4aFDqXSTOouxqvc4PFXcLdiCe5ezGMWVtWw3lSd+1qQBb4E73+s3\\\\\\\/mnVttpnwVj4FUz+0zGmQ1bKPFNhSoK1UspwCmpAAI8yQsf5OJZ63U3F\\\\\\\/7yigMtE1tbhsW3VxqwPa3vTkIAua3axrBq0SOmhrY0l\\\\\\\/KraObDia8gh7vHbR+C9lkuTdMngzpwK\\\\\\\/DrRroNtJEEM5KxsGLtpmdOG\\\\\\\/tQKhufwrY28AuNAuv\\\\\\\/CjI+SGMW0QeVKG9iR0Gdp6Ra5wGGYnQLyvgOPF0reaNQ5AVTHTw9ulQ5jtaWgto4qApMX7wnHCNVrGxNxrxUECXrAi3hJs2fs+PXjvw2T+H2G6aQtLm1DSZDF6JtOyR9QWMJMLOSLSpi1Ds21kNnX8sl3FzdoaG7kYraFdiFneO4zOei4vrph5G9NZuRZ2xpyQzyEm\\\\\\\/aUR58x5SNUaao4UFKXW5iBvT1Pztgk\\\\\\\/2nnyS9Bv76FYiUkuxFtICeCdSJOUpJ0aFRC\\\\\\\/1yqKird2123+Qh15t2e4fR3r0vHiA1i0uJMQ+aOspvxJIBlwcEr1DgynwsXZHm4i3thjYK8jss1qI+sJiN4u5oKyV9MKAs2ojHWxJlj48TTT2AutZpyaEuEphQq26H91QQ5zGHT62tP6dYRpC00fdFFRuGjfZoSMsDwIUFs1VOaXf6Cljhh\\\\\\\/1v5fOx8bGrHRyKFklDLzOIy0tPF389CWCs2reejiFPdM\\\\\\\/8ePU4jbf0WWMkokvXaJG0eGSThDjywVjXe+xLAfQLTRsEbSs5hF73xrbAIJtQfwVk62XtISLYQ\\\\\\\/FOoOxiegwwSjak6hbF1\\\\\\\/7ZoU4C4yQqvrhVjtVAamwk6C4ybz4RziJRCrrAUKdPrpm0yC29MYnHtIWmKE807aJvZalSnrY9MnitGfaZGOgllRS2fi6AY7PbZxqirSDwDrcfl1HUKJ\\\\\\\/XJBiARI3GDUuzU597ZIp0Hko9SaABsBvAOSpJKIYvAqB\\\\\\\/ToZwdCyc83c6JZlxDjK8fvMKTbK7U7Sujfn4FsQ2g\\\\\\\/2Rwbd\\\\\\\/\\\\\\\/PsCcdYY1UrKESxveqJyiFIzDVr8T2oKlez3sVE5pGO+c4cBH64ctJ2B0jZZawtxset+wPjjwbALI\\\\\\\/HCoHx3UHrUk5eP2gJKaNq6gLKcFga+wcyBOAuwq01oH0r7BpV7aQ6TWl9hMDAF\\\\\\\/Yl7RMD9bQ+h7MB9bTraT0k1vZ57aXS6MqNHT2KACQtgL1Go4befffWNR8aWlcNVf\\\\\\\/2sUvTiujPvXBJqBWaBNqo1jbfJ+oLOfFREsK8BbaS\\\\\\\/wLRpRSdGaMForCJGbHDIudS5K4boR0z8\\\\\\\/vBmyRdg86NUvwawDCickMccZDwAUfymCV+HSrjcwBwA\\\\\\\/5N\\\\\\\/J6xNo\\\\\\\/PAEwjVYqZMQGuPCYZwXgKthtPtzBGVn6ZmJpLEnPu\\\\\\\/OdKov5yLxRY3Z9EbHGbwrSOek\\\\\\\/2uwbqSkeLWW7fJ7XvXk4Yp2Q8\\\\\\\/Dx4DAETQVF30APWiMln+26ljINXF99wnSVf8I49w8i\\\\\\\/XnS+8xXpEu5p2I5cspsd1HTFdMUpql5nMryEDMY+4uRgI5tbKZxQFVKFqrl+\\\\\\\/gAqc8v89+V+9maRSJwNHFmseJRdRzXBudg\\\\\\\/xhHcekVY6uUu4r441Om9i16ic9x03QWQQD6sgZC80U5M1lok+N+RmUiWJs\\\\\\\/QfLV8lF65trdgQ+LKWQ4eGxQS\\\\\\\/FGaadUzNgsEbcaeDrDfQUJmUkAW9fd2VveTFQSKvH\\\\\\\/9mxSyQMyMxtgzbziL8qEobLzy8LUQe8dD7W\\\\\\\/NdhTQjBqzliIVW6\\\\\\\/qfmlp1cxz9DcFO9mvWMo1kRWMVuAGFZwJl9SdI82XjwfTZVK9h0qH0+lbky+su8xhfSUffgbTuCQxIc3AslPEtPxLStWiaOg\\\\\\\/hINmt16x9BOp1+us+CtjuHRzNNHmQTdf9eKn3lNB8rQU7B9tAKjs3PJUkV\\\\\\\/0jEIMpje586q+h3hu43DA6zWAVSfgL\\\\\\\/X0F9RyJPRQQC\\\\\\\/WI7+pht370eZ73UPVNzD\\\\\\\/Ajdpl13bFl2N7GQb+q3S9CNzhfEr\\\\\\\/SM11ZiZEY31HlGRzJgeLv\\\\\\\/0SRUj65yE7H\\\\\\\/sUEHHSS2JXkLamdj5cVJUT8fvZJ\\\\\\\/AiA7quuGBfCfDerd1lEYpSxSX3XsHRLxJnXyvn6cx4VbEi3fEwb7UzS+el7zpVFviSDOn9vbqIm4YA3Iz5pkn1uGq0vgc\\\\\\\/WJzPEH8dzkosmk\\\\\\\/kOlLO5K7BjZ\\\\\\\/N\\\\\\\/N3XZvR7ndEdPsyXMZf4ZVf\\\\\\\/pz2JI1j7CpWwk959XskRKyq1v1MPJBj\\\\\\\/FjNEtXsEakxHwEgsMrpnudNHLklo+0y4BcHmd2PJ7gFG7z54PGn5GquKwKyDktwq7PuR0UGQAyyHgbidWvSXNKf6kA\\\\\\\/H1QngpN6WaPkx82EWFUXfsZzJ5z24ev0wM5pNVbACtjTskOoJdPCfVwnA4aSnFIqNzzV9i05wB4X9ad04tIuTGhVRw54BtsODuLOzvupJd3WpZqCCVeW7Dd13hqReGYg6JIjFb0pDd\\\\\\\/2khNOwQF4D2\\\\\\\/R1LVJBnnBaDiU9+22UDLDulQoJ\\\\\\\/xx6ex\\\\\\\/crEYEs48M9KZ8eHpZjGEA2v863YaequgPQukF5vB3imSZViEBu0jgEqUT103opt2193Sl1kKRDjPbjp\\\\\\\/cxgbdea5JVVCHO9woJZyC026mFnzTHZ5A7sccXQmCxA6mLfgcax0Ak\\\\\\\/ce+OeBe8K8Lc7TYc91SHX0lnLeqw9CGbL7rsRGxnzw9qtC+G81DAlZo9xEE+hnFHxKhqKBMVKDhADVov0AruyHhs7d1ZRSdwQSumAf9N+UfUqKOIoWGPQhhR2qhgZTJyM4jzHunNZ6rpbu7UmrVxzGcst+NAy55k1KNzWJ8jUJeSftGx0AP8mgoD1BspI6M4uKHaI93GCOW5tST+wVsIAO0IqSqoTckRJAQr7oHlykwljhvtThkLi5QN03fylVXs+SWy2jLIFBDf4LcYpR2WNLmZ90EvlO9mNnQrCetVza8laDmnG39E3AFfICn7YFhPonNALA\\\\\\\/BPPDek8jKU1go68Sk3PLx8CVR+XWRSrvAFOiKvr8e2pdUkiuyqDHdc8\\\\\\\/EM31M6\\\\\\\/+6m8rsCRhCvxV4U\\\\\\\/+HHJGJWJYka17b0kWjNC2+8q1N+pMQ07UU4SLmW01+i81Vy5wPeXP0uqtx9UKMgwPDR1pZpHIqKSucqw4GoII6+QCyJllREW5I7CuX47LdCPWle6gtw5GMiKuvEZ2UN3yHU11b7E0iat5ZcxZtajqOVhH9xDhrnL2ilbWfd35zgjWRtbgt4b9yBgjBRYGvwSRokThzUXbkWvJU9oCI3gq1vKjtkXXlMP7r77mP437u07MreoAyiV2QUERQN1rIkSEgv+iJ1npfrFv\\\\\\\/1eKSjn48iCcC5Xk0+GoBXPapROqJO7hVUhLvSp+DiUDicygZThTeh+wwHdXWgOXw3IPK0hok6KbOtnz6ERITHVyIPdAJho5Dld4FD2scAv7RhuQWytgm+SmYqUcbSqW2xC+qE\\\\\\\/MnVC1C54FhWABTwmlhcD3z4qfuKCLFxrOKHCictSjzlUKvqbkFAqZqizE5wpUUCjhg78cd5Aqxolw3tuMU1U4MzMOoacR\\\\\\\/MdJwdW1w2tXEUHcWdBbMMG4j9MrGdUx4kf8uKXTOSoVQR8MyPrU\\\\\\\/UdOxrML4SfYCMmBN+I3Nn5A7XN6919+yUujJjwJbLFxuG41VWhr1WDeQ+sW04kS2EhsYwnmv7EaEIrqekzWd3xgfokU7Nr2kApKGOQO86NrUq6Yr9HZPX9Qpn+GzZXOlJSZCH\\\\\\\/Gyc7mGmXsTqaKMaM1NTica1n7s7HneM+X\\\\\\\/Cg61+VGEEiC9SqcZ\\\\\\\/R2aRoW0QMCmMv6hry7XYCVZVOTA+ma2UOLTpKexCeYpMqDzyn\\\\\\\/LMuQ89z9qRAjjvhWO26SltvIewkrj5jGPebfF1FcEMqCiNznOYa55a1c2kVKcKqrE+FdDIZ0FSxJ5RlTRYisc8CwFB89flIe\\\\\\\/NxJsHHFJ8T4Z6sAWNc3eknYu8+C6+hxKFH7lLrEg4fllOmv6moJ1D3wEH1gZcoz77DywuBTNXT4pdcRc1MrTA8sHPn\\\\\\\/BKiv73ieLWUj\\\\\\\/EoGOCA1JGpFls0GEkTW1FeBcXeTbG8ETyKcVc\\\\\\\/nnbmnX9N7E15hthBz3Im2fs4b7STqd+ABYkDsRR3fZSBD10j6yilQEDYEewrr+bGktWdpSZAgdPa1FJfTOeyVQG8sRWeH6pNn4sVtVk0+cFe+5jByr4kE8b9QbZ63HjEOPotd9BAXDXSwY\\\\\\\/rMI\\\\\\\/ke6Xy8N43msmYkftL2wJNCm0I7R68OiX1Ax2n7LRjlg9Z1CwpuiDeZzvf\\\\\\\/5oG0ZQn2hx0d0BCv5cP48CKMs5C9qwoGxdjkNH4M5KjpgYaUpkFZPZ01Bv\\\\\\\/M4rRqtgLPwXIRFuVXSQ1xHkuxU\\\\\\\/BQ3hzLyqVD73wGkOqG66YK62GFjGqlEWsA\\\\\\\/UCR4CAWVHBAVSTG4MtRiyJaTHk8iMOd0AB+\\\\\\\/Bi9ebjuTD1GWR2BJTTgX5bLRUdUiA1DtLhpIX9tN8M+\\\\\\\/NGBTFKViP0PEjvUhKZcmU\\\\\\\/VYsc7ilgMLGrhxy9eGJ0hCxJA3SRHdmM8UIFzm68GJcikhuZ1+C92HP+f2aWHu+J\\\\\\\/7Po90gFkHsr7wUSX7ukx33tGzXJCCLmqvK6ZpRL7edtuWu2uYXpkiCoUi96XkiOwh4AyAtFpMk403A\\\\\\\/0V4gjJHsbgQaYXdynGosEWFbhCdqfnUnsKmY53504lYMWyvAgysEfYNHnBSlpr8pN+uh2N0AdSIeRjjN8hNm9tK9sDwBNEfVZRo0zfgIiW1r84Cv37PteqFnIUfVHmN0LJEx4jeKZshoLmNgRVWFSyzgipUi3YetyOIXpt8auZgseYiNZcYm9dtrO888FBSwbvG3vn\\\\\\\/HPIcND\\\\\\\/ZU\\\\\\\/Cv2h+jhQ\\\\\\\/mFkzqaZpJRyjUO8rga2kB0khJaHEOKkqZfiIXrP92meyNvqg5WviXkDXes+Kc1PnOTROCxukVLlvtW2pRdhh\\\\\\\/yQ0sddu5Fs8wu\\\\\\\/RNZFlovW3PJ+PIntGfMKn46smNOOrISNLHFO9k8571EsvavM43yZpao9bBcUUoqjTQ3ScHdVPgZoTtZmBraniO5n3d5YxfqZZ7SCCOH5HS9H+4VrBhtXe\\\\\\\/h56+h6H5dwotcqblYA6ysnL5laQv15wtae+MpZFXAYZX2WKTUPP4OnUtYSBxgzq8qb9WX9+PSZLGVnTmiTbGgV88OsiNB9QiRYMccid\\\\\\\/13G8LEuM+2Lhx8WxhatqMlasJemfmfEtZDS5dtVN1uhjAt1S\\\\\\\/FsKhOgLHGYWhqirdCxOsZUyudKXlwhAzDqIMQFHV75p6cuIKnaFnKrHjU9Jm5mS6+ojADlO\\\\\\\/z66K9iH7v\\\\\\\/hv7FQ1+5ro1sAsFjnWS3B4IlqYo+gBdmSLvQ1tYqI22W\\\\\\\/EhG4lLlIOmbKYbc99APzOQPgQa7As+xhU8XKs8fAMCWWdk8omxK9T861uJ9Pt5bhEVhBrgDmw0APSOlJQVQlCTZQdCtvHRFeUwNbMUhCWCF0ZEPYxqtLocpS6ANic69cSglYjJgDecWoWyPkzGUDPE+veR5NMAkpsI5p6yAjLcBLbX7DDkSCkisSE5lS8caMBFvkB7LHJJa1Eom4zGT+sw3gvxnlP\\\\\\\/fqZYPKFhDAfntS9j+UDceZB0nIIdNlBIulSef59I\\\\\\\/KnkNOORIVvuH2FXVnVD9dPsNJshxUCDtbQf7MHx9poT29yNmw41PBeNHSU6CFKaG7U1aOoMgPSs3TVW8lb1acIjLPzSJaxf206yMXqAOC6D7whn6dDqklu7nKxfIGEJVfuV\\\\\\\/GMUxZVTzbfjrceE2fgd6fZBYKJqKaYISTsMCutaEn611m\\\\\\\/5jQi0Vs4R7p9kwlSQTP7CPwYWjcpRGQDQPc8fb\\\\\\\/3wqV1FW50aqOFEKy+1gg2KfglWhIlmRJIfPl\\\\\\\/5On5uxtvhZ7ASdlzQUb27wI5mBBV5scGmOxDiD18StJpf5PW8ORWrNpAOFh3xPCUKXGL9dD\\\\\\\/V7O+vJUG4gbe1mBHQZ99WqvsBHx09urCvcvwvzyRPY5oq+GJGFonAy26Re7pBf\\\\\\\/ya3jXdfIk\\\\\\\/hH3T6yBE2En79etBsSHIdUC1KCwCfStlSVNVdRA\\\\\\\/5gGBRSwWJDd51kr+DjZxwxNpshq5tYcuKZQsDaXgU41Gz79\\\\\\\/SidmEHEyCMwI4z\\\\\\\/p\\\\\\\/IL8MVGrCux7e8UIF\\\\\\\/USU5MA7kkSIUokdyVCjCjEYDzfQ1CxtvwWzigLel+K0KfOoDmyZcI63jGxMqHtGFKf3cV0QhCoirIWDqMMO44klb\\\\\\\/QCqWFprbZPVBOOK0jVtBHhbw4Ho6uE3QAqj4wx+lpw8OrGjknfjTCtpU6UWJtKHoPdd267iUgKK7jnzjLsl2FoPKJSBMRjNx7UkUvKnOjOnahF1fbA5dRILcqPCc60lkg1\\\\\\\/DPDG97i3aOfSdUfAUhEd6zJPD+9LGVffYUVyHF6F\\\\\\\/gRmOeLOHjGSqoECEnPcxFo3RY\\\\\\\/QX\\\\\\\/NfvenZqVS3IViV2RT3gHHEfATEUVdRtOeKZYa7Gou7LhhTlCwGQfKMNIeYR4z+A5gxDjF9750heWnJArfGIdMc7iMKZWr\\\\\\\/lLBp18mhfiGD2ujzlfEw2lMkBM7tfDuBIcUlcjGCIYRUgfmoQgzSM8rKo7R1vfK7Y8yzDwG7V6spk5KSBV4YlZ1NOOWzIrNHeSFIZveuFr2\\\\\\\/CLXZoMD0z89gJW+mVlp4sHcpftziwY7i\\\\\\\/F922NhAbE5a7nINzRXhorvu3DqYLJgOkrcXd834H+rvU60ZIJdbaWxdl\\\\\\\/ws1vyYUfqEvDAmXS4zW5EDw38nVjATZxXGMLdsTirsSejM87FFO2uVfEN2y\\\\\\\/cHcrh9LFwXry6Bg\\\\\\\/PGn+8h9jZPjiHXRKr6XQvsaMDmM8nFLFeqrSFoMAguVNtNgrB0uA\\\\\\\/k5U3w3I\\\\\\\/Rm3Oz+GflOrVIJenB2jL7q5ScGGLrNNRXdDDYs3q9xOO+x4KtgOrbNucqyihL\\\\\\\/xcFxJFvp2J8Jjz1Ktb+pLS0UZKicc2MgpcrF04OUaXFGii2GLCtMxZDqq3BwLq7gJdOSRFuWtCQWyWaZWi5VaaJPMFsg1R4zpSkAopGiNFjuA9Uqg6XxrE2eLUECIBmudg0jbEhpCi\\\\\\\/4hr7+18u9QlxLkz290+OHLgrFYyZu182eSlbf97rZ36wwCv4Zvb\\\\\\\/bjeqH52dwpV+ZuFgyVFKs3wCpZGMNXhc+3r7vpHfp8rhwMhDWURq0R0iIMl\\\\\\\/laCaY2onB4UUPeIgP0xz4qzaMrEUoL8BKBm2McuILgQAypXyS98zEQPwPaE\\\\\\\/Q7p2L64SZWoO4nIMKdFzTPlwAO+5Jl4Rc4fPBVK2T87yT+lesopWSktLqR6QZBFZXxxO7Dqrdh69yHwc3JhjjOH7AeVcLQedWiyXeq40Ro0oabpWxSB9rE5Pqge2\\\\\\\/BhV7F98KqRi9dvVoKZouVS4OF9tYuwJfGJE2owUX0uw+TqnCjWo\\\\\\\/BjjbCQ1kiaIggw2rb4rSO0Lcg0llbR9QABbmWBPLcLFb9AH9JmneTwpW04n1qkhrvI9PgYPhS4fiGsoK8otctf6SYiijWfHBgdXMj9eAEJ+r0iqnqKJJrUVx7M7WLjmSshNok+SL4HZUK7sZaxbLEGAu+KAZGQ7tyHas0QfqAP6SaVWl0hp5CtRzlVC6mmJdF4F45BDFVFVdLFia\\\\\\\/LSHRtfOsziWIZaycJOPx1UluCyZ50te4VjHAgRRUu+JwSCs\\\\\\\/6T93z0yB25fGe47Cr4vn2DrrDZyFPDW3fgFFNCgtZuaiXwtlj7nlxuXBMshxuFNyS8Bv0kZmx0JPXzQ6h9y7UVZ6tv8MHfvfk65q1BZr2zXaNlNWRG9oD1waJF1G4kIVIGK\\\\\\\/SW7JWwl\\\\\\\/N4SXLfbHt\\\\\\\/pmFlAlKY1\\\\\\\/4DjFWVwuS7ireIi+MbZWqN+zJg2DRBmAqq\\\\\\\/PoV3m0BlbGG8b+2JGU5MeT4U5D971aFLouzDclqBEsZCF4UuOwwlcnu1Ifv9Mhv0BdU3p0bOsnVxpyu5dEFNB021wG5blZM8QHXlXLvaWEeYjSBasAcN3GlYxYZ29XATjV86HLGcGTpury6FJ\\\\\\\/TJ2aT27dn5zUOvGyj5r4dMkRq88WhshA6ev2f2eUapPjmB\\\\\\\/q0ZSYCGJyVmSgIrmf41ImIKf+pDr7JVIh\\\\\\\/hvxLrlctRAm5m3yYd8pa4Bf+yeO4yn5Bw+qhsN6OqzDh2n51Z4F0zKW5M9PthzIoyspz0BPlODU4r5EZWIfpEg+gHYMX80FBcSJYNZ6zpq2TdwHBSvvjILjYWf7nCsJDJYCcF8xzTYPzJ8elVZcn4LNoRJeiEHAiAXggpT13LZ+gfgz1RdSC5AC2GV8Vb1184ehkTRHI+3f27bV127Z+yK\\\\\\\/W3YmhI8+LUQv9SumEvQ\\\\\\\/mVvUDCBIo8N6BIKWTIn3+PAX5OLdiQOqRWYiVtCA+AdkgXGAe2opO7vwrhMZFke5KtsbEvDTyjFLGn+Tg6bNo72PHhzg9eN6ktzRL+AIMKxpH0ElLUUjx7v7nMBhKB6aWRPaevzO4cIzeXrKpWMkNxx16SsdcCgbPV0nzgZCrpKnj2oh2aqDEHgCtJ7i7kvuf1+JZi6M3bo9bgK6sU8bqHUGGFPn4RAyojELqgTPyK0\\\\\\\/GOmW+SpFGA1y7JMQsGm6Su7FddVKxbc8CEk77tR4YTB3KKhFCJ24FQVqHU7Ezf+mZIVaLWqmT+TeZFKlnlwFqCo5BaSFd54Sbxv6FzyGlSDYxrxUXT+lv90fUdx8\\\\\\\/8aiaeAVUbKbsQfYtEwRDTuEVzFg9UyKRL3ZX93R3i592fmuRF54BNc4MSs55F81aXn1Os4YeaDQd4Wnb9SOpaAJFOalz0TMeu5i2zYQuWkriL7jpooeP48sU6WRFtodzQ07KUyUkhNT4\\\\\\\/F6SpuTbu2R2InUcAab33nPXpSUJg+DISXsLpC6s9krLOSPAm7KGXRRMN6KuzgSU4Ttun2NYMgZlMOXJYNdJsUHTIYpnrtlKst3yu5G8\\\\\\\/w1\\\\\\\/09B235Lgjom99pMmHGUbuv5ISyKu+nilVPfzd8Sg4gNRUWxsr26G6exvccEc4oIs9zsWy63N1P2z8GuWWsgOy4SbT4czwHU0EBzij+MIS+Eb6f3Q95w8erc+rG6Gtz59+PKXZaNjdOpRO8BQSB8N7pW4NXGcJmIAi7CccDTHb9R5I16fZfN3eb9fYwAi2AO\\\\\\\/eP5qavqbnf4LGFR7y7rw6ETIKLXOZ95Py1dGZdLCK9Wwja+1gkjseqOhd3BbOudxBJjQtTidDpBlC2LeXrr3kaU6A6\\\\\\\/69w58+I\\\\\\\/5fKOgozUsT1b6\\\\\\\/iJg488TdQsgovBwsWNjDUO0lQmksRgh3bWbP\\\\\\\/lh2lCKizLnkFlpwGva8psQuLhp1AFHdHBhaKiFNOForoOFuEyABCPQ65HFqJPyVAgD21BtXsppJ906+q3BYRUFkZlGknVU\\\\\\\/imF9MzAp6WzSxxXfsNBlhiyNOswegDYSCDNfOSeTxiJpei8kuNmu\\\\\\\/esOX2tZ9hBTfXFeu3W0O7TZBBPjgfrGR1Y78xLJ29yLMoEfsf8zNTC6EPLbcI1LqCMOMK67mjctNf48PiV0XcR5VcQGtTay9Bv8JQhNwzdG8MCh963UDjF0scA0SLaOAUYDvhm\\\\\\\/roomEPurcfr3L9Q8pCtyCg3JQ5VsLzP1jdjd2TCc1rsrvCgZN98++Ib\\\\\\\/lIBC7ZJ\\\\\\\/oRepfb5fEkVUKNmRRc23BOL+sDGug+dmd8WR9+vIduZWY53t1SvZDfIAwZyt9NH2LoDRHe9wqrxE6SNLSDPIEEAvxGV\\\\\\\/9oxAxN+5D4w\\\\\\\/cZ2kXSK9NpuVUMFwej3DiHRsH3H1Su3vnqhzz8rRFaet\\\\\\\/PmdwwSyGpNCUKDWTXL8D6HTdJDlLf0SwnHaFSulCbqgEma+kUEA8vFNJq8kRCrrBD8yXedZSQ0dU6k8rNcQFTYas1A6ch\\\\\\\/99MNE6hdYclb8ABbXVLTZg+QQeuP9rkXkZBufJLHEAV3HmDgetVGWtMaIniG6vFbmqZo4N91ccBxgFXE4IeJO2dS6c+chFu4nNs\\\\\\\/dNudimftN\\\\\\\/QSEUZuNLu4I0WyKTSdjJ2HTu8w0TV\\\\\\\/nreQBjXvgff23WJwiOvGudNROSOKCmGFiEfPZKhUQ35z5nM+1\\\\\\\/kQqUp60RDoldTtYo9u94G53UcvhAH\\\\\\\/ngfbBD+4+rgvufyQ28N+n6eqmqAbhl4M9e9NhsSWm449P\\\\\\\/Pc7CfUv34O9YhbC+OALRLXqvWsdlLb6YnXuWC5rn2Aflfq1TfEpIFSZsecOrQZKi6VuvoDq8ioW9AxUBkJ22eT6TmkNGNLyPrFx9SBuKmHcTeA+BMXQlQ+kue8gB\\\\\\\/pd9ou\\\\\\\/CENAvWA3EX1oV8M4uOK1GMhVP95uX1rdMai7DlzFM0WjeQ7pw8aVm8Q+gLMObTp0grYAHhdyKuEofHw5XR5n2J1kTH6WuY8OclRM78rgnLLBB2QvgYdomOz0n7rGimvArSqhS0eLEtVZCT506nS\\\\\\\/wPwqafJ+e6pc1BUhiXAJDolg9tn20+CdgKxB1x3gL1dPBBcieM3ntfz1PRsWDTBnIwlun\\\\\\\/82uwDnZCgFuDTNSdJ84+XSeWMhDetd0crrAskzmd2+ShZbCG01mqtR0xnUY9WXjniLnNOyHQAUJVTOaHrvC8ZAAOA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Bw3v5mF2JpT20CwHKUdE3g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229f41687ee0482b525896c9d7b58e9fa582be81f2fc0dafffe4a1875d2fbd4bff\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00226TbwKw06NHMP2AeRDDfRmiyg1ehbqJgtcdFqHCscRcsi\\\\\\\/snifISKtOk5qR\\\\\\\/JWEXcppV3svBEAQWkqX0CpwV\\\\\\\/4LVzDy38PyNMFhRRuTzW5r5OJbgwUianjC0eoDSqfCdbpYLA88C2QMal+Sxe9fJ3fsWXgzsZwTG\\\\\\\/yKoTi0x6AS1Jk951DiC85YR\\\\\\\/pj8dRN0v7mynW71CxKQCCsCrw0RNEfNd69DDetCmBXQlSYI8fx7nrz5BwEsQeMNVpxu19w0d6fcrIP0yGejVUCcyBf0TUzIT0Khq4k3P8tCTWDsqwxGZ3L\\\\\\\/otG4UvrA8mIaQ+nY1b39ZygCZGxl4zJgg97Ltn1yk9qyrtxmc+\\\\\\\/7yJNNN3sjknN2iCRq2pRKi233JNRvQev5i8df9KJ+pCc1AkIdCT4WuTQ8dgY2kcOKiyZ4W0MF1nCCcPuwpyNOpjWsFpQUufHln\\\\\\\/\\\\\\\/6RtWklUl3Xo2KvXDNH9L2oSz84Tu9H40N\\\\\\\/6IVVwEXdtgY4JfD\\\\\\\/LKF+\\\\\\\/cDbm08G4kgA5vaxyW1AdNJfXKYvliGhLGMnNtIwhCZQYjii7c3c27jj+cqPwA1nV0657mUS5mB0TENreprFWz\\\\\\\/+3QCuUv36+3wdjX7o26zLLh9rkcmVLbnL4AfCiMOOmXNuYrZCPM4NIYkGJytdn2\\\\\\\/\\\\\\\/H2I\\\\\\\/dKzOn6HCDDMRS4uGkHSHQzfLS9Ff7odlFSEkVJJP3EkXbdtNxevNDzVStKBWFhKe64FTxeQlNahY\\\\\\\/ZQQAsgwB0j0cYLh\\\\\\\/qIfIg\\\\\\\/45rQot2vq1Yu9KF2DP2tZicdaSlTRDXJ7X2lVSoTmq4O37u1bIua2Ela02GK57GintuQbMf+ui9Be7FX3zDoj0SJTmLFJt9lMCBaXgWYs8asicKVAVtC3Sw+zntqGrIkNwvhdS43TlgAKAWGzN1Z\\\\\\\/bDMQdOGaVRouDTclCCx2q7k8froC6VxJeYREayYMxS3jGoUGZSTNe+yqFc2MV82miqQ8X6gXtC0SPHlUo0Fj4TLpm8euoXeSSh\\\\\\\/9oYx\\\\\\\/kRroD\\\\\\\/z1VqJEmA+AULi3XWN1nBtWno2sluHmFK\\\\\\\/bZ2E5u60DLV4Jq4HtzuJKSkYYk3dlNYwJR+M8Wcger4q+u4yTP9oTGZJm5wNN1YFOX5Iaew5G1t5vzOICazCb3T+3OtYJi0S4\\\\\\\/EsDmvTFuLNI0u3KvqhTKTaPRh+1ilvgaMVLVlvWzTWBPsLkCUwm76OLYA51dSAJUsam0s00Z0aQ5Sgv18gaGEz5nqkUPOHM6wI7Qy5stgxJDIevobJRcpPmWlz5bt4wMlwbhY4zlSYPd4hC\\\\\\\/GpqX4l2xW1M7WQBjWWU4mwNyIrwESvvOEqGUIRsfRJmRf3AxXkxxpsHxMHwVMJOZeWEH9TR1J3B7Rm2fapUikQYqJWzVvj\\\\\\\/z06vN67Epc+gYsavPApUVsovTrtvBe\\\\\\\/dXIac+L\\\\\\\/0t1Wiui4oQSXk6lr6+CJUetDleMXluKaYG1WoFV53rxJAUbniZWR2stVkXkpck4+2BlUzhbdU6KA1sNNK8XgDEA1yMNXuuq1gNNotaZKAgs0vzrDLL\\\\\\\/pbVLXFgWM3ihv4XZN2jrRIxDCwRfREdfcpra5tqQl5Du5mIDhoSTuGriX0jWCcgakbQvmeLBSIReA7WJbkwTKkEw71KDJ11Q6y8CZtDHt4RWTHWNScSyis1QeLNLAd\\\\\\\/4AIw5ydjb1mk+l21hxCWFsVNNiSnwl\\\\\\\/MfEJjmS+hKgB+LqjHxHXFTQIW7GUG9b7LqNoHv57dqp7QXIqreIK5tjUnpO6bmawUpcujjzwa2YLNQpld+Q2VIfy6KQwq5ycufwUgi5DVh8HFlba6fSs2JnsPJSbuXvcaJnAlvuEXwjJej7GnaeciLxaTqtcBqBvi65oy3RREFH4NTRDpeyhg4UQhfqf19iQOnga5HVTJYZrDxcmaTjJfhLPHQ2JzWiSFmdN46OWr3HOuqxCFIgFeSxlAZSlosie2H8j7YYxxYq\\\\\\\/Q05fW1DbnSPSVvmqnTeHySHKNYBRNLZwig==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ot\\\\\\\/rR35\\\\\\\/EnZhB3v3Ymst1Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229dbd92e9d69a7ce114f545fefd125b1cb281f2b6e68c529f8027afb54a46ab19\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-493746 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/8/2511" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-U2gYYb7aQmXPy5atjdRRYtuMV7YCT2C9CzZKBt8KlQw" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/8/2511" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-I0n4F_90sOIl0xIxY9uJxUW4NrkkFF9LJ1ouz2cYG6o" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/8/2497" title="Structural Characterization of ABCB1, the Gene Underlying the d2 Dwarf Phenotype in Pearl Millet, Cenchrus Americanus (L.) Morrone" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/8/2521" title="Open Chromatin Profiling in Adipose Tissue Marks Genomic Regions with Functional Roles in Cardiometabolic Traits" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493746" id="node493746" data-pisa="ggg;9/8/2511" data-pisa-master="ggg;g3.119.400201" data-apath="/ggg/9/8/2511.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Molecular Traits of Long Non-protein Coding RNAs from Diverse Plant Species Show Little Evidence of Phylogenetic Relationships</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9832-7797" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Caitlin M. A. Simopoulos</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-0399-6232" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Elizabeth A. Weretilnyk</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7575-0282" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>G. Brian Golding</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2511-2520;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400201 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493746" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Caitlin M. A. Simopoulos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, McMaster University, Hamilton, Ontario, Canada, L8S 4K1</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Caitlin%2BM.%2BA.%2BSimopoulos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Simopoulos%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACaitlin%2BM.%2BA.%2BSimopoulos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9832-7797" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Caitlin M. A. Simopoulos</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Elizabeth A. Weretilnyk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, McMaster University, Hamilton, Ontario, Canada, L8S 4K1</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elizabeth%2BA.%2BWeretilnyk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weretilnyk%20EA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AElizabeth%2BA.%2BWeretilnyk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0399-6232" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Elizabeth A. Weretilnyk</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">G. Brian Golding </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, McMaster University, Hamilton, Ontario, Canada, L8S 4K1</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=G.%2BBrian%2BGolding%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Golding%20GB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AG.%2BBrian%2BGolding%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7575-0282" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for G. Brian Golding</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:golding@mcmaster.ca" class="" data-icon-position="" data-hide-link-title="0">golding@mcmaster.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/8/2511" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:493746" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:493746/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/8/2511.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:493746" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:493746/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/8/2511.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:493746" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:493746/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/8/2511.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:493746" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:493746/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Long non-coding RNAs (lncRNAs) represent a diverse class of regulatory loci with roles in development and stress responses throughout all kingdoms of life. LncRNAs, however, remain under-studied in plants compared to animal systems. To address this deficiency, we applied a machine learning prediction tool, Classifying RNA by Ensemble Machine learning Algorithm (CREMA), to analyze RNAseq data from 11 plant species chosen to represent a wide range of evolutionary histories. Transcript sequences of all expressed and/or annotated loci from plants grown in unstressed (control) conditions were assembled and input into CREMA for comparative analyses. On average, 6.4% of the plant transcripts were identified by CREMA as encoding lncRNAs. Gene annotation associated with the transcripts showed that up to 99% of all predicted lncRNAs for <em>Solanum tuberosum</em> and <em>Amborella trichopoda</em> were missing from their reference annotations whereas the reference annotation for the genetic model plant <em>Arabidopsis thaliana</em> contains 96% of all predicted lncRNAs for this species. Thus a reliance on reference annotations for use in lncRNA research in less well-studied plants can be impeded by the near absence of annotations associated with these regulatory transcripts. Moreover, our work using phylogenetic signal analyses suggests that molecular traits of plant lncRNAs display different evolutionary patterns than all other transcripts in plants and have molecular traits that do not follow a classic evolutionary pattern. Specifically, GC content was the only tested trait of lncRNAs with consistently significant and high phylogenetic signal, contrary to high signal in all tested molecular traits for the other transcripts in our tested plant species.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/lncrna" class="hw-term hw-article-keyword hw-article-keyword-lncrna" rel="nofollow">lncRNA</a></li><li class="kwd"><a href="/keyword/crema" class="hw-term hw-article-keyword hw-article-keyword-crema" rel="nofollow">CREMA</a></li><li class="kwd"><a href="/keyword/phylogenetic-signal" class="hw-term hw-article-keyword hw-article-keyword-phylogenetic-signal" rel="nofollow">phylogenetic signal</a></li><li class="kwd"><a href="/keyword/rnaseq" class="hw-term hw-article-keyword hw-article-keyword-rnaseq" rel="nofollow">RNASeq</a></li><li class="kwd"><a href="/keyword/evolution" class="hw-term hw-article-keyword hw-article-keyword-evolution" rel="nofollow">evolution</a></li></ul><p id="p-3">Long non-protein coding RNAs (lncRNAs), a heterogeneous class of regulatory transcripts, remain greatly understudied in plant species. Although these transcripts have been implicated in development and stress responses of plants, only 13 of these transcripts have been empirically functionally characterized to date (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Wang and Cheksnova 2017</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Nejat and Mantri 2018</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Zhao <em>et al.</em> 2018</a>). While researchers often focus on computational prediction of these transcripts, particularly lncRNAs expressed under stressful conditions, biological insights on the evolution, mechanisms and function of lncRNAs remain uncertain.</p><p id="p-4"><a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Simopoulos <em>et al.</em> (2018)</a> reported that the genome of <em>Eutrema salsugineum</em>, an extremophile, contains a lower proportion of putative lncRNAs in comparison to the genome of model plants <em>Arabidopsis thaliana</em> and <em>Oryza sativa</em>. A lower number of predicted lncRNAs in <em>E. salsugineum</em> is surprising due to the naturally high capacity of this species to tolerate extreme environmental conditions (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Champigny <em>et al.</em> 2013</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Kazachkova <em>et al.</em> 2018</a>) and the oft-cited association between expressed lncRNAs and stress responses (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Wang <em>et al.</em> 2017</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Xu <em>et al.</em> 2017</a>). <em>E. salsugineum</em>’s unexpectedly low number of predicted lncRNAs compared to its close and more stress sensitive relative <em>A. thaliana</em> leads to questions of potential natural variation in lncRNA number. However, the differences in predictions of lncRNAs in these species may be due to data availability as few plant species have had their reference annotation updated regularly in genomic databases. For example, novel gene information has yet to be updated for <em>E. salsugineum</em> since the official reference genome was released in 2013 (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Yang <em>et al.</em> 2013</a>) although <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Champigny <em>et al.</em> (2013)</a> presented an additional 665 transcriptional units for which the reference genome had no annotation. Recently, <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Yin <em>et al.</em> (2018)</a> have added to the number of novel transcripts in <em>E. salsugineum</em> with evidence of expression of an additional 65 transcripts, none of which are available in the reference annotation of <em>E. salsugineum</em>.</p><p id="p-5">LncRNAs may be missing from genome annotations because they are difficult to identify due to their low, tissue- and condition-dependent expression (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Derrien <em>et al.</em> 2012</a>). Further, contrary to protein-coding genes and other non-coding loci, the evolution of lncRNAs is not well understood. Limited nucleotide conservation has been identified in mammalian lncRNAs (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hezroni <em>et al.</em> 2015</a>), and structural conservation remains controversial (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Rivas <em>et al.</em> 2017</a>). Instead of using homology, distinguishing traits such as transcript length (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kapranov <em>et al.</em> 2007</a>), open reading frame (ORF) (or lack of) length (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Kapranov <em>et al.</em> 2007</a>), GC content (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Niazi and Valadkhan 2012</a>), and number of exons in a transcript (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Derrien <em>et al.</em> 2012</a>) are often used in lncRNA prediction studies. Detected phylogenetic signal in traits of transcripts, rather than sequence homology, can indicate that trait values follow the expected evolutionary patterns of tested species. For example, high phylogenetic signal implies traits are more similar in closely related species, whereas low phylogenetic signal suggests the opposite: less similarity in tested traits than expected in closely related species. However, identifying which evolutionary process may be influencing a significant phylogenetic signal is complex and many different processes are associated with both high or low signal estimates (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Revell <em>et al.</em> 2008</a>).</p><p id="p-6">Phylogenetic signal can be measured using many different indices, however three different approaches are prevalent throughout estimation methods: Brownian motion, an Ornstein-Uhlenbeck process and spatial autocorrelation. High phylogenetic signal detected by a signal estimation method that uses the theory of Brownian motion, or evolution following a random walk, can be observed in both natural selection and genetic drift scenarios (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Revell <em>et al.</em> 2008</a>). Conversely, low detected phylogenetic signal can be inferred as the lack of similarity in tested traits, as opposed to divergence of traits, and is common in adaptive radiation or other fast adaptive processes (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kamilar and Cooper 2013</a>). First described with applications to evolution by <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Hansen (1997)</a>, the Ornstein-Uhlenbeck process allows for a random walk, similar to Brownian motion, but also for species to evolve toward an adaptive peak or fitness optimum, thus suggesting data that fit an Ornstein-Uhlenbeck process as evidence of an adaptive process. Furthermore, local estimates of Moran’s I, based on the concept of spatial autocorrelation, estimate phylogenetic signal throughout evolutionary time. Positive autocorrelation indicates similarity of trait values at a given phylogenetic distance, while negative autocorrelation suggests dissimilarity at a given phylogenetic distance. <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Diniz-Filho (2001)</a> has shown, however, that it is the changes in local autocorrelation over phylogenetic distance beyond a significance threshold that are important for evolutionary process inference. A trait following an Ornstein-Uhlenbeck adaptive process would have a reduced phylogenetic distance at which Moran’s I changes in magnitude and crosses a threshold of no significant autocorrelation, also called a “phylogenetic patch”. Thus, it is necessary to test for phylogenetic signal in traits using multiple and diverse estimation methods to infer putative evolutionary processes that may be driving significant signal.</p><p id="p-7">In this study, we predicted lncRNAs from transcriptomes of 11 plant species with widely different evolutionary histories. Transcripts were assembled from RNASeq data without restriction of existing reference annotation in order to obtain a representation of all expressed loci in each study. Transcript sequences were then input into CREMA (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Simopoulos <em>et al.</em> 2018</a>) for accurate lncRNA prediction and ranking. Unlike other lncRNA prediction tools, CREMA is trained only on experimentally validated lncRNA transcripts and has been tested for use on transcript sequences of multiple plant species. Following lncRNA prediction, we observed that up to 99% of predicted lncRNAs may not be present in their corresponding reference annotation. Thus, we caution that researchers should not rely only on publicly available annotation for lncRNA research. Finally, as there has been little evidence for sequence conservation in lncRNAs between species in different families (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Nelson <em>et al.</em> 2016</a>), a phylogenetic signal was not expected in the distinguishing molecular traits of lncRNAs, such as transcript length and GC content. However, our comparative study detected a consistently high phylogenetic signal in GC content of lncRNAs with no similarity identified in the other traits tested in these regulatory transcripts. In particular, GC content differences relative to protein-coding RNA represent a trait that could help researchers distinguish putative functional lncRNAs from non-functional and spurious transcription, or fragmented protein-coding RNAs.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Data collection</h3><p id="p-8">RNASeq data from multiple plant species were downloaded from the SRA database (See <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> for accession and SRA IDs). All plants in this analysis have a publicly available sequenced genome. All RNASeq reads except for those from <em>E. salsugineum</em> were downloaded from the Sequence Read Archive (SRA) (<a href="https://www.ncbi.nlm.nih.gov/sra">https://www.ncbi.nlm.nih.gov/sra</a>). To be considered for analysis, plants must have been grown under control conditions without being subjected to stress. For consistency, preference was given to studies that used leaf tissue from mature plants, although use of older seedlings was accepted. Leaves or leaf-like tissue represented a shared morphological feature of photosynthetically active cells throughout all tested species and was used throughout the study to ensure an equal opportunity for capturing lncRNA expression. Additionally, only RNASeq reads from Illumina technology were considered, however both paired and single end reads were used.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493990/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493990/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Information on the data sources of RNASeq libraries used in this study</span><div class="sb-div caption-clear"></div></div></div><p id="p-9"><em>E. salsugineum</em> reads were sequenced using Illumina technology from Shandong ecotype rosette leaves grown under control, unstressed conditions as outlined in <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">MacLeod <em>et al.</em> (2015)</a>. Fully-expanded leaves were used for RNA sequencing, and were collected between 8 and 10 hr into the day cycle. RNA was extracted from leaves flash-frozen using liquid nitrogen using a modified hot borate method as described by <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Champigny <em>et al.</em> (2013)</a>. A quality control analysis was completed on the RNA using RNA Nano 600 chips on a Bioanalyzer 2100 and purified using three on-column purifications by Genelute mRNA miniprep kit (Cat. No. MRN10, Sigma). Finally, preparation of cDNA for sequencing was performed with the NEBNext multiplex cDNA synthesis kit for Illumina using random hexamers (Cat. No. E7335, New England Biolabs, Ipswich, MA). Cleanup of fragmented RNA was performed with Agencourt AMPure XP Beads (Cat. No. 163987, Beckman Coulter, Mississauga, ON) following the manufacturer’s protocol. Raw FASTQ files were deposited to the SRA with submission ID SRR7962298 and BioProject accession PRJNA494564.</p></div><div id="sec-3" class="subsection"><h3>Transcript assembly and lncRNA prediction</h3><p id="p-10">Reads from all plant species were trimmed using Trimmomatic v0.36 (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bolger <em>et al.</em> 2014</a>) and aligned to their corresponding genomes using STAR v2.5.2b (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dobin <em>et al.</em> 2013</a>) with default settings other than–outFilterIntronMotifs set to RemoveNoncanonical and–alignEndsType EndtoEnd. Aligned reads were assembled into transcripts by StringTie v1.3.4d (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Pertea <em>et al.</em> 2015</a>). GTF files of assembled transcripts were merged with GTF files of annotated genomes and are stored on GitHub: <a href="https://github.com/caitsimop/lncRNA-compGenomics">https://github.com/caitsimop/lncRNA-compGenomics</a>. Alignment quality was tested using gffcompare v0.10.4 (<a href="https://github.com/gpertea/gffcompare">https://github.com/gpertea/gffcompare</a>) by comparing assembled transcript GFF files with reference genome GFF files. Alignment quality metrics were used to confirm alignment quality and transcript assembly quality using accuracy and precision values (File S2). Gffcompare output was also used to identify novel transcripts and to quantify transcript exon numbers in each RNASeq library.</p></div><div id="sec-4" class="subsection"><h3>Identifying lncRNAs From RNASeq data</h3><p id="p-11">Assembled transcript sequences were input into CREMA (<a href="https://github.com/gbgolding/crema">https://github.com/gbgolding/crema</a>) (<a id="xref-ref-52-3" class="xref-bibr" href="#ref-52">Simopoulos <em>et al.</em> 2018</a>) for ranked lncRNA prediction. The number of lncRNAs in each species was calculated as a percentage of all transcripts (the sum of novel assembled transcripts and transcripts in reference annotation). The percentage of lncRNAs was used for normalization across all studied plant species to ensure appropriate comparisons to species with different sized transcriptomes.</p></div><div id="sec-5" class="subsection"><h3>Phylogenetic signal in lncRNA traits</h3><p id="p-12">Four continuous molecular traits were chosen for phylogenetic signal analysis on predicted lncRNAs: 1. Number of exons in transcript, 2. GC content of transcript, 3. Length of transcript, and 4. Length of maximal ORF. Features were extracted from transcript sequences and gffcompare outputs using a custom Python script. The phylosignal R package (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Keck <em>et al.</em> 2016</a>) was used to detect phylogenetic signal in lncRNAs, all other transcripts, and the differences between lncRNAs and all other transcripts for each trait in all species except for <em>Boea hygrometrica</em>. Separate phylogenetic signal tests were completed for each trait. Although we expect there to be correlation between transcript length and ORF length, we did not observe a correlation, particularly in lncRNAs. Phylogenetic signal of the mean value of the four traits was calculated using three separate methods: Moran’s I (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Moran 1948</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gittleman and Kot 1990</a>), Blomberg’s K (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Blomberg <em>et al.</em> 2003</a>) and Pagel’s λ (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Pagel 1999</a>). Local autocorrelation estimates at 100 phylogenetic distance points were also computed using Moran’s I and phylosignal to identify the location and sign of the detected autocorrelation. To identify significant autocorrelation estimates, 1000 bootstrap replicates were used for 95% confidence interval calculation. Autocorrelation estimates were considered significant if 95% confidence intervals did not overlap the null hypothesis threshold of -0.111. The null hypothesis that there is no detectable phylogenetic signal, or, autocorrelation, was a threshold of <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2511/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2511/embed/mml-math-1.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2511/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2511/embed/mml-math-2.gif"/></noscript></span></span>, or the number tested species, as suggested by <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Keck <em>et al.</em> (2016)</a>. Because branch lengths were required by the phylosignal package, branch lengths were estimated using the dnaml program in PHYLIP (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Felsenstein 1993</a>) and a MAFFT v.7.205 (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Katoh <em>et al.</em> 2002</a>) alignment of <em>rps16</em>, <em>atp2</em>, <em>18s</em>, <em>26s</em> and <em>SMC1</em> (File S4). <em>B. hygrometrica</em> was not included in phylogenetic signal analysis due to the limited percentage of annotated loci in genome annotation. The tree topology of land plants as reported by the <a id="xref-ref-1-3" class="xref-bibr" href="#ref-1">Amborella Genome Project (2013)</a> was used, and branch lengths were estimated from this topology. Branch lengths representing site changes were converted to relative age of branches using the R package ape (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Paradis and Schliep 2019</a>). A lambda value of 0 was chosen from 0, 0.1 and 1 after testing for the lowest log likelihood of lambda options.</p><p id="p-13">Trait values with high K values (K &gt;1) were chosen for further testing for better fit to models that consider an Ornstein-Uhlenbeck process and may indicate selection on traits. Traits values were fit with macroevolutionary models using the geiger (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Harmon <em>et al.</em> 2008</a>) R package and the fitContinuous function. Both “BM” (Brownian motion) and “OU” (Ornstein-Uhlenbeck) models were considered. Log likelihood estimates were used to test for the goodness of fit for “BM” and “OU” models while also considering the number of parameters that each model contains.</p></div><div id="sec-6" class="subsection"><h3>Data Availability</h3><p id="p-14">Raw FASTQ RNA sequencing data are available in the SRA with submission ID SRR7962298 and BioProject accession PRJNA494564. Ranked lncRNA prediction scores and GFF files of assembled transcripts are available on the author’s GitHub <a href="https://github.com/caitsimop/lncRNA-compGenomics">https://github.com/caitsimop/lncRNA-compGenomics</a>. Classifications of lncRNAs made by gffcompare are made available in File S1. Quality of transcriptome assemblies are available in File S2. The FASTA file containing sequences of the genes used in the estimation of branch lengths is available in File S3. The phylogenetic tree with branch lengths adjusted relative to time is found in Figure S4. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8250953">https://doi.org/10.25387/g3.8250953</a>.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Multispecies lncRNA prediction</h3><p id="p-15">We chose plant species with diverse and divergent evolutionary histories for lncRNA prediction comparisons. Specifically, we included Angiosperms (both monocots and dicots), a Lycophyte, a Bryophyte and an algal species for this work (see <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> in Materials and Methods). As novel transcripts were of importance to this work, RNASeq data from published experiments were used when available to assemble transcripts or were prepared <em>de novo</em> as in the case of one species (<em>E. salsugineum</em>). To be chosen for this study, data from the SRA database had to include samples produced with Illumina sequencing technology. Species chosen must have publicly available reference genome sequences and corresponding annotation. For consistency, leaf tissue from mature plants was preferred but younger leaves, or entire organisms in the case of <em>Chlamydomonas reinhardtii</em>, were used. To remove any potential biases toward transcripts induced by stress, only control, unstressed samples were chosen for analyses. After read mapping to appropriate plant genomes, transcripts were assembled using StringTie allowing for identification of novel transcripts. Sequences of assembled transcripts were input into CREMA, a lncRNA prediction tool (<a id="xref-ref-52-4" class="xref-bibr" href="#ref-52">Simopoulos <em>et al.</em> 2018</a>) and total lncRNA numbers in each plant species are described in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. Ranked prediction scores of all transcripts in each species are available on GitHub: <a href="https://github.com/caitsimop/lncRNA-compGenomics">https://github.com/caitsimop/lncRNA-compGenomics</a>. The percentage of total transcripts predicted as lncRNAs range from 3% in <em>E. salsugineum</em> to 16.6% in <em>Amborella trichopoda</em> with a mean percentage of 6.4% <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2511/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2511/embed/mml-math-3.gif"/></noscript></span></span> for the 11 analyzed plant species (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). On average, 52% of the predicted lncRNAs were found in intergenic regions leaving almost half of the predicted lncRNAs overlapping with a protein coding gene or as putative splicing variants (File S1).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2511/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Total predicted lncRNAs from 10 plant species. The counts of putative lncRNAs are categorized by transcripts that appear in the reference annotation of each species (purple) and novel transcripts, or those that did not appear in transcriptome annotation (coral). The percentages of novel transcripts (coral) predicted as lncRNAs appear above each bar." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-220780455" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Total predicted lncRNAs from 10 plant species. The counts of putative lncRNAs are categorized by transcripts that appear in the reference annotation of each species (purple) and novel transcripts, or those that did not appear in transcriptome annotation (coral). The percentages of novel transcripts (coral) predicted as lncRNAs appear above each bar.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2511/F1.medium.gif" width="440" height="168"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/8/2511/F1.medium.gif" width="440" height="168"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2511/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2511/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493724" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-16" class="first-child">Total predicted lncRNAs from 10 plant species. The counts of putative lncRNAs are categorized by transcripts that appear in the reference annotation of each species (purple) and novel transcripts, or those that did not appear in transcriptome annotation (coral). The percentages of novel transcripts (coral) predicted as lncRNAs appear above each bar.</p><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493897/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493897/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Number of predicted lncRNAs in each species ordered by phylogenetic relationship</span><div class="sb-div caption-clear"></div></div></div><p id="p-18">To determine how reference annotation may affect lncRNA research in plant systems, all assembled transcripts from each species, including novel transcripts, were compared to those found in the corresponding reference annotation. Transcripts that were not found in reference annotation and also predicted as a putative lncRNA were identified and are referred to as “novel” lncRNAs throughout this manuscript. The proportion of novel lncRNA in all predicted lncRNAs ranged among species from a low 4.5% predicted in <em>A. thaliana</em> and a high 99.6% in <em>Solanum tuberosum</em> (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Because <em>A. thaliana</em> is a well studied model plant with an almost fully annotated genome, we expected this species to have fewer novel transcripts assembled from the RNASeq data. Additionally, we expected that most lncRNAs predicted from the <em>A. thaliana</em> transcriptome to already be found in the reference annotation. The high percentage of lncRNAs found in the reference annotation of <em>A. thaliana</em> indicates that CREMA makes accurate predictions and suggests that the lower percentages of known lncRNAs identified in the other species are due to incomplete annotations (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>).</p></div><div id="sec-9" class="subsection"><h3>Phylogenetic signal in molecular traits of plant lncRNAs</h3><p id="p-19">We first considered overall trends in some typical distinguishing traits of lncRNAs: ORF length, GC content, number of exons, and transcript length. All species showed a similar trend where putative lncRNAs had a lower GC%, fewer exons, and shorter ORF length compared to the other transcripts in their corresponding transcriptomes (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). Length of transcripts, however, deviated from this trend where <em>Zea mays</em>, <em>Selaginella moellendorffi</em> and <em>A. trichopoda</em> all have putative lncRNAs longer than other transcripts in their transcriptome (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Deviation from the expected trend of shorter lncRNA transcripts in three of the selected species suggests that transcript length may not be a useful distinguishing trait in lncRNA prediction.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2511/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mean trait values of transcripts predicted as lncRNAs (coral, circle) and all other assembled transcripts (purple, triangle). Species are ordered as per phylogenetic relationships." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-220780455" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Mean trait values of transcripts predicted as lncRNAs (coral, circle) and all other assembled transcripts (purple, triangle). Species are ordered as per phylogenetic relationships.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2511/F2.medium.gif" width="440" height="169"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/8/2511/F2.medium.gif" width="440" height="169"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2511/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2511/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493900" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-20" class="first-child">Mean trait values of transcripts predicted as lncRNAs (coral, circle) and all other assembled transcripts (purple, triangle). Species are ordered as per phylogenetic relationships.</p><div class="sb-div caption-clear"></div></div></div><p id="p-21">We tested for phylogenetic signal in mean trait values of the four molecular traits previously mentioned in both lncRNAs and all transcripts other than lncRNAs. Phylogenetic signal estimates were calculated using three different methods that employ two different models of evolution, Moran’s I, Pagel’s λ and Blomberg’s K. We estimated phylogenetic signal in all species but <em>B. hygrometrica</em> due to the incomplete status of its genome annotation.</p><p id="p-22">Since each phylogenetic signal estimation method is based on different concepts, all estimates cannot be interpreted the same. First, Moran’s I is a measure of autocorrelation (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Gittleman and Kot 1990</a>). Autocorrelation, when referring to phylogenetic signal, indicates how correlated traits are in terms of phylogenetic distance. Due to the use of 10 species, Moran’s I must be compared to a calculated threshold of -0.111 (<a id="xref-ref-31-3" class="xref-bibr" href="#ref-31">Keck <em>et al.</em> 2016</a>) to determine significant autocorrelation and phylogenetic signal. A significant estimate greater than -0.111 indicates positive significant global autocorrelation and that trait values of closely related species are more similar to each other. Conversely, a significant estimate less than -0.111 suggests global significant negative autocorrelation. The Brownian motion model, originally used to describe the motion of particles suspended in fluid, is another model used to describe how traits evolve through time. In the case of phylogenetic signal, a trait following the Brownian motion model exhibits a random walk where the value of the trait can change in any direction at any time. Pagel’s λ uses this Brownian motion model and can be interpreted as the transformation the phylogeny requires to explain trait distribution if the trait followed Brownian motion (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Pagel 1999</a>). Thus, a value of 1 would indicate a phylogeny as expected under Brownian motion and high phylogenetic signal, and a significant value of 0 would mean a trait distribution that does not follow Brownian motion, consequently, low phylogenetic signal. Finally, Blomberg’s K, which also uses the Brownian motion model, can be interpreted as the ratio of observed values over expected values if the trait follows the Brownian motion model (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Blomberg <em>et al.</em> 2003</a>). A value of K = 1 can be interpreted as a trait distribution following Brownian motion, and as K becomes larger than 1, a stronger signal is detected. Conversely a value of K &lt; 1 indicates low phylogenetic signal, and less similarity between closely related tested species.</p><p id="p-23"><a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> shows phylogenetic signal estimates using all three methods for each of the four molecular traits. The mean trait values and phylogenetic relationships of the tested species are presented in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>. In predicted lncRNAs, GC content was the only trait that demonstrates high phylogenetic signal using all phylogenetic signal detection methods (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>). While ORF length was had a significant positive global autocorrelation (I = 0.040; <a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>), a value of K &lt; 1 indicates less similarity than expected under Brownian motion, suggesting unclear phylogenetic signal estimation. Blomberg’s K also suggests less similarity than expected in the number of exons of lncRNAs, however no other method indicated detectable significant phylogenetic signal. Transcript lengths of lncRNAs in tested species also demonstrate a moderate positive global autocorrelation. Conversely, all four traits consistently had significant phylogenetic signal estimates when all transcripts other than lncRNAs were evaluated, although λ for ORF length, number of exons and transcript length were slightly less than 1.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493844/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493844/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Phylogenetic signal estimates in transcript features</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Evolutionary processes and phylogenetic signal</h3><p id="p-27">We examined traits with an estimated K &gt; 1 with an evolutionary model that may suggest natural selection, the Ornstein-Uhlenbeck process (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Hansen 1997</a>), because high phylogenetic signal defined as K &gt; 1 (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Kamilar and Cooper 2013</a>) can indicate similarity by both genetic drift and natural selection. In lncRNAs, GC content is the only trait with K &gt; 1, and has significant phylogenetic signal detected using all three methods (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>). We compared the fit of a Brownian motion model <em>vs.</em> an Ornstein-Uhlenbeck model in our data using log likelihood values and a chi square test for significance estimates. Although the Ornstein-Uhlenbeck model had the smallest log-likelihood, a chi square test indicated that there was no significant fit difference when comparing the Brownian motion and Ornstein-Uhlenbeck model (<em>P</em> = 0.81). Because the Brownian motion model has the least number of parameters, this suggests that a Brownian motion model is the most reasonable fit for the data, and there is a lack of evidence for an adaptive process.</p><p id="p-28">All four traits of all transcripts other than lncRNAs had significant high phylogenetic signal when estimated using Blomberg’s K (K &gt;1), therefore we also tested for a better fit explained by the Ornstein-Uhlenbeck process. Again, the Ornstein-Uhlenbeck process was not a significantly better fit than a Brownian motion model when considering log likelihood values and a chi-square test (ORF length: <em>P</em> = 1, GC content: <em>P</em> = 1, number of exons: <em>P</em> = 1, transcript length: <em>P</em> = 0.75).</p><p id="p-29">We tested for local autocorrelation at 100 phylogenetic distances considering the most recent common ancestors of all species as a robust approach to an analysis on a small phylogeny. Confidence intervals were computed using 1000 bootstrapping replicates for a non-parametric significance estimate using a calculated threshold of -0.111. <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> visualizes the local correlations of traits in both lncRNAs and all other transcripts and are limited to the phylogenetic distances of the tested phylogeny (0-1 phylogenetic distance). We detected significant positive local autocorrelation at short phylogenetic distances in ORF length and GC content of lncRNAs (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). This suggests that closely related species contain lncRNAs with similar ORF lengths and GC content. There was no consistent significant autocorrelation at any short phylogenetic distances in any tested traits in all other transcripts (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>). Detected phylogenetic patches are shorter in the ORF and transcript lengths of lncRNAs compared to all other transripts. The opposite is true in the GC content of lncRNAs, where longer phylogenetic patches are observed. Shorter phylogenetic patches suggest an adaptive process as described by an Ornstein-Uhlenbeck model, rather than genetic drift.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2511/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Moran&#x2019;s I local correlogram of mean trait values in lncRNAs and All Other Transcripts. Coral points indicate significant phylogenetic signal at a particular phylogenetic distance. The horizontal line represents a value of the null hypothesis that no phylogenetic signal is detected. The null hypothesis value is -0.111, or  where , or the number of tested species. The 95% confidence intervals, computed using bootstrapping, are also plotted and were used to identify significant values." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-220780455" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Moran&#x2019;s I local correlogram of mean trait values in lncRNAs and All Other Transcripts. Coral points indicate significant phylogenetic signal at a particular phylogenetic distance. The horizontal line represents a value of the null hypothesis that no phylogenetic signal is detected. The null hypothesis value is -0.111, or &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-4&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; where &lt;img xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot; src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-5&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;, or the number of tested species. The 95% confidence intervals, computed using bootstrapping, are also plotted and were used to identify significant values.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2511/F3.medium.gif" width="440" height="212"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/8/2511/F3.medium.gif" width="440" height="212"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2511/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2511/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493797" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-30" class="first-child">Moran’s I local correlogram of mean trait values in lncRNAs and All Other Transcripts. Coral points indicate significant phylogenetic signal at a particular phylogenetic distance. The horizontal line represents a value of the null hypothesis that no phylogenetic signal is detected. The null hypothesis value is -0.111, or <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2511/F3/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2511/F3/embed/mml-math-4.gif"/></noscript></span></span> where <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2511/F3/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2511/F3/embed/mml-math-5.gif"/></noscript></span></span>, or the number of tested species. The 95% confidence intervals, computed using bootstrapping, are also plotted and were used to identify significant values.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-11"><h2 class="">Discussion</h2><p id="p-31">We used raw RNASeq data from multiple independent studies to make inferences on the numbers of predicted lncRNAs in 11 phylogenetically divergent plant species, and to identify putative phylogenetic signal in these regulatory loci. Our data-mining approach enabled us to use the same protocols for read mapping, transcript assembly, and lncRNA prediction for each species. In performing the same read-mapping and lncRNA prediction protocols, we were able to address a concern raised by <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kapusta and Feschotte (2014)</a> that comparisons between lncRNA numbers in animals can be misleading when prediction numbers are products of meta-analyses involving different prediction methods and lncRNA criteria. We found that the percentage of transcripts predicted as lncRNAs was, on average, 6.4% with two outliers: <em>A. trichopoda</em> with 16.6% and <em>E. salsugineum</em> with 3%. These estimates for lncRNA contributions to plant genomes are lower than comparable predicted values for humans, where Gencode (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Harrow <em>et al.</em> 2012</a>) v29 (<a href="https://www.gencodegenes.org">https://www.gencodegenes.org</a>) and NONCODE v5 (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Fang <em>et al.</em> 2017</a>) identified 29,566 and 172,216 putative lncRNA transcripts, respectively. However, there is a lack of evidence for function in these thousands of transcripts as they were identified by a transcript filtering method and the majority remain without experimental validation. In particular, NONCODE v5 has predicted function in only 1961 of their identified lncRNAs. Nonetheless, due to the methods used regarding read mapping, our results rely on the genome quality of each species. Fragmented genomes, as seen in <em>A. trichopoda</em>, <em>E. salsugineum</em> and <em>S. moellendorffi</em>, may have limited our predictive abilities, and low predicted lncRNA numbers may in part be due to data availability.</p><p id="p-32">The review by <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Kapusta and Feschotte (2014)</a> also included a meta-analysis describing variation in predicted lncRNA numbers among multiple animal species, a comparison similar to our observed prediction numbers in plants. In addition to their concern about transcriptome data arising from different methodologies, <a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Kapusta and Feschotte (2014)</a> also raised the issue of temporal and location specific lncRNA expression. We share a comparable concern in that plant lncRNAs have yet to be predicted in all tissue types for all developmental time points in all possible environments, so undoubtedly the number of putative lncRNAs detected in plants will increase over time. For example, in our study, we identified 2918 putative lncRNAs in <em>A. thaliana</em> plants that were grown under conditions designed to avoid exposing plants to sources of stress. In contrast, although using different prediction methods, <a id="xref-ref-66-2" class="xref-bibr" href="#ref-66">Zhao <em>et al.</em> (2018)</a> identified 6150 putative lncRNAs in <em>A. thaliana</em> plants undergoing cold, ABA and drought treatments. This difference in predicted lncRNAs is consistent with the expectation that lncRNAs likely play a role in stress responses and hence one expects to find increased diversity and abundance of lncRNAs in stressed relative to unstressed plants. Interestingly, <a id="xref-ref-66-3" class="xref-bibr" href="#ref-66">Zhao <em>et al.</em> (2018)</a> found that lncRNAs in <em>A. thaliana</em> are shorter and have fewer exons than all other transcripts, observations that agree with our study (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>). Thus our machine learning-based methodology that was trained on only empirically characterized, functional lncRNAs and the filtering method employed by <a id="xref-ref-66-4" class="xref-bibr" href="#ref-66">Zhao <em>et al.</em> (2018)</a> led to similar conclusions on traits shared by lncRNAs that distinguish them from other transcripts. This finding is also consistent with our previous, cross-validation estimates of CREMA’s accuracy (96%) and specificity (0.994) (<a id="xref-ref-52-5" class="xref-bibr" href="#ref-52">Simopoulos <em>et al.</em> 2018</a>) making CREMA a suitable method for lncRNA prediction from RNASeq data.</p><p id="p-33">The genome of <em>A. trichopoda</em>, the sister taxon to all other extant angiosperms, represents a species with a unique evolutionary history. During genome annotation, The <a id="xref-ref-1-4" class="xref-bibr" href="#ref-1">Amborella Genome Project (2013)</a> observed a larger number of the atypical 23 to 24nt plant miRNAs than expected as they were found in two times greater frequency than any other land plant. Additionally, eight predicted miRNA families in <em>A. trichopoda</em> have evidence of loss in more recent angiosperms (<a id="xref-ref-1-5" class="xref-bibr" href="#ref-1">Amborella Genome Project 2013</a>). The excess of miRNAs in <em>A. trichopoda</em> may reflect the high proportion of lncRNAs predicted in this study (at 16%; <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>) as miRNA progenitors are considered to be lncRNAs (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Saini <em>et al.</em> 2008</a>).</p><p id="p-34">Two plants, namely <em>E. salsugineum</em> and <em>B. hygrometrica</em>, were found to have the lowest proportion of lncRNAs in their transcriptomes (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). <em>E. salsugineum</em> represents a plant with a halophytic life strategy and a capacity to tolerate a variety of extreme environmental conditions (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Kazachkova <em>et al.</em> 2018</a>). Indeed, <em>E. salsugineum</em> has been used as a model plant in stress response studies due to its naturally high tolerance to abiotic stresses such as salt (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Taji <em>et al.</em> 2004</a>), cold (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Griffith <em>et al.</em> 2007</a>), drought (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">MacLeod <em>et al.</em> 2015</a>), and nutritional deficiencies (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Velasco <em>et al.</em> 2016</a>). Moreover, <em>E. salsugineum</em> shows constitutive expression of genes reported to be stress-responsive in many plants (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Taji <em>et al.</em> 2004</a>; Gong <em>et al.</em> 2005; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Wong <em>et al.</em> 2006</a>; <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Velasco <em>et al.</em> 2016</a>). <em>B. hygrometrica</em>, aptly named “the resurrection plant”, is also considered an extremophile by virtue of its capacity to survive desiccation (<a id="xref-ref-62-3" class="xref-bibr" href="#ref-62">Xiao <em>et al.</em> 2015</a>). However, <em>B. hygrometrica</em> shows different expression pattern changes when experiencing stress compared to <em>E. salsugineum</em>. <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Zhu <em>et al.</em> (2015)</a> did not observe constitutively high expression of stress tolerance genes in <em>B. hygrometrica</em> during desiccation. Instead, <em>B. hygrometrica</em> seemed to require gradual dehydration priming for survival after rehydration post-desiccation (<a id="xref-ref-67-2" class="xref-bibr" href="#ref-67">Zhu <em>et al.</em> 2015</a>). <em>B. hygrometrica</em> plants that have been subsequently rehydrated after this dehydration “training” have expression patterns more similar to desiccated plants than those without drought priming (<a id="xref-ref-67-3" class="xref-bibr" href="#ref-67">Zhu <em>et al.</em> 2015</a>). In other words, after experiencing a first gradual dehydration there are expression differences between well-watered <em>B. hygrometrica</em> plants and ones that experienced desiccation. The observation that <em>B. hygrometrica</em> plants can show “preparedness” among expressed genes normally responsive to a stressful condition is somewhat analogous to the constitutive nature of expressed genes in <em>E. salsugineum</em>. Specifically, <em>E. salsugineum</em> plants grown in the absence of high salt display the expression of genes reported to be salt-responsive in other plants (<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Taji <em>et al.</em> 2004</a>; <a id="xref-ref-60-2" class="xref-bibr" href="#ref-60">Wong <em>et al.</em> 2006</a>). Interestingly, both <em>E. salsugineum</em> and <em>B. hygrometrica</em> display a low proportion of predicted lncRNAs in their unstressed transcriptomes suggesting a possible connection between high natural stress tolerance and low lncRNA number. Conceivably, with stress-related genes constantly expressed under a primed condition, a plant adapted to an extreme environment may not require the precise regulation conferred by the recruitment of diverse lncRNAs, an important role proposed for the function of lncRNAs in plant stress responses. This hypothesis merits further work as little is known of the importance, or lack thereof, of lncRNAs in extremophile species.</p><p id="p-35"><em>E. salsugineum</em> and <em>A. trichopoda</em> have distinct evolutionary patterns and yet both species have few putative lncRNAs present in their reference annotations. In total, five of the ten tested plant species had less than 50% of predicted lncRNAs in their respective genome annotations. As genome annotation often relies on homology of predicted genes for functional annotation (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bolger <em>et al.</em> 2018</a>), particularly homology to <em>A. thaliana</em> protein-coding genes, lncRNAs can often be left out of genome annotation. Researchers studying plant lncRNAs frequently rely on bioinformatic analyses to assemble novel transcripts for lncRNA prediction (Liu <em>et al.</em> 2018; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Shuai <em>et al.</em> 2014</a>), indicating that missing lncRNA annotation should be taken into consideration in forthcoming genome annotation projects. Similar to our work, <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Jackson <em>et al.</em> (2018)</a> recently described misannotation of lncRNAs in mammalian genomes. Gaps in annotation and the ensuing problem with lncRNA identification is exacerbated by the fact that lncRNAs do not follow classic evolutionary conservation. Instead, in both plant and animal systems, lncRNAs often depict a positional conservation pattern, even with minimal sequence conservation, making functional predictions also difficult (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Hezroni <em>et al.</em> 2015</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Mohammadin <em>et al.</em> 2015</a>). A lack of extensive lncRNA conservation between species led to our investigation into phylogenetic signal detection in molecular traits of lncRNAs.</p><p id="p-36">In plant species, lncRNA homology has been shown to be virtually non-existent outside of the family classification. Only 1% of predicted <em>A. thaliana</em> (Brassicaceae) lncRNAs were identified as homologous in <em>Tarenaya hassleriana</em> (Cleomaceae) (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Nelson <em>et al.</em> 2016</a>). Interestingly, although distantly related, human and plant lncRNAs bear similarities in number of exons, and transcript and ORF length. For example, <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Derrien <em>et al.</em> (2012)</a> describe human lncRNAs as being shorter than protein coding genes, and by commonly having fewer exons which was also observed in our plant species analyses (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>). However, human lncRNAs are typically spliced and most often have two exons, while our study suggests plant lncRNAs are more often unspliced and comprised of a single exon (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>). Domination of single exon novel lncRNAs in our work may be indication of genomic contamination in source data or bioinformatic transcript assembly artifacts. Nevertheless, genomic contamination is unlikely seen in all ten independent experiments, and transcript assembly artifacts more likely result from <em>de novo</em> assembly rather than genome guided mapping. Further, <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Haerty and Ponting (2015)</a> also observed a lower GC content in lncRNAs than the protein coding genes of metazoan lncRNAs, with single-exon lncRNAs having the lowest GC content of all exon-types, reminiscent of the uni-exonic plant lncRNA majority of our analyses. However, the extent to which conserved traits typify plant and animal lncRNAs is difficult to assess at present. CREMA is trained on only validated lncRNA and may be subject to a prediction bias to animal-like lncRNA sequences given that non-plant sources currently comprise the majority of validated lncRNAs (<a id="xref-ref-52-6" class="xref-bibr" href="#ref-52">Simopoulos <em>et al.</em> 2018</a>). As more plant lncRNA undergo validation, the extent of conservation among lncRNAs from diverse organisms will be easier to detect and estimate with greater precision.</p><p id="p-37">Despite these concerns over bias, among the lncRNAs predicted by CREMA we found significant phylogenetic signal by at least one method in all four tested traits of lncRNAs: ORF length, GC content, the number of exons and transcript length (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a>). High phylogenetic signal detection in traits of lncRNAs was not expected given the lack of evidence for sequence conservation in this class of RNA (<a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Nelson <em>et al.</em> 2016</a>; <a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Hezroni <em>et al.</em> 2015</a>). Importantly, because the phylogenetic relationships of species are influencing the tested traits as indicated by detectable phylogenetic signal, the species can no longer be considered independent and statistical methods that assume independence cannot be used. While it is possible to detect phylogenetic signal in our data, it is difficult to infer evolutionary processes from phylogenetic signal estimates as many unique processes can invoke a similar signal estimation (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Revell <em>et al.</em> 2008</a>). However, high phylogenetic signal and consequent high similarity of GC content in lncRNAs of closely related species is not unexpected, as <a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Haerty and Ponting (2015)</a> demonstrated evidence of selection on the GC content of intergenic lncRNAs in animal species. Additionally, as previously mentioned, the GC content of animal lncRNAs is lower than that of protein-coding genes, mirroring what our work identified in plant lncRNAs.</p><p id="p-38">A lack of similarity in ORF length, number of exons, and transcript length of lncRNAs in close relatives may be due to a variety of processes, including but not limited to: stabilizing selection with high selective strength, selection with variable strength that is bounded by phenotypic limits, punctuated divergent selection, or genetic drift of which rate of drift began low and increased toward present time (<a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">Revell <em>et al.</em> 2008</a>). Because of the variety of possible complex interpretations of phylogenetic signal and process, <a id="xref-ref-46-5" class="xref-bibr" href="#ref-46">Revell <em>et al.</em> (2008)</a> do not recommend over-interpretting evolutionary processes from signal data. We have found, however, unique patterns in the phylogenetic signals of molecular traits of lncRNAs compared to all other transcripts in plant species that imply lncRNAs are not following similar evolutionary trends as most other transcripts. Moreover, the lack of similarity in three of the four tested molecular traits in lncRNAs is of interest and this observation implies that evolution of lncRNAs could be species specific, and is not easily defined by an over-arching evolutionary process. On the other hand, it is possible that there are subclasses of lncRNAs with conserved molecular traits yet to be defined due to a lack of validated transcripts.</p><p id="p-39">Because CREMA (<a id="xref-ref-52-7" class="xref-bibr" href="#ref-52">Simopoulos <em>et al.</em> 2018</a>) predicts lncRNAs using a complex ensemble machine learning model that initially uses ORF length, GC content and transcript length as features for transcript classification, it is possible that high detected phylogenetic signal in these features is a product of the lncRNA prediction tool. However, CREMA’s logistic regression ensemble classifier that is used for the final lncRNA prediction does not use molecular traits as prediction features, but instead binary outputs from eight gradient boosting models. Additionally, we have identified low phylogenetic signal in two of these three molecular traits (<a id="xref-table-wrap-3-6" class="xref-table" href="#T3">Table 3</a>) suggesting that CREMA is able to predict lncRNAs with varying ORF and transcript lengths.</p><p id="p-40">In this work we show that the annotation status of plant species can affect lncRNAs prediction with up to 99% of predicted lncRNAs missing from reference annotation. While researchers may be striving to increase the volume of lncRNA research, the effort to annotate genomes with lncRNAs is not reflective of the increased interest in this RNA class. As such, we caution researchers interested in these regulatory loci to be wary of relying solely upon genome and transcriptome annotations for lncRNA identification. Additionally, our work shows that plant lncRNAs have inconsistent detectable phylogenetic signal in sequence traits, further confirming the complex evolutionary history of lncRNAs. In particular, the differences in detected phylogenetic signal in lncRNAs compared to all other transcripts suggests that lncRNAs evolve, on average, differently than other loci.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-41">We thank the Ontario Research Fund-Research Excellence and Natural Science and Engineering Research Council of Canada for our funding support (EAW: RGPIN-2015-06530 and GBG: RGPIN-2015-04477). We also thank Dr Mitchell MacLeod for growing <em>E. salsugineum</em> plant material and for preparing the RNASeq library. Finally, we thank Dr Benjamin Furman for his invaluable comments on manuscript preparation.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-5"><p id="p-42">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8250953">https://doi.org/10.25387/g3.8250953</a>.</p></li><li class="fn" id="fn-6"><p id="p-43"><em>Communicating editor: S. Mathews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-05-09"><span class="received-label">Received </span>May 9, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-06-21"><span class="accepted-label">Accepted </span>June 21, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Simopoulos <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-3" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.1" data-doi="10.1126/science.1241089"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Amborella Genome Project</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The <em>Amborella</em> genome and the evolution of flowering plants</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">342</span>: <span class="cit-elocation-id">1241089</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1241089</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DAmborella%2BGenome%2BProject%26rft.auinit1%253D%2B%26rft.volume%253D342%26rft.issue%253D6165%26rft.spage%253D1241089%26rft.epage%253D1241089%26rft.atitle%253DThe%2BAmborella%2BGenome%2Band%2Bthe%2BEvolution%2Bof%2BFlowering%2BPlants%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1241089%26rft_id%253Dinfo%253Apmid%252F24357323%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNDIvNjE2NS8xMjQxMDg5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI1MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.8.2511.2" data-doi="10.1126/science.1203810"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Nishiyama</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hasebe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bowman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Gribskov</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The <em>Selaginella</em> genome identifies genetic changes associated with the evolution of vascular plants</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">332</span>: <span class="cit-fpage">960</span>–<span class="cit-lpage">963</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1203810</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1203810%26rft_id%253Dinfo%253Apmid%252F21551031%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMzIvNjAzMi85NjAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzgvMjUxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.3" data-doi="10.1554/0014-3820(2003)057[0717:TFPSIC]2.0.CO;2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blomberg</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Garland</span>  <span class="cit-name-suffix">Jr.</span>.</span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Ives</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Testing for phylogenetic signal in comparative data: behavioral traits are more labile</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">717</span>–<span class="cit-lpage">745</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.0014-3820.2003.tb00285.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DBlomberg%26rft.auinit1%253DS.%2BP.%26rft.volume%253D57%26rft.issue%253D4%26rft.spage%253D717%26rft.epage%253D745%26rft.atitle%253DTesting%2Bfor%2Bphylogenetic%2Bsignal%2Bin%2Bcomparative%2Bdata%253A%2Bbehavioral%2Btraits%2Bare%2Bmore%2Blabile.%26rft_id%253Dinfo%253Adoi%252F10.1554%252F0014-3820%25282003%2529057%255B0717%253ATFPSIC%255D2.0.CO%253B2%26rft_id%253Dinfo%253Apmid%252F12778543%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1554/0014-3820(2003)057[0717:TFPSIC]2.0.CO;2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12778543&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000182815100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.4" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lohse</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Usadel</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trimmomatic: a flexible trimmer for Illumina sequence data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu170</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24695404&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340049100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Arsova</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Usadel</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Plant genome and transcriptome annotations: from misconceptions to simple solutions</span>. <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">449</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft.volume%253D19%26rft.spage%253D437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.8.2511.6" data-doi="10.1038/ncomms10654"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cárdenas</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Sonawane</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Pollier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">VandenBossche</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Dewangan</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">GAME9 regulates the biosynthesis of steroidal alkaloids and upstream isoprenoids in the plant mevalonate pathway</span>. <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">10654</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms10654</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D7%26rft.spage%253D10654%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms10654%26rft_id%253Dinfo%253Apmid%252F26876023%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms10654&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26876023&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.7" data-doi="10.1186/1471-2164-14-578"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Champigny</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Sung</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Catana</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Salwan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Summers</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">RNA-Seq effectively monitors gene expression in <em>Eutrema salsugineum</em> plants growing in an extreme natural habitat and in controlled growth cabinet conditions</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">578</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-14-578</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D14%26rft.spage%253D578%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-14-578%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-14-578&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.8.2511.8" data-doi="10.1111/tpj.13415"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Krishnakumar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Chan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Thibaud-Nissen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Schobel</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Araport11: a complete reannotation of the <em>Arabidopsis thaliana</em> reference genome</span>. <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">89</span>: <span class="cit-fpage">789</span>–<span class="cit-lpage">804</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.13415</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D89%26rft.spage%253D789%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13415%26rft_id%253Dinfo%253Apmid%252F27862469%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13415&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27862469&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.9" data-doi="10.1101/gr.132159.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Derrien</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Johnson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Bussotti</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Tanzer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Djebali</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The GENCODE v7 catalog of human long noncoding RNAs: analysis of their gene structure, evolution, and expression</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1775</span>–<span class="cit-lpage">1789</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.132159.111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DDerrien%26rft.auinit1%253DT.%26rft.volume%253D22%26rft.issue%253D9%26rft.spage%253D1775%26rft.epage%253D1789%26rft.atitle%253DThe%2BGENCODE%2Bv7%2Bcatalog%2Bof%2Bhuman%2Blong%2Bnoncoding%2BRNAs%253A%2BAnalysis%2Bof%2Btheir%2Bgene%2Bstructure%252C%2Bevolution%252C%2Band%2Bexpression%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.132159.111%26rft_id%253Dinfo%253Apmid%252F22955988%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIyLzkvMTc3NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOC8yNTExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.10" data-doi="10.1111/j.0014-3820.2001.tb00630.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Diniz-Filho</span>,  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Phylogenetic autocorrelation under distinct evolutionary processes</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">55</span>: <span class="cit-fpage">1104</span>–<span class="cit-lpage">1109</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.0014-3820.2001.tb00630.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DDiniz-Filho%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D55%26rft.issue%253D6%26rft.spage%253D1104%26rft.epage%253D1109%26rft.atitle%253DPhylogenetic%2Bautocorrelation%2Bunder%2Bdistinct%2Bevolutionary%2Bprocesses.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0014-3820.2001.tb00630.x%26rft_id%253Dinfo%253Apmid%252F11475046%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0014-3820.2001.tb00630.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11475046&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169903000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.11" data-doi="10.1093/bioinformatics/bts635"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dobin</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Davis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Schlesinger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Drenkow</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Zaleski</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">STAR: ultrafast universal RNA-seq aligner</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">15</span>–<span class="cit-lpage">21</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts635</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts635%26rft_id%253Dinfo%253Apmid%252F23104886%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts635&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23104886&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000312654600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.12" data-doi="10.1093/nar/gkx1107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fang</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Guo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Niu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wu</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">NONCODEV5: a comprehensive annotation database for long non-coding RNAs</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">D308</span>–<span class="cit-lpage">D314</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkx1107</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D46%26rft.spage%253DD308%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx1107%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx1107&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-book" id="cit-9.8.2511.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Felsenstein</span>,  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-source"><em>PHYLIP (phylogeny inference package)</em>, <em>version 3.5 c</em></span>, <span class="cit-publ-name">Joseph Felsenstein. University of Washington</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.8.2511.14" data-doi="10.1093/molbev/msu303"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frank</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Scanlon</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Transcriptomic evidence for the evolution of shoot meristem function in sporophyte-dominant land plants through concerted selection of ancestral gametophytic and sporophytic genetic programs</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">355</span>–<span class="cit-lpage">367</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msu303</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu303%26rft_id%253Dinfo%253Apmid%252F25371433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu303&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25371433&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.15" data-doi="10.2307/2992183"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gittleman</span>,  <span class="cit-name-given-names">J. L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kot</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Adaptation: statistics and a null model for estimating phylogenetic effects</span>. <abbr class="cit-jnl-abbrev">Syst. Zool.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">227</span>–<span class="cit-lpage">241</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2307/2992183</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BZool.%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2992183%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2992183&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.8.2511.16" data-doi="10.1111/j.1365-313X.2005.02587.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gong</span>,  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Ma</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Indu Rupassara</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. J.</span> <span class="cit-name-surname">Bohnert</span></span></li></ol><cite>,<span class="cit-pub-date"> 2005</span> <span class="cit-article-title">Salinity stress adaptation competence in the extremophile <em>Thellungiella halophila</em> in comparison with its relative <em>Arabidopsis thaliana</em></span>. <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">826</span>–<span class="cit-lpage">839</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-313X.2005.02587.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DGong%26rft.auinit1%253DQ.%26rft.volume%253D44%26rft.issue%253D5%26rft.spage%253D826%26rft.epage%253D839%26rft.atitle%253DSalinity%2Bstress%2Badaptation%2Bcompetence%2Bin%2Bthe%2Bextremophile%2BThellungiella%2Bhalophila%2Bin%2Bcomparison%2Bwith%2Bits%2Brelative%2BArabidopsis%2Bthaliana.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2005.02587.x%26rft_id%253Dinfo%253Apmid%252F16297073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2005.02587.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16297073&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233407600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.8.2511.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonzalez-Munoz</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Avendano-Vazquez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Montes</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">deFolter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Andres-Hernandez</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The maize (<em>Zea mays</em> ssp. mays var. B73) genome encodes 33 members of the purple acid phosphatase family</span>. <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">341</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D6%26rft.spage%253D341%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.18" data-doi="10.1111/j.1365-3040.2007.01653.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Griffith</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Timonin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Gray</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Akhter</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title"><em>Thellungiella</em>: an <em>Arabidopsis</em>-related model plant adapted to cold temperatures</span>. <abbr class="cit-jnl-abbrev">Plant Cell Environ.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">529</span>–<span class="cit-lpage">538</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-3040.2007.01653.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%252C%2BCell%252C%2Band%2BEnvironment%2B%2528Print%2529%26rft.stitle%253DPlant%252C%2BCell%252C%2Band%2BEnvironment%2B%2528Print%2529%26rft.aulast%253DGriffith%26rft.auinit1%253DM.%26rft.volume%253D30%26rft.issue%253D5%26rft.spage%253D529%26rft.epage%253D538%26rft.atitle%253DThellungiella%253A%2Ban%2BArabidopsis-related%2Bmodel%2Bplant%2Badapted%2Bto%2Bcold%2Btemperatures.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-3040.2007.01653.x%26rft_id%253Dinfo%253Apmid%252F17407531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-3040.2007.01653.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17407531&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000245415100001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.19" data-doi="10.1261/rna.047324.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haerty</span>,  <span class="cit-name-given-names">W.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Ponting</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Unexpected selection to retain high GC content and splicing enhancers within exons of multiexonic lncRNA loci</span>. <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">333</span>–<span class="cit-lpage">346</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1261/rna.047324.114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.047324.114%26rft_id%253Dinfo%253Apmid%252F25589248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjIxLzMvMzIwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI1MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.20" data-doi="10.2307/2411186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hansen</span>,  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Stabilizing selection and the comparative analysis of adaptation</span>. <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">51</span>: <span class="cit-fpage">1341</span>–<span class="cit-lpage">1351</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.1997.tb01457.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.volume%253D51%26rft.spage%253D1341%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2411186%26rft_id%253Dinfo%253Apmid%252F28568616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2411186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28568616&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997YE98000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.21" data-doi="10.1093/bioinformatics/btm538"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harmon</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Weir</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Brock</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Glor</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Challenger</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Geiger: investigating evolutionary radiations</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">129</span>–<span class="cit-lpage">131</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btm538</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm538%26rft_id%253Dinfo%253Apmid%252F18006550%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm538&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18006550&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251865000018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.22" data-doi="10.1101/gr.135350.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harrow</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Frankish</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Gonzalez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Tapanari</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Diekhans</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">GENCODE: the reference human genome annotation for The ENCODE Project</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1760</span>–<span class="cit-lpage">1774</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.135350.111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DHarrow%26rft.auinit1%253DJ.%26rft.volume%253D22%26rft.issue%253D9%26rft.spage%253D1760%26rft.epage%253D1774%26rft.atitle%253DGENCODE%253A%2BThe%2Breference%2Bhuman%2Bgenome%2Bannotation%2Bfor%2BThe%2BENCODE%2BProject%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.135350.111%26rft_id%253Dinfo%253Apmid%252F22955987%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIyLzkvMTc2MCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOC8yNTExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.23" data-doi="10.1016/j.celrep.2015.04.023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hezroni</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Koppstein</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Schwartz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Avrutin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Bartel</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Principles of long noncoding RNA evolution derived from direct comparison of transcriptomes in 17 species</span>. <abbr class="cit-jnl-abbrev">Cell Reports</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">1110</span>–<span class="cit-lpage">1122</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.celrep.2015.04.023</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BReports%26rft.volume%253D11%26rft.spage%253D1110%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2015.04.023%26rft_id%253Dinfo%253Apmid%252F25959816%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2015.04.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25959816&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.24" data-doi="10.1038/s41586-018-0794-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jackson</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Kroehling</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Khitun</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Bailis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Jarret</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">The translation of non-canonical open reading frames controls mucosal immunity</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">564</span>: <span class="cit-fpage">434</span>–<span class="cit-lpage">438</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41586-018-0794-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41586-018-0794-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41586-018-0794-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.8.2511.25" data-doi="10.1105/tpc.16.00831"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">James</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Jayasena</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Berkowitz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Secco</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Evidence for Ancient Origins of Bowman-Birk Inhibitors from <em>Selaginella moellendorffii</em></span>. <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">461</span>–<span class="cit-lpage">473</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.16.00831</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.16.00831%26rft_id%253Dinfo%253Apmid%252F28298518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjI5LzMvNDYxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI1MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.26" data-doi="10.1098/rstb.2012.0341"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kamilar</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Cooper</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Phylogenetic signal in primate behaviour, ecology and life history</span>. <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">368</span>: <span class="cit-elocation-id">20120341</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rstb.2012.0341</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilos.%2BTrans.%2BR.%2BSoc.%2BLond.%2BB%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2012.0341%26rft_id%253Dinfo%253Apmid%252F23569289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2012.0341&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23569289&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.27" data-doi="10.1126/science.1138341"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kapranov</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Cheng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Dike</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Nix</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Duttagupta</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">RNA maps reveal new RNA classes and a possible function for pervasive transcription</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">316</span>: <span class="cit-fpage">1484</span>–<span class="cit-lpage">1488</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1138341</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1138341%26rft_id%253Dinfo%253Apmid%252F17510325%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTYvNTgzMC8xNDg0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI1MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.28" data-doi="10.1016/j.tig.2014.08.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kapusta</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Feschotte</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Volatile evolution of long noncoding RNA repertoires: mechanisms and biological implications</span>. <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">439</span>–<span class="cit-lpage">452</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2014.08.004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D30%26rft.spage%253D439%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2014.08.004%26rft_id%253Dinfo%253Apmid%252F25218058%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2014.08.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25218058&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.29" data-doi="10.1093/nar/gkf436"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katoh</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Misawa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kuma</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Miyata</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">3059</span>–<span class="cit-lpage">3066</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkf436</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DKatoh%26rft.auinit1%253DK.%26rft.volume%253D30%26rft.issue%253D14%26rft.spage%253D3059%26rft.epage%253D3066%26rft.atitle%253DMAFFT%253A%2Ba%2Bnovel%2Bmethod%2Bfor%2Brapid%2Bmultiple%2Bsequence%2Balignment%2Bbased%2Bon%2Bfast%2BFourier%2Btransform%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkf436%26rft_id%253Dinfo%253Apmid%252F12136088%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkf436&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12136088&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000177154300016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.30" data-doi="10.1104/pp.18.00863"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kazachkova</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Eshel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pantha</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Cheeseman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Dassanayake</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Halophytism: What Have We Learnt From <em>Arabidopsis thaliana</em> Relative Model Systems?</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">972</span>–<span class="cit-lpage">988</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.18.00863</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.18.00863%26rft_id%253Dinfo%253Apmid%252F30237204%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNzgvMy85NzIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzgvMjUxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.31" data-doi="10.1002/ece3.2051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keck</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Rimet</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bouchez</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Franc</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">phylosignal: an R package to measure, test, and explore the phylogenetic signal</span>. <abbr class="cit-jnl-abbrev">Ecol. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2774</span>–<span class="cit-lpage">2780</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/ece3.2051</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEcol.%2BEvol.%26rft.volume%253D6%26rft.spage%253D2774%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fece3.2051%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/ece3.2051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.8.2511.32" data-doi="10.1111/tpj.13801"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lang</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Ullrich</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Murat</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Fuchs</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jenkins</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">The <em>Physcomitrella patens</em> chromosome-scale assembly reveals moss genome structure and evolution</span>. <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">515</span>–<span class="cit-lpage">533</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.13801</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D93%26rft.spage%253D515%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13801%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13801&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.8.2511.33" data-doi="10.1371/journal.pone.0200002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Cheng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Lin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">XuHan</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Lai</span></span></li></ol><cite>,<span class="cit-pub-date"> 2018</span> <span class="cit-article-title">Genome-wide identification and characterization of mRNAs and lncRNAs involved in cold stress in the wild banana (<em>Musa itinerans</em>)</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">13</span>: <span class="cit-elocation-id">e0200002</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0200002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D13%26rft.spage%253De0200002%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0200002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0200002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.34" data-doi="10.1111/ppl.12316"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">MacLeod</span>,  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Dedrick</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Ashton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. W.</span> <span class="cit-name-surname">Sung</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Champigny</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Exposure of two <em>Eutrema salsugineum</em> (<em>Thellungiella salsuginea</em>) accessions to water deficits reveals different coping strategies in response to drought</span>. <abbr class="cit-jnl-abbrev">Physiol. Plant.</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">267</span>–<span class="cit-lpage">280</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/ppl.12316</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhysiol.%2BPlant.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fppl.12316%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/ppl.12316&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.8.2511.35" data-doi="10.1126/science.1143609"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Merchant</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Prochnik</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Vallon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Harris</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Karpowicz</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The <em>Chlamydomonas</em> genome reveals the evolution of key animal and plant functions</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">318</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">250</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1143609</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMerchant%26rft.auinit1%253DS.%2BS.%26rft.volume%253D318%26rft.issue%253D5848%26rft.spage%253D245%26rft.epage%253D250%26rft.atitle%253DThe%2BChlamydomonas%2BGenome%2BReveals%2Bthe%2BEvolution%2Bof%2BKey%2BAnimal%2Band%2BPlant%2BFunctions%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1143609%26rft_id%253Dinfo%253Apmid%252F17932292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMTgvNTg0OC8yNDUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzgvMjUxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.36" data-doi="10.1186/s12870-015-0603-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mohammadin</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Edger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Pires</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Schranz</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Positionally-conserved but sequence-diverged: identification of long non-coding RNAs in the <em>Brassicaceae</em> and <em>Cleomaceae</em></span>. <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">217</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12870-015-0603-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BPlant%2BBiol.%26rft.volume%253D15%26rft.spage%253D217%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12870-015-0603-5%26rft_id%253Dinfo%253Apmid%252F26362138%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12870-015-0603-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26362138&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moran</span>,  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1948</span> <span class="cit-article-title">The interpretation of statistical maps</span>. <abbr class="cit-jnl-abbrev">J. R. Stat. Soc. B</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">243</span>–<span class="cit-lpage">251</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BR.%2BStat.%2BSoc.%2BB%26rft.volume%253D10%26rft.spage%253D243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.38" data-doi="10.1080/07388551.2017.1312270"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nejat</span>,  <span class="cit-name-given-names">N.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Mantri</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Emerging roles of long non-coding RNAs in plant response to biotic and abiotic stresses</span>. <abbr class="cit-jnl-abbrev">Crit. Rev. Biotechnol.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">93</span>–<span class="cit-lpage">105</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/07388551.2017.1312270</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrit.%2BRev.%2BBiotechnol.%26rft.volume%253D38%26rft.spage%253D93%26rft_id%253Dinfo%253Adoi%252F10.1080%252F07388551.2017.1312270%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/07388551.2017.1312270&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.39" data-doi="10.1534/g3.116.030338"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Forsythe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Devisetty</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Clausen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Haug-Batzell</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A Genomic Analysis of Factors Driving lincRNA Diversification: Lessons from Plants</span>. <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2881</span>–<span class="cit-lpage">2891</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.030338</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.030338%26rft_id%253Dinfo%253Apmid%252F27440919%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjYvOS8yODgxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI1MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.40" data-doi="10.1261/rna.029520.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Niazi</span>,  <span class="cit-name-given-names">F.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Valadkhan</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Computational analysis of functional long noncoding RNAs reveals lack of peptide-coding capacity and parallels with 3′ UTRs</span>. <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">825</span>–<span class="cit-lpage">843</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1261/rna.029520.111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.029520.111%26rft_id%253Dinfo%253Apmid%252F22361292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjE4LzQvODI1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI1MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.8.2511.41" data-doi="10.1093/nar/gkl976"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ouyang</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Zhu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hamilton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Lin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Campbell</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The TIGR Rice Genome Annotation Resource: improvements and new features</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">D883</span>–<span class="cit-lpage">D887</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkl976</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl976%26rft_id%253Dinfo%253Apmid%252F17145706%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkl976&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17145706&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243494600171&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.42" data-doi="10.1038/44766"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pagel</span>,  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Inferring the historical patterns of biological evolution</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">401</span>: <span class="cit-fpage">877</span>–<span class="cit-lpage">884</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/44766</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPagel%26rft.auinit1%253DM.%26rft.volume%253D401%26rft.issue%253D6756%26rft.spage%253D877%26rft.epage%253D884%26rft.atitle%253DInferring%2Bthe%2Bhistorical%2Bpatterns%2Bof%2Bbiological%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F44766%26rft_id%253Dinfo%253Apmid%252F10553904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/44766&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10553904&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000083464700048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.8.2511.43" data-doi="10.1534/g3.114.015032"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Panchy</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Wu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Newton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Tsai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Prevalence, evolution, and cis-regulation of diel transcription in <em>Chlamydomonas reinhardtii</em></span>. <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">2461</span>–<span class="cit-lpage">2471</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.114.015032</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.114.015032%26rft_id%253Dinfo%253Apmid%252F25354782%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjQvMTIvMjQ2MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOC8yNTExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.44" data-doi="10.1093/bioinformatics/bty633"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Paradis</span>,  <span class="cit-name-given-names">E.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Schliep</span></span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">ape 5.0: an environment for modern phylogenetics and evolutionary analyses in R</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">35</span>:<span class="cit-fpage">526</span>–<span class="cit-lpage">528</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bty633</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbty633%26rft_id%253Dinfo%253Apmid%252F27016728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bty633&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27016728&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.45" data-doi="10.1038/nbt.3122"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Pertea</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Antonescu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Chang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Mendell</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">StringTie enables improved reconstruction of a transcriptome from RNA-seq reads</span>. <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">290</span>–<span class="cit-lpage">295</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.3122</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D33%26rft.spage%253D290%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.3122%26rft_id%253Dinfo%253Apmid%252F25690850%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.3122&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25690850&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.46" data-doi="10.1080/10635150802302427"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Revell</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Harmon</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Collar</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Phylogenetic signal, evolutionary process, and rate</span>. <abbr class="cit-jnl-abbrev">Syst. Biol.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">591</span>–<span class="cit-lpage">601</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/10635150802302427</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSyst.%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10635150802302427%26rft_id%253Dinfo%253Apmid%252F18709597%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10635150802302427&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18709597&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263702800005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.47" data-doi="10.1038/nmeth.4066"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rivas</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Clements</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Eddy</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A statistical test for conserved RNA structure shows lack of evidence for structure in lncRNAs</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">45</span>–<span class="cit-lpage">48</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.4066</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BMethods%26rft.volume%253D14%26rft.spage%253D45%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.4066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.4066&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.48" data-doi="10.1186/1471-2164-9-564"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Saini</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Enright</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Griffiths-Jones</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Annotation of mammalian primary microRNAs</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">564</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-9-564</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DSaini%26rft.auinit1%253DH.%2BK.%26rft.volume%253D9%26rft.spage%253D564%26rft.epage%253D564%26rft.atitle%253DAnnotation%2Bof%2Bmammalian%2Bprimary%2BmicroRNAs.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-9-564%26rft_id%253Dinfo%253Apmid%252F19038026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-9-564&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19038026&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.8.2511.49" data-doi="10.1126/science.1178534"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schnable</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Ware</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Fulton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Stein</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Wei</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The B73 maize genome: complexity, diversity, and dynamics</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">326</span>: <span class="cit-fpage">1112</span>–<span class="cit-lpage">1115</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1178534</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSchnable%26rft.auinit1%253DP.%2BS.%26rft.volume%253D326%26rft.issue%253D5956%26rft.spage%253D1112%26rft.epage%253D1115%26rft.atitle%253DThe%2BB73%2BMaize%2BGenome%253A%2BComplexity%252C%2BDiversity%252C%2Band%2BDynamics%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1178534%26rft_id%253Dinfo%253Apmid%252F19965430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjYvNTk1Ni8xMTEyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI1MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.8.2511.50" data-doi="10.1534/g3.113.007153"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharma</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Bolser</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">deBoer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Sonderkaer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Amoros</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Construction of reference chromosome-scale pseudomolecules for potato: integrating the potato genome with genetic and physical maps</span>. <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">2031</span>–<span class="cit-lpage">2047</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.113.007153</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.113.007153%26rft_id%253Dinfo%253Apmid%252F24062527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjk6IjMvMTEvMjAzMSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOC8yNTExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.51" data-doi="10.1093/jxb/eru256"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shuai</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Liang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Tang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Ye</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide identification and functional prediction of novel and drought-responsive lincRNAs in <em>Populus trichocarpa</em></span>. <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">4975</span>–<span class="cit-lpage">4983</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jxb/eru256</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Feru256%26rft_id%253Dinfo%253Apmid%252F24948679%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jxb/eru256&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24948679&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000342927200022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.52" data-doi="10.1186/s12864-018-4665-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simopoulos</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Weretilnyk</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Golding</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Prediction of plant lncRNA by ensemble machine learning classifiers</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">316</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-018-4665-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D19%26rft.spage%253D316%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-018-4665-2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-018-4665-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.8.2511.53" data-doi="10.1111/pce.12780"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sprenger</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Kurowsky</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Horn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Erban</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Seddig</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The drought response of potato reference cultivars with contrasting tolerance</span>. <abbr class="cit-jnl-abbrev">Plant Cell Environ.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">2370</span>–<span class="cit-lpage">2389</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pce.12780</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BEnviron.%26rft.volume%253D39%26rft.spage%253D2370%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpce.12780%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pce.12780&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.54" data-doi="10.1104/pp.104.039909"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Taji</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Seki</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Satou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Sakurai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kobayashi</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Comparative genomics in salt tolerance between <em>Arabidopsis</em> and <em>Arabidopsis</em>-related halophyte salt cress using <em>Arabidopsis</em> microarray</span>. <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">135</span>: <span class="cit-fpage">1697</span>–<span class="cit-lpage">1709</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.104.039909</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.104.039909%26rft_id%253Dinfo%253Apmid%252F15247402%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTM1LzMvMTY5NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOC8yNTExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.8.2511.55" data-doi="10.1038/nature11119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Tomato Genome Consortium</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The tomato genome sequence provides insights into fleshy fruit evolution</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">485</span>: <span class="cit-fpage">635</span>–<span class="cit-lpage">641</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature11119</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSato%26rft.auinit1%253DS.%26rft.volume%253D485%26rft.issue%253D7400%26rft.spage%253D635%26rft.epage%253D641%26rft.atitle%253DThe%2Btomato%2Bgenome%2Bsequence%2Bprovides%2Binsights%2Binto%2Bfleshy%2Bfruit%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11119%26rft_id%253Dinfo%253Apmid%252F22660326%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22660326&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304608000047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.56" data-doi="10.1111/pce.12750"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Velasco</span>,  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Mansbridge</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Bremner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Carruthers</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Summers</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Acclimation of the crucifer <em>Eutrema salsugineum</em> to phosphate limitation is associated with constitutively high expression of phosphate-starvation genes</span>. <abbr class="cit-jnl-abbrev">Plant Cell Environ.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1818</span>–<span class="cit-lpage">1834</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pce.12750</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BEnviron.%26rft.volume%253D39%26rft.spage%253D1818%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpce.12750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pce.12750&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.57" data-doi="10.1111/tpj.13481"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Qu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Yang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Transposable elements (TEs) contribute to stress-related long intergenic noncoding RNAs in plants</span>. <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">133</span>–<span class="cit-lpage">146</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/tpj.13481</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Ftpj.13481%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/tpj.13481&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-book" id="cit-9.8.2511.58" data-doi="10.1007/978-981-10-5203-3_5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">H. V.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Cheksnova</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Long Noncoding RNAs in Plants</span>, <span class="cit-source">Long Non Coding RNA Biology</span>, edited by <span class="cit-ed"> <span class="cit-name-given-names">M. R. S.</span> <span class="cit-name-surname">Rao</span></span> <span class="cit-publ-name">Springer, Singapore. Nature</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-981-10-5203-3_5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLong%2BNon%2BCoding%2BRNA%2BBiology%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-981-10-5203-3_5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-981-10-5203-3_5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.8.2511.59" data-doi="10.1105/tpc.16.00158"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilkins</span>,  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Hafemeister</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Plessis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Holloway-Phillips</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Pham</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">EGRINs (Environmental Gene Regulatory Influence Networks) in Rice That Function in the Response to Water Deficit, High Temperature, and Agricultural Environments</span>. <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2365</span>–<span class="cit-lpage">2384</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.16.00158</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.16.00158%26rft_id%253Dinfo%253Apmid%252F27655842%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjEwOiIyOC8xMC8yMzY1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI1MTEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.60" data-doi="10.1104/pp.105.070508"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Labbe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Guevara</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Nuin</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Transcriptional profiling implicates novel interactions between abiotic stress and hormonal responses in <em>Thellungiella</em>, a close relative of <em>Arabidopsis</em></span>. <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">140</span>: <span class="cit-fpage">1437</span>–<span class="cit-lpage">1450</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.105.070508</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.105.070508%26rft_id%253Dinfo%253Apmid%252F16500996%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTQwLzQvMTQzNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOC8yNTExLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.8.2511.61" data-doi="10.1104/pp.15.01929"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Woo</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Koo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Kim</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Jeong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Yang</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Programming of Plant Leaf Senescence with Temporal and Inter-Organellar Coordination of Transcriptome in <em>Arabidopsis</em></span>. <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">171</span>: <span class="cit-fpage">452</span>–<span class="cit-lpage">467</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.15.01929</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.15.01929%26rft_id%253Dinfo%253Apmid%252F26966169%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNzEvMS80NTIiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzgvMjUxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-3" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.62" data-doi="10.1073/pnas.1505811112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Yang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Yang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Zhao</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">The resurrection genome of <em>Boea hygrometrica</em>: A blueprint for survival of dehydration</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">5833</span>–<span class="cit-lpage">5837</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1505811112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1505811112%26rft_id%253Dinfo%253Apmid%252F25902549%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEyLzE4LzU4MzMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzgvMjUxMS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.63" data-doi="10.1186/s12870-017-0984-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Song</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Zhu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Tao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Kang</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Systematic comparison of lncRNAs with protein coding mRNAs in population expression and their response to environmental change</span>. <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">42</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12870-017-0984-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BPlant%2BBiol.%26rft.volume%253D17%26rft.spage%253D42%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12870-017-0984-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12870-017-0984-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.64" data-doi="10.3389/fpls.2013.00046"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Jarvis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Chen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Beilstein</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Grimwood</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The Reference Genome of the Halophytic Plant <em>Eutrema salsugineum</em></span>. <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">46</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2013.00046</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D4%26rft.spage%253D46%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2013.00046%26rft_id%253Dinfo%253Apmid%252F23518688%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2013.00046&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23518688&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.65" data-doi="10.1002/pld3.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yin</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Gosney</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. P.</span> <span class="cit-name-surname">Dilkes</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. V.</span> <span class="cit-name-surname">Mickelbart</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Dark period transcriptomic and metabolic profiling of two diverse <em>Eutrema salsugineum</em> accessions</span>. <abbr class="cit-jnl-abbrev">Plant Direct</abbr> <span class="cit-vol">2</span>: <span class="cit-elocation-id">e00032</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/pld3.32</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BDirect%26rft.volume%253D2%26rft.spage%253D00032e%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpld3.32%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/pld3.32&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.66" data-doi="10.1038/s41467-018-07500-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Lian</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Gu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Li</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Global identification of Arabidopsis lncRNAs reveals the regulation of MAF4 by a natural antisense RNA</span>. <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">5056</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41467-018-07500-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D9%26rft.spage%253D5056%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41467-018-07500-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41467-018-07500-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2511.67" data-doi="10.1093/pcp/pcv059"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Phillips</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Du</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Global Transcriptome Analysis Reveals Acclimation-Primed Processes Involved in the Acquisition of Desiccation Tolerance in <em>Boea hygrometrica</em></span>. <abbr class="cit-jnl-abbrev">Plant Cell Physiol.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">1429</span>–<span class="cit-lpage">1441</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/pcp/pcv059</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fpcp%252Fpcv059%26rft_id%253Dinfo%253Apmid%252F25907569%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/pcp/pcv059&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25907569&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2511.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/8/2511.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/8/2497" title="Structural Characterization of ABCB1, the Gene Underlying the d2 Dwarf Phenotype in Pearl Millet, Cenchrus Americanus (L.) Morrone" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/8/2521" title="Open Chromatin Profiling in Adipose Tissue Marks Genomic Regions with Functional Roles in Cardiometabolic Traits" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/8"><span>Volume 9 Issue 8, August 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/8.cover-source.jpg?itok=cSNYBBSf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/8/2511" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-BIuqMQ2YqYJK0f3kajVepO25TMFjYkadgXTyed9JKw0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/8/2511" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/8/2511" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/8/2511" class="active" data-icon-position="" data-hide-link-title="0">Molecular Traits of Long Non-protein Coding RNAs from Diverse Plant Species Show Little Evidence of Phylogenetic Relationships</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/493746" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-YLfRAfYn81Cto21d4aaUSyvsadacB6LKjbWnK6jAZY8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/8/2511.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/8/2511" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-J7Kg4bl7_5fa8r4xLcbEXO86DbPBlITJpdMb9PUoLXk" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/8/2511" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/8/2511.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493746" id="node493746--2" data-pisa="ggg;9/8/2511" data-pisa-master="ggg;g3.119.400201" data-apath="/ggg/9/8/2511.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Molecular Traits of Long Non-protein Coding RNAs from Diverse Plant Species Show Little Evidence of Phylogenetic Relationships</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9832-7797" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Caitlin M. A. Simopoulos</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-0399-6232" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Elizabeth A. Weretilnyk</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7575-0282" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>G. Brian Golding</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2511-2520;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400201 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493746--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Caitlin M. A. Simopoulos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, McMaster University, Hamilton, Ontario, Canada, L8S 4K1</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Caitlin%2BM.%2BA.%2BSimopoulos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Simopoulos%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACaitlin%2BM.%2BA.%2BSimopoulos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9832-7797" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Caitlin M. A. Simopoulos</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Elizabeth A. Weretilnyk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, McMaster University, Hamilton, Ontario, Canada, L8S 4K1</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elizabeth%2BA.%2BWeretilnyk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weretilnyk%20EA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AElizabeth%2BA.%2BWeretilnyk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0399-6232" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Elizabeth A. Weretilnyk</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">G. Brian Golding </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, McMaster University, Hamilton, Ontario, Canada, L8S 4K1</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=G.%2BBrian%2BGolding%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Golding%20GB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AG.%2BBrian%2BGolding%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7575-0282" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for G. Brian Golding</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:golding@mcmaster.ca" class="" data-icon-position="" data-hide-link-title="0">golding@mcmaster.ca</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/8/2511" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/8/2511&amp;title=Molecular%20Traits%20of%20Long%20Non-protein%20Coding%20RNAs%20from%20Diverse%20Plant%20Species%20Show%20Little%20Evidence%20of%20Phylogenetic%20Relationships" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/8/2511&amp;title=Molecular%20Traits%20of%20Long%20Non-protein%20Coding%20RNAs%20from%20Diverse%20Plant%20Species%20Show%20Little%20Evidence%20of%20Phylogenetic%20Relationships" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/8/2511&amp;title=Molecular%20Traits%20of%20Long%20Non-protein%20Coding%20RNAs%20from%20Diverse%20Plant%20Species%20Show%20Little%20Evidence%20of%20Phylogenetic%20Relationships" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/8/2511&amp;text=Molecular%20Traits%20of%20Long%20Non-protein%20Coding%20RNAs%20from%20Diverse%20Plant%20Species%20Show%20Little%20Evidence%20of%20Phylogenetic%20Relationships" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/8/2511&amp;title=Molecular%20Traits%20of%20Long%20Non-protein%20Coding%20RNAs%20from%20Diverse%20Plant%20Species%20Show%20Little%20Evidence%20of%20Phylogenetic%20Relationships" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/8/2511&amp;t=Molecular%20Traits%20of%20Long%20Non-protein%20Coding%20RNAs%20from%20Diverse%20Plant%20Species%20Show%20Little%20Evidence%20of%20Phylogenetic%20Relationships" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/8/2511&amp;title=Molecular%20Traits%20of%20Long%20Non-protein%20Coding%20RNAs%20from%20Diverse%20Plant%20Species%20Show%20Little%20Evidence%20of%20Phylogenetic%20Relationships" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/8/2511&amp;title=Molecular%20Traits%20of%20Long%20Non-protein%20Coding%20RNAs%20from%20Diverse%20Plant%20Species%20Show%20Little%20Evidence%20of%20Phylogenetic%20Relationships" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/493746/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493746" id="node493746--3" data-pisa="ggg;9/8/2511" data-pisa-master="ggg;g3.119.400201" data-apath="/ggg/9/8/2511.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Molecular Traits of Long Non-protein Coding RNAs from Diverse Plant Species Show Little Evidence of Phylogenetic Relationships</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9832-7797" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Caitlin M. A. Simopoulos</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-0399-6232" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Elizabeth A. Weretilnyk</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-7575-0282" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>G. Brian Golding</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2511-2520;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400201 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493746--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Caitlin M. A. Simopoulos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, McMaster University, Hamilton, Ontario, Canada, L8S 4K1</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Caitlin%2BM.%2BA.%2BSimopoulos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Simopoulos%20CM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ACaitlin%2BM.%2BA.%2BSimopoulos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9832-7797" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Caitlin M. A. Simopoulos</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Elizabeth A. Weretilnyk </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, McMaster University, Hamilton, Ontario, Canada, L8S 4K1</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Elizabeth%2BA.%2BWeretilnyk%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Weretilnyk%20EA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AElizabeth%2BA.%2BWeretilnyk%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0399-6232" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Elizabeth A. Weretilnyk</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">G. Brian Golding </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Biology, McMaster University, Hamilton, Ontario, Canada, L8S 4K1</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=G.%2BBrian%2BGolding%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Golding%20GB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AG.%2BBrian%2BGolding%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7575-0282" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for G. Brian Golding</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:golding@mcmaster.ca" class="" data-icon-position="" data-hide-link-title="0">golding@mcmaster.ca</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2511&amp;rft.epage=2520&amp;rft.atitle=Molecular%20Traits%20of%20Long%20Non-protein%20Coding%20RNAs%20from%20Diverse%20Plant%20Species%20Show%20Little%20Evidence%20of%20Phylogenetic%20Relationships&amp;rft.volume=9&amp;rft.issue=8&amp;rft.date=2019-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Simopoulos%2C+Caitlin+M.+A.&amp;rft.au=Weretilnyk%2C+Elizabeth+A.&amp;rft.au=Golding%2C+G.+Brian"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/493746/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/493746/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/493746/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/493746/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/493746/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/493746/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/493746/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/493746/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/493746/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/493746/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/493746/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/493746/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/8/2511" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-11" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/8/2511.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/8/2511.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/8/2511.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
