<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/8/2657" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/8/2657.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/8/2657.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/8/2657.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/8/2657.atom" />
<meta name="HW.pisa" content="ggg;9/8/2657" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Fragmentation of Pooled PCR Products for Highly Multiplexed TILLING" />
<meta name="DC.Identifier" content="10.1534/g3.119.400301" />
<meta name="DC.Date" content="2019-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Tramontano et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Improvements to massively parallel sequencing have allowed the routine recovery of natural and induced sequence variants. A broad range of biological disciplines have benefited from this, ranging from plant breeding to cancer research. The need for high sequence coverage to accurately recover single nucleotide variants and small insertions and deletions limits the applicability of whole genome approaches. This is especially true in organisms with a large genome size or for applications requiring the screening of thousands of individuals, such as the reverse-genetic technique known as TILLING. Using PCR to target and sequence chosen genomic regions provides an attractive alternative as the vast reduction in interrogated bases means that sample size can be dramatically increased through amplicon multiplexing and multi-dimensional sample pooling while maintaining suitable coverage for recovery of small mutations. Direct sequencing of PCR products is limited, however, due to limitations in read lengths of many next generation sequencers. In the present study we show the optimization and use of ultrasonication for the simultaneous fragmentation of multiplexed PCR amplicons for TILLING highly pooled samples. Sequencing performance was evaluated in a total of 32 pooled PCR products produced from 4096 chemically mutagenized Hordeum vulgare DNAs pooled in three dimensions. Evaluation of read coverage and base quality across amplicons suggests this approach is suitable for high-throughput TILLING and other applications employing highly pooled complex sampling schemes. Induced mutations previously identified in a traditional TILLING screen were recovered in this dataset further supporting the efficacy of the approach." />
<meta name="DC.Contributor" content="Andrea Tramontano" />
<meta name="DC.Contributor" content="Luka Jarc" />
<meta name="DC.Contributor" content="Joanna Jankowicz-Cieslak" />
<meta name="DC.Contributor" content="Bernhard J. Hofinger" />
<meta name="DC.Contributor" content="Katarzyna Gajek" />
<meta name="DC.Contributor" content="Miriam Szurman-Zubrzycka" />
<meta name="DC.Contributor" content="Iwona Szarejko" />
<meta name="DC.Contributor" content="Ivan Ingelbrecht" />
<meta name="DC.Contributor" content="Bradley J. Till" />
<meta name="article:published_time" content="2019-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Fragmentation of Pooled PCR Products for Highly Multiplexed TILLING" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Improvements to massively parallel sequencing have allowed the routine recovery of natural and induced sequence variants. A broad range of biological disciplines have benefited from this, ranging from plant breeding to cancer research. The need for high sequence coverage to accurately recover single nucleotide variants and small insertions and deletions limits the applicability of whole genome approaches. This is especially true in organisms with a large genome size or for applications requiring the screening of thousands of individuals, such as the reverse-genetic technique known as TILLING. Using PCR to target and sequence chosen genomic regions provides an attractive alternative as the vast reduction in interrogated bases means that sample size can be dramatically increased through amplicon multiplexing and multi-dimensional sample pooling while maintaining suitable coverage for recovery of small mutations. Direct sequencing of PCR products is limited, however, due to limitations in read lengths of many next generation sequencers. In the present study we show the optimization and use of ultrasonication for the simultaneous fragmentation of multiplexed PCR amplicons for TILLING highly pooled samples. Sequencing performance was evaluated in a total of 32 pooled PCR products produced from 4096 chemically mutagenized &lt;i&gt;Hordeum vulgare&lt;/i&gt; DNAs pooled in three dimensions. Evaluation of read coverage and base quality across amplicons suggests this approach is suitable for high-throughput TILLING and other applications employing highly pooled complex sampling schemes. Induced mutations previously identified in a traditional TILLING screen were recovered in this dataset further supporting the efficacy of the approach.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/08/01" />
<meta name="citation_mjid" content="ggg;9/8/2657" />
<meta name="citation_id" content="9/8/2657" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/8/2657" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/8/2657.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/8/2657.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/8/2657.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400301" />
<meta name="citation_pmid" content="31213514" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2657" />
<meta name="citation_lastpage" content="2666" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Andrea Tramontano" />
<meta name="citation_author_institution" content="Plant Breeding and Genetics Laboratory, Joint FAO/IAEA Division of Nuclear Techniques in Food and Agriculture, IAEA Laboratories Seibersdorf, International Atomic Energy Agency, Vienna International Centre, PO Box 100, A-1400 Vienna, Austria" />
<meta name="citation_author_institution" content="These authors contributed equally to this work." />
<meta name="citation_author" content="Luka Jarc" />
<meta name="citation_author_institution" content="Plant Breeding and Genetics Laboratory, Joint FAO/IAEA Division of Nuclear Techniques in Food and Agriculture, IAEA Laboratories Seibersdorf, International Atomic Energy Agency, Vienna International Centre, PO Box 100, A-1400 Vienna, Austria" />
<meta name="citation_author_institution" content="These authors contributed equally to this work." />
<meta name="citation_author" content="Joanna Jankowicz-Cieslak" />
<meta name="citation_author_institution" content="Plant Breeding and Genetics Laboratory, Joint FAO/IAEA Division of Nuclear Techniques in Food and Agriculture, IAEA Laboratories Seibersdorf, International Atomic Energy Agency, Vienna International Centre, PO Box 100, A-1400 Vienna, Austria" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9249-4700" />
<meta name="citation_author" content="Bernhard J. Hofinger" />
<meta name="citation_author_institution" content="Plant Breeding and Genetics Laboratory, Joint FAO/IAEA Division of Nuclear Techniques in Food and Agriculture, IAEA Laboratories Seibersdorf, International Atomic Energy Agency, Vienna International Centre, PO Box 100, A-1400 Vienna, Austria" />
<meta name="citation_author" content="Katarzyna Gajek" />
<meta name="citation_author_institution" content="Department of Genetics, Faculty of Biology and Environmental Protection, University of Silesia, Jagiellonska 28, 40-032, Katowice, Poland" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0235-7607" />
<meta name="citation_author" content="Miriam Szurman-Zubrzycka" />
<meta name="citation_author_institution" content="Department of Genetics, Faculty of Biology and Environmental Protection, University of Silesia, Jagiellonska 28, 40-032, Katowice, Poland" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-4661-5823" />
<meta name="citation_author" content="Iwona Szarejko" />
<meta name="citation_author_institution" content="Department of Genetics, Faculty of Biology and Environmental Protection, University of Silesia, Jagiellonska 28, 40-032, Katowice, Poland" />
<meta name="citation_author" content="Ivan Ingelbrecht" />
<meta name="citation_author_institution" content="Plant Breeding and Genetics Laboratory, Joint FAO/IAEA Division of Nuclear Techniques in Food and Agriculture, IAEA Laboratories Seibersdorf, International Atomic Energy Agency, Vienna International Centre, PO Box 100, A-1400 Vienna, Austria" />
<meta name="citation_author" content="Bradley J. Till" />
<meta name="citation_author_institution" content="Plant Breeding and Genetics Laboratory, Joint FAO/IAEA Division of Nuclear Techniques in Food and Agriculture, IAEA Laboratories Seibersdorf, International Atomic Energy Agency, Vienna International Centre, PO Box 100, A-1400 Vienna, Austria" />
<meta name="citation_author_institution" content="Present address: Centro de Genómica Nutricional Agroacuícola, Las Heras #350, Temuco, Chile." />
<meta name="citation_author_email" content="brad.till@cgna.cl" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1300-8285" />
<meta name="citation_reference" content="citation_journal_title=Plant Genet. Resour.;citation_author=R. Bovina;citation_author=V. Talame;citation_author=S. Silvio;citation_author=M. C. Sanguineti;citation_author=P. Trost;citation_title=Starch metabolism mutants in barley: A TILLING approach;citation_pages=170-173;citation_volume=9;citation_year=2011;citation_doi=10.1017/S147926211100044X" />
<meta name="citation_reference" content="citation_title=Biotechnologies for Plant Mutation Breeding;citation_title=Next-Generation Sequencing for Targeted Discovery of Rare Mutations in Rice;citation_pages=323-340;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_journal_abbrev=Bioinformatics;citation_author=T. Carver;citation_author=A. Bleasby;citation_title=The design of Jemboss: a graphical user interface to EMBOSS;citation_pages=1837-1843;citation_volume=19;citation_year=2003;citation_issue=14;citation_pmid=14512356;citation_doi=10.1093/bioinformatics/btg251" />
<meta name="citation_reference" content="citation_journal_title=Plant Biotechnol. J.;citation_author=S. Datta;citation_author=J. Jankowicz-Cieslak;citation_author=S. Nielen;citation_author=I. Ingelbrecht;citation_author=B. J. Till;citation_title=Induction and recovery of copy number variation in banana through gamma irradiation and low coverage whole genome sequencing;citation_pages=1644-1653;citation_volume=16;citation_year=2018;citation_doi=10.1111/pbi.12901" />
<meta name="citation_reference" content="citation_journal_title=Comput. Struct. Biotechnol. J.;citation_author=J. Duitama;citation_author=L. Kafuri;citation_author=D. Tello;citation_author=A. M. Leiva;citation_author=B. Hofinger;citation_title=Deep Assessment of Genomic Diversity in Cassava for Herbicide Tolerance and Starch Biosynthesis;citation_pages=185-194;citation_volume=15;citation_year=2017;citation_doi=10.1016/j.csbj.2017.01.002" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiology;citation_journal_abbrev=Plant Physiol.;citation_author=P. Gepts;citation_title=Who Owns Biodiversity, and How Should the Owners Be Compensated?;citation_pages=1295-1307;citation_volume=134;citation_year=2004;citation_issue=4;citation_pmid=15084724;citation_doi=10.1104/pp.103.038885" />
<meta name="citation_reference" content="citation_journal_abbrev=BMC Res Notes;citation_author=S. Gottwald;citation_author=P. Bauer;citation_author=T. Komatsuda;citation_author=U. Lundqvist;citation_author=N. Stein;citation_title=TILLING in the two-rowed barley cultivar 'Barke' reveals preferred sites of functional diversity in the gene HvHox1.;citation_pages=258-258;citation_volume=2;citation_year=2009;citation_pmid=20017921;citation_doi=10.1186/1756-0500-2-258" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=Y. Guo;citation_author=B. Abernathy;citation_author=Y. Zeng;citation_author=P. Ozias-Akins;citation_title=TILLING by sequencing to identify induced mutations in stress resistance genes of peanut (Arachis hypogaea);citation_pages=157;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12864-015-1348-0" />
<meta name="citation_reference" content="citation_journal_title=Cell. Mol. Biol.;citation_author=P. Gupta;citation_author=B. Reddaiah;citation_author=H. Salava;citation_author=P. Upadhyaya;citation_author=K. Tyagi;citation_title=Next-generation sequencing (NGS)-based identification of induced mutations in a doubly mutagenized tomato (Solanum lycopersicum) population. Plant J;citation_pages=495-508;citation_volume=92;citation_year=2017" />
<meta name="citation_reference" content="citation_journal_title=Biotechniques;citation_author=S. R. Head;citation_author=H. K. Komori;citation_author=S. A. LaMere;citation_author=T. Whisenant;citation_author=F. Van Nieuwerburgh;citation_title=Library construction for next-generation sequencing: Overviews and challenges;citation_pages=61;citation_volume=56;citation_year=2014;citation_pmid=24502796;citation_doi=10.2144/000114133" />
<meta name="citation_reference" content="citation_journal_title=Brief. Funct. Genomics;citation_author=C. D. Hirsch;citation_author=J. Evans;citation_author=C. R. Buell;citation_author=C. N. Hirsch;citation_title=Reduced representation approaches to interrogate genome diversity in large repetitive plant genomes;citation_volume=13;citation_year=2014;citation_pmid=24395692;citation_doi=10.1093/bfgp/elt051" />
<meta name="citation_reference" content="citation_title=Biotechnologies for Plant Mutation Breeding;citation_author=O. A. Huynh;citation_author=J. Jankowicz-Cieslak;citation_author=B. Saraye;citation_author=B. Hofinger;citation_author=B. J. Till;citation_title=Low-Cost Methods for DNA Extraction and Quantification;citation_year=2017;citation_doi=10.1007/978-3-319-45021-6_14" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. F. Mayer;citation_author=R. Waugh;citation_author=J. W. Brown;citation_author=A. Schulman;citation_author=P. Langridge;citation_title=A physical, genetic and functional sequence assembly of the barley genome.;citation_pages=711-716;citation_volume=491;citation_year=2012;citation_issue=7426;citation_pmid=23075845;citation_doi=10.1038/nature11543" />
<meta name="citation_reference" content="citation_title=Advances in Plant Breeding Strategies: Breeding, Biotechnology and Molecular Tools;citation_author=J. Jankowicz-Cieslak;citation_author=B. J. Till;citation_title=Forward and Reverse Genetics in Crop Breeding;citation_year=2015;citation_doi=10.1007/978-3-319-22521-0_8" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. V. Krasileva;citation_author=H. A. Vasquez-Gross;citation_author=T. Howell;citation_author=P. Bailey;citation_author=F. Paraiso;citation_title=Uncovering hidden variation in polyploid wheat;citation_volume=114;citation_year=2017;citation_pmid=28096351;citation_doi=10.1073/pnas.1619268114" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows–Wheeler transform;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=BioMed Res. Int.;citation_title=Comparison of Next-Generation Sequencing Systems;citation_pages=1-11;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=M. Mascher;citation_author=H. Gundlach;citation_author=A. Himmelbach;citation_author=S. Beier;citation_author=S. O. Twardziok;citation_title=A chromosome conformation capture ordered sequence of the barley genome;citation_pages=427-433;citation_volume=544;citation_year=2017;citation_pmid=28447635;citation_doi=10.1038/nature22043" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=C. M. McCallum;citation_author=L. Comai;citation_author=E. A. Greene;citation_author=S. Henikoff;citation_title=Targeted screening for induced mutations.;citation_pages=455-457;citation_volume=18;citation_year=2000;citation_issue=4;citation_pmid=10748531;citation_doi=10.1038/74542" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The Genome Analysis Toolkit: A MapReduce framework for analyzing next-generation DNA sequencing data;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=K. Okonechnikov;citation_author=A. Conesa;citation_author=F. García-Alcalde;citation_title=Qualimap 2: advanced multi-sample quality control for high-throughput sequencing data;citation_volume=32;citation_year=2016;citation_pmid=26428292;citation_doi=10.1093/bioinformatics/btv566" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=L. Pan;citation_author=A. N. Shah;citation_author=I. G. Phelps;citation_author=D. Doherty;citation_author=E. A. Johnson;citation_title=Rapid identification and recovery of ENU-induced mutations with next-generation sequencing and Paired-End Low-Error analysis;citation_pages=83;citation_volume=16;citation_year=2015;citation_pmid=25886285;citation_doi=10.1186/s12864-015-1263-4" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=D. Rigola;citation_author=J. van Oeveren;citation_author=A. Janssen;citation_author=A. Bonne;citation_author=H. Schneiders;citation_title=High-throughput detection of induced mutations and natural variation using KeyPoint technology.;citation_pages=e4761-e4761;citation_volume=4;citation_year=2009;citation_issue=3;citation_pmid=19283079;citation_doi=10.1371/journal.pone.0004761" />
<meta name="citation_reference" content="citation_title=Biotechnologies for Plant Mutation Breeding;citation_author=M. Slota;citation_author=M. Maluszynski;citation_author=I. Szarejko;citation_title=Bioinformatics-Based Assessment of the Relevance of Candidate Genes for Mutation Discovery;citation_year=2017;citation_doi=10.1007/978-3-319-45021-6_17" />
<meta name="citation_reference" content="citation_journal_title=DNA Repair (Amst.);citation_author=M. Stolarek;citation_author=D. Gruszka;citation_author=A. Braszewska-Zalewska;citation_author=M. Maluszynski;citation_title=Alleles of newly identified barley gene HvPARP3 exhibit changes in efficiency of DNA repair;citation_pages=116-130;citation_volume=28;citation_year=2015;citation_doi=10.1016/j.dnarep.2015.02.018" />
<meta name="citation_reference" content="citation_journal_title=Mutagenesis;citation_author=M. Stolarek;citation_author=D. Gruszka;citation_author=A. Braszewska-Zalewska;citation_author=M. Maluszynski;citation_title=Functional analysis of the new barley gene HvKu80 indicates that it plays a key role in double-strand DNA break repair and telomere length regulation;citation_volume=30;citation_year=2015;citation_pmid=25958390;citation_doi=10.1093/mutage/gev033" />
<meta name="citation_reference" content="citation_title=Biotechnologies for Plant Mutation Breeding;citation_author=I. Szarejko;citation_author=M. Szurman-Zubrzycka;citation_author=M. Nawrot;citation_author=M. Marzec;citation_author=D. Gruszka;citation_title=Creation of a TILLING Population in Barley After Chemical Mutagenesis with Sodium Azide and MNU;citation_year=2017;citation_doi=10.1007/978-3-319-45021-6_6" />
<meta name="citation_reference" content="citation_journal_abbrev=Plant Biotechnol J;citation_author=V. Talamè;citation_author=R. Bovina;citation_author=M. C. Sanguineti;citation_author=R. Tuberosa;citation_author=U. Lundqvist;citation_title=TILLMore, a resource for the discovery of chemically induced mutants in barley.;citation_pages=477-485;citation_volume=6;citation_year=2008;citation_issue=5;citation_pmid=18422888;citation_doi=10.1111/j.1467-7652.2008.00341.x" />
<meta name="citation_reference" content="citation_journal_title=USENIX Mag.;citation_author=O. Tange;citation_title=GNU Parallel: The Command-Line Power Tool;citation_pages=42-47;citation_volume=36;citation_year=2011" />
<meta name="citation_reference" content="citation_journal_title=Adv. Biochem. Eng. Biotechnol.;citation_author=B. J. Till;citation_author=S. Datta;citation_author=J. Jankowicz-Cieslak;citation_title=TILLING: The Next Generation;citation_pages=139-160;citation_volume=164;citation_year=2018" />
<meta name="citation_reference" content="citation_title=Low-Cost Methods for Molecular Characterization of Mutant Plants;citation_author=B. J. Till;citation_author=J. Jankowicz-Cieslak;citation_author=O. A. Huynh;citation_author=M. Beshir;citation_author=R. G. Laport;citation_year=2015;citation_doi=10.1007/978-3-319-16259-1" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=B. J. Till;citation_author=J. Jankowicz-Cieslak;citation_author=L. Sagi;citation_author=O. A. Huynh;citation_author=H. Utsushi;citation_title=Discovery of nucleotide polymorphisms in the Musa gene pool by Ecotilling.;citation_pages=1381-1389;citation_volume=121;citation_year=2010;citation_issue=7;citation_pmid=20589365;citation_doi=10.1007/s00122-010-1395-5" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=B. J. Till;citation_author=S. H. Reynolds;citation_author=E. A. Greene;citation_author=C. A. Codomo;citation_author=L. C. Enns;citation_title=Large-Scale Discovery of Induced Point Mutations With High-Throughput TILLING;citation_pages=524-530;citation_volume=13;citation_year=2003;citation_issue=3;citation_pmid=12618384;citation_doi=10.1101/gr.977903" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=B. J. Till;citation_author=S. H. Reynolds;citation_author=C. Weil;citation_author=N. Springer;citation_author=C. Burtner;citation_title=Discovery of induced point mutations in maize genes by TILLING.;citation_pages=12-12;citation_volume=4;citation_year=2004;citation_pmid=15282033;citation_doi=10.1186/1471-2229-4-12" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=B. J. Till;citation_author=T. Zerr;citation_author=E. Bowers;citation_author=E. A. Greene;citation_author=L. Comai;citation_title=High-throughput discovery of rare human nucleotide polymorphisms by Ecotilling;citation_pages=e99-e99;citation_volume=34;citation_year=2006;citation_issue=13;citation_pmid=16893952;citation_doi=10.1093/nar/gkl479" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=B. J. Till;citation_author=T. Zerr;citation_author=L. Comai;citation_author=S. Henikoff;citation_title=A protocol for TILLING and Ecotilling in plants and animals.;citation_pages=2465-2477;citation_volume=1;citation_year=2006;citation_issue=5;citation_pmid=17406493;citation_doi=10.1038/nprot.2006.329" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=H. Tsai;citation_author=T. Howell;citation_author=R. Nitcher;citation_author=V. Missirian;citation_author=B. Watson;citation_title=Discovery of Rare Mutations in Populations: TILLING by Sequencing;citation_volume=156;citation_year=2011;citation_pmid=21531898;citation_doi=10.1104/pp.110.169748" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=Y. Wang;citation_author=L. Xu;citation_author=R. Thilmony;citation_author=F. M. You;citation_author=Y. Q. Gu;citation_title=PIECE 2.0: an update for the plant gene structure comparison and evolution database;citation_volume=45;citation_year=2017;citation_pmid=27742820;citation_doi=10.1093/nar/gkw935" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Fragmentation of Pooled PCR Products for Highly Multiplexed TILLING" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8.cover-source.jpg" />
<meta name="twitter:description" content="Improvements to massively parallel sequencing have allowed the routine recovery of natural and induced sequence variants. A broad range of biological disciplines have benefited from this, ranging from plant breeding to cancer research. The need for high sequence coverage to accurately recover single nucleotide variants and small insertions and deletions limits the applicability of whole genome approaches. This is especially true in organisms with a large genome size or for applications requiring the screening of thousands of individuals, such as the reverse-genetic technique known as TILLING. Using PCR to target and sequence chosen genomic regions provides an attractive alternative as the vast reduction in interrogated bases means that sample size can be dramatically increased through amplicon multiplexing and multi-dimensional sample pooling while maintaining suitable coverage for recovery of small mutations. Direct sequencing of PCR products is limited, however, due to limitations in read lengths of many next generation sequencers. In the present study we show the optimization and use of ultrasonication for the simultaneous fragmentation of multiplexed PCR amplicons for TILLING highly pooled samples. Sequencing performance was evaluated in a total of 32 pooled PCR products produced from 4096 chemically mutagenized Hordeum vulgare DNAs pooled in three dimensions. Evaluation of read coverage and base quality across amplicons suggests this approach is suitable for high-throughput TILLING and other applications employing highly pooled complex sampling schemes. Induced mutations previously identified in a traditional TILLING screen were recovered in this dataset further supporting the efficacy of the approach." />
<meta name="og-title" property="og:title" content="Fragmentation of Pooled PCR Products for Highly Multiplexed TILLING" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/8/2657" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Improvements to massively parallel sequencing have allowed the routine recovery of natural and induced sequence variants. A broad range of biological disciplines have benefited from this, ranging from plant breeding to cancer research. The need for high sequence coverage to accurately recover single nucleotide variants and small insertions and deletions limits the applicability of whole genome approaches. This is especially true in organisms with a large genome size or for applications requiring the screening of thousands of individuals, such as the reverse-genetic technique known as TILLING. Using PCR to target and sequence chosen genomic regions provides an attractive alternative as the vast reduction in interrogated bases means that sample size can be dramatically increased through amplicon multiplexing and multi-dimensional sample pooling while maintaining suitable coverage for recovery of small mutations. Direct sequencing of PCR products is limited, however, due to limitations in read lengths of many next generation sequencers. In the present study we show the optimization and use of ultrasonication for the simultaneous fragmentation of multiplexed PCR amplicons for TILLING highly pooled samples. Sequencing performance was evaluated in a total of 32 pooled PCR products produced from 4096 chemically mutagenized Hordeum vulgare DNAs pooled in three dimensions. Evaluation of read coverage and base quality across amplicons suggests this approach is suitable for high-throughput TILLING and other applications employing highly pooled complex sampling schemes. Induced mutations previously identified in a traditional TILLING screen were recovered in this dataset further supporting the efficacy of the approach." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8.cover-source.jpg" />
    <title>Fragmentation of Pooled PCR Products for Highly Multiplexed TILLING | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"mp3dQwTnW_6Wd4lsOGO-4XZbFTgkSc1cqbCu7Loympg","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"493754","apath":"\/ggg\/9\/8\/2657.atom","pisa":"ggg;9\/8\/2657","ac":{"\/ggg\/9\/8\/2657.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/8\/2657.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/8\/2657"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/8\/2657"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/8\/2657":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LbK07ECCIvrEss3rkHWG3cToKjOBteb+9NtfmbZdDUGYROi7XQODckcOPve4hUi5q6o6eVyKA7wBMnAdkUl8qPTy8bB0o60SL1JtK\\\\\\\/VYgYxcNspZOEhntyITHccbmJFy9XfCwgfxkxZnLhmSWSxocZJw88X4c3H\\\\\\\/p+Vb1hZr2LIJdrfb25SpLG7p\\\\\\\/DYtnfhmwq+MZICfNHrsm6PLj+6S8nIejYGRid3XgHdsBKSORk0cNxi+rYEytTDjjqHCFq8g4BegcJBB3uii3I9q\\\\\\\/\\\\\\\/9y3V0zi5q8pz8P+oDpmN9j3AIzPAgPZdOfrzA\\\\\\\/k0nO3Czn1JnQMR1x6pWwwKe1ZVgrha\\\\\\\/ZZatdBYVyyFHrQSkgeytwgKFJmDgkXjK22jSYr2zJnVbny9FqosPcXFjxMBU1J82FyuxxexYK1Hk0qF67M4NwBlcoLG8HnxRMJtlrXuW\\\\\\\/eECvLDe1vjfhyPkaFVqjDSeCjzZEzA+xnqexDPxptpY9sEjNs20qtYow9ljDZqKJiEA8mIBhdYoO38wFakkZKXzEBNhyiai17TjXW5uDUr0Rlk0VQncYF+1ABqf9K777zEPcDWRoPSEM\\\\\\\/4f+m+5y+x3o0\\\\\\\/lMmyUbxMnrixsEsjMyJJohTWSECnjFzP4XJEeOzKvM08Tw7eLuTcWCercrF\\\\\\\/BxyNFcoZMtAVB5vGVWhitWVn6EO5re\\\\\\\/g9UEN5\\\\\\\/GbRQ+JQlbhqxRo1qqYNk3NOTnYuwqiM+QPYZ2nShRG1f4\\\\\\\/IGEkXw9my0f0BBgW42T00oSfZ4FI9HH8q5tImHJM06VWzynzW9TajlHNe01iO+j9GsoMNf6zNFSQkc+qHfnF3f1hADeBasVvnNiaHJOWkENAyp49vdCIcA3lRy\\\\\\\/SJaC7TR\\\\\\\/k6JxKZS9zF7vG06GFX7Nn0qpa3ORk7jQR7TMa\\\\\\\/oS68eeId\\\\\\\/dycmpPZUkkYCIzY7hm5spmQ4peh+shzRejsE\\\\\\\/DjMAH\\\\\\\/X7+6J3QPWOWMFut2BKz91ZPzJ0tnjwdqcP9lJLsLEdWh6+1ihz8fSiz3G6cSaiVXUdtrdlizAQIdQhkvmQozBIiwoK7SJpzcNlsDI+V39Zk45Sl8PFHjvk8r+i6A00mNIuEIksjmMYv9iqgScmTyycFbr5jP9YetRpNXVSErhdvyxvG4v7MWyn4WeelHCoWVcODazazNymPMRW+CuFATKrcFmSxOLRh662sduQskmrlIca4iJAWz4WPm86yD8eZTZX6nUX6Yp+gE9ihkCxTJACyf2m1VsUoNwtghKgiK+mR6MkaITZ8ofg9hX\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/LEfW1f5gEc8rBGQLJWxXQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002297122a9c93f5f1326b8d98a2496c3f05500160712630708e356c534801750196\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022WzfXqLdNEkUlZarbfF8tvtKxNB06peajhmnSQD339K1nFH6lccHknfDaiel7umf9ml8dQ6P+ZYECxfPLlrpSBfGJjesD8nYEco66hQg8+EJiX6hZCD4dBrNQbHISXxMhUXyEc23ej+DxITc+RrwFEOJ9m2btRmDKByb85alhr+DuzQ09+dyC1jesvBaJmJY4y0LR1F0ZSo4weGZlS04FRzD7UCsJ+F8hhC8IEdgu3O0mHsB3yEDvnRNfWaiWJji2xZRQYxFU15SHWahcrKP84tdw364DAnhe7lSsdbt7CRep53f85VeWkARk5bwRq8KsT2DnuU1zG4re\\\\\\\/MI9PuF7eRwuHVC+vCUjMjeKmgUOSaksofTbB3UyWUI7G4e7CYNyQCtknuYn8Hl5LhF6ZcK61umZvBdlAMr\\\\\\\/23uOKtfhv1jkrPfGnYDckthxLdu2pTOURHg\\\\\\\/ffSkbcGBUrmf0xk\\\\\\\/qBuueGN6WhbEuQcq+UdQj0k+0G8O1jy\\\\\\\/T8TPnpgXbpD+1FqSwT64vOR0i+kefrR\\\\\\\/QkjgNqrjalwcp4GWx7W6U0YKbqojs2yAJWVlz\\\\\\\/cruPegYr8IPwmMvwr5b4clOAsVqbFFVgzx4I4TFXkbgnuYpxQ3W7A6okCno3+ewg3DS4sbIl2CCKYuJS7WmaUE2BXHG+sdYx8l0X+OhOGTxzp7gHAPQCMJZbCA7yoV\\\\\\\/0zsOCZ6V6zN8v6F3N7ZxyVrZJkT0DxWWAJnKlL1byFS+TDHkO0EMiRvZz2Tw+XS1FriYIp2nirdX8TRkz+o5C3wYCnUDvPk4BwBnO6Y7+FTj1WD4QUSSgwZnJYJgAivToJ8GSLHjeOn4kMOpa8XjYlMCul6PWgw8ynjp8pnt32qullYJ0BjF3ceQrDZad+r9ftEPP2ErHPXLKHleFoiIDSbbXTNE++lgCYrWLs5g94RQsa9S7loE6QXdRo9TcIYlPtRYC\\\\\\\/dAvSd3sJnn14YgHdZx8AeUZt+cTunf07Kh+UVK7BJCTK7kfvyQUOgdB6d3e6KjPAJoDZgXBlnbaaIShTp0\\\\\\\/\\\\\\\/zopcT4QFuKn1qPwPCR4h8Gre8jr14HmsNABR3o99QEqhuPAM1i6Mqmr3WGPpYvRxJHPBrDQVIc\\\\\\\/NRAP8ItX4WGXApbpwLsTtM0+2qOerZILSCICagYKEA4F4TE+kydh2PHzIaNpo7ACJItIXC\\\\\\\/rSEOgNYAqmfu7p\\\\\\\/MRxRjE\\\\\\\/PA5AU0hx4Aj9qoQsK\\\\\\\/O0cZ6Hr98YiNJJYE3bo8\\\\\\\/Ou01t6+k\\\\\\\/4EZdqw77hrg7sOmr4Kbd2ksmuqNncHvJP30gQ86ag2EjAqAtWHJbhseYx66oCIUI4XcERSG54fgpTxmBKgNiFNms0PySSLA2qdRqAmA7shNJhJ8WM2guEvhXIg7vytugZi8nvsUaYSANsHACmcIhzKMZZn5fWlUWeGmOt0mqv\\\\\\\/oaX6y66M2byDk5BqdKuSM6mbiaPX\\\\\\\/RIs++VM0br6lQ42EKqe2pVaAxALtK39dNLshNZ1ZZNDzU+KYBQ23TzFM+nqmcDw4XkrTCvQVvdheIysM4ZXdmnCdeijJKKLF69rlroMI6ZZBWApQJRsuP7IB6JM2XFs0N\\\\\\\/ZqTSC3WNXXmdUZfgdrpQ1YO4m\\\\\\\/onyR+0FKYvr\\\\\\\/BcFp3wIP7Ay0s4YIHMYsLwtNtysTcZtps93O7k8lLP5FpRWgu48zVLMy1v\\\\\\\/ilCfnqeShz1ot60lfMu6CbH3axtz1QwzjzfBhQ+yCcWnEZl+FnPO4z5bImHS1vZwqhsXuZ3okyzKtNdFSfgnhevJOEM844NC4GoCAc3tlp6q+ZrkdNN91FUvfum92M\\\\\\\/nTzZ9EK7d6dg06RXwXvd8SLUqLMjag1b\\\\\\\/JAT4uD0iKoidhk72UGbHnbDFyHVvX4RYmWRwQFjHc2jq4tH+9o8AxZ9vZPnIyMcdoy82zvbNcsd1Ov0Yrz58ITmfAf5qXPBOIUn3+G50GkpS3FJ7jlU0T0NogWwnGEl9lN+DSC4iA+LyWxoQcoS07MxLrx1E+OTpRkyYZdPyRkMBflBageLNFLIlskfJJUJxwwe82MTUN8IKsf6jqXHU35FGt13KkiYUpOLxiNv2Ec3SRV6drKTq2Nnq8M4hHL\\\\\\\/h6bi58Ckxa0O3GorV+PulzhRqIqchRkV2vMarQnMS1uYyO\\\\\\\/xjaLNUJet5GSzj1EpjbmUKtG88Mq4bocaMS1Ldp9EwS\\\\\\\/XD4KbUoB\\\\\\\/l2pLJcx3pK3wfHtXqtkyqZZ93+E7r2FBX+GIlZzvFK\\\\\\\/0XKpUe8ce1PJ0syzY7Zdx6up2avdOQeLjOBXN8fS4eJe\\\\\\\/B1ZfgkT8Gl1whN0TpaA2OzjwzeRqfvJWsxH7I66viZ0Ba\\\\\\\/lXjQ4fskNGX6vqqp30dBwxGlXbtEFAJyKvmhGo5yFe6ModBBmVA8hkC6PRaxTs5EWdyke3PrLCjSswaiOTRVv6Y1mruomiOh2GVELO03+1vKzWQs\\\\\\\/RYimt2P4GSNg7t2Oy2aNajPSohyYASpref9CqRYvd9E0oSZautpI4gSJ6yzPJY6w2eUols7mfERGjpGs7Z64HPtSeSfIPYhIMtruVXIAXXdrPLzbygyGnxTGbJ4vEZuV9KH8gds0yIyBhvm+mxkLsRB9aE8qw54zjgN4s5r7g9eJ6B37tP+5NkR5P9q1ql\\\\\\\/+xV3kM8RmUa7jIUQLKnr7Z+JU3LL73DrpXtWMXBlA1iR9x\\\\\\\/JZangTJ\\\\\\\/KMuwdaFrwxhiVvnIsMHjCDYEZpvFSPzr3rsT4q3V2ClKjXzQJRdKruAtIv77+Yr2i8s8fc1brIC+BtjBlKQK4a+ciS718MpDkCDCrkPq75FXa1osAF16FqHLa6W9u74V+Sf7B4jRngcfgaNk+SEc4G3JTTOl7cEE3nudfXTpcLMlHO8wYn45Mdj3GUiUgSBYlZgVrYBC\\\\\\\/NNYpo1Lvx0M1rJ3YxIAWPC1zjnPPmt04CPvJ29kOhSfx0fwo+u26UpG0Dc49b7nSSnFuqvYVBPtxSW+rkOSPLoNQsPhiNVi6JcsowgPOt\\\\\\\/IpIY2EilVfeigabvAFWWOQNC4GLc6dfD2Gx3VRMJgnGyGdACfmq\\\\\\\/Zu7BPaq39x3u+UEFiOAAacnnJiOiH7QRp2Fp90ZOcPCECN9IGQWJxBNwsylL9ljp51F8SjDHI3ECkazAcOUxfftypJEeYgJZChIcTyvVc2\\\\\\\/so9jTIS2aj+czmp6MGE6dG0pMGpbv+meCunPkVqxFLPE+8wXCAgXNz6uBSToq5Hqezym677yU0VTLlMxPdZEirHBoPo1iLAU5m3R3dPabCTxZZyXKIwOqi8jEAA9nUvNQuKHARMrE9bnTFFItNzyC\\\\\\\/nOtc4BVmokGk+MN\\\\\\\/SlQT9zb9ED66h8SaK2+O4W0wLZE7fnyMm6F4CquQNP\\\\\\\/Ox\\\\\\\/X9AW3s0FvBdzwt0nHU6yfCd8dFsafnLKuDonbZhOipYZ24MVCKEkegZ2OLxO5Go+4MtvRV2AvC4x0e\\\\\\\/26\\\\\\\/Cn5xLYIRSNUCUzsVcN0q7AvRbIHB2D8ZnUv7N2QOFacuc8w51olc+Mj87W7CHBvUqUG8Yt6t58PiXQ5N4xKM6AFjmuZIOOyXdymzsqGg28SqfbD\\\\\\\/6+44cGHLtTAUI4ZFDtZFVoa3Wfk050S2ui\\\\\\\/iEEu7x\\\\\\\/H9nPxBKbNIssVvQ3V30lV1+mJ3gkpMomImblGJpUFTY\\\\\\\/qH4WSWE\\\\\\\/u3RjUYaApCRpbf0ZGy9YceJhwhg4qAcylth3I0uVg22KAyepV3S9X7FDZvWSzjOLfACPHRS5Hoh1S1xDGaxNUZ5KMMdY\\\\\\\/ddv6qw3VWeFn4rhrSCTecwXpd7+\\\\\\\/ddYQMHojP64SdLhR108Opp2SKVWXnUmxp5PgEcNimDxKV2RzTgleSs15XZ2D8fngaUb5vFaLP8yQ5\\\\\\\/86u4qLsZKdGM+REkpXe\\\\\\\/MwZBTMnB\\\\\\\/mWbS5ZSvUAQ1kKk76gFodHehMcDxLds89lzSfn82txAPwb6ah0gajYSBwfKy3PyiHZjZl8R21\\\\\\\/+pELoVs5nb00HipsSIteyaPSc\\\\\\\/mC+guMYEHZ3ZO37Pvtgor0d1G+gi9tJePYwjnfb9ONaA\\\\\\\/n6qIfwEUC+7XLmubntjpNED165nym7h53\\\\\\\/lWyKOqD+V5Ta2LCe9\\\\\\\/9Z5EI7VmQelNY9OwIpvGDTkIvTJeVD3refZF9rKaCwU8fztH5bMY\\\\\\\/bniU0Fn2Hbhs5FYIBu2qZ0xTfud+GGo+IipTJZkbuajRt\\\\\\\/PoFNmN910vkulA7k1aG585UBTn20u7uDokUdk0pRMt5wsN5E\\\\\\\/fyJKoAXAQklzitu8J6ylwCLmPZC8pbd0We7KYCORjcjreYCRg93Gmz42xHBT9m3kz+aaHd7NpAUwme3f+4lqGBhToMpOs3qCLbyrkj2dpU74NsyQwR4O5WW74+sfu1k+S78mhfTd7iNxfjYrU7kj1ea8mvQDpK5GrOLlcUfpb4THH4BilG9hqASyIbHjuWfhkPR59\\\\\\\/oHIPD7Yly0WJijTYcIuUw\\\\\\\/mwFmKzItmn8m1M4fSq1JdQvGfPjMvdOS1Wmts1BDlQiSCXl3aceKNKSMw4iOtz3oi2xREux2qWZSY4axDBbyPAzRamvmA4KMeuw6d3TsU9F1MyYiV+IpTt2ZBR9osK2wF4rS52U35Ki5qxsE2iRrb4lGv8260Lo+ltTQvJv8SFed1fQrMyn55paj0ik7wBhWuPW4O\\\\\\\/ky30bDQFUBW+ubbB7XTZvjN7Mw2szIZnFg5gKMa0AafpfQOob7a01sDHZVetjk28MisCdleu8YdQO9Dm8YfUSfvWJd4D2flt1cNuY+zN20hIINpkJgdHUDJ6Z\\\\\\\/BF\\\\\\\/j0c7GnQrd+iFu6wgtRWWg6OR97+Zz0hGS0Smd3dpkvwznJZvEOKbOKnOGsERmnDZypxsafeRKFp5Ck1MF79SK\\\\\\\/Aa+32llZEqYgFZaqTIgrxWyJxV492V9HtNouzkgmfdbyeXn0CXamGG7P3vvkO+G6C+cXX8rhxzpeqR\\\\\\\/gL\\\\\\\/4xdUf0y4bxaNmmTOjKbOaCDRD52EVV1fshFgpxiqPgZk+LZh9qRWOTBBZ6y5JtCv\\\\\\\/hCD\\\\\\\/NK1El0BTILFlDLckU7PjtAYhUSdqN5xNdcmsI+kFHCz9QTT73TmLCeoeKyoFzacl5KhfFNLAyLozvZ7p\\\\\\\/ZlZfAtrCiOcxr8gvY4aWK1rUAPtiH9vjqatPBQWJH0t\\\\\\\/koMAJIvHDFFohODW5oy7s8yiPMfT6p7ol4GEBvSJjguYza\\\\\\\/5KDgTKEGDCi6DC1vZNmjntzmJGbzKfWCf9DNwDEtsmU4Vcy28zKoJlkg5qvqU8c4g1L+zL8D\\\\\\\/GpO+CyFw7702eKu4S97jZw+7Lpn960OIgQQQQ2jCAlXfB\\\\\\\/WXc\\\\\\\/J1e7o5dnvtgeaBnCBmrwwMiY5IzfgBBAGdDuBpKytp24+jmQptPKNU+JeSNOA9Z18JbqFT9tyGxXpGZcJ0z\\\\\\\/336epN01Om4cXhsp1VxRCqKct3Oz2aSlRmJ91jo7I7b\\\\\\\/esrK9afPXQyMkFC8o7Lw5b7X5dxfaVun3iD4C6btAWtLQ65jo4i7\\\\\\\/BIAroUWi2OZGzzJwOVUhNI0gwWythNd2Pd3X+5nrfgOPyXq21ziYpQl2SiWbBeACldQ2YSVUHM9WEmEmpgTtcJJLymKtRtk+9jVkD8U6gpwtf0a25sqdlr5k2007VMNAOe5zC7is3dFjU2N8d\\\\\\\/b5pIx6sjAer8qA0n1BGSZOIU3lf1Sauw7mID\\\\\\\/l0QDZsA9XVCDjYfcjsotcOdX7g0YLcYGJkrI2h0B20NFkFmkshajVWajQoOsJlKm6nNI1wFqE\\\\\\\/0yeoMz9\\\\\\\/0Um\\\\\\\/AARCGxhekJ03qc2cReBEXvT6ZK6XO5IgToUrDwu2Goz090MhL3YkDpjTGNYY6wNk8p\\\\\\\/rVEEmPbD42ms5eabZZ3QrSFmG3HWft4ouvZQbqJ4hQCZRQzZgVrZPx5v\\\\\\\/OJAngLsXhlf3F\\\\\\\/nkHlM0BA0arYye4O40uiiQYTQh5OqnFamsRaDTZGZsfElNulwNAigBW1lMk\\\\\\\/f3Jm8bsM8z8Md2SRbu+XzqP7LdTuuj\\\\\\\/HeUKthfcnX8ZnBwssoKn3CiUYbWR39P+uJtGiEAmEUwBQGDnhYPQkcbUfvAhop99\\\\\\\/BMs9S0wS2AgyJOD\\\\\\\/SsN+YgswEcHMNr6wQ5wufdgT+Ywc\\\\\\\/FwddRr45VQ0cWMaMRKq3uDbfTp+1ch1LOnyRzndGXIydpMLMH8jggKCsexaUuF+4OXuyLbJN9k8J9lS\\\\\\\/TYgaXOc\\\\\\\/E0W0OZdkmSJUhhBkh4kRl6HUxVQh7czoMdocpDLgR6RbIe53W3sGKWZrCK5JGFhQSYFcYZ1O2fH7p5dvzjLFa1d985uRcJFcjYdBymUNwKPkYrQ6F6DfH\\\\\\\/rrthkLs5fvfbI6zxZWK6cK6IktsSMn\\\\\\\/MS21nN\\\\\\\/2P6DcqKS9R2m5tokfBllVze4AJRY1J\\\\\\\/tNVvkXjuls5GLqbsRdBb1COX\\\\\\\/1fgwGYSVLHgYVsLNSZKaebc3DN9QQn+mj5Y0cOC+DeBLgtDOm\\\\\\\/Xis4t9T4nYBCTx7ysYRtSV\\\\\\\/jypQaO788jos1mU2AvhMmHOoSq0mgfhAJU2iq7lgE9BQexDCnutCTDvHb3QZF\\\\\\\/to4McYJGg7d2\\\\\\\/kN8b5f7Lon\\\\\\\/YdMqcX6GwgpowEcZe\\\\\\\/4a0FpDXy0mA03MPcifnt9KPCwbEtfF\\\\\\\/918SkoTWEUxpMhWt4lfP+fWtfsR5cnJy+DboJwS\\\\\\\/pxJixbvOE6AjwJaCwejGv20DbNF+AKHk1f36UCGVTvAU5Dhc2Z5aLXFNR\\\\\\\/NrR44iY55EASWaBObKlYMo\\\\\\\/l2b8dqMBwnqugZXw7QoX6irxN939f19xhjIEDvYt6XkjqdtieDkRD2jx6Iiubg7geiMFeWA\\\\\\\/nCleEE5K0bw6HqJAPvxWu3c4u3v0Qrgwkz+IIM02XWp9upyFdCUUjcwTluxMvSK\\\\\\\/yDFYBlib43ALLK9ET\\\\\\\/2bLLkgNzmRno2j+Tazcy1nbbBHqXO6J47f9hNfvovI8gx\\\\\\\/WKIRLXgp36PJ8nhqOFZSM4zYCJXQeX\\\\\\\/mqZ2oCYoXz1heO+XvDCZ1We\\\\\\\/tO1+hz7FABbKEZtw3fNoeLEZ\\\\\\\/\\\\\\\/+J6MAsLGU4+D8CReAxenKnqwPVG7IX0ktLkhYlxqQbKYOSvMuTCYGgleXpRKcIHtvUhiZsQEJhJS3tUh\\\\\\\/dAApHbGHnMzNqrdcKPMosbTzU5Z7MlF5MmegLff5kdyMRLibigkK0ovSVnKLgTQWNGcR21GcTvh1uURfjwaAhK93W0TzrW1C3lDq+tXlh5xSXJa7Ebd1YeRIfCyilWe5VWzBOil\\\\\\\/qpI2Ss3cTdLOa7o+XywjjcJ93Xu6d6cP54e+aSwIn05q1bqEYBN4xnV+qdj\\\\\\\/vNiVMjoFMsnVCkjh8cr2XxJfilCPF9U0I1yWfs2BgfOIZL5B\\\\\\\/Sl4O2lPKUzdzVZcl\\\\\\\/TcGI2F5r5D4UtShyxCS0EBOlPufbFuBhCUwhe\\\\\\\/xvFPEcLi8Rr6w7xJZPZVEJ6krVcCpCqPkmOMbkxtaU2cxd4j8iuvUHNp6q\\\\\\\/ksGuylCM3a4flaJJxKf1wJzac+oazSJYBEISwMGkbXlyfg4xxRNPB63Q6WNb91x4vbmycStvdvvE+gL9msv4nBzsl25B57qvJ5uytziGAQOFghAdypQmB\\\\\\\/4h9Yeukzrj7Sb+nrIgOjZavGs86Q\\\\\\\/Szepc2hkvTjNnkkM\\\\\\\/er6tMiAgS6lWCgSwqZOGig931nSJ6\\\\\\\/mxbxWIZavLqEMoSHKzpinrL6K0S5NEXiFc6k3ihQqNkPC2JOU1p6pxVc\\\\\\\/LKoJ3EFSJ4gouj8W5zepMNvU9LbDZ5S9xe56qAn0Hq3QRJQNqmKEckei8aHryfVcQoyNhilOAEHM39HCxojxG2g96rLv5jUn6SmNnYmp6GJRF08KL45HLjTgkDvoWZby1qieDigZdpqKj0xQi7WAzRB5jPI6\\\\\\\/jt6dLZOukGQwQztxoo\\\\\\\/stpfNg5ymyJ7viSK8+AXb2LD37lbqA35lwywAlBh5v6f3v7OJH8lsc\\\\\\\/omM4fkRZn7uVy22SlG58ETfD2Le5PRZXXF78MYv\\\\\\\/dIZpQ2FKDsch3my\\\\\\\/SpawbCtH3u4QHMuJEFcp70UbiWBX\\\\\\\/6f5iM\\\\\\\/H3zvvETvBB0zcwwvK6wbRpCn41G5ekCjMznkG9cEfiayLq0ejOjbBLxtWLG6aX17+VwhcicaS4s9gij4LVigI3hkcbu0hj\\\\\\\/KE5tNookzDMO+iPGkfvT9fnaSMnBHqpziQGHv2yfwTcz99xvEfuGDkVfKjSDE+ltNIX+xgijNeMLqCRDtnNzUkn6+rIpgu3McyFJ181u9ZeDrwFY\\\\\\\/TRktU0twAzv2hqkTTklQ\\\\\\\/pPFdok+d5iARnJkvrV4a4otnVHy79gkFUuLyq8xqSFGZKZ7AjSrBFReSzUVP9D81Bjowk4zqW8fcy936SVi84dwhvXKCFBjfnCK2Y8EMXAqHn2iD88iN8fLJCUnbyZG3TR0cs4YqV9tRTej+5SgkPkChDLbqz\\\\\\\/V5op3qasTGdQoxFDSLw8lclRH6qIFwV3cJb7u7UYW38Wi5Q4Bb7KLz38A6q7FKQTfilWayzK2Mo7Y45zM++Y\\\\\\\/MATFdhoo4ga7\\\\\\\/M8GskWFDjyqTl2+t4asl8tV5LjATVZXcA5ge7P2DIdxX\\\\\\\/QEXOPdpvJDTgxaNQIoUWJ4N3juLVX\\\\\\\/6KyY63bzymqFSH8iAuzPgbwuVFmPhJLED6OeQ+wHNHrc4cX5kvtSEVg9mBF5GzfU\\\\\\\/Mw1r3z8dPw5Jl8UMvKTo3yTHl7tQymnGc5FZ3gruLLWihukd7X1Kd801lq863sgKdB6HumJT64J7pq88SZZW41udAaMJ6IC4Y\\\\\\\/TUeQHayJZQSP4mljtRVR3HPpTo14U3HyrEM+OO8DlV68QkdY4+CJ3uPZiPFULRpqiydgWdL+OyI\\\\\\\/FegTXrB\\\\\\\/5iAv68TXZa0ZHUNwlofgISxGbz4Y9rfrutUl4o9B\\\\\\\/BBEbfoVhsImSDjByT4r\\\\\\\/o+zKsiBGST\\\\\\\/cLt\\\\\\\/jbqff9XW+Ga4KgQKqPJtgWVEK31Bt+qEq\\\\\\\/H9KhBycJ3dnEhQsjIq0Tyh1+KgCmXip23WRtgXm6aIIz8I3JQjAgwWb96dG+eq07a25BXabWeoKqaLSpUPNTPHK74jAGUh4ZacPwi3UHCDwocoU3j2OHyzeVIVIbCxmUNiy5mYL0AcZ0PWBRj0y9YE9yKAjirfcSxHAs9K9nY+lS9oUF8w93BFScanqpviNZCXy0zouGWXrXqC2lSy0+3gjYOrt3mvXniQupPEvXhpPj34AEAQSKiV+atvIfoC41g\\\\\\\/AVvsXiewqEkt\\\\\\\/QMMc10nGqnEk7bdC06Lj5KM1IMRvxANE6ltYLjsHPiii5YJd6wPhmWZVgho7dS3RC6VG5H9f+gnYllihxCjRrYBdsEo1hBA\\\\\\\/1TuIpKG0ApFcVh2cRQ+Mw8x+qiZqYKKzFOQbjmEptvWtTdRLRF+qBtxRGfKVV9l0oYDtyl+K2j4ZDVHw0VRCCIBwLEPWNdBXnkW\\\\\\\/lvx43gq9dMFiebtN7U3smd6LYR4I+OTWojzW+JxRwnmJiagc1KUWw6tZVn4jzWhMz9yMvcDuyw+ShIwxdjSDrNhfjlghRzTCCKvNqeyLH5etVEuod6XZQN4bzE4VgihJa0eBgfx6p3nb7ucc\\\\\\\/+6yujR\\\\\\\/FF2yvqIVC3MBjbHpQlU9gldK4NftrHDKARRLIdm5zRnZUtAsC7yZ013PQC0Y2pheddRq+Qc38AHP0EBeNvt+oFsoncBk8MbFhrVkwdNEk4tjbF8LmgH+AjKbmo1fMEwsg+tq1pV0VfAPAjKhNCTkak5ahoe+ltIhRxAms5qqK89Qe3POM61nmwrSwNUHn2I0nDFcCZoDMBCS4kypCpCN7H0t347gTlLhtUPzX7uE+IIBfBDzE4zzBpIXScCGQpItjaMrS\\\\\\\/TF\\\\\\\/WY+6NImUZa6Yq7eRcO+yR2UTTtpjuYbg9N8iZ3I1LDYvXqe21AGV6a9DoDFHqS\\\\\\\/e7kmNipyG9i3svMZLzNZNudRkeKxCkhwEmMl5Y65xT+0mNAjOz1LHRH8hxB3ekkKuWg6QI5Aq20AsWurbmnY9RWuIffMuYYC48WWRqY9s2+x7Uh284xS5SLqMtY2nyI58\\\\\\\/To3eA\\\\\\\/ajM4EZJvMeBkuO88NOgghW8vwh62lCa05yryMeGfnZxG6BewXMUB\\\\\\\/1NuPNz0BwaA1MHI67ZnIBKQmaniAe0yZWI0AM3SChTpVtXvABJ2drnpng2cbIJue+qgmuCuKoPF+Egm67ypvKal7is73e+lmxVjLU0y3hFy6Mfc1XOaMJnAkNSNYBoMs3g9O5K8PE3VqdhVDKNPeirijpNylQx9GkAc6e76wG2TDpRHk3gmxlb1vsx5MXWVsV1gLjeKdO02Mf43ZuE08QpL67JL\\\\\\\/4k2ur5fi0Xz24gomkqSMGSzdHF6oxIYQPtK6ZNukQBVTBLzgobbbLON4cSVXrUGRvmSJfz8GtquH\\\\\\\/QJnka9TPr6FDMvO27LYUnfbAv4QjN31dMcD4R7CwMILyA4B8q1W\\\\\\\/0IYkwjyqMPdJSvxyBKxCK1ciV7yHuAak8iYjarTWLfSpLsr1tTnEE7dtuIowI8MiDO3jENLGWnR4uB3Ym3R2mEHh5eoxbejwdaN1MSrqvS48ynjodKpORmTIlpR0HU5TyAGTN8lkLj0hLKrxRXCDrXAKT9fmw7\\\\\\\/L6Fc45UXPRp00uK0XIEk0Rsd2chhrtNOLDdoP5ybYs5UMOSMjOYoxKuN0+u\\\\\\\/1zQkq\\\\\\\/P75r76Cih214pKBXsuk4Cr7jOvgJ8W1TRNOd0+JDkNeMFW3UvrvQRzrtzKT1qlGuRTKK3b1k9FWXftQzjIvS7DyvFag3RUOo8WSsfvNQZXGkwSZl\\\\\\\/SVm+l9Y0HA7\\\\\\\/rvRfgEUyQ\\\\\\\/Pe11L05+wMG0kDI1im8Tf5ZrRTz6HOu27ZS9UCZrS6dXyAMSay3SPK2RbHHTCGJLTa6W17XR+9oSu27T0+PPgcVOFK\\\\\\\/0IAZuE1XayZIgEVRSYKKGmPUq13OsMmFDFtnkJ1USXrd2CftXD5amcwGlGVL4MASdaAneIoWHt4mOfIHhr66YcwPgTcGzxgVldi500+2SAQa1BNt\\\\\\\/iHp6KmSUmvkUd6S9wJwbsupCiYrNyRSmIOSkL75d2AJcACBt3hCX0dQYL4l8a2kFAFN3e0KJtiZYw3yUrTfz5QVS83yU4xiC+yK5IjbKB+c0NS6rEHgjf2QfwpS2VXohWjKhHQD5MBRjashK2AZVIzdrHfOgUrOJQZRhtm+Dui8FyWpUjK7y9y5jnpevLsmJALapnMateJ9IVy5wP8eHPAN9SONgjoLfq+m0iHCwLz2DrWvV3PqrkXpqSzUdzbG\\\\\\\/1FVHT6Ygzn73fXRMc9QvU6jikbfn9XiRDoCC4hW2RbJDfcBdIx2gSJ5iUZk8rX98ygOJBJL0q42GKwtM\\\\\\\/pi1Xvc\\\\\\\/nS9wjd3r1pNc+nPK871LT37r77kyMZF39+fhYgswtd0HaI6ehIJAXg0kO+517GHPN5p+UNUAbZUFLAtvZzYn0PWkbQsUuiUru0lcPQN6dRFeTk5kAR6FP5hbeSGddRGfY7Rocvmog9l3EIeo8E9AAlPjeGgZ36O0y+nB\\\\\\\/Q1OibUmt4Q9vItqHC33W4L5AKfvRduIDtXgrdADF8SY3tHkhM1VHZreGlvyakofzJfFjW7Dq96GOxO+Sf\\\\\\\/d+DLHKGx4AzurzwBTDERVN0mOSGy2FyQUQcfcmCPuZluc72txPC\\\\\\\/1ZIHywfzxmtFd4VP+3GlbSFPHPS\\\\\\\/wvLV9btT3\\\\\\\/Qw84ioe3c8J6sJOUE0U4A3mY1HLy9sr3ttQx0wCs91mIGuq2B5Y\\\\\\\/SFD9gKtvp9NK\\\\\\\/zLn6hM7L9OzxbCFfsZGErN0dbfQpuJUhrnBEDq4GtcZe3yJyqqWC1E9IqlPaEDoa+yY1IeXWw9iXxr+Pw+9IKeBcNoJICn\\\\\\\/Zf5p3RbAJjby2tz2SwZCcjwuyDhkZTUcf+Vw6K0pOJFUGmkhh8w+Ry6EWI\\\\\\\/j3VHb2JWGbsFTIfzskn7BC2\\\\\\\/vHenX+YpQCsyGUvz5EbD6ZnBt3APDDFIccqDbTsESeX9y9GEClwXYTibr1lCP8TJ2dLfGs5CqV\\\\\\\/Z44cyEs05dKSBjfYRtWqKVcoffmuwRlV+fWOy09zuZZ7z4bmwu\\\\\\\/duJnoBuZXf3PvJBIo2QPlG05QbfFWE7uOOLBxxP3jFGZDQnyhmwUYvEdlAH7aXcZYFvT4tmRFyL06b7yhhZqadebBB2PwJgR7HJdlhma8tEZuk8qruFrdmiZOeCknmOSGTxxQ1s+ZSfm9w4kqkLPZAcj8qS94LnWVMbaIhis29EVYkos8BqZf02egfubJDL3emST+cUncLxw0tRsVwKyUfCYBZSrAe+p66bHCHdVbUqEH2xnTt7YWpPUU58FN6AxrYm8v0N0G7ggjl8AJpBrkEUsMmhnE+jdAA\\\\\\\/YIzvp6pkWjqwzKavpx4Tcg43dOBvVctCJ+WI7qVUu69JDKEX7PVX4FKU4VJcX4gUIdovf\\\\\\\/00F\\\\\\\/rWs4FPQSpmvmaLmSdHZLs7CPj8j9xvWSOrOiYymKUlqwnEPkFu7IHanV2NKzp\\\\\\\/O3pNH3YvemU3o2UxYt42WE+J7FzCFpJwSvnbSodzLV8294eQNgsTJqcc1gxfG7lojDLu2JC+b3ELWcjoaYJYg9Qsx6owI5pR6a0HQCLb8xgbOXdGnlN56mZMV\\\\\\\/Y7r4yVEuChQETX\\\\\\\/nHCubLgsqJcRB4M6T0BGH8v7GgmiHuZ30fmd8y4y\\\\\\\/t29\\\\\\\/+O\\\\\\\/tTpDid3oi\\\\\\\/d\\\\\\\/sEAPzvA4nOCXU4WXGOSIXbRnqVPH9EYki8SDM6sgWlE5dgtiISY37ezCe033koYuOh0wTgDpWQ\\\\\\\/XDtRmUr3QdA6\\\\\\\/60HFMKjcukbBmbBoxum7EB2olUtC1Tq54v42dXjUquNjo1REa8t756f9W7w3K+ZRsebc+55yvpvVIWn6eGbhhjyWaaOfkSpqD+nR06p1woQoCuAImKemDN6sbRBbaBfd1E2dhsbA0JTvooVfzANh21ohN9PaMjU5jNueN+Y9rOlwikfsVzo8cn9QWLq6yuEz\\\\\\\/uK580Qoblyj5ZgabXN\\\\\\\/plgLV9IfvWmKqpatUEdQuoas42Kaw7viK7iD65IsRbeCgGPrFMiw3gqHigd8rcPttXgBANfmozOAurEJcg8242J3tATXsQCggpDxTDAXZi2NFyLuuHCupuM6lgUVbESbk5WshqtBDZopAUJ63s0ctFhXH1KA9LkNlyL4EAZNz0cKSR6bS2xNBTgS11NT3WqJdfr3YRZLdigeLF5M0+PEMOeIeZ3OrUaQfB6fwr+i37TpXcqm6YsREyU4EN8MQFYUqKIXCOBVuMNHvajjm2lZfZV0pjvglrY9cplaySgrFnlvHnGpndq2dCAIxI7dJGhvjLGPPqctIyb6yxPIKxHvwqOI9CYlhfTx4hx+Rr0nZ9z+GImCRbgMNvnQJXJS9fj2ZSvDzee7b0WQI\\\\\\\/OMlmV3ZUos4Ek8fewrvAj77iUb+3xcsoyjYi+B09SF3spZwxWgv58eUzw79ESJJ0wNg+DIgKMk+XU657TZC25E5pMqlbjdfPxbIRGd8XpJJD7xU9vj0XMtHHNOL89mTDsrV8LnvIhnrSKLszcmEhxMzbhtbHnW7mhbBshYRWWSdgS6wywMc8lpXLBTSJEBwxK8oHsH+Xh4MuiMD\\\\\\\/+ufeW72NbKR4vjXGfgRJmPj2Ne7PRxbyyBpER56uu8aJr5k8wJl6B1BT7uZqkPK6LaBrU39W+Fno91wsGl4sepKVDBjIQ0\\\\\\\/ujefOo3hq21lm2V4C6gImfdo3avo9HvhAsrKgvKhd09w1kt1uED8qszAnUoFLbQDNDNI7lRAioSGDpK+JMTqNBsxGPsBev\\\\\\\/y3Iiozb4P+mrQ5xtbgxJpaDU9r9BPHwUXybpgo+LCDIwMlsYRUH3l2ehugJy\\\\\\\/BioSTERm8Hy+BVtP+JwwWnIQkOd80wpKWxI1Ofo7t0zp2P\\\\\\\/ufKmhTw\\\\\\\/eexQtmvOw\\\\\\\/NknKB\\\\\\\/AbC961CQPLeXF83+SJunsFLyDUfPM54N\\\\\\\/eR\\\\\\\/WusnJfdcA2M9V4xoTfiuHLrwznLNPMQDOXG4b9h6+ExuHv8HhTU9gGRBPihvsAeBSh7FPZeyahuFYkAeOjAc2vFvn94RErCXRBH2EJazHSZpQJxugdcFlCEmS19ftFnihn9nCCoJHMnThXUMPuPAmU7DN1BOlLz4GuH1RwvVajzmVKEoxbFK6vXVO3MCrQM9bwUzXgdXtIJaleh9gaNIXYQMMAFij8MJa9hxKw6x5kjHZ49I\\\\\\\/b3cNWoaR5scYEmC63Yw86oaIjr6s\\\\\\\/ewhsT78Y\\\\\\\/28tXhiaMumPnBu01mZGWn3HzWTp8TRnVCQ1ZpA32ZV1OdOzuI+h5yi4Gg7\\\\\\\/ouzW3cZK1ODNJZNI\\\\\\\/gP3wl9mArvoiaVqcYu6MCGIJvH9K5HYEqSbVLazHzOfni\\\\\\\/7hbZPjKENu\\\\\\\/2fnCzAj\\\\\\\/uBdWYC37zEPt571kaGP6+Nmbk+9+ZwI19Teqr8boUNO7eSFNbw5QUzJx\\\\\\\/clAbuOdmq9epk\\\\\\\/2SLhtxIVDcHCN4ko9w81U6rbQIQY81w0uFvRxD2AJub3DJn2NKdpH0PhQqaPZZ5kyZwCrtDt\\\\\\\/\\\\\\\/8L\\\\\\\/fNdIXN7J3Ifn9i6vkmf01ALREkvfZ+SWxE66wuVokEbB5P9MBbKqa5bBkRuk2vMRa790SpvdnqR7B6QRQ6QYTosuXDFxqo8ywOkvUlOyk00eFCyNaLhjDnGz5\\\\\\\/rTlO9C9kXDgnalbUwdVwlbv0JcWVaq7vhHg3bVUC2pXO1J2RQhcQUHoUwEABbO3U8SYa6JPY6zQuIOgkGasKFTTYKVzvSUY8tVeaCRJAMWtaCzF7xvmLk356Xqt4bd77TgFT+vqh8awYgMLv2KCJLXIjBSJfT6142qLeuRzXrGrMgScOJ16wVDg2vb7C+yiDwkfcbt1jfohh+LrXF7sE9fBPZVVso29u1WfObj9dTYI1n5Tj3MmWfbGWoYXRz+ZFtdT39wvUuEIpEG35IR8w01C5x3ZMCGEeOoIn0xq7jnGw5DvsHw8yWnUV7b5Yy\\\\\\\/+kBD3Dtm+moGjbh8zGR79UwbJyizyUqosEC0Y8wDTQhS0lypyruhh5eRFXoF3I1zUQRKwbckkmmPBzQLtGmKSSNct2tAO5u+zi+DPRFWdDkMdBCEEIGdSOqK7Jq\\\\\\\/NsvHlFHYaNSGPdvmOSNPJsQCTAmNXPrXKf9xK1u9mX8kNKjM3DiuJWDDpeUfYL9TzaRBXP2FetmsBHV51ROkfGEaK8O2Nkh5a6L+ZOKNmiVEnLDbq+yHWvTEhrHn+q+Ij0zXn13YXjFJ0zUIiqTGAZ5aeJZ0L99m19ZP6SxGs2UwDE5c3i\\\\\\\/lYtUT948SNSx9OUMlE49D3IeW7p+fkbwVxGjBpUdiyy\\\\\\\/J3XOo8oz73K6qE8c91tgtFiMkkvfe9JkoymS4xNKOb11gZlD+m2in0jNQBYeL4ORp9dqTIFEnuSuxBvOdBX3\\\\\\\/\\\\\\\/mCUguNgxIRq1\\\\\\\/AC0by9l\\\\\\\/l00rbegml4K0EbGnim2SmYDR1S5Ya5RUYIpaVlHDCgJj5HgVUs\\\\\\\/NCE+BvXM7lVobFLBYQZgvOFQidt9bQ+yIEhz0TIK1HMWcdUOb7T0nDqBIrj5iIJxDTyQeuohi+GHp\\\\\\\/TpvFxUUHjlX1ocA1PynyfCED\\\\\\\/TniO01kZ7oXGpWtEs6PyrHEovu7au6XXv7G9ALzomMehEw6f3LVTjSUF8Lm6eXm8yu5oKHbcLfG7Gk6UqYX\\\\\\\/sQx0g\\\\\\\/o5bKLHn3wgEQeaxK+TfKEll4S6b00\\\\\\\/hpmoG\\\\\\\/e9wXdACCI5w5tb7zAQMC0FLkkRT\\\\\\\/hCxbIrFLBvjT9tMdTz+d2o7S74\\\\\\\/VzzGGnp1S53A5lQHFkVPUwpTrqk6mo2EQN3I1GntP3lOW7zvASxXbcGaPHYhM+T+YNzRbyKX4XGWuJhpj9+1NjxfahEH7B8lJKkn12mrvc4\\\\\\\/Q7ydP3y9Qrk9rZq8BDAdlJlvUWbsWeQn7AZNJOd3hhGN5e52+FBLFh2SzivYAAuXR4klfdexZV9MsXoHTnTjcXrPnytDRlDBp+TugUWw5nDYqZzrzslpBL7LHym3pv6zICwy4XbNtgpS649kYpzO6HlyFyz1CRuWyaYWzv5tjwtQSnzaqOrv3HIOm+7ebGN21EFGsGrEkJIv6FGlcFmdqS2RqGrb5RD5qkEcn20vWCfPsNd5\\\\\\\/lAZzN1HBZ\\\\\\\/syd\\\\\\\/LbnvcjeEiO8n11um6ESNfjVA7UcaFrvRvza1f4e5xngZiqdOhjpMHMYpa+L8fj3L4ZWdtRn70+uBtR4daOdi67x9aRrjA3X46pz0jbccbnmNJA2Lg4tWF60EG2AhIMKlTeYjOu1Qjl+FPQsxkoOcZBCrbLKw+FfAptoJ42lMe+8GJTETUm+EDbuclg99YqUchkzMRh8yewUDtRiavJnhfzNwN5QAXEs1SoUo1hgsVatNLs8TuIVjGM5wXZv8z+QJgTEeezcNIojoGHuFIJy9We2XX0aGCWGjU2zkTFpZbNbVlIoV8733mTffM9uHv3AXc0PwZ5q1WN2mcWt5GszasBp831agMxecQZY\\\\\\\/uIgj7jKqlgqR6rRanyhzkkx1mb8KcludsGTuZ7GJ\\\\\\\/xsNSO34FT0LTkudpeRD68yOHzjizY0NoGexAYIFlk0zkI2gqc8yk5fpZOl4qLyQdYRIly0C\\\\\\\/KyL6LuUPDht8NEkv67pCCrugHX+IE3fRinag3EnSiooeXwv7SgEtdv8wx+gOyMNHecMdLcDJOf8zn8vzlnsllAzlED6wisVkj1aSH77wrUpBMJqBQKlP0w\\\\\\\/\\\\\\\/fcusieF\\\\\\\/uLLKpUgbuiEQrtBg5WX09CXJ9xQiIJKZWbVKFiRRticHWRtrLa2eaJb7YuywLCi+K\\\\\\\/z+s2DtzIj43Pl9IM\\\\\\\/Noc5EzpkfQYwXygSq3MvcarTtDccEsVq8B24o64WrNAcs\\\\\\\/jwpvphzsVUTEPLP+5aO9EsKlugnjweUGbfuTmIUq7gne3KU\\\\\\\/osxGdbOh3rps4nE1cBgkYMDFxqWPM199oVGSQ7ah+C\\\\\\\/9AYqIjeQKFAssJ7wglN15TxgLuurcixMHnikddQMl9pGqqN4SXfh4OuAT+mPEojddxj6xLyYPyVugFTqNo7427k0u3lHRd8GALQjuBV72qh2vFNAD5xTwapVjZIUSygTAp3P2AStM8VA7Jf0yf9Ffk3Zz8gwvyNIxfpzdt88hrqfGa3IZhtWmGxqaurKGp712UJvRZxkrkT1dvZLDKnYyBmyrBus4fJmMBM4mfPcZJKUptvKjx7x4PmrForFpUYvdBpQy3LM2HtAlLBfSKUuKfd7N\\\\\\\/bfEqnAEIG7piWv9goN2jJ1c6u1BLnvwsADXBsPGHvZ45lCIUFoEc96Y2EtX9ECOk0dru7bOhET1t7JEP2WEFLEasRhUOCp9WEeTOX8e3eL5ZIPbjFTvDgzPeb8sn6jNYChTtltAM0tmeK0s8W02TPhBikiCRfln+tyoPjefWmMXvBOKCfYxgne86Ws\\\\\\\/v1OUkj7dSgxhOMgt1+IBkm4+Oha1+iEKnDlGOx\\\\\\\/CRYFeXEel7axievo0kgVfQERWhOU690I7XoLTpjBuwUzB47cEnJ5Kd\\\\\\\/1760MJquoz7RplN8Za73vLCtI+ePar10qmWarNDE1tcfc57+m+5YLCuW0WNmTTfHqnZM7QaX9tYiEORDbHvYa+ql6UDPuhGSU\\\\\\\/MCo1Vyk1JwUzT6KLg7zHVqNHkP5\\\\\\\/0rQq\\\\\\\/8nJvfAxnm32YqkFXjI\\\\\\\/4pWoqQYRJCh\\\\\\\/1f\\\\\\\/iMSKJeGSmWtoLxIPXhW8AM0JVSiu8iDaewit55IrTShsw\\\\\\\/TaW0fgIINK\\\\\\\/y2FQTl6NxdBpCAYQRQrdJ+decZx1TI6heqsWE5+1ynfARebMUeU9tiQR5F+tS03gqY7CYt9kS07iBzvq3AyxpOML000LZEQDmf9bokM0Xjy7Ly1weReLrE1XRlIsGS6OtQuVvTpOHTv1UmFXVZmqFto4RpQhc+G9ODyYvc0O+habG+juDsMZxhB2Y+tzhvI3gQjS65v9O7x9dZPIkfrRmeGux6u7y+AC59cj2Rn6vPhEuzsTDzdpuAnsi24um2sLEwVBlJOFl\\\\\\\/Q4q5+o1UFTd+bYFHVajxa1ojlUK4CLP9Dk9Gmg7p7t9y2h3zfm625Bbsps9SkI+3JF780lMwOstSFhNqcWY7Rh6+QKTlvLkhnres1WM2XPjMgejt6Q6XChLN7ftlkI2GDPhz1iyvfAhkqt0we33yMLm7ZKCqHVVl96U7OfUUZC3GaMV7T4Fh220mVIdOW7CDPkoHwEFwh\\\\\\\/0VKOQ2ziu8hDiOblJFkK4G7SUmwo8SXQ71zSa7HgYD8iBzLrxWBia\\\\\\\/H47PKHaCZhBwy0mMFvTVG0dJqCs7hvp1cTcbcLNji3n0zz\\\\\\\/beiZ1t8bGd65PYWH\\\\\\\/92ZTcV51SQNHjBwQq3lG6K\\\\\\\/jzELu9nvOb6ECYBQMFOYchT+WN0IHCr3JOEsMAJAtP2j\\\\\\\/IkRxq34FGqLYL+uV7UT2JPdZfmQ+\\\\\\\/ith3b5+pqINfY5YGgiaTeKu7Ts3uaeo7GxbpnR4fzxTigWu7h6sqQ8gRsSlVLm3drtpnhtRK4UFyf3mJFw0rfTOKGb\\\\\\\/JjYJccTbc7NO1ffYH2GRGq7Z\\\\\\\/oQqwDO6HxuJYKsr3e3QvLMBQ\\\\\\\/KQlS4A8EFV\\\\\\\/NBrU7Ua8JZmORsPouSWmU\\\\\\\/Snat1BFtboxiZ15sgTQTXCPfOB4n1ewPgFJrffVKEMUj+cUqTxIpN38\\\\\\\/P\\\\\\\/1261zBLrF\\\\\\\/KreznZrhFx+AfvDNAgDeoqxFjaoJw0Gc6MchLG85jGCW7J8E6GmY0iRAd3o+dX9RmD6tjwQX0hS+bXAqU7OqHQ+BMtTmacn8LG\\\\\\\/NO1avSFeVsslne5XPE6H3x5sMDBwfKORyAp5h8tm3vhAf\\\\\\\/AN3+Fv2YA1w8nqC67wor09L7DGhgBvL2xTlUY4vcak6fyZlAbxc4kKUUNSdgSaJoWO9KL07tF0XAdnHwS1IdBE7gXIA1Rfh2+JuUNBCZqmAanldlLOmQJS2rh4ejB6ydbXPw9niJBvPKisC4zMMCfvnnzquFCMQCPSrYtvV3+jOb\\\\\\\/HNq1c8ejzTgg4MqZYakpuPBkSsT+uAjxugJL6SU4XNKAVvbZf88sPoDus\\\\\\\/BiWvUTdY+XgsBpx9qLIgBbWmNadshtlVJHJc5yLDKpXAM3Nxren9+\\\\\\\/8Lj55zmGAHjNpzu5Ju2lGhoQQLNKEl4d+I2ZhzdRKo\\\\\\\/m8cHUBT9zjkioQCi0AsAaE7GyciheBrmKfK2m4Mnyk33VuELGCqMFfadPzBC+do9TRVuCmAVPu+69sAn6gvx8vPx0GQuaLqwPMxgFB7NkPx19UfoLX3daAFiWcehvnIev7S32CJH2GHyJVRPjDeTdbHRNgOHAs7e0rbpdkPBNbtVSDQZHvbC4CAqbWb9UMay4VDIPv6kkM4ZKaT+Q5d47FZbfosMMe4y5rNyj7nWC3n+dfLGLKjApaFmmFByL+JD1vtXsB233\\\\\\\/qkUP87OioJojkyFrhoyWC1oM2v9KRC2vNKEKjrY3iMr1d4l6se+7iNuhY+uafylRSZMrvFDRlMaq2qpZKlDAhDNL8CVlDWvLTwtz0CasaFwGVvLjRA8oF3zNtPDT\\\\\\\/0cx2XssXN\\\\\\\/jFSdj15W8gtWjV\\\\\\\/\\\\\\\/lc77nUYrYRqtWFG+jZcXDf2pMiszFFyrf\\\\\\\/0M+s0oWXexKtxHQqpm7c1DNyGEyb4h2nuMee8gpy8q1fNukzresqdCrCY7OxskBZT16NOjL7yL1scTGogiOTIWCco3\\\\\\\/SO80uP\\\\\\\/nVVvoFvhLrCi\\\\\\\/\\\\\\\/ovgqDA6ZB8VtjfgduTBdGUDEgCVQOozZ0EtaW3Nh+ZjqRcopSEpXTSyrHzY\\\\\\\/VZMQhV9nszduBTEzOvfF2o5U8X7GofuGMkRcWn7wxQWZCP0gEObG8BLA1L2Vy+aK6Z5vnQOC8CUR5x0x21Mk+vyVJvXJBvpQ6qUBqwrbdrFAn5GfUKBd0eIt+MCX15R8Zs0I\\\\\\\/HFzQ2KNmRvOsLhfHBru8wBrb+EGr383qo9ndxiBogxQHP4N\\\\\\\/3SYonzuGWH\\\\\\\/NDtoxAT2cSe0e+MDmW85iTvDuF\\\\\\\/NR1+D2\\\\\\\/\\\\\\\/VaJ1jE0EALzCv7U4gXMiCpE3ZBs7Kk9lvza\\\\\\\/0ODVdKSwgOKY\\\\\\\/ROr9imxxvt2q1O+QX6cIcfLzbUKWGUppkc5Y9\\\\\\\/deIGG\\\\\\\/2e+ynFOQq4Wyue5Y0VpIUl7mKhk+\\\\\\\/RjunJGlueeTfVOMRRXelNUxN83CxD0SwD9MvNMOcdzlrGjTqURU\\\\\\\/FtCx6SzrrBzcL9wN5zLnpQ39XLizJrBfeIjy0Qs+pdxojvHmggCkEkNL6n7Cu015QT5mPZZRIkIHA1S4fxehfCqdIUyie6AjoCu\\\\\\\/34n+wccXy0ya5INPub2Ovb3bNFCwbqqdzzL\\\\\\\/\\\\\\\/DK2a6nBrl\\\\\\\/dwe+LICOdPwNtTfVvrxaC6f+Hvm\\\\\\\/cKCLvQXlVGzpj0VtQCjRE8FcHgDa3xQdQsxw2LX1OSr37uyIfyCJJg5j3coyb\\\\\\\/atvKhBbfJoERs5sHGr9OBkLmcTu6oipG\\\\\\\/K6utTZzrpJ2yuTq6zYoaYGhx6LDIfxSgeFn1xw8ukH6CvQyUIEERivEXPBwlpUhlCWac0OhS2xZKdwCj5Gzbl2BYtV3zXLwkKg7swXw5sVl6xEPrBazNB4f+Bpt7Farnn3+iUcUjDaSlGCsw4nmKtdLGUf4xuAsU17YkfVFlIkbVrVxMtHYoyFWxi\\\\\\\/rbWDrXhveTTdg0oOqVgLewfN8QVnqQuUG\\\\\\\/vkDSQqNo\\\\\\\/IqNoJAwyIHSh3stlPddzV\\\\\\\/23R4cjq4PnJ1tcGHDGEAMdq1yJ4MEg4sWpblsvlJDA6lGG4OK3YHEO3ji5rSRMwKIGjpGmnyyV9FVb3nQUzWCCxE3oaUpg7XL84x4HfMtKTWJf5awVj+yIw7CXYYQATQ8x9A6KEiEfdJ886aGXTFq8E+N7EW5Ht+NMiCjTIj04ucOF8uw6MAim3GK7jeXtceXmRLe4ec7lNcUKnYnMgDs1\\\\\\\/t5eB4e+Wyo\\\\\\\/zR1I6NYlmfp0hRkik3O6eBshSpYW07XrCxHEzwqx4ROozHH2ySjHFLBntvwO7ngeEJZ6yFfvg1appieFGBiiYtczNjxF9vgLF6d7NUidvL6DR2tSQU+v\\\\\\\/VWEh39nipbQ9R5+Y6q1Qxjqiv6Bx1DdTVsBukSL89Fazw1WqDdnkPMAmQ9zaupwOGJ5cQmVXcQL98StSgpDq0vqIUbSEmqPC1Eu145dBLSzN9d1FIJFhEoeySCkRGP66fmo\\\\\\\/rpM3hjzKzJ\\\\\\\/KMBeQVWYWIPH5l\\\\\\\/a+yNSyncTNbd+0gOCC0wss\\\\\\\/sPcZgjOfs1SdRlX4ubVI+SBOIRBFMl\\\\\\\/fEAlXQtHOU2u9BF87V\\\\\\\/PV5FGkMzOP5y94CVUSLvbbFMoX8wdx37UxUDCt7XYRQNmdeiOlEnGLArboWWhWtGPxpp0TJBVI9s9Q5G+u3P9wCR9aG51C\\\\\\\/xd2jmeYjecjbDDg0mULnNEjpFluincbNpFvmh6QcICkxOIUSDrIlXyEEdRBxocWC1l4SyvJu5l+3S7hBzfLPxPzvE9h8Q9M38zrJUbh\\\\\\\/uOoVVpiBcC+gmBvvOIG2vZ1NxqAnQASRiXYeF09yEl8lX+aTDKbM9vagYQ4CPVSV9h8TxA+gqb\\\\\\\/rw1dw7id9BoPYrNi9RwPGXmgvltiTJ9QlFU\\\\\\\/PaNsrgGY+SD1ChJaSazkpJGnvUpxzf+ENwHUTSlYpxsjysCldjmWv4vFd68SVsDpi\\\\\\\/oupUcO\\\\\\\/fLQ1IRBDfCk4dfavfcySYwlTmgg0worJIWE+ODclvypiu4t9oWEFbM9bbCwiwphm2gdNh2QuZr5wBAxBp1n\\\\\\\/4dQUkei81lKxpgpmm26RVMMv9LE7\\\\\\\/kMkf9MS8nAe24hn6npFPchXVJTmH7W9hIOk1i36YbZw64TH7g91+AhUdzdLHpmf50XCy2K3DGGGkbJdSLZE6e5XWGdmHuUmTFTCaoEBPoPX9e7hnUr2pXpZyH2n\\\\\\\/xY8wv9JOmqjwihlhVdXtlvHKn\\\\\\\/85T5Z5F4vhbze8Cm0a4DTTBjaOHhYqWSJzyoEfvsN9dGH8qIHm2lhMU4ui0jXp3NWnyVOfTJ9VnFNmpvYBSGmUMS54q0tw+CU6FS8U2drHW37HFf2UoxshrM9dCmGD9\\\\\\\/rypr1NMpdwQbJYpyvKxBO31IEQnTdFg9ZmHUpDKrfgWtYXqx6kZlUSrpj5BOoE+xt1bEYDp8CkG+wHnxHO8Sm5QcQmXG6T1Jx4lVMmRJu7jMBQ2tgtfsYXhxyrIkX5M\\\\\\\/iyu8D9KsV76YOFU6FHKgmaOUhx96WjieLxzpBX9D6IErjRVKrdae1esigonS5DFYOQJU0O6nYmIosWsyDNa19dMldVGUKiRXnyicgan+NCSqWTKVjbDY9csjIAEKXT8SSIhxSyKaWJbb\\\\\\\/WGmNC8YTzacygndA5epoC85fn+xTI8UwCi2iodQcNE6HNtn+0CpOdTNWgZ+FYz1pPRrh5B4tDX781TFFW5qBOsN8iJbYkxmWfO14L4slJ5Fco6Hp50yd3UZeCpC4qPh7Uz3lwJu1X93D9F00kXhx1K81OqAsP4uf51\\\\\\\/9fOBZW6nSLOtANzZOJDXQSsYbI\\\\\\\/e64bqDAD0P+EemPcuSHeqrjh4xgGl11LIRCBmpc7nRadUKZ+HJ3rGNeu2YnOrSxcXWUtMdJgFsqALkIAc8+BPFXngu29B\\\\\\\/ZwYDLzo4SWx0WlBhSaL7EWmplTXAIUur2k\\\\\\\/vDhIPlzDuxs1Bww6N6yIj2NUyo404K6JIC5EnHPkkhk9HqvvNztLAd0KpzjUol7o0BBCC\\\\\\\/x+h2GnAOuF3KR5dVthuOraQy91OoxQuCrDaV7zT+qABBKlHYJnrWXzhTT5i41vZaeOaCimW\\\\\\\/g14phJPkLAgybKgb0HNvmgO\\\\\\\/FR19n3VMz45jhMFqhazsZ+x5XjgBl1ExX+ouiaS4fs4e2YZQvQbN6heKAc5fyd25+GH\\\\\\\/SDGR6EcQ97jWJPNR7pENjwUC02zAAPSyRc\\\\\\\/uqQJvktdMSEBszGkAo9e1GrDMx6ygXfhvttGC0Rn\\\\\\\/16d9ah0HrbJ87wkDGTKHOUmfz5zAdop5yriObg7Nfix468+WgdWF7jAFzB5n9Y7sCPkCm1q\\\\\\\/GNI23xVhvVjLdMGRr7\\\\\\\/L0Lzf9+GJTipQ9zLzbt\\\\\\\/LrG7+Z3qgrPioGFdFL17jSJiqXd0uXBFrweVlqxiENI4l9JG9DUieYn6GhGyqBDq0dpjVbjp3DYAeeKhihyV32qh6I2+N8kY1Uq8sNJMjVVoRIyQ5TgSp0QEwnnziCyDMRg2QHZWT5X2zxVoNIK4u0A3wi7bXHrDolhfZ7J4ELsnDW5ihVxeakd\\\\\\\/XwetXXUAwYam3Ty9KCLclqfNbUqoOAQ\\\\\\\/tyyJstDVMsGGX0BXmOngNZjxGXHnAX\\\\\\\/qfXVQmwueWNmdDaZ1Ke1hiTQPFdqPT2Oqd5zOojrR\\\\\\\/B19eTCS8xL8wJwSGysbcGxvQD9fnMHLB4\\\\\\\/Q1UQ7KbZW9LZK8ZvdVcAONduC8tUoOjsGyJmGy8GXb\\\\\\\/CGibeM9CDsAqKivo+Jj+NCIh1O0ukCbbPo0WXD1s1n+727BV+hHoiySORqKA\\\\\\\/kLi99fbxTdw+64EZmK8V4Wg6hSILUkmY3ter67GGVjcBf2Y2dOF5cU\\\\\\\/hpQJXiLA3UMI53S8cIBWD84MW0g9ZLVLbl1cwTxgR4uZAUNili4qepFjiE7CECaHGLqkldeXFng8FGZCkfc4djyVbrP3RWdHikcnjbQF9kJs9TedxK19VpAmRkMIlW\\\\\\\/N0ljcpMSxE5\\\\\\\/EVGI0ngMSZ3jdhlLfNwUxAlCIZk29ld8Bv258F5OXg9y+W7K+ij7Lbbg5MLFCv+s0M\\\\\\\/NOEunIFmhhoychOWZ\\\\\\\/\\\\\\\/jVAAPjesOOLOIJYGpkFnQwA4y37+UKnQRUA7UR4gGQEWiZ2xLIJwrR6JR478iTUVZKcMZGcoJQnwQkSOtt8jOMisDnBu182oylx\\\\\\\/IiNX2LTfQTdNeBbZHAjwtgXA5UTmpFliK4fmCbCMAsBNJuQvmtoxVgj4jLhrroStisK1AiSAwxsDpW+Ebw6zdKv\\\\\\\/T37K+2slyb5\\\\\\\/IKa\\\\\\\/w8LoTEz\\\\\\\/uPnu2QR4QrZu3bbbZHHNrq1qYGQ+Gr71CVraotziFClr7UFg05An619i9thLFqWWWNNI77eTmktSP4HkhM0oxiUfYSKF+4rcs+CYkwLGl8Po\\\\\\\/6Tl4jAhJbQbjI3DU17bMN4\\\\\\\/i1\\\\\\\/+Tp5V3ykO+shC2SjkQSYdTijByqZiHzQDRRHAWOuWonMolODE\\\\\\\/nMq+NUO8MPOANqlekR6xmHtWxYjROzoYC4LEyBmqVJqI0hI7QKsKlOPq9jJ0KVKoay3n10cNykrrahNEnnqsWKUX\\\\\\\/S4HVtPWZJNQySqYElFOpD5zdk7HwtMpj\\\\\\\/au6CbqSg864r2O6J+EqNedJ0LZDEpECS08cwb5whD3a\\\\\\\/JIKCHIkSNdsYz8bqSfw\\\\\\\/INMHpMwvvWlsv+1k0lPPBEc6Yil\\\\\\\/Rqf0aQZV6VqlWiZhJ8iSrtwpnn4Bof2htwxXwYJw7v9rUR31XV2EbOwkDN5mFyMvMs2eR3KtQLDcAIFhnCZnUpC9YrFb+GBlN4xulr10owjLGklPI+QAnMu0u7BqCGzQRV1h7502Fp6X664Ws8zh6kuGnJm47n8q8M8LzEmyu3j4dgNk+Bd5cf2gYdD0oL\\\\\\\/BknmhL\\\\\\\/lPsiQdGCh688GIMRRtQ7A53EvXL1pVKzuqKVW3IxfHexX601thHNBw29i5uUaTS2jBrWQ6LjFcMcuy99cf4HBSSRmTSCFoCplNNJEAqDmaWcT1xnCPBcs6ME4+Kqo83w\\\\\\\/mlO5vDuAYNxVmsQDLUzchJxozIkQre7MgF2cpHCEIRDNq2pAlsxiVjpC\\\\\\\/9+Ki\\\\\\\/G\\\\\\\/OQRLEQ64eBvmaeLIWfJPIAFuIEEO1lZUxEiEFI2orXqhH359uKB8uZvndgQ\\\\\\\/QBU4it0jfCl9ezfV\\\\\\\/mLOkWnGkDHJtkI6M08r0ChzoGor98BoFNX35LlSIITDXMXv+UJ54rMTKa8rsBiTS2tidVh\\\\\\\/7MmfEGjZjPrYhlQemjr9LJZANfQ7CfZ1B2WLxR5yGn\\\\\\\/RXgsARr4FC2DBchBreKQLLzWRknRcFZPJREyT3OhPl5P29I4OhTqCedrrsZKYYdf1+s\\\\\\\/XNYY1xkANUT0UHf7acscTOODUU3URmdpVqREzTw446A1oUW9yF5rrbjHPHHKFB2spQ+s9cieWpILQHmcX3u5dIGdxNO+utTZg5IuLAp6J5i7tyU+FKjx4+fk4RCBxQt+HS12M\\\\\\\/7Qv\\\\\\\/yDx1ZX0SpbqrcBHFKVTfsXSbFEYrM\\\\\\\/im41AlwuwWr6NdLLUTCJTyTeZzeFEeKgKQAx\\\\\\\/YFb9cgChwX72RSZ+Ng2VXHVQE98OkG+S3nU+cNVL7SoVoU\\\\\\\/NXSnoGpOkO9dWDzjllNZVQ1H+uTQ2u\\\\\\\/QJuCfC2CQvfqmId+ixjZzMg5DbinKK7sgbnIk00W9JNnH+bDRMG0GYo49D5eh0ZHBDToVCc7EwOd0yYuzWygtPURHknCEkv3xYQnYNallvKXIgIvggAl85DoRaC7kQRnbyAdoQKb7gyAZRcYTlgBAHTLjQkbomUZg+Whfn9x2W6vz9jxhhfMa9bssrsXXOjr8FQAPXMYicuP22Flxm3C6Yf+WjLFnZCRKd+YJPOuSYnTjJxQZyWakMKnYcYrCo1HogADj845Zeg53o3F1LcgZhC9AAhq0ESLsH\\\\\\\/VcfW+CXSa6yu3Sky1hxiMMHKRZq0iqm8CPiD2guauanE8nPv9TleevgUYzfg8zWcUVE\\\\\\\/o176cp0rdzeDwh52U\\\\\\\/1+yp\\\\\\\/l4Z+oeGyBp8LUi0\\\\\\\/2kiry0gDDiNsUF+L6XI1F+wOYmkEhOXgL4wS+FAMsb6Wh4wbrm5304QqlRHLzxasRdXmHzisltuRqKbXLk4Q7Pde1h9pOQI0ERPM0v1\\\\\\\/gjUc4duquC7if6kx9wEaTbiz14+QTEL9W5VETYuUuuQsgAZodiNT4rfwdgY4ltQmyzMAi0q294ucur65uCJUADZ1xmat6V+kPqlp3JsU6SpZsbg7uN1nkVxyRqGkepDfkuSKNGphr3juKZMHkPxxzjfF44LSYMeAVGE+m9LR+hwlEybSPwOXZ9M1OySClkEWkP0sesOf836TSwr2wCo3Rf9LdPoHKdW+mGIqGSjCEzuuqcMxfDiZ0XJ73nbji87Jbr4BdgJAZRPSVai\\\\\\\/xJjBKeP5rHfGe9\\\\\\\/\\\\\\\/SVNPUv0ximjjz8kFZbc0UL5ShQZE\\\\\\\/pxdNeS5Kf1+3embtQF1Hma3+zmarsqonE37vUJAFzDHbVIe1CJRzAbjp8EWvL9A8gvI\\\\\\\/29pOusxLXcos8tCbG8vjC4ikKYxNTRyR9qpJE04AltwVsxMpGAvbOsqSgbxD+h9\\\\\\\/UfJ9CeVM6KSmlpp74RzFLCEdOGNbT7x\\\\\\\/2Sml\\\\\\\/1b0PNEpKyUXMT3gkF2fA4jZXVMRNNpyXJlvRHExjpPN9ULCX6XEl+e5ZENuI24Q6y9\\\\\\\/yrI7STaScf38MhS5RPBcqdAXruuwiZGgjefecvt+jnrBegF\\\\\\\/N8wwoTl+TSxUWtcMI1+LTUIBO6RXPAlEb2GIZFRq20lBXd6kCjKKyAzD0Gwotd99kzGM4j3ovnvHzaL73RrSO27pGaggs0hKv9vQNph2jGTnqSTjEVIIS3v0TGI\\\\\\\/CUm8F7v2zFYdTyUBa0\\\\\\\/ZLEJEF4WT3JRkp1j0xiKvVnF2rEj7CE+5lqyvcUyPD6hfgkJsd6g1Hxl2a7dy0LHPkYDLlNjAMAwyHDNMO251a94LSabRr8MKdHcHyglDLXVcxctOO4HX9B4HAoBOTtCRXDQXfl6Y5jBZBgPC9n3EXvjZ0EKLmX2PsIahIUh1tgM9Oa4U9qKQqwawXTLUSD6bSAWMKlNFu7dmH2g2ykM++nvAeP+DKoB45z+SupY9Jsf5xgYS2tbwF+DQ17Tjz6NNDSkQYBZC0B8kEKoF3KKohLnKYvs+E0bXDdTxqnKQQkW3Px7BJrfqJ6tFcN\\\\\\\/vNu5P4dgQbStG8l\\\\\\\/a2TV2O3OJz6eBt+KyHV5JfzGT47i7o7eWP+7\\\\\\\/r8msbKds5v2ShI71+5wA1wOrrojx5FFdxXEozfOymfEgEyUfxCmfRzoWm\\\\\\\/ScumM0e3SuLiwHu1lnhmNmNDuHYt5vYtGn6m4oVHI90syoASizWai1YldGQPNA3ySP3sX\\\\\\\/\\\\\\\/ZH8VQqX9RUN9N+RtTjE7Crvtpll21jyrVf\\\\\\\/iJLa6BdXxjHWqckRirC9PzyYX0D2LYb1QwP8e88yLt0KJqtorrweiDGZIMGPC5\\\\\\\/8wvjRkgxXgchKvEkrZOphvpMeDjbflmCYyNriSxafymaZE5i++aI7kczr2bWGvkdj3jTTeeTJs3Gfya0sbZ+cKDygadSTBcuoeVI0yJsbnK0gv1j4n7OJM9LChZMo9KCxkH7KG8OzEEmzNjsBKXlPiEb\\\\\\\/lbwDYJVN0t3+5320poSWu9yuzXXL2G9QODsJ1oJRVWVBh8EBb4gEqGsDYS182NjwdKtlMHxAdyBKOBk6cgj2TxtWRCpxDEAM7+yTvrIaf9KoxHSdRSevOpHs1MVrA7ABFv9KCFtPpfVC9o+80\\\\\\\/9ptg+155oBNWdQ+P7rKpA2EMJ3bEHz1UMrwEg1D2B96EK7oyNA6qlxocFKNummRXg8L\\\\\\\/MmlQ+7oBCEc8tYs3wMnATdAmcCkwjIYVLkY1xvn9fCgfOxSldQPh1GvsQ7l7Z7QEdFL9jjM1Rrvqr24xINVbNSHQYe1BcwEUuND9WTeC+waTxFk3r53dkM7EEXVj41yqa4kpY3PyZYHoCCx2pnM2jKzsZ3V5NQPKJg238D4Toj5GJkkqJi7DfiE02DG\\\\\\\/cPsDDCN4wy6vVCNMxXJzFrA457bg3ofD4JBn6bxtWiGbCji5l91RdqjWJFQj+Y5zoDAIdJGB7gldHR6F2O037Ht6e02zKMFeOpkwYjio0VFKrHfccqhd8bb8ModJkMB+GFs0s7j4c5Gm9ru28j4dYTu6wa\\\\\\\/eDrkMk1Bzpsri\\\\\\\/ebzGsAVQ2FSv+8NjcgkyFKxYr9yWVm9Uu512QsUGhB5b0DK9HLJ1zSN2XFWg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022suV7GoBiBmAkmcg2ZnfI3g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e391b8f3ee76e8481e90d3159b7099420089a5ef1fe250ee057a58f1bdee837b\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022L3jbVNrndFxprcH1yPSXh3BPe0kACiwHWVy5pnrP6FuU8mV4\\\\\\\/uls8kJjDaqB4w8HxNaUbx4oZnf0o\\\\\\\/W+3CmgB7fAw2i8q1VrjkXaMO\\\\\\\/rE\\\\\\\/FWTw54R5\\\\\\\/+lH1YxvvS6nBnJk1LNd4FvqqDgLguRucA6GmMpjsXAO5WdK4jtw\\\\\\\/Ih7x\\\\\\\/QUCkafZOs0X79vTJ3xrjQLIBm69jcH8JPl0OYJ8jME221XO1CUh5fUTk5SVKaJVUnGcRfmT8Z06LnqUnSDjC\\\\\\\/bGPSL2+EV7Sq8gwzI5sjdsWRjy9IEiOwsfS2S5Gm8dLGdGG5+\\\\\\\/q\\\\\\\/k6bi0sa\\\\\\\/BtFOorTr1Sx5HEmac7VV1cYoUT0okTrmv7EIQL6Vd81iBFbvtBX8PzEO14nTTsZRT8oZibOUifRAYyUE+0iG\\\\\\\/JlSGLDJlP1PMHpVIoQahGUTd9SDDjFFek1kGgZsolwAt6DqYfKldCftTX5FNDw6EY8HSUtFY1jz9GjnUSEZ+1CdiD1yuOesnG+PmvaiNaSER2p\\\\\\\/7L9O95p6LAr2gy\\\\\\\/tHHHY0iV69EI2jGM+s\\\\\\\/T8nqWqYHFRY1eIeFsaaBgGVypWt1vN1BdjOtei4pLB3JkA+MEnuMGnzrr0xC3epRIh+R0BqN\\\\\\\/hePuBfGyCWeo2vfoADQvedNSzWEIZaN0R0PacjZJ0NLVKyH1NoBdN6AiPHSFhLwtVEZr7asVD3\\\\\\\/5hHTK2M+qnuqvwp8OwTj7Wil21TU7Rrio8B2ZuHfYZjWg9s+CNe3anvnd3NajEjq3Y6BojrBdDfGi23sd2uwuby9Qm2g8UF2anTTl6fhUOSoYZKtDn+G+pUO\\\\\\\/kHKr0qHbLsR9+mnKTi7MTt87Uo8AEnwQEWf\\\\\\\/GxJt74aWi5clFXO1FhkD+jfrhSohvVD09DjWW5HOAo3WQ35lhKY2+j6qIvAe+gI9amSRLmu7x7Nx54WY8zU8qtCSBjAtJ57P\\\\\\\/VJ0pwgDYRthJlQq5e4cueZlGfqREq9BAdvdLBiMDcDcipugX5MmBIdfYBQL7UYLzNSdjpiE1UI\\\\\\\/H7qBMKTMkL1wJ5Cs4SpkFYI5WAvV+y4dk6fnBSAmw\\\\\\\/5kwIYufdNdr8r\\\\\\\/5aFIKwwkr7YWAHnBXzlc8w2Px5QxxgVthjP5W86IxDFRRl+fBL0rWNRfpzCG+IzceSTj8kv3n6lB7hNpTDJNI9smIKFlsWNMSSA\\\\\\\/N+IcTID\\\\\\\/k4s27TGeLFk5HQfK4EWt+JahLSXK2hrjpZlv4l\\\\\\\/MYozSdbva3GSw2gQH6Nv\\\\\\\/YO6SoIQ9chtPw2+QkSvYLhmWMrkXu\\\\\\\/8ZFchCD5GkRPboBGC+l+Y\\\\\\\/Lph\\\\\\\/1Z0sttqxDRiyNK6a+7M4UXE11koEbR+HXi89tULJGoPsiA2POOlBMmE2jJdFCoKNIHL2DhWkGBlyjH7+xhGV1UUlQl06FoQWcACAu22\\\\\\\/PqnZkER8SV9lyIARGaldURYimBbdG9NHq3hD0M0vfRuQadoBtNHh7WjXc+k0DOkT5M90nfFtZ+MzWUHCkTBOdLYzs1oeITtnw1hdz8GfCStw9l0CTGMv9IHDTfAYSXQd20CRWybOh\\\\\\\/BvroJwyUEnWrNtjkz9dzuWa5GQDl7wFuRNe2kD3o7DW9hrvEvhccxP0ewdHmu6ywlk73kY026sciImQNLAcu6YAY4+1a3RjVLCzNjmt11to5TEXYZTfks8QGL2yS7BvvY8Qk4obEw3m7vKs8\\\\\\\/IwhA2grwN58XTaBp17EZEN+sDQ5NVLG9U1TX6baAVnPXmaB0or4tI+A5s8z\\\\\\\/DUatd2xhXBHsYToWHSwHbwOR\\\\\\\/iaEhM9KgYZ3yXaEV2F5YyzDb7MJNiVcl85c0C120Yb717986K4gv11aqwrMC4dwn1uSWCOimQumu03YVccGuzyNxsKo7RYWqcKXY2gHb4gz5LjOpnyAgeHp0hZtwB5VSFcIXYXW87DBsVhkWlJPAKwiRVzo8j5oxh4UaKRiPk1aZhfmCCiyj3NFgEzUgkh204GxuoRkoX8UT9niuLikAt9Vnst2iTDQvbwk3Lg==\\\u0022,\\\u0022iv\\\u0022:\\\u00223A6pAiyVc\\\\\\\/pdbjKVcXV26g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221acde66f1e7964874da4d157afe7297ff2b2e374ede1e825c80d662006d1e149\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-493754 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/8/2657" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-xBm4zEJnnppEUym3Z6xwBA0GV2Hwds59QFl5Cut_SM0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/8/2657" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0rOzTqHMT2pgi9S-m2TStEO-oxyKmM2D671a00FqJUc" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/8/2647" title="De Novo Assembly and Annotation of the Larval Transcriptome of Two Spadefoot Toads Widely Divergent in Developmental Rate" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/8/2667" title="Coordinated Roles of the Putative Ceramide-Conjugation Protein, Cwh43, and a Mn2+-Transporting, P-Type ATPase, Pmr1, in Fission Yeast" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493754" id="node493754" data-pisa="ggg;9/8/2657" data-pisa-master="ggg;g3.119.400301" data-apath="/ggg/9/8/2657.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Fragmentation of Pooled PCR Products for Highly Multiplexed TILLING</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Andrea Tramontano</span>, <span class="highwire-citation-author" data-delta="1">Luka Jarc</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-9249-4700" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Joanna Jankowicz-Cieslak</span>, <span class="highwire-citation-author" data-delta="3">Bernhard J. Hofinger</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-0235-7607" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Katarzyna Gajek</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-4661-5823" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Miriam Szurman-Zubrzycka</span>, <span class="highwire-citation-author" data-delta="6">Iwona Szarejko</span>, <span class="highwire-citation-author" data-delta="7">Ivan Ingelbrecht</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0002-1300-8285" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bradley J. Till</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2657-2666;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400301 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493754" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andrea Tramontano </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Laboratory, Joint FAO/IAEA Division of Nuclear Techniques in Food and Agriculture, IAEA Laboratories Seibersdorf, International Atomic Energy Agency, Vienna International Centre, PO Box 100, A-1400 Vienna, Austria</div><div class='author-affiliation'><span class='nlm-sup'>1</span>These authors contributed equally to this work.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrea%2BTramontano%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tramontano%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrea%2BTramontano%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Luka Jarc </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Laboratory, Joint FAO/IAEA Division of Nuclear Techniques in Food and Agriculture, IAEA Laboratories Seibersdorf, International Atomic Energy Agency, Vienna International Centre, PO Box 100, A-1400 Vienna, Austria</div><div class='author-affiliation'><span class='nlm-sup'>1</span>These authors contributed equally to this work.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luka%2BJarc%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jarc%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuka%2BJarc%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Joanna Jankowicz-Cieslak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Laboratory, Joint FAO/IAEA Division of Nuclear Techniques in Food and Agriculture, IAEA Laboratories Seibersdorf, International Atomic Energy Agency, Vienna International Centre, PO Box 100, A-1400 Vienna, Austria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joanna%2BJankowicz-Cieslak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jankowicz-Cieslak%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoanna%2BJankowicz-Cieslak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9249-4700" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Joanna Jankowicz-Cieslak</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Bernhard J. Hofinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Laboratory, Joint FAO/IAEA Division of Nuclear Techniques in Food and Agriculture, IAEA Laboratories Seibersdorf, International Atomic Energy Agency, Vienna International Centre, PO Box 100, A-1400 Vienna, Austria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bernhard%2BJ.%2BHofinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hofinger%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABernhard%2BJ.%2BHofinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Katarzyna Gajek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Faculty of Biology and Environmental Protection, University of Silesia, Jagiellonska 28, 40-032, Katowice, Poland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katarzyna%2BGajek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gajek%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKatarzyna%2BGajek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0235-7607" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Katarzyna Gajek</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Miriam Szurman-Zubrzycka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Faculty of Biology and Environmental Protection, University of Silesia, Jagiellonska 28, 40-032, Katowice, Poland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miriam%2BSzurman-Zubrzycka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Szurman-Zubrzycka%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMiriam%2BSzurman-Zubrzycka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4661-5823" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Miriam Szurman-Zubrzycka</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Iwona Szarejko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Faculty of Biology and Environmental Protection, University of Silesia, Jagiellonska 28, 40-032, Katowice, Poland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Iwona%2BSzarejko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Szarejko%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIwona%2BSzarejko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Ivan Ingelbrecht </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Laboratory, Joint FAO/IAEA Division of Nuclear Techniques in Food and Agriculture, IAEA Laboratories Seibersdorf, International Atomic Energy Agency, Vienna International Centre, PO Box 100, A-1400 Vienna, Austria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ivan%2BIngelbrecht%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ingelbrecht%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIvan%2BIngelbrecht%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Bradley J. Till </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Laboratory, Joint FAO/IAEA Division of Nuclear Techniques in Food and Agriculture, IAEA Laboratories Seibersdorf, International Atomic Energy Agency, Vienna International Centre, PO Box 100, A-1400 Vienna, Austria</div><div class='author-affiliation'><span class='nlm-sup'>2</span>Present address: Centro de Genómica Nutricional Agroacuícola, Las Heras #350, Temuco, Chile.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bradley%2BJ.%2BTill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Till%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABradley%2BJ.%2BTill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1300-8285" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bradley J. Till</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:brad.till@cgna.cl" class="" data-icon-position="" data-hide-link-title="0">brad.till@cgna.cl</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/8/2657" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:493754" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:493754/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/8/2657.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:493754" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:493754/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/8/2657.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:493754" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:493754/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/8/2657.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:493754" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:493754/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Improvements to massively parallel sequencing have allowed the routine recovery of natural and induced sequence variants. A broad range of biological disciplines have benefited from this, ranging from plant breeding to cancer research. The need for high sequence coverage to accurately recover single nucleotide variants and small insertions and deletions limits the applicability of whole genome approaches. This is especially true in organisms with a large genome size or for applications requiring the screening of thousands of individuals, such as the reverse-genetic technique known as TILLING. Using PCR to target and sequence chosen genomic regions provides an attractive alternative as the vast reduction in interrogated bases means that sample size can be dramatically increased through amplicon multiplexing and multi-dimensional sample pooling while maintaining suitable coverage for recovery of small mutations. Direct sequencing of PCR products is limited, however, due to limitations in read lengths of many next generation sequencers. In the present study we show the optimization and use of ultrasonication for the simultaneous fragmentation of multiplexed PCR amplicons for TILLING highly pooled samples. Sequencing performance was evaluated in a total of 32 pooled PCR products produced from 4096 chemically mutagenized <em>Hordeum vulgare</em> DNAs pooled in three dimensions. Evaluation of read coverage and base quality across amplicons suggests this approach is suitable for high-throughput TILLING and other applications employing highly pooled complex sampling schemes. Induced mutations previously identified in a traditional TILLING screen were recovered in this dataset further supporting the efficacy of the approach.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/tilling-sequencing" class="hw-term hw-article-keyword hw-article-keyword-tilling-by-sequencing" rel="nofollow">TILLING by Sequencing</a></li><li class="kwd"><a href="/keyword/dna-shearing" class="hw-term hw-article-keyword hw-article-keyword-dna-shearing" rel="nofollow">DNA shearing</a></li><li class="kwd"><a href="/keyword/sonication" class="hw-term hw-article-keyword hw-article-keyword-sonication" rel="nofollow">sonication</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/hordeum-vulgare" class="hw-term hw-article-keyword hw-article-keyword-hordeum-vulgare" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Hordeum vulgare</em></a></li><li class="kwd"><a href="/keyword/barley" class="hw-term hw-article-keyword hw-article-keyword-barley" rel="nofollow">barley</a></li></ul><p id="p-3">Next generation sequencing (NGS) techniques have had a profound impact on biological research. While technologies continue to advance, whole genome approaches remain costly for projects involving the analysis of species with large genomes or those involving the interrogation of many individuals. A variety of reduced representation genome sequencing approaches have been described to circumvent this issue (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Hirsch <em>et al.</em> 2014</a>). One powerful approach to evaluate sequence variation in targeted regions is by sequencing PCR amplicons. Amplicon sequencing has been applied for the discovery and characterization of both natural and induced mutations in plant and animal populations (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Rigola <em>et al.</em> 2009</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Tsai <em>et al.</em> 2011</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Pan <em>et al.</em> 2015</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Duitama <em>et al.</em> 2017</a>). For example, sequencing has been used to increase throughput of mutation discovery for the reverse-genetics technique known as TILLING (Targeting Induced Local Lesions IN Genomes)(<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">McCallum <em>et al.</em> 2000</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Till <em>et al.</em> 2018</a>). For TILLING, mutant populations are typically created using mutagens such as ethyl methanesulfonate (EMS) or a combination of sodium azide and N-Nitroso-N-methylurea (MNU) that induce primarily single nucleotide variants (SNV). While the frequency of mutations varies between species and ploidy level, mutation densities in diploid species have been reported to range between 1 mutation per 150,000 and 1 mutation per 700,000 base pairs (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Jankowicz-Cieslak and Till 2015</a>). TILLING screens typically aim to recover multiple mutations from a single target to increase the chances of recovering deleterious alleles. Therefore, a typical TILLING assay involves the screening of 3000 or more mutant individuals. To increase throughput, genomic DNAs are pooled together prior to mutation screening. The application of new sequencing technologies to improve TILLING throughput has been termed TILLING by Sequencing (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Tsai <em>et al.</em> 2011</a>). In addition to pooling genomic DNA, multiple amplicons are produced from each gDNA pool and combined prior to massively parallel sequencing to further improve throughput. The quantitative nature of NGS methods means that rare induced and natural SNV mutations can be effectively recovered from pools of over 200 individuals (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Pan <em>et al.</em> 2015</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Duitama <em>et al.</em> 2017</a>). The most common sequencing platform used for TILLING by Sequencing is short read sequencing-by-synthesis from the company Illumina (<a id="xref-ref-37-3" class="xref-bibr" href="#ref-37">Tsai <em>et al.</em> 2011</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Guo <em>et al.</em> 2015</a>; <a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Pan <em>et al.</em> 2015</a>). Similar approaches have been used in population genetics studies to identify rare alleles in large germplasm collections (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Duitama <em>et al.</em> 2017</a>).</p><p id="p-4">Direct sequencing of PCR amplicons is potentially advantageous in that an even read coverage across the target can theoretically be achieved, thus providing consistent recovery of sequence variants. This approach is disadvantageous, however, due to the fact that the length of sequenced amplicons is limited to the maximum read lengths of the sequencer used (<em>e.g.</em>, 500-600 bp for the Illumina MiSeq)(<a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Pan <em>et al.</em> 2015</a>; <a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Duitama <em>et al.</em> 2017</a>). Thus, multiple amplicons are required to screen an entire gene, necessitating extra liquid handling steps and also additional work in adjusting PCR amplicon concentrations to a similar level prior to sequencing. An alternative approach is to produce longer PCR products and to fragment them prior to library construction. Several approaches for fragmentation have been described such as nebulization, enzymatic cleavage, and ultrasonication (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Head <em>et al.</em> 2014</a>). Most approaches are developed for the fragmentation of genomic DNA samples that contain high molecular weight molecules. We describe here the optimization of ultrasonication for PCR products amplified from the genomes of <em>Coffea arabica</em> (Arabica coffee) and <em>Hordeum vulgare</em> (barley) ranging between 670 and 1513 bp. We further show suitable coverage can be achieved in a pool of 32 distinct PCR products generated from PCR amplification of pools of 256 genomic DNA samples prepared from chemically mutagenized barley. This suggests that the approach can be easily adapted for pooled amplicon sequencing in different species.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Genomic DNA and PCR</h3><p id="p-5">Genomic DNA from <em>Coffea arabica</em> was provided by Margit Laimer. PCR primers were designed using the Primer3 program with parameters previously developed for TILLING assays(<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Till <em>et al.</em> 2006b</a>). Primer sequences were TCGATTCGATTCGTTGACACCCCTA and TGGATGATGGATGGGAATGTGGTTC. PCR was performed as previously described (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Till <em>et al.</em> 2015</a>). Amplification of a single PCR product was assayed using agarose gel electrophoresis and confirmed via capillary electrophoresis (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2657/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of different fragmentation parameters of a single PCR amplicon. Panel A shows the non-fragmented control of the single 964 bp PCR product as assayed by capillary electrophoresis. Panel B shows the results of test parameter 1 in Table 1. This is an example of incomplete fragmentation as the original 964 bp PCR product can be observed at a high concentration. Panel C and D are technical repeats showing results of fragmentation using test #13 parameters. The y axis in all graphs shows relative abundance of product in relative fluorescence units (RFU). The x axis shows molecular weight in base pairs. LM and UM are the lower and upper marker, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-6747667" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of different fragmentation parameters of a single PCR amplicon. Panel A shows the non-fragmented control of the single 964 bp PCR product as assayed by capillary electrophoresis. Panel B shows the results of test parameter 1 in Table 1. This is an example of incomplete fragmentation as the original 964 bp PCR product can be observed at a high concentration. Panel C and D are technical repeats showing results of fragmentation using test #13 parameters. The y axis in all graphs shows relative abundance of product in relative fluorescence units (RFU). The x axis shows molecular weight in base pairs. LM and UM are the lower and upper marker, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2657/F1.medium.gif" width="440" height="271"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/8/2657/F1.medium.gif" width="440" height="271"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2657/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2657/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493811" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-6" class="first-child">Comparison of different fragmentation parameters of a single PCR amplicon. Panel A shows the non-fragmented control of the single 964 bp PCR product as assayed by capillary electrophoresis. Panel B shows the results of test parameter 1 in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. This is an example of incomplete fragmentation as the original 964 bp PCR product can be observed at a high concentration. Panel C and D are technical repeats showing results of fragmentation using test #13 parameters. The y axis in all graphs shows relative abundance of product in relative fluorescence units (RFU). The x axis shows molecular weight in base pairs. LM and UM are the lower and upper marker, respectively.</p><div class="sb-div caption-clear"></div></div></div><p id="p-7">Genomic DNA from chemically mutagenized barley was prepared using a modified CTAB protocol (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Szarejko <em>et al.</em> 2017</a>). DNA samples were adjusted to a similar concentration and pooled. Two hundred and fifty-six samples were combined in each pool. Forty-eight unique pools were prepared. Primers were designed to target regions of the <em>H</em>. <em>vulgare</em> genome as previously described (Table S1)(<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Slota <em>et al.</em> 2017</a>). PCR primers amplify part of a coding sequence and thus target names are assigned for each amplicon that is unqiue from gene annotations. PCR was performed on pooled genomic DNA samples using 32 unique primer pairs. Individual PCR reactions were performed for each primer pair. The concentration of each PCR product was estimated by running products on a 96-well eGel system with lambda DNAs standard of known concentrations (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Huynh <em>et al.</em> 2017</a>). Average concentration was estimated visually and all samples adjusted to a final concentration of 200 ng in TE buffer. PCR products were then pooled together. DNA sequences of each resulting amplicon were prepared from the barley reference genome and combined into one multifasta file for downstream bioinformatic analysis (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Gupta <em>et al.</em> 2017</a>). GC content for each amplicon was extracted from the multifasta file using the Emboss infoseq tool (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Carver and Bleasby 2003</a>). Average GC content for barley was calculated from the coding sequences from the International Barley Genome Sequencing Consortium version 2 assembly (IBSC v2, INSDC Assembly database version 95.3) (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">International Barley Genome Sequencing Consortium <em>et al.</em> 2012</a>).</p><p id="p-8">GC percentage for each sequence was extracted using the Emboss infoseq tool. Minimum and maximum GC percentage was extracted using the following command: awk ′BEGIN{first = 1;} {if (first) {max = min = $3; first = 0; next;} if (max &lt; $3) max=$3; if (min &gt; $3) min=$3;} END {print min, max}′. Average GC was calculated using the following command: awk ′{sum = sum+$3 ; sumX2+=(($3)^2)} END { printf “Average: %f. Standard Deviation: %f \n”, sum/NR, sqrt(sumX2/(NR) - ((sum/NR)^2))}′ .</p></div><div id="sec-3" class="subsection"><h3>Sonication of PCR products</h3><p id="p-9">Sonication of PCR products was performed using a Covaris M220 ultrasonicator with microTUBE AFA bead split tubes (coffee amplicons) or microTUBE AFA FiberPre-slit (barley amplicons) in 60 μl volumes except where indicated with parameters adjusted according to <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Tables 1</a>, S2 and S3. Fragmentation of PCR products was assayed using a Fragment Analyzer with the low sensitivity 1kb separation matrix with 30 cm capillaries (cat #DNF935). Analysis of data were performed using the <em>PROSize</em> Data Analysis Software. Performance of sonication of pooled PCR products was independently tested in 48 pools. Pooling of genomic DNA for each of the 48 pools was done such that each individual genomic DNA is represented in triplicate in the experiment (in three different pools). A total of 4096 DNAs from unique mutant lines are represented in the 48 pools.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493937/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493937/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Test parameters for sonication of a 964 bp PCR product</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Sequencing and data analysis</h3><p id="p-10">Quantified PCR products were normalized to approximately 200ng and pooled together according to a three-dimensional pooling scheme whereby samples were arrayed in a 16 × 16 × 16 grid for pooling (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Gupta <em>et al.</em> 2017</a>). A total of 48 pools were created for library preparation. Sequencing libraries were prepared using TruSeq Nano DNA HT Library Prep Kit (Illumina, Catalogue # FC-121-9010DOC) with 200 ng of starting pooled PCR products according to manufacturer’s recommendations. Libraries were quantified using a Qubit fluorimeter, concentrations normalized to a common molarity and pooled as previously described (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Datta <em>et al.</em> 2018</a>). Sequencing was performed on an Illumina MiSeq using 2x300 PE chemistry according to manufacturer’s protocol. Reads were mapped to amplicon sequences during the sequencing run using BWA with the MiSeq BWA default settings (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Li and Durbin 2009</a>). Sequencing coverage and mapping quality were calculated using Qualimap version v.2.2.1 with default settings except that the number of windows was set to amplicon length to retrieve per base metrics used in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Okonechnikov <em>et al.</em> 2016</a>). Base qualities were prepared using pysamstats and the baseq feature (<a href="https://github.com/alimanfoo/pysamstats">https://github.com/alimanfoo/pysamstats</a>). Mean coverage data per amplicon for all 48 pools was extracted from the Qualimap output. Principle component analysis of all 48 pools was produced using the multi-sample BAM QC feature of Qualimap. Correlation coefficients were calculated using the LibreOffice CORREL function. Coverage data used to calculate mean coverage for tomato amplicon sequencing comes from <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Gupta <em>et al.</em> (2017)</a>. Coverage data used to calculate mean coverage for cassava sequencing was extracted using nucleotide number binning rather than amplicon name as data were mapped to a contiguous sequence rather than individual amplicons (<a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Duitama <em>et al.</em> 2017</a>). Data for barley, tomato and cassava were collected on the same MiSeq instrument using 2x300 PE sequencing.</p><p id="p-11">Screening of data for previously identified SNP mutations was performed using the CAMBa pipeline for three dimensionally pooled samples as previously described (<a id="xref-ref-37-4" class="xref-bibr" href="#ref-37">Tsai <em>et al.</em> 2011</a>; <a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Gupta <em>et al.</em> 2017</a>). The HaplotypeCaller tool from GATK 4.0.10.0 was used as a second method to call SNPs (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">McKenna <em>et al.</em> 2010</a>). Processing was carried out in parallel using the following command: ls *.bam | parallel java -jar gatk-package-4.0.10.0-local.jar -Xmx32g HaplotypeCaller -I {} -R reference.fa -O {.}.vcf–sample-ploidy N, where sample ploidy N was set to either 600 or 2000 in different analyses (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Tange 2011</a>). Resulting VCF files were first filtered to remove common variants appear above 5% allele frequency in pools (bcfftools view -i ‘AF &lt; 0.05’). True mutations should be present in only one sample in a pool and therefore the expected frequency is 1/256 for homozygous mutations and 0.5/256 for heterozygous mutations. The 16 VCF files from a single dimension pool where then grouped together for further analysis. True induced mutations should appear only one time in each dimension. Text files containing variant positions were created, and variants unique to a single file were extracted using awk ′END {for (R in r) {split(r[R], t, SUBSEP) if (!t[1]) print t[3], t[2]}}{k = $1 SUBSEP $2 SUBSEP $3r[k] = c[k]++ SUBSEP FILENAME SUBSEP $0 }′ *.txt &gt; output. This resulted in three files containing unique variant calls for each pooling dimension. True mutations should be present one time in each dimension and so should be represented in each of the three lists. Matching values fitting these criteria were extracted using grep -wFf. Original VCF files were evaluated in cases where positive control mutations were not identified using this method. This was how the mutation in VDE_2_27 in pool 14 was recovered.</p></div><div id="sec-5" class="subsection"><h3>Data availability</h3><p id="p-12">The raw datasets analyzed during the current study are available in the Sequence Read Archive, Accession PRJNA422048 at <a href="https://www.ncbi.nlm.nih.gov/bioproject/422048">https://www.ncbi.nlm.nih.gov/bioproject/422048</a>. M<sub>3</sub> seed from positive control mutants is available upon request while supplies last. Table S1 contains target names, primer sequences, amplicon lengths and percent GC of each amplicon. Table S2 contains pre-set parameters of sonication of genomic DNA. Table S3 contains additional tested sonication parameters. Table S4 contains sequencing statistics for mutant pool 10. Table S5 contains sequencing statistics for mutant pool 11. Table S6 contains sequencing statistics for mutant pool 15. Table S7 contains sequencing statistics for mutant pool 19. Table S8 contains sequencing statistics for mutant pool 20. Table S9 contains sequencing statistics for mutant pool 37. Table S10 contains sequencing statistics for mutant pool 41. Table S11 contains summary sequencing statistics for 48 barley pools. Table S12 contains coverage data and calculations for 32 barley amplicons in 48 pools. Table S13 contains coverage data from a tomato TILLING by Sequencing experiment using non-fragmented amplicons. Table S14 contains coverage data from cassava sequencing using non-fragmented amplicons. Table S15 contains frequencies of mutations called with the CAMBa and GATK HaplotypeCaller tools. Table S16 contains cost estimations for experiments using fragmented and non-fragmented pooled amplicons. Figure S1 shows Fragment Analyzer profiles of a single amplicon subjected to different default sonication settings. Figure S2 shows Fragment Analyzer profiles of a single amplicon subjected to different sonication parameters. Figure S3 shows Fragment Analyzer profiles of a single amplicon subjected to additional sonication parameters. Figure S4 shows Fragment Analyzer profiles of pooled amplicons before and after sonication. Figure S5 shows read coverage, base coverage and mapping quality for tomato amplicons not subjected to sonication. Figure S6 shows insert sizes for 48 barley sequencing pools. Figure S7 shows principle component analysis of sequencing statistics for 48 barley pools. Figure S8 contains a graph of mean coverage and amplicon size in 48 barley pools. Figure S9 contains a graph of mean coverage and percentage of GC in amplicons in 48 barley pools. Figure S10 contains a graph of percentage of GC and amplicon size in base pairs. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8052821">https://doi.org/10.25387/g3.8052821</a>.</p></div></div><div class="section results" id="sec-6"><h2 class="">Results</h2><div id="sec-7" class="subsection"><h3>Evaluation of sonication parameters established for genomic DNA on PCR products</h3><p id="p-13">A PCR product of 964 base pairs (bp) amplified from <em>Coffea arabica</em> genomic DNA was created for initial experiments to optimize DNA fragmentation. The aim of the work was to identify conditions resulting in a fragmented PCR product between 350 and 500 bp using a Covaris focused-ultrasonicator. The manufacturer’s parameters established for genomic DNA shearing were initially tested (Table S2). Fragment analysis showed limited fragmentation and retention of a high concentration of 964 bp product at all pre-defined parameters (Figure S1).</p></div><div id="sec-8" class="subsection"><h3>Optimization of parameters for sonication of a single PCR amplicon</h3><p id="p-14">Based on the results from the default parameters for genomic DNA we hypothesized that halving the power, duty factor and/or cycles/burst may improve fragmentation of lower molecular weight PCR products. Fourteen distinct sonication parameter combinations were chosen and evaluated (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Test numbers 1 through 3 were designed to assess the effect of the number of cycles/burst on the fragment size and distribution. Tests 4 through 6 were designed to assess the effect of the duty factor on the fragment size and distribution. Tests 7 through 9 were designed to assess the effect of the peak power on the fragment size and distribution. Tests 10 through 12 were designed to assess the effect of the duration of the sonication on the fragment size and distribution, and tests 13 and 14 were designed to test the combination of high duty factor with an average number of cycles/burst and very low duty factor with the highest cycles/burst respectively.</p><p id="p-15">The extent of PCR fragmentation was evaluated using a capillary Fragment Analyzer (FA), (Figure S2). Relative concentration of DNA fragments was evaluated owing to the fact that the absolute concentration varies according to amount of input PCR product used in the assay. Based on this analysis, parameters of test # 13 were chosen to provide the best fragmentation as they produced a broad distribution of fragments with a peak of 390 bp, which was close to 350 bp, one of two sizes recommended for Illumina library preparation (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). Additional modifications, whereby average power (Peak Power and Duty Factor) were held constant while time, cycles/ burst and volume were modified, showed no substantial improvement (Table S3, Figure S3).</p></div><div id="sec-9" class="subsection"><h3>Sonication of complex pools of PCR amplicons</h3><p id="p-16">Having established optimal conditions for fragmentation of a single amplicon, the parameters were next tested in a TILLING by Sequencing experiment containing 32 amplicons ranging between 670-1513 base pairs. The GC base percentage in the amplicons ranged from 35.16 to 68.66 with a median of 50.28 (Table S1). For comparison, GC values of the entire coding sequence assembly of the barley genome build 2 was calculated as having a minimum GC percentage of 22.86, maximum of 86.96 and median of 52.48 (see methods). The PCR primers producing amplicons tested in this study had been previously validated and used in traditional TILLING assays employing gel-based cleavage assays using an eightfold genomic DNA pooling strategy (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Stolarek <em>et al.</em> 2015a</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">2015b</a>; <a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Szarejko <em>et al.</em> 2017</a>). Thus, the amplicon set represents the complexity of a TILLING project, and also allows testing of the feasibility of applying sonication-based fragmentation and next generation sequencing to an already existing TILLING platform. Fragmentation was performed on 48 pools followed by preparation of Illumina libraries (Figure S4). Fragmentation profiles of the prepared libraries showed more complex patterns with average sizes of approximately 650 bp, that includes the size of the target DNA (known as the insert) plus the ligated adapter sequences. The presence of higher concentration peaks at specific molecular weights is also observed (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>).</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2657/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Fragmentation profiles of Illumina sequencing libraries prepared from 32 pooled amplicons produced by PCR amplification of genomic DNA from 256 pooled barley samples. PCR products from forty-eight genomic DNA pools were subjected to sonication using test parameter 13 in Table 1. Fragmentation profiles of two different libraries are shown (top and bottom panel)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-6747667" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Fragmentation profiles of Illumina sequencing libraries prepared from 32 pooled amplicons produced by PCR amplification of genomic DNA from 256 pooled barley samples. PCR products from forty-eight genomic DNA pools were subjected to sonication using test parameter 13 in Table 1. Fragmentation profiles of two different libraries are shown (top and bottom panel).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2657/F2.medium.gif" width="437" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/8/2657/F2.medium.gif" width="437" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2657/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2657/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493996" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-17" class="first-child">Fragmentation profiles of Illumina sequencing libraries prepared from 32 pooled amplicons produced by PCR amplification of genomic DNA from 256 pooled barley samples. PCR products from forty-eight genomic DNA pools were subjected to sonication using test parameter 13 in <a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>. Fragmentation profiles of two different libraries are shown (top and bottom panel).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-10" class="subsection"><h3>Sequencing libraries prepared from pooled fragmented amplicons</h3><p id="p-18">To evaluate the effect of fragmentation on sequencing coverage and quality, the Illumina libraries were subjected to 2x300 PE sequencing on a MiSeq instrument. Sequencing coverage profiles, per base mapping quality, and per base sequence quality of four unique amplicons are shown (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>, Tables S4, S5, S6, S7, S8, S9, S10). Similar data were graphed for two pools and two amplicons from a previously published project where TILLING screening was performed by direct sequencing (without fragmentation) of short amplicons derived from PCR from pools of 64 tomato genomic DNAs (<a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">Gupta <em>et al.</em> 2017</a>) (Figure S5). The minimum insert size in all 48 barley pools was 259 bp, maximum was 386 bp, and median was 348.23 bp (Table S11). Pools show a distribution of sizes, with accumulation of higher concentration of DNA at specific molecular weights. This is pronounced in the most highly concentrated pool number 24 (Figure S6). At the pool level, mean coverage ranged from 1492 to 50030 with a median of 6365. PCA analysis showed pool 24 to be the outlier with coverage of 50030 (Figure S7). Percent GC ranged from 47.64 to 54.33 and mapping quality from 59.06 to 59.76 (Table S11).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2657/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Read coverage, base quality and mapping quality from 2x300PE sequencing of targets EXPB4 (1350 bp) (A), RTH3 (1028 bp) (B), PRT1 (979 bp) (C) and ALS3_1 (670 bp) (D). The number of aligned reads (coverage) at each position in the amplicon is marked in blue. The average base quality per position is marked in yellow and mapping quality in red. Coverage is even across the majority of each amplicon with slight increases in coverage and concomitant decreases in quality score toward the end of each amplicon. Shorter amplicons have longer regions of increased coverage at the ends." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-6747667" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Read coverage, base quality and mapping quality from 2x300PE sequencing of targets EXPB4 (1350 bp) (A), RTH3 (1028 bp) (B), PRT1 (979 bp) (C) and ALS3_1 (670 bp) (D). The number of aligned reads (coverage) at each position in the amplicon is marked in blue. The average base quality per position is marked in yellow and mapping quality in red. Coverage is even across the majority of each amplicon with slight increases in coverage and concomitant decreases in quality score toward the end of each amplicon. Shorter amplicons have longer regions of increased coverage at the ends.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2657/F3.medium.gif" width="397" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/8/2657/F3.medium.gif" width="397" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2657/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2657/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493892" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-19" class="first-child">Read coverage, base quality and mapping quality from 2x300PE sequencing of targets EXPB4 (1350 bp) (A), RTH3 (1028 bp) (B), PRT1 (979 bp) (C) and ALS3_1 (670 bp) (D). The number of aligned reads (coverage) at each position in the amplicon is marked in blue. The average base quality per position is marked in yellow and mapping quality in red. Coverage is even across the majority of each amplicon with slight increases in coverage and concomitant decreases in quality score toward the end of each amplicon. Shorter amplicons have longer regions of increased coverage at the ends.</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">To compare sonicated PCR products to direct sequencing of PCR products, average coverage values for each amplicon in each of the 48 barley pools was tabulated (Table S12). Data were also tabulated from previously published tomato and cassava experiments that employed direct sequencing of PCR products without fragmentation (Table S13, S14) (<a id="xref-ref-5-6" class="xref-bibr" href="#ref-5">Duitama <em>et al.</em> 2017</a>; <a id="xref-ref-9-6" class="xref-bibr" href="#ref-9">Gupta <em>et al.</em> 2017</a>). In barley, 1536 coverage values (32 amplicons × 48 pools) were evaluated. The minimum mean coverage was observed in the 1153 bp amplicon named FC1_11 (value = 2093). Maximum coverage was in the 778 bp BAK1_1 (134456), with a mean coverage of 6621 for the total collection of amplicons. A weak negative correlation was observed between amplicon size and mean coverage (<em>r</em> = -0.38), and between percentage of GC in the amplicon and mean coverage (<em>r</em> = -0.41) (Figures S8, S9 and Table S12). A weak positive correlation was observed between amplicon size and percentage of GC (<em>r</em> = 0.25) (Figure S10 and Table S12). To estimate failures, data points were counted where coverage fell below 5% of the mean coverage for the tested amplicon. This detects genomic DNA pools where a single amplicon produces much less coverage than the same amplicon in other DNA pools. Using this criterion, 2.08% of the experiment failed. This is in comparison to 0.5% for tomato (n = 2408) and 1.19% for cassava (n = 2538).</p><p id="p-21">To evaluate the suitability of the experiment to recover induced mutations, the number of times a mutant allele would be sequenced was calculated. In the pooling scheme tested, a heterozygous mutation should be present at a frequency of 0.00195 (1/(256*2)). A coverage of 512 is therefore needed to identify the heterozygous mutation 1 time in a pool. Using this, the percentage of the assay where heterozygous mutations are expected at a specific coverage was calculated. One hundred percent of the experiment produced data for 1x coverage. This reduced to 96.74% for 2x coverage, 88.15% for 4x and 62.30% for 8x coverage (Table S12). Homozygous mutations are twice the concentration in an individual plant and therefore 88.15% of the experiment produced data for recovery of homozygous mutations at 8x coverage.</p></div><div id="sec-11" class="subsection"><h3>Mutation discovery in complex pools</h3><p id="p-22">To evaluate the use of sonication for TILLING assays involving complex pooled PCR products, the mapped reads from sequencing were subjected to two mutation discovery pipelines. Three plants harboring previously identified induced mutations were used to evaluate mutation discovery (<a id="xref-ref-27-3" class="xref-bibr" href="#ref-27">Szarejko <em>et al.</em> 2017</a>). The CAMBa pipeline discovered all positive controls (Table S15). Control 1, a plant harboring a C to T homozygous mutation at position 416 in the PRT1_2 target was identified at allele frequencies of 0.0078 in pool 11 (first dimension of pooling), 0.0103 in pool 19 (second dimension of pooling) and 0.0117 in pool 41 (third dimension of pooling). The expected frequency of a homozygous mutation is 0.0039. Control 2, a plant harboring a C to T heterozygous mutation in target EXPB4_23 was identified in pool 15 (first dimension) at a frequency of 0.00778, in pool 20 (second dimension) at 0.0024 and in pool 37 (third dimension) at 0.00199. The expected frequency of a heterozygous mutation is 0.00195. The heterozygous T to A mutation in control 3, VDE_2_27, was identified in pool 14 (0.0026), 26 (0.0051) and 46 (0.0060). GATK HaplotypeCaller was performed using two different ploidy levels to recover rare alleles. Variant calling at ploidy settings 600 and 2000 both recovered the homozygous mutation in control 1. Allele frequencies in pools 11, 19, and 41 for ploidy 600 were 0.0033, 0.0050 and 0.0067, respectively. For ploidy 2000 the frequencies were 0.0035, 0.0055 and 0.0070. Control 2 was not identified at either ploidy setting. The control 3 mutation was identified only in pool 14 at frequencies of 0.0033 and 0.0030 for ploidy settings 600 and 2000, respectively. No trend was observed between sequencing coverage of the amplicon in the pool and observed <em>vs.</em> expected allele frequencies (Table S15).</p></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><p id="p-23">The ability to rapidly discover novel nucleotide variation in germplasm collections and mutant populations is a fundamental tool for functional genomics and breeding of plants and animals. For example, large germplasm collections have been maintained for many species, but limited resources mean little has been done to characterize gene coding and regulatory sequence variation that may prove important for unlocking their full potential. Further, efficient methods for accurate discovery and cataloguing of nucleotide variation can be important to measure and protect biological diversity in developing countries (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Gepts 2004</a>). For functional genomic studies, screening of a TILLING population for the recovery of induced mutations predicted to alter gene function typically requires interrogation of thousands of individuals. While reduced-representation genome methods such as exome capture sequencing have been described to recover rare single nucleotide variants in large genome species such as wheat, the approach remains cost-prohibitive for smaller scale projects and for understudied species where research funds are limited (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Krasileva <em>et al.</em> 2017</a>). Amplicon sequencing provides a low-cost alternative that is advantageous in that it is highly flexible with regard to population size and the choice of sequence regions for evaluation.</p><p id="p-24">The flexibility and throughput of amplicon sequencing is increased through fragmentation of PCR products. More bases can be interrogated from a PCR product, and fragmentation into small pieces allows use of a range of different short-read sequencing platforms. Fragmentation of low molecular weight PCR products, however, is potentially more challenging when compared to working with genomic DNA where starting fragment sizes can exceed 20,000 bp when using standard genomic DNA extraction protocols (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Huynh <em>et al.</em> 2017</a>). For example, recovery of a 500 bp fragment from a 1000 bp PCR product requires that a single double strand break be induced at the mid-point of the amplicon. Owing to this, optimal conditions for sonication-based fragmentation of a single amplicon were evaluated. Trials showed that sonication for 30 sec with a peak power of 50 W, Duty factor of 40 and 200 cycles per burst produced a smooth distribution of fragment sizes that peaked near 400 base pairs, a suitable size for Illumina library preparation and sequencing. While different sized amplicons may require subtle optimizations, it was reasoned that sonication at this setting may be suitable for a complex pool of PCR products of varying sizes. A pool of amplicons ranging between 670 and 1513 bp was next evaluated because this is the size range of amplicons used in traditional gel-based TILLING approaches that have served as a benchmark for mutation discovery in pools (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Till <em>et al.</em> 2003</a>, <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">2004</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Talamè <em>et al.</em> 2008</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Gottwald <em>et al.</em> 2009</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bovina <em>et al.</em> 2011</a>). Conditions were sought to allow higher throughput screening of existing TILLING populations where gene targets and oligonucleotide primers have been previously validated.</p><p id="p-25">Evaluation of fragment size distribution of the libraries prepared from pooled PCR products showed a broad distribution of fragments, suggesting suitability for sequencing. Analysis of post-sonication fragment sizes after library preparation, and also of insert sizes post-mapping, showed a broad distribution of fragment sizes with median insert size of 348.23 bp, close 350 bp, one of two optimal insert sizes for Illumina library preparation. Over-accumulation of DNA fragments at specific molecular weights was observed. However, this did not appear to affect the experiment as all pools produced high base and mapping quality with the majority of amplicons in pools producing high read coverage. Over-accumulation of specific molecular weights may indicate that some sequence contexts or fragment sizes are less likely to experience sonication-induced cleavage. The GC content of the amplicon may influence this, as a weak negative correlation between GC content and mean coverage was observed. Internal sequences of DNA fragments longer than the sequencing read length will go unsequenced affecting coverage across an amplicon. However, a weak positive correlation between GC content and amplicon size was also observed, and coverage differences may be explained by the method used for amplicon quantification (see below). Further tests are required to determine if there is any dependence on the concentration of input PCR product, as the phenomenon of accumulation of specific molecular weight fragments was most pronounced in the highest concentration pool number 24. It is interesting to note that while read coverage is generally even across fragmented amplicons, there is a consistent pattern of higher coverage and reduced base quality near but not directly at amplicon termini. This pattern is expected in paired end reads if there is a lower probability for fragmentation of DNA sequences near the termini of the PCR amplicon. Importantly, with the exception of the increases near the amplicon termini, read coverage is consistent across the length of amplicons. A weak correlation was observed between fragment size and median sequencing coverage in the experiment. While the sonication procedure cannot be ruled out as a contributing factor, it is also likely that this observation reflects errors in quantification of PCR products prior to pooling. Intercalating DNA binding dyes will produce brighter band images (with a higher pixel density) in larger DNA fragments compared to smaller fragments of the same molar amount. The quantification used for the barley experiment did not correct for amplicon size. Therefore concentrations were over-estimated for larger fragments and therefore these amplicons were over-diluted compared to smaller fragments. While slightly underrepresented, quantity and coverage from the larger amplicons was suitable for mutation discovery. Based on the consistency of coverage across the length of amplicons, it was concluded that the mean coverage per amplicon in each pool is a useful metric to evaluate the performance of the entire population. The population consisted of 1536 data points (32 amplicons × 48 pools). In total, 2.08% of data points had coverage values less than 5% of the mean for the amplicon in all pools. These were considered assay failures even though some high quality reads were produced. Assay failures measured in this way were slightly lower in cassava and tomato non-fragmented amplicon experiments performed on the same MiSeq sequencer (0.5 and 1.19%, respectively). Assay failures may arise due to a combination of variables including genomic DNA quality, genomic DNA quantification, PCR quality, PCR quantification and library quantification (see below). Interestingly, PCR product evaluation and quantification methods were different in all three experiments. Every tomato PCR product was quantified by capillary electrophoresis using the Fragment Analyzer. This proved costly, and so for the cassava experiment all samples were analyzed using 96 well e-gels and digital image quantification using Image J (<a id="xref-ref-5-7" class="xref-bibr" href="#ref-5">Duitama <em>et al.</em> 2017</a>; <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Huynh <em>et al.</em> 2017</a>). For barley, the procedure used in cassava was streamlined by replacing digital image quantification with a rapid visual estimation to determine average yield in nanograms for each amplicon. The method of PCR product quantification trends with failure rates based on sequencing coverage, and therefore may be related. Nevertheless, assay failure is low in all experiments and from this we conclude that using a fragmented PCR approach does not significantly add to assay failures.</p><p id="p-26">Comparisons between sequencing metrics from fragmented and non-fragmented PCR products allow further evaluation of the sonication-based approach. Data from direct sequencing of non-fragmented PCR products pooled in a similar fashion (Figure S5) shows both coverage and base quality drop toward the center of amplicons that are near 600 base pairs. In contrast, coverage increases in the center of shorter amplicons as expected when applying paired-end sequencing. Based on this data, we conclude that the sonication-based approach produces more consistent data independent of starting amplicon size. This allows for more precise experimental design, taking into consideration genomic DNA sample pooling and amplicon pooling so that required depth of coverage can be obtained for accurate variant calling.</p><p id="p-27">There are many parameters in addition to amplicon fragmentation that are important for a successful assay. These include proper quantification and pooling of genomic DNA, proper quantification and pooling of PCR products, proper quantification and pooling of sequencing libraries, and producing sufficient read coverage to ensure recovery of rare mutations based on how many genomic DNA samples are pooled together. It is difficult to evaluate if a DNA quality issue results in a subset of genomic DNAs in a pool underperforming in PCR. All other parameters can be investigated. Quantification of PCR products is evaluated through coverage variations within the same amplicon between different pools, and also through coverage variations for all pools combined calculated between different amplicons. Ideally, all PCR products in all pools will be represented at the same molarity and thus produce the same sequencing coverage. This may be impossible to achieve. In the barley experiment, maximum coverage within the same amplicon ranged between 2 and 11 fold higher than the mean (Table S12). Coverage between different amplicons ranged between 2092.7 and 13445.8 (6.42 fold). This is in comparison to a 4.94 fold difference between the minimum and maximum coverage between amplicons in the tomato TILLING experiment. Thus, the careful quantification of each PCR product used for tomato TILLING, which utilized robotic liquid handling and capillary analysis of each non-pooled PCR product, may provide only incremental assay improvements. In addition to PCR concentration, variation in sequencing library quantification will affect the mean coverage of each pool in an experiment. Library quantification in the barley experiment was performed using fluorimetry. One noticeable error was observed whereby pool 24 was highly over-represented (mean coverage of 52170.41) compared to the next highest pool (mean coverage 8854.62, pool 31). Library quantification using fluorimetry allowed rapid quantification compared to quantitative PCR and the over-representation of pool 24 is assumed to have resulted from a human error. Further tests are required to determine if library to library mean coverage can be made more similar by adding additional quantification methods such as qPCR. Nevertheless, assay success can be measured as having achieved sufficient quality and coverage for pools and amplicons in order to recover mutations. Expected variations can be built into the experimental design when choosing the number of PCR amplicons to sequence and the level of genomic DNA pooling to employ. With regards to coverage of induced mutations, 88% of the experiment produced sufficient data for recovery of heterozygous mutations with a read coverage of the mutant allele of at least four. This coverage can be increased by increasing the number of sequencing reads in the experiment.</p><p id="p-28">Numerous DNA sequencers can potentially be used for pooled amplicon experiments when amplicons are fragmented. Quality and throughput can vary between different platforms and between different sequencer models of the same platform, as reported for Illumina sequencers <em>(</em><a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Liu <em>et al.</em> 2012</a>). In theory, any platform is suitable provided sufficient coverage and quality is achieved. MiSeq 2x300PE sequencing was used in this study to provide direct comparison with previous non-fragmented amplicon experiments run on the same machine. Higher throughput and lower per-base costs are possible on other sequencing platforms such as the Illumina HiSeq system. Cost estimations can also be considered to further evaluate the utility of sonication of larger amplicons <em>vs.</em> direct sequencing. For simplicity, quality and coverage are assumed to be identical for a non-fragmented 600 bp amplicon <em>vs.</em> a fragmented 1200 bp amplicon. When considering major consumables and assays employing capillary quantification of all PCR products, it is estimated that it will cost approximately 1.9x more to screen two 600 bp amplicons <em>vs.</em> one 1200 bp amplicon that has been sonicated (Table S16). Cost estimations do not take into account the extra time for PCR amplification and FA machine run times when performing more PCR reactions producing shorter amplicon sizes. Comparison of barley, tomato and cassava data suggests that capillary quantification may only provide small assay improvements. Without this step, sequencing of 600 bp amplicons using 2x300PE sequencing is 1.3x more costly than sequencing 1200 bp amplicons (Table S16). Additional time and cost savings may be achieved by using amplicons larger than those reported here. For example, while gene sizes vary, median gene length in plants such as barley can be thousands of base pairs (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Wang <em>et al.</em> 2017</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Mascher <em>et al.</em> 2017</a>). An elegant experimental design would be for one amplicon to cover an entire coding sequence. Long PCR, however, can be more challenging to optimize, and careful testing should be performed to avoid unforeseen biases and to ensure that PCR amplification produces and equal representation of molecules from complex pooled genomic DNAs. Errors from PCR must also be considered. Previous studies using traditional gel-based approaches showed using Taq polymerase with amplicon sizes up to 1500 bp resulted in less than a 5% false positive error rate (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Till <em>et al.</em> 2006a</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">2010</a>). PCR based errors are further mitigated when applying a three-dimensional genomic DNA pooling strategy whereby each genomic DNA is represented in three unique pools and true mutations are found only once in each pooling dimension (<a id="xref-ref-37-5" class="xref-bibr" href="#ref-37">Tsai <em>et al.</em> 2011</a>). When applying this criterion to mutation calling the likelihood of false positive errors due to PCR amplification of genomic DNA or through post-ligation library amplification is reduced. Further, library preparation kits that do not employ post-ligation amplification are available if errors from library preparation are a concern. However, more studies are needed before direct comparisons between traditional enzymatic-based TILLING assays and sequencing-based assays can be made. Higher pooling in sequencing assays (256x <em>vs.</em> 8x for enzymatic assays) means that rare errors will be more likely to be detected.</p><p id="p-29">An optimal experimental design for sequencing of fragmented pooled amplicons will balance assay throughput, cost, and accuracy. The ability to maximize recovery of true positives while minimizing false positives is important. The choice of bioinformatics tools is a critical component. Previous studies in tomato TILLING by Sequencing tested six different mutation calling tools and estimated a range of accuracy from 89.33 to 29.33 (<a id="xref-ref-9-7" class="xref-bibr" href="#ref-9">Gupta <em>et al.</em> 2017</a>). In that study, the CAMBa tool that was purpose built for TILLING performed the best. In the present study, CAMBa successfully identified all three positive controls, but only the homozygous mutation was identified in all three pools using GATK HaplotypeCaller with sample ploidy set at either 600 or 2000. The homozygous mutation is present in DNA pools at an expected frequency of 0.0039 while the heterozygous mutation is expected at 0.00195. Using the amplicon coverage values for the pools, the homozygous mutation is expected to be sequenced 14x in pool 11, 35.65x in pool 19 and 34.26x in pool 41. The heterozygous mutations are expected at 4.98x, 7.91x, 14.26x, 11.81x, 8.04x, and 20.15x respectively, in pool numbers 15, 20, 37, 14, 26, and 46. Lower representation of heterozygous mutations may have contributed to the failure to call the mutation with HaplotypeCaller. Analysis of novel putative mutations identified in the mutant screen is ongoing. True mutations resulting from this work will be used to further evaluate the performance of mutation discovery software. Variant calling tools are constantly evolving and their improvement will likely lead to improvements in pooled amplicon assays such as TILLING.</p><p id="p-30">We conclude that fragmentation of pooled amplicons by ultrasonication provides a suitable method for producing even coverage sequencing for targeted recovery of nucleotide variation in large populations of samples. Alternative methods for amplicon fragmentation, such as use of dsDNA fragmentase have been applied for TILLING with genomic pooling of 64-fold (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Burkart-Waco <em>et al.</em> 2017</a>). This suggests that multiple methods may be considered for experiments employing complex pools of PCR products. In theory, any method that can produce a distribution of fragments with optimal size ranges for the chosen library preparation method should be suitable. Further tests are needed, however, to determine if alternative fragmentation methods will produce similar results in highly pooled samples. Given all of the experimental variables in a pooled amplicon experiment that can affect variant discovery, and the fact that variant calling algorithms are still evolving, careful testing of amplicon fragmentation, amplicon pooling, genomic DNA pooling and bioinformatics tools using positive controls is advised prior to choosing an experimental design. In addition to recovery of induced mutations it is envisioned that combining sample pooling with amplicon fragmentation can improve screening efficiencies for natural allelic variations. For example, the same approach used for barley mutants can be applied for sequencing of germplasm banks where rare alleles may provide an important resource for understanding gene function and allow for the genetic improvement of domesticated and semi-domesticated species.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-31">We thank Keji Dada of the Cocoa Research Institute of Nigeria for assistance in coffee primer design and testing. Professor Margit Laimer of BOKU provided <em>Coffea arabica</em> genomic DNA. The Centro de Genómica Nutricional Agroacuícola kindly provided time on their computer cluster for parts of the data analysis provided in this manuscript. We thank Prateek Gupta of the University of Hyderabad, India for sharing coverage data for tomato TILLING by Sequencing. Authors BJT, JJC, IS &amp; II contributed to experimental design and project oversight. AT, LJ &amp; BJT prepared the manuscript. KG, JJC, MSZ, BJH, &amp; BJT conducted experiments, contributed to data analysis and interpretation and manuscript editing. Funding for this work was provided by the European Regional Development Fund through the Innovative Economy for Poland 2007–2013, project WND-POIG.01.03.01-00-101/08 POLAPGEN-BD “Biotechnological tools for breeding cereals with increased resistance to drought,” task 22 and by the Food and Agriculture Organization of the United Nations and the International Atomic Energy Agency through their Joint FAO/IAEA Program of Nuclear Techniques in Food and Agriculture. This work is part of IAEA Coordinated Research Project D24012 and D22005.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-32">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8052821">https://doi.org/10.25387/g3.8052821</a>.</p></li><li class="fn" id="fn-2"><p id="p-33"><em>Communicating editor: A. Paterson</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-04-30"><span class="received-label">Received </span>April 30, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-06-12"><span class="accepted-label">Accepted </span>June 12, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Tramontano <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.1" data-doi="10.1017/S147926211100044X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bovina</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Talame</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Silvio</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Sanguineti</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Trost</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Starch metabolism mutants in barley: A TILLING approach</span>. <abbr class="cit-jnl-abbrev">Plant Genet. Resour.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">170</span>–<span class="cit-lpage">173</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S147926211100044X</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenet.%2BResour.%26rft.volume%253D9%26rft.spage%253D170%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS147926211100044X%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S147926211100044X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-book" id="cit-9.8.2657.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burkart-Waco</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Tsai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Ngo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. M.</span> <span class="cit-name-surname">Henry</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Comai</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Next-Generation Sequencing for Targeted Discovery of Rare Mutations in Rice</span>, pp. <span class="cit-fpage">323</span>–<span class="cit-lpage">340</span> in <span class="cit-source">Biotechnologies for Plant Mutation Breeding</span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">Cham</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.3" data-doi="10.1093/bioinformatics/btg251"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carver</span>,  <span class="cit-name-given-names">T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bleasby</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The design of Jemboss: a graphical user interface to EMBOSS</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1837</span>–<span class="cit-lpage">1843</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btg251</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.stitle%253DBioinformatics%26rft.aulast%253DCarver%26rft.auinit1%253DT.%26rft.volume%253D19%26rft.issue%253D14%26rft.spage%253D1837%26rft.epage%253D1843%26rft.atitle%253DThe%2Bdesign%2Bof%2BJemboss%253A%2Ba%2Bgraphical%2Buser%2Binterface%2Bto%2BEMBOSS%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtg251%26rft_id%253Dinfo%253Apmid%252F14512356%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btg251&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14512356&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185701100015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.4" data-doi="10.1111/pbi.12901"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Datta</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jankowicz-Cieslak</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Nielen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Ingelbrecht</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Till</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Induction and recovery of copy number variation in banana through gamma irradiation and low coverage whole genome sequencing</span>. <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1644</span>–<span class="cit-lpage">1653</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/pbi.12901</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BBiotechnol.%2BJ.%26rft.volume%253D16%26rft.spage%253D1644%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fpbi.12901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/pbi.12901&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.5" data-doi="10.1016/j.csbj.2017.01.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Duitama</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Kafuri</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Tello</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Leiva</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Hofinger</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Deep Assessment of Genomic Diversity in Cassava for Herbicide Tolerance and Starch Biosynthesis</span>. <abbr class="cit-jnl-abbrev">Comput. Struct. Biotechnol. J.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">185</span>–<span class="cit-lpage">194</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.csbj.2017.01.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BStruct.%2BBiotechnol.%2BJ.%26rft.volume%253D15%26rft.spage%253D185%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.csbj.2017.01.002%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.csbj.2017.01.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.6" data-doi="10.1104/pp.103.038885"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gepts</span>,  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Who Owns Biodiversity, and How Should the Owners Be Compensated?</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">1295</span>–<span class="cit-lpage">1307</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.103.038885</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiology%26rft.stitle%253DPlant%2BPhysiol.%26rft.aulast%253DGepts%26rft.auinit1%253DP.%26rft.volume%253D134%26rft.issue%253D4%26rft.spage%253D1295%26rft.epage%253D1307%26rft.atitle%253DWho%2BOwns%2BBiodiversity%252C%2Band%2BHow%2BShould%2Bthe%2BOwners%2BBe%2BCompensated%253F%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.103.038885%26rft_id%253Dinfo%253Apmid%252F15084724%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTM0LzQvMTI5NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOC8yNjU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.7" data-doi="10.1186/1756-0500-2-258"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gottwald</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Bauer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Komatsuda</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Lundqvist</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Stein</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">TILLING in the two-rowed barley cultivar “Barke” reveals preferred sites of functional diversity in the gene HvHox1</span>. <abbr class="cit-jnl-abbrev">BMC Res. Notes</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">258</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1756-0500-2-258</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DBMC%2BRes%2BNotes%26rft.aulast%253DGottwald%26rft.auinit1%253DS.%26rft.volume%253D2%26rft.spage%253D258%26rft.epage%253D258%26rft.atitle%253DTILLING%2Bin%2Bthe%2Btwo-rowed%2Bbarley%2Bcultivar%2B%2527Barke%2527%2Breveals%2Bpreferred%2Bsites%2Bof%2Bfunctional%2Bdiversity%2Bin%2Bthe%2Bgene%2BHvHox1.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1756-0500-2-258%26rft_id%253Dinfo%253Apmid%252F20017921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1756-0500-2-258&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20017921&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.8" data-doi="10.1186/s12864-015-1348-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Abernathy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zeng</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Ozias-Akins</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">TILLING by sequencing to identify induced mutations in stress resistance genes of peanut (Arachis hypogaea)</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">157</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-1348-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D157%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1348-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1348-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Reddaiah</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Salava</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Upadhyaya</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Tyagi</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Next-generation sequencing (NGS)-based identification of induced mutations in a doubly mutagenized tomato (Solanum lycopersicum) population. Plant J</span>. <abbr class="cit-jnl-abbrev">Cell. Mol. Biol.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">495</span>–<span class="cit-lpage">508</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell.%2BMol.%2BBiol.%26rft.volume%253D92%26rft.spage%253D495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.10" data-doi="10.2144/000114133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Head</span>,  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. K.</span> <span class="cit-name-surname">Komori</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">LaMere</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Whisenant</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Van Nieuwerburgh</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Library construction for next-generation sequencing: Overviews and challenges</span>. <abbr class="cit-jnl-abbrev">Biotechniques</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">61</span> <span class="cit-comment">–passim.</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2144/000114133</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechniques%26rft.volume%253D56%26rft.spage%253D61%26rft_id%253Dinfo%253Adoi%252F10.2144%252F000114133%26rft_id%253Dinfo%253Apmid%252F24502796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2144/000114133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24502796&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000331491800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.11" data-doi="10.1093/bfgp/elt051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hirsch</span>,  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Evans</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. R.</span> <span class="cit-name-surname">Buell</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. N.</span> <span class="cit-name-surname">Hirsch</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Reduced representation approaches to interrogate genome diversity in large repetitive plant genomes</span>. <abbr class="cit-jnl-abbrev">Brief. Funct. Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">257</span>–<span class="cit-lpage">267</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bfgp/elt051</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BFunct.%2BGenomics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbfgp%252Felt051%26rft_id%253Dinfo%253Apmid%252F24395692%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bfgp/elt051&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24395692&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-book" id="cit-9.8.2657.12" data-doi="10.1007/978-3-319-45021-6_14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huynh</span>,  <span class="cit-name-given-names">O. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jankowicz-Cieslak</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Saraye</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Hofinger</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Till</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Low-Cost Methods for DNA Extraction and Quantification</span>, pp. <span class="cit-fpage">227</span>–<span class="cit-lpage">239</span> in <span class="cit-source">Biotechnologies for Plant Mutation Breeding</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Jankowicz-Cieslak</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">T. H.</span> <span class="cit-name-surname">Tai</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Kumlehn</span></span>, and <span class="cit-ed"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Till</span></span>. <span class="cit-publ-name">Springer International Publishing AG Switzerland</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-3-319-45021-6_14</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechnologies%2Bfor%2BPlant%2BMutation%2BBreeding%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-319-45021-6_14%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-3-319-45021-6_14&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.13" data-doi="10.1038/nature11543"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">International Barley Genome Sequencing Consortium</span></li><li><span class="cit-auth"><span class="cit-name-surname">Mayer</span>,  <span class="cit-name-given-names">K. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Waugh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">Brown</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Schulman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Langridge</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A physical, genetic and functional sequence assembly of the barley genome</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">491</span>: <span class="cit-fpage">711</span>–<span class="cit-lpage">716</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature11543</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMayer%26rft.auinit1%253DK.%2BF.%26rft.volume%253D491%26rft.issue%253D7426%26rft.spage%253D711%26rft.epage%253D716%26rft.atitle%253DA%2Bphysical%252C%2Bgenetic%2Band%2Bfunctional%2Bsequence%2Bassembly%2Bof%2Bthe%2Bbarley%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11543%26rft_id%253Dinfo%253Apmid%252F23075845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11543&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23075845&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000311606000035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-9.8.2657.14" data-doi="10.1007/978-3-319-22521-0_8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jankowicz-Cieslak</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Till</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Forward and Reverse Genetics in Crop Breeding</span>, pp. <span class="cit-fpage">215</span>–<span class="cit-lpage">240</span> in <span class="cit-source"><em>Advances in Plant Breeding Strategies: Breeding</em>, <em>Biotechnology and Molecular Tools</em></span>, edited by <span class="cit-ed"><span class="cit-name-surname">Al-Khayri</span>,  <span class="cit-name-given-names">J. M.</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Jain</span></span>, and <span class="cit-ed"> <span class="cit-name-given-names">D. V.</span> <span class="cit-name-surname">Johnson</span></span>. <span class="cit-publ-name">Springer International Publishing Switzerland</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-3-319-22521-0_8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2BPlant%2BBreeding%2BStrategies%253A%2BBreeding%252C%2BBiotechnology%2Band%2BMolecular%2BTools%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-319-22521-0_8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-3-319-22521-0_8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.15" data-doi="10.1073/pnas.1619268114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Krasileva</span>,  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. A.</span> <span class="cit-name-surname">Vasquez-Gross</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Howell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Bailey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Paraiso</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Uncovering hidden variation in polyploid wheat</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">114</span>: <span class="cit-fpage">E913</span>–<span class="cit-lpage">E921</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1619268114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1619268114%26rft_id%253Dinfo%253Apmid%252F28096351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTE0LzYvRTkxMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOC8yNjU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.16" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Durbin</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows–Wheeler transform</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp324</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp324&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19451168&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267665900006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Hu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">He</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Comparison of Next-Generation Sequencing Systems</span>. <abbr class="cit-jnl-abbrev">BioMed Res. Int.</abbr> 2012: <span class="cit-fpage">1</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.18" data-doi="10.1038/nature22043"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mascher</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Gundlach</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Himmelbach</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Beier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. O.</span> <span class="cit-name-surname">Twardziok</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A chromosome conformation capture ordered sequence of the barley genome</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">544</span>: <span class="cit-fpage">427</span>–<span class="cit-lpage">433</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature22043</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D544%26rft.spage%253D427%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature22043%26rft_id%253Dinfo%253Apmid%252F28447635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature22043&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28447635&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.19" data-doi="10.1038/74542"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCallum</span>,  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Comai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. A.</span> <span class="cit-name-surname">Greene</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Henikoff</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Targeted screening for induced mutations</span>. <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">455</span>–<span class="cit-lpage">457</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/74542</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DMcCallum%26rft.auinit1%253DC.%2BM.%26rft.volume%253D18%26rft.issue%253D4%26rft.spage%253D455%26rft.epage%253D457%26rft.atitle%253DTargeted%2Bscreening%2Bfor%2Binduced%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F74542%26rft_id%253Dinfo%253Apmid%252F10748531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/74542&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10748531&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086444300033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.20" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Hanna</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Banks</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sivachenko</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Cibulskis</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The Genome Analysis Toolkit: A MapReduce framework for analyzing next-generation DNA sequencing data</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.107524.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOC8yNjU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.21" data-doi="10.1093/bioinformatics/btv566"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Okonechnikov</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Conesa</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">García-Alcalde</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Qualimap 2: advanced multi-sample quality control for high-throughput sequencing data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">292</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtv566%26rft_id%253Dinfo%253Apmid%252F26428292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btv566&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26428292&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.22" data-doi="10.1186/s12864-015-1263-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. N.</span> <span class="cit-name-surname">Shah</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. G.</span> <span class="cit-name-surname">Phelps</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Doherty</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. A.</span> <span class="cit-name-surname">Johnson</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Rapid identification and recovery of ENU-induced mutations with next-generation sequencing and Paired-End Low-Error analysis</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">83</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-1263-4</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D83%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1263-4%26rft_id%253Dinfo%253Apmid%252F25886285%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1263-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25886285&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.23" data-doi="10.1371/journal.pone.0004761"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rigola</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">van Oeveren</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Janssen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Bonne</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Schneiders</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">High-throughput detection of induced mutations and natural variation using KeyPoint technology</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>: <span class="cit-elocation-id">e4761</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0004761</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DRigola%26rft.auinit1%253DD.%26rft.volume%253D4%26rft.issue%253D3%26rft.spage%253De4761%26rft.epage%253De4761%26rft.atitle%253DHigh-throughput%2Bdetection%2Bof%2Binduced%2Bmutations%2Band%2Bnatural%2Bvariation%2Busing%2BKeyPoint%2Btechnology.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0004761%26rft_id%253Dinfo%253Apmid%252F19283079%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0004761&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19283079&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-book" id="cit-9.8.2657.24" data-doi="10.1007/978-3-319-45021-6_17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slota</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Maluszynski</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Szarejko</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Bioinformatics-Based Assessment of the Relevance of Candidate Genes for Mutation Discovery</span>, pp. <span class="cit-fpage">263</span>–<span class="cit-lpage">280</span> in <span class="cit-source">Biotechnologies for Plant Mutation Breeding</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Jankowicz-Cieslak</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">T. H.</span> <span class="cit-name-surname">Tai</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Kumlehn</span></span>, and <span class="cit-ed"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Till</span></span>. <span class="cit-publ-name">Springer International Publishing AG Switzerland</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-3-319-45021-6_17</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechnologies%2Bfor%2BPlant%2BMutation%2BBreeding%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-319-45021-6_17%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-3-319-45021-6_17&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.25" data-doi="10.1016/j.dnarep.2015.02.018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stolarek</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gruszka</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Braszewska-Zalewska</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Maluszynski</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">a</span> <span class="cit-article-title">Alleles of newly identified barley gene HvPARP3 exhibit changes in efficiency of DNA repair</span>. <abbr class="cit-jnl-abbrev">DNA Repair (Amst.)</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">116</span>–<span class="cit-lpage">130</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.dnarep.2015.02.018</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDNA%2BRepair%2B%2528Amst.%2529%26rft.volume%253D28%26rft.spage%253D116%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.dnarep.2015.02.018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.dnarep.2015.02.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.26" data-doi="10.1093/mutage/gev033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stolarek</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gruszka</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Braszewska-Zalewska</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Maluszynski</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span><span class="cit-comment">b</span> <span class="cit-article-title">Functional analysis of the new barley gene HvKu80 indicates that it plays a key role in double-strand DNA break repair and telomere length regulation</span>. <abbr class="cit-jnl-abbrev">Mutagenesis</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">785</span>–<span class="cit-lpage">797</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/mutage/gev033</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMutagenesis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmutage%252Fgev033%26rft_id%253Dinfo%253Apmid%252F25958390%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/mutage/gev033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25958390&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-book" id="cit-9.8.2657.27" data-doi="10.1007/978-3-319-45021-6_6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szarejko</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Szurman-Zubrzycka</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Nawrot</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Marzec</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gruszka</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Creation of a TILLING Population in Barley After Chemical Mutagenesis with Sodium Azide and MNU</span>, pp. <span class="cit-fpage">91</span>–<span class="cit-lpage">111</span> in <span class="cit-source">Biotechnologies for Plant Mutation Breeding</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Jankowicz-Cieslak</span>,  <span class="cit-name-given-names">J.</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">T. H.</span> <span class="cit-name-surname">Tai</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Kumlehn</span></span>, and <span class="cit-ed"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Till</span></span>. <span class="cit-publ-name">Springer International Publishing</span>, <span class="cit-publ-loc">Cham</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-3-319-45021-6_6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechnologies%2Bfor%2BPlant%2BMutation%2BBreeding%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-319-45021-6_6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-3-319-45021-6_6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.28" data-doi="10.1111/j.1467-7652.2008.00341.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Talamè</span>,  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Bovina</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Sanguineti</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Tuberosa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Lundqvist</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">TILLMore, a resource for the discovery of chemically induced mutants in barley</span>. <abbr class="cit-jnl-abbrev">Plant Biotechnol. J.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">477</span>–<span class="cit-lpage">485</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1467-7652.2008.00341.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPlant%2BBiotechnol%2BJ%26rft.aulast%253DTalame%26rft.auinit1%253DV.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253D477%26rft.epage%253D485%26rft.atitle%253DTILLMore%252C%2Ba%2Bresource%2Bfor%2Bthe%2Bdiscovery%2Bof%2Bchemically%2Binduced%2Bmutants%2Bin%2Bbarley.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-7652.2008.00341.x%26rft_id%253Dinfo%253Apmid%252F18422888%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-7652.2008.00341.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18422888&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255724000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.29"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tange</span>,  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">GNU Parallel: The Command-Line Power Tool</span>. <abbr class="cit-jnl-abbrev">USENIX Mag.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">42</span>–<span class="cit-lpage">47</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DUSENIX%2BMag.%26rft.volume%253D36%26rft.spage%253D42%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.30"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Till</span>,  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Datta</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jankowicz-Cieslak</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">TILLING: The Next Generation</span>. <abbr class="cit-jnl-abbrev">Adv. Biochem. Eng. Biotechnol.</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BBiochem.%2BEng.%2BBiotechnol.%26rft.volume%253D164%26rft.spage%253D139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-book" id="cit-9.8.2657.31" data-doi="10.1007/978-3-319-16259-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Till</span>,  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jankowicz-Cieslak</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O. A.</span> <span class="cit-name-surname">Huynh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Beshir</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. G.</span> <span class="cit-name-surname">Laport</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-source">Low-Cost Methods for Molecular Characterization of Mutant Plants</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Till</span>,  <span class="cit-name-given-names">B. J.</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jankowicz-Cieslak</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">O. A.</span> <span class="cit-name-surname">Huynh</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Beshir</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">R. G.</span> <span class="cit-name-surname">Laport</span></span> <span class="cit-etal">et al.</span> <span class="cit-publ-name">Springer International Publishing AG Switzerland</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-3-319-16259-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLow-Cost%2BMethods%2Bfor%2BMolecular%2BCharacterization%2Bof%2BMutant%2BPlants%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-319-16259-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-3-319-16259-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.32" data-doi="10.1007/s00122-010-1395-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Till</span>,  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Jankowicz-Cieslak</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Sagi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O. A.</span> <span class="cit-name-surname">Huynh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Utsushi</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Discovery of nucleotide polymorphisms in the Musa gene pool by Ecotilling</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">1381</span>–<span class="cit-lpage">1389</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-010-1395-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DTill%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D121%26rft.issue%253D7%26rft.spage%253D1381%26rft.epage%253D1389%26rft.atitle%253DDiscovery%2Bof%2Bnucleotide%2Bpolymorphisms%2Bin%2Bthe%2BMusa%2Bgene%2Bpool%2Bby%2BEcotilling.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-010-1395-5%26rft_id%253Dinfo%253Apmid%252F20589365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-010-1395-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20589365&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.33" data-doi="10.1101/gr.977903"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Till</span>,  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. H.</span> <span class="cit-name-surname">Reynolds</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. A.</span> <span class="cit-name-surname">Greene</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Codomo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. C.</span> <span class="cit-name-surname">Enns</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Large-scale discovery of induced point mutations with high-throughput TILLING</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">524</span>–<span class="cit-lpage">530</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.977903</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DTill%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D13%26rft.issue%253D3%26rft.spage%253D524%26rft.epage%253D530%26rft.atitle%253DLarge-Scale%2BDiscovery%2Bof%2BInduced%2BPoint%2BMutations%2BWith%2BHigh-Throughput%2BTILLING%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.977903%26rft_id%253Dinfo%253Apmid%252F12618384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEzLzMvNTI0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI2NTcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.34" data-doi="10.1186/1471-2229-4-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Till</span>,  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. H.</span> <span class="cit-name-surname">Reynolds</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Weil</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Springer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Burtner</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Discovery of induced point mutations in maize genes by TILLING</span>. <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2229-4-12</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DTill%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D4%26rft.spage%253D12%26rft.epage%253D12%26rft.atitle%253DDiscovery%2Bof%2Binduced%2Bpoint%2Bmutations%2Bin%2Bmaize%2Bgenes%2Bby%2BTILLING.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-4-12%26rft_id%253Dinfo%253Apmid%252F15282033%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-4-12&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15282033&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.35" data-doi="10.1093/nar/gkl479"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Till</span>,  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Zerr</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Bowers</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. A.</span> <span class="cit-name-surname">Greene</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Comai</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span><span class="cit-comment">a</span> <span class="cit-article-title">High-throughput discovery of rare human nucleotide polymorphisms by Ecotilling</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-elocation-id">e99</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkl479</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DTill%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D34%26rft.issue%253D13%26rft.spage%253De99%26rft.epage%253De99%26rft.atitle%253DHigh-throughput%2Bdiscovery%2Bof%2Brare%2Bhuman%2Bnucleotide%2Bpolymorphisms%2Bby%2BEcotilling%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl479%26rft_id%253Dinfo%253Apmid%252F16893952%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkl479&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16893952&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.36" data-doi="10.1038/nprot.2006.329"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Till</span>,  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Zerr</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Comai</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Henikoff</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span><span class="cit-comment">b</span> <span class="cit-article-title">A protocol for TILLING and Ecotilling in plants and animals</span>. <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">2465</span>–<span class="cit-lpage">2477</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nprot.2006.329</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DTill%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D1%26rft.issue%253D5%26rft.spage%253D2465%26rft.epage%253D2477%26rft.atitle%253DA%2Bprotocol%2Bfor%2BTILLING%2Band%2BEcotilling%2Bin%2Bplants%2Band%2Banimals.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2006.329%26rft_id%253Dinfo%253Apmid%252F17406493%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2006.329&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17406493&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.37" data-doi="10.1104/pp.110.169748"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsai</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Howell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Nitcher</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Missirian</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Watson</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Discovery of Rare Mutations in Populations: TILLING by Sequencing</span>. <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">156</span>: <span class="cit-fpage">1257</span>–<span class="cit-lpage">1268</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.110.169748</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.110.169748%26rft_id%253Dinfo%253Apmid%252F21531898%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTU2LzMvMTI1NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOC8yNjU3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2657.38" data-doi="10.1093/nar/gkw935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Xu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Thilmony</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. M.</span> <span class="cit-name-surname">You</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y. Q.</span> <span class="cit-name-surname">Gu</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">PIECE 2.0: an update for the plant gene structure comparison and evolution database</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">1015</span>–<span class="cit-lpage">1020</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkw935</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw935%26rft_id%253Dinfo%253Apmid%252F27742820%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw935&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27742820&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2657.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/8/2657.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/8/2647" title="De Novo Assembly and Annotation of the Larval Transcriptome of Two Spadefoot Toads Widely Divergent in Developmental Rate" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/8/2667" title="Coordinated Roles of the Putative Ceramide-Conjugation Protein, Cwh43, and a Mn2+-Transporting, P-Type ATPase, Pmr1, in Fission Yeast" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/8"><span>Volume 9 Issue 8, August 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/8.cover-source.jpg?itok=cSNYBBSf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/8/2657" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Lth6TuSGKFM3-YdezPrQ1oGdYi2J-SL64xQn61eKG4o" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/8/2657" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/8/2657" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/8/2657" class="active" data-icon-position="" data-hide-link-title="0">Fragmentation of Pooled PCR Products for Highly Multiplexed TILLING</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/493754" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-9OBsUsCtvlhIIxlK6Cw7LFisW-OM-vGgFaB4p8z5PBo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/8/2657.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/8/2657" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-qvrCo2ULGdToxPfDlCbVksJLCeih40oOECzyZJ0lxz4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/8/2657" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/8/2657.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493754" id="node493754--2" data-pisa="ggg;9/8/2657" data-pisa-master="ggg;g3.119.400301" data-apath="/ggg/9/8/2657.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Fragmentation of Pooled PCR Products for Highly Multiplexed TILLING</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Andrea Tramontano</span>, <span class="highwire-citation-author" data-delta="1">Luka Jarc</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-9249-4700" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Joanna Jankowicz-Cieslak</span>, <span class="highwire-citation-author" data-delta="3">Bernhard J. Hofinger</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-0235-7607" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Katarzyna Gajek</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-4661-5823" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Miriam Szurman-Zubrzycka</span>, <span class="highwire-citation-author" data-delta="6">Iwona Szarejko</span>, <span class="highwire-citation-author" data-delta="7">Ivan Ingelbrecht</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0002-1300-8285" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bradley J. Till</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2657-2666;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400301 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493754--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andrea Tramontano </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Laboratory, Joint FAO/IAEA Division of Nuclear Techniques in Food and Agriculture, IAEA Laboratories Seibersdorf, International Atomic Energy Agency, Vienna International Centre, PO Box 100, A-1400 Vienna, Austria</div><div class='author-affiliation'><span class='nlm-sup'>1</span>These authors contributed equally to this work.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrea%2BTramontano%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tramontano%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrea%2BTramontano%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Luka Jarc </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Laboratory, Joint FAO/IAEA Division of Nuclear Techniques in Food and Agriculture, IAEA Laboratories Seibersdorf, International Atomic Energy Agency, Vienna International Centre, PO Box 100, A-1400 Vienna, Austria</div><div class='author-affiliation'><span class='nlm-sup'>1</span>These authors contributed equally to this work.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luka%2BJarc%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jarc%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuka%2BJarc%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Joanna Jankowicz-Cieslak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Laboratory, Joint FAO/IAEA Division of Nuclear Techniques in Food and Agriculture, IAEA Laboratories Seibersdorf, International Atomic Energy Agency, Vienna International Centre, PO Box 100, A-1400 Vienna, Austria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joanna%2BJankowicz-Cieslak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jankowicz-Cieslak%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoanna%2BJankowicz-Cieslak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9249-4700" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Joanna Jankowicz-Cieslak</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Bernhard J. Hofinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Laboratory, Joint FAO/IAEA Division of Nuclear Techniques in Food and Agriculture, IAEA Laboratories Seibersdorf, International Atomic Energy Agency, Vienna International Centre, PO Box 100, A-1400 Vienna, Austria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bernhard%2BJ.%2BHofinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hofinger%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABernhard%2BJ.%2BHofinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Katarzyna Gajek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Faculty of Biology and Environmental Protection, University of Silesia, Jagiellonska 28, 40-032, Katowice, Poland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katarzyna%2BGajek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gajek%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKatarzyna%2BGajek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0235-7607" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Katarzyna Gajek</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Miriam Szurman-Zubrzycka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Faculty of Biology and Environmental Protection, University of Silesia, Jagiellonska 28, 40-032, Katowice, Poland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miriam%2BSzurman-Zubrzycka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Szurman-Zubrzycka%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMiriam%2BSzurman-Zubrzycka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4661-5823" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Miriam Szurman-Zubrzycka</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Iwona Szarejko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Faculty of Biology and Environmental Protection, University of Silesia, Jagiellonska 28, 40-032, Katowice, Poland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Iwona%2BSzarejko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Szarejko%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIwona%2BSzarejko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Ivan Ingelbrecht </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Laboratory, Joint FAO/IAEA Division of Nuclear Techniques in Food and Agriculture, IAEA Laboratories Seibersdorf, International Atomic Energy Agency, Vienna International Centre, PO Box 100, A-1400 Vienna, Austria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ivan%2BIngelbrecht%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ingelbrecht%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIvan%2BIngelbrecht%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Bradley J. Till </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Laboratory, Joint FAO/IAEA Division of Nuclear Techniques in Food and Agriculture, IAEA Laboratories Seibersdorf, International Atomic Energy Agency, Vienna International Centre, PO Box 100, A-1400 Vienna, Austria</div><div class='author-affiliation'><span class='nlm-sup'>2</span>Present address: Centro de Genómica Nutricional Agroacuícola, Las Heras #350, Temuco, Chile.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bradley%2BJ.%2BTill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Till%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABradley%2BJ.%2BTill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1300-8285" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bradley J. Till</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:brad.till@cgna.cl" class="" data-icon-position="" data-hide-link-title="0">brad.till@cgna.cl</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/8/2657" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/8/2657&amp;title=Fragmentation%20of%20Pooled%20PCR%20Products%20for%20Highly%20Multiplexed%20TILLING" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/8/2657&amp;title=Fragmentation%20of%20Pooled%20PCR%20Products%20for%20Highly%20Multiplexed%20TILLING" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/8/2657&amp;title=Fragmentation%20of%20Pooled%20PCR%20Products%20for%20Highly%20Multiplexed%20TILLING" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/8/2657&amp;text=Fragmentation%20of%20Pooled%20PCR%20Products%20for%20Highly%20Multiplexed%20TILLING" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/8/2657&amp;title=Fragmentation%20of%20Pooled%20PCR%20Products%20for%20Highly%20Multiplexed%20TILLING" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/8/2657&amp;t=Fragmentation%20of%20Pooled%20PCR%20Products%20for%20Highly%20Multiplexed%20TILLING" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/8/2657&amp;title=Fragmentation%20of%20Pooled%20PCR%20Products%20for%20Highly%20Multiplexed%20TILLING" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/8/2657&amp;title=Fragmentation%20of%20Pooled%20PCR%20Products%20for%20Highly%20Multiplexed%20TILLING" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/493754/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493754" id="node493754--3" data-pisa="ggg;9/8/2657" data-pisa-master="ggg;g3.119.400301" data-apath="/ggg/9/8/2657.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Fragmentation of Pooled PCR Products for Highly Multiplexed TILLING</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Andrea Tramontano</span>, <span class="highwire-citation-author" data-delta="1">Luka Jarc</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0002-9249-4700" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Joanna Jankowicz-Cieslak</span>, <span class="highwire-citation-author" data-delta="3">Bernhard J. Hofinger</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0002-0235-7607" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Katarzyna Gajek</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0002-4661-5823" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Miriam Szurman-Zubrzycka</span>, <span class="highwire-citation-author" data-delta="6">Iwona Szarejko</span>, <span class="highwire-citation-author" data-delta="7">Ivan Ingelbrecht</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0002-1300-8285" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Bradley J. Till</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2657-2666;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400301 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493754--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Andrea Tramontano </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Laboratory, Joint FAO/IAEA Division of Nuclear Techniques in Food and Agriculture, IAEA Laboratories Seibersdorf, International Atomic Energy Agency, Vienna International Centre, PO Box 100, A-1400 Vienna, Austria</div><div class='author-affiliation'><span class='nlm-sup'>1</span>These authors contributed equally to this work.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Andrea%2BTramontano%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tramontano%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAndrea%2BTramontano%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Luka Jarc </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Laboratory, Joint FAO/IAEA Division of Nuclear Techniques in Food and Agriculture, IAEA Laboratories Seibersdorf, International Atomic Energy Agency, Vienna International Centre, PO Box 100, A-1400 Vienna, Austria</div><div class='author-affiliation'><span class='nlm-sup'>1</span>These authors contributed equally to this work.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Luka%2BJarc%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jarc%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALuka%2BJarc%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Joanna Jankowicz-Cieslak </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Laboratory, Joint FAO/IAEA Division of Nuclear Techniques in Food and Agriculture, IAEA Laboratories Seibersdorf, International Atomic Energy Agency, Vienna International Centre, PO Box 100, A-1400 Vienna, Austria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Joanna%2BJankowicz-Cieslak%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jankowicz-Cieslak%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJoanna%2BJankowicz-Cieslak%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-9249-4700" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Joanna Jankowicz-Cieslak</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Bernhard J. Hofinger </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Laboratory, Joint FAO/IAEA Division of Nuclear Techniques in Food and Agriculture, IAEA Laboratories Seibersdorf, International Atomic Energy Agency, Vienna International Centre, PO Box 100, A-1400 Vienna, Austria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bernhard%2BJ.%2BHofinger%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hofinger%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABernhard%2BJ.%2BHofinger%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Katarzyna Gajek </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Faculty of Biology and Environmental Protection, University of Silesia, Jagiellonska 28, 40-032, Katowice, Poland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Katarzyna%2BGajek%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gajek%20K&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AKatarzyna%2BGajek%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0235-7607" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Katarzyna Gajek</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Miriam Szurman-Zubrzycka </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Faculty of Biology and Environmental Protection, University of Silesia, Jagiellonska 28, 40-032, Katowice, Poland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Miriam%2BSzurman-Zubrzycka%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Szurman-Zubrzycka%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMiriam%2BSzurman-Zubrzycka%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-4661-5823" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Miriam Szurman-Zubrzycka</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Iwona Szarejko </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Genetics, Faculty of Biology and Environmental Protection, University of Silesia, Jagiellonska 28, 40-032, Katowice, Poland</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Iwona%2BSzarejko%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Szarejko%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIwona%2BSzarejko%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Ivan Ingelbrecht </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Laboratory, Joint FAO/IAEA Division of Nuclear Techniques in Food and Agriculture, IAEA Laboratories Seibersdorf, International Atomic Energy Agency, Vienna International Centre, PO Box 100, A-1400 Vienna, Austria</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ivan%2BIngelbrecht%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ingelbrecht%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIvan%2BIngelbrecht%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Bradley J. Till </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Plant Breeding and Genetics Laboratory, Joint FAO/IAEA Division of Nuclear Techniques in Food and Agriculture, IAEA Laboratories Seibersdorf, International Atomic Energy Agency, Vienna International Centre, PO Box 100, A-1400 Vienna, Austria</div><div class='author-affiliation'><span class='nlm-sup'>2</span>Present address: Centro de Genómica Nutricional Agroacuícola, Las Heras #350, Temuco, Chile.</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bradley%2BJ.%2BTill%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Till%20BJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ABradley%2BJ.%2BTill%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1300-8285" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Bradley J. Till</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:brad.till@cgna.cl" class="" data-icon-position="" data-hide-link-title="0">brad.till@cgna.cl</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2657&amp;rft.epage=2666&amp;rft.atitle=Fragmentation%20of%20Pooled%20PCR%20Products%20for%20Highly%20Multiplexed%20TILLING&amp;rft.volume=9&amp;rft.issue=8&amp;rft.date=2019-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Tramontano%2C+Andrea&amp;rft.au=Jarc%2C+Luka&amp;rft.au=Jankowicz-Cieslak%2C+Joanna&amp;rft.au=Hofinger%2C+Bernhard+J.&amp;rft.au=Gajek%2C+Katarzyna&amp;rft.au=Szurman-Zubrzycka%2C+Miriam&amp;rft.au=Szarejko%2C+Iwona&amp;rft.au=Ingelbrecht%2C+Ivan&amp;rft.au=Till%2C+Bradley+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/493754/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/493754/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/493754/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/493754/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/493754/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/493754/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/493754/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/493754/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/493754/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/493754/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/493754/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/493754/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/8/2657" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-6" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/8/2657.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/8/2657.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/8/2657.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
