<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/8/2463" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/8/2463.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/8/2463.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/8/2463.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/8/2463/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/8/2463.atom" />
<meta name="HW.pisa" content="ggg;9/8/2463" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic Selection with Allele Dosage in Panicum maximum Jacq." />
<meta name="DC.Identifier" content="10.1534/g3.118.200986" />
<meta name="DC.Date" content="2019-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 de C. Lara et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genomic selection is an efficient approach to get shorter breeding cycles in recurrent selection programs and greater genetic gains with selection of superior individuals. Despite advances in genotyping techniques, genetic studies for polyploid species have been limited to a rough approximation of studies in diploid species. The major challenge is to distinguish the different types of heterozygotes present in polyploid populations. In this work, we evaluated different genomic prediction models applied to a recurrent selection population of 530 genotypes of Panicum maximum , an autotetraploid forage grass. We also investigated the effect of the allele dosage in the prediction, i.e. , considering tetraploid (GS-TD) or diploid (GS-DD) allele dosage. A longitudinal linear mixed model was fitted for each one of the six phenotypic traits, considering different covariance matrices for genetic and residual effects. A total of 41,424 genotyping-by-sequencing markers were obtained using 96-plex and Pst 1 restriction enzyme, and quantitative genotype calling was performed. Six predictive models were generalized to tetraploid species and predictive ability was estimated by a replicated fivefold cross-validation process. GS-TD and GS-DD models were performed considering 1,223 informative markers. Overall, GS-TD data yielded higher predictive abilities than with GS-DD data. However, different predictive models had similar predictive ability performance. In this work, we provide bioinformatic and modeling guidelines to consider tetraploid dosage and observed that genomic selection may lead to additional gains in recurrent selection program of P. maximum ." />
<meta name="DC.Contributor" content="Letícia A. de C. Lara" />
<meta name="DC.Contributor" content="Mateus F. Santos" />
<meta name="DC.Contributor" content="Liana Jank" />
<meta name="DC.Contributor" content="Lucimara Chiari" />
<meta name="DC.Contributor" content="Mariane de M. Vilela" />
<meta name="DC.Contributor" content="Rodrigo R. Amadeu" />
<meta name="DC.Contributor" content="Jhonathan P. R. dos Santos" />
<meta name="DC.Contributor" content="Guilherme da S. Pereira" />
<meta name="DC.Contributor" content="Zhao-Bang Zeng" />
<meta name="DC.Contributor" content="Antonio Augusto F. Garcia" />
<meta name="article:published_time" content="2019-08-01" />
<meta name="article:section" content="Genomic Prediction" />
<meta name="citation_title" content="Genomic Selection with Allele Dosage in Panicum maximum Jacq." />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genomic selection is an efficient approach to get shorter breeding cycles in recurrent selection programs and greater genetic gains with selection of superior individuals. Despite advances in genotyping techniques, genetic studies for polyploid species have been limited to a rough approximation of studies in diploid species. The major challenge is to distinguish the different types of heterozygotes present in polyploid populations. In this work, we evaluated different genomic prediction models applied to a recurrent selection population of 530 genotypes of &lt;i&gt;Panicum maximum&lt;/i&gt;, an autotetraploid forage grass. We also investigated the effect of the allele dosage in the prediction, &lt;i&gt;i.e.&lt;/i&gt;, considering tetraploid (GS-TD) or diploid (GS-DD) allele dosage. A longitudinal linear mixed model was fitted for each one of the six phenotypic traits, considering different covariance matrices for genetic and residual effects. A total of 41,424 genotyping-by-sequencing markers were obtained using 96-plex and &lt;i&gt;Pst&lt;/i&gt;1 restriction enzyme, and quantitative genotype calling was performed. Six predictive models were generalized to tetraploid species and predictive ability was estimated by a replicated fivefold cross-validation process. GS-TD and GS-DD models were performed considering 1,223 informative markers. Overall, GS-TD data yielded higher predictive abilities than with GS-DD data. However, different predictive models had similar predictive ability performance. In this work, we provide bioinformatic and modeling guidelines to consider tetraploid dosage and observed that genomic selection may lead to additional gains in recurrent selection program of &lt;i&gt;P. maximum&lt;/i&gt;.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/08/01" />
<meta name="citation_mjid" content="ggg;9/8/2463" />
<meta name="citation_id" content="9/8/2463" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/8/2463" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/8/2463.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/8/2463.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/8/2463.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200986" />
<meta name="citation_pmid" content="31171567" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="13" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Prediction" />
<meta name="citation_firstpage" content="2463" />
<meta name="citation_lastpage" content="2475" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Letícia A. de C. Lara" />
<meta name="citation_author_institution" content="Luiz de Queiroz College of Agriculture / University of São Paulo (ESALQ/USP), Piracicaba, SP, Brazil," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0593-5003" />
<meta name="citation_author" content="Mateus F. Santos" />
<meta name="citation_author_institution" content="Embrapa Beef Cattle, Campo Grande, MS, Brazil, and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5324-906X" />
<meta name="citation_author" content="Liana Jank" />
<meta name="citation_author_institution" content="Embrapa Beef Cattle, Campo Grande, MS, Brazil, and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9436-3678" />
<meta name="citation_author" content="Lucimara Chiari" />
<meta name="citation_author_institution" content="Embrapa Beef Cattle, Campo Grande, MS, Brazil, and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3030-7532" />
<meta name="citation_author" content="Mariane de M. Vilela" />
<meta name="citation_author_institution" content="Embrapa Beef Cattle, Campo Grande, MS, Brazil, and" />
<meta name="citation_author" content="Rodrigo R. Amadeu" />
<meta name="citation_author_institution" content="Luiz de Queiroz College of Agriculture / University of São Paulo (ESALQ/USP), Piracicaba, SP, Brazil," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5127-4448" />
<meta name="citation_author" content="Jhonathan P. R. dos Santos" />
<meta name="citation_author_institution" content="Luiz de Queiroz College of Agriculture / University of São Paulo (ESALQ/USP), Piracicaba, SP, Brazil," />
<meta name="citation_author" content="Guilherme da S. Pereira" />
<meta name="citation_author_institution" content="North Carolina State University (NCSU), Raleigh, NC" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7106-8630" />
<meta name="citation_author" content="Zhao-Bang Zeng" />
<meta name="citation_author_institution" content="North Carolina State University (NCSU), Raleigh, NC" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3115-1149" />
<meta name="citation_author" content="Antonio Augusto F. Garcia" />
<meta name="citation_author_institution" content="Luiz de Queiroz College of Agriculture / University of São Paulo (ESALQ/USP), Piracicaba, SP, Brazil," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0634-3277" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=I. Aguilar;citation_author=I. Misztal;citation_author=D. L. Johnson;citation_author=A. Legarra;citation_author=S. Tsuruta;citation_title=Hot topic: a unified approach to utilize phenotypic, full pedigree, and genomic information for genetic evaluation of Holstein final score.;citation_pages=743-752;citation_volume=93;citation_year=2010;citation_issue=2;citation_pmid=20105546;citation_doi=10.3168/jds.2009-2730" />
<meta name="citation_reference" content="citation_journal_title=IEEE Trans. Automat. Contr.;citation_author=H. Akaike;citation_title=A new look at the statistical model identification;citation_pages=716-723;citation_volume=19;citation_year=1974;citation_doi=10.1109/TAC.1974.1100705" />
<meta name="citation_reference" content="citation_journal_title=American Journal of Botany;citation_journal_abbrev=Am. J. Bot.;citation_author=S. S. Aliscioni;citation_author=L. M. Giussani;citation_author=F. O. Zuloaga;citation_author=E. A. Kellogg;citation_title=A molecular phylogeny of Panicum (Poaceae: Paniceae): tests of monophyly and phylogenetic placement within the Panicoideae;citation_pages=796-821;citation_volume=90;citation_year=2003;citation_issue=5;citation_pmid=21659176;citation_doi=10.3732/ajb.90.5.796" />
<meta name="citation_reference" content="citation_year=1960" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=R. R. Amadeu;citation_author=C. Cellon;citation_author=J. W. Olmstead;citation_author=A. A. F. Garcia;citation_author=M. F. R. Resende;citation_title=AGHmatrix: R package to construct relationship matrices for autotetraploid and diploid species: A blueberry example;citation_volume=9;citation_year=2016;citation_doi=10.3835/plantgenome2016.01.0009" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=P. Annicchiarico;citation_author=N. Nazzicari;citation_author=X. Li;citation_author=Y. Wei;citation_author=L. Pecetti;citation_title=Accuracy of genomic selection for alfalfa biomass yield in different reference populations;citation_pages=1020;citation_volume=16;citation_year=2015;citation_doi=10.1186/s12864-015-2212-y" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=M. P. Arruda;citation_author=P. J. Brown;citation_author=A. E. Lipka;citation_author=A. M. Krill;citation_author=C. Thurber;citation_title=Genomic selection for predicting Head Blight Resistance in a Wheat Breeding Program;citation_pages=1-12;citation_volume=8;citation_year=2015;citation_doi=10.3835/plantgenome2014.05.0020" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=F. G. Asoro;citation_author=M. A. Newell;citation_author=W. D. Beavis;citation_author=M. P. Scott;citation_author=J.-L. Jannink;citation_title=Accuracy and training population design for genomic selection on quantitative traits in elite North American oats;citation_pages=132-144;citation_volume=4;citation_year=2011;citation_doi=10.3835/plantgenome2011.02.0007" />
<meta name="citation_reference" content="citation_journal_title=Plant Sci.;citation_author=F. M. Bassi;citation_author=A. R. Bentley;citation_author=G. Charmet;citation_author=R. Ortiz;citation_author=J. Crossa;citation_title=Breeding schemes for the implementation of genomic selection in wheat (Triticum spp.);citation_pages=23-36;citation_volume=242;citation_year=2016;citation_doi=10.1016/j.plantsci.2015.08.021" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=J. L. Bennetzen;citation_author=J. Schmutz;citation_author=H. Wang;citation_author=R. Percifield;citation_author=J. Hawkins;citation_title=Reference genome sequence of the model plant Setaria.;citation_pages=555-561;citation_volume=30;citation_year=2012;citation_issue=6;citation_pmid=22580951;citation_doi=10.1038/nbt.2196" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=R. Bernardo;citation_title=Genomewide selection when major genes are known;citation_pages=68-75;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2013.05.0315" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=E. Biazzi;citation_author=N. Nazzicari;citation_author=L. Pecetti;citation_author=E. C. Brummer;citation_author=A. Palmonari;citation_title=Genome-wide association mapping and genomic selection for alfalda (Medicago sativa) forage quality traits;citation_pages=e0169234;citation_volume=12;citation_year=2017;citation_doi=10.1371/journal.pone.0169234" />
<meta name="citation_reference" content="citation_year=2009" />
<meta name="citation_reference" content="citation_journal_abbrev=J Integr Plant Biol;citation_author=L. Cabrera-Bosquet;citation_author=J. Crossa;citation_author=J. von Zitzewitz;citation_author=M. D. Serret;citation_author=J. L. Araus;citation_title=High-throughput phenotyping and genomic selection: the frontiers of crop breeding converge.;citation_pages=312-320;citation_volume=54;citation_year=2012;citation_issue=5;citation_pmid=22420640;citation_doi=10.1111/j.1744-7909.2012.01116.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. P. L. Calus;citation_author=T. H. E. Meuwissen;citation_author=A. P. W. Roos;citation_author=R. F. Veerkamp;citation_title=Accuracy of Genomic Selection Using Different Methods to Define Haplotypes;citation_pages=553-561;citation_volume=178;citation_year=2008;citation_issue=1;citation_pmid=18202394;citation_doi=10.1534/genetics.107.080838" />
<meta name="citation_reference" content="citation_journal_title=J. Agric. Biol. Environ. Stat.;citation_author=B. R. Cullis;citation_author=A. B. Smith;citation_author=N. E. Coombes;citation_title=On the design of early generation variety trials with correlated data;citation_pages=381-393;citation_volume=11;citation_year=2006;citation_doi=10.1198/108571106X154443" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. D. Daetwyler;citation_author=M. P. L. Calus;citation_title=Whole-genome regression and prediction methods applied to plant and animal breeding;citation_volume=193;citation_year=2013;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_journal_title=Mol. Breed.;citation_author=A. A. de Oliveira;citation_author=M. M. Pastina;citation_author=R. A. da Costa Parrella;citation_author=R. W. Noda;citation_author=M. L. F. Simeone;citation_title=Genomic prediction applied to high-biomass sorghum for bioenergy production;citation_pages=49;citation_volume=38;citation_year=2018;citation_doi=10.1007/s11032-018-0802-5" />
<meta name="citation_reference" content="citation_journal_title=BMC Genet.;citation_author=J. P. R. Dos Santos;citation_author=L. P. M. Pires;citation_author=R. C. C. Vasconcellos;citation_author=G. S. Pereira;citation_author=R. G. V. Pinho;citation_title=Genomic selection to resistance to Stenocarpella maydis in maize lines using DArTseq markers;citation_pages=86;citation_volume=17;citation_year=2016;citation_doi=10.1186/s12863-016-0392-3" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=R. J. Elshire;citation_author=J. C. Glaubitz;citation_author=Q. Sun;citation_author=J. A. Poland;citation_author=K. Kawamoto;citation_title=A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species.;citation_pages=e19379-e19379;citation_volume=6;citation_year=2011;citation_issue=5;citation_pmid=21573248;citation_doi=10.1371/journal.pone.0019379" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. B. Endelman;citation_author=C. A. S. Carley;citation_author=P. C. Bethke;citation_author=J. J. Coombs;citation_author=M. E. Clough;citation_title=Genetic variance partitioning and genome-wide prediction with allele dosage information in autotetraploid potato;citation_volume=209;citation_year=2018;citation_pmid=29514860;citation_doi=10.1534/genetics.118.300685" />
<meta name="citation_reference" content="citation_title=Quantitative genetics in maize breeding;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=L. F. V. Ferrão;citation_author=R. G. Ferrão;citation_author=M. A. G. Ferrão;citation_author=A. F. A. Fonseca;citation_author=A. A. F. Garcia;citation_title=A mixed model to multiple harvest-location trials applied to genomic prediction in Coffea canephora;citation_pages=95;citation_volume=13;citation_year=2017;citation_doi=10.1007/s11295-017-1171-7" />
<meta name="citation_reference" content="citation_title=Quantitative genetics and breeding methods in autopolyploid plants;citation_year=2003" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=G. Galli;citation_author=D. H. Lyra;citation_author=F. C. Alves;citation_author=S. C. Granato;citation_author=R. Fritsche-Neto;citation_title=Impact of phenotypic correction method and missing phenotypic data on genomic prediction of maize hybrids;citation_pages=1481-1491;citation_volume=58;citation_year=2018;citation_doi=10.2135/cropsci2017.07.0459" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=A. A. F. Garcia;citation_author=M. Mollinari;citation_author=T. G. Marconi;citation_author=O. R. Serang;citation_author=R. R. Silva;citation_title=SNP genotyping allows an in-depth characterisation of the genome of sugarcane and other complex autopolyploids;citation_pages=3399;citation_volume=3;citation_year=2013;citation_pmid=24292365;citation_doi=10.1038/srep03399" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gerard;citation_author=L. F. V. Ferrão;citation_author=A. A. F. Garcia;citation_author=M. Stephens;citation_title=Genotyping polyploids from messy sequencing data;citation_volume=210;citation_year=2018;citation_pmid=30185430;citation_doi=10.1534/genetics.118.301468" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=L. Gidskehaug;citation_author=M. Kent;citation_author=B. J. Hayes;citation_author=S. Lien;citation_title=Genotype calling and mapping of multisite variants using an Atlantic salmon iSelect SNP array;citation_volume=27;citation_year=2010;citation_pmid=21149341;citation_doi=10.1093/bioinformatics/btq673" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. C. Glaubitz;citation_author=T. M. Casstevens;citation_author=F. Lu;citation_author=J. Harriman;citation_author=R. J. Elshire;citation_title=TASSEL-GBS: A high capacity genotyping by sequencing analysis pipeline;citation_pages=e90346;citation_volume=9;citation_year=2014;citation_pmid=24587335;citation_doi=10.1371/journal.pone.0090346" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. M. González-Camacho;citation_author=G. De los Campos;citation_author=P. Pérez;citation_author=D. Gianola;citation_author=J. E. Cairns;citation_title=Genome-enabled prediction of genetic values using radial basis function neural networks.;citation_pages=759-771;citation_volume=125;citation_year=2012;citation_issue=4;citation_pmid=22566067;citation_doi=10.1007/s00122-012-1868-9" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=D. M. Goodstein;citation_author=S. Shu;citation_author=R. Howson;citation_author=R. Neupane;citation_author=R. D. Hayes;citation_title=Phytozome: a comparative platform for green plant genommics;citation_volume=40;citation_year=2011;citation_pmid=22110026;citation_doi=10.1093/nar/gkr944" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. Gouy;citation_author=Y. Rousselle;citation_author=D. Bastianelli;citation_author=P. Lecomte;citation_author=L. Bonnal;citation_title=Experimental assessment of the accuracy of genomic selection in sugarcane;citation_pages=2575-2586;citation_volume=126;citation_year=2013;citation_doi=10.1007/s00122-013-2156-z" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=Z. Gu;citation_author=L. Gu;citation_author=R. Eils;citation_author=M. Schlesner;citation_author=B. Brors;citation_title=circlize implements and enhances circular visualization in R;citation_volume=30;citation_year=2014;citation_pmid=24930139;citation_doi=10.1093/bioinformatics/btu393" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=K. Kizilkaya;citation_author=D. J. Garrick;citation_title=Extension of the bayesian alphabet for genomic selection.;citation_pages=186-186;citation_volume=12;citation_year=2011;citation_pmid=21605355;citation_doi=10.1186/1471-2105-12-186" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=C. A. Hackett;citation_author=Z. W. Luo;citation_title=TetraploidMap: Construction of a Linkage Map in Autotetraploid Species;citation_pages=358-359;citation_volume=94;citation_year=2003;citation_issue=4;citation_pmid=12920109;citation_doi=10.1093/jhered/esg066" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=C. A. Hackett;citation_author=K. McLean;citation_author=G. Bryan;citation_title=Linkage analysis and QTL mapping using SNP dosage data in a tetraploid potato mapping population;citation_pages=e63939;citation_volume=8;citation_year=2013;citation_pmid=23704960;citation_doi=10.1371/journal.pone.0063939" />
<meta name="citation_reference" content="citation_journal_title=Maydica;citation_author=A. R. Hallauer;citation_author=M. J. Carena;citation_title=Recurrent selection methods to improve germplasm in maize;citation_pages=266-283;citation_volume=57;citation_year=2012" />
<meta name="citation_reference" content="citation_title=Quantitative genetics in maize breeding;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=E. L. Heffner;citation_author=A. Lorenz;citation_author=J.-L. Jannink;citation_author=M. E. Sorrells;citation_title=Plant breeding with genomic selection: gain per unit time and cost;citation_pages=1681-1690;citation_volume=50;citation_year=2010;citation_doi=10.2135/cropsci2009.11.0662" />
<meta name="citation_reference" content="citation_journal_title=Crop Science;citation_journal_abbrev=Crop Sci.;citation_author=E. L. Heffner;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Genomic Selection for Crop Improvement;citation_pages=1-12;citation_volume=49;citation_year=2009;citation_issue=1;citation_doi=10.2135/cropsci2008.08.0512" />
<meta name="citation_reference" content="citation_title=Genetic data analysis for plant and animal breeding;citation_author=F. Isik;citation_author=J. Holland;citation_author=C. Maltecca;citation_year=2017;citation_doi=10.1007/978-3-319-55177-7" />
<meta name="citation_reference" content="citation_journal_title=Crop Pasture Sci.;citation_author=L. Jank;citation_author=S. C. Barrios;citation_author=C. B. Valle;citation_author=R. M. Simeão;citation_author=G. F. Alves;citation_title=The value of improved pastures to Brazilian beef production;citation_pages=1132-1137;citation_volume=65;citation_year=2014;citation_doi=10.1071/CP13319" />
<meta name="citation_reference" content="citation_journal_title=Crop Breed. Appl. Biotechnol.;citation_author=L. Jank;citation_author=C. B. Valle;citation_author=R. M. S. Resende;citation_title=Breeding tropical forages;citation_pages=27-34;citation_volume=11;citation_year=2011;citation_doi=10.1590/S1984-70332011000500005" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=A. E. Lipka;citation_author=F. Lu;citation_author=J. H. Cherney;citation_author=E. S. Buckler;citation_author=M. D. Casler;citation_title=Accelerating the switchgrass (Panicum virgatum L.) breeding cycle using genomic selection approaches;citation_pages=e112227;citation_volume=9;citation_year=2014;citation_doi=10.1371/journal.pone.0112227" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=J. M. Massman;citation_author=H.-J. G. Jung;citation_author=R. Bernardo;citation_title=Genomewide selection vs. marker-assisted recurrent selection to improve grain yield and stover-quality traits for cellulosic ethanol in maize;citation_pages=58-66;citation_volume=53;citation_year=2013;citation_doi=10.2135/cropsci2012.02.0112" />
<meta name="citation_reference" content="citation_journal_title=Crop Breed. Appl. Biotechnol.;citation_author=F. I. Matias;citation_author=S. C. L. Barrios;citation_author=C. B. Valle;citation_author=R. G. Mateus;citation_author=L. B. Martins;citation_title=Estimate of genetic parameters in Brachiaria decumbens hybrids;citation_pages=115-122;citation_volume=16;citation_year=2016;citation_doi=10.1590/1984-70332016v16n2a18" />
<meta name="citation_reference" content="citation_title=Alfalfa and alfalfa improvement;citation_author=T. J. McCoy;citation_author=E. T. Bingham;citation_title=Cytology and cytogenetics of alfalfa;citation_pages=737-776;citation_volume=Vol. 29;citation_year=1988" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_title=Methods in Molecular Biology;citation_title=Quantitative SNP genotyping of polyploids with MassARRAY and other platforms;citation_pages=215-241;citation_year=2013" />
<meta name="citation_reference" content="citation_author=D. Müller;citation_author=P. Schopp;citation_author=A. E. Melchinger;citation_year=2017;citation_pmid=28064189;citation_doi=10.1534/g3.116.036582" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=T. C. Osborn;citation_author=J. Chris Pires;citation_author=J. A. Birchler;citation_author=D. L. Auger;citation_author=Z. Jeffery Chen;citation_title=Understanding mechanisms of novel gene expression in polyploids.;citation_pages=141-147;citation_volume=19;citation_year=2003;citation_issue=3;citation_pmid=12615008;citation_doi=10.1016/S0168-9525(03)00015-5" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=T. Park;citation_author=G. Casella;citation_title=The Bayesian Lasso;citation_pages=681-686;citation_volume=103;citation_year=2008;citation_doi=10.1198/016214508000000337" />
<meta name="citation_reference" content="citation_journal_title=Rev. Ceres;citation_author=E. A. Pereira;citation_author=M. DallAgnol;citation_author=K. M. Saraiva;citation_author=C. Simioni;citation_author=A. P. S. Leães;citation_title=Genetic gain in apomictic species of the genus Paspalum;citation_pages=60-67;citation_volume=64;citation_year=2017;citation_doi=10.1590/0034-737x201764010009" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=G. S. Pereira;citation_author=A. A. F. Garcia;citation_author=G. R. A. Margarido;citation_title=A fully automated pipeline for quantitative genotype calling from next generation sequencing data in autopolyploids;citation_pages=398;citation_volume=19;citation_year=2018;citation_doi=10.1186/s12859-018-2433-6" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Pérez;citation_author=G. de los Campos;citation_title=Genome-wide regression and prediction with the BGLR statistical package;citation_volume=198;citation_year=2014;citation_pmid=25009151;citation_doi=10.1534/genetics.114.164442" />
<meta name="citation_reference" content="citation_year=1975" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=R. M. S. Resende;citation_author=M. D. Casler;citation_author=M. D. V. de Resende;citation_title=Genomic selection in forage breeding: Accuracy and methods;citation_pages=143-156;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2013.05.0353" />
<meta name="citation_reference" content="citation_journal_title=Crop Sci.;citation_author=R. M. S. Resende;citation_author=L. Jank;citation_author=C. B. d. Valle;citation_author=A. L. V. Bonato;citation_title=Biometrical analysis and selection of tetraploid progenies of Panicum maximum using mixed model methods;citation_pages=143-156;citation_volume=54;citation_year=2014;citation_doi=10.2135/cropsci2013.05.0353" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=R. T. Resende;citation_author=M. D. V. Resende;citation_author=F. F. Silva;citation_author=C. F. Azevedo;citation_author=E. K. Takahashi;citation_title=Assessing the expected response to genomic selection of individuals and families in Eucalyptus breeding with an additive-dominant model;citation_pages=245-255;citation_volume=119;citation_year=2017;citation_doi=10.1038/hdy.2017.37" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. Riedelsheimer;citation_author=J. B. Endelman;citation_author=M. Stange;citation_author=M. E. Sorrells;citation_author=J.-L. Jannink;citation_title=Genomic predictability of interconnected bi-parental maize populations;citation_volume=194;citation_year=2013;citation_pmid=23535384;citation_doi=10.1534/genetics.113.150227" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=C. Riedelsheimer;citation_author=F. Technow;citation_author=A. E. Melchinger;citation_title=Comparison of whole-genome prediction models for traits with contrasting genetic architecture in a diversity panel of maize inbred lines.;citation_pages=452-452;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=22947126;citation_doi=10.1186/1471-2164-13-452" />
<meta name="citation_reference" content="citation_year=1978" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=C. A. Schmitz Carley;citation_author=J. J. Coombs;citation_author=D. S. Douches;citation_author=P. C. Bethke;citation_author=J. P. Palta;citation_title=Automated tetraploid genotype calling by hierarchical clustering;citation_pages=717-726;citation_volume=130;citation_year=2017;citation_doi=10.1007/s00122-016-2845-5" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Schopp;citation_author=D. Müller;citation_author=F. Technow;citation_author=A. E. Melchinger;citation_title=Accuracy of genomic prediction in synthetic populations depending on the number of parents, relatedness, and ancestral linkage disequilibrium;citation_volume=205;citation_year=2017;citation_pmid=28049710;citation_doi=10.1534/genetics.116.193243" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=T. Schulz-Streeck;citation_author=J. O. Ogutu;citation_author=H.-P. Piepho;citation_title=Comparisons of single-stage and two stage approaches to genomic selection;citation_pages=69-82;citation_volume=126;citation_year=2013;citation_pmid=22903736;citation_doi=10.1007/s00122-012-1960-1" />
<meta name="citation_reference" content="citation_journal_title=Ann. Stat.;citation_author=G. Schwarz;citation_title=Estimating a dimension of a model;citation_pages=461-464;citation_volume=6;citation_year=1978;citation_doi=10.1214/aos/1176344136" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=O. Serang;citation_author=M. Mollinari;citation_author=A. A. F. Garcia;citation_title=Efficient exact maximum a posteriori computation for bayesian SNP genotyping in polyploids.;citation_pages=e30906-e30906;citation_volume=7;citation_year=2012;citation_issue=2;citation_pmid=22363513;citation_doi=10.1371/journal.pone.0030906" />
<meta name="citation_reference" content="citation_journal_title=Plant Genome;citation_author=A. T. Slater;citation_author=N. O. I. Cogan;citation_author=J. W. Forster;citation_author=B. J. Hayes;citation_author=H. D. Daetwyler;citation_title=Improving genetic gain with genomic selection in autotetraploid potato;citation_volume=9;citation_year=2016;citation_doi=10.3835/plantgenome2016.02.0021" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. Spindel;citation_author=H. Begum;citation_author=D. Akdemir;citation_author=P. Virk;citation_author=B. Collard;citation_title=Genomic selection and association mapping in rice (Oryza sativa): Effect of trait, genetic architecture, training population composition, marker number and statistical model on accuracy of rice genomic selection in elite, tropical rice breeding line;citation_pages=e1004982;citation_volume=11;citation_year=2015;citation_doi=10.1371/journal.pgen.1004982" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=B. Stich;citation_author=D. V. Inghelandt;citation_title=Prospects and potential uses of genomic prediction of key performance traits in tetraploid potato;citation_pages=159;citation_volume=9;citation_year=2018;citation_doi=10.3389/fpls.2018.00159" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=E. Sverrisdóttir;citation_author=S. Byrne;citation_author=E. H. R. Sundmark;citation_author=H. O. Johnsen;citation_author=K. H. Grethe;citation_title=Genomic prediction of starch content and chipping quality in tetraploid potato using genotyping-by-sequencing;citation_pages=2091-2108;citation_volume=130;citation_year=2017;citation_doi=10.1007/s00122-017-2944-y" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=G. Toledo-Silva;citation_author=C. B. Cardoso-Silva;citation_author=L. Jank;citation_author=A. P. Souza;citation_title=De Novo transcriptome assembly for the tropical grass Panicum maximum Jacq;citation_pages=e70781;citation_volume=8;citation_year=2013;citation_pmid=23923022;citation_doi=10.1371/journal.pone.0070781" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=J. G. A. M. L. Uitdewilligen;citation_author=A.-M. A. Wolters;citation_author=B. B. D’hoop;citation_author=T. J. A. Borm;citation_author=R. G. F. Visser;citation_title=A next-generation sequencing method for genotyping-by-sequencing of highly heterozygous autotetraploid potato;citation_pages=1-14;citation_volume=8;citation_year=2013;citation_pmid=23620825;citation_doi=10.1371/journal.pone.0054583" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=P. M. VanRaden;citation_title=Efficient Methods to Compute Genomic Predictions;citation_pages=4414-4423;citation_volume=91;citation_year=2008;citation_issue=11;citation_pmid=18946147;citation_doi=10.3168/jds.2007-0980" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=R. E. Voorrips;citation_author=G. Gort;citation_author=B. Vosman;citation_title=Genotype calling in tetraploid species from bi-allelic marker data using mixture models.;citation_pages=172-172;citation_volume=12;citation_year=2011;citation_pmid=21595880;citation_doi=10.1186/1471-2105-12-172" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=P. G. Vos;citation_author=M. João Paulo;citation_author=R. E. Voorips;citation_author=R. G. F. Visser;citation_author=H. J. van Eck;citation_title=Evaluation of LD decay and various LD-decay estimators in simulated and SNP-array data of tetraploid potato;citation_pages=123-135;citation_volume=130;citation_year=2017;citation_doi=10.1007/s00122-016-2798-8" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Bot.;citation_author=H. E. Warmke;citation_title=Apomixis in Panicum maximum;citation_pages=5-11;citation_volume=41;citation_year=1954;citation_doi=10.2307/2438575" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=J. C. Whittaker;citation_author=R. Thompson;citation_author=M. C. Denham;citation_title=Marker-assisted selection using ridge regression.;citation_pages=249-252;citation_volume=75;citation_year=2000;citation_issue=2;citation_pmid=10816982;citation_doi=10.1017/S0016672399004462" />
<meta name="citation_reference" content="citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=K. K. Wu;citation_author=W. Burnquist;citation_author=M. E. Sorrells;citation_author=T. L. Tew;citation_author=P. H. Moore;citation_title=The detection and estimation of linkage in polyploids using single-dose restriction fragments;citation_pages=294-300;citation_volume=83;citation_year=1992;citation_pmid=24202510;citation_doi=10.1007/BF00224274" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=J. Yang;citation_author=B. Benyamin;citation_author=B. P. McEvoy;citation_author=S. Gordon;citation_author=A. K. Henders;citation_title=Common SNPs explain a large proportion of the heritability for human height.;citation_pages=565-569;citation_volume=42;citation_year=2010;citation_issue=7;citation_pmid=20562875;citation_doi=10.1038/ng.608" />
<meta name="citation_reference" content="citation_author=X. Zhang;citation_author=P. Pérez-Rodríguez;citation_author=J. Burgueño;citation_author=M. Olsen;citation_author=E. Buckler;citation_year=2017;citation_pmid=28533335;citation_doi=10.1534/g3.117.043141" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic Selection with Allele Dosage in Panicum maximum Jacq." />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8.cover-source.jpg" />
<meta name="twitter:description" content="Genomic selection is an efficient approach to get shorter breeding cycles in recurrent selection programs and greater genetic gains with selection of superior individuals. Despite advances in genotyping techniques, genetic studies for polyploid species have been limited to a rough approximation of studies in diploid species. The major challenge is to distinguish the different types of heterozygotes present in polyploid populations. In this work, we evaluated different genomic prediction models applied to a recurrent selection population of 530 genotypes of Panicum maximum , an autotetraploid forage grass. We also investigated the effect of the allele dosage in the prediction, i.e. , considering tetraploid (GS-TD) or diploid (GS-DD) allele dosage. A longitudinal linear mixed model was fitted for each one of the six phenotypic traits, considering different covariance matrices for genetic and residual effects. A total of 41,424 genotyping-by-sequencing markers were obtained using 96-plex and Pst 1 restriction enzyme, and quantitative genotype calling was performed. Six predictive models were generalized to tetraploid species and predictive ability was estimated by a replicated fivefold cross-validation process. GS-TD and GS-DD models were performed considering 1,223 informative markers. Overall, GS-TD data yielded higher predictive abilities than with GS-DD data. However, different predictive models had similar predictive ability performance. In this work, we provide bioinformatic and modeling guidelines to consider tetraploid dosage and observed that genomic selection may lead to additional gains in recurrent selection program of P. maximum ." />
<meta name="og-title" property="og:title" content="Genomic Selection with Allele Dosage in Panicum maximum Jacq." />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/8/2463" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genomic selection is an efficient approach to get shorter breeding cycles in recurrent selection programs and greater genetic gains with selection of superior individuals. Despite advances in genotyping techniques, genetic studies for polyploid species have been limited to a rough approximation of studies in diploid species. The major challenge is to distinguish the different types of heterozygotes present in polyploid populations. In this work, we evaluated different genomic prediction models applied to a recurrent selection population of 530 genotypes of Panicum maximum , an autotetraploid forage grass. We also investigated the effect of the allele dosage in the prediction, i.e. , considering tetraploid (GS-TD) or diploid (GS-DD) allele dosage. A longitudinal linear mixed model was fitted for each one of the six phenotypic traits, considering different covariance matrices for genetic and residual effects. A total of 41,424 genotyping-by-sequencing markers were obtained using 96-plex and Pst 1 restriction enzyme, and quantitative genotype calling was performed. Six predictive models were generalized to tetraploid species and predictive ability was estimated by a replicated fivefold cross-validation process. GS-TD and GS-DD models were performed considering 1,223 informative markers. Overall, GS-TD data yielded higher predictive abilities than with GS-DD data. However, different predictive models had similar predictive ability performance. In this work, we provide bioinformatic and modeling guidelines to consider tetraploid dosage and observed that genomic selection may lead to additional gains in recurrent selection program of P. maximum ." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8.cover-source.jpg" />
    <title>Genomic Selection with Allele Dosage in Panicum maximum Jacq. | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"FE_g38CIdBu9jlLyBKuijRRRjSuUuIl2alqHAhFPaig","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"493738","apath":"\/ggg\/9\/8\/2463.atom","pisa":"ggg;9\/8\/2463","ac":{"\/ggg\/9\/8\/2463.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/8\/2463.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/8\/2463"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/8\/2463"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/8\/2463":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00220hdXjzlrScKuPBJAUD4Is8tn69FBzXgj5ijV9f52Mwbcb\\\\\\\/i5j5VFxQoUIDP2law8RlBOofyQ6jZMr1gw\\\\\\\/97OZGcyzAUXS1vM0TZemTPKLzr9mO8CB2s6fJisQ1ter+hpUtNRE19I9FeWs6ra2PGG3UliK2xiJdKjK0hGAlM8KaRlh+XBHne\\\\\\\/QdwJDAknxg2UAePxsKc43l3liqOVVWf\\\\\\\/oEiDqGug4+iY41SL0E5nNFFXWBjrS5nLLuKIH2VZr8e7ALTIvNM2jknD7u2pqJvH41oFUE7Ve7ynx98pOX32j4ksb1mT+4wnJQd45zo6PlZ9RkuJk1lSESz6W\\\\\\\/l4bJwOcIX\\\\\\\/7f\\\\\\\/aFRxHzHG18ecpxA\\\\\\\/mfG6uZv7vcs0FHikDH7VBQY9lvEZy3JwlV88x+ssc2eCS5g1PT6RfeX1T+wj2oV6cd3C6g0GXnIM4mg2MsmUBIDxHY3WVC3KQfN3jPDrrYCvmW2OXByOi\\\\\\\/eP+RR09eHe7t+4DzOA\\\\\\\/jGMbtpjgyQFwRzkk0t5nQkBZsUria8wGVhmnmLd0qeqrfI0uwgwfT7wYdEdOCy42ixGq58WR6fZI1VUduwolSVNS7nv4708Jywu2BlnczyVZeFAq+xDIa8MNGag\\\\\\\/JlQYYX6foBchbcp78ZbnCDb9wH41V0Cu4Md9wWW24MbPPRcS+rPPCNYaVFxxDWOkPhhk1f9XR1Nl41RKcC7SZ8Xg6sdL7g0HogjhUYS05lXA5b9ammnd53UNTtzrxP8RypZFj+mmJymum4snZnxQHs4L1M+YU7EyxEyxQ6QhWWxrcFL24SO0o9MZs82rrFZUZc3caXq8+12MtMoCkuB5ZZUwdvzoR2JdpdPh941VYQFI5PxZBaKdfVKhcoTP3mnLlDRQPViXjedH6Bphtn0XP5ZVdlU\\\\\\\/wejamvEZ70xrlJrf+B7FkANgnIp00fqfbRS\\\\\\\/+2qGzdHlx+sjngdk\\\\\\\/4M1ZXRX4BrSRpwM1ROdRMg7kG9Y66hzIO\\\\\\\/FrQnnWY\\\\\\\/3h7XXWtwt1XLzdaKfc02cHHUdbQmXXPBLKd9S5empdXYaWi\\\\\\\/hvMxw5g4bxREgwNCLeCVNgFCjZRrTxk3fvJWrEwEl8PuASC94qBvukCLS8a1azz4YSAyMB99i0r+IMtuDErULjcvu+bqikA006p\\\\\\\/TQX0P2XhK404UeEkQM3eU5KJnYw\\\\\\\/bL\\\\\\\/jCu+Z7AvGOoK+IEFYjS8DpnsagSM38ZRsYmEpSWWcX+PCrbT35p4PJEdJL+Nqo\\\\\\\/R7tOzI4fZVLgkYTjt4ByC4InlzjHoSKSX76\\\u0022,\\\u0022iv\\\u0022:\\\u0022a9xtcR7iOIFyHZklgSuzXw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f7972b03928e915232e73158fedd019106aa4b642cd77b0beefc8c394b28857e\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022wWNoLa0zn2A8YD6aKvS8lOPxbUKcLm4qtdkYHl16p4\\\\\\\/P\\\\\\\/4NMSJUF++hDekK0X1\\\\\\\/OpgThPlzrXa2bwk4XOPkuB26ItnaDsK8\\\\\\\/HEroRarEXWC\\\\\\\/b9wa6F6Nd56Z1S2SnAJMIYFd9eL9vpTRAa0sxgKcpbVjWOZNPvJQQvzjlyJK2WuLXp0P7FLJWooF08ue2PAZckxEATmn9MIune82Pz8wic23g9GvpS7zNto7LKFdPnuTGP1kqZubvLKdxq8frd2dBVwiRgPkCbJ6++MgvjPGI04ik6r1yFEUJph7dfuO81grt0eUUf\\\\\\\/oUG6lySGqEe7dQPJnfJG0+z\\\\\\\/LAxC\\\\\\\/yIdwGs3xQroueQp+wCONMjR067WwIIEUCMyR1oOzKUu6eyEW3BouIzkBW3lUNMLzogmraeANHdB7ONwAfx2PtjNqDlKwWqeFz+w2HODgL0TaBeJbfD+x31jEjtnIqecuN1IKIrZmv\\\\\\\/owzej\\\\\\\/Ni4W3g2xnRa+qMLpwc6gRiWHYAMq4eYJgynUIL43OfBx+7u6B8a3Lq1PfifCHxnHnK5TeaThTiaEClPPoYraTEUTcnGFUx6AyHs+\\\\\\\/4Y89V5UZwbBDkuIi8eOjAJPpfM4TtL3l+St4MxQDnj1OURr6\\\\\\\/19E\\\\\\\/GnPu4pBsioYWZpdqHtFPT6Wr5ixa571jIlPV36WMJRdIHeO29hJXtE0U9Sgh6JOcP4g4wfogkJYgklTwdwOHsF5PZkPStEZpgVrI\\\\\\\/0DpkzOzl4H28yYQPoBPlO443A658dG4sjMJdhfQ2XXgKPI9YWisSU2mCKA3rZETOJE2ICqnmGSm4vCtrqVwOlWTL+WOLqCTWN3uVN670jMMdKHScs2\\\\\\\/rd0UHOslMDW2XAMakNl5BbO6WugyA5ol2ed2HEqyN\\\\\\\/miKLOUF21gTLv69X3XrNGnBRBl3sVjVQSrhoqaRyOg+cbESx16FdRSGvDcgdpcEjdwJOrUEKnJIm4sXD48U9kTJvuD98+W0lx+3zImyjNNeJu8E5K8QFREpyQohO+drAWE+Ebh4\\\\\\\/lm89MO8UCAkGvfO6VYvk6STkHf+hQ928\\\\\\\/AdKrBQmtQfgwsG+GztFvNQ+UqItJNqHooN3Wlnlujzjo1pymRrrjL3OJgaqbAO3mq1vqRKxqhvmImfsYG1hUMbnLspsnuXFPYIRRbhtnL2pfEBUQMq7Q4HoWFXbjLzkJrxUPLF1jtbnNpumRj5SIuZkVUxqPD4LKcEaedZKMssObZghvwJXl0zi80G2FsMR9JXwU3+9mNVonHZGv\\\\\\\/iSLarTp6zinLIgXLmHNr79Gyu9CK\\\\\\\/s9m\\\\\\\/oqZ69vkckpMJSy1hs0QjACn\\\\\\\/mTk2IKOB4LlTnqYO8jncKxozdGs6Ul2sBbfBz9NF8v6wFwNnlwDVQ5dFreFEQ7PkQnv6YvB6zLGqJFhOfJ9iyBXm8PXPjJ7OIHmIvjlDU2UH3c50hFCL97MQ4b414Luo233mAl1CigeYqHW7dE3NRTDI9xzQcbenqO3xfhZecmTVg8yYxKJ7pWyJ8cHGiXvOPtxYhUAHa7DuFEWmzszZi5syOFjxrb9LTTLIIioNGmfTxT21wW+4oLCPpn2beXJUocpGi73iMM2As8f8xByrCapozT3tSozAEuZz7\\\\\\\/3kZukYxuw28hz5Kh0Ekm1JiuA8ZAiQ4Hn3b2jR9BJCQdaVifJW6NLtk3PQ+aaq3BEMRE6kSSQQcAU\\\\\\\/dd\\\\\\\/pLP\\\\\\\/mI9O+4VHHWgImzMTXIyLFvSh2uRxmzdzCsQB7oSBDUH294Lnsy1ZOilA6k\\\\\\\/fiSK9n84TopPiz1WMx6219kO3sVE2M20vC5jV7ac948gQYMs4JoXW7xpcPDiwd\\\\\\\/db0LVLoWAZv1nfySlvyUStFwDvMLFQthE7W6rTWUhYZfVx7VLKCL11TofI7DSxr+LnS2h9qiyRb+yvcIv+6d68x5l53fr8ExjOCGwDlp9F1Cg5FTPt\\\\\\\/MX3iGBmKvLTDt757gxSis87+W8\\\\\\\/v\\\\\\\/oCml\\\\\\\/slnHBUsFqoNbkyh5laQXPkul1YlCGUxcdsd1YSSn2x1GrAfOi677atDCTIwRZI2shXuqK09RmBPBEvqNve9ca12TOK7hflWnMX7Ns\\\\\\\/oQypsEHPlIiSsxcB6tUYxW86+gqYDgI0pz51BR7uwSs2+Rd0YiyFIMzdm9\\\\\\\/s3FFROCuFMxy+yv3v8gq\\\\\\\/SH3mmVjsj6LgWfd6\\\\\\\/kOtcUEivuJ93FbOAzONRsPlTdt1\\\\\\\/Hul64GUPq+IjPMxhZsa7rkPs+ocl66W5QxXUrYeb1A52uv5aDSh8DQHi8b1e5gTxnf5I1LfZNN68JTCjKFroieQUCBQhxY7RyTLfAtln1Qf3Gnhk78s8uoEn0HOsqJB4p+hxFYpVWQuq04c2DvaLgAO8jGvY8LYm0lNc9U+wJNurnJiBLyt4aECIJdVMxQGs+V15d1gmOwXG88nVtAp5ZudUSBk2u8hXW2\\\\\\\/CdlpT12oqPFIVcX53gX9vDcDR\\\\\\\/QEQ\\\\\\\/XKoQB++2b2qeB7HqebJmz79WSHc7r81s1mze\\\\\\\/pQty+OSP+rWzrsLFLgDoYVILkOvoYS3VUwsE40s\\\\\\\/12DYxexAtemVWb93gLnoI\\\\\\\/+s8Wk29HvW9nZ144fis\\\\\\\/qQEqPK7pltI1xc0RV\\\\\\\/J7JUsVsLYx0WLE4EYTh7oelQKQMrSn1VHNip\\\\\\\/iLeg3eHS8mNvGb6aT9jSGqNJUUYfMT5NYSlzXRWOL4fPO0ItGwC8q66xFiVlbt67KXADmR4c5EdtVasbsz80yqH84IudHZegQ8+vaXGb9JuHtaSbJftchLmH5irpITXiivWpYrOw5CyUHUieh+zqsSnjOV6F+UEaHatzdNlNmdSWgPMANerV4w3srFa4ilI510Sj+gElmyiVDH+VV5ofxslcutrbR53Z\\\\\\\/ayWKgIMi869gp97yFaMW2i0VRP1Fq+QYS07BFg\\\\\\\/YBE8dpSXkC9BtUNVsg6Sefq5JfRx8ZrgeZ+QhZYN1\\\\\\\/ZF6N+A9zDU6JL130yuSRx5OrvUxKEn+q6mSiaXZ6w15BhYTrYhWG40l\\\\\\\/20ad0T2X+5+Vfcpx2Eyk2HgJ7XYXg1aEG3duKPzghJneyTIPoYVkcYUhQOGUTf17pOQi6IU49dCCbmNY0eHPufTSugHTcHLMqO14eaTbudhBImASyAWqaiYCcQ6BiABcq0l7wdOnWURU7Nxx0xjH9aeJcEWjf9XsO1b6GWW5feDPBy+CPbZBP8rwu1rAA6xhhPUlkyvb2jXAHIE\\\\\\\/zEVJAYBY\\\\\\\/a9c8Sj0Q121HFqgbcwgHCJr2Bde\\\\\\\/\\\\\\\/znK\\\\\\\/OiuG66YtyodzHLDqHbe0iYAr9V3khSEumXmatWu14BPOz3cQsIL19n\\\\\\\/FY9gIED3mtAI4nqwGdp8GGjpY8Stj0UO6JkXsiRj\\\\\\\/8g+HDpw1NFGMCQI+0r7rP0g6Ydh1Tc9LNOOAgs+xExrhFDopgpTXjJmmgnkeEq\\\\\\\/67uaAxGzEaUwnPmAzOHjIjHWjfqUySI9s2PmiAeDUziHa3Wkh40rWLOFeZu2Flu7SSytU6VXLcvusc1mzectwLTyfIgqX7P+9sQMru2JiolG057zoMSkuMwirpv3xTMq3nedge2RscRLinJw0tV9sqVKLHYdYzRoVCUgWx4mlAHpyeRFyuD25rZkmbb0qFwDPS+nICJxrq4lG2Op62NyPy6TFsZBMm1REJY5VOHKTZ2bOeAKrCI3RbAQ79CkJGjcmGbNxZtV1RQyk58r3dx2WZ06RA1TbjCg6E5IQtDd1E3JPOVcqtlZAgxvtOsFcaNKdS2dnK8Rp8NAIBZ9TQAJersfHG3UrPpEDwHwbwzDvhWxjByym8LM\\\\\\\/GdXDAnRMr0S4AoeHzOTXxZRvaS+g6\\\\\\\/0qwAUgi+W4BYe245du5ZWWNVwf0Z\\\\\\\/t\\\\\\\/wBe6GTutjB3VWOuoB2\\\\\\\/vicuMrjC4M0Xb+BUltIWAg9s5PbqYKzRioGE7cV0ENCjAvNBDQ1k5j3siH+CUTs3sZs2pQFCxW3lAAUkvp8JO0mpqnSKoiygmCHIwj9RvgmXKHfB7gV80EsZIurBEOu59y6lmZmpQqJWTRWep84T+ahNdPAMNbfHin1QZ8Ruan\\\\\\\/lFfLZgAUV+NuIVMxeHB8XcniV1ozylaBO4qhdeOq6yB1DSaisE6ENFLFtltPU66B+8GszIyA4vGE55AeiPIDwx\\\\\\\/GOI9jngzr\\\\\\\/1WuHKthA0xxbTsBXVuVZItiq2Gd9reWGS55GSTmaZ3o7VKC73CId8DwY9KE60Qk8ipxTFqoA7eTd5RzGgDOf8aUOHhqqVi44VDxPIZZAqBlNEzlYFIaJvaESMuSHjViqkKvBwkpkqBS\\\\\\\/z6Mqk2nbl27CEXrQd74nxhnjgu\\\\\\\/5x+J8O65rzoeTSLJOJLKuL\\\\\\\/LJWExAf5y5zdhsfTR\\\\\\\/t5ST0UH9HjZ4jHVh3ZOSvhFipaecDe30YiYBT6NLogp1wTKZIZNaCvCaBZtmwkNvN7QZC36hoYs1NLyO\\\\\\\/mAEHtpk1G6RPal0Kd1n+V3icIPZQdhi+I8nmT\\\\\\\/q3jC80NXvsAKE+5YRhze4FEZDkqRt3xxOOtrfkOZcO2ppcaf8GUY7lLhSFGlHr\\\\\\\/2qYVnYsIIAjJ3Vsb8K+kskge3wICgabACq5koi8zvu\\\\\\\/JanRTv82m\\\\\\\/Y8pYRDgv8pt\\\\\\\/9PzQFInG3Noz8oewR1l3cLH6giGL7748422g8pmqzHB4LRFcnQtciE21XQozDEEF\\\\\\\/4\\\\\\\/cHsTHCBb\\\\\\\/L3mcJgwnfyPX\\\\\\\/EeqsawDNGj\\\\\\\/nD+viQzIuJT3ZKhNY9xQgzp4+1v81MxHR2N43oLb88B\\\\\\\/oT\\\\\\\/5atJG6j8C0f4E4YiCIoE+oum3rWYcNgxd8oIVp5YlEH8Dx92sggKPGLbvpjcAoKmPzlCUa7vB6Hm5E4XfHY+AstiHaC32MCJvKzz71\\\\\\\/HxqiAVNHlC3rwQPA6NfAOLL1PiVorAk9nz36qDO0h7TjMgZJhuBG3BPnGr7PtPwPHt4dHaAe\\\\\\\/C8dFvEuBM6yyNbJyqi5nOj\\\\\\\/AkhlHPz1em\\\\\\\/+3XNgYhonRlXRkVVOTHR09L6f\\\\\\\/YUlowR35NjhIst96vKq1vbGPJ+WmXpsPH\\\\\\\/inIlAJ5qJ3fsk1lHSoktEFfZl+5gWYc+4My6u5hrZVgm2fB8y++Rt0ruCJcOnFB8ysxcnBJyjhr\\\\\\\/+vjbUlUNZoiYk5IVWKE07UyUg0UbDU8A4Nia1PQ7YIvd3Ev8Gom3cojvu70tQrCIx5IpNcqvuTzl0K2X9oWS4u05Mv7+k8\\\\\\\/LchAUaK4yd0l3HqArphrt3iRMuCTun33fXjLOX00eCFgWSKZpebLmwsLCwLyv8TVuTtgap\\\\\\\/m+Zq\\\\\\\/2Q6F1GMAeDcppQH0Ozag7O5Os0dF+R74jR0ec6LAHMvj4kx\\\\\\\/733OurNh891ExJCv365bitUG0Jj+W+SVtH0S2DDBWKwHMmzySXcIsay8Yvt8n6FD4yIX8JdpKdceG9el7mmB8aKMe\\\\\\\/mU7Jt4ssmbgT03MgXZbkaUoGUNuLRY8lDyuoXBHqm7vTh47H2b\\\\\\\/kkJ1GBYNR6pEbV2yamYvReN+\\\\\\\/VJ0nJ3zC1i4zKSmlj8F10LY19nq9jAcA8jz9DCNd4kpSY2sj+B1lJcYR9l+xWmrA0TICUmWbKeVa2VQUvvyMkokzVaIw2DAuX00PeFmZrY6rOdt1HNKrNQoCdtEB+7qjipfzDIEO2V0UOls\\\\\\\/5UcwXhyLaahnwsY4iiu4siwz1mkjcyZBq\\\\\\\/lWZg7u8CSYii4I4Wmc99fEhvtHIq9v44GQxOaTUGingiXEVUordXsNuFxNB45ap9jVH60JRJav9d3ybhBrkfPSjP5lasY2MIiVCnH7fRIQj5Vdq7rMWQKyIl7M2Htxhh1RYyjoC7wEawbsGO37j\\\\\\\/fF3pZ6bHPzg5z4GgAdbcDQaY+UHcVqa+N\\\\\\\/u17hB8zWttjOvJBK79Rwubbtg5bbsbNR7qsidPsg3T8zkXv1N7eWa9HOYg5tVQdz5zFcv\\\\\\\/HVykouNS26Nyb47EdA7QgtcHzV2zJXgINXRBRurfWUvJC8lLFaTA8SZbKY2hv0FOXxinO6StMxSVmqaz+g6+pN5tBKKB4g6WWEtmnSBaseeQySYRMIJ2sFi7jxIZH+1l1h4tiOWyrHHZR7X1VPTRCzwcQ9PWra8aB6NslUfJv3YJDwJf0lzV3rtzKGSmDyMb4PzX5G897k9mVEnDxVkzbJee2CXB8sO+QJH4ApUKI6HNeN5ddbpW3T\\\\\\\/tbN8AGnOlW9rQhfikz7WCWPWeoroNs8qycpJXVc+I6+279RZkSaYI7tbta+wZoC+oArUMHR7swnsLlx+840M5N0Sw8qqvpSd+Z2LImfWwXQNq3U08VKMBP8cs9QVqtr0Fof1UKY73\\\\\\\/rKiPPpsPEKAODm5ZKVudD3sy5YOZIL+Tcbcq9FfBk0Ah6X4T90i62us6CdGwZCQsrep317GM020Uh3d4NCAbQW+iH5Q9piMFIXBwbYPX+LiPizw3mWlBD7emdjkEkI68uWoFm0p4i+o4kDU6PHwY8s65UE0rW4M8w9CUrc2VDsLQaAEFDsV+ypWnTAorDNoZIBjrVEennDVj8CdxH88Ft85DwVQJ+Kuf26o7eVyUsT\\\\\\\/TZgIzObTcP681\\\\\\\/4aXPu0FpICmr9I4vmeLYGMJGjjqTi8mtV5AbK2xiA\\\\\\\/6h8lF9FDnDZ9YhwuWaugp4wUNvHNCUF85679nwV0xMu6+qXjvC4d0Tp0l2CizT8OKhz67iBdX97VHQ\\\\\\\/6fqXIyic+n9NWByFYyw93x\\\\\\\/j0Gbipi1iOXyJ3o91dWrjK+uq3ocvR2i5wtRXvUrRFvcT8I+eiHEjVMl81\\\\\\\/Ie3pxNpn9Tu8g5JCjhrNB+dnPRP\\\\\\\/8HdhgR6KjZ7UnA13Wsj3f50BkxYupKkKCwhNl5vUYKWgzI3p2X1Woqg6PrjciDd0svv\\\\\\\/85G93hWUDnRSa6mMgJ7M9Q\\\\\\\/DXApy0njHoF49r+bjrP2D051w4srDRPgKfEGg++fM\\\\\\\/XvWvgV09w22I6xxVUIjf5n\\\\\\\/ZpKne1mysModMx9LdMBYwolF3oHM6E9+I\\\\\\\/v0OUxxQEUO5yB23nqQtFWLXX\\\\\\\/kkjEQms4Ix7oCNeEc+j76CV8daFHimuBcFBvOxKyBP7JrC7nTbHGKPc6xZMPVYqaTtUwwzfo2IGgzgarRqsCXQzVt1QfONCFbsHCJ0avpIf90oIbo23P091jAlNd8JD9wrA\\\\\\\/3Qpu0ceGVBegblytWa70u3biLeU1vmSjTe3neNyOCnfBoItmnXkyYHXWunO56A5ZhO7pvGdah60DPgTUbyWkxhCSwOmDQZnzI8mZVDgjMlzy8UZunwpL3+Pg0figWXDK5tG9lSpyZtwvYY6q0bcxMqv4My8gO4nSvobHLlDkWMtdNclDn1H7rxyAb9y6OIMfjNvZXVmTIiEefxPDUEKflzRsPQOMQnJcgkVzNLWoTUwwb4SRbf9f6RDgTomQJ9nV7cwVTdXbnxZydQI7VlELh05Rf1DZ+RrnVFUER\\\\\\\/w7E+Q6W\\\\\\\/HmvZaGfLSTDQFvWZGktbnX8wbUr0Zbs6HUU9eCOIKZ26e39noa+x7Nhc\\\\\\\/emocbRfbURDtLcme0g5+unzQ+4EoK5RgVnOsA7FgRrOAWs0NUVD7PlsF+RqEPRt+pvylbocde5QP88r20NgGTQdrlcm6yXVAEmCeJzpGSGHlUBRKc657R9zkyIgywd6kusqoP5NE8E8sPMg3c5GVJB+bwzBYCJM8mdyPWgSe5mQOT7kCkMkL5ZMKHZIPnaCHLBncjdGGCsBPlHly2f++z4he0mN\\\\\\\/pDeKOCGJanELHkU5WthX0s2DNSnJgmwRRmAjCrV4pLvNJlMOSV6qjcLV0uU3ll0EIaZkfePaJrR6QYU8e336BPDuBFn8cDA6\\\\\\\/88GC5aTppupKF6b3GRUwDMCKQB5OyjfHBUMF0u9yYBKl0y7BfmQcyUIsEYJWflvJx0fZ6g2dmtSS1pgk74tf3h0Oeh5uNjqQq2PhcJlGj8sOWq0lV+Ycoh2p22oWGp6SjkIcgbCSf67z1H9GuChxmMUQc77DRXA27QlgxQjs5T6nD6R+fYs3IEyg74R3zFcU7yYMHuopHbv2zquy5jfxVFIKvQwx0rlYVqqTC6MFnQkwFmFt0X5nak9zBFgNEPLMIWErmfWdnJNxtGJbHamIOfIXV5lkBq7IoOyE5finAmPuNNtIqPUM2+ZFtCvfvH9Xs4Cc\\\\\\\/+Ln3JomasDfdDL1iXq\\\\\\\/EgkHnJK9ZjDgEdi1ODrZVga96QKXsjvTjE1n4j+Pq\\\\\\\/afqPiCuRi8daVHjpyn1J1HOEx7jmjqdedK7IGQnpUA1LgKe1kaRQAmjsByEY0i9lY\\\\\\\/hHf2uOwdq7U+HBIt8sCVrEMnFH75TcvLhx++eeAaF\\\\\\\/ek6kTVdIflPm9+M2BMOry39+xUNyp0q\\\\\\\/P33IOaFrUoPAGbqbchBlBLU0Ix1QcD44qMIG3iNm+nUzcCP13h\\\\\\\/m2frAFn7B298ko2l9hraND4zgqeKK\\\\\\\/FU5MZfm7mIfaNss920gK77lo0FHp6FizWsYlUb3U8p3JSjwxcrO2Hi5n8SoUlPcvNPkLeke6+VL4VbA6WSb52ExGPpBI3VDFZpvJjAPra4l8NyE\\\\\\\/anCFiP5XXOvuhKNd99UDRS4NSQjCGF8NpkGBngZDC73flP0rFQ3TQTEBJSxKFLjfvgGSLUsmCXCtZnAmFKqbEOxw8QAZo6g9BzDjVd4c\\\\\\\/Y43Hm6brbW6WEpfPNpmMprz1JOJbHJ+ZXe17P46AlKiIoXFtwWbVdxggiZIBJKeINa1uZLXOUnfQHm4tYt9mH+PVHtQmOlR4ZDJAqlVjvxoO90ku4P6Pj7H4\\\\\\\/Gs10gxUvpuJqoRjbO2bnWGTFCfTD7Wx9nCKdzrcLmmw9CmaEiEzCs6iP2swVWVZclWEoZSbpDoCstwMS+KxVU6Qn6wX3Hlo5pZab3ow2jW7oOLLXAg49oc0dQakjCdVG0HcGUuazQk9rHKpYKK4jtxVTKL9VH16IAKDxmC\\\\\\\/27K\\\\\\\/bKsuiFzun8lLt5QyEXWCZkyd72SyGiBR9ek4iQvurgKRJUIDrkLB7YTmd350G5qh6Mezy50uZr4rpOs3LW\\\\\\\/0hswUeTVWmzX11PuVdKI6qk2eqBzNxD0wxK\\\\\\\/7q+\\\\\\\/iOFmHJ+zxutgNN\\\\\\\/OghzLNwBDY+Q8+X4Phgc2xfx5YcxSx2G3CaqIPO\\\\\\\/YH2a0hLsxZLmx0RHGJ4IMWGxHILg0uYb\\\\\\\/askMlp1KgrY0gHSV4FWd2yS+twdY1qEwOzKV1ClGyXw53DCQXh\\\\\\\/gaAQl26MWGrgHHlbf4RBmKVta+l+AjYPvWvF9FKYubx0ukGGwyl\\\\\\\/UawpdNAtka3KB2PWGu2TVL6V1XPF4rH7nh8M8StGOcC3srySmYm4joQMwqyhg4GK3gBqwFWXA4DsRbycGekzyCo7VtNpIv9XAOeOC6cWAPHYDVdlGc207RffvWA45LiWIDURwDIlWQdkVNvkpD1k\\\\\\\/70oPmlkDlhif9NeCOtQzCoNODgNiE0qJIeO26y+xSrIu8QVr4MalSmpZ+wHp0LJ39y7yJBlV11YOmwDg\\\\\\\/JJ757uQg8WCuqUWVjWycqttQzNt67AsmVrpx+zGAB5Y0iHHc0Aze\\\\\\\/C0xSK0AHpmatSNibeHRyi3szpumUIAU9G7qAuk2jNPVbN41M5jaO4eM8l2yd9cytnZFNhXGL45SWqwV6GH4bd0AzfxAtfRs4ImZiPiws20iNZhpnN4CGK8L1JXC6RZ2AHUgIzm6YcmjpKmxnCpX+rRTS2wwhTGBN8TLJrOuPwVAsAK\\\\\\\/VpCiFlAXqPfFcrS\\\\\\\/xUYwv1Pv3mrrrrGfHnC3t7co85AOAe6AA4+YEwhj503wV3lDeq\\\\\\\/YVhBLD\\\\\\\/qZTzOFnrdCz9WsefD8Yo+Pvbwa7oUOUf7OeIO0T8OzlHF0VImMx3XKlO4Phy9f7zPgDXtuZDPfJt+o0UuZWPByQZ2npIwQthtBcEiqswrQtwv4momOMMRQnYaoF6QRChRbVjOgnc+n70sq3sCDnRS3fS8ZsvxoZjUfrA7OhMLsMYmK35MicI4NnQC3TWJm\\\\\\\/lgqMzghXQjWNqaM6tosbLKK2\\\\\\\/YZdcP5qQPQJ4O5hlfKm354HV+7Tagrt7E3iIrCiZdQgVHrmlYaoA1PrwV16RZhHP+FsleYOlJ7SF4jB2CGffV8fpKTnLvEu36DAIScFR6pqim1N8RvStdJwDm51OC8vEcveWDhEIVKGIM1+on6Qk8QV3o77QenqgcjcmxFRTqgLW84S6AVWxp6pgx76ylBBhPtDHkTJdHE\\\\\\\/OVoK\\\\\\\/vptT9vdO9S\\\\\\\/\\\\\\\/kBktUAEUdjjURMyrVo1NyxoD5g4VguEKDLM8FrwPw8fHCAuo6NykORfpz6QJck8goKAmOwbdr\\\\\\\/mYvaVCEXuVHdtI++i2xnb5WKUVyhuHgdN5e+DK1cSs2BvtuIZFXSGrsoTL07CDslavM\\\\\\\/mfU5XWgM036ApCgx412G3Oasd1g99ftKzUcARELfhXBILUMDrKoigOcP6SdftN8ATh1Ao\\\\\\\/TKmT+C0u1UkB8u+5H\\\\\\\/3uq2rQH1+rsogJpp7rq9S\\\\\\\/s07L11qEObvRGhUZF98pKAOpMW\\\\\\\/dY1Njsvxx2LP8l1kaYsFRKGhgGG4KhR3nobTN5rSdufDyre+8i0BGgkKJH\\\\\\\/hucDWs7lwLpBLffIjmBptLfty5pyr74P26NyhjRiJrzx3RKmBL6rHlxThP8E9L95a1gff7ikK5iXE\\\\\\\/t3oLZXwdS5oel+T6HoIr4YgI2PYzMIC9ied3b3clZUmhLDOQ\\\\\\\/qcl3uCbC5r8tKCUNw\\\\\\\/CjH8ticqPeA3lRhglBLaZjvyanDcx75lrs3APWlaS954IMcafyJ5kTnJ6tvumHHtvRYOcCQkVa1lju4LqN3C7p9W5vv9L6GMOGMGMnsiOUfy4b6cNEkzFZlIDF1erfGUtFWT7wDtuYlpbtoMzTTlRxmQej6MqmqiGkIh+X6l4zjT9vvv7Z4EB9M9ulivl4ym2MgPeqWkPFqZ0K9Mzt3l5DA2B13m2tfy1Wu\\\\\\\/k0+TGO0CMMycBc36lcHHRCvLz94ko5YZusZlzFuXRJiep8RJD3hLIgc\\\\\\\/LS2ekAMAi7YWzC+vqP60OqkB6mRks3WBYJjB5nOR\\\\\\\/p\\\\\\\/pREmcSMRAmR1SH4onpS3rVPNPxauI\\\\\\\/yEOd57AZd7EC+eAGXfLLjHso2l++M9F26brHT41HcCPcb6XTyQYCs8qooJIB5RGoJRBkx1mVJLAnWIayFENevrSjLkNi7TMcvPaS5wD1qKgIbJe\\\\\\\/MJwqlDbPrNDlhOmj+9nUU+F6H4uXOXTYwyv9Gqif++cXAWhidyiPUEqhVrT4AnMlqYT9gFh7n0TyhGfJ7Xxb1+SZizThO1\\\\\\\/22UOy7pz8+YB+uTvRFV+ZumMd5z0x\\\\\\\/HnMlp1SArGUWrZWCEvn3EUBsuMMyOVStG0p5ScHRFmENpaWGE2JwjJNTGNUz91mVRtHm6Vtl6JilK8\\\\\\\/xh0Yu237CqnShgAiloHO0w2duAFIjzDEmHEbuljVpZudbBvDkDYD\\\\\\\/LH5OB2NfNR\\\\\\\/rsGtmeDY8sbpvSkn0dlfC9QEeU4zVOsQe5OopEQJACOJZIDytElUWrQDfj2bs0QM80PHHgf80mPkSrl8PIVKZQea0Qo\\\\\\\/5yOXzefweUkSI11WxVSmbnSYQFHQ+c\\\\\\\/7DQ1liYT\\\\\\\/ies8pVGkIbvyFKy7b3Doi2XuN7GqnERX2P+ptiFJNp7\\\\\\\/0p9yNhwi8QM0aKM2\\\\\\\/hk14QMDKI0K3Q\\\\\\\/rgf9BYRnIFpBm020ACMfFtOVi1lzVh52dy4PTvESOWNUtXAQP284WObprfEKCT\\\\\\\/Sz3YNoAdnBFptCQIEY0c8l65QCNuzK+mmm6IHlkcdvmbOYfqsrR3dlc3jW50Fr+2o9LPSDWDd088\\\\\\\/MZ6NxtTPCAlrPWM8Xb6TRq7kgDUZ2p0i72CtP9hdXVE50\\\\\\\/cP\\\\\\\/\\\\\\\/ozGSpxn9\\\\\\\/atERYRDcCoDuUcLSACYcgsovj\\\\\\\/RSDkfdg3wh3Pivvt5z4XcKTlq3v8rjvSrJcL6wsbAyk7HDf0TLBQXYtDJkPhYDSyZa9fGTRYjML3NNU7R66vUmONImD3awJdTfR\\\\\\\/cnXKw3md0sD2QqZJgG+kXO70sw0Ld8j7gquqGoixX2L6Nj\\\\\\\/71T7GPmrwf64CcZJl+NeTD7icJkrPusBwwAtgpM4U9MCxsRcAU6iUuVz70KxwV0DtHLQv4Jx1meL5HVTWhYY2PoOak6LO7NNHHRCgGpeA1cv98efBGgulI3p2Hc5PNaJGdGYu1WujuIgs8IIRViA7+ViVJr2yhvf9beabPO3\\\\\\\/Rl61ufqZMkFj9KgislWozAVs5UbyFHhU2qi5OJ1kq6f8FM3fnLfDRR2REn1iERSUlPKychw8cdSstTn1htzaRPpMGxK72C7Ne+l7dSLgL\\\\\\\/l7yuhBfU89AatL2JRA3VOHsroUqnqtfn4RStn1Q9dsl\\\\\\\/RAvgk8ym2DKCtVzAxO2iJIbarMlzQx\\\\\\\/CrZIhM8vgEMgBq8o5nApoqRm5gj\\\\\\\/IYMz2Gfn9bo0FUneZVMi23fjfcgLRnFC7zjIDJjvokUQSBTp73pv58cwL4DjCbZOzzaEpm\\\\\\\/O80YKFXRHolwBtGdN4ezH\\\\\\\/jb+f2WP\\\\\\\/KDTc2KceVzoccSgiyDJuNxcjgwvQ9M2vZH2BR4sytbgYcwc+AMc\\\\\\\/5lG9AGNasiTMqUXjsx1JTLmHkrlaqalCbpqmVEld1DChkD4YaNzL1aTPznZHpBRAsEEYoxHxWn79ZXPyyHzNf7ExrXMlVYmvdPEQ9ArWGiRmLqWmHq+4gpBn9Qz4mG9XytUTdAxAmfWKDQHfLyPpKtsFeYukP2f2RjTtHbM8A4xLHg20t7rdGf8By0l50WJxmr9ytw1f8eERtwiY9J3b9DdhdCycZrqWdckC8aXC4PgAQPk8NblizdwjgNFZ8Gt0djFskKRajd8L727Ct7O79\\\\\\\/Ye8xumS5p3okONv4f5Oxz+QnYShiqJ+q1R+qo9b7NCnlfs+Fotvz\\\\\\\/\\\\\\\/elXyMeXK8ZN\\\\\\\/No\\\\\\\/LReC9Bf0bZ6DawrfONHNHPi8GNKv4BybLB0uWOw6yY0u\\\\\\\/gxHkw3YvPoAC2BQYutrfhJuzVzPpfww+6gdVZkP1OQxvmUqN0Y2wLGDAwVO54xBSt4OJhQkOYzhswmLyx0mfqHIlxk5ZZ6ruf+HJpiev+l5VqHj1SPCb7rlxZWgwifY2K5dv\\\\\\\/7\\\\\\\/8dFnWBhDgZJPRB+KiJZlpnI5NOGYkehBc+Br+WvHq7h\\\\\\\/rpx35Nx3t8701R4REMQW8EwSxeZTDgpSJhwOhIEjVNsB9pbv4BnXBAJsheiXuOS1uEAXNvU02rxyw\\\\\\\/TAwWWa7gKoqhavc0WNtfiToyz71s0LoeovGGhjr9edsVsPuGv46XLKx3vBscWytdQTYVPouRfTew0d1DhMQXAZ3Z8z6xgYX0a7ulfrHnUs11\\\\\\\/qO5Xw8BAat4akdVNDCseaYajjLMlnYAdJ\\\\\\\/Rte2YScyQbFVR83iAcei8xAKcV6eTLKxh3yjfri5SucZOA2yMlsBXiGAFvjP2LKBEk\\\\\\\/3TyZhKMdjztRnONDb897Se8jgz1NwVpyhW1lxJ47x8hXf6YodsADSYl0XGKgBULqqc8L6\\\\\\\/yy0SQfwL1w4tw7yCAns9lsp57k4kylUvvvwHEsl\\\\\\\/Xeql+8GH4fS3VbKP1tJKRMIknce0XUV578tkPy5qAbt0fkd8hSzYvzJuA7Yp7rYiGHNLpT48hvFcQnjo\\\\\\\/8jy3FSzOK1kMXF7hC4RJ6hK4lm5ok8S3z+PDuSIDghxXTAkCv7\\\\\\\/LaDB22XbNQDlcfPyIGhYs1593Zz5BIQprJ0d\\\\\\\/SVXPW3ITqMSQfHI+I33vMi8Drpib0kqoYyxzzdYi3G7qCaFgHZ5RboR8YcmYW35nzH4xdMlngeizVMrGrqBz13qRsndowvkQGbdJ+IdtBofO95z1pl1ydp4X6RocRpDQ0FyQmCeHhT85uCksz0CPnAsj8A1MSMP+OGV1Z60Z8MMUh1Nz5waeHcnvDQ+6CVRion8Mlz1YcdcpMbtvb0qCqxaPkUwTv9T9quKphl4B805qTD3NmixinIUoiPWm137CcTqZ27GC+ZOjcz6yvGzfpB9QAg9Qbo\\\\\\\/p0pRuY5uMXLMrWXPusvxNTVnxWxHi80TUzw0kM61Pkl8hL7PjKcRPX21ZOE9lE81stPcUQUaUQfsoSOEBsjnl6IKvcBIGJlpgtTq2\\\\\\\/EomjF6LILaR1D2\\\\\\\/Y\\\\\\\/\\\\\\\/CA+XFuaOAA\\\\\\\/JqS8V6Jt2bR0EeZCXU+R3DFT76DWm\\\\\\\/DChc1yPrlXdTy06rKfkifApO2RlcLVpxxBON71q+F0MDYnwukfpadpTvJfdKUyy0HgEuZgjkoACHZntP9AYmtT5HR3BkT8B65xl\\\\\\\/zTaFp9ZfqrMkdIq5E+JNpdL01vju7vCxFOG+4pmFPfi+UBKJIXAxRP20OaQcflAk8FBROhIU5qwyf3jubdty68zstka77khVUDBanNEDxv07BVVzAlTOXfUPN5BOiYGA6DY5pX1WB0\\\\\\\/M0gcMxIDCyWe\\\\\\\/fLLBcDim6UpQ0RsY5gxw41AD2fZ4KHWi0zJSPhsfrhSDC4yq5EaaNK00Gh2W4DC7BXpCoAiae9tpJjwj+X+TXZDsluSdlMkXjKSvmCgem67Iz7ACDslZz3ZNknyheukPf+2cJVY8B8GHZ5iAsxTO8O2KrIjoeQetY7uvykaPQwFHO\\\\\\\/s7A59yT\\\\\\\/WDK54xcQZ0WrN0C01t04rxQX4kTvFhkJar\\\\\\\/Dc7WljQtQ0LCJ0MW2yUMuUGECKPTP5NSG5sJ9P6ddS4Iio5zQW64vqJbgDaARFTVQIGh3aQI3k5HLmodEqaWmUYpfIJCmO0P\\\\\\\/qvEc7C6G5uMIdD\\\\\\\/tTPPKFdpgXdeSYqh5P9e0nnUnbU2BDK4afilUr3qDjL2b1U+l3U5LAUv7ZLbRhJkMMMIPx3rap2hiyPuSc2Azyz+xXonWU8IU4EknqB82clK\\\\\\\/sJWaimZU8UVK57ZX7XVNaeARx36pjX3lDdCiEb3LqKkMM6V7GZtD9ob3Qeyg0vKFOlpER6SmnEJaouQ\\\\\\\/Q0vaj8OjkpngMmcmP6hLcoanDwiW0AR6aT0vYyOW+r32J1rLxZQWopGMq8kGSYrQKXgX19CopSWfe4lDi02m8zHeS5i3qwwCNC08TQeUHTwIb5X3BG2T6J+1rN383Xf5xNIobs\\\\\\\/c4BdDVHxYeuUG09Wl9mtMY2o9dKlKWM8DMZKMIj3AsvrDso+THEWVPH5NAhc4LRpDSlivzDGdnLFIsfy0DxJQCgrpdp6fvqmS58v1wLsGopi8kLqWkxcvG34J2V5U55rywxaPLuF9lOB9aDIj7\\\\\\\/He9ApST0Um0UsdXDMxC5H\\\\\\\/x1hrjAxwvMqWfkiMPauYtQgeMnDUpocKKGA2+V4awkgTHJywURDaR9Xkqrh2yYHUY3uuUcD91oJyq36kcAFhSjCKWQaoa4kcRpHfVaHZPIW5ml\\\\\\\/suOcvETQ033RTnQREvBtFHoiLUJtNu4opwfY\\\\\\\/ZRZd0+mxX+0xC0dgGnM7++KTCLxgXMw315PCpA+MwsHmT7xpKrW6m4odks9JK+l8o3wbBNaa6YwMJ6pxZVjGrB+dB6PSgUI87x+4WWtcwmwkbVVlynTWyjJOcnEInztSyuSBUHxe94p9xYfPOBx5hKfg13Ysa3r\\\\\\\/wqieN66Su+tehd64Xxw37Y3TSe0eEAJ2uFZoVpAajXgg8SSnB45u85+Lu63poqES3TXr5o+PPHIpXntEgJsTnb7pNUyc58a7XJwfzROmkuYha\\\\\\\/mpWLemomB4RGObGfDldYeUP05pRu95x1V1WvPUYuZWr8i5tmVPJMoK1xGu2oFc1atQQbSksYlpMen1HfHFHPfSGN9S4cTYloiZe8GK2NPaifYQ+FzXFzpgAz+ZenNnj0h78FXuUZwAsQ7i3aIeaMuhOzJ4r9GzpHQeRBZDFA3dVl6H2si8jtYVtw6SgwiNmspcOuH8OkWUBXSkO868x+CprL1jvZ9otOYPjKBPMpXRqPbXF+7y\\\\\\\/iQ0eNTpQ4q3lLqlzSoz\\\\\\\/LU5ivqcKz6Z\\\\\\\/ArTdtDwuQqdwAJKQYuQ+q0\\\\\\\/rp4X5UWmEiJ0Ui1sob3nmKZMuiF1n204VRjfMIfd8pl5NGhFSeys02588UgX4+PgU9qCNnNJMg0AciAukZAPHVfDi733G2R1kt8fD7aYULAYEIZinOH+EApC1n6onmC47\\\\\\\/OEr7iuno5kkCdFTmsN2DOilRYD07hSN\\\\\\\/L\\\\\\\/BOhjAEAYHysryOOL6baZOWYQ6YuhPnphakRv1IAnqwLt+KLzJrRljOPjiWHReekvc1njkN1BJW7lrl3P7hARFwHrqg58wlOdSyIaYvIefFo4nN3Ez7KnNA7wPmDgXldUyigYW+mmnBrBOw0rfs5gMfB4kU7P+yYYQv87IM2A2NaOpWtw6Yj6w7FgVPgJkUaaZVw3Aazbn8kpubBRIACFG9z8kJmuhJQMFa\\\\\\\/STY0gDMVrwkt4uC1G0E5azo7FPGaN\\\\\\\/nYnsNzGgStEMPczBBeE9N9pJXwR8Uy2\\\\\\\/oK3KkQCNCRTsMoMG+BkrtMkbXydJwKIr2kgiTQA6pxXvrZKxx7gx0799ANAqkAOf8H+s6ZMViXXNpYGjKvzz9c1zclfXiS8m0JSiYd4jYe3BVrj+QkRCCEtv031mO9bWhRNQ5z9iLJYZFLhwB\\\\\\\/20NI\\\\\\\/Ybxq0xzHJvHT62IrHYHeujVRwZceluSSUOJBtnyotSA743I8h\\\\\\\/83H7Iyz98D21Dy86j5JVD5EwV1qwoNAGI3Ji2Q0TeIGzz2tnpTFU7aWHBQS12SN3D\\\\\\\/i9U3ZEr60jYKTGwQIHQOSSkm4dG9eoICrFQlYUUpSg2KLeEkGSYkmqXIx6YXYbZ0lf3hVOZq2Ifq1Yr1Yy1DjwCNNT4M4Ec1EYVrcx9P95nIJtKWVWyuLOx0ueYaPokB8FzPHl\\\\\\\/tVObtZlUv1hAbLbsuxD9vBHORj7lMx+wmXH8InEvYLqYo1lnGE69ZS4OjXi\\\\\\\/KiBvn16NaTmQ10em+hHU1knoRicRqVVzleZUfuH6SO\\\\\\\/VSEF+DwDbBlBlN5E5f4k10YLoKBInUqXrdmFhAug\\\\\\\/wbssarUArzl2SqiQINEcMjITH1HDj6O48hHin6ctBg6DYU7RfptBxfUTLOtdRS1qk8lIptDZmi8Xz5NNwLQLWpvzuE1oHrg5Nj+B\\\\\\\/st\\\\\\\/noMZFkiwPF0a+VMjfThVJMyi7BZfDajFndxXhD9L9bRRsMHrYZKNzMaWbQmpiXSdvEHm5YPmmxlH2gi5Iw7fR2tM1CfFuinp\\\\\\\/a0WCPp2cSeQOVNgq\\\\\\\/59gstOV89agLSER1DBImZ5\\\\\\\/yrn4CZgj8YqGhrs9H0r7M3kTbKr\\\\\\\/UnKCoTEoLtZNdDQIHtcV4axievZaC5WtTFQfxXVAJ4otH5FbzYXHaJXR6fW08kb\\\\\\\/seURA7glM3WXXEtdi0fKm3Rc3zJ8rlyWrmDUypvaCwwSxmNsR8dd1vEuXTlIkHkgDy2VPszN6G6DLlUnfmFuyVBZulXvsMApe5GODTVF86uIWrrCBmivxq1VPw8Dc9G4Q9HV62niVsDyi7SXQ7fdBQLeMg15iJRDDndJJ7NH3ceGIEFUcpCXpqMYCnbVBb1xcHJkuQErsDkysnB5Q43NONOdCJL5KsXfYooFgDsY8GycKZ0dceSQQa91ElyFgMW\\\\\\\/BgOLYwbmaxLWlILZr\\\\\\\/MtTD9n4tim9bT7\\\\\\\/mt+uQSk1grTZ5S\\\\\\\/QIJDO2gYP19zJ6kC9sspUmFEOH\\\\\\\/GOf1ZhZq0T1oEUnJKGyb3PEPzx\\\\\\\/iBMTPTmIxc66pvyeeLWaPj2bu9aNMCYSaO9VBUg4UPFLYYXxBQNCoXociS3jLaFzfCiDhRde7o++HVId6U3WQBTLPFPwc8P4G\\\\\\\/P7H8PlS1dZICQ09pYoakPo8rD1WX5RCpoTQM2DhgVzHAPVtqX4KNcN\\\\\\\/4hQhgewhef8TmulY1JMP2uYMwhZgbUMyWI8+foRC8SNdQti7L4OSLd1HCbAj0LNVCoctrVxHMg7Z99X5A5rTQupp27+RQwAgjbk6zNC2pJSDlVgIOOfsNYH81BgjT51FVnJvPLjceqRii4v8jqdjvifAnbMHscjxDtcY0v+6rNGkoFa8L0DOvsQehBZ9mMoid3eM66wYSwde55+FqLO2nvVkTUPxMi7wZ2P7HB12Ul0NchU+uqE+dCsQ8u2cEZ2rHOq1ZpddqMhLXohnb1f7U2l3YRlGY81VlqwHPZ0i3Cco72GFpKom3c0f9OPLN3IarT5cbF8fm9FzwdEIK\\\\\\\/SFwMCjkfArSgwcODSx91q1zBfwXS6rpUIHbCYfZ3ib7Nne4bpSdNECcRdHDcTgRe+Nccxmt6bEl1cf80mbxDtfw1DnohJGCJVJVQk2JLiiRJVzzbJVRl7Zn0WxN8HUAeRS8v6gOSTsUhMDv+tle+A2pyS3o8HX+Z5rqfln\\\\\\\/ZZAO44isqXO8CMG4+jnMWJuAE3RNlPAO+ZT6DGDinIBoJOn8qygJRyDgbqlws++X8xVSigkqgbR222rlhRST6XV1ocAPSg5hEKvgnHpAbQ68Kls0wWlj7DudNGq25LFzA77hGakX+08LllwLUeBk2STGBiniyflkIifmWR49qMtqzikSXVLIx3AHW1KZAezpO9IfwpJivMR3z+Gj4uE3x87JawkojNrym7tVVmOOfY2SAyukd2ll559F9B9MRgca\\\\\\\/zjs+2ES01neG4Y49OwIDZE3lPHjK4jTOxRQfLE9AjLDNQsZPA6hsEPRNUSX1krBKMylcHXh\\\\\\\/C5Y4S8MHVIcQKKtE7g+zXSInQgSNfNzVj84Kdk50vrEJp+lvCxiuSj2+ZKQN1C6q+d66vcbKseGBp6UaxXjgKei0ows+g2HB3Jv3Hj3KhZDX94exbRpPo74wSXU3i9NQmMqoFg1x4UEklaNjKoSyD6cqvBGP8Jw0L1ysxBS\\\\\\\/OSg02fttsxzbQe4vMf8g6GQBENNZt8mN7z5ZfMqnNXoNuHJRd6LOZ3E8EMjoN7uy3bg1KjopqxXFd2re\\\\\\\/CrXL6qPhFRVi+W1+iah9V7umdx5iE3A0qnd4CmUWxmZ2yzmAOvUxPE5J79GkMk9a6i4xvC\\\\\\\/klD7nprbY1JmIHHttgEKAg3pvm0erfd3rv4htF8DA+UOanQBg92qwtx2fpf\\\\\\\/aD0epI+QqWmogCegnH+P30NJnct2G+qRbe9AthewbdIBAnASYDc1aA3aYQjiDIkYzk9NOMn2EQsZtm3e4BnwDwA+zQyzGnkThrkriU\\\\\\\/Jpv0YdOPybpMNt+MunSsMtKn7U36i4iI6mnOvjVIvyZqg+ZD3v3b0gkZAUxF0cBRtXdksZrVoRUOH2WXC7XMWycoCJ77bktlBEbrCyYOSKGaZ0QixMRAQo+No9Pupv4\\\\\\\/u47tkVK9yjpAz1eMcHQ2+g3hDBKryvm7wPSfCtmA7RWMOzhG53+BR\\\\\\\/Y1YY8wKVNStmDcNlwwwpQHtX5hw3c9KIMvQXcb1nxOKvu0MdpekzZOZ6sf+5ya2YxTlPl\\\\\\\/+FvH460MFFpGYN1NQ+VlfHD1SrS7PU9HV7bTItByHKy1wI9JioiAAQeDThZkjoQnaBcBkuioZj5ui+g7nzeCsc7tU0wgoMVJhK5pCB64DDIIvGdVvtkSk6FLY78wOLSw0Z6c59UW+ZzDeNBpa0\\\\\\\/wPw0HkPR9piLcD+oi8TL8fkqo7NtCF4J9OR1OwrM6KoxPuVx6i6TDUFJn9nsKJr5USQv22brEgapmXe3Emhd+b+9q8IRy6AfGp1RypvmHc6e4064rQj3XKcBzQCtzwVJOyTLYCTArxj3ZhAeTa8Ud\\\\\\\/18XuOWJ1wIsfC2U1Ozuv1gFUInND2e0U08dFn\\\\\\\/x2eTPGDTWtWnA2PYpD4wYjuJqZ2BECxS+m9B\\\\\\\/L2MDoiK3KIVqOA0JoVUWw32HYZaauYZ+oanfVsqZJvSwlt0et5lQxi0Db9ly8mf9HV1c3DTY79mBeRcuLKA83ywN+LjA++1kUhkHbyJUoZldVk4oBtB9GTN6rJh5Ul1vyXqijmtVT5ab2682qobEpGTUF6xRCvmpHkVDRr4PTpwNE6nTnnLON0q5diud\\\\\\\/PJmQtDeKLQkKXd85m449WhSpNaCbFzT9kbHHgMOfQALyJy7N1UNl5oF6qTGQ+0\\\\\\\/rMa9mwP9O78icRWD2KquW6mnAi61cLfJMDwHchf+Rzzy3iw1Bun80henXGdgWx18RAoQdVmw9XqMGPYEBxRU+F\\\\\\\/ijNWmkFWXavrrjw8GnGPnWLrR1t6RYbu\\\\\\\/t7VBTwPCH52c6SeUTSbByPzP3udb6YKAXQ07qQZYvs21NYigcm4wm69VRxMz2Jg9XwYVC3rj3xg0jDbG+gqkE4WWhno9aATU0gt13yWOjd8qVbBSzo5jaCjJHJi4BBdHLv45u318CoCptoj2ZB\\\\\\\/viIL4oGypHLSKsuMnYkEAn2jbcVApRS3FLGYXcVmmk7BbqThiO+c+bPWG\\\\\\\/W+Hkg0vp9faVfwvDQRmsKXOh2K+Yw\\\\\\\/b+kB\\\\\\\/GSCooyHJyLxI0ln0tWkirOASNEE5HCPf24dLUVgvwr8ZUMxWnlZJxpX8AvucqHuECJwND7\\\\\\\/DcbEi\\\\\\\/8wgvqrKkVE66kcB3yDpZD9szoaCvOuiVIS+iSIIldnjax\\\\\\\/eBZ24CF+aR+6ssApP9EmmfaBESdupJadYulVQ2QpBG9VRG3o4ndKGpqXYOaL8clnhMMLIliI3UD2edAzUwxWdYBH5wmwmFP0CnFT6NBPtqtOLotXnf\\\\\\\/XimPSol1jg\\\\\\\/eSIVODlCM2CJeaHHBbDZcMQ86\\\\\\\/BLfh9roN02zqnpxZy40esDTbvuLetRVuy0BYTR0qfdtDKrS7ZKs4+oroSSry\\\\\\\/yyk3Mx+5dOey88ZVk45TU9lWGskaVsjm\\\\\\\/wXoTBZjb4nu\\\\\\\/ncGw3TMzz7jj7Q6x\\\\\\\/EThR1RLVL95PQ\\\\\\\/Cg\\\\\\\/K6WBvPTwJaioj1UqBUZHq7\\\\\\\/S54dz\\\\\\\/nc5IvK8TabzqcQ4YcyA\\\\\\\/9lnD7VcohGmT5\\\\\\\/t173umb6sOZDMpngbnuihkL1u86y3blUPJSqKRG4Yyi1b9E0fiYXJR\\\\\\\/XREieyPMk5UxKd0RMr8hPhCvCWf+4jMCOFRFUDGB0KvlaqQrPfWyUvWJHnbAFbQAC302rMMvEBHt1vAMs4qHXijbA1FKyDcqbjcJ9hHfuo1En+VrQjT63WXFOjI9lkhj\\\\\\\/0juTlgoMY3yQD4D32sf\\\\\\\/Gf\\\\\\\/1LTo13d1oj+bCgVplp42e4xHP7VBHcl+yOpzdHqG2RcJ1kU2kM8pVNU8QdYR3fBzgBbCBhebXiirrwoeNz1rINB913GGHC\\\\\\\/6Gc+mSQq\\\\\\\/T0yTRBbI9+X7AhY413aecSFyMlkGC\\\\\\\/4KceY9pYe9NeJ5CR+u1ISDB\\\\\\\/9HsLwAOkwfJzGEQeovMLqklIk4mBZFVEQ2P+FwU\\\\\\\/jiyofg0XlH4dl8uZuIarncWLsbN8F2whIKxVqS02VOR8z1rQae05NsbXYZoL6PwIMWo8Cws4Z8y0oZG5DogHzCU+3X\\\\\\\/YdBbIzsbKY1wpeCTbdX8HSpB2fUAHB9\\\\\\\/iTjlLUJv6mG4PW\\\\\\\/UbQ5xNxd63bd7yz8oucluBGhbZjcb08k\\\\\\\/QduSwViv1BUe1AQd7oa5VghnrgTLrcxec1gpkRb2bhDkZ3BniSuMp81QzQqVYvVbwzPJGuGbQhHL6y+3PccsDg+LaXMmYT5P3QVtW0xZITLce7uXQjn3\\\\\\\/uRNlABpkvsXgdOTsWOJ8uBgw588EiyntDUhYu+ZnTAPz5Nfj5yK1LsAnnDFX1yeadQoAr9321Ps2JHsovY3l2VAG8QclaoLcqw9Qpk1a08bMtoAdbNbQw\\\\\\\/uBcg\\\\\\\/wheeqwEUDPupSBJcDtIYwUQuoce2XzwEJy1A5h2q1486FJ\\\\\\\/ody6FC5AEFVmdeWvuDS0y+3oG62mNaaBdI0WFXDCPbFmFfOUf4++S\\\\\\\/L9X4XHnZPRUnx7x+bDvyYrEgP4gF5X0ZQnG68JKrQHHsfTTolKfk640SVS6dT+tzpLqSuItmTTi8z6dm4RHNAaXS1RPG30fQxNgw1CBLeXQJipn2LYKWb0WfCXwizFXIcdtQT2nKXaXRGhO212vyds9Nic60p\\\\\\\/apZFdDQqRAWybZ7tjCl1pob+UGnJFH1PSw9uKkG7p8g\\\\\\\/LdsoEpx5MDDVWwRfk3FSYXRb8Al\\\\\\\/FG4q3BpgDY\\\\\\\/GbQU7gRhR48D2XfwjKmM8J57TyVolDA8cZHJXcc7CgtTj1WmebK5dBM1f26XYMcczUPdDNuiAlcTcPPqelLQkLwpdWZnZwAAsqwjgEZeWVVY31pYg7hVKKmY9J46IrI9VeF4q1xLOQX0Rs+t3o3UiJYdk6P5Lcj37FiEM8B6mgY0rKcvwd\\\\\\\/SF7rG35U9ddpb9Kit1x4+oJ9\\\\\\\/rOHhGMe0ZQPLbURztvE3YI26xXmjTVKQRLHLNDegENK77VHhJIxLF28ltvrVODqv56cRUom6Ynf0ACE741fsNF98OIBsmkggPZcgKFUiR5RnHrF5\\\\\\\/cGBSPSj+l4\\\\\\\/brGTcJ9xlgya13mUE6hfr5pSfXOAGMEMOP4OU\\\\\\\/2GJdQ4Xpz+B8Hlu9gJOodZPJ0Yqp4IgZ6Q\\\\\\\/CMcpL7tI13yFCn6\\\\\\\/Ug8Gd\\\\\\\/VMp\\\\\\\/O1XMuFdX742Z9Hoej2ibgnvAvmLBfJVwN\\\\\\\/OuqlZcIapBIvZzUaEBFPaJkigXmuRIXlj8zilD5zXHgnob+AXqaiNjuSaU8ZaVnw5dMX5nRsyOgYQBUUtQdki\\\\\\\/u4gLWo9A79NI9OmTAFkfyT9Wm09nwzSGZKmtL160F9DYVAU9eXU2sz5C3eh0XUcWXS0DRGgFdaMC19Tt4wSH9uSgv2h5rAM7Ni1wuoS7eQ\\\\\\\/C6qtvGLJ+sbMmQxOT0NmVssHm48yvbKsGrZq7dA0oBuz2A4w+q5jH+G96LxUDf8HlZv7hEndL5PVAQQEzlaT\\\\\\\/oWCxrHsGWcYITtNKT2E2gwQ\\\\\\\/TeD2ymnTnMPQfTjzBEfp8c0BSdvoUDewAHW+Lq7chEPNvFeedi1qlwLnXG+WW9q74zGzIPzVH5BMTIHJBmriaPgyXLTBUpemDlN4Y5IDV7qZU2ZiCnjQTCt2Fg3RHNKm5OseAAt4sFKnG6q0pszmOl\\\\\\\/R9JWT5RlVblVuhIauESsppfmv4Ggff+5nuAO\\\\\\\/F+y62BhmcOq+2f3mWHHQuqHbiI8m014Fr4hz3+Rh4m\\\\\\\/Tig6S\\\\\\\/VhqaTXN9oBDg+d8jV\\\\\\\/ZoFuRQAFwT\\\\\\\/B4F6LAMlEhY6J6S6oCifuRb5cZKRL\\\\\\\/BBVBO2AJe0VoyOjhlbM2VsbfpJA1fNZpfQQzp27w14x0KaxmbHQslXr4VOjac8vQvr48CE3nOvBA7LwQqELH7cl7cJDnS5Tq73bPu5Lv3Gm1nd6co2VUQd4f+1xTcBLbT6s7GWrYOTd2sWp9Bhutwo10QjtfmEk269mQ+rTpgyz2ETvfV7als9KIjnhcxfSGC2Z\\\\\\\/IbdcaGzPSIplQbGuv28QWy35QyCbcQsuWVv5DS9Kxa7QED8Lanwn4beHSUoMH4TZ0ZzPUWOyjFhq\\\\\\\/Mk6E0cdmaIdj59g5h7Z552I7gOWpkbcEgCcPPOU5KpiMErtC3Q6EWdEF6J9XIYX96X\\\\\\\/uZv9O9PUwGmowyi9H1wLwZyF+Q5gWaKSJefnoFf3RcFUqQ4BCIAY1kXfCvnD98IVmI+GrRR7mHo\\\\\\\/YMQFMyUTILR0WlkNtto14p\\\\\\\/UfEKuhv98mdAMY\\\\\\\/qi\\\\\\\/vJX\\\\\\\/6RRwN8jiXFPnG7CVjmN9GITzWtrv1zkMOX2hMUA1ZW+utmc+aXy8VVJBfC2qvylJtVXC75RxJ3x7y9xEJTTYmg80ytWe7wWGr8OqWNE9nbhZIW6RJRv0Tw+xAu3SP6YoVJ6CKE\\\\\\\/HqLgjB817EDEt6ZfMtn0HZQMN4dE9aR2cf\\\\\\\/wQf2QdjXQu97QzIZtSXYsfJLZbJj3rC+7F70c3+0x8JysKAbgwdYSmbjGwZqrwDz\\\\\\\/6gLLfmSXvAezzrGj+FCeJkpmSnw8MZEFus0W5KbRiRnJe4iRBpZYjuTavn8XOFMPiviBJWQNXut8LZYvaLXffaX+WiTPBgNuOSt7E4C+ywT\\\\\\\/M2lhqFpqJjf0o78oANOBrzVOSqDxyk0K\\\\\\\/E1wwJ1VtTnWis4NyHE7G04tCtQxBgI\\\\\\\/6013KLYohTvhDeD+AV9flC+jatpS9tn\\\\\\\/YYA6amuUqWClJApksycHZN6nTHZzI7PMlY1IF0GSfQ5TcyUzhuQ3T5xNEZfbyqS8PIZDhxaGk8HmYx4xTxwIxYLPKyiLmStX5zwzR94MVAm1KaWKUTGALc7qsAOU0rcuD4vkr4BJyNNC4hh4Bj0jEAzHtvyXe13MQefrf0boMFpfiRuFNDIxwBCLek32umQOKxiTBjHghoq6URIWk52P77pjdx50kLalody61vQL\\\\\\\/e5YtCmqFsXtjwVusOpAWKynqjNSN82sG7Gr75rsLYkZhhHZJYeIer8flcbnJXETaMWUKJpc\\\\\\\/8rqzQ7pq6YPcn4c\\\\\\\/SC6dFDgBbvyEiRjcsBDcDH\\\\\\\/V94z1gXFMnrLN4bDclQdO6u5j+Ug+k9Ii5ad5g5uxs6O36rFtikOcTYI+Ns7IUuqcLb5CVAMlMJdgsdJan6Yr412YAL3FpKSyLEmi8yW0nUrp+LrQE6tVrfWoi64DEEo1vFo6DvzLW4GWnlCRg9eRHLq9qLhLbBZoyVWTHNfByxZfOzxaZKeR4pdEOoz+2i4Sw4mStDaBGpsnfiJUJJ79TB6cIOR+t5C8flV4SF8YueFoTmcaJ\\\\\\\/w6M9ce6\\\\\\\/3fzNqD4Hl0HnpxhSMaJFnmYKxJmicKjTFklEJNPTKgbV9A0vLs\\\\\\\/CfVVjwfPu9OCB6vuATPQ+IMB8APpiBjPVX7QcXwQt5rzzcUHxiSdb\\\\\\\/CzP5A5excr1HVyWsgckK6yznj\\\\\\\/ph2mUYI1hhLhWDz\\\\\\\/5fsjb3qtQ8\\\\\\\/zB9FNtoV4E5Igm6KOdJjMTmIRsqKyAN3nAe1kykmFl6+PELK64sE11+TWfyCaVtrsKW6rGBEhER+uWPfPp7AMN5od6\\\\\\\/Ae3WF\\\\\\\/0Z1bH+bWt5EZCiK+qfrfxobgzJUqgT4SNHnVVz118al1xQrNagdFAXw9NG66FMJlQMpzDh7ypFxr\\\\\\\/f4yx0PzpCFcZbxnBIr7nxtknhUw7d1JHwOcuYmbwMju0fM0J5wzOWtRHA66MXnET9tPmgRNJrEIURDxLDin+Uk+InCtCnBR4fhYd5zjrjYqaYyY\\\\\\\/SoMmd+P\\\\\\\/QpdSGhYoK7PVVA1m6x65\\\\\\\/HHPYqYHE4HNnyB\\\\\\\/LlDDES3Lq+ioO9rtuJPg+nIoNb5gy\\\\\\\/P4lbbiMlAb++nSI291T1Pb+Y2QEbP6nachqBcCu5juCEHmV\\\\\\\/hdz10VLqEabCrIf5D82qBtzXOZQIrPKhVWrmWsCtVM117souyD5err0G\\\\\\\/EcW5ShsD43LUAtzKW2HX0nVu2YMvenvnPkzyD98o1R8UbIQTFwdXGUu5EmqlqkGgzFL82W6C49x3zrtadj\\\\\\\/ACtI9TvLBiddbCerFxe\\\\\\\/eHsfosw03A4BT+DFhL2eHMhZd7s9XQ8qtwv\\\\\\\/AvLd235Zjfr889vRVlaRazhlgrXiv9VWLGm2XO0fLhw5DsY3iJouHOFe15YAKtu40HEvG+zTMO7tZ0aGG0TSkQLJJsJdcnI5JuYp4oxynnCdwXRyWdp\\\\\\\/ZOZBRh9Zm9sTVIUwQt0E6JnJQKb4R\\\\\\\/rcat3K4Jh3i\\\\\\\/GPNHKJtwCu1iNeUfAd3oynw+gpzqybns7tuGLEapH58FcI0ZGwOoFJgw2eZyb\\\\\\\/ZJQOsEqJI04VLZseByn4SyAKp5aOQDrMS0VU\\\\\\\/eMCIvEOzKgo\\\\\\\/O+S7sSuJYxutCpx+kfh5OvQQseGkGoUo5sFr37nBJTZdKqo8okQhk4gF7+TfYxbBo41ZidsXod9wFuqYuO+WRE8ZcTMsKrJ6j371Vx7qJ2r8ZNv3zrDdgKR9uL6wp2FxNw4gFbBSDa3CM7hG5t5u0BjOeX26F\\\\\\\/4pQSraVFmbRmJAW96hLJVsYNMMmeAGhNx0PL+Hii2cM+VcIYod3sOBPMgvBBvWmngW5yrE\\\\\\\/O3IbzhxTozAP8Lcx7gMKMDXgD84We3OoU0qEmXLVNs8gi5GGWy3EHCdaskkpBqW\\\\\\\/OEKVKz7MFuuGztyo\\\\\\\/CWuVbNxe7bqp\\\\\\\/3AhkMBoAYz9pC1fcRoWH1YYBtLbDuP+JNu+\\\\\\\/g3oMh4\\\\\\\/uel0SYscWWvPEH\\\\\\\/wB9g+S0M\\\\\\\/D86zlmH3MYKUxWxg4NMrd6L4c6g+\\\\\\\/9zFbbZnKEd2p\\\\\\\/E6Lozs4wzkMG1LIIYlrMJDiITiax1mZjUR7ki9xXlwSQ25IbKFvtaCod+0x2EyVAI0ycPF\\\\\\\/b5z+aEzxmO+2WQrVTu7sgY\\\\\\\/E0ko4WckI8g9V5+fCGVjCgltCuOguYqYUakkGAisy5kHhtK6ofXVJnY8TFnS\\\\\\\/49UFIA1gvOJCtOVxXAGVcSD0QHDVVd0APRL7eF4k0nJThIqNqIlqM19sx13o9Gw7GCk0+qUJfY0YOlH307VXzjC16fh\\\\\\\/zt5BJROWa8sgTkB\\\\\\\/ssplwqxY0RbEAcQNyR7ZgKwI71eDk6LW4EONBcQJpdd+N\\\\\\\/gsUhtF7\\\\\\\/nlDj+rpQFsDV8rqC6\\\\\\\/sUfDURsSvt+IzufZSTK02UNf\\\\\\\/58moIX48mZ2yMWlK\\\\\\\/59IkImJcCLE2tM3688Fruj0jxQiiMCVgz27AC3Kr2GLTDjBqul\\\\\\\/pPdsGbsJkcOgkCUK\\\\\\\/YE0kUHB74KPdh6gj91MVHdjKJFMJ0rQ+At+yWyVjcYd5zYYctzoRvkVZMsPmHkQXoGf3Miqz89MOcNsyc74NtbxCtqWum7z8SQijRU2hUIOioiWXrZtyT4ExuuePweYSpaB1AcTOnn1m60uJCoYSiTVkdD2RlPXJvpUAi6us+JDqOlFYRGJdrjEMlDRNedD607Zi2iPsr6kCY7kDSjLfjebY9z6UIEAW1VtAsy+28G602kBWRANEmgN0+QNDxZYbvO2aFfOEYF6rUBLUw0lQoUP4Mus9IzYOtVolvaseWUxj4r29Obr7vwsM9P+6M+zbEseSPoR9oiPKy87\\\\\\\/5tRAEsLr3Z76vumDttKFozWTw89CFP+fzaIAbQ6fm1gv0ZfbSuEkRt3W56ffkaU5OXvu6Mc69iBLC6jA2jedRpjgVp8U7+eJ8ZQGuRgBFLt+tcEEz7tFpDKCdiCBADTNwmitOhnUPwr7lSvuWMUKU7xSCZYsm4xXrE7OILeLvEcD6d1M6bSvaVlBw8nBD+FmK7DQ9mm\\\\\\\/yZFIVjU6RTRHEHbCqac3A0XbYClBHxJ5cBHYbzbZ6alOnIrMO3wYWWyaewAD2XSTmZopxq2eBarp9PqDQMbO82l9UFkdxPCe+Zplozw9mBPeL3Yd9CJ8doYF+591itopybKNbRuG2hsHfaUII1uWV2ZVpZeEho4YYi6duBBLibCMTlk8h\\\\\\\/hfeuQqe7ZTDXzk1Ih5IhcQravAQ8Xz+o\\\\\\\/f\\\\\\\/DCgaXJ8RKRpzH5kUZIU67\\\\\\\/mA6BA7ThH0US4fGgatY2ORYXqeEgjS\\\\\\\/l2uH0Eau+\\\\\\\/NDNdh+wffS1Z8iPl\\\\\\\/PzoGaanz\\\\\\\/uI\\\\\\\/fwBmsmnJW2feFUMskv1PHO1WcJsk2wbML6H+Rh9Iq0LowK2v9XeGB10Uoxq80b+QqG8kryaZLJSvYaLJ\\\\\\\/1yc8LXfq0ZblicUBobJ7J88o0HSpZ4kZkmEe+SRmqGGccXj3R5n25wQd53lXKn3CDJQp6ip4ufv+EkaPnOn9kkvVfJNqYucR6W6YfkdH5wkQp1RTWZ5wH44ZbxDpGJ\\\\\\\/fGInZR+tUQd+4wvCki3K8KpJjtSEYw9grH5fi3bovpsuFbNWTwsr1Di6gjpuR8nSa4XUD4+Q593y0197BLg\\\\\\\/4+Norprlp\\\\\\\/Q0Ml6R+HZsiK8qdesRR0uToY1PB7MW5LjtGlG2tsAb4uwM63SxggzZZHTkTFiVnBzGu1QehF2YHiBY1mR4mTgW5M+g+FMsDoVGEM61VI1nPFpqlb4gU8ceUHWe4tvU3ufEiKbEQDg6zNJwX+Mjdy26650qk58WJFzXB5QOOs5iwh2wrawRYeLWnuvR6BnvSBqZYSgBHvSywwyg18mlqiGZn9Y6f5CwNkBhmAJeoC1mnpblKI6XO4iL9GNqF0raWwRMtI4Tqg1TmXF2m\\\\\\\/QJpR9orQ0XQMjvRpb0ooHDKxv\\\\\\\/JvS2wh6SzxaDvpQS98iUWJj206T7\\\\\\\/0xhpV20UNIWsyu6A==\\\u0022,\\\u0022iv\\\u0022:\\\u00220m0BgWypoAP5L8Gf9CASFw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223301697ae72203a72be054e10e4ae04fa3276fafc302fcd1d7843077e86ca5d1\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022DFGCJol1ZHo2UA2p2CbUxP4EYLPr1gQ33CExdVfyCl5Pu1\\\\\\\/dnUUnwvoezlp3tDBDf\\\\\\\/dK\\\\\\\/FH1YN3TPYJxMkxp7vozzMt52hIUGQECt+1VQqLTRkkRipgQ\\\\\\\/RxGgPaifhpDoVOdzQ+JArYWfwOF8+so9c+lY5d092j8LQcdtklWBQOTTZcaZKceHe2cCeSpyw1hczFvai88Evv6q6oA\\\\\\\/mS8vidr8kH5xK1DGer6aS4ilD6fXdIQgbog3Ei\\\\\\\/YHHLd5PuVnlIyfbYldAEmOxsjadgd1AktpyTfzjGZeLbjPHAkRImBGj4wtqX8L\\\\\\\/o171KiffsQj56EyCb0ONpTc3v6xrAu2Yh++IBWMKE97Clo5NQfoIltnUnZC37pGJitDMH5f9N3B4Trh0p4MM+aAgg\\\\\\\/12HuBu+w7SWrVmvfb8x1uofBntpOrRSdZWg9KZk01wZHc6m3OyuUrhPRb1kbM+zMZr9ppVFrj7a1Xx148zflnHyOngNV9ltmJLROOrxVyGSGLg87EuWaRMXx6wfPAkHH1HgqmwQ24AVRAVWUZdX84wGc\\\\\\\/HR6Kd5hkL5eIQGKcBhAGqt1ZCl45b54UkcR14T7mS2cliWgPvC7Hu7N+GokQMCXZtuDRCUhyygiN2m6e0p5YlzNvQ7VVRIr5MgdrE+Rn8\\\\\\\/VzIWPbbAIJs5\\\\\\\/0u5MlakmlMqCItf31RQB1yT+tX7tgRSCc6sM7xBXdEVD3u36YJIJwip8lEU+afRAKRwLVFNMBvn3LJsBf59pJMj0SJSJCQJu2BPXNVfi\\\\\\\/wsLg3iR\\\\\\\/KZoLGMRKDLqgYt\\\\\\\/6mNdA8CFAO9n1Ccved9x\\\\\\\/I50cB4cmV0djEYhbVEGxD0wtqZzbTr+u+X8fsAPmQ\\\\\\\/Cpiq3xeROm62praMnL0JyiuWgyxCauQ9euG\\\\\\\/c3Zmvd8nFWErhGsg\\\\\\\/LE2PNcHHo3wH3l7invnNeCj5Kw85r3zJ5uHCkfg882a0xo6QMtSS3Dqk1fFjcHkvPgZMHwWA3dJjFg6AhXx7j\\\\\\\/To0o9ITkWpOh1q\\\\\\\/eqJxYYpH90N1agRuMwVGCFsno\\\\\\\/qprm\\\\\\\/iuN2iTkYQgo0I5ExRQY72\\\\\\\/VKfXcTk5A7vEcmO7LXINlKFzD7XFn\\\\\\\/JWWm7c7m82wDb4n2rJno0+XL+iFw8dl2ppp0Uja\\\\\\\/RBe+LS1f0Stpr5BM5JtDpS\\\\\\\/AVGtGoG3pYqmwlhpZXaq0ZVv0b2ZODn9ukvQNqx9NCEaf0RK9LpG8niwacOB9\\\\\\\/IPBLuxWbpAffUTGfIwwsvQeuUJjdxCCizqiGItVXQ2jW6qdV8Y4W7UcXerdDSGaZDmZyV+Q+64koq1fsjL0IryywLXl3yE3XnQYA5Hd278Zm9m6gOnzyM+6+Eurzr89Dk7mHzedO5EaEw86XRzWIR9q3YwRc7ulkmBKkOMeKZVDY4EtbH8D89gYld0JraEk+j3BRA+TCepG0FKFPKtXXm3Mk5L1RKlwe3KvKcMoS4Ldd2zdP5aQc9mzsfpg3L99bP4pjyDGYvd9J1fzPvCfaiN+vw71TCgtlcYpyQ2Rng7nXV7PhhgB0tW2ooRo92S+z\\\\\\\/z0JFxeE2RLOSLJ\\\\\\\/7EvgUyQ9564u4BBnRmdniBLAHsSgfIjem+\\\\\\\/Kfo4a6neMscEcEWmm\\\\\\\/kksWcp4ExJtVaDikgCSbEucWb4rLxqXDa+vbq5HeJAAWgOclvlvFzjW7ETXCNg9wn59JeVKqHY4hPCwPSpo\\\\\\\/NWN2HxTZjX9gQiCDLivuE3\\\\\\\/WUNu9Yy\\\\\\\/O7XzQPxFvLdv8k8NtYBFNhYZ\\\\\\\/Mjgz\\\\\\\/uXP9aM7NZO7lx5P26r8mZ9DEBu3L2NrzAHx\\\\\\\/Irqh1DaD4SP9KRumq6m02PlNP8w8B0VRe4x+dOygSjguRUsZ2PWBKvI68PgyJVrJg5HzPrqY0ifWkj+Qj2WP4aChIlwG+HjI1vc30s6JfBtTiKOShM8e57aWSirj6gsxKlwYWT2Ib65QMi1YZg7LnYOvgsK2MoPmb7f2XrQyvU1Rg++WtbRhwJ\\\\\\\/XJdwqqudUhf9t52Xw+6wdwmA\\\\\\\/GA==\\\u0022,\\\u0022iv\\\u0022:\\\u0022jWG2M6MVnEpFeIwOCzVAbw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022ce87af909b36ba6615472e8625a75fb8420713e462091253b98f766ca080258f\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-493738 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-prediction">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/8/2463" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-e7Yzl7nVTl0GryQM3aG66Xx-RdFUcvS2suGMAVZsIgc" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/8/2463" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-bAI031dwGXH33RWtTs89lhuYaCR_lPxB8QTHAWAn_q8" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/8/2447" title="GRAF-pop: A Fast Distance-Based Method To Infer Subject Ancestry from Multiple Genotype Datasets Without Principal Components Analysis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/8/2477" title="Protein Kinase D Is Dispensable for Development and Survival of Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493738" id="node493738" data-pisa="ggg;9/8/2463" data-pisa-master="ggg;g3.118.200986" data-apath="/ggg/9/8/2463.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Selection with Allele Dosage in <em>Panicum maximum</em> Jacq.</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0593-5003" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Letícia A. de C. Lara</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5324-906X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mateus F. Santos</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9436-3678" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Liana Jank</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-3030-7532" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lucimara Chiari</span>, <span class="highwire-citation-author" data-delta="4">Mariane de M. Vilela</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-5127-4448" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rodrigo R. Amadeu</span>, <span class="highwire-citation-author" data-delta="6">Jhonathan P. R. dos Santos</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-7106-8630" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Guilherme da S. Pereira</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0002-3115-1149" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Zhao-Bang Zeng</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0003-0634-3277" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Antonio Augusto F. Garcia</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2463-2475;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200986 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493738" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Letícia A. de C. Lara </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Luiz de Queiroz College of Agriculture / University of São Paulo (ESALQ/USP), Piracicaba, SP, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Let%C3%ADcia%2BA.%2Bde%2BC.%2BLara%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20C.%20Lara%20LA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALet%25C3%25ADcia%2BA.%2Bde%2BC.%2BLara%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0593-5003" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Letícia A. de C. Lara</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mateus F. Santos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Embrapa Beef Cattle, Campo Grande, MS, Brazil, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mateus%2BF.%2BSantos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Santos%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMateus%2BF.%2BSantos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5324-906X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mateus F. Santos</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Liana Jank </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Embrapa Beef Cattle, Campo Grande, MS, Brazil, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Liana%2BJank%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jank%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALiana%2BJank%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9436-3678" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Liana Jank</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Lucimara Chiari </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Embrapa Beef Cattle, Campo Grande, MS, Brazil, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lucimara%2BChiari%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chiari%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALucimara%2BChiari%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3030-7532" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lucimara Chiari</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Mariane de M. Vilela </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Embrapa Beef Cattle, Campo Grande, MS, Brazil, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mariane%2Bde%2BM.%2BVilela%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vilela%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMariane%2Bde%2BM.%2BVilela%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Rodrigo R. Amadeu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Luiz de Queiroz College of Agriculture / University of São Paulo (ESALQ/USP), Piracicaba, SP, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rodrigo%2BR.%2BAmadeu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Amadeu%20RR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARodrigo%2BR.%2BAmadeu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5127-4448" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rodrigo R. Amadeu</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jhonathan P. R. dos Santos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Luiz de Queiroz College of Agriculture / University of São Paulo (ESALQ/USP), Piracicaba, SP, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jhonathan%2BP.%2BR.%2Bdos%2BSantos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=dos%20Santos%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJhonathan%2BP.%2BR.%2Bdos%2BSantos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Guilherme da S. Pereira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>North Carolina State University (NCSU), Raleigh, NC</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guilherme%2Bda%2BS.%2BPereira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pereira%20GD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGuilherme%2Bda%2BS.%2BPereira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7106-8630" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Guilherme da S. Pereira</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Zhao-Bang Zeng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>North Carolina State University (NCSU), Raleigh, NC</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhao-Bang%2BZeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zeng%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZhao-Bang%2BZeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3115-1149" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Zhao-Bang Zeng</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Antonio Augusto F. Garcia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Luiz de Queiroz College of Agriculture / University of São Paulo (ESALQ/USP), Piracicaba, SP, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonio%2BAugusto%2BF.%2BGarcia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garcia%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAntonio%2BAugusto%2BF.%2BGarcia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0634-3277" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Antonio Augusto F. Garcia</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/8/2463" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:493738" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:493738/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/8/2463.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:493738" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:493738/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/8/2463.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:493738" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:493738/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/8/2463.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:493738" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:493738/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Genomic selection is an efficient approach to get shorter breeding cycles in recurrent selection programs and greater genetic gains with selection of superior individuals. Despite advances in genotyping techniques, genetic studies for polyploid species have been limited to a rough approximation of studies in diploid species. The major challenge is to distinguish the different types of heterozygotes present in polyploid populations. In this work, we evaluated different genomic prediction models applied to a recurrent selection population of 530 genotypes of <em>Panicum maximum</em>, an autotetraploid forage grass. We also investigated the effect of the allele dosage in the prediction, <em>i.e.</em>, considering tetraploid (GS-TD) or diploid (GS-DD) allele dosage. A longitudinal linear mixed model was fitted for each one of the six phenotypic traits, considering different covariance matrices for genetic and residual effects. A total of 41,424 genotyping-by-sequencing markers were obtained using 96-plex and <em>Pst</em>1 restriction enzyme, and quantitative genotype calling was performed. Six predictive models were generalized to tetraploid species and predictive ability was estimated by a replicated fivefold cross-validation process. GS-TD and GS-DD models were performed considering 1,223 informative markers. Overall, GS-TD data yielded higher predictive abilities than with GS-DD data. However, different predictive models had similar predictive ability performance. In this work, we provide bioinformatic and modeling guidelines to consider tetraploid dosage and observed that genomic selection may lead to additional gains in recurrent selection program of <em>P. maximum</em>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/plant-breeding" class="hw-term hw-article-keyword hw-article-keyword-plant-breeding" rel="nofollow">Plant Breeding</a></li><li class="kwd"><a href="/keyword/guinea-grass" class="hw-term hw-article-keyword hw-article-keyword-guinea-grass" rel="nofollow">Guinea Grass</a></li><li class="kwd"><a href="/keyword/quantitative-genotyping" class="hw-term hw-article-keyword hw-article-keyword-quantitative-genotyping" rel="nofollow">Quantitative Genotyping</a></li><li class="kwd"><a href="/keyword/polyploidy" class="hw-term hw-article-keyword hw-article-keyword-polyploidy" rel="nofollow">Polyploidy</a></li><li class="kwd"><a href="/keyword/genotyping-sequencing-gbs" class="hw-term hw-article-keyword hw-article-keyword-genotyping-by-sequencing-gbs" rel="nofollow">Genotyping-by-sequencing (GBS)</a></li><li class="kwd"><a href="/keyword/recurrent-genomic-selection" class="hw-term hw-article-keyword hw-article-keyword-recurrent-genomic-selection" rel="nofollow">Recurrent Genomic Selection</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">Genomic Prediction</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-3">Many agricultural and forage crops of economic importance are autotetraploids, such as potato (<em>Solanum tuberosum</em>) (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Allard 1960</a>), alfalfa (<em>Medicago sativa</em>) (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">McCoy and Bingham 1988</a>), and guinea grass (<em>Panicum maximum</em> Jacq.) (<a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Warmke 1954</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Jank <em>et al.</em> 2014</a>). For these species, marker alleles can be represented with different dosages ranging from 0 to 4. This allele dosage refers to the number of copies of the reference allele, ranging from <em>aaaa</em> for nulliplex, <em>Aaaa</em> for simplex (single dose), and so on up to <em>AAAA</em> (quadruplex), where <em>A</em> is the reference allele, for a biallelic marker, such as Single Nucleotide Polymorphisms (SNPs).</p><p id="p-4">Historically, genetic analysis in polyploids has been based on single dose markers, mostly for building genetic maps (<a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Wu <em>et al.</em> 1992</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Hackett and Luo 2003</a>). With the development of Next-Generation Sequencing (NGS) technologies and the advance of genetic and statistical methods, new possibilities have arisen to enable studies of the complex polyploid genomes for many crops (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Garcia <em>et al.</em> 2013</a>). In these species, the evaluation of SNP throughout the genome allows one to assess the relative abundance of each allele (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Voorrips <em>et al.</em> 2011</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Serang <em>et al.</em> 2012</a>; <a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Garcia <em>et al.</em> 2013</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Hackett <em>et al.</em> 2013</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Mollinari and Serang 2013</a>; <a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Schmitz Carley <em>et al.</em> 2017</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Gerard <em>et al.</em> 2018</a>).</p><p id="p-5">According to <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Osborn <em>et al.</em> (2003)</a>, the allele dosage is expected to be related to different expression levels of some target traits. Thus, polyploidy can increase the potential variation in its genic expression, reflecting in phenotypic variation. Therefore, the inclusion of allele dosage information has become important for genetic studies in polyploid species (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Garcia <em>et al.</em> 2013</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Hackett <em>et al.</em> 2013</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Endelman <em>et al.</em> 2018</a>). In addition, it will allow the development of new statistical models in autopolyploid species, such as in the forage <em>P. maximum</em>.</p><p id="p-6"><em>Panicum maximum</em> Jacq. (Syn. <em>Megathyrsus maximum</em> Jacq. B. K. Simon &amp; S. W. L. Jacobs) is a perennial tropical autotetraploid forage grass (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-1.gif"/></noscript></span></span>) that reproduces by facultative apomixis (<a id="xref-ref-79-2" class="xref-bibr" href="#ref-79">Warmke 1954</a>), which means a high percentage of asexual reproduction by seeds. Cross-breeding started with the doubling of chromosomes by colchicine of rare diploid sexual plants (<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-2.gif"/></noscript></span></span>) found in the center of origin in East Africa (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Pernès 1975</a>). This enabled the successful crossing between apomictic and sexual plants on a tetraploid level, giving rise to a progeny of sexual and apomictic hybrids in a 1:1 ratio (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Savidan 1978</a>). This means that it is possible to fix the hybrid vigor by apomixis in the <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-3.gif"/></noscript></span></span> generation. Also, superior sexual hybrids can be used as new parents in sexual × apomictic crosses or as donors of favorable alleles in sexual populations.</p><p id="p-7">Few breeding generations separate the founder diploid accessions from the current sexual parents, and increase in the frequency of favorable alleles by selection for traits such as disease resistance, regrowth, biomass and seasonal yield, and nutritive value is necessary. Currently, conventional methods such as half-sib recurrent selection (RS) scheme (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Hallauer and Carena 2012</a>) have been carried out in sexual populations to increase the general combining ability of sexual parents. However, RS is a long term program as each cycle requires from three to five years for developing, evaluating, selecting and recombining superior plants. The evaluation phase needs two years for phenotyping the target traits in different harvests and seasons (rainy and dry seasons), and is the most time and expensive phase of the RS program. Thus, methods that accelerate the RS cycles will speed up the development of superior sexual parents for crossing with apomictic plants and increase the probability of releasing improved apomictic cultivars in the market. This is one of the goals of EMBRAPA, the Brazilian Agricultural Research Corporation. The unit Embrapa Beef Cattle is responsible for forage breeding programs, where the main crops are <em>Brachiaria</em> spp. and <em>P. maximum</em> (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Jank <em>et al.</em> 2011</a>). This research was established into the <em>P. maximum</em> recurrent genomic selection program.</p><p id="p-8">Initially proposed by <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Meuwissen <em>et al.</em> (2001)</a>, genomic selection (GS) is an approach that uses statistical methods to predict breeding values from markers distributed throughout the genome. It uses all markers simultaneously to calculate the genomic estimated breeding values (GEBVs) of individuals for ranking and progeny selection (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Meuwissen <em>et al.</em> 2001</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Bernardo 2014</a>). GS has recently been proposed as a useful tool for rapid genetic gains in RS (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Massman <em>et al.</em> 2013</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Müller <em>et al.</em> 2017</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Zhang <em>et al.</em> 2017</a>). It is a promising approach to accelerate cycles of recurrent selection since an established prediction equation can be used repeatedly for multiple cycles of selection (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Müller <em>et al.</em> 2017</a>). Furthermore, GS promises to increase genetic gain per unit of time, reduce costs for phenotyping, and increase the accuracy of selection.</p><p id="p-9">Most of the available GS models were developed for diploids, and are not well adapted and evaluated for polyploid species. The huge challenge in working with polyploid species is in the correct distinction among different types of heterozygotes. Hidden heterozygotes lead to pseudo-diploid models, that are commonly applied in polyploid species (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Annicchiarico <em>et al.</em> 2015</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Biazzi <em>et al.</em> 2017</a>). However, the impact of this approximation for GS models has only been reported in a few studies (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Endelman <em>et al.</em> 2018</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Oliveira <em>et al.</em> 2019</a>).</p><p id="p-10">The goal of this research was to develop and to evaluate predictive models for genomic selection in <em>P. maximum</em>. For this purpose, we compared different predictive models considering tetraploid and diploid allele dosage to obtain the GEBVs. The genotype calling was performed to allow the identification of different dose levels, showing all heterozygous genotypes and six different GS models were extended to incorporate the tetraploid allele dosage. To our knowledge, this is the first study that includes tetraploid dosage in whole-genome regression models for genomic selection in this tropical forage grass, combined with a high throughput genotyping in a sexual recurrent selection population of <em>P. maximum</em>.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Panicum maximum population</h3><p id="p-11">We generated <em>P. maximum</em> multi-parental population using 20 selected plants (JA, S7, S13, S16, A42, B87, T103, T4610, A47, A72, B107, C48, C16, B22, Y34, C54, B74, B96, BX4, and B103) as male parents and 19 sexual plants (the same set of plants except B107) as female parents in a polycross mating design. These parents were selected based on relevant agronomic performance in the forage breeding program at Embrapa Beef Cattle during the last 30 years. After the crossing process, we synthesized 19 half-sibs progeny families, each family composed of 30 individuals, resulting in a total of 570 tetraploid sexual plants.</p></div><div id="sec-3" class="subsection"><h3>Phenotypic evaluation</h3><p id="p-12">The multi-parental population was evaluated in a split-plot randomized complete block design with six replications at Embrapa Beef Cattle, in Campo Grande city, Mato Grosso do Sul state, Brazil (20∘27’S, 54∘37’W, 530m). Half-sib progenies (represented by each female parent) were treated as the whole plot factor, while individuals were treated as sub-plot factors. Experimental units were 5 <em>m</em> by 2 <em>m</em> sized plots containing 5 individual plants of a family or 5 clones of one of three apomictic cultivars (B107, Mombaça, and Tanzânia). Thus, each block consisted of a total of 22 (19 half-sib families and three check cultivars) and a total of 110 individual plants (Figure S1). Therefore 660 plants were evaluated in the experiment, in which 570 were individuals from the 19 half-sib progenies and 90 were clones from the apomictic cultivars.</p><p id="p-13">Each individual plant was evaluated for six traits: leaf dry matter (LDM - g/plant), regrowth capacity (RC), percentage of leaf blade (PLB - %), organic matter (OM - %), crude protein (CP - %), and <em>in vitro</em> digestibility of organic matter (IVD - %). The RC trait was evaluated seven days after the harvest using density notes and speed notes of sprouted stems. We considered these traits due their agronomic importance for forage breeding, especially for better forage quality properties for animal consumption. Thus, forage value is measured indirectly by being converted into animal products (such as meat, milk, leather, and furs) (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Jank <em>et al.</em> 2011</a>). The first three traits were evaluated for eight harvests, during the years of 2013 (four harvests), 2014 (one harvest), and 2015 (three harvests), and the last three were evaluated for four harvests, during the years of 2013 and 2015 (two harvests each).</p></div><div id="sec-4" class="subsection"><h3>Statistical analysis of phenotypic data</h3><p id="p-14">For each trait, we fitted the following longitudinal linear mixed model to obtain predicted means later used in genomic selection analysis. The model is<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-4.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>where <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-5.gif"/></noscript></span></span> was the phenotypic value of the <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-6.gif"/></noscript></span></span> plant from the parent <em>j</em> in the block <em>k</em> at the harvest <em>l</em>, <em>μ</em> was the fixed overall mean, <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-7.gif"/></noscript></span></span> was the fixed effect of the <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-8.gif"/></noscript></span></span> harvest (<span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-9.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-10.gif"/></noscript></span></span> for LDM, RC, and PLB traits, and <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-11.gif"/></noscript></span></span> for OM, CP, and IVD traits), <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-12.gif"/></noscript></span></span> was the random effect of the <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-13.gif"/></noscript></span></span> block (<span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-14.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-15.gif"/></noscript></span></span>) at the harvest <em>l</em>, <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-16.gif"/></noscript></span></span> was the random effect of the <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-17.gif"/></noscript></span></span> parent (<span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-18.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-19.gif"/></noscript></span></span>) at the harvest <em>l</em>, <span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-20.gif"/></noscript></span></span> was the random interaction effect of block <em>k</em> and parent <em>j</em> at the harvest <em>l</em>, <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-21.gif"/></noscript></span></span> was the effect of the <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-22.gif"/></noscript></span></span> plant (<span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-23.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-24.gif"/></noscript></span></span>) at the harvest <em>l</em>, and <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-25.gif"/></noscript></span></span> was the random environmental error. The plant effects (<span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-26.gif"/></noscript></span></span>) were separated into two groups, in which <span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-27.gif"/></noscript></span></span> was the random effect of the <span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-28.gif"/></noscript></span></span> individual genotype (<span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-29.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-30.gif"/></noscript></span></span>) at the harvest <em>l</em>, and <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-31.gif"/></noscript></span></span> was the fixed effect of the <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-32.gif"/></noscript></span></span> apomictic cultivar (<span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-33.gif"/></noscript></span></span>, with <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-34.gif"/></noscript></span></span>) at the harvest <em>l</em>. For genotypes, the vector <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-35.gif"/></noscript></span></span> was assumed to follow a multivariate normal distribution with zero mean and genetic VCOV matrix <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-36.gif"/></noscript></span></span>, <em>i.e.</em>, <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-37.gif"/></noscript></span></span>. For residuals, <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-38.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-39.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-40.gif"/></noscript></span></span>. For parents, <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-41.gif"/></noscript></span></span> and for blocks, <span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-42.gif"/></noscript></span></span>.</p><p id="p-15">The choice of the genetic (<span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-43.gif"/></noscript></span></span>) and residual (<span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-44.gif"/></noscript></span></span>) VCOV matrices was performed hierarchically as follows. First, we picked the model with the best fit for the VCOV structures of the genetic effects among nine different VCOV structures (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). This was done assuming an identity matrix (ID) as the VCOV matrix of the residual effects. Then, this picked model was evaluated assuming nine different VCOV structures for residual effects. Finally, the model with the best fit for the VCOV structure of the residual effect was selected. The models’ goodness of fitness were computed with Akaike Information Criterion (AIC) (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Akaike 1974</a>) and Bayesian Information Criterion (BIC) (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Schwarz 1978</a>). These analyses were performed in the R package ASReml-R (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Butler <em>et al.</em> 2009</a>). The generalized heritability (broad-sense heritability) for each trait was calculated using the same model as before mentioned but considering <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-45.gif"/></noscript></span></span> matrix as CS and <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-46.gif"/></noscript></span></span> as ID, according to <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Cullis <em>et al.</em> (2006)</a> as:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-53.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>where, <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-54.gif"/></noscript></span></span> is the prediction error variance (average variance of individual plant comparisons) and <span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-55.gif"/></noscript></span></span> is the genetic variance.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493858/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493858/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Variance and covariance structures examined for genetic (<span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/T1/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/T1/embed/mml-math-47.gif"/></noscript></span></span> matrix) and residual effects (<span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/T1/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/T1/embed/mml-math-48.gif"/></noscript></span></span> matrix)</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-5" class="subsection"><h3>Molecular data</h3><p id="p-17">Due to losses of individuals in the field, we sequenced a total of 530 offspring. DNA of these 530 offspring were extracted using the DNeasy Plant kit (QIAGEN) and sequenced along with female parents each repeated twice. To provide a higher sequence depth, genotyping-by-sequencing (GBS) was conducted in NextSeq 500 platform for 96-plex <em>Pst</em>1 libraries and following the protocol from Genomic Diversity Facility, Cornell University (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Elshire <em>et al.</em> 2011</a>). Raw data were analyzed using Tassel-GBS pipeline (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Glaubitz <em>et al.</em> 2014</a>) modified to obtain the exact read counts for each SNP allele (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Pereira <em>et al.</em> 2018</a>). As this pipeline requires a reference genome and <em>P. maximum</em> does not have one, we aligned the GBS tags against six pseudo-references from other diploid and tetraploid forage genomes, and <em>P. maximum</em> transcriptomes: (i) <em>Panicum hallii</em> genome (v. 2.0, DOE-JGI; ∼554 Mb arranged in 9 chromosomes and 8,405 scaffolds; diploid forage); (ii) <em>Panicum virgatum</em> genome (v 1.0, DOE-JGI; ∼1,230 Mb arranged in total of 18 chromosomes, 9 chromosomes named as A and B each, and 220,646 contigs; allotetraploid forage); (iii) <em>Setaria italica</em> genome (v 2.2; ∼405.7 Mb arranged in 336 scaffolds; diploid forage) (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Bennetzen <em>et al.</em> 2012</a>); (iv) <em>Setaria viridis</em> genome (v 1.0, DOE-JGI; ∼394.9 Mb arranged in 9 chromosomes and 724 scaffolds; diploid forage); (v) transcriptome of <em>P. maximum</em> obtained by EMBRAPA institution (43,803 transcripts with mean length of 841.334 bases); and (vi) transcriptome of <em>P. maximum</em> obtained by UNICAMP institution (138,853 transcripts with mean length of 695.658 bases). All the genomes are available in Phytozome website (<a href="http://www.phytozome.jgi.doe.gov/">http://www.phytozome.jgi.doe.gov/</a>) (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Goodstein <em>et al.</em> 2011</a>). The transcriptomes were provided directly by the mentioned institutions. The transcriptome obtained by UNICAMP was published by <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">Toledo-Silva <em>et al.</em> (2013)</a>. The transcriptome obtained by EMBRAPA has not yet been published. Bowtie2 algorithm (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Langmead and Salzberg 2012</a>) was used to align tags against each reference with -D and -R parameters defined as 20 and 4, respectively, and with very-sensitive-local argument. Common reads that aligned to the different genomes were called as different SNP markers. Then, we performed a preliminary analysis evaluating the prediction accuracy of models containing all the markers against models containing only non-redundant markers in genomic prediction models. As predictive accuracies between them did not differ, we chose to use all the markers in the subsequent analyses.</p><p id="p-18">In Tassel-GBS pipeline, the minimum minor allele frequency (mnMAF) considered was 1%. The counting information derived from the Tassel-GBS pipeline was used to estimate the tetraploid allele dosage for each loci using SuperMASSA (<a id="xref-ref-69-2" class="xref-bibr" href="#ref-69">Serang <em>et al.</em> 2012</a>; <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Pereira <em>et al.</em> 2018</a>). This software is designed especially for SNP calling in polyploid species based on probabilistic graphical models. In SuperMASSA software, the minimum overall depth considered was 25 reads and the model used was “Generalized Population Model”. Markers were fitted and filtered to ploidy 4. Triallelic SNPs and markers with more than 5% of missing data were filtered out. Since the selection of markers with up to 5% of missing data are a very strict filter, it is expected to include few percentage of errors due to the imputation process. The imputation was made using random sampling considering the frequency of each dose within each marker. Redundancy among markers was analyzed and displayed using R package circlize (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Gu <em>et al.</em> 2014</a>). The name of the markers was formed by: reference genome plus chromosome number plus position of SNP in the chromosome.</p><p id="p-19">As linkage disequilibrium (LD) can affect the prediction accuracy of GS, the LD was estimated using squared Pearson correlation, <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-56.gif"/></noscript></span></span> (<a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Vos <em>et al.</em> 2017</a>). Correlations were calculated on tetraploid dosage (0, 1, 2, 3, and 4) between marker pairs with up 1500 bp of distance for each reference genome. The average <span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-57.gif"/></noscript></span></span> between adjacent markers were calculated and, subsequently, the pairwise correlations were pooled over all chromosomes for each reference genome. A principal component analysis (PCA) was performed on the genotype data to detect population structure and displayed using ggplot2 (<a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Wickham 2009</a>). The relatedness among individuals was assessed computing (i) a pedigree-based additive relationship matrix (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Amadeu <em>et al.</em> 2016</a>), and (ii) a marker-based genomic relationship matrix (<a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">VanRaden 2008</a>) using the R package AGHmatrix (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Amadeu <em>et al.</em> 2016</a>).</p></div><div id="sec-6" class="subsection"><h3>Genomic prediction models using tetraploid dosage</h3><p id="p-20">Here, we generalized well known GS models for diploid to tetraploid species using the information of tetraploid allele dosage. Our response variable was the marginal predicted values of individual plants across harvests (BLUPs considering all harvests simultaneously). We evaluated both bayesian and frequentist approaches: Bayesian Ridge Regression (BRR) (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Whittaker <em>et al.</em> 2000</a>; <a id="xref-ref-49-3" class="xref-bibr" href="#ref-49">Meuwissen <em>et al.</em> 2001</a>); Bayes A (BA) (<a id="xref-ref-49-4" class="xref-bibr" href="#ref-49">Meuwissen <em>et al.</em> 2001</a>); Bayes B (BB) (<a id="xref-ref-49-5" class="xref-bibr" href="#ref-49">Meuwissen <em>et al.</em> 2001</a>); Bayes C (BC) (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Habier <em>et al.</em> 2011</a>); Bayesian LASSO (BL) (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Park and Casella 2008</a>); and Genomic Best Linear Unbiased Predictor (GBLUP) (<a id="xref-ref-76-2" class="xref-bibr" href="#ref-76">VanRaden 2008</a>).</p><p id="p-21">All bayesian models expanded for tetraploid allele dosage (TD models) share the same predictive multiple linear regression model (a complete description of these models can be seen at <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Pérez and de los Campos (2014)</a> and <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">de los Campos <em>et al.</em> (2013)</a>),<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-58.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>where, y (<span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-59.gif"/></noscript></span></span>) is the <em>n</em> adjusted entry mean response vector (mean values across harvests obtained from the phenotypic analysis); <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-60.gif"/></noscript></span></span> is a vector of 1’s; <em>μ</em> is a scalar representing the population mean; X (<span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-61.gif"/></noscript></span></span>) is the tetraploid allele dosage incidence matrix of <em>p</em> marker loci coded as <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-62.gif"/></noscript></span></span> according to the copy number of the reference allele for individual <em>i</em> at marker <em>j</em>; β (<span class="inline-formula" id="inline-formula-60"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-63.gif"/></noscript></span></span>) is the vector of (unknown) marker with tetraploid dosage genetic (TDG) effects; and ε (<span class="inline-formula" id="inline-formula-61"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-64.gif"/></noscript></span></span>) is the vector of residual effects, <span class="inline-formula" id="inline-formula-62"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-65.gif"/></noscript></span></span>.</p><p id="p-22">Different assumptions of TDG effects were evaluated. BRR-TD model assumes that all marker loci share the same normal prior distribution, <span class="inline-formula" id="inline-formula-63"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-66.gif"/></noscript></span></span>, where the common genetic variance hyperparameter (<span class="inline-formula" id="inline-formula-64"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-67.gif"/></noscript></span></span>) follows a scaled inverse chi-squared hyperprior distribution <span class="inline-formula" id="inline-formula-65"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-68.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-66"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-69.gif"/></noscript></span></span> is the number of degrees of freedom and <span class="inline-formula" id="inline-formula-67"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-70.gif"/></noscript></span></span> is the scale parameter of the distribution.</p><p id="p-23">BA-TD model is an extension of the above model, which assumes that each TDG prior effects follows specific normal densities, <span class="inline-formula" id="inline-formula-68"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-71.gif"/></noscript></span></span>. As before, each specific genetic variance hyperparameter (<span class="inline-formula" id="inline-formula-69"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-72.gif"/></noscript></span></span>) follows a scaled inverse chi-squared distribution. Due to its property, we expect that BA-TD model tends to shrink TDG effects with different prior strength (desirable for highly parameterized models, <span class="inline-formula" id="inline-formula-70"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-73.gif"/></noscript></span></span>), as opposed to the BRR-TD that assumes a common genetic variance hyperparameter. Genetically, these assumptions mean that the analyzed traits are controlled by many genes of small effects and few genes of large effects.</p><p id="p-24">BB-TD model is an extension of BA-TD model, that takes into account the TDG effects prior as a mixture of two normal densities, <span class="inline-formula" id="inline-formula-71"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-74.gif"/></noscript></span></span>, where <em>δ</em> is the genetic variance hyperparameter assumed as a known infinitesimal small value. In this model, we can interpret the mixture proportion (<em>π</em>) as a known expectation of a Bernoulli random variable, that is, the expectation of which mixture component best describes the TDG effects (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Dos Santos <em>et al.</em> 2016</a>).</p><p id="p-25">BC-TD model is a parsimonious variant of the BB-TD model, which considers that all TDG effects follows a common mixture of two normal distributions, <span class="inline-formula" id="inline-formula-72"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-75.gif"/></noscript></span></span>.</p><p id="p-26">BL-TD model assumes that all markers follows specific normal priors, with genetic variance hyperprior given by the product <span class="inline-formula" id="inline-formula-73"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-76.gif"/></noscript></span></span>. However, the key difference of BL-TD is the assumption that one component of genetic variance hyperparameters follows exponential distributions, <span class="inline-formula" id="inline-formula-74"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-77.gif"/></noscript></span></span>. The hyperparameter <span class="inline-formula" id="inline-formula-75"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-78.gif"/></noscript></span></span> measures the knowledge (precision) about the genetic variance hyperparameter. As the usual procedure for all the above models, we assume the residual genetic variance hyperparameter follows <span class="inline-formula" id="inline-formula-76"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-79.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-79.gif"/></noscript></span></span>.</p><p id="p-27">The frequentist model GBLUP-TD was:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-80.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-80.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>where y, <span class="inline-formula" id="inline-formula-77"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-81.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-81.gif"/></noscript></span></span>, <em>μ</em>, and <span class="inline-formula" id="inline-formula-78"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-82.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-82.gif"/></noscript></span></span> are the same as previously defined; Z (<span class="inline-formula" id="inline-formula-79"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-83.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-83.gif"/></noscript></span></span>) is an identity indicence matrix, and g (<span class="inline-formula" id="inline-formula-80"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-84.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-84.gif"/></noscript></span></span>) is the vector mapping the individuals total dosage genetic effects (random effect). The GBLUP-TD model assumes that the random variable g follows a multivariate normal distribution, <span class="inline-formula" id="inline-formula-81"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-85.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-85.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-82"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-86.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-86.gif"/></noscript></span></span> represents the genetic variance of the population and <span class="inline-formula" id="inline-formula-83"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-87.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-87.gif"/></noscript></span></span>, where K is the genomic relationship matrix and A the additive relationship matrix (<a id="xref-ref-76-3" class="xref-bibr" href="#ref-76">VanRaden 2008</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Aguilar <em>et al.</em> 2010</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Isik <em>et al.</em> 2017</a>). This normalization is necessary to obtain a invertible genomic relationship matrix (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Isik <em>et al.</em> 2017</a>) and the use of such weight does not result in critical differences in the outputs (<a id="xref-ref-76-4" class="xref-bibr" href="#ref-76">VanRaden 2008</a>; <a id="xref-ref-1-2" class="xref-bibr" href="#ref-1">Aguilar <em>et al.</em> 2010</a>). The K matrix, for tetraploid organisms, can be expressed as <span class="inline-formula" id="inline-formula-84"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-88.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-88.gif"/></noscript></span></span>, were W is the centered marker score matrix. Each <span class="inline-formula" id="inline-formula-85"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-89.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-89.gif"/></noscript></span></span> on K can be interpreted as a correlation between genotypes of different individuals (genomic relationship), and each <span class="inline-formula" id="inline-formula-86"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-90.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-90.gif"/></noscript></span></span> as the correlation of the genotypes of an individual with itself (inbreeding). K and A matrices were computated using R package AGHmatrix (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Amadeu <em>et al.</em> 2016</a>).</p><p id="p-28">To fit all bayesian models, we used the R package BGLR (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Pérez and de los Campos 2014</a>), choosing the default package settings for all known hyperparameters. To obtain the posterior distribution of the unknown parameters and hyperparameters, we used the Gibbs sampler with 20,000 iterations; the first 2,000 cycles were discarded as burn in. The GBLUP-TD model was analyzed using the R package ASReml-R (<a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Butler <em>et al.</em> 2009</a>).</p></div><div id="sec-7" class="subsection"><h3>Model evaluation</h3><p id="p-29">The best statistical model was selected for each of the phenotypic traits. Cross-validation with fivefolds has been repeated 100 times for bayesian approaches (computationally intensive) and 1,000 times for the frequentist approach, to obtain an asymptotic empirical distribution of the predictive ability. In each replication, the population was randomly split into five disjoint subsets of genotypes. Whereas one subset was used as validation population (20% or 106 individuals), the remaining four were combined as training population (80% or 424 individuals) to predicte the left-out genotypes in the first population. Subsequently, another subset was used as validation population and the left-out genotypes of this set were predicted. These steps were repeated until all five subsets were used as validation population once.</p><p id="p-30">We calculated Pearson correlation between observed (y) and predicted (<span class="inline-formula" id="inline-formula-87"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-91.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-91.gif"/></noscript></span></span>) adjusted entry means for the validation sets, considering, simultaneously, all five cross-validations of each replication. Predictive ability was calculated as the mean of these correlations. In addition, we also derived the empirical distribution of the predicted residual error sums of squares (PRESS), given by the sum of squares of the difference between the predicted and observed adjusted entry means. The narrow-sense heritability was calculated as:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-92.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-92.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>where, <span class="inline-formula" id="inline-formula-88"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-93.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-93.gif"/></noscript></span></span> is the additive genetic variance and <span class="inline-formula" id="inline-formula-89"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-94.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-94.gif"/></noscript></span></span> is the phenotypic variance from GBLUP analysis. Therefore, this estimate corresponds to a genomic heritability, using the additive relationship matrix from molecular markers.</p></div><div id="sec-8" class="subsection"><h3>Genomic prediction models using diploid dosage</h3><p id="p-31">We also performed a comparison between GS models using diploid and tetraploid allele dosage (GS-DD and GS-TD models, respectively). To do so, the three possible heterozygotes (<em>Aaaa</em>, <em>AAaa</em>, and <em>AAAa</em>) were coded as a single heterozygote (<em>Aa</em>), while the two tetraploid homozygotes (<em>AAAA</em> and <em>aaaa</em>) as diploid homozygotes (<em>AA</em> and <em>aa</em>). Molecular data were filtered to eliminate markers containing only doses 0 and 1. This step was necessary since the offspring were closely related among them and most of the genotypes in the full data were classified as <em>aaaa</em> or <em>Aaaa</em> (84.27% and 12.57%, respectively). Therefore, the molecular data set decreased from 41,424 to 1,223 markers. In these new data, tetraploid molecular matrix contains 1,223 markers coded as 0, 1, 2, 3, and 4, and diploid molecular matrix contains the same 1,223 markers coded as 0, 1, and 2. Usual GBLUP model with diploid dosage (GBLUP-DD) was evaluated using R package ASReml-R (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Butler <em>et al.</em> 2009</a>), considering the genomics relationship matrix as described by <a id="xref-ref-76-5" class="xref-bibr" href="#ref-76">VanRaden (2008)</a>. Bayesian models with diploid dosage were evaluated using R package BGLR (<a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Pérez and de los Campos 2014</a>). The model evaluation was performed as previously described.</p></div><div id="sec-9" class="subsection"><h3>Data Availability</h3><p id="p-32">Phenotypic and molecular data and R code files can be found at: &lt;<a href="https://github.com/leticia-lara/PM_data">https://github.com/leticia-lara/PM_data</a>&gt;. GBS data have been submitted to the NCBI Sequence Read Archive with the BioProject ID: PR-JNA510446. Additional information can also be found in the supplementary material. Figure S1 contains an illustrative scheme of the experimental design. Figure S2 contains the proportion of missing data in molecular dataset. Figure S3 contains the heatmaps of relationship matrices. Figure S4 contains the principal component analysis. Figure S5 contains the linkage disequilibrium analysis. Table S1 contains model selection for phenotypic traits. Table S2 contains the alignment rate for each reference genome. Table S3 contains the PRESS of predictive models for each trait. Table S4 contains the mean predictive ability of genomic selection models using tetraploid dosage for all markers. Table S5 contains the comparison between individual plant values based on half-sib family means and the genomic prediction of individuals. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.7762958">https://doi.org/10.25387/g3.7762958</a>.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Phenotypic models</h3><p id="p-33">All VCOV structures selected for <span class="inline-formula" id="inline-formula-90"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-95.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-95.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-91"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-96.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-96.gif"/></noscript></span></span> matrices allowed for heterogeneous variation and/or correlations across harvests for genetic and residual effects (Table S1). When the selection by AIC was not in agreement with the selection by BIC, we calculated the differences between these two selected models considering both criteria and selected by the criteria that obtained the higher difference. For example, considering the OM trait and <span class="inline-formula" id="inline-formula-92"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-97.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-97.gif"/></noscript></span></span> matrix, the Po<em><sub>Het</sub></em> had the lowest AIC value (with AIC criteria of 2774.400 and BIC criteria of 2826.809) and the Po had the lowest BIC value (with AIC criteria of 2782.262 and BIC criteria of 2817.201). The differences between these two selected models were 7.862 for AIC (2782.262 - 2774.400) and 9.608 for BIC (2826.809 - 2817.201). As BIC criteria had the higher difference, this criteria was used and the Po matrix was selected for <span class="inline-formula" id="inline-formula-93"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-98.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-98.gif"/></noscript></span></span> matrix in OM trait. The most commom selected matrix for genetic effects was AR1<span class="inline-formula" id="inline-formula-94"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-99.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-99.gif"/></noscript></span></span>, which was selected for CP and IVD, and different VCOV matrices were selected for residual effects. Generalized heritabilities were 0.66, 0.42, 0.48, 0.89, 0.85, and 0.43 for OM, CP, IDV, LDM, RC, and PLB, respectively. On average, traits evaluated in four harvests had lower heritability than traits evaluated in eight harvests.</p></div><div id="sec-12" class="subsection"><h3>Genotype calling</h3><p id="p-34">Approximately 485 million of reads per lane were obtained from short read sequencing, where 81.73% were good barcoded reads. The alignment of 6,596,939 read tags using <em>P. hallii</em>, <em>P. virgatum</em>, <em>S. italica</em>, <em>S. viridis</em>, and two transcriptomes of <em>P. maximum</em> (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>) showed that the overall alignment ranged from 19.05 to 24.24%. Although transcriptomes obtained by UNICAMP had the highest overall alignment rate, transcriptomes obtained by EMBRAPA had the highest unique alignment rate, followed by <em>S. viridis</em> and <em>S. italica</em> (Table S2). A total of 476,904 markers were obtained as the sum of the final number of markers for each reference genome (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>).</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493905/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493905/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Overall alignment rate and total number of markers obtained after Tassel-GBS pipeline, allele dosage in SuperMASSA software, and selection of up to 5% of missing data (Filter NA), for each reference genome</span><div class="sb-div caption-clear"></div></div></div><p id="p-35">Due to the nature of GBS technique, the sequencing coverage of different samples are random. It is possible that the same genomic region was not sequenced for all samples. Furthermore, sequences that have mutation in the restriction site of the enzyme are also not observed (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Elshire <em>et al.</em> 2011</a>). Therefore, a large amount of missing data are expected. The reference genomes of <em>P. hallii</em>, <em>P. virgatum</em>, <em>S. italica</em>, and <em>S. viridis</em> had around 5,000 markers with 87% missing data (Figure S2). Markers with a high proportion of missing values were eliminated in subsequent steps. Besides, most part of the selected markers had 0% missing data after the Tassel-GBS pipeline. <em>S. italica</em>, <em>S. viridis</em>, and transcriptome obtained by EMBRAPA had more than 15,000 markers with 0% missing data.</p><p id="p-36">VCF files obtained from Tassel-GBS pipeline were used as input in the SuperMASSA software. A total of 78,289 markers was selected with minimum average depth for the population of 25 reads (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>). From this, 32,619 markers had more than 100 minimum overall depth. For example, the scatterplot of marker Hallii.1_3461595 (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>) shows the intuition of how SuperMASSA uses count reads of two alleles to classify individuals according to their genotype using a probabilistic graphical model (<a id="xref-ref-69-3" class="xref-bibr" href="#ref-69">Serang <em>et al.</em> 2012</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2463/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Tetraploid allele dosage for marker Hallii.1_3461595." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2090393975" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Tetraploid allele dosage for marker Hallii.1_3461595.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2463/F1.medium.gif" width="440" height="301"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/8/2463/F1.medium.gif" width="440" height="301"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2463/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2463/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493863" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-37" class="first-child">Tetraploid allele dosage for marker Hallii.1_3461595.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">High level of missing data can impact the performance of GS models, and precise imputation of missing entries can be complex, especially when considering polyploid genomes. Therefore, markers were selected with up to 5% of missing data, aiming to reduce imputation bias. The final number of markers was 41,424, which were used in GS models (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>). Subsequently, imputation was made using random sampling and considering the dose frequency for each marker.</p><p id="p-39">The redundancy among markers was inspected (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>) and a large similarity was verified within three specific groups of reference: <em>Panicum</em> genus, <em>Setaria</em> genus, and transcriptomes. This result is expected due to phylogenetic proximity of the groups (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Aliscioni <em>et al.</em> 2003</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Bennetzen <em>et al.</em> 2012</a>). More than half of the markers identified by the different reference genomes have non-redundant information, and 31,046 markers were classified as unique (74.95%). This may be due to the great genomic variability still persistent in each genome, since they are relatively new in terms of breeding.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2463/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Redundancy among markers. Regions in red represent redundant markers within each reference, while regions in black, pink, blue, green, and orange represent redundant markers among six, five, four, three, and two references, respectively. Gray regions represent markers with unique information for each reference." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2090393975" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Redundancy among markers. Regions in red represent redundant markers within each reference, while regions in black, pink, blue, green, and orange represent redundant markers among six, five, four, three, and two references, respectively. Gray regions represent markers with unique information for each reference.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2463/F2.medium.gif" width="440" height="372"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/8/2463/F2.medium.gif" width="440" height="372"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2463/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2463/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493791" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-40" class="first-child">Redundancy among markers. Regions in red represent redundant markers within each reference, while regions in black, pink, blue, green, and orange represent redundant markers among six, five, four, three, and two references, respectively. Gray regions represent markers with unique information for each reference.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Population structure and linkage disequilibrium</h3><p id="p-41">A trace of population structure was detected for the additive relationship matrix based on pedigree information (Figure S3A) as expected, since these pedigrees consider only female parental information. However, no clear population structure was detected for the genomic relationship matrix based on molecular markers (Figure S3B). We also confirmed a relevant trace of population structure for female parents by principal component analysis of their genotype data, in which the three first principal components explained, respectively, 20.88, 11.98, and 10.23% of the total variability in these parents (Figure S4A). On the other hand, the principal component analysis for the 19 half-sib families (530 individuals) did not detected the population structure in these population. Their first three principal components explained, respectively, 6.20, 4.91, and 4.03% of the total variability in the breeding population (Figure S4B).</p><p id="p-42">Average linkage disequilibrium (LD) between marker pairs was 0.3063, 0.3321, 0.3155, 0.3165, 0.2401, and 0.2810 for markers identified from <em>P. hallii</em>, <em>P. virgatum</em>, <em>S. italica</em>, <em>S. viridis</em>, and transcriptomes of <em>P. maximum</em> obtained by EMBRAPA and UNICAMP, respectively (Figure S5). The LD decay was lower for <em>P. hallii</em>, <em>S. italica</em>, and <em>S. viridis</em> than for others.</p></div><div id="sec-14" class="subsection"><h3>Genomic prediction using tetraploid dosage</h3><p id="p-43">Genomic selection models were evaluated using all markers (41,424 markers) and only non-redundant ones (31,046 markers). The predictive accuracy did not differ between these two data sets (results not shown), since the predictive models deal well with multicollinearity. From now on, only predictive models using all markers will be presented.</p><p id="p-44">Mean values of predictive ability ranged from 0.1610 (BL-TD for LDM) to 0.4229 (BB-TD for OM) (Table S4). LDM showed the lowest accuracies for all analyzed GS-TD models and OM showed the highest ones (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>). No clear difference in predictive ability was observed among models (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>). The BL-TD model for LDM showed the highest estimates of standardized PRESS (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>, Table S3).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2463/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison among six genomic selection models using tetraploid dosage (GS-TD) for organic matter (OM), in vitro digestibility of organic matter (IVD), crude protein (CP), leaf dry matter (LDM), regrowth capacity (RC), and percentage of leaf blade (PLB). Molecular data contains 41,424 markers. (A) Predictive ability. (B) Standardized Predicted Residual Error Sum of Squares." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2090393975" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison among six genomic selection models using tetraploid dosage (GS-TD) for organic matter (OM), &lt;em&gt;in vitro&lt;/em&gt; digestibility of organic matter (IVD), crude protein (CP), leaf dry matter (LDM), regrowth capacity (RC), and percentage of leaf blade (PLB). Molecular data contains 41,424 markers. (A) Predictive ability. (B) Standardized Predicted Residual Error Sum of Squares.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2463/F3.medium.gif" width="440" height="195"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/8/2463/F3.medium.gif" width="440" height="195"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2463/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2463/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493992" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-45" class="first-child">Comparison among six genomic selection models using tetraploid dosage (GS-TD) for organic matter (OM), <em>in vitro</em> digestibility of organic matter (IVD), crude protein (CP), leaf dry matter (LDM), regrowth capacity (RC), and percentage of leaf blade (PLB). Molecular data contains 41,424 markers. (A) Predictive ability. (B) Standardized Predicted Residual Error Sum of Squares.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Comparison between GS-TD and GS-DD models</h3><p id="p-46">The mean predictive ability showed clear differences between GS-TD and GS-DD models (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>). Almost all GS-TD models were superior to GS-DD models for most of the analyzed traits. The higest superiority was for LDM, with an average superiority of 50.96% for GS-TD model in relation to GS-DD model. The narrow-sense heritability ranged from 0.11 for LDM to 0.53 to OM. These results were consistent with those obtained for the predictive abilities, which is expected since both were calculated considering the additive effects (<a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2463/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison between genomic selection models using tetraploid dosage (GS-TD) and diploid dosage (GS-DD), for organic matter (OM), crude protein (CP), in vitro digestibility of organic matter (IVD), leaf dry matter (LDM), regrowth capacity (RC), and percentage of leaf blade (PLB). Molecular matrices containing 1,223 markers for each data set." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2090393975" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison between genomic selection models using tetraploid dosage (GS-TD) and diploid dosage (GS-DD), for organic matter (OM), crude protein (CP), &lt;em&gt;in vitro&lt;/em&gt; digestibility of organic matter (IVD), leaf dry matter (LDM), regrowth capacity (RC), and percentage of leaf blade (PLB). Molecular matrices containing 1,223 markers for each data set.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2463/F4.medium.gif" width="440" height="373"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/8/2463/F4.medium.gif" width="440" height="373"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2463/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2463/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493904" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-47" class="first-child">Comparison between genomic selection models using tetraploid dosage (GS-TD) and diploid dosage (GS-DD), for organic matter (OM), crude protein (CP), <em>in vitro</em> digestibility of organic matter (IVD), leaf dry matter (LDM), regrowth capacity (RC), and percentage of leaf blade (PLB). Molecular matrices containing 1,223 markers for each data set.</p><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493838/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493838/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Mean predictive ability of genomic selection models using tetraploid dosage (GS-TD) and diploid dosage (GS-DD) for organic matter (OM), crude protein (CP), <em>in vitro</em> digestibility of organic matter (IVD), leaf dry matter (LDM), regrowth capacity (RC), and percentage of leaf blade (PLB). GS-TD and GS-DD models with the highest mean predictive ability for each trait are indicated in bold. Molecular matrices containing 1,223 markers for each data set. The broad-sense heritability is <span class="inline-formula" id="inline-formula-95"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/T3/embed/mml-math-100.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/T3/embed/mml-math-100.gif"/></noscript></span></span> and the narrow-sense heritability is <span class="inline-formula" id="inline-formula-96"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/T3/embed/mml-math-101.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/T3/embed/mml-math-101.gif"/></noscript></span></span></span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><p id="p-48">The aim of this study was to develop predictive models considering allele dosage for autotetraploid species, with applications in <em>P. maximum</em>. We extended six different GS models to autotetraploid species and compared the accuracy of predicted breeding values of these models. To the best of our knowledge, this is for the first time where the efficiency of genomic selection models considering the information from hidden heterozygotes in the autotetraploid tropical forage <em>P. maximum</em> has been reported. Furthermore, we evaluated strategies for modeling residual effects during phenotypic analysis, and performing quantitative genotype calling in autotetraploid species. This methodology can be applied to other autotetraploid species as well as can be extended to species with higher ploidy levels.</p><p id="p-49">Before the development of GS models considering tetraploid allele dosage (GS-TD models), it is important to perform a phenotypic analysis carefully since the sucess of GS in breeding for quantitative traits largely depends of phenotyping process (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Cabrera-Bosquet <em>et al.</em> 2012</a>). We performed a two stage approach for genomic selection, where we considered only phenotypic data in the first stage and incorporated molecular data in the second stage. In the first stage, for each trait, we fitted a longitudinal linear mixed model and treated genotypes as random. We recognized there is a discussion about fit genotype as random in the first stage (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Schulz-Streeck <em>et al.</em> 2013</a>). However, as we have multiple harvests and we are modeling genotype effects nested within harvests, with VCOV structure for genotype values across harvests, the model requires to treat genotype as random. The gain in information by incorporating the correlations among harvests is corroborated in the VCOV structures selection, in which the matrices allowing heterogeneous genetic and residual variation as well as allowing correlation among harvests provided a better fit than other models for most analyzed traits (Table S1). Moreover, there was reported in the literature that the use of BLUPs (Best Linear Unbiased Predictor, and in this case, it refers to the marginal predicted values of individual plants across harvests) does not result in significant differences for selection purposes (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Galli <em>et al.</em> 2018</a>), and it has been already applied in other GS studies as a simple approach to include correlations among traits/harvests/environments (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Asoro <em>et al.</em> 2011</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Resende <em>et al.</em> 2017</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Ferrão <em>et al.</em> 2017</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">de Oliveira <em>et al.</em> 2018</a>). Furthermore, we calculated the deregressed BLUPs, but the GS models had lower predictive ability than using the marginal predicted values of individual plants across harvests (results not shown).</p><p id="p-50">In the second stage, molecular markers were considered in the predictive models. For this, the genotype calling took into consideration the allele dosage, discriminating among the five possible genotypes. According to <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Uitdewilligen <em>et al.</em> (2013)</a>, a high sequence depth is required to identify the genotypic class accurately for autotetraploid species, where 60–80 depth leads to 98.4% accuracy in genotypic calls. Reads were also selected with minimum overall depth of 100 reads. However, as the predictive ability of GS-TD models was similar for both criteria (results not shown), we chose to keep only the analysis with an overall depth of 25 reads (approximately 78.7% of markers selected with minimum overall depth of 25 reads were also selected with minimum overall depth of 100 reads). This is a less strict filter for the practical pipeline of breeding and contains a larger number of markers.</p><p id="p-51">The underlying assumption of genomic selection is the presence of SNPs at some loci in linkage disequilibrium (LD) with Quantitative Trait Loci (QTL) alleles that affect traits which are subject to selection (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Calus <em>et al.</em> 2008</a>). LD represents the non-random association between alleles at different loci and it can be estimated using the correlation among markers when the SNP alleles at those loci are coded with numerical values. <a id="xref-ref-78-2" class="xref-bibr" href="#ref-78">Vos <em>et al.</em> (2017)</a> calculated several estimators for LD in a simulated and real panel of tetraploid potato and concluded that LD<span class="inline-formula" id="inline-formula-99"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-104.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-104.gif"/></noscript></span></span> values provides the most consistent estimates of LD decay. This estimator consists of 90% percentile <span class="inline-formula" id="inline-formula-100"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-105.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-105.gif"/></noscript></span></span> the short-range LD. Short-range LD is calculated across a defined interval of physical distances between marker pairs (<a id="xref-ref-78-3" class="xref-bibr" href="#ref-78">Vos <em>et al.</em> 2017</a>). One major reason for the minor differences in prediction accuracies among prediction models is the high level of LD found in breeding populations (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Riedelsheimer <em>et al.</em> 2012</a>). <a id="xref-ref-63-2" class="xref-bibr" href="#ref-63">Riedelsheimer <em>et al.</em> (2012)</a> obtained similar accuracies of GS models in elite maize germplasm, which had high level of LD. Accuracies did not differ regardless whether the effect of large QTL were precisely captured or spread over a larger region. In this work, we also obtained a high level of LD between markers pairs with up 1500 bp of distance (<span class="inline-formula" id="inline-formula-101"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-106.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-106.gif"/></noscript></span></span> from 0.2401 to 0.3321), which can explain the similarity among GS-TD models for prediction purposes.</p><p id="p-52">Population structure is another important factor in GS and can result in biased estimates of the predict ability when it is not taken into account (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Riedelsheimer <em>et al.</em> 2013</a>). As our population is composed by half-sib families, a stratified sampling was performed for the cross-validation process, in which 20% of individuals in each family were taken from the validation population and 80% from the training population. Therefore, all families were represented in both populations. The GS models and the evaluation process were performed as described in Materials and Methods section. We did not observe differences in the predictive abilities between stratified sampling and random sampling (results not shown) and we are presenting in this paper only results using random sampling. This similarity is expected, since no clear population structure was detected in this population of <em>P. maximum</em> (Figure S3 and S4B). When we observe the genotypes of the parents, few parents are different from the others (Figure S4A). The absence of population structure occurs because most parents are closely related and the individuals analyzed have a high level of relationship among them. Similar results have been reported in wheat (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Arruda <em>et al.</em> 2015</a>) and in potato (<a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Sverrisdóttir <em>et al.</em> 2017</a>), with typical values for data with family structure, but without substantial population structure. As reported by <a id="xref-ref-73-2" class="xref-bibr" href="#ref-73">Sverrisdóttir <em>et al.</em> (2017)</a>, the absence of population structure could in principle be caused by an extremely narrow genetic base of the parents. This is in line with our study since all parents were derived from a common female tetraploidized ancestor.</p><p id="p-53">Synthetic populations are usually formed by crossing several parents and posteriorly cross-pollinating the F1 individuals for one or several generations (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Falconer and Mackay 1996</a>). These populations have played an important role in quantitative genetic research on gene action in complex heterotic traits and comparison of selection methods (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Hallauer <em>et al.</em> 2010</a>; <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Schopp <em>et al.</em> 2017</a>). As in our population, the number of parents is usually small and parents are often related, leading to small effective size (<span class="inline-formula" id="inline-formula-102"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-107.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-107.gif"/></noscript></span></span>) (<a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Müller <em>et al.</em> 2017</a>). We estimated broad and narrow sense heritabilities for the evaluated traits. Besides we did not expect high heritabilities for traits related with yield, we observed a high broad sense heritability (0.89) and low narrow sense heritability (0.11) for LDM (<a id="xref-table-wrap-3-3" class="xref-table" href="#T3">Table 3</a>). Such low value reflects the difficulty to breed for this trait. Differences in magnitude between these estimates were previously reported by <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Resende <em>et al.</em> (2014b)</a>, which observed broad-sense heritability ranging from 0.47 to 0.75 and narrow-sense heritability ranging from 0.15 to 0.24 for LDM considering different years of evaluation. For individual harvests, the same authors found a narrow-sense heritability of 0.05 for LDM. Similar to our results, broad-sense heritabilities were reported in <em>Paspalum</em> for LDM with 0.82 and PLB with 0.55 (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Pereira <em>et al.</em> 2017</a>). In respect to the nutritive traits, moderate values were reported by <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Matias <em>et al.</em> (2016)</a> for narrow-sense heritability considering the mean of seven harvests in <em>Brachiaria decumbens</em> progenies for CP (0.31) and IVD (0.14). Similar moderate values for CP and IVD were also reported in our study (<a id="xref-table-wrap-3-4" class="xref-table" href="#T3">Table 3</a>).</p><p id="p-54">The prediction of the breeding values was made using the marginal predicted values of individual plants across harvests that were obtained in the first stage of the analysis. The goals were to select individuals in the present recurrent selection cycle (already phenotypically evaluated) as well as to select non-phenotyped individuals from the next cycle. Since one selection cycle requires three to five years to complete (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Resende <em>et al.</em> 2014a</a>), <em>P. maximum</em> breeding program with genomic selection will reduce from five to one year for each recurrent cycle. This one year is necessary to grow the selected plants and cross them to obtain the new population. In addition, the <em>P. maximum</em> breeding programs for releasing cultivars will be benefited since superior sexual plants can be selected every year to, posteriorly, cross with apomictic plants. From these crosses, new apomictic hybrid combinations will be obtained and tested as new cultivars for releasing the best one in the market. <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Lipka <em>et al.</em> (2014)</a> applied genomic selection in <em>P. virgatum</em> species with the objective of evaluating genomic selection efficiency to accelerate breeding cycles in this species. Since <em>P. virgatum</em> is an allotetraploid species, the authors used diploid dosage and obtained high prediction accuracy for most of the traits, using association panel and considering seven morphological and thirteen biomass quality trais. Although analyzed traits were different from ours, the range of values were similar. The higher mean prediction accuracy obtained by <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Lipka <em>et al.</em> (2014)</a> was 0.52 for standability and the lower was -0.08 for minerals. Our higher mean predictive ability was 0.4229 for organic matter and the lower was 0.1610 for leaf dry matter (Table S4). Similar accuracies were also obtained for oats (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Asoro <em>et al.</em> 2011</a>), maize (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">González-Camacho <em>et al.</em> 2012</a>), and rice (<a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Spindel <em>et al.</em> 2015</a>). Despite the difference among traits, the accuracy did not differ among models. Besides that, we suggest to use GBLUP-TD model for all traits, since it is more intuitive for breeders and it is less computational demanding.</p><p id="p-55">We also compared the predictions of individual plant values based on genomic prediction at the level of half-sib family means with that at the individual level. To achieve this, we removed the maternal effect of the phenotypic model and included it in the genomic selection model. Then, we predicted individuals based only on the maternal effects and compared with prediction of individuals (Table S5). The genomic predicted values are more accurate because they capture not only information on the family means, but also the within-family deviations. The accuracy of prediction using only maternal effect ranged from 0.0452 for PLB to 0.1843 for LDM. The prediction of individuals using molecular markers and family effect ranged from 0.1786 for LDM to 0.3466 for OM. Therefore, the GS can operate on individual plants without needing trait data on sibs with considerable accuracy of prediction and genetic gains with selection.</p><p id="p-56">There are few studies on GS for autotetraploid species using allele dosage, most of them are for potato (<a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Slater <em>et al.</em> 2016</a>; <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Endelman <em>et al.</em> 2018</a>; <a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Stich and Inghelandt 2018</a>). To our knowledge, this is the first study that applied genomic selection with allele dosage in an autotetraploid tropical forage grass. <a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Slater <em>et al.</em> (2016)</a> developed an extension of genomic relationship matrix proposed by <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Yang <em>et al.</em> (2010)</a> for autotetraploids and applied genomic selection in potato. The authors found accuracies ranging from 0.2, under conditions of low heritability and small reference populations, to 0.8 in larger reference populations. <a id="xref-ref-21-4" class="xref-bibr" href="#ref-21">Endelman <em>et al.</em> (2018)</a> and <a id="xref-ref-72-2" class="xref-bibr" href="#ref-72">Stich and Inghelandt (2018)</a> analyzed the inclusion of additive and non-additive components (digenic dominance and two order epistasis) for the accuracy of predictive models. The inclusion of dominance effects in GS when selecting the per-se performance of the individuals is fundamental, mainly when working with species that are highly heterozygous, which allows dominance effects to contribute to phenotypic variation (<a id="xref-ref-72-3" class="xref-bibr" href="#ref-72">Stich and Inghelandt 2018</a>). Concordantly, <a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">Endelman <em>et al.</em> (2018)</a> extended the genomic relationship matrix proposed by <a id="xref-ref-76-6" class="xref-bibr" href="#ref-76">VanRaden (2008)</a> for autotetraploids and highlighted the importance of genotypic and additive values for selection. However, for both studies, when the goals is selecting new parents for several cycles, only the additive value needs to be considered since the contribution of digenic dominance diminishes exponentially over the generations (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Gallais 2003</a>; <a id="xref-ref-21-6" class="xref-bibr" href="#ref-21">Endelman <em>et al.</em> 2018</a>), in other words, non-additive effects are less efficiently transmitted to progeny. In our work, we modeled only additive effects since our study aims to select superior sexual parents in a recurrent genomic selection program in the current and next selection cycles.</p><p id="p-57">A comparison of GS-TD models with GS models considering the usual diploid allele dosage (GS-DD models) was also performed. Although being a rougly approximation, this strategy has been applied in several polyploid crops, such as alfafa (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Annicchiarico <em>et al.</em> 2015</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Biazzi <em>et al.</em> 2017</a>), and sugarcane (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Gouy <em>et al.</em> 2013</a>), as GS software for polyploids have only recently emerged. To investigate the impact of using diploidized markers, <a id="xref-ref-21-7" class="xref-bibr" href="#ref-21">Endelman <em>et al.</em> (2018)</a> also compared diploid and tetraploid dosage. The authors observed that the accuracy of the diploidized model was consistently lower. In our study, the accuracy of GS-DD models were also lower than for GS-TD models for all traits (<a id="xref-table-wrap-3-5" class="xref-table" href="#T3">Table 3</a> and <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). This reinforces that GS models using tetraploid allele dosage are superior for autotetraploid populations, and extensions can and should be applied to other ploidy levels.</p><div id="sec-17" class="subsection"><h3>Implementation of genomic selection in forage breeding</h3><p id="p-58">Recent research shows the potential of genomic selection to reshape crop breeding programs. In particular, the results obtained here imply that GS has great potential for <em>P. maximum</em> breeding programs, especially in a recurrent genomic selection program.</p><p id="p-59">Usually, one cycle of half-sib recurrent selection is split in: i) development of progenies; ii) phenotypic evaluation; and iii) selection and recombination of the best selected individuals to obtain an improved population (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). The recurrent genomic selection is a modification to get shorter breeding cycles and greater genetic gains with selection. Therefore, genomic prediction is implemented by genotyping and phenotyping the base population and estimating marker effects to predict hybrid performance in the subsequent recurrent selection cycles and recombine the best individuals based only on the GEBVs (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). The persistency of predictive accuracy in GS is fundamental for practical breeding because it determines the number of cycles that can be advanced until it is necessary to retrain the predictive models (<a id="xref-ref-51-4" class="xref-bibr" href="#ref-51">Müller <em>et al.</em> 2017</a>). This is because in each cycle of recombination and selection, the individuals of breeding population can accumulate genetic diversity and gene frequencies may differ from the training population (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Heffner <em>et al.</em> 2010</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bassi <em>et al.</em> 2016</a>). Therefore the breeder needs to update the GS models by phenotyping the population and re-estimating marker effects each three recurrent selection cycles (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Heffner <em>et al.</em> 2009</a>) or whenever necessary.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2463/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Forage breeding program in a: (A) recurrent phenotypic selection. (B) recurrent genomic selection." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2090393975" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Forage breeding program in a: (A) recurrent phenotypic selection. (B) recurrent genomic selection.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2463/F5.medium.gif" width="440" height="161"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/8/2463/F5.medium.gif" width="440" height="161"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2463/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2463/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493902" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-60" class="first-child">Forage breeding program in a: (A) recurrent phenotypic selection. (B) recurrent genomic selection.</p><div class="sb-div caption-clear"></div></div></div><p id="p-61">To compare the efficiency of GS approach with phenotypic breeding programs, the breeders’ equation can be used. In this equation, expected genetic gains per unit of time is defined as <span class="inline-formula" id="inline-formula-103"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-108.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-108.gif"/></noscript></span></span>, where <em>i</em> is the selection intensity, <em>r</em> is the selection accuracy, <span class="inline-formula" id="inline-formula-104"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-109.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-109.gif"/></noscript></span></span> is the square root of the additive genetic variance, and <em>T</em> is the length of time to complete one breeding selection cycle (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Falconer and Mackay 1996</a>). The success of GS approach is determined by its ability to predict phenotypes that were not evaluated as well as by its ability to increase the rate of <span class="inline-formula" id="inline-formula-105"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-110.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2463/embed/mml-math-110.gif"/></noscript></span></span> while maintaining affordable costs. Assuming similar selection intensity and similar genetic variance for both methods, greater gains per unit of time can be achieved as long as the reduction in the time of each breeding cycle by GS compensates for the reduction in selection accuracy (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Bassi <em>et al.</em> 2016</a>). Furthermore, GS becomes cheaper than phenotypic selection for traits that have a long generation time or are difficult and expensive to evaluate.</p><p id="p-62">Another crucial aspect for implementation of genomic selection in forage breeding programs is the generation of accurate genotypic data. Several softwares have been developed to perform quantitative genotype calling for autotetraploid species, such as R packages fitTetra (<a id="xref-ref-77-2" class="xref-bibr" href="#ref-77">Voorrips <em>et al.</em> 2011</a>) and ClusterCall (<a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Schmitz Carley <em>et al.</em> 2017</a>); for “diploidizing” tetraploid species, such as R package breadarrayMSV (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Gidskehaug <em>et al.</em> 2010</a>); and species with any ploidy level, such as R packages fitPoly (<a id="xref-ref-77-3" class="xref-bibr" href="#ref-77">Voorrips <em>et al.</em> 2011</a>), updog (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Gerard <em>et al.</em> 2018</a>), and software SuperMASSA (<a id="xref-ref-69-4" class="xref-bibr" href="#ref-69">Serang <em>et al.</em> 2012</a>; <a id="xref-ref-56-3" class="xref-bibr" href="#ref-56">Pereira <em>et al.</em> 2018</a>). Therefore, the evaluation of allele dosage for polyploid species has become more accessible by several available softwares.</p><p id="p-63">This is the first work of genomic selection in the tropical forage grass <em>P. maximum</em> which uses a high throughput genotyping and considers tetraploid allele dosage in Bayesian and frequentist models. GBS and allele dosage showed to be promising strategies for genomic analysis in autotetraploid species. Furthermore, the accuracy of predictive models and the time reduction in the breeding cycles justifies the implementation of genomic selection in <em>P. maximum</em> breeding programs.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-64">This work was supported by FAPESP (São Paulo Research Foundation), grants 2015/20659-2 and 2016/01279-7. Additional support was provided by CNPq (“Conselho Nacional do Desenvolvimento Científico e Tecnológico”), EMBRAPA (“Empresa Brasileira de Pesquisa Agropecuária”) number 02.14.01.013.00.03.003, UNIPASTO (“Associação para o Fomento à Pesquisa de Melhoramento de Forrageiras”), and CAPES (“Coordenação de Aperfeiçoamento de Pessoal de Nível Superior”). L. A. C. Lara was supported by FAPESP, grants 2015/20659-2 and 2016/01279-7. A. A. F. Garcia was supported by a productivity scholarship from CNPq and CAPES (“Bio Computacional 051/2013”). J. P. R. dos Santos was supported by FAPESP, grants 2017/03625-2 and 2017/25674-5. The authors thank all trainees of Embrapa Beef Cattle that helped on the phenotypic evaluations.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-65">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.7762958">https://doi.org/10.25387/g3.7762958</a>.</p></li><li class="fn" id="fn-3"><p id="p-66"><em>Communicating editor: J. Holland</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-12-19"><span class="received-label">Received </span>December 19, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-05-23"><span class="accepted-label">Accepted </span>May 23, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 de C. Lara <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.1" data-doi="10.3168/jds.2009-2730"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aguilar</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Misztal</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. L.</span> <span class="cit-name-surname">Johnson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Legarra</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Tsuruta</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title"><em>Hot topic</em>: A unified approach to utilize phenotypic, full pedigree, and genomic information for genetic evaluation of Holstein final score</span>. <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">743</span>–<span class="cit-lpage">752</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2009-2730</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DAguilar%26rft.auinit1%253DI.%26rft.volume%253D93%26rft.issue%253D2%26rft.spage%253D743%26rft.epage%253D752%26rft.atitle%253DHot%2Btopic%253A%2Ba%2Bunified%2Bapproach%2Bto%2Butilize%2Bphenotypic%252C%2Bfull%2Bpedigree%252C%2Band%2Bgenomic%2Binformation%2Bfor%2Bgenetic%2Bevaluation%2Bof%2BHolstein%2Bfinal%2Bscore.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2009-2730%26rft_id%253Dinfo%253Apmid%252F20105546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2009-2730&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20105546&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274102500032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.2" data-doi="10.1109/TAC.1974.1100705"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akaike</span>,  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">1974</span> <span class="cit-article-title">A new look at the statistical model identification</span>. <abbr class="cit-jnl-abbrev">IEEE Trans. Automat. Contr.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">716</span>–<span class="cit-lpage">723</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1109/TAC.1974.1100705</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DIEEE%2BTrans.%2BAutomat.%2BContr.%26rft.volume%253D19%26rft.spage%253D716%26rft_id%253Dinfo%253Adoi%252F10.1109%252FTAC.1974.1100705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1109/TAC.1974.1100705&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.3" data-doi="10.3732/ajb.90.5.796"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aliscioni</span>,  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. M.</span> <span class="cit-name-surname">Giussani</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. O.</span> <span class="cit-name-surname">Zuloaga</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. A.</span> <span class="cit-name-surname">Kellogg</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A molecular phylogeny of <em>Panicum</em> (Poaceae: Paniceae): tests of monophyly and phylogenetic placement within the Panicoideae</span>. <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">90</span>: <span class="cit-fpage">796</span>–<span class="cit-lpage">821</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3732/ajb.90.5.796</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BBotany%26rft.stitle%253DAm.%2BJ.%2BBot.%26rft.aulast%253DAliscioni%26rft.auinit1%253DS.%2BS.%26rft.volume%253D90%26rft.issue%253D5%26rft.spage%253D796%26rft.epage%253D821%26rft.atitle%253DA%2Bmolecular%2Bphylogeny%2Bof%2BPanicum%2B%2528Poaceae%253A%2BPaniceae%2529%253A%2Btests%2Bof%2Bmonophyly%2Band%2Bphylogenetic%2Bplacement%2Bwithin%2Bthe%2BPanicoideae%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.90.5.796%26rft_id%253Dinfo%253Apmid%252F21659176%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjg6IjkwLzUvNzk2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-9.8.2463.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Allard</span>,  <span class="cit-name-given-names">R. W.</span></span></li></ol><cite>, <span class="cit-pub-date">1960</span> Inheritance in autotetraploids. In <em>Principles of Plant Breeding</em>, edited by J. W. Sons, pp. 385–399, Wiley.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.5" data-doi="10.3835/plantgenome2016.01.0009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amadeu</span>,  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Cellon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">Olmstead</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A. F.</span> <span class="cit-name-surname">Garcia</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. F. R.</span> <span class="cit-name-surname">Resende</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">AGHmatrix: R package to construct relationship matrices for autotetraploid and diploid species: A blueberry example</span>. <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.01.0009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.01.0009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.01.0009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.6" data-doi="10.1186/s12864-015-2212-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Annicchiarico</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Nazzicari</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wei</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Pecetti</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Accuracy of genomic selection for alfalfa biomass yield in different reference populations</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">1020</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-2212-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D1020%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-2212-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-2212-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.7" data-doi="10.3835/plantgenome2014.05.0020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arruda</span>,  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. J.</span> <span class="cit-name-surname">Brown</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Lipka</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Krill</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Thurber</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic selection for predicting Head Blight Resistance in a Wheat Breeding Program</span>. <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2015.01.0003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D8%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2014.05.0020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2014.05.0020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.8" data-doi="10.3835/plantgenome2011.02.0007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Asoro</span>,  <span class="cit-name-given-names">F. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Newell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. D.</span> <span class="cit-name-surname">Beavis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. P.</span> <span class="cit-name-surname">Scott</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accuracy and training population design for genomic selection on quantitative traits in elite North American oats</span>. <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">132</span>–<span class="cit-lpage">144</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2011.02.0007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft.volume%253D4%26rft.spage%253D132%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2011.02.0007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2011.02.0007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.9" data-doi="10.1016/j.plantsci.2015.08.021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bassi</span>,  <span class="cit-name-given-names">F. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Bentley</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Charmet</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Ortiz</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Breeding schemes for the implementation of genomic selection in wheat (<em>Triticum</em> spp.)</span>. <abbr class="cit-jnl-abbrev">Plant Sci.</abbr> <span class="cit-vol">242</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">36</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.plantsci.2015.08.021</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BSci.%26rft.volume%253D242%26rft.spage%253D23%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.plantsci.2015.08.021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.plantsci.2015.08.021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.10" data-doi="10.1038/nbt.2196"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bennetzen</span>,  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Schmutz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Percifield</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hawkins</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Reference genome sequence of the model plant Setaria</span>. <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">555</span>–<span class="cit-lpage">561</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt.2196</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DBennetzen%26rft.auinit1%253DJ.%2BL.%26rft.volume%253D30%26rft.issue%253D6%26rft.spage%253D555%26rft.epage%253D561%26rft.atitle%253DReference%2Bgenome%2Bsequence%2Bof%2Bthe%2Bmodel%2Bplant%2BSetaria.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2196%26rft_id%253Dinfo%253Apmid%252F22580951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2196&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22580951&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.11" data-doi="10.2135/cropsci2013.05.0315"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bernardo</span>,  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomewide selection when major genes are known</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">68</span>–<span class="cit-lpage">75</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2013.05.0315</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D68%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2013.05.0315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2013.05.0315&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.12" data-doi="10.1371/journal.pone.0169234"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Biazzi</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Nazzicari</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Pecetti</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. C.</span> <span class="cit-name-surname">Brummer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Palmonari</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genome-wide association mapping and genomic selection for alfalda (<em>Medicago sativa</em>) forage quality traits</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">12</span>: <span class="cit-elocation-id">e0169234</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0169234</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D12%26rft.spage%253De0169234%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0169234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0169234&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>,  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. R.</span> <span class="cit-name-surname">Cullis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Gilmour</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Gogel</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <em>ASReml-R reference manual</em>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.14" data-doi="10.1111/j.1744-7909.2012.01116.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cabrera-Bosquet</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">von Zitzewitz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Serret</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Araus</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">High-throughput phenotyping and genomic selection: The frontiers of crop breeding converge</span>. <abbr class="cit-jnl-abbrev">J. Integr. Plant Biol.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">312</span>–<span class="cit-lpage">320</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1744-7909.2012.01116.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BIntegr%2BPlant%2BBiol%26rft.aulast%253DCabrera-Bosquet%26rft.auinit1%253DL.%26rft.volume%253D54%26rft.issue%253D5%26rft.spage%253D312%26rft.epage%253D320%26rft.atitle%253DHigh-throughput%2Bphenotyping%2Band%2Bgenomic%2Bselection%253A%2Bthe%2Bfrontiers%2Bof%2Bcrop%2Bbreeding%2Bconverge.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1744-7909.2012.01116.x%26rft_id%253Dinfo%253Apmid%252F22420640%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1744-7909.2012.01116.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22420640&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.15" data-doi="10.1534/genetics.107.080838"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>,  <span class="cit-name-given-names">M. P. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. H. E.</span> <span class="cit-name-surname">Meuwissen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. P. W.</span> <span class="cit-name-surname">Roos</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. F.</span> <span class="cit-name-surname">Veerkamp</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Accuracy of genomic selection using different methods to define haplotypes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">553</span>–<span class="cit-lpage">561</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.080838</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCalus%26rft.auinit1%253DM.%2BP.%2BL.%26rft.volume%253D178%26rft.issue%253D1%26rft.spage%253D553%26rft.epage%253D561%26rft.atitle%253DAccuracy%2Bof%2BGenomic%2BSelection%2BUsing%2BDifferent%2BMethods%2Bto%2BDefine%2BHaplotypes%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.080838%26rft_id%253Dinfo%253Apmid%252F18202394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc4LzEvNTUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.16" data-doi="10.1198/108571106X154443"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cullis</span>,  <span class="cit-name-given-names">B. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. B.</span> <span class="cit-name-surname">Smith</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. E.</span> <span class="cit-name-surname">Coombes</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">On the design of early generation variety trials with correlated data</span>. <abbr class="cit-jnl-abbrev">J. Agric. Biol. Environ. Stat.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">381</span>–<span class="cit-lpage">393</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1198/108571106X154443</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAgric.%2BBiol.%2BEnviron.%2BStat.%26rft.volume%253D11%26rft.spage%253D381%26rft_id%253Dinfo%253Adoi%252F10.1198%252F108571106X154443%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/108571106X154443&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.17" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Hickey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Pong-Wong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. D.</span> <span class="cit-name-surname">Daetwyler</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. P. L.</span> <span class="cit-name-surname">Calus</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-genome regression and prediction methods applied to plant and animal breeding</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.143313</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.18" data-doi="10.1007/s11032-018-0802-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Oliveira</span>,  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. M.</span> <span class="cit-name-surname">Pastina</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">da Costa Parrella</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. W.</span> <span class="cit-name-surname">Noda</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. L. F.</span> <span class="cit-name-surname">Simeone</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genomic prediction applied to high-biomass sorghum for bioenergy production</span>. <abbr class="cit-jnl-abbrev">Mol. Breed.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">49</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11032-018-0802-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBreed.%26rft.volume%253D38%26rft.spage%253D49%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11032-018-0802-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11032-018-0802-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.19" data-doi="10.1186/s12863-016-0392-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dos Santos</span>,  <span class="cit-name-given-names">J. P. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. P. M.</span> <span class="cit-name-surname">Pires</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. C. C.</span> <span class="cit-name-surname">Vasconcellos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. S.</span> <span class="cit-name-surname">Pereira</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. G. V.</span> <span class="cit-name-surname">Pinho</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genomic selection to resistance to <em>Stenocarpella maydis</em> in maize lines using DArTseq markers</span>. <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">86</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12863-016-0392-3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenet.%26rft.volume%253D17%26rft.spage%253D86%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12863-016-0392-3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12863-016-0392-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.20" data-doi="10.1371/journal.pone.0019379"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elshire</span>,  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Glaubitz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Sun</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Poland</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kawamoto</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A robust, simple genotyping-by-sequencing (GBS) approach for high diversity species</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-elocation-id">e19379</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0019379</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DElshire%26rft.auinit1%253DR.%2BJ.%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253De19379%26rft.epage%253De19379%26rft.atitle%253DA%2Brobust%252C%2Bsimple%2Bgenotyping-by-sequencing%2B%2528GBS%2529%2Bapproach%2Bfor%2Bhigh%2Bdiversity%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0019379%26rft_id%253Dinfo%253Apmid%252F21573248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0019379&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21573248&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.21" data-doi="10.1534/genetics.118.300685"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Endelman</span>,  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A. S.</span> <span class="cit-name-surname">Carley</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. C.</span> <span class="cit-name-surname">Bethke</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Coombs</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Clough</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genetic variance partitioning and genome-wide prediction with allele dosage information in autotetraploid potato</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">209</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">87</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.118.300685</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.300685%26rft_id%253Dinfo%253Apmid%252F29514860%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMjA5LzEvNzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzgvMjQ2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-book" id="cit-9.8.2463.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>,  <span class="cit-name-given-names">D. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. F. C.</span> <span class="cit-name-surname">Mackay</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source">Quantitative genetics in maize breeding</span>, <span class="cit-publ-name">Pearson</span>, ‎<span class="cit-publ-loc">London</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.23" data-doi="10.1007/s11295-017-1171-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferrão</span>,  <span class="cit-name-given-names">L. F. V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. G.</span> <span class="cit-name-surname">Ferrão</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A. G.</span> <span class="cit-name-surname">Ferrão</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. F. A.</span> <span class="cit-name-surname">Fonseca</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A. F.</span> <span class="cit-name-surname">Garcia</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A mixed model to multiple harvest-location trials applied to genomic prediction in <em>Coffea canephora</em></span>. <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">95</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11295-017-1171-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D13%26rft.spage%253D95%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-017-1171-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-017-1171-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-book" id="cit-9.8.2463.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gallais</span>,  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-source">Quantitative genetics and breeding methods in autopolyploid plants</span>, <span class="cit-publ-name">Institut National de la Recherche Agronomique</span>, <span class="cit-publ-loc">Paris, France</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.25" data-doi="10.2135/cropsci2017.07.0459"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galli</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. H.</span> <span class="cit-name-surname">Lyra</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. C.</span> <span class="cit-name-surname">Alves</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. C.</span> <span class="cit-name-surname">Granato</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Fritsche-Neto</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Impact of phenotypic correction method and missing phenotypic data on genomic prediction of maize hybrids</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">1481</span>–<span class="cit-lpage">1491</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2017.07.0459</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D58%26rft.spage%253D1481%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2017.07.0459%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2017.07.0459&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.26" data-doi="10.1038/srep03399"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garcia</span>,  <span class="cit-name-given-names">A. A. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Mollinari</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. G.</span> <span class="cit-name-surname">Marconi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O. R.</span> <span class="cit-name-surname">Serang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. R.</span> <span class="cit-name-surname">Silva</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">SNP genotyping allows an in-depth characterisation of the genome of sugarcane and other complex autopolyploids</span>. <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">3399</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep03399</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D3%26rft.spage%253D3399%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep03399%26rft_id%253Dinfo%253Apmid%252F24292365%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep03399&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24292365&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.27" data-doi="10.1534/genetics.118.301468"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerard</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. F. V.</span> <span class="cit-name-surname">Ferrão</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A. F.</span> <span class="cit-name-surname">Garcia</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Stephens</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Genotyping polyploids from messy sequencing data</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">210</span>: <span class="cit-fpage">789</span>–<span class="cit-lpage">807</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.118.301468</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.118.301468%26rft_id%253Dinfo%253Apmid%252F30185430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjEwLzMvNzg5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.28" data-doi="10.1093/bioinformatics/btq673"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gidskehaug</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kent</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Hayes</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Lien</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Genotype calling and mapping of multisite variants using an Atlantic salmon iSelect SNP array</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">303</span>–<span class="cit-lpage">310</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btq673</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq673%26rft_id%253Dinfo%253Apmid%252F21149341%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq673&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21149341&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286991300002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.29" data-doi="10.1371/journal.pone.0090346"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Glaubitz</span>,  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. M.</span> <span class="cit-name-surname">Casstevens</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Lu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Harriman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. J.</span> <span class="cit-name-surname">Elshire</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">TASSEL-GBS: A high capacity genotyping by sequencing analysis pipeline</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e90346</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0090346</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De90346%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0090346%26rft_id%253Dinfo%253Apmid%252F24587335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0090346&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24587335&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.30" data-doi="10.1007/s00122-012-1868-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">González-Camacho</span>,  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">De los Campos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pérez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Cairns</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-enabled prediction of genetic values using radial basis function neural networks</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">759</span>–<span class="cit-lpage">771</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-012-1868-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DGonzalez-Camacho%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D125%26rft.issue%253D4%26rft.spage%253D759%26rft.epage%253D771%26rft.atitle%253DGenome-enabled%2Bprediction%2Bof%2Bgenetic%2Bvalues%2Busing%2Bradial%2Bbasis%2Bfunction%2Bneural%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1868-9%26rft_id%253Dinfo%253Apmid%252F22566067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1868-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22566067&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.31" data-doi="10.1093/nar/gkr944"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goodstein</span>,  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Shu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Howson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Neupane</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. D.</span> <span class="cit-name-surname">Hayes</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Phytozome: a comparative platform for green plant genommics</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D1178</span>–<span class="cit-lpage">D1186</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkr944</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr944%26rft_id%253Dinfo%253Apmid%252F22110026%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr944&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22110026&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298601300176&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.32" data-doi="10.1007/s00122-013-2156-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gouy</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Rousselle</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Bastianelli</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Lecomte</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Bonnal</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Experimental assessment of the accuracy of genomic selection in sugarcane</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">2575</span>–<span class="cit-lpage">2586</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-013-2156-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D2575%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-013-2156-z%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-013-2156-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.33" data-doi="10.1093/bioinformatics/btu393"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Gu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Eils</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Schlesner</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Brors</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title"><em>circlize</em> implements and enhances circular visualization in R</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2811</span>–<span class="cit-lpage">2812</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu393</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu393%26rft_id%253Dinfo%253Apmid%252F24930139%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu393&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24930139&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000343082900017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.34" data-doi="10.1186/1471-2105-12-186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Fernando</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kizilkaya</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Garrick</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Extension of the bayesian alphabet for genomic selection</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">186</span>–<span class="cit-lpage">197</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-186</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.spage%253D186%26rft.epage%253D186%26rft.atitle%253DExtension%2Bof%2Bthe%2Bbayesian%2Balphabet%2Bfor%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-186%26rft_id%253Dinfo%253Apmid%252F21605355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21605355&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.35" data-doi="10.1093/jhered/esg066"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>,  <span class="cit-name-given-names">C. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z. W.</span> <span class="cit-name-surname">Luo</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">TetraploidMap: Construction of a linkage map in autotetraploid species</span>. <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">358</span>–<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/jhered/esg066</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DHackett%26rft.auinit1%253DC.%2BA.%26rft.volume%253D94%26rft.issue%253D4%26rft.spage%253D358%26rft.epage%253D359%26rft.atitle%253DTetraploidMap%253A%2BConstruction%2Bof%2Ba%2BLinkage%2BMap%2Bin%2BAutotetraploid%2BSpecies%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252Fesg066%26rft_id%253Dinfo%253Apmid%252F12920109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/jhered/esg066&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12920109&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000184922700012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.36" data-doi="10.1371/journal.pone.0063939"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hackett</span>,  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">McLean</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Bryan</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Linkage analysis and QTL mapping using SNP dosage data in a tetraploid potato mapping population</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e63939</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0063939</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De63939%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0063939%26rft_id%253Dinfo%253Apmid%252F23704960%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0063939&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23704960&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hallauer</span>,  <span class="cit-name-given-names">A. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Carena</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Recurrent selection methods to improve germplasm in maize</span>. <abbr class="cit-jnl-abbrev">Maydica</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">266</span>–<span class="cit-lpage">283</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMaydica%26rft.volume%253D57%26rft.spage%253D266%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-book" id="cit-9.8.2463.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hallauer</span>,  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Carena</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Miranda Filho</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-source">Quantitative genetics in maize breeding</span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.39" data-doi="10.2135/cropsci2009.11.0662"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>,  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Lorenz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Sorrells</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Plant breeding with genomic selection: gain per unit time and cost</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">1681</span>–<span class="cit-lpage">1690</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2009.11.0662</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D50%26rft.spage%253D1681%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2009.11.0662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2009.11.0662&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000281060300010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.40" data-doi="10.2135/cropsci2008.08.0512"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Heffner</span>,  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Sorrells</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Genomic selection for crop improvement</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">49</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2008.08.0512</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BScience%26rft.stitle%253DCrop%2BSci.%26rft.aulast%253DHeffner%26rft.auinit1%253DE.%2BL.%26rft.volume%253D49%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D12%26rft.atitle%253DGenomic%2BSelection%2Bfor%2BCrop%2BImprovement%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2008.08.0512%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2008.08.0512&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000263344300001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-book" id="cit-9.8.2463.41" data-doi="10.1007/978-3-319-55177-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Isik</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Holland</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Maltecca</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-source">Genetic data analysis for plant and animal breeding</span>, <span class="cit-publ-name">Springer Nature</span>, <span class="cit-publ-loc">New York</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-3-319-55177-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bdata%2Banalysis%2Bfor%2Bplant%2Band%2Banimal%2Bbreeding%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-3-319-55177-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-3-319-55177-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.42" data-doi="10.1071/CP13319"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jank</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. C.</span> <span class="cit-name-surname">Barrios</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. B.</span> <span class="cit-name-surname">Valle</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Simeão</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. F.</span> <span class="cit-name-surname">Alves</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The value of improved pastures to Brazilian beef production</span>. <abbr class="cit-jnl-abbrev">Crop Pasture Sci.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">1132</span>–<span class="cit-lpage">1137</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1071/CP13319</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BPasture%2BSci.%26rft.volume%253D65%26rft.spage%253D1132%26rft_id%253Dinfo%253Adoi%252F10.1071%252FCP13319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/CP13319&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.43" data-doi="10.1590/S1984-70332011000500005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jank</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. B.</span> <span class="cit-name-surname">Valle</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M. S.</span> <span class="cit-name-surname">Resende</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Breeding tropical forages</span>. <abbr class="cit-jnl-abbrev">Crop Breed. Appl. Biotechnol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">27</span>–<span class="cit-lpage">34</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1590/S1984-70332011000500005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BBreed.%2BAppl.%2BBiotechnol.%26rft.volume%253D11%26rft.spage%253D27%26rft_id%253Dinfo%253Adoi%252F10.1590%252FS1984-70332011000500005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1590/S1984-70332011000500005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.44" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>,  <span class="cit-name-given-names">B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Salzberg</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with Bowtie 2</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1923</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.45" data-doi="10.1371/journal.pone.0112227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lipka</span>,  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Lu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Cherney</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. S.</span> <span class="cit-name-surname">Buckler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Casler</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Accelerating the switchgrass (<em>Panicum virgatum</em> L.) breeding cycle using genomic selection approaches</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e112227</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0112227</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De112227%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0112227%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0112227&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.46" data-doi="10.2135/cropsci2012.02.0112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Massman</span>,  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.-J. G.</span> <span class="cit-name-surname">Jung</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Bernardo</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomewide selection <em>vs.</em> marker-assisted recurrent selection to improve grain yield and stover-quality traits for cellulosic ethanol in maize</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">58</span>–<span class="cit-lpage">66</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2012.02.0112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D53%26rft.spage%253D58%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2012.02.0112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2012.02.0112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.47" data-doi="10.1590/1984-70332016v16n2a18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Matias</span>,  <span class="cit-name-given-names">F. I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. C. L.</span> <span class="cit-name-surname">Barrios</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. B.</span> <span class="cit-name-surname">Valle</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. G.</span> <span class="cit-name-surname">Mateus</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. B.</span> <span class="cit-name-surname">Martins</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Estimate of genetic parameters in <em>Brachiaria decumbens</em> hybrids</span>. <abbr class="cit-jnl-abbrev">Crop Breed. Appl. Biotechnol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">122</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1590/1984-70332016v16n2a18</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BBreed.%2BAppl.%2BBiotechnol.%26rft.volume%253D16%26rft.spage%253D115%26rft_id%253Dinfo%253Adoi%252F10.1590%252F1984-70332016v16n2a18%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1590/1984-70332016v16n2a18&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-book" id="cit-9.8.2463.48"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McCoy</span>,  <span class="cit-name-given-names">T. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. T.</span> <span class="cit-name-surname">Bingham</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Cytology and cytogenetics of alfalfa</span>, pp. <span class="cit-fpage">737</span>–<span class="cit-lpage">776</span> in <span class="cit-source">Alfalfa and alfalfa improvement</span>, <span class="cit-vol">Vol. 29</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Hanson</span>,  <span class="cit-name-given-names">A. A.</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">D. K.</span> <span class="cit-name-surname">Barnes</span></span>, <span class="cit-ed"> <span class="cit-name-given-names">R. R.</span> <span class="cit-name-surname">Hill</span></span>. <span class="cit-publ-name">American Society of Agronomy</span>, <span class="cit-publ-loc">Madison, WI</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAlfalfa%2Band%2Balfalfa%2Bimprovement%26rft.volume%253D29%26rft.spage%253D737%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.49" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>,  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Hayes</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of total genetic value using genome-wide dense marker maps</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzgvMjQ2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-book" id="cit-9.8.2463.50"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mollinari</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Serang</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Quantitative SNP genotyping of polyploids with MassARRAY and other platforms</span>, pp. <span class="cit-fpage">215</span>–<span class="cit-lpage">241</span> in <span class="cit-source">Methods in Molecular Biology</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Walker</span>,  <span class="cit-name-given-names">J. M.</span></span> <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.51" data-doi="10.1534/g3.116.036582"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Müller</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Schopp</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Melchinger</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> Persistency of prediction accuracy and genetic gain in synthetic populations under recurrent genomic selection. G3 (Bethesda) 7: 801–811.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.116.036582</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.116.036582%26rft_id%253Dinfo%253Apmid%252F28064189%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjc6IjcvMy84MDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzgvMjQ2My5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oliveira</span>,  <span class="cit-name-given-names">I. d. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. F.</span> <span class="cit-name-surname">Resende</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. F. V.</span> <span class="cit-name-surname">Ferrão</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. R.</span> <span class="cit-name-surname">Amadeu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Endelman</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> Genomic prediction of autotetraploids; influence of relationship matrices, allele dosage, and continuous genotyping calls in phenotype prediction. G3 (Bethesda) 9: 1189–1198.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.119.400059</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.53" data-doi="10.1016/S0168-9525(03)00015-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Osborn</span>,  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Chris Pires</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Birchler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. L.</span> <span class="cit-name-surname">Auger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Jeffery Chen</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Understanding mechanisms of novel gene expression in polyploids</span>. <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">141</span>–<span class="cit-lpage">147</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0168-9525(03)00015-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DOsborn%26rft.auinit1%253DT.%2BC.%26rft.volume%253D19%26rft.issue%253D3%26rft.spage%253D141%26rft.epage%253D147%26rft.atitle%253DUnderstanding%2Bmechanisms%2Bof%2Bnovel%2Bgene%2Bexpression%2Bin%2Bpolyploids.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252803%252900015-5%26rft_id%253Dinfo%253Apmid%252F12615008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(03)00015-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12615008&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181584500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.54" data-doi="10.1198/016214508000000337"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Park</span>,  <span class="cit-name-given-names">T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Casella</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The Bayesian Lasso</span>. <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">681</span>–<span class="cit-lpage">686</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1198/016214508000000337</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D103%26rft.spage%253D681%26rft_id%253Dinfo%253Adoi%252F10.1198%252F016214508000000337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/016214508000000337&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000257897500025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.55" data-doi="10.1590/0034-737x201764010009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pereira</span>,  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">DallAgnol</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Saraiva</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Simioni</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. P. S.</span> <span class="cit-name-surname">Leães</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genetic gain in apomictic species of the genus <em>Paspalum</em></span>. <abbr class="cit-jnl-abbrev">Rev. Ceres</abbr> <span class="cit-vol">64</span>: <span class="cit-fpage">60</span>–<span class="cit-lpage">67</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1590/0034-737x201764010009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRev.%2BCeres%26rft.volume%253D64%26rft.spage%253D60%26rft_id%253Dinfo%253Adoi%252F10.1590%252F0034-737x201764010009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1590/0034-737x201764010009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.56" data-doi="10.1186/s12859-018-2433-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pereira</span>,  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A. F.</span> <span class="cit-name-surname">Garcia</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. R. A.</span> <span class="cit-name-surname">Margarido</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">A fully automated pipeline for quantitative genotype calling from next generation sequencing data in autopolyploids</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">398</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12859-018-2433-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D19%26rft.spage%253D398%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-018-2433-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-018-2433-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.57" data-doi="10.1534/genetics.114.164442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>,  <span class="cit-name-given-names">P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de los Campos</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide regression and prediction with the BGLR statistical package</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">495</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.164442</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164442%26rft_id%253Dinfo%253Apmid%252F25009151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzIvNDgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-thesis" id="cit-9.8.2463.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pernès</span>,  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <em>Organisation évolutive d’un groupe agamique: la section des Maximae du genre Panicum</em>. Ph.D. thesis, Orstom.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.59" data-doi="10.2135/cropsci2013.05.0353"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>,  <span class="cit-name-given-names">R. M. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Casler</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D. V.</span> <span class="cit-name-surname">de Resende</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">a</span> <span class="cit-article-title">Genomic selection in forage breeding: Accuracy and methods</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">143</span>–<span class="cit-lpage">156</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2135/cropsci2013.05.0353</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D143%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2013.05.0353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2013.05.0353&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.60" data-doi="10.2135/cropsci2013.05.0353"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>,  <span class="cit-name-given-names">R. M. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Jank</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. B. d.</span> <span class="cit-name-surname">Valle</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. L. V.</span> <span class="cit-name-surname">Bonato</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">b</span> <span class="cit-article-title">Biometrical analysis and selection of tetraploid progenies of <em>Panicum maximum</em> using mixed model methods</span>. <abbr class="cit-jnl-abbrev">Crop Sci.</abbr> <span class="cit-vol">54</span>: <span class="cit-fpage">143</span>–<span class="cit-lpage">156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCrop%2BSci.%26rft.volume%253D54%26rft.spage%253D143%26rft_id%253Dinfo%253Adoi%252F10.2135%252Fcropsci2013.05.0353%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2135/cropsci2013.05.0353&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.61" data-doi="10.1038/hdy.2017.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>,  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D. V.</span> <span class="cit-name-surname">Resende</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. F.</span> <span class="cit-name-surname">Silva</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. F.</span> <span class="cit-name-surname">Azevedo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. K.</span> <span class="cit-name-surname">Takahashi</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Assessing the expected response to genomic selection of individuals and families in <em>Eucalyptus</em> breeding with an additive-dominant model</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">255</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2017.37</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D119%26rft.spage%253D245%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2017.37%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2017.37&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.62" data-doi="10.1534/genetics.113.150227"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Endelman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Stange</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Sorrells</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-L.</span> <span class="cit-name-surname">Jannink</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic predictability of interconnected bi-parental maize populations</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">493</span>–<span class="cit-lpage">503</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.150227</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.150227%26rft_id%253Dinfo%253Apmid%252F23535384%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzIvNDkzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.63" data-doi="10.1186/1471-2164-13-452"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Riedelsheimer</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Technow</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Melchinger</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Comparison of whole-genome prediction models for traits with contrasting genetic architecture in a diversity panel of maize inbred lines</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">452</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-13-452</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DRiedelsheimer%26rft.auinit1%253DC.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D452%26rft.epage%253D452%26rft.atitle%253DComparison%2Bof%2Bwhole-genome%2Bprediction%2Bmodels%2Bfor%2Btraits%2Bwith%2Bcontrasting%2Bgenetic%2Barchitecture%2Bin%2Ba%2Bdiversity%2Bpanel%2Bof%2Bmaize%2Binbred%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-452%26rft_id%253Dinfo%253Apmid%252F22947126%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-452&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22947126&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.64"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Savidan</span>,  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> Genetic control of facultative apomixis and application in breeding <em>Panicum maximum</em>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.65" data-doi="10.1007/s00122-016-2845-5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmitz Carley</span>,  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Coombs</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. S.</span> <span class="cit-name-surname">Douches</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. C.</span> <span class="cit-name-surname">Bethke</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Palta</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Automated tetraploid genotype calling by hierarchical clustering</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">717</span>–<span class="cit-lpage">726</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2845-5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D717%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2845-5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2845-5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.66" data-doi="10.1534/genetics.116.193243"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schopp</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Müller</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Technow</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. E.</span> <span class="cit-name-surname">Melchinger</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Accuracy of genomic prediction in synthetic populations depending on the number of parents, relatedness, and ancestral linkage disequilibrium</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">205</span>: <span class="cit-fpage">441</span>–<span class="cit-lpage">454</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.116.193243</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.116.193243%26rft_id%253Dinfo%253Apmid%252F28049710%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMjA1LzEvNDQxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.67" data-doi="10.1007/s00122-012-1960-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schulz-Streeck</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. O.</span> <span class="cit-name-surname">Ogutu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.-P.</span> <span class="cit-name-surname">Piepho</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comparisons of single-stage and two stage approaches to genomic selection</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">82</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-012-1960-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D69%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1960-1%26rft_id%253Dinfo%253Apmid%252F22903736%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1960-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22903736&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.68" data-doi="10.1214/aos/1176344136"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwarz</span>,  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span> <span class="cit-article-title">Estimating a dimension of a model</span>. <abbr class="cit-jnl-abbrev">Ann. Stat.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">461</span>–<span class="cit-lpage">464</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1214/aos/1176344136</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BStat.%26rft.volume%253D6%26rft.spage%253D461%26rft_id%253Dinfo%253Adoi%252F10.1214%252Faos%252F1176344136%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1214/aos/1176344136&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1978EQ63300014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.69" data-doi="10.1371/journal.pone.0030906"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Serang</span>,  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Mollinari</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A. F.</span> <span class="cit-name-surname">Garcia</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Efficient exact maximum a posteriori computation for Bayesian SNP genotyping in polyploids</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e30906</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0030906</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DSerang%26rft.auinit1%253DO.%26rft.volume%253D7%26rft.issue%253D2%26rft.spage%253De30906%26rft.epage%253De30906%26rft.atitle%253DEfficient%2Bexact%2Bmaximum%2Ba%2Bposteriori%2Bcomputation%2Bfor%2Bbayesian%2BSNP%2Bgenotyping%2Bin%2Bpolyploids.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0030906%26rft_id%253Dinfo%253Apmid%252F22363513%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0030906&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22363513&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.70" data-doi="10.3835/plantgenome2016.02.0021"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Slater</span>,  <span class="cit-name-given-names">A. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. O. I.</span> <span class="cit-name-surname">Cogan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. W.</span> <span class="cit-name-surname">Forster</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Hayes</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. D.</span> <span class="cit-name-surname">Daetwyler</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Improving genetic gain with genomic selection in autotetraploid potato</span>. <abbr class="cit-jnl-abbrev">Plant Genome</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3835/plantgenome2016.02.0021</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.3835%252Fplantgenome2016.02.0021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3835/plantgenome2016.02.0021&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.71" data-doi="10.1371/journal.pgen.1004982"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Spindel</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Begum</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Akdemir</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Virk</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Collard</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic selection and association mapping in rice (<em>Oryza sativa</em>): Effect of trait, genetic architecture, training population composition, marker number and statistical model on accuracy of rice genomic selection in elite, tropical rice breeding line</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e1004982</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1004982</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D11%26rft.spage%253De1004982%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004982&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.72" data-doi="10.3389/fpls.2018.00159"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stich</span>,  <span class="cit-name-given-names">B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. V.</span> <span class="cit-name-surname">Inghelandt</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Prospects and potential uses of genomic prediction of key performance traits in tetraploid potato</span>. <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">159</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fpls.2018.00159</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D9%26rft.spage%253D159%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2018.00159%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2018.00159&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.73" data-doi="10.1007/s00122-017-2944-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sverrisdóttir</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Byrne</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. H. R.</span> <span class="cit-name-surname">Sundmark</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. O.</span> <span class="cit-name-surname">Johnsen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. H.</span> <span class="cit-name-surname">Grethe</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Genomic prediction of starch content and chipping quality in tetraploid potato using genotyping-by-sequencing</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">2091</span>–<span class="cit-lpage">2108</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-017-2944-y</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D2091%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-017-2944-y%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-017-2944-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.74" data-doi="10.1371/journal.pone.0070781"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Toledo-Silva</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. B.</span> <span class="cit-name-surname">Cardoso-Silva</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Jank</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">Souza</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">De Novo transcriptome assembly for the tropical grass <em>Panicum maximum</em> Jacq</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e70781</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0070781</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De70781%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0070781%26rft_id%253Dinfo%253Apmid%252F23923022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0070781&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23923022&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.75" data-doi="10.1371/journal.pone.0054583"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Uitdewilligen</span>,  <span class="cit-name-given-names">J. G. A. M. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.-M. A.</span> <span class="cit-name-surname">Wolters</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. B.</span> <span class="cit-name-surname">D’hoop</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. J. A.</span> <span class="cit-name-surname">Borm</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. G. F.</span> <span class="cit-name-surname">Visser</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A next-generation sequencing method for genotyping-by-sequencing of highly heterozygous autotetraploid potato</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0062355</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0054583%26rft_id%253Dinfo%253Apmid%252F23620825%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0054583&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23620825&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.76" data-doi="10.3168/jds.2007-0980"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">VanRaden</span>,  <span class="cit-name-given-names">P. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Efficient methods to compute genomic predictions</span>. <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">4414</span>–<span class="cit-lpage">4423</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2007-0980</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVanRaden%26rft.auinit1%253DP.%2BM.%26rft.volume%253D91%26rft.issue%253D11%26rft.spage%253D4414%26rft.epage%253D4423%26rft.atitle%253DEfficient%2BMethods%2Bto%2BCompute%2BGenomic%2BPredictions%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2007-0980%26rft_id%253Dinfo%253Apmid%252F18946147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2007-0980&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18946147&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260277200035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.77" data-doi="10.1186/1471-2105-12-172"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voorrips</span>,  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Gort</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Vosman</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genotype calling in tetraploid species from bi-allelic marker data using mixture models</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">172</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-172</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DVoorrips%26rft.auinit1%253DR.%2BE.%26rft.volume%253D12%26rft.spage%253D172%26rft.epage%253D172%26rft.atitle%253DGenotype%2Bcalling%2Bin%2Btetraploid%2Bspecies%2Bfrom%2Bbi-allelic%2Bmarker%2Bdata%2Busing%2Bmixture%2Bmodels.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-172%26rft_id%253Dinfo%253Apmid%252F21595880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-172&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21595880&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.78" data-doi="10.1007/s00122-016-2798-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vos</span>,  <span class="cit-name-given-names">P. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">João Paulo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. E.</span> <span class="cit-name-surname">Voorips</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. G. F.</span> <span class="cit-name-surname">Visser</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. J.</span> <span class="cit-name-surname">van Eck</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Evaluation of LD decay and various LD-decay estimators in simulated and SNP-array data of tetraploid potato</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">123</span>–<span class="cit-lpage">135</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-016-2798-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D130%26rft.spage%253D123%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-016-2798-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-016-2798-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.79" data-doi="10.2307/2438575"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Warmke</span>,  <span class="cit-name-given-names">H. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1954</span> <span class="cit-article-title">Apomixis in <em>Panicum maximum</em></span>. <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">5</span>–<span class="cit-lpage">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/j.1537-2197.1954.tb14297.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BBot.%26rft.volume%253D41%26rft.spage%253D5%26rft_id%253Dinfo%253Adoi%252F10.2307%252F2438575%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2307/2438575&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1954UF03700002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.80" data-doi="10.1017/S0016672399004462"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whittaker</span>,  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Thompson</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Denham</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Marker-assisted selection using ridge regression</span>. <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">75</span>: <span class="cit-fpage">249</span>–<span class="cit-lpage">252</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672399004462</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DWhittaker%26rft.auinit1%253DJ.%2BC.%26rft.volume%253D75%26rft.issue%253D2%26rft.spage%253D249%26rft.epage%253D252%26rft.atitle%253DMarker-assisted%2Bselection%2Busing%2Bridge%2Bregression.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672399004462%26rft_id%253Dinfo%253Apmid%252F10816982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672399004462&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10816982&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000086569100013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.81"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wickham</span>,  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <em>ggplot2: Elegant graphics for data analysis</em>. Springer-Verlag New York.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.82" data-doi="10.1007/BF00224274"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Burnquist</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Sorrells</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. L.</span> <span class="cit-name-surname">Tew</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. H.</span> <span class="cit-name-surname">Moore</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">The detection and estimation of linkage in polyploids using single-dose restriction fragments</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">83</span>: <span class="cit-fpage">294</span>–<span class="cit-lpage">300</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/BF00224274</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D83%26rft.spage%253D294%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00224274%26rft_id%253Dinfo%253Apmid%252F24202510%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00224274&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24202510&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1992HB49800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.83" data-doi="10.1038/ng.608"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Benyamin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. P.</span> <span class="cit-name-surname">McEvoy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Gordon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. K.</span> <span class="cit-name-surname">Henders</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Common SNPs explain a large proportion of the heritability for human height</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">565</span>–<span class="cit-lpage">569</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.608</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DJ.%26rft.volume%253D42%26rft.issue%253D7%26rft.spage%253D565%26rft.epage%253D569%26rft.atitle%253DCommon%2BSNPs%2Bexplain%2Ba%2Blarge%2Bproportion%2Bof%2Bthe%2Bheritability%2Bfor%2Bhuman%2Bheight.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.608%26rft_id%253Dinfo%253Apmid%252F20562875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.608&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20562875&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2463.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279242400007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2463.84" data-doi="10.1534/g3.117.043141"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pérez-Rodríguez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Burgueño</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Olsen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Buckler</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> Rapid cycling genomic selection in a multiparental tropical maize population. G3 (Bethesda) 7: 2315–2326.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.117.043141</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.117.043141%26rft_id%253Dinfo%253Apmid%252F28533335%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjcvNy8yMzE1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI0NjMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/8/2463.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/8/2447" title="GRAF-pop: A Fast Distance-Based Method To Infer Subject Ancestry from Multiple Genotype Datasets Without Principal Components Analysis" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/8/2477" title="Protein Kinase D Is Dispensable for Development and Survival of Drosophila melanogaster" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/8"><span>Volume 9 Issue 8, August 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/8.cover-source.jpg?itok=cSNYBBSf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/8/2463" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-rxEPD4IF7Tpan8mTL5oW_EXq575Eo1qyn_yGEiPZFSs" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-prediction" class="" data-icon-position="" data-hide-link-title="0">Genomic Prediction</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/8/2463" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/8/2463" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/8/2463" class="active" data-icon-position="" data-hide-link-title="0">Genomic Selection with Allele Dosage in Panicum maximum Jacq.</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/493738" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-lYfxdqrAEgfhacQ5vowYCwU4-GDJ9C4FRWc824DREOA" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/8/2463.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/8/2463" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-zIeYlJEHdDpsusoKv_lzR_EKNekv1CQP7EaBMNl8Exk" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/8/2463" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/8/2463.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493738" id="node493738--2" data-pisa="ggg;9/8/2463" data-pisa-master="ggg;g3.118.200986" data-apath="/ggg/9/8/2463.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Selection with Allele Dosage in <em>Panicum maximum</em> Jacq.</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0593-5003" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Letícia A. de C. Lara</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5324-906X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mateus F. Santos</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9436-3678" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Liana Jank</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-3030-7532" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lucimara Chiari</span>, <span class="highwire-citation-author" data-delta="4">Mariane de M. Vilela</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-5127-4448" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rodrigo R. Amadeu</span>, <span class="highwire-citation-author" data-delta="6">Jhonathan P. R. dos Santos</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-7106-8630" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Guilherme da S. Pereira</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0002-3115-1149" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Zhao-Bang Zeng</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0003-0634-3277" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Antonio Augusto F. Garcia</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2463-2475;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200986 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493738--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Letícia A. de C. Lara </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Luiz de Queiroz College of Agriculture / University of São Paulo (ESALQ/USP), Piracicaba, SP, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Let%C3%ADcia%2BA.%2Bde%2BC.%2BLara%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20C.%20Lara%20LA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALet%25C3%25ADcia%2BA.%2Bde%2BC.%2BLara%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0593-5003" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Letícia A. de C. Lara</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mateus F. Santos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Embrapa Beef Cattle, Campo Grande, MS, Brazil, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mateus%2BF.%2BSantos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Santos%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMateus%2BF.%2BSantos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5324-906X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mateus F. Santos</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Liana Jank </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Embrapa Beef Cattle, Campo Grande, MS, Brazil, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Liana%2BJank%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jank%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALiana%2BJank%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9436-3678" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Liana Jank</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Lucimara Chiari </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Embrapa Beef Cattle, Campo Grande, MS, Brazil, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lucimara%2BChiari%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chiari%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALucimara%2BChiari%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3030-7532" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lucimara Chiari</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Mariane de M. Vilela </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Embrapa Beef Cattle, Campo Grande, MS, Brazil, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mariane%2Bde%2BM.%2BVilela%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vilela%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMariane%2Bde%2BM.%2BVilela%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Rodrigo R. Amadeu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Luiz de Queiroz College of Agriculture / University of São Paulo (ESALQ/USP), Piracicaba, SP, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rodrigo%2BR.%2BAmadeu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Amadeu%20RR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARodrigo%2BR.%2BAmadeu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5127-4448" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rodrigo R. Amadeu</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jhonathan P. R. dos Santos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Luiz de Queiroz College of Agriculture / University of São Paulo (ESALQ/USP), Piracicaba, SP, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jhonathan%2BP.%2BR.%2Bdos%2BSantos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=dos%20Santos%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJhonathan%2BP.%2BR.%2Bdos%2BSantos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Guilherme da S. Pereira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>North Carolina State University (NCSU), Raleigh, NC</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guilherme%2Bda%2BS.%2BPereira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pereira%20GD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGuilherme%2Bda%2BS.%2BPereira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7106-8630" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Guilherme da S. Pereira</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Zhao-Bang Zeng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>North Carolina State University (NCSU), Raleigh, NC</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhao-Bang%2BZeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zeng%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZhao-Bang%2BZeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3115-1149" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Zhao-Bang Zeng</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Antonio Augusto F. Garcia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Luiz de Queiroz College of Agriculture / University of São Paulo (ESALQ/USP), Piracicaba, SP, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonio%2BAugusto%2BF.%2BGarcia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garcia%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAntonio%2BAugusto%2BF.%2BGarcia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0634-3277" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Antonio Augusto F. Garcia</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/8/2463" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/8/2463&amp;title=Genomic%20Selection%20with%20Allele%20Dosage%20in%20Panicum%20maximum%20Jacq." id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/8/2463&amp;title=Genomic%20Selection%20with%20Allele%20Dosage%20in%20Panicum%20maximum%20Jacq." id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/8/2463&amp;title=Genomic%20Selection%20with%20Allele%20Dosage%20in%20Panicum%20maximum%20Jacq." id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/8/2463&amp;text=Genomic%20Selection%20with%20Allele%20Dosage%20in%20Panicum%20maximum%20Jacq." id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/8/2463&amp;title=Genomic%20Selection%20with%20Allele%20Dosage%20in%20Panicum%20maximum%20Jacq." id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/8/2463&amp;t=Genomic%20Selection%20with%20Allele%20Dosage%20in%20Panicum%20maximum%20Jacq." id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/8/2463&amp;title=Genomic%20Selection%20with%20Allele%20Dosage%20in%20Panicum%20maximum%20Jacq." id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/8/2463&amp;title=Genomic%20Selection%20with%20Allele%20Dosage%20in%20Panicum%20maximum%20Jacq." id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/493738/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493738" id="node493738--3" data-pisa="ggg;9/8/2463" data-pisa-master="ggg;g3.118.200986" data-apath="/ggg/9/8/2463.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Selection with Allele Dosage in <em>Panicum maximum</em> Jacq.</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-0593-5003" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Letícia A. de C. Lara</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0001-5324-906X" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Mateus F. Santos</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9436-3678" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Liana Jank</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-3030-7532" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Lucimara Chiari</span>, <span class="highwire-citation-author" data-delta="4">Mariane de M. Vilela</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0001-5127-4448" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Rodrigo R. Amadeu</span>, <span class="highwire-citation-author" data-delta="6">Jhonathan P. R. dos Santos</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0002-7106-8630" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Guilherme da S. Pereira</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0002-3115-1149" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Zhao-Bang Zeng</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0003-0634-3277" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Antonio Augusto F. Garcia</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2463-2475;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200986 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493738--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Letícia A. de C. Lara </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Luiz de Queiroz College of Agriculture / University of São Paulo (ESALQ/USP), Piracicaba, SP, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Let%C3%ADcia%2BA.%2Bde%2BC.%2BLara%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20C.%20Lara%20LA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALet%25C3%25ADcia%2BA.%2Bde%2BC.%2BLara%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0593-5003" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Letícia A. de C. Lara</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Mateus F. Santos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Embrapa Beef Cattle, Campo Grande, MS, Brazil, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mateus%2BF.%2BSantos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Santos%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMateus%2BF.%2BSantos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5324-906X" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Mateus F. Santos</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Liana Jank </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Embrapa Beef Cattle, Campo Grande, MS, Brazil, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Liana%2BJank%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jank%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALiana%2BJank%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9436-3678" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Liana Jank</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Lucimara Chiari </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Embrapa Beef Cattle, Campo Grande, MS, Brazil, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lucimara%2BChiari%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chiari%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALucimara%2BChiari%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3030-7532" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Lucimara Chiari</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Mariane de M. Vilela </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Embrapa Beef Cattle, Campo Grande, MS, Brazil, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mariane%2Bde%2BM.%2BVilela%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vilela%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMariane%2Bde%2BM.%2BVilela%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Rodrigo R. Amadeu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Luiz de Queiroz College of Agriculture / University of São Paulo (ESALQ/USP), Piracicaba, SP, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Rodrigo%2BR.%2BAmadeu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Amadeu%20RR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ARodrigo%2BR.%2BAmadeu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-5127-4448" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Rodrigo R. Amadeu</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Jhonathan P. R. dos Santos </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Luiz de Queiroz College of Agriculture / University of São Paulo (ESALQ/USP), Piracicaba, SP, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jhonathan%2BP.%2BR.%2Bdos%2BSantos%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=dos%20Santos%20JP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJhonathan%2BP.%2BR.%2Bdos%2BSantos%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Guilherme da S. Pereira </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>North Carolina State University (NCSU), Raleigh, NC</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Guilherme%2Bda%2BS.%2BPereira%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pereira%20GD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AGuilherme%2Bda%2BS.%2BPereira%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-7106-8630" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Guilherme da S. Pereira</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Zhao-Bang Zeng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>North Carolina State University (NCSU), Raleigh, NC</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zhao-Bang%2BZeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zeng%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AZhao-Bang%2BZeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3115-1149" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Zhao-Bang Zeng</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Antonio Augusto F. Garcia </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Luiz de Queiroz College of Agriculture / University of São Paulo (ESALQ/USP), Piracicaba, SP, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonio%2BAugusto%2BF.%2BGarcia%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Garcia%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAntonio%2BAugusto%2BF.%2BGarcia%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0634-3277" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Antonio Augusto F. Garcia</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2463&amp;rft.epage=2475&amp;rft.atitle=Genomic%20Selection%20with%20Allele%20Dosage%20in%20Panicum%20maximum%20Jacq.&amp;rft.volume=9&amp;rft.issue=8&amp;rft.date=2019-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=de+C.+Lara%2C+Let%C3%ADcia+A.&amp;rft.au=Santos%2C+Mateus+F.&amp;rft.au=Jank%2C+Liana&amp;rft.au=Chiari%2C+Lucimara&amp;rft.au=Vilela%2C+Mariane+de+M.&amp;rft.au=Amadeu%2C+Rodrigo+R.&amp;rft.au=dos+Santos%2C+Jhonathan+P.+R.&amp;rft.au=Pereira%2C+Guilherme+da+S.&amp;rft.au=Zeng%2C+Zhao-Bang&amp;rft.au=Garcia%2C+Antonio+Augusto+F."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/493738/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/493738/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/493738/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/493738/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/493738/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/493738/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/493738/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/493738/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/493738/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/493738/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/493738/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/493738/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1113" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/769" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495557" id="node495557" data-pisa="ggg;10/2/539" data-pisa-master="ggg;g3.119.400866" data-apath="/ggg/10/2/539.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/539" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Prediction" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Prediction</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/8/2463" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/8/2463.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/8/2463.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/8/2463.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
