<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/8/2761" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/8/2761.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/8/2761.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/8/2761.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/8/2761.atom" />
<meta name="HW.pisa" content="ggg;9/8/2761" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Signatures of Recent Positive Selection in Enhancers Across 41 Human Tissues" />
<meta name="DC.Identifier" content="10.1534/g3.119.400186" />
<meta name="DC.Date" content="2019-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Moon et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Evolutionary changes in enhancers are widely associated with variation in human traits and diseases. However, studies comprehensively quantifying levels of selection on enhancers at multiple evolutionary periods during recent human evolution and how enhancer evolution varies across human tissues are lacking. To address these questions, we integrated a dataset of 41,561 transcribed enhancers active in 41 different human tissues (FANTOM Consortium) with whole genome sequences of 1,668 individuals from the African, Asian, and European populations (1000 Genomes Project). Our analyses based on four different metrics (Tajima’s D , F ST, H12, nS L) showed that ∼5.90% of enhancers showed evidence of recent positive selection and that genes associated with enhancers under very recent positive selection are enriched for diverse immune-related functions. The distributions of these metrics for brain and testis enhancers were often statistically significantly different and in the direction suggestive of less positive selection compared to those of other tissues; the same was true for brain and testis enhancers that are tissue-specific compared to those that are tissue-broad and for testis enhancers associated with tissue-enriched and non-tissue-enriched genes. These differences varied considerably across metrics and tissues and were generally in the form of changes in distributions’ shapes rather than shifts in their values. Collectively, these results suggest that many human enhancers experienced recent positive selection throughout multiple time periods in human evolutionary history, that this selection occurred in a tissue-dependent and immune-related functional context, and that much like the evolution of their protein-coding gene counterparts, the evolution of brain and testis enhancers has been markedly different from that of enhancers in other tissues." />
<meta name="DC.Contributor" content="Jiyun M. Moon" />
<meta name="DC.Contributor" content="John A. Capra" />
<meta name="DC.Contributor" content="Patrick Abbot" />
<meta name="DC.Contributor" content="Antonis Rokas" />
<meta name="article:published_time" content="2019-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Signatures of Recent Positive Selection in Enhancers Across 41 Human Tissues" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Evolutionary changes in enhancers are widely associated with variation in human traits and diseases. However, studies comprehensively quantifying levels of selection on enhancers at multiple evolutionary periods during recent human evolution and how enhancer evolution varies across human tissues are lacking. To address these questions, we integrated a dataset of 41,561 transcribed enhancers active in 41 different human tissues (FANTOM Consortium) with whole genome sequences of 1,668 individuals from the African, Asian, and European populations (1000 Genomes Project). Our analyses based on four different metrics (Tajima’s &lt;i&gt;D&lt;/i&gt;, &lt;i&gt;F&lt;/i&gt;&lt;sub&gt;ST&lt;/sub&gt;, H12, &lt;i&gt;nS&lt;/i&gt;&lt;sub&gt;L&lt;/sub&gt;) showed that ∼5.90% of enhancers showed evidence of recent positive selection and that genes associated with enhancers under very recent positive selection are enriched for diverse immune-related functions. The distributions of these metrics for brain and testis enhancers were often statistically significantly different and in the direction suggestive of less positive selection compared to those of other tissues; the same was true for brain and testis enhancers that are tissue-specific compared to those that are tissue-broad and for testis enhancers associated with tissue-enriched and non-tissue-enriched genes. These differences varied considerably across metrics and tissues and were generally in the form of changes in distributions’ shapes rather than shifts in their values. Collectively, these results suggest that many human enhancers experienced recent positive selection throughout multiple time periods in human evolutionary history, that this selection occurred in a tissue-dependent and immune-related functional context, and that much like the evolution of their protein-coding gene counterparts, the evolution of brain and testis enhancers has been markedly different from that of enhancers in other tissues.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/08/01" />
<meta name="citation_mjid" content="ggg;9/8/2761" />
<meta name="citation_id" content="9/8/2761" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/8/2761" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/8/2761.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/8/2761.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/8/2761.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400186" />
<meta name="citation_pmid" content="31213516" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2761" />
<meta name="citation_lastpage" content="2774" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jiyun M. Moon" />
<meta name="citation_author_institution" content="Department of Biological Sciences," />
<meta name="citation_author" content="John A. Capra" />
<meta name="citation_author_institution" content="Department of Biological Sciences," />
<meta name="citation_author_institution" content="Vanderbilt Genetics Institute, and" />
<meta name="citation_author_institution" content="Departments of Biomedical Informatics and Computer Science, Vanderbilt University, Nashville, TN, 37235" />
<meta name="citation_author" content="Patrick Abbot" />
<meta name="citation_author_institution" content="Department of Biological Sciences," />
<meta name="citation_author" content="Antonis Rokas" />
<meta name="citation_author_institution" content="Department of Biological Sciences," />
<meta name="citation_author_institution" content="Vanderbilt Genetics Institute, and" />
<meta name="citation_author_institution" content="Departments of Biomedical Informatics and Computer Science, Vanderbilt University, Nashville, TN, 37235" />
<meta name="citation_author_email" content="antonis.rokas@vanderbilt.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-7248-6551" />
<meta name="citation_reference" content="citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=R. Andersson;citation_author=C. Gebhard;citation_author=I. Miguel-Escalada;citation_author=I. Hoof;citation_author=J. Bornholdt;citation_title=An atlas of active enhancers across human cell types and tissues;citation_volume=507;citation_year=2014;citation_pmid=24670763;citation_doi=10.1038/nature12787" />
<meta name="citation_reference" content="citation_journal_title=Human Molecular Genetics;citation_journal_abbrev=Hum Mol Genet;citation_author=P. L. Balaresque;citation_author=S. J. Ballereau;citation_author=M. A. Jobling;citation_title=Challenges in human genetic diversity: demographic history and adaptation;citation_pages=R134-R139;citation_volume=16;citation_year=2007;citation_issue=R2;citation_pmid=17911157;citation_doi=10.1093/hmg/ddm242" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. Brawand;citation_author=M. Soumillon;citation_author=A. Necsulea;citation_author=P. Julien;citation_author=G. Csárdi;citation_title=The evolution of gene expression levels in mammalian organs.;citation_pages=343-348;citation_volume=478;citation_year=2011;citation_issue=7369;citation_pmid=22012392;citation_doi=10.1038/nature10532" />
<meta name="citation_reference" content="citation_journal_title=Phil. Trans. Roy. Soc. B: Biol. Sci.;citation_author=J. A. Capra;citation_author=G. D. Erwin;citation_author=G. McKinsey;citation_author=J. L. R. Rubenstein;citation_author=K. S. Pollard;citation_title=Many human accelerated regions are developmental enhancers;citation_volume=368;citation_year=2013;citation_pmid=24218637;citation_doi=10.1098/rstb.2013.0025;citation_georef=2015117316" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=S. B. Carroll;citation_title=Evolution at two levels: on genes and form.;citation_pages=e245-e245;citation_volume=3;citation_year=2005;citation_issue=7;citation_pmid=16000021;citation_doi=10.1371/journal.pbio.0030245" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. Casper;citation_author=A. S. Zweig;citation_author=C. Villarreal;citation_author=C. Tyner;citation_author=M. L. Speir;citation_title=The UCSC Genome Browser database: 2018 update;citation_pages=D762-D769;citation_volume=46;citation_year=2018;citation_pmid=29106570;citation_doi=10.1093/nar/gkx1020" />
<meta name="citation_reference" content="citation_journal_title=GigaSci;citation_author=C. C. Chang;citation_author=C. C. Chow;citation_author=L. C. Tellier;citation_author=S. Vattikuti;citation_author=S. M. Purcell;citation_title=Second-generation PLINK: rising to the challenge of larger and richer datasets;citation_pages=7;citation_volume=4;citation_year=2015;citation_pmid=25722852;citation_doi=10.1186/s13742-015-0047-8" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. Danecek;citation_author=A. Auton;citation_author=G. Abecasis;citation_author=C. A. Albers;citation_author=E. Banks;citation_title=The variant call format and VCFtools;citation_volume=27;citation_year=2011;citation_pmid=21653522;citation_doi=10.1093/bioinformatics/btr330" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. Ferrer-Admetlla;citation_author=M. Liang;citation_author=T. Korneliussen;citation_author=R. Nielsen;citation_title=On Detecting Incomplete Soft or Hard Selective Sweeps Using Haplotype Structure;citation_volume=31;citation_year=2014;citation_pmid=24554778;citation_doi=10.1093/molbev/msu077" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=M. Fumagalli;citation_author=M. Sironi;citation_author=U. Pozzoli;citation_author=A. Ferrer-Admettla;citation_author=L. Pattini;citation_title=Signatures of environmental genetic adaptation pinpoint pathogens as the main selective pressure through human evolution.;citation_pages=e1002355-e1002355;citation_volume=7;citation_year=2011;citation_issue=11;citation_pmid=22072984;citation_doi=10.1371/journal.pgen.1002355" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=N. R. Garud;citation_author=P. W. Messer;citation_author=E. O. Buzbas;citation_author=D. A. Petrov;citation_title=Recent selective sweeps in North American Drosophila melanogaster show signatures of soft sweeps;citation_pages=e1005004;citation_volume=11;citation_year=2015;citation_pmid=25706129;citation_doi=10.1371/journal.pgen.1005004" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=M. Gershoni;citation_author=S. Pietrokovski;citation_title=Reduced selection and accumulation of deleterious mutations in genes exclusively expressed in men;citation_volume=5;citation_year=2014;citation_doi=10.1038/ncomms5438" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=S. Gravel;citation_author=B. M. Henn;citation_author=R. N. Gutenkunst;citation_author=A. R. Indap;citation_author=G. T. Marth;citation_title=Demographic history and rare allele sharing among human populations;citation_volume=108;citation_year=2011;citation_pmid=21730125;citation_doi=10.1073/pnas.1019276108" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=X. Gu;citation_author=Z. Su;citation_title=Tissue-driven hypothesis of genomic evolution and sequence-expression correlations;citation_volume=104;citation_year=2007;citation_pmid=17301236;citation_doi=10.1073/pnas.0610797104" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=B. C. Haller;citation_author=P. W. Messer;citation_title=SLiM 2: Flexible, Interactive Forward Genetic Simulations;citation_volume=34;citation_year=2017;citation_pmid=27702775;citation_doi=10.1093/molbev/msw211" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=R. B. Harris;citation_author=A. Sackman;citation_author=J. D. Jensen;citation_title=On the unfounded enthusiasm for soft selective sweeps II: Examining recent evidence from humans, flies, and viruses;citation_pages=e1007859;citation_volume=14;citation_year=2018;citation_doi=10.1371/journal.pgen.1007859" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=R. Haygood;citation_author=O. Fedrigo;citation_author=B. Hanson;citation_author=K.-D. Yokoyama;citation_author=G. A. Wray;citation_title=Promoter regions of many neural- and nutrition-related genes have experienced positive selection during human evolution.;citation_pages=1140-1144;citation_volume=39;citation_year=2007;citation_issue=9;citation_pmid=17694055;citation_doi=10.1038/ng2104" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=K. A. Frazer;citation_author=D. G. Ballinger;citation_author=D. R. Cox;citation_author=D. A. Hinds;citation_author=L. L. Stuve;citation_title=A second generation human haplotype map of over 3.1 million SNPs.;citation_pages=851-861;citation_volume=449;citation_year=2007;citation_issue=7164;citation_pmid=17943122;citation_doi=10.1038/nature06258" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=E. K. Karlsson;citation_author=D. P. Kwiatkowski;citation_author=P. C. Sabeti;citation_title=Natural selection and infectious disease in human populations;citation_pages=379-393;citation_volume=15;citation_year=2014;citation_pmid=24776769;citation_doi=10.1038/nrg3734" />
<meta name="citation_reference" content="citation_journal_title=Hum. Mol. Genet.;citation_author=J. M. Karnuta;citation_author=P. C. Scacheri;citation_title=Enhancers: bridging the gap between gene control and human disease;citation_pages=R219-R227;citation_volume=27;citation_year=2018;citation_doi=10.1093/hmg/ddy167" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=W. J. Kent;citation_author=C. W. Sugnet;citation_author=T. S. Furey;citation_author=K. M. Roskin;citation_author=T. H. Pringle;citation_title=The human genome browser at UCSC;citation_volume=12;citation_year=2002;citation_pmid=12045153;citation_doi=10.1101/gr.229102. Article published online before print in May 2002" />
<meta name="citation_reference" content="citation_journal_title=PLoS Biol.;citation_author=P. Khaitovich;citation_author=G. Weiss;citation_author=M. Lachmann;citation_author=I. Hellmann;citation_author=W. Enard;citation_title=A neutral model of transcriptome evolution;citation_pages=e132;citation_volume=2;citation_year=2004;citation_pmid=15138501;citation_doi=10.1371/journal.pbio.0020132" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=P. Khaitovich;citation_title=Parallel patterns of evolution in the genomes and transcriptomes of humans and chimpanzees;citation_volume=309;citation_year=2005;citation_pmid=16141373;citation_doi=10.1126/science.1108296" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. C. King;citation_author=A. C. Wilson;citation_title=Evolution at two levels in humans and chimpanzees;citation_pages=107-116;citation_volume=188;citation_year=1975;citation_issue=4184;citation_pmid=1090005;citation_doi=10.1126/science.1090005" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=N. Kryuchkova-Mostacci;citation_author=M. Robinson-Rechavi;citation_title=Tissue-specific evolution of protein coding genes in human and mouse;citation_pages=e0131673;citation_volume=10;citation_year=2015;citation_pmid=26121354;citation_doi=10.1371/journal.pone.0131673" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=S. Kudaravalli;citation_author=J. B. Veyrieras;citation_author=B. E. Stranger;citation_author=E. T. Dermitzakis;citation_author=J. K. Pritchard;citation_title=Gene expression levels are a target of recent natural selection in the human genome;citation_volume=26;citation_year=2008;citation_pmid=19091723;citation_doi=10.1093/molbev/msn289" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=T. I. Lee;citation_author=R. A. Young;citation_title=Transcriptional regulation and its misregulation in disease;citation_pages=1237-1251;citation_volume=152;citation_year=2013;citation_pmid=23498934;citation_doi=10.1016/j.cell.2013.02.014" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=H. K. Long;citation_author=S. L. Prescott;citation_author=J. Wysocka;citation_title=EVER-CHANGING LANDSCAPES: TRANSCRIPTIONAL ENHANCERS IN DEVELOPMENT AND EVOLUTION;citation_pages=1170-1187;citation_volume=167;citation_year=2016;citation_pmid=27863239;citation_doi=10.1016/j.cell.2016.09.018" />
<meta name="citation_reference" content="citation_journal_title=Cell Reports;citation_author=V. J. Lynch;citation_author=M. C. Nnamani;citation_author=A. Kapusta;citation_author=K. Brayer;citation_author=S. L. Plaza;citation_title=Ancient transposable elements transformed the uterine regulatory landscape and transcriptome during the evolution of mammalian pregnancy;citation_pages=551-561;citation_volume=10;citation_year=2015;citation_doi=10.1016/j.celrep.2014.12.052" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. MacArthur;citation_author=E. Bowler;citation_author=M. Cerezo;citation_author=L. Gil;citation_author=P. Hall;citation_title=The new NHGRI-EBI Catalog of published genome-wide association studies (GWAS Catalog);citation_volume=45;citation_year=2017;citation_pmid=27899670;citation_doi=10.1093/nar/gkw1133" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. W. Messer;citation_title=SLiM: Simulating evolution with selection and linkage;citation_volume=194;citation_year=2013;citation_pmid=23709637;citation_doi=10.1534/genetics.113.152181" />
<meta name="citation_reference" content="citation_journal_title=Trends Ecol. Evol.;citation_author=P. W. Messer;citation_author=D. A. Petrov;citation_title=Population genomics of rapid adaptation by soft selective sweeps;citation_pages=659-669;citation_volume=28;citation_year=2013;citation_pmid=24075201;citation_doi=10.1016/j.tree.2013.08.003" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=J. M. Moon;citation_author=D. M. Aronoff;citation_author=J. A. Capra;citation_author=P. Abbot;citation_author=A. Rokas;citation_title=Examination of signatures of recent positive selection on genes involved in human sialic acid biology;citation_volume=8;citation_year=2018;citation_pmid=29467190;citation_doi=10.1534/g3.118.200035" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=S. Neph;citation_author=M. S. Kuehn;citation_author=A. P. Reynolds;citation_author=E. Haugen;citation_author=R. E. Thurman;citation_title=BEDOPS: high-performance genomic feature operations;citation_volume=28;citation_year=2012;citation_pmid=22576172;citation_doi=10.1093/bioinformatics/bts277" />
<meta name="citation_reference" content="citation_journal_title=Science Advances;citation_journal_abbrev=Sci Adv;citation_author=S. Neubauer;citation_author=J.-J. Hublin;citation_author=P. Gunz;citation_title=The evolution of modern human brain shape;citation_pages=eaao5961-eaao5961;citation_volume=4;citation_year=2018;citation_issue=1;citation_doi=10.1126/sciadv.aao5961" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=Y. Nédélec;citation_author=J. Sanz;citation_author=G. Baharian;citation_author=Z. A. Szpiech;citation_author=A. Pacis;citation_title=Genetic ancestry and natural selection drive population differences in immune responses to pathogens;citation_pages=657-669.e21;citation_volume=167;citation_year=2016;citation_pmid=27768889;citation_doi=10.1016/j.cell.2016.09.025" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=R. Nielsen;citation_author=J. M. Akey;citation_author=M. Jakobsson;citation_author=J. K. Pritchard;citation_author=S. Tishkoff;citation_title=Tracing the peopling of the world through genomics;citation_pages=302-310;citation_volume=541;citation_year=2017;citation_pmid=28102248;citation_doi=10.1038/nature21347" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=C.-T. Ong;citation_author=V. G. Corces;citation_title=Enhancer function: new insights into the regulation of tissue-specific gene expression.;citation_pages=283-293;citation_volume=12;citation_year=2011;citation_issue=4;citation_pmid=21358745;citation_doi=10.1038/nrg2957" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. E. Page;citation_author=S. Viguier;citation_author=M. Dyble;citation_author=D. Smith;citation_author=N. Chaudhary;citation_title=Reproductive trade-offs in extant hunter-gatherers suggest adaptive mechanism for the Neolithic expansion;citation_volume=113;citation_year=2016;citation_pmid=27071109;citation_doi=10.1073/pnas.1524031113" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=L. A. Pennacchio;citation_author=W. Bickmore;citation_author=A. Dean;citation_author=M. A. Nobrega;citation_author=G. Bejerano;citation_title=Enhancers: five essential questions;citation_pages=288-295;citation_volume=14;citation_year=2013;citation_pmid=23503198;citation_doi=10.1038/nrg3458" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=P. S. Pennings;citation_author=J. Hermisson;citation_title=Soft sweeps II—molecular population genetics of adaptation from recurrent mutation or migration;citation_volume=23;citation_year=2006;citation_pmid=16520336;citation_doi=10.1093/molbev/msj117" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=B. Pfeifer;citation_author=U. Wittelsbürger;citation_author=S. E. Ramos-Onsins;citation_author=M. J. Lercher;citation_title=PopGenome: an efficient Swiss army knife for population genomic analyses in R;citation_volume=31;citation_year=2014;citation_pmid=24739305;citation_doi=10.1093/molbev/msu136" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=B. J. Raney;citation_author=T. R. Dreszer;citation_author=G. P. Barber;citation_author=H. Clawson;citation_author=P. A. Fujita;citation_title=Track data hubs enable visualization of user-defined genome-wide annotations on the UCSC Genome Browser;citation_volume=30;citation_year=2014;citation_pmid=24227676;citation_doi=10.1093/bioinformatics/btt637" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Genet. Dev.;citation_author=M. Rebeiz;citation_author=M. Tsiantis;citation_title=Enhancer evolution and the origins of morphological novelty;citation_pages=115-123;citation_volume=45;citation_year=2017;citation_doi=10.1016/j.gde.2017.04.006" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=M. V. Rockman;citation_author=M. W. Hahn;citation_author=N. Soranzo;citation_author=F. Zimprich;citation_author=D. B. Goldstein;citation_title=Ancient and recent positive selection transformed opioid cis-regulation in humans.;citation_pages=e387-e387;citation_volume=3;citation_year=2005;citation_issue=12;citation_pmid=16274263;citation_doi=10.1371/journal.pbio.0030387" />
<meta name="citation_reference" content="citation_journal_title=Phil. Trans. Roy. Soc. B: Biol. Sci.;citation_author=M. Rubinstein;citation_author=F. S. J. de Souza;citation_title=Evolution of transcriptional enhancers and animal diversity;citation_volume=368;citation_year=2013;citation_pmid=24218630;citation_doi=10.1098/rstb.2013.0017" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. C. Sabeti;citation_author=S. F. Schaffner;citation_author=B. Fry;citation_author=J. Lohmueller;citation_author=P. Varilly;citation_title=Positive natural selection in the human lineage.;citation_pages=1614-1620;citation_volume=312;citation_year=2006;citation_issue=5780;citation_pmid=16778047;citation_doi=10.1126/science.1124309;citation_georef=2006066590" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=S. F. Schaffner;citation_title=The X chromosome in population genetics.;citation_pages=43-51;citation_volume=5;citation_year=2004;citation_issue=1;citation_pmid=14708015;citation_doi=10.1038/nrg1247" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=S. J. Sholtis;citation_author=J. P. Noonan;citation_title=Gene regulation and the origins of human biological uniqueness.;citation_pages=110-118;citation_volume=26;citation_year=2010;citation_issue=3;citation_pmid=20106546;citation_doi=10.1016/j.tig.2009.12.009" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=C. N. Simonti;citation_author=M. Pavličev;citation_author=J. A. Capra;citation_title=Transposable element exaptation into regulatory regions is rare, influenced by evolutionary age, and subject to pleiotropic constraints;citation_volume=34;citation_year=2017;citation_doi=10.1093/molbev/msx219" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=Z. A. Szpiech;citation_author=R. D. Hernandez;citation_title=selscan: an efficient multithreaded program to perform EHH-based scans for positive selection;citation_volume=31;citation_year=2014;citation_pmid=25015648;citation_doi=10.1093/molbev/msu211" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=A. R. Forrest;citation_author=H. Kawaji;citation_author=M. Rehli;citation_author=J. K. Baillie;citation_title=A promoter-level mammalian expression atlas;citation_pages=462-470;citation_volume=507;citation_year=2014;citation_pmid=24670764;citation_doi=10.1038/nature13182" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=S. A. Tishkoff;citation_author=F. A. Reed;citation_author=A. Ranciaro;citation_author=B. F. Voight;citation_author=C. C. Babbitt;citation_title=Convergent adaptation of human lactase persistence in Africa and Europe;citation_pages=31-40;citation_volume=39;citation_year=2006;citation_pmid=17159977;citation_doi=10.1038/ng1946" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Uhlen;citation_author=L. Fagerberg;citation_author=B. M. Hallstrom;citation_author=C. Lindskog;citation_author=P. Oksvold;citation_title=Tissue-based map of the human proteome;citation_pages=1260419-1260419;citation_volume=347;citation_year=2015;citation_issue=6220;citation_pmid=25613900;citation_doi=10.1126/science.1260419" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Varki;citation_author=P. Gagneux;citation_title=Human-specific evolution of sialic acid targets: Explaining the malignant malaria mystery?;citation_volume=106;citation_year=2009;citation_pmid=19717444;citation_doi=10.1073/pnas.0908196106" />
<meta name="citation_reference" content="citation_journal_title=Annu. Rev. Genet.;citation_author=J. J. Vitti;citation_author=S. R. Grossman;citation_author=P. C. Sabeti;citation_title=Detecting natural selection in genomic data;citation_pages=97-120;citation_volume=47;citation_year=2013;citation_pmid=24274750;citation_doi=10.1146/annurev-genet-111212-133526" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=B. F. Voight;citation_author=S. Kudaravalli;citation_author=X. Wen;citation_author=J. K. Pritchard;citation_title=A map of recent positive selection in the human genome.;citation_pages=e72-e72;citation_volume=4;citation_year=2006;citation_issue=3;citation_pmid=16494531;citation_doi=10.1371/journal.pbio.0040072" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=J. Z. Wang;citation_author=Z. Du;citation_author=R. Payattakool;citation_author=P. S. Yu;citation_author=C. F. Chen;citation_title=A new method to measure the semantic similarity of GO terms;citation_volume=23;citation_year=2007;citation_pmid=17344234;citation_doi=10.1093/bioinformatics/btm087" />
<meta name="citation_reference" content="citation_journal_title=Evol;citation_author=B. S. Weir;citation_author=C. C. Cockerham;citation_title=Estimating F-Statistics for the Analysis of Population Structure;citation_pages=1358-1370;citation_volume=38;citation_year=1984;citation_doi=10.1111/j.1558-5646.1984.tb05657.x" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=G. A. Wray;citation_title=The evolutionary significance of cis-regulatory mutations.;citation_pages=206-216;citation_volume=8;citation_year=2007;citation_issue=3;citation_pmid=17304246;citation_doi=10.1038/nrg2063" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. Yu;citation_author=F. Li;citation_author=Y. Qin;citation_author=X. Bo;citation_author=Y. Wu;citation_title=GOSemSim: an R package for measuring semantic similarity among GO terms and gene products;citation_volume=26;citation_year=2010;citation_pmid=20179076;citation_doi=10.1093/bioinformatics/btq064" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Signatures of Recent Positive Selection in Enhancers Across 41 Human Tissues" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8.cover-source.jpg" />
<meta name="twitter:description" content="Evolutionary changes in enhancers are widely associated with variation in human traits and diseases. However, studies comprehensively quantifying levels of selection on enhancers at multiple evolutionary periods during recent human evolution and how enhancer evolution varies across human tissues are lacking. To address these questions, we integrated a dataset of 41,561 transcribed enhancers active in 41 different human tissues (FANTOM Consortium) with whole genome sequences of 1,668 individuals from the African, Asian, and European populations (1000 Genomes Project). Our analyses based on four different metrics (Tajima’s D , F ST, H12, nS L) showed that ∼5.90% of enhancers showed evidence of recent positive selection and that genes associated with enhancers under very recent positive selection are enriched for diverse immune-related functions. The distributions of these metrics for brain and testis enhancers were often statistically significantly different and in the direction suggestive of less positive selection compared to those of other tissues; the same was true for brain and testis enhancers that are tissue-specific compared to those that are tissue-broad and for testis enhancers associated with tissue-enriched and non-tissue-enriched genes. These differences varied considerably across metrics and tissues and were generally in the form of changes in distributions’ shapes rather than shifts in their values. Collectively, these results suggest that many human enhancers experienced recent positive selection throughout multiple time periods in human evolutionary history, that this selection occurred in a tissue-dependent and immune-related functional context, and that much like the evolution of their protein-coding gene counterparts, the evolution of brain and testis enhancers has been markedly different from that of enhancers in other tissues." />
<meta name="og-title" property="og:title" content="Signatures of Recent Positive Selection in Enhancers Across 41 Human Tissues" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/8/2761" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Evolutionary changes in enhancers are widely associated with variation in human traits and diseases. However, studies comprehensively quantifying levels of selection on enhancers at multiple evolutionary periods during recent human evolution and how enhancer evolution varies across human tissues are lacking. To address these questions, we integrated a dataset of 41,561 transcribed enhancers active in 41 different human tissues (FANTOM Consortium) with whole genome sequences of 1,668 individuals from the African, Asian, and European populations (1000 Genomes Project). Our analyses based on four different metrics (Tajima’s D , F ST, H12, nS L) showed that ∼5.90% of enhancers showed evidence of recent positive selection and that genes associated with enhancers under very recent positive selection are enriched for diverse immune-related functions. The distributions of these metrics for brain and testis enhancers were often statistically significantly different and in the direction suggestive of less positive selection compared to those of other tissues; the same was true for brain and testis enhancers that are tissue-specific compared to those that are tissue-broad and for testis enhancers associated with tissue-enriched and non-tissue-enriched genes. These differences varied considerably across metrics and tissues and were generally in the form of changes in distributions’ shapes rather than shifts in their values. Collectively, these results suggest that many human enhancers experienced recent positive selection throughout multiple time periods in human evolutionary history, that this selection occurred in a tissue-dependent and immune-related functional context, and that much like the evolution of their protein-coding gene counterparts, the evolution of brain and testis enhancers has been markedly different from that of enhancers in other tissues." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8.cover-source.jpg" />
    <title>Signatures of Recent Positive Selection in Enhancers Across 41 Human Tissues | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"JK3GARwtPrZjyOIK3t8yqxQpKxBWjOmel8ntscocWlg","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"493760","apath":"\/ggg\/9\/8\/2761.atom","pisa":"ggg;9\/8\/2761","ac":{"\/ggg\/9\/8\/2761.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/8\/2761.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/8\/2761"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/8\/2761"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/8\/2761":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022MjALzWvCxQMhRFhWfPUPT0M+mdu81NAODIGhdHAsACQRzT1e220UuHl4BA7WBIRPRNWjjqn4Kzkn5r2l9kSdaiVpo6WbizEWJRGCXflKOB3L4OY90J4ivNEJrNBUKZAM+jOyWtvlaqRVpP2+BU+ZaoVtmWMW+m4oCTBGzXXEDdOZ6wNRN0iXyo2XwdcLTCXP\\\\\\\/xrbgNDUPrOTtEkEcl3w2QTa6MBnBYZ2y7zqq2N2HS3OC2ngb54fKqg4du2t5HuC5LZQs2zx7IB\\\\\\\/5h89A\\\\\\\/Fan\\\\\\\/24QOLe8c\\\\\\\/SKFMN+kFojxnClRbavO0fpsgVkBoOMCzT1+qHx\\\\\\\/7Ouu23S5du0noP9P3JDmjkwUSNjuPY46+NVgefzj57m10AHs\\\\\\\/AxCWG2O6Galj\\\\\\\/WjOwX8oSg9sA76h8A537CxuG2HN+9fXqyyHK1TjDNGyY297O71oAlAvBtTlHiOJ7yJ61u81m7jq2FoZTksHOUvcHAZdCBl3x3ZpZq49uFDzikEfemZVxUIY\\\\\\\/0x3122aClyflcXaurnAVxkdnOzCBS6TmQ6D\\\\\\\/6AnS3R2cgZMi3pWPVJ66N5N+zGd1KmsfWL9WME16auY1T2qfebEl0se\\\\\\\/9sAGiq0iXmzhnGc3OO+q3\\\\\\\/bLkOPgMAu8Zmn2sgSqYChBGQJSRw9hmfjFlkWChSim+rMmDhH9s5CnZdx\\\\\\\/IUHMl+w5QsxsXI8jr8I3DwB6modLqAa2wIG5xKTq2UqwUREPW+DUqcaGNw9BpUCQWRvtbgaala5gCLsfOZdqJPQaf8jZncLuRoF926X82S9euuZnr0U89u2iAJaqRXYItX4+KlJtcxKZ0ge1R8UWQ6TYsaKSlTStUSqIgFQ50TIYx0WD0aEovjvl1U\\\\\\\/92a+TIhUqIbf0vtjP6Nz2FY3eraIIHUdkR7WtVWRsvF\\\\\\\/nUq41nE9yeKyvDBMwaPiTTcEqhJER9Suj72gdQeUWfK1b3u+8SdcFIR7pNbLHaWErpXYyujyoL81wMG1kmClvIr6QAD5b8lVqLAETSu\\\\\\\/i2ToSbVqnvSVtz7b3tXArsVo1knNCkf1D+8FYf\\\\\\\/sOJ1mBlLf6IyVjoapdZ2Rrp2A3\\\\\\\/mPjk0IeOq6IphB0y9B2\\\\\\\/4aDiemyiPAnnSxnk0hSlbgPUdtIATNrZhi1mTEsSwFFDGrvhRVIEyoXIKUTuPlAotOEOD9Gow5WYO464yMsGsyOYTaIbiJiz4ElX6gndvZjcYpRmeqHHZKmBg1pT8UKfatw+1RMNM9bgzTezFmZU\\\\\\\/odGy3wP7FutHyeJeZlLiZAR4jYn63W\\\u0022,\\\u0022iv\\\u0022:\\\u0022oNYKzDfTffvqC0eLhfsupw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022dee45372823591a4b3c4828470b803270c4ac7a441895c261e66c962008f018b\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022v6Ke8Kx27QxNlyJ18cqyM9YAeGoSX8mVPF5UXV4Awvoz\\\\\\\/njV88PqgiwpLMqNqSnomzM1dohPV8h1rdELVExf\\\\\\\/GURGujtjFtd5rZyCuZqBkcfVi3mr9XfwN3YrkDmIib4NIRgatVcqyO11xn3QmqttzCiyBhV3\\\\\\\/J+VYHEodxBd8xr8oVrfImZ5v63iXERuWQxe1SyNhiMDckPIEaGMwuy6hzcQjxHuBJuysQsonAZ9JWzyQgKNefHTan2adOK0aUbIqUqQV0wFdZRS3ObgsfiMgnncfLFCYsIfhUor+zx8cqCZlGGDwrEdV34OrT1p5lsnM8Ky7cw\\\\\\\/Yk42uEL+7KjPLegslV5JnWvvg8LAuiMY6rH4FohWybzVoQMeTZx\\\\\\\/vqHJq2HnkssWng+CUsiFvx43rb6B4hA4iwuXEN67MMaMBU5FzEWDLfQ8FCFfYIurVPxUtO33mkgVJMcXwMh9pGfUp8msC3IJwVG4iVQZETQvc7wIUaVIeTcgGho3ZKx3DGNQfE8Y\\\\\\\/wvnqWeWL+dfOSGSpPfhbsjShJaojefQ9rnX643ZJpnR2OYiKipF0l2NJkYkd0nQjFBcETR4LNWOTbY5T9Jk+YcU+DRCyfocS5xUEtbxFfpOt1G0Z9NSsU2LSnV5W4sYGFekATKPYVPUoEZpJhFkQl\\\\\\\/5\\\\\\\/UX8Csg0D69zKszaH+rZp6FVSvkimjjAeFlofImDALWgdAuBx7nr9zbOa3F9zDK6G3cwExmsPuI3OfHGr1yuaaU1bz\\\\\\\/zQI8P5L+DSaF87B7GsmlLef+SBsjSb1OQsejSrIgl5RtTx90B4pFeQpRyPIa2Kvqk7c5NKFNyeWKxyIzW6UfRg9bk\\\\\\\/eqek\\\\\\\/\\\\\\\/3FR0uL1451bj9RTHjqL42C\\\\\\\/7PK1opwqfoFzz06kSv5ew6Xu80RUz35goZskaff91mrXfSV1Q7hUjcpNnBUz2K0w3SS9qdCOz7iXz+WZboWandk5DZstLXwvv2ayiYdYK7qM5SvTkG5C1S4HqrO+QHosixT2E2otgjOREcgtPv3qpXJdBfnVJ9mOqsZty\\\\\\\/Jjg2iOkMefqFu4Su9Q+EqfnYV1G6PW6inurY+GT+oaXB6WO4VAlooxHN+wMx6kbhn2ep5fW0DqEdayRIah3WjTcq2MI9Xiv0ixU2YT2X1nTuKiBRvCufOvKDVLqdybdfBXa4xlOLCU5tn38TwXZRwwyzkVnVhh6eTf9vnKAB8ARxlwB\\\\\\\/9DnePHMY4o+96mHW+3mswajVIQuotsCThFPtmD\\\\\\\/CcjTxfvDEV1hk6ODJ5dSgzp7X5NDk88lgU8NLCPsNovXAYVUqdRtktB3TrUx9fN9tMZVuPt9gTxYuNDQZ+ht7NBSD\\\\\\\/S3mAksuiYFJ+JBdtcWd\\\\\\\/wcjHklbS0sR4Ax51rTSVUXnEoV2EgsMQHSKGB8Hm76N7FBqzofeidN5g3Godnx2jcvBRzonwQ5F4Fh8zEvVsHjRcKFJ22TJeSwhbeQq0Cibb1HVT36EvvGH0ApJfJiCetfGxjyVgfoLfyb6dckdDNhZ9+hj6humeRs9iBD1WmfJpmXLrWYf\\\\\\\/UhPML2bYZtyub\\\\\\\/JWnnD7lHyNufizp8ORfhv3ZsYKlOqIV7Rl5wNkeEeGJ5wHPgNoj+mCV1s2TT2DCAE3Ft7dp1vbxKV\\\\\\\/oocjsugbsoM5NjdwgDlb8Ki4teriQ\\\\\\\/3QkVHmH3Sets1qW32uhJbe4icnlXTI\\\\\\\/nnfH5soBgSI\\\\\\\/vlVxEIZQucfzHDoDbSGMzXPzn\\\\\\\/RVqtQD7yWvSI8eD\\\\\\\/NwN3N4BLHyUmGEr3pc1dTuV8KNjeMSULHeop96x5R3ABNeJbObBivtov4irUUNUGRG1zFvJkgTnLwSb0REyVKAR8ichm2cjEVbHov2+YVPpOcGOxVB4ULtwed4IXMk5Oki5syRqeNRplnuCdI44W7qvc+SBmjmFpyvcIJEfLVDJSPRKkAJefITnenypnHI9Y3hDXIgQgUqWZ7gIPdQJj4\\\\\\\/YyJoV\\\\\\\/UxfOtickb\\\\\\\/unRjKDFf9M3m9mYJe\\\\\\\/8ER8kUT3rZoNIMCKYSAWZBUE0gk88WHb9YO5cWr9KeoCJiu6u14wlZ0TEYGcwEv28kfGQmSpGY\\\\\\\/+BGnpqKhKxpFhjuwMjccuXpS6GsUujq3T9kXu6kOsyJLwT697TA8rGQ709ZkklqPp1LUmiz9wKOVecA0gguAbTAaqf0gFjLhtcghDH+YhrEhOa3YD+KSfW4edAFObaET0kt0oaJaWbD1KuXuhPsmNeh2iV5VCzLBOxFv1NbyDs+\\\\\\\/V7c3Q4E45AW16qncAMi91nF35tSg4i1NwUAQ81P8kG9XH+S7AG9CqnqiYI1DWEcgZFfkwbRLObAVjlL1REYRZK19MS0HT1DGJUmIicxZhIrcOh5u5+xEFJeYjDdKF0ziSeYfc1lUCVwt8hXXfPMw6r4fe6OvRqIiiBC7Lc7pVXd4Y2ZCj7jg2hmKbavs9Epw6b5ApdBosYnfDDSqZXmxKLEb7CN6g8PJwHCFKA+GgIByjdjOWGIWNxEBvcKLTHi39XlWRkavu+k2umJqjG37oX\\\\\\\/6\\\\\\\/ZTkQFIWrcoppysWD\\\\\\\/y9E+KRoKdZfLG1LSt\\\\\\\/EKnp6Txuj7Xi1nL55xIoue\\\\\\\/OIy9XJRazGXzL8Zq2nKwMrmlOkuKovSGt2eNgAMfxk1TfJSej4XI+KajdKPAaKD1H+CtGVi1\\\\\\\/uW\\\\\\\/f8f99ExSKAFz1V7TJY16pP7B9OEVApOovUuYg+EmBnQQ1u8dGJRYlDLTuWpZhp1IoCR4aeBFxSCsZLWXZ7ShInlNlM7EtJVbo3grWubeitQlUuW4FCUu91YauWiufipgMFhXxRGMM83vDN8d2kmmKiWL5X+BaS9CCyuJE+dGIFEk4czlEftfKFvjdqZ9F48N2rxnljlGKgnonaGwYpcCh7vo2sLfHAKn8ir4POblLdJ3JINpPFbzj4syu1W1nVXeHhBbNNnqQuThZuwTVQjDtz5Z+h6GC0hfjgbJxbRx\\\\\\\/C4\\\\\\\/eOPAL4fUXikJETIk3LNEDk3i2ray9PAVhlrOC4iLRTE7J19FRVOzfgBrDf8\\\\\\\/FMmlAfoqk1RWZg6EyAPSuGQtdrcGFzgMFUVkNXHmaMFK5bfa8A8qTINzu2Ai+S\\\\\\\/i2968fILDIRHXd0VC\\\\\\\/biJznxrvtfSjnwCqWhpI2YJVpPb5LOnG2WqOznTrF5\\\\\\\/fcjreE3hFb3DymYPSjeWq4cX4bWbh7Bxrbz6FSjeIP+mgb6wzLyaEQ3VsrPs7Es4lTSpKaHG43U3CIB0wXL3EivMtxuhz4ZmOFYyt2HB\\\\\\\/3p1NWul81uDpzQzmrrD\\\\\\\/CL2NQJJDP3QVP4OZIidh5mWxOimlt1s9D+VG5sINIa3UGVfJKCE69L45oH1vjWHAY7m5RrnwqBPw63szHBOqvWepkch8H7nol1Zcny5bBM8u7W5dSl1EJ3OGUJ0yddx1kzbEEQqmlciKW9+fSSLP\\\\\\\/U6DbFe94dLxyZbl5neW1THiEMtVAMVR4qw6t\\\\\\\/CayODJFUGEAlTYjV3RJOe5IpL3s72aZ4yFqH1VkSzgAv8eGmPWtt8PYJ4y8gm+1kkpH73KT+gp7c2IlEVgAs2K0lRFJGELN06E74v+p3+sg6zVadHNgkcmzu7ia\\\\\\\/lz8K+EYg38IWr+Ih4yRjPWsTBQQpNpGuvKelD31nJcYSSG2rPiYrEb00U00c7BjkqU\\\\\\\/pFvqhYMHFWgQvIBZ7uQkRnSCrnLAxpMIto+ZNsKovDqaOWTD+W0EE1pbupjWVF9e8m48iSgk54dkDHnoccFWMWW53QfFf6Y9xd\\\\\\\/iu6SDMpmQjOQdDrjxHJ\\\\\\\/dtvxBQFX+TsA7vblAMCMqC1pD\\\\\\\/7T5FU5tRpv3PAHBxubyNBt45M5frBzqq+xZuvPHXRohaNswRQPCMtEVtlVyLtPGqZtH\\\\\\\/o7uY\\\\\\\/PiUJxyMA2GoU22846R1Xdrn9jkEtlkEmQKTxVQJlEBkyAi5gi9dcOjUkVm4zhGx\\\\\\\/kYOTGhR8z6faCKYD9uqXe\\\\\\\/IlTWN2fAWthkY1USHBnXuNmVoIRK2uq8o+eeE8FDIbjrotUbEAKNvw8G+SevzTd2+jG9FVeWp6WshgaglMogAsTFVJ13\\\\\\\/nkFR+uIThiRGpGLTA78wPTG7sFXk9cJeXHmiblHQwlzl\\\\\\\/4r2TjT+qQVU2luvXoprNdWfoyxLnGn+nFFWYpGJDK1AgRmIMPQAOFXma6mBhA01YCO+i3Wupy7IRowponT5bjWSvRj3YyIL6dTzjTSc\\\\\\\/VY8PqqKR5oMxJGHIpecjnuHQeAlib3PaxHKMKmUjJ7KitadBuR12B8H8wq44P7bypnpIduuIp8tdqRWL9hrgo8TQUBBmm9\\\\\\\/ZWWKraW6z4JrK53WKscd1vm7+FZKuHetch1Wm\\\\\\\/EoPoW1cDw6k9kSeckoahT7ltbq5ubpwV7q\\\\\\\/uEpCvWL4pnCsSMD\\\\\\\/UCmeqkq88ScUK0\\\\\\\/0lcMh+Dk93nzGdp1BvLuukAfZNwuJAeYAAZ\\\\\\\/t8zY5c5DcGHMpPYyWvy7AgHRo0ANE1rcNZNlDF3w5rSVn9R+naFGZpDx7zkp9C4Ckp3axScRtp1xTuPYlxDw5l9GZ5kFtleo+d9\\\\\\\/bd6DrafTmIkpM5ciMXML2YHYupxs3evYhAYn7JPul2Hf7CXGxKEczKFcoK0j5w7p85Lzgt84VmN8lF3AmhZlNhX7igJ6YU4MK2JwsiGNRjqgQXXO4lLpGPo7E\\\\\\\/u3AGt0Y455jYwatSBAXk9qJ8snGxjVZqFxlBbh\\\\\\\/QmPoKr61fFjTwy76mEVvWmnl4okQEY3mW2OFAbM+AESSMdUR4VfSUvAaRGaDNvfpPZJ2i7Yuah+m83B+8ju1jK6efJOIBj+Vt2QWPndtckMCXkrht5hVZfCZQFde2Jnd8GvqfY4rRXm5EajbYPBJ9lnqVcfPc36nRA3CDN3rupceIsgER\\\\\\\/wmf0pSc0USjQTYcrld2sYXrs5xCeoqU9MctxJ+J0YobLf\\\\\\\/UvZRmbGddGrWzlT8CZBlSjj5\\\\\\\/qwsRIMFugfZWhmn1P\\\\\\\/RTkb9dUSXSjvFLl7k1DbsTLMI6Q78NHsCwhzi32rwfpI1C1giN+8F2vq2h45YkzvsGJUjSigcRS4BcAOYX6OCSUlsR+wa6fuhnsejdPKwABDuLFE+8YbGssYNkA7GGmWOC8xTUKxplkLteB4g9E0yMrHzsSSMVsUjkrmjEpxSGLRc1nK91ZR4JQOPmnoEw+nxbiakmcl0CPdMrz3TdvrwCZmtnGyK6u2pY5axy37DNBFF\\\\\\\/EeMnm5lzpE1nVowJHWRkiyg0s8PxW8aQ4soxSsmZt9hptg7eMP16\\\\\\\/2Q0rtYm69uq2MpV0Cdx1arAaLrvGlMOMyZR04QkVE4w1Kc+SyVkA8qFDX\\\\\\\/eev53nh2DuKdSm+PIXVvERXy+VRRjAyF3R8t9kSQdVpH1l7B1rd7cxFRS8+VB83ky1S4O9DqEtGABtvDEOTFRKvtnHc6v++vsyw6iLOAanGhMj8lZ7Ic74kMLwVJ9GYseWs2fkSby4So\\\\\\\/EiPcgRvoxJBeCM9rgfFfH\\\\\\\/bv17992yv5CKsoMm6yAbRfGUn4tTy1hJwip7IZR6E8hUOHKiqgaLLiP9Z5BF6aNt0\\\\\\\/04U1nfDvLYVuUGeFXSlv9kw592u3YlAGxFYz79\\\\\\\/Sudx9tXetI8XgIgTwrN8KQHqL+BZKytOdq949N33Pa+Y5GNJz+zKmP05SEZNVqR2LnUJNO1NRA4kfsxoBdGSRqnE3K363Gr70iqZ+J3zQaKdClK98cjockrVajYmLCU5DJZ8X2Jifd3VqFq4yUAiF+4aAS9RR9dDTNOnfiJJuPgPPZDsPcoyDLnj1Gxpgb7NkY\\\\\\\/LXXKz0D54QXwsYDXd7MDvsFJjzu2OK3xMV7pVU6we\\\\\\\/paON86AlU7g3OMO1NU5+a+5ja5zqQ45UEybFyyR64N230b\\\\\\\/yF1u4b6BnbEhiSQNmgja3+lyoTD44EBX59KP3HgZD1yCulIby07EAl5GTUNjNKuCBppGuVIbTn5DcPD2p1X8v5xp1Zfbz408khe5PtKdsFI3hmMSlQLJi5TepvDGcYdZL8a4wxsMGtl9K44anPrm5Xj5hnrSbn7fHM52\\\\\\\/qmeTLf8m+5PvroUMm7JyXoa\\\\\\\/L3Eq2C4ZnQm5dIW\\\\\\\/Dgve5IcaQJ5OD72IivwRP0mFXH1h\\\\\\\/2LxU6iNTA3UwSVrEVyGXSnymAe3cqRhT3efgHXHEoSKHu2NyQQGGW244OaAexFfGyPhAIjozzOiPS1P7K07dm6Hj1izGdvAE2BybpSwzBd0FoKvYpRrxldgxhzp9Uc\\\\\\\/tMOW2gs3fgSmD5HfO\\\\\\\/8xgxAFmM76qNYkg7ap14Hd7xzJM8qZilshmSyUC1olvVyejCAS\\\\\\\/IApFr\\\\\\\/LMyM0A7H1YENQaTwnBp61eiUjqrnbhp5OP7S2aUYKznyZdQQKKK3EYq9jwD5zc+lAikh1sDT4qI5TEqtCvFUZbZbv3IpinHPd8YYejuDPItgeOU3wtmADtNdBozmk8ioT+3zxKOiMXiscsppViW3UKG\\\\\\\/R+FfMahff\\\\\\\/r3xL2AHJNbrdb1DqyKH5Ae5Ja3oXw9i97clZdD3J\\\\\\\/mJRnVKzg4OI9p0rF7og+aXsbbIrtnduIyEOM57\\\\\\\/yJn659lp4oF2I2oPLI9qoR8SDpX7pvM1x3XMa1Kop\\\\\\\/AFl49AAjjGq+gVxRBaSsEOyyahaaAbGUdl71pqaFDZw1Y37PrsfjBCnbbZqSL4+xxtJwi1A4UCqxVIKx0mNR2lhH0\\\\\\\/qrTqG3SxiAQ8zGMBkguNkhsToobOGE2KpiDdfdl8F6R4JBNAKjNNKUBxwf2qXNyz1Z6vDAquy59TG3zV\\\\\\\/XWDiiIc9M+B0czg5BHWPRndEZWKJh8imlrTTXEmuSd1gMU+c8pH1bqJ1X5ku36QaTtQRkc8t6bl1cwTKMa6VzYgnplP6ZKY93E5d6bL5um7Jlo0EHiknuNDa22aGeiaONodnRogBUe1P4FFBLhCzyjC+ZubTuo4jBCmy59xCqaHaW25x4oYGckMTQXIbZRJFI6zycXWg50c1ykklxq8PDS7UIhUZDtrvDSFAKJtRqwgt9uoUg1NIV0uV6tWJ6yeYdSuCOzfx\\\\\\\/dyLpuqc98xcVHh4HPRd3+P93Kzdn+yk+pPNCkxAa3z7Vs4L\\\\\\\/Gnf01b\\\\\\\/po7x3N7MJ7NMjU+B21MLcf03uqoSF0qNxF02ZEe7zDHB8hT1Q\\\\\\\/Wv\\\\\\\/lKC3OXEsqKlklHt4tioCKak5yJyutv9WYJ8PtEIBkEpNd2KeMPH8jxhpuqTlmVmzkaMyBVeeF1mr6svKxhr1fotryqp0ckf6Mv9x+\\\\\\\/rq\\\\\\\/vBaee093WttX1Wom+pncBOGqluisrgBb8v4JD5XZVscb33HqTNrnfBvTWnrdEPb4Lkxe4uIYtHzqihi1zedyMpn4QqwFlAlQ\\\\\\\/5OuzJOIJ4+ca2ioSEv8S8H5D+nWYYR0G5KeVTkUTpysJ3TT4zNJ4pTpiO8uZuDcuSAFQJrFsVrUGmCZRQH+59YbmRqd5zqMnZZe8+vzLgsP0hUS2so2HqWkDTKDIYYBsJ1uBpUYGpxAujfJoAWaZOXFcdwHqpQID9VEU31EBtq5u1jqMpPd6Iuz8sYIlt9LopbqGPyw2BxBTWeXWBG3l+a2xn3xONf\\\\\\\/KDPbACV+f9xxWW\\\\\\\/A8jo9bF9KDdxCsTY5HfCEFt+klMNgOA1Qq5n2ftgwFRTMojmKy09cSv5dmFfIya7juBE8J\\\\\\\/c3GnK1cyCYiA3NyKnZPYobjYX56Y\\\\\\\/W+IvBrUxspye19gK+0eNXwKQNjsE9y3oSMHTwsndCWYbbC1XsSS7yGY8P7hclKksehoujJvRpr1zAspHAuexE91Kmo6z+wrwc6fK4giIl7GVRsuGQ1Ve4x7dhQSsFkZ9Cv\\\\\\\/ldKaZwDNuXv4AY9HWERa5ZMfy8qz5AR2w9joU6VNTPcgUjOuSJUbsfMAA2ETh8YsI2aT9BUdJ9zjFJYyPd5ha4i+50y8qT\\\\\\\/QUi6HMydOEOaLoLnHl6c8oh5S0NXVP9Qa9Hm6qoRfho+hsKHZoe6KGsfHuXgQPq4VfFIMkV+API5QnwX9VWTR6ty\\\\\\\/fV8x40\\\\\\\/xAnACAfYEFdkhFPufBqP6aTeyqD1DYvTtSRqx9qlxvTA\\\\\\\/Qm4rM+K9OAoDGERCpQqYX+JvP9Nib2KPF\\\\\\\/RC4+YFXqQiujijA8cz7SVXbOuRfpr7qfdaDycWHQkC0N07umDU1anxOkVMPpwrz4\\\\\\\/DnS4VI3reMAWcMidnVo64ibqmn8Y09s0rO4vC6nSTD0oS5oLNBjNeYF8PCTBa1o4r1aSF5VX2AFqPvt5sS\\\\\\\/ypBH9ljz5D6BzMqo\\\\\\\/Mw2rYbqLP7gPhQYH17UDsdcSHwf9PmMlCyXyccoZMiqXaxGG0m9vAmUr9FPCE\\\\\\\/d1RIupWIzF8Wxlux6CTeDbXGwEpM04F3xpcaI\\\\\\\/5MBTaRRWFIdt0\\\\\\\/pjJxUTNljaIaqaYbxY166wFrojl61RmX0ZxQkNx2VDNqW48gVJ6x6G6caBAVoMo6i8i+9buAtttDXkQe9q9s4XhcwmO0NrRaFemmI2GkxLSpcBqQZXU9Sew3UZbR5KaImgdBaQpfBcJeFYawNrfpJ10k82o9TtlEU7xRPM6idG91z7ycyTeiJRizYz+va2RrChegy5SwOgMQM5eLisU3pgWJKkueceQ\\\\\\\/Hl59p07wcogq2PgAwAqeM489DNuN6s0y\\\\\\\/9n8t6zMFxzJgGNukRKlEfT2c02r5UtY4cTtiuYHtcCsqd5D2Jo7\\\\\\\/M4oTd67x15Ra03wlTy9MQGktAL8bmFHl0uun8RxaH2jjWDsB4K3bFWe3Obft5xmDFfEnFlgJV2oBMohjDntQ17eglDID8rWKA9FH2jC0IddDVjfYYCIt7dKEQq1dowWbhBr6xhSEeWKWYQYDFtgaioqe8AVSj3Qk2TrElm4MqPjBYvm9APlGl+7NsT1lBdT4FuPLT8q3o43Q9tftV54d7sYn2VvZSGKnK9\\\\\\\/poPZG0BO2PxVvHgefjPoEl7YmrIogWPdqXoxsQmX75N1JhQKFgmKXgdjcLMVSh1Q2G\\\\\\\/0qRcK1iNUuhbd9kqfrdK7r2C0ImZVmKnMRYm6PMGp6uNlphKoTcZwmtli+4dE7Esbru2atyuA\\\\\\\/+\\\\\\\/Xp6sClWimmgvtT+b5tva\\\\\\\/zsn1ew2GLLzFM82m+0Oyc0Cyds5TAZFp5Xq7Rz\\\\\\\/i6smSGqa8E0cCrhs9CgcFZHrwDyQqSMlW0Cz7M35QjwDxXFLYqn+9vHS7O+jWMP9fi78xc1c7ceu8HaeBCTAfBnT0OCxuEgmN8SHWUz22ZfgyAl8heAiy8k+JhWc7t4Nq6FQThXbS1inUgfg3th8q7sXbWwWcW7LnHw4hIOaYpWCEW4bv0ySkabuQTtOwO2a3UOyEI0awOSlh1c6oSy76Ln3mfA0V9CG06bhGlfNYYGeGLByQANlt5wv9j7GdNJ16VuTxRPbv2caLEn4tGNi6BmxfPKBvqaEaG0iwJT+G3RBdlryLickeYdBz1PJupAG8aBxFBOGW4GO8UGQ1XGZNLGG43go5WKZm\\\\\\\/\\\\\\\/+qFiBE4S3vE0Cq5KOdJErCWBjWXNwYo99hI+dZgNTIqJYu6FPNgU6eBhEsqu6xCtNfowShLb2Pg5a6MZkQThlNib8kP5ghWiMgT5An4mtzgV2d9bBulrNwg7SPEVlm0xOrpm3f7fum5ruBc3mY5huFXNWiX+zyqJ1bVQzX\\\\\\\/lHuXn7L8jBDx3J1zomFWKtmyTJCncXb0bgOwy2JSHTrSyQjKz8IwIKfggikgiXTN9w9dSIvsIknWk6N8jdzOT2xtnLTTWNnG1+qnCSQzyWNTqsVD\\\\\\\/\\\\\\\/QLro1SzfwK2Q7yVhVXO5UKm5399SaP8tFu5wC335QFDe5lqPx\\\\\\\/8pclxhRrrAokZ2JtWvcDvpBHXiYPMZY5sNUwtaWDvUVwT5yIJX+QVHXV2SIIq2ZXjf+jf95PGU3\\\\\\\/+JZVNsxe9RXM+mDrRmZAGtfHqjSq6wwlpo+liLeZTolt7FvBe88UFNTP\\\\\\\/48DCLv+CYv2Pa8VGe1k1yzXkd88YMAA9lyUJpNgKIyxrQEO+Fh9ukGlN1TnmXsrmObyoV09aAKaiKq31HM9d8\\\\\\\/\\\\\\\/A1PYDO8Duyyc3OJZrUg4Y0kjP7V0wK5XHA5CWIJ1iJuBcfiP85nRUf8GtAcKppku37JvFwyZjPdHHSw5PaCUBSr2xZqdg1fWUKq3fIa6cQShNAIsJGCiRzkd47l5d1518dm\\\\\\\/evo4rWT21du1eIPxpx4n+S1pepvj6T7nN8tFpOPjqcXOu258shePViIBxJ5RCkwuyKHeOn1LCJbBUiT3Nmfiq\\\\\\\/M2PJz+p8vvLiZpL8opAlLbuVbCrxaKRxTVojzHyOvtkiD21OHIpJenWcf2yVGZSDPaaIerNK1u7QXrmvGZDbX86zjJpDCF6VmcdVJLTIuCWTchqTW+lF1z1kIOjBgFHxYCa4ZTEaTGyCacQO4ZAGhg7wzys01VDO0Ry7cSaGgo\\\\\\\/MHwlQ4mE1ZpUv3FOvljE4e6Wvip83e1gwLjmJz4Ofd8lbcWxbXz8QvlhU\\\\\\\/K62iOfu630FPx+HFTW9MvjFjiVNQjWI8KHbp3OkDOc9DWweHZVVG4sNO9xvwnS+oq77PtLpQqEQzM6ugPsqCi32H3b4kFFD5atQurDPMnNJqwLwbizdG7qu6oALSoTv2B9q6j6xWTd8aAQHhhtSwyv5sPj9iJBy+lBfaGbzt4DgtQDmlEAnH3PUzuK4dpVJu5u6dnshSQaAoxzKL2HKN2KmAmHJHiFS24O+HzcZg9ArkZRiYKGwDNnkOAgsLxiGwRwJYEr9Ttms0GwP9\\\\\\\/Wz4J94wdXZfoFGvMjMz\\\\\\\/IiTfa6GfuWBDoLT3NOfBkrplbyL8CSce2Mf602H0D8dGHGLzkF4NFAaYOWn6j5MsF1HfZG4sv4nx6CqT5PDTqbyHvbawCr0JyO9FektdgFgo9wrj95YHgJFeX1++L8BoVgjHL\\\\\\\/9uP3ya2OMRovw4zoFGQrbfxPWdzl28D1GiYuF3AIidpN51HRhVOwgC0+RTs36sWA1X9fgpmkYDkPlciQoz\\\\\\\/ia55xajmbkRbrzKDRDn7w3FoBJQAHNHMhWQHRwRjIUNrCDRWJ4XRnjjeCGqdH04Ki2fIbrBIK6qvriRfWuLcfRkYiIyPaJKC0ZVcZ4VwthQ1W\\\\\\\/WPSAv1YEP92yv3mm+TuyDOBsoTAtHTvzlaLKMH84d5wDLIRws9BV0tn8jblOH71XizdZ+IPAJPikZsEHB6PAWiOIgsiDyOJKHPXqOqb3HZCOuGNKcu9jYptjJ4vMbqRJj0eiIHq8soev4Z4mwwmQy4Eo5DeQ\\\\\\\/r9a4khYZHsaxQRk\\\\\\\/IwMUJy1AzQH+81uLrWwWj2sG09qtzExRSuOMI3NGwkLdiIqhavF9y9Za7wgZplJrcudvIChSCAn8TV+aUCSih2OulDwgHkDViY\\\\\\\/KJ+8XHyJdN0l70\\\\\\\/S7B2aE2d7U3quMQkBTBPvjSaqoyDoFKHHsYvbShOxsH9hCo1daT+HdsmJxSkMKi0dqtsnF+EDxUJfdmXepObg68CPd\\\\\\\/Gy5LcKQv5pdqzZQoShDAc3eMJbENcDiMkXi2KA6DgF\\\\\\\/kwuuXHoG9wmwTv21CoPv4TzSIfwXJFrN+3OTnsOv6CsDIu+zvCA+jv00JbiUs0xFWwaA2GUos\\\\\\\/\\\\\\\/pgDcrFxmhYYK4e4+ZgGxtvoLJvSa4nwFXKNRH0WyHHunJbu3y3mdpl3viJzxkO5+qa+vb493qqMk9PgQ+s76cQxLK6BycDWRdN5iWx8OKHt0\\\\\\\/mPUmlKiJHf769eYk3k+BMe5YV+eakLsEgMHlkDJvw+s8AeyPcpXSKeyiyZ9m0jAmh\\\\\\\/B\\\\\\\/Vii9I8ZAsqlB\\\\\\\/zPeSiN3lBnr6vCXDte68QQVFwEbpHW3qc2cYpLxceZww1lf4vC1aE3oLaDcqcot3EXMP9+oqpkBf4v5lR+2p9O7H\\\\\\\/gUeVps1pQeGsAew8CZSElAAZ1J2GlBImqHpLjNvGGa6bq\\\\\\\/RFbmLCrSrFk9f8nrQ4Kse3SOE+CB\\\\\\\/ay\\\\\\\/1QVtxiltEtfUjaRaIN5YPOYaMVQsdf4ZXUmpe7GuXPxuSY3XZYQSPZV2cOPEbeT6hbC2a2plaS1lt0FN4Zx1on3L+KrNNTnetTh0eEUFlzbbyXjxDD0kmvpObOh10gL4tTpt24lvHPsMDMHx7G4es+mycgXz4VNW+fgjnXbE3\\\\\\\/9E3wp3aZ9KJf2TfPrUPw2u0hD4f78hrgwLU\\\\\\\/Q\\\\\\\/btneHALsFmtGgPH5lif4F5DIzLgSJ0VJOIzOboz3VVNJ6yLAePFUHWeB+G9\\\\\\\/5XpS7q3epRGcotXKxkapH\\\\\\\/FirluxA84uSdHn3y9EeIHzL4g37wreElWiu9VfYXErhZq0blFcuIftVgGqxnPMJBd\\\\\\\/iEXpD+TmYTkRtBFTOhMU2urI5f5Ijs2Riwy6t4PbbaiCGymcWYTgQM\\\\\\\/j7W1+sgSEtc++g2AhYmkcPbwPLlVkM4fJpa8SNTJxAYyrX90v5J+vTpZ\\\\\\\/yBj2y1KCpUZ39ZvVBNBq4xpPEcIfSwZdV1YBByOmI34uhSafEFU0x+S+fWZ4ej5yj9wKSUZlaN63AUB2VwHTaNZzHfmkLnAZ6XmwDgBM7ndggtNqhMPIVjPI4lzkUnu3elz5ly70OGeZ3HoNx7x9tRUCKdaIQhmUjgFQztP6w6WXKCaoe2qjIvxlZHiaq9OyJgj9lg2yHaKvh1UyRyvQoZoXr+w7Jf2yzOlqFMbnr9neZ2Twp0Jk2qa9g1m48A7rGfHbM6StNGLUoeru3BBv8krrjJfnTTQy9XAzk2Vyeef\\\\\\\/9tNbyJAMbYN0hnzCayFolLv6f6hX1OXIQDk1\\\\\\\/5QPrXCnnDlrIfR28sr8N7kLbXNZyPY2mTWcktrQnLqzVNZfpYdOzaDqlSgmx88E7ofdJCPENrQNznN3YK2IWQr1PkEF3nPJeYlhW8d2WPuBwG4gG87IsbMTWsKUlRQGsZr3n7YOXqJLrR3lySkKy4g2QKXsgOkDqjHa\\\\\\\/ISb6GUJFl8hOky9v6Fp91zYA7a4FMrK+svKCK9BN+7AnVjCTg8oiTfk0Kbau2AY\\\\\\\/kahWq9bWCOdFmMnVsdVTW3I5VQgKk01AZ03\\\\\\\/UiP4fNJgKHpSVdUv+2t14AsxE7wbptN0IH+MVKcPk1ipdDeKg24otCU4nYzqeEH1QlsXblXuG4CQkomedhnUfZl6MyxAqCwqr\\\\\\\/nJ+sSJktZytTZ3OU0SBhP\\\\\\\/6SM2c1kjJPgtBvn1Gj\\\\\\\/9fsUvwIzxbj\\\\\\\/YrpJ4Op8S3YZQZmabl8DxrdXftPmO8iAsDnpn1EOi7479C5\\\\\\\/mytbiRIruO5\\\\\\\/XqeDv4vi05Oms0jyrzAMfudr\\\\\\\/9ToyjvpD40lAdOjygEqZ\\\\\\\/p\\\\\\\/RYlTuym\\\\\\\/GFLzBhSJHZEd5mwYnyD61HJY9Fkpn9uHsSmn+jaqHE9wZYzQqWlZVTwXj5O\\\\\\\/0bj402mCFHxfXQ7xQR8EmVt4MNRmpbpqLHRNFoR9zEihZ1bSE5umKEa\\\\\\\/q6Z8B1+7TTrZxdX1XoD2g8Tj8Ot0lf+2tXp8g1y+8k4\\\\\\\/k8megDEUzfHBaX4BN6j40WKgP5FemNr+V4HKg+1FMeMNIfTy89KWRY3haZ1smB2E5xFIa5JUpI52xMALWtADrtCjhdbcJG35k2MldLWOjR8HB\\\\\\\/0GBpoS7Ep38pMuhuM1s0HJh0pMWTJ8o8OrUftL6m8QJGv6JXkQK9dyfjdV369B1NQFQCFEH\\\\\\\/sgsOeS1jmwyzsVt7yLjY5TNDMgbe4KIQE0d\\\\\\\/Qm8jaK+xtaMy9\\\\\\\/h0HTf8TVZxRhcZ68RP5YmLKj6f+kc6bT1xJojQYk8GbpKIKo5DKTRtmSgqIGpcmiErA34rox86HOy1sh2rfKkkkGvsBtKWcL10ePqpOsQGKUIADevbqt6SQ92don1aMNpvI8uYejppkHc1w2x80n8fPnAxM4\\\\\\\/t7QLNVoJfBxaam7ehVBm2B9sAvRSmiOSu1Y4IPqxop88wAmY1hSzPK1kduTlsTa4wOLC+NVFjvtRh1mHuyVVvc4+9mLEBl2KrMDN54vL2s02rOvvmjpoQP2LAzI6Msuz\\\\\\\/0A8cI047JYYfgQTGKUp+o30M5hT9\\\\\\\/RnYnlZWCNbzlORSHGuEsfT5Kv5AQRCGOJ64CYtHFc995P9u\\\\\\\/sQuwfxhAoWi3BG1EGKIutlQoXINXBp+cfwAZPNm0bqlJaTvCNlwGmT6\\\\\\\/XHka5PR\\\\\\\/ad02y1ynvqV6VVFqwV\\\\\\\/rUOBMPl5llaB2SLOtlUlF4BCStK+QoRJJn0+fL8StGf3XMd7WZo8joOv58wJSHwRckkkiJnaA\\\\\\\/pWIQgzNSxHwqSWGOZpiRrowiEdILCtlRx7rYrVo+0xz8M2Cg+IIN8a5y1Rq90GsKZV0r3GDLZUQBiubrlxm9IkMp25l\\\\\\\/Drl4ULK\\\\\\\/c8lwlPCyxAsODeVAWvTq6\\\\\\\/3jOp3KuLiOCA4rJzHtBfHufyUdExe6TinCPPwBZEEzIXfTO7A9WltCbSv4rryGUZtuWpDdJOQ9ZHGN7ZRBed97Zkzkb2NjhlXW\\\\\\\/0y44UvNdtPFnCKazSqCGol0pO\\\\\\\/vgRpikOpON2gbYX8glV55mqSFqsu\\\\\\\/05hj6ajyYll3Rlgqr1sAi52AEzZPkGbhZCii0p43ktMjo\\\\\\\/UEXDfMN0t9oTGIU7yifK8KMBMPWZNjtdc9MObvxZoLwJNXzpAdvBPwpebG9vPMpHsOkMjBoRAwEkvq7GXRpqxi\\\\\\\/JC3MYRi8Cw51KRqhyjKcFI1F+1wL1vZp4mL4M5PYD\\\\\\\/7wjpdASkmeNOscl1j2fCSMF2yW8zOhiJn8Bp3uvh+Q\\\\\\\/KMeoXlJeqhK2eLO3dEV5jvgQxfUMZKevmeLyO2PiXINduSNFbJidVBCJbhfGH9pEKJK5PXW9jAcnnJZZd4rYGSyp4SBPgPnMw9VcOk21IlRGMf5YaCY6XLoZvJ\\\\\\\/YBJXJJ0EUy2WM1opEcnAERqe0V9u+r4BEPY72WxiOMJIAshELlSWlde5pl6gqH0SaId7WkEKQfW6f4C\\\\\\\/lP39929XHtQHy4wEEQBHBX1VgnkxwSgG0pIPKmFYaxRljL3x0g4m4v2VmF5RZkFn94cxVMBu6973NTFzmVW3vdiD6nlkfQMbipPcxmf7Exprd+jWD1abQe7p\\\\\\\/+M1VI1fVsxRYhIrldC1aZmbK7lmssNmifbreFYFjR2KqAYsUNbfd2dovnwiQnur6eQPmKBaP43NlDTAYrSE3yt\\\\\\\/2ZFGvuAdYgGU\\\\\\\/Eqw+I51M17PRiMsjJ6P+n0Xf+zGVEgySP8PXQZvupsw+RSA1HLjUtiG4YgQlSpPxlFrvWHsYqbwgeGHqr+XUVe524r\\\\\\\/G2\\\\\\\/WuGVEhx61rHkzjscTo4IiXLnjMRBd4rbQ26qnAIFrRMaI5HEEKUFop6yE9DUzQmJL8G89iJrr8OJaGAnaFT1bRn8AHBPNBLYwOzjUfaOap6CIZJdXE1Apixyl9BMIQabIA1f+oBfMaW\\\\\\\/JBwvG4gMFmJ0wq+ZE4ArKTZyUGcqHv+yP0cthg48xbfnUZudoF4ALK6xXfXacJd3W9cFmcaL56Xs3uhDKIHs9VLXZrSdY3mYteBDqCRtPU2an+Vp84kpmcUIrj0Li+pdDvXt2wzlvBCFrcWxnqkfXy+ISDnt\\\\\\\/JEU2EO2PAYCHSDPQhP9YMIzjJxA9aeacd4nd363KQpDPo7z3BSMmVas1dSZdu9X7GX4zsHaIzoO+XE7nyU8jSFL+emNEtgygdYtU9YZCcz2noVFd\\\\\\\/qOIvbJTSw2UgVRJYpYSUMTSaAOIpesLe66e09c\\\\\\\/SDlQ\\\\\\\/xtb22AxNYwc5JEllkOPlOCGzV4cxwH0ohFkqstyhPaPXBTMuREYQ6QJVpy5xPySJQreJogwrkMKD52BsgycmJS6GmIaPkvpDo8xbOovr6vXj+Xtwe8L1OC8hbnutrGXBnb3w9j5gBG1yrRttfpUH89DBC3UjnlqX4Kk\\\\\\\/Dl+I+sfGAHGkt9oTvmsWpBAC6gHrHFf\\\\\\\/mKSCRXUAuBsLkg6FuuJRid5z8SppJLGJxrWAo4EAP1EUYCccNCSHEsU8yi\\\\\\\/NynOAUm6OgujZCNd\\\\\\\/LJJe1uoy9c64WU3TMbQgieGDy6lA3t4N1ETnpD+CZEXzKTU6iN3EWEn7oTVxYS6zZadGfqsNUoM98Kw+sh0+9GRvr5xAbDKSip9kwIE+ge9wAOKDenWiYQbGBhdczns2ewsh9tUW\\\\\\\/DIh4IdHneuIN9lgBDXULAC9kPrjTNI2eCdJ6mZvMm7p0JiwsMywQhgvcnhb3KC2DGKEDl8r3afVqy2vfKENw2zqdnZjwuvokSnEV1SJK3RvTSRlxidAaIi0SivOrQ4pGaOvfFNd\\\\\\\/tj3KxfWDRIMYsZt0yGjzsrqk8ZpK2Zm1dVvcoQAqJfcgWb7aLywaqoNziZoT+zKpGhfhrjrAgLNk+hNmcpulx9s9L3Caco2IucBIKAPjzC4d3WxxJ4tqHN2E3IE6w4+kW2rbSfm1PSocFBCPHx6Es+LzYa\\\\\\\/2rhPnhaLWP8BbuWQfGx8uBE1k5gJ1+GZsxbeoGwbMH5\\\\\\\/dtKn\\\\\\\/YdmwO5hMLgWWNx1B27Uc\\\\\\\/V7D0Svc6nX1u\\\\\\\/Wl3z7CmWH8jrMWv5RCzyOa5A0yI6P5K0EIyPLZj1TKDyXBJ0qtur3onlsz\\\\\\\/P7rMwUCqP9CCTYrW6yYBI3ItHUoqN2yol03lmL1RP0fpPMTYY+pup1ijm7nySrDkZLQBUWwjWCISy7XdPUNyzjp+C6u6jTSWFqc1Bcu5EXp7QUfvU+ov9km1CYNiAxVgGO8HaRPXGvkM9GKc3Q3xLI5+wQQEeW9dSxSZXJ7zGY+217L9SYjyvFNVWGg6r5bWkzwa+iJphFkLd1uu8YOP4VcUGbdz94a\\\\\\\/DhU\\\\\\\/+TEENw\\\\\\\/iAO5Ur4FECaPiXNGeeOkrsZnmXeomqpGd4lmlgbIXMmlJ\\\\\\\/2lPpxZnYHTHTsXN1Kvuy+p5f+GEHJVdWdOBsTWynPv2\\\\\\\/c4CaZAh5pcAMCjOy9GohEu0rbJctHN5KUyolDjyyDUebijCkbxeyvZjbwY5QSbE8QYmOYts1hSWF5OQjfcH+BfqPkB98qXFUrhQ51NYb\\\\\\\/zNqKol+NAZQUwsliSBTGjcr58oocj3HPbqgVcnoKZAwQ4gZxVH3lxqbWkXjP8HxrnLQJiiBVbuq+Im+k4pDyyT6j0L4VbXzkXSSZBQbKudWskiPCJzxbK53CTa344mp2WRlGqP9czw6S66ENYITncXxGC8o7Z+Q0FXVxCUrHhcODSQr7dSiS\\\\\\\/BtSOdYt5UVdEp\\\\\\\/g+5E9WQZAbmQdr51AKiGZ4FwoxuqDNQvTNwT8mbCc1v6TOGuQxaF1TKHjrKjxKwMqXjQ12uqTlTmy9REwXceLeY\\\\\\\/iPi4\\\\\\\/42SXqXwne44uHIa5HewF1GieYLraa3kUwQ3HNMX02fchQ+JNm60V6iwBc6Ebl45ZsMTyacQKBAcR66ECdeNhm+wYo4dZviuzRuxCVOTun1C463UA0v1\\\\\\\/sOlhEoH+iKXkf+Vr6pJjAnG1H20FvF0lDvbLJ6kmlPByqT+QwgXdBmryjv5o8GygwieXnlermlXpqI4THzZNDQ4dbtktudlDN7pP4Wmj+yT5gMaeEJB8ziunVN1etjjueqZxXySzSayOy8TZVnlkxPeFcDDGYWiHYSJQugrnr5NVPTkDCGKhVqSeMBOa2jl7A+Buzt9lJeOmLgPabkVZDiwx4I\\\\\\\/6JuR+DM\\\\\\\/2Eq799fLIe3MujWaQjMwdBcSbv04CL64qpC60jePnvgbjgRF6PliDVgYPvfviLtLnLiGzxWdlBZIa60xTLdMUGjbDg7cmNyUA32s30t6Q39M6oyCg1ptt7wbvztnkzsVQXeQWJyRin+f6DQFuS38WcsBbMNNNdo6yhZbDhofED6yxqVQIQ+FPxu53XsbwBUOSdovShtzV5Rbcr5CND+I99oo5MSdc7NRwg3mrozCxrZZWe6Qe1UFeUVfnNqp5JwWSdQDhGCl20RNd+GRhYc5k7z\\\\\\\/87ETZGGOa8nADZDOVQA9iVJlRcfbZDETIRtc4C6plE\\\\\\\/ScpgzJJ570B1nxIaNqEe27pjh81LKgrbO2anAiAWqmPlwBMQam26zG7YkSQUSjeqhyyzzmk7\\\\\\\/RX2AmuoLZC3keSh5Lxtb5FDV+2oexe1gBU1v2njwrxwF2uTCd16RH1srCG6L+n89qJV6Dd6rA5ahTA7USgN4yEH7Jw4eqyUIM2tINpPSMMZ6rItY0Hx0BKob7z8fFhF36nCtjgkXR7CUenWAQZ\\\\\\\/iRluUkmJWvZ2vXVXaCEtOxAQWNQUxqK5nZ\\\\\\\/yqhHoTg95cRwKu4IeAEChcmGoHAH8tjfQiLovJaOPiMyl4tnMf1t3iPRVkRVGeRihRVb\\\\\\\/xE66baoajlMYZXzh\\\\\\\/p5Ew2Fq+li4cHtxBz8MH8DC\\\\\\\/0DQqjNNy+2V4eeuPN4JFxi66EI20GHze0ZhLPs40Vq4f78omZO92Iws\\\\\\\/k8RaaurMAjJTVLZ\\\\\\\/VrCQkAxymy78y1qgbNtSbd6QztmUX+c7b2L2jELzm\\\\\\\/XvIAroIDFG9xq87UBzO4+VgAMFI4\\\\\\\/Xg4VU+gxKnUfYgNZgJUKAdIZCHi\\\\\\\/4XXMRsOhhppmJCELFMM3BTawcIADNWu0zaKIojGzaynT2KAIqOCMfjXZo5cAH0O9q\\\\\\\/8PVP9jC2mN1ycAimVJMIQCPiVG+tVWlODu6Kn6xsn2y8zeGV9+Te+5pg6+cHJ+KYoeHQLR9FVhTD9a\\\\\\\/nd617KD40suBJybsq2JBGX56hW7wlzMDnXgS+MeaUUDdo32Z4BaOe++7mmCS6NN6tfyWaCuqVDKSJBkThwz8bOHHr6e1H964cqOg0AZqwEcH4DA9hsuAsYRr\\\\\\\/xoD+qp19cu1smTLQ66\\\\\\\/tsKpHL7JzwNUzkm5lvvv+YJWarKSzzpbnET6JtkQPM5FexaoAgtEc6Pxid1YGHuPz73nLXrCWvq9d+F4fEswtuJVZUJDZq1VsLSsGSJNssT8SKmnJMMvX9Mr6iemz4KdlKx0Nd7Cxrx\\\\\\\/Cd\\\\\\\/eQ8EzBCGAQzggVAudlY4OXnsr5jOUZSjwjfTvhhAoXRxFFiRaIQtGZ6BuehJYid7bHRhz9McZKpIxMrwA0AtSZkQUwrX8W1iIdK0iHWy94z5\\\\\\\/H0FMQWyLWFQ2G1ISuydoqcYGjAsP8MmjM7nnSyELZ9gB9CR+U5BHKeFGTIB9i0KZ8jGQop3kx9pH1iT6cWLwhFPjTvVrTR84E4kkZLTDPCzg8RYjjMaWmvBeM3UbA8D7KjDN2oQUELmKbgiNEsA1YtHXKrn9a1E2PrvzM3J9xKtGe+Ap1MUHoovx1Fk+FfyGEqVMpYjEo46L5hEgISsUPg3vLjB3y3rriI25288fCffCDD1l9rr22gPl8hG+\\\\\\\/XFz61Or3VPpyntXkS4UydLczacT1s+ufaMlLEqOdGETcP0QOA7wJJPfFMAEJo4SOO2tsVj4hYjBqv1yVxJMvSdMWmII2dkitwyJgTmU+FJgo7L9sJSNb0OjSOEMgv5Z\\\\\\\/BIiuIR9dwNkzuNllCBzHx8VPVNEglkFWpye\\\\\\\/e1KoNPMDEZesWJU3ycHq0scKF1At7JOkJrKzOVuI++ykG02Zpjjcc4FX6UWyjRjAkOO5r07D59K1WhcgwKl8RG2Q9eXldHtszzTN2VNqzbEQdCSzcOfegZi4HooSkGrlb61zNlZbRAPSR0tmvkx0jdD2AvNFYSr7CJTV30Qs\\\\\\\/Q3pliHJVVwz+2SQONm6rN1LCdn3FD5zzCvurTsjBElphmjyKAge56yjJ9w0UWwXnQkqaH96N8rQsPmJfNmdrfrVGFaIQ9Rb6zmmb+hdrurxIvoi3N4xONY6Lpyn9l\\\\\\\/AGDL+XoyZo7EaAetTP1dZggjezXcg6QSxpMdbb9ocWPeAgomz2tATPFg54nioDwULFVDWzPjtMYh4gC5IqJbeKbH1WSdr0Gm4BFrPIFEbx78UGeIv5UMWX6BRyVcZpmdfCk4dU2uHpexYbuYpbtTgPKAM9jUkSNM9dv9iIQWvQexGazEedsylweOljK5znZvB9UdwX771TBKW7vINQ7lVJkKmAEu3WG8QiK3YmxVoQ4u8VNEDnMwAScCuiePfJvRfStcBDHe+vItwzJT0lFvRTfPfPwkKA\\\\\\\/T+o9gJRdGHcCDZYSMPbsKfe\\\\\\\/Qj+26vcBedb6gm0leHAAZ7RFtyz7SqhsQfmLj8HdblUVKQiQVG16WFkcN7+aDPM3y2wPURXXXKIHPWGNNZnexEF5CPtyI1bMmOlY4sWe7HgbFYG+M3yl9NBYi0c7ztz1doq7SEC9Qw0ZdWsOzlCyo2usEmKG1Vb7TpcreaW1Kre99WxImswk9sMkwkEfHDPqWUhxMPTpklM6uUtH7foeKiLpT84SFpl5zSrMef8bUBhwQQAAW06CnBqhNghIBkhz15ezGltvZdBQyF2kVa1kpuFsKKLpbac52ST5mscI8dEccf9u6XzkYMk1jdXsQoc6kAvJiSTg\\\\\\\/OvbWzdRJQ5TvyC39p\\\\\\\/YXIb7btbFdU8zgwGwSrQJJMZjHOlQ\\\\\\\/peLVdUSBo4FDltvqo7O33eUAG2FRHh\\\\\\\/TzYTY0uv0mtyjmZzZcSFBqzSa1j1PY\\\\\\\/38HBp\\\\\\\/Ll6xMOQAp49FWwQtZhuh4TjXK5AQL+\\\\\\\/yYWSeMis7o1irjAImm6Bx4eAkrGgpOgxBgcjFeyIYmiyBQGYPD01mJMwpEXZM66WkhgdYs9GJEjZ5TjjQscUrZvQgpTFF8jrhyl9mKJHXZsK1qaGJ5yIqDH35rNQBw5IloMblMq6B04hfsSKg9nAPLLRp9L0\\\\\\\/jttm4vjLH8LpDtn1+GmirFywFvDWbCVQYLoyWJMygPhEVEicjJ2M5KUHYS5knCMbqDQxr1DuVcGGrbu99Yt3nqR48Hi3d+N\\\\\\\/HBBqI4asdih9QCBbPgygNW4GiO1yPObR0dIoJFh5Pofi9GvKsEU8aUrbvP\\\\\\\/MeNqPZBK7lU0LkQTUhX32jWG+X5Sd+0mzLQX\\\\\\\/ZnpW1oPRrTJ7z\\\\\\\/BqpuNyCdCk3+pqWNJ7aYhORThUZll6NlMGVXZILZokGEUNRe35HlEeqcPKONoC+dB8kQ\\\\\\\/54Ql9UTS1wpsB8D9EoOWykv5BObbWRkPeYGrbwg03GUe8kmS2qE9gjJ\\\\\\\/KRGBmDVScI0Yx7V8Cu\\\\\\\/PHPRtVSzH4u8SbDqaHR+TddoXUhlSyUCtsxTE\\\\\\\/89Ilq5esqOyaQ049d7YLTt71jdoT33OmzgDPNV3mlxFVh6nvP8l1NA8fH+XRBO4PVlIoAY81fCKR9ScueHsPK2JIAZur2b4aoBKMTP2eTJ9OVJqP4rb1Ylh5QHxMmcBiIJ0e9k9mU81W8as5K9v6FoPVvaYmuW+KV5lCp5T45cdtzY+wSelEA6Dky7xXjoy9XvEA0GV8DDNTNQZe5axy+YnUOt05hj6KFYEBPVwSWyZDKkVmTFIU0wL4grm+P+tHqM6gYy4weHuYj2CInyE9AjHE59Oea0lb2R7o6+Emssxt9\\\\\\\/zhabtrP394SfHgzXH0+\\\\\\\/OHBGBM\\\\\\\/D4hRdcvC9ZX1ccMDQNgiIx++n98z6Jrh+8UtxcmpfC9gCQ4VJSdK4Lf2Ua4QrPkm7TCrUurQxx05hiKvCYJUfEMYNIF7vOxmPL5T640W6BW+C5qx1pyxuWQ88I6PxhzSmahKI6BIvktSjcdjOaSelU\\\\\\\/65nbZUVyLM96ku\\\\\\\/nIEKtRmz376flfuYRlnBqiJuFcHEqWuDMAuRSuihDD6La5RQIj6scroXSAgYixnWqMOYRYYEUXJ\\\\\\\/rDQ3p5RmZzU9nLVmlbtE\\\\\\\/gus\\\\\\\/EOyK6TV2synjaPlQvhK0h7x7ss\\\\\\\/rPs+Dn1FdIi9ZIq0tRY+9WP+DZ1Fd97YK47PrhnxhHKvk8z9fZpbEhgdHaA+SwlGgIdec8R3ijx5ebsky79p8oOSBWqfl5DHxjfiyGx9ieeBDIXafIg6SIzNAvthJP0Cq\\\\\\\/pwOTinDibszRDtp9flQq2NTL\\\\\\\/5pAp6n\\\\\\\/FFe+zdGGZyjtWOfOD6gS6sehsCPOgWXmm38SknZIPc4wkU4FOQQwedFKzk1eB+aXvYfTOO2epP4ptgAwrYNnm9Vs8g6c1iPDhC55cc36HV9TNURV5QY12OXVwSqOzl4QqSR+sVPue6oZGothZ+bDagaCLFpDvkFBFNYvWuD0MbWhfHuhJi0nUihZ1JNDMgHOLl6TA5yt4AvZfPO9AG0qbR4aux+ikiYIryQwWbUDshAM1oFkzbDENXb54Jpm4RaQy+G56\\\\\\\/QqL9Tt8odMuye8n+15p2ad9kB1syTENqOvNTx\\\\\\\/kEWGpftdkzO4xcO7eoAOrM7RSDAPjwtWTNq0HNpeHzTkRWF4cQ9JMYSYTBqey5AjzJWXPK\\\\\\\/8h5AzCm8B13mKNI075PKp9Dr\\\\\\\/RkAkhmTreCM5VtxXtxywHJVUOSwshmxELI7H\\\\\\\/StmcJxpTJflNH\\\\\\\/SEFNc6Vbu5ta+aQcv2Qtdv48DQMoQD3PgkCzL\\\\\\\/4DrATOHM4BvXjzauJYX2q6Ytp57DSJ5+04oh1\\\\\\\/9RREUdAVRmedcthnqAudzbY\\\\\\\/tj0nyvHA10Sx2ixYfF55e7zZ7O3Zv4R8EE4gTVS++7zzRqPMwRvieuWBfQ5RGlZu7xglWE1HMaetcyLdHzP2\\\\\\\/YgelLbD2DWnEzEKc99\\\\\\\/kvQbXKpvkHX4o4TbApUgTjRXrsDDMQDQMJt99nZtTndzAkjbLQZ\\\\\\\/n0PYrfvIbkVF+WHpfhlXfBso11CNBf526BHP9et2WoRge2juovaBn94BrxXf71lYT27GC7dIirom6FbLkWgAKYzYDBsTcvv2xZUfYizv4M7+bX3IRm4yzERsLHEtIOZLjOiBOFo4uDe2n02cKB4\\\\\\\/eLat9Se4jV6RpeNNc0wFLBmNxwjULSITyTFEgyFP2X9bqp+DBEqLvkmYdCiZ+fa8B1n6Q5dyEAU6qEBsLOWDTrx8ALp7XNDCVBEBWswnt4mp+oTw\\\\\\\/dVhnjnEOUm9loLsHRP4+hGGEcynCVocb5LeolQnyYy0j8q3iJohHjtsK0JWdoJaoWTIuvIYfx0js48XwGczplvU7WzGWSnwzU\\\\\\\/7rc1KqJozI3ArajLRYU5pINWcNShOkapRDwVsDTapue38R+jw1y319pa0aF39fbBmO9QSp2mgd7ZGoltOCoxTJPJmh40L9GU1sgmnJJmsxipyrc4+bR3uN4Dw42jN9\\\\\\\/Vt10dVlO0ZO31MTp2rEqYhfPsrzkLF4VI+G0Y2xIM\\\\\\\/MmBnf0DQhqme5dQKNO1HBXZKVGHtsbJRuTGjEI4JufZ8cu8eJYZiNK0StK8hOa739T46jYDiTmDNK2wav81MmB3miLawiCe057cGzeZWLPEoW0VkZkspkFnap1pj+43exRjc1\\\\\\\/pRg3KXgSnbTzGYiS9jeFYO+emYg\\\\\\\/1BYisowryrAUbpluawjXStvNB6bkpp1cZf7Mpi6S5CyZJBNLyJ8PYDdDy\\\\\\\/hHCrPl49yw4BKNg\\\\\\\/eyVp0RoIq+qwHJ8igHCAjz9EBlmy8gd8uRilScGkATizOGttpTDreaDf01jlLoEay59TUgwPb1LchPxMosvIT7GmHC+lC0yXAzCtDOIRzB2Xgpq1PQLB8bRo6WXP6FEChq+5u6r5tHMj7Qa\\\\\\\/Y7VfwNiuyfq0oTYZDArifKLRZ5NY4pr\\\\\\\/ySdu4ZsME4Rf5z+GfjhktsuEX6TyNvQJU1abtQuK8n1vTcJYkEoWVCNK99gbuy+p+r2d2XeMbEK6qlBZ6Ha0eLCIQwz5eJFtM2Vnz3PubxtdzR8BBmM++PwHGKGO7kWuQrue5ZBGYD2T1neLCX\\\\\\\/uS\\\\\\\/3zJBJ1vkF\\\\\\\/WpuZUi1bNWVLfE2zfNTgsbnv9zixVPhD27MRXG87FJWl5VpVevzhR6nv02TnSUYE8Jkeprj7Bk26hp19A7bxuJmuU2Ci\\\\\\\/XUwx\\\\\\\/JsB8bEpYYTqoGvRUTr6hOpcK33qWySW1U\\\\\\\/oyX352oB9ixd2kSHqEMhZORnbrYL6rrksn8\\\\\\\/u7mJUff9qxesmh5x7Rw56zqIty4GyCOzm8PvUKRAEFD7ju8H0UxHwOb62\\\\\\\/WEo5y+wJGdQEJBmW2SK7K\\\\\\\/RuYsD8vkfnH8FAzUHz1btyuEcBg0tOnujVNcC9gGb4Z4W4c2T\\\\\\\/JDSmPUHp3i20T5+yOVOJ52lHctDMeI1NjJdkqG7xg\\\\\\\/EXRg7FO+oW1ZaU5H1MPTKGvlIPJW3eovqrvSVeV39\\\\\\\/K+\\\\\\\/3JIh3g2JBvCjo0xj\\\\\\\/o\\\\\\\/SjTORiG97fK56kK3o3\\\\\\\/eUFcyAzwYjSyoR2vuvgT2xgcJ+zqrHeQ3s3+4GPSF0jMwOvb3e8Y5NLNiWS2K+dSUAlvYoj\\\\\\\/GuUVEJF4b66WeljOoHt6wBMEWMA8z1Vb6aNcD5eUqgrzNZhgaaAFJTcezPMZOwknUbBpQLYsw\\\\\\\/+82IsG80Pr0bAX1ESgHkWlFIVNRSRb\\\\\\\/L+IS4K6xvQ14LoiR5g3iGQxjm0p3DLZWqnVodfDUpCPwKc0z2+UDA5UnYt6hVZS8qwQ5GF8ql1VkwlxgGAPzbZWZYI\\\\\\\/+vIY3THEfyQ\\\\\\\/i+ZalSu1e6\\\\\\\/9zbG7C0LahbL8ECkqrfscu12WX7hUd9X6GI4xVGRbcU\\\\\\\/UpBY0wx7aZffjKQaqPQiIABC1L6PPudAWQDwoPdTv7q7AKmAx9dlY\\\\\\\/bY6N\\\\\\\/076smPcl4mQHPpvzs9eq\\\\\\\/p8yseX0dbUApHBEo5\\\\\\\/hdE7SXz3fp5UzBYbBKDQ0VVr0Jh4E0grnbifFi9wMB95XhUQQ533OWUFuJx27615aqraL1+INYOZnLyA\\\\\\\/7Z8wlOS9ZMWQKwpjGvDtKeH6MeSc9qTurozZBw0eoda44v9PblAz+6Q2vtpTwLlGZzJjKY8+iy52FarQwISVb+K13cCa7Cv82S8H1LvaXszQnb9PiGGICfyQtlEiJbKiyCvTpb9CGDzYRmaMdTdf1cpSkX5lkNCfqn5gNtPxIb7clDs7cClSJSowNjenix4C9d1gnSE+ANrri\\\\\\\/zex1dKu9HHUIwi89zG8EVxsA6w1so8f9m3vXzfyD9wN0tp2p+LPkQ+Xsv+vywPUcO4sUnKLvPOs91uLGXZO\\\\\\\/o606J4PvU8k4zg3pRUfwnTI14e9\\\\\\\/g54f4JCvi4djWvqnF+f6UpqLBFz2TCTiF++ZaJaERGlOensk7bZEF2WBgboEI8IxgTdqf0F5axGOS51k1gsmIL5WZSD1eBNT0ODFKBBnVLkNCabm87Ky6HlmLubrrQ1vY65wczLgnWYs6ej9Kxd5Fp3IQRGEk+RJXVegbMlt6BKrR4m2Y3B4INYa84kblLyxA4kFNVJaMa9bhkq8tjOtFmNZPKfr9CX0gZxPrl9WJom9QZpbpuWjckplpGno9Mr7ZUZ+i8y3BpJ9D+30yi4qPMVZ3L8C1VxEomMUVHJR+kWol8OSN\\\\\\\/Qj441ZQhme0uSmlosfrOmCykrrqxN\\\\\\\/tCnX4wfsF7CcFTs23B3leISKyW\\\\\\\/+s21IWTlTN3Gn9ltJZnMh+7ERbq9tTUfTb6n+Jk+bHJ4Fr9GL2YI9pAJWKSO3MFih+aZ07+N2dr0B09Ifymf1Xj67QPpQvnTLlPp7leRyUVqLWI+HBpHVUVwJeyPge6BR3RmykKOHVfGO1YrlQ3pr6JHafaWoZUW0a2OKqNk3btZgaubzaSLnJAai\\\\\\\/kuvSg9RJeaqyUnlEHE\\\\\\\/cpBNPr4UizvvEfGPPIeGRbF5cyouZQAcALnWvV8S6+X4TdJulxLmlpK8aUcuDP9Q0byThMJ3itoGRl4N6V88UWq\\\\\\\/2c3XhQ7dq8k7mdqovQ102yaBOdPAMxN0wMr6N2ddXxy7MBzVzYb3Zr489dQKEzg0BcOJMVNhm35ko3IPJ\\\\\\\/+6x51npX2TcPwu\\\\\\\/DprAW85eQOsh6jem+9XNZNzwoMDVE0jcPmfQTvmYrRb0DmPnKa3\\\\\\\/3UkJpUXRKp4SvJ6kmCz8qYFhjT2QHQcI0qqDcOvSBwq9\\\\\\\/amjScGcVFJsgxcDxN6zwGYffTXNLo9fWOCxcitmgXL1ZqWpJBBKG6m+vMv3dtvPiugqhBlS6VgdGcVCoSM2hr4dHi6LtT4uRYhtW6P3VGG0q8Vk9J1YohDqCD9YgyrVIW3SbrWUEgsMHqlL47UusgcvgkdfcUkh0e45FpU7kajOU0ofad0wLk75u3QiHzXquU+h7DZ1T\\\\\\\/azE4iU16wxlQqlYHHu4RXWApvCNQka9dDOmhPEX0qLB1t2NdwdVhFe8YbLb5yciBr72cJbTeU8IUUzveryMXhYGsSu5LVtpjJ2EhZyUuQ5QbyhjGTDJpYs2jNSh4dImqDBKx35Y9dsYpQlHas7r88yY2MCMD\\\\\\\/c6mCsh5LgvueiZ2TGrgvnj0HBcal2vvmJoqYq5FHF2j0yL0dClswQEDkTC0AmYF92QoiaWyIzu5jV8etRGUqfRRHhSEUUMLLdfyM4y97kwCyhszR\\\\\\\/HB4cnuGnXKXu3K2MNHQ9BBNqvqYWi7KA8eC3x69aaCFD3UfdZ0YeauQUXACs3kVQVe\\\\\\\/BfVqLnyJdgbdqaYinfXKSfMmgC5hnniMlxDaQVldhPBDlRX8hGnA2MSNjB++HwwHNerXIacWohQ2eKibqd4q4QMe3oRP6oJbKUSghfhDDSnZvBrqJxls\\\\\\\/BJLn1\\\\\\\/8IqJlDiqC+kxusvN4Rb0MskyDf0iBIVfUY4LKDLGUxXAEP3FoyuZ07\\\\\\\/eJrSz5J7UD0lSpGvu9sO4Cu41E+ZEQ3SBT4Y1iMpsapaZ3742\\\\\\\/h8e6GBwDyb1GMApbCkazv+jLDMxDAwWOWeSM+VSjbnQeU5Tl6zF25KZxsyCNxNaqBSqavv\\\\\\\/y1F9FFWqsl3Nbl0vA1Vn7RiWRm3lvTfo27cOA+XLwnefa2wbnsI2oOpy3h0tYyEiLy1i+3gGSVtDqDaWCs4L0cdItiyPp646BePB3o9zcXytHY6ZOZJSzuU36H4O+CIVx8AEBzo3USzo8AQWUZnTTzs6Jz3ardhDkAenSIPD4ABdSmW8DfG3ZzeaeTPlM4tLVMl6ktRUpd0J7vhiaAJYzJYu6VYQkaZHQI23KrUHMIIaxQLFjzM6fnGlMTUwrdjfv\\\\\\\/MW8V\\\\\\\/8C542K+Cm+x3W0cvB\\\\\\\/cOKqjClIaW6TJHoHATe+RBN7LAa7l7D7GFGGQ5GzHe3Mumo3LzYTUDogFDgWJvKoFoxItxjCN1QAKKWE5AHT2Zv9OIJZjM7hs7xGHM9JSdQtONBHfwqUEAZedRNlejyLFYrSQVIP1UlpF5lorV42Knuohdl7tvol1fhJW\\\\\\\/Uszz8cBmbiomdbdyg2Ef9\\\\\\\/wVp80FQEuU7IJT6RKnxV3okaAah\\\\\\\/0JAyVRzMYENADtocCW0ksqXkxiQ4+08IKLkhcLPN9rjKEk6COD2PRAR0GuyQI8BpdflYqj6IpBkFxP9l8ihvqWe3zoFI\\\\\\\/pGo9Y8\\\\\\\/Pd2a+iqtP8+2uzQpjVIz4PvhxL6dAZp6OZNF5hcWaRckERSFa7FOxT\\\\\\\/WXV52Kw0KXFhH4K5r+TlpU2sE7zRTVjfgafvr81IeBWNhd5gJ5cCNBJumvabD1wBlQHd5Tkl4dKGFQ3++PCdVV5eeDfkrqYRk5JP2V7XK3Tdg7jJOUqWwSNUa0vl5GDAIkGAWKVFYCuskGdxuXd+lY5ZvRvp4aph6DAeo\\\\\\\/fMVKz6gz3cP4sefl2xd5lTsLRCphTkJVKmDnKlVxWp3ejFAXGRrf405NTpLU2jgftSE3icNaIOKIrH6rRrcHw101QvYuxP0iidiE7lvZXjc+hluSEvoBlZu4Ee\\\\\\\/HET5nc0uJiaRwYf2EzVBVP1OpgIiQe8rJGs9scTf7Q9O0CiW+HgsTgUkJb73qp697mkH10\\\\\\\/+0FSLwuxmPYafa6dC3Pr3PlK0lyQjXiQ58oHcFRmB279nAr5+SsseEG3DAHvIArU6LI4Fv67Omp8A4UWSlKkspHczsJH6QqX6mpRVehYzGTo6hdQPSVg92iYnME\\\\\\\/gLTNFj1iQE0uXC6BaoEelkkQly0AnQknr4NiAfE7QVYikIB\\\\\\\/E4pHKrTaZmkilr4CYjTxKLvQxn9H7DkOuocw\\\\\\\/IJuR3MOTuH6SPLRfgL8r2UHnHunzeQDXqAbDc54+0t2wcFtqm+zAy5\\\\\\\/zlMnPdZ4dyXvXyht5+hM1u5JbUIRhBpxKXjz\\\\\\\/+Gnu829j7T\\\\\\\/g4Cdi+w\\\\\\\/3HUWZLV6FEqY9oarBvwHE9ZAlp1OMSw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Zs8rQZKzpuAPfpdaWjfeoA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f752d1aa16f2fe8772d6d11cce12e46e8cfc99e7dfb9d547499f5de6213e5a66\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022LLEz8kDBZ4F6a70mJioWAgUaLJCa4gmUqpun3CMF4MwL76bL9vXZcUYj2VlYAXitdeS3THAxFaSHYv1Gh5UcXiDmE6iYFhkVdiIs6xRQn\\\\\\\/JIQEeuNXUpJ\\\\\\\/8AYZe8fyE5xSL6B5HJGcBKxmbyqrZgQL7gwR7kp7W9AWrt\\\\\\\/w1cCwqYicfhFMQCZJB1WENtxehusYvG903kGYCWRoi6wJcMBeqlYjgq\\\\\\\/FyHMWVO\\\\\\\/072DmY2sfwSK6tZD5AGwUlQil\\\\\\\/5Uwt72KYRhh1Jc54bw4\\\\\\\/1OX8eonksd1tUw74efAZ+QNxhJOdgv8Igx3MwVX09jQWgmsIrM6rq+F9sIM\\\\\\\/iK2vjN1\\\\\\\/QPzTnDzRl\\\\\\\/vII47\\\\\\\/jAZkqncf7e+AS3sJZdlqC9\\\\\\\/3jyunbaYBKHvsBSgvuBzZT9CCiyI9adLccCCQTT23Qu4gFCgzouf27wI\\\\\\\/IcM2aa5n48j4vEem12xbbyT7jfU4ASV+jd+2ZL87UcMR9ws89jB3QJX0wIOCteRw\\\\\\\/dEpP36bj9pd63r96cnUnatmb+OKGO5xhrYCWPQWjFh0kUYDCJ7abXBae4O6X5fqCCQ958HxmGzvlFSs\\\\\\\/SOLLZBK0qEco3t1bifnPmvZW1WKmNIdZ4DS2z0eJchrsOU6kOECVRuhkId\\\\\\\/6L346AGRTUulgcOd1K95slIWs2GZGbfKgz7NqPyzhhVPuBc0IRXFcdLpRjbnwVfoDCBzk9cWM+ENc7dtvCs+Pu5Lq84S0Ag8LEFEYQx0uPnDA\\\\\\\/irYHN21Lq0MXLob9GSgq9AerFDkNgd7lbmnmVXjWgn5sUm+pVHsT+p6pEvEHBJryUlnaqZhf0dH7vt4n4eU7T53Z8ZGwyQIvSm99aWHPtR8t2ohRp6VhSDwIve024VM62+TvJxK5ntm24k8ECojSp0CtZtHct\\\\\\\/3ZutxNewxDH8SnM4KDgLL6mtvt+Ef+5IvEzJ842BPw1JIvmlZVChhxKON2Ene9HzLXHdIBC5g3An0rkpXvgm3Ob83vPR\\\\\\\/d0R3us3tT1BvI\\\\\\\/FM7huFYCmdh+CpYpzene3cHfi14u1xWnWcAGu6RX51202IkLxnn7pE2g+0S+fPcO9wFTYHidCCuAHgfp5nrvLGc28FCPpq1HxYy1D8NIOH1nT8yWMvCq+YOzdPLGJXWa03Op5cC5YnaLDkcwAFd8AkLlEHimHcXV\\\\\\\/Jk4qrjTU6jK1xNKUZXja3Fj27pAh45sUyRy2rk9QSF40bUJPlTVz7roP\\\\\\\/NNuW8AaYxkEK1iB8bvjH2APamyyYcUt+uvwSIrhBpsj+EYl4scD0s6yJgiMbFkUyr08GzFp93ngIxEW9ecUf7IQYTZxUjAUm\\\\\\\/HKbVMonhe661301btyPhGLJpnYIs6juusxL9Ci34CaAMekkFzv1GHy3DmDIxH3G0sJn6NbOR2GyKPssDV+cSQVTfuFGXdhzZPCL5lft+FYzERo3P0wzPkTN1Y08ylYYr\\\\\\\/Y+h4Bk07edUtR6fn4GK31QFnAesTx6fxZdou3o2pqVtnL66xya6DeRZv692JNw2JJA9HC23r7caTULqdR+sFcDHossqhcXD6gU13Jic1ZMXY80zv19gs+opY1nMoGQfBRaUYFPA5mU8jtglgxwpE7wYTGY+ibsCtYRBHXvUigzgvr4Jcpg5cLLjBHJz\\\\\\\/24Rp86MFs6+ZuL9WRjH6tVoauPkPnd21d872hiO08SHXfibw9Z5dFAcczw5zJmf8GR6D\\\\\\\/dbac5WbaqqRTQbZrt2qM8sP8JzXiAHqDFSGfB+hmmZYRVwsJcuOWEuxZo6rO3zzsCEWma9SMk9ILz7qRAtO4Z8avE4UHWjYtOUmvMoMaRrRiNIRuXzF+m2GD4JgeELK2Qd\\\\\\\/wcb5HsJPk4JKyOc\\\\\\\/Pm7Z4y+NFd4MRHaN6X1qtl5Gml4dcG5LnJdl3Is\\\\\\\/cW7qc0AoM7LOZre0GHZPqS7awwyGm4A4ptiBDXQ\\\\\\\/ssJI0huovbs2NZ2152Pf4eNEcif3fzrOLYSS8EovaPGr2Qpv2hCwAiOwLcSg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022RDVREnT1Wof+77yD0WgPtg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00225b7a9ca3c71712a8741cdb403b1d0293da6b990fe37de1088c5967b72e0e3860\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-493760 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/8/2761" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--KgWLchYUWzxpiFWw-vEndFs4ffle8cYFzNWYRNl0qo" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/8/2761" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-DFR4Lr5rZYAN1D_MP8FYmhrFO3fhfOpvmZQhzUZvidA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/8/2749" title="Genome-Wide Profiling of Polyadenylation Events in Maize Using High-Throughput Transcriptomic Sequences" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/8/2775" title="TNF-α Differentially Regulates Cell Cycle Genes in Promyelocytic and Granulocytic HL-60/S4 Cells" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493760" id="node493760" data-pisa="ggg;9/8/2761" data-pisa-master="ggg;g3.119.400186" data-apath="/ggg/9/8/2761.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Signatures of Recent Positive Selection in Enhancers Across 41 Human Tissues</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jiyun M. Moon</span>, <span class="highwire-citation-author" data-delta="1">John A. Capra</span>, <span class="highwire-citation-author" data-delta="2">Patrick Abbot</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-7248-6551" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Antonis Rokas</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2761-2774;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400186 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493760" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jiyun M. Moon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiyun%2BM.%2BMoon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moon%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJiyun%2BM.%2BMoon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John A. Capra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Vanderbilt Genetics Institute, and</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Departments of Biomedical Informatics and Computer Science, Vanderbilt University, Nashville, TN, 37235</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BA.%2BCapra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Capra%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BA.%2BCapra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Patrick Abbot </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BAbbot%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Abbot%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BAbbot%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Antonis Rokas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Vanderbilt Genetics Institute, and</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Departments of Biomedical Informatics and Computer Science, Vanderbilt University, Nashville, TN, 37235</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonis%2BRokas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rokas%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAntonis%2BRokas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7248-6551" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Antonis Rokas</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:antonis.rokas@vanderbilt.edu" class="" data-icon-position="" data-hide-link-title="0">antonis.rokas@vanderbilt.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/8/2761" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:493760" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:493760/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/8/2761.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:493760" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:493760/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/8/2761.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:493760" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:493760/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/8/2761.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:493760" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:493760/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Evolutionary changes in enhancers are widely associated with variation in human traits and diseases. However, studies comprehensively quantifying levels of selection on enhancers at multiple evolutionary periods during recent human evolution and how enhancer evolution varies across human tissues are lacking. To address these questions, we integrated a dataset of 41,561 transcribed enhancers active in 41 different human tissues (FANTOM Consortium) with whole genome sequences of 1,668 individuals from the African, Asian, and European populations (1000 Genomes Project). Our analyses based on four different metrics (Tajima’s <em>D</em>, <em>F</em><sub>ST</sub>, H12, <em>nS</em><sub>L</sub>) showed that ∼5.90% of enhancers showed evidence of recent positive selection and that genes associated with enhancers under very recent positive selection are enriched for diverse immune-related functions. The distributions of these metrics for brain and testis enhancers were often statistically significantly different and in the direction suggestive of less positive selection compared to those of other tissues; the same was true for brain and testis enhancers that are tissue-specific compared to those that are tissue-broad and for testis enhancers associated with tissue-enriched and non-tissue-enriched genes. These differences varied considerably across metrics and tissues and were generally in the form of changes in distributions’ shapes rather than shifts in their values. Collectively, these results suggest that many human enhancers experienced recent positive selection throughout multiple time periods in human evolutionary history, that this selection occurred in a tissue-dependent and immune-related functional context, and that much like the evolution of their protein-coding gene counterparts, the evolution of brain and testis enhancers has been markedly different from that of enhancers in other tissues.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/enhancers" class="hw-term hw-article-keyword hw-article-keyword-enhancers" rel="nofollow">enhancers</a></li><li class="kwd"><a href="/keyword/human-evolution" class="hw-term hw-article-keyword hw-article-keyword-human-evolution" rel="nofollow">human evolution</a></li><li class="kwd"><a href="/keyword/recent-positive-selection" class="hw-term hw-article-keyword hw-article-keyword-recent-positive-selection" rel="nofollow">recent positive selection</a></li><li class="kwd"><a href="/keyword/immunity" class="hw-term hw-article-keyword hw-article-keyword-immunity" rel="nofollow">immunity</a></li><li class="kwd"><a href="/keyword/testis" class="hw-term hw-article-keyword hw-article-keyword-testis" rel="nofollow">testis</a></li><li class="kwd"><a href="/keyword/brain" class="hw-term hw-article-keyword hw-article-keyword-brain" rel="nofollow">brain</a></li></ul><p id="p-3">Enhancers are <em>cis</em>-acting DNA segments that, either independently of or in concert with other regulatory elements, control spatial, temporal and quantitative aspects of gene expression (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Ong and Corces 2011</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Rubinstein and de Souza 2013</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Long <em>et al.</em> 2016</a>). While the precise architecture of enhancers is still debated (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Long <em>et al.</em> 2016</a>), a typical enhancer contains multiple transcription factor binding sites (TFBS) arranged in specific order and distance from one another. Enhancers facilitate initiation of gene transcription by helping to recruit RNA Polymerase II, general transcription factors, and additional components of the transcriptional machinery to the gene’s promoter (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Pennacchio <em>et al.</em> 2013</a>; <a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Rubinstein and de Souza 2013</a>). Multiple enhancers, each with its own repertoire of TFBS, can regulate the activities of a gene in a tissue-specific manner or across distinct developmental stages, enabling enhancers to alter its expression patterns in a particular context without affecting expression of other genes (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Wray 2007</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Sholtis and Noonan 2010</a>). Enhancers in the human genome are more numerous than protein-coding genes (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Pennacchio <em>et al.</em> 2013</a>), facilitating the induction of diverse gene expression programs in different spatial and temporal contexts (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">Long <em>et al.</em> 2016</a>).</p><p id="p-4">Changes in gene regulation have long been thought to play a major role in the adaptive evolution of human traits (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">King and Wilson 1975</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Carroll 2005</a>). One reason for regulatory regions in general, and enhancers in particular, as preferential targets of selection is that, compared to protein-coding regions, they tend to be modularly organized (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Sholtis and Noonan 2010</a>). This modular organization means that mutations in enhancers are less likely to have pleiotropic effects and more likely to contribute to phenotypic evolution (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Carroll 2005</a>; <a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Wray 2007</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Rebeiz and Tsiantis 2017</a>). In the context of human evolution, several studies suggest that evolutionary changes in enhancers might have played a major role in the acquisition of human-specific traits. For example, a considerable proportion of regions that has experienced accelerated evolution in the human lineage are developmental enhancers active in the brain (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Capra <em>et al.</em> 2013</a>), and <em>cis</em>-regulatory regions of genes with neurological and nutritional roles have been shown to exhibit evidence of accelerated evolution in the human lineage (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Rockman <em>et al.</em> 2005</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Haygood <em>et al.</em> 2007</a>). Accelerated evolution is one signature of positive selection, and several studies suggest that recent selection might also have preferentially acted on human <em>cis</em>-regulatory regions (<a id="xref-ref-62-3" class="xref-bibr" href="#ref-62">Wray 2007</a>). For instance, Rockman <em>et al.</em> found that the promoter region of a gene that encodes the precursor of an opioid neuropeptide (<em>PDYN</em>) exhibits a significant degree of population differentiation between human populations, especially between Europeans and East Asians, which is suggestive of local adaptation (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Rockman <em>et al.</em> 2005</a>). Similarly, patterns of variation in the promoter region of the <em>LCT</em> gene that confers lactase persistence in Africans are consistent with the action of selective sweeps (Tishkoff <em>et al.</em> 2007). Finally, SNPs with evidence of recent positive selection are more likely to be associated with expression of nearby genes than random SNPs, and this enrichment is strongest for Yorubans (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Kudaravalli <em>et al.</em> 2008</a>).</p><p id="p-5">More broadly, genome-wide studies have found that, compared to protein-coding regions, enhancers are enriched for variants that are statistically associated with various human diseases (<em>e.g.</em>, inflammatory diseases, metabolic diseases) (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Andersson <em>et al.</em> 2014</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Karnuta and Scacheri 2018</a>). Moreover, recent studies have argued that human population-level differences in transcriptional responses to infection likely resulted from local adaptation, further supporting the idea that selection on enhancers has contributed to recent human evolution (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Nédélec <em>et al.</em> 2016</a>). In addition, multiple events, such as migrations and shifts in cultural practices, have occurred in different time periods during recent human evolution (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Karlsson <em>et al.</em> 2014</a>), likely introducing novel selective agents. Given the role of enhancers in contributing to phenotypic differences, it is likely that enhancers experienced selection events in response to such selective pressures.</p><p id="p-6">Each human tissue serves a particular physiological function so it is reasonable to hypothesize that the genetic elements active in each tissue are influenced by distinct selective pressures that shape their evolutionary rates (<a id="xref-ref-62-4" class="xref-bibr" href="#ref-62">Wray 2007</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gu and Su 2007</a>). For example, both gene expression and protein-coding sequence divergence patterns might be expected to be more conserved in developmentally constrained tissues (<em>e.g.</em>, nervous tissues) than in developmentally relaxed tissues (<em>e.g.</em>, testis or endocrine tissues, such as pancreas). Alternatively, certain functions of tissues (<em>e.g.</em>, reproductive processes for testis) might have experienced increased levels of positive selection (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Khaitovich 2005</a>). Early studies examining patterns of divergence in gene expression and protein-coding sequence evolution among mammals found that both were lowest for nervous tissues (<em>e.g.</em>, brain, cerebellum) and greatest for testis (<a id="xref-ref-62-5" class="xref-bibr" href="#ref-62">Wray 2007</a>; <a id="xref-ref-24-2" class="xref-bibr" href="#ref-24">Khaitovich 2005</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Brawand <em>et al.</em> 2011</a>). More recent examination of a larger number of human tissues has provided further support for this hypothesis. For example, the correlation between expression levels of genes and the strength of purifying selection (as assessed by <em>d</em>N/<em>d</em>S) is strongest for the brain, while this correlation is lowest for liver, placenta and testis (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kryuchkova-Mostacci and Robinson-Rechavi 2015</a>).</p><p id="p-7">Hitherto, studies investigating the variation of evolutionary patterns among tissues have mainly focused on inter-species divergence of gene expression and of protein-coding sequences. However, studies comprehensively quantifying levels of selection on enhancers at multiple evolutionary time periods during recent human evolution and how enhancer evolution varies across human tissues are lacking. Furthermore, it is plausible, if not likely, that distinct selective pressures are also acting on regulatory elements in different tissues, further contributing to the global differences in gene expression patterns among the tissues described above (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Ong and Corces 2011</a>; <a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Rubinstein and de Souza 2013</a>).</p><p id="p-8">To examine the influence of selection on enhancers at multiple evolutionary time periods during recent human evolution and how enhancer evolution varies across human tissues, we used a dataset of 41,561 enhancers active in 41 human tissues and genotype data of 1,668 individuals from three different super-populations from the 1000 Genomes Project to calculate signatures of recent selection that happened at different time points and via different modes (<em>i.e.</em>, hard <em>vs.</em> soft selective sweeps). We found that on average, 5.90% of enhancers exhibit evidence of recent positive selection and that their putative target genes are enriched for immunity-related functions. Furthermore, we found that the distributions of positive selection metrics for enhancers expressed in the testis and brain exhibit were often statistically different and in the direction of lower positive selection than those of numerous other tissues. We found the same trend for the distributions of positive selection metrics for enhancers active in only brain or testis <em>vs.</em> enhancers that were also active in other tissues as well as when we studied enhancers associated with tissue-enriched and non-tissue-enriched genes in brain and testis. Our results suggest that human enhancers, in particular ones associated with immune-related genes, have experienced different modes of recent positive selection at different periods of recent human evolution. Furthermore, patterns of selection on human enhancers differ between tissues, including for brain and testis, a pattern reminiscent of their protein-coding counterparts.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Genetic Variation Data</h3><p id="p-9">To examine signatures of natural selection on human enhancers, we used the whole genome sequence data for 1,668 individuals from Phase 3 of the 1000 Genomes Project (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">The 1000 Genomes Project Consortium <em>et al.</em> 2015</a>) (for information on data sources, see File S1). The 1,668 individuals represent three major human populations (661 Africans, 503 Europeans, and 504 East Asians). We used these three populations because their demographic histories have been most confidently estimated and excluded the other two (Admixed Americans and South Asians) because their demographic histories are more complex.</p></div><div id="sec-3" class="subsection"><h3>FANTOM Enhancer Data</h3><p id="p-10">To study patterns of recent evolution on human enhancers active in different tissues, we used the enhancers detected in 41 human tissues compiled by the Functional Annotation of the Mammalian Genomes (FANTOM) project (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> &amp; File S1) (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Andersson <em>et al.</em> 2014</a>) based bidirectional transcription identified via cap analysis of gene expression (CAGE) in diverse human cell and tissue types (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">The FANTOM Consortium and the RIKEN PMI and CLST (DGT) <em>et al.</em> 2014</a>). We further classified the enhancers into ‘tissue-specific’ enhancers that are detected in only a single tissue, and ‘tissue-broad’ enhancers that are detected in two or more tissues. Of the tissue-broad enhancers, only a small proportion (833 out of 6,757 enhancers; 12.33%) belonged only to one organ system; 39.35% (2,659 enhancers), 18.12% (1,224 enhancers), 10.14% (685 enhancers), 6.16% (416 enhancers), 4.51% (305 enhancers), 2.86% (193 enhancers), 2.77% (187 enhancers), 1.75% (118 enhancers), and 2.03% (137 enhancers) of tissue-broad enhancers belonged to two, three, four, five, six, seven, eight, nine, and ten organ systems, respectively. As genomic regions located on sex chromosomes exhibit patterns of genetic variation that differ from those observed in autosomal genomic regions (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Schaffner 2004</a>), we restricted our analyses to only autosomal enhancers.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2761/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Visual summary of the FANTOM5 enhancer data set used in this study Each circle represents a tissue and all 41 tissues are grouped into non-overlapping organ systems and are color-coded accordingly. The size of the circle is proportional to the log10 of the number of enhancers active within each tissue (Table S1: from highest to lowest: brain: 4,883, blood: 3,657, spleen: 2,429, lung: 2,366, thymus: 1,741, heart: 1,737, testis: 1,621, meninx: 1,447, kidney: 1,294, large-intestine: 1,266, tonsil: 1,193, adipose-tissue: 1,161, spinal-cord: 1,115, eye: 1,019, blood-vessel: 982, small-intestine: 880, uterus: 877, liver: 875, internal-male-genitalia: 846, esophagus: 845, thyroid-gland: 795, skeletal-muscle: 784, throat: 782, placenta: 735, tongue: 708, female-gonad: 693, prostate-gland: 667, gallbladder: 654, urinary-bladder: 602, vagina: 512, salivary-gland: 387, olfactory-region: 338, lymph-node: 292, smooth-muscle: 270, pancreas: 243, parotid-gland: 184, skin: 161, submandibular-gland: 159, penis: 154, umbilical-cord: 115, stomach: 92)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-48003863" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Visual summary of the FANTOM5 enhancer data set used in this study Each circle represents a tissue and all 41 tissues are grouped into non-overlapping organ systems and are color-coded accordingly. The size of the circle is proportional to the log&lt;sub&gt;10&lt;/sub&gt; of the number of enhancers active within each tissue (Table S1: from highest to lowest: brain: 4,883, blood: 3,657, spleen: 2,429, lung: 2,366, thymus: 1,741, heart: 1,737, testis: 1,621, meninx: 1,447, kidney: 1,294, large-intestine: 1,266, tonsil: 1,193, adipose-tissue: 1,161, spinal-cord: 1,115, eye: 1,019, blood-vessel: 982, small-intestine: 880, uterus: 877, liver: 875, internal-male-genitalia: 846, esophagus: 845, thyroid-gland: 795, skeletal-muscle: 784, throat: 782, placenta: 735, tongue: 708, female-gonad: 693, prostate-gland: 667, gallbladder: 654, urinary-bladder: 602, vagina: 512, salivary-gland: 387, olfactory-region: 338, lymph-node: 292, smooth-muscle: 270, pancreas: 243, parotid-gland: 184, skin: 161, submandibular-gland: 159, penis: 154, umbilical-cord: 115, stomach: 92).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2761/F1.medium.gif" width="440" height="296"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/8/2761/F1.medium.gif" width="440" height="296"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2761/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2761/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493851" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-11" class="first-child">Visual summary of the FANTOM5 enhancer data set used in this study Each circle represents a tissue and all 41 tissues are grouped into non-overlapping organ systems and are color-coded accordingly. The size of the circle is proportional to the log<sub>10</sub> of the number of enhancers active within each tissue (Table S1: from highest to lowest: brain: 4,883, blood: 3,657, spleen: 2,429, lung: 2,366, thymus: 1,741, heart: 1,737, testis: 1,621, meninx: 1,447, kidney: 1,294, large-intestine: 1,266, tonsil: 1,193, adipose-tissue: 1,161, spinal-cord: 1,115, eye: 1,019, blood-vessel: 982, small-intestine: 880, uterus: 877, liver: 875, internal-male-genitalia: 846, esophagus: 845, thyroid-gland: 795, skeletal-muscle: 784, throat: 782, placenta: 735, tongue: 708, female-gonad: 693, prostate-gland: 667, gallbladder: 654, urinary-bladder: 602, vagina: 512, salivary-gland: 387, olfactory-region: 338, lymph-node: 292, smooth-muscle: 270, pancreas: 243, parotid-gland: 184, skin: 161, submandibular-gland: 159, penis: 154, umbilical-cord: 115, stomach: 92).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Human Protein Atlas Data</h3><p id="p-12">To compare the patterns of recent evolution between enhancers associated with genes that exhibit higher expression levels in one tissue compared to others and those that are not in our tissues of interest, the brain and testis, we downloaded the list of tissue-enriched genes for the brain and testis tissues from the Human Protein Atlas database (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Uhlen <em>et al.</em> 2015</a>) (version 88; download date: January 17<sup>th</sup>, 2018): the Human Protein Atlas Database defines a gene as tissue-enriched if its mRNA levels in a given tissue are at least fivefold higher compared to its levels of expression in all other tissues. We defined ‘non-tissue-enriched’ genes as those genes that are not labeled as ‘tissue-enriched’ according the to the Protein Atlas database.</p></div><div id="sec-5" class="subsection"><h3>Identifying Signatures of Recent Positive Selection</h3><p id="p-13">To identify signatures of recent positive selection on human enhancers, we calculated four metrics aimed to capture genetic signatures left from the action of selection at different time ranges during recent human evolution (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Sabeti <em>et al.</em> 2006</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Vitti <em>et al.</em> 2013</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Moon <em>et al.</em> 2018</a>) using previously described methodology (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Moon <em>et al.</em> 2018</a>). Briefly, the four metrics that we used can be divided into two categories: those that are designed to detect hard selective sweeps (Tajima’s <em>D</em>, Weir &amp; Cockerham’s <em>F</em><sub>ST</sub> and <em>nS</em><sub>L</sub>) and those designed to detect soft selective sweeps (H12). Furthermore, the three tests for hard selective sweeps are most sensitive to selection events that occurred at different time points in human history: Tajima’s <em>D</em> can detect selection that happened approximately 250,000 - 200,000 years ago; <em>F</em><sub>ST</sub> can identify signatures of local selection that occurred following the out-of-African migration approximately 75,000 - 50,000 years ago; finally, <em>nS</em><sub>L</sub> has power to detect signatures left by selection events that occurred approximately 20,000 - 10,000 years ago (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Sabeti <em>et al.</em> 2006</a>; <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Moon <em>et al.</em> 2018</a>). Following the methods of Moon <em>et al.</em> (<a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Moon <em>et al.</em> 2018</a>) for each enhancer region in every tissue we calculated Tajima’s <em>D</em> using the R package <em>PopGenome</em>, version 2.1.6 (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Pfeifer <em>et al.</em> 2014</a>) and the weighted Weir &amp; Cockerham’s fixation index (<em>F</em><sub>ST</sub>) (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Weir &amp; Cockerham 1984</a>) among all three populations using <em>VCFtools</em>, version 0.1.13 (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Danecek <em>et al.</em> 2011</a>). In addition, we calculated the <em>nS</em><sub>L</sub> statistic (number of segregating sites by length; (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Ferrer-Admetlla <em>et al.</em> 2014</a>)) using <em>Selscan</em>, version 1.2.0 (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Szpiech and Hernandez 2014</a>). For each enhancer, we calculated <em>nS</em><sub>L</sub> for the region extending 50 kilobases (kb) upstream and downstream using the default settings of <em>Selscan</em>, except for the minor allele frequency (MAF) cut-off value, for which we used 0.01. We calculated this metric for each of the three populations, as haplotype-based methods are known to detect selection events that occurred after the out-of-African migration (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Voight <em>et al.</em> 2006</a>; <a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Sabeti <em>et al.</em> 2006</a>). We used the maximum absolute un-standardized <em>nS</em><sub>L</sub> values calculated over the entire window to represent each enhancer region. Finally, for each enhancer region we also calculated the H12 index using a custom script, based on Garud <em>et al.</em>’s original script (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Garud <em>et al.</em> 2015</a>). The values of all enhancers in the 41 human tissues for all metrics can be found in File S2.</p><p id="p-14">To test for statistical differences in the distributions of each of the four metrics between any two tissues, we also carried out pairwise Kolmogorov-Smirnov tests (two-sided). Visualization of the data (Figure S1) revealed that assumptions of normality and equal variance could not be upheld, and therefore, we chose the non-parametric Kolmogorov-Smirnov test which makes no assumptions regarding the shape of the distributions and carried out this test in the R programming environment. We also conducted this test to examine whether there are any significant differences between the distributions of patterns of recent evolution between tissue-specific and tissue-broad enhancers, as well as between enhancers that are associated with tissue-enriched and non-tissue-enriched genes. All statistical tests were followed by <em>post hoc</em> Bonferroni corrections in R. The results of the statistical tests for all metrics can be found Tables S22-27, 28-33.</p></div><div id="sec-6" class="subsection"><h3>Simulations of Neutral Evolution</h3><p id="p-15">The action of non-adaptive processes and demographic changes, such as genetic drift and population expansion, can produce patterns of variation that these tests may mistake for positive selection (<a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">Sabeti <em>et al.</em> 2006</a>). To determine the likelihood of the empirical values being generated by the action of selection, we carried out simulations of neutral evolution based on a model of recent human demographic history and compared the observed values to expected values under neutrality. Simulations of neutrality were conducted using <em>SLiM</em>, version 2.4.1 (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Messer 2013</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Haller and Messer 2017</a>). Following Gravel <em>et al.</em>, we used previously calculated demographic parameters for the three populations included in our study (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Gravel <em>et al.</em> 2011</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Messer 2013</a>). Detailed information on the parameters used for the simulations can be found in the Supplementary File S1.</p><p id="p-16">We simulated the genotypes of 661, 503, and 504 individuals corresponding to the numbers of individuals from each of the three populations analyzed. Since there are 41,561 autosomal enhancers in the FANTOM dataset and anywhere from 92 (stomach) to 4,883 (brain) enhancers in any given tissue (Table S1), carrying out individual simulations for each enhancer was computationally prohibitive. To reduce the computational load of our simulations, we focused on simulating the mutational profile of an average enhancer for a given tissue under a model of neutral evolution. Specifically, to carry out the neutral simulations for enhancers in a given tissue, we used the average length of all the autosomal enhancers found in that tissue (these values can be found in File S3) and a fixed recombination rate of 1x10<sup>−8</sup>. For each tissue, we carried out 10,000 simulations. We next used these simulated sequences to calculate Tajima’s <em>D</em> values, Weir &amp; Cockerham’s <em>F</em><sub>ST</sub>, and H12 as described above. We calculated an empirical <em>p</em>-value for a given enhancer for any metric (including for <em>nS</em><sub>L</sub>, the simulation process of which is described below) as the proportion of simulated sequences that obtained scores equal to or more extreme than the observed value. We used a <em>p</em>-value of 0.05 as cutoff for significance; enhancers with <em>p</em>-values lower than 0.05 were considered to significantly deviate from neutral evolution and to have experienced selection.</p></div><div id="sec-7" class="subsection"><h3>Recombination Rate Interpolation and Simulation of nS<sub>L</sub> Values</h3><p id="p-17">As variation in recombination rates can affect the values of <em>nS</em><sub>L</sub> (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Ferrer-Admetlla <em>et al.</em> 2014</a>), we carried out a separate set of neutral simulations for the calculation of <em>nS</em><sub>L</sub> values by incorporating average recombination rates for each tissue. To do this, we used the genome-wide genetic map curated by the HapMap II consortium (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">International HapMap Consortium <em>et al.</em> 2007</a>), which provides pre-computed recombination rates (cM/Mb) for the variants included in the HapMap II project. More specifically, we carried out linear interpolation to infer the recombination rates of the variants in the 1000 Genomes Project dataset that are not included in the HapMap Phase II dataset. Next, we calculated the average recombination rate (cM/Mb) for the region spanning 50 kb up and downstream of each enhancer in a given tissue, and then used the average of all recombination rates of the enhancer regions in a given tissue as the recombination rate parameter (converted to probability of crossovers per bp) for the neutral simulations. The average recombination rates, as used in our <em>SLiM</em> simulations, for each tissue can be found in File S4. Using these calculated recombination rates, we then created 2,500 enhancers that span 50 kb upstream and downstream of the average length of all autosomal enhancers found in that given tissue and calculated <em>nS</em><sub>L</sub> values on these simulated enhancers using <em>Selscan</em> as described above. As with the actual enhancers, we only included variants with MAF greater than 0.01 in the calculations and used the maximum absolute un-standardized values to represent a given simulated window.</p></div><div id="sec-8" class="subsection"><h3>Functional Enrichment Analyses and Semantic Similarity Calculations</h3><p id="p-18">To gain insight into the functions of genes associated with enhancers that show evidence of selection, we next carried out functional enrichment analyses. To associate enhancers with putative target genes, we used the transcription start site (TSS)-enhancer mapping file generated by Andersson <em>et al.</em> (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">Andersson <em>et al.</em> 2014</a>) (file location information can be found in File S1), which was compiled by measuring the pairwise correlation between enhancer activity and transcription level of putative target genes. We carried out functional enrichment analyses on the list of these putative target genes using the R package <em>TopGO</em>, version 2.32.0 (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Alexa and Rahnenfuhrer 2016</a>). For each enrichment analysis within a given tissue, we used the set of all putative target genes associated with all the autosomal enhancers in that tissue as the background. In short, the GO terms associated with the putative target genes of enhancers with significant evidence of recent positive selection in a tissue were compared against the GO terms of all putative target genes of all autosomal enhancers in that tissue. Detailed description of the files and commands used can be found in File S1. In short, we used the ‘weight’ algorithm that compares the significance scores of the connected nodes to explicitly account for the hierarchy of the gene ontology tree and carried out analyses for the three general ontologies: Biological Process (BP), Molecular Function (MF), and Cellular Component (CC). Subsequent corrections for multiple comparisons were carried out by calculating the FDR-adjusted <em>p</em>-values in the R statistical environment; only those GO terms with an FDR-adjusted <em>p</em>-value less than 0.05 were retained for further analyses. All <em>TopGO</em> enrichment analysis results for all the metrics can be found in Files S5-21.</p><p id="p-19">To quantify the overall similarity of the patterns of gene functional enrichment among different tissues, we calculated pairwise semantic similarity of the GO terms between any two tissues using a graph-based method (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Wang <em>et al.</em> 2007</a>), which takes into consideration the topology of the GO graph structure (<em>i.e.</em>, the location of the GO terms in the graph and the relationship of the terms to their ancestor terms), as implemented in the R package <em>GOSemSim</em>, version 2.4.1 (<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Yu <em>et al.</em> 2010</a>): this analysis was carried out for only those tissues that have 5 or more significant GO terms. To determine if the semantic similarity scores thus calculated significantly deviated from random expectations, we randomly sampled the same number of GO terms for each tissue from the pool of all GO terms associated with a given tissue 1,000 times and calculated the pairwise semantic similarity score; scores equal or greater than the 95<sup>th</sup> percentile value of the distribution of semantic similarity scores were considered significant. The calculated semantic similarity values, along with the 95<sup>th</sup> percentile values obtained from the 1,000 randomly sampled GO terms for each tissue pair, can be found in Tables S7-15.</p></div><div id="sec-9" class="subsection"><h3>Transposable Element (TE) Data and Analyses</h3><p id="p-20">TE-derived sequences often contribute to the origin of new enhancers and the modification of regulatory networks (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lynch <em>et al.</em> 2015</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Simonti <em>et al.</em> 2017</a>). To determine if the enhancers with evidence of recent positive selection harbor higher proportions of TEs than those with nonsignificant deviations from neutral expectations, we used BEDOPS, version 2.4.35 (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Neph <em>et al.</em> 2012</a>) to overlap the enhancer regions with the RepeatMasker-annotated regions. We downloaded the RepeatMasker annotation track for the hg19 assembly from the UCSC Genome Browser (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kent <em>et al.</em> 2002</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Casper <em>et al.</em> 2018</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Raney <em>et al.</em> 2014</a>). The location of the RepeatMasker annotation file and the specific commands used for BEDOPS can be found in Supplementary File S1. To test for statistical differences in the proportions of TEs between enhancers with evidence of recent selection and those that do not exhibit such significant deviation from neutral expectations, we carried out a 2x2 chi-squared test with Yate’s continuity correction using R. The results of the chi-squared tests can be found in Tables S16-21.</p></div><div id="sec-10" class="subsection"><h3>GWAS Catalog Data Annotation</h3><p id="p-21">Previous studies have shown that many variants in human <em>cis</em>-regulatory regions are significantly associated with a broad range of complex traits and diseases (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lee and Young 2013</a>; <a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">Andersson <em>et al.</em> 2014</a>; <a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Karnuta and Scacheri 2018</a>). To investigate whether brain and testis enhancers that exhibit evidence of recent positive selection show enrichment of variants in the Genome-Wide Association Study (GWAS) catalog, we downloaded the NHGRI-EBI GWAS catalog, version1.0.2 (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">MacArthur <em>et al.</em> 2017</a>) (download date: June 2<sup>nd</sup>, 2018) and first compared the SNPs that reside in the enhancer regions with GWAS catalog SNPs. We also included SNPs that are outside the enhancer regions but in complete linkage disequilibrium (LD) with the SNPs within the enhancer regions. We used Plink, version 1.0.9 (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chang <em>et al.</em> 2015</a>) to obtain a list of SNPs that are in complete LD (r<sup>2</sup> = 1.0) with those that lie within the enhancers of interest (‘complete LD SNPs’) and carried out the same analyses as described above. Detailed description of the Plink analyses can be found in Supplementary File S1. For subsequent analyses, we combined the GWAS hits of the SNPs that reside within the enhancers and those that are in complete LD with SNPs inside the enhancers and considered them together. We also carried out the same analyses for brain and testis enhancers that exhibit nonsignificant deviations from neutral expectations. To determine if there is a statistical difference in the enrichment of variants in the GWAS hits between enhancers that exhibit evidence of recent positive selection and those that do not, we carried out 2x2 chi-squared tests with Yate’s continuity correction using R. The full list of all the GWAS hits that overlap with all the variants considered, as well as the chi-squared test results, can be found in Tables S36-37 and S38-39.</p></div><div id="sec-11" class="subsection"><h3>Data Availability</h3><p id="p-22">All the supplementary tables, figures and files from this study are in the G3 figshare repository. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8230757">https://doi.org/10.25387/g3.8230757</a>.</p></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><div id="sec-13" class="subsection"><h3>Enhancers experienced positive selection at different time periods during recent human history</h3><p id="p-23">Calculation of four metrics of selection (Weir &amp; Cockerham’s <em>F</em><sub>ST</sub>, Tajima’s <em>D</em>, H12 and <em>nS</em><sub>L</sub>) for all 41,561 enhancers in 41 tissues (these data can be found in File S2) revealed that an average 5.90% of enhancers have experienced recent positive selection considering all metrics and tissues (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). The proportions of enhancers that exhibit significant deviations from neutral expectations for each metric varied (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>; Table S2). Specifically, greater fractions of enhancers show evidence of recent positive selection according to H12 and Tajima’s <em>D</em> metrics (H12: from 6.57 to 16.67%; Tajima’s <em>D</em>: 6.75–13.92%) than other metrics (<em>F</em><sub>ST</sub>: 1.91–5.56%; <em>nS</em><sub>L</sub> in Africans: 1.12–7.08%; <em>nS</em><sub>L</sub> in Europeans: 2.25–5.39%; <em>nS</em><sub>L</sub> in East Asians: 2.25–5.93%). Furthermore, for 33/41 tissues examined, the H12 metric had the highest proportion of enhancers exhibiting evidence of recent selection. In addition, there were no significant differences in lengths between enhancers with significant and non-significant evidence of recent positive selection. The only exceptions were: kidney (<em>nS</em><sub>L</sub> in East Asians; adjusted <em>p</em>-value = 0.042), lung (<em>nS</em><sub>L</sub> in Europeans; adjusted <em>p</em>-value = 0.023), meninx (<em>nS</em><sub>L</sub> in Africans; adjusted <em>p</em>-value = 0.019), parotid-gland (Tajima’s <em>D</em>; adjusted <em>p</em>-value = 0.002), skin (<em>nS</em><sub>L</sub> in Africans; adjusted <em>p</em>-value = 0.024), throat (<em>nS</em><sub>L</sub> in Africans; adjusted <em>p</em>-value = 0.0003), thyroid-gland (<em>nS</em><sub>L</sub> in Africans; adjusted <em>p</em>-value = 0.0003), tongue (<em>nS</em><sub>L</sub> in Africans; adjusted <em>p</em>-value = 0.002), tonsil (<em>nS</em><sub>L</sub> in East Asians; adjusted <em>p</em>-value = 0.047), urinary-bladder (<em>nS</em><sub>L</sub> in Africans; adjusted <em>p</em>-value = 0.045), and vagina (<em>nS</em><sub>L</sub> in Africans; adjusted p-value = 0.044) (Tables S3-8). In short, these results imply that human enhancers experienced selection at different time points in human history, with substantial evidence for soft selective sweeps in which multiple haplotypes increase in frequency within a population (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Pennings and Hermisson 2006</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Messer and Petrov 2013</a>). However, we caution against overly interpreting differences in the proportions of enhancers across metrics as differences in the action of selection across different time periods since they likely vary in power to detect selection.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2761/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Proportions of enhancers exhibiting significant deviations from neutral expectations for different recent positive selection metrics across 41 tissues Each dot represents the proportion of enhancers (Y-axis) that exhibited significant deviation from the neutral model (i.e., p-value &lt; 0.05) in a given tissue (X-axis). Differently colored dots correspond to the different selection metrics used (Salmon: Tajima&#x2019;s D; Teal: FST; Green: H12; Blue Gray: nSL (Africans); Orange: nSL (Europeans); Purple: nSL (East Asians)). For each metric, the p-value for the observed value for an enhancer (i.e., the likelihood under neutral expectations of obtaining a value as or more extreme as the observed value) was assessed by comparing to 10,000 simulated values calculated on sequences generated from the neutral simulations. Differently colored horizontal lines correspond to the average of all proportions calculated for all 41 tissues according to the different selection metrics used (Salmon: Tajima&#x2019;s D; Teal: FST; Green: H12; Blue Gray: nSL (Africans); Orange: nSL (Europeans); Purple: nSL (East Asians)). Note the 1) differences in the proportion of enhancers exhibiting significant deviations from neutral expectations across tissues in any given metric, and 2) similarly, differences in the proportion of enhancers with significant deviations from the neutral model across all metrics in any given tissue." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-48003863" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Proportions of enhancers exhibiting significant deviations from neutral expectations for different recent positive selection metrics across 41 tissues Each dot represents the proportion of enhancers (Y-axis) that exhibited significant deviation from the neutral model (&lt;em&gt;i.e.&lt;/em&gt;, &lt;em&gt;p&lt;/em&gt;-value &lt; 0.05) in a given tissue (X-axis). Differently colored dots correspond to the different selection metrics used (Salmon: Tajima&#x2019;s &lt;em&gt;D&lt;/em&gt;; Teal: &lt;em&gt;F&lt;/em&gt;&lt;sub&gt;ST&lt;/sub&gt;; Green: H12; Blue Gray: &lt;em&gt;nS&lt;/em&gt;&lt;sub&gt;L&lt;/sub&gt; (Africans); Orange: &lt;em&gt;nS&lt;/em&gt;&lt;sub&gt;L&lt;/sub&gt; (Europeans); Purple: &lt;em&gt;nS&lt;/em&gt;&lt;sub&gt;L&lt;/sub&gt; (East Asians)). For each metric, the &lt;em&gt;p&lt;/em&gt;-value for the observed value for an enhancer (&lt;em&gt;i.e.&lt;/em&gt;, the likelihood under neutral expectations of obtaining a value as or more extreme as the observed value) was assessed by comparing to 10,000 simulated values calculated on sequences generated from the neutral simulations. Differently colored horizontal lines correspond to the average of all proportions calculated for all 41 tissues according to the different selection metrics used (Salmon: Tajima&#x2019;s &lt;em&gt;D&lt;/em&gt;; Teal: &lt;em&gt;F&lt;/em&gt;&lt;sub&gt;ST&lt;/sub&gt;; Green: H12; Blue Gray: &lt;em&gt;nS&lt;/em&gt;&lt;sub&gt;L&lt;/sub&gt; (Africans); Orange: &lt;em&gt;nS&lt;/em&gt;&lt;sub&gt;L&lt;/sub&gt; (Europeans); Purple: &lt;em&gt;nS&lt;/em&gt;&lt;sub&gt;L&lt;/sub&gt; (East Asians)). Note the 1) differences in the proportion of enhancers exhibiting significant deviations from neutral expectations across tissues in any given metric, and 2) similarly, differences in the proportion of enhancers with significant deviations from the neutral model across all metrics in any given tissue.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2761/F2.medium.gif" width="440" height="204"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/8/2761/F2.medium.gif" width="440" height="204"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2761/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2761/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493893" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-24" class="first-child">Proportions of enhancers exhibiting significant deviations from neutral expectations for different recent positive selection metrics across 41 tissues Each dot represents the proportion of enhancers (Y-axis) that exhibited significant deviation from the neutral model (<em>i.e.</em>, <em>p</em>-value &lt; 0.05) in a given tissue (X-axis). Differently colored dots correspond to the different selection metrics used (Salmon: Tajima’s <em>D</em>; Teal: <em>F</em><sub>ST</sub>; Green: H12; Blue Gray: <em>nS</em><sub>L</sub> (Africans); Orange: <em>nS</em><sub>L</sub> (Europeans); Purple: <em>nS</em><sub>L</sub> (East Asians)). For each metric, the <em>p</em>-value for the observed value for an enhancer (<em>i.e.</em>, the likelihood under neutral expectations of obtaining a value as or more extreme as the observed value) was assessed by comparing to 10,000 simulated values calculated on sequences generated from the neutral simulations. Differently colored horizontal lines correspond to the average of all proportions calculated for all 41 tissues according to the different selection metrics used (Salmon: Tajima’s <em>D</em>; Teal: <em>F</em><sub>ST</sub>; Green: H12; Blue Gray: <em>nS</em><sub>L</sub> (Africans); Orange: <em>nS</em><sub>L</sub> (Europeans); Purple: <em>nS</em><sub>L</sub> (East Asians)). Note the 1) differences in the proportion of enhancers exhibiting significant deviations from neutral expectations across tissues in any given metric, and 2) similarly, differences in the proportion of enhancers with significant deviations from the neutral model across all metrics in any given tissue.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Enhancers that experienced positive selection 20,000–10,000 years ago putatively regulate the activities of genes with immunity-related functions</h3><p id="p-25">To study the functions of the enhancers that show evidence of recent positive selection, we carried out enrichment analysis on the putative target genes associated with them (these results can be found in Files S5-21). We found that there was little functional enrichment among the enhancers identified by <em>F</em><sub>ST</sub>, Tajima’s <em>D</em> and H12 (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Analyzing all enhancers irrespective of their tissues produced similar results, with no significant GO terms for the <em>F</em><sub>ST</sub> metric and very few GO terms for the Tajima’s <em>D</em> and H12 metrics (these results can be found in the Tables S9-12). In contrast, many tissues had multiple significant enriched GO terms among the <em>nS</em><sub>L</sub> hits, the majority of which were immunity-related (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Tables 1</a>, <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">2</a>, <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">3</a>, and <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">4</a>). Importantly, analyses on the putative target genes of enhancers with no evidence of recent positive selection according to the <em>nS</em><sub>L</sub> metric showed that none of the tissues possessed any significantly enriched GO terms: the only exception was a single Cellular Component term in the parotid-gland for <em>nS</em><sub>L</sub> metric in Europeans (GO:0044444: cytoplasmic part, adjusted <em>p</em>-value = 0.047). These results suggest that approximately 20,000–10,000 years ago, enhancers that putatively regulate the activities of immunity-related genes in multiple tissues underwent selection, a finding consistent with the hypothesis that human populations faced novel local selective pressures as they moved into new environments (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Balaresque <em>et al.</em> 2007</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Fumagalli <em>et al.</em> 2011</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493994/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493994/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Number of tissues with one or more significantly enriched GO Terms</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493928/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493928/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Most frequently-occurring GO terms among enhancers with evidence of recent positive selection by <em>nS</em><sub>L</sub> across tissues in Africans</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493950/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493950/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Most frequently-occurring GO terms among enhancers with evidence of recent positive selection by <em>nS</em><sub>L</sub> across tissues in Europeans</span><div class="sb-div caption-clear"></div></div></div><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493972/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493972/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Most frequently-occurring GO terms among enhancers with evidence of recent positive selection by <em>nS</em><sub>L</sub> across tissues in East Asians</span><div class="sb-div caption-clear"></div></div></div><p id="p-26">To determine if the pattern of enrichment for immunity-related functions among the enhancers with significant evidence for positive selection according to the <em>nS</em><sub>L</sub> metric was shared across tissues, we quantified the semantic similarity (SS) of the functionally enriched terms among tissues (Figures S2a-i). The pairwise SS values among tissues were consistently very high (BP: Africans: 0.687 to 1.000; Europeans: 0.786 to 1.000; MF: Africans: 0.779 to 1.000; Europeans: 0.715 to 1.000; East Asians: 0.902 to 1.000; CC: Africans: 0.873 to 1.000; Europeans: 0.864 to 1.000; East Asians: 0.853 to 1.000): one exception to this general trend were the SS scores calculated on the Biological Process terms in East Asians (from 0.208 to 1.000) (Figure S2c). To determine if the range of SS values we observed were unusually high, we compared each pairwise SS value to the 95<sup>th</sup> percentile value of the SS values calculated on the 1,000 randomly sampled GO terms for the same pair of tissues (these values, as well as the empirical SS values, can be found in Tables S13-21). We found that most of the empirical SS values were higher than the 95<sup>th</sup> percentile values of the SS values calculated on the randomly sampled GO terms, with the exception of pairwise comparisons involving liver for the Biological Process terms in East Asians (Figure S2c). In other words, most pairs of tissues were associated with semantically similar GO terms, suggesting that patterns of functional enrichment were very similar across tissues. The sole exception to this were 13 comparisons involving the liver, which had three additional biological terms in East Asians (GO:0071294: cellular response to zinc ion, adjusted <em>p</em>-value = 0.005; GO:0046597: negative regulation of viral entry into host cell, adjusted <em>p</em>-value = 0.005; GO:007126: cellular response to cadmium ion, adjusted <em>p</em>-value = 0.014), which were found only in the liver. Overall, our analyses suggest that the enrichment for immunity-related functions found in putative target genes of enhancers with evidence of recent positive selection by the <em>nS</em><sub>L</sub> metric is not confined to specific tissues, but represents a general trend.</p></div><div id="sec-15" class="subsection"><h3>Positively selected enhancers are not enriched for transposable element origins</h3><p id="p-27">To examine whether enhancers that exhibit statistically significant signatures of recent positive selection tend to have arisen from TEs, we compared the proportions of TE overlap between enhancers that exhibit evidence of recent positive selection <em>vs.</em> enhancers with no evidence of recent positive selection across all 41 tissues (Tables S22-27). Of the 41 tissues examined, only the lung (χ<sup>2</sup> = 12.859, adjusted <em>p</em>-value = 0.002) and the female gonad (χ<sup>2</sup> = 8.632, adjusted <em>p</em>-value = 0.020) displayed statistically significant differences in terms of the proportions of TE-overlapping regions between the two groups of enhancers for the H12 metric (Figures S3a-f). Moreover, in these two tissues, enhancers with evidence of recent positive selection showed lower proportions of TE overlap than enhancers with no such evidence. Overall, these results suggest that enhancers that have arisen from TEs were not preferential targets of recent positive selection.</p></div><div id="sec-16" class="subsection"><h3>Enhancers active in testis and brain show different patterns of recent evolution</h3><p id="p-28">To test the hypothesis that signatures of selection during recent human history differed across enhancers active in different tissues, we compared the distributions of the four metrics across all 820 pairs of the 41 tissues (these results can be found in Tables S28-33). The number of tissue pairs exhibiting different distributions for Tajima’s <em>D</em> was 22 / 820, for <em>F</em><sub>ST</sub> was 5 / 820, for H12 was 56 / 820, and for <em>nS</em><sub>L</sub> was 2 / 820 in Africans, and 0 / 820 in Europeans and East Asians (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figures 3a-f</a>). All tissue pairs exhibiting significantly different distributions involved either brain or testis, with the exception of two <em>nS</em><sub>L</sub> comparisons (Africans: Adipose-tissue and Large-intestine: <em>p</em>-value = 0.014; Adipose-tissue and Thymus: <em>p</em>-value = 0.007). Specifically, brain enhancers exhibited significantly different distributions for Tajima’s <em>D</em>, <em>F</em><sub>ST</sub>, and H12 compared to enhancers from 5, 2, and 23 other tissues (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figures 3a-c</a>); similarly, testis enhancers exhibited significantly different distributions for Tajima’s <em>D</em>, <em>F</em><sub>ST</sub> and H12 compared to enhancers from fifteen, three, and thirty other tissues (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figures 3a-c</a>). We note that the high fractions of enhancers under selection in brain and testis are unlikely to be solely due to the high number of enhancers found in these tissues; specifically, there are several other tissues that have high number of enhancers (Table S1) and yet fail to exhibit statistically significant pairwise comparisons with other tissues.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2761/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Pairwise comparisons of the patterns of recent evolution among enhancers from 41 tissues Each graph shows the results of the pairwise Kolmogorov-Smirnov tests carried out between all pairs of 41 tissues (shown on the X- and Y-axes) for all recent positive selection metrics. Each cell represents a pairwise comparison between two specific tissues: red-filled cells represent pairwise comparisons that are statistically significant (i.e., adjusted p-value &lt; 0.05) and empty cells represent non-significant pairwise comparisons." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-48003863" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Pairwise comparisons of the patterns of recent evolution among enhancers from 41 tissues Each graph shows the results of the pairwise Kolmogorov-Smirnov tests carried out between all pairs of 41 tissues (shown on the X- and Y-axes) for all recent positive selection metrics. Each cell represents a pairwise comparison between two specific tissues: red-filled cells represent pairwise comparisons that are statistically significant (&lt;em&gt;i.e.&lt;/em&gt;, adjusted &lt;em&gt;p&lt;/em&gt;-value &lt; 0.05) and empty cells represent non-significant pairwise comparisons.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2761/F3.medium.gif" width="440" height="336"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/8/2761/F3.medium.gif" width="440" height="336"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2761/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2761/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493922" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-29" class="first-child">Pairwise comparisons of the patterns of recent evolution among enhancers from 41 tissues Each graph shows the results of the pairwise Kolmogorov-Smirnov tests carried out between all pairs of 41 tissues (shown on the X- and Y-axes) for all recent positive selection metrics. Each cell represents a pairwise comparison between two specific tissues: red-filled cells represent pairwise comparisons that are statistically significant (<em>i.e.</em>, adjusted <em>p</em>-value &lt; 0.05) and empty cells represent non-significant pairwise comparisons.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">In general, the significant differences in the distributions of these metrics between testis or brain and other tissues were manifested as differences in the magnitude of the peaks of the distributions and/or differences in shapes of the distributions rather than from shifts in the range of the distributions. For all metrics, most pairs of tissues with significant differences had very subtle shifts in the distributions, and in cases where the shifts were more noticeable, brain and testis enhancers’ distributions tended to be shifted to the left (for <em>F</em><sub>ST</sub>; Figure S5) or the right (for Tajima’s <em>D</em>; Figure S4), both in the direction suggestive of lower levels of positive selection for the brain or testis enhancers. The nature of the differences in the peaks of the magnitude varied for each metric: for <em>F</em><sub>ST</sub>, the peaks for both brain and testis were consistently higher compared to those of the other tissues. In contrast, for Tajima’s <em>D</em>, the peaks for brain and testis were consistently lower than the other tissues being compared; the sole exception was the comparison between brain and meninx and urinary-bladder, in which the peaks were almost equal or slightly higher for the brain, respectively. In addition, for Tajima’s <em>D</em> and H12, there were differences in the shapes of their distributions (Figures S4; S6-7). These results imply that enhancers active in the brain and testis experienced different selective pressures during recent human history compared to enhancers active in other tissues.</p><p id="p-31">To further determine if brain and testis stood out compared to other tissues in terms of the patterns of recent evolution of their enhancers, we also ranked the tissues according to the median values of each metric (Figure S8) and the proportion of enhancers that exhibit statistically significant evidence of recent positive selection for each metric (Figure S9). Overall, we did not find any general patterns regarding the recent evolution of brain and testis, in terms of either the median values of the metrics of recent selection (Figure S8) or the proportion of enhancers that have putatively been under selection (Figure S9). Exceptions to the general trend were that testis and brain ranked 1<sup>st</sup> and 2<sup>nd</sup>, respectively, for median values of the H12 metric (Figure S8c) and that testis ranked 2<sup>nd</sup> in the proportion of enhancers with evidence of recent selection according to Tajima’s <em>D</em> and <em>nS</em><sub>L</sub> in Africans (Figures S9a &amp; d).</p></div><div id="sec-17" class="subsection"><h3>Evolution of tissue-specific enhancers differs from that of tissue-broad enhancers in the brain and testis</h3><p id="p-32">We next compared the patterns of recent evolution between tissue-specific and tissue-broad enhancers for each tissue. We found that the number of tissues with statistically significant differences in the distributions of the metrics between enhancers active in a single tissue and those active in multiple tissues was as follows, sorted from highest to lowest: 20/41 for H12, 8/41 for Tajima’s <em>D</em>, 8/41 for <em>F</em><sub>ST</sub> and 1/41 for <em>nS</em><sub>L</sub> in Africans and East Asians (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figures 4a</a> &amp; S10-15). Tissue-specific and tissue-broad enhancers in the brain and testis show statistically significant differences in the patterns of recent evolution for all metrics except <em>nS</em><sub>L</sub> (Tajima’s <em>D</em>: brain: 1.882e-07; testis: 9.523e-11; <em>F</em><sub>ST</sub>: brain: 6.074e-07; testis: 1.639e-07; H12: brain: &lt; 2.2e-16; testis: &lt; 2.2e-16 (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figures 4b-d</a>). Regarding the nature of the differences between the two groups of enhancers in these tissues, we found contrasting patterns for H12 compared to Tajima’s <em>D</em> and <em>F</em><sub>ST</sub>: for Tajima’s <em>D</em> and <em>F</em><sub>ST</sub>, we found that the interquartile values of the tissue-specific enhancers were lower (or higher for Tajima’s <em>D</em>) than those of the tissue-broad enhancers in brain and testis, suggestive of less positive selection in tissue-specific enhancers in these two tissues. In contrast, values for the H12 metric were consistently higher for the tissue-specific enhancers compared to the tissue-broad enhancers in brain and testis. These results suggest that, in general, enhancers with different breadth of tissue activity did not differ in their patterns of recent evolution. Nevertheless, enhancers active only in the brain do show significant differences compared to those active in multiple tissues (including brain) and the same is true for testis, raising the possibility that the recent evolution of brain- and testis-specific enhancers may have been different from that of other tissue-specific enhancers.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2761/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparisons of patterns of recent evolution between tissue-specific and tissue-broad enhancers (a) The grid panel depicts which tissues exhibit significantly different distributions of metrics between tissue-specific and tissue-broad enhancers. Each cell represents a comparison between tissue-specific and tissue-broad enhancers in a given tissue: pink-filled cells represent the comparisons that are statistically significant (i.e., adjusted p-value &lt; 0.05) and empty cells represent non-significant comparisons. (b&#x2013;d) Violin plots depicting the distributions of the metrics for tissue-specific and tissue-broad enhancers for brain and testis for (b) Tajima&#x2019;s D, (c) Weir &amp; Cockerham&#x2019;s FST, and (d) H12. In all cases, the distributions of tissue-specific and tissue-broad enhancers were significantly different (see also panel a)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-48003863" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparisons of patterns of recent evolution between tissue-specific and tissue-broad enhancers (a) The grid panel depicts which tissues exhibit significantly different distributions of metrics between tissue-specific and tissue-broad enhancers. Each cell represents a comparison between tissue-specific and tissue-broad enhancers in a given tissue: pink-filled cells represent the comparisons that are statistically significant (&lt;em&gt;i.e.&lt;/em&gt;, adjusted &lt;em&gt;p&lt;/em&gt;-value &lt; 0.05) and empty cells represent non-significant comparisons. (b&#x2013;d) Violin plots depicting the distributions of the metrics for tissue-specific and tissue-broad enhancers for brain and testis for (b) Tajima&#x2019;s &lt;em&gt;D&lt;/em&gt;, (c) Weir &amp; Cockerham&#x2019;s &lt;em&gt;F&lt;/em&gt;&lt;sub&gt;ST&lt;/sub&gt;, and (d) H12. In all cases, the distributions of tissue-specific and tissue-broad enhancers were significantly different (see also panel a).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2761/F4.medium.gif" width="440" height="262"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/8/2761/F4.medium.gif" width="440" height="262"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2761/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2761/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493955" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-33" class="first-child">Comparisons of patterns of recent evolution between tissue-specific and tissue-broad enhancers (a) The grid panel depicts which tissues exhibit significantly different distributions of metrics between tissue-specific and tissue-broad enhancers. Each cell represents a comparison between tissue-specific and tissue-broad enhancers in a given tissue: pink-filled cells represent the comparisons that are statistically significant (<em>i.e.</em>, adjusted <em>p</em>-value &lt; 0.05) and empty cells represent non-significant comparisons. (b–d) Violin plots depicting the distributions of the metrics for tissue-specific and tissue-broad enhancers for brain and testis for (b) Tajima’s <em>D</em>, (c) Weir &amp; Cockerham’s <em>F</em><sub>ST</sub>, and (d) H12. In all cases, the distributions of tissue-specific and tissue-broad enhancers were significantly different (see also panel a).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>The recent evolution of testis enhancers associated with tissue-enriched genes differs from the evolution of enhancers associated with non-tissue-enriched genes</h3><p id="p-34">To determine whether there are differences in the patterns of recent evolution between enhancers associated with tissue-enriched genes <em>vs.</em> those associated with non-tissue-enriched genes, we compared the distributions of the four metrics between brain and testis enhancers stratified by the breadth of expression of their target genes (these results can be found in Tables S40-41). The only significant differences that we found were for H12 (adjusted <em>p</em>-value =3.667e-06) and Tajima’s <em>D</em> (adjusted <em>p</em>-value = 0.004) in the testis (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). More specifically, we found that the distribution of Tajima’s <em>D</em> values of the enhancers associated with testis-enriched genes were shifted to the left (<em>i.e.</em>, more negative values suggestive of stronger positive selection) compared to the enhancers associated with non-testis-enriched genes (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5a</a>). The difference between the same groups of enhancers for the H12 metric was comparatively subtle, with the main difference being that the peak of the distribution of the enhancers associated with the non-testis-enriched genes was higher than that of the enhancers associated with testis-enriched genes (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5b</a>). These results imply that enhancers associated with genes with enriched expression in the testis have experienced different degrees of selection in the form of both hard and soft selective sweeps compared to enhancers associated with genes that do not show enriched expression in the same tissue.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2761/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparisons of patterns of recent evolution between enhancers associated with tissue-enriched genes and enhancers associated with non-tissue-enriched genes in brain and testis The violin plots depicting the distributions of the metrics for enhancers associated with tissue-enriched and non-tissue-enriched genes for brain and testis for: (a) Tajima&#x2019;s D (b) Weir &amp; Cockerham&#x2019;s FST (c) H12 (d) nSL (Africans) (e) nSL (Europeans) (f) nSL (East Asians). Plots with asterisks above them indicate significant pairwise comparisons." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-48003863" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparisons of patterns of recent evolution between enhancers associated with tissue-enriched genes and enhancers associated with non-tissue-enriched genes in brain and testis The violin plots depicting the distributions of the metrics for enhancers associated with tissue-enriched and non-tissue-enriched genes for brain and testis for: (a) Tajima&#x2019;s &lt;em&gt;D&lt;/em&gt; (b) Weir &amp; Cockerham&#x2019;s &lt;em&gt;F&lt;/em&gt;&lt;sub&gt;ST&lt;/sub&gt; (c) H12 (d) &lt;em&gt;nS&lt;/em&gt;&lt;sub&gt;L&lt;/sub&gt; (Africans) (e) &lt;em&gt;nS&lt;/em&gt;&lt;sub&gt;L&lt;/sub&gt; (Europeans) (f) &lt;em&gt;nS&lt;/em&gt;&lt;sub&gt;L&lt;/sub&gt; (East Asians). Plots with asterisks above them indicate significant pairwise comparisons.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2761/F5.medium.gif" width="440" height="183"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/8/2761/F5.medium.gif" width="440" height="183"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2761/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2761/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493913" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-35" class="first-child">Comparisons of patterns of recent evolution between enhancers associated with tissue-enriched genes and enhancers associated with non-tissue-enriched genes in brain and testis The violin plots depicting the distributions of the metrics for enhancers associated with tissue-enriched and non-tissue-enriched genes for brain and testis for: (a) Tajima’s <em>D</em> (b) Weir &amp; Cockerham’s <em>F</em><sub>ST</sub> (c) H12 (d) <em>nS</em><sub>L</sub> (Africans) (e) <em>nS</em><sub>L</sub> (Europeans) (f) <em>nS</em><sub>L</sub> (East Asians). Plots with asterisks above them indicate significant pairwise comparisons.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-19" class="subsection"><h3>Brain and testis enhancers under recent positive selection are not significantly enriched for variants associated with complex human traits and diseases</h3><p id="p-36">To examine if genetic variants in brain and testis enhancers showing signatures of recent selection are associated with particular complex human traits and diseases, we queried the NHGRI-EBI GWAS catalog (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">MacArthur <em>et al.</em> 2017</a>). We found that for all metrics, most or all enhancers did not harbor SNPs that are associated with any human traits (brain: Tajima’s <em>D</em>: 372/421; <em>F</em><sub>ST</sub>: 131/150; H12: 430/437; <em>nS</em><sub>L</sub>(Africans): 150/167; <em>nS</em><sub>L</sub> (Europeans): 144/161; <em>nS</em><sub>L</sub> (East Asians): 152/170; testis: Tajima’s <em>D</em>: 143/163; <em>F</em><sub>ST</sub>: 47/52; H12: 102/103; <em>nS</em><sub>L</sub> (Africans): 64/72; <em>nS</em><sub>L</sub> (Europeans): 48/54; <em>nS</em><sub>L</sub> (East Asians): 60/65). In the few instances in which SNPs within enhancers showing evidence of selection were associated with human traits, these associations were with traits such as Alzheimer’s disease (brain: Tajima’s <em>D</em>), obesity-related traits (brain: Tajima’s <em>D</em>), and type 2 diabetes (brain: <em>nS</em><sub>L</sub> (Africans)) (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Tables 5</a> &amp; <a id="xref-table-wrap-6-1" class="xref-table" href="#T6">6</a>; the full list of overlapping GWAS hits, including variants in LD with those inside enhancers, can be found in the Tables S42-43). We next compared the proportions of enhancers that overlapped with GWAS hits between enhancers that exhibit evidence of recent positive selection and enhancers that do not and did not find any significant difference between them. More specifically, we saw no statistically significant differences in the proportions of overlap with GWAS hits between enhancers with or without evidence of recent positive selection: the only exception was the H12 metric in the brain (χ<sup>2</sup>= 30.552, adjusted <em>p</em>-value = 1.950e-07), which showed depletion of GWAS hits among enhancers with evidence of recent positive selection (These results can be found in Tables S44-45). These results show that overall, enhancer SNPs under recent positive selection are not preferentially associated with specific human traits or complex human diseases. However, we note that it is also possible that such lack of enrichment for GWAS hits among enhancers with significant evidence of recent selection is due to the fact that such enhancers are less likely to harbor potentially deleterious SNPs that have reached high frequencies within human populations.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493777/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493777/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Complex human traits and diseases associated with variants within the brain enhancers that exhibit evidence of recent selection</span><div class="sb-div caption-clear"></div></div></div><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493776/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493776/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Complex human traits and diseases associated with variants within the testis enhancers that exhibit evidence of recent selection</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-20"><h2 class="">Discussion</h2><p id="p-37">In this study, we examined signatures of recent positive selection in 41,561 enhancers active in 41 human tissues. We found that approximately 5.90% of enhancers across tissues and metrics exhibit significant evidence of recent positive selection, b) the putative target genes of enhancers that experienced selection in the last 20,000 – 10,000 years ago are enriched for immunity-related functions, and c) enhancers active in the brain and testis exhibited significantly different patterns of recent evolution compared to enhancers in other tissues.</p><p id="p-38">Our first key result is that the proportions of enhancers in each tissue that exhibited signatures of selection vary across the different metrics (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>). Since the origin of modern humans approximately 250,000 - 200,000 years ago, numerous episodes, including the out-of-Africa migration approximately 75,000 - 50,000 years ago, the Eurasian split that occurred approximately 45,000 - 36,000 years ago, and the Agricultural Revolution approximately 20,000 - 10,000 years ago, exposed the human populations to novel selective pressures (<em>e.g.</em>, pathogens, dietary changes) (<a id="xref-ref-48-5" class="xref-bibr" href="#ref-48">Sabeti <em>et al.</em> 2006</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Karlsson <em>et al.</em> 2014</a>). Adaptations in response to such changes would have likely altered the allele frequencies in both the genic and regulatory regions. As the different metrics of selection that we employed in our study are sensitive to distinct types of genomic signatures (<a id="xref-ref-48-6" class="xref-bibr" href="#ref-48">Sabeti <em>et al.</em> 2006</a>; <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Voight <em>et al.</em> 2006</a>), one possible interpretation is that within any given tissue, the differences in percentages of enhancers under selection between metrics reflect temporal variation in the action of selection during human history.</p><p id="p-39">There are two caveats to interpreting the results of <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a> to represent temporal variation in the action of selection. The first caveat is that these metrics likely have different power to detect selection. For instance, Tajima’s <em>D</em>, a metric that detects excess of rare alleles in a region of interest, is most sensitive to selective sweeps that have resulted in complete fixation of the target locus but has limited power otherwise (<a id="xref-ref-48-7" class="xref-bibr" href="#ref-48">Sabeti <em>et al.</em> 2006</a>; <a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Ferrer-Admetlla <em>et al.</em> 2014</a>). In contrast, <em>nS</em><sub>L</sub>, a haplotype-based metric, best detects ongoing or incomplete hard selective sweeps resulting in intermediate allele frequencies (<em>i.e.</em>, allele frequencies = 60–80%) and rapidly loses power as allele frequencies increase to 100% (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Ferrer-Admetlla <em>et al.</em> 2014</a>). Thus, the observed differences in the proportions of enhancers with evidence of recent positive selection across the metrics also reflect the disparity in power of the metrics to successfully detect incidences of selection that happened at a particular time period and cannot be directly compared to each other. The second caveat is that our inferences rely on an explicit neutral demographic model whose key parameter values are fixed rather than drawn from distributions (due to the substantially increased computational burden associated with the latter choice). Given the very large number of enhancers examined, it is possible that not accounting for neutral variation may have increased our numbers of false positives in a metric-dependent fashion (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Harris <em>et al.</em> 2018</a>). Nevertheless, the estimates of demographic parameters we used are consistent with those derived from other data sources (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Gravel <em>et al.</em> 2011</a>), and the inference of selection using large genomic segments by some of our metrics (<em>e.g.</em>, <em>nS</em><sub>L</sub>) makes them less susceptible to this potential type of error. Distinguishing whether our results are best explained by temporal variation in selection or by our experimental design choices and differences in the power of our metrics could be achieved via more realistic simulations in which selective events are introduced at specific time points and result in a fixed proportion of regions being selected. This is an important future research direction that has potential to shed light into the tempo of selection in the course of recent human evolution.</p><p id="p-40">Our second key result is that the putative target genes of enhancers exhibiting significant evidence of recent positive selection according to the <em>nS</em><sub>L</sub> metric are enriched for immunity-related functions (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Tables 2</a>-<a id="xref-table-wrap-4-2" class="xref-table" href="#T4">4</a>). Haplotype-based metrics such as <em>nS</em><sub>L</sub> are known to be sensitive to signatures resulting from selection that occurred approximately 20,000 - 10,000 years ago, which corresponds to the incidence of the Agricultural Revolution (<a id="xref-ref-59-3" class="xref-bibr" href="#ref-59">Voight <em>et al.</em> 2006</a>). The advent of farming practices, communal living in settlements, and migrations of farmers across the globe resulted in an increase of numbers and densities of humans in any given location, likely facilitating the spread of pathogens (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Varki and Gagneux 2009</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Page <em>et al.</em> 2016</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Nielsen <em>et al.</em> 2017</a>). In addition, recent studies suggest that selection on <em>cis</em>-regulatory regions, such as enhancers, might have been important in driving adaptation of modern human populations to distinct environments, due to their modular organization: change of expression pattern in one temporal or spatial context can often occur without affecting others, which could contribute to phenotypic changes without incurring negative pleiotropic effects (<a id="xref-ref-6-3" class="xref-bibr" href="#ref-6">Carroll 2005</a>; <a id="xref-ref-62-6" class="xref-bibr" href="#ref-62">Wray 2007</a>). Therefore, it is possible that around 20,000 - 10,000 years ago, enhancers regulating the activities of immunity-related genes underwent selection to allow refined fine-tuning of host defense processes in response to the stronger pressure from pathogens resulting from increased human population sizes.</p><p id="p-41">The third and perhaps most striking result of our study is that brain and testis enhancers exhibited different patterns of recent evolution, typically in the direction of lower levels of positive selection, compared to enhancers of other tissues (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a> &amp; S4-7). For both brain and testis, the high number of enhancers included in these tissues could partly explain our results. Brain, with 4,883 enhancers, has the most enhancers of all the tissues examined, while testis, with 1,621 enhancers, is ranked 7<sup>th</sup>. The effect of the large number of enhancers included in these tissues is reflected in the differences in the magnitudes of peaks of the distributions of the metrics between brain and testis and other tissues (Figure S4-7). In addition, significant differences in the lengths of enhancers included in the brain and testis compared with other tissues could further contribute to our results (Table S46). Overall, enhancers in testis tend to be shorter than those in other tissues (<em>i.e.</em>, distribution of lengths of testis enhancers exhibits subtle shifts to the left compared to other tissues; Figure S16). The same is true for brain enhancers, except when compared with testis enhancers (<em>i.e.</em>, the distribution of lengths of enhancers active in the brain are shifted to the right compared to the distribution of lengths of testis enhancers); however, for comparisons with some tissues (<em>e.g.</em>, heart, internal male genitalia, small-intestine, blood, and spinal-cord), the shift was more subtle, with the main difference instead occurring in the peaks of the magnitudes or the shapes of the distributions (Figure S17). Nevertheless, we also observed other types of differences in comparisons of the distributions of these metrics between brain and testis and other tissues: while subtle, some metrics (Tajima’s <em>D</em>, <em>F</em><sub>ST</sub>; Figure S4-5) showed shifts of the distributions of brain and testis enhancers toward weaker signatures of recent positive selection compared to other tissues, suggesting that the significant pairwise differences we see are not solely caused by disparities in the numbers of enhancers or lengths of enhancers within tissues. In addition, there are several other tissues (<em>e.g.</em>, lung, spleen, blood) whose numbers of enhancers are nearly on par with those of the brain and more than the testis or whose lengths significantly differed compared to numerous other tissues. However, these tissues did not show significant differences in the distribution of these metrics compared to other tissues. Furthermore, most tissues did not differ significantly in terms of the recombination rates of the enhancers: the only exception was between eye and testis (adjusted <em>p</em>-value = 0.05; Table S47). Collectively, these results further suggest that the observed differences in selection are likely biological.</p><p id="p-42">Why do signatures of recent positive selection tend to be lower for brain and testis enhancers? Answering these questions is challenging without additional functional experiments that shed light on the phenotypic effects of the selected variants. In the case of the brain, one possible explanation could be that brain enhancers are under stronger functional constraint (and hence less likely to exhibit positive selection) than enhancers in other tissues. Consistent with this hypothesis, existing evidence suggests that the brain size of modern humans has not changed substantially in the last 250,000 - 300,000 years (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Neubauer <em>et al.</em> 2018</a>). Similarly, expression patterns of genes expressed in neural tissues (<em>e.g.</em>, brain, cerebellum) show low levels of divergence across both species and within humans, suggesting that the overall structure of the neural network is highly conserved (<a id="xref-ref-24-3" class="xref-bibr" href="#ref-24">Khaitovich 2005</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Brawand <em>et al.</em> 2011</a>). Finally, the brain transcriptome exhibits low divergence due to the relatively strong developmental constraints acting on neural tissues (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Gu and Su 2007</a>). For instance, a study found that the correlation between expression levels and signals of purifying selection is strongest for brain, suggesting that genes expressed in the brain evolve slowly likely due to stronger functional constraints (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Kryuchkova-Mostacci and Robinson-Rechavi 2015</a>).</p><p id="p-43">In the case of testis, our result that enhancers active in the testis often show weaker signatures of positive selection compared to enhancers in other tissues is surprising and harder to reconcile with what is known about how this tissue’s regulatory landscape evolved. For example, previous studies have shown that testis exhibits the highest degree of expression pattern divergence between species, likely due to either strong action of positive selection on reproductive processes (<a id="xref-ref-24-4" class="xref-bibr" href="#ref-24">Khaitovich 2005</a>; <a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Brawand <em>et al.</em> 2011</a>), relaxation of purifying selection in the testis compared to other tissues (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gershoni and Pietrokovski 2014</a>), or both. Unfortunately, our results are not directly comparable with studies measuring gene expression divergence between species due to the difference in the features being compared and the temporal window examined; previous studies have looked at the divergence of the overall transcriptome, which is influenced not just by changes in enhancer sequences but by other types of changes and other regulatory elements as well, whereas we have specifically looked at changes in allele frequency and haplotypes in enhancer regions. In addition, previous studies have looked at divergence of the expression patterns between species over an evolutionary period spanning several millions of years, whereas we have examined selection events that have occurred in more recent evolutionary periods, on the order of hundreds or tens of thousands of years. Therefore, it is possible that there has been temporal variation (<em>i.e.</em>, ancient <em>vs.</em> recent) in the occurrence of selection events on these enhancer regions.</p><p id="p-44">More broadly, it is worth emphasizing that most of the 41,561 enhancers examined do not deviate from neutral expectations. Studies examining the divergence of brain and liver transcriptomes across species have previously suggested that tissue transcriptome differences accumulate approximately linearly with evolutionary time (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Khaitovich <em>et al.</em> 2004</a>), consistent with a broadly neutral model of transcriptome evolution. Quantifying the relative contributions of different evolutionary forces across such a large window of evolutionary time, between tissues with vastly different physiologies, and for a group of genetic elements whose evolution is only now beginning to be understood is, our study illustrates, non-trivial. Further dissection of how adaptive and non-adaptive processes shaped enhancers during the evolution of modern humans will greatly advance our understanding of the molecular evolutionary changes that gave rise to human phenotypic diversity.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-45">This research was supported by the March of Dimes through the March of Dimes Prematurity Research Center Ohio Collaborative, the Burroughs Wellcome Fund, the Guggenheim Foundation, and the National Institutes of Health (R01GM115836, R35GM127087). This work was conducted, in part, using the Advanced Computing Center for Research and Education at Vanderbilt University.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-46">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8230757">https://doi.org/10.25387/g3.8230757</a>.</p></li><li class="fn" id="fn-2"><p id="p-47"><em>Communicating editor: S. Tishkoff</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-03-13"><span class="received-label">Received </span>March 13, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-06-13"><span class="accepted-label">Accepted </span>June 13, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Moon <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alexa</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Rahnenfuhrer</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> Gene set enrichment analysis with topGO, R package version 2.30.1</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.2" data-doi="10.1038/nature12787"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Gebhard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Miguel-Escalada</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Hoof</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bornholdt</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">An atlas of active enhancers across human cell types and tissues</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">507</span>: <span class="cit-fpage">455</span>–<span class="cit-lpage">461</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature12787</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12787%26rft_id%253Dinfo%253Apmid%252F24670763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12787&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24670763&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.3" data-doi="10.1093/hmg/ddm242"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Balaresque</span>,  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. J.</span> <span class="cit-name-surname">Ballereau</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Jobling</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Challenges in human genetic diversity: demographic history and adaptation</span>. <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">R134</span>–<span class="cit-lpage">R139</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/hmg/ddm242</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2BMolecular%2BGenetics%26rft.stitle%253DHum%2BMol%2BGenet%26rft.aulast%253DBalaresque%26rft.auinit1%253DP.%2BL.%26rft.volume%253D16%26rft.issue%253DR2%26rft.spage%253DR134%26rft.epage%253DR139%26rft.atitle%253DChallenges%2Bin%2Bhuman%2Bgenetic%2Bdiversity%253A%2Bdemographic%2Bhistory%2Band%2Badaptation%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddm242%26rft_id%253Dinfo%253Apmid%252F17911157%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/hmg/ddm242&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17911157&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251165500003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.4" data-doi="10.1038/nature10532"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brawand</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Soumillon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Necsulea</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Julien</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Csárdi</span></span>, <span class="cit-etal">et al.</span></li></ol><cite> <span class="cit-pub-date">2011</span> <span class="cit-article-title">The evolution of gene expression levels in mammalian organs</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">478</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">348</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature10532</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBrawand%26rft.auinit1%253DD.%26rft.volume%253D478%26rft.issue%253D7369%26rft.spage%253D343%26rft.epage%253D348%26rft.atitle%253DThe%2Bevolution%2Bof%2Bgene%2Bexpression%2Blevels%2Bin%2Bmammalian%2Borgans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10532%26rft_id%253Dinfo%253Apmid%252F22012392%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10532&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22012392&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000296021100037&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.5" data-doi="10.1098/rstb.2013.0025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Capra</span>,  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. D.</span> <span class="cit-name-surname">Erwin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">McKinsey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L. R.</span> <span class="cit-name-surname">Rubenstein</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. S.</span> <span class="cit-name-surname">Pollard</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Many human accelerated regions are developmental enhancers</span>. <abbr class="cit-jnl-abbrev">Phil. Trans. Roy. Soc. B: Biol. Sci.</abbr> <span class="cit-vol">368</span>: <span class="cit-elocation-id">20130025</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rstb.2013.0025</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhil.%2BTrans.%2BRoy.%2BSoc.%2BB%253A%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2013.0025%26rft_id%253Dinfo%253Apmid%252F24218637%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2013.0025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2015117316&amp;link_type=GEOREF" class="cit-ref-sprinkles cit-ref-sprinkles-georef"><span>GeoRef</span></a><a href="/lookup/external-ref?access_num=24218637&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.6" data-doi="10.1371/journal.pbio.0030245"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carroll</span>,  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Evolution at two levels: On genes and form</span>. <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-elocation-id">e245</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.0030245</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DCarroll%26rft.auinit1%253DS.%2BB.%26rft.volume%253D3%26rft.issue%253D7%26rft.spage%253De245%26rft.epage%253De245%26rft.atitle%253DEvolution%2Bat%2Btwo%2Blevels%253A%2Bon%2Bgenes%2Band%2Bform.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0030245%26rft_id%253Dinfo%253Apmid%252F16000021%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0030245&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16000021&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.7" data-doi="10.1093/nar/gkx1020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casper</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. S.</span> <span class="cit-name-surname">Zweig</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Villarreal</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Tyner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. L.</span> <span class="cit-name-surname">Speir</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">The UCSC Genome Browser database: 2018 update</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">D762</span>–<span class="cit-lpage">D769</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkx1020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D46%26rft.spage%253DD762%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx1020%26rft_id%253Dinfo%253Apmid%252F29106570%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx1020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29106570&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.8" data-doi="10.1186/s13742-015-0047-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>,  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. C.</span> <span class="cit-name-surname">Chow</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. C.</span> <span class="cit-name-surname">Tellier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Vattikuti</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. M.</span> <span class="cit-name-surname">Purcell</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Second-generation PLINK: rising to the challenge of larger and richer datasets</span>. <abbr class="cit-jnl-abbrev">GigaSci</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13742-015-0047-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGigaSci%26rft.volume%253D4%26rft.spage%253D7%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13742-015-0047-8%26rft_id%253Dinfo%253Apmid%252F25722852%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13742-015-0047-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25722852&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.9" data-doi="10.1093/bioinformatics/btr330"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Auton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Abecasis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. A.</span> <span class="cit-name-surname">Albers</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Banks</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The variant call format and VCFtools</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">2156</span>–<span class="cit-lpage">2158</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btr330</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtr330%26rft_id%253Dinfo%253Apmid%252F21653522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btr330&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21653522&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292778700023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.10" data-doi="10.1093/molbev/msu077"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferrer-Admetlla</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Liang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Korneliussen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Nielsen</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">On Detecting Incomplete Soft or Hard Selective Sweeps Using Haplotype Structure</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1275</span>–<span class="cit-lpage">1291</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msu077</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu077%26rft_id%253Dinfo%253Apmid%252F24554778%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu077&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24554778&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000335914400020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.11" data-doi="10.1371/journal.pgen.1002355"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fumagalli</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Sironi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Pozzoli</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ferrer-Admettla</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Pattini</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Signatures of environmental genetic adaptation pinpoint pathogens as the main selective pressure through human evolution</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-elocation-id">e1002355</span> (erratum: PLoS Genet. 7: 10.1371/annotation/ca428083-dbcb-476a-956c-d7bb6e317cf7).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002355</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DFumagalli%26rft.auinit1%253DM.%26rft.volume%253D7%26rft.issue%253D11%26rft.spage%253De1002355%26rft.epage%253De1002355%26rft.atitle%253DSignatures%2Bof%2Benvironmental%2Bgenetic%2Badaptation%2Bpinpoint%2Bpathogens%2Bas%2Bthe%2Bmain%2Bselective%2Bpressure%2Bthrough%2Bhuman%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002355%26rft_id%253Dinfo%253Apmid%252F22072984%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002355&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22072984&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.12" data-doi="10.1371/journal.pgen.1005004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Garud</span>,  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. W.</span> <span class="cit-name-surname">Messer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. O.</span> <span class="cit-name-surname">Buzbas</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Petrov</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Recent selective sweeps in North American <em>Drosophila melanogaster</em> show signatures of soft sweeps</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e1005004</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005004%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005004%26rft_id%253Dinfo%253Apmid%252F25706129%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25706129&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.13" data-doi="10.1038/ncomms5438"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gershoni</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Pietrokovski</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Reduced selection and accumulation of deleterious mutations in genes exclusively expressed in men</span>. <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">4438</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms5438</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms5438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms5438&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.14" data-doi="10.1073/pnas.1019276108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gravel</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. M.</span> <span class="cit-name-surname">Henn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. N.</span> <span class="cit-name-surname">Gutenkunst</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Indap</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. T.</span> <span class="cit-name-surname">Marth</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Demographic history and rare allele sharing among human populations</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">11983</span>–<span class="cit-lpage">11988</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1019276108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1019276108%26rft_id%253Dinfo%253Apmid%252F21730125%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzI5LzExOTgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI3NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.15" data-doi="10.1073/pnas.0610797104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>,  <span class="cit-name-given-names">X.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Su</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Tissue-driven hypothesis of genomic evolution and sequence-expression correlations</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">104</span>: <span class="cit-fpage">2779</span>–<span class="cit-lpage">2784</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0610797104</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0610797104%26rft_id%253Dinfo%253Apmid%252F17301236%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTA0LzgvMjc3OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOC8yNzYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.16" data-doi="10.1093/molbev/msw211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haller</span>,  <span class="cit-name-given-names">B. C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. W.</span> <span class="cit-name-surname">Messer</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">SLiM 2: Flexible, Interactive Forward Genetic Simulations</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">230</span>–<span class="cit-lpage">240</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msw211</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsw211%26rft_id%253Dinfo%253Apmid%252F27702775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msw211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27702775&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.17" data-doi="10.1371/journal.pgen.1007859"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harris</span>,  <span class="cit-name-given-names">R. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Sackman</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Jensen</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">On the unfounded enthusiasm for soft selective sweeps II: Examining recent evidence from humans, flies, and viruses</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-elocation-id">e1007859</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1007859</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D14%26rft.spage%253De1007859%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1007859%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1007859&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.18" data-doi="10.1038/ng2104"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haygood</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Fedrigo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Hanson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.-D.</span> <span class="cit-name-surname">Yokoyama</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. A.</span> <span class="cit-name-surname">Wray</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Promoter regions of many neural- and nutrition-related genes have experienced positive selection during human evolution</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1140</span>–<span class="cit-lpage">1144</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng2104</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DHaygood%26rft.auinit1%253DR.%26rft.volume%253D39%26rft.issue%253D9%26rft.spage%253D1140%26rft.epage%253D1144%26rft.atitle%253DPromoter%2Bregions%2Bof%2Bmany%2Bneural-%2Band%2Bnutrition-related%2Bgenes%2Bhave%2Bexperienced%2Bpositive%2Bselection%2Bduring%2Bhuman%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2104%26rft_id%253Dinfo%253Apmid%252F17694055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2104&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17694055&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249122400025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.19" data-doi="10.1038/nature06258"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">International HapMap Consortium</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frazer</span>,  <span class="cit-name-given-names">K. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. G.</span> <span class="cit-name-surname">Ballinger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Cox</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Hinds</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. L.</span> <span class="cit-name-surname">Stuve</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A second generation human haplotype map of over 3.1 million SNPs</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">449</span>: <span class="cit-fpage">851</span>–<span class="cit-lpage">861</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature06258</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DFrazer%26rft.auinit1%253DK.%2BA.%26rft.volume%253D449%26rft.issue%253D7164%26rft.spage%253D851%26rft.epage%253D861%26rft.atitle%253DA%2Bsecond%2Bgeneration%2Bhuman%2Bhaplotype%2Bmap%2Bof%2Bover%2B3.1%2Bmillion%2BSNPs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature06258%26rft_id%253Dinfo%253Apmid%252F17943122%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature06258&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17943122&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000250230600036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.20" data-doi="10.1038/nrg3734"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karlsson</span>,  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. P.</span> <span class="cit-name-surname">Kwiatkowski</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. C.</span> <span class="cit-name-surname">Sabeti</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Natural selection and infectious disease in human populations</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">379</span>–<span class="cit-lpage">393</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3734</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D15%26rft.spage%253D379%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3734%26rft_id%253Dinfo%253Apmid%252F24776769%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3734&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24776769&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.21" data-doi="10.1093/hmg/ddy167"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karnuta</span>,  <span class="cit-name-given-names">J. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. C.</span> <span class="cit-name-surname">Scacheri</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Enhancers: bridging the gap between gene control and human disease</span>. <abbr class="cit-jnl-abbrev">Hum. Mol. Genet.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">R219</span>–<span class="cit-lpage">R227</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/hmg/ddy167</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BMol.%2BGenet.%26rft.volume%253D27%26rft.spage%253DR219%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fhmg%252Fddy167%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/hmg/ddy167&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.22" data-doi="10.1101/gr.229102. Article published online before print in May 2002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kent</span>,  <span class="cit-name-given-names">W. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. W.</span> <span class="cit-name-surname">Sugnet</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. S.</span> <span class="cit-name-surname">Furey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Roskin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. H.</span> <span class="cit-name-surname">Pringle</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The human genome browser at UCSC</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">996</span>–<span class="cit-lpage">1006</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.229102</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.229102.%2BArticle%2Bpublished%2Bonline%2Bbefore%2Bprint%2Bin%2BMay%2B2002%26rft_id%253Dinfo%253Apmid%252F12045153%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjEyLzYvOTk2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI3NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.23" data-doi="10.1371/journal.pbio.0020132"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khaitovich</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Weiss</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lachmann</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Hellmann</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Enard</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A neutral model of transcriptome evolution</span>. <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-elocation-id">e132</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.0020132</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BBiol.%26rft.volume%253D2%26rft.spage%253De132%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0020132%26rft_id%253Dinfo%253Apmid%252F15138501%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0020132&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15138501&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.24" data-doi="10.1126/science.1108296"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khaitovich</span>,  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Parallel patterns of evolution in the genomes and transcriptomes of humans and chimpanzees</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">309</span>: <span class="cit-fpage">1850</span>–<span class="cit-lpage">1854</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1108296</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1108296%26rft_id%253Dinfo%253Apmid%252F16141373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDkvNTc0Mi8xODUwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI3NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.25" data-doi="10.1126/science.1090005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">King</span>,  <span class="cit-name-given-names">M. C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. C.</span> <span class="cit-name-surname">Wilson</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Evolution at two levels in humans and chimpanzees</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">107</span>–<span class="cit-lpage">116</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1090005</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DKing%26rft.auinit1%253DM.%26rft.volume%253D188%26rft.issue%253D4184%26rft.spage%253D107%26rft.epage%253D116%26rft.atitle%253DEvolution%2Bat%2Btwo%2Blevels%2Bin%2Bhumans%2Band%2Bchimpanzees%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1090005%26rft_id%253Dinfo%253Apmid%252F1090005%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czozOiJzY2kiO3M6NToicmVzaWQiO3M6MTI6IjE4OC80MTg0LzEwNyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOC8yNzYxLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.26" data-doi="10.1371/journal.pone.0131673"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kryuchkova-Mostacci</span>,  <span class="cit-name-given-names">N.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Robinson-Rechavi</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Tissue-specific evolution of protein coding genes in human and mouse</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">10</span>: <span class="cit-elocation-id">e0131673</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0131673</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D10%26rft.spage%253De0131673%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0131673%26rft_id%253Dinfo%253Apmid%252F26121354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0131673&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26121354&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.27" data-doi="10.1093/molbev/msn289"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kudaravalli</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Veyrieras</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. E.</span> <span class="cit-name-surname">Stranger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. T.</span> <span class="cit-name-surname">Dermitzakis</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Pritchard</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Gene expression levels are a target of recent natural selection in the human genome</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">649</span>–<span class="cit-lpage">658</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msn289</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsn289%26rft_id%253Dinfo%253Apmid%252F19091723%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msn289&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19091723&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263420900016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.28" data-doi="10.1016/j.cell.2013.02.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">T. I.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Young</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Transcriptional regulation and its misregulation in disease</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">152</span>: <span class="cit-fpage">1237</span>–<span class="cit-lpage">1251</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2013.02.014</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D152%26rft.spage%253D1237%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.02.014%26rft_id%253Dinfo%253Apmid%252F23498934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.02.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23498934&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000316192500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.29" data-doi="10.1016/j.cell.2016.09.018"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Long</span>,  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Prescott</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wysocka</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Ever-changing landscapes: transcriptional enhancers in development and evolution</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">167</span>: <span class="cit-fpage">1170</span>–<span class="cit-lpage">1187</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2016.09.018</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D167%26rft.spage%253D1170%26rft.atitle%253DEVER-CHANGING%2BLANDSCAPES%253A%2BTRANSCRIPTIONAL%2BENHANCERS%2BIN%2BDEVELOPMENT%2BAND%2BEVOLUTION%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2016.09.018%26rft_id%253Dinfo%253Apmid%252F27863239%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2016.09.018&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27863239&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.30" data-doi="10.1016/j.celrep.2014.12.052"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>,  <span class="cit-name-given-names">V. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. C.</span> <span class="cit-name-surname">Nnamani</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kapusta</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Brayer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Plaza</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Ancient transposable elements transformed the uterine regulatory landscape and transcriptome during the evolution of mammalian pregnancy</span>. <abbr class="cit-jnl-abbrev">Cell Reports</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">551</span>–<span class="cit-lpage">561</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.celrep.2014.12.052</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BReports%26rft.volume%253D10%26rft.spage%253D551%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2014.12.052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2014.12.052&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.31" data-doi="10.1093/nar/gkw1133"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">MacArthur</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Bowler</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Cerezo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Gil</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Hall</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The new NHGRI-EBI Catalog of published genome-wide association studies (GWAS Catalog)</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">D896</span>–<span class="cit-lpage">D901</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkw1133</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw1133%26rft_id%253Dinfo%253Apmid%252F27899670%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw1133&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27899670&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.32" data-doi="10.1534/genetics.113.152181"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Messer</span>,  <span class="cit-name-given-names">P. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">SLiM: Simulating evolution with selection and linkage</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">1037</span>–<span class="cit-lpage">1039</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.152181</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.152181%26rft_id%253Dinfo%253Apmid%252F23709637%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5NC80LzEwMzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzgvMjc2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.33" data-doi="10.1016/j.tree.2013.08.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Messer</span>,  <span class="cit-name-given-names">P. W.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Petrov</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Population genomics of rapid adaptation by soft selective sweeps</span>. <abbr class="cit-jnl-abbrev">Trends Ecol. Evol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">659</span>–<span class="cit-lpage">669</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tree.2013.08.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BEcol.%2BEvol.%26rft.volume%253D28%26rft.spage%253D659%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tree.2013.08.003%26rft_id%253Dinfo%253Apmid%252F24075201%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tree.2013.08.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24075201&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000326666200007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.34" data-doi="10.1534/g3.118.200035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moon</span>,  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. M.</span> <span class="cit-name-surname">Aronoff</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Capra</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Abbot</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Rokas</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Examination of signatures of recent positive selection on genes involved in human sialic acid biology</span>. <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">1315</span>–<span class="cit-lpage">1325</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/g3.118.200035</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fg3.118.200035%26rft_id%253Dinfo%253Apmid%252F29467190%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2dnIjtzOjU6InJlc2lkIjtzOjg6IjgvNC8xMzE1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI3NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.35" data-doi="10.1093/bioinformatics/bts277"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neph</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. S.</span> <span class="cit-name-surname">Kuehn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. P.</span> <span class="cit-name-surname">Reynolds</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Haugen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. E.</span> <span class="cit-name-surname">Thurman</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">BEDOPS: high-performance genomic feature operations</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">1919</span>–<span class="cit-lpage">1920</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/bts277</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbts277%26rft_id%253Dinfo%253Apmid%252F22576172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/bts277&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22576172&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306136100017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.36" data-doi="10.1126/sciadv.aao5961"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Neubauer</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.-J.</span> <span class="cit-name-surname">Hublin</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Gunz</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> The evolution of modern human brain shape. Sci. Adv. 4: eaao5961.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/sciadv.aao5961</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2BAdvances%26rft.stitle%253DSci%2BAdv%26rft.aulast%253DNeubauer%26rft.auinit1%253DS.%26rft.volume%253D4%26rft.issue%253D1%26rft.spage%253Deaao5961%26rft.epage%253Deaao5961%26rft.atitle%253DThe%2Bevolution%2Bof%2Bmodern%2Bhuman%2Bbrain%2Bshape%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fsciadv.aao5961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJhZHZhbmNlcyI7czo1OiJyZXNpZCI7czoxMjoiNC8xL2VhYW81OTYxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI3NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.37" data-doi="10.1016/j.cell.2016.09.025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nédélec</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Sanz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Baharian</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z. A.</span> <span class="cit-name-surname">Szpiech</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Pacis</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genetic ancestry and natural selection drive population differences in immune responses to pathogens</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">167</span>: <span class="cit-fpage">657</span>–<span class="cit-lpage">669.e21</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2016.09.025</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D167%26rft.spage%253D657%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2016.09.025%26rft_id%253Dinfo%253Apmid%252F27768889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2016.09.025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27768889&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.38" data-doi="10.1038/nature21347"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Akey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Jakobsson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Pritchard</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Tishkoff</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Tracing the peopling of the world through genomics</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">541</span>: <span class="cit-fpage">302</span>–<span class="cit-lpage">310</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature21347</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D541%26rft.spage%253D302%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature21347%26rft_id%253Dinfo%253Apmid%252F28102248%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature21347&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28102248&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.39" data-doi="10.1038/nrg2957"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ong</span>,  <span class="cit-name-given-names">C.-T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">V. G.</span> <span class="cit-name-surname">Corces</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Enhancer function: new insights into the regulation of tissue-specific gene expression</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">283</span>–<span class="cit-lpage">293</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2957</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DOng%26rft.auinit1%253DC.%2BT.%26rft.volume%253D12%26rft.issue%253D4%26rft.spage%253D283%26rft.epage%253D293%26rft.atitle%253DEnhancer%2Bfunction%253A%2Bnew%2Binsights%2Binto%2Bthe%2Bregulation%2Bof%2Btissue-specific%2Bgene%2Bexpression.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2957%26rft_id%253Dinfo%253Apmid%252F21358745%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2957&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21358745&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.40" data-doi="10.1073/pnas.1524031113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Page</span>,  <span class="cit-name-given-names">A. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Viguier</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Dyble</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Smith</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Chaudhary</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Reproductive trade-offs in extant hunter-gatherers suggest adaptive mechanism for the Neolithic expansion</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">4694</span>–<span class="cit-lpage">4699</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1524031113</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1524031113%26rft_id%253Dinfo%253Apmid%252F27071109%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTEzLzE3LzQ2OTQiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzgvMjc2MS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.41" data-doi="10.1038/nrg3458"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pennacchio</span>,  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Bickmore</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Dean</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Nobrega</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Bejerano</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Enhancers: five essential questions</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">288</span>–<span class="cit-lpage">295</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg3458</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D14%26rft.spage%253D288%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3458%26rft_id%253Dinfo%253Apmid%252F23503198%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3458&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23503198&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.42" data-doi="10.1093/molbev/msj117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pennings</span>,  <span class="cit-name-given-names">P. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hermisson</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Soft sweeps II—molecular population genetics of adaptation from recurrent mutation or migration</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1076</span>–<span class="cit-lpage">1084</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msj117</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsj117%26rft_id%253Dinfo%253Apmid%252F16520336%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msj117&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16520336&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000237320800029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.43" data-doi="10.1093/molbev/msu136"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pfeifer</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Wittelsbürger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. E.</span> <span class="cit-name-surname">Ramos-Onsins</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Lercher</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">PopGenome: an efficient Swiss army knife for population genomic analyses in R</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">1929</span>–<span class="cit-lpage">1936</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msu136</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu136%26rft_id%253Dinfo%253Apmid%252F24739305%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu136&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24739305&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000339423800026&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.44" data-doi="10.1093/bioinformatics/btt637"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raney</span>,  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. R.</span> <span class="cit-name-surname">Dreszer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. P.</span> <span class="cit-name-surname">Barber</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Clawson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. A.</span> <span class="cit-name-surname">Fujita</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Track data hubs enable visualization of user-defined genome-wide annotations on the UCSC Genome Browser</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1003</span>–<span class="cit-lpage">1005</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btt637</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt637%26rft_id%253Dinfo%253Apmid%252F24227676%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt637&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24227676&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000334078300016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.45" data-doi="10.1016/j.gde.2017.04.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rebeiz</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Tsiantis</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Enhancer evolution and the origins of morphological novelty</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">123</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.gde.2017.04.006</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BGenet.%2BDev.%26rft.volume%253D45%26rft.spage%253D115%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2017.04.006%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gde.2017.04.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.46" data-doi="10.1371/journal.pbio.0030387"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rockman</span>,  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. W.</span> <span class="cit-name-surname">Hahn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Soranzo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Zimprich</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. B.</span> <span class="cit-name-surname">Goldstein</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Ancient and recent positive selection transformed opioid cis-regulation in humans</span>. <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">3</span>: <span class="cit-elocation-id">e387</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.0030387</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DRockman%26rft.auinit1%253DM.%2BV.%26rft.volume%253D3%26rft.issue%253D12%26rft.spage%253De387%26rft.epage%253De387%26rft.atitle%253DAncient%2Band%2Brecent%2Bpositive%2Bselection%2Btransformed%2Bopioid%2Bcis-regulation%2Bin%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0030387%26rft_id%253Dinfo%253Apmid%252F16274263%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0030387&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16274263&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.47" data-doi="10.1098/rstb.2013.0017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rubinstein</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. S. J.</span> <span class="cit-name-surname">de Souza</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Evolution of transcriptional enhancers and animal diversity</span>. <abbr class="cit-jnl-abbrev">Phil. Trans. Roy. Soc. B: Biol. Sci.</abbr> <span class="cit-vol">368</span>: <span class="cit-elocation-id">20130017</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rstb.2013.0017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhil.%2BTrans.%2BRoy.%2BSoc.%2BB%253A%2BBiol.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2013.0017%26rft_id%253Dinfo%253Apmid%252F24218630%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rstb.2013.0017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24218630&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.48" data-doi="10.1126/science.1124309"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sabeti</span>,  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. F.</span> <span class="cit-name-surname">Schaffner</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Fry</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Lohmueller</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Varilly</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Positive natural selection in the human lineage</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">312</span>: <span class="cit-fpage">1614</span>–<span class="cit-lpage">1620</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1124309</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSabeti%26rft.auinit1%253DP.%2BC.%26rft.volume%253D312%26rft.issue%253D5780%26rft.spage%253D1614%26rft.epage%253D1620%26rft.atitle%253DPositive%2Bnatural%2Bselection%2Bin%2Bthe%2Bhuman%2Blineage.%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1124309%26rft_id%253Dinfo%253Apmid%252F16778047%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTIvNTc4MC8xNjE0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI3NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.49" data-doi="10.1038/nrg1247"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schaffner</span>,  <span class="cit-name-given-names">S. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The X chromosome in population genetics</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">43</span>–<span class="cit-lpage">51</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg1247</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DSchaffner%26rft.auinit1%253DS.%2BF.%26rft.volume%253D5%26rft.issue%253D1%26rft.spage%253D43%26rft.epage%253D51%26rft.atitle%253DThe%2BX%2Bchromosome%2Bin%2Bpopulation%2Bgenetics.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1247%26rft_id%253Dinfo%253Apmid%252F14708015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1247&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14708015&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000187641100015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.50" data-doi="10.1016/j.tig.2009.12.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sholtis</span>,  <span class="cit-name-given-names">S. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. P.</span> <span class="cit-name-surname">Noonan</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Gene regulation and the origins of human biological uniqueness</span>. <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">110</span>–<span class="cit-lpage">118</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.tig.2009.12.009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DSholtis%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D26%26rft.issue%253D3%26rft.spage%253D110%26rft.epage%253D118%26rft.atitle%253DGene%2Bregulation%2Band%2Bthe%2Borigins%2Bof%2Bhuman%2Bbiological%2Buniqueness.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2009.12.009%26rft_id%253Dinfo%253Apmid%252F20106546%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2009.12.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20106546&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.51" data-doi="10.1093/molbev/msx219"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simonti</span>,  <span class="cit-name-given-names">C. N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Pavličev</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Capra</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Transposable element exaptation into regulatory regions is rare, influenced by evolutionary age, and subject to pleiotropic constraints</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">2856</span>–<span class="cit-lpage">2869</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msx219</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsx219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msx219&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.52" data-doi="10.1093/molbev/msu211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szpiech</span>,  <span class="cit-name-given-names">Z. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. D.</span> <span class="cit-name-surname">Hernandez</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">selscan: an efficient multithreaded program to perform EHH-based scans for positive selection</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">2824</span>–<span class="cit-lpage">2827</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msu211</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsu211%26rft_id%253Dinfo%253Apmid%252F25015648%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msu211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25015648&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">The 1000 Genomes Project Consortium</span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. A.</span> <span class="cit-name-surname">Gibbs</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Boerwinkle</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Doddapaneni</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Han</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A global reference for human genetic variation</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">526</span>: <span class="cit-fpage">68</span>–<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.54" data-doi="10.1038/nature13182"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">The FANTOM Consortium and the RIKEN PMI and CLST (DGT)</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Forrest</span>,  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Kawaji</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Rehli</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Baillie</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A promoter-level mammalian expression atlas</span>. <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">507</span>: <span class="cit-fpage">462</span>–<span class="cit-lpage">470</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nature13182</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D507%26rft.spage%253D462%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature13182%26rft_id%253Dinfo%253Apmid%252F24670764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature13182&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24670764&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000333402000033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.8.2761.55" data-doi="10.1038/ng1946"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tishkoff</span>,  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. A.</span> <span class="cit-name-surname">Reed</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Ranciaro</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. F.</span> <span class="cit-name-surname">Voight</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. C.</span> <span class="cit-name-surname">Babbitt</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Convergent adaptation of human lactase persistence in Africa and Europe</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">31</span>–<span class="cit-lpage">40</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1946</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D39%26rft.spage%253D31%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1946%26rft_id%253Dinfo%253Apmid%252F17159977%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1946&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17159977&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.56" data-doi="10.1126/science.1260419"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Uhlen</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Fagerberg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. M.</span> <span class="cit-name-surname">Hallstrom</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Lindskog</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Oksvold</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Tissue-based map of the human proteome</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">347</span>: <span class="cit-elocation-id">1260419</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.1260419</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DUhlen%26rft.auinit1%253DM.%26rft.volume%253D347%26rft.issue%253D6220%26rft.spage%253D1260419%26rft.epage%253D1260419%26rft.atitle%253DTissue-based%2Bmap%2Bof%2Bthe%2Bhuman%2Bproteome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1260419%26rft_id%253Dinfo%253Apmid%252F25613900%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE2OiIzNDcvNjIyMC8xMjYwNDE5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI3NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.57" data-doi="10.1073/pnas.0908196106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Varki</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Gagneux</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Human-specific evolution of sialic acid targets: Explaining the malignant malaria mystery?</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">14739</span>–<span class="cit-lpage">14740</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0908196106</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0908196106%26rft_id%253Dinfo%253Apmid%252F19717444%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA2LzM1LzE0NzM5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI3NjEuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.58" data-doi="10.1146/annurev-genet-111212-133526"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vitti</span>,  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Grossman</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. C.</span> <span class="cit-name-surname">Sabeti</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Detecting natural selection in genomic data</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-genet-111212-133526</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnu.%2BRev.%2BGenet.%26rft.volume%253D47%26rft.spage%253D97%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genet-111212-133526%26rft_id%253Dinfo%253Apmid%252F24274750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genet-111212-133526&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24274750&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000329816000005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.59" data-doi="10.1371/journal.pbio.0040072"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Voight</span>,  <span class="cit-name-given-names">B. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kudaravalli</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Wen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Pritchard</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">A map of recent positive selection in the human genome</span>. <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-elocation-id">e72</span> (erratum: PLoS Biol. 4: e154 and 5: e147).<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pbio.0040072</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DVoight%26rft.auinit1%253DB.%2BF.%26rft.volume%253D4%26rft.issue%253D3%26rft.spage%253De72%26rft.epage%253De72%26rft.atitle%253DA%2Bmap%2Bof%2Brecent%2Bpositive%2Bselection%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0040072%26rft_id%253Dinfo%253Apmid%252F16494531%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0040072&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16494531&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.60" data-doi="10.1093/bioinformatics/btm087"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Du</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Payattakool</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. S.</span> <span class="cit-name-surname">Yu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. F.</span> <span class="cit-name-surname">Chen</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A new method to measure the semantic similarity of GO terms</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">1274</span>–<span class="cit-lpage">1281</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btm087</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtm087%26rft_id%253Dinfo%253Apmid%252F17344234%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btm087&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17344234&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247348300013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.61" data-doi="10.1111/j.1558-5646.1984.tb05657.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weir</span>,  <span class="cit-name-given-names">B. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. C.</span> <span class="cit-name-surname">Cockerham</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">Estimating F-Statistics for the Analysis of Population Structure</span>. <abbr class="cit-jnl-abbrev">Evol</abbr>. <span class="cit-vol">38</span>: <span class="cit-fpage">1358</span>–<span class="cit-lpage">1370</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1558-5646.1984.tb05657.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvol%26rft.volume%253D38%26rft.spage%253D1358%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.1984.tb05657.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.1984.tb05657.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.62" data-doi="10.1038/nrg2063"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wray</span>,  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The evolutionary significance of cis-regulatory mutations</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">206</span>–<span class="cit-lpage">216</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrg2063</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DWray%26rft.auinit1%253DG.%2BA.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253D206%26rft.epage%253D216%26rft.atitle%253DThe%2Bevolutionary%2Bsignificance%2Bof%2Bcis-regulatory%2Bmutations.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2063%26rft_id%253Dinfo%253Apmid%252F17304246%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2063&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17304246&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000244274400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2761.63" data-doi="10.1093/bioinformatics/btq064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Qin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Bo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Wu</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">GOSemSim: an R package for measuring semantic similarity among GO terms and gene products</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">976</span>–<span class="cit-lpage">978</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btq064</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq064%26rft_id%253Dinfo%253Apmid%252F20179076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20179076&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2761.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276045800023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/8/2761.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/8/2749" title="Genome-Wide Profiling of Polyadenylation Events in Maize Using High-Throughput Transcriptomic Sequences" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/8/2775" title="TNF-α Differentially Regulates Cell Cycle Genes in Promyelocytic and Granulocytic HL-60/S4 Cells" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/8"><span>Volume 9 Issue 8, August 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/8.cover-source.jpg?itok=cSNYBBSf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/8/2761" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-9ORCE80v0OGxMTp1JCmS3z-TxCsy4OpHu1pGNaG8fiQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/8/2761" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/8/2761" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/8/2761" class="active" data-icon-position="" data-hide-link-title="0">Signatures of Recent Positive Selection in Enhancers Across 41 Human Tissues</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/493760" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-LLy2Q6YSHjIBemSG2VoVMM2xlP-e0GqH5pH-kuPuIKI" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/8/2761.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/8/2761" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-Y2etcCLT19WaVdaztZOfwzXnujRQ1WRxKyzM-_ddbhQ" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/8/2761" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/8/2761.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493760" id="node493760--2" data-pisa="ggg;9/8/2761" data-pisa-master="ggg;g3.119.400186" data-apath="/ggg/9/8/2761.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Signatures of Recent Positive Selection in Enhancers Across 41 Human Tissues</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jiyun M. Moon</span>, <span class="highwire-citation-author" data-delta="1">John A. Capra</span>, <span class="highwire-citation-author" data-delta="2">Patrick Abbot</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-7248-6551" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Antonis Rokas</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2761-2774;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400186 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493760--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jiyun M. Moon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiyun%2BM.%2BMoon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moon%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJiyun%2BM.%2BMoon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John A. Capra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Vanderbilt Genetics Institute, and</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Departments of Biomedical Informatics and Computer Science, Vanderbilt University, Nashville, TN, 37235</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BA.%2BCapra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Capra%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BA.%2BCapra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Patrick Abbot </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BAbbot%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Abbot%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BAbbot%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Antonis Rokas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Vanderbilt Genetics Institute, and</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Departments of Biomedical Informatics and Computer Science, Vanderbilt University, Nashville, TN, 37235</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonis%2BRokas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rokas%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAntonis%2BRokas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7248-6551" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Antonis Rokas</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:antonis.rokas@vanderbilt.edu" class="" data-icon-position="" data-hide-link-title="0">antonis.rokas@vanderbilt.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/8/2761" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/8/2761&amp;title=Signatures%20of%20Recent%20Positive%20Selection%20in%20Enhancers%20Across%2041%20Human%20Tissues" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/8/2761&amp;title=Signatures%20of%20Recent%20Positive%20Selection%20in%20Enhancers%20Across%2041%20Human%20Tissues" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/8/2761&amp;title=Signatures%20of%20Recent%20Positive%20Selection%20in%20Enhancers%20Across%2041%20Human%20Tissues" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/8/2761&amp;text=Signatures%20of%20Recent%20Positive%20Selection%20in%20Enhancers%20Across%2041%20Human%20Tissues" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/8/2761&amp;title=Signatures%20of%20Recent%20Positive%20Selection%20in%20Enhancers%20Across%2041%20Human%20Tissues" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/8/2761&amp;t=Signatures%20of%20Recent%20Positive%20Selection%20in%20Enhancers%20Across%2041%20Human%20Tissues" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/8/2761&amp;title=Signatures%20of%20Recent%20Positive%20Selection%20in%20Enhancers%20Across%2041%20Human%20Tissues" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/8/2761&amp;title=Signatures%20of%20Recent%20Positive%20Selection%20in%20Enhancers%20Across%2041%20Human%20Tissues" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/493760/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493760" id="node493760--3" data-pisa="ggg;9/8/2761" data-pisa-master="ggg;g3.119.400186" data-apath="/ggg/9/8/2761.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Signatures of Recent Positive Selection in Enhancers Across 41 Human Tissues</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jiyun M. Moon</span>, <span class="highwire-citation-author" data-delta="1">John A. Capra</span>, <span class="highwire-citation-author" data-delta="2">Patrick Abbot</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-7248-6551" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Antonis Rokas</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2761-2774;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400186 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493760--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jiyun M. Moon </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jiyun%2BM.%2BMoon%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Moon%20JM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJiyun%2BM.%2BMoon%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">John A. Capra </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Vanderbilt Genetics Institute, and</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Departments of Biomedical Informatics and Computer Science, Vanderbilt University, Nashville, TN, 37235</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=John%2BA.%2BCapra%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Capra%20JA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJohn%2BA.%2BCapra%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Patrick Abbot </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patrick%2BAbbot%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Abbot%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3APatrick%2BAbbot%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Antonis Rokas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biological Sciences,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Vanderbilt Genetics Institute, and</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Departments of Biomedical Informatics and Computer Science, Vanderbilt University, Nashville, TN, 37235</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Antonis%2BRokas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rokas%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAntonis%2BRokas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-7248-6551" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Antonis Rokas</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:antonis.rokas@vanderbilt.edu" class="" data-icon-position="" data-hide-link-title="0">antonis.rokas@vanderbilt.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2761&amp;rft.epage=2774&amp;rft.atitle=Signatures%20of%20Recent%20Positive%20Selection%20in%20Enhancers%20Across%2041%20Human%20Tissues&amp;rft.volume=9&amp;rft.issue=8&amp;rft.date=2019-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Moon%2C+Jiyun+M.&amp;rft.au=Capra%2C+John+A.&amp;rft.au=Abbot%2C+Patrick&amp;rft.au=Rokas%2C+Antonis"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/493760/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/493760/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/493760/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/493760/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/493760/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/493760/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/493760/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/493760/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/493760/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/493760/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/493760/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/493760/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/8/2761" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-20" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/8/2761.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/8/2761.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/8/2761.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
