<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/8/2489" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/8/2489.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/8/2489.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/8/2489.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/8/2489.atom" />
<meta name="HW.pisa" content="ggg;9/8/2489" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Systematic Analysis of Metabolic Pathway Distributions of Bacterial Energy Reserves" />
<meta name="DC.Identifier" content="10.1534/g3.119.400123" />
<meta name="DC.Date" content="2019-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Wang et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Previous bioinformatics studies have linked gain or loss of energy reserves with host-pathogen interactions and bacterial virulence based on a comparatively small number of bacterial genomes or proteomes. Thus, understanding the theoretical distribution patterns of energy reserves across bacterial species could provide a shortcut route to look into bacterial lifestyle and physiology. So far, five major energy reserves have been identified in bacteria due to their capacity to support bacterial persistence under nutrient deprivation conditions. These include polyphosphate (polyP), glycogen, wax ester (WE), triacylglycerol (TAG), and polyhydroxyalkanoates (PHAs). Although the enzymes related with metabolism of energy reserves are well understood, there is a lack of systematic investigations into the distribution of bacterial energy reserves from an evolutionary point of view. In this study, we sourced 8282 manually reviewed bacterial reference proteomes and combined a set of hidden Markov sequence models (HMMs) to search homologs of key enzymes related with the metabolism of energy reserves. Our results revealed that specific pathways like trehalose-related glycogen metabolism and enzymes such as wax ester synthase/acyl-CoA:diacylglycerol acyltransferase (WS/DGAT) are mainly restricted within specific types of bacterial groups, which provides evolutionary insights into the understanding of their origins and functions. In addition, the study also confirms that loss of energy reserves like polyP metabolism absence in Mollicutes is correlated with bacterial genome reduction. Through this analysis, a clearer picture about the metabolism of energy reserves in bacteria is presented, which could serve as a guide for further theoretical and experimental analyses of bacterial energy metabolism." />
<meta name="DC.Contributor" content="Liang Wang" />
<meta name="DC.Contributor" content="Jianye Yang" />
<meta name="DC.Contributor" content="Yue Huang" />
<meta name="DC.Contributor" content="Qinghua Liu" />
<meta name="DC.Contributor" content="Yaping Xu" />
<meta name="DC.Contributor" content="Xue Piao" />
<meta name="DC.Contributor" content="Michael J. Wise" />
<meta name="article:published_time" content="2019-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Systematic Analysis of Metabolic Pathway Distributions of Bacterial Energy Reserves" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Previous bioinformatics studies have linked gain or loss of energy reserves with host-pathogen interactions and bacterial virulence based on a comparatively small number of bacterial genomes or proteomes. Thus, understanding the theoretical distribution patterns of energy reserves across bacterial species could provide a shortcut route to look into bacterial lifestyle and physiology. So far, five major energy reserves have been identified in bacteria due to their capacity to support bacterial persistence under nutrient deprivation conditions. These include polyphosphate (polyP), glycogen, wax ester (WE), triacylglycerol (TAG), and polyhydroxyalkanoates (PHAs). Although the enzymes related with metabolism of energy reserves are well understood, there is a lack of systematic investigations into the distribution of bacterial energy reserves from an evolutionary point of view. In this study, we sourced 8282 manually reviewed bacterial reference proteomes and combined a set of hidden Markov sequence models (HMMs) to search homologs of key enzymes related with the metabolism of energy reserves. Our results revealed that specific pathways like trehalose-related glycogen metabolism and enzymes such as wax ester synthase/acyl-CoA:diacylglycerol acyltransferase (WS/DGAT) are mainly restricted within specific types of bacterial groups, which provides evolutionary insights into the understanding of their origins and functions. In addition, the study also confirms that loss of energy reserves like polyP metabolism absence in &lt;i&gt;Mollicutes&lt;/i&gt; is correlated with bacterial genome reduction. Through this analysis, a clearer picture about the metabolism of energy reserves in bacteria is presented, which could serve as a guide for further theoretical and experimental analyses of bacterial energy metabolism.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/08/01" />
<meta name="citation_mjid" content="ggg;9/8/2489" />
<meta name="citation_id" content="9/8/2489" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/8/2489" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/8/2489.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/8/2489.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/8/2489.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400123" />
<meta name="citation_pmid" content="31151997" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="8" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2489" />
<meta name="citation_lastpage" content="2496" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Liang Wang" />
<meta name="citation_author_institution" content="Department of Bioinformatics, School of Medical Informatics," />
<meta name="citation_author_institution" content="Jiangsu Key Lab of New Drug Research and Clinical Pharmacy, Xuzhou Medical University, Xuzhou, Jiangsu Province, 221000, China," />
<meta name="citation_author_email" content="leonwang@xzhmu.edu.cn" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-5339-7484" />
<meta name="citation_author" content="Jianye Yang" />
<meta name="citation_author_institution" content="Department of Bioinformatics, School of Medical Informatics," />
<meta name="citation_author" content="Yue Huang" />
<meta name="citation_author_institution" content="Department of Bioinformatics, School of Medical Informatics," />
<meta name="citation_author" content="Qinghua Liu" />
<meta name="citation_author_institution" content="Jiangsu Key Lab of New Drug Research and Clinical Pharmacy, Xuzhou Medical University, Xuzhou, Jiangsu Province, 221000, China," />
<meta name="citation_author" content="Yaping Xu" />
<meta name="citation_author_institution" content="Department of Bioinformatics, School of Medical Informatics," />
<meta name="citation_author" content="Xue Piao" />
<meta name="citation_author_institution" content="Department of Bioinformatics, School of Medical Informatics," />
<meta name="citation_author_institution" content="School of Information and Control Engineering, China University of Mining and Technology, Xuzhou, Jiangsu, 221116, China," />
<meta name="citation_author" content="Michael J. Wise" />
<meta name="citation_author_institution" content="Department of Computer Science and Software Engineering, and" />
<meta name="citation_author_institution" content="The Marshall Centre for Infectious Diseases Research and Training, University of Western Australia, Perth 6009, WA, Australia" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-6559-4303" />
<meta name="citation_reference" content="citation_journal_title=Biologia;citation_author=L. Achbergerová;citation_author=J. Nahálka;citation_title=PPK1 and PPK2 — which polyphosphate kinase is older?;citation_pages=263-269;citation_volume=69;citation_year=2014;citation_doi=10.2478/s11756-013-0324-x" />
<meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.;citation_author=A. Arabolaza;citation_author=E. Rodriguez;citation_author=S. Altabe;citation_author=H. Alvarez;citation_author=H. Gramajo;citation_title=Multiple Pathways for Triacylglycerol Biosynthesis in Streptomyces coelicolor;citation_volume=74;citation_year=2008;citation_pmid=18310412;citation_doi=10.1128/AEM.02638-07" />
<meta name="citation_reference" content="citation_journal_title=Chemistry &amp; biology;citation_journal_abbrev=Chem Biol;citation_author=D. Barkan;citation_author=Z. Liu;citation_author=J. C. Sacchettini;citation_author=M. S. Glickman;citation_title=Mycolic acid cyclopropanation is essential for viability, drug resistance, and cell wall integrity of Mycobacterium tuberculosis.;citation_pages=499-509;citation_volume=16;citation_year=2009;citation_issue=5;citation_pmid=19477414;citation_doi=10.1016/j.chembiol.2009.04.001" />
<meta name="citation_reference" content="citation_title=Biotechnol. J.;citation_author=U. Breuer;citation_title=Plastics from Bacteria - Natural Functions and Applications. By Guo-Qiang Chen (Editor), Alexander Steinbüchel (Series Editor);citation_pages=1351-1351;citation_volume=5;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Microbiology;citation_author=G. Chandra;citation_author=K. F. Chater;citation_author=S. Bornemann;citation_title=Unexpected and widespread connections between bacterial glycogen and trehalose metabolism;citation_volume=157;citation_year=2011;citation_pmid=21474533;citation_doi=10.1099/mic.0.044263-0" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. Dahlqvist;citation_author=U. Stahl;citation_author=M. Lenman;citation_author=A. Banas;citation_author=M. Lee;citation_title=Phospholipid:diacylglycerol acyltransferase: An enzyme that catalyzes the acyl-CoA-independent formation of triacylglycerol in yeast and plants;citation_volume=97;citation_year=2000;citation_pmid=10829075;citation_doi=10.1073/pnas.120067297" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=R. C. Edgar;citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput;citation_pages=1792-1797;citation_volume=32;citation_year=2004;citation_issue=5;citation_pmid=15034147;citation_doi=10.1093/nar/gkh340" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=R. C. Edgar;citation_title=Search and clustering orders of magnitude faster than BLAST;citation_volume=26;citation_year=2010;citation_pmid=20709691;citation_doi=10.1093/bioinformatics/btq461" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. Federhen;citation_title=The NCBI Taxonomy database;citation_volume=40;citation_year=2011;citation_pmid=22139910;citation_doi=10.1093/nar/gkr1178" />
<meta name="citation_reference" content="citation_journal_title=Geology;citation_journal_abbrev=Geology;citation_author=D. B. Finkelstein;citation_author=S. C. Brassell;citation_author=L. M. Pratt;citation_title=Microbial biosynthesis of wax esters during desiccation: Adaptation for colonization of the earliest terrestrial environments?;citation_pages=247-250;citation_volume=38;citation_year=2010;citation_issue=3;citation_doi=10.1130/G30398.1;citation_georef=2010030931" />
<meta name="citation_reference" content="citation_journal_title=International Journal of Systematic and Evolutionary Microbiology;citation_journal_abbrev=Int. J. Syst. Evol. Microbiol.;citation_author=G. Funke;citation_author=S. Stubbs;citation_author=M. Altwegg;citation_author=A. Carlotti;citation_author=M. D. Collins;citation_title=Turicella otitidis gen. nov., sp. nov., a Coryneform Bacterium Isolated from Patients with Otitis Media;citation_pages=270-273;citation_volume=44;citation_year=1994;citation_issue=2;citation_pmid=8186090;citation_doi=10.1099/00207713-44-2-270" />
<meta name="citation_reference" content="citation_journal_title=Microbiol. Spectr.;citation_author=M. Y. Galperin;citation_title=Genome Diversity of Spore-Forming Firmicutes;citation_pages=1-15;citation_volume=1;citation_year=2013;citation_doi=10.1128/microbiolspectrum.TBS-0015-2012" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_title=Predicting the Minimal Translation Apparatus: Lessons from the Reductive Evolution of Mollicutes;citation_volume=10;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Journal of Bacteriology;citation_journal_abbrev=J. Bacteriol.;citation_author=R. Handrick;citation_author=S. Reinhardt;citation_author=P. Kimmig;citation_author=D. Jendrossek;citation_title=The &quot;Intracellular&quot; Poly(3-Hydroxybutyrate) (PHB) Depolymerase of Rhodospirillum rubrum Is a Periplasm-Located Protein with Specificity for Native PHB and with Structural Similarity to Extracellular PHB Depolymerases;citation_pages=7243-7253;citation_volume=186;citation_year=2004;citation_issue=21;citation_pmid=15489436;citation_doi=10.1128/JB.186.21.7243-7253.2004" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=B. Henrissat;citation_author=E. Deleury;citation_author=P. M. Coutinho;citation_title=Glycogen metabolism loss: a common marker of parasitic behaviour in bacteria?;citation_pages=437-440;citation_volume=18;citation_year=2002;citation_issue=9;citation_pmid=12175798;citation_doi=10.1016/S0168-9525(02)02734-8" />
<meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.;citation_author=R. Kalscheuer;citation_author=A. Steinbüchel;citation_title=A Novel Bifunctional Wax Ester Synthase/Acyl-CoA:Diacylglycerol Acyltransferase Mediates Wax Ester and Triacylglycerol Biosynthesis inAcinetobacter calcoaceticusADP1;citation_volume=278;citation_year=2003;citation_pmid=12502715;citation_doi=10.1074/jbc.M210533200" />
<meta name="citation_reference" content="citation_title=Handbook of Hydrocarbon and Lipid Microbiology;citation_title=Actinobacteria;citation_pages=1819-1838;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=I. Letunic;citation_author=P. Bork;citation_title=Interactive tree of life (iTOL) v3: an online tool for the display and annotation of phylogenetic and other trees;citation_volume=44;citation_year=2016;citation_pmid=27095192;citation_doi=10.1093/nar/gkw290" />
<meta name="citation_reference" content="citation_journal_title=Chem. Biol.;citation_author=H. Marrakchi;citation_author=M.-A. Lanéelle;citation_author=M. Daffé;citation_title=Mycolic Acids: Structures, Biosynthesis, and Beyond;citation_pages=67-85;citation_volume=21;citation_year=2014;citation_pmid=24374164;citation_doi=10.1016/j.chembiol.2013.11.011" />
<meta name="citation_reference" content="citation_author=S. McGinnis;citation_author=T.L. Madden;citation_pages=20;citation_volume=32;citation_year=2004;citation_doi=10.1093/nar/gkh435" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=N. A. Moran;citation_title=Microbial minimalism: genome reduction in bacterial pathogens.;citation_pages=583-586;citation_volume=108;citation_year=2002;citation_issue=5;citation_pmid=11893328;citation_doi=10.1016/S0092-8674(02)00665-7" />
<meta name="citation_reference" content="citation_journal_title=Structure (London);citation_journal_abbrev=Structure (London);citation_author=C. A. Orengo;citation_author=A. D. Michie;citation_author=S. Jones;citation_author=D. T. Jones;citation_author=M. B. Swindells;citation_title=CATH--a hierarchic classification of protein domain structures.;citation_pages=1093-1108;citation_volume=5;citation_year=1997;citation_issue=8;citation_pmid=9309224;citation_doi=10.1016/S0969-2126(97)00260-8" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=W. R. Pearson;citation_author=S. R. Eddy;citation_title=Accelerated Profile HMM Searches;citation_pages=1-16;citation_volume=7;citation_year=2011;citation_doi=10.1371/journal.pcbi.1001120" />
<meta name="citation_reference" content="citation_title=The Prokaryotes;citation_title=The Family Dietziaceae;citation_pages=327-338;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Biochem. J.;citation_author=B. H. Rehm;citation_title=Polyester synthases: natural catalysts for plastics;citation_volume=376;citation_year=2003;citation_pmid=12954080;citation_doi=10.1042/bj20031254" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=A. Rottig;citation_author=A. Steinbuchel;citation_title=Acyltransferases in Bacteria;citation_pages=277-321;citation_volume=77;citation_year=2013;citation_issue=2;citation_pmid=23699259;citation_doi=10.1128/MMBR.00010-13" />
<meta name="citation_reference" content="citation_journal_title=Appl. Environ. Microbiol.;citation_author=A. Röttig;citation_author=C. S. Strittmatter;citation_author=J. Schauer;citation_author=S. Hiessl;citation_author=A. Poehlein;citation_title=Role of Wax Ester Synthase/Acyl Coenzyme A:Diacylglycerol Acyltransferase in Oleaginous Streptomyces sp. Strain G25;citation_volume=82;citation_year=2016;citation_pmid=27474711;citation_doi=10.1128/AEM.01719-16" />
<meta name="citation_reference" content="citation_journal_title=J. Bacteriol.;citation_author=K. Uchino;citation_author=T. Saito;citation_author=B. Gebauer;citation_author=D. Jendrossek;citation_title=Isolated Poly(3-Hydroxybutyrate) (PHB) Granules Are Complex Bacterial Organelles Catalyzing Formation of PHB from Acetyl Coenzyme A (CoA) and Degradation of PHB to Acetyl-CoA;citation_volume=189;citation_year=2007;citation_pmid=17720797;citation_doi=10.1128/JB.00752-07" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_title=UniProt: the universal protein knowledgebase;citation_volume=45;citation_year=2017;citation_pmid=27899622;citation_doi=10.1093/nar/gkw1099" />
<meta name="citation_reference" content="citation_journal_title=Journal of Applied Microbiology (Print);citation_journal_abbrev=Journal of Applied Microbiology (Print);citation_author=R. A. Verlinden;citation_author=D. J. Hill;citation_author=M. A. Kenward;citation_author=C. D. Williams;citation_author=I. Radecka;citation_title=Bacterial synthesis of biodegradable polyhydroxyalkanoates.;citation_pages=1437-1449;citation_volume=102;citation_year=2007;citation_issue=6;citation_pmid=17578408;citation_doi=10.1111/j.1365-2672.2007.03335.x" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=L. Wang;citation_author=Q. Liu;citation_author=X. Wu;citation_author=Y. Huang;citation_author=M. J. Wise;citation_title=Bioinformatics Analysis of Metabolism Pathways of Archaeal Energy Reserves;citation_pages=1034;citation_volume=9;citation_year=2019;citation_doi=10.1038/s41598-018-37768-0" />
<meta name="citation_reference" content="citation_journal_title=Front. Microbiol.;citation_author=L. Wang;citation_author=Z. Liu;citation_author=S. Dai;citation_author=J. Yan;citation_author=M. J. Wise;citation_title=The Sit-and-Wait Hypothesis in Bacterial Pathogens: A Theoretical Study of Durability and Virulence;citation_pages=2167;citation_volume=8;citation_year=2017;citation_doi=10.3389/fmicb.2017.02167" />
<meta name="citation_reference" content="citation_journal_title=Naturwissenschaften;citation_journal_abbrev=Naturwissenschaften;citation_author=L. Wang;citation_author=M. J. Wise;citation_title=Glycogen with short average chain length enhances bacterial durability.;citation_pages=719-729;citation_volume=98;citation_year=2011;citation_issue=9;citation_pmid=21808975;citation_doi=10.1007/s00114-011-0832-x" />
<meta name="citation_reference" content="citation_journal_title=Front. Microbiol.;citation_author=L. Wang;citation_author=J. Yan;citation_author=M. J. Wise;citation_author=Q. Liu;citation_author=J. Asenso;citation_title=Distribution Patterns of Polyphosphate Metabolism Pathway and Its Relationships With Bacterial Durability and Virulence;citation_pages=782;citation_volume=9;citation_year=2018;citation_pmid=29755430;citation_doi=10.3389/fmicb.2018.00782" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. M. Waterhouse;citation_author=J. B. Procter;citation_author=D. M. A. Martin;citation_author=M. Clamp;citation_author=G. J. Barton;citation_title=Jalview Version 2–a multiple sequence alignment editor and analysis workbench;citation_volume=25;citation_year=2009;citation_pmid=19151095;citation_doi=10.1093/bioinformatics/btp033" />
<meta name="citation_reference" content="citation_author=M.P. Whitehead;citation_author=L. Eagles;citation_author=P. Hooley;citation_author=M.R.W. Brown;citation_year=2014;citation_pmid=24600028;citation_doi=10.1099/mic.0.075366-0" />
<meta name="citation_reference" content="citation_journal_title=Exp. Cell Res.;citation_author=J. Wilkinson;citation_title=The problem of energy-storage compounds in bacteria;citation_pages=111-130;citation_volume=7;citation_year=1959;citation_doi=10.1016/0014-4827(59)90237-X" />
<meta name="citation_reference" content="citation_journal_title=Cladistics;citation_author=M. J. Wise;citation_title=No so HoT - heads or tails is not able to reliably compare multiple sequence alignments;citation_pages=438-443;citation_volume=26;citation_year=2010;citation_doi=10.1111/j.1096-0031.2009.00292.x" />
<meta name="citation_reference" content="citation_journal_title=BioEssays;citation_author=Y. I. Wolf;citation_author=E. V. Koonin;citation_title=Genome reduction as the dominant mode of evolution;citation_pages=829-837;citation_volume=35;citation_year=2013;citation_pmid=23801028;citation_doi=10.1002/bies.201300037" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=H. Zhang;citation_author=K. Ishige;citation_author=A. Kornberg;citation_title=A polyphosphate kinase (PPK2) widely conserved in bacteria;citation_volume=99;citation_year=2002;citation_pmid=12486232;citation_doi=10.1073/pnas.262655199" />
<meta name="citation_reference" content="citation_journal_title=ISME J.;citation_author=B. Zhu;citation_author=M. Ibrahim;citation_author=Z. Cui;citation_author=G. Xie;citation_author=G. Jin;citation_title=Multi-omics analysis of niche specificity provides new insights into ecological adaptation in bacteria;citation_pages=2072-2075;citation_volume=10;citation_year=2016;citation_doi=10.1038/ismej.2015.251" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Systematic Analysis of Metabolic Pathway Distributions of Bacterial Energy Reserves" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8.cover-source.jpg" />
<meta name="twitter:description" content="Previous bioinformatics studies have linked gain or loss of energy reserves with host-pathogen interactions and bacterial virulence based on a comparatively small number of bacterial genomes or proteomes. Thus, understanding the theoretical distribution patterns of energy reserves across bacterial species could provide a shortcut route to look into bacterial lifestyle and physiology. So far, five major energy reserves have been identified in bacteria due to their capacity to support bacterial persistence under nutrient deprivation conditions. These include polyphosphate (polyP), glycogen, wax ester (WE), triacylglycerol (TAG), and polyhydroxyalkanoates (PHAs). Although the enzymes related with metabolism of energy reserves are well understood, there is a lack of systematic investigations into the distribution of bacterial energy reserves from an evolutionary point of view. In this study, we sourced 8282 manually reviewed bacterial reference proteomes and combined a set of hidden Markov sequence models (HMMs) to search homologs of key enzymes related with the metabolism of energy reserves. Our results revealed that specific pathways like trehalose-related glycogen metabolism and enzymes such as wax ester synthase/acyl-CoA:diacylglycerol acyltransferase (WS/DGAT) are mainly restricted within specific types of bacterial groups, which provides evolutionary insights into the understanding of their origins and functions. In addition, the study also confirms that loss of energy reserves like polyP metabolism absence in Mollicutes is correlated with bacterial genome reduction. Through this analysis, a clearer picture about the metabolism of energy reserves in bacteria is presented, which could serve as a guide for further theoretical and experimental analyses of bacterial energy metabolism." />
<meta name="og-title" property="og:title" content="Systematic Analysis of Metabolic Pathway Distributions of Bacterial Energy Reserves" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/8/2489" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Previous bioinformatics studies have linked gain or loss of energy reserves with host-pathogen interactions and bacterial virulence based on a comparatively small number of bacterial genomes or proteomes. Thus, understanding the theoretical distribution patterns of energy reserves across bacterial species could provide a shortcut route to look into bacterial lifestyle and physiology. So far, five major energy reserves have been identified in bacteria due to their capacity to support bacterial persistence under nutrient deprivation conditions. These include polyphosphate (polyP), glycogen, wax ester (WE), triacylglycerol (TAG), and polyhydroxyalkanoates (PHAs). Although the enzymes related with metabolism of energy reserves are well understood, there is a lack of systematic investigations into the distribution of bacterial energy reserves from an evolutionary point of view. In this study, we sourced 8282 manually reviewed bacterial reference proteomes and combined a set of hidden Markov sequence models (HMMs) to search homologs of key enzymes related with the metabolism of energy reserves. Our results revealed that specific pathways like trehalose-related glycogen metabolism and enzymes such as wax ester synthase/acyl-CoA:diacylglycerol acyltransferase (WS/DGAT) are mainly restricted within specific types of bacterial groups, which provides evolutionary insights into the understanding of their origins and functions. In addition, the study also confirms that loss of energy reserves like polyP metabolism absence in Mollicutes is correlated with bacterial genome reduction. Through this analysis, a clearer picture about the metabolism of energy reserves in bacteria is presented, which could serve as a guide for further theoretical and experimental analyses of bacterial energy metabolism." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8.cover-source.jpg" />
    <title>Systematic Analysis of Metabolic Pathway Distributions of Bacterial Energy Reserves | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings, {"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"orDVc68tOAU2vj4A6c14vMz3U7jZ-yGqEE2TquGHrcY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"493756","apath":"\/ggg\/9\/8\/2489.atom","pisa":"ggg;9\/8\/2489","ac":{"\/ggg\/9\/8\/2489.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/8\/2489.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/8\/2489"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/8\/2489"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/8\/2489":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022\\\\\\\/8F9TniYEzXMguiicPYCi\\\\\\\/27P5akB3KKp0hYWs2GTg9\\\\\\\/ll2P5VlhP7m7Rl34uOgOMoEHhFxift8dKfGCCrDUronFf+FglMMXdOnBuuaik8X3QkRsMW7qOL9ibJN8V\\\\\\\/83frDNhih9i8fAUHB7GMoafdSQ+P05mEaJNy7Ne+zmjYqNzSak7VEc+uaGZB6YXB9oD1SOnYrSPYERu8x6vOd\\\\\\\/0+FMrNmHQLBtc7c6r0T\\\\\\\/YHL1HiPEffneACybun8Fi+32P4Ug141s\\\\\\\/xFFg6fcKgDlEfjnn0kBZP1IOP3PcRm2p8gKZBnvsH9MUkK0urlO1nNPM1e9Qr+ENhkurQne5fhH35iq93Q3BWiTnvps\\\\\\\/WprJvHYqrJje\\\\\\\/Nfxo8hk+R0pVsIqEpf2F5ZIguGZIF6ygITqt\\\\\\\/lJXmcJ2aBEyPDq5rwbFinaEwkbP2LnUiayyoN7NBcx+SxTm4+X5K3yT\\\\\\\/u3ofeQmMkCpq9a+JOLWroQgUtKFnsRNXJatYjGaI7RKux6cVi0ScKpq6SX\\\\\\\/HWxb\\\\\\\/uDTzxMNHuIV1ytMx8d01+o+i4CbJoZxsU+cPrKPl+v6GRUAWI3vqzDbicoAJSb52Rh2np0dJyPefhDzn1x9IxFRfjngXo8o1JXi+d4oS1UgtL9QufD1Db1Qt9AOjn400yAm2HiyRSRpYr4nGrg6bshc7x3BiLjS53yKa8Z2sFXsbBFKyRt3Rsx1zwnJGYM667TdvAWnpAOcycHuaoKjAQbhvPHKXiIOfb\\\\\\\/AA3AbvSz\\\\\\\/tbIKDeiwxaAdYxJxLb7tKIb3ablc6yaYd7L1RHsVToL70TM2ZZQcWq1uC+Ah7hAB5EhJg6YU25KIKbbwHltrFqEVesnnP3mZ1kqF6bldNiAAJxq\\\\\\\/2XP+fQtiTN1ywAif2Sd0CP+HfNPjWtD8ZZ2ps3oCNOGjIJuuT5Ze9l1R8vb1FVc+TsynLJ1XV7wyZPq7ahsogu5cvd7bf0kHuZeE6bqg4ie4dBtP9LDIBAhTCjwBbJDaBJlPM3KrhlIIBCZg1Lm\\\\\\\/dMuX8t0Dprsb4budp6wYn3RN3WVNVrMfk9v+8gtm2VXVYkWqyJsqjCd0okLcmlD5wW9ID9eG2AsSFWBWkHeW6GM80xN6AmXqIZwXWlunMuQ9u\\\\\\\/asXtOXoH\\\\\\\/8MyMApH1z8gnvUT3HN0Trdx\\\\\\\/Nm2hYmFBp+S+3lfOpGf0GB3X8fztZEiHOpDBLMv2PCAOkENc1B8OAuihbizuZslHvkUvEROTI0awP2I\\\\\\\/YOlRwVUcAqqT3xHjA08jRfkbfgqdIfUJr5r\\\u0022,\\\u0022iv\\\u0022:\\\u0022YHqSI3WwATVCkS3zJQM0tg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b5e3fdff68df02979360cf13dd8d599dd28d1fa7889768a40cb593a082efdbdb\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022pqd8LsXJLfB0\\\\\\\/kyhbWjpz2PR01U1xd+uSXZXpP216uRD2RBaRUNIssK4NSIZoB2\\\\\\\/KQexJZ09JoK4ci54bfwYmHCj5E45xY7uIAkZI3pEZ330W9loxjevQzE5BQj4daOPuRFRaX24nfuuwHQgwVkOtDjVEYD8I1qWudOuypCTv3ixHg6cHaHf\\\\\\\/LQTSuEs790PaLL71FRpIs8dXhkqhX\\\\\\\/CoE771i6jGlrS4AHfW\\\\\\\/nl7cBfqJED9YJ02Mxk3pNefuToBYfSEP29BjstnVgRzxXMKqZL84+eM187SuqBT9WwXfzG62o20dJe8XT77f37q8CCWxy8Jmyirf1tAex+VjVVeosHri9BLL6ZC1XDUbKVcdGcG4WEZlJc0pjfiPFoCJdpQIBTQYTXqclHCmav5rvldYWf62R16DfSYjCgGGcf4OJOoT7umT4YO1ecfvqDFrYvaEkmuWz5f27K1sxqVCYok8\\\\\\\/vuZpjIP0ZnEuCMCGGC6TkpI6qR0pTbxwThCBpEhDo9JTAdVPSTa\\\\\\\/5FBcKlUAj4bxcInxjGrfWilVyVSGkzddaE3io10sISFWpmTQ9WEbRUyJkzH72gbpyKyAun2QpuCC1ejU45b7ZmsXITwKjPruZpAlmX2qAbZmD0bCeORcuXmbUc+5uP48HfFDoMCrw3uFV1Ejwqjx9vSHunvhx6adPrMTlRDk9w6oT0NLiRcaaKUDzDqaxGVDGvDqzGI219SiuyRh3eIf2Z9VUSmqiT72FRnRRV68N4KLH0GtW0aHP+APoHMamYwZzTrjzaPZlYg32I9f\\\\\\\/tG3r+5\\\\\\\/ekiyxJA7XM6gQcrg3sdHEjHap\\\\\\\/QmXJ7xKdcxdE1bAQwEce9CNeaR1dewaR8I96Z\\\\\\\/invQ2jjbLgMw03myH7aml1d0tl1ZUh0TEAExr\\\\\\\/Uy+f2Q\\\\\\\/L89x+9asqdpa8zObs8Mzhd5XNoThYj7E3kc30NbaDUzEYEo6a7tjPRzoV4iArrhmJz+9rgQCHISoKoHmsCNm\\\\\\\/6Uo\\\\\\\/URUZZiustmk0ZVTGyzowwce55kG92an\\\\\\\/N+c1IqKfrx1xVLxiZYKOfiRJeKAR+olKrY3v4UDG1lOUHfkCJ7avJ6fhklr1lttebvdAcnwZKAls\\\\\\\/qrYp5A8NXwmiWx+aq1h\\\\\\\/+dTPRHsPep6xGF8aU1L4lKw2aBOH1DvW4cf8pNM5fi\\\\\\\/KfalPPrmxQ1vHXD8swssAfJxvTBpQuPEfd5iBh2txysRbCV081++m\\\\\\\/doemBgmHRHAgltJwYkpgn2avTdrpXeDkLmgoW4xi\\\\\\\/E0iOPaKrmvBFCGfcLcnYAnw8TrTdXSlOB84ARHWRB1YsuY0I0JcKuNgJmIStojfwaat9xLbIPyLW2K+HZDmWeTobiWKz5JhwZ21AIRLnr6GZs+WP7VWwP4Vpew6zezgMoPacfD+Zt3zKawiqoLKUFNibCX7ZKVXefjnIveD60fgDLxQBMY3yDe4du3esFznRc1aAAD\\\\\\\/wUXO8eABiZ1VX2w9D86+3qnSIWeyntYbkYuWZIGH6Nm0ei6kwjVpYZHfnriEVGrON1O2peBMsvsjKazbxVg+5kQnGrtJxqqMtRn5RZYkZP9xEHoJwrGS7dBoDnYZZi0J0Q+eaBkd4Cf1dqIQj834VFioSge5m4GRWn4bsXRwW9NzHk5b1P8Xp\\\\\\\/OEDl2OZeaG88SuA5ceWlcAvK3L8sApI+TeZzqI6QAXJ3mnVIDg9C2cuIqM08DKLBKe8MRtGtNn6TMejWGXrR5vKKQt0x+2dzfAWLiH2vEy3mVfxaRNEbZp\\\\\\\/jVrocn6NmrqMxjS9Ucd7X2k4pAFKkrusRGjEEB5HfrzK8gN5d5vVpjNFXghrE0vfWSYsZfLJLEPYaGMaqxZ26DPK4Zju3Iv15MpBb0Uvu6R6+AAG2Y5gGXgjEvT7YcTRfHIgLM7Vuzwjk\\\\\\\/+yPk+HM6qXyIC8Ev58nJmwqyygchGsBTBnT\\\\\\\/iIzU87o4JN3yfmGU2aK+LGGwX2dfzbyjss7+7dcyvRvoSMtjmM1mNALTPC8o0dikodXckqWta9Gt0doWWsNpwejFbo9e2+YB5d4wpILvPwPHLeRSPM6BVIXJ1INX1O0OYHTOZvXq9hcd832iylxg6emCSYrgd0yLJOVWfbppyQPtbyeYn81JMpDrNanqUqnL1OwH9D7sQMjLEM3+rONYgxkPce9SyyF9fet9nYFr9UyS5YMrtmLGSj+FtoHp6QxQdKKRgPOQiKDvMwHXyNUEsrI8+9XY0mlTLO5ZPRuwpfRaIWWYL\\\\\\\/tnd1lLX4sh8wti4dhOlKiq\\\\\\\/sxvJHI4o8ZkL37yYd1x1rXBubQN17BqLMGT3eLonVEgmZel+qksWLJConSXB87rIANnM+k4GSPRKwdjuwK3HrfTXZhgLoFg+zD24qDPa9f8koS6KPV6XKseDxGRD8kRgABpyKt+y7KLJdtX3UVeMHHU\\\\\\\/s41Bi6eG46V1w2WLuMeRpm2N85c2FOiTSQc8LXWoOTfOOLdsS63143pfMibgGGnqMYAduAYojlIrajN\\\\\\\/OE5C1ilhAwaPqZ9dGHIrj\\\\\\\/pXSBlMxQJ4PAy2gPQXKxFuhOoz8XO5XwF7d4wSIvcA7CvjYLg+M2CEJ7iNA9kZUtx+4VXMVAa+sef3cnqCJwk6Gmgo0AFcBHoh52TT8Y\\\\\\\/7klGCh\\\\\\\/viEgyBnQSdIRTCOeaWvw6PVFgydVBbhw75EXSVO6MdzFVL5U+MFQ7z6aN9wlhZg2wORQdkGvBDlLLvV9T97iu7kJ+6n+WOWTRYaiQLfBo2XUylpCUNXHyFOVYZtMu1M+aI5PTzk3Wu47rZJnoaZ0DRiwkEwizd+YxJvLF06layesKkJqjsi437OzdWiaMglRJCp2mZ0rt+2AlFzsFF58nPaLY9sCNFo6WbjLkYRs36QZSMAqhvIldvGWQlDLfVYQ\\\\\\\/gr+YDnps7N+pxz1A2+cQAnZtF+QAEpBdkP8PsPL5jmYgMcbfeJQ5U90ivGH3S2FyRskKm4EAuofh+bzJChYwBVioJZvVz5E3aaveWHTvd0ps4oK6i\\\\\\\/SlDUer\\\\\\\/9RHM8LsI9W0opDM+iEGC1ucswyXrn2+ZwI6r74\\\\\\\/3GYWfxwI9KDV8\\\\\\\/l35dWVRZ2JFI\\\\\\\/4E9unLd7W77OhpNdwxOviEihaPYgbQ9Jz\\\\\\\/k+YC2CLTMm2iVz\\\\\\\/mMw7\\\\\\\/N02hrXxKgrCvqgD5AI6HwVPbcX8NmKOlwGq69UUPw5l2Z3NWxk0HmE53yfrg97XR4EA9w8sii4cHPWGyttvt\\\\\\\/eUtOILxLljprC2SVzpybcpAFfhl1ShN4A6JZWUWeo7dAnHLWovd1QjaH+rkhvpr1alKDUFads8rdkmrNOqzAMzaoTqIanNkIN4+QrN3UsG4XGRYiD0QwnWw+6eVISUQqbP3ecrENNPpK0q7pyLuJf3UdwpJlODnLu6f6ZY64f4BFGtyb4TnlPIQgDc1EUkdNtRcQuirM1Rj4iG05eZ5BPtmVuyUyDUnEfuhY3LLhjEi8ZIbUTj91BYDJedlekiMG8\\\\\\\/\\\\\\\/kEiZ+44GHC2ZnRke8aic455tggCAejDNGJcWFbXkbnm45pNHVEgd+YrrgZDqkVEo1Qo\\\\\\\/4VxLppulOuk5RNcFhL32l6wXGGFh\\\\\\\/QnJUka752VtbUjn9ME096Hq96G0ZPsfy4+RYdLDauQ5r2VRDrXeiJd5GZhLxdmrPKKccVnJi75l0KiTAZa5cPXYTeMPLgQlf2SbO5p8A87i\\\\\\\/k7tOSDS4DrGVKn6or4Ziu7QZWve\\\\\\\/lbJZNTlXAYUpmjSveobI2JbFh59T28FJcVb3ZIR53xs5bb+cJzvafMHazmp1urxyP5ABMA6i09Hbdy9OLfrM6RsbDw9YnJLwBacud5s5c2tw73PcCpFkjXqF+Gg5v3VpCf5v5EK\\\\\\\/9rFRG14vE\\\\\\\/GkxaKXGI70RlmBIs4DWoJCGsDwo2kXkPO2bNfe7RAH\\\\\\\/\\\\\\\/8wxGpETkitGMZ367m3EoTC0+KC1p2MeUc9YlcPOTHlgV5MpBTCHxaq3e\\\\\\\/uaaCh830OjJ0k\\\\\\\/\\\\\\\/fNVMExBIhBAszr06BByxdMOu+YObEFmX\\\\\\\/OgGetrOwwYscXQujiavRRA4rVWwtSsg22JnXbyUNnTJ7BRNmg7VZACnaepXTALWJ2rGxyjKej\\\\\\\/cO2wc+H1xVKCRv9FjA9czR94+piD\\\\\\\/Y+WfWp1TC5m4zRSmd0eV+fBklV9M2P4fGHu3A6wjf+hXz\\\\\\\/LRU3odtDpFTS4czc1sVczpvHYNwY\\\\\\\/1ust5x9aZ3yL+NUjSXYa5Qmm9gp+U2D8INZMRhSAhyFJbULQo42jkIvxzyGJj\\\\\\\/a8gEeyKvYog8eOJSxlzANaycAkOHnJWv5Tc+4RbCLGR0ODI\\\\\\\/lZXfMM75R9lPMqWaSyL\\\\\\\/vw3Oak4FTX314NA+4VjBVyZ\\\\\\\/cv71EgvqjVb82XjprwQMkrM7ei3r50eeF6MpIZ4LHtBOERc9x3sb1lDEKkrpy6RXHtrvfGBZeHmy8BOhhhQXxWNH90KS4SrRMFVHowF3vCfM5R2mXwS8\\\\\\\/oT6rx30WeBN+A3D8EutvOL\\\\\\\/QzfJl85FkRLZkfMlRh96y4DwX7hurz1bDQ8HOfEBcuISL9kanKy4bKqK7a2DUv8wLzITaULckcqT7f\\\\\\\/3xKL6r\\\\\\\/d00aiblmU7JeJfKQG4qEvEyoyezQ6C\\\\\\\/p0XEwgj93WjMVEWTUF7foKXsVod0q8qa\\\\\\\/NHmtwF23wpREjRJK2dFrs5VpPB9QZvnWpU+aHX\\\\\\\/\\\\\\\/roxrkOU\\\\\\\/yDbOl8yRXDMK7fTg9MsXGhEsuw\\\\\\\/vKP3fMFMshV5qK\\\\\\\/MdBjMR4j+emdtVlvCPBoPPsZeC89lC8nfkEhFNvzELAVt0SRCsFMyYKfZdBhzl2Bj2fIHHsecMXNZYmC77PUFhSVlv9n8A7Pv3\\\\\\\/RCF8vkzXwPpROja9LJrilOvBF0f1pnjhcNGbyAr+31OHbBj5+XPgu0tWBVfli4DILZid9HjxJ6Hm0TihK18Bxn6z8bFOSZaix09Ey7AP4JqEzAnlz98IzCYrpSOZMd\\\\\\\/2foeR5qJeEJWEJcLXzf1HV6UNcSfOGfHQmexCUWyWhI7maWrZIuAwr9eNlih9+zcNB+qEmq0NmFD3MopQ5qM0b4okvVFbdxDE4NO\\\\\\\/98Kitx5pxGfK7pM5sMCkdx9CjwDwTAMGq2WeVpq4cYkKmUN4jYIW5dHQnWJkkte+ykU8reEq4\\\\\\\/5XOqx6A4MedkAf2ZbE+uNppliCzN4gQbrI5JRuzMX9JSxB96hAeHM1EIGaWjskNghbvdGHGarLn9cNtrtQzKgv8tI735RrS+h+e932vkoDNTb\\\\\\\/yN6hOfUSY0iA1CHRt1Z06EBrXzr38qjlnpwkApXKpkUrooZ5UDt1ESZy5OBWZjLw8D+ooenZU7PCtXQBwcobbr5SYjDMSGRke3y0QZdrQX8DuR1bpAUkf4Ot1UTaEL8fzfqYD0PMt\\\\\\\/JTdpFitF75afeP2f4qTudfw8mCnsI3iahA2ZyQAxrdAwTaKbaF2H6bQKDgzBi5JQQoWLn5649UOtc+7nWr96VdYFT6F0akAnWLquPvTVBkEIfezSB+AUMhK9zeAh7kAwTMnEMR0ZLr4NwkVQlnTm9jf5miacRHkR0iZ05Gde0qtFI9rmCE5ByjUD4lLHIyXGR8NOthUJC6aMNTM8O0EYeUtq7PAdE1vLSNAdj4pZr+yEORInbrd0\\\\\\\/1nazV2aaXYxGtNwkcUjiKowZ3rrEz9tpwVamvuQ6RZsMf99jLLGA7YPkoH5KE7Kw0XNo7Kzdl4E1joZNyHFARYNQb+vog72ZMqcJWwXL00tepnTbHdT0QHmbpaqs4fMvrmN2CkOaEDTAfhz9mLC01+plX1u3xR\\\\\\\/gVyr5QnhJYWgqkQG5y7oHBKDB8443v73ep1rDele9yZQE8KOizF8LjKML9bUA40O9t45LfLo8wYjs9ctGBuNDyyZJWQqiFosjVIxODPcznFRtE4dTXnlJ+TqTj2qIlxbRTn2tI7BTXYMmpn4X+05fBAh5KOYI2aYGNYPAbZ03MJAQ8DuKbGJX8SzXeGMlstTW9cYRkSgEU8jaImd+zkY3IssGHzi4CoyKDNr0iq9V1TFrZXCPt+WkhfZdRM+f3IByIObmeqEF+GqsnzHxq2SeuAVuvrZ7dnC3KqxEGeIisyIVnYLcqevlFpyz9KkXHRuLUSr2W6EPo0yg+eFUTyVsFDL1YoMtKlD24X9XXh2YG\\\\\\\/kvEj7jW2dvoA0TPGaxGfFKNmKTkbPgGBa2qjWomcPskl4c\\\\\\\/0VCzYnG2KBsP45nhr8GlSqI4PZpoV1+GCA9GNFObitDcaH9Z1CsqMlcIjqmcDsDagPsn3aw8Dl7u1HGjKucSEQdP8BRNmC\\\\\\\/oWdu7q2S32\\\\\\\/BuhGyHzDpr0kATjQwWqtJ4VFV8oj7jZI9st0N\\\\\\\/RcLFLrwAb8Nmqe+rWaMfpqXuyDxAX5cSiNsPTfoaU0BHeO6u1z+zOPJvDihihFUotkZK7Lvee9GF9A\\\\\\\/61UkznwSTAEiqrBf85uBaM7ckNShlG1fWasm9S0kBfX28sCbJRdTY9W8WVsPx46ug4O\\\\\\\/DUFkGPoopAkK+56o1C7KmHha464p15o63W9rUbtydk2nBwJwDE\\\\\\\/2+P5XbnFSpjHZ71736V9mpdOqdpTQnFb5l1bAHNVJAqDRLYudeKVDL7uKNoYJ7F\\\\\\\/edJGXVfuJRZLlXKpYUGOeQtFhIwgI1bIM+bUag5SnrqRnAoXuKXpUOzCcQ+M8+oK0YWmVUjXn6qCrIRGbOUmXeXCyFW7b0f89De251TO+4EvbVPYfOLs4hr5WKMjO0wtXr+aziZkt8K5K1AWVH9LEuODtKwvCXbQ005EUQid6tCmQz6aIGprZhsnZqT99dmiuHfhh3L7zyIQ+fW\\\\\\\/UhhNn9cCZP1KPAkBR5igwa6lbcGED9+jWjp8DcwDvSVf4fjWgxQRzsu6j7pHIbPBW6iuiklngPj1BXLmaBRCwmH2++0LtnSp6f5Q0eU9xJXdXYH7gJz7lwBW1p97LA8CBDS6vlASVfOpB4CutspLRc48jitswklBo7897NhoHq05PCjA8POKdVNXmCfkMjL\\\\\\\/AZvPKuvSGuGOT59beDIgB0DcZyT8qcCE45L3lI5hDtZNpX4q90uNte5KJG05o3OHRSHb2J3MvnI426GUS6alsaeNo6ukPl6ZnnnVCnJxGR1yg1RWoKSzCE4rjK5dp\\\\\\\/eBMeWl3cAyFQWJpO6A5iUtmT+b1YvNi+XiLOVLOcqYW5+kIuQehKKZGGJ2Ykf2yCoczBAXvUee+FlA5YmmirV9OYZdl+dLOLAPcZwYBDseZeJajtVIbNDdxlHuUV2lzDZaufPlm68C0+fwBkQqT6HcJB5ObvtcAiMtY+SlAHJza99o1ew3LocXTRH4k3s1JJeyNTsCyIMi2s\\\\\\\/zf0fqOfHoL0r8F49SQDIsS4pzKybnm9CKsu9EQbwWYfdqjzoFcnByb2r1higrnGpLJr8dADpOY6IoygUJXJA3WMyEC+RfWNt9YI0TtXkRn37fUFMjqKSwqmvtDfZuSoLXFl3n5ZQHZbT\\\\\\\/uRBwH5JgggyMUhbrf1KSw\\\\\\\/ny2qhj35uw+dPCb7LpZAr7GxALZbQVK6QMY5zoRX+hwAJQzicLi+sxmc8baQYDw8GZtg\\\\\\\/t5c4t1CHXL3jFwQycm5IbD+QykRhDVPclwSfGOCVoiROKdwlULyd4M+8E86D3yIqDtdf6TZ1bDvqWf3vdeKa9WKmPtH\\\\\\\/S6MVnu1jZ4MZN+yv\\\\\\\/zS\\\\\\\/padQZ5IZJF0d0c6lbbnyX34h+LK629AGN2d\\\\\\\/wCWAJhRQTtmYriQjX8ZbI6AD3ATVFEFzCpzUP50zSyiuAMtRoGu1V1UxznLRkN90azTHT4vtGQtpEch+attmrSlmChKPsUroTnCR4+dSe7N+DLT4la\\\\\\\/z09I3KkQAvLukyouBNZaOIzlEoFCg3EYseSZxQCiZQLhCVNci2EugjjfC4gDOPVeP78nwkyvEwiYzcKAMNtPPRKkBoQSQiw3En2dROqvoZL4jnfVm9jbRoWCxCWoxAM4G8\\\\\\\/HL8ri9GMeSiaH7zNNKLtvQwRG8Hn55I+k1yzDjuPciPBh2y0gx8jdLDzGWAH2vHMKDyig\\\\\\\/JQ1KiiCkSCdIflIFNJFhXYbifu0JwpD1R75xwYQ8p5R5cDloTQGbaB64+45QptD\\\\\\\/HqAKh5yjdqWY3qbVA8OCqgPqGH4JR3AW4f1UY9yfrvnE9XNPoeI+IELE2df4wSstVtjJzkDpprYV94mNkHodiWGufWR5Kw4IK\\\\\\\/ys0lhtKDqmsvpLPiHuddddx1cVTZK4k1Xcnyku7n4bMQw7JJyMu8rG\\\\\\\/VBUsPr8TRj\\\\\\\/9MCosoTVOTrF+lg9A0QQkeSEt5xP3qlAAF4CGTl4IwmS1J2jDZmvz2nl5A8DidOGPY8+2EpnfTJr0Vkx3fmBtLBhm3k0vFLcAFnRI2LHAe34FuADf4CmANDN06yBCBYg1idCy91SdFcJGeWECXgRfAc9b8x+QVrqIWrVYvRD09MONEq71xM5txi2RCJlFZ41HXX4D7iAi5TqCTKY4k3dT5fSgZDWiMpNbssBvEwGYD09CtX1wTh4dditCwrgzhtiY2eABETfvD9IciPc1Zuz4ttlOIQyS+AgFDlPPQysiHsMvbZqqI44jyFYPu0Ha+y+cFyeG+nQKV1cIsLYvFcL6xeY2CracAJUCP\\\\\\\/oSCnvRM3IanaLv3AExzzJ9xvCidSeMXcdt5s0kqq\\\\\\\/GggPGqhII0Ez7k4NZjmG7s8bKH+D7NuMM9z1TG9M53lh9S6xpMLh1nlbgrVLqTjEdOh8yfs0Jwcyv7PDbrGxky4IHj9cVR8M17JmBgxB6XsUfEq2ynBAYHEFwtT4UtALcbF3vsqGNarLmo0KLF5YCcaCbZZJh9rBRNblynoeVvu+bNV3I0NlGnXhEsidAULWjD9Z3VdQb\\\\\\\/IUMV21XUv4PIx4tq428kVh4genelUYCLIGzm8aN9gEF0BscMW3CXrFk4q0ls8YYpHAahDSyoEESv5v6KkdTgSC1Zc0+n\\\\\\\/UUTLff1w6U9TcPNTmsKhUKVQHikUz8TQDZRL0ttQdqWxfBc1H8i4TAZeKJALazLU94lKh2P\\\\\\\/cWQJEFynRu1N3yhEJUvEEcviQl4BywCLBszN9nHUSIIbeQ2byVlXHMl6S2dpLCd6HeosRymuc9K9H3q0DQTMXPLpSPZY9kKPP\\\\\\\/k3FZzewtYbMmwRXQ9gS4nS7qS60vhl2xc0S+cK1lmCkKUwD4vMC1ZO15xo5wJmzXZ9tVQ4YamlkWCH9J26dixExIIvHdke8BikclQlRH0WP\\\\\\\/oj3eDhBgbNB7Qgdka6zbxnQKRD1UYm0y\\\\\\\/gGmhz72L7F0yZKsDRtJhE6M4xkz8u778ZdhXm6bZO1blzFgk1FivVOsvkeKOgF4khSfJ0IyQls6Jd6qg6aFbFuqtQUZYTFCRvwIEAzdeLKQlsHsD5FfxM2bWx2o3kIg9HH9e4kOt3oaPdZqq027tujGxpdlc+NbdP5cFMaG8DCHbAQibVMAVuvEJ1pzuMq7Sne3PWtsCC8heMIdeCKCES1WglGPhH0pJmFVbFgpvelNoTefG9ms8Mbip7AW6f+RBMf2bl+XnOCAEkbM0kcxHWO\\\\\\\/i9psViWtjAtyhdgc6pV+zYNxN6Y8Bqo+0n79SojXUoYKTIdWqG+3Lh5oIatRf5K5YaOxV9e6M4qgKrXa4ZAlj00tzYoNIuquDcrQ0+DpIfrBXgdlJS1jfbWG45zGt7o6nrH+mnO7yZpRpi3qXkATts0U\\\\\\\/R259ENHht\\\\\\\/TdfKJ0Kes3MzeNWgGNbQMkj76Yej0aymMEn0WmNvAuRzjJPFx8hH8pYJ0tl6hXsxNEvpwPfkl\\\\\\\/QelxvxQ4zdrOT01eNFGBAl+hDMsIOn7Ox4kZxJKWprBeStPdHjOAuu5bLcXZT0oVTzvCo2X44iAC8osUUcqtXDLEfapDFBDzANGEXqgaxtkUYtPm5RzPcggeM\\\\\\\/uFNNkPkgTuy6mnltyxCNIzvW3uagHiV0bRhAADN7uC0McfOuc7BibTEfQslUrxT5vmor7oBCxKuhcCrcAp0OxvpKBv4VSMRkIP9JChMRmb3c8hi2VE0jVkmY1FesXqXUD5MHHujGFQI+hEP9tq5G8Il8ztqny5L25HWveIK78UbU+AOevorbr18eZ9pDV3maoQg9VoLmrtbeln9s5B56iEELiAIzZFH+diW531A3bWjl7X0iXxSgxMt+o\\\\\\\/84uWY8\\\\\\\/MEpUpK5PbnUXzacSQksTn82R9CNGfBZJWzNGibZY2+LhHlZFXo3OImJWVjonEJCV3KHr3hiDaNGKZvlVjbqUKwtbpN1Ht2xUBaWxRutUmfDGUiOpAfYImBgFfnPgMrZHbzEnHiBV7ZKuZDLpYJw7qZUgBSdXf4Hh5TiuEJnPgPkV0v5X+iiYYXUtJZEfuDS9e+YOitZ9bAC2ATzb4W3Wfy46MenOh7F+b4\\\\\\\/oG1AszDP97bSsE0srmlUz6WFAjX1upB18bcASe6KKCDu6nH9o6i6oQgjZ1VX1f5ZYg5+kutIOACP1N30vKfhEwivmepGqHbDHHh208KW9w0XGhOua66aGki10W\\\\\\\/bsCmwiuj804OXdCEKviFaG9a4eMNFcREN1NH0rb8pcMKMxpE9d06FOCxc13p85ZrjgbsafTjXx1uD\\\\\\\/WX0MwRIz0e4r7Z0kchLqLVR0xo+1bqHJNNBg9h+74qjiyNBhEeT9Ic8aO1gcpG2C1v9c99IVEq8vi7KzAARKU3ulYMTzL8DOhpW+oVpwd9gUP+ha6CaYV5Xg265LBXyQccnbn43P4C79Vna31AZlmI6rMhVNqpAxp7dNy5XCOWstyrkwC0nCM7akFFHFLHg2lgXGdmY8Ec3Ugi4arFDFmVsqyhCWVezyv\\\\\\\/fJGxql88YVwskKpM2TUaPrkdC+bDZtOAzMPsXZDoawvZiWYn45wKzwT+ygn1GjsJ7w98as21Fw2QfqJCiDYkcETJHCwJRterqLjaKqAHW3NJbM\\\\\\\/kjZ7F+xnMVMvQfyH7OVSNkys8MA3ZeXwDKnqGPAFuFEcyc5bFnRM7jQdmcl+w0ZEaosZIW9Iq24xJynk94lJnbqdGhbvMc+a+FXTypUCxRkNQZk22oRXkrrXInX0ySQv6YWV63py21uDoNdsBSLxCIqd+85LzTnsjjTrHHsNydAq1M89QGhSiMtj7OjRWVADaNl1mDwcLOmn6iIjxRfci6egYhA1xnE8LjHRErR\\\\\\\/dnbCtNW1RS61DMMx9+C3Z3aYvyINFR\\\\\\\/f2QcDPEGSM2a+\\\\\\\/YTiEj3F0amKTo5+Q9YI6JDHACQ8KQHHOxfyPzm0pZpogiGvT4bvjU\\\\\\\/oelSjMFqcPDZFjnOp8noeA9mUUiwtd3152MXRYe1Ls8ZnXWjqLwy6Iie7n6OEUf7SUoPnnKdhJJRIPCLWkIM39hZqpoDk+o19pRKf+HqA9E+u2h2VrFRSxMPgWqdRSaM8qkIZsT2qbO\\\\\\\/zjzMFEqN+ux9PcK6beC23FuRduBCfUl9jsVE+M8kc1ettq1kLF0gLNid22UnlazIFrRHV93RGkZhXMzlaM\\\\\\\/zFtkRmTA2+5qAhkmcpdl\\\\\\\/mirBMW\\\\\\\/SmgOarYe7U7moVY\\\\\\\/MbIGCBSThZncNydJ7p5BhUXjKdB6RvMgBwApk9plj51cPxgMcnZo5BJWdzvVVIOEFQCb7ZVX1CkBWWvq3ym9gXUs\\\\\\\/EUMKbAjstr77aV5+RCYw4DGYaGC82I1\\\\\\\/xJUAdwq+mJdv4xZo009z3zx4MtXq1ir4HJOHRFgitBb\\\\\\\/0Jw9gGCisKXcp3sWNBs8bnx1gVdJm6aO04GNj3YiQ4dgeYxAFqdbExqvrqKnSmA6QTzDIke889LohwZodYD3619wwWWbVWz7kyRiOYomOqsJzv\\\\\\\/ep87YplBgDeSomdiEzEvObOtudHybsS3GOTlQ+GPVXp\\\\\\\/LZHZOKdrVWOjya4aMdc4rvLp1hublUU5l5B1uZnXyicU4v6Kt\\\\\\\/Q1wix3AlOmF8DhVSDRwnUax11jappMAPDkyale+HO6w9ms2GC5qUecx3MUSO0Z769j6LH7feMb9jFxyjB6TYtsUqKqnvU9sEAoav5MgVVl\\\\\\\/QpDhGoYpf\\\\\\\/Kk67ZdVmOrCxxbqkF6lFxcF49xCPzcbNc92pXBsYHBqBdeQcr7tKyT3vJDPQJUc3jFl9rGcGOBnx9sdlRRJ1opsiCVJZBQaV8YFgfcGTHB6QR9P1oL\\\\\\\/oQEzEw+LRecTCXzoA5tUNc5YBpLQyEJo5TI4fq50\\\\\\\/WQSeirk+ow+urr\\\\\\\/xda0smOpNBF1TUcYQ9j21HMtDaMqxDEwpGv92AiI8KcHLVUXjnyYs9mAgVIFoxo+a0U1bYOFCnhcIlzohiY33GnHQaZwsF2Cj5bJr1j6eBFP\\\\\\\/b4dopAGtUAnBDhl5vAW367qQE1lezOAXSVRstP66E156uvJkXFCkKnXFatAnveGvnuB9hxdVboiMRHPVbGIkiWhSBdDtvRZV40BTg3Yoy4zxvPKIMxAn\\\\\\\/UqxHiu2U9UFr2qpJYzctsZs7cTbnZjbHZxLyNsF79YXqEQlFv56ZQ6JljwaiT6Nv0iGxLegi400CVj8c8amt5qgd098m7Yu4uIX6UrRtFeZwe9EWU8gGy7GZ5AyMruHjKra3YMel8JB1xERUov\\\\\\\/V2+kglZNUf9qFDClk4QISKov7iUMU0BR5CLxhHGVkcMS3OdWaNIF2eCFcmVZcZJCqZkvdj9O+OqfZZ5QezTD+QNvxIeuUO9yMKRa4\\\\\\\/ErIUhYnWeCmoQvmzv9B7peGBiV4I+vMTm9\\\\\\\/zELl6uP\\\\\\\/HBR+PnA\\\\\\\/uL9e4PGeJ\\\\\\\/AzXnAqJMYcY3vweksmq+z7DqhGSQ9QQLFUDPGW50h5nnZPDDO0LqujngweSckTfbVynw9Ya43NxvsEEYaPeuD\\\\\\\/U2\\\\\\\/Os18HYwUp77hrRzvp\\\\\\\/bTcdspfVkn7sDp+l7vpUIJoqKyKzcPHvhiVjosqlUTxbWsau0rX0xONGqUz6cv5D8g5qrea8lEDOq2OVTS63yLtXdU3u91qpeU5eOYMjPybYXRCvnTo2ClWJ+SwNRlR7bxdIZWcG\\\\\\\/PPAUrc2ak1yhcSTCUdaq7RMqLv8rQuH3nxDQBsbpxAB4WoFpXGFI+Dn5Dz5sFg4Sm7rFvRsrBUZQccAdevLez09eK6QZkd+YQ\\\\\\\/PK2t1A0WLk9m8KliwqoVHj0MEPfBw5RclZhzGgeN8uDm7Hsc661hkTWjHdR24onogSGWgYAEVdlqZ\\\\\\\/7glrO5V+0OO2ePs0GqsnVrvzjf89qn13RbXMNe6RGkd\\\\\\\/z3PYDfO0fGIq\\\\\\\/9g7bIXPPPJWfO3lew++FCXJDZJjoTqV6PF+dColrXBrPvlvPxpf\\\\\\\/atK34BhqNn3JcDgFOIhwYZ6Bvsa\\\\\\\/ImFAgPAIvCYyLUJaNjYHpbGEPayeTigOi69O5UEtfF3AeBCYnBnO0d6cVw0Da8376joWfIgF+7Uh\\\\\\\/5u+agVTwSVPT4Y+fCkmc6ls5XKXlenso4i1ufZS35GaxqOQaiPeZExEw3Z98Y72GNv+84nXwPBrhXJvVLY0+PWh+72kbTFxzVOn5MYf56PXTuvlYz6zYTxm8D4+BvCDc567CRz0a+QwpM0g+nhpFdsZ6V+RhougsIWEqSjZhvwFAhGI\\\\\\\/JhcaxpGcOvt6X6ueav8+D7DkwmyFLZyeobMkNLwJw\\\\\\\/9P+pUX+xrNCx2k9+k0yyT4AuCY40H4tIkYiQ49osrJUbYIGiTwk8A5jb\\\\\\\/UBjK2u1T5+eGQSbYF0\\\\\\\/tguo1pDjXDakhtonByPcHmedvurmq89FX3LbPP6KwhwRJgrNd0Q616XiFU9DEp0yerOXECri6hGN3qVJZZWzAsbHVpoUZNWEDzMW2i4K2pZiEd9QmIrOcDBbWuHM200VfpgUNgT5snfsvSHBg7GSJx\\\\\\\/w+ydiD\\\\\\\/fYy49nYRJCf1M8GcMxo2o5ab7W8WQuItcFlTpx15wMqw4OxJ2uyq+xNbKSbEnfyP\\\\\\\/lRkPIopmoA2iLHNOSqHLETRYNPAR+l2b8I27tIs+T6dvVVTXoHCPp3ODV08otywuhCZvIhyGrvPcXAS\\\\\\\/3UHVgNlh2XRyfL7caYhsEN0NZCu1R45q+1rgska\\\\\\\/7rawnEyUucSAmEN3\\\\\\\/cp8lUUQBjk0oro0DYuOtX0Pt8j0anApP4CDR3O9sGxBT0KsFnNRPbC35Srn4+65hP05EtxS+YedrOkypQV8Wjm4zcXAyj5jYagvGDqMman4W6jhi6Bwi5Z6j9bkxQx+zR\\\\\\\/n0r5qKNFa8hbKAZ2j0tynQWHlekhDKSZfNjXWqNdXRXIP7VIbSBkBvmQbakDLZq4yPI92x7unL7B+d6y1lSlq8Z+haPYMMJhzyF0kdo8E3q3B7P6FuREOP339jRlLorX7aae8DG1bVVYW\\\\\\\/IpEXsAPkfweVBEU3n3qJz5efZyvnV88gglhgs2bWBXr2yc7QWfuJNw9P0JbHdADBoE+F8Qm3grF0MXwFmHFKZn5ggM7kpGLyoCJXlRBxFjo4hid5B2CMYBOUbinG9hY2bUbMNt2AMSAEgtiMi\\\\\\\/nfk0ZbqBeLsQTycNFLol5XD+aIEMEU3Z7ZnsJqjEQ2aA5um4CW5786BBsxa+twA5vGVPo8XeLAq4xwu2S198xhhIKPfZkzpa0d358VQ9+ENDN+ZyZL7O4q0peIil9\\\\\\\/vK+RQE+nQs1ghw1EsdM6tngAxNO\\\\\\\/m9nhxbU\\\\\\\/p1HrW+fr2MhoPa+1fhpyzvyf3JQ2SeN+ysPj7EMYheXPnmF5ycNKfy\\\\\\\/oLPWL5KgfBaCsHFIZtnQDN1G2dQVCS6bgXOYNVg7REh9U3\\\\\\\/qhosTtyX\\\\\\\/Y+1g95LTn3g9I4EjWb+WM\\\\\\\/HdPy\\\\\\\/hyCsWMQWM9\\\\\\\/DRK04\\\\\\\/KvPXYFsZX4LMYWadYr+sI1xy5QK4kGtviCMwpkSl\\\\\\\/NBiE\\\\\\\/dppOwwD8YYJW2YpwA72Vl85hiGgSWPJj3pCG7APgjGZVuFeOhFoDA2l+vKalOABIWuTYHOKZPP36e87n9ZMJ7rvla8SJAE3su9qXNMrxQR2iDZLOBF0nB8mQelYw1ccv8Caj4Ro1+MEQX0NvOKZxcsOnKVK3nvd9I4F4Bs4uhPlwDHObCVtZQH64T8QgU4vnNqwJI5u1qNoCiqnTlZXnCu1C4eb0N2gag9Ajy\\\\\\\/RqAf80qAX+sd3+I\\\\\\\/CDaFaD\\\\\\\/d8y86xpAuBabChQiTRXbefbfiOZnjVki+wigl28bsSleVvQQZJqyGa8wenQqE7fjF17aeyCIJj3vIaII1vLY4OZ3mAP20lql2zOZzm318Ysgg\\\\\\\/u1m8WCxp+YoWphGg157k19LsIfCFjTQ0QkCBXxGJ\\\\\\\/q4abBMLfzcqNt1ObIuPsvyMDfvsI+ep\\\\\\\/osQuvPPFPvt43ZebSJimkfdIp1Paccog4qy9iC+0iu2UtAqh2nxJLNjrjUGmv8LvMI3e7wjHYyJyq1+KvkSPAS5bVEt5UYKQMDHnXuFscwLtozGkI7xqYzumHp7v9eAl5qZEo2GrRkWrX9XwFVQnyxIMnVY07f6knKMUdF0fhmIQbp2CWzl\\\\\\\/h6IhyoAu\\\\\\\/XlWGz73Va0P6EOZWekWUvkd5HBBvg7uGdrJeNaWLehXLsO1Ev6IGKIFz2Lg+m8DBdrdPJW6VbP5f3K4exmOZj4GKvZjiv2ks+I+0sABYNRwm3pU4piPX4LyvyJFtMTjsSdWWBdTVhUVG29V8JNmCEF0r7Y1oZHw3AOnYHmzOuWwaLeJ4h09T7RN25o\\\\\\\/+K8vmxoamuBoGCidu1kh+ojGuvmTWVML46wa8NwgbuSDmfnBt1V3R7MPsAB4vtqg91VodjVg7Weevr7Lmh6Z92H1b3ZwGUe5TaydSapnh2WDVDrsBIRnEkw9tji7Lasv2eSJT\\\\\\\/xjSdEFps7jGvQTkmiQycHMBXWxelUST0B48tM0v3QE6tou5PCEHckLlTEITUR9w\\\\\\\/A5S6VQ3PvnElt3LcsYON433t7rJLY39j4hKoLjSw9tCwTLhpyuQKVE6aaqI2xOG9TS+\\\\\\\/jyHCrpCA5nGVes5FfrhF9UyvPaQdJgdCKCF4rwjkmWyP3l2uKEHeLmrDTfvgeRnZY4FBqqXtwbp1qRnBIBaoHUDLAYL4d8UBTEEt+ro0yXtU3Wfq9Nbnae+g8J0pulbjFrRaIKUHHtjDrY72SBwB82AoFzKYSjAK\\\\\\\/qWsfeg0x6aPGIxXbgxcGzhr6mVX3RJ8rhbOlwbBfe5S42iL\\\\\\\/J5M5hhjt4XcOG25ZPW0OIWYCgC+825suEGEFYvvhAjnFuSuvBIbdmon43twp4er9\\\\\\\/AR+oSoU1SoiPKj\\\\\\\/evW17EPv5lFH7wPFkDR\\\\\\\/xbX+RfQL4TCPfR7vxhW7uBTdv450\\\\\\\/exlQA8j5CWTYerLaO+2a4QlDCDyHi\\\\\\\/5VCFToEhw5DPST1gh3AEWh8b55Twwd4fnAUVif+fvEo\\\\\\\/Rs7o9oYGoVdX3ZKrSUQcbctE1+uuL4Kpy7KOBI0xdx5pBNQOAcTlcdKRH+XfcWTcOlX9XyzIICCDom\\\\\\\/OpTW7O2v70eU+6B84sRTnwAHEqXxP8lk6GC\\\\\\\/hX36IVdigKEmDFju9JBPzY\\\\\\\/qqj2RGEGGYI1sG69HpYTspajxOkawvs9RWgCEcNevW6ODg7g5tolCcL8zEbEdYk6\\\\\\\/V3cZaN07GDTKZF5LfHBIe+Y0lu3cLGkaYV3ykAFst0DP907SJAKbXK42i8gV8rradY94L4uOaK9\\\\\\\/QhsWGp8ru7izedJ69bdyND9OwhNetREbZaDVfuIXehy+TyaIrwwKPZmCBrOvnpg7j\\\\\\\/BEPlseipeCUitIMOPsDDcUyislRsQBUBN\\\\\\\/r2vL0zN0ga2JafEdwRgXohQB7SqNfG97RcrD4Wuus6nwTGew+h\\\\\\\/F53FbDY6bYnToooNpk2weznNkLdd2JAf4oiVyDNypxkZAcZB2kl9Uubf72CMuW8QrzMu+Tb8KlsSZMNyZ09rnNnEnkRk\\\\\\\/iSFwiKo\\\\\\\/IxQyvaHRvaT48YLvDPhxoEgOGbgYBYPem717beCjTKFynFN8yDFP76jjY4oKCLbzpMvVXiX9WSqz3Sg3u7+vi2H0dPjdYB0wJciU3P3i2gG\\\\\\\/bDIp\\\\\\\/uaU9BLnrFWCdnyW5SOWP+t5BYxMXusq7YDfILXS\\\\\\\/LdLeBVpgRMnt1\\\\\\\/jvh7QmN64axa1MaimRoFbwA3tSTEAnUZQUDZsh364EQYvwbu1SF\\\\\\\/Mmg6nugcCZ+47fKuXjjjyGsr+IHyfo0aOy8jK5a\\\\\\\/rAJqaCPSLqo8sshZs6AMgOzzr7KNuXHZLUjhWTg1vPMhEuHz16NwN4sdqVZd1OnqQWMbeatM47OSrU5+\\\\\\\/3NLpYWK2QFkhyxBCylYpc9OcA30zi0gJuF3VbJ7TLIkv2E6zM\\\\\\\/YB6KAU83uOQmyOvRDwA8xOYida8LFDK\\\\\\\/ludPlKQPZV+KWGKqpRGFuSO3WDXEYb2vyfo1TSySeg44Afv7HbPbGW5EQYLkCnriIx\\\\\\\/7MArxzSceBOML1M0svGf1js+LVGabBmR4q\\\\\\\/dbBHBTElCZ2HuSuNahhHdBLIARCHATPNDMCyUjtwePWXZ48Qor7kZnyxjHP2owfIjK7qUIGYcvudcgwOno0UknVkrvQEdknjBGIP+xMVduY1SD6t6yBhDjpxEr3aHAsqMDbEk3Yt6Z5VrDZlKB8gFL0HZPGbmffxX6o00d+4zFOifzUFv4H+RhatSxU0sSzm2Z3u2iomtIgVWy5u33dZld9kJ4h1gvH\\\\\\\/uUmjLUjkv0y2tP39PynClDJ3NJ5JL2S9O8cS0+3MWx8S0Q4wiGYQ\\\\\\\/ejD16qM5bvbav5AdfoRcVRJZREl+lh7K9bsDuvf3T1t8Azwtuz\\\\\\\/M+NpGHQ2T1Tfqhtfzpv2jCoPt7IyLwKf6VzlY\\\\\\\/hDaR9zqct97Md1UlK+YmH9yrsd7wvU00kEh6UZVhG8nbQVMS5iqB3sMekxylQJ\\\\\\\/q1Pdi9OR1uDBHLHtUnH9X4+OEePEA17\\\\\\\/qvsNMO+9pRtJRBiru6rK+HNvZR+zmsjKafjiR0+2wX4U3ylERD2esicsxkBoRbkv0DD\\\\\\\/CPgRcx\\\\\\\/xamZsKSyT1giLRg\\\\\\\/O8zI6eVbZIzh03iv6rzswRLWu6MLkATXdG6\\\\\\\/Pb0RJWGTz9MQ1LX3CbZNDKB7rGGy+DVb+dClTtgO39foWBJXOy0iFFlsmhQxAodQOR2rDwBJsfM6vRYbuOQSC\\\\\\\/AJRIjFwhDL8rQVKX9UfQpBO+pM6rxJGkzgqWcRO1RNo2YOs3wnMs4tWWTvAyWnPrWJO17E9DLwLUua47fXpYrWYPbIamvYPidmhLyKIHsMBEsmKawpnUki2TxsVaJs0cRX\\\\\\\/AwbPjz\\\\\\\/KjMpbMIiPF5mB+\\\\\\\/iHMNwLeUVUWlUrLhKU\\\\\\\/s9k4sD5t8ibDzxUo3wWjylkaKW3PSLTj95fALS9IIZ6kMWIyQZABqKGt6Rws0tgmcjYqMUEAmp2nes+hLTy0f0atPWafWEF+JgjDb73FeIJ3AmCfzdni+9\\\\\\\/rcgFShWaE39vfLNZXd8QbDsd4U5ea2MILfpB5X0PK1oDKq1TCtNUt1c6Cs0u7kcwVqJAG4ejVnnOyTeZ2JqmABO+sMQYTfSHppGMIi2yXhp4\\\\\\\/ra17nJIN00NBFWjdJuduEboSz4z78Qhd2DiRb34DpNWaQM+z+Ujxe2vsopXg64FuROUy\\\\\\\/ii4pP5gb\\\\\\\/n7KrS6zzTQNjOGTphFKiPHN9tC2H+pmqCNJ8sGtdCB9wP2+pZQn36ckKFIZuhCI5JDrGOw1G4xeWh6BPNj3GoZzAo1cncAJvbcMR0sRLFDs4DtBAbRPvTaTETNaOsfjwTRwlfqRavrl9IipTMtOAJdCalGgAXZ0CyNXGbmWsxcsD0l1PkcC9\\\\\\\/sbC8OkiQhgSw0fbfKuI2nt7Z1+UUbE1tv9UVgN\\\\\\\/4FFQY6RRgM1rj59J+oivJnkQwg5evxeuqLZKe6eWHRxiQU\\\\\\\/JYIY4I8qznMkp\\\\\\\/rVD4hmGMHmcJ9cuX0LiR3gqts8pYlFI7Cj\\\\\\\/lqD1oL3rbF3Zw4B3Z4TnwTOuQXkP0cEr33BJWKW\\\\\\\/m8LAxxSQpwjwWTrvfPf6eUjpiC3tp+OyP7ZivCRUblBtG\\\\\\\/Gfcp1LADQnW2mA3JYLkEm0eApJL+p6lBLerJMNWRhKVcfNEXDvbcau1hxUnT+330Anp\\\\\\\/4E0ZAfVzwyuwaSQhotceUbgo8fprIjdyNJn7c50hqK+EUhG9hD+hotEnweop5D1F\\\\\\\/dY\\\\\\\/NfZlKO865vrH8NobtJFGOxzLf56dD00DaBvPxa\\\\\\\/C2r7vBNtj9nOGpEX7sOu3WF1Zb+SCOaXVB3bRfeLmc+ZXQLsvm\\\\\\\/149yPwgbXA1krsAgl\\\\\\\/yAWGVT086JA0Fjod1UGw8LblzLQeWTEQ9ZVgbrDH1RaQT5RIKMqYtDH+SOm15bV48GMr6PADF83hlrBQRPj8iUbQmCm6JmeccklzsU\\\\\\\/Rsxqi03U1BBzIGtY4Qc0pSK70IbFNGP3R9JDmSpsj+Vlumeaia9a8dGjiR8FdOf1HUlwFsXoKRRHduKtqq6a5dZkc3Vnbdmt5gVIdl0GIW5811Saql2yC2\\\\\\\/gyTMbLtU1cPZQr3og2pRiTVXTA2tpKBarO74ykY2NTz7dUVI8RVDosfMWFfKPRQPZvRiLRnuouQ1nCIAJDZ1tOotbg0CUFlpfs+LS06uIstDfEyS7YagfMmFlhcrh4ZkIdWwrXexGqYdKbp+dezKxMY8sswEShsW\\\\\\\/rcKxQYMoH01vNvYkUjo82BU8lvmUrmlTTNSU+WLygSZHeayfsk0KqqEZLYW+ZCks0U6JuowhpaGMBuMrZHw0\\\\\\\/SVD2t30Ryn2cfAGj93wS0VXrY00xrNI2vjYKRGiUiTxy+pZ83kPaOjb4ndb4nM\\\\\\\/RIev\\\\\\\/jYMSzJWyOPzJOdax+ZrqDDILdwGsBARM90Y8vjY8eS7DSgvjskFj\\\\\\\/tHj2iEcI6gqW5q8jJv\\\\\\\/YbYjq0opmTOkLQOB0KXgTtBYG3n7lqA35Q+ujSO5BylTTHHMv6LN9\\\\\\\/XMqkmsWqQR4qynHnCNaN6F1N1tXpbVbsbDJZCF9L9Vx2\\\\\\\/+EBL8CVVolv+NF6xr\\\\\\\/V+uW5oJwi8Pd2nAYFbzT1cfaFUnyzxeJgVnvrV8MirhKQFjwO0IZAYYhkGeP5rJ1P7KZ2fJDNwjHltOArzijIPKdJOxJj+TIO+fqTyhvcesP1CRLpSBNie6zHumihrnNhjpBLW+GiwM\\\\\\\/pJNQm2P5V6wBqimqtCMLoIFh+OtOVxttojvJMzKgcp7+W32TBqYHjjNZ7QCkcM3wKL2208MqxPqOBf9fvW3HYjLOYAj5TJ7RHJmB5nbUcgpeCc4mw2mFlx3NcCclGSKCtth6Y+iA29k8W8LAl3aATFugtn7LSOivpr+P8Yr+42iih+QK20XMvCkCfH73xtqZjgB+GWXJrjCVsx14vVuH57f40Vva2vKzO79kASLrvnMGokalnJr37wdkikoQyLLMqXITMeoN3CkM\\\\\\\/RLr6ChHT8agn8dv3fi0dbdDO2xosfsLl5NOmDfYWpXQiLuT2UbGYGo\\\\\\\/mFFj+MdIM3qkA\\\\\\\/vkY1SoSrau7CQvjK74T4zg7jd8ry0+Qu8o6dNifmY6sEstwHGKlAolPEnhWqQftpUxMWJzLDQ8Y2ezgW4CrsqEEIruwBABvu3pFOQYBBThtnHXE5p3T6QSeFUZORWnPjepEweJKU1nQ8CLiCARkOO5WEANcmvAHkgcgRzC0ihBf8uClX\\\\\\\/Pfhw25Tnt1j1hn9Xr2sXzh8oAmmmItGoqbIhyFQcFZEfw34b5k2zhEnjAT5dhATNcj4lJCICcm3DQq3nW2OjxnLc2rdvSaoVFOdlOPP7HmIRzncwCy7NMHIWcPTmCidSToaoFMe4GYmF3VS\\\\\\\/vvT0uGMqGmwX8blm0ch3xcLR3oh6wGBQ5DGx9y\\\\\\\/DpJFhD1grgnKh3RO08dDpOJNHG4WuD\\\\\\\/wuvxrC7Tno3aIeGFTY86QOTh12rvNjo2B\\\\\\\/BWtA3CNB0UmzEDj61C7t3l+CGEkiuiOpNjM2UM6dkLA+akcqlZlzJtTga\\\\\\\/eghDkjsbCeDMwHt6L7aHk6hbn476ZsWwLmUtWRYEYBa97bWrgDFh6QnYpzHO\\\\\\\/xRtjGWessRVah7OB+cnY85rLaTcF\\\\\\\/9YgNTxkLbhcq1Gx5hWxxjCq17SQ9reNIhBCYmGSkVZR6KE93u4HpLwwCIvTt1CRWV0HwoBViDwGgzaqvDs2igJHXkDWV7U8nCoGX9Mfm4VaU4rD3DJd+4UqZw1WlUHVw7HvmYLl55iXEi3l8bU5cMe4Zhm7v3tYotctmZAYtp1rlqHOXCdvvwn+qAek04yy0ZW6bXwW5LQ3Jsh6fY6+wQbaXLUq6MEGK4vB5xrkj9IlO8QifoLRJ\\\\\\\/0PvDDSZLWkK9ZCb0gonPz+giWv0xFGfSxnrAbMtauh1VxaE34wjpffK3g0w5+c0UWqoGnauXlATqZsHZwhl115Vsngrp+N1\\\\\\\/RGpOMeseL6N18nZIHLzJVgX0xly9zbFrwtAdm\\\\\\\/GvWpRXgs+KjWILJRA8vl95sGacruA1ArBjw0gHKu354iQ3K648q0RVjpwG\\\\\\\/qwBVEaQqPXR4z6RFdZhUF6XAcpbrBsfNr8jrbTBgwBwheT43EOhzlC082k+TYdG88w53p1iUaNsugtF4nV9Me+1VCovieWZu528JdrgTW1AtayLAqQjEvfoTG8VfnG5QlRs1ajxnuSOHQxdhg4tyB3352z2e0dgurha2YidARBNr9foKEqZzIrAzn+0CLZNkeMVMmrQHs9kvp2Thl2\\\\\\\/crlNQsn43CzWBeaAp18wsFUuDycvb5av1KKUhnvhVEtKyC3\\\\\\\/eek0BpokuwITL8J7qeCzWnvRobnwdiqJDw4dINzCuVybnpjt+1uO1jOC65jfJDE9O9RvvDQSpe6DdgqKClS5DUT9bg\\\\\\\/l8Icd5aq6l\\\\\\\/wWw8jIMfTYMLpI21vGWJoSEdZHhPN\\\\\\\/HyuHj2dGWa454UOB3tnRD8jHnFN2Wt7ctQhKkn2jnVaqPwUYIapPR9XpGUKNAbStL3M\\\\\\\/+Boldn5LY6eYBqhYm1DXa1ytaIUGAPBzZz678stckoj9x5GQvnDqGW4m4gpOJBciHSdTpcs1fX7sGjK54vM65NaAY34NUCokO22fsiRdX7MRrTZZIhSdZZcCfcTW8TZEpz16oix60r7XBkxaITDffy8stZsU40GVEXqukOc1mP2kNfvXgCBY9lz0113coTHMVE2Cmlt5I7WaN+IpHM+vVi76WtsOOMXeR+fuDBETlURTxc4yHCrmVO5lXXKgW7pG5UV1Lvqv+0s5xQKqxRFHAe7dIdG6gqkBByc9XfTLswqZf3HViI4tLesqq6i7ICH8FLxadj0LfbH7wILFxIMPOuWUcTa7GsYHQtnT80LOs7DhYz6GxW9LxKFPfls0exPrd\\\\\\\/fZBO5+BE6COrqQZ\\\\\\\/waXWaaFAjhvIRseHe3QwjW2y7OW6md3TkQuQmg3ocFszWKjqEw5xwiyDk4IdIW08VXFCo2EIpDXxAwYCf7ceq0\\\\\\\/IqwqYR53Ar1NEyic9d4VD63SxUDNB2TBFpyoGfKoePHpLKUSOAgGwtIebcO4JsMHEeQXvO2iUMaamPijUjRZsTe07\\\\\\\/+fUonRCfx\\\\\\\/QnHyFQaeY8MuBbcH5Gf7YMWjqH12rpI9nwMCkL8XlW5hCeQddmfSIw4fWY\\\\\\\/3AnL4QJWlJ1X0HShDLPJfLsQQiEWeaHBFuLWCQpjk6BRKP\\\\\\\/89AESeKKCz0fjKWWRg0sl0mvlX7OqFjTBHGHB0hv6kZ5wGJDjuBP4Rhlm0QePiPeegPdHoRazPnm4mviJ\\\\\\\/BpeqzS2Hf3W+K\\\\\\\/eRWPchduFBhfl9YPBKndk+MhWktXBcx0KjJL1rB\\\\\\\/fIyaS+00Lo8rLVGt6YcelwFsBmUqcMGG8Lp1QCsuDSVHSJJNvjJuvgttKSSnMHfngsBhhUIoZOEHHdk8jfCVuQe3C7yOFwTc614B0q9Owe4KWzoA0ryqpUltkO6b7p6RquEAHeSyp+Uk0B6NSNfKG6MxaRpt3Zn9M7SZFx+vJGF3gWcpePe5\\\\\\\/l7bhQj1+DNSKiPWf+0TvjSqATns8xJ4WDV4wuVAiOJlgyFyD9hyMF+2Esd2\\\\\\\/tZdHbGUPNS2eIu\\\\\\\/y\\\\\\\/j5jJz6ccdOUPGmspQvjgWGhSsmv6mtrMToYkvWJhPdSa1ULPyWr7I82LJYMebWSUm+hHAqUHgoPZ1Hu08xUCdhx\\\\\\\/vZURAKfCTCCp025U8OJg1wITPVQzcFYQ\\\\\\\/InwQzspx+FVVKfF72bvZxg4SVvk369qUL7\\\\\\\/NF3rz877CR2VNF3mh1Y3SRm0Rpe8agIPLSuxO7ngpdQPpmCSrRvTORkesCtN6q\\\\\\\/k10IpztOGM7QmqsgF1eu\\\\\\\/VDxrHKuiM71df9ClNp9egeNpb2hd+HNoiFPxa1OfhBYja05oq5xzX72Uu292U5Ch1c+lB\\\\\\\/2bRHYc4Q6b2VHae70UB34yJ1DE5\\\\\\\/p9vBhF\\\\\\\/RAsz8nrNVyY\\\\\\\/qYk0Lix94XnFgLFFA3VdaH+APELnP9AhX+bzCfEdn4WALTrO9erMfZ4Z4K\\\\\\\/FycU5+PpcIf\\\\\\\/M\\\\\\\/ApWO1TQV+PMyYrmBGF0SonjOva73DBkXFxzOFb7NTwr8O\\\\\\\/RfzLV1ut68ZHPFDgiI4geXOWSxrZgKxgE78vU\\\\\\\/S1hx3oipeHCI9iAQbPGAQp9buyWAUpI7mvDLw9pYlvQKe5Mct2OpqEwCyNdtwYdFs63t+OE4ix9PR841cyXFJL7wSGWWeW0DWxrqV+jz0ZAvxugqxr9Prm\\\\\\\/nEDXC0h6ox7izmZQr72WPcXI85MMYLhx6P8bb\\\\\\\/1v8q1S\\\\\\\/6MUtAj0rQoNKLd\\\\\\\/8ethxK\\\\\\\/r7R9fpGmkmrXvqmSqDR6bdLPVMe6tthM0JpWABiq2CWIcaXP\\\\\\\/qxewnW4K6\\\\\\\/x380JWnxaFRFlyLYEk\\\\\\\/AZa5waladoA+Uzzml22Zxt3UyHrmFM3gBSXzTIZFvHrymmt\\\\\\\/H\\\\\\\/jTHcrqrxp0oMwzeWCa2NgUA7j5kSj5xT9Ou\\\\\\\/G6o6S66UzLT+XaH3BK8Lxs2xGJQBEVhiPHO\\\\\\\/r7do3Ac6midKPytPevwJjZ1MvQwNIVPoaVYDK2WENEHIl6rbUEPshnfejw+badq0OmEfCcbEtjeXnDLpQJD865BogXl6B\\\\\\\/Us5VNe43qmTDlI2TcFhtvxGvxFtWXlUQJVhzvDe494ZGeK5sCdEEAa+qq1fwz635DA5ZHoyq\\\\\\\/cf7Ixo3Mmuh\\\\\\\/S15HfXkEd8Z2aduTK0vxqbCt2dWpXqkScWlUyrvloJka5CBKoMbOhEASIta1+guGHldwtATrDe8eC8ym9FZ4dZ1fzJSU7ZrwuGDrmmfFte3Q6r6BaalMXnsyy2FngM2DCtYSWP9gR7X0WxIy9PhBJXKewRWAP5BVtEzGdlK0niKWiNdPOCpF\\\\\\\/bd6lr0ewsfz6zz48rAZujDZdsYI+qGLsvRMvfp\\\\\\\/v97KYdJVSiFGeAQuUjk4L6WSXD1igzc1R9EwB8usqylri0E3XAUwqWPJUaKdAV7T+va1QAdbVsPdcQQBYYRTFSKS3CckbhBvK8gcSsj\\\\\\\/\\\\\\\/N7tVoHvBUAJfM1MAzWYaYQVsaRC79KW3Hi1H9VUjPl57wj8JsJOZQSGN+rw9ts3NEJhe1tadRRYOYJ9HBtkbrVg29kqjyLiWXNxSio4CYxSeAe+WRAgIrrToV6vBCrp9VR2OMa0EwFgI8+c+Iuqy1cdgUqs1qM6s6dl1\\\\\\\/5tiJtc4SFt+2y8wTdJJEqXZmehiXu7eGpgX6ledgQxxw56f4Ekhd5JW3LV3CSHFEmkwtGU9BABL29BCIW3VG78pCseersbCpdWSQ\\\\\\\/oFZXsv2JTjXy2\\\\\\\/T85nbKyzzbLcWDn2StMpn4O+TSvIRVgR7aU4XEroNFEgt4oLcJ+MCaHvMykfwEukAhHG6U8732iFhHfHYCwuFAB5lTqUYWnVuCk2AAQJhDgIR2I+Zl5J5WnBD11bKKNg7CrsrCZAIWHJZBEhjPVso82CrmH4lfNlBn2JvcNWf+8sIac0S4SJUVxJteZ++C9GwQH9EfDKYaHtlbLjv9HHYtOxK6lnqMelusdwDekP7tirvhULEvDYQXoG9yrR5WOUFbh4qV3FZGoBYJNTstuyE8eC\\\\\\\/9j\\\\\\\/d\\\\\\\/hhEEUc+m16xy2ln7qjAUb7E3J\\\\\\\/MSI3f27TH1OUeg+LMEFDhvovMHYOnA+b\\\\\\\/M173aeVEfKHpKKWvlUgVgRgTXtriU5wmkQT3SHJK1aDRMxES3PLzFrNmE7qcVBpn6l7ENYdNJfzte0e2AJFX0SB+fmMTbgEHnmlIb4FqEzAC908xET4b+Y7vxrY3LjHVsxm2fRBxtAf87wULocxBtwJxjVkZnsLFD0efEGcE24X8ZL66Ki+QcVELzR3aX3ADDcWf6Tl79Vtwu8hWFiAe3CEsYSTFXQS8f98yxG5jgRBrBnadK9Hq09wzpFZKifYXj4VHenD8o1lg8g8IyyUcCuH+sXphrOqR1cVn4G9EthRVyTc1Ez0TL+N7QyoFA7\\\\\\\/R9Z1kb7lsC+SRGVfIFJUHAhgYVw8WGdyq5vJBhRNMR9WfS3LL4TjJ\\\\\\\/Mna5rv\\\\\\\/Dq+hMV1GpPF3BruQI2JqDaqDW4\\\\\\\/pD7qzScXOEKCveNCX9zemDzfUW\\\\\\\/5benefNi8fV7m3oT\\\\\\\/fCAbQFFVTZURgtno\\\\\\\/zBcR5JJYmnTI5a9a1Y3poQ+Wl8fFA6Mmc9ODQZ7NH8\\\\\\\/DRup+FF7IVajCbXkjPIV+RLCbu0GXLWMb1d57cqFh91WfVsKimFdbA0w3I3a7ObTTKLLlg2tXsl7wdLcpkssfBZCwQVHTwY1f5xAoxaNjHOYGlmB5gtZGhsqPruwXdC20IpMTB2yrdJT1OGSgwnLcbRI40ejvxSkBGzZEoLlXFmete1bUp5m08lSYl\\\\\\\/RpouEvKl5Q02kAT9FOq5HSPs0nJKomFdkUFp\\\\\\\/rkBgZzSmKFPqPQhfh9m98HpQbr3llcHOhBbjFCvshzetOTh8Mh84+hN7TuMq4U5kYLBDxlaGU0F0+K47qr\\\\\\\/fUpL32CIxiUU3cxLEhC4kYaibjeVnKKoqBY3og05X5rF+CtVjhIlE+jsC9+3wrd86\\\\\\\/XxQJslirZ7cdLLLYVdUZpx2RzUPxGntTEGCQHkqQrrJ2QgKVVipAZgaubcgC+jqyB59XPY2ACibItebsnL+YzI5yEBzilv21jYPVxbaS75pybkAOgWeWReZe7YlVKW9WqcXdEU8JohPn+b3G7sP0S2v\\\\\\\/54Yq3Qrd6yprO0bJU9ZBBdINPqr+Z6KFiHI1OJ6kMHu41rfQ6B6gyiRTrnR8PvmagWAALzlKn4cQ+OKEGHzkzDyOIjAXkGLkQ4w9SfQC46c337evvxj8QIPKbXf+zHU5kzQEn75+uI9nwbQhyvAKz17X6TlhPT293xMjG53ykbC4QIsRv2QEGyyReQRMxKYmP9OdyVTyS5hjwJVfl1ZxVlCuwCtNlz2+rt6zP0js\\\\\\\/Srvm0oa5P1JwP+RcahpvjqE+XicXvGBjBnwQfn3YxLs7gsN31iB4gc5cFZz7AAWi8xu4jp2hNThLZT95z7rcrsb+QPSZZcgu5XeJNmGHG3bEs5Yt0mFOJ+pN8cFNTb+8qVDZ5wQtpi3jdzm8UYHVANpt7ucpkDH\\\\\\\/X09W7xlGy1u6oCfRjB9O0rYnC76UtlHWLLgPQVbH59lnaXUzXdTl8dj5QRZWQfivb6CZNHcWNccSo5nzXntNH\\\\\\\/Plj9Tq7QS9G7o7wJgrHhx6sp78e96J9O3DeYwb7sjlWLvDWDHbziVL0RM8ptrTgPBYiwG4eq\\\\\\\/Bu8p09ERf7qr5cOmPIaVV65Qj4DOxHOTb0CANs5bUyglZHpLgx+j6Pvoj2XLRGQlgsxN8B4TEa6f+KUG9+limGnywY\\\\\\\/I3UeV87eZrVizutX0Es8zE3gY8puQo34Kr6utexgyOZF6tkxFPpQ1+PmEaz1LDPeR+1aikiPoBySAgAj4fZt+4mXAk1hKpHu5Nf+ydFgWw1308hOS5EKowyd0Pn4qNeaatSwYdgjOmUvDZ6MpUUZHPBk74Ay+g+MWlsZfqYbn61gVCeLlu3hAh37GBOFFYT30VvLx3Nta\\\\\\\/neWVUNPVJBqZJeiYlgTcOczcu6ENMbJfFaqbjUl0OwzsgS3NPWxSUQm0qgfZh2MzRO7qWbkvNJISNcjUVcd93q1FGnO98DWxV3gkVodRMbStXuxaHz3leNShWdXCWaNNkC1B2d71wOqZbekbTN2hg+dL2f+28vtwKkOb6gvAc\\\\\\\/e5dK89hqA2+KQ967GOAHH6muq+fjqRuzlMsJ03IpPRTRMrm\\\\\\\/bEjYtufNnGa4f8dNjdAFTZyROcU3iAgUpXPBAreexMXJj3fJ3fhMP1OiskVPUwF9xQ7diXUbjDDLbYnAAcdiWBbRDR2kEb518dF4kaPaehoy32q5J6Rz4LWOaSyesFgY7eE1NpxpP5WgY9vFouEjf5ajuAYtUc+uZ677ejwuYFbpm2qW5uMOIjlZHIXcvNUAdQv9w\\\\\\\/pX8lsABT5h95sezoFJkIoM4W0Z+jXLHzfLFAswDq2AGlIuJma7BrHODDeiwe8MjlN7P+4hcXzQ5TgWzXaZIZ7JOXDiDeO7vdgls8zALk7acPvHN2MnNB4z7F5jem8dsIydijBPQryB7pWIYp5+XuvS0mVbWFPmELOeAgpEPVLS41vuJoubiFbGAj4vf3bM772cravV7luGuhSlGvQ1NgKzLzn+ym3+Du\\\\\\\/zwGczKtRqdlm\\\\\\\/vnfYD8stheM7VrLDHXQrw5HkLhB8vAZv9gONIG7iH\\\\\\\/fcx4ctY5AIthpPBEu51Unu+9swaTGMMAiZ9i+Am+BMYTUI5vFipXKU9AbLOFZ1nJYz8SsFU6fyOO0wF1phHZmuRZryhyQ7xX9xdc5XaHaW6AAF8nQkC4IlRboUDStHgNICtbPJJRbhrCeH8BVx+16bANGzg\\\\\\\/pcgXHfc5SqYzMDgpASX\\\\\\\/zjhkcGdIHxXRS2kc3y8QOM48B8kOnDRXozgdGqDVJ0SC6DFEdJten3+hRUPuUffjvFONOR0nHOgIxNrk3X+FyWIWtRlEiIZwZD4jE9ecJJKOjMtXg7I\\\\\\\/kIFqUUxc0afS9fqeEyCfaH3hevUxRtmOko5ZXKr1Z8FwQgCyLyHkRciEMdAlAduYMoB\\\\\\\/hEq0hyduZ9tk4xVmnzt7WbypT0+jf+Hoyz5UtIxtjhw6iVdpphWv\\\\\\\/oF9pppkSdB\\\u0022,\\\u0022iv\\\u0022:\\\u0022l7g41+UfcoPwJojr7F0jCg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022667f6eb269626084d61773df9f1d6400e6f6b6a57a1399621d30f080f187ee63\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022VUhuqbJpXIpV9rYs\\\\\\\/zeH+wlZ0QcyCQgMCAENljIW4+gdthpNXUL3c71DFW6B\\\\\\\/fu0vusvvfpZKfioquN4cXbOzqtdnD7JoXroUXtnPzO0TaJHWbjNtYvIfX6fNQMwlk7Oskgd1TxSETI6RYslZZmkBjRclXvav+mORWqvgXP0NfMqSrHolaLrjS2MKczVddVtQ8VlnAhF4kLzaDIt\\\\\\\/meqltcvA0WV8jNGHaoPMJdSo\\\\\\\/fSv4SE3l0nEdxyZGmAUVcoYfBXQqAbRfjVGusDqHpmugMNuZkJTCD2Nyrabew5GjDIUt2n1j1lrwjegJJcrpI97mxpEXHxMfi6GpfWcswkunNdZYpfMs6redFPsOb4vqrH5dPnZFTD763J79z2SzCy0lNHIS\\\\\\\/3oF7WPdauRP8yf3nG+YxJoJC7PlwHADjD1CsrWtYvL3VrP\\\\\\\/cP2sP\\\\\\\/i6IOFaWyo1QRfbjdoEfBupbKwPcBxHnHiHOknlpyJdiFy7fO6HPrsnEwLs6xA1KQwQU3xMIDH4RiYvEMdWbw2SMVuaoz61TK2Xb0Ld56BZ2\\\\\\\/BpE02lSZOBQUs0yKpEO5Tne3hZF5h\\\\\\\/qVDiUMcvP+bJ37Q3RMX\\\\\\\/KaRCNs3gYsdemHkfWqSR8NFCili+7fXI2MzVGEkt2QcTtnt+lw5hXnOpPabBy0Eseo7XTiy0OGcyKXGPuzssRtqzrJt88XDywD30tv8+gSnY4SR49f398x4cBAxV6ZZY2zejby+FXsMp0zW55rrm9R\\\\\\\/X3y2iZuH5UBelI91f23OvqNzQoR1GIEp4\\\\\\\/U2sNNSWaJQaUO8DxiKsYkzBxx\\\\\\\/1UnXtRWAJqHd8dbMVSbjU\\\\\\\/2R1k6hGLLWH8Sl\\\\\\\/rTbaRrlzV2buVszBuJVWei6wa2zrlTFupHKy17tuKaVgjY0W8WzuDdoplE2uGeLY2\\\\\\\/dbbdLJX+vfMiHvuWMSo6rT3ylodML\\\\\\\/wn1P5+N2vKgX0fEBxz\\\\\\\/gAF\\\\\\\/HlSEvkx6UPS3BtpLlFqGbkUmBWtZ\\\\\\\/ARON+It+JrIW1YLBMzLZpFuq2UNrnVaNMytn+IGp5hd8CzOv4U0vPZlCWMJm1ehWLKD1ux2a5F2ihPfUIg9nGyuKsQrfRGz4RYhn0MjcV7RdytRXkbKsLT+LPEnoOCEBn6WOrJUoCwRfuIZrcoqfDElawliFEfvBr0y9rDJLMpm+VviC984hwR2c3mhJiss9oCkiOQs4cNRUQ890Fayznb0ypRnwkn+Xyn9ZVUxgpldRO4QqQ\\\\\\\/A\\\\\\\/JI2Ns3DNlL3kZgl1ouqWw+6+jsf5tN1aATOwLluRw7Uf9AQtIK+k+Q46gxntmW9hTgVLhbIT+1WDveBCbbNNZ6SkyagwRB5pGcSC1uP0g6FjwtUgbOw0JwzKOfv4OsZUG186otwIUbKeQRwwkmZVAmQdPdf17gi+oMovaiqRzn+GFCXJrDWOKnCK8OFHrhS\\\\\\\/+ARcLb\\\\\\\/yuiunpRUp0ShChjrZayUWe3fpez204EWXxYGakIIo0gzJ1NvNvioLzqbwlMNi0qcT4VRCM7MnDuYsg4WtGf3uKywAfMr6uUGQPhb3d9iiVIBZaXqxVCuPDphtXr7kaSszJ3dUuwVQQ7BcnyRb3DyXWMK7S03vbpz7jzp+wdpvt2NN6yl\\\\\\\/ShGp7Tvo3TclNFYKbJckGfimJleh5E\\\\\\\/8T1wNFArqaGiFhQtq8ud5caDxJR29bLjbwZxk3SJSYWebHIaKChHKOxkEfDcjwmi5HcyNUdfJTHX8YkPYQQCo9p12oKRtSTJL+AsgI5rWp98PJSa\\\\\\\/3XWmsomWUIJnhNtD8YrYC4zaTq+gnvTms74S\\\\\\\/EDYtUD89WHX85Cu6g\\\\\\\/SGTbTAv6LuAStIPb1j4rSIctfSlC8degHlvEIV1pk0X\\\\\\\/0Sqp95X657NcZpWzIk3FPg0vRjmp61z3Mxz4sZWJNCl1nUPNKUDPitMpMqjHSaPX1D\\\\\\\/oKDFaecOGTwg5vLw+pNHD\\\\\\\/DeqQfsdQqbQjoO0eeckDEIan5JvwmVyyvjcJ4TFyd5Ov9n1eISLevNzCSPWL2LDZlgdgS5MyTrCTEDxJXb5rvcIEJVnqdUv\\\\\\\/UJ\\\u0022,\\\u0022iv\\\u0022:\\\u0022Jsi6wvwNZCk3VDJkD3mqpw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227ae307f6ecf987eb7449a2673dd46f1b64263a71b55062ab048cd1a43c6410c5\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-493756 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/8/2489" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-jBW64nuQqsooi7IVYWsbo0bR7eIoS8LrWV1Babgv2rc" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/8/2489" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-u7jMXHteoUIghCRr5WAkqbPVmZie4eDks6AjkJzgVk4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/8/2477" title="Protein Kinase D Is Dispensable for Development and Survival of Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/8/2497" title="Structural Characterization of ABCB1, the Gene Underlying the d2 Dwarf Phenotype in Pearl Millet, Cenchrus Americanus (L.) Morrone" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493756" id="node493756" data-pisa="ggg;9/8/2489" data-pisa-master="ggg;g3.119.400123" data-apath="/ggg/9/8/2489.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Systematic Analysis of Metabolic Pathway Distributions of Bacterial Energy Reserves</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5339-7484" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Liang Wang</span>, <span class="highwire-citation-author" data-delta="1">Jianye Yang</span>, <span class="highwire-citation-author" data-delta="2">Yue Huang</span>, <span class="highwire-citation-author" data-delta="3">Qinghua Liu</span>, <span class="highwire-citation-author" data-delta="4">Yaping Xu</span>, <span class="highwire-citation-author" data-delta="5">Xue Piao</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-6559-4303" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michael J. Wise</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2489-2496;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400123 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493756" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Liang Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Bioinformatics, School of Medical Informatics,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Jiangsu Key Lab of New Drug Research and Clinical Pharmacy, Xuzhou Medical University, Xuzhou, Jiangsu Province, 221000, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Liang%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALiang%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5339-7484" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Liang Wang</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:leonwang@xzhmu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">leonwang@xzhmu.edu.cn</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jianye Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Bioinformatics, School of Medical Informatics,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jianye%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJianye%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yue Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Bioinformatics, School of Medical Informatics,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yue%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYue%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Qinghua Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Jiangsu Key Lab of New Drug Research and Clinical Pharmacy, Xuzhou Medical University, Xuzhou, Jiangsu Province, 221000, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qinghua%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQinghua%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Yaping Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Bioinformatics, School of Medical Informatics,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yaping%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYaping%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Xue Piao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Bioinformatics, School of Medical Informatics,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Information and Control Engineering, China University of Mining and Technology, Xuzhou, Jiangsu, 221116, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xue%2BPiao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Piao%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXue%2BPiao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Michael J. Wise </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Computer Science and Software Engineering, and</div><div class='author-affiliation'><span class='nlm-sup'>**</span>The Marshall Centre for Infectious Diseases Research and Training, University of Western Australia, Perth 6009, WA, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BJ.%2BWise%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wise%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BJ.%2BWise%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6559-4303" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael J. Wise</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/8/2489" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:493756" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:493756/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/8/2489.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:493756" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:493756/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/8/2489.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:493756" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:493756/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/8/2489.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:493756" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:493756/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Previous bioinformatics studies have linked gain or loss of energy reserves with host-pathogen interactions and bacterial virulence based on a comparatively small number of bacterial genomes or proteomes. Thus, understanding the theoretical distribution patterns of energy reserves across bacterial species could provide a shortcut route to look into bacterial lifestyle and physiology. So far, five major energy reserves have been identified in bacteria due to their capacity to support bacterial persistence under nutrient deprivation conditions. These include polyphosphate (polyP), glycogen, wax ester (WE), triacylglycerol (TAG), and polyhydroxyalkanoates (PHAs). Although the enzymes related with metabolism of energy reserves are well understood, there is a lack of systematic investigations into the distribution of bacterial energy reserves from an evolutionary point of view. In this study, we sourced 8282 manually reviewed bacterial reference proteomes and combined a set of hidden Markov sequence models (HMMs) to search homologs of key enzymes related with the metabolism of energy reserves. Our results revealed that specific pathways like trehalose-related glycogen metabolism and enzymes such as wax ester synthase/acyl-CoA:diacylglycerol acyltransferase (WS/DGAT) are mainly restricted within specific types of bacterial groups, which provides evolutionary insights into the understanding of their origins and functions. In addition, the study also confirms that loss of energy reserves like polyP metabolism absence in <em>Mollicutes</em> is correlated with bacterial genome reduction. Through this analysis, a clearer picture about the metabolism of energy reserves in bacteria is presented, which could serve as a guide for further theoretical and experimental analyses of bacterial energy metabolism.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/energy-reserve" class="hw-term hw-article-keyword hw-article-keyword-energy-reserve" rel="nofollow">Energy reserve</a></li><li class="kwd"><a href="/keyword/hidden-markov-model" class="hw-term hw-article-keyword hw-article-keyword-hidden-markov-model" rel="nofollow">Hidden Markov model</a></li><li class="kwd"><a href="/keyword/evolution" class="hw-term hw-article-keyword hw-article-keyword-evolution" rel="nofollow">Evolution</a></li><li class="kwd"><a href="/keyword/proteome" class="hw-term hw-article-keyword hw-article-keyword-proteome" rel="nofollow">Proteome</a></li><li class="kwd"><a href="/keyword/metabolic-pathway" class="hw-term hw-article-keyword hw-article-keyword-metabolic-pathway" rel="nofollow">Metabolic pathway</a></li></ul><p id="p-3">Due to the diversity of environmental niches that bacteria have colonized through millions of years of adaptation and evolution, bacteria have evolved specialized sets of metabolic pathways to live optimally in these environments, which can be reflected in their characteristic genomes, gene transcription profiles, and proteomes (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Zhu <em>et al.</em> 2016</a>). Previously, comparative genomics studies have shown that the loss of glycogen metabolism could serve as an indicator for bacterial parasitic lifestyle while the gain of polyphosphate (polyP) metabolism seems to link with free-living lifestyle and higher bacterial virulence (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Wang <em>et al.</em> 2018</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Henrissat <em>et al.</em> 2002</a>). In addition, it has been observed that loss of glycogen or polyP metabolism is associated with reduced genome size, providing a hint about genome reduction (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Wang and Wise 2011</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Wang <em>et al.</em> 2018</a>). It is well known that both glycogen and polyP are important energy sources in bacteria. Thus, presence or absence of energy reserves in bacteria could be important for <em>in silico</em> analysis of bacterial physiology and lifestyle, especially when large numbers of sequenced bacterial genomes are available and many of those are unculturable by traditional laboratory techniques.</p><p id="p-4">It is well known now that energy reserves play essential roles in bacteria for their regular activities to sense and respond to changing environments and different types of stresses, such as temperature fluctuation and nutrient deprivation, <em>etc</em>. (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Wang and Wise 2011</a>). Although there are many different energy-related compounds, not all of them can be classified as energy reserves. According to Wilkinson, three principles should be satisfied for a compound to be considered as an energy reserve, which are: 1) accumulation when energy is over-supplied, 2) utilization when energy is insufficient, and 3) apparent advantages by consuming the compound when compared with those without it (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Wilkinson 1959</a>). Through physiological and biochemical studies, five energy storage compounds have been regarded to meet the criteria, which are wax ester (WE), triacylglycerol (TAG), polyhydroxyalkanoates (PHAs), polyphosphates (polyP) and glycogen (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Wang <em>et al.</em> 2017</a>; <a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Wang and Wise 2011</a>; <a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Wang <em>et al.</em> 2018</a>; <a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Wilkinson 1959</a>).</p><p id="p-5">Several studies have investigated the distribution patterns of energy reserves in bacteria, most of which were based on small sets of bacterial genomes or proteomes. No comprehensive and systematic phylogenetic analysis currently exists (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Zhang <em>et al.</em> 2002</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Whitehead <em>et al.</em> 2014</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Achbergerová and Nahálka 2014</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Kalscheuer and Steinbüchel 2003</a>; <a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Wang and Wise 2011</a>; <a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Wang <em>et al.</em> 2018</a>). In this study, we collected 8282 manually reviewed bacterial proteomes from UniProt database and sourced key enzymes directly involved in the metabolism of five energy reserves from public literature and database (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>) (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">The UniProt Consortium 2017</a>). Hidden Markov sequence models were used for searching homologs in bacterial proteomes. Distribution patterns of representative metabolic pathways of the fiver major reserves are presented in Table S1. In order to gain an explicit view about distributions of the enzymes along the evolutionary paths, we incorporated the enzyme distributions into phylogenetic trees constructed via NCBI taxonomy identifiers (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Federhen 2011</a>). Through a combinational analysis of the pathways in the phylogenetic trees, we have identified interesting distribution patterns of metabolic pathways that are linked with bacterial groups specific lifestyles, which may improve our understanding of the functions of energy reserves in bacteria. In addition, our systematic analysis also gives us an overview of enzyme distributions, which could serve a as guide for further theoretical and experimental analyses of energy reserves in bacteria.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493807/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493807/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Key enzymes and corresponding UniProt sequences used in this study for statistical modeling via HMMER package</span><div class="sb-div caption-clear"></div></div></div><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Proteomes and enzymes collection</h3><p id="p-7">Bacterial proteomes were downloaded from UniProt database by using two keywords, <em>Bacteria</em> and <em>Reference Proteomes</em>, as filters (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">The UniProt Consortium 2017</a>). A total of 8282 bacterial proteomes were collected, reflecting the state of knowledge as at 2018. Of these, 68 proteomes were removed due to outdated taxonomy identifiers that cannot be identified in NCBI taxonomy database when constructing phylogenetic trees (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Federhen 2011</a>). A complete list of all the 8282 bacteria with UniProt proteome identifiers, NCBI taxonomy identifiers, bacterial names, proteome sizes, and distribution patterns of key enzymes is available in the Table S1. For each of the five major energy reserves, only key enzymes were considered. For the synthesis of WE and TAG, the bifunctional enzyme wax ester synthase/acyl-CoA:diacylglycerol acyltransferase (WS/DGAT) was studied due to its pivotal role in these processes (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Kalscheuer and Steinbüchel 2003</a>). In addition, the enzyme phospholipid: diacylglycerol acyltransferase (PDAT) that catalyses the acyl-CoA-independent formation of triacylglycerol in yeast and plants were also screened in bacterial proteomes (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Dahlqvist <em>et al.</em> 2000</a>). For metabolism of polyP, the key enzyme polyphosphate kinase (PPK1) for synthesis and two degradation enzymes, intracellular polyphosphate kinase 2 (PPK2) and extracellular Ppx/GppA phosphatase (PPX), were included (<a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">Wang <em>et al.</em> 2018</a>). For glycogen metabolism, two synthesis pathways were considered. The first one involves glucose-1-phosphate adenylyltransferase (GlgC), glycogen synthase (GlgA), and glycogen branching enzyme (GlgB) (<a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Wang and Wise 2011</a>). The second pathway includes trehalose synthase/amylase (TreS), maltokinase (Pep2), α-1,4-glucan: maltose-1-phosphate maltosyltransferase (GlgE), and glycogen branching enzyme (GlgB) (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chandra <em>et al.</em> 2011</a>). Key enzyme Rv3032 for the elongation of α-1,4-glucan in another pathway relevant to glycogen metabolism and capsular glucan was included (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Chandra <em>et al.</em> 2011</a>). In addition, archaeal type GlgB belonging to the glycosyl hydrolase family 57 (GH57) was also investigated for comparative analysis due to its importance. As for PHAs, major enzymes responsible for synthesis such as Acetyl-CoA acetyltransferase (PhaA), acetoacetyl-CoA reductase (PhaB) and poly(3-hydroxyalkanoate) polymerase subunit C (PhaC) have been identified (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Handrick <em>et al.</em> 2004</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Verlinden <em>et al.</em> 2007</a>). Based on Pfam analysis, PhaC could be further divided into two groups, PhaC Group 1 with PFAM domain Abhydrolase_1 (PF00561) and Group 2 with PFAM domain PhaC_N (PF07167). In addition, other enzymes in the PHAs synthesis pathway were also studied, which are 3-oxoacyl-[acyl-carrier-protein] reductase (FabG), (R)-Enoyl-CoA hydratase/enoyl-CoA hydratase I (PhaJ), malonyl CoA-acyl carrier protein transacylase (FabD), succinic semialdehyde dehydrogenase (SucD), NAD-dependent 4-hydroxybutyrate dehydrogenase (4HbD), and 4-hydroxybutyrate CoA-transferase (OrfZ) (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Breuer 2010</a>). Together with poly(3-hydroxyalkanoate) polymerase subunit C, these enzymes are able to synthesize PHAs in bacteria. All the enzymes are screened for presence and absence in collected bacterial proteomes. For details of these enzymes, please refer to <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>.</p></div><div id="sec-3" class="subsection"><h3>De novo construction of HMMs</h3><p id="p-8">Each selected seed protein was used for building HMMs via HMMER package (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Pearson and Eddy 2011</a>). In specificity, after obtaining sequences for all seed proteins from UniProt database, remote BLAST was performed to collect homologous sequences for each seed protein from the NCBI non-redundant database of protein sequences (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">McGinnis and Madden 2004</a>). Usearch was used to remove the homologous sequences with more than 98% similarity from the selected proteins (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Edgar 2010</a>). The standalone command-line version of MUSCLE was used so the multiple-sequence alignments were created automatically (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Edgar 2004</a>). N- and C-termini of multiple sequence alignments tend to be more inconsistent (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Wise 2010</a>). Thus, all MSAs were manually edited to remove these parts by using JalView (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Waterhouse <em>et al.</em> 2009</a>). HMMER was selected for the construction of HMMs through hmmbuild command. Since HMMER only recognizes STOCKHOLM format, all MSAs results were converted from FASTA to STOCKHOLM format. In addition, another set of HMM models sourced directly from PFAM database based on the domain structures of seed proteins were collected. For seed proteins with more than two domains, <em>de novo</em> constructed HMM models were used for homologous screening, while those with two or less domains, established PFAM domains were used. For searching homologs in bacterial proteomes, routine procedures were performed by following HMMER User’s Guide (<a href="http://eddylab.org/software/hmmer3/3.1b2/Userguide.pdf">eddylab.org/software/hmmer3/3.1b2/Userguide.pdf</a>). Only those hits meeting the criteria of E-value less than 1e-10 and hit length greater than 60% of query domains will be considered. Results obtained from HMM screening can be found in Table S1, that is, the presence (copy numbers) or absence of a specific enzyme in a certain bacterial proteome.</p></div><div id="sec-4" class="subsection"><h3>Data visualization</h3><p id="p-9">Phylogenetic trees were first constructed based on NCBI taxonomy identifiers for all bacteria in this study via the commercial web server PhyloT (<a href="https://phylot.biobyte.de/">https://phylot.biobyte.de/</a>), and were then visualized through the online interactive Tree of Life (iTOL) server (<a href="https://itol.embl.de/">https://itol.embl.de/</a>) (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Letunic and Bork 2016</a>). Distribution patterns of enzymes and their combinations in terms of energy reserves were added to the trees through iTOL pre-defined tol_simple_bar template (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Letunic and Bork 2016</a>).</p></div><div id="sec-5" class="subsection"><h3>Statistical analysis</h3><p id="p-10">Unpaired two-tailed Student’s <em>t</em>-test was used for statistical analysis through R package. Significant difference was defined as <em>P-value</em> &lt; 0.05.</p></div><div id="sec-6" class="subsection"><h3>Data Availability Statement</h3><p id="p-11">Supplemental material available at <a href="https://gsajournals.figshare.com/s/08e2bf674f15b9704676">https://gsajournals.figshare.com/s/08e2bf674f15b9704676</a>. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8138951">https://doi.org/10.25387/g3.8138951</a>.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Wax ester and triacylglycerol</h3><p id="p-12">The key enzyme that is involved in both WE and TAG synthesis in bacteria is WS/DGAT. Through the screening of HMM-based statistical models, 673 out of 8282 bacterial species harbor a single copy or multiple copies of WS/DGAT homologs, which are mainly present in phylum <em>Actinobacteria</em> and the super-phylum <em>Proteobacteria</em>. Only a few bacteria in groups such as FCB (also known as <em>Sphingobacteria</em>) and PVC (also known as <em>Planctobacteria</em>), <em>etc</em>. have WS/DGAT. No species belonging to phylum <em>Firmicutes</em> has WS/DGAT. As for the unclassified bacteria, although no WS/DGAT was identified, they will not be studied due to their uncertain classification at current stage. For details, please refer to <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>. By comparing the proteome sizes of bacteria species with or without WS/DGAT, we found that bacteria with WS/DGAT have average proteome size of 5294 proteins/proteome while those without WS/DGAT have average proteome size of 3117 proteins/proteome (<em>P-value</em> &lt; 0.001).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2489/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution patterns of key enzymes and representative pathways along the NCBI taxonomy ID-based phylogenetic tree. (A) Distribution of key enzyme WS/DGAT (green circle) and PDAT (blue circle), together with bacterial proteome sizes (red circle); (B) distribution of the classical PHA synthesis pathway PhaABC (orange circle) and the two groups of PHA synthases, Group I with Abhydrolase_1 domain (blue circle) and Group II with PhaC_N domain (green circle); (C) distribution of key enzyme polyphosphate kinase PPK1 (blue circle), polyphosphate kinase 2 PPK2 (green circle), and exopolyphosphatase PPX (orange circle); (D) distribution of two glycogen synthesis pathways and four key enzymes, that is, classical synthesis pathway (yellow circle) including GlgC, GlgA, and GlgB, trehalose-based glycogen synthesis pathway including TreS, Pep2, GlgE, GlgB (black circle), glycogen synthases (Rv3032 in blue circle and GlgA in green circle) and glycogen branching enzymes (GH57 GlgB in orange circle and GH13 GlgB in violet circle). Five groups of bacteria are highlighted, which are Firmicutes (green), Actinobacteria (blue), Proteobacteria (orange), Unclassified Bacteria (violet), and FCB group (brown)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1487736011" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution patterns of key enzymes and representative pathways along the NCBI taxonomy ID-based phylogenetic tree. (A) Distribution of key enzyme WS/DGAT (green circle) and PDAT (blue circle), together with bacterial proteome sizes (red circle); (B) distribution of the classical PHA synthesis pathway PhaABC (orange circle) and the two groups of PHA synthases, Group I with Abhydrolase_1 domain (blue circle) and Group II with PhaC_N domain (green circle); (C) distribution of key enzyme polyphosphate kinase PPK1 (blue circle), polyphosphate kinase 2 PPK2 (green circle), and exopolyphosphatase PPX (orange circle); (D) distribution of two glycogen synthesis pathways and four key enzymes, that is, classical synthesis pathway (yellow circle) including GlgC, GlgA, and GlgB, trehalose-based glycogen synthesis pathway including TreS, Pep2, GlgE, GlgB (black circle), glycogen synthases (Rv3032 in blue circle and GlgA in green circle) and glycogen branching enzymes (GH57 GlgB in orange circle and GH13 GlgB in violet circle). Five groups of bacteria are highlighted, which are &lt;em&gt;Firmicutes&lt;/em&gt; (green), &lt;em&gt;Actinobacteria&lt;/em&gt; (blue), &lt;em&gt;Proteobacteria&lt;/em&gt; (orange), Unclassified Bacteria (violet), and FCB group (brown).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2489/F1.medium.gif" width="440" height="246"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/8/2489/F1.medium.gif" width="440" height="246"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2489/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2489/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493915" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-13" class="first-child">Distribution patterns of key enzymes and representative pathways along the NCBI taxonomy ID-based phylogenetic tree. (A) Distribution of key enzyme WS/DGAT (green circle) and PDAT (blue circle), together with bacterial proteome sizes (red circle); (B) distribution of the classical PHA synthesis pathway PhaABC (orange circle) and the two groups of PHA synthases, Group I with Abhydrolase_1 domain (blue circle) and Group II with PhaC_N domain (green circle); (C) distribution of key enzyme polyphosphate kinase PPK1 (blue circle), polyphosphate kinase 2 PPK2 (green circle), and exopolyphosphatase PPX (orange circle); (D) distribution of two glycogen synthesis pathways and four key enzymes, that is, classical synthesis pathway (yellow circle) including GlgC, GlgA, and GlgB, trehalose-based glycogen synthesis pathway including TreS, Pep2, GlgE, GlgB (black circle), glycogen synthases (Rv3032 in blue circle and GlgA in green circle) and glycogen branching enzymes (GH57 GlgB in orange circle and GH13 GlgB in violet circle). Five groups of bacteria are highlighted, which are <em>Firmicutes</em> (green), <em>Actinobacteria</em> (blue), <em>Proteobacteria</em> (orange), Unclassified Bacteria (violet), and FCB group (brown).</p><div class="sb-div caption-clear"></div></div></div><p id="p-14">Within the major phylum of <em>Proteobacteria</em>, WS/DGAT is not evenly distributed and γ-, δ-, and ε-<em>Proteobacteria</em> sub-divisions have more species harboring WS/DGAT genes. In addition, two orders, <em>Rhodobacterales</em> (305 species) and <em>Enterobacterales</em> (168 species) that belong to α- and γ-<em>Proteobacteria</em> phylum, respectively, do not have any WS/DGAT except for one species <em>Plesiomonas shigelloides 302-73</em> (NCBI taxonomy ID 1315976). As for the phylum <em>Actinobacteria</em>, two WS/DGAT abundant regions (R1 and R3) and one WS/DGAT absence region (R2) in the phylogenetic tree are worth further exploration. R1 region includes three closely related families with most species harboring WS/DGATs, which includes <em>Streptomycetaceae</em>, <em>Pseudonocardiaceae</em>, and <em>Nocardioidaceae</em>. R3 includes only one family <em>Mycobacteriaceae</em> (115 species) in which bacteria have up to 10 homologs of WS/DGAT. R4 is the family <em>Corynebacteriaceae</em> (69 species) that only includes WS/DGAT-free bacteria.</p><p id="p-15">As for phospholipid:diacylglycerol acyltransferase (PDAT), it is involved in TAG storage in yeasts and plants. Recently, PDAT activity was proven in <em>Streptomyces coelicolor</em> for the first time (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Arabolaza <em>et al.</em> 2008</a>). However, no bacterial homologs were known yet with similarity to the respective eukaryotic sequences (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Röttig <em>et al.</em> 2016</a>). Our systematic screening of <em>Saccharomyces cerevisiae</em> PDAT unexpectedly identified 46 proteins belonging to 42 out of 8282 bacterial species (Table S2), 38 homologs belong to small genome-sized unclassified bacteria while 8 homologs were present in <em>Terrabacteria</em> group. Both E-values (&gt;1E-10) and target sequence lengths (&gt; 60% query sequence length) were also given to confirm the significance of the results. Thus, presence of eukaryotic PDAT in bacteria is theoretically true, though functionality of the enzymes requires further experimental validation. Although most of the sequences were annotated as uncharacterized proteins in the UniProt database, some were described as acyltransferases. Interestingly, the PDAT homolog in <em>Clostridium butyricum</em> E4 str. BoNT E BL5262 was thought to be a putative prophage LambdaBa01 acyltransferase, which indicated that the enzyme could jump around among species via horizontal gene transfer.</p></div><div id="sec-9" class="subsection"><h3>Polyhydroxyalkanoates</h3><p id="p-16">Although TAG and WE are a more common storage lipid in several groups of bacteria, the majority of all bacteria store PHA rather than TAG or WE (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Rottig and Steinbuchel 2013</a>). Three major enzymes (PhaA, PhaB and PhaC) involved in the synthesis of PHAs, forming the classical synthesis pathway and two enzymes (intracellular PhaZ and extracellular PhaZ) involved in the utilization of PHAs in bacteria. In this study, we mainly focused on the distribution patterns of PHA synthesis pathway PhaABC. PhaC has been further divided into four classes depending on substrate specificities and subunit compositions, represented by <em>Cupriavidus necator</em> (class I), <em>Pseudomonas aeruginosa</em> (class II), <em>Allochromatium vinosum</em> (class III), and <em>Bacillus megaterium</em> (class IV) (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Rehm 2003</a>). However, based on Pfam analysis, the four classes of PhaC could be included into two groups, PhaC Group 1 with PFAM domain Abhydrolase_1 (PF00561) and Group 2 with PFAM domain PhaC_N (PF07167), which were also thoroughly investigated for their distribution patterns. However, PhaE and PhaR as PhaC subunits were not be considered in this study. On the other hand, there are another four confirmed pathways for PHAs synthesis, which are 1) FabG, 2) PhaJ, 3) FabD, 4) SucD, 4HbD and OrfZ (<a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Breuer 2010</a>). All the enzymes are screened for presence and absence in collected bacterial proteomes (Table S1).</p><p id="p-17">Preliminary analysis showed that 3166 bacterial species with average proteome size of 3772 proteins/proteome have PhaABC pathway while 1036 bacterial species with average proteome size of 1059 proteins/proteome do not have the pathway (<em>P-value</em> &lt; 0.001). In general, PHA synthesis was widely distributed across bacterial species. It is interesting to notice that Group II PHA synthase is dominantly present in <em>Proteobacteria</em> Phylum while Group I PHA synthase is abundantly present in <em>Actinobacteria</em> Phylum. In addition, three representative regions R1, R2, and R3 with loss of the classical PHA synthesis pathway were highlighted, that is, <em>Bacteroidia</em> Class, Epsilon-<em>Proteobacteria</em> Subdivision Class, and <em>Tenericutes</em> Phylum. In these regions, two bacterial species, <em>Lentimicrobium saccharophilum</em> and <em>Helicobacter</em> sp. 13S00477-4 actually harbored the complete PhaABC pathway. Distribution patterns of the other four PHA synthesis pathways mentioned above were not visualized along phylogenetic tree since they were distributed with no featured patterns (Table S1). Please refer to <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a> for the distribution patterns of the studied enzymes and the pathway.</p></div><div id="sec-10" class="subsection"><h3>Polyphosphate</h3><p id="p-18">Three key enzymes PPK1, PPK2, and PPX are related with polyP metabolism. PPK1 is responsible for polyP synthesis. In this study, a total of 5273 bacterial species have PPK1. PPK2 and PPX are used for intracellular and extracellular polyP degradation, respectively. 2584 bacterial species have PPK1, PPK2 and PPX enzymes while 2211 bacteria species do not have any of the three enzymes. The average proteome sizes of the two groups of bacteria are 4571 proteins/proteome and 1615 proteins/proteome, respectively, which are significantly different (<em>P-value</em> &lt; 0.001). In our analysis, we independently reviewed the distribution patterns of the three enzymes along phylogenetic trees and the result is displayed in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1C</a>. The three enzymes are widely distributed across bacterial species, which reflects the essentiality of the polymer in bacterial physiology. In addition, comparison shows that <em>Firmicutes</em> phylum seems to favor PPX more than PPK2 for polyP degradation. In addition, although it was observed that several regions had missing synthesis enzyme or degradation enzyme, only unclassified bacteria and <em>Mollicutes</em> class (94 bacterial species) showed apparent lack of the three polyP metabolism enzymes.</p></div><div id="sec-11" class="subsection"><h3>Glycogen</h3><p id="p-19">Glycogen metabolism in bacteria has multiple pathways, which include the classical pathway (GlgC, GlgA, GlgB, GlgP and GlgX) (<a id="xref-ref-33-6" class="xref-bibr" href="#ref-33">Wang and Wise 2011</a>), trehalose pathway (TreS, Pep2, GlgE, and GlgB), and the novel Rv3032 pathway (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Wang <em>et al.</em> 2019</a>). Rv3032 is an alternative enzyme for the elongation of α-1,4-glucan, which was compared for distribution patterns with GlgA. We also focused on the two glycogen synthesis pathways and compared their distribution patterns. In addition, there are two types of glycogen branching enzymes. One is the common bacterial GlgB, belonging to GH13 in CATH database, and the other one is known as archaeal GlgB, belonging to GH57 in CATH database (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Orengo <em>et al.</em> 1997</a>). We also looked into their distribution patterns in bacteria since GlgB is essential in determining the branched structure of glycogen. Our study showed that 3137 bacteria have the classical synthesis pathway (GlgC, GlgA, and GlgB) and their average proteome size is 3966 proteins/proteome while only 510 bacterial species (average proteome size of 1120 proteins/proteome) do not have these enzymes (<em>P-value</em> &lt; 0.001). Comparison of the two synthesis pathways confirmed that full classical glycogen synthesis pathway (yellow circle in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1D</a>) distributes widely in bacteria except for <em>Actinobacteria</em> phylum, FCB group and unclassified bacteria group. Random loss of the classical pathway can also be spotted. In contrast, trehalose pathway is tightly associated with <em>Actinobacteria</em> phylum. It was also shown that Rv3032 was actually more widespread than GlgA. As for the two GlgBs, GH13 GlgB is widely distributed in 4500 bacterial species with a trend of random loss, while GH57 GlgB is identified in only 785 bacterial species that are mainly fall into <em>Terrabacteria</em> Group and PVC group, <em>etc</em>. In addition, GH57 GlgB is also identified in PVC group, <em>Spirochaetes</em>, <em>Acidobacteria</em>, <em>Fusobacteria</em>, <em>Thermotogae</em>, <em>Nitrospirae</em>, <em>Aquificae</em>, <em>Synergistetes</em>, <em>Elusimicrobia</em>, <em>Nitrospinae/Tectomicrobia</em> group, <em>Thermodesulfobacteria</em>, <em>Rhodothermaeota</em>, and <em>Dictyoglomi</em>.</p></div></div><div class="section discussion" id="sec-12"><h2 class="">Discussion</h2><p id="p-20">From an evolutionary point of view, if an organism can obtain compounds from other sources, it would tend to discard the corresponding biosynthetic pathways (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Moran 2002</a>). For example, strictly intracellular bacteria such as <em>Rickettsia</em> species, <em>Mycoplasma</em> species, and <em>Buchnera</em>, <em>etc</em>. have extensively reduced genome sizes and common pathways relevant to energy metabolism have been eliminated (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Moran 2002</a>). Although a common belief is that organism should evolve toward high complexity, recent analysis reported that reduction and simplification could be the dominant mode of evolution while increasing complexity is just a transitional stage according to the neutral genetic loss and streamlining hypothesis (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Wolf and Koonin 2013</a>). Independent analyses of the distribution patterns of the five energy reserves in bacteria found a consistent and statistically significant correlation between energy reserve loss and reduced proteome size, as shown in the results. Previous studies also reported this correlation in terms of glycogen and polyP metabolism in bacteria (<a id="xref-ref-33-7" class="xref-bibr" href="#ref-33">Wang and Wise 2011</a>; <a id="xref-ref-34-6" class="xref-bibr" href="#ref-34">Wang <em>et al.</em> 2018</a>). In this study, we extended the conclusion by adding the neutral lipid reserves of WE, TAG, and PHA. It was also confirmed that bacteria losing energy reserve metabolism capacity tended to have a niche-dependent or host-dependent lifestyle (<a id="xref-ref-33-8" class="xref-bibr" href="#ref-33">Wang and Wise 2011</a>; <a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Wang <em>et al.</em> 2017</a>; <a id="xref-ref-34-7" class="xref-bibr" href="#ref-34">Wang <em>et al.</em> 2018</a>). Thus, by looking into bacterial energy reserve metabolism, we could obtain preliminary views in terms of bacterial lifestyles, though other evidence is required to verify this hypothesis. It is worth mentioning that proteomes that we used in this study were derived from translated coding genes, not based on condition-specific expression of proteins (<a id="xref-ref-29-3" class="xref-bibr" href="#ref-29">The UniProt Consortium 2017</a>). Thus, there is no bias in protein coverage or gene expression level. From this point of view, this equivalent to bacterial genome analysis, except for that HMM models perform much better for protein sequences than DNA sequences in terms of remote homolog identification. It is also worth mentioning that by comparing with other tools such as BLAST for ortholog identification, HMM provides more accuracy with less bias because it is a statistical sequence model built on the alignments of multiple homologous sequences (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Pearson and Eddy 2011</a>).</p><p id="p-21">WS/DGAT is a bifunctional enzyme and key to the biosynthesis of WE and TAG in bacteria. It was previously thought that WE and TAG are very uncommon lipid storage compounds in bacteria when compared with plants and animals, until this novel enzyme was identified (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Kalscheuer and Steinbüchel 2003</a>). From our analysis, it could be seen that many bacteria belonging to both Gram-positive and Gram-negative categories have the potential to synthesize WE and TAG. However, studies about WS/DGAT are mainly restricted to <em>Mycobacteria</em> genus (<em>Actinobacteria</em> phylum) and <em>Acinetobacter</em> genus (γ-Proteobacteria phylum) due to their clinical significance and potentially industrial use. WS/DGAT genes in the phylum <em>Actinobacteria</em> tend to have more paralogs than other phyla, especially for the bacteria in the R1 and R3 regions, which include <em>Mycobacteriaceae</em>, <em>Dietziaceae</em>, <em>Gordoniaceae</em>, <em>Nocardiaceae</em>, <em>Tsukamurellaceae</em>, <em>Williamsiaceae</em>, <em>Nocardioidaceae</em> and <em>Pseudonocardiales</em>. On the other hand, no WS/DGAT is found in the family of <em>Corynebacteriaceae</em> (R2 region), although <em>Corynebacteriaceae</em> is closely related with <em>Mycobacteriacea</em> (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Pukall 2014</a>). In addition, bacteria in phylum <em>Firmicutes</em> do not have any WS/DGAT enzymes. Screening of Phospholipid:diacylglycerol acyltransferase (PDAT), an enzyme that catalyses the acyl-CoA-independent formation of triacylglycerol in yeast and plants, found 46 homologs in bacteria (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Dahlqvist <em>et al.</em> 2000</a>). This contradicts that eukaryotic PDAT is exclusively present in higher organisms with no homologs in prokaryotic genomes (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Rottig and Steinbuchel 2013</a>).</p><p id="p-22">The family <em>Corynebacteriaceae</em> contains the genera <em>Corynebacterium</em> and monospecific genus <em>Turicella</em> (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Kämpfer 2010</a>). <em>Mycobacterium tuberculosis</em> is the dominant species in <em>Mycobacteriaceae</em> (97 species). Mycolic acid (MA), with wax ester as the oxidized form of MA in <em>Mycobacterium tuberculosis</em>, is present in the cell wall and plays essential roles in host invasion, environmental persistence, and also drug resistance (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Barkan <em>et al.</em> 2009</a>). In addition, <em>Mycobacterium tuberculosis</em> also relies on wax ester for dormancy, though specific functions of WE in <em>M. tuberculosis</em> require further investigation. Thus, abundance of WS/DGAT in <em>Mycobacteriacea</em> has selective advantages in evolution. Considering the abundance of wax ester and its slow degradation, it could also contribute to the long-term survival (more than 360 days) of <em>M. tuberculosis</em> in environment (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Wang <em>et al.</em> 2017</a>). On the other hand, <em>Corynebacterium</em> does not rely on oxidized mycolic acid while <em>Turicella</em> does not have mycolic acid at all (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Marrakchi <em>et al.</em> 2014</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Funke <em>et al.</em> 1994</a>). Thus, there is no need for them to be equipped with the WS/DGAT enzyme. However, how <em>Mycobacteriaceae</em> gains WS/DGAT, or <em>Corynebacteriaceae</em> loses it, is not clear and needs more investigation. As for <em>Firmicutes</em>, it is the low G+C counterpart of the high G+C <em>Actinobacteria</em>. Most of its species can form endospores and are resistant to extreme environmental conditions such as desiccation, temperature fluctuation, and nutrient deprivation, <em>etc</em>. (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Galperin 2013</a>). Thus, they may not need compounds such as WE or TAG for storing energy and dealing with harsh external conditions. How G+C content in the two phyla may impact on the gain or loss of wax ester metabolism is currently not known.</p><p id="p-23">PHAs are a group of compounds that include but are not limited to components such as polyhydroxybutyrate (PHB) and polyhydroxyvalerate (PHV), <em>etc</em>., among which PHB is the most common and most prominent member in bacteria (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Verlinden <em>et al.</em> 2007</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Uchino <em>et al.</em> 2007</a>). Currently, there are eight pathways responsible for the synthesis of PHB in bacteria (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Breuer 2010</a>). The complexity of the metabolic pathways in bacteria is worthy of a separate and complete investigation and is unable to be fully addressed in this study. Here, we only focused on the classical pathway that mainly involves PhaA, PhaB and PhaC (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Verlinden <em>et al.</em> 2007</a>). Its phylogenetic analysis revealed that PHA synthesis is widely distributed across bacterial species, which is consistent with its role as a dominant neutral lipid reserve in bacteria. As discussed above, PHA synthase is further divided into four classes. However, HMM analysis of representative sequences revealed that differences of the four-class enzymes at domain level only involves Abhydrolase_1 (PF00561) and PhaC_N (PF07167) domains. Further exploration of all bacterial PhaC domain structures shows higher heterogeneity, indicating that a more complex classification system for this enzyme should be introduced (L. Wang, unpublished data) and will be investigated in future study.</p><p id="p-24">PolyP is known to be ubiquitous in different life domains and claimed to be present in all types of cells in Nature due to its essential roles as energy and phosphate sources (<a id="xref-ref-34-8" class="xref-bibr" href="#ref-34">Wang <em>et al.</em> 2018</a>). Although a number of enzymes are directly linked with polyP metabolism, we only focused on PPK1, PPK2, and PPX in this study because they are most essential enzymes to polyP metabolism. <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1C</a> gives an overview of the distribution patterns of the three enzymes. Although 2212 bacterial species across the phylogenetic tree lack of all three enzymes, an apparent gap was only evident in the phylum <em>Tenericutes</em> and was further confirmed to be <em>Mollicutes</em>. A previous analysis of 944 bacterial proteomes showed that bacteria which have completely lost the polyP metabolic pathways (PPK1, PPK2, PAP, SurE, PPX, PpnK and PpgK) are heavily host-dependent and tend to adopt obligate intracellular or symbiotic lifestyles (<a id="xref-ref-34-9" class="xref-bibr" href="#ref-34">Wang <em>et al.</em> 2018</a>). Consistently, <em>Mollicutes</em> is a group of parasitic bacteria that have evolved from a common <em>Firmicutes</em> ancestor through reductive evolution (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Gojobori <em>et al.</em> 2014</a>). From here, we could infer that not only loss of complete metabolism pathway, but also even loss of key enzymes for energy reserve metabolism could give a hint at bacterial lifestyle.</p><p id="p-25">For glycogen metabolism, we compared two synthesis pathways, the classical pathway (GlgC, GlgA and GlgB) and the newly identified trehalose-related pathway (TreS, Pep2, GlgE and GlgB) (<a id="xref-ref-33-9" class="xref-bibr" href="#ref-33">Wang and Wise 2011</a>; <a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Chandra <em>et al.</em> 2011</a>). Although initial analysis via BLAST search showed in 2011 that GlgE pathway is represented in 14 % of sequenced genomes from diverse bacteria, our studies showed that, when searching for the complete GlgE pathway by including another three enzymes, it is dominantly restricted to <em>Actinobacteria</em> phylum while the classical pathway is widely present in the phylogenetic tree as seen in <a id="xref-fig-1-6" class="xref-fig" href="#F1">Figure 1D</a> (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Chandra <em>et al.</em> 2011</a>). In addition, the two types of GlgBs also showed interesting distribution patterns. Although GH13 GlgB is widely identified in 54.33% bacteria, GH57 GlgB is only present in 9.48% bacteria with skewed distribution in groups such as the <em>Terrabacteria</em> phylum and PVC group, <em>etc</em>. Another study of 427 archaea proteomes found that the 11 archaea have GH13 GlgB, while 18 archaea have GH57 GlgB (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Wang <em>et al.</em> 2019</a>). Thus, the two GlgBs are rarely present in archaea and mainly exist in bacteria. However, why trehalose-related glycogen metabolism pathway is markedly associated with <em>Actinobacteria</em> phylum still needs more experimental exploration.</p></div><div class="section conclusions" id="sec-13"><h2 class="">Conclusions</h2><p id="p-26">Distribution patterns of key enzymes and their combined pathways in bacteria provided a comprehensive view of how energy reserves are incorporated and lost. In general, polyP, PHA, and glycogen are widely distributed across bacterial species as energy storage compounds. The other two neutral lipids investigated in this study are comparatively minor energy reserves in bacteria and mainly found in the super phylum <em>Proteobacteria</em> and phylum <em>Actinobacteria</em>. Within the group, more bacteria have the capacity to accumulate WE and TAG due to the abundance of WS/DGAT homolog. Comparatively, polyP acts as a transient energy reserve while neutral lipids are a more sustainable energy provider (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Finkelstein <em>et al.</em> 2010</a>; <a id="xref-ref-33-10" class="xref-bibr" href="#ref-33">Wang and Wise 2011</a>). Thus, neutral lipids could be major players for bacterial persistence under harsh conditions such terrestrial and aquatic environments. As for glycogen, its ability to enhance bacterial environmental viability is still controversial. Its widespread distribution in bacteria indicates that its metabolism is tightly linked with bacterial essential activities. In sum, through this study, we obtained a much clearer picture about how key enzymes responsible for the metabolism of energy reserves are distributed in bacteria. Further investigation via incorporating bacterial physiology and lifestyle could supply additional explanations to illustrate the distribution patterns, although experimental evidence is indispensable to confirm the computational analysis.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-27">We thank Professor Ximiao He from Huazhong University of Science and Technology and the anonymous reviewers for genuine comments and suggestions, which greatly improves the quality of this study. The work was supported by the Startup Foundation for Excellent Researchers at Xuzhou Medical University [D2016007], Innovative and Entrepreneurial Talent Scheme of Jiangsu Province (2017), and Natural Science Foundation of Jiangsu Province [BK20180997].</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-28">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8138951">https://doi.org/10.25387/g3.8138951</a>.</p></li><li class="fn" id="fn-3"><p id="p-29"><em>Communicating editor: R. Kulathinal</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-03-01"><span class="received-label">Received </span>March 1, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-05-28"><span class="accepted-label">Accepted </span>May 28, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Wang <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.1" data-doi="10.2478/s11756-013-0324-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Achbergerová</span>,  <span class="cit-name-given-names">L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Nahálka</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">PPK1 and PPK2 — which polyphosphate kinase is older?</span> <abbr class="cit-jnl-abbrev">Biologia</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">263</span>–<span class="cit-lpage">269</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2478/s11756-013-0324-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiologia%26rft.volume%253D69%26rft.spage%253D263%26rft_id%253Dinfo%253Adoi%252F10.2478%252Fs11756-013-0324-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2478/s11756-013-0324-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.2" data-doi="10.1128/AEM.02638-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Arabolaza</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Rodriguez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Altabe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Alvarez</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Gramajo</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Multiple Pathways for Triacylglycerol Biosynthesis in Streptomyces coelicolor</span>. <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">74</span>: <span class="cit-fpage">2573</span>–<span class="cit-lpage">2582</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/AEM.02638-07</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.02638-07%26rft_id%253Dinfo%253Apmid%252F18310412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjk6Ijc0LzkvMjU3MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOC8yNDg5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.3" data-doi="10.1016/j.chembiol.2009.04.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barkan</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Sacchettini</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. S.</span> <span class="cit-name-surname">Glickman</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Mycolic Acid Cyclopropanation is Essential for Viability, Drug Resistance, and Cell Wall Integrity of Mycobacterium tuberculosis</span>. <abbr class="cit-jnl-abbrev">Chem. Biol.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">499</span>–<span class="cit-lpage">509</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.chembiol.2009.04.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChemistry%2B%2526%2Bbiology%26rft.stitle%253DChem%2BBiol%26rft.aulast%253DBarkan%26rft.auinit1%253DD.%26rft.volume%253D16%26rft.issue%253D5%26rft.spage%253D499%26rft.epage%253D509%26rft.atitle%253DMycolic%2Bacid%2Bcyclopropanation%2Bis%2Bessential%2Bfor%2Bviability%252C%2Bdrug%2Bresistance%252C%2Band%2Bcell%2Bwall%2Bintegrity%2Bof%2BMycobacterium%2Btuberculosis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chembiol.2009.04.001%26rft_id%253Dinfo%253Apmid%252F19477414%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chembiol.2009.04.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19477414&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2489.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-9.8.2489.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Breuer</span>,  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Plastics from Bacteria - Natural Functions and Applications. By Guo-Qiang Chen (Editor), Alexander Steinbüchel (Series Editor)</span>. <span class="cit-source">Biotechnol. J.</span> <span class="cit-vol">5</span>:<span class="cit-fpage">1351</span>–<span class="cit-lpage">1351</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechnol.%2BJ.%26rft.volume%253D5%26rft.spage%253D1351%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.5" data-doi="10.1099/mic.0.044263-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chandra</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. F.</span> <span class="cit-name-surname">Chater</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Bornemann</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Unexpected and widespread connections between bacterial glycogen and trehalose metabolism</span>. <abbr class="cit-jnl-abbrev">Microbiology</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1565</span>–<span class="cit-lpage">1572</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1099/mic.0.044263-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%26rft_id%253Dinfo%253Adoi%252F10.1099%252Fmic.0.044263-0%26rft_id%253Dinfo%253Apmid%252F21474533%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/mic.0.044263-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21474533&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2489.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292356200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.6" data-doi="10.1073/pnas.120067297"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dahlqvist</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Stahl</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lenman</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Banas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lee</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Phospholipid:diacylglycerol acyltransferase: An enzyme that catalyzes the acyl-CoA-independent formation of triacylglycerol in yeast and plants</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">6487</span>–<span class="cit-lpage">6492</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.120067297</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.120067297%26rft_id%253Dinfo%253Apmid%252F10829075%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiOTcvMTIvNjQ4NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOC8yNDg5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.7" data-doi="10.1093/nar/gkh340"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>,  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">1792</span>–<span class="cit-lpage">1797</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkh340</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DEdgar%26rft.auinit1%253DR.%2BC.%26rft.volume%253D32%26rft.issue%253D5%26rft.spage%253D1792%26rft.epage%253D1797%26rft.atitle%253DMUSCLE%253A%2Bmultiple%2Bsequence%2Balignment%2Bwith%2Bhigh%2Baccuracy%2Band%2Bhigh%2Bthroughput%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh340%26rft_id%253Dinfo%253Apmid%252F15034147%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkh340&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15034147&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2489.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000220487200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.8" data-doi="10.1093/bioinformatics/btq461"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Edgar</span>,  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Search and clustering orders of magnitude faster than BLAST</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">2460</span>–<span class="cit-lpage">2461</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btq461</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtq461%26rft_id%253Dinfo%253Apmid%252F20709691%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btq461&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20709691&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2489.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000282170000016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.9" data-doi="10.1093/nar/gkr1178"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Federhen</span>,  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The NCBI Taxonomy database</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">D136</span>–<span class="cit-lpage">D143</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkr1178</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkr1178%26rft_id%253Dinfo%253Apmid%252F22139910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkr1178&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22139910&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2489.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298601300021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.10" data-doi="10.1130/G30398.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Finkelstein</span>,  <span class="cit-name-given-names">D. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. C.</span> <span class="cit-name-surname">Brassell</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. M.</span> <span class="cit-name-surname">Pratt</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Microbial biosynthesis of wax esters during desiccation: Adaptation for colonization of the earliest terrestrial environments?</span> <abbr class="cit-jnl-abbrev">Geology</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">247</span>–<span class="cit-lpage">250</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1130/G30398.1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGeology%26rft.stitle%253DGeology%26rft.aulast%253DFinkelstein%26rft.auinit1%253DD.%2BB.%26rft.volume%253D38%26rft.issue%253D3%26rft.spage%253D247%26rft.epage%253D250%26rft.atitle%253DMicrobial%2Bbiosynthesis%2Bof%2Bwax%2Besters%2Bduring%2Bdesiccation%253A%2BAdaptation%2Bfor%2Bcolonization%2Bof%2Bthe%2Bearliest%2Bterrestrial%2Benvironments%253F%26rft_id%253Dinfo%253Adoi%252F10.1130%252FG30398.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZ2VvbG9neSI7czo1OiJyZXNpZCI7czo4OiIzOC8zLzI0NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOC8yNDg5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.11" data-doi="10.1099/00207713-44-2-270"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Funke</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Stubbs</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Altwegg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Carlotti</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D.</span> <span class="cit-name-surname">Collins</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Turicella otitidis gen. nov., sp. nov., a Coryneform Bacterium Isolated from Patients with Otitis Media</span>. <abbr class="cit-jnl-abbrev">Int. J. Syst. Bacteriol.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">270</span>–<span class="cit-lpage">273</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1099/00207713-44-2-270</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInternational%2BJournal%2Bof%2BSystematic%2Band%2BEvolutionary%2BMicrobiology%26rft.stitle%253DInt.%2BJ.%2BSyst.%2BEvol.%2BMicrobiol.%26rft.aulast%253DFUNKE%26rft.auinit1%253DG.%26rft.volume%253D44%26rft.issue%253D2%26rft.spage%253D270%26rft.epage%253D273%26rft.atitle%253DTuricella%2Botitidis%2Bgen.%2Bnov.%252C%2Bsp.%2Bnov.%252C%2Ba%2BCoryneform%2BBacterium%2BIsolated%2Bfrom%2BPatients%2Bwith%2BOtitis%2BMedia%26rft_id%253Dinfo%253Adoi%252F10.1099%252F00207713-44-2-270%26rft_id%253Dinfo%253Apmid%252F8186090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/00207713-44-2-270&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8186090&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2489.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.12" data-doi="10.1128/microbiolspectrum.TBS-0015-2012"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galperin</span>,  <span class="cit-name-given-names">M. Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome Diversity of Spore-Forming Firmicutes</span>. <abbr class="cit-jnl-abbrev">Microbiol. Spectr.</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">15</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/microbiolspectrum.TBS-0015-2012</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiol.%2BSpectr.%26rft.volume%253D1%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1128%252Fmicrobiolspectrum.TBS-0015-2012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/microbiolspectrum.TBS-0015-2012&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gojobori</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Grosjean</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Breton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Sirand-Pugnet</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Tardy</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Predicting the Minimal Translation Apparatus: Lessons from the Reductive Evolution of Mollicutes</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-elocation-id">e1004363</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.14" data-doi="10.1128/JB.186.21.7243-7253.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Handrick</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Reinhardt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Kimmig</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Jendrossek</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The “intracellular” poly(3-hydroxybutyrate) (PHB) depolymerase of Rhodospirillum rubrum is a periplasm-located protein with specificity for native PHB and with structural similarity to extracellular PHB depolymerases</span>. <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">7243</span>–<span class="cit-lpage">7253</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/JB.186.21.7243-7253.2004</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBacteriology%26rft.stitle%253DJ.%2BBacteriol.%26rft.aulast%253DHandrick%26rft.auinit1%253DR.%26rft.volume%253D186%26rft.issue%253D21%26rft.spage%253D7243%26rft.epage%253D7253%26rft.atitle%253DThe%2B%2522Intracellular%2522%2BPoly%25283-Hydroxybutyrate%2529%2B%2528PHB%2529%2BDepolymerase%2Bof%2BRhodospirillum%2Brubrum%2BIs%2Ba%2BPeriplasm-Located%2BProtein%2Bwith%2BSpecificity%2Bfor%2BNative%2BPHB%2Band%2Bwith%2BStructural%2BSimilarity%2Bto%2BExtracellular%2BPHB%2BDepolymerases%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.186.21.7243-7253.2004%26rft_id%253Dinfo%253Apmid%252F15489436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE4Ni8yMS83MjQzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI0ODkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.15" data-doi="10.1016/S0168-9525(02)02734-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henrissat</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Deleury</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Coutinho</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Glycogen metabolism loss: a common marker of parasitic behaviour in bacteria?</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">440</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0168-9525(02)02734-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DHenrissat%26rft.auinit1%253DB.%26rft.volume%253D18%26rft.issue%253D9%26rft.spage%253D437%26rft.epage%253D440%26rft.atitle%253DGlycogen%2Bmetabolism%2Bloss%253A%2Ba%2Bcommon%2Bmarker%2Bof%2Bparasitic%2Bbehaviour%2Bin%2Bbacteria%253F%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0168-9525%252802%252902734-8%26rft_id%253Dinfo%253Apmid%252F12175798%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0168-9525(02)02734-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12175798&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2489.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.16" data-doi="10.1074/jbc.M210533200"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kalscheuer</span>,  <span class="cit-name-given-names">R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Steinbüchel</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A Novel Bifunctional Wax Ester Synthase/Acyl-CoA:Diacylglycerol Acyltransferase Mediates Wax Ester and Triacylglycerol Biosynthesis inAcinetobacter calcoaceticusADP1</span>. <abbr class="cit-jnl-abbrev">J. Biol. Chem.</abbr> <span class="cit-vol">278</span>: <span class="cit-fpage">8075</span>–<span class="cit-lpage">8082</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1074/jbc.M210533200</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBiol.%2BChem.%26rft_id%253Dinfo%253Adoi%252F10.1074%252Fjbc.M210533200%26rft_id%253Dinfo%253Apmid%252F12502715%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamJjIjtzOjU6InJlc2lkIjtzOjExOiIyNzgvMTAvODA3NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOC8yNDg5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-book" id="cit-9.8.2489.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kämpfer</span>,  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Actinobacteria</span>, pp. <span class="cit-fpage">1819</span>–<span class="cit-lpage">1838</span> in <span class="cit-source">Handbook of Hydrocarbon and Lipid Microbiology</span>, Springer, Berlin, Heidelberg.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.18" data-doi="10.1093/nar/gkw290"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Letunic</span>,  <span class="cit-name-given-names">I.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Bork</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Interactive tree of life (iTOL) v3: an online tool for the display and annotation of phylogenetic and other trees</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">W242</span>–<span class="cit-lpage">W245</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkw290</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw290%26rft_id%253Dinfo%253Apmid%252F27095192%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw290&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27095192&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2489.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.19" data-doi="10.1016/j.chembiol.2013.11.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marrakchi</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.-A.</span> <span class="cit-name-surname">Lanéelle</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Daffé</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mycolic Acids: Structures, Biosynthesis, and Beyond</span>. <abbr class="cit-jnl-abbrev">Chem. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">67</span>–<span class="cit-lpage">85</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.chembiol.2013.11.011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChem.%2BBiol.%26rft.volume%253D21%26rft.spage%253D67%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chembiol.2013.11.011%26rft_id%253Dinfo%253Apmid%252F24374164%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chembiol.2013.11.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24374164&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2489.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.20" data-doi="10.1093/nar/gkh435"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McGinnis</span>,  <span class="cit-name-given-names">S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.L.</span> <span class="cit-name-surname">Madden</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> BLAST: at the core of a powerful and diverse set of sequence analysis tools. <em>Nucleic Acids Research</em> 32 (Web Server):W20–W25.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkh435</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.volume%253D32%26rft.spage%253D20%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkh435%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkh435&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.21" data-doi="10.1016/S0092-8674(02)00665-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moran</span>,  <span class="cit-name-given-names">N. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Microbial Minimalism</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">583</span>–<span class="cit-lpage">586</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0092-8674(02)00665-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMoran%26rft.auinit1%253DN.%2BA.%26rft.volume%253D108%26rft.issue%253D5%26rft.spage%253D583%26rft.epage%253D586%26rft.atitle%253DMicrobial%2Bminimalism%253A%2Bgenome%2Breduction%2Bin%2Bbacterial%2Bpathogens.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252802%252900665-7%26rft_id%253Dinfo%253Apmid%252F11893328%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(02)00665-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11893328&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2489.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174314800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.22" data-doi="10.1016/S0969-2126(97)00260-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Orengo</span>,  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. D.</span> <span class="cit-name-surname">Michie</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Jones</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. T.</span> <span class="cit-name-surname">Jones</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. B.</span> <span class="cit-name-surname">Swindells</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">CATH – a hierarchic classification of protein domain structures</span>. <abbr class="cit-jnl-abbrev">Structure</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1093</span>–<span class="cit-lpage">1109</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/S0969-2126(97)00260-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStructure%2B%2528London%2529%26rft.stitle%253DStructure%2B%2528London%2529%26rft.aulast%253DOrengo%26rft.auinit1%253DC.%2BA.%26rft.volume%253D5%26rft.issue%253D8%26rft.spage%253D1093%26rft.epage%253D1108%26rft.atitle%253DCATH--a%2Bhierarchic%2Bclassification%2Bof%2Bprotein%2Bdomain%2Bstructures.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0969-2126%252897%252900260-8%26rft_id%253Dinfo%253Apmid%252F9309224%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0969-2126(97)00260-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9309224&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2489.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.23" data-doi="10.1371/journal.pcbi.1001120"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pearson</span>,  <span class="cit-name-given-names">W. R.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. R.</span> <span class="cit-name-surname">Eddy</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Accelerated Profile HMM Searches</span>. <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">16</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1001120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1001120&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-book" id="cit-9.8.2489.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pukall</span>,  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">The Family Dietziaceae</span>, pp. <span class="cit-fpage">327</span>–<span class="cit-lpage">338</span> in <span class="cit-source">The Prokaryotes</span>, Springer-Verlag, Berlin, Heidelberg.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.25" data-doi="10.1042/bj20031254"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rehm</span>,  <span class="cit-name-given-names">B. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Polyester synthases: natural catalysts for plastics</span>. <abbr class="cit-jnl-abbrev">Biochem. J.</abbr> <span class="cit-vol">376</span>: <span class="cit-fpage">15</span>–<span class="cit-lpage">33</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1042/bj20031254</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochem.%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1042%252Fbj20031254%26rft_id%253Dinfo%253Apmid%252F12954080%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InBwYmlvY2hlbWoiO3M6NToicmVzaWQiO3M6ODoiMzc2LzEvMTUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzgvMjQ4OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.26" data-doi="10.1128/MMBR.00010-13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rottig</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Steinbuchel</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Acyltransferases in Bacteria</span>. <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">277</span>–<span class="cit-lpage">321</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/MMBR.00010-13</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DRo%2526%2523x0308%253Bttig%26rft.auinit1%253DA.%26rft.volume%253D77%26rft.issue%253D2%26rft.spage%253D277%26rft.epage%253D321%26rft.atitle%253DAcyltransferases%2Bin%2BBacteria%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMMBR.00010-13%26rft_id%253Dinfo%253Apmid%252F23699259%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI3Ny8yLzI3NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOC8yNDg5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.27" data-doi="10.1128/AEM.01719-16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Röttig</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. S.</span> <span class="cit-name-surname">Strittmatter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Schauer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Hiessl</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Poehlein</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Role of Wax Ester Synthase/Acyl Coenzyme A:Diacylglycerol Acyltransferase in Oleaginous Streptomyces sp. Strain G25</span>. <abbr class="cit-jnl-abbrev">Appl. Environ. Microbiol.</abbr> <span class="cit-vol">82</span>: <span class="cit-fpage">5969</span>–<span class="cit-lpage">5981</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/AEM.01719-16</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAppl.%2BEnviron.%2BMicrobiol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FAEM.01719-16%26rft_id%253Dinfo%253Apmid%252F27474711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYWVtIjtzOjU6InJlc2lkIjtzOjEwOiI4Mi8xOS81OTY5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI0ODkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.28" data-doi="10.1128/JB.00752-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Uchino</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Saito</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Gebauer</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Jendrossek</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Isolated Poly(3-Hydroxybutyrate) (PHB) Granules Are Complex Bacterial Organelles Catalyzing Formation of PHB from Acetyl Coenzyme A (CoA) and Degradation of PHB to Acetyl-CoA</span>. <abbr class="cit-jnl-abbrev">J. Bacteriol.</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">8250</span>–<span class="cit-lpage">8256</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1128/JB.00752-07</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BBacteriol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJB.00752-07%26rft_id%253Dinfo%253Apmid%252F17720797%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MjoiamIiO3M6NToicmVzaWQiO3M6MTE6IjE4OS8yMi84MjUwIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI0ODkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.29" data-doi="10.1093/nar/gkw1099"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">The UniProt Consortium</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">UniProt: the universal protein knowledgebase</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">D158</span>–<span class="cit-lpage">D169</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkw1099</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw1099%26rft_id%253Dinfo%253Apmid%252F27899622%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkw1099&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=27899622&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2489.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.30" data-doi="10.1111/j.1365-2672.2007.03335.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Verlinden</span>,  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Hill</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Kenward</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. D.</span> <span class="cit-name-surname">Williams</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Radecka</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Bacterial synthesis of biodegradable polyhydroxyalkanoates</span>. <abbr class="cit-jnl-abbrev">J. Appl. Microbiol.</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">1437</span>–<span class="cit-lpage">1449</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1365-2672.2007.03335.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BApplied%2BMicrobiology%2B%2528Print%2529%26rft.stitle%253DJournal%2Bof%2BApplied%2BMicrobiology%2B%2528Print%2529%26rft.aulast%253DVerlinden%26rft.auinit1%253DR.%2BA.%26rft.volume%253D102%26rft.issue%253D6%26rft.spage%253D1437%26rft.epage%253D1449%26rft.atitle%253DBacterial%2Bsynthesis%2Bof%2Bbiodegradable%2Bpolyhydroxyalkanoates.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-2672.2007.03335.x%26rft_id%253Dinfo%253Apmid%252F17578408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-2672.2007.03335.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17578408&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2489.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000247375200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.31" data-doi="10.1038/s41598-018-37768-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Wu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Huang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Wise</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span> <span class="cit-article-title">Bioinformatics Analysis of Metabolism Pathways of Archaeal Energy Reserves</span>. <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">1034</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/s41598-018-37768-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D9%26rft.spage%253D1034%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fs41598-018-37768-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/s41598-018-37768-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.32" data-doi="10.3389/fmicb.2017.02167"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Dai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Yan</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Wise</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The Sit-and-Wait Hypothesis in Bacterial Pathogens: A Theoretical Study of Durability and Virulence</span>. <abbr class="cit-jnl-abbrev">Front. Microbiol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">2167</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fmicb.2017.02167</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BMicrobiol.%26rft.volume%253D8%26rft.spage%253D2167%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffmicb.2017.02167%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fmicb.2017.02167&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.33" data-doi="10.1007/s00114-011-0832-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Wise</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Glycogen with short average chain length enhances bacterial durability</span>. <abbr class="cit-jnl-abbrev">Naturwissenschaften</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">719</span>–<span class="cit-lpage">729</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00114-011-0832-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNaturwissenschaften%26rft.stitle%253DNaturwissenschaften%26rft.aulast%253DWang%26rft.auinit1%253DL.%26rft.volume%253D98%26rft.issue%253D9%26rft.spage%253D719%26rft.epage%253D729%26rft.atitle%253DGlycogen%2Bwith%2Bshort%2Baverage%2Bchain%2Blength%2Benhances%2Bbacterial%2Bdurability.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00114-011-0832-x%26rft_id%253Dinfo%253Apmid%252F21808975%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00114-011-0832-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21808975&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2489.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.34" data-doi="10.3389/fmicb.2018.00782"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Yan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Wise</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Asenso</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Distribution Patterns of Polyphosphate Metabolism Pathway and Its Relationships With Bacterial Durability and Virulence</span>. <abbr class="cit-jnl-abbrev">Front. Microbiol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">782</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fmicb.2018.00782</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BMicrobiol.%26rft.volume%253D9%26rft.spage%253D782%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffmicb.2018.00782%26rft_id%253Dinfo%253Apmid%252F29755430%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fmicb.2018.00782&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=29755430&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2489.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.35" data-doi="10.1093/bioinformatics/btp033"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waterhouse</span>,  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Procter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. M. A.</span> <span class="cit-name-surname">Martin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Clamp</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. J.</span> <span class="cit-name-surname">Barton</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Jalview Version 2–a multiple sequence alignment editor and analysis workbench</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1189</span>–<span class="cit-lpage">1191</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp033</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp033%26rft_id%253Dinfo%253Apmid%252F19151095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp033&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19151095&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2489.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000265523300016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.36" data-doi="10.1099/mic.0.075366-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Whitehead</span>,  <span class="cit-name-given-names">M.P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Eagles</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Hooley</span></span>,and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.R.W.</span> <span class="cit-name-surname">Brown</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> Most bacteria synthesize polyphosphate by unknown mechanisms. <em>Microbiology</em> 160 (Pt_5):829–831.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1099/mic.0.075366-0</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft_id%253Dinfo%253Adoi%252F10.1099%252Fmic.0.075366-0%26rft_id%253Dinfo%253Apmid%252F24600028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1099/mic.0.075366-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24600028&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2489.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.37" data-doi="10.1016/0014-4827(59)90237-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wilkinson</span>,  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">1959</span> <span class="cit-article-title">The problem of energy-storage compounds in bacteria</span>. <abbr class="cit-jnl-abbrev">Exp. Cell Res.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">130</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/0014-4827(59)90237-X</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DExp.%2BCell%2BRes.%26rft.volume%253D7%26rft.spage%253D111%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0014-4827%252859%252990237-X%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0014-4827(59)90237-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.38" data-doi="10.1111/j.1096-0031.2009.00292.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wise</span>,  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">No so HoT - heads or tails is not able to reliably compare multiple sequence alignments</span>. <abbr class="cit-jnl-abbrev">Cladistics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">438</span>–<span class="cit-lpage">443</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1096-0031.2009.00292.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCladistics%26rft.volume%253D26%26rft.spage%253D438%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1096-0031.2009.00292.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1096-0031.2009.00292.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000279718000006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.39" data-doi="10.1002/bies.201300037"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wolf</span>,  <span class="cit-name-given-names">Y. I.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. V.</span> <span class="cit-name-surname">Koonin</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genome reduction as the dominant mode of evolution</span>. <abbr class="cit-jnl-abbrev">BioEssays</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">829</span>–<span class="cit-lpage">837</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/bies.201300037</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioEssays%26rft.volume%253D35%26rft.spage%253D829%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fbies.201300037%26rft_id%253Dinfo%253Apmid%252F23801028%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/bies.201300037&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23801028&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2489.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000327793500013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.40" data-doi="10.1073/pnas.262655199"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Ishige</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Kornberg</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A polyphosphate kinase (PPK2) widely conserved in bacteria</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">16678</span>–<span class="cit-lpage">16683</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.262655199</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.262655199%26rft_id%253Dinfo%253Apmid%252F12486232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiOTkvMjYvMTY2NzgiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzgvMjQ4OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2489.41" data-doi="10.1038/ismej.2015.251"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Ibrahim</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Cui</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Xie</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Jin</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Multi-omics analysis of niche specificity provides new insights into ecological adaptation in bacteria</span>. <abbr class="cit-jnl-abbrev">ISME J.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">2072</span>–<span class="cit-lpage">2075</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ismej.2015.251</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DISME%2BJ.%26rft.volume%253D10%26rft.spage%253D2072%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fismej.2015.251%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ismej.2015.251&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/8/2489.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/8/2477" title="Protein Kinase D Is Dispensable for Development and Survival of Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/8/2497" title="Structural Characterization of ABCB1, the Gene Underlying the d2 Dwarf Phenotype in Pearl Millet, Cenchrus Americanus (L.) Morrone" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/8"><span>Volume 9 Issue 8, August 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/8.cover-source.jpg?itok=cSNYBBSf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/8/2489" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-YziTL0d3N2J4r8DzloHBIjIDaFOfb9K6FR8RqMmTe1s" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/8/2489" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/8/2489" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/8/2489" class="active" data-icon-position="" data-hide-link-title="0">Systematic Analysis of Metabolic Pathway Distributions of Bacterial Energy Reserves</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/493756" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-kJWVgwnc9W6E3vRiUfu0Tt0XJkwTgxCsewhqfAWFygo" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/8/2489.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/8/2489" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-wRRaDJqM_gE-iZtEQ-PsvuTgqn1GUCkNcek5v7qiu5k" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/8/2489" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/8/2489.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493756" id="node493756--2" data-pisa="ggg;9/8/2489" data-pisa-master="ggg;g3.119.400123" data-apath="/ggg/9/8/2489.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Systematic Analysis of Metabolic Pathway Distributions of Bacterial Energy Reserves</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5339-7484" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Liang Wang</span>, <span class="highwire-citation-author" data-delta="1">Jianye Yang</span>, <span class="highwire-citation-author" data-delta="2">Yue Huang</span>, <span class="highwire-citation-author" data-delta="3">Qinghua Liu</span>, <span class="highwire-citation-author" data-delta="4">Yaping Xu</span>, <span class="highwire-citation-author" data-delta="5">Xue Piao</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-6559-4303" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michael J. Wise</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2489-2496;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400123 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493756--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Liang Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Bioinformatics, School of Medical Informatics,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Jiangsu Key Lab of New Drug Research and Clinical Pharmacy, Xuzhou Medical University, Xuzhou, Jiangsu Province, 221000, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Liang%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALiang%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5339-7484" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Liang Wang</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:leonwang@xzhmu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">leonwang@xzhmu.edu.cn</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jianye Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Bioinformatics, School of Medical Informatics,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jianye%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJianye%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yue Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Bioinformatics, School of Medical Informatics,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yue%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYue%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Qinghua Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Jiangsu Key Lab of New Drug Research and Clinical Pharmacy, Xuzhou Medical University, Xuzhou, Jiangsu Province, 221000, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qinghua%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQinghua%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Yaping Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Bioinformatics, School of Medical Informatics,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yaping%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYaping%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Xue Piao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Bioinformatics, School of Medical Informatics,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Information and Control Engineering, China University of Mining and Technology, Xuzhou, Jiangsu, 221116, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xue%2BPiao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Piao%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXue%2BPiao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Michael J. Wise </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Computer Science and Software Engineering, and</div><div class='author-affiliation'><span class='nlm-sup'>**</span>The Marshall Centre for Infectious Diseases Research and Training, University of Western Australia, Perth 6009, WA, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BJ.%2BWise%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wise%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BJ.%2BWise%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6559-4303" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael J. Wise</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/8/2489" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/8/2489&amp;title=Systematic%20Analysis%20of%20Metabolic%20Pathway%20Distributions%20of%20Bacterial%20Energy%20Reserves" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/8/2489&amp;title=Systematic%20Analysis%20of%20Metabolic%20Pathway%20Distributions%20of%20Bacterial%20Energy%20Reserves" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/8/2489&amp;title=Systematic%20Analysis%20of%20Metabolic%20Pathway%20Distributions%20of%20Bacterial%20Energy%20Reserves" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/8/2489&amp;text=Systematic%20Analysis%20of%20Metabolic%20Pathway%20Distributions%20of%20Bacterial%20Energy%20Reserves" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/8/2489&amp;title=Systematic%20Analysis%20of%20Metabolic%20Pathway%20Distributions%20of%20Bacterial%20Energy%20Reserves" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/8/2489&amp;t=Systematic%20Analysis%20of%20Metabolic%20Pathway%20Distributions%20of%20Bacterial%20Energy%20Reserves" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/8/2489&amp;title=Systematic%20Analysis%20of%20Metabolic%20Pathway%20Distributions%20of%20Bacterial%20Energy%20Reserves" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/8/2489&amp;title=Systematic%20Analysis%20of%20Metabolic%20Pathway%20Distributions%20of%20Bacterial%20Energy%20Reserves" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/493756/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493756" id="node493756--3" data-pisa="ggg;9/8/2489" data-pisa-master="ggg;g3.119.400123" data-apath="/ggg/9/8/2489.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Systematic Analysis of Metabolic Pathway Distributions of Bacterial Energy Reserves</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-5339-7484" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Liang Wang</span>, <span class="highwire-citation-author" data-delta="1">Jianye Yang</span>, <span class="highwire-citation-author" data-delta="2">Yue Huang</span>, <span class="highwire-citation-author" data-delta="3">Qinghua Liu</span>, <span class="highwire-citation-author" data-delta="4">Yaping Xu</span>, <span class="highwire-citation-author" data-delta="5">Xue Piao</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0001-6559-4303" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michael J. Wise</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2489-2496;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400123 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493756--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Liang Wang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Bioinformatics, School of Medical Informatics,</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Jiangsu Key Lab of New Drug Research and Clinical Pharmacy, Xuzhou Medical University, Xuzhou, Jiangsu Province, 221000, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Liang%2BWang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wang%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ALiang%2BWang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-5339-7484" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Liang Wang</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:leonwang@xzhmu.edu.cn" class="" data-icon-position="" data-hide-link-title="0">leonwang@xzhmu.edu.cn</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Jianye Yang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Bioinformatics, School of Medical Informatics,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jianye%2BYang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Yang%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJianye%2BYang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Yue Huang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Bioinformatics, School of Medical Informatics,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yue%2BHuang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Huang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYue%2BHuang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Qinghua Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Jiangsu Key Lab of New Drug Research and Clinical Pharmacy, Xuzhou Medical University, Xuzhou, Jiangsu Province, 221000, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Qinghua%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20Q&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AQinghua%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Yaping Xu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Bioinformatics, School of Medical Informatics,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yaping%2BXu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYaping%2BXu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Xue Piao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Bioinformatics, School of Medical Informatics,</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>School of Information and Control Engineering, China University of Mining and Technology, Xuzhou, Jiangsu, 221116, China,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xue%2BPiao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Piao%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXue%2BPiao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Michael J. Wise </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Computer Science and Software Engineering, and</div><div class='author-affiliation'><span class='nlm-sup'>**</span>The Marshall Centre for Infectious Diseases Research and Training, University of Western Australia, Perth 6009, WA, Australia</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BJ.%2BWise%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wise%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BJ.%2BWise%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-6559-4303" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael J. Wise</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2489&amp;rft.epage=2496&amp;rft.atitle=Systematic%20Analysis%20of%20Metabolic%20Pathway%20Distributions%20of%20Bacterial%20Energy%20Reserves&amp;rft.volume=9&amp;rft.issue=8&amp;rft.date=2019-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Wang%2C+Liang&amp;rft.au=Yang%2C+Jianye&amp;rft.au=Huang%2C+Yue&amp;rft.au=Liu%2C+Qinghua&amp;rft.au=Xu%2C+Yaping&amp;rft.au=Piao%2C+Xue&amp;rft.au=Wise%2C+Michael+J."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/493756/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/493756/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/493756/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/493756/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/493756/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/493756/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/493756/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/493756/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/493756/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/493756/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/493756/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/493756/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/8/2489" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/8/2489.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/8/2489.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/8/2489.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
