<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/8/2739" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/8/2739.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/8/2739.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/8/2739.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/8/2739/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/8/2739.atom" />
<meta name="HW.pisa" content="ggg;9/8/2739" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic Prediction of Additive and Non-additive Effects Using Genetic Markers and Pedigrees" />
<meta name="DC.Identifier" content="10.1534/g3.119.201004" />
<meta name="DC.Date" content="2019-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 de Almeida Filho et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The genetic merit of individuals can be estimated using models with dense markers and pedigree information. Early genomic models accounted only for additive effects. However, the prediction of non-additive effects is important for different forest breeding systems where the whole genotypic value can be captured through clonal propagation. In this study, we evaluated the integration of marker data with pedigree information, in models that included or ignored non-additive effects. We tested the models Reproducing Kernel Hilbert Spaces (RKHS) and BayesA, with additive and additive-dominance frameworks. Model performance was assessed for the traits tree height, diameter at breast height and rust resistance, measured in 923 pine individuals from a structured population of 71 full-sib families. We have also simulated a population with similar genetic properties and evaluated the performance of models for six simulated traits with distinct genetic architectures. Different cross validation strategies were evaluated, and highest accuracies were achieved using within family cross validation. The inclusion of pedigree information in genomic prediction models did not yield higher accuracies. The different RKHS models resulted in similar predictions accuracies, and RKHS and BayesA generated substantially better predictions than pedigree-only models. The additive-BayesA resulted in higher accuracies than RKHS for rust incidence and in simulated additive-oligogenic traits. For DBH, HT and additive-dominance polygenic traits, the RKHS- based models showed slightly higher accuracies than BayesA. Our results indicate that BayesA performs the best for traits with few genes with major effects, while RKHS based models can best predict genotypic effects for clonal selection of complex traits." />
<meta name="DC.Contributor" content="Janeo Eustáquio de Almeida Filho" />
<meta name="DC.Contributor" content="João Filipi Rodrigues Guimarães" />
<meta name="DC.Contributor" content="Fabyano Fonsceca e Silva" />
<meta name="DC.Contributor" content="Marcos Deon Vilela de Resende" />
<meta name="DC.Contributor" content="Patricio Muñoz" />
<meta name="DC.Contributor" content="Matias Kirst" />
<meta name="DC.Contributor" content="Marcio Fernando Ribeiro de Resende Júnior" />
<meta name="article:published_time" content="2019-08-01" />
<meta name="article:section" content="Genomic Prediction" />
<meta name="citation_title" content="Genomic Prediction of Additive and Non-additive Effects Using Genetic Markers and Pedigrees" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The genetic merit of individuals can be estimated using models with dense markers and pedigree information. Early genomic models accounted only for additive effects. However, the prediction of non-additive effects is important for different forest breeding systems where the whole genotypic value can be captured through clonal propagation. In this study, we evaluated the integration of marker data with pedigree information, in models that included or ignored non-additive effects. We tested the models Reproducing Kernel Hilbert Spaces (RKHS) and BayesA, with additive and additive-dominance frameworks. Model performance was assessed for the traits tree height, diameter at breast height and rust resistance, measured in 923 pine individuals from a structured population of 71 full-sib families. We have also simulated a population with similar genetic properties and evaluated the performance of models for six simulated traits with distinct genetic architectures. Different cross validation strategies were evaluated, and highest accuracies were achieved using within family cross validation. The inclusion of pedigree information in genomic prediction models did not yield higher accuracies. The different RKHS models resulted in similar predictions accuracies, and RKHS and BayesA generated substantially better predictions than pedigree-only models. The additive-BayesA resulted in higher accuracies than RKHS for rust incidence and in simulated additive-oligogenic traits. For DBH, HT and additive-dominance polygenic traits, the RKHS- based models showed slightly higher accuracies than BayesA. Our results indicate that BayesA performs the best for traits with few genes with major effects, while RKHS based models can best predict genotypic effects for clonal selection of complex traits.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/08/01" />
<meta name="citation_mjid" content="ggg;9/8/2739" />
<meta name="citation_id" content="9/8/2739" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/8/2739" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/8/2739.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/8/2739.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/8/2739.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.201004" />
<meta name="citation_pmid" content="31263059" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="10" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Genomic Prediction" />
<meta name="citation_firstpage" content="2739" />
<meta name="citation_lastpage" content="2748" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Janeo Eustáquio de Almeida Filho" />
<meta name="citation_author_institution" content="Genetic Plant Improvenent Lab, Norte Fluminense “Darcy Ribeiro” State University, Campos dos Goytacazes RJ, 28013-602, Brazil," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3906-8884" />
<meta name="citation_author" content="João Filipi Rodrigues Guimarães" />
<meta name="citation_author_institution" content="Futuragene Ltda, Itapetininga SP, 18207-780, Brazil," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3849-9834" />
<meta name="citation_author" content="Fabyano Fonsceca e Silva" />
<meta name="citation_author_institution" content="Animal Science Department, Federal University of Viçosa, Viçosa, MG 36570-000, Brazil," />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9536-1113" />
<meta name="citation_author" content="Marcos Deon Vilela de Resende" />
<meta name="citation_author_institution" content="EMBRAPA Forestry, Colombo, PR 83411-000, Brazil / Statistics Department, Federal University of Viçosa, Viçosa, MG 36570-000, Brazil," />
<meta name="citation_author" content="Patricio Muñoz" />
<meta name="citation_author_institution" content="Horticultural Sciences Department, University of Florida, Gainesville, FL 32611, and" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-8973-9351" />
<meta name="citation_author" content="Matias Kirst" />
<meta name="citation_author_institution" content="University of Florida Genetics Institute, School of Forest Resources and Conservation, Gainesville, FL 32611" />
<meta name="citation_author" content="Marcio Fernando Ribeiro de Resende Júnior" />
<meta name="citation_author_institution" content="Horticultural Sciences Department, University of Florida, Gainesville, FL 32611, and" />
<meta name="citation_author_email" content="mresende@ufl.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-2367-0766" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=T. Albrecht;citation_author=H.-J. Auinger;citation_author=V. Wimmer;citation_author=O. J. Ogutu;citation_author=C. Knaak;citation_title=Genome-based prediction of maize hybrid performance across genetic groups, testers, locations, and years;citation_pages=1375-1386;citation_volume=127;citation_year=2014;citation_pmid=24723140;citation_doi=10.1007/s00122-014-2305-z" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. E. de Almeida Filho;citation_author=J. F. R. Guimarães;citation_author=F. F. Silva;citation_author=M. D. V. De Resende;citation_author=P. Muñoz;citation_title=The contribution of dominance to phenotype prediction in a pine breeding and simulated population;citation_pages=33-41;citation_volume=117;citation_year=2016;citation_doi=10.1038/hdy.2016.23" />
<meta name="citation_reference" content="citation_journal_title=Can. J. For. Res.;citation_author=B. S. Baltunis;citation_author=D. A. Huber;citation_author=T. L. White;citation_author=B. Goldfarb;citation_author=H. E. Stelzer;citation_title=Genetic analysis of early field growth of loblolly pine clones and seedlings from the same full-sib families;citation_pages=195-205;citation_volume=37;citation_year=2007;citation_doi=10.1139/x06-203" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. Beaulieu;citation_author=T. Doerksen;citation_author=S. Clément;citation_author=J. Mackay;citation_author=J. Bousquet;citation_title=Accuracy of genomic selection models in a large population of open-pollinated families in white spruce;citation_pages=343-352;citation_volume=113;citation_year=2014;citation_pmid=24781808;citation_doi=10.1038/hdy.2014.36" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. Beaulieu;citation_author=T. K. Doerksen;citation_author=J. MacKay;citation_author=A. Rainville;citation_author=J. Bousquet;citation_title=Genomic selection accuracies within and between environments and small breeding groups in white spruce;citation_pages=1048;citation_volume=15;citation_year=2014;citation_pmid=25442968;citation_doi=10.1186/1471-2164-15-1048" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. Bouvet;citation_author=G. Makouanzi;citation_author=D. Cros;citation_author=P. Vigneron;citation_title=Modeling additive and non-additive effects in a hybrid population using genome-wide genotyping : prediction accuracy implications;citation_pages=146-157;citation_volume=116;citation_year=2016;citation_doi=10.1038/hdy.2015.78" />
<meta name="citation_reference" content="citation_journal_abbrev=J Anim Breed Genet;citation_author=M. P. L. Calus;citation_author=R. F. Veerkamp;citation_title=Accuracy of breeding values when using and ignoring the polygenic effect in genomic breeding value estimation with a marker density of one SNP per cM.;citation_pages=362-368;citation_volume=124;citation_year=2007;citation_issue=6;citation_pmid=18076473;citation_doi=10.1111/j.1439-0388.2007.00691.x" />
<meta name="citation_reference" content="citation_journal_title=Genet.;citation_author=J. Crossa;citation_author=Y. Beyene;citation_author=S. Kassa;citation_author=P. Pérez;citation_author=J. M. Hickey;citation_title=Genomic prediction in maize breeding populations with genotyping-by-sequencing. G3 Genes, Genomes;citation_pages=1903-1926;citation_volume=3;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. Crossa;citation_author=L. Campos Gde;citation_author=P. Pérez;citation_author=D. Gianola;citation_author=J. Burgueño;citation_title=Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers;citation_volume=186;citation_year=2010;citation_pmid=20813882;citation_doi=10.1534/genetics.110.118521" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=J. Crossa;citation_author=P. Pérez;citation_author=J. Hickey;citation_author=J. Burgueño;citation_author=L. Ornella;citation_title=Genomic prediction in CIMMYT maize and wheat breeding programs;citation_pages=48-60;citation_volume=112;citation_year=2014;citation_pmid=23572121;citation_doi=10.1038/hdy.2013.16" />
<meta name="citation_reference" content="citation_journal_title=Tree Genet. Genomes;citation_author=M. Denis;citation_author=J. Bouvet;citation_title=Efficiency of genomic selection with models including dominance effect in the context of Eucalyptus breeding;citation_pages=37-51;citation_volume=9;citation_year=2012;citation_doi=10.1007/s11295-012-0528-1" />
<meta name="citation_reference" content="citation_journal_title=Genet.;citation_author=O. G. El-dien;citation_author=B. Ratcliffe;citation_author=J. Klápstě;citation_author=I. Porth;citation_author=C. Chen;citation_title=Implementation of the Realized Genomic Relationship Matrix to Open-Pollinated White Spruce Family Testing for Disentangling Additive from Nonadditive Genetic Effects. G3 Genes, Genomes;citation_pages=743-753;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=J. Ertl;citation_author=A. Legarra;citation_author=Z. G. Vitezica;citation_author=L. Varona;citation_author=C. Edel;citation_title=Genomic analysis of dominance effects on milk production and conformation traits in Fleckvieh cattle;citation_pages=40;citation_volume=46;citation_year=2014;citation_doi=10.1186/1297-9686-46-40" />
<meta name="citation_reference" content="citation_title=Introduction to Quantitative Genetics;citation_year=1996" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_title=Priors in whole-genome regression: The Bayesian alphabet returns;citation_volume=194;citation_year=2013;citation_pmid=23636739;citation_doi=10.1534/genetics.113.151753" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_author=R. L. Fernando;citation_author=A. Stella;citation_title=Genomic-assisted prediction of genetic value with semiparametric procedures;citation_volume=173;citation_year=2006;citation_pmid=16648593;citation_doi=10.1534/genetics.105.049510" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Gianola;citation_author=J. B. C. H. M. van Kaam;citation_title=Reproducing Kernel Hilbert Spaces Regression Methods for Genomic Assisted Prediction of Quantitative Traits;citation_pages=2289-2303;citation_volume=178;citation_year=2008;citation_issue=4;citation_pmid=18430950;citation_doi=10.1534/genetics.107.084285" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. Gianola;citation_author=G. de los Campos;citation_author=W. G. Hill;citation_author=E. Manfredi;citation_author=R. Fernando;citation_title=Additive genetic variability and the Bayesian alphabet;citation_volume=183;citation_year=2009;citation_pmid=19620397;citation_doi=10.1534/genetics.109.103952" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=J. M. González-Camacho;citation_author=G. de los Campos;citation_author=P. Pérez;citation_author=D. Gianola;citation_author=J. E. Cairns;citation_title=Genome-enabled prediction of genetic values using radial basis function neural networks.;citation_pages=759-771;citation_volume=125;citation_year=2012;citation_issue=4;citation_pmid=22566067;citation_doi=10.1007/s00122-012-1868-9" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=J. M. Dekkers;citation_title=The Impact of Genetic Relationship Information on Genome-Assisted Breeding Values;citation_pages=2389-2397;citation_volume=177;citation_year=2007;citation_issue=4;citation_pmid=18073436;citation_doi=10.1534/genetics.107.081190" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. Habier;citation_author=R. L. Fernando;citation_author=K. Kizilkaya;citation_author=D. J. Garrick;citation_title=Extension of the bayesian alphabet for genomic selection.;citation_pages=186-186;citation_volume=12;citation_year=2011;citation_pmid=21605355;citation_doi=10.1186/1471-2105-12-186" />
<meta name="citation_reference" content="citation_title=Quantitative Genetics in Maize Breeding;citation_year=2010" />
<meta name="citation_reference" content="citation_title=Applications of Linear Models in Animal Breeding;citation_year=1984" />
<meta name="citation_reference" content="citation_journal_title=Genet.;citation_author=J. M. Hickey;citation_author=G. Gorjanc;citation_title=Simulated Data for Genomic Selection and Genome-Wide Association Studies Using a Combination of Coalescent and Gene Drop Methods. G3 Genes, Genomes;citation_pages=425-427;citation_volume=2;citation_year=2012" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=G. C. Kayihan;citation_author=D. A. Huber;citation_author=A. M. Morse;citation_author=T. L. White;citation_author=J. M. Davis;citation_title=Genetic dissection of fusiform rust and pitch canker disease traits in loblolly pine.;citation_pages=948-958;citation_volume=110;citation_year=2005;citation_issue=5;citation_pmid=15700146;citation_doi=10.1007/s00122-004-1915-2" />
<meta name="citation_reference" content="citation_journal_title=Can. J. For. Res.;citation_author=G. C. Kayihan;citation_author=C. D. Nelson;citation_author=D. A. Huber;citation_author=H. V. Amerson;citation_author=T. L. White;citation_title=Clonal evaluation for fusiform rust disease resistance: effects of pathogen virulence and disease escape;citation_pages=1042-1050;citation_volume=40;citation_year=2010;citation_doi=10.1139/X10-045" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Sci.;citation_author=G. de los Campos;citation_author=D. Gianola;citation_author=G. J. M. Rosa;citation_title=Reproducing kernel Hilbert spaces regression: a general framework for genetic evaluation;citation_volume=87;citation_year=2009;citation_pmid=19213705;citation_doi=10.2527/jas.2008-1259" />
<meta name="citation_reference" content="citation_journal_title=Genetical research;citation_journal_abbrev=Genet Res;citation_author=G. De los Campos;citation_author=D. Gianola;citation_author=G. J. M. Rosa;citation_author=K. A. Weigel;citation_author=J. Crossa;citation_title=Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods.;citation_pages=295-308;citation_volume=92;citation_year=2010;citation_issue=4;citation_pmid=20943010;citation_doi=10.1017/S0016672310000285" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=J. M. Hickey;citation_author=R. Pong-Wong;citation_author=H. D. Daetwyler;citation_author=M. P. L. Calus;citation_title=Whole-Genome Regression and Prediction Methods Applied to Plant and Animal Breeding;citation_volume=193;citation_year=2013;citation_pmid=22745228;citation_doi=10.1534/genetics.112.143313" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. de los Campos;citation_author=H. Naya;citation_author=D. Gianola;citation_author=J. Crossa;citation_author=A. Legarra;citation_title=Predicting quantitative traits with regression models for dense molecular markers and pedigree;citation_volume=182;citation_year=2009;citation_pmid=19293140;citation_doi=10.1534/genetics.109.101501" />
<meta name="citation_reference" content="citation_title=BGLR: Bayesian Generalized Linear Regression;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=G. de los Campos;citation_author=D. Sorensen;citation_author=D. Gianola;citation_title=Genomic Heritability: What Is It?;citation_pages=e1005048;citation_volume=11;citation_year=2015;citation_pmid=25942577;citation_doi=10.1371/journal.pgen.1005048" />
<meta name="citation_reference" content="citation_title=Genetics and Analysis of Quantitative Traits;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. H. E. Meuwissen;citation_author=B. J. Hayes;citation_author=M. E. Goddard;citation_title=Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps;citation_pages=1819-1829;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290733;citation_doi=10.1093/nar/gkv1344" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=G. Morota;citation_author=P. Boddhireddy;citation_author=N. Vukasinovic;citation_author=D. Gianola;citation_author=S. DeNise;citation_title=Kernel-based variance component estimation and whole-genome prediction of pre-corrected phenotypes and progeny tests for dairy cow health traits;citation_pages=56;citation_volume=5;citation_year=2014;citation_pmid=24715901;citation_doi=10.3389/fgene.2014.00056" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=G. Morota;citation_author=D. Gianola;citation_title=Kernel-based whole-genome prediction of complex traits: a review;citation_pages=363;citation_volume=5;citation_year=2014;citation_pmid=25360145;citation_doi=10.3389/fgene.2014.00363" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=G. Morota;citation_author=M. Koyama;citation_author=G. J. M. Rosa;citation_author=K. A. Weigel;citation_author=D. Gianola;citation_title=Predicting complex traits using a diffusion kernel on genetic markers with an application to dairy cattle and wheat data;citation_pages=17;citation_volume=45;citation_year=2013;citation_pmid=23763755;citation_doi=10.1186/1297-9686-45-17" />
<meta name="citation_reference" content="citation_title=Linear Models for the Prediction of Animal Breeding Values;citation_author=R. A. Mrode;citation_year=2014;citation_doi=10.1079/9781780643915.0000" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. R. Muñoz;citation_author=M. F. R. Resende;citation_author=S. A. Gezan;citation_author=M. D. V. Resende;citation_author=G. de los Campos;citation_title=Unraveling Additive from Non-Additive Effects Using Genomic Relationship Matrices;citation_volume=198;citation_year=2014;citation_pmid=25324160;citation_doi=10.1534/genetics.114.171322" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. Nishio;citation_author=M. Satoh;citation_title=Including Dominance Effects in the Genomic BLUP Method for Genomic Evaluation;citation_pages=e85792;citation_volume=9;citation_year=2014;citation_pmid=24416447;citation_doi=10.1371/journal.pone.0085792" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=P. Pérez;citation_author=G. de los Campos;citation_title=Genome-Wide Regression &amp; Prediction with the BGLR Statistical Package;citation_volume=198;citation_year=2014;citation_pmid=25009151;citation_doi=10.1534/genetics.114.164442" />
<meta name="citation_reference" content="citation_journal_title=Forests;citation_author=T. Quesada;citation_author=M. F. R. Resende;citation_author=P. Muñoz;citation_author=J. Wegrzyn;citation_author=D. Neale;citation_title=Mapping Fusiform Rust Resistance Genes within a Complex Mating Design of Loblolly Pine;citation_pages=347-362;citation_volume=5;citation_year=2014;citation_doi=10.3390/f5020347" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=M. F. R. Resende;citation_author=P. Muñoz;citation_author=J. J. Acosta;citation_author=G. F. Peter;citation_author=J. M. Davis;citation_title=Accelerating the domestication of trees using genomic selection: accuracy of prediction models across ages and environments.;citation_pages=617-624;citation_volume=193;citation_year=2012;citation_issue=3;citation_pmid=21973055;citation_doi=10.1111/j.1469-8137.2011.03895.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. F. R. Resende;citation_author=P. Muñoz;citation_author=M. D. V. Resende;citation_author=D. J. Garrick;citation_author=R. L. Fernando;citation_title=Accuracy of genomic selection methods in a standard data set of loblolly pine (Pinus taeda L.);citation_volume=190;citation_year=2012;citation_pmid=22271763;citation_doi=10.1534/genetics.111.137026" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=M. D. V. Resende;citation_author=M. F. R. Resende;citation_author=C. P. Sansaloni;citation_author=C. D. Petroli;citation_author=A. A. Missiaggia;citation_title=Genomic selection for growth and wood quality in Eucalyptus: capturing the missing heritability and accelerating breeding for complex traits in forest trees.;citation_pages=116-128;citation_volume=194;citation_year=2012;citation_issue=1;citation_pmid=22309312;citation_doi=10.1111/j.1469-8137.2011.04038.x" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=R. T. Resende;citation_author=M. D. V. Resende;citation_author=F. F. Silva;citation_author=C. F. Azevedo;citation_author=E. K. Takahashi;citation_title=Assessing the expected response to genomic selection of individuals and families in Eucalyptus breeding with an additive-dominant model;citation_pages=245-255;citation_volume=119;citation_year=2017;citation_doi=10.1038/hdy.2017.37" />
<meta name="citation_reference" content="citation_journal_title=Genetics, selection, evolution. :  GSE;citation_journal_abbrev=Genet Sel Evol;citation_author=M. A. Toro;citation_author=L. Varona;citation_title=A note on mate allocation for dominance handling in genomic selection.;citation_pages=33-33;citation_volume=42;citation_year=2010;citation_pmid=20699012;citation_doi=10.1186/1297-9686-42-33" />
<meta name="citation_reference" content="citation_journal_title=J. Anim. Breed. Genet.;citation_author=L. Tusell;citation_author=P. Pérez-Rodríguez;citation_author=S. Forni;citation_author=D. Gianola;citation_title=Model averaging for genome-enabled prediction with reproducing kernel Hilbert spaces: a case study with pig litter size and wheat yield;citation_pages=105-115;citation_volume=131;citation_year=2014;citation_pmid=24397267;citation_doi=10.1111/jbg.12070" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=A. I. Vazquez;citation_author=G. J. M. Rosa;citation_author=K. A. Weigel;citation_author=G. de los Campos;citation_author=D. Gianola;citation_title=Predictive ability of subsets of single nucleotide polymorphisms with and without parent average in US Holsteins.;citation_pages=5942-5949;citation_volume=93;citation_year=2010;citation_issue=12;citation_pmid=21094768;citation_doi=10.3168/jds.2010-3335" />
<meta name="citation_reference" content="citation_journal_title=Journal of Dairy Science;citation_journal_abbrev=J DAIRY SCI;citation_author=G. R. Wiggans;citation_author=P. M. Vanraden;citation_author=T. A. Cooper;citation_title=The genomic evaluation system in the United States: past, present, future.;citation_pages=3202-3211;citation_volume=94;citation_year=2011;citation_issue=6;citation_pmid=21605789;citation_doi=10.3168/jds.2010-3866" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=A. Willyard;citation_author=J. Syring;citation_author=D. S. Gernandt;citation_author=A. Liston;citation_author=R. Cronn;citation_title=Fossil calibration of molecular divergence infers a moderate mutation rate and recent radiations for pinus;citation_volume=24;citation_year=2007;citation_pmid=16997907;citation_doi=10.1093/molbev/msl131" />
<meta name="citation_reference" content="citation_journal_title=Genet.;citation_author=A. Xavier;citation_author=W. M. Muir;citation_author=K. M. Rainey;citation_title=Assessing Predictive Properties of Genome-Wide Selection in Soybeans. G3 Genes, Genomes;citation_pages=2611-2616;citation_volume=6;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Genet. Sel. Evol.;citation_author=J. Zeng;citation_author=A. Toosi;citation_author=R. L. Fernando;citation_author=J. C. Dekkers;citation_author=D. J. Garrick;citation_title=Genomic selection of purebred animals for crossbred performance in the presence of dominant gene action;citation_pages=11;citation_volume=45;citation_year=2013;citation_pmid=23621868;citation_doi=10.1186/1297-9686-45-11" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic Prediction of Additive and Non-additive Effects Using Genetic Markers and Pedigrees" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8.cover-source.jpg" />
<meta name="twitter:description" content="The genetic merit of individuals can be estimated using models with dense markers and pedigree information. Early genomic models accounted only for additive effects. However, the prediction of non-additive effects is important for different forest breeding systems where the whole genotypic value can be captured through clonal propagation. In this study, we evaluated the integration of marker data with pedigree information, in models that included or ignored non-additive effects. We tested the models Reproducing Kernel Hilbert Spaces (RKHS) and BayesA, with additive and additive-dominance frameworks. Model performance was assessed for the traits tree height, diameter at breast height and rust resistance, measured in 923 pine individuals from a structured population of 71 full-sib families. We have also simulated a population with similar genetic properties and evaluated the performance of models for six simulated traits with distinct genetic architectures. Different cross validation strategies were evaluated, and highest accuracies were achieved using within family cross validation. The inclusion of pedigree information in genomic prediction models did not yield higher accuracies. The different RKHS models resulted in similar predictions accuracies, and RKHS and BayesA generated substantially better predictions than pedigree-only models. The additive-BayesA resulted in higher accuracies than RKHS for rust incidence and in simulated additive-oligogenic traits. For DBH, HT and additive-dominance polygenic traits, the RKHS- based models showed slightly higher accuracies than BayesA. Our results indicate that BayesA performs the best for traits with few genes with major effects, while RKHS based models can best predict genotypic effects for clonal selection of complex traits." />
<meta name="og-title" property="og:title" content="Genomic Prediction of Additive and Non-additive Effects Using Genetic Markers and Pedigrees" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/8/2739" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The genetic merit of individuals can be estimated using models with dense markers and pedigree information. Early genomic models accounted only for additive effects. However, the prediction of non-additive effects is important for different forest breeding systems where the whole genotypic value can be captured through clonal propagation. In this study, we evaluated the integration of marker data with pedigree information, in models that included or ignored non-additive effects. We tested the models Reproducing Kernel Hilbert Spaces (RKHS) and BayesA, with additive and additive-dominance frameworks. Model performance was assessed for the traits tree height, diameter at breast height and rust resistance, measured in 923 pine individuals from a structured population of 71 full-sib families. We have also simulated a population with similar genetic properties and evaluated the performance of models for six simulated traits with distinct genetic architectures. Different cross validation strategies were evaluated, and highest accuracies were achieved using within family cross validation. The inclusion of pedigree information in genomic prediction models did not yield higher accuracies. The different RKHS models resulted in similar predictions accuracies, and RKHS and BayesA generated substantially better predictions than pedigree-only models. The additive-BayesA resulted in higher accuracies than RKHS for rust incidence and in simulated additive-oligogenic traits. For DBH, HT and additive-dominance polygenic traits, the RKHS- based models showed slightly higher accuracies than BayesA. Our results indicate that BayesA performs the best for traits with few genes with major effects, while RKHS based models can best predict genotypic effects for clonal selection of complex traits." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8.cover-source.jpg" />
    <title>Genomic Prediction of Additive and Non-additive Effects Using Genetic Markers and Pedigrees | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"YNMpDFC0c1tddjs2-Vgmlmg8jhNc8fn0YRL2goykYJ0","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"493732","apath":"\/ggg\/9\/8\/2739.atom","pisa":"ggg;9\/8\/2739","ac":{"\/ggg\/9\/8\/2739.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/8\/2739.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/8\/2739"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/8\/2739"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/8\/2739":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022uueAItePjZeygL7bzeo36I\\\\\\\/T8wU\\\\\\\/EAVDeb0qUmfN9AAPAd+O5Jd1V3MCns7A9LpWdfwcV42N49z+OX76xWuxTmKOM37hZhknd+yERkPZyNP+8nYlVaTanqeNBhKCcJD11pjtMEeV5b1cLBRLJiG7RWwyBZKWroILt6XiHNLMaKY8RUIQxIClPuITS2GHd00DJAq+FlJkKn7A3spBPUtVxLEIFMpmcwfkA5KX4lrcLHHx9LHL9yaD0gq1Y3ed0Wt4J5LqGy4tkiry0UhB\\\\\\\/OR1Oz1HITL2FhP5hg6ZRuSVW+LvEzQ5xNnaMHKl3wjwLcsrMMY1OdSDpZerRpDZ2d6ikdRo+rWWzBcFq\\\\\\\/+uj9\\\\\\\/mAwKQgyOKdysmkINGX3W1+qXuKyqtTPFKEGx9XFZIYHram5auhXhfKpuoCvg8FQTjFdWZYaO5kbLDFHUU3ULKHBKywrt4TFclyutwwqrPwd2r2bGK8zhlHTJGG3wIfO96JeZbX5ZqtipnYAZgvhnV4fI8j8rkeeTkmFMcKGpTbFTQuqHyrurg0jKxDT3i6J6Jh53Oq4miFJFIHoknstN1kfGN\\\\\\\/spr7j36tzFVm\\\\\\\/QTGdBUx6Lc9HYeFkchXl+QoyA6ydbNt5eupySKTAXAbg4I5Q6Ah4YPB+YF4oit9QWkcWaA5VEWh\\\\\\\/Y7Cp7mp1ZoppG6HzlwysgINueWVnUztEWmguC4LHh98un1mrKmfzlJlbBpc3Q6HAEi3r9oxn8wbM830YUB83et9lA+va2GLc0JRhJDmcXEP1yY5P+S0Vz5LtUsnRIGX7aOAj1dKOpBBSbvIgBEAQ9QQYBRgCKZF3FINFTonCUgaJHXR9WqDf4sXMQD01uTZyGHJND38we8wd6M+7GlaUgOU2\\\\\\\/7laMn9NgjJcJMfMZkCXoiCTFzqBOkC4id8ijsVuHKXATTCXFRMANMJkUEVoYR7gBWj5W5rNc5clIcJWxrfUrlE0Xvac2dWDVvznN+sUdlFyAUdVsapikSoZnagI\\\\\\\/O7B8Ivrz\\\\\\\/vKHrqHgILR0V\\\\\\\/hymGd4i3BpKueWRZhe+8f3KV4fsah9Ftif2uA7MLaN2uOIzccKFrP9g6lxo5WLvPIeI3U8Jq5Z6gSPCc184jyesJ1TBOFRIsWKDJ3wM0L4Wykr66VCR3KuspAm\\\\\\\/8ul7IqGlLoog5h1L4K1OXfJNclMd5cDNGnZ0rqtfACb9iWJo+7E\\\\\\\/BzzWXk9XLn7ec6U28HmF18YXmw3dISRogAt4o26wVzfGW\\\\\\\/JbglLCdWaZh4KtIZDIJEmJcCp4xWZ9\\\u0022,\\\u0022iv\\\u0022:\\\u0022p8rGwRdsTfC3GPEDqUM83g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f44c72229db2fd49de53696a03b8fc764e2b7023fb29bfd980af85281935b246\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00220irGrppmqF7Yxqf9416gqoSpaCb\\\\\\\/smhyUGcf5jM8\\\\\\\/xoxnCqfErzA7XeQEEBAg\\\\\\\/gD2TWAs+0wy6GJGpGRbRzlQifbqdGVq\\\\\\\/ITl7Z4SW6PC2CD1aTDrW9Smlusp5seKIpZ9vM\\\\\\\/pQUOtdiHXRNh509aZCOwuB19OwAMa39eygqw20p6ONwfRxSsbBqFbTksqeJrcbmwoBIBr75lWpYN2j92hEADg8Sqxu0YRCzlqcJkwp3ie+X5fbAjStYEbIZLcj0q8nXy51LYrV\\\\\\\/Tdp8bhxIe26DitdrGSA9onGiAL1PSN3JSU03Jhj+2KFc2zRMTAUi4vxHxc3lVc1fPiux6mgEHHgxVMTNzOJH3eX\\\\\\\/oNaddNmswYA4UC8MJCJNU66T8NQYqTqPhuZIYnKcDuTgM8S7shmLSG3s2hPJJ1572cVXWqTyBldOHSLkF5C5N8REJDLMgnVgSb\\\\\\\/lGpD+T1kJC90uFxV2VAHRFkOn54c1VKhqO4LZQKXC7JnqQlqGXLIu5ZswmdmKFPxrH3lFwbyZrcogY2Eo\\\\\\\/8m8OzcawSjnGS\\\\\\\/WObUg4dMrM5xX36KXZyfVQCMtbOhqvaos0+OaONy7seKNsp8GeZQjAfZf2DqqtdpXcs8sxxShLwFax8L8\\\\\\\/1aeSU9VFFuSF6rYKxk+kFzNouBwBFUFDVcmE8NHwknRX9XVY17E8e80x6y6FkU3Z1cJzkHgnv+ZQOzwyPRYizoj4XwqEURCyys\\\\\\\/4xwRSE2kXa0CFh0XBJIO9J4Ioa9WDEUEFOTBvhJn\\\\\\\/u8vgdfZT4eobBklJwvJaSsAUyMmmetKfsqyZLKKoDRvT9cBEhrNJNSEYMjNpbTCVwui1Rd1S69KtaVrbThEttI8JHuXu3REGxKnqu1hPz2vn\\\\\\\/YRvD\\\\\\\/9KEbfCEoEjFDlI\\\\\\\/kbqS2lN0cSFJNvPw0ra8PeXdQBpRXFvg2gHwn39sj2npjcF40yIs0AeTS7UmAs430fRSBUYIVkuUubn8s92b+41lyUV760m4SgBxGmHXmvsu+rVc3P5r6C3EQ9Fmc7JXrZ1L0ReXGZIBGwOal61X0lkFmrC3F+qZ\\\\\\\/9OyVzC4yF2twOHWeaigQw0Y2jMIbWRr2FAbN623gKU9g8IjYLSmFm3+7AvRvhS16kQr4Nai+ERLQcZxIFCc98nfq2kKtzuf9ns6tV2dVKaDNPKKdkPI1EXxBQvn5hyEQPuZ1TKOhUavtOSw9F+lcRIB4zEuNGjHlJ27pVhM7HiMx4aS24pPg273gpuZOKC0rWacrvkldOSDZsZqSjUpG8OJeBHvcjP7TM5h1LwfmYphbFRboTvokmwF\\\\\\\/8Ed\\\\\\\/7qmzOJd+kSLyac711O0MQrZToSa6U7TaQ0Dw2UX7jy\\\\\\\/58noiGrtn5mITK9yIqa7euXHQtSWyJ\\\\\\\/w4RcVRoD+\\\\\\\/DYBBRIFRaw5I5fb9njyQbuR4Vc5ruAve4FuoUG9uIlN1btcYHgwivO7JqVVb6tX81Qb0tqEo9N81SVNLtmZ98nDPDor7InoXpOmsu7zgHvfAjomoubTvKvPAiYlvqhWozqxmGfZmfQyR+6u1vDWC7UaLFYJrifIlihPCQkpaJXGc0EMKfjkXoCskSf45s4JfgGSa196RGFn7LZS\\\\\\\/BACdzrMuV0ue4pE+CxR763g6pBHUZ8HPwpQQU55lmkSGtRvbe07eUgvzTddZR9F2nK40jqaGSyZda12kfmJFkCLQUe07DscpQHMq\\\\\\\/HCz6QkZ5FQGXXjhwKr8BAtxY9XcelkUdwcgMD3m08rU5ixgAuH\\\\\\\/kUdqYk5MF9ymFGvZE0Mgdx1hYf9mDjeqsdRNuA\\\\\\\/vxrGhLrf7suy2JKRi1mo6pXEzzx5WmT3ZNJ8rZGDHXVfSwNZPdIfb1BCHimxx8ZYsPo3U7ZRvL1yjxuT2wgQwp0eIs9ddD6gDfNpVVaO1zuoL7igbY84zdwa\\\\\\\/\\\\\\\/8L\\\\\\\/hdmC+Tm5IAwCBUu8RJmoyZD3TnSP5XS6URU2eV\\\\\\\/2T\\\\\\\/41hnQTrMj2fGlVmHuMJZ+PBlDdcuf38DkbYV2w\\\\\\\/sbZ7IlT3\\\\\\\/Ovte7guMpUY3nLrESDX5TZl7XRa4GTQD0L8R3Fewmv8HbbQIa9poIvnp6VnX+Qh5wyJ5+2W0DhxY\\\\\\\/m2Rt\\\\\\\/hxNDGPaDl0PtFZ5x+pNzABV44WoUlBhBchpVuhDajBnPTcYfyZH3Lem6xTG7zmJIxDm+JZRL4FBJWH1JvtBPrN8Xa6xyF+0GnHPemnEAM34CTuluGyKeKC7UF+50sdy9tPDEOHn7m5igeOtsuefRxW+qL2zXivVsN1210TjUvrfrMdMGYGGLpgkgoDLwXUtmvhJD4VwkqTtKEdCoLjL3aXheKjxGv+0drhErJ+B\\\\\\\/R4gd8mbvAVOdjf5+oezkVBzAiSbCYxIGT2JA3RK7aTQyoi\\\\\\\/7mUWjUJb4vfM6Ttl6Zu11FnXBG9hFhDun72Kws+hjztxNGV7T82vK2kBqlMQfO0fErAZE6Gfo85DNEyVAJUR2+nB\\\\\\\/d7+2MVOpZZXtNr08C3604oneolSWABgaBtk\\\\\\\/JpgGjVHkeVuf0CoLzR8OnyV8CvU8Dqg9i8QMXTvjzMkvHBtdK1pULiLdFy1l9iZCtbXbEradIcp\\\\\\\/vhK\\\\\\\/7\\\\\\\/9VQ4d7pqtE0eG5zgRzO65tJtrJ8vcaYK6rYNGMm\\\\\\\/z\\\\\\\/WnDKP9iW+8Q8fnS6JIXRDIpbGyX5VPlAqAdhxfAKBCEVuAcccLw65YAv8W0Flm+PTtSLJ72qyEsew5\\\\\\\/0Nal\\\\\\\/iHUZG1UGac3rWV2qNE84PxKCXTAekKPqW4lAvTdIl6amwHGJUSrWI60F7DTSHJdCDhMpvjuZBXXKRx3R86udNC1QZ7MeyhtaUSRuNrW\\\\\\\/6XObLqLRTHdVLGLw9mkgnfv+vrGoAnsGSp+69R729CWDNRG+lHs+nUkMMCV0lAt6Tjpx7g0jnumTdMJEvDwK5+U\\\\\\\/YVP2z\\\\\\\/CS0axMAG295l30D8KM9iz7JI3yCJNBa9SFLPMSSnsU7cRzItAgWGh4QpXfLCrwzBLBwMxWFTOMpC+YUzBXuvYWtXdCdaFDd9PaxeIFfwnBCh2f2+k8x27i8EcA3Ruu+T7XulViUNz3ETjlA5y5+nb1or9DAnRrb6eFgZolXVbBVXweme5iHQ6cP01JCsWGXNbBemEr5XKXuDQDvhjpamgBOr9fIfAucn4STB95j0EV2vCE86FBUN1LptC9iIbhf+LHgqCu31io\\\\\\\/p0dcDiq7DzPEsXXLrp6ydjsR\\\\\\\/LnWBG\\\\\\\/qHlVp1+6DeMQZyrFXekvHQ2PrPeLW+tIakKMJxNHWLxos8J7s93H2VHtoyI8p7fBsjNjQRGDZ4Y2EqNRUSM+AF4Ln5V2xb8F5bNUsWOpOjxNyVWN2JXDPkqGKSRHEiYORkm0y7Dnt6m2Z20r4HJI0umEJsZ2fLStyNRIuqL5NoUN4JXdAisdvara6Ph+aZS7Cqal2ynz9kfDI5cbsTHa9rODYcMLxaJOkH4uAiZ\\\\\\\/PMEOaCWxIMeZNXq\\\\\\\/ytEjEDSvFkkpiQe4J8IhtzY7cgLWyrXgxsQ+lRP6guOMPG+QLgdmmYL8L+wQq+TgCIn0IPi\\\\\\\/d2CmSGo32AGzh1uPY4SIq3XmFrnDzcC9UmV7WIC0zVhTifU7Rb8VN4WOJwiC1Luyrp\\\\\\\/RnwEJ6tgyrio8mkg3EJtMiQoo4zCaD4zq5aLhNKgsnzwTdbtAe1NNrnIlSNwMYi28TXW4WCV5Z\\\\\\\/xZaxkpEdllJBdP9yWPd6mrIEAaGJxuD2WVykNKQl\\\\\\\/29OkH08zape+Rfl5nJLnSP2v1d2\\\\\\\/+Hh0FLudIqL4VSDt0Xj6oRTaya0FtnXM4FFr8SB9jkei5zRVGfqGcqrY2dwvDgnKvyOlrWAAjDk9XctLv3vT2Nke6FApzn4YWAcNDuCE5oAJeRheaoXi19m8k7k0hx7rJZRx1A0+xpiCi0RX6nGXwACHHXH2yWceKr3sg\\\\\\\/jx2dKJu+GWRndsgRTZkYYCKoFdbMS4wqHPABgggAsHQ75JAwXffReBgFCRLV05IZIQu6GwIg2R8+qHmbMrI6KtwIYkvwvvMPOMav3Z9wHFWERoUSoea7DDvWtdYcparPlqRL1bSuR1VYEOFkZxA6takRucjUk\\\\\\\/R1vGBBkVp6rSP5sM8gjQBPQNKhT5mD6yPLCuE5VHrKb1kFJyi8q6OitsdXPaXw9eTm+jX7kTBRmZ8OHvXEfIUIohuMFrVTgTdV1P5Q02KpKLE73wm8HfpC1UNk9DNS82XlFwk+UHd2+lgfHg87rXUIfBmSaaZfvuFXIy8I+qj3ZKqC81LjGdUUGY+Ay6OhWg6dlKW2W7obVg46AQeplWoMTMKN1pEhYCcH0AO9S5jk1d6QFp0yuz31jVirJFSc50OOAHLW7l+Vt6GsV0fx6rJwQ7m19WT7DL4vJ1axGYgnADbXZkXtVs8eufTgYZ8SDY\\\\\\\/P9M+BY\\\\\\\/VH2oX6YYKzCwyfbAVuBPWXUEu\\\\\\\/7d50ldadUX7QGPMroBZSA5AKeqfDNxLnIfEMeEoW8S5YfzHw\\\\\\\/nzSwch4o6BVb\\\\\\\/2aHckPIR7M7iI\\\\\\\/WyiwHAqvkFRcwwI+t+W0pYp4kQcLLafFEEYlIrjiMJVRowGMRdKLCDjGzbwDqKhscOH8FnmBhFghKJ5iCPbC2sD07zBYdtX4U+mju5R3bgTHbWIRQwtxyXfV7wos8rtwr8G8z2\\\\\\\/P6vmuW\\\\\\\/+6rOl3Mu2STBNseKu+xf0ELx4prsLvrbLtVOmqQyzmZEcuGZfL1YOEJeihZV8n+yDkOutwWNFF3HmTXTwY4i90rSgFHKqiuPUEBAg\\\\\\\/662IU2kjQTtamExzz\\\\\\\/RLocSKXs5AVj\\\\\\\/qEJD4d4Ei4QLHmSOge0\\\\\\\/SKE3y5YiSVvaxJf+4HrO3+yl2U0D1sFVoGLEyA0ygaZZtt3uvqVMiPSbaXBrYzt52uU6QFnur8a1R41rhm2yfnbn\\\\\\\/ZIEnDmeYTASVqbkMSBZ6Nn\\\\\\\/aZgP0iaQvJR1lAa6rOnQ4uM5FYcSkYSwv4J48seNi6A733fc3r3S7yW1QY3\\\\\\\/y4fKRbYtpR\\\\\\\/71tLJx+DjvB3IH9M\\\\\\\/t2KRxoh7s0VWUSnoxvdEhtyIYffKLoeIjOD4nrDJ8LVLqqUjKZNMT68yqHAIii91mOO6vkDf4t3tdJF\\\\\\\/iT7CTW\\\\\\\/KUbjXJjrBVGn8RU23ekBX\\\\\\\/uxuIHMidcxcrZxoeIovqO9dGO+2d7fv0qdnrWf1QOsdYzcsllr33kIT1FDTGcijZk5Um2whT6dc+RXYB9K5PfRpRZz4t5Hh\\\\\\\/8hW9JIuEJyGNT1Gx5MF975z8qX9XHql5Q9phPy6n+7l9tzSq4C7bS+q8QELi6UHJhbwhLSYPCRCTD9nWkwuTrKi0RYzXw6bfOV\\\\\\\/Lx9PjQhgj0YWNTNl4EpchGubGlD5s\\\\\\\/S+\\\\\\\/LONSEYrLkhMdpOv8I+Lp5L0\\\\\\\/rfkVZSVRj9oULzOcQyZ9jtvkmYZox0+pEgVwlTxK5K7yFg9ptwxyP8oQWm5jb1Vb5c2gvlrNXv64WaHdgZKylEO+cOQLBd58ffzSXHnJA9fDQJHMUj9JrUti4GmN9kzIXK0IHcP7zcRatnQoVJx1qntK7sQ8zUZxHxJN2VPD7TkVjGq\\\\\\\/Dpbp8HCb5q9GvbLmmzziVRv5uAIOA4b6HbQBUTO4xFXqyEes4loOa\\\\\\\/QwB73f9+RVuBPPC0JfmPN4bJfKQQtG0RaX3FvIf0fvyiEsezQhJiygVyXVMx3r5ndh8Sv4aN23VuPJqPZbcFGAcnSl3fXjl0XRrLu\\\\\\\/pktJqVDLpqtO8xd57o2U2ebY7HYcLZhVBf9a+TkdhumgC0t7cbeDZt2r+gl+0EqcQ2fw1222obyPiBEM6La5Obr9uLavoAbQm07ZBtNREQ5dmySoCuakytV379ERATjorx20D183xSk9Q7xY6Phiviw\\\\\\\/jRdmzUFa2tfAovY9dZw0WyJ52B8nW4xmVWfZ0NKWP99DjuLH\\\\\\\/ClRZDDzK3t9qZJuBuddkfbEwtXHHGD53fdgk\\\\\\\/rPxrp2dz3J27ej0EoAq+8seARgpf0VI+NrwhKl+12KMnspFb0WtNlNF+XOt+gvDTFafsCCVNcF7JpKEqY+F3bPLPYe8w82JDorvF0uTEKNUtWzSlKvhT1H5lEEPJ\\\\\\\/hPx+uZfDyyIocrS7DWzvjzYcPbrZQxZ\\\\\\\/9Wv9Y68zd4iy6DqNLgSlsiHb87ABzr13NxYrT++6HXvpS9YxZnKNAHBu\\\\\\\/s1g3kz4gmO9OZ9AyOL9A8pxhBKJOsB7ujnqUgEtbf2mA7tGxjJjjfrX5q0OCNFLtSYO\\\\\\\/myGy3fsZLV+1eiD3VqoqAf7LmqI9VDxNlSCDwx7dWbRxeWSs1BmDRZ2S0G\\\\\\\/HhPnRNNTcbQ90bJFsn\\\\\\\/Z55bKN2hCYiQVEgboYyodbKkqvVbbsTJUGBjKOkJ36rH79OzgFpKOO0mWdXuuE6C\\\\\\\/5SCE4mm2+J3INe16VrazHketZn3T+eGBhutIXcPutm6egVzf\\\\\\\/R8G4QCpoHd9KM07ihmH6o5MutycIXpBVgPjlYm4HPNjoxcUo7sJOzvHzxOknWYSqFbRc6uuNUF0z2H6Ei6+G7kyRCOBu0Z0EXITUDeVvmiEHVY9AhjobYc1PDEr0Clrquu5A\\\\\\\/4rsOwG5egFqRCjkSQvtRK6qRZgfMrcgbI6YuvOeuaZ0jNG3yo8gFMIDUZZ27ttWIMR8IlU3iHnWXzMjT3wad\\\\\\\/b9sByY906Uz4hhDES75dk3i2VTjKy+A7Lfnfd0DtmoeF6K9rzHC3DbKk0668kiWCaTUmtPIjzP1HsCpshIjJUiNYFZ3bnUaekVXlkshTacN74cNgiUZiEFSJOlFJ004KRfHAxGsp4suGh+x\\\\\\\/LC2Fj7pBOrCHMseUzbMjA\\\\\\\/CflILdyfCOZkj8OEa4SZjBBiVIixj1RechrgNdLlCo499ic91vxOhTSAquyzCQgNUSoQEPajZWpi0zBsHk\\\\\\\/Zzvjgt\\\\\\\/HQMFNXhG5E2KJFTqCCNldVX49f2gfevkb6e7oVMdID9fQyuGD656rTdjxXdEz27ObLpTdcmMAfNycQPn0q8TSzLCg0T4zMNckY6bBH9CN0J26\\\\\\\/gV4ztqtem2oZVhU7FFiyByscXfqVhbZyF6lkz9pv0TAoelXE2xztqMglv0kRgRSHalJuJkgkTZbFD8FB7kjNcyujMMDAMlmepkSZu7mtBTHXdWVPYuG4AK2BJYCELjtAXhtTIz9CJ+btZaknNUs\\\\\\\/gT9kQyeMKv\\\\\\\/sjsOmnqo+LMhyTlFOpa9aiRuKn\\\\\\\/v4KsE1TXbgrJjOPD7x1YxI2KzGjZJaa4OwPUM\\\\\\\/hZj+auvr7lVSBq4tPM9tgbjvgqFMRlqynbrM\\\\\\\/fd51N1W348Mm8fVz4Lv8WdCFr6HqxzM\\\\\\\/GExX+UOCItnCVnO\\\\\\\/IJ+Hl8BGt1BxJUwS3Kk\\\\\\\/vsChlzCIy\\\\\\\/kXPD7t0mA7YLRHwVlW3ZUXH0zHMrChgS\\\\\\\/N7ux\\\\\\\/z6xAXYD\\\\\\\/GqY\\\\\\\/1GHms6rNVdhMzMn5zCYCf0R8v8PNqQlgsAg1l1VmVfSYVmyCE2rrldg+n7Y\\\\\\\/Q7lzUqEY78HGepAUhIwvtGtRdzlibKXpvwiFTSrnmhM\\\\\\\/oTR9h0y8QL9nxGacuGgaK\\\\\\\/qQ3biRIWBz2K4Kfhf3b3rxgBVJ\\\\\\\/NbVnmQYqvAemvDiSsWLcU0ndykEbt6fLtmQhZxRG88OprWd6Iwf7BDimtueCc2bgl8QIF9OwmAEVrW1QUkXOrXqM+qAwVvRs1gUwvv+zGpOVSzS6\\\\\\\/q1LFHzKhL2dNm6H\\\\\\\/T8E3ZckTduasTUWjJxrqRjPnT1\\\\\\\/4MzCWsg9x9FU8OEYVaTlDsEAgJRb79XCpMiCVIUZSWoN2JKSB0jbG8SvF+N3G1rNdmyA6QZq87WIGDRYLOy5wzPVWjQuwkxnanG+ZCwCK8i6IToTjTZ+8zc7qObZkCZO3W14oht9fJrxInZf7DJ+GmlaHz1Eu0vdy\\\\\\\/KWUIO4yF4sdAHlxmxb35xKayNEA+QUEwglcDsinlcjODGjvHzzTJdFyUAp7WjDBKiE4\\\\\\\/ME0FUCfaVoN4RB+cSBSY3o28Sykp2NxHzMZq3wdT9JMMgLDMdOMybSkQE16UJHjessgJNtsJVO+VY0LbhCBMqA4v3RAMdcQFLkIMqVZieDuyXyj96BNGBjZwuoW\\\\\\\/QIadEZXTe+ln2dHwe6gs5j5cnxHKL0T9nCdbR\\\\\\\/rJpO1LUAyVt8Bn0OLPnt2MiW4o\\\\\\\/ZMqroqDpI70rgIc77nOOcKCSZsUR5BV3pFthRgtM\\\\\\\/TbxGAId1eyxKr8qSR1CvIZwvfWnL03VqYNQjRs\\\\\\\/kpV7Iy\\\\\\\/RwmxBdPP+NT9qfuKxjTlBbrpEs0ox\\\\\\\/Lh+ElrnPhALXabUG73oyFVBvg90cKhR8aS1LzrimXulTIc4RR0f2SPD4e6jm\\\\\\\/u9W+tScM5In3g9Fv3awSjs9Y5sUDz3GPxmrkx9Go+hc4Af2OONO4AmHCFU0gV4wLKGToUnUCOBdtYczKMqBc1Cq0UX+qa6t7q1jx5NgHr+Rb\\\\\\\/LzAQiw7IxrfyEPCRHV6D2xPuCPM5ZInNjvhKnu8nq7ZA4onWFyFaoF1vBEpIDM+nWAIh+OnLo3U9Qs5o11xXdKRzrjFYpp1QaQbApV+txr2oy7eS3S5A9eZgwEwgjUg3k4jC7zijZ9Uc8zvn8SVUHZ6T4kSYX4BFlLSaC5tGshrcJu9VZAnylnb9x5R+9fXd\\\\\\\/j7fU1jgK7x6M7Oi\\\\\\\/xGDWDj3\\\\\\\/mvseqp+ru7IITjMYzQXu8bgbyyV2lmdbks10q123nZopaPSmbHeyTLjvI+3REENAPGPKL9BKU8FBAQvmYMDhF0\\\\\\\/ZwzoYf2ITd03+FBzE8oC6MGcYgn1OflI35dmFfLmNSTHQJUhpvdv9rrlWBjSL2++0KnPWyj0a4T7pZxGl8AmHCuwEpnWRES9JzDy+PD4dKHv7dO1dMYxkUgfpIbEKxcv9TuDGIKrTkHj+Ya0rFTsOqZQmyfIlcb1vZO9Zw6vbv2gCcNE8auLpJbmYtBTw1EBgUy+pKcMTRR5E3EApm7j+YnHe3D8kCJycGIYESe5CcsbjyqOdNHKlgvfeFd6OEcV0IlNHZmkQNJRKQU2imnsXiIh1TXP0t6Q2hhCTrRCK5\\\\\\\/IcPPu8G5L3ZErN4Avfv8DgRPy+zc+ZBdviSWmCbB3gZ8gyquverLrvqAkQDHKOWMcQkzeZJTGWWrZ5nBbel5Bne5\\\\\\\/oDD5orNcJ7ITNJbUaunMtBZVpo0Qah+bdKLPW45HtiEvfUE98ZRaTOwKkOX4aXm2Ep4LxLoEEkWiCXwx6CnGOihp0djvjfbET+Bec1ZGd7JOB0HSmAhFv2a0DTN8PWgNgDaQLw2u8tbIaoW9L6CcB8+3eS+K0e3F+fLn3pqeWsYPUNShroha6A758BVEQ8453aEe\\\\\\\/AjKr00QfpurHecvdAKt4MmGptIAwEVgwf+aft4XVfowVLxVfQw2agldLEDJmEboc\\\\\\\/131DFqU4kSpEVhInIjS2YRULF6vDqPhXeu7R1d51Tk1DpAQBPX9+w6oz7ZEOVvqTKc4RvCpt0COCbUpQ+OyFv1v\\\\\\\/hl3zOO2M4ybA4qDxjBe1sTFSNKYsgPokn3Vvpjs+qGg\\\\\\\/y5xOdH41kgsgth884rYCGgx9T3FvIUnkdlAzY5NTLJqQ8p8MPf2sYacwXCMJQRE5SFDjnbD\\\\\\\/UpgJeybwF63HBtKnicM4ImT9zm5KuXegXIxEmL3tPh8Df3HKLa56Kv\\\\\\\/3WqW813ApBjt79I0vzgaeRh2bvDvBN0OKcq1LoXAUauXWWxPP8v7qbHZWK9qGcPWw4D4Ee9Hrs8wyueGygfbWDgRtUdeWymgjsble\\\\\\\/I1SKcWa8beiGSvUjmQJxQeCF+Wqx3HbP336orYej7KIHp9hLcgaB11oRQBhNi6FBSOVjJlEhaVFBgAcYz33Yuih4vmMJSrRvh8Qk0GTA5rOOBf0kM5D35NnAUq8Bl5uP5HPSHQPU0c0dazfZpTZSi4s26DX7OCVPsM1F5NXuBd9SMO\\\\\\\/SnI28I\\\\\\\/19Lanqj6nUwCs07F17VlZVjYuqMtEnSh1yKX6cz3rSvmBjNihsJaKlfe24QWi3Hkqi5sWT12y9I2CscvoIs3uYUhpo7snIZDmSdznZEq1QmzsqJFNOhGx36S8PVLiHLATHPtPdNEifWWzzu3nE6wFL+jIgOuzhaOhlx9jj7xfeJpsJQwzS5BOTHu29ROhvCYO2z9RfWKWNgkBu2LwQIWAlEAm9guOLbg+112iZY7jYJO6i6vyE9gtDPSktECy1drw\\\\\\\/WLkFpNitiKKeQ+ZC6SRcW88\\\\\\\/kVvcEMK8lBTcgWdhxI5uqALMCBOujl+RLRZQjss+yrPOL0nR6B69gfHfZxVUTZuZriBe0QTltRDaO3dzKTsiTM+vH8zVB36Kt1qdE9XrnltCNyFHoyNcxIRH2kJyxkCLaDFk1FDWsuWzOqTdhLNgT36BLL81GpnRnCjJCMKnrBdC6Dif\\\\\\\/IztnrItK83MUy4zP57L1ZpsxbpkkyqB0G3ofylzTDFmysQf2sqjynjnwVVV1U4qxyaKPfpUulirEk0q3Y6RheblIXHrGvwi2QytHhbJNAcY1T6PH50N0dPQAAm9ide1VDgDWuepdfanGCIv3HJWh3iPY4cbP3Cct2RV7LzEUrosaGszAjlYy09A58XsoLgXy\\\\\\\/kxFhoj2AhwGhHPYpy8gyX02kBR8jZ8BlMdUZMLgycQ3qULwMIRXCT6fcgzDvzbiLZesS9+OhMGMrEAC+wLQEbozCLOVgKOY9KHBQbo0Xe+Dyc7jcNBEuJ9fl4uGyrmbuBxB2fO3VPcxeJFPvRxkDSLO\\\\\\\/JYILwA3bFisXrygMeM80akntS4+T0hpVcsihXtvatVyGZBvq8SstucwntB93+VPpO4mvkerM\\\\\\\/U8YxN950PdX9D1nEnn3sXgfYGZn3adsMI86P3Y0lZxqMhfjceo0gpgTATrPNjpo4vGgPaaHkJo4yQgm3yEjAYbVTLeQtI1h3BR\\\\\\\/x9BpaFQUUJSlQBSHuPn\\\\\\\/0DDuedp6MdcNCt0ZRAPwGU8O9AaPJJ1FBBdcGAIuZkVdNBNehNL5lcFY\\\\\\\/Gha9u\\\\\\\/Pnu361qkXGOmzn6ur6r1C0vx1GAwgZ24cxa\\\\\\\/5a40p3WBpk4iGHnpl0Mx3NPQ7UdyMNiAgSAjaY2aY14OWP63ZyuhzUiW2V6UOsgaG6YMKok9TYdrAFqm038b36zLGIs+gm+wSTObCDG51ilSC7d2u19S3+qpYH0\\\\\\\/OlfTtTqkAzTXSmtkiW4uQuVsLZUf6jeE9DGqAVHpQk2izp8qLs374OdIFK4z1SXpcsVizf9VY7b6Q28fksRyMvkGiY+cC68GKV3+Z8iDjASk16+\\\\\\\/Wd77JkjId5epkFMwdtIUC4SJmpcVM\\\\\\\/NcIARoan02zytgSwwHdhNxXeHK4jw6uEXOEno6LGQ4ZfSGGjtEX6Fpm+5iSqQEm5reg2T5g31dPqoFJ3S+3dsz2oTl0FSwY8rtxajbKofzjvrRPHwBH2kOSEMacmc7eroxekckXpjHE7Yw9ULZ9Sv5hRAPcbAUInuZilLx8Vo7ZHGy2rMMBK\\\\\\\/k1iAOyXDUg6UNRqLpd77FtpQnKCcr21kyByEefe8R2ySpStq1Ux40+cfHKo9DsdjSzzRTIaqorZd3Q2vKTwg978+uvdhtMP9RQFpDXkXRE+eE0xPgiq\\\\\\\/LwQixYeWwVWJHcfvGra1rxxyn6dWvu9oWVCdJifIJQtb9jopc+5IVtJmYN9gwg4Hw0cTAHW0eFI5GvTFmIyjkpWM6t7AAmTbIPIPiP3HiodM8fRmg4FwJgyLhhCllYM5l3XrSgIv4At6pkiusGTiKVSuNITOHO8PCXU9xPPD0w2Ioh2fAUkG5xRqmn7h2bTGaZv7\\\\\\\/olOH6vb0kcqZHt\\\\\\\/DQKULKxS4I6TiwJmbAJR41I\\\\\\\/Tm68wVxvNAKCdtdrePz1Cw9rLNnu7JqZlKCl8AMrBi8dXjM0iounZJtUy7JDB5gd54rZqR5xvALbA3ODfJq5EZnjDHhSsJyCpgpLOAJoW4el9JuAvdrW8HCL1QJUy1ypT3YiC5M7uAsTeEA+zsk68PPiZ5pp9NbUpfGfSKHW7JheWLTGzU\\\\\\\/AxhFdQ5Kz1L4QZnkip+DQbG5eSUTfIkU3Y\\\\\\\/wLRz9+l9uDfVHUjKR0gbp1sevXZCsCXgj5c0+Vfyjg1y2Hd20x4Jtmh4kly67C5ZFBbmfjkLgtxKuAU5xw9GUeYeXx3NxxzHUdIhlBQt7UbD1MvAZp1SjoeBEp37U+9rLwFJfEAU90A0gOBYEX05Z1PomOgFd5+ETEfTqQnlKXPXHv5PhvJAY+V4TCWMkdEaH5IAIoYVvQGIN4\\\\\\\/gtfWClRiGUZm1kADymfDTOSW6yoPLRkh6\\\\\\\/7pdOGwHoqWWGNxeE8BbqfpyOi8CQMD6i2lSfW09pu9xppXtgMy9YmEwm15aqHl\\\\\\\/E43p3hwYcbxuhYOHHU0gyAeG\\\\\\\/J1IAb0m9L44DNwA66ceDLxHmRtqRmX7\\\\\\\/d1tsT+AKBSqPZpUPdZxuYgvHHXUKnUNoNKsgkgvOO6ENLxxVMi13YQ9YVSiGvF+v\\\\\\\/S5KSXQDB4dbG4tRGSN94yCg0ls7zMQyWgqNjiBiumyrniHz+d8\\\\\\\/X7bMXjNfYBBmLyfXwoTFfgDuDn58ELbn9lSMMvK1ft0oIvx5IPef1u8FBZzEBs5e1hGySo1BQFyok5CpvoLULdZ2cL+Z2h10y310+TswXb4i47TXv7hXF+J0wnBe+sFjziIq5tNsWTxqEoy7EMN+ArnBqAEBeFLN3t3qLaKOxK4Qi6lTjLmQt4q+bf9Ci6IeD3rpoNaELcdZmIOAkJMO5PY7ZEPvurLERe+RfKq8jVT+bZ\\\\\\\/\\\\\\\/kAUyLgnr9UjwvSoZCQ8ghstcrHQh0OfZxZiBaoJ6TGR\\\\\\\/I6Qa+iYE7016VdFvwoRIBgS20RjqZQYPvVy6mcmDZZB2DsiIgHQoT+V9mJ9EnpZgfTnLDjDLXpGfWwgNziw5cT4ArHZBF+QTG26S2LjaDRgeb63ZjVsFyJpmNG3R7jpimr4cP4\\\\\\\/+4bagE1fFwqdN\\\\\\\/0M5mEZArUuCZsut2fPdMzm\\\\\\\/GHq\\\\\\\/rmoQ\\\\\\\/MtH8dW\\\\\\\/9t1CVpP1h+5QTahXrdzquGvh7PqVXh13W9gk0rjGMwokkYUnqRRK7OJriXH5lh0uyJ0mfc3JbwzyuK0d0WvrK0gAyDlfJB669laphHUy9Y943wu3b2q9XYKL4s0wCoReICEYhHk3fYxrg5KPGQL2F7hm4TPYJFxNxdQISl8sYHTsOK+AKLKkYteDbRj+ZIE76bb5xB0vGXmS1GIrG+KscA7Tl\\\\\\\/d+AV4JnGb5hmVQ8YowDsdqW855M3VXIokL8KiKKlNpoE9hQk8GvAcaFIQxNi\\\\\\\/Jqtq2j\\\\\\\/p6ug7oYAAyVq\\\\\\\/tGpwCXG80hQB8dkq8REQ+a9b9xIn4Zp7i21nS0X1w6jbrJoKoZTrpy2RXgx7C\\\\\\\/N5S3MoBGtPmTTplL9IaoMvT5wsDCfCH4M44RBn3Yca25SKSr1dh0WgWanZPF2qsrpYjKLNAbu3cL2arwkx+AP\\\\\\\/GsuHgbulXuTtsB2nhl4MA4fXiCztDtX0qTePjeHCFBvMHDdUUy1I6qMwWg3kLYWqALJSWNIq06oCzV96nn94DJ3yg2eGhjHRl0ZmChk+7Hl6eZU+9XvU1doth49Ph+b40OC\\\\\\\/z8xcQo6bYYKtkXU+QcWl+RZgroaWf8x\\\\\\\/tgKHHTkD5Zv5I8zWMKDG81FaQ+bAXSVY38Ll+bESbxLkBKJO2gcwWV6IMMx6PJvKEwziHLoYinwRdiq62z+4S2rYqvg0tqgcRuKHXExy69PwJ4ck6mfPbaZEfm2yJaB9GoMsy2ntPZ06H0NilPZ7FPViwIrpOus+He1gcmwEQCO8hgmUj0tuc6w9sUe0Be4kDzpcVgPAWcM4QfvB2clMvXRFhQ+brFltKOlMSnlcjdeMRtcaS36Sg330Jj3Da7NGNUf8EyWZmJbe7aRM00zQ2I12+polmUDk9PVfNPz+QOgrnEe09WrUmHsvRMxU5lbCv+BnyZnGMzN0DjPm9EtF6asGJKtxZz1vByJhu2SFPegMNMgpYugUGJKIhf8fTEfvusngkPcnBsOfVSdtFGe1tVvix\\\\\\\/mjBnjFcOjKBSzbByTNtDuqW2AxvxDkSjRtCT2\\\\\\\/2Eu1Z9LxWcETcYAdPRHce0PfLhhduy08f1Z5XZ0t5uFSIJsRU\\\\\\\/qFqPA8Ty1FygU\\\\\\\/+gPg02KuqprL05WY1wLIG7K9QjD7Rj67pItI1+6CZrYa5i7txHVeXlpPJ8XH\\\\\\\/bCSuH5TwxBBODsXxi8\\\\\\\/ZY8p3RlL6XTvm1n0duBzNIAegPhhGbmgYI6A7pYITMUU\\\\\\\/TCnY0Zap5TvrkiLMEtTEhKsURyIbH6RhdNMxBdNWPB5dRzjj06+DwLikQFpzmG0GqjcttKp1hHmlnobKNjGxnfJ3UUfDcgOlFENHuD0Ibv011VjPM2+tU4cTYUGmkBvLl9hTeHIOHVlgUio8JKGLZVBTtyz6qGEOlKyGqCMm0J7rq16p88wGzmDV9ug+aX+l7pdtSw+py5NJXoZ0P1UQG04Jhx5B5KK4ed86ZHN9tWw6CkIjei69uWIZrnt31dR1L\\\\\\\/hAUHHQmmBUNeR71XEj9WzGcmG5QMBDaVUzzmMuq7hxCUBdPpCIWK\\\\\\\/Rh\\\\\\\/xsayl3Tl7uzG4RJ7+wCfXTNPbpIYbd9wqAZVlmpi1yUoy3zJwRbZ+sCXqf90cfE0HMjh9EiYOl3zGxByzKPLYGauspnsFD\\\\\\\/tygONH9zWF3J\\\\\\\/p+8BAyPew\\\\\\\/dYvp3fZsjEFqPJiwrnVwJ0VTobVYozHYmodMmyxIBjXeGGsMGihEbMUEXV09QaYTXcvtJYTfE+jx4x5X4T0tF2TXLn0iRGZc20oFSETv8rP7inK65u6Lo1hCHGplGCWHwxnNocbzDrhYnBOZkjteJOnpoiTgge88zxiH3kQOdwjG1ATP18YJ4sBCCIg1nvKhGvNHkOJvPXbc3SOIOWHEEE6mhX4GKRopD1sO96TKl8tC2b\\\\\\\/iM6e\\\\\\\/54\\\\\\\/CqApyb5NwJgC9DW6vzqMmhHoBcViYpNT5YfBoiTl+xSkLR9Okjohj8dTFkO6RE\\\\\\\/EekgQ1cyfAz1MgYBNqxkW8n+EBQAKhjYaHsAHbhUScpC46jl83uWISjcTMK73mBIeOihxvXQUbprhHyU3J0rK9VlHd0qJAQDd8mvseMkwvqfLH7tDD2z2YbdeDz\\\\\\\/l6OAaeEGVP3NENf4yivoN3TVA+2kpl5XC4iBflSyGAvDDgVqA37e8Uo36c6ROzEKsASCqZDJlbnsFJgl3FSinsWEIcSljUBw0HJMYVMlo+lows2LcvpRt7DqYQvyv3IZ7r1RP8OYorsXJ\\\\\\\/AX+Ekic2O4DALsAjuPKU\\\\\\\/EbrIt7kjRDBmIK9bbM33CjtaF\\\\\\\/pAtxjOucJ22lmdJetoicLlvLEThdCGYk63H4cGOnj05lf09ManXEXsc0km9C2ZryHuYJusfSKUUMRLxbtkY3bA4u6NQ6AQNDAtK9gS9JCpKPnGbUfUeQ61HM+DJEyw85IX9MfcwcY\\\\\\\/cfk6XixJFAtXZTSwUHWHZGdZQXyieqUGX5rmqBq0niPHx+wFx\\\\\\\/5Ic80v8qV5PathiW6bCVpflu3Pz3onF2UwLFMlEujj5ICZSAp27mFrryFs0NM5cnBj+u6k0\\\\\\\/bugVRoGU14tDUUW6aYbxmXdUETAX6Sp9cFUiFNPXUUxesMUVV502IdEM+Pzjh5dfXJwXKRQyEyJdLn0VkqYP8joqx8BqT07ZqIfkZXRwEEyMIgcCuTdo4NWItnUjrh\\\\\\\/\\\\\\\/m8mbMMbHYi0f7Ug9HMzCsX8CvCbe+5KaT\\\\\\\/V2\\\\\\\/Ec7N1eG6WFo2++xbp1koKOvr5sf6PJxTmGvTZVPjKIjnHzTSAUIyqMG44278aPgYAm\\\\\\\/KiN1vx8zVpwBtdObnwZvXOZIeLQpPhmx6W29lX8PNEQv6myhSAla3eiWcdfG19NTbSEvEZZHd4VaKHwrJA4wrT6f9wk0ndn8sYtdRZbdbanZ+lwaY3L7uFOp6yKF4f2yQ1tyhf\\\\\\\/vWllpk1GVkecANGryHZE6e6rVKAIlv\\\\\\\/H+plJohm7fBms+9xLaJMc0wcG7zDx16dLZGsDQWA6QIJKXGgj2IJqfwnXm2\\\\\\\/V4vvmSg+NGId1NeBQ4qNKIG265UJ21OJv6fN2DIoWH5YjitZ7r612VbfwA5lMcR00vV0ILfupScEqdLonPZlZ5GqFEr4oEHvQ4hTkjkDRzMNOutisLm+6Cm\\\\\\\/GGdWkMuNOD3URBSK0Ysl1EX\\\\\\\/hfj9I6xTWWa+M83\\\\\\\/cSUfzE8iI7eVP8H0T9JJlSh0WuRh6Bg7hXqHeXtKQyP11LzGxUyNjwNk8nwRuvZDmp+E1vElslEj\\\\\\\/pnbJSwXE02GcWeHMbYc4XTYsxqR6d3ti\\\\\\\/nhs3PS6K5xkI\\\\\\\/G3n90cY0gnAR+dTBL5xtaxiexnnaJpQS6nkqIFE+SXH9PqzIiOVyuqq5JZ29BXGjUJC9FiN9ZD2JXsXsH7dXW3YrT47qWK0bMzcSMa5lhI3oEAgHPDxh06wfniSs813nX2WT3paCUAR7ECcpOBfhrx+\\\\\\\/AGXo+zdXZHiNxiZDtx5OrkQQRNfAhi0l7hqmD4n\\\\\\\/l0ITm64rctSWo560IG09YVwht4lTAkGE0DjxLaO89GcDGG9tKCk5dG+vG37SCaKcutONrPwMHvhTsniutz6QRpGQ1TCNPsKDRBhP1lDDO7InXfcTjyLmiLDsWh2EpAu\\\\\\\/nvrandmMULQGGYU3Oc8ArKtOoeTXeu4BqcVZyI4RGF4llbYBizZoWSxOgfUi3Ky\\\\\\\/qmwEQX9GJdFaNcUrtWf8vhbmgynfpQK2oQ+qGzq6dYs41O5AG1LAEEuZdqXZT4b0e4Evm2cj9iP+diJkRElbbjcAPNzjXuTM++j2\\\\\\\/pPPmY0g4g8RLmW07v+afU16l2UDU3pcPiekFSlrk0CxhxU0qr8v61OWqLgl1f7TBE6kmUyExwPl8kPs8jwTPxTdUSsEjfKNbK62X6I7jinTGIX49mnIw2B3r5ZztnTYpFww4hB90vcGawaDC7cR+OkJS8lD8oIM7t2QUfAVCMOu1y\\\\\\\/ZP9yFZR7eG5OxWndWWsCnLDEhkLR0udhYsLDhuz3hYDkTeKmbJx8WzjvC6EkRWYq\\\\\\\/Rjei2az8xhMHrpFTexueY1IX4CiOwwNy3ed2JmNhxrOn7OAfv4YIM5Gu4A+FXMazbWIL0dbOKjdtNHOitqqXdKjuyUGuMfwKpfHNnA\\\\\\\/YjQ1MyU+zcv1s5znqHpyHsP+7DX\\\\\\\/IInofA+knY+sJ9\\\\\\\/JR2Mm+8Z6mkvZzTWRyfBKuNvE8t6kLlhoT+Msaxe3xLwaysek9UR4Q3y1tWFaUCUAVemQtGdcMmBvtXB2bUaQ80T8NEprprwVbaGb\\\\\\\/spn+yoBxf6LsteM4rz3rtK078KK9fLWaWnKg4IPtPGkFw4Q0s3VqzOkbKLs8CP8s\\\\\\\/eShI23wY1bDTmqOCfNQbpMOx4i7vnnnvNFxiRyu2csjzaLVruG7qk\\\\\\\/Nlm+wg5P3lkVmyBmiEs4eYekcTBF1K40lrZ1i2Xy7DLwpxznh\\\\\\\/odl0jiI110lTT67iO8xj9BYrxTG\\\\\\\/SiIVsV+3\\\\\\\/vn7ScxwLfoXYWNS+899AZuU0D3Z1rK44AbAuZ4koTcxtFYW7Znc2E9a+mO5Uyo+flGrkpKlodEYukoelxwLjJqcGlpFXsMCilw9PVtCARqfsSSrAhw9wIo4zIeH4I0wp4fci3Av+aWaonNDwLZcIoIleypTQSEmBUgc2LiTCYuRiWm1KXcv7r2lZNP5CUO095iop4BDFfduP7Y0EKupnNhMszN01c6NrcyXmR1mO9IBst4Hi4RXej\\\\\\\/CJl5AvpFb\\\\\\\/gv8PVlebacA+Fi9WKkktlmX9efnJvHNAYvi9ipngu6b5YFbT\\\\\\\/WeoNquFeVnNRZ1wf1pEMkEqBb7Pn5VIYVmCcW\\\\\\\/WorNS5gsVMGNeQzjSIXaIFRmtc8fi2sTTicVaKzFNuwziVs6OWxxPkB2t5Tt9hQSwrmBInbdv3Gv0KGoVBH3cLPLqptXSZqhyWjiTbtGJMmJUEBKj+mcq8ufW5ptMcgLnVYiY1ZjwpGiC0hlxoPjq2KpOCgbF3GU74nANHRVbRP8yS6jXAm\\\\\\\/BZjfOQr8ao+v0+qwNNch9B4N7PYe37DDKFabYaDES6TEEl7RHXFPgQCF5B3HsJXh9ec6XdhATkW\\\\\\\/6ZcLQ4326vqzZJnNKrNfl9ZIaVSeiXYwCiDojbGEtQJDJzDPVNSQMei7uRbD3xA5qdxxZfNyIQWu6FvuE1EKE7ztrbUdZ3rQoRc49gsEROWJhbs4WkD1y45bWttA6lOVgpe7VMPhZNpi3zXI\\\\\\\/9paFnP\\\\\\\/i+2JJgV6TNUL9ac4XLNi4NQpwrtDgOgGd+NvHPoNeizx2whp6Totl\\\\\\\/tXSHz8KB5kJ7lBggW37IHmtwGAxMK7yFWFl+odlXbI3pAQcB3HAu+Di1NM7wlcIVZUrcazpBY\\\\\\\/\\\\\\\/S7ma83i2azTwbZYgDoy0aphtxgKHzfb6RqMFRN+yjpg0YznvYH+SyIqebanaxftO2PjVGFevKGJOcq1GM3LV+36M17gy89gSdAkcvQOeBdpzizIsVpmz1uzhUsoM2RHnHJlSM0jNv3qOiMTtMQ83yZ+c5qWVULC0F1qWU1owkPJjGxmi5LBM55YVX5NeKYdoP\\\\\\\/NF7\\\\\\\/su31CtJZ\\\\\\\/DoTTZR+eyQW80NusfKTXN0rLWXA5lJf3drnRlwOMZYJ2bqStkDXfSPIWVKVYTyMxNQYcht7vBaUOACsXdNMZz99VHG5yfCohmJojb6zigMGrVWOPMfbYIwMhixC0IFexhmsfBRNUGfmQ+LymX4H3IaJNvyTJsTzUtbQ1\\\\\\\/dagaovtRDNz9YkExWI5qubptmWLuuj\\\\\\\/SrgBBBCb8cD5JVG\\\\\\\/I63VTnphpGKPhBnoRmXZI4GUKl\\\\\\\/\\\\\\\/Ddk79EtwTHrmlxnTPU36WMAta6nnx+Gj8hbqQfMOD7UTwq9r77y4s5vrRnSAgyrxbrh\\\\\\\/j2Ag4DWRKXzV4NFGB\\\\\\\/Jt8C6+5YElYmJLdPZ4AvRSO4iRXJviHF0\\\\\\\/+MIx8LDv+PLfEfyEqni+H5GVjIp8am5WHIyQVJ043bzMVuAavfqE44AaAF+Ebyty6C78Ve4iuwyXbKuiXBY0cg7Fp4vMaV+s6BaReSbZSQLYZhmuzFpyG9Jtc73lS6Kf+HcP6aCWisKh52yZf5BHrlUttftxb6p1WC6bjnuMVM4pMGi+TYShHMtduK3GlNyQifj7CepdtUHFgCsFn8ktowc\\\\\\\/Aj3BM2O\\\\\\\/rCiIefyIOLhqZCA9mZxBv7GyEbA1HzFG5OYfqGxp8mvUNk3gqHUZl7wUxAHjxEnUOVL2yaLp\\\\\\\/\\\\\\\/qX9gJOIz0iJ8LTVdjV1dsHEqIiIVjbg+fQvJ7NrP62+iciu4GWskks4EE76yoqx2Ohln6l9eIU2XD5xWpyves84y68507sKU9A2SRIY8leEQ8Ln+FJXiRVOfEzJRL8w2wkcK67Y1OpuIT8i8nPJ\\\\\\\/UklchiRnk5n9oluA704XBp4o9NH\\\\\\\/0kceZpOuyJN0tTzyQrHz\\\\\\\/UwgzsLcTNFcDL0wUgM5oVJo4bUoief7GVYaDqQTS8PIv+rGA0CJItCd0rYalYBPBrnPCX0CK3CptFjCuF9nO6G3QHhVtOW8\\\\\\\/fCTbsqFnNGq3e5Q9mBqy7YNqOLgFV5ug9K5X\\\\\\\/RtaPWxRELThax5RRAnvNC7Qj3w6paORTG2ZHcd24Bk6WozcbIDzOo54d6gjkBV351lk\\\\\\\/ucBZU9fZB9rDo0xtBv6WY6hH4TEboG6ChyLIA0fW0FPiXYtXjygqu\\\\\\\/3\\\\\\\/K263Gl15oGj7DQ6gLlDtmeAPHvfBynMjjIAv3zl1BzNWOu49pxUJnTT7rKn14q1N5Gi\\\\\\\/cZpqWr0jh1MBi60H\\\\\\\/hpONc4okk2gTig0g2mZeCd7al3V4CX3JmZeArccJwh36RMNzsCqi8Vz9caawSJx3F\\\\\\\/CcrwOdkRw\\\\\\\/NsZsPofoPNPaRLxZdf1aU7TeOq4WHbfYebq1F+5tV+U2RRKf\\\\\\\/8zNg2PqJDyVU5PmtNYN5nTNBca+Hfc6Wb+pQeucZrZatX1CeQ7DNVvlzoIOeE8PlEnijnk3FH6V42kWT2h0ERxL8uMF8Q1\\\\\\\/lYcDyK8K1Jy2mVGr0D5MCrvcpkp4ZUMPg1PDucErFxa0VzzuVhf2zDGPgaosoRPj7umyEK\\\\\\\/EE7Ui4aVVnAjIPY047dupta8Zf87uCYFeaiQoIGusiHYrEJ6YycpykfSK\\\\\\\/KSPWGifqOOqOF6IBUFbW3NM9sX2KMKaNSb0UhA4jwNnchwRWcz0q251oAljbJ00smbI7V77GIhm8WcDpF9Ew9fJv4M2uo8OUl5BuQtCqgEeC2jDt+A7IyoAgW1de5dDvOk9iXcsDxsAOnY8fjfsG61qWYlUXINVZmunPshY29n+tqDErpVWvSpwekWtWsg5fG1J2\\\\\\\/2j0wE1xctxU5awPmpuVPl4Btk6ZCALld653e2z\\\\\\\/rETuORVOv82p0Ej53NsVgXKW\\\\\\\/3klJNo83zGOx+lLwL\\\\\\\/k\\\\\\\/Wah\\\\\\\/15zrUpZrJt4Gb\\\\\\\/n9mcUeHj1LG2F64tNCVV4JEmUGVu7d7U8u9imAfakOJXNHhYlDMxbUYgkWEwBTrPhhruI3GJaqi41z4lZmNWBUphMYG2qmtjWUmI4TD8G5Z8lHB5dUhoSN8U7VxgaSYRThtBzRDpc29T5z0E5aj\\\\\\\/20IPjFzh7EJU30Z1sUprcUoh4eoonRGaYJEmftiIJ40lrPbxHPiTDn3X+oUJcPs\\\\\\\/1OdY1LoJwv\\\\\\\/nic6iAfWBGNj+NVNX01A7WZJTDO4FxdA352JmkLSDVjvWzqdBR9HdT1PhyWIfz3aXfXs9g5zXuj9hypYN2NODgs9NZHaHjFAqsq5c2dbM\\\\\\\/U1zaoKisztxGaUmZlrCyHriuwb4421cdVHVqBMAZd4OuZy1Vb7XWzmI2JYNYHf4+2eZ9UbCTXutomNnh5LrL05rgV7xmZ3Hq7Bk5waHlcpnCQbgtF403mwRTWTr22WDy\\\\\\\/rOoBE\\\\\\\/5lfUvsNV224V5ZyRyyvgorgWjWZM4X4KzjIMfz5s2dh40v7Ch07rmlgCB\\\\\\\/DkgjHCK3A8l+A5C4xhtUuepfjrRdMKisT7cxgeLZ1rRaSBZ0FU\\\\\\\/7ZmMe7guxqZHRSJwF0KmzZ5VaXgTJwH+LMp\\\\\\\/2xjWqKkCkzr+Ow2hLQ8ynQCUwPG0PUPSZ563s29JwwH54DvE\\\\\\\/BvY6TSLjayNI0VzADHecZlQ+MB58Wntuy005B5y6nYo070vPOt4XNI8c0xnuQQdeKdgE9uQ7Nz14Kt9mANudCk0s2khfoSqE1gKsi29jqBgo7P17WwtrmOFZjN+\\\\\\\/IwXgYOXP02rnQ2SXgXpbinmXdYcraU6X8rDiRaZs1fEBqY+c3zt4Sh\\\\\\\/dtzec0ZwuDP4oOYHjF0U15HyNPNK17x0uK2FasqFsCT3VytdacJE0a97PtsEf8EVR1pj4K0NWMbhJOTutWaW+LvP8wNM6cvrlvhFsDHAbyGMvfn9PQ6jMaq8Fe87SyHYyCZ32aq5KIVODojyuPaBqhva7iehaY0QdbbPHMnwpyQHucnVuVxZySmMVHGgP2XAA3poA0dh0AhGAiKtl2TsS4diOPjd\\\\\\\/V7JWeE71nwk53g2l9yN\\\\\\\/eMVfhqD9lgPqpD4Cr+paPP+UscrtXysMMY4Z58DQMlaUrxWF0VMlPqMDaVJG4Zl+ATX6VgydPfLBVgUtr4qUUHA\\\\\\\/YG+GpG\\\\\\\/wknP4w7kgUx+uT1N93ZPkCT9mcnHIarONPq3bNe5IzKHl91grus\\\\\\\/vLmDWMQicOkhZmZA4pLVFAYCp8LkEiOpqS6h48Q5zMzCiSpcZKuxltFV+0AY03n9e\\\\\\\/MbwTU379mHaYPs+uQMCXDm1u8decm\\\\\\\/7JBFqwh7PFXHDclC7fIArLiZH389F+rszMJLzIqUSC2lMQctH2\\\\\\\/XP6MzZYVRWhF3BfS7kxcHnlQ+vm1HOGdyHYtYc+pQLkAL7YQj4dbLOBFAQCR0fiL13Nq9WTSQ7fUocFNWPK8ZEHFZxsa\\\\\\\/S\\\\\\\/9jESuVPltKJ+9pxBwO0wWIck2DokZau1DGozQn3Z1X0xAORNXUZeVjeIAq02Ua8k0a8ZPcqsYiMEKbsJeJnL\\\\\\\/E\\\\\\\/hHRcpK4QpmpPa7m61BMXuXGkvgWU7A5vfdklkkeur5QVN4MB8fnDSygj0GEc4Ou4OOpgD7wvQwbVupSc0P8h8Dvdhspji4mL+0IMM3834s2KTeujhCye5i7B1j7l1hrNcS3USaRm52iuhOn9j5r3n1VvKGq+uXE+AE6jF4T7k5YGiRND3VelEdaZJDrjfP0PVk3+HWTMcgf20a+dRdGhErEcrnwetOpZd8ZsLjyhe02eOkRcZCZoWNoFBRPqTA+Nh1HzACSE20pw1aRJOn0IykS2n2U6uunoDoY59BezKj6HeRs7WZtnsm+UkISzfIy8GY7o2bFl4oeEZ07BUkI8u1iyf+71vUg6FjNKc9el+63mM3wHz9ix\\\\\\\/Zg\\\\\\\/mXkGKeMcFBQYAOfrLMlaToSOeA+6iIyAhRYpABjEaYqBKlnvh4gQBlPnRbztMmXTqpprcz4sittgoUX2V4g88ewJBjdubkkJBMso43PyuyG9w+GVZWHkks50\\\\\\\/h63HEjkKU+p7K8DXq61dlFqheFKPSEo45wHLrxZ7I1kaHJYUKTgTjRwBOLS\\\\\\\/pil8kdoZWp\\\\\\\/vcaM2QYM13GaRcpW43t8HonkwzMkrsAYiYOf9WlcGGDPUhiL07u5jmepZsm3HND9mtCNp8g+L9eMl3vBfUPOeIobMOSfHO6Z8tm\\\\\\\/TK9k7Jbv8n3IPthwHmbvMZkJlrWfLkiq0EPuz\\\\\\\/snWuzx66QW7FEkAhKenBkONdtaRGhM0WGbqmpej51a9q636i4g2cqZGWdMg9ugTzLH2xeaEv\\\\\\\/GYCySU\\\\\\\/nXhwF4bgWOPGQt\\\\\\\/K0fpyCpba54XmdWiygb9Hv7Ps5A1QPVyPT2ISrTOmNX3Wo15kFbo5IUugEsliYDUon02B3Xy8VnqTqxMGAzXlJ6VF\\\\\\\/PlzM2VAeU44FtRjjX4t\\\\\\\/sMNW2xqtmMgpHD0WSibSSBiJODPdbukZQvxX+dy6KHfB2QKLsusmBdxAACQce+qVYk1s8ioExts0iz\\\\\\\/00J+POgIH5jBQ\\\\\\\/IPzRHmc1mqPmRQMfvxFrhX8EeDPOBZMeK0GQH6goV8ge0C+SzmzgbsX+uPFQREYDs9eXVFUPc1Wcn2u+PHdk+DKU8J0n\\\\\\\/6G76GIcD1XMVFgqQHRA4gF4NvuBGLV53TOsaqpZ0aa0UFWzj5hsA5nlWkRzo3+ftrY7C6ttvOnK3tutgJZNbQ2Sh1CIkcbf1na2LuZ+pUi\\\\\\\/f0FOABJiD7MljILgnrWOTqurbrZrguo0A7TF4zYnRqt56qzLR9qgUtreqkOHDemhQmTRezOqY9uxSIUryS0y5tVTVJrDK\\\\\\\/i4bw9Hi+xG3WJWVc2yFsBlBstky7IcybjOA0gkPLpyRjzDAqokqonvdhHRtZLeiNSTSKCg6\\\\\\\/jmy+JconZNgRmS+bg5Fc0r88Hy0117bVsLTSgyzrLFL0EJXVZ8Kc+3jymFsTJpDxO5x6uaHvnP8R0x9dLEPjEPPFcKSlnMvRGDyRO+rKQDkJlkCuyGb17UfTkywLiQ+HKYfvhVU3pVUYj7MHt\\\\\\\/buJFqxQ+CynDYXFzmY1J+fFE6ZsqhUcoba2zk74Lz83x3Kzh5Ksapw4R6M1TNuNFXj7JAi5tRT3gGt4wgJZ\\\\\\\/QDgbylJwhUJFN4LBXJJWFW3leJ4OflxDG7IP1TpSd+ybJs\\\\\\\/YwpNCPk+QmwCyZPhdf8qQ1p8f2Xh849MO7BYPLfIXhkEGk6XhjQU\\\\\\\/ekpgfUyemsQfCIF7bK2lJ0oSVu0P\\\\\\\/F92QSvD+0fnJcVQsbBkXsFOknjsXuT4w\\\\\\\/CkCo9f4+9Xad8DQ9UF0z5uRNtNgruBRDg+Vt5BU+fvx\\\\\\\/mTTC93OF3wSjXSwNbrA6Bur7jBg7kBP+cQKU6XNeYepO\\\\\\\/BZPz7Y+HXi46\\\\\\\/CDlnUwbME+o2+8KXTBBs3FcxWWpcSDqkRLwBO0f6li7xdWBEXOVG5W8Ip2fFO3Nfd\\\\\\\/TpF8GbUERil9abJ5CfTu+JhtY0CKSGaDUDR8uxS8DnrkRMJ+k2T9D5Y9Acdtw5UfjcJvAX5fiHL9YbQE3PCE77g1atoflTZzBXAvzd\\\\\\\/XIhpZ5PuGrpRTY7I3iNNMN1XBuUg7BTruAi7UF9+2ueXK0vUQGC6AUpl0pnveYX79\\\\\\\/VcpNz1QCVJSLODjM6eGnTpxTqlONVvXXis1hEjjReDFGTIiea92yap1urfHlUcHcr\\\\\\\/nQ9BHwfCUrE90Teghd5p\\\\\\\/We2sosB9SHa9HeQ+FOdOP2f+PmS+M\\\\\\\/Crik537SxKorMhx+UgtGv8JLLtklDuM153LILKoYIs75YmTueBU0GLEcULJElhTQEGLmYoTzjjAzqJbFw64QTNdj8fXlDHZYgQdnw79XTi7KekiHK64pbLQ\\\\\\\/CYAuzBcvlA43NLWM\\\\\\\/imhL6h6\\\\\\\/jnYVtFhZBWjaQSKSSYtMTvnyOcz56SUlqCIifoEkj53ZVF5jpTzT5bch6RQggGUWa+IojN88f63\\\\\\\/jHu+au+P\\\\\\\/wyeiUyg\\\\\\\/+S1La8nGNKHy1mNuaKPMRiFPgpZ\\\\\\\/9DUaaN4w0IEn8u0Cd4k0i\\\\\\\/535z9i4gxP0FlTzQAI9ISaxpp+IVWoc24tfuIL9UjA0KVOa2goUERlayjUJ5DArkjwjckAKZ4Kiphp1VBCTdyX+t6oJVDosZrn9M\\\\\\\/ejr6IZX8pVDuOxnSIQAuEO1P9AQ8PcdgMbAQGLc06BgH6762t4niTuTjrklD3N+nNR6s4BDADvHoxY9\\\\\\\/hxxKG45ZGj9YrYkRVmuqKMSVxr45khuY8onWAwKBsTP580FSyJNHRRJAiBSpaCre1Oid5i3NJqASnDOOAPH4375AeVlrq7dZJ\\\\\\\/vzV3PvwSzR6dN9x1\\\\\\\/17Lg11\\\\\\\/+LtGmPpsVV0j7lb3SCQBdqsP2wTO6EL6CzXvj386S80yCM4Nli1+byvgWFghKgNlFLINYW9jizwlbmjo2JC5\\\\\\\/2iN3S0zYQAb0tMQ1DfCDlh0yLuiOPU6J5E1T6y8OavT2i\\\\\\\/I\\\\\\\/FK\\\\\\\/X5KMvzFT5vyau41BXzZ0wsPGdZaovTJUS0sw6qkS3IMH8ummyXQXbXIWLhkKcwevMrR9THKr8+uqIrIurB168hxnxZmGPbB9ihc2woFbK6gLQ5iK3ONIzIx5d\\\\\\\/pjZc\\\\\\\/8Nu8Chs\\\\\\\/Cmu5UrZ1WnPMzHIYd523ZppZ7KIlegr\\\\\\\/8kJ90frGgfHa27YENrJeeb\\\\\\\/pNYmLile\\\\\\\/4vzX+Ak0MT4KTK0YMGqyK9udaTVUHDNCHNw\\\\\\\/h1TN8q9\\\\\\\/itE49jNodqjkJL1Al5pwABWgix6nMrpDb0lmC9PaPSRDghQMEmbdY9eHdcWfsDY1jJaONgwyWcgrLE4V13refHMDGuklGj8nkgRvFrzOugzrXoC8pJlynthurWSXtHAA6KLmamvPLZlXYgyzjR2\\\\\\\/IcyVaQeITThwMLAg6Mdkvi1aUNIrf7+ZSDcPokP8tIIEou9xM71i1NTRxJepiQFrn2HJvJvA7qaEgThTszX9ypA3g86Xa5dn6EfW4vCNuG2FfNxOtmgA0Q+sTe8fqNu4wTVZeTQkZcWfvjWgmk6shr7lnrEc\\\\\\\/5skhWEFBmBIC5tAma12z1tfEocOMoOij\\\\\\\/Y5UOr8xvDRsuvZ8QJ5F+VBahxj\\\\\\\/XvoxWWseW3X7xHzYq7p6bKB1v1R5YOfQLAky1eile42hbmg63YPLC6xDZnQfjCH9fviXtZPttBKJEP4\\\\\\\/EIedFNFNY3oKWN29zJc7oTTK6BTUYOZSU8XPKfdrnkxi3aDdA7F2JTqQSwcAVpNeNcykGzyV\\\\\\\/8LNnYBfTNu0uc1zotgq76OJiVnzTTWbkcHni6c+4na4qBZKODqkyTDW+qMfpOLUrLZMeLhW18Xzp72F0\\\\\\\/ZSB0IWsEX4VR2dJWkr8WlBdvH\\\\\\\/2HRgGRlT3OJ44vhGF9ADLiwlqmOXOSROZojWmNRodT+o9X0rIrBQyaGDF9bWfk2MOBoxtBPDIf+AU838AhnC7uUKU5Co3dje6CrM+1nCaY5XPnzs9UCL\\\\\\\/Z187wjypQ6d5hcBXzc7vmTh5DtflDLu5fKgT6wb6dIqNJEKvBoBGg12iR+Y7UNeZo0riHZvbkPg6sxeND1dx+T+r7w1Pvo4OPVbLYGe61zMPMWYrB6gxjSWerJpnh2OsmcolweeDCAqRhOstImF4e\\\\\\\/pJfA+I0OqsSOW6+pAJly0IhxrRYuzTI9t++oGs8qiNVqmek3WdzjAHddvbT9uuUzt5etd0NYMWHT6r4Dc8YJbA20Rj9R+PSywB0NmymU7aGMaTtfm9cMKwPIRswrUsUx3eTInOezBRx9\\\\\\\/0MU8SncZyeP7z0vZjjyhnnRSOEGxnGEX\\\\\\\/TQM0eZNQmQAPNNa7LlLeRrUR\\\\\\\/45TOy\\\\\\\/+8Go4kien3OXBTU4ESq9TZG2y3+hxOSLqylmr3AapIWQf68LJriKPC2NUs6iDfIuH425hEtNrulDrF+mcg6ExNYh0fBkLnVC+AbzUvYLMa8JUC8R3de01zLRNrI7pBpYMrWexFWnCA1kPukL3djiS2ZaESawbk9lI+nhxRSact5fLnwpJlzkbLGXR1X3oLMGWFGW8EG30liIS3yO7aZ5Ra+33p3bvk2jXNe4yX+uN3FmUqm0I6Iho29u9rt+hbZFPygyiKRn97iLy9m7kuDx6376L5NetBpyrfx0se957inE6yHlb72dlwYSK7VBC4NxF3V0vb0cVBCwtswybGafEkbU+qSY1EA7gMZ\\\\\\\/nlhzttakdjs57+hzF0eY7NZiSu+vlSHdLAHgfCFj0MMQQbZ99TrO3fVCPQY4OW825jYoNSWVdX\\\\\\\/ily4tKIznF1sUPNr02QnWpSylzA7X2KeqysMl5D3w06hssMIhYawRwGYflfgHv3kf08xTPekQgzbDil1PXcz+yT80+amhrsNiGJkuxyPeMOVdopXzaL60+XIUkDUAKPbcbnXEMZy33XE6IKAvdCfV9MnYleZ2tMsiLUwVp9kjgbW2CZm0uxSIpQD582vB\\\\\\\/iTMxYUwJY4bRiOfgnLy2h2z\\\\\\\/OWB5IlAHFikoOkf\\\\\\\/tX5NOKJ7uNGFnM74j2nU2aGTfLzLtF0OQEpIAoPwBiEooGJy62RKxFg2z0qam+UwwgU7fPQlt3AP8GVeEKjccdyKWRSEmph7T0z3JTVWxB6qHbPjeUYZWO9qRIzr+Vi\\\\\\\/0nCepbXbsoT0BqJpWmRlcmb311RFQP3OxhbH5OCrlfES0YbhbDpMNGNvyVJDIqtXqFHn3BDpEGtvixlg35ERQTNhkTiXlv1rH8Xpoc38D9e93IGfCZIpyXTNBrxbQWJH1V+rYgeOwv\\\\\\\/PgLFoWuCK8O\\\\\\\/TJPpTa5fXa5\\\\\\\/80MsJ\\\\\\\/1WgAa23yUfhuxxKgLMN52QDUnapmnCIdVKzl3YTa+ZNdb76y7jnj3AYPD45hErMwRlqm2uBszD8Vl68uC1\\\\\\\/CKbGQM\\\\\\\/V2ErH9wysccCqQCompBb96NAimhVH2ix2vvaFigBP5U\\\\\\\/hzjq7RAKZOx9JlsoXLl+ujmpL1GyXKOtmabvUePCTZytFJxzazW4D3DwygQqEKhPm2qyvNQJw+rsQrNuu3cAeMkyXK72kQtiQlpvq7toWXpiyhOovWOzElM5MAAaeSSbLu4Eg8Qr4ResQ5AnJeYkm+w3MalPLvxCCxaSvwFl1XCRWJSDyfXiZFXz0bXLXRYuRcb2OlFnSc+WjhrAvHBTsdPC2\\\\\\\/WGvl4Nxdo+lP6sydKgbI4pc\\\\\\\/\\\\\\\/IvsVLVNtOxl+jZWNV6zumF1L195QK+z2dgLl6JgtFx1W\\\\\\\/AJly3xMW44o9Y9kR8qzqb+Ubr7nSD3d8vOxi1yhWE6zuGJ7wbzBaBwshcJLqqtv6K7syJKmiRgv2J+pxEJ1EzWFL9UYfXiuCyZsB9dvRRYxEqHSLYs1AETgmSNYQ8a7JFVd\\\\\\\/rPoOGuQpOzIHE8qF6qp+voe1f07wUzInfe0R7ysngOAHAuaE\\\\\\\/KYy3kLgEL2e4+QKS1iVrMdEuXnZm6brFcZTAf0l4cGmfkluo+vzXa1sWkQzmH1l9lqKfUSoc2iO8LP3oipMP1ndp5iGNAnR058NimlQ5QpyczhShk81zrx05Q==\\\u0022,\\\u0022iv\\\u0022:\\\u002260FfkmcktRTMLU1fqLlfPw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002255e19d03a161c403de1ce3a617824946829e1384b2d0e351c2937ba886fa2c58\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022b+FsOCV8UKLdiOo6Y0DqcoLdpybQ4LTv6qnjha74J2RPW042DYLxH0E1TJJmKVwKva0qiFjMCt8yuMc0KphwX0o2DTKUtqcZxhP1iKgSMUOEIOb42dC1QKJP\\\\\\\/O5WZE2Fr3piQ5VifQPAJbOEXcH7+loIhPJkJXdXUMNb2cYbB90cxutA2q2konEWWpo0H1KMQ0z8QeZx8E4l93+HJcjwDQuYJ\\\\\\\/PrylDS\\\\\\\/mSPmUsN7hJK2g232NEyRtC4XnxK3vRVZtVP4D3m79apxdm5ynIe\\\\\\\/RD34AI5xiJ9zbUGZ5dZMrCBJqPSoKB4gv9MYQkTk1PSAfuzv0+vhBV44FbtjPZ8XBGcA0p7gT3yCC1ukEtSky33hs+eMbXGhECKH1utw0\\\\\\\/JFBApkz14PMhEe0vfgLQP6fJa2H8r4tzs73sZr1ZBaJdDbQqRI32GrRk9l5HXV0xwEXxz8LpxJ+hI9lS4SOquo97NBRvBsa3zpMWCG1R2z+jJqO7hi80smx1lfP8vrGPbM7c2FyxN0ZK\\\\\\\/LAW3F\\\\\\\/f1hboHF9q\\\\\\\/Zomv59oOd8s2aVJRXNBMEeAjD0ydBlg+aW70BsB7IAAt6BBoGLRK\\\\\\\/aFKkVaeJEfb58o+g+KRMUzgzuTEQCcMr0xfIhNcRdUPS0GdNlwz\\\\\\\/18SZJEPpjLGfLgeTYMipEBN3TmC+yn0K9Ks7A0CGpVOHjRDCEWPWsnRFY0jGJgc2S\\\\\\\/sNB6YKvOnJ8DkG7XoD9FCqCiJgSaDW4lChtOG80lj0zPoRirhr\\\\\\\/5F5W+9Qny4fpfexplGW+6596xF2TE\\\\\\\/2vJIBirHq9cWhacxiJ1xu9rms7EMJHWaSfiGuG\\\\\\\/5JNPhT+KsDwoIOlCKLTzajgcojjKTzEdLmw8QiTdqVs2Veeht3ej\\\\\\\/9fgsd1\\\\\\\/cEJhZhhlhYsyaGuEA48zpJExP8P185j1WRoGS+c7+r6O\\\\\\\/5X\\\\\\\/sy4+yFtc3TjUvQdDMh5YSZfI7aP7\\\\\\\/CrF6RkbJ7V6t\\\\\\\/rTwiJy\\\\\\\/0LLZxdwUEOl78ncfqi\\\\\\\/Pbpcj8JG+TsgJK75MqSyaWcY0JbWGWYpZNbvaVxuHxaQ5XDjur02owXZAWEsSQNsgCxSZlO3XHy5fOF0Ux2Ho7PZfCGVaN53y+hIjqKaOuXjLH3Zk2CCQqcMz1sGB4hDOcQM7yRmDDr8ky5jVzhWOC+guYgODeePaZUEqoUImZZaZh7QX6H0TkDGhAw1p3Dg3ZhlBvp7RGF5vxKI7qZk+kKk3jhbnkwHbduY0Kwj5fKbcpuIRqzweQvfvIJe0OHJD4M7LGaz64ybEEYIkgs1o0csoEhir6ytxPYUlnfZ6s1IWaCPg3WJpsSSTn7nTM5bPSufYh8coyHIIvNvRYVu64KyXZ0xvkSD7VkR2ZhJhVWjYaP2vrQxubZqJdGedQskI\\\\\\\/5GUvySnyUUxucdTHLqPESRBhk6ipLVIh8hGhYWE4FqcDwrw8ty8eyCu0Qr8t0mNcQRb+L9MefLp7vrncj75qTXyfch\\\\\\\/J7wjS+pCEs7s5JT6OZwC5eKfWn2F2y8cYjfATF+CyEc9QqjYB5KYntjCkHXk9rjJ7Yx8j1cNcK90KVoLmugVVFnN2miVkSH0ZWsKa9r8y4+ApX6gU34+kRQlpCrGLAf4ISUOJI2cEX35dgkP+EipyIgEwBbmAr9Pkvp5s7JLwzHUGHNxDeyTAOgh0zDx+vFj53GmROSbBGQZsvO49LoNKXzEv+7aV4UXJtrgqXzcc\\\\\\\/PJIWuA6joNKH\\\\\\\/2QnMs0R62Ru5tc5QI0cTflBJKrV1rofYXx57485pMDCf80gwrI6BVgb456Y2EwtwRzhZygqfbOJ6YT1scNq8CXB\\\\\\\/+J0Utm7AhnEKzdhmfq7iCUYbNdbyReIG4td\\\\\\\/mholOkGSABml6U6ZuBAjXgHyVrrZxQSy\\\\\\\/DswmxUFEAAhOIvGoUdeZtA8tIU2ltdxY24x9LkLekIh7n\\\\\\\/\\\\\\\/\\\\\\\/UI87HQz5mM0CW\\\\\\\/fLYj4LWk3h\\\\\\\/Fp0QLjiqiQTX5ZqWlKzxCaB1Ej92OXqTxkCHKIgcVtfuQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022ATFHShfU1iVvDCCqF18aVQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022602b95ba06da0b466cf147de4e67128c4120b1d01fb766a78a045b3ba16cd46b\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-493732 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-genomic-prediction">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/8/2739" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-onzIYOxzoX834o5n-fxJV7aRBeAsw-ED3zvDTB96l_Y" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/8/2739" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-4Hlxc2pGUGUCL8JQMVmrit0-ckRrJSP6Xbk9yoXOUuk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/8/2723" title="Comparative Transcriptome Analyses Revealed Conserved and Novel Responses to Cold and Freezing Stress in Brassica napus L" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/8/2749" title="Genome-Wide Profiling of Polyadenylation Events in Maize Using High-Throughput Transcriptomic Sequences" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493732" id="node493732" data-pisa="ggg;9/8/2739" data-pisa-master="ggg;g3.119.201004" data-apath="/ggg/9/8/2739.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction of Additive and Non-additive Effects Using Genetic Markers and Pedigrees</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3906-8884" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Janeo Eustáquio de Almeida Filho</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-3849-9834" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>João Filipi Rodrigues Guimarães</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9536-1113" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fabyano Fonsceca e Silva</span>, <span class="highwire-citation-author" data-delta="3">Marcos Deon Vilela de Resende</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-8973-9351" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patricio Muñoz</span>, <span class="highwire-citation-author" data-delta="5">Matias Kirst</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-2367-0766" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marcio Fernando Ribeiro de Resende Júnior</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2739-2748;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.201004 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493732" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Janeo Eustáquio de Almeida Filho </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Plant Improvenent Lab, Norte Fluminense “Darcy Ribeiro” State University, Campos dos Goytacazes RJ, 28013-602, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Janeo%2BEust%C3%A1quio%2Bde%2BAlmeida%2BFilho%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Almeida%20Filho%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJaneo%2BEust%25C3%25A1quio%2Bde%2BAlmeida%2BFilho%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3906-8884" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Janeo Eustáquio de Almeida Filho</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">João Filipi Rodrigues Guimarães </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Futuragene Ltda, Itapetininga SP, 18207-780, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jo%C3%A3o%2BFilipi%2BRodrigues%2BGuimar%C3%A3es%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guimar%C3%A3es%20JF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJo%25C3%25A3o%2BFilipi%2BRodrigues%2BGuimar%25C3%25A3es%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3849-9834" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for João Filipi Rodrigues Guimarães</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Fabyano Fonsceca e Silva </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Animal Science Department, Federal University of Viçosa, Viçosa, MG 36570-000, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fabyano%2BFonsceca%2Be%2BSilva%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fonsceca%20e%20Silva%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFabyano%2BFonsceca%2Be%2BSilva%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9536-1113" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fabyano Fonsceca e Silva</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Marcos Deon Vilela de Resende </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>EMBRAPA Forestry, Colombo, PR 83411-000, Brazil / Statistics Department, Federal University of Viçosa, Viçosa, MG 36570-000, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcos%2BDeon%2BVilela%2Bde%2BResende%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vilela%20de%20Resende%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarcos%2BDeon%2BVilela%2Bde%2BResende%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Patricio Muñoz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Horticultural Sciences Department, University of Florida, Gainesville, FL 32611, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patricio%2BMu%C3%B1oz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mu%C3%B1oz%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatricio%2BMu%25C3%25B1oz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8973-9351" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patricio Muñoz</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Matias Kirst </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>University of Florida Genetics Institute, School of Forest Resources and Conservation, Gainesville, FL 32611</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matias%2BKirst%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kirst%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatias%2BKirst%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Marcio Fernando Ribeiro de Resende Júnior </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Horticultural Sciences Department, University of Florida, Gainesville, FL 32611, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcio%2BFernando%2BRibeiro%2Bde%2BResende%2BJ%C3%BAnior%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Resende%20J%C3%BAnior%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarcio%2BFernando%2BRibeiro%2Bde%2BResende%2BJ%25C3%25BAnior%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2367-0766" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marcio Fernando Ribeiro de Resende Júnior</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mresende@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">mresende@ufl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/8/2739" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:493732" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:493732/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/8/2739.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:493732" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:493732/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/8/2739.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:493732" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:493732/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/8/2739.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:493732" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:493732/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">The genetic merit of individuals can be estimated using models with dense markers and pedigree information. Early genomic models accounted only for additive effects. However, the prediction of non-additive effects is important for different forest breeding systems where the whole genotypic value can be captured through clonal propagation. In this study, we evaluated the integration of marker data with pedigree information, in models that included or ignored non-additive effects. We tested the models Reproducing Kernel Hilbert Spaces (RKHS) and BayesA, with additive and additive-dominance frameworks. Model performance was assessed for the traits tree height, diameter at breast height and rust resistance, measured in 923 pine individuals from a structured population of 71 full-sib families. We have also simulated a population with similar genetic properties and evaluated the performance of models for six simulated traits with distinct genetic architectures. Different cross validation strategies were evaluated, and highest accuracies were achieved using within family cross validation. The inclusion of pedigree information in genomic prediction models did not yield higher accuracies. The different RKHS models resulted in similar predictions accuracies, and RKHS and BayesA generated substantially better predictions than pedigree-only models. The additive-BayesA resulted in higher accuracies than RKHS for rust incidence and in simulated additive-oligogenic traits. For DBH, HT and additive-dominance polygenic traits, the RKHS- based models showed slightly higher accuracies than BayesA. Our results indicate that BayesA performs the best for traits with few genes with major effects, while RKHS based models can best predict genotypic effects for clonal selection of complex traits.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/genotypic-value" class="hw-term hw-article-keyword hw-article-keyword-genotypic-value" rel="nofollow">Genotypic Value</a></li><li class="kwd"><a href="/keyword/polygenic" class="hw-term hw-article-keyword hw-article-keyword-polygenic" rel="nofollow">Polygenic</a></li><li class="kwd"><a href="/keyword/oligogenic" class="hw-term hw-article-keyword hw-article-keyword-oligogenic" rel="nofollow">Oligogenic</a></li><li class="kwd"><a href="/keyword/rkhs" class="hw-term hw-article-keyword hw-article-keyword-rkhs" rel="nofollow">RKHS</a></li><li class="kwd"><a href="/keyword/bayesa" class="hw-term hw-article-keyword hw-article-keyword-bayesa" rel="nofollow">BayesA</a></li><li class="kwd"><a href="/keyword/genomic-prediction" class="hw-term hw-article-keyword hw-article-keyword-genomic-prediction" rel="nofollow">Genomic Prediction</a></li><li class="kwd"><a href="/keyword/genpred" class="hw-term hw-article-keyword hw-article-keyword-genpred" rel="nofollow">GenPred</a></li><li class="kwd"><a href="/keyword/shared-data-resources" class="hw-term hw-article-keyword hw-article-keyword-shared-data-resources" rel="nofollow">Shared Data Resources</a></li></ul><p id="p-3">Pedigree and high-density DNA markers have been used to predict genetic merit of individuals in animal (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Wiggans <em>et al.</em> 2011</a>) and plant breeding (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Resende <em>et al.</em> 2012a</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Resende <em>et al.</em> 2012</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Crossa <em>et al.</em> 2014</a>). Initially, genomic prediction models were developed to select sires in animal breeding, accounting only for additive effects (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Meuwissen <em>et al.</em> 2001</a>). However, the prediction of dominance effects represents an important component of models designed for breeding program that focus on crossbred populations and/or hybrid production (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Zeng <em>et al.</em> 2013</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Nishio and Satoh 2014</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Resende <em>et al.</em> 2017</a>). In vegetatively-propagated species, non-additive effects are especially relevant because selection takes advantages of dominance effects. Hence, breeders can transfer whole-genotypic values of individuals to the next generation through clonal selection strategies.</p><p id="p-4">Many statistical methods for genomic prediction have been proposed, which differ with regards to the assumed trait genetic architecture (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gianola 2013</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">de los Campos <em>et al.</em> 2013</a>). For instance, BayesA consists of a whole-genome multiple regression (WGR) model where each marker regression coefficient assumes one specific variance (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Meuwissen <em>et al.</em> 2001</a>). Consequently, BayesA should provide a good fit for oligogenic traits where few genes explain a large proportion of the observed genetic variation (<a id="xref-ref-34-3" class="xref-bibr" href="#ref-34">Meuwissen <em>et al.</em> 2001</a>). However, prediction of non-additive effects introduces new SNP-covariates associated with these effects (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Toro and Varona 2010</a>), which may prohibitively increase the number of estimated parameters. The semi-parametric Reproducing Kernel Hilbert Space (RKHS) method has been proposed to account for non-additive effects, and is less computationally demanding than BayesA, especially when the number of individuals is substantially lower than the number of markers. RKHS combines features of non-parametric kernel regression with mixed-effect linear models (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gianola <em>et al.</em> 2006</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Gianola and van Kaam 2008</a>). RKHS is equivalent to the animal model in BLUP (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">de los Campos <em>et al.</em> 2009a</a>), but uses kernel matrices that are different than the numerator relationship matrix (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Morota <em>et al.</em> 2013</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Tusell <em>et al.</em> 2014</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Morota and Gianola 2014</a>). In fact, the numerator relationship matrix is a special case of RKHS matrices.</p><p id="p-5">Empirical results showed that the inclusion of dominance in BayesA improved the prediction accuracy in complex real and simulated traits in a pine breeding context. The additive-dominance BayesA was also superior to other Bayesian models for oligogenic traits (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">de Almeida Filho <em>et al.</em> 2016</a>). In addition, reports that analyzed complex traits in other forest species showed improvement of accuracy in genomic selection models with inclusion of pedigree information in additive-only models (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Beaulieu <em>et al.</em> 2014a</a>, <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">b</a>). Thus, it is hypothesized that inclusion of pedigree information in additive-dominance BayesA can also increase prediction accuracy in structured breeding populations. The same hypothesis can be considered for alternative methods to BayesA, such as RKHS, which may also provide higher prediction accuracies when combined with pedigree data (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Crossa <em>et al.</em> 2010</a>, <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">2013</a>).</p><p id="p-6">To date, few studies have compared the prediction of non-additive effects from models using only genetic marker data, with models that integrate genetic markers and pedigree information. Moreover, the ability of RKHS models to predict whole genotypic values has not been assessed in the context of forest tree breeding. This study had as objectives: 1) evaluate the impact of including pedigree information in genomic prediction models applied to forest tree breeding, 2) evaluate RKHS and compare it with BayesA when it includes non-additive effects or not and 3) contrast different cross-validation strategies in the context of a structured breeding population. We applied and evaluated these methods to traits with distinct genetic architecture measured in a loblolly pine breeding population, and to simulated oligogenic and polygenic traits with different levels of dominance.</p><div class="section materials-methods" id="sec-1"><h2 class="">Material and Methods</h2><div id="sec-2" class="subsection"><h3>Loblolly pine population and phenotypes</h3><p id="p-7">In this study, we used the traits tree height (HT), diameter at breast height (DBH) and two measures of fusiform rust infection: presence or absence of rust (RFbin) and gall volume (RFgall). These traits were measured in the loblolly pine (<em>Pinus taeda)</em> CCLONES breeding population, as previously described (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">de Almeida Filho <em>et al.</em> 2016</a>). The traits HT and DBH are polygenic (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Resende <em>et al.</em> 2012b</a>), and HT also shows a significant non-additive genetic contribution (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Muñoz <em>et al.</em> 2014</a>). In contrast, rust resistance is thought to be controlled by fewer genes, including loci of large effect (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Resende <em>et al.</em> 2012b</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Quesada <em>et al.</em> 2014</a>). The CCLONES population where these traits were measured was created from 42 founders, which were crossed to create 71 full-sib families, with an average of 13 individuals per family (SD = 5). In total, 923 individuals were genotyped for 7,216 single nucleotide polymorphic markers (SNP) as previously described (<a id="xref-ref-2-3" class="xref-bibr" href="#ref-2">de Almeida Filho <em>et al.</em> 2016</a>). From the total set, all 4,722 polymorphic loci were used in this study. Traits HT and DBH were measured in a field trial established in Nassau under alpha lattice design (Florida, USA), when the plants were six years old, in eight clonal replicates (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Baltunis <em>et al.</em> 2007</a>). Rust resistance traits (RFbin and RFgall) were measured in greenhouse, under randomized complete block design, as previously described (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kayihan <em>et al.</em> 2005</a>, <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">2010</a>). The phenotypes for these traits were adjusted with the following linear model:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-1.gif"/></noscript></span></span>Where <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-2.gif"/></noscript></span></span> is phenotype of the i<sup>th</sup> clone evaluated in the j<sup>th</sup> repetition and the k<sup>th</sup> incomplete block,<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-3.gif"/></noscript></span></span> is the intercept,<span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-4.gif"/></noscript></span></span> is the random effect of k<sup>th</sup> incomplete block nested in the j<sup>th</sup> repetition <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-5.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-6.gif"/></noscript></span></span> is the fixed effect of the j<sup>th</sup> repetition and <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-7.gif"/></noscript></span></span> is the effect of the i<sup>th</sup> clone (considered as fixed to estimate the least-square means or adjusted means) and <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-8.gif"/></noscript></span></span> is the error of observation ijk <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-9.gif"/></noscript></span></span>. This model was used for analysis of DBH and HT. For rust resistance, the incomplete block term was excluded. The analysis of variance of these traits is described in Table S1.</p></div><div id="sec-3" class="subsection"><h3>Simulated population and phenotypes</h3><p id="p-8">We also analyzed six traits with different genetic architecture, in a simulated population described previously (<a id="xref-ref-2-4" class="xref-bibr" href="#ref-2">de Almeida Filho <em>et al.</em> 2016</a>). This analysis included traits with oligogenic and polygenic inheritance, and three levels of dominance (none, median and high). The simulated population was created following a standard forest breeding program model, in two steps. First, a base population with 1,000 individuals was created by randomly sampling 2,000 haplotypes from a population with effective size of 10,000, generated by 1,000 generations of a neutral coalescence model, with mutation rate 2.5 × 10<sup>−8</sup> per generation (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Willyard <em>et al.</em> 2007</a>). Next, the breeding population was created by phenotypic selection of 100 individuals from the base population. These selected individuals were randomly mated to create 1,000 individuals to be used in the first breeding cycle. From the 1,000 individuals of the first breeding cycle, 42 were phenotypically selected and crossed following the same mating design used in the loblolly pine CCLONES population (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Baltunis <em>et al.</em> 2007</a>). Ten independent replicates were carried out for each simulated trait.</p><p id="p-9">The simulated genome had 12 chromosomes with 100 cM. A set of 10,000 bi-allelic loci (<em>e.g.</em>, SNP) were used in developing prediction models, and 30 or 1,000 QTL were simulated for oligogenic and polygenic traits, respectively. All traits had narrow sense heritability of 0.25. Three levels of dominance were evaluated: <em>d<sup>2</sup></em>: 0, 0.1 and 0.2, where <em>d<sup>2</sup></em> is the proportion of phenotypic variance explained by dominance deviation - <span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-10.gif"/></noscript></span></span>; <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-11.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-12.gif"/></noscript></span></span> are dominance deviation and phenotypic variances respectively (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Falconer and Mackay 1996</a>). The additive effect of a gene (a) was defined as half of the difference between the alternative homozygotes, and the dominance effect (d) was estimated by the difference between the heterozygote and the mean of the homozygotes. The distribution used for a in oligogenic traits were gamma (rate = 1.66, shape = 0.4) with sign (positive or negative) sampled with equal probability (<a id="xref-ref-34-4" class="xref-bibr" href="#ref-34">Meuwissen <em>et al.</em> 2001</a>). For polygenic traits, a was simulated with standard normal distribution (mean = 0, sd = 1) (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hickey and Gorjanc 2012</a>). When dominance was present, it was simulated by: <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-13.gif"/></noscript></span></span>, where <span class="inline-formula" id="inline-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-14.gif"/></noscript></span></span> was sampled from a normal distribution with mean zero and standard deviation 1 and 2 for traits with medium- and high-dominance-levels, respectively. To achieve the targeted values of <em>d<sup>2</sup></em>, only samples that provided <em>d<sup>2</sup></em> between 0.09 and 0.11 for medium-dominance traits and between 0.19 and 0.21 for high-dominance traits were kept. The simulated populations showed very similar allele frequencies, when compared across scenarios. Hence, <em>d<sup>2</sup></em> was mostly defined by a larger or smaller sampled value of <span class="inline-formula" id="inline-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-15.gif"/></noscript></span></span>.</p></div><div id="sec-4" class="subsection"><h3>Statistical methods</h3><p id="p-10">We evaluated models that consider just SNP or pedigree information, and models that combined both. The DNA marker (SNP) component was fitted with the following methods: a) semi parametric Reproducing Kernel Hilbert Space models (RKHS), using different kernels (Ka and Ka-Kd), and b) BayesA considering additive and additive-dominant effects. BayesA was the choice of WGR because it was previously shown to generate better predictions than the Bayesian Lasso and Bayesian Ridge Regression, and to have similar accuracies as BayesB, for oligogenic traits (<a id="xref-ref-2-5" class="xref-bibr" href="#ref-2">de Almeida Filho <em>et al.</em> 2016</a>). The full base model can be represented by:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-16.gif"/></noscript></span></span>Where <span class="inline-formula" id="inline-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-17.gif"/></noscript></span></span> is the phenotype (adjusted clonal mean in real traits) of individual <em>j</em>; µ is the intercept; <span class="inline-formula" id="inline-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-18.gif"/></noscript></span></span> is the genotypic value for each j<em><sup>th</sup></em> individual, estimated from SNP data — this term changed among methods adopted, as can be seen bellow; <em>u<sub>j</sub></em> is the additive polygenic effect (when included) of individual j; <em>δ<sub>j</sub></em> is the dominant polygenic effect (when included) of individual j; <span class="inline-formula" id="inline-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-19.gif"/></noscript></span></span> is the error term. The joint data distribution, the prior distribution for the constant <em>μ</em>, and the prior distributions for the vectors g, <em>u</em> and <em>δ</em> (containing information of all evaluated individuals) are given by:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-20.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-21.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-22.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-23.gif"/></noscript></span></span>Where A and D are the additive relationship matrix and dominance relationship matrix, respectively, calculated using standard methods (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Henderson 1984</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Lynch and Walsh 1998</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Mrode 2014</a>). In the pedigree model, the <span class="inline-formula" id="inline-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-24.gif"/></noscript></span></span> term was absent and u and δ are the breeding values and dominance deviation vectors, respectively.</p><div id="sec-5" class="subsection"><h4>BayesA:</h4><p id="p-11">The BayesA model adopted in this study used the changes in the formulation proposed by <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Gianola <em>et al.</em> (2009)</a>, modified from the original version (<a id="xref-ref-34-5" class="xref-bibr" href="#ref-34">Meuwissen <em>et al.</em> 2001</a>) to estimate the shape parameter of the inverted chi-square (<span class="inline-formula" id="inline-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-25.gif"/></noscript></span></span>). This modification is expected to reduce the influence of the hyperparameter and consequently improve the bayesian learn. The full BayesA model included additive and dominant effects from SNP and pedigree. This model can be represented by:<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-26.gif"/></noscript></span></span>Where <span class="inline-formula" id="inline-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-27.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-21"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-28.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-28.gif"/></noscript></span></span> are the functions of SNP <em>i</em> in individual <em>j</em>, for genotypes AA, Aa and aa. <span class="inline-formula" id="inline-formula-22"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-29.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-29.gif"/></noscript></span></span> is composed of values 1 (AA), 0 (Aa) and -1 (aa) and <span class="inline-formula" id="inline-formula-23"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-30.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-30.gif"/></noscript></span></span> of 0 (AA), 1 (Aa) and 0 (aa). <span class="inline-formula" id="inline-formula-24"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-31.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-31.gif"/></noscript></span></span> and <em>d<sub>i</sub></em> are the additive and dominance effect of marker <em>i</em>, respectively. The dominance effect was fitted only in the additive-dominance model. The priors used in linear regression coefficients for additive-dominance and additive models are described below:</p><p id="p-12"><span class="inline-formula" id="inline-formula-25"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-32.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-32.gif"/></noscript></span></span>;<span class="inline-formula" id="inline-formula-26"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-33.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-33.gif"/></noscript></span></span>;<span class="inline-formula" id="inline-formula-27"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-34.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-34.gif"/></noscript></span></span>; <span class="inline-formula" id="inline-formula-28"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-35.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-35.gif"/></noscript></span></span>; <span class="inline-formula" id="inline-formula-29"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-36.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-36.gif"/></noscript></span></span>; <span class="inline-formula" id="inline-formula-30"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-37.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-37.gif"/></noscript></span></span>. Where <span class="inline-formula" id="inline-formula-31"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-38.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-38.gif"/></noscript></span></span> and <em>G</em> represent the scaled inverse chi square and Gamma distributions, respectively.</p></div><div id="sec-6" class="subsection"><h4>RKHS Kernel averaging model:</h4><p id="p-13">From the full base model represented above, the term <em>g</em> in RKHS represents the whole genotypic values explained by the markers, including additive and non-additive effects such as dominance and gene interactions (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Gianola <em>et al.</em> 2006</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Gianola and van Kaam 2008</a>). Here, g was modeled in two forms referred to as RKHS Ka and RKHS Ka-Kd.</p><div id="sec-7" class="subsection"><h5>RKHS-Ka:</h5><p id="p-14">In the RKHS-Ka, the whole genotypic effect (g) was explained by 3 SNP functions:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-39.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-39.gif"/></noscript></span></span>where<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-40.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-40.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-41.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-41.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-42.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-42.gif"/></noscript></span></span><span class="inline-formula" id="inline-formula-32"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-43.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-43.gif"/></noscript></span></span> is the square of the Euclidean distance matrix among the individuals using the SNP additive incidence matrix <strong>X</strong> that takes values -1, 0 and 1. The <span class="inline-formula" id="inline-formula-33"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-44.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-44.gif"/></noscript></span></span> is a bandwidth parameter that controls for the relationship measure between individuals j and j’, for a given distance (squared Euclidean in this case). Large positive values of bandwidth result in the relationship of j and j’ being close (or equal) to 0. Small positive values result in the relationship of j and j’ being close (or equal) to 1. The kernel averaging method (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">De los Campos <em>et al.</em> 2010</a>) was used to determinate the bandwidth components in this study. In this approach, each SNP function g is replaced for two or more SNP functions with the same distance (squared Euclidean in this case), but with different bandwidth values — in kernel averaging these bandwidths are not regular parameters, they are constants. The bandwidth values <span class="inline-formula" id="inline-formula-34"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-45.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-45.gif"/></noscript></span></span>) used in <span class="inline-formula" id="inline-formula-35"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-46.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-46.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-36"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-47.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-47.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-37"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-48.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-48.gif"/></noscript></span></span> are 5/h, 1/h and 0.2/h respectively, where <em>h</em> is 5<sup>th</sup> percentile of <span class="inline-formula" id="inline-formula-38"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-49.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-49.gif"/></noscript></span></span> leading to local, intermediate and global kernels, respectively (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">González-Camacho <em>et al.</em> 2012</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Tusell <em>et al.</em> 2014</a>).</p></div><div id="sec-8" class="subsection"><h5>RKHS-Ka-Kd:</h5><p id="p-15">In addition to the information contained in the <strong>X</strong> matrix described above, RKHS-Ka-Kd also includes <strong>W</strong>, that is the SNP incidence matrix for dominance effects in additive-dominance Bayes A. The g in this case for each SNP matrix (X and W) used three SNP functions:<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-50.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-50.gif"/></noscript></span></span>The <span class="inline-formula" id="inline-formula-39"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-51.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-51.gif"/></noscript></span></span> consists of the same SNP functions used in RKHS-Ka, and for <span class="inline-formula" id="inline-formula-40"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-52.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-52.gif"/></noscript></span></span> we assumed:<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-53.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-53.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-54.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-54.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-55.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-55.gif"/></noscript></span></span><span class="inline-formula" id="inline-formula-41"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-56.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-56.gif"/></noscript></span></span> is the square of the Euclidean distance matrix using the dominance SNP matrix <strong>W</strong> that takes values 1 for heterozygote and 0 for both homozygotes, as described in the additive-dominance-BayesA model. The same bandwidth values used in RKHS-KA were used for <span class="inline-formula" id="inline-formula-42"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-57.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-57.gif"/></noscript></span></span> in RKHS-Ka-Kd, and for <span class="inline-formula" id="inline-formula-43"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-58.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-58.gif"/></noscript></span></span>, <span class="inline-formula" id="inline-formula-44"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-59.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-59.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-45"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-60.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-60.gif"/></noscript></span></span>, the bandwidths values (<span class="inline-formula" id="inline-formula-46"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-61.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-61.gif"/></noscript></span></span>) were 5/h<sub>d</sub>, 1/h<sub>d</sub> and 0.2/h<sub>d</sub> respectively, where h<sub>d</sub> is 5<sup>th</sup> percentile of <span class="inline-formula" id="inline-formula-47"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-62.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-62.gif"/></noscript></span></span>, similar to <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Morota <em>et al.</em> (2014)</a>. In both RKHS models (Ka and Ka-Kd) the whole genotypic value was predicted but cannot be separated into the components: breeding value, dominance deviation and epistasis. The summary of models compared are available in Table S2.</p></div></div></div><div id="sec-9" class="subsection"><h3>Models validation</h3><p id="p-16">The prediction accuracies were calculated using 10-fold cross-validation (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Resende <em>et al.</em> 2012b</a>; <a id="xref-ref-2-6" class="xref-bibr" href="#ref-2">de Almeida Filho <em>et al.</em> 2016</a>). In order to infer the impact of the training population (TP) on the prediction accuracy, the TP was defined following three different schemes: a) prediction across families –individuals from a group of families were used to fit the model and the genetic potential was predicted in individuals from different families; b) prediction within families – the genetic merit of was predicted in individuals belonging to the same families included in the TP; c) random sample of individuals for TP. These validation approaches were also applied to each of the ten replicates of the simulated data. For data collected from the CCLONES population, the 10-fold based on random sample process was applied 10 times with independent groups of individuals in each fold. Prediction of accuracies and regression coefficients of parametric values on validation data were estimated for each of the 10 folds. A graphical representation of the 10-folds cross-validation schemes used in this study is available in Figure S1.</p></div><div id="sec-10" class="subsection"><h3>Breeding values and dominance deviation</h3><p id="p-17">The expected breeding value (EBV) and the expected dominance deviation (EDD) were estimated as described below:<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-63.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-63.gif"/></noscript></span></span>and<span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-64.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-64.gif"/></noscript></span></span>Where <em>p<sub>i</sub></em> is allele frequency of allele A of SNP <em>i</em>, <em>q<sub>i</sub> = 1-p<sub>i</sub></em>, <span class="inline-formula" id="inline-formula-48"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-65.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-65.gif"/></noscript></span></span> is the average effect of substitution, <span class="inline-formula" id="inline-formula-49"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-66.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-66.gif"/></noscript></span></span>, and I is an indicator function of SNPs; <span class="inline-formula" id="inline-formula-50"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-67.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-67.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-51"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-68.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-68.gif"/></noscript></span></span> are terms from additive and dominant polygenic effects, respectively. The whole genotypic value is the sum of <span class="inline-formula" id="inline-formula-52"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-69.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-69.gif"/></noscript></span></span> and<span class="inline-formula" id="inline-formula-53"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-70.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-70.gif"/></noscript></span></span>. In RKHS-based models, the whole genotypic value is predicted, and therefore the partitions between <span class="inline-formula" id="inline-formula-54"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-71.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-71.gif"/></noscript></span></span> and<span class="inline-formula" id="inline-formula-55"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-72.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-72.gif"/></noscript></span></span> cannot be estimated.</p></div><div id="sec-11" class="subsection"><h3>Variance components</h3><p id="p-18">The variance components from WGR are extensions of estimators previously reported (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Zeng <em>et al.</em> 2013</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Ertl <em>et al.</em> 2014</a>) that assume absence of epistasis, linkage equilibrium among markers and Hardy-Weinberg equilibrium (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Gianola <em>et al.</em> 2009</a>). The general estimator of additive variance (<em>V<sub>A</sub></em>) and the variance due to dominance deviation (<em>V<sub>D</sub></em>) are:<span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-73.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-73.gif"/></noscript></span></span>and<span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-74.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-74.gif"/></noscript></span></span>The components <span class="inline-formula" id="inline-formula-56"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-75.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-75.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-57"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-76.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-76.gif"/></noscript></span></span> in each model above are associated with the polygenic effect. The remaining equation of <span class="inline-formula" id="inline-formula-58"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-77.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-77.gif"/></noscript></span></span> and <span class="inline-formula" id="inline-formula-59"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-78.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2739/embed/mml-math-78.gif"/></noscript></span></span> are due to marker effects. The whole genotypic variance is the sum of additive and dominance variance. The <em>h<sup>2</sup></em>, <em>d<sup>2</sup></em> and <em>H<sup>2</sup></em> are the proportion of additive, dominance and genotypic variance in the phenotypic variance. In RKHS models the genetic variance estimated by markers is the whole genotypic variance. All models were fitted with the R package BLGR (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">de los Campos and Perez 2014</a>), using 100,000 iterations, burn-in of 20,000, thinning of 3 and default hyperparameters previously described (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Pérez and de los Campos 2014</a>).</p></div><div id="sec-12" class="subsection"><h3>Data availability</h3><p id="p-19">All phenotypic and genotypic data utilized in this study have been previously published as a standard data set for development of genomic prediction methods (<a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Resende <em>et al.</em> 2012b</a>). Simulated data available from the Dryad Digital Repository: <a href="http://dx.doi.org/10.5061/dryad.3126v">http://dx.doi.org/10.5061/dryad.3126v</a>. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8379059">https://doi.org/10.25387/g3.8379059</a>.</p></div></div><div class="section results" id="sec-13"><h2 class="">Results</h2><div id="sec-14" class="subsection"><h3>The prediction within families was more accurate than prediction across families</h3><p id="p-20">To infer the impact of population structure and the choice of training population (TP) in the estimates of prediction accuracy, we carried the cross-validation across families, within families and at random. For all models and traits, the prediction accuracies were lowest when the predicted individuals came from families not included in the TP (across family cross-validation). The reductions in accuracy ranged from ∼15–59% in both real and simulated data (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>-<a id="xref-fig-2-1" class="xref-fig" href="#F2">2</a>). Conversely, the within-family cross validation resulted in highest accuracy for the majority of cases. In CCLONES, the superiority of the within-family TP ranged from 1.5–2.4%, 2.7–6.8% and 3.6–5.6% for HT, RFbin and RFgall, respectively (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>). In simulated traits, the within-family cross validation was slightly superior (0.15–3.3%) than the random sample TP in ∼85% and ∼80% of cases for prediction of breeding values and whole genotypic values respectively (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). There model comparison, however, was very similar across TP schemes, and no interaction between models and TP was observed (data not shown). Hence, the following model comparisons and general conclusions, were performed using random-sample cross-validation.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2739/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Accuracy distribution of all genomic prediction models fitted for tree height (HT), diameter at breast height (DBH) and two measures of fusiform rust infection: presence or absence of rust (RFbin) and gall volume (RFgall). These results were achieved from three different 10-fold cross validation orientations: a) Across families: Each fold is a group of distinct families; b) Within families: The folds were grouped inside families; and c) Random sample: Each fold is a group of distinct individuals random sampled ignoring family information." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1899125587" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Accuracy distribution of all genomic prediction models fitted for tree height (HT), diameter at breast height (DBH) and two measures of fusiform rust infection: presence or absence of rust (RFbin) and gall volume (RFgall). These results were achieved from three different 10-fold cross validation orientations: a) Across families: Each fold is a group of distinct families; b) Within families: The folds were grouped inside families; and c) Random sample: Each fold is a group of distinct individuals random sampled ignoring family information.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2739/F1.medium.gif" width="440" height="121"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/8/2739/F1.medium.gif" width="440" height="121"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2739/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2739/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493961" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-21" class="first-child">Accuracy distribution of all genomic prediction models fitted for tree height (HT), diameter at breast height (DBH) and two measures of fusiform rust infection: presence or absence of rust (RFbin) and gall volume (RFgall). These results were achieved from three different 10-fold cross validation orientations: a) Across families: Each fold is a group of distinct families; b) Within families: The folds were grouped inside families; and c) Random sample: Each fold is a group of distinct individuals random sampled ignoring family information.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2739/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Accuracy average of prediction of breeding values and genotypic values for all genomic prediction models fitted in six simulated traits: Oligogenic and Polygenic with three degrees of dominance (d2 = 0; d2 = 0.1 and d2 = 0.2). These results were achieved from three different 10-fold cross validation orientations: a) Across families: Each fold is a group of distinct families; b) Within families: The folds were grouped inside families; and c) Random sample: Each fold is a group of distinct individuals random sampled ignoring family information." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1899125587" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Accuracy average of prediction of breeding values and genotypic values for all genomic prediction models fitted in six simulated traits: Oligogenic and Polygenic with three degrees of dominance (d&lt;sup&gt;2&lt;/sup&gt; = 0; d&lt;sup&gt;2&lt;/sup&gt; = 0.1 and d&lt;sup&gt;2&lt;/sup&gt; = 0.2). These results were achieved from three different 10-fold cross validation orientations: a) Across families: Each fold is a group of distinct families; b) Within families: The folds were grouped inside families; and c) Random sample: Each fold is a group of distinct individuals random sampled ignoring family information.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2739/F2.medium.gif" width="440" height="237"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/8/2739/F2.medium.gif" width="440" height="237"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2739/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2739/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493803" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-22" class="first-child">Accuracy average of prediction of breeding values and genotypic values for all genomic prediction models fitted in six simulated traits: Oligogenic and Polygenic with three degrees of dominance (d<sup>2</sup> = 0; d<sup>2</sup> = 0.1 and d<sup>2</sup> = 0.2). These results were achieved from three different 10-fold cross validation orientations: a) Across families: Each fold is a group of distinct families; b) Within families: The folds were grouped inside families; and c) Random sample: Each fold is a group of distinct individuals random sampled ignoring family information.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-15" class="subsection"><h3>Pedigree information in model prediction</h3><p id="p-23">The use of pedigree information has been shown previously to improve the accuracy of genotypic predictions (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Crossa <em>et al.</em> 2014</a>). Here, we evaluated if this gain in accuracy is also observed in our populations, based on correlation using parametric genetic values from simulated data, and phenotypic values from real data. Initially we compared the prediction accuracy of models based only on pedigree information against models with markers-only data (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Overall, models based only on pedigree data had modestly lower accuracy than those using marker information for traits measured in the pine CCLONES population. The improvement in accuracy for breeding and genotypic values prediction was much more apparent when markers were used in the simulated population, compared to when only pedigree data were utilized. This improvement was particularly pronounced for simulated oligogenic traits (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). Next, we extended the comparison of accuracies to include models that combined both pedigree and marker information. We observed that in only few instances the expanded models had higher accuracies, and that in general the accuracies were comparable to those with marker only in the real and simulated data (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>-<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">2</a>). One of such cases was the accuracy in predicting dominance deviation in the simulated traits (<a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a>).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493995/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493995/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Average of accuracies for prediction of phenotypic values for all models based in pedigree-only, in markers-only and models combining pedigree and markers information</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493966/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493966/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Average of accuracies of breeding values, dominance deviation, genotypic values and phenotypic values prediction for all models based in pedigree-only, in markers-only and models combining pedigree and markers information</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Genotypic predictive model accuracy depends on trait architecture</h3><p id="p-26">The two genomic prediction methods tested in this study differ with respect to the assumptions regarding the genetic architecture of the trait being predicted. BayesA is a linear regression model that assumes that each marker has different variance. Consequently, some markers can explain the effect of major loci, such as in oligogenic traits (<em>e.g.</em>, rust resistance). On the other hand, RKHS is a semi-parametric model that assumes that all markers with the same MAF contribute equally to the relationship among individuals. Thus, this assumption has closer resemblance with polygenic trait regulation. For phenotypic predictions, RKHS had higher accuracy for DBH and HT (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>) when compared to BayesA. Similar results were observed in the polygenic traits with presence of dominance effects (<em>d<sup>2</sup></em> = 0.1 and <em>d<sup>2</sup></em> = 0.2). In both cases, the RKHS method resulted in higher accuracies than BayesA models for genotypic and phenotypic prediction (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, Table S3). In contrast, BayesA resulted in higher correlation for RFbin (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>) and all oligogenic simulated traits (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>), independent of the inclusion of pedigree in the genomic prediction and independent of the simulated dominance effect. The superiority of the additive-dominance models was trait dependent and in some cases opposite accuracy patterns were observed, as it is the case for HT and RFbin (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2739/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Average of phenotypic prediction accuracies and standard error for four markers-only models: additive- and additive-dominance-BayesA, RKHS-Ka and RKHS Ka-Kd for diameter at breast height (DBH), height (HT) and Rust resistance evaluated as gall volume (RFgall) and presence or absence (RFbin) in loblolly pine. The standard errors (s.e.=s.d.(x)/sqrt(10)) were calculated for each ten-fold procedure. The error bars are the averages of s.e. across the ten independent cross validations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1899125587" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Average of phenotypic prediction accuracies and standard error for four markers-only models: additive- and additive-dominance-BayesA, RKHS-Ka and RKHS Ka-Kd for diameter at breast height (DBH), height (HT) and Rust resistance evaluated as gall volume (RFgall) and presence or absence (RFbin) in loblolly pine. The standard errors (s.e.=s.d.(x)/sqrt(10)) were calculated for each ten-fold procedure. The error bars are the averages of s.e. across the ten independent cross validations.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2739/F3.medium.gif" width="440" height="133"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/8/2739/F3.medium.gif" width="440" height="133"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2739/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2739/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493921" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-27" class="first-child">Average of phenotypic prediction accuracies and standard error for four markers-only models: additive- and additive-dominance-BayesA, RKHS-Ka and RKHS Ka-Kd for diameter at breast height (DBH), height (HT) and Rust resistance evaluated as gall volume (RFgall) and presence or absence (RFbin) in loblolly pine. The standard errors (s.e.=s.d.(x)/sqrt(10)) were calculated for each ten-fold procedure. The error bars are the averages of s.e. across the ten independent cross validations.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2739/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Accuracy average of prediction of breeding values and genotypic values for four marker-only models: additive-, additive-dominance-BayesA, RKHS-Ka and RKHS Ka-Kd for six simulated traits: Oligogenic and Poligenic with three degrees of dominance (d2 = 0; d2 = 0.1 and d2 = 0.2). Error bars are standard error (s.e.) considering the 100 independent samples used to calculate the mean (s.e.=s.d.(x)/sqrt(100))." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1899125587" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Accuracy average of prediction of breeding values and genotypic values for four marker-only models: additive-, additive-dominance-BayesA, RKHS-Ka and RKHS Ka-Kd for six simulated traits: Oligogenic and Poligenic with three degrees of dominance (d&lt;sup&gt;2&lt;/sup&gt; = 0; d&lt;sup&gt;2&lt;/sup&gt; = 0.1 and d&lt;sup&gt;2&lt;/sup&gt; = 0.2). Error bars are standard error (s.e.) considering the 100 independent samples used to calculate the mean (s.e.=s.d.(x)/sqrt(100)).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2739/F4.medium.gif" width="440" height="420"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/8/2739/F4.medium.gif" width="440" height="420"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2739/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2739/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493879" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-28" class="first-child">Accuracy average of prediction of breeding values and genotypic values for four marker-only models: additive-, additive-dominance-BayesA, RKHS-Ka and RKHS Ka-Kd for six simulated traits: Oligogenic and Poligenic with three degrees of dominance (d<sup>2</sup> = 0; d<sup>2</sup> = 0.1 and d<sup>2</sup> = 0.2). Error bars are standard error (s.e.) considering the 100 independent samples used to calculate the mean (s.e.=s.d.(x)/sqrt(100)).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-17" class="subsection"><h3>Genotypic predictive model strength depends on non-additive effects</h3><p id="p-29">The prediction of genotypic values is valuable for genetic improvement programs where breeders can capture additive and non-additive effects by cloning selected individuals. Prediction of dominance effects for each locus is also critical to optimize crossing designs, such as in mate-pair allocation. The inclusion of the Kd kernels matrix in RKHS was equal or worse than the Ka-only kernels for both real traits (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3</a>) and simulated traits (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>). On the other hand, the inclusion of dominance effects in the BayesA model resulted in a better prediction for HT (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>). For simulated traits, the additive-dominance-BayesA showed considerably higher genotypic (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a>) and phenotypic (Table S3) prediction accuracy compared to the additive-BayesA in traits with high dominance.</p></div><div id="sec-18" class="subsection"><h3>BayesA models generated higher accuracies in breeding value prediction</h3><p id="p-30">The breeding value of one individual represents the component of its genotypic value that is directly transmitted to the progeny. Thus, the breeding value is a critical parameter in the selection of individuals to be used extensively in mating with other individuals in the population. With BayesA and pedigree-based models, it was possible to predict directly the breeding values. The RKHS models, however, generate a prediction of the whole genotypic values, which cannot be split in breeding values, dominance deviation and epistasis. In the simulated the traits, we still reported the correlation between predicted genotypic values and parametric breeding values to assess the accuracy in breeding value prediction. The traditional additive-BayesA and additive-dominance-BayesA based only in marker information estimated the higher accuracy for breeding value prediction for all simulated traits (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a>), while pedigree-based models showed the worst accuracies (<a id="xref-table-wrap-2-4" class="xref-table" href="#T2">Table 2</a>).</p></div><div id="sec-19" class="subsection"><h3>Variance components and heritability</h3><p id="p-31">One of the most important tasks for a breeder is to make decisions regarding the breeding strategy. Several genetic parameters are evaluated to determine breeding strategies, including the estimation of variance components and the proportion of the genetic variance relative to the phenotypic variance. Here these parameters were estimated using genetic markers, pedigree information or both, for the real and simulated traits. For simulated traits, the parametric values are known, which allowed us to evaluate the capacity of the methods and models to accurately estimate them.</p><p id="p-32">Considering the parametric values of <em>h<sup>2</sup></em> and <em>d<sup>2</sup></em> in the simulated population, the BayesA models that used only genetic markers resulted in the least biased estimate of the genetic parameters (Table S4). The inclusion of pedigree information in BayesA models increased the estimates of heritabilities, and in most cases these parameters were overestimated. Models based only on pedigree also overestimated these genetics parameters: the additive-only pedigree model over estimated <em>h<sup>2</sup></em>, and the additive-dominance pedigree models estimated high <em>d<sup>2</sup></em> for traits with no dominance.</p><p id="p-33">The RKHS based models predict the whole genotypic values. Consequently, estimation of both <em>h<sup>2</sup></em> and <em>d<sup>2</sup></em> is not possible. Nonetheless, <em>H<sup>2</sup></em> can be obtained. The <em>H<sup>2</sup></em> was calculated for the simulated traits and showed that all RKHS based models, regardless of the pedigree inclusion, overestimated this parameter substantially (on average by ∼106%, Table S4). Overestimation is higher in models that included the Kd kernel (RKHS Ka-Kd). Table S5 reports the results of heritabilities estimated with all models for real traits. Similarly, to the simulated data, the estimation of <em>h<sup>2</sup></em> and <em>d<sup>2</sup></em> increased with the pedigree information, and the RKHS based models resulted in much larger estimates of <em>H<sup>2</sup></em>.</p></div><div id="sec-20" class="subsection"><h3>Prediction bias</h3><p id="p-34">The regression coefficient (slope) of observed values <em>vs.</em> predicted values was used to measure model bias – a slope of one indicates the absence of any bias. The linear regression of simulated data included parametric genotypic values and prediction. For the real data, where the parametric values are unknown, we calculated the slope using phenotype values. The predictions using real data (Table S6) resulted in regression coefficients near one. Most predictions using simulated data resulted in a slope similar to one, with the exception of dominance deviation predictions. In this case, the slopes showed a biased dominance prediction. More specifically, the slope for dominance deviation prediction was less than one in the marker-only models with additive-dominance effects, indicating an underestimation of dominance effects. In contrast, when the pedigree information was considered, the slopes were higher than one, suggesting that the pedigree information contributed to an overestimation of dominance effects (Table S7).</p></div></div><div class="section discussion" id="sec-21"><h2 class="">Discussion</h2><p id="p-35">We tested the performance of genomic- and pedigree-based models, with and without non-additive effects, for the prediction of genetic values in complex traits. We used real data from a standard pine tree breeding program in its third generation. Pine traits used for model testing included plant height (HT), diameter at breast height (DBH), and the rust resistance measures RFbin and RFgall, whose narrow sense heritability, prior to least square mean adjustment, were previous reported as 0.31, 0.31, 0.21 and 0.12, respectively (<a id="xref-ref-44-5" class="xref-bibr" href="#ref-44">Resende <em>et al.</em> 2012b</a>). These traits have different genetic architecture; DBH and HT are polygenic whereas rust resistance is thought to be an oligogenic trait (<a id="xref-ref-44-6" class="xref-bibr" href="#ref-44">Resende <em>et al.</em> 2012b</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Quesada <em>et al.</em> 2014</a>). Moreover, HT has significant non-additive effects (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Muñoz <em>et al.</em> 2014</a>). To expand and validate our conclusions, we simulated six traits with distinct genetic architectures (polygenic and oligogenic) and three dominance levels.</p><div id="sec-22" class="subsection"><h3>Pedigree information in genomic predictions</h3><p id="p-36">Pedigree and marker information were used separately and in combination, to predict genetic values. In the prediction of breeding and whole genotypic values, the genomic selection model and the combined model (genetic markers + pedigree) were substantially better than the models accounting only for pedigree information. However, the combined model did not improve the prediction accuracies. Previous studies suggest that the use of pedigree data only results in an improvement in prediction when a low marker density is used, as seen in simulated studies (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Calus and Veerkamp 2007</a>), as well as in wheat (<a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Crossa <em>et al.</em> 2010</a>) and dairy cattle (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Vazquez <em>et al.</em> 2010</a>). In contrast, pedigree information did not improve model predictions when a higher density of genetic markers were used (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Calus and Veerkamp 2007</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">de los Campos <em>et al.</em> 2009b</a>; <a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Vazquez <em>et al.</em> 2010</a>). An exception was reported in maize, when prediction models generated with a high-density SNP panel from genotyping-by-sequencing (GBS) was improved with the addition of pedigree data (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Crossa <em>et al.</em> 2013</a>). A possible explanation for this outcome rests in the fact that the GBS data contains a high frequency of missing data (<a id="xref-ref-8-3" class="xref-bibr" href="#ref-8">Crossa <em>et al.</em> 2013</a>), resulting in incomplete genomic information.</p></div><div id="sec-23" class="subsection"><h3>Semi-parametric kernel choice</h3><p id="p-37">Different kernels can be used to improve the predictions of complex traits in semi-parametric RKHS models. In the current study, the genomic predictions of RKHS Ka-Kd models and the simpler RKHS Ka models yielded similar results. These findings are in agreement with those previously reported by <a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Morota <em>et al.</em> (2014)</a> for dairy cattle. These authors did not find increased accuracies with the inclusion of extra kernels in the RKHS Ka. Other kernel comparisons in RKHS also showed that RKHS Ka is a robust choice for the prediction of additive and non-additive effects (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Morota <em>et al.</em> 2013</a>; <a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Tusell <em>et al.</em> 2014</a>).</p></div><div id="sec-24" class="subsection"><h3>BayesA resulted in the highest breeding values accuracy</h3><p id="p-38">The BayesA models based only on genetic marker information resulted in the highest accuracies of breeding values prediction in all the simulated scenarios, regardless of the dominance effects. The additive-BayesA and additive-dominance-BayesA models also resulted in similar accuracies for breeding values in additive-dominance traits, despite the fact that the breeding value is also function of dominance effects (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Falconer and Mackay 1996</a>). While this result was unexpected, one possible reason for this is the overall low accuracy in the prediction of dominance. The results were also similar to another simulation study that showed higher genetic gain with the additive model compared to an additive-dominance model, even in presence of higher dominance effects (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Denis and Bouvet 2012</a>). Furthermore, <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Nishio and Satoh (2014)</a> also showed that dominance inclusion did not provide higher accuracies in breeding value prediction, even in traits with dominance effects.</p></div><div id="sec-25" class="subsection"><h3>Prediction of whole-genotypic values of traits with distinct genetic architecture</h3><p id="p-39">Models with built-in assumptions that some loci have major effects, such as BayesA, usually provide better genomic predictions for simulated oligogenic traits (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">de los Campos <em>et al.</em> 2013</a>), and also for real traits controlled by few genes, such as fat percentage in milk (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Habier <em>et al.</em> 2011</a>). For genotypic prediction in this study, BayesA generated better predictions than RKHS for RFbin and simulated oligogenic traits. This finding is in agreement with other studies that suggest that rust resistance is an oligogenic trait (<a id="xref-ref-44-7" class="xref-bibr" href="#ref-44">Resende <em>et al.</em> 2012b</a>; <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Quesada <em>et al.</em> 2014</a>). In the case of simulated additive-dominance oligogenic traits, the additive-dominant-BayesA model resulted in the best predictions of genotypic values.</p><p id="p-40">Our analyses of polygenic simulated traits showed that, for additive-dominant polygenic traits, the RKHS models were better than the additive-BayesA for whole genotypic predictions. These findings agree with those of other authors who argue that RKHS addresses non-additive variation in a non-explicit manner (<a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">Gianola <em>et al.</em> 2006</a>; <a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Gianola and van Kaam 2008</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Morota and Gianola 2014</a>). In addition, RKHS was modestly more accurate when compared to the additive-dominant-BayesA, confirming that RKHS can be explored for predictions in polygenic traits.</p><p id="p-41">In CCLONES population, the RKHS models was slightly more accurate than additive- and additive-dominant-BayesA for HT and DBH. In addition, the additive-dominant-BayesA model was slightly more accurate than the additive-BayesA for HT. This result suggests the presence of non-additive effects in <em>P. taeda</em> tree height, which was also previously suggested by <a id="xref-ref-39-3" class="xref-bibr" href="#ref-39">Muñoz <em>et al.</em> (2014)</a>. Similar results were also observed in a genomic selection study by <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bouvet <em>et al.</em> (2016)</a> which reported higher prediction accuracy of HT in Eucalyptus using models with non-additive effects. In contrast, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">El-dien <em>et al.</em> (2016)</a>, did not achieved prediction accuracy increase with inclusion of non-additive effects in HT in an open-pollinated white Spruce population. Altogether, and as expected, we conclude that the increase of prediction accuracy with inclusion of dominance depend of the population and trait.</p><p id="p-42">Similar to <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Xavier <em>et al.</em> (2016)</a>, we also fit a model that combined BayesA and RKHS in a single model. This combined model resulted in lower accuracies than additive BayesA for RFbin and simulated oligogenic traits with d<sup>2</sup> = 0 and d<sup>2</sup> = 0.1. In addition, the BayesA+RKHS combined model provides lower accuracy than RKHS for HT, DBH and polygenic additive-dominance traits (Table S8). However, the BayesA+RKHS model was slightly more accurate than BayesA for genotypic prediction in oligogenic traits with high dominance (d<sup>2</sup> = 0.2) and more accurate than RKHS in additive polygenic traits for breeding and genotypic values prediction (Tables S9). In soybean, <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Xavier <em>et al.</em> (2016)</a> reported that BayesA+RKHS and BayesB+RKHS were the most accurate genomic prediction models. These authors suggested that this model could be capturing the breeding values through BayesA and the remaining genetic components with RKHS. In our dataset, this advantage of the combined model was not clear, but further evaluation of this model could be made.</p></div><div id="sec-26" class="subsection"><h3>Heritabilities estimation</h3><p id="p-43">When using RKHS based models and pedigree information, the parameters <em>h<sup>2</sup></em>, <em>d<sup>2</sup></em> and <em>H<sup>2</sup></em> were often overestimated. Similarly, others genomic prediction studies in forest species also suggested that pedigree based models generate large and unrealistic estimates of h<sup>2</sup>, when compared with GBLUP (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Bouvet <em>et al.</em> 2016</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">El-dien <em>et al.</em> 2016</a>).</p><p id="p-44">In quantitative genetics theory, the additive and non-additive effects are typically assumed to be orthogonal and can be divided as independent components from the whole genotypic variance (<a id="xref-ref-14-3" class="xref-bibr" href="#ref-14">Falconer and Mackay 1996</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Hallauer <em>et al.</em> 2010</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">de los Campos <em>et al.</em> 2015</a>). However, in practical situations (absence of Hardy-Weinberg equilibrium, non-random matting), these presuppositions do not hold, and additive and dominance effects are related to each other and not orthogonal. The inability to partition the components can lead to bias estimates of the variance components, affecting the estimates of selection gain as well as the interpretation of the trait architecture. <a id="xref-ref-39-4" class="xref-bibr" href="#ref-39">Muñoz <em>et al.</em> (2014)</a> indicated that this bias was stronger in pedigree models and recommended the use of genetic markers to partition the additive and non-additive components.</p><p id="p-45">The marker specific variances and therefore the variance component estimates can be influenced by the effect of the priors in Bayesian models (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Gianola <em>et al.</em> 2009</a>). Nonetheless, in our study, the BayesA models using only markers, generated more reasonable variance components estimates, compared to the parametric values. The inclusion of dominance effects in BayesA resulted in less biased heritabilities estimation for additive-dominance traits, and the traditional additive-BayesA were less biased for <em>h<sup>2</sup></em> estimation in additive traits. This suggests that additive-dominant models should be used for estimation of heritability in cases where the inclusion of dominance effects increase the prediction accuracy. Based on this observation, the estimates for tree height of <em>h<sup>2</sup></em> and <em>d<sup>2</sup></em> using the additive-dominant model equal 0.37 and 0.17 respectively. For DBH, RFbin and RFgall, the additive model is suggested and theirs respective <em>h<sup>2</sup></em> are estimated in 0.52, 0.39 and 0.29 (Table S13). The <em>h<sup>2</sup></em> estimated for HT in this report were similar to the value found in <a id="xref-ref-44-8" class="xref-bibr" href="#ref-44">Resende <em>et al.</em> (2012b)</a>, but higher than reported in <a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Resende <em>et al.</em> (2012a)</a>. The estimates of <em>d<sup>2</sup></em> for HT were similar to previous report (<a id="xref-ref-39-5" class="xref-bibr" href="#ref-39">Muñoz <em>et al.</em> 2014</a>), which used the same population but different methods. For DBH, and rust resistance, the <em>h<sup>2</sup></em> obtained in this study, were higher than other authors (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Resende <em>et al.</em> 2012a</a>, <a id="xref-ref-44-9" class="xref-bibr" href="#ref-44">2012b</a>). <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Calus and Veerkamp (2007)</a> reported that the inclusion of polygenic effects resulted in better estimates of variance components when compared with models that included only markers - this result that was not replicated in our study.</p></div><div id="sec-27" class="subsection"><h3>Prediction accuracy is affected by the choice of training population</h3><p id="p-46">This study contrasted three schemes of training population (TP). The results showed that across families cross validation resulted the lowest accuracy, whereas the setup to within families cross validation produced the best results in most part of cases, being slightly superior than random sample individuals for TP. These results were expected, and further emphasize the importance of genetic relationship among individuals used to fit the model and target individuals for genomic prediction. Genomic Selection models have been shown to capture not only linkage disequilibrium between the markers and causal alleles, but also genetic relationship (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Habier <em>et al.</em> 2007</a>). The scheme based on random sampling has been commonly used, because it represents a balance between within and across families. These results agree with other studies in forest breeding that showed higher accuracies in the prediction within families than across families (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Beaulieu <em>et al.</em> 2014b</a>, <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">2014a</a>). In addition, <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Albrecht <em>et al.</em> (2014)</a> reported higher accuracies in the prediction inside genetic groups than across genetics groups. Hence, one of the most important factors when outlining a genomic selection breeding program is the definition of the target population. The TP needs to resemble, as close as possible, the level of relationship of the target population where GS will be applied.</p></div><div id="sec-28" class="subsection"><h3>CONCLUSION</h3><p id="p-47">In summary, we conclude that, in the CCLONES pine population and in our simulation studies, genomic selection is effective compared to phenotypic pedigree selection. The inclusion of pedigree information did not improve the prediction accuracies, suggesting that pedigree computation is not required in genomic prediction programs with reasonable SNP panels. This study also supports the findings that the individuals used to compose the training population should be genetically related to the individuals in the target breeding population. Finally, the BayesA models overcame the RKHS-based models for breeding and genotypic prediction for oligogenic traits, while in polygenic traits, BayesA was suitable for breeding value prediction and RKHS for whole-genotypic prediction.</p></div></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-48">This work was supported by the US Department of Agriculture National Institute of Food and Agriculture Plant Feedstock for Bioenergy Program (Award No. 2013-67009-21200 to MK and PM) and Plant Health and Production and Plant Products Program (Award No. 2013-67013-21159 to MK and PM). The authors thank the Coordenação de Aperfeiçoamento de Pessoal de Nível Superior (CAPES) and the Conselho Nacional de Desenvolvimento Científico e Tecnológico (CNPq) for the graduate fellowship.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-49">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8379059">https://doi.org/10.25387/g3.8379059</a>.</p></li><li class="fn" id="fn-4"><p id="p-50"><em>Communicating editor: J.-L. Jannink</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-03-31"><span class="received-label">Received </span>March 31, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-06-20"><span class="accepted-label">Accepted </span>June 20, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 de Almeida Filho <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.1" data-doi="10.1007/s00122-014-2305-z"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Albrecht</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.-J.</span> <span class="cit-name-surname">Auinger</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Wimmer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">O. J.</span> <span class="cit-name-surname">Ogutu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Knaak</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-based prediction of maize hybrid performance across genetic groups, testers, locations, and years</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">1375</span>–<span class="cit-lpage">1386</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-014-2305-z</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D127%26rft.spage%253D1375%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-014-2305-z%26rft_id%253Dinfo%253Apmid%252F24723140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-014-2305-z&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24723140&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.2" data-doi="10.1038/hdy.2016.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Almeida Filho</span>,  <span class="cit-name-given-names">J. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. F. R.</span> <span class="cit-name-surname">Guimarães</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. F.</span> <span class="cit-name-surname">Silva</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D. V.</span> <span class="cit-name-surname">De Resende</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Muñoz</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">The contribution of dominance to phenotype prediction in a pine breeding and simulated population</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">33</span>–<span class="cit-lpage">41</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2016.23</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D117%26rft.spage%253D33%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2016.23%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2016.23&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.3" data-doi="10.1139/x06-203"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baltunis</span>,  <span class="cit-name-given-names">B. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Huber</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. L.</span> <span class="cit-name-surname">White</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Goldfarb</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. E.</span> <span class="cit-name-surname">Stelzer</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genetic analysis of early field growth of loblolly pine clones and seedlings from the same full-sib families</span>. <abbr class="cit-jnl-abbrev">Can. J. For. Res.</abbr> <span class="cit-vol">37</span>: <span class="cit-fpage">195</span>–<span class="cit-lpage">205</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1139/x06-203</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BFor.%2BRes.%26rft.volume%253D37%26rft.spage%253D195%26rft_id%253Dinfo%253Adoi%252F10.1139%252Fx06-203%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/x06-203&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.4" data-doi="10.1038/hdy.2014.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beaulieu</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Doerksen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Clément</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Mackay</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bousquet</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">a</span> <span class="cit-article-title">Accuracy of genomic selection models in a large population of open-pollinated families in white spruce</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">343</span>–<span class="cit-lpage">352</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2014.36</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D113%26rft.spage%253D343%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2014.36%26rft_id%253Dinfo%253Apmid%252F24781808%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2014.36&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24781808&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000342564000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.5" data-doi="10.1186/1471-2164-15-1048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Beaulieu</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. K.</span> <span class="cit-name-surname">Doerksen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">MacKay</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Rainville</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bousquet</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">b</span> <span class="cit-article-title">Genomic selection accuracies within and between environments and small breeding groups in white spruce</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1048</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-1048</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D1048%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-1048%26rft_id%253Dinfo%253Apmid%252F25442968%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-1048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25442968&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.6" data-doi="10.1038/hdy.2015.78"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bouvet</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Makouanzi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Cros</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Vigneron</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Modeling additive and non-additive effects in a hybrid population using genome-wide genotyping : prediction accuracy implications</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">146</span>–<span class="cit-lpage">157</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2015.78</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D116%26rft.spage%253D146%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2015.78%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2015.78&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.7" data-doi="10.1111/j.1439-0388.2007.00691.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Calus</span>,  <span class="cit-name-given-names">M. P. L.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. F.</span> <span class="cit-name-surname">Veerkamp</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Accuracy of breeding values when using and ignoring the polygenic effect in genomic breeding value estimation with a marker density of one SNP per cM</span>. <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">362</span>–<span class="cit-lpage">368</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1439-0388.2007.00691.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BAnim%2BBreed%2BGenet%26rft.aulast%253DCalus%26rft.auinit1%253DM.%2BP.%26rft.volume%253D124%26rft.issue%253D6%26rft.spage%253D362%26rft.epage%253D368%26rft.atitle%253DAccuracy%2Bof%2Bbreeding%2Bvalues%2Bwhen%2Busing%2Band%2Bignoring%2Bthe%2Bpolygenic%2Beffect%2Bin%2Bgenomic%2Bbreeding%2Bvalue%2Bestimation%2Bwith%2Ba%2Bmarker%2Bdensity%2Bof%2Bone%2BSNP%2Bper%2BcM.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1439-0388.2007.00691.x%26rft_id%253Dinfo%253Apmid%252F18076473%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1439-0388.2007.00691.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18076473&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000251432800006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Beyene</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Kassa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pérez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Hickey</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic prediction in maize breeding populations with genotyping-by-sequencing. G3 Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">1903</span>–<span class="cit-lpage">1926</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%26rft.volume%253D3%26rft.spage%253D1903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.9" data-doi="10.1534/genetics.110.118521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Campos Gde</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pérez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Burgueño</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Prediction of genetic values of quantitative traits in plant breeding using pedigree and molecular markers</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">724</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.110.118521</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.118521%26rft_id%253Dinfo%253Apmid%252F20813882%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg2LzIvNzEzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI3MzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.10" data-doi="10.1038/hdy.2013.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crossa</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pérez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hickey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Burgueño</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Ornella</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic prediction in CIMMYT maize and wheat breeding programs</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">112</span>: <span class="cit-fpage">48</span>–<span class="cit-lpage">60</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2013.16</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D112%26rft.spage%253D48%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2013.16%26rft_id%253Dinfo%253Apmid%252F23572121%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2013.16&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23572121&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.11" data-doi="10.1007/s11295-012-0528-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Denis</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Bouvet</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Efficiency of genomic selection with models including dominance effect in the context of Eucalyptus breeding</span>. <abbr class="cit-jnl-abbrev">Tree Genet. Genomes</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">37</span>–<span class="cit-lpage">51</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s11295-012-0528-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTree%2BGenet.%2BGenomes%26rft.volume%253D9%26rft.spage%253D37%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs11295-012-0528-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s11295-012-0528-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">El-dien</span>,  <span class="cit-name-given-names">O. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Ratcliffe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Klápstě</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Porth</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Chen</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Implementation of the Realized Genomic Relationship Matrix to Open-Pollinated White Spruce Family Testing for Disentangling Additive from Nonadditive Genetic Effects. G3 Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">743</span>–<span class="cit-lpage">753</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%26rft.volume%253D6%26rft.spage%253D743%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.13" data-doi="10.1186/1297-9686-46-40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ertl</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Legarra</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z. G.</span> <span class="cit-name-surname">Vitezica</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Varona</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Edel</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic analysis of dominance effects on milk production and conformation traits in Fleckvieh cattle</span>. <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">40</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-46-40</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D46%26rft.spage%253D40%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-46-40%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-46-40&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-book" id="cit-9.8.2739.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Falconer</span>,  <span class="cit-name-given-names">D. S.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. F. C.</span> <span class="cit-name-surname">Mackay</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-source">Introduction to Quantitative Genetics</span>, <span class="cit-publ-name">Longman</span>. <span class="cit-publ-loc">Harlow, UK</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.15" data-doi="10.1534/genetics.113.151753"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>,  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Priors in whole-genome regression: The Bayesian alphabet returns</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">573</span>–<span class="cit-lpage">596</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.113.151753</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.151753%26rft_id%253Dinfo%253Apmid%252F23636739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzMvNTczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI3MzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.16" data-doi="10.1534/genetics.105.049510"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Fernando</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Stella</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Genomic-assisted prediction of genetic value with semiparametric procedures</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">173</span>: <span class="cit-fpage">1761</span>–<span class="cit-lpage">1776</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.105.049510</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.105.049510%26rft_id%253Dinfo%253Apmid%252F16648593%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3My8zLzE3NjEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzgvMjczOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.17" data-doi="10.1534/genetics.107.084285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>,  <span class="cit-name-given-names">D.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B. C. H. M.</span> <span class="cit-name-surname">van Kaam</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Reproducing Kernel Hilbert Spaces Regression Methods for Genomic Assisted Prediction of Quantitative Traits</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">2289</span>–<span class="cit-lpage">2303</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.107.084285</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DGianola%26rft.auinit1%253DD.%26rft.volume%253D178%26rft.issue%253D4%26rft.spage%253D2289%26rft.epage%253D2303%26rft.atitle%253DReproducing%2BKernel%2BHilbert%2BSpaces%2BRegression%2BMethods%2Bfor%2BGenomic%2BAssisted%2BPrediction%2Bof%2BQuantitative%2BTraits%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.084285%26rft_id%253Dinfo%253Apmid%252F18430950%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC80LzIyODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzgvMjczOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.18" data-doi="10.1534/genetics.109.103952"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gianola</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de los Campos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. G.</span> <span class="cit-name-surname">Hill</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Manfredi</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Fernando</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Additive genetic variability and the Bayesian alphabet</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">363</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.103952</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.103952%26rft_id%253Dinfo%253Apmid%252F19620397%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgzLzEvMzQ3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI3MzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.19" data-doi="10.1007/s00122-012-1868-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">González-Camacho</span>,  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de los Campos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pérez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. E.</span> <span class="cit-name-surname">Cairns</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-enabled prediction of genetic values using radial basis function neural networks</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">759</span>–<span class="cit-lpage">771</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-012-1868-9</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DGonzalez-Camacho%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D125%26rft.issue%253D4%26rft.spage%253D759%26rft.epage%253D771%26rft.atitle%253DGenome-enabled%2Bprediction%2Bof%2Bgenetic%2Bvalues%2Busing%2Bradial%2Bbasis%2Bfunction%2Bneural%2Bnetworks.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-012-1868-9%26rft_id%253Dinfo%253Apmid%252F22566067%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-012-1868-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22566067&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.20" data-doi="10.1534/genetics.107.081190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Fernando</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Dekkers</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The impact of genetic relationship information on genome-assisted breeding values</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">2389</span>–<span class="cit-lpage">2397</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D177%26rft.issue%253D4%26rft.spage%253D2389%26rft.epage%253D2397%26rft.atitle%253DThe%2BImpact%2Bof%2BGenetic%2BRelationship%2BInformation%2Bon%2BGenome-Assisted%2BBreeding%2BValues%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.081190%26rft_id%253Dinfo%253Apmid%252F18073436%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ny80LzIzODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzgvMjczOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.21" data-doi="10.1186/1471-2105-12-186"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Habier</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Fernando</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Kizilkaya</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Garrick</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Extension of the bayesian alphabet for genomic selection</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">186</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-186</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHabier%26rft.auinit1%253DD.%26rft.volume%253D12%26rft.spage%253D186%26rft.epage%253D186%26rft.atitle%253DExtension%2Bof%2Bthe%2Bbayesian%2Balphabet%2Bfor%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-186%26rft_id%253Dinfo%253Apmid%252F21605355%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-186&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21605355&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-book" id="cit-9.8.2739.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hallauer</span>,  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Carena</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. B.</span> <span class="cit-name-surname">Miranda Filho</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-source">Quantitative Genetics in Maize Breeding</span>, <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-book" id="cit-9.8.2739.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>,  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-source">Applications of Linear Models in Animal Breeding</span>, <span class="cit-publ-name">University of Guelph</span>. <span class="cit-publ-loc">Guelph, Ontario</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hickey</span>,  <span class="cit-name-given-names">J. M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Gorjanc</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Simulated Data for Genomic Selection and Genome-Wide Association Studies Using a Combination of Coalescent and Gene Drop Methods. G3 Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">425</span>–<span class="cit-lpage">427</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%26rft.volume%253D2%26rft.spage%253D425%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.25" data-doi="10.1007/s00122-004-1915-2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kayihan</span>,  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Huber</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. M.</span> <span class="cit-name-surname">Morse</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. L.</span> <span class="cit-name-surname">White</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Davis</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Genetic dissection of fusiform rust and pitch canker disease traits in loblolly pine</span>. <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">948</span>–<span class="cit-lpage">958</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s00122-004-1915-2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DKayihan%26rft.auinit1%253DG.%2BC.%26rft.volume%253D110%26rft.issue%253D5%26rft.spage%253D948%26rft.epage%253D958%26rft.atitle%253DGenetic%2Bdissection%2Bof%2Bfusiform%2Brust%2Band%2Bpitch%2Bcanker%2Bdisease%2Btraits%2Bin%2Bloblolly%2Bpine.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-004-1915-2%26rft_id%253Dinfo%253Apmid%252F15700146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-004-1915-2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15700146&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.26" data-doi="10.1139/X10-045"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kayihan</span>,  <span class="cit-name-given-names">G. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. D.</span> <span class="cit-name-surname">Nelson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. A.</span> <span class="cit-name-surname">Huber</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. V.</span> <span class="cit-name-surname">Amerson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. L.</span> <span class="cit-name-surname">White</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Clonal evaluation for fusiform rust disease resistance: effects of pathogen virulence and disease escape</span>. <abbr class="cit-jnl-abbrev">Can. J. For. Res.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1042</span>–<span class="cit-lpage">1050</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1139/X10-045</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCan.%2BJ.%2BFor.%2BRes.%26rft.volume%253D40%26rft.spage%253D1042%26rft_id%253Dinfo%253Adoi%252F10.1139%252FX10-045%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1139/X10-045&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.27" data-doi="10.2527/jas.2008-1259"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. J. M.</span> <span class="cit-name-surname">Rosa</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">a</span> <span class="cit-article-title">Reproducing kernel Hilbert spaces regression: a general framework for genetic evaluation</span>. <abbr class="cit-jnl-abbrev">J. Anim. Sci.</abbr> <span class="cit-vol">87</span>: <span class="cit-fpage">1883</span>–<span class="cit-lpage">1887</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.2527/jas.2008-1259</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.2527%252Fjas.2008-1259%26rft_id%253Dinfo%253Apmid%252F19213705%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.2527/jas.2008-1259&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19213705&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.28" data-doi="10.1017/S0016672310000285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">De los Campos</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. J. M.</span> <span class="cit-name-surname">Rosa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Weigel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Semi-parametric genomic-enabled prediction of genetic values using reproducing kernel Hilbert spaces methods</span>. <abbr class="cit-jnl-abbrev">Genet. Res.</abbr> <span class="cit-vol">92</span>: <span class="cit-fpage">295</span>–<span class="cit-lpage">308</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1017/S0016672310000285</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetical%2Bresearch%26rft.stitle%253DGenet%2BRes%26rft.aulast%253DDE%2BLos%2BCampos%26rft.auinit1%253DG.%26rft.volume%253D92%26rft.issue%253D4%26rft.spage%253D295%26rft.epage%253D308%26rft.atitle%253DSemi-parametric%2Bgenomic-enabled%2Bprediction%2Bof%2Bgenetic%2Bvalues%2Busing%2Breproducing%2Bkernel%2BHilbert%2Bspaces%2Bmethods.%26rft_id%253Dinfo%253Adoi%252F10.1017%252FS0016672310000285%26rft_id%253Dinfo%253Apmid%252F20943010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1017/S0016672310000285&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20943010&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.29" data-doi="10.1534/genetics.112.143313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Hickey</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Pong-Wong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H. D.</span> <span class="cit-name-surname">Daetwyler</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. P. L.</span> <span class="cit-name-surname">Calus</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Whole-Genome Regression and Prediction Methods Applied to Plant and Animal Breeding</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">327</span>–<span class="cit-lpage">345</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.112.143313</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.112.143313%26rft_id%253Dinfo%253Apmid%252F22745228%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkzLzIvMzI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI3MzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.30" data-doi="10.1534/genetics.109.101501"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Naya</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Crossa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Legarra</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span><span class="cit-comment">b</span> <span class="cit-article-title">Predicting quantitative traits with regression models for dense molecular markers and pedigree</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">375</span>–<span class="cit-lpage">385</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.109.101501</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101501%26rft_id%253Dinfo%253Apmid%252F19293140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzEvMzc1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI3MzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>,  <span class="cit-name-given-names">G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. R.</span> <span class="cit-name-surname">Perez</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">BGLR: Bayesian Generalized Linear Regression</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.32" data-doi="10.1371/journal.pgen.1005048"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de los Campos</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Sorensen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genomic Heritability: What Is It?</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">11</span>: <span class="cit-elocation-id">e1005048</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1005048</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D11%26rft.spage%253De1005048%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1005048%26rft_id%253Dinfo%253Apmid%252F25942577%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1005048&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25942577&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-book" id="cit-9.8.2739.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Walsh</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-source">Genetics and Analysis of Quantitative Traits</span>, <span class="cit-publ-name">Sinauer</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.34" data-doi="10.1093/nar/gkv1344"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meuwissen</span>,  <span class="cit-name-given-names">T. H. E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Hayes</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Goddard</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Prediction of Total Genetic Value Using Genome-Wide Dense Marker Maps</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1819</span>–<span class="cit-lpage">1829</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMeuwissen%26rft.auinit1%253DT.%2BH.%2BE.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1819%26rft.epage%253D1829%26rft.atitle%253DPrediction%2Bof%2BTotal%2BGenetic%2BValue%2BUsing%2BGenome-Wide%2BDense%2BMarker%2BMaps%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkv1344%26rft_id%253Dinfo%253Apmid%252F11290733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE4MTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzgvMjczOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.35" data-doi="10.3389/fgene.2014.00056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morota</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Boddhireddy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Vukasinovic</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">DeNise</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Kernel-based variance component estimation and whole-genome prediction of pre-corrected phenotypes and progeny tests for dairy cow health traits</span>. <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">56</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2014.00056</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D5%26rft.spage%253D56%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2014.00056%26rft_id%253Dinfo%253Apmid%252F24715901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2014.00056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24715901&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.36" data-doi="10.3389/fgene.2014.00363"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morota</span>,  <span class="cit-name-given-names">G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Kernel-based whole-genome prediction of complex traits: a review</span>. <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">363</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2014.00363</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D5%26rft.spage%253D363%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2014.00363%26rft_id%253Dinfo%253Apmid%252F25360145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2014.00363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25360145&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.37" data-doi="10.1186/1297-9686-45-17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morota</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Koyama</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. J. M.</span> <span class="cit-name-surname">Rosa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Weigel</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Predicting complex traits using a diffusion kernel on genetic markers with an application to dairy cattle and wheat data</span>. <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">17</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-45-17</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D45%26rft.spage%253D17%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-45-17%26rft_id%253Dinfo%253Apmid%252F23763755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-45-17&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23763755&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-book" id="cit-9.8.2739.38" data-doi="10.1079/9781780643915.0000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mrode</span>,  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-source">Linear Models for the Prediction of Animal Breeding Values</span>, <span class="cit-publ-name">CABI Pub.</span><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1079/9781780643915.0000</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DLinear%2BModels%2Bfor%2Bthe%2BPrediction%2Bof%2BAnimal%2BBreeding%2BValues%26rft_id%253Dinfo%253Adoi%252F10.1079%252F9781780643915.0000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1079/9781780643915.0000&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.39" data-doi="10.1534/genetics.114.171322"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muñoz</span>,  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. F. R.</span> <span class="cit-name-surname">Resende</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Gezan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D. V.</span> <span class="cit-name-surname">Resende</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de los Campos</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Unraveling Additive from Non-Additive Effects Using Genomic Relationship Matrices</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">1759</span>–<span class="cit-lpage">1768</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.171322</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.171322%26rft_id%253Dinfo%253Apmid%252F25324160%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5OC80LzE3NTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzgvMjczOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.40" data-doi="10.1371/journal.pone.0085792"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nishio</span>,  <span class="cit-name-given-names">M.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Satoh</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Including Dominance Effects in the Genomic BLUP Method for Genomic Evaluation</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e85792</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0085792</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De85792%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0085792%26rft_id%253Dinfo%253Apmid%252F24416447%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0085792&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24416447&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.41" data-doi="10.1534/genetics.114.164442"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez</span>,  <span class="cit-name-given-names">P.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de los Campos</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-Wide Regression &amp; Prediction with the BGLR Statistical Package</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">483</span>–<span class="cit-lpage">495</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.164442</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164442%26rft_id%253Dinfo%253Apmid%252F25009151%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk4LzIvNDgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI3MzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.42" data-doi="10.3390/f5020347"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Quesada</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. F. R.</span> <span class="cit-name-surname">Resende</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Muñoz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wegrzyn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Neale</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Mapping Fusiform Rust Resistance Genes within a Complex Mating Design of Loblolly Pine</span>. <abbr class="cit-jnl-abbrev">Forests</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">362</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3390/f5020347</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DForests%26rft.volume%253D5%26rft.spage%253D347%26rft_id%253Dinfo%253Adoi%252F10.3390%252Ff5020347%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/f5020347&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000336086000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.43" data-doi="10.1111/j.1469-8137.2011.03895.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>,  <span class="cit-name-given-names">M. F. R.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Muñoz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. J.</span> <span class="cit-name-surname">Acosta</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. F.</span> <span class="cit-name-surname">Peter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Davis</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Accelerating the domestication of trees using genomic selection: accuracy of prediction models across ages and environments</span>. <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">193</span>: <span class="cit-fpage">617</span>–<span class="cit-lpage">624</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-8137.2011.03895.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DResende%26rft.auinit1%253DM.%2BF.%26rft.volume%253D193%26rft.issue%253D3%26rft.spage%253D617%26rft.epage%253D624%26rft.atitle%253DAccelerating%2Bthe%2Bdomestication%2Bof%2Btrees%2Busing%2Bgenomic%2Bselection%253A%2Baccuracy%2Bof%2Bprediction%2Bmodels%2Bacross%2Bages%2Band%2Benvironments.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2011.03895.x%26rft_id%253Dinfo%253Apmid%252F21973055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2011.03895.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21973055&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298984900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.44" data-doi="10.1534/genetics.111.137026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>,  <span class="cit-name-given-names">M. F. R.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Muñoz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D. V.</span> <span class="cit-name-surname">Resende</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Garrick</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Fernando</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">Accuracy of genomic selection methods in a standard data set of loblolly pine (Pinus taeda L.)</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">1503</span>–<span class="cit-lpage">1510</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.111.137026</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.137026%26rft_id%253Dinfo%253Apmid%252F22271763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MC80LzE1MDMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzgvMjczOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.45" data-doi="10.1111/j.1469-8137.2011.04038.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>,  <span class="cit-name-given-names">M. D. V.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. F. R.</span> <span class="cit-name-surname">Resende</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. P.</span> <span class="cit-name-surname">Sansaloni</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. D.</span> <span class="cit-name-surname">Petroli</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Missiaggia</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genomic selection for growth and wood quality in Eucalyptus: capturing the missing heritability and accelerating breeding for complex traits in forest trees</span>. <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">116</span>–<span class="cit-lpage">128</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1469-8137.2011.04038.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DResende%26rft.auinit1%253DM.%2BD.%26rft.volume%253D194%26rft.issue%253D1%26rft.spage%253D116%26rft.epage%253D128%26rft.atitle%253DGenomic%2Bselection%2Bfor%2Bgrowth%2Band%2Bwood%2Bquality%2Bin%2BEucalyptus%253A%2Bcapturing%2Bthe%2Bmissing%2Bheritability%2Band%2Baccelerating%2Bbreeding%2Bfor%2Bcomplex%2Btraits%2Bin%2Bforest%2Btrees.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2011.04038.x%26rft_id%253Dinfo%253Apmid%252F22309312%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2011.04038.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22309312&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000300719400014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.46" data-doi="10.1038/hdy.2017.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resende</span>,  <span class="cit-name-given-names">R. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. D. V.</span> <span class="cit-name-surname">Resende</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F. F.</span> <span class="cit-name-surname">Silva</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. F.</span> <span class="cit-name-surname">Azevedo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. K.</span> <span class="cit-name-surname">Takahashi</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Assessing the expected response to genomic selection of individuals and families in Eucalyptus breeding with an additive-dominant model</span>. <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">119</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">255</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/hdy.2017.37</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D119%26rft.spage%253D245%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fhdy.2017.37%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/hdy.2017.37&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.47" data-doi="10.1186/1297-9686-42-33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Toro</span>,  <span class="cit-name-given-names">M. A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Varona</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A note on mate allocation for dominance handling in genomic selection</span>. <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">33</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-42-33</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%252C%2Bselection%252C%2Bevolution.%2B%253A%2B%2BGSE%26rft.stitle%253DGenet%2BSel%2BEvol%26rft.aulast%253DToro%26rft.auinit1%253DM.%2BA.%26rft.volume%253D42%26rft.spage%253D33%26rft.epage%253D33%26rft.atitle%253DA%2Bnote%2Bon%2Bmate%2Ballocation%2Bfor%2Bdominance%2Bhandling%2Bin%2Bgenomic%2Bselection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-42-33%26rft_id%253Dinfo%253Apmid%252F20699012%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-42-33&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20699012&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.48" data-doi="10.1111/jbg.12070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tusell</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Pérez-Rodríguez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Forni</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Model averaging for genome-enabled prediction with reproducing kernel Hilbert spaces: a case study with pig litter size and wheat yield</span>. <abbr class="cit-jnl-abbrev">J. Anim. Breed. Genet.</abbr> <span class="cit-vol">131</span>: <span class="cit-fpage">105</span>–<span class="cit-lpage">115</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/jbg.12070</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAnim.%2BBreed.%2BGenet.%26rft.volume%253D131%26rft.spage%253D105%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjbg.12070%26rft_id%253Dinfo%253Apmid%252F24397267%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jbg.12070&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24397267&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.49" data-doi="10.3168/jds.2010-3335"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vazquez</span>,  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G. J. M.</span> <span class="cit-name-surname">Rosa</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. A.</span> <span class="cit-name-surname">Weigel</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">de los Campos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gianola</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Predictive ability of subsets of single nucleotide polymorphisms with and without parent average in US Holsteins</span>. <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">5942</span>–<span class="cit-lpage">5949</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2010-3335</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DVazquez%26rft.auinit1%253DA.%2BI.%26rft.volume%253D93%26rft.issue%253D12%26rft.spage%253D5942%26rft.epage%253D5949%26rft.atitle%253DPredictive%2Bability%2Bof%2Bsubsets%2Bof%2Bsingle%2Bnucleotide%2Bpolymorphisms%2Bwith%2Band%2Bwithout%2Bparent%2Baverage%2Bin%2BUS%2BHolsteins.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2010-3335%26rft_id%253Dinfo%253Apmid%252F21094768%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2010-3335&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21094768&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.50" data-doi="10.3168/jds.2010-3866"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wiggans</span>,  <span class="cit-name-given-names">G. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. M.</span> <span class="cit-name-surname">Vanraden</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. A.</span> <span class="cit-name-surname">Cooper</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The genomic evaluation system in the United States: past, present, future</span>. <abbr class="cit-jnl-abbrev">J. Dairy Sci.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">3202</span>–<span class="cit-lpage">3211</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3168/jds.2010-3866</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BDairy%2BScience%26rft.stitle%253DJ%2BDAIRY%2BSCI%26rft.aulast%253DWiggans%26rft.auinit1%253DG.%2BR.%26rft.volume%253D94%26rft.issue%253D6%26rft.spage%253D3202%26rft.epage%253D3211%26rft.atitle%253DThe%2Bgenomic%2Bevaluation%2Bsystem%2Bin%2Bthe%2BUnited%2BStates%253A%2Bpast%252C%2Bpresent%252C%2Bfuture.%26rft_id%253Dinfo%253Adoi%252F10.3168%252Fjds.2010-3866%26rft_id%253Dinfo%253Apmid%252F21605789%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3168/jds.2010-3866&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21605789&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.51" data-doi="10.1093/molbev/msl131"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Willyard</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Syring</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. S.</span> <span class="cit-name-surname">Gernandt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Liston</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Cronn</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Fossil calibration of molecular divergence infers a moderate mutation rate and recent radiations for pinus</span>. <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">90</span>–<span class="cit-lpage">101</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/molbev/msl131</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsl131%26rft_id%253Dinfo%253Apmid%252F16997907%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/molbev/msl131&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16997907&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000243069700012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xavier</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. M.</span> <span class="cit-name-surname">Muir</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. M.</span> <span class="cit-name-surname">Rainey</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Assessing Predictive Properties of Genome-Wide Selection in Soybeans. G3 Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">2611</span>–<span class="cit-lpage">2616</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%26rft.volume%253D6%26rft.spage%253D2611%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2739.53" data-doi="10.1186/1297-9686-45-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Toosi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. L.</span> <span class="cit-name-surname">Fernando</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Dekkers</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Garrick</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Genomic selection of purebred animals for crossbred performance in the presence of dominant gene action</span>. <abbr class="cit-jnl-abbrev">Genet. Sel. Evol.</abbr> <span class="cit-vol">45</span>: <span class="cit-fpage">11</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1297-9686-45-11</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BSel.%2BEvol.%26rft.volume%253D45%26rft.spage%253D11%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1297-9686-45-11%26rft_id%253Dinfo%253Apmid%252F23621868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1297-9686-45-11&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23621868&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2739.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/8/2739.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/8/2723" title="Comparative Transcriptome Analyses Revealed Conserved and Novel Responses to Cold and Freezing Stress in Brassica napus L" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/8/2749" title="Genome-Wide Profiling of Polyadenylation Events in Maize Using High-Throughput Transcriptomic Sequences" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/8"><span>Volume 9 Issue 8, August 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/8.cover-source.jpg?itok=cSNYBBSf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/8/2739" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-qY3qFlxYP20wpbrlFuqp-Cr9OdArjeAcn26soIRgiRg" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/genomic-prediction" class="" data-icon-position="" data-hide-link-title="0">Genomic Prediction</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/8/2739" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/8/2739" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/8/2739" class="active" data-icon-position="" data-hide-link-title="0">Genomic Prediction of Additive and Non-additive Effects Using Genetic Markers and Pedigrees</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/493732" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-zoif9JxWO4tUBU60DGmCA12j-Mplu_amEZIp91eZM84" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/8/2739.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/8/2739" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-aKsBq4VizdjbdmKTEtwUECZUIxlMF5OD2LKoTWaOt1Y" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/8/2739" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/8/2739.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493732" id="node493732--2" data-pisa="ggg;9/8/2739" data-pisa-master="ggg;g3.119.201004" data-apath="/ggg/9/8/2739.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction of Additive and Non-additive Effects Using Genetic Markers and Pedigrees</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3906-8884" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Janeo Eustáquio de Almeida Filho</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-3849-9834" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>João Filipi Rodrigues Guimarães</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9536-1113" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fabyano Fonsceca e Silva</span>, <span class="highwire-citation-author" data-delta="3">Marcos Deon Vilela de Resende</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-8973-9351" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patricio Muñoz</span>, <span class="highwire-citation-author" data-delta="5">Matias Kirst</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-2367-0766" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marcio Fernando Ribeiro de Resende Júnior</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2739-2748;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.201004 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493732--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Janeo Eustáquio de Almeida Filho </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Plant Improvenent Lab, Norte Fluminense “Darcy Ribeiro” State University, Campos dos Goytacazes RJ, 28013-602, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Janeo%2BEust%C3%A1quio%2Bde%2BAlmeida%2BFilho%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Almeida%20Filho%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJaneo%2BEust%25C3%25A1quio%2Bde%2BAlmeida%2BFilho%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3906-8884" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Janeo Eustáquio de Almeida Filho</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">João Filipi Rodrigues Guimarães </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Futuragene Ltda, Itapetininga SP, 18207-780, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jo%C3%A3o%2BFilipi%2BRodrigues%2BGuimar%C3%A3es%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guimar%C3%A3es%20JF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJo%25C3%25A3o%2BFilipi%2BRodrigues%2BGuimar%25C3%25A3es%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3849-9834" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for João Filipi Rodrigues Guimarães</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Fabyano Fonsceca e Silva </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Animal Science Department, Federal University of Viçosa, Viçosa, MG 36570-000, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fabyano%2BFonsceca%2Be%2BSilva%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fonsceca%20e%20Silva%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFabyano%2BFonsceca%2Be%2BSilva%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9536-1113" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fabyano Fonsceca e Silva</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Marcos Deon Vilela de Resende </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>EMBRAPA Forestry, Colombo, PR 83411-000, Brazil / Statistics Department, Federal University of Viçosa, Viçosa, MG 36570-000, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcos%2BDeon%2BVilela%2Bde%2BResende%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vilela%20de%20Resende%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarcos%2BDeon%2BVilela%2Bde%2BResende%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Patricio Muñoz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Horticultural Sciences Department, University of Florida, Gainesville, FL 32611, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patricio%2BMu%C3%B1oz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mu%C3%B1oz%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatricio%2BMu%25C3%25B1oz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8973-9351" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patricio Muñoz</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Matias Kirst </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>University of Florida Genetics Institute, School of Forest Resources and Conservation, Gainesville, FL 32611</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matias%2BKirst%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kirst%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatias%2BKirst%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Marcio Fernando Ribeiro de Resende Júnior </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Horticultural Sciences Department, University of Florida, Gainesville, FL 32611, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcio%2BFernando%2BRibeiro%2Bde%2BResende%2BJ%C3%BAnior%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Resende%20J%C3%BAnior%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarcio%2BFernando%2BRibeiro%2Bde%2BResende%2BJ%25C3%25BAnior%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2367-0766" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marcio Fernando Ribeiro de Resende Júnior</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mresende@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">mresende@ufl.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/8/2739" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/8/2739&amp;title=Genomic%20Prediction%20of%20Additive%20and%20Non-additive%20Effects%20Using%20Genetic%20Markers%20and%20Pedigrees" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/8/2739&amp;title=Genomic%20Prediction%20of%20Additive%20and%20Non-additive%20Effects%20Using%20Genetic%20Markers%20and%20Pedigrees" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/8/2739&amp;title=Genomic%20Prediction%20of%20Additive%20and%20Non-additive%20Effects%20Using%20Genetic%20Markers%20and%20Pedigrees" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/8/2739&amp;text=Genomic%20Prediction%20of%20Additive%20and%20Non-additive%20Effects%20Using%20Genetic%20Markers%20and%20Pedigrees" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/8/2739&amp;title=Genomic%20Prediction%20of%20Additive%20and%20Non-additive%20Effects%20Using%20Genetic%20Markers%20and%20Pedigrees" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/8/2739&amp;t=Genomic%20Prediction%20of%20Additive%20and%20Non-additive%20Effects%20Using%20Genetic%20Markers%20and%20Pedigrees" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/8/2739&amp;title=Genomic%20Prediction%20of%20Additive%20and%20Non-additive%20Effects%20Using%20Genetic%20Markers%20and%20Pedigrees" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/8/2739&amp;title=Genomic%20Prediction%20of%20Additive%20and%20Non-additive%20Effects%20Using%20Genetic%20Markers%20and%20Pedigrees" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/493732/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493732" id="node493732--3" data-pisa="ggg;9/8/2739" data-pisa-master="ggg;g3.119.201004" data-apath="/ggg/9/8/2739.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Prediction of Additive and Non-additive Effects Using Genetic Markers and Pedigrees</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3906-8884" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Janeo Eustáquio de Almeida Filho</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0002-3849-9834" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>João Filipi Rodrigues Guimarães</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="2"><a href="http://orcid.org/0000-0001-9536-1113" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Fabyano Fonsceca e Silva</span>, <span class="highwire-citation-author" data-delta="3">Marcos Deon Vilela de Resende</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0001-8973-9351" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Patricio Muñoz</span>, <span class="highwire-citation-author" data-delta="5">Matias Kirst</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0002-2367-0766" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Marcio Fernando Ribeiro de Resende Júnior</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2739-2748;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.201004 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493732--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Janeo Eustáquio de Almeida Filho </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Genetic Plant Improvenent Lab, Norte Fluminense “Darcy Ribeiro” State University, Campos dos Goytacazes RJ, 28013-602, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Janeo%2BEust%C3%A1quio%2Bde%2BAlmeida%2BFilho%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Almeida%20Filho%20JE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJaneo%2BEust%25C3%25A1quio%2Bde%2BAlmeida%2BFilho%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3906-8884" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Janeo Eustáquio de Almeida Filho</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">João Filipi Rodrigues Guimarães </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Futuragene Ltda, Itapetininga SP, 18207-780, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jo%C3%A3o%2BFilipi%2BRodrigues%2BGuimar%C3%A3es%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Guimar%C3%A3es%20JF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJo%25C3%25A3o%2BFilipi%2BRodrigues%2BGuimar%25C3%25A3es%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3849-9834" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for João Filipi Rodrigues Guimarães</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Fabyano Fonsceca e Silva </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Animal Science Department, Federal University of Viçosa, Viçosa, MG 36570-000, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fabyano%2BFonsceca%2Be%2BSilva%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fonsceca%20e%20Silva%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AFabyano%2BFonsceca%2Be%2BSilva%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9536-1113" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Fabyano Fonsceca e Silva</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Marcos Deon Vilela de Resende </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>EMBRAPA Forestry, Colombo, PR 83411-000, Brazil / Statistics Department, Federal University of Viçosa, Viçosa, MG 36570-000, Brazil,</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcos%2BDeon%2BVilela%2Bde%2BResende%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vilela%20de%20Resende%20MD&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarcos%2BDeon%2BVilela%2Bde%2BResende%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Patricio Muñoz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Horticultural Sciences Department, University of Florida, Gainesville, FL 32611, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Patricio%2BMu%C3%B1oz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Mu%C3%B1oz%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APatricio%2BMu%25C3%25B1oz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-8973-9351" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Patricio Muñoz</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Matias Kirst </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>University of Florida Genetics Institute, School of Forest Resources and Conservation, Gainesville, FL 32611</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matias%2BKirst%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kirst%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatias%2BKirst%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Marcio Fernando Ribeiro de Resende Júnior </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Horticultural Sciences Department, University of Florida, Gainesville, FL 32611, and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marcio%2BFernando%2BRibeiro%2Bde%2BResende%2BJ%C3%BAnior%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Resende%20J%C3%BAnior%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMarcio%2BFernando%2BRibeiro%2Bde%2BResende%2BJ%25C3%25BAnior%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-2367-0766" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Marcio Fernando Ribeiro de Resende Júnior</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:mresende@ufl.edu" class="" data-icon-position="" data-hide-link-title="0">mresende@ufl.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2739&amp;rft.epage=2748&amp;rft.atitle=Genomic%20Prediction%20of%20Additive%20and%20Non-additive%20Effects%20Using%20Genetic%20Markers%20and%20Pedigrees&amp;rft.volume=9&amp;rft.issue=8&amp;rft.date=2019-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=de+Almeida+Filho%2C+Janeo+Eust%C3%A1quio&amp;rft.au=Guimar%C3%A3es%2C+Jo%C3%A3o+Filipi+Rodrigues&amp;rft.au=Fonsceca+e+Silva%2C+Fabyano&amp;rft.au=Vilela+de+Resende%2C+Marcos+Deon&amp;rft.au=Mu%C3%B1oz%2C+Patricio&amp;rft.au=Kirst%2C+Matias&amp;rft.au=de+Resende+J%C3%BAnior%2C+Marcio+Fernando+Ribeiro"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/493732/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/493732/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/493732/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/493732/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/493732/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/493732/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/493732/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/493732/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/493732/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/493732/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/493732/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/493732/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495864" id="node495864" data-pisa="ggg;10/3/1113" data-pisa-master="ggg;g3.119.400968" data-apath="/ggg/10/3/1113.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1113" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Multi-trait Genomic Prediction Model Increased the Predictive Ability for Agronomic and Malting Quality Traits in Barley (<em>Hordeum vulgare</em> L.)</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495543" id="node495543" data-pisa="ggg;10/2/769" data-pisa-master="ggg;g3.119.400759" data-apath="/ggg/10/2/769.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/769" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Novel Bayesian Networks for Genomic Prediction of Developmental Traits in Biomass Sorghum</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495557" id="node495557" data-pisa="ggg;10/2/539" data-pisa-master="ggg;g3.119.400866" data-apath="/ggg/10/2/539.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/2/539" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Optimized Genetic Testing for Polledness in Multiple Breeds of Cattle</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Genomic%20Prediction" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Genomic Prediction</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/8/2739" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Material and Methods</a></li><li class="odd"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-21" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/8/2739.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/8/2739.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/8/2739.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
