<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/8/2749" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/8/2749.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/8/2749.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/8/2749.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/8/2749.atom" />
<meta name="HW.pisa" content="ggg;9/8/2749" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genome-Wide Profiling of Polyadenylation Events in Maize Using High-Throughput Transcriptomic Sequences" />
<meta name="DC.Identifier" content="10.1534/g3.119.400196" />
<meta name="DC.Date" content="2019-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2019 Jafar et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Polyadenylation is an essential post-transcriptional modification of eukaryotic transcripts that plays critical role in transcript stability, localization, transport, and translational efficiency. About 70% genes in plants contain alternative polyadenylation (APA) sites. Despite availability of vast amount of sequencing data, to date, a comprehensive map of the polyadenylation events in maize is not available. Here, 9.48 billion RNA-Seq reads were analyzed to characterize 95,345 Poly(A) Clusters (PAC) in 23,705 (51%) maize genes. Of these, 76% were APA genes. However, most APA genes (55%) expressed a dominant PAC rather than favoring multiple PACs equally. The lincRNA genes with PACs were significantly longer in length than the genes without any PAC and about 48% genes had APA sites. Heterogeneity was observed in 52% of the PACs supporting the imprecise nature of the polyadenylation process. Genomic distribution revealed that the majority of the PACs (78%) were located in the genic regions. Unlike previous studies, large number of PACs were observed in the intergenic (n = 21,264), 5′-UTR (735), CDS (2,542), and the intronic regions (12,841). The CDS and introns with PACs were longer in length than without PACs, whereas intergenic PACs were more often associated with transcripts that lacked annotated 3′-UTRs. Nucleotide composition around PACs demonstrated AT-richness and the common upstream motif was AAUAAA, which is consistent with other plants. According to this study, only 2,830 genes still maintained the use of AAUAAA motif. This large-scale data provides useful insights about the gene expression regulation and could be utilized as evidence to validate the annotation of transcript ends." />
<meta name="DC.Contributor" content="Zehra Jafar" />
<meta name="DC.Contributor" content="Salma Tariq" />
<meta name="DC.Contributor" content="Irfan Sadiq" />
<meta name="DC.Contributor" content="Tayyab Nawaz" />
<meta name="DC.Contributor" content="Malik Nadeem Akhtar" />
<meta name="article:published_time" content="2019-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genome-Wide Profiling of Polyadenylation Events in Maize Using High-Throughput Transcriptomic Sequences" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Polyadenylation is an essential post-transcriptional modification of eukaryotic transcripts that plays critical role in transcript stability, localization, transport, and translational efficiency. About 70% genes in plants contain alternative polyadenylation (APA) sites. Despite availability of vast amount of sequencing data, to date, a comprehensive map of the polyadenylation events in maize is not available. Here, 9.48 billion RNA-Seq reads were analyzed to characterize 95,345 Poly(A) Clusters (PAC) in 23,705 (51%) maize genes. Of these, 76% were APA genes. However, most APA genes (55%) expressed a dominant PAC rather than favoring multiple PACs equally. The lincRNA genes with PACs were significantly longer in length than the genes without any PAC and about 48% genes had APA sites. Heterogeneity was observed in 52% of the PACs supporting the imprecise nature of the polyadenylation process. Genomic distribution revealed that the majority of the PACs (78%) were located in the genic regions. Unlike previous studies, large number of PACs were observed in the intergenic (n = 21,264), 5′-UTR (735), CDS (2,542), and the intronic regions (12,841). The CDS and introns with PACs were longer in length than without PACs, whereas intergenic PACs were more often associated with transcripts that lacked annotated 3′-UTRs. Nucleotide composition around PACs demonstrated AT-richness and the common upstream motif was AAUAAA, which is consistent with other plants. According to this study, only 2,830 genes still maintained the use of AAUAAA motif. This large-scale data provides useful insights about the gene expression regulation and could be utilized as evidence to validate the annotation of transcript ends.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/08/01" />
<meta name="citation_mjid" content="ggg;9/8/2749" />
<meta name="citation_id" content="9/8/2749" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/8/2749" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/8/2749.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/8/2749.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/8/2749.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.119.400196" />
<meta name="citation_pmid" content="31239292" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2749" />
<meta name="citation_lastpage" content="2760" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Zehra Jafar" />
<meta name="citation_author_institution" content="Computational Biology and Bioinformatics Laboratory, Department of Biosciences, COMSATS University Islamabad, Islamabad 45550, Pakistan" />
<meta name="citation_author" content="Salma Tariq" />
<meta name="citation_author_institution" content="Computational Biology and Bioinformatics Laboratory, Department of Biosciences, COMSATS University Islamabad, Islamabad 45550, Pakistan" />
<meta name="citation_author" content="Irfan Sadiq" />
<meta name="citation_author_institution" content="Computational Biology and Bioinformatics Laboratory, Department of Biosciences, COMSATS University Islamabad, Islamabad 45550, Pakistan" />
<meta name="citation_author" content="Tayyab Nawaz" />
<meta name="citation_author_institution" content="Department of Mathematics, COMSATS University Islamabad, Islamabad 45550, Pakistan" />
<meta name="citation_author" content="Malik Nadeem Akhtar" />
<meta name="citation_author_institution" content="Computational Biology and Bioinformatics Laboratory, Department of Biosciences, COMSATS University Islamabad, Islamabad 45550, Pakistan" />
<meta name="citation_author_email" content="nadeemakhtar@comsats.edu.pk" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=A. M. Bolger;citation_author=M. Lohse;citation_author=B. Usadel;citation_title=Trimmomatic: a flexible trimmer for illumina sequence data;citation_volume=30;citation_year=2014;citation_pmid=24695404;citation_doi=10.1093/bioinformatics/btu170" />
<meta name="citation_reference" content="citation_journal_title=Genomics Proteomics Bioinformatics;citation_author=W. Chen;citation_author=Q. Jia;citation_author=Y. Song;citation_author=H. Fu;citation_author=G. Wei;citation_title=Alternative polyadenylation: methods, findings, and impacts;citation_pages=287-300;citation_volume=15;citation_year=2017;citation_doi=10.1016/j.gpb.2017.06.001" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. Derti;citation_author=P. Garrett-Engele;citation_author=K. D. MacIsaac;citation_author=R. C. Stevens;citation_author=S. Sriram;citation_title=A quantitative atlas of polyadenylation in five mammals;citation_volume=22;citation_year=2012;citation_pmid=22454233;citation_doi=10.1101/gr.132563.111" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=D. C. Di Giammartino;citation_author=K. Nishida;citation_author=J. L. Manley;citation_title=Mechanisms and consequences of alternative polyadenylation.;citation_pages=853-866;citation_volume=43;citation_year=2011;citation_issue=6;citation_pmid=21925375;citation_doi=10.1016/j.molcel.2011.08.017" />
<meta name="citation_reference" content="citation_title=Polyadenylation in Plants;citation_author=M. Dong;citation_author=G. Ji;citation_author=Q. Q. Li;citation_author=C. Liang;citation_title=Extraction of poly (a) sites from large-scale rna-seq data;citation_year=2015;citation_doi=10.1007/978-1-4939-2175-1_3" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=Z. Du;citation_author=X. Zhou;citation_author=Y. Ling;citation_author=Z. Zhang;citation_author=Z. Su;citation_title=agrigo: a go analysis toolkit for the agricultural community;citation_volume=38;citation_year=2010;citation_pmid=20435677;citation_doi=10.1093/nar/gkq310" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=H. Fu;citation_author=D. Yang;citation_author=W. Su;citation_author=L. Ma;citation_author=Y. Shen;citation_title=Genome-wide dynamics of alternative polyadenylation in rice;citation_volume=26;citation_year=2016;citation_pmid=27733415;citation_doi=10.1101/gr.210757.116" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=M. Gonzàlez-Porta;citation_author=A. Frankish;citation_author=J. Rung;citation_author=J. Harrow;citation_author=A. Brazma;citation_title=Transcriptome analysis of human tissues and cell lines reveals one dominant transcript per gene;citation_pages=R70;citation_volume=14;citation_year=2013;citation_pmid=23815980;citation_doi=10.1186/gb-2013-14-7-r70" />
<meta name="citation_reference" content="citation_journal_title=Wiley Interdiscip. Rev. RNA;citation_author=A. R. Gruber;citation_author=G. Martin;citation_author=W. Keller;citation_author=M. Zavolan;citation_title=Means to an end: mechanisms of alternative polyadenylation of messenger rna precursors;citation_pages=183-196;citation_volume=5;citation_year=2014;citation_pmid=24243805;citation_doi=10.1002/wrna.1206" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=C. Guo;citation_author=M. Spinelli;citation_author=M. Liu;citation_author=Q. Q. Li;citation_author=C. Liang;citation_title=A genomewide study of “non-3utr” polyadenylation sites in arabidopsis thaliana;citation_pages=28060;citation_volume=6;citation_year=2016;citation_doi=10.1038/srep28060" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=K. Hanada;citation_author=X. Zhang;citation_author=J. O. Borevitz;citation_author=W.-H. Li;citation_author=S.-H. Shiu;citation_title=A large number of novel coding small open reading frames in the intergenic regions of the arabidopsis thaliana genome are transcribed and/or under purifying selection;citation_volume=17;citation_year=2007;citation_pmid=17395691;citation_doi=10.1101/gr.5836207" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=P. F. Harrison;citation_author=D. R. Powell;citation_author=J. L. Clancy;citation_author=T. Preiss;citation_author=P. R. Boag;citation_title=Pat-seq: a method to study the integration of 30 -utr dynamics with gene expression in the eukaryotic transcriptome;citation_volume=21;citation_year=2015;citation_pmid=26092945;citation_doi=10.1261/rna.048355.114" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=A. G. Hunt;citation_author=D. Xing;citation_author=Q. Q. Li;citation_title=Plant polyadenylation factors: conservation and variety in the polyadenylation complex in plants.;citation_pages=641-641;citation_volume=13;citation_year=2012;citation_issue=1;citation_pmid=23167306;citation_doi=10.1186/1471-2164-13-641" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=H. Jeong;citation_author=A. Nasir;citation_title=A preliminary list of horizontally transferred genes in prokaryotes determined by tree reconstruction and reconciliation;citation_pages=112;citation_volume=8;citation_year=2017;citation_doi=10.3389/fgene.2017.00112" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=B. Langmead;citation_author=S. L. Salzberg;citation_title=Fast gapped-read alignment with Bowtie 2.;citation_pages=357-359;citation_volume=9;citation_year=2012;citation_issue=4;citation_pmid=22388286;citation_doi=10.1038/nmeth.1923" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=J. Y. Lee;citation_author=Z. Ji;citation_author=B. Tian;citation_title=Phylogenetic analysis of mrna polyadenylation sites reveals a role of transposable elements in evolution of the 30 -end of genes;citation_volume=36;citation_year=2008;citation_pmid=18757892;citation_doi=10.1093/nar/gkn540" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. Leinonen;citation_author=H. Sugawara;citation_author=M. Shumway;citation_title=The sequence read archive;citation_volume=39;citation_year=2011;citation_pmid=21062823;citation_doi=10.1093/nar/gkq1019" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=B. Handsaker;citation_author=A. Wysoker;citation_author=T. Fennell;citation_author=J. Ruan;citation_title=The sequence alignment/map format and samtools;citation_volume=25;citation_year=2009;citation_pmid=19505943;citation_doi=10.1093/bioinformatics/btp352" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=X. Q. Li;citation_author=D. Du;citation_title=Rna polyadenylation sites on the genomes of microorganisms, animals, and plants;citation_pages=e79511;citation_volume=8;citation_year=2013;citation_doi=10.1371/journal.pone.0079511" />
<meta name="citation_reference" content="citation_journal_title=BMC Evol. Biol.;citation_author=X. Q. Li;citation_author=D. Du;citation_title=Motif types, motif locations and base composition patterns around the rna polyadenylation site in microorganisms, plants and animals;citation_pages=162;citation_volume=14;citation_year=2014;citation_doi=10.1186/s12862-014-0162-7" />
<meta name="citation_reference" content="citation_journal_title=Genomics Proteomics Bioinformatics;citation_author=X. Liu;citation_author=L. Hao;citation_author=D. Li;citation_author=L. Zhu;citation_author=S. Hu;citation_title=Long non-coding rnas and their biological roles in plants;citation_pages=137-147;citation_volume=13;citation_year=2015;citation_doi=10.1016/j.gpb.2015.02.003" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=J. C. Loke;citation_author=E. A. Stahlberg;citation_author=D. G. Strenski;citation_author=B. J. Haas;citation_author=P. C. Wood;citation_title=Compilation of mrna polyadenylation signals in arabidopsis revealed a new signal element and potential secondary structures;citation_volume=138;citation_year=2005;citation_pmid=15965016;citation_doi=10.1104/pp.105.060541" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=F. Lopez;citation_author=S. Granjeaud;citation_author=T. Ara;citation_author=B. Ghattas;citation_author=D. Gautheret;citation_title=The disparate nature of “intergenic” polyadenylation sites;citation_volume=12;citation_year=2006;citation_pmid=16931874;citation_doi=10.1261/rna.136206" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=B. C. Meyers;citation_author=T. H. Vu;citation_author=S. S. Tej;citation_author=H. Ghazal;citation_author=M. Matvienko;citation_title=Analysis of the transcriptional complexity of Arabidopsis thaliana by massively parallel signature sequencing.;citation_pages=1006-1011;citation_volume=22;citation_year=2004;citation_issue=8;citation_pmid=15247925;citation_doi=10.1038/nbt992" />
<meta name="citation_reference" content="citation_journal_title=RNA Biol.;citation_author=M. Movassat;citation_author=T. L. Crabb;citation_author=A. Busch;citation_author=C. Yao;citation_author=D. J. Reynolds;citation_title=Coupling between alternative polyadenylation and alternative splicing is limited to terminal introns;citation_pages=646-655;citation_volume=13;citation_year=2016;citation_doi=10.1080/15476286.2016.1191727" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=N. T. T. Nguyen;citation_author=B. Contreras-Moreira;citation_author=J. A. Castro-Mondragon;citation_author=W. Santana-Garcia;citation_author=R. Ossio;citation_title=Rsat 2018: regulatory sequence analysis tools 20th anniversary;citation_pages=W209-W214;citation_volume=46;citation_year=2018;citation_doi=10.1093/nar/gky317" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=E. Pauws;citation_author=A. Van Kampen;citation_author=S. Van de Graaf;citation_author=J. De Vijlder;citation_title=Heterogeneity in polyadenylation cleavage sites in mammalian mRNA sequences: implications for SAGE analysis;citation_pages=1690-1694;citation_volume=29;citation_year=2001;citation_issue=8;citation_pmid=11292841;citation_doi=10.1093/nar/29.8.1690" />
<meta name="citation_reference" content="citation_title=PostTranscriptional Control of Gene Expression in Plants;citation_author=H. M. Rothnie;citation_title=Plant mrna 30 -end formation;citation_year=1996;citation_doi=10.1007/978-94-009-0353-1_2" />
<meta name="citation_reference" content="citation_journal_title=F1000 Res.;citation_author=J. F. Schmitz;citation_author=E. Bornberg-Bauer;citation_title=Fact or fiction: updates on how protein-coding genes might emerge de novo from previously non-coding dna;citation_pages=57;citation_volume=6;citation_year=2017;citation_doi=10.12688/f1000research.10079.1" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=Y. Shen;citation_author=G. Ji;citation_author=B. J. Haas;citation_author=X. Wu;citation_author=J. Zheng;citation_title=Genome level analysis of rice mrna 30 -end processing signals and alternative polyadenylation;citation_volume=36;citation_year=2008;citation_pmid=18411206;citation_doi=10.1093/nar/gkn158" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=Y. Shen;citation_author=R. Venu;citation_author=K. Nobuta;citation_author=X. Wu;citation_author=V. Notibala;citation_title=Transcriptome dynamics through alternative polyadenylation in developmental and environmental responses in plants revealed by deep sequencing;citation_volume=21;citation_year=2011;citation_pmid=21813626;citation_doi=10.1101/gr.114744.110" />
<meta name="citation_reference" content="citation_journal_abbrev=Cell Rep;citation_author=P. Smibert;citation_author=P. Miura;citation_author=J. O. Westholm;citation_author=S. Shenker;citation_author=G. May;citation_title=Global patterns of tissue-specific alternative polyadenylation in Drosophila.;citation_pages=277-289;citation_volume=1;citation_year=2012;citation_issue=3;citation_pmid=22685694;citation_doi=10.1016/j.celrep.2012.01.001" />
<meta name="citation_reference" content="citation_year=2009" />
<meta name="citation_reference" content="citation_journal_abbrev=WIREs RNA;citation_author=B. Tian;citation_author=J. H. Graber;citation_title=Signals for pre-mRNA cleavage and polyadenylation.;citation_pages=385-396;citation_volume=3;citation_year=2012;citation_issue=3;citation_pmid=22012871;citation_doi=10.1002/wrna.116" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=B. Tian;citation_author=J. Hu;citation_author=H. Zhang;citation_author=C. S. Lutz;citation_title=A large-scale analysis of mRNA polyadenylation of human and mouse genes;citation_pages=201-212;citation_volume=33;citation_year=2005;citation_issue=1;citation_pmid=15647503;citation_doi=10.1093/nar/gki158" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Mol. Cell Biol.;citation_author=B. Tian;citation_author=J. L. Manley;citation_title=Alternative polyadenylation of mrna precursors;citation_pages=18-30;citation_volume=18;citation_year=2017;citation_doi=10.1038/nrm.2016.116" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. Tian;citation_author=Z. Pan;citation_author=J. Y. Lee;citation_title=Widespread mrna polyadenylation events in introns indicate dynamic interplay between polyadenylation and splicing;citation_volume=17;citation_year=2007;citation_pmid=17210931;citation_doi=10.1101/gr.5532707" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_author=I. Ulitsky;citation_author=D. P. Bartel;citation_title=lincrnas: genomics, evolution, and mechanisms;citation_pages=26-46;citation_volume=154;citation_year=2013;citation_pmid=23827673;citation_doi=10.1016/j.cell.2013.06.020" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=I. Ulitsky;citation_author=A. Shkumatava;citation_author=C. H. Jan;citation_author=A. O. Subtelny;citation_author=D. Koppstein;citation_title=Extensive alternative polyadenylation during zebrafish development;citation_volume=22;citation_year=2012;citation_pmid=22722342;citation_doi=10.1101/gr.139733.112" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. Wang;citation_author=M. Regulski;citation_author=E. Tseng;citation_author=A. Olson;citation_author=S. Goodwin;citation_title=A comparative transcriptional landscape of maize and sorghum obtained by single-molecule sequencing;citation_volume=28;citation_year=2018;citation_pmid=29712755;citation_doi=10.1101/gr.227462.117" />
<meta name="citation_reference" content="citation_journal_title=Nat. Commun.;citation_author=B. Wang;citation_author=E. Tseng;citation_author=M. Regulski;citation_author=T. A. Clark;citation_author=T. Hon;citation_title=Unveiling the complexity of the maize transcriptome by single-molecule long-read sequencing;citation_pages=11708;citation_volume=7;citation_year=2016;citation_doi=10.1038/ncomms11708" />
<meta name="citation_reference" content="citation_journal_title=Sci. Rep.;citation_author=H. Wang;citation_author=R. Li;citation_author=X. Zhou;citation_author=L. Xue;citation_author=X. Xu;citation_title=Genome-wide analysis and functional characterization of the polyadenylation site in pigs using rnaseq data;citation_pages=36388;citation_volume=6;citation_year=2016;citation_doi=10.1038/srep36388" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=R. Wang;citation_author=R. Nambiar;citation_author=D. Zheng;citation_author=B. Tian;citation_title=Polya_db 3 catalogs cleavage and polyadenylation sites identified by deep sequencing in multiple genomes;citation_pages=D315-D319;citation_volume=46;citation_year=2017;citation_doi=10.1093/nar/gkx1000" />
<meta name="citation_reference" content="citation_journal_title=Mol. Plant;citation_author=Y. Wang;citation_author=X. Wang;citation_author=W. Deng;citation_author=X. Fan;citation_author=T.-T. Liu;citation_title=Genomic features and regulatory roles of intermediate-sized non-coding rnas in arabidopsis;citation_volume=7;citation_year=2014;citation_pmid=24398630;citation_doi=10.1093/mp/sst177" />
<meta name="citation_reference" content="citation_journal_title=Plant Direct;citation_author=K. Wimalanathan;citation_author=I. Friedberg;citation_author=C. M. Andorf;citation_author=C. J. Lawrence-Dill;citation_title=Maize go annotation—methods, evaluation, and review (maize-gamer);citation_pages=e00052;citation_volume=2;citation_year=2018;citation_doi=10.1002/pld3.52" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=X. Wu;citation_author=B. Gaffney;citation_author=A. G. Hunt;citation_author=Q. Q. Li;citation_title=Genome-wide determination of poly (a) sites in medicago truncatula: evolutionary conservation of alternative poly (a) site choice;citation_pages=615;citation_volume=15;citation_year=2014;citation_pmid=25048171;citation_doi=10.1186/1471-2164-15-615" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=X. Wu;citation_author=M. Liu;citation_author=B. Downie;citation_author=C. Liang;citation_author=G. Ji;citation_title=Genome-wide landscape of polyadenylation in arabidopsis provides evidence for extensive alternative polyadenylation;citation_volume=108;citation_year=2011;citation_pmid=21746925;citation_doi=10.1073/pnas.1019732108" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=X. Wu;citation_author=Y. Zeng;citation_author=J. Guan;citation_author=G. Ji;citation_author=R. Huang;citation_title=Genome-wide characterization of intergenic polyadenylation sites redefines gene spaces in arabidopsis thaliana;citation_pages=511;citation_volume=16;citation_year=2015;citation_pmid=26155789;citation_doi=10.1186/s12864-015-1691-1" />
<meta name="citation_reference" content="citation_journal_title=Cell Syst.;citation_author=C. Xu;citation_author=J. Zhang;citation_title=Alternative polyadenylation of mammalian transcripts is generally deleterious, not adaptive;citation_pages=734-742.e4;citation_volume=6;citation_year=2018;citation_doi=10.1016/j.cels.2018.05.007" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=L. You;citation_author=J. Wu;citation_author=Y. Feng;citation_author=Y. Fu;citation_author=Y. Guo;citation_title=Apasdb: a database describing alternative poly (a) sites and selection of heterogeneous cleavage sites downstream of poly (a) signals;citation_volume=43;citation_year=2014;citation_pmid=25378337;citation_doi=10.1093/nar/gku1076" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Z. Zhao;citation_author=X. Wu;citation_author=P. K. R. Kumar;citation_author=M. Dong;citation_author=G. Ji;citation_title=Bioinformatics analysis of alternative polyadenylation in green alga chlamydomonas reinhardtii using transcriptome sequences from three different sequencing platforms. G3: Genes, Genomes;citation_pages=871-883;citation_volume=4;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=D. Zheng;citation_author=X. Liu;citation_author=B. Tian;citation_title=30 reads+, a sensitive and accurate method for 30 end sequencing of polyadenylated rna;citation_volume=22;citation_year=2016;citation_pmid=27512124;citation_doi=10.1261/rna.057075.116" />
<meta name="citation_reference" content="citation_journal_title=Front. Endocrinol.;citation_author=C. Ziegler;citation_author=M. Kretz;citation_title=The more the merrier—complexity in long non-coding rna loci;citation_pages=90;citation_volume=8;citation_year=2017;citation_doi=10.3389/fendo.2017.00090" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genome-Wide Profiling of Polyadenylation Events in Maize Using High-Throughput Transcriptomic Sequences" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8.cover-source.jpg" />
<meta name="twitter:description" content="Polyadenylation is an essential post-transcriptional modification of eukaryotic transcripts that plays critical role in transcript stability, localization, transport, and translational efficiency. About 70% genes in plants contain alternative polyadenylation (APA) sites. Despite availability of vast amount of sequencing data, to date, a comprehensive map of the polyadenylation events in maize is not available. Here, 9.48 billion RNA-Seq reads were analyzed to characterize 95,345 Poly(A) Clusters (PAC) in 23,705 (51%) maize genes. Of these, 76% were APA genes. However, most APA genes (55%) expressed a dominant PAC rather than favoring multiple PACs equally. The lincRNA genes with PACs were significantly longer in length than the genes without any PAC and about 48% genes had APA sites. Heterogeneity was observed in 52% of the PACs supporting the imprecise nature of the polyadenylation process. Genomic distribution revealed that the majority of the PACs (78%) were located in the genic regions. Unlike previous studies, large number of PACs were observed in the intergenic (n = 21,264), 5′-UTR (735), CDS (2,542), and the intronic regions (12,841). The CDS and introns with PACs were longer in length than without PACs, whereas intergenic PACs were more often associated with transcripts that lacked annotated 3′-UTRs. Nucleotide composition around PACs demonstrated AT-richness and the common upstream motif was AAUAAA, which is consistent with other plants. According to this study, only 2,830 genes still maintained the use of AAUAAA motif. This large-scale data provides useful insights about the gene expression regulation and could be utilized as evidence to validate the annotation of transcript ends." />
<meta name="og-title" property="og:title" content="Genome-Wide Profiling of Polyadenylation Events in Maize Using High-Throughput Transcriptomic Sequences" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/8/2749" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Polyadenylation is an essential post-transcriptional modification of eukaryotic transcripts that plays critical role in transcript stability, localization, transport, and translational efficiency. About 70% genes in plants contain alternative polyadenylation (APA) sites. Despite availability of vast amount of sequencing data, to date, a comprehensive map of the polyadenylation events in maize is not available. Here, 9.48 billion RNA-Seq reads were analyzed to characterize 95,345 Poly(A) Clusters (PAC) in 23,705 (51%) maize genes. Of these, 76% were APA genes. However, most APA genes (55%) expressed a dominant PAC rather than favoring multiple PACs equally. The lincRNA genes with PACs were significantly longer in length than the genes without any PAC and about 48% genes had APA sites. Heterogeneity was observed in 52% of the PACs supporting the imprecise nature of the polyadenylation process. Genomic distribution revealed that the majority of the PACs (78%) were located in the genic regions. Unlike previous studies, large number of PACs were observed in the intergenic (n = 21,264), 5′-UTR (735), CDS (2,542), and the intronic regions (12,841). The CDS and introns with PACs were longer in length than without PACs, whereas intergenic PACs were more often associated with transcripts that lacked annotated 3′-UTRs. Nucleotide composition around PACs demonstrated AT-richness and the common upstream motif was AAUAAA, which is consistent with other plants. According to this study, only 2,830 genes still maintained the use of AAUAAA motif. This large-scale data provides useful insights about the gene expression regulation and could be utilized as evidence to validate the annotation of transcript ends." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8.cover-source.jpg" />
    <title>Genome-Wide Profiling of Polyadenylation Events in Maize Using High-Throughput Transcriptomic Sequences | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"_3Aw6pbfvEflgvmkD9Mk5exbXMKC3D9MYSU4TABNBcs","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"493750","apath":"\/ggg\/9\/8\/2749.atom","pisa":"ggg;9\/8\/2749","ac":{"\/ggg\/9\/8\/2749.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/8\/2749.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/8\/2749"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/8\/2749"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/8\/2749":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022tTW6d\\\\\\\/Za3Ln7WCEcBAmzVwtozt7KeVusx9NlI9u1DUDG4eLzZNpKUCjQWDuSj\\\\\\\/7DkNTV3XSrszNgQ92DgZEGaMyGH1DvgsnJaH+x7Tl36cDkgIzshlHtMMGyabDHVRPjfz+JJKn4c9o+2XytDJZ\\\\\\\/nCq1rfmPTF4esgLBE\\\\\\\/fVhikM9ube0WA8ORecHQnrnwKFfkUomPiLWjp3V5zHfOW2dc9V1w0GkTykXhsp0L\\\\\\\/k1Key2m+c9IEqgAXnCoe\\\\\\\/+AEYGcTpqEm79IslrZI3wer0wnvbBfZt52Ah5vVw7mY6VTvpOqfHhVg2O5G5RmGFA6Tz0a8YAvxkEi5\\\\\\\/oiuXX4nJDDSlCtz3RQCvcBM0WIc3OvdZyMBWh0awrYQQXHxLd+729r+w5kAEVqyATj9s68Sjkcbmb2ZADLyfxCVxBApcYun45MNmvvyDVwGPlCowJQqZYXsGodSYhcBw6mG6\\\\\\\/6evTZ\\\\\\\/qzmWRhk7lDiw2aeE1W3Ky2l5q4iPg48gRHiE9jtyKhe8\\\\\\\/C74QKOAcKGWeysvH5BDt29qAQYG6nrG6us0mYoOHfjMcmT3uMaZh7IwTDlIOfh\\\\\\\/bBiooV8BiZZVm3q3Jo5q9qPjaEjFY2OyZ\\\\\\\/JUNB0bbw5pc422O6nxegJlVo7GE61XsnUcFOxtsI21oMOYJ6YD8o5\\\\\\\/H62o+V5O44Fm16v2GFT5ru\\\\\\\/74EzLLAKA26rgHUdi5z7jrFvieSAS8wmB3UhxD4Nkm4YLyV89nUHHAwERMQFN9P3Xdyn8k\\\\\\\/zw9fxTZSIFTytJ\\\\\\\/yMND8LspaDEkJZGPIbanen2TjAgU3jYi16qk7Yo7FEmIcAD+fZUfJhy\\\\\\\/QjBM9DDLfLioHVa24tUJ1i2e0SV31LO7HOSEpU\\\\\\\/a2+PLRlSUhZ7sGymtzrf6oOBqsgrGSxB+mw\\\\\\\/TevqWppzkhcnqWqpFRmoD278cUJ+FlHU4xVVpHHxKK+93qzWNYGWDjx9dgC6HlAtNShnlHUwhYGptU2Lc5VQnYFoboRChGkucsugksyprM9zs7HLqBzNwBrojVO2uJmW4shaSTjiE9Rb10X6HzlZ0VXFbSlFTfwwyBqQLTed0BYVVjZ1XF9MLbPYDJB7P0Wv5Pyp3M9P0Mx6ByTk\\\\\\\/FLMKa\\\\\\\/UCD2wWwDbPwe4Ssd510ECNrcMH6u1SrEyJEQUtVRlQ87F4NPoppofmgSni4TrIrPqLv8WTGemH6bOhDV8IKSvj+4S9A4nRHLMprK5NL2q6VgPQD8s2spzshGnTrXnUxzxqvOhCfaDUdtcrKrHOf5oXzXpAp\\\\\\\/CF\\\u0022,\\\u0022iv\\\u0022:\\\u0022yWLm3W+RmEZ27Z\\\\\\\/URtDVJA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00221c3743aa26a551640e7630756e71bff1b0927802525c33b0c8456cf3d57ad948\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022FmZx0AtLUHpMS040jGdfICEyZqQmtToAVumH9II6BkwLgw\\\\\\\/0Hbe7jpOzgk0hi1VBmEwXTnw+MP90v8HqIP2jO0d+UCQBQdbaTVotJ\\\\\\\/odWI2IvhchxRlAWRK5cC18aAoNRwWRKHCDcR4CwYRMCf0zHBekxM8rZjgmp2hRvTMFpF6xZ6AIeMp9NqRGMZssPA1wbmkrEnYViL6SJEoi85nHWZK0cLv7LUOhrDT3z+9Qm+53Q13T6yHIZL\\\\\\\/836dcS\\\\\\\/EvbJ55vIZcXHczkyf1RQToz6KyaO3KH455XVe5uw9jTZgnM+94CcdqBqy9yT5C3MnJ5GlvQZ5kww0kd2XJOy8kF9VAyB8pmZZr8LgOEUbI\\\\\\\/ZoX66CDC4k+Fl9hJFIQOHEDPycrqsqy\\\\\\\/uIHMQ0pbCfVm7nWgpeW4l3fhm2L5weTj6abHnOj8pwMhV5WJFYAgMmEnnhxlyMckcCpi8UuTyhbqwoY49UFwSadwaEvWCEs85MiBSz5EQWSF8TOEbwWRrYmKgbOvvzpPEtKU1YT4RG6RvTJ0hMeQsxYxQ3qrLC6aF6my9N4GcwtuAMXVM2WV+aBwR0JM4YZOK+dzmpP\\\\\\\/IPQAmgWcc5+doYV54lxZ3tJVMK4UxYBMtx4AClTieaLwfvrvPJ4tSNcSJJ+yg2tinf\\\\\\\/HnC8BL0t\\\\\\\/NsEEOjxKL6LkHZ6SMFm4MGWAQykuSztFI\\\\\\\/FUZSdFF1pLCPXgJspFkiq2tHuVwVjKZeLi9cRl2IsgmSxtFcbyVMxOwroy\\\\\\\/pGKjT3L8TSG+tDj1IK3sqHSP4xYLrSQXwhBKxQO9MJgnb5gI3K9LE1b+YSmfe\\\\\\\/UQXVYcfV1YtnHXbH+BNHxK4acnrHbr3uv3kmkpDWlI9ApAOuwlV0XdviuAffEw+jEI6FoXzNPqQeQTvmWHMztrwS7PQN0mndL5cZdyA1J3PomzgEsfXRLz+Q\\\\\\\/4kbwrT\\\\\\\/EZ6zxLMWS7s5IbkShFCBYMyPV74HERVYz68l+TjFJ0sfyP9mWh1EJjmpvieCR+VLIT0Sho\\\\\\\/2GanJzPdHcyAt\\\\\\\/wCXenKG1l2a+B34Z7TxjdeYvMQ67UUTzv44Ekpd4RhDfCQb+3eQK2CRJDSouPrcVcp4VtyBtoVv3jSSNfw+2rULKaK\\\\\\\/i04Qh93r+oWPwCFWmih5xpZjrU5i8CIZ3KIFVIXzC0RUwV9ZBKSXHmEwjFAhUoOyxLNTsnKiXwwVzbmkhRaSdj2syH3RXm96wuKRDChBecTWye5fFXL71nP9eM7pEY1IWRp3F3T\\\\\\\/MkYaz2ExTpoYwjBqgaTCJ6s25LynaJ5YzgeAfsPIVO+H8t2Z35SrZXSi5I2p\\\\\\\/FjryhZCqP\\\\\\\/9SOfubjk2zYhRbbJ0vTjyrneTYMMHIaeKe64VIm+3NqNVMpilaYKak4BEYa8mRYsZf11YzXUDPVzOKfxZXmx8IAnp2t248In1EDKN7wSIIb+kZNOyNuxOsvzaquY23W7sZ6JNJ+JvdRzWnemXCxUZU6FY9YMsVOsFqDAmJUCWxMwfM0URJ29fY7PMQ7XtwEpX8nE4nUGhevkd2R0T1Rr0rRZISdag1PqyZHoQR9lhco8gk7smOk6YuMoHxhZ9oG+V+gcTyTjRShPfFjWjEZuPC2MkytzJX9eqNcw+2ln+9NfnPk8DKoKiDo0JjePrdmHiC77eE\\\\\\\/q33ZMrAWC3J1S9xRfqpQKfsRsSEBpQVL1Hj407fvLXG9sl3XQAm8SllKvrS+GquNvOv1zJtfPIgBORfcQcguFZ5mXUrZvCemzhoR9M5HJBFdHXmbAwmVS9WF6cW+dMTk7nSUs8uHTdDOsRXLn\\\\\\\/m7zXL7jsNz1JMZLg0Dkg0i1fMqJchNhvBTz1VWGMIZWTZWBVjsxnyP5ftuaEdyZ8efA2yPQSKDq3ARzbacBP1dqymt\\\\\\\/\\\\\\\/HiDe9FnD3jQSMasYaQl3flwtmVUSzC1quGvNuEyYgmejFjtFiIXG6\\\\\\\/Zgj7TZUVxF4aTd0vs6kSAEJGik5iLhu8fa09JbClY1NgLHfcnUx1s\\\\\\\/M8YMt0obWTJg+ykcsHxNj5M6Zy1j4YCeyvKgInKGYScavYskv9+lyhXpHxim26wcllbYuh2WeSvpl7HXHv3aZT140TAg4E\\\\\\\/Jzq2AQxXGolcvZJ28+C\\\\\\\/k3jOxTJjLhTVsCa7IGVkw5eXnBXQ2dprdOi9Xsy8SK\\\\\\\/oD0RLZf6B0bX5vcpAzDkD71hIr\\\\\\\/0xp0nozNU7eymjAtpQcICAS19XyNOK6hoeKlFJ1nGhUN5YI1+KexCAPVzCcy0heAjv6gz5N9xpWn4aZGyBJp+oG+VVCwTDVA4xAYTvDuMhdMoj\\\\\\\/2uqz+uT+nxfG8MVYZS0MquEtGeZSIoRy4wzBpiaQfHeChnaLXUqGhqrpG\\\\\\\/NGS5FnJwHQnOYcploFHKnwbWnKuK7IDlOXOPxy3KG2ZB52VrN2nGuX98CgEzSBei2i2rWRNkFsV0G86rnxR3XDg4w6fMHHIK2bCCrPCJ0E0OszfyqGCSZWfzLNKyLSCwXy06noq2vQVyqgAx0QLXJU4mf4txqL6nhisexxxkT7g\\\\\\\/jLU5niRvj3MqiWCtUdvcj0qPQIcK8fGxwsM7sh7PbnpFfoZh5BjsIzP8BOw1xQwOnIPSrJA3YhLoVQnWeDbzelQt0Qycckln6jctmPstMZeW+Vp\\\\\\\/uSc1Myyz7xIE5aqYNQ8G\\\\\\\/84DTtb7eiPH2hHwnSj\\\\\\\/F2TuEAhK9OKUa2c\\\\\\\/ZEmv+URKpy5QKkuZVLoNCwSAfOf+Oj9zESF9MzlbixE+VrlZRNZRJjIhEHaI4Gqm\\\\\\\/Eo3KnVG36OBtEnk\\\\\\\/pwAlfi9WAfy+06ShMyfC9Y6ZYTrRwkAQ5d51hI9KjRMbIcyL3NSfO944rs+VkB3DGxhUe9UTm8NUS4rNlL2DdUSUcRHOGVLaMsbw40T+eVEAddGg4jUIaLtgrUduBfzHaCloBHmZPb5DMavvmN7LIVEzy\\\\\\\/G8JBztQoggJFiHa3j3qrGC5BA\\\\\\\/Cwo\\\\\\\/oQpBw\\\\\\\/VXzeQkXcScYooArSnM4+cBDGz48z7kkof9Idsw+5fvBOcCrdqHJEhKefq7NT+J4vHvXS4a3\\\\\\\/3DGVPTasutlbxidoBcCBKiLcM9sX61cv4+VjsuSaHeCN3mfMfhgOlqs62JLZs9aWYCOAMrz10DgBxQd\\\\\\\/ogO65wBtpJAMu\\\\\\\/PT5cGrivP+Ew\\\\\\\/Coo0+eySlm6J9JSOQzeBms3fmapuvdx+Lizs8R2ptzJ8PKCXw4YhgtPHECeDBzRjo7Ub5JMZvRjnRPLlLkj97PcpO3mMNq1eqlojgNb5s410lLCufKt+l9BF54IFoz7NmMh2bmjeLrjt75ueyAKnpZVeJzaEoVWl+TXvzP7uVUW2sDlfDL3tuKvkCt\\\\\\\/U\\\\\\\/uWHc93tLzE+iFbsZqjiaValhgDp\\\\\\\/MY3krfQrEdOwMEEYghBF2pUrOF7Ig7XAogxlbNVCQvRc9EEef9WlmEK9NYe53F3I\\\\\\\/S44ojVFiWEL\\\\\\\/Ebg1ic1vUde0Z8tcaDRL82OMI41auhbzHtBIEBDQKObG9bAM6UlypIF5PDtsGsmGZXWAZb3gWf5lTf4jBRUCQLJZCV0VSWFSc0XPZxebYnwdp2\\\\\\\/2IcdA9yaSCpvEGe7mN0P5s41VXb4CePELkHciaoP+2oEpmNE2UgBW6fcp4xDz9K3gs8vNch0Qu3pWOwdSNCIF8fDijDbU2sNqdH\\\\\\\/qsPSKnitCpX3x6wFgHmkWz8xfp58BTX5wLl5HzMgRZghq48XG4bINgVFm5Btj\\\\\\\/AOuUGrx9lJPjCM0CZ9GxheM7oZW9vVguCXOyudpM\\\\\\\/\\\\\\\/cIx3dEfFyV1xbR8owBafElm2d+7+xHuMZGYSAhkl++27JBmwvkLRELy9tT60QGQ8yVJO3xF+\\\\\\\/Bq\\\\\\\/PVai3XWI5iq55iB4EuDiBLyyMXav7haGC8SejPqNtQjBjt+++H8oXYbGstINLGEh+IRMRfyO1jcdL0or15jYd8NB+IFvszj1aHPfx+nuX25SbPv+fzRTaFkkHvSYDYyOYCfWU8RI4sYInjURiY2M4KFwRUuVMtQl0IB0C45m3L6CRKwT+YLZtLodPIESzdZFwKYUSzOgcAjlWl9\\\\\\\/vIZ3QW0PXbMFPAsKCQnodNE+KUVmpVH33+yzAHSp8IjuMKYdkLJ0Ojlu4LqrEo1XXwFWTWXSr0A8IVkupncxZUt\\\\\\\/VMWZqufaW+Q3MQkDeLaoKCBQ77BvR4YD17dFObjGdpemAIaT0imFjK8qhAJbu7w3czTt+B3H6Iqbdsbl\\\\\\\/TcMKEpwImEifhjQXbOtqyLldpAynEBE2ywDNAyyMHvo0rVtnJk\\\\\\\/Svz3jneJJC1fcOnjobvntPD2JoMUc3vocCW0R36617ct\\\\\\\/FlC3J01Bo2TuOpru7e6KyRwjn1BmkGWljuriVjR\\\\\\\/bfeA35DOzagjVGwaSnqXJztMBlGK5rdHdbPRZUZNXLbbWoB4H39MIZ2Qy9tsNt8+mOp9VHtUJAh0MZM6BmnGpSPGIJBe1mpAB1Ch7UvEA4KDnTBP6za2ewY7Zu5iokbcGJW21wUU6uLiGOh1eXC8FJq0essuGBjbIA9KoEh4oO9SFlmlrFOhal9VzZ7ZgvCgoOGgHXFq45GhpeFrP8d94C1NRpb37TC6bHNoUECYBDIyP7yMJIeQcEe44M4cdDU39gyOiDdwMHK5wyTXm0ixkYXBoTWNuAZueyRFYJTIxYEGZDZHVZ3dZzhRG8t2ZhgxPX7gPEJc0ZmwoeDWjOJfSZaQYNjZGZ3v0EET0zroqLZf2h9gbz6gguwqWdwoUeqQbmcuWpnGydFwHoua7Dorh0DcE1tNA169cJ8OE5khRlhOufFlVGgTcMrDVDeyV5D3Jv6CZiMFnVsULsewSFWL9cOJe6wcEsTC1lnx4\\\\\\\/rF7IZHDIWHbaSIJQCnpKf35H0l6bQ\\\\\\\/imlKhiHGNhkWx5QYTJf6yS6ZarrpwkiGxJl9ip48Ox4Qp9E46GlfSJBJ8Y4V+cEZyIG0BPffhYZfb37QzukLnmsxMEVgaDCtZXh\\\\\\\/ERMG0vAF5x7Vd8x5ynmkKMaW2wd0JycMCWZeDDisGXyMHMqWBrXFIc8lW1VeZB\\\\\\\/aGOZoAXwlir7PWY17Oy2Dvv0WK7e5XRSwTTVA0aZkhgb02rD9iurVAP0Pfe0PybQlgiE49cjenmuxo7G+ld7Govn8Rt+gmSfI\\\\\\\/lUTp1ovGEogCTFWpBxjlt8kUM00rwEzk\\\\\\\/sLF9s2qkwVHu8SX4TztbpeXijVHkaL2ifXvWaAvcOOxF7AJzBHcJD52Gmn7hzLfQygtXmLbUJC\\\\\\\/EdVsBVbvykUN\\\\\\\/u6plLT5uvobAUiGTIi4JhSifBAe6aXY28yD4KpzScV5XTDRPK7DJZkntJ+I2BRHscVAwd4lKTS21GqWAM6Pop4fPgxSTmFdJfAz6IsaIFHr9RE77pP+bRJoX5uQca2IClMnrWef1CVkjx6\\\\\\\/fUX4nK\\\\\\\/1JSDGYRoQaQXv1IBiD88JyxtHbTijYgYu9UClTNLNb1HqiQx7lmikJvSsMF92EyA0Kjs+yIRaM5qWWxW1vLGz34ZNSYqKqfdIiUt6FhFDE2I0gVVmOYo9xGmw3bkh2CTqVAoz7zo6z4jayOwln+32+hI6Hgy0EA6KE5V4MwXOpZbstNR6hQ1Xv1jHYF7OzNEcaED2a3Oi7wdXjFOL2SYCxbAyxv\\\\\\\/q8MDQ1O5X\\\\\\\/pcldy9vjG\\\\\\\/4+ZwcxHjcx\\\\\\\/HxzJb2ObL3kp9\\\\\\\/y4oNE386glyNZJg7TlyUjvn4qGVCcnedE9sxVhNIfBIDu5jpU2Fa1Pq5J91kJ\\\\\\\/XohdrNIdskDwC9lCN65JCbgxYl9fr6hwCz30MKVIqwyCb5cNRstjyZ8WSVoB\\\\\\\/H7HePD+DBFVYgIpy8Lko2\\\\\\\/gMzyiuiulh4K5H4gxgLX1ErOBdHLr+c01+8N0IC+4cJoqbKyB+5aQ9rX0NvJqASlcOEYNdUfhPegInDDN8mMQL2EP5UOFgVnlf+pnIOa1m8rhq5UntGUv1cZt4d5a9G3u8EILKhgb894+FSJiqNw+RCuUa6jE\\\\\\\/KJOqlwcMWiBa3UL7mI7orL\\\\\\\/s5shn26wPjptsXONZDvpf9oPeUIKFh10aQN+wK3a4s2lCVDwBB8qxC6w9ULLfl6TtZlaKvwfh2ASE\\\\\\\/RNyYEUSQMDOz\\\\\\\/MITiDijd9NTlyBlLLP1J+h\\\\\\\/lYgxhMaQrcUtdRBpmpkEnfE+d8fL788ov7hwZPmsajh4EmVElR67fQpAyIU1IUKQnbtrQL6CQUPqX4mhSyD\\\\\\\/WA\\\\\\\/n8IEu9fH0THL4Emzk92d2GWRH7lhltlExM4hwInwMtXjdO8h0jocIOq63BMMCucrj99jsnfJ5+6lgjmOjAPMKIwoLkUPlZ2zK1u613sA3IhDGV9LvrZNbi99jPKfBXUlLIK9IDPmYxzJlXpKCMzS33GGLUWjbzWAHGrtEeD6BM\\\\\\\/Rz7v1CU4TXcP8gX\\\\\\\/Ffb2pRpFAynqQtLcwBU7oGP4lFoyQeNZHk\\\\\\\/fRJ0Tt6THYBeyDeEMgvfX5\\\\\\\/L8EYGRmqdflDllC7M3Vr2s\\\\\\\/v8ccC9KgiEW1X7mqn7MgN24CW2w5HZ8T6KUioKRxGuQMfmnWgBD+UrnjqG0YGIF8jtKdXzqTKQWmu8gQxdiauWJQKvqy\\\\\\\/X26i3n4I2+Ks2ZLwrIvHw8FcFPeqFw\\\\\\\/KOoTnBOOfLyA4B+TdK6Vm3uAQ29kcX+MXpzBOYF6qodDcEDQvoEKe2cbfMUh0\\\\\\\/vz13zKQVirsa8wOjjw63yvgHYkzBTmT0belrdz54gYlE3FVsseFrTiq5bVBakQq\\\\\\\/qgetApPUHYjuKtN1tVXXD\\\\\\\/CHxaa3JQRKDEXLYVCJYUIEVwLmCOHwCin\\\\\\\/JAvuL0ujBD6jtveJ1qSCTcXkUSCQ0bD5kgifLssEyIptdEVyADC+dlqrI5\\\\\\\/Qci619eVta7fTZPKzWGbglCFe0RiRIABx5oW6rRSBMhP5xTl80p601LIhmVS9QDlCvPpeIpiiJBW8Mqcm53auIENxE\\\\\\\/4dbyGO+LDoof6Om8Sa\\\\\\\/zo1QG28FJOroanLbfaj+y2OYZxPAwMZekLjU3h8hfwJx0WNPnh99lEZVfvk2cPAXG15JVLCN9DMsTM+bHwC1GYHL3xHfe0cUZYS5tHmCpYF1FoQvfTsI7+BeN1N6VR44rrhMwt9l\\\\\\\/KjiN7j3ARGPnfqeJVtDh50BwlI2ikNMz5S3MIDIBo8dIbO4BEJU6jxXTVQFUnFzNOofd6BgskJb29kVsTaKA9DqZmCd3zZkMAo\\\\\\\/LULqHold62xqV7IIzhVkRdzCgRCZ8JW1YsYSKW8FF77jTsVvCia\\\\\\\/sjDuhCYYGnsqX2WY7vpT9n5IundWDj6+EyfgHEhAw5iPbYllIa\\\\\\\/Ozbbodo8wT9YlhMiigINoSpJ6SWAExq2ySCIWbZetaIrqX+xTiJtE\\\\\\\/hK1\\\\\\\/0d\\\\\\\/O1AKXsJCtLkuLkNgHnPr82mAkIMYtTPHhulkBzf8NvXZWMfxbtumt24bjay9+YVdXRcl\\\\\\\/AD1Wlj9rd7riOMF1uBxF5r5QmuSsT+OcYzbfr0Y1LOcTsspTIiZBF+ElxJNiB4Ay0GNXgN7\\\\\\\/l1t8GiuBco68hY6hf3qLvl0VUPiVT7EV0JU1OVNR6iX5kiPXOtDsCwAtTL9DRSVBDM4D9qqXAss5kNwpE3OK3W3Y19OXR\\\\\\\/nOGQYOA8soHM7bMNsYFi7WDM8UhoDoSDt9ID7Aj7Hs7OxQEQvRrqCgQR6oKKyg+ycVnd2rolUhEe+IckBPhkfr6UR08YnKF1gFkGjuXGV6o3AyM20b2o13bphbS10WdvcCg6+2tkZH9f2yfeIDndt1YbCzJuQYGQ7SRvrXMgWuyNPKZ5BBtjedfS4nnt9zvjDKit7\\\\\\\/nRW5rtvaJoZGAEPlGFw13DOdBY3Hp0UZ3TggxRpQ2YprjAwuTAHTSm4E4YtrCghOeJyTSDzJYgCySRgM8k092B+Ag81\\\\\\\/4ZTuH0SfpJBnTjIrV2bEVOhrqTvsJkrioX\\\\\\\/W\\\\\\\/1+eaj3yZHu4OnGWZLO50De+aK92Qvr8wcQVSIowI+2EC3HWeRFgSDiMlYBb7I4ySrLNkAgHhtmqanQa1bNDLFVNV\\\\\\\/Rlb6E0FHxOQdm5N5Yh3GSgr2Ndx6dJT7euMRJVm2tlx18BVBKDK50cNN4bSgMnOXcZAvEXFjKssmr3hvIljbkj6e\\\\\\\/YOy2J0KUgcQrFj7HAEy53WIVVtjQjtkTf1Yhez1Pweu28dZ6PmP1H9\\\\\\\/rMn21iycr+FP8fRQH+92wvgIxu+xZPVF5yJ3CoEayLyW4x+wLVGFvOCEqqTaDzijQO9xEDWpSp68O1aqlqeLKoKVgcURjpShcd6EsMVoBczMlOC04pvLpW7zQRByuR0RVIVpQ\\\\\\\/cibpyFtbhGEAk8+eJ31hh3KLceYI\\\\\\\/P\\\\\\\/XStV43Q\\\\\\\/363D6v7ZwNMaEOtS99q5ambOG9bJBQsy1RJ+wJl4UJhdnIM3VJatMpgTdYPkpVTRwQgvKRr1JhSRZiEieyoOrvs27UkXVCPjvSo94Amf7x1nf95dXeI43sEIz3IL2akLsTxTzTeaavpkTrGYLRfDf8R\\\\\\\/Y\\\\\\\/O96hre5MMkhQtl+5I+5zSLyA7HDfvYkeuWUsvxt1DeVIGgsZa9vTv06Go3T4bP6yHGk5aKIVZQpMsOIdKguKQSBELrRJcHah\\\\\\\/Unm0B3CgOjNfwmRlmSZePI4HE7KCURX9NivAcGO2QjDWYMQbsVOR+Mk2\\\\\\\/aN5IhP0iMLiTJayO4VzCov7AmOvm0d+8uIEsv\\\\\\\/Ex01OFzSbHSwtAQZv19KDBCiLO0zZ7dmJGLJFnzMgZ54MQZxMQbS6Hv2fMEJi0yZ4Qi7G+pC7jsROFhrzQrTKrNZfzyCo4thy1yO0IZxuziTYh+GN9BZrZxJCK5N\\\\\\\/4ietq+2ajsISZ9QMHjtB+6+yNmyWEq2C\\\\\\\/B+97LiS6f644DRHCapWZdrBc8gVj69riqxasz3nPt\\\\\\\/vAFp20wphmnJPsYoQhoQrBdPcs9B5JMTg\\\\\\\/1dgQ+vq9FvIALndl2EH+4be2DGTs4M1MAta5SNLtWqEi79+iAwqlw5Lgj9dRnP\\\\\\\/nuvOD\\\\\\\/ZzXg4eAghkEuCOmTG\\\\\\\/OV+ifJepIyMrVwJ\\\\\\\/sjRmX50PTyU\\\\\\\/3l3OjTZPVeLC73DrbQYu8QB9HHgebtCVA8TEXQ+0ME9Q8iNFXP9kpG2R0\\\\\\\/fGU4H8Ex3l7usOaR9xQ5iwPh+EiyqNfqmP3WBtseilbRnqJjL06MHODsFsNfXaWjsHc2G9Jjk9yZAnEqIX9MKSjsnMqIuL0U54tcaiOcRV18edoJ\\\\\\\/KuI2DonWP\\\\\\\/FWSLEx+MCTjCMsZKiJhaPkAvYPrDQRplzufkBof7KKaluyW2tfL36I5PRHOBLxqHo9qBtu6pfPRlhlN6HRZ3EThj2UO3yZzkWZekynSdPX3Otu2BkZPsFzJT80w+o\\\\\\\/5j+JnxkZYjp3INwsfabY\\\\\\\/zW9gKAaNyFexzQbuwoc21x2fFKz3xnU3ewVTkJUc93w+PfW0\\\\\\\/djiFWwJNPXaM9iwar6AbHh3+C96RIWRf82HS6V+BjTwEygeAaXM5H8o7un0syBGwvilxkoBU07qfPb52bgA4u8GIseaIZ1A0oJ8PFw9rDht6S4nLsvPc8dx2kxlv5PO5N1q5B1Di9v0ioiksRwIgSHgtv9EmJp2G+9KTwM629GEclbtyydDONu6mEBHb7QS8m6vfmt0dAkRfVGpMPfS5KqOm2Bm96wwSPlXv5T45p5rN4q6h7O3kBDDU26c9nGyyDWVfuIfLg9y++zV8lK8DnyDjf3NvMlzHBR\\\\\\\/yEGgQHLTblraWFwKrxDMzVBPnjv6PPX9XJt51XfbVTIpucbBcHQao8TdH\\\\\\\/bTVaO6AK1AfV\\\\\\\/DjvWjOZKLy2QrcAeF1\\\\\\\/0moLKG5jUBgQcmWTy6vNFgxIE7MlXz+IStGTNuxIM3ypIAf9kxQayc0gNAVzVdvJ0inWSZDAAGxs5aUYXZMyp9KX3ObUrinjK\\\\\\\/WmwoKorLqn1BRwZtF8ypnnkzB4d5rwb4SItsGsHjaxDM5PWbbcQq5CjERQm8\\\\\\\/hoYyjXv7eh4dzfB0oeEaG6JVmpU1f+f3+exZ2RgZ7y6cJNxHMQ31\\\\\\\/7y4Phl3pp4cU5SXRnNpVwRW\\\\\\\/w4utJC4mkaiSodoTqiPaG0XGATwH7\\\\\\\/gkwTCzNnZPgYJ1cpREw88K7+3Rq8839f5TuDvelqiQC4wX+qgQAlrWd91NOuwsPd8Nm147moPkAGcV+OMnOZC76\\\\\\\/HGT2zNz0q8LGFt4bWYQFhixoH0WrqMqEG1n7USWg92L+1MBl\\\\\\\/EqktIY2WncxrYyQYexhZTkFG\\\\\\\/s\\\\\\\/4CY2v5A88sgTMAELxJ1BMMg4+BibA44WtExl0OXu\\\\\\\/NzUzzF+55kZd0aoAhzpuu3mHAp5V8yXubqxkdEkeiqTSWEJ9OSEY4KfhOD77BlHYNKFj2R5do3cljY5liin+jplVLOdcp5apbB5lRxMnB3BtiNC1EccQYOf2DD6cBQUL04PCYppFeKqR\\\\\\\/beUWS4k+D8Zps8VoDDomOjdje1P6oafYMEsBgW5kBDyh2h0DUn6XYadkIbF2ZGANwezT6nA31uVBoksQjKyvA60KqOoAg89yo7VRiUR\\\\\\\/XD+sdJqRawQ+eDNVQxHIVZ+LMM1L5CvLNfDMEyAHMVIEroIvc7xiUrIVmXw7e1oRD1M+GIGJlyKahPA\\\\\\\/DNJR4wFQsjbhZAskBNM7cAxJSrZxGjddw\\\\\\\/xYIzNQqzb3hCdheQ3HBRL\\\\\\\/V+DxYGz42gyigfS5ZwJRQs22kiHkXdYn\\\\\\\/xsYkmhkxxHOBcCu2AkClObpLuSAU6EKuDE10O803iKBn0CZ\\\\\\\/it+gyxpVkc3cAPOFqVmhVFtDo3rPAIWuIUcSCQ\\\\\\\/7PLcZ6jlre\\\\\\\/DcKLXU0nfPwCb\\\\\\\/Te6+XFBw2PqBLiZXUDzc4jrodSvVpl9Oz1bfihCRb\\\\\\\/BcRtpkGGBkRh7VA0csHOH+phUTNyKuQm6gWX2TSnatnaFNajNxCzR4CWb+LqZAaYgt5uz4zBXxzGhmi1ae9qUAj4IetYJQj9IzXS966HKdCBjnMKTwdrSYFM1ngtuU1A+Dxr5eD+jaFDFd7rnNuiB9eQKbDdexP9T0LjWRHjOkR6J5yvJaz9KVNngf0+lUrDzw\\\\\\\/mNIv3bntrL5F8xzNdysYF9\\\\\\\/TgX3cDqASysKYQXs+g1CnNe0QHvu++FuTBlvhtlnrlK+jJO2SIKX76gMV0sf8cwWPRYu7y6\\\\\\\/hix+g\\\\\\\/nG80c3Js1bTD6P23frHTfGFCyQDegMzHKH7ZVqbh9FJ9je8zZg6wu7ltRTR431St1\\\\\\\/zViKWodEQuzufMuQKUIriY9t14mHVZePGfs2AUfuc9CNWuqyLG+AkbSMVgHYfxYJE9EoQgRAhPz5LuDhPxIes\\\\\\\/o\\\\\\\/dL88M+SlvjB53wUfycaa0bs1dtdBQ+Gr0RDbDyFnZ63KI4XMBvpOhCNSeuOl8TZ\\\\\\\/BvGonpNRvJyD8\\\\\\\/S\\\\\\\/oKw5MdXaOSPU4nISqL0AhCnqLNv50xuO7J7xOoAhLt7cFrN9S8U3YSSN+ESy2sK2N04\\\\\\\/byo1P5a1pic8Y9kxKARNQyZeaVI6iKOx\\\\\\\/51LJwFqcISSpT5hpxH1d41FmmqedyzF88fSFq\\\\\\\/GaW40hbyKPeKgF7MGE0kwNiIlnG3mKHUJXmshrlXyiaG8w7Ia41v\\\\\\\/3dDLvhJBxLqPxV42n7Rc6Pq\\\\\\\/KEUSxBSAUexNrYrShJkcLUkvv4pGM5TsGC7WJIxI9+UlPkiXa9nag4RTWxtgxkTZKUuWcVjihmLrq0Q0jfhsGZKrSbrfLvMdD3jpBDjsVGat2dYttcshz26sOytxZSwKOHTxGXwjz+EXDLMgSQ3\\\\\\\/Xg7Q5qnBKyOZP\\\\\\\/SsVB1n6b1AD6NTy9Z7T92kWZyljftLY48eNR+CyVImtDOnvKu7NasgZscQ1i4jal4JjZLrwqAk9+LztgUfcFhINZJ4hK\\\\\\\/ukli1VVdoNHdLMGdip\\\\\\\/V45ftsKYU9COtpLQyovJtyfhId7ec8udd1ZCCsSAi4NvyKMwpZVBod\\\\\\\/B\\\\\\\/CcNlY52ld\\\\\\\/m9g64QfGcz+\\\\\\\/24E+oeh6QcXyEJ9+ysLssU2WP3dOTr2hfA\\\\\\\/NEyJoEEIPXAT77+PUBRubztfoWoh6st9ZoyR\\\\\\\/eANRWiVU2CrcToa2j9\\\\\\\/d8ADc2EdPy6crb++ZsPzmK9YpSbOIYT9uO6xPG\\\\\\\/6Fg+RswdjW5lNsWnNqKFSuVR27RTwWRu47PMtqyPvrxxgL0bcluhPhAXOYy+viQj9LupNyDWZ3jMtFUBT4Op7TQGPJEc86nBMpYr1CtDu9Rlr1fhfKJKUnPXFwW30qeU4BswrJnA6qxj1Jhw+bZqq7T9vYUhBh\\\\\\\/CSFxGrZ72a6IdiG56kdz1i1jpXvyZqs2tBCrU28FJ7pmD887gIcYv9krzhatrYE5Wevh+V\\\\\\\/p+WVr\\\\\\\/hrrEK\\\\\\\/cssC4Mxk220Vig7xrtJDpGfUwgdO2aCKT2GpLNGhiRiKcSAUiFHW+mDKYXhlspnim1S7JQ\\\\\\\/Zm4W1NEoS0kTCXYDgYmwpsyd6\\\\\\\/vg3yjdxu\\\\\\\/khNE0ZqOEejMnw1Ch7Crng1BtYAjHMXpE8jXgYTHEStzu8hqV0BXWxbI\\\\\\\/RilZgFxGI3OaPuu+aQQXWVH3PAyvkEyD+Eu7lpUX3TG0b66v6WSYGqMPV2QLVLEYjOSLQAXV3f+DSTPRBqiy1tgFYF\\\\\\\/4KoLV+Nc5Pp+i0+4L3Y8JoJanJ7KC5gglsHlyoJDB4M88ke5FGZcyLk0nVLZmstGxj0Nx5CdgZIODK9KJZTHN4EHOn8DGQF9Y\\\\\\\/WMb7ngJwIz6GRGa\\\\\\\/6du4BTEilxgHWbSezysFN95uSyV2heKaVGJ0wko0nepNedQGRBBKeaZZg0C0lXkTU69WiEh25GgPPU2qlIGOmZ6X8MnlAArR00LHzHJEziO7mJ7t67dyZjiGKfLdmxvQef5kZ76241aHyALAuUMJYbDYAGuLYZ7i7hhjZgbo4jxI4ftGL1xX9kol7EuiqF90PDzZf\\\\\\\/h86DgxKX6p9S9uBLtCjv9u962xmMHNHUBMed7WbNnrMYTDU4YBfuayuqy6wlvO4dpzgyQQUw\\\\\\\/BOd56S0bm4+K\\\\\\\/3mK7BlB7nwnMpcoCBbObhdF5ahh0UAEqYfunkCJMt9f9HY+M9JVfls70wtvcvpcFsRwilU15GAQUNDb6D77psrIrdM5RIYkhQcQgERWa3koAEuIPvRUU8KB4dmI64\\\\\\\/pfjVz7ngNX0yM5JkwXDkJOsjryHTlhui2fmfsVkiyKFOzoposEHCANtsNokdFw63uvjvDxL6nzi29+hLb2QMBug8feEfDON2bX3ekyj6ZgLT5b3d+wLwq0p\\\\\\\/YNSBHjR4oTGMhn1n9SoGoMSfVQuWMLC9v988fN8im3AgyJ0KHiLZOiVJtt9XGe62fwM8QOHkUqU4Gd9JK0k\\\\\\\/4hlLhbNu+oeLPcIG\\\\\\\/grXkwB35\\\\\\\/ItTUkmISS+eOs3v37EYmXtua1fkt33ePH4iMcORbMea\\\\\\\/nOoXsdH36fiBn\\\\\\\/HxDmx7n5JLugP62GCMrt8nqdzHbog287Dhh1OGDuu58nOAuxoRKmojFfa30L84hgtPqz\\\\\\\/1dt8FSrrvqrfjvyKGdeBZbJjMcIQnK\\\\\\\/8AGCU4+7UOpw8T16VLPEPYLNRsrm+GSMl6KOVy+CGt3hC6\\\\\\\/neLUtOKRL\\\\\\\/1Uh1+t9hHimAP1Em43kFG+aTGfCfFEV2XLOAje6+WZMF3S1N3NxGumBFFsrDptgjNlbEg5iQhbERgy6FQDkl48uyzeDloEDAXNQPAje64i3HXdVi9\\\\\\\/B0sXaTsXND49uPmCiVVcFRJnCE8mPn9PQr7MiK3ech9+ikdfzjpN5Cgt4XC0S3\\\\\\\/RJ2bGa9XVyFPSk2h6U9Hyv4dZz8yo62jS1nTEP+B5NOmFvIyabYFh9+KYbFnwyJa8BXZVUaUX7OHH9bdeXHst6jXrrvo+yYRY0HVspaKhtelOzgbjJPSXoFbeilyyJOMk9WjtFTk9mY19B7Do4+NhJ9gvC6ondq94bhTrE9Z4xi\\\\\\\/JEjTQODBKnYv9vsiMcVeqOtydXa81mt6PH0ZXqQqKDph55O3yVY0vi56dHkqZucTZLisx5SiEPDYanhNjRrkeM96EeuC7Qzrg27w2ZsfMglMAXTQiHRNkBvEowY4kvMUz177om\\\\\\\/\\\\\\\/VjaZ8dHAsg571LI6UuODPWnf\\\\\\\/XuqbofDEo0Gp6CTkB7zZxLakubG0OxsZp067TULU+xxBkVCJmb7OTLG8GwI14DIcagQMqKzNUUXNT\\\\\\\/KwhsJ9QYFQFIXlNIlSM2MLCbHoDuxjG+fRo4j5rdWkDqsP2d8cal8PL7NUvR1IRAkTO75Ig7MN8lrNjMP3olcVIZpaQFlpYfDOKZOTCfszUtgPcvH64wrL\\\\\\\/uJ3TatHvquQseDAOWvLGh7h8SHUbs7\\\\\\\/zqXJ+DIpleA1ok43r83qliSLPSWk9LDo\\\\\\\/e\\\\\\\/FimDAbQ2LxW2+K0lGqWRS6Y2Q0xd5eXkFn9NqszYVIEupWccg8f00YTnNN51l3bWfYM9WFb9n3c8Sq6h23dfWAo\\\\\\\/Pn\\\\\\\/Zpt+CiyywdwblTJ7D0KN93hyhAdaZjQz9We991AisX+P9SDjPOWpK3Snoxf6YEa3k3WGne4LOlq7t10PBDvlSrsV89QX+9pPoWWnoyrsH9VZlvrrCOE\\\\\\\/hw2KePRS3b3vE4LrEtJzFdkFvvbJYigM5wYcHnvNKdsgcuufxzWyql2Z6FeGLLAFIJrUfoAFp+zbCiQGceOY4Gab4Y58ET2f9jTqRwFGKFjzxaNE+X\\\\\\\/xr\\\\\\\/Bud3ztQ8xPNb21zEQQoeMYxhTXeLohAtwuXdz6ErOclH\\\\\\\/ugWat\\\\\\\/JpC0ZuUOjOm4pz2gX+RfZFCKYUPWckcI5CD8lfkRzS4bRdRoKdHL6Zga\\\\\\\/S3S3ZO5+8WU2TJJkKxh1\\\\\\\/exR2ZOnb2sK0i06gHo1i5C1R5DxJnttf7X0P7mVpkBUWX\\\\\\\/j60xIvFNnwuLiTj0ftje6iGygA0A\\\\\\\/juemxtZ6l0ShuzFIBCi4Tk9qoyJUP6B84DLi82BNRcUZyN3GFLwkPfhidu0GC49+Cba02fkrLDWKHk7itY\\\\\\\/d0YDccnJc4At1YWnhfkc3+6L\\\\\\\/WnZhG4OWkh\\\\\\\/SX9tK66SvBfet8Fz8B3Z195kMizWI7dEcFLvo8yjBxABrJHJM4YzQDw74loiQS8+eNRKVRhKn7Onbbg56+QrJCGuyC+fxTrIegfEX3mfghhyDGbYVwmeseWH6RRFyyZbQf0nId9BY1HwgLq1\\\\\\\/fzBreK97NBsRNtDbN1nK\\\\\\\/aMXHcBWcuyT3ddBrasKSU\\\\\\\/ztKP2wvuGJ7Bdw8VqW54o9qZbyWeCCeYNbeC157wuKUqvESYZObYA8A4cQMFlIp7mnrqSupP718d5ZJIT5t9KpdO4Mhj+Lr\\\\\\\/bqt1Sp2swTN0IktOWiEPj8a4x\\\\\\\/KMn4IqZgous8QZKLyO7nzmTOEaeAcbZ1IZJitM7fcWiIlASqPTFo1uFr1hGq6a\\\\\\\/5KjMJuH1hef5t23tpjakJxia4O4IEgabuGBOdOEMHhNLqYRPvZnzE3ZH8czU55YBOD4KSOnZsKfiIFSXl5Qv10+5TnVVqrWb05lT\\\\\\\/xQkEMU+Mz2taIYvRrT2APCvVQeJUtzX6yUdlG13ottfOPfyjgViBMGsULdV49GtagHEjJEzHx\\\\\\\/e0tsA96kSYzGHdDQxtTjRKY18njnvlPc4+lLvE\\\\\\\/Z99KPkJhMprVe6CfBGU9Bnov\\\\\\\/mCOGMGik+a3g2etyX3Gr98FP+GmEtyNS4aaSiymUCzQaFLjtcA\\\\\\\/FsaZTDuF1OtNXVJf590T+nOeYMAgGk535\\\\\\\/cr6SI\\\\\\\/Dx1RUOzA4LSAytuvc9Bn37pxBVVluc3TL2x2Fs0OE9DJKiYU1gZ64aAsFGfMVyQb19Rp9Ee8khYl3wAZOHA9dLwlaOkT1N7zEwqTrcoqnU4CVeZSpCdWzOnwcNNNNI\\\\\\\/ICZkBCZE3iVbh+vFkzJu3qufhgT2o7rrpTfAWToyi6hL16WpBXgrg9+N1JDFDIxRytyU3VeF1Egf2kWyz4qlS+FQZ04\\\\\\\/NHyVaoyMW+eR\\\\\\\/hZbEL25PFOmdM4hYvBHIi2bn0gy4I3zvYQpjVWlMGMl6fpUcSi2TAfED7AUZlu\\\\\\\/H17YSBEgl47KqLZ5DCBasz+uxaaUbUQ84j35jNRqsyobyrx+x6Z4\\\\\\\/rDdpkRgCI\\\\\\\/lxhQMyHbON1uhNldQ5ZKy2y6ZePHfM\\\\\\\/\\\\\\\/qWVK+qB0R1376bkAqCkSPOt1FH5dzbVookKzU02R5WnLfxuBvOYhXjA\\\\\\\/4St18fa85\\\\\\\/x53mbYwEgmgw7KoNTwf5gIARilXy3FnsQ4uYKMuPYR6DVLT3af3p\\\\\\\/AdWNRWeu2VHfj4Xv6LMGdzeO+czjnbvftPXGYahouWguXuVvU9ZW16w0BK8dEkMJsWUolClxkFc47Vom27C4uBrKmSMIF\\\\\\\/5aIvHBMFUSzDDwlMCfURsKstfpivcnAiJDCQkvyDZIcqP4YL09Dfn8FCt+WhUtQnB5Uk+d4uaV6\\\\\\\/AanRqHRf+LKXLB9RfkrVkhLAyHHOhFPbnb9t6XnAbD0gq7r3k5pLPjQTEDbjca3TJ8a+t7P4JVwn6CinS6VPlQVH3TC2nR1qPDP7xrzO+Ay2po4Ssu31\\\\\\\/BIbGJ\\\\\\\/1UN\\\\\\\/O8zA4bWNzCv09BZmuKOVkxUJY3LBuFZloK5be9Rt420QxGpo5yXuEfm0OsAB5jN\\\\\\\/uio2r3C\\\\\\\/dU3nW4R0e5pIrxDWzZ2WwbL1VkOwyyzb1jPsW6Lo3Sh7Yuc78mhsv5Gmbc3NirB0x3Ap5DdR2rCjUoZSAB1ESwoFMpeCA4xdiXRoq1OgSg9dZREbCBsUuEKFFZMbnORZjeFEc6+6b6r6ERMYsn\\\\\\\/8rWQ++Dy+CGL+mbwjaTUxS4JVbzGbxJrzksVHqHMFPBSN1QY+JoXBH3rNkJ7pLwweHILLCt\\\\\\\/K5Mev3YJ1VlhsD4ylr+fxuZHCkP+kHTenT6KwIw2UOMC2p4VG3eHcGhleANoi4dnNZaqbZ0\\\\\\\/sohxqch4u\\\\\\\/YqY9XaGoQYLwBsRGwDz3qMfRGL9M1jdFdadJSE\\\\\\\/h+hdkEsiziSL3eqe518R48cTX45uplcAKJOHetYzN4veBQUeoPUaae1Y6UAbaXucmlFzhcihDuuSP1g3atiMkcow00Eimw\\\\\\\/K6lMkSRxd096z7Azmo4N0XnKbxU6Uq88pV361j2Pvjqrg9mm0de6w8NBfzQZZVyaGk4qDwLUnlyuEDTP4P29yN1Chxtp3EpIytYxtGyF3aKoW8Cudjqkl3kMPlre26V4K7KOm7sjx5zGy\\\\\\\/DI0taaGe5b2tNDxgICUsfnDgyrFQ7q2P06ZZyZKpsZTjzeJRDYkCqL3Po4GltDT+pVmZ8kItC1ZMdEO0h8oJX+125gHYPLbZlcn\\\\\\\/HYhmmguaKa1zqjBc2YYkq2jy62rubF6OW0qwx02ruI3LyTIeBmPNDZFpTGrx6vetkhq\\\\\\\/71YU+WjN9JVqHxOK\\\\\\\/eXAgGmZdEQr8wmAdNQTBaKywA3OXH0uoKmmtrRZeN2GPtNGycTweM5gHj5ON2r1y+dzSvegMb2gzC3WJWgXmFGaLKpJzdtdvU8GMtUvaQIhej3sV3mAR7XSgoBTIrnK\\\\\\\/cN6hjzDPr6OgBHjrkCfe37Qpg88WvfthkXkt6dZce2yFw5AdJHdJ+UGzPlNJzoYS4wbMWLuAQQqBNAgMRR10yOq9yj8TUXu2fDCfylG04XlNPdM0cglrq7MlBbfjVoL5K2g4a+XVbnuXegGq92ZTB1jsnzqVxKa1pz6fpDH\\\\\\\/c2G\\\\\\\/moNGQVXJytmx1pIvSePdM0pFNiIRKx7dwbL11ETkrED7PtdMfwuXh7mOFiQcCP2zoQHwAXiKmVuPyEIWv6OSGmNk2GcChgJvWwP8xQJTNPGLjbWvmguMgZOHNOPLxGVoYX\\\\\\\/8Nli5zazIePGsEIw6aDeYqeH8X1ahF5P5U+Rmnp9d\\\\\\\/kEN+Mlw7hwPq5BUDtHDsKUSctQYUI3G+2wwLJ+niuwUi0+jxXX9co4hPqRM90JY7bblh+mCQCJQQKnNGgXcoZRiHIO7apFO8qfcDiRAeHLbCBfkXREBEMpYVAVyHhpwHBEDHcmxJmkVFaOypuao4S8cYQiZ5vKAr38t6lOWShikXYesm5FKyzu3PBMnvkCIFXu1Ac33qux1750nWFc\\\\\\\/mqsp2fabzkgnnxshzc7gqIdktEUno8SMBU62db3wQzdS+tQewFEgwg8izTTkTbQ8IHgnAWka5SO06lSr1BN9QQtyMc1ajYMledD+qMT9\\\\\\\/Hz0uDRlV\\\\\\\/pRJh0rer2aCi7eafKbZfHZQ6Zcntq\\\\\\\/CxtZpN9XUs\\\\\\\/eET7y6IljEEZgQ4T8ng3PMx6P+9p7GRvKwIAdRj0XLS+78EAVSSi5OjAgmV19r5IwLwphzrgNmWbJrocWhMmLohWeQuTYalu90WG2sCW3Zyw92y\\\\\\\/sC6ZY3XmGvh9SaVSJZexIA3xJqUkfBO6flB04ZK5gCnXa3PD8fGtk29BjyuegWMx5PKdebz1It8SxaWekBvj3lBgSQchl24EojVpyGSF97Xr5PEi\\\\\\\/R3KySfz4Wk2GUejeXRWGR3zskqPP0fZi6bqufdmx\\\\\\\/zRgWEOdrMWJ07rHOPCMFhy0lWlETLopQpa0Pm2L2ItMjjJFpIf54hR0BmdkdrB8qXuyVOz4ajvh35F3O+a9boRCWExlLCMtIB7kvuAMvjS\\\\\\\/bUFtDPm5u1sMBGA\\\\\\\/Z3HMXZUU9Y+vaaOf6dGhDoHpgdlf8eZbm3SbqcBkjC1VWDYwJD9JF5oekzjshc+co+wRIqE6Wy68v2GFYpoHYC8fBsCnderHL11efIGuHR3LOqx9rAiaro99lYUAj8GNRGILswu6bWwaRWjuX\\\\\\\/EG8DJmBUINVGWff7fUIursBqn+34vK8LjojBcjuPZTpVEaB7d4hntGURsu+n91bkHA1IK4qI24BFn2LMwyaSvC6lPgtE2wNDj8i9d292+reVkO82mm7l3L8Cjw5i43IoWGk30hfKpf5Tvz9Vat1VlZHLBzyDqkGWiJMhpdIRRPHEtdStrnUXAnCY7LR+KPnNoC3aCEY1a6ZgKjKOTUMAG7SgjjaA7aT24WjHCmKgY1dTcvNa6YArhFW95vj9zGnlNf7T5ojiK3zP94fmHITNMAbfg5r4tR5Gc+xJ4Nr0C\\\\\\\/+18tVNaurnZso4cW+yKSmswrPtPjlhejnjRdSz51GlD1x3whUNEDnQl6RlQQ1XgqZOhK\\\\\\\/KTtACg0C8ozD\\\\\\\/RdmVrG85WFv34lEa42ZYE4apLNPvpkYTzoKKssmhMCYyhsv+Bm3JGaBQLqpDoWR9Fgu0q3lBg8pc+qS1EIi0z+H3CIf9q1W0mBaXR+k5K8essTgG5eOQ6CXqnI8rmY8l38g1PsKRULkD4dsWoEJmse5A8zGLnOsX3xNGl73y8qqBBML8ZjfPJzvwX+ilVyAJsxl+fXBbTrtAxTSNQhbjeiMrll8hI\\\\\\\/GEzgaEuITkDB0akAylzT\\\\\\\/B9gXKxRP5STqB0QAarfHgY0AF\\\\\\\/Wr3iPZxOf7k\\\\\\\/OjuxWr85OLFlgMtjWgkzLGYIj58T0josFa1XXEp3gTMUu\\\\\\\/wxUcZERt000cu2Y6YrcKFSoLM1+PQLoanmJj+eZVnuvGbzBF514BE3Ho7t+581FPqXZD3f\\\\\\\/4Sl0YotyVai4z1BepVnX+PM\\\\\\\/kjUG+P1FPXdwQAyDKIN1f+A2qOx\\\\\\\/HeTgj\\\\\\\/abQCnIhDFfs187dL\\\\\\\/gRfmnv44GwsSnu+PPYCSPPSGEq1oFR2Eb9ML8IVzHcww0FKPE1hOIjNoIjT1DgnaHNSIwIs\\\\\\\/mQ4pK0ioLHG2+TdxKIoKOTMl5RLmaMAIRPIpsW6zct51pYJ2BKcQY+dLDVVKr9uHTlQQutxISvF7+g5V9oMjh8Gr2ttcyAX+f7xDKDusgF1WnvlUx4LZOriEmC8GLFflniqdtcD8rI9Fk8EvgryqgOmnSD6fTSdEcUu3VtYhrn3fOVfaxqxjf1DJDoPu0HkcKdT9QfkL7o4j3OfrnL4ND9Zvs6D9UMV2hviS\\\\\\\/xXDRI5cTTJ1M0bJoyi70BHWxuEU9DP4wvv5B9JEOIN8mrW1TJsfrEcJ0ZJ9Lz8DDXmmp7DGc20cN2IzoEwFSt4JOTTlCW6L2IbrLuzgaDlMVU9jiy9ujQvWAyzGu6QFUPptOPuMs4K4Lek3765u0d8ZJ4VC5Qu3RC8K4+DcHjjWhJrP5Tmc3wNhZHChmrDstt278K9gM3iUkx+KP21JhUwKwKpXqYphpOPNn9b+FQIYb1rRbyYhtkPjUoivYKsRVpXN\\\\\\\/dyJMFwoMOIZogE8tKlF5Xb7gUWOIz3cuMAvxEPX4YkxQLFu46A0RjIEfkRAN+Uxx00c9447wxcu1yfQjN0UyX0sWY4DxqsCjv8uf0bSShAoDZA40Icm2k+G9BTw80x2AdFgMhZQQcOQYhijDgw4BebPCF59tqX0c5rak+GgOHB9bnuGi\\\\\\\/Vg9lkvwp+PbR3J2IY5lgFgYbgMDnxfN0FEkJXxvXH00ObURnOjtkukTGjT3nS\\\\\\\/eGfC8Nub\\\\\\\/sCZfRBlYVaNqCEZShJ\\\\\\\/FFik8Ar\\\\\\\/2BrV+lTVThD2RHtrRptvDSxn7Q3ZpU9\\\\\\\/b\\\\\\\/Dp8lgJNYVWYaFb78ndWyfWcIAkGqgDd8k7FPSQaMvXoW+giNzV3E\\\\\\\/RG+QXbjIyTB3+Px+44jvXlcQnBqA3fSL+dC\\\\\\\/Cp9o6vbw6v6GIoTghmp\\\\\\\/7ecn93Yog8ETnho8Fwd\\\\\\\/IEyyA5uEIA0favKOYjK2eo\\\\\\\/b\\\\\\\/pKM52A433UDII1UaJ\\\\\\\/iiNNe7xqqf7a200c0Z8IPjMYljiHHCLlXae\\\\\\\/JoOynHjfdNIDODNZP9xMQT9k2FnSIzU2\\\\\\\/pK6Yp9EItnp57KKXTJoxz7F6qN19MUjGnD8BCySepolb5+TpR0Z1Yf4ouYnoLCEqLQiM3Gol9NQ1UiEn9kAYK7of3odOvYaV9USD5myzz2xC1hKoHjmRR4NmsoeJstNF+itUpv+QOs9Q5PJGHQy+QJsv4cL1PcSol29AbFTtW5OI6v\\\\\\\/CMkmG41wKrhgKaBphevvPxKn3pW3HZ02AQwk+x76pRgEkpxzpbaTCAXHT1JQ1wGrVdpTeeGRmMIM+DNpI5xE9tQebmfY2XQqkfCRK0S\\\\\\\/HsK7d28EqiGyEdkYfmZtRYSP6BqsgapL48pAckOIVwedDYCR5cqJouWTU6mvwcnb3Q8UxUkgIyhd6jlhjBO\\\\\\\/FNBzZJ5ayz82B+daceeMXvsrvG2YlzAbPIhfnlflyIvUxAFSW0Ksfn0dLx33TlhaDJTYXdAgCuaiRTekTkNXoJkqAQKmXNg9qCI4wFYbd3wOIJcaRkQ65EcWSm2+IR1xGNyyv5tAmSNHsVnbDn1lY5ZwgY6xTIwlRtqOJeEUcc88ZnTq\\\\\\\/bNuDbZ96an\\\\\\\/1dc14rdKbsRo1dxvIu3rdgF3H05WXDuCruVRzriO\\\\\\\/vwPYilNw\\\\\\\/4Bica+gaEAnlBWPtC5nCXyOjkrrLqdJl\\\\\\\/WF++CnPWoRB1l5b3nnQmGLIi8qzON6HvpS\\\\\\\/xgt4jZa+5NwbLRIo7HHj\\\\\\\/RpkvWoEAQW3QX4y0KJURNF8k9Byffi1oHnykFHU0pd9fHF8QZn5Onxx1n7j6sq4OW9qGJcBxL78EMZxgAr5XR2CuvJoLKc2KGC35\\\\\\\/wQQoLG3imr0adIY9E6nrLgt9mtLhj5oGXTmyeX1ET9uctMElkWh4D5qguoIy0I0w8+aT5IApGHnyvJCx4Mlc\\\\\\\/H3K0d8TKzlHR2dv2BTwpcbgIcwcAMkghC98y2rJdShSPu6OKs5+OC+Ivc5De1FN7xxNbq52T7a\\\\\\\/IeSTB\\\\\\\/7xTxqPyU3OlM5wfwF7aTHzrMym9oEQ95ShWE7rWHrOCz51LnMB43kbG4Wycsoyc6e9ZYSXAyriaafMU286fD4p77SOdvTMsbzHx7AhAghtrKKq7ARhPBLcaEa0TZC8vb8nKKrbU3SX4mmuEIAn5vjzMmMkdHQNlzo9hcDL9kajq1dilcCpre7BocqvECN\\\\\\\/+gfxZlk5tw7ouzeMzTDOPjxnLloYbqp7stftqYwK0Lomb4reCKmsPbwANYdNa685Z43oFrVL+igq2nB8ftESr2d4nzLPKddEkOwPtRh5joYbxogPCVOR9UssoGvmm98LNi5Fapz4k+yOj5rh3dRJZqqZq7i0OUgqOi3WZW8RV2HbHq7y71wZ3gFWIqtczf9cskI0usamFLmyPdmirzIbvQdhQzKxbDh8zpctBjPu9GBF263hFt6kKIH8w8wpsTYxj\\\\\\\/Xqtorm0Za\\\\\\\/RnpOJ2TC\\\\\\\/uLlT1Wzz6IXfbiYJwKYzL0mGpJXB+ziCzPziIJcp29eUl4STKIMySPgpASv05n\\\\\\\/cBWI4Y+h1G9Xsk2Kzr\\\\\\\/M+aYT0Efm6GvXbQx0rq+w\\\\\\\/eHFNe2Dd4hJMGKdSJO1eSrf6ywLELNPpOgLRMf35uB0OBW1xl3yCVKQln2PYluOE6WlKENi00sX3fSTHW7IhKEuLWIiECLFw3fTpbMDx0e+41SMJXKUD\\\\\\\/aWK8kUJ6iErwbmbAUXFsBYqDcPuAWi+364K2ojn61LsvUh1kQoDYG9W4m5k5x4inIq5Ig+aUFHL9exEVjhDEHiuMSujloar7xF6hShlEGLv\\\\\\\/xV4FREBTy7IFcrJi0rGslVZouWW5i7YvG3eFm8eeVBmSLfGu5dAnu6XByH5wiUGd4YadbY7VXlfDg2h\\\\\\\/gpAZ+rFBqCj\\\\\\\/IZOoiUn55HCuEVSryT4jq4S32XzPsyWmKGAVpXgBKFEp1eVJXnK4gWEIF0ocWJGzJVhZrSwmLz7l7IOPkVAqF9\\\\\\\/TuVRdBaimSlxT+0YH1dqulTF6pwqOKrd7\\\\\\\/9Hd8ybl+ZhnLJyH89HS+2XSi02JoUrRA0F87eHHE+DxokjBY\\\\\\\/YKDlTMFjdiiIiF42Sja2jLLo4WSP+qsfKjF3v2br\\\\\\\/TE6h7kiRtQ71zaa9McFcnixfoVyKRzbD79+a5SqCc9i5ZEnXB0XiCv6APR2w+VVWQcUVXT5Iasw9u\\\\\\\/CdMQSE1tbP7SS4mwStdQvYEdesSE3Gs7G3ZPMMocjJYn+KtYe+AmP9f1EYXya7AEJIWyVmTEZ2Fh7Q0fUrIMp\\\\\\\/eq61w\\\\\\\/m0hlQgn+bF+Q5LjVQONYqhLstJ\\\\\\\/ts3xsNsH8Yl9A4BCpe+iOUHE0RBP2n1RYQQbi3fCZeP+mdEzo4fBHLos8pmGoNuTW8lq3zskIvYFDG78Q5oS\\\\\\\/MrjtUAb8twqps+UFwuFDr6\\\\\\\/u9NTUvcvPj2SHPAUusqpMVSlPw\\\\\\\/FVcKPCDistGQMXnEQtCdZ+UREZphJ98L5Vn2IW87VOGIyEgHHpLlTcRZEuPgQ61T66888sFw2JrMgDFEZYG58drs0fnZxJrSuzIBRd\\\\\\\/OQYnghO\\\\\\\/1J0KZTTm7pnGaURt5oHvbc8jfb63IIfmr+TDHPQarIim3rEDx2L4yc50bK1HdiKwQy3ihY7iijOTWR\\\\\\\/BKHndrHKXr8zBPgQPTWMVP3ndKYca8qy88WQawpZAYQALH2px\\\\\\\/vHARdYRT8knqGiSJ7CA5\\\\\\\/mjhzcFha3O229kPqGoOk\\\\\\\/bxGygfNvQUGR3VVt09OCItirs4xB\\\\\\\/1CfwwNb18F0GqhHN5PYASQYTqookBby+kXiwKU7OcLjb5SXP9IaK829KUoT5G5gc2bDjS6zEF5558UPtZ6NFbFpLiiTCIb7WHfG5VS+E9QEj0RRkxJif0S1hCL\\\\\\\/ATsE\\\\\\\/xRyNlYPCFoViiIri\\\\\\\/StWDlopUY8an0Vk3MRj\\\\\\\/YCnkMza4EgU1R6KZWmIGFY7a8yM8U+8MV5+fwjX\\\\\\\/ZjknbSRND0FeuqTnmYUAngbqpDgVO6brgyWKKDAJc0gXJ6Raw2yKvIw6sE1UfHn3MXvoAb2FZq7zhcEXS8BQ+66BHrJttAIqymYKUtKGpMjMdPSX\\\\\\\/sro01oPU+aQBrN\\\\\\\/5rCCMWMu5qJ\\\\\\\/7EtF8VW4zqCbe6m+SRragWEXryja8nUwbJ984XvGOrM8T7qTZo4MwWmV35lQr\\\\\\\/Q7Jmo\\\\\\\/dSgMFuMseSMDb67NNC\\\\\\\/ZJHke+ifYAWRwIIEvX0vRDPNxhIO3018Aq42APfsWO7PoKfG3veg2+BOtQ4kid3Mq8fkK7EQt30GCi240+T\\\\\\\/ix8RAkmFR9uGk3jJ\\\\\\\/JyFFH9MF9MvF1nU\\\\\\\/EbyyG3DprnJTApPsXKVqBbjaNMzRILYnzQ4IArdfQc+mnYWQuHxQeTHYI+VIarDajOzw5jC+fdD6wJhFMhPK6PzWWgb2irGpV4WXCUNVnPV2zEVCaBsKTZWoAh6Hygueq70cQ7oUNfLNSax0q0dpgrRE83dX2K9PzR004B+CTR9YMg9rr+BgG3bRH3BFSA2yFE5PJQHs4l+l9qnkMVj8ZHNoRAfFK1JGrxnOIpInsjj1xUoYPt\\\\\\\/\\\\\\\/dWvO2gD7MWVax\\\\\\\/fD5N86VJG05f1tJAIwh0agVsiLbL2DycjHWtVqIStpaaZwVndr1P9jWKKUfc4aeDC9Elvtbdm0230sw8WU3ZGGZbfvmamlg2oKUGuq83ZbyWvodIdgySSZX+hKdjni108gvhPOYCZ2gnPT6SZTW9mox0\\\\\\\/IlGTWfDB47j4H7xjCMtLOmPZDrEUTChBk+NtXMRv2Uxeujklv46opXryOEN8sscrV79GL2WvJbMfUXKOgkaQV4otao05iWZhYPDNegDbm8fAbB\\\\\\\/INRKC4TL6TGnwA6MzSptN39u3+S4lIGMJk0HJGDGRl3HDanD\\\\\\\/cOUTlDTT2ge2\\\\\\\/AHb4mj\\\\\\\/pyjPzC7JLqG21UiCx9eVDHTkkikT+SGvu0l444QRZkQrIez3zvSkIf7RW60tWUi6ptdg3JK12ZcJFTdnB0NoM5oVH8tCu8hBKlopspYK6Dpk09RrgYOk57BAWn2nXUvvfAbYdehK3V4W1rNVAy3SliCoyJ0\\\\\\\/5IKWJZbxlVlVsznl8dXGuDHnFrihqKeM9tCv4YOGmNr811KopfcRIgI7SeCHK9OxMBOJ9DDryKjBtIKBkjnuJwaTd6HHtYfkuX3ZFwfScTazFeeFMvzeYoHndcJKK1tzC21Gh7ZWo5sVyxcQh\\\\\\\/w6mPfrt14kAR76iQoMoRgLRKySuPtbdrFlJTmrKDSUS+IdEJvvKDJ2wXiyRiv9v4DZoSAzuBUnGB+h5Y+ys0WaMVA0QDzi\\\\\\\/A7nvPL5p4VFMOargM8qmbGqdz5TfUs22IwUeeDu821aV\\\\\\\/KUdnlaMNHz7Wrtt9BTRfMRRLjhf\\\\\\\/p4ZRq4oVGOZhRkYc+E5Mvp66EezP+blqp+FO8Pl73Emv\\\\\\\/eO6z2pX3iAB9ouV0yywD39+YzAXeubtpZDOPAE+o41fxjYVis+FBYTL\\\\\\\/pTUsOm8mYaouTf+j6UZaRWjHCqpc5l18d7xNYJ6QtqLMQ7JsvQsoYZXcDtesZg0lbDwI0QebnXn7lTdEGUq5hAttb4bMdmenox2++8YMRkTiMWwd798cH\\\\\\\/+7T8SM3RFC0+A6CjJODorTQF\\\\\\\/pRvfSHk2hNkf3hDCfwz6GIET1\\\\\\\/lDPHqEUUOXeWFu+ryO1+mESmbNQO87lajT5LFhM\\\\\\\/YfGWrC10rMVgXTdSezNlCgkLEvx8V1v1Pv\\\\\\\/stYSskW2rtQ+bRlGLcaL442nTokemBsmx1WwBnjrM5zMtAcqxQTXilMvKda+t2y8oVfYkeACf9SAIa4yxbosEICpsPQbUn3eRWvcEoGSxe7uZjrA032H6ykM5FQWexWF06nb+hnQib3emx1biPgV80a7tIHqE1zoB+x8wMZCgjF047kmyX3W6LvW7NwstFGUU+pSbEFF\\\\\\\/rQXehobV8tfH5GW\\\\\\\/zur0EJivGpr2U+x2xCU+EOwXWdUvowFUT\\\\\\\/3N\\\\\\\/J9xTHj3Epr\\\\\\\/FjrPpxBxzHO7xRz8hVhA2w02gHtF7ENna7rHSbznZZia0UTDRtDbCFVX9n9QomGa36SPv773ujCytDVQy3RSksIJXbOWc6jKaULUV6mtOGWl1BlJlyWthgr2mW0s1GssKuS5ZQl7kp6pbF9A\\\\\\\/P0sktmLDJ5rBSf\\\\\\\/e6MipdnjpI3qtvhj6cEBPuUYEREx6i\\\\\\\/KHfvu\\\\\\\/EfeegwncpfEPe05UnIRpQIx+rwblnYsbpTb1FrgOrUNdfU\\\\\\\/dtNV2Brc\\\\\\\/a+c9HEqpP+Gz4T8FY0BKKHUAsZJR64Nx13MNL4ZouMiXIUOYx91kJ+9yYwHwp\\\\\\\/DwpFixl196tTJGmtvOHfj+km0rw3duPh5nDI5YgaqQL0dUUG2QKBe2hfT8\\\\\\\/s+2Z\\\\\\\/CsfH6v+KCaiEC+q64m64Yvw1ATmazPGk+r5iNwmuZPrE3bj4zsrWa528xUvyvZ4oSPioqKJPn2shYE4U9H58pghszS2G5hBD+RwBJnBScU90sTfcqV8ET3oZuoPwcFzo1Hw6tJq+U3xGcUMAx57ubkUF7LUEpKewGL95y3yyUerZ+UnoKe7yGdcTMmg4d7Hsmg2Ge7aSr0\\\\\\\/63Z3a4ngQ7mjUOLi3ab+ehg4k+g58qVltuE4AqmFufnKSQaQAK96lRwmYXs2HCegFWFe9795RjCxz6VsJKJoGg\\\\\\\/cQh2zkZZC3tSf2zAMZD7FZjXGuN6W9FVFMZMffiOHm\\\\\\\/wMX2eTBeZvCb68RyEwWA5NEf8LdzCHHQijnW3FS\\\\\\\/iQQyS4hpv40lKwsoqJ0CRlWqxF8HP5Hk6Zlpm73KlRXysarTuBVYVWG+lAaLuXOZBJf2dtQ2HIb\\\\\\\/iYWC9K3pw6ZiT2Ll0VNDZsxi1Xje8qAwnXZwPBvz+T0HLX+F\\\\\\\/EHF2oPnHDvaF+V620dTHMZxn7fyxyY0Qvgp35oDRPvSRozzB7DkV4s\\\\\\\/BLEcK6woyx5Te6k6fS+Mo9RMlXE3T3jiVNRAg9SQYF9YyonnSsZbyN5IslNlsPj\\\\\\\/mh8i+ZB66UePZW7iWY3ng6QT+SVHwmFjkikH9tohuPHNeSE8IQ7c70+U+iLsvffgBH1tPDrVzVE6IHfAIfvt0MqNKWmv4sOQQbdqbeu0AC9VldRpB0dCFpxXcmBv9yTTUhT9NR0jrBZAs60jzhfZaIPLGIkIYykrLIEaHEMEDH3ySCNDB3jxGqwykyX9SABTkQlUgZAr8RkUHQfCmIMrb1VK\\\\\\\/i4S5BFjKn8Nknh\\\\\\\/83yTDONkk5RRvp9di1RfJtk81xP64DyMZXWarQb5LG6HUp6d3RlNSfRB0a+dRgwnS1YNRe7JSdXOxANgJEishPTGhZx4WCFHE\\\\\\\/6kx2mpUmaSfsBjiq27+vodqHRjxwZRXrrIKSjxsqwVtF1NGRffo2AUeoATrvlPTEoREuf7WPslBbXhNAIn\\\\\\\/Y9XWeiC\\\\\\\/TTJsQtQy+7IcjXKCLXY\\\\\\\/WseLd5JK1zinGqk8fsjj8KRHn2wvM7HOmg1yG6C2AqAdcWPcx+OZ5l9D5D1WhT\\\\\\\/6A3FPdfHbqdGWfTufvxqUw93et47i1Y1kanMBCXjKZL2+xxejCGQbWX4\\\\\\\/8ngY98lAXyOblAGRgTNTVO+cNgcrqPgr32LHLrQh1Ri8JPMUlvLKLXZCJY4vSlQxHWB29FHBAE2ut9vGaQvLPz51wvff8z7KeQPavaw\\\\\\\/Krlh+nTCYX0rA8F7V+cd1YfWHx1Rtc4Mtx2Fm1qYAwpvX49SOGuU5i6MreLRJ0OHV7VBgXRF7qo4n5Yg9MT+li\\\\\\\/I5Bg62xSibxflNeq5fd\\\\\\\/5Xmo5HsVnuC8Vok1u8wD3nUmkdx0kJRRf1uLx0yZhMcGkFzW2MIDLYDSZrOkQHFSq5l5isBsSsT5jIZT8cynGIsvj8rlhP1oAFYkJOCMZIbI6LZLUod4kkeOgqhBX5N6JFgyvU2ttZUaAgsbo8lvyR6DTpBetSg1yX90khutM9oar6xNdea6OohslednQ4qMWqxSe1odid2K7m+T5TNfJASD3r8XquHknIa4FBUY5J6I1cdhQrfVjBqmNjrk0RvlRdAwyBrFO0cnh1zOL4Qc4afuL8hpdAsrGtGRfIwwBxPGdxuy51Kz+xBmtr0OuAkZnmNdRIaKEr8ddi3SncyNKr6YHWpjqefjh1edw2CX0mNSCIFu1lU1j2u0\\\\\\\/H6XWDYVS4cX5Cuk3jxndcm+ws5k05eRccGI5MTKe25r3qVff0hjumALOPYgmK2BBEjgZbogR9K9HlE8fQoL8C\\\\\\\/4dZiM4JPKh1UVITWyl3TstekMQgzeDCZ\\\\\\\/fTe7+2HlgbQ9wI0FUsZovcCznWb2QFqnWT8vZ3zxikkPdihv+3467SNikIBoMmS\\\\\\\/SvZNtiSJ3FUqpcX\\\\\\\/oMUQJAwiug==\\\u0022,\\\u0022iv\\\u0022:\\\u0022fiii\\\\\\\/VgyPmwS9P0eF4BvgQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227db2005607640a906600e4c807ef54f9d8602a7f62e34fb4449fda249d3eaf8f\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022UohOflI5yQswIe7NRO95gELOgJ3Gbxtx8+jcmXc6On\\\\\\\/2UWVYQKlpmn7pwX0Q6p7VmZu9aw5EhRRz5eEBc3d8nWrJl5cMFovIG00u+7JGixc1vytxl3SJRCaRyPG1vrnsbmZe8XHnv4w\\\\\\\/HlayHc\\\\\\\/OJ52ijtdksYHLpk7zFr20e7rlCql3+rQVbMJCu30z\\\\\\\/Ng6amMs25cc7KDF1SIlQMZdMC0Zsa0JXqFAw1USoh7NyeF+N83kJmxkC+GY7ugXbX8XImM6IGEVZzz6ls6J8iHrRM+d6SJEqrGnlXDcJUmLp\\\\\\\/FSIEJk7fHSRT8rtVI3IfLgPEzY7Obt2S2LVeNZxV8v2S7gt0Lj6FLPqGqGyOIaDhdJwBlGH6Mjbs+iiOvNxmkHeXDRDwAIWvCaYLEVsBfA3VTnPihQLXQBnNq1a9ELX883hQG+waX3eQgAAjTNB9LWfVeZF0Hf2ymkNWzy\\\\\\\/xQa3WIkNhU0Qalia0TwrvSCniHX4vMDAKiVm3sPAv2W8TlIkXlbuq71CchNJnhoXZJr9hK2DrWrVbTffjg7fJCHBP87551zpkSXh9ykcDTrURzar9VrSdeUKijCai1ZhkxLOgA10y6JDCawMRAiEKrMpomHgFWDMgLQgHSYn5ZvlnOGU8POQYWJheNIah6r0Uyquw\\\\\\\/Ms9WhtWRhOLilV46EmCrUAeyff9I20ORX\\\\\\\/st8inxCgPla\\\\\\\/I7ua3hkZ8ysUWO9MY0wn1kBvJrjxCxC+IyxQjV0Hm3bt7OaYaFykAg+jVfDSEJ3\\\\\\\/9oNPja9MupuaSwgg9raSD+mxVK5qvYzVxFAz3w\\\\\\\/Tfg9iu2tzLjKBOTaI2+XAvfqRPZ46eZol0vyOJqUYsOvP9QAC9J\\\\\\\/heBcDxO9bHEXU9GD9ilqEVYufIXI3Hjjt990FhN99inL7DkVCqUjmilMtvBP0V5m4t9zSzRNIQ4qpZqTrYxLPuFU82Jxv5+nadiynpdCK9pgvHn+GgdBNzFWsT7IpY2Cx10AClw+imHCrqc84Lw8NUHs\\\\\\\/W9KKIA7CNFq3T3bGxqKYkU3iCJOJsDTR3DaeFf4prrNxa78S4DVFcktJhYy5cHtFChRpYFY5+l34o6qxNSJGxUmRZeJ0fzulV3GsP+WNIfZ6WTBSqoLW6cvlNa7p2bcaG6YePP1MaoPMGRSQ3FsGWrZztzSLfVAxk79DNzF\\\\\\\/keVWSwDm64T9bptB9VjnNFGdTLKkNtaq9ZBthAtoMmuCS4oL7qNhO7PvVHEeS5qi4r7+f10Z3\\\\\\\/FA4R8N\\\\\\\/dIj2yxuiI3H73CBpn39gbqklBb+C1euRRgx4qhylDnHLHfI2SDPEJxRmF8ROTBSS1CZzf8atcYlZe4fLLpoBjEoBgVgk6myIf1y4PpvKRolYEdM0IPpIx5SoDtSLMenplNWCH6B3GJQ1PyzGcH9FpId7Pa6176x9eyRNrNXrNwRbDkwj9k0wlbKq86ztrDBAG2D2rxcpKUKZXhw6DJOtCuSlmLSyhTucuR9xT4IhuxlrQZGtUEy7I\\\\\\\/ANg8Um\\\\\\\/0MDWvYyDuBGEDK8k8Qzz3WClk9ZQvk4uCuW8nL57XaP3Nuci\\\\\\\/Ngoiwhzx3CssI1xgDd6HbOM7j+hdtdJac9AR3OiWc2zrxzCU2Y4azSVcMJQPrmsfHsdDA1Z9A4DFZYjaRxr6YkDaV3Tj6L19+GYKy7ydZyx2QWJk\\\\\\\/2WmQYIREBj2VDWjudcQFXoRODmWmg6\\\\\\\/MyCErInB8l2dEgMCvB8vgJHhDkC0StxFTE1wkeCR5t6FE5GcWSBCOtS\\\\\\\/n5bjW2RKyyKmKfgRgthHCH1nYesH\\\\\\\/R\\\\\\\/xfUM8KnA0B+L\\\\\\\/kHS+UcbbpKh6rN4H1zIsVrTAOxHuA2CdRdwmvJpMSNySB04RhWKfXS4dJpibyxGhkH7nNKGFPuAiwbztgzqdh1z1z0V6Q0yj9x7WWY9v2br7jmyewVCVJKcdYj7dcEiTVqETMFoW7YnedcclBqOPmjboWZk1e9GffpImaUx8JABRcHiOowUbxExDoW18vRftEw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022DQnf6bML+VgRXr2EAoqg\\\\\\\/Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00222e1f28356b55f7869da23d0883bd139f9c60caabb6166f2bf94b0e0915f6373f\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-493750 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/8/2749" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-i3r9vY4BOhzTNaTYC4q3mwQxogaVRkTYfT_cJQBDuB0" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/8/2749" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-LbNJLPFIGBIf4eukbJw3YPHnQZFCMfKQxOMo2q9EjjA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/8/2739" title="Genomic Prediction of Additive and Non-additive Effects Using Genetic Markers and Pedigrees" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/8/2761" title="Signatures of Recent Positive Selection in Enhancers Across 41 Human Tissues" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493750" id="node493750" data-pisa="ggg;9/8/2749" data-pisa-master="ggg;g3.119.400196" data-apath="/ggg/9/8/2749.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Profiling of Polyadenylation Events in Maize Using High-Throughput Transcriptomic Sequences</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zehra Jafar</span>, <span class="highwire-citation-author" data-delta="1">Salma Tariq</span>, <span class="highwire-citation-author" data-delta="2">Irfan Sadiq</span>, <span class="highwire-citation-author" data-delta="3">Tayyab Nawaz</span> and <span class="highwire-citation-author" data-delta="4">Malik Nadeem Akhtar</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2749-2760;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400196 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493750" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zehra Jafar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Computational Biology and Bioinformatics Laboratory, Department of Biosciences, COMSATS University Islamabad, Islamabad 45550, Pakistan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zehra%2BJafar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jafar%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZehra%2BJafar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Salma Tariq </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Computational Biology and Bioinformatics Laboratory, Department of Biosciences, COMSATS University Islamabad, Islamabad 45550, Pakistan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Salma%2BTariq%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tariq%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASalma%2BTariq%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Irfan Sadiq </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Computational Biology and Bioinformatics Laboratory, Department of Biosciences, COMSATS University Islamabad, Islamabad 45550, Pakistan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Irfan%2BSadiq%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sadiq%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIrfan%2BSadiq%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Tayyab Nawaz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Mathematics, COMSATS University Islamabad, Islamabad 45550, Pakistan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tayyab%2BNawaz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nawaz%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATayyab%2BNawaz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Malik Nadeem Akhtar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Computational Biology and Bioinformatics Laboratory, Department of Biosciences, COMSATS University Islamabad, Islamabad 45550, Pakistan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Malik%2BNadeem%2BAkhtar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Akhtar%20MN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMalik%2BNadeem%2BAkhtar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nadeemakhtar@comsats.edu.pk" class="" data-icon-position="" data-hide-link-title="0">nadeemakhtar@comsats.edu.pk</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/8/2749" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:493750" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:493750/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/8/2749.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:493750" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:493750/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/8/2749.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:493750" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:493750/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/8/2749.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:493750" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:493750/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Polyadenylation is an essential post-transcriptional modification of eukaryotic transcripts that plays critical role in transcript stability, localization, transport, and translational efficiency. About 70% genes in plants contain alternative polyadenylation (APA) sites. Despite availability of vast amount of sequencing data, to date, a comprehensive map of the polyadenylation events in maize is not available. Here, 9.48 billion RNA-Seq reads were analyzed to characterize 95,345 Poly(A) Clusters (PAC) in 23,705 (51%) maize genes. Of these, 76% were APA genes. However, most APA genes (55%) expressed a dominant PAC rather than favoring multiple PACs equally. The lincRNA genes with PACs were significantly longer in length than the genes without any PAC and about 48% genes had APA sites. Heterogeneity was observed in 52% of the PACs supporting the imprecise nature of the polyadenylation process. Genomic distribution revealed that the majority of the PACs (78%) were located in the genic regions. Unlike previous studies, large number of PACs were observed in the intergenic (n = 21,264), 5′-UTR (735), CDS (2,542), and the intronic regions (12,841). The CDS and introns with PACs were longer in length than without PACs, whereas intergenic PACs were more often associated with transcripts that lacked annotated 3′-UTRs. Nucleotide composition around PACs demonstrated AT-richness and the common upstream motif was AAUAAA, which is consistent with other plants. According to this study, only 2,830 genes still maintained the use of AAUAAA motif. This large-scale data provides useful insights about the gene expression regulation and could be utilized as evidence to validate the annotation of transcript ends.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/zea-mays" class="hw-term hw-article-keyword hw-article-keyword-zea-mays" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Zea mays</em></a></li><li class="kwd"><a href="/keyword/alternative-polyadenylation-apa" class="hw-term hw-article-keyword hw-article-keyword-alternative-polyadenylation-apa" rel="nofollow">Alternative polyadenylation (APA)</a></li><li class="kwd"><a href="/keyword/lncrna" class="hw-term hw-article-keyword hw-article-keyword-lncrna" rel="nofollow">lncRNA</a></li><li class="kwd"><a href="/keyword/non-coding-rna" class="hw-term hw-article-keyword hw-article-keyword-non-coding-rna" rel="nofollow">non-coding RNA</a></li><li class="kwd"><a href="/keyword/3%E2%80%B2-utr-annotations" class="hw-term hw-article-keyword hw-article-keyword-3&#x2032;-utr-annotations" rel="nofollow">3′-UTR annotations</a></li></ul><p id="p-3">Polyadenylation is a crucial cellular process to generate mature 3′ ends of the nascent transcripts. Maturation of 3′ ends begins with the endonucleolytic cleavage of the nascent transcripts followed by the addition of several adenosine residues to the free 3′ end of the transcript (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Di Giammartino <em>et al.</em> 2011</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Tian and Manley 2017</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Chen <em>et al.</em> 2017</a>). The location on the transcript where cleavage occurs is known as poly(A) cleavage site and the newly added stretch of A-residues is known as the poly(A) tail. The poly(A) tail provides stability to the transcript, promotes the translational efficiency, plays role in the transport of transcript from nucleus to cytoplasm, and determines localization. Polyadenylation is a widespread modification that occurs in almost all-eukaryotic transcripts and many non-coding RNA transcripts that are encoded by RNA polymerase II (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Gruber <em>et al.</em> 2014</a>; <a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Tian and Manley 2017</a>; <a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Chen <em>et al.</em> 2017</a>).</p><p id="p-4">Most genes in eukaryotic species use more than one poly(A) sites, a phenomenon known as the alternative polyadenylation (APA). The configuration of APA sites could lead to transcripts with altered 3′-UTRs (when multiple sites occur in the same last exon) or altered final protein products (when sites occur in different exons or intronic regions) (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Wu <em>et al.</em> 2011</a>). Genome-wide polyadenylation studies in different species have revealed the widespread nature of APA events. At least two poly(A) sites have been detected in 70% of the mammalian genes (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Derti <em>et al.</em> 2012</a>) and approximately 50% of the genes in flies (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Smibert <em>et al.</em> 2012</a>), and about 80% of the genes in <em>Danio rerio</em> (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Ulitsky <em>et al.</em> 2012</a>). In <em>Chlamydomonas reinhardtii</em>, 68% of the genes use APA sites (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Zhao <em>et al.</em> 2014</a>). Similarly, in higher plants, 64% of <em>Medicago truncatula</em> genes and 70% of <em>Arabidopsis thaliana</em> genes have more than one poly(A) sites (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Wu <em>et al.</em> 2011</a>, <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">2014</a>). Studies based on poly(A) tag sequencing (PAT-seq), expressed sequenced tags (ESTs), and massively parallel signature sequencing (MPSS) place the number of <em>Oryza sativa</em> genes using APA sites between 48–82% (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Shen <em>et al.</em> 2008</a>, <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">2011</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Fu <em>et al.</em> 2016</a>).</p><p id="p-5">The choice of a particular poly(A) site is triggered by the interaction between <em>cis</em>-elements and polyadenylation machinery (also known as <em>trans</em>-elements). The <em>trans</em>-elements and <em>cis</em>-elements of plants are largely known (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Hunt <em>et al.</em> 2012</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Wang <em>et al.</em> 2018</a>). <em>Cis</em>-elements in plant are less conserved than the ones present in mammals. Near Upstream Element (NUE) is the most conserved <em>cis</em>-element usually positioned between 10 and 40 nucleotides (nts) upstream from the poly(A) sites (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Loke <em>et al.</em> 2005</a>). In terms of sequence similarity and relative strength, NUE is analogous to mammalian polyadenylation signal AAUAAA (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Tian and Graber 2012</a>). However, hexamer AAUAAA is known to be present in only 9–15% of the plant poly(A) sites (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Loke <em>et al.</em> 2005</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Li and Du 2014</a>). The Far Upstream Element (FUE) controls the efficiency of polyadenylation sites and is found between 30-100 nts upstream of the poly(A) site. Overall, the FUE elements are considered less conserved (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Rothnie 1996</a>). The composition of plant cleavage site revealed the preference of YA (CA or UA) dinucleotide that is located between two U-rich region known as cleavage elements (<a id="xref-ref-22-3" class="xref-bibr" href="#ref-22">Loke <em>et al.</em> 2005</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Li and Du 2014</a>).</p><p id="p-6">Maize (<em>Zea mays</em>; corn) is an important cereal crop of great agronomic value that has been considered as model organism for various genomic studies (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Strable and Scanlon 2009</a>). Recently, motif distribution and nucleotide composition has been analyzed in 10,491 maize poly(A) sites generated from the NCBI transcripts (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Li and Du 2014</a>). The nucleotide composition displayed a characteristic “U-A-U-A-U” rich pattern that is conserved across animals, plants, and microorganisms. The canonical motif AAUAAA is predominant in the NUE region of the poly(A) sites in maize (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Li and Du 2014</a>). Similarly, <em>Wang et al</em> observed that APA is common and the AAUAAA or its variants are predominant the motifs in the 3′-UTR region of the full-length transcripts assembled from the RNA-Seq reads in maize (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Wang <em>et al.</em> 2018</a>). However, both these studies were limited in several aspects. First, both studies only tested 3′-UTR poly(A) sites and provide no information about the genomic distribution of “non 3’-UTR” poly(A) sites. Second, none of the studies reported the extent of APA in the maize genome. Considering the importance of APA in gene expression regulation, understanding how the choice of APA is regulated is important. Third, no information has been available about the heterogeneity in the maize genome.</p><p id="p-7">Here, we present a comprehensive catalog of the genome-wide polyadenylation events in maize based on 9.48 billion RNA-Seq reads obtained from the NCBI SRA database. The reads were mapped to maize genome and 95,345 distinct polyadenylation events are identified. The analysis presents the annotation, genome wide distribution, and extent of heterogeneity, single nucleotide profiles, and NUE motif distribution of polyadenylation events in maize. Furthermore, the vast majority of the maize genes has more than one poly(A) sites and could contribute in overall transcriptome and proteome diversity.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Data collection and determination of polyadenylation sites</h3><p id="p-8">The RNA-Seq datasets used to identify polyadenylation events in the B73 variety of maize were acquired from the Sequencing Read Archive (SRA; <a href="https://www.ncbi.nlm.nih.gov/sra">https://www.ncbi.nlm.nih.gov/sra</a>) database. Details of these datasets (as available in SRA database) are provided in the File S1 and a brief description of each dataset is provided in the File S2. The RNA-Seq data were extracted from the SRA files in the FASTQ format using the <em>Fastq-dump</em> software of the <em>SRA Toolkit</em> (version 2.8.0) (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Leinonen <em>et al.</em> 2011</a>). For the paired end data, the additional “–<em>split-files</em>” parameter was used to extract the first and second reads separately. <em>Trimmomatic</em> software (version 0.36) was used to evaluate the read quality and read lengths (parameters: ILLUMINACLIP:AdapterFile:2:30:10 SLIDINGWINDOW:4:15 MINLEN:25) (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Bolger <em>et al.</em> 2014</a>).</p><p id="p-9">The reads containing eight or more A-residues at the 3′ end or T-residues at 5′ end (according to the strand specificity) were defined as the polyadenylated reads as described previously (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Wang <em>et al.</em> 2016b</a>). Briefly, the <em>FindTail</em> program (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Dong <em>et al.</em> 2015</a>) with “–endgap 2” parameter was used to select the polyadenylated reads. This script identifies the perfect poly(A) sequences at first and then it combines the sequences using an user-defined size of non-A bases to form a longer poly(A) tail. For the non-stranded data, reads with at least eight A-residues at the 3′ end or T-residues at the 5′ end were selected. For the strand-specific data, the first reads with at least eight T-residues at the 5′ end or second reads with at least eight A-residues at the 3′ end were selected. The N (Uncalled) residues in the polyadenylated tail were replaced randomly with A, T, C, and G using the <em>ReplaceN</em> script. The reads with up to 10% of non-A or non-T residues were finally selected as the polyadenylated reads to account for the possible imperfect homopolymeric tails (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Dong <em>et al.</em> 2015</a>).</p><p id="p-10">The poly(A) tails in the reads were added post-transcriptionally and should not map to the genome (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Zhao <em>et al.</em> 2014</a>). To ensure that the detected poly(A) tails were post-transcriptional, the reads from each dataset were mapped to the maize B73 (V4) reference genome (<a href="ftp://ftp.gramene.org/pub/gramene/release-55/fasta/zea_mays/dna">ftp://ftp.gramene.org/pub/gramene/release-55/fasta/zea_mays/dna</a>) using the <em>Bowtie2</em> software (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Langmead and Salzberg 2012</a>). The <em>Bowtie2</em> alignments were performed in the “end-to-end” mode with stringent settings (-D 15 -R 2 -L 22 -i S,1,1.15 -N 1). The “<em>–un</em>” parameter of the <em>Bowtie2</em> was used to retrieve alignment results of the reads that failed to map the genome in SAM format (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Li <em>et al.</em> 2009</a>).</p><p id="p-11">In the subsequent step, <em>TrimPolyA</em> program (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Dong <em>et al.</em> 2015</a>) (released on 20-12-2013) was used to trim terminal A- or T-residues from the extracted reads and the trimmed reads that were less than 20 nts in length were discarded. The remaining trimmed reads were mapped with the maize genome using <em>Bowtie2</em> in the “end-to-end” mode with stringent settings (-D 15 -R 2 -L 22 -i S,1,1.15 -N 1). Only uniquely mapped reads were retained. The downstream genomic regions of the mapped reads were checked with the <em>CleanSam.pl</em> script to filtered out the internally primed reads (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Dong <em>et al.</em> 2015</a>). The filtered reads from all datasets were pooled together to generate the unique genomic coordinates, which were termed as cleavage sites.</p><p id="p-12">As most of the cleavage sites exist in the form of clusters due to imprecise nature of the cleavage and polyadenylation process (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Pauws <em>et al.</em> 2001</a>), iterative clustering approach was employed to cluster adjacent cleavage sites that were observed within 24 nts of each other on the same chromosome and strand (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Tian <em>et al.</em> 2005</a>). Only high quality cleavage sites with ≥3 reads were clustered and rest were discarded. Within each poly(A) site cluster (PAC), the cleavage site having the highest number of mapped reads was considered as the representative cleavage site of the cluster and sum of the mapped reads from all individual cleavage sites was used as the support for the cluster.</p></div><div id="sec-3" class="subsection"><h3>Annotation of PACs and de novo transcript assembly</h3><p id="p-13">The respective genic locations of PACs were crucial to distinguish between different types of PACs identified in this study. The latest gene annotations for the B73 variety of the maize were downloaded from the Ensembl (<a href="ftp://ftp.ensemblgenomes.org/pub/release-39/plants/gff3/zea_mays">ftp://ftp.ensemblgenomes.org/pub/release-39/plants/gff3/zea_mays</a>). For the genes with multiple transcripts, only the longest transcript was used (<a id="xref-ref-51-3" class="xref-bibr" href="#ref-51">Zhao <em>et al.</em> 2014</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Wang <em>et al.</em> 2017</a>). These transcript annotations were used to identify the relative genomic locations of 3′-UTR, 5′-UTR, introns, exons, CDS, and intergenic regions in the maize genome. The intergenic region was defined as the un-annotated region between two genes on the same strand and chromosome. Previous polyadenylation studies (<a id="xref-ref-47-3" class="xref-bibr" href="#ref-47">Wu <em>et al.</em> 2011</a>; <a id="xref-ref-51-4" class="xref-bibr" href="#ref-51">Zhao <em>et al.</em> 2014</a>) in plants highlighted possible incompleteness of the 3′-UTR annotations which can lead to the inaccurate classification of genic PAC as intergenic PAC (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Wu <em>et al.</em> 2015</a>). The 3′-UTR regions were extended using the median of the intergenic PAC distribution to improve annotations (<a id="xref-ref-51-5" class="xref-bibr" href="#ref-51">Zhao <em>et al.</em> 2014</a>).</p><p id="p-14">To further validate the intergenic PACs, <em>de novo</em> transcripts were assembled in the maize genome using 68 strand-specific RNA-Seq samples obtained at different developmental stages from the multiple tissues (SRP029238). <em>Trimmomatic</em> software was used to remove the adapters and filter low quality reads (parameters: ILLUMINACLIP:AdapterFile:2:30:10 SLIDINGWINDOW:4:15 MINLEN:25). Only reads that were greater than 25 nts in length were retained further. The quality filtered reads were aligned onto the reference maize genome using HISAT2 (version 2.1.0; flags:–rna-strandness R -q). The HISAT2 alignments in SAM format were converted into BAM format followed by sorting using SAMtools (version 1.3.1). The sorted BAM files were used to identify transcripts from each RNA-Seq sample with StringTie (version 1.3.5; flag:–rf) using Ensembl transcripts as a reference guide. The transcriptome assemblies from all samples were merged together using StringTie “merged” option in a reference-guided manner to generate a master transcriptome (transcripts = 242,307 and genes = 73,965). Of these genes, 12,719 overlapped completely with Ensembl annotations, whereas remaining 61,246 genes either had all novel transcripts (41,095 genes) or at least one novel transcript (20,151 genes). Only genes with at least one novel transcript (n = 61,246 genes) were considered further to validate intergenic PACs. Multiple transcripts from the same gene were merged and the 5′-most and 3′-most transcript termini were used to define the gene start and end coordinates. The intergenic PACs that were located within the assembled <em>de novo</em> gene coordinates were considered as the validated PACs.</p></div><div id="sec-4" class="subsection"><h3>Analysis of PAC expression levels in APA genes</h3><p id="p-15">The genes with two or more PAC were considered as the APA genes. The relative expression level (RE) of the PACs within each APA gene was measured as the ratio of the number of unique poly(A) reads mapped to a given PAC <em>vs.</em> the total number of the mapped poly(A) reads on that gene. Using a gene-centric approach (<a id="xref-ref-51-6" class="xref-bibr" href="#ref-51">Zhao <em>et al.</em> 2014</a>), any PAC with an RE value greater than an empirically selected threshold of 0.7 was categorized as the “strong” PAC, while all other PACs in that gene were categorized as “weak” PACs. All the PACs in a gene with no “strong” PAC were considered as the “medium” PACs. Similarly, gene with strong PACs was categorized as “strong” gene, while the genes without a strong PAC were classified as the medium gene. The PACs from each gene were ranked in descending order based on their RE values. For PAC having the same rank from all genes, the total supporting reads were estimated and divided by the total reads mapped to all APA genes to calculate the fraction of mapped reads.</p></div><div id="sec-5" class="subsection"><h3>Analysis of sequences Around PACs</h3><p id="p-16">The [-300:+100] sequence around each PAC was extracted from the genome. Single nucleotide composition in the extracted sequences were analyzed using the <em>SignalSleuth2</em> software with parameters k = 1 and gap = 0 (<a id="xref-ref-51-7" class="xref-bibr" href="#ref-51">Zhao <em>et al.</em> 2014</a>). The deviations in the single nucleotide composition of PACs found in different types of genomic regions (CDS, intron, 5′-UTR, and intergenic) from the annotated 3′-UTR PACs in Ensembl genes was compared using the χ<sup>2</sup>-test (<a id="xref-ref-47-4" class="xref-bibr" href="#ref-47">Wu <em>et al.</em> 2011</a>). Furthermore, the 10-40 nts upstream region of the PACs was analyzed using the <em>SignalSleuth2</em> (parameters: k = 6 and once = T) to identify the potential over represented NUE motifs. Top five NUE motifs observed in PACs from each genomic region (3′-UTR, CDS etc) were used for the comparison among PACs from various genomic regions. The significance of the observed hexamers was measured as <em>Z-score</em> using the <em>RSAT</em> software (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Nguyen <em>et al.</em> 2018</a>).</p></div><div id="sec-6" class="subsection"><h3>Gene Ontology (GO) analysis</h3><p id="p-17">Domain-based InterProScan GO terms annotations for the maize protein coding genes were obtained from the maize-<em>GAMER</em> (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Wimalanathan <em>et al.</em> 2018</a>). Singular Enrichment Analysis (SEA) tool (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Du <em>et al.</em> 2010</a>) with “<em>Plant GO slim</em>” parameter in the AgriGO web server (<a href="http://bioinfo.cau.edu.cn/agriGO/analysis.php">http://bioinfo.cau.edu.cn/agriGO/analysis.php</a>) was used to find the association between GO terms and the specific gene sets. The significance of the GO enrichment was calculated using the <em>hypergeometric</em> test. The <em>Benjamini–Hochberg (FDR)</em> method was employed to adjust the <em>p-values</em> for the multiple hypothesis testing. GO terms with FDR <em>p-values</em> &lt; 0.05 were considered significant. The keyword counts on the biological processes were used to summarize the GO terms associated with specific gene sets (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Jeong and Nasir 2017</a>).</p></div><div id="sec-7" class="subsection"><h3>Comparison of PACs With known transcript ends</h3><p id="p-18">To compare PACs with annotated transcript termini, 131,496 known transcripts from 39,498 annotated protein-coding genes were obtained from the Ensembl (release-39). Of these, only 113,678 transcripts with an annotated 3′-UTR (regardless of 3′-UTR length) were selected. All non-protein coding genes were also excluded from further analysis due to lack of annotated 3′-UTRs. To simplify, only the 3′ most transcript terminus was taken from the genes with multiple transcripts. Furthermore, 136,745 PacBio transcripts were obtained from the previous study (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Wang <em>et al.</em> 2018</a>) and transcript termini that were within 24 nts of each other were iteratively clustered. Finally, 25,424 Ensembl transcript termini with 3′-UTR and 70,741 PacBio transcript termini were compared against the PACs generated in this study.</p></div><div id="sec-8" class="subsection"><h3>Data availability</h3><p id="p-19">Publicly available datasets were analyzed in this study that can be obtained from the SRA database (<a href="https://www.ncbi.nlm.nih.gov/sra">https://www.ncbi.nlm.nih.gov/sra</a>). The data generated and analyzed in this article is present within the article, figures, tables, and supplementary files. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8034695">https://doi.org/10.25387/g3.8034695</a>.</p></div></div><div class="section results" id="sec-9"><h2 class="">Results</h2><div id="sec-10" class="subsection"><h3>Identified cleavage sites in Maize using RNA-Seq data</h3><p id="p-20">To determine the poly(A) sites in the maize genome, a total of 401 samples from the 24 RNA-Seq datasets of the B73 maize variety were systematically retrieved from the SRA database (File S1 and S2). From the 9,485,003,995 total raw reads in the 24 datasets, 9,254,967,996 reads survived the low-quality and read length filtering criteria using the <em>Trimmomatic</em> software. The reads with eight or more A- or T-residues were considered as the polyadenylated reads and the rest were discarded. The 27,065,201 polyadenylated reads that remained unaligned to the maize genome in an initial alignment step using <em>Bowtie2</em> were retained further to ensure that the selected reads were post-transcriptional. Terminal A- or T-residues were removed from the unaligned reads and remapped to the maize genome using <em>Bowtie2</em>. Only 21,697,103 reads that mapped uniquely to the genome and were not internally primed were used to define cleavage sites in each sample. Summary of the total raw reads, number of filtered reads, total polyadenylated reads, uniquely mapped reads, and total number of cleavage sites obtained from each SRA dataset are listed in the Table S1. The cleavage sites from all 401 samples were pooled together to define 1,316,852 distinct cleavage sites in maize. Only 561,066 high quality cleavage sites that were supported by at least three mapped reads were retained for further analysis. All the cleavage sites that were within 24 nts of each other were iteratively clustered (referred as PAC from here onwards) to define 95,345 distinct PACs.</p><p id="p-21">To assess whether the same numbers of PACs could be identified using fewer numbers of polyadenylated reads. The polyadenylated reads were randomly down sampled to define the number of PACs, starting from 20 million and decreasing one million with each step. The number of PACs decreased with decrease in the number of sampled reads suggesting that all (21 million) polyadenylated reads were required to identify 95,345 polyadenylation events (Table S2).</p></div><div id="sec-11" class="subsection"><h3>Heterogeneity of cleavage sites</h3><p id="p-22">Heterogeneous nature of the 3′ end formation generates many cleavage sites that exist next to each other in the form of clusters (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Pauws <em>et al.</em> 2001</a>). A considerable heterogeneity was also observed in the maize cleavage sites and a large number of cleavage sites (83%) were within 24 nts of each other. The genomic distances between the adjacent cleavage sites were calculated and the relative frequency of the distances is shown in the <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>. For the PACs containing multiple cleavage sites, the distance between the 5′-most cleavage site and 3′-most cleavage site was calculated. The distances for the majority of PACs (n = 68,453, 72%) were within 24 nts, highlighting that the choice of 24 nts window was sufficient to reduce the heterogeneity among the cleavage sites.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2749/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Micro-heterogeneity of cleavage sites in the maize genome. (A) Distribution of the distances between the adjacent cleavage sites. (B) Pearson correlation between natural logs of the number of cleavage sites and the number of supporting reads per PAC (R = 0.88, p-value = 2.2 &#xD7; 10&#x2212;16)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1315534593" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Micro-heterogeneity of cleavage sites in the maize genome. (A) Distribution of the distances between the adjacent cleavage sites. (B) Pearson correlation between &lt;em&gt;natural logs&lt;/em&gt; of the number of cleavage sites and the number of supporting reads per PAC (&lt;em&gt;R&lt;/em&gt; = 0.88, &lt;em&gt;p-value&lt;/em&gt; = 2.2 &#xD7; 10&lt;sup&gt;&#x2212;16&lt;/sup&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2749/F1.medium.gif" width="440" height="226"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/8/2749/F1.medium.gif" width="440" height="226"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2749/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2749/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493883" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-23" class="first-child">Micro-heterogeneity of cleavage sites in the maize genome. (A) Distribution of the distances between the adjacent cleavage sites. (B) Pearson correlation between <em>natural logs</em> of the number of cleavage sites and the number of supporting reads per PAC (<em>R</em> = 0.88, <em>p-value</em> = 2.2 × 10<sup>−16</sup>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">A total of 49,382 (52%) PACs had more than one cleavage sites with an average of 5.9 (SD = 11.5) cleavage sites per PAC. The cleavage process is stochastic and the discovery of heterogeneity depends mainly on the number of supporting reads (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Tian <em>et al.</em> 2005</a>). To study whether the number of cleavage sites per PAC in maize also increased with increase in the number of supporting reads, the Pearson correlation coefficient was measured between the natural logs of the number of supporting reads for the representative site in PAC and the number of cleavage sites per PAC (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). The heterogeneous nature of the cleavage process was strongly correlated with the number of supporting reads (Pearson correlation = 0.88, <em>p-value</em> = 2.2 × 10<sup>−16</sup>).</p></div><div id="sec-12" class="subsection"><h3>Genomic distribution of the PACs</h3><p id="p-25">A total of 95,345 PACs were mapped to the Ensembl annotated transcripts in the maize genome (File S3). The maize genome consisted of a total 46,272 genes, of which 39,498 were protein-coding genes. The 46,272 genes expressed 138,270 transcripts, and of which 131,496 originated from the protein-coding genes. Only the longest transcript from each gene was selected to annotate the PACs if the genes had more than one annotated transcripts. About 40% of the PACs were distributed in the annotated 3′-UTR regions of the transcripts. The majority of the PACs (42.99%) were not located within any annotated gene (these PACs were termed as the intergenic PACs). Previous studies have shown that the majority of the intergenic PACs in plant species are located within few hundred nucleotides of the annotated genes (<a id="xref-ref-47-5" class="xref-bibr" href="#ref-47">Wu <em>et al.</em> 2011</a>; <a id="xref-ref-51-8" class="xref-bibr" href="#ref-51">Zhao <em>et al.</em> 2014</a>; <a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">Wu <em>et al.</em> 2015</a>), highlighting the potential incompleteness of the 3′-UTR annotations in plants. About 37% (14,583 out of 39,498) of the longest annotated protein-coding transcripts in the maize genome also lacked an annotated 3′-UTR indicating that maize annotation is far from being complete as reported previously (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Wang <em>et al.</em> 2016a</a>, <a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">2018</a>). To further investigate the possible incompleteness of the 3′-UTR annotations in the maize genome, the intergenic PACs that were within 1,000 nts of the nearby-annotated 3′-end of the genes were analyzed (n = 24,594) (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). Of these 24,594 intergenic PACs, approximately 50% of the PACs were situated within 150 nts of the annotated transcript termini. In maize, if indeed the intergenic PACs represent incomplete 3′-UTR annotations, intergenic PACs are expected to be more commonly associated with the genes that lacked annotated 3′-UTRs than the genes with annotated 3′-UTRs. To test this, PAC associated protein-coding genes were categorized into two groups. First group included 13,019 genes with an observed PAC in the 1,000 nts intergenic region, while the second group included 8,430 genes that had lacked a PAC in the 1,000 nts intergenic region. A total of 4,908 (38%) and 534 (6%) genes lacked an annotated 3′-UTR in group one and group two genes, respectively. This finding strongly suggested that the majority of the “intergenic” PACs could be probable real poly(A) sites that define the 3′ ends of the adjacent genes.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2749/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Assessment of genic and intergenic properties of the mapped polyadenylated reads. (A) The distribution of distances between the intergenic PACs and the 3&#x2032;-end of the closest annotated genes. The relative frequency of PACs that were within the 1,000 nts of the annotated gene end was calculated in 5-nt intervals and a running sum of these values was plotted. (B) Distribution of PACs in genic and intergenic regions of the maize genome. (C) Most Ensembl transcript termini were within the boundaries of poly(A) clusters as shown in main graph. A similar trend was observed in case of PacBio transcripts (inset graph). (D) Distribution of polyadenylated reads mapped to different genomic features. As expected, most reads were mapped to the annotated 3&#x2032;-UTR regions." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1315534593" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Assessment of genic and intergenic properties of the mapped polyadenylated reads. (A) The distribution of distances between the intergenic PACs and the 3&#x2032;-end of the closest annotated genes. The relative frequency of PACs that were within the 1,000 nts of the annotated gene end was calculated in 5-nt intervals and a running sum of these values was plotted. (B) Distribution of PACs in genic and intergenic regions of the maize genome. (C) Most Ensembl transcript termini were within the boundaries of poly(A) clusters as shown in main graph. A similar trend was observed in case of PacBio transcripts (inset graph). (D) Distribution of polyadenylated reads mapped to different genomic features. As expected, most reads were mapped to the annotated 3&#x2032;-UTR regions.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2749/F2.medium.gif" width="440" height="375"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/8/2749/F2.medium.gif" width="440" height="375"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2749/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2749/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493722" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-26" class="first-child">Assessment of genic and intergenic properties of the mapped polyadenylated reads. (A) The distribution of distances between the intergenic PACs and the 3′-end of the closest annotated genes. The relative frequency of PACs that were within the 1,000 nts of the annotated gene end was calculated in 5-nt intervals and a running sum of these values was plotted. (B) Distribution of PACs in genic and intergenic regions of the maize genome. (C) Most Ensembl transcript termini were within the boundaries of poly(A) clusters as shown in main graph. A similar trend was observed in case of PacBio transcripts (inset graph). (D) Distribution of polyadenylated reads mapped to different genomic features. As expected, most reads were mapped to the annotated 3′-UTR regions.</p><div class="sb-div caption-clear"></div></div></div><p id="p-27">To portray the maize annotation more precisely, the 3′-ends of the transcripts with annotated 3′-UTR were extended by 150 nts based on the median of the intergenic PAC distribution in the <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>. The 3′-ends of the transcripts that lacked annotated 3′-UTRs were extended by 460 nucleotides (where 310 was the median 3′-UTR length of all protein coding transcripts (n = 131,496) + 150 nts extension based on median of the intergenic distribution). By this definition, 20.69% (n = 19,731) of the PACs were classified as the “extended” 3′-UTR PACs that were associated with 12,361 genes (11,740 protein-coding genes). The mean distance between the extended PACs and the 3′-end of the closest upstream-annotated genes was 137 nts (SD = 119; median = 108). <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a> summarizes the distribution of the 95,345 PACs in different regions of the maize genome. A total of 77.70% PACs were located in the extended gene annotated regions, while remaining 22.30% were located in the intergenic region. The 60.79% of the total PACs were located within the extended 3′-UTR regions of the gene. The PACs in the introns, CDS and 5′-UTR regions were 13.47%, 2.67%, and 0.77%, respectively. All 95,345 detected along with updated assigned genomic regions are provided in File S3.</p><p id="p-28">To evaluate how well identified PACs agree with the known transcript termini, PACs were compared with 70,741 and 25,424 PacBio transcript termini and Ensembl transcript termini, respectively (see Methods for details). Majority of the Ensembl and PacBio transcript termini showed an excellent agreement with the PAC cluster boundaries (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>). About 50.26% (n = 12,779) and 48.72% (n = 34,463) of the Ensembl and PacBio transcript termini were within 24 nts of the poly(A) clusters boundaries. A total of 36,866 PACs in our data were within 24 nts of either PacBio or Ensembl transcript termini. Of these, 10,376 were supported by both transcript sources. As expected, 98.30% and 85.48% of the supported Ensembl and PacBio transcript termini were either 3′-UTR or extended 3′-UTR PACs (Table S3). Among non-3′-UTR regions, supported PacBio sites were also observed for about 6%, 0.4%, 0.4%, and 8% PACs in intronic, CDS, 5′-UTR, and intergenic regions, respectively.</p><p id="p-29">According to the Ensembl annotation, 22.30% (n = 21,264) PACs were located in the intergenic region. To assess whether these intergenic PACs overlap with the novel transcripts, the <em>de novo</em> transcripts were identified in the maize genome using 68 strand-specific RNA-Seq samples. About 47% (n = 9,975) intergenic PACs were validated by the <em>de novo</em> assembled genes (File S4).</p><p id="p-30">To examine the contribution of different genomic region in the total uniquely mapped polyadenylated reads that were used to define 95,345 PACs, the percentage of the polyadenylated reads mapped to each genomic region was calculated. Most of the polyadenylated reads were mapped to the annotated genes in the maize genome. Of the genomic features, majority (86.84%) of the polyadenylated reads overlapped with the known extended 3′-UTR regions, demonstrating the agreement between the identified PACs and the known 3′-UTR gene annotations (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2D</a>). About 9% polyadenylated reads mapped in the intergenic regions, while only 0.18–3.41% of the polyadenylated reads were mapped to the non 3′-UTR regions of the genes.</p></div><div id="sec-13" class="subsection"><h3>APA is widespread, however most APA genes express a predominant PAC</h3><p id="p-31"><a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a> summarizes the distribution of the number of PACs per gene in the maize <em>genome</em>. Overall, the 74,081 PACs were distributed in a total of 23,705 genes, which accounted for the 51% of the total annotated genes in the maize genome. A total of 23.71% (n = 5,620) of the gene contained a single PAC (termed as the constitutive genes). The two or more PACs were observed in 76.29% (n = 18,085) of the polyadenylated genes (termed as the APA genes), which corresponds to 68,461 (92%) of the total genic PACs. The APA genes with multiple PACs had an average of 3.79 (SD = 2.01 and median = 3) per gene. However, the quantification of multiple PACs within a gene based on RE indicated that in general most abundant PACs in the APA genes were expressed at much higher levels than the rest of the PACs, instead of expressing several PACs at a similar level (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>). The mean RE of the most abundant PAC (mean = 72%; median = 73%) was much higher than the mean RE for the second most abundant PAC (mean = 18%; median = 17%). The difference in the mean RE values of the most abundant and second most abundant PACs in the APA genes was statistically significant (wilcox.test <em>p-value</em> &lt; 2.2 × 10<sup>−16</sup>, one-tailed greater).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2749/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of alternative polyadenylation events. (A) Percentage of the number of PACs per gene. (B) Relative abundance of the PACs at each rank in the APA genes. For each APA gene, PACs were ranked based on the decreasing order of the relative abundances. In general, there exists one dominant PAC in APA genes. The difference between the first and second ranked PAC was statistically significant (wilcox.test p-value &lt; 2.2 &#xD7; 10&#x2212;16, one-tailed greater) (C) Percentage of the total polyadenylated reads of the APA genes that were mapped to each position in the ranking. Approximately, 90% reads were mapped to the most abundant PAC. The &#x201C;Strong&#x201D; PACs from the 65% of the APA genes contributed the 80% of the total polyadenylated reads. (D) Majority of the non-3&#x2032;UTR PACs either had a &#x201C;weak&#x201D; or &#x201C;medium&#x201D; expression levels." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1315534593" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Analysis of alternative polyadenylation events. (A) Percentage of the number of PACs per gene. (B) Relative abundance of the PACs at each rank in the APA genes. For each APA gene, PACs were ranked based on the decreasing order of the relative abundances. In general, there exists one dominant PAC in APA genes. The difference between the first and second ranked PAC was statistically significant (wilcox.test &lt;em&gt;p-value&lt;/em&gt; &lt; 2.2 &#xD7; 10&lt;sup&gt;&#x2212;16&lt;/sup&gt;, one-tailed greater) (C) Percentage of the total polyadenylated reads of the APA genes that were mapped to each position in the ranking. Approximately, 90% reads were mapped to the most abundant PAC. The &#x201C;Strong&#x201D; PACs from the 65% of the APA genes contributed the 80% of the total polyadenylated reads. (D) Majority of the non-3&#x2032;UTR PACs either had a &#x201C;weak&#x201D; or &#x201C;medium&#x201D; expression levels.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2749/F3.medium.gif" width="440" height="382"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/8/2749/F3.medium.gif" width="440" height="382"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2749/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2749/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493802" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-32" class="first-child">Analysis of alternative polyadenylation events. (A) Percentage of the number of PACs per gene. (B) Relative abundance of the PACs at each rank in the APA genes. For each APA gene, PACs were ranked based on the decreasing order of the relative abundances. In general, there exists one dominant PAC in APA genes. The difference between the first and second ranked PAC was statistically significant (wilcox.test <em>p-value</em> &lt; 2.2 × 10<sup>−16</sup>, one-tailed greater) (C) Percentage of the total polyadenylated reads of the APA genes that were mapped to each position in the ranking. Approximately, 90% reads were mapped to the most abundant PAC. The “Strong” PACs from the 65% of the APA genes contributed the 80% of the total polyadenylated reads. (D) Majority of the non-3′UTR PACs either had a “weak” or “medium” expression levels.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">In order to quantify the overall contribution of the most abundant PACs in the total polyadenylated reads mapped to the APA genes in maize genome, the percentage of reads mapped to the most abundant PACs out of total poly(A) reads was calculated (as described in the methods section). Approximately 90% of the poly(A) reads were mapped to the most abundant PACs of the APA genes, while of the remaining, 8% belonged to the second most abundant PACs (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3C</a>). To identify the potential predominant PACs within a given gene that were expressed at much higher levels than any other PACs in that gene, an empirical RE threshold of 0.70 was used. By this definition, a total of 13.48% (n = 9,985) PAC in 55.21% (n = 9,985) polyadenylated APA genes were considered as the strong PACs, while the remaining 32.82% (n = 24,313) PACs in these genes were termed as the weak PACs (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). Around 44.78% of the APA genes (n = 8,100) with 34,163 (46.12%) PACs that lacked a predominant PAC were classified as the medium genes. The genes with strong PACs contributed majority of the total polyadenylated reads (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a>). Most “non 3’-UTR” PACs were expressed at lower levels relative to 3′-UTR PACs that were either expressed at higher levels or were single PAC in a gene (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3D</a>). The complete list of enriched biological processes, molecular functions, and cellular components GO terms in APA, strong, and medium genes are provided in Table S4. The keyword count on the biological process gene ontology (GO) terms (Table S4) indicated that the APA genes were involved in the wide range of functions such as metabolic (n = 8 words) that includes both biosynthetic (4) and catabolic (1) processes, modification (2), regulation (2), localization (2), cell proliferation (1), generation of metabolites and energy (1), homeostasis (1), transport (1), response to stimulus (1), and signal transduction (1). Comparison of the enriched GO terms between the dominant and medium genes revealed that two sets of genes were involved in the different sets of functions. The keyword count on the biological processes for the strong genes revealed involvement in biosynthetic processes (n = 4 words), catabolic (1), energy and metabolites generation (1), translation (1), and signal transduction (1). The keyword count on the biological processes of GO terms for the medium genes indicated functional involvement in the regulation (n = 5 words), modification (2), transcription (1), development (1), and response to extracellular stimulus (1).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493830/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493830/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Numbers of PACs in different PAC categories</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Polyadenylation in non-coding RNA genes</h3><p id="p-34">More than 1,400 PACs were found in the non-protein coding genes (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>). All the PACs observed in the non-coding genes were either located in the extended 3′ ends (n = 788) or intronic regions of the genes (n = 646). Majority of the extended 3′ end PACs associated with RNA genes were within few base pairs of the annotated 3′ ends of the genes (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>). More than 50% of the extended PACs in all the RNA genes were located within 25 nts from the annotated 3′ end of the genes (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a>). Among different classes of RNA genes, 50% of the extended PACs in the tRNA, lincRNA, and snoRNA genes were within 15 nts, 60 nts, and 85 nts of the annotated 3′ ends of the genes, respectively.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493861/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493861/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Number of PACs in different non-coding RNA genes</span><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2749/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Analysis of polyadenylation events in non-coding RNA genes. (A) In addition to PACs in the intronic region of non-coding RNA genes, many PACs were located in the intergenic region within few bps of the annotated RNA genes. (B) Difference in lengths of LincRNA genes with and without PACs. LincRNA genes with PACs (PA) were significantly longer in length (wilcoxon.test p-value = 2.2 &#xD7; 10&#x2212;16) than those without annotated PACs (Control 1, Control 2, and Control 3)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1315534593" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Analysis of polyadenylation events in non-coding RNA genes. (A) In addition to PACs in the intronic region of non-coding RNA genes, many PACs were located in the intergenic region within few bps of the annotated RNA genes. (B) Difference in lengths of LincRNA genes with and without PACs. LincRNA genes with PACs (PA) were significantly longer in length (wilcoxon.test &lt;em&gt;p-value&lt;/em&gt; = 2.2 &#xD7; 10&lt;sup&gt;&#x2212;16&lt;/sup&gt;) than those without annotated PACs (Control 1, Control 2, and Control 3).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2749/F4.medium.gif" width="440" height="227"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/8/2749/F4.medium.gif" width="440" height="227"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2749/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2749/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493975" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-36" class="first-child">Analysis of polyadenylation events in non-coding RNA genes. (A) In addition to PACs in the intronic region of non-coding RNA genes, many PACs were located in the intergenic region within few bps of the annotated RNA genes. (B) Difference in lengths of LincRNA genes with and without PACs. LincRNA genes with PACs (PA) were significantly longer in length (wilcoxon.test <em>p-value</em> = 2.2 × 10<sup>−16</sup>) than those without annotated PACs (Control 1, Control 2, and Control 3).</p><div class="sb-div caption-clear"></div></div></div><p id="p-37">The highest numbers of PAC (n = 628) were observed in the lincRNA genes. These genes were analyzed further as lincRNA genes are known not to overlap with other genes (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Ulitsky and Bartel 2013</a>) and mapped PACs in this study can be assigned to these genes in an undisputed manner. Furthermore, only the intronic PACs in lincRNA genes were considered as the occurrence of PACs upstream of the 3′-most exons is much more common than the PACs in last exon of the lincRNA genes (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Ziegler and Kretz 2017</a>). A total of 470 intronic PACs were distributed in the 220 lincRNA genes of maize. Of which 115 were constitutive genes while remaining 105 were APA genes with more than one PACs. The APA genes on the average had 3.38 PACs per gene (SD = 1.97 and median = 3). To investigate the difference in gene lengths between lincRNA genes with and without PACs, gene lengths of all annotated lincRNA genes were extracted from the Ensembl gene annotations. As a negative control, three groups each of 220 genes were randomly selected (Control 1, Control 2, and Control 3) from the remaining 2,312 lincRNA genes in the maize genome with no observed intronic PAC. The mean lengths of the lincRNA genes with PAC and without PACs were 2,990 (median = 848) and 1,110 (median = 536), respectively (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>). The mean lengths of the polyadenylated lineRNA genes were significantly longer than the three pooled non-polyadenylated control datasets (wilcoxon.test <em>p-value</em> &lt; 2.2 × 10<sup>−16</sup>).</p></div><div id="sec-15" class="subsection"><h3>Intron and CDS with PACs were longer in length</h3><p id="p-38">A total of 2,542 PACs were detected in the CDS regions of the 1,809 protein-coding genes. To compare whether the CDS with PAC were significantly longer in length relative to the CDS without PACs (<a id="xref-ref-47-6" class="xref-bibr" href="#ref-47">Wu <em>et al.</em> 2011</a>; <a id="xref-ref-51-9" class="xref-bibr" href="#ref-51">Zhao <em>et al.</em> 2014</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Guo <em>et al.</em> 2016</a>), three random samples each of 2,542 CDS without PACs were selected as the control datasets (Control 1, Control 2, Control 3). The mean lengths of the CDS with PACs and without PACs were 787 (median = 474) and 207 (median = 126), respectively (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>; one-tailed greater wilcox.test <em>p-value</em> &lt; 2.2 × 10<sup>−16</sup>). Similarly, PACs in the introns showed preference for the longer introns. A total 12,195 intronic PACs were detected in a total of 6,193 genes. The mean length of the introns with PACs was compared against the three groups each of 6,193 randomly selected introns without any PAC (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). The mean lengths of the introns with PAC and introns without PACs were 2,057 (median = 973) and 654 (median = 142) respectively (one-tailed greater wilcoxon.test <em>p-value</em> &lt; 2.2 × 10<sup>−16</sup>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2749/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Difference in lengths of CDS and intronic regions with and without annotated PACs. CDS and introns with PACs (PA) were significantly longer in lengths than the lengths of those without annotated PACs (Control 1, Control 2, and Control 3). (A) CDS (one-tailed wilcoxon.test p-value = 2.2 &#xD7; 10&#x2212;16). (B) Introns (one-tailed wilcoxon.test p-value = 2.2 &#xD7; 10&#x2212;16)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1315534593" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Difference in lengths of CDS and intronic regions with and without annotated PACs. CDS and introns with PACs (PA) were significantly longer in lengths than the lengths of those without annotated PACs (Control 1, Control 2, and Control 3). (A) CDS (one-tailed wilcoxon.test &lt;em&gt;p-value&lt;/em&gt; = 2.2 &#xD7; 10&lt;sup&gt;&#x2212;16&lt;/sup&gt;). (B) Introns (one-tailed wilcoxon.test &lt;em&gt;p-value&lt;/em&gt; = 2.2 &#xD7; 10&lt;sup&gt;&#x2212;16&lt;/sup&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2749/F5.medium.gif" width="440" height="224"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/8/2749/F5.medium.gif" width="440" height="224"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2749/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2749/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493800" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-39" class="first-child">Difference in lengths of CDS and intronic regions with and without annotated PACs. CDS and introns with PACs (PA) were significantly longer in lengths than the lengths of those without annotated PACs (Control 1, Control 2, and Control 3). (A) CDS (one-tailed wilcoxon.test <em>p-value</em> = 2.2 × 10<sup>−16</sup>). (B) Introns (one-tailed wilcoxon.test <em>p-value</em> = 2.2 × 10<sup>−16</sup>).</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-16" class="subsection"><h3>Conserved “U-A-U-A-U” single nucleotide pattern Around PACs</h3><p id="p-40">To analyze the single nucleotide profiles, the sequences spanning -300 to +100 nt were extracted from the maize genome surrounding each PAC and profiles were generated using <em>SignalSleuth2</em> program. The analysis of longer upstream region (not shown) indicated a gradual transition from a relatively neutral nucleotide composition to a conserved “U-rich—A-rich—U-rich—A-site—U-rich” nucleotide composition. This conserved pattern coincides with the positions of the four groups of poly(A) signals found in plant kingdom including FUE, NUE, poly(A) site, and cleavage element before and after poly(A) site (<a id="xref-ref-22-4" class="xref-bibr" href="#ref-22">Loke <em>et al.</em> 2005</a>). <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a> depicts nucleotide composition in surrounding regions of different PAC types.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2749/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Average single nucleotide composition profiles computed in the surrounding -100 to + 50 base pairs around different PAC categories. The x-axis denotes position relative to the PAC location with negative numbers showing upstream region from the PAC and positive numbers showing downstream region from the PAC. The y-axis is the relative frequency of each base or chi-square values multiplied by 103. (A) Nucleotide composition of PACs from the 3&#x2032;-UTR region. (B) Nucleotide composition of PACs from the intronic region. (C) Nucleotide composition of PACs from the CDS region. (D) Nucleotide composition of PACs from the 5&#x2032;-UTR region. (E) Nucleotide composition of PACs from the intergenic region. (F) In a pair-wise manner position-by-position nucleotide composition from the 3&#x2032;-UTR PACs (&#x201C;expected value&#x201D;) was compared with the nucleotide composition from other regions (&#x201C;observed value&#x201D;) using the &#x3C7;2 test. The &#x3C7;2 values were multiplied with 103 and plotted on y-axis." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1315534593" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Average single nucleotide composition profiles computed in the surrounding -100 to + 50 base pairs around different PAC categories. The &lt;em&gt;x&lt;/em&gt;-axis denotes position relative to the PAC location with negative numbers showing upstream region from the PAC and positive numbers showing downstream region from the PAC. The &lt;em&gt;y&lt;/em&gt;-axis is the relative frequency of each base or chi-square values multiplied by 10&lt;sup&gt;3&lt;/sup&gt;. (A) Nucleotide composition of PACs from the 3&#x2032;-UTR region. (B) Nucleotide composition of PACs from the intronic region. (C) Nucleotide composition of PACs from the CDS region. (D) Nucleotide composition of PACs from the 5&#x2032;-UTR region. (E) Nucleotide composition of PACs from the intergenic region. (F) In a pair-wise manner position-by-position nucleotide composition from the 3&#x2032;-UTR PACs (&#x201C;expected value&#x201D;) was compared with the nucleotide composition from other regions (&#x201C;observed value&#x201D;) using the &#x3C7;&lt;sup&gt;2&lt;/sup&gt; test. The &#x3C7;&lt;sup&gt;2&lt;/sup&gt; values were multiplied with 10&lt;sup&gt;3&lt;/sup&gt; and plotted on &lt;em&gt;y&lt;/em&gt;-axis.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2749/F6.medium.gif" width="383" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/8/2749/F6.medium.gif" width="383" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2749/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2749/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493929" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-41" class="first-child">Average single nucleotide composition profiles computed in the surrounding -100 to + 50 base pairs around different PAC categories. The <em>x</em>-axis denotes position relative to the PAC location with negative numbers showing upstream region from the PAC and positive numbers showing downstream region from the PAC. The <em>y</em>-axis is the relative frequency of each base or chi-square values multiplied by 10<sup>3</sup>. (A) Nucleotide composition of PACs from the 3′-UTR region. (B) Nucleotide composition of PACs from the intronic region. (C) Nucleotide composition of PACs from the CDS region. (D) Nucleotide composition of PACs from the 5′-UTR region. (E) Nucleotide composition of PACs from the intergenic region. (F) In a pair-wise manner position-by-position nucleotide composition from the 3′-UTR PACs (“expected value”) was compared with the nucleotide composition from other regions (“observed value”) using the χ<sup>2</sup> test. The χ<sup>2</sup> values were multiplied with 10<sup>3</sup> and plotted on <em>y</em>-axis.</p><div class="sb-div caption-clear"></div></div></div><p id="p-42">The single nucleotide profiles investigated in the 3′-UTR PACs (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6A</a>) indicated that the FUE region (-100 to -30 nts) was U rich (32–34%) and preferred nucleotides in the order U &gt;&gt; A &gt; G &gt;&gt; C. The A content remained relatively stable (mean = 25%), while G and C content showed a gradual decrease from -100 to -30 positions. The NUE region (-28 to -13) had high A content with mean of 33% and peak at -19 (37%). The NUE region overall preferred nucleotides in the order of: A &gt; U &gt; G &gt;&gt; C, where U, G, and C content had an average of 30%, 21%, and 16%, respectively. The U-rich region -10 to -1 before poly(A) site coincided with the left cleavage element and showed the highest U content (mean = 43%) and peak at -7 (49%). This region preferred nucleotides in the order of: U &gt;&gt; G &gt; C &gt; A, where rise in U content could be attributed to the sudden drop in the previously stable or high A content (mean = 17%). At -1 position, overall the preferred nucleotide order was U &gt;&gt; C &gt;&gt; G &gt;&gt; A. The A content dropped to 6% at this position. At poly(A) site (0 position), A was the preferred residue followed by the U, C, and G, respectively. The U-rich region (+1 to +10) that coincides with the right cleavage element allowed nucleotides in the order of: U &gt;&gt; A &gt;&gt; G &gt; C. This region can be differentiated from the upper U-rich region (-10 to -1) in terms of lower U content (mean: 34% <em>vs.</em> 43%) and higher A content (mean: 28% <em>vs.</em> 17%). The C and G contents showed almost similar occurrences in both left and right cleavage element regions.</p><p id="p-43">More than 16,000 (16.91%) of the PACs were located in the annotated introns, CDS, or 5′-UTR regions of the genes. Nucleotide compositions around these “non-3’-UTR” and intergenic PACs are shown in <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figures 6B-E</a>. The single nucleotide compositions from these PACs were compared in position-by-position manner with single nucleotide composition of PACs from the 3′-UTR regions using the χ<sup>2</sup> test (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6F</a>). As expected, the intronic PACs were indistinguishable from the 3′-UTR PACs. The single nucleotide profiles of intergenic PACs were similar to profiles noted in the 3′-UTR PAC except at and around poly(A) site where U content was higher. The χ<sup>2</sup> values indicated that the PACs in the 3′-UTR and CDS were very different. The variation can be attributed to the overall decrease in the U content and rise in A+G+C content in the CDS PACs (<a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6A</a> and <a id="xref-fig-6-6" class="xref-fig" href="#F6">Figure 6C</a>). Moving toward the poly(A) site, the two PAC groups diverged gradually in the FUE region (-100 to -30) with lower U content in CDS (29%) than the 3′-UTR (33%), however a typical 3′-UTR like conserved pattern (U &gt; A &gt; G &gt; C) was still observed. The NUE region (-28 to -13) in CDS had a similar high A content (mean = 32%) with a peak further three bases upstream than the 3′-UTR at -22 position (36%). The region indicated lower U content and higher G+C content in the CDS PACs. Although, the cleavage element left (-10 to -1) in CDS preferred the same nucleotide order as 3′-UTR (U &gt;&gt; G &gt; C &gt; A), the overall U content was much lower than the 3′-UTR (34% <em>vs.</em> 43%). The cleavage element right (+1 to +10) of CDS PACs preferred A over T unlike 3′-UTR PACs. The preferred nucleotide order in the CDS PACs was A &gt;&gt; U &gt; G &gt; C. At -1 position, a clear preference for the C nucleotide was observed in CDS PACs. Similar to the 3′-UTR PACs, the poly(A) site (0) showed preference for the A residue, however among other nucleotides C was more preferred than U.</p></div><div id="sec-17" class="subsection"><h3>Comparison of NUE motif frequency in different PAC types</h3><p id="p-44">Previous studies indicated that the plant poly(A) sites are surrounded by various poly(A) signals (<a id="xref-ref-22-5" class="xref-bibr" href="#ref-22">Loke <em>et al.</em> 2005</a>). Among these, the NUE is considered as the most conserved poly(A) signal. To compare PACs from the different genomic regions, the poly(A) signals in the -40 to -10 NUE region of different genomic PAC were examined using the <em>SignalSleuth2</em> and <em>RSAT</em> programs. The overall frequency of occurrence and the significance (measured in terms of <em>Z-score</em>) of the observed hexamers were compared (Table S5). The frequency distributions of the hexamers revealed that the top five motifs were similar in the NUE region of 3′-UTR, introns, and intergenic region with few motifs changing their ranks. However, in the intergenic region CG rich motifs were more significant among the top motifs. Canonical AAUAAA was the most frequent and significant motif in the NUE region of 3′-UTR, introns, and intergenic PACs. However, it was observed in only 5% of the PACs in all three PAC types. In CDS PACs, the AAUAAA was not among the top frequent motifs and frequency was low (2.71%) compared with other regions. The NUE region in CDS indicated higher occurrence of G + A rich motifs relative to U + A rich motifs found in 3′-UTR, intronic, and intergenic PACs. The motif composition was in accordance with a drop in U content and rise in G content observed in the single nucleotide profiles of the CDS PACs. Only two motifs were detected in the 5′-UTR PACs and among these AAUAAA was not detected. The 5′-UTR motifs were different from those observed in the other regions, however, this could be attributed to the low number of PACs in 5′-UTR.</p><p id="p-45">About 3,016 PACs were associated with the canonical AAUAAA motif in the upstream NUE region. These PACs were distributed either in 3UTR (1,882) or extended 3′-UTR (1,134) regions of the 2,830 maize protein-coding genes (File S5). Out of 2,830 genes, the 454 genes were constitutive (<em>i.e.</em>, had single PAC), while rest of the genes were APA genes. To further examine which specific sets of genes that still maintain the use of AAUAAA as the poly(A) signal in maize, the enriched GO terms in genes containing AAUAAA were investigated. The analysis indicated that such genes are functionally involved in the metabolic biological processes including cellular amino acid and derivative metabolic process (GO:0006519, number of genes n = 86, FDR = 1.5 × 10<sup>−08</sup>), carbohydrate metabolic process (GO:0005975, n = 133, FDR = 5 × 10<sup>−6</sup>), catabolic process (GO:0009056, n = 98, FDR = 8.4 × 10<sup>−5</sup>), secondary metabolic process (GO:0019748, n = 16, FDR = 7.1 × 10<sup>−3</sup>), and generation of precursor metabolites and energy (GO:0006091, n = 37, FDR = 1.6 × 10<sup>−2</sup>). Furthermore, evaluation of the relationship between the occurrence of AAUAAA and expression level of PACs in APA genes (n = 2562 PACs) indicated that AAUAAA motifs were more strongly associated with PACs showing strong (32.40%, n = 830) or medium (45.32%, n = 1161) expression relative to the weak expression (22.29%, n = 571).</p></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-46">Availability of large numbers of transcriptomic sequences provides an unprecedented opportunity to explore genome-wide polyadenylation profiles. The standard RNA-seq reads are commonly used to define the genome wide polyadenylation profiles in species with no available or limited polyadenylation data (<a id="xref-ref-51-10" class="xref-bibr" href="#ref-51">Zhao <em>et al.</em> 2014</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Wang <em>et al.</em> 2016b</a>). Here, 9.48 billion RNA-Seq reads from the 24 high-throughput transcriptomic studies were analyzed to comprehensively characterize genome-wide polyadenylation profiles in the maize. The datasets used in this study consists of 401 pooled samples collected from different tissues under various physiological and developmental conditions (Table S1). Using a robust PAC identification process (<a id="xref-ref-5-5" class="xref-bibr" href="#ref-5">Dong <em>et al.</em> 2015</a>) with slight modifications, 21 million transcriptomic reads with eight or more terminal A- or T-residues were used to find the evidence for 95,345 polyadenylation events (PACs) in the maize genome.</p><p id="p-47">Heterogeneity in the polyadenylation events is a common phenomenon that indicates the imprecise nature of the cleavage process in general (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Tian <em>et al.</em> 2005</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">You <em>et al.</em> 2014</a>). In this phenomenon, cleavage occurs at multiple locations that are few bases apart (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">You <em>et al.</em> 2014</a>). This study demonstrated that 52% of the PACs in maize have more than one cleavage sites, which is similar to the proportion of heterogeneity observed in <em>human</em> (51%) and <em>mouse</em> (47%) (<a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Tian <em>et al.</em> 2005</a>). Similarly, heterogeneity was also noted in the previous studies on plants including <em>Arabidopsis</em> (<a id="xref-ref-47-7" class="xref-bibr" href="#ref-47">Wu <em>et al.</em> 2011</a>), <em>rice</em> (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Shen <em>et al.</em> 2008</a>), <em>Mediga truncatula</em> (<a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Wu <em>et al.</em> 2014</a>), <em>chlamadomoas reinhardtii</em> (<a id="xref-ref-51-11" class="xref-bibr" href="#ref-51">Zhao <em>et al.</em> 2014</a>). A strong positive correlation (0.88) observed between the numbers of supporting reads and the number of cleavage sites per PAC supports the stochastic nature of the cleavage process, as reported previously (<a id="xref-ref-35-5" class="xref-bibr" href="#ref-35">Tian <em>et al.</em> 2005</a>). The heterogeneous cleavage sites in maize may provide valuable insights about the mechanisms of cleavage sites selection and can be compared to heterogeneity datasets reported in other species (<a id="xref-ref-50-3" class="xref-bibr" href="#ref-50">You <em>et al.</em> 2014</a>).</p><p id="p-48">Ensembl gene annotations (version 39) with more than 46,000 (∼39,000 protein coding) genes were used to annotate PACs. However, about 37% of the protein coding transcripts lacked an annotated 3′-UTR making annotation of PACs more challenging. According to Ensembl annotations, majority (43%) of the identified PACs are distributed in the intergenic regions. In general we observed that the intergenic PACs close to the 3′-ends genes are more commonly associated with transcripts that lacked an annotated 3′-UTR (38% <em>vs.</em> 6%) indicating that the existing 3′-UTR annotations are probably incomplete or inaccurate. The occurrence of PACs in intergenic regions could be attributed to incomplete 3′-UTR (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lopez <em>et al.</em> 2006</a>), novel unannotated transcripts (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11"><em>Hanada et al.</em> 2007</a>) or orphan genes (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Schmitz and Bornberg-Bauer 2017</a>). Similar reports of incomplete 3′-UTR annotations were also reported in related plants (<a id="xref-ref-47-8" class="xref-bibr" href="#ref-47">Wu <em>et al.</em> 2011</a>; <a id="xref-ref-51-12" class="xref-bibr" href="#ref-51">Zhao <em>et al.</em> 2014</a>). After expanding the 3′-ends of transcripts, majority (78%) of the PACs are observed within the genic region, while remaining PACs are distributed in the intergenic PACs. Of these intergenic PACs 47% were present within the novel <em>de novo</em> transcripts. Similarly, intergenic PACs in Arabidopsis were also found to be associated with the <em>de novo</em> transcripts (<a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">Wu <em>et al.</em> 2015</a>). However, around 53% PACs in maize are still localized in the intergenic regions and the significance of these intergenic sites in identifying novel transcripts in maize needs further verification in the future studies. Consistent with the previous studies (<a id="xref-ref-47-9" class="xref-bibr" href="#ref-47">Wu <em>et al.</em> 2011</a>; <a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Ziegler and Kretz 2017</a>), most poly(A) sites are found in the 3′-UTR regions, whereas only small fraction of PACs are situated in the other non-3′-UTR regions. The introns and CDS with PACs are longer in length than without PACs and this is similar to results reported in mammals and plants (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Tian <em>et al.</em> 2007</a>; <a id="xref-ref-47-10" class="xref-bibr" href="#ref-47">Wu <em>et al.</em> 2011</a>; <a id="xref-ref-51-13" class="xref-bibr" href="#ref-51">Zhao <em>et al.</em> 2014</a>; <a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Guo <em>et al.</em> 2016</a>). The preference of PACs in longer introns has been linked with the interaction between splicing and polyadenylation (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Tian <em>et al.</em> 2007</a>). Also, such PACs are known to occur more often in terminal introns (<a id="xref-ref-46-3" class="xref-bibr" href="#ref-46">Wu <em>et al.</em> 2014</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Movassat <em>et al.</em> 2016</a>;). A comprehensive study is required to further uncover the significance and biasness of PACs toward the longer introns and CDS across various eukaryotic species.</p><p id="p-49">At least one PAC is reported for the 51% of the total maize genes annotated in Ensembl version 39. About 23% of these genes are constitutive and use single predominant PAC site. However, more than 76% these genes use multiple PACs indicating that the APA is a widespread mechanism in maize, which is similar to 50–70% APA events reported in other plants (<a id="xref-ref-30-3" class="xref-bibr" href="#ref-30">Shen <em>et al.</em> 2008</a>; <a id="xref-ref-47-11" class="xref-bibr" href="#ref-47">Wu <em>et al.</em> 2011</a>; <a id="xref-ref-51-14" class="xref-bibr" href="#ref-51">Zhao <em>et al.</em> 2014</a>; <a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Fu <em>et al.</em> 2016</a>). To understand how APA is regulated, we evaluated the relative expression of PACs within APA genes. In APA genes, not all PACs are expressed at equal levels (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3B</a>). Dominant PACs are expressed at much higher levels than rest of the PACs in the genes and contributed 90% of the polyadenylated reads. Most APA PACs are expressed at low levels and may be attributed to transcriptional noise. Similar trend of APA regulation was observed in mammals (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Derti <em>et al.</em> 2012</a>). Further investigation of APA strengths in a separate study indicated that most APA events except the most dominant are deleterious for the cell (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Xu and Zhang 2018</a>). Furthermore, most protein-coding genes that expressed multiple transcripts in humans also preferred a single dominant transcript and transcription start site rather than expressing multiple isoforms at equal levels (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Gonzàlez-Porta <em>et al.</em> 2013</a>). Using a gene-centric approach, we quantified the number of genes that expressed strong PAC relative to others. More than 55% of APA genes expressed a strong PAC and all other PACs in such genes are expressed at much lower levels. And, even still a significant portion of APA genes (44%) expressed more than one preferred PACs (termed as medium) in our analysis.</p><p id="p-50">Fewer PACs resides in the non-coding RNA genes. According to the current genome annotation, these PACs are situated in the extended 3′ ends or introns of these genes. In different classes of non-coding RNA genes, 50% of the extended PACs are within few bases of the annotated gene ends (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4A</a>). However, the exact nature of these 3′ end extensions remains unresolved as 3′ end extension criterion used in this study is mainly dominated by the PACs originating from the protein-coding genes (see results section). Furthermore, poly(A) tails are known to play important role in the degradation of defective tRNA, rRNA, and snRNA genes (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Li and Du 2013</a>). Considering these factors, only lincRNA genes with at least one annotated intronic PAC were evaluated further, as these are known to posses mRNA like features including poly(A) tail and these do not overlap with other genes (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Ulitsky and Bartel 2013</a>). Analysis of 470 intronic PACs in the lincRNA genes indicates that 48% of the 220 genes are APA genes, whereas rests are constitutive genes. Similarly, the APA events in the introns of human and mouse lncRNA genes were also reported previously (<a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Ziegler and Kretz 2017</a>). Previous studies have revealed that polyadenylated lncRNAs genes are longer in length than non-polyadenylated lncRNA genes (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Wang <em>et al.</em> 2014</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Liu <em>et al.</em> 2015</a>). Consistent with the previous studies, in maize the lincRNA genes with PACs are significantly longer in length than the lincRNA genes without any PACs (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4B</a>).</p><p id="p-51">Conservation of nucleotide composition surrounding the polyadenylation sites acquired from the NCBI mRNA sequences has been studied across animals, plants, and microorganisms (<a id="xref-ref-19-2" class="xref-bibr" href="#ref-19">Li and Du 2013</a>, <a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">2014</a>). This nucleotide composition pattern is generally correlated with the location of key <em>cis</em>-elements around PACs (<a id="xref-ref-22-6" class="xref-bibr" href="#ref-22">Loke <em>et al.</em> 2005</a>). Nucleotide composition in maize is similar to the nucleotide composition reported previously in maize and closely related plant species (<a id="xref-ref-22-7" class="xref-bibr" href="#ref-22">Loke <em>et al.</em> 2005</a>; <a id="xref-ref-20-6" class="xref-bibr" href="#ref-20">Li and Du 2014</a>; <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Fu <em>et al.</em> 2016</a>). The analysis of longer upstream region clearly indicated a gradual transition from a relatively neutral nucleotide composition to a conserved “U-rich—A-rich—U-rich—A-site—U-rich” nucleotide composition reported in plants (<a id="xref-ref-20-7" class="xref-bibr" href="#ref-20">Li and Du 2014</a>). This conserved pattern corresponds to “FUE—NUE—CE—Cleavage site—CE” order of <em>cis</em>-element (<a id="xref-ref-22-8" class="xref-bibr" href="#ref-22">Loke <em>et al.</em> 2005</a>). The nucleotide composition of 3′-UTR was in agreement with nucleotide composition of PACs in the intronic, intergenic, and 5′-UTR regions. Single nucleotide profiles around these PACs are overall A + T rich. However, CDS profiles are more G-rich and different from the nucleotide composition noted in other PAC types. These nucleotide composition variations are similar to results reported in Arabidopsis and Green Algae (<a id="xref-ref-47-12" class="xref-bibr" href="#ref-47">Wu <em>et al.</em> 2011</a>; <a id="xref-ref-51-15" class="xref-bibr" href="#ref-51">Zhao <em>et al.</em> 2014</a>). At cleavage site, a peak of A residues is observed which is consistent with reports in plants (<a id="xref-ref-19-3" class="xref-bibr" href="#ref-19">Li and Du 2013</a>, <a id="xref-ref-20-8" class="xref-bibr" href="#ref-20">2014</a>). At -1 position, U was preferred over C as reported (<a id="xref-ref-19-4" class="xref-bibr" href="#ref-19">Li and Du 2013</a>). In contrast, <em>Li and Du</em> observed that the C is more enriched than U, however, the difference in the relative frequencies of U and C residues was minor (<a id="xref-ref-20-9" class="xref-bibr" href="#ref-20">Li and Du 2014</a>). In this study, preference of C was observed at this position only in case of CDS PACs.</p><p id="p-52">Previous investigations of motifs in the NUE region of maize based on NCBI mRNAs and transcript ends assembled from the RNA-Seq data indicated that the canonical hexamer AAUAAA has the highest frequency (<a id="xref-ref-20-10" class="xref-bibr" href="#ref-20">Li and Du 2014</a>; <a id="xref-ref-40-5" class="xref-bibr" href="#ref-40">Wang <em>et al.</em> 2018</a>). The top three observed NUE motifs (AAUAAA, AUAUAU, UAUAUA) in this study are consistent with previous reports (<a id="xref-ref-40-6" class="xref-bibr" href="#ref-40">Wang <em>et al.</em> 2018</a>). The AAUAAA is the top most frequent and significant occurring motif in 3′-UTRs, introns, and intergenic regions. The CDS and 5′-UTR preferred different NUE motifs over AAUAAA. The different nucleotide profiles and NUE motifs for the CDS region relative to the 3′-UTR region were also reported in case of green algae (<a id="xref-ref-51-16" class="xref-bibr" href="#ref-51">Zhao <em>et al.</em> 2014</a>). However, the estimates of NUE motif distribution in the 5′-UTR regions needs further validation in future upon availability of more data.</p><p id="p-53">The frequency of AAUAAA observed in 3′-UTR is lower than reported previously in maize (<a id="xref-ref-40-7" class="xref-bibr" href="#ref-40">Wang <em>et al.</em> 2018</a>). Two factors might have resulted in overestimation of the AAUAAA motif occurrence in the previous studies (<a id="xref-ref-20-11" class="xref-bibr" href="#ref-20">Li and Du 2014</a>; <a id="xref-ref-40-8" class="xref-bibr" href="#ref-40">Wang <em>et al.</em> 2018</a>). First, use of un-clustered transcript ends to estimate the NUE motif frequency relative to the 24 nts clustered PACs used in the current study. Second, a much wider search region (-50 to -1 and -40 to -1) used in previous studies relative to -40 to -10 nts region used in this study. According to our data, about 2,800 genes still maintained the use of AAUAAA predominantly in the upstream regions of the strong or medium PACs (File S5). These genes with AAUAAA motif are involved in core biological processes including cellular amino acid and derivative metabolic process, carbohydrate metabolic process, catabolic process, secondary metabolic process, and generation of precursor metabolites and energy. Further research is required to understand the functional and evolutionary importance of genes that still use AAUAAA motif.</p><p id="p-54">Although large numbers of the PACs were identified using RNA-Seq data, however limited poly(A) reads (0.29%) were detected in whole dataset, which is typical of standard RNA-Seq reads (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Wang <em>et al.</em> 2016b</a>). Different number of poly(A) reads were identified in each SRA dataset. However, for the present study only few SRA datasets provided majority of the poly(A) reads. Highest numbers of reads were observed in samples associated with early development followed by the germinating seedlings. This could be due to the fast growing nature of these tissues leading to tissue-specific APA regulation. Significantly higher rate of distinct APA events were also observed previously in the seedlings of Arabidopsis (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Shen <em>et al.</em> 2011</a>). Considering the over representation of certain libraries in our data, it is likely that the difference in the abundance of different PACs (based on RE values) for the same gene might simply reflect the difference in the proportion of polyadenylated reads from different tissue/cell types. However, even with low coverage in other tissues, highly expressed poly(A) events have more chances of being observed relative to ones with low expression.</p><p id="p-55">The polyadenylation events reported here are still underrepresented and cover only 51% of the maize genes due to low coverage of poly(A) tails in the RNA-Seq data. Its challenging to comprehensively identify all polyadenylation events in each tissues using RNA-Seq. Alternatively, to increase coverage MPSS (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Meyers <em>et al.</em> 2004</a>) high throughput technology can be employed to profile poly(A) sites in maize. Moreover, PAT-seq had been used extensively to profile poly(A) sites in model plant species (<a id="xref-ref-47-13" class="xref-bibr" href="#ref-47">Wu <em>et al.</em> 2011</a>, <a id="xref-ref-46-4" class="xref-bibr" href="#ref-46">2014</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Harrison <em>et al.</em> 2015</a>;). Many other poly(A) specific protocols including PolyA_seq (<a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Derti <em>et al.</em> 2012</a>), and 3′ READS+ (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Zheng <em>et al.</em> 2016</a>) could also be considered in order to overcome the limited number of poly(A) reads observed in this study. Furthermore, there is an immense need of studying correlation of maize poly(A) data with additional data types such as transposable elements (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Lee <em>et al.</em> 2008</a>) and single nucleotide polymorphism (<a id="xref-ref-49-2" class="xref-bibr" href="#ref-49">Xu and Zhang 2018</a>). The study, without a doubt, will help further research on the APA patterns in maize. The findings of this study will improve our understanding about the gene expression regulation in maize.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-56">ZJ and ST collected data and sample details. ZJ analyzed all the datasets and performed initial analysis. ST analyzed GO enrichments. TN and MNA performed statistical analysis. MNA designed and conceived this study. MNA, IS, ZJ, and TN wrote and edited the manuscript. All authors analyzed data, discussed results, and approved the final manuscript version. We thank Arshan Nasir and Noreen Akhtar for the valuable discussions related to results for this study. The authors received no funds from any source for the present study.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-57">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8034695">https://doi.org/10.25387/g3.8034695</a>.</p></li><li class="fn" id="fn-3"><p id="p-58"><em>Communicating editor: J. Ma</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2019-04-24"><span class="received-label">Received </span>April 24, 2019.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-06-21"><span class="accepted-label">Accepted </span>June 21, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2019 Jafar <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.1" data-doi="10.1093/bioinformatics/btu170"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>,  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Lohse</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Usadel</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Trimmomatic: a flexible trimmer for illumina sequence data</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">2114</span>–<span class="cit-lpage">2120</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btu170</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu170%26rft_id%253Dinfo%253Apmid%252F24695404%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btu170&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24695404&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2749.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000340049100004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.2" data-doi="10.1016/j.gpb.2017.06.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Jia</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Song</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Fu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Wei</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Alternative polyadenylation: methods, findings, and impacts</span>. <abbr class="cit-jnl-abbrev">Genomics Proteomics Bioinformatics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">287</span>–<span class="cit-lpage">300</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.gpb.2017.06.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%2BProteomics%2BBioinformatics%26rft.volume%253D15%26rft.spage%253D287%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gpb.2017.06.001%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gpb.2017.06.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.3" data-doi="10.1101/gr.132563.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Derti</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Garrett-Engele</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K. D.</span> <span class="cit-name-surname">MacIsaac</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. C.</span> <span class="cit-name-surname">Stevens</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Sriram</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A quantitative atlas of polyadenylation in five mammals</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1173</span>–<span class="cit-lpage">1183</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.132563.111</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.132563.111%26rft_id%253Dinfo%253Apmid%252F22454233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIyLzYvMTE3MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOC8yNzQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.4" data-doi="10.1016/j.molcel.2011.08.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Di Giammartino</span>,  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Nishida</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Manley</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mechanisms and consequences of alternative polyadenylation</span>. <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">853</span>–<span class="cit-lpage">866</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.molcel.2011.08.017</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DDi%2BGiammartino%26rft.auinit1%253DD.%2BC.%26rft.volume%253D43%26rft.issue%253D6%26rft.spage%253D853%26rft.epage%253D866%26rft.atitle%253DMechanisms%2Band%2Bconsequences%2Bof%2Balternative%2Bpolyadenylation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2011.08.017%26rft_id%253Dinfo%253Apmid%252F21925375%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2011.08.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21925375&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2749.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000295309800002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-book" id="cit-9.8.2749.5" data-doi="10.1007/978-1-4939-2175-1_3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Ji</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q. Q.</span> <span class="cit-name-surname">Li</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Liang</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Extraction of poly (a) sites from large-scale rna-seq data</span>, pp. <span class="cit-fpage">25</span>–<span class="cit-lpage">37</span> in <span class="cit-source">Polyadenylation in Plants</span>, <span class="cit-publ-name">Springer</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-1-4939-2175-1_3</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPolyadenylation%2Bin%2BPlants%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-1-4939-2175-1_3%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-1-4939-2175-1_3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.6" data-doi="10.1093/nar/gkq310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Du</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Ling</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Zhang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Su</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">agrigo: a go analysis toolkit for the agricultural community</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">W64</span>–<span class="cit-lpage">W70</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkq310</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq310%26rft_id%253Dinfo%253Apmid%252F20435677%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq310&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20435677&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2749.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000284148900012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.7" data-doi="10.1101/gr.210757.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Yang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Su</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Ma</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Shen</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Genome-wide dynamics of alternative polyadenylation in rice</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1753</span>–<span class="cit-lpage">1760</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.210757.116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.210757.116%26rft_id%253Dinfo%253Apmid%252F27733415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyNi8xMi8xNzUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI3NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.8" data-doi="10.1186/gb-2013-14-7-r70"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonzàlez-Porta</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Frankish</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Rung</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Harrow</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Brazma</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Transcriptome analysis of human tissues and cell lines reveals one dominant transcript per gene</span>. <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R70</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/gb-2013-14-7-r70</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR70%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-7-r70%26rft_id%253Dinfo%253Apmid%252F23815980%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-7-r70&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23815980&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2749.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.9" data-doi="10.1002/wrna.1206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gruber</span>,  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Martin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Keller</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Zavolan</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Means to an end: mechanisms of alternative polyadenylation of messenger rna precursors</span>. <abbr class="cit-jnl-abbrev">Wiley Interdiscip. Rev. RNA</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">183</span>–<span class="cit-lpage">196</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/wrna.1206</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWiley%2BInterdiscip.%2BRev.%2BRNA%26rft.volume%253D5%26rft.spage%253D183%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fwrna.1206%26rft_id%253Dinfo%253Apmid%252F24243805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/wrna.1206&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24243805&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2749.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.10" data-doi="10.1038/srep28060"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>,  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Spinelli</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q. Q.</span> <span class="cit-name-surname">Li</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Liang</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">A genomewide study of “non-3utr” polyadenylation sites in arabidopsis thaliana</span>. <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">28060</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep28060</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D28060%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep28060%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep28060&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.11" data-doi="10.1101/gr.5836207"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hanada</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. O.</span> <span class="cit-name-surname">Borevitz</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.-H.</span> <span class="cit-name-surname">Li</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.-H.</span> <span class="cit-name-surname">Shiu</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">A large number of novel coding small open reading frames in the intergenic regions of the arabidopsis thaliana genome are transcribed and/or under purifying selection</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">632</span>–<span class="cit-lpage">640</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.5836207</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5836207%26rft_id%253Dinfo%253Apmid%252F17395691%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE3LzUvNjMyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI3NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.12" data-doi="10.1261/rna.048355.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Harrison</span>,  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. R.</span> <span class="cit-name-surname">Powell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Clancy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Preiss</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. R.</span> <span class="cit-name-surname">Boag</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Pat-seq: a method to study the integration of 30 -utr dynamics with gene expression in the eukaryotic transcriptome</span>. <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1502</span>–<span class="cit-lpage">1510</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1261/rna.048355.114</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.048355.114%26rft_id%253Dinfo%253Apmid%252F26092945%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjk6IjIxLzgvMTUwMiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOC8yNzQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.13" data-doi="10.1186/1471-2164-13-641"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hunt</span>,  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Xing</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q. Q.</span> <span class="cit-name-surname">Li</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Plant polyadenylation factors: conservation and variety in the polyadenylation complex in plants</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">641</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-13-641</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DHunt%26rft.auinit1%253DA.%2BG.%26rft.volume%253D13%26rft.issue%253D1%26rft.spage%253D641%26rft.epage%253D641%26rft.atitle%253DPlant%2Bpolyadenylation%2Bfactors%253A%2Bconservation%2Band%2Bvariety%2Bin%2Bthe%2Bpolyadenylation%2Bcomplex%2Bin%2Bplants.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-641%26rft_id%253Dinfo%253Apmid%252F23167306%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-641&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23167306&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2749.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.14" data-doi="10.3389/fgene.2017.00112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jeong</span>,  <span class="cit-name-given-names">H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Nasir</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">A preliminary list of horizontally transferred genes in prokaryotes determined by tree reconstruction and reconciliation</span>. <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">112</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2017.00112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D8%26rft.spage%253D112%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2017.00112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2017.00112&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.15" data-doi="10.1038/nmeth.1923"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>,  <span class="cit-name-given-names">B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Salzberg</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fast gapped-read alignment with bowtie 2</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">357</span>–<span class="cit-lpage">359</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1923</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D9%26rft.issue%253D4%26rft.spage%253D357%26rft.epage%253D359%26rft.atitle%253DFast%2Bgapped-read%2Balignment%2Bwith%2BBowtie%2B2.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1923%26rft_id%253Dinfo%253Apmid%252F22388286%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1923&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22388286&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2749.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000302218500017&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.16" data-doi="10.1093/nar/gkn540"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Ji</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Tian</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Phylogenetic analysis of mrna polyadenylation sites reveals a role of transposable elements in evolution of the 30 -end of genes</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">5581</span>–<span class="cit-lpage">5590</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkn540</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn540%26rft_id%253Dinfo%253Apmid%252F18757892%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkn540&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18757892&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2749.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260424200013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.17" data-doi="10.1093/nar/gkq1019"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Leinonen</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Sugawara</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Shumway</span></span>, <span class="cit-auth cit-collab">International Nucleotide Sequence Database Collaboration</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The sequence read archive</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">D19</span>–<span class="cit-lpage">D21</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkq1019</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkq1019%26rft_id%253Dinfo%253Apmid%252F21062823%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkq1019&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21062823&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2749.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285831700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.18" data-doi="10.1093/bioinformatics/btp352"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Handsaker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Wysoker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Fennell</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Ruan</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">The sequence alignment/map format and samtools</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2078</span>–<span class="cit-lpage">2079</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btp352</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp352%26rft_id%253Dinfo%253Apmid%252F19505943%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btp352&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19505943&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2749.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268808600014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.19" data-doi="10.1371/journal.pone.0079511"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">X. Q.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Du</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Rna polyadenylation sites on the genomes of microorganisms, animals, and plants</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e79511</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0079511</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De79511%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0079511%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0079511&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.20" data-doi="10.1186/s12862-014-0162-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">X. Q.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Du</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Motif types, motif locations and base composition patterns around the rna polyadenylation site in microorganisms, plants and animals</span>. <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">162</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12862-014-0162-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvol.%2BBiol.%26rft.volume%253D14%26rft.spage%253D162%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12862-014-0162-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12862-014-0162-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.21" data-doi="10.1016/j.gpb.2015.02.003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Hao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Zhu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Hu</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Long non-coding rnas and their biological roles in plants</span>. <abbr class="cit-jnl-abbrev">Genomics Proteomics Bioinformatics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">137</span>–<span class="cit-lpage">147</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.gpb.2015.02.003</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%2BProteomics%2BBioinformatics%26rft.volume%253D13%26rft.spage%253D137%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gpb.2015.02.003%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gpb.2015.02.003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.22" data-doi="10.1104/pp.105.060541"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Loke</span>,  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. A.</span> <span class="cit-name-surname">Stahlberg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. G.</span> <span class="cit-name-surname">Strenski</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Haas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. C.</span> <span class="cit-name-surname">Wood</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Compilation of mrna polyadenylation signals in arabidopsis revealed a new signal element and potential secondary structures</span>. <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">1457</span>–<span class="cit-lpage">1468</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1104/pp.105.060541</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.105.060541%26rft_id%253Dinfo%253Apmid%252F15965016%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTM4LzMvMTQ1NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOC8yNzQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.23" data-doi="10.1261/rna.136206"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lopez</span>,  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Granjeaud</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Ara</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Ghattas</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Gautheret</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The disparate nature of “intergenic” polyadenylation sites</span>. <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">1794</span>–<span class="cit-lpage">1801</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1261/rna.136206</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.136206%26rft_id%253Dinfo%253Apmid%252F16931874%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjEwOiIxMi8xMC8xNzk0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI3NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.24" data-doi="10.1038/nbt992"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Meyers</span>,  <span class="cit-name-given-names">B. C.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. H.</span> <span class="cit-name-surname">Vu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. S.</span> <span class="cit-name-surname">Tej</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Ghazal</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Matvienko</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Analysis of the transcriptional complexity of arabidopsis thaliana by massively parallel signature sequencing</span>. <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1006</span>–<span class="cit-lpage">1011</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nbt992</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DMeyers%26rft.auinit1%253DB.%2BC.%26rft.volume%253D22%26rft.issue%253D8%26rft.spage%253D1006%26rft.epage%253D1011%26rft.atitle%253DAnalysis%2Bof%2Bthe%2Btranscriptional%2Bcomplexity%2Bof%2BArabidopsis%2Bthaliana%2Bby%2Bmassively%2Bparallel%2Bsignature%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt992%26rft_id%253Dinfo%253Apmid%252F15247925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt992&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15247925&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2749.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223039500030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.25" data-doi="10.1080/15476286.2016.1191727"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Movassat</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. L.</span> <span class="cit-name-surname">Crabb</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Busch</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Yao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Reynolds</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Coupling between alternative polyadenylation and alternative splicing is limited to terminal introns</span>. <abbr class="cit-jnl-abbrev">RNA Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">646</span>–<span class="cit-lpage">655</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1080/15476286.2016.1191727</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%2BBiol.%26rft.volume%253D13%26rft.spage%253D646%26rft_id%253Dinfo%253Adoi%252F10.1080%252F15476286.2016.1191727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/15476286.2016.1191727&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.26" data-doi="10.1093/nar/gky317"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>,  <span class="cit-name-given-names">N. T. T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Contreras-Moreira</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Castro-Mondragon</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Santana-Garcia</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Ossio</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Rsat 2018: regulatory sequence analysis tools 20th anniversary</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">W209</span>–<span class="cit-lpage">W214</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gky317</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D46%26rft.spage%253DW209%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgky317%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gky317&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.27" data-doi="10.1093/nar/29.8.1690"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pauws</span>,  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Van Kampen</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Van de Graaf</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">De Vijlder</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Heterogeneity in polyadenylation cleavage sites in mammalian mrna sequences: implications for sage analysis</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1690</span>–<span class="cit-lpage">1694</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/29.8.1690</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DPauws%26rft.auinit1%253DE.%26rft.volume%253D29%26rft.issue%253D8%26rft.spage%253D1690%26rft.epage%253D1694%26rft.atitle%253DHeterogeneity%2Bin%2Bpolyadenylation%2Bcleavage%2Bsites%2Bin%2Bmammalian%2BmRNA%2Bsequences%253A%2Bimplications%2Bfor%2BSAGE%2Banalysis%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252F29.8.1690%26rft_id%253Dinfo%253Apmid%252F11292841%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/29.8.1690&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11292841&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2749.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000168229100005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-book" id="cit-9.8.2749.28" data-doi="10.1007/978-94-009-0353-1_2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rothnie</span>,  <span class="cit-name-given-names">H. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">Plant mrna 30 -end formation</span>, pp. <span class="cit-fpage">43</span>–<span class="cit-lpage">61</span> in <span class="cit-source">PostTranscriptional Control of Gene Expression in Plants</span>, <span class="cit-publ-name">Springer</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/978-94-009-0353-1_2</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPostTranscriptional%2BControl%2Bof%2BGene%2BExpression%2Bin%2BPlants%26rft_id%253Dinfo%253Adoi%252F10.1007%252F978-94-009-0353-1_2%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/978-94-009-0353-1_2&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.29" data-doi="10.12688/f1000research.10079.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schmitz</span>,  <span class="cit-name-given-names">J. F.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Bornberg-Bauer</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Fact or fiction: updates on how protein-coding genes might emerge de novo from previously non-coding dna</span>. <abbr class="cit-jnl-abbrev">F1000 Res.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">57</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.12688/f1000research.10079.1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DF1000%2BRes.%26rft.volume%253D6%26rft.spage%253D57%26rft_id%253Dinfo%253Adoi%252F10.12688%252Ff1000research.10079.1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.12688/f1000research.10079.1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.30" data-doi="10.1093/nar/gkn158"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Ji</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. J.</span> <span class="cit-name-surname">Haas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Wu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zheng</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Genome level analysis of rice mrna 30 -end processing signals and alternative polyadenylation</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">3150</span>–<span class="cit-lpage">3161</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkn158</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkn158%26rft_id%253Dinfo%253Apmid%252F18411206%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkn158&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18411206&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2749.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000256173200039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.31" data-doi="10.1101/gr.114744.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shen</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Venu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Nobuta</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Wu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Notibala</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Transcriptome dynamics through alternative polyadenylation in developmental and environmental responses in plants revealed by deep sequencing</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1478</span>–<span class="cit-lpage">1486</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.114744.110</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.114744.110%26rft_id%253Dinfo%253Apmid%252F21813626%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzkvMTQ3OCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOC8yNzQ5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.32" data-doi="10.1016/j.celrep.2012.01.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smibert</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Miura</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. O.</span> <span class="cit-name-surname">Westholm</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Shenker</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">May</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Global patterns of tissue-specific alternative polyadenylation in drosophila</span>. <abbr class="cit-jnl-abbrev">Cell Reports</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">277</span>–<span class="cit-lpage">289</span>. Erratum: 969.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.celrep.2012.01.001</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCell%2BRep%26rft.aulast%253DSmibert%26rft.auinit1%253DP.%26rft.volume%253D1%26rft.issue%253D3%26rft.spage%253D277%26rft.epage%253D289%26rft.atitle%253DGlobal%2Bpatterns%2Bof%2Btissue-specific%2Balternative%2Bpolyadenylation%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2012.01.001%26rft_id%253Dinfo%253Apmid%252F22685694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2012.01.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22685694&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2749.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000309710800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Strable</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. J.</span> <span class="cit-name-surname">Scanlon</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> Maize (zea mays): a model organism for basic and applied research in plant biology. Cold spring harbor protocols 2009: pdb–emo132.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.34" data-doi="10.1002/wrna.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>,  <span class="cit-name-given-names">B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. H.</span> <span class="cit-name-surname">Graber</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Signals for pre-mrna cleavage and polyadenylation</span>. <abbr class="cit-jnl-abbrev">Wiley Interdiscip. Rev. RNA</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">385</span>–<span class="cit-lpage">396</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/wrna.116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DWIREs%2BRNA%26rft.aulast%253DTian%26rft.auinit1%253DB.%26rft.volume%253D3%26rft.issue%253D3%26rft.spage%253D385%26rft.epage%253D396%26rft.atitle%253DSignals%2Bfor%2Bpre-mRNA%2Bcleavage%2Band%2Bpolyadenylation.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fwrna.116%26rft_id%253Dinfo%253Apmid%252F22012871%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/wrna.116&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22012871&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2749.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.35" data-doi="10.1093/nar/gki158"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Hu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Zhang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. S.</span> <span class="cit-name-surname">Lutz</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A large-scale analysis of mrna polyadenylation of human and mouse genes</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">201</span>–<span class="cit-lpage">212</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gki158</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DTian%26rft.auinit1%253DB.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D201%26rft.epage%253D212%26rft.atitle%253DA%2Blarge-scale%2Banalysis%2Bof%2BmRNA%2Bpolyadenylation%2Bof%2Bhuman%2Band%2Bmouse%2Bgenes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki158%26rft_id%253Dinfo%253Apmid%252F15647503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gki158&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15647503&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2749.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000226477000019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.36" data-doi="10.1038/nrm.2016.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>,  <span class="cit-name-given-names">B.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Manley</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Alternative polyadenylation of mrna precursors</span>. <abbr class="cit-jnl-abbrev">Nat. Rev. Mol. Cell Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">18</span>–<span class="cit-lpage">30</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nrm.2016.116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BMol.%2BCell%2BBiol.%26rft.volume%253D18%26rft.spage%253D18%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrm.2016.116%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrm.2016.116&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.37" data-doi="10.1101/gr.5532707"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Z.</span> <span class="cit-name-surname">Pan</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. Y.</span> <span class="cit-name-surname">Lee</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Widespread mrna polyadenylation events in introns indicate dynamic interplay between polyadenylation and splicing</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">156</span>–<span class="cit-lpage">165</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.5532707</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.5532707%26rft_id%253Dinfo%253Apmid%252F17210931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE3LzIvMTU2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI3NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.38" data-doi="10.1016/j.cell.2013.06.020"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ulitsky</span>,  <span class="cit-name-given-names">I.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. P.</span> <span class="cit-name-surname">Bartel</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">lincrnas: genomics, evolution, and mechanisms</span>. <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">154</span>: <span class="cit-fpage">26</span>–<span class="cit-lpage">46</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cell.2013.06.020</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.volume%253D154%26rft.spage%253D26%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2013.06.020%26rft_id%253Dinfo%253Apmid%252F23827673%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2013.06.020&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23827673&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2749.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000321327900008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.39" data-doi="10.1101/gr.139733.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ulitsky</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Shkumatava</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. H.</span> <span class="cit-name-surname">Jan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. O.</span> <span class="cit-name-surname">Subtelny</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Koppstein</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Extensive alternative polyadenylation during zebrafish development</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">2054</span>–<span class="cit-lpage">2066</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.139733.112</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.139733.112%26rft_id%253Dinfo%253Apmid%252F22722342%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8xMC8yMDU0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI3NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.40" data-doi="10.1101/gr.227462.117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Regulski</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Tseng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Olson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Goodwin</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">A comparative transcriptional landscape of maize and sorghum obtained by single-molecule sequencing</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">921</span>–<span class="cit-lpage">932</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.227462.117</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.227462.117%26rft_id%253Dinfo%253Apmid%252F29712755%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI4LzYvOTIxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI3NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.41" data-doi="10.1038/ncomms11708"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Tseng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Regulski</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. A.</span> <span class="cit-name-surname">Clark</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Hon</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">a</span> <span class="cit-article-title">Unveiling the complexity of the maize transcriptome by single-molecule long-read sequencing</span>. <abbr class="cit-jnl-abbrev">Nat. Commun.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">11708</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ncomms11708</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BCommun.%26rft.volume%253D7%26rft.spage%253D11708%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fncomms11708%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ncomms11708&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.42" data-doi="10.1038/srep36388"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Li</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhou</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Xue</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Xu</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span><span class="cit-comment">b</span> <span class="cit-article-title">Genome-wide analysis and functional characterization of the polyadenylation site in pigs using rnaseq data</span>. <abbr class="cit-jnl-abbrev">Sci. Rep.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">36388</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/srep36388</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BRep.%26rft.volume%253D6%26rft.spage%253D36388%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsrep36388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/srep36388&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.43" data-doi="10.1093/nar/gkx1000"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Nambiar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Zheng</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Tian</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">Polya_db 3 catalogs cleavage and polyadenylation sites identified by deep sequencing in multiple genomes</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">D315</span>–<span class="cit-lpage">D319</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gkx1000</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D46%26rft.spage%253DD315%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkx1000%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gkx1000&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.44" data-doi="10.1093/mp/sst177"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W.</span> <span class="cit-name-surname">Deng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Fan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.-T.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genomic features and regulatory roles of intermediate-sized non-coding rnas in arabidopsis</span>. <abbr class="cit-jnl-abbrev">Mol. Plant</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">514</span>–<span class="cit-lpage">527</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/mp/sst177</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BPlant%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmp%252Fsst177%26rft_id%253Dinfo%253Apmid%252F24398630%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/mp/sst177&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24398630&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2749.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000335895200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.45" data-doi="10.1002/pld3.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wimalanathan</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I.</span> <span class="cit-name-surname">Friedberg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Andorf</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. J.</span> <span class="cit-name-surname">Lawrence-Dill</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Maize go annotation—methods, evaluation, and review (maize-gamer)</span>. <abbr class="cit-jnl-abbrev">Plant Direct</abbr> <span class="cit-vol">2</span>: <span class="cit-elocation-id">e00052</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/pld3.52</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BDirect%26rft.volume%253D2%26rft.spage%253De00052%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fpld3.52%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/pld3.52&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.46" data-doi="10.1186/1471-2164-15-615"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Gaffney</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. G.</span> <span class="cit-name-surname">Hunt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q. Q.</span> <span class="cit-name-surname">Li</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Genome-wide determination of poly (a) sites in medicago truncatula: evolutionary conservation of alternative poly (a) site choice</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">615</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2164-15-615</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D15%26rft.spage%253D615%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-15-615%26rft_id%253Dinfo%253Apmid%252F25048171%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-15-615&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25048171&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2749.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.47" data-doi="10.1073/pnas.1019732108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Downie</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Liang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Ji</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide landscape of polyadenylation in arabidopsis provides evidence for extensive alternative polyadenylation</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">12533</span>–<span class="cit-lpage">12538</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.1019732108</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1019732108%26rft_id%253Dinfo%253Apmid%252F21746925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzMwLzEyNTMzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI3NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.48" data-doi="10.1186/s12864-015-1691-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>,  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Zeng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Guan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Ji</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Huang</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Genome-wide characterization of intergenic polyadenylation sites redefines gene spaces in arabidopsis thaliana</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">511</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-015-1691-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D16%26rft.spage%253D511%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-015-1691-1%26rft_id%253Dinfo%253Apmid%252F26155789%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-015-1691-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26155789&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2749.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.49" data-doi="10.1016/j.cels.2018.05.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>,  <span class="cit-name-given-names">C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Zhang</span></span></li></ol><cite>, <span class="cit-pub-date">2018</span> <span class="cit-article-title">Alternative polyadenylation of mammalian transcripts is generally deleterious, not adaptive</span>. <abbr class="cit-jnl-abbrev">Cell Syst.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">734</span>–<span class="cit-lpage">742.e4</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cels.2018.05.007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BSyst.%26rft.volume%253D6%26rft.spage%253D734%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cels.2018.05.007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cels.2018.05.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.50" data-doi="10.1093/nar/gku1076"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">You</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Wu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Feng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Fu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Guo</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Apasdb: a database describing alternative poly (a) sites and selection of heterogeneous cleavage sites downstream of poly (a) signals</span>. <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">D59</span>–<span class="cit-lpage">D67</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/nar/gku1076</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1076%26rft_id%253Dinfo%253Apmid%252F25378337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/nar/gku1076&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25378337&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2749.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Wu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. K. R.</span> <span class="cit-name-surname">Kumar</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Dong</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Ji</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Bioinformatics analysis of alternative polyadenylation in green alga chlamydomonas reinhardtii using transcriptome sequences from three different sequencing platforms. G3: Genes, Genomes</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">871</span>–<span class="cit-lpage">883</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.volume%253D4%26rft.spage%253D871%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.52" data-doi="10.1261/rna.057075.116"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>,  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Liu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Tian</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">30 reads+, a sensitive and accurate method for 30 end sequencing of polyadenylated rna</span>. <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1631</span>–<span class="cit-lpage">1639</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1261/rna.057075.116</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.057075.116%26rft_id%253Dinfo%253Apmid%252F27512124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjEwOiIyMi8xMC8xNjMxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI3NDkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-9.8.2749.53" data-doi="10.3389/fendo.2017.00090"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ziegler</span>,  <span class="cit-name-given-names">C.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kretz</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span> <span class="cit-article-title">The more the merrier—complexity in long non-coding rna loci</span>. <abbr class="cit-jnl-abbrev">Front. Endocrinol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">90</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fendo.2017.00090</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BEndocrinol.%26rft.volume%253D8%26rft.spage%253D90%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffendo.2017.00090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fendo.2017.00090&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/8/2749.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/8/2739" title="Genomic Prediction of Additive and Non-additive Effects Using Genetic Markers and Pedigrees" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/8/2761" title="Signatures of Recent Positive Selection in Enhancers Across 41 Human Tissues" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/8"><span>Volume 9 Issue 8, August 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/8.cover-source.jpg?itok=cSNYBBSf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/8/2749" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-6XR8TCJd-Io1kkJMIHyOsD2gY81Lq0rCZfeZl9WE0ks" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/8/2749" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/8/2749" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/8/2749" class="active" data-icon-position="" data-hide-link-title="0">Genome-Wide Profiling of Polyadenylation Events in Maize Using High-Throughput Transcriptomic Sequences</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/493750" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-W_9Yp8bju_16BrrmydwVhw_yPyfm6wlqnShJGwmkuig" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/8/2749.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/8/2749" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-S1ecWjDxcpAFCySRO-D_9vRp1JU2xRhjY7-7F6Ky9KI" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/8/2749" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/8/2749.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493750" id="node493750--2" data-pisa="ggg;9/8/2749" data-pisa-master="ggg;g3.119.400196" data-apath="/ggg/9/8/2749.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Profiling of Polyadenylation Events in Maize Using High-Throughput Transcriptomic Sequences</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zehra Jafar</span>, <span class="highwire-citation-author" data-delta="1">Salma Tariq</span>, <span class="highwire-citation-author" data-delta="2">Irfan Sadiq</span>, <span class="highwire-citation-author" data-delta="3">Tayyab Nawaz</span> and <span class="highwire-citation-author" data-delta="4">Malik Nadeem Akhtar</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2749-2760;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400196 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493750--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zehra Jafar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Computational Biology and Bioinformatics Laboratory, Department of Biosciences, COMSATS University Islamabad, Islamabad 45550, Pakistan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zehra%2BJafar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jafar%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZehra%2BJafar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Salma Tariq </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Computational Biology and Bioinformatics Laboratory, Department of Biosciences, COMSATS University Islamabad, Islamabad 45550, Pakistan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Salma%2BTariq%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tariq%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASalma%2BTariq%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Irfan Sadiq </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Computational Biology and Bioinformatics Laboratory, Department of Biosciences, COMSATS University Islamabad, Islamabad 45550, Pakistan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Irfan%2BSadiq%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sadiq%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIrfan%2BSadiq%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Tayyab Nawaz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Mathematics, COMSATS University Islamabad, Islamabad 45550, Pakistan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tayyab%2BNawaz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nawaz%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATayyab%2BNawaz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Malik Nadeem Akhtar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Computational Biology and Bioinformatics Laboratory, Department of Biosciences, COMSATS University Islamabad, Islamabad 45550, Pakistan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Malik%2BNadeem%2BAkhtar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Akhtar%20MN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMalik%2BNadeem%2BAkhtar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nadeemakhtar@comsats.edu.pk" class="" data-icon-position="" data-hide-link-title="0">nadeemakhtar@comsats.edu.pk</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/8/2749" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/8/2749&amp;title=Genome-Wide%20Profiling%20of%20Polyadenylation%20Events%20in%20Maize%20Using%20High-Throughput%20Transcriptomic%20Sequences" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/8/2749&amp;title=Genome-Wide%20Profiling%20of%20Polyadenylation%20Events%20in%20Maize%20Using%20High-Throughput%20Transcriptomic%20Sequences" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/8/2749&amp;title=Genome-Wide%20Profiling%20of%20Polyadenylation%20Events%20in%20Maize%20Using%20High-Throughput%20Transcriptomic%20Sequences" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/8/2749&amp;text=Genome-Wide%20Profiling%20of%20Polyadenylation%20Events%20in%20Maize%20Using%20High-Throughput%20Transcriptomic%20Sequences" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/8/2749&amp;title=Genome-Wide%20Profiling%20of%20Polyadenylation%20Events%20in%20Maize%20Using%20High-Throughput%20Transcriptomic%20Sequences" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/8/2749&amp;t=Genome-Wide%20Profiling%20of%20Polyadenylation%20Events%20in%20Maize%20Using%20High-Throughput%20Transcriptomic%20Sequences" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/8/2749&amp;title=Genome-Wide%20Profiling%20of%20Polyadenylation%20Events%20in%20Maize%20Using%20High-Throughput%20Transcriptomic%20Sequences" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/8/2749&amp;title=Genome-Wide%20Profiling%20of%20Polyadenylation%20Events%20in%20Maize%20Using%20High-Throughput%20Transcriptomic%20Sequences" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/493750/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493750" id="node493750--3" data-pisa="ggg;9/8/2749" data-pisa-master="ggg;g3.119.400196" data-apath="/ggg/9/8/2749.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Profiling of Polyadenylation Events in Maize Using High-Throughput Transcriptomic Sequences</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Zehra Jafar</span>, <span class="highwire-citation-author" data-delta="1">Salma Tariq</span>, <span class="highwire-citation-author" data-delta="2">Irfan Sadiq</span>, <span class="highwire-citation-author" data-delta="3">Tayyab Nawaz</span> and <span class="highwire-citation-author" data-delta="4">Malik Nadeem Akhtar</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2749-2760;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.119.400196 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493750--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Zehra Jafar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Computational Biology and Bioinformatics Laboratory, Department of Biosciences, COMSATS University Islamabad, Islamabad 45550, Pakistan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Zehra%2BJafar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jafar%20Z&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AZehra%2BJafar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Salma Tariq </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Computational Biology and Bioinformatics Laboratory, Department of Biosciences, COMSATS University Islamabad, Islamabad 45550, Pakistan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Salma%2BTariq%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tariq%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASalma%2BTariq%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Irfan Sadiq </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Computational Biology and Bioinformatics Laboratory, Department of Biosciences, COMSATS University Islamabad, Islamabad 45550, Pakistan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Irfan%2BSadiq%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sadiq%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AIrfan%2BSadiq%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Tayyab Nawaz </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Department of Mathematics, COMSATS University Islamabad, Islamabad 45550, Pakistan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Tayyab%2BNawaz%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Nawaz%20T&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ATayyab%2BNawaz%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Malik Nadeem Akhtar </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Computational Biology and Bioinformatics Laboratory, Department of Biosciences, COMSATS University Islamabad, Islamabad 45550, Pakistan</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Malik%2BNadeem%2BAkhtar%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Akhtar%20MN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMalik%2BNadeem%2BAkhtar%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nadeemakhtar@comsats.edu.pk" class="" data-icon-position="" data-hide-link-title="0">nadeemakhtar@comsats.edu.pk</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2749&amp;rft.epage=2760&amp;rft.atitle=Genome-Wide%20Profiling%20of%20Polyadenylation%20Events%20in%20Maize%20Using%20High-Throughput%20Transcriptomic%20Sequences&amp;rft.volume=9&amp;rft.issue=8&amp;rft.date=2019-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Jafar%2C+Zehra&amp;rft.au=Tariq%2C+Salma&amp;rft.au=Sadiq%2C+Irfan&amp;rft.au=Nawaz%2C+Tayyab&amp;rft.au=Akhtar%2C+Malik+Nadeem"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/493750/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/493750/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/493750/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/493750/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/493750/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/493750/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/493750/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/493750/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/493750/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/493750/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/493750/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/493750/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/8/2749" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-9" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/8/2749.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/8/2749.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/8/2749.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
