<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/9/8/2447" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/9/8/2447.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/9/8/2447.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/9/8/2447.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/9/8/2447/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/9/8/2447.atom" />
<meta name="HW.pisa" content="ggg;9/8/2447" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="GRAF-pop: A Fast Distance-Based Method To Infer Subject Ancestry from Multiple Genotype Datasets Without Principal Components Analysis" />
<meta name="DC.Identifier" content="10.1534/g3.118.200925" />
<meta name="DC.Date" content="2019-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2019 Jin et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Inferring subject ancestry using genetic data is an important step in genetic association studies, required for dealing with population stratification. It has become more challenging to infer subject ancestry quickly and accurately since large amounts of genotype data, collected from millions of subjects by thousands of studies using different methods, are accessible to researchers from repositories such as the database of Genotypes and Phenotypes (dbGaP) at the National Center for Biotechnology Information (NCBI). Study-reported populations submitted to dbGaP are often not harmonized across studies or may be missing. Widely-used methods for ancestry prediction assume that most markers are genotyped in all subjects, but this assumption is unrealistic if one wants to combine studies that used different genotyping platforms. To provide ancestry inference and visualization across studies, we developed a new method, GRAF-pop, of ancestry prediction that is robust to missing genotypes and allows researchers to visualize predicted population structure in color and in three dimensions. When genotypes are dense, GRAF-pop is comparable in quality and running time to existing ancestry inference methods EIGENSTRAT, FastPCA, and FlashPCA2, all of which rely on principal components analysis (PCA). When genotypes are not dense, GRAF-pop gives much better ancestry predictions than the PCA-based methods. GRAF-pop employs basic geometric and probabilistic methods; the visualized ancestry predictions have a natural geometric interpretation, which is lacking in PCA-based methods. Since February 2018, GRAF-pop has been successfully incorporated into the dbGaP quality control process to identify inconsistencies between study-reported and computationally predicted populations and to provide harmonized population values in all new dbGaP submissions amenable to population prediction, based on marker genotypes. Plots, produced by GRAF-pop, of summary population predictions are available on dbGaP study pages, and the software, is available at &lt;https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/Software.cgi&gt;." />
<meta name="DC.Contributor" content="Yumi Jin" />
<meta name="DC.Contributor" content="Alejandro A. Schaffer" />
<meta name="DC.Contributor" content="Michael Feolo" />
<meta name="DC.Contributor" content="J. Bradley Holmes" />
<meta name="DC.Contributor" content="Brandi L. Kattman" />
<meta name="article:published_time" content="2019-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="GRAF-pop: A Fast Distance-Based Method To Infer Subject Ancestry from Multiple Genotype Datasets Without Principal Components Analysis" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Inferring subject ancestry using genetic data is an important step in genetic association studies, required for dealing with population stratification. It has become more challenging to infer subject ancestry quickly and accurately since large amounts of genotype data, collected from millions of subjects by thousands of studies using different methods, are accessible to researchers from repositories such as the database of Genotypes and Phenotypes (dbGaP) at the National Center for Biotechnology Information (NCBI). Study-reported populations submitted to dbGaP are often not harmonized across studies or may be missing. Widely-used methods for ancestry prediction assume that most markers are genotyped in all subjects, but this assumption is unrealistic if one wants to combine studies that used different genotyping platforms. To provide ancestry inference and visualization across studies, we developed a new method, GRAF-pop, of ancestry prediction that is robust to missing genotypes and allows researchers to visualize predicted population structure in color and in three dimensions. When genotypes are dense, GRAF-pop is comparable in quality and running time to existing ancestry inference methods EIGENSTRAT, FastPCA, and FlashPCA2, all of which rely on principal components analysis (PCA). When genotypes are not dense, GRAF-pop gives much better ancestry predictions than the PCA-based methods. GRAF-pop employs basic geometric and probabilistic methods; the visualized ancestry predictions have a natural geometric interpretation, which is lacking in PCA-based methods. Since February 2018, GRAF-pop has been successfully incorporated into the dbGaP quality control process to identify inconsistencies between study-reported and computationally predicted populations and to provide harmonized population values in all new dbGaP submissions amenable to population prediction, based on marker genotypes. Plots, produced by GRAF-pop, of summary population predictions are available on dbGaP study pages, and the software, is available at https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/Software.cgi.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2019/08/01" />
<meta name="citation_mjid" content="ggg;9/8/2447" />
<meta name="citation_id" content="9/8/2447" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/9/8/2447" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/9/8/2447.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/9/8/2447.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/9/8/2447.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.118.200925" />
<meta name="citation_pmid" content="31151998" />
<meta name="citation_volume" content="9" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="15" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="2447" />
<meta name="citation_lastpage" content="2461" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Yumi Jin" />
<meta name="citation_author_institution" content="National Center for Biotechnology Information, National Institutes of Health, Department of Health and Human Services, Bethesda, Maryland 20894 and" />
<meta name="citation_author_email" content="jinyu@ncbi.nlm.nih.gov" />
<meta name="citation_author" content="Alejandro A. Schaffer" />
<meta name="citation_author_institution" content="Cancer Data Science Laboratory, National Cancer Institute, National Institutes of Health; Department of Health and Human Services; Bethesda, Maryland 20892" />
<meta name="citation_author_institution" content="National Center for Biotechnology Information, National Institutes of Health, Department of Health and Human Services, Bethesda, Maryland 20894 and" />
<meta name="citation_author" content="Michael Feolo" />
<meta name="citation_author_institution" content="National Center for Biotechnology Information, National Institutes of Health, Department of Health and Human Services, Bethesda, Maryland 20894 and" />
<meta name="citation_author" content="J. Bradley Holmes" />
<meta name="citation_author_institution" content="National Center for Biotechnology Information, National Institutes of Health, Department of Health and Human Services, Bethesda, Maryland 20894 and" />
<meta name="citation_author" content="Brandi L. Kattman" />
<meta name="citation_author_institution" content="National Center for Biotechnology Information, National Institutes of Health, Department of Health and Human Services, Bethesda, Maryland 20894 and" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=G. Abraham;citation_author=M. Inouye;citation_title=Fast principal component analysis of large-scale genome-wide data;citation_pages=e93766;citation_volume=9;citation_year=2014;citation_pmid=24718290;citation_doi=10.1371/journal.pone.0093766" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=G. Abraham;citation_author=Y. Qiu;citation_author=M. Inouye;citation_title=FlashPCA2: principal component analysis of Biobank-scale genotype datasets;citation_pages=2776-2778;citation_volume=33;citation_year=2017;citation_pmid=28475694;citation_doi=10.1093/bioinformatics/btx299" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=D. H. Alexander;citation_author=K. Lange;citation_title=Enhancements to the ADMIXTURE algorithm for individual ancestry estimation.;citation_pages=246-246;citation_volume=12;citation_year=2011;citation_pmid=21682921;citation_doi=10.1186/1471-2105-12-246" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. H. Alexander;citation_author=J. Novembre;citation_author=K. Lange;citation_title=Fast model-based estimation of ancestry in unrelated individuals;citation_volume=19;citation_year=2009;citation_pmid=19648217;citation_doi=10.1101/gr.094052.109" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=V. Bansal;citation_author=O. Libiger;citation_title=Fast individual ancestry inference from DNA sequence data leveraging allele frequencies for multiple populations;citation_pages=4;citation_volume=16;citation_year=2015;citation_pmid=25592880;citation_doi=10.1186/s12859-014-0418-7" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=K. Bryc;citation_author=A. Auton;citation_author=M. R. Nelson;citation_author=J. R. Oksenberg;citation_author=S. L. Hauser;citation_title=Genome-wide patterns of population structure and admixture in West Africans and African Americans;citation_volume=107;citation_year=2010;citation_pmid=20080753;citation_doi=10.1073/pnas.0909559107" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=K. Bryc;citation_author=E. Y. Durand;citation_author=J. M. Macpherson;citation_author=D. Reich;citation_author=J. L. Mountain;citation_title=The genetic ancestry of African Americans, Latinos, and European Americans across the United States;citation_pages=37-53;citation_volume=96;citation_year=2015;citation_pmid=25529636;citation_doi=10.1016/j.ajhg.2014.11.010" />
<meta name="citation_reference" content="citation_journal_title=Forensic Sci. Int. Genet.;citation_author=O. Bulbul;citation_author=W. C. Speed;citation_author=C. Gurkan;citation_author=U. Soundararajan;citation_author=H. Rajeevan;citation_title=Improving ancestry distinctions among Southwest Asian populations;citation_pages=14-20;citation_volume=35;citation_year=2018;citation_doi=10.1016/j.fsigen.2018.03.010" />
<meta name="citation_reference" content="citation_journal_title=BMC Genomics;citation_author=J. Byun;citation_author=Y. Han;citation_author=I. P. Gorlov;citation_author=J. A. Busam;citation_author=M. F. Seldin;citation_title=Ancestry inference using principal component analysis and spatial analysis: a distance-based analysis to account for population substructure;citation_pages=789;citation_volume=18;citation_year=2017;citation_doi=10.1186/s12864-017-4166-8" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. Y. Chen;citation_author=S. Pollack;citation_author=D. J. Hunter;citation_author=J. N. Hirschhorn;citation_author=P. Kraft;citation_title=Improved ancestry inference using weights from external reference panels;citation_volume=29;citation_year=2013;citation_pmid=23539302;citation_doi=10.1093/bioinformatics/btt144" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=L. Clarke;citation_author=X. Zheng-Bradley;citation_author=R. Smith;citation_author=E. Kulesha;citation_author=C. Xiao;citation_title=The 1000 Genomes Project: data management and community access.;citation_pages=459-462;citation_volume=9;citation_year=2012;citation_issue=5;citation_pmid=22543379;citation_doi=10.1038/nmeth.1974" />
<meta name="citation_reference" content="citation_journal_title=Genet. Epidemiol.;citation_author=M. P. Conomos;citation_author=M. B. Miller;citation_author=T. A. Thornton;citation_title=Robust inference of population structure for ancestry prediction and correction of stratification in the presence of relatedness;citation_pages=276-293;citation_volume=39;citation_year=2015;citation_pmid=25810074;citation_doi=10.1002/gepi.21896" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=M. Daya;citation_author=L. van der Merwe;citation_author=U. Galal;citation_author=M. Moller;citation_author=M. Salie;citation_title=A panel of ancestry informative markers for the complex five-way admixed South African coloured population;citation_pages=e82224;citation_volume=8;citation_year=2013;citation_doi=10.1371/journal.pone.0082224" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=J. M. Galanter;citation_author=J. C. Fernandez-Lopez;citation_author=C. R. Gignoux;citation_author=J. Barnholtz-Sloan;citation_author=C. Fernandez-Rozadilla;citation_title=Development of a panel of genome-wide ancestry informative markers to study admixture throughout the Americas.;citation_pages=e1002554-e1002554;citation_volume=8;citation_year=2012;citation_issue=3;citation_pmid=22412386;citation_doi=10.1371/journal.pgen.1002554" />
<meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.;citation_author=K. J. Galinsky;citation_author=G. Bhatia;citation_author=P. R. Loh;citation_author=S. Georgiev;citation_author=S. Mukherjee;citation_title=Fast Principal-Component Analysis Reveals Convergent Evolution of ADH1B in Europe and East Asia;citation_pages=456-472;citation_volume=98;citation_year=2016;citation_doi=10.1016/j.ajhg.2015.12.022" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=S. Gravel;citation_author=F. Zakharia;citation_author=A. Moreno-Estrada;citation_author=J. K. Byrnes;citation_author=M. Muzzio;citation_title=Reconstructing Native American migrations from whole-genome and whole-exome data;citation_pages=e1004023;citation_volume=9;citation_year=2013;citation_pmid=24385924;citation_doi=10.1371/journal.pgen.1004023" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=G. Greenbaum;citation_author=A. R. Templeton;citation_author=S. Bar-David;citation_title=Inference and Analysis of Population Structure Using Genetic Data and Network Theory;citation_volume=202;citation_year=2016;citation_pmid=26888080;citation_doi=10.1534/genetics.115.182626" />
<meta name="citation_reference" content="citation_journal_title=Human mutation;citation_journal_abbrev=Hum Mutat;citation_author=I. Halder;citation_author=M. Shriver;citation_author=M. Thomas;citation_author=J. R. Fernandez;citation_author=T. Frudakis;citation_title=A panel of ancestry informative markers for estimating individual biogeographical ancestry and admixture from four continents: utility and applications.;citation_pages=648-658;citation_volume=29;citation_year=2008;citation_issue=5;citation_pmid=18286470;citation_doi=10.1002/humu.20695" />
<meta name="citation_reference" content="citation_journal_title=PLoS One;citation_author=Y. Jin;citation_author=A. A. Schaffer;citation_author=S. T. Sherry;citation_author=M. Feolo;citation_title=Quickly identifying identical and closely related subjects in large databases using genotype data;citation_pages=e0179106;citation_volume=12;citation_year=2017;citation_doi=10.1371/journal.pone.0179106" />
<meta name="citation_reference" content="citation_journal_title=Eng. Sci.;citation_author=I. T. Jolliffe;citation_author=J. Cadima;citation_title=Principal component analysis: a review and recent developments. Philos Trans A Math Phys;citation_volume=374;citation_year=2016;citation_pmid=26953178;citation_doi=10.1098/rsta.2015.0202" />
<meta name="citation_reference" content="citation_journal_title=Forensic Sci. Int. Genet.;citation_author=K. K. Kidd;citation_author=W. C. Speed;citation_author=A. J. Pakstis;citation_author=M. R. Furtado;citation_author=R. Fang;citation_title=Progress toward an efficient panel of SNPs for ancestry inference;citation_pages=23-32;citation_volume=10;citation_year=2014;citation_pmid=24508742;citation_doi=10.1016/j.fsigen.2014.01.002" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=I. Lappalainen;citation_author=J. Almeida-King;citation_author=V. Kumanduri;citation_author=A. Senf;citation_author=J. D. Spalding;citation_title=The European Genome-phenome Archive of human data consented for biomedical research;citation_pages=692-695;citation_volume=47;citation_year=2015;citation_pmid=26111507;citation_doi=10.1038/ng.3312" />
<meta name="citation_reference" content="citation_journal_title=Annual review of genomics and human genetics;citation_journal_abbrev=Annu Rev Genomics Hum Genet;citation_author=D. J. Lawson;citation_author=D. Falush;citation_title=Population identification using genetic data.;citation_pages=337-361;citation_volume=13;citation_year=2012;citation_pmid=22703172;citation_doi=10.1146/annurev-genom-082410-101510" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=A. B. Lee;citation_author=D. Luca;citation_author=L. Klei;citation_author=B. Devlin;citation_author=K. Roeder;citation_title=Discovering genetic ancestry using spectral graph theory.;citation_pages=51-59;citation_volume=34;citation_year=2010;citation_issue=1;citation_pmid=19455578;citation_doi=10.1002/gepi.20434" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=Y. Li;citation_author=J. Byun;citation_author=G. Cai;citation_author=X. Xiao;citation_author=Y. Han;citation_title=FastPop: a rapid principal component derived method to infer intercontinental ancestry using genetic data;citation_pages=122;citation_volume=17;citation_year=2016;citation_doi=10.1186/s12859-016-0965-1" />
<meta name="citation_reference" content="citation_journal_title=Hum. Genomics;citation_author=Y. Liu;citation_author=T. Nyunoya;citation_author=S. Leng;citation_author=S. A. Belinsky;citation_author=Y. Tesfaigzi;citation_title=Softwares and methods for estimating genetic ancestry in human populations;citation_pages=1;citation_volume=7;citation_year=2013;citation_pmid=23289408;citation_doi=10.1186/1479-7364-7-1" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. D. Mailman;citation_author=M. Feolo;citation_author=Y. Jin;citation_author=M. Kimura;citation_author=K. Tryka;citation_title=The NCBI dbGaP database of genotypes and phenotypes.;citation_pages=1181-1186;citation_volume=39;citation_year=2007;citation_issue=10;citation_pmid=17898773;citation_doi=10.1038/ng1007-1181" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=G. McVean;citation_title=A genealogical interpretation of principal components analysis.;citation_pages=e1000686-e1000686;citation_volume=5;citation_year=2009;citation_issue=10;citation_pmid=19834557;citation_doi=10.1371/journal.pgen.1000686" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=P. Menozzi;citation_author=A. Piazza;citation_author=L. Cavalli-Sforza;citation_title=Synthetic maps of human gene frequencies in Europeans;citation_pages=786-792;citation_volume=201;citation_year=1978;citation_issue=4358;citation_pmid=356262;citation_doi=10.1126/science.356262" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=R. Nassir;citation_author=R. Kosoy;citation_author=C. Tian;citation_author=P. A. White;citation_author=L. M. Butler;citation_title=An ancestry informative marker set for determining continental origin: validation and extension using human genome diversity panels.;citation_pages=39-39;citation_volume=10;citation_year=2009;citation_pmid=19630973;citation_doi=10.1186/1471-2156-10-39" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Genet. Dev.;citation_author=J. Novembre;citation_author=B. M. Peter;citation_title=Recent advances in the study of fine-scale population structure in humans;citation_pages=98-105;citation_volume=41;citation_year=2016;citation_doi=10.1016/j.gde.2016.08.007" />
<meta name="citation_reference" content="citation_journal_title=Front. Genet.;citation_author=B. Padhukasahasram;citation_title=Inferring ancestry from population genomic data and its applications;citation_pages=204;citation_volume=5;citation_year=2014;citation_pmid=25071832;citation_doi=10.3389/fgene.2014.00204" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS Genet;citation_author=N. Patterson;citation_author=A. L. Price;citation_author=D. Reich;citation_title=Population structure and eigenanalysis.;citation_pages=e190-e190;citation_volume=2;citation_year=2006;citation_issue=12;citation_pmid=17194218;citation_doi=10.1371/journal.pgen.0020190" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. L. Price;citation_author=N. J. Patterson;citation_author=R. M. Plenge;citation_author=M. E. Weinblatt;citation_author=N. A. Shadick;citation_title=Principal components analysis corrects for stratification in genome-wide association studies.;citation_pages=904-909;citation_volume=38;citation_year=2006;citation_issue=8;citation_pmid=16862161;citation_doi=10.1038/ng1847" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. K. Pritchard;citation_author=M. Stephens;citation_author=P. Donnelly;citation_title=Inference of Population Structure Using Multilocus Genotype Data;citation_pages=945-959;citation_volume=155;citation_year=2000;citation_issue=2;citation_pmid=10835412;citation_doi=10.1111/j.1471-8286.2005.01155.x" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=S. Purcell;citation_author=B. Neale;citation_author=K. Todd-Brown;citation_author=L. Thomas;citation_author=M. A. Ferreira;citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses.;citation_pages=559-575;citation_volume=81;citation_year=2007;citation_issue=3;citation_pmid=17701901;citation_doi=10.1086/519795" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Raj;citation_author=M. Stephens;citation_author=J. K. Pritchard;citation_title=fastSTRUCTURE: variational inference of population structure in large SNP data sets;citation_volume=197;citation_year=2014;citation_pmid=24700103;citation_doi=10.1534/genetics.114.164350" />
<meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics;citation_author=S. S. Shringarpure;citation_author=C. D. Bustamante;citation_author=K. Lange;citation_author=D. H. Alexander;citation_title=Efficient analysis of large datasets and sex bias with ADMIXTURE;citation_pages=218;citation_volume=17;citation_year=2016;citation_doi=10.1186/s12859-016-1082-x" />
<meta name="citation_reference" content="citation_journal_title=Comput. Graph.;citation_author=V. Skala;citation_title=Barycentric coordinates computation in homogeneous coordinates;citation_pages=120-127;citation_volume=32;citation_year=2008;citation_doi=10.1016/j.cag.2007.09.007" />
<meta name="citation_reference" content="citation_journal_abbrev=Talanta;citation_author=I. Stanimirova;citation_author=M. Daszykowski;citation_author=B. Walczak;citation_title=Dealing with missing values and outliers in principal component analysis.;citation_pages=172-178;citation_volume=72;citation_year=2007;citation_issue=1;citation_pmid=19071598;citation_doi=10.1016/j.talanta.2006.10.011" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=H. Tang;citation_author=M. Coram;citation_author=P. Wang;citation_author=X. Zhu;citation_author=N. Risch;citation_title=Reconstructing genetic ancestry blocks in admixed individuals.;citation_pages=1-12;citation_volume=79;citation_year=2006;citation_issue=1;citation_pmid=16773560;citation_doi=10.1086/504302" />
<meta name="citation_reference" content="citation_journal_title=Genetic epidemiology;citation_journal_abbrev=Genet Epidemiol;citation_author=H. Tang;citation_author=J. Peng;citation_author=P. Wang;citation_author=N. J. Risch;citation_title=Estimation of individual admixture: analytical and study design considerations.;citation_pages=289-301;citation_volume=28;citation_year=2005;citation_issue=4;citation_pmid=15712363;citation_doi=10.1002/gepi.20064" />
<meta name="citation_reference" content="citation_journal_title=BMC Proc.;citation_author=T. Thornton;citation_author=M. P. Conomos;citation_author=S. Sverdlov;citation_author=E. M. Blue;citation_author=C. Y. Cheung;citation_title=Estimating and adjusting for ancestry admixture in statistical methods for relatedness inference, heritability estimation, and association testing;citation_pages=S5;citation_volume=8;citation_year=2014;citation_doi=10.1186/1753-6561-8-S1-S5" />
<meta name="citation_reference" content="citation_title=Barycentric Calculus in Euclidean and Hyperbolic Geometry;citation_author=A. A. Ungar;citation_year=2010;citation_doi=10.1142/7740" />
<meta name="citation_reference" content="citation_journal_title=Forensic Sci. Int. Genet.;citation_author=Z. Wang;citation_author=G. He;citation_author=T. Luo;citation_author=X. Zhao;citation_author=J. Liu;citation_title=Massively parallel sequencing of 165 ancestry informative SNPs in two Chinese Tibetan-Burmese minority ethnicities;citation_pages=141-147;citation_volume=34;citation_year=2018;citation_doi=10.1016/j.fsigen.2018.02.009" />
<meta name="citation_reference" content="citation_journal_title=Adv. Comput. Math.;citation_author=J. Warren;citation_author=S. Schaefer;citation_author=A. N. Hirani;citation_author=M. Desbun;citation_title=Barycentric Coordinates for Convex Sets;citation_pages=319-338;citation_volume=27;citation_year=2007;citation_doi=10.1007/s10444-005-9008-6" />
<meta name="citation_reference" content="citation_journal_title=Comput. Graph. Forum;citation_author=O. Weber;citation_author=M. Ben-Chen;citation_author=C. Gotsman;citation_title=Complex Barycentric Coordinates with Applications to Planar Shape Deformation;citation_pages=587-597;citation_volume=28;citation_year=2009;citation_doi=10.1111/j.1467-8659.2009.01399.x" />
<meta name="citation_reference" content="citation_journal_title=Investig. Genet.;citation_author=A. Wollstein;citation_author=O. Lao;citation_title=Detecting individual ancestry in the human genome;citation_pages=7;citation_volume=6;citation_year=2015;citation_doi=10.1186/s13323-015-0019-x" />
<meta name="citation_reference" content="citation_journal_title=Forensic Sci. Int. Genet.;citation_author=S. Zhao;citation_author=C. M. Shi;citation_author=L. Ma;citation_author=Q. Liu;citation_author=Y. Liu;citation_title=AIM-SNPtag: A computationally efficient approach for developing ancestry-informative SNP panels;citation_pages=245-253;citation_volume=38;citation_year=2019;citation_doi=10.1016/j.fsigen.2018.10.015" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="GRAF-pop: A Fast Distance-Based Method To Infer Subject Ancestry from Multiple Genotype Datasets Without Principal Components Analysis" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8.cover-source.jpg" />
<meta name="twitter:description" content="Inferring subject ancestry using genetic data is an important step in genetic association studies, required for dealing with population stratification. It has become more challenging to infer subject ancestry quickly and accurately since large amounts of genotype data, collected from millions of subjects by thousands of studies using different methods, are accessible to researchers from repositories such as the database of Genotypes and Phenotypes (dbGaP) at the National Center for Biotechnology Information (NCBI). Study-reported populations submitted to dbGaP are often not harmonized across studies or may be missing. Widely-used methods for ancestry prediction assume that most markers are genotyped in all subjects, but this assumption is unrealistic if one wants to combine studies that used different genotyping platforms. To provide ancestry inference and visualization across studies, we developed a new method, GRAF-pop, of ancestry prediction that is robust to missing genotypes and allows researchers to visualize predicted population structure in color and in three dimensions. When genotypes are dense, GRAF-pop is comparable in quality and running time to existing ancestry inference methods EIGENSTRAT, FastPCA, and FlashPCA2, all of which rely on principal components analysis (PCA). When genotypes are not dense, GRAF-pop gives much better ancestry predictions than the PCA-based methods. GRAF-pop employs basic geometric and probabilistic methods; the visualized ancestry predictions have a natural geometric interpretation, which is lacking in PCA-based methods. Since February 2018, GRAF-pop has been successfully incorporated into the dbGaP quality control process to identify inconsistencies between study-reported and computationally predicted populations and to provide harmonized population values in all new dbGaP submissions amenable to population prediction, based on marker genotypes. Plots, produced by GRAF-pop, of summary population predictions are available on dbGaP study pages, and the software, is available at &lt;https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/Software.cgi&gt;." />
<meta name="og-title" property="og:title" content="GRAF-pop: A Fast Distance-Based Method To Infer Subject Ancestry from Multiple Genotype Datasets Without Principal Components Analysis" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/9/8/2447" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Inferring subject ancestry using genetic data is an important step in genetic association studies, required for dealing with population stratification. It has become more challenging to infer subject ancestry quickly and accurately since large amounts of genotype data, collected from millions of subjects by thousands of studies using different methods, are accessible to researchers from repositories such as the database of Genotypes and Phenotypes (dbGaP) at the National Center for Biotechnology Information (NCBI). Study-reported populations submitted to dbGaP are often not harmonized across studies or may be missing. Widely-used methods for ancestry prediction assume that most markers are genotyped in all subjects, but this assumption is unrealistic if one wants to combine studies that used different genotyping platforms. To provide ancestry inference and visualization across studies, we developed a new method, GRAF-pop, of ancestry prediction that is robust to missing genotypes and allows researchers to visualize predicted population structure in color and in three dimensions. When genotypes are dense, GRAF-pop is comparable in quality and running time to existing ancestry inference methods EIGENSTRAT, FastPCA, and FlashPCA2, all of which rely on principal components analysis (PCA). When genotypes are not dense, GRAF-pop gives much better ancestry predictions than the PCA-based methods. GRAF-pop employs basic geometric and probabilistic methods; the visualized ancestry predictions have a natural geometric interpretation, which is lacking in PCA-based methods. Since February 2018, GRAF-pop has been successfully incorporated into the dbGaP quality control process to identify inconsistencies between study-reported and computationally predicted populations and to provide harmonized population values in all new dbGaP submissions amenable to population prediction, based on marker genotypes. Plots, produced by GRAF-pop, of summary population predictions are available on dbGaP study pages, and the software, is available at &lt;https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/Software.cgi&gt;." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8.cover-source.jpg" />
    <title>GRAF-pop: A Fast Distance-Based Method To Infer Subject Ancestry from Multiple Genotype Datasets Without Principal Components Analysis | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"STCu21tvImMoLKvPpd1wPpjco4TK4Lo2U4VYr_wGgtM","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"493745","apath":"\/ggg\/9\/8\/2447.atom","pisa":"ggg;9\/8\/2447","ac":{"\/ggg\/9\/8\/2447.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/9\/8\/2447.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;9\/8\/2447"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;9\/8\/2447"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/9\/8\/2447":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022c7\\\\\\\/c0Fny02UFDzze8Ox6uNiZrdOFnM+Y+RaxsZu5cNVchrNLpvXsEV6lvGmdwphxD9U7QXi9+r8I6mQ6mH83g59+phCbhBe1kvGOHRLKas3geWqbMnad3+f+wYHwns0PmR+7LywTZedhiitFTpVdbnE7Nbm4T9Mti69mXcXnQfI9mD9JZixvahrR0I\\\\\\\/c6RlcbcH6xrCqc46l+7YmELiRdNqSlCy3TYVShsPfGLT4b6cIMp\\\\\\\/LIahC8jxNfCdWnCb6qUpcyVQ9NdYGg8i45FQQGv5jmV78Gm6u7ZhdQyMhaoo1GRZ\\\\\\\/fDrmEHsgXqRnrU21UVKXvFy\\\\\\\/ImBWK2P4bnhyZO\\\\\\\/jSfoI9doRqIYh+6IbGCcGvdSSS7kV7OWa8tHxXk+lVt\\\\\\\/+Ilah6uZ5bzMpza6c4DhJdZMzHVw63ksYV8VM9460FEyCQ1fySYMaA8lWkJlBvDxm4ieCmfbp4IpbQ2gNamH6lhRLPSHuZnwbCxikwMt13rR22qh3S\\\\\\\/rxiE4sYXDYraEE\\\\\\\/i3jKjAphjsKZm8REmWbK6xjD8qV180ozjFx6\\\\\\\/jUG0bA0RDJ\\\\\\\/U0HqqviHHJHqgcSv6ubKMgf2EId7kGTMCG8eB7GMtvZSo5uIqVgX2QN+LycpEGFddIDCcT+4tNU4JbSjGSsMUIcpH\\\\\\\/Fyu8RvTLNxG+PiKdh\\\\\\\/D0HOog2PYih3y1qaiKJIKvWqnHrEg08WO\\\\\\\/um16eA\\\\\\\/sxmgxPb24CMqi9s4IqHGRfQMC59mifnjjbEGkwz14hSVW7O2rq9p0q5hc3a6QpBQ5Y+1Ts\\\\\\\/dYquaFP\\\\\\\/fu6FV9hTPeEjvmKeNKpuZlXZ7N0qA8BHE7tsWw7+LUZ81xVT917DzLRsdAqKJUqGQ1I1mKZJkoXoHJ1JZgQ9duVZDGTr\\\\\\\/NnlgaRBZwiyVcpEX2X2nugDn7KXD4LRtUI7oWKSOE4qA9AK0fblMnmGKmgVw2ZqxVEjDceLQwkLeNKN8PkLpRN7cZnB75Dg62sqBWRxZvqtYe3bi+ONBba\\\\\\\/wAATeu9RiT1xIH8j+q+R+1N0tYVaCZ3eWAbdZbr+rw16huNHVXxBWqGBF9IQb7N6BvvKWDGXBZa4J\\\\\\\/bcFrUuZ\\\\\\\/snQf8MEhZ0+DPSp3KpoJH3MjDIRJmzYr4ubsD6DJ80SWFFggDmK6aP7Sq8M07GldollcUxUV52+8FSWSGmX6ZoytRnfKAfdQUY8WYL\\\\\\\/2FNYKF20YNnvCsSbm6ZoawINgO9Jlhc9\\\\\\\/DGB7eChSzlakDKjdj0\\\\\\\/vhV9MJNkBTN32xsQdkS+Eb6KmKyGFy9npP\\\u0022,\\\u0022iv\\\u0022:\\\u0022MYWJpq6m3hj2UYmDlecinw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223b8f6f3b94be83966289ef50cdcff6a9ca665d452bfa6cb162389565258eaa62\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022GIuTCesKd5PJMh3xrx7UUBxta5JZ9mZQ9eescmZC0hBrzkZ5dgqnird4hCYhMMiKuYPG9om6UyedL+ErneSzFY+jO8zx1IFsgzoL4BxPEygWgaEb+qGBceOP+oz1jTRkjujzt0ES764Dx+FSYKJAwjt28lwJ81oYTOnvkpLRwcm9Wl63KU96DiK1oIGXiNdsapgx\\\\\\\/37MSN0S+2Pewg+ObrIFmu\\\\\\\/xtlrbj8H4qwHGtO\\\\\\\/8q5DrYSncwsGKKlZaqQ6\\\\\\\/0QWwAbJwhgjIDqckkQtYu3jxhDZdfOx+9MSjXBMmG7\\\\\\\/QQOvuMZNDjovr0L2yjzVTABOv\\\\\\\/NxbHlQ2fpj4sQIanSumStP\\\\\\\/5Z+f8XeLjTamwiKQB54aqrk64a0zRyy+0rIiYCI2W5IL3xWHYhrFxK2KkuzLuwHMQQCE3HvhzILYZOmGBlQzS1I\\\\\\\/lo\\\\\\\/NmEoEd6mBeuuIygeVBso3lH2QuXjfLYhEH4cjCrNBwdaym5mOu70eWKPEN7kFiqO1VOUfRO27UkYwefMieZH+hFV0xRC9ZqMW0aOdroQuLliYhKY+RKGFOmzUj99bWCgH3KOyUTArBDDkwxevqIgdevP8g0A3GIvqaPUwIT801hiGaIQOAtDzuqM\\\\\\\/u+iDVwE1alo3vXM1vmgt3+fWPF26RxWe37Z\\\\\\\/s\\\\\\\/h7wCyRwc6SkJEJa5Qapixpm3QvaUMs1u1VcPdiqwkQ2QTVdFZK1OVJruZsRBwlHX2Al6XB0fekMjV1f9eHIQss6TvELOZ50TPlqd786UkSZNhk\\\\\\\/aXHBDfo7MMeuaarj0NME5sw7DrbR+y+7jPgpkOwbLUBSR5mG7SrqrUwjw+\\\\\\\/Ds8+I6cMJhOizq5WOixIUuWY\\\\\\\/mmqnhKixi9VF2w1pZPOvRZs9o5c\\\\\\\/sx4uDLxb1o7l+9PqMlLTsXcwVbxqOtNWew2\\\\\\\/2OgW6oE8YaLPdZ1vwxYnjMQe21IHlXMATF2PpZdog\\\\\\\/Q2bfheqhdymR3j1bE2f4\\\\\\\/JKdn2dA\\\\\\\/Cdep14CosIBPPgxongx2kJbekBPtPknuSABDlk2UHgm02wHURxJ7Js2vlt21iZsGl3UVg5Z+I\\\\\\\/SWEY3EVsCfThpxIygdw8X36z2HMzy4H0equ+3v+jDINFJI3DuDdYoKo5FqFcjJskKnCJKb7as5hW2ziLYkoInZvw6Dzm37WnthLeSyYYS1QFKaRbR20MR6gxZJ7BdxG0\\\\\\\/sN4Zu4X0ha3sYO+1PeIQdUIHBm1gzPNGa7Y\\\\\\\/V6M0XGumWN6CJxVvX7RjM\\\\\\\/1bldGTikZRDO1ID+quQ6dagegRKL3RNg8AE8SRQg3RNia4mtwIJvgD36JdWhIyH63+BhAgQhAI+3x4rArTpuGXfAjP\\\\\\\/b0QHpnsRH+msvIBc\\\\\\\/4mPZXUMcRMJ80C0NTqELbMLI9rsyczJ\\\\\\\/PDx6DyGPCBIdDe1stxM9AxZziNREHwqLPgPvIDDMXZAAuVeAiDsJi6rX89ScMUUaGmm43NS3SUD+O5KQHaXk9nS4Y5+sh4tMCPBawtiEjPqgeatK8WufHuHGcZRAv\\\\\\\/k\\\\\\\/YvxEClusAKX3I711k9tbqc6rTALwBQXUuWAsoUhHnuTXCORuRyPTmf4FcZF5trA\\\\\\\/Dts0HbqzCIbw\\\\\\\/aKOeUSHgl90i\\\\\\\/q4RzgGvoS8R1Mwcpwd\\\\\\\/GVC6wGyXyCypcaIo75AIoNmM3NSxaHaZhcQ3uRg9TYzyHr79zA8AYpK3ZzS3RbDCxjSVNLqh+\\\\\\\/I2UmwkqP1c6LcX1qMv71iwfR66ryq0D63iR71XTYMxIeyHzJIy\\\\\\\/qrnaDbeAefmQsBTh0u0r1aVyTN67P4x0+fVEI8yRMfwphCbX4rUfQbACTZTdoSG+b7ZQXDKdN7SkxavalYe7IbrIDLGb9vffCCZem+4RNV\\\\\\\/hi4dED4oTqJpEl5+mxPKccMty7RJMc89UnRek67CeT80vXsUKzSz9hvSpCeG+919vK2aU41gsuV+V5gIfjxcfZsiATpRNWDXTkRptv2EjM2FbX8OqyJa8oiXNirAUk6Chq01Rt+MGNgWahH\\\\\\\/TJym5Fdp1r54JYP2Vccq2dcvn9VpNrw+okacGX6khHM7lBPnjxvPLptwghdNQihWANqkP+Y8yoE5A4DYfJUGFe8j0unYPL4gd6\\\\\\\/GrfvjFA\\\\\\\/JrRg+klF53+iufQ6XtCiIPpXoi8W+0V5ZMfQZdvhRjDDv8EpT6UNc5USJFj+V+Z8ttoCtw+gNVRHoHZ+DWnqcPV7Pfj5NvVL5Yh6E6f68OlRi8ESvUpC3JEOTPEbykg35dpw2vgCZ2jHUsIBK7PRRc1zG1CHJxkNzXlVRnZzIxW\\\\\\\/aEvGJuPceSMBJY6okaKkwh9H37Dmj4StMvsvXgYyF\\\\\\\/15fvTY5vtIawO7yw82per+d8Y5ciRol7uz57YU\\\\\\\/7cpIwR5e0Jj3BDxlq4m6JNp2oJCjioBxei2\\\\\\\/91G3ZX2oI4ew7nCy8NHjt0Bu8n1PkyoLYY5ITy\\\\\\\/ULygM\\\\\\\/\\\\\\\/mExlBaQIZfQkh35NRlU4xV1BQcYvl1DTDRvK\\\\\\\/LW\\\\\\\/YHLO3sasZbghyoxF92x\\\\\\\/AbjBTpV7hKTlGiIK8ENuXtROdvU2HXH3Oqr78t27rG98kCY2JIWTF2KZP6FweQnSaBhVDjw6NdOVK8edYNG5cOcot8rfzVU4diTG5MfEZ1W+oMq7DvRIkYveDR8AY\\\\\\\/SvicHzTWSiohGScY\\\\\\\/VOY5lr3zk2\\\\\\\/2mVfvvqgSnNk+\\\\\\\/TGM3wRzBslWwItNsQXipZ\\\\\\\/f5oRVD2bHxFkozkl\\\\\\\/1GlPHc0mN069P\\\\\\\/Xz13aXfpaImz7VBsCeY+ulx7HO1hM0U1Zz\\\\\\\/ep37ug120ej5Yhd1NwTBl+mWt\\\\\\\/8hXiNvu2KixY\\\\\\\/XCTRPurS2eIqM+vYqbdsuRgZU7yIisk3TSr+JSLWoqKYm+c3dxB83QBCAX4jXLi2jrjSOjEyU3wRSz\\\\\\\/ArZBjNeElm3rwpvbLQNxf5hrLJ32oJnWWuueHdmEp\\\\\\\/gsfjfq1q9DFXNhI3PWEbJn\\\\\\\/6KQiEx58hqekI5TrMW+XYrL40L2rIicasPLnsc9IPebi8dIx9jMXSRZ0SaqoBZlQRM53iZ1+6fMM+z+\\\\\\\/LSNse5GKT+hdVhYF1vFy8oSc5HKihgSNAihKC9aDsZvAJTLdPBUe9CuiL4r8ppOrgGl0k6jBjFWgDCgroITK+HzQsKU21Eq2WLV2m1hp1PnwT6dnIgsbsaJOgqfhudZKp09ETxScjz213i8xWQI8eyZrmoPKmfjAu07pIcBnhf4iqDu2tTmlUJggaNbZEEA\\\\\\\/qJ1mdYj6K9n42G0\\\\\\\/++aEJZNNVUkr3GRaIm35fL92OguOf5qHkx7yh7bGjUpG8ylG6sdgFDbd\\\\\\\/sJ2Cm55Mnb4IG\\\\\\\/xoTSuHbJXgIwxZIwGis\\\\\\\/LjjbWA5wvzpf0\\\\\\\/io1oyXnAzo0QQ8NVnWG4qX3OCeXXEg\\\\\\\/914h+anu+qttZ\\\\\\\/Gvim0PAgCKMg5XHm6Ebg+YWfFSRyUkN8Ht+Va\\\\\\\/CJmyKUsSzLgTjABWz2oeM4SCcG86KFi8UKM3N9W4WBE0EMtaM\\\\\\\/nbHPQ7jkA4Z8BDrRVI0NwqS8OEYSV1B9vKe97dG3ruWUHLsH0taSpm8nJTF8Ul\\\\\\\/GaY8bHZNZzgFnPUIdU+iX+Kv7u7\\\\\\\/1BZsJ1rZPVmhN1ZFNAn1pO5eYMiyEop2yjEWsJACjjnGPeXCAFzqtFkWQltRg24IGy7VL+5qVa4+ek6DN3VhB77kp1s39W5Gxkv9b+\\\\\\\/UaJCa5zR4EdWKm\\\\\\\/AulDQxoaviR4QMVGi8g+uI9RWKIuIM84P+oohOfekV\\\\\\\/u+65GogJ68Oo5QGUpL4LMNLTAHyLvjgvSF7x1mdHorHp\\\\\\\/5YKVbYw3IHx8D5OG1Utdt0J4Sa4Bi\\\\\\\/WmuQPHnOwVGXQlv2eO6E9T6wtU9nCVjJA+BjhNpWaWQWeGrCC3fwCHZEstuMCicQdR\\\\\\\/jQgPCaTY1pJPtFvjd1oU1qjeQP011NRbwrR3VgkqYbx+dwxj1much2lAcfaSr31+paOMfgOQnZO4n3oI4AKg\\\\\\\/WP9MFgWJTacwoP3MF1dah0WnXxStIDOaIVIhX\\\\\\\/y4uysM15CRYYixnnmn1r9JjcPvMXuaw1BglMA892xguQ5RIDjywjcpZCYXuuq+hjWRQFuOniGHQw6mtNgLv7Gt9dns\\\\\\\/iJxZlUKrZbdMUuk+lU6DrbyRnWoAigKexba6MMsxDfy0r2mJcgIJgvXIufZpNW60u7ZRwJqhLGxfCmAJw1He5YlV+XuDe4SZDkdsqnAIex7xVx4Bs371uwrKUKXaHfAMU5dUJ7bD3v4qJg7x\\\\\\\/MbFniJdVMN9sv06wyFvSTWKaZSRkQZtOTUqtTAp9VB\\\\\\\/OW0WXqBdy\\\\\\\/EkqODCPMUN4ndKjr7RXSS1h9aMhVlGTELOIpOKuBmIflbfJQcnC9qgdZ0PrsFB62O+FFgGUuJEdmvuMDNRAHwLnMBz+L2d+BbnK+DVUJrluiaS8WtbEm6dfOodOIhKjMXZLFfRJ9mfPOJhICIyEk8CHn8+TyILrmPy2zZTIL6rtiYzOS5klKdlpyOxy5pyZ9dx7hp\\\\\\\/dd\\\\\\\/DYWvdJ3dPQkPDpqvVTwhMiZVSLExuqw8WLNXVp+I3tibOJAm\\\\\\\/1qpkquCyoKowkLAKthCV0A\\\\\\\/Haa8YZqKUIApOBBkgEnIdAlExnkgz0f54TC8NKRI\\\\\\\/+4x4OLA\\\\\\\/dipulNK0btWRd8sy3kSIA5oLa93fDb5I0xLOWOvfkkCobBXe\\\\\\\/h4VcShqxrU2ifAPh0K3HyKQ6qbgibnxABaD8SVTDhbBj81fNQz6tepB\\\\\\\/IMCWdhlXKtitRfK4awTcjA\\\\\\\/gg7Iw5P6tdAn2KhIk92p17ih1uQAcP\\\\\\\/lBs5IqF\\\\\\\/2TLZBYXWByTGyAhXZm2VlsuE\\\\\\\/Ybj1JbSW4cxXOX7qhnkEtGp7bIFF31mp5540K10kWytD4l0P6\\\\\\\/fBknVlGWQGKji6wIn1SYU1+In5PCKr5XNgH13a4JcFseqlS99+pDy38lryhPFa025DOzr1Eht1SQ78C1U4oaXiqT2Q9Un1+ZCJAuCJ5kMpdhSmnNMR2aU2NK3n9gKNC5iQuPtWgDGeioiKKekKq+2WkVT4ifjLaowG8p3HpYUYqoeLgZudq1glu81rC5UNHgJm9yEOWW30VaUtt77T6uZSWpMU5RrVSJYxQz70u3fsGZ6DjDQo3K3t4teevO9mtSWpTPHdZd+TwRPur8AtcNI6\\\\\\\/MM0vbcDepRKr68VK7dv3wNyFjA9l8630oeKJ+fPq5EwRSGZZ6Kh8kZVLIH9nWxXhGRO2o5sVjYPHPD8ukYS6hDrnt1BZE+D3MGEueL632O03LW1cflT6t+1CEWC9gBy6VwS5D5t9isa8dfuHT4PBOAXyxoZot8s47rzqixB8YAhG4Zl8Gftc+0Tjunjh9nCjbT76a2GROWqmhOzhGTOoKpl6Got+LrFQQ7SS6s3Y\\\\\\\/k4ib1d0ODxGfVeiFCSJ3TD9AogBddpZl4G3Gv4nGsprg8Xl+UF2wHYAWGwHjioQI0v9dX+t3CW1+1j\\\\\\\/pbeeuFc\\\\\\\/TeMXGTx3VQFqICjcIyDdQf1PWrJ8FiyYEhWrtqUPn2dMLyY0l+enpbYf70Z5KBuRCbpIaPiuzIhcy9fNsruwA2wQHq6e65DdWRbQbV8v\\\\\\\/+SrUXI33hJgYdotnsLLMOUeqiZAwlm5D0s52e4ly1c\\\\\\\/ymJC0lkj4XFv9GXo48wVCz4QJhKx4vbJ\\\\\\\/CU68V3phJU8kPkgHgs7YGwpvFczSmr6bXBw0r9Y4grHa\\\\\\\/NcuIakaUmBOaBG8rI0hvAjaOdvUpr+2LwgA9Hgn0oTb1W0IPRktw3YteNe8sbN\\\\\\\/fDu\\\\\\\/GAR5R6qHk1AFn12ep6277c4nLenwARBHlHn60fqWNS2+jfL\\\\\\\/zsYQYl0JSp\\\\\\\/NdhrLxoe\\\\\\\/WitdbuALMZ\\\\\\\/4E1WeuyiDEFITyh0HxDGEoAbWO7jgJZvakMJhaJRFyHwqSHcrgpToQetcBT+Z\\\\\\\/YCd15Qbz4Yrg8WVmjaAARjC7YCdo\\\\\\\/GmwfFvFfroP98QK70o7VBBmMGqVCBF1mhHP+VhyV1HISJJA0kQ\\\\\\\/uUaMxDWENr\\\\\\\/Lj1\\\\\\\/7PWVJw9zX88rtK3TfW5IAKrJVV20UDq8X79wvzS6fHCG\\\\\\\/\\\\\\\/sCRXNa\\\\\\\/DNsHLwORfEyFFh5K0mecT0drKw5EwxzuEh1FJzMc7ZX8vROcrQux5U10JT2yYUfBvCMQn1MMeWYB251AVhQ2Txd5iMUhIX2u1F7pabX4TTx7+4d3CGVHbPFVyM50KNeq86WTijq9bUbT9IClHSY6H3z1tDUUoCOMl2IeQ8ZcfeyMsr8faS2J8R5hJKThRVo3mbuFIT8U2bM2jvZasinVO+v94rieVfB1xDUn7MbCFgMmzxRdb+Y1yciO+dt6F7m\\\\\\\/XkUxznYmegzyWj99OjEVTgbiOT2Gbb5TzfHmRVF6Wq0t8ktQDFWDGLEtrAwP7De7t+VqxZsLnTwI\\\\\\\/krvhbXopfBvwq2QnD1fDDfxaxtgrMwuAuv1o7FtICeYPix0RSMJMFnxC0lqo6RVchEUF6m5z8GuQsVxX1z4R1zq21553ujSVPxQ8Pd4wnoBHACX3ekuXTIxisQic9gfESCXvwFwTvElr0UEQmAGSc9g02G+dw672LdxvR84PFPL5N0fv1mmEfQpKKPl9M\\\\\\\/M098uRhWeQDtOMEzbc3zA5gSAOpHTLLtBADH7a6jJZwaUQMIK65Juz30F+0wbzr+oKhaBfSO\\\\\\\/GXyMms+Kw7PnYR3sQfhV2E15r\\\\\\\/+fIcjm1T7tFtJe5JjHfysQL15XSrv5B9bhS22mNE7dNQI+0tJcwHjgjoYwIAH9SbqJkRYrH0CLDfLidAm\\\\\\\/aXbGPXWXsEb+GPgejr3Jo3+y1wxBukt1jznWa6211E2Hx3O03S4yHpwxTRWUGujfElfl6LB7OiDfpLeIKa0wju53psZ4UTtoGBtg+JOluCgT8HdIlQwefKpbqFB2FoWjaiUT\\\\\\\/lRpNx2qk3ZOjO4UpEQqkTqqbvBT1qlairlIAZaNe29\\\\\\\/Wqpnb8tLjabhoyYzop+ERW\\\\\\\/gaJKyiOTurABS5dICU2Ef08A2c1c2Pty\\\\\\\/1fm1L92AXnNAh6cPDOi5V5pz2mehf1DF1fci6TqvmGY\\\\\\\/d3uBsPUaXW1lDSmua8N4Te5fqq9mZ4\\\\\\\/m2MvWF9rlnT2tU7zBWqwNijVJ9TUJCVHX3vKXj1Cukjm8J54zbRlOXn6GOKLoLYuMwNM0qqxqNoeOebTPGhWxaQ2y+vGiWOg7yjaNJKIq8YVGueid4bNh3kVDSVAA4rziIExiwwL9e0F6Xn7QKU7yjchWgMW+CFnnRDmwMbG2YBJwYO5SBg6E7p2dz10bK8HhX6hAQtaUUO1cgx0\\\\\\\/EpohvybKVuz4BfWoDlbyKosUE6trNxrFYqLVDsjzFMBulFr3IM7zWW8Gpw8Dh3mtBLmaEB6Qv18rSPDS30YdgaHbD9jBAAdgrFNS9r3a2uzjENm2X7iR6KeTFjCLC\\\\\\\/lHKe2imMZ4QqVZEcInq6yg+Kt6WhxCnuR1g6ii6GJ2DAhmbhmSZgE\\\\\\\/y+PpkAcCEFg8A\\\\\\\/CueQiOAE\\\\\\\/ryliDy11Jsin8MX4povceyCXGdDZlwnLikgyZ3nD+U4i+Ew8NsxexhY+mlfTkHrETW9zxKA0ENB0BTPCk1pmeNH8lwYiB6juQSZKb17A+lEtpl73Iy113cIFbT4ju2vY59cYW300Ft8P0rpmUCeYzQRh7GqINjuaw9BrD1VvxCth\\\\\\\/NKKvYtTS2Q7Y3KAx0VVcHPtgXfUrKBvMof0YfmNyn5CoIFrdOkM08PJBlGVyC0HWSlnYz6FXTXy1BthuKsdUzSpLor6snT\\\\\\\/+uNyokrbr1W365Gsnp14Cm6FbChmWtxeDb9DTAKe6tMK9xYA80MJGq+bmFHZ8N3uctFPUUVu\\\\\\\/zpCz2m9bJIZQhNIGy9T7VI1yRECMkzDmRKqTF6INKveX4l3uZQp\\\\\\\/7WlxAWaA\\\\\\\/W\\\\\\\/LbkhRUi\\\\\\\/eSnDJFG77uTOICl0CRq5nA4i4y1wB39sY\\\\\\\/6gk5mbon2WLYjeKE60mqRKEtCk5JYcrbW8H1M6ULjx7RCFi1QIOtIRwtggKJOyrUbosBHrlipKIyWZ08FUClarbUMVhNUUDwYVbfZmFhrrdq34UmYslO2JlgxhwLoE+8oa2k9q6Ai5njUgUqjHynvx9fQlc6JcNipwWdR2SAYXqA5ICRLnNW6aszHW5yBKjZe+cCIvdVTWWidksp6mwGn9wj2gbl4etwomq3lDRxnFC123OAKlY+3DxexOvuxX26y13VW+bOSEkIrB\\\\\\\/RSgawzMuFZjL24pLGOsFUkP7BtCxB0J57n8JgQA0wDboMzCsLRZo36ClvqhxbjNzdfoLOfpX0DsTqEVKuZ7fWqFYU0SK1gFHWSXKbqdR6QP4\\\\\\\/ZSMuH1iwYp7xZIfG1ZMQEhhGiXVsQFZKeQsRT3PtGHxpgurMtzFyKYxqD8r8YnvRz\\\\\\\/EYAsA89kpv1lZrk7RuGTK+6kRUJnFh1tcbJzjoC3GdHcD7QARsCqBDd86rUGGlvkWUT0NKB9YewmThPKZCX8YfzV0O7GQ3bpHurFQx1OLFq4sXwtJlFaBrWxNB\\\\\\\/qqohEbKRdk4ZsEWTHOuBYin3KNR2rQvUnnbO\\\\\\\/GKkwKQkIqPIvTIdmEQ33QG9kgBjnwtA\\\\\\\/dYSwAfxAjMBtmLV\\\\\\\/3YB8IFuY7CAnCZQzjeGkKPUjr\\\\\\\/HJXnOxR9NSe+bdceHBQlZjN+yN36fe+m3DyhMIPz065\\\\\\\/azKviGk8q9ivhFIt3GmiScUf0B\\\\\\\/xhRcgnUqf0lo806cTtNUP6dln+r66fI9L+KQ3xwW5aPXV2nzinLAOneZCyQ1+VBc\\\\\\\/xEuBN92IeD8yDzXcajry94KQoslSTRe3wTWMn+TXn\\\\\\\/CTBiLBhnUZOM+\\\\\\\/dgsjxel5FfQ6STVlmkQzO4+BhT4KThaBOh0dZ50PCGLsuCPR35VNUFjL3RpIRO6B5xgjChXmVdzQbDi8clBK4iIN8it9P1ZClZpTNTYWiykq2u6tJ5RIwQE4IScWmS+IgtPQaZK\\\\\\\/3G02TXFMHR7vMN8+\\\\\\\/BgprnZwRXceEhX8ZaFSv21Qual1GMBq0sCHgmSUN26ASMjtLkAbP\\\\\\\/+rkUlwjSSCmKKpuSGMyRINrx\\\\\\\/BZpvydQfdWNzZSnGRR5xAzpK\\\\\\\/LKVPiE6dFwI\\\\\\\/0XVaDicGWaN2E2tcLaaDQOYu8Eik+4yAl5BBbRGfE5vJhO0SfJxlboV9oty6iowXNDqGF9OESEPippct4Bg96k7+mZhynUjwf3Q4TFokwUTlDjScNGiWkbL3VwJl1fO7xOkQKAAyjj\\\\\\\/b6Q296DA\\\\\\\/xh4ZPycRm4O7I9ZpfAYgDxztyYb8czszE7Ux10yDiRRmImur5ykTvBq2Pt13x6oCKJHO9n4bvC1XaxOpQfiXZTcRy97dCB0JwWuzeZdgqhV4ei2oJBQqOtleJsfPm3lXCmMz7+Ul9qPvBXRJTNoxgOTro5sOnfAMhMSUG4kqMC2xp40\\\\\\\/LuD69urBlHS53WrsjPESqSv7yiBYsYiDify5CBkborDtL7gvmpbx74c27EfUNvuK4WmyTEuQPAgaRpKCt6saMFXW6xUbR6auGB7b+sO3vEJEWg2zFKoU\\\\\\\/qCgYjBkBlQacd7pGWoi9Lxj6mtZeA4Pmop7fNXVP8MUeFu55Ly++29LY4KMm1rO3ouZIF8jVsVEMj06fz9o69ofXToV3tLqYhzCz8BDrAXfsdCgwS\\\\\\\/fFBndFuXSMm3mQ8lOpdTKMbKqEYPucKxP4iwV4goCk7judiMfdUf3znApmWZzF2CehPMsTo3kN0bq+8N7h6tlTG7dxKf6W2JuQwl8SRkFydryiWud05vLAnBinJJcbwYS8fhd7jiRTF50D5DMK3fNi9qD4H9xTTGysgSTa5DIXIs48u9R0PW4YPqaBrtvnmZ3rzRFzp8\\\\\\\/HmxCwRuvWZUdARNZuHHR3\\\\\\\/\\\\\\\/dXj6+bJZUlEweVK5tx1soJfp3uh\\\\\\\/oRTWyTmV8V40zR1CKWHDEyOL\\\\\\\/HRyqQY5SeyGHtiymyrHt1QfFOZtqdpD5b3YYKqjzDPuGLImQ35I6LKMmhCzvQfnIcMseB6Q14XlQUOwgu8l3mZ+CxbNa2L76+wGvgbtxkvyXEWZYHLHgjnxrv\\\\\\\/QWiY+VvNTZ0oIA77RDpnUJ0zdJZN9JkA\\\\\\\/TP732DCg5j94YUcBtU4EasRy6EdYPaiQvrfsTmkTRj7qmh6\\\\\\\/H1pah3mtSppib2SUtdr7bwej3s0BlXS1JPhkpyVEREZ0xRRAND+nFEgz166VBV5EJvzChPWxtbnmZe\\\\\\\/4G\\\\\\\/NfPrgwTsOkYQSq5Ig3U7mqd1eznu0K\\\\\\\/zXhojDFbsIM8pp\\\\\\\/66k2u5bPFzVWHHVMbKklidjBqQe8rxMzMO9MvoPrh81qSzgsx0qiPRpFOge05T6W2QkFgDt6ezXUqE7ypVIu7zw21+mm39s07lxpLYTeabiiuQ28C7duEaH4XZOcYMnoUFplMMMu3npnv4g+nc2lsvEyGcpk+2Mhie6q6\\\\\\\/6w8sR8sMBWKtrdIkbkfiTbxnqJxHre4jyCSXNHDOQThvMnRqgQk1tr6YUQWeJfvYq4vsQbwMj+8MxJ7bUaJv4RSlG3p6iZhBeOhGIYAf90tCxKYG3qIo3bLP0\\\\\\\/M7k9T97hsGzVbK4uTaocQs4ouE\\\\\\\/NgYoVDkoKDeYJeAkznJ+uVqQiNsJF9o6pRyFo+lVVclvr5M0BLa\\\\\\\/ue9eCWfUTczYk7PKzwWrGDWgJS9Nm+52bd5kNkjpVh1TbBH7Lp4Boy2O2rnd4M+VumLXwb9BRB4qoqE27\\\\\\\/uWCjbeAM1xZSwHkF9yzwt7nI2R+rZsqH6qULLnaIiEgpycmcWG1N+4k2GDoAFCXoJqgZTRePOG5TnIw8iWGEgJTJvTe5dEXOHXJCU5XWQI12rB91S7DzYuPHWj\\\\\\\/EGfG396Dm99BdlvzehgxmbdJdHfp7zqofEqKsjonx48Cr6zsTclJR6KcuIlv8fncv2awlKTsd2hhLmyaGUan8l3NodZH953FaCdWW1iyWM+\\\\\\\/\\\\\\\/nKtQqga7\\\\\\\/+xDOeAajkZqiAyZ\\\\\\\/Xidn0z1NKMeru\\\\\\\/jH5L9KZcGZdemPwdIVH9oPKgFx8h25IO4\\\\\\\/\\\\\\\/mh+d+FvxarMz\\\\\\\/cvzwY9zIVygTmWJc4k6tkldWEP\\\\\\\/qzlQqRwdjKE\\\\\\\/I91yQSdq5Ry494IHAn8e3NzBIVGzxDtxH100k+xWN+O6EidacYeS0swPoV1afBgNBDmZG8FhvweD4NGIYwE0efKFovfgyK+EchrttjNxgRcw2tTrgDa8i6OR7KnIdhqIh9zAvQsINRg8sBSqb+V5vogCP6RxwvUswrhDActXwq0ehvQ+POlanLdywn4T1NR\\\\\\\/eapjQmjNaUkag3UqHVWazq4yeuL8OMbkV\\\\\\\/GAKWVLJcKO6ku9AkcTTcjFCUC738\\\\\\\/2iIQbdg6BMtT32WGdkJ8wbTeO2a9d87CpKkHde4ZgNAUMWlt1kC\\\\\\\/eyWsaxbIvlBaIKd2HVscn4oaX9ZspPZrnUstejDnI241uWJjGL9j7uWNo3bODXUttjkw+5K1A1csNagnntK3izbhH5kttc5yrv0w6xAMWSnGLnD4\\\\\\\/Nyv7jp49h6sITzIKl+Vly65Tyb\\\\\\\/+4RPmcPJRtnvEfoPJ9wsQf2Nv\\\\\\\/Pi0DcKndXOD4kgRF\\\\\\\/AymObhqteLyrfFJRTBSkPYDwx0vnx+qqc7wYFFQkSmeACAshhEhSosFHXcEvb3KiNMPTb7X\\\\\\\/E0YG6PpdWDfq73zQIpKr7fdeaNHczv8zhEZJ\\\\\\\/5nSGtZWv4Av8OPEJXtm+QM2rsOPeWszxQiKd1xrER1B2TQ4a\\\\\\\/iZ\\\\\\\/vkEzlsQw4U15F43+VpRLm2ibElNhKz1iW9ZC9WLCQlVch6L5BXLvlu1FUaOcCEqQeW24Cd90AI3SP+GigY9FOHQ1\\\\\\\/GV7dft7Yop\\\\\\\/10EMmKc9FyYe2vyzDl\\\\\\\/E9oIIgDlsjSuROQbNIaSWQnIiiMT7KXQ\\\\\\\/7Q0vLhrKmex3ny23IEc3d00aFGPEjBMu10HJLL5GL5io1EbNxGloKqHlOcDjJKFmaPbpUW2ZbsMn2mnQlw0M9t2kaA+0LqpHD0fnQY9PuMkCZ3mbhv6RwgW4ilQRs7siOdJqUmoz+gb+vtKkx9P1pceTyAys05I8OXMVB6Lc+H+fQch0Fwdk2n6fth9T6s0Ag\\\\\\\/AmBlXfjwSIweQpc6+v+ifbqypwhmBImbTMLHNEQh4nHq7PVF9g9l888fVua9Hl2l5id\\\\\\\/w+86gZBpNk17ODIJfUP4TkNdNKW\\\\\\\/if5s5DpUOHIliS+0TdWS\\\\\\\/8VzCl7hQI2QfQsHqhsHpeBkWrBVy2oaaoP7u9txAIaIl6VtcrVMlMVcCY3usifSIay1zoIQx\\\\\\\/YBw84O5jO578WLwqQSODp5dyCj03Z0J+zbwgvUqKWUKuyMCvNu3+YU657zFd9FA2oMfNlOhzB2qBPOEktKzN2kfVwdQmLzWHWqKkisKWgGd1k9KUgECGUiRGZSmHYpqxFT97+Zd1uelxTktb1CUDKcBwhNznWMmpThc2S8WnndrENPMFcS298hsvBsg+Iv3w3DD8H6m4yAsqQfNxPTDEpR3anIX0eaxJksFHdebVUDbXarIlyHZEWzvHkCdijLGYVHcJ+NL3vYnaKsaDpcXdgOpr7DS\\\\\\\/UcfW6b1VSivX03fhPA0JX+5lpLpalWtSC2EgnkZscWQiAZ+9U\\\\\\\/WQsA5boU66H4ITGe55ZvFKH5OrBzeDqN7XlOjL9FqWEC8DWf3qUn7Z6cda4UTOu\\\\\\\/kCDWATwdpKCGuY92qY5t5xexNXT8MQ70SlorYJPhLazqraiYXZasETxmrpCHVkvulvsDSDL3zgm64JSVrLTbzybrjIZXyNva5lK5FaUqa\\\\\\\/4a7mV05RhGqcFyyYanBDY6g5TcyFhyUFQEhhYcX9LMakO+ImhGCpoKJUl12\\\\\\\/f4mXJA1H27p1Q7Fh9iZmse7M5zKSj\\\\\\\/aoTyYsXO4G9lfPnwqWr7LrVld2EUZZoulLCFMW7V3xTpBXMC7HFxB55wrFkXdoU3\\\\\\\/CjDl2S4U6o9rJAcqdbBApYLByViiY29sD9oM2LuSlVwXg1YsKgHvX25iVVFWadHuXVdQEq+Iuw2a0L1NTdRd5m5O\\\\\\\/v08Vn+wkQFllXPTjApVKyy1VBrQMqzkj7Q8acByG7F9RVnE0ws+MB4\\\\\\\/xEv1Hls9gxZQx9tu9IAQflFFYIdUhaEy2+8aNalDIbp1TJ5Nk2J7A1IknWscVcnCL3erldgqqq\\\\\\\/VJ31VpxpAisIl0vwe5xYIod\\\\\\\/bdOGpp0UA59zOSI0SptUXENjfhm\\\\\\\/UAePd3KtLlJWW\\\\\\\/5k52+\\\\\\\/A8zmjKstAhPra9u0eJuKdSO\\\\\\\/wW689eSA\\\\\\\/dPsI1uq+naLZlIPsPozF6mtsuBA4K+gv1pLSM2J5HAbCViKf4AggO7FtCwUCcPDlk3tnTsazBsYV71FaO+\\\\\\\/w3XWs0GhYPLxtgYmEM5hPCMLos+YCTClw+I7ZdpF9mBM5600vPmjf09O1GgqJbLl\\\\\\\/7JjxNRkekwUH1huRYuMQq+olTmqCN+DtUSRfiEoV6AGc7A\\\\\\\/hbeoEJ4wN5b2WyC6hp5aj53d\\\\\\\/IvdViZtGIFy7Myjuw4QGGCImtFxIWjZZWEzJjrVnVsOVsiB4WtQvrWK\\\\\\\/YLHneuY1ISXNiE8oZ0APJWfS0vw5xUgFVqrUogem0oyCgKdQyhAdfFhy0pOftWKWnk7Gw4ZURL9h8o8P3VciE4orlgN08o6bYtoVDLQEkyx1RPvVJOFkhXVBCtYN0VRKonTGTETUZ8cyrt80HYgqeFU4ujHj4S41Nxtv0wdf7JT6WrslGEooXNQGlVSOqHvjKa3xh6Dagu5Y3SwhBAPlqEsImX5BBPB+iM01eAiTznftfPVrQkeH\\\\\\\/fEejvY5O1sWl\\\\\\\/cSgAGPIuIoQHgG3gBMqNAn0xP3qFvroN9uqqawJk\\\\\\\/hFn7h41Eu1ewTJviy+xXLKojsl3wil5Psx0ea5CAZ0WGh93fUH6k415kIp35xH0vxGBQ3QMyekyCS5YZcB6nmDaXxw+nAn9O7yiwR5Vb3AsNDDSWM05rTLSOptAo89rAEHS\\\\\\\/btUf8yTKq3WrlQXNO8arDlWq38iTh4ktNBk5Opgh+e+v9m5p+7OX2pGwAiKZ2TLGY6vdzZknTcmGAUlyeNCkZVPbup969jizmKPYjgmFKVDYRjn4A+g50xoR58kqkzsJwMhPyY1fX8JOejAFOR1AdBM2q5UbDj0ySrJJlVbQBSupzTTd4zexReCLLgBZd9QpPO6gHPmxiiadT8Wh52sDLgTB12Wz7NUELm2bThzCIRw2FLhapc3\\\\\\\/WimKEMnMce0zjeJ2xDOglQxZ8EFKFm9ZdqqZjh3BRBZKvRrhYm2s2KIYGcMAfOyuIHxTpI4gbYyfxgLtV\\\\\\\/2YfOeH0+CSNhN9Abt5dVBEbDSmXCmDmqLaZnuPEd5bxGeCbhSygRnRH5IvpjSn8FCMkVsFO28smUYLSx+j8R8NS4JPI8NYCk29l4KHoQyNxKmLlghQ0m5WxaH7ltaTR3xQN\\\\\\\/YzBxkkI5HwDa7QM1EokG+St4+p5VC57O97+GxffX1D\\\\\\\/vCDNjk1yz3C9Afb0PP15vJvLew+BH4LWfCWBC0HndpfSXvlWudUkDzeeNgizSggUqlJPApEdHCDwHgYGD+EPYPe\\\\\\\/BuNYWMgplJNPS\\\\\\\/5Gktg23YvInlhdhzvkc3ROzj+6cnLfZ7K+QkPZk12uKNEDFznU1qSuVZk534ai+rdhT9BF90t1Z56qrNBsKfi1\\\\\\\/asZvmBmXI7kI03Rnnxzhty1Q1tYFQWA4l7AjIxj3W\\\\\\\/I87m7Ik0NDVn1HJsj0axZ7XXIu8v2oc9Kegp13hSgLkTGOiUmjtX5IMqk4l9hoWAoKi2WyeDOvePvrHV65E3C4FmT6PI79ASUGmVFMYF\\\\\\\/vEPwo0njwBDq7YfmM8191czv6ExPqozNEUZWQ\\\\\\\/tUOnx+BPXbuWWHJZ4j7El5sFA7CK3paNJwHZgH6wFrxjudAJ11laP4FChVOo4J+FPhwal+rG4KdfzfXmw6iSrpnpyp6Flp1JjTEByBK3FyGIYXHWCYXRNE7bq5n4QIbNo9nPuXtLlFFjd\\\\\\\/LQNidfKdcli+U699ydt52rrhy2rwaajfbFM1khABP\\\\\\\/DEtWT\\\\\\\/0ZfzzwfRgVr0JicPVNkZb6Rs6O1rnABZ1Bt22xphymE0u2QMzeWRhGp8pFu+aKMybqWElSetKN9zxY\\\\\\\/+htZdT43lYA\\\\\\\/yzJCEGA5mQoJYN\\\\\\\/YZAcAWGSXLg8FH0dsd7Bt2qOLRQrrYC+sZI1iYnJkLuYMUtnrlz5eii9rujnzYpYct0A30L\\\\\\\/OhUOVWVqGzUT3ISx5nkwMCgLKKRxYQlk71upZKNch36ciY2+MkNW4R+UfbpgUrafupNdcaePJegpsjemqwMvGUXhvmnrrRMnIpwr6IUl\\\\\\\/Q9OOvMk5uTXSO2FGSJuDSqfc7aw7XwdVRvt+KFQpXCuXf9ZSIUd4NCpJUBZSBJFgh2\\\\\\\/W9\\\\\\\/IjCI2TEyi21v0MKlQQuwfuU185rOokqLf123sOu4fUThyZBXCjE59R\\\\\\\/y1u++kS8ZYOjNQy2AJFvh+o55RjmadbatL2q+I4XFAl+EQYrZroYP5FX0hs6WZLBbwVzsSq1IZzezil+UQj+B7Ci1ugtlDqaau\\\\\\\/r8SAHQTQNttXy85nIs5EJzSJoX18sFuMCnr+HpHOhm\\\\\\\/LXK5QyQpyGvtnm9pMzrBE2J+AUZcP1fovx67psRyNnrHW6y6yF+bEnZbWAEdEaKXyBS7CelK5vM3OiYR9AUcJJ9nr1+1duR74qRR0AsDghNzjvX6RZpIHyY8BtgBceFLVjh9ctqHIk+5Pere0BU172jaPLwSueK43Uf6WosQvsejhPX+rTezCLddgsyDO\\\\\\\/RnyoH+12j50Mstv5b0HxNpkoNLM92+jBl2f4lzB\\\\\\\/hp\\\\\\\/TKycoqxOz92lo+cM6S33W\\\\\\\/nHiMKRgBjw4qwvrmgMEdNJP5L4ptg\\\\\\\/811x2zpCNJEJT3SOlUhzdCfJnrLe01ypf\\\\\\\/ch1jhtsaxJts7upSlgzAVMWrF61ojrKuQbagEK6AWNYtJmC7uiQh6Peip+ORKPojZEF+QZwTY20r91NTKCgSPSXdarP7y0oLzd\\\\\\\/eEONbJcjSLoHSc\\\\\\\/9c\\\\\\\/4I6rMzcJRhUWEJ\\\\\\\/hUaiSBXv4tfoa\\\\\\\/tjVE1JvajRlm\\\\\\\/vPkEInoBtJ4fYXo2LIGzIJHdZD1rNgpyEQib0XNtKXL4yAaAlnumjkEVJTiyS+WpbaeBUoh5C+dpaC8+fC0vkelmCYwJOcwf7WkAStofY+t4UOzDAtfcoJjWXCZyvUC5bGwWCrw8ppopLjShdo+y\\\\\\\/Zq3LktaLx9lm2j7fnaZLGu0+FCjvWHiY0bqqaM1YeV\\\\\\\/077NaPMcUMm1lhvGmdTCQQlB9zGGYX7xO\\\\\\\/w1twJdQve7Vrl+\\\\\\\/QhP6cIAaJicx7nnOkq05rzoowoURVZ2L0qidE4MLzdPabtwf9ZPVzkpmzWVCPNGiEKspLLMhfIOXgwGA7\\\\\\\/l4GS7nS9XZ23FaEfx+myMXnHFYRUi0XrMiYxs\\\\\\\/bj6s793W+S51RXwsv2kdY5GlWuBpzLwfkeJdMJu3+d8zpl6B9V6AeCKFRmZK4sb6VwJhAEUmMC6tw8gb9I09YH9z1L4Mx\\\\\\\/EPKZpRshg9+WHi82sL3k071Fy2QU5\\\\\\\/N5T80KCa2Ks2HttKGldUtYaXBvityiaZ8g4Ag11QULnMChLpO7WfsrWUtSOTkmy4g5LI4mM3FuBelUuEoUr5wXFXNr5GGOXhy6BcOGmqXYw7S\\\\\\\/yXQ6EvWrDwMgRkjmdEXTrdqLO86IyPq9QuG7FnvNydT06Rb8SlAoRvzRiyw6S0ftJ7b6+oe\\\\\\\/b0kCcgTsYi1jTbqZtqnbfU21nHQk8qOCLPsDKg6ClMj5F+9JJQJusZ4l1I7mYyvoUAwbkNbikG6Y11m+D2xq+kfUCRobCETXW60va4n30dYO0+HBCsxWp0f81JQdlN+RoE1VLQ9T9V1LJiDzIoJJ4ecN98r3OKmmHJQXQeDRKaWtTVj1ph+qeL8Bhu+a0plv0m\\\\\\\/Wk362M2UnR3m9YCpRrWbelEU203RYQnAGwOGLsryU+DanFZE9Y5GSRDI8KytuQuoH8R0Nip9Xo31V5RH23S2Pg5pvwp03rTAuJZ\\\\\\\/CmXf1gPMYcvGGDz\\\\\\\/wh3yThdQ305\\\\\\\/n9Eui+zglnPolexkOQIXbMr10yDXPe8wbUaWTmy5R0mfikJL0WcBdsXIbjeH1wPsLeYt0Z1gN2xNHsLyRkIObWaB01CmgQFU\\\\\\\/DezgwSUm0zAsCg\\\\\\\/mYpgYph5uWloRvWqoDEL06BMkT9jRm4cGAWlYp4YLpqpKDp3gA9TBTP5bDZvspkcJRp9PNMYLvo8t35tFHK0E1dIk35Ydt3hAkgjkD1A1zVP6evzJ\\\\\\\/Y3DZkJtI4X2deZuOqlqZ3K1TTA6HS9vuS8dzoODHraZmkwOAeJonxwLaLlEM+FA6Hj4Acw1K9UBdGJVtY6quXNNV1i+8pF1PeiyCMPiSbeyOqbGJUy9iph4mE2k7Qrk8gOvmqHOR72nSuHjkKBPpEPNoZ+JRJu+l\\\\\\\/EElxHe28qlPnMmq04oWNH3PP0gWy9et70+86TavSIgc+861xgM0BytzP8FKCrp81lbVinLlDT0LGQtwD35RCZkjLpcUZ89Os2PumPgO7+3+CqCb7VaaZddZjvTw8pEYKu1YyKkZ26bveAUbKkbQToALCmLXNiw5uHyWGqjJatbIZzS0ReUmJGOAnXH4vChVVd2ij59pj5EIxfORTDbFGqTvmGd\\\\\\\/FGhe+wwT9cYXr8mwvvizycENQvFoyeaaZ1D7cdIMYiaqh7+i1wxVkxxOksRK9zB7bDQr4+KvO1fyHZjcGZcHFn7XijPL\\\\\\\/rm3I9LBg9OGK5ET7c0n6vhc+1+JnRZO4RpFmM8QT9ZHb0InJlp\\\\\\\/VI69CisXGgEBr4bKUq5n6s7h1zypxNVPqa0ZlvvbP4O2ZgyN41ewCfvq+XE0Q9M\\\\\\\/ACQ0AIpIuTZRUrirEWPJ6zv1xNLGfvyvl0\\\\\\\/Y\\\\\\\/kx8EaQtz6IXzLmd5SbZLz2GdxNIoVa73sQ2UtX+Q6CtvZNaxm4Xx8jPzFbC55eJayKmHfZn\\\\\\\/IysWJExIUoBj9AHGaOCJ7Y0kSCvMJnVIxSuUtI6whQr9lR51P0jpfkNM7t\\\\\\\/GUtSZ49XKkUo1utHypXaNd3uMJVFB4t8\\\\\\\/zTAsQcx+XqpiVbqiqi1IonNHw2Il+JXVPRSOlAeb2imYIXQnrJm0fYaS8w5Yqhn7IzxOeNSDNrdBLE0Il\\\\\\\/403sqiQ+fOjM3VsK7T1LoCop3y7NPeNysn3g98mfTmkB13uHAI+wT\\\\\\\/k976HfTonQt1u8D5kZGbiau0hIeYbjT8hpi5uLKJGFl8WeiVAPb2MG22WYyQ7E9QHq4oryQkP4Afw0t8nnUxHSTA4\\\\\\\/bxznNru7CFIG6wUM\\\\\\\/kQIygPSi6SZjLrbaDkOJ2s\\\\\\\/tQtpuyjbQPlIUlwLQ2Lbtz1DYTefPGT1w83hOecHfORMzCBLIZBSOWiu\\\\\\\/AF9etQMQr14lWTqDCywKoJFvbGz3jsjkDpvlkxYLZXo0+y1\\\\\\\/OrKQt16GjPta16l5p8yezZGl1g0CXw+IW65PpTpNIt\\\\\\\/o9TybLrPS5nUq0YJsanVOyY0Savy8oNEMSNt8Lno3wHGDc6i6GeYiC\\\\\\\/R+TEIjg6gHrl+b5FND6NZ2bSlHDRzFF6Y8OJqYJRiVGg2\\\\\\\/N8lkWe2DN59YqXlNPKiYa0YWzLldyzLLOLztCXU2VRBcANtSVyBZCfTyse1Yrq6n1EUgdiwZV4AnDaFPYx8CJ\\\\\\\/++EyAzrJy2ZJqrOwQfwvXoxJWd5d4VcNNjCO+0YlN\\\\\\\/XDn6nggPEYrproq0L7cmKte9P9LNuSCvL8\\\\\\\/mL2w2qUyaYcrC+HxwRWDXEme41WLVtsskuMPhMPA3cblUjOgJ\\\\\\\/paX2GT5lNzUYwUByyqW1hXu8E7Z+ZOnCsU90aIJBXhx3a4ej+PxcLPvt4k59TgDPZE1gAlBEYwz63Z2xxKTjWGafE85+eCIUUsn8w6FBvDi8fbWIbP+kr3vIPOA1MgIGS\\\\\\\/v8gnOwhqtI3npfQUqn8Far7xgncZAXeFTuuWENHcFh7g8sGeUjiufaXVe9XH6TCoM1UOEBMi8OAzxKoIiQgxQ6fYWmYW7Dzr3cDX1bOgmvwb04jkJwzs+3GrhieWeGzhHE1eVGefO4Ibq9Nt06FL1h2NT1WCxhphqCFcpZQVLFkLeUgG7EB7Zw\\\\\\\/x7jzkKZDhPS6b9V5mdoHAvP58ghm2NDkPCDbOh3Fu0oO+mp\\\\\\\/K2vduRUEGtHCBZkjAEHZFLpAIsGuIBavVxwPnNkRqnnZCiig0TaLvkDIhMPUovgvy7HAk1DpuOs4DMMbzboNMpJlTQTngOq64WyL6ui\\\\\\\/OB13aoN1CHCUwD8G+FOHouYgyyLR0LidbC\\\\\\\/aSQmjlEYduR6nKt+S7xGPkcB5bSfSE3TSAuIU8QnouC73khe5SS4lZt2TAy5m74mIuegcjrKtPl\\\\\\\/Nb5iXH6C7Mafpaxsdjj\\\\\\\/svg\\\\\\\/uI2i0W3Du+BCyn+aoJTnx4txC9UEKifxpGnuaTTVLeg3bVTCwvWMtOwyBBAfC1d\\\\\\\/NLJcdZORsRQ1sdriSeCFoi+7hwCBU0FeNrw\\\\\\\/fQPzGmFE6RDMTDz975EOYN45r8uyPXV2i\\\\\\\/NLxxQMlJSsefRRtfuWIFgD+\\\\\\\/VpgkR\\\\\\\/av4wUDKRxxMPJbMgCl+cjnGeQqAX\\\\\\\/7hq8t0oUshQPwkjZUYi+PM50mos1zaIGFQPfMHTdn8QGPwjdRWiCYDrarObeo+MVdKkY8HQXQpprKX\\\\\\\/uyZTG3h2W6Y8abmZ48BTE+4o\\\\\\\/4TvMe\\\\\\\/ZJRJsAeNyTg9Iowe\\\\\\\/Uqjuu7iG8V+XwsXAsRat5k\\\\\\\/OqY0Ib4MFXl2cPgDDaV1VSM0OTl6FgbHiaHOrUY+hfAbKPSmnFW7jeZm2f48Z3DCoMTthgx6GoyL2HHgvbE8dx1k\\\\\\\/1UlOc+8cxRSfBslZ4cisovy+lAtSjqjNmtkxjpzc1BJxGtED0RAD5V56Hq+aN1x6cgdbOGC10\\\\\\\/srx3vjdI0UrI\\\\\\\/sypAAz\\\\\\\/V\\\\\\\/PMaVo22hRxmZaLJjgEFV9YMOiiJVlcKxueoG0X70IeV7XrtqADDhqcfjUllV\\\\\\\/558PkEeSoDpZi9t9rtXPRgl\\\\\\\/1PPhy6D+rIoI40BgTqFTjTfuRWtBnCStb7Bv7CpN+Tkmo3GJK668ziWLP9oddbJdoSX97H6lmISZMiaZeX97+gO5r3ioOWGuF7XTXWJd0GC5J9r\\\\\\\/Pe9eDPUPe7QU+pRv\\\\\\\/nIMT1gAEYKKdZMm4Tb1KL4tdA\\\\\\\/iIA24y3mgW1Nx64i7MolBV4PNmMaaapNNfGtnhWEPx9FnIAJmrXKnIHpdq5B69YKnxFXVGuR5YCwb8mSMHgb6oLy0dcEpUba77\\\\\\\/EZtjacTOySaDoAslhWyIAPj5sYMpj0p5\\\\\\\/ieELWDjT7ZNsHnD6Beuse2i8CQcJs\\\\\\\/Ad0ZVCOgTMEI6Vo03HkKhXZVuj0vz7gbeYLDoOG+nUeAMHT+2W+7o7IHPO11CyUSAK4Dv6ssOOQ7FiKoMtRixZNYU7tIKVkhNlIMQho+YEQQqX\\\\\\\/+WDEMSNTMKo26usMlHOboj0YUnJ\\\\\\\/oOncNSlhrTYIHPbnDrmBcQMJ6sOdGa9SncwLBQsr6O443YeJwnVh3IsbN6DJVHMFVn+U5VJ7GJxn3Z7rRA\\\\\\\/6ORu2p57pubRABABVnZXbom6yJ9z5ow9zp92OBOXyIr34OsEVSVpKmyHg\\\\\\\/9kwV7zpYko27toEAftOl4E905ep4ddofi2sBnWwT0GxH5wSq3xI331p0ywCZfSdhryTFduRADEF8D+7BSgNGV68Ns9GoOssZzjOcJN02L0q5MJJZYQVjaM85fLHbdbKyvNoUi3vgHvmepbZ84OnSbaOpJgYWXf0ASO60yVOG24Azttut3kHZNS7fkhcyWSmTQ2tX9lS+hCSDPJvyDwNQPMrfwJEC++JboLziwipt7B6yC42X19BvfLsMprZJtV+\\\\\\\/aBs+u4M38l9+KvTdMxr89fKFY1Rd2UzGSl5BNTStGdNcKRV9vVD20vKa0E+U+NE+CLBdKU82N2muIC49bBPKZTzs0hDnO9yiSUaZ+vCsHho6tO++kYCszTppStIri4zGdiEde8uIBMIc4akpraIuCee\\\\\\\/TsdN+LxN1smeQCXLHM1U1LCOInEfmnmTzKmJd0A8Yxrzpn7ApDqypJvs27ilhWWk9EJ4dmCvz+P5fqDKGvoWHhWNBqurKdPJFasS\\\\\\\/rQLR0a0+n831Z2Fw6hTZ5I2xf4py057HbloMof5LyCsQvTtzBbPXZLDhpuWwvhYP4seD\\\\\\\/ppH5Co\\\\\\\/GjvcT+G1DlEWO9Y4FRb9GKupC+WVns7nCy8mmnI3yIcd0H7iE\\\\\\\/lNr0ykQZXjDtX1wF2AS9MRUFg\\\\\\\/JcPQAU16zBSD2gQEc2xMm4ep3eOXmamKKoEqIeN8eeNQ5fAd04NN+dV92r3fk2492xSS1SO2Lr\\\\\\\/BPER6G\\\\\\\/izAgUqVRFgKI1o2uni1sDRZEZ4\\\\\\\/Zt8F\\\\\\\/Vjjjm2cx4r6UtZ8HVUzmaJhOJdl8xbvxXai+On4WI6kos1aYAzXpxTsyUDMVa8ia4bDPGPCA+6bvgeVScT7A8Z9+Ho+QpO6E9UNldBmTERSHwGC0UFnNaosnsxKaxI66rEtHPiWXx2\\\\\\\/FaJNKE06xh9bgONBKfsE2xz7s1IFd9JvVQQjNvDnW\\\\\\\/qRWVpyqQyC6H2Vb\\\\\\\/xJdQXBj9HnxrntZ5XBH4ThattgLpaN4xt\\\\\\\/zTKZJo+o1cDnpb\\\\\\\/B6em+dYRsgs2m180\\\\\\\/cmKOeWWLza9Xa3EXdpprSbAs7w13JBUTE5bL2haGDpz4FWdRlB5nvrdmEgmWoeYRvjBD8kQrKuI218x0K\\\\\\\/SYZCe7RI0XNlZninqUW2F3bOLn9Qyimuwf\\\\\\\/iNDzupcvzGpaFrJKoNnswckT\\\\\\\/zJoaEY5Rzba5R3YMUTlhN2FNq7HAPfoK92\\\\\\\/VF7NztPvtAjvKkF8AM5Bu37H\\\\\\\/VGE0S8QEuBfToUSPzCH2zYZOumZl9IFymqCZ7HRY42oEoAHI\\\\\\\/WTNkSWKUiw41TujJpDLz+WoFGJ1TNf7E3xoD0ExxMuwewnrNqgpoAaPRP0cPIHA227BmcNX3v7DseMi1kHii9AFk8Wx4XHbaB5Uue8vv0FwQlNj9LFTbkW\\\\\\\/ymwPkFDvx3KeMbRPb0hY\\\\\\\/8Z7xufcpmyf+\\\\\\\/SC2NTkUtBk+NDKMQ2FkpYb8I0aAgwlDYLI4W\\\\\\\/POyXk2K01adOnQXfeNSNuR97CGPYVz5qP+FFErZyFw5yOC+hSLzl2XSiVnEbT\\\\\\\/o7Ug5p9Pssz42WNCkaQc7Pgw41jKTRUcBW6qBKJ\\\\\\\/F5lZYLi2sgkWIy3\\\\\\\/4n+zvOMsGbdncJyRDUpjbT5YrkaZ0QhtvEpuZqWwpunzeMIELx8hHeb+9oPlqM1qAND+uHSEyjPWQD4Ipxnd0WwH6hN6eTvkeVjwaHj2UVShfwTPfU8Q3oM\\\\\\\/gKhHW2qf5GXKafGQn5eCAZT7PJN7UqvOOpUII9QNV9wnZLn7KH2zR9KozAnykA4GtRVEmTtX1620rJw1SPLF+F4iiOyxJBfuZVnif42e+IapvumC81l4HBzvipuoMwYzQu+El47MTTy1u1qoGYZwe+HG3kjTlncsa4\\\\\\\/rj7mCuNxj5mkbgGLdHMQ3vQ55tQf7ztxkbRBtFA23lVWcUrAeDmJR6fpOXovyDsffo3oM\\\\\\\/HyAZhvEzw3S2nKz7Vt3C+DCGMnmu0gydNPZVmhmN4hPQvHCjL\\\\\\\/I4pzXtj4wMLdAmJjY0cFrt\\\\\\\/BUMGNJz4NomRR3irZRRB4PHsH7SmTLox5lXODuMJxRTI8OZuocPxpSOcQAmalT6BClxqpESCmnIvnlK86sbgLiZ5HJnj6wtVu2cewsRYxQZo5jxBnEdFbYyoLUrOe9iIWtV7i\\\\\\\/5\\\\\\\/4YPZwDzq4LfqHy6yew05oYLWHhRFLcyPj2a2eR1b\\\\\\\/CZXdCbhG6MTCOcUByyWL5vrRNx0iwLm3UQWdeOXYcHiu9mJcS1mdOgE1KWHlzmJGzW8aC0StwWo7HCOvnWh\\\\\\\/nvWZnuNRsjesyz3dI1+vmpQ2m3Jv53kaPsQw1SciAsvMsYSqVvtljgSUMIn2klv9ZB+5GO5d5my+YVCbaCKJcJRhs+q6C5PUumkfUOgbXag0O7VHDeuLAr+mCc+PVz\\\\\\\/HPFvGTGpJ06go0lpn4oLk25biUz3bHlcON4F5tFeN27Dd\\\\\\\/w5aWGexVRd1\\\\\\\/gRb9rOjhLa5B+S7toi9XKUiBOjF1nGQ0tajdWidKsGHlcrLw\\\\\\\/xDUjhzy\\\\\\\/k2fYsWoANSb83wvwOxUL8uQXGnNDyeUBWBsuIor9c0eK3bx+FSXguTpE0g9jeOi7rv4hdr2lf6IyZfNqpSuB5bKZmPvKvQw+SOK3hrRvGO+IBHrN36Jd2JAop7fhsVx4smR1Db6DD4J2HL4oMe6fW2Vl\\\\\\\/ZnvP0J5KvwF5xFweq\\\\\\\/MvavU4Z5vooP7MaJnfhQ0MMNRe2\\\\\\\/1lLQukCGJwACFdKLCPd+ekjeFt8jAO5kziyVC7wBEhxHKu3EO2mQ0yaGwxVpX+LEO32u3susjUGP6D++D5ks7CVRpcI1D7T4+Zm0Ys3PFqn1vsQ+TcCMAYsRFW6z+87ojebgxI8qWucLlPlpEnjp1vqfH3ANGeZdrZDwdYstPtYZGrG1zZw3L\\\\\\\/0iHAyfue\\\\\\\/+bW3aFEtGqSGWiH6Iog65V6T9krUzratt4rpEyNTxQxnf2ibOmnbpPFDuJunNPOcX0A\\\\\\\/zIUb1+UzGiwiAGaIlZjPPPlY7yE80vFZCT3Rih1\\\\\\\/bL7yQV6MBgmNB04x\\\\\\\/OLnf2Wp5Q6Ktq3GAY\\\\\\\/exjo+NcfTxnuajcwWXfe+i+fj8hENoy4igA99Do8Olg5Wi9O9tC8Dehw8Lnk2erVJJ8yTMcps9Cry5BdKlRwETzibRp7eiJEjrOl52l+9PauY85kihQn8dNA5jaFfzbFVXHTZzzrT7fGgXj7BX2ttirFXikG0K4cqIiSJE2kH2amni8dWqOAzMabzfq5bG6ESGcXXqSczUFDu3qv5Uuqvx7wMARoOr7xTV9bKCu084xTbnaTc2aGxvctN+WnbjGyYPmMMNA\\\\\\\/XDKK7wdRdrbiMwW4ckSUUNsAnlu7VI4VepdD3sfBA8p99lTUsnzwiB8BJkhbA2+IRBUoBgqpD6faTuPzvGNCXWvO4NNfhY3HQOWivDV4I\\\\\\\/BzpTi1CyF3IqxnhOjgQdDMg+6mKK9nvOruJDFEFtI5GVREyO4GpLpooS0FoODc6Ysn+G3wO0LFptPuXePZrWXCzteAvYjzfhyTjwIernAo08ckWyYJPgjFTXsIESseW+dFuQBM25aBzEVNqLRXoa\\\\\\\/3Bswg4mY7nhChlLmMaURN+7OeBrbr+59CUmZBlbn+dHruW69ABjJ+itZ5ZeO8L8IhJn2FDzt8Ic78VCzadqHj5jCrX0tDoZeB\\\\\\\/NsBgzqE3eDN4A7Ncueml9ZDNxeYKoKxmzkbCo3skwIpcTLjjd7MS0jHx3IZwbeUht38p4tbtebSc9qTyQs2KA2ar5vhuFDlqDn2gfxDhqb\\\\\\\/P3RCAYaZN9W2yuWSq9NV0oAnBgaKYtkuICfQxwCv8jgz1MpPuECHNrrv3E6OjKbg8Wf70rblx8cDTKKyRi1jQHAvuVOC52dPAo91DFVntmjQU7qaxUSsyshDo\\\\\\\/HOw7aFNlRIK5ZGn85bUc\\\\\\\/CWycDGFr1Ut0qCAVkA9dTq4gr1MZBvjD4Tm4CvemyHgDUP++WKQO6InwpjY0ZYD981dYcz5h0rfC2fKXqz\\\\\\\/6BhzY4kDo730WPTEJZh+n7dI78nOdmih1lpRY2YB+bx9\\\\\\\/\\\\\\\/+mkAA13KE3F1MOGxZ+XzpfAcL3nY8QmtL07kJRoWASZXIOxjHoS\\\\\\\/6FnxkrGQHkmYvmqTZ06monUc63UcF\\\\\\\/Qv+tWstYWluA\\\\\\\/o0qH27MzqRU8XLAhGGcHCZYz6gFWl7d3zBfM6uX8TIGvztcoxsmpdSuACLhS3Cn5ymyCNiwElyNVpirqEE0vagY3jkbRIGrfO9X68a6\\\\\\\/LOd+97znL8mbH9YkXEa0yl+SQplSMUhNsDU11WS+Cvn2iinHAu+W9d9pvdHKKtOWD6QaMypUJR5e\\\\\\\/KaBL2aX6xGusaxn\\\\\\\/7TAjzfLo4s6Sn9jB\\\\\\\/FTezx20J3BJJ2QIorGFdDDrlNoS9BfC5rkOYGwqgc+bKdpm+xkrzqxmT7LLlVfuCc9GujZwAZQQxc229uWisJT9incAWu1luozcWvggkiBMxuFkfjsdMLAbLjZ2AvB700xjtrxzTiuG0El2xtB0wSDQVjc8Z23v42MvwKNxHvzToVPLdte08STWsEKf3TswlFNSsSKCSIzc4LLfjCzQWEla7WEYwhqpS+AAvEEUAcqKbYY63hvfkI7rZFOxgbO1U2YNH7BoMyHkjLugDEsB30tczIKCcyjQ2jooHmq8zNEW5XRlmvrCVM\\\\\\\/YnpglimkdbL+rFqrGBKCytae3Ah0xMDjXf6wkb3RdZ3RoVn2T7ULfFnFso2n9L4V1jZjBo1JC7Z+zrdmNINVD8dLfHx54Kd\\\\\\\/jVUOS2fGG\\\\\\\/Iy+SFcwZHSC6LxIflnnRpQAd9l\\\\\\\/N8VoGrDEWV4zPbuUzCdTYSJU+AMuPo4K8h6vDtyvdX1GatdD1CXD98Dbp07P9XItkfJW7fkT3k3W4A25yKPXW+FdSRaB05QJ8iRz60jqOOLyx9R4tbTRHswo+dBZAAlth\\\\\\\/4xmTqM4oeByy1tNEFuKQZ4IWSu1EUz+6OSUB5fhWqdX5A7nyP7zv7hNzinwlnlbetPqr+Qq6lI5OLebgEtqYUG9smCgzg1C5OhhVBXHV2Pe3DT4amJXkdOpW1ZIUxC\\\\\\\/MsGPszl8vb90edRWc0BS90HlTXsCFbVtffHPlEM8\\\\\\\/KxEpvmUonZ7yUXqbMcETU3GWfkiniA0LQY+fcdaTkuGfs8JORo8f+ADPDoTIyJIdSzQ3I0iRI4apGrpWv1dEUcy1z\\\\\\\/Z2kyChxR1ZWZNe\\\\\\\/dcb6\\\\\\\/w5eCncoQsX49A5g0VjuDMqp686DhLSLs9D6scAeLrhkxtX0J38\\\\\\\/lMcpgxYuDxPD6PEzsQHT92x1xmDrJWSWmlXZfeYaZsTXnyg2za57jjaG0O656ozbG5oieDXt7mJ5uXyiTH2SdFCnrfJm1huf2tdYtPMvwwqZ3bsKLgmbMYmfxe2a54GNimClPnX6IRdbQdxsxDkWBK30SLwu2AKbLoEwAXOjYyRRVt4LciTmgrYDfRb8sJIMhD2DrSqSRedA7uB25PGdLKHrG2pe2ZMXVWgULLpQASmA2JhKj7+azlxKlFWXhLdhgYyPznNJ0RCWUa4h625drHB7XclZTo\\\\\\\/QoevKzVQY1jFtXzzDnhfq0\\\\\\\/Jb1+7vTDNyu\\\\\\\/VvMxYIKHmBeSp\\\\\\\/OlGNOWNhmfLMYqGiGCB1DVNCJ6JLVxvEjW9i3Uct\\\\\\\/Mw2atpMk\\\\\\\/ZMe2owyJHKGesyeqgoGBsj5HQX3Ac4\\\\\\\/RtatQpf\\\\\\\/vBOyFxy3YHW7TP\\\\\\\/sSAHIcjVqMMf1KARH445gH0mQE5cpaM1baSkGm13qWEHDsri2dD4R6AGhZQ6hgMuzUo+iY1SnVgxKA0Cc2u06wcwKNQBJjMYpLBy+jsWla1xzSsX6gYuA5AE\\\\\\\/mzpJkpuEHaU3CNXjDI17NoX06SCFhS\\\\\\\/6GCxR2RkVpBG91ghxZHKSPn+kMWrSb8WaxAzHywrOWseJ2kJcJMsTLJpw5fHAMjYoN4m2CkYyXm2LbseHRYCJA9HCmm8RZHj6mzD7U3reV1DqlDfL+mRf5TiwPOqZHqiv2Z1PfuVKKgGn37\\\\\\\/iR33BRVF1AefOvYdrM\\\\\\\/I8lCyuXt2GmNGrjK+dDCtyTSuVxeONbwNveNqLrS0X4KVv3TgUtOZXuP074U\\\\\\\/5Nk2p0ky7mtJ\\\\\\\/gnyK5o7+KKLCcDjkvfzT\\\\\\\/RURmIz9rNVCaCtuchWtiUkwhguVAupTaLe90DwRCTR+twQX4070K5+BzMwrlaBIt553pyP6LIDM5osje9yBzJtJ5k6GhwksmlJuERshOTugyvJvQsXm09siSDTQNJKbvn1WoXLHEM3GIEyViEOFOwDCtUZ8UsbmijH27x5fNKM77wvJVW4C9Pjjny5az9+m1apmZ95yc5lnomrFAnx9+K+sgJz6Q+GTvmfaIM2+bEnITImgcetIyqfhlRfntmu7wFHjX6xMKFoAeTDaggYqlWFzWXy\\\\\\\/RoV6oKKPj5faywoQm1KzzVoQnkKOjLAO10gSZiXzTCk14OYKLkC4LyRTnDPhlZTibzdJcWOxBYR3tQtIFujThhppXjIXGm7woSeGlFtYrxTL63klLm46xUnbMyOfbjxKNN9XQUV0o1fm14+bu4hc6+xo+pi1K5xxnzegJjaz+HWPSKrnT5jKno\\\\\\\/cxb61JQN0nteQ75+uZQ93x1+ltQwtPboL6jNvSh\\\\\\\/b1TfrRX3NwRm4MekDMkNiOOxCTpQc7D32ZRr7JVUWZisXEvODclctsFJ1xJ1uOw6tcStXqbwLjgDyMP4G\\\\\\\/Gkk1WolzTU\\\\\\\/Fmadqk4RiIbcxVfqVKQk9o+xi6PY5BFbeF+twCaaTVKpWWG85lbgZgXl896h9y8AmlqHJjGCsXxV\\\\\\\/zq4LjpriIKPHLNINtI79\\\\\\\/o\\\\\\\/rrgp6Err+BQQACpLRq9eMxj7IwHx15i6gNr7hOC9qdDAZKvZX\\\\\\\/GbmqNsH+ZO3YrdW2HciZC52En7cbtqtWb8XZlHLIB32teVs4yrNw4eohv8F0B4WN4ks1W9sNHePxbHrhqKmDHrXt19Ej7w7QcfubKFTO9\\\\\\\/LIvkFKIEXFnzmv5\\\\\\\/pyeWvmOVRF9Jy+wWGwrpShVTyVJes7hmFxbmdeWtzdtirNQQwjrvYDVrtK8KlYhfF5ppK84WGZtFh9QcCe1ackkeEv5pk3WpFUyQNAEu4cchVUWVS+2Bfd3aN7Jb5fBh4CFVFfEjlopAsTOGVuph2jDXC8PqjXQXTA5CIl2sKCrA498T0UYlnHRbR1w8b6pGMhybY7pTUS51P4LDQImkKp04EudOUlmaLozMyljNmAoiUie4Bsr+dciQJsgEo\\\\\\\/5r\\\\\\\/QautsPTuF0UWOwIH42M1sCYhK80HAjhrnpXXxgOEHUHrxoLJr7CeIjsJEoQwtONYfKld37JgqGihLKecZUkITnDnx4vHa7At55bniQ==\\\u0022,\\\u0022iv\\\u0022:\\\u00226RWmOA\\\\\\\/Vj0k5LDW4jOCAuw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002207a4def557da245e4fabc7bf0a383890204573722a8ee3d72df2a08aea2fc3c7\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Kc+PNMq0FBsD5e+3kKd\\\\\\\/ibUlzv3lljRJlWsPVYOZySR5i\\\\\\\/IvUddtr6LgvWT81+I\\\\\\\/WdCQq18o32SBZvU\\\\\\\/fEJ8Hjp+XNVFFEjEuArD\\\\\\\/Hj2w66T\\\\\\\/P95GvgX8SFcB1Tu0PrdIMaZpK0rsrLPrCf+Rm9m\\\\\\\/k9CMfXW5g7UWt3JqmVL8u5yocD0+ZcSCq3dyK17wMxkUmDPPaf95teNtwQgFUVcBFbPQSqx\\\\\\\/QrpxArdM1T+5xv1OtIJUOoniD9WuFI2DoKk4CU+lGOnKWDn85SfrcNA0Kbe8BwiyRGBfDHcjUGQimXHpeAbAryONkg4UG5pO1QYIVmXfA3QCQbmAGNBaRocTmn4xX+8xwPcWoc0BHVdVi2EgVzxDRxYqG1vhFLnIavgSHB5p4ro5YExHy2Ta0u31UkUrepS\\\\\\\/4M\\\\\\\/Lj51GHjNl4BY\\\\\\\/25qmd7X4JmsE0Aqj81mewo7i9rSYmfAIP7vSA4oMpQw3\\\\\\\/uKyva+VGKy7wrMkCDeLmPaXGDY0a8qmnsmRRdZZoygWCK\\\\\\\/mOP0ASAClQD58fMPVOr9ck8FG0WM7BOjzZ4Cf6fApBxzLeX7ItANSbelGDAhd97SGSIune3pvy3g5gq9U3YjIt5OjtUdCumnvZlZqgol4UK6lpznjxs1DEjFsLb5+E5zoZrskrFN1O9USYY7TxKoDV+Y8\\\\\\\/D7BEmTxuCTOnGddvOl4urmdUxO6J6agj6GDhGxYjPj\\\\\\\/byMYcJ6\\\\\\\/Lg3zZ7kRQ0D11c3y8LNN315K5XNo1joc5i\\\\\\\/Y+nJbI79xvN71p+gllOJYckYs\\\\\\\/1yRYZXRxdpcX3KEh0Tkfb1\\\\\\\/Kgx9gNEudDhouAQ0yR4mOr6T5QYrkThbkPxoDVSUCwKnxYXQXJXW5Oh5eP7chXdTxOKGEv\\\\\\\/Oy+GF5l4fuG42qTyXFZlQqF3HW651Wkq84OaCy2d5isZevAxHInEAyZ0K7GUD5n4It2v+cftjAONe42fVt\\\\\\\/pDhMEC3lR0UhVLSvvnYzuNNSAwl0vTZ661nPYAt5tYU1pepQO75FYq2AuRekwCH+ftBUGELMSBoj2Q3FJXgT7EL26ysd1shNYTmuuZBoVa0RGoJVf6ZLDU06EJEbP+r\\\\\\\/rJytXYKRyehRoBgCGX+D9Rx9vpXHH+55eggJ0jI44e4h4N3WgIJlMA5zv5BLY4on\\\\\\\/wv2ZzrJ9Hd7twR9SM0DghqqqewUHJzOv28u4LJ\\\\\\\/TzNeHDxIEcoExeqkj7b\\\\\\\/\\\\\\\/YOY1i4fAum0WRPR6Qk1P7Xh9sDA4aU0HRS0rcohsbuWusNuiQHDy\\\\\\\/+w9x0VJFmd+E+Rj9qvecfaCfb2AmurmRedBHW5f54M+W7DTy2JQ49CJKmSRjgO2xrr7DLfoBOUx6D\\\\\\\/tEKu\\\\\\\/ZQDTHnAUoBIHaijO3Z3eR1MY5F62FTEw5iOaUAD9W+rR14giGl\\\\\\\/bdfj\\\\\\\/TwDq6di8GCvAfkxfegbn8LWTIJmGanx\\\\\\\/YsbwzGN8FT94+VMPfFnNsjF1eXirh6ORDSiZNkTW6j1ynjuFIfoQKgVJ6l7IZNBwb4p2rFhfGydYTjPil28hUn8vXFlQO3HMn7hM031cbyIvZTWEYtxl5n2J\\\\\\\/2atMJA1iMdpPp59LdC\\\\\\\/uZnHGXzBAdE8NwFYYMCabC1sm1hwi\\\\\\\/7dMGJoGi4t0+V0yteP8WNoKJ6wKagqiPtLQaVGnEjO7m8BvEBCEfLuatwuE\\\\\\\/jB9JES91Lz\\\\\\\/tXVEvw9RToiO6vrl9JBWsjr3b6MBn9wC4gr9Crs\\\\\\\/BTi4fB+RUWuYuY\\\\\\\/V6GAC97Jd1MT7Kl1X7TMRbY9OTq9YSfGlUmTiP3EDAqoTCT1nMacPsdKkVI\\\\\\\/iGN6svqtzAUDmiu2EfjpwQLdknRX2Haa6zrtFSlkPt5QcpI9RZhai5WNU376tANAzIYbZLgyEJgSR42Yxy+ezl7nWU4duZ3or1nM2+IfYfb\\\\\\\/Nz4pyHsgr9B6qOeW\\\\\\\/j+xahKQjgkJhzszHcX04y33HEGvhQBC17wJDZXCD8\\\\\\\/h3pM7soR1X0QYweno2IKD3bP\\\\\\\/8SA40M8E\\\\\\\/TiIjU7c2g==\\\u0022,\\\u0022iv\\\u0022:\\\u0022z6rqiOy1E4wTKZHdRCDFuw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227ee7f5613050269dae4422d7a34c9e895ef96bc31859cf392e7e2f670a1d37b9\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-493745 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/8/2447" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-JkdoxeHGuX_ZiCgajWJzgUnWS-1uXB9tytoollktHR8" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/8/2447" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-bY0hRCOEMFNYIjBOmFrFcaRiBeM5Rj2xdsoutPd9gUI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/8/2441" title="Evolutionary and Experimental Loss of Gene Body Methylation and Its Consequence to Gene Expression" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/8/2463" title="Genomic Selection with Allele Dosage in Panicum maximum Jacq." rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493745" id="node493745" data-pisa="ggg;9/8/2447" data-pisa-master="ggg;g3.118.200925" data-apath="/ggg/9/8/2447.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">GRAF-pop: A Fast Distance-Based Method To Infer Subject Ancestry from Multiple Genotype Datasets Without Principal Components Analysis</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yumi Jin</span>, <span class="highwire-citation-author" data-delta="1">Alejandro A. Schaffer</span>, <span class="highwire-citation-author" data-delta="2">Michael Feolo</span>, <span class="highwire-citation-author" data-delta="3">J. Bradley Holmes</span> and <span class="highwire-citation-author" data-delta="4">Brandi L. Kattman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2447-2461;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200925 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493745" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yumi Jin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Center for Biotechnology Information, National Institutes of Health, Department of Health and Human Services, Bethesda, Maryland 20894 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yumi%2BJin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jin%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYumi%2BJin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jinyu@ncbi.nlm.nih.gov" class="" data-icon-position="" data-hide-link-title="0">jinyu@ncbi.nlm.nih.gov</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alejandro A. Schaffer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Cancer Data Science Laboratory, National Cancer Institute, National Institutes of Health; Department of Health and Human Services; Bethesda, Maryland 20892</div><div class='author-affiliation'><span class='nlm-sup'>*</span>National Center for Biotechnology Information, National Institutes of Health, Department of Health and Human Services, Bethesda, Maryland 20894 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alejandro%2BA.%2BSchaffer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schaffer%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlejandro%2BA.%2BSchaffer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Michael Feolo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Center for Biotechnology Information, National Institutes of Health, Department of Health and Human Services, Bethesda, Maryland 20894 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BFeolo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Feolo%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BFeolo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">J. Bradley Holmes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Center for Biotechnology Information, National Institutes of Health, Department of Health and Human Services, Bethesda, Maryland 20894 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BBradley%2BHolmes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Holmes%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ.%2BBradley%2BHolmes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Brandi L. Kattman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Center for Biotechnology Information, National Institutes of Health, Department of Health and Human Services, Bethesda, Maryland 20894 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brandi%2BL.%2BKattman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kattman%20BL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrandi%2BL.%2BKattman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/9/8/2447" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:493745" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:493745/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/8/2447.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:493745" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:493745/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/9/8/2447.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:493745" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:493745/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/9/8/2447.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:493745" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:493745/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-2">Inferring subject ancestry using genetic data is an important step in genetic association studies, required for dealing with population stratification. It has become more challenging to infer subject ancestry quickly and accurately since large amounts of genotype data, collected from millions of subjects by thousands of studies using different methods, are accessible to researchers from repositories such as the database of Genotypes and Phenotypes (dbGaP) at the National Center for Biotechnology Information (NCBI). Study-reported populations submitted to dbGaP are often not harmonized across studies or may be missing. Widely-used methods for ancestry prediction assume that most markers are genotyped in all subjects, but this assumption is unrealistic if one wants to combine studies that used different genotyping platforms. To provide ancestry inference and visualization across studies, we developed a new method, GRAF-pop, of ancestry prediction that is robust to missing genotypes and allows researchers to visualize predicted population structure in color and in three dimensions. When genotypes are dense, GRAF-pop is comparable in quality and running time to existing ancestry inference methods EIGENSTRAT, FastPCA, and FlashPCA2, all of which rely on principal components analysis (PCA). When genotypes are not dense, GRAF-pop gives much better ancestry predictions than the PCA-based methods. GRAF-pop employs basic geometric and probabilistic methods; the visualized ancestry predictions have a natural geometric interpretation, which is lacking in PCA-based methods. Since February 2018, GRAF-pop has been successfully incorporated into the dbGaP quality control process to identify inconsistencies between study-reported and computationally predicted populations and to provide harmonized population values in all new dbGaP submissions amenable to population prediction, based on marker genotypes. Plots, produced by GRAF-pop, of summary population predictions are available on dbGaP study pages, and the software, is available at <a href="https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/Software.cgi">https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/Software.cgi</a>.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/ancestry-inference" class="hw-term hw-article-keyword hw-article-keyword-ancestry-inference" rel="nofollow">ancestry inference</a></li><li class="kwd"><a href="/keyword/population-structure" class="hw-term hw-article-keyword hw-article-keyword-population-structure" rel="nofollow">population structure</a></li><li class="kwd"><a href="/keyword/admixture-mapping" class="hw-term hw-article-keyword hw-article-keyword-admixture-mapping" rel="nofollow">admixture mapping</a></li><li class="kwd"><a href="/keyword/gwas" class="hw-term hw-article-keyword hw-article-keyword-gwas" rel="nofollow">GWAS</a></li><li class="kwd"><a href="/keyword/barycentric-coordinates" class="hw-term hw-article-keyword hw-article-keyword-barycentric-coordinates" rel="nofollow">barycentric coordinates</a></li></ul><p id="p-3">Inference of individual ancestry from genetic data arises in genome wide association studies (GWAS), personalized medicine, and human migration studies. Several reviews on ancestry inference methods are available (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Lawson and Falush 2012</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Liu <em>et al.</em> 2013</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Padhukasahasram 2014</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Wollstein and Lao 2015</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Novembre and Peter 2016</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hellwege <em>et al.</em> 2017</a>).</p><p id="p-4">The â(global) problem of ancestry inferenceâ of human genetics takes as input genotype data on <em>M</em> markers for <em>N</em> subjects and produces as output some sort of clustering or assignment of the <em>N</em> subjects into populations. There have been two popular classes of solutions to this problem: model-based and distance-based methods.</p><p id="p-5">Model-based approaches, represented by the software packages STRUCTURE (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Pritchard <em>et al.</em> 2000</a>), fastSTRUCTURE (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Raj <em>et al.</em> 2014</a>), ADMIXTURE (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Alexander <em>et al.</em> 2009</a>), and FRAPPE (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Tang <em>et al.</em> 2006</a>), estimate ancestral proportions based on a statistical model, <em>e.g.</em>, individuals being randomly drawn from <em>K</em> distinct populations and may also use the technique of Hidden Markov Models (HMMs). Distance-based methods are based on embedding points in a high-dimensional space and combining algebraic and geometric tools to represent key features. Distance-based methods use principal components analysis (PCA) (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Patterson <em>et al.</em> 2006</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Price <em>et al.</em> 2006</a>), multidimensional scaling (MDS) (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Purcell <em>et al.</em> 2007</a>), spectral graph theory (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lee <em>et al.</em> 2010</a>), or network theory (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Greenbaum <em>et al.</em> 2016</a>). The focus of our study is a new distance-based method, which we have implemented in freely available software, called GRAF-pop.</p><p id="p-6">PCA-based approaches are the most used model-free ancestry inference methods. PCA analyzes a pairwise genetic similarity or genetic distance matrix between individuals and finds ancestral clusters by projecting the data to a lower-dimensional space. Compared to model-based methods, PCA-based approaches are usually faster and easier to use. Model-based methods estimate parameters using computationally intensive algorithms such as Markov Chain Monte Carlo algorithm (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Pritchard <em>et al.</em> 2000</a>) or maximum likelihood estimation (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Tang <em>et al.</em> 2005</a>; <a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Tang <em>et al.</em> 2006</a>; <a id="xref-ref-4-2" class="xref-bibr" href="#ref-4">Alexander <em>et al.</em> 2009</a>). Multiple parameters need to be tuned to run model-based software. Choosing the number of populations is difficult (<a id="xref-ref-4-3" class="xref-bibr" href="#ref-4">Alexander <em>et al.</em> 2009</a>), and usually also requires knowledge of the populationâs history.</p><p id="p-7">The first PCA methods such as EIGENSTRAT, have running times that grow as the square of <em>N</em> (more formally O(<em>MN<sup>2</sup></em>)). Advances in random matrix theory led to the implementation of asymptotically faster methods with running time linearly proportional to <em>N</em>, such as FlashPCA (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abraham and Inouye 2014</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Abraham <em>et al.</em> 2017</a>) and FastPCA (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Galinsky <em>et al.</em> 2016</a>). PCA analysis <em>across</em> datasets is not necessarily robust.</p><p id="p-8">PC coordinates in the PCA plots are not directly interpretable (<a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Lee <em>et al.</em> 2010</a>) in contrast to the coefficients estimated by model-based methods. In addition, unlike model-based methods, traditional distance-based approaches do not necessarily assign individuals to populations or estimate ancestral proportions for admixed individuals.</p><p id="p-9">Model-based and PCA-based methods can work well for datasets with samples genotyped for the same set of variants using the same genotyping or sequencing platform. However, researchers often wish to do meta-analyses combining multiple datasets genotyped using different methods. With the establishment of public genotype databases, such as the database of Genotypes and Phenotypes (dbGaP) (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Mailman <em>et al.</em> 2007</a>) and the European Genotype-phenotype Archive (EGA) (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Lappalainen <em>et al.</em> 2015</a>), investigators now can access genotype datasets of millions of individuals. Most PCA-based methods, as well as model-based approaches, generate different ancestry results for the same individual when different individuals are included in the datasets being analyzed. Moreover, PC scores are not comparable when different sets of markers are analyzed.</p><p id="p-10">Because of the public availability of outside data that can guide ancestry inference, several research groups considered modifications to problem of ancestry inference, which we describe here together as methods that incorporate reference data. The reference data for these methods typically consist of genotypes of individuals whose ancestries are extrememly reliable and not very admixed (<em>e.g.</em>, the original HapMap samples). One way to use such reference data is to find ancestry informative markers (AIMs) using the subjects with reliable ancestries and to use AIMs to infer ancestry of other subjects. Many AIM panels have been built, especially for forensic analyses (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Halder <em>et al.</em> 2008</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Nassir <em>et al.</em> 2009</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Galanter <em>et al.</em> 2012</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Daya <em>et al.</em> 2013</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Kidd <em>et al.</em> 2014</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bulbul <em>et al.</em> 2018</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Wang <em>et al.</em> 2018</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Zhao <em>et al.</em> 2019</a>). Another way to infer ancestry with reference data can be seen in ADMIXTURE and FRAPPE that perform supervised analyses using reference panels to improve the estimation accuracy (<a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Tang <em>et al.</em> 2006</a>; <a id="xref-ref-4-4" class="xref-bibr" href="#ref-4">Alexander <em>et al.</em> 2009</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Thornton <em>et al.</em> 2014</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Conomos <em>et al.</em> 2015</a>).</p><p id="p-11">A third formulation of methods using reference data, which we consider here, is to infer ancestry with the aid of only <em>the summary data</em> of the reference panels. This formulation is attractive because one does not need to spend effort or obtain permission to download individual genotypes. iAdmix (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bansal and Libiger 2015</a>) is a model-based method that infers ancestry using a reference set of population allele frequencies. ADMIXTURE has been upgraded to estimate individual ancestry by projecting the new samples to the pre-calculated population structure of reference panels (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Shringarpure <em>et al.</em> 2016</a>). Similarly, for PCA-based methods, subject ancestry can be inferred by projecting samples in one dataset to the SNP weights calculated using another dataset. PCAiR (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Conomos <em>et al.</em> 2015</a>) partitions subjects in the target dataset into a âreferenceâ subset containing individuals known to be unrelated and another subset containing all other individuals, and calculates SNP weights using the reference subset and applies the weights to all individuals. SNPweights (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chen <em>et al.</em> 2013</a>) and FastPop (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Li <em>et al.</em> 2016</a>) use fixed sets of pre-selected SNPs from outer reference populations to calculate SNP weights and infer subject ancestry for target samples by projecting their genotypes to these SNP weights. Using SNP weights pre-calculated from the same reference populations makes the results comparable across target datasets and allows the software to execute faster.</p><p id="p-12">The previous methods using reference data employ a fixed set of SNPs from reference panels. These methods require that the target samples have genotypes available for most of these SNPs. This requirement is hard to meet when analyzing multiple datasets downloaded from databases like dbGaP. To date, dbGaP has genotypes of more than a million subjects spanning hundreds of studies, genotyped using many different platforms. No SNP is included in all genotyping platforms found in dbGaP. Nevertheless, both dbGaP staff (for harmonization and QA/QC) and outside researchers (so they can reanalyze the data), want to infer ancestry of dbGaP subjects across studies.</p><p id="p-13">No previous methods for ancestry inference are robust in datasets with high rates of non-randomly missing genotypes. Most existing tools filter out individuals with high genotype missing rates, <em>e.g.</em>, 5% or more. When encountering missing genotypes in calculations, existing methods, <em>e.g.</em>, EIGENSTRAT (<a id="xref-ref-35-2" class="xref-bibr" href="#ref-35">Price <em>et al.</em> 2006</a>), SNPweights (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Chen <em>et al.</em> 2013</a>), and AIPS (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Byun <em>et al.</em> 2017</a>), usually replace them with the mean genotype values. As more missing genotypes are replaced, the more likely that an individual is predicted to have an average ancestry background, where average depends on the other members of the target or reference dataset used.</p><p id="p-14">To find duplicate samples and closely related subjects across studies in dbGaP, we previously developed a statistical method and software tool called GRAF (standing for Genetic Relationship And Fingerprinting) (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Jin <em>et al.</em> 2017</a>). One contribution of GRAF was to define a set of 10,000 unlinked, autosomal biallelic SNPs with high minor allele frequency (MAF), good coverage on many of the widely used genotyping platforms, and other desirable characteristics. GRAF extracts genotypes of these âfingerprint SNPsâ for all samples.</p><p id="p-15">Here, we describe GRAF-pop, a new distance-based method for ancestry inference using reference, that uses the fingerprint genotypes extracted by GRAF and 1) does not assume that the reference populations and target samples have been genotyped at mostly the same markers, 2) does not impute missing genotypes, 3) only requires allele frequencies of reference populations, and 4) does not use PCA. GRAF-pop can give reliable population assignments when the rate of missing genotypes far exceeds 5%. GRAF-pop makes accurate ancestry estimations even if the missing genotypes are non-random. The main continental populations can be well separated from one another even when there are only 100-200 fingerprint SNPs with genotypes. GRAF-pop is fast, with running time linear in the product of number of individuals and SNPs. In addition, GRAF-pop incorporates a visualization tool to plot all individuals into the same plot and uses the same standards to assign them into different populations, regardless of the genotyping platform.</p><div class="section methods" id="sec-1"><h2 class="">Methods</h2><div id="sec-2" class="subsection"><h3>Use of dbGaP data for the development of validation tools</h3><p id="p-16">GRAF and GRAF-pop have been developed to improve quality control in curation, and to harmonize population values across studies. NCBI staff have access to data within dbGaP for the purposes of managing data, QC, and tool development, such as the development of GRAF previously, and here GRAF-pop. The analyses shown here involve six dbGaP studies: phs000050 (HapMap), phs000403, phs000420, phs000456, phs000517, phs000788. The study-reported population summary information for these studies are published both in literature, and on public dbGaP. This manuscript was reviewed in accordance with NIH Genomic Data Sharing Policy governace.</p></div><div id="sec-3" class="subsection"><h3>Reference population data and subject group data from within dbGaP</h3><p id="p-17">Herein, the phrase âstudy-reported populationâ refers to the population term assigned to a subject submitted to dbGaP by the group producing/collecting data. The study-reported population terms and granularity of the terms vary across studies and may have been either collected from participant interviews or questionnaires or inferred computationally using genotypes. Some studies submitted to dbGaP do not submit any population values.</p><p id="p-18">The phrase âreference populationsâ refers to five large sets of dbGaP subjects with the following study-reported population values: (1) White, Caucasian, European, European American, and other equivalent terms (2) Black, African, African American, Ghana, Yoruba, etc. (3) Asian, East Asian, Chinese, Japanese, etc. (4) Asian Indian, Pakistani; (5) Mexican, Latino. After closely related subjects and outliers were removed using GRAF, allele frequencies were calculated for each reference population.</p><p id="p-19">The term âsubject group(s)â refers to smaller sets of subjects from dbGaP: 8,475 study-reported CEU (denoted E), 904 individuals from Ghana (denoted F), 7,612 Chinese, and 4,138 Japanese (combined and denoted A). The subject groups are subsets of reference populations 1, 2, and 3, respectively, in which the study-reported population is especially trustworthy.</p></div><div id="sec-4" class="subsection"><h3>Clustering subjects using genetic distances</h3><p id="p-20">Suppose there are <em>K</em> mutually exclusive, random-mating reference populations, with <em>n<sub>j</sub></em> individuals for each population <em>j</em>, and <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-1.gif"/></noscript></span></span>. Consider <em>S</em> independent, biallelic SNPs. For each SNP <em>l</em> and population <em>j</em>, suppose the reference and alternate allele frequencies are <em>p<sub>jl</sub></em> and <em>q<sub>jl</sub></em>, respectively. The fingerprint SNPs used by GRAF and GRAF-pop were previously selected to be spread out, so that no pairs have substantial linkage disequilibrium and on the initial test sets were in Hardy-Weinberg equilibrium. Let <em>g<sub>il</sub></em><span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-2.gif"/></noscript></span></span><em>0</em>, <em>1</em>, <em>2</em>} be the count of reference alleles of subject <em>i</em> at SNP <em>l</em>. The probabilities to have each <em>g<sub>il</sub></em> value are:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-3.gif"/></noscript></span></span>Denoting <em>g<sub>i</sub></em> as the genotype of the subject <em>i</em> over all the <em>S</em> SNPs, we have<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-4.gif"/></noscript></span></span>In principle, the probability that subject <em>i</em> belongs to population <em>j</em> could be calculated using Bayesâ theorem:<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-5.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>In practice, most individuals cannot be unambiguously classified into populations due to admixture. <a id="xref-disp-formula-3-1" class="xref-disp-formula" href="#disp-formula-3">Equation (1)</a> does imply that the probability <em>P(g<sub>i</sub>|population = j)</em> reflects the similarity between subject <em>i</em> and population <em>j</em>. Taking negative logarithm and normalizing it on number of SNPs with genotypes, we define a âdistanceâ:<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-6.gif"/></noscript></span></span>It can be rewritten as:<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-7.gif"/></noscript></span><span class="disp-formula-label">(2)</span></span>Since <em>D<sub>ij</sub></em> is greater when the genotypes of subject <em>i</em> are more different from the genotypes of population <em>j</em>, we use <em>D<sub>ij</sub></em> as an estimate of the genetic distance between subject <em>i</em> and population <em>j</em>.</p><p id="p-21">For each subject <em>i</em>, we can calculate <em>K D<sub>ij</sub></em> values, and represent them as a point in <em>K</em>-dimensional space. If we plot all subjects in the <em>K</em>-dimensional space, subjects will be clustered based on their genetic similarities. We treat the centroids of the <em>K</em> reference population clusters as vertices of a (<em>K-1</em>)-dimensional simplex (<em>e.g.</em>, a triangle if <em>K = 3</em>).</p></div><div id="sec-5" class="subsection"><h3>Estimating ancestry proportions using barycentric coordinates</h3><p id="p-22">Assuming all subjects are admixtures of <em>R</em> (1 &lt; <em>R</em> â¤ <em>K</em>) ancestry populations, we can define a simplex based on the <em>R</em> centroids and then use <em>barycentric coordinates</em> (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Ungar 2010</a>) (defined below) to estimate ancestry proportions by mapping each subject to a point relative to the (R-1)-dimensional simplex. We select <em>R</em> groups of subjects with genotypes of the same set of <em>S</em> SNPs mentioned above. Suppose all subjects in each group have one of the <em>R</em> ancestries. For each subject, we calculate the <em>D</em> values with reference to <em>R-1</em> populations and represent that subject by a position in (<em>R</em>-1)-dimensional space. For each ancestry group, we calculate the centroid of all its subjects.</p><p>The <em>R</em> centroids, denoted <strong><em>V<sub>1</sub></em></strong>, â¦, <strong><em>V</em></strong><em><sub>R</sub></em>, become vertices of a simplex. For any point <strong><em>Q</em></strong> representing a subject, we calculate the unique absolute barycentric coordinates (<em>Î»<sub>1</sub></em>, â¦, <em>Î»<sub>R</sub></em>) for <strong><em>Q</em></strong> by solving the following equations:<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-8.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-9.gif"/></noscript></span></span>The unique solution of the equations can be found by taking determinants of some 3Ã3 matrices when <em>R</em> = 3, as described below, or by an equivalent geometric method (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Ungar 2010</a>). We use R = 3, rather than R = 5 partly because 3 dimensions are the most that are easy to visualize. If <strong><em>Q</em></strong> is within the simplex, then barycentric coordinates are all positive. Letting <em>P<sub>r</sub></em> be the proportion of ancestry <em>r</em>, we estimate the proportion as <em>P<sub>r</sub></em> = <em>Î»<sub>r</sub></em>. In general <strong><em>Q</em></strong> may fall outside of the simplex, so we estimate <em>P<sub>r</sub></em> as follows:</p><ol class="list-ord " id="list-1"><li id="list-item-1"><p id="p-24">Let <em>Î»â<sub>r</sub></em> = <em>max</em>(0, <em>Î»<sub>r</sub></em>), <span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-10.gif"/></noscript></span></span><em>1</em>, .., <em>R</em>}</p></li><li id="list-item-2"><p id="p-25">Set <em>P<sub>r</sub></em> as:<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-11.gif"/></noscript></span></span></p></li></ol><p>Starting from a triangle, where each vertex represents one of the three HapMap populations, SNPweights (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Chen <em>et al.</em> 2013</a>) uses the same algebra to compute ancestry proportions for points inside the triangle, but handles points outside the triangle differently.</p></div><div id="sec-6" class="subsection"><h3>Normalizing scores using barycentric coordinates</h3><p id="p-26">In previous subsections, we assumed that all subjects have genotypes on the same SNPs. In dbGaP usage, genotypes are missing because different genotyping platforms were used in different dbGaP studies. We normalize the <em>D</em> values when there are missing genotypes. For an ancestry group <em>r</em>, suppose the two allele frequencies of SNP <em>l</em> are <em>u<sub>rl</sub></em> and <em>v<sub>rl</sub></em>. If we compare all subjects in <em>r</em> to population <em>j</em> over <em>S</em> SNPs, the mean genetic distance is expected to be:<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-12.gif"/></noscript></span><span class="disp-formula-label">(3)</span></span>The subscript âallâ conveys that all 10,000 SNPs are used at this stage. For a subject <em>i</em>, if all <em>S</em> SNPs have genotypes, we can 1) use <a id="xref-disp-formula-9-1" class="xref-disp-formula" href="#disp-formula-9">Equation (3)</a> to calculate the expected centroids for the <em>R</em> ancestry groups, 2) use the simplex formed by these centroids, and 3) calculate the ancestry proportions of each new subject as barycentric coordinates with respect to the simplex. If there are missing genotypes, we ignore the SNPs with genotypes missing, and calculate the genetic distance from subject <em>i</em> to each population <em>j</em>.</p><p id="p-27">Suppose subject <em>i</em> has genotypes for <em>S</em>â (<em>Sâ &lt; S)</em> fingerprint SNPs, <em>i.e.</em>, <em>l<sub>1</sub></em>, <em>l<sub>2</sub></em>, <em>â¦</em>, <em>l<sub>S</sub></em><sub>â</sub>. We use an equation like <a id="xref-disp-formula-9-2" class="xref-disp-formula" href="#disp-formula-9">Equation (3)</a> to calculate the expected distance <em>E<sub>Di</sub></em>, from each group <em>r</em> to population <em>j</em>, for the subset of SNPs genotyped for subject <em>i</em>.<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-13.gif"/></noscript></span><span class="disp-formula-label">(4)</span></span>Using <em>E<sub>Di</sub></em> values of the <em>R</em> ancestry groups as vertices, we can build a simplex and use it as a reference to calculate the barycentric coordinates for subject <em>i</em>. The barycentric coordinates are converted back to the Cartesian coordinates using <em>E<sub>D_all</sub></em> values as references for plotting.</p></div><div id="sec-7" class="subsection"><h3>Implementing the ancestry inference method Into the GRAF software package</h3><p id="p-28">The above method is implemented in the GRAF-pop feature of the GRAF software package (version 2.3). As in GRAF 1.0, the main program is implemented in C++, auxiliary programs are implemented in Perl, and graphic displays rely on the Perl GD Graphics Library (<a href="http://search.cpan.org/%E2%88%BClds/GD-1.38/GD.pm">http://search.cpan.org/â¼lds/GD-1.38/GD.pm</a>).</p><p id="p-29">We use the same 10,000 well-separated, autosomal, biallelic SNPs used in GRAF v1.0 (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Jin <em>et al.</em> 2017</a>) to determine subject ancestry. Five reference populations, listed above, are used to calculate genetic distances. The allele frequencies at these SNPs (<em>p<sub>jl</sub></em> and <em>q<sub>jl</sub></em>) are estimated using the subjects submitted to dbGaP and reported by the submitters as having these ancestry backgrounds. For each subject <em>i</em>, we calculate five <em>D</em> values: <em>D<sub>i1</sub></em>, <em>D<sub>i2</sub></em>, <em>D<sub>i3</sub></em>, <em>D<sub>i4</sub></em>, <em>D<sub>i5</sub></em>, for the five reference populations.</p><p id="p-30">The first three <em>D</em> values, <em>D<sub>i1</sub></em>, <em>D<sub>i2</sub></em> and <em>D<sub>i3</sub></em>, are treated as the <em>x</em>, <em>y</em>, <em>z</em> Cartesian coordinates for each subject <em>i</em>, and are used for calculating barycentric coordinates. Three subject groups (E, F, A) are used to build the reference triangle, denoted as ÎEFA, whose vertices are the centroids of these three groups. This triangle is mapped to the x-y plane by translation and rotation, so that the side FA is parallel to the <em>x</em>-axis. The transformed triangle is used for calculating the barycentric coordinates and ancestry proportions. All 3-D points representing subjects being checked are also projected onto the plane where ÎEFA is located. We use numerical subscripts 1,2,3 in equations and subscripts E, F, A, when referring to the triangle.</p><p id="p-31">We estimate the ancestry proportions for ancestries {E, F, A}, for each subject using barycentric coordinates. Let (<em>x<sub>i</sub></em>, <em>y<sub>i</sub></em>) be the Cartesian coordinates of subject <em>i</em> on the 2-D plane after transformation, and (<em>x<sub>e</sub></em>, <em>y<sub>e</sub></em>), (<em>x<sub>f</sub></em>, <em>y<sub>f</sub></em>), (<em>x<sub>a</sub></em>, <em>y<sub>a</sub></em>) be the coordinates of the three vertices of ÎEFA. Denote matrix <strong><em>T</em></strong> as:<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-14.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-14.gif"/></noscript></span></span>The barycentric coordinates (<em>Î»<sub>ie</sub></em>, <em>Î»<sub>if</sub></em><sub>,</sub>, <em>Î»<sub>ia</sub></em>) of subject <em>i</em> with respect to ÎEFA are calculated using the following equations:<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-15.gif"/></noscript></span><span class="disp-formula-label">(5)</span></span>The ancestry proportions <em>P<sub>ie</sub></em>, <em>P<sub>if</sub></em>, <em>P<sub>ia</sub></em> are calculated using the following equation:<span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-16.gif"/></noscript></span><span class="disp-formula-label">(6)</span></span>where <em>P<sub>im</sub></em> <span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-17.gif"/></noscript></span></span><em>P<sub>ie</sub></em>, <em>P<sub>if</sub></em>, <em>P<sub>ia</sub></em>} and <em>Î»<sub>im</sub></em> <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-18.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-18.gif"/></noscript></span></span><em>Î»<sub>ie</sub></em>, <em>Î»<sub>if</sub></em>, <em>Î»<sub>ia</sub></em>}.</p><p id="p-32">In GRAF-pop, we use the expected <em>D</em> values calculated using <a id="xref-disp-formula-9-3" class="xref-disp-formula" href="#disp-formula-9">Equation (3)</a> to find the vertices of ÎEFA, and use it as the reference triangle to calculate barycentric coordinates. When there are missing genotypes, we first calculate the barycentric coordinates using the SNPs with genotypes, then map the coordinates onto the reference triangle when no genotypes are missing. Specifically, we do the following steps to normalize the genetic distances and plot the results:</p><ol class="list-ord " id="list-2"><li id="list-item-3"><p id="p-33">For each subject group <em>U</em> â<em>{E</em>, <em>F</em>, <em>A}</em>, calculate the expected genetic distances <em>D<sub>U1</sub></em>, <em>D<sub>U2</sub></em>, <em>D<sub>U3</sub></em>, to the first three reference populations, using <a id="xref-disp-formula-9-4" class="xref-disp-formula" href="#disp-formula-9">Equation (3)</a> for the full set of 10,000 fingerprint SNPs.</p></li><li id="list-item-4"><p id="p-34">Represent each group with a 3-D point in space by treating <em>D<sub>U1</sub></em>, <em>D<sub>U2</sub></em>, <em>D<sub>U3</sub></em> as the <em>x</em>, <em>y</em>, <em>z</em> Cartesian coordinates. Build a triangle by connecting the three points representing the three subject groups.</p></li><li id="list-item-5"><p id="p-35">Rotate and translate the triangle so that it is in the plane <em>z</em> = 0, and side FA is parallel to the <em>x</em>-axis. Denote this triangle as ÎEFA<sub>0</sub>, to be used as the reference triangle for calculating barycentric coordinates.</p></li><li id="list-item-6"><p id="p-36">Given a subject <em>i</em>, find the subset, <em>T</em>, of fingerprint SNPs with genotypes for this subject.</p></li><li id="list-item-7"><p id="p-37">For each subject group <em>U</em> â<em>{E</em>, <em>F</em>, <em>A}</em>, calculate the expected genetic distances <em>D<sub>U1T</sub></em>, <em>D<sub>U2T</sub></em>, <em>D<sub>U3T</sub></em>, to the first three reference populations using <a id="xref-disp-formula-10-1" class="xref-disp-formula" href="#disp-formula-10">Equation (4)</a> for subset <em>T</em>. Represent each group with a 3-D point in the space.</p></li><li id="list-item-8"><p id="p-38">Calculate the genetic distances <em>D<sub>i1</sub></em>, <em>D<sub>i2</sub></em> and <em>D<sub>i3</sub></em> from subject <em>i</em> to the first three reference populations. Represent subject <em>i</em> as a 3-D point, <strong><em>Q</em>,</strong> by treating <em>D<sub>i1</sub></em>, <em>D<sub>i2</sub></em>, <em>D<sub>i3</sub></em> as the <em>x</em>, <em>y</em>, <em>z</em> Cartesian coordinates.</p></li><li id="list-item-9"><p id="p-39">Build a triangle by connecting the three points representing the three subject groups. Rotate and translate the three points, together with point <strong><em>Q</em></strong>, so that the triangle is in the plane <em>z</em> = 0. Denote this triangle as ÎEFA<sub>T</sub>.</p></li><li id="list-item-10"><p id="p-40">Using the <em>x</em>, <em>y</em> coordinates of point <strong><em>Q</em></strong> after transformation, calculate the barycentric coordinates (<em>Î»<sub>ie</sub></em>, <em>Î»<sub>if</sub></em>, <em>Î»<sub>ia</sub></em>) for subject <em>i</em> with respect to ÎEFA<sub>T</sub>, using <a id="xref-disp-formula-12-1" class="xref-disp-formula" href="#disp-formula-12">Equations (5)</a> and (<a id="xref-disp-formula-13-1" class="xref-disp-formula" href="#disp-formula-13">6</a>).</p></li><li id="list-item-11"><p id="p-41">Convert the barycentric coordinates back to the Cartesian coordinates (<em>x<sub>i0</sub></em><sub>,</sub> <em>y<sub>i0</sub></em>) using reference triangle ÎEFA<sub>0</sub>:<span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-19.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-20.gif"/></noscript></span></span></p><p id="p-42">where (<em>x<sub>e0</sub></em>, <em>y<sub>e0</sub></em>), (<em>x<sub>f0</sub></em>, <em>y<sub>f0</sub></em>), (<em>x<sub>a0</sub></em>, <em>y<sub>a0</sub></em>) are the Cartesian coordinates of the three vertices of ÎEFA<sub>0</sub>.</p></li><li id="list-item-12"><p id="p-43">Plot the converted Cartesian coordinates of subject <em>i</em>, together with ÎEFA<sub>0</sub>, on the <em>x-y</em> plane. The final <em>x<sub>i0</sub></em><sub>,</sub> <em>y<sub>i0</sub></em> values are the normalized genetic distances, called GD1 and GD2 scores in this article and in GRAF.</p></li><li id="list-item-13"><p id="p-44">The <em>z</em> coordinate of point <strong><em>Q</em></strong> after transformation (step 7), called GD3 score, is also used for plotting results.</p></li><li id="list-item-14"><p id="p-45">Calculate the genetic distances <em>D<sub>i4</sub></em> and <em>D<sub>i5</sub></em> from subject <em>i</em> to the last two reference populations South Asian and Mexican/Latino. The difference <em>D<sub>i5</sub></em> - <em>D<sub>i4</sub></em>, called GD4 score, is plotted against GD1 scores to separate South Asians from Latin Americans.</p></li></ol><p id="p-46">A population ID is assigned to each subject based on the ancestor proportions <em>P<sub>e</sub></em>, <em>P<sub>f</sub></em> and <em>P<sub>a</sub></em>, together with GD1 and GD4 scores. The cutoff standards are described in Results.</p></div><div id="sec-8" class="subsection"><h3>Testing GRAF-pop in comparison with existing ancestry-prediction software packages</h3><p id="p-47">We compared the performances and prediction accuracies between GRAF-pop and existing software EIGENSTRAT (<a id="xref-ref-35-3" class="xref-bibr" href="#ref-35">Price <em>et al.</em> 2006</a>), FastPCA (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Galinsky <em>et al.</em> 2016</a>), SNPweights (<a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Chen <em>et al.</em> 2013</a>), and FlashPCA2 (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Abraham <em>et al.</em> 2017</a>) using the dbGaP studies listed in the first subsection. The first three programs are included in the software package EIGENSOFT 7.1.2 (<a href="https://www.hsph.harvard.edu/alkes-price/software/">https://www.hsph.harvard.edu/alkes-price/software/</a>).</p><p id="p-48">We extracted genotypes of the 10,000 fingerprint SNPs and saved them into PLINK sets. We compared the performances of different software packages using the dataset of phs000420.v6.p3, as well as the datasets combined from two or three studies. Missing genotypes were retained in some datasets to evaluate the software packages in the presence of missing genotypes. Since PCA results are displayed in different scales and directions by different PCA programs, PC1 and PC2 values generated by the PCA software packages were normalized using the following method: 1) Genotypes of the HapMap subjects were combined with the datasets to be tested, 2) PC1 and PC2 values were treated as the <em>x</em>, <em>y</em> coordinates, 3) The centroids of the three HapMap populations CEU, YOR and ASN were calculated and used as the vertices of the reference triangle ÎEFA as mentioned above, 4) The barycentric coordinates with respect to ÎEFA of all subjects were calculated, and converted back to Cartesian coordinates using reference triangle ÎEFA<sub>0</sub> as mentioned above, and 5) The converted Cartesian coordinates were plotted on scatter plots.</p><p id="p-49">Since GRAF-pop can also estimate the ancestry proportion for each subject, like model-based approaches, we compared GRAF-pop with ADMIXTURE (<a id="xref-ref-4-5" class="xref-bibr" href="#ref-4">Alexander <em>et al.</em> 2009</a>). Because ADMIXTURE requires that no subjects in the dataset be closely related, we used GRAF (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Jin <em>et al.</em> 2017</a>) software to find the related subjects and created a dataset including only unrelated subjects to test the software tools.</p></div><div id="sec-9" class="subsection"><h3>Data Availability</h3><p id="p-50">Phenotypic and genotypic data required to confirm the results presented in this study are available through the dbGaP Authorized Access System, with the following accessions: phs000050.v1.p1, phs000403.v3.p3, phs000420.v6.p3, phs000456.v1.p1, phs000517.v3.p1, and phs000788.v1.pl. Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8061485">https://doi.org/10.25387/g3.8061485</a>.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>Clustering dbGaP subjects and assigning subject populations using GRAF-pop</h3><p id="p-51">As of November 15, 2018, dbGaP includes data on 1,987,542 subjects in 1,276 studies and extracted fingerprint genotypes of 1,387,305 subjects from 501 studies; 1,186,387 of these subjects in 478 studies have more than 4,000 fingerprint SNPs genotyped (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). The study-reported populations were submitted to dbGaP as text strings. When non-English characters, UPPER CASE/lower case and singular/plural differences are ignored, there are 264 distinct values for populations.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493842/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493842/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Distribution of numbers of fingerprint SNPs with genotypes per subject in dbGaP</span><div class="sb-div caption-clear"></div></div></div><p id="p-52">We calculated GD1, GD2, GD3, GD4 scores and estimated ancestry proportions for all subjects with fingerprint genotypes extracted. <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a> shows the scores of the 1,186,387 subjects with more than 4,000 fingerprint SNPs genotyped.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2447/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="GRAF-pop results of subjects with at least 4,000 fingerprint SNPs genotyped across 478 dbGaP studies. (A, B) All subjects are plotted. (C, D) 10,000 randomly selected subjects are plotted. Cyan lines show the cutoff standards suggested by GRAF-pop to assign dbGaP subjects into GRAF-calculated populations." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1041302944" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;GRAF-pop results of subjects with at least 4,000 fingerprint SNPs genotyped across 478 dbGaP studies. (A, B) All subjects are plotted. (C, D) 10,000 randomly selected subjects are plotted. Cyan lines show the cutoff standards suggested by GRAF-pop to assign dbGaP subjects into GRAF-calculated populations.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2447/F1.medium.gif" width="440" height="268"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/9/8/2447/F1.medium.gif" width="440" height="268"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2447/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2447/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493957" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-53" class="first-child">GRAF-pop results of subjects with at least 4,000 fingerprint SNPs genotyped across 478 dbGaP studies. (A, B) All subjects are plotted. (C, D) 10,000 randomly selected subjects are plotted. Cyan lines show the cutoff standards suggested by GRAF-pop to assign dbGaP subjects into GRAF-calculated populations.</p><div class="sb-div caption-clear"></div></div></div><p id="p-54"><a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> shows the distributions of GRAF-pop scores of eight dbGaP study-reported populations: European, African (represented by study-reported populations Ghana and Yoruba), East Asian (represented by Chinese and Japanese), African American (including African), Latin American 1 (represented by Puerto Rican and Dominican), Latin American 2 (represented by Mexican and Mexican American), South Asian (represented by Asian Indian and Pakistani), and Asian-Pacific Islander (including East Asian but not South Asian) . For each population, 1,000 randomly selected subjects are plotted.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2447/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of GRAF-pop scores of subjects with different groups of study-reported population of subjects across 478 dbGaP studies. Each ellipse in panel (A) shows the area including 95% of the subjects for a certain study-reported population. Only 1,000 randomly selected subjects are plotted for each population to avoid overplotting. Study-reported populations are grouped and harmonized. The population Asian-Pacific Islander does not include South Asian subjects." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1041302944" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of GRAF-pop scores of subjects with different groups of study-reported population of subjects across 478 dbGaP studies. Each ellipse in panel (A) shows the area including 95% of the subjects for a certain study-reported population. Only 1,000 randomly selected subjects are plotted for each population to avoid overplotting. Study-reported populations are grouped and harmonized. The population Asian-Pacific Islander does not include South Asian subjects.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2447/F2.medium.gif" width="336" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/9/8/2447/F2.medium.gif" width="336" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2447/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2447/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493911" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-55" class="first-child">Distribution of GRAF-pop scores of subjects with different groups of study-reported population of subjects across 478 dbGaP studies. Each ellipse in panel (A) shows the area including 95% of the subjects for a certain study-reported population. Only 1,000 randomly selected subjects are plotted for each population to avoid overplotting. Study-reported populations are grouped and harmonized. The population Asian-Pacific Islander does not include South Asian subjects.</p><div class="sb-div caption-clear"></div></div></div><p id="p-56">In <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a>, we can see that dbGaP subjects are separated into six clusters by GD1, GD2 and GD4. On the GD2 <em>vs.</em> GD1 plots, three main clusters at the vertices of the reference triangle, corresponding to continental populations European, African/African American, and Asian. Below the European cluster, there are two clusters. The left cluster in <a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a> has more African components, and the location is corresponding to that of the study-reported Latin American 1 in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>. The right cluster is located in the area where the study-reported South Asians, and Latin American 2 are found in <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>. On the GD4 <em>vs.</em> GD1 plots, the right cluster is separated into three sub-clusters: the top one corresponds to South Asian and the bottom one to Latin American 2. The middle sub-cluster is connected to the Asian-Pacific Islander cluster. The Latin American 1, Latin American 2, and South Asian populations are not homogeneous, so the tightness of the clustering is surprising. We emphasize that the individuals sampled in <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a> represent the study-reported populations in dbGaP, but may not comprise a representative sample of these populations as a whole. Naturally, dbGaP staff must check all submissions of data sets, and accept all submissions of data sets that pass quality control, but it would be neither appropriate nor possible (since any research group can submit data) for dbGaP staff to check whether submitted data sets are representative. Analogously, sequence databases maintained at NCBI have a gross excess of sequences from model organisms out of proportion with the diversity of organisms on earth.</p><p id="p-57">We assign each dbGaP subject a unique population ID using the thresholds in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> and <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>. These simple cutoff lines, plotted in <a id="xref-fig-1-4" class="xref-fig" href="#F1">Figure 1</a> using cyan lines, were set to be able to separate the clusters of dbGaP subjects shown in <a id="xref-fig-1-5" class="xref-fig" href="#F1">Figure 1</a>, and the ellipses that include 95% subjects of populations shown in <a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>. There are no clear separations between African (PopID 2) and African American (PopID 4), and between East Asian (PopID 3) and Asian-Pacific Islanders (not including East Asian, PopID 7). Note that these GRAF-calculated populations, different from the eight study-reported populations used to calculate the ellipse areas in <a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2</a>, are mutually exclusive. For example, the GRAF-calculated Asian-Pacific Islander population does not include East Asian or South Asian subjects, but in <a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2</a>, the ellipse area of Asian-Pacific Islander includes subjects from both East Asia and other Asia-Pacific regions.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493925/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493925/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Populations assigned by GRAF-pop based on the estimated ancestry proportions</span><div class="sb-div caption-clear"></div></div></div><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493895/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493895/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Separating Asians and Hispanics using GD1 and GD4 scores</span><div class="sb-div caption-clear"></div></div></div><p id="p-59">The cutoff lines were determined empirically to aid in grouping the dbGaP subjects. We sought to base the cutoff lines and curves on as few parameters as possible, using the ancestry proportions and GD1-GD4. These lines were selected based on the distribution of dbGaP subjects with known study-reported populations on the GD spaces (<a id="xref-fig-1-6" class="xref-fig" href="#F1">Figures 1</a> and <a id="xref-fig-2-8" class="xref-fig" href="#F2">2</a>). When selecting these cutoff lines, we paid some special attention to the ellipse areas that include 95% of the subjects with the eight known populations. The lines are included in the GRAF software package as a suggested cutoff standard to assign subjects to populations. Other users should only use cutoff lines, as well as the ellipses, as references to compare their own data with dbGaP. GRAF-pop users are supposed to plot the GD scores and ancestry proportions calculated by GRAF-pop and see how the subjects in their own data sets are clustered in the GD space and then select their own standards to group the subjects based on the purposes of data usage.</p><p id="p-60"><a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> shows the percentages of subjects, among those with â¥ 4,000 fingerprint SNPs genotyped, assigned to different population IDs for different study-reported populations. <a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a> includes the 94 study-reported populations with â¥ 100 subjects with â¥ 4,000 fingerprint SNPs genotyped. We combined the study-reported populations and GRAF-assigned populations into five continental populations and estimated the prediction accuracies of GRAF-pop.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493849/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493849/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Prediction accuracies (%) of GRAF-pop for continental populations</span><div class="sb-div caption-clear"></div></div></div><p id="p-62">Assuming the study-reported populations correspond to the expected population based on genetic marker data, GRAF-pop correctly determined populations for more than 98% of European and 97% of Asian and African American subjects. The prediction accuracy for South Asian subjects is also greater than 98%, but the number of subjects is small. Approximately 98% of the Asian subjects were predicted by GRAF-pop either as Asians or Asian-Pacific Islanders or South Asians. These predictions were consistent since self-reported Asians can be South Asians. It was most difficult to predict the population of subjects reported as âHispanicsâ, or Latin Americans as the harmonized term, which is not surprising considering the complex demographic history of Latin America. Only 80% of study-reported Latin American subjects were predicted by GRAF-pop as Latin American 1 or Latin American 2. Most of the remaining were classified as European, African, or Other, with a small percentage (0.41%) reported by GRAF-pop as Asians or South Asians.</p><p id="p-63">Study-reported populations are sometimes misassigned which makes it difficult to use them to evaluate the performance of GRAF-pop. Figure S1 shows the distributions of subjects with different study-reported populations on GRAF-pop scatter plots. Some subjects with the same study-reported populations are separated by GRAF-pop into several clusters. For example, Figure S1C shows that there is a small cluster in the position near the centroid of Europeans for subjects with a study-reported population of African. It is likely that these subjects are Europeans misassigned as Africans in the dbGaP submission. Figure S1E and S1F shows subjects reported as Asians that are separated into several clusters by GRAF-pop. The top cluster on Figure S1F is in the position near the centroid of South Asian. It is understandable that South Asian participants were reported by the study as Asians. However, the bottom cluster is located where Latin Americans are expected, therefore they are most likely also misassigned. If GRAF-pop predictions are correct, then a possible explanation is that these study-reported populations were estimated by data submitters using genotypes. When PCA is used to infer ancestry and only the first two PC scores are used, it is difficult to distinguish Latin Americans, South Asians and European/East Asian admixtures from one another (see <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6B</a> below).</p><p id="p-64">The study-reported populations of the 1000 Genomes Project (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Clarke <em>et al.</em> 2012</a>) individuals are more reliable. We used genotype data of 2,504 individuals from 1000 Genomes as a test for GRAF-pop. For HapMap defined âsuper populationsâ EUR, AFR, EAS, SAS (<a href="http://www.internationalgenome.org/category/population/">http://www.internationalgenome.org/category/population/</a>), the accuracies are greater than 99% (<a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a>). For the admixed population AMR, more than 81% are predicted as Latin Americans. The super population AFR includes two admixed populations from Americas: ASW (Americans of African ancestry in SW USA) and ACB (African Caribbeans in Barbados). If we exclude these two populations from AFR, then the prediction precision of GRAF-pop in non-admixed continental populations is 100% (see Figures S2A and S2B).</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/493906/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/493906/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Prediction accuracies (%) of GRAF-pop using data of 1000 Genomes Project</span><div class="sb-div caption-clear"></div></div></div><p id="p-65">Among 1.2 million dbGaP subjects with â¥4000 fingerprint SNPs, about 76% are Europeans, and the percentages of African Americans, Latin Americans and Asians are about 12%, 6% and 5%, respectively.</p></div><div id="sec-12" class="subsection"><h3>Populations of dbGaP subjects predicted by GRAF-pop for each dbGaP study</h3><p id="p-66">For each dbGaP study with genotype data submitted, we use GRAF-pop to estimate subject populations and display the results on the study report page within the dbGaP web site. As an example, <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows population reports generated by GRAF-pop for study phs000788.v1.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2447/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Subject populations determined by GRAF-pop for dbGaP study phs000788.v1: Research Program on Genes, Environment and Health (RPGEH). Each subject has 4028 &#x2013; 5872 (average 4239) fingerprint SNPs with genotypes. In panels (A-C) and (G-I), study-reported populations are color coded. In panels (D-G), GRAF-calculated populations are color coded. Black cutoff lines are the same as those in Figure 1. Each ellipse shows the area including 95% of the subjects for a study-reported population European, African American, Asian-Pacific Islander, or Latin American 2, same as in Figure 2. Total 78,419 subjects with the following study-reported populations are plotted: White: 65,540 subjects; Asian: 5,510; Hispanic: 4,641; Black: 2,183; Other 545." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1041302944" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Subject populations determined by GRAF-pop for dbGaP study phs000788.v1: Research Program on Genes, Environment and Health (RPGEH). Each subject has 4028 &#x2013; 5872 (average 4239) fingerprint SNPs with genotypes. In panels (A-C) and (G-I), study-reported populations are color coded. In panels (D-G), GRAF-calculated populations are color coded. Black cutoff lines are the same as those in Figure 1. Each ellipse shows the area including 95% of the subjects for a study-reported population European, African American, Asian-Pacific Islander, or Latin American 2, same as in Figure 2. Total 78,419 subjects with the following study-reported populations are plotted: White: 65,540 subjects; Asian: 5,510; Hispanic: 4,641; Black: 2,183; Other 545.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2447/F3.medium.gif" width="440" height="310"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/9/8/2447/F3.medium.gif" width="440" height="310"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2447/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2447/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493894" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-67" class="first-child">Subject populations determined by GRAF-pop for dbGaP study phs000788.v1: Research Program on Genes, Environment and Health (RPGEH). Each subject has 4028 â 5872 (average 4239) fingerprint SNPs with genotypes. In panels (A-C) and (G-I), study-reported populations are color coded. In panels (D-G), GRAF-calculated populations are color coded. Black cutoff lines are the same as those in <a id="xref-fig-1-7" class="xref-fig" href="#F1">Figure 1</a>. Each ellipse shows the area including 95% of the subjects for a study-reported population European, African American, Asian-Pacific Islander, or Latin American 2, same as in <a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2</a>. Total 78,419 subjects with the following study-reported populations are plotted: White: 65,540 subjects; Asian: 5,510; Hispanic: 4,641; Black: 2,183; Other 545.</p><div class="sb-div caption-clear"></div></div></div><p id="p-68">The study phs000788.v1 is a multi-ethnic study with 78,419 genotyped subjects with an average of 4,259 fingerprint SNPs genotyped per subject. Most subjects are reported as having one of four populations: White, Asian, Hispanic, or Black. <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a> shows that most of the subjects are within the areas expected. Each panel G-I plots subjects with one study-reported population. Panel G indicates that some of the study-reported Asians have mostly European backgrounds. The cluster between the European and Asian vertices most likely contains individuals who have both European and Asian ancestry. The subjects within the South Asian expected area are probably South Asians reported as Asians by the data submitter. <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a> shows the graph after it is rotated around the bottom of the triangle by 90Â°; <em>i.e.</em>, the GD3 values are shown on the <em>y</em>-axis. Hispanics are separated from European/Asian admixtures in the rotated graph. <a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a> plots GD4 values on the <em>y</em>-axis. Study-reported Hispanics are almost completely separated from Asians, including East Asians, European/Asian admixtures, and South Asians. Panels D-F color code the subjects by populations computed by GRAF-pop. The black lines are the cutoff lines to separate different populations. <a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3F</a> shows that the cluster of study-reported Asians near the expected area for Europeans is composed of South Asians.</p><p id="p-69">Different numbers of fingerprint SNPs are genotyped across dbGaP studies. We selected studies with fingerprint SNPs ranging from fewer than 200 to almost all 10,000 SNPs. <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> shows the population reports generated by GRAF-pop and displayed on dbGaP webpages for these studies. European subjects are restricted in a small area if there are more than 2,000 fingerprint SNPs genotyped. The subjects become more widely spread out when there are fewer than 1,000 SNPs genotyped. Roughly 150 fingerprint SNPs genotyped suffice to separate the three main populations (European, African American, and Asian).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2447/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="GRAF-pop results of dbGaP studies with different numbers of fingerprint SNPs genotyped. Each panel shows result of one dbGaP study. Study-reported populations are color coded. Each ellipse shows the area including 95% of the subjects for a study-reported population European, African American, Asian-Pacific Islander, same as in Figure 2. (A) Study accession: phs000169.v1. White: 1,663 subjects. Black: 1,139. (B) phs000944.v1. White: 4,132. Black: 2,665. Asian: 81. (C) phs001306.v1. White: 9,695. (D) phs000281.v8. White: 1,020. Black: 884. (E) phs000918.v1. White: 11,402. (F) phs000948.v1. White: 1,786. Black: 49. Asian: 69. From Panels (A) to (F), the average fingerprint SNPs with genotypes per subject are: 9982, 5072, 2040, 471, 244, and 157, respectively." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1041302944" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;GRAF-pop results of dbGaP studies with different numbers of fingerprint SNPs genotyped. Each panel shows result of one dbGaP study. Study-reported populations are color coded. Each ellipse shows the area including 95% of the subjects for a study-reported population European, African American, Asian-Pacific Islander, same as in Figure 2. (A) Study accession: phs000169.v1. White: 1,663 subjects. Black: 1,139. (B) phs000944.v1. White: 4,132. Black: 2,665. Asian: 81. (C) phs001306.v1. White: 9,695. (D) phs000281.v8. White: 1,020. Black: 884. (E) phs000918.v1. White: 11,402. (F) phs000948.v1. White: 1,786. Black: 49. Asian: 69. From Panels (A) to (F), the average fingerprint SNPs with genotypes per subject are: 9982, 5072, 2040, 471, 244, and 157, respectively.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2447/F4.medium.gif" width="440" height="419"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/9/8/2447/F4.medium.gif" width="440" height="419"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2447/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2447/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493977" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-70" class="first-child">GRAF-pop results of dbGaP studies with different numbers of fingerprint SNPs genotyped. Each panel shows result of one dbGaP study. Study-reported populations are color coded. Each ellipse shows the area including 95% of the subjects for a study-reported population European, African American, Asian-Pacific Islander, same as in <a id="xref-fig-2-10" class="xref-fig" href="#F2">Figure 2</a>. (A) Study accession: phs000169.v1. White: 1,663 subjects. Black: 1,139. (B) phs000944.v1. White: 4,132. Black: 2,665. Asian: 81. (C) phs001306.v1. White: 9,695. (D) phs000281.v8. White: 1,020. Black: 884. (E) phs000918.v1. White: 11,402. (F) phs000948.v1. White: 1,786. Black: 49. Asian: 69. From Panels (A) to (F), the average fingerprint SNPs with genotypes per subject are: 9982, 5072, 2040, 471, 244, and 157, respectively.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Comparing GRAF-pop to other software tools</h3><p id="p-71">We compared performance of GRAF-pop to EIGENSTRAT, FastPCA, and SNPweights included in the EIGENSOFT 7.2.1 package (<a href="https://www.hsph.harvard.edu/alkes-price/software/">https://www.hsph.harvard.edu/alkes-price/software/</a>), as well as FlashPCA2 (<a href="https://github.com/gabraham/flashpca">https://github.com/gabraham/flashpca</a>, version 2.0). We used data from the following five dbGaP studies to test these programs: phs000050.v1.p1, phs000403.v3.p3, phs000420.v6.p3, phs000456.v1.p1 and phs000517.v3.p1. These studies include subjects of all continental populations, and genotype datasets covering both large (&gt;80%) and small (&lt;5%) portions of the fingerprint SNPs. The 270 subjects of phs000050 were from HapMap Phase I and were genotyped using the PERLEGEN-600K platform. We used HapMap data as a cross-dataset positive control because the population assignments are reliable and have been used to test other ancestry inference software. Phs000420 (NHLBI MESA SHARe) is a sub-study of the Multi-Ethnic Study of Atherosclerosis (MESA) Cohort (phs000209.v13.p3) with 8,295 subjects genotyped using the AFFY_6.0 platform. Phs000403 (NHLBI GO-ESP) is another sub-study of MESA, with genotypes obtained using exome sequencing platform Genome Analyzer IIX for 404 subjects. Phs000456 (Risk Assessment of Cerebrovascular Events (RACE) Study) contains 2,493 Pakistani subjects, and phs000517 (Multiethnic Cohort (MEC) Breast Cancer Genetics) includes 3,708 subjects with study-reported populations Black or African American, Latino and Japanese. Both RACE and MEC Breast Cancer studies genotyped subjects using the platform Human660W-Quad_v1_A. Table S2 shows a summary of the five selected studies. For NHLBI MESA SHARe, we combined study-reported populations âAfrican Americanâ, âBlackâ, and âBlack, African-Americanâ into âAfrican Americanâ, and we combined âWhite, Caucasianâ and âEuropean Americanâ into âEuropean Americanâ.</p><p id="p-72">We extracted the genotypes of the 10,000 fingerprint SNPs and filtered out SNPs with genotype missing rates greater than 90%. Two PLINK sets were created to compare the software tools. The first dataset contains combined genotypes from HapMap, NHLBI GO-ESP and NHLBI MESA SHARe. It consists of genotypes of 3,343 SNPs and 8,898 subjects; the counts of subjects from HapMap, NHLBI GO-ESP and NHLBI MESA SHARe are 270, 404 and 8224, respectively. While subjects of HapMap and NHLBI MESA SHARe have genotype missing rates &lt; 0.1%, the NHLBI GO-ESP subjects have genotype missing rate &gt; 95%, which is useful to test the software tools when the genotype missing rates are high. The second dataset was generated by combining genotypes of HapMap, RACE Study and MEC Breast Cancer, containing genotypes of 7,970 SNPs and 6,616 subjects, with genotype missing rate &lt; 5% for all subjects.</p><p id="p-73">We deliberately chose to show the performance of GRAF-pop on real individual data sets in dbGaP, rather than on blended or simulated data sets. The decision to use individual data sets refleects the reality of the quality-control process. Each data set is idiosyncratic and each submitting group may use different procedures to assign study-reported populations and may make unique errors in managing data. Some of the systematic submitter errors that GRAF-pop has detected would have been quite unrealistic to simulate.</p><p id="p-74"><a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> shows results from SNPweights, EIGENSTRAT, and GRAF-pop using the first dataset. Comparing the normalized results of EIGENSTRAT (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5D</a>) and those of GRAF-pop (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5E</a>), we can see that both methods can separate the continental populations from one another well, when genotype missing rates are low. The resolutions of EIGENSTRAT and GRAF-pop are similar, and both are much higher than the resolution of SNPweights (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5A-B</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2447/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results generated by applying SNPweights, EIGENSTRAT and GRAF-pop on the combined dataset of phs000050.v1 (HapMap), phs000403.v3 (NHLBI GO-ESP) and phs000420.v6 (NHLBI MESA SHARe) to infer subject ancestry. Average number of SNPs with genotypes per subject in the test data set: HapMap and NHLBI MESA SHARe: 3,433; NHLBI GO-ESP: 160. For ease of comparison, SNPweights and EIGENSTRAT results are plotted on the three-population triangle as explained in Methods. (A) SNPweights, PC2 vs. PC1. (B) SNPweights, after normalization onto the GRAF-pop results. (C) EIGENSTRAT, PC2 vs. PC1. (D) EIGENSTRAT, after normalization. (E) GRAF-pop, GD2 vs. GD1. The two small triangles on Panels (B, D, E), each of which is pointed by a black arrow, show the centroids of the subject locations of the two study-reported populations in NHLBI GO-ESP predicted by each software tool: blue triangle for European, brown for African American. Since the subjects in NHLBI GO-ESP have very high genotype missing rates, these centroids indicate whether a software tool can estimate subject ancestry unbiasedly in the presence of missing genotypes." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1041302944" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Results generated by applying SNPweights, EIGENSTRAT and GRAF-pop on the combined dataset of phs000050.v1 (HapMap), phs000403.v3 (NHLBI GO-ESP) and phs000420.v6 (NHLBI MESA SHARe) to infer subject ancestry. Average number of SNPs with genotypes per subject in the test data set: HapMap and NHLBI MESA SHARe: 3,433; NHLBI GO-ESP: 160. For ease of comparison, SNPweights and EIGENSTRAT results are plotted on the three-population triangle as explained in Methods. (A) SNPweights, PC2 &lt;em&gt;vs.&lt;/em&gt; PC1. (B) SNPweights, after normalization onto the GRAF-pop results. (C) EIGENSTRAT, PC2 &lt;em&gt;vs.&lt;/em&gt; PC1. (D) EIGENSTRAT, after normalization. (E) GRAF-pop, GD2 &lt;em&gt;vs.&lt;/em&gt; GD1. The two small triangles on Panels (B, D, E), each of which is pointed by a black arrow, show the centroids of the subject locations of the two study-reported populations in NHLBI GO-ESP predicted by each software tool: blue triangle for European, brown for African American. Since the subjects in NHLBI GO-ESP have very high genotype missing rates, these centroids indicate whether a software tool can estimate subject ancestry unbiasedly in the presence of missing genotypes.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2447/F5.medium.gif" width="424" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/9/8/2447/F5.medium.gif" width="424" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2447/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2447/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493890" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-75" class="first-child">Results generated by applying SNPweights, EIGENSTRAT and GRAF-pop on the combined dataset of phs000050.v1 (HapMap), phs000403.v3 (NHLBI GO-ESP) and phs000420.v6 (NHLBI MESA SHARe) to infer subject ancestry. Average number of SNPs with genotypes per subject in the test data set: HapMap and NHLBI MESA SHARe: 3,433; NHLBI GO-ESP: 160. For ease of comparison, SNPweights and EIGENSTRAT results are plotted on the three-population triangle as explained in Methods. (A) SNPweights, PC2 <em>vs.</em> PC1. (B) SNPweights, after normalization onto the GRAF-pop results. (C) EIGENSTRAT, PC2 <em>vs.</em> PC1. (D) EIGENSTRAT, after normalization. (E) GRAF-pop, GD2 <em>vs.</em> GD1. The two small triangles on Panels (B, D, E), each of which is pointed by a black arrow, show the centroids of the subject locations of the two study-reported populations in NHLBI GO-ESP predicted by each software tool: blue triangle for European, brown for African American. Since the subjects in NHLBI GO-ESP have very high genotype missing rates, these centroids indicate whether a software tool can estimate subject ancestry unbiasedly in the presence of missing genotypes.</p><div class="sb-div caption-clear"></div></div></div><p id="p-76">When genotype missing rates are high, as in NHLBI GO-ESP, EIGENSTRAT did not determine the populations correctly. All subjects from NHLBI GO-ESP, both European Americans or African Americans, were placed near the center (<a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5C-D</a>). Similar results were obtained by running EIGENSTRAT with option <em>lsqproject</em>. SNPweights put these subjects in widely spread areas, with centroids of European Americans and African Americans far from the centroids of the same populations in other studies (<a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5A-B</a>). Only GRAF-pop could clearly separate African Americans from European Americans, and the centroids of these two populations were close to those from other studies (see the two small triangles pointed by black arrows in panels B, D, E).</p><p id="p-77"><a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a> compares the results of EIGENSTRAT and GRAF-pop on the combined dataset of HapMap, RACE Study and MEC Breast Cancer. Panels A-C show that study-reported Europeans, African Americans, Asians, and Hispanics can be well separated from one another using PC1 and PC2, or the GD1 and GD2 scores of GRAF-pop. The resolutions of these two programs were close. However, South Asians (Pakistanis in this test) could not be well separated from Hispanics. <a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6D</a> shows that South Asians could be separated from Hispanics using PC3 of EIGENSTRAT. Similarly, <a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6E</a> shows that these two populations could be distinguished from each other using the GD4 scores of GRAF-pop.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2447/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Results generated by applying EIGENSTRAT and GRAF-pop on the combined dataset of phs000050.v1 (HapMap), phs000456.v1 (Risk Assessment of Cerebrovascular Events (RACE) Study) and phs000517.v3 (Multiethnic Cohort (MEC) Breast Cancer Genetics) to infer subject ancestry. 7,790 SNPs are included in the test data set, with genotype missing rate less than 5% for each subject. (A) EIGENSTRAT, PC2 vs. PC1. (B) EIGENSTRAT, PC2 vs. PC1, after normalization. (C) GRAF-pop, GD2 vs. GD1. (D) EIGENSTRAT, PC3 vs. PC1. (E) GRAF-pop, GD4 vs. GD1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1041302944" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Results generated by applying EIGENSTRAT and GRAF-pop on the combined dataset of phs000050.v1 (HapMap), phs000456.v1 (Risk Assessment of Cerebrovascular Events (RACE) Study) and phs000517.v3 (Multiethnic Cohort (MEC) Breast Cancer Genetics) to infer subject ancestry. 7,790 SNPs are included in the test data set, with genotype missing rate less than 5% for each subject. (A) EIGENSTRAT, PC2 &lt;em&gt;vs.&lt;/em&gt; PC1. (B) EIGENSTRAT, PC2 &lt;em&gt;vs.&lt;/em&gt; PC1, after normalization. (C) GRAF-pop, GD2 &lt;em&gt;vs.&lt;/em&gt; GD1. (D) EIGENSTRAT, PC3 &lt;em&gt;vs.&lt;/em&gt; PC1. (E) GRAF-pop, GD4 &lt;em&gt;vs.&lt;/em&gt; GD1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2447/F6.medium.gif" width="440" height="437"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/9/8/2447/F6.medium.gif" width="440" height="437"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2447/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2447/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493891" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-78" class="first-child">Results generated by applying EIGENSTRAT and GRAF-pop on the combined dataset of phs000050.v1 (HapMap), phs000456.v1 (Risk Assessment of Cerebrovascular Events (RACE) Study) and phs000517.v3 (Multiethnic Cohort (MEC) Breast Cancer Genetics) to infer subject ancestry. 7,790 SNPs are included in the test data set, with genotype missing rate less than 5% for each subject. (A) EIGENSTRAT, PC2 <em>vs.</em> PC1. (B) EIGENSTRAT, PC2 <em>vs.</em> PC1, after normalization. (C) GRAF-pop, GD2 <em>vs.</em> GD1. (D) EIGENSTRAT, PC3 <em>vs.</em> PC1. (E) GRAF-pop, GD4 <em>vs.</em> GD1.</p><div class="sb-div caption-clear"></div></div></div><p id="p-79">We also tested FastPCA and FlashPCA2. For all the datasets tested, FastPCA and FlashPCA2 generated results almost identical to EIGENSTRAT, which is not surprising since they all use PCA.</p><p id="p-80">We compared the running times of these software tools on an Intel Xeon machine with 16 2.67 GHz CPUs using the combined dataset of HapMap, NHLBI GO-ESP and NHLBI MESA SHARe (8,898 individuals). The running times were: FlashPCA2, 8 sec; FastPCA, 22 sec; GRAF-pop, 75 sec; SNPweights, 9 min; EIGENSTRAT, 94 min. Only 3,343 SNPs with low genotype missing rates are included in the dataset. FlashPCA2 and FastPCA only need to analyze genotypes of these SNPs, but GRAF-pop checks genotypes of all 10,000 fingerprint SNPs.</p><p id="p-81">We also compared GRAF-pop with the widely-used, model-based method ADMIXTURE (v.1.3.0, <a href="http://software.genetics.ucla.edu/admixture/download.html">http://software.genetics.ucla.edu/admixture/download.html</a>). The above first dataset containing combined genotypes from HapMap, NHLBI GO-ESP and NHLBI MESA SHARe was used for testing ADMIXTURE. Since many of the 14,003 subjects in the dataset are closely related, the data set had to be reduced to meet ADMIXTUREâs requirements. We used our GRAF software to create a subset of data of 6,974 subjects, among which no two subjects are closely related (<em>i.e.</em>, with HGMR &lt; 15%). ADMIXTURE was run using option <em>k</em> = 3. It converged in 36 iterations with running time 139 sec. GRAF-pop spent 36 sec to calculate the results. <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a> shows that the results obtained using ADMIXTURE and GRAF-pop are very similar, including the NHLBI GO-ESP subject with very high genotype missing rates. Both methods can easily cluster the non-admixed populations correctly. Out of the 1,410 subjects with study-reported population Hispanic, 74 and 6 subjects are determined to be Europeans and Asians, respectively, by GRAF-pop using the default cutoff values. For those subjects, very high European (86%) and Asian (87%) ancestry proportions are also predicted by ADMIXTURE.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/9/8/2447/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of population structure determined using ADMIXTURE and GRAF-pop. The data set tested contains genotypes of 3,433 well separated SNPs and 6,794 unrelated subjects from dbGaP studies HapMap, NHLBI MESA SHARe and NHLBI GO-ESP. Subjects from NHLBI GO-ESP have average 160 SNPs with genotypes, while subjects from the other two studies have average genotype missing rate &lt; 0.1%. Each bar represents one subject, and results of same subject are plotted on the same horizontal position to compare the two software tools. For NHLBI MESA SHARe, only about 40% randomly selected subjects are plotted. Three colors are used to represent the three ancestry components: light purple: European; purple: African; mid blue: East Asian." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1041302944" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Comparison of population structure determined using ADMIXTURE and GRAF-pop. The data set tested contains genotypes of 3,433 well separated SNPs and 6,794 unrelated subjects from dbGaP studies HapMap, NHLBI MESA SHARe and NHLBI GO-ESP. Subjects from NHLBI GO-ESP have average 160 SNPs with genotypes, while subjects from the other two studies have average genotype missing rate &lt; 0.1%. Each bar represents one subject, and results of same subject are plotted on the same horizontal position to compare the two software tools. For NHLBI MESA SHARe, only about 40% randomly selected subjects are plotted. Three colors are used to represent the three ancestry components: light purple: European; purple: African; mid blue: East Asian.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/9/8/2447/F7.medium.gif" width="410" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/9/8/2447/F7.medium.gif" width="410" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/9/8/2447/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/9/8/2447/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/493954" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-82" class="first-child">Comparison of population structure determined using ADMIXTURE and GRAF-pop. The data set tested contains genotypes of 3,433 well separated SNPs and 6,794 unrelated subjects from dbGaP studies HapMap, NHLBI MESA SHARe and NHLBI GO-ESP. Subjects from NHLBI GO-ESP have average 160 SNPs with genotypes, while subjects from the other two studies have average genotype missing rate &lt; 0.1%. Each bar represents one subject, and results of same subject are plotted on the same horizontal position to compare the two software tools. For NHLBI MESA SHARe, only about 40% randomly selected subjects are plotted. Three colors are used to represent the three ancestry components: light purple: European; purple: African; mid blue: East Asian.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Applying GRAF-pop in dbGaP curation</h3><p id="p-83">During development, GRAF-pop was used informally to harmonize population values for searching and as a quality control tool to check study-reported populations in newly submitted dbGaP studies. Advice from curators aided improvements to the visualization tools illustrated in the Figures. Since February 2018, GRAF-pop has been used formally to check the genotype data and population assignments before each new study is publicly released by dbGaP.</p><p id="p-84">Populations predicted by GRAF-pop and those reported in the submitted files are displayed and compared on a webpage for curators and submitters to use to find potential errors in the phenotype and genotype files. The page also highlights missing information to be filled in before the submission is complete.</p><p id="p-85">Below is an example URL for one of these webpages:</p><p id="p-86"><a href="https://www.ncbi.nlm.nih.gov/projects/gap/population/cgi-bin/StudySubjectAncestry.cgi?phs=420&amp;version=6&amp;exp1=1&amp;exp2=1&amp;exp3=1">https://www.ncbi.nlm.nih.gov/projects/gap/population/cgi-bin/StudySubjectAncestry.cgi?phs=420&amp;version=6&amp;exp1=1&amp;exp2=1&amp;exp3=1</a></p><p id="p-87">Our expectation has been that GRAF-pop would catch omissions and errors in submittersâ assignment of subject populations. GRAF-pop has already also identified some inconsistencies in the content of PLINK-formatted genotype files. GRAF-pop can flag files that have a syntactically valid PLINK format, but have incorrect marker assignments, which leads to unexpected GRAF-pop predictions of subject population.</p></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-88">PCA is a linear algebraic method applied in many scientific disciplines (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Jolliffe and Cadima 2016</a>), and has long been used to analyze genetic data (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Menozzi <em>et al.</em> 1978</a>). Since the development of EIGENSOFT (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Patterson <em>et al.</em> 2006</a>; <a id="xref-ref-35-4" class="xref-bibr" href="#ref-35">Price <em>et al.</em> 2006</a>), the use of PCA has become central to many ancestry inference methods. However, PCA has two important limitations when applied to ancestry inference.</p><p id="p-89">One limitation is that PCA is a dimensionality reduction method without a statistical model. Mcvean (2009) has shown that PCA scores give an accurate representation of the parameters of the coalescent process. However, PCA scores do not have physical meaning. They can only be used to separate individuals within the dataset being analyzed (<a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Lee <em>et al.</em> 2010</a>), and therefore, cannot be generalized across datasets. Recent methods, such as SNPweights (<a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Chen <em>et al.</em> 2013</a>) and FastPop (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Li <em>et al.</em> 2016</a>) mitigate this limitation by using reference panels as well as algorithmic steps before and after PCA.</p><p id="p-90">The second limitation is that PCA has difficulty with missing data (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Stanimirova <em>et al.</em> 2007</a>). Most of the existing PCA ancestry inference approaches filter out individuals and SNPs with missing rates greater than some threshold, usually 5% or 1%. Missing values remaining in the datasets are then replaced with the mean value (<a id="xref-ref-35-5" class="xref-bibr" href="#ref-35">Price <em>et al.</em> 2006</a>; <a id="xref-ref-10-6" class="xref-bibr" href="#ref-10">Chen <em>et al.</em> 2013</a>; <a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Li <em>et al.</em> 2016</a>; <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Byun <em>et al.</em> 2017</a>), which shifts individuals toward the centroid in the scatter plot generated by PCA. Li <em>et al.</em> noticed that false populations were reported by FastPop when samples with genotype missing rates higher than 5% were included (<a id="xref-ref-26-4" class="xref-bibr" href="#ref-26">Li <em>et al.</em> 2016</a>).</p><p id="p-91">To infer the population assignment for dbGaP subjects genotyped on various platforms, we have developed an inference method named GRAF-pop to improve quality control in curation, and harmonize subject population values across studies. It uses a distance-based approach that overcomes these two limitations of PCA. Instead of comparing an individualâs genotypes with those of other individuals in the dataset being analyzed, GRAF-pop compares the individuals to fixed sets of reference panels with known populations. Since the number of human populations at a coarse level is limited, the genetic distance data generated by GRAF-pop is low-dimensional, and PCA is not necessary. GRAF-pop utilizes dbGaP subjects with known populations as references. Due to the paucity of Native American samples, we calculate the genetic distances to three reference continental populations: European, African, and Asian, and obtain results of data in 3D. By projecting the initial prediction onto the fixed EFA triangle, GRAF-pop produces results that can be generalized across different datasets. <a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5</a> and <a id="xref-fig-6-5" class="xref-fig" href="#F6">Figure 6</a> show that the GRAF-pop GD2 <em>vs.</em> GD1 plots, which show the EFA projection, separate continental populations from one another almost as well as the PC2 <em>vs.</em> PC1 plots, which show the PCA 2D projection to the plane of the first two PCs. Using the EFA triangle, we can also estimate the ancestral proportions by calculating barycentric coordinates.</p><p id="p-92">Barycentric coordinates were developed by August Ferdinand MÃ¶bius (1790-1868) in his book â<em>Der barycentrische calcul</em>â (1827) (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Ungar 2010</a>). Nowadays, MÃ¶bius is better known for discovering the non-orientable three-dimensional shape known as the MÃ¶bius strip, which has been cut in three pieces and stylized into the universal symbol for recycling. In that spirit, our work recycles the technique of barycentric coordinates for application to ancestry inference. Readers may have seen barycentric coordinates used in computer graphics and animation to interpolate colors over any convex shape and to deform shapes smoothly, and for other applications (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Warren <em>et al.</em> 2007</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Skala 2008</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Weber <em>et al.</em> 2009</a>).</p><p id="p-93">For a triangle on a 2D plane, barycentric coordinates solve the following problem: given a point, <em>p</em>, inside the triangle, find the unique masses of three objects summing to 1 such that when they are placed on the three vertices, <em>p</em> is the center of mass. Applying barycentric coordinates to the EFA triangle and treating the mass proportions as the ancestral proportions, we can estimate the population components for each individual based on the projection of that individualâs distance data relative to the vertices of the reference triangle.</p><p id="p-94">Both SNPweights (<a id="xref-ref-10-7" class="xref-bibr" href="#ref-10">Chen <em>et al.</em> 2013</a>) and FastPop (<a id="xref-ref-26-5" class="xref-bibr" href="#ref-26">Li <em>et al.</em> 2016</a>) estimate subject ancestral proportions using distances from the subject to the centroids of reference populations. Surprisingly, these two methods give two <em>different algebraic</em> solutions. In each paper, the necessary algebra is shown, but without any justification or comparison to the alternative. One of our contributions is to point out that the method of SNPweights has a natural <em>geometric</em> interpretation as barycentric coordinates. Usage of barycentric coordinates allows GRAF-pop to characterize other populations beyond E, F, A, geometrically (see Results), unlike SNPweights and FastPop.</p><p id="p-95">GRAF-pop also uses barycentric coordinates to treat the missing data problem. By using reference panels, GRAF-pop can infer ancestry proportions for individuals independent of others in the dataset. In contrast to SNPweights and FastPop, which require that the same SNPs be genotyped in all target individuals, GRAF-pop does not need to impute missing data. GRAF-pop simply skips the SNPs without genotypes. When the average genetic distances from individuals in each vertex subpopulation to the reference population are available for the subset of genotyped SNPs, then barycentric coordinates can be calculated using these mean distances as the reference triangle. <em>Expected</em> mean values for a subject group can be predicted using the allele frequencies of the group, for any combination of SNPs. We use the expected mean genetic distances from the three subject groups to the three reference populations to build the reference triangle. The GD2 <em>vs.</em> GD1 scatter plots of ancestry results of dbGaP studies calculated using GRAF-pop (<em>e.g.</em>, <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-fig-5-8" class="xref-fig" href="#F5">Figure 5</a>) confirm that the centroid of each population remains nearly unaffected when different numbers of SNPs are missing.</p><p id="p-96">GD1 and GD2 separate continental populations (European, African/African American, Asian-Pacific Islander) from one another (<a id="xref-fig-2-11" class="xref-fig" href="#F2">Figures 2A</a>). Latin Americans comprise an admixed population with genetic contributions from European, African and/or Native American populations. Most subjects self-described as âHispanicsâ or âLatinosâ in the United States either have large proportions of European and Native American components, or European and African components (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bryc <em>et al.</em> 2010</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bryc <em>et al.</em> 2015</a>). Ancestry inference using GRAF-pop shows similar results. On the GD2 <em>vs.</em> GD1 plots, study-reported Hispanics and Latinos form two sweeping clusters (<a id="xref-fig-2-12" class="xref-fig" href="#F2">Figures 2</a>, <a id="xref-fig-3-6" class="xref-fig" href="#F3">3</a>): one with more African proportion (labeled Latin American1 by GRAF-pop) than the other (labeled Latin American2).</p><p id="p-97">The following three admixed populations are more difficult to distinguish: Latin American 2, South Asian, and individuals experiencing recent European/East Asian admixture. On the GD2 <em>vs.</em> GD1 graph, they show up roughly in the same region of the graph. These populations can be separated by GD4, which is the difference of the genetic distances from each individual to the reference populations of Latin American 2 and South Asian (<a id="xref-fig-1-8" class="xref-fig" href="#F1">Figures 1</a>-<a id="xref-fig-2-13" class="xref-fig" href="#F2">2</a>). Although not normalized, GD4 is not sensitive to non-random missing genotypes. Indeed, any such difference of two genetic distances with two reference populations can be rewritten as:<span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/9/8/2447/embed/mml-math-21.gif"/></noscript></span><span class="disp-formula-label">(7)</span></span>Note that the last term in <a id="xref-disp-formula-5-1" class="xref-disp-formula" href="#disp-formula-5">Equation (2)</a> cancels in the subtraction since the subject genotype <em>g<sub>il</sub></em> is the same no matter what reference population is used. Only the <em>ratios</em> of the allele frequencies are used for calculating the difference, and hence the genetic distance differences are robust against missing data.</p><p id="p-98">GRAF-pop has been used by dbGaP curators and submitters as a quality assurance tool, and to generate displays of the ancestry inference results to potential requestors of the data. For example, on the study report page of phs000001.v1.p1: <a href="https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000001.v1.p1">https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000001.v1.p1</a>, there is an âAncestry Componentâ link that opens the webpage showing GRAF-pop results:</p><p id="p-99"><a href="https://www.ncbi.nlm.nih.gov/projects/gap/population/cgi-bin/StudySubjectAncestry.cgi?phs=1&amp;version=1&amp;exp1=1&amp;exp2=1&amp;exp3=1">https://www.ncbi.nlm.nih.gov/projects/gap/population/cgi-bin/StudySubjectAncestry.cgi?phs=1&amp;version=1&amp;exp1=1&amp;exp2=1&amp;exp3=1</a>. These displays can call attention to unusual results, <em>e.g.</em>, abnormal data distributions that manifest as discordance between study-reported populations and those reported by GRAF-pop.</p><p id="p-100">Two limitations of the current implementation of GRAF-pop are: 1) it relies on the 10,000 dbGaP fingerprint SNPs, which were selected for other purposes (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Jin <em>et al.</em> 2017</a>) and 2) only five reference populations are used to calculate the genetic distances, and these do not include Native American. After applying GRAF-pop to hundreds of dbGaP studies, we noticed that about 2,000 fingerprint SNPs are needed to obtain good resolution in separating continental populations. <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a> shows some examples. When fewer than 1,000 fingerprint SNPs have genotypes, PCA tools such as EIGENSTRAT will perform better than the current version of GRAF-pop, since they can use all available SNPs in the data sets, although some SNPs within linkage disequilibrium (LD) regions might need to be pruned. Missing data can be imputed if possible. However, GRAF-pop introduces a new method of inferring ancestry that does not use PCA. In the future, we would like to improve the software using more SNPs. Theoretically, GRAF-pop can be implemented using all the variants in the whole genome, so that all SNPs available in any given data set can be used to do ancestry inference. Similar LD-pruning can be done for GRAF-pop as for PCA. However, imputation would not be needed no matter how many genotypes are missing in the data sets. Therefore, we did not and will not try to do imputation for the current and future versions of the GRAF-pop software.</p><p id="p-101">We also would like to improve GRAF-pop by using more reference populations, especially Native American. Some researchers tried to estimate allele frequencies of Native Americans (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Gravel <em>et al.</em> 2013</a>). When these allele frequencies become available, we can use the centroid of Native Americans as a fourth vertex to build a tetrahedron instead of the EFA triangle, and continue to use barycentric coordinates to estimate ancestry proportions.</p><p id="p-102">Most existing ancestry inference methods analyze genotypes as âunsupervised inferenceâ. Some investigators have developed methods that instead include âsupervised analysesâ, making use of genotype data of individuals with known ancestral histories (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Alexander and Lange 2011</a>). Some methods, including GRAF-pop, do not require that the raw genotypes in the reference panels be publicly available. Instead, only the allele frequencies or pre-computed SNP weights of the reference panels are needed to infer ancestry (<a id="xref-ref-10-8" class="xref-bibr" href="#ref-10">Chen <em>et al.</em> 2013</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bansal and Libiger 2015</a>; <a id="xref-ref-26-6" class="xref-bibr" href="#ref-26">Li <em>et al.</em> 2016</a>), which increases efficiency. As stated by Bansal and Libiger, the accuracies of these methods depend on the availability of accurate allele frequencies for reference populations. Large public databases and genotyping consortia will participate in creating future reference panels that will enable improvements in GRAF-pop.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-103">Thanks to our colleagues working on dbGaP for testing GRAF-pop and for making suggestions for improvement. The development of GRAF and GRAF-pop is supported by the Intramural Research Program of the National Institutes of Health, National Library of Medicine and National Cancer Institute.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-3"><p id="p-104">Supplemental material available at FigShare: <a href="https://doi.org/10.25387/g3.8061485">https://doi.org/10.25387/g3.8061485</a>.</p></li><li class="fn" id="fn-4"><p id="p-105"><em>Communicating editor: A. Sethuraman</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2018-12-12"><span class="received-label">Received </span>December 12, 2018.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2019-05-23"><span class="accepted-label">Accepted </span>May 23, 2019.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2019 Jin <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-1">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.1" data-doi="10.1371/journal.pone.0093766"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abraham</span>,  <span class="cit-name-given-names">G.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Inouye</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Fast principal component analysis of large-scale genome-wide data</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e93766</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0093766</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D9%26rft.spage%253De93766%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0093766%26rft_id%253Dinfo%253Apmid%252F24718290%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0093766&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24718290&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.2" data-doi="10.1093/bioinformatics/btx299"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abraham</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Qiu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Inouye</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">FlashPCA2: principal component analysis of Biobank-scale genotype datasets</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">2776</span>â<span class="cit-lpage">2778</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btx299</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D33%26rft.spage%253D2776%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtx299%26rft_id%253Dinfo%253Apmid%252F28475694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btx299&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=28475694&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.3" data-doi="10.1186/1471-2105-12-246"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>,  <span class="cit-name-given-names">D. H.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Lange</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Enhancements to the ADMIXTURE algorithm for individual ancestry estimation</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">12</span>: <span class="cit-fpage">246</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2105-12-246</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DAlexander%26rft.auinit1%253DD.%2BH.%26rft.volume%253D12%26rft.spage%253D246%26rft.epage%253D246%26rft.atitle%253DEnhancements%2Bto%2Bthe%2BADMIXTURE%2Balgorithm%2Bfor%2Bindividual%2Bancestry%2Bestimation.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-12-246%26rft_id%253Dinfo%253Apmid%252F21682921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-12-246&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21682921&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000293266800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.4" data-doi="10.1101/gr.094052.109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alexander</span>,  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Novembre</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Lange</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Fast model-based estimation of ancestry in unrelated individuals</span>. <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1655</span>â<span class="cit-lpage">1664</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1101/gr.094052.109</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.094052.109%26rft_id%253Dinfo%253Apmid%252F19648217%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjE5LzkvMTY1NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzkvOC8yNDQ3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.5" data-doi="10.1186/s12859-014-0418-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bansal</span>,  <span class="cit-name-given-names">V.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Libiger</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Fast individual ancestry inference from DNA sequence data leveraging allele frequencies for multiple populations</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">4</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12859-014-0418-7</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D16%26rft.spage%253D4%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-014-0418-7%26rft_id%253Dinfo%253Apmid%252F25592880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-014-0418-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25592880&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.6" data-doi="10.1073/pnas.0909559107"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bryc</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Auton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Nelson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Oksenberg</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. L.</span> <span class="cit-name-surname">Hauser</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Genome-wide patterns of population structure and admixture in West Africans and African Americans</span>. <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">107</span>: <span class="cit-fpage">786</span>â<span class="cit-lpage">791</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1073/pnas.0909559107</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0909559107%26rft_id%253Dinfo%253Apmid%252F20080753%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiIxMDcvMi83ODYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzgvMjQ0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.7" data-doi="10.1016/j.ajhg.2014.11.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bryc</span>,  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. Y.</span> <span class="cit-name-surname">Durand</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Macpherson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Reich</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. L.</span> <span class="cit-name-surname">Mountain</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">The genetic ancestry of African Americans, Latinos, and European Americans across the United States</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">96</span>: <span class="cit-fpage">37</span>â<span class="cit-lpage">53</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2014.11.010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D96%26rft.spage%253D37%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2014.11.010%26rft_id%253Dinfo%253Apmid%252F25529636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2014.11.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25529636&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.8" data-doi="10.1016/j.fsigen.2018.03.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bulbul</span>,  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. C.</span> <span class="cit-name-surname">Speed</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Gurkan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Soundararajan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">H.</span> <span class="cit-name-surname">Rajeevan</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span>â<span class="cit-article-title">Improving ancestry distinctions among Southwest Asian populations</span>. <abbr class="cit-jnl-abbrev">Forensic Sci. Int. Genet.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">14</span>â<span class="cit-lpage">20</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fsigen.2018.03.010</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DForensic%2BSci.%2BInt.%2BGenet.%26rft.volume%253D35%26rft.spage%253D14%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fsigen.2018.03.010%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fsigen.2018.03.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.9" data-doi="10.1186/s12864-017-4166-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Byun</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Han</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">I. P.</span> <span class="cit-name-surname">Gorlov</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. A.</span> <span class="cit-name-surname">Busam</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. F.</span> <span class="cit-name-surname">Seldin</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">Ancestry inference using principal component analysis and spatial analysis: a distance-based analysis to account for population substructure</span>. <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">789</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12864-017-4166-8</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BGenomics%26rft.volume%253D18%26rft.spage%253D789%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12864-017-4166-8%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12864-017-4166-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.10" data-doi="10.1093/bioinformatics/btt144"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>,  <span class="cit-name-given-names">C. Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Pollack</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. J.</span> <span class="cit-name-surname">Hunter</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. N.</span> <span class="cit-name-surname">Hirschhorn</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Kraft</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Improved ancestry inference using weights from external reference panels</span>. <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">1399</span>â<span class="cit-lpage">1406</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1093/bioinformatics/btt144</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtt144%26rft_id%253Dinfo%253Apmid%252F23539302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1093/bioinformatics/btt144&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23539302&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.11" data-doi="10.1038/nmeth.1974"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Clarke</span>,  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zheng-Bradley</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Smith</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E.</span> <span class="cit-name-surname">Kulesha</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Xiao</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">The 1000 Genomes Project: data management and community access</span>. <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">459</span>â<span class="cit-lpage">462</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/nmeth.1974</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DClarke%26rft.auinit1%253DL.%26rft.volume%253D9%26rft.issue%253D5%26rft.spage%253D459%26rft.epage%253D462%26rft.atitle%253DThe%2B1000%2BGenomes%2BProject%253A%2Bdata%2Bmanagement%2Band%2Bcommunity%2Baccess.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1974%26rft_id%253Dinfo%253Apmid%252F22543379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1974&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22543379&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.12" data-doi="10.1002/gepi.21896"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conomos</span>,  <span class="cit-name-given-names">M. P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. B.</span> <span class="cit-name-surname">Miller</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T. A.</span> <span class="cit-name-surname">Thornton</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Robust inference of population structure for ancestry prediction and correction of stratification in the presence of relatedness</span>. <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">276</span>â<span class="cit-lpage">293</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/gepi.21896</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenet.%2BEpidemiol.%26rft.volume%253D39%26rft.spage%253D276%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.21896%26rft_id%253Dinfo%253Apmid%252F25810074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.21896&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25810074&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.13" data-doi="10.1371/journal.pone.0082224"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Daya</span>,  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">van der Merwe</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">U.</span> <span class="cit-name-surname">Galal</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Moller</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Salie</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">A panel of ancestry informative markers for the complex five-way admixed South African coloured population</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e82224</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0082224</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D8%26rft.spage%253De82224%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0082224%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0082224&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.14" data-doi="10.1371/journal.pgen.1002554"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galanter</span>,  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. C.</span> <span class="cit-name-surname">Fernandez-Lopez</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. R.</span> <span class="cit-name-surname">Gignoux</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Barnholtz-Sloan</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Fernandez-Rozadilla</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Development of a panel of genome-wide ancestry informative markers to study admixture throughout the Americas</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-elocation-id">e1002554</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1002554</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DGalanter%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D8%26rft.issue%253D3%26rft.spage%253De1002554%26rft.epage%253De1002554%26rft.atitle%253DDevelopment%2Bof%2Ba%2Bpanel%2Bof%2Bgenome-wide%2Bancestry%2Binformative%2Bmarkers%2Bto%2Bstudy%2Badmixture%2Bthroughout%2Bthe%2BAmericas.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1002554%26rft_id%253Dinfo%253Apmid%252F22412386%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1002554&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22412386&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=22412386&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.15" data-doi="10.1016/j.ajhg.2015.12.022"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Galinsky</span>,  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Bhatia</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. R.</span> <span class="cit-name-surname">Loh</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Georgiev</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Mukherjee</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Fast Principal-Component Analysis Reveals Convergent Evolution of ADH1B in Europe and East Asia</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">456</span>â<span class="cit-lpage">472</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.ajhg.2015.12.022</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAm.%2BJ.%2BHum.%2BGenet.%26rft.volume%253D98%26rft.spage%253D456%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ajhg.2015.12.022%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ajhg.2015.12.022&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.16" data-doi="10.1371/journal.pgen.1004023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gravel</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">F.</span> <span class="cit-name-surname">Zakharia</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Moreno-Estrada</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Byrnes</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Muzzio</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Reconstructing Native American migrations from whole-genome and whole-exome data</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-elocation-id">e1004023</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.1004023</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1004023%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004023%26rft_id%253Dinfo%253Apmid%252F24385924%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24385924&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.17" data-doi="10.1534/genetics.115.182626"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Greenbaum</span>,  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. R.</span> <span class="cit-name-surname">Templeton</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Bar-David</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Inference and Analysis of Population Structure Using Genetic Data and Network Theory</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">202</span>: <span class="cit-fpage">1299</span>â<span class="cit-lpage">1312</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.115.182626</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.115.182626%26rft_id%253Dinfo%253Apmid%252F26888080%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjIwMi80LzEyOTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzgvMjQ0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.18" data-doi="10.1002/humu.20695"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Halder</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Shriver</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Thomas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. R.</span> <span class="cit-name-surname">Fernandez</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Frudakis</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">A panel of ancestry informative markers for estimating individual biogeographical ancestry and admixture from four continents: utility and applications</span>. <abbr class="cit-jnl-abbrev">Hum. Mutat.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">648</span>â<span class="cit-lpage">658</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/humu.20695</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHuman%2Bmutation%26rft.stitle%253DHum%2BMutat%26rft.aulast%253DHalder%26rft.auinit1%253DI.%26rft.volume%253D29%26rft.issue%253D5%26rft.spage%253D648%26rft.epage%253D658%26rft.atitle%253DA%2Bpanel%2Bof%2Bancestry%2Binformative%2Bmarkers%2Bfor%2Bestimating%2Bindividual%2Bbiogeographical%2Bancestry%2Band%2Badmixture%2Bfrom%2Bfour%2Bcontinents%253A%2Butility%2Band%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fhumu.20695%26rft_id%253Dinfo%253Apmid%252F18286470%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/humu.20695&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18286470&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255385800010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hellwege</span>,  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. M.</span> <span class="cit-name-surname">Keaton</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Giri</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Gao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.R.</span> <span class="cit-name-surname">Velez Edwards</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2017</span>âPopulation Stratification in Genetic Association Studies. Curr Protoc Hum Genet 95: 1 22 21â21 22 23.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/cphg.48</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.20" data-doi="10.1371/journal.pone.0179106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jin</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. A.</span> <span class="cit-name-surname">Schaffer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. T.</span> <span class="cit-name-surname">Sherry</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Feolo</span></span></li></ol><cite>, <span class="cit-pub-date">2017</span>â<span class="cit-article-title">Quickly identifying identical and closely related subjects in large databases using genotype data</span>. <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">12</span>: <span class="cit-elocation-id">e0179106</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pone.0179106</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BOne%26rft.volume%253D12%26rft.spage%253De0179106%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0179106%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0179106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.21" data-doi="10.1098/rsta.2015.0202"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jolliffe</span>,  <span class="cit-name-given-names">I. T.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Cadima</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Principal component analysis: a review and recent developments. Philos Trans A Math Phys</span>. <abbr class="cit-jnl-abbrev">Eng. Sci.</abbr> <span class="cit-vol">374</span>: <span class="cit-elocation-id">20150202</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1098/rsta.2015.0202</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEng.%2BSci.%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frsta.2015.0202%26rft_id%253Dinfo%253Apmid%252F26953178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1098/rsta.2015.0202&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26953178&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.22" data-doi="10.1016/j.fsigen.2014.01.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kidd</span>,  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">W. C.</span> <span class="cit-name-surname">Speed</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. J.</span> <span class="cit-name-surname">Pakstis</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. R.</span> <span class="cit-name-surname">Furtado</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Fang</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Progress toward an efficient panel of SNPs for ancestry inference</span>. <abbr class="cit-jnl-abbrev">Forensic Sci. Int. Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">23</span>â<span class="cit-lpage">32</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fsigen.2014.01.002</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DForensic%2BSci.%2BInt.%2BGenet.%26rft.volume%253D10%26rft.spage%253D23%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fsigen.2014.01.002%26rft_id%253Dinfo%253Apmid%252F24508742%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fsigen.2014.01.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24508742&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.23" data-doi="10.1038/ng.3312"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lappalainen</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Almeida-King</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">V.</span> <span class="cit-name-surname">Kumanduri</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Senf</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. D.</span> <span class="cit-name-surname">Spalding</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">The European Genome-phenome Archive of human data consented for biomedical research</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">692</span>â<span class="cit-lpage">695</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng.3312</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D47%26rft.spage%253D692%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.3312%26rft_id%253Dinfo%253Apmid%252F26111507%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.3312&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26111507&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.24" data-doi="10.1146/annurev-genom-082410-101510"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lawson</span>,  <span class="cit-name-given-names">D. J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Falush</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span>â<span class="cit-article-title">Population identification using genetic data</span>. <abbr class="cit-jnl-abbrev">Annu. Rev. Genomics Hum. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">337</span>â<span class="cit-lpage">361</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1146/annurev-genom-082410-101510</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bgenomics%2Band%2Bhuman%2Bgenetics%26rft.stitle%253DAnnu%2BRev%2BGenomics%2BHum%2BGenet%26rft.aulast%253DLawson%26rft.auinit1%253DD.%2BJ.%26rft.volume%253D13%26rft.spage%253D337%26rft.epage%253D361%26rft.atitle%253DPopulation%2Bidentification%2Busing%2Bgenetic%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-genom-082410-101510%26rft_id%253Dinfo%253Apmid%252F22703172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-genom-082410-101510&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22703172&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310143800015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.25" data-doi="10.1002/gepi.20434"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>,  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Luca</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Klei</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Devlin</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Roeder</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-article-title">Discovering genetic ancestry using spectral graph theory</span>. <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">51</span>â<span class="cit-lpage">59</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/gepi.20434</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DLee%26rft.auinit1%253DA.%2BB.%26rft.volume%253D34%26rft.issue%253D1%26rft.spage%253D51%26rft.epage%253D59%26rft.atitle%253DDiscovering%2Bgenetic%2Bancestry%2Busing%2Bspectral%2Bgraph%2Btheory.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20434%26rft_id%253Dinfo%253Apmid%252F19455578%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20434&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19455578&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273513000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.26" data-doi="10.1186/s12859-016-0965-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Byun</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">Cai</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Xiao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Han</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">FastPop: a rapid principal component derived method to infer intercontinental ancestry using genetic data</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">122</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12859-016-0965-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D17%26rft.spage%253D122%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-016-0965-1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-016-0965-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.27" data-doi="10.1186/1479-7364-7-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>,  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Nyunoya</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Leng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S. A.</span> <span class="cit-name-surname">Belinsky</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Tesfaigzi</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Softwares and methods for estimating genetic ancestry in human populations</span>. <abbr class="cit-jnl-abbrev">Hum. Genomics</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">1</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1479-7364-7-1</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHum.%2BGenomics%26rft.volume%253D7%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1479-7364-7-1%26rft_id%253Dinfo%253Apmid%252F23289408%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1479-7364-7-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23289408&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.28" data-doi="10.1038/ng1007-1181"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mailman</span>,  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Feolo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Jin</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Kimura</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Tryka</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">The NCBI dbGaP database of genotypes and phenotypes</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1181</span>â<span class="cit-lpage">1186</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1007-1181</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DMailman%26rft.auinit1%253DM.%2BD.%26rft.volume%253D39%26rft.issue%253D10%26rft.spage%253D1181%26rft.epage%253D1186%26rft.atitle%253DThe%2BNCBI%2BdbGaP%2Bdatabase%2Bof%2Bgenotypes%2Band%2Bphenotypes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1007-1181%26rft_id%253Dinfo%253Apmid%252F17898773%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1007-1181&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17898773&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249737400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-9.8.2447.29" data-doi="10.1371/journal.pgen.1000686"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>,  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">A genealogical interpretation of principal components analysis</span>. <abbr class="cit-jnl-abbrev">PLoS Genet</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e1000686</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DMcVean%26rft.auinit1%253DG.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253De1000686%26rft.epage%253De1000686%26rft.atitle%253DA%2Bgenealogical%2Binterpretation%2Bof%2Bprincipal%2Bcomponents%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000686%26rft_id%253Dinfo%253Apmid%252F19834557%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000686&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19834557&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.30" data-doi="10.1126/science.356262"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Menozzi</span>,  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A.</span> <span class="cit-name-surname">Piazza</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Cavalli-Sforza</span></span></li></ol><cite>, <span class="cit-pub-date">1978</span>â<span class="cit-article-title">Synthetic maps of human gene frequencies in Europeans</span>. <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">786</span>â<span class="cit-lpage">792</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1126/science.356262</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DMenozzi%26rft.auinit1%253DP%26rft.volume%253D201%26rft.issue%253D4358%26rft.spage%253D786%26rft.epage%253D792%26rft.atitle%253DSynthetic%2Bmaps%2Bof%2Bhuman%2Bgene%2Bfrequencies%2Bin%2BEuropeans%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.356262%26rft_id%253Dinfo%253Apmid%252F356262%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyMDEvNDM1OC83ODYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy85LzgvMjQ0Ny5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.31" data-doi="10.1186/1471-2156-10-39"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nassir</span>,  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R.</span> <span class="cit-name-surname">Kosoy</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Tian</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P. A.</span> <span class="cit-name-surname">White</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L. M.</span> <span class="cit-name-surname">Butler</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">An ancestry informative marker set for determining continental origin: validation and extension using human genome diversity panels</span>. <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">39</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1471-2156-10-39</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DNassir%26rft.auinit1%253DR.%26rft.volume%253D10%26rft.spage%253D39%26rft.epage%253D39%26rft.atitle%253DAn%2Bancestry%2Binformative%2Bmarker%2Bset%2Bfor%2Bdetermining%2Bcontinental%2Borigin%253A%2Bvalidation%2Band%2Bextension%2Busing%2Bhuman%2Bgenome%2Bdiversity%2Bpanels.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-10-39%26rft_id%253Dinfo%253Apmid%252F19630973%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-10-39&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19630973&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.32" data-doi="10.1016/j.gde.2016.08.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Novembre</span>,  <span class="cit-name-given-names">J.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B. M.</span> <span class="cit-name-surname">Peter</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Recent advances in the study of fine-scale population structure in humans</span>. <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">98</span>â<span class="cit-lpage">105</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.gde.2016.08.007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BGenet.%2BDev.%26rft.volume%253D41%26rft.spage%253D98%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2016.08.007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gde.2016.08.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.33" data-doi="10.3389/fgene.2014.00204"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Padhukasahasram</span>,  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Inferring ancestry from population genomic data and its applications</span>. <abbr class="cit-jnl-abbrev">Front. Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">204</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.3389/fgene.2014.00204</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BGenet.%26rft.volume%253D5%26rft.spage%253D204%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffgene.2014.00204%26rft_id%253Dinfo%253Apmid%252F25071832%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fgene.2014.00204&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25071832&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.34" data-doi="10.1371/journal.pgen.0020190"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Patterson</span>,  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. L.</span> <span class="cit-name-surname">Price</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D.</span> <span class="cit-name-surname">Reich</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Population structure and eigenanalysis</span>. <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">2</span>: <span class="cit-elocation-id">e190</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1371/journal.pgen.0020190</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BGenet%26rft.aulast%253DPatterson%26rft.auinit1%253DN.%26rft.volume%253D2%26rft.issue%253D12%26rft.spage%253De190%26rft.epage%253De190%26rft.atitle%253DPopulation%2Bstructure%2Band%2Beigenanalysis.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.0020190%26rft_id%253Dinfo%253Apmid%252F17194218%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.0020190&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17194218&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.35" data-doi="10.1038/ng1847"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Price</span>,  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. J.</span> <span class="cit-name-surname">Patterson</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">R. M.</span> <span class="cit-name-surname">Plenge</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. E.</span> <span class="cit-name-surname">Weinblatt</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. A.</span> <span class="cit-name-surname">Shadick</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Principal components analysis corrects for stratification in genome-wide association studies</span>. <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">904</span>â<span class="cit-lpage">909</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1038/ng1847</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTo%26rft.auinit1%253DM.%2BD.%26rft.volume%253D38%26rft.issue%253D8%26rft.spage%253D904%26rft.epage%253D909%26rft.atitle%253DPrincipal%2Bcomponents%2Banalysis%2Bcorrects%2Bfor%2Bstratification%2Bin%2Bgenome-wide%2Bassociation%2Bstudies.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1847%26rft_id%253Dinfo%253Apmid%252F16862161%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1847&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16862161&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239325700019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.36" data-doi="10.1111/j.1471-8286.2005.01155.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pritchard</span>,  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Stephens</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Donnelly</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span>â<span class="cit-article-title">Inference of population structure using multilocus genotype data</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">945</span>â<span class="cit-lpage">959</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPritchard%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D155%26rft.issue%253D2%26rft.spage%253D945%26rft.epage%253D959%26rft.atitle%253DInference%2Bof%2BPopulation%2BStructure%2BUsing%2BMultilocus%2BGenotype%2BData%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1471-8286.2005.01155.x%26rft_id%253Dinfo%253Apmid%252F10835412%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzIvOTQ1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI0NDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.37" data-doi="10.1086/519795"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Purcell</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Neale</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Todd-Brown</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Thomas</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. A.</span> <span class="cit-name-surname">Ferreira</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">PLINK: a tool set for whole-genome association and population-based linkage analyses</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">559</span>â<span class="cit-lpage">575</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/519795</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPurcell%26rft.auinit1%253DS.%26rft.volume%253D81%26rft.issue%253D3%26rft.spage%253D559%26rft.epage%253D575%26rft.atitle%253DPLINK%253A%2Ba%2Btool%2Bset%2Bfor%2Bwhole-genome%2Bassociation%2Band%2Bpopulation-based%2Blinkage%2Banalyses.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F519795%26rft_id%253Dinfo%253Apmid%252F17701901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/519795&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17701901&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.38" data-doi="10.1534/genetics.114.164350"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Raj</span>,  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Stephens</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">J. K.</span> <span class="cit-name-surname">Pritchard</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">fastSTRUCTURE: variational inference of population structure in large SNP data sets</span>. <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">197</span>: <span class="cit-fpage">573</span>â<span class="cit-lpage">589</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1534/genetics.114.164350</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.164350%26rft_id%253Dinfo%253Apmid%252F24700103%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk3LzIvNTczIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvOS84LzI0NDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.39" data-doi="10.1186/s12859-016-1082-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shringarpure</span>,  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. D.</span> <span class="cit-name-surname">Bustamante</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">K.</span> <span class="cit-name-surname">Lange</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">D. H.</span> <span class="cit-name-surname">Alexander</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span>â<span class="cit-article-title">Efficient analysis of large datasets and sex bias with ADMIXTURE</span>. <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">218</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s12859-016-1082-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BBioinformatics%26rft.volume%253D17%26rft.spage%253D218%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs12859-016-1082-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s12859-016-1082-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.40" data-doi="10.1016/j.cag.2007.09.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Skala</span>,  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span>â<span class="cit-article-title">Barycentric coordinates computation in homogeneous coordinates</span>. <abbr class="cit-jnl-abbrev">Comput. Graph.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">120</span>â<span class="cit-lpage">127</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.cag.2007.09.007</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BGraph.%26rft.volume%253D32%26rft.spage%253D120%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cag.2007.09.007%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cag.2007.09.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.41" data-doi="10.1016/j.talanta.2006.10.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stanimirova</span>,  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Daszykowski</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">B.</span> <span class="cit-name-surname">Walczak</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Dealing with missing values and outliers in principal component analysis</span>. <abbr class="cit-jnl-abbrev">Talanta</abbr> <span class="cit-vol">72</span>: <span class="cit-fpage">172</span>â<span class="cit-lpage">178</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.talanta.2006.10.011</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DTalanta%26rft.aulast%253DStanimirova%26rft.auinit1%253DI.%26rft.volume%253D72%26rft.issue%253D1%26rft.spage%253D172%26rft.epage%253D178%26rft.atitle%253DDealing%2Bwith%2Bmissing%2Bvalues%2Band%2Boutliers%2Bin%2Bprincipal%2Bcomponent%2Banalysis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.talanta.2006.10.011%26rft_id%253Dinfo%253Apmid%252F19071598%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.talanta.2006.10.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19071598&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246225900023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.42" data-doi="10.1086/504302"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Coram</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Wang</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhu</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N.</span> <span class="cit-name-surname">Risch</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Reconstructing genetic ancestry blocks in admixed individuals</span>. <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">1</span>â<span class="cit-lpage">12</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1086/504302</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DTang%26rft.auinit1%253DH.%26rft.volume%253D79%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D12%26rft.atitle%253DReconstructing%2Bgenetic%2Bancestry%2Bblocks%2Bin%2Badmixed%2Bindividuals.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F504302%26rft_id%253Dinfo%253Apmid%252F16773560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/504302&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16773560&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238341200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.43" data-doi="10.1002/gepi.20064"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>,  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Peng</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">P.</span> <span class="cit-name-surname">Wang</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">N. J.</span> <span class="cit-name-surname">Risch</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">Estimation of individual admixture: analytical and study design considerations</span>. <abbr class="cit-jnl-abbrev">Genet. Epidemiol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">289</span>â<span class="cit-lpage">301</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1002/gepi.20064</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetic%2Bepidemiology%26rft.stitle%253DGenet%2BEpidemiol%26rft.aulast%253DTang%26rft.auinit1%253DH.%26rft.volume%253D28%26rft.issue%253D4%26rft.spage%253D289%26rft.epage%253D301%26rft.atitle%253DEstimation%2Bof%2Bindividual%2Badmixture%253A%2Banalytical%2Band%2Bstudy%2Bdesign%2Bconsiderations.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fgepi.20064%26rft_id%253Dinfo%253Apmid%252F15712363%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/gepi.20064&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15712363&amp;link_type=MED&amp;atom=%2Fggg%2F9%2F8%2F2447.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000228573700001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.44" data-doi="10.1186/1753-6561-8-S1-S5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>,  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M. P.</span> <span class="cit-name-surname">Conomos</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Sverdlov</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">E. M.</span> <span class="cit-name-surname">Blue</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. Y.</span> <span class="cit-name-surname">Cheung</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Estimating and adjusting for ancestry admixture in statistical methods for relatedness inference, heritability estimation, and association testing</span>. <abbr class="cit-jnl-abbrev">BMC Proc.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">S5</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/1753-6561-8-S1-S5</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BProc.%26rft.volume%253D8%26rft.spage%253DS5%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1753-6561-8-S1-S5%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1753-6561-8-S1-S5&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">âµ</a><div class="cit ref-cit ref-book" id="cit-9.8.2447.45" data-doi="10.1142/7740"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ungar</span>,  <span class="cit-name-given-names">A. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span>â<span class="cit-source">Barycentric Calculus in Euclidean and Hyperbolic Geometry</span>, <span class="cit-publ-name">World Scientific</span>, <span class="cit-publ-loc">Singapore</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1142/7740</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBarycentric%2BCalculus%2Bin%2BEuclidean%2Band%2BHyperbolic%2BGeometry%26rft_id%253Dinfo%253Adoi%252F10.1142%252F7740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1142/7740&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.46" data-doi="10.1016/j.fsigen.2018.02.009"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>,  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">G.</span> <span class="cit-name-surname">He</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">T.</span> <span class="cit-name-surname">Luo</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">X.</span> <span class="cit-name-surname">Zhao</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">J.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2018</span>â<span class="cit-article-title">Massively parallel sequencing of 165 ancestry informative SNPs in two Chinese Tibetan-Burmese minority ethnicities</span>. <abbr class="cit-jnl-abbrev">Forensic Sci. Int. Genet.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">141</span>â<span class="cit-lpage">147</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fsigen.2018.02.009</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DForensic%2BSci.%2BInt.%2BGenet.%26rft.volume%253D34%26rft.spage%253D141%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fsigen.2018.02.009%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fsigen.2018.02.009&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.47" data-doi="10.1007/s10444-005-9008-6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Warren</span>,  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">S.</span> <span class="cit-name-surname">Schaefer</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">A. N.</span> <span class="cit-name-surname">Hirani</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Desbun</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span>â<span class="cit-article-title">Barycentric Coordinates for Convex Sets</span>. <abbr class="cit-jnl-abbrev">Adv. Comput. Math.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">319</span>â<span class="cit-lpage">338</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1007/s10444-005-9008-6</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BComput.%2BMath.%26rft.volume%253D27%26rft.spage%253D319%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10444-005-9008-6%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10444-005-9008-6&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.48" data-doi="10.1111/j.1467-8659.2009.01399.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>,  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">M.</span> <span class="cit-name-surname">Ben-Chen</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">C.</span> <span class="cit-name-surname">Gotsman</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span>â<span class="cit-article-title">Complex Barycentric Coordinates with Applications to Planar Shape Deformation</span>. <abbr class="cit-jnl-abbrev">Comput. Graph. Forum</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">587</span>â<span class="cit-lpage">597</span> <span class="cit-comment">(Proceedings of Eurographics)</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1111/j.1467-8659.2009.01399.x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DComput.%2BGraph.%2BForum%26rft.volume%253D28%26rft.spage%253D587%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-8659.2009.01399.x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-8659.2009.01399.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.49" data-doi="10.1186/s13323-015-0019-x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wollstein</span>,  <span class="cit-name-given-names">A.</span></span>, and </li><li><span class="cit-auth"> <span class="cit-name-given-names">O.</span> <span class="cit-name-surname">Lao</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span>â<span class="cit-article-title">Detecting individual ancestry in the human genome</span>. <abbr class="cit-jnl-abbrev">Investig. Genet.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">7</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1186/s13323-015-0019-x</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DInvestig.%2BGenet.%26rft.volume%253D6%26rft.spage%253D7%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fs13323-015-0019-x%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/s13323-015-0019-x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">âµ</a><div class="cit ref-cit ref-journal" id="cit-9.8.2447.50" data-doi="10.1016/j.fsigen.2018.10.015"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>,  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">C. M.</span> <span class="cit-name-surname">Shi</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">L.</span> <span class="cit-name-surname">Ma</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Q.</span> <span class="cit-name-surname">Liu</span></span>, </li><li><span class="cit-auth"> <span class="cit-name-given-names">Y.</span> <span class="cit-name-surname">Liu</span></span>, <span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2019</span>â<span class="cit-article-title">AIM-SNPtag: A computationally efficient approach for developing ancestry-informative SNP panels</span>. <abbr class="cit-jnl-abbrev">Forensic Sci. Int. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">245</span>â<span class="cit-lpage">253</span>.<span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1016/j.fsigen.2018.10.015</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DForensic%2BSci.%2BInt.%2BGenet.%26rft.volume%253D38%26rft.spage%253D245%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.fsigen.2018.10.015%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.fsigen.2018.10.015&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/9/8/2447.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/9/8/2441" title="Evolutionary and Experimental Loss of Gene Body Methylation and Its Consequence to Gene Expression" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/9/8/2463" title="Genomic Selection with Allele Dosage in Panicum maximum Jacq." rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/9/8"><span>Volume 9 Issue 8, August 2019</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/9/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/9/8.cover-source.jpg?itok=cSNYBBSf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 9 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/9/8/2447" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-9vFRoTnxPrAwL1VWC-QkrnJcwGyGRBq8igXsxayd4i4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/9/8/2447" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/9/8/2447" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/9/8/2447" class="active" data-icon-position="" data-hide-link-title="0">GRAF-pop: A Fast Distance-Based Method To Infer Subject Ancestry from Multiple Genotype Datasets Without Principal Components Analysis</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/493745" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-0jLUASurnruPaxpjxe11cfda97c7f46CBmqut6a_a_Q" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/8/2447.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/9/8/2447" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-3FMjeJQk9i6V1W2gAGtmi0oRqyhB_aVLlYmB_5xMMiY" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/9/8/2447" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/9/8/2447.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493745" id="node493745--2" data-pisa="ggg;9/8/2447" data-pisa-master="ggg;g3.118.200925" data-apath="/ggg/9/8/2447.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">GRAF-pop: A Fast Distance-Based Method To Infer Subject Ancestry from Multiple Genotype Datasets Without Principal Components Analysis</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yumi Jin</span>, <span class="highwire-citation-author" data-delta="1">Alejandro A. Schaffer</span>, <span class="highwire-citation-author" data-delta="2">Michael Feolo</span>, <span class="highwire-citation-author" data-delta="3">J. Bradley Holmes</span> and <span class="highwire-citation-author" data-delta="4">Brandi L. Kattman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2447-2461;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200925 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493745--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yumi Jin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Center for Biotechnology Information, National Institutes of Health, Department of Health and Human Services, Bethesda, Maryland 20894 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yumi%2BJin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jin%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYumi%2BJin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jinyu@ncbi.nlm.nih.gov" class="" data-icon-position="" data-hide-link-title="0">jinyu@ncbi.nlm.nih.gov</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alejandro A. Schaffer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Cancer Data Science Laboratory, National Cancer Institute, National Institutes of Health; Department of Health and Human Services; Bethesda, Maryland 20892</div><div class='author-affiliation'><span class='nlm-sup'>*</span>National Center for Biotechnology Information, National Institutes of Health, Department of Health and Human Services, Bethesda, Maryland 20894 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alejandro%2BA.%2BSchaffer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schaffer%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlejandro%2BA.%2BSchaffer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Michael Feolo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Center for Biotechnology Information, National Institutes of Health, Department of Health and Human Services, Bethesda, Maryland 20894 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BFeolo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Feolo%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BFeolo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">J. Bradley Holmes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Center for Biotechnology Information, National Institutes of Health, Department of Health and Human Services, Bethesda, Maryland 20894 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BBradley%2BHolmes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Holmes%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ.%2BBradley%2BHolmes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Brandi L. Kattman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Center for Biotechnology Information, National Institutes of Health, Department of Health and Human Services, Bethesda, Maryland 20894 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brandi%2BL.%2BKattman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kattman%20BL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrandi%2BL.%2BKattman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/9/8/2447" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/9/8/2447&amp;title=GRAF-pop%3A%20A%20Fast%20Distance-Based%20Method%20To%20Infer%20Subject%20Ancestry%20from%20Multiple%20Genotype%20Datasets%20Without%20Principal%20Components%20Analysis" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/9/8/2447&amp;title=GRAF-pop%3A%20A%20Fast%20Distance-Based%20Method%20To%20Infer%20Subject%20Ancestry%20from%20Multiple%20Genotype%20Datasets%20Without%20Principal%20Components%20Analysis" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/9/8/2447&amp;title=GRAF-pop%3A%20A%20Fast%20Distance-Based%20Method%20To%20Infer%20Subject%20Ancestry%20from%20Multiple%20Genotype%20Datasets%20Without%20Principal%20Components%20Analysis" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/9/8/2447&amp;text=GRAF-pop%3A%20A%20Fast%20Distance-Based%20Method%20To%20Infer%20Subject%20Ancestry%20from%20Multiple%20Genotype%20Datasets%20Without%20Principal%20Components%20Analysis" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/9/8/2447&amp;title=GRAF-pop%3A%20A%20Fast%20Distance-Based%20Method%20To%20Infer%20Subject%20Ancestry%20from%20Multiple%20Genotype%20Datasets%20Without%20Principal%20Components%20Analysis" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/9/8/2447&amp;t=GRAF-pop%3A%20A%20Fast%20Distance-Based%20Method%20To%20Infer%20Subject%20Ancestry%20from%20Multiple%20Genotype%20Datasets%20Without%20Principal%20Components%20Analysis" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/9/8/2447&amp;title=GRAF-pop%3A%20A%20Fast%20Distance-Based%20Method%20To%20Infer%20Subject%20Ancestry%20from%20Multiple%20Genotype%20Datasets%20Without%20Principal%20Components%20Analysis" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/9/8/2447&amp;title=GRAF-pop%3A%20A%20Fast%20Distance-Based%20Method%20To%20Infer%20Subject%20Ancestry%20from%20Multiple%20Genotype%20Datasets%20Without%20Principal%20Components%20Analysis" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/493745/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="493745" id="node493745--3" data-pisa="ggg;9/8/2447" data-pisa-master="ggg;g3.118.200925" data-apath="/ggg/9/8/2447.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">GRAF-pop: A Fast Distance-Based Method To Infer Subject Ancestry from Multiple Genotype Datasets Without Principal Components Analysis</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Yumi Jin</span>, <span class="highwire-citation-author" data-delta="1">Alejandro A. Schaffer</span>, <span class="highwire-citation-author" data-delta="2">Michael Feolo</span>, <span class="highwire-citation-author" data-delta="3">J. Bradley Holmes</span> and <span class="highwire-citation-author" data-delta="4">Brandi L. Kattman</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2019 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 9 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">2447-2461;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.118.200925 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node493745--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Yumi Jin </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Center for Biotechnology Information, National Institutes of Health, Department of Health and Human Services, Bethesda, Maryland 20894 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yumi%2BJin%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Jin%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYumi%2BJin%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:jinyu@ncbi.nlm.nih.gov" class="" data-icon-position="" data-hide-link-title="0">jinyu@ncbi.nlm.nih.gov</a></span></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Alejandro A. Schaffer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Cancer Data Science Laboratory, National Cancer Institute, National Institutes of Health; Department of Health and Human Services; Bethesda, Maryland 20892</div><div class='author-affiliation'><span class='nlm-sup'>*</span>National Center for Biotechnology Information, National Institutes of Health, Department of Health and Human Services, Bethesda, Maryland 20894 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alejandro%2BA.%2BSchaffer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schaffer%20AA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlejandro%2BA.%2BSchaffer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Michael Feolo </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Center for Biotechnology Information, National Institutes of Health, Department of Health and Human Services, Bethesda, Maryland 20894 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BFeolo%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Feolo%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMichael%2BFeolo%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">J. Bradley Holmes </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Center for Biotechnology Information, National Institutes of Health, Department of Health and Human Services, Bethesda, Maryland 20894 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=J.%2BBradley%2BHolmes%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Holmes%20JB&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJ.%2BBradley%2BHolmes%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Brandi L. Kattman </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>National Center for Biotechnology Information, National Institutes of Health, Department of Health and Human Services, Bethesda, Maryland 20894 and</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Brandi%2BL.%2BKattman%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kattman%20BL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABrandi%2BL.%2BKattman%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=2447&amp;rft.epage=2461&amp;rft.atitle=GRAF-pop%3A%20A%20Fast%20Distance-Based%20Method%20To%20Infer%20Subject%20Ancestry%20from%20Multiple%20Genotype%20Datasets%20Without%20Principal%20Components%20Analysis&amp;rft.volume=9&amp;rft.issue=8&amp;rft.date=2019-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Jin%2C+Yumi&amp;rft.au=Schaffer%2C+Alejandro+A.&amp;rft.au=Feolo%2C+Michael&amp;rft.au=Holmes%2C+J.+Bradley&amp;rft.au=Kattman%2C+Brandi+L."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/493745/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/493745/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/493745/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/493745/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/493745/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/493745/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/493745/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/493745/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/493745/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/493745/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/493745/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/493745/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/9/8/2447" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/9/8/2447.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/9/8/2447.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/9/8/2447.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
