<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/8/1479" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/8/1479.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/8/1479.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/8/1479.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/4/8/1479/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/8/1479.atom" />
<meta name="HW.pisa" content="ggg;4/8/1479" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Interplay of Interlocus Gene Conversion and Crossover in Segmental Duplications Under a Neutral Scenario" />
<meta name="DC.Identifier" content="10.1534/g3.114.012435" />
<meta name="DC.Date" content="2014-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Hartasánchez et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Interlocus gene conversion is a major evolutionary force that drives the concerted evolution of duplicated genomic regions. Theoretical models successfully have addressed the effects of interlocus gene conversion and the importance of crossover in the evolutionary fate of gene families and duplications but have not considered complex recombination scenarios, such as the presence of hotspots. To study the interplay between interlocus gene conversion and crossover, we have developed a forward-time simulator that allows the exploration of a wide range of interlocus gene conversion rates under different crossover models. Using it, we have analyzed patterns of nucleotide variation and linkage disequilibrium within and between duplicate regions, focusing on a neutral scenario with constant population size and validating our results with the existing theoretical models. We show that the interaction of gene conversion and crossover is nontrivial and that the location of crossover junctions is a fundamental determinant of levels of variation and linkage disequilibrium in duplicated regions. We also show that if crossover activity between duplications is strong enough, recurrent interlocus gene conversion events can break linkage disequilibrium within duplicates. Given the complex nature of interlocus gene conversion and crossover, we provide a framework to explore their interplay to help increase knowledge on molecular evolution within segmental duplications under more complex scenarios, such as demographic changes or natural selection." />
<meta name="DC.Contributor" content="Diego A. Hartasánchez" />
<meta name="DC.Contributor" content="Oriol Vallès-Codina" />
<meta name="DC.Contributor" content="Marina Brasó-Vives" />
<meta name="DC.Contributor" content="Arcadi Navarro" />
<meta name="article:published_time" content="2014-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Interplay of Interlocus Gene Conversion and Crossover in Segmental Duplications Under a Neutral Scenario" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Interlocus gene conversion is a major evolutionary force that drives the concerted evolution of duplicated genomic regions. Theoretical models successfully have addressed the effects of interlocus gene conversion and the importance of crossover in the evolutionary fate of gene families and duplications but have not considered complex recombination scenarios, such as the presence of hotspots. To study the interplay between interlocus gene conversion and crossover, we have developed a forward-time simulator that allows the exploration of a wide range of interlocus gene conversion rates under different crossover models. Using it, we have analyzed patterns of nucleotide variation and linkage disequilibrium within and between duplicate regions, focusing on a neutral scenario with constant population size and validating our results with the existing theoretical models. We show that the interaction of gene conversion and crossover is nontrivial and that the location of crossover junctions is a fundamental determinant of levels of variation and linkage disequilibrium in duplicated regions. We also show that if crossover activity between duplications is strong enough, recurrent interlocus gene conversion events can break linkage disequilibrium within duplicates. Given the complex nature of interlocus gene conversion and crossover, we provide a framework to explore their interplay to help increase knowledge on molecular evolution within segmental duplications under more complex scenarios, such as demographic changes or natural selection.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/08/01" />
<meta name="citation_mjid" content="ggg;4/8/1479" />
<meta name="citation_id" content="4/8/1479" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/8/1479" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/8/1479.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/8/1479.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/8/1479.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.012435" />
<meta name="citation_pmid" content="24906640" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1479" />
<meta name="citation_lastpage" content="1489" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Diego A. Hartasánchez" />
<meta name="citation_author_institution" content="Institute of Evolutionary Biology (Universitat Pompeu Fabra – CSIC), PRBB, Barcelona, Catalonia, Spain, 08003" />
<meta name="citation_author" content="Oriol Vallès-Codina" />
<meta name="citation_author_institution" content="Institute of Evolutionary Biology (Universitat Pompeu Fabra – CSIC), PRBB, Barcelona, Catalonia, Spain, 08003" />
<meta name="citation_author" content="Marina Brasó-Vives" />
<meta name="citation_author_institution" content="Institute of Evolutionary Biology (Universitat Pompeu Fabra – CSIC), PRBB, Barcelona, Catalonia, Spain, 08003" />
<meta name="citation_author" content="Arcadi Navarro" />
<meta name="citation_author_institution" content="Institute of Evolutionary Biology (Universitat Pompeu Fabra – CSIC), PRBB, Barcelona, Catalonia, Spain, 08003" />
<meta name="citation_author_institution" content="National Institute for Bioinformatics (INB), Barcelona, Catalonia, Spain" />
<meta name="citation_author_institution" content="Institució Catalana de Recerca i Estudis Avançats (ICREA), Barcelona, Catalonia, Spain" />
<meta name="citation_author_institution" content="Centre for Genomic Regulation (CRG), Barcelona, Catalonia, Spain, 08003" />
<meta name="citation_author_email" content="arcadi.navarro@upf.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2162-8246" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=P. Andolfatto;citation_author=M. Nordborg;citation_title=The Effect of Gene Conversion on Intralocus Associations;citation_pages=1397-1399;citation_volume=148;citation_year=1998;citation_issue=3;citation_pmid=9539452" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=K. Ardlie;citation_author=S. N. Liu-Cordero;citation_author=M. A. Eberle;citation_author=M. Daly;citation_author=J. Barrett;citation_title=Lower-than-expected linkage disequilibrium between tightly linked markers in humans suggests a role for gene conversion.;citation_pages=582-589;citation_volume=69;citation_year=2001;citation_issue=3;citation_pmid=11473344;citation_doi=10.1086/323251" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. A. Bailey;citation_author=E. E. Eichler;citation_title=Primate segmental duplications: crucibles of evolution, diversity and disease.;citation_pages=552-564;citation_volume=7;citation_year=2006;citation_issue=7;citation_pmid=16770338;citation_doi=10.1038/nrg1895" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=J. A. Bailey;citation_author=Z. Gu;citation_author=R. A. Clark;citation_author=K. Reinert;citation_author=R. V. Samonte;citation_title=Recent Segmental Duplications in the Human Genome;citation_pages=1003-1007;citation_volume=297;citation_year=2002;citation_issue=5583;citation_pmid=12169732;citation_doi=10.1126/science.1072047" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=D. Baltimore;citation_title=Gene conversion: some implications for immunoglobulin genes.;citation_pages=592-594;citation_volume=24;citation_year=1981;citation_issue=3;citation_pmid=7018693;citation_doi=10.1016/0092-8674(81)90082-9" />
<meta name="citation_reference" content="citation_journal_title=Genomics;citation_journal_abbrev=Genomics;citation_author=D. Benovoy;citation_author=G. Drouin;citation_title=Ectopic gene conversions in the human genome.;citation_pages=27-32;citation_volume=93;citation_year=2009;citation_issue=1;citation_pmid=18848875;citation_doi=10.1016/j.ygeno.2008.09.007" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=E. Bosch;citation_author=M. E. Hurles;citation_author=A. Navarro;citation_author=M. A. Jobling;citation_title=Dynamics of a Human Interparalog Gene Conversion Hotspot;citation_pages=835-844;citation_volume=14;citation_year=2004;citation_issue=5;citation_pmid=15123583;citation_doi=10.1101/gr.2177404" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=C. Casola;citation_author=U. Zekonyte;citation_author=A. D. Phillips;citation_author=D. N. Cooper;citation_author=M. W. Hahn;citation_title=Interlocus gene conversion events introduce deleterious mutations into at least 1% of human genes associated with inherited disease.;citation_volume=22;citation_year=2012;citation_pmid=22090377;citation_doi=10.1101/gr.127738.111" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. M. Chen;citation_author=D. N. Cooper;citation_author=N. Chuzhanova;citation_author=C. Férec;citation_author=G. P. Patrinos;citation_title=Gene conversion: mechanisms, evolution and human disease.;citation_pages=762-775;citation_volume=8;citation_year=2007;citation_issue=10;citation_pmid=17846636;citation_doi=10.1038/nrg2193" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=D. F. Conrad;citation_author=D. Pinto;citation_author=R. Redon;citation_author=L. Feuk;citation_author=O. Gokcumen;citation_title=Origins and functional impact of copy number variation in the human genome.;citation_pages=704-712;citation_volume=464;citation_year=2010;citation_issue=7289;citation_pmid=19812545;citation_doi=10.1038/nature08516" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. Dover;citation_title=Molecular drive: a cohesive mode of species evolution.;citation_pages=111-117;citation_volume=299;citation_year=1982;citation_issue=5879;citation_pmid=7110332;citation_doi=10.1038/299111a0" />
<meta name="citation_reference" content="citation_journal_title=PLoS ONE;citation_author=B. L. Dumont;citation_author=E. E. Eichler;citation_title=Signals of historical interlocus gene conversion in human segmental duplications.;citation_pages=e75949;citation_volume=8;citation_year=2013;citation_pmid=24124524;citation_doi=10.1371/journal.pone.0075949" />
<meta name="citation_reference" content="citation_journal_title=Trends Genet.;citation_author=J. A. Fawcett;citation_author=H. Innan;citation_title=The role of gene conversion in preserving rearrangement hotspots in the human genome.;citation_pages=561-568;citation_volume=29;citation_year=2013;citation_pmid=23953668;citation_doi=10.1016/j.tig.2013.07.002" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=L. Frisse;citation_author=R. R. Hudson;citation_author=A. Bartoszewicz;citation_author=J. D. Wall;citation_author=J. Donfack;citation_title=Gene conversion and different population histories may explain the contrast between polymorphism and linkage disequilibrium levels.;citation_pages=831-843;citation_volume=69;citation_year=2001;citation_issue=4;citation_pmid=11533915;citation_doi=10.1086/323612" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=E. Gazave;citation_author=F. Darré;citation_author=C. Morcillo-Suarez;citation_author=N. Petit-Marty;citation_author=A. Carreño;citation_title=Copy number variation analysis in the great apes reveals species-specific patterns of structural variation.;citation_volume=21;citation_year=2011;citation_pmid=21824994;citation_doi=10.1101/gr.117242.110" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=P. Hallast;citation_author=L. Nagirnaja;citation_author=T. Margus;citation_author=M. Laan;citation_title=Segmental duplications and gene conversion: Human luteinizing hormone/chorionic gonadotropin {beta} gene cluster;citation_pages=1535-1546;citation_volume=15;citation_year=2005;citation_issue=11;citation_pmid=16251463;citation_doi=10.1101/gr.4270505" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=M. V. Han;citation_author=J. P. Demuth;citation_author=C. L. McGrath;citation_author=C. Casola;citation_author=M. W. Hahn;citation_title=Adaptive evolution of young gene duplicates in mammals;citation_pages=859-867;citation_volume=19;citation_year=2009;citation_issue=5;citation_pmid=19411603;citation_doi=10.1101/gr.085951.108" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=M. Hanikenne;citation_author=J. Kroymann;citation_author=A. Trampczynska;citation_author=M. Bernal;citation_author=P. Motte;citation_title=Hard selective sweep and ectopic gene conversion in a gene cluster affording environmental adaptation.;citation_pages=e1003707;citation_volume=9;citation_year=2013;citation_pmid=23990800;citation_doi=10.1371/journal.pgen.1003707" />
<meta name="citation_reference" content="citation_journal_title=Genes;citation_author=P. J. Hastings;citation_title=Mechanisms of ectopic gene conversion.;citation_pages=427-439;citation_volume=1;citation_year=2010;citation_doi=10.3390/genes1030427" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Innan;citation_title=A Method for Estimating the Mutation, Gene Conversion and Recombination Parameters in Small Multigene Families;citation_pages=865-872;citation_volume=161;citation_year=2002;citation_issue=2;citation_pmid=12072480" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Innan;citation_title=The Coalescent and Infinite-Site Model of a Small Multigene Family;citation_pages=803-810;citation_volume=163;citation_year=2003;citation_issue=2;citation_pmid=12618415" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=H. Innan;citation_title=Population genetic models of duplicated genes.;citation_pages=19-37;citation_volume=137;citation_year=2009;citation_pmid=19266289;citation_doi=10.1007/s10709-009-9355-1" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=R. C. Iskow;citation_author=O. Gokcumen;citation_author=C. Lee;citation_title=Exploring the role of copy number variants in human adaptation.;citation_pages=245-257;citation_volume=28;citation_year=2012;citation_issue=6;citation_pmid=22483647;citation_doi=10.1016/j.tig.2012.03.002" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. J. Jeffreys;citation_author=C. A. May;citation_title=Intense and highly localized gene conversion activity in human meiotic crossover hot spots.;citation_pages=151-156;citation_volume=36;citation_year=2004;citation_issue=2;citation_pmid=14704667;citation_doi=10.1038/ng1287" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. J. Jeffreys;citation_author=L. Kauppi;citation_author=R. Neumann;citation_title=Intensely punctate meiotic recombination in the class II region of the major histocompatibility complex.;citation_pages=217-222;citation_volume=29;citation_year=2001;citation_issue=2;citation_pmid=11586303;citation_doi=10.1038/ng1001-217" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=V. Katju;citation_author=E. L. LaBeau;citation_author=K. J. Lipinski;citation_author=U. Bergthorsson;citation_title=Sex change by gene conversion in Caenorhabditis elegans fog-2 mutant.;citation_volume=180;citation_year=2008;citation_pmid=18757925;citation_doi=10.1534/genetics.108.090035" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=O. Khakhlova;citation_author=R. Bock;citation_title=Elimination of deleterious mutations in plastid genomes by gene conversion.;citation_pages=85-94;citation_volume=46;citation_year=2006;citation_issue=1;citation_pmid=16553897;citation_doi=10.1111/j.1365-313X.2006.02673.x" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=M. Kimura;citation_title=Average time until fixation of a mutant allele in a finite population under continued mutation pressure: Studies by analytical, numerical, and pseudo-sampling methods;citation_pages=522-526;citation_volume=77;citation_year=1980;citation_issue=1;citation_pmid=16592764;citation_doi=10.1073/pnas.77.1.522" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=A. Kong;citation_author=D. F. Gudbjartsson;citation_author=J. Sainz;citation_author=G. M. Jonsdottir;citation_author=S. A. Gudjonsson;citation_title=A high-resolution recombination map of the human genome.;citation_pages=241-247;citation_volume=31;citation_year=2002;citation_issue=3;citation_pmid=12053178;citation_doi=10.1038/ng0302-241" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=B. Lorente-Galdos;citation_author=J. Bleyhl;citation_author=G. Santpere;citation_author=L. Vives;citation_author=O. Ramírez;citation_title=Accelerated exon evolution within primate segmental duplications.;citation_pages=R9;citation_volume=14;citation_year=2013;citation_pmid=23360670;citation_doi=10.1186/gb-2013-14-1-r9" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Lynch;citation_author=J. S. Conery;citation_title=The Evolutionary Fate and Consequences of Duplicate Genes;citation_pages=1151-1155;citation_volume=290;citation_year=2000;citation_issue=5494;citation_pmid=11073452;citation_doi=10.1126/science.290.5494.1151" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. Mano;citation_author=H. Innan;citation_title=The evolutionary rate of duplicated genes under concerted evolution.;citation_volume=180;citation_year=2008;citation_pmid=18757936;citation_doi=10.1534/genetics.108.087676" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S. P. Mansai;citation_author=H. Innan;citation_title=The power of the methods for detecting interlocus gene conversion.;citation_volume=184;citation_year=2010;citation_pmid=19948889;citation_doi=10.1534/genetics.109.111161" />
<meta name="citation_reference" content="citation_journal_title=Genes;citation_author=S. P. Mansai;citation_author=T. Kado;citation_author=H. Innan;citation_title=The rate and tract length of gene conversion between duplicated genes.;citation_pages=313-331;citation_volume=2;citation_year=2011;citation_doi=10.3390/genes2020313" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol. Evol.;citation_author=G. A. B. Marais;citation_author=P. R. A. Campos;citation_author=I. Gordo;citation_title=Can intra-Y gene conversion oppose the degeneration of the human Y chromosome? A simulation study.;citation_volume=2;citation_year=2010;citation_pmid=20624739;citation_doi=10.1093/gbe/evq026" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. Marques-Bonet;citation_author=J. M. Kidd;citation_author=M. Ventura;citation_author=T. A. Graves;citation_author=Z. Cheng;citation_title=A burst of segmental duplications in the genome of the African great ape ancestor.;citation_pages=877-881;citation_volume=457;citation_year=2009;citation_issue=7231;citation_pmid=19212409;citation_doi=10.1038/nature07744" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. L. McGrath;citation_author=C. Casola;citation_author=M. W. Hahn;citation_title=Minimal effect of ectopic gene conversion among recent duplicates in four mammalian genomes.;citation_volume=182;citation_year=2009;citation_pmid=19307604;citation_doi=10.1534/genetics.109.101428" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. E. Mills;citation_author=K. Walter;citation_author=C. Stewart;citation_author=R. E. Handsaker;citation_author=K. Chen;citation_title=Mapping copy number variation by population-scale genome sequencing.;citation_pages=59-65;citation_volume=470;citation_year=2011;citation_issue=7332;citation_pmid=21293372;citation_doi=10.1038/nature09708" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=S. Myers;citation_author=C. Freeman;citation_author=A. Auton;citation_author=P. Donnelly;citation_author=G. McVean;citation_title=A common sequence motif associated with recombination hot spots and genome instability in humans.;citation_pages=1124-1129;citation_volume=40;citation_year=2008;citation_issue=9;citation_pmid=19165926;citation_doi=10.1038/ng.213" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Nagylaki;citation_title=THE EVOLUTION OF MULTIGENE FAMILIES UNDER INTRACHROMOSOMAL GENE CONVERSION;citation_pages=529-548;citation_volume=106;citation_year=1984;citation_issue=3;citation_pmid=6706112" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. Nagylaki;citation_author=T. D. Petes;citation_title=INTRACHROMOSOMAL GENE CONVERSION AND THE MAINTENANCE OF SEQUENCE HOMOGENEITY AMONG REPEATED GENES;citation_pages=315-337;citation_volume=100;citation_year=1982;citation_issue=2;citation_pmid=7106560" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=K. M. Nielsen;citation_author=J. Kasper;citation_author=M. Choi;citation_author=T. Bedford;citation_author=K. Kristiansen;citation_title=Gene conversion as a source of nucleotide diversity in Plasmodium falciparum.;citation_volume=20;citation_year=2003;citation_pmid=12679555;citation_doi=10.1093/molbev/msg076" />
<meta name="citation_reference" content="citation_title=Evolution by Gene Duplication;citation_author=S. Ohno;citation_year=1970" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=T. Ohta;citation_title=Allelic and nonallelic homology of a supergene family.;citation_pages=3251-3254;citation_volume=79;citation_year=1982;citation_issue=10;citation_pmid=6954477;citation_doi=10.1073/pnas.79.10.3251" />
<meta name="citation_reference" content="citation_journal_title=Theoretical population biology;citation_journal_abbrev=Theor Popul Biol;citation_author=T. Ohta;citation_title=On the evolution of multigene families.;citation_pages=216-240;citation_volume=23;citation_year=1983;citation_issue=2;citation_pmid=6612633;citation_doi=10.1016/0040-5809(83)90015-1" />
<meta name="citation_reference" content="citation_journal_title=Genes;citation_author=T. Ohta;citation_title=Gene conversion and evolution of gene families: an overview.;citation_pages=349-356;citation_volume=1;citation_year=2010;citation_doi=10.3390/genes1030349" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=B. Padhukasahasram;citation_author=P. Marjoram;citation_author=J. D. Wall;citation_author=C. D. Bustamante;citation_author=M. Nordborg;citation_title=Exploring Population Genetic Models With Recombination Using Efficient Forward-Time Simulations;citation_pages=2417-2427;citation_volume=178;citation_year=2008;citation_issue=4;citation_pmid=18430959;citation_doi=10.1534/genetics.107.085332" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=V. Plagnol;citation_author=B. Padhukasahasram;citation_author=J. D. Wall;citation_author=P. Marjoram;citation_author=M. Nordborg;citation_title=Relative influences of crossing over and gene conversion on the pattern of linkage disequilibrium in Arabidopsis thaliana.;citation_volume=172;citation_year=2006;citation_pmid=16299388;citation_doi=10.1534/genetics.104.040311" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=O. V. Plotnikova;citation_author=F. A. Kondrashov;citation_author=P. K. Vlasovb;citation_author=A. P. Grigorenkoa;citation_author=E. K. Ginterc;citation_title=Conversion and compensatory evolution of the gamma-crystallin genes and identification of a cataractogenic mutation that reverses the sequence of the human CRYGD gene to an ancestral state.;citation_pages=32-43;citation_volume=81;citation_year=2007;citation_issue=1;citation_pmid=17564961;citation_doi=10.1086/518616" />
<meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.;citation_author=H. S. Rane;citation_author=J. M. Smith;citation_author=U. Bergthorsson;citation_author=V. Katju;citation_title=Gene conversion and DNA sequence polymorphism in the sex-determination gene fog-2 and its paralog ftr-1 in Caenorhabditis elegans.;citation_volume=27;citation_year=2010;citation_pmid=20133352;citation_doi=10.1093/molbev/msq039" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Rozen;citation_author=H. Skaletsky;citation_author=J. D. Marszalek;citation_author=P. J. Minx;citation_author=H. S. Cordum;citation_title=Abundant gene conversion between arms of palindromes in human and ape Y chromosomes.;citation_pages=873-876;citation_volume=423;citation_year=2003;citation_issue=6942;citation_pmid=12815433;citation_doi=10.1038/nature01723" />
<meta name="citation_reference" content="citation_journal_title=Molecular Biology and Evolution;citation_journal_abbrev=Mol Biol Evol;citation_author=S. Sawyer;citation_title=Statistical tests for detecting gene conversion;citation_pages=526-538;citation_volume=6;citation_year=1989;citation_issue=5;citation_pmid=2677599" />
<meta name="citation_reference" content="citation_journal_title=American journal of human genetics;citation_journal_abbrev=Am J Hum Genet;citation_author=A. J. Sharp;citation_author=D. P. Locke;citation_author=S. D. McGrath;citation_author=Z. Cheng;citation_author=J. A. Bailey;citation_title=Segmental duplications and copy-number variation in the human genome.;citation_pages=78-88;citation_volume=77;citation_year=2005;citation_issue=1;citation_pmid=15918152;citation_doi=10.1086/431652" />
<meta name="citation_reference" content="citation_journal_title=Annual review of medicine;citation_journal_abbrev=Annu Rev Med;citation_author=P. Stankiewicz;citation_author=J. R. Lupski;citation_title=Structural variation in the human genome and its role in disease.;citation_pages=437-455;citation_volume=61;citation_year=2010;citation_pmid=20059347;citation_doi=10.1146/annurev-med-100708-204735" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=K. M. Teshima;citation_author=H. Innan;citation_title=The Effect of Gene Conversion on the Divergence Between Duplicated Genes;citation_pages=1553-1560;citation_volume=166;citation_year=2004;citation_issue=3;citation_pmid=15082568;citation_doi=10.1534/genetics.166.3.1553" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. M. Teshima;citation_author=H. Innan;citation_title=The coalescent with selection on copy number variants.;citation_volume=190;citation_year=2012;citation_pmid=22174068;citation_doi=10.1534/genetics.111.135343" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. R. Thornton;citation_title=The neutral coalescent process for recent gene duplications and copy-number variants.;citation_volume=177;citation_year=2007;citation_pmid=17720930;citation_doi=10.1534/genetics.107.074948" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. Uddin;citation_author=M. Sturge;citation_author=L. Peddle;citation_author=D. D. O’Reilly;citation_author=P. Rahman;citation_title=Genome-wide signatures of 'rearrangement hotspots' within segmental duplications in humans.;citation_pages=e28853-e28853;citation_volume=6;citation_year=2011;citation_issue=12;citation_pmid=22194928;citation_doi=10.1371/journal.pone.0028853" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. B. Walsh;citation_title=Sequence-Dependent Gene Conversion: Can Duplicated Genes Diverge Fast Enough to Escape Conversion?;citation_pages=543-557;citation_volume=117;citation_year=1987;citation_issue=3;citation_pmid=3692140" />
<meta name="citation_reference" content="citation_journal_title=BMC Evol. Biol.;citation_author=C. S. Willett;citation_title=Gene conversion yields novel gene combinations in paralogs of GOT1 in the copepod Tigriopus californicus.;citation_pages=148;citation_volume=13;citation_year=2013;citation_pmid=23845062;citation_doi=10.1186/1471-2148-13-148" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. Wiuf;citation_author=J. Hein;citation_title=The Coalescent With Gene Conversion;citation_pages=451-462;citation_volume=155;citation_year=2000;citation_issue=1;citation_pmid=10790416" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Interplay of Interlocus Gene Conversion and Crossover in Segmental Duplications Under a Neutral Scenario" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8.cover-source.jpg" />
<meta name="twitter:description" content="Interlocus gene conversion is a major evolutionary force that drives the concerted evolution of duplicated genomic regions. Theoretical models successfully have addressed the effects of interlocus gene conversion and the importance of crossover in the evolutionary fate of gene families and duplications but have not considered complex recombination scenarios, such as the presence of hotspots. To study the interplay between interlocus gene conversion and crossover, we have developed a forward-time simulator that allows the exploration of a wide range of interlocus gene conversion rates under different crossover models. Using it, we have analyzed patterns of nucleotide variation and linkage disequilibrium within and between duplicate regions, focusing on a neutral scenario with constant population size and validating our results with the existing theoretical models. We show that the interaction of gene conversion and crossover is nontrivial and that the location of crossover junctions is a fundamental determinant of levels of variation and linkage disequilibrium in duplicated regions. We also show that if crossover activity between duplications is strong enough, recurrent interlocus gene conversion events can break linkage disequilibrium within duplicates. Given the complex nature of interlocus gene conversion and crossover, we provide a framework to explore their interplay to help increase knowledge on molecular evolution within segmental duplications under more complex scenarios, such as demographic changes or natural selection." />
<meta name="og-title" property="og:title" content="Interplay of Interlocus Gene Conversion and Crossover in Segmental Duplications Under a Neutral Scenario" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/8/1479" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Interlocus gene conversion is a major evolutionary force that drives the concerted evolution of duplicated genomic regions. Theoretical models successfully have addressed the effects of interlocus gene conversion and the importance of crossover in the evolutionary fate of gene families and duplications but have not considered complex recombination scenarios, such as the presence of hotspots. To study the interplay between interlocus gene conversion and crossover, we have developed a forward-time simulator that allows the exploration of a wide range of interlocus gene conversion rates under different crossover models. Using it, we have analyzed patterns of nucleotide variation and linkage disequilibrium within and between duplicate regions, focusing on a neutral scenario with constant population size and validating our results with the existing theoretical models. We show that the interaction of gene conversion and crossover is nontrivial and that the location of crossover junctions is a fundamental determinant of levels of variation and linkage disequilibrium in duplicated regions. We also show that if crossover activity between duplications is strong enough, recurrent interlocus gene conversion events can break linkage disequilibrium within duplicates. Given the complex nature of interlocus gene conversion and crossover, we provide a framework to explore their interplay to help increase knowledge on molecular evolution within segmental duplications under more complex scenarios, such as demographic changes or natural selection." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8.cover-source.jpg" />
    <title>Interplay of Interlocus Gene Conversion and Crossover in Segmental Duplications Under a Neutral Scenario | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"NiH5uAM3cbtdEfMoOzgVOb90EhpVMtKQ0jkUPMqSTw8","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459948","apath":"\/ggg\/4\/8\/1479.atom","pisa":"ggg;4\/8\/1479","ac":{"\/ggg\/4\/8\/1479.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/8\/1479.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/8\/1479"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/8\/1479"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/8\/1479.full":true,"\/content\/4\/8\/1479":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022PywnqplnRT1fhDXNkC4bCAn+bcext3sooua5lZRh6HkPdPbVM47PT7dxQ7mC9ZHAv5JV9jUhhnJcW8v3klEO2IK41PYlhj7JWEnPzux6dF3ZiURUr9tQPDhtTnUkyFSoN1U3Y1KuLJQ1IxKKqW4+WaFF+xPK4k9VG4KmJj+ieii3kqEuozV7dU1Ro8r\\\\\\\/9lZOZPRYlbeePvY5l8Rqbd\\\\\\\/yGM3VHcW0oHS95H58aDlAtsp+NQR+6Br9HEGglodJrCal+FP2\\\\\\\/q24N8919ov0Tc1QWvDAiYFdPbkCsWZ6LQHOoxSeZdvF9c7v5LTjLvVnk6Cg6PHSXPafnVeyfCGeBlND7OukVgAupHNJdrQwB\\\\\\\/u\\\\\\\/Ego4K6OPo7+3Hrj++CNjfoN5C35oeZ9YZBPhsDBtWnJzTWswGQJzrtSClbBv5DKSF32antdJC6V\\\\\\\/92lbhxwHvvGzq+NW+zY5esaQtBomD2kXiGvaxI5TEcMxvDudd33CKaKeSF6Apaddsd90AVSg4EHEUOh7e6jqSUQ7NL01HGzoJs8AzCGbPLDVFAvLtg2SDjwD0WSWM6YaIuZmT7fscqj5gOnhhuHIBcszkG4RTv2XO3GUj4klXB+DEnBI8VLxjvk2fUFT8AoLGHljuIYiXC3vagaZpmt+7Lg7QjKcttCc4BjHWgHn1qMJ+kJbQZCOPfsU6ZqanJUR0VlZNnym6QPfiFLQgodgG06zD+TosMoF5\\\\\\\/mJhi6VZHf8rxF3mtD085WvD4wo9FXaDehkMce6HvnSviM794ueBRnqUoe\\\\\\\/lNMjwWk3lQl\\\\\\\/TkMJ1FDnTBlu4AUfX+qL6\\\\\\\/oa+fd7tkm9f\\\\\\\/oI4lKimvEUmN1DpqThuNt3MqsjRApbf2ze68tZmVsBjwrswayMLdaSsoFzf9DrJjuZRKvjrSsDvthw+M8tENLMlmofDUvIpIQMmWCbpW2D5ezKkb\\\\\\\/Rt+2UCQ+7gUOpSWt2\\\\\\\/9r9Yz+72TWoE7126DyX3NTIfH9uyIkuh\\\\\\\/gUXS0lVrd2S7FtAfE81KYHTG8PPCFEGt2lqx8+Sc2+C1Vn\\\\\\\/vaDED8c8vH7MPwPLiC5QrLyB3iW+K6n97UgMJX4x\\\\\\\/nwvSVIuJ9sh6VpKRHHv0pKoQG1FFQ2Xz8\\\\\\\/FBTIw9kVGIyNV5qjhkilZhBfr\\\\\\\/ZYTJgtmh48XP3o9bdrhCv\\\\\\\/0LL4tIK6a7kfcwEfZOqHt6SCB7Np+uRNn0cdu0+OW5gC5BLat6Ca0yG8WXqCrIx4pElDrxo0I5IcDTookwtNG1ed+tRZdoBN3bqwGPtZLyGzr3+lF1fF\\\u0022,\\\u0022iv\\\u0022:\\\u0022V4U1jEgLudRhg61PDldPMQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00228166dfa3a5d72d29b2d887c225196c76d58feea33830de417c0792634b57ffba\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022z6+dSLh1ipLDWIOGh2k+y4TtUmftR5g8lt4ZvH1\\\\\\\/36IefBpXIAE6L8ThhUp8\\\\\\\/n1Zg268vgTfS0SNOQ12AN+Vv13YAFGri3sMdJbL1so\\\\\\\/RvRU6GwZwQyFgd2wt2eZZfbiYEdZ9Y6sCCCWrta6axnDxLR7lFO0CYL5Tw9bVMRK8+oIMrffTjYn829JcUhEWRdxh6WFTkzbZAvow0bZ2BSVTE31flsFA6y8y93+bF8Y4m5MAMI1sJ0okRyKs38ciq3q7rupwxwBQpoKyqr3ilsFvzvnMF4LJprDzuQjahqLw\\\\\\\/7FLCk5HasNIyxcaEYpylb5zgFtyX6CHJqYp2uFyTy63G9OigTTNQ0eGOxQwiGQxUBEMNzumNwDKzWBcQSyxehZ6N0Cukjbm42QxGBdGSSwN+zVAhYvrEfGioZx+bun\\\\\\\/8G57QQkrOWYJKIZszERtHCMDMn3AqJVRy0qyJZgHsHHduAE7eNtkMonb5BiOQXhkj2Id9E2botx+eg4p2ufkNHPwILwAUmge8kHef2+2X7ICTYQcbkKYAdgqr6qxPsrw1ancqImBuZFVCjLfvy2gLimqGRbraOHTsppuJMhOKYyNPqvbdDZbWHzM2bxnyuw4yQAT+ceDiahWfwXcjkbFHpKcXwnfblXTqYMRfP1Iuhajfxo30y0btJak1GZ02sUSQqsokiGNeWqJnhPlixaU\\\\\\\/yZ+zX0D1fg5RAvYsso9O2dFL4spAn\\\\\\\/UsERjPLdvKD5BnfZbo8xvUxRayarFlKCkH5ePfxu\\\\\\\/SN98C2DXsl1pBFS9JxKKuIDcAuhpl1o\\\\\\\/\\\\\\\/nb3RAj4v5s4pYIW48MD3X3l1frwdUxRvFWY2a9+53CgHISctL8ht\\\\\\\/iOn\\\\\\\/Kr7yatfCYuZEsF774brPu8k\\\\\\\/sr4OY2hFXc8NmKFFEVq\\\\\\\/N+hETRjMRI02B\\\\\\\/PUleyKqT2fcNsogKO6Trv\\\\\\\/wDflNqDUHRvIykdiAtNOdKEnXoxe8UWU8dYom5S2jj9rQUAbwy\\\\\\\/z691tFGL9Jsx9S6s\\\\\\\/9YqPVOM4H3G8iddcjg6XD7MuMC5cyVJuvlEJNRM4HYhiAsbGJ1Abnzfa8xkX9sECl3eJUZoe7W8iGpyYcwmL\\\\\\\/26\\\\\\\/0saZdmYLyDalLiGrAUlJJjDgHFE\\\\\\\/WAOjuR1Hgk8Cc9KXI1lxHXxKcWylZlzlpbS+fB3swiCSJjQH\\\\\\\/UVAujzBZf8pk3G\\\\\\\/fw+p\\\\\\\/miK754tmjEOfbd2Bf4L6H8PdNXisrLuR+5SATla20Y\\\\\\\/Bds5x9EyRD\\\\\\\/mb35tmkYpBJ9yNpR76VzbYb2rp5C37rA3DsyFofNMmvQsWqRug6j0f\\\\\\\/LWz2uP7uA0gAGK78TUH5TAMTVnHxAE74MCqr\\\\\\\/4G8LATFDEDzG0HYaAsjh09rZ9em0tMxd1mUOjrc9BUH6lzbXqvzZ01Hq2y+EqHKuNgERsGxP\\\\\\\/UvxtwC0xwYAQCSlN90yuoaXwFY8A3UnaT67RHvSlmAlCdno1HNb0OMllpDgIeVryodgoR6YGmJcF9ggOIpa9IH5jDUWJ9K32vBia0pG0lmhsDptCmmVACEILWTQsiFb76wONU1HwDBWs2wYpYSMjpaGoQOhVIyVFEFUAPww7r1RpiPy4UgQ24aQkmoNLvgP2ZV+9qo5h3S0ftRCDpijkmyIRFYZjHe4xz1ozyoXWY0l\\\\\\\/lx00+KD1LiU9BE6cqd+wK9I+\\\\\\\/14leCEeSfsAdyzO63aOJNgLlneEbZ8eIjIwPrYvdiEs5dD5GfGqql\\\\\\\/B7RhaoeMBOpW39wV1yNtmQ0X5CS0KLTiYaBpoxYUHDZcRybKRjrXz6MTLdW9rXkQcTiKZ9PgJewc7rP6iOiIT7EytrJzCLtKIX3gYWPsGls5H118iXw8ouxErc6J2PXA2LCDFOCkWt5WZv9U6hVYdQ+gD8BKzo31LN9VYi4G5Dz8JIXr3Ei7WJF4M0yh9AGjdcQccFKTxXWqurD2YTckpkNRgwsFsgHqNeaBWeoAyYDBOYDahVRaTDo4PWIvQpZtXw3Sfe\\\\\\\/rwQyJYyzJquj1vl4+1LJoZGHYG1ZKe9tQcOvR3cLDgSsYs0j1Qx67O7gzPMUZrIXHx+wJ1EVrpAfxzEw7noLvWd1GOYxD6ZdGqqCzl+FjpbO0KIYlzujsQDf0gxjRxIay+JYOmJrE0pUof\\\\\\\/7FJWI\\\\\\\/oeb\\\\\\\/qKuz+aiGIcG05vOD3IwjZ1Dt9d97BHZyhYIMzyXG\\\\\\\/c2gpGUTCtumPiUnYQuoojLeS7aT\\\\\\\/l5qtiK\\\\\\\/D7H5BcStxxvZuV4SsoLLlqB3oipxKUxnRTEJJVuGwxvt\\\\\\\/taR+uClzGs37xGs6s2skdKumlr0HNSUNr2h5sbNJN+QvgKH7JQghOSrciu53wyhZAxdmTXMBZb9YY9m\\\\\\\/5wit7wuaFv0cZOXoFfIAtfg7ApXj714Ae1MK\\\\\\\/gBeRJd0n3aKeu\\\\\\\/T1lGl2WjbJYt9M7ZPJ5IKwSrSHjVR01QwJVMPDfL\\\\\\\/RjCzERzX16h3tXSKAmqJuOPvt5Dl0xRhvH0z0jKsoG17xzsgj\\\\\\\/0PqP8XK5OE7s6HFdcG0pMg8jdY4mpFX9c6XfP4BB6gksvLkCrmQ59zQs2ILm+4cDLtS1KFOlDT\\\\\\\/17mkBxytCwA1cC4SjNqcO8rB0SobPEmrdGaiUQNM9swpvmY9PRCQnklAFsgcS6lQNagcABjaVLMnJmU7gSAo7xadVXcf0jW0XzLT+eCWWDnqHTHSn2yYig0WMy2cVFvy1XLnTjMDUhmADIkbgKG8z1D\\\\\\\/KiLwxSjgI0DfOiw5CreFe2oDOpGPpRugy7XXJLHRX2tPUxIGGIxjo6XlywkwK8FxDSSdnChJg2rAiYeKC6dt3laQA0fgsnOTb6k73QBqPb\\\\\\\/nM6jPYMgwUftU0sHrm3RI3Dowb4tJrHTXbC762YjMn8G8NFYxetO8dKM\\\\\\\/eMl+ZU7T7v2ZJTIGZojuKvtH8bWgpBj4kliX4Pz+rugHcI3EzPETfv\\\\\\\/VrFQfOTl2l7QVIxD8vUYldBKQFO0\\\\\\\/JJAxTyhmxxfQEqDJ7hA1zcmhr4sV9duzd8QCwdHG8PCdRbS0S9IC\\\\\\\/4hB9gRPosTQ9+Oi\\\\\\\/VRx7KoEcvOt1eTyOH667QOsYgGdRphw5NPZX\\\\\\\/apSXwslOB7pF2WS2cV5sff106sSphxvyaxHE\\\\\\\/9mTvOvygXfbvoIc0YGsUfK3f2AGeWaDq7o65WzcQum6mZcHtX71yrTZpcfy3qnk06m3OI4Od+s0jMFySiHa0HezkVOMo8YKETq9Pf2FDBYm6hQScyEA15dE99ZwmM5Vk0XpSMvQa37gkb0HLy78rWSZf8QoYpzJ1oZvYdKkxBhH7UbJmGieJBIhy\\\\\\\/XXuflOOdBiMwEJNBp2G9uZsm8EJJ4XcGJQjEimjkmuXztFSUFTvd5rwxBmwhA5Vc1zaXoUrB+\\\\\\\/LtnDQ+NapXokHN9l5Hy53\\\\\\\/\\\\\\\/1nMMDrmhMWkUJeRWpYp3wMW0+KotjUpRBOrYlKNwR6Curl1xCNoUSk36jIbp\\\\\\\/8QrB4TMuoyxilvqxC7hmLIFIiqDjfgcwC7v1DN\\\\\\\/rDR\\\\\\\/Ml0B2BbLUjG6OZzZJm2zmRAOlv0aRyn\\\\\\\/FoLCeQNXlQABc\\\\\\\/YVkb3fO6O\\\\\\\/9bXHJnlitmnoth0eZtrOAYeixYi54qMuKGY23iwXX4MQ663zYfsz80pp45vh1fGsQmsvyCSrigaomUuO4Q7FyRVj4L1M9NE4GS9ryFMl8hoJB2v\\\\\\\/IMmHqcul6EloSfm0DbRmZYATVp6\\\\\\\/p7WxKzk5M\\\\\\\/SfijstcZfe9YXdcjFGfFB0lTa3O14iU6rgQyuTdOclQD91SRB4Q21n2ooBxhIEhIZfAs6C\\\\\\\/8ssDtBjkrArwQExglipmWhlx\\\\\\\/V\\\\\\\/eskBq7fZNsxnNdN7yyi5+uu7lOq\\\\\\\/R649QmPoimjOviUFV1ZWZyzN5kUp6Hh6QovcEanZ70RUA0zbkrOVe\\\\\\\/iqaWcRCMjYZUcafmtVEE7kZerh0LWvRaFa+7S0xbUl0VfomOTlTj3QO6JIJXXQwc3FYf52f1Y\\\\\\\/CgqaRNHQzmE2tptSoi+\\\\\\\/c2O4+vQw3YZrnaw5vGQ3DUsJ38coAW79sXE0ZyjK7j3OsoZMsabsKarqK3xEEo+\\\\\\\/gIQ6vlEGewenKzqbltM9zZn94edoNrGojfTwV4f1nKY1dQgY01fYQrQDsFjbi8qEBFxVwajXj4ap6QZm23o1EDB0jH3MnxuXV182B7jZ+MCZoZH5zapp4vb0lL\\\\\\\/MMrMgYO658LY+7EYxdsyLhfAcwVbniawjjJ11gDO6o5BnrauCr2pnUPQbRn+Mz0mWy0v9\\\\\\\/gq7g8oicDAnLYflcx3CrsVYQuk1LfDL6JMsJp8PCjgZ+J8h1p1ZCeThzxxOIpfcpvkhDl+JBT3MzIBIrp+12s4m3TLFZSSAtOCkmx\\\\\\\/ywZR1BLwfn6aeGzd8h5kIR94g+zBMELtpnGlVoJUBBUsgpUj0sGn71ezlfC4cpu8Jvk7gN1gwS5g7LJG1fwRZw1qYg3gizkNynRV2MVWKdGcjTxLLfban+ovJUdqTUg6FHaSNJdrBsSGauHjX5tSPWR87scq8IB66D2ZlzsNl3U4CeFHS2K7Jpt7IM9SZ1NW0J3gSzJQtXeeWq0jFks+uwOpEsW\\\\\\\/xzGHjpW+PdniZjS+pEmInW4Co5Pq4UmgjNP5mRle6363WCx49kKrowwUSI6Ynwnmr9utLpF3EPowRhuquZs6hAA0JXJVKt70JcSzXCwiJK0KdceAjjDpViRjF2VLHVELFJDYHTheqSqEhIoN3wcQpEWiPeqDNmTYHVdGLQqqPcTpu4ciuvEKSmYdeTuduJ28yNrSxTkJIf8S+6jK8hNdbaZpb01YnUsBhYVrPvj+I6jLSAfsBYIkI6fY+GbD4nEwkWkZfoQ8JvLO8X1mtxxAFP94uCh3yBiZVAI5tadzgdJNK731LMrQ2IAGkZ6W+xeULERD7rap6iQxiemUyuWD+6+qeyXGgg0bbmPxk\\\\\\\/P8JhvlnDpnkcOOZQCNDZcShU9WOKEJEx+bB1EbwrtM71omkli7I543CaEyg\\\\\\\/NertOctpNPdqLy1JnC6jgZN+6e97hbIGdeNKi0Kehk+F4a2CaS6wWgCMl9vxVQLthA9UkqrAoH2fQPBdJAME5QXI91KdXFlTcKdZUwJib6FEgo1+CFNfkzKTTYsaJ2CIiErzF\\\\\\\/BfQl3j1iHiKIbzL86v5hRlTcp2HBE4RB8rkMLwgzwxv6MZuT2n0DlQSPhU0F07dNWxsGMLb0jWUfp+Xgs5Dyyt82dpVyNjXv87vE+9muNPHwps6f0As4bQOXnjmYkw\\\\\\\/RMF\\\\\\\/IXdJbKi6VU0GdjtUYishkm9buFbuiHjkYNKB4o5O0ChTK++RqHB7\\\\\\\/s9EQMnJP6xlaPXDZ0+7D82LqfRT8d4y2\\\\\\\/W5thRfVOLEUKeX2iE3W6V6ZFyqJcsSebCevVZ7Erge17\\\\\\\/B3iLLOTbe2nr8rIlhX79o36SkjSLpkt6h40QdZVtUr3yUd2+XJr\\\\\\\/f8O2CQusRCY6lDoYt4ix95UTKYTULRXUh9L65V6NBoYqiegs9zr1xBByi8UxLBwgnhjcTXs6P5CFJu7bf2lN5IVJEWe6qUkVP+JiRhFuLNfWyu7TZ22zoe+5zQN4RrMxoyJPukJ3U6P5ibkf9gh0BDRgRPCSsRhS5CHj\\\\\\\/eQMh7KjiRbZma\\\\\\\/w67bdHToqXPut\\\\\\\/3zgxBGm0VJ9yb7tbuGMw\\\\\\\/un9EYoAjYrmGht79UXeADU7fDaGD4SQIiKR9JoXTVMDzBcZAsc4aU8sfuDMqSVmPF9Xx5N3WgmMBDSL8J6Ql7CkG2zwkOBqB0Dr9P4KR1aS\\\\\\\/4N+odMyqy7Sz9kuNS5p7WVl7YUI5peREXRtkG3lWagnzLX0\\\\\\\/\\\\\\\/TXOj2DxbsbTsTbIkLcISFZ8oLxOub2UzowwGIzTMPpsDo0excAKFLyyQdGGJVXAYzJ0+8uJMTvSXwrQlxCSjjmbLaxl9IO5EcxVQ7GEDaW1e5ebTYcVwdkHsiqD\\\\\\\/cgUd7OoP+biG2DS8WKYLiV0mJpAeFPQVx0UOJBEIhP8IGyvGT\\\\\\\/8s0vKdxGivo287kezv\\\\\\\/iGJevTZY4SgO\\\\\\\/oGCV2FUsg+7kFWp8JMF4h5WpXTURx8qeuYOrnSkpFclDgDGlxFNbTjoWkBIjnZfmSR0q3QW63uVj180i+V6SGh2+Esf0aTJUBJ\\\\\\\/0IZaslZX5GLe9219n0X+2sm\\\\\\\/HMF2YgjMCpkUc0Yt2aa+49xDyJ1AmCsmRYpAIm7yiPGD+rCbdFStEgLQfpcn5MK6PEk5k8TggcVzbUyamvZi\\\\\\\/sf2Fsfn1ct19ZP8KPc8e5yVzOlpknSm25UaEuEYGlxU7+MjnmEtiXUTHJ812M7hz4XB5m67J\\\\\\\/zRbUkaolCvZwcMCo\\\\\\\/Unz1Cd522XxctCzxyZoOxR0Vxk7fqGvdwpHME4gK7V\\\\\\\/3v3w\\\\\\\/OBhN3mvUTNX8oksOmEGUtL5WZTeV1DcHw7oKRa2KQ9DN4RAGXLmjAs+59BuvgySANMjNPASc+4J92FmFRRdTfTWXItUyV37tmSYan6TxffIlZCyQUyEoQuFolFcq2AB9K+IsvF2dYwo1NwINWduqWk6YfdaWSMQH1xoW1XDLnsuzcny1HqusWGbXnhPbgTU4+9QhoF7ekwVX0L0sPiST21Y6OBanxNJUUGwWn42nuznyMMXuHOh4H3C09Ybb4ED79n\\\\\\\/o4SecjWqNEHYW4wB7gtYMi2CEJHYT5i7mAdfqNZYd42ukqA7KH\\\\\\\/SUZB37F+MQlBWTSS5HskqyjxW6HtwKUxQLhQy7SRxE4XdDV3kKvHvBUWrxhOV69mW3rEwjNmvMU+KXOTF\\\\\\\/Vm9Fi2dVe9x0DLz8Dr88hpIRH3Ncemyo3gIvdGhtShZ\\\\\\\/mIXDEKEoEZyzZK31Oa6I8USVzpgqMHizjedRNnLYAV0c6oeLbIUq4\\\\\\\/aqT7BLsjjNUHd+sjX8C8XFj9ccIeVuDU8mTnQV+lyV3+UAgEMvD5ntmLT2ANSxElSktfY5fbUIE39WmnNdn9EO+y0chHPw+e2GCfrlW0YZ7l77\\\\\\\/IdDr8hreMkRa5Fiw7k1gh0yyhi9DtgtH1iDI6p7GK+QXdpRnxjpU2ixb3VAByyn7QpxwtR99ZpCFRUZASOf0ToOPGoWgnm2LDI3Q4VdNLUbe0rwhJtd4\\\\\\\/\\\\\\\/CR17nPgYwhd0gCH9qKa\\\\\\\/3GJKhOHlxubcQ6TFXfnLg+d06f8Ewv78mqJDdWztmI5LXpmMDj9pv6bwk1AcnNasMrVGZAEnxesiAMvoAIq9ippSdbIznKSyGNJs5tr2XZ8g+yh5NviMukl3znqGm3ue5+TYoCSHbmEhHCT1rF76\\\\\\\/Z1g1rQ4lBaqh77R+OAvVOuVGF5O9W\\\\\\\/klWR+5gPDmnocWcChGff1nzDlLeBAlHZGTVZHdGIPI4OC25rbLyeD\\\\\\\/LGjE3vjjqoBP+3QgntB9PWxGBOUCXKAuedNxDAdEptA3Uxp2TQlyR9grvp5vCIQSCip3D8j1wHZi0P3ljB5FkL\\\\\\\/5FIvgofUdDJ22qxbGJT44TOywm5uv6SZARFYFS6MEgedPHBOIR\\\\\\\/+M1Vw8xbUoF5OH6Xiu3+FcYxYuBXQxN\\\\\\\/u4gSYHvzpZtDXe38lrW6jpjYwplHTTvWBWGXO61nUBJqj7qlSrTNrRcL+G247Zgt4airkld7U6+Ks3a+TG21rZR+3ofrR\\\\\\\/gvi8Wb6t\\\\\\\/AfEVZmWs3\\\\\\\/O213EPGeKKlKwQn1ptorHhPFrp+gP0nqtb2nRLWcisr8KfJtpcdQFbDjNlLEDZPf0XetKJ5uEk8y31U\\\\\\\/JHbIoL5PXyVOya3qQvjgq0s00Jr5oFzh\\\\\\\/ZjUY2Gmns3nGfCkGsQKnYUm1e\\\\\\\/CB7odRuuFXdR28PBL\\\\\\\/sb78qnsx2HvpJSNFhWPKestPNRdn9BLiLV7JGF7c2Svh8ynawZpko8n+FPmtYY83HPUQT85zGizt2NW+Rm0HfWWG7TneKN0bmg++btOI6eVZxpessVwUMY9kaemVPlsOqIQ7L4IV+02RFIeGs\\\\\\\/gSyBLEjxUaXBZq9vkQqK2CALgBebYPOkQ4Um5o7WlpqEicCEpR2f6KdNUJPFOyGuKMfmvFbWrlZkeOmWwPxt1Vz0eBmEjhGvaUivR+6qH4YZnmaR4y4dN9z4YAz8CLkqrt5sBKunPWCeJuWcmlzme9UtKeQSKOoMbEW8bzNBupTYCoeG\\\\\\\/HBYjXGHpCCi7IYZA93WzhJ5sLMc8gbI13RgXyt6KHo6hJhZbAfY3FD3TXYM4C75ImzuTOOfkMODZ1olmaNuL3K5k4mHKVX47uTjetGNe3RTSu8BiBRdMjcFaxcIDH5Hpxba54MM4xhCEmbD1DCYpiX5rYp4NDMZ0ew3s29xbXaCDc8T7w6R4cjrI\\\\\\\/3Oc8v7G1FWXAczpFZ0STtXD04qtgaAROsSQnt+XemvhaNOmG5PtseZLpB\\\\\\\/17OpAKlhRnq1Tn\\\\\\\/t+Hb6wvlFqJ3w3XyeJwYyoTT7Uq\\\\\\\/xeZCcOWgHATgyurHgxQkoyK3QXoKCHufaErUxgoqu9fyEryn12+SrFfwkJ5cUxFYIDOFxfyWlgkKholjLuCaM7KB1JBuHHRZez8iJRKfMXzQBqgzy9lMt\\\\\\\/LlwZQBqZa4Di6IPjKfJvcwP2tgFw4K1g+L3cEqp2RHpJTRuPznes75BfgUgJedAQg0I5ifjb510XsGa5Qasv48hriysmdj4FAmcFdJskl2+hEyKV+1ZsNhOMXxOPbWKcYUPnhNe\\\\\\\/JEUPdgbDDblCicZ3SgZ9pXyaM1Gn3wqMwF+annmKMRe9inPGmFfaJMrJgR\\\\\\\/lPFsJBcABJynWPe9nDfQ44R+wUeUW45CTVU95uDj701kRBLqe7CIUK28\\\\\\\/bLtsjl6TWfuVdeX8awB6P1ZbcA1SnccGkpb2DFMDlJHCI0G0RVQXvNB8LT7GXqYRv1w8rWzqOSyAluUN7EGWjqN6mKBBljXZcwe+LmPu8b+mgfuwZZ8iJg3QWqAfnOO68cElTacUhY5IMbAmpp+ZRpFEkF7eE68NLKis+IswaXMHky0FSoID+PVcuCucMjwkkzOzUvtmoCsmxy+H8buMRrQtpDI7Eqq4c3sNAkqz96uhv\\\\\\\/SgRCxINqKf5a2836WKKu0T\\\\\\\/aa09tSBwgerHz4G4CriXwJs0f1Ib0zVuXOpt\\\\\\\/wxeIG6pxX1jRj2cSaBoiN9UdqRyLor+Rwhn2Ys973T8te3UUnDNBWMilRAvYUXvvVbyGPQ9g4n4fusoFVS5Bv6eNGSV3IW03Wq+2bTxntjgJNGNbMsCK2PB\\\\\\\/g8l0uA9zlWaK46DmB8dgzQgHusCIMQ7fAAkWBlZVhwYaLcJo7pXY0fJiGn2qgghKvHjoXdtzrPW15woqVWDjNk88\\\\\\\/D8GFoByUezUATHn3d4QNhgEsfhMv3S8jSsx0NiZXxuvBh4MRiNRv5qTGbbo7PPQJM9zql314JcS4iqg\\\\\\\/tGTEYuKc\\\\\\\/AX5ZPUpNyRZfq6WlEPvVKi8kifdCfm5kbAMEM4ZKn8fUT0OemXPf7Dc84K9N3572uHwqabl5Ng8nCEX77v8Opn6sgEtILbbt7QMDDqJoHFjTQNRHS6x6qb2cz2xRZAsleu4odLP8MXdSi0hmb6pXZ36SsDzwXt1Rm2dEPjATDVYAUanrl3B\\\\\\\/u4Hvva5GcdmDpaWuCmpkQxjMmpYX1OdVX5a82jjt5kzuQyYHQHlyMx6GEc1r7Qjvz6z1gJDKI1lt2hK0htr+fyhV99UT6SGV57McABs9USNSHVOEP4dZ81vANTWC\\\\\\\/+tCi9bNaB6HCKqvYJBQ4ghwLlZ6glYHvNAv7atnRV7N36SGmh1+PhgcZSrimZnQ1skNkoMoxEug+Ri5LsbxcqtXUphUdurtZHDqFkX\\\\\\\/KEpgJaXmhe12CAivyfYinNt+nDxgS3rfo+q49V+u7c4Z27nPF\\\\\\\/kG3SRlnL7bWSQg1NarO2f8VqAte5hPoNOG1YVgs2VI53cxct06PX29pEFJj74Ea4SQ4sLr9dNJMQqK+Fh8xeJtUffVJl5y2pXB+2JL0MFeZFNRMmpNTRcARu6sYuN7u9QaeDDBchHJI5vtKVgTFCULs43MQME5n7v2FhM62zELe20Z76p7vERO+6ecUJMi2a\\\\\\\/JiWN1Eayscjdwxym\\\\\\\/c5lth+zt6I2SGnkyHi7SSlttg9GbxFJYGsiuJs6K8pJ7JF39LC+1+eKziKcQifdbEr3Wkrh7Gw8qqjP4+qrKR\\\\\\\/8OKN0oupunS6W\\\\\\\/ZQx6uxoAV53ALif2aqls\\\\\\\/RMlODVJ5YY6hjYF83LfIV4eet2UV1UdLvwFkjncx\\\\\\\/t9kAmfHE04DaZoziT04AGv+gSVIR2GHrRTgZT5\\\\\\\/OMnad6vjzdxPKM6ecZH6VaDoU3qtmDsG2XNyQYqGF9X78heRhSSpbG3kZAOxADIdvlDGc\\\\\\\/vDZxQj5881piUgZ0kfNkxD6lNriD7vAhqDGNarAxTMUOs3De8dfink+mUBFSYuW4\\\\\\\/0\\\\\\\/odhdr1mDWhX3VOSjwZpTrknba7+sKctcIFoH+Z6x24se42l\\\\\\\/jxEo6KT+NLvuyFtxbqYPVFfb5Wguj3RmPWJ057TViBXhcpC+xN2b2UoU0UXFETLYp3aBItDOJteBjpFJxwoQHqkdlFMx2C6XDuHTR3stximqqUR5WMyXJkG9D+lChxwqgImUtKNjFNslUffdVq3MBbW1iVO88xz07pL9\\\\\\\/yXqrTOtCOmOgTeqFFap9HzzHAgID9j5sPZYW+O\\\\\\\/t7JxcI1OYVr+07ViMNKrTeoWnE50pGuILiqjjv91Hr4tMtRR+lRUbGKK84cbmmw9iJV3ALrjmkC2EobSyKS4Sw9LH5L6VN1CyvZkoi3Cq\\\\\\\/69a3lmG8TpNY2DsmFrR8I62JKrgngp+H6czs2PF+IuX7+T3yWnvyXK5LFVB+dAE7MblIqgj0+6m11cSV7AuOVdt+J9KuyVf0KTJIHglJiKU2yUvV1rduJ9gmCo8jJjmBEwsXKYo9fRqx0moCgVGwBFsN7Z+IRp3HPcTRsjC9vy+KLIOUXFrW\\\\\\\/9TFqhiV+ptYuMiYzfawJFSkirZiHdiQUyHzFYgWcirCNEesxkITbmnF5X0YyDN63Q67tpwfZXLDkq\\\\\\\/XTUWQ9LvwXFOFxcko2rT9SIBH82ctJVmWmtA18Z0pNmEm5Iq0fzegeM5LK9QJvRNeD+8KpelUrRn4g8qWlm9+b0ZM0FdlUKnEJKlzYmGryZAfNgsUujfNEmrVwIJzKaV3MRfmekk0EbkDpyBcPPMXNdKoZ7grd8T8XZoDLASmOMnRr\\\\\\\/J9wuTKKvR54QamsZWAHsC\\\\\\\/uVB9\\\\\\\/xBbBCewDz6kBktBD2hipcs6xwqEgDoc3ndVMhinf1W8C9jjGgEE4X5c0c3UdJatTWogIhL4bEKmGVidAkAqOwyG0rMAKt4PxtlfZUjPO30nbduplbwETh0yzc\\\\\\\/6b+4+k\\\\\\\/d6iyndvUCCD6U6f9lOV3OFmxaxlBqWEJ6nqlGXE+qTEtdrGmPbtzHeQqUc2xgTNvS0HvyVKagu42mgFrXo0\\\\\\\/rV3EIwUVRBGrxw\\\\\\\/j1HzGb9Ib4kBJFJ9JEai5FQzkXY9eHZmBhHCpenv5j+V87TkjRPD1BCPHq\\\\\\\/kESY08L2rdKpfp1PFa8Mnxhzuy3HB9o+b0GD5c\\\\\\\/B13j6bUXEAwsqb3NSCMkQnc2+plLGbKMN5odxr+p61EQ7VoqRoJqsjW\\\\\\\/JrkkgLAuSJNCKQh8uFxXMP28omzFq3lBhAdKyVgVxrczKvdZsdhqJ7wHqwLWAS8AQVXuFhQIczOMof1WtTiBx2dg4O52wuhDY7v5YR1glxUa49ycTbfuDksFXMKfk+zUgr\\\\\\\/KX4SE3RW4bvoXBs4WT1Tfp6Td++JvaMZmM6IQak1flIpFRWKKeAgyRdLTLHEqvLdI+n3oqK++YsupPLkF4GcUXPBLhIrB85ZUw9m8jO7l\\\\\\\/PDX0z5zBzA0NX57hz33GdyVdP6dTifeEyvrTM9ZEKEG23GJ+sJ2auqNJKnDE6gTfLvlieLhk+qIwcC\\\\\\\/Fec5UKvHasXrxOwOaI0dr8L1PGni4tEG8pf7anDLifjE8dKDpYkIegfVgE8LCpo5ncIfCKMX7raPjQRm+L5Ah3vrFanUae0Bi+XLr7YdJ4NANXoSLozCIR4agoXOMRzgiydDmxeray0+0z0128C7Y94iWuVZSE+ehgUMtJPZJBEnDRuN62Jgc56hkcKu4iXcj5a5KrZb4GX+vuWtK\\\\\\\/+KR9Oc8UGUEyPnFNVGD948BMGR+P1MeJQBisqVGrM7kWFyUKjQ90Zc1W4wPGAzB9f3Wv6NQdIWipI0EUGbRbFvxUmdc8Mtxg6ShFz\\\\\\\/RBoWMv9ECBaVea+DyAxaQ1eCTJAFeNcDfZadeQxUhKuy2+o5oDS2lr09i0lWrJLVusgMU7GLmPY+cOwl4kjB0MgpnvGyywcGC+FBh38s1YY+6UrnaSelRFb+e0\\\\\\\/ZrR82yVzz1UY20RZQJrPqX181PItT9qThUD7N2caWAPgWVDUAxmwmf6xPD9Q1LF1w9Y8bDBW8KqF0sJ5L8tYc6SQ06\\\\\\\/\\\\\\\/fdtQDj16udqf+bVnTYxqfXOL7DnB3FEH9Cq9+1GC6+WzpBMqgT6Fi3CCXUMfKRKvxC3FIAzszZz8r6yYJt\\\\\\\/W\\\\\\\/68BRtQzNaD75dOLs5fN2XwkdF5iYv5V8g+NF7RzLNrlWkSvo98\\\\\\\/D+7w\\\\\\\/IxNVPjG6eXPlL9EfqPyT6CEU22l+A17hnQ8zuA0RxRCfeTA9wOa6VRU9017SAwrG+Or2fxcifGy\\\\\\\/sI\\\\\\\/Yie1luAg4YtwmTEDt8H1c7YHp1Ar3fEU2R5ZSAtvT9vnS0R\\\\\\\/2okgoC177sEBhD3lYIHPB7ZfbYoxLl5I1B6G4trmehfTC8CsE2euiqitTykWvjA4Tlzkdn+Dijovp9DrtaMixxJyPt3WTh9zNQT2ZNKGGsDCd\\\\\\\/\\\\\\\/yrfL\\\\\\\/AK0Cd7+ScoMSKzduVZ\\\\\\\/+PUdEmBQh7QI6EqpqegAtgDTjJCt929vowZuKupsDhrCmolEqEmmbxliwGJfRlVKl\\\\\\\/kPZMWYX3mvAmiKOkGCArImUcBqEqcWSNsQFqXm3i2KvlzeLowuxT22VSeKSRlVEvScs1eCdW+p6rDbVGmyBaaq3w+s+bkdPj8gtNmgeG14\\\\\\\/k1h7K3E7Cv+QoPEd8jRUn9QQl7Bh+MCyfjdqlOIQH9JlJf0w2Bl1tmqLZcSaegOYN7PIB7HFrpD3Goe9DsEpV0ORLNZgTYXK5OuDBcNatw6kXGAY0Iif1T8Xibbu6zqw58tOKT6T0tud3+cKs9J3qn\\\\\\\/r3HqM1FIixg6EkIGtF4kHLrSNhjJrtZlnbUtAl1IHWG7cWdE15z2TR7edhxOqBoRf3kNsqFO52orOT7LSiiW+NWJr9RnFI1y6epimwOqtFlf\\\\\\\/IyMeIYsr88Lqe2htSq98tRbIyLVB6L0g2XwpubvP+ce4Hkv\\\\\\\/8GXP1MpuZajcAcVrIGGs7lUV+aELGXuN7xrOuBzbyPxlvAGkPdYGb8Hz2rnKeHjm4kvh22P\\\\\\\/Xzwcxe0RDwv+BpVhAerHPj\\\\\\\/ua8BSNINC6WD28ojxeMTrveK2373Fjt5khZENRWhdpN18XGacwRTsEqs1jvQZfc+HfgXmmIAoh7MUIN0GBSFEXdUlOlEKrIvCY+zo1psjoseQOJB\\\\\\\/k+\\\\\\\/Kl0k48eKxYbHhjTSIGOOGR0pi6iLI6IL54KydUAw3xT0fsRfIeBosPC9KROMnBw7LiXFnOHAv+1nKzKm2YMZCJpkGwb8WYJ06YAfnZI\\\\\\\/MJfILOH83ON8yX93MEdrwagNLYDyo5D8\\\\\\\/MFefiWuqvR9FQYmLG012Qn8riyjm5+JoxBrhWxxL7KfwYH+ggG6MA4rk8f6M2L3f\\\\\\\/w4d0CYFWNaIlEQvaNgEc\\\\\\\/7myp2U4AS8biDLR4nBr5k6r0BX5GVvu2jv0Ac5Ws\\\\\\\/bEA5hQdxCtRGVGeDGn6Du4RjqZ4R8+13cSwUniKmsrJkW8\\\\\\\/m3rNuw6XnEYl\\\\\\\/4RdcjyV2RoiPgQ+8qyg9nXs8OApMOC7Euw6zsu3ecn4gjF6z9TAWn1UPL11zNqjXEQLMg5sgyuglT+Dyh9AcPOAnp0argZ7CN4jmkUYZyxh2oYMYNMBpe5Ye+t4R6J6F+ze4z267swuF85O1BJj\\\\\\\/cVHeeetTJHM2m6\\\\\\\/0Ag5IF80Mg8tJNgKN37cNKwXCyv4\\\\\\\/yJ0DBiWbeQbPIk+Iox5+Ym7AHwU8\\\\\\\/\\\\\\\/2tX8vsg4oSBRtZaOyYwXBVXbKz1TWSUqadUw+\\\\\\\/r8as7uJ7GOQSfjvuQLRZyS0YRwsdMendlMnpMAhHn8FloZ2KfKRHHRGKeEjyWm\\\\\\\/+OqhDZKBtwlOxB3L2kTCdiZhXWJx+gsfusXZiGl\\\\\\\/rT2k5\\\\\\\/zevOeEKV+Jg6BoMlGb6RBoSdZoFw4O11+e8CPcBAg\\\\\\\/cJ59ov6jiVbpszKzCbRXrsFb1FLTGHJ0WdTnofv+AJQaenpDekwvgK4cY2Xz2Uq\\\\\\\/4bw4HAO4kpMbE48U1JRJWeByoe2oML\\\\\\\/J\\\\\\\/KXkb8DMmlCBDcpEI+\\\\\\\/1trc10DKBCnan50+DRgxn3NL9KSfi6qJ6tLkguiqt79urJ3EeTO93l1WuOKwpingyBIex4SacaR4haI+YvfFDMUkwMJIGJQmG+s+54IcSg4rWtskIerL6koVAFR6kKJT3wPcOMVsG7QVrlCz01PumrwFdfoiGi74PfWyCRhGAqxeFFG6odbapuQ+2\\\\\\\/CyCrQVLJxZoMdRM\\\\\\\/xgOGfFXYE1oCNLrC\\\\\\\/QvQw03O63L2hcCqpttZ\\\\\\\/acHLGTFww5t2pP0wYVSrxhM38ZJEHLSCWTRz2JBawUtZbZFltrrMmZkH4fQutJ0LwQnHY6P6EufVFYTEiGuqPGs30iMcqJqshlKyd3+izMANjfFqZtcwto176TtriJRKcAbAXlQAh4HD3awJr8GWP2v+ar5mvajR\\\\\\\/tHO+0Tdo3m43L8pL+BlXdQMbKIMLbPod\\\\\\\/KLvkKxKMMJIGcVV68dAAK6adATaErcqzA+KjEs066hEc2TP1MsKXwo4p6p1ePv+23F46NcRQ0IbhdW9ZDmRrEBpKLgDl9iuRQjdkdoc24gQnoHrOrS4vnnEPK\\\\\\\/UU+kDzB+rqCf5\\\\\\\/ug5Cr+SwzcAGacynAi0nWHyFT+gzTdW0potu2LAIbSL4MOvMffkdalRf71bPcEe7dHvFAAj25WpmcLGjuWBiQub9Oij3Tcj5Ci8TsEnVQRjTptQrLU+JoRmQkh1gEjM+XBoipNBMVZ4rY7TriIvW3DgSWLXBYLuOJbLEAqnWz6mlaAoBkXWwv2V5DNJutEaL8TdEs2AX4KvWDT4ikwkPsTqCYV7LeeeFGnEBCQYNBYQ8m1CwzxjMqsK9G98PEyUR5r7rqwbXKh6en1\\\\\\\/kAIftwQCPJyqEndDforQ4jwgP3ELx3xIKiicnGgyu8HtGXT07Bvw+TvYwg26n1aJtcNv1YKcgjdVKaQgITExhcuP8sFBIpB7OGHHlTQTf\\\\\\\/3CjJJo3U0QVz+jtQEUhmScU+5eol69u46UV9fjDTs8IO3MdEtjY45m8fEzQBiGRPNfjgza5FLlqne87a71CQHt0a4734UI\\\\\\\/rNvNUR5nEiL0CnNEGFFNySK7ynDZHoXLgFYZf1jFMOphLU\\\\\\\/SQ\\\\\\\/Acss4trJW\\\\\\\/Mry8d0hyFTDAEwRvcVcrlfVG27TvQ7S678iDIwFLZ36n8mAxPLpnW84A6m6h0hun0FZ2KKvTTnmmxw8Zsfn3zUrbFZknEYaOgBkSBkIo3PySeg9ZJMRPIEUpAoTBxS54u+2iONoEz\\\\\\\/0heXhPB0OAg7IE68g0jUw60hQmrWj22L9geVmUu1YAi7ZpUE45ekF8GCn1M+MdjTiFR1Qk4Q2Fd8uJR\\\\\\\/pQqDg4J0gWkupxiL1eigG64QDrxhytE62Mqib4OwdJxL6XXeoqFrgeDNHqbDlrI3j1tqR22TJVFeydPUlrtsN2zqJGcuRLwPmYJbb6VMaja\\\\\\\/ELpNvVG1skDyx3nhDcLgxeST7KnDML1M6aTwkWh1ps5DBz4c4cT7lINk3BNxHaYttGbWBQT83+m4wVTz0Yjr9GuVyX85lbdX5Kr2QSXiQbp1EwDEoXuVRwP1IBPmnJtFZuKLwk2EEiToI6dhfukeWcXPQG2fzgLWyIVLe7zGUKxZm4KzqmB2saFka0a\\\\\\\/nMoR+UAopjJOiqbOchiVYbGeVDRZBu\\\\\\\/HevIEWY1VIvjKij422ClZCl5halkUMcURzHmBraOtKynI6GBazgG082SD25QZTLwW+uqZo51OvX3KiAl9XLROqn9OU8uIXHwAeOm+Hweo2LQt0iNX5Kz+afn4Jc61KvV6G1sLkjT15L5c3lJJsapu6rypxYkZ6FRlfXd\\\\\\\/pvmavz2wygv0nyYHN1477vYuP70xF+DAnTNLwqQsyVpdIdn9F+av\\\\\\\/cgpPElJ2nDcyTYzN1ztGb6+pC65oKhARYAgokAd4hHnLJIXWkF72iYZihVnTHT1OENMSOHhtSwW3f2w6HYlHcyJ3cqtSPpTYM0E3TbvbbpcmomtvX5XMKky9uZBlTSsuYfZINcBPj35Dk6KyDMwi1YRT9zTPSl9CcjrKeTku\\\\\\\/\\\\\\\/V72+hzBWf61MuGPSfFVNr38hMG9cwA7tdpWFqRe6EOhesgLbNzZs0vtYCEIUszpUGubkksY7YeLJdiE5Cy0dYmaEWRlREcQlPfnR\\\\\\\/QqQypNNBYVz5vPFHTpq6I\\\\\\\/MqfKfveEQ8xfgGBglOJK9suYLmmSH7dPVVlmeFIDrRBMDoPD3K89bkeBM\\\\\\\/QiCAJAlou+OALKeh\\\\\\\/keHlpS8yiMBfsohNozlzJ+5j5PoSYRzEpDxj7bxH0XanTvSRzrpzndthIO4qJHRz3gZBYTEafnWwWtwIbc0\\\\\\\/KQrAtjnCeAakL29\\\\\\\/8mBd+ygQxYO6aQ7jyUPKe\\\\\\\/z6fnOm4FlGtu1tRDcEJ831XAdcwx2GgxCZw7esiSgHEO+y+cqdqkNE7vB0zSaXqJLAuK4ZQsEE4IbcK6e5h0fRVd0tgMtUThvHWhcqvCGEB1xuONCF4HGEldvSvcCUcoa3CWDQpVlgWtOJeWa0ciYjqNoU5bqRlBhd1veszlyFizyONDLhFIWygmvlS7mwcDPoUSYxginmPEsTvLrQqFgElu2uIw7pp6Q6\\\\\\\/JK3h6Key1\\\\\\\/jBF\\\\\\\/jjSiP2fu3sJ\\\\\\\/6xgf7vEU76webWKvzAjACQ6b4gMDSkvpRtJEhcw4zG8TUbILHSdhE+xyz7dWcEQkVTF2brzHAcM1ZBzv3H7uax1J4Q2MeIdHDQjUgbzDjVPAY1Wmv6Z\\\\\\\/9MfVAGKJua8pJdMPlD+IiUIYa4umrQJyWQGPZ2ZQSQdtIiqgIzfPj1bKDz7Wb\\\\\\\/\\\\\\\/95eO+QyzewS0J4aZrMfGdaSvo\\\\\\\/9JfL5ONI7RHu\\\\\\\/WL6GWi0+67v8KyoT9JaMayvq12iH7Ds24sMm7L+5yeeiD8+67YdG+wQiP5yYvLxztM0hy9m6nuwoYnxJdskpgIcwFyGDUhqB3J7yBKdtcHEgjzwSxAoKWxvpzeZ\\\\\\\/GNF\\\\\\\/ovzslne6LryXlnTchE3i539sbbEXI7AvBVaGg+HWA2nsFAKm5gJhfYOufJnFRzntPz40QQ+\\\\\\\/epsedCIqH1ZX1pItT7C2kzWT40\\\\\\\/Bi3mpyypy91oeiz0s5IR+pRsF+YfWwiJ0y94ozwl0r6twNbU3XCiRZ7CYry2GR9DO\\\\\\\/R4+k2pO9Ayjf1O2dc9Agyol+aMOVC\\\\\\\/YDgPU+9hPi11dfVu6PULD1g8Qk0+tFgIetqKPCn0tRU4qdmqFIuTy\\\\\\\/fskSUel0lkI4RlkLnKTlVx1nk67\\\\\\\/6hUtlXdfla3HMkIsohEauU6hCeuApGoqkYHXO3FYpqe6RjRoEOo9HRDFEmdwfl9n7Dax2F5miuTgic7SukW3yb2WACwYODev8m+ZXjMaJ7gpXqRpDzoxAK1xWcJ543ddQx9A8SRq1ZSFODBzITn62jUlIBEtydmhLap0qYjotoMAzZ6DpeXTkwa+nyIr8QgBl3ONoZVqngpoAGDoimfshcjIdYufO1nPlIKGl+fEyUNl+xQ+eKkgORfcEQfJ1\\\\\\\/WtrkUKktJJJ0YNnQCrCujmqFu3LCm4QqdtEKYuJHvN3u\\\\\\\/d7KS5Iwpb19461C7CvA7wMzXHHnVxwHf4ucDktQidDOMwlQ0chQsGajitT0rwN5aDoHsGYNRF5ziF2MaGUHqII7exbvBe0W4X9NYCfNWqpmksG70ryNMBRfNEfTfjUNyWNU6cO2BFoBlB2I0+EgQ8arp\\\\\\\/C6pOqN\\\\\\\/6i+MQWc7AnUpu4WzTbGp83pKxbgyf3OiVmI+\\\\\\\/a3VgvEs+5i6nbQDSVsktNAz8nK3rzMQQpGlyxCrAhdyC\\\\\\\/LPOX+eSW6Jt\\\\\\\/YR3pqo9m99uMS7uDfj6Aw39VQuJnAL1MLL2Rz5Odp64PXOtN15fCYMCkDzCnUVUhHaEgIYRAl3h0ikPYIc1f5uySbZzmQJkTWM2\\\\\\\/H\\\\\\\/Y9mFIKtpAEqymsFYJGRKENg\\\\\\\/MUx9MhwKVNJhBi1UNYTuCwCA+64p0BC\\\\\\\/S7Ave7Df9IT+OjCAtSIx8GFUfgykUvuBCWzlaMziwgq0IEwxW6WvmtCx4m+rNo7v31IQZ9e8r1QlPiHr9XkEsb0zO7aK0Xg23HZ97XoTPi6tkTwkpjocqPMrMP5fu7G9QOzWrwbXfcNEBXv8RU68HhHyNl0pUuyb8gcRP75f7JQCzA+t0INwQdcnY1gLGjJLoR1aridw0nbl1xJYH31FiTmtI62dT4802168GnegbOz1V10QlpYrVzLE6+vsAVMrLcwt4ACl29pqRtE3UC65i8yCTfEyBnXjOBpt7XRaKVxXp2eAMmw3Ew\\\\\\\/S2JrRY24pTn7YxECYDbQS0GaU9yZYOl24sXcyCszDfYpX3a0922KyPsnHE4jkBpuS8wwnHJhD0pcA4xggNH9drb6NN+Ao0Gz1Ynjm7NdGQEsFmzWJLx3JLi+OnCE3TK5RDeaJPSnyxR14OPw2ige892Va7jLTbpE3HYQnJOfZGn1mB6tw6EHchqCw6kSB08sehOFdjbuuhwjXBtUyMJVGK4+SIWW1PhqItHIGM2F2dcyl1ou3ccueqcQxFf5+lq4a+rcZaZMPVjYXnvQlYSXNYlmWjOUI3QF66iS\\\\\\\/qKRPItfOiFOIUwHMr6aVBIiz\\\\\\\/hXNtbMsxyX+0bwTyvfUpKBXpjdYd3C8chD90XOJCG7D340ye2ZOb\\\\\\\/2+TInIDsJ0VgwlBejSApRWmteB0x+tIhf9OL6oK+5tpZMF3mnHY0rbUAz77Wkv9OKIeEM4nU2XHj64s3sozCGqzABFuuu28edUPL9ikMWY2by+mHmVy\\\\\\\/Bl01ZuxHv3DG6KhWShxn+dnlfcy1fl+0RDRBAVFeE\\\\\\\/d2vAM2OiAOjYVV0Mu3zCoTfGRji\\\\\\\/2bTeksETbLLmxuMAuD+8S5LdOw0bssUJ4qy\\\\\\\/Z8K9FwZJj1XJgKUke+8kwswTPqNyzeiP26YqUGoE+lqK8UwD97k6RLu2HyUV2wmfW3iPxQ0uK5M1BYqb\\\\\\\/C0cgxveBJQg5CK8ZU34uUExu4nPJD8s3jaBvmVo17VyjcTaMpK\\\\\\\/4BAWN3OWdof6vT+ThcuTdPbpiv3fQAu6frBJo+KBXfyAx5mGwF7ojhJO4SL25rpFczyhrqHitlEa7\\\\\\\/Bx9z7fNYIrD13H1Ay55omu+TIzYTaCcZZHIJbFTDw\\\\\\\/LbkFA6zx17pXlIgiYJ9zGm8IvJUx825VlizmPTp9Bg7QHUaiLjs8DMo2vO1UguMjJAdBhzZ9kjCYjK6G5gOFB1o\\\\\\\/xpMTSenfC997ZiF99Tp9bKwdNaHaGvD39Y2MGPqqOxklCSqZ37A30lgsefra0FLR7qQGZtTLN82wqUzhVMpMxe3BfQMfdqVzSA7QG6jdeT0NwG7udeMnkjQKnD4lOtFQSawQkcAFm\\\\\\\/t\\\\\\\/ZkAYYicVXXiw5ucW0sHo7TwLLT0UHjRPygR8RHoExpZcdYK6KJNFbG6Ty2kqcFPzrg0bz+Ibz0EivquAtJW4U3+6iHfJCjkuwcBEU7aVANIS4e5+o1TlaWPtNlc4A6dtJgB\\\\\\\/+eEhjMgmRDKcxm9ZkhnqudGniCA7\\\\\\\/f3Ldyst2NkrJ6yqbgkR8+Iu9PPN+BOBZFhMzptkDlnMjA7QAotVZJLA2TKR5UzgjWAkHplQhQ8mQdV2A+PMEAtQWuKAdEBbzkJQgU30MLkoV\\\\\\\/Ut4h3uXqRMf+MEIKLuD+Qow6VzOpcG30UzJwb6Lhl\\\\\\\/LIBep2ofFY9zPYngHgwTxyDeVDafBtZ\\\\\\\/dMvPkExds3EHihSJxw09NJRywpvLGY0xYhZ6hOd91lT0XWTy\\\\\\\/l5RFEPzq6dVjeRfZgNlttv9aQtZk8WJSnAkkz+Yk6p5wNVQhiMkkUxHEFUxYzOXSbfZtq+uT3z4s\\\\\\\/DgQW2+wwVz53yIv8rMQKcDbdOnLWyQhfIYZl84Stto+4ztw4v+VPLQkLWfdj5URdoyo8K2Wgk\\\\\\\/cGldzzTJWJl5I\\\\\\\/hjmc+REJ3gd6dC\\\\\\\/vo267sjEefqnjoj1bwJqouTARXvJi0AB2dVCYGNlIP4MnikJ1hVzagIh0hhuz2BoMM0voD5s9lcb7lnFK2eReQRnQV8cWIclbRsk9854mVDbdoiQ2zUPgkIXu3WGnqe2wZHGBcnaaHbTpmp7g8ZmqyTzIPtTkgC7aNV7ud1y2Nhqgl\\\\\\\/KdGkSscAfY\\\\\\\/y12qttoPgwOx3jGnnQx2+UMnDNiHHOOlXxkHY3yTeTtlmFeuNnxaUAtX2xpQTVY4tV4Bh4LU3LggjP4t7rN8rUXfLjRKCVcTIG2dJaSyJdBej05p9XtjLSn1PesEse85QBSpKyejRMQ0a\\\\\\\/7pJj+qFwLYUCsmE7jUTqU8m5VqSgUKUS\\\\\\\/c8hxddR+Esf9sluaXcs5vRHHmGJCa\\\\\\\/0qgEVIsfwTjh4ixKqdh500eO\\\\\\\/+N49bO\\\\\\\/WqdIw+jLaPPBTJzJ\\\\\\\/IGFSy2EpP8aBgUGH8\\\\\\\/RnXR+TRTO4SXmWicsbas9NpeGFEnmECEM6DawLtZ4jKijWIETg9u7\\\\\\\/Fz27Q4xnTeBcQyQCs1U4KoyyN6Ey28L0u0S4s+tXgHhmAoP9vkVmeIUCwYnQP1GmlZJMaRq8lMVVHngkI\\\\\\\/e1NWyqXiGHRuSEKFf6jagT7Oq3feMR2iXRIAYXjuFoh8m1vXKYsgqWnwksp5HsUneDUb9VmTNkJdhCTqMKXYSiQYouyUcaEwDuBmy39iWd5GxRlGmseHgBdMb1UDE\\\\\\\/wIJyJAKgXkgSevUtKzFMNJyJ+egDuvO1BaoIcqczs5CcYcFPN6U4aleZz1Xs+kykn18pi1lDXMa9MtlHPuP2YgyhOiERlTQOYucgY6Nbir6O5za\\\\\\\/AA850JDpuJwdGDzbpa9f02EFEQm5HVs88fpvw33kVMGsc6ChyIYdVAvX5fnK8OPLVX+iYnY8B0QFzA4+qX2v5SfC0JNxCxyspjkqNCTSwDHgBrxxEVDsas0Qf6CRz8DCfktInKpxkYMUCGugAhOHJYyEqZ+OenxXy6CAoeVoYw82hL1QVshhPXAmnwGUnEVQLkPkxz441lvKHD5uAkbNHhkY9oDztxgOiP3prkYlb4pogVjBruIfY\\\\\\\/P6+OTY4\\\\\\\/1uoEo2mvXiREFAT\\\\\\\/wfyGl5ijTwrvgpLCCGCL1DkZrwSsUfA03Jd\\\\\\\/4GXqEG4TNX\\\\\\\/qrKp7xesDpamI5nDzE+XSgSNwQXMPSCLVluX0YdGZxLyH8cIIWnDZdD89jLGYGBwkW1sfppt1vnJRAKU0Bip\\\\\\\/gyABhNFX7EhIQNaNTvnRhc\\\\\\\/pGO\\\\\\\/lpR\\\\\\\/DVGkO474Mlv44VBD3U2ms6orgM1vBnsti6muhM5gA1zc2KP98WeM1MXq9FTPx9WrdjOFvgGbY8veaq1FC8E\\\\\\\/R\\\\\\\/RrIFJNsF\\\\\\\/UUNIa3ZcbdbG0lOxA0fBLrSteqT3wx4vcTIj\\\\\\\/HpRl0CnOQ1CGP+rOOiGRiq1tekPvzV4PALJ0jgGmoloFfshC643X1iXh+AVYioklsRN\\\\\\\/DidRCFxCsfb+OIjJee+\\\\\\\/DEKEIF3OODo1UX40dF6rTgDc9VGWkEElXyRmqlVidSp5YErftWMlrmZTwlEjHCqXBgyeZvHEhDQGGYax48v6qGucUMXpcCOi61v5qxgXW6LJNNIS0cTMkVVzPXR8zhqT2MaVsfCp3Rbx7vHkjquMTUClNAgGHreFVxFlvffyviudTvKzqfZdxBEQ8d5nSPVdWw+rkN5wQnwmLvxjAd6FHKG\\\\\\\/ArJzoIStLM+Vpa5PE9xRiAQAEbU5mRD3+0xUsbrL3QeAqcTpHnLoMjB3OJCtiPJPgJ5HtVJ0M4HqYoZOVB8bFr8jOFfb2dZQ9gBq\\\\\\\/g\\\\\\\/Y+Al+tf2ZHlFVXbyONpTohlyHNcWx0K021qjPkwHw6ugrPsRIytmDv+UUVy9XiikYZ4wtZ6wiIMUlKgb\\\\\\\/rKo3cAnstA0o77O5Sgk4tMPzxV6RvpCW4PQ7qApf3rLWmGQn3ahyTsC5NXBUAT1lBGggdafY2DpKWQDVmk0tJMcSvwdbIyXHlJsccN8bwjTze9mU6DIEAcuK7nPeVSFr84B2fvfZOlhYOywKmT5HhpB4zmnx1hegzQ8soFGU67T5gFYmnh6g5\\\\\\\/y0CQ5tNslwEl7Ncs1JDxBy5+3rYLgYdxoXyde1k0JdGjLqHNk5qimW4ubCYu3VItQ4\\\\\\\/Hczlo44bzXigMdVJgesYOxTijbRUQybnuqui3OpZLgW3c6NfVBtJbjZiDAJLX5Dp\\\\\\\/CHNG+kEv6oWJYxTlG0Jv3sjtQEV3lAfSgLq15v7r7hVINoWx7SDmw8+20nmdgr74OzQm2KYforMEgh+uebBpv7qaa0um2B4DC2XH\\\\\\\/m6rWVHMBFT7rit96\\\\\\\/x0d4HxVuo569+jW665PnKXrvCBJlhVpkxpdcFpgqQzpUuPx2vnWjir4rEmhEoXt3Umm9TudRqCfH2QAfuVLh8syx6MVyaNSIW2py9ErgvS3pIIWoxGQDN0qxkoY\\\\\\\/fn8YN\\\\\\\/qyrzM069R+7OELrMjIqG1FFTAddlr4GLIa+ZnBxaGXiRKPBDJ1f1cXmxpoDQk27bmv6HyGY\\\\\\\/8GtWkLAFTT42jVuFWrlcOvVt+2wNAfr7U12cR0huGTmIs7WBp0zRfzkztEW+2m9vf6PDDQHAwMXh4x74AS2OecN6xkmZJY\\\\\\\/0BGFcxxEFTxl1Gj\\\\\\\/x2kAYLXCENDPMS0S6d+J2YBa8cdaMB9I4hs4vCwTzrj8iyW7v93FDeCCTeb7pEvBbKtv4EhPTesR6pe\\\\\\\/fJCtX9DIjZbvNx4+00JPTOZJLcEHziV0h07OyUR0plC4NRmLvajZ0c438gESI+NgtKMvcnaowp9Vlo\\\\\\\/zQ+PPVNKUF\\\\\\\/EeW5tFSR1GO2Con5pY+3lmpLXMaiJflNobyNBk4TG6qfBrN9\\\\\\\/pPJxsOoFe96owDE9Vk5p\\\\\\\/4jGACHa3pld+mfXgUiwj935OWFmozyGNDAlkEhyen4YUvFMsl48hniUjjG7SgQ19fNH6e7lhACJznSphRsaFW78KIxmvllpm+DgINr0YWOS5YpRDFLfwVwvPMJ8DibVcobLiLJ3XSCT6oobIGR\\\\\\\/IEKt8+6IMbcXE4FDGni3mXC55VCMl4hybU6vSlOvcuUe363oiS2a39X1bOl55NZ2h573mViM6O8q4kLxhA+AvUh6nCFQj6Zcu7bSBKfr4dU6O677FiV7X76nax88RGf1rTAuD60C8iWb4ZinhsAQddNeM0gvwqspfwZnUVsqEyAMvxdR8hzEdiu0suG5m6drqa7Q6GqgOsMQoA9JsUtsMzMjavoVjg+7yb3rLO\\\\\\\/MsNhsvaRRagXIN9NY2PXN\\\\\\\/tFcQG\\\\\\\/Bjhvs9Y1cs4synapZBtItVFZ5F7UvezsVnflS1zmWc452SyYTZ+iXff574\\\\\\\/oYVi4fYHW\\\\\\\/deh3LNTIU25tcR0a\\\\\\\/AQ\\\\\\\/nV0+7wHDsFPN9c8t6yRxJslna9xjUsd9XU0ZknRVMDBJ9lvsBagc7Pp+8+S7HJE3EVyrr3unavPwJ6SV8wgLwqu7m4Er\\\\\\\/+\\\\\\\/e8YXkQEay\\\\\\\/XqAp5F0txd4oKiIDSvAJ01O4+GJ2pWCbdvhdX3juzrJcLjtGC65bSMkf4tp47kMKOcbfKu0rdKMs2zgOxJU7OjXF9m3zbicEkNBrl3oWVdC0md8SO9l+584jdGdqsdU94xdHkKSfP+qjyBRJPyjrbVYhcidxahbLIF1ukk6O2rgQ1ELlmvyEXFkzDLMUXjLsiUTRP1C\\\\\\\/LgNezrQsnY5lVqpMFeC60xBl7EOlZ5Go1qE273p5b6v9\\\\\\\/n0KRpLAbF4H7X4OPUfju73EZO6XmfAsUjcicU5K9h4U6IV+NqKTfyUod3iBaAtY+qNHrzobkJiOyLnFHvY0846NqAY8uLl87X1nxBwH65Z0AnBv7NApCSAFD72ReWQVZwTdAwEhnAeVv+YEaqgPSJGpoj82S0+oW8X2UooSqBdRsNLDHeo63sxsODzKTD\\\\\\\/GE6yCHskuNMtNfSP3jA64a4piks9aAd8tQhCl9uZxAfREbtlJisrjxBmtvi7ekv\\\\\\\/gKiPXjd86rUUpuINVAm\\\\\\\/SHRXp0zoD7Y9UVsLCjU+zkzTazPPI75aPsCdnKaDCerJTP1HWrp4VFR5oV4g5tCfOb7hQEtt\\\\\\\/AHVYNNHRUKAPjJOqp0gtVg3UzJh7W0TYGJ\\\\\\\/TDYR1cflCryFosn0OWcHm\\\\\\\/BMEW7xXvSydDjN7zzv9Q1FkDLBJ6RsHRy603DrDAASHsZM\\\\\\\/5AnVNKxoOWiQYt34MDFAOdCuQA1U97UGwoDBlhEC9uZqLltT7Rpy5LHfP5YOgGqfA746bwqN2mAmH6St+2zDpn5OTaWobi98Az8tGnww6Nux5GaTNJgvnGf1Rg0IoocXMXevzrcV9GFTyoL7hVXvzWxKXn710WYrMGRrSH+dXhdJ+QkgSVetnGinbl9WOkCcsYwd\\\\\\\/XujYRdWfYJVu+82wOXx6IoPOx4\\\\\\\/dPZWN0aMnA5yOmjkRslKL\\\\\\\/I4V68w75JEHN\\\\\\\/WWXqjVkuh6wmVuk5BRwhqNsdjT0brQ2Jg34HNqEA7avawTCvRGVqYSS+IfpE0ueR2b9FcXTMNgmPqc4chN2gHAMsZ4B+wtPLxnug7xn68nJLQwHURA+VKR0wXyfTrUkKy6ZfH52rG0x83Tg\\\\\\\/KL1B0DF8FU+uAMki6IzOrTBQKMCnlMXeRqb5SjGunPZr1Ir2O\\\\\\\/GRlND1IXomuJIf74IhdtI1PkGJUosYHtrmDnYRtymm9mo62\\\\\\\/PruoxNjKsQFjJcqCRBUJiExz7qHXLUP2S2OnLVJJ0HkAH1x8tMCWinPH4i1OlgRZMX0PL22VlOlGGaaREs9aGk4g8yi+UkLqBwVwXd\\\\\\\/1xS5pBw3ASTR5fQqYBG9xdLmxrcTYYsFtl1t90TtbXlr\\\\\\\/P2Mtez48L8Igq02KE77LQFX+upYnzBFCtIenjvFaOeGvRCcVz0yzBnoLZ79N4zoHtpuaIL4Rf2nEcQmhYZtddrOxoni2\\\\\\\/2USYmYhjka6P1+6t8YDk2RFsEja9fqFYfYlsCN\\\\\\\/qdyvh93AF3q6xC7vJ4Gi1XX08dWKBCLH9UU+16L+LE0Z5NWdu0VcPIifRiw23Vj\\\\\\\/9aN7F0p6RHPCDlQuUf4MetlenE\\\\\\\/kAXTfNxG5X80gmZGicXyeg\\\\\\\/ON\\\\\\\/RMCw3SAdVN27bkX16tmuGpfPkq7Mmh+Xiggi5pLpk1Zqn+6CbcsWydz07OtiQiG9PxjA9AVmsz9fKKzJdD1++ZTgV8uWjlquQ1tcDgYJDTL4cNAJ3k2r4EjpNJ9H+sB45NypCHgSWQ+acmmCptZKzmMtZSvFaZPinz4xi\\\\\\\/4wRE3Y\\\\\\\/wV0ClsNxc\\\\\\\/diTZHtrJxz+y21CEn0ivpCyqwTfBcgEDYfCBL3V4sdb9kkRM+lyJ5ik+V\\\\\\\/l5mcDExFk6w8nie22ngrQBAAYcLQu9edEMnvbuGzqM88fyvINQtHsz5owi0\\\\\\\/soHwKiC0QfAtIC0eWOxbRsWLxnDjN25k1Kb2qo1sWytkpBnEYITMIynitNm0w0uflyrus7gAOCSOaLmZKddUzyYct4lYmXVxxH+AyXUx0WMkayXGtUSK4dr20nLjt\\\\\\\/opsg8\\\\\\\/zqIpYgoNXdBEnr4kvYKQOMq5hThrRcK\\\\\\\/kr5Ywir1AliIvT3YGLrtU9fYH5X1p59wfw9eFtGTGEIDFIa8Wo99Z6y2VWm072XVdibAjcwZy7yodaxFMo7vBi+nFmRg\\\\\\\/R5dFipUX6tgpuHYy6hVnnNC2eN6Vae9cAsf9k6Do\\\\\\\/BcbwRjYCqtb5Dyz2ESSsAq\\\\\\\/xDWTbeH46XIcFquAHoFq7dIMmZbLjTocS4DEYvYsJGuNw7AC6MdCp3+LsUWPlgL\\\\\\\/y8LU9WOCtpdj\\\\\\\/TODKMcGGIfPaFtas\\\\\\\/UkEmN2CkIHD5aByReiWvdmRnX5zl10V\\\\\\\/tciTAnlY5Dc9LgWPb\\\\\\\/D53IIv\\\\\\\/WIjk0nE163BB5SWsgZNQ4SXlhGkkTPP29vyEQe4WkvCbRBRNaS7Qzw0SBPvyvWKTbCi4W1OlFcR4rWeCVLHewSQ1s7Mhn9ofaCw\\\\\\\/\\\\\\\/A1YHN+dCxijj5opoScTfvJ\\\\\\\/oMG1PHMRaC3sS1qvQoKuupDgGjU41GSX1hvnXPcar+GRlDhISiczU52AzxdADmDsBG4+LxssIqJi7nBK5YrwDTBaw+GFa60XKjKZMbYAeRucsm+HFupVM+GT5nAc+K33o81Sg88D7UiHOnekI6hvmkqKrUfFcs0XzbocJA3poROJctG4+5pJjdVaTVT4KEtT6ArxJqfdNLjWQiOtS7br+xeKIAN0SGuZmlhXEzrw4hVFCYOUPNQ8Kv1cWfdFGwJ55zPsR6sDHf6iUm6VJWn0zVv6EZ3QtsWH2mhnio5Fp32\\\\\\\/19TvAyTT9gRrqvt8t0k0YCvifv3urudb3yBMN1GyOswpu7EIMiNV0OkDeTgBPR43od\\\\\\\/BATCTMfpXoeGiO1azGOokJP8eKygiWmqV3VmOxY+MHSNQkiTp9+XavDKXP7FTInoS4keRJg6BtarXp9fyLgedp84RfS3sXMQVPvlpvLimnV7zDz92JIhP96hzvK1H5w13pHJBh4CowrHJtqWYFZIqdhhuV0bXQB8mp+cb07d9PmthS3Bs6CeCW+dKcY5lxZ850chvm92AunkW7+xLMYUKFeqc82eTe+\\\\\\\/R9ZTw7a2Lvi9VLwb9p9dr+JaWatV94yh3lbPQbdb5osdzheWPnpHMjXaAzuqTEzqlHPtAuypz6NCAsv+iDHn2xTbFZMRIhWearHRtIfm894j9Anen9WPBXfJwNt\\\\\\\/+00taOQnAgMy\\\\\\\/MIzHW1DOUPe7IRKrrCBZxgJQ6i8aOcwtE7c2yd1\\\\\\\/zu9lIaDdsKFLuh5NTwv7JOn6F\\\\\\\/hGD1lKDrF49Jig+iVeZHPTbsinF+RE8q9SULPAkv6EgH8zqbVLxZHtOc9D1CqYwZnZm14WO7gVjSIwqPN\\\\\\\/Fjz2E+AoGc8L57sfWKTdbe\\\\\\\/ABdCGfs29rE000V6smYc04rjcNWbDaQbSQXnxh7WAgCIN67el06tvG2Jw7LB\\\\\\\/N9\\\\\\\/Eg0YC7yTT+CYJWtx5srqt1sIJNvzIBBMuoIL73t7oxn7jstjHRM6Jy0O3UDVW32G2Z+lTqsKlnIBeLIA3p+mxNf6L5aimgGMsuTJYs7Bp4RbE0i29kdhCuoEyZjsZhc0LQpNGAsh4Ts98CF7E\\\\\\\/pddeXgQYPp4Sr0gEmvi0aZWIiTSGI3BsI1cqQ9pSMNP899w0rxIUfGPWmR2+RB2qLc0N+eDZw3QoXqz+SSBApQsOvgDyhjpOJA9mp0WTaFRKYUL1thv35mxJLHSKrRaqDKV99mLbGQ7mls4UyATC4wEyWHKeKy59ggr0oNEtFA1WX0xlXQ970s9s4A2xYHnmiQZRKueYTelT6PxexoLKPJ5N+bsA+8Fs9I7URuVAlB1dOd1Bm7rM9Sdm+Cqx+X2WLYUb8\\\\\\\/93f5FiJu\\\\\\\/4Im6chNfB7YEos71unvY4EYC9MZcV2S4bDMPVYlsgsMli7aw\\\\\\\/aRZRFBW9UQXgcs\\\\\\\/q7OUKf07e1w\\\\\\\/avc08p7X1GtUFj5yWxJl9KG\\\\\\\/QhAYWKpKVtlQonYCz9cohKCIptLZ2IUAU2nP80hObByCqeD7irAqgIErEvC8C\\\\\\\/o6gV8lxxxQ5eYAZeYRBI1lVtVnP6nX4orFCm8054iUb4TruVVi2A\\\\\\\/D0EOKUkY9LbN5uX2SnAlm2QSKTR5\\\\\\\/Ybaizyl2vQERLfaM6Gnek5qJnQ4HYNWmw2WSe+nnb+7uFOnS\\\\\\\/+KZ859p48R1YUnGXIQukiSP11Iu0WcqDg3fBoC4K6XxOyn51o\\\u0022,\\\u0022iv\\\u0022:\\\u002216Eb97lY7VhZ5W5tWhw5Sg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022d2fa12e48d0054dbae4fc9add18646f4674a5080dec0920eda9cc7a56b4bbad8\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022kyPUnQfRN0aJoLFXJnaV72d9hvM0RdV1LHJ+39ihvBUsMe02nC2NGdkqoaT0Vhrr56F5GsuxQPCh3Dw+6D6Ov2B76TMl5WoWgq+SsWogFsZzmcIY52FuEpyPGbAIXF5jODnItElzhBVvl8TzAnZCvlwcf+yKBiIT\\\\\\\/5RBU+pDZ4v+Ra8lHWi7f9TWyR0RbptW+Bkjppo5hxASemCLDhLxEbaIxmXq1KJCFctTOwhIqRIXBRfJ93UyKqs0\\\\\\\/OA8rmZDA7jXxLEEkQZTGrCQHQB\\\\\\\/DkC\\\\\\\/89vAkPv4NtSil2kALHM8lgBNyRy8xIHY6UNCM8gi67K\\\\\\\/uxF+ihRZn0Kr9Gr\\\\\\\/So1\\\\\\\/BEwVSIyYM3UpxIb+N12asJZp4Wmpvtgp3718\\\\\\\/BQ6qL9kaiYDL0qvUTM+tfkcfwsECxlkX5M6oqpjjPlTxFgpwXS0sIm1hw6P3SaH5wdZdVycR3sljSeD361+hkcrePW5FEX12Iw7Fb0M07ZTKVeD3iYAdwgXgB7lUoJfZxbAx8xaNs5FC2ho7dL374ZrQMp0YynworrTItfhX1awZSLJHTinq+iZRc69IhWXmlAbGUa9OerIQfcGiOqMk3EBLp4MuUxbms5Vu4bU4vH05JSZDo0RIEJDaT96Ve1YxSusHMKtn+PFTG0cmsAKtESQCGWU1yhIXHT\\\\\\\/xNx1Y+hxc8qwn8ZOr9Je9x+8oyp00AeXEZJTMSCVpnaNff6\\\\\\\/1CMi\\\\\\\/9v+mNamDFuc2uVybQxx2GoWLwGXPgT8nZhkkZuyJh\\\\\\\/CgANSk9GRPTFWgZoeE6JdZH4LZMZ\\\\\\\/IagUwdN0X+zY2q9YCdotTQC64XybHQfsd6Tup7h8sKqMqkYvg8fu2KHBLwb8820Z54HFJYzzPawJ2q0eBaXRPbdre0Y8yQ4IMtFSxe+dystp940Q40kSPWFSUE5zlCUL66Pznj4AvEFvGFexLJ8M2FLugMaU26s9lXTID5PTULyv8OSjaeex2AGuuNH9gWJRtksfNmJDzZSCgPE4TqErfu9968m15KYJ\\\\\\\/vj9mH0rQbClJj8KNdc1\\\\\\\/uxAlMsnppU7DBLfxUfrEBMisRHGhmVrmdrMdKQ3OhV\\\\\\\/umCxDQGD3C5\\\\\\\/lLavuy1Ti6J8MceGJlYNeUm+6Epes2yGkvSl09rlsV3LfAFdvEfhIlLC49tPYEGefSQ3nM+iR3R1tErg8WULwQeThJ8xUvf3x4\\\\\\\/VMwwCnyQmKZDTLzScVH9XK+x05laRnhTM1dsWimBO2sI08BVyQIrwT61vy2Dtwv9MexJXP82MOEWksb0a1Dj8RpbXUB6H1awBnZbIl\\\\\\\/Oy57AmViInEYAkk9ng+SWysy6HPadm9AGDQeqZBb0ot7cTrq0DD97D0TL3uHai0BXv4JVP9wPQhbiNLU4XB94HlXp+l6LQxxe6fWHWEzzBpXbwhto+v2PZRnmFtV3WGwrbIf7BgEYV3JrDLESQcUhNHjLVnZ+7FqLzjN8mCj+B2O9dqRGASVr3N3frvmDI1PEHaPUvgVbC7yhhvhp+\\\\\\\/wt1PObQxtDIkC2G2bn8+UQOH\\\\\\\/HvzQTVqfE58zeZvgq4zpHOko57j6lJ4b3vEKIRq08GzeXSJ6IhGW+zjbCHXgjZL\\\\\\\/eYT69+sTXEy+b73QoPBNNuMrFT+KNGCxqCGhsYwXqgFFaWZDVHBTEjvE7vqon4EGkmD5OMmPe3Ldx0an4fjsIXkW1FHXnp7EoJ6nwo9O+CgyguX7Lf7CZl4j6Wst7mFmsk3oQfn\\\\\\\/11mrRuLTGrSexkLmJ1FYHOI8b+PYCDQA\\\\\\\/leGeivT1kuxw7w4QeqMZ\\\\\\\/XvkOn+MWAK1G9GHVoJL6Eiwu+dDkc7gN487Tmv9muuW4N\\\\\\\/V+zoB+0T2hTV5jTx2uXctjljIgRycsm0ENQlHm2rPs\\\\\\\/7UIQ+Iu5U+CT3zeLdZSRD2W8oSfFpLkKuRBildsFurWq7rylYLFKfwubNPLFIjzdd8H8psSZbSVTpF1JI\\\\\\\/G4kBCTildfk5kk14N02ZMGUvUPMES3ONKD1C\\\\\\\/SorLaXOA8KcKTtq4pcOUuZlJYppu298=\\\u0022,\\\u0022iv\\\u0022:\\\u00221\\\\\\\/V9h0PtSvPwvhU+RZeOLg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002275ad3c04993bfd417ec354987ab5ab9795e35a4322a0aacd42fd46ace007cc4c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459948 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/8/1479" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0cPmLz-2K8EFNnXE5goVx7KgXioFLx6VHsL3QyI2Y8s" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/8/1479" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-9HrxbL_jnvmK4UMoFI-XqD0rydZSFtUB3dJxa1lFU-4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/8/1465" title="Association Mapping of Germination Traits in Arabidopsis thaliana Under Light and Nutrient Treatments: Searching for G×E Effects" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/8/1491" title="Linking Genetics to Structural Biology: Complex Heterozygosity Screening with Actin Alanine Scan Alleles Identifies Functionally Related Surfaces on Yeast Actin" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459948" id="node459948" data-pisa="ggg;4/8/1479" data-pisa-master="ggg;g3.114.012435" data-apath="/ggg/4/8/1479.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Interplay of Interlocus Gene Conversion and Crossover in Segmental Duplications Under a Neutral Scenario</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Diego A. Hartasánchez</span>, <span class="highwire-citation-author" data-delta="1">Oriol Vallès-Codina</span>, <span class="highwire-citation-author" data-delta="2">Marina Brasó-Vives</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-2162-8246" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Arcadi Navarro</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1479-1489;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012435 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459948" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Diego A. Hartasánchez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolutionary Biology (Universitat Pompeu Fabra – CSIC), PRBB, Barcelona, Catalonia, Spain, 08003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BA.%2BHartas%C3%A1nchez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hartas%C3%A1nchez%20DA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADiego%2BA.%2BHartas%25C3%25A1nchez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Oriol Vallès-Codina </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolutionary Biology (Universitat Pompeu Fabra – CSIC), PRBB, Barcelona, Catalonia, Spain, 08003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Oriol%2BVall%C3%A8s-Codina%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vall%C3%A8s-Codina%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOriol%2BVall%25C3%25A8s-Codina%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Marina Brasó-Vives </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolutionary Biology (Universitat Pompeu Fabra – CSIC), PRBB, Barcelona, Catalonia, Spain, 08003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marina%2BBras%C3%B3-Vives%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bras%C3%B3-Vives%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarina%2BBras%25C3%25B3-Vives%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Arcadi Navarro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolutionary Biology (Universitat Pompeu Fabra – CSIC), PRBB, Barcelona, Catalonia, Spain, 08003</div><div class='author-affiliation'><span class='nlm-sup'>†</span>National Institute for Bioinformatics (INB), Barcelona, Catalonia, Spain</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institució Catalana de Recerca i Estudis Avançats (ICREA), Barcelona, Catalonia, Spain</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Genomic Regulation (CRG), Barcelona, Catalonia, Spain, 08003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Arcadi%2BNavarro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Navarro%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AArcadi%2BNavarro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2162-8246" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Arcadi Navarro</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:arcadi.navarro@upf.edu" class="" data-icon-position="" data-hide-link-title="0">arcadi.navarro@upf.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/8/1479" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459948" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459948/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/8/1479.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459948" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459948/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/8/1479.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459948" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459948/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/8/1479.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459948" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459948/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/8/1479.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459948" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459948/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Interlocus gene conversion is a major evolutionary force that drives the concerted evolution of duplicated genomic regions. Theoretical models successfully have addressed the effects of interlocus gene conversion and the importance of crossover in the evolutionary fate of gene families and duplications but have not considered complex recombination scenarios, such as the presence of hotspots. To study the interplay between interlocus gene conversion and crossover, we have developed a forward-time simulator that allows the exploration of a wide range of interlocus gene conversion rates under different crossover models. Using it, we have analyzed patterns of nucleotide variation and linkage disequilibrium within and between duplicate regions, focusing on a neutral scenario with constant population size and validating our results with the existing theoretical models. We show that the interaction of gene conversion and crossover is nontrivial and that the location of crossover junctions is a fundamental determinant of levels of variation and linkage disequilibrium in duplicated regions. We also show that if crossover activity between duplications is strong enough, recurrent interlocus gene conversion events can break linkage disequilibrium within duplicates. Given the complex nature of interlocus gene conversion and crossover, we provide a framework to explore their interplay to help increase knowledge on molecular evolution within segmental duplications under more complex scenarios, such as demographic changes or natural selection.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/concerted-evolution" class="hw-term hw-article-keyword hw-article-keyword-concerted-evolution" rel="nofollow">concerted evolution</a></li><li class="kwd"><a href="/keyword/linkage-disequilibrium" class="hw-term hw-article-keyword hw-article-keyword-linkage-disequilibrium" rel="nofollow">linkage disequilibrium</a></li><li class="kwd"><a href="/keyword/recombination-hotspots" class="hw-term hw-article-keyword hw-article-keyword-recombination-hotspots" rel="nofollow">recombination hotspots</a></li><li class="kwd"><a href="/keyword/forward-simulations" class="hw-term hw-article-keyword hw-article-keyword-forward-simulations" rel="nofollow">forward simulations</a></li><li class="kwd"><a href="/keyword/increased-variation" class="hw-term hw-article-keyword hw-article-keyword-increased-variation" rel="nofollow">increased variation</a></li></ul><p id="p-4">Gene duplication has been recognized as a primary source of genetic innovation since Ohno’s seminal work on this topic in 1970 (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Ohno 1970</a>). The accumulation of genomic sequence data from a wide range of species has shown that segmental duplications (SDs) spanning more than 1 kb and retaining a high degree of sequence homogeneity (&gt;90%) are a pervasive feature of eukaryotic genomes (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Lynch and Conery 2000</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bailey <em>et al.</em> 2002</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Marques-Bonet <em>et al.</em> 2009</a>). Understanding the molecular evolution of duplicated nucleotide sequences is also of great relevance because SDs define hotspots of chromosomal rearrangements and are known to give rise to copy-number variants (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Sharp <em>et al.</em> 2005</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Mills <em>et al.</em> 2011</a>; <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Uddin <em>et al.</em> 2011</a>), which in turn frequently are implicated in disease susceptibility (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Conrad <em>et al.</em> 2010</a>; <a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Stankiewicz and Lupski 2010</a>) and are targets of natural selection (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Gazave <em>et al.</em> 2011</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Iskow <em>et al.</em> 2012</a>; <a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Lorente-Galdos <em>et al.</em> 2013</a>).</p><p id="p-5">During the 1980s, extensive theoretical work was carried out on the subject of multigene family evolution (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Baltimore 1981</a>; <a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dover 1982</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Ohta 1982</a>, <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">1983</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Nagylaki and Petes 1982</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Nagylaki 1984</a>). More recently, Innan and collaborators (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Innan 2002</a>, <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">2003</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Teshima and Innan 2004</a>; <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Mano and Innan 2008</a>) and <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Thornton (2007)</a> have successfully combined analytical results and coalescent simulations to study diversity patterns and divergence times between duplicates and have analyzed various selection scenarios. Still, many unique features of SDs have not yet been fully addressed, neither analytically or by simulation.</p><p id="p-6">The foremost feature of duplicated genomic regions is that they undergo concerted evolution due to the exchange of genetic information via interlocus gene conversion (IGC) (<a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Nagylaki and Petes 1982</a>; <a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Ohta 1982</a>). IGC can be described as a copy-paste event (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Innan 2009</a>) whereby a fragment of one of the duplicated segments is copied onto the corresponding segment of the duplicate (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Wiuf and Hein 2000</a>). IGC is also referred to as nonallelic, interparalog, or ectopic gene conversion. IGC is known to be a major player in small (two-copy) multigene family evolution (<a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Ohta 1983</a>), and empirical estimates of IGC rates span several orders of magnitude (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Chen <em>et al.</em> 2007</a>; <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Benovoy and Drouin 2009</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">McGrath <em>et al.</em> 2009</a>). The extent to which the theory from multigene family evolution can be applied to the evolution of SDs and copy-number variants remains an open question. In particular, because SDs can span large regions, some genomic features, such as the pervasiveness of crossover hotspots in the human genome (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Jeffreys <em>et al.</em> 2001</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Kong <em>et al.</em> 2002</a>), might have non-negligible effects on the levels of variation within and between SDs. The evolution of SDs under different crossover models, including the overlap of IGC susceptible regions with crossover hotspots, or the linkage disequilibrium (LD) patterns within and between SDs, are some of the issues that, to our knowledge, have not been addressed before.</p><p id="p-7">To tackle some of these questions, a flexible forward-time simulator should prove to be a powerful tool. Although coalescent simulators have the advantage of being extremely fast, forward-time simulators allow the exploration of a wider parameter space and the implementation of different scenarios (such as sophisticated crossover models) in a straightforward way. We here introduce a forward-time algorithm in C++ and explore a wide range of realistic parameter values in a model of neutral evolution of duplicated chromosomal regions undergoing mutation, IGC, and crossover under different models.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><p id="p-8">We model a Wright-Fisher population of <em>N</em> diploid individuals, where each of the <em>2N</em> chromosomes consists of either two or three blocks. Each block is <em>L</em> nucleotides long and is characterized by an ordered set of integers (representing nucleotide positions) between 0 and <em>L−1</em>.</p><div id="sec-2" class="subsection"><h3>Simulation procedure: phases</h3><p id="p-9">Simulations have three phases, the <em>burn-in phase</em>, the <em>structured phase</em>, and the <em>concerted evolution phase</em>. Each simulation begins with a burn-in phase of <em>T<sub>1</sub></em> generations in which the population undergoes random mating, mutation, and recombination by crossover. During this first phase, each chromosome is composed of two blocks that we will refer to as <em>original</em> and <em>single-copy</em> blocks. We run this first phase long enough for the population to reach equilibrium. At <em>t = T<sub>1</sub></em> + 1, the structured phase starts with a duplication event in which the original block of a randomly chosen chromosome is duplicated and “pasted” to the right of its single-copy block (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). We shall refer to this third block as <em>duplicated</em> block. This unique duplication is fixed by drift. To achieve fixation in a computationally efficient way, a neutral fixation trajectory is simulated using Kimura’s pseudo-sampling method (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Kimura 1980</a>). During the structured phase, of average <em>T<sub>2</sub></em> = 4<em>N</em> generations (and maximum <em>T<sub>2</sub></em> = 20<em>N</em> generations), the number of chromosomes carrying the duplication in each generation is determined by the fixation trajectory. Once the fixation of the duplication has occurred, the program proceeds to a concerted evolution phase (of <em>T<sub>3</sub></em> = <em>T<sub>T</sub></em> – <em>T<sub>1</sub></em> – <em>T<sub>2</sub></em> generations) until <em>t = T<sub>T</sub></em>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/8/1479/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="The three phases of every simulation run. Each simulation begins with a burn-in phase, in which a population formed by chromosomes with two single-copy blocks (dark blue and orange) is brought to mutation-drift equilibrium. The duplication of the first of these blocks (original block: dark blue, duplicated block: light blue) marks the initiation of the structured phase, during which the duplication becomes fixed. Finally, during the concerted evolution phase, the population reaches a new equilibrium in which the interplay of interlocus gene conversion between duplicated blocks and crossover determines levels and patterns of variability." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-701161318" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;The three phases of every simulation run. Each simulation begins with a burn-in phase, in which a population formed by chromosomes with two single-copy blocks (dark blue and orange) is brought to mutation-drift equilibrium. The duplication of the first of these blocks (original block: dark blue, duplicated block: light blue) marks the initiation of the structured phase, during which the duplication becomes fixed. Finally, during the concerted evolution phase, the population reaches a new equilibrium in which the interplay of interlocus gene conversion between duplicated blocks and crossover determines levels and patterns of variability.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/8/1479/F1.medium.gif" width="440" height="154"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/8/1479/F1.medium.gif" width="440" height="154"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/8/1479/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/8/1479/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464801" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-10" class="first-child">The three phases of every simulation run. Each simulation begins with a burn-in phase, in which a population formed by chromosomes with two single-copy blocks (dark blue and orange) is brought to mutation-drift equilibrium. The duplication of the first of these blocks (original block: dark blue, duplicated block: light blue) marks the initiation of the structured phase, during which the duplication becomes fixed. Finally, during the concerted evolution phase, the population reaches a new equilibrium in which the interplay of interlocus gene conversion between duplicated blocks and crossover determines levels and patterns of variability.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-3" class="subsection"><h3>Mutation</h3><p id="p-11">Mutation is active during all phases and occurs independently in each block at a rate μ per site per generation. We use a pseudo infinite-sites model (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Padhukasahasram <em>et al.</em> 2008</a>) in which the number of sites in each block is finite (<em>L</em>), but mutations can only appear in nonpolymorphic positions. For mutational purposes, a variant that occurs in either the original or the duplicated block is still considered as polymorphic even if it is segregating only in one of them, and therefore any mutation is only considered as <em>fixed</em> if it is present in all chromosomes on both blocks. Multiple mutations are not allowed in the same block and generation. We have verified that allowing multiple mutations does not change the results presented in this work.</p></div><div id="sec-4" class="subsection"><h3>Crossover</h3><p id="p-12">Homologous crossover occurs at rate <em>r</em> per meiosis, with a maximum of one crossover allowed per meiosis. Most previous studies of multigene family evolution (<a id="xref-ref-45-3" class="xref-bibr" href="#ref-45">Ohta 1983</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Nagylaki 1984</a>; <a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Innan 2002</a>; <a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Thornton 2007</a>) consider intergenic crossover only. In other words, previous models are such that meiotic crossover junctions are only allowed between duplicated blocks (usually called “genes” in these papers) but not within them. We will refer to this model as the single-copy crossover (SCC) model since all crossover junctions fall within the single-copy block. Here, we explore two additional crossover models: the whole-region crossover (WRC) model, that allows crossover junctions from 5′ end of one duplicate to 3′ end of the other duplicate, which so far has only been used by <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Teshima and Innan (2012)</a>; and the hotspot crossover (HSC) model, in which a particular region of the simulated segment is predefined as a crossover hotspot such that all crossover junctions fall within this region. Choosing an adequate definition of <em>r</em> is not straightforward, given that chromosomes with different lengths (two or three blocks) segregate at changing frequencies in the population, and that different models have different <em>a priori</em> distributions of crossover events. To facilitate comparison among models, we define <em>r</em> per meiosis, irrespective of the lengths of the chromosomes involved. In other words, the same average number of crossovers occurs per meiosis per generation under any model and phase. This implies, for example, that under the WRC model, the effective crossover rate per base pair is higher during the burn-in phase (two blocks) than during the concerted evolution phase (three blocks). The effects of this choice are always small, do not affect measures taken at equilibrium, and are easier to account for than those that would be introduced by other potential definitions of <em>r</em>. We will refer to the population scaled crossover rate as <em>R</em> = 4N<em>r</em>. At each crossover event, a crossover junction is randomly selected from the region allowed by the underlying crossover model.</p></div><div id="sec-5" class="subsection"><h3>Interlocus gene conversion</h3><p id="p-13">At every generation, if a chromosome contains the duplication, there is a per site probability of an IGC event being initiated, <em>g</em>. An IGC event can be described as a copy-paste event (<a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Innan 2009</a>) in which the information (<em>i.e.</em>, all derived mutations) contained along an IGC tract on one of the duplicate blocks is copied and pasted onto the other duplicate substituting any variants that were previously within the paralogous tract in that duplicate. In our model, an IGC event can be initiated at any position of any duplicate block according to a uniform distribution. The length of IGC tracts is determined according to a geometric distribution with average size <em>λ</em>, which fits available empirical evidence (<a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Wiuf and Hein 2000</a>). Following <a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Wiuf and Hein (2000)</a>, the tract length for every IGC event, <em>l</em>, is extracted from a geometric distribution with parameter <em>q = λ/L:</em><span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1479/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1479/embed/mml-math-1.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>Instead of randomly choosing the direction in which the IGC tract extends (either 3′ or 5′), we determine the IGC tract extending <em>l/2</em> sites to the left and <em>l/2</em> to the right of the initiation point or junction. <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Mansai <em>et al.</em> (2011)</a> proposed a similar model in which there is an independent exponential elongation of the gene conversion tract in both directions from the initiation point. We deviate from previous models (<a id="xref-ref-61-4" class="xref-bibr" href="#ref-61">Wiuf and Hein 2000</a>; <a id="xref-ref-57-3" class="xref-bibr" href="#ref-57">Thornton 2007</a>; <a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Mansai <em>et al.</em> 2011</a>) in limiting IGC tracts to the duplicate regions by the simple procedure of truncating any tract that extends beyond the duplicate blocks. We expect that this simplification has a negligible impact on our results. In our model, IGC only occurs between paralogous regions on the same chromosome and both copies act as donor or receptor with equal probability. The mutation positions found in the donor tract are imported to the receptor, and the mutations in the receptor tract are erased. Finally, we allow IGC to occur regardless of the divergence between blocks on the same chromosome (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012435/-/DC1/012435SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012435/-/DC1/FileS1.pdf">File S1</a>). Other authors, notably <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Teshima and Innan (2004)</a>, implemented a divergence threshold above which IGC is terminated. The IGC rate per site per generation will be <em>c</em> = <em>gλ</em>, where <em>λ</em> is the effective average length of the IGC tracts (although not accounting for possible truncations) and the population IGC rate will be <em>C =</em> 4<em>Nc</em>.</p></div><div id="sec-6" class="subsection"><h3>Eras: simulating neutral genealogies</h3><p id="p-14">The neutral scenario allows us to decrease the running time of simulations by avoiding the explicit simulation of chromosomes that will eventually leave no offspring in the population (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Padhukasahasram <em>et al.</em> 2008</a>). This requires simulating a genealogy for <em>k</em> generations in advance and then tracing it back to ascertain which chromosomes, at each generation, will make a contribution to the final population. For the sake of clarity, we will refer to each period of <em>k</em> generations as an <em>Era</em>. Because there is a trade-off between the time it takes to simulate the genealogy and the time saved by not simulating those chromosomes destined to be lost, we need to choose an optimal value of <em>k</em>. This value varies considerably with population size, crossover rate, and other parameters, so we selected <em>k</em> by performing trial runs for every combination of parameters (not shown).</p><p id="p-15">Eras start at generation <em>t’ =</em> 0. To construct the genealogy for each Era, each chromosome at each of the <em>k</em> generations is randomly assigned a <em>parental</em> chromosome from the previous generation. When <em>R</em> = 0, the process is straightforward: first, all chromosomes at generation <em>t’ = k</em> − 1, are tagged as <em>fertile</em>; next, the parental chromosome of every fertile individual at <em>t’ = k</em> − 1 is tagged as fertile at <em>t’ = k</em> − 2; and the same is repeated for every generation until <em>t</em>’ = 0. The case with crossover (<em>R &gt;</em> 0) is essentially the same except that when tagging the parental chromosomes as fertile, there is a probability, <em>r</em>, that the <em>partner</em> of that chromosome (<em>i.e.</em>, the other chromosome from the same individual) is also tagged as fertile.</p><p id="p-16">During the structured phase, the number of chromosomes carrying the duplication (<em>s(t’))</em> is determined by a neutral fixation trajectory. As mentioned previously, the structured phase begins with a unique duplication event in a randomly chosen chromosome (<em>i.e. s</em>(<em>t’=0</em>) = 1). At <em>t’ = 1</em>, <em>s(1)</em> randomly chosen chromosomes will be assigned the chromosome carrying the duplication as their parental chromosome. The rest of the 2<em>N – s(1)</em> chromosomes will be randomly assigned a parental chromosome not carrying the duplication (at this point, any other chromosome). At <em>t</em>’ = 2, <em>s</em>(2) randomly chosen chromosomes will be assigned any of the <em>s(1)</em> chromosomes carrying the duplication at <em>t’ = 1</em> as their parental chromosome and the rest of the 2<em>N – s(2)</em> chromosomes will be randomly assigned a parental chromosome not carrying the duplication at <em>t’ = 1</em>. This process is continued until <em>t’ = k</em> – 1. Typically, because the average neutral fixation time is 4<em>N</em> generations, fixation trajectories will be larger than <em>k</em>. Thus, the aforementioned process will frequently exceed <em>k</em> generations.</p><p id="p-17">Before the duplication reaches fixation, crossover might occur between chromosomes with a different number of blocks. In this case, irrespective of the underlying crossover model, the position of the crossover junction will be chosen from their shared length. Without loss of generality, the <em>daughter</em> chromosome will have the same number of blocks as its parental chromosome, regardless of the number of blocks of its parental chromosome’s partner. To ensure this, the daughter chromosome will inherit the region from the 5′ end until the crossover junction (including the junction) from its parental chromosome’s partner and the region from the junction until the 3′ end from its parental chromosome. At the end of every Era, the mutations present on every block in a random sample of <em>n</em> = 50 individuals (<em>i.e.</em>, 100 chromosomes) are recorded. Fixed derived mutations are erased from simulated chromosomes after being recorded (mutations on duplicated blocks are only erased if they are fixed in both blocks), which saves memory and running time. At the end of each simulation run, we will have recorded data every <em>k</em> generations.</p></div><div id="sec-7" class="subsection"><h3>Variation measures</h3><p id="p-18">From the data recorded every Era, we extract site-frequency spectra and variation measurements such as the number of segregating sites (<em>S</em>) and the average number of pairwise differences (π). Although historically it makes sense to talk about sequence diversity and divergence when referring to duplicated regions of the genome, from here on and for the sake of simplicity, we will use the term <em>variation</em> to refer to both because we will measure both diversity and divergence by calculating π within and between copies. To compare the evolution of the duplicate blocks under different parameter values, we explore nucleotide variation with the following measures (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>): variation <span class="underline">w</span>ithin a duplicate block (π<sub>w</sub>) (at equilibrium, variation within the original block and within the duplicated block will be the same); variation <span class="underline">b</span>etween the original and duplicated blocks on different chromosomes (π<sub>b</sub>); and variation between the original and duplicated blocks on the <span class="underline">s</span>ame chromosome (π<sub>s</sub>). In addition, we also calculate the variation present within the single-copy block, which will serve as control.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/8/1479/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Variation measures. We measure nucleotide variation among segmental duplications as follows: (w) variation within duplicate blocks (at equilibrium, variation within the original and duplicated blocks will be the same); (b) variation between the original and duplicated blocks on different chromosomes; (s) variation between the original and duplicated blocks on the same chromosome. We use average pairwise differences (&#x3C0;) to measure all these types of variation (&#x3C0;w, &#x3C0;b, &#x3C0;s)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-701161318" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Variation measures. We measure nucleotide variation among segmental duplications as follows: (w) variation within duplicate blocks (at equilibrium, variation within the original and duplicated blocks will be the same); (b) variation between the original and duplicated blocks on different chromosomes; (s) variation between the original and duplicated blocks on the same chromosome. We use average pairwise differences (&#x3C0;) to measure all these types of variation (&#x3C0;&lt;sub&gt;w&lt;/sub&gt;, &#x3C0;&lt;sub&gt;b&lt;/sub&gt;, &#x3C0;&lt;sub&gt;s&lt;/sub&gt;).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/8/1479/F2.medium.gif" width="440" height="302"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/8/1479/F2.medium.gif" width="440" height="302"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/8/1479/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/8/1479/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464797" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-19" class="first-child">Variation measures. We measure nucleotide variation among segmental duplications as follows: (w) variation <span class="underline">w</span>ithin duplicate blocks (at equilibrium, variation within the original and duplicated blocks will be the same); (b) variation <span class="underline">b</span>etween the original and duplicated blocks on different chromosomes; (s) variation between the original and duplicated blocks on the <span class="underline">s</span>ame chromosome. We use average pairwise differences (π) to measure all these types of variation (π<sub>w</sub>, π<sub>b</sub>, π<sub>s</sub>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-20">Average values from 10,000 simulations of π within the original, the single-copy, and the duplicated blocks are represented in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>. During the burn-in phase, variation within the original and single-copy blocks reaches its neutral expectation π<sub>w</sub> = Θ. Once the duplication appears (at <em>t = T<sub>1</sub></em> + 1 <em>=</em> 30<em>N</em>), variation begins to increase in the duplicated block, and IGC activity begins between the original and the duplicated blocks. Distributions of the variation within the original and duplicated blocks at different time points are shown in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/8/1479/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Changes in variation within blocks along simulations. Average results from 10,000 simulation runs are shown. (A) Dark blue, orange, and light blue curves correspond to the average pairwise differences found within the original, single-copy, and duplicated blocks, respectively. Gray-shaded areas correspond to the burn-in phase, structured phase, and concerted evolution phase. Duplication occurs at t = 30N. Although we depict the structured phase as ending at t = 50N, this is actually an arbitrary upper limit, because the neutral trajectory of the duplicated chromosomes and length of the structured phase is different every simulation. As expected, variation at equilibrium for the single-copy block is &#x398; = &#x3B8;L = 4N&#x3BC;L. The original and duplicated blocks attain higher variation (~1.95&#x398;) due to IGC activity among them. Parameters for this simulation are N = 1000, k = 1000, L = 5000, &#x3B8; = 0.001, C = 0.5, R = 50, and &#x3BB; = 100. (B) Distributions of &#x3C0;wsim for the original (top) and duplicated (bottom) blocks at different times after the appearance of the duplication (t = 31N, 32N, 34N, 38N, 50N, and 120N) are colored in different shades of green." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-701161318" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Changes in variation within blocks along simulations. Average results from 10,000 simulation runs are shown. (A) Dark blue, orange, and light blue curves correspond to the average pairwise differences found within the original, single-copy, and duplicated blocks, respectively. Gray-shaded areas correspond to the burn-in phase, structured phase, and concerted evolution phase. Duplication occurs at &lt;em&gt;t&lt;/em&gt; = 30&lt;em&gt;N&lt;/em&gt;. Although we depict the structured phase as ending at &lt;em&gt;t =&lt;/em&gt; 50&lt;em&gt;N&lt;/em&gt;, this is actually an arbitrary upper limit, because the neutral trajectory of the duplicated chromosomes and length of the structured phase is different every simulation. As expected, variation at equilibrium for the single-copy block is &#x398; = &#x3B8;&lt;em&gt;L&lt;/em&gt; = 4&lt;em&gt;N&lt;/em&gt;&#x3BC;&lt;em&gt;L&lt;/em&gt;. The original and duplicated blocks attain higher variation (~1.95&#x398;) due to IGC activity among them. Parameters for this simulation are &lt;em&gt;N&lt;/em&gt; = 1000, &lt;em&gt;k&lt;/em&gt; = 1000, &lt;em&gt;L&lt;/em&gt; = 5000, &#x3B8; = 0.001, &lt;em&gt;C&lt;/em&gt; = 0.5, &lt;em&gt;R&lt;/em&gt; = 50, and &#x3BB; = 100. (B) Distributions of &#x3C0;&lt;sub&gt;w&lt;/sub&gt;&lt;sup&gt;sim&lt;/sup&gt; for the original (top) and duplicated (bottom) blocks at different times after the appearance of the duplication (&lt;em&gt;t&lt;/em&gt; = 31&lt;em&gt;N&lt;/em&gt;, 32&lt;em&gt;N&lt;/em&gt;, 34&lt;em&gt;N&lt;/em&gt;, 38&lt;em&gt;N&lt;/em&gt;, 50&lt;em&gt;N&lt;/em&gt;, and 120&lt;em&gt;N&lt;/em&gt;) are colored in different shades of green.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/8/1479/F3.medium.gif" width="440" height="237"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/8/1479/F3.medium.gif" width="440" height="237"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/8/1479/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/8/1479/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464765" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-21" class="first-child">Changes in variation within blocks along simulations. Average results from 10,000 simulation runs are shown. (A) Dark blue, orange, and light blue curves correspond to the average pairwise differences found within the original, single-copy, and duplicated blocks, respectively. Gray-shaded areas correspond to the burn-in phase, structured phase, and concerted evolution phase. Duplication occurs at <em>t</em> = 30<em>N</em>. Although we depict the structured phase as ending at <em>t =</em> 50<em>N</em>, this is actually an arbitrary upper limit, because the neutral trajectory of the duplicated chromosomes and length of the structured phase is different every simulation. As expected, variation at equilibrium for the single-copy block is Θ = θ<em>L</em> = 4<em>N</em>μ<em>L</em>. The original and duplicated blocks attain higher variation (~1.95Θ) due to IGC activity among them. Parameters for this simulation are <em>N</em> = 1000, <em>k</em> = 1000, <em>L</em> = 5000, θ = 0.001, <em>C</em> = 0.5, <em>R</em> = 50, and λ = 100. (B) Distributions of π<sub>w</sub><sup>sim</sup> for the original (top) and duplicated (bottom) blocks at different times after the appearance of the duplication (<em>t</em> = 31<em>N</em>, 32<em>N</em>, 34<em>N</em>, 38<em>N</em>, 50<em>N</em>, and 120<em>N</em>) are colored in different shades of green.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-8" class="subsection"><h3>LD measures</h3><p id="p-22">To analyze how LD patterns vary with IGC rates and crossover models, we have devised a simple way of extracting LD measurements from independent simulation runs and of obtaining average values from them. To do so, we have binned every block in 100-bp windows. For every run, we analyze a sample of <em>n</em> = 50 individuals at <em>t</em> = <em>T<sub>T</sub></em>, once the population has reached equilibrium. For every pair of windows <em>x</em> and <em>y</em>, we calculate <em>D</em>’ and <em>r<sup>2</sup></em> between all possible pairs of mutations (one taken from <em>x</em> and the other from <em>y</em>). We then calculate the average value of all these measurements and thus obtain a value of <em>D</em>’ and <em>r<sup>2</sup></em> for every pair of windows that is independent of the number of mutations that fall within a particular window in a particular run. We repeat this process for every simulation run and calculate the average values for every pair of windows. During the analysis of LD patterns, we will differentiate between LD within the duplicate regions and LD between paralogous windows of the original and duplicated blocks (from here on, we refer to the latter as <em>LD between duplicates</em>).</p></div><div id="sec-9" class="subsection"><h3>Parameter values</h3><p id="p-23">In this paper, we concentrate on the effect of two parameters: the IGC rate (<em>C</em>), and the homologous crossover rate (<em>R</em>). Other parameters, such as the point mutation rate, only modify the scale and variance of our results. To allow for accurate comparisons with results from previous works, all our parameters are population scaled. Population size is an important parameter in so far as it does have an effect in populations undergoing natural selection. However, because we are exclusively considering a neutral scenario, we have fixed our population effective size to <em>N</em> = 1000. The range of parameters we have explored is based primarily on previous coalescent simulations (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Innan 2003</a>; <a id="xref-ref-57-4" class="xref-bibr" href="#ref-57">Thornton 2007</a>) and a literature survey selecting a range of empirical estimates from various species to guarantee that we are exploring realistic scenarios. In <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>, the range of parameters we explore here is compared to those in previous works. Details about how they were selected can be found in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012435/-/DC1/FileS1.pdf">File S1</a>. In addition, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012435/-/DC1/FileS2.pdf">File S2</a> shows how rates of IGC vary according to the distance between duplicated blocks. In this paper, we exclusively explore duplicate blocks of length <em>L</em> separated by a block of equal length. The Perl and R scripts to construct the LD plots, as well as the C++ forward simulator used throughout this study, are available upon request.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464627/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464627/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Comparison between parameter ranges explored through simulations by <a id="xref-ref-21-3" class="xref-bibr" href="#ref-21">Innan 2003</a>, <a id="xref-ref-57-5" class="xref-bibr" href="#ref-57">Thornton 2007</a>, and this work</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><p id="p-25">The action of gene conversion between the original and duplicated blocks increases the amount of variation found within each of them. Variation within blocks will attain an equilibrium value between Θ and 2Θ, which will depend on IGC and crossover rates. <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a> shows the case for <em>C</em> = 0.5, <em>R</em> = 50, and <em>N</em> = 1000, for which π<sub>w</sub> ≈ 1.95Θ. IGC increases variation by transferring mutations that appear in one block onto the other block while at the same time preventing the fixation of different derived mutations in each block. However, a clear increment in variation is observed only for a limited range of IGC rates. On the one hand, very high IGC rates can homogenize copies to such a point that they remain nearly identical and maintain a level of variation similar to that of single-copy regions. On the other hand, very low IGC rates can increase variation up to 2Θ, but waiting times to reach this equilibrium can be extremely high (<a id="xref-ref-40-3" class="xref-bibr" href="#ref-40">Nagylaki 1984</a>).</p><p id="p-26">Increased variation within duplicates caused by IGC activity has already been extensively studied and modeled for small and large multigene families. We have selected three different models (<a id="xref-ref-45-4" class="xref-bibr" href="#ref-45">Ohta 1983</a>; <a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Innan 2002</a>, <a id="xref-ref-21-5" class="xref-bibr" href="#ref-21">2003</a>) to validate the results of our simulations on variation (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012435/-/DC1/FileS3.pdf">File S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012435/-/DC1/FigureS1.pdf">Figure S1</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012435/-/DC1/FigureS2.pdf">Figure S2</a>) and on LD (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012435/-/DC1/FileS4.pdf">File S4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012435/-/DC1/FigureS3.pdf">Figure S3</a>). Given some disparity in nomenclature among this body of work, we have introduced a new notation differentiating models, which is explained in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. After validation, we proceeded to examine in detail the interplay between IGC and crossover in determining levels and patterns of variation and LD in duplications. In order to do so, we analyzed three different crossover models: SCC, in which crossover is limited to the <span class="underline">s</span>ingle-<span class="underline">c</span>opy region; WRC, in which crossover can occur in the <span class="underline">w</span>hole simulated <span class="underline">r</span>egion; and HSC, in which crossover happens at a previously defined <span class="underline">h</span>ot<span class="underline">s</span>pot or hotspots.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464739/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464739/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Nomenclature used throughout this work to distinguish between different types of variation, and allowing for comparison between theoretical models by <a id="xref-ref-45-5" class="xref-bibr" href="#ref-45">Ohta (1983)</a> and <a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Innan (2002</a>, <a id="xref-ref-21-6" class="xref-bibr" href="#ref-21">2003</a>)</span><div class="sb-div caption-clear"></div></div></div><div id="sec-11" class="subsection"><h3>SCC model: single-copy crossover</h3><p id="p-27">Under this model, crossover occurs only in the single-copy region between duplicates. The levels of variation found for the SCC model fall directly on the theoretical predictions provided by <a id="xref-ref-45-7" class="xref-bibr" href="#ref-45">Ohta (1983)</a> and <a id="xref-ref-20-6" class="xref-bibr" href="#ref-20">Innan (2002</a>, <a id="xref-ref-21-8" class="xref-bibr" href="#ref-21">2003</a>) with a very high degree of accuracy (for a detailed description and comparison between these models see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012435/-/DC1/FileS3.pdf">File S3</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012435/-/DC1/FileS5.pdf">File S5</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012435/-/DC1/FigureS1.pdf">Figure S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012435/-/DC1/FigureS2.pdf">Figure S2</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012435/-/DC1/FigureS4.pdf">Figure S4</a>). Under the infinite-site model, variation within a duplicate at equilibrium is negatively correlated with IGC rate and positively correlated with crossover rates. The explanation for the latter is that the higher the crossover rate, the higher the probability for two duplicate blocks that have already undergone IGC to become separated, thus allowing for another IGC event to effectively transfer different variants from one block to another.</p><p id="p-28"><a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012435/-/DC1/FigureS5.pdf">Figure S5</a> show LD patterns (D′ and r<sup>2</sup>, respectively) for different crossover models and IGC rates. Row 1 shows the case for <em>R</em> = 0 and provides a null expectation for the effect of IGC in the absence of crossover: high LD between duplicates (see <em>Materials and Methods</em>), represented by a diagonal dark blue line. Under the SCC model (row 2), we observe that in the presence of crossover, the pattern of LD in regions undergoing IGC changes considerably. LD within the single-copy block will be low due to recurrent crossover within it, relative to the higher levels of LD in the original and duplicated blocks where no crossover occurs. However, this only holds for low IGC rates, because moderate-to-high IGC between duplicates also breaks down LD within them. Furthermore, LD between duplicates will always increase with increasing IGC rate but will decrease with the rate of crossover.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/8/1479/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="LD patterns under different crossover models. Average values for 1000 simulations are shown. LD between each pair of windows along the sequence (D&#x2032;) is coded with a number between 0 and 1 and represented with a color (from white to dark blue to light green). Three different IGC rates (0, 1, and 50) are represented in columns and three different crossover conditions are shown in rows: no crossover, SCC (R = 50), and WRC (R = 50). The red lines below the names identifying each block show regions undergoing crossover. In the first row, where no crossover is acting, a dark blue diagonal line appears when IGC is active (and increasing with IGC rate) representing LD between paralogous windows of duplicate blocks. LD within the duplicate region is high when no IGC is acting and when IGC is high (C = 50) but decreases with a medium IGC rate. LD between duplicates (dark blue diagonal line) decreases when crossover is active on the single-copy region (SCC model, row 2) and on the whole region (WRC model, row 3) with respect to R = 0. As expected, crossover breaks LD blocks in the regions where it is acting. IGC also breaks LD blocks within duplicates if crossover is active (both in the SCC model and in the WRC model)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-701161318" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;LD patterns under different crossover models. Average values for 1000 simulations are shown. LD between each pair of windows along the sequence (D&#x2032;) is coded with a number between 0 and 1 and represented with a color (from white to dark blue to light green). Three different IGC rates (0, 1, and 50) are represented in columns and three different crossover conditions are shown in rows: no crossover, SCC (&lt;em&gt;R&lt;/em&gt; = 50), and WRC (&lt;em&gt;R&lt;/em&gt; = 50). The red lines below the names identifying each block show regions undergoing crossover. In the first row, where no crossover is acting, a dark blue diagonal line appears when IGC is active (and increasing with IGC rate) representing LD between paralogous windows of duplicate blocks. LD within the duplicate region is high when no IGC is acting and when IGC is high (&lt;em&gt;C&lt;/em&gt; = 50) but decreases with a medium IGC rate. LD between duplicates (dark blue diagonal line) decreases when crossover is active on the single-copy region (SCC model, row 2) and on the whole region (WRC model, row 3) with respect to &lt;em&gt;R&lt;/em&gt; = 0. As expected, crossover breaks LD blocks in the regions where it is acting. IGC also breaks LD blocks within duplicates if crossover is active (both in the SCC model and in the WRC model).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/8/1479/F4.medium.gif" width="440" height="376"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/8/1479/F4.medium.gif" width="440" height="376"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/8/1479/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/8/1479/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464799" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-29" class="first-child">LD patterns under different crossover models. Average values for 1000 simulations are shown. LD between each pair of windows along the sequence (D′) is coded with a number between 0 and 1 and represented with a color (from white to dark blue to light green). Three different IGC rates (0, 1, and 50) are represented in columns and three different crossover conditions are shown in rows: no crossover, SCC (<em>R</em> = 50), and WRC (<em>R</em> = 50). The red lines below the names identifying each block show regions undergoing crossover. In the first row, where no crossover is acting, a dark blue diagonal line appears when IGC is active (and increasing with IGC rate) representing LD between paralogous windows of duplicate blocks. LD within the duplicate region is high when no IGC is acting and when IGC is high (<em>C</em> = 50) but decreases with a medium IGC rate. LD between duplicates (dark blue diagonal line) decreases when crossover is active on the single-copy region (SCC model, row 2) and on the whole region (WRC model, row 3) with respect to <em>R</em> = 0. As expected, crossover breaks LD blocks in the regions where it is acting. IGC also breaks LD blocks within duplicates if crossover is active (both in the SCC model and in the WRC model).</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">In the absence of crossover, moderate IGC decreases the amount of LD within duplicate blocks. This very weak effect is expected because IGC breaks linkage within blocks by transferring new variants from the other duplicate. High IGC does not break this linkage since variants are always exchanged between duplicates on the same chromosome (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>, row 1). However, when crossover is active, the effectiveness of IGC in breaking LD blocks increases dramatically and correlates positively with IGC rate and crossover rates (data not shown for different crossover rates) (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>, row 2).</p></div><div id="sec-12" class="subsection"><h3>WRC model: whole-region crossover</h3><p id="p-31">Because it is likely that crossover is not restricted to the single-copy region between duplications, we explore the interplay between IGC and crossover by allowing crossover to extend over the whole simulated region. Allowing crossover to overlap with regions subject to IGC events decreases the expected within-block variation for all IGC and crossover rates (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>). Points for the WRC model fall on curves corresponding to <em>R</em>’ = (2/3)<em>R</em>. That is, given that we are simulating three blocks of equal length, allowing crossover to occur on the whole region has an effect on variation equivalent to decreasing the crossover rate by one third under the SCC model (for a detailed explanation, see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012435/-/DC1/FileS6.pdf">File S6</a>). This effect is observed for variation within blocks (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5A</a>), variation between blocks (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5B</a>), and LD between duplicates (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5C</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/8/1479/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison between data from simulations under the SCC model and the WRC model. Results are shown for (A) &#x3C0;wC, (B) &#x3C0;sA, and (C) E(Dsum). Continuous lines are based on the SCC model and are shown for different crossover rates (R = 10, 50, and 100). As expected, results from SCC simulations are in very good agreement with theoretical expectations. Results from WRC simulations agree with theoretical expectations (discontinuous lines) for R&#x2019; = (2/3)R, showing that allowing crossover to occur in the duplicate regions has an effect identical to that of effectively reducing the crossover rate by one third on the SCC model. Although we have not implemented any IGC rate dependence on sequence similarity between duplicates, according to Walsh (1987) IGC rates C &gt; 0.2 would ensure the prevalence of stable concerted evolution at least temporarily in the face of genetic drift (see File S1). The shaded area indicates the region that lies beyond this threshold, where both theoretical predictions and results from our simulations might not be biologically realistic." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-701161318" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison between data from simulations under the SCC model and the WRC model. Results are shown for (A) &#x3C0;&lt;sub&gt;w&lt;/sub&gt;&lt;sup&gt;C&lt;/sup&gt;, (B) &#x3C0;&lt;sub&gt;s&lt;/sub&gt;&lt;sup&gt;A&lt;/sup&gt;, and (C) E(D&lt;sub&gt;sum&lt;/sub&gt;). Continuous lines are based on the SCC model and are shown for different crossover rates (&lt;em&gt;R&lt;/em&gt; = 10, 50, and 100). As expected, results from SCC simulations are in very good agreement with theoretical expectations. Results from WRC simulations agree with theoretical expectations (discontinuous lines) for &lt;em&gt;R&lt;/em&gt;&#x2019; = (2/3)&lt;em&gt;R&lt;/em&gt;, showing that allowing crossover to occur in the duplicate regions has an effect identical to that of effectively reducing the crossover rate by one third on the SCC model. Although we have not implemented any IGC rate dependence on sequence similarity between duplicates, according to Walsh (1987) IGC rates &lt;em&gt;C&lt;/em&gt; &gt; 0.2 would ensure the prevalence of stable concerted evolution at least temporarily in the face of genetic drift (see File S1). The shaded area indicates the region that lies beyond this threshold, where both theoretical predictions and results from our simulations might not be biologically realistic.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/8/1479/F5.medium.gif" width="440" height="140"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/8/1479/F5.medium.gif" width="440" height="140"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/8/1479/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/8/1479/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464665" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-32" class="first-child">Comparison between data from simulations under the SCC model and the WRC model. Results are shown for (A) π<sub>w</sub><sup>C</sup>, (B) π<sub>s</sub><sup>A</sup>, and (C) E(D<sub>sum</sub>). Continuous lines are based on the SCC model and are shown for different crossover rates (<em>R</em> = 10, 50, and 100). As expected, results from SCC simulations are in very good agreement with theoretical expectations. Results from WRC simulations agree with theoretical expectations (discontinuous lines) for <em>R</em>’ = (2/3)<em>R</em>, showing that allowing crossover to occur in the duplicate regions has an effect identical to that of effectively reducing the crossover rate by one third on the SCC model. Although we have not implemented any IGC rate dependence on sequence similarity between duplicates, according to <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Walsh (1987)</a> IGC rates <em>C</em> &gt; 0.2 would ensure the prevalence of stable concerted evolution at least temporarily in the face of genetic drift (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012435/-/DC1/FileS1.pdf">File S1</a>). The shaded area indicates the region that lies beyond this threshold, where both theoretical predictions and results from our simulations might not be biologically realistic.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">Row 3 of <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a> shows the pattern of LD under the WRC model. As expected, LD blocks present within the original and duplicated blocks under the SCC model disappear when allowing crossover to occur within duplicate blocks. As the rate of IGC increases, LD within duplicate blocks decreases, while LD levels and patterns remain constant within the single-copy block, where no IGC is acting. In accordance with the reduction of the strength of crossover depicted in <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a>, there is an increase of LD between duplicates under the WRC model compared to the SCC model.</p></div><div id="sec-13" class="subsection"><h3>HSC model: hotspot crossover</h3><p id="p-34">Finally, the HSC model considers that crossover occurs in short and specific segments of the sequence. We first explore the effect of the presence of a crossover hotspot in one of the copies (in this case, the original copy). We show that if the hotspot is located toward the left of the whole simulated region (this is, furthest away from the single-copy region in our model) the average variation within the whole original block is significantly reduced compared with that found under the SCC model (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). In contrast, if the hotspot is located adjacent to the single-copy region, variation is reduced to a much lesser extent. Simulations with centered hotspots reach intermediate levels of variation. Minimum and maximum variation are achieved for <em>R</em> = 0 and under the SCC model, respectively. The average variation corresponding to any hotspot location falls between these two extreme values for a given IGC rate.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/8/1479/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Variation within a duplicate block under different crossover conditions. The plot shows variation within duplicates under the HSC model. Each point corresponds to the average equilibrium value over 1000 simulation runs. Columns indicate different hotspot locations on the original block (illustrated by the red lines in the diagram). Expected values (&#x3C0;wC) for R = 0 and for the SCC model (R = 10) are shown to the left and to the right of the plot respectively. Regardless of the width and localization of the hotspot, variation within duplicates is decreased in comparison with the SCC model but increased with respect to R = 0. Hotspots located the furthest away from the duplicated block (to the left) have the strongest effect in lowering the amount of variation, while those localized closest to the duplicated block still lower the variation but to a lesser degree (to the right). Hotspots centered in the original block have an intermediate effect." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-701161318" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Variation within a duplicate block under different crossover conditions. The plot shows variation within duplicates under the HSC model. Each point corresponds to the average equilibrium value over 1000 simulation runs. Columns indicate different hotspot locations on the original block (illustrated by the red lines in the diagram). Expected values (&#x3C0;&lt;sub&gt;w&lt;/sub&gt;&lt;sup&gt;C&lt;/sup&gt;) for &lt;em&gt;R&lt;/em&gt; = 0 and for the SCC model (&lt;em&gt;R&lt;/em&gt; = 10) are shown to the left and to the right of the plot respectively. Regardless of the width and localization of the hotspot, variation within duplicates is decreased in comparison with the SCC model but increased with respect to &lt;em&gt;R&lt;/em&gt; = 0. Hotspots located the furthest away from the duplicated block (to the left) have the strongest effect in lowering the amount of variation, while those localized closest to the duplicated block still lower the variation but to a lesser degree (to the right). Hotspots centered in the original block have an intermediate effect.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/8/1479/F6.medium.gif" width="440" height="305"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/4/8/1479/F6.medium.gif" width="440" height="305"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/8/1479/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/8/1479/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464722" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-35" class="first-child">Variation within a duplicate block under different crossover conditions. The plot shows variation within duplicates under the HSC model. Each point corresponds to the average equilibrium value over 1000 simulation runs. Columns indicate different hotspot locations on the original block (illustrated by the red lines in the diagram). Expected values (π<sub>w</sub><sup>C</sup>) for <em>R</em> = 0 and for the SCC model (<em>R</em> = 10) are shown to the left and to the right of the plot respectively. Regardless of the width and localization of the hotspot, variation within duplicates is decreased in comparison with the SCC model but increased with respect to <em>R</em> = 0. Hotspots located the furthest away from the duplicated block (to the left) have the strongest effect in lowering the amount of variation, while those localized closest to the duplicated block still lower the variation but to a lesser degree (to the right). Hotspots centered in the original block have an intermediate effect.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">To better understand the reasons behind this decrease in variation, we calculated nucleotide variation in 1-kb bins and analyzed results for several hotspot locations. Results are depicted in <a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7</a>. We observe that IGC between paralogous regions makes the variation pattern identical for original and duplicated blocks. We also observe that the HSC model affects variation levels on each bin depending on their position relative to the hotspot. Bins to the left of the hotspot have the same levels of variation as they have under the SCC model; bins to the right of the hotspot have the same level of variation as they have under null crossover; and bins within the hotspot have intermediate values of variation between these two extremes. In the latter case, the fall in variation is gradual (from left to right) and the variation on each bin within the hotspot depends on the hotspot length and the position of the bin within the hotspot. These results show that hotspot location within regions undergoing constant IGC can affect levels of variation within duplicate regions.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/8/1479/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of variation along the sequence under different crossover conditions. Comparison between variation along the sequence when R = 0 (top row) and when crossover occurs (R = 10) on different conditions. Red rectangles indicate the regions undergoing crossover. In the bottom, variation along the sequence on SCC model is shown. Plots in the middle show different HSC model cases (different locations and lengths of the crossover hotspot region). Circles correspond to average pairwise differences calculated by bins within the original, single-copy and duplicated blocks, respectively. Each block is divided into five bins. Bins to the left of the hotspot have an amount of variation similar to that found under the SCC model whereas those to the right have a variation level equivalent to that of a model with no crossover (R = 0). Bins within the hotspot have intermediate levels of variation, which are lower for bins that are closer to the single-copy region. Additionally, original and duplicated blocks have identical (non-symmetrical) patterns of variation within them. This figure is for C = 0.5. Equivalent results are attained for greater values of C." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-701161318" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of variation along the sequence under different crossover conditions. Comparison between variation along the sequence when &lt;em&gt;R&lt;/em&gt; = 0 (top row) and when crossover occurs (&lt;em&gt;R&lt;/em&gt; = 10) on different conditions. Red rectangles indicate the regions undergoing crossover. In the bottom, variation along the sequence on SCC model is shown. Plots in the middle show different HSC model cases (different locations and lengths of the crossover hotspot region). Circles correspond to average pairwise differences calculated by bins within the original, single-copy and duplicated blocks, respectively. Each block is divided into five bins. Bins to the left of the hotspot have an amount of variation similar to that found under the SCC model whereas those to the right have a variation level equivalent to that of a model with no crossover (&lt;em&gt;R&lt;/em&gt; = 0). Bins within the hotspot have intermediate levels of variation, which are lower for bins that are closer to the single-copy region. Additionally, original and duplicated blocks have identical (non-symmetrical) patterns of variation within them. This figure is for &lt;em&gt;C&lt;/em&gt; = 0.5. Equivalent results are attained for greater values of &lt;em&gt;C&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/8/1479/F7.medium.gif" width="306" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/4/8/1479/F7.medium.gif" width="306" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/8/1479/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/8/1479/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464647" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-37" class="first-child">Distribution of variation along the sequence under different crossover conditions. Comparison between variation along the sequence when <em>R</em> = 0 (top row) and when crossover occurs (<em>R</em> = 10) on different conditions. Red rectangles indicate the regions undergoing crossover. In the bottom, variation along the sequence on SCC model is shown. Plots in the middle show different HSC model cases (different locations and lengths of the crossover hotspot region). Circles correspond to average pairwise differences calculated by bins within the original, single-copy and duplicated blocks, respectively. Each block is divided into five bins. Bins to the left of the hotspot have an amount of variation similar to that found under the SCC model whereas those to the right have a variation level equivalent to that of a model with no crossover (<em>R</em> = 0). Bins within the hotspot have intermediate levels of variation, which are lower for bins that are closer to the single-copy region. Additionally, original and duplicated blocks have identical (non-symmetrical) patterns of variation within them. This figure is for <em>C</em> = 0.5. Equivalent results are attained for greater values of <em>C</em>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-38">Given the evidence that crossover hotspot location is dependent on sequence motifs, at least in the case of humans (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Myers <em>et al.</em> 2008</a>), it would make sense for hotspots to be present in both duplicates. Since the effect of crossover is symmetrical with respect to the center of the simulated region while IGC acts between paralogous regions, the decrease in variation within duplicate blocks observed under the HSC model with one hotspot is not maintained with two hotspots (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012435/-/DC1/FigureS6.pdf">Figure S6</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012435/-/DC1/FigureS7.pdf">Figure S7</a>).</p><p id="p-39"><a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012435/-/DC1/FigureS8.pdf">Figure S8</a> show LD patterns (D′ and r<sup>2</sup>, respectively) under the HSC model with one and two hotspots for different IGC rates. As expected, if <em>C</em> = 0, highly identifiable LD blocks form to the left and right of the hotspot or hotspots. Contrary to the SCC model, LD blocks extend from the original to the duplicated block. For <em>C</em> &gt; 0 and a single hotspot, a complex pattern appears driven by the breakdown of LD blocks within duplicates. As we have observed for the SCC model and the WRC model, LD blocks within duplicates break down strongly with IGC only if crossover is active between them. Under the HSC model with a single hotspot, LD breaks down to the left of the hotspot and in the corresponding paralogous region in the duplicated block; this breakdown is unnoticeable, although present, to the right of the hotspot for <em>C</em> = 1. LD between duplicates is greater to the right of the hotspot location than to the left of the hotspot location since crossover is acting between paralogous regions only to the left of the hotspot.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/8/1479/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Comparison of LD between scenarios with one or two crossover hotspots. Average values for 1000 simulations are shown. LD between each pair of windows along the sequence (D&#x2032;) is coded with a number between 0 and 1 and represented with a color (from white to dark blue to light green). Three different IGC rates (0, 1, and 50) are represented in columns. Rows show the effect of crossover (R = 50) located in one specific region of the original block or in both the original and duplicated blocks (in paralogous regions). The red line below the names identifying each block shows regions undergoing crossover. In the first column, the effect of crossover delimiting LD blocks is clear. When IGC is active (in the second and third column), a complex pattern of LD appears along the sequence together with the dark blue diagonal line representing LD between duplicate regions (stronger when no crossover is acting between paralogous windows of the duplicate blocks). With the presence of one hotspot on the original block, there are paralogous windows to the right of each duplicate between which there is no crossover and, thus, IGC has lower power to break LD in these fragments. In the case of two hotspots, this situation disappears and the combination of crossover and IGC breaks LD within duplicate blocks." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-701161318" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Comparison of LD between scenarios with one or two crossover hotspots. Average values for 1000 simulations are shown. LD between each pair of windows along the sequence (D&#x2032;) is coded with a number between 0 and 1 and represented with a color (from white to dark blue to light green). Three different IGC rates (0, 1, and 50) are represented in columns. Rows show the effect of crossover (&lt;em&gt;R&lt;/em&gt; = 50) located in one specific region of the original block or in both the original and duplicated blocks (in paralogous regions). The red line below the names identifying each block shows regions undergoing crossover. In the first column, the effect of crossover delimiting LD blocks is clear. When IGC is active (in the second and third column), a complex pattern of LD appears along the sequence together with the dark blue diagonal line representing LD between duplicate regions (stronger when no crossover is acting between paralogous windows of the duplicate blocks). With the presence of one hotspot on the original block, there are paralogous windows to the right of each duplicate between which there is no crossover and, thus, IGC has lower power to break LD in these fragments. In the case of two hotspots, this situation disappears and the combination of crossover and IGC breaks LD within duplicate blocks.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/8/1479/F8.medium.gif" width="440" height="269"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/4/8/1479/F8.medium.gif" width="440" height="269"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/8/1479/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/8/1479/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464800" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-40" class="first-child">Comparison of LD between scenarios with one or two crossover hotspots. Average values for 1000 simulations are shown. LD between each pair of windows along the sequence (D′) is coded with a number between 0 and 1 and represented with a color (from white to dark blue to light green). Three different IGC rates (0, 1, and 50) are represented in columns. Rows show the effect of crossover (<em>R</em> = 50) located in one specific region of the original block or in both the original and duplicated blocks (in paralogous regions). The red line below the names identifying each block shows regions undergoing crossover. In the first column, the effect of crossover delimiting LD blocks is clear. When IGC is active (in the second and third column), a complex pattern of LD appears along the sequence together with the dark blue diagonal line representing LD between duplicate regions (stronger when no crossover is acting between paralogous windows of the duplicate blocks). With the presence of one hotspot on the original block, there are paralogous windows to the right of each duplicate between which there is no crossover and, thus, IGC has lower power to break LD in these fragments. In the case of two hotspots, this situation disappears and the combination of crossover and IGC breaks LD within duplicate blocks.</p><div class="sb-div caption-clear"></div></div></div><p id="p-41">Under the HSC model with two hotspots, crossover generates a symmetrical pattern of LD blocks in both duplicates. As a consequence, when IGC is active between paralogous regions, it breaks down LD not only in the external regions with respect to the hotspots but also in their corresponding paralogous (internal) regions, resulting in a complete breakdown of the LD along the duplicated blocks.</p></div><div id="sec-14" class="subsection"><h3>Overview</h3><p id="p-42">We have described the effects of IGC under different crossover models and highlighted the importance of the distribution of crossover junctions in the attainment of variation within and between duplicates. As we have shown, crossover between paralogous regions provides the opportunity for IGC to effectively transfer new variants between paralogous regions and for IGC to break LD within duplicates more efficiently. If crossover is not acting between copies, variation will be lower and LD will be higher within paralogous regions for the same IGC rates. In the case of the HSC model, we can find both situations in the same duplicated region, creating a complex pattern of nucleotide variation and LD along the duplicates. We have summarized the different possible scenarios in <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a>.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464661/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464661/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Summary of variation and LD measures under different IGC and crossover conditions</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-44">The complexity behind the interplay between IGC and crossover was already evident from theoretical models of the effect IGC and crossover on levels of variation within and between duplicates under neutrality (<a id="xref-ref-44-3" class="xref-bibr" href="#ref-44">Ohta 1982</a>, <a id="xref-ref-45-8" class="xref-bibr" href="#ref-45">1983</a>; <a id="xref-ref-40-4" class="xref-bibr" href="#ref-40">Nagylaki 1984</a>; <a id="xref-ref-20-7" class="xref-bibr" href="#ref-20">Innan 2002</a>, <a id="xref-ref-21-9" class="xref-bibr" href="#ref-21">2003</a>). However, extending its application not only to duplicated genes but also to SDs, which can span large areas of the genome and are not necessarily in tandem, demands the incorporation of more realistic crossover models. To tackle SD evolution, forward-time simulators promise to be an efficient tool for intense exploration of broad ranges of parameter values. Here, we have presented a first glimpse at what forward-time simulations can offer, limiting our analysis to neutral evolution and thus, providing a null-model for future scans of areas under selective pressure within SDs.</p><p id="p-45">Increased variation within duplicates undergoing IGC is an accepted phenomenon (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Baltimore 1981</a>; <a id="xref-ref-44-4" class="xref-bibr" href="#ref-44">Ohta 1982</a>, <a id="xref-ref-45-9" class="xref-bibr" href="#ref-45">1983</a>; <a id="xref-ref-40-5" class="xref-bibr" href="#ref-40">Nagylaki 1984</a>; <a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Walsh 1987</a>; <a id="xref-ref-20-8" class="xref-bibr" href="#ref-20">Innan 2002</a>, <a id="xref-ref-21-10" class="xref-bibr" href="#ref-21">2003</a>; <a id="xref-ref-57-7" class="xref-bibr" href="#ref-57">Thornton 2007</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Ohta 2010</a>; <a id="xref-ref-56-2" class="xref-bibr" href="#ref-56">Teshima and Innan 2012</a>) and it has been observed in humans (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Bosch <em>et al.</em> 2004</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Hallast <em>et al.</em> 2005</a>) as well as in other species (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Nielsen <em>et al.</em> 2003</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Rane <em>et al.</em> 2010</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Willett 2013</a>). In this study, we have shown that the extent to which variation within duplicates is increased due to IGC is highly dependent upon the distribution of crossover junctions. In the case of crossover hotspots, if they happen to fall inside duplications, the distribution of variation within these regions might be altered, with a strong dependence on the hotspot location. This complex pattern of variation might affect the fate of SDs and could be important to take into account when calculating IGC rates. The implications of increased variation on the evolutionary fate of SDs are largely unexplored. For instance, high variation might increase the possibility of fixation of compensatory mutations (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Plotnikova <em>et al.</em> 2007</a>) and afford more opportunities for natural selection to act upon standing variation (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Katju <em>et al.</em> 2008</a>). This will not be independent of the underlying recombination scenario under which certain levels of variability are achieved.</p><p id="p-46">Abundant IGC between duplicates may facilitate parallel selective sweeps if increased gene dosage is positively selected (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Hanikenne <em>et al.</em> 2013</a>). IGC is additionally thought to be one of the mechanisms responsible for the elimination of deleterious mutations in polyploid asexual genetic systems (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Khakhlova and Bock 2006</a>) and in the human Y chromosome (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Rozen <em>et al.</em> 2003</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Marais <em>et al.</em> 2010</a>), allowing them to escape Muller’s ratchet. Alternatively, deleterious effects of IGC also have been reported; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Casola <em>et al.</em> (2012)</a> demonstrated that the introduction of deleterious alleles via IGC has happened in at least 1% of human genes and identified thousands of potentially deleterious mutations that could be disease-causing if they were to have the same fate.</p><p id="p-47">Another interesting result concerns LD. We have observed that the relationship between LD and IGC rate is nonlinear because both very low and very high IGC rates will produce high LD within duplicates, whereas intermediate IGC rates produce intermediate levels of LD. This effect cannot be observed if crossover is active between duplicates because the interplay between IGC and crossover drastically increases the breakdown of LD within duplicates. We have also analyzed the levels of LD between duplicates and found that they have a positive correlation with IGC rates and a negative correlation with crossover rates.</p><p id="p-48">Gene conversion is already known to affect patterns of LD and is considered to be the main cause of LD breakdown over short distances (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Andolfatto and Nordborg 1998</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ardlie <em>et al.</em> 2001</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Frisse <em>et al.</em> 2001</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Plagnol <em>et al.</em> 2006</a>). However, this observation is restricted to gene conversion within the same loci on different chromosomes, referred to as intralocus gene conversion, interallelic gene conversion, or simply gene conversion. It has been suggested that similar molecular mechanisms might be behind both intralocus and ICG (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Jeffreys and May 2004</a>). However, even if a similar mechanism could be illustrated (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Hastings 2010</a>), it does not imply that their effects on variation or on patterns of LD should be equal. In this work, we have exclusively dealt with gene conversion between different loci (IGC) and have analyzed, through simulations, the effect of IGC on LD patterns (D’ and r<sup>2</sup>) not only within but also between duplicates. Patterns of LD inside gene families have received some attention (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">Hallast <em>et al.</em> 2005</a>), but the causes for the appearance and maintenance of these patterns have not been explored thoroughly. We have demonstrated that in the case of IGC, the interplay with crossover is crucial to determine the levels of LD. We stressed the importance of differentiating between LD within and LD between duplicates because they do not respond in the same way to the action of IGC and crossover and might prove to be useful in ascertaining biases such as donor-acceptor bias.</p><p id="p-49">Under neutrality, there are at least two possible mechanisms by means of which different levels of variation and LD can be attained along a duplicated sequence: nonhomogeneous distribution of IGC events and nonhomogeneous distribution of crossover junctions. IGC rates are known to be dependent on sequence similarity between paralogous regions. We suspect that setting minimum identity thresholds for IGC to occur (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012435/-/DC1/FileS1.pdf">File S1</a>) may cause patterning inside SDs. Our simulator incorporates the possibility of setting these thresholds but the effects of these are beyond the scope of this paper. Patterning could also be caused by the nonhomogeneous distribution of crossover junctions that we have explored here. All crossover models share a common characteristic: given a homogeneous IGC rate, crossover is effective in increasing variation and in breaking LD within duplicates only if it is acting between paralogous regions. Of course, crossover does not play an “active” role in these effects; it only sets the ground for IGC to cause them.</p><p id="p-50">The presence of crossover hotspots within SDs is of special relevance given the asymmetrical patterns it can generate within them. If a crossover hotspot is located inside a copy of the duplication but not in its paralog, increased variation might only be found in the region that is separated from its paralog by the hotspot and not in the rest of the SD. This asymmetry might tend to disappear via IGC (if it were caused by a sequence motif), either by the elimination of the hotspot or its transfer to the duplicate. For a short period of time, however, this asymmetry might by decisive in determining the evolutionary fate of genes located in different regions within the SDs.</p><p id="p-51">The analysis that we presented here covers a broad range of IGC rates that spans the observed ranges in a wide variety of species. The applicability of our results to real data depends on the extent to which the measurement of IGC rates between duplicates is accurate. As evidenced by <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Mansai and Innan (2010)</a> some methods to detect IGC rates, such as GENECONV (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Sawyer 1989</a>), can underestimate IGC rates if they are very high. New methods are still being developed (<em>e.g.</em>, <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dumont and Eichler 2013</a>). We believe that for tandem duplications, the LD measurement method that we have implemented here would be able to detect rates of IGC between duplicates, especially if they are high. A formal comparison between our method and others would be an interesting focus of further research.</p><p id="p-52">Given the relevance of IGC between SDs in primate and human recent evolution (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Fawcett and Innan 2013</a>) and their likely role in adaptation (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Bailey and Eichler 2006</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Han <em>et al.</em> 2009</a>) we have provided a testing ground by analyzing the neutral scenario. Exploring the effects of natural selection on the evolution of duplicated regions of the genome would be a natural next step and an important and surely interesting endeavor.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-53">We acknowledge Urko M. Marigorta, Gabriel Santpere, Javier Prado-Martinez, Juan A. Rodríguez, Nino Spataro, Pierre Luisi, Txema Heredia, and David A. Hughes for technical assistance, valuable discussion, and smart insights. We also thank Hideki Innan and two anonymous reviewers for helpful comments and suggestions. This work has been supported by the Spanish National Institute of Bioinfomatics, a platform of the Instituto de Salud Carlos III (PT13/0001/0026), and the Spanish Government, Grant BFU2012-38236 to A.N.; by grants to D.H. from Conacyt and CSIC (Predoctoral JAE grant); and by the Fondo Europeo de Desarrollo Regional (FEDER) and the Fondo Social Europeo (FSE).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012435/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.012435/-/DC1</a></p></li><li class="fn" id="fn-4"><p id="p-54"><em>Communicating editor: Y. Kim</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-03-14"><span class="received-label">Received </span>March 14, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-06-03"><span class="accepted-label">Accepted </span>June 3, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Hartasánchez <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The effect of gene conversion on intralocus associations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">148</span>: <span class="cit-fpage">1397</span>–<span class="cit-lpage">1399</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAndolfatto%26rft.auinit1%253DP.%26rft.volume%253D148%26rft.issue%253D3%26rft.spage%253D1397%26rft.epage%253D1399%26rft.atitle%253DThe%2BEffect%2Bof%2BGene%2BConversion%2Bon%2BIntralocus%2BAssociations%26rft_id%253Dinfo%253Apmid%252F9539452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0OC8zLzEzOTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzgvMTQ3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.2" data-doi="10.1086/323251"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ardlie</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu-Cordero</span>  <span class="cit-name-given-names">S. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eberle</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Daly</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Lower-than-expected linkage disequilibrium between tightly linked markers in humans suggests a role for gene conversion.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">582</span>–<span class="cit-lpage">589</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DArdlie%26rft.auinit1%253DK.%26rft.volume%253D69%26rft.issue%253D3%26rft.spage%253D582%26rft.epage%253D589%26rft.atitle%253DLower-than-expected%2Blinkage%2Bdisequilibrium%2Bbetween%2Btightly%2Blinked%2Bmarkers%2Bin%2Bhumans%2Bsuggests%2Ba%2Brole%2Bfor%2Bgene%2Bconversion.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F323251%26rft_id%253Dinfo%253Apmid%252F11473344%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/323251&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11473344&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170271400010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.3" data-doi="10.1038/nrg1895"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eichler</span>  <span class="cit-name-given-names">E. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Primate segmental duplications: crucibles of evolution, diversity and disease.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">552</span>–<span class="cit-lpage">564</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DBailey%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253D552%26rft.epage%253D564%26rft.atitle%253DPrimate%2Bsegmental%2Bduplications%253A%2Bcrucibles%2Bof%2Bevolution%252C%2Bdiversity%2Band%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg1895%26rft_id%253Dinfo%253Apmid%252F16770338%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg1895&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16770338&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000238377200016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.4" data-doi="10.1126/science.1072047"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gu</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clark</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reinert</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samonte</span>  <span class="cit-name-given-names">R. V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Recent segmental duplications in the human genome.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">297</span>: <span class="cit-fpage">1003</span>–<span class="cit-lpage">1007</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBailey%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D297%26rft.issue%253D5583%26rft.spage%253D1003%26rft.epage%253D1007%26rft.atitle%253DRecent%2BSegmental%2BDuplications%2Bin%2Bthe%2BHuman%2BGenome%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1072047%26rft_id%253Dinfo%253Apmid%252F12169732%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTcvNTU4My8xMDAzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzE0NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.5" data-doi="10.1016/0092-8674(81)90082-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baltimore</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1981</span> <span class="cit-article-title">Gene conversion: some implications for immunoglobulin genes.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">592</span>–<span class="cit-lpage">594</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBaltimore%26rft.auinit1%253DD.%26rft.volume%253D24%26rft.issue%253D3%26rft.spage%253D592%26rft.epage%253D594%26rft.atitle%253DGene%2Bconversion%253A%2Bsome%2Bimplications%2Bfor%2Bimmunoglobulin%2Bgenes.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252881%252990082-9%26rft_id%253Dinfo%253Apmid%252F7018693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(81)90082-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7018693&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1981MG29600001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.6" data-doi="10.1016/j.ygeno.2008.09.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Benovoy</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Drouin</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Ectopic gene conversions in the human genome.</span> <abbr class="cit-jnl-abbrev">Genomics</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">27</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenomics%26rft.stitle%253DGenomics%26rft.aulast%253DBenovoy%26rft.auinit1%253DD.%26rft.volume%253D93%26rft.issue%253D1%26rft.spage%253D27%26rft.epage%253D32%26rft.atitle%253DEctopic%2Bgene%2Bconversions%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ygeno.2008.09.007%26rft_id%253Dinfo%253Apmid%252F18848875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ygeno.2008.09.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18848875&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000262492000007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.7" data-doi="10.1101/gr.2177404"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bosch</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hurles</span>  <span class="cit-name-given-names">M. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navarro</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jobling</span>  <span class="cit-name-given-names">M. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Dynamics of a human interparalog gene conversion hotspot.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">835</span>–<span class="cit-lpage">844</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DBosch%26rft.auinit1%253DE.%26rft.volume%253D14%26rft.issue%253D5%26rft.spage%253D835%26rft.epage%253D844%26rft.atitle%253DDynamics%2Bof%2Ba%2BHuman%2BInterparalog%2BGene%2BConversion%2BHotspot%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.2177404%26rft_id%253Dinfo%253Apmid%252F15123583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE0LzUvODM1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzE0NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.8" data-doi="10.1101/gr.127738.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Casola</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zekonyte</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Phillips</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">D. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Interlocus gene conversion events introduce deleterious mutations into at least 1% of human genes associated with inherited disease.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">429</span>–<span class="cit-lpage">435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.127738.111%26rft_id%253Dinfo%253Apmid%252F22090377%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIyLzMvNDI5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzE0NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.9" data-doi="10.1038/nrg2193"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">D. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chuzhanova</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Férec</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Patrinos</span>  <span class="cit-name-given-names">G. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Gene conversion: mechanisms, evolution and human disease.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">762</span>–<span class="cit-lpage">775</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DChen%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253D762%26rft.epage%253D775%26rft.atitle%253DGene%2Bconversion%253A%2Bmechanisms%252C%2Bevolution%2Band%2Bhuman%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg2193%26rft_id%253Dinfo%253Apmid%252F17846636%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg2193&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17846636&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.10" data-doi="10.1038/nature08516"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Conrad</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pinto</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Redon</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feuk</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gokcumen</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Origins and functional impact of copy number variation in the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">464</span>: <span class="cit-fpage">704</span>–<span class="cit-lpage">712</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DConrad%26rft.auinit1%253DD.%2BF.%26rft.volume%253D464%26rft.issue%253D7289%26rft.spage%253D704%26rft.epage%253D712%26rft.atitle%253DOrigins%2Band%2Bfunctional%2Bimpact%2Bof%2Bcopy%2Bnumber%2Bvariation%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08516%26rft_id%253Dinfo%253Apmid%252F19812545%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08516&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19812545&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276205000035&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.11" data-doi="10.1038/299111a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dover</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Molecular drive: a cohesive mode of species evolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">299</span>: <span class="cit-fpage">111</span>–<span class="cit-lpage">117</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDover%26rft.auinit1%253DG.%26rft.volume%253D299%26rft.issue%253D5879%26rft.spage%253D111%26rft.epage%253D117%26rft.atitle%253DMolecular%2Bdrive%253A%2Ba%2Bcohesive%2Bmode%2Bof%2Bspecies%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F299111a0%26rft_id%253Dinfo%253Apmid%252F7110332%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/299111a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7110332&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1982PF45500025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.12" data-doi="10.1371/journal.pone.0075949"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dumont</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eichler</span>  <span class="cit-name-given-names">E. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Signals of historical interlocus gene conversion in human segmental duplications.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">e75949</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BONE%26rft.volume%253D8%26rft.spage%253De75949%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0075949%26rft_id%253Dinfo%253Apmid%252F24124524%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0075949&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24124524&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.13" data-doi="10.1016/j.tig.2013.07.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fawcett</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Innan</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The role of gene conversion in preserving rearrangement hotspots in the human genome.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">561</span>–<span class="cit-lpage">568</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2BGenet.%26rft.volume%253D29%26rft.spage%253D561%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2013.07.002%26rft_id%253Dinfo%253Apmid%252F23953668%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2013.07.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23953668&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000325300800004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.14" data-doi="10.1086/323612"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frisse</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">R. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartoszewicz</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donfack</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Gene conversion and different population histories may explain the contrast between polymorphism and linkage disequilibrium.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">831</span>–<span class="cit-lpage">843</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DFrisse%26rft.auinit1%253DL.%26rft.volume%253D69%26rft.issue%253D4%26rft.spage%253D831%26rft.epage%253D843%26rft.atitle%253DGene%2Bconversion%2Band%2Bdifferent%2Bpopulation%2Bhistories%2Bmay%2Bexplain%2Bthe%2Bcontrast%2Bbetween%2Bpolymorphism%2Band%2Blinkage%2Bdisequilibrium%2Blevels.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F323612%26rft_id%253Dinfo%253Apmid%252F11533915%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/323612&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11533915&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171008200015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.15" data-doi="10.1101/gr.117242.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gazave</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Darré</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morcillo-Suarez</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petit-Marty</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carreño</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Copy number variation analysis in the great apes reveals species-specific patterns of structural variation.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1626</span>–<span class="cit-lpage">1639</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.117242.110%26rft_id%253Dinfo%253Apmid%252F21824994%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyMS8xMC8xNjI2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzE0NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.16" data-doi="10.1101/gr.4270505"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hallast</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nagirnaja</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Margus</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laan</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Segmental duplications and gene conversion: human luteinizing hormone/chorionic gonadotropin β gene cluster.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">1535</span>–<span class="cit-lpage">1546</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DHallast%26rft.auinit1%253DP.%26rft.volume%253D15%26rft.issue%253D11%26rft.spage%253D1535%26rft.epage%253D1546%26rft.atitle%253DSegmental%2Bduplications%2Band%2Bgene%2Bconversion%253A%2BHuman%2Bluteinizing%2Bhormone%252Fchorionic%2Bgonadotropin%2B%257Bbeta%257D%2Bgene%2Bcluster%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.4270505%26rft_id%253Dinfo%253Apmid%252F16251463%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIxNS8xMS8xNTM1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzE0NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.17" data-doi="10.1101/gr.085951.108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">M. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Demuth</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGrath</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casola</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Adaptive evolution of young gene duplicates in mammals.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">859</span>–<span class="cit-lpage">867</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DHan%26rft.auinit1%253DM.%2BV.%26rft.volume%253D19%26rft.issue%253D5%26rft.spage%253D859%26rft.epage%253D867%26rft.atitle%253DAdaptive%2Bevolution%2Bof%2Byoung%2Bgene%2Bduplicates%2Bin%2Bmammals%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.085951.108%26rft_id%253Dinfo%253Apmid%252F19411603%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE5LzUvODU5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzE0NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.18" data-doi="10.1371/journal.pgen.1003707"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hanikenne</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kroymann</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trampczynska</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bernal</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Motte</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Hard selective sweep and ectopic gene conversion in a gene cluster affording environmental adaptation.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003707</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1003707%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1003707%26rft_id%253Dinfo%253Apmid%252F23990800%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1003707&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23990800&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.19" data-doi="10.3390/genes1030427"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hastings</span>  <span class="cit-name-given-names">P. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Mechanisms of ectopic gene conversion.</span> <abbr class="cit-jnl-abbrev">Genes</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">427</span>–<span class="cit-lpage">439</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%26rft.volume%253D1%26rft.spage%253D427%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fgenes1030427%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/genes1030427&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.20"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Innan</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A method for estimating the mutation, gene conversion and recombination parameters in small multigene families.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">865</span>–<span class="cit-lpage">872</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DInnan%26rft.auinit1%253DH.%26rft.volume%253D161%26rft.issue%253D2%26rft.spage%253D865%26rft.epage%253D872%26rft.atitle%253DA%2BMethod%2Bfor%2BEstimating%2Bthe%2BMutation%252C%2BGene%2BConversion%2Band%2BRecombination%2BParameters%2Bin%2BSmall%2BMultigene%2BFamilies%26rft_id%253Dinfo%253Apmid%252F12072480%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYxLzIvODY1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzE0NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Innan</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The coalescent and infinite-site model of a small multigene family.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">163</span>: <span class="cit-fpage">803</span>–<span class="cit-lpage">810</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DInnan%26rft.auinit1%253DH.%26rft.volume%253D163%26rft.issue%253D2%26rft.spage%253D803%26rft.epage%253D810%26rft.atitle%253DThe%2BCoalescent%2Band%2BInfinite-Site%2BModel%2Bof%2Ba%2BSmall%2BMultigene%2BFamily%26rft_id%253Dinfo%253Apmid%252F12618415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYzLzIvODAzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzE0NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.22" data-doi="10.1007/s10709-009-9355-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Innan</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Population genetic models of duplicated genes.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">19</span>–<span class="cit-lpage">37</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D137%26rft.spage%253D19%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-009-9355-1%26rft_id%253Dinfo%253Apmid%252F19266289%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10709-009-9355-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19266289&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268326400003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.23" data-doi="10.1016/j.tig.2012.03.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iskow</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gokcumen</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Exploring the role of copy number variants in human adaptation.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">245</span>–<span class="cit-lpage">257</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DIskow%26rft.auinit1%253DR.%2BC.%26rft.volume%253D28%26rft.issue%253D6%26rft.spage%253D245%26rft.epage%253D257%26rft.atitle%253DExploring%2Bthe%2Brole%2Bof%2Bcopy%2Bnumber%2Bvariants%2Bin%2Bhuman%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2012.03.002%26rft_id%253Dinfo%253Apmid%252F22483647%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2012.03.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22483647&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000305094000001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.24" data-doi="10.1038/ng1287"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jeffreys</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">C. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Intense and highly localized gene conversion activity in human meiotic crossover hot spots.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DJeffreys%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D36%26rft.issue%253D2%26rft.spage%253D151%26rft.epage%253D156%26rft.atitle%253DIntense%2Band%2Bhighly%2Blocalized%2Bgene%2Bconversion%2Bactivity%2Bin%2Bhuman%2Bmeiotic%2Bcrossover%2Bhot%2Bspots.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1287%26rft_id%253Dinfo%253Apmid%252F14704667%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1287&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14704667&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000188542700016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.25" data-doi="10.1038/ng1001-217"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jeffreys</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kauppi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neumann</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Intensely punctate meiotic recombination in the class II region of the major histocompatibility complex.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">217</span>–<span class="cit-lpage">222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DJeffreys%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D29%26rft.issue%253D2%26rft.spage%253D217%26rft.epage%253D222%26rft.atitle%253DIntensely%2Bpunctate%2Bmeiotic%2Brecombination%2Bin%2Bthe%2Bclass%2BII%2Bregion%2Bof%2Bthe%2Bmajor%2Bhistocompatibility%2Bcomplex.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng1001-217%26rft_id%253Dinfo%253Apmid%252F11586303%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng1001-217&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11586303&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000171374100031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.26" data-doi="10.1534/genetics.108.090035"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Katju</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">LaBeau</span>  <span class="cit-name-given-names">E. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipinski</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergthorsson</span>  <span class="cit-name-given-names">U.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Sex change by gene conversion in <em>Caenorhabditis elegans fog-2</em> mutant.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">669</span>–<span class="cit-lpage">672</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.090035%26rft_id%253Dinfo%253Apmid%252F18757925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgwLzEvNjY5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzE0NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.27" data-doi="10.1111/j.1365-313X.2006.02673.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khakhlova</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bock</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Elimination of deleterious mutations in plastid genomes by gene conversion.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DKhakhlova%26rft.auinit1%253DO.%26rft.volume%253D46%26rft.issue%253D1%26rft.spage%253D85%26rft.epage%253D94%26rft.atitle%253DElimination%2Bof%2Bdeleterious%2Bmutations%2Bin%2Bplastid%2Bgenomes%2Bby%2Bgene%2Bconversion.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2006.02673.x%26rft_id%253Dinfo%253Apmid%252F16553897%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2006.02673.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16553897&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236035700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.28" data-doi="10.1073/pnas.77.1.522"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Average time until fixation of a mutant allele in a finite population under continued mutation pressure: studies by analytical, numerical, and pseudo-sampling methods.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">522</span>–<span class="cit-lpage">526</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DKimura%26rft.auinit1%253DM.%26rft.volume%253D77%26rft.issue%253D1%26rft.spage%253D522%26rft.epage%253D526%26rft.atitle%253DAverage%2Btime%2Buntil%2Bfixation%2Bof%2Ba%2Bmutant%2Ballele%2Bin%2Ba%2Bfinite%2Bpopulation%2Bunder%2Bcontinued%2Bmutation%2Bpressure%253A%2BStudies%2Bby%2Banalytical%252C%2Bnumerical%252C%2Band%2Bpseudo-sampling%2Bmethods%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.77.1.522%26rft_id%253Dinfo%253Apmid%252F16592764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI3Ny8xLzUyMiI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOC8xNDc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.29" data-doi="10.1038/ng0302-241"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kong</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gudbjartsson</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sainz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jonsdottir</span>  <span class="cit-name-given-names">G. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gudjonsson</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A high-resolution recombination map of the human genome.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">241</span>–<span class="cit-lpage">247</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DKong%26rft.auinit1%253DA.%26rft.volume%253D31%26rft.issue%253D3%26rft.spage%253D241%26rft.epage%253D247%26rft.atitle%253DA%2Bhigh-resolution%2Brecombination%2Bmap%2Bof%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng0302-241%26rft_id%253Dinfo%253Apmid%252F12053178%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng0302-241&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12053178&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176494100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.30" data-doi="10.1186/gb-2013-14-1-r9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lorente-Galdos</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bleyhl</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Santpere</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vives</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ramírez</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Accelerated exon evolution within primate segmental duplications.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">R9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D14%26rft.spage%253DR9%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2013-14-1-r9%26rft_id%253Dinfo%253Apmid%252F23360670%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2013-14-1-r9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23360670&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.31" data-doi="10.1126/science.290.5494.1151"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lynch</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Conery</span>  <span class="cit-name-given-names">J. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The evolutionary fate and consequences of duplicate genes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">290</span>: <span class="cit-fpage">1151</span>–<span class="cit-lpage">1155</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DLynch%26rft.auinit1%253DM.%26rft.volume%253D290%26rft.issue%253D5494%26rft.spage%253D1151%26rft.epage%253D1155%26rft.atitle%253DThe%2BEvolutionary%2BFate%2Band%2BConsequences%2Bof%2BDuplicate%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.290.5494.1151%26rft_id%253Dinfo%253Apmid%252F11073452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTAvNTQ5NC8xMTUxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzE0NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.32" data-doi="10.1534/genetics.108.087676"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mano</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Innan</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">The evolutionary rate of duplicated genes under concerted evolution.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">493</span>–<span class="cit-lpage">505</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.087676%26rft_id%253Dinfo%253Apmid%252F18757936%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgwLzEvNDkzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzE0NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.33" data-doi="10.1534/genetics.109.111161"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mansai</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Innan</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The power of the methods for detecting interlocus gene conversion.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">184</span>: <span class="cit-fpage">517</span>–<span class="cit-lpage">527</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.111161%26rft_id%253Dinfo%253Apmid%252F19948889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg0LzIvNTE3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzE0NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.34" data-doi="10.3390/genes2020313"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mansai</span>  <span class="cit-name-given-names">S. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kado</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Innan</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The rate and tract length of gene conversion between duplicated genes.</span> <abbr class="cit-jnl-abbrev">Genes</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">313</span>–<span class="cit-lpage">331</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%26rft.volume%253D2%26rft.spage%253D313%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fgenes2020313%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/genes2020313&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.35" data-doi="10.1093/gbe/evq026"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marais</span>  <span class="cit-name-given-names">G. A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campos</span>  <span class="cit-name-given-names">P. R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gordo</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Can intra-Y gene conversion oppose the degeneration of the human Y chromosome? A simulation study.</span> <abbr class="cit-jnl-abbrev">Genome Biol. Evol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">347</span>–<span class="cit-lpage">357</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fgbe%252Fevq026%26rft_id%253Dinfo%253Apmid%252F20624739%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiZ2JlIjtzOjU6InJlc2lkIjtzOjc6IjIvMC8zNDciO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzgvMTQ3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.36" data-doi="10.1038/nature07744"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marques-Bonet</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kidd</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ventura</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graves</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">A burst of segmental duplications in the African great ape ancestor.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">457</span>: <span class="cit-fpage">877</span>–<span class="cit-lpage">881</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMarques-Bonet%26rft.auinit1%253DT.%26rft.volume%253D457%26rft.issue%253D7231%26rft.spage%253D877%26rft.epage%253D881%26rft.atitle%253DA%2Bburst%2Bof%2Bsegmental%2Bduplications%2Bin%2Bthe%2Bgenome%2Bof%2Bthe%2BAfrican%2Bgreat%2Bape%2Bancestor.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07744%26rft_id%253Dinfo%253Apmid%252F19212409%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07744&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19212409&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000263266700044&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.37" data-doi="10.1534/genetics.109.101428"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McGrath</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Casola</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hahn</span>  <span class="cit-name-given-names">M. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Minimal effect of ectopic gene conversion among recent duplicates in four mammalian genomes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">615</span>–<span class="cit-lpage">622</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101428%26rft_id%253Dinfo%253Apmid%252F19307604%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgyLzIvNjE1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzE0NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.38" data-doi="10.1038/nature09708"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mills</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walter</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stewart</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Handsaker</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mapping copy number variation by population-scale genome sequencing.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">470</span>: <span class="cit-fpage">59</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMills%26rft.auinit1%253DR.%2BE.%26rft.volume%253D470%26rft.issue%253D7332%26rft.spage%253D59%26rft.epage%253D65%26rft.atitle%253DMapping%2Bcopy%2Bnumber%2Bvariation%2Bby%2Bpopulation-scale%2Bgenome%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09708%26rft_id%253Dinfo%253Apmid%252F21293372%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09708&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21293372&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286886400033&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.39" data-doi="10.1038/ng.213"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freeman</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Auton</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Donnelly</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A common sequence motif associated with recombination hot spots and genome instability in humans.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">40</span>: <span class="cit-fpage">1124</span>–<span class="cit-lpage">1129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DMyers%26rft.auinit1%253DS.%26rft.volume%253D40%26rft.issue%253D9%26rft.spage%253D1124%26rft.epage%253D1129%26rft.atitle%253DA%2Bcommon%2Bsequence%2Bmotif%2Bassociated%2Bwith%2Brecombination%2Bhot%2Bspots%2Band%2Bgenome%2Binstability%2Bin%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.213%26rft_id%253Dinfo%253Apmid%252F19165926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.213&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19165926&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258761200028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagylaki</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span> <span class="cit-article-title">The evolution of multigene families under intrachromosomal gene conversion.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">529</span>–<span class="cit-lpage">548</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DNagylaki%26rft.auinit1%253DT.%26rft.volume%253D106%26rft.issue%253D3%26rft.spage%253D529%26rft.epage%253D548%26rft.atitle%253DTHE%2BEVOLUTION%2BOF%2BMULTIGENE%2BFAMILIES%2BUNDER%2BINTRACHROMOSOMAL%2BGENE%2BCONVERSION%26rft_id%253Dinfo%253Apmid%252F6706112%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTA2LzMvNTI5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzE0NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nagylaki</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Petes</span>  <span class="cit-name-given-names">T. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Intrachromosomal gene conversion and the maintenance of sequence homogeneity among repeated genes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">315</span>–<span class="cit-lpage">337</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DNagylaki%26rft.auinit1%253DT.%26rft.volume%253D100%26rft.issue%253D2%26rft.spage%253D315%26rft.epage%253D337%26rft.atitle%253DINTRACHROMOSOMAL%2BGENE%2BCONVERSION%2BAND%2BTHE%2BMAINTENANCE%2BOF%2BSEQUENCE%2BHOMOGENEITY%2BAMONG%2BREPEATED%2BGENES%26rft_id%253Dinfo%253Apmid%252F7106560%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTAwLzIvMzE1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzE0NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.42" data-doi="10.1093/molbev/msg076"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nielsen</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kasper</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bedford</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kristiansen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Gene conversion as a source of nucleotide diversity in <em>Plasmodium falciparum</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">726</span>–<span class="cit-lpage">734</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsg076%26rft_id%253Dinfo%253Apmid%252F12679555%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjg6IjIwLzUvNzI2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzE0NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-book" id="cit-4.8.1479.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohno</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1970</span> <span class="cit-source">Evolution by Gene Duplication</span>. <span class="cit-publ-name">Springer</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.44" data-doi="10.1073/pnas.79.10.3251"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohta</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1982</span> <span class="cit-article-title">Allelic and nonallelic homology of a supergene family.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">3251</span>–<span class="cit-lpage">3254</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DOhta%26rft.auinit1%253DT.%26rft.volume%253D79%26rft.issue%253D10%26rft.spage%253D3251%26rft.epage%253D3254%26rft.atitle%253DAllelic%2Band%2Bnonallelic%2Bhomology%2Bof%2Ba%2Bsupergene%2Bfamily.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.79.10.3251%26rft_id%253Dinfo%253Apmid%252F6954477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiNzkvMTAvMzI1MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOC8xNDc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.45" data-doi="10.1016/0040-5809(83)90015-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohta</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">On the evolution of multigene families.</span> <abbr class="cit-jnl-abbrev">Theor. Popul. Biol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">216</span>–<span class="cit-lpage">240</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheoretical%2Bpopulation%2Bbiology%26rft.stitle%253DTheor%2BPopul%2BBiol%26rft.aulast%253DOhta%26rft.auinit1%253DT.%26rft.volume%253D23%26rft.issue%253D2%26rft.spage%253D216%26rft.epage%253D240%26rft.atitle%253DOn%2Bthe%2Bevolution%2Bof%2Bmultigene%2Bfamilies.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0040-5809%252883%252990015-1%26rft_id%253Dinfo%253Apmid%252F6612633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0040-5809(83)90015-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6612633&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1983QU75700006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.46" data-doi="10.3390/genes1030349"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ohta</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Gene conversion and evolution of gene families: an overview.</span> <abbr class="cit-jnl-abbrev">Genes</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">349</span>–<span class="cit-lpage">356</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%26rft.volume%253D1%26rft.spage%253D349%26rft_id%253Dinfo%253Adoi%252F10.3390%252Fgenes1030349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3390/genes1030349&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.47" data-doi="10.1534/genetics.107.085332"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Padhukasahasram</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marjoram</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bustamante</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Exploring population genetic models with recombination using efficient forward-time simulations.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">178</span>: <span class="cit-fpage">2417</span>–<span class="cit-lpage">2427</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPadhukasahasram%26rft.auinit1%253DB.%26rft.volume%253D178%26rft.issue%253D4%26rft.spage%253D2417%26rft.epage%253D2427%26rft.atitle%253DExploring%2BPopulation%2BGenetic%2BModels%2BWith%2BRecombination%2BUsing%2BEfficient%2BForward-Time%2BSimulations%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.085332%26rft_id%253Dinfo%253Apmid%252F18430959%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OC80LzI0MTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzgvMTQ3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.48" data-doi="10.1534/genetics.104.040311"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plagnol</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Padhukasahasram</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wall</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marjoram</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nordborg</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Relative influences of crossing over and gene conversion on the pattern of linkage disequilibrium in <em>Arabidopsis thaliana</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">2441</span>–<span class="cit-lpage">2448</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.104.040311%26rft_id%253Dinfo%253Apmid%252F16299388%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi80LzI0NDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzgvMTQ3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.49" data-doi="10.1086/518616"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Plotnikova</span>  <span class="cit-name-given-names">O. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kondrashov</span>  <span class="cit-name-given-names">F. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vlasovb</span>  <span class="cit-name-given-names">P. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grigorenkoa</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ginterc</span>  <span class="cit-name-given-names">E. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Conversion and compensatory evolution of the γ-cristallin genes and identification of cataractogenic mutation that reverses the sequence of human CRYGD gene to an ancestral state.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">32</span>–<span class="cit-lpage">43</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DPlotnikova%26rft.auinit1%253DO.%2BV.%26rft.volume%253D81%26rft.issue%253D1%26rft.spage%253D32%26rft.epage%253D43%26rft.atitle%253DConversion%2Band%2Bcompensatory%2Bevolution%2Bof%2Bthe%2Bgamma-crystallin%2Bgenes%2Band%2Bidentification%2Bof%2Ba%2Bcataractogenic%2Bmutation%2Bthat%2Breverses%2Bthe%2Bsequence%2Bof%2Bthe%2Bhuman%2BCRYGD%2Bgene%2Bto%2Ban%2Bancestral%2Bstate.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F518616%26rft_id%253Dinfo%253Apmid%252F17564961%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/518616&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17564961&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.50" data-doi="10.1093/molbev/msq039"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rane</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bergthorsson</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katju</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Gene conversion and DNA sequence polymorphism in the sex-determination gene <em>fog-2</em> and its paralog <em>ftr-1</em> in <em>Caenorhabditis elegans</em>.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">1561</span>–<span class="cit-lpage">1569</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BBiol.%2BEvol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fmolbev%252Fmsq039%26rft_id%253Dinfo%253Apmid%252F20133352%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjk6IjI3LzcvMTU2MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOC8xNDc5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.51" data-doi="10.1038/nature01723"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rozen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skaletsky</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marszalek</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Minx</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cordum</span>  <span class="cit-name-given-names">H. S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Abundant gene conversion between arms of palindromes in human and ape Y chromosomes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">423</span>: <span class="cit-fpage">873</span>–<span class="cit-lpage">876</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRozen%26rft.auinit1%253DS.%26rft.volume%253D423%26rft.issue%253D6942%26rft.spage%253D873%26rft.epage%253D876%26rft.atitle%253DAbundant%2Bgene%2Bconversion%2Bbetween%2Barms%2Bof%2Bpalindromes%2Bin%2Bhuman%2Band%2Bape%2BY%2Bchromosomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01723%26rft_id%253Dinfo%253Apmid%252F12815433%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01723&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12815433&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000183585300047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.52"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sawyer</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span> <span class="cit-article-title">Statistical tests for detecting gene conversion.</span> <abbr class="cit-jnl-abbrev">Mol. Biol. Evol.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">526</span>–<span class="cit-lpage">538</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BBiology%2Band%2BEvolution%26rft.stitle%253DMol%2BBiol%2BEvol%26rft.aulast%253DSawyer%26rft.auinit1%253DS%26rft.volume%253D6%26rft.issue%253D5%26rft.spage%253D526%26rft.epage%253D538%26rft.atitle%253DStatistical%2Btests%2Bfor%2Bdetecting%2Bgene%2Bconversion%26rft_id%253Dinfo%253Apmid%252F2677599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTE6Im1vbGJpb2xldm9sIjtzOjU6InJlc2lkIjtzOjc6IjYvNS81MjYiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzgvMTQ3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.53" data-doi="10.1086/431652"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Locke</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGrath</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Segmental duplications and copy-number variation in the human genome.</span> <abbr class="cit-jnl-abbrev">Am. J. Hum. Genet.</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">78</span>–<span class="cit-lpage">88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2Bjournal%2Bof%2Bhuman%2Bgenetics%26rft.stitle%253DAm%2BJ%2BHum%2BGenet%26rft.aulast%253DSharp%26rft.auinit1%253DA.%2BJ.%26rft.volume%253D77%26rft.issue%253D1%26rft.spage%253D78%26rft.epage%253D88%26rft.atitle%253DSegmental%2Bduplications%2Band%2Bcopy-number%2Bvariation%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1086%252F431652%26rft_id%253Dinfo%253Apmid%252F15918152%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1086/431652&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15918152&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000229794500007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.54" data-doi="10.1146/annurev-med-100708-204735"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stankiewicz</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lupski</span>  <span class="cit-name-given-names">J. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Structural variation in the human genome and its role in disease.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Med.</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">437</span>–<span class="cit-lpage">455</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bmedicine%26rft.stitle%253DAnnu%2BRev%2BMed%26rft.aulast%253DStankiewicz%26rft.auinit1%253DP.%26rft.volume%253D61%26rft.spage%253D437%26rft.epage%253D455%26rft.atitle%253DStructural%2Bvariation%2Bin%2Bthe%2Bhuman%2Bgenome%2Band%2Bits%2Brole%2Bin%2Bdisease.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-med-100708-204735%26rft_id%253Dinfo%253Apmid%252F20059347%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-med-100708-204735&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20059347&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000274857600029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.55" data-doi="10.1534/genetics.166.3.1553"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teshima</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Innan</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">The effect of gene conversion on the divergence between duplicated genes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">166</span>: <span class="cit-fpage">1553</span>–<span class="cit-lpage">1560</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTeshima%26rft.auinit1%253DK.%2BM.%26rft.volume%253D166%26rft.issue%253D3%26rft.spage%253D1553%26rft.epage%253D1560%26rft.atitle%253DThe%2BEffect%2Bof%2BGene%2BConversion%2Bon%2Bthe%2BDivergence%2BBetween%2BDuplicated%2BGenes%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.166.3.1553%26rft_id%253Dinfo%253Apmid%252F15082568%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2Ni8zLzE1NTMiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzgvMTQ3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.56" data-doi="10.1534/genetics.111.135343"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Teshima</span>  <span class="cit-name-given-names">K. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Innan</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The coalescent with selection on copy number variants.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">1077</span>–<span class="cit-lpage">1086</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.135343%26rft_id%253Dinfo%253Apmid%252F22174068%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5MC8zLzEwNzciO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzgvMTQ3OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.57" data-doi="10.1534/genetics.107.074948"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thornton</span>  <span class="cit-name-given-names">K. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The neutral coalescent process for recent gene duplications and copy-number variants.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">177</span>: <span class="cit-fpage">987</span>–<span class="cit-lpage">1000</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.074948%26rft_id%253Dinfo%253Apmid%252F17720930%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc3LzIvOTg3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzE0NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.58" data-doi="10.1371/journal.pone.0028853"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Uddin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sturge</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peddle</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Reilly</span>  <span class="cit-name-given-names">D. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rahman</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide signatures of “rearrangement hotspots” within segmental duplications in humans.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">e28853</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DUddin%26rft.auinit1%253DM.%26rft.volume%253D6%26rft.issue%253D12%26rft.spage%253De28853%26rft.epage%253De28853%26rft.atitle%253DGenome-wide%2Bsignatures%2Bof%2B%2527rearrangement%2Bhotspots%2527%2Bwithin%2Bsegmental%2Bduplications%2Bin%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0028853%26rft_id%253Dinfo%253Apmid%252F22194928%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0028853&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22194928&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.59"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Walsh</span>  <span class="cit-name-given-names">J. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span> <span class="cit-article-title">Sequence-dependent gene conversion: can duplicated genes diverge fast enough to escape conversion?</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">543</span>–<span class="cit-lpage">557</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWalsh%26rft.auinit1%253DJ.%2BB.%26rft.volume%253D117%26rft.issue%253D3%26rft.spage%253D543%26rft.epage%253D557%26rft.atitle%253DSequence-Dependent%2BGene%2BConversion%253A%2BCan%2BDuplicated%2BGenes%2BDiverge%2BFast%2BEnough%2Bto%2BEscape%2BConversion%253F%26rft_id%253Dinfo%253Apmid%252F3692140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTE3LzMvNTQzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzE0NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.60" data-doi="10.1186/1471-2148-13-148"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Willett</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Gene conversion yields novel gene combinations in paralogs of GOT1 in the copepod <em>Tigriopus californicus</em>.</span> <abbr class="cit-jnl-abbrev">BMC Evol. Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">148</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2BEvol.%2BBiol.%26rft.volume%253D13%26rft.spage%253D148%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2148-13-148%26rft_id%253Dinfo%253Apmid%252F23845062%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2148-13-148&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23845062&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1479.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1479.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wiuf</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hein</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The coalescent with gene conversion.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">155</span>: <span class="cit-fpage">451</span>–<span class="cit-lpage">462</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DWiuf%26rft.auinit1%253DC.%26rft.volume%253D155%26rft.issue%253D1%26rft.spage%253D451%26rft.epage%253D462%26rft.atitle%253DThe%2BCoalescent%2BWith%2BGene%2BConversion%26rft_id%253Dinfo%253Apmid%252F10790416%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTU1LzEvNDUxIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzE0NzkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/8/1479.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/8/1465" title="Association Mapping of Germination Traits in Arabidopsis thaliana Under Light and Nutrient Treatments: Searching for G×E Effects" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/8/1491" title="Linking Genetics to Structural Biology: Complex Heterozygosity Screening with Actin Alanine Scan Alleles Identifies Functionally Related Surfaces on Yeast Actin" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/8"><span>Volume 4 Issue 8, August 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/8.cover-source.jpg?itok=uF7Z9Bah" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/8/1479.full" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form--yVFuK3Io1R5nt1-cVjsDkg1Y7xdYG202CnF6iqbCYA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/8/1479" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/8/1479" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/8/1479" class="active" data-icon-position="" data-hide-link-title="0">Interplay of Interlocus Gene Conversion and Crossover in Segmental Duplications Under a Neutral Scenario</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459948" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-KTG3axEOjuKWqwYfhqnALNWy_2d4gutDF8zA5un17VY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/8/1479.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/8/1479" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-MkKgET6EPR-kxpv2iIiBDHV62p7mhKlE6eMOgJlwZTM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/8/1479" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/8/1479.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459948" id="node459948--2" data-pisa="ggg;4/8/1479" data-pisa-master="ggg;g3.114.012435" data-apath="/ggg/4/8/1479.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Interplay of Interlocus Gene Conversion and Crossover in Segmental Duplications Under a Neutral Scenario</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Diego A. Hartasánchez</span>, <span class="highwire-citation-author" data-delta="1">Oriol Vallès-Codina</span>, <span class="highwire-citation-author" data-delta="2">Marina Brasó-Vives</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-2162-8246" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Arcadi Navarro</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1479-1489;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012435 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459948--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Diego A. Hartasánchez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolutionary Biology (Universitat Pompeu Fabra – CSIC), PRBB, Barcelona, Catalonia, Spain, 08003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BA.%2BHartas%C3%A1nchez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hartas%C3%A1nchez%20DA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADiego%2BA.%2BHartas%25C3%25A1nchez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Oriol Vallès-Codina </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolutionary Biology (Universitat Pompeu Fabra – CSIC), PRBB, Barcelona, Catalonia, Spain, 08003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Oriol%2BVall%C3%A8s-Codina%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vall%C3%A8s-Codina%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOriol%2BVall%25C3%25A8s-Codina%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Marina Brasó-Vives </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolutionary Biology (Universitat Pompeu Fabra – CSIC), PRBB, Barcelona, Catalonia, Spain, 08003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marina%2BBras%C3%B3-Vives%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bras%C3%B3-Vives%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarina%2BBras%25C3%25B3-Vives%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Arcadi Navarro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolutionary Biology (Universitat Pompeu Fabra – CSIC), PRBB, Barcelona, Catalonia, Spain, 08003</div><div class='author-affiliation'><span class='nlm-sup'>†</span>National Institute for Bioinformatics (INB), Barcelona, Catalonia, Spain</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institució Catalana de Recerca i Estudis Avançats (ICREA), Barcelona, Catalonia, Spain</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Genomic Regulation (CRG), Barcelona, Catalonia, Spain, 08003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Arcadi%2BNavarro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Navarro%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AArcadi%2BNavarro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2162-8246" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Arcadi Navarro</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:arcadi.navarro@upf.edu" class="" data-icon-position="" data-hide-link-title="0">arcadi.navarro@upf.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/8/1479" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/8/1479&amp;title=Interplay%20of%20Interlocus%20Gene%20Conversion%20and%20Crossover%20in%20Segmental%20Duplications%20Under%20a%20Neutral%20Scenario" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/8/1479&amp;title=Interplay%20of%20Interlocus%20Gene%20Conversion%20and%20Crossover%20in%20Segmental%20Duplications%20Under%20a%20Neutral%20Scenario" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/8/1479&amp;title=Interplay%20of%20Interlocus%20Gene%20Conversion%20and%20Crossover%20in%20Segmental%20Duplications%20Under%20a%20Neutral%20Scenario" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/8/1479&amp;text=Interplay%20of%20Interlocus%20Gene%20Conversion%20and%20Crossover%20in%20Segmental%20Duplications%20Under%20a%20Neutral%20Scenario" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/8/1479&amp;title=Interplay%20of%20Interlocus%20Gene%20Conversion%20and%20Crossover%20in%20Segmental%20Duplications%20Under%20a%20Neutral%20Scenario" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/8/1479&amp;t=Interplay%20of%20Interlocus%20Gene%20Conversion%20and%20Crossover%20in%20Segmental%20Duplications%20Under%20a%20Neutral%20Scenario" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/8/1479&amp;title=Interplay%20of%20Interlocus%20Gene%20Conversion%20and%20Crossover%20in%20Segmental%20Duplications%20Under%20a%20Neutral%20Scenario" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/8/1479&amp;title=Interplay%20of%20Interlocus%20Gene%20Conversion%20and%20Crossover%20in%20Segmental%20Duplications%20Under%20a%20Neutral%20Scenario" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459948/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459948" id="node459948--3" data-pisa="ggg;4/8/1479" data-pisa-master="ggg;g3.114.012435" data-apath="/ggg/4/8/1479.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Interplay of Interlocus Gene Conversion and Crossover in Segmental Duplications Under a Neutral Scenario</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Diego A. Hartasánchez</span>, <span class="highwire-citation-author" data-delta="1">Oriol Vallès-Codina</span>, <span class="highwire-citation-author" data-delta="2">Marina Brasó-Vives</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0003-2162-8246" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Arcadi Navarro</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1479-1489;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.012435 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459948--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Diego A. Hartasánchez </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolutionary Biology (Universitat Pompeu Fabra – CSIC), PRBB, Barcelona, Catalonia, Spain, 08003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Diego%2BA.%2BHartas%C3%A1nchez%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hartas%C3%A1nchez%20DA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADiego%2BA.%2BHartas%25C3%25A1nchez%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Oriol Vallès-Codina </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolutionary Biology (Universitat Pompeu Fabra – CSIC), PRBB, Barcelona, Catalonia, Spain, 08003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Oriol%2BVall%C3%A8s-Codina%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Vall%C3%A8s-Codina%20O&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AOriol%2BVall%25C3%25A8s-Codina%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Marina Brasó-Vives </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolutionary Biology (Universitat Pompeu Fabra – CSIC), PRBB, Barcelona, Catalonia, Spain, 08003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Marina%2BBras%C3%B3-Vives%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bras%C3%B3-Vives%20M&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMarina%2BBras%25C3%25B3-Vives%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Arcadi Navarro </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute of Evolutionary Biology (Universitat Pompeu Fabra – CSIC), PRBB, Barcelona, Catalonia, Spain, 08003</div><div class='author-affiliation'><span class='nlm-sup'>†</span>National Institute for Bioinformatics (INB), Barcelona, Catalonia, Spain</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Institució Catalana de Recerca i Estudis Avançats (ICREA), Barcelona, Catalonia, Spain</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Centre for Genomic Regulation (CRG), Barcelona, Catalonia, Spain, 08003</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Arcadi%2BNavarro%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Navarro%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AArcadi%2BNavarro%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2162-8246" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Arcadi Navarro</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:arcadi.navarro@upf.edu" class="" data-icon-position="" data-hide-link-title="0">arcadi.navarro@upf.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1479&amp;rft.epage=1489&amp;rft.atitle=Interplay%20of%20Interlocus%20Gene%20Conversion%20and%20Crossover%20in%20Segmental%20Duplications%20Under%20a%20Neutral%20Scenario&amp;rft.volume=4&amp;rft.issue=8&amp;rft.date=2014-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Hartas%C3%A1nchez%2C+Diego+A.&amp;rft.au=Vall%C3%A8s-Codina%2C+Oriol&amp;rft.au=Bras%C3%B3-Vives%2C+Marina&amp;rft.au=Navarro%2C+Arcadi"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459948/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459948/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459948/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459948/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459948/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459948/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459948/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459948/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459948/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459948/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459948/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459948/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/8/1479" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/8/1479.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/8/1479.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/8/1479.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/8/1479.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
