<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/8/1503" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/8/1503.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/8/1503.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/8/1503.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/4/8/1503/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/8/1503.atom" />
<meta name="HW.pisa" content="ggg;4/8/1503" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Pattern of Mutation Rates in the Germline of Drosophila melanogaster Males from a Large-Scale Mutation Screening Experiment" />
<meta name="DC.Identifier" content="10.1534/g3.114.011056" />
<meta name="DC.Date" content="2014-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright Â© 2014 Gao et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The sperm or eggs of sexual organisms go through a series of cell divisions from the fertilized egg; mutations can occur at each division. Mutations in the lineage of cells leading to the sperm or eggs are of particular importance because many such mutations may be shared by somatic tissues and also may be inherited, thus having a lasting consequence. For decades, little has been known about the pattern of the mutation rates along the germline development. Recently it was shown from a small portion of data that resulted from a large-scale mutation screening experiment that the rates of recessive lethal or nearly lethal mutations differ dramatically during the germline development of Drosophila melanogaster males. In this paper the full data set from the experiment and its analysis are reported by taking advantage of a recent methodologic advance. By analyzing the mutation patterns with different levels of recessive lethality, earlier published conclusions based on partial data are found to remain valid. Furthermore, it is found that for most nearly lethal mutations, the mutation rate at the first cell division is even greater than previous thought compared with those at other divisions. There is also some evidence that the mutation rate at the second division decreases rapidly but is still appreciably greater than those for the rest of the cleavage stage. The mutation rate at spermatogenesis is greater than late cleavage and stem-cell stages, but there is no evidence that rates are different among the five cell divisions of the spermatogenesis. We also found that a modestly biased sampling, leading to slightly more primordial germ cells after the eighth division than those reported in the literature, provides the best fit to the data. These findings provide conceptual and numerical basis for exploring the consequences of differential mutation rates during individual development." />
<meta name="DC.Contributor" content="Jian-Jun Gao" />
<meta name="DC.Contributor" content="Xue-Rong Pan" />
<meta name="DC.Contributor" content="Jing Hu" />
<meta name="DC.Contributor" content="Li Ma" />
<meta name="DC.Contributor" content="Jian-Min Wu" />
<meta name="DC.Contributor" content="Ye-Lin Shao" />
<meta name="DC.Contributor" content="Shi-Meng Ai" />
<meta name="DC.Contributor" content="Shu-Qun Liu" />
<meta name="DC.Contributor" content="Sara A. Barton" />
<meta name="DC.Contributor" content="Ronny C. Woodruff" />
<meta name="DC.Contributor" content="Ya-Ping Zhang" />
<meta name="DC.Contributor" content="Yun-Xin Fu" />
<meta name="article:published_time" content="2014-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Pattern of Mutation Rates in the Germline of Drosophila melanogaster Males from a Large-Scale Mutation Screening Experiment" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The sperm or eggs of sexual organisms go through a series of cell divisions from the fertilized egg; mutations can occur at each division. Mutations in the lineage of cells leading to the sperm or eggs are of particular importance because many such mutations may be shared by somatic tissues and also may be inherited, thus having a lasting consequence. For decades, little has been known about the pattern of the mutation rates along the germline development. Recently it was shown from a small portion of data that resulted from a large-scale mutation screening experiment that the rates of recessive lethal or nearly lethal mutations differ dramatically during the germline development of &lt;i&gt;Drosophila melanogaster&lt;/i&gt; males. In this paper the full data set from the experiment and its analysis are reported by taking advantage of a recent methodologic advance. By analyzing the mutation patterns with different levels of recessive lethality, earlier published conclusions based on partial data are found to remain valid. Furthermore, it is found that for most nearly lethal mutations, the mutation rate at the first cell division is even greater than previous thought compared with those at other divisions. There is also some evidence that the mutation rate at the second division decreases rapidly but is still appreciably greater than those for the rest of the cleavage stage. The mutation rate at spermatogenesis is greater than late cleavage and stem-cell stages, but there is no evidence that rates are different among the five cell divisions of the spermatogenesis. We also found that a modestly biased sampling, leading to slightly more primordial germ cells after the eighth division than those reported in the literature, provides the best fit to the data. These findings provide conceptual and numerical basis for exploring the consequences of differential mutation rates during individual development.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/08/01" />
<meta name="citation_mjid" content="ggg;4/8/1503" />
<meta name="citation_id" content="4/8/1503" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/8/1503" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/8/1503.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/8/1503.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/8/1503.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.011056" />
<meta name="citation_pmid" content="24924332" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="12" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1503" />
<meta name="citation_lastpage" content="1514" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Jian-Jun Gao" />
<meta name="citation_author_institution" content="Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China" />
<meta name="citation_author" content="Xue-Rong Pan" />
<meta name="citation_author_institution" content="Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China" />
<meta name="citation_author" content="Jing Hu" />
<meta name="citation_author_institution" content="Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China" />
<meta name="citation_author" content="Li Ma" />
<meta name="citation_author_institution" content="Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China" />
<meta name="citation_author" content="Jian-Min Wu" />
<meta name="citation_author_institution" content="Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China" />
<meta name="citation_author" content="Ye-Lin Shao" />
<meta name="citation_author_institution" content="Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China" />
<meta name="citation_author" content="Shi-Meng Ai" />
<meta name="citation_author_institution" content="Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China" />
<meta name="citation_author" content="Shu-Qun Liu" />
<meta name="citation_author_institution" content="Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China" />
<meta name="citation_author" content="Sara A. Barton" />
<meta name="citation_author_institution" content="Division of Biostatistics and Human Genetics Center, School of Public Health, The University of Texas at Houston, Houston, Texas" />
<meta name="citation_author" content="Ronny C. Woodruff" />
<meta name="citation_author_institution" content="Department of Biological Sciences, Bowling Green State University, Bowling Green, Ohio" />
<meta name="citation_author" content="Ya-Ping Zhang" />
<meta name="citation_author_institution" content="Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China" />
<meta name="citation_author_institution" content="State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming, Yunnan, China" />
<meta name="citation_author" content="Yun-Xin Fu" />
<meta name="citation_author_institution" content="Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China" />
<meta name="citation_author_institution" content="Department of Biological Sciences, Bowling Green State University, Bowling Green, Ohio" />
<meta name="citation_author_email" content="yunxin.fu@uth.tmc.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1224-4075" />
<meta name="citation_reference" content="citation_title=Chemical Mutagens;citation_title=The detection of mutations in Drosophila melanogaster;citation_author=S. Abrahamson;citation_author=E. B. Lewis;citation_pages=461-484;citation_year=1971" />
<meta name="citation_reference" content="citation_title=Drosophila: A Laboratory Handbook;citation_author=M. Ashburner;citation_year=1989" />
<meta name="citation_reference" content="citation_journal_title=Environmental and molecular mutagenesis;citation_journal_abbrev=Environ Mol Mutagen;citation_author=R. K. Brodberg;citation_author=M. J. Mitchell;citation_author=S. L. Smith;citation_author=R. C. Woodruff;citation_title=Specific reduction of N,N-dimethylnitrosamine mutagenicity in Drosophila melanogaster by dimethyl sulfoxide.;citation_pages=425-432;citation_volume=10;citation_year=1987;citation_issue=4;citation_pmid=3119334;citation_doi=10.1002/em.2850100411" />
<meta name="citation_reference" content="citation_title=J. Radiat. Res.;citation_author=J. F. Crow;citation_title=Age and sex effects on human mutation rates: an old problem with new complexities.;citation_volume=47;citation_year=2006;citation_pmid=17019055;citation_doi=10.1269/jrr.47.B75" />
<meta name="citation_reference" content="citation_journal_title=Environ. Mol. Mutagen.;citation_author=J. B. Drost;citation_author=W. R. Lee;citation_title=Biological basis of germline mutation: Comparisons of spontaneous germline mutation rates among Drosophila, mouse and human.;citation_pages=48-64;citation_volume=25;citation_year=1995;citation_pmid=7789362;citation_doi=10.1002/em.2850250609" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=J. B. Drost;citation_author=W. R. Lee;citation_title=The developmental basis for the germline mosaicism in mouse and Drosophila melanogaster.;citation_pages=421-443;citation_volume=102/103;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. X. Fu;citation_title=Statistical methods for analyzing Drosophila germline mutation rates.;citation_volume=194;citation_year=2013;citation_pmid=23636740;citation_doi=10.1534/genetics.113.151571" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Y. X. Fu;citation_author=H. Huai;citation_title=Estimating Mutation Rate: How to Count Mutations?;citation_pages=797-805;citation_volume=164;citation_year=2003;citation_issue=2;citation_pmid=12807798" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=J. J. Gao;citation_author=X. R. Pan;citation_author=J. Hu;citation_author=L. Ma;citation_author=J. M. Wu;citation_title=Highly variable recessive lethal or nearly lethal mutation rates during germline development of male Drosophila melanogaster.;citation_volume=108;citation_year=2011;citation_pmid=21890796;citation_doi=10.1073/pnas.1100233108" />
<meta name="citation_reference" content="citation_title=Developmental Biology;citation_author=S. F. Gilbert;citation_year=2003" />
<meta name="citation_reference" content="citation_title=Drosophila Genetics: A Practical Course;citation_author=U. Graf;citation_author=N. van Schaik;citation_author=F. E. Wurgler;citation_year=1992" />
<meta name="citation_reference" content="citation_title=Fly Pushing: The Theory and Practice of Drosophila Genetics;citation_author=S. F. Greenspan;citation_year=1997" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. Huai;citation_author=R. C. Woodruff;citation_title=Clusters of Identical New Mutations Can Account for the ``Overdispersed'' Molecular Clock;citation_pages=339-348;citation_volume=147;citation_year=1997;citation_issue=1;citation_pmid=9286693" />
<meta name="citation_reference" content="citation_title=Molecular Evolution;citation_author=W. H. Li;citation_year=1997" />
<meta name="citation_reference" content="citation_title=The Genome of Drosophila melanogaster;citation_author=D. L. Lindsley;citation_author=G. G. Zimm;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Environmental mutagenesis;citation_journal_abbrev=Environ Mutagen;citation_author=J. M. Mason;citation_author=R. Valencia;citation_author=R. C. Woodruff;citation_author=S. Zimmering;citation_title=Genetic drift and seasonal variation in spontaneous mutation frequencies in Drosophila.;citation_pages=663-676;citation_volume=7;citation_year=1985;citation_issue=5;citation_pmid=3930236;citation_doi=10.1002/em.2860070506" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harbor Symposia on Quantitative Biology;citation_journal_abbrev=Cold Spring Harb Symp Quant Biol;citation_author=T. Miyata;citation_author=H. Hayashida;citation_author=K. Kuma;citation_author=K. Mitsuyasa;citation_author=T. Yasunaga;citation_title=Male-driven Molecular Evolution: A Model and Nucleotide Sequence Analysis;citation_pages=863-867;citation_volume=52;citation_year=1987;citation_issue=0;citation_pmid=3454295;citation_doi=10.1101/SQB.1987.052.01.094" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. J. Muller;citation_title=THE MEASUREMENT OF GENE MUTATION RATE IN DROSOPHILA, ITS HIGH VARIABILITY, AND ITS DEPENDENCE UPON TEMPERATURE;citation_pages=279-357;citation_volume=13;citation_year=1928;citation_issue=4;citation_pmid=17246553" />
<meta name="citation_reference" content="citation_title=Methodology in Basic Genetics;citation_title=Some mutational techniques in Drosophila;citation_author=H. J. Muller;citation_author=I. I. Oster;citation_pages=249-278;citation_year=1963" />
<meta name="citation_reference" content="citation_journal_title=Biochim. Biophys. Acta;citation_author=C. Rathke;citation_author=W. R. Baarends;citation_author=S. Awe;citation_author=R. Renkawitz-Pohl;citation_title=Chromatin dynamics during spermiogenesis.;citation_pages=155-168;citation_volume=1839;citation_year=2014;citation_doi=10.1016/j.bbagrm.2013.08.004" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=L. Shimmin;citation_author=B. H.-J. Chang;citation_author=W. H. Li;citation_title=Male-driven evolution of DNA sequences.;citation_pages=745-747;citation_volume=362;citation_year=1993;citation_issue=6422;citation_pmid=8469284;citation_doi=10.1038/362745a0" />
<meta name="citation_reference" content="citation_title=Biology of Drosophila;citation_title=The early embryology of Drosophila melanogaster;citation_author=B. P. Sonnenblick;citation_pages=62-167;citation_year=1965" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=J. N. Thompson;citation_author=R. C. Woodruff;citation_title=Increased mutation in crosses between geographically separated strains of Drosophila melanogaster.;citation_pages=1059-1062;citation_volume=77;citation_year=1980;citation_issue=2;citation_pmid=6767240;citation_doi=10.1073/pnas.77.2.1059" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=V. Thompson;citation_title=RECOMBINATION AND RESPONSE TO SELECTION IN DROSOPHILA MELANOGASTER;citation_pages=125-140;citation_volume=85;citation_year=1977;citation_issue=1;citation_pmid=402302" />
<meta name="citation_reference" content="citation_title=Human Genetics: Problems and Approaches;citation_author=F. Vogel;citation_author=A. G. Motulsky;citation_year=1997" />
<meta name="citation_reference" content="citation_journal_title=Aging cell;citation_journal_abbrev=Aging Cell;citation_author=M. R. Wallenfang;citation_author=R. Nyak;citation_author=S. DiNardo;citation_title=Dynamics of the male germline stem cell population during aging of Drosophila melanogaster.;citation_pages=297-304;citation_volume=5;citation_year=2006;citation_issue=4;citation_pmid=16800845;citation_doi=10.1111/j.1474-9726.2006.00221.x" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_journal_abbrev=Genetica;citation_author=R. C. Woodruff;citation_author=J. N. J. Thompson;citation_title=The fundamental theorem of neutral evolution: rates of substitution and mutation should factor in premeiotic clusters.;citation_pages=333-339;citation_volume=125;citation_year=2005;citation_issue=2-3;citation_pmid=16247704;citation_doi=10.1007/s10709-005-4982-7" />
<meta name="citation_reference" content="citation_journal_title=Heredity;citation_author=R. C. Woodruff;citation_author=J. J. N. Thompson;citation_author=M. A. Seeger;citation_author=W. E. Spivey;citation_title=Variation in spontaneous mutation and repair in natural population lines of Drosophila melanogaster.;citation_pages=223-234;citation_volume=58;citation_year=1984" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_journal_abbrev=Genetica;citation_author=R. C. Woodruff;citation_author=H. Huai;citation_author=J. N. Thompson;citation_title=Clusters of identical new mutation in the evolutionary landscape.;citation_pages=149-160;citation_volume=98;citation_year=1996;citation_issue=2;citation_pmid=8976063;citation_doi=10.1007/BF00121363" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Pattern of Mutation Rates in the Germline of Drosophila melanogaster Males from a Large-Scale Mutation Screening Experiment" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8.cover-source.jpg" />
<meta name="twitter:description" content="The sperm or eggs of sexual organisms go through a series of cell divisions from the fertilized egg; mutations can occur at each division. Mutations in the lineage of cells leading to the sperm or eggs are of particular importance because many such mutations may be shared by somatic tissues and also may be inherited, thus having a lasting consequence. For decades, little has been known about the pattern of the mutation rates along the germline development. Recently it was shown from a small portion of data that resulted from a large-scale mutation screening experiment that the rates of recessive lethal or nearly lethal mutations differ dramatically during the germline development of Drosophila melanogaster males. In this paper the full data set from the experiment and its analysis are reported by taking advantage of a recent methodologic advance. By analyzing the mutation patterns with different levels of recessive lethality, earlier published conclusions based on partial data are found to remain valid. Furthermore, it is found that for most nearly lethal mutations, the mutation rate at the first cell division is even greater than previous thought compared with those at other divisions. There is also some evidence that the mutation rate at the second division decreases rapidly but is still appreciably greater than those for the rest of the cleavage stage. The mutation rate at spermatogenesis is greater than late cleavage and stem-cell stages, but there is no evidence that rates are different among the five cell divisions of the spermatogenesis. We also found that a modestly biased sampling, leading to slightly more primordial germ cells after the eighth division than those reported in the literature, provides the best fit to the data. These findings provide conceptual and numerical basis for exploring the consequences of differential mutation rates during individual development." />
<meta name="og-title" property="og:title" content="Pattern of Mutation Rates in the Germline of Drosophila melanogaster Males from a Large-Scale Mutation Screening Experiment" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/8/1503" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The sperm or eggs of sexual organisms go through a series of cell divisions from the fertilized egg; mutations can occur at each division. Mutations in the lineage of cells leading to the sperm or eggs are of particular importance because many such mutations may be shared by somatic tissues and also may be inherited, thus having a lasting consequence. For decades, little has been known about the pattern of the mutation rates along the germline development. Recently it was shown from a small portion of data that resulted from a large-scale mutation screening experiment that the rates of recessive lethal or nearly lethal mutations differ dramatically during the germline development of Drosophila melanogaster males. In this paper the full data set from the experiment and its analysis are reported by taking advantage of a recent methodologic advance. By analyzing the mutation patterns with different levels of recessive lethality, earlier published conclusions based on partial data are found to remain valid. Furthermore, it is found that for most nearly lethal mutations, the mutation rate at the first cell division is even greater than previous thought compared with those at other divisions. There is also some evidence that the mutation rate at the second division decreases rapidly but is still appreciably greater than those for the rest of the cleavage stage. The mutation rate at spermatogenesis is greater than late cleavage and stem-cell stages, but there is no evidence that rates are different among the five cell divisions of the spermatogenesis. We also found that a modestly biased sampling, leading to slightly more primordial germ cells after the eighth division than those reported in the literature, provides the best fit to the data. These findings provide conceptual and numerical basis for exploring the consequences of differential mutation rates during individual development." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8.cover-source.jpg" />
    <title>Pattern of Mutation Rates in the Germline of Drosophila melanogaster Males from a Large-Scale Mutation Screening Experiment | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"zEwhLL9d2LJje9yhJOMQBsrIyS7b9lSvNeVUgQaOW4Q","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459947","apath":"\/ggg\/4\/8\/1503.atom","pisa":"ggg;4\/8\/1503","ac":{"\/ggg\/4\/8\/1503.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/8\/1503.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/8\/1503"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/8\/1503"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/8\/1503":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022l6QuwkG1uyt4+RGBni0FwADrsTYBQDqmXCb9yCv6TajO7s3OTY6Gj+way2Drd2yqeDEGePrkmby6uRO4u5SENfApzGJWRsAtSD05mfvzNz45U1NVSToSrQBY78CNICwt3VRP0lLtZuYYQ\\\\\\\/cMJwceDiA2CtmHWRrVb0S5YxdITzNtB2s7QOk4vnXvU08ajtvkh25k8psnGI+SWDhd7l07kLyMQndp4hVeWGhO\\\\\\\/faPvuTdbzn2pv6hspAQBsR0nZJfwe1u2a+5TW0wR6N0FYaQAqbZOtjQppk2wifxw5yY8T9pJiCFs1j2rb89Ol0DsiTuBuXfdhuiiubgxzlZ80wr0ViEZORFh3JHmBGaDhg6PkZPsegc1OQiFHYcPkTtcGGk7JYktnm8DwPUugv4sFcO5MpfHBQZGhPZ8WjIC5flThI+xbFE+msNOrvKVNc8d7t56G3XRQY33bL1+OuL5eTmk4W5HkMjn42hgXWScK5+UjvPgUKyccolJ8PY5gEeLGsoRjphTsGVfg2\\\\\\\/iXcwZi+YbE94mqZzBfTny9t6PesrP57yz8\\\\\\\/xl1Ro4krhdR+2HmX17\\\\\\\/6ShwidlVs14p7BzxtHec4Z+sUHTnYaQ3fHVryEIzUBNRD70g1T8uvfhv0JdB\\\\\\\/95W4M\\\\\\\/hzRJo4QosRdol9PtPs2LKRG5iUF4CGbCdpCIxzT6c4iUTZCWgI0NVSMrEfu0KLLb+6XSHop3JNmPcAHKZYzp4fEvsuTWkTVak8uTYA7e\\\\\\\/bJsIz5TPlPmK+\\\\\\\/pEoiGhLs9KLc1ioCwzpopZIbA+cgAW8SCpRULJ\\\\\\\/dtHyzM5ZMX4DbXNvRfz\\\\\\\/p2X2\\\\\\\/O3Fj5tRv5ModgXCTmRowKstvI+IlgTTWub04OZH4\\\\\\\/jkXCYrAbOxEJoR0KgG74in4iWTcPyvmoD9PfsAoNZpl7\\\\\\\/hIlkZLTw1nQ86ugBNM9iokerXoF+uTkcKwZ4oCAnAYWjuELV97NfJLkAFTTSzwQD+DAf6sdmveRshhT6USwTXudU0VHYstTob+TFEaxcEZ4S95YBB7IKbF3dmWq6vTKcuaH4kT2qqCYE9tpo2qwBjIWqMJ2Oxj4bY68nsBZDThY4jKL9ER9fbrtuXGdwNS8PTA4WVxCJPgKJibhM+Ud9FekjtEG2yDKkfmCihLeKRmtHv3jDtlzKEMx03MIBNrBHxJgDtK\\\\\\\/DBeY8iQkKDeApS8VDKrKLPS\\\\\\\/ZDcheg9Mhkm+caj8NINyIR\\\\\\\/p8tU\\\\\\\/hv3yoPusMnpZ2wVnC2+9+Swv7+qpCw3n\\\\\\\/JY91jCvTt4GBrIr1xI\\\u0022,\\\u0022iv\\\u0022:\\\u0022hGIiYKyomygVnuAGIAleXw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00229ba23e70322b56ac52bdf0d3d4f5a7f2ebb129686d335c984b5a54dda1255b1c\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022vsnC7vtEajYmO704Vi148JkcN1IM3yaGRdPHJwlVoySairz1bX2vSOX8MpdvLpTyVACHPelq5whvxz7vvRdK0iIKq3O7zMVPfmULntGIQFnlTF8DA8bA3E7UzaOc+r3eDjwg4d5b7QhskSY22QqW1Dtd7C57KsyaX\\\\\\\/TPC56EE2Hbyw\\\\\\\/L3AvxCGt4bgCiBScHjqqNZBZSpHbogM9kR6dc9LzogrKXbOZUQTE4t06W56+xRcKMKVUP+pf8Eu2XlKuT0w4ahv+NxiUKA+WuRsZAVZIgOZwlzLGMXpln2GLxB29o+EjcGrnlaM5j8GNVrfq80I5wtt2eNvxNTiyUjHblug\\\\\\\/7Y9MLLJ1WVpa4qAyi4qg8FUiLHuLYpWF+3CMM2P++xZ1FOvuW58FK7yV4jL9whpM2d2l4DNI+kU71QqNgzbnWJXnTjZgPlMxrCb1Dlov0dxv0oxQUq3vubzhxsxmzHFUoqftCCu1QARExbytfYELfu\\\\\\\/GkLtmR48jsZ1KgvAxi1O5XPt0d8ikit94rHMtPoAabH8dRlMwkwy3dzF9nXi8Smga0R7V3G3JMjpknmbkiIljUOgMqnj85u14wILhZn6FtVwT6VkVmGBqXtNJ+7uV33i8p5z5q7hUTYGQrsjIm8DC1XqWoHH\\\\\\\/0Kq9nldbdeR3PwQ2Jzje85+bgdA+hbk3tReeLg2gjqUxSd+JUBvKAOpRGzJDvMrAyB\\\\\\\/eUB+pWmQ+HM5u9I99aoe7XtroLDRZ5TAeDAz7vfLeR3wBnoNIrpaB\\\\\\\/TEtUQ5gfWzquhwGLsKThOcsGDrMuGocxazVUc4yCsGFLvF1t7Qi0BqLRK6iM2U1P26Pm77bd9tSwJV6XbbjQJiK6eyNSygoaupN0uTXlevsTr\\\\\\\/AgIz+Ahkq657LMZIa+4a0IeSMcdw0nJII3+fkWP4h3Qx0mnoaD0\\\\\\\/WcZBT\\\\\\\/nopksskFzjUvazdnjJaGo7uQCjc8iR4k9sKVQCJx5\\\\\\\/N6D0MwR+cuzcUi3H6NwZb0e1tihX37FrI04l0rqzWw6rcGfeR7wY0uZ+ty9zxacZlj60IGJP0Rs49pyDn4XcYXMOyoewewyIVRRd9+y6u+On1LW9MAlJkbg1PWmtZJ5XxHbW8Gu1Pe1pr0p+QIvpo4uaTRiUp8AglXj\\\\\\\/tFApv9Xqa2+4TaLjYazLUnZDEmZ66gWTEi++UBPTISehsxOd6GvwBXAD+dEhOELm35lFYT2hnk7PhbNIFBuEMmCeP+naepDTScAG47sqiYW57ZEtXrcQSCXxEzrgQiHnFhEAtN9x1k9IDloDlcBFx\\\\\\\/Nle7d03PF8dKvNhET17yNq+\\\\\\\/TIm1G61MYi64do5\\\\\\\/hUjriEG65ScZWl1ruOo\\\\\\\/X3dgUPdVPvyPh\\\\\\\/OIWD37eQsi8FNn\\\\\\\/ism7elfW4qy7yBrzCZTrB\\\\\\\/MFyX86JD3QGQtT3lLK4S+GvGGo4XIc3pIohuYVlOijGcVBe8odXx1f0J+M\\\\\\\/C6n8yxudc5WNZT72kRtGQldDzWP0XHW956\\\\\\\/vtfAiBPzDYrod70knbmjPClyNY405IVvD3RXT9\\\\\\\/kLJD3uK8DLfGafHHVvCIhwVk22kS59L0mi7y48UjHyDqYCjDr5KA0EHFDPvfV65ST0vX3ls839e15AkvLRb\\\\\\\/VlahPKjuQMj6adDPSOAeI54erX5MIVrTMVWPpQ41Sz+Gi4JWpfLaJfEH3e5XAQFkXOlfAIzfkDk2H9KE8vZQyBKovbzJjW+zZ4QMy6B9EdUuJdNfbUBrY1ReROCV3Ur\\\\\\\/qfm+jzIObbnzIQSNVm49OWXhvxWoPbOEbV2axiZT4zgyLTGbgPdDa5ak3dCxj7zDYcLvJycp8tqxcGEIUTXOi1RdcB6B1QQNxa2SuXvpKsGqaUS25EHU\\\\\\\/C2KInPWk+Mu7JH5465kfyCgklS+dampf9KnVWXqhEB2qhfpl2KyED6XpaZGMOkbyF5JpsCx5zmTuv8LS1Ihba8uSOOpXtKbHARpk+0Dmao9evEvRgOl22FuTrhlGEfSZkvGGIFuM\\\\\\\/M+SxKlQikdbFy92NCLmGCVx6UCrgkLbK2ZRjn1VPmdLtayu4QSsNbaMeg0FbqsEvNkkyxBw8D+7JQrp+JtPs7FIkI+UEbdgIQ04PPG+COW639cEYBG\\\\\\\/sruGhp3tnhG2i3wQKxOP4lr5ZQWnib0EdSW65zWx57ClewanHYZTzirNOBnpvvq90TaR77QnxjWzg47QVmOnx6SqdXmnqHM7wRmdHEcVgNjpsTRL2RtHEJVG+1BHsli5ACikEVv75XRM8YzUhT3\\\\\\\/eJhHzLPNxvEYGLtd0E8Cep7r3GCUQjmHZSSjMPQCQ0fSpW7qdH\\\\\\\/Kbv0D85TaRwFeq8E71wekSdh3VdOrPI9+\\\\\\\/c\\\\\\\/uuXBcfCA7YTp8Q+2sHzCXG0YAk0MnjCNHlIAKnPtx9rtq8oPMDl+ZGSxCT9eJtBh8CKHtnSVct6x8m9hKwXOpbbQNe+l7su2VWDTf4HjOn1VIp64Nwi9NSIq+lzUuyl8U4vj8+BI6KwXAFug1tblbG+4C8LulD+u1Gg+oqqgYOk1r4SDYbd7Q3R5rwafOmOEodrxIDgriwLPBZKFAvYwjealSxvmiBFowg3TLcMbKFjFOYP9N0TDnAaW264wKgOYDd\\\\\\\/46pmrDF\\\\\\\/iMRGo7wsXohLi1CcZYiV\\\\\\\/nBJwFtZ4BxCbq7AoF1lrv6lmGY9YGvjf+1rP5mMLHcPzt8wTCE1vPRblATSqdPppgi\\\\\\\/nM0roO\\\\\\\/0G7vGZ5QaCi2bnlQ6ilVpDu4G\\\\\\\/Qq8ZuAMGCGEwEMUsThEZNKz1Qv\\\\\\\/KA9CdbrYo9f6a9hVBfPQBEJEN\\\\\\\/T0wAl6dNXosEqA92+OJkZ4I0Wjf1etLZCitjoc6RwsrU1niKqBWYWvMyBq3tsuUk7REJ\\\\\\\/2P88VrBqJqruPbYYfyd+R+LsY5xljobpPSTm0PNuNO89M47NNonrmW77Uk7+lK0SyBjSpDsQmZNUIfc+\\\\\\\/yL23i59mlmJwDMTJeAtV35GBX9bSOuwwF4q1lXQACnjvifnCK6Bv9J88rsRwdd1YjyQRfLVAhSzwuzQEfsv2j6avGHLykxtserg0VWcxGCKmSGKNstihXFXNenNzN3JrEw\\\\\\\/ELPbVJMNY3bELoJVOk1489hwd\\\\\\\/nioBIv53egfXX4gx744i6F6rAnFaTxrlKxyV3QLkdp3zsd1ug6V6lfOeqU0CF7s+bmtUir\\\\\\\/eNGxV36FWMXEiFxZb8zpxDpOUkpdTrY2etVPLg\\\\\\\/FQe3NxYR7VfJeCYAIfvOf+VeEmibzcyD\\\\\\\/amsWOB+qBOiLzW8S36Yuq9O27qMOZh+rqFyiwOsqG5toVjI7oPJke\\\\\\\/x2+zn8UgcZ5uv7iUE9\\\\\\\/QihfzZ+A43dUiRzuCNsmi4JZA6nNyO31s6GB7ujqKE73+HCrluDKyOVc2UXfVnvIJjVxHqhyVkoBuNw5B8Ff7QwlfejD8nqNZOwPE+jfIgZyxmeTTndOV6ycHkQcfc\\\\\\\/I5XRx\\\\\\\/BID6udJsDvdemxSbVIGHtO9DchoogRr1Bba4ppdyqmrlGGw6dbHik4EUAmZPoLPtohQ+KVXbjYqopyARDfQAYdriIUDce5NzvtawsK9F0yNChnlXHn3FW0rvlFe+qe58kKuArT0G3HDx837Er9\\\\\\\/LQRMT7Uzl\\\\\\\/noRbJ\\\\\\\/7Fjw+1ciqSomW9C7FiV0lYfQUNA0RXOKKeluhtSt5Hs3C3IyJ+gjo0qhwdAejC8kIn8mGc4JXoVFlRUwBotMOq7AXpbfxPYC9Dxr5W+TLWa7f65CN\\\\\\\/iN6FK\\\\\\\/54p5TjXgQT6g7W8tXP8IBQn0y1F+OyxPkb1\\\\\\\/4c7rnqSOvn6990Jg9ejtXT+q1OoTeN\\\\\\\/LKBI1A1tRvJrv2ziXLCEk+7sRamRFObRS5z7cvOEx\\\\\\\/Y3VCF8vyj3ts9+M5vLlhxfko3zISYTg7G7ZKXCUB8Tjq7Q5Pb3qw5uOsL\\\\\\\/MgZNgZoUnNbELmLbcCJ9pppkkzDPKzM48pbptNO56ZLfDMfpfO4WDytPoeRoMK+aEmvZZkiDN0QgT1qH6yd93IwZIUHO0f7o9IqYhJc0ie1gj7Y1vmzzDGJ350kb+cX4J12cXUZgzFL9IduXnY\\\\\\\/PO8lCM\\\\\\\/Czw9PLYh3mFrgkN9PQzVaMgR73shABmpaKHOTjFfIRz9BrRKLmBdqANecypx7Pciv52LhNcqArLwcJ8Y6UZ1TzEjnloJRXpkhllaFFd1YGWPTFGGsRfR81XM285Hfl+kT+4SQXnHXnviY9NbzktbOYPQWkIn7Q9p1gAsfBokyaN2LZy47ml6lXrm3aQmxzY7t8j9BU93V1vNTuBvBl4tZMnQ4R0PvkxoBsusGEGD7U2SdoR1\\\\\\\/\\\\\\\/OoXQBxOlYMNcni8FAABsE8Mamiurc3j5CEnBYo3H8YGEQLlJCfpjjvFxHUEE085PewLEHASsJfDSLvg4bw4DGL+T38CeczuCXX\\\\\\\/b5Mw9QYVIVKZ7ZgChY0W+6zYN16WxCzWK11d0v2g+JwPnc+552mTCzMwgiiXpKs0WqIE39QTEWmKuMx603\\\\\\\/P3QR+KaMk9PoOFVDpc6gQRwjy325oMgozvhlPlwDVkL0wm0\\\\\\\/c6xHo45NnfGPqKo3ZzJSsNlk8hyK2lReMfSglbUZlpmxDYsRrB9NadtB\\\\\\\/4S8h9aULH5J0xgmhoeIJ2ljb4z407VnnMChKAlq4QqPajyINdYiTL57pIN6JfdAGkPdF8o0TdrYifgOOgNje+k0EeLvLYa9CC43BUOPyAPWEdQc2O3H0q0+1E7Qh\\\\\\\/PoEmhScKMRESeFTAKls\\\\\\\/IYUL75nnG8EQvJusoZKYOJTlFsVHyl9dc\\\\\\\/DWlh\\\\\\\/BrZEbavWCWqE3Wgz4r\\\\\\\/4Lve5BdVSjT2ig1OA20xCRupMw0ima8+Ky1164QwDsBw6fm3KHmY8tj7FPjr8ONGr\\\\\\\/aO965Gi23JA14MCTH0znfiEbMC8Ve8Yfzrl8zyZjQjrw0U3Zq125mT2l1W\\\\\\\/xZ5Yj8HY36+FC\\\\\\\/GPfV24dnTKtHvUUD4XXSnmxjSm8VY+6ZxwYxFgJm8g6qS5PbAGYsPP2h6RLpoGrp+S5KCLIMoTptSp4DgUzPREKXYjPFj3IkGmaHIg3fXc\\\\\\\/1CV6MYJosyQ2\\\\\\\/DZCt8bTZeR4SlKVaS72lM\\\\\\\/kZqD3cSFuoHDYLsf\\\\\\\/Kj4eMPXDMPBLlhKJA2tbnhPqy5y6MkAocdP4xj1aIqFtyCzSXfvCMr5j7b8SZw\\\\\\\/s5k4r4xl3PhWlhcXy67jbWoG13uM6\\\\\\\/MApTarkFC3nMRNLJ7eINlRfWAB+TTzhNCAjrJHRYvp5YopZ90crwnpC1+zNKS4lz0jPYuSZXSOpWp+1mp8GDUBSxEdTaA7tXNoY+AqNeXsvPWPQDnZQAE8jSgj5oorl4u30awcCySLIexRu9XccUvFvHiicdEU1Z6yHZej0cAK6mftM5\\\\\\\/UR1284TfZiFLzsf1O9lS8GjtSODMTdyteeHesHViSovuvR4FFUEjEwlq\\\\\\\/iuxkP85aQ0t0+Od28r3YbNEFjwxr7dHgElNshhrlhDQxEBE19ZxgDNwsbFRC57OcY09TZp194jtjz7RHFqkDF7chsi9aavj72VerjJELVOUzPzJqbY4DTEo1Kbyp3tyBB7pGU62jXwr5ip2RvGzlIXCIA8TgUAu0feADUGzfwq2bmLTLblSEhbA8qMNCgWQnR11Dp8wZFKsDd\\\\\\\/bq8PtNXPhmkdQnChKB+\\\\\\\/k9IVSiHBIvKTRgYuNOnUhofqB42LeGolq5fnPtSeWEwMALgGMpw8Rdgc6vLhSYE7yfFCeWiFH42vgWfRVmneXGKr5abucXZmXnEKgNPO+ze6qU3ZWwyeD3HlxPIx0BS7h5Kw1TvB0yEmBWrrK4a65tffPxLY2OBUlrEu3lIIx9xk1yO8Uu645q4JT4XpSnNgwaB\\\\\\\/EeyfnVDOCn1sOM23rh8g39BpJPzq6fJWELCaXBq3pp4UTeIjmzIHNrfkzeLxyab4nH8JNM1+S2HtvTMudcozhQPl6vk\\\\\\\/KBcF1c\\\\\\\/8wbadao91v7t1sVmIBslOdIkYDtuUU5iL1cNXD1YKU9fCmuvbPUIGjrELm9V62OJ4Lvac+FeIRgmwLxPaj06rDEJaFr1wSkMKepSCsdpbIAKkbsrixYPsJEPoEBTb2dtKo7dkDyjz50BZ+xZSMQuv4VyDWGqMvtf1L9BFrZn27cGGIH4C77Lxp2cTF4\\\\\\\/uuvKY8RPV\\\\\\\/9j1R6VaAkl4M1AqbK5neRX9jovOxEczka5kGcqOx5Ua+jxLy2N\\\\\\\/IrDexfloPQuQgFPb+mhUcUcUhu8s06heOIYy0hsxnksmNYu6YVE5mSMew03vS+9E5BniRTrciCzkmsqAu10jlyF\\\\\\\/5x6Hgr0ef6uu+aF9sRumKx3Zox888bhP82XO\\\\\\\/xcrM\\\\\\\/2yfLi9j5AUXIUgYNmFC1V03rL5Qrovn0vJ1dVxJk92Gi\\\\\\\/u55Iy2Eia6985XH4Z85xZeK4dr+LOhk68zMThWKqaHIE+31JIjGlBdA1+DU4avPTn0cNlITIP0cLndU6qa3cmBlhJz+40tc3jKzTuNPdu7BwsyKzBQt5QHkAt3xaKKFQccxXbV0bxYgf3pacbrD4LZsqmblRxQq5ABhrqYxn\\\\\\\/dbcVDFhyz6HLmWL+v\\\\\\\/JAcE7RrCkg1aOdbI4yUpw9iFlphVTjf\\\\\\\/HVwAabQCx0QFgTvpOLLWVqWWDqpFvOwcc6CPUJ7eT1i6llNdo3ylCiwAm+uXxo9q8K7Um6Uss7jTOtiKizVaOVROJ\\\\\\\/3VZRPdTT+4gvwb+FheL33gHfbRBR0wck5\\\\\\\/fXAh1fR1ZhS3xnpDyha0x2MHNFiH3cSvqIFwq21xAlfjJf1ZfUgxvR6s1tZAFZqCN1HPP3zCeS\\\\\\\/+\\\\\\\/VG6IgyCPD9XS7CBdp+gMwmHyoPQfXbWkuyy3bWhJzPkRrMDGsVb9klR7T8mUuNSueKGd5VRg5ORZKH+2yMEurl4HhCkwRUlt96kPuPw8oNIzR4p7BQRZaNdsxU6kz7SqnZ5hkRAlxJMuaEYs1p47BuEblhSLx8GLO06tS4Nn\\\\\\\/Xl2W29TcBrcFlSfmZBOCimV34A27JhgdlzUaUtXGLPlZvl7WirPDPjTGfkxMlLnPyMA51pglj2hG6kmxEwcp43jVmdWJLmcnVo6HkUDt4nViKZ8ayFO4t6qkoT\\\\\\\/oVhpFgAtDPMkgPuJL27IpFS\\\\\\\/4zDFgNFoQClAzlz47llUIzyqtBgmkpxWATF3n2T3WkbbNiiOb3IQw3nZf\\\\\\\/E9IrrUe2VPGNORbwkOoGFkwQoisFk8sau1XG4tIqTiCigsHcChnBbW0hIi\\\\\\\/JqanI5VBNkDb\\\\\\\/+sAv7hGh2GC9dC4uhAka6VSF42HDT3JXtBNBNpjvDWSa3cWBpDmV44kCa1pcjvg9gRweaHKGVAkEUJJvhVXny6kTV8FYip+IavZD9CTR4v\\\\\\\/2t+0AMzZfRLqZRvWRHFu0J+jogrr9apU+YCEEebz+M1N6gFuuytfwWlKfiEgjnbsIjYFJH1\\\\\\\/B96QtKF2KsLYcKmUY+ESAJXuvzUxdg9pqXHA1IYClBSZQBE\\\\\\\/rZtKmmdhJIihOJIUuiM8jrej0EfqDq4Ena8UnVWaDTKHmpw0BnJkcmFVGjyWAgWYeDKbYv\\\\\\\/9LqWXTbbmeURZan6\\\\\\\/183ys5RjE1caJUxRAb14jFWUtEohye0MSuc+J1\\\\\\\/x+8xGSDPR2OL+amwPXTsNp4+5GB03WXzoWpDKnDiLptjMGWilNhdFjiyy0KOVX1MWTFZ\\\\\\\/6TgFbxzBYKOOsDYec+K3+3+ttCtfHmIaffv\\\\\\\/zlQ6geExXUQBvTHYOMVW3axge9bB2ioNC4VGoK3AQVEqmwhn9OOHNRpwA2tAe17Fv\\\\\\\/X37yQUtWg0m7n1Wm9hMSJ8GXSC0jZQcvouiK\\\\\\\/MjXVRWwWDLGlBxZzweLF9shrzdGlRo3xuCrQAfaA1LZvKU5mayUssxHkW0owrMktGlrmbzedlu7B1B3n5vll0xawi4lGHfCtI4Bbtkcxv\\\\\\\/5z8KMA\\\\\\\/daZHA7BRS6vgdt\\\\\\\/zWboJUDZjet\\\\\\\/WPwATpG9GzDDo\\\\\\\/M3ibOUlRH\\\\\\\/XNnmdxLVSv9OrVM31Z\\\\\\\/rYb2DqZzHFnJu8R33+7CuofvWAITNzWfEFzpFVIF12YEPQSNZOXr1MNBXemqpAUbeOfY8mwCuoppuBY\\\\\\\/4nP7BoZY8\\\\\\\/PudPSkADmXTD73Xu\\\\\\\/zZTo+VsxdfWVy3JC2UgtUapmK5b\\\\\\\/rtxUplwI+1hAzrtuQJKp53qcfDgVFIwNaFFsVD5\\\\\\\/fKbRt+zN\\\\\\\/65d2g0VMt1lwt4oIf3E30nbin3jOmzydO3ty0GC71wW5oA2Z0fgGdWU7LITQtnmS52JfP\\\\\\\/hAPiBjhvmCn+o9QIddXSnpmQ4aYm\\\\\\\/0DZO7XqFQu1rOvJKN1ki7YFtyeS3UZvLSSQmjp1X1UUoMwdka2PB7bAZq379xTMZeEW76fsUFehkbtCA+jzouenftEJvvPyoM5p9\\\\\\\/V++BCfhgx+zAB6ZFfxeGwo4uc9BeNo9HfQgj+r4JFKNZOuagDsOyw\\\\\\\/Ro8sRDJJ4JaccNFF8JnEHto1oIxs9IWoCvSW8c+Q1TLab0norw6lK66feLYmIHNHLZFzt7FJxteBLDho2VC2AK85gynE3pw8Fq7hDlVehVFi6R8SSwxALwEVb1j22mntAG5vIl\\\\\\\/i0dQPxVPR+EXh02N6HIlsXmvzLSsWV3ezYV84YR\\\\\\\/QaTlNHmCK3dswnM3zLxIGmaY6FQa8Tm\\\\\\\/wgXuXIvfuVcVcL5hQ2wJoXXqotir9jHf5MBxWHPL83bNQ9qxXQ+e03ivGORAWTuX7ht6A5zRqHwQ6qzIZ8fVWtvCVUQqVf6brgpTlrJY6V\\\\\\\/pl6lDci1UAWvpZGQx1DcFiIBTQ4StLypE15SiZZS5Ndeo6Pg5R6B4fXvr8a8gIUk3lJ8tNII4mdr1w9wdkTaQaADQZyhniSw\\\\\\\/WHBXwLW5RIEZDBlo3NFAej6o8nFUlA5aBuRyhsgMbbx9j4pmqP4p4Rt9HSwe24atly5E3QOYHryu2uHXxy16vj9cKA5zmgC+BUibaEScSO+mDUWlK2SLKuC1bJHaWPU037\\\\\\\/ZBMqM2tEAbz+kZJ\\\\\\\/9OjaXm\\\\\\\/yBrXftXYSMVBywGcYGJ0xYfAZ9NYt\\\\\\\/oCtUOn+3K1LfjQYDZa0zJxpcACQyk0MALyBgZh7zz5E9F7bABeEtGtcwExEEzuXRGzU0s3ANNRZOwdp9wuOJpqk\\\\\\\/mD7nklhG0WiTNwyxzC8tuAkichC+7m+xPIk6shvrTdF7cyoz7h82wTKzS0+7DEd\\\\\\\/xO4aCn6ozMnm7EJmnCWGzIglVBCZ\\\\\\\/wXyAmP\\\\\\\/yXpoAk6z7FE5T5vnjqU\\\\\\\/vEGozjfGRuD1bPB1Jn7m8Id6BQdrsssqyNB2KwvzOpfmSUScZGLIaBPaspoX\\\\\\\/mS9E5qwqLqT7Cc\\\\\\\/AZ73pv8Y\\\\\\\/XgiLYn+5627NjDa4Q1M2quZR1pcpgpZIFjM5Z09trXjgBs\\\\\\\/vd9z7l+tmLEeeyfRuyN0ORiG48u6VH2L+ebMUZiWJJVy6Fy4foD9O2IAoL+nlX\\\\\\\/L\\\\\\\/t0JotHmco6GkgQVynFv2rEeIyDty78nSB46cbbV0wLriVq8aV7XeeNfi3Jkp1WNMvDglC5nq4tutPTwOlrxG9Fo9XJSR\\\\\\\/tmpijWdVv+TfGY1BZE7myzLxvGQMs41eNZjKZiPQGuwqGo8uN+LN9CD3KW9afJVoA9W1XyrTjfqz\\\\\\\/FKZ35PUUQKwXy8dSsTKFHrE+N3p\\\\\\\/hHsXmneyiEKFhrw9Ns6tUyiUQLFQpxopjOMdnOFm5GEI+11b548V0m72K1AGc4qgLV9w6WCVXvRCK1gWjj0tYKamyPeZ94+NrbG\\\\\\\/MpGqCHwEMBiSD8\\\\\\\/EvusTQvmvdy0oJ03T2iCxjgZl2zsR4wrhhEvvkqPb4DzJsCtSGi+DeYW5QJeSyho5qSvr8Ykg\\\\\\\/RWNBp0pJePASPHnamln6LvSAXyQPmr8xVuIo4D+RcTZYLKs6UtvdOWEilVu4+SNMuZMP3svoQvxz2C1Rg0WRoHSuFLuNmeB9nSK8zFQX+hGNfWBqCXb\\\\\\\/0TWQWHG9uILTmcPVD2Wg8jeUYoAeQefh04KTuuJSWMC7ClTi107ysjuHAI2NzMsVFZG812NznJ6zPy2MAjfqsfht7ghapZieun4RAUXxj4m3kORCS3FRJAxeE1isz31E5ediOkoXlQn9ig15Th\\\\\\\/Njv\\\\\\\/asw6RU5sDef3wGkaaRdsw\\\\\\\/X3+q2VXh10Ub3pnKIFkfo8ONLfhWKCaFVqeq5cekF+PoaYO25btcSFN15TesK7yV9DXC6NY9mrROlD\\\\\\\/HScdgIjCKJQMQXGv87SludKg+azMb0b3VOG1VD10S\\\\\\\/oub3YivVDH+Ab\\\\\\\/AGmGLCosGKsxDbbsfVvN1xccEWwq\\\\\\\/dcv6TeENvLY+SNZb\\\\\\\/XihfKLlRxm1GRIh7i9YfxVj5iMaNCRlh60g\\\\\\\/6ZdlNm8bPrdTNIBUx+g0LybfuANC2lEOgoQ\\\\\\\/ue29bJ5+YPKjavWhK\\\\\\\/G6t84jLmw0xjmNOkgSqOT\\\\\\\/hE9GmGfKWWzRkOrP0f5GNwGjv4HkQtAfe77ckHPJkPd+zRIRZKDSbtHEoqW9Dr073\\\\\\\/d3IdrVpic9JNXoWZFyE2Xqj3sPUeubTJp3zMI5025EkPE5FjE7VM+zDVxt1FmhzZm1h\\\\\\\/nmBPgULdVrVTnmFW5TxCc5AJ56PHuw9ixW7DOkNFvbGsWKnBuEINJ1Vbj13QFujMFbCe5w+RocY6U7euMEvspna3eG90\\\\\\\/20IvVkzbr4imEvGZISJDVVQDASo0ZCb9K6B\\\\\\\/lUwybaHI6FJpLPLvm02nItUeex\\\\\\\/PMJujc9tYuB4k1wHhIu30efmxCN+N2xoxQY53kAiIETI68CftFdcKNy52GVhI\\\\\\\/tPHdRqau6tmwlM3Ca3zUtm0wfhYcMqr\\\\\\\/qzdTvqAVmVxrmbwAkzswNiePoXjIiLzv486RM3J6UzYcQfwTzrCYisnDszmfKvztO1lZgzZGScOqpp8M2DYhZ2\\\\\\\/RND4x1tzfh5Ese4khkeFFShgzRkRo5xxaLQa40nK\\\\\\\/AW39TQtxnxkPyVtuBAVRCzHrdU2YSEgFVnUujuj9A3pgCA8xAI7pvN9OEzQI+AGcq25ZvU3kY5qGZJQsD\\\\\\\/IBCopcgBPsGVsTZ6SNyGDdp9GFGR8tHTW0A2QulOTiYFzH38S+d9Ikw3doIzpGUWxfkIdDoGe50g1GFmtFSlkJdq7di8HdQNib4gNkX4asVD7akYmamtgXTdtIkDIDpv3RL7yc94KDdK\\\\\\\/q+XiIxr5E9UsxXanbx+TwDGICcOearx1ltM7wzG5f6Yq91ftjI4JDchcZ+9fxjvc5sb\\\\\\\/POAMjYa6hLz7yCwHYgE+5yizRP9MheOn14\\\\\\\/u5Cg1yDPeymJDpigwqlKH53Ms\\\\\\\/X1FLkg7ZwpNNcyGcp++y4L8WKEO0NlZTNE\\\\\\\/2M0BTjkqIWbVIWLuSbBKrng6zI72c1npql+pZqMn3R32Dam5du8psb\\\\\\\/g4bWszwADIUiHDw1rYHrIxxkj6Por+7pPp2voqtlmSqcKvoOsnYIOFpaABltWzhg7IwP+Y0vfomruwD2IcRlyGTaax8QvBkE5aY5jPGKxngbwE9oNVSL4E4Qx7nhRG5SwmLiiKiOZE+4gPS\\\\\\\/5Od0az6pWEP87MLDaRFDcQ\\\\\\\/uA748+XdOWKwfhgER6Obp7KEAxwLXeLxo3YF3uPDcv0SYpwz\\\\\\\/VDcmSaWOoiiZXRO7Z7q5TTX85og4Wnrx3UOFgOWtnzwwhPW5BqBnvpiHNJrmT3nADA393jsO6F246kGE\\\\\\\/bqNzidCbDLQ06t5vAecEYcOZyYAXLvXkbKdEuUU32rKjS3E6P6HHqJD08lWIFvL0mDKQL+0MzZ5GGV7SL306ubWyez4jKR0leHZPhu1GLZTeFwdRv547oszzKIArlq7vii4Ba5MjUeW+4p8iLfI\\\\\\\/8jSKDTKQ+JLfKAoLfDqb7pdI17oSx\\\\\\\/NjFHPAXGAocJOpT28p3OZZpaRVILJuxj8BuHyD8ewgmsMlyRarn0qDrQi+cvp7kGWC0F0j42jFmt9O4GEoeNNbwn6CVP3hrpej0\\\\\\\/4FYPCU2O1GOWahhyxmrRyFcKEi6qhNm3uicL8+xYVlKgOaAbty51wVk6I8H7dcUAc0K3U6K8a25jyPEVJJjp1o5wB7KNrI2GExcxRaodyVy1xjvGIWpaQlPKPvjDtqC2pDZ9Ca9nJr0qIJr8iUqaQWaKFPfd5MvPqVWsb5\\\\\\\/FboUNg+kMNidcyXKMVSNDq75F84V0D8JX208P8tfQBIqbnyXf2i09u0EerySa9fvhcuCqnR1T1fmH6Imtgh6N2UW0i+ySSwEg4vEvBmjUjw5uwTHkRikLyp7IBb\\\\\\\/nTj2Oz3K77qsWEW82mw2zIUT0tdlXnIbIGt5ACKuDmfofLu6K9SfiEvOLHqKMOwfL5j8alPzllLIcr8tDEkkS5CW8\\\\\\\/pljrsg\\\\\\\/LMnNn3qn72Mkm\\\\\\\/7eath6vL8OW8YbFhzi2DfI5SFeQB6GlRSLOhvHMqtE\\\\\\\/QdpejT6TncaE8opv0s9B2FAzrsOM8jEBM5U\\\\\\\/YaaU3ug9qIMOsS5X5kodTY2+0IORGSysC2T6h3wm4wXXzUbvVaiKZ8rLpUj1MzELbj7CjF2ZR6PXEPs0KTOMBX4Q5gECiZPUT+FVY086wbTS+ZAVkYLkvgBFhyHPLKO52UcU13+C2iKnQwuBwOseyGfFgunEAe\\\\\\\/t3twI4cXhf8dqvMI\\\\\\\/neqvPZzh6lcyHCz\\\\\\\/+4ivCOUuHJ48O7sZSfyar+RKOhw1jAOeapl4JezYceOQUulPa3uoqDM+qqRI4j\\\\\\\/5BLtob1\\\\\\\/jnAnoAuQReQzpxeagdFEym0rg4W8K9OsXnwGDHXSu4OjsbuG0DDuzvon80LblDjEeY36AjUOTMjh1f9NHEqUj1djxeYHvMFHea6mfTjWu1I5HEwP+c5A7JWkB5nU5AXnSqGkfDkbeZlr0nh7HVYASCQvl1khow9VSZ+QJGpQmaG7odgK8VyD2x4TKkdhFaGT9wRN7BfqngV1qU2Nyon9t601J2PN0iXGMYv7Bef12qy4UhL63Tb6gF1DPD34UnJg0+gLYTg6IFE4lzODP5Q2AB4lMzrLaLRerwe1X6jZgI8dSn\\\\\\\/v2xwa+vGecy6QOxzTWZ7COItTzs2IE\\\\\\\/cI3dWTcfNS7e5iTOwM9DDfHC7eYuE9mYFiF49v742DXiwQEvgCDOUVH5bPrpkZNAT3RfuJnIG2lWlfJsonipAsfMd6xwhyWwdQCf7ajNlLPgW4TRfZpS7CjQXZjz9h4eprOgB\\\\\\\/b2+7vDxeMTtpLi80bY2Sf2w\\\\\\\/+IkqjA\\\\\\\/fQV5Fil1hv6dcKJn0Brl1u1vrQAXT52a5qaPotaCITG3pyXB6budrFzHzQi4e\\\\\\\/UXZz13zY87zyyq8TFZD+Ikus+43Hg3tuUO+sikigrnoPm4ofW3lCMdLdfSW6q94BbTILZH9dv1yJCXNi1frNOWJbXiXChyXR9tXRmbBfQDGwkWq\\\\\\\/7RaGnuERlSHA4mcLTceGEEf8k2935EgIRjC7iIvOQvA4c5TKWUzF5nDaV\\\\\\\/scYFF6xnrTvLujA7Gds1f3YiQFKd4HIl4jw1p6z\\\\\\\/S\\\\\\\/BzYYYasFya8aKLm2hm+u5JeYF93sYyygP2qBtB40aE4IM4amJIleLkd27+5dV+0vqZdS1nKPno5O9NP9bGgDSg764cQ5+pHQRuZUlHl\\\\\\\/dMxYbIGksiC3oJIPk0OIO9CHsbVjyU5ChA51cruH86P\\\\\\\/W2SUeikdGWFuRxOyBcZThIgVtjFjS+sGgtzzmI0nq+iFtco58dB0Sq+igy5vx4RhK8NhP7avNzjmywqs1lXTy1yTN+mfJ5Kj\\\\\\\/uUwmNWp32NPb13wKuOr0pcmCRw7u0EHy1UCqa8pBuqLml3irBs2IzHt3esDJSnlhm\\\\\\\/W71jgDXME+OOp6Ke7gqBzZxT6rkslWolbw5hAJLOOQmz0Gsz+lKj51etRIKBWF6HckBVZx4kcW6dS8iKFPB+TH7vUrn6+uD0VnBz4QuiOARhNgWMh7rGHl9rH0xWvJ0NuermW8GINalGrB7SMB2U35JyE5UEAmQYzxvqlV5m5dUmPtQYCrP\\\\\\\/ns+5arAAS54zRZAjWjf3QomAet5K5bgqtSwdMXia3bSlIGHwMC7BwOme0UOnWacBEVT1pvEAYnb5RzSL63IVFVtbM5w5uq5Cedl1nx0kshGM5ry45LLyMF2bAkMcdp1fd\\\\\\\/AiumIfhdSHcu04iucH0Z1HZ5HVnHycwVQPv1Db84xlTxz8qnb0+xhPW5iWqeXrHoCjTUBx05ftReGXuhH5jsOzmJhy6Ym8pxors2Hv56sS86XMavhg5W+oEqPm1oOWQcZ8EZKD1PCsOnOcAxuUX\\\\\\\/wk+YymUJwT0k+d1z6JExUh5zxavvPIf9WaKoOZftQUEEM11q9AsbydLuIVzUuBKdYUVYG9JC89hoO0Zgc3hOEKmu\\\\\\\/SBVEKiw+GbUxX\\\\\\\/wcW4m0ezZVhL9yi5gqy4KQko2ONehXIIgb7EdOM5rRFnApZuwT0l+XJuS5nLzVxF2lKIo2qFZ9O3bTMEFUOrgyDjm1xG3Oxuvw0hSDUALJ6XwY2BhvAipmDjsM3df78tBSjrKJOp3uqLVAXD2lM2k8\\\\\\\/hAgPMpe\\\\\\\/XiTe+xQijcfELDPLi\\\\\\\/VXD2O+zy5vO0xxt26Nt006d+eQATaFU4KPL6DRFVYJAHWITfsiKAyHvp6kWXwq5Nv2lEmUqsqAw95wa3OIt0IWqOdC\\\\\\\/4eJ0Mm2f2Y4fiClM78OvUsaX4SsTgZS1N0Vf76DlNthKAMvwo6xfhyGZBBhZjKuEW2cL8tdtnoe58gmAvnmO74mh7LurSrLN7lM932tp+da\\\\\\\/CcIFhoHLE7lNoQTCUHBX2JhAurUMwKrao\\\\\\\/JuJiryU+IZnGTo\\\\\\\/JeoxfPF22sheKoFQcmAethhSim5INdhEowCKQegI0k9Ki9PHDtT2IvMr3iXjY+efKUhkkZWkXPSbTNLQN3tYWqA3XvHiHPhwKLBfcVPdiaLxUlJJTl36t\\\\\\\/RfDMK2G4D8TbNbqBHyEhVx6C81VG9PQF1BRpdddtvGj0TSbYHAGVc8JRi4JJy7CIsPtrX8WQ9e+o+O\\\\\\\/qXFjTVYpOIlNNtmG5dyBiJNuKy2aHjc04xGbuMVW48VpaLQCHSEJsf1z6VJAIkZtkizLr6IPWxf4HInvLZYiovjVq8\\\\\\\/+G+dQKq8HFJuJ2WR74DanrWRZLvATcM3giTWlDIuvhOy02Xau6EaUTOtrxDDXawRaEBSLGHrvQZ8BpJI\\\\\\\/\\\\\\\/AOV1U\\\\\\\/bgBCI5\\\\\\\/tvu2\\\\\\\/kb7iBeMKoTOzMrlfplsrql7jXoZS9sPw0ql7fZ07qRxnCUvT2ESIOeLVr7bO\\\\\\\/Lw3Uoo4eRYOJ6MXlJ\\\\\\\/xJXI72v49ClRJeQ3MX1UzMwFsfPC9CodeLwhny\\\\\\\/fM0y43eLcYmL9\\\\\\\/zI16aCGknnJuNaao2RdEsMaWSKBVgK+D6k2+AY0MmwYMum1th\\\\\\\/n\\\\\\\/wweQpurWtI5xY6vCZrj2QMunvIvppTw\\\\\\\/6qgsVmn0kP2Vl9ZE3kXN\\\\\\\/ZyCuqZ6owYLS4rkj1Ndax7j4mhzEF2qcgGefrhn\\\\\\\/pxZEz8ZRUQh8VNaPJRH2AZG6OHipwG3O6XxeJtWItNJwrKl\\\\\\\/7e18pXM+OInipXv4K51Lp9pi1Ra5oR7XNzxi52FlBoSO4fnd8YVuUmvtXSafzlUjsdIiLxOsGCibDVrU8yrtrMAXBbMAWqtX2p3cmRzy0HHR5L+U+VKNdfc8hztR3UZr9HGF1o38rrOyS6GbIEbOKsQwEfuNH19sgKw43zMry1hPorDX89xoxZE7leBRkpKLh8X8FwIildnaoNLNuMoJ+nwVbhB4CmTeW81RAYzdY5VT7z\\\\\\\/yH1e5Z6NaQ0FkHeLsGiM27+f1TkCUv800VlEshYP4xqY1P+nESAiLT1dQGER09A\\\\\\\/djCLU90rSjHVtDVfp8EAUoaIIP8Laq3zmPe3lVATN67dyV1VaaRqAqbR5iAQ5N1T0a0AUB7QQm+YgLNB77Ig4DmBxMxiFjcoPD5e2\\\\\\\/PpuKAhNdxtRu0Vav7cX3RgcsGpri+bofUnqSchrO\\\\\\\/6M+QhUSpoo8piFqupe\\\\\\\/scEBLFKnwYxAd0t4wEHMlN3WnF008B5heiInfDEPbb+e7qAy9eaPVGYHjzqmUqW0dYNWOe59udWxS2VgCK1yW9LdRVuFOo85VrATSNCkJmjSg12u9SbLCs1lEmZXP35x2FHqkflsaXWfQtjA8fzhzemyR5y85+zAZo2\\\\\\\/lCK1E6cfq+eq0QZ6hjcLm0nLQXG8Py4SS5gLd+Kckzv523q5a9MHWRlMTaYIN1VKC\\\\\\\/czyCgrwBN3Sb8doVf1FhAeQ5ClVZ0NnNwc7R+Tn8y8rWkZ74GKuETJkaZiw6\\\\\\\/iyWKi91k2+PtC+Lv7W7t89QhWFoeVO8MKGtFMBqBL9FbjPNwQcXDhWX9gJCHMQYsR\\\\\\\/UTBIyacj5SnhfejlfcJGkATzAPhyRO3rLDvWQWd2OSLnY2VsBEwV6ZceuCWhc1FUsaiPRJf706sdxhAbL1ng9SrvgKMHMpyiJIgm9x9kgFljI2TVbLwLXKJOxKPVmMkmNSIc54UkfqYm1J9WnB+Cj9I9tmAkQhU\\\\\\\/0dO7Ot1UDIsGQ9+WMKbKtIXva\\\\\\\/i8q35bwJo7xpaR3ScJoSR5q9p6wVvh5kOVvryGJRejAAG8++P2QjD14GtxhzJaiAIHx34evyMyCV964cSbKPxTixlfrfMFSIgLNHglRXhe4SMBbX8fITJ+278SUUNHIH0vRLKWFiqpE7uflPAt9G1gEpsQV7GrH9gdIkp0LDE\\\\\\\/e8\\\\\\\/pmtuXoeRTSKLVh8LpAMhYA0rUre\\\\\\\/XdaQDZwgJEXe0MAbVa\\\\\\\/oR6jeOok4F+C7it1L57wQmQeJBlBnuKBIUaTYp+c8YjfgOtDIpgRFhGVA8Bw+BdfMuu9hbxkiDb5anPXi7JQxIlDeMjPiDuq1XCACDoHU55zg7yXEMbiNxX4sWZmqD7BEMz8hcl6rVl+qiDKA5owAtD3n4SnP7XtG4GplmjQXcrp7BJroexzTbnZm9WIKXEWs45t\\\\\\\/+8muT51c9kC62rTIVMCpQhXtUKypOxokXi21zQoq\\\\\\\/RMlFXJ+nSnyKxsRiDX0qv4ycIsHVct9Ux544LQ2JMYfK\\\\\\\/O6GCPCHs7ossQJpL7mjdBZ\\\\\\\/7I0YJuQOkldFq6m1zePy5fYcCU5qrqu73fHCZsFB4s+VJGgnL00uI2jexmyqRrff+xeVi5C4XM7VtJ2HQHB0YhoD8CHUZeY2r\\\\\\\/ynaLixYuC\\\\\\\/9g4f3Vy2FsPMDRSjArkPT8DF9TJ9lt15\\\\\\\/xpn2GRl6lmV9aHCMDou9gG1mss0m4qeNwdsc8y2MgO0uPxpv4+pQujKEcZs+ErudPdArGGE35hqHPlxIUzHR+OdK7QtnH9vA7UjT+E8oIiIXb27DegmW4arCq\\\\\\\/hhC4CogO2Q5Ujl5Bcy84LHjzkk86ZJ2+B83odt5fpVhbiekqqb\\\\\\\/ZH5rA\\\\\\\/OXjbvixTxum3wAM8N\\\\\\\/7iawlvo+Cp35BnwWc\\\\\\\/3oZUUy43wNs3ATO0qWbyn34Q3dVWDQ9kZt\\\\\\\/Ja+XiPqKzVmKa+6Tts7UnlB3+z62dcZWtkhR1qJB24wO2RxycomO0bAQoejBbdg0UvMQl45yNZM9zhk3d6OGcRCFG\\\\\\\/M66MKcKB1z+PfMmNwAmES3KctPsbM77+XhcK08vC\\\\\\\/xklEvMtNuKa5jH6+t+fWBbDd5sRaM5eIiZL\\\\\\\/zcsFslTiLRm52r7MyyJTBKXeN\\\\\\\/1ra+LmIGyk1nAq2OpiPKE0SMrPcSDRbjYaKGYPOCJnuj4w2LnkIDp9Is+KaLkYFhgq1SKqPLNgNLzu7ixj0KxUpp84emkYkl54SmLy1Ju1gCfRvynjGY1B7ZG+hWIngZvDY2hrWKsYDDC1KUTUJZBhyYQSessR7GKkSwVMIWdXHq4qqMl77WJgzACaH1YumizZPDh7+Mxl7INUyE0SGjylKB2M0BFaBgjrmb0ewibcYMlfOTsYCRkCf4QZPF7TzI9sizdzJr0TD2PjleM8gmn4NJfQpzLHx4zl2ZQajV2DMOmFGa8B1iQwt61SCYnt45ctwfC6BVWhAuMDtj8BiIS1cZLokJSXhhGXAXmZrAOP4LVGv0QVESSVJ1CyKUqo9nK3JvLNJoCpe0phxo51r7DCiNsgtd7idRb2yXAQBwy012yOKc+heZvHDLNDWSRU08HcbGQBqosfX1+CLYGD08Z74EhNyWfF3+H9mUmtyaYBkTFkVtE0DOzEJI0q1EbcC1kqRGiuTz+smJnMX5+4jDpRPxLUG3PFRRTLGdrNn6jairH40ntEvkVQGVz8sa2o7OQZPa5nwrvHrqruhJ7mdIsS+\\\\\\\/4KlyS8kgMNW02BTiPnzIV7MDXb6ifKw4PdzZqj9nvcklOztz3f18zDrYwJwT+94OBN5xmcNpUpIxftC1XT0JJPB8+Ryvp7q6TDbs6ZRcBMk+zHR1DWc3MaIZw+7t8LHFbM+mrS6ODaKK5kvjE7+yHiruvrfb4m0y07H5plIDHJg1fOueELcgifLg3g3FoaEu2Y3aO+shZorD\\\\\\\/kLnmyvj16gGncPTfQO44msnfkQ6EJNOGkGu67OwIC6bdPwEli8Xd\\\\\\\/\\\\\\\/p1df2AE2fmgzYhh6lzFlqqth7U2BORHzi66Z55wSGydarZ6oim2ATPuHo0qDDov3fg5H7YQLaKPxPYhxAQ6vwZIBJ5RbuDR5ZBaSj\\\\\\\/X4pq1fXT\\\\\\\/As1Ic0FpAS5BpDDaHYNkVvfcVYJoiQrf77UqU8bEvjmYEa5XmiTNn1n0OBdAm3mesAJB\\\\\\\/mD7b8DULyRwCP9a4UgtNee5uyfxNsYkWjRUQf9hB9GiiaHt4yFiNHezCVADPLZdGT8X6Q\\\\\\\/wa1kde1Csz75b7WLGvRXrGK\\\\\\\/sfbQUCc2ri7nDSzZoBmBwbbpovmZb7k7pTTomD8WgIkfyN5z\\\\\\\/7UjeAk0VXWPzaikEqcq5RcGQh7c+Xh\\\\\\\/L4\\\\\\\/XqLRTiinqgWzI\\\\\\\/xGCJzYZx1sfe\\\\\\\/rPYh3gKNaA+81fJ67UqKIqrlmn6xP\\\\\\\/2n18y+D2mtspesFK49OPludTGSFs88rwwuZLb2hiVALFKAzHHmZi4C2aFS9zC3AqC0YXQE4eExtZ8HeVHAz\\\\\\\/95P5hGsb6a+U4ls+yCeZQeGFfnWeTC59ugAT3wyRvIN1XOprZ3sFKvEtbddK1G+TVXzT2Pyf0jzlMuBn32QnWWVCXXRHyvEn\\\\\\\/mAWzpJ+SZvAQDpylwU3P88IqPV1DecQzlmhsXTlG56oKEq5lMSUOEu8RHnVcmHMgA\\\\\\\/pFMiabZnODAUwbf0uGFLflmAVwTGjA6O38iv5LqzF+dlbco6aPTdW3toiT3tds+bGi791O7mRdbAA8Jq0NycvaBxW0ZXEL\\\\\\\/qZptmGGuOYvuWS0\\\\\\\/l5U+PcQ5xbsRpbVf3Mdgf4blS2UIB9QYH94ouCxVjfzZzgKhEjdR1xcRXu2v9I6jyY41z+GvpZN82a6s2POAUP5LZQNo3c6NJMWlpv6HWaCymZbvLsE6Xu8ltLcl\\\\\\\/iKsbkGwiFRFA3iKhFiU1YNGWwHlLX+7wfSWD9tcit6SI+Gcs+FZdUaRJ55m1CXySoJD6rCOH0lkNKxIBNyPdrHIgo2K1Mx1+2+rx7u2hV+OviQOWTNhF47QhnRYVc65R10kdsq5vTzMyViJcuN8xenYOdWuujjonKxDiQy4BAYnGYoixuaEMWKyr5R38FmYpXYb+dZ8KVRPEYJkM\\\\\\\/O1S3qg+eWku3MkJ0\\\\\\\/qw4od23npUzYTmXkIgmNcvy8osSlhCnL76KLJmgTZfeLL18RkLn1QEuqcCFxN47WDXMmvshy18krRXH1DKYXaH1+DmCVXneMg5M9mVsX5xj9VY0g1f34wsMqLRGVx7fTXZePXlHUXOIABgwEOTQhUWhWasVtXs1xO5T\\\\\\\/jBeqHmyFAr7SflMedh6FSWKk+7jW7WoiNYfTC0kJj1WGsqDcLeXqBX1Ds9\\\\\\\/kmutWG7sT2nNdwAXnezEnsIThPNaY6lm6VJomqw0WFdPx93BSxJchi8DkkKMFtgVcQ7HSIAwo3z8jvrZ1drk0cp\\\\\\\/gHy002gtVtPNoU\\\\\\\/50pVChzciPsN3qaDcXlrp0zH2npXyKtk8vCl31+22MmNRFDn1Ha8ZVDuINvjf133+IAKnzLSp2OUp3ZYGjyP7jUnhkPKIQJLSkPzMQ4I\\\\\\\/SrWwbjxAibQgyA2xh0bk5pD1OLHfxjfDf\\\\\\\/0B3ppkox3QFEtXbKxlESEBU6vP75hlhufHM3uNTseoZaAFihURhbamugCdUv6ycTsdHshtnSfMnkBndsEgaZrdG+tf9CuRJeyDaWHhkyXJHjAf9Cw1FMA0A1vVZ7XSGVZocEiV6zcEM1s5R717pPfTXpLHOMD1xMYotapeixUtSGlYyXVS\\\\\\\/4iILD6KIucr7ck930mzBU3n+YsMJEli3AcLyjFwg3dnaItaJB\\\\\\\/v3si6cCe1ahnOXOmey\\\\\\\/Db12YHq5z0mWVGjNIidrc\\\\\\\/0sl6Euic1BLnHeRBehhM2OLGyDcWvocCcU\\\\\\\/xS70Q9B8Qu6hVHl9cDB0kivwKjtLcimmjT9auDD+MeQintxizUP3TR2ZkPkOBMealeaQfXnJAlvD8j2Pk1ZeMQSz72AAzhUnQWYe9SExcgr3ICQgBZr+UFPzaGPCHfbKm67CfiMGGcWaHLTb2Uu189Kmy5MR3XvmrDuBfkQWwLQpKlCKny21ISksC7BYUTrrXp1CGWSsFi9CXwX5scEBwDMr5Op1MQwZORCWCwjzwTpw5m9OgKsF3ehFc0SIIUPpBxQeK95Qc2xHllPMIPEplYxiv9uYKDOcPfSgoQN\\\\\\\/nYvX8uam13po+w\\\\\\\/N3uiM5u9\\\\\\\/vtyagb69PDbgVhPMCCTwe9pDzL1DwFuVy6u6bIV+nPPCRhEBs0bMG84laT+v8bxsPW0naV4luontip+4zLAvyiEIoEr3ApJn582hJ4xaHyLUJADlCosGjHhHOCc1wTd4ubyClABYDKMTErWyCE\\\\\\\/tPy5DmpeiYAemvsFBLK0U\\\\\\\/qgz0Lm1QqpHHyVAF2T+nvx\\\\\\\/zN\\\\\\\/Eqz3Mz7fM0IxVy51Sj5\\\\\\\/BTyW8RJYphU3710pq04OcZzhoOh47d\\\\\\\/60M\\\\\\\/KF8Edraq4DQyqrGtcmleIwXNOEcAwuocPyYU7b\\\\\\\/1fWT8Zk0JpRVj2+8GE8N56SaFejZYLMkC2zaQHFUjSA91CPthC5asOSh98yZ0SOkM9TF+n2hUBpXBW7SWfyR3opha6RyNSdOzj3Ok6TYxoTP8MC69AMigsCx2zh8xJtOXY0XGrcPmNMkLvAQ8rALLgM4UT6hoFnPlZrhIZiMMVqxsZ+MfQ3Spa9EGsCETbldlEC+pOs0OhBxfkEx6LCmG35uDdjCZpqtZbAln19dh00Ion\\\\\\\/B5DiLeYKp\\\\\\\/pQzhOmcZHqoM9aUA5R8SHiGuEt97JgAb7NHIz1pPjtX4tf0dFFjraW\\\\\\\/kL7D+Ln\\\\\\\/xAqGko2izS60zdvOmz+2KvPeqCIC9Ibuk7K6rCJ4gkLIeVKiM+WzZN5QdUHItqY5Po1ulwY5784kaOxDw2BeTM9PdJ3rXLI4tX71wUeL9n4ipHDFTVj+Vl2tIG2oPuqghfeCzifMYx6CldbGI6zgLG9BGCNuyQoPA009IDj6MW54rW8a5wnljFMe44PyJksRNmlS64hEwi4yrg43xUIXQWSdJlnAHm2ybeP6v9YoZ+C4m2sPVInNXdOHcQIBuKlbpGQzK0nf1+2Q27W4dhVKO5DwKEZQcaMN6NcXsuekGi4xz6Z7iv2jemk+yUHt4fD69Vdu8c5wOjh46hllPRcoFMd0WYrd43Lq4unjuUTgiJHzh5qel\\\\\\\/yNelNowljil+Rmi49OGV3T55SzArA\\\\\\\/aGceOxoAcB0i6r816SYKCAUxDLAVk\\\\\\\/YWpA9LQ7eS4JnAJ2D2bG2vaXI\\\\\\\/nRD0ffMfU91Aqnpxw1Ys30Zy+8J0FWMOWPc3vvjnW+cUi4qmU5s0CTi7gq52rSFWTJZ8m0dTAbo2EqL6Cq7bq+6g4bNJ1VvSSXYyy501YJabSPvqLcCz6ZLvquvupRX2HAXxH2ESsb57NdMjmEnC\\\\\\\/\\\\\\\/QUrbybJNFzxUVl4A6uFOdau9gXpKDcq\\\\\\\/cxFljcgKMU4fD9w48tiJnGETIOzkLrG31l6RoPMTPaZC8yug+G2xeXZqXt5j1qKO0UxaXz2776KCvz8ktGC2djUOzhU5TZwFmyuqgohJng59\\\\\\\/qE9\\\\\\\/CmHwBNKe\\\\\\\/ifGdlXZsVghLxh5mU7v6TWEYObAfe\\\\\\\/ouTOMZEui2Z6hc4Trup1Z9MXh3qHmM6h6gRDkz4SQlwZE5eyIfQUSnmPCCG2re84jkF4DxTiDWOkxSsJRNgxm5+bXuTLB+QDSENPdxznNCUGHAZjYg726BO+HAh1csR\\\\\\\/2vcCqGCASOiRdDfLcPeQ4vt6EgDUWmeHIaaD\\\\\\\/3ix8a3J9rAJnGDs5qaXHxBXRDhugD+IbhIcCVZsxIeh3MQzoRNhpLZZY8um0QQURNiELlcyiBzjzT2mlh4twe3Sq4WcgtuMwmbGfa+9U+yMn\\\\\\\/I0PGOmi8TJazI3LgpSxXyYllbPN8FkNyqy8kzx0Xv0ubCI86iAZtAQUqAZ6fTA8l3hm+A6yNwd\\\\\\\/Aqp3sGjCy\\\\\\\/rN5KR9Ebh4yZ40ZSP5+b5TcS3cbt0PcOUaX4LKSq\\\\\\\/S8sDXXk7u0hOKhxodsiGUdeK3dHOhv4DrSAvi4K2fOYA8XjB2oGgk5I\\\\\\\/9AbZrbtb0InjXF5u1KIExZqPHKflJUsLBcFiEXSJ6TEjpY0+\\\\\\\/C5Gny+MDyUvpPhFhZTtvCSMvdL2LZi4yBxpBd6irrj91qr7\\\\\\\/xA0U0QnmBK7Nqoh9HGDEfp\\\\\\\/AeZ3R99jJ05OS7W04\\\\\\\/l798lUT3pJrV0N9BfzdyvYc4DNMJZUJnfPhZp\\\\\\\/UOd4a7eXmQvmlOORBj1WG7+aFp+Bdprb55fAxiOh4vMEss4NtQkqpRFz7xRsPJeOVVFKI9weV480kUl4W8eQLe10a32GJBbPjp4D1Cs6zAKvJonOc4l5W7MnDGbxeEJuzciUs2ReRuC5jAd694clxmDhyZKolu6xzh2Nc+WPZjMs2YkASOznVRkNbX93dF\\\\\\\/je3v8J10UNPv1OtM51XrUH+r+2dXjyjT8QhC6NgkPajIxoiWJhxXfhJ9rVgu2vkiqrKSGnbiScKN0irBQmsPsHGdo9QNvbnaiWOyzw0Apy64lunqqSLSRcayI2eOtEJ7uKsbFoEjeIf\\\\\\\/VL5fDm+Cowwm4wMvpmOiZO+8sWnbiROl3eC3guw4amqdmBP8O6D5dKGpgnPbe+tQrER3SAOZKclaqDSaS8dHWYjrinxgkrmVP+e6odMNIM6qQq08yYWvMnABkMnC13DXTk7j56r9f+OJqOD3U\\\\\\\/araGvCsQzmF1BnxAYEDchgB8xki85W4uE7UGRWalx7MBvdwvqiFDvWrfmzn5AHMaNrK0jVcaOuOniewkFAwpHfXVShwIL3+lpWiamsneCfN0NgXZAAQHq7pro42olpHy94rm\\\\\\\/zWODP2mkUDHExYbWxEwGW2m5oE9YfxAEUgafKCYf8sqQfiCPYQcBIOZAlkYCfHBGPcQTflA4XpFl3ZmaHWhY84Qw6NCjtQEpTSVzhcaLkd4mdWcQ2Bmzsm6EbzMSlZ7SB34xIUD27pe3QGtfa+yldt\\\\\\\/QTQWfiWbCVe3vACPD4hHykNPutVfPbzuiI8OU1YdATRAfEx\\\\\\\/OEnKgBt3hTnpKIp8iyhA27BdT6nYVKXzTh1okJsxMX9GS3h8dbHgV8VTxhl8NYfF1\\\\\\\/Rr6R4LWZTxRlDk6nIrxdFhTgNNmD\\\\\\\/kOos7ABvmxBFT1bKD8M7p29WU8xM+Scwn8wlgKHgGiWftMtcNacZReS86GFyAe74JgBjXV1w+usUbBEsGX\\\\\\\/ArA5ETtEmz\\\\\\\/IfKyPPKqMrwTeOk+sWllEQoP++cQK9cX6b113m8ttzxm6P+f\\\\\\\/9GmH+eiaZ2ZnsIUzLRYb\\\\\\\/7jOVVK9CuYT6BTu5CI1n7I8SNvprD5U6ts1i7cMDP5vBOjseExk3BetRbKDgaOaAgaqGAEEvBFydRPFv2MtZG2baHMIbviBWKCDemAEkAOoeiuUK3emXyKmposxp78p\\\\\\\/XCIzOSydrrWgEX9LGTswal0bkiIzJunryF\\\\\\\/Fj25QFpRIjU05iTnYf5kcR2Yl\\\\\\\/+79MeTu9lChaLL3xbWa4B\\\\\\\/cfHKxGf\\\\\\\/EYInPSBLKTRj5f12BIqcsZnWmIS3+SGWvEkGWMAppTVzcJLcW+A4vqO35zKG5VqwjrZuhOIjgzRGwZfFvc+R2wBav+tX8\\\\\\\/KSDDhRDAzFMjMbpxtnN8lMh67n0JClQV9YZq\\\\\\\/ysG8DnLqF9nAzGsJUwAoiuwJE9nzLHye3gTbDH9puZjRZOSGuoqiqZbYzWsyKzdI9PCHMGwKVYq+rUejSrZJlCcQvbfXpCMuxs1GW2wb36LLq2D85e23SsUdEc\\\\\\\/liGchFp7G8ngoln5qOv+aSqHpwuQCJkXe9A1FPfzpOTJ4nY7BQsAAKQeDIHqp\\\\\\\/WLeO8hfikomU4q3lxvODHcr1E+rU221vTwWjgJh8F3NWs8cPHoWG0wb5yFGm+t+m6tE7gj1pDctu2dxiAONgTmHAg9jHosEU3OrNdqV1R0Rd34k0buI7MeU9zGi8G2ktFy9tHYHtBKO6jTwJYNkrtmHHDVvZdPoo1B5Sj2loUjP9aHz+CaSptWS77M4I8PZkj6hzvaHrc3vzvKraVR08BmQGbmnbaXOAeSu5mm6jcr69jCzz0rHfJYdPPA1XpuKBHzC7FOL3qvYadq3SpemD4iyec5HeWGqbRSipxBt7Xf9YjTN6pUBWv7DjC0ELeZ77hq3CIx\\\\\\\/TQ\\\\\\\/l+bcpV1wbCnLWx49fKY3UIVccVtPgNVsXOTkhqcvskh4f+Nf17ry\\\\\\\/yPDrt+NBsjaCVEUdSY87NG+yjnrdWUSUWJyR3CBg8NYrm3H+9ASTRMuWIXC3mMLuKzvs\\\\\\\/6mZmBpI0m7X26V7UvJ5QxQeuxEzMCFtu0fpoRIWoPJYAJS+errNQYx4dqOtNrT9dgAiJYPB5EPmoPGsWhJkhnulnEuYZnk9fmegNk5FlXCEq7d6fObb3hRsP3tdXZxmKXXsxKxjJpIIE5cAcRK5mLYccytBgMcWtr08pcxHWlFRQ51MHFPLh7jtWAEQe6vM4zAJGueT7clrEqvUw3Xhe\\\\\\\/BMbJikT9zd3IcWP\\\\\\\/osiIBkPIsiKrB6TYdLLOf9rL8OkBDTApKkXSiK+4HhAZcBsAqg6wKpdjs2n\\\\\\\/F4bOUuoj3baNqDzn\\\\\\\/ojPBxOMwwI3o6t2x3cP6uYFDzI8lF12Ip0Qalg9jDwQ11Ai\\\\\\\/EGNNEaB15gwOjF11e9TZVE6tm30aLBPKH2ieUQW+LwIvCYJ\\\\\\\/Je\\\\\\\/r3qZ1Gtpda17JEz9F6ZsgQJKQjLVLuDqTtAxo6YbDeDG\\\\\\\/H8LVJxdzm8jLWQ\\\\\\\/UeWf+aYCEYKB3Tqe2EnMRkgLPo9W+BmL3Z+uplYAr0Tsv+bIUnKuZ9ie2nsisGSmE\\\\\\\/WNp187yYBtOqTGlRlponRQeU8MbpwgNVMYppOvgJ1x3uI8lFBNs9SZvAyglK3uIzxXYG8fo\\\\\\\/apJunH26keeInJXPvbh3M73M6HdqUcJlR2z3rEj6U98XZ+VXdZ5SEZu0AfizgT+Ir4vibFFWaQyWMfuMT7hcx0qUTH23FX+qSjbcTDWvapyGWSIQuvj22lJkmRj4XrEn\\\\\\\/RwJS+1g6YZB1DrR1j8ItGBJmvgwiRK8pePmjkTlmzv21aWQOwAcRQFYLsp4s4JBdJ2PM3ZC+4sv3rN3lmcf\\\\\\\/8ZOvJ8kOZID72xfTlA\\\\\\\/zIX+nPgW5lhF1qrBpRpOI+LE5QOBITgrP34BthuOQmKwSUMmk+fh4kZWgjsoOH2xVHCMbiMMcQhZZ4cwsDQcIMMiqDX2Ga6\\\\\\\/VV4Tos0gBqTBFNIOVVFv2ffQA7ZmMi2Tl1KiI2uEIS7aGjI7Gg+FWm0SsU22uVzWLW5ITXJouWnvlz8NBEW5NtQzHumvlMC4Cajol5dbhx+ylFlSt\\\\\\\/pasGCLFyfhjN+on5WNU8\\\\\\\/jyMhBgUJrJQeVAXMYyFYSJcPgE9PpNRNONTZenFsv7LRPMUl5WpxkH4p5syOuAxEfs4GMg4Lra5faL4UuCBfICbhX0QW396hbAIqzwq0K+5QBfnNkp0QUxyat+7lXYbT17mBw518MIkiNqLIERqlHkvfhFBG9Y6ySrPz5yEf8pxVOY3ahXjwxsMtep4laOokj7GJpg5N7i0rI3aIoS1+NR+LtBXPFAshfAS3zZvO1WXhmm4ERcllRvHaaUQ7SUD98hjSr2X9HIFX7iTz6iwB85LuR8YrOFUwdjt9tZVA6CT2+d8Pr73Dah\\\\\\\/xTL3nFZKqDPlLyNCIS66X9Vyakk6G16aUmzR+VZlgrzgQ4Q4mqT\\\\\\\/\\\\\\\/QQwTZ8EcRyI6fkhlbhyZwXy5UO0DkmM0Nanbn8C6a9HcZMP\\\\\\\/MY1GHwX2JcXGcbLuxL2SgcOiCmWJSWTmNAXsnojGW9Jh2HEMOiGt850JDK6ZOUShSWgU+UNusaXalGixWlLUykOznnnHvKyzvPXa69Rj9nGNaXhue03dsyox4o3Hu75TpXHwMEnATUXOgfxCKKUBt2or9DfYnYV+6+CLPt\\\\\\\/i1Lb+JNCmzdAtveCAQkk3Wzs+XAZdy2m6m8V+OVNKwwH5b3es\\\\\\\/sNJZPngqxyaviDhLee2GLCjxXVcFNy97DgdGGUxqoyRFqC4WyxID9uvi5atS7vGay\\\\\\\/bQtQ\\\\\\\/iWjLGx\\\\\\\/QE4nWWqZgsJVtGSK+RErc8peFyYm84wceNUHeo\\\\\\\/2jxLhN8IbnwFo3\\\\\\\/XJzu4TKuW372e+1TQKDI+wwTRiAEpUQq9m+lOO\\\\\\\/LfzBoNUnRffaysBaRo91hkYbhGvxOncIJIu7YbJ9QpLqGReJLJriTLzhjpC37Jd2CmreFGj3BZz2FjxDQEtpkgaR3\\\\\\\/7ax+j22vOPR\\\\\\\/ubzjALpST0sj6eNQgE3fWEgZa1UcY46HtATkYbn4KjanpeDbdKo2K9qu2XESEVzhUf1YywiEUaYTsmj\\\\\\\/Wd6D3+KsoJWL0ZLZMJe3uVXAjWpKGWDsBZFnte+XukVarHBdUOk6BKPsesubHmWY77K4aoVQUdcTjnZNDv8EFptLQqknZ9aB8LUoJ1\\\\\\\/1xsDUZaHRb\\\\\\\/XTItbOWAj9JdL3TBiLHSW3z3DkRhuCLs7smHAoHkqWhQH2qoM9jEGVVAKZOg37sv0IRdMy0PQWG22Rz0DTwnW4OYwQMITY6IgfBLWvImg\\\\\\\/sW9KGpbSnxtRSmcZan8zKByJW+yvH6+SDCmFY6YfF2HMUgR80EUGjPdKkc8Frli+RfpcpI4xbOx0xf6+yv+nZGek6JAUWNH+VsnrFfa\\\\\\\/Z2OluGw7g+vDL2BkvOa+WFabtQ2vY8Mdcl4F02HxIhq1ZIv0NO8TdqQxfEDs77eysgm6Bkl4SxKSuZqZ1JOPztYHTpmkrOW19K3\\\\\\\/M0yvuEaWebHop652IOM1IIApnjL6hW5mEsVw+SlpxXKK08anmw6FngijCahYIBReCWyG3Uya\\\\\\\/awM7k1OVZaQHW6Z091KzrKNtT3TbjJh9\\\\\\\/gNn0Ppnnlvo3isvhRYeevKHtOfZrx\\\\\\\/yvhI7uNgX7fIvzIlNbHb5AbsvGm0RTSIobMvstVitCY6YIYnKF6CtHnP+hpf6SseDxk43PVCxCGWz05idBIJvJurnkHrH7c3DICLnMooyKQwVCC7BvuqLDng7DVZ2nIT3gshpnsDID6qlZr+9nA1lYAFk1P0k1fKU11lQwE2FfEUzXIaHi8MqK3qNAJRGhaiT\\\\\\\/LC27pNsWIVeF18dYkBQFwL\\\\\\\/NC6XQwyWY5F4Mq7xLJLV4GgunJ0ADZLXh1sUZ4F\\\\\\\/YgTTl2nO2owpAXg\\\\\\\/l2YsawMZ9bUXobagvImMaoyioEABYNc97pgpYQzBxpExMQzDAeKLJkKXIThDNtcBhMjrZkxZ8I0wtGmFayDfAcuWnsnRvmZlmJzHbieGLbCcW2EYz8+sRCU7Fij+BvYVLssuarVmZJoC8yA5\\\\\\\/C6jVFz3ZS0rB+cps8HVvEiLDG4wi8iNxuACnJaTUH0tElxCIygyHGkMg9G+lnsI5OrDP+n13utd0t\\\\\\\/h0N8DToKufU8GBQRg5YuRcMsvYJ7Vi617A6fEWEanedho6UZZAU5s5fEg==\\\u0022,\\\u0022iv\\\u0022:\\\u00222Z2wOpvT\\\\\\\/PbxZIl9KuXhRA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223accc964b921d5e57671680a566e7152e6f7313818db60e4894d926833548dd3\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022c5qTU7rHqIXbIouOk4UtNPyjDWACjBzWc8K1MydHAcc0\\\\\\\/6tPw5Ell0XrvopPXPJYc7vLytu6721r3DK7jV+esTjYiFZgL54oESpbAaqbcZSBaY16BELa95klONznTn2z3n20oCjm41qSCt14fhu2M+NdX4IBUuIu+wkWmxOnvOW0gaAqKteFTuICBUzRdGeK572N2h2mBhucE5C2nCe+vgX8xCDexAOBvyTK3o9+aswifKsIcB6CqbbXrWloUv8o9cr1mTl67uEegA0eZM6XYj0J4zcNJuQBCIg7L87hIVKd3upGt12UH4+O3z6w4Yt9FT672Jtk9u8Yx1jIt1YhmwaiMBkthzjJO6NT\\\\\\\/aCNojjkP0hr7d7Zqle3w3xZZbwUa4M0i+Vf3c3Zn37HUJwEYUHYtADXHdYnUKIq6Li3HyxRszc76oxPxpPZ1SDxwccv5cgUmIfG7eAe62wsdkNN+5ov4arC6PWCDSWcJKLkPib21OiqzkEdp2sUfrFkJ1\\\\\\\/Oo2HbuEZc9oJx9a5eUwabdUmEe0OBvccUfNYbs6P0tw8807h+RfycMmr9A7QwbIKHUGlmkfxrKNQqy5vq7KW+AksJQv5qBHyKIoNffNrr0SRKmklodBbcHuLqkr9FShLYZxW7trN+eFR8nrnbfLyrFfLyLl\\\\\\\/tF2MSoJu281FUmd1VM2CWn4z6bQdMo5xsa5\\\\\\\/EsbTBDSZTu5ZEudyP4jUqw0s007+nsSdHj40Wc2xk7qSH3zjN6yfqEmLcG+B2R97y355tQx+fiY2XOIcTFfwT7ZpZpFoMZV8zPlKizljaHO5CS\\\\\\\/qdCecJkmCmmhg964FpUipemwgFmV4WQD65V5G2uUg2tJDiczI\\\\\\\/jMdKBeEoIgN8Fg9Oy8uYcOOdM59k9zgLKUw2bMs+bIXg7UNKxkiSdeAvnhf9Ob\\\\\\\/dYtKT1hs6A4qfapk5Z3O0cfsnmU\\\\\\\/iueTV+HiWrp7iOn8e4FWmf59SnPZ3FMpwEl1nYKczQfPCn\\\\\\\/pAc1D20OVcIRyzbCEd0bIbpQ1HPlLS6HkHJKJkTCt0czvQ3USfgl95NVL\\\\\\\/\\\\\\\/so+e3MqlV\\\\\\\/mxyZUwD5k86tPtSNkDcDzIj3in6laMuwFN9aTxRwwl4V3x5IQelRXGMpX5jbsk2LEpqexuJmOH36NzPnp8wz4cnj4LJkgzYzQiulTLi6ynGP4py0VeM3UAo+GODn0sTx8GdlyRp7SjvNT5lXOfs7bkda+S8QWzsdFBJm4MQW4F0jnAdUg4Bv\\\\\\\/Jn6t00P6BDvDOcV0yog+Haq3ihIsoqUlkQWnKDipdkGCaa4tPJFQKtxtFWtEwKzSpKfcwE3Ubdt9xRceneRW\\\\\\\/Jb92d2wwf1UkCr9zc2apl1Ij9X9+HDsnRW4bASNlaZ0KgvY\\\\\\\/lkta8goONa5gGQardm3btyJnKfhpLxy44CUNI09Y0Z\\\\\\\/JNSveWE3IPmx8It0IpWXr7nUADlwwOc1p59jf\\\\\\\/najC8DlmeIHnb2Hlcc9HS0iiMoChU6r\\\\\\\/Dp47iOeA5mGLg\\\\\\\/QM7N8MZEc1nKuRM1reOwzs3EnWrtvJXaiOzPEoMjGRGb2I6xRZwqtYvztqUnMYKwtZnUGJ245jcXlpHFaKXVDrv\\\\\\\/PlFkHxwmMFE74FvCa6sl0e5LEvMXNYFjk1wbd1Y079KQ\\\\\\\/ssc0Z0WHwNDf+8NZFi9laTzIyXdmaxnL3zJGpo8yhl2AAxyMHATbFiWiTEJTjd+oWnvm1hyy4flcNmKWv\\\\\\\/B6XtLiay\\\\\\\/s0WcsVAMDOJxUBZcK5y9fB0RYDG9Ek5ddYXFbKJ2voQ5pWH11mDkYAnwJn2IKzXxkvfrbxbpUh+HAUqT3WtapUxt974fnljyjlPwJk067+hlZiHEh93G2by9n9G2jE3SMoP3ypu\\\\\\\/Elwr6Q67bkq3ueV44ZGSU2\\\\\\\/epVdY7Q30xTzz2gHmVXkPGgQ6zJUsu0wBsE991egdozRvcsPjrjuFORhzUvDJWeD1Z0kerD0kNFvUQ7dsv4tZNBQD16kuaxNP2V\\\\\\\/b5rvUJYJt8w==\\\u0022,\\\u0022iv\\\u0022:\\\u00223yP+InXCgbh3lkBoabA\\\\\\\/QQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022e5e546ff9f4e86c8ee4ea43a201420d3009524466c7021c56e32e7bfe4b18332\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459947 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/8/1503" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-zqC-1ur9EQqA5i9zVLGZbutAN0Yax-rqL9oPmc-FCZo" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/8/1503" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-n9Gwicpka9pFI4gjgTw11eZX7sleESm-9CrF9wZQ6FA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/8/1491" title="Linking Genetics to Structural Biology: Complex Heterozygosity Screening with Actin Alanine Scan Alleles Identifies Functionally Related Surfaces on Yeast Actin" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/8/1515" title="Differentiation State-Specific Mitochondrial Dynamic Regulatory Networks Are Revealed by Global Transcriptional Analysis of the Developing Chicken Lens" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459947" id="node459947" data-pisa="ggg;4/8/1503" data-pisa-master="ggg;g3.114.011056" data-apath="/ggg/4/8/1503.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Pattern of Mutation Rates in the Germline of <em>Drosophila melanogaster</em> Males from a Large-Scale Mutation Screening Experiment</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jian-Jun Gao</span>, <span class="highwire-citation-author" data-delta="1">Xue-Rong Pan</span>, <span class="highwire-citation-author" data-delta="2">Jing Hu</span>, <span class="highwire-citation-author" data-delta="3">Li Ma</span>, <span class="highwire-citation-author" data-delta="4">Jian-Min Wu</span>, <span class="highwire-citation-author" data-delta="5">Ye-Lin Shao</span>, <span class="highwire-citation-author" data-delta="6">Shi-Meng Ai</span>, <span class="highwire-citation-author" data-delta="7">Shu-Qun Liu</span>, <span class="highwire-citation-author" data-delta="8">Sara A. Barton</span>, <span class="highwire-citation-author" data-delta="9">Ronny C. Woodruff</span>, <span class="highwire-citation-author" data-delta="10">Ya-Ping Zhang</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="11"><a href="http://orcid.org/0000-0002-1224-4075" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yun-Xin Fu</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1503-1514;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.011056 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459947" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jian-Jun Gao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jian-Jun%2BGao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gao%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJian-Jun%2BGao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Xue-Rong Pan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xue-Rong%2BPan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pan%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXue-Rong%2BPan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jing Hu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jing%2BHu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJing%2BHu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Li Ma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Li%2BMa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ma%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALi%2BMa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jian-Min Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jian-Min%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJian-Min%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Ye-Lin Shao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ye-Lin%2BShao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shao%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYe-Lin%2BShao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Shi-Meng Ai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shi-Meng%2BAi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ai%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShi-Meng%2BAi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Shu-Qun Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shu-Qun%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShu-Qun%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Sara A. Barton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Division of Biostatistics and Human Genetics Center, School of Public Health, The University of Texas at Houston, Houston, Texas</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sara%2BA.%2BBarton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barton%20SA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASara%2BA.%2BBarton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Ronny C. Woodruff </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Biological Sciences, Bowling Green State University, Bowling Green, Ohio</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ronny%2BC.%2BWoodruff%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Woodruff%20RC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARonny%2BC.%2BWoodruff%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Ya-Ping Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China</div><div class='author-affiliation'><span class='nlm-sup'>Â§</span>State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming, Yunnan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ya-Ping%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYa-Ping%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Yun-Xin Fu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Biological Sciences, Bowling Green State University, Bowling Green, Ohio</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yun-Xin%2BFu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYun-Xin%2BFu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1224-4075" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yun-Xin Fu</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:yunxin.fu@uth.tmc.edu" class="" data-icon-position="" data-hide-link-title="0">yunxin.fu@uth.tmc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/8/1503" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459947" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459947/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/8/1503.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459947" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459947/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/8/1503.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459947" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459947/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/8/1503.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459947" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459947/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">The sperm or eggs of sexual organisms go through a series of cell divisions from the fertilized egg; mutations can occur at each division. Mutations in the lineage of cells leading to the sperm or eggs are of particular importance because many such mutations may be shared by somatic tissues and also may be inherited, thus having a lasting consequence. For decades, little has been known about the pattern of the mutation rates along the germline development. Recently it was shown from a small portion of data that resulted from a large-scale mutation screening experiment that the rates of recessive lethal or nearly lethal mutations differ dramatically during the germline development of <em>Drosophila melanogaster</em> males. In this paper the full data set from the experiment and its analysis are reported by taking advantage of a recent methodologic advance. By analyzing the mutation patterns with different levels of recessive lethality, earlier published conclusions based on partial data are found to remain valid. Furthermore, it is found that for most nearly lethal mutations, the mutation rate at the first cell division is even greater than previous thought compared with those at other divisions. There is also some evidence that the mutation rate at the second division decreases rapidly but is still appreciably greater than those for the rest of the cleavage stage. The mutation rate at spermatogenesis is greater than late cleavage and stem-cell stages, but there is no evidence that rates are different among the five cell divisions of the spermatogenesis. We also found that a modestly biased sampling, leading to slightly more primordial germ cells after the eighth division than those reported in the literature, provides the best fit to the data. These findings provide conceptual and numerical basis for exploring the consequences of differential mutation rates during individual development.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/drosophila-melanogaster" class="hw-term hw-article-keyword hw-article-keyword-drosophila-melanogaster" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Drosophila melanogaster</em></a></li><li class="kwd"><a href="/keyword/cell-coalescent" class="hw-term hw-article-keyword hw-article-keyword-cell-coalescent" rel="nofollow">cell coalescent</a></li><li class="kwd"><a href="/keyword/germline-mutation-rate" class="hw-term hw-article-keyword hw-article-keyword-germline-mutation-rate" rel="nofollow">germline mutation rate</a></li><li class="kwd"><a href="/keyword/likelihood-inference" class="hw-term hw-article-keyword hw-article-keyword-likelihood-inference" rel="nofollow">likelihood inference</a></li></ul><p id="p-4">Sperm from any organism results from a series of cell divisions during individual development, and each of these divisions may lead to mutations (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Gilbert 2003</a>). Because mutation is the ultimate cause of genetic variation, numerous studies have focused on understanding various aspects of mutation, including estimations of its rate. The overall mutation rate per generation is an essential quantity for various evolutionary/population genetic studies and is thus the focus of much research. However, a detailed understanding of mutation rates along individual development is also indispensable and an integral knowledge of biology and its importance has been widely recognized in medical genetics, particularly in the study of cancer/tumor development. A mutation that occurred early in the development will likely lead to more descendants (somatic or germ cells) than one that occurred later and thus will likely have more impact on the host as well as on its chance of survival in the population (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Woodruff <em>et al.</em> 1996</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Huai and Woodruff 1997</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Woodruff and Thompson 2005</a>). Differential mutation rates during development also provide new insights on male-driven evolution (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Gao <em>et al.</em> 2011</a>).</p><p id="p-5">Until recently, there has been little progress on dissecting mutation rates during germline development at the level of individual cell divisions. Although next-generation sequencing may hold great promise for providing rich information for such purposes, well-developed classic experiments can still be a powerful and cost-effective approach, particularly for some model organisms. <a id="xref-ref-9-2" class="xref-bibr" href="#ref-9">Gao <em>et al.</em> (2011)</a> reported an analysis of the mutation patterns from a large-scale experiment for screening recessive lethal or nearly lethal mutations and found that mutation rates differ substantially in the germline lineage. In particular, the first cell division harbors the greatest mutation rate, followed by the divisions in spermatogenesis, whereas cell divisions in between have at least a magnitude smaller mutation rate. <a id="xref-ref-9-3" class="xref-bibr" href="#ref-9">Gao <em>et al.</em> (2011)</a> analyzed only those mutations of extreme recessive lethality, which are only a small fraction of the available data, due to a technical difficulty that also limited the analysis to families of exactly 20 offspring with at most two mutations per family. <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Fu (2013)</a> extended the previous inference framework (<a id="xref-ref-9-4" class="xref-bibr" href="#ref-9">Gao <em>et al.</em> 2011</a>) with a new method for approximating the probability of a mutation pattern and a refined coalescent algorithm for simulating sample genealogies which are necessary for deriving coefficients used in the analysis. The refined inference framework not only removes the limitation of at most two mutations per families but also allows families of different sizes. This framework also established the confidence for conclusions derived from likelihood ratios based statistical tests for mutation screening data.</p><p id="p-6">Taking advantage of the aforementioned methodologic progress, we report in this paper the analysis of the complete data set from the experiment, which consists of 9872 families of various sizes and which contains three times more mutations than previously reported. The greater resolution of the data as well as the contrast of results from analyzing mutations with different recessive lethalities allows us to obtain more accurate/stable estimates of mutation rates, to explore some aspects of the mutation process, and to test hypotheses that were unattainable previously. As a result, a deeper understanding of the mutation rate patterns along germline development is achieved. Furthermore, new hypotheses are presented and the male-driven hypothesis is re-evaluated in light of the new results from the current analysis.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Materials</h3><p id="p-7"><em>Drosophila melanogaster</em> stocks from Woodruffâs laboratory in Bowling Green State University were used. These flies were maintained by taking advantage of the balancer chromosomes that were pioneered by H. J. Muller (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Muller 1928</a>) for the purpose of maintaining newly isolated mutations, including recessive lethals, without selection (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Muller and Oster 1963</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abrahamson and Lewis 1971</a>; <a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Ashburner 1989</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Greenspan 1997</a>). Balancers for each of the major chromosomes of these <em>D. melanogaster</em> contain multiple inversions and one or more dominant visible mutations. The inversions, which were mapped by using giant polytene chromosomes, act as crossover suppressors and the clearly visible dominant mutations allows for the identification of heterozygotes. With these chromosome stocks, new lethal or nearly lethal mutations are maintained in the heterozygous state against the balancer chromosomes without the concern of being lost due to recombination. The experiment employed three types of autosomal haploid chromosomes (genomes), which are denoted by <em>Î²</em>, <em>Î³</em>, and z. More specifically they are as follows:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-1.gif"/></noscript></span></span>The <em>Î²</em> type balancer is homozygous lethal and is marked with the dominant visible, and recessive lethal mutations, Curly (<em><a href="http://flybase.org/reports/FBgn0000403.html">Cy</a></em>) wings, Lobe (<em><a href="http://flybase.org/reports/FBgn0001332.html">L</a></em>) eye, and Ultrabithorax (<em><a href="http://flybase.org/reports/FBgn0003944.html">Ubx</a></em>) enlarged halteres. This balancer segregates as a unit and suppresses crossing over on both the second and third chromosomes effectively (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Lindsley and Zimm 1992</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Thompson 1977</a>). Similarly, the <em>Î³</em> chromosome is also homozygous lethal and carries dominant visible markers. Type z represents a haploid genome with wild-type second and third chromosomes that are free of lethal mutations at the start of experiment. Recessive lethal or nearly lethal mutations in z are the screening target of the experiment.</p></div><div id="sec-3" class="subsection"><h3>Experiment</h3><p id="p-8"><a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Graf <em>et al.</em>âs (1992)</a> methods for culturing the flies were followed with some modifications. Drosophila medium containing water, glucose, agar, corn meal, and the antifungal agent methyl-p-hydroxybenzoate was cooked and dispensed into glass culture vials (10 cm in height and 3.5 cm in diameter) via a self-made dispenser. Also, self-made vial holders (designed to hold vials in a 10 Ã 10 array to match the dispenser) were used to facilitate the work. After drying and cooling the medium, a small piece of sterilized filter paper was folded and inserted into the medium with forceps to increase the surface and to regulate humidity within the vial, and then a small amount of live bakerâs yeast was seeded into the vial. The culture vials, with cotton plugs added, were used to start the cultures. The flies were reared in a chamber, which allowed simultaneous culturing of more than 20,000 culture vials of flies under standard conditions (25Â°, approximately 60% relative humidity and 16-hr light:8-hr dark). The temperature in the culturing chamber was adjusted with air-conditioners and a self-regulating electric heating system, and humidity was manually controlled with a humidifier.</p><p id="p-9">As briefly described in <a id="xref-ref-9-5" class="xref-bibr" href="#ref-9">Gao <em>et al.</em> (2011)</a>, the mutation screening experiment, which is similar to protocols that have been used in various laboratories (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Thompson and Woodruff 1980</a>; <a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Woodruff <em>et al.</em> 1984</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Mason <em>et al.</em> 1985</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Brodberg <em>et al.</em> 1987</a>; <a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Woodruff <em>et al.</em> 1996</a>), consists of two parts. The first is to employ a three-generation assay to identify autosomal-recessive lethal or nearly lethal mutations in approximately 1200 genes on the second and third chromosomes in <em>D. melanogaster</em>. The second component of the experiment is known as the <em>allelism test</em>, which is to delineate the identity of mutations leading to different mutants.</p><p id="p-10">The first component of the experiment was designed to screen <em>Î²</em>/z male offspring of crosses between single <em>Î²</em>/z males and multiple <em>Î²</em>/<em>Î³</em> virgin females to see whether a new lethal or nearly lethal mutation occurred in the z chromosomes during germline development of the father. In essence, the mating scheme within each family derived from a single <em>Î²</em>/z male is as follows:</p><span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-2.gif"/></noscript></span></span><p id="p-11">The intention here was to follow 20 offspring (lines) per family, but because some lines would not succeed for a variety of reasons, including death and failing quality control, up to 35 lines were initiated per family. As a result, the number of offspring in a family ranged from 2 to 35. Parental flies were removed from culture vials well before we examined the progeny and collected virgin females. The progeny adult flies were etherized using the Drosophila Fly Anesthetizer (Burco) and examined for phenotype and sex under a stereo microscope. Virgin <em>Î²</em>/<em>Î³</em> or <em>Î²</em>/z females were collected within about 8 hr after we removed parental flies and then collection was repeated every 8 hr (usually at about 8:00 <span class="sc">am</span>, 4:00 <span class="sc">pm</span>, and midnight) during the eclosion.</p><p id="p-12">Care was taken to ensure that <em>Î²</em>/<em>Î³</em> females were virgin during the experiment. As a quality control measure, if <em>Î²</em>/<em>Î³</em> offspring in the <em>F</em><sub>3</sub> were observed, the line was deemed disqualified because such an event can only result from nonvirgin <em>Î²</em>/z females. Also, to make sure the degree <em>d</em> of recessive lethality or simply <em>lethality</em> (<a id="xref-ref-7-2" class="xref-bibr" href="#ref-7">Fu 2013</a>), which is one minus the percent of z/z homozygote among all offspring, is adequately estimated, 40 offspring were set to be the minimum number of offspring examined in <em>F</em><sub>3</sub>. A line is declared to be tentatively a mutant with lethality <em>d</em> if the percentage of its z/z homozygotes among all surviving offspring is not larger than <em>d</em>.</p><p id="p-13">The <em>Î²</em>/z males used in the Parental stage were selected from families of <em>F</em><sub>3</sub> in which no mutation of detectable lethality was found (<em>i.e.</em>, the percentage of z/z offspring is normal). In addition, the selection favored vigorous young males with distinct phenotypes. The process ensured that the chromosome z in the <em>Î²</em>/z male was devoid of recessive lethal mutation in general, but newly arisen recessive lethal mutation in F3 may escape such surveillance. The counting of <em>F</em><sub>3</sub> offspring was performed when there were a sufficient number of matured offspring. By the time a line was judged to be devoid of the targeted mutations, it would be typically 2â4 d after the minimum age for a matured adult.</p><p id="p-14">The second component of the experiment is the <em>allelism test</em>, which is to delineate the identity of the mutation leading to each mutant. This was achieved by a series of crosses between mutant lines, with one line contributing <em>Î²</em>/z males and another virgin <em>Î²</em>/z females. There are usually many ways the crosses can be arranged, but each mutant line must be involved in at least one cross. The percentage of z/z individuals among offspring of a cross is expected to be similar to those of the parental lines if the mutations in the two different lines are the same and significantly higher when they are different (if mutations are indeed recessive). When there are multiple mutant lines in a family, a series of interconnected crosses between different mutant lines are necessary to resolve ambiguities.</p></div><div id="sec-4" class="subsection"><h3>Statistical method</h3><p id="p-15">Using the same notation as <a id="xref-ref-7-3" class="xref-bibr" href="#ref-7">Fu (2013)</a>, the information conveyed by mutation(s) in a family can be represented by a mutation pattern<span class="disp-formula" id="disp-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-3.gif"/></noscript></span> <span class="disp-formula-label">(1)</span></span>in which each element represents a mutation and its value is the number of offspring carrying the mutation, or simply the <em>size of the mutation</em>. After a line obtains a mutation of lethality â¥ <em>d</em>%, a further mutation will likely do nothing or increase the level of lethality, but the effect is usually difficult to distinguish from the first one under our experimental setting. This often led to masking or nondetection of the second mutation (<a id="xref-ref-7-4" class="xref-bibr" href="#ref-7">Fu 2013</a>). As a result, each identified mutant offspring is associated with one and only one mutation in the mutation pattern. Therefore, there is a natural constraint that <em>i</em> + <em>j</em> + <em>k</em> â¦ is not larger than the family size. The aggregation of mutation patterns for families of size <em>f</em> can be concisely represented as<span class="disp-formula" id="disp-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-4.gif"/></noscript></span> <span class="disp-formula-label">(2)</span></span>where <em>n</em>(<em>Îº<sub>i</sub></em>) is the number of occurrences of pattern <em>Îº<sub>i</sub></em> and for brevity can be omitted if its value is 1. The statistical analysis of the experimental results consists of determining the mutation pattern in each family, estimating mutation rates, and testing hypotheses.</p><div id="sec-5" class="subsection"><h4>Determination of the mutation pattern in a family:</h4><p id="p-16">The log-likelihood ratio test is used to test the null hypothesis that two mutant lines share the same causal mutation against the alternative that they result from independent mutations. Under the null hypothesis, the likelihood is the product of three binomial distributions all having the same frequency for z/z offspring, while under the alternative the binomial distribution for the cross has different frequencies for z/z offspring. Suppose <em>z<sub>i</sub></em> and <em>n<sub>i</sub></em> are the numbers of z/z and total offspring for parental line <em>i</em>(<em>i</em> = 1, 2) respectively, and <em>z</em><sub>3</sub> and <em>n</em><sub>3</sub> are the corresponding numbers for the cross. Then the test statistic is<span class="disp-formula" id="disp-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-5.gif"/></noscript></span> <span class="disp-formula-label">(3)</span></span>where <em>p</em><sub>123</sub> = (<em>z</em><sub>1</sub> + <em>z</em><sub>2</sub> + <em>z</em><sub>3</sub>)/(<em>n</em><sub>1</sub> + <em>n</em><sub>2</sub> + <em>n</em><sub>3</sub>), <em>p</em><sub>12</sub> = (<em>z</em><sub>1</sub> + <em>z</em><sub>2</sub>)/(<em>n</em><sub>1</sub> + <em>n</em><sub>2</sub>) and <em>p</em><sub>3</sub> = <em>z</em><sub>3</sub>/<em>n</em><sub>3</sub>. The test statistic follows asymptotically a Ï<sup>2</sup> distribution with one degree of freedom under the null hypothesis that the cross results in the same z/z percentage as the two parental lines. Therefore, a significant test result indicates that the two lines result from different mutations. Often, multiple crosses are performed among lines within a family. If the overall significance level is <em>Î±</em>, and there are <em>m</em> crosses, then the level of significance for each test should be set to about <em>Î±</em>/<em>m</em>. In general, larger values of <em>Î±</em> will result in more significant tests and thus more independent mutations will be inferred, but the number of mutants for a given lethality does not change significantly, which are mostly determined by the percentage of z/z offspring in the <em>F</em><sub>3</sub>. The results presented in this paper corresponds to <em>Î±</em> = 0.10.</p></div><div id="sec-6" class="subsection"><h4>Estimating mutation rate and hypothesis testing:</h4><p id="p-17">Suppose the development from the fertilized egg to the sperms are divided into <em>I</em> consecutive time intervals [<em>t<sub>i</sub></em> <sub>â 1</sub> + 1, <em>t<sub>i</sub></em>] (i = 1,â¦I) (<em>t</em><sub>0</sub> = 0 and <em>t<sub>I</sub></em> = maximum number of cell divisions), and let <em>u<sub>i</sub></em> be the mutation rate per cell division within the ith interval. The estimate of <strong><em>Î¼</em></strong> = (<em>u</em><sub>1</sub>, â¦ <em>u<sub>I</sub></em>) and statistical tests about them were performed through the maximum likelihood approach developed in <a id="xref-ref-9-6" class="xref-bibr" href="#ref-9">Gao <em>et al.</em> (2011)</a> and further refined in <a id="xref-ref-7-5" class="xref-bibr" href="#ref-7">Fu (2013)</a>. The inference makes use of the information about population dynamics, intervals of cell divisions, and coalescent structure of the sample genealogy (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>), but in essence, the inference framework is based on evaluating the likelihood function<span class="disp-formula" id="disp-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-6.gif"/></noscript></span> <span class="disp-formula-label">(4)</span></span>where <em>f</em> represents family size, <em>Îº</em> is a mutation pattern for a given family size <em>f</em>, <em>n<sub>f</sub></em>(<em>Îº</em>) is the number of occurrences of pattern <em>Îº</em> in families of size <em>f</em>, and <em>p<sub>f</sub></em>(<em>Îº</em>) is the probability of mutation pattern <em>Îº</em>, which is a function of <em>Î¼</em> and coefficients that are derived from the dynamics of the germline population. The probability can be computed through a combined approach of both analytic derivation and coalescent simulation of samples. The first product enumerates over all the different family sizes, while for each family size the second product enumerates over all observed patterns. The maximum likelihood estimate <span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-7.gif"/></noscript></span></span> is the value of <em>Î¼</em>that maximizes the above likelihood function.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/8/1503/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Population dynamics and an example of the genealogy of four sperm sampled at the time at which maximal 38th cell division has occurred (adapted from Fu 2013)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-793805354" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Population dynamics and an example of the genealogy of four sperm sampled at the time at which maximal 38th cell division has occurred (adapted from Fu 2013).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/8/1503/F1.medium.gif" width="354" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/8/1503/F1.medium.gif" width="354" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/8/1503/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/8/1503/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464754" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-18" class="first-child">Population dynamics and an example of the genealogy of four sperm sampled at the time at which maximal 38th cell division has occurred (adapted from <a id="xref-ref-7-6" class="xref-bibr" href="#ref-7">Fu 2013</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">The overall mutation rate <em>Î¼</em> is defined as the sum of mutation rates over all divisions in the development. Thus the maximum likelihood estimate of <em>Î¼</em> is<span class="disp-formula" id="disp-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-8.gif"/></noscript></span> <span class="disp-formula-label">(5)</span></span>Alternatively, an unbiased estimate of <em>Î¼</em> can be obtained by a classical approach (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Fu and Huai 2003</a>), which is<span class="disp-formula" id="disp-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-9.gif"/></noscript></span> <span class="disp-formula-label">(6)</span></span>This estimate has the advantage of being unbiased regardless of the assumptions made on the dynamics of the germline lineage development.</p></div></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Data and summary</h3><p id="p-20">More than 10,000 families were screened in a 4-yr period (2004â2008), from which 9872 succeeded with at least one survival line at <em>F</em><sub>3</sub>. To obtain positional information about observed mutation(s), we further require at least two completed lines for a family. To make sure <em>Î²</em>/<em>Î³</em> females in <em>F</em><sub>2</sub> are virgin, we further require that no <em>Î²</em>/<em>Î³</em> offspring among the <em>F</em><sub>3</sub> was observed. Furthermore, when the recessive lethality of a line exceeds a given threshold and the identity of the mutation cannot be determined, the line was also removed. This occurs when such a line was not used in crosses with other mutant lines (if they exist). After the cleanup, 9594 families passed the quality control, which resulted in a total of 271,794 lines, which is 90% of the 300,737 examined lines. The mean number of lines per families is 28.3 and the mean number of offspring in the qualified lines in <em>F</em><sub>3</sub> is 117.9. The distribution of the family sizes is given in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/8/1503/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of family sizes in 9594 screened families." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-793805354" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Distribution of family sizes in 9594 screened families.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/8/1503/F2.medium.gif" width="440" height="259"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/8/1503/F2.medium.gif" width="440" height="259"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/8/1503/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/8/1503/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464713" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-21" class="first-child">Distribution of family sizes in 9594 screened families.</p><div class="sb-div caption-clear"></div></div></div><p id="p-22">The histogram of the percentage of z/z offspring in the 271,794 lines is given in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a>. There are two obvious modes in the histogram, one at 1% and another at 18%. The first one corresponds to those with relatively high lethality mutations, and the latter corresponds to the normal z/z homozygotes. Note that because <em>Î²</em>/<em>Î²</em> is lethal, only <em>Î²</em>/z and z/z genotypes can potentially survive. If the genotypes are of the same fitness, their proportions will be 2/3 and 1/3, respectively. <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a> shows that z/z appears to be less fit than the <em>Î²</em>/z genotype, resulting in an average of approximately 17.5% only. One possible reason may be that the normal z chromosome used in the initial experiment contained some mildly deleterious recessive mutations. The reduced fitness of the z/z homozygotes does not affect significantly the identification of recessive mutations of high lethality but makes the dissection of those of lower lethality more challenging.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/8/1503/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Distribution of the percentage of z/z offspring among 271,794 lines in F3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-793805354" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Distribution of the percentage of z/z offspring among 271,794 lines in &lt;em&gt;F&lt;/em&gt;&lt;sub&gt;3&lt;/sub&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/8/1503/F3.medium.gif" width="440" height="283"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/8/1503/F3.medium.gif" width="440" height="283"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/8/1503/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/8/1503/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464719" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-23" class="first-child">Distribution of the percentage of z/z offspring among 271,794 lines in <em>F</em><sub>3</sub>.</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">The identification of mutations starts with lines having a low percentage of z/z offspring in the <em>F</em><sub>3</sub> and proceeds with cross experiments and inference. One example is presented to illustrate the process. <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a> shows an example of the <em>F</em><sub>3</sub> results of a family early in the experiment. The family starts with 30 lines but after initial quality control (one with <em>Î²</em>/<em>Î³</em>offspring and two with too few total offspring and a few lines did not survive), a total of 17 lines are recorded.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464809/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464809/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">An example of F3 data for family 140</span><div class="sb-div caption-clear"></div></div></div><p id="p-25">It is clear that all lines, except for line 5, are suspects of new mutations of relatively high lethality. For a line to be declared as a mutant line for lethality level <em>d</em>, at minimum it should have a percentage of z/z offspring not larger than 1 â <em>d</em> and it is crossed with at least one other line if available. We were not sure how low the <em>d</em> can be so it was decided to perform as many crosses as feasible. As a result, all the lines except 5 and 2 are used for cross experiments. The exclusion of line 2 was not intentional but was due to the late completion of <em>F</em><sub>3</sub> for that line.</p><p id="p-26">There are multiple ways to carry out allelism tests but the principle is that ambiguity needs to be resolved. <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> shows the diagram of crosses used and their results. It is clear that there is no evidence that the six lines, represented by white circles, resulted from different mutations because the percentages of z/z offspring from crosses among them are similar to their parental lines. The crosses between lines 1 and 3, lines 19 and 30, and lines 12 and 24, resulted in significant test results, but after adjusting multiple tests as described in the <em>Statistical Method</em> section, only the cross between 1 and 3 is significant at the 10% level. Therefore, two mutations are identified, the first one includes line 3 with the z/z percentage being 0, and the second one includes the other 13 lines tested with an overall z/z percentage of 0.0171.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/8/1503/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Cross and results for family 140. a/b (e.g. 0/101) beside a line indicates the numbers of z/z and total offspring of the cross, respectively. Three crosses, between lines 19 and 30, between 12 and 24, and between lines 1 and 3 are individually significant (with * and *** representing, respectively, significance at 5 and 1% level). After adjusting for multiple tests, all three crosses are significant at the 50% significance level but only the cross between lines 1 and 3 remains significant at the 10% significance level." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-793805354" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Cross and results for family 140. &lt;em&gt;a&lt;/em&gt;/&lt;em&gt;b&lt;/em&gt; (e.g. 0/101) beside a line indicates the numbers of z/z and total offspring of the cross, respectively. Three crosses, between lines 19 and 30, between 12 and 24, and between lines 1 and 3 are individually significant (with * and *** representing, respectively, significance at 5 and 1% level). After adjusting for multiple tests, all three crosses are significant at the 50% significance level but only the cross between lines 1 and 3 remains significant at the 10% significance level.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/8/1503/F4.medium.gif" width="440" height="284"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/8/1503/F4.medium.gif" width="440" height="284"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/8/1503/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/8/1503/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464641" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-27" class="first-child">Cross and results for family 140. <em>a</em>/<em>b</em> (e.g. 0/101) beside a line indicates the numbers of z/z and total offspring of the cross, respectively. Three crosses, between lines 19 and 30, between 12 and 24, and between lines 1 and 3 are individually significant (with * and *** representing, respectively, significance at 5 and 1% level). After adjusting for multiple tests, all three crosses are significant at the 50% significance level but only the cross between lines 1 and 3 remains significant at the 10% significance level.</p><div class="sb-div caption-clear"></div></div></div><p id="p-28">The number of identifiable mutations of relatively high lethality depends on the extensiveness of lines used in the allelism test. Although we were interested in determining as many mutations as feasible, it becomes too laborious when the number of lines to be crossed is large. Therefore, a compromise had to be made, which was achieved progressively. <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a> shows the mean minimum z/z percentage for lines in a family that are not subjected to the allelism crossing experiment. It can be seen that after the initial 3000 families, the minimum was raised from approximately 8% to approximately 12% for a short period of time. The practice was deemed impractical, and the minimum then dropped back to approximately 7%. The overall average is 8.2%.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/8/1503/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Mean minimum percentage of z/z offspring among nontested lines in windows of 200 families." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-793805354" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Mean minimum percentage of z/z offspring among nontested lines in windows of 200 families.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/8/1503/F5.medium.gif" width="440" height="276"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/8/1503/F5.medium.gif" width="440" height="276"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/8/1503/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/8/1503/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464806" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-29" class="first-child">Mean minimum percentage of z/z offspring among nontested lines in windows of 200 families.</p><div class="sb-div caption-clear"></div></div></div><p id="p-30">Despite the fact that lines with a recessive lethality of more than 92% were generally subjected to the allelism test, there is an increasing level of difficulties and ambiguity in determining the identity of mutations with decreasing lethality. As a result, we are cautious about those mutations identified with a recessive lethality lower than 95%. To be conservative, yet allowing for sufficient contrast, we will focus on those with lethality of 97% or greater. The mutation patterns identified through the allelism test depends on how stringent the criteria is, which is controlled by the value of <em>Î±</em> as described in the <em>Materials and Methods</em> section. Because <em>Î±</em> is the overall error rate, the larger it is, there will be more independent mutations declared. <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a> provides the summary of mutations identified under different lethality intervals under two values of <em>Î±</em>. Both the number of mutations and the number of mutants are greatest in the lethality interval [97%, 98%), followed by the lethality interval [98%, 99%). Overall, the pattern appears to agree with that of the z/z percentages shown in <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>, where the second percentile is the most frequent mutant type and is followed by the third. As predicted previously, the number of mutations under <em>Î±</em> = 0.5 is larger in most cases, particularly for greater lethalities. However, the overall mutation rate does not change significantly in any of the cases. It turns out that subsequent analysis of mutations from different <em>Î±</em> values also leads to a relatively small difference. Therefore, we will focus on the presentation of analysis resulting from the mutations identified with <em>Î±</em> = 0.10.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464748/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464748/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Distributions of 9594 families by mutation count under different lethality intervals</span><div class="sb-div caption-clear"></div></div></div><p id="p-32">The data analyzed by <a id="xref-ref-9-7" class="xref-bibr" href="#ref-9">Gao <em>et al.</em> (2011)</a> is a subset of those with lethality [99%,100%] (or â¥99%). However, a 5% significance level without adjustment for multiple tests was used, so their data are not strictly a subset of column 1 of <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a> from either <em>Î±</em> = 0.10 or <em>Î±</em> = 0.50. Because the number of crosses in a family increases with the number of likely mutants, the major effect of adjusting for multiple tests is that it makes the declaration of different mutations easier for families with a smaller number of likely mutants and harder for families with many likely mutants. Comparatively, because the mean number of crosses for families with a cross is approximately 17 (data not shown), this translates into about a 3% significance level on average per family, even <em>Î±</em> = 0.5 still appears slightly more stringent than the criteria used by <a id="xref-ref-9-8" class="xref-bibr" href="#ref-9">Gao <em>et al.</em> (2011)</a>. It is reassuring that the subsequent inference is rather robust and as a result no qualitative conclusions made previously need to be revoked, which will be seen from the inference in a later section.</p><p id="p-33">It is also useful to consider mutations for given minimum lethality levels. <a id="xref-table-wrap-3-1" class="xref-table" href="#T3">Table 3</a> provides the distribution under four minimum lethalities (notice that lethality â¥99% is the same as [99%, 100%]). It should be pointed out that mutation patterns for a given minimum lethality, say 97%, cannot be obtained as a simple aggregation of those in the lethality intervals [99â100%], [98â99%), and [97â98%). This occurs because sometimes mutations of lethalities falling into different intervals may occur in the same family, which can be seen from <a id="xref-table-wrap-3-2" class="xref-table" href="#T3">Table 3</a> in which there are families with four mutations, whereas there are none when considering lethality intervals separately.</p><div id="T3" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464731/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464731/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 3</span> <span class="caption-title">Distributions of 9594 families by mutation count under different minimum lethalities</span><div class="sb-div caption-clear"></div></div></div><p id="p-35">There are 111 families of size 20 (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>) and under the lethality level [99%,100%] the collection of mutation patterns is<span class="disp-formula" id="disp-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-15.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-15.gif"/></noscript></span></span>which indicates that 11 have one mutation of pattern &lt;1&gt;, one each for patterns of &lt;2&gt;, &lt;3&gt;, and &lt;17&gt;, respectively and two have a mutation of pattern &lt;1, 1&gt;. The number of mutant families is 11 + 1 + 1 + 1 + 2 = 16 and, thus, the number of nonmutant families is 111 â 16 = 95. Similarly, the mutation patterns for a family of size 20 under lethality level [98%, 99%) and [97%,98%) are, respectively,<span class="disp-formula" id="disp-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-16.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-16.gif"/></noscript></span></span>and<span class="disp-formula" id="disp-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-17.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-17.gif"/></noscript></span></span>The complete mutation patterns of the data corresponding to different lethality levels are given Table A1, Table A2, and Table A3 in the Appendix. For the sake of space, mutation patterns for given minimum lethalities will not be listed but can be obtained from the authors.</p></div><div id="sec-9" class="subsection"><h3>Inference about mutation rates</h3><p id="p-36">Although it is desirable to make an inference about the mutation rate for every cell division along germline development, the lack of sufficient resolution in the experimental data and the prohibiting computational burden limit our inference to six or fewer intervals. As described in the <em>Statistical Method</em> section, the intervals can be represented by a series of integers defining the boundary locations. For example, the sequence 1, 2, 14, and 31 means that there are five intervals: [1, 1], [2, 2], [3, 14], [15, 31], and [32, <em>n</em>] where <em>n</em> is the last cell division (<em>e.g.</em>, 36). However, because different sperm may experience different numbers of divisions (but at least 36), while regardless of the number, the last five are spermatogenesis, it is more logical to put the last five divisions into the last interval and divisions from the 15th to just prior to spermatogenesis as the fourth. This interval definitions is conveniently represented by 1, 2, 14, â5.</p><p id="p-37">The evaluation of the likelihood function makes use of coefficients computed from simulated samples from the germline population, which is dependent on the assumptions about the population dynamics. Let <em>N</em>(<em>i</em>) be the population size of germline lineage after the <em>i</em>th cell division. The assumptions about <em>N</em>(<em>i</em>) used in <a id="xref-ref-9-9" class="xref-bibr" href="#ref-9">Gao <em>et al.</em> (2011)</a> are provided in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> in <a id="xref-ref-7-7" class="xref-bibr" href="#ref-7">Fu (2013)</a> and one assumption that has significant impact on the analysis is about <em>N</em>(8). Previous knowledge (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Drost and Lee 1995</a>, <a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">1998</a>) suggests that after the eighth division, about four to six cells move to the posterior region and become the primordial germ cells (PGCs). For a relatively small number of PGCs, are the PGCs a random sample out of the 256 cells available after the eighth division, or are they more closely related being descendants of an ancestral cell a few divisions earlier, or something in between? We will refer to this factor as sampling bias leading to N(8). This issue can be investigated effectively by examining the value of <em>N</em>(5), because after three divisions a cell in N(5) will have eight descendant cells in N(8), making it possible that all the PGCs are derived from a single cell in N(5). On the other hand, if N(5) is set to 32, it is equivalent to assuming that N(8) is a random sample out of the 356 cells.</p><p id="p-38">The maximum number <em>D</em> of cell divisions is one factor that impacts the likelihood analysis. Because 36 cell divisions is the minimum and young mature males were used in the experiment, D was set to 36 as in <a id="xref-ref-9-10" class="xref-bibr" href="#ref-9">Gao <em>et al.</em> (2011)</a>, although large values also were examined to some extent. As pointed out in the <em>Experiment</em> section, the likely range of cell divisions for sperm leading to <em>F</em><sub>1</sub> offspring is between 36 to about 40. Hence, we conducted likelihood analysis with <em>D</em> from 36 to 42, and found that <em>D</em> = 38 yields the overall best fit, which is significantly better than the case with <em>D</em> = 36, but goodness of fit declines gradually after <em>D</em> &gt; 40. Even so, the pattern of likelihood estimates differ only marginally for D in the range between 36 and 42 and the impact on the interpretation of inference is relatively low. Because we are reasonably confident that the majority of sperm used in the experiments are from young males, we shall report results throughout the paper with <em>D</em> = 38.</p></div><div id="sec-10" class="subsection"><h3>Assumptions on the dynamics of the germline population</h3><p id="p-39">Although we are primarily interested in the distribution of mutation rates during germline development, it is important to evaluate the impact of the assumptions about the dynamics of the population on the inference. One such assumption concerns how the four to six cells are selected after the eighth cell division. The primary analysis in <a id="xref-ref-9-11" class="xref-bibr" href="#ref-9">Gao <em>et al.</em> (2011)</a> assumed that these cells are randomly drawn from 256 cells which resulted from the eighth cell division. Although this does seem consistent with the Drosophila embryonic development literature (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Sonnenblick 1965</a>), the impact of biased sampling were evaluated to some extent. We carried out more extensive analysis using the full data set.</p><p id="p-40">We first investigated the impact of the values of <em>N</em>(5) and <em>N</em>(8) on the value of the likelihood function. To make the computation manageable, we divided the range (1â32) of N(5) into groups each containing two consecutive integers, and similarly the range (1â256) of <em>N</em>(8) into groups each containing three consecutive integers. Because the dynamics of the germline population are properties of the germline and thus should be more or less independent of types of mutations, one uses as much data as possible for judging its goodness of fit. Therefore, we examine the effect of gradually adding more data into the analysis on the strength of conclusions. <a id="xref-table-wrap-4-1" class="xref-table" href="#T4">Table 4</a> shows the log-likelihood values for a number of combinations of <em>N</em>(5) and <em>N</em>(8) for data under three lethality levels.</p><div id="T4" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464716/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464716/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 4</span> <span class="caption-title">Differential decrease to the maximum likelihood with intervals [1, 1], [2, 2], [3, 14], [15 â 6], [â5, 38]</span><div class="sb-div caption-clear"></div></div></div><p id="p-41"><a id="xref-table-wrap-4-2" class="xref-table" href="#T4">Table 4</a> shows that regardless of recessive lethality the maximum likelihood is achieved at the combination of <em>N</em>(5) = 5 ~ 6 and <em>N</em>(8) = 7 ~ 9. Because twice the difference is the value of the log-likelihood ratio test (with one degree of freedom when one of the <em>N</em>(5) and <em>N</em>(8) is fixed), it follows that for 98% and 97% recessive lethality, any other combination of <em>N</em>(5) and <em>N</em>(8) can be rejected at the 1% significance level. For 99% recessive lethality, the resolution is less, but all other combinations except <em>N</em>(5) = 5 ~ 6 and <em>N</em>(8) = 10 ~ 12 can be rejected at the 5% level (including the random sampling). These results imply that <em>N</em>(5) is about 5â6, which represents a sampling for the eighth population that is far more restrictive than random sampling, which corresponds to <em>N</em>(5) = 32. Note that <a id="xref-ref-9-12" class="xref-bibr" href="#ref-9">Gao <em>et al.</em> (2011)</a> categorically defined <em>N</em>(5) = 4 as mild sampling bias largely due to mistaking the value as the number of ancestral sequences after the fifth cell division. Given the amount of reduction from 32 to 5â6, its classification as modest bias (at least) appears to be in order. These results also suggest that <em>N</em>(8) = 4 â 6, based on various experimental observations, appears to be conservative. Hence, a slightly larger number for <em>N</em>(8) may be more the norm.</p></div><div id="sec-11" class="subsection"><h3>Estimates of mutation rates</h3><p id="p-42">In the previous section, we established that <em>N</em>(5) = 5 ~ 6 and <em>N</em>(8) = 7 ~ 9 is the best assumption for the germline population dynamics; therefore, we will use this assumption for subsequent analysis. Because we investigate the mutation rates for different lethality levels, it is desirable to know whether a slight deviation of the optimal parameters will lead to a significant difference in the subsequent mutation rate estimation and hypothesis testing. We performed full likelihood analysis for several lethalities around <em>N</em>(5) = 5 â 6 and <em>N</em>(8) = 7 â 9 and found that indeed the impact is rather marginal unless the deviation leads to a substantially smaller likelihood value. <a id="xref-table-wrap-5-1" class="xref-table" href="#T5">Table 5</a> lists, as examples, the maximum likelihood estimates of mutation rates for combinations of <em>N</em>(5) and <em>N</em>(8) that differs no more than one step from the optimal. As can be seen, overall the estimates are rather similar to those under the optimal, but there are some notable differences in <em>u</em><sub>2</sub> for cases two steps away from the optimal; but then the difference of likelihood to that of the optimal is substantial. Therefore, we are confident that conclusions based on the optimal <em>N</em>(5) and <em>N</em>(8) will be robust.</p><div id="T5" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464775/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464775/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 5</span> <span class="caption-title">Full maximum likelihood estimates of <em>u</em> Ã 10<sup>3</sup> for lethality â¥ 97% and <em>N</em>(5) = 5 ~ 6 and <em>N</em>(8) = 7 ~ 9 with intervals [1,1], [2,2],[3,3], [4 14], [15 -6], and [-5 38]</span><div class="sb-div caption-clear"></div></div></div><p id="p-43"><a id="xref-table-wrap-6-1" class="xref-table" href="#T6">Table 6</a> gives the estimates of <strong><em>u</em></strong> for three recessive lethalities for 38 cell divisions and the optimal combination of <em>N</em>(5) and <em>N</em>(8). The most striking pattern is that the mutation rate for the first division varies considerably, but regardless of how the data are examined, it is markedly larger than those for subsequent divisions. Similar to the pattern observed in <a id="xref-ref-9-13" class="xref-bibr" href="#ref-9">Gao <em>et al.</em> (2011)</a>, the spermatogenesis has a relatively higher mutation rate than the interval divisions. With data aggregation, there appears to be a trend of appreciable mutation rate for the second division, and a similar pattern is observed for the stem cell stage although the magnitude is less appreciable. The cleavage stage, excluding the first (and perhaps also the second), harbors the smallest mutation rate. The ratio of the first division mutation rate to the mean mutation rate of the internal divisions are all larger than 100, with the highest ratio of 620 for lethality [97%, 98%).</p><div id="T6" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464811/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464811/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 6</span> <span class="caption-title">Full maximum likelihood estimates of <em>u</em> Ã 10<sup>3</sup> for several lethalities</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Mutation rate hypotheses testing</h3><p id="p-45">As defined in <em>Materials and Methods</em>, <strong><em>Î¼</em></strong> = (<em>u</em><sub>1</sub>, <em>u</em><sub>2</sub>, â¦ <em>u<sub>I</sub></em>) with <em>u<sub>i</sub></em> being the mutation rate per cell division in the <em>i</em>th interval. Then different constraints on the rates will affect the estimates of <strong><em>Î¼</em></strong> and the associated log-likelihood values are used as the basis for testing the hypotheses about the mutation rates. The following nine hypotheses will be considered:<span class="disp-formula" id="disp-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-19.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-19.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-13"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-20.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-20.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-14"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-21.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-21.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-15"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-22.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-22.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-16"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-23.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-23.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-17"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-24.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-24.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-18"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-25.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-25.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-19"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-26.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-26.gif"/></noscript></span></span><span class="disp-formula" id="disp-formula-20"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-27.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8/1503/embed/mml-math-27.gif"/></noscript></span></span>All the hypotheses except <em>H</em><sub>4</sub><em><sub>b</sub></em> are labeled the same as those in <a id="xref-ref-9-14" class="xref-bibr" href="#ref-9">Gao <em>et al.</em> (2011)</a> and <a id="xref-ref-7-8" class="xref-bibr" href="#ref-7">Fu (2013)</a>. <a id="xref-table-wrap-7-1" class="xref-table" href="#T7">Table 7</a> gives the values of these tests against the <em>H</em><sub>8</sub>.</p><div id="T7" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464650/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464650/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 7</span> <span class="caption-title">The values of the log-likelihood ratio test for various hypotheses <em>H<sub>i</sub></em> against <em>H</em><sub>8</sub></span><div class="sb-div caption-clear"></div></div></div><p id="p-47">As in <a id="xref-ref-9-15" class="xref-bibr" href="#ref-9">Gao <em>et al.</em> (2011)</a>, the assumption of equal mutation rate during germline development is soundly rejected and the evidence is stronger with data aggregation. As mentioned previously, with data aggregation there appears to be a trend that the second division and the stem-cell stage also have appreciable mutations (<a id="xref-table-wrap-6-2" class="xref-table" href="#T6">Table 6</a>), but the log-likelihood ratio tests in <a id="xref-table-wrap-7-2" class="xref-table" href="#T7">Table 7</a> does not provide significant support for the trend. In fact, there is no significant evidence to reject the hypothesis that all internal divisions, that is, from the second until right before gametogenesis, share the same mutation rate.</p><p id="p-48">To investigate the sensitivity of our inference with regard to possible sporadic inclusions of families with pre-existing mutations, we can exclude a certain fraction of families with a high percentages of mutants among offspring. However, it is not easy to decide the proper fractions to use, so a conservative approach was taken by removing all families with a percentage of mutants exceeding a given threshold value. <a id="xref-table-wrap-8-1" class="xref-table" href="#T8">Table 8</a> shows the results of the likelihood ratio tests for two different thresholds. At a 90% threshold value, for example, families of size 20 with 18 or more mutants were excluded, and families of size 30 with 27 or more mutants were excluded. As expected, the number of families excluded increases with the width of the lethality interval for a given threshold value, and decreasing the threshold from 90% to 85% results in doubling the number of families being removed. For the latter threshold value, as high as 20% of mutant families were removed. Even with such extreme exclusions, all the cases with significant test results shown in <a id="xref-table-wrap-7-3" class="xref-table" href="#T7">Table 7</a> remain the same.</p><div id="T8" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464676/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464676/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 8</span> <span class="caption-title">The values of the log-likelihood ratio test for various hypotheses <em>H<sub>i</sub></em> against <em>H</em><sub>8</sub> excluding families with high percentage of mutants</span><div class="sb-div caption-clear"></div></div></div><p id="p-50">So far we have grouped the five cell divisions in the spermatogenesis as one interval and thus assumed that mutation rates are constant within the interval. This hypothesis can be investigated by dividing the gametogenesis into two intervals. <a id="xref-table-wrap-9-1" class="xref-table" href="#T9">Table 9</a> lists the maximum likelihood estimates for different partitions of the five cell divisions in gametogenesis (to reduce the amount of computation, the third cell division and the remaining cleavage divisions are combined into one interval, since there is little evidence to suggest that their rates are different). It follows that the maximum difference in the log-likelihood value to that for the case of equal rate (i = 0) is 0.7. Therefore, there is little evidence to suggest different mutation rates in the process of gametogenesis, despite the trend that early cell divisions have a greater estimated mutation rate than the later divisions.</p><div id="T9" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464730/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464730/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 9</span> <span class="caption-title">Likelihood estimates of mutation rates with lethality â¥97% with gametogenesis split into two intervals with [1, 1], [2, 2], [3, 14], [15 â 6][â5, â(<em>i</em> + 1)], [â<em>i</em>, 38]</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-13"><h2 class="">Discussion</h2><p id="p-52">The full data set from our large-scale mutation screening experiment provides rich information for exploration in much more detail than previously possible for various aspects of the mutation process during germline development. The inference, taking advantage of the improved framework, leads to the following conclusions: (1) mutation rates during germline development are not equal, (2) the first division harbors the greatest mutation rate, (3) gametogenesis has mutation rates greater than other divisions except the first (and perhaps the second as well), (4) after the first cell division, the rate drops rapidly, and after the second division the rate becomes flat throughout the cleavage stage, (5) no evidence of rate difference during the gametogenesis, and (6) the number of PGCs after the eighth division is likely greater than that reported in the literature and they are not derived at random from the 256 cells at that stage nor from one or two ancestral cells at the fifth division.</p><p id="p-53">Because of reduced DNA repair efficiency during gametogenesis, greater mutation rate at gametogenesis is expected. Although it is generally known that zygotic control starts toward the end of the cleavage stage which for <em>Drosophila</em> is around the 10â14th divisions, current knowledge of <em>Drosophila</em> development does not provide an adequate explanation of why the first division (or with the second division) harbors a much elevated mutation rate compared with the rest of the cell divisions during the cleavage stage, which was pointed out earlier by <a id="xref-ref-9-16" class="xref-bibr" href="#ref-9">Gao <em>et al.</em> (2011)</a>. However, it is now known that some mutations in the sperm may be repaired in the early cleavage after fertilization (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Rathke <em>et al.</em> 2014</a>), so a portion of observed mutations in the early cleavage may be the result of incomplete repairing of pre-existing mutations. If all early cleavage mutations are derived as such, one would expect that the mutation rate for the first cell division would be similar to or smaller than that of gametogenesis; therefore, the much elevated mutation rate for the first cell division remains to be illuminated biologically.</p><p id="p-54">To safeguard our conclusions against artifacts in both the experiment and inference, we also carried out analysis with combinations of parameters deviated from the reported optimal set. Examining the impact of assumptions on the dynamics of population size is one such effort, another is to examine the impact of sporadic pre-existing mutations of high lethality that had managed to escape our surveillance. Such mutations would be identified by the experiment as ones that lead to 100% mutant offspring (if the experiment was perfect) or close to 100% mutant offspring if the z/z percentages in a few lines fluctuate upward to escape both the initial screening and subsequent allelism tests. Our analysis (<a id="xref-table-wrap-8-2" class="xref-table" href="#T8">Table 8</a>) shows that the aggressive removal of families with a high percentage of mutants does not change the main conclusions. Therefore, the possibility of some sporadic pre-existing mutations escaping our surveillance cannot be the primary cause of the sharp contrast of mutation rates along the germline development. The experiment and subsequent dissection of mutations was not perfect, which led to our investigation of the impact of the overall significance level on assigning mutants into mutational groups and subsequent inference. Again, all the main conclusions remains intact. One can conclude that the consistency of conclusions under various adjustments come from overwhelming information (both quantity and quality) in the data.</p><p id="p-55">The greater resolutions in data from 98% and 97% recessive lethality have resolved some ambiguity previously encountered but also reveals a striking pattern. Although the rate for gametogenesis increases from 1 to 1.5 and 1.9, the rate increase of the first cell division is much more profound, from 4 to 30 for 98% lethality and 60 for 97% lethality. Although some bias might have been introduced in the delineation of mutations from cross data, the impact of such factors is likely modest at best. This is because the same analysis was conducted for mutation patterns generated by using two different <em>Î±</em> values (0.5 and 0.95) and the conclusions remain mostly intact except for some relatively small changes in numerical values. Therefore, we are confident that the elevated mutation rate for the first cleavage with decreasing lethality is beyond reasonable doubt. However, there is an increasing discrepancy between the overall mutation rate estimated from likelihood analysis and that from the classical estimator when the rate is significantly higher than 1%. We are not certain about the cause of this discrepancy but again different criteria for delineating cross data are not the major cause. This is an issue that deserves further investigation and one possible reason may be due to increasing inaccuracy of approximating the larger probabilities of mutation patterns used in the likelihood analysis. If this is true, it implies that the mutation rate for the first cell division might have been underestimated. Nevertheless, the qualitative conclusions from the current analysis are unlikely to be altered significantly.</p><p id="p-56">The pattern of the ratio of the mutation rate of the first cell division and the mean rate of the interval divisions also deserve to be examined carefully. The lower ratio for greater lethality level (â¥99%) than those of lower lethality, say [97%,98%) for example, is apparent. One possible cause may be that a significant number of mutations, which are completely lethal or nearly lethal, have a dominant effect and the degree of dominance increases with the lethality in general. When the lethality is reduced, the second division starts to harbor a greater mutation rate, which also helps to lower the ratio. Another way to look at the phenomenon is to examine the average cluster size of mutations for different levels of lethality. <a id="xref-table-wrap-2-3" class="xref-table" href="#T2">Table 2</a> shows that on average (<em>i.e.</em>, <em>n<sub>m</sub></em>/<em>m<sub>t</sub></em>), mutations of lethality â¥ 99 is 2.2 whereas the average for lethalities of level [98%, 99%) and [97%, 98%) are 8.4 and 12.8, respectively. In general, mutations leading to smaller clusters tend to occur later than those to larger clusters and thus the selective disadvantage of mutations of high lethality may be one reason which prevents them from reaching a high frequency in the cell population within a host.</p><p id="p-57">It is clear that our experimental data have the resolution to test the validity of some assumptions about the dynamics of the population. The number of cell divisions for the sperm in the parental stage was initially thought to be 36, but after examining the experimental procedure carefully, we realized that by the time the <em>Î²</em>/z males were introduced into the parental stage, additional two to three stem cell divisions might have occurred because on average it takes 32 hr for a stem-cell cycle (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Wallenfang <em>et al.</em> 2006</a>). Furthermore, each male can only mate a limited number of times during 24 hr; thus, the offspring in the <em>F</em><sub>1</sub> may result from sperm of an even wider range of ages. Therefore, sperm in the Parental stage likely range from a minimal of 36 cell divisions to approximately 40 divisions. Indeed, the likelihood analysis suggests that 38 cell divisions provide the best overall fit. The population size after the eighth division was thought to be in the range of four to six as reported in the literature (<a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Drost and Lee 1995</a>, <a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">1998</a>), but the best fit suggests that may be a slight underestimate, and a more appropriate range should be 7â9. Although the aforementioned two quantities have previous knowledge to judge their validity, little can be found from the literature about how the PGCs after the eighth division are derived or selected among the 256 available cells. Constraints of physical space and the tendency that recently derived cells tend to cluster near each other suggests that the PGCs may not be a random sample from the 256 cells as assumed by default. This analysis provides strong evidence that they are derived from a relatively small number of ancestral cells at the fifth division, but the assumption that their common ancestor is a cell at the fifth generation can be soundly rejected.</p><p id="p-58">Regardless of the mechanism leading to the increased mutation rate ratio with decreasing lethality between the first division and gametogenesis, the trend is clear, and one can envisage that for neutral or nearly neutral mutations, the ratio may be even greater. This prospect lends stronger support to the explanation in <a id="xref-ref-9-17" class="xref-bibr" href="#ref-9">Gao <em>et al.</em> (2011)</a> concerning a lower ratio of male <em>vs.</em> female mutation rate than expected due to the difference in the number of cell divisions between sperm and eggs. This study further suggests that the rate differential between the first one or two with the remaining cell divisions can differ for mutations of different nature (here different lethalities). It is conceivable that this may be true for different genes/regions in the genome as well. Human genetic disease cases (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Vogel and Motulsky 1997</a>) as well as DNA sequence data (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Miyata <em>et al.</em> 1987</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Shimmin <em>et al.</em> 1993</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Li 1997</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Crow 2006</a>) has led to the conclusion known as male-driven evolution, <em>i.e.</em>, males dominate females in generating inheritable mutations in evolution. For 30-yr-old human males and females, there are roughly 400 and 30 cell divisions leading to sperm and eggs, respectively (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Drost and Lee 1995</a>), so the expected ratio of male-to-female mutation rate is larger than 10; however, estimates from sequence data vary and in general are lower than this ratio. Several possible causes have been put forward for this apparent discrepancy, but if the first (or first and second) cell division has a mutation rate many-fold larger than the average mutation rates for the remaining cell division, the phenomenon becomes easily explainable. For example, suppose the ratio of the mutation rate of the first cell division and the mean of the internal divisions is 100, then the male to female ratio would be expected to be (100 + 399)/(100 + 29) â 3.9 and if the ratio is 500, then the male to female mutation ratio would be about 1.7. Because different mutation types or regions may exhibit rather different ratios between the first division and the average, the ratio between male-to-female mutation rates can be expected to be quite variable.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-59">We thank those who performed part of the experiment, including Ji-fen Li, Zhen Xie, Fang Chen, Jian-rui Zhou, Qun-li Wang, Lei-hua He, Qian-qian Zhao, Zong-jun Luo, Rui-lin Zhang, Ji Yao, Rui-hong Zhang, Xing-qin Yin, Xiao-qing Yang, Ji-qin Liu, Ji-xin Yang, Xing-yun Wang, Tian-fen Zhang, Yong-ping Meng, Qiu-qi Li, Yan-hong Du, Shu-li Xiong, Mei Zhang, Mei-lan Huo, Li-xian Lou, Xiao-yun Jiang, and Ya-ting Liu. This work was partly support by grants from the Chinese NSF (30570248 and 91231120 YF) and the Betty Wheless Trotter Endowment Fund from The University of Texas Health Science Center at Houston, TX.</p></div><div class="section app" id="app-1"><h2 class="">Appendix</h2><div class="section" id="sec-14"><h3>Mutation patterns for different recessive lethalities</h3><div id="T10" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464727/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464727/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table A1</span> <span class="caption-title">Mutation patterns for recessive lethality [99â100%)</span><div class="sb-div caption-clear"></div></div></div><div id="T11" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464704/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464704/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table A2</span> <span class="caption-title">Mutation patterns for recessive lethality [98â99%)</span><div class="sb-div caption-clear"></div></div></div><div id="T12" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464751/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464751/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table A3</span> <span class="caption-title">Mutation patterns for recessive lethality [97â98%)</span><div class="sb-div caption-clear"></div></div></div></div></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1"><a class="rev-xref" href="#xref-fn-1-1">âµ</a><span class="fn-label">1</span> These authors are first authors.</p></li><li class="fn" id="fn-8"><p id="p-60"><em>Communicating editor: K. S. McKim</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-03-13"><span class="received-label">Received </span>March 13, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-06-10"><span class="accepted-label">Accepted </span>June 10, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright Â© 2014 Gao <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">âµ</a><div class="cit ref-cit ref-book" id="cit-4.8.1503.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abrahamson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lewis</span>  <span class="cit-name-given-names">E. B.</span></span></li></ol><cite>, <span class="cit-pub-date">1971</span>â<span class="cit-article-title">The detection of mutations in <em>Drosophila melanogaster</em></span>, pp. <span class="cit-fpage">461</span>â<span class="cit-lpage">484</span> in <span class="cit-source">Chemical Mutagens</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Hollaender</span>  <span class="cit-name-given-names">A.</span></span>. <span class="cit-publ-name">Plenum Press</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">âµ</a><div class="cit ref-cit ref-book" id="cit-4.8.1503.2"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ashburner</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1989</span>â<span class="cit-source">Drosophila: A Laboratory Handbook</span>. <span class="cit-publ-name">Cold Spring Harbor Laboratory</span>, <span class="cit-publ-loc">Cold Spring Harbor, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.8.1503.3" data-doi="10.1002/em.2850100411"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brodberg</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woodruff</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span>â<span class="cit-article-title">Specific reduction of N-dimethylnitro-samine mutagenicity in <em>Drosophila melanogaster</em> by dimethyl sulfoxide.</span> <abbr class="cit-jnl-abbrev">Environ. Mol. Mutagen.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">425</span>â<span class="cit-lpage">432</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnvironmental%2Band%2Bmolecular%2Bmutagenesis%26rft.stitle%253DEnviron%2BMol%2BMutagen%26rft.aulast%253DBrodberg%26rft.auinit1%253DR.%2BK.%26rft.volume%253D10%26rft.issue%253D4%26rft.spage%253D425%26rft.epage%253D432%26rft.atitle%253DSpecific%2Breduction%2Bof%2BN%252CN-dimethylnitrosamine%2Bmutagenicity%2Bin%2BDrosophila%2Bmelanogaster%2Bby%2Bdimethyl%2Bsulfoxide.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fem.2850100411%26rft_id%253Dinfo%253Apmid%252F3119334%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/em.2850100411&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3119334&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1503.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">âµ</a><div class="cit ref-cit ref-other" id="cit-4.8.1503.4" data-doi="10.1269/jrr.47.B75"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Crow</span>  <span class="cit-name-given-names">J. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Age and sex effects on human mutation rates: an old problem with new complexities.</span> <span class="cit-source">J. Radiat. Res.</span> <span class="cit-vol">47</span> <span class="cit-supplement">Suppl B</span><strong>:</strong> <span class="cit-fpage">B75</span>â<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BRadiat.%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1269%252Fjrr.47.B75%26rft_id%253Dinfo%253Apmid%252F17019055%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianJyIjtzOjU6InJlc2lkIjtzOjE0OiI0Ny9TdXBwbF9CL0I3NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOC8xNTAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.8.1503.5" data-doi="10.1002/em.2850250609"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drost</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">W. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span>â<span class="cit-article-title">Biological basis of germline mutation: Comparisons of spontaneous germline mutation rates among <em>Drosophila</em>, mouse and human.</span> <abbr class="cit-jnl-abbrev">Environ. Mol. Mutagen.</abbr> <span class="cit-vol">25</span>(<span class="cit-supplement">Suppl 26</span>)<strong>:</strong> <span class="cit-fpage">48</span>â<span class="cit-lpage">64</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnviron.%2BMol.%2BMutagen.%26rft.volume%253D25%26rft.spage%253D48%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fem.2850250609%26rft_id%253Dinfo%253Apmid%252F7789362%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/em.2850250609&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7789362&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1503.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1995RD91000008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.8.1503.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Drost</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">W. R.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span>â<span class="cit-article-title">The developmental basis for the germline mosaicism in mouse and <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">102/103</span>: <span class="cit-fpage">421</span>â<span class="cit-lpage">443</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D102103%26rft.spage%253D421%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.8.1503.7" data-doi="10.1534/genetics.113.151571"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y. X.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span>â<span class="cit-article-title">Statistical methods for analyzing <em>Drosophila</em> germline mutation rates.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">194</span>: <span class="cit-fpage">927</span>â<span class="cit-lpage">936</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.113.151571%26rft_id%253Dinfo%253Apmid%252F23636740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk0LzQvOTI3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzE1MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.8.1503.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huai</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-article-title">Estimating mutation rate: how to count mutations?</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">797</span>â<span class="cit-lpage">805</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DFu%26rft.auinit1%253DY.-X.%26rft.volume%253D164%26rft.issue%253D2%26rft.spage%253D797%26rft.epage%253D805%26rft.atitle%253DEstimating%2BMutation%2BRate%253A%2BHow%2Bto%2BCount%2BMutations%253F%26rft_id%253Dinfo%253Apmid%252F12807798%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTY0LzIvNzk3IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzE1MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.8.1503.9" data-doi="10.1073/pnas.1100233108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gao</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">X. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span>â<span class="cit-article-title">Highly variable recessive lethal or nearly lethal mutation rates during germline development of male <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">15914</span>â<span class="cit-lpage">15919</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1100233108%26rft_id%253Dinfo%253Apmid%252F21890796%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzM4LzE1OTE0IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzE1MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">âµ</a><div class="cit ref-cit ref-book" id="cit-4.8.1503.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gilbert</span>  <span class="cit-name-given-names">S. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span>â<span class="cit-source">Developmental Biology</span>. <span class="cit-edition">Ed. 7</span>. <span class="cit-publ-name">Sinauer Associates, Inc.</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">âµ</a><div class="cit ref-cit ref-book" id="cit-4.8.1503.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Graf</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Schaik</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wurgler</span>  <span class="cit-name-given-names">F. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span>â<span class="cit-source">Drosophila Genetics: A Practical Course</span>. <span class="cit-publ-name">Springer-Verlag</span>, <span class="cit-publ-loc">Berlin, Heidelberg</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">âµ</a><div class="cit ref-cit ref-book" id="cit-4.8.1503.12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Greenspan</span>  <span class="cit-name-given-names">S. F.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-source">Fly Pushing: The Theory and Practice of Drosophila Genetics</span>. <span class="cit-publ-name">Cold Spring Harbor Laboratory</span>, <span class="cit-publ-loc">Cold Spring Harbor, NY</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.8.1503.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huai</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woodruff</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-article-title">Clusters of identical new mutations can account for the overdispersed molecular clock.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">147</span>: <span class="cit-fpage">339</span>â<span class="cit-lpage">348</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHuai%26rft.auinit1%253DH.%26rft.volume%253D147%26rft.issue%253D1%26rft.spage%253D339%26rft.epage%253D348%26rft.atitle%253DClusters%2Bof%2BIdentical%2BNew%2BMutations%2BCan%2BAccount%2Bfor%2Bthe%2B%2560%2560Overdispersed%2527%2527%2BMolecular%2BClock%26rft_id%253Dinfo%253Apmid%252F9286693%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQ3LzEvMzM5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzE1MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">âµ</a><div class="cit ref-cit ref-book" id="cit-4.8.1503.14"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-source">Molecular Evolution</span>. <span class="cit-publ-name">Sinauer Associates, Inc.</span>, <span class="cit-publ-loc">Sunderland, MA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">âµ</a><div class="cit ref-cit ref-book" id="cit-4.8.1503.15"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lindsley</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zimm</span>  <span class="cit-name-given-names">G. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span>â<span class="cit-source">The Genome of Drosophila melanogaster</span>. <span class="cit-publ-name">Academic Press, Inc.</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.8.1503.16" data-doi="10.1002/em.2860070506"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mason</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valencia</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woodruff</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zimmering</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1985</span>â<span class="cit-article-title">Genetic drift and seasonal variation in spontaneous mutation frequencies in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Environ. Mutagen.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">663</span>â<span class="cit-lpage">676</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEnvironmental%2Bmutagenesis%26rft.stitle%253DEnviron%2BMutagen%26rft.aulast%253DMason%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D7%26rft.issue%253D5%26rft.spage%253D663%26rft.epage%253D676%26rft.atitle%253DGenetic%2Bdrift%2Band%2Bseasonal%2Bvariation%2Bin%2Bspontaneous%2Bmutation%2Bfrequencies%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fem.2860070506%26rft_id%253Dinfo%253Apmid%252F3930236%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/em.2860070506&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=3930236&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1503.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1985ARL3900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.8.1503.17" data-doi="10.1101/SQB.1987.052.01.094"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Miyata</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hayashida</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuma</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitsuyasa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yasunaga</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1987</span>â<span class="cit-article-title">Male-driven molecular evolution: a model and nucleotide sequence analysis.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Symp. Quant. Biol.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">863</span>â<span class="cit-lpage">867</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarbor%2BSymposia%2Bon%2BQuantitative%2BBiology%26rft.stitle%253DCold%2BSpring%2BHarb%2BSymp%2BQuant%2BBiol%26rft.aulast%253DMiyata%26rft.auinit1%253DT.%26rft.volume%253D52%26rft.issue%253D0%26rft.spage%253D863%26rft.epage%253D867%26rft.atitle%253DMale-driven%2BMolecular%2BEvolution%253A%2BA%2BModel%2Band%2BNucleotide%2BSequence%2BAnalysis%26rft_id%253Dinfo%253Adoi%252F10.1101%252FSQB.1987.052.01.094%26rft_id%253Dinfo%253Apmid%252F3454295%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic3FiIjtzOjU6InJlc2lkIjtzOjg6IjUyLzAvODYzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzE1MDMuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.8.1503.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1928</span>â<span class="cit-article-title">The measurement of gene mutation rate in <em>Drosophila</em>, its high variability, and its dependence upon temperature.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">279</span>â<span class="cit-lpage">357</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DMuller%26rft.auinit1%253DH.%2BJ.%26rft.volume%253D13%26rft.issue%253D4%26rft.spage%253D279%26rft.epage%253D357%26rft.atitle%253DTHE%2BMEASUREMENT%2BOF%2BGENE%2BMUTATION%2BRATE%2BIN%2BDROSOPHILA%252C%2BITS%2BHIGH%2BVARIABILITY%252C%2BAND%2BITS%2BDEPENDENCE%2BUPON%2BTEMPERATURE%26rft_id%253Dinfo%253Apmid%252F17246553%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiIxMy80LzI3OSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOC8xNTAzLmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">âµ</a><div class="cit ref-cit ref-book" id="cit-4.8.1503.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muller</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oster</span>  <span class="cit-name-given-names">I. I.</span></span></li></ol><cite>, <span class="cit-pub-date">1963</span>â<span class="cit-article-title">Some mutational techniques in <em>Drosophila</em></span>, pp. <span class="cit-fpage">249</span>â<span class="cit-lpage">278</span> in <span class="cit-source">Methodology in Basic Genetics</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Burdette</span>  <span class="cit-name-given-names">W. J.</span></span>. <span class="cit-publ-name">Holden-Day, Inc.</span>, <span class="cit-publ-loc">San Francisco, CA</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.8.1503.20" data-doi="10.1016/j.bbagrm.2013.08.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rathke</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baarends</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Awe</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Renkawitz-Pohl</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span>â<span class="cit-article-title">Chromatin dynamics during spermiogenesis.</span> <abbr class="cit-jnl-abbrev">Biochim. Biophys. Acta</abbr> <span class="cit-vol">1839</span>: <span class="cit-fpage">155</span>â<span class="cit-lpage">168</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiochim.%2BBiophys.%2BActa%26rft.volume%253D1839%26rft.spage%253D155%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.bbagrm.2013.08.004%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.bbagrm.2013.08.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.8.1503.21" data-doi="10.1038/362745a0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shimmin</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">B. H.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span>â<span class="cit-article-title">Male-driven evolution of DNA sequences.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">362</span>: <span class="cit-fpage">745</span>â<span class="cit-lpage">747</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DShimmin%26rft.auinit1%253DL.%2BC.%26rft.volume%253D362%26rft.issue%253D6422%26rft.spage%253D745%26rft.epage%253D747%26rft.atitle%253DMale-driven%2Bevolution%2Bof%2BDNA%2Bsequences.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F362745a0%26rft_id%253Dinfo%253Apmid%252F8469284%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/362745a0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8469284&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1503.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1993KY45000054&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">âµ</a><div class="cit ref-cit ref-book" id="cit-4.8.1503.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sonnenblick</span>  <span class="cit-name-given-names">B. P.</span></span></li></ol><cite>, <span class="cit-pub-date">1965</span>â<span class="cit-article-title">The early embryology of <em>Drosophila melanogaster</em></span>, pp. <span class="cit-fpage">62</span>â<span class="cit-lpage">167</span> in <span class="cit-source">Biology of Drosophila</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Demerec</span>  <span class="cit-name-given-names">M.</span></span>. <span class="cit-publ-name">Hafner Publishing Company</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.8.1503.23" data-doi="10.1073/pnas.77.2.1059"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">J. N.</span>  <span class="cit-name-suffix">Jr.</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woodruff</span>  <span class="cit-name-given-names">R. C.</span></span></li></ol><cite>, <span class="cit-pub-date">1980</span>â<span class="cit-article-title">Increased mutation in crosses between geographically separated strains of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">77</span>: <span class="cit-fpage">1059</span>â<span class="cit-lpage">1062</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DThompson%26rft.auinit1%253DJ.%2BN.%26rft.volume%253D77%26rft.issue%253D2%26rft.spage%253D1059%26rft.epage%253D1062%26rft.atitle%253DIncreased%2Bmutation%2Bin%2Bcrosses%2Bbetween%2Bgeographically%2Bseparated%2Bstrains%2Bof%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.77.2.1059%26rft_id%253Dinfo%253Apmid%252F6767240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI3Ny8yLzEwNTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzgvMTUwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.8.1503.24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span>â<span class="cit-article-title">Recombination and response to selection in Drosophila melanogaster.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">85</span>: <span class="cit-fpage">125</span>â<span class="cit-lpage">140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DThompson%26rft.auinit1%253DV.%26rft.volume%253D85%26rft.issue%253D1%26rft.spage%253D125%26rft.epage%253D140%26rft.atitle%253DRECOMBINATION%2BAND%2BRESPONSE%2BTO%2BSELECTION%2BIN%2BDROSOPHILA%2BMELANOGASTER%26rft_id%253Dinfo%253Apmid%252F402302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiODUvMS8xMjUiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzgvMTUwMy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">âµ</a><div class="cit ref-cit ref-book" id="cit-4.8.1503.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vogel</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Motulsky</span>  <span class="cit-name-given-names">A. G.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span>â<span class="cit-source">Human Genetics: Problems and Approaches</span>. <span class="cit-edition">Ed. 3</span>. <span class="cit-publ-name">Springer-Verlag</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.8.1503.26" data-doi="10.1111/j.1474-9726.2006.00221.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wallenfang</span>  <span class="cit-name-given-names">M. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nyak</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DiNardo</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span>â<span class="cit-article-title">Dynamics of the male germline stem cell population during aging of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Aging Cell</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">297</span>â<span class="cit-lpage">304</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAging%2Bcell%26rft.stitle%253DAging%2BCell%26rft.aulast%253DWallenfang%26rft.auinit1%253DM.%2BR.%26rft.volume%253D5%26rft.issue%253D4%26rft.spage%253D297%26rft.epage%253D304%26rft.atitle%253DDynamics%2Bof%2Bthe%2Bmale%2Bgermline%2Bstem%2Bcell%2Bpopulation%2Bduring%2Baging%2Bof%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1474-9726.2006.00221.x%26rft_id%253Dinfo%253Apmid%252F16800845%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1474-9726.2006.00221.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16800845&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1503.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000239121100003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.8.1503.27" data-doi="10.1007/s10709-005-4982-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Woodruff</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">J. N. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span>â<span class="cit-article-title">The fundamental theorem of neutral evolution: Rates of substitution and mutation should factor in premeiotic clusters.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">333</span>â<span class="cit-lpage">339</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.stitle%253DGenetica%26rft.aulast%253DWoodruff%26rft.auinit1%253DR.%2BC.%26rft.volume%253D125%26rft.issue%253D2-3%26rft.spage%253D333%26rft.epage%253D339%26rft.atitle%253DThe%2Bfundamental%2Btheorem%2Bof%2Bneutral%2Bevolution%253A%2Brates%2Bof%2Bsubstitution%2Band%2Bmutation%2Bshould%2Bfactor%2Bin%2Bpremeiotic%2Bclusters.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs10709-005-4982-7%26rft_id%253Dinfo%253Apmid%252F16247704%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s10709-005-4982-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16247704&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1503.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000232841800020&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.8.1503.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Woodruff</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">J. J. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seeger</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Spivey</span>  <span class="cit-name-given-names">W. E.</span></span></li></ol><cite>, <span class="cit-pub-date">1984</span>â<span class="cit-article-title">Variation in spontaneous mutation and repair in natural population lines of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Heredity</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">223</span>â<span class="cit-lpage">234</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DHeredity%26rft.volume%253D58%26rft.spage%253D223%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">âµ</a><div class="cit ref-cit ref-journal" id="cit-4.8.1503.29" data-doi="10.1007/BF00121363"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Woodruff</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huai</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">J. N.</span>  <span class="cit-name-suffix">Jr</span>.</span></li></ol><cite>., <span class="cit-pub-date">1996</span>â<span class="cit-article-title">Clusters of identical new mutation in the evolutionary landscape.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">149</span>â<span class="cit-lpage">160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.stitle%253DGenetica%26rft.aulast%253DWoodruff%26rft.auinit1%253DR.%2BC.%26rft.volume%253D98%26rft.issue%253D2%26rft.spage%253D149%26rft.epage%253D160%26rft.atitle%253DClusters%2Bof%2Bidentical%2Bnew%2Bmutation%2Bin%2Bthe%2Bevolutionary%2Blandscape.%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00121363%26rft_id%253Dinfo%253Apmid%252F8976063%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00121363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8976063&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1503.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1996VY80000004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/8/1503.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/8/1491" title="Linking Genetics to Structural Biology: Complex Heterozygosity Screening with Actin Alanine Scan Alleles Identifies Functionally Related Surfaces on Yeast Actin" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/8/1515" title="Differentiation State-Specific Mitochondrial Dynamic Regulatory Networks Are Revealed by Global Transcriptional Analysis of the Developing Chicken Lens" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/8"><span>Volume 4 Issue 8, August 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/8.cover-source.jpg?itok=uF7Z9Bah" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/8/1503" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tceYGwmJo2mtihEWozosRHg0iK-qfliU8Db-_b8ysyQ" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/8/1503" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/8/1503" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/8/1503" class="active" data-icon-position="" data-hide-link-title="0">Pattern of Mutation Rates in the Germline of Drosophila melanogaster Males from a Large-Scale Mutation Screening Experiment</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459947" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form--8hwMIJXnPHZTh6HoM1mScxQ8E8itOZ3AnnP4vrWR8w" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/8/1503.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/8/1503" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-HnHN08u1fxEiUjF2zQrhTw4SqGEWJuqdaM4w44zSz5Q" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/8/1503" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/8/1503.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459947" id="node459947--2" data-pisa="ggg;4/8/1503" data-pisa-master="ggg;g3.114.011056" data-apath="/ggg/4/8/1503.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Pattern of Mutation Rates in the Germline of <em>Drosophila melanogaster</em> Males from a Large-Scale Mutation Screening Experiment</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jian-Jun Gao</span>, <span class="highwire-citation-author" data-delta="1">Xue-Rong Pan</span>, <span class="highwire-citation-author" data-delta="2">Jing Hu</span>, <span class="highwire-citation-author" data-delta="3">Li Ma</span>, <span class="highwire-citation-author" data-delta="4">Jian-Min Wu</span>, <span class="highwire-citation-author" data-delta="5">Ye-Lin Shao</span>, <span class="highwire-citation-author" data-delta="6">Shi-Meng Ai</span>, <span class="highwire-citation-author" data-delta="7">Shu-Qun Liu</span>, <span class="highwire-citation-author" data-delta="8">Sara A. Barton</span>, <span class="highwire-citation-author" data-delta="9">Ronny C. Woodruff</span>, <span class="highwire-citation-author" data-delta="10">Ya-Ping Zhang</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="11"><a href="http://orcid.org/0000-0002-1224-4075" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yun-Xin Fu</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1503-1514;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.011056 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459947--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jian-Jun Gao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jian-Jun%2BGao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gao%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJian-Jun%2BGao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Xue-Rong Pan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xue-Rong%2BPan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pan%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXue-Rong%2BPan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jing Hu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jing%2BHu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJing%2BHu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Li Ma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Li%2BMa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ma%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALi%2BMa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jian-Min Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jian-Min%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJian-Min%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Ye-Lin Shao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ye-Lin%2BShao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shao%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYe-Lin%2BShao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Shi-Meng Ai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shi-Meng%2BAi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ai%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShi-Meng%2BAi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Shu-Qun Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shu-Qun%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShu-Qun%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Sara A. Barton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Division of Biostatistics and Human Genetics Center, School of Public Health, The University of Texas at Houston, Houston, Texas</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sara%2BA.%2BBarton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barton%20SA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASara%2BA.%2BBarton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Ronny C. Woodruff </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Biological Sciences, Bowling Green State University, Bowling Green, Ohio</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ronny%2BC.%2BWoodruff%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Woodruff%20RC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARonny%2BC.%2BWoodruff%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Ya-Ping Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China</div><div class='author-affiliation'><span class='nlm-sup'>Â§</span>State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming, Yunnan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ya-Ping%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYa-Ping%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Yun-Xin Fu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Biological Sciences, Bowling Green State University, Bowling Green, Ohio</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yun-Xin%2BFu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYun-Xin%2BFu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1224-4075" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yun-Xin Fu</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:yunxin.fu@uth.tmc.edu" class="" data-icon-position="" data-hide-link-title="0">yunxin.fu@uth.tmc.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/8/1503" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/8/1503&amp;title=Pattern%20of%20Mutation%20Rates%20in%20the%20Germline%20of%20Drosophila%20melanogaster%20Males%20from%20a%20Large-Scale%20Mutation%20Screening%20Experiment" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/8/1503&amp;title=Pattern%20of%20Mutation%20Rates%20in%20the%20Germline%20of%20Drosophila%20melanogaster%20Males%20from%20a%20Large-Scale%20Mutation%20Screening%20Experiment" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/8/1503&amp;title=Pattern%20of%20Mutation%20Rates%20in%20the%20Germline%20of%20Drosophila%20melanogaster%20Males%20from%20a%20Large-Scale%20Mutation%20Screening%20Experiment" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/8/1503&amp;text=Pattern%20of%20Mutation%20Rates%20in%20the%20Germline%20of%20Drosophila%20melanogaster%20Males%20from%20a%20Large-Scale%20Mutation%20Screening%20Experiment" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/8/1503&amp;title=Pattern%20of%20Mutation%20Rates%20in%20the%20Germline%20of%20Drosophila%20melanogaster%20Males%20from%20a%20Large-Scale%20Mutation%20Screening%20Experiment" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/8/1503&amp;t=Pattern%20of%20Mutation%20Rates%20in%20the%20Germline%20of%20Drosophila%20melanogaster%20Males%20from%20a%20Large-Scale%20Mutation%20Screening%20Experiment" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/8/1503&amp;title=Pattern%20of%20Mutation%20Rates%20in%20the%20Germline%20of%20Drosophila%20melanogaster%20Males%20from%20a%20Large-Scale%20Mutation%20Screening%20Experiment" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/8/1503&amp;title=Pattern%20of%20Mutation%20Rates%20in%20the%20Germline%20of%20Drosophila%20melanogaster%20Males%20from%20a%20Large-Scale%20Mutation%20Screening%20Experiment" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459947/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459947" id="node459947--3" data-pisa="ggg;4/8/1503" data-pisa-master="ggg;g3.114.011056" data-apath="/ggg/4/8/1503.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Pattern of Mutation Rates in the Germline of <em>Drosophila melanogaster</em> Males from a Large-Scale Mutation Screening Experiment</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Jian-Jun Gao</span>, <span class="highwire-citation-author" data-delta="1">Xue-Rong Pan</span>, <span class="highwire-citation-author" data-delta="2">Jing Hu</span>, <span class="highwire-citation-author" data-delta="3">Li Ma</span>, <span class="highwire-citation-author" data-delta="4">Jian-Min Wu</span>, <span class="highwire-citation-author" data-delta="5">Ye-Lin Shao</span>, <span class="highwire-citation-author" data-delta="6">Shi-Meng Ai</span>, <span class="highwire-citation-author" data-delta="7">Shu-Qun Liu</span>, <span class="highwire-citation-author" data-delta="8">Sara A. Barton</span>, <span class="highwire-citation-author" data-delta="9">Ronny C. Woodruff</span>, <span class="highwire-citation-author" data-delta="10">Ya-Ping Zhang</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="11"><a href="http://orcid.org/0000-0002-1224-4075" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yun-Xin Fu</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1503-1514;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.011056 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459947--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Jian-Jun Gao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jian-Jun%2BGao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gao%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJian-Jun%2BGao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Xue-Rong Pan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xue-Rong%2BPan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pan%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXue-Rong%2BPan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jing Hu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jing%2BHu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJing%2BHu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Li Ma </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Li%2BMa%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ma%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALi%2BMa%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Jian-Min Wu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jian-Min%2BWu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Wu%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJian-Min%2BWu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Ye-Lin Shao </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ye-Lin%2BShao%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shao%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYe-Lin%2BShao%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Shi-Meng Ai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shi-Meng%2BAi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ai%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShi-Meng%2BAi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Shu-Qun Liu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Shu-Qun%2BLiu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Liu%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AShu-Qun%2BLiu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Sara A. Barton </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â </span>Division of Biostatistics and Human Genetics Center, School of Public Health, The University of Texas at Houston, Houston, Texas</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sara%2BA.%2BBarton%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Barton%20SA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASara%2BA.%2BBarton%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Ronny C. Woodruff </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Biological Sciences, Bowling Green State University, Bowling Green, Ohio</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ronny%2BC.%2BWoodruff%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Woodruff%20RC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARonny%2BC.%2BWoodruff%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Ya-Ping Zhang </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China</div><div class='author-affiliation'><span class='nlm-sup'>Â§</span>State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of Zoology, Chinese Academy of Sciences, Kunming, Yunnan, China</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ya-Ping%2BZhang%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zhang%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AYa-Ping%2BZhang%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Yun-Xin Fu </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Laboratory for Conservation and Utilization of Bio-Resources, Yunnan University, Kunming, Yunnan, China</div><div class='author-affiliation'><span class='nlm-sup'>â¡</span>Department of Biological Sciences, Bowling Green State University, Bowling Green, Ohio</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yun-Xin%2BFu%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fu%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYun-Xin%2BFu%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1224-4075" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yun-Xin Fu</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:yunxin.fu@uth.tmc.edu" class="" data-icon-position="" data-hide-link-title="0">yunxin.fu@uth.tmc.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1503&amp;rft.epage=1514&amp;rft.atitle=Pattern%20of%20Mutation%20Rates%20in%20the%20Germline%20of%20Drosophila%20melanogaster%20Males%20from%20a%20Large-Scale%20Mutation%20Screening%20Experiment&amp;rft.volume=4&amp;rft.issue=8&amp;rft.date=2014-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Gao%2C+Jian-Jun&amp;rft.au=Pan%2C+Xue-Rong&amp;rft.au=Hu%2C+Jing&amp;rft.au=Ma%2C+Li&amp;rft.au=Wu%2C+Jian-Min&amp;rft.au=Shao%2C+Ye-Lin&amp;rft.au=Ai%2C+Shi-Meng&amp;rft.au=Liu%2C+Shu-Qun&amp;rft.au=Barton%2C+Sara+A.&amp;rft.au=Woodruff%2C+Ronny+C.&amp;rft.au=Zhang%2C+Ya-Ping&amp;rft.au=Fu%2C+Yun-Xin"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459947/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459947/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459947/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459947/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459947/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459947/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459947/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459947/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459947/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459947/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459947/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459947/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/8/1503" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-13" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#app-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Appendix</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/8/1503.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/8/1503.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last odd"><a href="/content/4/8/1503.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
