<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/8/1395" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/8/1395.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/8/1395.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/8/1395.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/8/1395.atom" />
<meta name="HW.pisa" content="ggg;4/8/1395" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Fluorescence In Situ Hybridization and Optical Mapping to Correct Scaffold Arrangement in the Tomato Genome" />
<meta name="DC.Identifier" content="10.1534/g3.114.011197" />
<meta name="DC.Date" content="2014-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Shearer et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="The order and orientation (arrangement) of all 91 sequenced scaffolds in the 12 pseudomolecules of the recently published tomato ( Solanum lycopersicum , 2 n = 2 x = 24) genome sequence were positioned based on marker order in a high-density linkage map. Here, we report the arrangement of these scaffolds determined by two independent physical methods, bacterial artificial chromosome–fluorescence in situ hybridization (BAC-FISH) and optical mapping. By localizing BACs at the ends of scaffolds to spreads of tomato synaptonemal complexes (pachytene chromosomes), we showed that 45 scaffolds, representing one-third of the tomato genome, were arranged differently than predicted by the linkage map. These scaffolds occur mostly in pericentric heterochromatin where 77% of the tomato genome is located and where linkage mapping is less accurate due to reduced crossing over. Although useful for only part of the genome, optical mapping results were in complete agreement with scaffold arrangement by FISH but often disagreed with scaffold arrangement based on the linkage map. The scaffold arrangement based on FISH and optical mapping changes the positions of hundreds of markers in the linkage map, especially in heterochromatin. These results suggest that similar errors exist in pseudomolecules from other large genomes that have been assembled using only linkage maps to predict scaffold arrangement, and these errors can be corrected using FISH and/or optical mapping. Of note, BAC-FISH also permits estimates of the sizes of gaps between scaffolds, and unanchored BACs are often visualized by FISH in gaps between scaffolds and thus represent starting points for filling these gaps." />
<meta name="DC.Contributor" content="Lindsay A. Shearer" />
<meta name="DC.Contributor" content="Lorinda K. Anderson" />
<meta name="DC.Contributor" content="Hans de Jong" />
<meta name="DC.Contributor" content="Sandra Smit" />
<meta name="DC.Contributor" content="José Luis Goicoechea" />
<meta name="DC.Contributor" content="Bruce A. Roe" />
<meta name="DC.Contributor" content="Axin Hua" />
<meta name="DC.Contributor" content="James J. Giovannoni" />
<meta name="DC.Contributor" content="Stephen M. Stack" />
<meta name="article:published_time" content="2014-08-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Fluorescence In Situ Hybridization and Optical Mapping to Correct Scaffold Arrangement in the Tomato Genome" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;The order and orientation (arrangement) of all 91 sequenced scaffolds in the 12 pseudomolecules of the recently published tomato (&lt;i&gt;Solanum lycopersicum&lt;/i&gt;, 2&lt;i&gt;n &lt;/i&gt;= 2&lt;i&gt;x =&lt;/i&gt; 24) genome sequence were positioned based on marker order in a high-density linkage map. Here, we report the arrangement of these scaffolds determined by two independent physical methods, bacterial artificial chromosome–fluorescence &lt;i&gt;in situ&lt;/i&gt; hybridization (BAC-FISH) and optical mapping. By localizing BACs at the ends of scaffolds to spreads of tomato synaptonemal complexes (pachytene chromosomes), we showed that 45 scaffolds, representing one-third of the tomato genome, were arranged differently than predicted by the linkage map. These scaffolds occur mostly in pericentric heterochromatin where 77% of the tomato genome is located and where linkage mapping is less accurate due to reduced crossing over. Although useful for only part of the genome, optical mapping results were in complete agreement with scaffold arrangement by FISH but often disagreed with scaffold arrangement based on the linkage map. The scaffold arrangement based on FISH and optical mapping changes the positions of hundreds of markers in the linkage map, especially in heterochromatin. These results suggest that similar errors exist in pseudomolecules from other large genomes that have been assembled using only linkage maps to predict scaffold arrangement, and these errors can be corrected using FISH and/or optical mapping. Of note, BAC-FISH also permits estimates of the sizes of gaps between scaffolds, and unanchored BACs are often visualized by FISH in gaps between scaffolds and thus represent starting points for filling these gaps.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/08/01" />
<meta name="citation_mjid" content="ggg;4/8/1395" />
<meta name="citation_id" content="4/8/1395" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/8/1395" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/8/1395.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/8/1395.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/8/1395.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.011197" />
<meta name="citation_pmid" content="24879607" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="1395" />
<meta name="citation_lastpage" content="1405" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Lindsay A. Shearer" />
<meta name="citation_author_institution" content="Department of Biology, Colorado State University, Fort Collins, Colorado 80523" />
<meta name="citation_author" content="Lorinda K. Anderson" />
<meta name="citation_author_institution" content="Department of Biology, Colorado State University, Fort Collins, Colorado 80523" />
<meta name="citation_author" content="Hans de Jong" />
<meta name="citation_author_institution" content="Laboratory of Genetics, Wageningen University and Research Centre (WUR), Droevendaalsesteeg 1, 6708 PB Wageningen, The Netherlands" />
<meta name="citation_author" content="Sandra Smit" />
<meta name="citation_author_institution" content="Laboratory of Bioinformatics, WUR, Droevendaalsesteeg1, 6708 PB Wageningen, The Netherlands" />
<meta name="citation_author" content="José Luis Goicoechea" />
<meta name="citation_author_institution" content="Arizona Genomics Institute, University of Arizona, Tucson, Arizona 85721" />
<meta name="citation_author" content="Bruce A. Roe" />
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, Stephenson Research and Technology Center, University of Oklahoma, Norman, Oklahoma 73019" />
<meta name="citation_author" content="Axin Hua" />
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, Stephenson Research and Technology Center, University of Oklahoma, Norman, Oklahoma 73019" />
<meta name="citation_author" content="James J. Giovannoni" />
<meta name="citation_author_institution" content="Department of Plant Biology, Cornell University, Ithaca, New York 14853" />
<meta name="citation_author" content="Stephen M. Stack" />
<meta name="citation_author_institution" content="Department of Biology, Colorado State University, Fort Collins, Colorado 80523" />
<meta name="citation_author_email" content="Stephen.Stack@colostate.edu" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_title=Cytogenetic and genome research;citation_journal_abbrev=Cytogenet Genome Res;citation_author=L. K. Anderson;citation_author=P. A. Covey;citation_author=L. R. Larsen;citation_author=P. A. Bedinger;citation_author=S. M. Stack;citation_title=Structural differences in chromosomes distinguish species in the tomato clade.;citation_pages=24-34;citation_volume=129;citation_year=2010;citation_issue=1-3;citation_pmid=20551606;citation_doi=10.1159/000313850" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=S. Chamala;citation_author=A. S. Chanderbali;citation_author=J. P. Der, T. Lan, B. Walts;citation_title=Assembly and Validation of the Genome of the Nonmodel Basal Angiosperm Amborella;citation_pages=1516-1517;citation_volume=342;citation_year=2013;citation_issue=6165;citation_pmid=24357320;citation_doi=10.1126/science.1241130" />
<meta name="citation_reference" content="citation_title=Cytogenetic and molecular studies on tomato chromosomes using diploid tomato and tomato monosomic additions in tetraploid potato;citation_author=S.-B. Chang;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=S.-B. Chang;citation_author=L. K. Anderson;citation_author=J. D. Sherman;citation_author=S. M. Royer;citation_author=S. M. Stack;citation_title=Predicting and testing physical locations of genetically mapped loci on tomato pachytene chromosome 1.;citation_volume=176;citation_year=2007;citation_pmid=17565940;citation_doi=10.1534/genetics.107.074138" />
<meta name="citation_reference" content="citation_journal_title=Chrom. Res.;citation_author=S.-B. Chang;citation_author=T.-J. Yang;citation_author=E. Datema;citation_author=J. V. Vugt;citation_author=B. Vosman;citation_title=FISH mapping and molecular organization of the major repetitive sequences of tomato.;citation_pages=919-933;citation_volume=16;citation_year=2008;citation_pmid=18688733" />
<meta name="citation_reference" content="citation_journal_title=Chrom. Res.;citation_author=Z. Cheng;citation_author=C. R. Buell;citation_author=R. A. Wing;citation_author=J. Jiang;citation_title=Resolution of fluorescence in-situ hybridization mapping on rice mitotic prometaphase chromosomes, meiotic pachytene chromosomes and extended DNA fibers.;citation_pages=379-387;citation_volume=10;citation_year=2002;citation_pmid=12296520" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=Z. Cheng;citation_author=G. G. Presting;citation_author=C. R. Buell;citation_author=R. A. Wing;citation_author=J. Jiang;citation_title=High-Resolution Pachytene Chromosome Mapping of Bacterial Artificial Chromosomes Anchored by Genetic Markers Reveals the Centromere Location and the Distribution of Genetic Recombination Along Chromosome 10 of Rice;citation_pages=1749-1757;citation_volume=157;citation_year=2001;citation_issue=4;citation_pmid=11290728" />
<meta name="citation_reference" content="citation_journal_title=Trends in plant science;citation_journal_abbrev=Trends Plant Sci;citation_author=J. H. de Jong;citation_author=P. F. Fransz;citation_author=P. Zabel;citation_title=High resolution FISH in plants - techniques and applications.;citation_pages=258-263;citation_volume=4;citation_year=1999;citation_issue=7;citation_pmid=10407441;citation_doi=10.1016/S1360-1385(99)01436-3" />
<meta name="citation_reference" content="citation_journal_title=Gene;citation_author=M. Di Filippo;citation_author=A. Traini;citation_author=N. D’Agostino;citation_author=L. Frusciante;citation_author=M. L. Chiusano;citation_title=Euchromatic and heterochromatic compositional properties emerging from the analysis of Solanum lycopersicum BAC sequences.;citation_pages=176-181;citation_volume=499;citation_year=2012;citation_pmid=22391094" />
<meta name="citation_reference" content="citation_journal_title=Analytical Chemistry (Washington, DC);citation_journal_abbrev=Analytical Chemistry (Washington, DC);citation_author=E. T. Dimalanta;citation_author=A. Lim;citation_author=R. Runnheim;citation_author=C. Lamers;citation_author=C. Churas;citation_title=A microfluidic system for large DNA molecule arrays.;citation_pages=5293-5301;citation_volume=76;citation_year=2004;citation_issue=18;citation_pmid=15362885;citation_doi=10.1021/ac0496401" />
<meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.;citation_author=Y. Dong;citation_author=M. Xie;citation_author=Y. Jiang, N. Xiao, X. Du;citation_title=Sequencing and automated whole-genome optical mapping of the genome of a domestic goat (Capra hircus).;citation_pages=135-141;citation_volume=31;citation_year=2013;citation_pmid=23263233;citation_doi=10.1038/nbt.2478" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=A. Frary;citation_author=Y. Xu;citation_author=J. Liu;citation_author=S. Mitchell;citation_author=E. Tedeschi;citation_title=Development of a set of PCR-based anchor markers encompassing the tomato genome and evaluation of their usefulness for genetics and breeding experiments.;citation_pages=291-312;citation_volume=111;citation_year=2005;citation_issue=2;citation_pmid=15926074;citation_doi=10.1007/s00122-005-2023-7" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=T. M. Fulton;citation_author=R. Van der Hoeven;citation_author=N. T. Eannetta;citation_author=S. D. Tanksley;citation_title=Identification, Analysis, and Utilization of Conserved Ortholog Set Markers for Comparative Genomics in Higher Plants;citation_pages=1457-1467;citation_volume=14;citation_year=2002;citation_issue=7;citation_pmid=12119367;citation_doi=10.1105/tpc.010479" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. Iovene;citation_author=S. M. Wielgus;citation_author=P. W. Simon;citation_author=C. R. Buell;citation_author=J. Jiang;citation_title=Chromatin structure and physical mapping of chromosome 6 of potato and comparative analyses with tomato.;citation_volume=180;citation_year=2008;citation_pmid=18791232;citation_doi=10.1534/genetics.108.093179" />
<meta name="citation_reference" content="citation_journal_title=BMC plant biology [electronic resource];citation_journal_abbrev=BMC Plant Biol;citation_author=S.-H. Jo;citation_author=D.-H. Koo;citation_author=J. F. Kim;citation_author=C.-G. Hur;citation_author=S. Lee;citation_title=Evolution of ribosomal DNA-derived satellite repeat in tomato genome.;citation_pages=42-42;citation_volume=9;citation_year=2009;citation_pmid=19351415;citation_doi=10.1186/1471-2229-9-42" />
<meta name="citation_reference" content="citation_journal_title=Journal of Biotechnology;citation_journal_abbrev=Journal of Biotechnology;citation_author=S. Joos;citation_author=T. M. Fink;citation_author=A. Rätsch;citation_author=P. Lichter;citation_title=Mapping and chromosome analysis: the potential of fluorescence in situ hybridization.;citation_pages=135-153;citation_volume=35;citation_year=1994;citation_issue=2-3;citation_pmid=7765054;citation_doi=10.1016/0168-1656(94)90032-9" />
<meta name="citation_reference" content="citation_journal_title=Chrom. Res.;citation_author=M. Karafiátová;citation_author=J. Bartoš;citation_author=D. Kopecký;citation_author=L. Ma;citation_author=K. Sato;citation_title=Mapping nonrecombining regions in barley using multicolor FISH.;citation_pages=739-751;citation_volume=21;citation_year=2013;citation_pmid=24026304" />
<meta name="citation_reference" content="citation_journal_title=Biotechnic &amp; histochemistry :  official publication of the Biological Stain Commission;citation_journal_abbrev=Biotech Histochem;citation_author=A. Kato;citation_author=P. S. Albert;citation_author=J. M. Vega;citation_author=J. A. Birchler;citation_title=Sensitive fluorescence in situ hybridization signal detection in maize using directly labeled probes produced by high concentration DNA polymerase nick translation.;citation_pages=71-78;citation_volume=81;citation_year=2006;citation_issue=2-3;citation_pmid=16908431;citation_doi=10.1080/10520290600643677" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=G. S. Khush;citation_author=C. M. Rick;citation_title=Meiosis in hybrids between Lycopersicon esculentum and Solanum pennellii.;citation_pages=167-183;citation_volume=33;citation_year=1963;citation_doi=10.1007/BF01725760" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D.-H. Koo;citation_author=S. H. Jo;citation_author=J.-W. Bang;citation_author=H.-M. Park;citation_author=S. Lee;citation_title=Integration of cytogenetic and genetic linkage maps unveils the physical architecture of tomato chromosome.;citation_volume=179;citation_year=2008;citation_pmid=18622034;citation_doi=10.1534/genetics.108.089532" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=G. L. Koumbaris;citation_author=H. W. Bass;citation_title=A new single-locus cytogenetic mapping system for maize (Zea mays L.): overcoming FISH detection limits with marker-selected sorghum (S. propinquum L.) BAC clones.;citation_pages=647-659;citation_volume=35;citation_year=2003;citation_issue=5;citation_pmid=12940957;citation_doi=10.1046/j.1365-313X.2003.01829.x" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=O. Kulikova;citation_author=G. Gualtieri;citation_author=R. Geurts;citation_author=D.-J. Kim;citation_author=D. Cook;citation_title=Integration of the FISH pachytene and genetic maps of Medicago truncatula.;citation_pages=49-58;citation_volume=27;citation_year=2001;citation_issue=1;citation_pmid=11489182;citation_doi=10.1046/j.1365-313x.2001.01057.x" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=R. Li;citation_author=Y. Li;citation_author=K. Kristiansen;citation_author=J. Wang;citation_title=SOAP: short oligonucelotide alignment program.;citation_volume=24;citation_year=2008;citation_pmid=18227114;citation_doi=10.1093/bioinformatics/btn025" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=S. A. Peters;citation_author=E. Datema;citation_author=D. Szinay;citation_author=M. J. van Staveren;citation_author=E. G. W. M. Schijlen;citation_title=Solanum lycopersicum cv. Heinz 1706 chromosome 6: distribution and abundance of genes and retrotransposable elements.;citation_pages=857-869;citation_volume=58;citation_year=2009;citation_issue=5;citation_pmid=19207213" />
<meta name="citation_reference" content="citation_journal_title=Genome;citation_author=D. G. Peterson;citation_author=W. R. Pearson;citation_author=S. M. Stack;citation_title=Characterization of the tomato (Lycopersicon esculentum) genome using in vitro and in situ DNA reassociation.;citation_pages=346-356;citation_volume=41;citation_year=1998" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=D. G. Peterson;citation_author=H. J. Price;citation_author=J. S. Johnston;citation_author=S. M. Stack;citation_title=DNA content of heterochromatin and euchromatin in tomato (Lycopersicon esculentum) pachytene chromosomes.;citation_pages=77-82;citation_volume=39;citation_year=1996;citation_issue=1;citation_pmid=18469880" />
<meta name="citation_reference" content="citation_journal_title=Genetica;citation_author=M. S. Ramanna;citation_author=R. Prakken;citation_title=Structure and homology between pachytene and somatic metaphase chromosomes of the tomato.;citation_pages=115-133;citation_volume=38;citation_year=1967;citation_doi=10.1007/BF01507452" />
<meta name="citation_reference" content="citation_journal_title=Adv. Genet.;citation_author=C. M. Rick;citation_author=L. Butler;citation_title=Cytogenetics of the tomato.;citation_pages=267-382;citation_volume=8;citation_year=1956;citation_doi=10.1016/S0065-2660(08)60504-0" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. M. Rick;citation_title=CONTROLLED INTROGRESSION OF CHROMOSOMES OF SOLANUM PENNELLII  INTO LYCOPERSICON ESCULENTUM: SEGREGATION AND RECOMBINATION;citation_pages=753-768;citation_volume=62;citation_year=1969;citation_issue=4;citation_pmid=17248457" />
<meta name="citation_reference" content="citation_journal_title=Biol. Zbl.;citation_author=C. M. Rick;citation_title=Further studies on segregation and recombination in backcross derivatives of a tomato species hybrid.;citation_pages=209-229;citation_volume=91;citation_year=1972" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=C. M. Rick;citation_title=Tomato paste: a concentrated review of genetic highlights from the beginnings to the advent of molecular genetics.;citation_pages=1-5;citation_volume=128;citation_year=1991;citation_issue=1;citation_pmid=2060773" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=M. T. Sadder;citation_author=N. Ponelies;citation_author=U. Born;citation_author=G. Weber;citation_title=Physical localization of single-copy sequences on pachytene chromosomes in maize (Zea mays L.) by chromosome in situ suppression hybridization.;citation_pages=1081-1083;citation_volume=43;citation_year=2000;citation_issue=6;citation_pmid=11195341" />
<meta name="citation_reference" content="citation_journal_title=Methods Enzymol.;citation_author=I. Schubert;citation_author=P. F. Fransz;citation_author=J. Fuchs;citation_author=H. de Jong;citation_title=Chromosome painting in plants.;citation_pages=57-69;citation_volume=23;citation_year=2001" />
<meta name="citation_reference" content="citation_title=Practical in situ hybridization;citation_author=T. Schwarzacher;citation_author=P. Heslop-Harrison;citation_year=2000" />
<meta name="citation_reference" content="citation_journal_title=Genome;citation_author=J. D. Sherman;citation_author=S. M. Stack;citation_title=Two-dimensional spreads of synaptonemal complexes from solanaceous plants. V. Tomato (Lycopersicon esculentum) karyotype and idiogram.;citation_pages=354-359;citation_volume=35;citation_year=1992" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. D. Sherman;citation_author=S. M. Stack;citation_title=Two-dimensional spreads of synaptonemal complexes from solanaceous plants. VI. High-resolution recombination nodule map for tomato (Lycopersicon esculentum).;citation_pages=683-708;citation_volume=141;citation_year=1995;citation_issue=2;citation_pmid=8647403" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=K. Shirasawa;citation_author=E. Asamizu;citation_author=H. Fukuoka;citation_author=A. Ohyama;citation_author=S. Sato;citation_title=An interspecific linkage map of SSR and intronic polymorphism markers in tomato.;citation_pages=731-739;citation_volume=121;citation_year=2010;citation_issue=4;citation_pmid=20431859;citation_doi=10.1007/s00122-010-1344-3" />
<meta name="citation_reference" content="citation_journal_title=Chrom. Res.;citation_author=S. M. Stack;citation_author=L. K. Anderson;citation_title=Crossing over as assessed by late recombination nodules is related to the pattern of synapsis and distribution of early recombination nodules in maize.;citation_pages=329-345;citation_volume=10;citation_year=2002;citation_pmid=12199146" />
<meta name="citation_reference" content="citation_journal_title=Cytogenetic and genome research;citation_journal_abbrev=Cytogenet Genome Res;citation_author=S. M. Stack;citation_author=S. M. Royer;citation_author=L. A. Shearer;citation_author=S.-B. Chang;citation_author=J. J. Giovannoni;citation_title=Role of fluorescence in situ hybridization in sequencing the tomato genome.;citation_pages=339-350;citation_volume=124;citation_year=2009;citation_issue=3-4;citation_pmid=19556785;citation_doi=10.1159/000218137" />
<meta name="citation_reference" content="citation_journal_title=Cytogenetic and genome research;citation_journal_abbrev=Cytogenet Genome Res;citation_author=D. Szinay;citation_author=Y. Bai;citation_author=R. Visser;citation_author=H. de Jong;citation_title=FISH applications for genomics and plant breeding strategies in tomato and other solanaceous crops.;citation_pages=199-210;citation_volume=129;citation_year=2010;citation_issue=1-3;citation_pmid=20628252" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=D. Szinay;citation_author=S.-B. Chang;citation_author=L. Khrustaleva;citation_author=S. Peters;citation_author=E. Schijlen;citation_title=High-resolution chromosome mapping of BACs using multi-colour FISH and pooled-BAC FISH as a backbone for sequencing tomato chromosome 6.;citation_pages=627-637;citation_volume=56;citation_year=2008;citation_issue=4;citation_pmid=18643986;citation_doi=10.1111/j.1365-313X.2008.03626.x" />
<meta name="citation_reference" content="citation_journal_title=New Phytologist;citation_journal_abbrev=New Phytologist;citation_author=D. Szinay;citation_author=E. Wijnker;citation_author=R. van den Berg;citation_author=R. G. F. Visser;citation_author=H. de Jong;citation_title=Chromosome evolution in Solanum traced by cross-species BAC-FISH.;citation_pages=688-698;citation_volume=195;citation_year=2012;citation_issue=3;citation_pmid=22686400;citation_doi=10.1111/j.1469-8137.2012.04195.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. D. Tanksley;citation_author=M. W. Ganal;citation_author=M. C. Prince;citation_author=M. C. de Vicente;citation_author=M. W. Bonierbale;citation_title=High density molecular linkage maps of the tomato and potato genomes.;citation_pages=1141-1160;citation_volume=132;citation_year=1992;citation_issue=4;citation_pmid=1360934" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_title=A physical, genetic and functional sequence assembly of the barley genome.;citation_pages=711-716;citation_volume=491;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=The tomato genome sequence provides insights into fleshy fruit evolution.;citation_pages=635-641;citation_volume=485;citation_year=2012;citation_issue=7400;citation_pmid=22660326;citation_doi=10.1038/nature11119" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=R. Van der Hoeven;citation_author=C. Ronning;citation_author=J. Giovannoni;citation_author=G. Martin;citation_author=S. Tanksley;citation_title=Deductions about the Number, Organization, and Evolution of Genes in the Tomato Genome Based on Analysis of a Large Expressed Sequence Tag Collection and Selective Genomic Sequencing;citation_pages=1441-1456;citation_volume=14;citation_year=2002;citation_issue=7;citation_pmid=12119366;citation_doi=10.1105/tpc.010478" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=J. van Oeveren;citation_author=M. de Ruiter;citation_author=T. Jesse;citation_author=H. van der Poel;citation_author=J. Tang;citation_title=Sequence-based physical mapping of complex genomes by whole genome profiling.;citation_volume=21;citation_year=2011;citation_pmid=21324881;citation_doi=10.1101/gr.112094.110" />
<meta name="citation_reference" content="citation_journal_title=Chrom. Res.;citation_author=C. M. Wai;citation_author=P. H. Moore;citation_author=R. E. Paull;citation_author=R. Ming;citation_author=Q. Yu;citation_title=An integrated cytogenetic and physical map reveals unevenly distributed recombination spots along the papaya sex chromosomes.;citation_pages=753-767;citation_volume=20;citation_year=2012;citation_pmid=23007683" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=C.-J. R. Wang;citation_author=L. Harper;citation_author=W. Z. Cande;citation_title=High-resolution single-copy gene fluorescence in situ hybridization and its use in the construction of a cytogenetic map of maize chromosome 9.;citation_volume=18;citation_year=2006;citation_pmid=16461583;citation_doi=10.1105/tpc.105.037838" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=Y. Wang;citation_author=X. Tang;citation_author=Z. Cheng;citation_author=L. Mueller;citation_author=J. Giovannoni;citation_title=Euchromatin and pericentromeric heterochromatin: Comparative composition in the tomato genome.;citation_volume=172;citation_year=2006;citation_pmid=16489216;citation_doi=10.1534/genetics.106.055772" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=L. Yang;citation_author=D.-H. Koo;citation_author=Y. Li;citation_author=X. Zhang;citation_author=F. Luan;citation_title=Chromosome rearrangements during domestication of cucumber as revealed by high-density genetic mapping and draft genome assembly.;citation_pages=895-906;citation_volume=71;citation_year=2012;citation_issue=6;citation_pmid=22487099;citation_doi=10.1111/j.1365-313X.2012.05017.x" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=N. D. Young;citation_author=F. Debellé;citation_author=G. E. D. Oldroyd, R. Geurts, S. B. Cannon;citation_title=The Medicago genome provides insight into the evolution of rhizobial symbioses.;citation_pages=520-524;citation_volume=480;citation_year=2011;citation_issue=7378;citation_pmid=22089132" />
<meta name="citation_reference" content="citation_journal_title=Mol. Gen. Genet.;citation_author=D. Zamir;citation_author=S. D. Tanksley;citation_title=Tomato genome is comprised largely of fast-evolving, low copy number sequences.;citation_pages=254-261;citation_volume=213;citation_year=1988;citation_doi=10.1007/BF00339589" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=H. Zhang;citation_author=A. Koblízková;citation_author=K. Wang;citation_author=Z. Gong;citation_author=L. Oliveira;citation_title=Boom-bust turnovers of megabase-sized centromeric DNA in Solanum species: Rapid evolution of DNA sequences associated with centromeres;citation_year=2014;citation_doi=10.1105/tpc.114.123877." />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_author=H. B. Zhang;citation_author=X. Zhao;citation_author=X. Ding;citation_author=A. H. Paterson;citation_author=R. A. Wing;citation_title=Preparation of megabase-sized DNA from plant nuclei.;citation_pages=175-184;citation_volume=7;citation_year=1994" />
<meta name="citation_reference" content="citation_journal_title=Plant Mol. Biol. Rep.;citation_author=X. Zhong;citation_author=P. F. Fransz;citation_author=J. Wennekes-van Eden;citation_author=P. Zabel;citation_author=A. van Kammen;citation_title=High-resolution mapping on pachytene chromosomes and extended DNA fibres by fluorescence in-situ hybridization.;citation_pages=232-242;citation_volume=14;citation_year=1996;citation_doi=10.1007/BF02671658" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=X.-B. Zhong;citation_author=P. M. Lizardi;citation_author=X. H. Huang;citation_author=P. Bray-Ward;citation_author=D. C. Ward;citation_title=Visualization of oligonucleotide probes and point mutations in interphase nuclei and DNA fibers using rolling circle DNA amplification;citation_pages=3940-3945;citation_volume=98;citation_year=2001;citation_issue=7;citation_pmid=11274414;citation_doi=10.1073/pnas.061026198" />
<meta name="citation_reference" content="citation_journal_title=Molecular and biochemical parasitology;citation_journal_abbrev=Mol Biochem Parasitol;citation_author=S. Zhou;citation_author=A. Kile;citation_author=E. Kvikstad;citation_author=M. Bechner;citation_author=J. Severin;citation_title=Shotgun optical mapping of the entire Leishmania major Friedlin genome.;citation_pages=97-106;citation_volume=138;citation_year=2004;citation_issue=1;citation_pmid=15500921;citation_doi=10.1016/j.molbiopara.2004.08.002" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=S. Zhou;citation_author=M. C. Bechner;citation_author=M. Place;citation_author=C. P. Churas;citation_author=L. Pape;citation_title=Validation of rice genome sequence by optical mapping.;citation_pages=278-278;citation_volume=8;citation_year=2007;citation_pmid=17697381;citation_doi=10.1186/1471-2164-8-278" />
<meta name="citation_reference" content="citation_title=New Methods for DNA Sequencing;citation_title=A single molecule system for whole genome analysis;citation_author=S. Zhou;citation_author=J. Herschleb;citation_author=D. C. Schwartz;citation_pages=269-304;citation_year=2007" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=S. Zhou;citation_author=F. Wei;citation_author=J. Nguyen;citation_author=M. Bechner;citation_author=K. Potamousis;citation_title=Single molecule scaffold for the maize genome.;citation_pages=1-14;citation_volume=5;citation_year=2009;citation_doi=10.1371/journal.pgen.1000521" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Fluorescence In Situ Hybridization and Optical Mapping to Correct Scaffold Arrangement in the Tomato Genome" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8.cover-source.jpg" />
<meta name="twitter:description" content="The order and orientation (arrangement) of all 91 sequenced scaffolds in the 12 pseudomolecules of the recently published tomato ( Solanum lycopersicum , 2 n = 2 x = 24) genome sequence were positioned based on marker order in a high-density linkage map. Here, we report the arrangement of these scaffolds determined by two independent physical methods, bacterial artificial chromosome–fluorescence in situ hybridization (BAC-FISH) and optical mapping. By localizing BACs at the ends of scaffolds to spreads of tomato synaptonemal complexes (pachytene chromosomes), we showed that 45 scaffolds, representing one-third of the tomato genome, were arranged differently than predicted by the linkage map. These scaffolds occur mostly in pericentric heterochromatin where 77% of the tomato genome is located and where linkage mapping is less accurate due to reduced crossing over. Although useful for only part of the genome, optical mapping results were in complete agreement with scaffold arrangement by FISH but often disagreed with scaffold arrangement based on the linkage map. The scaffold arrangement based on FISH and optical mapping changes the positions of hundreds of markers in the linkage map, especially in heterochromatin. These results suggest that similar errors exist in pseudomolecules from other large genomes that have been assembled using only linkage maps to predict scaffold arrangement, and these errors can be corrected using FISH and/or optical mapping. Of note, BAC-FISH also permits estimates of the sizes of gaps between scaffolds, and unanchored BACs are often visualized by FISH in gaps between scaffolds and thus represent starting points for filling these gaps." />
<meta name="og-title" property="og:title" content="Fluorescence In Situ Hybridization and Optical Mapping to Correct Scaffold Arrangement in the Tomato Genome" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/8/1395" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="The order and orientation (arrangement) of all 91 sequenced scaffolds in the 12 pseudomolecules of the recently published tomato ( Solanum lycopersicum , 2 n = 2 x = 24) genome sequence were positioned based on marker order in a high-density linkage map. Here, we report the arrangement of these scaffolds determined by two independent physical methods, bacterial artificial chromosome–fluorescence in situ hybridization (BAC-FISH) and optical mapping. By localizing BACs at the ends of scaffolds to spreads of tomato synaptonemal complexes (pachytene chromosomes), we showed that 45 scaffolds, representing one-third of the tomato genome, were arranged differently than predicted by the linkage map. These scaffolds occur mostly in pericentric heterochromatin where 77% of the tomato genome is located and where linkage mapping is less accurate due to reduced crossing over. Although useful for only part of the genome, optical mapping results were in complete agreement with scaffold arrangement by FISH but often disagreed with scaffold arrangement based on the linkage map. The scaffold arrangement based on FISH and optical mapping changes the positions of hundreds of markers in the linkage map, especially in heterochromatin. These results suggest that similar errors exist in pseudomolecules from other large genomes that have been assembled using only linkage maps to predict scaffold arrangement, and these errors can be corrected using FISH and/or optical mapping. Of note, BAC-FISH also permits estimates of the sizes of gaps between scaffolds, and unanchored BACs are often visualized by FISH in gaps between scaffolds and thus represent starting points for filling these gaps." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8.cover-source.jpg" />
    <title>Fluorescence In Situ Hybridization and Optical Mapping to Correct Scaffold Arrangement in the Tomato Genome | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"S9NT18nr-kfH86lvo5wli87pDn57GYinTHn_950rSc4","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459935","apath":"\/ggg\/4\/8\/1395.atom","pisa":"ggg;4\/8\/1395","ac":{"\/ggg\/4\/8\/1395.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/8\/1395.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/8\/1395"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/8\/1395"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/8\/1395":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00220e9Oq5HPkXKksWYeEz+KipOIXgkHLgW9sFZOhMGPm2wtnNg+mzz5EYaZuZxU7xWplCUBascG2a9fhd2JrIiXWu+2EA5s55H11tcdWxFrih9IWJmwU8HnKcDzZsCWMn4hmDG6J1pGcxf1vbwKV\\\\\\\/aCoOQeoeRfeCEIEsvTOQUYBWqUNpgTXLeu+AyGUfAAeTmuJ2X2kyMzoPrPNDta8ghd\\\\\\\/8Ffah2h09RxwSBKPoRPOV3yceFoK95RCF57CLzLKs6B4ZnJzOxWDHGNxp0Jfm\\\\\\\/tRpQHSSmjthqHIxKtq72023Or1ik\\\\\\\/fAjRrS0TbEBeGejb80X9rweHOf497q\\\\\\\/6mv6wsgBAS7icMfeVM2ahbPEdvAkg2vX796I\\\\\\\/6BdIG2sYbXgcTghfylwfqWZwLej9H8\\\\\\\/NLTWKhAb6erCAFwJh1KvoGlWfU7+4ziVqwSHNKQljfNgxXAw0FnZcdP3ITK4ZzeqmzqK0UNTijbRUTwBJgtPovNZ5xqGez\\\\\\\/HCvr6tzuu1mj1ljf29j8L0XWSshvM8OPNbtE8SYmahDZK+c4DZng1L4+LDAXlxRNouL4NlnqnP1VdzQk0sZklRWEGfu7Sy1d\\\\\\\/s6iI9+huqEhe0omBJi+HpidyeNbChru0ldk8p6CEmDCu\\\\\\\/mS0GdpCaJlifIu2OythR6+v6zloc6tQyoUANIFLTwVZYovQQhQIfVjE9nkdTGjoGpEuBDC2TIs8Sj7\\\\\\\/1Jpn35VnKr5yO03d3KsX+tFIvwJt66JhG3KnVUIMGE3JhhkJ\\\\\\\/cvi9Iy3cBpvJkL1geTSk\\\\\\\/MzzU4aDBIs3JkLvVvGJOZ2ltrjeij+uSaxiRPmQo8ZT8D2M3gqxf+T5TawAMDa1tunQALCrgTuv8f4oAn1kVSlwyF8Zb1SosEAMRiYj8QZf7JaF96c2rS9EMIXjVwXOQbIoSIMonGA+ykgRuJzxniXhtNNyvbhvlc45BZIHkWMXPi1gC8iXzZNhwS64FMOHG9YvI4ImdGRzHEsDX2FmGnS23qxcqDCT+8+7wAdOs1XMj5gl5mbNpnvw8DfK8WQ9kH37VlKvnMyJDA4iYFZ9BFPIcORPJp9zGoj\\\\\\\/6+ndCn7azvMmqvxHR20kCa15wIUkyummBhwxmSj4Vg0HGPBbQ4tkSnYwoOTmLMh7yAKy4qwKOoKqXOPrySyRS5\\\\\\\/pGKR3+zKOUr+yxnaSbBrBa902t1JxfrKZ1rOLve0GMjLkD2jIjI3XMURTelilBj7PJwPnL0eOtDmsxLO717xpVpREFxRe0VUKZrgwaxyRGLxtD08s\\\u0022,\\\u0022iv\\\u0022:\\\u0022USdzzEutraTfLfQ1kh+nQg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022718688055816ddba0bca3792dfe5de94091afac7ac910dcf607ca4fec596da3f\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022EIiPlH8rK+i8329CKcNM48iFNnjs1Pc3p0bei13lfX6rrED9uZoObwvW0YA8meWylJTJUwVKnd9bX9HTIAkU2GySZLAadB2PW3S1nrMNjrJuRMI06yokKKeN6mV4kjnD5GaVT9YUE4ycSCF9mWzx8d\\\\\\\/jKbw7muzBXujUL2p4IbZAbnp+j3MU7H\\\\\\\/MhNbUFtWXbD6iYsTbDHOz9KF4LEatxrbCCpKjRfqZngL19ca11WlC9yz5JLaecBGQKozbsAnLVHWNz4mcviez1EXLqvKP4Vql8ZnL2POiqkinjKCAy7MNPPXkskvYIQIjK6xl3djzhILj4JcXjdqmEHvjMN0qLGsJ9zKd6Dcznb4rH\\\\\\\/qlY0HzdHszUrEgIbH\\\\\\\/UDC6ANZYPfYjyY3yYR1HRJ5X0udGLqbuvaAe6amFuAFTvy+WZrmwIU0jzK8ObnZEJKlr6C4Z3uJhSGa\\\\\\\/S3LCY7uwknXSMPsJW7SqlWDHDgq+2+ztzWhReGY+dAXzAxU8rgXBL0rLtIHgFc8wWFn1am+uDnoeX4eK90EsMXyzVo8PYz4SGGSX8sVC8HYckk78VYt0Xk5fUK5Z4Nwk2xaOPM4EnKzmth1HTPcf4oYWQMaW8TXh6cw6A6xD8WQPxpzhvgPsTVmC+iW1c0apFHTqxY1eJapercaQgL89LKhujLiDrUqygHtDRVVC79g1lLJjgOC48r+Nv8CbrKq8zClkyZWoPq3WqsgyifhCd7OL8O6OI\\\\\\\/f9k346BHH1G2RSiEt0QouNa8DvP5mT7zz1xt3Je8FmH51ejCsgeLUe41L6\\\\\\\/\\\\\\\/daEThlzK928xV7FxaaCjHxhVeSs4i5OpeWpdLhyd9fc+DQBFVDuDopajdqZei50VTamM8JQZfU\\\\\\\/I2VTQNJz8Ju5PLUmEljMePrcbRalNyCfQ\\\\\\\/jZRLguZzl3df2H659\\\\\\\/5AU\\\\\\\/77h54ufm84M\\\\\\\/s\\\\\\\/v6SWNX95yBlorov53rUdkrXdmLHQ9+sJ0zFMAYZ\\\\\\\/nzasDP3S6ZRfyA4m5t009R2KuVGseK9dT37BhsBS2O1+2X2r7aklV1SaCDsKANTJWtLuLQQ02EvWkBU8QBCJO8la3a5xRMZWc9Wqx3WVnRvhdoB\\\\\\\/VdqCTqnMq1Vs6z94ySwQkwAcG+gG32+PGjZga\\\\\\\/e4EyXUYfR4UCpdFGOSKTork9MvdSGD9IfxVpLIrL6hUyPp3NqTD71zoOAkwCsnpxQcIJQKCQfNnChGtb8IC9BtFVkONdGbYBNpriJv909qPCVRCvhFahPRQ9VSELnCeprgMfcL3UYanhyJKPmPYjIaoflFE2qxIW0\\\\\\\/NR5nOmGKNnIBczsXn9FaLR\\\\\\\/C5fTET2p4ED4CDSAjk2IuxLye3jFnz00pqqBbWBgQSott9QxAqvc7FW\\\\\\\/RW5VF42BLmQ0SBLwTn\\\\\\\/Y2E1WoNcXmdJ7Syw9rnXcA9PnH63V7o4tFlEQJ\\\\\\\/+Bf8kgCLHWC2Dui5KqfkMDHuM+8cNpWOyzYVFuFKcbUMPQqF0IX9XsXWJ58E9GQL+gjuxZJVyU07vqZ7QmdYBhtw5y1z6bHOgluLN6cx+nJYIJvvutj4cTSuV+Gzx3bNhq1jDShL7kLKbhqJL\\\\\\\/MfWH7\\\\\\\/zNRQNROE+PRc68mQd01XmGR0kr33WggTQHvRyumRlneGkN5HMrk5LzC\\\\\\\/VF3+WP\\\\\\\/oDO2\\\\\\\/2CjYFcGmanJlVVuxUOXly25Lf4n8EBE239U+NQ\\\\\\\/ewzhNApYL+mA\\\\\\\/Q1z\\\\\\\/dxRZJB76joyMcQfAId6GZUeMbl135HUWOVxbtNnZWGiuiHlL3dkp3fdKe06BMQMyepqXCThXMAxUKzZNLYBUSUx1+LTBBSpAmZm\\\\\\\/DyjXM70McSf\\\\\\\/HeM6+s6lGG\\\\\\\/8rjcimAtmVAQK6sYimB+3Wbkt8BdfKiM+nBLIk8Uec7S13Hh8bUa1A3PDar4qfCa5MKL1uNUmjUe6wRm8Y\\\\\\\/udRsdI6gMBGET\\\\\\\/Ba\\\\\\\/o7EZLQDZhyjN5Tfw2Lsxvo\\\\\\\/TLRYZfozVkge+3zJDvpSQHgj0lLsMlrmhwJIQkptdxMga4InVDwLJocfqHFmcUTlzj7dKj0bnuP5xdIq9zcmVMPyxIf2hJWoNIlC64T5LK5cJc5DQKSrvUdQATfPB6hae5KyWB4M6O\\\\\\\/+tY\\\\\\\/1Q2AqRgfF4D3nYkoDm5fgYNuFSITKccVIy9hnTnVLNy\\\\\\\/keEb8BuSMFuZbIaFNUF6EjiKgMWxJQfwKTT9mGFZvRv3Z+ZMVeNeqm8Nn9rgFfyiF8SqMfEEXgr1Td5KsX\\\\\\\/u+qP6JmEZWd6cTtbQDordh7XSEJjYqxqHlwSt2aCnSQLj090zarMWywvOLWQIbABviTVdKpq\\\\\\\/gwDF7Shji\\\\\\\/nWAa9K9kQlHH9p1hvv4oPweypM4BaeSr7+pmCpJ+\\\\\\\/Enj280RlyoPJCoqn03Qwxbao7zJNHeVs6eTRn3gyhuO0f1lvd3\\\\\\\/q3ehHro22ib1F2AlV5bikiO9U3oVn7kQnNALB5w+r\\\\\\\/xeo35iGI\\\\\\\/3NCxYnGxm4a9pnPLf8VRsc2g2c35HqUb+eePy7VjyWO9o27QcduclapsY+8f6NFms4USkJW3nSU1uQav7g2R0niUF4y4SxZ9Q82lBVd2e87hNkRYZtYupjoj52bxSgzjz8RYyzdggghgqTPa+iRx9Tioq2sWS5FvCQ4Kx9DTrYblKZvp46Ly0TBlNXGCGVxSdP1wlo7l9+SzOgYGL75GrEuqV1etzQ2479zJkn0b0sQqY0Z5+4bDqEOrqQa7bhwS\\\\\\\/ZFcc5DE1KJqDXEjRpu+DXIAvmDuWGRZmabegEwuvAlOEV8YgZmKgcki9A1PbsCanc3uEUWz+\\\\\\\/BlU5emWiMeopyC5w+8\\\\\\\/GMzn+\\\\\\\/qNNuc26bzKx36oEhcuMNw4UdMT7Jo4y5N+UL6NX\\\\\\\/1s4Xk+lU3xbdipLI9n6mDqKs9uTc8Ie6EvjgodNPC8uNE0JVP+Q8Wdfr8og2TDmCUHxG+e9qDTYdj4lICIEwAGu0razPYaWPErZ\\\\\\\/saP1Z\\\\\\\/+SIGUJueJfh0jpX1bRjHLVuA7yl2JZpyRPliyD185jRsKeyiYT277JGM7GbcWBJcweSpAsoucvInDL8xyBRoss8qh5DR7S56y8nHdoprnMYx6GxUtYRPmzz1tA7uyY2tyMSsP6exVRObfm4d7TB0bJplyvAZTioKJjpfA+tsJ81QNj2BTwZ4vjieyPYDu2s2KlOcv66pP0jSb74bwl0aYpFdkIFG+qpFsz4CTgVQ6DoqP0\\\\\\\/mrZ0r3VUjxM1vxZWvAcq3puhPteGJnP0iCBJoz5\\\\\\\/4IhdIAeX0OV1tvuEw4O1RLbE7kgNR2kxmUnKFonvuFVoLYHEXZnDd\\\\\\\/RSkcbSD3vQ1YxhZSzMyTmxrVO\\\\\\\/gvp7K97XmLphj3x3LbKHSGEIrRN6LAfGx2f8uUJOrVOk0Pbb1yC1GttZBfryDiQZMlDMg28b0ADB5x0rOIMrUjwD51K13m8K9pCZjtE5\\\\\\\/sb\\\\\\\/RXj9FRl65cMfmz9k8Jzgb2qd5T801R3iPgn2hcRl1q6NG6KaOQZkuVKy6tCuU2guNUlbtAxR25QoMT3Wh13ECurPsy2oYQUYy0mKKXi\\\\\\\/lf\\\\\\\/5fLm3zswGY\\\\\\\/m+78dm8mYb\\\\\\\/6KiBdpM4S1fNy5a74c6lgmsSOQ3XhFodwekmm+h1Mo1fL5dovRtQCW5KAzkODe7yjcS6oKtOyefu2JIBVy82Fa+y8sBBeWBX2E0dc6pSyg6gwhLcn3nwllwH8dTBLBtigY36\\\\\\\/jpMQeYKNYR+Qcp12wIwugC7aSa09HL4\\\\\\\/BPP2bUd8C9Ey0oQ5WRsE81CPzgKfdrfEjdl28Y9TxiR6p079JZQf5O7RvEQdToa5zQ+YNLo58X\\\\\\\/I6b+fupehDNn0gwbGjcWrfMsHtRlari1LtztGvTsHrLsBS\\\\\\\/9p7p7q1f911LOzEXgQbswNfCzY\\\\\\\/WBT8FFPM7kmZxMbkowXLinKHsiTOtzISgT3HH7F4+y8IekH53auuMPoOFN7tnHFui6XLXGM\\\\\\\/70tCndeVu+bgjl9P\\\\\\\/zeGDStkb5PK0pE\\\\\\\/n88Bfl6NZ8jaWQ5DvI0+lg4ImkPI2Ir4NOftfGdX2fmv\\\\\\\/DKURcKa+NECvqnMSCLqOXHY45VtoM0OWzk3WFk8c5a6\\\\\\\/e5qtrp9XDh18Ta0060epynCBTjYfnwdhYec09b\\\\\\\/5TP1ec\\\\\\\/cmZbgV5BFmVwlj70iF9z3\\\\\\\/lvEPPHZI0mCIqdK7aITXWGUjrcueHyIxdjQ2oT33+QbLvaSMfJ3kktC97wTbF452iQHvmmkUOT\\\\\\\/qP6NmuLn7ApXGsdRRCK3lZK+CZ\\\\\\\/pdh1pwW3Opie2AHx34oWzBZaKFojJHjrFLZ5n4OoKEsWcA03wZcKl8n9deKHUlbCUDffXZVBbh5\\\\\\\/vFgLCCxdiNsDw\\\\\\\/IRFQ9E2l+htddEAkZdfUK\\\\\\\/Fkoou0QqrsjM0qhPnL3VnKqsINluielzlxJwH31BrrXndxn2YHs2CEcVa2VY+dl7mT1HrRzYLXCL2z6F7WYgiRomWlAZ7t8sLirFXt5Cq9BKs6sYON8j6BtfGEx6Wfum+Ixvx1G8Ti1tTphZE\\\\\\\/bOjqlXtIMH\\\\\\\/Z+XeKM6\\\\\\\/yxfpt3BxS\\\\\\\/o1z\\\\\\\/Ex0OmjQb7y3m1G\\\\\\\/jOJzLNlqte3c7vMyg+uNz0uhpYC9iwfTS1daPwiDen0f+8YOv8\\\\\\\/NG1e9EK4uE6+e2rLXX1OR23hhB78kH6f\\\\\\\/ozFWufex\\\\\\\/nUJD+16BJb6YNGcRuU\\\\\\\/j6OWl6zFVwQEj6uhQnqysBMrps8UboRNWEMaLQS\\\\\\\/93h\\\\\\\/zO6+J39zua32dPAk6UT7kBrSrYIVFeqNJggJ9RUBNJuazruhQcjyqGlOEM3OLUCOPuWp5mE219E6rUad7e9PzBIlfR0iKKdrONjg8DYM6kq8+bZ7UrArrrodX7rmGUEZiWBOBtnq48rFM5kAKDAWfHEDYm7RMA1dSsEoDXw5FxPlYiEjMhtcRkkITPhFj5f1eDyoXLoctqP7tlvNX6hb0C7Hd9XOBvfpYAhGo83bsqTiyg46N5uOGN6cSGvwk15FDgEIPcaEUo8MRgr65MQyYVzyk5cQkI9eUAe4\\\\\\\/XMiQUFcSS+eBebw56HxwMWhF\\\\\\\/2GYyvCdhIYyP26KgfG6Pk11zIR+pC1BFzneL892VqVDMvcon91aXRGRs60Wv8rMRc6f9+mHMr42HoFHU50\\\\\\\/sp2uoHJaq9RDUPzVuF8vFM7LClUQO7pZCn3tJsw\\\\\\\/yOBpxtAzjzb\\\\\\\/M9\\\\\\\/YaqtgePwcutR8Iak\\\\\\\/twcHjzziVgRxoFrMU167ouEwpppOewpZh+O9AMQS0y2eH8sG2hEAjLSlQItrT8khdXxeREwQG0b\\\\\\\/uMVyNi9KYONT1fhjV+i+HKyxGek6WsxE\\\\\\\/O6J5ZK49QMHgMfUuA7vkaSoHQft5ClsSwqqCUDyfg1NHpiQlMRHXZc\\\\\\\/rw6pyliXGiaTB9sHqhPIGiLyyIZz\\\\\\\/pPDX39NCGvzolBJmxHk+RmvpdN7G4jej9NE4zqHMoYgUC2AsdbOiM9ZWxtT2EJd5bf7AftSRZ6lFE56prKMG+Ct9kZabb125M4Aeq5bXIAyiLQX6O0n+UgNpGy+hQFYMrUQprwzgWtLCEOX074u8JcI2GXWEONH1qauWIloUV6arAQYmcmCJmFTuji8vxXCY4mBc6ORPBz0p5dDD3X22gPBBaIL5Kb5bbg6Ro30tp0Cl3CgBvXI0d8wvKOr4We4HhfJWNeP57pvoeP3h2cfQ3VuW31ybIkM\\\\\\\/cZFRRz2Q7EYnGWxeOQ4yOUA39Lo6+qE7TDiSLTwdLtpKgRfyRoOKH\\\\\\\/3RdCgxvOYsUWak8etAgb5fMLjcd6zqqC\\\\\\\/nE0a7pL3U2NsRfIpZsPEGah2SjE\\\\\\\/Y+QR5PBkDAuyCadTz5B3K9c3uJIbrwROri4Jbv0s1SWu1AJW8EcZC9wVceB1pzlX2VMrT95RNZEA+U5kyRvfPsM21LsscPeVPsmRLP5FIqso7jC5d3INL7IMn\\\\\\\/4cu50TxI1ylUhdEEqSCSunccvXGPEEEDnX7gIpbH+b1kVhNqchk5Esft5+4zv1nRQ+dVVjcjC9z+cmKIIKH9zwBvifp0KNHfD2lh4zcak1emjl0qmuOHcEEzHPx1kvxa40liEqphUOWB1EKqN5hodZr04\\\\\\\/u\\\\\\\/dZPsLkyxQXArrcMBzQT63vlEhYzk2+SYAkBQpkp+EWRUXqfCG+7byxp6voobiTeejmHzWFDW0sq4bvl0aUqrcgfLfxtBHztyQJZDbxR7pDIHBCf+NTn1TsBdX4G6w7zYafxvuIhfHEOzxOtyc0CXpMGdewPX\\\\\\\/N06Nqi5Jik+sOwtz6Nsnow7TCsSSANt48F9NWoNH4srjNfXlwYDlyoRXtihRPreTOWsYdaKEBBwC490fPtGSeS89BxGO5bQ3R1nUpEyGBPmRRwUCOvG9uzJz317TYaGwz5WNi4Q6xi1Y6fk2PX0U0gwbkHUm6lEgzftROBYSsUOJ8v5qK0Z4liwx5wCcB9M4McpnQQwZGZiYohHAlaqUTp45ln+iKcg5crIWoyqEzQfpkyhgRI4zbpGXZ2qk2A6cRA0l8e3DZzA2+OGLq028re+U\\\\\\\/Qr4CxbAVH1ceyHefN+ILL0K2nDEGgPw7\\\\\\\/flHX0Uq8HYEzLzUm5bqp8cGxQ+FQ7V5XmtvnIQTgMBP3QpsqPTsPmFD2BweoD8tczeECIzjJwkDvkLSZBT2NphCEIHyizCZeaMDZ6WZ38dfiAZuxhDD5N1bB5IWHeF\\\\\\\/ZWQDwZRPjydfaZcO40qDA\\\\\\\/uor4GTgUK2TAJU2W1jszqfvTNHHWer0CvFoyEYDU99wYiDSCTeGvcmfoKV2ervoJaQZJLvIJAJ8N4L7A9TAjI4cgPCM2p2+5mVQSEk4U7XkMTjHfzNU\\\\\\\/Wph\\\\\\\/Wkgw\\\\\\\/2ON8R5Q6\\\\\\\/ULuGa8\\\\\\\/hF+GwQ8UspQu7qrbSCrjgLcH66hE2ZozFqOVds5cGkdfIgHxWiZbxQQSPv1X09bvVfRqJb+pb86e44vJfcmqL+ChumiXPXH+0ut+Vb0Q+CzM9FeWPJ7ozmrtsfLiAoe7mGhpqX6pH8MLEdyrIWsLxTJkvMgG05ncwByPZRljIBxFqWKybofEcxZc9Xr9QGAeEUcOq0adjXVkI6+T7lRvPO4juTh+f75PkoregRPqPVFpByVspyh0c0lxX0V+dkkZ62TEDz2HgtRnke2yE8A46jfV+BJcurV06c5M4ooU1l38Ue+VrwDnlAurgDcYQPPvaB\\\\\\\/9WdKR3kuDBv87\\\\\\\/WO9RKuJvhpr+JrgFCXuuZh7jQS0ytSQTGtIWc2C5pV2FHV5D6PlLOKxme9lWwgT6p1DqG+Acnb70gYg+IzPSxp1dH5o2I+gI4Exp76QMCtRx47LFE8OTlLNGEKbfjigUB7dS4d1p+wBA5g7cMAzMfJtT4fu9fEi+Efx73PkfbhGBsja4OPgy15O8oboX12jQeY+bh7KG45NL+Ab0lDc9vOXWqemEO4NGo6o+9tigipJeiTHwE+NL0dkBAyKy0RchveCli\\\\\\\/7ADIm\\\\\\\/v0F6\\\\\\\/omJWCPJ7zYi7NIysJiiSGbLq2hcjX5FIg9w72jDkvZiECJnqCYfOzjXEmC05IdjpQSgSd03e9bzEgW\\\\\\\/YaqiT+z0wD3tytgTc0xR6jScVgtW\\\\\\\/JkX7CjsP4v1PS\\\\\\\/Dqx+cstj+76nOQ6d+ywP\\\\\\\/UYiWz6iO7w9xD\\\\\\\/KxfOXofXG+Fbl8ulkQfcUM5D2gRj9nv4CehGEZAMwgkyoLHzgKg\\\\\\\/ugnTyjTTmWMM9Od\\\\\\\/oUL1vM9YdezGDoyHnTpJG\\\\\\\/DcYX99vnUjgJN593jUuN90BspqF3U6D+UAYrEVFA9IfDJu5OQC1Gp6RhNpHGnWQTTiw0pYLYi+3EuX3aT+DNWhA1e9ekKCKvpYEbUXdwK6TT4fNGD13hjDpCjxqxMotyaC675epDm00TsitGvKUnibBpZINjVqo3IeZOpW9MLTeoDENgTdvwddHwCZC99TW2yZObwUweh3OMovSmCl6Ut4sExqj0GZmTcqK2aUSn7RkcGC7VYtA33cVSCMi0UIcD8ke5gV400s4wFYYZcV+ANa8zTZaGeOrkK2w\\\\\\\/2O87EMfS9rPr\\\\\\\/hRg\\\\\\\/d6mDZExzlmeIgGeZfGAfSXEHNw7SpSrIq91d88LN1FFlsLKoc\\\\\\\/H48fenCfQevp4BH7jF\\\\\\\/1mnVmLFmD6gFfb7a2rNwHn9kO6ii1ST4uvw6MV2pJpiX3yODn7Xo+UQz93jGTogjSe29VwN2vveL\\\\\\\/CTvL5auCXI6nFlxyAbXd76VWyebVWoSYqEvq+jOA41eU2Xd5koHeJgC3WhdhTGHj7Iq\\\\\\\/IsXxojWbqj79+QXRcWJHzk4fyh7bVaiIW1W6qt6RV7Ev\\\\\\\/wgfavUqDQUvO7PxYJTarkXN4GEPXAmwgDNqUOBdMWLGw9Uf4+bLmIDq4iJUpL7CwqyjhR0IUAbjtAvosqpxQI5FaR3GEQsS2rbV9lfiFbDphaGgps5Is+77vFP8LIivEYQYsz7fR6uxFpcNR\\\\\\\/oLwgD818HZteKcWzzieXib7hm7Sl+9HHM8UF6PFrt2rlMDcPuO12B0BB4p5G6UnBD8LR4D3BW54IpAyWPzwNKtVU07541c4RHWK41y\\\\\\\/YM0KwTR5Vm0rfH9TToDmtkZqdUx7lpoz8M0XOLo8FjlzGYbeH+EZbKknfw+4lRaEmPOcOysnF\\\\\\\/40nQANqzxPKCOAIcS1tMDCuZrRC9KjBPfroA0ev\\\\\\\/tzQvL2RTcM6Jw5bhk\\\\\\\/l6Ga5E61aYc8vDzG\\\\\\\/2wh+YC0i8d9yX3Qrz1NNmLHVq+5Ur0fxy3CEIZCCmCBLIAlhkvjzkxub\\\\\\\/G1lAQpMILTRA7Y13Sez29mgDdbfne4ot24QOpR936SdmKljrJL8elNhDFRj8JjSvpCG5ubNq9OOZo97E50OsK9sZsVkEc9HzJwYyct2SbsLbW6Yu8lyk8sWWr3Z+dn8tRKFuRn7YBhxXIVz4v68O2cjJ+n21Ai9KQ1Lj64OaNlS66a57jdRpvvdb\\\\\\\/XREE4E3Dx86wIAdreDamzXrlMZ1UvkgRi8SpMWhVCkWES1DSu9mZE6323e\\\\\\\/FBD7sK14MTYrAep5tZbx2iol\\\\\\\/FEQwiRXGKeNAVfwLk1qZl+y3xWCc5TA3wVMzg8Wbe0QT5a9OLZ6kfX\\\\\\\/t0HXB12+xGqZ+YwM7K4XLapz5fvwCoOi+GS34tlYjN+fdAwhs5pz0FwTiskbOjz1egG7ZKksBzhNb8o9DLr9b6dsMaukgjEt8RYOm\\\\\\\/QWeGMJEuH\\\\\\\/DRuPnSf\\\\\\\/imSKGoszTAdUD\\\\\\\/2v\\\\\\\/RhICaxeZBI+tvz4rW3XZ95ecyfqjwITTO6XkFFJ4XGmjo5ba0oASEfIGZuMyQdRvFqC7XacRIM+k8QGnjJoHgYDpRxfiKSpKdKbpn+dPB5xrtFiK4WFj8vmVr0sHy7kZ6Fjf+Z2fnIsvN80ZazUEldbjZCYO7GUPqk6ub\\\\\\\/l0M5wKTRWQ8dVmVqZL8ZrGfiC0mIz1D9JbijF53Yu5ukBPxbGX4ysV6iH1shhdpVVHQln9nsbo0kjwoejOuVv1ceUHUaJy3ZV18N5udDR3gZIXy2tCwg3S0vGU9\\\\\\\/md2Aq6LOAGo\\\\\\\/fdKafJNvRVhqXA2xO9KSbXGzrZvhAKYBBvLCrreiDrPNIi1d2MaxWHyCvaToiWfTtWop9x8XZZZxhmxxEVcR4fwHGhWwH+Lt0PG6QCuK4O5ryKxnXI5Xmpi1dX3zCUeU9fY+cLnApuIeNd0liRP2FIghujhjbOGoCUMLcS6jmfx0W5UpEGAufGVQEYmocAMyx6DuYMWc1R6aeAiSl9YDUzzlJF762Tvx9lclmauR3yjAUQSE74nPhHDuuLhp\\\\\\\/5z9Q\\\\\\\/DCG+ZvYfZlOcjkbvzmWpkWUgNBDLOuvYCmUnFMSBohzptHLq1xc3P4yeLkO4FxrrTsNbf54ciiq76Hy4UZYEBJG2iZYlnLKL+DO1G\\\\\\\/EJoICEi5KfZpYpB+HFRBLRK+jOgPA9X7qhwwfFPC\\\\\\\/YKxtvCXd2qqtGKRiQGlWW5TVT91VAeF0+6XRUzp1vcs0mzpex03xUmlFDVuVpijE+lJ2f3TnA7qc86nGPidgVRnQzl5+snJJcnQOpXgokjAeIHZnIc63BCnt7KmRuWJVnF13QXV8zOeHYOmD7oeXpXYNqH\\\\\\\/nb\\\\\\\/lnkMLvCwGdzH1zUbYT39nXsG1ZlKtP5pDOs56\\\\\\\/ObKh08kgeXVJFJEnw9w7HKvU7ptrZt7fm9lOea4s5qWQmk4hHFDyx+twxhplp1eBVJzu3ICkgV046w+oBgT6yZqw8Doo6han06zwzbSb8zajTj2Io3NdHD7mp0YcZ6crPFVNQpszt8sADAkrt4+4+THkIaXCr3lEKBBeY0qYW4NpJP2Y1Qxx+m5Dp\\\\\\\/Oli06K49m0D2x6KPtgg3MpYUe+UxcI0ADW+dUEy2bPXNDPV10eyWtzUGG+QzOndrmxF2\\\\\\\/HMk1hBqcOF29rCiqoNzEuquyuNikhExFuyagQiB+4A+ILn6\\\\\\\/WkQc9F8LkocYXvt2jshHfACqBzil5In9oQ9zAbZ4L2EINKOInCNO41AxfeQAAgwsAE10J26HrH5EQsy1wJJN0Bf+9fjEwCBxoR\\\\\\\/f9YdgTLDoFa9dN8YnOPXZUFleyVpZjHAd9lMRkuJ7yvxVq\\\\\\\/KqOLgm7cY7bNs3B7GhPSxHpcapyioiFvmB1QEVIriAmnRZCh5Twg1lDSjJO7c9ViemSHUqpT0rmNvzQDITxUASGlLFwLeYCJRgrxKN336akDMpOsd7CsHXhG31XDSQhJwaDyDnWmqiS7FpEUs6l5u5KOq5QhNnrp28YafkS7BOMwdxl2cPlhqH6fLCHceEEAafsZxBwsB5wglUHrNvaCuO\\\\\\\/kyijAc8Dy+bBlOjvEKxACgDs4WH+3djvX1xn+s+SUj0O4Bek\\\\\\\/UbV6MyKxdvV\\\\\\\/Mry107NiuRpFn5UizTPJ1KGk0gQVOX1TByaxcRQwY0p85TMevrDd+J7l5pJ0oJuM27MrRJTKvc5lndnRYbhhMV489bHJHN4Uy8IUOJXUEVzjZ9PSNYbK8nmsRVdOeGKvl5nr60XBP4W6w273COUBtiyssHXhRlFoXOUmsxy0f4nV\\\\\\\/+OQFo8hI5qUOaF7\\\\\\\/HiPXxWaMCq66ON5S6lyxPtmjc+uZ0vcqUFVwdWW36QNZEIhsR1BVNa69ks+en2IGIx2yMHq51ZKG9iJmC+tdp9QAgzMitewwRvAlbyotXKSyUPlcqI+FMlXeHBVM3rZ44ChFgBJcdiwy0eel\\\\\\\/aiZEQyss7BsL5iWve1GoBWIABvHwsoY1jKrDWmdH9YdDk1AM7jEG1DTj\\\\\\\/a5HItbdF09wx1Bi0I4Rvp04y1wfZB1v6G5IVQvKOYeR2bRYZjyiYGVogvMtGCDlM+qs0pMw43qI08FAc05Waln1Ukx4JaUSHEizmB+M+UV6qk+9JgXZmZXfT2dRtwlRhONHygfP2uw2QKUGeikyoIjc1+SRB73f4GWRqX1hAhd4VmWqv4fn\\\\\\\/GY5gA9dvmken3WVB4QFqlj0rzS\\\\\\\/eByPue2boVP75\\\\\\\/WIrh4rZY201Mp0Lzly6iJ+Ffoa8cPMQ\\\\\\\/p4WjMh9rJgJEgydYc7VrVOL9zSTN+kSjyuNoE3AAGxOhh26kQBZ3brf2rC2ZzNk2bGITZZICOSNsduc3G4V5mqsqhXUcJLsCNUFbqJv9sGFg5znPqBOZsicqEcNFoeuDmlrVyXtz0\\\\\\\/JTvLyH3BpmwZOz\\\\\\\/z6\\\\\\\/lYMB\\\\\\\/NsPT3H8Lz8blmewMB\\\\\\\/yryOuA\\\\\\\/ShkzVqBqefSv9Xp67KoE02glexbz38IS\\\\\\\/efKfMj6sQtLknRp9wdc+ea1uO6eX95ig6lDzJSy0U6SGJWDx8sKEg5fgC8QsYxffMhILv9wiv5oId35eG16GLlEjcyd9WffIM2ajq6nkhZh2x3U+wfdZZr4Z9084+JQ68a2fzpoK5Qp9LtmylWbd+hF6UVKbPNq4Wqh2ML74W1CKCwV2MH7U311GMqR3SRiqzc\\\\\\\/bHHxh8BuMaxGkuWvpzUP9Q2TRDXbETn5iINfAkGm2jmNmBYu4pn0B46Ju7Zhrv\\\\\\\/E2Ur5JQEMmDR6NI51kQM10pBLRWnKo\\\\\\\/tIrGo1k76Bleji5RmgFXePtd7zQWgG9\\\\\\\/f+MvQRCO7ciCS\\\\\\\/Y0FEQ6C+lZ07FSrZ4rfqH088FhExzucKJUGBxzqSUkDGFtHi6950SIZVpE\\\\\\\/Vm1pKnEK9oNJWg6YONQmSgVdPBBk+zTWK38wTzTYGifhJlwQZ3SHH+EJSpTVOB9nNgpK3lSlsxxj3JvY6tTsgMfcgrSFNohLuUFJ+NKGiLdT95ZKLzBmwgFCUnz2iGVNdx6HD7kHsq91gSh5dofJTBJFhDxRXdevAhhMWOMUA35Y3jDaz+HRqGEX855nPZEKX8UdSMPMn+k6Sbw07hgIE+87S5mqoXTV6v1xTDsV6M\\\\\\\/GTlfJBxrBSpDWA7SgnY0wA41YRDapO7\\\\\\\/d0QYxvakwFsuCccIxlP8wAzJx3uNz8FPDza1WMPcPN89mVXInSsxeodGrevpS8LmjApZltnxMKPJI1VkCiRu1EtP99B5HXyc+ZotOjP420jLMdg4O1mw\\\\\\\/DLWWjzn3EcUmjaVCq9b4O9SyRuacNHTe8kRz6ycRKOM7JIAHR8ol9Co6LfIAOyknbvG7vGstZwjH4iU7POvNjuTs22g8LmXE9TDiLJKNc8LDhl1raW5JpGMKpsnF+1WweuA3xUwVAvnJZyUnVcgWaBCaDaS+fkrKBehey9z7O3SnQeq74BHQJkk\\\\\\\/qNQGxWE9OJH0Trd5r5NLj2CIrkGKRez4LqkFSXrci5lUYlj\\\\\\\/++RDYffg3o0I1Mr6wmjGzVWfXMsLj\\\\\\\/xZ2fY7fO9ft7xbCJk4qz5MVRkrYT92U4ftKydsnkugrJ1KzsrXx0UppeH\\\\\\\/iesfsEtLsuQN4XP0PytQt6p46xEF1RZLKUyV0NhdLxTuONvIGR5PKkfve5rhjAS\\\\\\\/MgM5LSFyWjFJ1nWTsmlDqZM\\\\\\\/0HD5reesjIsSCRYtMSCVYGLUV3HtJH7rQ2qrx+UtDZoMmf5EcO4UooQNqThHaQRpYYcUWk5IbBePeQ+suxjSypVOI0BV2tGV6TuBTy5NZztTX92WY4wdNihT7N5le+q928IiArwm9SrQ4U2B9m2DrJgx2qFzbUIHvMOvzphgULKWswETRDY\\\\\\\/TQ5cZdkentOtkAPoprdznD6thgX+GBG8pkA4VYKlxEfQnwIorO4og7F6I6gNWRVXtPOLBI2DgCs5nVeA9QhIGoAJwrfQfMxhEwa4s2W+V4Bz1xi+raPFH8486eSBf8Ec3i1431bxtedKMCjc6SKMbzqvTxldZi7ccfuL+goKISnQb8pmxbuB5BQLR3dQ9yGnivIcw6NBgRGGi3AXeS5dPFwav3Arg22VC8SKhovqlTCO+cPUEGtgfS+JGnzWzy0Bsv1t6lRlQgKFwiYAU2cGicLwRxLEuKsA2OI9Lp0W8DzBt728kBMiKTjS1P0Ac0wjUqjh65HyaI21Ad8ZnA11lkmFl8dL8i95GvoiJW+4\\\\\\\/FCorqmihjhDo+8w9dm1EAp60uuMxwQT2daIrlSPPcYt182AlDwKZVN28NCJNAjnnaXprtmnCQAHIsxFuu+KFpifRiqtQ4Hg6Lx9qjOfG6GNQyOY98zYvlCK2vzMnS0WQSCvLKe41dLM52Mv4vETRBXhgdVuxLLXc17tPLoqVxVdW9+L391tSjnUTS3d22m5LsT6dXmfKEcPmat2uX7P\\\\\\\/u5hTZGqm606cEvig81ZhLUULk\\\\\\\/7cRQ1Zy4sRUWpnIIn479nxMEO+SKb4pKByzMEczUvmUr0KtG9Fd\\\\\\\/ZSrNolrcRj48Gh2Juzf\\\\\\\/Hvyiyc7HuguDnd79WPyOqSg4vFdQL4L6AVE8Ri0IPaPelCiTLIGKTR5nDN\\\\\\\/38+7kLcBfUNS56+RR730dZSapuNNE4ImbdOBH\\\\\\\/NTYzUdSpSOtTfo3bzITTycg2SNQ6UYREfhKKV1cqcymjnIFpoIsqxBlZq+uV2RTomjeM373nrAi2SLE8lRv2K2j3DWrcbc8LW6+rTmLWsEUJCgFR65niF4Di9RtjHr7NGPRR3eUIJ6sbl1q\\\\\\\/FsbJ0iK1FAmnTNMooerh\\\\\\\/2vJo1t0SWqvRRJIDTjLZor8Z4s4njuaVLPw8XgomifmwjdecgxXLT5BI7lL\\\\\\\/VmpIq6wKj38+qCEdYkB4+uLS2g6dFqnOSxmgs6\\\\\\\/C4nXMv0rUjrF8UkGKVo0RI8oqnQ4ZmIdxkpVAXUwHuZSnRJG2pZ+a5paOYe\\\\\\\/pLi9J74MxbJmO54OVNwRwqvB+DeUtJCmC1S+QnokYkjm\\\\\\\/q3XzWW+IHbop9NQbR9ZT+mYNw2IkWPJyQ0FkBlj8NvxJP\\\\\\\/13wwWAKJjy0CivKqw+UqCVM\\\\\\\/Cuj\\\\\\\/loeBvQGQVlm1\\\\\\\/+TeEPiJtDBb1SciYqaSKI7iJ2nGttfouMH2q\\\\\\\/U67V+zW9pk+W1zFr1PGvEyuc2eni4fgN\\\\\\\/Fz55vjsv\\\\\\\/cKP8Y\\\\\\\/bozEAwFYAseXdTQ67EFKCa0UPCu338OoeLg5Vf9op91nLCplvv\\\\\\\/qKp8uPQqZtpE\\\\\\\/41SXSVX6sr3O8GGusO4bxYpxHaBltqTMQ4\\\\\\\/KV+IRud0XpU4Qt9zLmgpdNWVTUTirDxtFpemsug8qdUuHsxnpen2kaMYJWPMXlicFx3sl1MApmr2yX5kyR65Ncb27iFUkS0tm02vXdNJIyY2MjU1n5woEYPDQBszWX1aHeMvR8g8DEQ1jdtz5OMhRzbM\\\\\\\/WJ+q4m0EuhIXMXcz5v9n2ZBxR13Wwlvr+VnwMVTI7CyHp2UsaytLKbb1ov3llkp591iiSUpI++ZceRvFzM0FJC2b2g7PTKrXprfrgXt4G9rQo7GPXN\\\\\\\/F6fd\\\\\\\/NW+zK0YjuTgQ7ZCaVNgjXJv+z2dgeyamJ3J5O3aPVPakr5SvlsCDlf7HBoZsIds8FNEOnAyKk591rHhwsFnXIMa9WUjPSvnrTgGAHltdBTbt8aszGeHqLivhk81TymYh2r\\\\\\\/yScqnt4IkaPc9Y0U3iekiePZEET5eHD5X\\\\\\\/QjVTM1qAOq9a6Dptd7yPbDsyIlJBT2IIcKaMBh4T+JIiJ\\\\\\\/r75cCowFcnma9Kdeta7dZHKqiWwJCbvxWBRCkynzUwTTMYSLqVrcAi5SMU+s11Ew+Noj5RrEkRZkZqA1cZRPCQ3fkQb+riEY6l2jiA3nsu4s4tkBe08+gWwLuBFbtaPe+SQNoVWaVz14jsIva3goZ+1o3BF6SarDSYC7LXugZeUQi3U2sWFcaooHWUGFxEBoS2Odd0iIybeDbZG6Q0GnLhZ\\\\\\\/d8kNPvCPur99CPUK68mPj8WJkmAf5YJSZxeNgjmFKasdBti6HqoM9Fpa+cRc0fJ7RFy6+nY9n16+XsP8m0vcdnmz3Q4lP9siSe1Oo1Kd6WaUvEA2xZ+7o65tqeKqKeHlnJpcy0YKhSyqG\\\\\\\/BKmGhDdqT8ynb3tyyGaEfuUwh0o5sbisQhwYil6TY2cVkK6Adj37ejAdMNrex9wOn26\\\\\\\/wKydhNZ+hqe+1lCXDunadXuXeDuIY00vTdVALJwldAGV6jXrx9Ok9paEa5jReE7sR63ziW1iO8gwQjEGrRANJe118XS\\\\\\\/ZXFxKGUAJQVhnD5+w6D5PgNGxk6QAiogBPrNVIbKxlkcdSmiRaJQ7WRuCwmG0Icw+WqKde3b0O1KJ5lvjOQ0P2KT+jK4Zupbzr36zrOHBr231bmra3yQBvI0qDv9T8R9N5s2sCNRX+NPhYoNdMfevdjfvjFYO+vdrfOliLkxeknpmyVfUGZttBvSJnsk8JWOxhd26yOPT4gRYCvzbH\\\\\\\/QGSVmaIJ5fhqloqzbBPLpDG0GdK8vM2RXVJafoKrXhXKedd1OIPPjdmvUlGrZ1zfVEjKANaJklx0n\\\\\\\/5V959a554TRyNnUpBRsmiZcYjH5xdiJsR4X51\\\\\\\/jNfiqbKIGNOvk21W2tnIJfu3krDgkBjMXH7Ydz9nlkw3MKjJGP1JWXhfPHSIqJw99TrY7HnwmL6WbeBWDhy2PkU47t2Mkr1nDp9zSoVwAk7M6ezTuweG5QK4CpdwHoJApiRLjpnjBoDxQube4JpwtsLV0eJb3+L8Na5XwkMs1LMw9eV23Gklo\\\\\\\/Z7X56kOo31kPWorjk0RAGIoM0IXspWkFIh0Bh5KzNHo3PBas9vhgMOwESyw4NelJjn49tQqdm+Nzjl8JTKnonsaF9sdnCbHL87X\\\\\\\/TYklLav\\\\\\\/perHw1aY8FQvSMM\\\\\\\/dbX6qcRZgR1YPBgzoMXgQoh06gP\\\\\\\/PqWokYxkfQQp\\\\\\\/CerV2I3G4v9uLypCFkk4utaNSTcejFUbidLg2fhRo229it8A7H6IWmhdoAs572BJXxXAzZduVVezNzz\\\\\\\/KckvGOdwmUs+v3S27G16rjauRCcu+pMkvH8cEp+T5RXB6sUimOpZtD+QWr6TTE3L2JjapvXyTf3N5al9r\\\\\\\/tb+QcMARjuxH+wTE181UcSn+fxOS0vE7pXq6maFVTz1kA0JDfQtYtnbvaHfOr6i6LSmE1o3IVdYVbNjnthlC2I+XuXwnTYeFACqX2G9zSehrm3GmhtlXJ+j0tDkQ+8DifjD5qv5546O1hzwMxtHZcF9hxygN94fSVAHLS6r+J8xYpYb\\\\\\\/++omAkalfqgs07EXMhKYRYhrW+v6xIl6HC7limlYuPHh8t7orVlKUOT2Bnla4yAB5Fb5VMdoQdo+GsBPuuV0f1hbh6iAxgmLC9z26Ks7nknSOpex63++819l9pft5HEYDevSFq2nO5R14BMWZ\\\\\\\/DTGKOy06zduNXgeKSnfDG9WpVoxOgirj8Dq7eN2C+EadCQKON5CVi2a0vWP\\\\\\\/17H1YobXMrdp\\\\\\\/XdKeC1cy9+k2QoBVlApWecYkW0EgDqouVhIo60Y+aX1Z9B\\\\\\\/kMmAsCcZw3LonFHa9MbL984Ye+UgYkhDtwZNodQm9wRN1cS9q7lRKlcI3Hdht\\\\\\\/liQFpHnMdh8UIWd3cMyAfXJBuQ0IxdJTjj439EijtIOApT27kX72x6nX9\\\\\\\/UFdqhJVS3tEjlX13OROKjVrO0by17SNKWwPt8ivKH5WCWkhzblLJ168NfSnMKC2HOVBJ8sa23r69FyvoU27IecGxysMP+aLxcWeMrU3FzUwboSjjv6yUuJVpd0Zisc1x7D4qIDa5XNRiVkUuF83BP3lf93nC1AuzPG4DS4r24TfWxJ1IZEKqX40kSpT+yTRo5DLgsDghor3qOl6AOJRxYm8Kc+cNrJNB0t2aWR8F8Q6FiVoLyJd+77fBfiDR+zq9\\\\\\\/jkq7GkIL2s2ecXKI5k5LJ5kLWL1Wxnovnf5++bYj58GjLsVIHOQBF5L6pZ610U8wE7JHItuY3EIgJfRT\\\\\\\/Us+7N5j\\\\\\\/pF6xP\\\\\\\/4E\\\\\\\/s9Ze3vZaP4paZySYIZwVv398IsvevTJkuEZv3M3sd1QICjOa7OetrarMeTMT8xSPWO0EfKThtGbMgH+s3CneLqBtJUnzfdJml9Wf0zan\\\\\\\/zG2tqgZfJmIzm9tY1DhEKxeZVmu7bM\\\\\\\/963kCBXbvO+RB6NKVmBZknCztmyU7jD8FAyps3HRAYfLkiIVyPaHFdn4FQMuO\\\\\\\/nATBeR6pNrY95jTc\\\\\\\/bQvymEdi+AM9feAYRTHLF0or7lBcitqfdqw6O7UNDdREE6qFbEi2Mw3E9Tp0YpDo5AEkJJQhPG\\\\\\\/zvOlpwmBcq9TXNszELyJbAR8xVcd0M2TCh8ppyU6FpXwTfONpapEnsArKqk8UEehatAqhXvWpi5t8BknuV+vpCF+ZlDcSVC\\\\\\\/GynZ69JvYY7jCVu4y8YuJmKe9CEOth\\\\\\\/jKn65GZIatOmhL8WTT6zZ4Wv5KlyeqMlcqwosez8\\\\\\\/qF+Cp49TmLsdBfUCvzThITyZnfKYIgJVjIHfm\\\\\\\/MGLO7Qosa5dudJVg1SWnI6Vxh1MhYO5CQkA5Pzwr+rZlcKubqgsEE5EUI+JUHKE+ERFEsDsg09Is\\\\\\\/zgu+wmy0wEdjRG9Ul\\\\\\\/4nNb1gIEE3ODC4yUeXsTjct7aYpOHv\\\\\\\/HGz3z2BbDUvWn5b59+S8wHtSJRHoHWg35zt2gV58iMV9JZ3J52bHOdNQVptdA8UuivaKe+JXi7Zlan32U283uXsmbCZmRe\\\\\\\/v+NDHu6yT+WQVL0Q7jH8ps3uB5JLOh4l6FwD5XOJ7Dkk+V6HRG\\\\\\\/7F6DUBxRGfC05XBNKUGFv9C7p0qhatwybm3SrK2S1RQg9CW8IevUHexvJ0teRld2nW8bjsC+QGWFONvX\\\\\\\/0TKZ0Lc8Sj\\\\\\\/dAcSe11qTO+i\\\\\\\/EZvAk96mSkL0c96tBJ41RPHWaGihzjl1GT6z52WASQUP5Hcj76Yms+mhLA3OLNaSgtQ1QRK6FvvpC7i0PmoBmcc7ZD2A2wdyuEX7RdEeziygD6nbmykA3XSl9xRxUVXipF81HXbNHHAKc8S94TLbfYmSUDPR4hLk3k5qJIzGLNnm8ZAd33qxu5U18TbgIDil+0H+5SV605eKuJDrUwwTUFeDTMO6ESD\\\\\\\/n1F1xizJFy6A1\\\\\\\/hJjQpyNnpFhhjTq0yL+UJuePF9ZyrQUJNPhJKq4YMtauX+CdF8LvMskhC14Gh3Al9d5F5hflwevchP6nIdcm5WyfWAp3Q78yI0YjC5glGRaQstG3rlwtMu8oJVCEzAwCtsnaca5cn5WhMrX8h+1++fX62icRBgEUKU0L+8iK8tBHWY47cApc2vmphRnKsBo4u50j\\\\\\\/640qUAmPRY1pe5xPi7\\\\\\\/Wv6JRASOkjEnML7CBjsAh920r7fCBeqS7otG1TnE6AUWQfyr2vxUMAOpIonjUuYfYx9dXzDezscoWV2ghDc2XVaf\\\\\\\/cgI+mb\\\\\\\/Au9zw0SZExAMFKar8QAnYJOzn+GbbEH6wwDsmgmfoVyHye8W6U+ATVVCc4RDMfEvAz5MlYCAT8kgtMhPP3VD6frUllsHmMU+9ulTXuEB++r8Yd4o8zSlUYeVHEBvjfJqU+SURXWGEYIT8ooWDiZ7xdRtuUWiQYc7HW9BGKxzGWQYqUPGNW0IpPDzxGtBlmLDde9juFS1jUCaz7+JqiKytTn+GH2EOTWfbEDVHiAxoIoR+CJ\\\\\\\/AeqDa2E8Ga4UdIsO660dbuM+2sLFzNlqVYelTiwBfHKkKMo3tbd2ZSqMgQbOq\\\\\\\/8DnHAJpA\\\\\\\/v7fs7t883B7n1tURzXadkku+MNBLOBeIGIrosBzrzzRnq2nmWOsNroN7kSXHfhb1Sx9l0r8Dz0ycZDD+yVP+2DuDBRayHRTHpVfUUJROxHZyeVPGr1k3F4Z9\\\\\\\/IJT610\\\\\\\/97jJD4iw2W40nWFQhHew2+u6zEu94505VYxRx4SaZlxVtEuzgrZRNQdu5DfFTDOlXDbqZfhXvT\\\\\\\/Prv0+viTx39\\\\\\\/6MHazbmWjcJVy6PKUmFRg+5iZjNKZ7bwDDNVYDt6CQ4fvFUql+7TX1SWvx5isr8bU+53PvuWFTVhFM8Tv0PN++haNd15NKvxko0YBKNbIuqSSWh2okOuIfWDBe4PFKtdOJP8DuNgZSKqUvefjYcCCQqN140NYUSS0YQRUaOzI58KZP1pI4SX\\\\\\\/6O6zjJ1kxzt47AxJ7ZoH\\\\\\\/0Zdj7jiAcyRPtfAvwnUlaJrcmlyJntITH+MnI6Z86cv+HYVditglB1\\\\\\\/AVwKLHL4MBktygWOh6SGsokLJUvv72RYbBYY00Nf5d3uWKdal4ZZx3CtyRgIFtWxfBkYuppPAoVABmmoDO3fWDOk8oixQ9hko1tPpmXgdqEO27aCrMDOBtHxPUyFYS3C\\\\\\\/UgOWy5Vj9qn9kUoSZaCPyj80XTBv9saETLqlhKYK5Z6cwRfiWqOOw3naBq\\\\\\\/tHajV2X\\\\\\\/NQqTs1svF3H5ksIjviT2ozx6Wkz80BeE6BjW3ss5TtgzrGqXJSeubnQS1XcWhHlJemEBGFqHXw9hRbpW9uhYnoWg40bYTAsQrDeD\\\\\\\/+aqHcVpZHpQqdOSa78DMgDvNkaHK9NNjB+xPbJ0d+MgokkbNvG85sk3rRFL+32D2dsPNVA0tVVKHR\\\\\\\/GvJFUZEdUUuGPne8LjgFfYe653Hs8\\\\\\\/E\\\\\\\/CKVW5OqrixSt74GTPS+eF3wVDCE2kwmeJTb+cD0OtGIbCqyQOwHAGb31fr61U5qodaeOkOt\\\\\\\/FDOah0FM0B55T6GydZu71bbd3+S3XMosri7BjIcpGBcNeZYtxysE6sfYUZooVtBTTXH0tkB+TAJxU+kAHluZXzjCWvgT4VF3nXCXbPKLFhwG+DkdOgfu8nNXsco71MGbhFWqyRVC\\\\\\\/OIFLTDVTTViPZNugoA+T99a6kYGWAidU2KiGOt3hgdHPKiKm3uAHEuI5DYyd1k2zRVIGuLpKhpZz19cQStKJhHJW8pPNtMuWWLfexaUMBTKjufhLpk5zo95Ac5\\\\\\\/hC\\\\\\\/8D992eYm8EZuiIU5j8\\\\\\\/\\\\\\\/Gc9WNrUYSWIK4oc+0FI2l4i+S7NBA3DpeHlizqg\\\\\\\/UX6T+NOPdU8zsZRA4onqquwso3PvqtOmpmfZYQub3gLulynghySExrQGJBjed2IArmulWZaSGIeLb0hBNIpBBroVf4X6c71QPsBhEHqKSzy+fWvc8qBceR6NY3BkXWvx8Lx2ybUvsukZmIU4MqIsWA8jQawegyrULPyS3LxsW9CLTv3JM7hxugrfr5dm65Ri1MP073TtIVNeI0Bg7LYuFFTn5RFtxjMFudO5i\\\\\\\/x\\\\\\\/zspDeKeLkj1UTdGAj6WAVixGsfh4fQ27STLf9ubNGOi02llto1IRHJEkeHQJa5WyhZXv9WggRQadH6b7+0\\\\\\\/+4WedfjFcp3gWTUIcWP7B9Bzkt++tyV5yYovY\\\\\\\/YmaqlQrMC7On7WUkHzt1h7ixnL5EphI35Z+suUMMGLnzirtwh\\\\\\\/bu9YT3fbOZqQubrUBkjQS+teEfrM5RUj25DwETe+t4IiB3fzxJAmNxPOHgPeJuj76r6W5Br1\\\\\\\/eoPUcJBKXL4IPWCmKuDXhMJZkRx+3Y3wteldBYzNC+koW8lUSS7ig6eZERMBuPqWwOs5w2ebNyb83yw2ks2lzEbAoZGP6KMaynGKc1deagAfuIVvsAQnBp4\\\\\\\/d1gYR2tADooKCYmYC4z01C7LtsQdq4+kUZsWAz8MqPf8I\\\\\\\/+ckc2PkBzbvIzhLXan3TLCUO02QrLnwiw\\\\\\\/e7Y8sS\\\\\\\/kQ+2JTp2hV4Lu2ZqiDYUjhjI1yr7ZtYLeSIdw4h7j1ulSCj2FKh5ZOh\\\\\\\/W7VPGsZk+cJRx9uONDGxI2dYcg86D+yXsFJq3ZsxzL7faLhovw7GAw0hLmDa5IQcLQl34rfmNz72ypNzPeqffnG+g2PqJjk21vTEMcHt7hmuylxFIW1cuAJbEpHZqtRYCfx3P29KiO3XeKW1eakRJBv8JFGbnrH765Y+PjKFx5L0FuhgHUbn5V8o4Ar1cWiplGzUUc9n7PlKPuPYFQuOFWypVT6I+pY3x46TcB309JykoL74JFGlO1iFoRIhFC8zGuDvNOwJpyPfS1gWputGqqc\\\\\\\/sGVxJtlirETqKMgZ8hSiGbQrcX6S7BZHwDpVZnycbq13UhUCpSYEwacj5JRxD6EUk0XyDAFXM1Ae8zmGNbq9WU\\\\\\\/2+9sBV75ZsCuicnPD3z\\\\\\\/WsHJkXiuo\\\\\\\/90rLvjtpVBKl97t\\\\\\\/zHxEj2cIkRIBCw0P7PuB9bZy4P4g3\\\\\\\/8GCbqOyfWfRvTOswljBNxHOAsH8hrpPi6Hqd17pOBmM2wsqamxE0axKXEC43wT1QlsOlOai04H4vUDU9VDstry9dQPec+ujbDaeuRPsQl6Oj5SlmBHEm\\\\\\\/Bn6sK1n5u5V7eNzr3MFpmlre9tvLprQ43yfZ8D1LBSN64vJ9TiMt6tE+FB9vJwp2AAqTgGMcgmFuavInVeANTaGL\\\\\\\/tRkNXClZvljGKoh4LSizT8K2KVNQjN55aIIEvxyQfxg7\\\\\\\/q5HnySG0wIGqfkVrUbTSkCbqZAGIQoSvZLR4sKuKKNYKbKkN8l+3iMPH5YwgMqhWijjkQRNRqqO+kCY7fO1vY2XiOuScQtWM2IQFUApeLAgukFdnU+\\\\\\\/N5dfHSq5hlSAXem+BdKUoRbt\\\\\\\/aTXmCrbQ5d6A+c9\\\\\\\/JNzPVU4AQ6\\\\\\\/NaM1Z5pKlenN7B8+jHtqxX1e+rs6EizYbejX5KU7+2HVTD9W2jzw9kNyNTAUQPUsHG6ft5\\\\\\\/BIpb0Rl04BZROlJ5pFwyS8DIVOOpTs0DCGmo82w54znW6PC8CiIBwRUm9ZWNzct9l1qfDijnSWBVvOOKNdQEJ9d1nfIr3oGGfasAJSbGntH7KrOUJgKqOauCgJ4LwfiXO+sr3TvMSI8WmfjO9lTyQJ872DXmMW48\\\\\\\/gYRDIOBeCoFrjxqTOZbVz8hGHxa6frmE0zM9Lmn0HEoGzgRafiyqsJmfgxuitAtTFDjW2fiMpDbD0sDVTxE7pCUEz0RNvqLvK8tPld4ZpWY+bLbR4OKn+UGSxudpZclW+H5Y+CwrfPmytKqTKkTBUytil9t+P2M2lhTHYON6ww5xmAfvsJeXEfGlKkphxyvWmt5ObfZY4ceAPhauLJltjcSqUtiXiggpzv1LTM5NDqyYo6NJVN8cLdgUX0rsD7x9fPxBrHJmtjjQQwMqnJ9tnQjAKjIu\\\\\\\/0hNQnLvux9d7Z\\\\\\\/xLc9ZaJ0sg549xWqY245cR+Ih+l7Jc3Eb3F0Hphp4rub1ELg0T1VrDRfyxQ46zivQqO6Ziidk+RAMIMCQH2OqKGCHGCSCvFJIyzRckALPSydL9uYcqxUbLdeuBwW3ZodUs7vzxb7mN3Pu4F2nM6kscGz6+qgDGC38CAhrHN401krjmYQFg3oaLaH84rSNqu7x9JGbV3qA\\\\\\\/pEJqX1gjhfVlVsMQ\\\\\\\/zfRrxOHDeuNFwI+LCzHFhqKS6+vvh8+P9U9oxScbBrJhV4EjRRRfN0mxg5Ca5rX9jRoXDTDCzBvNBy7NGoyK1YIXM\\\\\\\/Nv\\\\\\\/jl2Q7xY8zYahym\\\\\\\/kGWHTjpLJs7nl5s0LmGbkyWs\\\\\\\/8SZ4lIvGT+pg9w0t0TGZkcYqbmcfa0GPd2drHbFhc\\\\\\\/D2OSJXfQOaKvcK\\\\\\\/HB2t0U7YtEPeyAuJnqD6kwlBP9RKqhfe0EYiGbSggIMCgjUumM5ey1VpZs4GXqLmc38mx83rM9denTdncGcIkd8OI8\\\\\\\/Zvkx8ZPSNdHzRKot3KTAoEIRK\\\\\\\/iRPtsyU\\\\\\\/32DxgXip04Q8DnnWAMmuT5USMKGigUNS+c2sLFkvO4X1eKDfjh\\\\\\\/CLi67XVLCo+JW8823\\\\\\\/cvo97TPKabs434yZ1ysHlm9ZGpJxXZ8wtqM9DzmgT3OrrcF4kGYYE2RksEvNsSgj9H5Ry9etgRrhM735mrxXG\\\\\\\/uHT6lxh6oRp2Ahe7UOgSpGgbwaqCQDiYMwU33oMqjwovgPYvm53DCcwa6FE4N5xUrPgeetJ0xWpKKFMtPmAMCknusjqUuPgOqthaAUDltBlidTS5o2KtoT+8HhvmdfZV6KbpTQCcEWRND9TAaSy\\\\\\\/ks6c6osLbQvK7\\\\\\\/Aplr1LWRplzJ7vLxBmBWkzMwLhWR6+5pKPY0RiqmpY8tPULxmNvEV9QMvVfCcM+NfxPk6noTCUFv3YeXMiyd1yvk5kCyUAzVVYYaTsdiFJPsimNHd0lsGjBJvk1Ba3aDjJ2FQk7P6n5+9qS2Kai0tCnhPTXZNANhIKkJj5xlTXooKcuVLl0qbA8G9BgQ94MqR\\\\\\\/lD\\\\\\\/fcoN25DbeuyR7YThN6NbS9F0RaaWuzR6GaRhyqR\\\\\\\/IkNW6flkNjAUqi93Ti1cs5+oIf8JpWgpOnZ6McasmcasBIv4ZA2lgMz2SHLUpjC7t\\\\\\\/RWkBD5C6atfVEhqXbSXU40O9no+3tIOtYoHadh2w0k3Nv3MOlaCX1D84MMoc7r36GG5GRJpthtxLnJKa0qPVKErCw7hJ4xK2zno\\\\\\\/7aiSE78nY6d6J8HgFNuvGQDqaQmJ+YjZnAd2IsokfHBO998+AZS+0ZoO7iiA\\\\\\\/q4Jlv8dVXiPEiBz9Q\\\\\\\/1xwSWUblHagHUa7fmSRE2oAN1oWcEA9\\\\\\\/owQaJCwwOoMFle+4o6qqUM2cmQybkt7hVwQFAJ\\\\\\\/cjp5E8JIkGsyspiZBseKe9HUQI7BC4LECZ3K20z\\\\\\\/yzqwBKD0ORuZ+DA0uLggzTelNpP7uPqgCcKm4Ym1m4K13tcLrOHB\\\\\\\/gbiRXb0qpLfmhUUdy7AWhoBAqImOjKFmZ1y1kaQEIlUbRrsgXaN795\\\\\\\/l5G4MVWsFCZI1txbx4yrSV6MGxKIR7CShyxRBToe8aTL\\\\\\\/5KPZ3ojux7a7uY8J1ACL1guennOfn\\\\\\\/9a2PEtpLSNic1I88VmPl2iljBcDA+oMfGcu9RPUR4Q9jaT0S5MqLjJa3DbcRhQinql5K6\\\\\\\/2Qh2ITWymH8QJE1tLekS\\\\\\\/98Lcp5nZ7LN+PLGZXUv66e3D0aVF06Ie+MSpOMaJPyF\\\\\\\/U9Yo13afkGATMS84gMGGvotdUVFrUL42rb5tbwTg5R0dCPJyMwjfXL6NpfdpWKIhzgNpyFRnUmRfiLApxM3ATpGWruvCui7YDscEKTrOPnWNa1mvukS5mLmvMIblBowI6P0VeFT+JJc32t3d8+\\\\\\\/aaNTnebTmhSIraJrKBdG1SVZ5Dg6vFrpMsObm8LXi3ApoPq4RRHWqtNMSLAZVzE1WbOd33ExQZa4dQ7TFtD\\\\\\\/zttqdErxwEDOMrqIjAjM023+1XMFKgvPfOqTKVZg+6JG+SfBdhqiIBGFjcQBdYyJrVKWEDQ75sZosDzkMKF8NwY3dmtYfzTmTweCEaGEFOw98W224AktAcExPBezxFc5UhT123smL5FO7y\\\\\\\/a9heWCl9R3QH8plwzCyCiBAxUFHu+cPj4ZhMc4m8ZAEtkofjIxRKm\\\\\\\/sZuRjlrckIpbKTm9iTc\\\\\\\/M0V2EB4IteXFZoj7pzOtfeyUj94AcH8MAzN4V9tWR8kwMYPwuZqyDNQgQxq2Fv61ogIwAwjvZpnHnq5VsWS5L7VSdSCZiL5acXeSqq4eBifArnOZ1J6NPu2Jz4lK+9ApAOoYOYIpIow8dd2agVYcGIl12TRSIEmKgWlykCKojZKQW3BxhLkIEiu3wrvakjyMruypN3F+xawRiLa1SDiSCr75v1XWfqofUetIIJk2aTlf8l4jfx2xSaeLABktLrmHI9T+KQzCWCZcMvadbXe7\\\\\\\/QJ1NeXG3GBqL84U1zwISLsblMTDHxzG3hZCeQzxkDGhXv75QTLuhJZ46\\\\\\\/rbtgHg3IjfGpQwn6a001Iat51mGr025wsYrZk8jrJUoFmO6zLzVgMxYLLwANlPG9im4h1vbfh5AJKPHfghe0CPcHJd6ShiSH3SbuOGN2PgfW9KjSqZBGI2Qok7Hp7q7VjVhZkQMcsTes5\\\\\\\/08TBH4MniEnQFCrzLXKlMQj5pPhZdoWvHUTZ32qr4+jRfQgZR9FgKwZWw2zZWGzlfoS6FTQBPPGmGCLWZ5yM9aIXK6fK4g9uMu2qZETANCMlwR45FDHYWx7NTkczDRK08j3\\\\\\\/2J78D5FWYJVwjlo0N+ude3SZilwmeGf9IS1hwG6i3KL9atVCFh2mjYZSEScsI6+h+XW639Hl+i9hPmgFSH5287lVXeJ8Jhx+m6i7iSk9UpAcXLaWOpcbiTaWhU1E6CSYFYEmLYi44dMDuUw9Pvb05KWyRqDOWxejL\\\\\\\/B6JejQZDsnlXuVOxRSPOz1yrxngogc2387pxfmXKW\\\\\\\/qCECxSHg1luTLeiSIupsk0iAK5G4bHyFemCUL28jhZ905HyRANOpC3c7aDSHNXRkpEfO\\\\\\\/ZJVbhh27GOcBVCsJFPPOUIYjWS0HSy8nD0JnnNHW0gXpJooy8CZIET7gyYB6pW2pi+WnhYCfmh2D0o9g5upu44XoozLleqKh00x6HBzWXe\\\\\\\/CMfQGuyjDaTwBV1OkRNUTu5dlLPYsXRPnyu8rHNiFC9zFLuGADq06Cy4NDDSi0rfi0Ce2JH+C05SnEAm8CIoyc8HZafthkXVZFDg7RU2LzaO5XKtSt3AQmea3uNl3KzRSUjTwrkKDleBGPp1O3RHUED4Bhnk3YccxORbPY4QQKxh9MykE2pWD0aBiRDfbT15SHxH7Kn9OhSo+ZCDrKuP5pVOJN4H4zAt1iQZxWmU0z17ZiepM3eJ7ZfYH9i1qb\\\\\\\/ShhjIaTK6CU\\\\\\\/qsen8NloQNpOgz7SkxzH8Sol6mJyCwMAAXIaIjNIADmBpV4pkms9g0SgjC\\\\\\\/8QDPPJHDUbAfRRaZNwTFJhUjtO3X1deuN\\\\\\\/vO598qOCq6z+edn9Pe9gSZ3GFigm+tUzFt5pCF6J6\\\\\\\/JGotsRWFYDMZNAwSCW+t52bP9T1450mYlJicJpFFwP6t5f8hFBXQQ1LRdL7q8QqUPi2fCQ+2zAv5qLJE3KwiDElGaRi6jb1L4MdmIvBIGxpZYkAXrdpY2rDpEeLerBah2GBIE5X+PWN1+mp2T1kf4m8PW249KTQ4OfYwKNI6lLz7PDiI0+HXGa9VAh1zfP0Itod8Fr8ihO0cxCSduoAoRvEkc8RcF5FjfSzzbfmteNLr6KCzO4Miep3p+sF6pe9d3HkASq8LBBcWKXNyXpSPwsMFj0TGYaK0JM7l7M2HhJoVdoRypYoCJM6fmv61THytxOW4dV2S9tH46pfmdZKRL2Lxe6+ml3OCAysaLrAYp82wcejEGoDtNB+d1O9yMK7ThhxOtRuuLUJEuUAA7smGlzQuG7HUrQKJhE374pp9Y5Lh5Pzr+DvlOGnigGMfiSEZKtBbAMfeFrfkZE08b4s4bC7gCRPpbDDDp9exbx5jxzpJXaszQRCF8w2FQubTyY3rSD+r+YzXWkSK\\\\\\\/Lycmyjs39vivsWnD2Dp9VRVEYKk5pbAzhUt\\\\\\\/SVqxIi0GV\\\\\\\/SDNAhEQHOvek+Zu4w+6vVKi73GkM0031UqLp5Y1dTsZpPeA+nkN0rjlxz2JIpssp+6\\\\\\\/FwxNcXGHDI7tUmPQU6\\\\\\\/efuAQBQwUIieeJcBZr68\\\\\\\/Kntzb4eEEvKHa6i02cfnUIsg+BVMOZNtqMS2N8pPUa4v9vkKL1tM0mnBgkX1guV1QjmTUbYKXIC9B\\\\\\\/U2zUhi\\\\\\\/FVTYwnzVi5fnIyq517w4dd55I1niL4xzeg1MTD7KxMUaJNSckCF6X7oExW+DbbIfyO8pCP9dIEkCsOFSsQdkGY+Dum3Zwk1z65ZqXOyRpXMwGFwYTBAoQJ2czILHNNqx72vMK4ha86vmmm3zJd2UfWw\\\\\\\/\\\\\\\/vOhfSL50ojD2dCvmVRTK6Wduqhkjwvshto9LNshoXO29cDbquA\\\\\\\/iq\\\\\\\/rVDtYGGbqhA6UQgviPwbf\\\\\\\/2gaLSwih3bPKbSy5sTP5L8kQ\\\\\\\/1K73EKhfRFY8VxBTVOnLpviy2N8\\\\\\\/KCwQlTeLFq15QjLbV\\\\\\\/RitCcXilO17vlU5WAYoMQp+Iol9slct65yQ8wUCZIJXbdNZTJwIzEbNcyEoZqw5iNBNT+BggRcRjV\\\\\\\/eHLdtOpnwea49pV2bm3Tio9mkEYeKjXPxVrSu181GczXGfCMEG4QNEg9w0UbtOqEQMrreWXHEhtlwU5Q67Oc2cErRmir9+UBPidOEQR1yqhUteW\\\\\\\/r74PuCVKCdS+\\\\\\\/vgmi2kf7HpG57GBv51nLp8OeyfLMqDqaIFbwc9OM+CjAlylQYWCWTD6iCXFwr\\\\\\\/5BCDd15zZOYpsR8rL8rXxpaeEwccFGHTDytK097K77W1cSwYEKQnX5kDVLxrs2zVqoRt5BVehpdtKTOet7xdTh3EHuo+iBSnVWf\\\\\\\/dADEH2KqQwdWraZL29f8LjUaKffuvijyDrDskjuUkDunM67IuGywvpMmlihMwFQDkpLP6h2RQKn4QPABuxqHfORKW\\\\\\\/vCdSTzRQZPGr1mrTXGzYHVxbM2pYZsLhlXvFfRI+3AbIwlrsdryoPq3Uwhas2M2wI6uj6u0pWdBG19y7ShBxvST\\\\\\\/7vT2HMAR3DZyouyTq49uThyJbo5eM3L9osTs2bBm5cyllfsvS01sY08N7s+M19WxmMEew6D5Xs2wCTq4OpwBfBgdozlupW0aPvWi\\\\\\\/9C2DQkmRxgRQm0UWA2bQseT6cCnziz+p11TVar3weJhsbWjWTN2KliIm+xJLdJuKJn8gRSKC1QrPS+xJ2ggPAO\\\\\\\/J8UKrdD2faxs6TLbuzhVajTA8TuB5jYwgqfwHGnpYE6S7DrYJm\\\\\\\/HtaEFXneAEw3dhDvNyAN6jxDRWsKBT0Gj9X48k9J\\\\\\\/C80UvHS216HAgETItxnnWeB53XJDhQ1Nwuv8PNN9JXEFJ0PVO3R+LySUFGA0Qa3\\\\\\\/WJj\\\\\\\/XagnOtY8aFeRVuQ==\\\u0022,\\\u0022iv\\\u0022:\\\u0022d9SU87HETu\\\\\\\/c1jbGtdtfZA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00223ffde1067cbcfe0a987d6a2008b39554403b91d7b9e962ffa3f324abf1ec9d96\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022l6xURknVZHNIkDQNhKLvTEEHEY+vM5f8iUo3CcVdBWtnt3k0JpFGL9KLbbAC\\\\\\\/muLnD5iPf\\\\\\\/NoLLi3n2mBLDKVmPmS67zuQk4Y1HV8wYAYCmMjXSLo29PPzB+u9OG0eIRR5qmAldo4HkjleZbhdEF5CoXwmIK1NQN3WX8iNBFP+X9n5j6x+5k1fJwtI9DvNTcbeClHNAdfhVcc3R+PEKcJWIE5iCy0b2ur6m8xrECBAFPpH9Yjxzkp88FeZJ6Rmx5o60bv2cWxdB9Lq2avpPTsCN4lZubBumluHGHXfxh6O5mR++QzIRAuI\\\\\\\/MJwaHNPomwKdpAB+3KMmOC8w2b8zkH7n2Y\\\\\\\/Z054xg\\\\\\\/pszyI0VlbWmEdpCDFTehpjCdtpkgwUS89b4v1j6rebpsFDr9ED0aNqvZptGW4iUTHemY+KTlx\\\\\\\/v1uEqZSk2vHhTPGwLDWeujj\\\\\\\/SPPUVFgfJwG1wJPjEvPU6pJvV1pMnwo+ivCP76Vsf7xoh0U\\\\\\\/qe\\\\\\\/hukP+SAn+dZ1qwQEJfIdxGM\\\\\\\/EEuttgQ3ai3nVjsyCac+1n8mqVky+iR9N\\\\\\\/id3jit37jCLygSVzTOGv6BXYk46suFcymyyqoIbwmkiIsSOLShzQWv13TVn57ZhMvKuzDvqw+8JLM3I5\\\\\\\/9v9ATTDgIASBpu22BnPjcoL3YeD120tyMgkbH8tg4K3Lx5Is+A3CwJLytTUKusB6yXNe45mtYwBxQLqLqyDflwUVSddK0Vd75YBg6Jh0da47xIv2MB08p+cHNYUSo5QJuudVVFLgJYGDcKX\\\\\\\/FrEfzqzPrvjhZe+hBwRMlSoUtmA4ET7OsKsQWirtXs75OWHV4l1QPs\\\\\\\/aQy\\\\\\\/DoycRcuiyKoNQH2rpX15ylKzpvYP+o5vCyCUND8QtdVcj99deyX3ft1heTM2+31oCDfhI\\\\\\\/\\\\\\\/a2OkRq+pn2P93IX5y8QWHLo7aPx6j4La6ZHyfAZv0HHoZuTOYSyiHdig40zGoWWW8S81cj3pdyvixNBA3H0EFfm40hXttQoOtOPEcHpjdO2jKh8W2R813uNtha\\\\\\\/Ns7Qbxuc9meawWRA+GtCdo2zaOdXnpwhZwL5iX0gpM6V04ydliYyJN7l7O9+iNiIHh9MMCDUFSjJWfXQaPELvuSoLVD6CPkz2a9sGQX5aLdjBsXrqsN74Bqyz8OTvD+Ha7T5kRmit0PmGCzldWw4VSFe2T\\\\\\\/5tcyzxEBhsAoJZ2yAJtMoJqGu4SyXYtkeIEocS2tmf5Nna1a7JJwdrTIeia8oHKlc5q3tx76eizPHg7q8WRa6z9Hkl03CD93JS\\\\\\\/mXcSLaorF5BtJX6a28wMnnFoNryHhq89naNFtcTsA+N7uZh9h0\\\\\\\/hJ2WXRp99ivFz9DEb54OsSuv+bDbOSH93PHVdU7bOZDjsdjG+rTAhNZNqjKsQwylf8WMdHG+k9jifQaJkOTLx4GuMIovjAmF9FBqRWs3XfOVv8WZ722Z1rcPqzHQVgizeEgD+WuSIyLAKLxI7S1enfT55KFmUYSxNsUCBN5TJOFPWUQ54xy2Prn\\\\\\\/M6FpZa7jY+ozumuJG2FJHTMEHzb8KNMkILSRtpJD1lHgOBWFPgEUTktYtQ9eQaWA5SMqUDmYi4Q7+OmNS8AzTs6455kmLKibJ\\\\\\\/A2EJLcsOt3eFQrXFnRW1H3MHg9nWPaQOvI\\\\\\\/A4l3AQyQe7x2ntfmjsJ\\\\\\\/DBTBTR02DEbNUwyEjB91bI6hHnatGDwoTKW+wO+9eLGz7EHP28eUUgQwPTKgD14FPy1xBfqKKM8YL3hsF\\\\\\\/RV08pNJdzEqZHxrqvAYX2jq1mXXwzwhj\\\\\\\/u5ySZR9K1XEifZBSzwHGFTTa7EEGLmOMEDg8KizpigZ4G1IjeKkxs2+V\\\\\\\/5\\\\\\\/S7GdH3JU2bU03EJ\\\\\\\/voyNKnUpqgRS+4fg2LVntD4dyNwNxA8M8dS7\\\\\\\/uTaThXRRWQ1vxAMANjXZIJdVrNG0IvPmSott4n81gcoIXRd8l5N3qSoXrQO2eU2XcqEULQICVdMLfvAolx4Bc2gdG7\\\\\\\/2JY2r6mw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022YGVgZhTdPvoHz2WS\\\\\\\/z+04g==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002209301f88f885fd5a4ea77cdced28002e26834b032494bb86cb5b3c9ac63dcd58\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459935 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/8/1395" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tPglyp2v1dVyBnfmTPovTAh56pkQXPf4psCDx4SxMYQ" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/8/1395" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-CJZ81Ss-SDEMYt0uob34VIv9krWZLXU2mBRE7W1HJ2E" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/8/1385" title="Validation of Six Genetic Determinants of Susceptibility to Estrogen-Induced Mammary Cancer in the Rat and Assessment of Their Relevance to Breast Cancer Risk in Humans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/8/1407" title="Population Dynamics and Evolutionary History of the Weedy Vine Ipomoea hederacea in North America" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459935" id="node459935" data-pisa="ggg;4/8/1395" data-pisa-master="ggg;g3.114.011197" data-apath="/ggg/4/8/1395.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Fluorescence <em>In Situ</em> Hybridization and Optical Mapping to Correct Scaffold Arrangement in the Tomato Genome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Lindsay A. Shearer</span>, <span class="highwire-citation-author" data-delta="1">Lorinda K. Anderson</span>, <span class="highwire-citation-author" data-delta="2">Hans de Jong</span>, <span class="highwire-citation-author" data-delta="3">Sandra Smit</span>, <span class="highwire-citation-author" data-delta="4">José Luis Goicoechea</span>, <span class="highwire-citation-author" data-delta="5">Bruce A. Roe</span>, <span class="highwire-citation-author" data-delta="6">Axin Hua</span>, <span class="highwire-citation-author" data-delta="7">James J. Giovannoni</span> and <span class="highwire-citation-author" data-delta="8">Stephen M. Stack</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1395-1405;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.011197 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459935" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Lindsay A. Shearer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Colorado State University, Fort Collins, Colorado 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lindsay%2BA.%2BShearer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shearer%20LA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALindsay%2BA.%2BShearer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lorinda K. Anderson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Colorado State University, Fort Collins, Colorado 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lorinda%2BK.%2BAnderson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Anderson%20LK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALorinda%2BK.%2BAnderson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Hans de Jong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Genetics, Wageningen University and Research Centre (WUR), Droevendaalsesteeg 1, 6708 PB Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hans%2Bde%2BJong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Jong%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHans%2Bde%2BJong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Sandra Smit </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratory of Bioinformatics, WUR, Droevendaalsesteeg1, 6708 PB Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sandra%2BSmit%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smit%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASandra%2BSmit%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">José Luis Goicoechea </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Arizona Genomics Institute, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BLuis%2BGoicoechea%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goicoechea%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJos%25C3%25A9%2BLuis%2BGoicoechea%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Bruce A. Roe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Chemistry and Biochemistry, Stephenson Research and Technology Center, University of Oklahoma, Norman, Oklahoma 73019</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bruce%2BA.%2BRoe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Roe%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABruce%2BA.%2BRoe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Axin Hua </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Chemistry and Biochemistry, Stephenson Research and Technology Center, University of Oklahoma, Norman, Oklahoma 73019</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Axin%2BHua%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hua%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAxin%2BHua%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">James J. Giovannoni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Plant Biology, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BJ.%2BGiovannoni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giovannoni%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BJ.%2BGiovannoni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Stephen M. Stack </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Colorado State University, Fort Collins, Colorado 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BM.%2BStack%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stack%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStephen%2BM.%2BStack%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Stephen.Stack@colostate.edu" class="" data-icon-position="" data-hide-link-title="0">Stephen.Stack@colostate.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/8/1395" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459935" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459935/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/8/1395.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459935" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459935/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/8/1395.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459935" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459935/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/8/1395.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459935" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459935/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/8/1395.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459935" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459935/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">The order and orientation (arrangement) of all 91 sequenced scaffolds in the 12 pseudomolecules of the recently published tomato (<em>Solanum lycopersicum</em>, 2<em>n </em>= 2<em>x =</em> 24) genome sequence were positioned based on marker order in a high-density linkage map. Here, we report the arrangement of these scaffolds determined by two independent physical methods, bacterial artificial chromosome–fluorescence <em>in situ</em> hybridization (BAC-FISH) and optical mapping. By localizing BACs at the ends of scaffolds to spreads of tomato synaptonemal complexes (pachytene chromosomes), we showed that 45 scaffolds, representing one-third of the tomato genome, were arranged differently than predicted by the linkage map. These scaffolds occur mostly in pericentric heterochromatin where 77% of the tomato genome is located and where linkage mapping is less accurate due to reduced crossing over. Although useful for only part of the genome, optical mapping results were in complete agreement with scaffold arrangement by FISH but often disagreed with scaffold arrangement based on the linkage map. The scaffold arrangement based on FISH and optical mapping changes the positions of hundreds of markers in the linkage map, especially in heterochromatin. These results suggest that similar errors exist in pseudomolecules from other large genomes that have been assembled using only linkage maps to predict scaffold arrangement, and these errors can be corrected using FISH and/or optical mapping. Of note, BAC-FISH also permits estimates of the sizes of gaps between scaffolds, and unanchored BACs are often visualized by FISH in gaps between scaffolds and thus represent starting points for filling these gaps.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/tomato" class="hw-term hw-article-keyword hw-article-keyword-tomato" rel="nofollow">tomato</a></li><li class="kwd"><a href="/keyword/genome" class="hw-term hw-article-keyword hw-article-keyword-genome" rel="nofollow">genome</a></li><li class="kwd"><a href="/keyword/scaffolds" class="hw-term hw-article-keyword hw-article-keyword-scaffolds" rel="nofollow">scaffolds</a></li><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/fluorescence-situ-hybridization" class="hw-term hw-article-keyword hw-article-keyword-fluorescence-in-situ-hybridization" rel="nofollow">fluorescence <em xmlns:default="http://www.w3.org/1999/xhtml">in situ</em> hybridization</a></li><li class="kwd"><a href="/keyword/optical-mapping" class="hw-term hw-article-keyword hw-article-keyword-optical-mapping" rel="nofollow">optical mapping</a></li></ul><p id="p-4">The recently sequenced tomato (<em>Solanum lycopersicum</em>, 2<em>n</em> = 2<em>x</em> = 24) genome consists of 12 DNA pseudomolecules corresponding to the 12 tomato chromosomes (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">The Tomato Genome Consortium 2012</a>). Each pseudomolecule is a linear series of sequenced DNA scaffolds interrupted by gaps of unknown size. Ideally, the arrangement (order and orientation) of scaffolds from the head of a pseudomolecule (end of the short arm of a chromosome, starting with telomere sequence) to the tail of the pseudomolecule (the end of the long arm of a chromosome, ending with telomere sequence) is supposed to be the same as the DNA double helix that runs the length of the corresponding chromosome.</p><p id="p-5">Two methods were used to arrange scaffolds in tomato pseudomolecules (<a id="xref-ref-48-2" class="xref-bibr" href="#ref-48">The Tomato Genome Consortium 2012</a>). One relies on identifying mapped molecular markers in scaffolds and then ordering and orienting scaffolds according to the locations of these markers in a high-resolution linkage map. The other relies on fluorescence <em>in situ</em> hybridization (FISH) of bacterial artificial chromosomes (BACs) to localize scaffold DNA to spreads of synaptonemal complexes (SCs = pachytene chromosomes) and then ordering and orienting the scaffolds according to the arrangement of the fluorescent signals on SCs. At the time of publication, disagreements in scaffold arrangements based on the two methods were resolved in favor of the Kazusa EXPEN 2000 linkage map. This was appropriate because most of the needed FISH localizations were not yet available. However, now the FISH localizations have been completed, and we find that FISH and the linkage map scaffold arrangements disagree for one-third of the genome. Most of the discrepancies occur in pericentric heterochromatin that includes 77% of the tomato genome and approximately 10% of tomato’s estimated 35,000 nuclear genes (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Peterson <em>et al.</em> 1996</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Van der Hoeven <em>et al.</em> 2002</a>; <a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Wang <em>et al.</em> 2006b</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Peters <em>et al.</em> 2009</a>; <a id="xref-ref-48-3" class="xref-bibr" href="#ref-48">The Tomato Genome Consortium 2012</a>). It is significant that when scaffolds are arranged using an independent physical method called optical mapping (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Dong <em>et al.</em> 2013</a>), the results are completely compatible with FISH-based arrangements, while often contradicting linkage map-based arrangements. FISH and optical mapping are based on actual visualization of relative sequence locations, and thus are more likely to reflect biological reality than linkage mapping (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Peterson 2014</a>). Consequently, these results suggest that similar problems are likely to occur in other genomes where scaffold arrangements were based exclusively on linkage maps.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Tomato plants</h3><p id="p-6"><em>Solanum lycopersicum</em>, var. <em>Cherry</em>, accession LA4444, and <em>S. lycopersicum</em>, var. Heinz 1706, and reciprocal hybrids of these two lines were grown from seeds to flowering in a greenhouse. Even though var. Heinz 1706 was used for sequencing, almost all BAC-FISH were performed on accession LA4444 because of its characteristics of indeterminate growth and abundant flowering.</p></div><div id="sec-3" class="subsection"><h3>Spreading tomato SCs for FISH and electron microscopy</h3><p id="p-7">Tomato SC spreads were prepared as described previously (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Stack <em>et al.</em> 2009</a>; <a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Stack and Anderson 2009</a>). Briefly, primary microsporocytes in pachytene were squeezed out of anthers, and cell walls were removed with cytohelicase. The resulting protoplast suspension was mixed with an aqueous hypotonic bursting medium [0.05% v/v IGEPAL CA-630 (Sigma), 0.3% w/v (para)formaldehyde, and 0.001% w/v potassium dextran sulfate] and placed on either a glow-discharged glass microscope slide for FISH or a glow-discharged plastic-coated slide for electron microscopy (EM). Slides were sprayed with aqueous 4% w/v (para)formaldehyde, air-dried, washed briefly in deionized water, air-dried again, and then stored up to 2 yr in sealed boxes at −80° for later FISH.</p></div><div id="sec-4" class="subsection"><h3>Probes for FISH</h3><p id="p-8">To determine which BACs to use for locating the ends of scaffolds by FISH, it was first necessary to determine the order of BAC clones in scaffolds. We used clone-end sequences [T7 and SP6 and 26-nucleotide sequence tags (<a href="http://solgenomics.net">http://solgenomics.net</a>)] from the Whole Genome Profiling physical map of tomato (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Van Oeveren <em>et al.</em> 2011</a>) to map BACs to the genome assembly using BLAST and SOAP (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Altschul <em>et al.</em> 1990</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Li <em>et al.</em> 2008</a>). We also wrote several custom scripts to parse the mapping results and determine reliable BAC locations in the scaffolds (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/011197SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/TableS1.xls">Table S1</a>). Once the order of BAC clones in scaffolds was determined, BACs with sequence at or near the head (toward the end of the short arm of the chromosome) and tail (toward the end of the long arm) of every scaffold were selected to use as probes. Probes were prepared from the tomato <em>Hind</em>III, <em>Mbo</em>I, <em>Eco</em>RI, sheared BAC, and fosmid libraries located at Cornell University (<a href="http://solgenomics.net/">http://solgenomics.net/</a>). Bacteria were grown by standard protocols. BACs and fosmids were isolated using the plasmid kit from AquaPlasmid (MultiTarget Pharmaceuticals, Salt Lake City, Utah) with modifications of the manufacturer’s instructions for BACs and fosmids (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/FileS1.pdf">File S1</a>). Isolated DNAs were labeled with digoxygenin, biotin, or dinitrophenol (DNP) using a nick translation kit according to the manufacturer’s instructions (Roche Applied Science), except digestion time was reduced from the recommended 2 hr to 0.5 hr or 1 hr. With only four exceptions, each end of each scaffold was marked by hybridization of one BAC, which showed the head–tail orientation and location of each scaffold on an SC. The four exceptions were scaffolds shorter than 400 kb, where only one BAC per scaffold was localized to show the position but not the orientation of these scaffolds on SCs. To determine the relative positions of adjacent scaffolds and the gap sizes between them, two BACs, one each from the two scaffold ends facing the gap, were hybridized at the same time. In some cases, four BACs marking the four ends of two adjacent scaffolds were hybridized simultaneously.</p></div><div id="sec-5" class="subsection"><h3>Fluorescence <em>in situ</em> hybridization</h3><p id="p-9">FISH was performed as described by (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Zhong <em>et al.</em> 1996</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Chang <em>et al.</em> 2007</a>). Briefly, air-dried slides were scanned by phase microscopy, good SC spreads were imaged, and their microscope stage coordinates were recorded. The slides were incubated in 45% v/v acetic acid for 1 min, fixed with 1:3 acetic ethanol for 1 min, and then digested with RNase followed by pepsin. After additional fixation in 1% w/v (para)formaldehyde, 20 µl of hybridization mixture [aqueous 2×SSC that was 50% (v/v) formamide, 10% (w/v) sodium dextran sulfate, 0.25% (w/v) sodium dodecyl sulfate with 50–1000 ng of one or more labeled probes and 1–5 µg of unlabeled Cot 100 tomato DNA to block repeated sequences] was placed on each slide, and a cover glass was added. Slides were incubated at 80° on an aluminum block for 2.5 min to denature the DNA, and then slides were incubated at 37° for at least 12 hr to permit hybridization. Slides were then washed three times in aqueous 2×SSC that was 50% v/v formamide at 42° for 80% stringency (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Schwarzacher and Heslop-Harrison 2000</a>). Blocking and antibody incubations were performed at 37° in 1-hr increments with three 3-min washes in a solution of 100 mM Tris, 150 mM NaCl, and 0.05% v/v Tween-20, pH 7.5, after each incubation step. Antibodies (from Jackson ImmunoResearch Laboratories except when otherwise indicated) included mouse anti-biotin (1:100), biotinylated donkey anti-mouse (1:125), rat anti-DNP (1:100, Invitrogen), sheep anti-digoxigenin conjugated to tetramethyl rhodamine isothiocyanate (TRITC; 1:100, Roche), donkey anti-rat conjugated to Dylight 649 (1:100), streptavidin conjugated to fluorescein isothiocyanate (FITC; 1:200), and donkey anti-sheep conjugated to TRITC (1:100). Donkey serum (5%) was added to the blocking buffer when appropriate. After immunolabeling, slides were dehydrated through an ethanol series and air-dried. Cover glasses were mounted with Vectashield (Vector Laboratories) containing 5 µg/ml 4′,6-diamidino-2-phenylindole (DAPI).</p></div><div id="sec-6" class="subsection"><h3>Microscopy</h3><p id="p-10">Microscopy and photography were performed with Leica DM 5000B and DM 5500B microscopes, both equipped for phase contrast and fluorescence microscopy with DAPI, FITC, TRITC, and Cy5 (to detect Dylight 649 fluorophore) filter cubes and zero pixel shift. Images were captured with cooled Hamamatsu monochrome 1344×1044 pixel cameras using IP Lab software (version 4).</p></div><div id="sec-7" class="subsection"><h3>Measuring positions of BACs on SCs</h3><p id="p-11">After hybridization and immunolabeling, SC spreads were located and photographed using filter cubes appropriate for the fluorescent probes. Pseudocolored fluorescent images were overlaid on corresponding phase images to mark sites of hybridization. The location of each BAC was determined as a percentage of arm length from the center of the kinetochore on at least 10 different SC spreads, and the percentiles were averaged. To express BAC location in micrometers, the percentage of the arm location was multiplied by the average length of the SC arm (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Sherman and Stack 1992</a>; <a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Peterson <em>et al.</em> 1996</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Chang <em>et al.</em> 2007</a>). Seven of the 12 SCs were identified reliably using relative length and arm ratio, but within SC group <em>7</em>, <em>9</em>, and <em>10</em> and within SC group <em>5</em> and <em>12</em>, SCs were indistinguishable by light microcopy. When a BAC FISHed to one of these SCs, the BAC in question was FISHed again, along with appropriate marker BACs to verify the SC and arm involved.</p></div><div id="sec-8" class="subsection"><h3>Determining linear DNA density on SCs</h3><p id="p-12">The length in micrometers of a scaffold on an SC can be determined by measuring the distance between FISH foci on the scaffold’s borders. This length divided by the megabases in the scaffold provides an estimate of the amount of DNA per chromatid per micrometer of SC in the type of chromatin involved (the linear density of DNA). However, a correction is required because the amount of DNA between the FISH signals is an overestimate. To explain, the location of a signal is considered to be at the center of a BAC so the amount of DNA between the BAC signals on the borders of the scaffold should be the scaffold size minus half the DNA in the left border BAC and minus half of the DNA in the right border BAC. In addition, sometimes the border BACs were unsuitable for FISH, so a more proximal BAC had to be used. In this case, all of the DNA distal to the BAC signal needs to be subtracted from the scaffold size to determine the amount of DNA between the BAC signals. Knowing the distance in micrometers between BAC signals on the borders of a scaffold, the corrected amount of DNA between the BAC signals, and the type of chromatin between the signals, the linear densities of kinetochores (centromeres), euchromatin, and heterochromatin were estimated (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/FileS1.pdf">File S1</a> for further explanation).</p></div><div id="sec-9" class="subsection"><h3>Determining gap sizes between adjacent scaffolds</h3><p id="p-13">The amount of DNA in each gap was estimated by first measuring the distance in micrometers between the two fluorescent foci on the borders of the gap. This distance was multiplied by the linear DNA density of the chromatin type involved to give the amount of DNA between the signals. However, again, a correction is required because the amount of DNA between the FISH signals is an overestimate of the size of the gap (see above and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/FileS1.pdf">File S1</a>)</p><p id="p-14">Larger gaps between foci could be measured more accurately than smaller gaps due to the resolution of light microscopy. In some cases, the two BAC foci on adjacent scaffolds were too close to each other to measure with confidence, so the gap was arbitrarily assigned a default length of 0.1 µm (just below the resolution of the light microscope). In such small gaps, subtracting scaffold DNA extending toward the gaps from the amount of DNA calculated between the FISH signals sometimes resulted in a negative value for the gap size. Negative values were recorded as 0 kb to indicate very small gaps at or below the resolution of FISH.</p></div><div id="sec-10" class="subsection"><h3>Optical mapping</h3><p id="p-15">DNA was isolated from 100 g of expanding leaves of Heinz 1706 seedlings harvested 21 d after planting, with the final 3 d of growth in the dark to reduce starch content. High-molecular-weight DNA was prepared by first isolating nuclei, embedding them in agarose, and then lysing the nuclei while in agarose to reduce shearing as described by <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Zhang <em>et al.</em> (1994)</a>. Optical mapping was performed by OpGen, Inc. using the protocol previously described for other plant genomes (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Zhou <em>et al.</em> 2009</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Young <em>et al.</em> 2011</a>; <a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Dong <em>et al.</em> 2013</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Chamala <em>et al.</em> 2013</a>). Briefly, this entailed mounting the genomic DNA (∼400–500 kb) onto derivatized glass surfaces using a silastic micro channel system (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Dimalanta <em>et al.</em> 2004</a>). After digesting the immobilized DNA with <em>Bam</em>H1, the linear strands were imaged by a charge-coupled device (CCD) camera at OpGen. The restriction fragment sizes were determined by measuring the distances between the visualized gaps caused by DNA cleavage with the restriction enzyme. The OpGen Mapper software was used to analyze the images channel-by-channel. Nonlinear distorted and short fragments were filtered out prior to generating an ordered restriction map for each genomic DNA molecule. Details of these procedures were previously described (<a id="xref-ref-11-2" class="xref-bibr" href="#ref-11">Dimalanta <em>et al.</em> 2004</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Zhou <em>et al.</em> 2004</a>, <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">2007a</a>). When optical mapping spanned gaps between adjacent sequenced scaffolds, it was possible to determine the order and orientation of the scaffolds relative to each other and to estimate the sizes of their included gaps.</p></div><div id="sec-11" class="subsection"><h3>Electron microscopy of SC spreads</h3><p id="p-16">SCs on plastic coated slides were prepared as described by <a id="xref-ref-41-2" class="xref-bibr" href="#ref-41">Stack and Anderson (2009)</a>. Briefly, SC spreads were digested with DNase I and then fixed in a combination of 2% formaldehyde and 2% glutaraldehyde. After washing, SCs were stained in alcoholic phosphotungstic acid, washed, and dried. Copper EM grids were placed over SC spreads. The grids and plastic film were lifted from the slides and air-dried. SC spreads on grids were examined and photographed using a JEOL 2000 electron microscope.</p></div></div><div class="section results" id="sec-12"><h2 class="">Results</h2><div id="sec-13" class="subsection"><h3>BAC-FISH localizations</h3><p id="p-17">For most individual BACs, FISH on SC spreads resulted in a single, discrete site of hybridization (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). BAC positions were mapped with a high degree of accuracy because there is little or no distortion of SCs in spreads and because BAC locations were averages of measurements from 10 or more different SC spreads. In addition, blocking repeated sequences during hybridization made BAC localizations in heterochromatin often as unique as localizations in euchromatin. Using this technique, we localized 627 BACs to unique sites on tomato SCs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/FigureS1.pdf">Figure S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/TableS2.pdf">Table S2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/TableS3.pdf">Table S3</a>) (see <a href="http://solgenomics.net/cview/map.pl?map_version_id=25">http://solgenomics.net/cview/map.pl?map_version_id=25</a> for identification of every BAC on the idiogram and supporting FISH images). An additional 12 BACs localized to two sites, either on the same SC (five BACS) or to two different SCs (seven BACs) [<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/FigureS2.pdf">Figure S2</a> and Figure 8 in (<a id="xref-ref-42-2" class="xref-bibr" href="#ref-42">Stack <em>et al.</em> 2009</a>)].</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/8/1395/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="BAC-FISH on tomato SC spreads is effective in both euchromatin and heterochromatin. (A) Digitally reversed, phase contrast image of a complete set of tomato SCs (pachytene chromosomes). Kinetochores appear as fuzzy white disks/ellipses approximately 1 &#x3BC;m in diameter on the SCs. SC in distal euchromatin is relatively thick compared with SC in proximal pericentric heterochromatin. The kinetochore of SC 12 is marked &#x201C;12.&#x201D; (B) Enlarged, digitally reversed, phase image of the same SC 12 illustrated in (A) showing FISH localization of two BACs. The approximate borders between distal euchromatin and pericentric heterochromatin are marked with transverse white lines in each arm. The location of BAC LE_HBa0017P17 is indicated by a green focus in distal euchromatin of the long arm, and the location of BAC SL_MboI0038L04 is indicated by a red focus in pericentric heterochromatin of the short arm. The bar in (A) represents 10 &#xB5;m, and the bar in (B) represents 2 &#xB5;m." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-443195453" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;BAC-FISH on tomato SC spreads is effective in both euchromatin and heterochromatin. (A) Digitally reversed, phase contrast image of a complete set of tomato SCs (pachytene chromosomes). Kinetochores appear as fuzzy white disks/ellipses approximately 1 &#x3BC;m in diameter on the SCs. SC in distal euchromatin is relatively thick compared with SC in proximal pericentric heterochromatin. The kinetochore of SC &lt;em&gt;12&lt;/em&gt; is marked &#x201C;12.&#x201D; (B) Enlarged, digitally reversed, phase image of the same SC &lt;em&gt;12&lt;/em&gt; illustrated in (A) showing FISH localization of two BACs. The approximate borders between distal euchromatin and pericentric heterochromatin are marked with transverse white lines in each arm. The location of BAC LE_HBa0017P17 is indicated by a green focus in distal euchromatin of the long arm, and the location of BAC SL_&lt;em&gt;Mbo&lt;/em&gt;I0038L04 is indicated by a red focus in pericentric heterochromatin of the short arm. The bar in (A) represents 10 &#xB5;m, and the bar in (B) represents 2 &#xB5;m.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/8/1395/F1.medium.gif" width="440" height="203"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/8/1395/F1.medium.gif" width="440" height="203"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/8/1395/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/8/1395/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464629" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-18" class="first-child">BAC-FISH on tomato SC spreads is effective in both euchromatin and heterochromatin. (A) Digitally reversed, phase contrast image of a complete set of tomato SCs (pachytene chromosomes). Kinetochores appear as fuzzy white disks/ellipses approximately 1 μm in diameter on the SCs. SC in distal euchromatin is relatively thick compared with SC in proximal pericentric heterochromatin. The kinetochore of SC <em>12</em> is marked “12.” (B) Enlarged, digitally reversed, phase image of the same SC <em>12</em> illustrated in (A) showing FISH localization of two BACs. The approximate borders between distal euchromatin and pericentric heterochromatin are marked with transverse white lines in each arm. The location of BAC LE_HBa0017P17 is indicated by a green focus in distal euchromatin of the long arm, and the location of BAC SL_<em>Mbo</em>I0038L04 is indicated by a red focus in pericentric heterochromatin of the short arm. The bar in (A) represents 10 µm, and the bar in (B) represents 2 µm.</p><div class="sb-div caption-clear"></div></div></div><p id="p-19">We also determined the locations of certain repeated sequences on SCs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/FigureS1.pdf">Figure S1</a>). Telomeric sequence was observed at the ends of every chromosome, mitochondrial DNA at a distinct site on the long arm of chromosome <em>11</em>, 45S rDNA on the distal half of the short arm of chromosome <em>2</em>, and 5S rDNA near the kinetochore on the short arm of chromosome <em>1</em>. We also found several BACs (<em>e.g.</em>, SL_<em>Mbo</em>I0034IOA, <em>Mbo</em>I034I08, <em>Eco</em>RI001K05, <em>Hin</em>dIII074704) that all hybridize to the same sites on chromosomes <em>1</em>, <em>2</em>, <em>3</em>, <em>6</em>, <em>8</em>, <em>11</em>, as well as throughout the nucleolar organizer region (NOR) on chromosome <em>2</em>. This hybridization pattern is probably due to a satellite repeat consisting of 45S rDNA intergenic spacer sequence (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Jo <em>et al.</em> 2009</a>). All of these sites are in heterochromatin, and many are located in gaps between scaffolds.</p></div><div id="sec-14" class="subsection"><h3>Ordering and orienting scaffolds in pseudomolecules by FISH</h3><p id="p-20">Using FISH to localize BACs with sequence at or near the head and tail ends of most scaffolds, we determined the location, order, and head–tail orientation of 87 of the 91 scaffolds. However, only one BAC was localized for each of the four scaffolds that were &lt;400 kb, so order, but not orientation, was determined for these (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/TableS4.pdf">Table S4</a>). <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3</a> shows an example of simultaneous FISH localizations of four BACs marking the head and tail ends of two adjacent scaffolds on the short arm of chromosome <em>3</em>. Because the chromatin is somewhat dispersed laterally from the SC during the spreading procedure, FISH signals may extend out to either side of the SC and/or may be located directly on the SC (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1</a> and <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3</a>). Scaffold numbering (1, 2, 3, <em>etc</em>.) for each pseudomolecule is based on the Kazusa EXPEN 2000 linkage map starting from the end of the short arm (head) of the pseudomolecule (<a id="xref-ref-48-4" class="xref-bibr" href="#ref-48">The Tomato Genome Consortium 2012</a>). To facilitate comparisons, these scaffold numbers were also used in pseudomolecules showing FISH-based scaffold orders. Based on FISH, scaffolds 1 and 4 have the same head–tail orientation, but a different order from that based on the linkage map. Details of the differences between linkage map–based <em>vs.</em> FISH-based pseudomolecules are given in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/TableS4.pdf">Table S4</a> for all 12 chromosomes and are diagrammed in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/8/1395/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Idiogram of the 12 tomato pachytene SCs (bivalent chromosomes) with FISH localizations of selected BACs. SCs are represented by vertical lines with thinner black segments in distal euchromatin and thicker gray segments in pericentric heterochromatin. The distal half of the short arm of SC 2 is the heterochromatic nucleolus organizer. Kinetochores (at centromeres) are represented by black disks, and gray dots on SCs 4 and 8 represent chromomeres. Horizontal blue lines are localization sites of BACs at or near the ends of scaffolds (Table S4). Brackets to the left of each SC show the chromosomal locations and boundaries of scaffolds. Spaces between brackets are gaps in sequencing and/or assembly between scaffolds. Some scaffolds are so small that they appear only as lines in gaps, and some gaps are so small that no space is visible between adjacent brackets. Horizontal red lines are localization sites of BACs that are not assigned to any of the 12 pseudomolecules, i.e., chromosome 0 BACs (Table S10). Note that most, but not all, unassigned BACs localize to gaps between scaffolds." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-443195453" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Idiogram of the 12 tomato pachytene SCs (bivalent chromosomes) with FISH localizations of selected BACs. SCs are represented by vertical lines with thinner black segments in distal euchromatin and thicker gray segments in pericentric heterochromatin. The distal half of the short arm of SC &lt;em&gt;2&lt;/em&gt; is the heterochromatic nucleolus organizer. Kinetochores (at centromeres) are represented by black disks, and gray dots on SCs &lt;em&gt;4&lt;/em&gt; and &lt;em&gt;8&lt;/em&gt; represent chromomeres. Horizontal blue lines are localization sites of BACs at or near the ends of scaffolds (Table S4). Brackets to the left of each SC show the chromosomal locations and boundaries of scaffolds. Spaces between brackets are gaps in sequencing and/or assembly between scaffolds. Some scaffolds are so small that they appear only as lines in gaps, and some gaps are so small that no space is visible between adjacent brackets. Horizontal red lines are localization sites of BACs that are not assigned to any of the 12 pseudomolecules, &lt;em&gt;i.e.&lt;/em&gt;, chromosome &lt;em&gt;0&lt;/em&gt; BACs (Table S10). Note that most, but not all, unassigned BACs localize to gaps between scaffolds.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/8/1395/F2.medium.gif" width="440" height="321"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/8/1395/F2.medium.gif" width="440" height="321"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/8/1395/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/8/1395/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464680" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-21" class="first-child">Idiogram of the 12 tomato pachytene SCs (bivalent chromosomes) with FISH localizations of selected BACs. SCs are represented by vertical lines with thinner black segments in distal euchromatin and thicker gray segments in pericentric heterochromatin. The distal half of the short arm of SC <em>2</em> is the heterochromatic nucleolus organizer. Kinetochores (at centromeres) are represented by black disks, and gray dots on SCs <em>4</em> and <em>8</em> represent chromomeres. Horizontal blue lines are localization sites of BACs at or near the ends of scaffolds (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/TableS4.pdf">Table S4</a>). Brackets to the left of each SC show the chromosomal locations and boundaries of scaffolds. Spaces between brackets are gaps in sequencing and/or assembly between scaffolds. Some scaffolds are so small that they appear only as lines in gaps, and some gaps are so small that no space is visible between adjacent brackets. Horizontal red lines are localization sites of BACs that are not assigned to any of the 12 pseudomolecules, <em>i.e.</em>, chromosome <em>0</em> BACs (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/TableS10.pdf">Table S10</a>). Note that most, but not all, unassigned BACs localize to gaps between scaffolds.</p><div class="sb-div caption-clear"></div></div></div><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/8/1395/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Representative example of simultaneous FISH localization of four BACs to define the borders of two scaffolds and the gap between them on SC 3. Because scaffold numbering is based on the linkage map, the two adjacent scaffolds are numbered 1 and 4, whereas scaffolds 2 and 3 were localized by FISH to positions on the long arm of chromosome 3 (see Figure 4). (A) Reversed phase image of a complete SC set with SC 3 marked &#x201C;3&#x201D; at its kinetochore. (B) Fluorescent image of the same SC set in (A) with DNA stained blue with DAPI and showing colored foci that are FISH localizations of BACs in the distal euchromatin of the short arm of SC 3. (C) Enlarged reversed phase contrast image of the short arm of SC 3 with the BAC-FISH localizations shown in (B). The upper lobe of the white, dumbbell-shaped structure to the right is the kinetochore, while the lower lobe is debris visible by phase contrast microscopy. BAC SL_s0009C01 (purple) is at the head (H = toward the end of the short arm) and BAC SL_s0086D22 (green) is at the tail (T = toward the end of the long arm) of scaffold 1 (SL2.40sc04439). BAC SL_s0018K15 (red) is at the head and BAC SL_s0002G24 (turquoise) is at the tail of the adjacent scaffold 4 (SL2.40sc4696). Scaffolds 1 and 4 are in distal euchromatin. The space between the green signals and the red signals is the gap between scaffolds 1 and 4. The purple and the turquoise foci mark the location of DNP-labeled BAC probes that were the same color in the original image but that have been given different pseudo colors here. In the diagram beneath the SC, the thick gray segments labeled 1, 4, 5, and 6 represent scaffolds SL2.40sc04439, SL2.40sc4696, SL2.40sc05330, and SL2.40sc4126, respectively, with their lengths proportional to the amounts of DNA they represent (Table S5). BAC-FISH localizations used to order and orient scaffolds 5 and 6 are not illustrated. Based on FISH, these scaffolds have the same head&#x2013;tail orientation, but a different order from that derived from the linkage map (Figure 4 and Table S4). Gaps between scaffolds are named according to the scaffolds on either side, e.g., 1-4, 4-5, etc., and gap lengths (white lines between the scaffolds) are proportional to the amount of DNA they are estimated to represent (Table S7). The bar in (B) represents 10 &#xB5;m for (A) and (B). In (C), the upper bar represents 1 &#xB5;m in reference to the SC segment, while the lower bar represents 2 Mb in reference to the pseudomolecule." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-443195453" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Representative example of simultaneous FISH localization of four BACs to define the borders of two scaffolds and the gap between them on SC &lt;em&gt;3&lt;/em&gt;. Because scaffold numbering is based on the linkage map, the two adjacent scaffolds are numbered 1 and 4, whereas scaffolds 2 and 3 were localized by FISH to positions on the long arm of chromosome &lt;em&gt;3&lt;/em&gt; (see Figure 4). (A) Reversed phase image of a complete SC set with SC &lt;em&gt;3&lt;/em&gt; marked &#x201C;3&#x201D; at its kinetochore. (B) Fluorescent image of the same SC set in (A) with DNA stained blue with DAPI and showing colored foci that are FISH localizations of BACs in the distal euchromatin of the short arm of SC &lt;em&gt;3&lt;/em&gt;. (C) Enlarged reversed phase contrast image of the short arm of SC &lt;em&gt;3&lt;/em&gt; with the BAC-FISH localizations shown in (B). The upper lobe of the white, dumbbell-shaped structure to the right is the kinetochore, while the lower lobe is debris visible by phase contrast microscopy. BAC SL_s0009C01 (purple) is at the head (H = toward the end of the short arm) and BAC SL_s0086D22 (green) is at the tail (T = toward the end of the long arm) of scaffold 1 (SL2.40sc04439). BAC SL_s0018K15 (red) is at the head and BAC SL_s0002G24 (turquoise) is at the tail of the adjacent scaffold 4 (SL2.40sc4696). Scaffolds 1 and 4 are in distal euchromatin. The space between the green signals and the red signals is the gap between scaffolds 1 and 4. The purple and the turquoise foci mark the location of DNP-labeled BAC probes that were the same color in the original image but that have been given different pseudo colors here. In the diagram beneath the SC, the thick gray segments labeled 1, 4, 5, and 6 represent scaffolds SL2.40sc04439, SL2.40sc4696, SL2.40sc05330, and SL2.40sc4126, respectively, with their lengths proportional to the amounts of DNA they represent (Table S5). BAC-FISH localizations used to order and orient scaffolds 5 and 6 are not illustrated. Based on FISH, these scaffolds have the same head&#x2013;tail orientation, but a different order from that derived from the linkage map (Figure 4 and Table S4). Gaps between scaffolds are named according to the scaffolds on either side, &lt;em&gt;e.g.&lt;/em&gt;, 1-4, 4-5, &lt;em&gt;etc&lt;/em&gt;., and gap lengths (white lines between the scaffolds) are proportional to the amount of DNA they are estimated to represent (Table S7). The bar in (B) represents 10 &#xB5;m for (A) and (B). In (C), the upper bar represents 1 &#xB5;m in reference to the SC segment, while the lower bar represents 2 Mb in reference to the pseudomolecule.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/8/1395/F3.medium.gif" width="440" height="406"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/8/1395/F3.medium.gif" width="440" height="406"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/8/1395/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/8/1395/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464729" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-22" class="first-child">Representative example of simultaneous FISH localization of four BACs to define the borders of two scaffolds and the gap between them on SC <em>3</em>. Because scaffold numbering is based on the linkage map, the two adjacent scaffolds are numbered 1 and 4, whereas scaffolds 2 and 3 were localized by FISH to positions on the long arm of chromosome <em>3</em> (see <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). (A) Reversed phase image of a complete SC set with SC <em>3</em> marked “3” at its kinetochore. (B) Fluorescent image of the same SC set in (A) with DNA stained blue with DAPI and showing colored foci that are FISH localizations of BACs in the distal euchromatin of the short arm of SC <em>3</em>. (C) Enlarged reversed phase contrast image of the short arm of SC <em>3</em> with the BAC-FISH localizations shown in (B). The upper lobe of the white, dumbbell-shaped structure to the right is the kinetochore, while the lower lobe is debris visible by phase contrast microscopy. BAC SL_s0009C01 (purple) is at the head (H = toward the end of the short arm) and BAC SL_s0086D22 (green) is at the tail (T = toward the end of the long arm) of scaffold 1 (SL2.40sc04439). BAC SL_s0018K15 (red) is at the head and BAC SL_s0002G24 (turquoise) is at the tail of the adjacent scaffold 4 (SL2.40sc4696). Scaffolds 1 and 4 are in distal euchromatin. The space between the green signals and the red signals is the gap between scaffolds 1 and 4. The purple and the turquoise foci mark the location of DNP-labeled BAC probes that were the same color in the original image but that have been given different pseudo colors here. In the diagram beneath the SC, the thick gray segments labeled 1, 4, 5, and 6 represent scaffolds SL2.40sc04439, SL2.40sc4696, SL2.40sc05330, and SL2.40sc4126, respectively, with their lengths proportional to the amounts of DNA they represent (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/TableS5.pdf">Table S5</a>). BAC-FISH localizations used to order and orient scaffolds 5 and 6 are not illustrated. Based on FISH, these scaffolds have the same head–tail orientation, but a different order from that derived from the linkage map (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/TableS4.pdf">Table S4</a>). Gaps between scaffolds are named according to the scaffolds on either side, <em>e.g.</em>, 1-4, 4-5, <em>etc</em>., and gap lengths (white lines between the scaffolds) are proportional to the amount of DNA they are estimated to represent (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/TableS7.pdf">Table S7</a>). The bar in (B) represents 10 µm for (A) and (B). In (C), the upper bar represents 1 µm in reference to the SC segment, while the lower bar represents 2 Mb in reference to the pseudomolecule.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/8/1395/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Diagrammatic representations of tomato SCs 1&#x2013;6 with corresponding linkage map&#x2013;based pseudomolecules and FISH-based pseudomolecules. (A) SCs are represented as horizontal black lines with solid black ellipses indicating the positions of kinetochores at centromeres. All SCs are oriented with short arms to the left. Pericentric heterochromation is represented as a gray layer to either side of kinetochores, and the approximate location of this heterochromatin is projected onto the DNA pseudomolecules below. The NOR heterochromatin in the distal half of the short arm of chromosome 2 is represented by darker gray. In the pairs of pseudomolecules with scaffolds arranged according to the linkage map (B) and according to FISH (C), thick segments represent sequenced scaffolds and gaps between scaffolds are represented by black lines. Arrowheads in the scaffolds indicate the orientation of each scaffold according to the linkage map from head to tail (i.e., arrowheads point toward the end of the long arm of the chromosome), and numbers above scaffolds indicate their order from the head to the tail of the pseudomolecule based on the linkage map (see Table S4 for scaffold names). Gaps in map-based pseudomolecules are shown of equal length because the linkage map does not contribute to estimates of gap sizes. Scaffolds that show different arrangements between the linkage-based and FISH-based pseudomolecules are indicated by colors other than gray, and scaffolds that show different orientations have red arrowheads. In FISH-based pseudomolecules, changes in scaffold order (position) relative to that predicted from the linkage map are obvious, changes in orientation are shown by red arrowheads that are reversed in direction, and the lengths of gaps are proportional to the amount of DNA estimated to be in the gaps (see Table S7 for gap sizes). The upper bar for each pachytene chromosome (A) represents 1 &#x3BC;m of SC length, and the lower bar for each pair of pseudomolecules (B and C) represents 5 Mb." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-443195453" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Diagrammatic representations of tomato SCs &lt;em&gt;1&#x2013;6&lt;/em&gt; with corresponding linkage map&#x2013;based pseudomolecules and FISH-based pseudomolecules. (A) SCs are represented as horizontal black lines with solid black ellipses indicating the positions of kinetochores at centromeres. All SCs are oriented with short arms to the left. Pericentric heterochromation is represented as a gray layer to either side of kinetochores, and the approximate location of this heterochromatin is projected onto the DNA pseudomolecules below. The NOR heterochromatin in the distal half of the short arm of chromosome &lt;em&gt;2&lt;/em&gt; is represented by darker gray. In the pairs of pseudomolecules with scaffolds arranged according to the linkage map (B) and according to FISH (C), thick segments represent sequenced scaffolds and gaps between scaffolds are represented by black lines. Arrowheads in the scaffolds indicate the orientation of each scaffold according to the linkage map from head to tail (&lt;em&gt;i.e.&lt;/em&gt;, arrowheads point toward the end of the long arm of the chromosome), and numbers above scaffolds indicate their order from the head to the tail of the pseudomolecule based on the linkage map (see Table S4 for scaffold names). Gaps in map-based pseudomolecules are shown of equal length because the linkage map does not contribute to estimates of gap sizes. Scaffolds that show different arrangements between the linkage-based and FISH-based pseudomolecules are indicated by colors other than gray, and scaffolds that show different orientations have red arrowheads. In FISH-based pseudomolecules, changes in scaffold order (position) relative to that predicted from the linkage map are obvious, changes in orientation are shown by red arrowheads that are reversed in direction, and the lengths of gaps are proportional to the amount of DNA estimated to be in the gaps (see Table S7 for gap sizes). The upper bar for each pachytene chromosome (A) represents 1 &#x3BC;m of SC length, and the lower bar for each pair of pseudomolecules (B and C) represents 5 Mb.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/8/1395/F4.medium.gif" width="440" height="370"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/8/1395/F4.medium.gif" width="440" height="370"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/8/1395/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/8/1395/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464757" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-23" class="first-child">Diagrammatic representations of tomato SCs <em>1–6</em> with corresponding linkage map–based pseudomolecules and FISH-based pseudomolecules. (A) SCs are represented as horizontal black lines with solid black ellipses indicating the positions of kinetochores at centromeres. All SCs are oriented with short arms to the left. Pericentric heterochromation is represented as a gray layer to either side of kinetochores, and the approximate location of this heterochromatin is projected onto the DNA pseudomolecules below. The NOR heterochromatin in the distal half of the short arm of chromosome <em>2</em> is represented by darker gray. In the pairs of pseudomolecules with scaffolds arranged according to the linkage map (B) and according to FISH (C), thick segments represent sequenced scaffolds and gaps between scaffolds are represented by black lines. Arrowheads in the scaffolds indicate the orientation of each scaffold according to the linkage map from head to tail (<em>i.e.</em>, arrowheads point toward the end of the long arm of the chromosome), and numbers above scaffolds indicate their order from the head to the tail of the pseudomolecule based on the linkage map (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/TableS4.pdf">Table S4</a> for scaffold names). Gaps in map-based pseudomolecules are shown of equal length because the linkage map does not contribute to estimates of gap sizes. Scaffolds that show different arrangements between the linkage-based and FISH-based pseudomolecules are indicated by colors other than gray, and scaffolds that show different orientations have red arrowheads. In FISH-based pseudomolecules, changes in scaffold order (position) relative to that predicted from the linkage map are obvious, changes in orientation are shown by red arrowheads that are reversed in direction, and the lengths of gaps are proportional to the amount of DNA estimated to be in the gaps (see <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/TableS7.pdf">Table S7</a> for gap sizes). The upper bar for each pachytene chromosome (A) represents 1 μm of SC length, and the lower bar for each pair of pseudomolecules (B and C) represents 5 Mb.</p><div class="sb-div caption-clear"></div></div></div><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/8/1395/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Diagrammatic representations of tomato SCs 7&#x2013;12 (with corresponding linkage map&#x2013;based pseudomolecules and FISH-based pseudomolecules below. This figure is a continuation of Figure 4 (see the legend of Figure 4 for details)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-443195453" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Diagrammatic representations of tomato SCs &lt;em&gt;7&#x2013;12&lt;/em&gt; (with corresponding linkage map&#x2013;based pseudomolecules and FISH-based pseudomolecules below. This figure is a continuation of Figure 4 (see the legend of Figure 4 for details).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/8/1395/F5.medium.gif" width="392" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/8/1395/F5.medium.gif" width="392" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/8/1395/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/8/1395/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464701" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-24" class="first-child">Diagrammatic representations of tomato SCs <em>7–12</em> (with corresponding linkage map–based pseudomolecules and FISH-based pseudomolecules below. This figure is a continuation of <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4</a> (see the legend of <a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4</a> for details).</p><div class="sb-div caption-clear"></div></div></div><p id="p-25">In general, scaffolds located in euchromatic regions of the chromosomes are likely to be in the same arrangement in both linkage map–based and FISH-based pseudomolecules, while scaffolds located in pericentric heterochromatin often differ between the two pseudomolecules. In some cases, scaffolds found in one arm by linkage mapping are observed to occur in the other arm by FISH, and scaffolds expected to include the centromere (as indicated by the physical presence of the kinetochore) differ in the two pseudomolecules. While centromere positions in linkage map–based scaffolds are difficult to assign with confidence, the exact locations of kinetochores relative to scaffolds were determined by FISH (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/FigureS1.pdf">Figure S1</a>). Details of differences between the linkage map-based and FISH-based pseudomolecules for each chromosome are described below.</p><ul class="list-simple " id="list-1"><li id="list-item-1"><p id="p-26"><strong>Chromosome <em>1:</em></strong> Nine scaffolds. Four of the smaller scaffolds (2, 3, 7, 8) differ in order and scaffold 2 also differs in orientation. Scaffolds 2 and 3 are within pericentric heterochromatin while scaffolds 7 and 8 are in distal euchromatin. Scaffold 7 is very small (400 kb), and its orientation was not determined by FISH. The kinetochore is in the gap between scaffolds 2 and 3.</p></li><li id="list-item-2"><p id="p-27"><strong>Chromosome <em>2:</em></strong> Seven scaffolds. All six scaffolds in or near pericentric heterochromatin (1–6) differ in order and two (4 and 6) also differ in orientation. The kinetochore is in scaffold 2. The distal half of the short arm is composed of heterochromatin (including the NOR), and the DNA in this segment is unsequenced and/or unassembled. Although no estimate of its DNA content was made in the linkage map–based pseudomolecule, the same length is shown in both pseudomolecules to simplify comparison of scaffolds and gaps in the two pseudomolecules. Based on our measurements, we estimate that this 3-μm distal segment includes at least (8.8 Mb/µm × 3 µm =) 26.4 megabases.</p></li><li id="list-item-3"><p id="p-28"><strong>Chromosome <em>3</em>:</strong> Thirteen scaffolds. Nine scaffolds (2–9, 12) differ in order, and two (2 and 3) also differ in orientation. The most notable discrepancies include scaffold 2 that was placed in euchromatin of the short arm by linkage mapping but located by FISH in the heterochromatin of the long arm, scaffold 3 (a small scaffold of only ∼400 kb) that was placed in the short arm near/in the heterochromatin by linkage mapping but located in the euchromatin of the long arm by FISH, and scaffold 12 that was located in the middle of the euchromatic portion of the long arm by linkage mapping but located by FISH in or near the relocated scaffold 2 in the heterochromatin of the long arm. The kinetochore is located in the gap between scaffolds 6 and 9. Possibly because we were unable to adequately suppress repeat sequences in the vicinity of scaffolds 2 and 12, the positions of the tail of scaffold 8, both ends of scaffolds 2 and 12, and the head of scaffold 10 are relatively uncertain and, as a consequence, the sizes of gaps 8-12, 12-2, and 2-10 are uncertain as well. Similar difficulties in localizing BACs in this area were encountered using both spreads of SCs and 1:3 acetic ethanol–fixed pachytene chromosomes (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Chang 2004</a>) from the two tomato varieties <em>Cherry</em> LA4444 and Heinz 1706, so this problem does not appear to be due to a structural difference between the two tomato lines.</p></li><li id="list-item-4"><p id="p-29"><strong>Chromosome <em>4:</em></strong> Six scaffolds. Two scaffolds in heterochromatin (2, 3) differ in both order and orientation by a simple inversion of the sequences. The kinetochore is in the gap between scaffolds 3 and 2.</p></li><li id="list-item-5"><p id="p-30"><strong>Chromosome <em>5:</em></strong> Three scaffolds. The only difference between the two pseudomolecules is the orientation of scaffold 2 that is located in heterochromatin. The kinetochore remains in scaffold 1.</p></li><li id="list-item-6"><p id="p-31"><strong>Chromosome <em>6:</em></strong> Eight scaffolds. Two scaffolds (2, 3), both located in heterochromatin, differ in order, and scaffolds (3 and 5) differ in orientation. The kinetochore is in scaffold 2.</p></li><li id="list-item-7"><p id="p-32"><strong>Chromosome <em>7:</em></strong> Four scaffolds. The arrangement of all four scaffolds is the same in the linkage-based and FISH-based pseudomolecules. The kinetochore is in scaffold 1.</p></li><li id="list-item-8"><p id="p-33"><strong>Chromosome <em>8:</em></strong> Nine scaffolds. Five scaffolds (3–7), all in heterochromatin, differ in order, and scaffolds 3–6 differ in orientation. The small scaffold 7 (200 kb) was localized by only one BAC, so its orientation is not known. The kinetochore is in scaffold 6.</p></li><li id="list-item-9"><p id="p-34"><strong>Chromosome <em>9:</em></strong> Ten scaffolds. Three scaffolds (2–4) differ in order, and scaffold 3 also differs in orientation. Using one fosmid, the tiny scaffold 2 (length not yet determined) was localized by FISH to euchromatin of the long arm compared to its mapped position in heterochromatin of the short arm. The orientation of scaffold 2 was not determined. The kinetochore is mostly in scaffold 4.</p></li><li id="list-item-10"><p id="p-35"><strong>Chromosome <em>10:</em></strong> Six scaffolds. Two scaffolds (3, 4) differ in order with scaffold 4 moving from euchromatin in the long arm to heterochromatin near the kinetochore in the long arm. The kinetochore is in scaffold 4.</p></li><li id="list-item-11"><p id="p-36"><strong>Chromosome <em>11:</em></strong> Six scaffolds. Three scaffolds (2–4) differ in order, and scaffold 3 also differs in orientation. All of these scaffolds are located in heterochromatin. The kinetochore is in scaffold 2.</p></li><li id="list-item-12"><p id="p-37"><strong>Chromosome <em>12:</em></strong> Ten scaffolds. Seven scaffolds (2–8) differ in order, and scaffold 7 also differs in orientation. Notably, scaffolds 5–8 were predicted to be located in euchromatin of the long arm using linkage mapping but were found by FISH to be in the pericentric heterochromatin of the short arm. The kinetochore is in scaffold 5. Unlike any of the other scaffolds, we found an apparent overlap of scaffolds 3 and 5 by FISH on chromosome <em>12</em> from LA4444. Using SC spreads from Heinz 1706 for FISH with the same BAC probes, the apparent overlap disappeared, indicating the presence of a small structural inversion difference between the two lines. The inversion includes the kinetochore, the tail of scaffold 5, and the head of scaffold 3 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/FigureS3.pdf">Figure S3</a>). Even so, a hybrid between LA4444 and Heinz 1706 shows only normal straight synapsis throughout the length of all 12 bivalents, including the presumably nonhomologously synapsed inverted segment around the kinetochore of chromosome <em>12</em> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/FigureS4.pdf">Figure S4</a>). This small inversion was the only structural difference that we identified between the two tomato varieties.</p></li></ul><p id="p-38">Overall, 46 of the scaffolds, mostly in euchromatin and representing 66% (= 500.2 Mb/760.0 Mb) of the assembled genome, had the same arrangement based on the linkage map compared to FISH. The remaining 45 scaffolds, mostly in heterochromatin and representing 34% (259.8 Mb) of the assembled genome, had different arrangements based on the linkage map compared to FISH. Of these, 28 differed only in order, three differed only in orientation, and 14 differed in both order and orientation. We observed no instance in which a scaffold was located on one chromosome by linkage mapping and on another chromosome by FISH.</p></div><div id="sec-15" class="subsection"><h3>Optical mapping to order and orient scaffolds in pseudomolecules</h3><p id="p-39">Optical mapping resulted in 14 superscaffolds bridging two or more adjacent scaffolds (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/TableS5.pdf">Table S5</a>). Ten chromosomes had at least one superscaffold, and these superscaffolds were observed in both euchromatin and heterochromatin (<a id="xref-fig-4-6" class="xref-fig" href="#F4">Figure 4</a>, <a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/TableS5.pdf">Table S5</a>). However, the superscaffolds included only 38 of the 91 total scaffolds, with 53 scaffolds resolving as uninformative singles. The 38 optical scaffolds represent approximately 240.5 Mb (32%) of the total 760 Mb of sequenced tomato genome. The order and orientation of all 38 scaffolds arranged by optical mapping are compatible with FISH results, while the arrangement of only 22 of these 38 scaffolds are compatible with map-based results (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/TableS5.pdf">Table S5</a>).</p></div><div id="sec-16" class="subsection"><h3>Determining gap sizes between scaffolds</h3><p id="p-40">We estimated the linear densities of DNA in kinetochores (centromeres), euchromatin, and heterochromatin to be 3.3, 1.5, and 8.8 Mb/µm, respectively (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/TableS6.pdf">Table S6</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/FileS1.pdf">File S1</a>). Using this information together with the measured distance between two adjacent scaffolds, we estimated individual gap sizes to range from 3.2 Mb down to 0 Mb (very small) (<a id="xref-fig-4-7" class="xref-fig" href="#F4">Figure 4</a>, <a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/TableS7.pdf">Table S7</a>). We were able to compare size estimates for 24 gaps using FISH and optical mapping (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/TableS8.pdf">Table S8</a>). The two values differed from 0 kb to 1.3 Mb, with the paired estimates usually being more similar in euchromatin than in heterochromatin.</p><p id="p-41">With FISH we estimate the total amount of DNA in all 79 gaps in the tomato pseudomolecules to be 43.9 Mb (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/TableS7.pdf">Table S7</a>) or approximately (43.9 Mb/919 Mb =) 5% of the genome (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/TableS9.pdf">Table S9</a>). Of this, 34.4 Mb occurs in pericentric heterochromatin, 2.3 Mb occurs in kinetochores, 4.5 Mb occurs in euchromatin/heterochromatin borders, 1.6 Mb occurs in a heterochromatic chromomere, and 1.0 Mb occurs in euchromatin.</p></div><div id="sec-17" class="subsection"><h3>Localizing unassigned (chromosome <em>0</em>) BACs</h3><p id="p-42">Unassigned BACs, also referred to as chromosome <em>0</em> BACs, lack sequence alignment and mapped markers so they could not be reliably assembled into any of the pseudomolecules. Of the 93 unassigned BACs that we FISHed, 75 were found at distinct single sites on SCs, and 70 (93%) of these were located in gaps between scaffolds (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/FigureS1.pdf">Figure S1</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/TableS10.pdf">Table S10</a>). Five unassigned BACs were located within scaffolds and require further characterization to explain their unassigned designation. The remaining 18 unassigned BACs either did not produce a FISH signal or produced diffuse signals over large areas of heterochromatin. All unassigned BACs are listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/TableS11.xls">Table S11</a>.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/8/1395/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="DAPI-stained SC spread showing FISH localizations of an unassigned (chromosome 0) BAC in a gap between BACs at the ends of adjacent scaffolds on SC 9. Arrows indicate the ends of the short (S) and long (L) arms of SC9 as well as the kinetochore (K). BAC SL_MboI0025N23 (red) is at the tail of scaffold SL2.40sc03771 (scaffold 1 in Figure 5), BAC SL_EcoRI0015A15 (green) is at the head of adjacent scaffold SL2.40sc04008 (scaffold 4 in Figure 5), and unassigned BAC SL_MboI0045G03 (white) is between the red and green signals. The inset shows the hybridization signals and kinetochore of a segment of SC 9 on a magnified reversed phase contrast image of the boxed area. The bar for the large image represents 5 &#xB5;m, and the bar for the inset represents 1 &#xB5;m." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-443195453" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;DAPI-stained SC spread showing FISH localizations of an unassigned (chromosome &lt;em&gt;0&lt;/em&gt;) BAC in a gap between BACs at the ends of adjacent scaffolds on SC &lt;em&gt;9&lt;/em&gt;. Arrows indicate the ends of the short (S) and long (L) arms of SC&lt;em&gt;9&lt;/em&gt; as well as the kinetochore (K). BAC SL_&lt;em&gt;Mbo&lt;/em&gt;I0025N23 (red) is at the tail of scaffold SL2.40sc03771 (scaffold 1 in Figure 5), BAC SL_&lt;em&gt;Eco&lt;/em&gt;RI0015A15 (green) is at the head of adjacent scaffold SL2.40sc04008 (scaffold 4 in Figure 5), and unassigned BAC SL_&lt;em&gt;Mbo&lt;/em&gt;I0045G03 (white) is between the red and green signals. The inset shows the hybridization signals and kinetochore of a segment of SC &lt;em&gt;9&lt;/em&gt; on a magnified reversed phase contrast image of the boxed area. The bar for the large image represents 5 &#xB5;m, and the bar for the inset represents 1 &#xB5;m.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/8/1395/F6.medium.gif" width="440" height="381"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/4/8/1395/F6.medium.gif" width="440" height="381"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/8/1395/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/8/1395/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464759" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-43" class="first-child">DAPI-stained SC spread showing FISH localizations of an unassigned (chromosome <em>0</em>) BAC in a gap between BACs at the ends of adjacent scaffolds on SC <em>9</em>. Arrows indicate the ends of the short (S) and long (L) arms of SC<em>9</em> as well as the kinetochore (K). BAC SL_<em>Mbo</em>I0025N23 (red) is at the tail of scaffold SL2.40sc03771 (scaffold 1 in <a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5</a>), BAC SL_<em>Eco</em>RI0015A15 (green) is at the head of adjacent scaffold SL2.40sc04008 (scaffold 4 in <a id="xref-fig-5-5" class="xref-fig" href="#F5">Figure 5</a>), and unassigned BAC SL_<em>Mbo</em>I0045G03 (white) is between the red and green signals. The inset shows the hybridization signals and kinetochore of a segment of SC <em>9</em> on a magnified reversed phase contrast image of the boxed area. The bar for the large image represents 5 µm, and the bar for the inset represents 1 µm.</p><div class="sb-div caption-clear"></div></div></div></div></div><div class="section discussion" id="sec-18"><h2 class="">Discussion</h2><p id="p-44">The Kazusa tomato EXPEN 2000 linkage map was used to arrange scaffolds in the recently published tomato genome sequence (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Shirasawa <em>et al.</em> 2010</a>; <a id="xref-ref-48-5" class="xref-bibr" href="#ref-48">The Tomato Genome Consortium 2012</a>). Using FISH as an independent means of ordering and orienting these scaffolds (<a id="xref-fig-1-3" class="xref-fig" href="#F1">Figure 1</a>, <a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2</a>, and <a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3</a>), we find that 45 of the 91 total scaffolds disagree with the linkage map–based arrangement (<a id="xref-fig-4-8" class="xref-fig" href="#F4">Figure 4</a>, <a id="xref-fig-5-6" class="xref-fig" href="#F5">Figure 5</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/TableS4.pdf">Table S4</a>). Although most of these scaffolds are in heterochromatin, they represent a substantial fraction (34%) of the genome that includes several thousand genes (<a id="xref-ref-53-2" class="xref-bibr" href="#ref-53">Wang <em>et al.</em> 2006b</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Peters <em>et al.</em> 2009</a>; <a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Di Filippo <em>et al.</em> 2012</a>). So what is the justification for using FISH to order and orient scaffolds when there is a disagreement with linkage mapping?</p><div id="sec-19" class="subsection"><h3>FISH is a direct physical means of ordering and orienting scaffolds on chromosomes in both euchromatin and heterochromatin</h3><p id="p-45">A pseudomolecule is a sequencing construct that ideally corresponds to the chromosomal DNA molecule. FISH demonstrates this relationship directly by physically hybridizing scaffold DNA to complementary sites on long pachytene chromosomes. In tomato, BAC-FISH was equally successful in both euchromatin and heterochromatin, <em>i.e.</em>, most BACS carrying DNA from heterochromatic parts of chromosomes hybridized to single sites when repeated sequences were blocked (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Joos <em>et al.</em> 1994</a>; <a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Sadder <em>et al.</em> 2000</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Schubert <em>et al.</em> 2001</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Szinay <em>et al.</em> 2010</a>). This result also indicates that most BAC inserts carried single-copy sequence, which agrees with reports that at least 50% of the DNA in tomato heterochromatin shows single copy reassociation kinetics (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Zamir and Tanksley 1988</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Peterson <em>et al.</em> 1998</a>).</p><p id="p-46">In comparison, the linkage map acts as an intermediary between scaffolds and corresponding chromosomal DNA molecules, so scaffold placement can be no more accurate than the linkage map. Linkage maps are most accurate in regions of the genome that have high rates of recombination, <em>e.g.</em>, distal euchromatin, where map-based scaffold arrangements often agree with FISH and optical mapping-based arrangements. However, crossing over is suppressed in heterochromatin, with the result that accuracy of the linkage maps in heterochromatin is correspondingly low (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Tanksley <em>et al.</em> 1992</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Sherman and Stack 1995</a>). This can result in mistakes in arranging scaffolds and, indeed, most discrepancies in scaffold arrangement occur in pericentric heterochromatin (<a id="xref-fig-4-9" class="xref-fig" href="#F4">Figure 4</a> and <a id="xref-fig-5-7" class="xref-fig" href="#F5">Figure 5</a>). Similar discrepancies in scaffold arrangements have been reported in heterochromatin of a cucumber chromosome and a barley chromosome (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Yang <em>et al.</em> 2012</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Karafiátová <em>et al.</em> 2013</a>). Another problem with arranging scaffolds by linkage maps is that scaffolds must include mapped markers, and this requirement may not be met by some small scaffolds.</p></div><div id="sec-20" class="subsection"><h3>Scaffolds are accurately positioned by FISH on pachytene chromosomes</h3><p id="p-47">In tomato, SCs are more than 10-times longer than corresponding C-metaphase chromosomes, and SC spreads have little or no distortion (<a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Sherman and Stack 1995</a>; <a id="xref-ref-42-3" class="xref-bibr" href="#ref-42">Stack <em>et al.</em> 2009</a>). These two features contribute to precise and reproducible BAC-FISH localizations with SDs of approximately 0.1 µm for the final position of each BAC based on measurements from at least 10 SC spreads. An SC length of 0.1 µm is equivalent to a resolution of approximately (1.5 Mb/µm × 0.1 µm =) 150 kb in euchromatin and (8.8 Mb/µm × 0.1 µm =) 880 kb in heterochromatin. Similar BAC-FISH resolution has been reported for pachytene chromosomes of tomato and other plant species when anthers were fixed with 1:3 acetic ethanol that lengthens pachytene chromosomes approximately two-fold compared to the formaldehyde fixation we used (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">De Jong <em>et al.</em> 1999</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Kulikova <em>et al.</em> 2001</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Cheng <em>et al.</em> 2001</a>, <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">2002</a>; <a id="xref-ref-53-3" class="xref-bibr" href="#ref-53">Wang <em>et al.</em> 2006b</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Szinay <em>et al.</em> 2008</a>; <a id="xref-ref-25-3" class="xref-bibr" href="#ref-25">Peters <em>et al.</em> 2009</a>).</p></div><div id="sec-21" class="subsection"><h3>The Kazusa tomato EXPEN 2000 linkage map is based on a hybrid</h3><p id="p-48">Tomato has unusually low genetic diversity because it went through more than one genetic bottleneck during domestication (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Rick 1991</a>). To obtain more heterozygosity for genetic mapping, the Kazusa tomato EXPEN 2000 linkage map is based on a hybrid cross between the <em>Solanum lycopersicum</em> cultivar LA925 and the wild tomato species <em>S. pennellii</em> cultivar LA716 (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Fulton <em>et al.</em> 2002</a>; <a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Frary <em>et al.</em> 2005</a>; <a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Shirasawa <em>et al.</em> 2010</a>). For the EXPEN 2000 linkage map to correspond precisely to cultivated tomato chromosomes, the genomes of the two species have to be structurally identical, <em>i.e.</em>, no major inversion, translocation, duplication, or deletion differences. While this was generally thought to be the case when the precursor EXPEN 1992 map was published (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Khush and Rick 1963</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Tanksley <em>et al.</em> 1992</a>), we now know that the two species differ in both genome size and organization (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Anderson <em>et al.</em> 2010</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Szinay <em>et al.</em> 2012</a>). The genome of <em>S. pennellii</em> is 20–30% larger than the tomato genome, and electron microscopic examination of SC spreads from an F1 hybrid revealed a large segment of foldback synapsis, a small inversion loop, and at least five SCs with mismatched kinetochores (<a id="xref-ref-2-2" class="xref-bibr" href="#ref-2">Anderson <em>et al.</em> 2010</a>). Mismatched kinetochores indicate proximal nonhomologous synapsis and/or neocentromere formation that are consistent with reports of enhanced suppression of proximal crossing over in a tomato × <em>S. pennellii</em> F1 hybrid compared to tomato controls (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Rick 1969</a>, <a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">1972</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Zhang <em>et al.</em> 2014</a>). Such structural differences between chromosomes of the two species may cause problems in the linkage map that interfere with its use in ordering and orienting scaffolds. However, the generally good agreement between map-based and FISH-based pseudomolecules in distal euchromatin indicates that most of the major differences between the two genomes occur in pericentric heterochromatin.</p></div><div id="sec-22" class="subsection"><h3>Optical mapping strongly supports the FISH-based scaffold arrangement</h3><p id="p-49">Optical mapping is an independent physical method used for arranging the scaffolds of pseudomolecules in several eukaryotic genomes (<a id="xref-ref-64-2" class="xref-bibr" href="#ref-64">Zhou <em>et al.</em> 2009</a>; <a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Young <em>et al.</em> 2011</a>; <a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Dong <em>et al.</em> 2013</a>; <a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Chamala <em>et al.</em> 2013</a>). Like FISH, optical mapping works in euchromatin and heterochromatin. Though useful for only part of the tomato genome, optical mapping was always consistent with FISH-based arrangements of scaffolds but often differed from linkage-based arrangements (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/TableS5.pdf">Table S5</a>).</p><p id="p-50">Because the results of BAC-FISH and optical mapping strongly indicate that many scaffolds were arranged incorrectly, the FISH-based arrangement of scaffolds is being used in the new release of the tomato genome build (SL2.5, <a href="http://solgenomics.net/organism/Solanum_lycopersicum/genome">http://solgenomics.net/organism/Solanum_lycopersicum/genome</a>). In a role reversal, the new scaffold arrangement based on physical methods should be used to rearrange hundreds of markers in the linkage map, particularly in heterochromatin.</p></div><div id="sec-23" class="subsection"><h3>Gap sizes between adjacent scaffolds were estimated by FISH</h3><p id="p-51">While linkage mapping provides little help with determining the sizes of gaps in sequencing and/or assembly between scaffolds, FISH permits estimates of the amounts of DNA in gaps based on gap lengths and linear DNA densities of 3.3 Mb/µm for kinetochores, 1.5 Mb/µm for euchromatin, and 8.8 Mb/µm for heterochromatin (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/TableS6.pdf">Table S6</a>). Approximately similar estimates of linear DNA densities have been made before for euchromatin and heterochromatin in 1:3 acetic ethanol-fixed tomato pachytene chromosomes (<a id="xref-ref-44-2" class="xref-bibr" href="#ref-44">Szinay <em>et al.</em> 2008</a>). We found individual gap sizes ranging between 0 and 3.2 Mb. Overall, we estimate that 43.9 Mb or approximately (43.9 Mb/919 Mb =) 5% of the genome is in gaps between scaffolds and thus unincorporated into the genome assembly (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/TableS7.pdf">Table S7</a>).</p><p id="p-52">A caveat to our current estimates of linear DNA density is the assumption that the sequenced scaffold lengths are correct. In euchromatin, where the scaffolds are more likely to be completely sequenced, our estimate of 1.5 Mb/µm matches the value determined using microspectrophotometry of pachytene chromosomes (<a id="xref-ref-28-3" class="xref-bibr" href="#ref-28">Peterson <em>et al.</em> 1996</a>; <a id="xref-ref-42-4" class="xref-bibr" href="#ref-42">Stack <em>et al.</em> 2009</a>). However, our estimate of 8.8 Mb/µm for the linear density of DNA in heterochromatin is lower than the estimate of 9.2 Mb/µm based on microspectrophotometry (<a id="xref-ref-28-4" class="xref-bibr" href="#ref-28">Peterson <em>et al.</em> 1996</a>; <a id="xref-ref-42-5" class="xref-bibr" href="#ref-42">Stack <em>et al.</em> 2009</a>). This may indicate that 8.8 Mb/µm is an underestimate for heterochromatin where repeated sequences are more likely to interfere with sequencing and/or assembly.</p><p id="p-53">We were unable to make such a genome-wide estimate of gap sizes using optical mapping because 53 of the 91 scaffolds resolved as singletons. However, the gap sizes estimated by FISH were almost always larger than those estimated by optical mapping (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/TableS8.pdf">Table S8</a>), so if optical mapping is more accurate for estimating gap sizes, then the true amount of DNA in gaps may be less than our estimate based on FISH. Although it is not clear why gap size estimates by FISH tend to be larger, the resolution limits of light microscopy as well as the use of average values to estimate linear DNA density could be involved. Regardless of the approach used to assess the sizes of gaps, a number of gaps are so small (&lt;100 kb) that it is surprising that no BACs (that average ∼100 kb inserts) have been found to bridge the gaps. Possibly some of these gaps are caused by simple sequence repeats (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Chang <em>et al.</em> 2008</a>) that make assembly difficult, and incorrect arrangement of some scaffolds using the linkage map may also have made it difficult to find BACs that bridge gaps. If so, then the rearrangement of scaffolds based on FISH results should aid in joining at least some adjacent scaffolds.</p></div><div id="sec-24" class="subsection"><h3>FISH locates most unassigned (chromosome <em>0</em>) BACs in gaps between scaffolds</h3><p id="p-54">BACs with sequence that does not fit in any scaffold would be predicted to be located in gaps between scaffolds and, indeed, FISH indicates this is the case (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2</a>, <a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/FigureS1.pdf">Figure S1</a>). These unassigned BACs are entrées for sequencing and/or assembling sequence in the gaps where they are located, while the few unassigned BACs that are located by FISH within scaffolds suggest assembly errors that can be corrected.</p></div><div id="sec-25" class="subsection"><h3>FISH relates pseudomolecules and linkage maps to the structure of pachytene chromosomes</h3><p id="p-55">FISH has been used before to relate physical maps and pachytene chromosomes to linkage maps, <em>e.g.</em>, in rice (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Cheng <em>et al.</em> 2001</a>), potato (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Iovene <em>et al.</em> 2008</a>), papaya (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wai <em>et al.</em> 2012</a>), and maize (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Wang <em>et al.</em> 2006a</a>). In tomato, also, pseudomolecules can be superimposed on the structure of pachytene chromosomes so euchromatin, heterochromatin, and kinetochores can be accurately related to scaffolds and pseudomolecules (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figures 2</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/FigureS1.pdf">Figure S1</a>). Similarly, molecular markers in the Kazusa EXPEN 2000 linkage map have known positions in pseudomolecules, so the tomato linkage map should be superimposable on pachytene chromosomes as well (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Koo <em>et al.</em> 2008</a>; <a id="xref-ref-48-6" class="xref-bibr" href="#ref-48">The Tomato Genome Consortium 2012</a>). However, this requires reconciliation of the linkage map with the arrangement of scaffolds determined by FISH and optical mapping, especially in pericentric heterochromatin where reduced crossing-over makes the linkage map less reliable.</p><p id="p-56">A previous effort to superimpose the tomato linkage map on tomato pachytene chromosome <em>1</em> took advantage of the distribution of recombination nodules that are cytological markers for crossovers on SCs (<a id="xref-ref-5-3" class="xref-bibr" href="#ref-5">Chang <em>et al.</em> 2007</a>). While FISH showed generally good agreement between observed marker positions on the SCs and the positions predicted from combining the RN and EXPEN 2000 linkage maps, one exceptional segment of SC <em>1</em> had three adjacent marker BACs that did not fit well with their expected locations [see Figure 3B in (<a id="xref-ref-5-4" class="xref-bibr" href="#ref-5">Chang <em>et al.</em> 2007</a>)]. It probably is not a coincidence that these BACs are located in the same region as scaffolds 7 and 8 that appear by FISH to be out of order in the linkage map–based pseudomolecule for chromosome <em>1</em> (<a id="xref-fig-4-10" class="xref-fig" href="#F4">Figure 4</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/TableS4.pdf">Table S4</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1/TableS5.pdf">Table S5</a>).</p></div><div id="sec-26" class="subsection"><h3>FISH shows that recent duplications in the tomato genome are rare</h3><p id="p-57">With 627 of 639 BACs localizing to single sites, there appear to be few duplications in the tomato genome based on hybridization at 80% stringency. This is a bit surprising considering that the tomato lineage is thought to have undergone a genome triplication 71 (±19) million years ago (<a id="xref-ref-48-7" class="xref-bibr" href="#ref-48">The Tomato Genome Consortium 2012</a>). Apparently, there has been enough gene loss and mutation since the most recent polyploidy event to make tomato effectively diploid, <em>i.e.</em>, most genes may have been duplicated, but in the meantime one copy was lost or the two copies diverged sufficiently to no longer hybridize (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Rick 1991</a>; <a id="xref-ref-48-8" class="xref-bibr" href="#ref-48">The Tomato Genome Consortium 2012</a>).</p></div><div id="sec-27" class="subsection"><h3>Limitations of BAC-FISH on spreads of pachytene chromosomes for assembling pseudomolecules</h3><p id="p-58">Like most cytological techniques, BAC-FISH is not high-throughput. For instance, starting with bacterial cultures, it may take a skilled worker 1 month to precisely localize BACs to define the location of a scaffold, although FISH experiments can be staggered and run in parallel. Even so, for organisms in which the genome sequence is assembled into hundreds or thousands of scaffolds, BAC-FISH may not be a cost-effective approach to build pseudomolecules corresponding to chromosomes. Also, locating BACs only at the ends of long scaffolds by FISH does not test the correctness of contig arrangement within scaffolds. Our localization of five unassigned BACs within scaffolds in heterochromatin may indicate problems with assembly and/or sequencing (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Karafiátová <em>et al.</em> 2013</a>). Furthermore, the accuracy of BAC-FISH is limited both by the resolving power of light microscopy and by differences in levels of chromatin compaction along SCs. In addition, utilizing BAC-FISH for genome assembly requires prior knowledge of chromosome architecture and the ability to prepare clean, well-separated, distortion-free, pachytene chromosome or SC spreads suitable for FISH [see Supplementary Figure 24 in <a id="xref-ref-48-9" class="xref-bibr" href="#ref-48">The Tomato Genome Consortium (2012)</a>]. In this regard, tomato was an ideal candidate for arranging scaffolds using BAC-FISH because it has been a cytogenetic model for many years (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Rick and Butler 1956</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Ramanna and Prakken 1967</a>; <a id="xref-ref-42-6" class="xref-bibr" href="#ref-42">Stack <em>et al.</em> 2009</a>; <a id="xref-ref-45-2" class="xref-bibr" href="#ref-45">Szinay <em>et al.</em> 2012</a>), and the proportion of highly repeated sequences in the tomato genome is low (<a id="xref-ref-27-2" class="xref-bibr" href="#ref-27">Peterson <em>et al.</em> 1998</a>). However, species with large genomes and many repeated sequences, <em>e.g.</em>, maize, may require SC spreads, single copy probes, more sensitive FISH techniques, and/or more effective means of blocking repeated sequences to effectively use BAC-FISH for checking genome assembly (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Zhong <em>et al.</em> 2001</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Stack and Anderson 2002</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Koumbaris and Bass 2003</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Kato <em>et al.</em> 2006</a>; <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Wang <em>et al.</em> 2006a</a>). When BAC-FISH is impractical, optical mapping may be a feasible alternative (or a supplement) to linkage map–based genome assemblies (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Zhou <em>et al.</em> 2007a</a>,<a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">b</a>; <a id="xref-ref-3-3" class="xref-bibr" href="#ref-3">Chamala <em>et al.</em> 2013</a>), with the caveat that optical mapping was capable of ordering fewer than half of the scaffolds in the tomato genome. However, BAC-FISH is a powerful technique for determining overall scaffold arrangement for organisms in which genome assembly is relatively complete and in which adequate chromosome spreads can be prepared.</p></div></div><div class="section conclusions" id="sec-28"><h2 class="">Conclusions</h2><p id="p-59">The accuracy of the FISH-based arrangement of scaffolds in tomato is strongly supported by an independent physical method, optical mapping, and the internal consistency of FISH results. By this we mean that the FISH-based arrangement of scaffolds covers most of the length of the tomato chromosomes with only small gaps, scaffolds do not overlap, and unanchored BACs localize primarily to gaps between scaffolds. Because of this, the FISH-based arrangement of scaffolds is being used in the new release of the tomato genome build (SL2.5) on the SGN website (<a href="http://solgenomics.net/organism/Solanum_lycopersicum/genome">http://solgenomics.net/organism/Solanum_lycopersicum/genome</a>). Furthermore, because tomato scaffolds have linkage markers, the new build can be used to improve the tomato linkage map, particularly in regions where crossing over is suppressed. Finally, our results suggest the existence of similar problems in the arrangement of scaffolds in other large genomes that were assembled according to linkage maps (<a id="xref-ref-18-3" class="xref-bibr" href="#ref-18">Karafiátová <em>et al.</em> 2013</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">The International Barley Genome Sequencing Consortium 2014</a>) while providing possible approaches for correcting those problems.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-60">This work was supported by National Science Foundation grants DBI-0116076, DBI-0421634, and DBI-0820612 (JJG, BAR, SMS), and by the Centre for BioSystems Genomics (CBSG), which is part of the Netherlands Genomics Initiative/Netherlands Organization for Scientific Research and by the European Commission (EU-SOL project PL 016214) (HdJ, SS).</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011197/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-61"><em>Communicating editor: K. Dawe</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-03-21"><span class="received-label">Received </span>March 21, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-05-27"><span class="accepted-label">Accepted </span>May 27, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Shearer <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.1" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.2" data-doi="10.1159/000313850"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Covey</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Larsen</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bedinger</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stack</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Structural differences in chromosomes distinguish species in the tomato clade.</span> <abbr class="cit-jnl-abbrev">Cytogenet. Genome Res.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">24</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenetic%2Band%2Bgenome%2Bresearch%26rft.stitle%253DCytogenet%2BGenome%2BRes%26rft.aulast%253DAnderson%26rft.auinit1%253DL.%2BK.%26rft.volume%253D129%26rft.issue%253D1-3%26rft.spage%253D24%26rft.epage%253D34%26rft.atitle%253DStructural%2Bdifferences%2Bin%2Bchromosomes%2Bdistinguish%2Bspecies%2Bin%2Bthe%2Btomato%2Bclade.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000313850%26rft_id%253Dinfo%253Apmid%252F20551606%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000313850&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20551606&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.3" data-doi="10.1126/science.1241130"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chamala</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chanderbali</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Der, T. Lan, B. Walts</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Assembly and validation of the genome of the nonmodel basal angiosperm <em>Amborella</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">342</span>: <span class="cit-fpage">1516</span>–<span class="cit-lpage">1517</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DChamala%26rft.auinit1%253DS.%26rft.volume%253D342%26rft.issue%253D6165%26rft.spage%253D1516%26rft.epage%253D1517%26rft.atitle%253DAssembly%2Band%2BValidation%2Bof%2Bthe%2BGenome%2Bof%2Bthe%2BNonmodel%2BBasal%2BAngiosperm%2BAmborella%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1241130%26rft_id%253Dinfo%253Apmid%252F24357320%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzNDIvNjE2NS8xNTE2IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzEzOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-book" id="cit-4.8.1395.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">S.-B.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-source">Cytogenetic and molecular studies on tomato chromosomes using diploid tomato and tomato monosomic additions in tetraploid potato</span>. <span class="cit-publ-name">Wageningen University</span>, <span class="cit-publ-loc">Wageningen, The Netherlands</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.5" data-doi="10.1534/genetics.107.074138"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">S.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">L. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Royer</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stack</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Predicting and testing physical locations of genetically mapped loci on tomato pachytene chromosome <em>1</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">176</span>: <span class="cit-fpage">2131</span>–<span class="cit-lpage">2138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.107.074138%26rft_id%253Dinfo%253Apmid%252F17565940%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Ni80LzIxMzEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzgvMTM5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">S.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">T.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Datema</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vugt</span>  <span class="cit-name-given-names">J. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vosman</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">FISH mapping and molecular organization of the major repetitive sequences of tomato.</span> <abbr class="cit-jnl-abbrev">Chrom. Res.</abbr> <span class="cit-vol">16</span>: <span class="cit-fpage">919</span>–<span class="cit-lpage">933</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChrom.%2BRes.%26rft.volume%253D16%26rft.spage%253D919%26rft_id%253Dinfo%253Apmid%252F18688733%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18688733&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buell</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wing</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Resolution of fluorescence <em>in-situ</em> hybridization mapping on rice mitotic prometaphase chromosomes, meiotic pachytene chromosomes and extended DNA fibers.</span> <abbr class="cit-jnl-abbrev">Chrom. Res.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">379</span>–<span class="cit-lpage">387</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChrom.%2BRes.%26rft.volume%253D10%26rft.spage%253D379%26rft_id%253Dinfo%253Apmid%252F12296520%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12296520&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Presting</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buell</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wing</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">High-resolution pachytene chromosome mapping of bacterial artificial chromosomes anchored by genetic markers reveals the centromere location and the distribution of genetic recombination along chromosome 10 of rice.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">1749</span>–<span class="cit-lpage">1757</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCheng%26rft.auinit1%253DZ.%26rft.volume%253D157%26rft.issue%253D4%26rft.spage%253D1749%26rft.epage%253D1757%26rft.atitle%253DHigh-Resolution%2BPachytene%2BChromosome%2BMapping%2Bof%2BBacterial%2BArtificial%2BChromosomes%2BAnchored%2Bby%2BGenetic%2BMarkers%2BReveals%2Bthe%2BCentromere%2BLocation%2Band%2Bthe%2BDistribution%2Bof%2BGenetic%2BRecombination%2BAlong%2BChromosome%2B10%2Bof%2BRice%26rft_id%253Dinfo%253Apmid%252F11290728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE1Ny80LzE3NDkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzgvMTM5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.9" data-doi="10.1016/S1360-1385(99)01436-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Jong</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fransz</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zabel</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">High resolution FISH in plants - techniques and applications.</span> <abbr class="cit-jnl-abbrev">Trends Plant Sci.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">258</span>–<span class="cit-lpage">263</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bplant%2Bscience%26rft.stitle%253DTrends%2BPlant%2BSci%26rft.aulast%253DHans%2Bde%2BJong%2BJ%26rft.volume%253D4%26rft.issue%253D7%26rft.spage%253D258%26rft.epage%253D263%26rft.atitle%253DHigh%2Bresolution%2BFISH%2Bin%2Bplants%2B-%2Btechniques%2Band%2Bapplications.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS1360-1385%252899%252901436-3%26rft_id%253Dinfo%253Apmid%252F10407441%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S1360-1385(99)01436-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10407441&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000081294700009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Di Filippo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Traini</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">D’Agostino</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frusciante</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiusano</span>  <span class="cit-name-given-names">M. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Euchromatic and heterochromatic compositional properties emerging from the analysis of <em>Solanum lycopersicum</em> BAC sequences.</span> <abbr class="cit-jnl-abbrev">Gene</abbr> <span class="cit-vol">499</span>: <span class="cit-fpage">176</span>–<span class="cit-lpage">181</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGene%26rft.volume%253D499%26rft.spage%253D176%26rft_id%253Dinfo%253Apmid%252F22391094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22391094&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.11" data-doi="10.1021/ac0496401"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dimalanta</span>  <span class="cit-name-given-names">E. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lim</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Runnheim</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lamers</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churas</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">A microfluidic system for large DNA molecule arrays.</span> <abbr class="cit-jnl-abbrev">Anal. Chem.</abbr> <span class="cit-vol">76</span>: <span class="cit-fpage">5293</span>–<span class="cit-lpage">5301</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnalytical%2BChemistry%2B%2528Washington%252C%2BDC%2529%26rft.stitle%253DAnalytical%2BChemistry%2B%2528Washington%252C%2BDC%2529%26rft.aulast%253DDimalanta%26rft.auinit1%253DE.%2BT.%26rft.volume%253D76%26rft.issue%253D18%26rft.spage%253D5293%26rft.epage%253D5301%26rft.atitle%253DA%2Bmicrofluidic%2Bsystem%2Bfor%2Blarge%2BDNA%2Bmolecule%2Barrays.%26rft_id%253Dinfo%253Adoi%252F10.1021%252Fac0496401%26rft_id%253Dinfo%253Apmid%252F15362885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1021/ac0496401&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15362885&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.12" data-doi="10.1038/nbt.2478"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Dong</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang, N. Xiao, X. Du</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Sequencing and automated whole-genome optical mapping of the genome of a domestic goat (<em>Capra hircus</em>).</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">135</span>–<span class="cit-lpage">141</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BBiotechnol.%26rft.volume%253D31%26rft.spage%253D135%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.2478%26rft_id%253Dinfo%253Apmid%252F23263233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.2478&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23263233&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.13" data-doi="10.1007/s00122-005-2023-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frary</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mitchell</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tedeschi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Development of a set of PCR-based anchor markers encompassing the tomato genome and evaluation of their usefulness for genetics and breeding experiments.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">291</span>–<span class="cit-lpage">312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DFrary%26rft.auinit1%253DA.%26rft.volume%253D111%26rft.issue%253D2%26rft.spage%253D291%26rft.epage%253D312%26rft.atitle%253DDevelopment%2Bof%2Ba%2Bset%2Bof%2BPCR-based%2Banchor%2Bmarkers%2Bencompassing%2Bthe%2Btomato%2Bgenome%2Band%2Bevaluation%2Bof%2Btheir%2Busefulness%2Bfor%2Bgenetics%2Band%2Bbreeding%2Bexperiments.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-005-2023-7%26rft_id%253Dinfo%253Apmid%252F15926074%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-005-2023-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15926074&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000230689800013&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.14" data-doi="10.1105/tpc.010479"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fulton</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Van der Hoeven</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eannetta</span>  <span class="cit-name-given-names">N. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanksley</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Identification, analysis, and utilization of conserved ortholog set markers for comparative genomics in higher plants.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1457</span>–<span class="cit-lpage">1467</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DFulton%26rft.auinit1%253DT.%2BM.%26rft.volume%253D14%26rft.issue%253D7%26rft.spage%253D1457%26rft.epage%253D1467%26rft.atitle%253DIdentification%252C%2BAnalysis%252C%2Band%2BUtilization%2Bof%2BConserved%2BOrtholog%2BSet%2BMarkers%2Bfor%2BComparative%2BGenomics%2Bin%2BHigher%2BPlants%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.010479%26rft_id%253Dinfo%253Apmid%252F12119367%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjE0LzcvMTQ1NyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOC8xMzk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.15" data-doi="10.1534/genetics.108.093179"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iovene</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wielgus</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Simon</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buell</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Chromatin structure and physical mapping of chromosome 6 of potato and comparative analyses with tomato.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">1307</span>–<span class="cit-lpage">1317</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.093179%26rft_id%253Dinfo%253Apmid%252F18791232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4MC8zLzEzMDciO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzgvMTM5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.16" data-doi="10.1186/1471-2229-9-42"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jo</span>  <span class="cit-name-given-names">S.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koo</span>  <span class="cit-name-given-names">D.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hur</span>  <span class="cit-name-given-names">C.-G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Evolution of ribosomal DNA-derived satellite repeat in tomato genome.</span> <abbr class="cit-jnl-abbrev">BMC Plant Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bplant%2Bbiology%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BPlant%2BBiol%26rft.aulast%253DJo%26rft.auinit1%253DS.%2BH.%26rft.volume%253D9%26rft.spage%253D42%26rft.epage%253D42%26rft.atitle%253DEvolution%2Bof%2Bribosomal%2BDNA-derived%2Bsatellite%2Brepeat%2Bin%2Btomato%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2229-9-42%26rft_id%253Dinfo%253Apmid%252F19351415%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2229-9-42&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19351415&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.17" data-doi="10.1016/0168-1656(94)90032-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Joos</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fink</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rätsch</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lichter</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Mapping and chromosome analysis: the potential of fluorescence in situ hybridization.</span> <abbr class="cit-jnl-abbrev">J. Biotechnol.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">135</span>–<span class="cit-lpage">153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BBiotechnology%26rft.stitle%253DJournal%2Bof%2BBiotechnology%26rft.aulast%253DJoos%26rft.auinit1%253DS.%26rft.volume%253D35%26rft.issue%253D2-3%26rft.spage%253D135%26rft.epage%253D153%26rft.atitle%253DMapping%2Band%2Bchromosome%2Banalysis%253A%2Bthe%2Bpotential%2Bof%2Bfluorescence%2Bin%2Bsitu%2Bhybridization.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0168-1656%252894%252990032-9%26rft_id%253Dinfo%253Apmid%252F7765054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0168-1656(94)90032-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=7765054&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.18"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karafiátová</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartoš</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kopecký</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ma</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mapping nonrecombining regions in barley using multicolor FISH.</span> <abbr class="cit-jnl-abbrev">Chrom. Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">739</span>–<span class="cit-lpage">751</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChrom.%2BRes.%26rft.volume%253D21%26rft.spage%253D739%26rft_id%253Dinfo%253Apmid%252F24026304%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24026304&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.19" data-doi="10.1080/10520290600643677"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kato</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Albert</span>  <span class="cit-name-given-names">P. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vega</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Birchler</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Sensitive fluorescence in situ hybridization signal detection in maize using directly labeled probes produced by high concentration DNA polymerase nick translation.</span> <abbr class="cit-jnl-abbrev">Biotech. Histochem.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">71</span>–<span class="cit-lpage">78</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiotechnic%2B%2526%2Bhistochemistry%2B%253A%2B%2Bofficial%2Bpublication%2Bof%2Bthe%2BBiological%2BStain%2BCommission%26rft.stitle%253DBiotech%2BHistochem%26rft.aulast%253DKato%26rft.auinit1%253DA.%26rft.volume%253D81%26rft.issue%253D2-3%26rft.spage%253D71%26rft.epage%253D78%26rft.atitle%253DSensitive%2Bfluorescence%2Bin%2Bsitu%2Bhybridization%2Bsignal%2Bdetection%2Bin%2Bmaize%2Busing%2Bdirectly%2Blabeled%2Bprobes%2Bproduced%2Bby%2Bhigh%2Bconcentration%2BDNA%2Bpolymerase%2Bnick%2Btranslation.%26rft_id%253Dinfo%253Adoi%252F10.1080%252F10520290600643677%26rft_id%253Dinfo%253Apmid%252F16908431%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1080/10520290600643677&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16908431&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000240796000003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.20" data-doi="10.1007/BF01725760"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Khush</span>  <span class="cit-name-given-names">G. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rick</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1963</span> <span class="cit-article-title">Meiosis in hybrids between <em>Lycopersicon esculentum</em> and <em>Solanum pennellii</em>.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">167</span>–<span class="cit-lpage">183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D33%26rft.spage%253D167%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01725760%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01725760&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A19631869B00002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.21" data-doi="10.1534/genetics.108.089532"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koo</span>  <span class="cit-name-given-names">D.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jo</span>  <span class="cit-name-given-names">S. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bang</span>  <span class="cit-name-given-names">J.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">H.-M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Integration of cytogenetic and genetic linkage maps unveils the physical architecture of tomato chromosome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">179</span>: <span class="cit-fpage">1211</span>–<span class="cit-lpage">1220</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.089532%26rft_id%253Dinfo%253Apmid%252F18622034%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3OS8zLzEyMTEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzgvMTM5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.22" data-doi="10.1046/j.1365-313X.2003.01829.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koumbaris</span>  <span class="cit-name-given-names">G. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bass</span>  <span class="cit-name-given-names">H. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A new single-locus cytogenetic mapping system for maize (<em>Zea mays</em> L.): overcoming FISH detection limits with marker-selected sorghum (<em>S. propinquum</em> L.) BACs clones.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">647</span>–<span class="cit-lpage">659</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DKoumbaris%26rft.auinit1%253DG.%2BL.%26rft.volume%253D35%26rft.issue%253D5%26rft.spage%253D647%26rft.epage%253D659%26rft.atitle%253DA%2Bnew%2Bsingle-locus%2Bcytogenetic%2Bmapping%2Bsystem%2Bfor%2Bmaize%2B%2528Zea%2Bmays%2BL.%2529%253A%2Bovercoming%2BFISH%2Bdetection%2Blimits%2Bwith%2Bmarker-selected%2Bsorghum%2B%2528S.%2Bpropinquum%2BL.%2529%2BBAC%2Bclones.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-313X.2003.01829.x%26rft_id%253Dinfo%253Apmid%252F12940957%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-313X.2003.01829.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12940957&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000185079200009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.23" data-doi="10.1046/j.1365-313x.2001.01057.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kulikova</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gualtieri</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Geurts</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D.-J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cook</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Integration of the FISH pachytene and genetic maps of <em>Medicago truncatula</em>.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">27</span>: <span class="cit-fpage">49</span>–<span class="cit-lpage">58</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DKulikova%26rft.auinit1%253DO.%26rft.volume%253D27%26rft.issue%253D1%26rft.spage%253D49%26rft.epage%253D58%26rft.atitle%253DIntegration%2Bof%2Bthe%2BFISH%2Bpachytene%2Band%2Bgenetic%2Bmaps%2Bof%2BMedicago%2Btruncatula.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-313x.2001.01057.x%26rft_id%253Dinfo%253Apmid%252F11489182%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-313x.2001.01057.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11489182&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000170396100006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.24" data-doi="10.1093/bioinformatics/btn025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kristiansen</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">SOAP: short oligonucelotide alignment program.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">713</span>–<span class="cit-lpage">714</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn025%26rft_id%253Dinfo%253Apmid%252F18227114%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNC81LzcxMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOC8xMzk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peters</span>  <span class="cit-name-given-names">S. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Datema</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szinay</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Staveren</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schijlen</span>  <span class="cit-name-given-names">E. G. W. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title"><em>Solanum lycopersicum</em> cv. Heinz 1706 chromosome 6: distribution and abundance of genes and retrotransposable elements.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">58</span>: <span class="cit-fpage">857</span>–<span class="cit-lpage">869</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DPeters%26rft.auinit1%253DS.%2BA.%26rft.volume%253D58%26rft.issue%253D5%26rft.spage%253D857%26rft.epage%253D869%26rft.atitle%253DSolanum%2Blycopersicum%2Bcv.%2BHeinz%2B1706%2Bchromosome%2B6%253A%2Bdistribution%2Band%2Babundance%2Bof%2Bgenes%2Band%2Bretrotransposable%2Belements.%26rft_id%253Dinfo%253Apmid%252F19207213%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19207213&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-book" id="cit-4.8.1395.26"><div class="cit-metadata unstructured">Peterson, D. G., 2014 Evolution of plant genome analysis. pp. 13–46 in <em>Genomes of Herbaceous Land Plants</em>, <em>Volume 69</em>, <em>Advances in Botanical Research</em>, edited by A.H. Paterson. Elsevier, Paris.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.27"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pearson</span>  <span class="cit-name-given-names">W. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stack</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Characterization of the tomato (<em>Lycopersicon esculentum</em>) genome using <em>in vitro</em> and <em>in situ</em> DNA reassociation.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">41</span>: <span class="cit-fpage">346</span>–<span class="cit-lpage">356</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%26rft.volume%253D41%26rft.spage%253D346%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peterson</span>  <span class="cit-name-given-names">D. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Price</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">J. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stack</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">DNA content of heterochromatin and euchromatin in tomato (<em>Lycopersicon esculentum</em>) pachytene chromosomes.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">77</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DPeterson%26rft.auinit1%253DD.%2BG.%26rft.volume%253D39%26rft.issue%253D1%26rft.spage%253D77%26rft.epage%253D82%26rft.atitle%253DDNA%2Bcontent%2Bof%2Bheterochromatin%2Band%2Beuchromatin%2Bin%2Btomato%2B%2528Lycopersicon%2Besculentum%2529%2Bpachytene%2Bchromosomes.%26rft_id%253Dinfo%253Apmid%252F18469880%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=18469880&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.29" data-doi="10.1007/BF01507452"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ramanna</span>  <span class="cit-name-given-names">M. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prakken</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">1967</span> <span class="cit-article-title">Structure and homology between pachytene and somatic metaphase chromosomes of the tomato.</span> <abbr class="cit-jnl-abbrev">Genetica</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">115</span>–<span class="cit-lpage">133</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetica%26rft.volume%253D38%26rft.spage%253D115%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF01507452%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF01507452&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.30" data-doi="10.1016/S0065-2660(08)60504-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rick</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Butler</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1956</span> <span class="cit-article-title">Cytogenetics of the tomato.</span> <abbr class="cit-jnl-abbrev">Adv. Genet.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">267</span>–<span class="cit-lpage">382</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdv.%2BGenet.%26rft.volume%253D8%26rft.spage%253D267%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0065-2660%252808%252960504-0%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0065-2660(08)60504-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1956WY36800008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.31"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rick</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1969</span> <span class="cit-article-title">Controlled introgression of chromosomes of <em>Solanum pennellii</em> into <em>Lycopersicon esculentum</em>: segregation and recombination.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">753</span>–<span class="cit-lpage">768</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRick%26rft.auinit1%253DC.%2BM.%26rft.volume%253D62%26rft.issue%253D4%26rft.spage%253D753%26rft.epage%253D768%26rft.atitle%253DCONTROLLED%2BINTROGRESSION%2BOF%2BCHROMOSOMES%2BOF%2BSOLANUM%2BPENNELLII%2B%2BINTO%2BLYCOPERSICON%2BESCULENTUM%253A%2BSEGREGATION%2BAND%2BRECOMBINATION%26rft_id%253Dinfo%253Apmid%252F17248457%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo4OiI2Mi80Lzc1MyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOC8xMzk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rick</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1972</span> <span class="cit-article-title">Further studies on segregation and recombination in backcross derivatives of a tomato species hybrid.</span> <abbr class="cit-jnl-abbrev">Biol. Zbl.</abbr> <span class="cit-vol">91</span>: <span class="cit-fpage">209</span>–<span class="cit-lpage">229</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiol.%2BZbl.%26rft.volume%253D91%26rft.spage%253D209%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.33"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rick</span>  <span class="cit-name-given-names">C. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1991</span> <span class="cit-article-title">Tomato paste: A concentrated review of genetic highlights from the beginnings to the advent of molecular genetics.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">128</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">5</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DRick%26rft.auinit1%253DC.%2BM.%26rft.volume%253D128%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D5%26rft.atitle%253DTomato%2Bpaste%253A%2Ba%2Bconcentrated%2Breview%2Bof%2Bgenetic%2Bhighlights%2Bfrom%2Bthe%2Bbeginnings%2Bto%2Bthe%2Badvent%2Bof%2Bmolecular%2Bgenetics.%26rft_id%253Dinfo%253Apmid%252F2060773%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6MzoiUERGIjtzOjExOiJqb3VybmFsQ29kZSI7czo4OiJnZW5ldGljcyI7czo1OiJyZXNpZCI7czo3OiIxMjgvMS8xIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzEzOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.34"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sadder</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ponelies</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Born</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weber</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Physical localization of single-copy sequences on pachytene chromosomes in maize (<em>Zea mays</em> L.) by chromosome in situ suppression hybridization.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">1081</span>–<span class="cit-lpage">1083</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DSadder%26rft.auinit1%253DM.%2BT.%26rft.volume%253D43%26rft.issue%253D6%26rft.spage%253D1081%26rft.epage%253D1083%26rft.atitle%253DPhysical%2Blocalization%2Bof%2Bsingle-copy%2Bsequences%2Bon%2Bpachytene%2Bchromosomes%2Bin%2Bmaize%2B%2528Zea%2Bmays%2BL.%2529%2Bby%2Bchromosome%2Bin%2Bsitu%2Bsuppression%2Bhybridization.%26rft_id%253Dinfo%253Apmid%252F11195341%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=11195341&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.35"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schubert</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fransz</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fuchs</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Jong</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Chromosome painting in plants.</span> <abbr class="cit-jnl-abbrev">Methods Enzymol.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">69</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%2BEnzymol.%26rft.volume%253D23%26rft.spage%253D57%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-book" id="cit-4.8.1395.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schwarzacher</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Heslop-Harrison</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-source">Practical in situ hybridization</span>, <span class="cit-publ-name">BIOS Scientific Publishers Ltd.</span>, <span class="cit-publ-loc">New York</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stack</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">Two-dimensional spreads of synaptonemal complexes from solanaceous plants. V. Tomato (<em>Lycopersicon esculentum</em>) karyotype and idiogram.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">354</span>–<span class="cit-lpage">359</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%26rft.volume%253D35%26rft.spage%253D354%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sherman</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stack</span>  <span class="cit-name-given-names">S. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1995</span> <span class="cit-article-title">Two-dimensional spreads of synaptonemal complexes from solanaceous plants. VI. High-resolution recombination nodule map for tomato (<em>Lycopersicon esculentum</em>).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">683</span>–<span class="cit-lpage">708</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DSherman%26rft.auinit1%253DJ.%2BD.%26rft.volume%253D141%26rft.issue%253D2%26rft.spage%253D683%26rft.epage%253D708%26rft.atitle%253DTwo-dimensional%2Bspreads%2Bof%2Bsynaptonemal%2Bcomplexes%2Bfrom%2Bsolanaceous%2Bplants.%2BVI.%2BHigh-resolution%2Brecombination%2Bnodule%2Bmap%2Bfor%2Btomato%2B%2528Lycopersicon%2Besculentum%2529.%26rft_id%253Dinfo%253Apmid%252F8647403%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTQxLzIvNjgzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzEzOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.39" data-doi="10.1007/s00122-010-1344-3"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shirasawa</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Asamizu</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fukuoka</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ohyama</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sato</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">An interspecific linkage map of SSR and intronic polymorphism markers in tomato.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">121</span>: <span class="cit-fpage">731</span>–<span class="cit-lpage">739</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DShirasawa%26rft.auinit1%253DK.%26rft.volume%253D121%26rft.issue%253D4%26rft.spage%253D731%26rft.epage%253D739%26rft.atitle%253DAn%2Binterspecific%2Blinkage%2Bmap%2Bof%2BSSR%2Band%2Bintronic%2Bpolymorphism%2Bmarkers%2Bin%2Btomato.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-010-1344-3%26rft_id%253Dinfo%253Apmid%252F20431859%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-010-1344-3&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20431859&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000280276000011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stack</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anderson</span>  <span class="cit-name-given-names">L. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Crossing over as assessed by late recombination nodules is related to the pattern of synapsis and distribution of early recombination nodules in maize.</span> <abbr class="cit-jnl-abbrev">Chrom. Res.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">329</span>–<span class="cit-lpage">345</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChrom.%2BRes.%26rft.volume%253D10%26rft.spage%253D329%26rft_id%253Dinfo%253Apmid%252F12199146%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12199146&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-book" id="cit-4.8.1395.41"><div class="cit-metadata unstructured">Stack, S. M., and L. K. Anderson, 2009 Electron microscopic immunogold localization of recombination-related proteins in spreads of synaptonemal complexes from tomato microsporocytes, pp. 147–169 in <em>Meiosis Volume 2 Cytological Methods</em>, edited by S. Keeney. Humana Press, Inc., Totowa, NJ.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.42" data-doi="10.1159/000218137"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stack</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Royer</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shearer</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">S.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giovannoni</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Role of fluorescence <em>in situ</em> hybridization in sequencing the tomato genome.</span> <abbr class="cit-jnl-abbrev">Cytogenet. Genome Res.</abbr> <span class="cit-vol">124</span>: <span class="cit-fpage">339</span>–<span class="cit-lpage">350</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenetic%2Band%2Bgenome%2Bresearch%26rft.stitle%253DCytogenet%2BGenome%2BRes%26rft.aulast%253DStack%26rft.auinit1%253DS.%2BM.%26rft.volume%253D124%26rft.issue%253D3-4%26rft.spage%253D339%26rft.epage%253D350%26rft.atitle%253DRole%2Bof%2Bfluorescence%2Bin%2Bsitu%2Bhybridization%2Bin%2Bsequencing%2Bthe%2Btomato%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1159%252F000218137%26rft_id%253Dinfo%253Apmid%252F19556785%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1159/000218137&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19556785&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000267648000014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.43"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szinay</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bai</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visser</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Jong</span>  <span class="cit-name-given-names">H.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">FISH applications for genomics and plant breeding strategies in tomato and other Solanaceous crops.</span> <abbr class="cit-jnl-abbrev">Cytogenet. Genome Res.</abbr> <span class="cit-vol">129</span>: <span class="cit-fpage">199</span>–<span class="cit-lpage">210</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCytogenetic%2Band%2Bgenome%2Bresearch%26rft.stitle%253DCytogenet%2BGenome%2BRes%26rft.aulast%253DSzinay%26rft.auinit1%253DD.%26rft.volume%253D129%26rft.issue%253D1-3%26rft.spage%253D199%26rft.epage%253D210%26rft.atitle%253DFISH%2Bapplications%2Bfor%2Bgenomics%2Band%2Bplant%2Bbreeding%2Bstrategies%2Bin%2Btomato%2Band%2Bother%2Bsolanaceous%2Bcrops.%26rft_id%253Dinfo%253Apmid%252F20628252%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20628252&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.44" data-doi="10.1111/j.1365-313X.2008.03626.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szinay</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chang</span>  <span class="cit-name-given-names">S.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khrustaleva</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peters</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schijlen</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">High-resolution chromosome mapping of BACs using multi-colour FISH and pooled-BAC FISH as a backbone for sequencing tomato chromosome 6.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">627</span>–<span class="cit-lpage">637</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DSzinay%26rft.auinit1%253DD.%26rft.volume%253D56%26rft.issue%253D4%26rft.spage%253D627%26rft.epage%253D637%26rft.atitle%253DHigh-resolution%2Bchromosome%2Bmapping%2Bof%2BBACs%2Busing%2Bmulti-colour%2BFISH%2Band%2Bpooled-BAC%2BFISH%2Bas%2Ba%2Bbackbone%2Bfor%2Bsequencing%2Btomato%2Bchromosome%2B6.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2008.03626.x%26rft_id%253Dinfo%253Apmid%252F18643986%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2008.03626.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18643986&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000260824700010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.45" data-doi="10.1111/j.1469-8137.2012.04195.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Szinay</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wijnker</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van den Berg</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Visser</span>  <span class="cit-name-given-names">R. G. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Jong</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Chromosome evolution in Solanum traced by cross-species BAC-FISH.</span> <abbr class="cit-jnl-abbrev">New Phytol.</abbr> <span class="cit-vol">195</span>: <span class="cit-fpage">688</span>–<span class="cit-lpage">698</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNew%2BPhytologist%26rft.stitle%253DNew%2BPhytologist%26rft.aulast%253DSzinay%26rft.auinit1%253DD.%26rft.volume%253D195%26rft.issue%253D3%26rft.spage%253D688%26rft.epage%253D698%26rft.atitle%253DChromosome%2Bevolution%2Bin%2BSolanum%2Btraced%2Bby%2Bcross-species%2BBAC-FISH.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1469-8137.2012.04195.x%26rft_id%253Dinfo%253Apmid%252F22686400%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1469-8137.2012.04195.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22686400&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306179200021&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.46"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tanksley</span>  <span class="cit-name-given-names">S. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ganal</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prince</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Vicente</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bonierbale</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1992</span> <span class="cit-article-title">High density molecular linkage maps of the tomato and potato genomes.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">132</span>: <span class="cit-fpage">1141</span>–<span class="cit-lpage">1160</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DTanksley%26rft.auinit1%253DS.%2BD.%26rft.volume%253D132%26rft.issue%253D4%26rft.spage%253D1141%26rft.epage%253D1160%26rft.atitle%253DHigh%2Bdensity%2Bmolecular%2Blinkage%2Bmaps%2Bof%2Bthe%2Btomato%2Band%2Bpotato%2Bgenomes.%26rft_id%253Dinfo%253Apmid%252F1360934%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjEzMi80LzExNDEiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzgvMTM5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.47"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">The International Barley Genome Sequencing Consortium</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">A physical, genetic and functional sequence assembly of the barley genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">491</span>: <span class="cit-fpage">711</span>–<span class="cit-lpage">716</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D491%26rft.spage%253D711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.48" data-doi="10.1038/nature11119"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">The Tomato Genome Consortium</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The tomato genome sequence provides insights into fleshy fruit evolution.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">485</span>: <span class="cit-fpage">635</span>–<span class="cit-lpage">641</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DSato%26rft.auinit1%253DS.%26rft.volume%253D485%26rft.issue%253D7400%26rft.spage%253D635%26rft.epage%253D641%26rft.atitle%253DThe%2Btomato%2Bgenome%2Bsequence%2Bprovides%2Binsights%2Binto%2Bfleshy%2Bfruit%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11119%26rft_id%253Dinfo%253Apmid%252F22660326%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11119&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22660326&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000304608000047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.49" data-doi="10.1105/tpc.010478"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Van der Hoeven</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ronning</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giovannoni</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanksley</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Deductions about the number, organization, and evolution of genes in the tomato genome based on analysis of a large expressed sequence tag collection and selective genomic sequencing.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1441</span>–<span class="cit-lpage">1456</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.aulast%253DVan%2Bder%2BHoeven%26rft.auinit1%253DR.%26rft.volume%253D14%26rft.issue%253D7%26rft.spage%253D1441%26rft.epage%253D1456%26rft.atitle%253DDeductions%2Babout%2Bthe%2BNumber%252C%2BOrganization%252C%2Band%2BEvolution%2Bof%2BGenes%2Bin%2Bthe%2BTomato%2BGenome%2BBased%2Bon%2BAnalysis%2Bof%2Ba%2BLarge%2BExpressed%2BSequence%2BTag%2BCollection%2Band%2BSelective%2BGenomic%2BSequencing%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.010478%26rft_id%253Dinfo%253Apmid%252F12119366%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjE0LzcvMTQ0MSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOC8xMzk1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.50" data-doi="10.1101/gr.112094.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">van Oeveren</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">de Ruiter</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jesse</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Poel</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Sequence-based physical mapping of complex genomes by whole genome profiling.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">618</span>–<span class="cit-lpage">625</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.112094.110%26rft_id%253Dinfo%253Apmid%252F21324881%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzQvNjE4IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzEzOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wai</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">P. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paull</span>  <span class="cit-name-given-names">R. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ming</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">Q.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">An integrated cytogenetic and physical map reveals unevenly distributed recombination spots along the papaya sex chromosomes.</span> <abbr class="cit-jnl-abbrev">Chrom. Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">753</span>–<span class="cit-lpage">767</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DChrom.%2BRes.%26rft.volume%253D20%26rft.spage%253D753%26rft_id%253Dinfo%253Apmid%252F23007683%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23007683&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.52" data-doi="10.1105/tpc.105.037838"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.-J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Harper</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cande</span>  <span class="cit-name-given-names">W. Z.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span><span class="cit-comment">a</span> <span class="cit-article-title">High-resolution single-copy gene fluorescence in situ hybridization and its use in the construction of a cytogenetic map of maize chromosome 9.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">529</span>–<span class="cit-lpage">544</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.105.037838%26rft_id%253Dinfo%253Apmid%252F16461583%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjE4LzMvNTI5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzEzOTUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.53" data-doi="10.1534/genetics.106.055772"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mueller</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Giovannoni</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span><span class="cit-comment">b</span> <span class="cit-article-title">Euchromatin and pericentromeric heterochromatin: Comparative composition in the tomato genome.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">172</span>: <span class="cit-fpage">2529</span>–<span class="cit-lpage">2540</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.055772%26rft_id%253Dinfo%253Apmid%252F16489216%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE3Mi80LzI1MjkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzgvMTM5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.54" data-doi="10.1111/j.1365-313X.2012.05017.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koo</span>  <span class="cit-name-given-names">D.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luan</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Chromosome rearrangements during domestication of cucumber as revealed by high-density genetic mapping and draft genome assembly.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">71</span>: <span class="cit-fpage">895</span>–<span class="cit-lpage">906</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DYang%26rft.auinit1%253DL.%26rft.volume%253D71%26rft.issue%253D6%26rft.spage%253D895%26rft.epage%253D906%26rft.atitle%253DChromosome%2Brearrangements%2Bduring%2Bdomestication%2Bof%2Bcucumber%2Bas%2Brevealed%2Bby%2Bhigh-density%2Bgenetic%2Bmapping%2Band%2Bdraft%2Bgenome%2Bassembly.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2012.05017.x%26rft_id%253Dinfo%253Apmid%252F22487099%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2012.05017.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22487099&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.55"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Young</span>  <span class="cit-name-given-names">N. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Debellé</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oldroyd, R. Geurts, S. B. Cannon</span>  <span class="cit-name-given-names">G. E. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The <em>Medicago</em> genome provides insight into the evolution of rhizobial synbioses.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">480</span>: <span class="cit-fpage">520</span>–<span class="cit-lpage">524</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DYoung%26rft.auinit1%253DN.%2BD.%26rft.volume%253D480%26rft.issue%253D7378%26rft.spage%253D520%26rft.epage%253D524%26rft.atitle%253DThe%2BMedicago%2Bgenome%2Bprovides%2Binsight%2Binto%2Bthe%2Bevolution%2Bof%2Brhizobial%2Bsymbioses.%26rft_id%253Dinfo%253Apmid%252F22089132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=22089132&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298318000060&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.56" data-doi="10.1007/BF00339589"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zamir</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanksley</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Tomato genome is comprised largely of fast-evolving, low copy number sequences.</span> <abbr class="cit-jnl-abbrev">Mol. Gen. Genet.</abbr> <span class="cit-vol">213</span>: <span class="cit-fpage">254</span>–<span class="cit-lpage">261</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMol.%2BGen.%2BGenet.%26rft.volume%253D213%26rft.spage%253D254%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00339589%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00339589&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.57" data-doi="10.1105/tpc.114.123877."><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koblízková</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gong</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliveira</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Boom-bust turnovers of megabase-sized centromeric DNA in Solanum species: Rapid evolution of DNA sequences associated with centromeres</span>. <abbr class="cit-jnl-abbrev">Plant Cell</abbr><span class="cit-pub-id-sep cit-pub-id-doi-sep"> </span><span class="cit-pub-id cit-pub-id-doi"><span class="cit-pub-id-scheme-doi">doi:</span>10.1105/tpc.114.123877.</span></cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.114.123877.%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1105/tpc.114.123877.&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wing</span>  <span class="cit-name-given-names">R. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">Preparation of megabase-sized DNA from plant nuclei.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">184</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BJ.%26rft.volume%253D7%26rft.spage%253D175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.59" data-doi="10.1007/BF02671658"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fransz</span>  <span class="cit-name-given-names">P. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wennekes-van Eden</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zabel</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Kammen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">High-resolution mapping on pachytene chromosomes and extended DNA fibres by fluorescence <em>in-situ</em> hybridization.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol. Rep.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">232</span>–<span class="cit-lpage">242</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BMol.%2BBiol.%2BRep.%26rft.volume%253D14%26rft.spage%253D232%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF02671658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF02671658&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1996VN71300005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.60" data-doi="10.1073/pnas.061026198"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhong</span>  <span class="cit-name-given-names">X.-B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lizardi</span>  <span class="cit-name-given-names">P. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huang</span>  <span class="cit-name-given-names">X. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bray-Ward</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ward</span>  <span class="cit-name-given-names">D. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Visualization of oligonucleotide probes and point mutations in interphase nuclei and DNA fibers using rolling circle DNA amplification.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">3940</span>–<span class="cit-lpage">3945</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DZhong%26rft.auinit1%253DX.-b.%26rft.volume%253D98%26rft.issue%253D7%26rft.spage%253D3940%26rft.epage%253D3945%26rft.atitle%253DVisualization%2Bof%2Boligonucleotide%2Bprobes%2Band%2Bpoint%2Bmutations%2Bin%2Binterphase%2Bnuclei%2Band%2BDNA%2Bfibers%2Busing%2Brolling%2Bcircle%2BDNA%2Bamplification%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.061026198%26rft_id%253Dinfo%253Apmid%252F11274414%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5OC83LzM5NDAiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzgvMTM5NS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.61" data-doi="10.1016/j.molbiopara.2004.08.002"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kile</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kvikstad</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bechner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Severin</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Shotgun optical mapping of the entire <em>Leishmania major</em> Friedlin genome.</span> <abbr class="cit-jnl-abbrev">Mol. Biochem. Parasitol.</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2Bbiochemical%2Bparasitology%26rft.stitle%253DMol%2BBiochem%2BParasitol%26rft.aulast%253DZhou%26rft.auinit1%253DS.%26rft.volume%253D138%26rft.issue%253D1%26rft.spage%253D97%26rft.epage%253D106%26rft.atitle%253DShotgun%2Boptical%2Bmapping%2Bof%2Bthe%2Bentire%2BLeishmania%2Bmajor%2BFriedlin%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molbiopara.2004.08.002%26rft_id%253Dinfo%253Apmid%252F15500921%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molbiopara.2004.08.002&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15500921&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000225128100012&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.62" data-doi="10.1186/1471-2164-8-278"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bechner</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Place</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churas</span>  <span class="cit-name-given-names">C. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pape</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span><span class="cit-comment">a</span> <span class="cit-article-title">Validation of rice genome sequence by optical mapping.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">278</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DZhou%26rft.auinit1%253DS.%26rft.volume%253D8%26rft.spage%253D278%26rft.epage%253D278%26rft.atitle%253DValidation%2Bof%2Brice%2Bgenome%2Bsequence%2Bby%2Boptical%2Bmapping.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-8-278%26rft_id%253Dinfo%253Apmid%252F17697381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-8-278&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17697381&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1395.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-book" id="cit-4.8.1395.63"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Herschleb</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwartz</span>  <span class="cit-name-given-names">D. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span><span class="cit-comment">b</span> <span class="cit-article-title">A single molecule system for whole genome analysis</span>, pp. <span class="cit-fpage">269</span>–<span class="cit-lpage">304</span> in <span class="cit-source">New Methods for DNA Sequencing</span>, edited by <span class="cit-ed"><span class="cit-name-surname">Mitchelson</span>  <span class="cit-name-given-names">K. R.</span></span> <span class="cit-publ-name">Elsevier B.V.</span>, <span class="cit-publ-loc">Amsterdam</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1395.64" data-doi="10.1371/journal.pgen.1000521"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhou</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wei</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nguyen</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bechner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Potamousis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Single molecule scaffold for the maize genome.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D5%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1000521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1000521&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/8/1395.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/8/1385" title="Validation of Six Genetic Determinants of Susceptibility to Estrogen-Induced Mammary Cancer in the Rat and Assessment of Their Relevance to Breast Cancer Risk in Humans" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/8/1407" title="Population Dynamics and Evolutionary History of the Weedy Vine Ipomoea hederacea in North America" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/8"><span>Volume 4 Issue 8, August 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/8.cover-source.jpg?itok=uF7Z9Bah" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/8/1395" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-0AV7IFUgE3_Aj5sW52Om1dTK1g1oQicLYI0gHCv8WG0" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/8/1395" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/8/1395" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/8/1395" class="active" data-icon-position="" data-hide-link-title="0">Fluorescence In Situ Hybridization and Optical Mapping to Correct Scaffold Arrangement in the Tomato Genome</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459935" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-Ek2Qkg5224ki4bhjCQJAyHV5MHj5Skbr6MWCRySIoKQ" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/8/1395.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/8/1395" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-6WYS-bQM7GnJkPBHg_rirkS00bMDTnQOf250qDSdOKM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/8/1395" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/8/1395.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459935" id="node459935--2" data-pisa="ggg;4/8/1395" data-pisa-master="ggg;g3.114.011197" data-apath="/ggg/4/8/1395.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Fluorescence <em>In Situ</em> Hybridization and Optical Mapping to Correct Scaffold Arrangement in the Tomato Genome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Lindsay A. Shearer</span>, <span class="highwire-citation-author" data-delta="1">Lorinda K. Anderson</span>, <span class="highwire-citation-author" data-delta="2">Hans de Jong</span>, <span class="highwire-citation-author" data-delta="3">Sandra Smit</span>, <span class="highwire-citation-author" data-delta="4">José Luis Goicoechea</span>, <span class="highwire-citation-author" data-delta="5">Bruce A. Roe</span>, <span class="highwire-citation-author" data-delta="6">Axin Hua</span>, <span class="highwire-citation-author" data-delta="7">James J. Giovannoni</span> and <span class="highwire-citation-author" data-delta="8">Stephen M. Stack</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1395-1405;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.011197 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459935--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Lindsay A. Shearer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Colorado State University, Fort Collins, Colorado 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lindsay%2BA.%2BShearer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shearer%20LA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALindsay%2BA.%2BShearer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lorinda K. Anderson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Colorado State University, Fort Collins, Colorado 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lorinda%2BK.%2BAnderson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Anderson%20LK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALorinda%2BK.%2BAnderson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Hans de Jong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Genetics, Wageningen University and Research Centre (WUR), Droevendaalsesteeg 1, 6708 PB Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hans%2Bde%2BJong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Jong%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHans%2Bde%2BJong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Sandra Smit </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratory of Bioinformatics, WUR, Droevendaalsesteeg1, 6708 PB Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sandra%2BSmit%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smit%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASandra%2BSmit%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">José Luis Goicoechea </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Arizona Genomics Institute, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BLuis%2BGoicoechea%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goicoechea%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJos%25C3%25A9%2BLuis%2BGoicoechea%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Bruce A. Roe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Chemistry and Biochemistry, Stephenson Research and Technology Center, University of Oklahoma, Norman, Oklahoma 73019</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bruce%2BA.%2BRoe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Roe%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABruce%2BA.%2BRoe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Axin Hua </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Chemistry and Biochemistry, Stephenson Research and Technology Center, University of Oklahoma, Norman, Oklahoma 73019</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Axin%2BHua%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hua%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAxin%2BHua%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">James J. Giovannoni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Plant Biology, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BJ.%2BGiovannoni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giovannoni%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BJ.%2BGiovannoni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Stephen M. Stack </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Colorado State University, Fort Collins, Colorado 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BM.%2BStack%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stack%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStephen%2BM.%2BStack%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Stephen.Stack@colostate.edu" class="" data-icon-position="" data-hide-link-title="0">Stephen.Stack@colostate.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/8/1395" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/8/1395&amp;title=Fluorescence%20In%20Situ%20Hybridization%20and%20Optical%20Mapping%20to%20Correct%20Scaffold%20Arrangement%20in%20the%20Tomato%20Genome" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/8/1395&amp;title=Fluorescence%20In%20Situ%20Hybridization%20and%20Optical%20Mapping%20to%20Correct%20Scaffold%20Arrangement%20in%20the%20Tomato%20Genome" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/8/1395&amp;title=Fluorescence%20In%20Situ%20Hybridization%20and%20Optical%20Mapping%20to%20Correct%20Scaffold%20Arrangement%20in%20the%20Tomato%20Genome" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/8/1395&amp;text=Fluorescence%20In%20Situ%20Hybridization%20and%20Optical%20Mapping%20to%20Correct%20Scaffold%20Arrangement%20in%20the%20Tomato%20Genome" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/8/1395&amp;title=Fluorescence%20In%20Situ%20Hybridization%20and%20Optical%20Mapping%20to%20Correct%20Scaffold%20Arrangement%20in%20the%20Tomato%20Genome" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/8/1395&amp;t=Fluorescence%20In%20Situ%20Hybridization%20and%20Optical%20Mapping%20to%20Correct%20Scaffold%20Arrangement%20in%20the%20Tomato%20Genome" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/8/1395&amp;title=Fluorescence%20In%20Situ%20Hybridization%20and%20Optical%20Mapping%20to%20Correct%20Scaffold%20Arrangement%20in%20the%20Tomato%20Genome" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/8/1395&amp;title=Fluorescence%20In%20Situ%20Hybridization%20and%20Optical%20Mapping%20to%20Correct%20Scaffold%20Arrangement%20in%20the%20Tomato%20Genome" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459935/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459935" id="node459935--3" data-pisa="ggg;4/8/1395" data-pisa-master="ggg;g3.114.011197" data-apath="/ggg/4/8/1395.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Fluorescence <em>In Situ</em> Hybridization and Optical Mapping to Correct Scaffold Arrangement in the Tomato Genome</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Lindsay A. Shearer</span>, <span class="highwire-citation-author" data-delta="1">Lorinda K. Anderson</span>, <span class="highwire-citation-author" data-delta="2">Hans de Jong</span>, <span class="highwire-citation-author" data-delta="3">Sandra Smit</span>, <span class="highwire-citation-author" data-delta="4">José Luis Goicoechea</span>, <span class="highwire-citation-author" data-delta="5">Bruce A. Roe</span>, <span class="highwire-citation-author" data-delta="6">Axin Hua</span>, <span class="highwire-citation-author" data-delta="7">James J. Giovannoni</span> and <span class="highwire-citation-author" data-delta="8">Stephen M. Stack</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1395-1405;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.011197 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459935--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Lindsay A. Shearer </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Colorado State University, Fort Collins, Colorado 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lindsay%2BA.%2BShearer%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Shearer%20LA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALindsay%2BA.%2BShearer%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Lorinda K. Anderson </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Colorado State University, Fort Collins, Colorado 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lorinda%2BK.%2BAnderson%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Anderson%20LK&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALorinda%2BK.%2BAnderson%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Hans de Jong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Laboratory of Genetics, Wageningen University and Research Centre (WUR), Droevendaalsesteeg 1, 6708 PB Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hans%2Bde%2BJong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=de%20Jong%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHans%2Bde%2BJong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Sandra Smit </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>‡</span>Laboratory of Bioinformatics, WUR, Droevendaalsesteeg1, 6708 PB Wageningen, The Netherlands</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Sandra%2BSmit%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Smit%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASandra%2BSmit%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">José Luis Goicoechea </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Arizona Genomics Institute, University of Arizona, Tucson, Arizona 85721</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jos%C3%A9%2BLuis%2BGoicoechea%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Goicoechea%20JL&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJos%25C3%25A9%2BLuis%2BGoicoechea%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Bruce A. Roe </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Chemistry and Biochemistry, Stephenson Research and Technology Center, University of Oklahoma, Norman, Oklahoma 73019</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Bruce%2BA.%2BRoe%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Roe%20BA&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ABruce%2BA.%2BRoe%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Axin Hua </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Department of Chemistry and Biochemistry, Stephenson Research and Technology Center, University of Oklahoma, Norman, Oklahoma 73019</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Axin%2BHua%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Hua%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAxin%2BHua%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">James J. Giovannoni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>††</span>Department of Plant Biology, Cornell University, Ithaca, New York 14853</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=James%2BJ.%2BGiovannoni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Giovannoni%20JJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJames%2BJ.%2BGiovannoni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Stephen M. Stack </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Biology, Colorado State University, Fort Collins, Colorado 80523</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Stephen%2BM.%2BStack%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Stack%20SM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AStephen%2BM.%2BStack%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:Stephen.Stack@colostate.edu" class="" data-icon-position="" data-hide-link-title="0">Stephen.Stack@colostate.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1395&amp;rft.epage=1405&amp;rft.atitle=Fluorescence%20In%20Situ%20Hybridization%20and%20Optical%20Mapping%20to%20Correct%20Scaffold%20Arrangement%20in%20the%20Tomato%20Genome&amp;rft.volume=4&amp;rft.issue=8&amp;rft.date=2014-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Shearer%2C+Lindsay+A.&amp;rft.au=Anderson%2C+Lorinda+K.&amp;rft.au=de+Jong%2C+Hans&amp;rft.au=Smit%2C+Sandra&amp;rft.au=Goicoechea%2C+Jos%C3%A9+Luis&amp;rft.au=Roe%2C+Bruce+A.&amp;rft.au=Hua%2C+Axin&amp;rft.au=Giovannoni%2C+James+J.&amp;rft.au=Stack%2C+Stephen+M."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459935/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459935/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459935/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459935/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459935/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459935/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459935/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459935/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459935/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459935/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459935/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459935/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/8/1395" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-12" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#sec-28" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Conclusions</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/8/1395.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/8/1395.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/8/1395.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/8/1395.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
