<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/4/8/1429" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/4/8/1429.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/4/8/1429.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/4/8/1429.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/4/8/1429.atom" />
<meta name="HW.pisa" content="ggg;4/8/1429" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genomic Profiling of Collaborative Cross Founder Mice Infected with Respiratory Viruses Reveals Novel Transcripts and Infection-Related Strain-Specific Gene and Isoform Expression" />
<meta name="DC.Identifier" content="10.1534/g3.114.011759" />
<meta name="DC.Date" content="2014-08-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2014 Xiong et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Genetic variation between diverse mouse species is well-characterized, yet existing knowledge of the mouse transcriptome comes largely from one mouse strain (C57BL/6J). As such, it is unlikely to reflect the transcriptional complexity of the mouse species. Gene transcription is dynamic and condition-specific; therefore, to better understand the mouse transcriptional response to respiratory virus infection, we infected the eight founder strains of the Collaborative Cross with either influenza A virus or severe acute respiratory syndrome coronavirus and sequenced lung RNA samples at 2 and 4 days after infection. We found numerous instances of transcripts that were not present in the C57BL/6J reference annotation, indicating that a nontrivial proportion of the mouse genome is transcribed but poorly annotated. Of these novel transcripts, 2150 could be aligned to human or rat genomes, but not to existing mouse genomes, suggesting functionally conserved sequences not yet recorded in mouse genomes. We also found that respiratory virus infection induced differential expression of 4287 splicing junctions, resulting in strain-specific isoform expression. Of these, 59 were influenced by strain-specific mutations within 2 base pairs of key intron–exon boundaries, suggesting cis -regulated expression. Our results reveal the complexity of the transcriptional response to viral infection, previously undocumented genomic elements, and extensive diversity in the response across mouse strains. These findings identify hitherto unexplored transcriptional patterns and undocumented transcripts in genetically diverse mice. Host genetic variation drives the complexity and diversity of the host response by eliciting starkly different transcriptional profiles in response to a viral infection." />
<meta name="DC.Contributor" content="Hao Xiong" />
<meta name="DC.Contributor" content="Juliet Morrison" />
<meta name="DC.Contributor" content="Martin T. Ferris" />
<meta name="DC.Contributor" content="Lisa E. Gralinski" />
<meta name="DC.Contributor" content="Alan C. Whitmore" />
<meta name="DC.Contributor" content="Richard Green" />
<meta name="DC.Contributor" content="Matthew J. Thomas" />
<meta name="DC.Contributor" content="Jennifer Tisoncik-Go" />
<meta name="DC.Contributor" content="Gary P. Schroth" />
<meta name="DC.Contributor" content="Fernando Pardo-Manuel de Villena" />
<meta name="DC.Contributor" content="Ralph S. Baric" />
<meta name="DC.Contributor" content="Mark T. Heise" />
<meta name="DC.Contributor" content="Xinxia Peng" />
<meta name="DC.Contributor" content="Michael G. Katze" />
<meta name="article:published_time" content="2014-08-01" />
<meta name="article:section" content="Mouse Genetic Resources" />
<meta name="citation_title" content="Genomic Profiling of Collaborative Cross Founder Mice Infected with Respiratory Viruses Reveals Novel Transcripts and Infection-Related Strain-Specific Gene and Isoform Expression" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Genetic variation between diverse mouse species is well-characterized, yet existing knowledge of the mouse transcriptome comes largely from one mouse strain (C57BL/6J). As such, it is unlikely to reflect the transcriptional complexity of the mouse species. Gene transcription is dynamic and condition-specific; therefore, to better understand the mouse transcriptional response to respiratory virus infection, we infected the eight founder strains of the Collaborative Cross with either influenza A virus or severe acute respiratory syndrome coronavirus and sequenced lung RNA samples at 2 and 4 days after infection. We found numerous instances of transcripts that were not present in the C57BL/6J reference annotation, indicating that a nontrivial proportion of the mouse genome is transcribed but poorly annotated. Of these novel transcripts, 2150 could be aligned to human or rat genomes, but not to existing mouse genomes, suggesting functionally conserved sequences not yet recorded in mouse genomes. We also found that respiratory virus infection induced differential expression of 4287 splicing junctions, resulting in strain-specific isoform expression. Of these, 59 were influenced by strain-specific mutations within 2 base pairs of key intron–exon boundaries, suggesting &lt;i&gt;cis&lt;/i&gt;-regulated expression. Our results reveal the complexity of the transcriptional response to viral infection, previously undocumented genomic elements, and extensive diversity in the response across mouse strains. These findings identify hitherto unexplored transcriptional patterns and undocumented transcripts in genetically diverse mice. Host genetic variation drives the complexity and diversity of the host response by eliciting starkly different transcriptional profiles in response to a viral infection.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2014/08/01" />
<meta name="citation_mjid" content="ggg;4/8/1429" />
<meta name="citation_id" content="4/8/1429" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/4/8/1429" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/4/8/1429.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/4/8/1429.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/4/8/1429.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.114.011759" />
<meta name="citation_pmid" content="24902603" />
<meta name="citation_volume" content="4" />
<meta name="citation_issue" content="8" />
<meta name="citation_num_pages" content="16" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Mouse Genetic Resources" />
<meta name="citation_firstpage" content="1429" />
<meta name="citation_lastpage" content="1444" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Hao Xiong" />
<meta name="citation_author_institution" content="Department of Microbiology, School of Medicine, University of Washington, Seattle, Washington" />
<meta name="citation_author_institution" content="Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon" />
<meta name="citation_author" content="Juliet Morrison" />
<meta name="citation_author_institution" content="Department of Microbiology, School of Medicine, University of Washington, Seattle, Washington" />
<meta name="citation_author_institution" content="Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon" />
<meta name="citation_author" content="Martin T. Ferris" />
<meta name="citation_author_institution" content="Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina-Chapel Hill, Chapel Hill, North Carolina" />
<meta name="citation_author" content="Lisa E. Gralinski" />
<meta name="citation_author_institution" content="Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon" />
<meta name="citation_author_institution" content="Department of Epidemiology, University of North Carolina-Chapel Hill, Chapel Hill, North Carolina" />
<meta name="citation_author" content="Alan C. Whitmore" />
<meta name="citation_author_institution" content="Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina-Chapel Hill, Chapel Hill, North Carolina" />
<meta name="citation_author" content="Richard Green" />
<meta name="citation_author_institution" content="Department of Microbiology, School of Medicine, University of Washington, Seattle, Washington" />
<meta name="citation_author_institution" content="Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon" />
<meta name="citation_author" content="Matthew J. Thomas" />
<meta name="citation_author_institution" content="Department of Microbiology, School of Medicine, University of Washington, Seattle, Washington" />
<meta name="citation_author_institution" content="Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon" />
<meta name="citation_author" content="Jennifer Tisoncik-Go" />
<meta name="citation_author_institution" content="Department of Microbiology, School of Medicine, University of Washington, Seattle, Washington" />
<meta name="citation_author_institution" content="Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon" />
<meta name="citation_author" content="Gary P. Schroth" />
<meta name="citation_author_institution" content="Illumina, Inc., San Diego, California" />
<meta name="citation_author" content="Fernando Pardo-Manuel de Villena" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina-Chapel Hill, Chapel Hill, North Carolina" />
<meta name="citation_author" content="Ralph S. Baric" />
<meta name="citation_author_institution" content="Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon" />
<meta name="citation_author_institution" content="Department of Epidemiology, University of North Carolina-Chapel Hill, Chapel Hill, North Carolina" />
<meta name="citation_author" content="Mark T. Heise" />
<meta name="citation_author_institution" content="Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon" />
<meta name="citation_author_institution" content="Department of Genetics, University of North Carolina-Chapel Hill, Chapel Hill, North Carolina" />
<meta name="citation_author" content="Xinxia Peng" />
<meta name="citation_author_institution" content="Department of Microbiology, School of Medicine, University of Washington, Seattle, Washington" />
<meta name="citation_author_institution" content="Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon" />
<meta name="citation_author" content="Michael G. Katze" />
<meta name="citation_author_institution" content="Department of Microbiology, School of Medicine, University of Washington, Seattle, Washington" />
<meta name="citation_author_institution" content="Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon" />
<meta name="citation_author_email" content="honey@uw.edu" />
<meta name="citation_reference" content="citation_journal_title=J. Autoimmun.;citation_author=J. An;citation_author=T. Nakajima;citation_author=H. Shibata;citation_author=T. Arimura;citation_author=M. Yasunami;citation_title=A novel link of HLA locus to the regulation of immunity and infection: NFKBIL1 regulates alternative splicing of human immune-related genes and influenza virus M gene.;citation_pages=25-33;citation_volume=47;citation_year=2013;citation_pmid=23953137;citation_doi=10.1016/j.jaut.2013.07.010" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. Anders;citation_author=W. Huber;citation_title=Differential expression analysis for sequence count data.;citation_pages=R106-R106;citation_volume=11;citation_year=2010;citation_issue=10;citation_pmid=20979621;citation_doi=10.1186/gb-2010-11-10-r106" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=D. L. Aylor;citation_author=W. Valdar;citation_author=W. Foulds-Mathes;citation_author=R. J. Buus;citation_author=R. A. Verdugo;citation_title=Genetic analysis of complex traits in the emerging Collaborative Cross.;citation_volume=21;citation_year=2011;citation_pmid=21406540;citation_doi=10.1101/gr.111310.110" />
<meta name="citation_reference" content="citation_journal_title=The Journal of Immunology;citation_journal_abbrev=J. Immunol.;citation_author=U. Boehm;citation_author=L. Guethlein;citation_author=T. Klamp;citation_author=K. Ozbek;citation_author=A. Schaub;citation_title=Two Families of GTPases Dominate the Complex Cellular Response to IFN-{gamma};citation_pages=6715-6723;citation_volume=161;citation_year=1998;citation_issue=12;citation_pmid=9862701" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=S. Carpenter;citation_author=D. Aiello;citation_author=M. K. Atianand;citation_author=E. P. Ricci;citation_author=P. Gandhi;citation_title=A long noncoding RNA mediates both activation and repression of immune response genes.;citation_volume=341;citation_year=2013;citation_pmid=23907535;citation_doi=10.1126/science.1240925" />
<meta name="citation_reference" content="citation_journal_title=BMC infectious diseases [electronic resource];citation_journal_abbrev=BMC Infect Dis;citation_author=W. P. Chong;citation_author=W. E. Ip;citation_author=G. H. Tso;citation_author=M. W. Ng;citation_author=W. H. Wong;citation_title=The interferon gamma gene polymorphism +874 A/T is associated with severe acute respiratory syndrome.;citation_pages=82-82;citation_volume=6;citation_year=2006;citation_pmid=16672072;citation_doi=10.1186/1471-2334-6-82" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=G. A. Churchill;citation_title=Misleading Results: Don't Blame the Mice;citation_pages=370-370;citation_volume=343;citation_year=2014;citation_issue=6169;citation_pmid=24458625;citation_doi=10.1126/science.343.6169.370-a" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=. Collaborative Cross Consortium;citation_title=The Genome Architecture of the Collaborative Cross Mouse Genetic Reference Population;citation_pages=389-401;citation_volume=190;citation_year=2012;citation_issue=2;citation_pmid=22345608;citation_doi=10.1534/genetics.111.132639" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=T. Derrien;citation_author=R. Johnson;citation_author=G. Bussotti;citation_author=A. Tanzer;citation_author=S. Djebali;citation_title=The GENCODE v7 catalog of human long noncoding RNAs: Analysis of their gene structure, evolution, and expression;citation_pages=1775-1789;citation_volume=22;citation_year=2012;citation_issue=9;citation_pmid=22955988;citation_doi=10.1101/gr.132159.111" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. Djebali;citation_author=C. A. Davis;citation_author=A. Merkel;citation_author=A. Dobin;citation_author=T. Lassmann;citation_title=Landscape of transcription in human cells.;citation_pages=101-108;citation_volume=489;citation_year=2012;citation_issue=7414;citation_pmid=22955620;citation_doi=10.1038/nature11233" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=C. Durrant;citation_author=H. Tayem;citation_author=B. Yalcin;citation_author=J. Cleak;citation_author=L. Goodstadt;citation_title=Collaborative Cross mice and their power to map host susceptibility to Aspergillus fumigatus infection.;citation_volume=21;citation_year=2011;citation_pmid=21493779;citation_doi=10.1101/gr.118786.110" />
<meta name="citation_reference" content="citation_journal_title=J. Virol.;citation_author=P. L. Earl;citation_author=J. L. Americo;citation_author=B. Moss;citation_title=Lethal monkeypox virus infection of CAST/EiJ mice is associated with a deficient gamma interferon response.;citation_volume=86;citation_year=2012;citation_pmid=22696658;citation_doi=10.1128/JVI.00162-12" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_title=An integrated encyclopedia of DNA elements in the human genome.;citation_pages=57-74;citation_volume=489;citation_year=2012;citation_issue=7414;citation_pmid=22955616;citation_doi=10.1038/nature11247" />
<meta name="citation_reference" content="citation_journal_title=PLoS Pathog.;citation_author=M. T. Ferris;citation_author=D. L. Aylor;citation_author=D. Bottomly;citation_author=A. C. Whitmore;citation_author=L. D. Aicher;citation_title=Modeling host genetic regulation of influenza pathogenesis in the Collaborative Cross.;citation_pages=e1003196;citation_volume=9;citation_year=2013;citation_pmid=23468633;citation_doi=10.1371/journal.ppat.1003196" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=J. Flint;citation_author=E. Eskin;citation_title=Genome-wide association studies in mice.;citation_pages=807-817;citation_volume=13;citation_year=2012;citation_issue=11;citation_pmid=23044826;citation_doi=10.1038/nrg3335" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=M. G. Grabherr;citation_author=B. J. Haas;citation_author=M. Yassour;citation_author=J. Z. Levin;citation_author=D. A. Thompson;citation_title=Full-length transcriptome assembly from RNA-Seq data without a reference genome.;citation_pages=644-652;citation_volume=29;citation_year=2011;citation_issue=7;citation_pmid=21572440;citation_doi=10.1038/nbt.1883" />
<meta name="citation_reference" content="citation_journal_title=The Tohoku journal of experimental medicine;citation_journal_abbrev=Tohoku J Exp Med;citation_author=R. Igusa;citation_author=T. Sakakibara;citation_author=T. Shibahara;citation_author=K. Sakamoto;citation_author=H. Nishimura;citation_title=Complicated secondary pneumonia after swine-origin influenza A virus infection in an immunocompetent patient.;citation_pages=117-120;citation_volume=226;citation_year=2012;citation_issue=2;citation_pmid=22251837;citation_doi=10.1620/tjem.226.117" />
<meta name="citation_reference" content="citation_author=L. Josset;citation_author=N. Tchitchek;citation_author=L. E. Gralinski;citation_author=M. T. Ferris;citation_author=A. J. Eisfeld" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. M. Keane;citation_author=L. Goodstadt;citation_author=P. Danecek;citation_author=M. A. White;citation_author=K. Wong;citation_title=Mouse genomic variation and its effect on phenotypes and gene regulation.;citation_pages=289-294;citation_volume=477;citation_year=2011;citation_issue=7364;citation_pmid=21921910;citation_doi=10.1038/nature10413" />
<meta name="citation_reference" content="citation_journal_title=WJG;citation_author=C.-F. Lee;citation_author=Z.-Q. Ling;citation_author=T. Zhao;citation_author=K.-R. Lee;citation_title=Distinct expression patterns in hepatitis B virus-and hepatitis C virus-infected hepatocellular carcinoma. World journal of gastroenterology;citation_pages=6072;citation_volume=14;citation_year=2008;citation_pmid=18932288;citation_doi=10.3748/wjg.14.6072" />
<meta name="citation_reference" content="citation_journal_title=BMC medical genetics [electronic resource];citation_journal_abbrev=BMC Med Genet;citation_author=M. Lin;citation_author=H.-K. Tseng;citation_author=J. A. Trejaut;citation_author=H.-L. Lee;citation_author=J.-H. Loo;citation_title=Association of HLA class I with severe acute respiratory syndrome coronavirus infection.;citation_pages=9-9;citation_volume=4;citation_year=2003;citation_pmid=12969506;citation_doi=10.1186/1471-2350-4-9" />
<meta name="citation_reference" content="citation_journal_title=Wiley Interdisciplinary Reviews: RNA;citation_author=R. E. Lloyd;citation_title=Regulation of stress granules and P-bodies during RNA virus infection.;citation_pages=317-331;citation_volume=4;citation_year=2013;citation_pmid=23554219;citation_doi=10.1002/wrna.1162" />
<meta name="citation_reference" content="citation_journal_title=The Journal of Immunology;citation_journal_abbrev=J. Immunol.;citation_author=K. K. McKinstry;citation_author=T. M. Strutt;citation_author=A. Buck;citation_author=J. D. Curtis;citation_author=J. P. Dibble;citation_title=IL-10 Deficiency Unleashes an Influenza-Specific Th17 Response and Enhances Survival against High-Dose Challenge;citation_pages=7353-7363;citation_volume=182;citation_year=2009;citation_issue=12;citation_pmid=19494257;citation_doi=10.4049/jimmunol.0900657" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=G. A. McVean;citation_author=D. M. Altshuler;citation_title=An integrated map of genetic variation from 1,092 human genomes.;citation_pages=56-65;citation_volume=491;citation_year=2012;citation_issue=7422;citation_pmid=23128226;citation_doi=10.1038/nature11632" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=S. B. Montgomery;citation_author=M. Sammeth;citation_author=M. Gutierrez-Arcelus;citation_author=R. P. Lach;citation_author=C. Ingle;citation_title=Transcriptome genetics using second generation sequencing in a Caucasian population.;citation_pages=773-777;citation_volume=464;citation_year=2010;citation_issue=7289;citation_pmid=20220756;citation_doi=10.1038/nature08903" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_title=An encyclopedia of mouse DNA elements (Mouse ENCODE).;citation_pages=418-418;citation_volume=13;citation_year=2012;citation_issue=8;citation_pmid=22889292;citation_doi=10.1186/gb-2012-13-8-418" />
<meta name="citation_reference" content="citation_journal_title=Science (New York, N.Y.);citation_author=R. J. Mural;citation_author=M. D. Adams;citation_author=E. W. Myers;citation_author=H. O. Smith;citation_author=G. L. G. Miklos;citation_title=A comparison of whole-genome shotgun-derived mouse chromosome 16 and the human genome.;citation_pages=1661-1671;citation_volume=296;citation_year=2002;citation_doi=10.1126/science.1069193" />
<meta name="citation_reference" content="citation_journal_title=Pediatrics;citation_journal_abbrev=Pediatrics;citation_author=P. C. Ng;citation_author=C. W. K. Lam;citation_author=A. M. Li;citation_author=C. K. Wong;citation_author=F. W. T. Cheng;citation_title=Inflammatory Cytokine Profile in Children With Severe Acute Respiratory Syndrome;citation_pages=e7-e14;citation_volume=113;citation_year=2004;citation_issue=1;citation_pmid=14702488;citation_doi=10.1542/peds.113.1.e7" />
<meta name="citation_reference" content="citation_journal_abbrev=Eur J Intern Med;citation_author=D. Pabst;citation_author=J. Kuehn;citation_author=S. Schuler-Luettmann;citation_author=K. Wiebe;citation_author=P. Lebiedz;citation_title=Acute Respiratory Distress Syndrome as a presenting manifestation in young patients infected with H1N1 influenza virus.;citation_pages=e119-e124;citation_volume=22;citation_year=2011;citation_issue=6;citation_pmid=22075296;citation_doi=10.1016/j.ejim.2011.08.014" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=S. Pal;citation_author=R. Gupta;citation_author=H. Kim;citation_author=P. Wickramasinghe;citation_author=V. Baubet;citation_title=Alternative transcription exceeds alternative splicing in generating the transcriptome diversity of cerebellar development.;citation_volume=21;citation_year=2011;citation_pmid=21712398;citation_doi=10.1101/gr.120535.111" />
<meta name="citation_reference" content="citation_journal_title=MBio;citation_author=X. Peng;citation_author=L. Gralinski;citation_author=C. D. Armour;citation_author=M. T. Ferris;citation_author=M. J. Thomas;citation_title=Unique signatures of long noncoding RNA expression in response to virus infection and altered innate immune signaling.;citation_pages=e00206-e00210;citation_volume=1;citation_year=2010;citation_pmid=20978541;citation_doi=10.1128/mBio.00206-10" />
<meta name="citation_reference" content="citation_journal_title=Genes Immun. 15: 38–46.;citation_title=Using the emerging Collaborative Cross to probe the immune system.;citation_author=J. Phillippi;citation_author=Y. Xie;citation_author=D. R. Miller;citation_author=T. A. Bell;citation_author=Z. Zhang;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. K. Pickrell;citation_author=J. C. Marioni;citation_author=A. A. Pai;citation_author=J. F. Degner;citation_author=B. E. Engelhardt;citation_title=Understanding mechanisms underlying human gene expression variation with RNA sequencing.;citation_pages=768-772;citation_volume=464;citation_year=2010;citation_issue=7289;citation_pmid=20220758;citation_doi=10.1038/nature08872" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=C. Pommerenke;citation_author=E. Wilk;citation_author=B. Srivastava;citation_author=A. Schulze;citation_author=N. Novoselova;citation_title=Global transcriptome analysis in influenza-infected mouse lungs reveals the kinetics of innate and adaptive host immune responses.;citation_pages=e41169-e41169;citation_volume=7;citation_year=2012;citation_issue=7;citation_pmid=22815957;citation_doi=10.1371/journal.pone.0041169" />
<meta name="citation_reference" content="citation_journal_title=eLife;citation_author=N. A. Rapicavoli;citation_author=K. Qu;citation_author=J. Zhang;citation_author=M. Mikhail;citation_author=R.-M. Laberge;citation_title=A mammalian pseudogene lncRNA at the interface of inflammation and anti-inflammatory therapeutics.;citation_pages=00762e;citation_volume=2;citation_year=2013" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=M. D. Robinson;citation_author=D. J. McCarthy;citation_author=G. K. Smyth;citation_title=edgeR: a Bioconductor package for differential expression analysis of digital gene expression data.;citation_pages=139-140;citation_volume=26;citation_year=2009;citation_pmid=19910308" />
<meta name="citation_reference" content="citation_journal_title=Virology;citation_journal_abbrev=Virology;citation_author=B. Sainz;citation_author=E. C. Mossel;citation_author=C. J. Peters;citation_author=R. F. Garry;citation_title=Interferon-beta and interferon-gamma synergistically inhibit the replication of severe acute respiratory syndrome-associated coronavirus (SARS-CoV).;citation_pages=11-17;citation_volume=329;citation_year=2004;citation_issue=1;citation_pmid=15476870;citation_doi=10.1016/j.virol.2004.08.011" />
<meta name="citation_reference" content="citation_journal_title=Philosophical Transactions of the Royal Society B: Biological Sciences;citation_journal_abbrev=Phil Trans R Soc B;citation_author=J. H. Schickli;citation_author=A. Flandorfer;citation_author=T. Nakaya;citation_author=L. Martinez-Sobrido;citation_author=A. Garcia-Sastre;citation_title=Plasmid-only rescue of influenza A virus vaccine candidates;citation_pages=1965-1973;citation_volume=356;citation_year=2001;citation_issue=1416;citation_pmid=11779399;citation_doi=10.1098/rstb.2001.0979" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=P. Staeheli;citation_author=R. Grob;citation_author=E. Meier;citation_author=J. G. Sutcliffe;citation_author=O. Haller;citation_title=Influenza virus-susceptible mice carry Mx genes with a large deletion or a nonsense mutation.;citation_pages=4518-4523;citation_volume=8;citation_year=1988;citation_issue=10;citation_pmid=2903437;citation_doi=10.1128/MCB.8.10.4518" />
<meta name="citation_reference" content="citation_journal_title=J. Virol.;citation_author=O. Terrier;citation_author=V. Marcel;citation_author=G. Cartet;citation_author=D. P. Lane;citation_author=B. Lina;citation_title=Influenza A viruses control expression of proviral human p53 isoforms p53β and Δ133p53α.;citation_volume=86;citation_year=2012;citation_pmid=22647703;citation_doi=10.1128/JVI.07143-11" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=C. Trapnell;citation_author=A. Roberts;citation_author=L. Goff;citation_author=G. Pertea;citation_author=D. Kim;citation_title=Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks.;citation_pages=562-578;citation_volume=7;citation_year=2012;citation_issue=3;citation_pmid=22383036;citation_doi=10.1038/nprot.2012.016" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=C. Trapnell;citation_author=B. A. Williams;citation_author=G. Pertea;citation_author=A. Mortazavi;citation_author=G. Kwan;citation_title=Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation.;citation_pages=511-515;citation_volume=28;citation_year=2010;citation_issue=5;citation_pmid=20436464;citation_doi=10.1038/nbt.1621" />
<meta name="citation_reference" content="citation_journal_title=J. Virol.;citation_author=T. M. Tumpey;citation_author=K. J. Szretter;citation_author=N. V. Hoeven;citation_author=J. M. Katz;citation_author=G. Kochs;citation_title=The Mx1 gene protects mice against the pandemic 1918 and highly lethal human H5N1 influenza viruses.;citation_volume=81;citation_year=2007;citation_pmid=17652381;citation_doi=10.1128/JVI.01116-07" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. T. Wang;citation_author=R. Sandberg;citation_author=S. Luo;citation_author=I. Khrebtukova;citation_author=L. Zhang;citation_title=Alternative isoform regulation in human tissue transcriptomes.;citation_pages=470-476;citation_volume=456;citation_year=2008;citation_issue=7221;citation_pmid=18978772;citation_doi=10.1038/nature07509" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=T. D. Wu;citation_author=C. K. Watanabe;citation_title=GMAP: a genomic mapping and alignment program for mRNA and EST sequences.;citation_volume=21;citation_year=2005;citation_pmid=15728110;citation_doi=10.1093/bioinformatics/bti310" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=H. Yang;citation_author=T. A. Bell;citation_author=G. A. Churchill;citation_author=F. Pardo-Manuel de Villena;citation_title=On the subspecific origin of the laboratory mouse.;citation_pages=1100-1107;citation_volume=39;citation_year=2007;citation_issue=9;citation_pmid=17660819;citation_doi=10.1038/ng2087" />
<meta name="citation_reference" content="citation_journal_abbrev=Cell Host Microbe;citation_author=A. K. Zaas;citation_author=M. Chen;citation_author=J. Varkey;citation_author=T. Veldman;citation_author=A. O. Hero;citation_title=Gene expression signatures diagnose influenza and other symptomatic respiratory viral infections in humans.;citation_pages=207-217;citation_volume=6;citation_year=2009;citation_issue=3;citation_pmid=19664979;citation_doi=10.1016/j.chom.2009.07.006" />
<meta name="citation_reference" content="citation_journal_title=Genome Medicine;citation_author=Z. Zhang;citation_author=S. Pal;citation_author=Y. Bi;citation_author=J. Tchou;citation_author=R. V. Davuluri;citation_title=Isoform level expression profiles provide better cancer signatures than gene level expression profiles.;citation_pages=33;citation_volume=5;citation_year=2013;citation_pmid=23594586;citation_doi=10.1186/gm437" />
<meta name="citation_reference" content="citation_journal_title=Trends in genetics :  TIG;citation_journal_abbrev=Trends Genet;citation_author=M. Zheng;citation_author=D. Dill;citation_author=G. Peltz;citation_title=A better prognosis for genetic association studies in mice.;citation_pages=62-69;citation_volume=28;citation_year=2012;citation_issue=2;citation_pmid=22118772;citation_doi=10.1016/j.tig.2011.10.006" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genomic Profiling of Collaborative Cross Founder Mice Infected with Respiratory Viruses Reveals Novel Transcripts and Infection-Related Strain-Specific Gene and Isoform Expression" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8.cover-source.jpg" />
<meta name="twitter:description" content="Genetic variation between diverse mouse species is well-characterized, yet existing knowledge of the mouse transcriptome comes largely from one mouse strain (C57BL/6J). As such, it is unlikely to reflect the transcriptional complexity of the mouse species. Gene transcription is dynamic and condition-specific; therefore, to better understand the mouse transcriptional response to respiratory virus infection, we infected the eight founder strains of the Collaborative Cross with either influenza A virus or severe acute respiratory syndrome coronavirus and sequenced lung RNA samples at 2 and 4 days after infection. We found numerous instances of transcripts that were not present in the C57BL/6J reference annotation, indicating that a nontrivial proportion of the mouse genome is transcribed but poorly annotated. Of these novel transcripts, 2150 could be aligned to human or rat genomes, but not to existing mouse genomes, suggesting functionally conserved sequences not yet recorded in mouse genomes. We also found that respiratory virus infection induced differential expression of 4287 splicing junctions, resulting in strain-specific isoform expression. Of these, 59 were influenced by strain-specific mutations within 2 base pairs of key intron–exon boundaries, suggesting cis -regulated expression. Our results reveal the complexity of the transcriptional response to viral infection, previously undocumented genomic elements, and extensive diversity in the response across mouse strains. These findings identify hitherto unexplored transcriptional patterns and undocumented transcripts in genetically diverse mice. Host genetic variation drives the complexity and diversity of the host response by eliciting starkly different transcriptional profiles in response to a viral infection." />
<meta name="og-title" property="og:title" content="Genomic Profiling of Collaborative Cross Founder Mice Infected with Respiratory Viruses Reveals Novel Transcripts and Infection-Related Strain-Specific Gene and Isoform Expression" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/4/8/1429" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Genetic variation between diverse mouse species is well-characterized, yet existing knowledge of the mouse transcriptome comes largely from one mouse strain (C57BL/6J). As such, it is unlikely to reflect the transcriptional complexity of the mouse species. Gene transcription is dynamic and condition-specific; therefore, to better understand the mouse transcriptional response to respiratory virus infection, we infected the eight founder strains of the Collaborative Cross with either influenza A virus or severe acute respiratory syndrome coronavirus and sequenced lung RNA samples at 2 and 4 days after infection. We found numerous instances of transcripts that were not present in the C57BL/6J reference annotation, indicating that a nontrivial proportion of the mouse genome is transcribed but poorly annotated. Of these novel transcripts, 2150 could be aligned to human or rat genomes, but not to existing mouse genomes, suggesting functionally conserved sequences not yet recorded in mouse genomes. We also found that respiratory virus infection induced differential expression of 4287 splicing junctions, resulting in strain-specific isoform expression. Of these, 59 were influenced by strain-specific mutations within 2 base pairs of key intron–exon boundaries, suggesting cis -regulated expression. Our results reveal the complexity of the transcriptional response to viral infection, previously undocumented genomic elements, and extensive diversity in the response across mouse strains. These findings identify hitherto unexplored transcriptional patterns and undocumented transcripts in genetically diverse mice. Host genetic variation drives the complexity and diversity of the host response by eliciting starkly different transcriptional profiles in response to a viral infection." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/4/8.cover-source.jpg" />
    <title>Genomic Profiling of Collaborative Cross Founder Mice Infected with Respiratory Viruses Reveals Novel Transcripts and Infection-Related Strain-Specific Gene and Isoform Expression | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__L27CoAClcEsxY9a29Y0lWctM-B9DY7Wom4B41-wuQsk__hwC62gz5iT0HOH5FHQ1euI6uc2_KNqFWQY8gJMTjS4M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"XW4SV929NM0DtDwQZE-8CUxh6Wqa5gN6behBFgd5-qU","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"459943","apath":"\/ggg\/4\/8\/1429.atom","pisa":"ggg;4\/8\/1429","ac":{"\/ggg\/4\/8\/1429.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/4\/8\/1429.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;4\/8\/1429"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;4\/8\/1429"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/4\/8\/1429":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022nXTzSSETkXaRVus4TtF4h146m1901GIqMmFrXkCHyrnnbEB7CETwzzQ+koCgmuuvKWxvtPrNZW20yYYYSvvBbA2SvM5dcRU3CLheSMrq8B9TUS8J4nBk+OYR5I6\\\\\\\/eLqaQ9yHgL5QoFhxjJI3Id5ZsE9H2rBtVB+L7D9xJqTqHYy19sxB1\\\\\\\/cWyt2KDLb1D1LVTNZleoI8ZdAYDIzzXyzrSAYHdb5nm0S1hxEQNNDi6jmR5CwfKAWkk\\\\\\\/KuzwVEwhgicU1nstCH5ezx9lJG46Nvdt9YsaweQ0p1R3GVlK2nJwGZ7CuSGClcbAxQurSAOzViX4DxUU6IgNfeA9O+H8rkRzOATggTsixE4VnzrnJhThvfU7R10HZq5xyMIKlrMCY3oOaMQCgvOcztngSl5RzwOSsU4XAjI\\\\\\\/tLrlJm51IE3SsxXF4jZa3COy6MoEn6hRjH87p2NehGAwA+teiQyQ1mhlG5CW0yQTqjqeWVJltn5LBiTGMqiDBKjkSz8Qa64QIOMsvJhMxUULoycVZ3eGeWiyEhPUORaIP+JfDUxr6JDGashWamLsSy53Q5QNDR1yejFY+RXvZRZDvOftjXhxvs2VSN2QQzHQeD+2F5\\\\\\\/qEo\\\\\\\/IxP7Lg6dafVg1nB1XRRIdqaLxZflE2H0Qwhlqg4fDfnECY\\\\\\\/wczXkMaGsWxE9qNr429ExRlE73IPv+XlZ\\\\\\\/ZKRl2Onv\\\\\\\/swTa8U8KSzAHpS6vd2ydpmFevZcPTa\\\\\\\/ok60vU26n8q1ihbl6kLrEgZ9KPIKoaPsiLsUNERFpozXDE0EVB2MLzx5YQffhnVPH2QqBtrJoaJ5wweBJVO1u\\\\\\\/I6HsQbIwynJZNnFQlqYbElKF8R7FMqzrrhRrWEoXU5Z7dfaNOs924RrDhF+3H\\\\\\\/h9zWpqDWkIh9wHeYnhDRpvL6t+IznOKleb7fHGQUd1FbE5SnczxgasBTV\\\\\\\/Z4sTM8D6X6R8ezBwrOQMnL2xCoyTHYHrlKjg\\\\\\\/dQqctNJ3ypWzRFVOykiXjRlV6qtDLn8fPMpTCGjnJw9294qHCaJbAU52R+hkHWcwz6Zc\\\\\\\/8h9y0U2jQGZyLLE+5u9SPcirGXnphdSZ1TWW16OqJI8KUvxFDpd0DzmE7Jx3gArFaeOs3gG27RP6yxHvKYYwfpahslOF5O+KMRySPEAFT5GOqsoWkHFPM5m5Ajl0litrgPbD1ljxH8U0rOHnk0\\\\\\\/qCsqBwomxn7voyODHvnMImbcdETXRoIx7InVJKwx6Mk8Z4wdSP0XnbIipFzr9qiVD1hAx4pUoC2nHlVtL6q\\\u0022,\\\u0022iv\\\u0022:\\\u0022gFJkGJVkXRGlwMKwWPVF9Q==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022b559112db52bd92f894a44b0851ca65435ae3f7bc383b9b323f4e582ee43e2d2\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022rfU8+VJtTQ1BXO0BzuSh\\\\\\\/PZ49hEwC7Bd\\\\\\\/BWPYiN+lFtIBxvovcShKHoLDItqNeYtLU+FUhRmo4Vfr4kve14mQnZIi4PltwKqlhK1Qa1\\\\\\\/a8A8MHrQKt69\\\\\\\/KPgKoPJZqV47YRzgIpV3TIZrdpxBqFdpMKGphtJc91SRSAsRHC8FcoL1OoHdZeYnHGtbLXPFKQSyTPDFgl7SJlp8\\\\\\\/xwkw2XnyjReLhJrFr1Gd6NeZyn3SukmQWNij4tzsmUdPLY36N7V7GvknNBU+yOF4U\\\\\\\/ayb4u0u8J0aIuNgJ+iQKpBaEVm2U07ybI8zG2YXAVncftubcrJnUYYTaqH5XvcMjc4a\\\\\\\/DFihOf5lBK15cmgNBFZPWJKtu8QO+qixgf70nrY9IL5dpRBcl9JOoFL+xfCB0zBjjk\\\\\\\/yKEyu9YYj71asdfl3b5l0+RFaRPzqU78EB2Zk9F21MRPnL0tmHmcjj2cLs6sNntF79NhRmtDi2b0+Gt+bMjWTcX0uiZ1e91H5+Pri2BsZpyjoEwUG+avVBVBY6vbIUvhIJrXPSjZESO0i9tEP8vSs5IsIza\\\\\\\/lrvCK4C9w8N\\\\\\\/nbrbpXIu9xwvQmEHDjlClJ\\\\\\\/CBTgwh0r\\\\\\\/EApK7bM0rekVci0LhvKBxt\\\\\\\/Tw35JavzA9t0YdX1NN0w3Yhxs8vCJFV4i6i1X06kJn4lv09VSLRrrw3k6E+g8\\\\\\\/QriierjGi5CycCuCPFzVwmcFRHqc7Zu3jhf\\\\\\\/GGrdju3X6+GhWXocJ0rfn5UyWaVyJH14dyN65eMdNRvdKP0ANs0AdMMaOnD8Uqcam1oLzVmYNWNLgS443r05qMCmUi9YprYnpKtD7zHepjEY338GEHKfX0vNciydoWtm8K9Du4HYYKsieePOol6jCh6\\\\\\\/Go66eGc8D1J52udN7nHUtkRjZpE2L4j4mtSWSnQfQ+367EBeZShin8gVFX5A8wW4RYAtPIePh1wkgYd0nfVxJD\\\\\\\/V3ldV4blIjJNzhvyE8ZA4H0AqlWA8y9cvUpq\\\\\\\/awu5bSeZ\\\\\\\/VtJ1S\\\\\\\/xns9DFwHJlFlr8evch99NF9Oso7FsyxifnWKkTWpqM4dBP+NXbtmGvGya6Zu6\\\\\\\/pWcZsAA\\\\\\\/P8mR1mp08jnp4Z9rHhNE\\\\\\\/nZ5jTbZmXbPN9eY7hq84qqN54R5Ljm9klBqaS0pFseNH2mh3uBzavxk8KlhEj1pzUOmRQCwiEUmMnwzlKKo8uNBT6BV6RB4X7UA\\\\\\\/UMOuLUDn4vWgTvCLysdyK07OEjYqykUFcruuut\\\\\\\/WsETCwv7lRQ3ywpFdRPj75kSEKG8CuKtoq9yCuxirNcRycDjka1L9v8Fr9InDm9jwJmUi\\\\\\\/c6GJrffzp8MF2lrMK+U7YgABp9qU59G1TBvPFxay14CZWfU7hU0d3noqLmpixwoh0+\\\\\\\/InC8KUQFeZI4V4ths0tQdaW82mO8lg+EGEX0qdHz+\\\\\\\/WxG\\\\\\\/i\\\\\\\/oZOz9f0\\\\\\\/HSXYMNBbH\\\\\\\/9TkIHR3FpiHKe6W9sFscd9jTueg+SEDXrxcalTRHpVyfOVfAbE7yFFefJgZr4Wn00Ui1AInbOAOjB8ESeSDwYd1yufObnonoOUr+Hhtv5IvFdpqy\\\\\\\/5cY\\\\\\\/37T3jezr\\\\\\\/9KFy9W0MynkZv7GiibD6mfZKExAGvyFSS0Gskka1qgX6HljewZcRlg34md635DE1SsmOInLQBxs0TTFSGOklPzxtfQwOI8iZ8s2fHJ5Y+NN2FaHEXxtzWMUcHxWJD3wKxZl\\\\\\\/2ou0H1R\\\\\\\/plWSzUhbhCrP1zdDE5Hn5Hwvc6R8jGKUyold9sQRk505SlablkDHIsIZc7DmIsfSkEAPBwEGDWdQLj\\\\\\\/blcqGsQD4RpeCeYzf7H\\\\\\\/fhInoTWIhJfFAtZKjotkNg6\\\\\\\/XI5OhBFqF18\\\\\\\/i2VC++NchESUeJjDl0g6yYH0fWHqv+Da3wQx1cv5sxybSv\\\\\\\/zCFR3omIfZJ8h5FD1isdzbWIuhxuhLpAUVH1WnailpXJgymzEbhM2cHD9VZ6sLSl9GKPEebR8FbuUkBEiP+PoKFDEwA\\\\\\\/g560p+24uBQMnHeiuvhjukst6Unfn\\\\\\\/ot4337fJbmZtzl8kifaMLTDrngNgPP1LIrDcqHP4\\\\\\\/IU6Q4yR+yu+A\\\\\\\/nUEkHDF0t696Uaxl5KmZhdGERPE33QAZzT00LDlvwJYMSs1Sou5UsFHr9XWIO2icU1cVrx7qeqLg\\\\\\\/aRcv8VBokIjcy2P73eFpeC1XH\\\\\\\/VPY0xP24kZFqzFEO05PdCt72YTBGrbUgxYpT3ZP11EQRKPTdtxA4halw7hOeMULfhOLrgOcfXLYL4RQ5Ql5cUirGQe335U8VGIR\\\\\\\/gbmhhILrMa9MteD86wEDEClWFaZH2yzzRV7S3SgL\\\\\\\/IOcYR+6Ih+6fYwq7A7rQUT5N+2cY47J3vIaO3N5Efx4HT0przMS4DcteErzNyW2l1lxYd0yRoEMD8cM3PHwJCgPBmo42eun6OdeSRxAEFNWqzaXOMjpy4DzCwIj8VxQ67jBOak+jmVfhXh7\\\\\\\/Xjg0hMfLCJWL4hAxBtMGroTIapwCtiq2rgzXFSn1LkUBUk\\\\\\\/X\\\\\\\/B7DTVfKHp7ysfmc2GvaJPg\\\\\\\/\\\\\\\/PLOGYz5z2szT5NK10Y0eS\\\\\\\/eKLrb7S+8QeBom41Itkyd0Y9k5iItdYEMr5qOntLipdSMQ6TsVvjetq47pYDPNhPVMiI1GrFK3Wqxz9W8OIeL45hA2RuStA1icxCAGmRe5t8AyzYoNqNmdwf+pEhbQPnOkUVb8tUmLvdTdRpNTbXze5r8eoUggqYnstjhzI2albZ3b\\\\\\\/OZykrVAxYYq2Xs7Y0jBVL\\\\\\\/AGU5KHKb8fD1+G6EH4jnuG4DMzKUpJQe+cusvg+Rc+g7xV2Tzntn+8ajrdUGB207N6eRoPii+xyAFLbv4xKoFldkkVETag8l0zD3MrKdJC+JzQMxpfHcbnvnhG\\\\\\\/v7KT5UXN\\\\\\\/H+1HM6HDGeyD+OeTNEJUVPxLNiKNVxrf89FCaqEeraIYq+7MSO7jEmplDoYneeg1NZGUoJ5U7Y37tu7\\\\\\\/sA404WUbXsVP5G\\\\\\\/9Lyd9e0aQh6agdYs0p43zO0EID2shydduhvD23XrcX5UaZjJwqpI+97EJtRamaC5tFPrnAnH2RUaTV5sSIgxiK7Ycj5Fc1Tne1rBUEA0RSHOZ9Yf0+Xz+2Dm+yNT3GlZu\\\\\\\/KZcukIIhvHXg3OlUOXS5\\\\\\\/+E8sBx6BeDGAAehzELWEgbAcpvUnMUYTLhpRHbuJJcR+XPdudLYg28jduWuwp9iNzKbToFo7ePrvZyba85+eBArgEm9VEf29Sl0AGH+swarvO4\\\\\\\/ZD6OihYYHGI+DD8WvSPLnkzIiuLjH8pKhZdeyLyAnDPVh2q1Cpx4kqaqSpaN16qxEwNaBC2iIDYOWC+RzUYqD2jryyublXdrFf8V\\\\\\\/2pSRKNqDRf2Las1YjTxNDFuUJzKlJPJOIFiqq4JSpuSI5kBy1LNEITbNdZ1375xWAR9TQ1d9wilVr+V9pl8CkwBRa\\\\\\\/l2PEUZU5lvbmmgtATraEP7EXHrZreYa67r\\\\\\\/n6DAnCWk32vGfSyLjnDPucGEYUEm8x6dq5F7G9VzyRQK2y+NMofEkWhN67foPW+Ufc+AL9yVXKK0UPVlbRhabzNFSwu4vgRhOmsj2NSXLnRMuB1B6qyZVmqISBgtS7nzCdJOO1WGkWAwzB6pCP4P+qyvxUx9ZAS3OM0QsESI4ho7nJ+WAnTXL8mZRSJfsdgZdRuf7ttsGM\\\\\\\/5AVoYtpfZzvhicsRg\\\\\\\/M2p5dc1BD0UJAwAov2Kp1rz9lE0deD412y1npylbMYPzLoZ4vcLxJMbyt92vp42fFhZYKrDV0PUWwUHNXZy\\\\\\\/yuT9i3KZB4heQHck3XpUv6o+E988INGmGOBMGdi6Iix180zMdP\\\\\\\/dFpFFSUEIzoCWHGtd3igSWV2PfF0xo2qI97Pfh2r2BjCRsk8HOAY9NgikZFU5Q8rJeyqTi0hgUOKy1KoYkROA3faqv2wagK7kbyxnazOLQHEzY1sqy9CVWdHJc1t3C3GmwilCga7KvftWsR4YXlmkWeJN4jzJgKbpzgXOFcJPB217aA3\\\\\\\/bubsDRGM27oXeInSMfNC8uBa12RF6K8BIZ626ZvDyRxvYSFFaiXX8\\\\\\\/5GDJYpkbA3PH\\\\\\\/nVeoYQaZZbZluHCGNOmnRjnp2plVdpvwFy8VWXmPnlz7nfvM4dUFzPCzkPi+fGmwAQHsI1FrSZxLk8ekcQuuMmKgGlITgTislcTBy0kFB9SrYY5S6o9QnfU9ElI\\\\\\\/4atWRaDb0RjyzhQB86KzMng4xXJQz2mKlj4ba2h\\\\\\\/ooYqEx7fdcFIQ19nAnIRQHSIlV\\\\\\\/El0HchEq74xru2ENYw4oPmuJWkx9dm6U3hfGBZ0VrxI6eEUhxxRGVK6AeRpQnii0dW8ARylGmvWK5DcUgA\\\\\\\/eDUPdQl98gRiGhHW+K8tPTq4smUyC7wiEUOmrfqGI8wXO7CMdlLusCgW+Lu1X0RvaN3ZHXD\\\\\\\/bMgYDxTD0m0deHtS7yf+LjPlJBmOYMX7n6GvpU2xeWxPitlnOTSgCxzlavx07Ro0N4RkL7EpaXRea7bVyz1q3FajTqMSg2g8vBmpXNcj2aI2\\\\\\\/nHn6ed6QyrvKc0ExEXR+LUf9zk9o7y4Kbls\\\\\\\/IM7FMjWsc6qHXXP+jGaaRSGWXcuLIbwgjRIgqMEwFMkq0Mmfbnr2Cy\\\\\\\/kvQscAmC1haF0ui5Z480dZtq8yhcMHw10bhd2gnqNr9k2\\\\\\\/MPYIkuZf\\\\\\\/5dwxYscMUgOjx7MGcuomkBVRNK\\\\\\\/1N9rBzwBxGy+e1m5MxI2i70y+MOJjQzMxFU3sONiIzDZe+x1ABgX7eJBEngkfjhxKFLu7Q6D5Q6XdOQWoTc6U1T2QIxMozCU7Hwmswhel4e8RvO7h7DQ8Dp72uiE2GYqGkBkvjStnYmS\\\\\\\/Q5lgkV+KYI4AnEYeSgavl+ahBBT3gRJ7HcO9amax9XACcDKiR8IQRASITVFlafIjXG3QxFCjEi1IbPy7ajQ80TBVveRBd5V46peWmv+VewD2y4BQG65hCw402ETlFo2XPLYbAHUphF7RDxuNxpKOaAU3yak6H39g7M4Bu+nK+eaA02hxH\\\\\\\/qedOYnrjusFm\\\\\\\/ipIC+\\\\\\\/9A7c9lFp0JvmkICD7bMCbQlu3IcbC5aHUN8XXnRmFa9P32wbl5L4aTh82fgosB8tWfr60jnrYtKJD33qvuWNKD9\\\\\\\/W2grG\\\\\\\/ReKM8av17AHvmFSIazraHIrFgQrPDdS9amQ2X1ixP98HEN9kOd6xsYWXrGHLqwrwMzJb61PZbRKNSOh9FuOSjdGRwgVcS0EX6cV+vNrrJ7ss1oIMHdeb\\\\\\\/IYnvPIgbGIlrZeOl1+GgkwOIEGX89wj+ar5PgONKl\\\\\\\/Ya4D2YdwwXxTtVRkXTEWbU4wzGiKdjKJIo2o6JdowL3udHytdqQJ6ksMZGYaywS4eNKSbRhinKoqQtnU4DBYZAgl8vF+s3TsfaJr8PGLQj5ilxFMP+VND2mKq3l\\\\\\\/SX2qdCRZXuZ29GlnGD4zBf11Ymm5x0S7tVKPHyHu030vGGdBUxRwun7g1MpIIGCG6hktQQveLT520NRCuxvvels33tVK7AaT1iaXxWx\\\\\\\/aLo4lmVIpNK4GoXWYBkFgb4\\\\\\\/wlWMHJMhLLsfsRR5WF8n556x5CLvFlKXVGT7LdQ\\\\\\\/w6cPaAy+ARlKmY4NwfQClGNbEqDhBfG4TFzhi+yB8OBL2yQ+yNm+Ewvw0N8uvVlTzpUKdLEj0rTHBxK3hs9euUjoIYiPB6Hy5Q03GlKf2pkX\\\\\\\/KzC1YfFQCspUZTFeDvDoVY3drRdS8yKiAj16+r3mtCI7eOG0Qk0yV6WU4ANSAjCBVV07sOGV4KQk+P8qxBGS9s0e\\\\\\\/UgncXt3ZHhp4qovsZLsJh9gaXIs1SlIxpwpPqQno93yx6O7Sft9ZdS+YEzBDqncePTMK7YEF2+GvNtnKsuz\\\\\\\/Brg1HgFGQl6WKt8Tl8irAvJSeBo18JJFSGkl0rT+igeXUmx4zHplnxfWnOtlFUwOwoB+VILyWJyzjWmGL1qHhlD60sLhtcWMBXx2gnhFGYDAsHqXTGKVwJXqmm6epYb\\\\\\\/jC+rIJOgItMOBfPHD5yH4BPCKorq0W7VoaM\\\\\\\/VADKtYbQeX0HvQMz5mRkeiwCuV3XJzqFZ6akK0qP0RtdoFqDOA9LHF0ruJZ07QNE+9FMT\\\\\\\/XwCEcOBoaGtBKmjOW6GRTTc+uYCAIFyU95WS6inmvyrPQ9la31RBMnwEixwEGxqiG4qd2u6yIJW\\\\\\\/EFmvS5Z9rigYhrRThYCG7jYSNoyp0bAEwbODwmtrJM3Zv9PwWanqD70Lb1BIuQJNh\\\\\\\/dE4\\\\\\\/5gvlJKQI4s9kHL1idBlLpyG2Wq4Kw57QNt3eDlvdBLKasWTnLf51S5smCo58K1JRKOZzCTkm01ehDQsas23LbODoHkj88Hd3AFTRMM+7wTVfXzM\\\\\\\/6kATbCX31x+q3zQ\\\\\\\/mAPcF2mE5azSPhD+xwjVNT+642YlFpHjsB2g3b5GlQz2KXSl8QQ+4hEZg9aDDZCntR+WfPILaRV4mEvdc\\\\\\\/DO\\\\\\\/42dphxuqNoVxks2tvn1cW10ae67IQmMUOy2r01b0DT0dpY29+mgW7JafkUWa9CIYzTE567Coa5AB7DdtgdmoKDNvALeQe\\\\\\\/2vcYr4l8+M0NbYk+bHu64XVrhMZRKHJOH0AUpnKnxv9lDje45xXYOjckL4NTbkPPQFM8CloywP4TQdb76Y\\\\\\\/VBoCLcNTJJL6wSyoFVrAL87\\\\\\\/+xwduh4YDmQyaq5lU6okX7sGQfQROujhVqdGzNnECgG2BXLqbjeN4NEKdWRhtxr01WkdqKENIWh6SM1ZU8YhvYUPZft+SPihYjb5\\\\\\\/C7JqwaiQKkzAESqKd1liWUlCAfTCCft9cAJvVFofidU\\\\\\\/e0JT\\\\\\\/Uedx1+Ofn8ide6r0Dmr4zl6T61XrP6kPe8ch5qWqL0AKMMQiWI9iV8qz6D7uK4XRjM\\\\\\\/GLaoamCUel93rmgNIWtvLfa\\\\\\\/6C7PQjgOWVbwfE3iboKQL6D4wYFSrVHJ8TLHe6sHfCx29GDrA5\\\\\\\/X+NhvhxEuaMnMTEwkuOnL4qfom\\\\\\\/GiSzajN7sh6aoc1rnGkMve6ByC1IdYW9nZGBt3h0g9ZdYye+AGfmpOKNwzZJHQN\\\\\\\/kWkA5STnPGaMpuq0usn3PJ1TQxoRO+n5qVzPSSOiYsQbS2SZkIQ49Ndb\\\\\\\/GH\\\\\\\/XZFliNsSU6ESSgLAeByz9CF432YKq8ZZgLGC5TXlAu31yjbrIppnDxQEcInuTOvPWQXDPysOhtekTR3WyEYrAIq+9uhqILFEtpSOFhYMaJaO9RMRv3S2qYyN0TGSvyNN22pOWYJwHD65kM38YXplE2yt81GWztL6o+RvHUm0WQ\\\\\\\/70KBOMLdyrMrA9qBjWxzHuI3Ljb43zllImlu9n44MwgtkVzaO7AZqRFJ+aL4xokc3\\\\\\\/cOcF42DBDQIn9RTB3Rhac5tOjbIwje2fJhUkvOyC84YBngd02i\\\\\\\/K9ByUcOuK9hktmdFjcGjdorfRR7a+5LuFZFQle8AqaoIF28jTGjsFB2PzOleIJE7hhZ\\\\\\\/\\\\\\\/1jU6JJTH2CgeFyr8bX1crjQQ4WAhvc\\\\\\\/vfnpHsTw9JB65m8Va8o8pFAnW0uuAFRixQtOZarvn5FuMTt9rGghiM9PYK7MGG1Z5XfCWej5r94RS4F1CWaz458lp22My3XEABBYZxNN5iQ1C6LKcpeLiFk+jrijysBAwnec3r49Le+h5cloa1AHJLRN07v90kvGiglKaIE1HL72Se2BaOPdXW82KAFi+3xicC2FsE7z1gAyPEr8J9Cy7fUzrL42s+VozjQ4P0osOsC\\\\\\\/lx1ix5F553BwNQlOe\\\\\\\/YdasjHjUC33oSKr+Z+o7vIbUnRVTe2C1ZsYBINUCVLdiB7\\\\\\\/sSTYtLFb1Kb7\\\\\\\/6ZlwVQziqQeYhakA4puiEWJXMsQaM5QcpJ1ou2dns9CxrzWH3th30jAYw5ETc4DEd0coW+5zk0mhEsV5o+TPGxNImIdaoClgYcyWdSVXxZPopI7+jOc5G4aEMUGNjtSfW1ngMlToe7XnVaTqGCaV8z7vAx+fQ8eVZ2r1YRU8H\\\\\\\/8gB+Iirs369FQRGzrzP2B51AptfXlSqSmsvMK0XAhRmrLdPGoEDpxgyPUGNGcV9TPn31JW1DP9nu5PVDNLNYy0dGYE3WV6wbpb0kPaKYBKw+FCYEGQD\\\\\\\/KmqF2\\\\\\\/qW5eNb8oCkphRwE2bs6P\\\\\\\/khmhhCANnV7UsI7pc4DpSaHw6tH91Fqf+MuVJsShUAIbFU7cSmzPC7SszWNPR1MgcrfXCvPCKRWhArETZaTLj4KFYuT2XkFZjdgVX4wd5PZdxMIpD6ZJub4eX9AcltliapO6TCUOCTd6rotBIvo0OnUHeUfvqiS7186Cjp7OaOqpDk80scb5bZT4IBzJPr2hNFF58DiidbG\\\\\\\/t82FqwJ8ideborIbjxo9vhGt5C2y3rF2jLY4cDSSyrXmoipZ54LSVUIc8JmgTVFK+OSKx0M7GxdoKKqBqX7ere6jZjro+M+fFJcVWuRkz9JDbl8b33fTgCQD8yMVBNP+SBD69kWUV1DEEwi1phNs\\\\\\\/naLsjW2m0rrfyn7HNlWfyjpIT9A2x1k1V4tZTEKd6gcAvXUfI6N3TzX5YVxRma8u\\\\\\\/vFTxKqMhshl10Fqi0UFGzYUOrfmlh4dbZjc1\\\\\\\/LaH8EOFQxZ8oE6DhsqpY6StDFUNwdKo82CO0eWsO9r1dFrSc8+PPxTrRMIeU5jvx\\\\\\\/yanq9PFCKr1H2\\\\\\\/rHOkLobDbI4A49gDfcARWgVHuNA3spwp02rC8cESWjqNwJB4CZKG8SW+hN7N5aMGp5XnjtOLQ6diIQPMxHvM0ZBaO4lYX7JRQu3R5cr3G89iwJDLWYWsX7U6LwfBwV9vaUpQeDFiaq7HlegkUh+cy75XViKFny9e3yYFXOkPB3dyFEjdOVI3RO3W7CAcWlYlQAyMrIzukHkw9zuUPGtjUldmrvulvRZIARFph9S7vKqW5zDiouR1lQzTfZETfkG3GNoiNX20awuzEjnuBn3GDHZrDkfZ5DvgHQkfoRNalucxrFTI\\\\\\\/5C6\\\\\\\/0hRYgWKr2lByWpo6CX95HH1I4HAVxvaY7jFMAddom2UFH77YV+igibeNZx6vIh8PByYAtfoL7AqYsLyxyZzTj6ZGIBCfowF1imWW\\\\\\\/PXk0vJo8t99Gd8o6i7J4oXnzCE1ZbVxiTdOnV\\\\\\\/HGi0hWeDCs5gSb8weDXnZRT\\\\\\\/S6MvK8YPdjbDny5bSxf5ielUwXs8ilJiz84pr+C8a1rzs02aF6WqJIiMPD4tiD9nSliBh6IYW0+bWwUy6PlyUFJTaJjy2qzHZfl4N\\\\\\\/JpgpTcmr0z3lLogVLSo\\\\\\\/VuufM+Sy15IUz8Wf0lfeZ4jSYJ6n9jsjjD18tD9pJoYFvMpXvQi5Xz\\\\\\\/h6Du08sGUJuP21GaZPXBoNh3T\\\\\\\/mCxl0jwmgISgH55vVJchjZ\\\\\\\/YynXsS5jJc1UhP420fWqwSxC1TM3f88XDeq+6b3pghNWXZSE7ailgkag+y+6v94CHM3dWHHgJZDYVm1+umcT2oLazkyndpVqtg5sjcpRHBAsyt6dJ0NdNb45Nr5QgGgJdL1EqxmGHxD6NTDyavGgUhypV9uVEsnE5nxOxCFqJdIi1m3K4r8FjY3TSG1DEIJbgNPBEM+PXxJVyR4pOYo3RLylXMYJgubj5OjqBmU26FCz7z1wsnYueaZWzpShamsiEvCg9atnvEm54LWzaaJipZTqYxoNWX5HQMSozEuxPfcpeJ7mMSAt2KhHllvtohidz3gMhZMnr\\\\\\\/vL2a+XfytQ2g+qCu+OmxIfjck1bbroiX5e6E8Ia466tUHbQ0UQIAwX+VgHXumHZnwT6wTPWE\\\\\\\/WmsmLaC7H2m25T4n1PWyU34E5Colf+thdxsyjN30YQ2j9ueQyGHP5niUoyS7y4VMThUM\\\\\\\/QXlSD9Zu2eKdKx4dxVxPbGyZ1p6WZlBsVniaH2x1rVf1VNb8Rbu0rgPJt0gTcpJSCDhNF6ibgFWRGBywJXCmz3ieAOOgAEIb6Ez3U9NrhgdE21D3UyPgqhv9wq2FH7D35AS6VS0qyHC9yNlxs4mDLuKiRJTGK\\\\\\\/NUh8c5550ABkAJAjl6Hqkyn8W4LM6XQfTywN8FafQQAYKvrQonjvMq6202iybqT9Mcx9O+EiIZ1lIToCqtTsfwmOZo1zQRWEQwqdJ9ay0C2l2QLOTW3t7kWm3ZeaQ1NvO792VvO89gIIIII\\\\\\\/NYO5LSBizpuTsOdCEqRI6pEwkijDhaPz71uLPI4wjj5KLMa3PvJLy9T7sbWKxdgKzrNJcCop8XsmorXCLMApRG30BklsbHKn8ClN0Q4Z6nhh8XUT5e+nil8\\\\\\\/QBFZInXzAZOTisZGVtiy3jqc3Y6egO1VB8Y+W\\\\\\\/NVm+L9oK3rF1G2\\\\\\\/xQ73mJ8dVBW3rVfSx8t8paalZ0kX0X2nbhS3yfPW3DCwu8rR3Z134l8hwlsDbI7KF6celS5HtPgkG05JwE71MKqZHFpAJhdXPB9TxOCEIylynMcgkyGNE5OqxfSovko8ctIa4oCF6yufpITOnqOKO5r1kkWPhhtgrOjLYqUqOVYQHKzBG1tcJFmmoeWHmbEbgFnFPEciDSOSzj4F0+h1q4iXgZU4mIT8uX7YfajHmnvN8MyMPQoMb60OpSUfl4Sn7bPbRfgA\\\\\\\/\\\\\\\/K6vNEOpCEZKefID\\\\\\\/0lM5CCOhp01Dw2CjO6HjA8ByASn3gbIGIwGhHgXr5fRXZaAHYw5+Au2kPXfIbAQmSAdEcb4dKf8HjIdFLPFWWVC4pjBIktJ2KOOPw2v\\\\\\\/kgIUqlG2jWaNMSCU34spT5agssaPJUMIhlOkA8rh0lceRIfH9RbymnscNG3DE5z4S\\\\\\\/ZTUGl+Hjkin6C8Ca+EZRwqJdZdOEvnfguILdQeQABsox4WDchbuf5WxSUXyTt85IFWbe1SN5\\\\\\\/zoi87GhpJAa93QttyQ8GgIenYQe6Z3NdAD1c5emUvfkr3WHGOeO1vso0sYZ\\\\\\\/vNN60Uzm1rVOvyd9TqmxZ9\\\\\\\/VvoLbsPx5j\\\\\\\/0eHcHVthpLduToTf8hJ8BhyKhDigXWP5eadHpTrO66LN4EhfW3wZKsg3bIoQl1H+UUDdg3MtgiUt6bJWLD28tIIwjMDKQWkRDeXhr\\\\\\\/kAjgTHKPI25t6vh311blkh9LVGR7ml+9Q1DOjezRjoWZLah9PBZ61LJMHR\\\\\\\/6FecF87+HVxCiZBm\\\\\\\/ZijXWjnXD2EDkUZiAiIoin7C74xTTEpAEbuvYCJIWouAQAsXUcH8wpOYzKj5muHdgVOSqgC3a8oGky4DwgsEmrRTJKHiogtjQhd9CzLuXQzHZa8bw\\\\\\\/zOOKKZwIaTKORLSBQd1uvRnQ5MJ2Q4\\\\\\\/I0xnvpK9BTDWw1afP21iVy2eWCZH\\\\\\\/4OOxDB49D32by7YL+tGnpOOG+WR5PqQhMxZnDRceQJ8mnIQGFDs9h6C0XQJ76E3heiM+4+7dyybIAYn2TVMZ\\\\\\\/xfmez4\\\\\\\/xNVCfE+eas4edc9LuZPNycH3wVQiM\\\\\\\/gGVEdQKTwZp\\\\\\\/tc6L6aqvjtOpIJ3j7OUicMWDSB+nydUzkvWDlYIyeZImLFPyjRd2b5yMAeAWovOtwkgbdk8Xlg30ElD6B0sBkPtyRSeKE4N0AVPzK\\\\\\\/Fi352qMRoMExQwllnkDeG\\\\\\\/8QUii6XTggwpc4nNqZLWOJ6SP\\\\\\\/n\\\\\\\/lCuUZGIAAPu2Mqvsmwxf9O+XNgo9BywZ3MnFlQrwV74z8tokORIdPkmMhhII79BeJeQ+weG4SyHP6+zwrlXCJ\\\\\\\/J142GYU\\\\\\\/gLKQYxvgk0VdFTfB4njOsnfqMb8iEietAVtWgIpjnqgllK2O6LNWaWyC2wRd7SWo8SEFwhDG750typf6ZU3kKSLQHqexSYlk3i8CZfmPVTkGZEjpOxOyboVRSasAFneTfUSC\\\\\\\/+P9uVmFnqL\\\\\\\/CpJ6QDnGFBdE0A6zUXXkSQGzOgFR0RwWYyDS0dYUpNkhPXkwYn6VTuSgPx\\\\\\\/Bmuiv2cmTeJrs7vuSAw8BZr3DWImOziFGeoMgmd3DoYCyflwrAbxIJrWjenDfDyUfkQkvENC7R1x1aJRfzMvse5m5DNhqgmhSWYbdkiytEU1pGaUNqdYnn15wMU9W3h0h1ai2Iqu6G+0u00Etl92t1Fd+iiZCry8BH+2grNgEidunvEj+unlNljEsKZOALbF8tKOrSOKY4\\\\\\\/YFBbno1PTwBRripIQ2py98VRBit9v1F\\\\\\\/HI3RY\\\\\\\/eItBdIww1esKGKC7hCIR4FwSKOTUps1c5w31iiAraLz271ox5gC7U\\\\\\\/O6jG2VDWRwBeZkZPFp2qNU0NMawgVHjpZTus\\\\\\\/U3Aba\\\\\\\/vTAAaWdsfGuH\\\\\\\/oZi6crV9mhhXwBjj7y0Mh2HpA3XOzk1XR7x7\\\\\\\/K5sV9RVfwmYxq9ruMZ8WoDoQu0JuZYUMZbcDLnf4jUdD3tvfIVUSiK4IxK9gH3WbZ8S9ayjFInnAaMdctBRCu3yFU17gZj2Jd84lLKmLs020tzAhdwkzn0kQsnHeM59ceL87sxsckObmmxVB64pQv6u+o1Dti9YrpnpBAsPi4c+FvxzMM0rfjLWRQ8D3AH+l5IHuyz78kZuQ85QRxGCtOj6veqzz4bhTMGbhGpvJ4VJqeFaZiqqY6rB7iMzkPavqkBTmBBTAFpXGrQnOJ70RJjlhagAmVKx4wuEXPu+Y7\\\\\\\/6lUBgMqb624l6aF744P2nB2ZedE66h0MMus5hVjNDFI4zrXalTLvbIZDe1t3AYSVgwN96UjU23TtRqovoOuRiEfCmi+qCuI1LW+\\\\\\\/RxpKyV8E\\\\\\\/eMcHK8kJISG8Sa+lC+irAvw0ofdE8oGW0IlcEpswC0atRutVah+6FyhLrju4Y+cHtViCyP9AeZAbzaurdGFd5GntmAxM4Lyl\\\\\\\/qQFHnIRYywIRC5fs8EMNcT19jFRTWbwNpfQQkYoZrnxiT4sexderZ5h377EdBME4rK8axiEh3dKOk3ME12Sn2QeFqgdfdRT0kbvge\\\\\\\/nuvfyBRJE0ARv7eGGHtaOe0siYarzy4ac1AkMiHbbTATkDgtMZ5l6FSi\\\\\\\/pixq\\\\\\\/eYYwMpNC3BXHfUNzHFhKbdQXIpYGbODUqJ5zMQwDeRmtv9ze0Liq0tnoZP3FP7+rKZ\\\\\\\/\\\\\\\/K7LbIiHhrXagJ0UW6jvztiaiRuE860JoJySQP2q14c9US1C9hrgWQwFy7xUBZJYlFzQHVt\\\\\\\/KWz0YbOT6CdOnojK4IFoIIl33aFjcVFtA0vqM\\\\\\\/31v\\\\\\\/Ai24bdRYlgrHk+hIdLWSNbANQgZ9SzBbCNP7sDc7e+cI09tmUtz18lgv3MUvz3SVosGAyAlkw0JWO4L8XvbNAC6naioPWQL9ID+aB47EiIyNjrCMA3EOMGggxSONr5XToNb20fHxF8hYLjzNQAB70g\\\\\\\/eNmlWXAxil93GXWQhUtbl6BGi\\\\\\\/e1K462EBD9SkRAbY+PbrN1ZoqYui1CnRZyH2N2p\\\\\\\/ttLwDUIunw0JMKyvvQjBdY7wWyqP2j2wQ\\\\\\\/xXBVo1XTKPNFcItUeIyTuxfgU1kunUaoH5OwvVTSOAOpuDYnewm\\\\\\\/6g7bs8TCZlh9PxZGp+ghbMIr5w\\\\\\\/UkU\\\\\\\/wHzbsjy0UrtITrcISHREnYf1vtN7w7d4LW3sl7jcsdzi7gi9g4FWcR3Ts4zZ9Qsg\\\\\\\/2lxLp4yt6M+aauMnHFKH3wJLOAMYkNukV\\\\\\\/0sTO\\\\\\\/Ri8TaI7Z9b7qtq+1ng\\\\\\\/mP9+wV5UwbTyiFEkQOznZntz9vU7nkZJqsx2WRMWomV9jfCv8+pPZoY7i\\\\\\\/QWFQMVImOXom\\\\\\\/1BqLppe257RKrLQfK16wRx6ZlqsuwnMY64K2qlLMjYhGYz46DbX3cP3O8BwcCS8cUi94rBTURK3TXie4+Jw9gqd\\\\\\\/AnJ3p84fInypZ6sgDidIqpcHns9YkxWteA97wuX\\\\\\\/6tT3P3v42N0UJ+Cmt0suypIXs2tjNnfYYiiBvPq+BEsJWbAqZHzjDECPfrxcsvljEhHV3awGQHdQ+kJFHRVTos2VNk5P9cdmrXHlj0tNJ2EK8JZ6sKTSf4\\\\\\\/jKUHAbh54+7MvXymjLeRx7qdNseyL6XUlMXRsRYo7Z1h+C9FxkCjA5f0A6SnDSjZ+svR6U5OD1q24SIXSPNUe49\\\\\\\/bnDFECpB\\\\\\\/LP6WnPqfX9SX1xQVKc7\\\\\\\/UCGFVvYPQgS5P1HUUkSedJubbHWqjAQ+f9+I12cWt8P4k2YQnby6b55rC0kb8h+f+4ETm7KBhKmh36hxfLWPLB03n3Nrt1Q0rNm8X\\\\\\\/AnvELuNTiQ95wuy7VkdyM4QBRz90RILBInDq6QaNYyAe2\\\\\\\/k5QJ8lRNI+BTE\\\\\\\/ctRoITsHKXmvaPIub8DyhMXCNoncFeexxUE1oyPaoX9uV4ZbbnXt\\\\\\\/r+3wWlPNLlrvhMIfa1dzl6FAozK7OtTAkUEjc3nWPBj+b3f1ev9xQh6crBkgfZI44V0+E3FuEjmK+sHq33no2nSv7uKSwOAjrznAKdEnFH5ChwKTjgloCA2nFXGT7g4XnZ8InpJI4TBJZFTCsSJMO8vsjTdYOX+ilPb5Q8ldzAt9h0KwPBAWss78WmPxCyjScwJABsW56QZegx5uuuYyZDEvrSGaWTiNsXy\\\\\\\/VbYMMoFvMZnXjs9H0g\\\\\\\/ciy+WPfgQG\\\\\\\/wWkhEyAC1DAgfqhZ5KUGU9BFbn5Q\\\\\\\/4DQl8gtkOGWhunTe+tUXDXlLvVjk\\\\\\\/pVpOjgcqgZeR1TDNy8S9jgqvxP7ghT+NzC+L\\\\\\\/qXQ1wbsu9ePq75TOBppj8H1V265asCdTiFfd+l5ApCN2WULNLgnBaOH7iT+9iq6MPQQMWlvpHvHzCZquYIxiZ3oLwuNQBDO1UDESwfRsx7LSvDzpXxW7Sd9rTOnA928NXfpReiSEXn0zhXaEGRaiLHGIMJb10InQKKFZzFByNv+7G8ONowqRuuk3BkvWD9XH5R9tRemT\\\\\\\/f57z2GgFAyZ4JEKg2R3yNxUvZoKRBCPmosx6BHlq6tGyp1jTisc\\\\\\\/uqStbNfRrmCBYL8TR7a+yc0DDJ5WYLZuBdot+WaX5mrKzR7F1bP+mSphuAobg\\\\\\\/WpyO99Q0jobVTrLAqJnqVAKB3TxX6ojmw6ybmuDu8Ymkd38pCA7zkRWOerIIvPTwsCDnZzkn8FfOVuyNnPO2oQzZsGsmmcjNjtwBKOkeGqHS+krSMenZGdJgDaMV8BNb0iDgHYlkpDdA28mFbwrlCp0DgtPk6Xr1tHSvPVON8InFnA+c2opVXbPa\\\\\\\/rBH3yeQJUpiVbg1lw0S79XK1DDe517wKaYCXV2ucmj4kSIPGdO4L6CE+j+V8V99oBBA+Zrba8EnH4Xt8FYxWDnftxUYWfepL4d9z4NyxVtVVEzXbT+\\\\\\\/PEGEdB7680H+EblquFOtnslvb5lEnmwhPOeAF4+FEGt82CtCTNylsP1igtJ2RBuO8xmF+Vqw6rXBnkMLyWO4imAuXTBzK1vONZ7Coe6agjbR5+EBl3I1Is7P43ozmdawMpgZP13DxNvBJ+gzsQfvtt87vwEUwGEIJLjmGj0x8pN25uaXH9euZdBdumPFzh3YZETan6cShPy5QXDCUKlwtkZZ\\\\\\\/2ha\\\\\\\/Oz0dyGDFoiUN8C9F4h0uDNlbom+ZopOij6rVxC1Gv1S60kI2pzJX8WvqFBN8peTh5Pu19Aj8uPNuHsG8I5\\\\\\\/9pMxPTJYK4NiRvjd3u8GnXz87l9oA4+amPPVQOdGG1gO3VQpEyX4uN9DTfyrvrX6vTCDS26rTemZaH2B5p79VUZoL8ydCW\\\\\\\/WiPNpdkQReLQyGay7chAIaXS9o13ZmfITV4Kpw1nFsc8BGyhAVsOnm04aTN8jrKyxhgsWOUxrzDjpCAGgukRt4k0hg5unDrq53oQHeprbKHdTywRUTyv87EkyfaiSbWfeFx0RIkXHIAonb5SUcmpq1UJ83FqNYeS8HmwZ0nEvZCCnsS0WUYGIH\\\\\\\/Grz9wgJxPKD4QIeiCzw8DH8izbZA1eWYOfKLIW+g4W9432i9X\\\\\\\/qdG9vDZA7XU\\\\\\\/CXdn5pTqA58\\\\\\\/7ARR8s10zu8lL4Sss2iLMxCss4YhJvDKefcXAUvdlZhLlK22LRCj7jcP25HRk0K+TK+lpTORXsp49F1dZ4DSrnyv4xY3rAKfaA\\\\\\\/Ey6hUHH03aOYORWcGiZzpCK9kxt2kqhQNVFnNPJcioY6eTdkNm4mqN+cFB50YmLhEu1X\\\\\\\/oDSx8JSvpbfOmS9O7iLKtPQlgFjfFKSItE3l9M7qNBNpSbXuXpdzqZY8GQ4cutj\\\\\\\/B6CuzPUg4KQpFIJXa718H12Jj5Vtx2ype5\\\\\\\/Il0iz+3xtqVzUux2svrcLUCpjatVkwUrstY9KOWu3NVWay7gMRLx+2aUDvsKgTaqLUeED8xjuPCZ\\\\\\\/gGCOefaUFT4tkJSR\\\\\\\/q8Be9glsqGz7Bk1cDT5iAwI8I5Pg+JasAu6RZwzSztaGBltnW8G7DCxuIpm1qx44sdHXf+MFaimVD21HYjrFXZ8kgA1+QhRa7oRZyw4mOCJqAHL3ngoedo6TsQrhsIOAibnXRhKKSKYsuR8OFZPlx0\\\\\\\/8zw\\\\\\\/x4MrXnlfnwllwG7YQO7bFxEQtl+IN1YFB+65iGEAMF8aaSP2fqPNYGDRMFbo1bQ2ejyAj8p3tc+tg6qQgq7hgDfLAvsDirT2SuiLu94psYgNVvTItq6Gg1LHMZ3Ewrrr+j8vNuZJGkKCa9MbT+04T3e2tMp7v9+fQO+S9oiyaP54YD\\\\\\\/aGTefY2WYDlyOQwsR5vio1PTwm60ujC8yrBt8sHmWoRVdZTNYC6Uqk12cQ4L8pax1ZSMmJowLwvZpPpJU24UtCuFZ\\\\\\\/+ncvuVI3+WbXV5B03RWpuQkb\\\\\\\/LAzETlv+A1qP6zYNXLtW+rtU\\\\\\\/peLedC2dFkMWlz2YvYi1klq5PUQxEADg6eP8Gra4bf2tV2HeM3ONYssZysEgIPMoWY+evnIu3cE3lMZ2WItTKuqB2X0rTukaxgtwqNkruhK63W2dvaQ1EpSKt9rfeecte3tcpMEDsdqSWG\\\\\\\/UdTDPHUs4stOcyFaxcM3qJ8Yh2du8\\\\\\\/oOUUcWLBIz649MG7XPBpcFLd1iO5jn1pH6j3Zy5toJxMjT7+esJe9SIZ62eVcHqSzVpl0lsng2exAfMs7O8yyIV117apuN9fB+akQHTxsJhg3TLjDWlPgPzvOJeFPY6evE211m\\\\\\\/9fZChpMjhX8xbVTOdtwAxVUzBBnQUWPZM1lEiIbgb5P6GQQbeVOmjrR3yAr20fbs0xqq2VnenUMDn4CE+0lFD4cCOIRI\\\\\\\/JB7W5L7MS0195\\\\\\\/H+3dC1+kiVag4TM9B8Qf6dAW9e4wlHnuUHJFloIjIxX8940MoIcFph4sWQrA7bC0H6Pk3ZLi80TKVO4S43MS4UoPLJ5sbEVAve37JQEvQcy3Rw7WX6oCJZyp7XCxqXY7BJnaaPnbDz5Vllfro1CIS0qPE8QeuIahcK344f6V6Dd9ifYs4joLkNOaR9TaN\\\\\\\/2z0oUi+OUYcdTNQbQlhjbBrO6SYvvawm9gnQfm7M6BTYVHvJlmM0DKdILDnTQLbY7bcaMcBrehez4aBJkNV8b4cvfhpwydMQJXW+ceM+kpe1QUhGFdEjkXMl5B+uPwr5+Y4wg3Ck38yYn\\\\\\\/q8DIK0utVMHolYLCNqccPq14fEt2mS3+v61km2IaOb5dn+e8mDAVaLEkqYzPRYd9QfdNCyR7e4CG1Rt3jV7v6WSGvxHDZPNAzo5ViWblu69l5aCVEilWU7WVJ1tr8Xj8dNDagh9Bv7vGcJq62D7grpw+5K68NmwSiejz+E+weueTJSPqrHZhotr6d5qvNumopK1BFDq0WT+LVH7cz\\\\\\\/qgJPxn8z9g7okz5+Fo69FPGihDGVxKQxGWqUTdQG8MX4BWLoCv5C7mN6G0pqSY9ZRIiuDd\\\\\\\/Et7icO\\\\\\\/nc6t\\\\\\\/jy\\\\\\\/v7+2W9dMyqwnPjZzC5dzeXtv\\\\\\\/Sm4\\\\\\\/veDQS+XyOQX\\\\\\\/w3\\\\\\\/T9ucYo88IUp1VoQitORKNXcJpQ77AF+VIKs4dLz2fjaDs2oUhORcbWJ24dDDEHN2yE61Hd65wPsW9aRIr9ucQrSUBilXCK0i6n818f1BDOr9ElXMiQ4CLX\\\\\\\/qwbomsK6fddLj3VkelPAeS4KMxLz+Ic4T3kBvhDuJ3NIiqIUQ6fwoh6ZUHi1jIg2nSOvI6yJBeBI8QwlYh5fcpilLylAXaOSpKqXMaUQXtxmJpMiWScING2+cEaqI+W9+HfaZ5QTBvbsdhn6JCTN\\\\\\\/7CCepitIz+wqQdItGPZe+w\\\\\\\/UV4YnpJfudKG0JCsamYm9JAeTbTkxRZlZsUKw4QFOrqk+R+iff+izhBA7fb9KndLGrVEu5VgZ6vHKDOSqA5vC4ObwpRNVMXmkI0D3jtEKaI0gKRN1hzO2CtZB5OOu4L\\\\\\\/iMFlpiyNvo82lHqVlScXAV25z5VrblLS84i6VYcv56VNX4Y9btighJc9BwvJjTHDCXxWJH0Qy5fR3uDUY\\\\\\\/Mv960wdXHRDnmVL5kCCawlLOTm8yajdSZ1FOiDw5b2VG9ZV8DwMH4jMObudy437HaEF4LFbndLABmq7UIatvpOwTMAv2ki85jmYvAAEDI0apJRCoHcxHbgnkGi2lzjwP6VrMtMEtRJEOFViQvIsITpqguR\\\\\\\/xS5\\\\\\\/qkKq7HhqmsdYv7aHzouLGK04jLHRdcBjpuggJQrkjmGRG24ZUH3a3nxtWBMUzUaDCZBpwEza+CceN7pzCvaFlOu1DFFurKy1UKGR0iDeK4annujIbrjp7hck1UVuK29wvT0dvBr1k6wNFZ3+0yHefHEkuOc+9R4sDtxFXj5IMGtw6AHzYv2yYZMkk8N9KJ+u7LGJZPiRbuon6+gcTjZdXxrgfoQBSCHGL8kjMSj6iY7\\\\\\\/2wwlZDgVReegXKRJjUMFRg41VE5O+CGhhoB8yHj\\\\\\\/2MKvC\\\\\\\/5MQ8Ls0sjRNe3C42x7qKcPt7RDBejOHA5mBn7NY01MoAbsjl2LGcsfBxs\\\\\\\/IW10qh68ifW1mu7QTsf0YdLEqrRQBP64NiEdGY7ncqXqlSwmZHEQR95i4K4i\\\\\\\/PKKsOQFk6gQSoGhFPSYUEJSU2CdItMRBogWuzhtGBWr7msyBOi775k6\\\\\\\/0S\\\\\\\/GlM57G4Ca058YCcjxuzxtWrNb1AeVlIc0ufz6RlB\\\\\\\/52CIQ3E2mDCRigC3DqxXX9DVW7CrkPWtRp52162xUVZOXljJG66H93J2WYpWo2hmByO0BSIlpCIAni6wKl244C3FGOld5dwZ\\\\\\\/DYaMIf26G9cpBnxmmbiRdfLoCnhJmXfg6Ou9UrR+XcekhYizO\\\\\\\/nyaKrH9aFtGdm6\\\\\\\/nm6O6f7\\\\\\\/A792XQe0f5aaYLtuyEEQ49i\\\\\\\/H+dLIILL25eoCQ7dPSPgwPMCjcr4tPhcEeMKPcv8PkZbi1i2ewhfSJOVujxCjchGr5Cdr0kxfXe5cCPS1Eh\\\\\\\/f8gaKGXRfzy\\\\\\\/P5bn+9jSI2ERCVNXwXXox\\\\\\\/ix8JpqE\\\\\\\/ZksG\\\\\\\/hTt+CPAdfzEXLGm1\\\\\\\/4\\\\\\\/oN7oKV4nlbLpBLD2VAh1mb9dzebVrwA3lgS3CFyre\\\\\\\/epfySPZZSxy6WLHu4hzV6DSfolBSlun24p84I6P\\\\\\\/liB1akzJ9qWgT8z5agiaOiKm83kCm+B\\\\\\\/ybWHx3bNzlzOCVIHHjFUPU6xRd8gTdLY88Z8KHPDbErdji69AuASFhPyrp23+E6F+LhX1QTGMn1HHqu2OS\\\\\\\/2l4GjH6ZS75a4cSwf20K+WzWSRdab7mCW3lBV7lPDE\\\\\\\/K26aQThzumYqOaiiwAAlNEhIg2NumIXJvchsaxSSviSx97kF\\\\\\\/ZLUhFmL+65GPsjh\\\\\\\/GLFOqSNsrZlHyES9ziquRBCZPedgDwwyRQworcVyZEvwDaLtT+UHoPRGC6W1XnMDWZTniMDef9DUtW\\\\\\\/lBr8cMwpbvui1Ie5hgCcACC13zocY1\\\\\\\/QqYyf\\\\\\\/xAguW+1LXbssIKc8WmFicN9jDGDWc0AAbb1etbbyNPxijsR\\\\\\\/6+MNj9+mHzS0KQdl5u7UBZajXJr72eSuQkQCpH9Ncvt01BP72AJLNn\\\\\\\/1dbinGqHo67qx2hqX4kzdL9a4HuVVnyaNcTG0lKRfZssAk0oGpmNYKhhI0569GVtfdQOxICJPXzswk6n5dKyswH\\\\\\\/redaY5VhXLKrzdZNxXVkJkgb6iyns6eCH4Ea3xb4nkJ2IXpAzBbGOQiIvPyKazt4bDhsYL3jGzuTiI6RRBwt57RYyB8z2AKfFlHDaHIyfGlc16PKH4wDPGBthVG8UEsMQRPrrTgLAqa1hGPJbl\\\\\\\/fMe8ITuKmZO17V7OkbcGhkKbCU\\\\\\\/hc2VXMoyJ5\\\\\\\/e9rNejhNzmLROEvBRWtgh6ayjLwj8+jU4A35Nonyyu97hejVQKTatHvqEV3Ul2U3QgFknRwQQps8CupihUNSGO00tah8b4An4HPbjSsavnAMLNhcxF8q6wYaNi\\\\\\\/Fm\\\\\\\/kdRjJ+TVwZAIPIcEfsjpEPjH1PDXxlCNPGa9Yo4iGSeeiReLvTc\\\\\\\/BGA1be5etqvSdZP9keYJBDRJ9+jr6fYrN7BkDffJYyETCgj7lNTZ7XbSkJcHrtmNFXE23r4Cux8uk9UmU0ug2d1XfVVHubscpKzxKxqNva+p9fNLEm32xfbCoGMSnin597j4YaFNUu+N9GtxCdz4D1+stIsv7K2kC9wG8zKMwX8IIlvaQ0OlYYWrOOUQkEuoWnfYcGmWRRXypcLCCQgF7pf2arsKoXJvmyYd72lV8xbAyKxyhGf8DxddWmw22i4oVAdL9tqc77v3q9mN8a8jtxf4waMpZvagR1OHHcA2Fgb8TSAxzVoXRLw4AXJPG1I4vMoB4ySa6d9OhnZHoRn8JPwDTplQ+WaU1JmZXTrL2l4\\\\\\\/SwteD3\\\\\\\/DatfK4BuYZWX0G3Wu1C3oNjPmISJs+cRRUbofTy70DQmnkcSOD7a24pkwgjSQkhSDhQ\\\\\\\/II6myOVxSdBjtow+z4X\\\\\\\/rRbIYzXOe+3Jb1e3LZJkRfSbIx7ODdBLaQAY+lSSeOxojxnSRNz8S8KF1WyaW89zX7x+4bLktcxUkV4e54VFRERlDwAyesewRvLs31QBm0LVgfXvN7H7wEOvGDl0kD9qoqkNDL7ZzEnfg9+pSbYZUVlnn3h6wJ9EsIUubpWsHiUx+N+VPwwDguKXRlK9wndV4xkOn2FhkTzVCGs5Dy9SNG0tSvR80PLxrfA9O8b4J\\\\\\\/3pfgnC44w1mVWI+735D\\\\\\\/8XbMJu6Zag8VJGzSQRViDZ1YpxhqpotzATOZ+Vk+esJ+0S\\\\\\\/pNV6G45kSJ+\\\\\\\/\\\\\\\/2tbK\\\\\\\/UbGzTsYHL\\\\\\\/u4qorCWnhaB10cfV31YnJd+X+Iez1VWCvV7Vburf7zAz66f9V8cHKt6FQvEwy0yUxJC4lDIw4pM3hjOf4SAV50LFXAB4X0iNt1UMUQ43234pL5Daai0LY\\\\\\\/dYQop8Y5ygdwUKq7+4PWAzIV580lOiGyvQRB6ES21cx2KhUcBx58r95sfTiQLR2RMA66QY8Plknmb6XiNk+fhupsIuai9zLAJl\\\\\\\/BoFLd4KtqZdpffOrJVVtK35XdMoonNLxzpSJITRx2v\\\\\\\/0TeQ6Q5W+grYLds35quJzrAjP5Som63xb0TiGiK+f0NmS8zjwpe8fQTPvh4RG7zv\\\\\\\/cmgG4viwv2\\\\\\\/eb0rLmwW6enQ9Oqfrf\\\\\\\/lO+6ORMCyCuk4xLaVaFFSJkpDTXMxPlTF4U0E45ELwQsCmAW6qUJ0r1QNNGSLNR95FubrqDH5aRe4GYySrPjK9Xzk7ztdWWwOsdvhm67VCFStd4zJAPHIhVxUoTFPLnuanbzmzuJ1WGR+q8aK6o1QIHt7A7dOB4WEy3I2nDiuS4vYHtSqkjpVc8lW+SCFI0tzgkvJDwqONEmOd0y9rrqg2f21ZWgSvpls5XfB86SSuFsZ3osYMWqMBiNdahO0vRVXIbafVpeX3e5UYULGSLE+yzbXB60Ychsv1p09PZAJK0e69Dsofpp3cZoRqvB9D4scAwnQ5KFRNQsCpZxGsgQ+bUg7cTB9Z91VyMrdgrPEC68PUgNonUtW7RLbiE\\\\\\\/N1BFHVRS3cqfJkjhQ7oFbU2MrvUJBAaAvjDV8LUtN63usBxHcZPQPE5\\\\\\\/QD1s94qo34JSlDEiP6vd+MX0pYP3PSf77WLMsyp56ZCY35UAf0+e3HjcJlsywWjF1HVW760e6zxT9yIarnszJHsCi26DrDmsL4YmEzXjwsPN0s0\\\\\\\/pp8bwaWYLB4r5ptuObtpk2lHm5pjCUy34FnPjTJ2T1nIGehV9OoKtYNVYvtupVfHAxPSg9azXyEsDmUKetQX3OjxyfrSC1WWNnbN9m5AwGM9mk05GRTKkCE\\\\\\\/N2M333AEXmjFfO4fSdGKxLzqjLqydqw8YMq1uW1i0B7O0\\\\\\\/wd+LeZaOrt6ucx+pCXUIEa\\\\\\\/hiSh776VmbbAw2\\\\\\\/Z9W5qe6l4aNDrUbW5BXgn27zRpbf6YNfo4CLnl5Qq1+DXl71Cglpm5IaZywhchARq6qkrv2Z0dEAfo9DMWAh7aSFsfgew\\\\\\\/xg+Bm1uRrFY\\\\\\\/bumZNavSqTab\\\\\\\/R+UJxCra9hiPUGurhmaQW+QvV7DRbAesKfYv6nlp8DZMOU7gglRgIhECG\\\\\\\/p5l1sQeLK0sjdRD8FE810QS9nJgp+KkKQrRRQZ16C7j3LL\\\\\\\/JMVFuUUlU8Vs9Pt0aNRG9qymUZYkrl0Vl5M0IGQIT8cT2ukCjerj+xWV2FH+8fC5HT1c4jhEfO6\\\\\\\/RvhhCsOauHQHlY8ItbZ3C0hKCR5UNlaR5AXCP1fRqs3xCFMfeLt+P4R1depbaPzyrD81w3OfTdv83l\\\\\\\/ZqeVgqM\\\\\\\/L8oQ\\\\\\\/gpj4KLvOS49no9xzAsLqKBSHvqsuoAgIAHJFNF8y\\\\\\\/gdyFGN5Ng6l0uqZpPReIvqenKI9Zn+REPHdF2o1fEvGkV25v97xjisgjQLFQK9nPvqC6HkFmOUIeS4wIhFKy2IQEiyC3JePbP4Ou8nRlOxWEeY1GRkw\\\\\\\/3Bic7FoUeV\\\\\\\/Jftoh8lraQdoEldTICl4Qdf4OoEj43ozJgfoYrgwBCuaLvxf1JquTdnqGevHhY8cKjriwjgkZS29vKx261RvbKdCG5uN++2z58NGoIIIy8tN8QT85a2V1ujQgnfeGaGuZEOVjAf+s8HzzGy0ksmdyN7XAZ2hSWSvmSAtyQGyp9WuF5TGSP0gAaO0VxRQxRygpCBkDH2MjoLBwfCS0BivWHuXC+Ei19VvH8Sav9Vyog6T3Htes9JvZzuAgY4OPIHPUycPyExqiAY3pjcp5hye9eyoUL4kJDadvB\\\\\\\/iOixsSjCbU2edq0G2tc3h1aC8x9bVw4HYhHrg809bCLk8+O3IREVOvJzhnu0VYFK3Qs2GFC7Q1wpXh\\\\\\\/RsGNacNlUsOOMBGkubTwOUxx5add4FDxIbsIlnS09xK0\\\\\\\/PDBl3DFQyE6hEexOFUV6u40edK3OkVdf1eSFXGKMG44h\\\\\\\/OlhhDT88kfm5lkRXLVBsCIFx\\\\\\\/Z6HqBR1wktX17Aq51BX3BeuaCKQUYDG1TUiwwlYQ2HHBLxyosWqvMco3K+I4DqWlk3yAcW8TPRYCeCQtfLqLVrlEimK\\\\\\\/QFDM2ECrk3O7Ve6pZmIYi9cwMM+Ht8dp6xoLZjqYV3aodYUbZyZRaM2rH7CPtCK2AOrV4S\\\\\\\/jsz+vJr4N+gPaLghfBSX6MDeGALxZ9fE3Agc+23q5bh3197\\\\\\\/ZESP6vBSA585RIvRoJCzud9OmdLqJsQlu1peAPlJ2rnFi2UXp5dKev+yjSVnDNb\\\\\\\/l7StG8dWHG8X9SMJ8wvtALg7XJ2kf+wDQ4clXC9Jk6lpVQY2+1FHmxjoLjy4E440hXmx4CbHokiaWrQZ+Cy2f6uhYDCWPIILnkjEb5GQfbKXOMeB7vJE3fzXUUP3cr+oC0pt21H8pLIOqJqUkIIJjt7SSQpznYZlXbv3zlFjmIFfBc5srO1Z1PpD2gvCNR0+7IhI+ptHiS0AfoADm+AaTqleIdLmSO5XvOkkuBuUSvY3OyPz+\\\\\\\/BAakoLVfmoSS47QBCn7rE4qsuetD6n\\\\\\\/pY7pn6sMyJOnJeie8YjrxbX06ekGtkJ8UWHKMJPoU4EGYX++B91WUO69TSvFKxYZPuS+bmvG6KtuDhXbaakBO9bDJcmGHwmy9X0OzuZuuEA\\\\\\\/7oLh1CkcNa5o7Gm+nZSuSe\\\\\\\/FYrVnng4+E24iE9LaLlnWB4PJdR2AddW1ELvarV1fmV5DTsLK2C4knOgaVNxLDVbqeKa6aIUwivMX\\\\\\\/xWf6Ct9QImPx4\\\\\\\/uhZm8P45HryDKngJIR09a\\\\\\\/6M4IQSqmB+cFHLcjbvH7ylhfQZH9fkFpeoHtTyX72x7vsYdBEsLVEoCuRCE4oR1LyTuU5wrCw6tw6ymBGxwYeLdD6Cnrp0QEO7Suv2JZ5mplE1IdyHa7NtnEvpbZlHcwENv9zoDDpeiMjykjYOp69sgjn3LTAvW4TLR9fpLQu91\\\\\\\/6uigI0rGkwQW+rU2+VTXczn80BkfDrOStQImlKMT\\\\\\\/RRQtRqyKbrNCuSldN3PYxXb6amgK+vZ\\\\\\\/r7+eZ4UG9lRzGCxmYmSnogVj2YRnTgRSfSF6trp0JlHoBZkzsq\\\\\\\/YzM+RbKNGc8ZIOqpIWSKEteT+o9yFQamY8ksufVcGKlliATNClx84V3jR2fJc6LH4hnelcM3vzU5l5ChM3g58LpTMe71u17uvq01B\\\\\\\/t8RgJ0xYLsHhZwMb+PuTKWfjyjxBy3g1m4l6NX3e6KTL2s8ULFDEOoOn95xkckFKPXqDw3hc5gPg1mKUx4+bnZJm07\\\\\\\/MOA4HkiLby3X5fGlZQKXk09ypbiTXAMqIc5jQrVzFCuJRDUfnESUBg9inJ4e0DmcdA23gEakwbj8oabuetEV+trmFrtz+alxTWfAw5q2rJ92MM0srUH+sU0nMAeZZw73khPk1oVAP5+cgPo4aFnDPBdJxisdCwGJntd8HT+D63Crk3EniKOneCx+HSnTioo2aPWpSKgxPOv2\\\\\\\/bKL7dg980j4RL5DJUrPYnCOqKDbo+FEObQ8lPfF8h3cj\\\\\\\/HGl9TwRVIPntarny+wKH2Q5v2F\\\\\\\/al5vZV68ZuUuIoOhxeFC9omkkEZgX5hDe6xX2OKxXOW6h19pf\\\\\\\/WKM\\\\\\\/hmDjquxI4\\\\\\\/uZkFb40LkDud5+XahJXumyf9YwdiA+vgkkgtbDNN8LI1MuSlIAGq4uMUR+JFmVJeF+5zFHPHuvrJfJAS2OLmnE7+0dVhJROHEdAYDzFatUIHhUIq4HMnRzuwzzGKOsAxLtETWPMYLg1cotEeThzyvgKg04bbFm3lzr89CybaZdGzjqsBeL6t6qWiTGfIrX6xyvmu79bs5DCCT8877VKmQIYtn8HjaZUnqr31DUYsUDf2ReosiPKOAixVcYPPFJUtbrWY+YNFLYOu8BxctmfLvUNF39opGYI83SJjCyGo8Fz5+ZQNVjTDIXIa\\\\\\\/wKEFz353KWofrXaLd+HtMj0wIDYmp0LfHjqBcgXAERP+JrrrpisYGluStw4ghtJmPSWnBPAhKbg5H5Nglsd+MPJoDLncQx8fmZqugYzqUKwydb77e66pMYV6+lsz0c60CCQgXDqjv5SDOcNYbphmUFEmLDoJ29s09UB+ZzRwECg\\\\\\\/d63du66y1r2y7+tdnwuySkGg+dIYhCo+xPdpz+7aphmYPsWIG47QCDHWubYT1GQimPZvDMs9I6nVkepc4l0QBmPaZ5235G1mbdeW\\\\\\\/ab4OcBH1GdSQCoRkRR1IyoEcGHOEdA8P5pJXFs6L7zPjppvU3j9vBla8wAYcNMOHnsyek4DMb0lejCg7e9+LIBQbpI3+M7cceQ5GzoVTjgbUQ4LOhAoNU\\\\\\\/T3e6OPFvnLXQWpOCoy9AH0XIG0hiiZRF3d57fAg\\\\\\\/VdwgmBxjYpetdqqEbd6vI7y6RUpVtW99PJXIGKtzn+aIxpA8TwC449jnBbAs2kqeLnGdlAD8L09Eju7kF\\\\\\\/RKQkGimQQ1\\\\\\\/bJOoSDgdbwj\\\\\\\/staStdOVI0OcvYx2eyuq9q7I6MgGORKUfeB9nSwTc8WmQv9d0DfTNQ0by8ymABpw1EF4PyVvZqbGmg1Dp3SgiB4kBQAcf3gHe3b9s7FOKHEyXPNS4iUQ3S\\\\\\\/bWZm4UdCg05NxsxvkFNQvzeYXtNQTrDwpy0osu1RyfdkdTz3lbXDkCa8XkqW9oVfvBhGIUgSWtlvw3Osz018sZ3XyHYjHh2RqWUINu5wbX4S0OTo+aAipxPqmY7K2iRh274Axjb1UX2bQ0Cihj8heDPJlAddpor1di1W8B9srQNcxJ6\\\\\\\/mYySZRsSC\\\\\\\/gzMlB3UZB6NNhWDyO5I7yS9c3OSA86WQ39VA89DQ6l6dLa\\\\\\\/bTj3HrjX\\\\\\\/ss4RgAnpcGnMeCniV2HgLr+Akmw9RrYuqFc0gHJysXHlmfAYD9s22WB+b1Ipl0ppCV33pdmlm9lem9EUUFJ6afFTR3fuw3gAdpWjSOF23gvPmUTohbhWhbdwQkqhD0edFi+os\\\\\\\/tDTApkoGd8y\\\\\\\/w5oG\\\\\\\/a5TcSY2mpBM4noXDJDDe6rMoMGAXFTuVpBwP5IrxbsYXK0W\\\\\\\/FYC4NG\\\\\\\/\\\\\\\/iFRiap+NoGUeH6NvfOB0LJCCG8J31wpQdVvpvKX1yJaL0h2gfed33SCU+9fQKwHCycGSxZSEK1qEZwmyH4icKRV5kYixBh2UdJW2UF\\\\\\\/T+at89Af5UEdK6QCd7nWGClUBZpP6BCpo2hU1FJLrjD+z9kcbdymxvsSrx6P0Lz+FlQ5quQW5zN2M+6AeK3T8rhXb2OucmRmsecEcAGegb3xoauCjr+tnzLXO60fENbYJeBCFOUNVo0fCi5NgybuiqSrFgEktNr86tjJFny8lyDYSyFIyEbOw+rhHNfG9a4WSU5F43oZIiWdsC4a8H3OHYHFiMnMMwF3OO9AWg\\\\\\\/7yzdAlqxhWAkAbi0Ei0nN\\\\\\\/Bj5IuSfZ\\\\\\\/u4laAqOSDv+ZFeZxszn8liEcNmgm2IutvgAIfTI4YrNOxk+dMqg+1\\\\\\\/oNr0vDtPDzJ+e7ITwwXqZQ3Pqem1JSmK\\\\\\\/gYiAggk8iavw3vIbRgI4+bZsW5qgOVtl9mykQ0mLtfi57hv+uvFl22u1DoRhgm+bIPHq\\\\\\\/MfwdKd8YDz11KkebQX5iu3TabKYgwkiXEsZSfKaI6VjNhpH0mZj85AzHEBTWf8Ja27yn1UtBgm7xb3aYAGjw3C4UqbhJjJ0Wb4THYRUhrtEvBjyLQiTdpm5K4icXOsqP767dWc0m8qftf+OILSJnQYI4ejoVKO4Qeu9w4ja4SJi6RcyX0DtnSBoskMPt5UP8YP3RZtHMGJP7BFEk+kLBRHzWfzL8t\\\\\\\/2dTVgRxq8K9bdvMG8gfTVrfVTpoGHBG4t+QZR1eXUjRhnuXDNbMNF7UacDfSxMRulq7IV73ykBczbdLwSZ2IVFud9j\\\\\\\/Ktkfc7I7o82gVEfN4lPg1tqcjeZ5N1uewHzg6XFbciMqns2TrlgGDkiROTO06uLi+faXHAvzWdKQM1eXv1k69RY63pD1vVRSqVfjZ0R6xJor9wcATf9GhegqGaW66X+FIon0PnPCY1c+\\\\\\\/fA6jSOZyYGrMkyM2NXw8PM96d1vZ+zfgMKwFrNphMkWVl3eMGaF\\\\\\\/kTWNqAGCSAEea3TlpiMZlQ4FO2UHFQLSmO3fkVvAN\\\\\\\/9U+IFeBgxS8pBu1xdBiSRvzdGL5hXgoVvnE0AyjLKL0KX0AZhdsJiN\\\\\\\/K0ML50\\\\\\\/PPxWALCJ+fXM1zsExwdqG5S09crJ\\\\\\\/6U046cSgW\\\\\\\/aHV1Uhlv77eQUsqwNQxBSGBCxS1bR\\\\\\\/ju8DaoPvjr2Nx4v6IGVu81CQMLil9OlZDB\\\\\\\/TnFr58nqOfooaI23r1BgfBTZcZxguLvNXkVtTjCWeTMFgStu+ObL0uNH5NKUqrp2dXDMv9ZNkqc4w90vVbKlTrFGpfqra8VCtOeBFiHIBwhyq0cnNqMPHd5jZkdzr3m7G\\\\\\\/3zNe2aEjqxDH\\\\\\\/wQIGVoZ0gCdGpQ0qAkwlGZOE+x7OxS8nBPZUBq+EYqKb4eGkaLHg1nOjAz3S98DVdQGJmHRExLThQ0GUgp5EO2dsUPHMED8AFT\\\\\\\/3QL6Hqy5uV\\\\\\\/\\\\\\\/dc9LaLC8opVecdo1k\\\\\\\/2MFps66te58OpXhU5KC0m1ip4N5817llvYd6axorBI3sBvDTg1OWXENXGuyoagwrEzApxMk7BaXcb2C4ykZb0guTUSejA2EAsTm8fDGTVMjKPD1eE2Zx+yKTzWwrcDze9p72g1zkFRjGzz2U0KfSIWqvmZo9j2Mg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022m90UTVdrFr0ssqv3BaPsPA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022a76616efd3a5e2891d928ec8469d90b0ade652eef118c18fa8898d87c49a5953\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00226XTg8VVeF31WbqNC+kgjwyAJxiK7+aOQ8pTCh596raewruIQ2KCyNZ\\\\\\\/kBjHDSSijHv9LqVkF9BrLQY81Y8T7VPoXhVKP57VZks9+QaGTjRj3F4JMnsWK6VSe5sWZi0z58rPlmLOpeKRGlbQUylqaKwdeCJLl8rrWppRmyYm1kO\\\\\\\/xpXzg9eF2bAYAL1U9OeHtfWYxioPhNLhOlPPyXycHEAq+KBGQYJ8pvoWaZxDmKnXvnIAU\\\\\\\/+EjMnoaKwMRRMQH27l4tgP8tgXqlU5d81pbbCbNADxN2YvHV5ODNYU1T\\\\\\\/Z5lpjGtFhDOs2XejqMGInSUQGlU38XDDC4zEA6\\\\\\\/eCef11bIi8Y1C2lRvWnX09J4iGerKEtN8PrW9iovbz\\\\\\\/XF7sZOdb25f9r0sJGF2\\\\\\\/45pGn3pv1Yz+vWXixZVDPuLHrwyci1rlgfpi1esJzyutQQ+QTnPNbUq+I4G\\\\\\\/SNbRh2+kxVMM+nCo9SYxZCWf7EaZlY3P+1JZPGJjUvmGlPrfj64OHF+MWIFrLBjkE+r52vgM3ygrn68cb9LuCyR0EjQQpo9CtWwsbvCEq1gTNiG4toG3vGZszioFwLRy\\\\\\\/J\\\\\\\/QFz4JGqdJuLHmJVT9gC7uE3\\\\\\\/NjbhF5fthZu9WoXPg\\\\\\\/mLS+PUXQGeTFsL5++u1vooUPGL534QExKdLK4KrW4EQZp4Ho0Bf35wW7j81mm9xXxd8o1IyLeND2g9QvVH3pHEPgKgqR0qWpkk7+Ycd8RyOBRi5Uhp0BK4j\\\\\\\/v8LDidsQHYfzVhGUOiNBG0\\\\\\\/EdKGTfT3GQ5seoNSxjsgNxV2aT0RYTjPg2ADLjJL0AquufOs2XCGewLpqUJyWHrJCEg+sWrAqydX5pA2pTfJXjO+wo5hJncsVQ3W2P2gF7mcPFOqXnWUBz5c283sXSWJ+BwT1p3k3cgDKUjwRwbyODuJwv7F8IGAGUWQ81Mh1cC8yZAMzAbpbhDLAA\\\\\\\/KbOGqUmZw2WADagFHbKgwC6IaOA5luoTa6Y8kFb6Z85YeqRQKrjFlF7Z1jTa71iROzo\\\\\\\/LrbhgIEDQTTxDbuzt9rsn0n0m+Uir\\\\\\\/F7CkAe1TE9V1wpcgfjZ5R+zTI0eRYUOqebMV2CHPPXF4WiYlITiRs2FHISv80iiWaLBFtgaqTki716lBE61K9xZ1Jk8WoFNQxHyT1u4qGriovuElNfkHTNRTrxjTKDHDmwlOg4LlZgB3aoMzHi0X1iS134qbAdrSDvSITdFE\\\\\\\/\\\\\\\/jY68jR4uytHK5Plq4WwpgKqo31WxvJe\\\\\\\/Bu7QRwuPfOBu4Nl+it0kNnKNltiZeWDCH1LpUUuS+tbe1y6MwDCg6ov8af3xMAECYb8Ni2aJLZGmjjqty35ho91vUAwSejV0cKhzARcsXkn9RoDAN91TW\\\\\\\/DEhoP73kd6TgostgyJ7JCmLNQE1sF8rybwYrkJm3DXxxg1jGACaC3+wjX8hvkwlb0u4tRSV026BRdbQSabf8FQqMNmncRZdUz\\\\\\\/fzE4aeTUMePgPl3lDtmRvpSxfNL89czZmX8CtWVR6UXCcnI1ear4qzJ420CGg6oOs9vwQUSk7aUbM7XWkmdRctD5tD4mb6ajYQpBA+svSiCCzBloIu+WRifB+cLsodq8LhGjTl53uLgPEXVlBrWPriSRzCb0wtbxbex9sj07dO77BS+2Bii8IJnqJ\\\\\\\/PYvCUoMeJWZvTKN\\\\\\\/AZrQse7LcHzZI9Uk4iITy00AKVJSZ64rMh2CquNx7WZH\\\\\\\/KS6\\\\\\\/bWTT6SPe3zFQ2VLzqFBUkLPtDNyDNyQUYO1W9\\\\\\\/7JDR6UwHWGIA5w+W09HYgmSQRcDAPP4zBOvczLyQQBEq+333WPkNWYOqsULs02hj+2MMwlF6wNbjmgjCK+pS8CiaNxTjKTqhBWyhvaVCvFJqYL15kiSkMZid9rn0I4o+QHHGTc0yFP\\\\\\\/IKJbeOLW3K9T5KywA8JcOPPwuZlIYxx5xPy0gYN2KQpzI22DqOdXbDcOLWKAU9PdREfe\\\\\\\/IQ9pnEFiSFvSnDGElR+BNF\\\\\\\/d0gWOnw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022EutagzOP3OW5Q0cCacrsQQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002255fff0fb97ed5a40e0ac10aa4ee6eee703f0f378f97a1dfd0e33f22257121abb\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-459943 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-mouse-genetic-resources">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/8/1429" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-d2R4F1D4kmajiIHO87Br7Q_dPoylUKnJZCUD3NbTuzE" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/8/1429" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-pH5qzm6R77Tmv6vri-OjLagmroajCdz8k0W_lEpnIMM" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/8/1417" title="Genetic Interactions Between P Elements Involved in piRNA-Mediated Repression of Hybrid Dysgenesis in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/8/1445" title="BZcon1, a SANT/Myb-Type Gene Involved in the Conidiation of Cochliobolus carbonum" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459943" id="node459943" data-pisa="ggg;4/8/1429" data-pisa-master="ggg;g3.114.011759" data-apath="/ggg/4/8/1429.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Profiling of Collaborative Cross Founder Mice Infected with Respiratory Viruses Reveals Novel Transcripts and Infection-Related Strain-Specific Gene and Isoform Expression</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hao Xiong</span>, <span class="highwire-citation-author" data-delta="1">Juliet Morrison</span>, <span class="highwire-citation-author" data-delta="2">Martin T. Ferris</span>, <span class="highwire-citation-author" data-delta="3">Lisa E. Gralinski</span>, <span class="highwire-citation-author" data-delta="4">Alan C. Whitmore</span>, <span class="highwire-citation-author" data-delta="5">Richard Green</span>, <span class="highwire-citation-author" data-delta="6">Matthew J. Thomas</span>, <span class="highwire-citation-author" data-delta="7">Jennifer Tisoncik-Go</span>, <span class="highwire-citation-author" data-delta="8">Gary P. Schroth</span>, <span class="highwire-citation-author" data-delta="9">Fernando Pardo-Manuel de Villena</span>, <span class="highwire-citation-author" data-delta="10">Ralph S. Baric</span>, <span class="highwire-citation-author" data-delta="11">Mark T. Heise</span>, <span class="highwire-citation-author" data-delta="12">Xinxia Peng</span> and <span class="highwire-citation-author" data-delta="13">Michael G. Katze</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1429-1444;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.011759 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459943" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hao Xiong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, School of Medicine, University of Washington, Seattle, Washington</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hao%2BXiong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xiong%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHao%2BXiong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Juliet Morrison </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, School of Medicine, University of Washington, Seattle, Washington</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juliet%2BMorrison%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morrison%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuliet%2BMorrison%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Martin T. Ferris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Genetics, University of North Carolina-Chapel Hill, Chapel Hill, North Carolina</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BT.%2BFerris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferris%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BT.%2BFerris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Lisa E. Gralinski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Epidemiology, University of North Carolina-Chapel Hill, Chapel Hill, North Carolina</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lisa%2BE.%2BGralinski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gralinski%20LE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALisa%2BE.%2BGralinski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Alan C. Whitmore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Genetics, University of North Carolina-Chapel Hill, Chapel Hill, North Carolina</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BC.%2BWhitmore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Whitmore%20AC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlan%2BC.%2BWhitmore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Richard Green </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, School of Medicine, University of Washington, Seattle, Washington</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BGreen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Green%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARichard%2BGreen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Matthew J. Thomas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, School of Medicine, University of Washington, Seattle, Washington</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BJ.%2BThomas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thomas%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BJ.%2BThomas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jennifer Tisoncik-Go </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, School of Medicine, University of Washington, Seattle, Washington</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BTisoncik-Go%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tisoncik-Go%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BTisoncik-Go%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Gary P. Schroth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Illumina, Inc., San Diego, California</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BP.%2BSchroth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schroth%20GP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGary%2BP.%2BSchroth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Fernando Pardo-Manuel de Villena </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Genetics, University of North Carolina-Chapel Hill, Chapel Hill, North Carolina</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BPardo-Manuel%2Bde%2BVillena%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pardo-Manuel%20de%20Villena%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFernando%2BPardo-Manuel%2Bde%2BVillena%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Ralph S. Baric </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Epidemiology, University of North Carolina-Chapel Hill, Chapel Hill, North Carolina</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ralph%2BS.%2BBaric%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baric%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARalph%2BS.%2BBaric%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Mark T. Heise </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Genetics, University of North Carolina-Chapel Hill, Chapel Hill, North Carolina</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BT.%2BHeise%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Heise%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BT.%2BHeise%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Xinxia Peng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, School of Medicine, University of Washington, Seattle, Washington</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xinxia%2BPeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Peng%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXinxia%2BPeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Michael G. Katze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, School of Medicine, University of Washington, Seattle, Washington</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BG.%2BKatze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Katze%20MG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BG.%2BKatze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:honey@uw.edu" class="" data-icon-position="" data-hide-link-title="0">honey@uw.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/4/8/1429" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:459943" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:459943/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/8/1429.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:459943" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:459943/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/8/1429.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:459943" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:459943/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/4/8/1429.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:459943" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:459943/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/4/8/1429.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:459943" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:459943/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-4">Genetic variation between diverse mouse species is well-characterized, yet existing knowledge of the mouse transcriptome comes largely from one mouse strain (C57BL/6J). As such, it is unlikely to reflect the transcriptional complexity of the mouse species. Gene transcription is dynamic and condition-specific; therefore, to better understand the mouse transcriptional response to respiratory virus infection, we infected the eight founder strains of the Collaborative Cross with either influenza A virus or severe acute respiratory syndrome coronavirus and sequenced lung RNA samples at 2 and 4 days after infection. We found numerous instances of transcripts that were not present in the C57BL/6J reference annotation, indicating that a nontrivial proportion of the mouse genome is transcribed but poorly annotated. Of these novel transcripts, 2150 could be aligned to human or rat genomes, but not to existing mouse genomes, suggesting functionally conserved sequences not yet recorded in mouse genomes. We also found that respiratory virus infection induced differential expression of 4287 splicing junctions, resulting in strain-specific isoform expression. Of these, 59 were influenced by strain-specific mutations within 2 base pairs of key intron–exon boundaries, suggesting <em>cis</em>-regulated expression. Our results reveal the complexity of the transcriptional response to viral infection, previously undocumented genomic elements, and extensive diversity in the response across mouse strains. These findings identify hitherto unexplored transcriptional patterns and undocumented transcripts in genetically diverse mice. Host genetic variation drives the complexity and diversity of the host response by eliciting starkly different transcriptional profiles in response to a viral infection.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/rna-seq" class="hw-term hw-article-keyword hw-article-keyword-rna-seq" rel="nofollow">RNA-seq</a></li><li class="kwd"><a href="/keyword/mouse-transcriptome-annotation" class="hw-term hw-article-keyword hw-article-keyword-mouse-transcriptome-annotation" rel="nofollow">mouse transcriptome annotation</a></li><li class="kwd"><a href="/keyword/isoform-differential-expression" class="hw-term hw-article-keyword hw-article-keyword-isoform-differential-expression" rel="nofollow">isoform differential expression</a></li><li class="kwd"><a href="/keyword/collaborative-cross" class="hw-term hw-article-keyword hw-article-keyword-collaborative-cross" rel="nofollow">collaborative cross</a></li><li class="kwd"><a href="/keyword/viral-infection" class="hw-term hw-article-keyword hw-article-keyword-viral-infection" rel="nofollow">viral infection</a></li></ul><p id="p-5">Genetic diversity is an important consideration when assessing the clinical applicability of data gained from mouse studies (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Churchill 2014</a>). This has led to the development of genetic diversity resources such as the Collaborative Cross (CC), a group of genetically diverse recombinant inbred (RI) mice descending from eight founder strains: five laboratory strains (C57BL/6J, A/J, NOD/ShiLtJ, NZO/HILtJ, and 129S1/SvImJ) and three wild-derived strains (CAST/EiJ, PWK/PhJ, and WSB/EiJ). The complex phenotypes that are exhibited by CC mice are rooted in their genetic variation (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Ferris <em>et al.</em> 2013</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Aylor <em>et al.</em> 2011</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Durrant <em>et al.</em> 2011</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Kelada <em>et al.</em> 2012</a>) and, as such, these mice are valuable tools for genetic mapping and systems research. While genetic variations between commonly used mouse strains have been characterized to a large extent (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Collaborative Cross Consortium 2012</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Keane <em>et al.</em> 2011</a>), the transcriptional landscape of genetically diverse mice has received little attention, with the Mouse ENCODE project focusing exclusively on the <em>Mus musculus domesticus</em> C57BL/6J strain (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Mouse ENCODE Consortium <em>et al.</em> 2012</a>). Although some mouse strains are closely related, such as the classical laboratory inbred strains C57BL/6J and A/J, wild-derived strains such as PWK/PhJ and CAST/EiJ are more genetically variable and, in fact, represent the <em>M. musculus castaneus</em> and <em>M. musculus musculus</em> subspecies, respectively (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Collaborative Cross Consortium 2012</a>). Wild-derived inbred strains can harbor tens of millions of SNPs, hundreds of thousands of small indels, and tens of thousands of structural variations compared with the reference mouse genome (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Keane <em>et al.</em> 2011</a>). This means that the genetic variation that exists among a few mouse strains can be as large as the genetic variation that exists among the 1092 humans sequenced by the 1000 Genomes Project (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">McVean <em>et al.</em> 2012</a>). In fact, there are more large structural differences between the CAST/EiJ strain and the mouse reference annotation than among all of the humans in the 1000 Genomes Project—25,000 <em>vs.</em> 14,000. Given these enormous genetic differences, and the fact that the mouse reference annotation is based on the C57BL/6J strain, it is likely that there are numerous new transcriptional elements with unknown functions that have yet to be recorded in the reference annotation.</p><p id="p-6">Our understanding of transcription has been steadily augmented since the advent of RNA-seq (<a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Mouse ENCODE Consortium <em>et al.</em> 2012</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">ENCODE Project Consortium <em>et al.</em> 2012</a>). More than half of the human genome is transcribed, yet no single cell line exhibits more than 60% of the total human transcriptome (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Djebali <em>et al.</em> 2012</a>). Many transcripts have been found in intergenic regions, particularly the long intergenic noncoding RNAs (lincRNAs) (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Derrien <em>et al.</em> 2012</a>). This suggests both transcriptional complexity and transcriptional diversity. In addition, transcription is dynamic and condition-dependent and is impacted by genetic variations (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Montgomery <em>et al.</em> 2010</a>; <a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Pickrell <em>et al.</em> 2010</a>). Therefore, the large genetic diversity of laboratory mouse strains leads to more transcriptional diversity than is reflected in the reference annotation. For example, the interferon-inducible <em>Mx1</em> gene is one of the first host genes that was linked to influenza A virus (IAV) pathogenesis, and it limits disease by inhibiting viral replication (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Staeheli <em>et al.</em> 1988</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Tumpey <em>et al.</em> 2007</a>). <em>Mx1</em> is naturally polymorphic; the <em>Mx1</em> gene has three exons that are absent in C57BL/6J mice but present in CAST/EiJ mice (<a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Ferris <em>et al.</em> 2013</a>). As such, these exons are not recorded in the reference annotation.</p><p id="p-7">The absence of alternative transcripts in the reference annotation has implications for understanding disease pathogenesis. For example, transcriptional control mechanisms, such as alternative promoter and terminator usage, are prevalent in genes associated with neurological disease (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Pal <em>et al.</em> 2011</a>); and novel alternatively spliced transcripts have been linked to lung fibrosis development (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Deng <em>et al.</em> 2013</a>). Alternative transcription is also important in the host response to virus infection. Transcriptional start site (TSS) selections and posttranscriptional modification to host transcripts have been linked to dengue virus virulence (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Sessions <em>et al.</em> 2013</a>). Long noncoding RNAs have been linked to activation and repression of the inflammatory response to a TLR2 ligand (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Carpenter <em>et al.</em> 2013</a>; <a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Chang <em>et al.</em> 2011</a>) and to inflammatory signaling (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Rapicavoli <em>et al.</em> 2013</a>), and unique noncoding RNA signatures have been associated with lethality of severe acute respiratory syndrome coronavirus (SARS-CoV) (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Peng <em>et al.</em> 2010</a>). As genetically diverse mouse populations such as the CC mice become increasingly used for expression quantitative trait loci (eQTL) (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bottomly <em>et al.</em> 2012</a>) and transcriptomic studies (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Pommerenke <em>et al.</em> 2012</a>), the need for a well-annotated mouse transcriptome increases commensurately.</p><p id="p-8">The goal of this study was to characterize the transcriptional landscape of genetically diverse mice in response to respiratory viral infection and to identify new transcriptional elements that have yet to be recorded in the reference annotation. To this end, we sequenced day 2 and day 4 lung samples from the eight founder strains of the CC project that had been mock-infected or infected with mouse-adapted strains of two important respiratory pathogens, IAV or SARS-CoV. We built a custom bioinformatics pipeline to filter the results of <em>de novo</em> assembly by focusing on intergenic novel transcripts and aligning them with mouse expressed sequence tag (EST) and clone databases and with rat and human genomes. We found that many novel transcripts with potential roles in the host response to viral infection existed outside of the reference annotation. A smaller number of novel transcripts could not be aligned with the reference mouse genome or the pseudogenomes generated from the reference genome and genome sequencing of the founder mice, further demonstrating missing information in existing mouse genome sequence data. We found strain-specific differentially expressed (DE) isoforms between samples from virus-infected and mock-infected animals, some of which were likely <em>cis</em>-regulated by genetic variations near intron–exon boundaries. Strain-specificity was also observed in the functional enrichment of DE genes, with one subset of DE genes closely reflecting the underlying genetic variations and another subset suggesting differences in the immune response to viral infection among the different founder strains.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Animal infection and RNA extraction</h3><p id="p-9">Female mice of the eight CC founder strains [A/J (AJ), C57BL/6J (C57BL6J), 129S1/SvImJ (129S1), NOD/ShiLtJ (NOD), NZO/HILtJ (NZO), CAST/EiJ (CAST), PWK/PhJ (PWK), and WSB/EiJ (WSB)] were bred in-house at the University of North Carolina at Chapel Hill and were infected when they were between 8 and 16 weeks old. They were kept in a pathogen-free environment until infection. After being lightly anesthetized, the mice received 50 µL of PBS and were infected with 5 × 10<sup>2</sup> PFU of PR8 (<a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Schickli <em>et al.</em> 2001</a>) or 10<sup>5</sup> PFU of MA15. Mock-infected animals received 50 µL of PBS. The animals were monitored daily for their morbidity. At day 2 and day 4 after infection, three of the infected mice and two mock-infected animals were killed and their lungs were collected. All experiments were approved by the University of North Caroline Chapel Hill Institutional Animal Care and Use Committee.</p><p id="p-10">RNA was extracted from lung tissue homogenate with the miRNeasy mini kit with on-column DNase I treatment (Qiagen). RNA concentration was quantified on an ND-2000c UV-Vis spectrophotometer (Nanodrop, Wilmington, DE) and controlled for integrity and purity on a capillary electrophoresis system (Agilent 2100 Bioanalyzer; Agilent Technologies, Santa Clara, CA).</p></div><div id="sec-3" class="subsection"><h3>Strand-specific total RNA library preparation and sequencing</h3><p id="p-11">The total RNA-Seq library was constructed using the Illumina TruSeq Stranded Total RNA Sample Prep Kit with Ribo-Zero Gold (catalog #RS-122-2201), which targets RNA for cDNA synthesis. This kit uses Ribo-Zero to remove rRNA prior to library preparation. The libraries were quality-controlled for appropriate mass and insert size (also for evidence of DNA contamination) and quantitated using the BioAnalyzer 2100 system and qPCR (Kapa Biosystems, Woburn, MA). Sequencing was performed initially with the Illumina HiSeq 2000 system using HiSeq version 3 sequencing reagents. Some additional data were generated using the Genome Analyzer IIx system using GA version 5 sequencing reagents to supplement the number of reads up to a minimum level for all libraries. The libraries were clonally amplified on a cluster generation station using Illumina HiSeq version 3 and GA version 4 cluster generation reagents to achieve a target density of approximately 700,000/mm<sup>2</sup> in a single channel of a flow cell. Image analysis, base calling, and error estimation were performed using Illumina Analysis Pipeline (version 2.8). RTA analysis (RTA version 1.13 for the HiSeq and RTA version 1.9 for the GAIIx) for both the HiSeq and Genome Analyzer reproduces base-calling and quality thresholds because both machines use similar SBS chemistry. The raw reads have been uploaded with accession number GSE52405 (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Josset <em>et al.</em> 2014</a>).</p></div><div id="sec-4" class="subsection"><h3>Quality control and alignment</h3><p id="p-12">Read quality was checked with FASTQC Illumina Sequence Analysis Viewer (SAV) and reads were trimmed to remove primers to length 97 using in-house custom scripts. Additional quality control was performed at the individual-run level by comparing log-read counts between replicates and between mocks of the same strain. Evidence of PCR artifacts and DNA contamination were not visible during sequencing quality control and alignment. One sequencing run was deleted because of obvious discrepancy with other replicates. Viral infection was monitored by checking viral reads, and sample mice for which no viral reads could be detected were removed from further analysis.</p><p id="p-13">Prior to aligning to host genomes, short reads were aligned to ribosomal sequences using Bowtie1. Any read aligned to ribosomes was removed. In the next step, the remaining reads were mapped to viral genomes, PR8 and MA15, and any mapped read was similarly removed. The number of reads removed at this step was recorded as viral read counts. The leftover reads were then aligned to pseudogenomes downloaded from the University of North Carolina compgene web site, version 0.1, using STAR version 2.2.0c. Alignment was also performed on the reference genome (version mm9) as a comparison.</p></div><div id="sec-5" class="subsection"><h3>Transcript discovery</h3><p id="p-14">Novel transcript discovery had to take into account the existence of the reference annotation and reference genome and strain-adapted pseudogenomes. As a result, we first downloaded the reference annotation’s sequences from Ensembl’s ftp server using version GRCm38, including both coding transcripts and noncoding transcripts, totaling 92,179 transcripts. Transcripts not yet placed on chromosomes but on scaffolds were extracted and kept separately for later processing. Then, the sequence of the transcripts was mapped to the pseudogenomes developed by the University of North Carolina Systems Genetics group (<a href="http://csbio.unc.edu/CCstatus/index.py?run=Pseudo">http://csbio.unc.edu/CCstatus/index.py?run=Pseudo</a>), version 0.1, using GMAP version 2013-08-14 (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Wu and Watanabe 2005</a>). The command line used was “-B 5 –t 10 –O–split-output–format=gff3_gene,” in addition to the genomes' location. The genome indices were built using a companion program of GMAP (“gmap_build”) with default options.</p><p id="p-15">The mapping of RNA-seq short reads to different strains’ genomes resulted in two sets, one in which reads were mapped to genomes and another in which reads were not mapped. <em>De novo</em> transcript assembly using unmapped reads was performed with Trinity. Transcript assembly using mapped reads was performed using Cufflinks version 2.0.2 (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Trapnell <em>et al.</em> 2012</a>). The command line used was “–library-type fr-firststrand.” The Cufflinks’ output annotation file was then compared with adapted annotation to filter out all transcripts overlapping known transcripts or, in other words, to extract intergenic transcripts. The output from Trinity was first filtered by mapping to viral and ribosomal genomes and then mapped to genomes to obtain transcripts mapped to existing genomes and truly <em>de novo</em> transcripts unmappable to existing genomes; the mapping was performed using GMAP with the same command line options as above. Single-exon transcripts were oriented to positive strand or negative strand, depending on which strand resulted in higher read counts. The read counts were generated by htseq-count, part of HTSeq software affiliated with the DESeq package (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Anders and Huber 2010</a>). The command line option was default except “–stranded=reverse.” Afterward, additional filtering for intergenic transcripts was performed using cuffcompare with “reference” annotation set using “-r” option. The intergenic transcripts from the output of Cufflinks and Trinity were then merged using cuffmerge with default options. The <em>de novo</em> transcripts from Trinity’s output were compacted using the sequencing clustering program cd-hit (<a href="http://www.bioinformatics.org/cd-hit/">http://www.bioinformatics.org/cd-hit/</a>) version 4.5.4 and option “’-c 0.95 -n 9 -g 1 -M 0 -T 20.” Then, the sequences of novel transcripts were matched against all known human or mouse DNA or RNA sequences downloaded from NCBI. The program used was “blastn” version 2.2.28 downloaded and run locally. The command line options used were “-db nt -perc_identity 90 -num_threads 20 -outfmt “6 qseqid sseqid pident length qlen slen mismatch gapopen staxids salltitles””. The database of sequences was downloaded to a companion program “update_blastdb.pl” downloaded along with blastn.</p></div><div id="sec-6" class="subsection"><h3>Differential expression and functional analysis</h3><p id="p-16">The read counts were generated using htseq-count version 0.5.3p9. The command line options were as specified above. Differential expression analysis was performed using edgeR software version 3.2.4 in R version 3.0.1. All statistical tests were performed using GLM-related functions. Normalization was performed by function "calcNormFactors" in edgeR. Filtering of genes was performed based on whether a gene’s read counts had at least two samples with one count per million reads. The counts-per-million is a normalized read count returned by function “cpm” in the edgeR package. The design matrix was generated using “MOCK” as the baseline target. Variance estimation was by a sequence of steps, “estimateGLMCommonDisp,” “estimateGLMTrendedDisp,” “estimateGLMTagwiseDisp.” The statistical test was performed by “glmLRT” function and fitting of the negative binomial model was performed by “glmFit.” The cut-off criterion for statistical significance was false-discovery rate of 0.01 and log-fold change absolute value of at least 1.</p><p id="p-17">The primary functional analysis was performed with Ingenuity Pathway Analysis (IPA) from Ingenuity Systems Inc. (<a href="http://www.ingenuity.com">http://www.ingenuity.com</a>). The output of edgeR was fed to IPA and filtering of significant genes was performed within IPA to give a proper background of genes for enrichment analysis. GO term enrichment analysis was provided by “GOSim” and “topGO” from Biodconductor with version 1.4.0 and 2.14.0, respectively. The enrichment was computed based on a background of DE genes to reduce bias.</p></div></div><div class="section results" id="sec-7"><h2 class="">Results</h2><div id="sec-8" class="subsection"><h3>Genetic background determines viral load and disease severity in response to IAV and SARS-CoV infection</h3><p id="p-18">The eight founder strains of the CC were infected with either 5 × 10<sup>2</sup> PFU of mouse-adapted influenza A/Puerto Rico/8/34 virus (PR8) or 1 × 10<sup>5</sup> PFU of mouse-adapted SARS-CoV (MA15), and lung samples were collected at day 2 and day 4 after infection, with three replicates at each time point. Time-matched lung samples from mock-infected animals were also collected, resulting in a total of 124 lung samples. Total RNA was then sequenced using an Illumina HiSeq with a stranded library preparation protocol that generated 2 × 100 nt paired-ends reads. Each sample was ensured to have at least 30 million reads and, on average, there were 48 million reads per sample. We then verified the sequence quality and filtered out ribosomal RNA reads and viral reads. The viral read counts varied among the founder strains, suggesting that the genotypes of the animals had an effect on viral loads in the lung (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1/011759SI.pdf">Supporting Information</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1/FigureS1.pdf">Figure S1</a>). Some strains, such as 129S1/SvImJ and CAST/EiJ, had high levels of MA15 RNA on day 4 after infection whereas others, such as NZO/HILtJ and NOD/ShiLtJ, had low levels of MA15 RNA (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1/FigureS2.pdf">Figure S2</a>). Strain-dependent variations in viral RNA levels were also observed in PR8-infected mice, although the pattern of high <em>vs.</em> low viral RNA levels was not necessarily the same. For example, although NOD/ShiLtJ mice had lower levels of MA15 RNA than did the other mouse strains on day 4, the inverse was true for PR8 RNA levels in this strain (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1/FigureS2.pdf">Figure S2</a>). The pattern of viral read-counts for PR8 can be well-explained by <em>Mx1</em> alleles in the eight founders (<a id="xref-ref-17-3" class="xref-bibr" href="#ref-17">Ferris <em>et al.</em> 2013</a>).</p><p id="p-19">Weight loss is a common and quantifiable readout of disease severity in mouse models of IAV and SARS-CoV infection. To assess the effect of genetic background on the severity of IAV and SARS disease, we measured the daily body weights of the mice from time of infection to time of kill. The daily body weights of the mice are provided in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1/FigureS3.pdf">Figure S3</a> for MA15-infected mice and in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1/FigureS4.pdf">Figure S4</a> for PR8-infected mice. A wide variety of phenotypes emerged, ranging from 20% weight loss to essentially no weight loss. Strikingly, NZO/HlLtJ mice, alone among the eight strains, did not lose weight when infected with either PR8 or MA15.</p></div><div id="sec-9" class="subsection"><h3>Thousands of novel intergenic transcripts have potential roles in the host response to IAV and SARS-CoV</h3><p id="p-20">The mouse reference annotation is largely based on one founder strain, C57BL/6J, but large genetic variations separate all eight founders, particularly among the three wild-derived strains as well as between the classical laboratory strains and the wild-derived strains (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Keane <em>et al.</em> 2011</a>). Indiscriminately using the reference annotation in CC transcriptome studies could bias results and lead to inaccurate conclusions. In addition, the advent of RNA-seq has led to large discoveries of novel transcripts (<a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">ENCODE Project Consortium <em>et al.</em> 2012</a>). Therefore, we built a custom bioinformatics pipeline (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1/FigureS5.pdf">Figure S5</a>) for novel transcript discovery. We based our pipeline on the popular novel transcript discovery programs, Cufflinks (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Trapnell <em>et al.</em> 2012</a>) and Trinity (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Grabherr <em>et al.</em> 2011</a>), with the former for genome-based transcript discovery and the latter for <em>de novo</em> assembly of transcripts not based on genomes. The pipeline accounted for the genetic variations of the founders’ genomes compared with the reference genome, on which the reference annotation is built, by mapping transcripts in the reference annotation to pseudogenomes (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Huang <em>et al.</em> 2013</a>). The mapped transcripts were used in place of the reference annotation. Because we had total RNA data, we focused on novel intergenic transcripts and filtered novel transcripts by comparing them against known transcripts. Additional filtering steps were required for <em>de novo</em> assembled transcripts to exclude false positives, so we mapped them against viral and ribosomal sequences and compared them with all known mouse, human, and rat ESTs and clones using BLAST. We also required all novel transcripts to be at least 200 bp long.</p><p id="p-21">The numbers of new intergenic transcripts in all founder strains are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1/FigureS6.pdf">Figure S6</a>A. C57BL/6J had the fewest (20887) new transcripts because the reference annotation is based on this strain and therefore is the most complete. We also observed that NOD/ShiLtJ mice had 30695 new transcripts and that all other strains had more new intergenic transcripts than C57BL/6J (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1/FigureS6.pdf">Figure S6</a>A). Surprisingly, wild-derived strains did not have more new transcripts than laboratory strains (except C57BL/6J), although overall differences in the numbers of novel transcripts among the other seven strains were not large. This could be due, in part, to the imperfect state of founder pseudogenomes, which are rooted in the reference genome and thus may not accurately describe the genomes of the other seven strains. This is particularly problematic for the wild-derived strains, whose genomes are more divergent from C57BL/6J. When we plotted the density of the novel transcripts onto mouse genomes, we found that although novel transcripts were spread throughout the eight genomes, they did cluster around certain locations (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). To further investigate this phenomenon, we zoomed-in on chromosome 2 (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). The clustering patterns were remarkably consistent between founders, further validating the presence of these novel transcripts (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>). However, not all locations were identical for the eight founders; some regions showed clear differences between the wild-derived strains and the classical laboratory strains. One such region is found between 45.05 Mb and 45.2 Mb on chromosome 2, where the wild-derived strains tended to have shorter and fewer novel transcripts than the laboratory strains, as illustrated by <a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>. This observation led us to cluster the founder strains based on the novel transcript density of chromosome 2. The resulting similarity distance showed that the wild-derived strains were more distant from the laboratory strains, with the WSB/EiJ strain being the most closely related to the laboratory strains (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1/FigureS6.pdf">Figure S6</a>B). This observation is similar to that of a previous phylogenetic characterization of inbred mouse strains (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Yang <em>et al.</em> 2007</a>).</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/8/1429/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Circos plot of genome-wide novel transcript density. Chromosomes 1 through 19 and X are represented as 20 different segments with their genomic coordinates labeled in megabases. The novel transcript density was defined as the total number of nucleotides within all novel transcripts of a given 100-KB region divided by the total number of nucleotides in that region. We considered imputed introns to be part of novel transcripts to regions of high density often resulting from multiple, overlapping novel transcripts. Because all mice in this report were female, no Y chromosome information is available." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1981492922" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Circos plot of genome-wide novel transcript density. Chromosomes 1 through 19 and X are represented as 20 different segments with their genomic coordinates labeled in megabases. The novel transcript density was defined as the total number of nucleotides within all novel transcripts of a given 100-KB region divided by the total number of nucleotides in that region. We considered imputed introns to be part of novel transcripts to regions of high density often resulting from multiple, overlapping novel transcripts. Because all mice in this report were female, no Y chromosome information is available.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/8/1429/F1.medium.gif" width="437" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/4/8/1429/F1.medium.gif" width="437" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/8/1429/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/8/1429/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464687" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-22" class="first-child">Circos plot of genome-wide novel transcript density. Chromosomes 1 through 19 and X are represented as 20 different segments with their genomic coordinates labeled in megabases. The novel transcript density was defined as the total number of nucleotides within all novel transcripts of a given 100-KB region divided by the total number of nucleotides in that region. We considered imputed introns to be part of novel transcripts to regions of high density often resulting from multiple, overlapping novel transcripts. Because all mice in this report were female, no Y chromosome information is available.</p><div class="sb-div caption-clear"></div></div></div><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/8/1429/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Novel transcript density on chromosome 2. (A) The novel transcript density on chromosome 2 is represented in further detail. The nucleotide position is shown at the top of the figure. Each of the eight sub-panels denotes a single mouse strain. A red box marks a region between 45 Mb and 45.2 Mb where the wild-derived strains, CAST, PWK, and WSB, have shorter or fewer novel transcripts than the other five strains. A magnified view of this region is shown in (B)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1981492922" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Novel transcript density on chromosome 2. (A) The novel transcript density on chromosome 2 is represented in further detail. The nucleotide position is shown at the top of the figure. Each of the eight sub-panels denotes a single mouse strain. A red box marks a region between 45 Mb and 45.2 Mb where the wild-derived strains, CAST, PWK, and WSB, have shorter or fewer novel transcripts than the other five strains. A magnified view of this region is shown in (B).&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/8/1429/F2.medium.gif" width="377" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/4/8/1429/F2.medium.gif" width="377" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/8/1429/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/8/1429/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464623" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-23" class="first-child">Novel transcript density on chromosome 2. (A) The novel transcript density on chromosome 2 is represented in further detail. The nucleotide position is shown at the top of the figure. Each of the eight sub-panels denotes a single mouse strain. A red box marks a region between 45 Mb and 45.2 Mb where the wild-derived strains, CAST, PWK, and WSB, have shorter or fewer novel transcripts than the other five strains. A magnified view of this region is shown in (B).</p><div class="sb-div caption-clear"></div></div></div><p id="p-24">Because the novel transcripts were induced during viral infection, they may play functional roles in the host response. To gain insight into possible functional roles, we tested for DE genes by comparing samples from virus-infected and mock-infected animals. New intergenic transcripts that were DE in response to infection are summarized in <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>. There were considerably more transcripts in 129S1/SvImJ, WSB/EiJ, CAST/EiJ, and NOD/ShiLtJ mice on day 4 after PR8 infection than on day 4 after MA15 infection, and significantly more new transcripts in 129S1/SvImJ, WSB/EiJ, and CAST/EiJ mice on day 4 after PR8 infection than on day 2 after PR8 infection. The pattern of strain variation with regard to the number of novel DE transcripts was similar to the pattern shown by known DE genes. We present an example of a novel DE transcript in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3B</a>. There were thousands or more reads mapped to this transcript, which was present in C57BL/6J on day 2 after MA15 infection. The differential expression of this transcript was striking, with abundant expression in samples from virus-infected animals and little expression in samples from mock-infected animals.</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/8/1429/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Differential expression of novel transcripts. (A) The number of intergenic novel transcripts that are differentially expressed between infected samples and mocks. Computation was performed using the Bioconductor package &#x201C;edgeR.&#x201D; A large proportion of new intergenic transcripts showed differential expression in response to infection, pointing to their potential functional role in the host response. (B) An example of differential expression of novel transcripts in C57BL/6J mice at day 2 after MA15 infection. Note the difference between virus-infected animals (top three samples) and mock-infected animals (bottom four samples). This locus has no annotated gene nearby and is not within known genes, as can be observed by the blank panel of &#x201C;Refseq genes.&#x201D;" class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1981492922" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Differential expression of novel transcripts. (A) The number of intergenic novel transcripts that are differentially expressed between infected samples and mocks. Computation was performed using the Bioconductor package &#x201C;edgeR.&#x201D; A large proportion of new intergenic transcripts showed differential expression in response to infection, pointing to their potential functional role in the host response. (B) An example of differential expression of novel transcripts in C57BL/6J mice at day 2 after MA15 infection. Note the difference between virus-infected animals (top three samples) and mock-infected animals (bottom four samples). This locus has no annotated gene nearby and is not within known genes, as can be observed by the blank panel of &#x201C;Refseq genes.&#x201D;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/8/1429/F3.medium.gif" width="440" height="433"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/4/8/1429/F3.medium.gif" width="440" height="433"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/8/1429/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/8/1429/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464655" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-25" class="first-child">Differential expression of novel transcripts. (A) The number of intergenic novel transcripts that are differentially expressed between infected samples and mocks. Computation was performed using the Bioconductor package “edgeR.” A large proportion of new intergenic transcripts showed differential expression in response to infection, pointing to their potential functional role in the host response. (B) An example of differential expression of novel transcripts in C57BL/6J mice at day 2 after MA15 infection. Note the difference between virus-infected animals (top three samples) and mock-infected animals (bottom four samples). This locus has no annotated gene nearby and is not within known genes, as can be observed by the blank panel of “Refseq genes.”</p><div class="sb-div caption-clear"></div></div></div><p id="p-26">To further characterize the novel transcripts, we selected novel transcripts within four previously studied QTL regions (<a id="xref-ref-17-4" class="xref-bibr" href="#ref-17">Ferris <em>et al.</em> 2013</a>) and associated them with annotated genes. A genome browser view of one QTL region, HrI3, is shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1/FigureS7.pdf">Figure S7</a>. The QTL study used PR8 virus and pre-CC mice, mice that are descendants of the founders in the current study and therefore are particularly relevant. First, we used linear regression to choose novel transcripts associated with weight loss; after multiple-testing correction, we obtained three significant novel transcripts. The three novel transcripts fall into regions previously associated with mouse body weight, pulmonary edema, or airway neutrophils of the mouse response to PR8 and their expression levels have further been associated with weight loss, an indicator of morbidity. Two of these transcripts were found in PWK/PhJ mice and the third transcript was found in C67BL/6J mice. When we used linear regression to characterize the functional roles of the three novel transcripts by associating their expression levels with those of annotated genes, one transcript in PWK/PhJ mice failed to significantly associate with any genes, but the other two transcripts were associated with many. These transcripts included XLOC_000093 in PWK/PhJ mice on chromosome 1 between 21.7 Mb and 29 Mb and XLOC_008559 in C57BL/6J mice on chromosome 15 between 77.4 Mb and 86.6 Mb. By using a cut-off value of 10<sup>−5</sup> as the threshold, we identified more than 200 annotated genes that associated with these two novel transcripts. When we used IPA functional analysis to associate annotated genes with their predicted biological functions, we found that genes associated with XLOC_000093 were associated with TREM1 signaling, B cell receptor signaling, and NF-κB activation. Given the fact that these novel transcripts are within a QTL region associated with the host response to PR8, are highly associated with weight loss after infection, and are associated with many important immune genes, it is likely that they have a functional role in the host response to viral infection.</p></div><div id="sec-10" class="subsection"><h3>Lack of alignment between <em>de novo</em> transcripts and mouse genomes reveals multiple gaps in existing mouse genome sequences</h3><p id="p-27">Our assembly pipeline not only identified many <em>de novo</em> transcripts that did not overlap with known genes but also identified transcripts that could not be placed on either the reference genome or the various founder pseudogenomes. We took a conservative approach to mapping these transcripts to minimize false positives. First, we selected the parts of the output from Trinity that did not map to pseudogenomes or the reference genome. We then filtered the data by first removing transcripts mapped to viral and ribosomal genomes, then by retaining only those transcripts that aligned to known human, rat, and mouse sequences (ESTs or DNA clones), and finally by requiring a minimum length of 200 bp per transcript. A summary of the remaining <em>de novo</em> transcripts is presented in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4A</a>. Because both the reference annotation and the reference genome are based on samples from the C57BL/6J strain, it is no surprise that C57BL/6J had the fewest <em>de novo</em> transcripts among all the founders. However, it is somewhat surprising that there were ∼600 <em>de novo</em> transcripts in this strain. On average, the laboratory strains (A/J, 129S1/SvImJ, C57BL/6J, NOD/ShiLtJ, NZO/HILtJ) had fewer <em>de novo</em> transcripts than the wild-derived strains (CAST/EiJ, PWK/PhJ, WSB/EiJ), reflecting the greater genomic difference between the wild-derived strains and C57BL/6J. Some of the <em>de novo</em> transcripts were more than 1000 bp long, with the transcripts shown in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4B</a> being prominent examples. These transcripts have very low similarity to the reference genome and pseudogenomes but could be mapped to both rat and human genomes with coverage approximately 70% or more. Interestingly, these transcripts could also be mapped to a lesser known mouse genome assembly called Mm_Celera (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Mural <em>et al.</em> 2002</a>), which was generated by Celera genomics based on multiple mouse strains. One prominent example is the <em>de novo</em> transcripts that spanned multiple conserved exons of the gene <em>Spata5l1</em> and aligned well with respective exons of human and rat genomes (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>). These data provide strong evidence that the existing reference genomes and various pseudogenomes have gaps within well-known coding genes.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/8/1429/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="De novo transcripts that do not map to existing mouse genes. The de novo transcripts were assembled from RNA-seq reads not mappable to pseudogenomes and then the transcripts were filtered by mapping the transcripts to pseudogenomes. The de novo transcripts were filtered using BLAST to probe the &#x201C;nx&#x201D; database of known mouse, rat, and human sequences. (A) The number of de novo transcripts. The de novo transcripts were fewest in the C57BL/6J strain, whereas the wild-derived strains, CAST, PWK, and WSB, had more de novo transcripts than laboratory strains. (B) An example showing that multiple de novo transcripts from all eight founders could be mapped to the human Spata5l1 gene, spermatogenesis-associated 5-like 1." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1981492922" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;De novo&lt;/em&gt; transcripts that do not map to existing mouse genes. The &lt;em&gt;de novo&lt;/em&gt; transcripts were assembled from RNA-seq reads not mappable to pseudogenomes and then the transcripts were filtered by mapping the transcripts to pseudogenomes. The &lt;em&gt;de novo&lt;/em&gt; transcripts were filtered using BLAST to probe the &#x201C;nx&#x201D; database of known mouse, rat, and human sequences. (A) The number of &lt;em&gt;de novo&lt;/em&gt; transcripts. The &lt;em&gt;de novo&lt;/em&gt; transcripts were fewest in the C57BL/6J strain, whereas the wild-derived strains, CAST, PWK, and WSB, had more &lt;em&gt;de novo&lt;/em&gt; transcripts than laboratory strains. (B) An example showing that multiple &lt;em&gt;de novo&lt;/em&gt; transcripts from all eight founders could be mapped to the human &lt;em&gt;Spata5l1&lt;/em&gt; gene, spermatogenesis-associated 5-like 1.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/8/1429/F4.medium.gif" width="440" height="309"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/4/8/1429/F4.medium.gif" width="440" height="309"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/8/1429/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/8/1429/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464792" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-28" class="first-child"><em>De novo</em> transcripts that do not map to existing mouse genes. The <em>de novo</em> transcripts were assembled from RNA-seq reads not mappable to pseudogenomes and then the transcripts were filtered by mapping the transcripts to pseudogenomes. The <em>de novo</em> transcripts were filtered using BLAST to probe the “nx” database of known mouse, rat, and human sequences. (A) The number of <em>de novo</em> transcripts. The <em>de novo</em> transcripts were fewest in the C57BL/6J strain, whereas the wild-derived strains, CAST, PWK, and WSB, had more <em>de novo</em> transcripts than laboratory strains. (B) An example showing that multiple <em>de novo</em> transcripts from all eight founders could be mapped to the human <em>Spata5l1</em> gene, spermatogenesis-associated 5-like 1.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">To further illustrate the power of our RNA-seq data to uncover gaps in the mouse genome, we recapitulated genomic differences between founders for <em>Mx1</em>, an important immune gene. Our groups have shown previously that four of the founders, A/J, C57BL/6J, 129S1/SvImJ, and NOD/ShiLtJ, have a deletion of three exons in the middle of <em>Mx1</em>, whereas the other four founders have a full complement of exons (<a id="xref-ref-17-5" class="xref-bibr" href="#ref-17">Ferris <em>et al.</em> 2013</a>). We compared our <em>de novo</em> transcripts that could not be placed on the chromosomes of the pseudogenome with the sequences of exons 9 to 11, the deleted exons. We were able to completely recapitulate the finding of <a id="xref-ref-17-6" class="xref-bibr" href="#ref-17">Ferris <em>et al.</em> (2013)</a> so that there were transcripts with more than 600-bp overlaps for four founders with full gene models and no match for founders with deleted exons. The alignment of these transcripts to the rat genome showed a similar pattern as observed in <a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4B</a>, with transcripts showing affinity to conserved exons in the genome of a closely related species (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1/FigureS8.pdf">Figure S8</a>).</p></div><div id="sec-11" class="subsection"><h3>Differential isoform expression and the host response to viral infection</h3><p id="p-30">The majority of mammalian genes have multiple isoforms (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Wang <em>et al.</em> 2008</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Zhang <em>et al.</em> 2013</a>), the significance of which is increasingly being recognized (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Trapnell <em>et al.</em> 2010</a>). Isoform expression has been linked to the host response to IAV (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Terrier <em>et al.</em> 2012</a>), and isoforms of immune loci such as HLA have long been recognized as being important to immune function (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">An <em>et al.</em> 2013</a>; <a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Lin <em>et al.</em> 2003</a>). RNA-seq is theoretically capable of distinguishing between different isoforms in an unbiased survey, but current short-read technologies make isoform quantification challenging. We circumvented the technical challenges of isoform quantification by focusing on the expression profiles of splice junctions. Because splice junctions are intimately tied to genomic coordinates, we mapped RNA reads to the reference genome and excluded false positives by permitting only canonical splice sites.</p><p id="p-31">The first thing we observed was the distinct expression patterns of alternative splice junctions among the founders (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1/TableS1.pdf">Table S1</a>). We looked for alternative splice junctions expressed in one strain and not the other. A minimum read count of 10 was required for a splice junction to be considered expressed, whereas the maximum read count for a junction to be considered not expressed was five; the gap is a conservative filtering criterion. As expected, there were large strain-specific differences in alternative splice expression with thousands of splice junctions expressed in one founder strain and not the other (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1/TableS1.pdf">Table S1</a>). Wild-derived PWK/PhJ had the highest number of splice junctions expressed that could not be found in other strains. The strain-specific differences in splice junction expression foreshadowed pervasive differences in isoform expression patterns.</p><p id="p-32">Next, we studied differential isoform expression in the context of IAV and SARS-CoV infection. At one extreme, we looked for splice junctions expressed in samples from virus-infected animals but not in samples from mock-infected animals of the same strain (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1/TableS2.pdf">Table S2</a>). Strain differences were clearly evident. The number of splice junctions expressed in samples from virus-infected animals ranged from a few hundred to a few thousand. There were also large differences for individual strains with respect to different viruses and time points.</p><p id="p-33">We also performed a detailed analysis of differential isoform expression for individual genes. Because we were using differential alternative splice junction expression as a proxy for differential isoform expression, we excluded splice junctions inside DE genes to disambiguate real from ambiguous differential alternative splicing due to differential gene expression. Using this approach, we obtained the number of differentially expressed alternative splice junctions (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1/TableS3.pdf">Table S3</a>). Some of them were in known immune genes such as <em>Irak1</em> (interleukin-1 receptor-associated kinase 1). The <em>Irak1</em> gene only showed differential isoform expression in CAST/EiJ mice, indicating strain-specific expression (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1/FigureS9.pdf">Figure S9</a>). The other DE isoforms we discovered also tended to be strain-specific, but this might be due, in part, to our extremely conservative approach to discovering differential isoform expression and the small number of DE isoforms we uncovered.</p><p id="p-34">There are two ways in which genetic variation can influence isoform expression. One is through distant regulation, by changing the expression of transcription factors or their affinity for distant promoters. The other is by changing local sequences such as spliceosome binding sites or intron–exon boundaries. While genetic variation was an obvious source of splicing variations, it was not clear whether differential expression was due to distant regulation or direct sequence alteration of local sites. Distant regulators are difficult to ascertain without a full genetic study, but local sequence variations could be assessed with our dataset. We therefore compared our list of DE splice junctions with published genetic variations obtained through a next-generation sequencing project (<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Keane <em>et al.</em> 2011</a>) and extracted DE isoforms with genetic variations within 2 bp of their intron–exon boundaries. Because of the proximity of these genetic variations to intron–exon boundaries, this subset of DE isoforms is likely the result of <em>cis</em>-regulated gene expression. One example is found in the <em>Atp5f1</em> gene (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). <em>Atp5f1</em> is an ATP synthase and transport gene that has been linked to hepatitis B infection (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Lee <em>et al.</em> 2008</a>). This gene, although not differentially expressed overall, had two splice junctions showing elevated expression in samples from PR8-infected WSB/EiJ mice on day 4 after infection. Furthermore, there was a SNP (rs33689100) at 105758959 on chromosome 3 that was within 2 bp of an intron–exon boundary. Another example was the <em>Tdrd3</em> gene, which showed differential isoform expression in CAST/EiJ mice on day 2 after MA15 infection. This gene has been linked to stress granules, which are co-opted by several viruses during replication (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lloyd 2013</a>). At 87886156 on chromosome 14, there was an insertion (rs234605399) within 2 bp of an intron–exon boundary, likely causing the splice junction spanning that position to be differentially expressed (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a>). The splice junction neighboring rs234605399 had far fewer reads in samples from virus-infected animals, although the overall gene expression was similar for all samples.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/8/1429/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Cis-regulated isoform differential expression. Genes Atp5f1 (A) and Tdrd3 (B) exhibited cis-regulated differential isoform expression in response to infection. Although neither gene showed differential expression overall, they both expressed select splice junctions differentially and contained strain-specific genetic variations within 2 bp of the splice junctions. Gene Atp5f1 (A) had a splice junction on chromosome 3 from 105758960 to 105761779 that was differentially expressed at day 4 after PR8 infection in WSB mice, which contains a SNP, rs33689100, at 105758959 on chromosome 3 that changes nucleotide C to A. (B) Tdrd3 is a large gene spanning over 128 kb that exhibits isoform differential expression at day 2 after infection with MA15 in CAST mice. The splice junction on chromosome 14 from 87886156 to 87886968 had substantially fewer read counts in samples from infected animals (top three) than in samples from mock-infected animals (bottom four). The adjusted p-value was 2.93&#xD7;10&#x2212;5 for the rightmost junction and 0.35 for the entire gene. In addition, Tdrd3 in CAST mice had an insertion at 87886156 bp, which impacted splicing and made the splicing pattern of this strain different from other founder strains." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1981492922" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Cis&lt;/em&gt;-regulated isoform differential expression. Genes &lt;em&gt;Atp5f1&lt;/em&gt; (A) and &lt;em&gt;Tdrd3&lt;/em&gt; (B) exhibited &lt;em&gt;cis&lt;/em&gt;-regulated differential isoform expression in response to infection. Although neither gene showed differential expression overall, they both expressed select splice junctions differentially and contained strain-specific genetic variations within 2 bp of the splice junctions. Gene &lt;em&gt;Atp5f1&lt;/em&gt; (A) had a splice junction on chromosome 3 from 105758960 to 105761779 that was differentially expressed at day 4 after PR8 infection in WSB mice, which contains a SNP, rs33689100, at 105758959 on chromosome 3 that changes nucleotide C to A. (B) &lt;em&gt;Tdrd3&lt;/em&gt; is a large gene spanning over 128 kb that exhibits isoform differential expression at day 2 after infection with MA15 in CAST mice. The splice junction on chromosome 14 from 87886156 to 87886968 had substantially fewer read counts in samples from infected animals (top three) than in samples from mock-infected animals (bottom four). The adjusted p-value was 2.93&#xD7;10&lt;sup&gt;&#x2212;5&lt;/sup&gt; for the rightmost junction and 0.35 for the entire gene. In addition, Tdrd3 in CAST mice had an insertion at 87886156 bp, which impacted splicing and made the splicing pattern of this strain different from other founder strains.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/8/1429/F5.medium.gif" width="402" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/4/8/1429/F5.medium.gif" width="402" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/8/1429/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/8/1429/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464669" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-35" class="first-child"><em>Cis</em>-regulated isoform differential expression. Genes <em>Atp5f1</em> (A) and <em>Tdrd3</em> (B) exhibited <em>cis</em>-regulated differential isoform expression in response to infection. Although neither gene showed differential expression overall, they both expressed select splice junctions differentially and contained strain-specific genetic variations within 2 bp of the splice junctions. Gene <em>Atp5f1</em> (A) had a splice junction on chromosome 3 from 105758960 to 105761779 that was differentially expressed at day 4 after PR8 infection in WSB mice, which contains a SNP, rs33689100, at 105758959 on chromosome 3 that changes nucleotide C to A. (B) <em>Tdrd3</em> is a large gene spanning over 128 kb that exhibits isoform differential expression at day 2 after infection with MA15 in CAST mice. The splice junction on chromosome 14 from 87886156 to 87886968 had substantially fewer read counts in samples from infected animals (top three) than in samples from mock-infected animals (bottom four). The adjusted p-value was 2.93×10<sup>−5</sup> for the rightmost junction and 0.35 for the entire gene. In addition, Tdrd3 in CAST mice had an insertion at 87886156 bp, which impacted splicing and made the splicing pattern of this strain different from other founder strains.</p><div class="sb-div caption-clear"></div></div></div><p id="p-36">We also identified unannotated untranslated regions (UTRs) in our results, although we did not systematically search for unannotated UTRs. <em>Iigp1</em> (interferon-inducible GTPase 1) has an unannotated 5′ UTR that is highly expressed in samples from infected animals (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). This is an example of differential isoform expression with different starting sites. Although the functional roles of the unannotated 5′ UTR and differential isoform expression are not clear, given the role that <em>Iigp1</em> has in modulating the interferon-γ (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Boehm <em>et al.</em> 1998</a>) response, it is possible that its 5′ UTR plays a role in driving the host response.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/8/1429/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Differential expression of untranslated regions. The 5&#x2032; end of the annotated gene Iigp1 (interferon inducible GTPase 1) contains an unannotated untranslated region (UTR) that is expressed only during infection. The annotated portion was differentially expressed but samples from mock-infected animals had hundreds if not thousands of reads whereas the unannotated 5&#x2032; UTR had practically none. This was observed in C57BL/6J mice, suggesting that the Iigp1 gene model is incomplete in the reference annotation." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1981492922" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Differential expression of untranslated regions. The 5&#x2032; end of the annotated gene &lt;em&gt;Iigp1&lt;/em&gt; (interferon inducible GTPase 1) contains an unannotated untranslated region (UTR) that is expressed only during infection. The annotated portion was differentially expressed but samples from mock-infected animals had hundreds if not thousands of reads whereas the unannotated 5&#x2032; UTR had practically none. This was observed in C57BL/6J mice, suggesting that the &lt;em&gt;Iigp1&lt;/em&gt; gene model is incomplete in the reference annotation.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/8/1429/F6.medium.gif" width="440" height="232"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/4/8/1429/F6.medium.gif" width="440" height="232"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/8/1429/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/8/1429/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464747" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-37" class="first-child">Differential expression of untranslated regions. The 5′ end of the annotated gene <em>Iigp1</em> (interferon inducible GTPase 1) contains an unannotated untranslated region (UTR) that is expressed only during infection. The annotated portion was differentially expressed but samples from mock-infected animals had hundreds if not thousands of reads whereas the unannotated 5′ UTR had practically none. This was observed in C57BL/6J mice, suggesting that the <em>Iigp1</em> gene model is incomplete in the reference annotation.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-12" class="subsection"><h3>Genome sequence variations and tight regulation drive strain-specific gene expression</h3><p id="p-38">We performed differential expression analysis on annotated genes and found widespread strain specificity in the numbers and the functional enrichment of DE genes; the details of which can be found in the <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1/FileS1.pdf">Supplementary Materials in File S1</a>. Although it was clear that genetic differences drove the strain-specific expression patterns that were described above, it was unclear whether this was by indirect genetic regulation or by direct genome sequence changes. Examples of direct sequence alteration include premature transcriptional stops and reduced transcription factor–binding affinity. To tease apart these sources of distinct expression patterns and their likely functional roles, we first separated the DE genes into three groups: those not expressed in all the samples of one mouse strain (group A); those expressed in all strains (group B); and the remainder. In other words, group A genes were differentially expressed during virus infection for at least one strain but were not expressed for another strain for all time points and all viruses (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>). This group includes genes with extreme expression difference between strains. We used group A gene expression levels to cluster all founder strains and discovered that the resulting similarity graph (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7B</a>) resembled the phylogenetic tree based on genetic variations (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Yang <em>et al.</em> 2007</a>). Using the remaining DE genes, or all DE genes, did not yield the same similarity graph. This suggests that genome sequence variations between founders had a large and immediate impact on the expression of these genes. The genome sequence variation driving selective expression of these genes is likely nearby in upstream binding sites or promoters, which are different between strains because of SNPs, small indels, or structural variations. In extreme cases, whole genes may be deleted in certain strains. Although genetic regulation is always a factor in gene expression, for these genes, expression differences between founder strains may be more directly explained by genome sequence variations within or near those genes.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/8/1429/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Some differentially expressed genes are not expressed in one or more founder strains. Of the 9149 DE genes, 2150 lacked expression in one or more founder strains. (A) Log-fold changes are shown as a heat map. The pattern of white space, which denotes nonexpression, clearly discriminates the founder strains. (B) Clustering the strains based on nonexpression pattern resulted in a graph resembling a previous phylogenetic tree computed from genetic variations of the founder strains (Yang et al. 2007)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1981492922" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Some differentially expressed genes are not expressed in one or more founder strains. Of the 9149 DE genes, 2150 lacked expression in one or more founder strains. (A) Log-fold changes are shown as a heat map. The pattern of white space, which denotes nonexpression, clearly discriminates the founder strains. (B) Clustering the strains based on nonexpression pattern resulted in a graph resembling a previous phylogenetic tree computed from genetic variations of the founder strains (Yang &lt;em&gt;et al.&lt;/em&gt; 2007).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/8/1429/F7.medium.gif" width="403" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/4/8/1429/F7.medium.gif" width="403" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/8/1429/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/8/1429/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464818" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-39" class="first-child">Some differentially expressed genes are not expressed in one or more founder strains. Of the 9149 DE genes, 2150 lacked expression in one or more founder strains. (A) Log-fold changes are shown as a heat map. The pattern of white space, which denotes nonexpression, clearly discriminates the founder strains. (B) Clustering the strains based on nonexpression pattern resulted in a graph resembling a previous phylogenetic tree computed from genetic variations of the founder strains (<a id="xref-ref-54-3" class="xref-bibr" href="#ref-54">Yang <em>et al.</em> 2007</a>).</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">We further divided the DE genes belonging to group B into two categories using median absolute deviation (MAD) of log-fold change as the metric. If a gene had a MAD score less than 0.1 within a unique combination of time point and virus, then it was considered a consistently expressed gene (<a id="xref-fig-8-1" class="xref-fig" href="#F8">Figure 8A</a>); if the MAD score was more than 1, then it was considered a gene with highly variable expression (<a id="xref-fig-8-2" class="xref-fig" href="#F8">Figure 8B</a>). Consistently expressed genes included genes that were similarly expressed across all eight founders, whereas genes with highly variable expression between strains are likely to be ultimately responsible for differing phenotypes. This is a further gradation of genes with strain-specific expression patterns; the highly variable genes have strain-specific pattern, but the stably expressed genes do not. Interestingly, of four combinations, PR8 <em>vs.</em> MA15 at day 2 or day 4 after infection, consistently expressed genes did not overlap among different viruses and time points, and neither did genes with highly variable expression (not shown). This suggests a tightly controlled host-response program in response to infection. The expression patterns shown in <a id="xref-fig-8-3" class="xref-fig" href="#F8">Figure 8, A and B</a> have 445 consistently expressed genes and 835 genes with highly variable expression. When we looked for pathway enrichment using GO terms, we found the consistently expressed genes were dominated by general biological processes and cellular functions, with low enrichment of immune functions or pathways. The top 10 enriched pathways for consistently expressed genes are in <a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>. There, only one pathway, pro-B-cell differentiation biological process, was related to immune function. The opposite was true for genes with highly variable expression, whose GO terms were predominantly immune functions such as innate immune system and interferon stimulation. The top 10 enriched pathways are listed in <a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>. Functional enrichment of immune functions remained high regardless of whether all known genes or just the DE genes were used as the background, so the high functional enrichment of genes with highly variable expression was not confounded by the enrichment of DE genes in immune functions. It is possible that consistently expressed genes are largely responsible for maintaining basic biological functions, whereas genes with highly variable expression between strains are responsible for the manifestation of different host responses.</p><div id="F8" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/4/8/1429/F8.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Highly variably and very stably expressed DE genes. Of the genes that were differentially expressed in all strains between infected and mock-infected animals, 445 were consistently expressed whereas 836 were highly variably expressed. Their respective log-fold change patterns are shown in (A) and (B), respectively. The functional enrichments of the respective genes are shown in Table 1 and Table 2." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1981492922" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Highly variably and very stably expressed DE genes. Of the genes that were differentially expressed in all strains between infected and mock-infected animals, 445 were consistently expressed whereas 836 were highly variably expressed. Their respective log-fold change patterns are shown in (A) and (B), respectively. The functional enrichments of the respective genes are shown in Table 1 and Table 2.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 8" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/4/8/1429/F8.medium.gif" width="283" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 8" src="https://www.g3journal.org/content/ggg/4/8/1429/F8.medium.gif" width="283" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/4/8/1429/F8.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 8" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/4/8/1429/F8.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/464746" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 8</span> <p id="p-41" class="first-child">Highly variably and very stably expressed DE genes. Of the genes that were differentially expressed in all strains between infected and mock-infected animals, 445 were consistently expressed whereas 836 were highly variably expressed. Their respective log-fold change patterns are shown in (A) and (B), respectively. The functional enrichments of the respective genes are shown in <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a> and <a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>.</p><div class="sb-div caption-clear"></div></div></div><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464742/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464742/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Functional enrichment of consistently expressed genes</span><div class="sb-div caption-clear"></div></div></div><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/464732/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/464732/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Functional enrichment of highly variably expressed genes.</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-13" class="subsection"><h3>Widespread strain-specific gene expression points to diverse host responses to viral infection</h3><p id="p-44">We mapped short RNA-seq reads to pseudogenomes and generated read counts based on the reference annotation adapted to pseudogenomes to obtain expression profiles for all 124 samples. Five samples were excluded from the gene expression analysis due to unusually low viral read counts and expression similar to that of the mock samples. The number of replicates for each comparison is listed in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1/TableS4.pdf">Table S4</a>. The remaining samples showed a clear separation of infected and mock samples, which is apparent in an MDS plot for all samples (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1/FigureS10.pdf">Figure S10</a>). For samples from A/J strain day 2 after being infected with MA15, the MDS plot shows clear separation between infected samples and mock samples (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1/FigureS11.pdf">Figure S11</a>). Differential expression analysis comparing infected samples against mock samples within each founder strain was performed using edgeR (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Robinson <em>et al.</em> 2009</a>) with a stringent criterion of FDR &lt;0.01 and a fold change of ±2.</p><p id="p-45">There were considerable strain-specific differences for both PR8 and MA15 on days 2 and 4 after infection. These were apparent even at the summary level of the numbers of differentially expressed genes (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1/FigureS12.pdf">Figure S12</a>). For example, PWK/PhJ had far fewer DE genes (223) than the other strains on day 2 after PR8 infection, whereas NOD/ShiLtJ had the most DE genes (2454). At day 4 after infection, all PR8-infected strains had markedly increased numbers of DE genes, except for A/J, which had a substantial decrease (from 1868 to 702). However, MA15-infected animals had fewer DE genes on day 4 than on day 2 or similar numbers of DE genes on both days. The largest number of DE genes (3748 in PR8-infected 129S1/SvImJ on day 4) was approximately an order of magnitude greater than the smallest number of DE genes (223 in PR8-infected PWK/PhJ on day 2), indicating that genetic variation among the founders was driving differences in the host response to viral infection.</p><p id="p-46">Strain variation in the number of DE genes was reflected in functional analysis. When IPA was used to identify pathways that were enriched in DE genes, key immune pathways such as granulocyte adhesion and diapedesis, agranulocyte adhesion and diapedesis, crosstalk between dendritic cells and natural killer cells, and TREM1 signaling were among the top 10 enriched pathways with enrichment scores (-log p-value) ranging from more than 3 to more than 20. Strain differences are profound for individual pathways. One such pathway was “granulocyte adhesion and diapedesis,” which is a key innate immune pathway related to vascular cuffing. As shown in a radial plot of the enrichment scores of all eight founders, PR8-infected PWK/PhJ mice had the lowest enrichment score (4.34) on day 2 after infection, whereas NZO/HILtJ had the highest enrichment score (22.67) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1/FigureS13.pdf">Figure S13</a>).</p><p id="p-47">Select genes important to immune response were largely responsible for the vastly different enrichment scores for pathways such as “interferon signaling.” Two examples were interferon-γ and IL-10. Interferon-γ has been shown to protect mice from influenza virus if administrated at an early stage of infection (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Sainz <em>et al.</em> 2004</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Weiss <em>et al.</em> 2010</a>), and genetic variation within the <em>IFNG</em> gene has been linked to SARS in humans (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Chong <em>et al.</em> 2006</a>). IL-10 is an anti-inflammatory cytokine that is linked to influenza recovery (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">McKinstry <em>et al.</em> 2009</a>) and the effectiveness of SARS treatment (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Ng <em>et al.</em> 2004</a>). Its expression can also predict patient survival among people infected with emergent H7N9 virus (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Wang <em>et al.</em> 2013</a>). Given their pivotal role in fighting IAV and SARS, it was intriguing that these genes showed strain-specific differential expression. Among mice infected with PR8 at day 2 after infection, <em>IFNG</em> was only expressed in C57BL/6J, CAST/EiJ, PWK/PhJ, and WSB/EiJ, whereas it was expressed in all strains except CAST/EiJ on day 4 after infection. A recent article also reported an absence of <em>IFNG</em> expression in the lungs of CAST/EiJ mice (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Earl <em>et al.</em> 2012</a>). For SARS-CoV infection, <em>IFNG</em> was expressed in all but the CAST/EiJ strain at day 2 and in all strains at day 4.</p></div><div id="sec-14" class="subsection"><h3>Viral-read percentages are highly predictive of weight loss</h3><p id="p-48">We separated short reads mapped to ribosomes and viruses from those mapped to mouse genomes. The numbers of short reads mapped to host, ribosomes, and viruses are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1/FigureS1.pdf">Figure S1</a>. To better compare viral reads, viral read counts were normalized to host read counts, which varied as a result of sequencing technology and our sequencing depth requirement, to yield viral read percentages (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1/FigureS2.pdf">Figure S2</a>). Viral read counts of mocks are in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1/TableS5.pdf">Table S5</a>.</p><p id="p-49">The number of viral reads and viral titer both measure viral concentration and could be related and to some extent can gauge the extent of viral replication. Total viral read counts and viral read percentages varied greatly among founders, and with different viruses and at different time points (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1/FigureS1.pdf">Figure S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1/FigureS2.pdf">Figure S2</a>). In particular, the CAST/EiJ strain had far more viral reads than did the other strains on day 2 after MA15 virus infection, especially compared to A/J strain, which had 11-fold fewer viral read counts; this further elaborated earlier observations about the CAST/EiJ strain among four founders (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Peng <em>et al.</em> 2010</a>). Clear but less extreme variations could be observed in other strains.</p><p id="p-50">Mouse weights were monitored daily because weight loss is considered a clinical sign of morbidity caused by viral infection. All mice except those killed at day 2 had body weights monitored up to day 4 after infection. The weight loss curves, defined as the percentage difference between day 0 weights and subsequent days’ weights, are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1/FigureS3.pdf">Figure S3</a> for MA15 samples and in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1/FigureS4.pdf">Figure S4</a> for PR8 samples. In general, the NZO/HILtJ strain showed little weight loss for both viruses and on all days. NOD/ShiLtJ, A/J, and C57BL/6J strains recovered some weight loss on day 4 after MA15 infection, but no strains showed similar weight recovery with PR8 infection. CAST/EiJ mice lost substantial weight by day 4, particularly when infected with MA15, with weight loss of more than 20% being observed. Interestingly, the CAST/EiJ strain had high viral read counts on day 2 after MA15 infection that decreased substantially by day 4, whereas the mice continued to lose almost 10% of their weight.</p><p id="p-51">Previous reports have shown that viral titer could predict weight loss (<a id="xref-ref-17-7" class="xref-bibr" href="#ref-17">Ferris <em>et al.</em> 2013</a>). We also found that there was a strong correlation between viral-read percentages and day 4 body weight loss (Spearman correlation coefficient = −0.73) and that viral-read percentages were strongly predictive of weight loss at day 4 (p-value &lt;1.34×10<sup>−10</sup>). There was no correlation between viral-read percentages and day 2 weight loss, probably because it was too early in the course of infection for body weight to be impacted by infection. The scatter plot of viral-read percentages and day 4 weight loss is shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1/FigureS14.pdf">Figure S14</a>, in which a clear linear relation can be observed.</p></div></div><div class="section discussion" id="sec-15"><h2 class="">Discussion</h2><p id="p-52">In this study, we characterized the transcriptional landscape of the eight CC founder strains in response to respiratory viral infection and identified thousands of new transcriptional elements yet to be recorded in the mouse reference annotation. The transcriptional patterns were starkly different between the eight founders, mirroring the phenotypic diversity observed in our study as well as other studies. We observed extreme phenotypes in weight loss and survival among the CC founder strains, similar to that reported in previous studies of mouse genetic regulation of IAV pathogenesis (<a id="xref-ref-17-8" class="xref-bibr" href="#ref-17">Ferris <em>et al.</em> 2013</a>) and immune cell associations (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Phillippi <em>et al.</em> 2014</a>) using incipient CC lines. The diversity of phenotypes observed in these studies mimics the diverse clinical symptoms of patients infected with either IAV or SARS-CoV (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Pabst <em>et al.</em> 2011</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Igusa <em>et al.</em> 2012</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Zaas <em>et al.</em> 2009</a>).</p><p id="p-53">As more quantitative trait loci (QTL) studies with CC mice (<a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Durrant <em>et al.</em> 2011</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Zheng <em>et al.</em> 2012</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Flint and Eskin 2012</a>) are undertaken, the need for a complete reference genome and annotation will increase. Even though C57BL/6J is the reference strain, we nevertheless found hundreds of missing sequences and tens of thousands of intergenic transcripts in this strain. The other seven strains had even more novel features. It is likely that many of the intergenic transcripts were the extended fragments of known genes, or that multiple new transcripts came from one new gene, so that the total numbers of novel transcripts are lower than estimated. However, the existence of large numbers of unannotated or extended genes is undoubtedly real in the mouse as well as other species. The implication of incomplete reference genome and annotation for QTL and eQTL studies is substantial. Although the transcriptome data presented here will help overcome these shortcomings in mouse studies, the concern remains for other systems, such as human GWAS. The potential pitfalls of incomplete genomes were illustrated by the example of <em>Spata5l1</em>, which had multiple exons missing in all eight strains. For studies of genetic control of transcription, our results can help by pointing out potentially incomplete gene models in the reference annotation and can also help complete the mouse annotation with new intergenic transcripts. Our results have particular relevance for viral infection studies, particularly respiratory viruses, as these new transcripts in lung may have functional roles, as evidenced by their differential expression in response to infection.</p><p id="p-54">We also observed that genes showing strain-specific differential expression patterns were heavily enriched in immune pathways, while genes showing generic differential expression patterns had more generic biological functions. This phenomenon may be due, in part, to immune cell infiltration. It is possible that the genes with strain-specific expression are genes that are expressed by different infiltrating cell populations, which may vary between the founders, whereas the nonstrain-specific genes are not active in infiltrating cells. An alternative explanation is that gene expression changes that were not strain-specific are so fundamentally important to mouse survival that they are similarly regulated in all founders. Further study of a variety of CC lines and/or by cell sorting and cell-specific transcriptional analysis could help to explain this phenomenon and its implications on host survival.</p><p id="p-55">Our results may also be used to prioritize candidate genes identified by QTL analysis. For example, the 1700026D08Rik gene is associated with day 4 weight loss after PR8 infection (<a id="xref-ref-17-9" class="xref-bibr" href="#ref-17">Ferris <em>et al.</em> 2013</a>). We found that the same gene had DE isoforms in NOD/ShiLtJ mice at day 2 after PR8 infection. Although there was no genetic variation near intron–exon boundaries of this gene in NOD/ShiLtJ mice, this strain-specific isoform expression pointed to possible genetic influence and therefore higher likelihood of being the causal gene among the 69 genes and 10 noncoding RNAs that were significantly associated with day 4 weight loss along with 1700026D08Rik. This is especially important because many genes, such as 1700026D08Rik, have poorly described or no known functions. We found that genes such as <em>IL6</em>, <em>Sh3gl3</em>, and <em>Atxn10</em>, which had previously been associated with different phenotypic outcomes in response to IAV infection, also expressed different isoforms in response to SARS-CoV infection. In this case, they have not been identified as QTL candidate genes for SARS-CoV infection, but their likely importance in the host response is increased because their genetic variations may cause large phenotypic variations and their isoforms were differentially expressed in samples from SARS-CoV–infected animals. Because CC mice still have relatively large recombination intervals, the resolution of QTL using CC mice is not at the gene level, and QTL often will have many candidate genes within a significant region. Isoform differential expression results could prove very helpful in finding causal genes in future QTL studies using CC mice by searching for strain-specific DE isoforms among candidate genes. Candidate genes whose protective alleles differentially express isoforms and whose deleterious alleles do not differentially express isoforms are likely the causal genes.</p><p id="p-56">This study is a first important step in extracting full information from transcripts in CC mice. We have provided examples and catalogs of discrepancies with the reference genome and reference annotation. Our differential expression analysis and subsequent functional analysis revealed distinct and focused host responses to viral infection across the eight CC founder strains. The full extent and importance of isoform differential expression are currently unclear. However, it is clear that genetic diversity drives diverse, intricate, and multi-faceted immune responses at the isoform, gene, and pathway levels through direct alteration of gene sequences, indirect regulation of expression, or a combination of both. The transcriptome data from this study are a resource to the mouse genetics community and are of direct benefit to researchers studying transcriptional complexity in genetically diverse populations.</p><p id="p-57">Our study is but one step toward characterizing the transcriptome of genetically diverse mice. mRNA-seq data and Cap analysis gene expression (CAGE) data in addition to our total RNA-seq data can improve gene models for annotated genes and uncover alternative transcription start and end sites. A dearth of good, full genomes for the founders hampered our comparison across strains. Relative small sample sizes for unique combinations of strain, virus, and days after infection are problematic. Considerable variability in mouse body weight within one strain has been observed, but there were too few mice to properly estimate true variance. Future studies can solve these issues by leveraging our data and results.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-58">This work was supported by Public Health Service grant U19 AI100625 from the National Institute of Allergy and Infectious Diseases, National Institutes of Health.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-1"><p id="p-1">Supporting information is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1">http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.114.011759/-/DC1</a></p></li><li class="fn" id="fn-2"><p id="p-2">Data access: GSE52405</p></li><li class="fn" id="fn-5"><p id="p-59"><em>Communicating editor: D. W. Threadgill</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2014-04-18"><span class="received-label">Received </span>April 18, 2014.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2014-05-31"><span class="accepted-label">Accepted </span>May 31, 2014.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2014 Xiong <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-3">This is an open-access article distributed under the terms of the Creative Commons Attribution Unported License (<a href="http://creativecommons.org/licenses/by/3.0/" rel="license">http://creativecommons.org/licenses/by/3.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.1" data-doi="10.1016/j.jaut.2013.07.010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">An</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakajima</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shibata</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arimura</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yasunami</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A novel link of HLA locus to the regulation of immunity and infection: NFKBIL1 regulates alternative splicing of human immune-related genes and influenza virus M gene.</span> <abbr class="cit-jnl-abbrev">J. Autoimmun.</abbr> <span class="cit-vol">47</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAutoimmun.%26rft.volume%253D47%26rft.spage%253D25%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.jaut.2013.07.010%26rft_id%253Dinfo%253Apmid%252F23953137%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.jaut.2013.07.010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23953137&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.2" data-doi="10.1186/gb-2010-11-10-r106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Anders</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Differential expression analysis for sequence count data.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">R106</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DAnders%26rft.auinit1%253DS.%26rft.volume%253D11%26rft.issue%253D10%26rft.spage%253DR106%26rft.epage%253DR106%26rft.atitle%253DDifferential%2Bexpression%2Banalysis%2Bfor%2Bsequence%2Bcount%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2010-11-10-r106%26rft_id%253Dinfo%253Apmid%252F20979621%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2010-11-10-r106&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20979621&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.3" data-doi="10.1101/gr.111310.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aylor</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Valdar</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Foulds-Mathes</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buus</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verdugo</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genetic analysis of complex traits in the emerging Collaborative Cross.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1213</span>–<span class="cit-lpage">1222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.111310.110%26rft_id%253Dinfo%253Apmid%252F21406540%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzgvMTIxMyI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOC8xNDI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Boehm</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guethlein</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Klamp</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ozbek</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaub</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">Two families of GTPases dominate the complex cellular response to IFN-γ.</span> <abbr class="cit-jnl-abbrev">J. Immunol.</abbr> <span class="cit-vol">161</span>: <span class="cit-fpage">6715</span>–<span class="cit-lpage">6723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BImmunology%26rft.stitle%253DJ.%2BImmunol.%26rft.aulast%253DBoehm%26rft.auinit1%253DU.%26rft.volume%253D161%26rft.issue%253D12%26rft.spage%253D6715%26rft.epage%253D6723%26rft.atitle%253DTwo%2BFamilies%2Bof%2BGTPases%2BDominate%2Bthe%2BComplex%2BCellular%2BResponse%2Bto%2BIFN-%257Bgamma%257D%26rft_id%253Dinfo%253Apmid%252F9862701%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTE6IjE2MS8xMi82NzE1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzE0MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-other" id="cit-4.8.1429.5"><div class="cit-metadata unstructured">Bottomly, D., M. T. Ferris, L. D. Aicher, E. Rosenzweig, A. Whitmore <em>et al.</em>, 2012 Expression quantitative trait loci for extreme host response to influenza A in pre-Collaborative Cross mice. G3 (Bethesda) 2: 213–221.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.6" data-doi="10.1126/science.1240925"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carpenter</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aiello</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Atianand</span>  <span class="cit-name-given-names">M. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ricci</span>  <span class="cit-name-given-names">E. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gandhi</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A long noncoding RNA mediates both activation and repression of immune response genes.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">341</span>: <span class="cit-fpage">789</span>–<span class="cit-lpage">792</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1240925%26rft_id%253Dinfo%253Apmid%252F23907535%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzNDEvNjE0Ny83ODkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzgvMTQyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-other" id="cit-4.8.1429.7"><div class="cit-metadata unstructured">Chang, S. T., P. Sova, X. Peng, J. Weiss, G. L. Law <em>et al.</em>, 2011 Next-generation sequencing reveals HIV-1-mediated suppression of T cell activation and RNA processing and regulation of noncoding RNA expression in a CD4+ T cell line. MBio 2: e00134–11.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.8" data-doi="10.1186/1471-2334-6-82"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chong</span>  <span class="cit-name-given-names">W. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ip</span>  <span class="cit-name-given-names">W. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tso</span>  <span class="cit-name-given-names">G. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">W. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">The interferon gamma gene polymorphism +874 A/T is associated with severe acute respiratory syndrome.</span> <abbr class="cit-jnl-abbrev">BMC Infect. Dis.</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Binfectious%2Bdiseases%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BInfect%2BDis%26rft.aulast%253DChong%26rft.auinit1%253DW.%2BP.%26rft.volume%253D6%26rft.spage%253D82%26rft.epage%253D82%26rft.atitle%253DThe%2Binterferon%2Bgamma%2Bgene%2Bpolymorphism%2B%252B874%2BA%252FT%2Bis%2Bassociated%2Bwith%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2334-6-82%26rft_id%253Dinfo%253Apmid%252F16672072%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2334-6-82&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16672072&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.9" data-doi="10.1126/science.343.6169.370-a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Misleading results: don’t blame the mice.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">343</span>: <span class="cit-fpage">370</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DChurchill%26rft.auinit1%253DG.%2BA.%26rft.volume%253D343%26rft.issue%253D6169%26rft.spage%253D370%26rft.epage%253D370%26rft.atitle%253DMisleading%2BResults%253A%2BDon%2527t%2BBlame%2Bthe%2BMice%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.343.6169.370-a%26rft_id%253Dinfo%253Apmid%252F24458625%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjE0OiIzNDMvNjE2OS8zNzAtYSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOC8xNDI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.10" data-doi="10.1534/genetics.111.132639"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Collaborative Cross Consortium</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The genome architecture of the collaborative cross mouse genetic reference population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">190</span>: <span class="cit-fpage">389</span>–<span class="cit-lpage">401</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DCollaborative%2BCross%2BConsortium%26rft.auinit1%253D%2B%26rft.volume%253D190%26rft.issue%253D2%26rft.spage%253D389%26rft.epage%253D401%26rft.atitle%253DThe%2BGenome%2BArchitecture%2Bof%2Bthe%2BCollaborative%2BCross%2BMouse%2BGenetic%2BReference%2BPopulation%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132639%26rft_id%253Dinfo%253Apmid%252F22345608%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTkwLzIvMzg5IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzE0MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-book" id="cit-4.8.1429.11"><div class="cit-metadata unstructured">Deng, N., C. G. Sanchez, J. A. Lasky, and D. Zhu, 2013 Detecting splicing variants in idiopathic pulmonary fibrosis from non-differentially expressed genes. PLoS One 8: e68352.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.12" data-doi="10.1101/gr.132159.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Derrien</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bussotti</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanzer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Djebali</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The GENCODE v7 catalog of human long noncoding RNAs: Analysis of their gene structure, evolution, and expression.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1775</span>–<span class="cit-lpage">1789</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DDerrien%26rft.auinit1%253DT.%26rft.volume%253D22%26rft.issue%253D9%26rft.spage%253D1775%26rft.epage%253D1789%26rft.atitle%253DThe%2BGENCODE%2Bv7%2Bcatalog%2Bof%2Bhuman%2Blong%2Bnoncoding%2BRNAs%253A%2BAnalysis%2Bof%2Btheir%2Bgene%2Bstructure%252C%2Bevolution%252C%2Band%2Bexpression%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.132159.111%26rft_id%253Dinfo%253Apmid%252F22955988%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIyLzkvMTc3NSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOC8xNDI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.13" data-doi="10.1038/nature11233"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Djebali</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merkel</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dobin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lassmann</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Landscape of transcription in human cells.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">489</span>: <span class="cit-fpage">101</span>–<span class="cit-lpage">108</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DDjebali%26rft.auinit1%253DS.%26rft.volume%253D489%26rft.issue%253D7414%26rft.spage%253D101%26rft.epage%253D108%26rft.atitle%253DLandscape%2Bof%2Btranscription%2Bin%2Bhuman%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11233%26rft_id%253Dinfo%253Apmid%252F22955620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11233&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22955620&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308347000043&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.14" data-doi="10.1101/gr.118786.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Durrant</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tayem</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yalcin</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cleak</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodstadt</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Collaborative Cross mice and their power to map host susceptibility to Aspergillus fumigatus infection.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1239</span>–<span class="cit-lpage">1248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.118786.110%26rft_id%253Dinfo%253Apmid%252F21493779%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzgvMTIzOSI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOC8xNDI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.15" data-doi="10.1128/JVI.00162-12"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Earl</span>  <span class="cit-name-given-names">P. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Americo</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moss</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Lethal monkeypox virus infection of CAST/EiJ mice is associated with a deficient gamma interferon response.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">9105</span>–<span class="cit-lpage">9112</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.00162-12%26rft_id%253Dinfo%253Apmid%252F22696658%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Ni8xNy85MTA1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzE0MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.16" data-doi="10.1038/nature11247"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">ENCODE Project Consortium</span></li></ol><cite>, B. E. Bernstein, E. Birney, I. Dunham, E. D. Green <em>et al</em>., <span class="cit-pub-date">2012</span> <span class="cit-article-title">An integrated encyclopedia of DNA elements in the human genome.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">489</span>: <span class="cit-fpage">57</span>–<span class="cit-lpage">74</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DBernstein%26rft.auinit1%253DB.%2BE.%26rft.volume%253D489%26rft.issue%253D7414%26rft.spage%253D57%26rft.epage%253D74%26rft.atitle%253DAn%2Bintegrated%2Bencyclopedia%2Bof%2BDNA%2Belements%2Bin%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11247%26rft_id%253Dinfo%253Apmid%252F22955616%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11247&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22955616&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000308347000039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.17" data-doi="10.1371/journal.ppat.1003196"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferris</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aylor</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bottomly</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Whitmore</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aicher</span>  <span class="cit-name-given-names">L. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Modeling host genetic regulation of influenza pathogenesis in the Collaborative Cross.</span> <abbr class="cit-jnl-abbrev">PLoS Pathog.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BPathog.%26rft.volume%253D9%26rft.spage%253De1003196%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.ppat.1003196%26rft_id%253Dinfo%253Apmid%252F23468633%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.ppat.1003196&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23468633&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.18" data-doi="10.1038/nrg3335"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flint</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eskin</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide association studies in mice.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">807</span>–<span class="cit-lpage">817</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DFlint%26rft.auinit1%253DJ.%26rft.volume%253D13%26rft.issue%253D11%26rft.spage%253D807%26rft.epage%253D817%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudies%2Bin%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3335%26rft_id%253Dinfo%253Apmid%252F23044826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3335&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23044826&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.19" data-doi="10.1038/nbt.1883"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Grabherr</span>  <span class="cit-name-given-names">M. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haas</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yassour</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levin</span>  <span class="cit-name-given-names">J. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thompson</span>  <span class="cit-name-given-names">D. A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">29</span>: <span class="cit-fpage">644</span>–<span class="cit-lpage">652</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DGrabherr%26rft.auinit1%253DM.%2BG.%26rft.volume%253D29%26rft.issue%253D7%26rft.spage%253D644%26rft.epage%253D652%26rft.atitle%253DFull-length%2Btranscriptome%2Bassembly%2Bfrom%2BRNA-Seq%2Bdata%2Bwithout%2Ba%2Breference%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1883%26rft_id%253Dinfo%253Apmid%252F21572440%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1883&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21572440&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-confproc" id="cit-4.8.1429.20"><div class="cit-metadata unstructured">Huang, S., C.-Y. Kao, L. McMillan, and W. Wang, 2013 Transforming genomes using MOD files with applications, pp. 595–604 in <em>Proceedings of the ACM Conference on Bioinformatics</em>, <em>Computational Biology and Biomedicine</em>, ACM Press, New York, NY.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.21" data-doi="10.1620/tjem.226.117"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Igusa</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakakibara</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shibahara</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sakamoto</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishimura</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Complicated secondary pneumonia after swine-origin influenza A virus infection in an immunocompetent patient.</span> <abbr class="cit-jnl-abbrev">Tohoku J. Exp. Med.</abbr> <span class="cit-vol">226</span>: <span class="cit-fpage">117</span>–<span class="cit-lpage">120</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BTohoku%2Bjournal%2Bof%2Bexperimental%2Bmedicine%26rft.stitle%253DTohoku%2BJ%2BExp%2BMed%26rft.aulast%253DIgusa%26rft.auinit1%253DR.%26rft.volume%253D226%26rft.issue%253D2%26rft.spage%253D117%26rft.epage%253D120%26rft.atitle%253DComplicated%2Bsecondary%2Bpneumonia%2Bafter%2Bswine-origin%2Binfluenza%2BA%2Bvirus%2Binfection%2Bin%2Ban%2Bimmunocompetent%2Bpatient.%26rft_id%253Dinfo%253Adoi%252F10.1620%252Ftjem.226.117%26rft_id%253Dinfo%253Apmid%252F22251837%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1620/tjem.226.117&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22251837&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.22"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Josset</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tchitchek</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gralinski</span>  <span class="cit-name-given-names">L. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferris</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eisfeld</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, 2014 Annotation of long non-coding RNAs expressed 1 in Collaborative 2 Cross founder mice in response to respiratory virus infection 3 reveals a new class of interferon-stimulated transcripts. RNA Biology. Jun 12:11(7).</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.23" data-doi="10.1038/nature10413"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keane</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goodstadt</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Danecek</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">White</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Mouse genomic variation and its effect on phenotypes and gene regulation.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">477</span>: <span class="cit-fpage">289</span>–<span class="cit-lpage">294</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKeane%26rft.auinit1%253DT.%2BM.%26rft.volume%253D477%26rft.issue%253D7364%26rft.spage%253D289%26rft.epage%253D294%26rft.atitle%253DMouse%2Bgenomic%2Bvariation%2Band%2Bits%2Beffect%2Bon%2Bphenotypes%2Band%2Bgene%2Bregulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10413%26rft_id%253Dinfo%253Apmid%252F21921910%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10413&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21921910&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000294852400022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-other" id="cit-4.8.1429.24"><div class="cit-metadata unstructured">Kelada, S. N. P., D. L. Aylor, B. C. E. Peck, J. F. Ryan, U. Tavarez <em>et al.</em>, 2012 Genetic analysis of hematological parameters in incipient lines of the collaborative cross. G3 (Bethesda) 2: 157–165.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.25" data-doi="10.3748/wjg.14.6072"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">C.-F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ling</span>  <span class="cit-name-given-names">Z.-Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">K.-R.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Distinct expression patterns in hepatitis B virus-and hepatitis C virus-infected hepatocellular carcinoma. World journal of gastroenterology</span>. <abbr class="cit-jnl-abbrev">WJG</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">6072</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWJG%26rft.volume%253D14%26rft.spage%253D6072%26rft_id%253Dinfo%253Adoi%252F10.3748%252Fwjg.14.6072%26rft_id%253Dinfo%253Apmid%252F18932288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3748/wjg.14.6072&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18932288&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.26" data-doi="10.1186/1471-2350-4-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tseng</span>  <span class="cit-name-given-names">H.-K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trejaut</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">H.-L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loo</span>  <span class="cit-name-given-names">J.-H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Association of HLA class I with severe acute respiratory syndrome coronavirus infection.</span> <abbr class="cit-jnl-abbrev">BMC Med. Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">9</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bmedical%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BMed%2BGenet%26rft.aulast%253DLin%26rft.auinit1%253DM.%26rft.volume%253D4%26rft.spage%253D9%26rft.epage%253D9%26rft.atitle%253DAssociation%2Bof%2BHLA%2Bclass%2BI%2Bwith%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome%2Bcoronavirus%2Binfection.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2350-4-9%26rft_id%253Dinfo%253Apmid%252F12969506%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2350-4-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12969506&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.27" data-doi="10.1002/wrna.1162"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lloyd</span>  <span class="cit-name-given-names">R. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Regulation of stress granules and P-bodies during RNA virus infection.</span> <abbr class="cit-jnl-abbrev">Wiley Interdisciplinary Reviews: RNA</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">317</span>–<span class="cit-lpage">331</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DWiley%2BInterdisciplinary%2BReviews%253A%2BRNA%26rft.volume%253D4%26rft.spage%253D317%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fwrna.1162%26rft_id%253Dinfo%253Apmid%252F23554219%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/wrna.1162&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23554219&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.28" data-doi="10.4049/jimmunol.0900657"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKinstry</span>  <span class="cit-name-given-names">K. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Strutt</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buck</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Curtis</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dibble</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">IL-10 deficiency unleashes an influenza-specific Th17 response and enhances survival against high-dose challenge.</span> <abbr class="cit-jnl-abbrev">J. Immunol.</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">7353</span>–<span class="cit-lpage">7363</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BJournal%2Bof%2BImmunology%26rft.stitle%253DJ.%2BImmunol.%26rft.aulast%253DMcKinstry%26rft.auinit1%253DK.%2BK.%26rft.volume%253D182%26rft.issue%253D12%26rft.spage%253D7353%26rft.epage%253D7363%26rft.atitle%253DIL-10%2BDeficiency%2BUnleashes%2Ban%2BInfluenza-Specific%2BTh17%2BResponse%2Band%2BEnhances%2BSurvival%2Bagainst%2BHigh-Dose%2BChallenge%26rft_id%253Dinfo%253Adoi%252F10.4049%252Fjimmunol.0900657%26rft_id%253Dinfo%253Apmid%252F19494257%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiamltbXVub2wiO3M6NToicmVzaWQiO3M6MTE6IjE4Mi8xMi83MzUzIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzE0MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.29" data-doi="10.1038/nature11632"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McVean</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Altshuler</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, R. M. Durbin, G. R. Abecasis, D. R. Bentley <em>et al</em>., <span class="cit-pub-date">2012</span> <span class="cit-article-title">An integrated map of genetic variation from 1,092 human genomes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">491</span>: <span class="cit-fpage">56</span>–<span class="cit-lpage">65</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DAbecasis%26rft.auinit1%253DG.%2BR.%26rft.volume%253D491%26rft.issue%253D7422%26rft.spage%253D56%26rft.epage%253D65%26rft.atitle%253DAn%2Bintegrated%2Bmap%2Bof%2Bgenetic%2Bvariation%2Bfrom%2B1%252C092%2Bhuman%2Bgenomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature11632%26rft_id%253Dinfo%253Apmid%252F23128226%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature11632&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23128226&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000310434500030&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.30" data-doi="10.1038/nature08903"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Montgomery</span>  <span class="cit-name-given-names">S. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sammeth</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gutierrez-Arcelus</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lach</span>  <span class="cit-name-given-names">R. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ingle</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Transcriptome genetics using second generation sequencing in a Caucasian population.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">464</span>: <span class="cit-fpage">773</span>–<span class="cit-lpage">777</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMontgomery%26rft.auinit1%253DS.%2BB.%26rft.volume%253D464%26rft.issue%253D7289%26rft.spage%253D773%26rft.epage%253D777%26rft.atitle%253DTranscriptome%2Bgenetics%2Busing%2Bsecond%2Bgeneration%2Bsequencing%2Bin%2Ba%2BCaucasian%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08903%26rft_id%253Dinfo%253Apmid%252F20220756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08903&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20220756&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276205000048&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.31" data-doi="10.1186/gb-2012-13-8-418"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth cit-collab">Mouse ENCODE Consortium</span></li></ol><cite>, J. A. Stamatoyannopoulos, M. Snyder, R. Hardison, B. Ren <em>et al</em>., <span class="cit-pub-date">2012</span> <span class="cit-article-title">An encyclopedia of mouse DNA elements (Mouse ENCODE).</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">418</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DStamatoyannopoulos%26rft.auinit1%253DJ.%2BA.%26rft.volume%253D13%26rft.issue%253D8%26rft.spage%253D418%26rft.epage%253D418%26rft.atitle%253DAn%2Bencyclopedia%2Bof%2Bmouse%2BDNA%2Belements%2B%2528Mouse%2BENCODE%2529.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2012-13-8-418%26rft_id%253Dinfo%253Apmid%252F22889292%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2012-13-8-418&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22889292&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.32" data-doi="10.1126/science.1069193"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mural</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">H. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miklos</span>  <span class="cit-name-given-names">G. L. G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">A comparison of whole-genome shotgun-derived mouse chromosome 16 and the human genome.</span> <abbr class="cit-jnl-abbrev">Science (New York, N.Y.)</abbr> <span class="cit-vol">296</span>: <span class="cit-fpage">1661</span>–<span class="cit-lpage">1671</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%2B%2528New%2BYork%252C%2BN.Y.%2529%26rft.volume%253D296%26rft.spage%253D1661%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1069193%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1126/science.1069193&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.33" data-doi="10.1542/peds.113.1.e7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lam</span>  <span class="cit-name-given-names">C. W. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wong</span>  <span class="cit-name-given-names">C. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheng</span>  <span class="cit-name-given-names">F. W. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Inflammatory Cytokine Profile in Children With Severe Acute Respiratory Syndrome.</span> <abbr class="cit-jnl-abbrev">Pediatrics</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">e7</span>–<span class="cit-lpage">e14</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPediatrics%26rft.stitle%253DPediatrics%26rft.aulast%253DNg%26rft.auinit1%253DP.%2BC.%26rft.volume%253D113%26rft.issue%253D1%26rft.spage%253De7%26rft.epage%253De14%26rft.atitle%253DInflammatory%2BCytokine%2BProfile%2Bin%2BChildren%2BWith%2BSevere%2BAcute%2BRespiratory%2BSyndrome%26rft_id%253Dinfo%253Adoi%252F10.1542%252Fpeds.113.1.e7%26rft_id%253Dinfo%253Apmid%252F14702488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTA6InBlZGlhdHJpY3MiO3M6NToicmVzaWQiO3M6ODoiMTEzLzEvZTciO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzgvMTQyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.34" data-doi="10.1016/j.ejim.2011.08.014"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pabst</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuehn</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schuler-Luettmann</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wiebe</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lebiedz</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Acute Respiratory Distress Syndrome as a presenting manifestation in young patients infected with H1N1 influenza virus.</span> <abbr class="cit-jnl-abbrev">Eur. J. Intern. Med.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">e119</span>–<span class="cit-lpage">e124</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DEur%2BJ%2BIntern%2BMed%26rft.aulast%253DPabst%26rft.auinit1%253DD.%26rft.volume%253D22%26rft.issue%253D6%26rft.spage%253De119%26rft.epage%253De124%26rft.atitle%253DAcute%2BRespiratory%2BDistress%2BSyndrome%2Bas%2Ba%2Bpresenting%2Bmanifestation%2Bin%2Byoung%2Bpatients%2Binfected%2Bwith%2BH1N1%2Binfluenza%2Bvirus.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ejim.2011.08.014%26rft_id%253Dinfo%253Apmid%252F22075296%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ejim.2011.08.014&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22075296&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.35" data-doi="10.1101/gr.120535.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pal</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gupta</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wickramasinghe</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baubet</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Alternative transcription exceeds alternative splicing in generating the transcriptome diversity of cerebellar development.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1260</span>–<span class="cit-lpage">1272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.120535.111%26rft_id%253Dinfo%253Apmid%252F21712398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIxLzgvMTI2MCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOC8xNDI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.36" data-doi="10.1128/mBio.00206-10"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gralinski</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Armour</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ferris</span>  <span class="cit-name-given-names">M. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thomas</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Unique signatures of long noncoding RNA expression in response to virus infection and altered innate immune signaling.</span> <abbr class="cit-jnl-abbrev">MBio</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">e00206</span>–<span class="cit-lpage">e00210</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMBio%26rft.volume%253D1%26rft.spage%253De00206%26rft_id%253Dinfo%253Adoi%252F10.1128%252FmBio.00206-10%26rft_id%253Dinfo%253Apmid%252F20978541%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1128/mBio.00206-10&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20978541&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.37"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Phillippi</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xie</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">D. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Using the emerging Collaborative Cross to probe the immune system.</span> <abbr class="cit-jnl-abbrev">Genes Immun. 15: 38–46.</abbr></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.38" data-doi="10.1038/nature08872"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pickrell</span>  <span class="cit-name-given-names">J. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marioni</span>  <span class="cit-name-given-names">J. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pai</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Degner</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Engelhardt</span>  <span class="cit-name-given-names">B. E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Understanding mechanisms underlying human gene expression variation with RNA sequencing.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">464</span>: <span class="cit-fpage">768</span>–<span class="cit-lpage">772</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DPickrell%26rft.auinit1%253DJ.%2BK.%26rft.volume%253D464%26rft.issue%253D7289%26rft.spage%253D768%26rft.epage%253D772%26rft.atitle%253DUnderstanding%2Bmechanisms%2Bunderlying%2Bhuman%2Bgene%2Bexpression%2Bvariation%2Bwith%2BRNA%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature08872%26rft_id%253Dinfo%253Apmid%252F20220758%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature08872&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20220758&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000276205000047&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.39" data-doi="10.1371/journal.pone.0041169"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pommerenke</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilk</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Srivastava</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulze</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Novoselova</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Global transcriptome analysis in influenza-infected mouse lungs reveals the kinetics of innate and adaptive host immune Responses.</span> <abbr class="cit-jnl-abbrev">PLoS ONE</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e41169</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DPommerenke%26rft.auinit1%253DC.%26rft.volume%253D7%26rft.issue%253D7%26rft.spage%253De41169%26rft.epage%253De41169%26rft.atitle%253DGlobal%2Btranscriptome%2Banalysis%2Bin%2Binfluenza-infected%2Bmouse%2Blungs%2Breveals%2Bthe%2Bkinetics%2Bof%2Binnate%2Band%2Badaptive%2Bhost%2Bimmune%2Bresponses.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0041169%26rft_id%253Dinfo%253Apmid%252F22815957%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0041169&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22815957&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.40"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rapicavoli</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Qu</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mikhail</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Laberge</span>  <span class="cit-name-given-names">R.-M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A mammalian pseudogene lncRNA at the interface of inflammation and anti-inflammatory therapeutics.</span> <abbr class="cit-jnl-abbrev">eLife</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e00762</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DeLife%26rft.volume%253D2%26rft.spage%253D00762e%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robinson</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCarthy</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smyth</span>  <span class="cit-name-given-names">G. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">edgeR: a Bioconductor package for differential expression analysis of digital gene expression data.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">139</span>–<span class="cit-lpage">140</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft.volume%253D26%26rft.spage%253D139%26rft_id%253Dinfo%253Apmid%252F19910308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=19910308&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000273116100025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.42" data-doi="10.1016/j.virol.2004.08.011"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sainz</span>  <span class="cit-name-given-names">B.</span>  <span class="cit-name-suffix">Jr</span>.</span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mossel</span>  <span class="cit-name-given-names">E. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peters</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garry</span>  <span class="cit-name-given-names">R. F.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Interferon-beta and interferon-gamma synergistically inhibit the replication of severe acute respiratory syndrome-associated coronavirus (SARS-CoV).</span> <abbr class="cit-jnl-abbrev">Virology</abbr> <span class="cit-vol">329</span>: <span class="cit-fpage">11</span>–<span class="cit-lpage">17</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DVirology%26rft.stitle%253DVirology%26rft.aulast%253DSainz%26rft.auinit1%253DB.%26rft.volume%253D329%26rft.issue%253D1%26rft.spage%253D11%26rft.epage%253D17%26rft.atitle%253DInterferon-beta%2Band%2Binterferon-gamma%2Bsynergistically%2Binhibit%2Bthe%2Breplication%2Bof%2Bsevere%2Bacute%2Brespiratory%2Bsyndrome-associated%2Bcoronavirus%2B%2528SARS-CoV%2529.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.virol.2004.08.011%26rft_id%253Dinfo%253Apmid%252F15476870%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.virol.2004.08.011&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15476870&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.43" data-doi="10.1098/rstb.2001.0979"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schickli</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flandorfer</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nakaya</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinez-Sobrido</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garcia-Sastre</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Plasmid-only rescue of influenza A virus vaccine candidates.</span> <abbr class="cit-jnl-abbrev">Philos. Trans. R. Soc. Lond. B Biol. Sci.</abbr> <span class="cit-vol">356</span>: <span class="cit-fpage">1965</span>–<span class="cit-lpage">1973</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPhilosophical%2BTransactions%2Bof%2Bthe%2BRoyal%2BSociety%2BB%253A%2BBiological%2BSciences%26rft.stitle%253DPhil%2BTrans%2BR%2BSoc%2BB%26rft.volume%253D356%26rft.issue%253D1416%26rft.spage%253D1965%26rft.epage%253D1973%26rft.atitle%253DPlasmid-only%2Brescue%2Bof%2Binfluenza%2BA%2Bvirus%2Bvaccine%2Bcandidates%26rft_id%253Dinfo%253Adoi%252F10.1098%252Frstb.2001.0979%26rft_id%253Dinfo%253Apmid%252F11779399%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Njoicm95cHRiIjtzOjU6InJlc2lkIjtzOjEzOiIzNTYvMTQxNi8xOTY1IjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzE0MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-book" id="cit-4.8.1429.44"><div class="cit-metadata unstructured">Sessions, O. M., Y. Tan, K. C. Goh, Y. Liu, P. Tan <em>et al.</em>, 2013 Host cell transcriptome profile during wild-type and attenuated dengue virus infection. PLoS Neglect. Trop. Dis. 7: e2107.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.45" data-doi="10.1128/MCB.8.10.4518"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Staeheli</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grob</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Meier</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sutcliffe</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haller</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">1988</span> <span class="cit-article-title">Influenza virus-susceptible mice carry Mx genes with a large deletion or a nonsense mutation.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">4518</span>–<span class="cit-lpage">4523</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DStaeheli%26rft.auinit1%253DP%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253D4518%26rft.epage%253D4523%26rft.atitle%253DInfluenza%2Bvirus-susceptible%2Bmice%2Bcarry%2BMx%2Bgenes%2Bwith%2Ba%2Blarge%2Bdeletion%2Bor%2Ba%2Bnonsense%2Bmutation.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.8.10.4518%26rft_id%253Dinfo%253Apmid%252F2903437%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjgvMTAvNDUxOCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOC8xNDI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.46" data-doi="10.1128/JVI.07143-11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Terrier</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Marcel</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cartet</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lane</span>  <span class="cit-name-given-names">D. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lina</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Influenza A viruses control expression of proviral human p53 isoforms p53β and Δ133p53α.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">86</span>: <span class="cit-fpage">8452</span>–<span class="cit-lpage">8460</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.07143-11%26rft_id%253Dinfo%253Apmid%252F22647703%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjEwOiI4Ni8xNi84NDUyIjtzOjQ6ImF0b20iO3M6MTg6Ii9nZ2cvNC84LzE0MjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.47" data-doi="10.1038/nprot.2012.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roberts</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goff</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">562</span>–<span class="cit-lpage">578</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DTrapnell%26rft.auinit1%253DC.%26rft.volume%253D7%26rft.issue%253D3%26rft.spage%253D562%26rft.epage%253D578%26rft.atitle%253DDifferential%2Bgene%2Band%2Btranscript%2Bexpression%2Banalysis%2Bof%2BRNA-seq%2Bexperiments%2Bwith%2BTopHat%2Band%2BCufflinks.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2012.016%26rft_id%253Dinfo%253Apmid%252F22383036%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2012.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22383036&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.48" data-doi="10.1038/nbt.1621"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">B. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pertea</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mortazavi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kwan</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">511</span>–<span class="cit-lpage">515</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DTrapnell%26rft.auinit1%253DC.%26rft.volume%253D28%26rft.issue%253D5%26rft.spage%253D511%26rft.epage%253D515%26rft.atitle%253DTranscript%2Bassembly%2Band%2Bquantification%2Bby%2BRNA-Seq%2Breveals%2Bunannotated%2Btranscripts%2Band%2Bisoform%2Bswitching%2Bduring%2Bcell%2Bdifferentiation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt.1621%26rft_id%253Dinfo%253Apmid%252F20436464%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt.1621&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20436464&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000277452700032&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.49" data-doi="10.1128/JVI.01116-07"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tumpey</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Szretter</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hoeven</span>  <span class="cit-name-given-names">N. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Katz</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kochs</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The Mx1 gene protects mice against the pandemic 1918 and highly lethal human H5N1 influenza viruses.</span> <abbr class="cit-jnl-abbrev">J. Virol.</abbr> <span class="cit-vol">81</span>: <span class="cit-fpage">10818</span>–<span class="cit-lpage">10821</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BVirol.%26rft_id%253Dinfo%253Adoi%252F10.1128%252FJVI.01116-07%26rft_id%253Dinfo%253Apmid%252F17652381%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoianZpIjtzOjU6InJlc2lkIjtzOjExOiI4MS8xOS8xMDgxOCI7czo0OiJhdG9tIjtzOjE4OiIvZ2dnLzQvOC8xNDI5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.50" data-doi="10.1038/nature07509"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">E. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandberg</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Khrebtukova</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Alternative isoform regulation in human tissue transcriptomes.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">456</span>: <span class="cit-fpage">470</span>–<span class="cit-lpage">476</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DWang%26rft.auinit1%253DE.%2BT.%26rft.volume%253D456%26rft.issue%253D7221%26rft.spage%253D470%26rft.epage%253D476%26rft.atitle%253DAlternative%2Bisoform%2Bregulation%2Bin%2Bhuman%2Btissue%2Btranscriptomes.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature07509%26rft_id%253Dinfo%253Apmid%252F18978772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature07509&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18978772&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261170500031&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-confproc" id="cit-4.8.1429.51"><div class="cit-metadata unstructured">Wang, Z., A. Zhang, Y. Wan, X. Liu, C. Qiu <em>et al.</em>, 2013 Early hypercytokinemia is associated with interferon-induced transmembrane protein-3 dysfunction and predictive of fatal H7N9 infection. Proc. Nat. Acad. Sci. U S A.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-other" id="cit-4.8.1429.52"><div class="cit-metadata unstructured">Weiss, I. D., O. Wald, H. Wald, K. Beider, M. Abraham <em>et al.</em>, 2010 IFN-gamma treatment at early stages of influenza virus infection protects mice from death in a NK cell-dependent manner. J. Interferon Cytokine Res. 30: 439–449.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.53" data-doi="10.1093/bioinformatics/bti310"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">T. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Watanabe</span>  <span class="cit-name-given-names">C. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">GMAP: a genomic mapping and alignment program for mRNA and EST sequences.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">1859</span>–<span class="cit-lpage">1875</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbti310%26rft_id%253Dinfo%253Apmid%252F15728110%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo5OiIyMS85LzE4NTkiO3M6NDoiYXRvbSI7czoxODoiL2dnZy80LzgvMTQyOS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.54" data-doi="10.1038/ng2087"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Yang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bell</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Churchill</span>  <span class="cit-name-given-names">G. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pardo-Manuel de Villena</span>  <span class="cit-name-given-names">F.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">On the subspecific origin of the laboratory mouse.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">1100</span>–<span class="cit-lpage">1107</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DYang%26rft.auinit1%253DH.%26rft.volume%253D39%26rft.issue%253D9%26rft.spage%253D1100%26rft.epage%253D1107%26rft.atitle%253DOn%2Bthe%2Bsubspecific%2Borigin%2Bof%2Bthe%2Blaboratory%2Bmouse.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2087%26rft_id%253Dinfo%253Apmid%252F17660819%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2087&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17660819&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000249122400019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.55" data-doi="10.1016/j.chom.2009.07.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zaas</span>  <span class="cit-name-given-names">A. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Varkey</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Veldman</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hero</span>  <span class="cit-name-given-names">A. O.</span>  <span class="cit-name-suffix">III</span>.</span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Gene expression signatures diagnose influenza and other symptomatic respiratory viral infections in humans.</span> <abbr class="cit-jnl-abbrev">Cell Host Microbe</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">207</span>–<span class="cit-lpage">217</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DCell%2BHost%2BMicrobe%26rft.aulast%253DZaas%26rft.auinit1%253DA.%2BK.%26rft.volume%253D6%26rft.issue%253D3%26rft.spage%253D207%26rft.epage%253D217%26rft.atitle%253DGene%2Bexpression%2Bsignatures%2Bdiagnose%2Binfluenza%2Band%2Bother%2Bsymptomatic%2Brespiratory%2Bviral%2Binfections%2Bin%2Bhumans.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.chom.2009.07.006%26rft_id%253Dinfo%253Apmid%252F19664979%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.chom.2009.07.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19664979&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000270290700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.56" data-doi="10.1186/gm437"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pal</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tchou</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davuluri</span>  <span class="cit-name-given-names">R. V.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Isoform level expression profiles provide better cancer signatures than gene level expression profiles.</span> <abbr class="cit-jnl-abbrev">Genome Medicine</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">33</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BMedicine%26rft.volume%253D5%26rft.spage%253D33%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgm437%26rft_id%253Dinfo%253Apmid%252F23594586%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gm437&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23594586&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-4.8.1429.57" data-doi="10.1016/j.tig.2011.10.006"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zheng</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dill</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peltz</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A better prognosis for genetic association studies in mice.</span> <abbr class="cit-jnl-abbrev">Trends Genet.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">62</span>–<span class="cit-lpage">69</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTrends%2Bin%2Bgenetics%2B%253A%2B%2BTIG%26rft.stitle%253DTrends%2BGenet%26rft.aulast%253DZheng%26rft.auinit1%253DM.%26rft.volume%253D28%26rft.issue%253D2%26rft.spage%253D62%26rft.epage%253D69%26rft.atitle%253DA%2Bbetter%2Bprognosis%2Bfor%2Bgenetic%2Bassociation%2Bstudies%2Bin%2Bmice.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.tig.2011.10.006%26rft_id%253Dinfo%253Apmid%252F22118772%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.tig.2011.10.006&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22118772&amp;link_type=MED&amp;atom=%2Fggg%2F4%2F8%2F1429.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000300532200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/4/8/1429.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/4/8/1417" title="Genetic Interactions Between P Elements Involved in piRNA-Mediated Repression of Hybrid Dysgenesis in Drosophila melanogaster" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/4/8/1445" title="BZcon1, a SANT/Myb-Type Gene Involved in the Conidiation of Cochliobolus carbonum" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/4/8"><span>Volume 4 Issue 8, August 2014</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/4/8.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/4/8.cover-source.jpg?itok=uF7Z9Bah" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 4 (8)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/4/8/1429" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-TFNu64562ZGH1XzznnOjHYAAfZPWt-PcPe_1dIVQJmI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/mouse-genetic-resources" class="" data-icon-position="" data-hide-link-title="0">Mouse Genetic Resources</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/4/8/1429" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/4/8/1429" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/4/8/1429" class="active" data-icon-position="" data-hide-link-title="0">Genomic Profiling of Collaborative Cross Founder Mice Infected with Respiratory Viruses Reveals Novel Transcripts and Infection-Related Strain-Specific Gene and Isoform Expression</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/459943" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-K3pWVJGm0ffxa-3Ss0QWguyRgrMkZ6vpXPv3pVEkSKY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/8/1429.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/4/8/1429" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-byHCHieK4OQA2RMcz4rAIgE8B4hEIgl6YaB7PtCc0Pc" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/4/8/1429" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/4/8/1429.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459943" id="node459943--2" data-pisa="ggg;4/8/1429" data-pisa-master="ggg;g3.114.011759" data-apath="/ggg/4/8/1429.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Profiling of Collaborative Cross Founder Mice Infected with Respiratory Viruses Reveals Novel Transcripts and Infection-Related Strain-Specific Gene and Isoform Expression</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hao Xiong</span>, <span class="highwire-citation-author" data-delta="1">Juliet Morrison</span>, <span class="highwire-citation-author" data-delta="2">Martin T. Ferris</span>, <span class="highwire-citation-author" data-delta="3">Lisa E. Gralinski</span>, <span class="highwire-citation-author" data-delta="4">Alan C. Whitmore</span>, <span class="highwire-citation-author" data-delta="5">Richard Green</span>, <span class="highwire-citation-author" data-delta="6">Matthew J. Thomas</span>, <span class="highwire-citation-author" data-delta="7">Jennifer Tisoncik-Go</span>, <span class="highwire-citation-author" data-delta="8">Gary P. Schroth</span>, <span class="highwire-citation-author" data-delta="9">Fernando Pardo-Manuel de Villena</span>, <span class="highwire-citation-author" data-delta="10">Ralph S. Baric</span>, <span class="highwire-citation-author" data-delta="11">Mark T. Heise</span>, <span class="highwire-citation-author" data-delta="12">Xinxia Peng</span> and <span class="highwire-citation-author" data-delta="13">Michael G. Katze</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1429-1444;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.011759 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459943--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hao Xiong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, School of Medicine, University of Washington, Seattle, Washington</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hao%2BXiong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xiong%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHao%2BXiong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Juliet Morrison </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, School of Medicine, University of Washington, Seattle, Washington</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juliet%2BMorrison%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morrison%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuliet%2BMorrison%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Martin T. Ferris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Genetics, University of North Carolina-Chapel Hill, Chapel Hill, North Carolina</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BT.%2BFerris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferris%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BT.%2BFerris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Lisa E. Gralinski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Epidemiology, University of North Carolina-Chapel Hill, Chapel Hill, North Carolina</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lisa%2BE.%2BGralinski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gralinski%20LE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALisa%2BE.%2BGralinski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Alan C. Whitmore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Genetics, University of North Carolina-Chapel Hill, Chapel Hill, North Carolina</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BC.%2BWhitmore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Whitmore%20AC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlan%2BC.%2BWhitmore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Richard Green </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, School of Medicine, University of Washington, Seattle, Washington</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BGreen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Green%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARichard%2BGreen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Matthew J. Thomas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, School of Medicine, University of Washington, Seattle, Washington</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BJ.%2BThomas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thomas%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BJ.%2BThomas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jennifer Tisoncik-Go </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, School of Medicine, University of Washington, Seattle, Washington</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BTisoncik-Go%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tisoncik-Go%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BTisoncik-Go%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Gary P. Schroth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Illumina, Inc., San Diego, California</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BP.%2BSchroth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schroth%20GP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGary%2BP.%2BSchroth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Fernando Pardo-Manuel de Villena </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Genetics, University of North Carolina-Chapel Hill, Chapel Hill, North Carolina</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BPardo-Manuel%2Bde%2BVillena%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pardo-Manuel%20de%20Villena%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFernando%2BPardo-Manuel%2Bde%2BVillena%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Ralph S. Baric </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Epidemiology, University of North Carolina-Chapel Hill, Chapel Hill, North Carolina</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ralph%2BS.%2BBaric%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baric%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARalph%2BS.%2BBaric%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Mark T. Heise </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Genetics, University of North Carolina-Chapel Hill, Chapel Hill, North Carolina</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BT.%2BHeise%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Heise%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BT.%2BHeise%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Xinxia Peng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, School of Medicine, University of Washington, Seattle, Washington</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xinxia%2BPeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Peng%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXinxia%2BPeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Michael G. Katze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, School of Medicine, University of Washington, Seattle, Washington</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BG.%2BKatze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Katze%20MG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BG.%2BKatze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:honey@uw.edu" class="" data-icon-position="" data-hide-link-title="0">honey@uw.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/4/8/1429" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/4/8/1429&amp;title=Genomic%20Profiling%20of%20Collaborative%20Cross%20Founder%20Mice%20Infected%20with%20Respiratory%20Viruses%20Reveals%20Novel%20Transcripts%20and%20Infection-Related%20Strain-Specific%20Gene%20and%20Isoform%20Expression" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/4/8/1429&amp;title=Genomic%20Profiling%20of%20Collaborative%20Cross%20Founder%20Mice%20Infected%20with%20Respiratory%20Viruses%20Reveals%20Novel%20Transcripts%20and%20Infection-Related%20Strain-Specific%20Gene%20and%20Isoform%20Expression" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/4/8/1429&amp;title=Genomic%20Profiling%20of%20Collaborative%20Cross%20Founder%20Mice%20Infected%20with%20Respiratory%20Viruses%20Reveals%20Novel%20Transcripts%20and%20Infection-Related%20Strain-Specific%20Gene%20and%20Isoform%20Expression" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/4/8/1429&amp;text=Genomic%20Profiling%20of%20Collaborative%20Cross%20Founder%20Mice%20Infected%20with%20Respiratory%20Viruses%20Reveals%20Novel%20Transcripts%20and%20Infection-Related%20Strain-Specific%20Gene%20and%20Isoform%20Expression" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/4/8/1429&amp;title=Genomic%20Profiling%20of%20Collaborative%20Cross%20Founder%20Mice%20Infected%20with%20Respiratory%20Viruses%20Reveals%20Novel%20Transcripts%20and%20Infection-Related%20Strain-Specific%20Gene%20and%20Isoform%20Expression" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/4/8/1429&amp;t=Genomic%20Profiling%20of%20Collaborative%20Cross%20Founder%20Mice%20Infected%20with%20Respiratory%20Viruses%20Reveals%20Novel%20Transcripts%20and%20Infection-Related%20Strain-Specific%20Gene%20and%20Isoform%20Expression" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/4/8/1429&amp;title=Genomic%20Profiling%20of%20Collaborative%20Cross%20Founder%20Mice%20Infected%20with%20Respiratory%20Viruses%20Reveals%20Novel%20Transcripts%20and%20Infection-Related%20Strain-Specific%20Gene%20and%20Isoform%20Expression" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/4/8/1429&amp;title=Genomic%20Profiling%20of%20Collaborative%20Cross%20Founder%20Mice%20Infected%20with%20Respiratory%20Viruses%20Reveals%20Novel%20Transcripts%20and%20Infection-Related%20Strain-Specific%20Gene%20and%20Isoform%20Expression" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/459943/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459943" id="node459943--3" data-pisa="ggg;4/8/1429" data-pisa-master="ggg;g3.114.011759" data-apath="/ggg/4/8/1429.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genomic Profiling of Collaborative Cross Founder Mice Infected with Respiratory Viruses Reveals Novel Transcripts and Infection-Related Strain-Specific Gene and Isoform Expression</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Hao Xiong</span>, <span class="highwire-citation-author" data-delta="1">Juliet Morrison</span>, <span class="highwire-citation-author" data-delta="2">Martin T. Ferris</span>, <span class="highwire-citation-author" data-delta="3">Lisa E. Gralinski</span>, <span class="highwire-citation-author" data-delta="4">Alan C. Whitmore</span>, <span class="highwire-citation-author" data-delta="5">Richard Green</span>, <span class="highwire-citation-author" data-delta="6">Matthew J. Thomas</span>, <span class="highwire-citation-author" data-delta="7">Jennifer Tisoncik-Go</span>, <span class="highwire-citation-author" data-delta="8">Gary P. Schroth</span>, <span class="highwire-citation-author" data-delta="9">Fernando Pardo-Manuel de Villena</span>, <span class="highwire-citation-author" data-delta="10">Ralph S. Baric</span>, <span class="highwire-citation-author" data-delta="11">Mark T. Heise</span>, <span class="highwire-citation-author" data-delta="12">Xinxia Peng</span> and <span class="highwire-citation-author" data-delta="13">Michael G. Katze</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">August 1, 2014 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 4 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 8 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">1429-1444;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.114.011759 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node459943--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Hao Xiong </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, School of Medicine, University of Washington, Seattle, Washington</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Hao%2BXiong%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Xiong%20H&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AHao%2BXiong%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Juliet Morrison </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, School of Medicine, University of Washington, Seattle, Washington</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Juliet%2BMorrison%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Morrison%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJuliet%2BMorrison%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Martin T. Ferris </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Genetics, University of North Carolina-Chapel Hill, Chapel Hill, North Carolina</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Martin%2BT.%2BFerris%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ferris%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMartin%2BT.%2BFerris%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Lisa E. Gralinski </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Epidemiology, University of North Carolina-Chapel Hill, Chapel Hill, North Carolina</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lisa%2BE.%2BGralinski%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Gralinski%20LE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALisa%2BE.%2BGralinski%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Alan C. Whitmore </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Genetics, University of North Carolina-Chapel Hill, Chapel Hill, North Carolina</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Alan%2BC.%2BWhitmore%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Whitmore%20AC&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AAlan%2BC.%2BWhitmore%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Richard Green </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, School of Medicine, University of Washington, Seattle, Washington</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Richard%2BGreen%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Green%20R&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARichard%2BGreen%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Matthew J. Thomas </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, School of Medicine, University of Washington, Seattle, Washington</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Matthew%2BJ.%2BThomas%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Thomas%20MJ&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMatthew%2BJ.%2BThomas%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Jennifer Tisoncik-Go </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, School of Medicine, University of Washington, Seattle, Washington</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jennifer%2BTisoncik-Go%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Tisoncik-Go%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJennifer%2BTisoncik-Go%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Gary P. Schroth </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>Illumina, Inc., San Diego, California</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gary%2BP.%2BSchroth%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Schroth%20GP&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGary%2BP.%2BSchroth%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Fernando Pardo-Manuel de Villena </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Genetics, University of North Carolina-Chapel Hill, Chapel Hill, North Carolina</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Fernando%2BPardo-Manuel%2Bde%2BVillena%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Pardo-Manuel%20de%20Villena%20F&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AFernando%2BPardo-Manuel%2Bde%2BVillena%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Ralph S. Baric </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Department of Epidemiology, University of North Carolina-Chapel Hill, Chapel Hill, North Carolina</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Ralph%2BS.%2BBaric%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baric%20RS&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ARalph%2BS.%2BBaric%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-11"><div class="author-tooltip-name">Mark T. Heise </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div><div class='author-affiliation'><span class='nlm-sup'>§</span>Department of Genetics, University of North Carolina-Chapel Hill, Chapel Hill, North Carolina</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Mark%2BT.%2BHeise%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Heise%20MT&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AMark%2BT.%2BHeise%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-12"><div class="author-tooltip-name">Xinxia Peng </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, School of Medicine, University of Washington, Seattle, Washington</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Xinxia%2BPeng%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Peng%20X&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AXinxia%2BPeng%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-13"><div class="author-tooltip-name">Michael G. Katze </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Microbiology, School of Medicine, University of Washington, Seattle, Washington</div><div class='author-affiliation'><span class='nlm-sup'>†</span>Pacific Northwest Regional Center of Excellence for Biodefense and Emerging Infectious Diseases Research, Portland, Oregon</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BG.%2BKatze%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Katze%20MG&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BG.%2BKatze%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:honey@uw.edu" class="" data-icon-position="" data-hide-link-title="0">honey@uw.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=1429&amp;rft.epage=1444&amp;rft.atitle=Genomic%20Profiling%20of%20Collaborative%20Cross%20Founder%20Mice%20Infected%20with%20Respiratory%20Viruses%20Reveals%20Novel%20Transcripts%20and%20Infection-Related%20Strain-Specific%20Gene%20and%20Isoform%20Expression&amp;rft.volume=4&amp;rft.issue=8&amp;rft.date=2014-08-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Xiong%2C+Hao&amp;rft.au=Morrison%2C+Juliet&amp;rft.au=Ferris%2C+Martin+T.&amp;rft.au=Gralinski%2C+Lisa+E.&amp;rft.au=Whitmore%2C+Alan+C.&amp;rft.au=Green%2C+Richard&amp;rft.au=Thomas%2C+Matthew+J.&amp;rft.au=Tisoncik-Go%2C+Jennifer&amp;rft.au=Schroth%2C+Gary+P.&amp;rft.au=Pardo-Manuel+de+Villena%2C+Fernando&amp;rft.au=Baric%2C+Ralph+S.&amp;rft.au=Heise%2C+Mark+T.&amp;rft.au=Peng%2C+Xinxia&amp;rft.au=Katze%2C+Michael+G."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/459943/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/459943/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/459943/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/459943/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/459943/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/459943/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/459943/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/459943/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/459943/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/459943/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/459943/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/459943/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459232" id="node459232" data-pisa="ggg;2/2/199" data-pisa-master="ggg;2/2/199" data-apath="/ggg/2/2/199.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/2/2/199" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Haplotype Probabilities in Advanced Intercross Populations</span></a></div>
</cite>
</div></li><li class="last even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="459235" id="node459235" data-pisa="ggg;2/2/191" data-pisa-master="ggg;2/2/191" data-apath="/ggg/2/2/191.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/2/2/191" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Accelerating the Inbreeding of Multi-Parental Recombinant Inbred Lines Generated By Sibling Matings</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Mouse%20Genetic%20Resources" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Mouse Genetic Resources</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/4/8/1429" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-7" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-15" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/4/8/1429.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/4/8/1429.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/4/8/1429.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/4/8/1429.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
