<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/10/3207" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/10/3207.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/10/3207.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/10/3207.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10.cover-source.jpg" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/10/3207.atom" />
<meta name="HW.pisa" content="ggg;6/10/3207" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Whole-Genome Sequencing and iPLEX MassARRAY Genotyping Map an EMS-Induced Mutation Affecting Cell Competition in Drosophila melanogaster" />
<meta name="DC.Identifier" content="10.1534/g3.116.029421" />
<meta name="DC.Date" content="2016-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Lee et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Cell competition, the conditional loss of viable genotypes only when surrounded by other cells, is a phenomenon observed in certain genetic mosaic conditions. We conducted a chemical mutagenesis and screen to recover new mutations that affect cell competition between wild-type and RpS3 heterozygous cells. Mutations were identified by whole-genome sequencing, making use of software tools that greatly facilitate the distinction between newly induced mutations and other sources of apparent sequence polymorphism, thereby reducing false-positive and false-negative identification rates. In addition, we utilized iPLEX MassARRAY for genotyping recombinant chromosomes. These approaches permitted the mapping of a new mutation affecting cell competition when only a single allele existed, with a phenotype assessed only in genetic mosaics, without the benefit of complementation with existing mutations, deletions, or duplications. These techniques expand the utility of chemical mutagenesis and whole-genome sequencing for mutant identification. We discuss mutations in the Atm and [Xrp1][1] genes identified in this screen.

 [1]: http://flybase.org/reports/FBgn0261113.html" />
<meta name="DC.Contributor" content="Chang-Hyun Lee" />
<meta name="DC.Contributor" content="Gerard Rimesso" />
<meta name="DC.Contributor" content="David M. Reynolds" />
<meta name="DC.Contributor" content="Jinlu Cai" />
<meta name="DC.Contributor" content="Nicholas E. Baker" />
<meta name="article:published_time" content="2016-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Whole-Genome Sequencing and iPLEX MassARRAY Genotyping Map an EMS-Induced Mutation Affecting Cell Competition in Drosophila melanogaster" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Cell competition, the conditional loss of viable genotypes only when surrounded by other cells, is a phenomenon observed in certain genetic mosaic conditions. We conducted a chemical mutagenesis and screen to recover new mutations that affect cell competition between wild-type and RpS3 heterozygous cells. Mutations were identified by whole-genome sequencing, making use of software tools that greatly facilitate the distinction between newly induced mutations and other sources of apparent sequence polymorphism, thereby reducing false-positive and false-negative identification rates. In addition, we utilized iPLEX MassARRAY for genotyping recombinant chromosomes. These approaches permitted the mapping of a new mutation affecting cell competition when only a single allele existed, with a phenotype assessed only in genetic mosaics, without the benefit of complementation with existing mutations, deletions, or duplications. These techniques expand the utility of chemical mutagenesis and whole-genome sequencing for mutant identification. We discuss mutations in the &lt;i&gt;Atm&lt;/i&gt; and &lt;i&gt;Xrp1&lt;/i&gt; genes identified in this screen.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/10/01" />
<meta name="citation_mjid" content="ggg;6/10/3207" />
<meta name="citation_id" content="6/10/3207" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/10/3207" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/10/3207.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/10/3207.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/10/3207.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.029421" />
<meta name="citation_pmid" content="27574103" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="11" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3207" />
<meta name="citation_lastpage" content="3217" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Chang-Hyun Lee" />
<meta name="citation_author_institution" content="Department of Genetics, Albert Einstein College of Medicine, Bronx, New York 10461" />
<meta name="citation_author" content="Gerard Rimesso" />
<meta name="citation_author_institution" content="Department of Genetics, Albert Einstein College of Medicine, Bronx, New York 10461" />
<meta name="citation_author" content="David M. Reynolds" />
<meta name="citation_author_institution" content="Department of Genetics, Albert Einstein College of Medicine, Bronx, New York 10461" />
<meta name="citation_author" content="Jinlu Cai" />
<meta name="citation_author_institution" content="Department of Genetics, Albert Einstein College of Medicine, Bronx, New York 10461" />
<meta name="citation_author" content="Nicholas E. Baker" />
<meta name="citation_author_institution" content="Department of Genetics, Albert Einstein College of Medicine, Bronx, New York 10461" />
<meta name="citation_author_email" content="nicholas.baker@einstein.yu.edu" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=R. T. Abraham;citation_title=Cell cycle checkpoint signaling through the ATM and ATR kinases;citation_pages=2177-2196;citation_volume=15;citation_year=2001;citation_issue=17;citation_pmid=11544175;citation_doi=10.1101/gad.914401" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. D. Adams;citation_author=S. E. Celniker;citation_author=R. A. Holt;citation_author=C. A. Evans;citation_author=J. D. Gocayne;citation_title=The Genome Sequence of Drosophila melanogaster;citation_pages=2185-2195;citation_volume=287;citation_year=2000;citation_issue=5461;citation_pmid=10731132;citation_doi=10.1126/science.287.5461.2185" />
<meta name="citation_reference" content="citation_journal_title=Oncogene;citation_journal_abbrev=Oncogene;citation_author=F. Akdemir;citation_author=A. Christich;citation_author=N. Sogame;citation_author=J. Chapo;citation_author=J. M. Abrams;citation_title=p53 directs focused genomic responses in Drosophila.;citation_pages=5184-5193;citation_volume=26;citation_year=2007;citation_issue=36;citation_pmid=17310982;citation_doi=10.1038/sj.onc.1210328" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=M. Amoyel;citation_author=E. A. Bach;citation_title=Cell competition: how to eliminate your neighbours;citation_pages=988-1000;citation_volume=141;citation_year=2014;citation_issue=5;citation_pmid=24550108;citation_doi=10.1242/dev.079129" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=S. Andersson;citation_author=S. Saeboe-Larssen;citation_author=A. Lambertsson;citation_author=J. Merriam;citation_author=M. Jacobs-Lorena;citation_title=A Drosophila third chromosome Minute locus encodes a ribosomal protein.;citation_pages=513-520;citation_volume=137;citation_year=1994;citation_issue=2;citation_pmid=8070662" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=N. E. Baker;citation_title=Cell competition.;citation_pages=R11-R15;citation_volume=21;citation_year=2011;citation_issue=1;citation_pmid=21215926;citation_doi=10.1016/j.cub.2010.11.030" />
<meta name="citation_reference" content="citation_journal_title=Methods;citation_author=N. E. Baker;citation_author=K. Li;citation_author=M. Quiquand;citation_author=R. Ruggiero;citation_author=L.-H. Wang;citation_title=Eye development.;citation_pages=252-259;citation_volume=68;citation_year=2014;citation_pmid=24784530;citation_doi=10.1016/j.ymeth.2014.04.007" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=H. J. Bellen;citation_author=R. W. Levis;citation_author=Y. He;citation_author=J. W. Carlson;citation_author=M. Evans-Holm;citation_title=The Drosophila gene disruption project: progress using transposons with distinctive site specificities.;citation_volume=188;citation_year=2011;citation_pmid=21515576;citation_doi=10.1534/genetics.111.126995" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=W. Bender;citation_author=M. Akam;citation_author=F. Karch;citation_author=P. A. Beachy;citation_author=M. Peifer;citation_title=Molecular Genetics of the Bithorax Complex in Drosophila melanogaster;citation_pages=23-29;citation_volume=221;citation_year=1983;citation_issue=4605;citation_pmid=17737996;citation_doi=10.1126/science.221.4605.23" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. P. Blumenstiel;citation_author=A. C. Noll;citation_author=J. A. Griffiths;citation_author=A. G. Perera;citation_author=K. N. Walton;citation_title=Identification of EMS-induced mutations in Drosophila melanogaster by whole-genome sequencing.;citation_volume=182;citation_year=2009;citation_pmid=19307605;citation_doi=10.1534/genetics.109.101998" />
<meta name="citation_reference" content="citation_journal_title=Molecular and Cellular Biology;citation_journal_abbrev=Mol. Cell. Biol.;citation_author=M. H. Brodsky;citation_author=B. T. Weinert;citation_author=G. Tsang;citation_author=Y. S. Rong;citation_author=N. M. McGinnis;citation_title=Drosophila melanogaster MNK/Chk2 and p53 Regulate Multiple DNA Repair and Apoptotic Pathways following DNA Damage;citation_pages=1219-1231;citation_volume=24;citation_year=2004;citation_issue=3;citation_pmid=14729967;citation_doi=10.1128/MCB.24.3.1219-1231.2004" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=K. H. Buetow;citation_author=M. Edmonson;citation_author=R. MacDonald;citation_author=R. Clifford;citation_author=P. Yip;citation_title=High-throughput development and characterization of a genomewide collection of gene-based single nucleotide polymorphism markers by chip-based matrix-assisted laser desorption/ionization time-of-flight mass spectrometry;citation_pages=581-584;citation_volume=98;citation_year=2001;citation_issue=2;citation_pmid=11136232;citation_doi=10.1073/pnas.98.2.581" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=T. B. Chou;citation_author=N. Perrimon;citation_title=The Autosomal FLP-DFS Technique for Generating Germline Mosaics in Drosophila melanogaster;citation_pages=1673-1679;citation_volume=144;citation_year=1996;citation_issue=4;citation_pmid=8978054" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. L. Cooper;citation_author=E. A. Greene;citation_author=B. J. Till;citation_author=C. A. Codomo;citation_author=B. T. Wakimoto;citation_title=Retention of induced mutations in a Drosophila reverse-genetic resource.;citation_volume=180;citation_year=2008;citation_pmid=18780737;citation_doi=10.1534/genetics.108.092437" />
<meta name="citation_reference" content="citation_journal_title=Developmental dynamics :  an official publication of the American Association of Anatomists;citation_journal_abbrev=Dev Dyn;citation_author=S. de Beco;citation_author=M. Ziosi;citation_author=L. A. Johnston;citation_title=New frontiers in cell competition.;citation_pages=831-841;citation_volume=241;citation_year=2012;citation_issue=5;citation_pmid=22438309;citation_doi=10.1002/dvdy.23783" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=M. A. DePristo;citation_author=E. Banks;citation_author=R. Poplin;citation_author=K. V. Garimella;citation_author=J. R. Maguire;citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data.;citation_pages=491-498;citation_volume=43;citation_year=2011;citation_issue=5;citation_pmid=21478889;citation_doi=10.1038/ng.806" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=C. Ding;citation_author=C. R. Cantor;citation_title=A high-throughput gene expression analysis technique using competitive PCR and matrix-assisted laser desorption ionization time-of-flight MS.;citation_volume=100;citation_year=2003;citation_pmid=12624187;citation_doi=10.1073/pnas.0630494100" />
<meta name="citation_reference" content="citation_journal_title=BMC genetics [electronic resource];citation_journal_abbrev=BMC Genet;citation_author=C. Ding;citation_author=E. Maier;citation_author=A. A. Roscher;citation_author=A. Braun;citation_author=C. R. Cantor;citation_title=Simultaneous quantitative and allele-specific expression analysis with real competitive PCR.;citation_pages=8-8;citation_volume=5;citation_year=2004;citation_pmid=15128429;citation_doi=10.1186/1471-2156-5-8" />
<meta name="citation_reference" content="citation_author=S. Gabriel;citation_author=L. Ziaugra;citation_author=D. Tabbaa;citation_year=2009" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. R. Gerhold;citation_author=D. J. Richter;citation_author=A. S. Yu;citation_author=I. K. Hariharan;citation_title=Identification and characterization of genes required for compensatory growth in Drosophila.;citation_volume=189;citation_year=2011;citation_pmid=21926302;citation_doi=10.1534/genetics.111.132993" />
<meta name="citation_reference" content="citation_journal_title=Biology (Basel);citation_author=M. A. Gonzalez;citation_author=D. V. Booven;citation_author=W. Hulme;citation_author=R. H. Ulloa;citation_author=R. F. Acosta Lebrigio;citation_title=Whole genome sequencing and a new bioinformatics platform allow for rapid gene identification in D. melanogaster EMS screens.;citation_pages=766-777;citation_volume=1;citation_year=2012;citation_pmid=24832518" />
<meta name="citation_reference" content="citation_journal_title=Free radical biology &amp; medicine;citation_journal_abbrev=Free Radic Biol Med;citation_author=C. Gruenewald;citation_author=J. A. Botella;citation_author=F. Bayersdorfer;citation_author=J. A. Navarro;citation_author=S. Schneuwly;citation_title=Hyperoxia-induced neurodegeneration as a tool to identify neuroprotective genes in Drosophila melanogaster.;citation_pages=1668-1676;citation_volume=46;citation_year=2009;citation_issue=12;citation_pmid=19345730;citation_doi=10.1016/j.freeradbiomed.2009.03.025" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=N. A. Haelterman;citation_author=L. Jiang;citation_author=Y. Li;citation_author=V. Bayat;citation_author=H. Sandoval;citation_title=Large-scale identification of chemically induced mutations in Drosophila melanogaster.;citation_volume=24;citation_year=2014;citation_pmid=25258387;citation_doi=10.1101/gr.174615.114" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. A. Hoskins;citation_author=J. W. Carlson;citation_author=C. Kennedy;citation_author=D. Acevedo;citation_author=M. Evans-Holm;citation_title=Sequence Finishing and Mapping of Drosophila melanogaster Heterochromatin;citation_pages=1625-1628;citation_volume=316;citation_year=2007;citation_issue=5831;citation_pmid=17569867;citation_doi=10.1126/science.1139816" />
<meta name="citation_reference" content="citation_journal_title=Cell Death Differ.;citation_author=A. Kale;citation_author=W. Li;citation_author=C. H. Lee;citation_author=N. E. Baker;citation_title=Apoptotic mechanisms during competition of ribosomal protein mutant cells: roles of the initiator caspases Dronc and Dream/Strica.;citation_pages=1300-1312;citation_volume=22;citation_year=2015;citation_pmid=25613379;citation_doi=10.1038/cdd.2014.218" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Protoc;citation_author=P. Kumar;citation_author=S. Henikoff;citation_author=P. C. Ng;citation_title=Predicting the effects of coding non-synonymous variants on protein function using the SIFT algorithm.;citation_pages=1073-1081;citation_volume=4;citation_year=2009;citation_issue=7;citation_pmid=19561590;citation_doi=10.1038/nprot.2009.86" />
<meta name="citation_reference" content="citation_journal_title=Advances in genetics;citation_journal_abbrev=Adv Genet;citation_author=A. Lambertsson;citation_title=The minute genes in Drosophila and their molecular functions.;citation_pages=69-134;citation_volume=38;citation_year=1998;citation_pmid=9677706;citation_doi=10.1016/S0065-2660(08)60142-X" />
<meta name="citation_reference" content="citation_journal_title=Neuron;citation_journal_abbrev=Neuron;citation_author=T. Lee;citation_author=L. Luo;citation_title=Mosaic analysis with a repressible cell marker for studies of gene function in neuronal morphogenesis.;citation_pages=451-461;citation_volume=22;citation_year=1999;citation_issue=3;citation_pmid=10197526;citation_doi=10.1016/S0896-6273(00)80701-1" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=R. Levayer;citation_author=E. Moreno;citation_title=Mechanisms of cell competition: themes and variations.;citation_volume=200;citation_year=2013;citation_pmid=23509066;citation_doi=10.1083/jcb.201301051" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=H. Li;citation_author=R. Durbin;citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform.;citation_volume=25;citation_year=2009;citation_pmid=19451168;citation_doi=10.1093/bioinformatics/btp324" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=T. F. Mackay;citation_author=S. Richards;citation_author=E. A. Stone;citation_author=A. Barbadilla;citation_author=J. F. Ayroles;citation_title=The Drosophila melanogaster Genetic Reference Panel.;citation_pages=173-178;citation_volume=482;citation_year=2012;citation_issue=7384;citation_pmid=22318601;citation_doi=10.1038/nature10811" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=S. J. Marygold;citation_author=J. Roote;citation_author=G. Reuter;citation_author=A. Lambertsson;citation_author=M. Ashburner;citation_title=The ribosomal protein genes and Minute loci of Drosophila melanogaster.;citation_pages=R216-R216;citation_volume=8;citation_year=2007;citation_issue=10;citation_pmid=17927810;citation_doi=10.1186/gb-2007-8-10-r216" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. McKenna;citation_author=M. Hanna;citation_author=E. Banks;citation_author=A. Sivachenko;citation_author=K. Cibulskis;citation_title=The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.;citation_volume=20;citation_year=2010;citation_pmid=20644199;citation_doi=10.1101/gr.107524.110" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=G. Morata;citation_author=P. Ripoll;citation_title=Minutes: mutants of drosophila autonomously affecting cell division rate.;citation_pages=211-221;citation_volume=42;citation_year=1975;citation_issue=2;citation_pmid=1116643;citation_doi=10.1016/0012-1606(75)90330-9" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=E. Moreno;citation_author=K. Basler;citation_author=G. Morata;citation_title=Cells compete for decapentaplegic survival factor to prevent apoptosis in Drosophila wing development.;citation_pages=755-759;citation_volume=416;citation_year=2002;citation_issue=6882;citation_pmid=11961558;citation_doi=10.1038/416755a" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=T. P. Newsome;citation_author=B. Åsling;citation_author=B. J. Dickson;citation_title=Analysis of Drosophila photoreceptor axon guidance in eye-specific mosaics;citation_pages=851-860;citation_volume=127;citation_year=2000;citation_issue=4;citation_pmid=10648243" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=S. R. Oikemus;citation_author=N. McGinnis;citation_author=J. Queiroz-Machado;citation_author=H. Tukachinsky;citation_author=S. Takada;citation_title=Drosophila atm/telomere fusion is required for telomeric localization of HP1 and telomere position effect.;citation_volume=18;citation_year=2004;citation_pmid=15256487;citation_doi=10.1101/gad.1202504" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=M. Pedersen;citation_author=S. Tiong;citation_author=S. D. Campbell;citation_title=Molecular genetic characterization of Drosophila ATM conserved functional domains.;citation_pages=778-786;citation_volume=53;citation_year=2010;citation_issue=10;citation_pmid=20962884" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=J. Roote;citation_author=S. Russell;citation_title=Toward a complete Drosophila deficiency kit.;citation_pages=149-149;citation_volume=13;citation_year=2012;citation_issue=3;citation_pmid=22445094;citation_doi=10.1186/gb4010" />
<meta name="citation_reference" content="citation_journal_abbrev=Nat Methods;citation_author=S. Sarin;citation_author=S. Prabhu;citation_author=M. M. O’Meara;citation_author=I. Pe’er;citation_author=O. Hobert;citation_title=Caenorhabditis elegans mutant allele identification by whole-genome sequencing.;citation_pages=865-867;citation_volume=5;citation_year=2008;citation_issue=10;citation_pmid=18677319;citation_doi=10.1038/nmeth.1249" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=W. Shi;citation_author=A. Stampas;citation_author=C. Zapata;citation_author=N. E. Baker;citation_title=The pineapple eye Gene Is Required for Survival of Drosophila Imaginal Disc Cells;citation_pages=1869-1879;citation_volume=165;citation_year=2003;citation_issue=4;citation_pmid=14704172" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=E. Silva;citation_author=S. Tiong;citation_author=M. Pedersen;citation_author=E. Homola;citation_author=A. Royou;citation_title=ATM is required for telomere maintenance and chromosome stability during Drosophila development.;citation_pages=1341-1347;citation_volume=14;citation_year=2004;citation_issue=15;citation_pmid=15296750;citation_doi=10.1016/j.cub.2004.06.056" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=P. Simpson;citation_title=Parameters of cell competition in the compartments of the wing disc of Drosophila.;citation_pages=182-193;citation_volume=69;citation_year=1979;citation_issue=1;citation_pmid=446891;citation_doi=10.1016/0012-1606(79)90284-7" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=D. St. Johnston;citation_title=The art and design of genetic screens: Drosophila melanogaster.;citation_pages=176-188;citation_volume=3;citation_year=2002;citation_issue=3;citation_pmid=11972155;citation_doi=10.1038/nrg751" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=D. M. Tyler;citation_author=W. Li;citation_author=N. Zhuo;citation_author=B. Pellock;citation_author=N. E. Baker;citation_title=Genes affecting cell competition in Drosophila.;citation_volume=175;citation_year=2007;citation_pmid=17110495;citation_doi=10.1534/genetics.106.061929" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=K. J. Venken;citation_author=E. Popodi;citation_author=S. L. Holtzman;citation_author=K. L. Schulze;citation_author=S. Park;citation_title=A molecularly defined duplication set for the X chromosome of Drosophila melanogaster.;citation_volume=186;citation_year=2010;citation_pmid=20876565;citation_doi=10.1534/genetics.110.121285" />
<meta name="citation_reference" content="citation_journal_title=J. Evol. Biol.;citation_author=C. J. Vermeulen;citation_author=P. Sorensen;citation_author=K. Kirilova Gagalova;citation_author=V. Loeschcke;citation_title=Transcriptomic analysis of inbreeding depression in cold-sensitive Drosophila melanogaster shows upregulation of the immune response.;citation_pages=1890-1902;citation_volume=26;citation_year=2013;citation_pmid=23944235;citation_doi=10.1111/jeb.12183" />
<meta name="citation_reference" content="citation_journal_title=Developmental biology;citation_journal_abbrev=Dev Biol;citation_author=J. Vincent;citation_author=C. Girdham;citation_author=P. O’Farrell;citation_title=A cell-autonomous, ubiquitous marker for the analysis of Drosophila genetic mosaics.;citation_pages=328-331;citation_volume=164;citation_year=1994;citation_issue=1;citation_pmid=8026635;citation_doi=10.1006/dbio.1994.1203" />
<meta name="citation_reference" content="citation_journal_title=Essays In Biochemistry;citation_journal_abbrev=Essays Biochem.;citation_author=S. Vivarelli;citation_author=L. Wagstaff;citation_author=E. Piddini;citation_title=Cell wars: regulation of cell survival and proliferation by cell competition;citation_pages=69-82;citation_volume=53;citation_year=2012;citation_issue=0;citation_pmid=22928509;citation_doi=10.1042/bse0530069" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=M. F. Wangler;citation_author=S. Yamamoto;citation_author=H. J. Bellen;citation_title=Fruit flies in biomedical research.;citation_volume=199;citation_year=2015;citation_pmid=25624315;citation_doi=10.1534/genetics.114.171785" />
<meta name="citation_reference" content="citation_journal_title=Genome Research;citation_journal_abbrev=Genome Res;citation_author=S. Winkler;citation_author=A. Schwabedissen;citation_author=D. Backasch;citation_author=C. Bökel;citation_author=C. Seidel;citation_title=Target-selected mutant screen by TILLING in Drosophila;citation_pages=718-723;citation_volume=15;citation_year=2005;citation_issue=5;citation_pmid=15867432;citation_doi=10.1101/gr.3721805" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=T. Xu;citation_author=G. M. Rubin;citation_title=Analysis of genetic mosaics in developing and adult Drosophila tissues;citation_pages=1223-1237;citation_volume=117;citation_year=1993;citation_issue=4;citation_pmid=8404527;citation_doi=10.1126/science.1197142" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=R. G. Zhai;citation_author=P. R. Hiesinger;citation_author=T. W. Koh;citation_author=P. Verstreken;citation_author=K. L. Schulze;citation_title=Mapping Drosophila mutations with molecularly defined P element insertions.;citation_volume=100;citation_year=2003;citation_pmid=12960394;citation_doi=10.1073/pnas.1832753100" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Whole-Genome Sequencing and iPLEX MassARRAY Genotyping Map an EMS-Induced Mutation Affecting Cell Competition in Drosophila melanogaster" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10.cover-source.jpg" />
<meta name="twitter:description" content="Cell competition, the conditional loss of viable genotypes only when surrounded by other cells, is a phenomenon observed in certain genetic mosaic conditions. We conducted a chemical mutagenesis and screen to recover new mutations that affect cell competition between wild-type and RpS3 heterozygous cells. Mutations were identified by whole-genome sequencing, making use of software tools that greatly facilitate the distinction between newly induced mutations and other sources of apparent sequence polymorphism, thereby reducing false-positive and false-negative identification rates. In addition, we utilized iPLEX MassARRAY for genotyping recombinant chromosomes. These approaches permitted the mapping of a new mutation affecting cell competition when only a single allele existed, with a phenotype assessed only in genetic mosaics, without the benefit of complementation with existing mutations, deletions, or duplications. These techniques expand the utility of chemical mutagenesis and whole-genome sequencing for mutant identification. We discuss mutations in the Atm and [Xrp1][1] genes identified in this screen.

 [1]: http://flybase.org/reports/FBgn0261113.html" />
<meta name="og-title" property="og:title" content="Whole-Genome Sequencing and iPLEX MassARRAY Genotyping Map an EMS-Induced Mutation Affecting Cell Competition in Drosophila melanogaster" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/10/3207" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Cell competition, the conditional loss of viable genotypes only when surrounded by other cells, is a phenomenon observed in certain genetic mosaic conditions. We conducted a chemical mutagenesis and screen to recover new mutations that affect cell competition between wild-type and RpS3 heterozygous cells. Mutations were identified by whole-genome sequencing, making use of software tools that greatly facilitate the distinction between newly induced mutations and other sources of apparent sequence polymorphism, thereby reducing false-positive and false-negative identification rates. In addition, we utilized iPLEX MassARRAY for genotyping recombinant chromosomes. These approaches permitted the mapping of a new mutation affecting cell competition when only a single allele existed, with a phenotype assessed only in genetic mosaics, without the benefit of complementation with existing mutations, deletions, or duplications. These techniques expand the utility of chemical mutagenesis and whole-genome sequencing for mutant identification. We discuss mutations in the Atm and [Xrp1][1] genes identified in this screen.

 [1]: http://flybase.org/reports/FBgn0261113.html" />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10.cover-source.jpg" />
    <title>Whole-Genome Sequencing and iPLEX MassARRAY Genotyping Map an EMS-Induced Mutation Affecting Cell Competition in Drosophila melanogaster | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"IoZeQWsDf8mOcg-rz58H6EwOxnH5EOmhgE12LlV6TEo","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460342","apath":"\/ggg\/6\/10\/3207.atom","pisa":"ggg;6\/10\/3207","ac":{"\/ggg\/6\/10\/3207.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/10\/3207.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/10\/3207"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/10\/3207"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/10\/3207":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022bq\\\\\\\/JzH1F068CVFgy0tds8k7Xn1VTlvKsIZ9zaG4RScolch+04iTabBfWNqcIWzLMBhXVvLtOTrCQsp826xScuC5Sg408NCkJk+JVvkxEu6MSuBK4rlRCZkCH5S5ZaisbYk9QBCwwNb005ehDZztlzYdIZPXzqj1hIn+xVOAgeSYamk\\\\\\\/27VI2adlPc+8mXMOWsZCrdlZ2l1Z12195yPzN4Va1CN5PzoOXn8ClJoSdjHwRqkFiRRswGDyW\\\\\\\/fnIvhHF4aAsohCTqBvGKZKZS3RWr0Zheq\\\\\\\/A6dPEJStbuL1dYLAGr9v7Zhh5M12GcJycKVevmCi0UAlUoQ4FZn+GLvnF5GsA6RseyfbAQZn47RFIfAQOdI2zWO6UPCKaC0Y4D75NCl8EaysL4H8mvYQb7okxUhDhCuPVm4DUPYCvrKVPm98uZqTRHYGQo1a5uJsIks6VipIz1xBND1+JvZF+ytX4zO2Ld+OmOF08RFdwbqpRR\\\\\\\/SQg4Qd\\\\\\\/hH1MRWeRNS6D8\\\\\\\/FJUpIl+ahowgTohwGJbawh8NKav7IzRkWQ1hFaehmSJzaDBfr7+wXpnjjz6l8amQCmLBba3pyUpbEMzukTuhO3q9CdRoUnlcXbOqgrOmFfpQCgaApOU27T7jAqYLDzYMQK8tnXEGn6lI4ag6A9QBWlxbjklzfeHutLGiQVWMNAaz6XuzvuNNIbvLmsw2O2xXcgq0AGS0dC84aSTlW4oZEyp1OkMenPUohd6PyfRdw9qaruSQEQ7FIkTyrBpN9fn4xjHrGWf0Zsdb594HZJkM6SA0+03oIdRt7VcNi8lqpvOxt1iMvM9gMr7rbq9Hzm1Ovg1Mu8GI2GMye9lYrzNOmP+vKxbFd+LGxi50peX8WlVh+C\\\\\\\/reP2GJcHzgwj2hs9UhgZRTrUtRpjIapaFuyUtiQTWf4L8QIQqI3ydghioBvQrPhJgbYpGWzcPbeugdCJo85FCcPQ51F3jJANuTnLSgsQCDrDFTHa3X4omuFFzFYNLaufs1Tx11mIbN4pAXofMoANVcvxtG3EhEyX6hRqnwgza92yZW7n+NFCXc2\\\\\\\/ELAQGrrc24JDNxaZkOGhKbIVZaXZoKRLd1Tg79KjX3wYCkFTCE4B69kLVy4N3Q6E5N1r\\\\\\\/2jxUGAuWicw2LZK6K5Xm1BcMCfs7KWkHrpWG8zkEnhkxhcdM9pKjaX\\\\\\\/W2htg9PGxR7i6ezYeutdhrCrvtwL8vXfAcd1xSAqqIRojuDd4o8gYBGjuOuOmZVMoHCFsRyYBAbVgvQDDoURYWUgd61Je6\\\u0022,\\\u0022iv\\\u0022:\\\u0022UwBmKj9iPLJeEkPQ8jZ+Kg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002261d9fd64d7b04521e8f61481d45c34a14d3a20264ddd989e21b99136b5c5b38d\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022Nos16Hc3ZIjhJ6gQ5HYcEMoPtkW4Uri+GEWZ+iW8pWNjWrY5v\\\\\\\/K5q8bS0H3\\\\\\\/4622WzaMQ6TMz1EwuAfvUeCD0h+r56kXXFeYK8SC\\\\\\\/+YOgG7s6p2Nw71vNA4NbUaQjHMHev76CnY6+RRQXSLCMiv2VKRGcVzxurR1FC5Z2BV7meLyNBt9WAG728OB8MtzmHAnvioj2+ETGrjd4XNHb1HKvrYRjlg4Ec\\\\\\\/CanUMtYLyvMMv9QVWnPduk5kHCq8lkxVZJ1zApdjxm1XHrMxp0T77+KD8dRd+LUNOJwAAkpW4RQF\\\\\\\/yUh1RK8sQNtEhu62pjfr7vNywc7VC727us9LjX3xCv83lL0oG+8UJwTycPSiU5iO4+Bstvt4tLrxXE9MLUi6oXHdNh+OwKTDZ1LwwCioN5ZOVXJzU9w0CJxN9uPxcoB+wfZXB5hQsMU7wcyfM0KpGZQtniymmDHzPvX9vsM93M2aXqcuw\\\\\\\/ptXnOlD\\\\\\\/D+09W2qbva1lyTye1w74rvKjMlMQa5PaMcXJ+sXW6bXjm5Blp3rA+GkmLXscqoijBciESKIAf8aLtyIkiqe5yrjJHG7WL8x7Oa37QkgHllxt+55PTJe2qRDiruF97wn4MDj59B9Z+TVIZokWYyhdJ1zP6272dwn7gImZy\\\\\\\/5CNuz6Q4HZSQcLMhy\\\\\\\/633E\\\\\\\/YIrq+zlKHNxKFeZplGdHQkFd5L8+0Oz17C0HNInBAErFHaxeb0yEd9zV9IgumH3dZK1jRlfwK\\\\\\\/Jx804rNVBfJplIWdrgGNQCCr4Hkww66wmwk7KvAnLmfC8JAHIEV5rwQuJZddgv1hGrU5INroSDcMG26anxGBFLriZjCw27XImN+vHxMHWzsRisy8agsQkQpnGo1cC4laDXyzs2PruR6d2v+lRzJSXU+6d953oAXPlHijCozJExP6w3bWiP0rCakp+pZVtPVeA0dWktDm0MHRTFG42xYS8VbBw7rhJSyCn8YLo2CTskfWlV8LngsCIrrd1RBdbk7J1rop2BAbJQgldejD6stQLrxklSLuOzc5FTEf6cJpsxxzsFFM73G\\\\\\\/jIEjfdmaITwTqcMV8RK277BmmI1YzpO9ebYW1RlwgsKylwXi\\\\\\\/tuR6Bl0B8tU1PzA3g1LmqrxUFP4Vs8xlHYu+M4m1BqPGT+YCQ7w6\\\\\\\/G4BTwuiCOEgkyXD6kUzA4RKWGgjG7ABofbqIOTq\\\\\\\/mTdzp+B3uC6bqJ9avDRw3Hg32Plue8Lmum0zRXkkDcX+P0Qo3MlEG19+Ad+bctoj2FhQ1ZorwcgoRRL9EXcREVpWG\\\\\\\/WB53GSv\\\\\\\/0FM0hhDQgR+lpzkCnDbvHzfsqKCrKR2XbcsLyhJS+WIED\\\\\\\/ydwOagc65M1vV1WAxDPMZpjovsW+XL+rTZFT5HGQedELWPXO5Kw4wPzKG4pn+U7WgG4fwyrnKEJ7dby\\\\\\\/1q7EN8MBGl8vJqycJzMo8sN2c5gIbTXxFcjz\\\\\\\/6srWQCR6ezH0G+snTHGrgcsnKgRT4Vt2yjswVImL+Duz7Plu4EOHtWuZtbGc1tSFqhS\\\\\\\/W5AF+4QL2v6cBGMflQoh5PBWFpxjm8CNSNQRoIBXgjn+nFxbnBjvSQNbuhvytl6sVjKuAY6nJ4tTCy4T0AYZwSRQZgrNjCcx274Oo5W2oUiGUPxFHsFf2ItLs0GjE0PksDBzpCfgSCDq2Y1nrJjlwBznzPHE7y4i6h9LBaJ1bJbS0Pyrf8AP85rMZfDSgucCnYHBjm8xFiThPnfN3DMPSGDVORBgYxYTjZqNpwqx2elBrDWQaa9CFZqO2wB2Sgc0FwlXFH2ibEY8hNBlEpoJkl0ZGRmekq4CVE6\\\\\\\/Kcif6PqVgeAWDXgPcaWUtY86ZURNBI5AkuM9T1xCPMDW3gO0kEbfsZ0FVHC\\\\\\\/VRNoEvzTCcy3iAX1hiGw\\\\\\\/oxsnKq92ZOa+tW+fXRFCUNJgpeQmQw9x7n9MxLEaN14xQHXFJyy2PdfMd5FkRTcb1yuOLA1Z2Js4gsBm5Z+cb2jc+LVEH6GxkUrGXb\\\\\\\/rnObGjxvr9TsYjCXzTJxhxA5YWP9S0fABR+UjKNgtL98Rj9x2lQY3Zt5Lg7EOz+gO99x6P\\\\\\\/BHpdQZd6KiRoFMPVEdFnQgB9Mxe0C3PxRDMCA4VdPHNj8jJ\\\\\\\/vJU3WvLn\\\\\\\/p5gCuvFshOU+ZWqY9T6HcyR5WNngIIGlRQ8RLx4At5kD2946NwudXX81pmDUiV+ukch6\\\\\\\/lVs5tJrQsVXABK\\\\\\\/MX1ypxgXFai3ERpNj9OpwuOlK9pKN3Fz4PluyBxKst\\\\\\\/dTQfcfs3HkuDkkoojhJQwkxmfWg5ZHyNN6MLh4jklQheR8OviPkNg3+aVk0QI5BgH07UXI1ppfJtytLbChmhRXgD\\\\\\\/09jPMFY2GXrHoC1JM+k3oueEobjVG\\\\\\\/sVURUzr9jPZnGTJmqk\\\\\\\/W0J4VSYBINQIk734pLFGxI38z2nan4flCsIo\\\\\\\/RsQJGq9t4ZOyuFy2SmYp1acDjDs0O8e3yngzINOf2IWyjTaqT268iDCeQHRAEaLVv34i6fJUKlKmXV8OZJZxAo8aMQm7BGu25ZDGjI15O+J8b6y5nTTRmWnIg6vVMuFurtLGtqEM4LpbIZYrFzvPcadGqeEBgkjhkB0O7fCKc85kcQ2YBO3xyAAuUCDN+G4s+kUl1ZC7VQWp5BIG\\\\\\\/d0145k9t5aViNrf2gIszkEDqXvjMwPXBUXdobn7Ns81gRBA1f4PpKzSRsxjA4\\\\\\\/G\\\\\\\/fZYLrB\\\\\\\/U6IJnBqaLlzKGpbc4enV8zkKFsDgnWKL3IjXsPDSG5t7pXXhpSiCj3p8TnAOczG+Dd+2TRU65eFCquXR5jyCC9SbwBctHWha26WcfqT+LSk9buwVTnS\\\\\\\/7C3uhL8wL8TDdgj7+sspPfV4IgzNoWjx5lcvbqB0akqvKIz2A+GA91zkZO8E4wASpDsBt+Kj8kGU5JyskL9Rv0R4KnK9iykRiJ8N1IeXBqf+KbUoHl\\\\\\\/zkXI33huh3n\\\\\\\/kWhDDfKtwelGRzWjLpgQKTadhCR+hXbc986lVijGJGCqCjo+A\\\\\\\/qbAcRvLij0ka1P7QNoSLkz42RkXXaSpWoPZYn5Y+Fqczk9QV92fvDx7pWUzYrYQOpbBuBoEvtryfGnbE8FIUQl9dCQtPpbV2B94FzIKgT0aAfy3lJbWXkjof2xUXvV1\\\\\\\/ukjgEvUUsMlcr+YqzWe13dzxr3LmUiGiWirRBbYvODAr4MKXhjt5CW9KfhwQXhI+AmLGiKnMuIqJ0tBjtNYkOOmiVpdh8xBS4m3cWnDBMdyPUESOR8VkG5zQHE6epV+F44SqFMTElEF4CAOCTthoPBqCqaO+4byA96Ki4wbYBWMgkshzwucfzEXQjcQ9UhyerJsuH1JZ\\\\\\\/jO86Dvylv9tgkUL8hnPwwxm+9QfXrF7S2m8zKRyB0pzEdjNZCxvDroSA2kWn4qzA0IymxefV5CbfzQOFVV48Ins42xAXjMlgN6uLSEfnRRBNrMJ\\\\\\\/2jZlZDwtkoquHzjSsCL+h87h7qz5l\\\\\\\/iee7BRsYvbCAOcbCgFBTucp1nj8dcsh44bwdn31y9JGNCS+yixKktbLTNNlm9gT5qp2WouxU43\\\\\\\/G6inUj0Tne2Ok\\\\\\\/61\\\\\\\/Yf+7MaBuKHgpN87FpoIRQUiSIEt4UqWT1I1ICXJSaJfeHxDyq7qcNBiSjs15MLV1g0KrUo11eBahwrNztebHCqmzwN5UOzHYrSueea2DAHeLZnQ5XFo6hCvqJ33soDbVTp+Af4d\\\\\\\/5rGfC5LY2eyJHSjFlyHLCNpiy6lztKWRniDC562BO9H3AXI94MZ8Dkwaad5eGODcO6lQnpDdSvR7Ys2mc16i2tVcLJlKcWcqQlslGmWov+Vjq5Yk714JL+3DB9am9pvpRMvRQZXOEGcBP1\\\\\\\/tTVfePE8dxy0\\\\\\\/RNKx6v1YeWoQkKWG8Xo8xOm95tNuowl+wowAHGHIvu2VOSIYNOoQDSFCt0f37YemA7fv2R+hzPEyOvqQIYFwozTsWJQWaendl4Z5FIG2h2BLTCFfpvUqsQ18005w2HKI8zUteQ06XQt1y1AyPlazlbzZOY3TgsWMYf2j25b21NI9c4sK7mj5lt\\\\\\\/DZ60Yh\\\\\\\/aF6N86q10SPqK6+QvvcosJInP1Plbf\\\\\\\/0EyLi8sCe5ZMlrCm0LEOma+mpyIzLjNDh9ZkV11qF\\\\\\\/bWrAc1OnXoJidK0hUttNbfq67bebbtoSAKqzixTRBM+BNKtp186jD6NPYzkA3QrNlhlmU81eqInD+YDaK0iC8n8dbkpai8eMTjbooe+mCgodHsuLTlTzFtpzQn3UOUwc\\\\\\\/4eiCEAkXYSl9MhhLinnYpJOm8G94KVEJ3Cg\\\\\\\/K2GXATg2\\\\\\\/XyHIh0\\\\\\\/YGrNZay2z0RnBzXYHdTWGClXmEUnLNEsNjFKjx79ueBZe8Fter+MLyxHZHvc3oEgu2l4dTbLk9BfzPaCYNFd2bwrJo2gZlI0qFHEkHm5lB1nO9VuZDerUvOivkkNKEfSg\\\\\\\/o\\\\\\\/eDoYjUlDMDdVVAZn+LxHqa8Mz1bEQvBLLFoo5dTF\\\\\\\/9lUfhuvK9jWna3qkTVex\\\\\\\/zs3ouH3tGJlCt9OBj4CBp52Spug+M34WofileUqXmlfXv9GptzPmfolD\\\\\\\/T4eS6A2kv9qkqNjHe1QxD7BlEIQDOJxo0kdOQNJ280rkukMKcXjOrLx65a8XGL\\\\\\\/pK9jgiPPTj79eHVFYc05TZv6EhkSUmniIFVgmottg7NlxbL+P6nsxWk9JfeE20BWYW5rLj2\\\\\\\/2rMeVclNEro9omN+C6yhkgkfRa2cXmsO2iqjlf8OMuSGcQGflJZWKAuX2bh9609u5rU4MnwG8XobF\\\\\\\/UYOPJRs3\\\\\\\/Yao7OYHhlxnYKeYm1Y2QjMOiIaHmos4osq3ggEzswij4lpBL5cJw2oC9h16TUb64sMt34NdS6c8NT6bIFwlyOkL1MGLqpiKl+bWoZEmpEr9LvZY2HXc5ownTbNDX69c5YZ1lA0+UVz3XCW+2IVqEmUljERzK8LSbrawmDFlT0KrjTKT8DYXJuy22mdA0AGqbWfpnNwuaghxGUB1ivAQi6\\\\\\\/ubf6mUNi7X0JOm4SgpY19RoMXygbzc7+4hAIYZJ7CWZywdrNOGQpfQLp4pXaVnwCFcTGsEHHTXsRW4QFUbZ8iSCS4WqeqsEHZlJG8cpazmHdD2g4gYtNxeXW57KuCUs4x58hpbHJmMUv1f4DXdZgW8KgjzfPxVg+y7QWTImmWi+DjpnrZpeqy8F\\\\\\\/+HjTx\\\\\\\/kuZY\\\\\\\/GFv48dC0T07IouMMpIszjpm0OqF1mwXWa42r4kvkRs\\\\\\\/IFQIRo1WcZpYxXkjszqeklLQNrl6QgVstV7msHQAbAOHocizu4PbCdaOhnNL9hahP3GptZGBlRoI5dJMEEUl2ukMAZOjZjbpLH80m5qhX4zy10d4I\\\\\\\/WbPJLZdTq6Z\\\\\\\/jdFLNViSZclEW1N25L2rg+8RmSgjax9knUS2yRb5Jj64NX3HUfIvc\\\\\\\/SCIqzsTKD6DqZZp99YnoAI2\\\\\\\/xfHXk86eBFupe41LTYpkY7BQ+rDWRfmvsp52a\\\\\\\/DLEbqdlpuZtH92HEnqQazcnWiaMhRQdDZrOqoU0qLzeah6WHLR89GId50wfrQ3dsaEgFZXLLsljVUt7ds+V0UUoFOsvmPekCMFk+BXecLOKRjFw51hkmE+M3oa8WdYZA\\\\\\\/eli7CreC5yBxqmqhO9LmmSiSxggi1ZgbKdC5Wm6WbyZiPr4rDfrKJFL5ua0Cj1A5jOJaCsPATuPji2OePgaEPgAK3ZOXHaPof4Le8LnBh5Io3Oujv9uzUrd5ry5w3HqUgnMRDYh8ZSfAc62aRxL79VlHLP\\\\\\\/UkVopzKor9YjnPF0CsZJVZ2f1ocIBinqfhlwN4ci7QvwOlFQVg\\\\\\\/j4mtYuI1NJRuFzvafuYotXxFPtXIuSlGpEPxwAYI60Z\\\\\\\/iVjSI8y9+kV48nFiV\\\\\\\/\\\\\\\/5uNhFEthF6cjZO8EHbf95fYfZWUmNc1C0HjCrNPLgKzf3eI5rjfRjDtXQ6vTLF5V6OI257erensoYudoqdm+j1dhxtiHPuPPB5rzebc3iJR2T3kiEuYqFhYMOgFRAVp76Qw6ng2I15mn8ChJ8UKzeH1r578SahVjXR0VHxMkIadbG+8ZKjrMFrfmObkxrC2\\\\\\\/aA\\\\\\\/Jp5mbnVXnglTxujU0arab5OMqSHGEv8fEP1tB6hEY54BiY0Mjfy59drT4LH9rcPy8uSNlEupqYV62kQZZPMOBX\\\\\\\/2xfl+44F2qd4v\\\\\\\/lITOCmZajX+HWHOT7gnSc9QF0SNS8a0tGKHuue4b08fxi4kDa2v+nR3wBmSr\\\\\\\/SNFrq4K1kOXipzCW2qIXqXQcnLc7zxQQn1WGywrUYX3KpYFckO9b\\\\\\\/pCzq5aZySGgNAmI5clfMKh+JMGd5Gd2L35ysVUHjbbHEIWTGTQKx1T082SpHLS36Hh709WtfdnWDK9GbzX+d5xxEb2JJhAOXtllN009vTX2StMfkTGyjjFXgNz7ph0YU+kcXMNnvn+rKQu78B7SZ+x\\\\\\\/RU0TJDz5WmAUsJnzZz4N7wvJfDcCUwT5a5q5CGisBGyRzCE1GWvWJEJ8P6ZeRpYn0u9CcaNwXH2u8SmzV7zdJNugoYMEDLDJIpn+Caqwme885qAO1\\\\\\\/kp8WgD5e0o+XvqlOjouRtPrwGMRu2zIBdPsPAFUYWOSUmpIc626FChuPTzMpY6zPGPCo0MsZwy7sFHJCl1NVHjUN0H7Vz70MDcaI42sEv7bVaFnUwPRbtk0RhM7UvB2GR45bh7K+3rfa6QPCtAHMhpsAxfwVK4C0IEnrEsHYRWvHdE3R3VO1ZtveAkeXUxNuDC6JT\\\\\\\/qr1eWqseZLxStJJu52ITAnNWsabxKVppLZ+1VHdTBaXm79\\\\\\\/XG7+z+u6fgS74\\\\\\\/5KqQVs71qO1uyQovBCDrT78kch5rAcvNWu8a11nlY+PMn\\\\\\\/mK2qZEcwtLIbJS2IGRbum62svL2WW7B9o4+IKWp3bRo6eXqRTgm8uLm+f3+0dk\\\\\\\/h\\\\\\\/xxedNAYnXCpro+u76yWifuKbwnB2iWkJ5qzyv5D4veMlyfwHkBKosPAsicDe8KXJYsPsbTlprt85mrHl+Y4dwKQ7d2cTxtpUbgB3X6gGvlWAmlAJF+dnPuTt+L8nBq8PY7e6uD03typfSM+0Rb\\\\\\\/eDSrd7qe\\\\\\\/Aqdwz1Wy64iWZOFdPF0ezDTBx9S5fHXX7FOIunTaXFH5ubStZpsSybw9aIsZVTVb8RJ5pVeD0xxPgl0TlJdm1zU8exQ31gtTtdgx2iaMkGqFAR9Z13AURMqmkryLyDV87PQsQnBbfFY4QOJ2af1vJ9k8oLi4CsPU2RO5iVrVwvOKw19XRhC8ZHHMjtAdNGHzEnQFPTWK0hx6\\\\\\\/h0klu0AZHbR\\\\\\\/Kqyn4I7ebo+BELoTuik5EXtUEqu9yJ+RcGC\\\\\\\/vXAnhxdxMf8yEO+fevBqRYzCW9a52GIuqBg5g8CDlIzoDfjd1G920IIu0N0pJPT3aR9DkkCBFvBGcHmWpkMQ8A7dvfg0TRje\\\\\\\/R69fRIzGctW8SaIh00p0qlpZ15nH5+mAeIzTkUSPK8rCpcmsQeVdYboSFdR0SCTvtqueafFHogAvEwMEL+8tsVWq6ZTUznE5QIuZFzBOiZcUdybzGJYLVexu5tEnP3QnBMu0HeoJ6EE87dqCQrx\\\\\\\/4Q8vv8mRpQnMKehkA\\\\\\\/oYKGsKpbCBdZ9z9hg9qhbj\\\\\\\/7KQSH5otxp3xN4xmDFjXlMlLkS1NsxLsi4IBQ7u+MOxWJXBQjdnwdiMt+H49hhKEt2fQaK8WO8kcAmWPYIPzszAIPR+jbsloxi7DFwTPCzNZxjSEIG5BJztcN9lu6FCQtOqyrV4gGd9W\\\\\\\/2xsopR9fEw0VwSUCQp38pI5mZ0JVpCcP\\\\\\\/ZsCjlwN5biA30ZPKEsL06UtpefxuU82rmDbDwFxNmWV12QFZulaARkVyZfvt1pbIoelDRrRCXXAzQK6ZmTcoZurK9R0Q43ufQSB+RsDVC1g+hV97vqFD+8iUic97Pg5vHjlzaF25rizTx4kDHINrq3TUZcibdzqsVMbJOM7cN2mdEixlc4BFuZyH6h8mWkSFHnZOmPAMdh9keV8wusxoh7axxPOTr4KWRaZnTIqF2P44ndtrcRTddoKiq63sI0egj+2LPYS07VhFLAjcifumqpxj0SfKZk70BpUDJf9rppnZKMVn3sL94xXsO8Uws3fW3SEjiWNJjGZrZ05xFZ+DVrvanel\\\\\\\/JCNsic7E5yC+iD\\\\\\\/bP78avKH9NJ7+h4yI1XHrNm6XjuWeMfdTcg1fHumSk417ay+hHhOUA\\\\\\\/K2pe045bdZ5EfuM+Sli2BpHA2USnTVSfGYf0ozYc\\\\\\\/0M9JfzDRi0BkX8iTU8rwgSZBjguR9rRTRvi2IQGF304f9EmJ2aRjWth5umzdmLYH2UkD+0Lg+fEQAa5GCGy2Obpqo1fid9e5Qmvw+fPLqdilQbyjjIiWaoNWSSzcsF0NL6AlNsYK\\\\\\\/aO0ZVyFGqQaiBgjBIABuUVElFmYXKTdq+sREPhj4UHvRac8AEARtCpZxYRbz8ye8KkjTLhYfZ7ogg6fGALQzkD63OmnGK9+drVGUDHbntgPIqcKdD1F00jf7RsFbgp4cCzOfr6v7Fmzcs8smC\\\\\\\/Bd2\\\\\\\/HlqusLhXAn4gdMp62oEAVF5ak08ni++AX0YFGfrRShJgAAYBtBEwsuap8rrOKuAEINEyQ5fMpxerQ2CEy\\\\\\\/2XX8SQZba++vE7tDOa5hLZyb1gTdWmY4CN7F\\\\\\\/9s4Q95e0bWVqd20Xjc7VInuDnv4+\\\\\\\/w+ZQcayKR2imPyjbpBhKLGdP06VA3KA\\\\\\\/ZbrI6PLqS3FweUhJWt6ZNXpkIVOHk3p5OhjMbZFv+P845wlk\\\\\\\/l2qqRUuMatbyX4c4c6qx6sQwka8HlPQdaQ\\\\\\\/ism40SyxH\\\\\\\/kR14hO7TEAxpw0FBFG5eLo9Pu4JUy0aM6f5k38zW7dZruRhZ1jzknhqcHPY94Vf8WULrikPqJZ5\\\\\\\/4X5jqxt92yTdVEcUUWVlDjNy5j+TIQsXBhLGO0YNSd8wB+PAtY6VIErqpR5ctUJT+y3jjCxcGN7K+T3jupOwdHu2w5dJ7Sg0JIH4C+FlvnSJCL\\\\\\\/59uoV1qOYv09+l2lbOcPqdzQ5km08G5CPwetAbVf4+aD22J68lay61Nym8lj8aGeEqYpYCcGDbnR2qRCHKyL\\\\\\\/G92x6B2yVrcRH0YCDHPbVyN3qLhGO1aSAFCvVLdfOSevMIgVt\\\\\\\/Pa51KeSI9O9Oz+PllyCvsT\\\\\\\/\\\\\\\/RiCM3oZcBwbm6NMCaPsbLBj2iD9JtnKY8YNyjqYscyCE9+x53PGin58xPNaE3vSBXS8EzOhKniPY6OQbKZ++u3PZ\\\\\\\/wZqeenRIUpJOuRrwPGdVcLI+JQPutemMpqVHvh5GdS5V742K6X+LXxiLC9fFGaYZFnU7BLldwKw0HxqWUZYU9w65fRTxq7Hq\\\\\\\/kX+0uyYKRb3\\\\\\\/8jsWHJQMDD1lrWHcLugs8hkalC1EYwlb0LT9vl7uldUXd+qr9FoPwySb+YJzDI36H2DqQ1EcSKrBddvbdksFnmECymMGPr96cReUX6OJyEraRMUezKiUbUnQRZtArLw3LS1nMYMevtxAc4mL+6khcMy+nRIQj6Yl\\\\\\\/+c++8TUpQXwOgDFlETKKE1b61+J1YJmltG6s2yytk5tK3TqfmMZYHr67nq8Q+XY3oIx1atVI0SKG\\\\\\\/EyULNXkeIkcFusoHm3mjmtpo4AxFB0LJ9YRKh7h6GRx1ivck3iVIMLuzjgc7DOICKzXOzyVqjx3\\\\\\\/l060cOqTAbMczt2+3jhRCpiJt\\\\\\\/eWLiFXW1cD1lEWY4Aik3TQaiAKH6Qc2Q0KezL\\\\\\\/Sr4baZWexmoT09FEz0Q67OqrdWdj62Xrt\\\\\\\/JCWriS1igEzysA\\\\\\\/Uoy2J2tlhvU\\\\\\\/8MlM3CQqVcw5zK2bJhri4kX5Yusn7KhFdw7W6gLqVsfUUzC5M0NFtNFNFubVomElQwyox9ghqj9TKPJJM0uiAK5rADI3at0CXKLC32ETOrwVHoGFhwQqn7YrkryMOHIPnDhTYNc\\\\\\\/DZ\\\\\\\/29M+QaUD2MRq6yPdAb3dyBG6ShLSt\\\\\\\/KUzFdnq54zKgN4UyzXWbJ4BGPt2nGcAP4v7hnwHmMWUHXFdYFObDtwEdqqYjX5+4sgmeGx8HKFa1jgXa0A01dWpKjaHqxK7uo2PTgK1DKA3ZD0FfXl\\\\\\\/ZxIhZwd6LQm+YtCBAWjuYGIC0AMx9+uVYWZrjezeYmQelvIFs7BsiwKQJ2YFvl30LQ9KMgkwndMCfIZ56cf1EDJUSV2r4TKh4DBXE49wEVCeb1VFEKDtLV7MTOE4GBCBJECkG3Tk0lE7fkAAeiePYBUu3GO+CZanhK7Hvh2+aYHMThRU9fk\\\\\\\/WwTpFyXZsurOBOGMDVSyilENUkW6omJ\\\\\\\/5WlLsTeWEOJD0uNOSYroa3RwHtH2SESDS+7ycYV49\\\\\\\/Lil84N9UckWUjMY6+cOr3n2ae6LguWarjqTr6wgzSAbOXn0hYQBtTlF1L2A5ZWx3sjLcx9usx4FNJHSHbBCHHabhIgvJQRYtUBgWOuyzHDu3zu4k\\\\\\\/Dpn4lUaSKeRl8fvsaAE4kGadO\\\\\\\/WfI1560YAVGHC9HFsrhM9UltbWYGfsujL5BBPmyxoPZJJ66XPex1TkdF5gjw0M+GRByZJPCTYWkSTufGFl5nXZWjAK66BeBTl3MpljwGQd\\\\\\\/pKSulWbqBYzuINagbtiJz9XS5unsd3WObK5mMdWcqG7QNsA+IM\\\\\\\/M7zA\\\\\\\/6myB7OpKISy6civ0NkyCQIsuI87ag51myu22plAbE2bqYmCfy8V1UtLWVjS3o\\\\\\\/QnV6LP1i+h7UPQWwxlAn1L7YET\\\\\\\/ZGrxyrAPfBUAfHFt8wtHK0X4dAupsQS\\\\\\\/y\\\\\\\/QZjn9nEfZo9Il\\\\\\\/wYuz7dMfulKH7aLW7N7sO9W0K2\\\\\\\/6ErtT+cMmwOHGhhtuIlTVNp5t2IhQHtufdS8Ljb2difLcQx8PVXloICxlLL7iKsLrEsX0PG6UMPhkzeUvG7PNRAoiVU8E5FxLy\\\\\\\/kB7m8WeVu4xIqdviu9HhERgvCoOJi6uYZPHDl6AdSkFLqu\\\\\\\/E65xP7mB1CC+s6TojPFa4Tp54zMpbmRSqm8TfBz8tb6fjFRqeoGdb248lr46A7+3dq8igLg6KXY05ZoOZtj2eZ8FX7aNVq7YChlH40p+YqYMH3JqLU2kedJ02CCUhqRy0JIRIdJXg1e7Yefyp8V7gtkNmlBxLm6DoLuHh\\\\\\\/KyHvF0idiZZs2Iu4B5NSrDpEnNZysBupY3+FSeUurQ+\\\\\\\/XPFVbb0m92S3T+RgBENmGYTSP\\\\\\\/3yoMeXM5XUpN9oOdRHidJlIYpB4DHur4NCgaiU9G1Jjwm9MJPUdaYnnb1WnBg7Iy1NC6hA+D+Gt0BmgKpMODVis9rFzb9g9zwlhoqdO4Kd8X\\\\\\\/BUQnWq\\\\\\\/KMBC4RWQiifXqQtZ3sQuJFOlhrUhMIzJxYa9VeOhepJPJN4Eh+Kv+7AkbFQGGp13MPyT0N3Tn0bX9wbtzjHAnSrukDRgV5KPdyMXWD9HvTeaqBBUqcuQN\\\\\\\/FvX3dfNhZmL6k9zyxzrNAy31rgzl5C8Zb7eQjc3+sR9das0VjDmjiqKtXzGZMMXawYdg77IDrtTAe1oT2VhOh2ZZdd7XKgOuXXMyVnbc7j+VT64hQqszl2QdgXAFOTzit5fNgOZd2QNICGnd9avcNM4EQ3KWV95kP5QJSupcff+lSYEVBbqxn29j5R\\\\\\\/PXZVP6dH3\\\\\\\/Qpb53\\\\\\\/Pw2UBQ4L1AQQh\\\\\\\/lxSmjHpV9UwCktBi+\\\\\\\/Z8X0lISDU2lhVqNixmb3JuRdn\\\\\\\/jK\\\\\\\/JFOEgPzcCqGyh3EAJvpmbR4p8KUoM\\\\\\\/C94xrecOrqCXdb7leN6GhYp9DLHAf1Hvj7q3SuzQolutC+Fgh17E8RrqyNJtpy4yiVF+P49Hw\\\\\\\/kYjBcz13ZIvf01pc5PupfYRUIy8PItNvOJC2ELAEMXiCW3oHLS2XyBWgQmC16MYTgTd6mMjOq2Aoz22d2jgLgvUWjjJM\\\\\\\/yAIbIepzz+cScv+W6VoVII2eiNl6opLuuuaFJCG35GTyLD1lNF+x35z9XE2bEoMekgpFLPqX\\\\\\\/ZD4a1P9Y+QqeMGVMOU+BHYYJUL1QQTtm7UmPscrWAGtaQp3bOdHMfAeLqg9\\\\\\\/tORreN0zOsoYA4IbWdeKxyMLr6GAtH\\\\\\\/bWUm3U3IynVX9byocU23jwmeGx4xwTYEw2vDAgCusyQteG1zUtQk7h8f\\\\\\\/vH++BZEYDH2PhrMUNcUk3H\\\\\\\/oDZLI1nF3Tposi\\\\\\\/kP\\\\\\\/etmFiRkGKHWfEM8h2+MbBl8lVNLKrHJy2BY9qWGEo7d+aWr\\\\\\\/MARXS0virPKpv8a+JFjAIVv4sOvykhxfSlJNPvddcXEcLh9AnYIkEQTfYtQBOhwDWUd1LHUXTR1Rwgyi6Rw9acAahcTXXm9UTtQewO5uYRDlAcNe39NKwVyG9vLPkgNBsgITmEr8EWJ6V2btS2OCrMPVlkJWrUrFt02y+2LBglN9Bn7j0QxLer80V\\\\\\\/gxCcvlYjH2o0DeQ3kcglRRyH+gk7zUFdynVe1uRbEOHhPOk3B5U0evat7j+TLMptz\\\\\\\/LQODe7GnmkfHfsOaJSbHtcnVI7tstxyjCvDN\\\\\\\/vTTWM\\\\\\\/zEnM2NTIN4w7t2JifLTnJiQTsWJbbBT5E07XN8a6vqInqtgDhGbaS4iqAUs7KzeV7ofiiuBhQZD9u26C+8Rgd3IjUYLH+fACIVKwkDfd1o1DzDf1FBp9PqQFflb+A4mjMRkzujj9eS+EdJ87TK0qr4PmJ8vsRZLYNrL6YcjkNY1mgvBHT6LY2NFWtWUg9QB4lGznmx3W5Tu17k7JWQXQCFvrvBra0ehPQmMgSjn8G80cZOCM94K1hyIdwU3tvF\\\\\\\/2CVJEEvUZTCM1obWzXsijhFvdaKmdEQbhy0YIVQy+qpH9\\\\\\\/Y9CrbAFJUHdmKFq4OIEWXKf4w5U07Uq5QpxmV7GTgWg+\\\\\\\/CSLEE\\\\\\\/z7tuM9iF\\\\\\\/ttkVR1OoyKnTNtnKBBFREIRTa+zRcCzZb0NWkKIfP5VemJyYaUb8H1tihNOXG9SL3RwZZXQ0Y9DK8n+N9RP7OtC3Y1US5fC0VMLifaiNy\\\\\\\/FAIOEJ24KgMMhKO2s1+ZsIaR6h25wKbzLm5PCP10ioMBHn\\\\\\\/qt35W8VCAZNw\\\\\\\/AvZ03j2vv5RzZnV3SSgmfRy+SeRwL2m9SczbIVGz8FhCrlnCTZezlmS5tt0F9XCJ0KSiJYUrUO7HbRaaTF+mBTDv8skRxdKOKXqi8pgkkLZR+YK+SMq6jTdEgyrlKHvTQtIiZS53CXJg2Hu\\\\\\\/Q1b+M2gYNdxs1+n3oiamgzK54ZfW0\\\\\\\/zyeNjH9ee94Tr7wU0zJs8IkaYOyk9NjaagTHJmXMzsZoEPGTH\\\\\\\/qMLKnRSN2Qb1mipJTHHu+ay6rIWEJ773QnUUmRY7lRSsm0clGPsrwmlE8xb5xjuqF2IzG2SzhLdSgFiinRkMjHJ+zF0YMNYqsxylrRDs7+LiEvyEXl9wK3Ze6AaKIG9YaCpG1O5VAn7rjYkxLbnl94vru2VWd1OaHD8vy4svwOxbjK9IsTEyySJ3o0uLdFOq2pZsg2XEYYh3SwuTgcrfG2fi9PUCxGPHAb1gnc2TOAB+Nh5MtrdFx\\\\\\\/eTuJNrC8pHQZUI1ogNEXaboMQkRd30HIfFcCmknpRCAeaxqMRCYI8t6\\\\\\\/ArcXI0\\\\\\\/Pp6llYZ2QVpwQ1JXu+TFMyHXsp1tW3flO6sKB65xCN7S2uqq8HsFtrbgTdBX8FqvpFcm09J3AkF6OHefIRoLhSELu+rkkW0YmdJqWzWuYPf5WfJo11ddn2PKAWRyBPnVlNiQoov9\\\\\\\/ieL9eRKCjLv9pzKY1BYGNHwP2Zk2MxdppYkikvGcEqVad7G5H2AhYwkAR84cQWYkcuV8k4vzG1tdo4UzvdDKPJBIWH2qqGx7c3Ph5h5a1l1PBW8w8gjnXh77fmDKxvpxXwqjGuV\\\\\\\/9nT+BPwnM6jywz4eFdANzUf\\\\\\\/FNnX7jXUVK1h4heVxujAX9kcWEa7NdU0iXPs7Ok99vf1awkD9M5h4IGFd+DtiaXGO6FwII79SjXKB4cqfOwvXLMK58i6tV76CT8GEhSbjSk7yjxpadnWpT\\\\\\\/vsuHF02EaKmogdINgwpbuPkGmu6Liwg+hzcF5kC+CDtzsAlT8O1ennG77tKJM2fYHH23N4V+54Os4e9HrwFioLMzO6vt+5xS4lORy6sArwbeEg6p08cPvzFg9SvXOvF6aE4VBvuMQfOVUj0\\\\\\\/L0kEg8hWPiPVg76iqtL6c\\\\\\\/zzIvkloOp3IiCdfSFIO+EanVpgMWtKAjF6Gqakj+omHRG28agGypkyxe30yBRpMf78L+gDsWARgwFaUWPZq9nDANd5eO1vergpIZbISpzlpabNR1VGqPPJjUV2KbPR5birDcTl5Ht4\\\\\\\/BvE65fu31DKGk\\\\\\\/NqK\\\\\\\/SJ2xywYOkjvAkDo+NT0M\\\\\\\/r5bCSHTx\\\\\\\/KA+8Bg2fPQNdIdhMfVbGK1Urfzea3ihlDYDIf5Ijy0g4IwyKWLzXBUxeyRIFZtIRJ\\\\\\\/m8wEcoiO\\\\\\\/GuZpr8gsrl\\\\\\\/ZVjpkYCLJO2rwOBs5Jnnc+JyKk5hrfEivUIZQVMxFhDbmxCMic5lIB7+f09M4OE0xijKGIo+dETO7f9PDrHcvHj42eksiX0IkE5d8+Eq+11uSuJ+9tkklV+Z7iw2zWIznqHcUrwQDTIwNNjCou2aLIipHrbTG\\\\\\\/sEXfhTKNQonjxdsh37Wsp8NgitUhsr1P9SivVTXyAAvEldd2PB6cYiS2W1rJzg8VJpEwIKyYxzpuCkDHa\\\\\\\/BlXubhAPyCrs0lgMiPvL4mLqMbfQ14zShBhnT7HqpPo1zSAohuOuz0Z0YWPdpbmpgU0tsYmCac67MuM4TZMZt3glPyX7EmsaJW80X7ZWRvmnV86zIrmdW5BwKkiUk5Sk2HCZ8bi0dIXAoyIqM3yBlA4ZSnRUD3GDU8EQW\\\\\\\/L2kVzpH5QMUIHPzlkzgpNCIDB9aMu7QZnixof8Y2E6wtCeGcr9TKxTB\\\\\\\/\\\\\\\/FKIr6XsnSWWnEOPU0PqPY2V8ncwe6n2jOfIvfqu3IQikqjvXtMQusGjaS\\\\\\\/T8i2ULnGeHvTtHLoGFtia\\\\\\\/owgE56YYXUBAGhK9eE9MZ4f33Hyx2zKO7T28enHSQLJvQLSbUhQaCTSL8K+TzSP7n4N9q+w00JFUJCRysOV3iXZAegTixQFfH344bBEutSmUVJP3rrtDoampeXVQQTwxG9TwZastKcxfGFEN669dhu2IImp0+dBgPhO8UbAzGvgt85\\\\\\\/yKT4\\\\\\\/OFb90JXQJgyNREdkNJeoaq94UhHiXuHB3v02YolXzjgEQakRdfs8v5wPyXfMnqFxjx3c2KGbCkTDHpFe6ctrpxWjkcnIFP8zIy2XXDfs\\\\\\\/sLZb\\\\\\\/U1PXH57npYqUwVvleHfOWdxLIghJr9Z1ENkSBtI\\\\\\\/HvUUlO3ljXCchW+Z0\\\\\\\/yPIwuGKhK4UM3sPt+s2dMq7EHrGSckcuQkNlVhUnYpiT9cUVFdgbs+s+i0wqFboIlfDjSZNbcEuBZHLfyGuYyBpAB2RWL0zGsfx6zqS+TsobDEWHvbzv57j++blc44hHYLOUBTGFs3vNSaiPd5Hlya1YMmUSbnrmMQIQfQ7HgRTiWfkK5b2NqASEbCA8QAxdynpldGWkeogp7qNeSx+hdKIUOFxexhqof9MIv64z84l95vbJUglz+GI1xY1Nb5qJxcBtWhLTMznvn46+qQcufYIV+5lrVGonZ5+fhwpbBsk8bd\\\\\\\/DldwtvIDnUeBuZHkqWD5glFwFX1aYMuG51gn4MYvXq8cc0I8+6LspQCXwc9IMFBloqGvqnxru+r4P5+kg9qij4SoV3GuASXAhzDe8p3aIr0\\\\\\\/XH4QwuhAcHgHg4x5cu5I40FHVbghP4sKIxPzuRBpmc2831ZLv844Pej0B8VDjppD3NV4+xom7O2za23N85BFPmKemC8TZNA3mKZAFFjwNRVZzYEgKJ4utMDWGDmBHf1gdUKDxQrGbQ1hshASzHgCu+bTgXtQGOprjsdnzTSsU0yPpJfpdfXtnfzUuAD+wt2cELzbBkx+z+\\\\\\\/xs9\\\\\\\/pw4mj1OWLpbPWR8OHwt0hM9sGLz0VARBhZo6wtX5Up9WSyCV2Dc8WfWi7B5ANdRcdE3cns5+VOKthLJQ\\\\\\\/ioayJmpdeL+f2LlPCiKsnzoY2cbK1p4X\\\\\\\/bXFyn3XcjvJjUKqs9747juym6+CB9xG5Szw8qSK\\\\\\\/aihnCpJziF08biYJT1RV+k8tEoJLwC3SfFQsCfscS42EjviIHl8ojDs8FV7GugG6R7KgPvUyolKxYlxDvCHca8zsRRqeBR7GLXGdG0YdmDlO+52RTRRr2gnyAzfDi+JWI4PBIL0hNyxi2jVb0tAybYy1DdjYhFL\\\\\\\/6Qnz8GIAkXHHS23cypmJsDuFT0t2ek78RRLJk9iUMMSRDPSpgW73nr0hjpQdXZsSxQxl8Ry2e9y85ro62lX\\\\\\\/hWKFJ+02O5VyW4kUHE\\\\\\\/ScpQAHUP1PQ0q2Xffw3LjWRgnNh4OGD2DFlo0sos2mwBi7KoKhzYxmLCzDrL1jts\\\\\\\/rcU1HeV9\\\\\\\/H72LrVLzL4bRA4j4YkFmbe1FUGoHCUYcsIfXyVIl2JC2K3czPzr7\\\\\\\/djK2wbXUXaNmZ8r5stzGM9IsbL8oalDIVNY\\\\\\\/oe+drQM29+\\\\\\\/5Wn9q0hA58otCX4FCBvlp6AzkRprg\\\\\\\/n0esUYAcbJCTt0U2+qf7YkGeXu7xtTbK9X15\\\\\\\/KFIs4mTYeXy95m7JJV722hljFGCv48vulL\\\\\\\/Av\\\\\\\/W44scEpsBhB4gpvK9Qh02XIyOyqzgNb20UIYDeZuKf2oZLl\\\\\\\/R2ow2Uzd69jDHtlHwmlTXyGWIdpDly2qBeWN2CEN1Nhz0i4ld62\\\\\\\/P1FwTs1nPhHEMEMhzKkDdV91GXFgH0\\\\\\\/TxML1aQJPKsNK6aEkcQlGTBt43bPussLm9czMeX9MgjD6Gh10BcY8IKGATzKc002np5h4Mrwrto2WAKigdlH3oxmYeP60Wmw9YE9dQqkN0\\\\\\\/WKrBEugaseG7cIDt\\\\\\\/ZjQKkDJWrOo\\\\\\\/spRAqfiDX5cvsreeNu2IrzBGsKPLDco9n6lkn+RsWC7j3igoq9DqQzUrSE\\\\\\\/ZNzqb9unLO5OM9pWF8LVwGkWWOoJyuZfFUNC4gQhrpJUUj1Xo5lkr7UvOpEjWzbZAV\\\\\\\/J5T\\\\\\\/s4y50\\\\\\\/Y8U9AxfHnyU0LWxNgE5Jn8UUOAPHBYA+znDEeoqLFSR6vs9opqixJ1+RT\\\\\\\/EqCXNTCiePMtg\\\\\\\/Ug+gJ2GAy5M0MVeilg\\\\\\\/vRlDtf0OcEWgj1NJD+pmO3M4tq\\\\\\\/UWfYtNb+rVFtPK7ggUkU1AYT022\\\\\\\/WU8e27mi9xCYBnNVh2JmbZccs45fIvHJEFg+HcB2OG1Zb5dP9UaTPRgadsAo3+UA0EUXfXxsk7S8Ua7m1DQlzntiridvqIR4l9Qyq2CN+axGK7q1i9Te3FRMXzkQoRftUNtMe18Yw1sQoN0kgnF3ceXOLVrpz91JNS\\\\\\\/MsWnobvmi5r24gxZrkjfqP0uBxVe3nnu3dNL2GLJVt5RBiblwZ5SmtDyJ5cmUsKdIXD6UTx+FXqOMN\\\\\\\/TFgHlF40d2EOw2+PpfvppA8T506nLrstUERN\\\\\\\/Csl\\\\\\\/l2+o+BJL+PHQ\\\\\\\/plpp0BxPzLePORrPM2o4Iigz0FfWqodo3Hi2wtRofPSaPfLziKEUAlUSjYgCF0eA0IdsJkQML3I5SUnfQiSnZL9yVOTmDzOxixu4UG3vd4d2B+67YZfBD5h0SCemG9RJujrZF5lNQmCUfeX01UiEFSnegqiJIhenKzH7urZjnm3b0Z3xrR3dPevrwPzmigHKSLAVAwXtgFENIOfNZVrAaEdadSszAl5+R9+o\\\\\\\/NuTJtmyXvuHVhOxKTZUeyGoGWOVw+GYKjxJ5ZbG3E3MIngZZmr\\\\\\\/wO6IqhLOl3wlE6T+oQkCgMdpcjHm4c73s7DeY9XXsiEyCpEqxItHlCaa\\\\\\\/QNSdpSmVBsaDOwQKKqaGUqiPTuX5TXaZQTwVlNPr7\\\\\\\/difb3JJ3H+mqXphVDIBtTab3NSMC9Pg9zVUfd+oAd0xFDpeHcrEJFx5Yo0sDYp32zmhD\\\\\\\/wHk0lUSxJLZK2+pxDLivk1wuh+ghahBNxV5KVV8MOrHJTRKisHXZXlYOYzUcXDJ6N2Vonol+hAgfsvG8tGloakudCZnCAP8dsQ3g31y\\\\\\\/bSo7t2QsFadjEN+VHj+JysrBgRfxrSgMkHCz68+iiov2HD0mDY6\\\\\\\/1XEp5V1VxzumjW2dqUGKcqnr0\\\\\\\/vp62M2czxo1U+IYembYbpt+I75zc5gCxOtwtgeaswLFRlLKd7Umf0zyGCsBhnxm0Wgt1zdyJ82mqxRNZR+RpHPg552sT1MPJM+bEbgCEgOvxGDWhLFS9TMhA7G0lrajvGinu5aw7Y\\\\\\\/JrybvsxLqnI+fb732UKs9wxN3tu6aSEBfl7ht5gtizmzePNCAK4L0WpF3guykTJmn9hnzqy3WCJMPa2z25JiMqrbYzCehS9V9kIld3uwDGbHJ0s8nqU3selmJRHFvc5H\\\\\\\/C8ykUlKd+n\\\\\\\/kU2VDyNgfAh\\\\\\\/HcabM7T9g9YPxHRrUdElP2KJz3yNscQXK4QuFcGa8UzZwgww4HOTe08A\\\\\\\/9kGEovZpoTOzvtlf6qp7BbCxD4HQcUaKn6RIIRm1R88UetQLy1tCtxg8yuQZdJVqB3s3lFiuGKD+0sCG1QxhHDWuPbtV0SncGPfiLDh\\\\\\\/ROE4awWdiXJyyl4bLbMGkStir28KrZ\\\\\\\/CjyWKN5WCAMl3dRtdnOILYhFIagJKQ8c8LXztmmolRmEHCLxo0u\\\\\\\/Oox+z7lCvtfqVUlqUQLVHuXfAqd5IG9ixAuhp\\\\\\\/MFrGqNb3NJbMTvzLFgmX4njedFhRfPmreOPD141r5nHwoQ7IIt2Q3t7+q+3WozfDs9fFRZLgRmxFVeEWppjNdrAC0z7Ouv8vkuy+zbTquToQwEgyhaT4aOgaMY1rxBgHUY3GZtQKblrDU08t1nYJRHqHsl5YPYhAPitv4DGfHMVHb6PlrF1bCcGDAxf20HqrROWYYDlRyzfKOYbwCPOHnA1x6EDF7qv3C361YmM4jMWS7iHXugowHeEpTmJHZIGd\\\\\\\/Ei2au\\\\\\\/I4poGdjkkJzOcafs+TREjG2R3IG7cqxglB1BJxskdzgYuR2wipjeCEp+MfXRgo7SCsiwFERITqp+2JnGSlBN6U08bS+3zQrRJ1TnLLIqVbq38PZ5Cblp1HiFONH\\\\\\\/T486WyuCcipDhTQuK12fchqi9sYDeGGnfAMcO+P+uvfs+QxDXWhUiZskYIzw6QL7+gmOci4bGC+X7QqLcPZ1M47TDMFFG0QOOAuNY2GT9ef+4EyRnsa6yg9ouK2Uxz2t\\\\\\\/mVH3jcTjT3UEnZf6Atgn5u48qWCgIkpCtL1nK0nVXGWqyJJ7cOX8WQwIxrBofj1xBWQiUmaNpNsASJsMmqLWQwi+BuIYZhqxnGfO2ErbGKoM+4sN1MEVs3z0S87bx95bCeZIlH7UyRlGtvbJ1qxzFcO01yFHtCw1HdB5pYy0Lf6igEWnhu3SXpclVlVaHLNn5NvdcLqIZQGZT\\\\\\\/SeJT8WkaW6AZ444FS4pUWx\\\\\\\/uF88u3zWgrck0SR3w+GsIpgusCDcDO3ZcbK+Oijnu8q8W++9BOU7OvKIbJhLldt+ArUWDQzocLd9zT3FuHxwFojGe7mYoSh4+D1r8SrimbrX5Gdpm54jys92uhogfBLnhPBeDXYpfkT1kklyJIJPtOqR1W2im8c3+Mn4pM0qHAyY\\\\\\\/5qrWyxH+lO7jlRIpyRqUlV02oHg0NrH5PN0ydJMCdGCt4PdEjRExB5SVlaS\\\\\\\/ldFx1j5EP0QHbdrqqRCJ+LH+6rl5Eug0Ce5SqGIOMA6mi5FETXoZe7xfDlbVud\\\\\\\/ExCB1efIyT\\\\\\\/Cs2MdOFBr8WeBcDHyhTtVYpIaxUqjGT5QLYowS3qPGRkwoeMElWR8Z66aHmw74jgDQNO0hAdv5vvo5r6ztOJIZv5g+ikTpzW3HwYTpjurD96Hoh2Yiba8ehU3OPZHASYmmHKfdwOTu0\\\\\\\/N1g9USvvpyN0IlXHAv7IxVUuB4VoAGJ6kdjTkRj\\\\\\\/eHoy+ZEJZ0juK0JnzVWGQYcxkTPTEtVaU0tAddnlO9zSkDAEUubuDBFifuzm4rnYxceUCIciHj1ywOJD2kq4LonueAkB5tign1xwbVa2G9iZPdPq7grRnYvBy\\\\\\\/xOS7PijHyuxmMGlkYFRUrbnZF\\\\\\\/FCQpvVcAjFw9YDKs45ImACzgfN2qs3MrI+RBvgxpVrhemlfdsUSPlJWJXXkH+gXIqCp3F158TKe+WZfebqnEJhHNRBYjp6CV+BwAYmZH98b9MBnrnNm1YGzxo+s9KwO04W3SbFPNpFYFp6b\\\\\\\/DCaR2nqr9c7XzS\\\\\\\/wRB\\\\\\\/oEAlTP9e176OpEsN3qa0BHYOzkJYJF8hnXVcPYu5kY8ra0mw7jaDArpedcngFq4j2m1Or33JvVVh96Zi30Bo\\\\\\\/RLDkOOXhyyTww6RzRSDLWsBx1kzxemzc5okNYWBYHYSz0ugGc5I1LmVhHqLKfivUtex\\\\\\\/7twZ2tkU5djeAUuUigHxFZ8FI8F7EQl6xoPyRL+fSgAZytdsLspBURd+EFrxbp82iYTy7SzOcX7bKYkgUWjuegPNLTM\\\\\\\/IRVXaUadH5netc31bM2Oci7oaMnLLEghnlXGmk2lhUz9+0WSznqkCGBMp8I2lwZikz6nl9l5owUOLttZA97n2gwQB7m2uFkfM5qnzJJ5cfwJakwou\\\\\\\/AceLHpibDAs8eNPW4MllADJVWUIwKKDm7DnsgShZCm9hstR0g+0KhNyciURm2ubsUQ1NA37PsTRPk44lZLa5qK8njJOCF9voz0vf+SWvNViXbOUh+SBN\\\\\\\/H3oLqP8tcUPH5\\\\\\\/2k8jHLHmLuGbsihLBM10r2F1mtyMvT1ZpzLVL01ziQfmMBmd4H2UBfwkYSHdXGzhtXvmtX8r6JBPrwOpfrwQnOZiGi8F1bN40wL0ZZSKIrYRSLH4PDtWDMSnPvia067NN0xZZTUxZrJvn+q6HHYNlDTUfjGI1Dn+d8D6HsJmtLnxCnKm+Bto1ZGHWvG1JHSQIhXH5xp2ktqtHc+kAVy59qP\\\\\\\/d5QeRVvbWMFEN72ykcLqndXnt5IcHUMAAvPNHFs5osZ8UWizXMbK68EQHgQTYvjVtFAB0gGWtxTVpvwQEtx2Qr5VTX\\\\\\\/DnQP2QKhmKpHbumx5iO+yPT2\\\\\\\/ExLxVx1q+R1Yw5aqH78EdhfevTK+3H\\\\\\\/PW48jnfqhR9awCZJhS+XLjNPt7nglX5YG8OFP0Vnp\\\\\\\/Gg5ZQqVhDkgtTtch\\\\\\\/0nXL2GV3sgjw5d\\\\\\\/MdDj\\\\\\\/nBOqcT0IJRMHgJWQ\\\\\\\/Bozdt\\\\\\\/Sb8XIrRlnuipqxO7+7J+sY5+\\\\\\\/IwqruRnLg3W27d9uC3ECkdbRdifuZt3KKDhtBgvWF4XDYsC4DAqvMIZmwd1XTRDqLviy+TzMjvkf0IzD43v7T+0Ui+yQMO9eckcMQKbwdIKI\\\\\\\/s+3SHQ0aBjWUwU4VVck6Oz5xTI7dm9q0meLhtRJgiePVl6Ji3HLz6tI1\\\\\\\/yPL9FEuw5UC3tka1CDyjn7so3hQIgBFpcD6vsmbcwvovewX1AjG01\\\\\\\/OZF6BgPTbI3Q08f3tKhR2mFhaA76hlYKqCjDrP+uMXhhV3vlYe0GeDtyO+89dZo64tzr3kgJ+kX5VICksMLBpDLcNdxNARaGs4fmPxModdRHISQofcD1eRrZUWYBpCLdZwAx\\\\\\\/mZcpq1Try36JB0hKByz55w\\\\\\\/iZUDVdHEj8KbOK0xrSbp\\\\\\\/AvjOqWq00OZJhLu+xVMSODgLQj6BZr\\\\\\\/EYhaBeqV7td0uCp1oLRCsvJCHdMpUG61rrzOyorB9nR4uGuKpp4+n9sndFFYb++wex4v1GGhgIOcqcTJYqqh2HCMBYXaHq9V6n\\\\\\\/Q1hQT608KhBH43RVpvYwBsi2LpEkDVXUagp2DFUK+B8unUsNWo1YK5EU2yslmHOi7rejdS5ZDyhf4U1FZxFxacQGPpMF3ra27EqaVdYqPZGo3Du6P\\\\\\\/0OxVqybDcxVc3E8fyZu7Uxb8Q4lHHcuM9AHDy1yR\\\\\\\/YZLv95dN1\\\\\\\/0iT+Zc4vTT1Crg3m\\\\\\\/PtWjf+e+K8JvOu1Z35eWZxzaA0b1vXY\\\\\\\/904tnjzSzRPu6QpGUxyKDTnKL2egRCuEBAAKZfnJHlG9LAjmuwu+awqyUJRzKNstwfk6I11kY1P1qE5WjyBUGY9ad6ICy\\\\\\\/6A+CFOO\\\\\\\/q2AoR8rAOCu\\\\\\\/8K0G+L8t2Rs2pbJC5CubK\\\\\\\/oXCTF\\\\\\\/ZPqSdT7B2KCFk8g5ydv8ewCzeEBYW+MTajMDPhZBtofXa7ruHUQ+Ft+ujllLxkzxoWvnqIdDk\\\\\\\/OuOPqM\\\\\\\/qX1Z1L\\\\\\\/AE+bPtCu+EYB2AWcBtZJPchP8ug4Usv8ZiAV\\\\\\\/\\\\\\\/PM05onYPbO5SH1jWXwSAwCKbRC61SN7t7q9pvtimGs3FN79vgeItHpwoZdc7kt52TfTk50YAQ8VgXbMclxXJt3hPii1in4j\\\\\\\/0mFXeBLTTNSVIfTGgXYDH7QB7fg4L7Idy8sjoZrkIL+\\\\\\\/MEFlB\\\\\\\/71qfB7nGKUTG056lnqPWyNPxIISAx7HNAQ1HAVY\\\\\\\/XGq\\\\\\\/sgvZgbce\\\\\\\/DZJY0RC2JLlLUbEL+zb5YX5IXhUE5Qfni9p0MdWyzxV78+cLFkks7362OE\\\\\\\/fXEOnb+zuBNWZJSlpENeZGqWkCOe6pW9OGYodAhNVX1zwe8BDqzmUa424A0FxoTsh8sxQrIigSft04wydmFu\\\\\\\/tsUx9VkB4mc\\\\\\\/ieK1jhQk75pY3nA2kbXH8IJg6tYF8zbsgH7ohL06pEa5zqhnh2FUvVMKv1N9oCOl1gGDKuGYhHW47C7spkB9Ib9ywh9D6SRFgjF8WUvsFbH4PyI04goZrEKDW1IJQuGXEzj4htCNYlFJNX\\\\\\\/olh5C1Vne5rljW8tjGe8Z10nt2D3tHHUoV5tx1sEM9Vn75ddqPBGnFvEmfoRT1M9ZAOkrNGiRmW2TBDlcq2lgVK9LR00eUx9lpzBFGZLSc+0k\\\\\\\/\\\\\\\/l+OZJ\\\\\\\/cuiXUtTdDbao\\\\\\\/KCXALULdd4QTNA8KRS7I0sC9BhVl5Jsle4O7YvQkt2jxbLMnoc4mbI2kb\\\\\\\/fhNoZM\\\\\\\/CTxMRowr6cZAUaPY0fpffK7QER8d7tkgdNGcCOkUna\\\\\\\/jfyM8R6KYWx2HyWjbs1a80sZbKtGSwv70NpopA\\\\\\\/h7t4+QU6uHkhXA0Ew4nun0XIyJT7VLLv7uYkWXKHft3RkMgNIgIp3IUGOa7JG+rW7wNKvd5KHnJXy2ym6xKUqg+mHD92wkV\\\\\\\/UJc4DpL73l2rghw7ZSXTZziJBLNqzckYg1dDzBzc8Uq2cCwJWLHxKvVwFS\\\\\\\/oHFaV0VWx4BLBMHmjTlido+o9O3JCIJOWHbJSC8GaQcTXFkIrw5p1o23h3p4ZDtkokcSIqaq0XM4ausatzowTD5bnPnkLlATYiqCmAOPJgzCoz\\\\\\\/\\\\\\\/mRR+CCSWFUunm11VUucbeFtr6FXhfxKRBoYhMaBbcPzaMWDqf7Ar0\\\\\\\/FUEKf9CqaKrXmqKhjCwwsGpBiZNyM63nZxdzT\\\\\\\/p4WOJRILWCGDZqHnSl53iqmubth+wIwdIbiyseJhFw9lDdv63RwrdOGOvSGAKpspLVUmUnm6ehdKofmjDyRWnWKZOGyy1uTp8wetMdiCMe3BKlbCZ1ugLAoAVY0a4u38EfIJEi2GhUlZFHa+o0LHFqytULjHvrQ1y4UN\\\\\\\/HNSdti3\\\\\\\/Bu\\\\\\\/0sZ\\\\\\\/NIJZU5H9mL3GhSB9MG6i4HvfHuC0NHvnjEH2RRHDMixLlFZh75TkRN6Dv0aa9YU4g54yA0cojp5MeFkfquT8ERllUvZpx1zRFvM8MZ3RJxLbHe02BMlwhfEKVeYPBHbmXVS\\\\\\\/rIgLhw\\\\\\\/iHBXQp+BeGMjXePhwX9uXSVRPVP\\\\\\\/IFQQhs2T99Crbdeu7SyvLdRK9ZT5iBRF+HdQ2l62MLVeVkErzQTtksS0zOR+5FYIpoRvWsY+4FYWgt6qb225HGBD24U38lctEl4erpIJTEFuIavS4dIQM9vQ8SRkh1VkY9Ug85bMMfIEIjPph+gPvBFK6ZR+2slgFQrri2OEXmmqjNNNGPajXGjIGPmKEot6z9sJFt7MAH7zVMEYKQfGZ5QlE5zD8yvjvOoOTO+oZS47cGq9EJO49fdPXrJtfyQ1FXT6AKJu5SBTM7JEqNgwUhRjd6QkUlL3y++Vrru0J7vc8KJAEJvQGFxLGl4Pkp\\\\\\\/22WldBFQyXSObJpKbF147f9R80ptpXu5CFDQ68mehPheX4TXw3SwuvCVhq9IYZ75OVZV6+V5pmIJPnjvU2gL8PGs\\\\\\\/SUk+b76nhNxea2Xu4VxAeV9eIu4Y9H5gQr93twRyoXbRa9usmthg+RUR\\\\\\\/19lod1tJTmCaA9cY6tC9knooA77PAej1jOVWeAhf\\\\\\\/bFlZSe+aHBmlB7JDoA72w5DMfu5De3ANlFRMKZ3xQqxZB71lfivCwi88nvA5akpoYwVVKUeuEnEc91vaVuFJXhD4ZqlfLP1wT+oSbFzRx2Jtv7gfIZnzCm0I7AfzuegKieTxJIbqmO7ODU+HOUmLPFGJ8wfrOA9jJFtoObXCnReGeESj7n4xmQ9m9Q7VtBSUuykxkAr4vuL7i\\\\\\\/1vpWVXvA58\\\\\\\/IjUKNWLCKs7xAdKiCBeLgqfRs6Ccg6m\\\\\\\/n\\\\\\\/IfMnQkICFrSkUhnAPPp\\\\\\\/yWjEImYiF0IazfX7z\\\\\\\/5c46tFGnnsgcEzPxBLqY+br++3cVZ4aOtczv8lvXWwsap5h1Y9CS4PtQmqoR2f9KJeWBl63En7lc1gpj1yYOKkqeasg3xCj1lBX\\\\\\\/KXZQ+I3Zlsw2GtoG5tQKelG+Kjm5UmF2yw\\\\\\\/upEQDRLw8Qe8SbtB03g4gmz8Iolg3aBPeMwcgi1KIUqwow0JybKJT3RIs4xBa8vT7ViC8E7EfCBYeU3GtAjmtrDyXkNN4v\\\\\\\/YTi5EtRihH2OktUi5pvsMjHGs9gL7DYERHOMzH6LR45Zxga6wneJ2RqLZBDu6HmepbnxuLqIJpPLdIR8lY+bG5izCw41ka1D\\\\\\\/1GBU1XQyhKRjTeMjF393ipbyfowxHU0Lt\\\\\\\/UNixjv1ngXpgP9VqQbU+Rvv\\\\\\\/IHxR4ba3DoWC4VYbWT9tAMEucH\\\\\\\/ZfWBopaQch67oU0hCuG499bHacfP9nvy3EsaFjwb\\\\\\\/VNjgysNhxSFONDnXKV27iK4h\\\\\\\/vwRv9fVhkh+Ig9oxeTtEhVNnhX5KNRs6A7Tn1nolvxk4x0H+ZuPtYBWGHWOJuR3uUiU2Ne3Hhb6o7aojyyeTktpbYp9lkrnot5zx\\\\\\\/Wzab6d3FchFH+nILGuZTQBSr\\\\\\\/koj1ivncm3RfPk8TrNzVua+0nRXWJU3K\\\\\\\/vIrqdluGHmeRt2DUhIp89aclCidoKeTgeJk32ECnB7RfAZ\\\\\\\/zZUK1Ej7+cGFGbKuJhLmcx8MOw53iWJtVzQIeUwlgSQ3xZ\\\\\\\/Z7dNvmCh3LC2C3XZ6OSjr3Ns2P1FO73r5xIfDSvh8teU1gYu47TauMDRoiNwbr19jF04y5zryedgZtZHSeBbWg95kdRKVK1TX2VV3VYH4Peh5EBa7w7KjrtQCnV0IBeVRj\\\\\\\/pT7iLYktYiAls65hOZj7Kqtl+mFJT7WveRtq6KTQAX4TVaVUzHOC1ttw\\\\\\\/NwMAcWau0pvau02ArtFtniOutMwZOL0WCLGY8zxPwKq7LpsULCMT1eVtIlS\\\\\\\/eOiz5jabIGBb+F4C22H94JBUUEENH83e2nYgtc4r9Ntge5+3DnhM1G\\\\\\\/Z10Islh1fu6BImJrIMONt4FHg3tnyG9jJnpd+UVANuLXjleqHFLcqWWmLvqnZhTfEWjhPJncITbYznl2XQd\\\\\\\/i\\\\\\\/A8vDARZQ8Fuvc19ZY2hXJo6IR9BDhI46HuL0b6NS4MYdrGqzm\\\\\\\/U8NM5oS3uM8aDxEVQ7NfDRO3vn1sDG9pVxNOMdrDccsUa7v4pG7SkZCHDUNQXJK0SuJxM1JmZQ96vTmy4QwK8BcPdIYSnndzHyyQEWbKr6h3It3PWGwZD1XdU895P6JsUuff3ONidkvMPfAFpD6POWAUmZYH+pAM3ffbHWHr5WhjkascIwT8GE5634TqFlCbFdK0vjITr7CZ+vLOInW+pEzwEUBOu+nlELFzLPD2YG7gHX\\\\\\\/zTe5mrXPSgNm7Qo4Mf5X+3Z\\\\\\\/2FFTjUwZ5zhEY3JWgts\\\\\\\/Piz9pPLNL4JheJGQKFFikuhJHnLC8Ya5ZI9v9DxsYDkVFfyyWGF8dPTX9fEr1Fedgc0qnLQ16RXHe5c17oXDF2fTSIOyNQ88ymtSyg0PHpQQ1mUNVuMffC7uy7qyeMA9jTBXMjkFxLnlUVQMaUmAxhRioLCfC4fp3JG5n5egWP8mvEUlPQa3YUXBBpPT\\\\\\\/8CjmLZCza88XuyKclc+hVRvIuHDVtFEbv1EyiuOhLOVrhOk2fDVyBwSzvTVCZSFkpYNu+nqQEE9\\\\\\\/VF1yF96OsjmHA7Qhhzm0RFOxBinTIBfVT\\\\\\\/tO70qywLZRlGFZxL7nn6byHlf4IP9H\\\\\\\/cmguGOVmgJgw0rO3eT7gMYqHeY52C1WrNdHXfP0sxEu8t3C9JwJH8ld9D7HRvebQeJ6kjDamB\\\\\\\/A5NkGNYvkhGMydRi0rCXosjj8tvQXaL\\\\\\\/+Z2ieToHoV5CqBr9cWMXjrAycVaakZFyE\\\\\\\/Yh0A8mrK0NAkZno54eBG06GMeGb7kKHVkJmZFvLcfg2T6rabiDBC2nVW8NJfRWCk38VBZLJCrdam4nW6NQVXm0ct6pZZGJMkoQ1jWgUu8vCQwkx5Q+GXNRvJL\\\\\\\/sJYvcwW\\\\\\\/mUEwugmMApBHtvgg7oKT\\\\\\\/0Z80FdOq5I3BBP\\\\\\\/krwq84hSZd19iygWoY35C9T4v4NGF08wMLPDO3ZqP00yK7RpOJ+KeRP5cgCZrgxZ9izt6n\\\\\\\/4hMrCX9\\\\\\\/PpWl6uOrMr5KGBYrxTFW1V2bm\\\\\\\/KbrX0oJqiILzs60KhnlhV6rV8lGBfjGf\\\\\\\/bcJETjbTus3p7\\\\\\\/DppMo+O3E32xf8o1GuVzoakAF9JG7BWahJNlgWxR6j68qp9Ce\\\\\\\/f+I7w\\\\\\\/2PTFCg3MW83bRwzx39TV6u6aWkKabeRgSLtZYpTnJsCJe+Jw99KV2UDbjo107CNbgs8W0lnxQFRvwruFY8johdWU51SWROnq+RGY\\\\\\\/vaiyoIzW\\\\\\\/22kQirCY\\\\\\\/4YOHg3+ce3CgMIqtbJjEud\\\\\\\/aloWQHRex3hzgJqVx6jQpKIjGcexrim+lVk4OzDA1SU4KtIORtOOKj5AVj3EwVXcS9a7+ak4IPAZ1q1RmYXiffntCODCOQvyI6GXjwMn48msNEFnMb0IQGqsR2vcKGKGO5YHcB+4Zi+1e7v4hzk6MvOVsY9woQdNDW8UO3jmwK0q+L3WGGPXXLNti3Adn3NCWc2dc4kQAK49dofk6s7GRQx0KDs2stVubfcaZU4g8q9eOqHrEvR+ctDeMqtBli4uKVFVVEY\\\\\\\/c2s+Vcc1yDug1nI3F3gbCwyxD+8vUttdL921gi\\\\\\\/FEdoNbnEyLOW0R\\\\\\\/vDBCY8F0wtounF9koAhLlEEBTwsD8dmvpvFVCxlxP150bYntG5ZVm4t2B6R0RIMuv0v17IcI8rwKKIYxQI1JDF6OPozO5DfFO7UP39XGpyH7+nBfyYjNHNBpAW1Us1BSLeXX3E80uWSFOSejsDZtcIc+YIH841wkbs+THCRAVvOLuGI8Iz9b839gvnGQ0J2GNZVoof0QZgoVmdQ8DSmz\\\\\\\/f4LZ80\\\\\\\/yNPYz4OxRj2emylhJLTNGQ48\\\\\\\/I7B7aE5CLAfZi\\\\\\\/j70ke31LMGTMJIOE1uliODxVQjEHnb1mThq3sIPoKF79\\\\\\\/7+ZNAavcLFqAWqtGibPjzlm4BdO+Ow5BWnDcOyyKFaTanEFWGDlW23xnrlt0wgyk+AFGFMyijoVKNbGKnm2nX9Y1E\\\\\\\/HAwO8w4fi+9RwnjEJwvbFCyP5\\\\\\\/vAknPn26Yk15H\\\\\\\/AXND0PXeZPQLNODPntc2\\\\\\\/RjNNveT\\\\\\\/tkA\\\\\\\/gKnQVBX3GmODFMA2RsMLNp6ne5HVyJKTHq26mGNPZDfEaemLdNdZWnPP9dx3I16MOVyInuKtgxS9oWF0GuoTa15Ehed2Maat4SE4SI7bsnOisQpPNmiphVvjae9ZZ\\\\\\\/wMNO3jqZYVSHo0VDkSaIRIxINozdLyFJCuxM51+ZbiTKAVRnfgcGq0m15ISxXZktbT0a4Y5erWHf8vz9D\\\\\\\/V2sCFkR3Ke\\\\\\\/Z+9mtMFBPI6k\\\u0022,\\\u0022iv\\\u0022:\\\u0022qC3RJ8pkEdonIi\\\\\\\/VLzYpTg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226474d78c3349aec60abd7e6161a37573bfc1bd058b6d3420dfc63baecf7b6b8c\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022eBAGzzx7xx43MPNxZMIOvwOb7xCSHIuQUrVUdwwpuZzze9DUB59QDIqQmxUdYBGK8zNt8gMD4GpfEOq7goCtQSQtP8fdpblwjk7S6y0\\\\\\\/esRAfxDBZALMUk2GZXwkvR6hIVgqOMCS+Z6TtLiNW+LMJxNXxfeyNTdXHChWPXk0vvcgV2+DGquFeRL5IZXz0HS9E0K9Mh7lmCdInAY27NgRjP\\\\\\\/FEYEmEK8LX8bXpy0\\\\\\\/EiaFK2kqxLvHsRjpkst0KcSdLyBJjUHUNFAdw\\\\\\\/3qG4SDQ0VhKTqXvhE6lF2FqTwzOJ2rUXVqP12WDrqJBvS7tcv8Zui8dnU3jD+nRhmuGnMcEQeuWLDsv4RlAcp83h9jreEOXMW4j35X4pOTYVtsHD9DjXu2uD6QRaV1SZwIMwfirivbdzsslDm5CCMR18tD+amNCI3V+K99I15iX84ggZZHke+yD4wbwES9cMWDQ6+h6SUO4SM5znmQl3IWVV8+jxEN8xeuJ1ICgqZzg1V3mXxobWYthhNHjB\\\\\\\/5WV32+1rEnCW07ZlclL29T5MVSt6NNmVq3y58p3VCQBO\\\\\\\/A9XpdCL5xXQLh2EzBbU9FpJPvqz3HV\\\\\\\/5nMTh41h+BpsQpznpqiopWLHY3MWOWzrWP0dNuEvEIqyrR7\\\\\\\/6JEgaeJrvGGSyrCXae03YQy4Xf2pXHb+Gkgu8xXJ\\\\\\\/owfSnguLSLFcU2TMgRSY7QbygFVX62VNBRPm75pe79Qp6F3L8SvI\\\\\\\/G4P1nmSMaT69GPXZFDQWjfM6lqdSJ6TQWUwFvzmV\\\\\\\/Ux6YuoKiZhbLI9O04kj1ORoWOSRFcpcd3YA9+XReDqTVKH1pNk3BNeN\\\\\\\/FnOLd07caZurZHbCa9ecDOCCLOqO+n7jWMmGMTPcTr1hQJ53nauBDKh53kMYF48a97ftKpGLXCs8iuzH48DaKDCfrh\\\\\\\/451weiL0FiZeuLXeCx0eoPRzfIhdIKT2USl0UionzfpDC+Rrupk1f\\\\\\\/QRPKAl5VDMyItuZM8sUh96s1Yye+bP2YpFyZYwwUgu0Krch7BCeQO\\\\\\\/5rm7kSgbu86HdvN4YamK0zVDPeXFq0WlkretAyVSAQqQNFhPryaUs9wcMhCbL2CCe5B0nr0gjvHJI44JWhGWIOCHEIilfXpE7hoJ25G+gfn\\\\\\\/kDgeCXYdicDVSJ78FdFQ9hq0s27TMZmcexSPLW9EyvRgEwwSZuFMhFRKPyArCYOj5L7jGApVpUB3qLaVZ3oJloAu8zO4RF2WjY3UtIxlPyp55Bn6ikS2\\\\\\\/xpjV+aaQosRRaFSzZ3m7nD0QBP9BwptmVmqhQPjFExi5jFDK6XPogDdMCl6zI7c8prEXEi9+7tfWzY5zZRFBraH6bOOCHZkOW8YVu7ovenLVvLUShjk8IF0zreJQ2R8KBjLnU2oYv2W9r8k\\\\\\\/MGRx+W5KACSg4Fm8sMuNEYW4t64JuyzZKEOZEGXQBKfSwihuuA8Sf6RS5upD3DvVtQ8nRmc2+s8BzqmjCYqRF6uOJRKL8omZCKUWwDZ2lJq5qcYS6OWGArb519o4\\\\\\\/xOa5Gsey0M86mqBXoTlW0Kxyzv8QSNdxUjfXPkkHSJvfDmOmFxBF3pzXLKXn7U\\\\\\\/hA2ik4Tv\\\\\\\/5gzfvy\\\\\\\/EkqDkJCOqVIbBjoXIFVtHQ6tRqUxH5bNE29E33IDR8Vvhh8BYjLCIPFWAH\\\\\\\/PQ\\\\\\\/WVWw+k251HrshcPa\\\\\\\/3A5vHIvyfwh1lfe\\\\\\\/GeDIrlADxbCGf522PSQ5cXi0Y0+0rQ5ZiAqAhPgHhxM\\\\\\\/uKp4LRw8G6SCrLUe7EOQQSEAj+scnSdxIacWCtoNgrzExEZ6wFsXffIzQZPsdT4kaUh3HdkXO9kwDug4TSrj7cmARBWovziesFn21R\\\\\\\/B6cDqI4jx\\\\\\\/GfDWvc\\\\\\\/Tg6APIuyYNAx7\\\\\\\/VQBa8P8oNJ3gBFCv0nrfCkrkflxEDlspWBkf208hvGZT46R0RFFpdQxiRvxwWE5nRsqqlGIgthU+VnJadkAvqVZAgI8yM\\\\\\\/OsylEXBKtUy8j1ZcJzmuDvaw038G2LwMIlREc9ja3SYh3ARd4Gz9FVooNWmGH48WDHKhUJI\\\u0022,\\\u0022iv\\\u0022:\\\u0022feWbaOZx8jLudqio+86UPw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022eb447757c901b8ba2132fc2f764bc17e70396573d23afee609262d8563006b1c\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460342 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/10/3207" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-gmvjZ5YEu8hdhLiNDPEPX0TcFmpotEeuByjCxNKPc9A" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/10/3207" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-d-ovzx4v-7QM4UDtjlo423ZOga5-elHcAAetWFHgmr4" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/10/3197" title="Rapid Screening for CRISPR-Directed Editing of the Drosophila Genome Using white Coconversion" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/10/3219" title="The Genetic Architecture of Noise-Induced Hearing Loss: Evidence for a Gene-by-Environment Interaction" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460342" id="node460342" data-pisa="ggg;6/10/3207" data-pisa-master="ggg;g3.116.029421" data-apath="/ggg/6/10/3207.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Whole-Genome Sequencing and iPLEX MassARRAY Genotyping Map an EMS-Induced Mutation Affecting Cell Competition in <em>Drosophila melanogaster</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Chang-Hyun Lee</span>, <span class="highwire-citation-author" data-delta="1">Gerard Rimesso</span>, <span class="highwire-citation-author" data-delta="2">David M. Reynolds</span>, <span class="highwire-citation-author" data-delta="3">Jinlu Cai</span> and <span class="highwire-citation-author" data-delta="4">Nicholas E. Baker</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3207-3217;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.029421 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460342" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Chang-Hyun Lee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Albert Einstein College of Medicine, Bronx, New York 10461</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chang-Hyun%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lee%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChang-Hyun%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gerard Rimesso </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Albert Einstein College of Medicine, Bronx, New York 10461</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gerard%2BRimesso%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rimesso%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGerard%2BRimesso%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David M. Reynolds </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Albert Einstein College of Medicine, Bronx, New York 10461</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BM.%2BReynolds%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reynolds%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BM.%2BReynolds%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jinlu Cai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Albert Einstein College of Medicine, Bronx, New York 10461</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jinlu%2BCai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cai%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJinlu%2BCai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Nicholas E. Baker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Albert Einstein College of Medicine, Bronx, New York 10461</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholas%2BE.%2BBaker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baker%20NE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicholas%2BE.%2BBaker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nicholas.baker@einstein.yu.edu" class="" data-icon-position="" data-hide-link-title="0">nicholas.baker@einstein.yu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/10/3207" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460342" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460342/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/10/3207.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460342" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460342/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/10/3207.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460342" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460342/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/10/3207.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460342" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460342/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/10/3207.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460342" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460342/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">Cell competition, the conditional loss of viable genotypes only when surrounded by other cells, is a phenomenon observed in certain genetic mosaic conditions. We conducted a chemical mutagenesis and screen to recover new mutations that affect cell competition between wild-type and RpS3 heterozygous cells. Mutations were identified by whole-genome sequencing, making use of software tools that greatly facilitate the distinction between newly induced mutations and other sources of apparent sequence polymorphism, thereby reducing false-positive and false-negative identification rates. In addition, we utilized iPLEX MassARRAY for genotyping recombinant chromosomes. These approaches permitted the mapping of a new mutation affecting cell competition when only a single allele existed, with a phenotype assessed only in genetic mosaics, without the benefit of complementation with existing mutations, deletions, or duplications. These techniques expand the utility of chemical mutagenesis and whole-genome sequencing for mutant identification. We discuss mutations in the <em>Atm</em> and <em><a href="http://flybase.org/reports/FBgn0261113.html">Xrp1</a></em> genes identified in this screen.</p></div><ul class="kwd-group"><li class="kwd"><a xmlns:default="http://www.w3.org/1999/xhtml" href="/keyword/drosophila-melanogaster" class="hw-term hw-article-keyword hw-article-keyword-drosophila-melanogaster" rel="nofollow"><em xmlns:default="http://www.w3.org/1999/xhtml">Drosophila melanogaster</em></a></li><li class="kwd"><a href="/keyword/whole-genome-sequencing" class="hw-term hw-article-keyword hw-article-keyword-whole-genome-sequencing" rel="nofollow">whole-genome sequencing</a></li><li class="kwd"><a href="/keyword/iplex-massarray" class="hw-term hw-article-keyword hw-article-keyword-iplex-massarray" rel="nofollow">iPLEX MassARRAY</a></li><li class="kwd"><a href="/keyword/cell-competition" class="hw-term hw-article-keyword hw-article-keyword-cell-competition" rel="nofollow">cell competition</a></li><li class="kwd"><a href="/keyword/xrp1" class="hw-term hw-article-keyword hw-article-keyword-xrp1" rel="nofollow">Xrp1</a></li><li class="kwd"><a href="/keyword/flybook" class="hw-term hw-article-keyword hw-article-keyword-flybook" rel="nofollow">Flybook</a></li></ul><p id="p-4">Forward genetic screens in <em>Drosophila</em> remain important for the discovery of gene functions and to the genetic characterization of biological processes (<a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">St. Johnston 2002</a>; <a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Wangler <em>et al.</em> 2015</a>). Accessible whole-genome sequencing permits rapid identification of newly isolated mutations, and several studies have described whole-genome sequencing approaches to mutation identification in <em>Drosophila</em> (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Blumenstiel <em>et al.</em> 2009</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Gerhold <em>et al.</em> 2011</a>; <a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Gonzalez <em>et al.</em> 2012</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Haelterman <em>et al.</em> 2014</a>). There are, however, additional steps required between the determination of DNA sequence for a newly mutagenized strain and the final identification of a single mutation responsible for a particular phenotype. One challenge is that <em>Drosophila</em> strains are not identical in sequence, even before mutagenesis (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Bender <em>et al.</em> 1983</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Mackay <em>et al.</em> 2012</a>). Strain polymorphisms greatly outnumber newly induced mutations when most mutant sequences are compared to the reference. A second challenge is that mutagenesis with the typical 25 mM dose of EMS results in one mutation per 273–480 kb (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Cooper <em>et al.</em> 2008</a>), corresponding to multiple new mutations on each chromosome.</p><p id="p-5">In principle, conventional genetic mapping should be able to associate a single novel mutation with the novel phenotype (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Sarin <em>et al.</em> 2008</a>). In practice, <em>Drosophila</em> studies often concentrate on loci where multiple alleles have been isolated, and prefer complementation testing with extensive collections of pre-existing mutant lines, deficiencies, and duplications to identify the gene (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Venken <em>et al.</em> 2010</a>; <a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Bellen <em>et al.</em> 2011</a>; <a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Roote and Russell 2012</a>). For example, Blumenstiel <em>et al.</em> identified <em><a href="http://flybase.org/reports/FBgn0004875.html">encore</a></em> as a mutated gene responsible for disrupting eggshell chamber morphology using complementation tests with publicly available mutant strains (<a id="xref-ref-10-2" class="xref-bibr" href="#ref-10">Blumenstiel <em>et al.</em> 2009</a>). Such an approach is not universally applicable, since mutant strains do not yet exist for all <em>Drosophila</em> genes. Gerhold <em>et al.</em> used genetic mapping to identify loci required for compensatory growth following imaginal disc damage (<a id="xref-ref-20-2" class="xref-bibr" href="#ref-20">Gerhold <em>et al.</em> 2011</a>). Gonzalez <em>et al.</em> recognized <em>ect4</em> as a gene required for Wallerian degeneration from its mutation in each of three noncomplementing mutant strains (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Gonzalez <em>et al.</em> 2012</a>). In an extensive recent study, mutant X-linked loci causing defects in neural development or homeostasis were identified in 274 of 394 sequenced strains using complementation with duplications covering portions of the X-chromosome as a mapping strategy (<a id="xref-ref-23-2" class="xref-bibr" href="#ref-23">Haelterman <em>et al.</em> 2014</a>), an approach analogous to deficiency mapping for autosomal mutations.</p><p id="p-6">Whole-genome sequencing remains sparingly used for novel mutant identification in <em>Drosophila</em>, partly because the approaches described above are not always applicable. For example, it may not always be straightforward to obtain multiple alleles of new loci, particularly for genetic backgrounds that grow poorly or for phenotypes that cannot be assessed rapidly, either of which may limit the size of genetic screens. In addition, recognition of allelism requires complementation analysis, which is also a prerequisite for deletion or duplication mapping, or for complementation with existing mutant strains. Complementation analysis is not straightforward for phenotypes that are only apparent in genetic mosaics. Genetic mosaicism is an important tool in <em>Drosophila</em> for investigating function in later developmental stages or adults of genes that are essential early in development. The usual method makes use of inducible FLP recombinase expression to achieve mitotic recombination between homologous chromosomes bearing FRT sites, yielding daughter cells homozygous for particular chromosome arms from heterozygous parents (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Xu and Rubin 1993</a>). In this way clones of homozygously mutant cells may be obtained in animals that are largely heterozygous, which enables characterization of the roles of essential genes in particular tissues. Mitotic recombination does not produce cells that are transheterozygotes of different alleles, or of alleles and deficiencies, unless an unlinked rescue transgene or gene duplication is employed, which is usually not feasible for unidentified new mutants.</p><p id="p-7">One process that is only revealed by genetic mosaic studies is cell competition. Cell competition describes the elimination of certain otherwise viable genotypes from genetic mosaics. Cell competition was originally described for the Minute class of mutations (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Morata and Ripoll 1975</a>), but has also been described for mosaics of certain other genotypes (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Baker 2011</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">de Beco <em>et al.</em> 2012</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Vivarelli <em>et al.</em> 2012</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Levayer and Moreno 2013</a>; <a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Amoyel and Bach 2014</a>). Minutes are dominant slow growing mutations now known to correspond to the majority of the ribosomal protein loci, which affect growth dominantly because of haploinsufficiency (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Marygold <em>et al.</em> 2007</a>). Minute mutations are typically lethal as homozygotes, but survive as heterozygotes with slow growth and thin or small (“Minute”) bristles (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Lambertsson 1998</a>). Despite the viability and fertility of Minute (<em>i.e.</em>, <em>Rp</em>/+) flies, clones of <em>Rp</em>/+ cells are not generally recovered in tissues derived from otherwise wild-type imaginal discs, since they are eliminated by competitive apoptosis (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Morata and Ripoll 1975</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Simpson 1979</a>; <a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Moreno <em>et al.</em> 2002</a>; <a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Kale <em>et al.</em> 2015</a>). Because cell competition only occurs in genetically mosaic animals, the effect of mutations on cell competition can only be assessed in genetic mosaics. This makes for a cumbersome phenotypic assay and largely precludes complementation analysis. For example, tumor suppressors of the <em>Salvador-Warts-Hippo</em> pathway were recognized in a screen for mutations that are required for cell competition by virtue of their other roles in growth regulation and organismal viability, but mutations that affected cell competition specifically and lacked other apparent phenotypes, although recovered, were not identified (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Tyler <em>et al.</em> 2007</a>).</p><p id="p-8">Here, we describe a genetic screen for further EMS-induced mutations affecting cell competition, using methods for mutation identification and mapping that avoid genetic complementation assays. We describe a Burrows–Wheeler Aligner/Genome Analysis Toolkit (BWA/GATK) pipeline for the identification of novel mutations from whole-genome sequences that automatically excludes strain differences between the reference genome and the FRT genetic background that is essential for mosaic generation. To permit mapping of a single mutation without recourse to complementation tests, we employ iPLEX MassARRAY genotyping. The iPLEX MassARRAY uses matrix-assisted laser desorption ionization time of flight (MALDI-TOF) mass spectrometry to detect sequence variants. Mass spectrometry is sufficiently quantitative that allele frequencies can be estimated from pools of recombinants, greatly facilitating mapping and reducing its expense (<a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Buetow <em>et al.</em> 2001</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Ding and Cantor 2003</a>; <a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Ding <em>et al.</em> 2004</a>). A flowchart outlining the sequencing and mapping process is shown in <a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>. These methods expand the utility of whole-genome sequencing to identify single mutations and mutations affecting mosaic traits in <em>Drosophila</em>.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/10/3207/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Flowchart of steps leading to identification and mapping of a single mutant allele without recourse to genetic complementation assays or collections of known mutants. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1423119758" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Flowchart of steps leading to identification and mapping of a single mutant allele without recourse to genetic complementation assays or collections of known mutants. &lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/10/3207/F1.medium.gif" width="255" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/10/3207/F1.medium.gif" width="255" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/10/3207/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/10/3207/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469402" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-9" class="first-child">Flowchart of steps leading to identification and mapping of a single mutant allele without recourse to genetic complementation assays or collections of known mutants. </p><div class="sb-div caption-clear"></div></div></div><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Fly strains and husbandry</h3><p id="p-10">Strains used in this study: FRT82B (<a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Xu and Rubin 1993</a>), P{w, <em><a href="http://flybase.org/reports/FBgn0000117.html">arm</a></em>-LacZ} (<a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Vincent <em>et al.</em> 1994</a>), P{w, <em><a href="http://flybase.org/reports/FBgn0003882.html">tub</a></em>-Gal80} (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Lee and Luo 1999</a>), <em><a href="http://flybase.org/reports/FBal0038901.html">RpS3<sup>Plac92</sup></a></em> (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Andersson <em>et al.</em> 1994</a>), <em>Atm</em><sup>3</sup> (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Silva <em>et al.</em> 2004</a>).</p><p id="p-11">For clones in the adult eye, we used an eye-specific enhancer-induced FLP (<em><a href="http://flybase.org/reports/FBgn0005558.html">ey</a></em>FLP2) insertion on the X chromosome (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Newsome <em>et al.</em> 2000</a>). For heat-shock-induced clones, we used a FLP122 insertion on the X chromosome (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Chou and Perrimon 1996</a>). Larvae were subjected to 60 min at 37°, 72 hr after egg deposition. Dissection and fixation were performed 96 hr after clone induction. All crosses were made at 25° on cornmeal agar medium.</p></div><div id="sec-3" class="subsection"><h3>Mutagenesis and screening</h3><p id="p-12">Male flies from an isogenized stock (<em>y w ey</em>FLP; FRT82B/TM3, <em><a href="http://flybase.org/reports/FBgn0004197.html">Ser</a></em><sup>1</sup>) were fed with 15 mM EMS in sucrose solution as described (<a id="xref-ref-36-2" class="xref-bibr" href="#ref-36">Newsome <em>et al.</em> 2000</a>). Subsequent crosses were made as shown in <a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2B</a>. Mutagenized males were crossed to non-Minute <em><a href="http://flybase.org/reports/FBgn0005558.html">ey</a></em>FLP virgins (<em>y w ey</em>FLP; FRT82B P{w+, <em><a href="http://flybase.org/reports/FBgn0000117.html">arm</a></em>-LacZ}/TM3, <em><a href="http://flybase.org/reports/FBgn0004197.html">Ser</a></em><sup>1</sup>) to generate mosaic eyes (<a id="xref-ref-36-3" class="xref-bibr" href="#ref-36">Newsome <em>et al.</em> 2000</a>). Single F1 males were crossed to Minute virgin females (<em>y w ey</em>FLP; FRT82B P{w+, <em><a href="http://flybase.org/reports/FBgn0000117.html">arm</a></em>-LacZ} P{w+, <em><a href="http://flybase.org/reports/FBgn0003882.html">tub</a></em>-Gal80} <em><a href="http://flybase.org/reports/FBgn0002622.html">RpS3</a></em><sup>Plac92</sup>/TM3, <em><a href="http://flybase.org/reports/FBgn0004197.html">Ser</a></em><sup>1</sup>) to screen for cell competition inhibiting mutants in the F2 generation as in <a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>. Stocks were recovered from the balanced F2 siblings.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/10/3207/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Scheme of a forward genetic screen to find genes required during cell competition. (A) The F2 crossing scheme for screening mutants on chromosomal arm 3R. In the F1 generation, single F1 male progeny with equal contributions of red and white eye tissues were selected (illustrated in C, D). After breeding with RpS3Plac92 females, the F2 progeny with appropriate genotype were screened for candidate mutant strains (illustrated in E, F). Mutants were recovered and bred from balanced siblings. (B) Cartoon of the screen concept. FLP-FRT mediated mitotic recombination in RpS3 heterozygous cells produces unpigmented, non-Minute FRT82B homozygous cells and reciprocally recombinant cells homozygous for the mutation in the RpS3 gene. The latter die, leaving the FRT82B homozygous clones to compete with unrecombined genotype RpS3 heterozygous genotype. (C) Adult eye image of an unmutagenized control fly in the F1 generation. Genotype: y w eyFLP; FRT82B w+ arm-LacZ/FRT82B (unmutagenized). (D) Schematic of the F1 generation. Mosaic flies that are indistinguishable from controls in proportions of red and white eye cells were selected. These should lack novel mutants that cell-autonomously alter growth, in which the unpigmented clones should either be relatively smaller (in case of a growth deficit) or larger (in case of a mutation causing overgrowth). (E) Adult eye image of an unmutagenized control fly in the F2 generation. Few pigmented cells remain in the eye, which is largely homozygous for the unmutagenized FRT82B progenitor chromosome. (F) Schematic of the F2 screen. Mutants that might affect cell competition are selected from F2 genotypes where more pigmented cells survive than in controls (see Figure 3, B and D for examples). Genotype: y w eyFLP; FRT82B/FRT82B P{w+, arm-LacZ} P{w+, tub-Gal80} RpS3." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1423119758" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Scheme of a forward genetic screen to find genes required during cell competition. (A) The F2 crossing scheme for screening mutants on chromosomal arm 3R. In the F1 generation, single F1 male progeny with equal contributions of red and white eye tissues were selected (illustrated in C, D). After breeding with &lt;em&gt;RpS3&lt;/em&gt;&lt;sup&gt;Plac92&lt;/sup&gt; females, the F2 progeny with appropriate genotype were screened for candidate mutant strains (illustrated in E, F). Mutants were recovered and bred from balanced siblings. (B) Cartoon of the screen concept. FLP-FRT mediated mitotic recombination in &lt;em&gt;RpS3&lt;/em&gt; heterozygous cells produces unpigmented, non-Minute FRT82B homozygous cells and reciprocally recombinant cells homozygous for the mutation in the &lt;em&gt;RpS3&lt;/em&gt; gene. The latter die, leaving the FRT82B homozygous clones to compete with unrecombined genotype &lt;em&gt;RpS3&lt;/em&gt; heterozygous genotype. (C) Adult eye image of an unmutagenized control fly in the F1 generation. Genotype: &lt;em&gt;y w ey&lt;/em&gt;FLP; FRT82B &lt;em&gt;w&lt;/em&gt;+ &lt;em&gt;arm&lt;/em&gt;-LacZ/FRT82B (unmutagenized). (D) Schematic of the F1 generation. Mosaic flies that are indistinguishable from controls in proportions of red and white eye cells were selected. These should lack novel mutants that cell-autonomously alter growth, in which the unpigmented clones should either be relatively smaller (in case of a growth deficit) or larger (in case of a mutation causing overgrowth). (E) Adult eye image of an unmutagenized control fly in the F2 generation. Few pigmented cells remain in the eye, which is largely homozygous for the unmutagenized FRT82B progenitor chromosome. (F) Schematic of the F2 screen. Mutants that might affect cell competition are selected from F2 genotypes where more pigmented cells survive than in controls (see Figure 3, B and D for examples). Genotype: &lt;em&gt;y w ey&lt;/em&gt;FLP; FRT82B/FRT82B P{w+, &lt;em&gt;arm&lt;/em&gt;-LacZ} P{w+, &lt;em&gt;tub&lt;/em&gt;-Gal80} &lt;em&gt;RpS3&lt;/em&gt;.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/10/3207/F2.medium.gif" width="440" height="207"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/10/3207/F2.medium.gif" width="440" height="207"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/10/3207/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/10/3207/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469300" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-13" class="first-child">Scheme of a forward genetic screen to find genes required during cell competition. (A) The F2 crossing scheme for screening mutants on chromosomal arm 3R. In the F1 generation, single F1 male progeny with equal contributions of red and white eye tissues were selected (illustrated in C, D). After breeding with <em>RpS3</em><sup>Plac92</sup> females, the F2 progeny with appropriate genotype were screened for candidate mutant strains (illustrated in E, F). Mutants were recovered and bred from balanced siblings. (B) Cartoon of the screen concept. FLP-FRT mediated mitotic recombination in <em>RpS3</em> heterozygous cells produces unpigmented, non-Minute FRT82B homozygous cells and reciprocally recombinant cells homozygous for the mutation in the <em>RpS3</em> gene. The latter die, leaving the FRT82B homozygous clones to compete with unrecombined genotype <em>RpS3</em> heterozygous genotype. (C) Adult eye image of an unmutagenized control fly in the F1 generation. Genotype: <em>y w ey</em>FLP; FRT82B <em>w</em>+ <em>arm</em>-LacZ/FRT82B (unmutagenized). (D) Schematic of the F1 generation. Mosaic flies that are indistinguishable from controls in proportions of red and white eye cells were selected. These should lack novel mutants that cell-autonomously alter growth, in which the unpigmented clones should either be relatively smaller (in case of a growth deficit) or larger (in case of a mutation causing overgrowth). (E) Adult eye image of an unmutagenized control fly in the F2 generation. Few pigmented cells remain in the eye, which is largely homozygous for the unmutagenized FRT82B progenitor chromosome. (F) Schematic of the F2 screen. Mutants that might affect cell competition are selected from F2 genotypes where more pigmented cells survive than in controls (see <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3, B and D</a> for examples). Genotype: <em>y w ey</em>FLP; FRT82B/FRT82B P{w+, <em>arm</em>-LacZ} P{w+, <em>tub</em>-Gal80} <em>RpS3</em>.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-4" class="subsection"><h3>Immunohistochemistry</h3><p id="p-14">Imaginal discs immunohistochemistry and confocal imaging was performed as described using paraformaldehyde-lysine-periodate fixation (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Baker <em>et al.</em> 2014</a>). Primary antibodies used were mouse anti-β-galactosidase (mAb40-1a) obtained from Developmental Studies Hybridoma Bank, and rabbit anti-activated Dcp-1 (Cell Signaling Technology, Danvers, MA). Secondary antibodies were multilabeling antibodies from Jackson Immunoresearch Laboratories (West Grove, PA).</p></div><div id="sec-5" class="subsection"><h3>DNA preparation for sequencing and bulk segregant analysis</h3><p id="p-15">Genomic DNA for sequencing was prepared from transheterozygous late third instar larvae grown at 18°, to maximize polytene amplification in which euchromatin is over-represented, as described (<a id="xref-ref-10-3" class="xref-bibr" href="#ref-10">Blumenstiel <em>et al.</em> 2009</a>).</p><p id="p-16">Genomic DNA for iPLEX MassARRAY was prepared from phenotyped adult mosaics transheterozygous for recombinant and parental FRT82B P{w+, <em><a href="http://flybase.org/reports/FBgn0000117.html">arm</a></em>-LacZ} P{w+, <em><a href="http://flybase.org/reports/FBgn0003882.html">tub</a></em>-Gal80} <em><a href="http://flybase.org/reports/FBal0038901.html">RpS3<sup>Plac92</sup></a></em> chromosomes. A total of 50 independent recombinants were pooled from each of the phenotypically wild-type and phenotypically M2-73 populations. Adults were ground in liquid nitrogen using a pestle and mortar, and DNA was recovered using the same procedure as for sequencing.</p></div><div id="sec-6" class="subsection"><h3>Genome sequencing and variant calling</h3><p id="p-17">Library construction for whole-genome sequencing was performed according to the manufacturer’s protocol (TruSeq DNA Sample Prep Kit v1, Illumina, San Diego, CA). Whole-genome resequencing was performed on Illumina HiSequation 2000 platform with paired-end 2 × 100 reads (Illumina). Adapter or low quality bases were trimmed from sequence reads. Then the cleared sequences were aligned to the reference genome dm3 using the BWA (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Li and Durbin 2009</a>). The Mark Duplicates program from Picard (<a href="http://picard.sourceforge.net/">http://picard.sourceforge.net/</a>) was used to remove duplicate reads. Local realignment, the recalibration of base quality values, and the adjustment of per-base alignment qualities were performed by components from the GATK pipeline (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">McKenna <em>et al.</em> 2010</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">DePristo <em>et al.</em> 2011</a>). The Unified Genotyper of GATK was applied to call the sequence differences or indels. Due to the fairly aggressive manner of the Unified Genotyper in making either SNP or indel calls, the raw call sets was filtered to reduce false-positive results based on call quality, depth, strand bias, <em>etc</em>., according to GATK best practice (version 4).</p><p id="p-18">Three mutant strains were exceptional in showing much higher frequencies of sequence differences. In each case there was a single large hypervariable region on chromosome 3R, within which apparent mutations occurred at much higher rate than normal (one mutation per 323 bp). These hypervariable regions were distinct but each partially overlapped another. This suggests that these three mutant chromosomes each contained sequences derived from a common progenitor that differs substantially from both the FRT82B chromosome and the reference genome sequence. We are unsure when these sequences entered our FRT82B strain, but sequences from these three anomalous regions have been excluded from our sequence analysis results.</p></div><div id="sec-7" class="subsection"><h3>Sanger sequencing</h3><p id="p-19">Primers of 20–22 bp length and annealing temperature of 57–63° were designed for 500–700 bp PCR products. NCBI/Primer-BLAST was used to optimize primer design and minimize off-target sites. QIAquick PCR Purification Kit (Qiagen) was used to purify PCR products for Sanger sequencing.</p></div><div id="sec-8" class="subsection"><h3>Recombination for mapping</h3><p id="p-20">The M2-73 chromosome was allowed to recombine with an isogenized chromosome marked with <em><a href="http://flybase.org/reports/FBgn0243586.html">Tb</a></em><sup>1</sup>. Recombinant offspring containing <em><a href="http://flybase.org/reports/FBgn0243586.html">Tb</a></em><sup>1</sup> and FRT82B were selected by Geneticin (G418) resistance. Single recombinant males were bred with the <em>y w ey</em>FLP; FRT82B P{w+, <em><a href="http://flybase.org/reports/FBgn0000117.html">arm</a></em>-LacZ} P{w+, <em><a href="http://flybase.org/reports/FBgn0003882.html">tub</a></em>-Gal80} <em><a href="http://flybase.org/reports/FBgn0002622.html">RpS3</a></em>/TM3, <em><a href="http://flybase.org/reports/FBgn0004197.html">Ser</a></em><sup>1</sup> tester strain and the offspring scored as phenotypically either mutant (cell competition defective) or nonmutant (cell competition manifested).</p></div><div id="sec-9" class="subsection"><h3>iPLEX MassARRAY</h3><p id="p-21">PCR and extension primers were designed from sequences containing each target SNP and ∼100 upstream and downstream bases with Assay Design Suite (<a href="http://agenabio.com/assay-design-suite-20-software">http://agenabio.com/assay-design-suite-20-software</a>) using the default settings. The PCR primers were pooled to a final concentration of 500 nM. The extend oligonucleotides were ranked according to mass and divided into three groups. The lowest mass group was rehydrated to 88 μM, the middle group to 110 μM, and the highest mass group to 165 μM. A 50 μl aliquot from each of the extend oligonucleotides was pooled to make an unextended extend primer (UEP) pool. The PCR primer pool was used to amplify 10 ng of DNA in a 5 μl volume reaction with 1 U of FastStart Taq Polymerase (Roche, Indianapolis, IN) and 4 mM magnesium chloride. The tubes were cycled 45 times with an annealing temperature of 56°. After PCR, shrimp alkaline phosphatase (SAP) was used to dephosphorylate any remaining dNTPs to render them unusable for future polymerase reactions. A total of 0.5 U of SAP and buffer (Agena Bioscience, San Diego, CA) was added to the PCR tubes and incubated for 40 min at 37°, followed by 5 min at 85°. Single base extension reactions were performed on the PCR reactions with the iPLEX Gold Kit (Agena Bioscience) and 0.8 μl of the custom UEP pool. The kit contains mass modified terminator nucleotides that increase the mass difference between extended UEPs, allowing for greater accuracy in genotyping. The mass difference with unmodified terminator nucleotides ranges from 9 to 40 kDa, depending on the two nucleotides compared. With the mass modified terminator nucleotides the mass difference increases to 16–80 kDa. The single base extension reactions were cycled with a nested PCR protocol that used five cycles of annealing and extension nested with a denaturation step in a cycle that was repeated 40 times for a total of 200 annealing and extension steps. The goal was to extend nearly all of the UEPs. Following single base extension, the reactions were diluted with 16 μl of water and deionized with 6 mg of resin. After deionizing for 20 min the reactions were dispensed onto SpectroChip Arrays with a Nanodispenser (Agena Bioscience). The speed of dispensation was optimized to deliver an average of 18 nl of each reaction to a matrix pad on the SpectroChip. An Agena Bioscience Compact Mass Array Spectrometer was used to perform MALDI-TOF mass spectrometry according to the iPLEX Gold Application Guide (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Gabriel <em>et al.</em> 2009</a>). Availability of MassARRAY Spectrometer facilities can be determined through Agena Bioscience. The Typer 4 software package (Agena Bioscience) was used to analyze the resulting spectra and the composition of the target bases was determined from the mass of each extended oligo.</p></div></div><div class="section results" id="sec-10"><h2 class="">Results</h2><div id="sec-11" class="subsection"><h3>EMS mutagenesis in search for genes required during cell competition</h3><p id="p-22">Although a number of genes affecting cell competition have been identified, significant questions remain. An interaction between competing genotypes is thought to trigger cell death only in the loser cells, but the identities of the interacting molecules that initiate cell competition have not been identified. To help address this and other outstanding questions, we conducted a chemical mutagenesis to screen for further genes required during cell competition. The breeding scheme used, as well as the principle of the screen, are described in <a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2A</a>. We mutagenized a strain with an FRT site near the centromere of chromosome 3R, at position 82B, in order make clones homozygous for this arm. Mutagenized chromosomes were bred with FRT82B P{w+, <em><a href="http://flybase.org/reports/FBgn0000117.html">arm</a></em>-LacZ} P{w+, <em><a href="http://flybase.org/reports/FBgn0003882.html">tub</a></em>-Gal80} <em><a href="http://flybase.org/reports/FBal0038901.html">RpS3<sup>Plac92</sup></a></em> strains in a background of the <em><a href="http://flybase.org/reports/FBgn0005558.html">ey</a></em>FLP transgene to obtain flies in which cells transheterozygous for the mutagenized and <em><a href="http://flybase.org/reports/FBal0038901.html">RpS3<sup>Plac92</sup></a></em> chromosomes were subject to mitotic recombination in the head (<a id="xref-ref-36-4" class="xref-bibr" href="#ref-36">Newsome <em>et al.</em> 2000</a>). The <em><a href="http://flybase.org/reports/FBal0038901.html">RpS3<sup>Plac92</sup></a></em> mutation is a typical Minute mutation, corresponding to the classically defined gene M(3)95A (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Marygold <em>et al.</em> 2007</a>). Since the <em><a href="http://flybase.org/reports/FBal0038901.html">RpS3<sup>Plac92</sup></a></em> mutation is cell lethal when homozygous (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2B</a>), the resulting adult eyes typically consist almost entirely of recombined FRT82B/FRT82B cells, distinguished by the lack of eye pigment (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2E</a>). We reasoned that the few remaining, pigmented, <em><a href="http://flybase.org/reports/FBgn0002622.html">RpS3</a></em> heterozygous cells have survived two processes: first, they represent the minority of cells where chronic FLP expression did not lead to mitotic recombination before the end of development; second, they survived competition with the increasing population of non-Minute (<em><a href="http://flybase.org/reports/FBgn0002622.html">RpS3</a></em><sup>+/+</sup>) FRT82B homozygous cells. We hypothesized that mutations that diminished cell competition (or mitotic recombination) would lead to increased representation of unrecombined, pigmented, <em>RpS3<sup>Plac92/+</sup></em> cells in this assay (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2F</a>). Therefore, increased contribution of pigmented cells offers an easily visualized screen for mutations that might affect cell competition. To help discriminate against mutations affecting recombination, or recessively diminishing growth, which could also result in a higher proportion of pigmented cells, an F2 screening procedure was adopted (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2A</a>). In the F1 generation, heterozygotes between mutagenized chromosomes and control chromosomes were screened to identify mosaic flies where the two genotypes were equally represented in the adult eyes, selecting against mutations with effects on recombination or growth (<a id="xref-fig-2-7" class="xref-fig" href="#F2">Figure 2, C and D</a>). This procedure should also eliminate any mutations dominantly affecting FLP-mediated mitotic recombination, should such mutations exist. F1 flies satisfying this condition were then individually bred with a FRT82B P{w+, arm-LacZ} <em><a href="http://flybase.org/reports/FBal0038901.html">RpS3<sup>Plac92</sup></a></em> strain to identify mutations potentially required for cell competition using the F2 generation.</p><p id="p-23">A total of 2731 such F2 crosses were screened, recovering 19 mutated lines that reproducibly enhanced the contribution of <em><a href="http://flybase.org/reports/FBgn0002622.html">RpS3</a></em> heterozygous cells in the F2 generation without enhancing the contribution of non-Minute cells in the F1 generation. Only one of these mutagenized chromosomes was homozygously viable. Crosses between all the other strains revealed lethal complementation groups of four alleles and two alleles (although these apparent complementation results were misleading, as described in the section on Atm mutations and cell competition and 13 mutant lines that complemented all the other lethal lines.</p></div><div id="sec-12" class="subsection"><h3>Whole-genome sequencing of mutants</h3><p id="p-24">We identified the novel mutations in the mutant strains by sequencing the genomes of all 15 distinct complementation groups, as well as the second allele from the two-allele complementation group, for a total of 16 genotypes. Since most mutant homozygotes were lethal, genomic DNA from heterozygotes was sequenced. To facilitate the differentiation of new mutations induced by chemical mutagenesis from strain polymorphisms between the reference genome and the FRT82B chromosome that was necessary in this mosaic screen, we adapted a transheterozygote sequencing approach pioneered by Gerhold <em>et al.</em> (<a id="xref-ref-20-3" class="xref-bibr" href="#ref-20">Gerhold <em>et al.</em> 2011</a>). For example, to identify the mutations present in four mutant strains M2-73, M9-22, M9-25, and M10-26, whole-genome sequencing was performed on the four transheterozygous genotypes M2-73 +/+ M9-22, M9-22 +/+ M10-26, M10-26 +/+ M9-25, and M9-25 +/+ M2-73. Interpretation of these transheterozygous sequences is described below.</p><p id="p-25">Sequencing was performed on an Illumina HiSeq2000 sequencing platform, as two groups of eight sequences, each with multiplexed 100 bp paired-end reads in a single lane platform (see <em>Materials and Methods</em> for details). One group of eight samples was then resequenced in a further lane platform, which doubled the read depth for these samples.</p></div><div id="sec-13" class="subsection"><h3>Identification of mutations after whole-genome sequencing</h3><p id="p-26">We used BWA, which is fast and accurate, to align sequences (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">Li and Durbin 2009</a>). BWA was employed to align sequence output to the <em>Drosophila melanogaster</em> genome version dm3 (+BDGP Release 5) (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hoskins <em>et al.</em> 2007</a>). Since Illumina HiSeq2000 can generate up to 35 Gbp of sequence per lane, and the <em>Drosophila</em> genome sequence comprises 180 Mb (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Adams <em>et al.</em> 2000</a>), we anticipated around 24× read depth for eight samples sequenced from a single lane, and 48× read depth for eight samples sequenced in two lanes. About 23× and 50× read depth were actually achieved (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). A further statistic that is useful when assessing the reliability of heterozygous mutant detection is the percentage of the genome sequenced to ≥10× read depth. Surprisingly, this was ∼95% in both cases (<a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). This relative independence from overall read depth suggests that the remaining 5% of the genome for which &lt;10× coverage is available may represent problematic sequences, for example repetitive regions that are not aligned well, and that additional sequencing might yield higher overall read depth but little more information in such difficult regions.</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469488/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469488/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Sequencing statistics</span><div class="sb-div caption-clear"></div></div></div><p id="p-28">Gerhold <em>et al.</em> were the first to exploit transheterozygote sequencing in this way, using a customized Perl script to identify induced mutations (<a id="xref-ref-20-4" class="xref-bibr" href="#ref-20">Gerhold <em>et al.</em> 2011</a>). We were able to use the publicly available Unified Genotyper of GATK for this purpose (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">McKenna <em>et al.</em> 2010</a>; <a id="xref-ref-16-2" class="xref-bibr" href="#ref-16">DePristo <em>et al.</em> 2011</a>). The algorithm compares four transheterozygous sequences to identify heterozygous variants that are detected in exactly two sequences out of four. This procedure automates the removal of strain differences between the FRT82B background and the reference genome, which are expected both to be homozygous and present in all four samples, and the removal of the large majority of alignment errors, since these will usually be distributed randomly among the four sequences (regions of inconsistent alignment, for example those related to indels or repeats, may lead to mixed base calls that resemble heterozygous variants in a single sequence). The procedure should also select against variants from regions that appear “difficult to sequence,” and produce high false-positive mutation rates (<a id="xref-ref-23-3" class="xref-bibr" href="#ref-23">Haelterman <em>et al.</em> 2014</a>). Specifically, our algorithm for each mutant calls for 0/1 in the two relevant sequences, and 0/0 in the remaining two samples, where “0” corresponds to reference allele and “1” to any variant. In case a novel mutation affected a site already differing between the reference sequence and the FRT82B strain, we also allowed for a 1/2 combination in two sequences with a 1/1 combination in the remainder, where “2” represents another distinct allele. At least regarding mutations in coding regions, our primary interest, no such event was observed.</p><p id="p-29">Initially, filters were set to prioritize regions of more reliable sequence. We required a threshold of read depth ≥15, genotype quality ≥30 (a Phred-scaled confidence measure for the reported genotype) (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">McKenna <em>et al.</em> 2010</a>; <a id="xref-ref-16-3" class="xref-bibr" href="#ref-16">DePristo <em>et al.</em> 2011</a>). These analyses found, on average, 83.7 (±25.3) candidate mutations on chromosomes arm 3R for the once-sequenced samples, and 139.6 (±25.4) for the twice-sequenced samples, of which 12.5 (±4.8) and 16.4 (±0.9) were the average numbers of nonsynonymous mutations affecting coding regions (<a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>; errors are ±1S.D.).</p><p id="p-30">To evaluate the accuracy of the variant calls, genomic DNA was amplified from transheterozygote genomic DNA samples and subject to Sanger sequencing. From 16 candidate coding mutations identified from the samples sequenced in a single Illumina lane, and 21 candidate coding mutations predicted from the samples sequenced twice, each one was confirmed by Sanger sequencing.</p><p id="p-31">Given the high apparent accuracy observed when sequence quality filters were applied, whereby no false positives were observed, sequences were re-evaluated without read depth or genotype quality filters, identifying an average 226.5 additional, lower confidence variants from samples sequenced once, of which 11.3 were nonsynonymous coding mutations, and 89.8 additional variants in samples sequenced twice, of which 5.4 were nonsynonymous coding mutations (<a id="xref-table-wrap-1-4" class="xref-table" href="#T1">Table 1</a>). A sample of these lower-confidence mutations were checked by amplification and Sanger sequencing. Out of 16 lower confidence coding mutations predicted from the samples sequenced in a single Illumina lane, Sanger sequencing confirmed 14. Out of another 18 lower confidence coding mutations predicted from the samples sequenced twice, 16 were confirmed.</p><p id="p-32">Taken together, these data indicate that even without applying filters for sequence quality, the false-positive rate of mutant identification is low. If the results are pooled as if no sequence filters had been applied from the outset, the measured false-positive rates for predicted coding mutations were 5.9% for samples sequenced to 23× read depth and 2.8% for samples sequenced to 50× read depth. Filtering for genome quality eliminated false positives, but appeared to conceal about one-third of the mutations present in samples sequenced to 50× read depth, and the majority of those present in the once-sequenced samples, because many fewer mutations were identified when filters were applied (<a id="xref-table-wrap-1-5" class="xref-table" href="#T1">Table 1</a>). Without filters, the overall number of coding mutations identified is very similar in the samples of different read depth. An average of 23.8 coding mutations were predicted from samples sequences in one lane, of which 5.9% are expected to be false positives, and 21.8 predicted from samples sequenced in two lanes, of which 2.8% are expected to be false positives. Similar mutation recovery independent of read depth may argue that false-negative detection rates are also low (see <em>Discussion</em>). However, there was an effect of read depth on recovery of noncoding mutations (<a id="xref-table-wrap-1-6" class="xref-table" href="#T1">Table 1</a>).</p><p id="p-33">G/C-to-A/T transitions accounted for 74.4% of the predicted coding mutations, in agreement with previous studies of EMS mutagenesis in <em>Drosophila</em> (<a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Winkler <em>et al.</em> 2005</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Cooper <em>et al.</em> 2008</a>; <a id="xref-ref-10-4" class="xref-bibr" href="#ref-10">Blumenstiel <em>et al.</em> 2009</a>). The results indicate that one mutation occurred every 149 kb in 3R euchromatin, more than predicted by earlier whole-genome sequencing studies (<a id="xref-table-wrap-2-1" class="xref-table" href="#T2">Table 2</a>) (<a id="xref-ref-10-5" class="xref-bibr" href="#ref-10">Blumenstiel <em>et al.</em> 2009</a>; <a id="xref-ref-20-5" class="xref-bibr" href="#ref-20">Gerhold <em>et al.</em> 2011</a>), which is a further indication of improved mutation detection.</p><div id="T2" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469364/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469364/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 2</span> <span class="caption-title">Frequency of mutations</span><div class="sb-div caption-clear"></div></div></div></div><div id="sec-14" class="subsection"><h3>Atm mutations and cell competition</h3><p id="p-37">The M9-25 mutant strain, one of the four-member complementation group, contained a mutation in the Ataxia telangiectasia mutated (<em>Atm</em>) gene, a locus important in DNA repair (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abraham 2001</a>). Unexpectedly, the other three noncomplementing strains all harbored the identical allele, as did two other mutants that had been sequenced at the whole-genome level, M2-17 and M10-22. This mutation, a C to A alteration at position 15,241,548 (+BDGP Release 6) generated a premature stop codon predicted to truncate the 2767 amino acid Atm polypeptide after residue 162. Despite the presence of the identical <em>Atm</em> mutation, M9-25, M2-17, and M10-22 each showed a typical spectrum of mutations, distinct from one another. A total of 39 of the 52 coding mutations observed in these strains reflected alkylation events typical for EMS mutagenesis. By contrast, the 15,241,548 C to A transversion is not typical of EMS. These data argue against the accidental isolation of the same EMS-induced allele multiple times, and suggest that a spontaneous <em>Atm</em> mutant allele was segregating in the isogenized FRT82B progenitor strain and was recovered on multiple occasions after passing through the mutagenesis.</p><p id="p-38">To verify that the mutation in <em>Atm</em> was responsible for the phenotype, the independent <em>Atm</em><sup>3</sup> mutation was studied. The <em>Atm</em><sup>3</sup> mutation corresponds to a premature stop codon in place of amino acid 600 (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Pedersen <em>et al.</em> 2010</a>). Mosaic Minute eyes containing homozygous <em>Atm</em><sup>3</sup> clones showed an increased contribution of <em><a href="http://flybase.org/reports/FBgn0002622.html">RpS3</a></em> heterozygous cells in the adult; in addition, the mosaic adult eyes were slightly small and rough in all of these mutants (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3, A–C</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/10/3207/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Atm and Xrp1 mosaic phenotypes in Minute and non-Minute backgrounds. (A&#x2013;D) Adult eye phenotypes from F2 screen (see Figure 2A for method and genotypes) (A) FRT82B control; (B) FRT82B M9-25; (C) FRT82B Atm3; (D) FRT82B M2-73. (B&#x2013;D) Homozygous M9-25, Atm3, and M2-73 cells were less able to eliminate pigmented RpS3 heterozygous cells. (E, F) Late third instar wing imaginal disc. The RpS3/+ background is labeled for &#x3B2;-galactosidase expression in green; non-Minute clones are unlabeled. Cell death is detected with &#x3B1;-activated-Dcp-1 labeling in red. (E) Competitive cell death of RpS3/+ cells is largely limited to boundaries between the genotypes. (F) High cell death levels were observed throughout Atm3 clones. (G&#x2013;I) Atm3 homozygous clones induced between 48 and 72 hr in a non-Minute background and dissected at late third instar larval stage from 48 to 72 hr after clone induction (ACI) as indicated. The FRT82B Atm3 heterozygous background was labeled for &#x3B2;-galactosidase expression, the FRT82B homozygous twin-spots were labeled more brightly, the FRT82B Atm3 homozygous clones were unlabeled. Atm3 homozygous clones were under-represented compared to twin-spots after 72 hr but not after shorter times. Scalebars represent 250 microns for panels A&#x2013;D and 50 microns for panels E&#x2013;I. Genotypes: (A) y w eyFLP;FRT82B/FRT82B P{w+, arm-LacZ} P{w+, tub-Gal80} RpS3, (B) y w eyFLP;FRT82B M9-25/FRT82B P{w+, arm-LacZ} P{w+, tub-Gal80} RpS3, (C) y w eyFLP;FRT82B Atm3/FRT82B P{w+, arm-LacZ} P{w+, tub-Gal80} RpS3, (D) y w eyFLP;FRT82B M2-73/FRT82B P{w+, arm-LacZ} P{w+, tub-Gal80} RpS3, (E) hsFLP;FRT82B/FRT82B P{w+, arm-LacZ} P{w+, tub-Gal80} RpS3, (F) hsFLP;FRT82B Atm3/FRT82B P{w+, arm-LacZ} P{w+, tub-Gal80} RpS3, (G&#x2013;I) y w hsFLP;FRT82B Atm3/FRT82B P{w+, arm-LacZ}." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1423119758" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Atm&lt;/em&gt; and &lt;em&gt;Xrp1&lt;/em&gt; mosaic phenotypes in Minute and non-Minute backgrounds. (A&#x2013;D) Adult eye phenotypes from F2 screen (see Figure 2A for method and genotypes) (A) FRT82B control; (B) FRT82B M9-25; (C) FRT82B &lt;em&gt;Atm&lt;/em&gt;&lt;sup&gt;3&lt;/sup&gt;; (D) FRT82B M2-73. (B&#x2013;D) Homozygous M9-25, &lt;em&gt;Atm&lt;/em&gt;&lt;sup&gt;3&lt;/sup&gt;, and M2-73 cells were less able to eliminate pigmented &lt;em&gt;RpS3&lt;/em&gt; heterozygous cells. (E, F) Late third instar wing imaginal disc. The &lt;em&gt;RpS3&lt;/em&gt;/+ background is labeled for &#x3B2;-galactosidase expression in green; non-Minute clones are unlabeled. Cell death is detected with &#x3B1;-activated-&lt;em&gt;Dcp&lt;/em&gt;-1 labeling in red. (E) Competitive cell death of &lt;em&gt;RpS3&lt;/em&gt;/+ cells is largely limited to boundaries between the genotypes. (F) High cell death levels were observed throughout &lt;em&gt;Atm&lt;/em&gt;&lt;sup&gt;3&lt;/sup&gt; clones. (G&#x2013;I) &lt;em&gt;Atm&lt;/em&gt;&lt;sup&gt;3&lt;/sup&gt; homozygous clones induced between 48 and 72 hr in a non-Minute background and dissected at late third instar larval stage from 48 to 72 hr after clone induction (ACI) as indicated. The FRT82B &lt;em&gt;Atm&lt;/em&gt;&lt;sup&gt;3&lt;/sup&gt; heterozygous background was labeled for &#x3B2;-galactosidase expression, the FRT82B homozygous twin-spots were labeled more brightly, the FRT82B &lt;em&gt;Atm&lt;/em&gt;&lt;sup&gt;3&lt;/sup&gt; homozygous clones were unlabeled. &lt;em&gt;Atm&lt;/em&gt;&lt;sup&gt;3&lt;/sup&gt; homozygous clones were under-represented compared to twin-spots after 72 hr but not after shorter times. Scalebars represent 250 microns for panels A&#x2013;D and 50 microns for panels E&#x2013;I. Genotypes: (A) &lt;em&gt;y w ey&lt;/em&gt;FLP;FRT82B/FRT82B P{w+, &lt;em&gt;arm&lt;/em&gt;-LacZ} P{w+, &lt;em&gt;tub&lt;/em&gt;-Gal80} &lt;em&gt;RpS3&lt;/em&gt;, (B) &lt;em&gt;y w ey&lt;/em&gt;FLP;FRT82B M9-25/FRT82B P{w+, &lt;em&gt;arm&lt;/em&gt;-LacZ} P{w+, &lt;em&gt;tub&lt;/em&gt;-Gal80} &lt;em&gt;RpS3&lt;/em&gt;, (C) &lt;em&gt;y w ey&lt;/em&gt;FLP;FRT82B &lt;em&gt;Atm&lt;/em&gt;&lt;sup&gt;3&lt;/sup&gt;/FRT82B P{w+, &lt;em&gt;arm&lt;/em&gt;-LacZ} P{w+, &lt;em&gt;tub&lt;/em&gt;-Gal80} &lt;em&gt;RpS3&lt;/em&gt;, (D) &lt;em&gt;y w ey&lt;/em&gt;FLP;FRT82B M2-73/FRT82B P{w+, &lt;em&gt;arm&lt;/em&gt;-LacZ} P{w+, &lt;em&gt;tub&lt;/em&gt;-Gal80} &lt;em&gt;RpS3&lt;/em&gt;, (E) &lt;em&gt;hs&lt;/em&gt;FLP;FRT82B/FRT82B P{w+, &lt;em&gt;arm&lt;/em&gt;-LacZ} P{w+, &lt;em&gt;tub&lt;/em&gt;-Gal80} &lt;em&gt;RpS3&lt;/em&gt;, (F) &lt;em&gt;hs&lt;/em&gt;FLP;FRT82B &lt;em&gt;Atm&lt;/em&gt;&lt;sup&gt;3&lt;/sup&gt;/FRT82B P{w+, &lt;em&gt;arm&lt;/em&gt;-LacZ} P{w+, &lt;em&gt;tub&lt;/em&gt;-Gal80} &lt;em&gt;RpS3&lt;/em&gt;, (G&#x2013;I) &lt;em&gt;y w hs&lt;/em&gt;FLP;FRT82B &lt;em&gt;Atm&lt;/em&gt;&lt;sup&gt;3&lt;/sup&gt;/FRT82B P{w+, &lt;em&gt;arm&lt;/em&gt;-LacZ}.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/10/3207/F3.medium.gif" width="438" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/10/3207/F3.medium.gif" width="438" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/10/3207/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/10/3207/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469243" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-39" class="first-child"><em>Atm</em> and <em>Xrp1</em> mosaic phenotypes in Minute and non-Minute backgrounds. (A–D) Adult eye phenotypes from F2 screen (see <a id="xref-fig-2-8" class="xref-fig" href="#F2">Figure 2A</a> for method and genotypes) (A) FRT82B control; (B) FRT82B M9-25; (C) FRT82B <em>Atm</em><sup>3</sup>; (D) FRT82B M2-73. (B–D) Homozygous M9-25, <em>Atm</em><sup>3</sup>, and M2-73 cells were less able to eliminate pigmented <em>RpS3</em> heterozygous cells. (E, F) Late third instar wing imaginal disc. The <em>RpS3</em>/+ background is labeled for β-galactosidase expression in green; non-Minute clones are unlabeled. Cell death is detected with α-activated-<em>Dcp</em>-1 labeling in red. (E) Competitive cell death of <em>RpS3</em>/+ cells is largely limited to boundaries between the genotypes. (F) High cell death levels were observed throughout <em>Atm</em><sup>3</sup> clones. (G–I) <em>Atm</em><sup>3</sup> homozygous clones induced between 48 and 72 hr in a non-Minute background and dissected at late third instar larval stage from 48 to 72 hr after clone induction (ACI) as indicated. The FRT82B <em>Atm</em><sup>3</sup> heterozygous background was labeled for β-galactosidase expression, the FRT82B homozygous twin-spots were labeled more brightly, the FRT82B <em>Atm</em><sup>3</sup> homozygous clones were unlabeled. <em>Atm</em><sup>3</sup> homozygous clones were under-represented compared to twin-spots after 72 hr but not after shorter times. Scalebars represent 250 microns for panels A–D and 50 microns for panels E–I. Genotypes: (A) <em>y w ey</em>FLP;FRT82B/FRT82B P{w+, <em>arm</em>-LacZ} P{w+, <em>tub</em>-Gal80} <em>RpS3</em>, (B) <em>y w ey</em>FLP;FRT82B M9-25/FRT82B P{w+, <em>arm</em>-LacZ} P{w+, <em>tub</em>-Gal80} <em>RpS3</em>, (C) <em>y w ey</em>FLP;FRT82B <em>Atm</em><sup>3</sup>/FRT82B P{w+, <em>arm</em>-LacZ} P{w+, <em>tub</em>-Gal80} <em>RpS3</em>, (D) <em>y w ey</em>FLP;FRT82B M2-73/FRT82B P{w+, <em>arm</em>-LacZ} P{w+, <em>tub</em>-Gal80} <em>RpS3</em>, (E) <em>hs</em>FLP;FRT82B/FRT82B P{w+, <em>arm</em>-LacZ} P{w+, <em>tub</em>-Gal80} <em>RpS3</em>, (F) <em>hs</em>FLP;FRT82B <em>Atm</em><sup>3</sup>/FRT82B P{w+, <em>arm</em>-LacZ} P{w+, <em>tub</em>-Gal80} <em>RpS3</em>, (G–I) <em>y w hs</em>FLP;FRT82B <em>Atm</em><sup>3</sup>/FRT82B P{w+, <em>arm</em>-LacZ}.</p><div class="sb-div caption-clear"></div></div></div><p id="p-40">To understand the effect of <em>Atm</em> mutations, homozygous <em>Atm</em><sup>3</sup> clones were induced in <em><a href="http://flybase.org/reports/FBgn0002622.html">RpS3</a></em> heterozygous wing discs using <em>hs</em>FLP. An antibody against the activated caspase <em>Dcp</em>-1 revealed a strikingly high level of apoptosis throughout the <em>Atm</em><sup>3</sup> clones (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3, E and F</a>), consistent with a previous report of spontaneous apoptosis in <em>Atm</em><sup>−/−</sup> cells in the wing imaginal disc (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Oikemus <em>et al.</em> 2004</a>). This may explain the retention of <em><a href="http://flybase.org/reports/FBgn0002622.html">RpS3</a></em> heterozygous cells in our screen. Chronic apoptosis can reduce clonal growth (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Shi <em>et al.</em> 2003</a>), so reduced growth of homozygous <em>Atm</em><sup>3</sup> clones likely leads to a greater proportion of the eye being descended from the remaining <em><a href="http://flybase.org/reports/FBgn0002622.html">RpS3</a></em> heterozygous cells. A plausible explanation for why a mutation deleterious for clonal growth was not excluded in the F1 generation was suggested by inducing <em>Atm</em><sup>3</sup> homozygous clones in a wild-type background at different time points (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3, G–I</a>). When induced 48 or 60 hr before dissection in the later third instar, <em>Atm</em><sup>3</sup> homozygous clones were present at sizes and numbers similar to the reciprocal recombinant twin clones in wing imaginal discs. <em>Atm</em><sup>3</sup> homozygous clones were only under-represented when dissected 72 hr after clone induction (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3I</a>). This suggests that <em>Atm</em><sup>3</sup> homozygous clones survive without significant growth effects for many hours, only later reducing in viability (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3H</a>). The <em>Atm</em><sup>3</sup> homozygous clones seen in eyes of the F1 generation of our screen may have been induced by <em><a href="http://flybase.org/reports/FBgn0005558.html">ey</a></em>FLP too late to exhibit strong growth effects in a non-Minute background, by contrast to the <em>RpS3<sup>Plac92</sup>/+</em> F2 generation that grew for ∼2 d longer.</p><p id="p-41">Two alleles of a second putative lethal complementation group were sequenced. Both mutants, however, showed an identical spectrum of novel mutations on chromosome 3R, indicating that they represented reisolation of the same mutant, presumably induced premeiotically. Therefore, the specific mutation responsible for the phenotype could not be revealed as a locus independently mutated in the two lines.</p></div><div id="sec-15" class="subsection"><h3>Mapping a single allele using recombination and iPLEX MassARRAY</h3><p id="p-42">None of the other 13 mutant strains shared any alleles that affected the open reading frames of common loci. In addition, no mutations were observed in genes yet implicated in cell competition. Since each mutant strain contained mutations affecting ∼20 protein coding regions on chromosome 3R, as well as many alterations in noncoding regions, other methods were needed to identify the responsible genes. We decided to focus on the M2-73 mutant, which gives a clear and reproducible phenotype in the mosaic eye assay (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3D</a>).</p><p id="p-43">In other genome sequencing projects, identification of causative mutations has been facilitated by complementation with existing alleles (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Zhai <em>et al.</em> 2003</a>). Complementation is difficult to apply to phenotypes scored in mosaics. It would be possible to test existing mutations for similar phenotypes to M2-73, but loss-of-function mutations existed for only five out of the 22 coding mutations in M2-73 and many are not recombined onto FRT82B chromosomes and carry transgenic <em>w</em>+ alleles that would interfere with the adult eye assay. Finally, there is no guarantee that existing alleles would present identical phenotypes to uncharacterized missense mutations present on the M2-73 chromosome. Software prediction of which amino acid substitutions affect protein function using SIFT (<a href="http://sift.jcvi.org/">http://sift.jcvi.org/</a>) (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Kumar <em>et al.</em> 2009</a>) predicted that 17 of the 22 nonsynonymous mutations present on M2-73 would be deleterious, doing little to prioritize likely mutations.</p><p id="p-44">Recombinational mapping required phenotypic scoring and genotyping of recombinant chromosomes in mosaic flies for the phenotypic assessment of recombinant chromosomes. We applied iPLEX MassARRAY to map the mutation responsible for the M2-73 phenotype, an approach that is both generally applicable, accurate, efficient, and economical. In iPLEX MassARRAY, a primer that precisely flanks a mutant site is extended from an amplified genomic DNA template using a mass modified dideoxynucleotide, then MALDI-TOF mass spectrometry was used to determine the allele (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Buetow <em>et al.</em> 2001</a>; <a id="xref-ref-17-2" class="xref-bibr" href="#ref-17">Ding and Cantor 2003</a>). The accuracy of mass spectrometry permits measurement of an allele frequency from pooled samples, and multiple SNPs can be assessed simultaneously if their specific primers differ in mass (<a id="xref-ref-18-2" class="xref-bibr" href="#ref-18">Ding <em>et al.</em> 2004</a>).</p><p id="p-45">We first analyzed amplified DNA from FRT82B M2-73/FRT82B P{w+, <em><a href="http://flybase.org/reports/FBgn0000117.html">arm</a></em>-LacZ} P{w+, <em><a href="http://flybase.org/reports/FBgn0003882.html">tub</a></em>-Gal80} <em><a href="http://flybase.org/reports/FBal0038901.html">RpS3<sup>Plac92</sup></a></em> transheterozygotes to confirm heterozygosity for 11 novel SNPs identified after the mutagenesis (the identity of all 236 novel SNPs recovered on the M2-73 chromosome is shown in Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029421/-/DC1/TableS1.pdf">Table S1</a>). These were distributed across 3R at 1.6–2.6 Mb intervals. Ideally, the peaks for wild-type and mutant products from heterozygous DNA should be equal. We found nine loci where wild-type and mutant signals were within 10% of one another, one locus where only the wild-type allele was detected, and another that yielded 80% wild-type product (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029421/-/DC1/FigureS1.tif">Figure S1</a>). Such deviations from equality can occur if either chromosome differs from the reference sequence used to guide primer design, so that the primers do not amplify each genotype equally.</p><p id="p-46">Genetic recombinants were obtained after breeding FRT82B M2-73 with an isogenized <em><a href="http://flybase.org/reports/FBgn0243586.html">Tb</a></em><sup>1</sup> strain. The dominant marker <em><a href="http://flybase.org/reports/FBgn0243586.html">Tb</a></em><sup>1</sup> maps distal on chromosome 3R, distant from FRT82B. Recombinant FRT82B <em><a href="http://flybase.org/reports/FBgn0243586.html">Tb</a></em><sup>1</sup> strains were assessed in mosaics for presence of the M2-73 phenotype, and DNA extracted from a pool of 50 independent recombinants exhibiting the M2-73 phenotype and 50 independent recombinants exhibiting the control phenotype. Allele frequencies were determined for each pool by iPLEX MassARRAY. It is expected that the mutant allele should be present in all the recombinants exhibiting the mutant phenotype, and none of the recombinants lacking the phenotype (<a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4, A and B</a>, respectively). In this initial experiment, the pool of phenotypically nonmutant recombinants exhibited only <em><a href="http://flybase.org/reports/FBgn0243586.html">Tb</a></em><sup>1</sup> alleles from the distal tip of 3R to 19,588,401 bp (+BDGP Release 6, cytological band 92A10), and the pool of phenotypically mutant recombinants exhibited only <em>M2-73</em> alleles from the proximal end of 3R to 17,720,801 bp (+BDGP Release 6, cytological band 90C2). This indicates the causative mutation mapped to the interval 17,720,801–19,588,401 bp (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029421/-/DC1/FigureS1.tif">Figure S1</a>).</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/10/3207/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="iPLEX MassARRAY genotyping maps the M2-73 mutant. (A, B) Recombinant mapping principles. &#x2018;X&#x2019; illustrates that meiotic recombination occurs between homologs. (A) A cartoon representation of recombinant chromosome arms 3R expected to retain the M2-73 phenotype. These should retain M2-73 SNPs proximal to and including the locus responsible (asterisk), and exhibit various crossover points distal to the locus. When DNA from 50 such recombinants is pooled, the frequency of M2-73 SNPs is expected to decrease distal to the locus as the frequency of Tb SNPs increases. (B) A cartoon representation of recombinant chromosome arms 3R expected to lack the M2-73 phenotype. These should retain Tb SNPs distal to and including locus responsible for the M2-73 phenotype, and exhibit crossover points proximal to the locus. When DNA from such recombinants is pooled, the frequency of M2-73 SNPs should increase proximally to the locus as the frequency of Tb SNPs decreases. Comparing observed SNP frequencies for the two pools should limit the mutant locus to a chromosome segment that retains 100% M2-73 SNPs in phenotypically mutant recombinants and 100% Tb SNPs in phenotypically wild-type recombinants. (C) Distribution of the 17 SNPs that were assessed by iPLEX (crosses) and 22 exonic variants on the M2-73 chromosome (circles). An open circle shows the position of the coding mutation in Xrp1, closed circles show the other mutations. Some of the coding alleles were used as SNPs, in addition to noncoding SNPs. (D) Allele frequencies determined by iPLEX for 17 SNPs in the FRT82B M2-73/FRT82B P{w+, arm-LacZ} P{w+, tub-Gal80} RpS3 genotype. Deviation from 50% reflects amplification and detection bias in the iPLEX method (see section on Mapping a single allele using recombination and iPLEX MassARRAY). This heterozygote genotype is used because the recombinant DNA is also heterozygous with the FRT82B P{w+, arm-LacZ} P{w+, tub-Gal80} RpS3 chromosome, in which background the M2-73 phenotype is assessed. (E) Normalized frequencies of reference alleles (derived from the Tb1 chromosome). Solid line, pooled recombinants exhibiting the M2-73 phenotype; dashed line, pooled recombinants lacking the M2-73 phenotype. Normalized allele frequency is calculated as (measured % Tb allele in recombinants)/(measured % Tb allele in (D) control) &#xD7; 100, to account for any amplification or detection bias in the MassARRAY method. The arrow indicates SNP 19,029,906 (+BDGP Release 6) where Tb SNPs are recovered in &#x223C;0% of phenotypically M2-73 recombinants and &#x223C;100% of phenotypically wild-type recombinants. This SNP is &#x223C;100 kb from a coding mutation in the Xrp1 gene (open circle in A)." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-1423119758" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;iPLEX MassARRAY genotyping maps the M2-73 mutant. (A, B) Recombinant mapping principles. &#x2018;X&#x2019; illustrates that meiotic recombination occurs between homologs. (A) A cartoon representation of recombinant chromosome arms 3R expected to retain the M2-73 phenotype. These should retain M2-73 SNPs proximal to and including the locus responsible (asterisk), and exhibit various crossover points distal to the locus. When DNA from 50 such recombinants is pooled, the frequency of M2-73 SNPs is expected to decrease distal to the locus as the frequency of &lt;em&gt;Tb&lt;/em&gt; SNPs increases. (B) A cartoon representation of recombinant chromosome arms 3R expected to lack the M2-73 phenotype. These should retain &lt;em&gt;Tb&lt;/em&gt; SNPs distal to and including locus responsible for the M2-73 phenotype, and exhibit crossover points proximal to the locus. When DNA from such recombinants is pooled, the frequency of M2-73 SNPs should increase proximally to the locus as the frequency of &lt;em&gt;Tb&lt;/em&gt; SNPs decreases. Comparing observed SNP frequencies for the two pools should limit the mutant locus to a chromosome segment that retains 100% M2-73 SNPs in phenotypically mutant recombinants and 100% &lt;em&gt;Tb&lt;/em&gt; SNPs in phenotypically wild-type recombinants. (C) Distribution of the 17 SNPs that were assessed by iPLEX (crosses) and 22 exonic variants on the M2-73 chromosome (circles). An open circle shows the position of the coding mutation in Xrp1, closed circles show the other mutations. Some of the coding alleles were used as SNPs, in addition to noncoding SNPs. (D) Allele frequencies determined by iPLEX for 17 SNPs in the FRT82B M2-73/FRT82B P{w+, &lt;em&gt;arm&lt;/em&gt;-LacZ} P{w+, &lt;em&gt;tub&lt;/em&gt;-Gal80} &lt;em&gt;RpS3&lt;/em&gt; genotype. Deviation from 50% reflects amplification and detection bias in the iPLEX method (see section on Mapping a single allele using recombination and iPLEX MassARRAY). This heterozygote genotype is used because the recombinant DNA is also heterozygous with the FRT82B P{w+, &lt;em&gt;arm&lt;/em&gt;-LacZ} P{w+, &lt;em&gt;tub&lt;/em&gt;-Gal80} &lt;em&gt;RpS3&lt;/em&gt; chromosome, in which background the M2-73 phenotype is assessed. (E) Normalized frequencies of reference alleles (derived from the &lt;em&gt;Tb&lt;/em&gt;&lt;sup&gt;1&lt;/sup&gt; chromosome). Solid line, pooled recombinants exhibiting the M2-73 phenotype; dashed line, pooled recombinants lacking the M2-73 phenotype. Normalized allele frequency is calculated as (measured % &lt;em&gt;Tb&lt;/em&gt; allele in recombinants)/(measured % &lt;em&gt;Tb&lt;/em&gt; allele in (D) control) &#xD7; 100, to account for any amplification or detection bias in the MassARRAY method. The arrow indicates SNP 19,029,906 (+BDGP Release 6) where &lt;em&gt;Tb&lt;/em&gt; SNPs are recovered in &#x223C;0% of phenotypically M2-73 recombinants and &#x223C;100% of phenotypically wild-type recombinants. This SNP is &#x223C;100 kb from a coding mutation in the &lt;em&gt;Xrp1&lt;/em&gt; gene (open circle in A).&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/10/3207/F4.medium.gif" width="440" height="236"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/10/3207/F4.medium.gif" width="440" height="236"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/10/3207/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/10/3207/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469437" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-47" class="first-child">iPLEX MassARRAY genotyping maps the M2-73 mutant. (A, B) Recombinant mapping principles. ‘X’ illustrates that meiotic recombination occurs between homologs. (A) A cartoon representation of recombinant chromosome arms 3R expected to retain the M2-73 phenotype. These should retain M2-73 SNPs proximal to and including the locus responsible (asterisk), and exhibit various crossover points distal to the locus. When DNA from 50 such recombinants is pooled, the frequency of M2-73 SNPs is expected to decrease distal to the locus as the frequency of <em>Tb</em> SNPs increases. (B) A cartoon representation of recombinant chromosome arms 3R expected to lack the M2-73 phenotype. These should retain <em>Tb</em> SNPs distal to and including locus responsible for the M2-73 phenotype, and exhibit crossover points proximal to the locus. When DNA from such recombinants is pooled, the frequency of M2-73 SNPs should increase proximally to the locus as the frequency of <em>Tb</em> SNPs decreases. Comparing observed SNP frequencies for the two pools should limit the mutant locus to a chromosome segment that retains 100% M2-73 SNPs in phenotypically mutant recombinants and 100% <em>Tb</em> SNPs in phenotypically wild-type recombinants. (C) Distribution of the 17 SNPs that were assessed by iPLEX (crosses) and 22 exonic variants on the M2-73 chromosome (circles). An open circle shows the position of the coding mutation in Xrp1, closed circles show the other mutations. Some of the coding alleles were used as SNPs, in addition to noncoding SNPs. (D) Allele frequencies determined by iPLEX for 17 SNPs in the FRT82B M2-73/FRT82B P{w+, <em>arm</em>-LacZ} P{w+, <em>tub</em>-Gal80} <em>RpS3</em> genotype. Deviation from 50% reflects amplification and detection bias in the iPLEX method (see section on Mapping a single allele using recombination and iPLEX MassARRAY). This heterozygote genotype is used because the recombinant DNA is also heterozygous with the FRT82B P{w+, <em>arm</em>-LacZ} P{w+, <em>tub</em>-Gal80} <em>RpS3</em> chromosome, in which background the M2-73 phenotype is assessed. (E) Normalized frequencies of reference alleles (derived from the <em>Tb</em><sup>1</sup> chromosome). Solid line, pooled recombinants exhibiting the M2-73 phenotype; dashed line, pooled recombinants lacking the M2-73 phenotype. Normalized allele frequency is calculated as (measured % <em>Tb</em> allele in recombinants)/(measured % <em>Tb</em> allele in (D) control) × 100, to account for any amplification or detection bias in the MassARRAY method. The arrow indicates SNP 19,029,906 (+BDGP Release 6) where <em>Tb</em> SNPs are recovered in ∼0% of phenotypically M2-73 recombinants and ∼100% of phenotypically wild-type recombinants. This SNP is ∼100 kb from a coding mutation in the <em>Xrp1</em> gene (open circle in A).</p><div class="sb-div caption-clear"></div></div></div><p id="p-48">In a refinement step, eight additional polymorphic sites within the critical region were assessed with the same recombinant pools. The combined results with all 17 polymorphisms are shown in <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4E</a>. The combined results show that the pool of phenotypically wild-type recombinants exhibited only wild-type alleles from the distal tip of 3R to the SNP at 19,029,906 bp (91E4), whereas the pool of phenotypically mutant recombinants exhibited only mutant alleles from the proximal end of 3R to the SNP at 19,029,906 bp. This indicates that the mutation causing the M2-73 phenotype mapped close to 19,029,906 bp (+BDGP Release 6, 91E4); the results exclude loci left of 17,807,692 bp, where some phenotypically wild-type recombinants have M2-73 alleles, and right of 19,588,401 bp, where some phenotypically mutant recombinants have wild-type alleles (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4E</a>). Among the 22 nonsynonymous coding mutations on the right arm of the M2-73 third chromosome, one satisfied these criteria. This was the G to T transition in the second exon of the <em><a href="http://flybase.org/reports/FBgn0261113.html">Xrp1</a></em> gene (position 18,925,491 bp, +BDGP Release 6), which predicted a nonsense mutation in the open reading frame (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4C</a>).</p></div></div><div class="section discussion" id="sec-16"><h2 class="">Discussion</h2><p id="p-49">We applied whole-genome sequencing and iPLEX MassARRAY mapping strategies to mutations with potential effects on cell competition, a process occurring in genetic mosaics. Molecular identification of a mutation with one allele was possible without use of complementation tests. These methods may be useful for mutations responsible for other phenotypes in genetic mosaics or that are otherwise problematic in more conventional approaches.</p><p id="p-50">Genome sequencing has advanced significantly since first applied to <em>Drosophila</em> mutants. The Illumina HiSeq2000 platform achieves 35 Gb of sequence per lane. Our experience suggests that 35 Gb of sequence is sufficient to identify coding mutations from at least eight mutant strains. We did not see much increase in the rate of coding mutation identification from a further doubling of read depth. In practice, the proportion of the genome adequately covered (∼95%) increased little with greater read depth, suggesting that the easily sequenced (or easily assembled) genome regions have already been adequately covered. Haelterman <em>et al.</em> reported similar conclusions (<a id="xref-ref-23-4" class="xref-bibr" href="#ref-23">Haelterman <em>et al.</em> 2014</a>).</p><p id="p-51">We adapted a simple yet effective strategy, first implemented by Gerhold <em>et al.</em> (<a id="xref-ref-20-9" class="xref-bibr" href="#ref-20">Gerhold <em>et al.</em> 2011</a>), whereby sequencing transheterozygotes between mutant strains automates the removal of strain polymorphisms from the analysis. It is now possible to implement this strategy using publicly available software. Using this method, none of 65 mutations we confirmed by Sanger sequencing represented a polymorphism between FRT82B and reference strains. Under our least stringent sequence analysis, false-positive rates due to other causes were only 2.8–5.9%, depending on read depth. By contrast, a recent study that sequenced balanced mutant stocks in a conventional manner required three additional filter steps after variant calling to eliminate various other sources of variation. These were: removing polymorphisms distinguishing the FRT and reference genes; removing polymorphisms segregating in the progenitor strain; filtering genes that repeatedly yield false-positive results, at the end of which only 1% of the initial sequence variants remained as candidate novel mutations (<a id="xref-ref-23-5" class="xref-bibr" href="#ref-23">Haelterman <em>et al.</em> 2014</a>).</p><p id="p-52">Our results suggest little reason to impose sequence quality filters to identify coding mutations. Although unfiltered sequence led to more false positives, it permitted the recovery of significantly more mutations, especially at lower read depth (an average of 9.9 more verified coding mutations confirmed at lower read depth, 4.8 more coding mutations confirmed at higher read depth).</p><p id="p-53">We do not have a direct estimate of the false-negative rate, <em>i.e.</em>, mutations that were not detected by whole-genome sequencing. When the estimated false-positive rate is subtracted, we estimate that on average 22.4 novel 3R coding mutations would be confirmed among samples at 23× coverage, and 21.2 among samples at 50× coverage. The failure of increased coverage to identify more mutations is consistent with a low false-negative rate. We also notice that the mutation frequency in our study was one mutation per 94.0–124.6 kb (corresponding to 8.0–10.6 mutations per Mb), which is as high or higher than in previous studies, even though we employed a lower mutagen concentration (<a id="xref-table-wrap-2-2" class="xref-table" href="#T2">Table 2</a>). This suggests that earlier studies might have had higher false-negative rates, perhaps as a result of lower read depths and application of filters to eliminate the large number of background sequence variants. If these interpretations are valid, then following the procedures outlined in this paper should identify EMS-induced mutations with reasonably high accuracy and completeness.</p><p id="p-54">The data in <a id="xref-table-wrap-1-8" class="xref-table" href="#T1">Table 1</a> suggest that prediction of noncoding variants may be less accurate than for coding variants. Because increased read depth resulted in a significant reduction in the total number of sequence variants predicted on 3R when no sequence quality filters were applied, ∼90% of which are noncoding, it seems likely that 50× coverage significantly reduces false positives in noncoding sequences. We performed too few Sanger sequencing validations of noncoding mutations to quantify this (five noncoding mutations were sequenced and verified in the course of designing primer pairs for iPLEX mapping). Reasons that higher false-positive rates might be expected in noncoding sequences include reduced sequence complexity and more repeated sequences in noncoding DNA.</p><p id="p-55">We have isolated 13 mutants that reduce the elimination of <em>Rp</em>/+ cells from mosaic eye discs and identified two of these loci, <em>Atm</em> and <em><a href="http://flybase.org/reports/FBgn0261113.html">Xrp1</a></em>. These loci were identified following whole-genome sequencing through distinct approaches. Multiple <em>Atm</em> mutants identified a lethal complementation group, so it was evident that the locus in common between the strains was likely responsible, as was confirmed by replicating the phenotype with a pre-existing allele. Such approaches often permit the identification of mutants after whole-genome sequencing (<a id="xref-ref-10-8" class="xref-bibr" href="#ref-10">Blumenstiel <em>et al.</em> 2009</a>; <a id="xref-ref-20-10" class="xref-bibr" href="#ref-20">Gerhold <em>et al.</em> 2011</a>; <a id="xref-ref-23-6" class="xref-bibr" href="#ref-23">Haelterman <em>et al.</em> 2014</a>). Other mutations did not fall into lethal complementation groups, however, and genome sequencing identified no other mutant loci in common. Since the cell competition phenotype can only be assessed in genetic mosaics, approaches based on collections of deficiencies, duplications, or pre-exiting mutations were not appropriate. The remaining strategy was to use recombination to map phenotypes to intervals containing only one mutation. A limitation then is how many recombinant chromosomes must be both genotyped and assessed phenotypically in genetic mosaics. In principle, the results could also have been obtained using DNA sequencing of individual PCR products to characterize recombinants. To save time and expense, we exploited iPLEX MassARRAY methods to map cell competition phenotypes in an efficient and economical manner. iPLEX MassARRAY provides a way to genotype multiple SNPs in pools of recombinants in a single step, without additional sequencing reactions. This approach may be generally useful in mutation mapping following genome sequencing, especially mutations were allelism is not established and where phenotypic evaluation may be time-consuming. iPLEX MassARRAY technology is widely used in human genetics and available in many medical schools, as well as from the spectrometer manufacturer (Agena Bioscience). Mapping could also have been conducted using Sanger sequencing for genotyping, but this might have required thousands of sequencing reactions.</p><p id="p-56">Mutations in <em>Atm</em> appear to select for <em>Rp</em>/+ cells in the mosaic eyes by eliminating <em>Atm</em><sup>−/−</sup> cells through a mechanism independent of cell competition. Elevated cell death in <em>Atm</em> mutant cells occurs due to chromosome segregation defects including telomere fusion and anaphase bridges (<a id="xref-ref-37-2" class="xref-bibr" href="#ref-37">Oikemus <em>et al.</em> 2004</a>). Although we aimed to avoid mutations with competition-independent effects on growth through a prescreen of mosaic flies at the F1 generation (<a id="xref-fig-2-9" class="xref-fig" href="#F2">Figure 2, A, C, and D</a>), perdurance may have allowed the <em>Atm</em> mutation to pass this filter: <em>Atm</em> homozygous mutant clones are only lost after 72 hr of growth in wing imaginal discs. Because the F1 generation screened <em>Atm</em> homozygous clones in a non-Minute background, in which larval growth is completed ∼2 d earlier than in the Minute background where the effect of cell competition is assessed, the F1 generation may not provide a sufficiently stringent assay to identify mutations that affect noncompetitive growth but exhibit significant perdurance effects.</p><p id="p-57">A mutation in <em><a href="http://flybase.org/reports/FBgn0261113.html">Xrp1</a></em>, by contrast, remains a candidate to affect cell competition. <em><a href="http://flybase.org/reports/FBgn0261113.html">Xrp1</a></em> encodes a DNA-binding bZIP protein whose expression is induced in response to irradiation and other stresses (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Brodsky <em>et al.</em> 2004</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Akdemir <em>et al.</em> 2007</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Gruenewald <em>et al.</em> 2009</a>; <a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Vermeulen <em>et al.</em> 2013</a>). <em><a href="http://flybase.org/reports/FBgn0261113.html">Xrp1</a></em> is not required for cell or organismal viability (<a id="xref-ref-3-2" class="xref-bibr" href="#ref-3">Akdemir <em>et al.</em> 2007</a>). We expect to describe the role of <em><a href="http://flybase.org/reports/FBgn0261113.html">Xrp1</a></em> in cell competition in more detail elsewhere.</p><p id="p-58">Identification of newly-induced point mutations following genetic screens can be a bottleneck, especially when multiple alleles are not obtained or when complementation analyses with pre-existing mutant, deficiency or duplication collections are inconvenient. We present evidence that whole-genome sequencing and variant mapping methods are adequate to identify even single alleles of nonessential genes.</p></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-59">We thank A. Jenny, M. Kiparaki, and J. Secombe for comments on the manuscript, and H. Bellen and I. Hariharan for advice. Confocal Imaging was performed at the Analytical Imaging Facility, Albert Einstein College of Medicine, supported by National Cancer Institute cancer center support grant (P30CA013330). Whole-genome sequencing, subsequent analysis for variant calling, and iPLEX MassARRAY were carried out by the Epigenomics Core, Computational Genomics Core, and the Genomics Core of Albert Einstein College of Medicine, respectively. Stocks obtained from the Bloomington <em>Drosophila</em> Stock Center [National Institutes of Health (NIH) P40OD018537] were used in this study. Stocks obtained from the Bloomington <em>Drosophila</em> Stock Center (NIH P40OD018537) were used in this study. Research supported by grants from the NIH (EY018910 and GM104213) and Research to Prevent Blindness (2013 Senior Investigator Award to N.E.B. and unrestricted grant to the Department of Ophthalmology and Visual Sciences). Some data in this paper are from a thesis submitted in partial fulfillment of the requirements for the Degree of Doctor of Philosophy in the Graduate Division of Medical Sciences, Albert Einstein College of Medicine, Yeshiva University to C.H.L.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-6"><p id="p-60">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029421/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.029421/-/DC1</a>.</p></li><li class="fn" id="fn-7"><p id="p-61"><em>Communicating editor: B. Oliver</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-07-06"><span class="received-label">Received </span>July 6, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-08-04"><span class="accepted-label">Accepted </span>August 4, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Lee <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.1" data-doi="10.1101/gad.914401"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abraham</span>  <span class="cit-name-given-names">R. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Cell cycle checkpoint signaling through the ATM and ATR kinases.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">2177</span>–<span class="cit-lpage">2196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DAbraham%26rft.auinit1%253DR.%2BT.%26rft.volume%253D15%26rft.issue%253D17%26rft.spage%253D2177%26rft.epage%253D2196%26rft.atitle%253DCell%2Bcycle%2Bcheckpoint%2Bsignaling%2Bthrough%2Bthe%2BATM%2Band%2BATR%2Bkinases%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.914401%26rft_id%253Dinfo%253Apmid%252F11544175%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE1LzE3LzIxNzciO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMyMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.2" data-doi="10.1126/science.287.5461.2185"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Celniker</span>  <span class="cit-name-given-names">S. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holt</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gocayne</span>  <span class="cit-name-given-names">J. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">The genome sequence of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">287</span>: <span class="cit-fpage">2185</span>–<span class="cit-lpage">2195</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DAdams%26rft.auinit1%253DM.%2BD.%26rft.volume%253D287%26rft.issue%253D5461%26rft.spage%253D2185%26rft.epage%253D2195%26rft.atitle%253DThe%2BGenome%2BSequence%2Bof%2BDrosophila%2Bmelanogaster%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.287.5461.2185%26rft_id%253Dinfo%253Apmid%252F10731132%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyODcvNTQ2MS8yMTg1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMjA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.3" data-doi="10.1038/sj.onc.1210328"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Akdemir</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Christich</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sogame</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chapo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Abrams</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">p53 directs focused genomic responses in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Oncogene</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">5184</span>–<span class="cit-lpage">5193</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DOncogene%26rft.stitle%253DOncogene%26rft.aulast%253DAkdemir%26rft.auinit1%253DF.%26rft.volume%253D26%26rft.issue%253D36%26rft.spage%253D5184%26rft.epage%253D5193%26rft.atitle%253Dp53%2Bdirects%2Bfocused%2Bgenomic%2Bresponses%2Bin%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.onc.1210328%26rft_id%253Dinfo%253Apmid%252F17310982%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/sj.onc.1210328&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17310982&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3207.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000248674200002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.4" data-doi="10.1242/dev.079129"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Amoyel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bach</span>  <span class="cit-name-given-names">E. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Cell competition: how to eliminate your neighbours.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">988</span>–<span class="cit-lpage">1000</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DAmoyel%26rft.auinit1%253DM.%26rft.volume%253D141%26rft.issue%253D5%26rft.spage%253D988%26rft.epage%253D1000%26rft.atitle%253DCell%2Bcompetition%253A%2Bhow%2Bto%2Beliminate%2Byour%2Bneighbours%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.079129%26rft_id%253Dinfo%253Apmid%252F24550108%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czo5OiIxNDEvNS85ODgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMyMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andersson</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saeboe-Larssen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lambertsson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Merriam</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jacobs-Lorena</span>  <span class="cit-name-given-names">M.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">A Drosophila third chromosome Minute locus encodes a ribosomal protein.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">513</span>–<span class="cit-lpage">520</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DAndersson%26rft.auinit1%253DS.%26rft.volume%253D137%26rft.issue%253D2%26rft.spage%253D513%26rft.epage%253D520%26rft.atitle%253DA%2BDrosophila%2Bthird%2Bchromosome%2BMinute%2Blocus%2Bencodes%2Ba%2Bribosomal%2Bprotein.%26rft_id%253Dinfo%253Apmid%252F8070662%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM3LzIvNTEzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMjA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.6" data-doi="10.1016/j.cub.2010.11.030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">N. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Cell competition.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">R11</span>–<span class="cit-lpage">R15</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DBaker%26rft.auinit1%253DN.%2BE.%26rft.volume%253D21%26rft.issue%253D1%26rft.spage%253DR11%26rft.epage%253DR15%26rft.atitle%253DCell%2Bcompetition.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2010.11.030%26rft_id%253Dinfo%253Apmid%252F21215926%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2010.11.030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21215926&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3207.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.7" data-doi="10.1016/j.ymeth.2014.04.007"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">N. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Quiquand</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruggiero</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">L.-H.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Eye development.</span> <abbr class="cit-jnl-abbrev">Methods</abbr> <span class="cit-vol">68</span>: <span class="cit-fpage">252</span>–<span class="cit-lpage">259</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMethods%26rft.volume%253D68%26rft.spage%253D252%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.ymeth.2014.04.007%26rft_id%253Dinfo%253Apmid%252F24784530%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.ymeth.2014.04.007&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24784530&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3207.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.8" data-doi="10.1534/genetics.111.126995"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bellen</span>  <span class="cit-name-given-names">H. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levis</span>  <span class="cit-name-given-names">R. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">He</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans-Holm</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The <em>Drosophila</em> gene disruption project: progress using transposons with distinctive site specificities.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">188</span>: <span class="cit-fpage">731</span>–<span class="cit-lpage">743</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.126995%26rft_id%253Dinfo%253Apmid%252F21515576%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTg4LzMvNzMxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMjA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.9" data-doi="10.1126/science.221.4605.23"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bender</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Akam</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karch</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beachy</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peifer</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1983</span> <span class="cit-article-title">Molecular genetics of the bithorax complex in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">221</span>: <span class="cit-fpage">23</span>–<span class="cit-lpage">29</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DBender%26rft.auinit1%253DW.%26rft.volume%253D221%26rft.issue%253D4605%26rft.spage%253D23%26rft.epage%253D29%26rft.atitle%253DMolecular%2BGenetics%2Bof%2Bthe%2BBithorax%2BComplex%2Bin%2BDrosophila%2Bmelanogaster%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.221.4605.23%26rft_id%253Dinfo%253Apmid%252F17737996%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIyMjEvNDYwNS8yMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTAvMzIwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.10" data-doi="10.1534/genetics.109.101998"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blumenstiel</span>  <span class="cit-name-given-names">J. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Noll</span>  <span class="cit-name-given-names">A. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Griffiths</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perera</span>  <span class="cit-name-given-names">A. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walton</span>  <span class="cit-name-given-names">K. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Identification of EMS-induced mutations in <em>Drosophila melanogaster</em> by whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">182</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">32</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.101998%26rft_id%253Dinfo%253Apmid%252F19307605%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6ODoiMTgyLzEvMjUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMyMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.11" data-doi="10.1128/MCB.24.3.1219-1231.2004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brodsky</span>  <span class="cit-name-given-names">M. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weinert</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tsang</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rong</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGinnis</span>  <span class="cit-name-given-names">N. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title"><em>Drosophila melanogaster</em> MNK/Chk2 and p53 regulate multiple DNA repair and apoptotic pathways following DNA damage.</span> <abbr class="cit-jnl-abbrev">Mol. Cell. Biol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1219</span>–<span class="cit-lpage">1231</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Band%2BCellular%2BBiology%26rft.stitle%253DMol.%2BCell.%2BBiol.%26rft.aulast%253DBrodsky%26rft.auinit1%253DM.%2BH.%26rft.volume%253D24%26rft.issue%253D3%26rft.spage%253D1219%26rft.epage%253D1231%26rft.atitle%253DDrosophila%2Bmelanogaster%2BMNK%252FChk2%2Band%2Bp53%2BRegulate%2BMultiple%2BDNA%2BRepair%2Band%2BApoptotic%2BPathways%2Bfollowing%2BDNA%2BDamage%26rft_id%253Dinfo%253Adoi%252F10.1128%252FMCB.24.3.1219-1231.2004%26rft_id%253Dinfo%253Apmid%252F14729967%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibWNiIjtzOjU6InJlc2lkIjtzOjk6IjI0LzMvMTIxOSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTAvMzIwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.12" data-doi="10.1073/pnas.98.2.581"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Buetow</span>  <span class="cit-name-given-names">K. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Edmonson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">MacDonald</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Clifford</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yip</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">High-throughput development and characterization of a genomewide collection of gene-based single nucleotide polymorphism markers by chip-based matrix-assisted laser desorption/ionization time-of-flight mass spectrometry.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">98</span>: <span class="cit-fpage">581</span>–<span class="cit-lpage">584</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DBuetow%26rft.auinit1%253DK.%2BH.%26rft.volume%253D98%26rft.issue%253D2%26rft.spage%253D581%26rft.epage%253D584%26rft.atitle%253DHigh-throughput%2Bdevelopment%2Band%2Bcharacterization%2Bof%2Ba%2Bgenomewide%2Bcollection%2Bof%2Bgene-based%2Bsingle%2Bnucleotide%2Bpolymorphism%2Bmarkers%2Bby%2Bchip-based%2Bmatrix-assisted%2Blaser%2Bdesorption%252Fionization%2Btime-of-flight%2Bmass%2Bspectrometry%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.98.2.581%26rft_id%253Dinfo%253Apmid%252F11136232%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo4OiI5OC8yLzU4MSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTAvMzIwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.13"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chou</span>  <span class="cit-name-given-names">T. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perrimon</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1996</span> <span class="cit-article-title">The autosomal Flp-Dfs technique for generating germline mosaics in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">144</span>: <span class="cit-fpage">1673</span>–<span class="cit-lpage">1679</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DChou%26rft.auinit1%253DT.%2BB.%26rft.volume%253D144%26rft.issue%253D4%26rft.spage%253D1673%26rft.epage%253D1679%26rft.atitle%253DThe%2BAutosomal%2BFLP-DFS%2BTechnique%2Bfor%2BGenerating%2BGermline%2BMosaics%2Bin%2BDrosophila%2Bmelanogaster%26rft_id%253Dinfo%253Apmid%252F8978054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE0NC80LzE2NzMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMyMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.14" data-doi="10.1534/genetics.108.092437"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cooper</span>  <span class="cit-name-given-names">J. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greene</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Till</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Codomo</span>  <span class="cit-name-given-names">C. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wakimoto</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Retention of induced mutations in a <em>Drosophila</em> reverse-genetic resource.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">180</span>: <span class="cit-fpage">661</span>–<span class="cit-lpage">667</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.108.092437%26rft_id%253Dinfo%253Apmid%252F18780737%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTgwLzEvNjYxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMjA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.15" data-doi="10.1002/dvdy.23783"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">de Beco</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ziosi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Johnston</span>  <span class="cit-name-given-names">L. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">New frontiers in cell competition.</span> <abbr class="cit-jnl-abbrev">Dev. Dyn.</abbr> <span class="cit-vol">241</span>: <span class="cit-fpage">831</span>–<span class="cit-lpage">841</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bdynamics%2B%253A%2B%2Ban%2Bofficial%2Bpublication%2Bof%2Bthe%2BAmerican%2BAssociation%2Bof%2BAnatomists%26rft.stitle%253DDev%2BDyn%26rft.aulast%253Dde%2BBeco%26rft.auinit1%253DS.%26rft.volume%253D241%26rft.issue%253D5%26rft.spage%253D831%26rft.epage%253D841%26rft.atitle%253DNew%2Bfrontiers%2Bin%2Bcell%2Bcompetition.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fdvdy.23783%26rft_id%253Dinfo%253Apmid%252F22438309%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/dvdy.23783&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22438309&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3207.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.16" data-doi="10.1038/ng.806"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">DePristo</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poplin</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garimella</span>  <span class="cit-name-given-names">K. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maguire</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">491</span>–<span class="cit-lpage">498</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DDepristo%26rft.auinit1%253DM.%2BA.%26rft.volume%253D43%26rft.issue%253D5%26rft.spage%253D491%26rft.epage%253D498%26rft.atitle%253DA%2Bframework%2Bfor%2Bvariation%2Bdiscovery%2Band%2Bgenotyping%2Busing%2Bnext-generation%2BDNA%2Bsequencing%2Bdata.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.806%26rft_id%253Dinfo%253Apmid%252F21478889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.806&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21478889&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3207.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000289972600023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.17" data-doi="10.1073/pnas.0630494100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantor</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">A high-throughput gene expression analysis technique using competitive PCR and matrix-assisted laser desorption ionization time-of-flight MS.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">3059</span>–<span class="cit-lpage">3064</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0630494100%26rft_id%253Dinfo%253Apmid%252F12624187%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTAwLzYvMzA1OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTAvMzIwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.18" data-doi="10.1186/1471-2156-5-8"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ding</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maier</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roscher</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Braun</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cantor</span>  <span class="cit-name-given-names">C. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Simultaneous quantitative and allele-specific expression analysis with real competitive PCR.</span> <abbr class="cit-jnl-abbrev">BMC Genet.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenetics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenet%26rft.aulast%253DDing%26rft.auinit1%253DC.%26rft.volume%253D5%26rft.spage%253D8%26rft.epage%253D8%26rft.atitle%253DSimultaneous%2Bquantitative%2Band%2Ballele-specific%2Bexpression%2Banalysis%2Bwith%2Breal%2Bcompetitive%2BPCR.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2156-5-8%26rft_id%253Dinfo%253Apmid%252F15128429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2156-5-8&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15128429&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3207.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.19"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gabriel</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ziaugra</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tabbaa</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-comment">Curr. Protoc. Hum. Genet. Chapter 2: Unit 2.12.</span></cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.20" data-doi="10.1534/genetics.111.132993"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gerhold</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richter</span>  <span class="cit-name-given-names">D. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yu</span>  <span class="cit-name-given-names">A. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hariharan</span>  <span class="cit-name-given-names">I. K.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Identification and characterization of genes required for compensatory growth in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">189</span>: <span class="cit-fpage">1309</span>–<span class="cit-lpage">1326</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.111.132993%26rft_id%253Dinfo%253Apmid%252F21926302%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4OS80LzEzMDkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMyMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.21"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonzalez</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Booven</span>  <span class="cit-name-given-names">D. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hulme</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ulloa</span>  <span class="cit-name-given-names">R. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acosta Lebrigio</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Whole genome sequencing and a new bioinformatics platform allow for rapid gene identification in <em>D. melanogaster</em> EMS screens.</span> <abbr class="cit-jnl-abbrev">Biology (Basel)</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">766</span>–<span class="cit-lpage">777</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiology%2B%2528Basel%2529%26rft.volume%253D1%26rft.spage%253D766%26rft_id%253Dinfo%253Apmid%252F24832518%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=24832518&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3207.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.22" data-doi="10.1016/j.freeradbiomed.2009.03.025"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gruenewald</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Botella</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bayersdorfer</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Navarro</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneuwly</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Hyperoxia-induced neurodegeneration as a tool to identify neuroprotective genes in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Free Radic. Biol. Med.</abbr> <span class="cit-vol">46</span>: <span class="cit-fpage">1668</span>–<span class="cit-lpage">1676</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFree%2Bradical%2Bbiology%2B%2526%2Bmedicine%26rft.stitle%253DFree%2BRadic%2BBiol%2BMed%26rft.aulast%253DGruenewald%26rft.auinit1%253DC.%26rft.volume%253D46%26rft.issue%253D12%26rft.spage%253D1668%26rft.epage%253D1676%26rft.atitle%253DHyperoxia-induced%2Bneurodegeneration%2Bas%2Ba%2Btool%2Bto%2Bidentify%2Bneuroprotective%2Bgenes%2Bin%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.freeradbiomed.2009.03.025%26rft_id%253Dinfo%253Apmid%252F19345730%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.freeradbiomed.2009.03.025&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19345730&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3207.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000266358000015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.23" data-doi="10.1101/gr.174615.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Haelterman</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bayat</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sandoval</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Large-scale identification of chemically induced mutations in <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">1707</span>–<span class="cit-lpage">1718</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.174615.114%26rft_id%253Dinfo%253Apmid%252F25258387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjEwOiIyNC8xMC8xNzA3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMjA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.24" data-doi="10.1126/science.1139816"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoskins</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">J. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kennedy</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Acevedo</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Evans-Holm</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Sequence finishing and mapping of <em>Drosophila melanogaster</em> heterochromatin.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">316</span>: <span class="cit-fpage">1625</span>–<span class="cit-lpage">1628</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DHoskins%26rft.auinit1%253DR.%2BA.%26rft.volume%253D316%26rft.issue%253D5831%26rft.spage%253D1625%26rft.epage%253D1628%26rft.atitle%253DSequence%2BFinishing%2Band%2BMapping%2Bof%2BDrosophila%2Bmelanogaster%2BHeterochromatin%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1139816%26rft_id%253Dinfo%253Apmid%252F17569867%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTYvNTgzMS8xNjI1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMjA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.25" data-doi="10.1038/cdd.2014.218"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kale</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">N. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Apoptotic mechanisms during competition of ribosomal protein mutant cells: roles of the initiator caspases Dronc and Dream/Strica.</span> <abbr class="cit-jnl-abbrev">Cell Death Differ.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1300</span>–<span class="cit-lpage">1312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BDeath%2BDiffer.%26rft.volume%253D22%26rft.spage%253D1300%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fcdd.2014.218%26rft_id%253Dinfo%253Apmid%252F25613379%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/cdd.2014.218&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25613379&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3207.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.26" data-doi="10.1038/nprot.2009.86"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henikoff</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ng</span>  <span class="cit-name-given-names">P. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Predicting the effects of coding non-synonymous variants on protein function using the SIFT algorithm.</span> <abbr class="cit-jnl-abbrev">Nat. Protoc.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">1073</span>–<span class="cit-lpage">1081</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BProtoc%26rft.aulast%253DKumar%26rft.auinit1%253DP.%26rft.volume%253D4%26rft.issue%253D7%26rft.spage%253D1073%26rft.epage%253D1081%26rft.atitle%253DPredicting%2Bthe%2Beffects%2Bof%2Bcoding%2Bnon-synonymous%2Bvariants%2Bon%2Bprotein%2Bfunction%2Busing%2Bthe%2BSIFT%2Balgorithm.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnprot.2009.86%26rft_id%253Dinfo%253Apmid%252F19561590%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nprot.2009.86&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19561590&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3207.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000268858700007&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.27" data-doi="10.1016/S0065-2660(08)60142-X"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lambertsson</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The Minute genes in <em>Drosophila</em> and their molecular functions.</span> <abbr class="cit-jnl-abbrev">Adv. Genet.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">134</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAdvances%2Bin%2Bgenetics%26rft.stitle%253DAdv%2BGenet%26rft.aulast%253DLambertsson%26rft.auinit1%253DA.%26rft.volume%253D38%26rft.spage%253D69%26rft.epage%253D134%26rft.atitle%253DThe%2Bminute%2Bgenes%2Bin%2BDrosophila%2Band%2Btheir%2Bmolecular%2Bfunctions.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0065-2660%252808%252960142-X%26rft_id%253Dinfo%253Apmid%252F9677706%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0065-2660(08)60142-X&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9677706&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3207.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000079081300003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.28" data-doi="10.1016/S0896-6273(00)80701-1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">L.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Mosaic analysis with a repressible cell marker for studies of gene function in neuronal morphogenesis.</span> <abbr class="cit-jnl-abbrev">Neuron</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">451</span>–<span class="cit-lpage">461</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNeuron%26rft.stitle%253DNeuron%26rft.aulast%253DLee%26rft.auinit1%253DT.%26rft.volume%253D22%26rft.issue%253D3%26rft.spage%253D451%26rft.epage%253D461%26rft.atitle%253DMosaic%2Banalysis%2Bwith%2Ba%2Brepressible%2Bcell%2Bmarker%2Bfor%2Bstudies%2Bof%2Bgene%2Bfunction%2Bin%2Bneuronal%2Bmorphogenesis.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0896-6273%252800%252980701-1%26rft_id%253Dinfo%253Apmid%252F10197526%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0896-6273(00)80701-1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10197526&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3207.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000079483900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.29" data-doi="10.1083/jcb.201301051"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Levayer</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moreno</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Mechanisms of cell competition: themes and variations.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">200</span>: <span class="cit-fpage">689</span>–<span class="cit-lpage">698</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.201301051%26rft_id%253Dinfo%253Apmid%252F23509066%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjIwMC82LzY4OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTAvMzIwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.30" data-doi="10.1093/bioinformatics/btp324"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Durbin</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Fast and accurate short read alignment with Burrows-Wheeler transform.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">1754</span>–<span class="cit-lpage">1760</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtp324%26rft_id%253Dinfo%253Apmid%252F19451168%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjUvMTQvMTc1NCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTAvMzIwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.31" data-doi="10.1038/nature10811"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mackay</span>  <span class="cit-name-given-names">T. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Richards</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stone</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Barbadilla</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayroles</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The <em>Drosophila melanogaster</em> genetic reference panel.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">482</span>: <span class="cit-fpage">173</span>–<span class="cit-lpage">178</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMackay%26rft.auinit1%253DT.%2BF.%26rft.volume%253D482%26rft.issue%253D7384%26rft.spage%253D173%26rft.epage%253D178%26rft.atitle%253DThe%2BDrosophila%2Bmelanogaster%2BGenetic%2BReference%2BPanel.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10811%26rft_id%253Dinfo%253Apmid%252F22318601%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10811&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22318601&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3207.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000299994100029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.32" data-doi="10.1186/gb-2007-8-10-r216"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Marygold</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roote</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Reuter</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lambertsson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ashburner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The ribosomal protein genes and Minute loci of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">R216</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DMarygold%26rft.auinit1%253DS.%2BJ.%26rft.volume%253D8%26rft.issue%253D10%26rft.spage%253DR216%26rft.epage%253DR216%26rft.atitle%253DThe%2Bribosomal%2Bprotein%2Bgenes%2Band%2BMinute%2Bloci%2Bof%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2007-8-10-r216%26rft_id%253Dinfo%253Apmid%252F17927810%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2007-8-10-r216&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17927810&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3207.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.33" data-doi="10.1101/gr.107524.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McKenna</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Banks</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sivachenko</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cibulskis</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">1297</span>–<span class="cit-lpage">1303</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.107524.110%26rft_id%253Dinfo%253Apmid%252F20644199%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIwLzkvMTI5NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTAvMzIwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.34" data-doi="10.1016/0012-1606(75)90330-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Morata</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ripoll</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1975</span> <span class="cit-article-title">Minutes: mutants of <em>Drosophila</em> autonomously affecting cell division rate.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">42</span>: <span class="cit-fpage">211</span>–<span class="cit-lpage">221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DMorata%26rft.auinit1%253DG.%26rft.volume%253D42%26rft.issue%253D2%26rft.spage%253D211%26rft.epage%253D221%26rft.atitle%253DMinutes%253A%2Bmutants%2Bof%2Bdrosophila%2Bautonomously%2Baffecting%2Bcell%2Bdivision%2Brate.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0012-1606%252875%252990330-9%26rft_id%253Dinfo%253Apmid%252F1116643%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0012-1606(75)90330-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=1116643&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3207.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1975V703400002&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.35" data-doi="10.1038/416755a"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moreno</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Basler</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Morata</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Cells compete for decapentaplegic survival factor to prevent apoptosis in <em>Drosophila</em> wing development.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">416</span>: <span class="cit-fpage">755</span>–<span class="cit-lpage">759</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMoreno%26rft.auinit1%253DE.%26rft.volume%253D416%26rft.issue%253D6882%26rft.spage%253D755%26rft.epage%253D759%26rft.atitle%253DCells%2Bcompete%2Bfor%2Bdecapentaplegic%2Bsurvival%2Bfactor%2Bto%2Bprevent%2Bapoptosis%2Bin%2BDrosophila%2Bwing%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F416755a%26rft_id%253Dinfo%253Apmid%252F11961558%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/416755a&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11961558&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3207.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000175033500046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Newsome</span>  <span class="cit-name-given-names">T. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Åsling</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dickson</span>  <span class="cit-name-given-names">B. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Analysis of <em>Drosophila</em> photoreceptor axon guidance in eye-specific mosaics.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">127</span>: <span class="cit-fpage">851</span>–<span class="cit-lpage">860</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DNewsome%26rft.auinit1%253DT.%26rft.volume%253D127%26rft.issue%253D4%26rft.spage%253D851%26rft.epage%253D860%26rft.atitle%253DAnalysis%2Bof%2BDrosophila%2Bphotoreceptor%2Baxon%2Bguidance%2Bin%2Beye-specific%2Bmosaics%26rft_id%253Dinfo%253Apmid%252F10648243%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czo5OiIxMjcvNC84NTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMyMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.37" data-doi="10.1101/gad.1202504"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Oikemus</span>  <span class="cit-name-given-names">S. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McGinnis</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Queiroz-Machado</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tukachinsky</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Takada</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title"><em>Drosophila</em> atm/telomere fusion is required for telomeric localization of HP1 and telomere position effect.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">1850</span>–<span class="cit-lpage">1861</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.1202504%26rft_id%253Dinfo%253Apmid%252F15256487%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjE4LzE1LzE4NTAiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMyMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.38"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pedersen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Campbell</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Molecular genetic characterization of <em>Drosophila</em> ATM conserved functional domains.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">778</span>–<span class="cit-lpage">786</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DPedersen%26rft.auinit1%253DM.%26rft.volume%253D53%26rft.issue%253D10%26rft.spage%253D778%26rft.epage%253D786%26rft.atitle%253DMolecular%2Bgenetic%2Bcharacterization%2Bof%2BDrosophila%2BATM%2Bconserved%2Bfunctional%2Bdomains.%26rft_id%253Dinfo%253Apmid%252F20962884%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=20962884&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3207.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.39" data-doi="10.1186/gb4010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Roote</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Russell</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Toward a complete <em>Drosophila</em> deficiency kit.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">149</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DRoote%26rft.auinit1%253DJ.%26rft.volume%253D13%26rft.issue%253D3%26rft.spage%253D149%26rft.epage%253D149%26rft.atitle%253DToward%2Ba%2Bcomplete%2BDrosophila%2Bdeficiency%2Bkit.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb4010%26rft_id%253Dinfo%253Apmid%252F22445094%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb4010&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22445094&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3207.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.40" data-doi="10.1038/nmeth.1249"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sarin</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Prabhu</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Meara</span>  <span class="cit-name-given-names">M. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pe’er</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hobert</span>  <span class="cit-name-given-names">O.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title"><em>Caenorhabditis elegans</em> mutant allele identification by whole-genome sequencing.</span> <abbr class="cit-jnl-abbrev">Nat. Methods</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">865</span>–<span class="cit-lpage">867</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DNat%2BMethods%26rft.aulast%253DSarin%26rft.auinit1%253DS.%26rft.volume%253D5%26rft.issue%253D10%26rft.spage%253D865%26rft.epage%253D867%26rft.atitle%253DCaenorhabditis%2Belegans%2Bmutant%2Ballele%2Bidentification%2Bby%2Bwhole-genome%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnmeth.1249%26rft_id%253Dinfo%253Apmid%252F18677319%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nmeth.1249&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18677319&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3207.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000259650100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.41"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stampas</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zapata</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">N. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The pineapple eye gene is required for survival of <em>Drosophila</em> imaginal disc cells.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">1869</span>–<span class="cit-lpage">1879</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DShi%26rft.auinit1%253DW.%26rft.volume%253D165%26rft.issue%253D4%26rft.spage%253D1869%26rft.epage%253D1879%26rft.atitle%253DThe%2Bpineapple%2Beye%2BGene%2BIs%2BRequired%2Bfor%2BSurvival%2Bof%2BDrosophila%2BImaginal%2BDisc%2BCells%26rft_id%253Dinfo%253Apmid%252F14704172%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NS80LzE4NjkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMyMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.42" data-doi="10.1016/j.cub.2004.06.056"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Silva</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tiong</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pedersen</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Homola</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Royou</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">ATM is required for telomere maintenance and chromosome stability during Drosophila development.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1341</span>–<span class="cit-lpage">1347</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DSilva%26rft.auinit1%253DE.%26rft.volume%253D14%26rft.issue%253D15%26rft.spage%253D1341%26rft.epage%253D1347%26rft.atitle%253DATM%2Bis%2Brequired%2Bfor%2Btelomere%2Bmaintenance%2Band%2Bchromosome%2Bstability%2Bduring%2BDrosophila%2Bdevelopment.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2004.06.056%26rft_id%253Dinfo%253Apmid%252F15296750%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2004.06.056&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15296750&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3207.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000223306200025&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.43" data-doi="10.1016/0012-1606(79)90284-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1979</span> <span class="cit-article-title">Parameters of cell competition in the compartments of the wing disc of <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">182</span>–<span class="cit-lpage">193</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DSimpson%26rft.auinit1%253DP.%26rft.volume%253D69%26rft.issue%253D1%26rft.spage%253D182%26rft.epage%253D193%26rft.atitle%253DParameters%2Bof%2Bcell%2Bcompetition%2Bin%2Bthe%2Bcompartments%2Bof%2Bthe%2Bwing%2Bdisc%2Bof%2BDrosophila.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0012-1606%252879%252990284-7%26rft_id%253Dinfo%253Apmid%252F446891%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0012-1606(79)90284-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=446891&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3207.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1979GN89400015&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.44" data-doi="10.1038/nrg751"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">St. Johnston</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">The art and design of genetic screens: <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">3</span>: <span class="cit-fpage">176</span>–<span class="cit-lpage">188</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DSt%2BJohnston%26rft.auinit1%253DD.%26rft.volume%253D3%26rft.issue%253D3%26rft.spage%253D176%26rft.epage%253D188%26rft.atitle%253DThe%2Bart%2Band%2Bdesign%2Bof%2Bgenetic%2Bscreens%253A%2BDrosophila%2Bmelanogaster.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg751%26rft_id%253Dinfo%253Apmid%252F11972155%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg751&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11972155&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3207.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174159100014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.45" data-doi="10.1534/genetics.106.061929"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tyler</span>  <span class="cit-name-given-names">D. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhuo</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pellock</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baker</span>  <span class="cit-name-given-names">N. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genes affecting cell competition in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">643</span>–<span class="cit-lpage">657</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.061929%26rft_id%253Dinfo%253Apmid%252F17110495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc1LzIvNjQzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMjA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.46" data-doi="10.1534/genetics.110.121285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Venken</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Popodi</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holtzman</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulze</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Park</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A molecularly defined duplication set for the X chromosome of <em>Drosophila melanogaster</em>.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">186</span>: <span class="cit-fpage">1111</span>–<span class="cit-lpage">1125</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.110.121285%26rft_id%253Dinfo%253Apmid%252F20876565%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4Ni80LzExMTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMyMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.47" data-doi="10.1111/jeb.12183"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vermeulen</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sorensen</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kirilova Gagalova</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Loeschcke</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Transcriptomic analysis of inbreeding depression in cold-sensitive <em>Drosophila melanogaster</em> shows upregulation of the immune response.</span> <abbr class="cit-jnl-abbrev">J. Evol. Biol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">1890</span>–<span class="cit-lpage">1902</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BEvol.%2BBiol.%26rft.volume%253D26%26rft.spage%253D1890%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fjeb.12183%26rft_id%253Dinfo%253Apmid%252F23944235%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/jeb.12183&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23944235&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3207.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.48" data-doi="10.1006/dbio.1994.1203"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vincent</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Girdham</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">O’Farrell</span>  <span class="cit-name-given-names">P.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">A cell-autonomous, ubiquitous marker for the analysis of <em>Drosophila</em> genetic mosaics.</span> <abbr class="cit-jnl-abbrev">Dev. Biol.</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">328</span>–<span class="cit-lpage">331</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopmental%2Bbiology%26rft.stitle%253DDev%2BBiol%26rft.aulast%253DVincent%26rft.auinit1%253DJ.%2BP.%26rft.volume%253D164%26rft.issue%253D1%26rft.spage%253D328%26rft.epage%253D331%26rft.atitle%253DA%2Bcell-autonomous%252C%2Bubiquitous%2Bmarker%2Bfor%2Bthe%2Banalysis%2Bof%2BDrosophila%2Bgenetic%2Bmosaics.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fdbio.1994.1203%26rft_id%253Dinfo%253Apmid%252F8026635%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/dbio.1994.1203&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=8026635&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3207.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1994NW44400029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.49" data-doi="10.1042/bse0530069"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Vivarelli</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wagstaff</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Piddini</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Cell wars: regulation of cell survival and proliferation by cell competition.</span> <abbr class="cit-jnl-abbrev">Essays Biochem.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">69</span>–<span class="cit-lpage">82</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEssays%2BIn%2BBiochemistry%26rft.stitle%253DEssays%2BBiochem.%26rft.aulast%253DVivarelli%26rft.auinit1%253DS.%26rft.volume%253D53%26rft.issue%253D0%26rft.spage%253D69%26rft.epage%253D82%26rft.atitle%253DCell%2Bwars%253A%2Bregulation%2Bof%2Bcell%2Bsurvival%2Band%2Bproliferation%2Bby%2Bcell%2Bcompetition%26rft_id%253Dinfo%253Adoi%252F10.1042%252Fbse0530069%26rft_id%253Dinfo%253Apmid%252F22928509%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoicHBlYmlvIjtzOjU6InJlc2lkIjtzOjc6IjUzLzAvNjkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMyMDcuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.50" data-doi="10.1534/genetics.114.171785"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Wangler</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yamamoto</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bellen</span>  <span class="cit-name-given-names">H. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Fruit flies in biomedical research.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">199</span>: <span class="cit-fpage">639</span>–<span class="cit-lpage">653</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.171785%26rft_id%253Dinfo%253Apmid%252F25624315%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTk5LzMvNjM5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMjA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.51" data-doi="10.1101/gr.3721805"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Winkler</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schwabedissen</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Backasch</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bökel</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Seidel</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Target-selected mutant screen by TILLING in <em>Drosophila</em>.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">15</span>: <span class="cit-fpage">718</span>–<span class="cit-lpage">723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BResearch%26rft.stitle%253DGenome%2BRes%26rft.aulast%253DWinkler%26rft.auinit1%253DS.%26rft.volume%253D15%26rft.issue%253D5%26rft.spage%253D718%26rft.epage%253D723%26rft.atitle%253DTarget-selected%2Bmutant%2Bscreen%2Bby%2BTILLING%2Bin%2BDrosophila%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.3721805%26rft_id%253Dinfo%253Apmid%252F15867432%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE1LzUvNzE4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMjA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.52" data-doi="10.1126/science.1197142"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rubin</span>  <span class="cit-name-given-names">G. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">Analysis of genetic mosaics in the developing and adult <em>Drosophila</em> tissues.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">117</span>: <span class="cit-fpage">1223</span>–<span class="cit-lpage">1236</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DXu%26rft.auinit1%253DT%26rft.volume%253D117%26rft.issue%253D4%26rft.spage%253D1223%26rft.epage%253D1237%26rft.atitle%253DAnalysis%2Bof%2Bgenetic%2Bmosaics%2Bin%2Bdeveloping%2Band%2Badult%2BDrosophila%2Btissues%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1197142%26rft_id%253Dinfo%253Apmid%252F8404527%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMDoiMTE3LzQvMTIyMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTAvMzIwNy5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3207.53" data-doi="10.1073/pnas.1832753100"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhai</span>  <span class="cit-name-given-names">R. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hiesinger</span>  <span class="cit-name-given-names">P. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koh</span>  <span class="cit-name-given-names">T. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Verstreken</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schulze</span>  <span class="cit-name-given-names">K. L.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Mapping <em>Drosophila</em> mutations with molecularly defined P element insertions.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">10860</span>–<span class="cit-lpage">10865</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1832753100%26rft_id%253Dinfo%253Apmid%252F12960394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAwLzE5LzEwODYwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMjA3LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/10/3207.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/10/3197" title="Rapid Screening for CRISPR-Directed Editing of the Drosophila Genome Using white Coconversion" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/10/3219" title="The Genetic Architecture of Noise-Induced Hearing Loss: Evidence for a Gene-by-Environment Interaction" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/10"><span>Volume 6 Issue 10, October 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/10.cover-source.jpg?itok=eRzcKCYf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/10/3207" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-BeazNIP7B_6IxoNE-lfx7ywQV4qMf-FG41gDkIFw5vk" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/10/3207" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/10/3207" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/10/3207" class="active" data-icon-position="" data-hide-link-title="0">Whole-Genome Sequencing and iPLEX MassARRAY Genotyping Map an EMS-Induced Mutation Affecting Cell Competition in Drosophila melanogaster</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460342" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-NHpAXiKjUe36mwAhEr3qPSqNivuA8RgKL5OvZIrkjg0" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/10/3207.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/10/3207" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-alOPnDUQRaq6UoiqqmmQ8x3hAiw4qmeCDvdqjbLQEg4" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/10/3207" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/10/3207.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460342" id="node460342--2" data-pisa="ggg;6/10/3207" data-pisa-master="ggg;g3.116.029421" data-apath="/ggg/6/10/3207.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Whole-Genome Sequencing and iPLEX MassARRAY Genotyping Map an EMS-Induced Mutation Affecting Cell Competition in <em>Drosophila melanogaster</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Chang-Hyun Lee</span>, <span class="highwire-citation-author" data-delta="1">Gerard Rimesso</span>, <span class="highwire-citation-author" data-delta="2">David M. Reynolds</span>, <span class="highwire-citation-author" data-delta="3">Jinlu Cai</span> and <span class="highwire-citation-author" data-delta="4">Nicholas E. Baker</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3207-3217;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.029421 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460342--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Chang-Hyun Lee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Albert Einstein College of Medicine, Bronx, New York 10461</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chang-Hyun%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lee%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChang-Hyun%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gerard Rimesso </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Albert Einstein College of Medicine, Bronx, New York 10461</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gerard%2BRimesso%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rimesso%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGerard%2BRimesso%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David M. Reynolds </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Albert Einstein College of Medicine, Bronx, New York 10461</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BM.%2BReynolds%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reynolds%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BM.%2BReynolds%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jinlu Cai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Albert Einstein College of Medicine, Bronx, New York 10461</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jinlu%2BCai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cai%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJinlu%2BCai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Nicholas E. Baker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Albert Einstein College of Medicine, Bronx, New York 10461</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholas%2BE.%2BBaker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baker%20NE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicholas%2BE.%2BBaker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nicholas.baker@einstein.yu.edu" class="" data-icon-position="" data-hide-link-title="0">nicholas.baker@einstein.yu.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/10/3207" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/10/3207&amp;title=Whole-Genome%20Sequencing%20and%20iPLEX%20MassARRAY%20Genotyping%20Map%20an%20EMS-Induced%20Mutation%20Affecting%20Cell%20Competition%20in%20Drosophila%20melanogaster" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/10/3207&amp;title=Whole-Genome%20Sequencing%20and%20iPLEX%20MassARRAY%20Genotyping%20Map%20an%20EMS-Induced%20Mutation%20Affecting%20Cell%20Competition%20in%20Drosophila%20melanogaster" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/10/3207&amp;title=Whole-Genome%20Sequencing%20and%20iPLEX%20MassARRAY%20Genotyping%20Map%20an%20EMS-Induced%20Mutation%20Affecting%20Cell%20Competition%20in%20Drosophila%20melanogaster" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/10/3207&amp;text=Whole-Genome%20Sequencing%20and%20iPLEX%20MassARRAY%20Genotyping%20Map%20an%20EMS-Induced%20Mutation%20Affecting%20Cell%20Competition%20in%20Drosophila%20melanogaster" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/10/3207&amp;title=Whole-Genome%20Sequencing%20and%20iPLEX%20MassARRAY%20Genotyping%20Map%20an%20EMS-Induced%20Mutation%20Affecting%20Cell%20Competition%20in%20Drosophila%20melanogaster" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/10/3207&amp;t=Whole-Genome%20Sequencing%20and%20iPLEX%20MassARRAY%20Genotyping%20Map%20an%20EMS-Induced%20Mutation%20Affecting%20Cell%20Competition%20in%20Drosophila%20melanogaster" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/10/3207&amp;title=Whole-Genome%20Sequencing%20and%20iPLEX%20MassARRAY%20Genotyping%20Map%20an%20EMS-Induced%20Mutation%20Affecting%20Cell%20Competition%20in%20Drosophila%20melanogaster" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/10/3207&amp;title=Whole-Genome%20Sequencing%20and%20iPLEX%20MassARRAY%20Genotyping%20Map%20an%20EMS-Induced%20Mutation%20Affecting%20Cell%20Competition%20in%20Drosophila%20melanogaster" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460342/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460342" id="node460342--3" data-pisa="ggg;6/10/3207" data-pisa-master="ggg;g3.116.029421" data-apath="/ggg/6/10/3207.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Whole-Genome Sequencing and iPLEX MassARRAY Genotyping Map an EMS-Induced Mutation Affecting Cell Competition in <em>Drosophila melanogaster</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first" data-delta="0">Chang-Hyun Lee</span>, <span class="highwire-citation-author" data-delta="1">Gerard Rimesso</span>, <span class="highwire-citation-author" data-delta="2">David M. Reynolds</span>, <span class="highwire-citation-author" data-delta="3">Jinlu Cai</span> and <span class="highwire-citation-author" data-delta="4">Nicholas E. Baker</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3207-3217;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.029421 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460342--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Chang-Hyun Lee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Albert Einstein College of Medicine, Bronx, New York 10461</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Chang-Hyun%2BLee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lee%20C&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AChang-Hyun%2BLee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Gerard Rimesso </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Albert Einstein College of Medicine, Bronx, New York 10461</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Gerard%2BRimesso%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Rimesso%20G&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AGerard%2BRimesso%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">David M. Reynolds </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Albert Einstein College of Medicine, Bronx, New York 10461</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=David%2BM.%2BReynolds%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Reynolds%20DM&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADavid%2BM.%2BReynolds%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Jinlu Cai </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Albert Einstein College of Medicine, Bronx, New York 10461</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jinlu%2BCai%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Cai%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJinlu%2BCai%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Nicholas E. Baker </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'>Department of Genetics, Albert Einstein College of Medicine, Bronx, New York 10461</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Nicholas%2BE.%2BBaker%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Baker%20NE&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANicholas%2BE.%2BBaker%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nicholas.baker@einstein.yu.edu" class="" data-icon-position="" data-hide-link-title="0">nicholas.baker@einstein.yu.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3207&amp;rft.epage=3217&amp;rft.atitle=Whole-Genome%20Sequencing%20and%20iPLEX%20MassARRAY%20Genotyping%20Map%20an%20EMS-Induced%20Mutation%20Affecting%20Cell%20Competition%20in%20Drosophila%20melanogaster&amp;rft.volume=6&amp;rft.issue=10&amp;rft.date=2016-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Lee%2C+Chang-Hyun&amp;rft.au=Rimesso%2C+Gerard&amp;rft.au=Reynolds%2C+David+M.&amp;rft.au=Cai%2C+Jinlu&amp;rft.au=Baker%2C+Nicholas+E."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460342/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460342/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460342/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460342/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460342/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460342/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460342/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460342/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460342/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460342/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460342/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460342/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/10/3207" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-10" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/10/3207.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/10/3207.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/10/3207.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/10/3207.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
