<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/10/3035" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/10/3035.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/10/3035.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/10/3035.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/10/3035/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/10/3035.atom" />
<meta name="HW.pisa" content="ggg;6/10/3035" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="Genome-Wide Analysis of Polyadenylation Events in Schmidtea mediterranea" />
<meta name="DC.Identifier" content="10.1534/g3.116.031120" />
<meta name="DC.Date" content="2016-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Lakshmanan et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="In eukaryotes, 3′ untranslated regions (UTRs) play important roles in regulating posttranscriptional gene expression. The 3′UTR is defined by regulated cleavage/polyadenylation of the pre-mRNA. The advent of next-generation sequencing technology has now enabled us to identify these events on a genome-wide scale. In this study, we used poly(A)-position profiling by sequencing (3P-Seq) to capture all poly(A) sites across the genome of the freshwater planarian, Schmidtea mediterranea , an ideal model system for exploring the process of regeneration and stem cell function. We identified the 3′UTRs for ∼14,000 transcripts and thus improved the existing gene annotations. We found 97 transcripts, which are polyadenylated within an internal exon, resulting in the shrinking of the ORF and loss of a predicted protein domain. Around 40% of the transcripts in planaria were alternatively polyadenylated (ApA), resulting either in an altered 3′UTR or a change in coding sequence. We identified specific ApA transcript isoforms that were subjected to miRNA mediated gene regulation using degradome sequencing. In this study, we also confirmed a tissue-specific expression pattern for alternate polyadenylated transcripts. The insights from this study highlight the potential role of ApA in regulating the gene expression essential for planarian regeneration." />
<meta name="DC.Contributor" content="Vairavan Lakshmanan" />
<meta name="DC.Contributor" content="Dhiru Bansal" />
<meta name="DC.Contributor" content="Jahnavi Kulkarni" />
<meta name="DC.Contributor" content="Deepak Poduval" />
<meta name="DC.Contributor" content="Srikar Krishna" />
<meta name="DC.Contributor" content="Vidyanand Sasidharan" />
<meta name="DC.Contributor" content="Praveen Anand" />
<meta name="DC.Contributor" content="Aswin Seshasayee" />
<meta name="DC.Contributor" content="Dasaradhi Palakodeti" />
<meta name="article:published_time" content="2016-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="Genome-Wide Analysis of Polyadenylation Events in Schmidtea mediterranea" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;In eukaryotes, 3′ untranslated regions (UTRs) play important roles in regulating posttranscriptional gene expression. The 3′UTR is defined by regulated cleavage/polyadenylation of the pre-mRNA. The advent of next-generation sequencing technology has now enabled us to identify these events on a genome-wide scale. In this study, we used poly(A)-position profiling by sequencing (3P-Seq) to capture all poly(A) sites across the genome of the freshwater planarian, &lt;i&gt;Schmidtea mediterranea&lt;/i&gt;, an ideal model system for exploring the process of regeneration and stem cell function. We identified the 3′UTRs for ∼14,000 transcripts and thus improved the existing gene annotations. We found 97 transcripts, which are polyadenylated within an internal exon, resulting in the shrinking of the ORF and loss of a predicted protein domain. Around 40% of the transcripts in planaria were alternatively polyadenylated (ApA), resulting either in an altered 3′UTR or a change in coding sequence. We identified specific ApA transcript isoforms that were subjected to miRNA mediated gene regulation using degradome sequencing. In this study, we also confirmed a tissue-specific expression pattern for alternate polyadenylated transcripts. The insights from this study highlight the potential role of ApA in regulating the gene expression essential for planarian regeneration.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/10/01" />
<meta name="citation_mjid" content="ggg;6/10/3035" />
<meta name="citation_id" content="6/10/3035" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/10/3035" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/10/3035.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/10/3035.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/10/3035.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.031120" />
<meta name="citation_pmid" content="27489207" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="14" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3035" />
<meta name="citation_lastpage" content="3048" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Vairavan Lakshmanan" />
<meta name="citation_author_institution" content="Institute for Stem Cell Biology and Regenerative Medicine, NCBS campus, GKVK, Bellary Road, Bangalore 560065, Karnataka, India" />
<meta name="citation_author_institution" content="SASTRA University, Thirumalaisamudram, Thanjavur 613401, Tamil Nadu, India" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-9468-9571" />
<meta name="citation_author" content="Dhiru Bansal" />
<meta name="citation_author_institution" content="Institute for Stem Cell Biology and Regenerative Medicine, NCBS campus, GKVK, Bellary Road, Bangalore 560065, Karnataka, India" />
<meta name="citation_author_institution" content="Manipal University, Madhav Nagar, Near Tiger Circle, Manipal 576104, Karnataka, India" />
<meta name="citation_author" content="Jahnavi Kulkarni" />
<meta name="citation_author_institution" content="Institute for Stem Cell Biology and Regenerative Medicine, NCBS campus, GKVK, Bellary Road, Bangalore 560065, Karnataka, India" />
<meta name="citation_author" content="Deepak Poduval" />
<meta name="citation_author_institution" content="University of Bergen, Department of Clinical Science, Bergen 5020, Norway" />
<meta name="citation_author" content="Srikar Krishna" />
<meta name="citation_author_institution" content="Institute for Stem Cell Biology and Regenerative Medicine, NCBS campus, GKVK, Bellary Road, Bangalore 560065, Karnataka, India" />
<meta name="citation_author_institution" content="SASTRA University, Thirumalaisamudram, Thanjavur 613401, Tamil Nadu, India" />
<meta name="citation_author" content="Vidyanand Sasidharan" />
<meta name="citation_author_institution" content="Institute for Stem Cell Biology and Regenerative Medicine, NCBS campus, GKVK, Bellary Road, Bangalore 560065, Karnataka, India" />
<meta name="citation_author_institution" content="Manipal University, Madhav Nagar, Near Tiger Circle, Manipal 576104, Karnataka, India" />
<meta name="citation_author" content="Praveen Anand" />
<meta name="citation_author_institution" content="Institute for Stem Cell Biology and Regenerative Medicine, NCBS campus, GKVK, Bellary Road, Bangalore 560065, Karnataka, India" />
<meta name="citation_author_email" content="dasaradhip@instem.res.in" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2219-4785" />
<meta name="citation_author" content="Aswin Seshasayee" />
<meta name="citation_author_institution" content="National Centre for Biological Sciences, GKVK Campus, Bangalore 560065, Karnataka, India" />
<meta name="citation_author_email" content="dasaradhip@instem.res.in" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0001-7028-1803" />
<meta name="citation_author" content="Dasaradhi Palakodeti" />
<meta name="citation_author_institution" content="Institute for Stem Cell Biology and Regenerative Medicine, NCBS campus, GKVK, Bellary Road, Bangalore 560065, Karnataka, India" />
<meta name="citation_author_email" content="dasaradhip@instem.res.in" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-2219-4785" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=C. Addo-Quaye;citation_author=W. Miller;citation_author=M. J. Axtell;citation_title=Cleaveland: a pipeline for using degradome data to find cleaved small RNA targets.;citation_volume=25;citation_year=2009;citation_pmid=19017659;citation_doi=10.1093/bioinformatics/btn604" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=F. W. Alt;citation_author=A. L. Bothwell;citation_author=M. Knapp;citation_author=E. Siden;citation_author=E. Mather;citation_title=Synthesis of secreted and membrane-bound immunoglobulin mu heavy chains is directed by mRNAs that differ at their 3' ends.;citation_pages=293-301;citation_volume=20;citation_year=1980;citation_issue=2;citation_pmid=6771018;citation_doi=10.1016/0092-8674(80)90615-7" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=S. F. Altschul;citation_author=W. Gish;citation_author=W. Miller;citation_author=E. W. Myers;citation_author=D. J. Lipman;citation_title=Basic local alignment search tool.;citation_pages=403-410;citation_volume=215;citation_year=1990;citation_issue=3;citation_pmid=2231712;citation_doi=10.1006/jmbi.1990.9999" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=T. L. Bailey;citation_author=N. Williams;citation_author=C. Misleh;citation_author=W. W. Li;citation_title=MEME: discovering and analyzing DNA and protein sequence motifs.;citation_pages=W369373;citation_volume=34;citation_year=2006" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=S. M. Barabino;citation_author=W. Keller;citation_title=Last but not least: regulated poly(A) tail formation.;citation_pages=9-11;citation_volume=99;citation_year=1999;citation_issue=1;citation_pmid=10520989;citation_doi=10.1016/S0092-8674(00)80057-4" />
<meta name="citation_reference" content="citation_journal_title=Cellular and molecular life sciences :  CMLS;citation_journal_abbrev=Cell Mol Life Sci;citation_author=L. W. Barrett;citation_author=S. Fletcher;citation_author=S. D. Wilton;citation_title=Regulation of eukaryotic gene expression by the untranslated gene regions and other non-coding elements.;citation_pages=3613-3634;citation_volume=69;citation_year=2012;citation_issue=21;citation_pmid=22538991;citation_doi=10.1007/s00018-012-0990-9" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=B. D. Berkovits;citation_author=C. Mayr;citation_title=Alternative 3′ UTRs act as scaffolds to regulate membrane protein localization.;citation_pages=363-367;citation_volume=522;citation_year=2015;citation_pmid=25896326;citation_doi=10.1038/nature14321" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=M. J. Blythe;citation_author=D. Kao;citation_author=S. Malla;citation_author=J. Rowsell;citation_author=R. Wilson;citation_title=A dual platform approach to transcript discovery for the planarian Schmidtea mediterranea to establish RNAseq for stem cell and regeneration biology.;citation_pages=e15617-e15617;citation_volume=5;citation_year=2010;citation_issue=12;citation_pmid=21179477;citation_doi=10.1371/journal.pone.0015617" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_title=PlanMine - a mineable resource of planarian biology and biodiversity.;citation_author=H. Brandl;citation_author=H. Moon;citation_author=M. Vila-Farre;citation_author=S. Y. Liu;citation_author=I. Henry;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Journal of molecular biology;citation_journal_abbrev=J Mol Biol;citation_author=C. Burge;citation_author=S. Karlin;citation_title=Prediction of complete gene structures in human genomic DNA.;citation_pages=78-94;citation_volume=268;citation_year=1997;citation_issue=1;citation_pmid=9149143;citation_doi=10.1006/jmbi.1997.0951" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=B. L. Cantarel;citation_author=I. Korf;citation_author=S. M. Robb;citation_author=G. Parra;citation_author=E. Ross;citation_title=MAKER: an easy-to-use annotation pipeline designed for emerging model organism genomes.;citation_volume=18;citation_year=2008;citation_pmid=18025269;citation_doi=10.1101/gr.6743907" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=A. A. Cass;citation_author=J. H. Bahn;citation_author=J.-H. Lee;citation_author=C. Greer;citation_author=X. Lin;citation_title=Global analyses of endonucleolytic cleavage in mammals reveal expanded repertoires of cleavage-inducing small rnas and their targets.;citation_volume=44;citation_year=2016;citation_pmid=26975654;citation_doi=10.1093/nar/gkw164" />
<meta name="citation_reference" content="citation_journal_title=Genes &amp; Development;citation_journal_abbrev=Genes Dev.;citation_author=D. F. Colgan;citation_author=J. L. Manley;citation_title=Mechanism and regulation of mRNA polyadenylation;citation_pages=2755-2766;citation_volume=11;citation_year=1997;citation_issue=21;citation_pmid=9353246;citation_doi=10.1101/gad.11.21.2755" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=M. Davila Lopez;citation_author=T. Samuelsson;citation_title=Early evolution of histone mRNA 3′ end processing.;citation_volume=14;citation_year=2008;citation_pmid=17998288;citation_doi=10.1261/rna.782308" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=A. Derti;citation_author=P. Garrett-Engele;citation_author=K. D. Macisaac;citation_author=R. C. Stevens;citation_author=S. Sriram;citation_title=A quantitative atlas of polyadenylation in five mammals.;citation_volume=22;citation_year=2012;citation_pmid=22454233;citation_doi=10.1101/gr.132563.111" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=P. Early;citation_author=J. Rogers;citation_author=M. Davis;citation_author=K. Calame;citation_author=M. Bond;citation_title=Two mRNAs can be produced from a single immunoglobulin mu gene by alternative RNA processing pathways.;citation_pages=313-319;citation_volume=20;citation_year=1980;citation_issue=2;citation_pmid=6771020;citation_doi=10.1016/0092-8674(80)90617-0" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=E. Elkayam;citation_author=C.-D. Kuhn;citation_author=A. Tocilj;citation_author=A. D. Haase;citation_author=E. M. Greene;citation_title=The structure of human argonaute-2 in complex with miR-20a.;citation_pages=100-110;citation_volume=150;citation_year=2012;citation_issue=1;citation_pmid=22682761;citation_doi=10.1016/j.cell.2012.05.017" />
<meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.;citation_author=R. Elkon;citation_author=A. P. Ugalde;citation_author=R. Agami;citation_title=Alternative cleavage and polyadenylation: extent, regulation and function.;citation_pages=496-506;citation_volume=14;citation_year=2013;citation_pmid=23774734;citation_doi=10.1038/nrg3482" />
<meta name="citation_reference" content="citation_journal_title=Annual review of neuroscience;citation_journal_abbrev=Annu Rev Neurosci;citation_author=S. W. Flavell;citation_author=M. E. Greenberg;citation_title=Signaling mechanisms linking neuronal activity to gene expression and plasticity of the nervous system.;citation_pages=563-590;citation_volume=31;citation_year=2008;citation_pmid=18558867;citation_doi=10.1146/annurev.neuro.31.060407.125631" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=Y. Fu;citation_author=Y. Sun;citation_author=Y. Li;citation_author=J. Li;citation_author=X. Rao;citation_title=Differential genome-wide profiling of tandem 3′ UTRs among human breast cancer and normal cells by high-throughput sequencing.;citation_volume=21;citation_year=2011;citation_pmid=21474764;citation_doi=10.1101/gr.115295.110" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=M. A. German;citation_author=M. Pillay;citation_author=D. H. Jeong;citation_author=A. Hetawal;citation_author=S. Luo;citation_title=Global identification of microRNA-target RNA pairs by parallel analysis of RNA ends.;citation_pages=941-946;citation_volume=26;citation_year=2008;citation_issue=8;citation_pmid=18542052;citation_doi=10.1038/nbt1417" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. Griffiths-Jones;citation_author=H. K. Saini;citation_author=S. van Dongen;citation_author=A. J. Enright;citation_title=miRBase: tools for microRNA genomics.;citation_volume=36;citation_year=2008;citation_pmid=17991681;citation_doi=10.1093/nar/gkm952" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=V. Hilgers;citation_author=M. W. Perry;citation_author=D. Hendrix;citation_author=A. Stark;citation_author=M. Levine;citation_title=Neural-specific elongation of 3′ UTRs during Drosophila development.;citation_volume=108;citation_year=2011;citation_pmid=21896737;citation_doi=10.1073/pnas.1112672108" />
<meta name="citation_reference" content="citation_journal_title=Epigenetics;citation_author=A. Hubert;citation_author=J. M. Henderson;citation_author=K. G. Ross;citation_author=M. W. Cowles;citation_author=J. Torres;citation_title=Epigenetic regulation of planarian stem cells by the SET1/MLL family of histone methyltransferases.;citation_pages=79-91;citation_volume=8;citation_year=2013;citation_pmid=23235145;citation_doi=10.4161/epi.23211" />
<meta name="citation_reference" content="citation_journal_title=BMC bioinformatics [electronic resource];citation_journal_abbrev=BMC Bioinformatics;citation_author=J. Huerta-Cepas;citation_author=J. Dopazo;citation_author=T. Gabaldon;citation_title=ETE: a python Environment for Tree Exploration.;citation_pages=24-24;citation_volume=11;citation_year=2010;citation_pmid=20070885;citation_doi=10.1186/1471-2105-11-24" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. H. Jan;citation_author=R. C. Friedman;citation_author=J. G. Ruby;citation_author=D. P. Bartel;citation_title=Formation, regulation and evolution of Caenorhabditis elegans 3'UTRs.;citation_pages=97-101;citation_volume=469;citation_year=2011;citation_issue=7328;citation_pmid=21085120;citation_doi=10.1038/nature09616" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=Z. Ji;citation_author=B. Tian;citation_title=Reprogramming of 3' untranslated regions of mRNAs by alternative polyadenylation in generation of pluripotent stem cells from different cell types.;citation_pages=e8419-e8419;citation_volume=4;citation_year=2009;citation_issue=12;citation_pmid=20037631;citation_doi=10.1371/journal.pone.0008419" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Z. Ji;citation_author=J. Y. Lee;citation_author=Z. Pan;citation_author=B. Jiang;citation_author=B. Tian;citation_title=Progressive lengthening of 3′ untranslated regions of mRNAs by alternative polyadenylation during mouse embryonic development.;citation_volume=106;citation_year=2009;citation_pmid=19372383;citation_doi=10.1073/pnas.0900028106" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Genet. Dev.;citation_author=J. Jia;citation_author=P. Yao;citation_author=A. Arif;citation_author=P. L. Fox;citation_title=Regulation and dysregulation of 3′UTR-mediated translational control.;citation_pages=29-34;citation_volume=23;citation_year=2013;citation_pmid=23312843;citation_doi=10.1016/j.gde.2012.12.004" />
<meta name="citation_reference" content="citation_journal_title=PLoS biology;citation_journal_abbrev=PLoS Biol;citation_author=B. John;citation_author=A. J. Enright;citation_author=A. Aravin;citation_author=T. Tuschl;citation_author=C. Sander;citation_title=Human MicroRNA targets.;citation_pages=e363-e363;citation_volume=2;citation_year=2004;citation_issue=11;citation_pmid=15502875;citation_doi=10.1371/journal.pbio.0020363" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=F. V. Karginov;citation_author=S. Cheloufi;citation_author=M. M. W. Chong;citation_author=A. Stark;citation_author=A. D. Smith;citation_title=Diverse endonucleolytic cleavage sites in the mammalian transcriptome depend upon microRNAs, Drosha, and additional nucleases.;citation_pages=781-788;citation_volume=38;citation_year=2010;citation_issue=6;citation_pmid=20620951;citation_doi=10.1016/j.molcel.2010.06.001" />
<meta name="citation_reference" content="citation_journal_title=The EMBO Journal;citation_journal_abbrev=EMBO J.;citation_author=I. Kaufmann;citation_author=G. Martin;citation_author=A. Friedlein;citation_author=H. Langen;citation_author=W. Keller;citation_title=Human Fip1 is a subunit of CPSF that binds to U-rich RNA elements and stimulates poly(A) polymerase;citation_pages=616-626;citation_volume=23;citation_year=2004;citation_issue=3;citation_pmid=14749727;citation_doi=10.1038/sj.emboj.7600070" />
<meta name="citation_reference" content="citation_journal_title=Stem Cells;citation_journal_abbrev=Stem Cells;citation_author=R. M. Labbé;citation_author=M. Irimia;citation_author=K. W. Currie;citation_author=A. Lin;citation_author=S. J. Zhu;citation_title=A comparative transcriptomic analysis reveals conserved features of stem cell pluripotency in planarians and mammals.;citation_pages=1734-1745;citation_volume=30;citation_year=2012;citation_issue=8;citation_pmid=22696458;citation_doi=10.1002/stem.1144" />
<meta name="citation_reference" content="citation_journal_title=Genome biology;citation_journal_abbrev=Genome Biol;citation_author=B. Langmead;citation_author=C. Trapnell;citation_author=M. Pop;citation_author=S. L. Salzberg;citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.;citation_pages=R25-R25;citation_volume=10;citation_year=2009;citation_issue=3;citation_pmid=19261174;citation_doi=10.1186/gb-2009-10-3-r25" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=W. Li;citation_author=A. Godzik;citation_title=Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences.;citation_volume=22;citation_year=2006;citation_pmid=16731699;citation_doi=10.1093/bioinformatics/btl158" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_author=S. Y. Liu;citation_author=C. Selck;citation_author=B. Friedrich;citation_author=R. Lutz;citation_author=M. Vila-Farre;citation_title=Reactivating head regrowth in a regeneration-deficient planarian species.;citation_pages=81-84;citation_volume=500;citation_year=2013;citation_pmid=23883932;citation_doi=10.1038/nature12414" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=C. R. Mandel;citation_author=S. Kaneko;citation_author=H. Zhang;citation_author=D. Gebauer;citation_author=V. Vethantham;citation_title=Polyadenylation factor CPSF-73 is the pre-mRNA 3'-end-processing endonuclease.;citation_pages=953-956;citation_volume=444;citation_year=2006;citation_issue=7121;citation_pmid=17128255;citation_doi=10.1038/nature05363" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_author=M. Mangone;citation_author=A. P. Manoharan;citation_author=D. Thierry-Mieg;citation_author=J. Thierry-Mieg;citation_author=T. Han;citation_title=The landscape of C. elegans 3′UTRs.;citation_volume=329;citation_year=2010;citation_pmid=20522740;citation_doi=10.1126/science.1191244" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=C. Mayr;citation_author=D. P. Bartel;citation_title=Widespread shortening of 3'UTRs by alternative cleavage and polyadenylation activates oncogenes in cancer cells.;citation_pages=673-684;citation_volume=138;citation_year=2009;citation_issue=4;citation_pmid=19703394;citation_doi=10.1016/j.cell.2009.06.016" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=S. Millevoi;citation_author=S. Vagner;citation_title=Molecular mechanisms of eukaryotic pre-mRNA 3′ end processing regulation.;citation_volume=38;citation_year=2010;citation_pmid=20044349;citation_doi=10.1093/nar/gkp1176" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=Y. Moran;citation_author=D. Fredman;citation_author=D. Praher;citation_author=X. Z. Li;citation_author=L. M. Wee;citation_title=Cnidarian microRNAs frequently regulate targets by cleavage.;citation_volume=24;citation_year=2014;citation_pmid=24642861;citation_doi=10.1101/gr.162503.113" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=C. G. Noble;citation_author=B. Beuth;citation_author=I. A. Taylor;citation_title=Structure of a nucleotide-bound Clp1-Pcf11 polyadenylation factor.;citation_volume=35;citation_year=2007;citation_pmid=17151076;citation_doi=10.1093/nar/gkl1010" />
<meta name="citation_reference" content="citation_journal_title=EMBO J.;citation_author=P. Onal;citation_author=D. Grun;citation_author=C. Adamidi;citation_author=A. Rybak;citation_author=J. Solana;citation_title=Gene expression of pluripotency determinants is conserved between mammalian and planarian stem cells.;citation_volume=31;citation_year=2012;citation_pmid=22543868;citation_doi=10.1038/emboj.2012.110" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=B. J. Pearson;citation_author=A. Sanchez Alvarado;citation_title=A planarian p53 homolog regulates proliferation and self-renewal in adult stem cell lineages;citation_pages=213-221;citation_volume=137;citation_year=2010;citation_issue=2;citation_pmid=20040488;citation_doi=10.1242/dev.044297" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=A. M. Resch;citation_author=D. Palakodeti;citation_author=Y. C. Lu;citation_author=M. Horowitz;citation_author=B. R. Graveley;citation_title=Transcriptome analysis reveals strain-specific and conserved stemness genes in Schmidtea mediterranea.;citation_pages=e34447-e34447;citation_volume=7;citation_year=2012;citation_issue=4;citation_pmid=22496805;citation_doi=10.1371/journal.pone.0034447" />
<meta name="citation_reference" content="citation_journal_title=Genesis;citation_author=S. M. Robb;citation_author=K. Gotting;citation_author=E. Ross;citation_author=A. Sanchez Alvarado;citation_title=SmedGD 2.0: The Schmidtea mediterranea genome database.;citation_pages=535-546;citation_volume=53;citation_year=2015;citation_pmid=26138588;citation_doi=10.1002/dvg.22872" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=J. Rogers;citation_author=P. Early;citation_author=C. Carter;citation_author=K. Calame;citation_author=M. Bond;citation_title=Two mRNAs with different 3' ends encode membrane-bound and secreted forms of immunoglobulin mu chain.;citation_pages=303-312;citation_volume=20;citation_year=1980;citation_issue=2;citation_pmid=6771019;citation_doi=10.1016/0092-8674(80)90616-9" />
<meta name="citation_reference" content="citation_journal_title=Cellular and molecular life sciences :  CMLS;citation_journal_abbrev=Cell Mol Life Sci;citation_author=L. Rossi;citation_author=A. Salvetti;citation_author=R. Batistoni;citation_author=P. Deri;citation_author=V. Gremigni;citation_title=Planarians, a tale of stem cells.;citation_pages=16-23;citation_volume=65;citation_year=2008;citation_issue=1;citation_pmid=18030424;citation_doi=10.1007/s00018-007-7426-y" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. M. Rothberg;citation_author=W. Hinz;citation_author=T. M. Rearick;citation_author=J. Schultz;citation_author=W. Mileski;citation_title=An integrated semiconductor device enabling non-optical genome sequencing.;citation_pages=348-352;citation_volume=475;citation_year=2011;citation_issue=7356;citation_pmid=21776081;citation_doi=10.1038/nature10242" />
<meta name="citation_reference" content="citation_author=L. Rouhana;citation_author=N. Shibata;citation_author=O. Nishimura;citation_author=K. Agata;citation_year=2010" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_author=L. Rouhana;citation_author=J. A. Weiss;citation_author=R. S. King;citation_author=P. A. Newmark;citation_title=PIWI homologs mediate histone H4 mRNA localization to planarian chromatoid bodies.;citation_volume=141;citation_year=2014;citation_pmid=24903754;citation_doi=10.1242/dev.101618" />
<meta name="citation_reference" content="citation_journal_title=Current opinion in genetics &amp; development;citation_journal_abbrev=Curr Opin Genet Dev;citation_author=A. Sanchez Alvarado;citation_title=The freshwater planarian Schmidtea mediterranea: embryogenesis, stem cells and regeneration.;citation_pages=438-444;citation_volume=13;citation_year=2003;citation_issue=4;citation_pmid=12888018;citation_doi=10.1016/S0959-437X(03)00082-0" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=R. Sandberg;citation_author=J. R. Neilson;citation_author=A. Sarma;citation_author=P. A. Sharp;citation_author=C. B. Burge;citation_title=Proliferating Cells Express mRNAs with Shortened 3' Untranslated Regions and Fewer MicroRNA Target Sites;citation_pages=1643-1647;citation_volume=320;citation_year=2008;citation_issue=5883;citation_pmid=18566288;citation_doi=10.1126/science.1155390" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=V. Sasidharan;citation_author=Y. C. Lu;citation_author=D. Bansal;citation_author=P. Dasari;citation_author=D. Poduval;citation_title=Identification of neoblast- and regeneration-specific miRNAs in the planarian Schmidtea mediterranea.;citation_volume=19;citation_year=2013;citation_pmid=23974438;citation_doi=10.1261/rna.038653.113" />
<meta name="citation_reference" content="citation_journal_title=Genes Dev.;citation_author=L. Schonemann;citation_author=U. Kuhn;citation_author=G. Martin;citation_author=P. Schafer;citation_author=A. R. Gruber;citation_title=Reconstitution of CPSF active in polyadenylation: recognition of the polyadenylation signal by WDR33.;citation_volume=28;citation_year=2014;citation_pmid=25301781;citation_doi=10.1101/gad.250985.114" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=P. J. Shepard;citation_author=E. A. Choi;citation_author=J. Lu;citation_author=L. A. Flanagan;citation_author=K. J. Hertel;citation_title=Complex and dynamic landscape of RNA polyadenylation revealed by PAS-Seq.;citation_volume=17;citation_year=2011;citation_pmid=21343387;citation_doi=10.1261/rna.2581711" />
<meta name="citation_reference" content="citation_journal_title=RNA;citation_author=Y. Shi;citation_title=Alternative polyadenylation: new insights from global analyses.;citation_volume=18;citation_year=2012;citation_pmid=23097429;citation_doi=10.1261/rna.035899.112" />
<meta name="citation_reference" content="citation_journal_title=Molecular cell;citation_journal_abbrev=Mol Cell;citation_author=C. Shin;citation_author=J.-W. Nam;citation_author=K. K.-H. Farh;citation_author=H. R. Chiang;citation_author=A. Shkumatava;citation_title=Expanding the microRNA targeting code: functional sites with centered pairing.;citation_pages=789-802;citation_volume=38;citation_year=2010;citation_issue=6;citation_pmid=20620952;citation_doi=10.1016/j.molcel.2010.06.005" />
<meta name="citation_reference" content="citation_journal_title=Molecular Systems Biology;citation_journal_abbrev=Mol Syst Biol;citation_author=F. Sievers;citation_author=A. Wilm;citation_author=D. Dineen;citation_author=T. J. Gibson;citation_author=K. Karplus;citation_title=Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega;citation_pages=539-539;citation_volume=7;citation_year=2011;citation_issue=1;citation_pmid=21988835;citation_doi=10.1038/msb.2011.75" />
<meta name="citation_reference" content="citation_journal_title=Cancer Res.;citation_author=P. Singh;citation_author=T. L. Alley;citation_author=S. M. Wright;citation_author=S. Kamdar;citation_author=W. Schott;citation_title=Global changes in processing of mRNA 3′ untranslated regions characterize clinically distinct cancer subtypes.;citation_volume=69;citation_year=2009;citation_pmid=19934316;citation_doi=10.1158/0008-5472.CAN-09-2236" />
<meta name="citation_reference" content="citation_journal_title=Cell Reports;citation_author=P. Smibert;citation_author=P. Miura;citation_author=J. O. Westholm;citation_author=S. Shenker;citation_author=G. May;citation_title=Global patterns of tissue-specific alternative polyadenylation in Drosophila.;citation_pages=277-289;citation_volume=1;citation_year=2012;citation_pmid=22685694;citation_doi=10.1016/j.celrep.2012.01.001" />
<meta name="citation_reference" content="citation_journal_title=PLoS Genet.;citation_author=J. Solana;citation_author=C. Gamberi;citation_author=Y. Mihaylova;citation_author=S. Grosswendt;citation_author=C. Chen;citation_title=The CCR4-NOT complex mediates deadenylation and degradation of stem cell mRNAs and promotes planarian stem cell differentiation.;citation_pages=e1004003;citation_volume=9;citation_year=2013;citation_pmid=24367277;citation_doi=10.1371/journal.pgen.1004003" />
<meta name="citation_reference" content="citation_journal_title=J. Mol. Cell Biol.;citation_author=Y. Sun;citation_author=Y. Fu;citation_author=Y. Li;citation_author=A. Xu;citation_title=Genome-wide alternative polyadenylation in animals: insights from high-throughput technologies.;citation_volume=4;citation_year=2012;citation_pmid=23099521;citation_doi=10.1093/jmcb/mjs041" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Research;citation_journal_abbrev=Nucleic Acids Res;citation_author=B. Tian;citation_author=J. Hu;citation_author=H. Zhang;citation_author=C. S. Lutz;citation_title=A large-scale analysis of mRNA polyadenylation of human and mouse genes;citation_pages=201-212;citation_volume=33;citation_year=2005;citation_issue=1;citation_pmid=15647503;citation_doi=10.1093/nar/gki158" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=I. Ulitsky;citation_author=A. Shkumatava;citation_author=C. H. Jan;citation_author=H. Sive;citation_author=D. P. Bartel;citation_title=Conserved function of lincRNAs in vertebrate embryonic development despite rapid sequence evolution.;citation_pages=1537-1550;citation_volume=147;citation_year=2011;citation_issue=7;citation_pmid=22196729;citation_doi=10.1016/j.cell.2011.11.055" />
<meta name="citation_reference" content="citation_journal_title=Brief. Bioinform.;citation_author=O. Westesson;citation_author=M. Skinner;citation_author=I. Holmes;citation_title=Visualizing next-generation sequencing data with JBrowse.;citation_volume=14;citation_year=2013;citation_pmid=22411711;citation_doi=10.1093/bib/bbr078" />
<meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.;citation_author=L. You;citation_author=J. Wu;citation_author=Y. Feng;citation_author=Y. Fu;citation_author=Y. Guo;citation_title=Apasdb: a database describing alternative poly(a) sites and selection of heterogeneous cleavage sites downstream of poly(a) signals.;citation_volume=43;citation_year=2015;citation_pmid=25378337;citation_doi=10.1093/nar/gku1076" />
<meta name="citation_reference" content="citation_journal_title=J. Cell Biol.;citation_author=A. Zeng;citation_author=Y. Q. Li;citation_author=C. Wang;citation_author=X. S. Han;citation_author=G. Li;citation_title=Heterochromatin protein 1 promotes self-renewal and triggers regenerative proliferation in adult stem cells.;citation_volume=201;citation_year=2013;citation_pmid=23629965;citation_doi=10.1083/jcb.201207172" />
<meta name="citation_reference" content="citation_journal_title=Microbiology and Molecular Biology Reviews;citation_journal_abbrev=Microbiol. Mol. Biol. Rev.;citation_author=J. Zhao;citation_author=L. Hyman;citation_author=C. Moore;citation_title=Formation of mRNA 3' Ends in Eukaryotes: Mechanism, Regulation, and Interrelationships with Other Steps in mRNA Synthesis;citation_pages=405-445;citation_volume=63;citation_year=1999;citation_issue=2;citation_pmid=10357856" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="Genome-Wide Analysis of Polyadenylation Events in Schmidtea mediterranea" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10.cover-source.jpg" />
<meta name="twitter:description" content="In eukaryotes, 3′ untranslated regions (UTRs) play important roles in regulating posttranscriptional gene expression. The 3′UTR is defined by regulated cleavage/polyadenylation of the pre-mRNA. The advent of next-generation sequencing technology has now enabled us to identify these events on a genome-wide scale. In this study, we used poly(A)-position profiling by sequencing (3P-Seq) to capture all poly(A) sites across the genome of the freshwater planarian, Schmidtea mediterranea , an ideal model system for exploring the process of regeneration and stem cell function. We identified the 3′UTRs for ∼14,000 transcripts and thus improved the existing gene annotations. We found 97 transcripts, which are polyadenylated within an internal exon, resulting in the shrinking of the ORF and loss of a predicted protein domain. Around 40% of the transcripts in planaria were alternatively polyadenylated (ApA), resulting either in an altered 3′UTR or a change in coding sequence. We identified specific ApA transcript isoforms that were subjected to miRNA mediated gene regulation using degradome sequencing. In this study, we also confirmed a tissue-specific expression pattern for alternate polyadenylated transcripts. The insights from this study highlight the potential role of ApA in regulating the gene expression essential for planarian regeneration." />
<meta name="og-title" property="og:title" content="Genome-Wide Analysis of Polyadenylation Events in Schmidtea mediterranea" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/10/3035" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="In eukaryotes, 3′ untranslated regions (UTRs) play important roles in regulating posttranscriptional gene expression. The 3′UTR is defined by regulated cleavage/polyadenylation of the pre-mRNA. The advent of next-generation sequencing technology has now enabled us to identify these events on a genome-wide scale. In this study, we used poly(A)-position profiling by sequencing (3P-Seq) to capture all poly(A) sites across the genome of the freshwater planarian, Schmidtea mediterranea , an ideal model system for exploring the process of regeneration and stem cell function. We identified the 3′UTRs for ∼14,000 transcripts and thus improved the existing gene annotations. We found 97 transcripts, which are polyadenylated within an internal exon, resulting in the shrinking of the ORF and loss of a predicted protein domain. Around 40% of the transcripts in planaria were alternatively polyadenylated (ApA), resulting either in an altered 3′UTR or a change in coding sequence. We identified specific ApA transcript isoforms that were subjected to miRNA mediated gene regulation using degradome sequencing. In this study, we also confirmed a tissue-specific expression pattern for alternate polyadenylated transcripts. The insights from this study highlight the potential role of ApA in regulating the gene expression essential for planarian regeneration." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10.cover-source.jpg" />
    <title>Genome-Wide Analysis of Polyadenylation Events in Schmidtea mediterranea | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"DKZrLqi3ruMM9AVe4DSSwVhxXRHnq7xAjnB2jcLUu2g","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460332","apath":"\/ggg\/6\/10\/3035.atom","pisa":"ggg;6\/10\/3035","ac":{"\/ggg\/6\/10\/3035.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/10\/3035.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/10\/3035"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/10\/3035"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/10\/3035":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022EUPetcYGWh0NkkSKz0IqLc9T0c5ktE\\\\\\\/J7vA0GqqfFwWLyWb8kSZW9JqGdpKxDeUy8li7MYowXc5Lfd2VefEFomwsIBUOWSgT19azZneo81CjDavC\\\\\\\/05023tuMJNSYyHBDp3X+XfQivIZhnJ2kViA7kUenz2HDZ7JDYaDPA1Cg21EpzALIJ2dQ1nTnt\\\\\\\/gBeWHRZUG8o3clj3ZkdJwFHs7rb\\\\\\\/pHo0YtPB84H4h9KBFi2MDGJLR21F+UuJfcuHsUpts\\\\\\\/\\\\\\\/GGiunSJrelibSdKE9TZbWRxpCG\\\\\\\/XXGD1MuKseTpqEpzw0ozRrpHJwukAnEGAEzFgU7CduvY0kNpx4F+bOzm00KcHpVtOWGRIRJx\\\\\\\/s0o+fkKfK01T8eD15\\\\\\\/9OhSJ9qy\\\\\\\/W64zhd9AT14eQygWLiIwdr\\\\\\\/n5Y3ylV\\\\\\\/DsbPbgFuNde4RxHQGz+b4wC\\\\\\\/Hn9Mw60uTnJ6GLClrTSGKP2s8S7IvbaqV2G0I+z64p+knXwIrjtuhe9kkcV92ZUD8wIQuGXUJJRgzb9\\\\\\\/\\\\\\\/2Jdu8Pn\\\\\\\/cIJN\\\\\\\/cV+qsdAej6KPSIJ33enbhX7RC7bFCdAgh+kC+sIDEYe5j9REtYVT967OBzpQEDLLSc8VG0FJvQ6fkXPxHB+UInh3oMkEkLMhUwev+TLTIq1+MrOnWxntXlTtG8aWZJGMJJbftSrQLTFk0RrisVhdT2RNP6wq4N+6wacbNa\\\\\\\/ekCwFpN+YksqYYrU8C4auS2fzDwVYujUDU6zt\\\\\\\/G662pXiTzoXJofYaEqfHKNPn5xkvuGrmzUAqxl\\\\\\\/yfpAGzCIuL8cJHzl\\\\\\\/tmjuEVfxPDYnk6xDgybX5BBtDvJs6E0Hj\\\\\\\/4Bm3r2y9ZQKaoFPOyuWtgkbOslYxihPGLjbCZA19ZwxQ9Kl+VKopxi9S3wPkM6XXEHlxuSCB8LxJBU737+WfPtNPHiAdpbmU4CsBcjqDawvUA9xiNG2M+hLuNZ8z6JNasu9GNH5mhTz8Z3makuDziLOSo+dtggmaYz5MXIdejO4+T9uJJRYcbCYyRXRQU6iz9opweEdrue2dgvp3C0XqX95bAMXm\\\\\\\/OJrSybjdT810kFdH2\\\\\\\/BekjG8lewo1xWboPhd0hqzqKZYfKqZVdRmestY6DXCPdHoyj4Ve3vqOp1Vm6CXuicf3oyQwcEo1Z\\\\\\\/lACMoRf\\\\\\\/M+yESnMHo8M5VMpW\\\\\\\/iNbkYT6Ofh4Bp1qBrL3NKQJEdR0\\\\\\\/TvPZnkwsZk3GI\\\\\\\/PTvFJFIXqHl55vLfkbqKeqX4bxexTZfJn4uHngbx\\\\\\\/Sknz4leoq5TKqowKFJk02F2MAR0\\\u0022,\\\u0022iv\\\u0022:\\\u0022+Jx+QDHj34qVRdlqRXGFZQ==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00226f6a8be893b7c763824d8c4d37838a33bf4a5459a70bf836fb625e33632e5956\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022T5gcpr0GNSyY2ul4ApudgxtmEnUS2J5QzvRfmmmAMPc3xZK6fnC4XPP4USdTydxFQKisdJInPu4GSwZ8v+Pe\\\\\\\/luH9AH8O+Pf1hqiK2ufRdV6cTzIZy609w6H02PGpgn5As2eVIetJVfkNv15kreElCcu1p4JeFZZN51P9cHYtW1pN1WI25x9eSes+sRKza5QJVkCd5OuZv54GbvfmPh8hG8vcFbVdO9pKIw0UUK7bc21JM1Okuj9dpzU1Z\\\\\\\/HQoYht+cdshSkKvGrJITq1TJ03pf1KgFoXU4c1jN1L25q99CdTjMmSm2d3Hhk8iHrWKZMEnk1Fa\\\\\\\/bgPgsuDQHzO1Wg8wA+yAH0hEoNdK67z0ztovc3YO3Cp4eKb+5ZYHsxz\\\\\\\/RGSCtvCYEjsKVZzaJajRAj84IscDO8ybbIkdeldiDE2\\\\\\\/+niKivYra8vwWaqtYXFb4mRniYtbuZ+QsrkjFdF35fnLOPKHfltxJeCvCzF1lkxee8zlW6FMxjt4PsF271p5GG3k6YDnQAv9Rk0SApgiWywl8AUbDi71mN4k5ONwMESBHvb+uRCdSGbfKErhrvSKbJ4RdtCxOK5G9bJOGu9yiKLhQc0HcnAcjDiVmRjW0QPA8t0MpHzRktnReM\\\\\\\/xTIkdP7VHEsh64KI1Yf1saufIBNgFOPgss2\\\\\\\/E5qYPXJEv0mj7Rm1SBSxxyPMQX8t1sKupg3LPsCqzjd4XNUvEqNAk9yT2FNPTBq74brld8Vt65RWvDNHve0JnQO5bjrqSGo6pSSmLWPlRgGm3iEuTjPYbgugCEP\\\\\\\/PNftYpETf\\\\\\\/IOsS50t+pQbdc9I8iezT0JbfOhEidbOdIFmoFezNBE5m2zMrRvP0JQEm2SBoLemnZV83OnanPpFIRt24w8yZJW3uKkxG9rvWA\\\\\\\/iUBuKMgCD0EiOFtEkjzI0GyNjEMicHB7Cc\\\\\\\/h7j+24JUwLlVhUO8Fb5b\\\\\\\/QGhiku9o341yIUeiJyri1TPe+MN\\\\\\\/nHL3GBRMSO+EtK\\\\\\\/n8iEw1w+Osnbjuw1igss8jMFX6mPYkomQ1cznJwzFYatDUl8i+0+ijnPQv+kSYzwVzmoHUsFY9eFtd\\\\\\\/3xixaRSqDWVR2FpfZEevY0zzRIg5Z6jJy2yYQtaTUy8spNE2NNtOXIcCWZybHivx38Jl+TuHB4Q9809yp+3P2jNiiUk0PcxQcWkTb1f2LPg2v8higTGv+iuI+7H\\\\\\\/f7sKHUmiL8VyUCL7+\\\\\\\/jAzgwD6TUvhPMU0ZlnJnC2vVKtM6fQzYpvOLv6sn\\\\\\\/MVBhMHTkhQFoJzk6\\\\\\\/V\\\\\\\/LKbA9ISkP3mvs1r07139xtQ8EjL+Ohcj5Z0yBawDNquCJFVIoNZykwQ2pnkDdhkB1Tnd7UyhNsxCHDNr0eGU5sysbTGNLFPA+dfmK9zDhG+F952HLHxrNKqhIupgVY5g8LaB\\\\\\\/m6xgRf\\\\\\\/HVjSQjJeD72XKCR1IcFXVeD9PLfnwGxXw4oqDoE0\\\\\\\/vjMVceJZFJnqfBramUHuojY0s+BrOkFWzZ1hfxmGeE4TWl0\\\\\\\/7A5yBrEsbqxJBduIL7OvvfVJEBj5DxM9QsaJPz0tNnosduXRvzLpXpHy90cFmUmbc7ugQHd8FcF1eoZ\\\\\\\/H7vIDK3ciPoZNJsVX5EBPGcu+zve5B0UU\\\\\\\/Oa7xYCP2AJjNU9VgW3Jix53gTcFlP7FH8GG5EH+jc50UfZdxZwZ0INVYDExykewx+38tZTAHP8UEQNmSENYyty2nW3WhkBX2dY8DTyJYcLWdQJYx3VbJ5aUh0CdBH55IcNsgdCmcjXYgg5MGE9VuqEqu0l3fCbSQVo8BK61ZEkJIEDMIeZGWIdEVqvkxDOFfdrMjnNJundoI0CiVQ23fKXGgHBpni0mpWKMD\\\\\\\/fSnoJuaUpObd1RKZSbw5VJIYPN2ou3DHJ17+8nnIEa\\\\\\\/qcsvszXYnZ4bh\\\\\\\/Yaa\\\\\\\/dUcX+nLKoSmbbCDCiByBD1XQCjUDvgQvby7H3HUceTlMOpD9Rh7B2iEFOHF1a7GZaoMQyaKLFo6XbJksJJ\\\\\\\/e\\\\\\\/4v5vKjNCk5Bzqzt9L8uf8p+ghtKzjijYhC326DPEstvyIeinDxuL8thLsFWUZRL9fbnxM8A8J6BSO0QNKt8pG6fwfLdDADL6EtLMeO+i7qjKhqnbakx7Ga7mNFSiX5bHm77bj8f+zQlfI6z2IdNfa0jhpP2BZcAL3Yf\\\\\\\/Jz\\\\\\\/PIn3bOwbuNaO7e9EWKY6Z\\\\\\\/HXIuqkU\\\\\\\/OpJXojddXyF29djd32nykWw7DzZbn0L43E\\\\\\\/\\\\\\\/Wb8J4AsECZdeKi2lrlxl\\\\\\\/WdfDHEzti5KBGKA+MxCTDIftfJIaRDbJAtyOPN9Z+rmfy9wI6wQKjePI6x9xu43zWO4T1CvB89z7b50rrzkKq5OF117yG2yWquB0Bgmd9eqQ1a6xxVUDXiXocIIvXDHVjMIj7EQg1sWv2T0ugQ7v1209AJA9bwaciJSMTbmJCQYG7q8id2tdIce5yLoKE1O2AcmMAz4Ds74O+dKrVnxMCIsfmwnyPk1zfn7TMqsWYlAB0q09jDMHMtWXKsJVfIEw3g\\\\\\\/9PA5Hw67lFHLUc6jbLJjHMoagcn43w0IAivC9bDvUWChQ\\\\\\\/S1zTloHX6f2thcCSXKh16kDPimMSvHbuLGdDF7GHfxURPW5UT\\\\\\\/fy5YsMxB3S7aaGlNAO19Rk6\\\\\\\/NjJ\\\\\\\/Oc5HqkzrdHIEJBCISyGgHcG3iXyO+5Jf99E65VsouQ1Y\\\\\\\/qqQv30heDhN35e4xBeVc8Wh0EB7emBDU3vcDe0dv8zJv\\\\\\\/E5MAdfFePEPxfQ\\\\\\\/YyyPEouwXQPrFppPeAS4RC5G0xb\\\\\\\/iWJ8v2RItkqbDdKBt7zy6ddEG7qTWP1pQXvPOfO5oaev\\\\\\\/U2+VkPXxvBlWsUoBtyzTHnG1xbW5pp3mLG8F3bjueK37BpPDKgv3RVZpRanbn7NumVNiWQ+cM5Ucb19hXs23l06P3Ukhl4H8SjYQl481dp7J31cq8G03rc1XfbnCyxULegO5ublFm187OyGqwmtbYfzqdYbKrMp3hdyoKWKBQGlmCNNGCjAoD43cg98lTW9+I6WCeFRGxSANC+m\\\\\\\/oYJycYq8aOYikdMKk9kTeIfSm8Ox+C4ILEgVJgXl\\\\\\\/gtZXl6IvBuM82m8cuf+0fi2m6tQ8zWUUetPx3jOCf+wuQIGyhLdLLCZzhE1IYruq2sLQOIUFqdZdhL\\\\\\\/ldBZtsvHBbwZjF74rbbJrt90V89VxJff0lAgv6Y2m\\\\\\\/dkJHey3+Wld6OfbpFHgaLq4TdLGtfMqE5TVhmCe8\\\\\\\/4ZCmmvzoWUBhyydkBob7v\\\\\\\/aT0e19wBf9EDN8zOzFmCeJLRM0xzvGQUn9WgBDydO3dG39GxC3P3mQM7WqbA409Hw2VwjOhxNrzmJ5dZOG48hCdX6DGxXTByEbEGHLRSWGkE84jI1vBOeuKC1h2jrpKKseHBwrLxGIQR2DR2+TFF8VKcoYfO7qOPN92cwX5MfBioNqmf9cSD\\\\\\\/g0o3xbjX3nTDsWRJtx3tl9IYW5KOlIlgVwgMzCPy0k+PutpYQEkT0jwaAI8ASH2THlIJU4eRlZScwkpRG1U3tpcgr06Nil6RhX0X1Dtmxu2gkTfjhmhViVCVfIDugO5ivQJO\\\\\\\/u\\\\\\\/AdjjnROviwCvJns+6LVUhQwJS8tcBSWsbiKyNU3pKPf5xtWoOeGKVeqMQ+S+LjIpXg+e+OOzBPwYDwhEOuYzSNbIyDmd+Y3g7ql7D9MFvxBHXLdTesfpejMoMVykjZe\\\\\\\/6YNv+DkMPaul\\\\\\\/+re2jhC366KxUw0wSI+6jJenlqHPDBuQxFaWPBXQtu7h7CiNcUjXDPKMGCkQ\\\\\\\/fRwZtgTZRfTfiGAPvXxZ48qJkxSxOUTiasLrqOPmfSdP6fsydm5pR+DVGV6WAtEO1DS5NvSeMju09CNq4O0nRs7iivhv+mopE48M3r0LN9Z4Q6VbU1hzH\\\\\\\/7We96qvqC8Kmhh9Zg1ikSl+bl45MlBJk8L4\\\\\\\/TxglDxyEJCEkoH9fbJBra3JVxhpLvJbAk2pG+0boMteN4ZfB6THQoElo7VXlpKJocNd\\\\\\\/Nxk6YJLTTcYSFfyCLIR79+XjfobPXgK60EZJNyWOXzPMhm\\\\\\\/SNSrMKx1vQgCoO3r+clcobXm9w\\\\\\\/dUYVk6dS+9VFBqGZdEQQ3eVjGUnMlf6wAR+02ROmyMpXdGm1ze\\\\\\\/7TNfVYC0iuttP066VFhXZ4+UQ9EAdUO5xEJDM+aK+sHfMrFePUuepnUpbM0qeKbIzw6CLZ3vTBIqmJjlm+6tOmRPcUfa74Vs7sZS1T0HNdUnF7SwSFHUz6DA1J08SLGT+pXLSQn7Xy+YWtOT0qfUMUBASftD8bP9tvFVpRM6FFf4XrcJOXqj3Co7NM5efEEUvlgIY0t767ELIN+DjC6gukqhKoMIVk9uF8Z1x9bC9cjqtM1D8+Tk6BbXbvRdRlcYlA1LJJy+FGlmOyt3GTdzCG4kRRBMQL7+J\\\\\\\/M4WW03PSssF7Pg\\\\\\\/icCQxAUrnL1D+9jAcEvkAlymdiRnfhlNGMKuM4KhUPEV23JH9V97JJnt5p70YmS73xf\\\\\\\/eaw51mVZ9yB3C6\\\\\\\/m90GJ8mAceZUw6\\\\\\\/+H+HhJrSlS75qnuSK\\\\\\\/WkEAf0ZkBcjz\\\\\\\/coCOajRyiOurJsDffFj1wbGAo2EEztI1CoLXy4ouAO\\\\\\\/OpGXmtUhJTX4rA+JdXxImx69es1rrIqvTMb0XjybTNUGDf0gZSjWxfR8WJPd1PwBPI60ac\\\\\\\/62XoJW\\\\\\\/3B4jlusRbq2onkWOFQE9csRTlTvJJtdcyIOhln3AcOYMoFQQh7vFpaOIfjTvl42CCcs6Hm+XjoOSjcrcBiZLQFplyfB32THRODVorIy52NZNcyBNTUqSPZGVLQbUUWvqc+NdR5w6+1E6ag7C7jDKPMhmCSZ+9MpZAqdJI8+SJYvLg\\\\\\\/aPUBLp12a82AIiusDlOGepWaerHaWEPzC+9\\\\\\\/kasVopwUcVfMzw\\\\\\\/TEVRaqcBcrUMqLJVBpOfaIckx\\\\\\\/vMwnNmwwl\\\\\\\/PDarcaOOW7Wcxm6hO\\\\\\\/RDmEorFOJk3MRSfboG+fg7+hzsUT+\\\\\\\/40qAcX1vNjv6r5mraibR6v5jtO65gUH+SRwMRiIKz\\\\\\\/jk9S+KwO2iFZ0pHop4oJ0nZtiwG7WzZUnCr7Euhbf1wWlX5ChexjEL6bjdDa8MzhYT8hswLv4ISB+LK5sGoUGd4CPRVXO1YMtZPXASdkzVoRdkkAPqwpME663wiesxHcNQ1zsdqhI+k93ES+SCxPDCHLkDQiKGwj5oMkRYnan50wsroPSsCSLWcFENOueCIvKEpOTdcsDLehauLLAYAWFW6be2ALXZieQe8ce0VKD1QBIMWIlpb7YxR39iwGnGUEiCpKsljWUFgn2p2c2Kb0OfbqHobnbbTmqopezs3UVA1XxL6N4dRrW7tlC+SVur3W697OMLMD0zjlls1QYEwqCHDLC9zm0bx+WWe0thcdO\\\\\\\/+guV\\\\\\\/1rDY1z3maJ64v+mem1bguLtNvxBJojZZytfdyLQJroxUQvdcYZ0Z5JOFitNweHPeYMAN78XmL+3h21fntssI\\\\\\\/FqHxCkaJ6GZ+DsKMOcBQ66NG5HKEDpuQJsGyqq2a1mw\\\\\\\/XMPZ1PmAj26w5CQy9SmfWn2BEplAZMJ4NWw\\\\\\\/0hMAbl6kNUk16FSd0zW6DhhQYaXrs7scFQgyS7coae7jeLkC447DKugK5XNidxe7vrQUIhG76l8B5pIte2\\\\\\\/xrsWcumJjd1pl4cHWEewc\\\\\\\/jAlHGVRmrAgWyFVqQExXmZP4usZPqH+cl4o1aR10Vwoc5Tb7ogU5qZWgTsrbLIkHKkh1mWG+HjReECPbSDJrSfE0GjDcsgCEUESQB280aVWK\\\\\\\/4aJ+4FV+1FzalI3ZQ8wGXoRgZeM403wKFStchYAltEDFUjRrZiYElTsL\\\\\\\/uqyg9wt\\\\\\\/2f4pMJFc5tft4171OrLdNe6OKhLoi9sQelw7okU8nwFQARtQvCeYORqz79mGtveYw+BsxWbSnBAe+TSYE53YYrfDNQb3uHyWmSLCTD4g6W2ZY53qtf0BLaWoocuu+ieNFSY5Q0dNkmJu4hC002BbsxzhfHq3mwr8Qv7oQFG9gA+uzmYnh8g97WePrT0Ba5ScnuQ4E6m89trSz1u\\\\\\\/gtJwrnmF2iDUHvY09X3KodEfD\\\\\\\/1uv6wA0HSxxr1C5udTs1MhTP3Zy1ge9J2f9mOo0NQs343DDjrrj3o7AHvVgNlKfJ5055S6MXzA\\\\\\\/A3D1JmRI\\\\\\\/VMe1Gr4dwFSGqRTh\\\\\\\/ga6sobZWTpPc3j7UYoB6N5kNiiFvY9133z0J0fxuiQYoCQi8Gp\\\\\\\/tN1eom0V9phFTOdk4GgjA+hb7GfLY68JMT4slkShXfQ0LqI4Nc15R38c2plBTQ6XGnOHoebUvKy4qTnb6VjVVj5JQ+95Fs+7Zb5Mpq1tA4rByJdZd12jm9yBKaFazkHcCMWE8VNl4bOC6I9xq6uce4hmkoarTF2ubAqta6ns5hbQvvYOT9bpE7YK80cyAmhV7T7LfbOvYkIPFNudHrkXo4IhpXo93jSchHiwPuuUoLDLXIt1tAvXhxsq9NCfa3Odfia0RHz+0S6o2X75ioosmMuRnxPpsb1Mbwwjr2ZPGeSUQU0XizqBJZjAf9pexivq1Een0SbzgK14vhETuXUQme4n65ymjMT8nkKlcPTEybApkin2Ud2iwGj9Qk8Qx7aKbuUEAxceZIVwPsYv6eeyc8UyfONgXv+eexQM9tnOcwjJnyxJnYwNGVoVDh3H2kY7wvcj1ZcHYNtWJPw3V5FT0EIngTAimNr5S3orXKHEZOBsGpwPNyRbziXFxOKq8vnaVIAyN7kSxg6xlboP8CP+tlEt0xyX6aGI4LqjN4+3UZzSXU8iUOT2OI8S8oFWhB1Wdj5OQUFEOAaMrzDZjJARdIcqdupB\\\\\\\/152W7qYwB\\\\\\\/bp+SsuIRTzQ\\\\\\\/5fZLCVM5pirPcAsM8dGxPb1bcSiIJgxHPz6bcWApgXnKBWDwvgASl5YRYkFfUVGWpvHlIFaZVVjb7w8ZaZDryEJOIoBdXZg9Tcl8v38bQhUbkyOv0CqCBD5FeyXSbY9HzQQg3K8pc57aOBjsoMXoYrRJSrORShupUdmnYJV1\\\\\\\/PZV8I46u4hbf0efvEQ3Sr3krlwaoWhxRaBaCvl7We+KQDmIz6m3DtD5w+4sWBguL+hpF\\\\\\\/ZqxB5QtBAmhjwXczveGexXt\\\\\\\/WWPXnA7vWabmaQ9jhKtEaQKOND8OK6wLklsMwxaE9vv4L8Z+EI1S\\\\\\\/bf3K9WUeKANrJQPp4z0aViJNFudxOTYgl7X\\\\\\\/DKJqUQ5Bvceb5Yy1b4++q6sUmkNubTzC5Lcv5P7BSIwjG1FKNkdA+uT8zPpo4gtHUqMVEI+bVgxuUZPuLo1L36xj8KWY\\\\\\\/dFPd\\\\\\\/76OyhcLE11WmQ4g5RbD9DBV5UGFhRCoUKQO7sQPnon9MQBCr+ggaUUCZEpi3pMxIYBiFSlaBUiVLQCIYx4HECjF2rFrGKxMGmE5MvH9IWRQF1ua8INPaWYOri5ZZPG1y6VqZ47rlVRH\\\\\\\/HkR6CWwj9iCrJdtP1vfSjDMNnwLDgVWcc2DieKM1yuxUTaiPdprpO171oRH54h16q8PRnzEmXHUFai3Z9eYpd7rlsXmwX4AVMbWEisK2HQZoYv4zyCXodvbjUdHAFnBx9mDMHudPkZlUTpTpoZXOV\\\\\\\/VYmhzPQNapdc\\\\\\\/eeU6o+qamZwO1RRqOqY7S\\\\\\\/Kt9iRdF8YkpGt0WKntlbBi7P0cdhh1w1PUM5hqa0vK6ObPYAsIfp37rv6Qu4T6YmvidQh9+jpQ8fdiF3cATdjQrkEArJawuX7nfsmcRy9e5LeOkS9UB3RKrAM1jD0XglSTOz9SXVztowQeRt4SAZrQ8hpvd0+lpLfjHPG5Rx++se29CzyKcZAtLreYfc8lRIrf0lzgWJWwt59AWgKWeJDWVfBGlMRODUESgrNbN68ew8+We+a4DQtKyrKCBQnzxVufseoAcTLrW8JgFIcteDZGuGMFtnMUIjntfZ9NfCK\\\\\\\/S7i3nvme4V5dZkq\\\\\\\/mLf7jUpb+Nm69pJpRWy7gBFabkMKTr9oP8SOL4hGcpN+Lv81ljZ4wi3VsvZAEEHDE4nXKaVx8mvw4R45lMYSZMatUbn8gXBz+dn13h3lXYuhBO1VygT3xUMg2xGV9Fc6q65d1P\\\\\\\/I2v90dkqVvkwpi+fp1gyqPFIzCxmNxco4oRKuNI7QsGge1wmUsQfe0UbT4T\\\\\\\/ZIUyZZPCdt4QkMmwJnKNBN4Pu+6IKtGO70M\\\\\\\/wL3gylIU+\\\\\\\/Q8UVaIFFxTJZYiuuvNKEnw2Cbaa4j+QY\\\\\\\/YyWp5+KyJgPv8AoT7zZEK1drO3HFqVdCbouJa3CMjShIiUemrvTk6lCDCKO7Rmqdz7vq0nVVBQBdH6oVFAmiasVeo2DmZHAMOjYDAZO3Nt3YvPNNiHwLgr+lKfHEg7n4gyAAa1VVqbbJg28gUbE7DqZamyeBX3LBg3JvW548r3LUTlhkIo2yZJ9jWXeDzb\\\\\\\/kyzQDTHAMi7\\\\\\\/yfxFwPHQXTddiUVHdvGMoXu1LlUzf0vB99RkTJlpszAhrhfJG1dYmy1IRqzrUkeFTVJsva+SCL95B5tbeBsJ2Ywd7theExPiFPieuvRUu1ACd6\\\\\\\/8Eyjj2M21g5xwRu5ZTmSCDDKCVti1jQ+n\\\\\\\/xwhi82UDppyAwLkTJmAPTyAvRm+VTlW9ZxjYXQqj\\\\\\\/ZoNRDv2NxTlqOTATp5X2NjZOuk9oL1dYmcb5pkwf99fEiiTy0JdoNlFnezNuUsInZzNjshfzAX3JDNrNmbpR+Z5aB+Wl2Hfp53Mx9ZoF0mVyQOnhubcsVpuv8QbQU0T6NXhtXgecElhMjjFLocTbCvld4e0KAH9lwz7Oy2FpMP8wF1QfVL2IcWsdYehvJ97stbNIfIzHlb4FSk9+bcTrvhv9qHSKp7ePSgVnsCBZ5ZI4BekR0pkh2b24cH0w+OOg2daZDqsIzB532a4bRMPNL1pTRUwVRrqSovVld3Fjurab2moECurx8VlgLEdDV+Nhmd\\\\\\\/0UVjtr83KYelFIWyaFI0wiR5tkJ8ZCmLe7Jt9IH6YnxXSGb2ZsPwAeNDvGLVZbeTB4MipMLi3XOM5s\\\\\\\/JRFHLOcGsxDHCItJnKwaumSwcQ6fy8ZwzI+UkouDplU3Mot5UDlxX7u2GEY\\\\\\\/qxkyTX4Puqz\\\\\\\/pJShtSIgFeIOMVmQTLUoh99Z1QTtf0iMd2bI9CjfKrUoVfEjBP5g5Mdw0n5v1ccGF0fGUNL9WfR9LyRBaep80dYtBZwTMchRoh0Q3NZFQ9JRpWfeIfPVB\\\\\\\/XB0d9Ak1lhR3uw4YJd0HVaCs17ukecUPAiZVKBOLAxSAZLQjEuAEOuWXqW7piVt8cZn\\\\\\\/va\\\\\\\/7LvGTVhHOgrPcUyofaa3\\\\\\\/JJMHd7BuWBdGSP1buiKbqxv2QlbvVu4UHmbfBevKzECdhdo2LkJ0wLga2obnUTopBreOTwrimXD9rNnaDx3aiac3TaEKf3TA5ITf8nhl0X2+LxLt0zQYMB3vAuTtExZmr2+YLH3\\\\\\\/FpO3yd7Ndk6litnzi3CWpi9jRz888XCh9\\\\\\\/fpZvDr61d2OGU3Z1nMGlxv2\\\\\\\/D1bvTgPhC5hGXDPS+JFVinFNvspBtsAO795evBFKt9y19z\\\\\\\/p8uQjdKzdkKs3hVYw+2SoDHXhW6GTzj0IieKX8sYaLLQwT2T\\\\\\\/D60lupQNnhEpRj3u4O23MODca3GYIvVHtT2OIUu4IMJlBlda\\\\\\\/L1Tthej+si\\\\\\\/loRpu35QSdWkE7oRf1Po\\\\\\\/sol+6qTaaalbK+MVZPIXdKmYzphJECuTyYbfsZZBZUaIGtr4fJSxrcdMSNa9\\\\\\\/iBeKoHVwtiH1ufqpMA5pfeDTMuY05YCcVZe1pHdunG92C6swXiAXkRsFth0\\\\\\\/zOq3wU1LgTpfhXAjhDrr8KwThDfOhCoDgYFb0waxVTVV4Dh8Te1\\\\\\\/kP+uWlTKfUChv1MxoHilLQc8wYcIaVDhz4vHQN8rNOqiT\\\\\\\/e5FK8UifHe+geQHRktocuTmnE2cHcGNXDU6tqi4m5EpiPz7SbMqZwkg887+S+l\\\\\\\/3Isgq6ap99au6klnDQNW9vsJH8djcCM4vMxnGLk9hnHHulaLczcKcFKM\\\\\\\/prwmca4UBdY2+ZVX1wiiN1JnnlfdzQdsu98tbemmS3KYA2v8YCUu1s6jmZaPThZju3+PamFfrG2o5LEQBR0NhFOG2fK61vrhbYUIGamP+TcPahTNa838TpPgZdWNVMoXZtHImm89SVNhZwMbQFjvFjEyUmd\\\\\\\/A7teAdGweIgpGkya0YugQiRWmkaFfsfwENnuiinhDALvJwHc20SPgMOgNOBG8Mi66xfps\\\\\\\/UZC1i\\\\\\\/HpQQuCkcIFXYFKPX4TZ6mNwUn8\\\\\\\/lte8oP9GcYCz7GQonpIKFtlw8aga9owUPzLSlFk1+dLWew1Vg4+E+j2hnL9p7AE45PeRJ0cyE8w9n8cCWLSbCReXg9jgOILaS+lQw+A7MFM9TN\\\\\\\/ysGyh2hZEhx5Hi8x2A+ArjMRtg59F9e5awHtdlDtlJy40mFx4z8cpbix2c9jDRDVvxdz8tARO2pxgv5Z\\\\\\\/ZTZwGUUZpWCCXeD0zH\\\\\\\/vtd3Rv+xizP\\\\\\\/ZB65NeNMXZJ9B02lMOqsqHrQohSRvgMa1Ks8DX1MHLR3WUi49zRK4632PvDMtUlOM2e5X3DGh4SsvJyeRwVGhEr\\\\\\\/CNy2BwV329lhrGbp3qXTE4VTEd3ZaV+5AdwOCUf+38zz5GtQbKBdE\\\\\\\/oniySbnDU7SrZl6j\\\\\\\/mTtgIYxZoj1HdV20d4v6Idm76vdCgI0aBvoslmuudtb7PXu5EVi7Rpt4Skjv8K9Ofi51OSyI5MD2M\\\\\\\/RlZHSjCwZG0u5oWQXaMUwS\\\\\\\/YpHbw8KR2pfw9G2TjLTAt+Ztlluwct9nq+nbHaxVwfo6KmFgErP8Fo+EiJwIINTkDfCV1YI9Jl+GIXk6IbT0bj5MsUyyNRMrZqXRdEAQeSdwvmwPrWlk0n0Bgo51BXKMvfgb29t2juWhgr6OYJFyxZwim3\\\\\\\/kOr\\\\\\\/KgwJEleldUfU5xs82AgrRem\\\\\\\/\\\\\\\/17Ia8CYZ+J2b6xkFXErjGDfLBTHwMCysA3lMEfA1wFqCuREt0qnAI2cvGrWkT00i0tiiA9T1RA+WIKAmZTAVMR3KuJesYbJzPIuWDYbmnBb\\\\\\\/OVP2pbUhmuV\\\\\\\/2\\\\\\\/dm1zUPOL1wsCTsTS9vsDqvklFNW2hztKWqwJ0Qf4Im0AgRYzcYd76vFx8QrJXR\\\\\\\/Ru6UdhADK8zh4buQL2xOUZm6+oA92CzU4HDxpMldv\\\\\\\/7+R3PurgXxceswD5oWlr1ak2WFcwiIh0wBnZonyJnFRs9YcLLE1V10ONJuE72FaM2aF2lvzJVBtlKJ2tmlSYKrdOzMnrOdT\\\\\\\/aMQFY1bPnZEzFq44MwiZ4cgZnG\\\\\\\/LM3XHZPhHS0+7eDCMUpc2kOVemipSQ+Huli4yZplcVP0uL1n\\\\\\\/6DFnnuxVUnFquvsEBvz6RNgZXf2E2D08eqeujJo70KJxbE8eK7f\\\\\\\/rHNYXD6ChCexZtNefYYVG8823K6xwGgu22RR5B8Kd\\\\\\\/hueAbl3eAwxbKBvD1pLa6a9pUJINOEhd4L3LrElhTsDz1hehNjAJdljJUjVUi4slOHYMSKj4sdMZDY18Qe913nzIkxx9DfZnOuE639s+glhoW4tU4yfRb15tUYjdYrKKEJxqWS367nBzogg\\\\\\\/zqRuFCDRHQsaIbLjSIDy\\\\\\\/MMzL+DZHQtVXPT1MdA2Su0OBUGAy1z+\\\\\\\/e7c223cnQ\\\\\\\/zqodIAdWlUj1unhl2GvKYXBZg1MMUQa24dAhhVIggv4Xn0+dOBbE8chw1vmmp9ne+PcCfpevG6fDEdDFm8SxH\\\\\\\/dgL5OgOGlhamhhX68V8Zy+omQt7zRTnhyjym+gi2G\\\\\\\/bF75HoTf\\\\\\\/41pS3E+6Wg29iK2XysFPAGB7p42E0GnoSPoE\\\\\\\/p5ecnIgwxevDpG+J6b50VcY9RhTaRC3bmUiuwuGI8lfmN2LoVbo5ZXyXfyndmyVdEf+P96LxxE6YQtC0rx7Ki7w13uvi5PPEr3VhERzvJQ9tkk7z+dirRTwQ17pTwyg0i4fyisHNf7\\\\\\\/57Hgr+\\\\\\\/\\\\\\\/+rBlmdp9SlQLvX3O0s134aTTkzs43fh7EnuWgl\\\\\\\/1JU2jm41mnPs5LWN08W3Hz88ZFmTET\\\\\\\/6d7EuQhu2xSZSR2kKcio8eex7im5kxikGG5+kx+Fz+6K78FwODEnu3n+virA4LkwQ0xbZw2NP7YOoiMgl0Hi3MDxq8B\\\\\\\/4RE40MxfZjrC9sshspLE6pQN\\\\\\\/\\\\\\\/6ko26pipILgwrw\\\\\\\/JivG\\\\\\\/W+hBUIF7CThkiH4nYS1aUD70DN8Zoju8RY76ssM3QSc1FAzXSyvUzlztsLuqnwcz0hrDYL9PtN6gSugebsEGnLKt0HQvmjNaGgsVpXTjxSXpjk6U0qGU4kQwU59pjWhZlY4X8WGs7VVJF47xFEgdz\\\\\\\/yQ1bbluvfQ6T76+S9oNC0TnovaPgXfKa+Z24FlgXQA6y\\\\\\\/6zOwJB3y\\\\\\\/5FoVvESECGL0nc35jgvMOvby67SGBp5AQs1i2zLlu\\\\\\\/+KLTqOudcEzwQoaclRVjjqlSFPwUT7xRkjBSh8RY4irSOWt0yzi21iF6AMj9DXA8j9paFfv7tNrtaSVTvaP2NnmrMhzaRY8jDFe2tQlslQ4EVmwsWuRokllWyzmJB8DlSRg9GJCIaH5fVG1PjuZvKgaK6gGTnYHMIIKBs86a9sJ51W3ws3k7oKtCyHYV0KOWTo6qZWlqdHVZhGiKGJGZVU3Lim7T992AXHXiZxPFZlfD9AfgTtwN9LyJL7HtJcOk7SlaVq5JDDl3iiCUQV+KnOclM\\\\\\\/uROhpMiGNVIJPgdYuH9C+NLKiLLi38dGSGFKlr2Tk\\\\\\\/QJmKPeHtrLhZsCYQ+IIGhdoUUaM9qUuunCQM1ywR1Pv2w98dtAOOokwnsetdFpea32pBzVbrujmTP\\\\\\\/LW+OWswNwFyCVE+KDJIe4lPiUGKmAkYo+4OdDqT24CHGxJr2WBFWjcLc29kIxT3SH316vzTGGSA8JrOzYq2eyOtQqEJtC18c1HAuIsr+JBgn3v8PUgT8mnwZHNztdqnufT3gC0NJkcCocQKq0boY7FulVO5U8JFrx0Uodih3WkcUDz28dtJ5gmjnPqSrqlvMybhsi50mRA4ccmJYa7lCC+VTdPSJzr6JvzynmW2YfYyP1mYa8q+J8NOM3gfcf2nS6GUc9hqlP0W5u1zuRSkGU4yaj+eVgipBPs5uEXaoumXLAoBxfQn6I+MvGcy9C9f1ypZlGG6Qb9bbZ8G7bSa\\\\\\\/9FIQ54eCAQDsltTXvf6IT81xiWTbVgTghpwCuvgcV7O\\\\\\\/hOoVDWLgq3JuUp0c3IZ+42L6+\\\\\\\/pCXDVdhDopWk9qNa1a\\\\\\\/AaLPk\\\\\\\/\\\\\\\/0T6Z6uRQcZdngIye0YwbAW9VjG5X+lImRuBjZklqreJvQvXFh2oTy\\\\\\\/X++UQGvWDuoIQTF1lejp8B7qD9U6z\\\\\\\/zkcHm2q9vg\\\\\\\/\\\\\\\/\\\\\\\/l5mojHnQcWfGvwrvf5kLZ3cwV6MBF8OWZPkXRl5l04zV3QW0k0B1d\\\\\\\/EMOjd8Gf5sFkFzPhxCfQzI+94tt6IkIdMYFmMX7Di+jaH\\\\\\\/nwfK0+ksSjGLxo2jfK3k7XoRQFFbfaXjE8J6psMA9YqA06Pgk9mY8fi7qEIVDv\\\\\\\/TvGHksvmCRCRAvpjAnUtvOM8wbLg2MnZNs9LCIlrqb+YUEHjpWEvPNQkbN5pWmArvHMBwuhsA5lx92k3ZeYHz0kBBmEHqv5+VKF5gMSDBlEeVE+F1qhQWyv0EMyX6AqQhSdmfFUcJx\\\\\\\/SIVo5Rbo573YNtD9lXeNKCB+1Lxd4BVSlrevZ\\\\\\\/mtnr7\\\\\\\/qyAGKyRE155uneExfkzLwUn5ArEkLdOoGBJHWFnCGDGkK03D4abzQSYjvEwKldqiLtF29Y+l59gz+TjGJP6rOQyawjivOR9NNQV+80fabO\\\\\\\/FyB8iXIrtyPEHWLDlJljQ+zi0rrYlYExvHAyqr66GKQ8ayPKmnhtDqP3sPQwOpJZxC+1KlsTb+knVW5Jo4pdbmNHIk+W7Aj8IAA58nxYQcPy5mqZ77gM9H03tfCPzfqCrKTNCDOACe\\\\\\\/4Zwakl\\\\\\\/nWSSp\\\\\\\/F0ysq3jZzJRDDpylNBQNW6\\\\\\\/WX+UpBSRWYYJAaXuwDN8+3IQ+6usEGLHF\\\\\\\/COFGIFQriah64E0hTLYtr91q0RZNFXjelCtIAgwBsSolpDcJbmxSiTUvAs1BZzLNruaZ53SeC0mdzpZ24XJ47o6ePrmW2r9pV7Va7d7EUV6oDlRutU3ZHyVZm7Jg9he0i\\\\\\\/tZxKnxRxlfMzimFKbJ4zPgP1kGhusJresKPXEXmijEyJlCcGVj2jN7unutUqwvEaTiVOCmV2aHQzCeDvIWWGDQRLW0L9NI+geiiUNOujsYMBSc2MYLYIyXDt0DcjAbaGaWqIgajp1B6WuYn5QlibbLdY7kyLwWPrpBy6+vYNwDIOcT1lVdvQ+3h21tSVFDrdPBvGy2YfrTT0bj+\\\\\\\/vVkIJIb60YP3LMOvxuTFpDwk5o30L\\\\\\\/zCV1wK\\\\\\\/guWXg4oxl0+Siwi3P\\\\\\\/Y+RanBAcPWZbUiXbp+mUIZVFX7p3Q1xHAox7+zMtSMyx2tbkzqe6trNlWSYMmU2jQrdXGIGIGKGcXFv1eA2Yt9hDdC\\\\\\\/WtMhXjkpgI2AO3cCMzTrfKAj4KolyR1mJreHjuwY3lpPeDMyw4uOJIIwAUS5lMt7s0ARh\\\\\\\/oF5Tsrlk67LZ94G2+0qKR2CR7Cbbzf4XhPjJ67Utu05U6y1bIJQ5\\\\\\\/JY8mqdKJ39kLdBtZ818V71rDWVchjd1fuYnnOQu0KaDIDinQc9uu2xpsVvrGLXr5IpfPQbPuAM0D10KXY93tsUejGAqo1I8oN\\\\\\\/jzFEyuM9CBFLUTuAiDEfNPEo2vqtdJXVV0+RiIDZIfuPOoiXKvOBOi0efH\\\\\\\/31hTTbyA5NCIOz3gdtUZW1WrpXaf+fatpKhrWHU0QwoTOjSxhwG6qzGRTZaH5jp\\\\\\\/SC4eSAASdHbHfXodS8D7lLtvzrCx\\\\\\\/HZuRqbmkR+5CKZwEoL2vDQpkFKbm2PNduFUKCBCr2lX4AqSuYOSLgCKTLawT8g1gqo3Ohl6budA9aDAd4BmhJ9BXIWtQFfZqQcfaZR0gOqpnr3a8uRZXeHBZ9IXmcCsRd45RTJMVH8Un7cRZVYlmF9NavOnOfqo2ui8xBVcdvrBIFDSXRy4OGFwKakiQazZOyDpHLiK\\\\\\\/0HBn0mezEhZJrMGJUktUq0RUwZgxZfRbCfGe9iaTmxqiVoSIr0F5iYoFjttGOyHjK0boErG+j+AcNAkzKG0wjlRV0uRtorMY6Y7rac5zC0JiEIuD6CJ6+sIvbAx2PvvayJ9\\\\\\\/qOKZY67vhaSxWeCyOTyn8Olsgps6IhEj9IRqer+5OxS79AjgWDd66sA5W5eE+N1nEYytixtc5\\\\\\\/YC95Zg+pGotKQifwNM9Kv4mzN7CvwSWgOp\\\\\\\/iFIyKOatntp+MSetdQCgngahA5PODiONhVkXTfEOjbJyrKaIIeRPIiL3DReiO4ZYaS0s7YzdjaeX693Bl6Z\\\\\\\/WMH3CWSPVWzwDiDRBxONu+KS1kSZoN9f+jU6uiHXDj7F4jvXx1lOcSqA6Td6Ykzj3W78+3XBz9eWZTsMV2T8AnU7LvQiOXI9lTpBLKe6MnfI0wvwedoIMJCHZftmi7W\\\\\\\/KwZG48Kw1O0QWMQJNJkeiAylUg70hYbM2xFM\\\\\\\/ts40NQ2aoDrQTtyx7Y6eFVeFC6Emw8c62oomiBFuriGmwxNCY1HuIq8\\\\\\\/Rf1N0vYEC6IglBj5brVtl2qHR4tLQsnIo6t\\\\\\\/Kmo4UzJgSN8+EsoMTo9d5hwsENzBpwZ6hWWdEbYzDyDe7t9DlxjFlMNgeZVpkCImJTK9auUzBhxiSncNYnvXr\\\\\\\/r1mRTv1GVpvSK9\\\\\\\/dJhZSl1isP\\\\\\\/MltM\\\\\\\/pb0lq8KdnRqd\\\\\\\/NdQE2qwWTLU\\\\\\\/eFZRRD7ra88ZIqNO38XfGFmWAXYYXE22B2CzGIamysrGwN51oATD2lX8O62+p3qmfU+6O5spycoC14MgE+PnTG4FiHwtBxEOyFZGjhCTvtxUPrx8MPbzjCuvb5yeRABPu+EiXvCla3nvzfF9c7iHZvRzIvJBmK5\\\\\\\/nhPTfqXI36XPp1v\\\\\\\/tu+EcBRFpJnGKaQ41kzlWN4FasLEaJaty\\\\\\\/FvPwSTBoTxqJvY6G9loUmEA5vQvBE0zQyqDOrRB7fsEpWTk457Yq0snyrZOqXEUN9b7TCr53CZtd7A6Z2rrGOl+EwITAobf7XOIE5rf5X7mTB3PBMQp7h6ltmlpFGAZCtdZJ4q8J699\\\\\\\/F2cWRt9EqlLMlhcS3elws62S\\\\\\\/6NECZxZW2nwrOWglWl+\\\\\\\/MK6eIwCjGDUhPy0YZTvCU0VcRPW2ZPggd1URtuFBJqFYljGQU+K6ivjKd1v8\\\\\\\/DwhuUpyNxBumJa5ul8zMjNM1y\\\\\\\/AKEHUdh2eWpcM\\\\\\\/1DJ0wtKWJQIqFBAR2v\\\\\\\/GRStfMNZgL45wUa27KLJTrOIeVocQ512Od3MrLE8sY+\\\\\\\/ST1AP0mkCgikMJ8lyhDNEzXkLxXlZaP5B6sRAgfecnJNQwJGre9DnbjuzUKDcIxLdXypoKrnVb3RfRhG7KdGHcCTNU1RsRMuXvy4av5SSgd8nd2qG8EWGgKJPrrVb1UjzyURhuuiaGj3PeSVm9qS9LIynRaEgmEzCVcVD9oewwaPVa6U+F5HX7mZ0v05r+anIfmQ9YYjK49cUG9XTJkVAkcLflQVjF7RSh0CNKPlaEMc8B3Iz2CdezxS7YpkR3qVrDXgOUjHD92LTptn1qPmJRetlPJ\\\\\\\/U9V7DLdgaUH+PX08ClOAh0CDcZggXLUrjOeXx30MlXxsT\\\\\\\/egbOu0rnN+dyoqWHTjcpQhoyXcm1+4t\\\\\\\/8i9OWKC4A1mZ\\\\\\\/sJKY3Ozzki3GTBqmpBua3uCElTbYBv0Zut7UG+K7VeCZ1pLQ0REBFKXowQ17eeBD3SpItFI9Y1t3AjjkvmmR5hqs+8Pq2sVHAofinC9FXv9zk4mn47dhQ0h8uieGO0NOp1wEKFLk\\\\\\\/bQP4sMJoTcw0XJL5RB+JEHhoHDzG9TPrgiWRFnpZb4T0n+n5Y1HU4gR+8zIvEJto9EA9Zffrj8z5vAl7CGlgkoUyW3E+LF1U65\\\\\\\/vgdR9BeIKWhf+V40YqD91WJsyJMiqtEs6VfzkjoDPT\\\\\\\/y\\\\\\\/BtZ14cxyMoN3vQr8AUtk5dj5gJCE2lUEnQ\\\\\\\/ydyEA+zdFUPR3ccYic2imcartKk3MoUNBpTj5bkfjVeFuy43Eteg2S1r055a3sEq1uXNFUg9eRC3WufnKJE29ACvn2siiq4rO04lh8cC3Z0EtbRhAlspJpyaKuqtY\\\\\\\/YfKhxfU\\\\\\\/mdARBod3Fs\\\\\\\/HgrgA6KzASRCpSskxAXJL7Z8Cr+bXfwhd3C0gXoXGi00POlqR6d29kkb9weuTeOM+79sJRw+jPedzxIcAMH32AAeOFljxpZo0T1FnT5Vcf96u6v7HeBVMnWhsIYvDE1Tql2stEQathgfLT91QnZgHoPzJ5oldHXwbyfJpiI5C\\\\\\\/8bUJpKXGnIV4lYIwo3zJGXvfv5Dd1+aWgXi25gvFec8DZ\\\\\\\/9+5nY+UIwd\\\\\\\/DwbDYgExtvM4jQKCwIr24wxRRZKKGEmTx\\\\\\\/\\\\\\\/xiwiNF7PCn4KrsdtHfGaY1ROZ8riPvW1dv61l90vCDK2S6krutJUu\\\\\\\/VWTV74A6QO9v9M9cZfIWPQrl7ddDSUTGlJBInUdxhCFmOO2buMFufQ7OeTRJLgP+BTIyJbYXV9PH1qeYQ6YxDYh6Xgdroif6leIjQml2UMXTvfEO9IAQ2gelFd+Ce9XF+x9cvdAcvZZpPf7yFoqoCXmfXE0DOfiTAwxvA9Km4h7mzWwnn2NA7eo+UPdVFqOTvx5UAeUUDL9guF1w93bUuquQ22v9JsKcVWDe+f0OLzaB1jJl5sMktA4GJtqhce9RvzwcNAHhupBQlhaua2YFWJ+tXQvxadlufVbYtw0nD47dBhLjLjINT14MluD0BSASZOcNwvpez9Ja\\\\\\\/cQxS6sB\\\\\\\/UXKNO5H\\\\\\\/2Y35rrTIUFCux\\\\\\\/K735aP9OFb77wcBAPQT32jamvKtZNPbJbYZCnwFwIHNq0nabzQWm4Kylqa7moddtDN7722EYiMzkC6YtPv5uKQem4vxbVKtPEEaQphkSmAHvaiHIkg\\\\\\\/ts2cyiwiXsOk3H8M8yeYyUoaBlQ0EEBkaSGLVnn0SYn8WBEfGhMjJDhstoHi9IjzPqlY2M7T+ZTairdduXYcXI0XSCqL6tKxa0nRjq5JUV64B1JtAru8Zj0kFWapV+sh4O8Kfa+pRti1fvspS6s27\\\\\\\/gawzANNlqzvIk31wiQZ5cqAUNUNfVHXmq6iAxc13wsa5McEOv0O0Gzgs0\\\\\\\/i010srg1UHdy4+3uAOXkIT6PdTdTKK0JSiJAUohy1C+OA8Sp9domTWbpZxgljSLmpKncmKLFNG9fOn3jgzVQQMt2e0BQk\\\\\\\/MMifWu7TGHmOTWs10bgyddLPGmflqkn0dssBwLplS23WCrnZ91zeqfRBAgXtBF7kqWPiffaH0lcNWlHHrg\\\\\\\/e0aeFKGkhcH6tXgR\\\\\\\/ZHhvCquQrnxGOmLfpSS3rpkxvWo4O\\\\\\\/5ZFpAxZqyXarAqvXy7Z3BzXCwAkOiWaVfnx9RL7F9qu+JxNX7zW48O3HT601k34pohEO9b5w9GQGlQs+d5J+dEfJPzhwRrqOo7WIBmqXBkmRE6dJEce3rOTIF0xKcz9QCCYy6JeXvi+q\\\\\\\/l2nicWEV5r0s7rRuRybcakMl4f64jdnFK+JjLWBp6ub2WtH0cfK+RgTG3+xnMfV7JBeR5mBKMw3\\\\\\\/c80pdrnkLQtTu3ot6FN2CUPtyW1s3osWmL6KFA+\\\\\\\/VPUsHFSBCDJPZ0XY7FvbOxtZYFBnGdZnuKbOUm32mofLmT0S+BGzUMdUc83DprfAVs3QGL+yoi24afeF7CWuSHMlEQn66KP14Bnu91XzcDW2POgt0JRdbudl2MuI6R\\\\\\\/SZbLb1VXevsoj146Gd5OTFJlkum3Ftk\\\\\\\/81e27yS108SUHQ6feDpjEFea0l\\\\\\\/UQfzCZ9TQ\\\\\\\/lJxJjD7zqK5GX6f\\\\\\\/6VwUyTEnmMO0PQWK6ssv2SIMxuejDlBYHmq4oU2lyxZlfVFfLPiAmBHQuey3NBb2do3mH110vcPJA2nEp05ZZHmHXDyp9U9W1PfTqPj0OXCx0tCdBQOOKgaoXzd2lpdUMAvqfSSRHQup1SyKF9eCsww78308FVsEeZssGG+ezrnDQZMPnIFMK1ylJLDTpl833kGq8Q5Dk2YhU1WBRjgoq1qomG7WLjFZ\\\\\\\/yinkb9eE\\\\\\\/uGs\\\\\\\/SqmaGbu8wiB0AFxb5TNkUReCDSc7eVeoyOrg\\\\\\\/gHbliQBbCyH8RXYgS98o6fE3ZsAT6jwPLTryiTVQMG91iPZ34CqRgsHSluwUD\\\\\\\/Are129n3TMGCW5m88KlvyVaofeeE3NEYlzR\\\\\\\/hIfX5pAvW7wjIYH6y3QvjDgOuhQEtxgiH7MPaxykn\\\\\\\/Ga120cu81ISn2bxLqo\\\\\\\/3BMPgjtoi\\\\\\\/oEhRBifb7dtBrr6+lhs\\\\\\\/4zWwDR0WdjJ0pVkth0Cgk8SF5n3GElZlvgHW\\\\\\\/VqPkltTYglkwUGdACdiFv69EGbnYbMU4XUE9IVnAhBBnPkINCLBffIfuZRFsI4EHhnyRdDn4pPqxPreqk\\\\\\\/O4llkIex1ScZjjlbHBYftV9+a4VJp9Z6qDlODpwqedKxEUr5UVfezIGYBV0G516UQwK9Hgc3ZvCC5LfTg2ki7qjmzpXpP5LwlWMzSy8TpAuB4EZ2b7xmNja6v282mYiH1cbEmufOjJgBW1gffR7RI3FMgvwEcyXllFACr1Z0dVeM2j6O5IshOLPbQDsfMq0gDIB+j1Hh5WVLa9C7y\\\\\\\/9OkN\\\\\\\/Xq+lU835jfgmQyLKFE2cWDR4dQh7Rvqi9CRasS3UR7g93sZIiYatkr614k3gSTs93HbWVUXNRxfEf89y\\\\\\\/B7IWqeEqOBKlyCyd2q3MHZzjApxnHITqklmXMfjDZWBi99jVPtRN2Ujut5TtO4ZpyTRAub+u00WNN1p6CMeF+TjyHBLrLlQ50cMpLxWJmpgYFWYNcYwl9f8IKnZUTrOnz+S730Lp9laiHf\\\\\\\/CHLHd8tFxvVgUNa+munBkwOtcK7GQUli\\\\\\\/A9acU\\\\\\\/+L5KEN0AfLrtOohCWap3bxf6kdhiaeygXrJt5kxVqqRMBeEiQnS+Yf44NmVgfvY4OREiOupkSUrRnk0bV3jDr3y+GyQ1Vcub\\\\\\\/+l78OkHBzE4ZquBZV3dI7Ug8otsXa3TJgsFL809pA2LN0rLTGs5adKYQMxbMxmTseDV8lK6pQxCHBMnzxBgdMhQFcIs\\\\\\\/vQUpQHE05twxfOAvav3TZwC4VmUvZMvIkC6Vy3OMDr9ubM1yQr0n\\\\\\\/COY991Cx3i\\\\\\\/I2MI3+sAissdujnA6nxPemgByC4fgYyzzakYKxVuJjPpez4H0Ut0h1v+GxUVLwrcnuc0thEd6jLKXGnHXL0Z41Vti4h7Weh4FZd4zxz4Hf5KnM7fVnZRQXs6\\\\\\\/buLMuQU3+pueHXJa5qsKvCPVdXeMz4aWSoo2QwMR5luqB1UFEXeJoYXSMu16WShUruBGhY+NQvKVQkqaHflj7\\\\\\\/FerRPAGjaeDKZdunL1uxZtFl9MWLjd24gZ7SHW3Sh8DOzodkbvaJ41oWFOEOEsrL0TJeTMmcJboxgWFOcju0cHOuh0kp3qMcX8Inqz53pfwg647v1HGh3uGzjbGWkP\\\\\\\/lDSfc9vZk\\\\\\\/hYl6RehLnHoCf7InrWh7qknpYYrMbY2zvJ1RvKcsenFkxHeWvE3DjcMmXhcs4NctCDnbaN3laj9ApgMG2fRyXe1YCvFiSWNGGTJmvL5gKmnlqPuMv6xbxh5GGvYv+F\\\\\\\/4PWrcoJZAWNH9bq4N068DVQiaBR4k4ox5bHrzfquQ6pblEKlpmBcsdd7qpipDM8ppeEAHYY1zEz2p5BiJ0LanAkN2FutCiYcxf4I\\\\\\\/Cw5Fxexwt4pVYpgKw14LE6Uqu0dp0PdGdXh1n6rupEx\\\\\\\/lD0ERW3wjB\\\\\\\/eedYXZVNdJ7aWn+9O0paRF0nq+Y6SPILX\\\\\\\/379ZTWALYcX7iEzOdL5FURcqdVY4p24C\\\\\\\/I3Wz\\\\\\\/BzvkCCEzBFvpNZIoj3wq+6yZo0gPiJzcIPnrXkOJb5M3\\\\\\\/jTTwuSZnsFGTVGMi6g4XaVilEy49cuJ1UP1RY73iRGWORW5S9QLb88pN7VfDjOPfJkWiNBVyE8GlTnEs23RA3pKJqH781UKYKnwm6iPFVd1FSUKE9Afu5QqvkJcLYFXKMwnpMx0aLBr\\\\\\\/vEsdMTy4SWtBCCkSVF7FW+iXqaV\\\\\\\/kPWJ6KBsSSZZmaD96RV6\\\\\\\/EYbGH9AJjcua8QnMCibvdNeT9JK3trOJNCHYczHXfbxKyjiSeMrX6HtRZDMl7FFgmc\\\\\\\/5LUHHS4nXvmqlULNbiP1CEi6TdzAEuY\\\\\\\/g84CfzYWfKuFtPotm+fhoSavZrVEEpJllGTfAyeeTluhgxZcCiFi1TFhycXnig\\\\\\\/Eb74LtnN7YwvculYs1nlG7i6WVgEQ7qJqcEshHbtE39CrAXHunDwnhCxt4igFLCxiJ8G2NkH5w0yjeArQtkLL8lf5hHtE6F2skQgLETZl1ofZBd\\\\\\\/50ZWxFTcNCA2YbUK7GmPeHDqLfzk2DQdJIETUGUdYeLEVBzXVUGnpA25jNdXBqoTdb6hGAvzH0wjDFZ7ZtA4eV94UEOKrcchPDQUeqofuDinr8DB5xtPNpDLlr7UASMtW8VO6lthIxkZsoWOUzLvHA5xceT\\\\\\\/1QhLchU+fxC+sf7pLDyU8F1OuQcrGCadz6V8l\\\\\\\/ASKqRFQrVrcicvnJ\\\\\\\/X6eadqeY9SZHQjKDOAlyi9mseluPhgUQ6IItc+1Ki2ZTB6HNjovnYJyTKXvmVsU61zhWDFygLoWe4erqEphMOUn+xZiwR7n+qlLjLHZgw0QEE4+iA5sfzgXj+ZpMZqi9Rw8cw5ksao9vS7VLB6gMkamCKcSgDEY2Vne4pDkNYWxZrHBtkiLrm0\\\\\\\/Xr8zOjW6r0a2F4HCO2w2pDSagQ57VufL\\\\\\\/wfK51YYdyNZGi\\\\\\\/3HDO4\\\\\\\/R6XBWswCzoJ5ySC1vfKq7UbUQ\\\\\\\/K0Kdin+4eTx7FQj4dpR7NhqPOpRk3U4fiuWkoZR1NzJUK084YczCmZ9WNal5LBXDpQabI\\\\\\\/TYX9yxKxeXr2BHFRbhvMyyijO3YX94hYTO\\\\\\\/\\\\\\\/UWcLlpKp9Kq986z0275N1zaVhK2I1xHTeKZKWg53YPBPA8jdqxJsQ9p5AD9dn0tyzgScmfuL3ChLkJ97S2kVRQeLbTV0QqyUlKo4dz+PomDidOt4TIMzbFThDJyWchUaCxclqLCyYRpQu49RanMz8ZKNeDBSU\\\\\\\/C1ocfVqoJS0VKgHQBOhv2Cai8vfPnujKwEHXn\\\\\\\/hmFN5UEKfWul7UVFylgi4m1EXgZZ2WkQyzdMzqS1H1urtDJLi1e3jQz6ZpYNy4wKqDjYa54p67h1HXZFerrAvH66MYMLoxozwW\\\\\\\/spyfF56TbyoFmbGSVvmuJItbbWdS9Lv33vkaEb3dZqosfoC43M\\\\\\\/hZbpEtiiwcyw5pTACUsr6Kb6H69Y7fpUXY\\\\\\\/00ARPTE6lKpOE5tBdl4UYfl\\\\\\\/77iE0e1XqPFH4i5uG0e1cYwevsbiTiWWjHTAxNsW9MNgHxPctXKPzY+Pd\\\\\\\/lRLcsyenMhWH2W\\\\\\\/HAjrYV8ScZQIjrxfrA8DQEiOMZ1PbFz3RwA\\\\\\\/YJDW6qR8qbBW7R7n7W8DJ\\\\\\\/hAfPGbF5ky4rjyz\\\\\\\/iBjS5Vrwroj\\\\\\\/qO+8AKIaU\\\\\\\/xl4he6fMy1BK3+mCroOmXR+pSbxehjYWVq4PFtyKNPH\\\\\\\/SetiwvsvmqkP\\\\\\\/l88weNAr2LXuDzrS7aWE274NFe\\\\\\\/gv93CpOLWTVNkXLQ\\\\\\\/pqOb+XCg4fXBYgIpzL4n5NJFG1DGGvUnTH7KYiA+bKynvIjMWlpDO0PAGQlvKUgvTJ9Jys7yhsXjFlXRlQQ51v46pUgrYGwR+s2gvqKju2FDNZTyAm23\\\\\\\/pGIxbVpIXlKf2ppv00hX238+2YKeyA90Lsi1KgvulAfeLajp+vU5T\\\\\\\/q78i0G1FwkcaE6m0kVSAOcBpEjbaRfbieFf2RrPFYUeSWlJKklv+2mAS205FnTQ7vXMdwPJKR2U8H2e1QZYbuLVrNLaaXLQMYDf88UGP2wB0GguakMpiswe1WKz1NxkkBdHt6XMngHmDw48p6dJYXXbZztIHwGDADmPVsBJofx74ysB\\\\\\\/APnWVgEI3muo\\\\\\\/U0jUBhb3H5ejWlGwr2xdY8Ey9CG2X4qAa2Uizn8E1gU3IgVbXyyzc\\\\\\\/Y1rCHnjafPt34Sfz+ZGAHAWSMcZ84atot4gCg50zBgqZeYvW+U3cgIU97Fod2TxyBnXTVAoGtri9kvIaBrw\\\\\\\/mUxdhPyk\\\\\\\/b+fd4zFMwzi43SZD0\\\\\\\/3g9Xn+aonmRorQTh8Dq3oCa\\\\\\\/a00Z529UDzRYnBTowlV44TmDvYxc5JLIFAwmn7hJBiN2L94VbA7s5kjUGfZt1bU7g8aCZp27v8yupA6r\\\\\\\/uS5LtL6mbj6PQ33sMA7C2hY\\\\\\\/mK+30WsrAC0vKTaRKCfsWO7CLMXs0jted1GiDxRIzcRT6qVqlDkwAoT+rocL5gbvvKBv7KUx0nGqeZoDsK5FWRYyLveNPwVrQ9MFC81PWm3Ah0SAnzHNfRK7yemLktpWV56usHiJ\\\\\\\/eaH6IvDPAeU4RNo\\\\\\\/CgPe4k6ii1WMD\\\\\\\/pBrLUzFgjHi1hDojqMQWvzMli08vY2z4HO5gRaTjOTYaVa89Dis8YS5bLwCY+KXmN72BFWYh3RBFxOaMjar18yoH1xfXByLIMQWUwLttv8RW9Q69ofUcwh6I7nbOLPYV3HH2L4S7bcTv\\\\\\\/rjOenNfFaN+AWlviQYdqaDWvNJ9uFcQGQzMUxDSUHb3BcP86WS6BPGtgmUhMUcGOLlE+u7OjsUMEobRNBnzBgYTFwhfZJTIa2ROgpVidXbhBFPM8EEE+I6ptOqtq+3KbXgPdLLn8FJ7VmUWcWQ12OdY+naiYifo9WYvUEH8RWh+jKmx9JldZQdcKxOLEYsydqfaBA6OnSrQeptiWWHtUssFDZoURvx963qs5z2cWnFC+fNIF\\\\\\\/zK8zzLtmGhkj5uqjRVH4PQKOCfsz1puosPlVUv1VQ3jiJb9K47XkedxwU2Qf4Id3YnG4OG+8WtAfTZ72SQ+7sPH32Aa\\\\\\\/Q4A15M4OWvLs167rHWTA64gfaiGcK5umEA0t2nYP0miPvfKVQ7u7qXhVFbHOdQ4DwwUfkkcMpFVEEENiuAkQ3iFsAGjHkA8zWZ0oROXJ0VzsgmesBt6LzSWTlfnrwufCmBYPRWCWZeXQ8PysJrbvVnULqxUaQexSPpWLSYY4U1GH516KQS6LCaPGwaskrSmemhxX4RBCV0JfWQ0dHuJ\\\\\\\/Rk2QwNE1a8enveS117nFoXGdTVrVnRfcT08euNDmGcouatZQSqFeYrm5eD3MsXTi6eEjMz8UAS9YNOtbVvnA3ZiDjEGIP0rSAlEeDoedRn6EFtRjbiEl7Aj5U8g9h4Tczvrr6FnDKgRwpnSmHT1YeJwdgQk1qmEh6ZZa1GcS0ksRdUrq8yKtJgAL0AUdc8tU7t7+LPpa24LfZGllZJBXvpdloShrLB4cbFl2oYRa95FZEJYa3kB7x3umexeAfWfTdZ1rKGq5f6V6FrlNlBva0oFgcAjxZIzZ1dLkZ8kw9DuXd\\\\\\\/1LJlBrt8xKsS7lgKwlaDn6eGYLhbmqz5etSiAhdaByg++\\\\\\\/24ZjG6eyQ7xKnk0Y6e8YWIF7XSv3RzixE71fNYlHOnxjs9C+\\\\\\\/bLTtC8w5MBZaPejzP2kOnHWQKnf6ew9Y6\\\\\\\/Pe+zUicRtxPJf56FiX5Y+HzhbOYdoW5G7BHZ5dJ1x\\\\\\\/AnGPiAmZvO4qa49bZLILnL2XQjB4Wf9oqjZuGFhea0Ey25nYb2d\\\\\\\/WKIB0ROfZlOzdOhZeqOe\\\\\\\/YDUrGPMoj1hwE7GNFp3b5W+I5cENOLfJX004bu5dDI4zVl7IUAxkFIgwFY7N4zkQpO\\\\\\\/1x3pdae1OJyv8ExuVD95aU2FkN\\\\\\\/s1luJw0SeqJZm9X1Bwv5UC30E25Svdqo4+Cys33I\\\\\\\/qtiSGAO8izXVut2tszbM982um5FzvgAvhYka+ndMLsvbOrPOU+lNk07rp8ERUOP6LKcgRUnsBoubDKH5VKgqWkZQPA5tGIfEZo8gl6v0e1SyZ9uXuUgW91ywEGJAQAb+asNVjTt3iUABLnfTxHY1TSPWJFsztaWTDMgLE\\\\\\\/KCE8LMPbMAtqcs7gAWi9YRZw\\\\\\\/YYfTmjcxN90A8Zu\\\\\\\/6HZ8CRqSdjgToqfeDJd6jWALmNhEHpXVkl+kkLpZe79Zb8AytUoMxRo7IfWn4e1tkphueeTsHirnn\\\\\\\/QAHnK0jEmaVRDmReyDsqg406GYfq8gKmcMC4qV+F3UgyaMmNNrnMXzLoyrVebyuqmUa6BGIvL6HWh0+rebmkZRRB3rfhQT1MbtZ+1cD3jF7s1FX5gGbxzGRO8Sbhk5pJwbpY6nMffuMAyVvka7fPPR77dn5TiX9u0x9+RrhanzlOaMbprV9W+KZzSWfzPh01ICSCx+aMpWrvn0tO2PXQvCVkI+l49plQ9NIJCGrAxsDGT1asZMZQUSXzMDHhftxya3wK7iho\\\\\\\/WKOrgiOhtB7QAs5yDgBufE6LBmfXzAm0tNvG0IkrA+K6x1Xlng2HxGs9F4ZDvNjtEwE23eOYgiSCzlYbtEoOU3aiNEuXclLfnJpiCPVQzOXhYeHoRHiZ7hZRLi4RiY352LFiQKYYrP2bQimtruhcAaa6vXFaNOoMMBy7unoolWcChFOsshvqgm\\\\\\\/WWnjan983lxjD1sg5glyKKna+iz8pmxZ2psYs3vJvvbzdp+EYA0UVjLOoesjhfn7fqUGD17eP3yQvOblH4Oyb2cbxTEm+v\\\\\\\/YojZkzOGWehwVWLbHo\\\\\\\/\\\\\\\/a5QmafKbLqoHGEXnsV+3aMJe+7xzxsyut5GPuLknNo6UmuH7oXYFSlrpNxKR0VhNpKm06CI53ag9556JMzJ1Xr8ifYbmToJiigvQPcDZFWIBHaMIWWbd5g6Q7Q+\\\\\\\/krunw6TuRdZPHFjZkVAVDm6u0Or52McDUIZTSx\\\\\\\/fKM4BvEaMAbQ9EiSYxwNaneVsvxBCN1PxkhYZbXpdkWIOe09uktZ2MOopzd5lnqnhHmFT36AqZe96x1ocQCf\\\\\\\/\\\\\\\/Dpxv8LGFpoeBT+DVvWaDSdFlr\\\\\\\/rGwiDr3o9aQcOzWZC34ow+7axemypbXKUxWyjxeaWVyvuTqOnWWnIPtTvbxT7upIbcdou+OMKo+TL6IhGhSyw1E7qJs32qItAc5r3DUSZgAeuXzFBDXb\\\\\\\/y3svsHdY0k9avxrRNBsmeyE62rcc9BleDfc+ewdyZwP2ZeFP5HMBc39fRchWMTi120YxrM3ihnQvRqw8k4Tz7YcQ+O8s8CNQQq9Jt2RDspSxhWu1tMPPXZF88\\\\\\\/agobULCjro38a8+XeP58ahgTdjK+bFuH+SHSg6+3f2hWp\\\\\\\/I4GGJxyJiyb0ATIMvUWS9XWmTeJUHK2SnKCjIHAratx+TuwXnANOXQRfDGXfTApSHHoJKrVGqTU3pdGyq7ElgjA04q96omNQ+mPerObc1lf8rBK501PxALSzNOhI4A7tVTXxHf1mSW+gMp6BBGJKYw5lLf28JfsLSD7GM29vkMZ6hjXJwSF5zKyCg4PIJ4k7GM5AO2i2URwrzNmflJsqEEp3cX47fO8CRuTqY+tUJH5sssBp5PNjYyLgmMSo+R97ed0lP2Yo832m4A+C2j44OdcZBU8PKvMJFUtDdZo2Dlh\\\\\\\/Z7Ut4JULiWG5NcYUDyw7f89\\\\\\\/Ah++7V60ybCBPB711DwZRE6KpbP2GHTlAE8F2F0uYOoAgydwiG8bgv\\\\\\\/m41L3Kp\\\\\\\/lpPCf6QpcRfU+HtS7atRa6KzEkiR+DjSin9ui28DVRKoFnPQHq5GZ+mIFdrJMFp50FwQ1vklGUnBDDViTKTYq1fiVgSffNGsewqTe3mzlzAShsn4mXTmLVckSC94DK8Az5\\\\\\\/geGjLpuJw48Pj9nvjUGjwB5+8yIIGyB2qF8muXPIl286BRDvJn+hf3n97\\\\\\\/Oln+BhDnRlXXMsTZ5o\\\\\\\/OSqswV2x90yGVff9r3QfSfBIqtb3clBHn9fxmXVpkjVY51oUWDWH3HSPekGFPqTEpoe3k4DcT1iAx3FFX8hGTIOsdv\\\\\\\/fzymtzKvOH9bDA15BR\\\\\\\/0sVkQ\\\\\\\/khRrGGi1w3hqW\\\\\\\/Ar+zeSioeqoRsW\\\\\\\/dP84VJ3Odi2lBS4txz8z0Cbu\\\\\\\/4IoeQC99Ia0MygtmwmFn4Q72yZVROqVRBnVe0vH2ZBbJ6ZUT5KV+1QURIxZ\\\\\\\/CL3TXgsvg0FTfKDiGUc6Gx33LNd2P4arQJ8WeZ4sI1rBy51VEYhAh+s3MS6EWdp6\\\\\\\/3ersmBFoNCsFcsYrrno03qFu11CUa7GyvW0A\\\\\\\/uHLrJS01xybC7QMXCPZHH7eaZRcyaNhtYGQlJyeQdZAzqtJpsRouIT\\\\\\\/rK6EG2ZiLA7g0TEGT81J6GO8eEps1sohAPu3elIDFu\\\\\\\/BDGLH01jgziVJdDyrfgbtkluC4rHGa5dqw9c3bhl5xQJdTov4WY3lewmcjGYGvOB2VL2Kt3nc\\\\\\\/+1Zlc+gDYrLEEYzPkZ3UNxhD4aZdciOnYsFATQgxOv14lHRnkHhgSo3+1hM+Iy9a1msKHArHa\\\\\\\/hrlo7h2MHABNNC2uAQgXXSIzeVl8Nsst81HWaxDPK23HuHS\\\\\\\/hwYJBwpsyVtWfTknXR\\\\\\\/OIgFo1zdAzUmeyfhf75BSo6k0NqjmokCfTphKdWB+0HJZ54l9ge7QdO8a3gNTevq9e0uZ6i8ninLWS5sppjwtUPy3P\\\\\\\/nFyylbAPuuU1MoMXUVlwKdUNWcgz67WdEZUogKinfYYSpDU5WWQabDThr8IvCi5ofQc9S+IK7\\\\\\\/p1WFEu0uhK9DiqFykMJSl5MRbh3i9qTs4ilN49uAkfC8ZWmCEpjB6pCOc+w+MXrufw5ffU42RTgfVaYEOb5Sg9pcd1F9XIEuQF5pP4lvtEhZji5u2bLCzL2f67r1qKZYZOioiOvE6FcYVt2jBb2QZgR+9ocmELzxFR+LLt6KaIWtbs3RG8TpjlCd+IBQtzPHV2kVsb36pLA3ZthAgB+w2KZdR2M4I+hQ6PCQg==\\\u0022,\\\u0022iv\\\u0022:\\\u0022\\\\\\\/DgJj6v4VGZbDjrh0Hfdmg==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u00227ffed4aced672526c2d1fa44e4e2c7726655182cdbd50e54691b267d2ebdc8ce\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u00228o\\\\\\\/gcgVYiYL5tsZcXAtg7ZT8E55iJxKNX7bOEF5plZwleevBm8UdD4uaCZqOtnZbJHC2onaODwcnYoZHPVVNA8TbHXzBxnuluVPQED0\\\\\\\/L4C7pOwGrW4sodKVFcJqDbhz1S8bMy3vw8d9FogyxKLIstbkrp56QHhkYALgoZVGR3ZV1wv+eUYEum356nuZXD+8Tkdq8Vw1XsLF4WuPAgzK4moen4yXmpMckth6gmZxz2oBFZjyqj+QiM+RyT2ORRrL092AXjzhmUKq1muxJZg1H0UXSk1KY2bg4jJuMP43Gbtt55ZUk2\\\\\\\/aCIcoDB2BtDWOggN\\\\\\\/vn6BxjoTFL0EfERCEMozOrxXeYHvCurp6h3IZqYA1Yvf\\\\\\\/gLPoWp714WtSdwF7QW4y6\\\\\\\/F7fV\\\\\\\/32REoTCl8HDPxfJ+F1dobBxVinY70Uu5b+I6eW4bw2Z25mXDrk33mtj0d42er\\\\\\\/1NC01i0vVSu7WJYMkfXLgetgUxKydPzvgjC6CxV7cDYirLspXMAgmVN3TCzeSI4veIPNQIVkxS\\\\\\\/KYGx1PrN2YkI30CvP6Hlk7D1\\\\\\\/tRdteuPEXZ1vvEyM2tsiD1uRmhyAhoSU7+ZGHDpsm9yTEMu+t8Q4TaCtCqmDZOAGLwAOEV5Q8H5eKmpti1Y7WkjaH2Kfy91xCROyxNNkj\\\\\\\/Hau6KefSxkVm8r84+kqCWF3vV+tujsmOWpdllxb22z1JJWLKX5uKN18YjDX1FSYP711b9rwwjM9sw9FjQBKWsPg3PtFhUmS31cFWFVKS0d75rHOMBhhesNo\\\\\\\/UsrYdmd56079frSuPyRfcO3gFPeqS6iN05mveM5\\\\\\\/S38N0RI86ip5JBVj9Qk44uYILWb7MKq81nqnqv7XtEfC2y28V2S36ZT9535jro+CgnQ8jDBDPjhU7AqWP5P+Xx0BqK17flsoGI06QcYmX51HkcUmeJ9anZiTvZJcHTdNHKiFUXIjweB083UR9i93MqL7y2wbxkEWOdWM5GXlsvJWcI+lxCPassALXelqYzf38ukMLeyZIGJt5Z643bcLXKmu2dsHNiHTrjTsiUjSeroFc7pCvvD7spDuaZUhliKqjfn5vM8ZLrK9ZpJ\\\\\\\/lbJ1\\\\\\\/g6NSzKYbC4a\\\\\\\/WQDtVjQgey5W2YkEJyxjF3wY5wRlc3aryOajOeWBEkvUsTEgD3qswptfjTHrg8eLDdzLoMgmzBnkmeEjoKTS+cBd7160WumbPd9KYcJb\\\\\\\/pcHpF3Bsawx98u52PUW5iMIvwxadl8pcZvwN3CMO\\\\\\\/ppUhoTWhNqmVhcKw8v8e28X2obLdf35rn8yhB2oQZtDCfGtNGEvx70eodN66RJkOvoGF2rPTemwwsb\\\\\\\/AWS2QzV0Ad1fWgpCyy8xweCwoWd7WWbTEYXRiEkY0M+FRIYa5wxp15rWJbbDrRmg8lT9iueaZVkhadJX++n56ZriTIjGKB11nNpqZ2IEhzSWv5RierEAi1wGaJMjKKAhfBzLNSSPUaNjqYTOvbvlLHgHxozJix8GhpYt\\\\\\\/dNwUKZ0Cz11UrHHItBWk\\\\\\\/nXJwYGQBsbVsTA1pJfxLvathx8seD63pYumqcwZm5NBJFrJqrS8aoY8RK5m9bOPMk3zh3zN9PFzKXNZYQEKyTBFdUWZLQoXkgARb9GJUqD2TThkHCJbsc\\\\\\\/\\\\\\\/sel17WHfZmC1DTBME+ZV\\\\\\\/tnK9pVPLla1EW6tv4YirSDzM83tvRVQT7\\\\\\\/qn7ky6\\\\\\\/dLRtCSuQ6OK2nC6sFUybYnYeLatmMZun7oHDXpQubeWF7HEI+nXLnbwfCoqO+nMhVEjpFvbLD7b\\\\\\\/ScPxrQ7eRzmNFXcjEIUDRm4bRjqN\\\\\\\/pawOUOwh0hHXQCm7Tv\\\\\\\/Rfs1PToBrlvGDqSzHG3YFGkAlbHCK\\\\\\\/S38\\\\\\\/4i9q3b+rqnW7tDLw0IADFu7\\\\\\\/Z4ZtlNpIRiz0Q4L7pEzSOUZs8J4kAczSzjCIIEj5J1PxA+TctrdDHuHW0E82kd51Y5AZz8EqdHbpuqzC60yo6P2ep6dYa4ibheo9aNDe\\\\\\\/RB+lOmMdQvG\\\\\\\/nw==\\\u0022,\\\u0022iv\\\u0022:\\\u0022Q6HOdy\\\\\\\/BeF8LPuPoKfgGJw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002242447876f0913a84c168b26fa4f922b9d7f86c2a3531917efba41069a8e1f573\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460332 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/10/3035" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-1pNsMhqNOF1YbSgpNCI8erwM0qjuBEvDQzJ2OjLDz8o" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/10/3035" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-tCZTEFAfv75Rd2qwLBDkXbywkg7UW_ZZiYVx4HyB_ew" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/10/3027" title="Tandem Duplication Events in the Expansion of the Small Heat Shock Protein Gene Family in Solanum lycopersicum (cv. Heinz 1706)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/10/3049" title="Characterization of a Novel MMS-Sensitive Allele of Schizosaccharomyces pombe mcm4+" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460332" id="node460332" data-pisa="ggg;6/10/3035" data-pisa-master="ggg;g3.116.031120" data-apath="/ggg/6/10/3035.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Analysis of Polyadenylation Events in <em>Schmidtea mediterranea</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9468-9571" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Vairavan Lakshmanan</span>, <span class="highwire-citation-author" data-delta="1">Dhiru Bansal</span>, <span class="highwire-citation-author" data-delta="2">Jahnavi Kulkarni</span>, <span class="highwire-citation-author" data-delta="3">Deepak Poduval</span>, <span class="highwire-citation-author" data-delta="4">Srikar Krishna</span>, <span class="highwire-citation-author" data-delta="5">Vidyanand Sasidharan</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-2219-4785" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Praveen Anand</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0001-7028-1803" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aswin Seshasayee</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0003-2219-4785" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dasaradhi Palakodeti</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3035-3048;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.031120 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460332" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Vairavan Lakshmanan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Stem Cell Biology and Regenerative Medicine, NCBS campus, GKVK, Bellary Road, Bangalore 560065, Karnataka, India</div><div class='author-affiliation'><span class='nlm-sup'>†</span>SASTRA University, Thirumalaisamudram, Thanjavur 613401, Tamil Nadu, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vairavan%2BLakshmanan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lakshmanan%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVairavan%2BLakshmanan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9468-9571" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vairavan Lakshmanan</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dhiru Bansal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Stem Cell Biology and Regenerative Medicine, NCBS campus, GKVK, Bellary Road, Bangalore 560065, Karnataka, India</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Manipal University, Madhav Nagar, Near Tiger Circle, Manipal 576104, Karnataka, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dhiru%2BBansal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bansal%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADhiru%2BBansal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jahnavi Kulkarni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Stem Cell Biology and Regenerative Medicine, NCBS campus, GKVK, Bellary Road, Bangalore 560065, Karnataka, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jahnavi%2BKulkarni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kulkarni%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJahnavi%2BKulkarni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Deepak Poduval </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>University of Bergen, Department of Clinical Science, Bergen 5020, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Deepak%2BPoduval%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poduval%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADeepak%2BPoduval%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Srikar Krishna </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Stem Cell Biology and Regenerative Medicine, NCBS campus, GKVK, Bellary Road, Bangalore 560065, Karnataka, India</div><div class='author-affiliation'><span class='nlm-sup'>†</span>SASTRA University, Thirumalaisamudram, Thanjavur 613401, Tamil Nadu, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Srikar%2BKrishna%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Krishna%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASrikar%2BKrishna%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Vidyanand Sasidharan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Stem Cell Biology and Regenerative Medicine, NCBS campus, GKVK, Bellary Road, Bangalore 560065, Karnataka, India</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Manipal University, Madhav Nagar, Near Tiger Circle, Manipal 576104, Karnataka, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vidyanand%2BSasidharan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sasidharan%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVidyanand%2BSasidharan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Praveen Anand </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Stem Cell Biology and Regenerative Medicine, NCBS campus, GKVK, Bellary Road, Bangalore 560065, Karnataka, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Praveen%2BAnand%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Anand%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APraveen%2BAnand%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2219-4785" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Praveen Anand</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dasaradhip@instem.res.in" class="" data-icon-position="" data-hide-link-title="0">dasaradhip@instem.res.in</a></span></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Aswin Seshasayee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>National Centre for Biological Sciences, GKVK Campus, Bangalore 560065, Karnataka, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aswin%2BSeshasayee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Seshasayee%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAswin%2BSeshasayee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7028-1803" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aswin Seshasayee</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dasaradhip@instem.res.in" class="" data-icon-position="" data-hide-link-title="0">dasaradhip@instem.res.in</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Dasaradhi Palakodeti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Stem Cell Biology and Regenerative Medicine, NCBS campus, GKVK, Bellary Road, Bangalore 560065, Karnataka, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dasaradhi%2BPalakodeti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palakodeti%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADasaradhi%2BPalakodeti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2219-4785" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dasaradhi Palakodeti</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dasaradhip@instem.res.in" class="" data-icon-position="" data-hide-link-title="0">dasaradhip@instem.res.in</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/10/3035" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460332" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460332/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/10/3035.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460332" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460332/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/10/3035.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460332" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460332/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/10/3035.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460332" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460332/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/10/3035.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460332" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460332/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-3">In eukaryotes, 3′ untranslated regions (UTRs) play important roles in regulating posttranscriptional gene expression. The 3′UTR is defined by regulated cleavage/polyadenylation of the pre-mRNA. The advent of next-generation sequencing technology has now enabled us to identify these events on a genome-wide scale. In this study, we used poly(A)-position profiling by sequencing (3P-Seq) to capture all poly(A) sites across the genome of the freshwater planarian, <em>Schmidtea mediterranea</em>, an ideal model system for exploring the process of regeneration and stem cell function. We identified the 3′UTRs for ∼14,000 transcripts and thus improved the existing gene annotations. We found 97 transcripts, which are polyadenylated within an internal exon, resulting in the shrinking of the ORF and loss of a predicted protein domain. Around 40% of the transcripts in planaria were alternatively polyadenylated (ApA), resulting either in an altered 3′UTR or a change in coding sequence. We identified specific ApA transcript isoforms that were subjected to miRNA mediated gene regulation using degradome sequencing. In this study, we also confirmed a tissue-specific expression pattern for alternate polyadenylated transcripts. The insights from this study highlight the potential role of ApA in regulating the gene expression essential for planarian regeneration.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/posttranscriptional-regulation" class="hw-term hw-article-keyword hw-article-keyword-posttranscriptional-regulation" rel="nofollow">posttranscriptional regulation</a></li><li class="kwd"><a href="/keyword/polyadenylation" class="hw-term hw-article-keyword hw-article-keyword-polyadenylation" rel="nofollow">polyadenylation</a></li><li class="kwd"><a href="/keyword/alternate-polyadenylation" class="hw-term hw-article-keyword hw-article-keyword-alternate-polyadenylation" rel="nofollow">alternate polyadenylation</a></li><li class="kwd"><a href="/keyword/3p-seq-polya-position-profiling" class="hw-term hw-article-keyword hw-article-keyword-3p-seq-polya-position-profiling" rel="nofollow">3P-Seq (poly(A)-position profiling)</a></li><li class="kwd"><a href="/keyword/planaria" class="hw-term hw-article-keyword hw-article-keyword-planaria" rel="nofollow">planaria</a></li></ul><p id="p-4">The 3′ untranslated region (3′UTR) governs the stability, localization, and translational efficiency of mRNA within the cell (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Barrett <em>et al.</em> 2012</a>; <a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Jia <em>et al.</em> 2013</a>). Various <em>cis</em>-regulatory elements present within the 3′UTR bind to <em>trans</em>-acting RNA-binding proteins (RBPs) and small noncoding RNAs to regulate eukaryotic gene expression (<a id="xref-ref-29-2" class="xref-bibr" href="#ref-29">Jia <em>et al.</em> 2013</a>; <a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Millevoi and Vagner 2010</a>). Hence, 3′UTR processing can have a direct influence on posttranscriptional gene regulation. The generation of a defined 3′UTR involves the cleavage/polyadenylation of pre-mRNA resulting in a stretch of untemplated polyadenosine monophosphate residues [poly(A) tail] of varying length at the 3′ end (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Colgan and Manley 1997</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Zhao <em>et al.</em> 1999</a>). Almost all transcripts in eukaryotes, excluding histone mRNA (<a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Davila Lopez and Samuelsson 2008</a>), are subjected to this process. Furthermore, cleavage/polyadenylation can occur at multiple sites on the same pre-mRNA, leading to different transcript isoforms; this is called alternative polyadenylation (ApA). Often, these ApA events alter the length of the 3′UTRs. In some scenarios, cleavage/polyadenylation events occur within the ORF of the transcript leading to the production of different protein isoforms. Currently, more than 30–50% of genes in higher eukaryotes are estimated to be alternatively polyadenylated (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Tian <em>et al.</em> 2005</a>; <a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Derti <em>et al.</em> 2012</a>). Moreover, the nature of polyadenylated genes and the site of polyadenylation on the mRNA change with the status of the cell. For example, proliferative cells derived upon T-cell activation express transcripts with shortened 3′UTRs (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Sandberg <em>et al.</em> 2008</a>). Reprogramming of somatic cells to iPS cells also favors transcripts with shorter ApA isoforms (<a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Ji and Tian 2009</a>). Transcripts with longer 3′UTRs are observed during mouse embryonic development, differentiation of monocytes, and in mammalian neurons (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Ji <em>et al.</em> 2009</a>; <a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Shepard <em>et al.</em> 2011</a>). In contrast, the average 3′UTR length was observed to decrease during the developmental stages in <em>Caenorhabditis elegans</em> (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Mangone <em>et al.</em> 2010</a>). Further, ApA was shown to contribute to oncogene activation through the loss of 3′UTR suppressor elements (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Mayr and Bartel 2009</a>). Recently, ApA was also implicated in the regulation of membrane protein localization (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Berkovits and Mayr 2015</a>).</p><p id="p-5">The planarian <em>Schmidtea mediterranea</em> serves as an ideal model system to study stem cell function and regeneration (Oviedo <em>et al.</em> 2008). The unique property of planaria to regenerate any organ, including the brain, is attributed to the presence of adult pluripotent somatic stem cells called neoblasts (<a id="xref-ref-53-1" class="xref-bibr" href="#ref-53">Sanchez Alvarado 2003</a>). Various genes and gene regulatory mechanisms have been implicated in neoblast function (<a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Rossi <em>et al.</em> 2008</a>; <a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Pearson and Sanchez Alvarado 2010</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Onal <em>et al.</em> 2012</a>; <a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Hubert <em>et al.</em> 2013</a>; <a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Sasidharan <em>et al.</em> 2013</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Solana <em>et al.</em> 2013</a>; <a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Zeng <em>et al.</em> 2013</a>). Recent efforts have identified several RNA binding proteins, mostly involved in translation regulation and mRNA stability, that are enriched in neoblasts. These were shown to be essential for stem cell maintenance and differentiation (<a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Rouhana <em>et al.</em> 2014</a>, <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">2010</a>).</p><p id="p-6">At present, the study of posttranscriptional gene regulation in the planarian model system is limited by the lack of well-annotated 3′UTRs. In order to address this issue, we carried out poly(A) position profiling (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Jan <em>et al.</em> 2011</a>) on the transcriptome of <em>S. mediterranea</em>. Using the 31,377 poly(A) sites that were identified, we could reliably annotate the 3′UTRs for 44% of the protein coding genes. Around 40% of the annotated transcripts were associated with more than one poly(A) site, hinting at the predominant role played by ApA in regulating gene expression. We identified around 97 transcripts in planaria that are polyadenylated within their ORFs. Furthermore, we explored the functional association between miRNA-mediated gene regulation and ApA, which revealed that miRNA binding sites are either clustered around the proximal or distal polyadenylation sites. We confirmed tissue-specific expression of 3′UTR isoforms arising from alternate polyadenylation. Thus, the work presented here highlights the potential role played by ApA in planarian regeneration. This study will also help to identify the regulators essential for the ApA mechanism in planarians.</p><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Identification of polyadenylation machinery in planaria</h3><p id="p-7">We used a polyadenylation machinery protein sequence from humans as a query and blasted it against the PlanMine database to determine the corresponding homologous in planaria (<a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Brandl <em>et al.</em> 2015</a>). For 12 of the proteins (CFIm25, CPSF100, CPSF30, CPSF160, CPSF73, CstF50, CstF77, CstF64, ClpI, PAPOA, PAPOB, and PAPOC) the homolog could be easily determined. The corresponding IDs from the Ox_Smed_V1 version of the transcriptome could be identified and the presence of these were experimentally verified through RT-PCR. For the four proteins from the polyadenylation machinery (CFIm59, CFIm68, hFIP1, and Pcf11), the sequence search through the PlanMine database did not result in a reliable hit. To rule out the possibility that these could be missed due to poor genome assembly/annotation, we searched for similar sequences in the recent version of the Smed genome (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Robb <em>et al.</em> 2015</a>). The query through BLAST resulted in hits that had partial similarity to the human homologs. All the orthologous sequences for these proteins were downloaded from the gene tree available at the Ensemble database. The gene tree was reconstructed by including the hits from the Smed genome along with orthologs from other organisms using <em>ete3</em> (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Huerta-Cepas <em>et al.</em> 2010</a>). The percentage identity matrix (PIM) was also constructed through multiple sequence alignment using clustal <em>omega</em> (<a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Sievers <em>et al.</em> 2011</a>). Both the constructed gene tree and the PIM confirmed the sequence divergence of these four proteins in planaria (Supplemental Material, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS1.pdf">Figure S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS2.pdf">Figure S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS3.pdf">Figure S3</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS4.pdf">Figure S4</a>).</p></div><div id="sec-3" class="subsection"><h3>Preparation of 3P-Seq library</h3><p id="p-8">The 3P libraries were prepared as per the standard protocol previously described in the literature (<a id="xref-ref-26-2" class="xref-bibr" href="#ref-26">Jan <em>et al.</em> 2011</a>) with some modifications. The total RNA amounting to 200 μg was extracted using the trizol method from both sexual and asexual planaria, <em>S. mediterranea</em>, to prepare 3P libraries. The extracted RNA was subjected to RNase H treatment for 25 min. The cDNA was prepared from the library using Superscript III RT (Catalog no. 18080044) and the derived product was directly amplified using PCR (25 cycles) without desalting. The adaptor (Illumina genomic sequencing adapter) ligated product was sequenced using an Illumina GAIIx platform.</p></div><div id="sec-4" class="subsection"><h3>Analysis of 3P-Seq reads</h3><p id="p-9">In order to determine the polyadenylation sites in <em>S. mediterranea</em>, poly(A) RNAs were isolated from the whole organism (both sexual and asexual forms separately) and then subjected to 3P-Seq. Around 165 million and 103 million reads were obtained for the sexual and asexual forms, respectively. These reads were combined, processed, and mapped to the SxlV3.1 genome using bowtie (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Langmead <em>et al.</em> 2009</a>). The raw reads obtained from the 3P-Seq protocol (<a id="xref-ref-26-3" class="xref-bibr" href="#ref-26">Jan <em>et al.</em> 2011</a>) were reverse complemented. The reads ending with a poly(A) stretch were trimmed down to contain a maximum of two A’s at the end. These were then selectively mapped onto the genome and the reads were considered as 3P-tags if: a) the alignment ended with the mismatch in the 3′, end as it would signify the untemplated adenosine nucleotide added by poly(A) machinery; and b) the minimum length of read was above 20 nucleotides (nt). Raw numbers are shown in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS5.pdf">Figure S5</a>. The reads were aligned to the <em>S. mediterranea</em> genome using bowtie with –m 4 and –v 2 options. Around 77 million reads aligned to the genome, among which 34 million reads qualified as 3P-tags. The density profile of 3P-tags across the whole genome revealed that 81% of the genomic locus is covered with a minimum depth of three 3P-Seq reads (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS5.pdf">Figure S5</a>). Hence, we used a minimum cut-off of three 3P-tags to identify the 3P-peaks. The 3P-peak has a minimum coverage of three 3P-tags associated across the complete length. This stringent scan across the entire genome involving the above cut-offs, resulted in 31,377 unique polyadenylation sites. The identified cleavage sites have a median length of 50 nt (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS5.pdf">Figure S5</a>) and were highly AT-rich (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS5.pdf">Figure S5</a>). The in-house python scripts used to process the raw FASTQ data can be obtained from github (<a href="https://github.com/VairavanL/3PSeq_analysis">https://github.com/VairavanL/3PSeq_analysis</a>).</p></div><div id="sec-5" class="subsection"><h3>Ion torrent sequencing</h3><p id="p-10">The total RNA, amounting to 7 μg extracted from both the sexual and asexual worms, was subjected to cDNA conversion using OligodT primers from an AffinityScript QPCR cDNA Synthesis Kit. The cDNA was fragmented using Covaris. The fragmented DNA was quantified and subsequently utilized for library preparation. End-repair and adapter ligation was carried out according to the protocol (Ion plus fragment library kit, # 4471252). The samples were bar-coded at this step and then cleaned using Ampure XP beads. The library was amplified using eight cycles of PCR for enrichment of adapter-ligated fragments. The amplified product was again cleaned using Ampure XP beads. The prepared library was quantified using Qubit and validated for quality by running an aliquot on a High Sensitivity Bioanalyzer Chip (Agilent). Ion PGM sequencing was performed on this sample. Around 2.3 million reads were derived from the transcriptome through ion torrent sequencing after poly(A) pulldown. Approximately 70% of these sequenced reads were &gt; 50 nt in length (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS12.pdf">Figure S12</a>). The reads containing poly(A/T) (28.6%) were selected as these would represent the polyadenylated mRNAs. The selected reads were subsequently trimmed to remove poly(A). We used blast (<a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Altschul <em>et al.</em> 1990</a>) and bowtie to align the trimmed reads to the genome. Around 0.37 million reads were successfully aligned and these covered 18,834 of 31,377 3P-peaks.</p></div><div id="sec-6" class="subsection"><h3>Scanning for poly(A) signals (PAS)</h3><p id="p-11">In order to identify a statistically significant hexamer that is enriched across the identified 31,377 peaks, we adopted a pipeline depicted in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS6.pdf">Figure S6</a>. A random dataset was generated by shuffling the 3P-peak sequences using the Knuth-shuffle algorithm. The probability of hexamer occurrences [4096 (<span class="inline-formula" id="inline-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10/3035/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10/3035/embed/mml-math-1.gif"/></noscript></span></span>)] is calculated across all positions in both shuffled and unshuffled sequence datasets. The z-score is calculated from the occurrence value for each position. The z-scores are then converted into <em>P</em>-values using the <em>pnorm</em> function in R and the <em>P</em>-value is adjusted using Bonferroni corrections. A particular signal was considered as an enriched hexamer motif at a specific position if it had an adjusted <em>P</em>-value &lt; 0.01 and at least a twofold enriched occurrence compared to a random sequence. We identified 737 enriched hexamer signals across various positions in the peak. These 737 hexamers are position-dependent enriched motifs. Among these, the canonical signal 5‘-AAUAAA-3’ was present in 14,183 peaks. A randomized dataset of 3P-peaks was generated to test for the significance of these hexamers and the whole pipeline was repeated on this dataset. We did not find any significant hexamer enrichment. This shows the robustness of our methodology. We also looked for pentamer, heptamer, and octamer sequence enrichment in the identified peaks. We did not find any enriched motifs of these sizes.</p><p id="p-12">To identify other signals (minor) apart from 5‘-AAUAAA-3’ that could also act as PAS, we adopted a method as illustrated in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS6.pdf">Figure S6</a>. We performed Fisher’s exact test to identify minor signals. Total peaks were divided into two categories: one that housed the canonical PAS (14,183 peaks) and another that did not contain canonical PAS (17,194 peaks). We considered 736 signals (apart from the canonical signal) that were enriched and performed a two-category association test to determine in which category they were enriched. Based on the association value (<em>i.e.</em>, odds ratio &lt; 0.75 and <em>P</em>-value &lt; 0.01), 82 signals were identified that could likely represent the minor signals. In order to test the robustness of this protocol, the whole analysis was also repeated with different cut-offs of 3P-tags (ranging from four 3P-tags up to 100 3P-tags). Although the number of 3P-peaks reduced with the increase in cut-off, the positional enrichment of the major PAS signal, along with the secondary (<span class="inline-formula" id="inline-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10/3035/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10/3035/embed/mml-math-2.gif"/></noscript></span></span> retained) and the minor signal (<span class="inline-formula" id="inline-formula-3"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10/3035/embed/mml-math-3.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10/3035/embed/mml-math-3.gif"/></noscript></span></span> retained), did not change (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS7.pdf">Figure S7</a>).</p><p id="p-13">A directed search was also performed to identify the PAS associated with each of 31,377 3P-peaks by scanning the sequence within the distance of 10–40 nt from the cleavage site (3′ end of the sequence). The first priority was given to the canonical signal 5‘-AAUAAA-3’, and around 40.8% peaks were found to house this signal within 40 nt from the cleavage site. The second priority was given to all hexameric sequences that had just one mismatch to the canonical signal 5‘-AAUAAA-3’. The one closest to the cleavage site was considered as PAS for the peak. Additionally, of the remaining peaks, around 40.4% could now be associated with PAS containing a variant of the canonical signal with one mismatch. Similarly, the same process was repeated with PAS containing two mismatches. Finally, around 97.5% (30,994) of the polyadenylation sites were assigned with a PAS signal (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS6.pdf">Figure S6</a>).</p></div><div id="sec-7" class="subsection"><h3>Identification of hexameric motifs using MEME</h3><p id="p-14">We used MEME 4.10.2 (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Bailey <em>et al.</em> 2006</a>) to predict PAS along with two other strategies. We used 3′UTR sequences from zebrafish, <em>Drosophila</em>, humans, <em>C. elegans</em>, and mouse as positive background datasets for MEME. We categorized the identified polyadenylation sites into three bins based on its coverage as top, median, and last 1000 sites. When we individually ran MEME on these three datasets, we identified 5‘-AAUAAA-3’ as a major enriched signal. Along with 5‘-AAUAAA-3’, we got other variant signals, which overlapped with 737 signals that we identified before (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS6.pdf">Figure S6</a>).</p></div><div id="sec-8" class="subsection"><h3>Transcript and peak statistics</h3><p id="p-15">There are 43,209 contigs in the smed genome (Version 3.1). In total, 31,067 maker transcripts are spread across 16,053 contigs. Among these, 7219 contigs (9449 maker transcripts) have only maker-annotated transcripts and no identified 3P-peaks falls into these contigs. On the contrary, there are 3793 contigs (5988 3P-peaks) that have only 3P-peaks but no annotated transcripts that fall within them. There are 8834 contigs that have both annotated transcripts and 3P-peaks associated with them. These 8834 contigs constitute 21,577 transcripts and 29,134 3P-peaks. Similarly, in the Ox_Smed_v1 transcriptome, 2432 contigs (3175 transcripts) have only annotated transcripts. 5145 contigs (7916 3P-peaks) have only peaks associated without containing any transcripts. In the dd_Smed_v4 transcriptome, 3888 contigs (6459 transcripts) have only transcripts and 4341 contigs (6566 3P-peaks) have only 3P-peaks coming from them.</p></div><div id="sec-9" class="subsection"><h3>Stitching of transcripts</h3><p id="p-16">We could classify the association of polyadenylation sites to the transcript into five different categories. Polyadenylation sites that falls inside the transcript were excluded from this analysis, except for the ones that are present in the last exon and end after the annotated transcript model. The transcripts that have a 3P-peak outside the annotated end were extended to get the complete gene model by utilizing the genomic information (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS9.pdf">Figure S9</a>). We used four datasets (Genscan, Genpred, Cufflinks from RNA-seq, and the EST database) to fill the gap between known transcript model ends and the polyadenylation site start. At least two of the four datasets should support a region to extend the existing transcript model. If a transcript is associated with more than one polyadenylation site, we considered it as a different 3′UTR isoform and stitched each polyadenylation site individually. For comparison of the 3′UTR lengths across different organisms, we derived the information from biomart (Ensemble Genes 84 version). The longest transcript isoform was chosen for each gene to calculate the 3′UTR length.</p></div><div id="sec-10" class="subsection"><h3>Using RNA-seq data as evidence for stitch and validation of cleavage site</h3><p id="p-17">We considered those transcripts that have one cleavage site and filtered them by genomic length. A cut-off of 1 kb between transcript end and peak start was used. Transcripts that satisfied these two conditions were taken for analysis. We divided these transcripts into two bins. One was the Upstream bin, which had sequences corresponding to the region between the transcript end and peak start. We also took exactly the same genomic length sequence as the Upstream bin after the cleavage site and called it the Downstream bin. We made sure that there was no transcript that started in the Downstream bin. To calculate RNA-seq coverage we used the following formula.<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10/3035/embed/mml-math-4.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10/3035/embed/mml-math-4.gif"/></noscript></span></span>The total numbers of reads are reduced by half as it represents paired-end data. Coverage is normalized to the total length of the transcript.</p></div><div id="sec-11" class="subsection"><h3>Validation of the candidates through RT-PCR</h3><p id="p-18">The forward primers were designed from the region within the annotated transcripts and the reverse primers were designed from the region spanning the derived 3P-peak. Candidates were chosen based on the strength of the 3P-peaks (number of 3P-tags within the 3P-peak region). Candidates having a similar number of 3P-tags on both the proximal and distal sites were selected. As the 3P-Seq was performed from the RNA isolated from the whole organism, this would ensure that both the isoforms of the transcripts are expressed. These were further filtered to ensure that the distance between the proximal and distal peaks was ∼2 kb. 1 μg of RNA was converted to cDNA using SSII RT (Invitrogen). The PCR primers for cDNA were designed as shown in <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4E</a> (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/TableS9.xls">Table S9</a>). The RT-PCR was performed using LA Taq (TAKARA). Amplicons were run on agarose gel and subsequently sequenced.</p></div><div id="sec-12" class="subsection"><h3>Degradome sequencing</h3><p id="p-19">We adopted parallel analysis of RNA ends (PARE), a specific sequencing method, to identify degraded RNA products (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">German <em>et al.</em> 2008</a>; <a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Karginov <em>et al.</em> 2010</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Cass <em>et al.</em> 2016</a>). These degraded RNAs are the by-products of AGO-DROSHA processing. Total RNA was isolated from sexual and asexual strains of <em>S. mediterranea</em>. Around 500 ng of RNA was directly adapter ligated using T4 RNA ligase, to select only for degraded mRNA that would have a free 5′ phosphate. The ligated RNA was then reverse transcribed using oligo dT (with an attached adapter sequence) and PCR amplified using primers from the TruSeq Illumina Small RNA library. The library was purified using Ampure XP beads and sequencing was performed on an Illumina HiSeq-1000. We got 61 and 72 million reads from the sexual and asexual strains, respectively. These reads were adapter trimmed and the first 15 nt of the reads were used for alignment to the genome and transcriptome using bowtie allowing no mismatches. To find miRNA correspondence for each tag, we used the FASTA v36 program using the parameters -n -H -Q -f -16 -r +15/-10 -g -10 -w 100 -W 25 -E 100 -i –U and scored the alignments using a match-mismatch profile in the seed and guide region. The analysis pipeline used here has been published previously in the literature (<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">Moran <em>et al.</em> 2014</a>). In metazoans, the requirement for complementarity between the miRNA and its corresponding target for AGO cleavage is still debatable. The previously published literature hints at the requirement of complementarity at the ninth, 10th, and 11th nucleotides, along with the seed region for AGO-mediated mRNA degradation (<a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Shin <em>et al.</em> 2010</a>; <a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Elkayam <em>et al.</em> 2012</a>; <a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Addo-Quaye <em>et al.</em> 2009</a>). Hence, we selected for sequence complementarity between the degradome tag and the miRNA, even at the ninth, 10th, and 11th positions. This would ensure that the derived degradome tags were a result of AGO cleavage. We then specifically analyzed these degradome tags based upon their location on the proximal or distal 3′UTR region.</p></div><div id="sec-13" class="subsection"><h3>miRNA predictions using miRanda</h3><p id="p-20">All of the transcripts that had two cleavage sites associated with them were chosen for prediction of miRNA targets. Transcripts that had both the cleavage sites within them were not considered further for the analysis. We extracted the sequences corresponding to the proximal and distal UTRs from the transcripts. These sequences were specifically scanned for miRNA binding sites using a tool called miRanda (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">John <em>et al.</em> 2004</a>). We downloaded all the known miRNAs of <em>S. mediterranea</em> from miRbase (<a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Griffiths-Jones <em>et al.</em> 2008</a>). The predicted binding sites from miRanda were filtered based on the complementarity score (score ≥ 140) calculated from an algorithm, as well as the free energy of optimal complementary base pair interaction between the 3′UTR and miRNA sequence (<span class="inline-formula" id="inline-formula-4"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10/3035/embed/mml-math-5.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10/3035/embed/mml-math-5.gif"/></noscript></span></span> G ≤ <span class="inline-formula" id="inline-formula-5"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10/3035/embed/mml-math-6.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10/3035/embed/mml-math-6.gif"/></noscript></span></span>). These filtered binding sites were normalized to the length of the 3′UTR region.</p></div><div id="sec-14" class="subsection"><h3>Control dataset for distribution of miRNA binding sites</h3><p id="p-21">A randomized dataset was generated to determine the statistical significance for bimodal distribution of miRNA binding sites in transcripts with two 3P-peak candidates. The dataset was generated by randomizing the association of a second 3P-peak to single 3P-peak-containing transcripts. Around 1550 (same size as the actual dataset) single 3P-peaks were also randomly chosen 100 times. In order to capture the native distance distribution between two 3P-peaks, only the distal 3P-peak from two 3P-peak candidates were randomly chosen, along with the region between the peaks and stitched to single 3P-peak candidates. We predicted miRNA binding sites and normalized the dataset as described above. We performed Hartigan’s dip test for multi-modality for the randomized 100 datasets to see how many of the randomized datasets followed a multi-modal distribution as an actual dataset. Significant <em>P</em>-values and a higher Dip score suggested multimodal distribution. Randomized datasets with <em>P</em>-values &lt; 0.05 were considered to be multimodal (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS14.pdf">Figure S14</a>).</p></div><div id="sec-15" class="subsection"><h3>Data availability</h3><p id="p-22">The data generated in this study has been deposited in the National Center for Biotechnology Information Sequence read archive (SRA) database. Accession ID: SRP070102. All the NGS data used in this study can also be visualized through JBrowse applet (<a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Westesson <em>et al.</em> 2013</a>) at <a href="http://bugbears.ncbs.res.in/planaria3p">http://bugbears.ncbs.res.in/planaria3p</a>.</p></div></div><div class="section results" id="sec-16"><h2 class="">Results</h2><div id="sec-17" class="subsection"><h3>Identification of polyadenylation machinery in planaria</h3><p id="p-23">The process of polyadenylation involves two major reactions. The first reaction is the endonucleolytic cleavage of pre-mRNA at a specific site. This is followed by the polymerase reaction that adds the polyadenylated tail to the cleaved product. These reactions require various protein factors that are specifically guided to the cleavage site, dictated by the <em>cis</em>-elements present within the pre-mRNA (<a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Millevoi and Vagner 2010</a>; <a id="xref-ref-13-2" class="xref-bibr" href="#ref-13">Colgan and Manley 1997</a>; <a id="xref-ref-70-2" class="xref-bibr" href="#ref-70">Zhao <em>et al.</em> 1999</a>; <a id="xref-ref-14-2" class="xref-bibr" href="#ref-14">Davila Lopez and Samuelsson 2008</a>; <a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Tian <em>et al.</em> 2005</a>; <a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Barabino and Keller 1999</a>). The polyadenylation machinery comprises four multi-protein complexes: CPSF (cleavage and polyadenylation specificity factor), CstF (cleavage stimulation factor), CFI, and CFII (cleavage factor I and II, respectively).</p><p id="p-24">The components of the polyadenylation machinery in planaria were computationally identified based on their sequence similarity to the known components in the human genome. Of the 16 protein subunits that comprise the polyadenylation machinery, 12 had homologs in planaria (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1A</a>). For the four proteins [hFIP1 (CPSF subunit), CFIm 59/68 (components of CFI), and Pcf11 (component of CFIIm)], confident planarian homologs could not be identified (due to low sequence identity to human homologs; <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS1.pdf">Figure S1</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS2.pdf">Figure S2</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS3.pdf">Figure S3</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS4.pdf">Figure S4</a>). The presence of transcripts coding for these polyadenylation machinery components were subsequently confirmed through PCR, followed by sequencing (<a id="xref-fig-1-2" class="xref-fig" href="#F1">Figure 1B</a>). This suggests that the classical eukaryotic mRNA 3′end-processing machinery is well-conserved in planarians.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/10/3035/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Polyadenylation machinery in planaria. (A) A schematic representing all the components of the eukaryotic polyadenylation machinery in planaria. These specifically include multi-subunit protein complexes such as: Cleavage Factor I (CFIm), Cleavage and Polyadenylation Specificity Factor (CPSF), Cleavage stimulation Factor (CstF), Cleavage factor II (CFIIm), and poly(A) polymerases (PAPOA, PAPOB, or PAPOG). The different protein subunits for each of these complexes are colored based upon the negative E-value exponent returned by BLAST-hit against its human homolog. The components for which the homolog could not be determined confidently in planaria (hFip1, CFIm-59/68, and Pcf11) are marked with dotted lines. The corresponding cis-elements recognized by these protein complexes are also shown in the schematic. (B) RT-PCR products showing the expression of the transcripts coding for proteins of the polyadenylation machinery, transcript ID, their corresponding human homolog, percentage sequence identity, and coverage (derived from blast2seq using BLOSUM62 matrix). ID, identity; PCR, polymerase chain reaction." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-884262096" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Polyadenylation machinery in planaria. (A) A schematic representing all the components of the eukaryotic polyadenylation machinery in planaria. These specifically include multi-subunit protein complexes such as: Cleavage Factor I (CFIm), Cleavage and Polyadenylation Specificity Factor (CPSF), Cleavage stimulation Factor (CstF), Cleavage factor II (CFIIm), and poly(A) polymerases (PAPOA, PAPOB, or PAPOG). The different protein subunits for each of these complexes are colored based upon the negative E-value exponent returned by BLAST-hit against its human homolog. The components for which the homolog could not be determined confidently in planaria (hFip1, CFIm-59/68, and Pcf11) are marked with dotted lines. The corresponding &lt;em&gt;cis&lt;/em&gt;-elements recognized by these protein complexes are also shown in the schematic. (B) RT-PCR products showing the expression of the transcripts coding for proteins of the polyadenylation machinery, transcript ID, their corresponding human homolog, percentage sequence identity, and coverage (derived from blast2seq using BLOSUM62 matrix). ID, identity; PCR, polymerase chain reaction.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/10/3035/F1.medium.gif" width="440" height="295"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/10/3035/F1.medium.gif" width="440" height="295"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/10/3035/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/10/3035/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469458" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-25" class="first-child">Polyadenylation machinery in planaria. (A) A schematic representing all the components of the eukaryotic polyadenylation machinery in planaria. These specifically include multi-subunit protein complexes such as: Cleavage Factor I (CFIm), Cleavage and Polyadenylation Specificity Factor (CPSF), Cleavage stimulation Factor (CstF), Cleavage factor II (CFIIm), and poly(A) polymerases (PAPOA, PAPOB, or PAPOG). The different protein subunits for each of these complexes are colored based upon the negative E-value exponent returned by BLAST-hit against its human homolog. The components for which the homolog could not be determined confidently in planaria (hFip1, CFIm-59/68, and Pcf11) are marked with dotted lines. The corresponding <em>cis</em>-elements recognized by these protein complexes are also shown in the schematic. (B) RT-PCR products showing the expression of the transcripts coding for proteins of the polyadenylation machinery, transcript ID, their corresponding human homolog, percentage sequence identity, and coverage (derived from blast2seq using BLOSUM62 matrix). ID, identity; PCR, polymerase chain reaction.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-18" class="subsection"><h3>Characterization of polyadenylation sites reveals a conserved mechanism in planarians</h3><p id="p-26">In order to determine polyadenylation sites in <em>S. mediterranea</em>, poly(A) RNAs were isolated from the whole organism (both sexual and asexual forms separately) and then subjected to 3P-Seq. Sequencing reads thus obtained were mapped to the SxlV3.1 genome using bowtie (<a id="xref-ref-34-2" class="xref-bibr" href="#ref-34">Langmead <em>et al.</em> 2009</a>). A read was considered as a 3P-tag if the alignment to the genome resulted in a mismatch of “A” at the 3′ end, which would be an untemplated adenosine from the poly(A) tail (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS5.pdf">Figure S5</a>). The alignment thus obtained was used to derive 3P-peaks, defined as discrete regions on the genome covered by the 3P-tags. Each of these peaks could represent an individual polyadenylation event. In total, 31,377 polyadenylated sites were identified from this analysis (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/TableS1.xls">Table S1</a>).</p><p id="p-27">Next, we looked for <em>cis</em>-regulatory elements associated with the 3P-peaks identified in our study. We found an enrichment of “U” around the cleavage site, a property which is conserved across eukaryotes (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2A</a>). These U-rich regions are specifically recognized by the CFI and CFII complexes. The A-rich region, corresponding to PAS recognized by CPSF, was also observed 10–30 nt upstream of the cleavage site. The CPSF complex cleaves adjacent to a “CA” dinucleotide, which is conserved in eukaryotes (<a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Mandel <em>et al.</em> 2006</a>). In planarians, in addition to “CA,” we also observed an enrichment of “UA” and “UU” near the cleavage site (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2B</a>). The CFIm-59/68 (CFI subunit), which recognizes the “UGUA” site, could not be detected in planarians using a sequence homolog search. This is also further supported by the lack of a “UGUA” motif upstream of the PAS in planaria.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/10/3035/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Conservation of polyadenylation signals in planaria. A line-plot showing the (A) mononucleotide profile and (B) the dinucleotide profile (CA, UU, and UA) of all the 31,377 3P-peaks derived from 3P-Seq (poly(A)-position profiling by sequencing). The 0-position on the x-axis denotes the cleavage site (annotated with dotted line and arrow). The upstream elements (USE) and the downstream elements (DSE), along with the polyadenylation signal (PAS) have been separately highlighted in blue and red, respectively. The U-rich element (URE) flanking the cleavage site has been highlighted in gray. (C) A line plot showing the distribution of different hexameric polyadenylation signals on the 3P-peaks, which are specifically recognized by the CPSF (Cleavage and Polyadenylation Specificity Factor subunit). (D) The matrix plot highlighting the distribution of secondary hexameric signals (other than canonical &#x201C;AAUAAA&#x201D;) across the 3P-peaks. The histogram in the top panel represents the positional distribution of the hexameric PAS across the 3P-peaks. The panel on the right is the bar plot reporting the frequency of the corresponding hexameric sequences across polyadenylation sites. The hexameric signals that passed the Fisher&#x2019;s test of independence are highlighted in red. (E) The histogram showing the number of 3P-peaks that were binned based on the genomic distances. The mononucleotide scan was performed for 3P-peaks by fixing the distal cleavage sites that were separated by (F) 10 and (G) 30&#x2013;35 nucleotides." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-884262096" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Conservation of polyadenylation signals in planaria. A line-plot showing the (A) mononucleotide profile and (B) the dinucleotide profile (CA, UU, and UA) of all the 31,377 3P-peaks derived from 3P-Seq (poly(A)-position profiling by sequencing). The 0-position on the x-axis denotes the cleavage site (annotated with dotted line and arrow). The upstream elements (USE) and the downstream elements (DSE), along with the polyadenylation signal (PAS) have been separately highlighted in blue and red, respectively. The U-rich element (URE) flanking the cleavage site has been highlighted in gray. (C) A line plot showing the distribution of different hexameric polyadenylation signals on the 3P-peaks, which are specifically recognized by the CPSF (Cleavage and Polyadenylation Specificity Factor subunit). (D) The matrix plot highlighting the distribution of secondary hexameric signals (other than canonical &#x201C;AAUAAA&#x201D;) across the 3P-peaks. The histogram in the top panel represents the positional distribution of the hexameric PAS across the 3P-peaks. The panel on the right is the bar plot reporting the frequency of the corresponding hexameric sequences across polyadenylation sites. The hexameric signals that passed the Fisher&#x2019;s test of independence are highlighted in red. (E) The histogram showing the number of 3P-peaks that were binned based on the genomic distances. The mononucleotide scan was performed for 3P-peaks by fixing the distal cleavage sites that were separated by (F) 10 and (G) 30&#x2013;35 nucleotides.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/10/3035/F2.medium.gif" width="440" height="413"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/10/3035/F2.medium.gif" width="440" height="413"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/10/3035/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/10/3035/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469296" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-28" class="first-child">Conservation of polyadenylation signals in planaria. A line-plot showing the (A) mononucleotide profile and (B) the dinucleotide profile (CA, UU, and UA) of all the 31,377 3P-peaks derived from 3P-Seq (poly(A)-position profiling by sequencing). The 0-position on the x-axis denotes the cleavage site (annotated with dotted line and arrow). The upstream elements (USE) and the downstream elements (DSE), along with the polyadenylation signal (PAS) have been separately highlighted in blue and red, respectively. The U-rich element (URE) flanking the cleavage site has been highlighted in gray. (C) A line plot showing the distribution of different hexameric polyadenylation signals on the 3P-peaks, which are specifically recognized by the CPSF (Cleavage and Polyadenylation Specificity Factor subunit). (D) The matrix plot highlighting the distribution of secondary hexameric signals (other than canonical “AAUAAA”) across the 3P-peaks. The histogram in the top panel represents the positional distribution of the hexameric PAS across the 3P-peaks. The panel on the right is the bar plot reporting the frequency of the corresponding hexameric sequences across polyadenylation sites. The hexameric signals that passed the Fisher’s test of independence are highlighted in red. (E) The histogram showing the number of 3P-peaks that were binned based on the genomic distances. The mononucleotide scan was performed for 3P-peaks by fixing the distal cleavage sites that were separated by (F) 10 and (G) 30–35 nucleotides.</p><div class="sb-div caption-clear"></div></div></div><p id="p-29">In higher eukaryotes, the canonical hexameric sequence of “AAUAAA” is located 10–30 nt upstream of the cleavage site (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2C</a>) (<a id="xref-ref-13-3" class="xref-bibr" href="#ref-13">Colgan and Manley 1997</a>). This motif is specifically recognized by the CPSF160 subunit (<a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">Schonemann <em>et al.</em> 2014</a>). The machinery also tolerates a single/double nucleotide variation within this canonical hexamer. In planaria, we found “AAUAAA” and its four single-nucleotide variants (AAUAUA, AUUAAA, AAUACA, and AAUAGA) enriched 10–30 nt upstream of the cleavage site (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2D</a>). These four hexameric variants have also been reported to act as PAS in other metazoans (<a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">You <em>et al.</em> 2015</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS6.pdf">Figure S6</a>). This is consistent with the results of a motif search using the MEME software (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS6.pdf">Figure S6</a>). The PAS hexameric signal, including the single and double-nucleotide variants, were found in 98.8% of 3P-peaks. Around 41% of these 3P-peaks contained the canonical “AAUAAA” signal; another 40.4% contained a single-nucleotide variant of the canonical signal; and the remaining 16.3% peaks had two nucleotide variations (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS6.pdf">Figure S6</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/TableS1.xls">Table S1</a>).</p><p id="p-30">We also determined the resolution of the polyadenylation sites by measuring the genomic distances between adjacent 3P-peaks. The distance is defined by the number of nucleotides between the end of one 3P-peak and the beginning of the neighboring peak. We observed 1390 peaks that were separated by a distance of only 100 nt. The gap between a majority of these 3P-peak pairs was between 10–35 nt (<a id="xref-fig-2-5" class="xref-fig" href="#F2">Figure 2E</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS8.pdf">Figure S8</a>). To rule out the possibility that these short distances are not merely single unresolved polyadenylation sites, we tested for the presence of polyadenylation signals within them. For this analysis, 150 bp spanning both upstream and downstream of the distal cleavage site was used. The enrichment of the U-rich signal (<a id="xref-fig-2-6" class="xref-fig" href="#F2">Figure 2, F and G</a>) along with the PAS (determined by fixing the proximal cleavage site, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS8.pdf">Figure S8</a>) confirmed these to be genuine 3P-peaks.</p></div><div id="sec-19" class="subsection"><h3>Annotation of 3′UTRs in planaria</h3><p id="p-31">We associated each transcript to its nearest 3P-peak to determine its polyadenylation site. For this, we used three different transcriptome models of planaria: a) Maker, a computational pipeline that performs abinitio gene prediction and annotation of the genome (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Cantarel <em>et al.</em> 2008</a>); b) Ox_Smed_v1 (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Blythe <em>et al.</em> 2010</a>); and c) dd_Smed_v4 (<a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Liu <em>et al.</em> 2013</a>), both of which are derived from <em>de novo</em> transcriptome assemblies of RNA-Seq data. The polyadenylation sites obtained could be categorized depending upon its position with respect to the transcript model (<a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/TableS2.xls">Table S2</a>). Around 92% of these polyadenylation sites were present on the terminal exon (Category 4, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS9.pdf">Figure S9</a>). We also observed polyadenylation sites (43.51%) outside the transcript models (Category 1 in <a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a>), emphasizing that these transcripts could have longer 3′UTRs than previously annotated. We found that ∼40% of transcripts were associated with more than one polyadenylation site (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>), which is close to the estimated ApA in other eukaryotes (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Derti <em>et al.</em> 2012</a>). For those transcripts with a polyadenylation site outside the annotated transcript, we extended the transcript to include the nearest 3P-peak. Various sources of evidence such as RNA-Seq reads, Genscan (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Burge and Karlin 1997</a>), and Genpred were used to confirm the extended transcripts (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3C</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS9.pdf">Figure S9</a>). Around 70% of the extended transcripts, across the three transcriptomes, had a length &lt; 2 kb (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3D</a>). For 213 transcripts, we were able to extend the protein coding region by more than 15 amino acids (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS9.pdf">Figure S9</a>). For 21 of these transcripts, we were able to assign pfam domains to the extended region (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS9.pdf">Figure S9</a>). Around 9972 unique 3P-peaks that were mapped to genomic contigs had no transcript annotations (Category 6, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS9.pdf">Figure S9</a>). These could be putative long noncoding RNAs (<a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Ulitsky <em>et al.</em> 2011</a>), which needs to be further characterized. We also observed cleavage sites that were in the opposite orientation to a neighboring transcript, which suggests the presence of antisense transcripts in these loci (Category 7, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS9.pdf">Figure S9</a>). To remove the redundancy arising from the pooling of transcripts from three different datasets, we clustered all protein sequences (coded by longest ORFs) based on sequence similarity using Cd-hit (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Li and Godzik 2006</a>). A high sequence identity cut-off of 90% was used to retain the paralogs. This resulted in 31,990 gene clusters (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS10.pdf">Figure S10</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/TableS3.xls">Table S3</a>). The longest protein sequence from each cluster was chosen as representative of that cluster to identify the length of 3′UTRs. The median length of the 3′UTR was found to be around 176 bp, as compared to 138 in <em>C. elegans</em>, 235 in <em>Drosophila</em>, 518 in zebrafish, and 1198 in humans (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3E</a>).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/10/3035/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Annotation of 3&#x2032;UTR sequences in planaria. (A) The association of each 3P-peak to the corresponding transcript was categorized based on the relative positioning of the 3P-peak on the existing transcript model. The red bar represents the annotated part of the transcript and the dotted line represents the unannotated region, downstream of the transcript. The arrows represent the cleavage site that was derived from the 3P-Seq analysis in these unannotated (black) and annotated (blue) regions of the transcript. The association was carried out using three different transcriptomes (Makers, Ox_Smed_v1, and dd_smed_v4). The percentage association of each category across different transcriptomes are also depicted. (B) A bar plot showing the number of 3P-peaks [poly(A) peaks] associated with each of the transcripts across different transcriptomes. (C) Stacked bar plot representing the number of transcripts that could be extended further based upon the 3P-peaks associated with them. (D) The bar plot highlights the length of the stitched region for the transcripts across different transcriptomes. The percentage denotes the total number of transcripts that were extended by specific length across the three different transcriptomes. (E) Box plot illustrating the distribution of 3&#x2032;UTR length across different species: C. elegans, S. mediterranea, D. melanogaster, Danio rerio, and Homo sapiens. The data for the length of the 3&#x2032;UTRs were derived from the Ensemble database. (F) Pie chart revealing the presence of the canonical &#x201C;AAUAAA&#x201D; signal across transcripts associated with two polyadenylation sites. (G) Stacked bar plot illustrating the occurrences of different hexameric PAS across proximal and distal polyadenylation sites. (H) Stacked bar plot reflecting the abundance of different hexameric PAS across transcripts associated with one and multiple 3P-peaks. 3P-Seq, poly(A)-position profiling by sequencing; PAS, poly(A) signals; UTR, untranslated region." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-884262096" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Annotation of 3&#x2032;UTR sequences in planaria. (A) The association of each 3P-peak to the corresponding transcript was categorized based on the relative positioning of the 3P-peak on the existing transcript model. The red bar represents the annotated part of the transcript and the dotted line represents the unannotated region, downstream of the transcript. The arrows represent the cleavage site that was derived from the 3P-Seq analysis in these unannotated (black) and annotated (blue) regions of the transcript. The association was carried out using three different transcriptomes (Makers, Ox_Smed_v1, and dd_smed_v4). The percentage association of each category across different transcriptomes are also depicted. (B) A bar plot showing the number of 3P-peaks [poly(A) peaks] associated with each of the transcripts across different transcriptomes. (C) Stacked bar plot representing the number of transcripts that could be extended further based upon the 3P-peaks associated with them. (D) The bar plot highlights the length of the stitched region for the transcripts across different transcriptomes. The percentage denotes the total number of transcripts that were extended by specific length across the three different transcriptomes. (E) Box plot illustrating the distribution of 3&#x2032;UTR length across different species: &lt;em&gt;C. elegans&lt;/em&gt;, &lt;em&gt;S. mediterranea&lt;/em&gt;, &lt;em&gt;D. melanogaster&lt;/em&gt;, &lt;em&gt;Danio rerio&lt;/em&gt;, and &lt;em&gt;Homo sapiens&lt;/em&gt;. The data for the length of the 3&#x2032;UTRs were derived from the Ensemble database. (F) Pie chart revealing the presence of the canonical &#x201C;AAUAAA&#x201D; signal across transcripts associated with two polyadenylation sites. (G) Stacked bar plot illustrating the occurrences of different hexameric PAS across proximal and distal polyadenylation sites. (H) Stacked bar plot reflecting the abundance of different hexameric PAS across transcripts associated with one and multiple 3P-peaks. 3P-Seq, poly(A)-position profiling by sequencing; PAS, poly(A) signals; UTR, untranslated region.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/10/3035/F3.medium.gif" width="440" height="418"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/10/3035/F3.medium.gif" width="440" height="418"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/10/3035/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/10/3035/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469393" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-32" class="first-child">Annotation of 3′UTR sequences in planaria. (A) The association of each 3P-peak to the corresponding transcript was categorized based on the relative positioning of the 3P-peak on the existing transcript model. The red bar represents the annotated part of the transcript and the dotted line represents the unannotated region, downstream of the transcript. The arrows represent the cleavage site that was derived from the 3P-Seq analysis in these unannotated (black) and annotated (blue) regions of the transcript. The association was carried out using three different transcriptomes (Makers, Ox_Smed_v1, and dd_smed_v4). The percentage association of each category across different transcriptomes are also depicted. (B) A bar plot showing the number of 3P-peaks [poly(A) peaks] associated with each of the transcripts across different transcriptomes. (C) Stacked bar plot representing the number of transcripts that could be extended further based upon the 3P-peaks associated with them. (D) The bar plot highlights the length of the stitched region for the transcripts across different transcriptomes. The percentage denotes the total number of transcripts that were extended by specific length across the three different transcriptomes. (E) Box plot illustrating the distribution of 3′UTR length across different species: <em>C. elegans</em>, <em>S. mediterranea</em>, <em>D. melanogaster</em>, <em>Danio rerio</em>, and <em>Homo sapiens</em>. The data for the length of the 3′UTRs were derived from the Ensemble database. (F) Pie chart revealing the presence of the canonical “AAUAAA” signal across transcripts associated with two polyadenylation sites. (G) Stacked bar plot illustrating the occurrences of different hexameric PAS across proximal and distal polyadenylation sites. (H) Stacked bar plot reflecting the abundance of different hexameric PAS across transcripts associated with one and multiple 3P-peaks. 3P-Seq, poly(A)-position profiling by sequencing; PAS, poly(A) signals; UTR, untranslated region.</p><div class="sb-div caption-clear"></div></div></div><p id="p-33">We found that 3639 transcripts contained two polyadenylation sites. We analyzed the occurrence of hexameric sequence motifs in these 3P-peaks. The canonical hexameric sequence “AAUAAA” was enriched in the distal compared to the proximal site (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3, F and G</a>; <em>t</em>-test unpaired: <em>P</em>-value &lt; 2 x <span class="inline-formula" id="inline-formula-6"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10/3035/embed/mml-math-7.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10/3035/embed/mml-math-7.gif"/></noscript></span></span>). The overall abundance of this canonical “AAUAAA” hexameric signal decreases with the increase in the number of polyadenylation sites per transcript (<a id="xref-fig-3-8" class="xref-fig" href="#F3">Figure 3H</a>, <em>t</em>-test unpaired: <em>P</em>-value &lt; 2 x <span class="inline-formula" id="inline-formula-7"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10/3035/embed/mml-math-8.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10/3035/embed/mml-math-8.gif"/></noscript></span></span>). This suggests that most of the transcripts associated with multiple 3P-peaks have a combination of strong (canonical), and a variant of the canonical, signals, which could have evolved to play a regulatory role. Further, we also explored the association of the canonical signal and its variants with the expression levels of the transcripts, but we did not find any correlation between the expression levels of the transcripts and the corresponding hexameric PAS signals associated with them (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS11.pdf">Figure S11</a>).</p></div><div id="sec-20" class="subsection"><h3>Validation and confirmation of cleavage sites and polyadenylation signals in planarians</h3><p id="p-34">The association of transcripts with polyadenylated sites derived from 3P-Seq were validated using various approaches. Transcripts with a single 3P-peak were specifically selected and their 3′ UTR expression probed using publicly available RNA-Seq data (<a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Resch <em>et al.</em> 2012</a>). We found that the region upstream of the polyadenylation site has more RNA-Seq coverage in comparison to its downstream region (<a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS12.pdf">Figure S12</a>; Wilcox test, paired: <em>P</em>–value &lt; 2.2 × <span class="inline-formula" id="inline-formula-8"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10/3035/embed/mml-math-9.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10/3035/embed/mml-math-9.gif"/></noscript></span></span>). This difference in RNA-Seq coverage increases with the strength (number of 3P-tags) of the 3P-peak (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS12.pdf">Figure S12</a>). Although we do see a few outliers with more reads in the downstream, most of these outliers may code for putative transcripts (as determined by Genscan, Genpred, and EST databases) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS12.pdf">Figure S12</a>). The other possible explanation for these outliers could be the presence of a downstream polyadenylation signal that could not be captured by the current depth of 3P-Seq.</p><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/10/3035/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Validation of polyadenylation sites and transcript association. (A) A schematic highlighting the coverage of ion-torrent reads (black) and RNA-seq reads in the regions upstream (red) and downstream (blue) of polyadenylation sites determined from 3P-Seq (i). The bar plot shows the comparison between the number of transcripts associated with one 3P-peak and those transcripts that were also mapped by the RNA-seq reads (ii). The scatter plot depicts the transcripts with the coverage of RNA-seq reads in the inner (blue) and outer (red) region of the polyadenylation site (iii). The bar plot depicts comparison of the number of transcripts associated with one 3P-peak and those transcripts that were mapped by ion-torrent reads across different transcriptomes (iv). The scatter plot depicts the transcripts with the coverage derived from the ion-torrent reads aligned to the upstream and downstream regions of the cleavage site (v). (B) The coverage of ion-torrent reads for the region spanning the 3P-peaks. (C) A coverage plot showing the coverage of RNA-seq reads and ion-torrent reads across different transcript models for ARS2 domain-containing protein. The corresponding 3P-peak derived for it supports the ox_Smed_V1 and dd_Smed_V4 models. (D) Single peak candidates that were experimentally validated through PCR amplification. A schematic highlighting the strategy for designing forward (blue) and reverse primers (red). (E) The transcripts with two 3P-peaks were experimentally validated using PCR. The upper band represents the amplification of longer transcript isoform and the lower band represents the shorter form of the transcript. 3P-Seq, poly(A)-position profiling by sequencing; PCR, polymerase chain reaction; RNA-Seq, RNA sequencing. " class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-884262096" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Validation of polyadenylation sites and transcript association. (A) A schematic highlighting the coverage of ion-torrent reads (black) and RNA-seq reads in the regions upstream (red) and downstream (blue) of polyadenylation sites determined from 3P-Seq (i). The bar plot shows the comparison between the number of transcripts associated with one 3P-peak and those transcripts that were also mapped by the RNA-seq reads (ii). The scatter plot depicts the transcripts with the coverage of RNA-seq reads in the inner (blue) and outer (red) region of the polyadenylation site (iii). The bar plot depicts comparison of the number of transcripts associated with one 3P-peak and those transcripts that were mapped by ion-torrent reads across different transcriptomes (iv). The scatter plot depicts the transcripts with the coverage derived from the ion-torrent reads aligned to the upstream and downstream regions of the cleavage site (v). (B) The coverage of ion-torrent reads for the region spanning the 3P-peaks. (C) A coverage plot showing the coverage of RNA-seq reads and ion-torrent reads across different transcript models for ARS2 domain-containing protein. The corresponding 3P-peak derived for it supports the ox_Smed_V1 and dd_Smed_V4 models. (D) Single peak candidates that were experimentally validated through PCR amplification. A schematic highlighting the strategy for designing forward (blue) and reverse primers (red). (E) The transcripts with two 3P-peaks were experimentally validated using PCR. The upper band represents the amplification of longer transcript isoform and the lower band represents the shorter form of the transcript. 3P-Seq, poly(A)-position profiling by sequencing; PCR, polymerase chain reaction; RNA-Seq, RNA sequencing. &lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/10/3035/F4.medium.gif" width="440" height="398"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/10/3035/F4.medium.gif" width="440" height="398"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/10/3035/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/10/3035/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469362" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-35" class="first-child">Validation of polyadenylation sites and transcript association. (A) A schematic highlighting the coverage of ion-torrent reads (black) and RNA-seq reads in the regions upstream (red) and downstream (blue) of polyadenylation sites determined from 3P-Seq (i). The bar plot shows the comparison between the number of transcripts associated with one 3P-peak and those transcripts that were also mapped by the RNA-seq reads (ii). The scatter plot depicts the transcripts with the coverage of RNA-seq reads in the inner (blue) and outer (red) region of the polyadenylation site (iii). The bar plot depicts comparison of the number of transcripts associated with one 3P-peak and those transcripts that were mapped by ion-torrent reads across different transcriptomes (iv). The scatter plot depicts the transcripts with the coverage derived from the ion-torrent reads aligned to the upstream and downstream regions of the cleavage site (v). (B) The coverage of ion-torrent reads for the region spanning the 3P-peaks. (C) A coverage plot showing the coverage of RNA-seq reads and ion-torrent reads across different transcript models for ARS2 domain-containing protein. The corresponding 3P-peak derived for it supports the ox_Smed_V1 and dd_Smed_V4 models. (D) Single peak candidates that were experimentally validated through PCR amplification. A schematic highlighting the strategy for designing forward (blue) and reverse primers (red). (E) The transcripts with two 3P-peaks were experimentally validated using PCR. The upper band represents the amplification of longer transcript isoform and the lower band represents the shorter form of the transcript. 3P-Seq, poly(A)-position profiling by sequencing; PCR, polymerase chain reaction; RNA-Seq, RNA sequencing. </p><div class="sb-div caption-clear"></div></div></div><p id="p-36">The 3′UTRs derived from 3P-Seq were further confirmed through ion-torrent sequencing (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Rothberg <em>et al.</em> 2011</a>) of polyadenylated mRNAs. This technology adopts a different chemistry optimized to derive reads of longer length (100–300 bp, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS12.pdf">Figure S12</a>). The reads derived from this ion-torrent sequencing were trimmed to remove the polyadenylated tail and subsequently mapped to the genome. The alignment covered around 42% of the 3P-peaks derived from 3P-Seq, and the majority of these reads mapped upstream of the cleavage site, thus validating the 3P-seq data (<a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4B</a>). Thus, RNA-Seq reads and ion-torrent reads coupled to 3P-Seq data from this study can help to obtain complete gene models in planaria (<a id="xref-fig-4-4" class="xref-fig" href="#F4">Figure 4C</a>).</p><p id="p-37">Some of the transcripts associated with polyadenylation sites were also experimentally validated using RT-PCR and Sanger sequencing. The validation (<a id="xref-fig-4-5" class="xref-fig" href="#F4">Figure 4D</a>) clearly suggests that existing transcript models coding for TNF2, Rabl2, and Ppp2r1b can be extended to capture the complete 3′UTR. Some transcripts with two polyadenylation sites were also experimentally validated through RT-PCR. We found that genes involved in the regulation of cell fate and patterning like <em>smed-wnt</em>, <em>smed-noggin</em>, and <em>smed-ybox-4</em> undergo ApA.</p></div><div id="sec-21" class="subsection"><h3>Biological functions mediated by alternate polyadenylation</h3><p id="p-38">Around 40% of the transcripts in planaria were associated with more than one polyadenylation site, which emphasizes the need to explore the biological significance of ApA. A Gene Ontology (GO) enrichment analysis was carried out for these transcripts. The following biological processes were significantly enriched: hemophilic cell adhesion via plasma membrane (GO:0007156), transmembrane transport (GO:0055085), the G-protein coupled receptor-signaling pathway (GO:0007186), and ion transport (GO:0006811). We also observed the enrichment of genes involved in cell cycle arrest (GO:0007050) and the regulation of sequence-specific DNA-binding transcription factors (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/TableS4.xls">Table S4</a>). The molecular functions that were enriched included: zinc-ion binding (GO:0008270), DNA/nucleic acid binding (GO:0003677, GO:0003676), protein serine/threonine kinase activity (GO:0004674), protein dimerization activity (GO:0046983). and ribonuclease activity (GO:0004540) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS13.pdf">Figure S13</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/TableS4.xls">Table S4</a>).</p><p id="p-39">Candidate-based approaches have previously reported that ApA plays a role in regulating biological processes such as the production of membrane-bound and secreted immunoglobins (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Alt <em>et al.</em> 1980</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Rogers <em>et al.</em> 1980</a>; <a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">Early <em>et al.</em> 1980</a>), T-cell activation (<a id="xref-ref-54-2" class="xref-bibr" href="#ref-54">Sandberg <em>et al.</em> 2008</a>), hippocampal neuron stimulation (<a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Flavell and Greenberg 2008</a>), embryonic development (<a id="xref-ref-28-2" class="xref-bibr" href="#ref-28">Ji <em>et al.</em> 2009</a>; <a id="xref-ref-38-2" class="xref-bibr" href="#ref-38">Mangone <em>et al.</em> 2010</a>; <a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Hilgers <em>et al.</em> 2011</a>), lymphomas (<a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Singh <em>et al.</em> 2009</a>), and breast cancer (<a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Fu <em>et al.</em> 2011</a>). We performed a literature survey to assemble a list of genes known to be regulated by ApA (<a id="xref-ref-39-2" class="xref-bibr" href="#ref-39">Mayr and Bartel 2009</a>; <a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Smibert <em>et al.</em> 2012</a>; <a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Sun <em>et al.</em> 2012</a>). Around 70 genes were identified by this survey (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/TableS5.xls">Table S5</a>) and homologs for 22 of them could be found in planaria. Of these, 10 were associated with more than one polyadenylation signal.</p></div><div id="sec-22" class="subsection"><h3>Functional consequences of alternate polyadenylation (ApA)</h3><p id="p-40">For transcripts containing two 3P-peaks, we investigated their location relative to the known transcript model. These peaks could either be located on the 3′UTR, giving rise to tandem 3′UTR ApA, or on the coding regions resulting in coding region polyadenylation (CR-ApA) (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Elkon <em>et al.</em> 2013</a>). For a majority of transcripts (∼80.1%, 1213 transcripts), the multiple ApA sites were all located on the last exon resulting in tandem 3′ UTR ApA (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5A</a>). This tandem 3′UTR ApA results in mRNA with an altered 3′UTR sequence and thus, could regulate posttranscriptional gene expression. The altered 3′UTR produced by ApA can result in a loss or gain of miRNA binding sites (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6A</a>). Hence, a systematic search was performed to predict miRNA binding sites within the 3′UTRs of the transcripts using miRanda (<a id="xref-ref-30-2" class="xref-bibr" href="#ref-30">John <em>et al.</em> 2004</a>). This is the first comprehensive prediction of miRNA binding sites in the planarian genome (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>). We defined the proximal 3′UTR to include the region from the end of the last exon to the proximal cleavage site. The distal 3′UTR involved the region between the proximal and the distal cleavage site. Thus, we ensured that both these regions had no overlap for miRNA binding site prediction. For transcripts with two polyadenylated sites, we observed that miRNA binding sites are preferentially located either at the proximal or the distal 3′UTR. This is reflected by the bimodal distribution of the occurrence of these sites along the transcript. This trend was rarely observed in a randomized control dataset (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6C</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS14.pdf">Figure S14</a>). Around 14 out of 100 randomized datasets displayed a bimodal distribution (as reported by Hartigan’s dip test with <em>P</em>-value &lt; 0.05), but seven of these bimodal distributions were qualitatively different from the actual profile (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS14.pdf">Figure S14</a>).</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/10/3035/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Functional consequences of alternate polyadenylation. (A) Schematic highlighting the classification of the alternate polyadenylation events based upon the existing transcriptome model. (B) Pie chart showing the polyadenylation events in the internal exons that could potentially lead to either loss of domain or no change in ORF. (C) The RNA-Seq coverage plot for OX_Smed.1.0.03529 () from X1, Xins population along with 3P-peaks is shown on the left panel. The corresponding whole mount in situ hybridization (WISH) patterns for the  is shown on the right. (D) Similar to (C), RNA-seq coverage and WISH patterns for the dd_smed_V4_3757_0_1 (slmb) are shown. ORF, open reading frame; RNA-Seq, RNA sequencing; UTR, untranslated region." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-884262096" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot; xmlns:hwp=&quot;http://schema.highwire.org/Journal&quot; xmlns:l=&quot;http://schema.highwire.org/Linking&quot;&gt;Functional consequences of alternate polyadenylation. (A) Schematic highlighting the classification of the alternate polyadenylation events based upon the existing transcriptome model. (B) Pie chart showing the polyadenylation events in the internal exons that could potentially lead to either loss of domain or no change in ORF. (C) The RNA-Seq coverage plot for OX_Smed.1.0.03529 (&lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-10&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt;) from X1, Xins population along with 3P-peaks is shown on the left panel. The corresponding whole mount &lt;em&gt;in situ&lt;/em&gt; hybridization (WISH) patterns for the &lt;img src=&quot;pending:yes&quot; l:ref-type=&quot;journal&quot; hwp:journal=&quot;&quot; hwp:article=&quot;&quot; l:sub-ref=&quot;mml-math-11&quot; l:type=&quot;image/*&quot; class=&quot;math mml&quot; alt=&quot;Formula&quot;/&gt; is shown on the right. (D) Similar to (C), RNA-seq coverage and WISH patterns for the dd_smed_V4_3757_0_1 (&lt;em&gt;slmb&lt;/em&gt;) are shown. ORF, open reading frame; RNA-Seq, RNA sequencing; UTR, untranslated region.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/10/3035/F5.medium.gif" width="365" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/10/3035/F5.medium.gif" width="365" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/10/3035/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/10/3035/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469233" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-41" class="first-child">Functional consequences of alternate polyadenylation. (A) Schematic highlighting the classification of the alternate polyadenylation events based upon the existing transcriptome model. (B) Pie chart showing the polyadenylation events in the internal exons that could potentially lead to either loss of domain or no change in ORF. (C) The RNA-Seq coverage plot for OX_Smed.1.0.03529 (<span class="inline-formula" id="inline-formula-9"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10/3035/F5/embed/mml-math-10.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10/3035/F5/embed/mml-math-10.gif"/></noscript></span></span>) from X1, Xins population along with 3P-peaks is shown on the left panel. The corresponding whole mount <em>in situ</em> hybridization (WISH) patterns for the <span class="inline-formula" id="inline-formula-10"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10/3035/F5/embed/mml-math-11.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10/3035/F5/embed/mml-math-11.gif"/></noscript></span></span> is shown on the right. (D) Similar to (C), RNA-seq coverage and WISH patterns for the dd_smed_V4_3757_0_1 (<em>slmb</em>) are shown. ORF, open reading frame; RNA-Seq, RNA sequencing; UTR, untranslated region.</p><div class="sb-div caption-clear"></div></div></div><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/10/3035/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Effect of altered 3&#x2032;UTR on miRNA-mediated gene regulation. (A) Schematic describing the location of miRNA binding sites with respect to the proximal and distal polyadenylation events. (B) The miRNA binding sites that were predicted to be associated with stitched transcripts across different transcriptomes. (C) The distribution of miRNA binding sites on the transcripts at the regions spanning the proximal and distal polyadenylation sites. (D) The bar plot shows the number of the degradome tags mapped to the transcripts on either the proximal (positive y-axis) or distal (negative y-axis) polyadenylation site. miRNA, microRNA; UTR, untranslated region." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-884262096" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Effect of altered 3&#x2032;UTR on miRNA-mediated gene regulation. (A) Schematic describing the location of miRNA binding sites with respect to the proximal and distal polyadenylation events. (B) The miRNA binding sites that were predicted to be associated with stitched transcripts across different transcriptomes. (C) The distribution of miRNA binding sites on the transcripts at the regions spanning the proximal and distal polyadenylation sites. (D) The bar plot shows the number of the degradome tags mapped to the transcripts on either the proximal (positive y-axis) or distal (negative y-axis) polyadenylation site. miRNA, microRNA; UTR, untranslated region.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/10/3035/F6.medium.gif" width="440" height="393"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/10/3035/F6.medium.gif" width="440" height="393"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/10/3035/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/10/3035/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469397" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-42" class="first-child">Effect of altered 3′UTR on miRNA-mediated gene regulation. (A) Schematic describing the location of miRNA binding sites with respect to the proximal and distal polyadenylation events. (B) The miRNA binding sites that were predicted to be associated with stitched transcripts across different transcriptomes. (C) The distribution of miRNA binding sites on the transcripts at the regions spanning the proximal and distal polyadenylation sites. (D) The bar plot shows the number of the degradome tags mapped to the transcripts on either the proximal (positive y-axis) or distal (negative y-axis) polyadenylation site. miRNA, microRNA; UTR, untranslated region.</p><div class="sb-div caption-clear"></div></div></div><p id="p-43">We further performed degradome sequencing to experimentally identify the miRNA targets in the planarian transcriptome. This method has been previously used to identify miRNA targets in other metazoans (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Cass <em>et al.</em> 2016</a>; <a id="xref-ref-31-2" class="xref-bibr" href="#ref-31">Karginov <em>et al.</em> 2010</a>). The reads that map to the miRNA targets are those that are cleaved by the AGO machinery because of the complementarity at the ninth, 10th, and 11th nucleotides. The targets that were translationally repressed will not be captured by this method. The mapped reads that represent the miRNA binding sites on the transcript were annotated as degradome tags (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/TableS6.xls">Table S6</a>). We identified around 219 tags that mapped to 259 transcripts having two polyadenylation sites associated with them. These tags mapped exclusively either to the proximal (135 transcripts) or the distal (104 transcripts) 3′UTRs of the transcripts. Very few transcripts (20 transcripts) had an equal number of degradome tags on both the proximal and the distal 3′UTR (<a id="xref-fig-6-4" class="xref-fig" href="#F6">Figure 6D</a>).</p><p id="p-44">Later, we also investigated ApA events that alter the protein coding sequence and subsequently its biological function. Around 19% of the transcripts had polyadenylation sites located within introns/exons; these are called coding region alternate polyadenylation (CR-ApA) sites. We found ApA sites on more than one exon for around 270 transcripts (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5B</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/TableS7.xls">Table S7</a>). Of these, 97 could potentially result in truncated proteins (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/TableS7.xls">Table S7</a>). We found that truncation could result in the loss of certain domains from these proteins. These domains include an EF-hand domain pair, a WD domain, tetracopeptide repeats, dynein heavy chain region D6 of dynein motor, a myosin tail, Ras family, and a methyltransferase domain.</p><p id="p-45">We performed whole mount <em>in situ</em> hybridization (WISH) to study the expression pattern of alternatively polyadenylated transcripts dd_smed_V4_650_0_1 (p25α) and dd_smed_V4_3757_0_1 (<em>slmb</em>). The <em>in situ</em> probe designed from the region upstream of the proximal PAS signal will recognize both the isoforms, whereas the probe designed from the region immediately upstream of the distal PAS signal will specifically hybridize to the longer isoform of the transcript. The whole mount <em>in situ</em> hybridization using the probe, which recognizes both the isoforms of the transcript <span class="inline-formula" id="inline-formula-11"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10/3035/embed/mml-math-12.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10/3035/embed/mml-math-12.gif"/></noscript></span></span> showed expression in the anterior-most region of the head, with characteristic expression at the dorsal tip and around the pharynx. The longer isoform of the transcript was expressed in the anterior-most region of the head but its expression was not seen at the dorsal tip. Weak expression of the longer isoform was seen around the pharyngeal region. The other transcript, dd_smed_V4_3757_0_1 (<em>slmb</em>), has also been reported to be alternatively polyadenylated in <em>Drosophila</em> (<a id="xref-ref-62-2" class="xref-bibr" href="#ref-62">Smibert <em>et al.</em> 2012</a>). The <em>in situ</em> hybridization for dd_smed_V4_3757_0_1 using the probe designed to recognize both the isoforms showed its expression in the pharynx, photoreceptor, and cephalic ganglion, whereas the probe designed to recognize the longer isoform revealed its expression only within the pharyngeal region (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5D</a>). These results demonstrate the tissue-specific expression of ApA isoforms, which implies the role of ApA in tissue organization and function in planaria.</p></div></div><div class="section discussion" id="sec-23"><h2 class="">Discussion</h2><p id="p-46">This study is the first to report genome-scale polyadenylation events in <em>S. mediterranea</em> using poly(A) position profiling (3P-Seq). An in-house computational pipeline was developed to reliably identify the cleavage site and to determine the PAS associated with it. Nearly 14,000 transcripts could be annotated with their 3′UTRs. The analysis presented here was done using the older version of the genome. Recently, a newer version of the Smed genome (SxlV4 and AsxlVs1) was released that contains 15,334 scaffolds in comparison to the 43,294 contigs in the previous SxlV3.1 genome (<a id="xref-ref-47-2" class="xref-bibr" href="#ref-47">Robb <em>et al.</em> 2015</a>). We repeated the analysis using the new genome to determine the extent of overlap of 3P-peaks between the new and old genome. Around 83% of the 3P-peaks determined using the new genome overlapped (≥ 50% overlap) with the previously determined polyadenylation sites using the SxlV3.1 genome (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/FigureS15.pdf">Figure S15</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/TableS8.xls">Table S8</a>), which lends support to our approach of annotating the 3′ UTR from the 3P-seq data.</p><p id="p-47">Our study also identified most of the proteins involved in the cleavage/polyadenylation process with high confidence, except for FIP-1, Pcf11, CFI, and CFII. The best hits obtained for these four genes, along with 12 other genes of the polyadenylation machinery, showed the evidence for their expression in the transcriptome databases (<a id="xref-ref-43-2" class="xref-bibr" href="#ref-43">Onal <em>et al.</em> 2012</a>; <a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Labbé <em>et al.</em> 2012</a>; <a id="xref-ref-46-2" class="xref-bibr" href="#ref-46">Resch <em>et al.</em> 2012</a>). This suggests that they are not merely hypothetical proteins and are actively expressed within the planarian system. Interestingly, the cell type-specific expression data from these transcriptomes revealed the enrichment (&gt; twofold) of almost all the transcripts of the polyadenylation machinery excluding PAPγ (no change), in the proliferating neoblast (X1) when compared to differentiated cells (Xins). Further experiments would be required to decipher the biological significance of the enrichment. In other metazoans, FIP-1 typically binds to U-rich elements (URE) spanning the cleavage/polyadenylated site (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Kaufmann <em>et al.</em> 2004</a>). Though we identified URE in the 3′ UTR regions of the planarians, the inability to identify the FIP-1 homolog could be due to the high sequence divergence, which cannot be detected using standard homology based searches. The proteins CFIm and Pcf11 (<a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Noble <em>et al.</em> 2007</a>), subunits of the CFII complex, recognize a “UGUA” signal in other eukaryotes. In planarians, lack of a “UGUA” signal at the cleavage site in 3′ UTR regions could potentially explain the absence/divergence of CFIm and Pcf11 protein subunits.</p><p id="p-48">In planarians, the canonical hexameric signal “AAUAAA” is the most prevalent polyadenylation signal. We identified several variants of this canonical signal, which were also reported in other metazoans. They function as secondary signals, which were less preferred for polyadenylation compared to the canonical signals (<a id="xref-ref-68-2" class="xref-bibr" href="#ref-68">You <em>et al.</em> 2015</a>). In planarians, for those transcripts with more than one 3P-peak, the distal 3P-peaks were mostly associated with a canonical hexameric signal, whereas the proximal 3P-peaks were associated with the secondary signal variants. This trend was also observed in other metazoans, which supports the hypothesis proposed by <a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Shi (2012</a>) that distal PAS may be recognized as default polyadenylation site by the machinery as it is highly conserved, whereas the proximal PAS could mostly play a regulatory role and is less conserved.</p><p id="p-49">Here, we also showed that &gt; 40% of the planarian transcripts were alternately polyadenylated. We identified transcripts that contained the polyadenylation signal in the internal exon that could lead to altered protein products. One such candidate, <span class="inline-formula" id="inline-formula-12"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10/3035/embed/mml-math-13.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10/3035/embed/mml-math-13.gif"/></noscript></span></span>, was experimentally validated using <em>in situ</em> hybridization. This protein is reported to stimulate the process of tubulin polymerization and also play a key role in α-synuclein aggregation. However, most of the transcripts that undergo ApA have alternate PAS in the 3′ UTR regions, which will result in mRNAs with distinct 3′ UTRs. We validated the presence of altered 3′UTR for <em>slmb</em> (supernumerary limb gene) through <em>in situ</em> hybridization (ISH). The <em>slmb</em> gene is a member of the F-box/WD40 protein family of the ubiquitin ligase SCF complex that targets phosphorylated proteins for degradation. In addition to these transcripts that undergo tissue-specific alternate polyadenylation, we also identified 1606 ApA transcripts that were enriched by at least twofold in the neoblast population by analyzing the previously published RNA-Seq data (<a id="xref-ref-43-3" class="xref-bibr" href="#ref-43">Onal <em>et al.</em> 2012</a>) (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1/TableS10.xls">Table S10</a>). However, the ApA status of these candidates in the neoblast population needs to be validated. This observation suggests that ApA could potentially play an important role in the regulation of gene expression in the neoblast population. The 3′UTR is one of the key regulatory elements that determine the fate of the mRNA within the cell. By altering the length of the 3′ UTR in a given cell type, the transcript may either get destabilized leading to translational repression or undergo active translation, mediated by RNA-binding proteins that bind to the 3′ UTR. The microRNAs, which are translational repressors, have their binding sites on the 3′ UTRs. microRNA binding could be altered by changing the length of the 3′ UTR. In planarians, we observed that miRNA binding sites on the transcripts with two 3P-peaks were either present upstream of the proximal 3P-peak or the distal 3P-peak in a mutually exclusive manner. Together, these results support the observation that miRNA-mediated regulation of gene expression could be altered by ApA events.</p><p id="p-50">In summary, our study demonstrates 3P-Seq as a powerful tool to identify alternate polyadenylation events on a genome-wide scale. Our analysis is one of the first to provide a detailed account of the 3′UTR landscape in planaria. This study will facilitate the exploration of various posttranscriptional regulatory events mediated by the 3′UTRs. More detailed study of polyadenylation events across tissue types and regenerating tissues will provide better insights into the role of ApA in the regulation of tissue homeostasis, stem cell function, and regeneration in planarians. This study will also provide a platform to explore the posttranscriptional regulatory mechanisms mediated through 3′UTRs in planaria.</p></div><div class="section" id="sec-24"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-51">We would like to acknowledge Dr. Karthi Sivaraman for his valuable inputs on computational analyses. High-throughput sequencing was performed at the Next-Generation sequencing facility of the Centre for Cellular Molecular Platform (c-CAMP) and Genotypic Technologies, Bangalore. We thank Palakodeti lab members for proofreading and helpful comments. This work was generously supported by the Wellcome Trust/DBT (Department of Biotechnology) India Alliance in addition to: a Ramanujan Fellowship from Department of Science and Technology, India (to A.S.); a Campus Fellowship Program at the National Centre for Biological Sciences (NCBS) ( to P.A.); a Council of Scientific and Industrial Research (CSIR) Junior research fellowship (to D.B.); and a Wellcome Trust/DBT India Alliance Intermediate Fellowship (to D.P.). Funding from a Wellcome/DBT India Alliance Intermediate fellowship awarded to D.P. supported V.L. and V.S. Funding for open access charge: the publication charges will be covered by the Wellcome Trust/DBT India Alliance Intermediate Fellowship.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-2"><p id="p-52">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.031120/-/DC1</a></p></li><li class="fn" id="fn-3"><p id="p-53"><em>Communicating editor: B. J. Andrews</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-05-12"><span class="received-label">Received </span>May 12, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-07-20"><span class="accepted-label">Accepted </span>July 20, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Lakshmanan <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-2">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.1" data-doi="10.1093/bioinformatics/btn604"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Addo-Quaye</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Axtell</span>  <span class="cit-name-given-names">M. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Cleaveland: a pipeline for using degradome data to find cleaved small RNA targets.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">130</span>–<span class="cit-lpage">131</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtn604%26rft_id%253Dinfo%253Apmid%252F19017659%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czo4OiIyNS8xLzEzMCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTAvMzAzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.2" data-doi="10.1016/0092-8674(80)90615-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Alt</span>  <span class="cit-name-given-names">F. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bothwell</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Knapp</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Siden</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mather</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Synthesis of secreted and membrane-bound immunoglobulin mu heavy chains is directed by mRNAs that differ at their 3′ ends.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">293</span>–<span class="cit-lpage">301</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DAlt%26rft.auinit1%253DF.%2BW.%26rft.volume%253D20%26rft.issue%253D2%26rft.spage%253D293%26rft.epage%253D301%26rft.atitle%253DSynthesis%2Bof%2Bsecreted%2Band%2Bmembrane-bound%2Bimmunoglobulin%2Bmu%2Bheavy%2Bchains%2Bis%2Bdirected%2Bby%2BmRNAs%2Bthat%2Bdiffer%2Bat%2Btheir%2B3%2527%2Bends.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252880%252990615-7%26rft_id%253Dinfo%253Apmid%252F6771018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(80)90615-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6771018&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1980JY00700003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.3" data-doi="10.1006/jmbi.1990.9999"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Altschul</span>  <span class="cit-name-given-names">S. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gish</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Myers</span>  <span class="cit-name-given-names">E. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lipman</span>  <span class="cit-name-given-names">D. J.</span></span></li></ol><cite>, <span class="cit-pub-date">1990</span> <span class="cit-article-title">Basic local alignment search tool.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">215</span>: <span class="cit-fpage">403</span>–<span class="cit-lpage">410</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DAltschul%26rft.auinit1%253DS.%2BF.%26rft.volume%253D215%26rft.issue%253D3%26rft.spage%253D403%26rft.epage%253D410%26rft.atitle%253DBasic%2Blocal%2Balignment%2Bsearch%2Btool.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1990.9999%26rft_id%253Dinfo%253Apmid%252F2231712%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1990.9999&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=2231712&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1990ED16700008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bailey</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Williams</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Misleh</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W. W.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">MEME: discovering and analyzing DNA and protein sequence motifs.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">34</span>: <span class="cit-fpage">W369–373</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft.volume%253D34%26rft.spage%253DW369373%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.5" data-doi="10.1016/S0092-8674(00)80057-4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barabino</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Last but not least: regulated poly(A) tail formation.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">99</span>: <span class="cit-fpage">9</span>–<span class="cit-lpage">11</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DBarabino%26rft.auinit1%253DS.%2BM.%26rft.volume%253D99%26rft.issue%253D1%26rft.spage%253D9%26rft.epage%253D11%26rft.atitle%253DLast%2Bbut%2Bnot%2Bleast%253A%2Bregulated%2Bpoly%2528A%2529%2Btail%2Bformation.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252980057-4%26rft_id%253Dinfo%253Apmid%252F10520989%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)80057-4&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10520989&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000082981600003&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.6" data-doi="10.1007/s00018-012-0990-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Barrett</span>  <span class="cit-name-given-names">L. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fletcher</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilton</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Regulation of eukaryotic gene expression by the untranslated gene regions and other non-coding elements.</span> <abbr class="cit-jnl-abbrev">Cell. Mol. Life Sci.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">3613</span>–<span class="cit-lpage">3634</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCellular%2Band%2Bmolecular%2Blife%2Bsciences%2B%253A%2B%2BCMLS%26rft.stitle%253DCell%2BMol%2BLife%2BSci%26rft.aulast%253DBarrett%26rft.auinit1%253DL.%2BW.%26rft.volume%253D69%26rft.issue%253D21%26rft.spage%253D3613%26rft.epage%253D3634%26rft.atitle%253DRegulation%2Bof%2Beukaryotic%2Bgene%2Bexpression%2Bby%2Bthe%2Buntranslated%2Bgene%2Bregions%2Band%2Bother%2Bnon-coding%2Belements.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00018-012-0990-9%26rft_id%253Dinfo%253Apmid%252F22538991%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00018-012-0990-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22538991&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.7" data-doi="10.1038/nature14321"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Berkovits</span>  <span class="cit-name-given-names">B. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mayr</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Alternative 3′ UTRs act as scaffolds to regulate membrane protein localization.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">522</span>: <span class="cit-fpage">363</span>–<span class="cit-lpage">367</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D522%26rft.spage%253D363%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature14321%26rft_id%253Dinfo%253Apmid%252F25896326%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature14321&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25896326&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.8" data-doi="10.1371/journal.pone.0015617"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blythe</span>  <span class="cit-name-given-names">M. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kao</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Malla</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rowsell</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilson</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A dual platform approach to transcript discovery for the planarian Schmidtea mediterranea to establish RNAseq for stem cell and regeneration biology.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">e15617</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DBlythe%26rft.auinit1%253DM.%2BJ.%26rft.volume%253D5%26rft.issue%253D12%26rft.spage%253De15617%26rft.epage%253De15617%26rft.atitle%253DA%2Bdual%2Bplatform%2Bapproach%2Bto%2Btranscript%2Bdiscovery%2Bfor%2Bthe%2Bplanarian%2BSchmidtea%2Bmediterranea%2Bto%2Bestablish%2BRNAseq%2Bfor%2Bstem%2Bcell%2Band%2Bregeneration%2Bbiology.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0015617%26rft_id%253Dinfo%253Apmid%252F21179477%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0015617&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21179477&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Brandl</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moon</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vila-Farre</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henry</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">PlanMine - a mineable resource of planarian biology and biodiversity.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> 44: D764–D773.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.10" data-doi="10.1006/jmbi.1997.0951"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Burge</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karlin</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Prediction of complete gene structures in human genomic DNA.</span> <abbr class="cit-jnl-abbrev">J. Mol. Biol.</abbr> <span class="cit-vol">268</span>: <span class="cit-fpage">78</span>–<span class="cit-lpage">94</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2Bmolecular%2Bbiology%26rft.stitle%253DJ%2BMol%2BBiol%26rft.aulast%253DBurge%26rft.auinit1%253DC.%26rft.volume%253D268%26rft.issue%253D1%26rft.spage%253D78%26rft.epage%253D94%26rft.atitle%253DPrediction%2Bof%2Bcomplete%2Bgene%2Bstructures%2Bin%2Bhuman%2Bgenomic%2BDNA.%26rft_id%253Dinfo%253Adoi%252F10.1006%252Fjmbi.1997.0951%26rft_id%253Dinfo%253Apmid%252F9149143%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1006/jmbi.1997.0951&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9149143&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1997WV87100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.11" data-doi="10.1101/gr.6743907"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cantarel</span>  <span class="cit-name-given-names">B. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korf</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Robb</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Parra</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">MAKER: an easy-to-use annotation pipeline designed for emerging model organism genomes.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">188</span>–<span class="cit-lpage">196</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.6743907%26rft_id%253Dinfo%253Apmid%252F18025269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjE4LzEvMTg4IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMDM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.12" data-doi="10.1093/nar/gkw164"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Cass</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bahn</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greer</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Global analyses of endonucleolytic cleavage in mammals reveal expanded repertoires of cleavage-inducing small rnas and their targets.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">44</span>: <span class="cit-fpage">3253</span>–<span class="cit-lpage">3263</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkw164%26rft_id%253Dinfo%253Apmid%252F26975654%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjQ0LzcvMzI1MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTAvMzAzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.13" data-doi="10.1101/gad.11.21.2755"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Colgan</span>  <span class="cit-name-given-names">D. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manley</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Mechanism and regulation of mRNA polyadenylation.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">2755</span>–<span class="cit-lpage">2766</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2B%2526%2BDevelopment%26rft.stitle%253DGenes%2BDev.%26rft.aulast%253DColgan%26rft.auinit1%253DD.%2BF.%26rft.volume%253D11%26rft.issue%253D21%26rft.spage%253D2755%26rft.epage%253D2766%26rft.atitle%253DMechanism%2Band%2Bregulation%2Bof%2BmRNA%25A0polyadenylation%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.11.21.2755%26rft_id%253Dinfo%253Apmid%252F9353246%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjExLzIxLzI3NTUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMwMzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.14" data-doi="10.1261/rna.782308"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Davila Lopez</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Samuelsson</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Early evolution of histone mRNA 3′ end processing.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">10</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.782308%26rft_id%253Dinfo%253Apmid%252F17998288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjY6IjE0LzEvMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTAvMzAzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.15" data-doi="10.1101/gr.132563.111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Derti</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garrett-Engele</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Macisaac</span>  <span class="cit-name-given-names">K. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sriram</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A quantitative atlas of polyadenylation in five mammals.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1173</span>–<span class="cit-lpage">1183</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.132563.111%26rft_id%253Dinfo%253Apmid%252F22454233%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjk6IjIyLzYvMTE3MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTAvMzAzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.16" data-doi="10.1016/0092-8674(80)90617-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Early</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calame</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bond</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Two mRNAs can be produced from a single immunoglobulin mu gene by alternative RNA processing pathways.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">313</span>–<span class="cit-lpage">319</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DEarly%26rft.auinit1%253DP.%26rft.volume%253D20%26rft.issue%253D2%26rft.spage%253D313%26rft.epage%253D319%26rft.atitle%253DTwo%2BmRNAs%2Bcan%2Bbe%2Bproduced%2Bfrom%2Ba%2Bsingle%2Bimmunoglobulin%2Bmu%2Bgene%2Bby%2Balternative%2BRNA%2Bprocessing%2Bpathways.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252880%252990617-0%26rft_id%253Dinfo%253Apmid%252F6771020%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(80)90617-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6771020&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1980JY00700005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.17" data-doi="10.1016/j.cell.2012.05.017"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elkayam</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">C.-D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tocilj</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Haase</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greene</span>  <span class="cit-name-given-names">E. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The structure of human argonaute-2 in complex with mir-20a.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">150</span>: <span class="cit-fpage">100</span>–<span class="cit-lpage">110</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DElkayam%26rft.auinit1%253DE.%26rft.volume%253D150%26rft.issue%253D1%26rft.spage%253D100%26rft.epage%253D110%26rft.atitle%253DThe%2Bstructure%2Bof%2Bhuman%2Bargonaute-2%2Bin%2Bcomplex%2Bwith%2BmiR-20a.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2012.05.017%26rft_id%253Dinfo%253Apmid%252F22682761%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2012.05.017&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22682761&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000306115000010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.18" data-doi="10.1038/nrg3482"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elkon</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ugalde</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agami</span>  <span class="cit-name-given-names">R.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Alternative cleavage and polyadenylation: extent, regulation and function.</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">496</span>–<span class="cit-lpage">506</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BRev.%2BGenet.%26rft.volume%253D14%26rft.spage%253D496%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnrg3482%26rft_id%253Dinfo%253Apmid%252F23774734%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nrg3482&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23774734&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.19" data-doi="10.1146/annurev.neuro.31.060407.125631"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Flavell</span>  <span class="cit-name-given-names">S. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenberg</span>  <span class="cit-name-given-names">M. E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Signaling mechanisms linking neuronal activity to gene expression and plasticity of the nervous system.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Neurosci.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">563</span>–<span class="cit-lpage">590</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bneuroscience%26rft.stitle%253DAnnu%2BRev%2BNeurosci%26rft.aulast%253DFlavell%26rft.auinit1%253DS.%2BW.%26rft.volume%253D31%26rft.spage%253D563%26rft.epage%253D590%26rft.atitle%253DSignaling%2Bmechanisms%2Blinking%2Bneuronal%2Bactivity%2Bto%2Bgene%2Bexpression%2Band%2Bplasticity%2Bof%2Bthe%2Bnervous%2Bsystem.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev.neuro.31.060407.125631%26rft_id%253Dinfo%253Apmid%252F18558867%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev.neuro.31.060407.125631&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18558867&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257992200023&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.20" data-doi="10.1101/gr.115295.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rao</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Differential genome-wide profiling of tandem 3′ UTRs among human breast cancer and normal cells by high-throughput sequencing.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">741</span>–<span class="cit-lpage">747</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.115295.110%26rft_id%253Dinfo%253Apmid%252F21474764%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzUvNzQxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMDM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.21" data-doi="10.1038/nbt1417"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">German</span>  <span class="cit-name-given-names">M. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pillay</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jeong</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hetawal</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Luo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Global identification of microRNA-target RNA pairs by parallel analysis of RNA ends.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">941</span>–<span class="cit-lpage">946</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DGerman%26rft.auinit1%253DM.%2BA.%26rft.volume%253D26%26rft.issue%253D8%26rft.spage%253D941%26rft.epage%253D946%26rft.atitle%253DGlobal%2Bidentification%2Bof%2BmicroRNA-target%2BRNA%2Bpairs%2Bby%2Bparallel%2Banalysis%2Bof%2BRNA%2Bends.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt1417%26rft_id%253Dinfo%253Apmid%252F18542052%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt1417&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18542052&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000258325500029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.22" data-doi="10.1093/nar/gkm952"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Griffiths-Jones</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Saini</span>  <span class="cit-name-given-names">H. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van Dongen</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Enright</span>  <span class="cit-name-given-names">A. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">miRBase: tools for microRNA genomics.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">36</span>: <span class="cit-fpage">D154</span>–<span class="cit-lpage">D158</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkm952%26rft_id%253Dinfo%253Apmid%252F17991681%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjE1OiIzNi9zdXBwbF8xL0QxNTQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMwMzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.23" data-doi="10.1073/pnas.1112672108"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hilgers</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Perry</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hendrix</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stark</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Levine</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Neural-specific elongation of 3′ UTRs during Drosophila development.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">108</span>: <span class="cit-fpage">15864</span>–<span class="cit-lpage">15869</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1112672108%26rft_id%253Dinfo%253Apmid%252F21896737%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTA4LzM4LzE1ODY0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMDM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.24" data-doi="10.4161/epi.23211"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hubert</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henderson</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross</span>  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cowles</span>  <span class="cit-name-given-names">M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Torres</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Epigenetic regulation of planarian stem cells by the SET1/MLL family of histone methyltransferases.</span> <abbr class="cit-jnl-abbrev">Epigenetics</abbr> <span class="cit-vol">8</span>: <span class="cit-fpage">79</span>–<span class="cit-lpage">91</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEpigenetics%26rft.volume%253D8%26rft.spage%253D79%26rft_id%253Dinfo%253Adoi%252F10.4161%252Fepi.23211%26rft_id%253Dinfo%253Apmid%252F23235145%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.4161/epi.23211&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23235145&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000313282900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.25" data-doi="10.1186/1471-2105-11-24"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Huerta-Cepas</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dopazo</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gabaldon</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">ETE: a python Environment for Tree Exploration.</span> <abbr class="cit-jnl-abbrev">BMC Bioinformatics</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">24</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bbioinformatics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BBioinformatics%26rft.aulast%253DHuerta-Cepas%26rft.auinit1%253DJ.%26rft.volume%253D11%26rft.spage%253D24%26rft.epage%253D24%26rft.atitle%253DETE%253A%2Ba%2Bpython%2BEnvironment%2Bfor%2BTree%2BExploration.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2105-11-24%26rft_id%253Dinfo%253Apmid%252F20070885%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2105-11-24&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20070885&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.26" data-doi="10.1038/nature09616"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jan</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">R. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ruby</span>  <span class="cit-name-given-names">J. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartel</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Formation, regulation and evolution of Caenorhabditis elegans 3′UTRs.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">469</span>: <span class="cit-fpage">97</span>–<span class="cit-lpage">101</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DJan%26rft.auinit1%253DC.%2BH.%26rft.volume%253D469%26rft.issue%253D7328%26rft.spage%253D97%26rft.epage%253D101%26rft.atitle%253DFormation%252C%2Bregulation%2Band%2Bevolution%2Bof%2BCaenorhabditis%2Belegans%2B3%2527UTRs.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature09616%26rft_id%253Dinfo%253Apmid%252F21085120%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature09616&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21085120&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000285921600039&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.27" data-doi="10.1371/journal.pone.0008419"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Reprogramming of 3′ untranslated regions of mRNAs by alternative polyadenylation in generation of pluripotent stem cells from different cell types.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">e8419</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DJi%26rft.auinit1%253DZ.%26rft.volume%253D4%26rft.issue%253D12%26rft.spage%253De8419%26rft.epage%253De8419%26rft.atitle%253DReprogramming%2Bof%2B3%2527%2Buntranslated%2Bregions%2Bof%2BmRNAs%2Bby%2Balternative%2Bpolyadenylation%2Bin%2Bgeneration%2Bof%2Bpluripotent%2Bstem%2Bcells%2Bfrom%2Bdifferent%2Bcell%2Btypes.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0008419%26rft_id%253Dinfo%253Apmid%252F20037631%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0008419&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20037631&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.28" data-doi="10.1073/pnas.0900028106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ji</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lee</span>  <span class="cit-name-given-names">J. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pan</span>  <span class="cit-name-given-names">Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">B.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Progressive lengthening of 3′ untranslated regions of mRNAs by alternative polyadenylation during mouse embryonic development.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">7028</span>–<span class="cit-lpage">7033</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0900028106%26rft_id%253Dinfo%253Apmid%252F19372383%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA2LzE3LzcwMjgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMwMzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.29" data-doi="10.1016/j.gde.2012.12.004"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jia</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yao</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arif</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fox</span>  <span class="cit-name-given-names">P. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Regulation and dysregulation of 3′UTR-mediated translational control.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">29</span>–<span class="cit-lpage">34</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BGenet.%2BDev.%26rft.volume%253D23%26rft.spage%253D29%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.gde.2012.12.004%26rft_id%253Dinfo%253Apmid%252F23312843%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.gde.2012.12.004&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23312843&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.30" data-doi="10.1371/journal.pbio.0020363"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">John</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Enright</span>  <span class="cit-name-given-names">A. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aravin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tuschl</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sander</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Human MicroRNA targets.</span> <abbr class="cit-jnl-abbrev">PLoS Biol.</abbr> <span class="cit-vol">2</span>: <span class="cit-fpage">e363</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2Bbiology%26rft.stitle%253DPLoS%2BBiol%26rft.aulast%253DJohn%26rft.auinit1%253DB.%26rft.volume%253D2%26rft.issue%253D11%26rft.spage%253De363%26rft.epage%253De363%26rft.atitle%253DHuman%2BMicroRNA%2Btargets.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pbio.0020363%26rft_id%253Dinfo%253Apmid%252F15502875%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pbio.0020363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=15502875&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.31" data-doi="10.1016/j.molcel.2010.06.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Karginov</span>  <span class="cit-name-given-names">F. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cheloufi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chong</span>  <span class="cit-name-given-names">M. M. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stark</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Smith</span>  <span class="cit-name-given-names">A. D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Diverse endonucleolytic cleavage sites in the mammalian transcriptome depend upon microRNAs, Drosha, and additional nucleases.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">781</span>–<span class="cit-lpage">788</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DKarginov%26rft.auinit1%253DF.%2BV.%26rft.volume%253D38%26rft.issue%253D6%26rft.spage%253D781%26rft.epage%253D788%26rft.atitle%253DDiverse%2Bendonucleolytic%2Bcleavage%2Bsites%2Bin%2Bthe%2Bmammalian%2Btranscriptome%2Bdepend%2Bupon%2BmicroRNAs%252C%2BDrosha%252C%2Band%2Badditional%2Bnucleases.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2010.06.001%26rft_id%253Dinfo%253Apmid%252F20620951%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2010.06.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20620951&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279297500004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.32" data-doi="10.1038/sj.emboj.7600070"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kaufmann</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedlein</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langen</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Keller</span>  <span class="cit-name-given-names">W.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Human Fip1 is a subunit of CPSF that binds to U-rich RNA elements and stimulates poly(A) polymerase.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">23</span>: <span class="cit-fpage">616</span>–<span class="cit-lpage">626</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BEMBO%2BJournal%26rft.stitle%253DEMBO%2BJ.%26rft.aulast%253DKaufmann%26rft.auinit1%253DI.%26rft.volume%253D23%26rft.issue%253D3%26rft.spage%253D616%26rft.epage%253D626%26rft.atitle%253DHuman%2BFip1%2Bis%2Ba%2Bsubunit%2Bof%2BCPSF%2Bthat%2Bbinds%2Bto%2BU-rich%2BRNA%2Belements%2Band%2Bstimulates%2Bpoly%2528A%2529%2Bpolymerase%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fsj.emboj.7600070%26rft_id%253Dinfo%253Apmid%252F14749727%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czo4OiIyMy8zLzYxNiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTAvMzAzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.33" data-doi="10.1002/stem.1144"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Labbé</span>  <span class="cit-name-given-names">R. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Irimia</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Currie</span>  <span class="cit-name-given-names">K. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lin</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhu</span>  <span class="cit-name-given-names">S. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A comparative transcriptomic analysis reveals conserved features of stem cell pluripotency in planarians and mammals.</span> <abbr class="cit-jnl-abbrev">Stem Cells</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1734</span>–<span class="cit-lpage">1745</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DStem%2BCells%26rft.stitle%253DStem%2BCells%26rft.aulast%253DLabbe%26rft.auinit1%253DR.%2BM.%26rft.volume%253D30%26rft.issue%253D8%26rft.spage%253D1734%26rft.epage%253D1745%26rft.atitle%253DA%2Bcomparative%2Btranscriptomic%2Banalysis%2Breveals%2Bconserved%2Bfeatures%2Bof%2Bstem%2Bcell%2Bpluripotency%2Bin%2Bplanarians%2Band%2Bmammals.%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fstem.1144%26rft_id%253Dinfo%253Apmid%252F22696458%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/stem.1144&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22696458&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.34" data-doi="10.1186/gb-2009-10-3-r25"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langmead</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Trapnell</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pop</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salzberg</span>  <span class="cit-name-given-names">S. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">10</span>: <span class="cit-fpage">R25</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2Bbiology%26rft.stitle%253DGenome%2BBiol%26rft.aulast%253DLangmead%26rft.auinit1%253DB.%26rft.volume%253D10%26rft.issue%253D3%26rft.spage%253DR25%26rft.epage%253DR25%26rft.atitle%253DUltrafast%2Band%2Bmemory-efficient%2Balignment%2Bof%2Bshort%2BDNA%2Bsequences%2Bto%2Bthe%2Bhuman%2Bgenome.%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2009-10-3-r25%26rft_id%253Dinfo%253Apmid%252F19261174%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2009-10-3-r25&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19261174&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.35" data-doi="10.1093/bioinformatics/btl158"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Godzik</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1658</span>–<span class="cit-lpage">1659</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtl158%26rft_id%253Dinfo%253Apmid%252F16731699%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMjIvMTMvMTY1OCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTAvMzAzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.36" data-doi="10.1038/nature12414"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">S. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selck</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedrich</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lutz</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vila-Farre</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Reactivating head regrowth in a regeneration-deficient planarian species.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">500</span>: <span class="cit-fpage">81</span>–<span class="cit-lpage">84</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.volume%253D500%26rft.spage%253D81%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature12414%26rft_id%253Dinfo%253Apmid%252F23883932%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature12414&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23883932&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.37" data-doi="10.1038/nature05363"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mandel</span>  <span class="cit-name-given-names">C. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kaneko</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gebauer</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vethantham</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Polyadenylation factor CPSF-73 is the pre-mRNA 3′-end-processing endonuclease.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">444</span>: <span class="cit-fpage">953</span>–<span class="cit-lpage">956</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMandel%26rft.auinit1%253DC.%2BR.%26rft.volume%253D444%26rft.issue%253D7121%26rft.spage%253D953%26rft.epage%253D956%26rft.atitle%253DPolyadenylation%2Bfactor%2BCPSF-73%2Bis%2Bthe%2Bpre-mRNA%2B3%2527-end-processing%2Bendonuclease.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature05363%26rft_id%253Dinfo%253Apmid%252F17128255%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature05363&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17128255&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000242805400062&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.38" data-doi="10.1126/science.1191244"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mangone</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Manoharan</span>  <span class="cit-name-given-names">A. P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thierry-Mieg</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thierry-Mieg</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">The landscape of C. elegans 3′UTRs.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">329</span>: <span class="cit-fpage">432</span>–<span class="cit-lpage">435</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1191244%26rft_id%253Dinfo%253Apmid%252F20522740%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIzMjkvNTk5MC80MzIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMwMzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.39" data-doi="10.1016/j.cell.2009.06.016"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mayr</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartel</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Widespread shortening of 3′UTRs by alternative cleavage and polyadenylation activates oncogenes in cancer cells.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">138</span>: <span class="cit-fpage">673</span>–<span class="cit-lpage">684</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DMayr%26rft.auinit1%253DC.%26rft.volume%253D138%26rft.issue%253D4%26rft.spage%253D673%26rft.epage%253D684%26rft.atitle%253DWidespread%2Bshortening%2Bof%2B3%2527UTRs%2Bby%2Balternative%2Bcleavage%2Band%2Bpolyadenylation%2Bactivates%2Boncogenes%2Bin%2Bcancer%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2009.06.016%26rft_id%253Dinfo%253Apmid%252F19703394%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2009.06.016&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=19703394&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000269156100010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.40" data-doi="10.1093/nar/gkp1176"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Millevoi</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vagner</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Molecular mechanisms of eukaryotic pre-mRNA 3′ end processing regulation.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">2757</span>–<span class="cit-lpage">2774</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkp1176%26rft_id%253Dinfo%253Apmid%252F20044349%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjM4LzkvMjc1NyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTAvMzAzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.41" data-doi="10.1101/gr.162503.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moran</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fredman</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Praher</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">X. Z.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wee</span>  <span class="cit-name-given-names">L. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Cnidarian microRNAs frequently regulate targets by cleavage.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">651</span>–<span class="cit-lpage">663</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.162503.113%26rft_id%253Dinfo%253Apmid%252F24642861%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjI0LzQvNjUxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMDM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.42" data-doi="10.1093/nar/gkl1010"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Noble</span>  <span class="cit-name-given-names">C. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Beuth</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">I. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Structure of a nucleotide-bound Clp1-Pcf11 polyadenylation factor.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">35</span>: <span class="cit-fpage">87</span>–<span class="cit-lpage">99</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgkl1010%26rft_id%253Dinfo%253Apmid%252F17151076%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjc6IjM1LzEvODciO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMwMzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.43" data-doi="10.1038/emboj.2012.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Onal</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grun</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adamidi</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rybak</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Solana</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Gene expression of pluripotency determinants is conserved between mammalian and planarian stem cells.</span> <abbr class="cit-jnl-abbrev">EMBO J.</abbr> <span class="cit-vol">31</span>: <span class="cit-fpage">2755</span>–<span class="cit-lpage">2769</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEMBO%2BJ.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Femboj.2012.110%26rft_id%253Dinfo%253Apmid%252F22543868%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZW1ib2pubCI7czo1OiJyZXNpZCI7czoxMDoiMzEvMTIvMjc1NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTAvMzAzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-web no-rev-xref" id="cit-6.10.3035.44"><div class="cit-metadata unstructured">Oviedo, N. J., C. L. Nicolas, D. S. Adams, and M. Levin, 2008 Planarians: a versatile and powerful model system for molecular studies of regeneration, adult stem cell regulation, aging, and behavior. Cold Spring Harb. Protoc. 2008. Available at: http://cshprotocols.cshlp.org/content/2008/10/pdb.emo101.long.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.45" data-doi="10.1242/dev.044297"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pearson</span>  <span class="cit-name-given-names">B. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez Alvarado</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">A planarian p53 homolog regulates proliferation and self-renewal in adult stem cell lineages.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">137</span>: <span class="cit-fpage">213</span>–<span class="cit-lpage">221</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DPearson%26rft.auinit1%253DB.%2BJ.%26rft.volume%253D137%26rft.issue%253D2%26rft.spage%253D213%26rft.epage%253D221%26rft.atitle%253DA%2Bplanarian%2Bp53%2Bhomolog%2Bregulates%2Bproliferation%2Band%2Bself-renewal%2Bin%2Badult%2Bstem%2Bcell%2Blineages%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.044297%26rft_id%253Dinfo%253Apmid%252F20040488%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czo5OiIxMzcvMi8yMTMiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMwMzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.46" data-doi="10.1371/journal.pone.0034447"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Resch</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Palakodeti</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Horowitz</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Graveley</span>  <span class="cit-name-given-names">B. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Transcriptome analysis reveals strain-specific and conserved stemness genes in Schmidtea mediterranea.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e34447</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DResch%26rft.auinit1%253DA.%2BM.%26rft.volume%253D7%26rft.issue%253D4%26rft.spage%253De34447%26rft.epage%253De34447%26rft.atitle%253DTranscriptome%2Banalysis%2Breveals%2Bstrain-specific%2Band%2Bconserved%2Bstemness%2Bgenes%2Bin%2BSchmidtea%2Bmediterranea.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0034447%26rft_id%253Dinfo%253Apmid%252F22496805%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0034447&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22496805&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.47" data-doi="10.1002/dvg.22872"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Robb</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gotting</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ross</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez Alvarado</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">SmedGD 2.0: The Schmidtea mediterranea genome database.</span> <abbr class="cit-jnl-abbrev">Genesis</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">535</span>–<span class="cit-lpage">546</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenesis%26rft.volume%253D53%26rft.spage%253D535%26rft_id%253Dinfo%253Adoi%252F10.1002%252Fdvg.22872%26rft_id%253Dinfo%253Apmid%252F26138588%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1002/dvg.22872&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=26138588&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.48" data-doi="10.1016/0092-8674(80)90616-9"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rogers</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Early</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carter</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Calame</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bond</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1980</span> <span class="cit-article-title">Two mRNAs with different 3′ ends encode membrane-bound and secreted forms of immunoglobulin mu chain.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">303</span>–<span class="cit-lpage">312</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DRogers%26rft.auinit1%253DJ.%26rft.volume%253D20%26rft.issue%253D2%26rft.spage%253D303%26rft.epage%253D312%26rft.atitle%253DTwo%2BmRNAs%2Bwith%2Bdifferent%2B3%2527%2Bends%2Bencode%2Bmembrane-bound%2Band%2Bsecreted%2Bforms%2Bof%2Bimmunoglobulin%2Bmu%2Bchain.%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0092-8674%252880%252990616-9%26rft_id%253Dinfo%253Apmid%252F6771019%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0092-8674(80)90616-9&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=6771019&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=A1980JY00700004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.49" data-doi="10.1007/s00018-007-7426-y"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rossi</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Salvetti</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Batistoni</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Deri</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gremigni</span>  <span class="cit-name-given-names">V.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Planarians, a tale of stem cells.</span> <abbr class="cit-jnl-abbrev">Cell. Mol. Life Sci.</abbr> <span class="cit-vol">65</span>: <span class="cit-fpage">16</span>–<span class="cit-lpage">23</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCellular%2Band%2Bmolecular%2Blife%2Bsciences%2B%253A%2B%2BCMLS%26rft.stitle%253DCell%2BMol%2BLife%2BSci%26rft.aulast%253DRossi%26rft.auinit1%253DL.%26rft.volume%253D65%26rft.issue%253D1%26rft.spage%253D16%26rft.epage%253D23%26rft.atitle%253DPlanarians%252C%2Ba%2Btale%2Bof%2Bstem%2Bcells.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00018-007-7426-y%26rft_id%253Dinfo%253Apmid%252F18030424%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00018-007-7426-y&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18030424&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.50" data-doi="10.1038/nature10242"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rothberg</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hinz</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rearick</span>  <span class="cit-name-given-names">T. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schultz</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mileski</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">An integrated semiconductor device enabling non-optical genome sequencing.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">475</span>: <span class="cit-fpage">348</span>–<span class="cit-lpage">352</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DRothberg%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D475%26rft.issue%253D7356%26rft.spage%253D348%26rft.epage%253D352%26rft.atitle%253DAn%2Bintegrated%2Bsemiconductor%2Bdevice%2Benabling%2Bnon-optical%2Bgenome%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature10242%26rft_id%253Dinfo%253Apmid%252F21776081%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature10242&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21776081&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000292911200036&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.51"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rouhana</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shibata</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nishimura</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Agata</span>  <span class="cit-name-given-names">K.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> Different requirements for conserved post-transcriptional regulators in planarian regeneration and stem cell maintenance. Dev. Biol. 341: 429–443.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.52" data-doi="10.1242/dev.101618"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rouhana</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Weiss</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">King</span>  <span class="cit-name-given-names">R. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Newmark</span>  <span class="cit-name-given-names">P. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">PIWI homologs mediate histone H4 mRNA localization to planarian chromatoid bodies.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">141</span>: <span class="cit-fpage">2592</span>–<span class="cit-lpage">2601</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.101618%26rft_id%253Dinfo%253Apmid%252F24903754%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTQxLzEzLzI1OTIiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMwMzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-53-1" title="View reference  in text" id="ref-53">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.53" data-doi="10.1016/S0959-437X(03)00082-0"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sanchez Alvarado</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The freshwater planarian Schmidtea mediterranea: embryogenesis, stem cells and regeneration.</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Genet. Dev.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">438</span>–<span class="cit-lpage">444</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bopinion%2Bin%2Bgenetics%2B%2526%2Bdevelopment%26rft.stitle%253DCurr%2BOpin%2BGenet%2BDev%26rft.aulast%253DS%25C3%25A1nchez%2BAlvarado%26rft.auinit1%253DA.%26rft.volume%253D13%26rft.issue%253D4%26rft.spage%253D438%26rft.epage%253D444%26rft.atitle%253DThe%2Bfreshwater%2Bplanarian%2BSchmidtea%2Bmediterranea%253A%2Bembryogenesis%252C%2Bstem%2Bcells%2Band%2Bregeneration.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0959-437X%252803%252900082-0%26rft_id%253Dinfo%253Apmid%252F12888018%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0959-437X(03)00082-0&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12888018&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.54" data-doi="10.1126/science.1155390"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sandberg</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Neilson</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sarma</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sharp</span>  <span class="cit-name-given-names">P. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Burge</span>  <span class="cit-name-given-names">C. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Proliferating cells express mRNAs with shortened 3′ untranslated regions and fewer microRNA target sites.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">320</span>: <span class="cit-fpage">1643</span>–<span class="cit-lpage">1647</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSandberg%26rft.auinit1%253DR.%26rft.volume%253D320%26rft.issue%253D5883%26rft.spage%253D1643%26rft.epage%253D1647%26rft.atitle%253DProliferating%2BCells%2BExpress%2BmRNAs%2Bwith%2BShortened%2B3%2527%2BUntranslated%2BRegions%2Band%2BFewer%2BMicroRNA%2BTarget%2BSites%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1155390%26rft_id%253Dinfo%253Apmid%252F18566288%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMjAvNTg4My8xNjQzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMDM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.55" data-doi="10.1261/rna.038653.113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sasidharan</span>  <span class="cit-name-given-names">V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">Y. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bansal</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dasari</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poduval</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Identification of neoblast- and regeneration-specific miRNAs in the planarian Schmidtea mediterranea.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">19</span>: <span class="cit-fpage">1394</span>–<span class="cit-lpage">1404</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.038653.113%26rft_id%253Dinfo%253Apmid%252F23974438%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjEwOiIxOS8xMC8xMzk0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMDM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.56" data-doi="10.1101/gad.250985.114"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schonemann</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhn</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martin</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schafer</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gruber</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Reconstitution of CPSF active in polyadenylation: recognition of the polyadenylation signal by WDR33.</span> <abbr class="cit-jnl-abbrev">Genes Dev.</abbr> <span class="cit-vol">28</span>: <span class="cit-fpage">2381</span>–<span class="cit-lpage">2393</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenes%2BDev.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgad.250985.114%26rft_id%253Dinfo%253Apmid%252F25301781%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXNkZXYiO3M6NToicmVzaWQiO3M6MTA6IjI4LzIxLzIzODEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMwMzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.57" data-doi="10.1261/rna.2581711"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shepard</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">E. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Flanagan</span>  <span class="cit-name-given-names">L. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hertel</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Complex and dynamic landscape of RNA polyadenylation revealed by PAS-Seq.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">17</span>: <span class="cit-fpage">761</span>–<span class="cit-lpage">772</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.2581711%26rft_id%253Dinfo%253Apmid%252F21343387%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjg6IjE3LzQvNzYxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMDM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.58" data-doi="10.1261/rna.035899.112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shi</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Alternative polyadenylation: new insights from global analyses.</span> <abbr class="cit-jnl-abbrev">RNA</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">2105</span>–<span class="cit-lpage">2117</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DRNA%26rft_id%253Dinfo%253Adoi%252F10.1261%252Frna.035899.112%26rft_id%253Dinfo%253Apmid%252F23097429%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoicm5hIjtzOjU6InJlc2lkIjtzOjEwOiIxOC8xMi8yMTA1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMDM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.59" data-doi="10.1016/j.molcel.2010.06.005"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Shin</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nam</span>  <span class="cit-name-given-names">J.-W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farh</span>  <span class="cit-name-given-names">K. K.-H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiang</span>  <span class="cit-name-given-names">H. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shkumatava</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Expanding the microRNA targeting code: functional sites with centered pairing.</span> <abbr class="cit-jnl-abbrev">Mol. Cell</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">789</span>–<span class="cit-lpage">802</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2Bcell%26rft.stitle%253DMol%2BCell%26rft.aulast%253DShin%26rft.auinit1%253DC.%26rft.volume%253D38%26rft.issue%253D6%26rft.spage%253D789%26rft.epage%253D802%26rft.atitle%253DExpanding%2Bthe%2BmicroRNA%2Btargeting%2Bcode%253A%2Bfunctional%2Bsites%2Bwith%2Bcentered%2Bpairing.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.molcel.2010.06.005%26rft_id%253Dinfo%253Apmid%252F20620952%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.molcel.2010.06.005&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20620952&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000279297500005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.60" data-doi="10.1038/msb.2011.75"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sievers</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wilm</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dineen</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibson</span>  <span class="cit-name-given-names">T. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Karplus</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Fast, scalable generation of high-quality protein multiple sequence alignments using Clustal Omega.</span> <abbr class="cit-jnl-abbrev">Mol. Syst. Biol.</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">539</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMolecular%2BSystems%2BBiology%26rft.stitle%253DMol%2BSyst%2BBiol%26rft.aulast%253DSievers%26rft.auinit1%253DF.%26rft.volume%253D7%26rft.issue%253D1%26rft.spage%253D539%26rft.epage%253D539%26rft.atitle%253DFast%252C%2Bscalable%2Bgeneration%2Bof%2Bhigh-quality%2Bprotein%2Bmultiple%2Bsequence%2Balignments%2Busing%2BClustal%2BOmega%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fmsb.2011.75%26rft_id%253Dinfo%253Apmid%252F21988835%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibXNiIjtzOjU6InJlc2lkIjtzOjc6IjcvMS81MzkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMwMzUuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.61" data-doi="10.1158/0008-5472.CAN-09-2236"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Singh</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alley</span>  <span class="cit-name-given-names">T. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">S. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kamdar</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schott</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Global changes in processing of mRNA 3′ untranslated regions characterize clinically distinct cancer subtypes.</span> <abbr class="cit-jnl-abbrev">Cancer Res.</abbr> <span class="cit-vol">69</span>: <span class="cit-fpage">9422</span>–<span class="cit-lpage">9430</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCancer%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1158%252F0008-5472.CAN-09-2236%26rft_id%253Dinfo%253Apmid%252F19934316%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiY2FucmVzIjtzOjU6InJlc2lkIjtzOjEwOiI2OS8yNC85NDIyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMDM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.62" data-doi="10.1016/j.celrep.2012.01.001"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Smibert</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miura</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Westholm</span>  <span class="cit-name-given-names">J. O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shenker</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">May</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Global patterns of tissue-specific alternative polyadenylation in Drosophila.</span> <abbr class="cit-jnl-abbrev">Cell Reports</abbr> <span class="cit-vol">1</span>: <span class="cit-fpage">277</span>–<span class="cit-lpage">289</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%2BReports%26rft.volume%253D1%26rft.spage%253D277%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.celrep.2012.01.001%26rft_id%253Dinfo%253Apmid%252F22685694%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.celrep.2012.01.001&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22685694&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.63" data-doi="10.1371/journal.pgen.1004003"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Solana</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gamberi</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mihaylova</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grosswendt</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">The CCR4-NOT complex mediates deadenylation and degradation of stem cell mRNAs and promotes planarian stem cell differentiation.</span> <abbr class="cit-jnl-abbrev">PLoS Genet.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1004003</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLoS%2BGenet.%26rft.volume%253D9%26rft.spage%253De1004003%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pgen.1004003%26rft_id%253Dinfo%253Apmid%252F24367277%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pgen.1004003&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=24367277&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.64" data-doi="10.1093/jmcb/mjs041"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Xu</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Genome-wide alternative polyadenylation in animals: insights from high-throughput technologies.</span> <abbr class="cit-jnl-abbrev">J. Mol. Cell Biol.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">352</span>–<span class="cit-lpage">361</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BMol.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjmcb%252Fmjs041%26rft_id%253Dinfo%253Apmid%252F23099521%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoiam1jYiI7czo1OiJyZXNpZCI7czo3OiI0LzYvMzUyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMDM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.65" data-doi="10.1093/nar/gki158"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lutz</span>  <span class="cit-name-given-names">C. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">A large-scale analysis of mRNA polyadenylation of human and mouse genes.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">201</span>–<span class="cit-lpage">212</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BResearch%26rft.stitle%253DNucleic%2BAcids%2BRes%26rft.aulast%253DTian%26rft.auinit1%253DB.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D201%26rft.epage%253D212%26rft.atitle%253DA%2Blarge-scale%2Banalysis%2Bof%2BmRNA%2Bpolyadenylation%2Bof%2Bhuman%2Band%2Bmouse%2Bgenes%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgki158%26rft_id%253Dinfo%253Apmid%252F15647503%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjg6IjMzLzEvMjAxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMDM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.66" data-doi="10.1016/j.cell.2011.11.055"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ulitsky</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Shkumatava</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jan</span>  <span class="cit-name-given-names">C. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sive</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bartel</span>  <span class="cit-name-given-names">D. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Conserved function of lincRNAs in vertebrate embryonic development despite rapid sequence evolution.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">147</span>: <span class="cit-fpage">1537</span>–<span class="cit-lpage">1550</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DUlitsky%26rft.auinit1%253DI.%26rft.volume%253D147%26rft.issue%253D7%26rft.spage%253D1537%26rft.epage%253D1550%26rft.atitle%253DConserved%2Bfunction%2Bof%2BlincRNAs%2Bin%2Bvertebrate%2Bembryonic%2Bdevelopment%2Bdespite%2Brapid%2Bsequence%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2011.11.055%26rft_id%253Dinfo%253Apmid%252F22196729%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2011.11.055&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22196729&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3035.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000298403400018&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.67" data-doi="10.1093/bib/bbr078"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Westesson</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Skinner</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Holmes</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Visualizing next-generation sequencing data with JBrowse.</span> <abbr class="cit-jnl-abbrev">Brief. Bioinform.</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">172</span>–<span class="cit-lpage">177</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBrief.%2BBioinform.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbib%252Fbbr078%26rft_id%253Dinfo%253Apmid%252F22411711%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiYmliIjtzOjU6InJlc2lkIjtzOjg6IjE0LzIvMTcyIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMDM1LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.68" data-doi="10.1093/nar/gku1076"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">You</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fu</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Apasdb: a database describing alternative poly(a) sites and selection of heterogeneous cleavage sites downstream of poly(a) signals.</span> <abbr class="cit-jnl-abbrev">Nucleic Acids Res.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">D59</span>–<span class="cit-lpage">D67</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNucleic%2BAcids%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fnar%252Fgku1076%26rft_id%253Dinfo%253Apmid%252F25378337%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoibmFyIjtzOjU6InJlc2lkIjtzOjk6IjQzL0QxL0Q1OSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTAvMzAzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.69" data-doi="10.1083/jcb.201207172"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zeng</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">Y. Q.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Han</span>  <span class="cit-name-given-names">X. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Li</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Heterochromatin protein 1 promotes self-renewal and triggers regenerative proliferation in adult stem cells.</span> <abbr class="cit-jnl-abbrev">J. Cell Biol.</abbr> <span class="cit-vol">201</span>: <span class="cit-fpage">409</span>–<span class="cit-lpage">425</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BCell%2BBiol.%26rft_id%253Dinfo%253Adoi%252F10.1083%252Fjcb.201207172%26rft_id%253Dinfo%253Apmid%252F23629965%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MzoiamNiIjtzOjU6InJlc2lkIjtzOjk6IjIwMS8zLzQwOSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTAvMzAzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3035.70"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hyman</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Formation of mRNA 3′ ends in eukaryotes: mechanism, regulation, and interrelationships with other steps in mRNA synthesis.</span> <abbr class="cit-jnl-abbrev">Microbiol. Mol. Biol. Rev.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">405</span>–<span class="cit-lpage">445</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DMicrobiology%2Band%2BMolecular%2BBiology%2BReviews%26rft.stitle%253DMicrobiol.%2BMol.%2BBiol.%2BRev.%26rft.aulast%253DZhao%26rft.auinit1%253DJ.%26rft.volume%253D63%26rft.issue%253D2%26rft.spage%253D405%26rft.epage%253D445%26rft.atitle%253DFormation%2Bof%2BmRNA%2B3%2527%2BEnds%2Bin%2BEukaryotes%253A%2BMechanism%252C%2BRegulation%252C%2Band%2BInterrelationships%2Bwith%2BOther%2BSteps%2Bin%2BmRNA%2BSynthesis%26rft_id%253Dinfo%253Apmid%252F10357856%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoibW1iciI7czo1OiJyZXNpZCI7czo4OiI2My8yLzQwNSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTAvMzAzNS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/10/3035.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/10/3027" title="Tandem Duplication Events in the Expansion of the Small Heat Shock Protein Gene Family in Solanum lycopersicum (cv. Heinz 1706)" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/10/3049" title="Characterization of a Novel MMS-Sensitive Allele of Schizosaccharomyces pombe mcm4+" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/10"><span>Volume 6 Issue 10, October 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/10.cover-source.jpg?itok=eRzcKCYf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/10/3035" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-D8kbZ2VvkvVWIXczHYXqBnEHRzf0ZbYluswEPqoDEDI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/10/3035" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/10/3035" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/10/3035" class="active" data-icon-position="" data-hide-link-title="0">Genome-Wide Analysis of Polyadenylation Events in Schmidtea mediterranea</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460332" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-5khZ7wDEUAwldWmn_LHUoGI9KSQEa7NkuBcYEgE4MP8" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/10/3035.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/10/3035" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-kg6oJwr0qU9l2DI3R_XxfIE7VX93l7KcYrgNktuOvQM" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/10/3035" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/10/3035.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460332" id="node460332--2" data-pisa="ggg;6/10/3035" data-pisa-master="ggg;g3.116.031120" data-apath="/ggg/6/10/3035.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Analysis of Polyadenylation Events in <em>Schmidtea mediterranea</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9468-9571" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Vairavan Lakshmanan</span>, <span class="highwire-citation-author" data-delta="1">Dhiru Bansal</span>, <span class="highwire-citation-author" data-delta="2">Jahnavi Kulkarni</span>, <span class="highwire-citation-author" data-delta="3">Deepak Poduval</span>, <span class="highwire-citation-author" data-delta="4">Srikar Krishna</span>, <span class="highwire-citation-author" data-delta="5">Vidyanand Sasidharan</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-2219-4785" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Praveen Anand</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0001-7028-1803" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aswin Seshasayee</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0003-2219-4785" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dasaradhi Palakodeti</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3035-3048;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.031120 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460332--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Vairavan Lakshmanan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Stem Cell Biology and Regenerative Medicine, NCBS campus, GKVK, Bellary Road, Bangalore 560065, Karnataka, India</div><div class='author-affiliation'><span class='nlm-sup'>†</span>SASTRA University, Thirumalaisamudram, Thanjavur 613401, Tamil Nadu, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vairavan%2BLakshmanan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lakshmanan%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVairavan%2BLakshmanan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9468-9571" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vairavan Lakshmanan</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dhiru Bansal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Stem Cell Biology and Regenerative Medicine, NCBS campus, GKVK, Bellary Road, Bangalore 560065, Karnataka, India</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Manipal University, Madhav Nagar, Near Tiger Circle, Manipal 576104, Karnataka, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dhiru%2BBansal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bansal%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADhiru%2BBansal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jahnavi Kulkarni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Stem Cell Biology and Regenerative Medicine, NCBS campus, GKVK, Bellary Road, Bangalore 560065, Karnataka, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jahnavi%2BKulkarni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kulkarni%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJahnavi%2BKulkarni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Deepak Poduval </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>University of Bergen, Department of Clinical Science, Bergen 5020, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Deepak%2BPoduval%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poduval%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADeepak%2BPoduval%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Srikar Krishna </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Stem Cell Biology and Regenerative Medicine, NCBS campus, GKVK, Bellary Road, Bangalore 560065, Karnataka, India</div><div class='author-affiliation'><span class='nlm-sup'>†</span>SASTRA University, Thirumalaisamudram, Thanjavur 613401, Tamil Nadu, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Srikar%2BKrishna%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Krishna%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASrikar%2BKrishna%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Vidyanand Sasidharan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Stem Cell Biology and Regenerative Medicine, NCBS campus, GKVK, Bellary Road, Bangalore 560065, Karnataka, India</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Manipal University, Madhav Nagar, Near Tiger Circle, Manipal 576104, Karnataka, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vidyanand%2BSasidharan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sasidharan%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVidyanand%2BSasidharan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Praveen Anand </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Stem Cell Biology and Regenerative Medicine, NCBS campus, GKVK, Bellary Road, Bangalore 560065, Karnataka, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Praveen%2BAnand%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Anand%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APraveen%2BAnand%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2219-4785" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Praveen Anand</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dasaradhip@instem.res.in" class="" data-icon-position="" data-hide-link-title="0">dasaradhip@instem.res.in</a></span></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Aswin Seshasayee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>National Centre for Biological Sciences, GKVK Campus, Bangalore 560065, Karnataka, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aswin%2BSeshasayee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Seshasayee%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAswin%2BSeshasayee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7028-1803" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aswin Seshasayee</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dasaradhip@instem.res.in" class="" data-icon-position="" data-hide-link-title="0">dasaradhip@instem.res.in</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Dasaradhi Palakodeti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Stem Cell Biology and Regenerative Medicine, NCBS campus, GKVK, Bellary Road, Bangalore 560065, Karnataka, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dasaradhi%2BPalakodeti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palakodeti%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADasaradhi%2BPalakodeti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2219-4785" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dasaradhi Palakodeti</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dasaradhip@instem.res.in" class="" data-icon-position="" data-hide-link-title="0">dasaradhip@instem.res.in</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/10/3035" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/10/3035&amp;title=Genome-Wide%20Analysis%20of%20Polyadenylation%20Events%20in%20Schmidtea%20mediterranea" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/10/3035&amp;title=Genome-Wide%20Analysis%20of%20Polyadenylation%20Events%20in%20Schmidtea%20mediterranea" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/10/3035&amp;title=Genome-Wide%20Analysis%20of%20Polyadenylation%20Events%20in%20Schmidtea%20mediterranea" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/10/3035&amp;text=Genome-Wide%20Analysis%20of%20Polyadenylation%20Events%20in%20Schmidtea%20mediterranea" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/10/3035&amp;title=Genome-Wide%20Analysis%20of%20Polyadenylation%20Events%20in%20Schmidtea%20mediterranea" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/10/3035&amp;t=Genome-Wide%20Analysis%20of%20Polyadenylation%20Events%20in%20Schmidtea%20mediterranea" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/10/3035&amp;title=Genome-Wide%20Analysis%20of%20Polyadenylation%20Events%20in%20Schmidtea%20mediterranea" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/10/3035&amp;title=Genome-Wide%20Analysis%20of%20Polyadenylation%20Events%20in%20Schmidtea%20mediterranea" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460332/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460332" id="node460332--3" data-pisa="ggg;6/10/3035" data-pisa-master="ggg;g3.116.031120" data-apath="/ggg/6/10/3035.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">Genome-Wide Analysis of Polyadenylation Events in <em>Schmidtea mediterranea</em></h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0001-9468-9571" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Vairavan Lakshmanan</span>, <span class="highwire-citation-author" data-delta="1">Dhiru Bansal</span>, <span class="highwire-citation-author" data-delta="2">Jahnavi Kulkarni</span>, <span class="highwire-citation-author" data-delta="3">Deepak Poduval</span>, <span class="highwire-citation-author" data-delta="4">Srikar Krishna</span>, <span class="highwire-citation-author" data-delta="5">Vidyanand Sasidharan</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-2219-4785" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Praveen Anand</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="7"><a href="http://orcid.org/0000-0001-7028-1803" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aswin Seshasayee</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="8"><a href="http://orcid.org/0000-0003-2219-4785" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Dasaradhi Palakodeti</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3035-3048;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.031120 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460332--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Vairavan Lakshmanan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Stem Cell Biology and Regenerative Medicine, NCBS campus, GKVK, Bellary Road, Bangalore 560065, Karnataka, India</div><div class='author-affiliation'><span class='nlm-sup'>†</span>SASTRA University, Thirumalaisamudram, Thanjavur 613401, Tamil Nadu, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vairavan%2BLakshmanan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Lakshmanan%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AVairavan%2BLakshmanan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0001-9468-9571" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Vairavan Lakshmanan</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Dhiru Bansal </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Stem Cell Biology and Regenerative Medicine, NCBS campus, GKVK, Bellary Road, Bangalore 560065, Karnataka, India</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Manipal University, Madhav Nagar, Near Tiger Circle, Manipal 576104, Karnataka, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dhiru%2BBansal%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Bansal%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADhiru%2BBansal%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Jahnavi Kulkarni </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Stem Cell Biology and Regenerative Medicine, NCBS campus, GKVK, Bellary Road, Bangalore 560065, Karnataka, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Jahnavi%2BKulkarni%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Kulkarni%20J&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AJahnavi%2BKulkarni%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Deepak Poduval </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>§</span>University of Bergen, Department of Clinical Science, Bergen 5020, Norway</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Deepak%2BPoduval%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Poduval%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADeepak%2BPoduval%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Srikar Krishna </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Stem Cell Biology and Regenerative Medicine, NCBS campus, GKVK, Bellary Road, Bangalore 560065, Karnataka, India</div><div class='author-affiliation'><span class='nlm-sup'>†</span>SASTRA University, Thirumalaisamudram, Thanjavur 613401, Tamil Nadu, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Srikar%2BKrishna%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Krishna%20S&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ASrikar%2BKrishna%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Vidyanand Sasidharan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Stem Cell Biology and Regenerative Medicine, NCBS campus, GKVK, Bellary Road, Bangalore 560065, Karnataka, India</div><div class='author-affiliation'><span class='nlm-sup'>‡</span>Manipal University, Madhav Nagar, Near Tiger Circle, Manipal 576104, Karnataka, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Vidyanand%2BSasidharan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sasidharan%20V&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AVidyanand%2BSasidharan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Praveen Anand </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Stem Cell Biology and Regenerative Medicine, NCBS campus, GKVK, Bellary Road, Bangalore 560065, Karnataka, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Praveen%2BAnand%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Anand%20P&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3APraveen%2BAnand%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2219-4785" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Praveen Anand</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dasaradhip@instem.res.in" class="" data-icon-position="" data-hide-link-title="0">dasaradhip@instem.res.in</a></span></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Aswin Seshasayee </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>**</span>National Centre for Biological Sciences, GKVK Campus, Bangalore 560065, Karnataka, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aswin%2BSeshasayee%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Seshasayee%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAswin%2BSeshasayee%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0001-7028-1803" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aswin Seshasayee</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dasaradhip@instem.res.in" class="" data-icon-position="" data-hide-link-title="0">dasaradhip@instem.res.in</a></span></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Dasaradhi Palakodeti </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Institute for Stem Cell Biology and Regenerative Medicine, NCBS campus, GKVK, Bellary Road, Bangalore 560065, Karnataka, India</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Dasaradhi%2BPalakodeti%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Palakodeti%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADasaradhi%2BPalakodeti%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0003-2219-4785" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Dasaradhi Palakodeti</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:dasaradhip@instem.res.in" class="" data-icon-position="" data-hide-link-title="0">dasaradhip@instem.res.in</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3035&amp;rft.epage=3048&amp;rft.atitle=Genome-Wide%20Analysis%20of%20Polyadenylation%20Events%20in%20Schmidtea%20mediterranea&amp;rft.volume=6&amp;rft.issue=10&amp;rft.date=2016-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Lakshmanan%2C+Vairavan&amp;rft.au=Bansal%2C+Dhiru&amp;rft.au=Kulkarni%2C+Jahnavi&amp;rft.au=Poduval%2C+Deepak&amp;rft.au=Krishna%2C+Srikar&amp;rft.au=Sasidharan%2C+Vidyanand&amp;rft.au=Anand%2C+Praveen&amp;rft.au=Seshasayee%2C+Aswin&amp;rft.au=Palakodeti%2C+Dasaradhi"></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460332/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460332/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460332/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460332/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460332/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460332/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460332/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460332/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460332/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460332/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460332/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460332/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/10/3035" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-16" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results</a></li><li class="even"><a href="#sec-23" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Discussion</a></li><li class="odd"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="even"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last odd"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/10/3035.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/10/3035.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/10/3035.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/10/3035.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__c-64NG7CB_fwZbyex-B49GkX1AvVESfu7M9yxTNDG_o__Uvi1liJMEKbHtHHv_1fo_FCLvimvbq9jMwPcCPR364Q__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
