<!DOCTYPE html>
<html lang="en" dir="ltr" 
  xmlns="http://www.w3.org/1999/xhtml"
  xmlns:mml="http://www.w3.org/1998/Math/MathML">
  <head prefix="og: http://ogp.me/ns# article: http://ogp.me/ns/article# book: http://ogp.me/ns/book#" >
    <!--[if IE]><![endif]-->
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="dns-prefetch" href="//fonts.gstatic.com" />
<link rel="dns-prefetch" href="//fonts.googleapis.com" />
<link rel="dns-prefetch" href="//www.googletagservices.com" />
<link rel="dns-prefetch" href="//stats.g.doubleclick.net" />
<link rel="dns-prefetch" href="//www.google-analytics.com" />
<link rel="dns-prefetch" href="//scholar.google.com" />
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=3, minimum-scale=1, user-scalable=yes" />
<link rel="shortcut icon" href="https://www.g3journal.org/sites/default/files/images/favicon.ico" type="image/vnd.microsoft.icon" />
<link rel="canonical" href="https://www.g3journal.org/content/6/10/3169" />
<meta name="Generator" content="Drupal 7 (http://drupal.org)" />
<link rel="alternate" type="application/pdf" title="Full Text (PDF)" href="/content/6/10/3169.full.pdf" />
<link rel="alternate" type="text/plain" title="Full Text (Plain)" href="/content/6/10/3169.full.txt" />
<link rel="alternate" type="application/vnd.ms-powerpoint" title="Powerpoint" href="/content/6/10/3169.ppt" />
<meta name="issue_cover_image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10.cover-source.jpg" />
<meta name="article_thumbnail" content="https://www.g3journal.org/content/ggg/6/10/3169/embed/mml-math-1.gif" />
<meta name="type" content="article" />
<meta name="category" content="research-article" />
<meta name="HW.identifier" content="/ggg/6/10/3169.atom" />
<meta name="HW.pisa" content="ggg;6/10/3169" />
<meta name="DC.Format" content="text/html" />
<meta name="DC.Language" content="en" />
<meta name="DC.Title" content="A New Advanced Backcross Tomato Population Enables High Resolution Leaf QTL Mapping and Gene Identification" />
<meta name="DC.Identifier" content="10.1534/g3.116.030536" />
<meta name="DC.Date" content="2016-10-01" />
<meta name="DC.Publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="DC.Rights" content="Copyright © 2016 Fulop et al.. This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited." />
<meta name="DC.AccessRights" content="open-access" />
<meta name="DC.Description" content="Quantitative Trait Loci (QTL) mapping is a powerful technique for dissecting the genetic basis of traits and species differences. Established tomato mapping populations between domesticated tomato ( Solanum lycopersicum ) and its more distant interfertile relatives typically follow a near isogenic line (NIL) design, such as the S. pennellii Introgression Line (IL) population, with a single wild introgression per line in an otherwise domesticated genetic background. Here, we report on a new advanced backcross QTL mapping resource for tomato, derived from a cross between the M82 tomato cultivar and S. pennellii . This so-called Backcrossed Inbred Line (BIL) population is comprised of a mix of BC2 and BC3 lines, with domesticated tomato as the recurrent parent. The BIL population is complementary to the existing S. pennellii IL population, with which it shares parents. Using the BILs, we mapped traits for leaf complexity, leaflet shape, and flowering time. We demonstrate the utility of the BILs for fine-mapping QTL, particularly QTL initially mapped in the ILs, by fine-mapping several QTL to single or few candidate genes. Moreover, we confirm the value of a backcrossed population with multiple introgressions per line, such as the BILs, for epistatic QTL mapping. Our work was further enabled by the development of our own statistical inference and visualization tools, namely a heterogeneous hidden Markov model for genotyping the lines, and by using state-of-the-art sparse regression techniques for QTL mapping." />
<meta name="DC.Contributor" content="Daniel Fulop" />
<meta name="DC.Contributor" content="Aashish Ranjan" />
<meta name="DC.Contributor" content="Itai Ofner" />
<meta name="DC.Contributor" content="Michael F. Covington" />
<meta name="DC.Contributor" content="Daniel H. Chitwood" />
<meta name="DC.Contributor" content="Donelly West" />
<meta name="DC.Contributor" content="Yasunori Ichihashi" />
<meta name="DC.Contributor" content="Lauren Headland" />
<meta name="DC.Contributor" content="Daniel Zamir" />
<meta name="DC.Contributor" content="Julin N. Maloof" />
<meta name="DC.Contributor" content="Neelima R. Sinha" />
<meta name="article:published_time" content="2016-10-01" />
<meta name="article:section" content="Investigations" />
<meta name="citation_title" content="A New Advanced Backcross Tomato Population Enables High Resolution Leaf QTL Mapping and Gene Identification" />
<meta name="citation_abstract" lang="en" content="&lt;p&gt;Quantitative Trait Loci (QTL) mapping is a powerful technique for dissecting the genetic basis of traits and species differences. Established tomato mapping populations between domesticated tomato (&lt;i&gt;Solanum lycopersicum&lt;/i&gt;) and its more distant interfertile relatives typically follow a near isogenic line (NIL) design, such as the &lt;i&gt;S. pennellii&lt;/i&gt; Introgression Line (IL) population, with a single wild introgression per line in an otherwise domesticated genetic background. Here, we report on a new advanced backcross QTL mapping resource for tomato, derived from a cross between the M82 tomato cultivar and &lt;i&gt;S. pennellii&lt;/i&gt;. This so-called Backcrossed Inbred Line (BIL) population is comprised of a mix of &lt;i&gt;BC&lt;sub&gt;2&lt;/sub&gt;&lt;/i&gt; and &lt;i&gt;BC&lt;sub&gt;3&lt;/sub&gt;&lt;/i&gt; lines, with domesticated tomato as the recurrent parent. The BIL population is complementary to the existing &lt;i&gt;S. pennellii&lt;/i&gt; IL population, with which it shares parents. Using the BILs, we mapped traits for leaf complexity, leaflet shape, and flowering time. We demonstrate the utility of the BILs for fine-mapping QTL, particularly QTL initially mapped in the ILs, by fine-mapping several QTL to single or few candidate genes. Moreover, we confirm the value of a backcrossed population with multiple introgressions per line, such as the BILs, for epistatic QTL mapping. Our work was further enabled by the development of our own statistical inference and visualization tools, namely a heterogeneous hidden Markov model for genotyping the lines, and by using state-of-the-art sparse regression techniques for QTL mapping.&lt;/p&gt;" />
<meta name="citation_journal_title" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publisher" content="G3: Genes, Genomes, Genetics" />
<meta name="citation_publication_date" content="2016/10/01" />
<meta name="citation_mjid" content="ggg;6/10/3169" />
<meta name="citation_id" content="6/10/3169" />
<meta name="citation_public_url" content="https://www.g3journal.org/content/6/10/3169" />
<meta name="citation_abstract_html_url" content="https://www.g3journal.org/content/6/10/3169.abstract" />
<meta name="citation_full_html_url" content="https://www.g3journal.org/content/6/10/3169.full" />
<meta name="citation_pdf_url" content="https://www.g3journal.org/content/ggg/6/10/3169.full.pdf" />
<meta name="citation_issn" content="2160-1836" />
<meta name="citation_doi" content="10.1534/g3.116.030536" />
<meta name="citation_pmid" content="27510891" />
<meta name="citation_volume" content="6" />
<meta name="citation_issue" content="10" />
<meta name="citation_num_pages" content="16" />
<meta name="citation_article_type" content="Research Article" />
<meta name="citation_section" content="Investigations" />
<meta name="citation_firstpage" content="3169" />
<meta name="citation_lastpage" content="3184" />
<meta name="citation_access" content="all" />
<meta name="citation_author" content="Daniel Fulop" />
<meta name="citation_author_institution" content="Department of Plant Biology, University of California at Davis, California 95616" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-3778-4107" />
<meta name="citation_author" content="Aashish Ranjan" />
<meta name="citation_author_institution" content="Department of Plant Biology, University of California at Davis, California 95616" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-3447-4442" />
<meta name="citation_author" content="Itai Ofner" />
<meta name="citation_author_institution" content="Institute of Plant Sciences and Genetics in Agriculture, The Hebrew University of Jerusalem, Rehovot, Israel" />
<meta name="citation_author" content="Michael F. Covington" />
<meta name="citation_author_institution" content="Department of Plant Biology, University of California at Davis, California 95616" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-0555-6409" />
<meta name="citation_author" content="Daniel H. Chitwood" />
<meta name="citation_author_institution" content="Department of Plant Biology, University of California at Davis, California 95616" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-4875-1447" />
<meta name="citation_author" content="Donelly West" />
<meta name="citation_author_institution" content="Department of Plant Biology, University of California at Davis, California 95616" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-0624-8034" />
<meta name="citation_author" content="Yasunori Ichihashi" />
<meta name="citation_author_institution" content="Department of Plant Biology, University of California at Davis, California 95616" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0003-1935-4397" />
<meta name="citation_author" content="Lauren Headland" />
<meta name="citation_author_institution" content="Department of Plant Biology, University of California at Davis, California 95616" />
<meta name="citation_author" content="Daniel Zamir" />
<meta name="citation_author_institution" content="Institute of Plant Sciences and Genetics in Agriculture, The Hebrew University of Jerusalem, Rehovot, Israel" />
<meta name="citation_author" content="Julin N. Maloof" />
<meta name="citation_author_institution" content="Department of Plant Biology, University of California at Davis, California 95616" />
<meta name="citation_author_email" content="nrsinha@ucdavis.edu" />
<meta name="citation_author_email" content="jnmaloof@ucdavis.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-9623-2599" />
<meta name="citation_author" content="Neelima R. Sinha" />
<meta name="citation_author_institution" content="Department of Plant Biology, University of California at Davis, California 95616" />
<meta name="citation_author_email" content="nrsinha@ucdavis.edu" />
<meta name="citation_author_email" content="jnmaloof@ucdavis.edu" />
<meta name="citation_author_orcid" content="http://orcid.org/0000-0002-1494-7065" />
<meta name="citation_reference" content="citation_journal_title=Biophotonics international;citation_author=M. D. Abramoff;citation_author=P. J. Magalhães;citation_author=S. J. Ram;citation_title=Image processing with ImageJ.;citation_pages=36-42;citation_volume=11;citation_year=2004" />
<meta name="citation_reference" content="citation_journal_title=Genome Res.;citation_author=P. Andolfatto;citation_author=D. Davison;citation_author=D. Erezyilmaz;citation_author=T. T. Hu;citation_author=J. Mast;citation_title=Multiplexed shotgun genotyping for rapid and efficient genetic mapping.;citation_volume=21;citation_year=2011;citation_pmid=21233398;citation_doi=10.1101/gr.115402.110" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Soc. Hortic. Sci.;citation_author=L. H. Aung;citation_title=Effects of photoperiod and temperature on vegetative and reproductive responses of Lycopersicon esculentum Mill.;citation_pages=358-360;citation_volume=101;citation_year=1976" />
<meta name="citation_reference" content="citation_journal_title=J. Stat. Softw.;citation_author=D. Bates;citation_author=M. Mächler;citation_author=B. Bolker;citation_author=S. Walker;citation_title=Fitting Linear Mixed-Effects Models Using lme4.;citation_pages=1-48;citation_volume=67;citation_year=2015" />
<meta name="citation_reference" content="citation_journal_title=Dev. Sci.;citation_author=G. Bharathan;citation_author=T. E. Goliber;citation_author=C. Moore;citation_author=S. Kessler;citation_author=T. Pham;citation_title=Homologies in Leaf Form Inferred from KNOXI Gene Expression During;citation_pages=1858-1860;citation_volume=296;citation_year=2002" />
<meta name="citation_reference" content="citation_journal_title=Curr. Opin. Biotechnol.;citation_author=M. Blümel;citation_author=N. Dally;citation_author=C. Jung;citation_title=Flowering time regulation in crops — what did we learn from Arabidopsis?;citation_pages=121-129;citation_volume=32;citation_year=2015;citation_pmid=25553537;citation_doi=10.1016/j.copbio.2014.11.023" />
<meta name="citation_reference" content="citation_journal_title=Plant molecular biology;citation_journal_abbrev=Plant Mol Biol;citation_author=L. Carmel-Goren;citation_author=Y. S. Liu;citation_author=E. Lifschitz;citation_author=D. Zamir;citation_title=The SELF-PRUNING gene family in tomato.;citation_pages=1215-1222;citation_volume=52;citation_year=2003;citation_issue=6;citation_pmid=14682620;citation_doi=10.1023/B:PLAN.0000004333.96451.11" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=S. B. Carroll;citation_title=Evo-devo and an expanding evolutionary synthesis: a genetic theory of morphological evolution.;citation_pages=25-36;citation_volume=134;citation_year=2008;citation_issue=1;citation_pmid=18614008;citation_doi=10.1016/j.cell.2008.06.030" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=D. H. Chitwood;citation_author=L. R. Headland;citation_author=D. L. Filiault;citation_author=R. Kumar;citation_author=J. M. Jiménez-Gómez;citation_title=Native environment modulates leaf size and response to simulated foliar shade across wild tomato species.;citation_pages=e29570-e29570;citation_volume=7;citation_year=2012;citation_issue=1;citation_pmid=22253737;citation_doi=10.1371/journal.pone.0029570" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=D. H. Chitwood;citation_author=L. R. Headland;citation_author=R. Kumar;citation_author=J. Peng;citation_author=J. N. Maloof;citation_title=The Developmental Trajectory of Leaflet Morphology in Wild Tomato Species.;citation_volume=158;citation_year=2012;citation_pmid=22247269;citation_doi=10.1104/pp.111.192518" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=D. H. Chitwood;citation_author=R. Kumar;citation_author=L. R. Headland;citation_author=A. Ranjan;citation_author=M. F. Covington;citation_title=A Quantitative Genetic Basis for Leaf Morphology in a Set of Precisely Defined Tomato Introgression Lines.;citation_volume=25;citation_year=2013;citation_pmid=23872539;citation_doi=10.1105/tpc.113.112391" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=D. H. Chitwood;citation_author=A. Ranjan;citation_author=R. Kumar;citation_author=Y. Ichihashi;citation_author=K. Zumstein;citation_title=Resolving Distinct Genetic Regulators of Tomato Leaf Shape within a Heteroblastic and Ontogenetic Context.;citation_volume=26;citation_year=2014;citation_pmid=25271240;citation_doi=10.1105/tpc.114.130112" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=D. H. Chitwood;citation_author=A. Ranjan;citation_author=C. C. Martinez;citation_author=L. R. Headland;citation_author=T. Thiem;citation_title=A Modern Ampelography: A Genetic Basis for Leaf Shape and Venation Patterning in Grape.;citation_volume=164;citation_year=2014;citation_pmid=24285849;citation_doi=10.1104/pp.113.229708" />
<meta name="citation_reference" content="citation_journal_abbrev=PLoS ONE;citation_author=Y. Choi;citation_author=G. E. Sims;citation_author=S. Murphy;citation_author=J. R. Miller;citation_author=A. P. Chan;citation_title=Predicting the functional effect of amino Acid substitutions and indels.;citation_pages=e46688-e46688;citation_volume=7;citation_year=2012;citation_issue=10;citation_pmid=23056405;citation_doi=10.1371/journal.pone.0046688" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=M. C. deVicente;citation_author=S. D. Tanksley;citation_title=QTL analysis of transgressive segregation in an interspecific tomato cross.;citation_pages=585-596;citation_volume=134;citation_year=1993;citation_issue=2;citation_pmid=8100788" />
<meta name="citation_reference" content="citation_journal_title=G3 (Bethesda);citation_author=U. K. Devisetty;citation_author=M. F. Covington;citation_author=A. V. Tat;citation_author=S. Lekkala;citation_author=J. N. Maloof;citation_title=Polymorphism Identification and Improved Genome Annotation of Brassica rapa Through Deep RNA Sequencing.;citation_pages=2065-2078;citation_volume=4;citation_year=2014" />
<meta name="citation_reference" content="citation_journal_title=Cold Spring Harbor Symposia on Quantitative Biology;citation_journal_abbrev=Cold Spring Harb Symp Quant Biol;citation_author=J. Doebley;citation_author=R.-L. Wang;citation_title=Genetics and the Evolution of Plant Form: An Example from Maize;citation_pages=361-367;citation_volume=62;citation_year=1997;citation_issue=0;citation_pmid=9598370;citation_doi=10.1101/SQB.1997.062.01.043" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=I. Efroni;citation_author=Y. Eshed;citation_author=E. Lifschitz;citation_title=Morphogenesis of Simple and Compound Leaves: A Critical Review.;citation_volume=22;citation_year=2010;citation_pmid=20435903;citation_doi=10.1105/tpc.109.073601" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=T. Elitzur;citation_author=H. Nahum;citation_author=Y. Borovsky;citation_author=I. Pekker;citation_author=Y. Eshed;citation_title=Co-ordinated regulation of flowering time, plant architecture and growth by FASCICULATE: the pepper orthologue of SELF PRUNING.;citation_volume=60;citation_year=2009;citation_pmid=19174461;citation_doi=10.1093/jxb/ern334" />
<meta name="citation_reference" content="citation_journal_title=Euphytica;citation_author=Y. Eshed;citation_author=D. Zamir;citation_title=A genomic library of Lycopersicon pennellii in L. esculentum: A tool for fine mapping of genes.;citation_pages=175-179;citation_volume=79;citation_year=1994;citation_doi=10.1007/BF00022516" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=P. Fan;citation_author=A. M. Miller;citation_author=A. L. Schilmiller;citation_author=X. Liu;citation_author=I. Ofner;citation_title=In vitro reconstruction and analysis of evolutionary variation of the tomato acylsucrose metabolic network.;citation_volume=113;citation_year=2016;citation_pmid=26715757;citation_doi=10.1073/pnas.1517930113" />
<meta name="citation_reference" content="citation_journal_title=Ann. Bot. (Lond.);citation_author=K. G. Ferris;citation_author=T. Rushton;citation_author=A. B. Greenlee;citation_author=K. Toll;citation_author=B. K. Blackman;citation_title=Leaf shape evolution has a similar genetic architecture in three edaphic specialists within the Mimulus guttatus species complex.;citation_volume=116;citation_year=2015;citation_pmid=26070644;citation_doi=10.1093/aob/mcv080" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=A. Fournier-Level;citation_author=L. L. Cunff;citation_author=C. Gomez;citation_author=A. Doligez;citation_author=A. Ageorges;citation_title=Quantitative Genetic Bases of Anthocyanin Variation in Grape (Vitis vinifera L. ssp. sativa) Berry: A Quantitative Trait Locus to Quantitative Trait Nucleotide Integrated Study.;citation_volume=183;citation_year=2009;citation_pmid=19720862;citation_doi=10.1534/genetics.109.103929" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=A. Frary;citation_author=T. C. Nesbitt;citation_author=A. Frary;citation_author=S. Grandillo;citation_author=E. van der Knaap;citation_title=fw2.2: A Quantitative Trait Locus Key to the Evolution of Tomato Fruit Size;citation_pages=85-88;citation_volume=289;citation_year=2000;citation_issue=5476;citation_pmid=10884229;citation_doi=10.1126/science.289.5476.85" />
<meta name="citation_reference" content="citation_journal_title=PNAS;citation_journal_abbrev=Proc. Natl. Acad. Sci. USA;citation_author=E. Fridman;citation_author=T. Pleban;citation_author=D. Zamir;citation_title=A recombination hotspot delimits a wild-species quantitative trait locus for tomato sugar content to 484 bp within an invertase gene;citation_pages=4718-4723;citation_volume=97;citation_year=2000;citation_issue=9;citation_pmid=10781077;citation_doi=10.1073/pnas.97.9.4718" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=E. Fridman;citation_author=F. Carrari;citation_author=Y.-S. Liu;citation_author=A. R. Fernie;citation_author=D. Zamir;citation_title=Zooming In on a Quantitative Trait for Tomato Yield Using Interspecific Introgressions;citation_pages=1786-1789;citation_volume=305;citation_year=2004;citation_issue=5691;citation_pmid=15375271;citation_doi=10.1126/science.1101666" />
<meta name="citation_reference" content="citation_journal_title=Current topics in developmental biology;citation_journal_abbrev=Curr Top Dev Biol;citation_author=T. Goliber;citation_author=S. Kessler;citation_author=J. J. Chen;citation_author=G. Bharathan;citation_author=N. Sinha;citation_title=Genetic, molecular, and morphological analysis of compound leaf development.;citation_pages=259-290;citation_volume=43;citation_year=1999;citation_pmid=9891889" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=A. Gonzalez;citation_author=M. Zhao;citation_author=J. M. Leavitt;citation_author=A. M. Lloyd;citation_title=Regulation of the anthocyanin biosynthetic pathway by the TTG1/bHLH/Myb transcriptional complex in Arabidopsis seedlings.;citation_pages=814-827;citation_volume=53;citation_year=2008;citation_issue=5;citation_pmid=18036197;citation_doi=10.1111/j.1365-313X.2007.03373.x" />
<meta name="citation_reference" content="citation_journal_abbrev=J Stat Softw;citation_author=J. Hadfield;citation_title=Regularization Paths for Generalized Linear Models via Coordinate Descent.;citation_pages=1-22;citation_volume=33;citation_year=2010;citation_issue=1;citation_pmid=20808728;citation_doi=10.1111/j.1467-9868.2005.00503.x" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=H. E. Hoekstra;citation_author=J. A. Coyne;citation_title=The locus of evolution: evo devo and the genetics of adaptation.;citation_pages=995-1016;citation_volume=61;citation_year=2007;citation_issue=5;citation_pmid=17492956;citation_doi=10.1111/j.1558-5646.2007.00105.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=H. E. E. Holtan;citation_author=S. Hake;citation_title=Quantitative Trait Locus Analysis of Leaf Dissection in Tomato Using Lycopersicon pennellii Segmental Introgression Lines;citation_pages=1541-1550;citation_volume=165;citation_year=2003;citation_issue=3;citation_pmid=14668401" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y. Ichihashi;citation_author=J. A. Aguilar-Martínez;citation_author=M. Farhi;citation_author=D. H. Chitwood;citation_author=R. Kumar;citation_title=Evolutionary developmental transcriptomics reveals a gene network module regulating interspecific diversity in plant leaf shape.;citation_volume=111;citation_year=2014;citation_pmid=24927584;citation_doi=10.1073/pnas.1402835111" />
<meta name="citation_reference" content="citation_journal_title=Journal of Heredity;citation_journal_abbrev=J Hered;citation_author=H. Iwata;citation_author=Y. Ukai;citation_title=SHAPE: A Computer Program Package for Quantitative Evaluation of Biological Shapes Based on Elliptic Fourier Descriptors;citation_pages=384-385;citation_volume=93;citation_year=2002;citation_issue=5;citation_pmid=12547931;citation_doi=10.1093/jhered/93.5.384" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=C. Jiang;citation_author=R. J. Wright;citation_author=S. S. Woo;citation_author=T. A. DelMonte;citation_author=A. H. Paterson;citation_title=QTL analysis of leaf morphology in tetraploid Gossypium (cotton).;citation_pages=409-418;citation_volume=100;citation_year=2000;citation_doi=10.1007/s001220050054" />
<meta name="citation_reference" content="citation_journal_title=Genome (Ottawa. Print);citation_journal_abbrev=Genome (Ottawa. Print);citation_author=J. M. Jiménez-Gómez;citation_author=C. Alonso-Blanco;citation_author=A. Borja;citation_author=G. Anastasio;citation_author=T. Angosto;citation_title=Quantitative genetic analysis of flowering time in tomato.;citation_pages=303-315;citation_volume=50;citation_year=2007;citation_issue=3;citation_pmid=17502904" />
<meta name="citation_reference" content="citation_journal_title=American Journal of Botany;citation_journal_abbrev=Am. J. Bot.;citation_author=C. M. Jones;citation_author=C. M. Rick;citation_author=D. Adams;citation_author=J. Jernstedt;citation_author=R. T. Chetelat;citation_title=Genealogy and fine mapping of obscuravenosa, a gene affecting the distribution of chloroplasts in leaf veins, and evidence of selection during breeding of tomatoes (Lycopersicon esculentum; Solanaceae);citation_pages=935-947;citation_volume=94;citation_year=2007;citation_issue=6;citation_pmid=21636462;citation_doi=10.3732/ajb.94.6.935" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=R. A. Kerstetter;citation_author=K. Bollman;citation_author=R. A. Taylor;citation_author=K. Bomblies;citation_author=R. S. Poethig;citation_title=KANADI regulates organ polarity in Arabidopsis.;citation_pages=706-709;citation_volume=411;citation_year=2001;citation_issue=6838;citation_pmid=11395775;citation_doi=10.1038/35079629" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=J. J. B. Keurentjes;citation_author=L. Bentsink;citation_author=C. Alonso-Blanco;citation_author=C. J. Hanhart;citation_author=H. B.-D. Vries;citation_title=Development of a Near-Isogenic Line Population of Arabidopsis thaliana and Comparison of Mapping Power With a Recombinant Inbred Line Population.;citation_volume=175;citation_year=2007;citation_pmid=17179089;citation_doi=10.1534/genetics.106.066423" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=M. Kim;citation_author=S. McCormick;citation_author=M. Timmermans;citation_author=N. Sinha;citation_title=The expression domain of PHANTASTICA determines leaflet placement in compound leaves.;citation_pages=438-443;citation_volume=424;citation_year=2003;citation_issue=6947;citation_pmid=12879073;citation_doi=10.1038/nature01820" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=M. Kim;citation_author=T. Pham;citation_author=A. Hamidi;citation_author=S. McCormick;citation_author=R. K. Kuzoff;citation_title=Reduced leaf complexity in tomato wiry mutants suggests a role for PHAN and KNOX genes in generating compound leaves;citation_pages=4405-4415;citation_volume=130;citation_year=2003;citation_issue=18;citation_pmid=12900456;citation_doi=10.1242/dev.00655" />
<meta name="citation_reference" content="citation_journal_title=Current biology :  CB;citation_journal_abbrev=Curr Biol;citation_author=S. Kimura;citation_author=D. Koenig;citation_author=J. Kang;citation_author=F. Y. Yoong;citation_author=N. Sinha;citation_title=Natural variation in leaf morphology results from mutation of a novel KNOX gene.;citation_pages=672-677;citation_volume=18;citation_year=2008;citation_issue=9;citation_pmid=18424140;citation_doi=10.1016/j.cub.2008.04.008" />
<meta name="citation_reference" content="citation_journal_title=Sci. Hortic. (Amsterdam);citation_author=J. M. Kinet;citation_title=Effect of light conditions on the development of the inflorescence in tomato.;citation_pages=15-26;citation_volume=6;citation_year=1977;citation_doi=10.1016/0304-4238(77)90074-7" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=D. Koenig;citation_author=E. Bayer;citation_author=J. Kang;citation_author=C. Kuhlemeier;citation_author=N. Sinha;citation_title=Auxin patterns Solanum lycopersicum leaf morphogenesis;citation_pages=2997-3006;citation_volume=136;citation_year=2009;citation_issue=17;citation_pmid=19666826;citation_doi=10.1242/dev.033811" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. Koenig;citation_author=J. M. Jiménez-Gómez;citation_author=S. Kimura;citation_author=D. Fulop;citation_author=D. H. Chitwood;citation_title=Comparative transcriptomics reveals patterns of selection in domesticated and wild tomato.;citation_volume=110;citation_year=2013;citation_pmid=23803858;citation_doi=10.1073/pnas.1309606110" />
<meta name="citation_reference" content="citation_journal_title=The Plant journal :  for cell and molecular biology;citation_journal_abbrev=Plant J;citation_author=H. Koltai;citation_author=D. M. Bird;citation_title=Epistatic repression of PHANTASTICA and class 1 KNOTTED genes is uncoupled in tomato.;citation_pages=455-459;citation_volume=22;citation_year=2000;citation_issue=5;citation_pmid=10849361;citation_doi=10.1046/j.1365-313X.2000.00754.x" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=L. X. Ku;citation_author=J. Zhang;citation_author=S. L. Guo;citation_author=H. Y. Liu;citation_author=R. F. Zhao;citation_title=Integrated multiple population analysis of leaf architecture traits in maize (Zea mays L.).;citation_volume=63;citation_year=2012;citation_pmid=21984652;citation_doi=10.1093/jxb/err277" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=N. B. Langlade;citation_author=X. Feng;citation_author=T. Dransfield;citation_author=L. Copsey;citation_author=A. I. Hanna;citation_title=Evolution through genetically controlled allometry space.;citation_volume=102;citation_year=2005;citation_pmid=16009935;citation_doi=10.1073/pnas.0504210102" />
<meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.;citation_author=M. Lawrence;citation_author=W. Huber;citation_author=H. Pagès;citation_author=P. Aboyoun;citation_author=M. Carlson;citation_title=Software for Computing and Annotating Genomic Ranges.;citation_pages=e1003118;citation_volume=9;citation_year=2013;citation_pmid=23950696;citation_doi=10.1371/journal.pcbi.1003118" />
<meta name="citation_reference" content="citation_journal_title=J. Exp. Bot.;citation_author=E. Lifschitz;citation_author=Y. Eshed;citation_title=Universal florigenic signals triggered by FT homologues regulate growth and flowering cycles in perennial day-neutral tomato.;citation_volume=57;citation_year=2006;citation_pmid=17005925;citation_doi=10.1093/jxb/erl106" />
<meta name="citation_reference" content="citation_journal_title=Front. Plant Sci.;citation_author=E. Lifschitz;citation_author=B. G. Ayre;citation_author=Y. Eshed;citation_title=Florigen and anti-florigen – a systemic mechanism for coordinating growth and termination in flowering plants.;citation_pages=465;citation_volume=5;citation_year=2014;citation_pmid=25278944;citation_doi=10.3389/fpls.2014.00465" />
<meta name="citation_reference" content="citation_author=Y. S. Liu;citation_pages=26-30;citation_volume=49;citation_year=1999" />
<meta name="citation_reference" content="citation_journal_title=Bioinformatics;citation_author=P. C. Lott;citation_author=I. Korf;citation_title=StochHMM: a flexible hidden Markov model tool and C++ library.;citation_volume=30;citation_year=2014;citation_pmid=24489371;citation_doi=10.1093/bioinformatics/btu057" />
<meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.;citation_author=R. Mazumder;citation_author=J. H. Friedman;citation_author=T. Hastie;citation_title=SparseNet: Coordinate Descent With Nonconvex Penalties.;citation_pages=1125-1138;citation_volume=106;citation_year=2011;citation_pmid=25580042;citation_doi=10.1198/jasa.2011.tm09738" />
<meta name="citation_reference" content="citation_journal_title=Nature;citation_journal_abbrev=Nature;citation_author=J. R. McConnell;citation_author=J. Emery;citation_author=Y. Eshed;citation_author=N. Bao;citation_author=J. Bowman;citation_title=Role of PHABULOSA and PHAVOLUTA in determining radial patterning in shoots.;citation_pages=709-713;citation_volume=411;citation_year=2001;citation_issue=6838;citation_pmid=11395776;citation_doi=10.1038/35079635" />
<meta name="citation_reference" content="citation_journal_title=BMC genomics [electronic resource];citation_journal_abbrev=BMC Genomics;citation_author=J. Monson-Miller;citation_author=D. C. Sanchez-Mendez;citation_author=J. Fass;citation_author=I. M. Henry;citation_author=T. H. Tai;citation_title=Reference genome-independent assessment of mutation density using restriction enzyme-phased sequencing.;citation_pages=72-72;citation_volume=13;citation_year=2012;citation_pmid=22333298;citation_doi=10.1186/1471-2164-13-72" />
<meta name="citation_reference" content="citation_journal_title=The Plant Cell Online;citation_journal_abbrev=Plant Cell;citation_author=A. Mouradov;citation_author=F. Cremer;citation_author=G. Coupland;citation_title=Control of Flowering Time: Interacting Pathways as a Basis for Diversity;citation_pages=s111-130;citation_volume=14;citation_year=2002;citation_issue=90001;citation_pmid=12045273" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=L. C. Moyle;citation_title=Ecological and evolutionary genomics in the wild tomatoes (Solanum sect. Lycopersicon).;citation_pages=2995-3013;citation_volume=62;citation_year=2008;citation_issue=12;citation_pmid=18752600;citation_doi=10.1111/j.1558-5646.2008.00487.x" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_author=C. D. Muir;citation_author=J. B. Pease;citation_author=L. C. Moyle;citation_title=Quantitative Genetic Analysis Indicates Natural Selection on Leaf Phenotypes Across Wild Tomato Species (Solanum sect. Lycopersicon; Solanaceae).;citation_volume=198;citation_year=2014;citation_pmid=25298519;citation_doi=10.1534/genetics.114.169276" />
<meta name="citation_reference" content="citation_journal_title=Nat. Genet.;citation_author=N. A. Müller;citation_author=C. L. Wijnen;citation_author=A. Srinivasan;citation_author=M. Ryngajllo;citation_author=I. Ofner;citation_title=Domestication selected for deceleration of the circadian clock in cultivated tomato.;citation_pages=89-93;citation_volume=48;citation_year=2016;citation_pmid=26569124" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=U. Nath;citation_author=B. C. W. Crawford;citation_author=R. Carpenter;citation_author=E. Coen;citation_title=Genetic Control of Surface Curvature;citation_pages=1404-1407;citation_volume=299;citation_year=2003;citation_issue=5611;citation_pmid=12610308;citation_doi=10.1126/science.1079354" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=A. A. Naz;citation_author=S. Raman;citation_author=C. C. Martinez;citation_author=N. R. Sinha;citation_author=G. Schmitz;citation_title=Trifoliate encodes an MYB transcription factor that modulates leaf and shoot architecture in tomato.;citation_volume=110;citation_year=2013;citation_pmid=23341595;citation_doi=10.1073/pnas.1214300110" />
<meta name="citation_reference" content="citation_journal_title=Funct. Plant Biol.;citation_author=A. B. Nicotra;citation_author=A. Leigh;citation_author=C. K. Boyce;citation_author=C. S. Jones;citation_author=K. J. Niklas;citation_title=The evolution and functional significance of leaf shape in the angiosperms.;citation_pages=535-552;citation_volume=38;citation_year=2011;citation_doi=10.1071/FP11057" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=J. Ning;citation_author=G. D. Moghe;citation_author=B. Leong;citation_author=J. Kim;citation_author=I. Ofner;citation_title=A Feedback-Insensitive Isopropylmalate Synthase Affects Acylsugar Composition in Cultivated and Wild Tomato.;citation_volume=169;citation_year=2015;citation_pmid=25986128;citation_doi=10.1104/pp.15.00474" />
<meta name="citation_reference" content="citation_journal_title=Plant J.;citation_title=Solanum pennellii backcross inbred lines (BILs) link small genomic bins with tomato traits.;citation_author=I. Ofner;citation_author=J. Lashbrooke;citation_author=T. Pleban;citation_author=A. Aharoni;citation_author=D. Zamir;citation_year=2016" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=N. Ori;citation_author=A. R. Cohen;citation_author=A. Etzioni;citation_author=A. Brand;citation_author=O. Yanai;citation_title=Regulation of LANCEOLATE by miR319 is required for compound-leaf development in tomato.;citation_pages=787-791;citation_volume=39;citation_year=2007;citation_issue=6;citation_pmid=17486095;citation_doi=10.1038/ng2036" />
<meta name="citation_reference" content="citation_journal_title=Genetics;citation_journal_abbrev=Genetics;citation_author=J. M. Pérez-Pérez;citation_author=J. Serrano-Cartagena;citation_author=J. L. Micol;citation_title=Genetic Analysis of Natural Variations in the Architecture of Arabidopsis thaliana Vegetative Leaves;citation_pages=893-915;citation_volume=162;citation_year=2002;citation_issue=2;citation_pmid=12399398" />
<meta name="citation_reference" content="citation_journal_title=Development;citation_journal_abbrev=Development;citation_author=L. Pnueli;citation_author=L. Carmel-Goren;citation_author=D. Hareven;citation_author=T. Gutfinger;citation_author=J. Alvarez;citation_title=The SELF-PRUNING gene of tomato regulates vegetative to reproductive switching of sympodial meristems and is the ortholog of CEN and TFL1;citation_pages=1979-1989;citation_volume=125;citation_year=1998;citation_issue=11;citation_pmid=9570763" />
<meta name="citation_reference" content="citation_journal_title=Genome Biol.;citation_author=A. Ranjan;citation_author=Y. Ichihashi;citation_author=N. R. Sinha;citation_title=The tomato genome: implications for plant breeding, genomics and evolution.;citation_pages=1-8;citation_volume=13;citation_year=2012;citation_doi=10.1186/gb-2012-13-1-r1" />
<meta name="citation_reference" content="citation_journal_title=TAG. Theoretical and applied genetics.  Theoretische und angewandte Genetik;citation_journal_abbrev=Theor Appl Genet;citation_author=M. C. Rousseaux;citation_author=C. M. Jones;citation_author=D. Adams;citation_author=R. Chetelat;citation_author=A. Bennett;citation_title=QTL analysis of fruit antioxidants in tomato using Lycopersicon pennellii introgression lines.;citation_pages=1396-1408;citation_volume=111;citation_year=2005;citation_issue=7;citation_pmid=16177901;citation_doi=10.1007/s00122-005-0071-7" />
<meta name="citation_reference" content="citation_journal_title=Nature biotechnology;citation_journal_abbrev=Nat Biotechnol;citation_author=N. Schauer;citation_author=Y. Semel;citation_author=U. Roessner;citation_author=A. Gur;citation_author=I. Balbo;citation_title=Comprehensive metabolic profiling and phenotyping of interspecific introgression lines for tomato improvement.;citation_pages=447-454;citation_volume=24;citation_year=2006;citation_issue=4;citation_pmid=16531992;citation_doi=10.1038/nbt1192" />
<meta name="citation_reference" content="citation_journal_title=Plant Cell;citation_author=N. Schauer;citation_author=Y. Semel;citation_author=I. Balbo;citation_author=M. Steinfath;citation_author=D. Repsilber;citation_title=Mode of Inheritance of Primary Metabolic Traits in Tomato.;citation_volume=20;citation_year=2008;citation_pmid=18364465;citation_doi=10.1105/tpc.107.056523" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=Y. Semel;citation_author=J. Nissenbaum;citation_author=N. Menda;citation_author=M. Zinder;citation_author=U. Krieger;citation_title=Overdominant quantitative trait loci for yield and fitness in tomato.;citation_volume=103;citation_year=2006;citation_pmid=16938842;citation_doi=10.1073/pnas.0604635103" />
<meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA;citation_author=D. K. Seymour;citation_author=D. L. Filiault;citation_author=I. M. Henry;citation_author=J. Monson-Miller;citation_author=M. Ravi;citation_title=Rapid creation of Arabidopsis doubled haploid lines for quantitative trait locus mapping.;citation_volume=109;citation_year=2012;citation_pmid=22371599;citation_doi=10.1073/pnas.1117277109" />
<meta name="citation_reference" content="citation_journal_title=Theor. Appl. Genet.;citation_author=M. Sharlach;citation_author=D. Dahlbeck;citation_author=L. Liu;citation_author=J. Chiu;citation_author=J. M. Jiménez-Gómez;citation_title=Fine genetic mapping of RXopJ4, a bacterial spot disease resistance locus from Solanum pennellii LA716.;citation_pages=601-609;citation_volume=126;citation_year=2012;citation_pmid=23117718" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=G. G. Simpson;citation_author=C. Dean;citation_title=Arabidopsis, the Rosetta Stone of Flowering Time?;citation_pages=285-289;citation_volume=296;citation_year=2002;citation_issue=5566;citation_pmid=11951029;citation_doi=10.1126/science.296.5566.285" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=M.-C. Steinhauser;citation_author=D. Steinhauser;citation_author=Y. Gibon;citation_author=M. Bolger;citation_author=S. Arrivault;citation_title=Identification of Enzyme Activity Quantitative Trait Loci in a Solanum lycopersicum × Solanum pennellii Introgression Line Population.;citation_volume=157;citation_year=2011;citation_pmid=21890649;citation_doi=10.1104/pp.111.181594" />
<meta name="citation_reference" content="citation_journal_title=Plant Physiol.;citation_author=R. Stevens;citation_author=M. Buret;citation_author=P. Duffé;citation_author=C. Garchery;citation_author=P. Baldet;citation_title=Candidate Genes and Quantitative Trait Loci Affecting Fruit Ascorbic Acid Content in Three Tomato Populations.;citation_volume=143;citation_year=2007;citation_pmid=17277090;citation_doi=10.1104/pp.106.091413" />
<meta name="citation_reference" content="citation_journal_title=Nature genetics;citation_journal_abbrev=Nat Genet;citation_author=F. Tian;citation_author=P. J. Bradbury;citation_author=P. J. Brown;citation_author=H. Hung;citation_author=Q. Sun;citation_title=Genome-wide association study of leaf architecture in the maize nested association mapping population.;citation_pages=159-162;citation_volume=43;citation_year=2011;citation_issue=2;citation_pmid=21217756;citation_doi=10.1038/ng.746" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. C. Timmermans;citation_author=A. Hudson;citation_author=P. W. Becraft;citation_author=T. Nelson;citation_title=ROUGH SHEATH2: A Myb Protein That Represses knox Homeobox Genes in Maize Lateral Organ Primordia;citation_pages=151-153;citation_volume=284;citation_year=1999;citation_issue=5411;citation_pmid=10102816;citation_doi=10.1126/science.284.5411.151" />
<meta name="citation_reference" content="citation_journal_title=Annual review of plant biology;citation_journal_abbrev=Annu Rev Plant Biol;citation_author=B. T. Townsley;citation_author=N. R. Sinha;citation_title=A new development: evolving concepts in leaf ontogeny.;citation_pages=535-562;citation_volume=63;citation_year=2012;citation_pmid=22404465;citation_doi=10.1146/annurev-arplant-042811-105524" />
<meta name="citation_reference" content="citation_journal_title=Nature reviews. Genetics;citation_journal_abbrev=Nat Rev Genet;citation_author=M. Tsiantis;citation_author=A. Hay;citation_title=Comparative plant development: the time of the leaf?;citation_pages=169-180;citation_volume=4;citation_year=2003;citation_issue=3;citation_pmid=12610522" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=M. Tsiantis;citation_author=R. Schneeberger;citation_author=J. F. Golz;citation_author=M. Freeling;citation_author=J. A. Langdale;citation_title=The Maize rough sheath2 Gene and Leaf Development Programs in Monocot and Dicot Plants;citation_pages=154-156;citation_volume=284;citation_year=1999;citation_issue=5411;citation_pmid=10102817;citation_doi=10.1126/science.284.5411.154" />
<meta name="citation_reference" content="citation_journal_title=Cell;citation_journal_abbrev=Cell;citation_author=R. Waites;citation_author=H. R. N. Selvadurai;citation_author=I. R. Oliver;citation_author=A. Hudson;citation_title=The PHANTASTICA gene encodes a MYB transcription factor involved in growth and dorsoventrality of lateral organs in Antirrhinum.;citation_pages=779-789;citation_volume=93;citation_year=1998;citation_issue=5;citation_pmid=9630222;citation_doi=10.1016/S0092-8674(00)81439-7" />
<meta name="citation_reference" content="citation_journal_title=Evolution;citation_journal_abbrev=Evolution;citation_author=A. Westerbergh;citation_author=J. Doebley;citation_title=Morphological traits defining species differences in wild relatives of maize are controlled by multiple quantitative trait loci.;citation_pages=273-283;citation_volume=56;citation_year=2002;citation_issue=2;citation_pmid=11926495;citation_doi=10.1111/j.0014-3820.2002.tb01337.x" />
<meta name="citation_reference" content="citation_journal_title=Science;citation_journal_abbrev=Science;citation_author=H. Xiao;citation_author=N. Jiang;citation_author=E. Schaffner;citation_author=E. J. Stockinger;citation_author=E. van der Knaap;citation_title=A Retrotransposon-Mediated Gene Duplication Underlies Morphological Variation of Tomato Fruit;citation_pages=1527-1530;citation_volume=319;citation_year=2008;citation_issue=5869;citation_pmid=18339939;citation_doi=10.1126/science.1153040" />
<meta name="citation_fulltext_world_readable" content="" />
<meta name="twitter:title" content="A New Advanced Backcross Tomato Population Enables High Resolution Leaf QTL Mapping and Gene Identification" />
<meta name="twitter:card" content="summary_large_image" />
<meta name="twitter:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10.cover-source.jpg" />
<meta name="twitter:description" content="Quantitative Trait Loci (QTL) mapping is a powerful technique for dissecting the genetic basis of traits and species differences. Established tomato mapping populations between domesticated tomato ( Solanum lycopersicum ) and its more distant interfertile relatives typically follow a near isogenic line (NIL) design, such as the S. pennellii Introgression Line (IL) population, with a single wild introgression per line in an otherwise domesticated genetic background. Here, we report on a new advanced backcross QTL mapping resource for tomato, derived from a cross between the M82 tomato cultivar and S. pennellii . This so-called Backcrossed Inbred Line (BIL) population is comprised of a mix of BC2 and BC3 lines, with domesticated tomato as the recurrent parent. The BIL population is complementary to the existing S. pennellii IL population, with which it shares parents. Using the BILs, we mapped traits for leaf complexity, leaflet shape, and flowering time. We demonstrate the utility of the BILs for fine-mapping QTL, particularly QTL initially mapped in the ILs, by fine-mapping several QTL to single or few candidate genes. Moreover, we confirm the value of a backcrossed population with multiple introgressions per line, such as the BILs, for epistatic QTL mapping. Our work was further enabled by the development of our own statistical inference and visualization tools, namely a heterogeneous hidden Markov model for genotyping the lines, and by using state-of-the-art sparse regression techniques for QTL mapping." />
<meta name="og-title" property="og:title" content="A New Advanced Backcross Tomato Population Enables High Resolution Leaf QTL Mapping and Gene Identification" />
<meta name="og-url" property="og:url" content="https://www.g3journal.org/content/6/10/3169" />
<meta name="og-site-name" property="og:site_name" content="G3: Genes | Genomes | Genetics" />
<meta name="og-description" property="og:description" content="Quantitative Trait Loci (QTL) mapping is a powerful technique for dissecting the genetic basis of traits and species differences. Established tomato mapping populations between domesticated tomato ( Solanum lycopersicum ) and its more distant interfertile relatives typically follow a near isogenic line (NIL) design, such as the S. pennellii Introgression Line (IL) population, with a single wild introgression per line in an otherwise domesticated genetic background. Here, we report on a new advanced backcross QTL mapping resource for tomato, derived from a cross between the M82 tomato cultivar and S. pennellii . This so-called Backcrossed Inbred Line (BIL) population is comprised of a mix of BC2 and BC3 lines, with domesticated tomato as the recurrent parent. The BIL population is complementary to the existing S. pennellii IL population, with which it shares parents. Using the BILs, we mapped traits for leaf complexity, leaflet shape, and flowering time. We demonstrate the utility of the BILs for fine-mapping QTL, particularly QTL initially mapped in the ILs, by fine-mapping several QTL to single or few candidate genes. Moreover, we confirm the value of a backcrossed population with multiple introgressions per line, such as the BILs, for epistatic QTL mapping. Our work was further enabled by the development of our own statistical inference and visualization tools, namely a heterogeneous hidden Markov model for genotyping the lines, and by using state-of-the-art sparse regression techniques for QTL mapping." />
<meta name="og-type" property="og:type" content="article" />
<meta name="og-image" property="og:image" content="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10.cover-source.jpg" />
    <title>A New Advanced Backcross Tomato Population Enables High Resolution Leaf QTL Mapping and Gene Identification | G3: Genes | Genomes | Genetics</title>  
    <link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vxii7zjLbHE_mluGa1TpMbaZnhXywieuF-UZ-NETudc__C50WLLQjU6DD7u7QNXoS7vQ5uXweRJQIXDDQ1GgbA94__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__MyxrLRsFchWI4LpVByJReMDj5ndfJGr0cUbK-Q778fw__4VTwo1PxHe54HfsX5kwvTPVuywpv7wG4dR5jDmP2ciM__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="screen" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__EsoyBF96RxC3jFlWqnHP8guLJmdJnWhpSMAaElBwkF8__43t1vLxvmdCcaslZvKFrd6KmwrbQ4K-wg_R5M51WSb8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/all/modules/highwire/highwire/highwire.style.highwire.css?q7euys" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__evkL-uop-oob581o-ZFQFtnZgazCTUSKuCk8NODCD9E__Eoi6tusyhGT9I21YDUT_kixHVpPoBqdQv9oniwaPyLY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__iVfxUxZ5aeeJ7DHp0lcrpailSGaBhtFTgMIlkfiYZ_I__GcDCKO3QUGkIWuLBFGRZc9EooWszdBC7Ar3DjfH-Kns__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if lte IE 7]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ElJr3PIJEvw3qLXc1cnYiLj2G4KgDPSXFOfm6Phf8hw__JdWGm15cDWjsK6KrFlQVXQix9YgNeYysf22XZHj-Y-c__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__vP96nkAt3EeLU62QRdHXKZuS9tqQ1EMDJC_PSKd7hEk__tdZg4qOJpqrCbkbGYbpNeWgJZN-ltlK3uYdisQZuHX0__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__ALeQonusrm59NZgxD21Znyz9APleYBFYxc1qb0CaF8I__RlWB_t5Pyx1oIJssmAj_IeB8LyUEBL9m4TYyt8ErKyo__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />

<!--[if (lt IE 9)&(!IEMobile)]>
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__XH6bpcI0f2dImc-p674DLCZtWBGb-QwxJK1YexVGtno__vUceGprdo5nIhV6DH93X7fI3r8RcTJbChbas9TQXeW4__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<![endif]-->

<!--[if gte IE 9]><!-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__2WBMox6sOrN42ss5lCnH7WWVRdFdJCxtTKnQJYRwTE4__yqNvNYLvMpjy3ffuJrjjm9uW2i-Me1c23KLYuWHaqio__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<!--<![endif]-->
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__0SZ9cqJNbiiTLUQC23dujBjMSM1jVZWnUL_WhfzbpEc__Y6eRs9yQZGZ5f8sbtlP-lFE6TB1gf0DpLTZuWOyHAcc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
<link type="text/css" rel="stylesheet" href="https://www.g3journal.org/sites/default/files/advagg_css/css__dETUFs9gQK6PkYZRl6PK4_azmd9UGV4p1gK-vWfZo10__lx6R_FvE4_zWuTq2uzbBb2MP9ashR3pFxHi8C3v5I6M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.css" media="all" />
    <script type="text/javascript">
<!--//--><![CDATA[//><!--
/*!
 * yepnope1.5.4
 * (c) WTFPL, GPLv2
 */
(function(a,b,c){function d(a){return"[object Function]"==o.call(a)}function e(a){return"string"==typeof a}function f(){}function g(a){return!a||"loaded"==a||"complete"==a||"uninitialized"==a}function h(){var a=p.shift();q=1,a?a.t?m(function(){("c"==a.t?B.injectCss:B.injectJs)(a.s,0,a.a,a.x,a.e,1)},0):(a(),h()):q=0}function i(a,c,d,e,f,i,j){function k(b){if(!o&&g(l.readyState)&&(u.r=o=1,!q&&h(),l.onload=l.onreadystatechange=null,b)){"img"!=a&&m(function(){t.removeChild(l)},50);for(var d in y[c])y[c].hasOwnProperty(d)&&y[c][d].onload()}}var j=j||B.errorTimeout,l=b.createElement(a),o=0,r=0,u={t:d,s:c,e:f,a:i,x:j};1===y[c]&&(r=1,y[c]=[]),"object"==a?l.data=c:(l.src=c,l.type=a),l.width=l.height="0",l.onerror=l.onload=l.onreadystatechange=function(){k.call(this,r)},p.splice(e,0,u),"img"!=a&&(r||2===y[c]?(t.insertBefore(l,s?null:n),m(k,j)):y[c].push(l))}function j(a,b,c,d,f){return q=0,b=b||"j",e(a)?i("c"==b?v:u,a,b,this.i++,c,d,f):(p.splice(this.i++,0,a),1==p.length&&h()),this}function k(){var a=B;return a.loader={load:j,i:0},a}var l=b.documentElement,m=a.setTimeout,n=b.getElementsByTagName("script")[0],o={}.toString,p=[],q=0,r="MozAppearance"in l.style,s=r&&!!b.createRange().compareNode,t=s?l:n.parentNode,l=a.opera&&"[object Opera]"==o.call(a.opera),l=!!b.attachEvent&&!l,u=r?"object":l?"script":"img",v=l?"script":u,w=Array.isArray||function(a){return"[object Array]"==o.call(a)},x=[],y={},z={timeout:function(a,b){return b.length&&(a.timeout=b[0]),a}},A,B;B=function(a){function b(a){var a=a.split("!"),b=x.length,c=a.pop(),d=a.length,c={url:c,origUrl:c,prefixes:a},e,f,g;for(f=0;f<d;f++)g=a[f].split("="),(e=z[g.shift()])&&(c=e(c,g));for(f=0;f<b;f++)c=x[f](c);return c}function g(a,e,f,g,h){var i=b(a),j=i.autoCallback;i.url.split(".").pop().split("?").shift(),i.bypass||(e&&(e=d(e)?e:e[a]||e[g]||e[a.split("/").pop().split("?")[0]]),i.instead?i.instead(a,e,f,g,h):(y[i.url]?i.noexec=!0:y[i.url]=1,f.load(i.url,i.forceCSS||!i.forceJS&&"css"==i.url.split(".").pop().split("?").shift()?"c":c,i.noexec,i.attrs,i.timeout),(d(e)||d(j))&&f.load(function(){k(),e&&e(i.origUrl,h,g),j&&j(i.origUrl,h,g),y[i.url]=2})))}function h(a,b){function c(a,c){if(a){if(e(a))c||(j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}),g(a,j,b,0,h);else if(Object(a)===a)for(n in m=function(){var b=0,c;for(c in a)a.hasOwnProperty(c)&&b++;return b}(),a)a.hasOwnProperty(n)&&(!c&&!--m&&(d(j)?j=function(){var a=[].slice.call(arguments);k.apply(this,a),l()}:j[n]=function(a){return function(){var b=[].slice.call(arguments);a&&a.apply(this,b),l()}}(k[n])),g(a[n],j,b,n,h))}else!c&&l()}var h=!!a.test,i=a.load||a.both,j=a.callback||f,k=j,l=a.complete||f,m,n;c(h?a.yep:a.nope,!!i),i&&c(i)}var i,j,l=this.yepnope.loader;if(e(a))g(a,0,l,0);else if(w(a))for(i=0;i<a.length;i++)j=a[i],e(j)?g(j,0,l,0):w(j)?B(j):Object(j)===j&&h(j,l);else Object(a)===a&&h(a,l)},B.addPrefix=function(a,b){z[a]=b},B.addFilter=function(a){x.push(a)},B.errorTimeout=1e4,null==b.readyState&&b.addEventListener&&(b.readyState="loading",b.addEventListener("DOMContentLoaded",A=function(){b.removeEventListener("DOMContentLoaded",A,0),b.readyState="complete"},0)),a.yepnope=k(),a.yepnope.executeStack=h,a.yepnope.injectJs=function(a,c,d,e,i,j){var k=b.createElement("script"),l,o,e=e||B.errorTimeout;k.src=a;for(o in d)k.setAttribute(o,d[o]);c=j?h:c||f,k.onreadystatechange=k.onload=function(){!l&&g(k.readyState)&&(l=1,c(),k.onload=k.onreadystatechange=null)},m(function(){l||(l=1,c(1))},e),i?k.onload():n.parentNode.insertBefore(k,n)},a.yepnope.injectCss=function(a,c,d,e,g,i){var e=b.createElement("link"),j,c=i?h:c||f;e.href=a,e.rel="stylesheet",e.type="text/css";for(j in d)e.setAttribute(j,d[j]);g||(n.parentNode.insertBefore(e,n),m(c,0))}})(this,document);

//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/all/libraries/modernizr/modernizr.min.js?q7euys"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
yepnope({
  test: Modernizr.matchmedia,
  nope: '/sites/all/libraries/media-match/media.match.min.js'
});
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__wH8myWXKi4A-QKdUKDXzJAnGs-FRS1A2ZzTTNzjRAdI__4MCtOZK_Ov0QKyRlXNLGQ2LRlYRGWj9ArT-NpFox3QA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
var googletag = googletag || {};
googletag.cmd = googletag.cmd || [];
googletag.slots = googletag.slots || {};
//--><!]]>
</script>
<script type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__e1dl8PurVlybtG0bsvbi28fsCWMzhJhJw-9CuLi13ww__QFHvDMPamxn6BHsrQ4Z923sYGkAGsrArniz8NYXuaSA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
document.createElement( "picture" );
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__UK1ygaa8bCfT-jsjqYRdRLDZ5N17950T7K5WPC9XtpI__NqLYT1T1SMBsKukLWIe82VgCZYYCMHZX_f609-a1rj8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
googletag.cmd.push(function() {
  googletag.pubads().enableAsyncRendering();
  googletag.pubads().enableSingleRequest();
  googletag.pubads().collapseEmptyDivs();
  googletag.pubads().addEventListener("slotRenderEnded", function (event) {
    if (event.isEmpty) {
      var id = event.slot.getSlotElementId();
      var x = document.getElementById(id);
      if (x.parentElement.classList.contains("dfp-tag-wrapper")) {
        x.parentElement.style.display = "none";
      }
    }
  });
});

googletag.enableServices();
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__3vwOA0BwlnyBkt9H4CLg6FgNyKAh--OmfOxFOIu_5S4__EXTgktcHbQ7BQfelY6CnrqgPRc4i4eD6XcZ4i3xs5RU__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" async="async" src="https://scholar.google.com/scholar_js/casa.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__AQCcIzbw6PlLGr6AoruxzF2X11FaoDDsHA-FoLZsu4s__Mu2qoX4T4xtDCOXMf7GBbQDEjD5J9IBHdOGdiyL-BW8__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="//cdn.jsdelivr.net/qtip2/2.2.1/jquery.qtip.min.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ByQuN5ZxGu5iOx8AHVa8H3fQIJHen2f5ddYyAbkE2Hc__WDlx5EK3HX7IIXvXmVywDbelU8FyG68raiGtOAh7ebE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
if(typeof window.MathJax === "undefined") window.MathJax = { menuSettings: { zoom: "Click" } };
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__ilS4K29t6_JtwUxaczKJ6BFEP4D7uEM4J0B-32BL12k__UCXMIpg8H4VKpXMTfyUrDRhIurQCmU0b4xBEpRkTYvc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
(function(i,s,o,g,r,a,m){i["GoogleAnalyticsObject"]=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,"script","//www.google-analytics.com/analytics.js","ga");ga("create", "UA-35990348-1", {"cookieDomain":"auto"});ga("set", "anonymizeIp", true);ga("send", "pageview");ga('create', 'UA-76034214-10', 'auto', {'name': 'hwTracker'});
ga('set', 'anonymizeIp', true);
ga('hwTracker.send', 'pageview');
//--><!]]>
</script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__5V9kRH6SAXxWPJx4qmsl8wLt3K_K3o7dsZC8LWHfcis__MS89XoGFSTYvshLHZFl1kK-HQoLR3h3yY9sQyaX3OUE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript">
<!--//--><![CDATA[//><!--
jQuery.extend(Drupal.settings,{"basePath":"\/","pathPrefix":"","ajaxPageState":{"theme":"jcore_1","theme_token":"v4JPy_twcmNjtsJEsdCQFUyekrHPEJhjYQFK6vpWwLY","jquery_version":"1.8","css":{"modules\/system\/system.base.css":1,"modules\/system\/system.menus.css":1,"modules\/system\/system.messages.css":1,"modules\/system\/system.theme.css":1,"misc\/ui\/jquery.ui.core.css":1,"misc\/ui\/jquery.ui.theme.css":1,"misc\/ui\/jquery.ui.button.css":1,"misc\/ui\/jquery.ui.resizable.css":1,"misc\/ui\/jquery.ui.dialog.css":1,"misc\/ui\/jquery.ui.tooltip.css":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/themes\/base\/minified\/jquery.ui.accordion.min.css":1,"sites\/all\/modules\/contrib\/date\/date_api\/date.css":1,"sites\/all\/modules\/contrib\/date\/date_popup\/themes\/datepicker.1.7.css":1,"modules\/field\/theme\/field.css":1,"sites\/all\/modules\/contrib\/flexslider\/assets\/css\/flexslider_img.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_draw\/css\/highwire-draw.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_folders\/highwire_folders.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/css\/highwire-responsive.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_saved_searches\/highwire_saved_searches.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/highwire_user.css":1,"modules\/node\/node.css":1,"sites\/all\/modules\/contrib\/picture\/picture_wysiwyg.css":1,"modules\/search\/search.css":1,"modules\/user\/user.css":1,"sites\/all\/modules\/contrib\/views\/css\/views.css":1,"sites\/all\/modules\/contrib\/ckeditor\/css\/ckeditor.css":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.css":1,"sites\/all\/modules\/contrib\/ctools\/css\/ctools.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.highwire.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/nlm-elements.css":1,"sites\/all\/modules\/contrib\/panels\/css\/panels.css":1,"public:\/\/ctools\/css\/263bfdb18936424f0ade19d6bd885f37.css":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.css":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/css\/panels_ajax_tab.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire.style.markup.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_lens\/css\/highwire_lens.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/layouts\/highwire_2col_stacked\/highwire-2col-stacked.css":1,"sites\/all\/modules\/contrib\/forward\/forward.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_share_link.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_copy_permalink.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_corrections.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.css.less":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus.css":1,"sites\/all\/modules\/contrib\/nice_menus\/css\/nice_menus_default.css":1,"sites\/all\/modules\/contrib\/panels\/plugins\/layouts\/onecol\/onecol.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons.css":1,"sites\/all\/modules\/highwire\/highwire\/highwire_theme_tools\/css\/font-hwicons-glyphs.css":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/font-titillium-web.css":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/css\/highwire_reports_feed.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-article-citation-list.css":1,"sites\/default\/modules\/jnl_genetics\/plugins\/highwire_citation\/jcore_title_only.css":1,"sites\/all\/themes\/highwire\/jcore_1\/eu_cookie_compliance.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome-ie7.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-reset.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-mobile.css":1,"sites\/all\/themes\/contrib\/omega\/alpha\/css\/alpha-alpha.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/formalize.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-text.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-branding.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-menu.css":1,"sites\/all\/themes\/contrib\/omega\/omega\/css\/omega-forms.css":1,"sites\/all\/modules\/highwire\/highwire\/css\/highwire-forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/mobile-reset.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/font-awesome.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/forms.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/global.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/jquery-ui-elements.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/text.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/colors.css.less":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"ie::normal::sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"narrow::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-narrow.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/narrow\/jcore-default-narrow-30.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"normal::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-normal.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/normal\/jcore-default-normal-30.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default.css":1,"wide::sites\/all\/themes\/highwire\/jcore_1\/css\/jcore-1-jcore-default-wide.css":1,"sites\/all\/themes\/highwire\/jcore_1\/css\/grid\/jcore_default\/wide\/jcore-default-wide-30.css":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/fonts.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/typography.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/shared-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/main-menu.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/ajax-tabs.css.less":1,"sites\/default\/modules\/jnl_genetics_styles\/css\/image-carousels.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-styles.css.less":1,"sites\/default\/modules\/jnl_genetics_g3_styles\/css\/g3-typography.css.less":1},"js":{"sites\/all\/libraries\/enquire.js\/enquire.min.js":1,"sites\/all\/modules\/contrib\/picture\/picturefill2\/picturefill.min.js":1,"sites\/all\/modules\/contrib\/picture\/picture.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_openurl.js":1,"sites\/all\/libraries\/lazysizes\/lazysizes.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_figures.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_tables.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_google_scholar_sprinkle.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/plugins\/content_types\/js\/highwire_user_salutation.js":1,"sites\/default\/modules\/jnl_genetics_styles\/js\/genetics.js":1,"sites\/all\/libraries\/modernizr\/modernizr.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/jquery\/1.8\/jquery.min.js":1,"misc\/jquery-extend-3.4.0.js":1,"misc\/jquery.once.js":1,"https:\/\/www.googletagservices.com\/tag\/js\/gpt.js":1,"misc\/drupal.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.core.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.widget.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.button.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.mouse.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.draggable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.position.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.resizable.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.dialog.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/external\/jquery.cookie.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/misc\/jquery.form.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.tooltip.min.js":1,"sites\/all\/modules\/contrib\/jquery_update\/replace\/ui\/ui\/minified\/jquery.ui.accordion.min.js":1,"misc\/ajax.js":1,"sites\/all\/modules\/contrib\/jquery_update\/js\/jquery_update.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.equal-heights.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.panels-ajax-tab.js":1,"sites\/all\/libraries\/colorbox\/jquery.colorbox-min.js":1,"sites\/all\/modules\/contrib\/colorbox\/js\/colorbox.js":1,"sites\/all\/modules\/contrib\/colorbox\/styles\/default\/colorbox_style.js":1,"https:\/\/scholar.google.com\/scholar_js\/casa.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_log\/highwire_log.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_user\/js\/highwire_user_meta.js":1,"sites\/all\/modules\/contrib\/panels\/js\/panels.js":1,"\/\/cdn.jsdelivr.net\/qtip2\/2.2.1\/jquery.qtip.min.js":1,"sites\/all\/modules\/contrib\/panels_ajax_tab\/js\/panels_ajax_tab.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_article_reference_popup.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/highwire_markup_process\/js\/highwire_at_symbol.js":1,"misc\/textarea.js":1,"misc\/progress.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_share_dialog.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/clipboard.min.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_clipboard.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_citation\/plugins\/content_types\/js\/highwire_citation_export.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-access.js":1,"sites\/all\/modules\/highwire\/highwire\/js\/highwire.article-citation-author-tooltip.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/minipanel_dialog_link.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/plugins\/styles\/highwire_responsive_toggle\/highwire_responsive_toggle.js":1,"sites\/all\/modules\/contrib\/google_analytics\/googleanalytics.js":1,"sites\/all\/modules\/contrib\/panels_ajax_pane\/panels_ajax_pane.js":1,"sites\/all\/modules\/contrib\/panels_accordion\/js\/panels_accordion.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_article_nav.js":1,"sites\/all\/modules\/highwire\/highwire\/plugins\/content_types\/js\/highwire_nav_float.js":1,"sites\/all\/themes\/highwire\/jcore_1\/js\/theme-scripts.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/jquery.formalize.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-mediaqueries.js":1,"sites\/all\/themes\/contrib\/omega\/omega\/js\/omega-equalheights.js":1,"sites\/all\/modules\/highwire\/highwire\/highwire_responsive\/js\/highwire-mediaqueries.js":1}},"colorbox":{"opacity":"0.85","current":"{current} of {total}","previous":"\u00ab Prev","next":"Next \u00bb","close":"Close","maxWidth":"98%","maxHeight":"98%","fixed":true,"mobiledetect":true,"mobiledevicewidth":"480px"},"highwire":{"nid":"460344","apath":"\/ggg\/6\/10\/3169.atom","pisa":"ggg;6\/10\/3169","ac":{"\/ggg\/6\/10\/3169.atom":{"access":{"full":true},"pisa_id":"","apath":"\/ggg\/6\/10\/3169.atom","jcode":"ggg"}},"processed":["highwire_math"],"markup":[{"requested":"long","variant":"full-text","view":"full","pisa":"ggg;6\/10\/3169"},{"requested":"full-text","variant":"full-text","view":"full","pisa":"ggg;6\/10\/3169"}],"modal_window_width":"560","share_modal_width":"560","share_modal_title":"Share this Article"},"jcarousel":{"ajaxPath":"\/jcarousel\/ajax\/views"},"instances":"{\u0022highwire_abstract_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:20,\u0022height\u0022:20,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-abstract-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-abstract-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022right center\u0022,\u0022my\u0022:\u0022left center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022shift\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter click \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_author_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-author-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-author-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022top center\u0022,\u0022my\u0022:\u0022bottom center\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}},\u0022highwire_reflinks_tooltip\u0022:{\u0022content\u0022:{\u0022text\u0022:\u0022\u0022},\u0022style\u0022:{\u0022tip\u0022:{\u0022width\u0022:15,\u0022height\u0022:15,\u0022border\u0022:1,\u0022mimic\u0022:\u0022top center\u0022,\u0022offset\u0022:0,\u0022corner\u0022:true},\u0022classes\u0022:\u0022qtip-custom hw-tooltip hw-ref-link-tooltip qtip-shadow qtip-rounded\u0022,\u0022classes_custom\u0022:\u0022hw-tooltip hw-ref-link-tooltip\u0022},\u0022position\u0022:{\u0022at\u0022:\u0022bottom left\u0022,\u0022my\u0022:\u0022top left\u0022,\u0022viewport\u0022:true,\u0022adjust\u0022:{\u0022method\u0022:\u0022flip\u0022}},\u0022show\u0022:{\u0022event\u0022:\u0022mouseenter \u0022,\u0022solo\u0022:true},\u0022hide\u0022:{\u0022event\u0022:\u0022mouseleave \u0022,\u0022fixed\u0022:1,\u0022delay\u0022:\u0022100\u0022}}}","qtipDebug":"{\u0022leaveElement\u0022:0}","highwire_panel_tabs":[{"panel_name":"jnl_genetics_tab_pdf","panel_ajax_tab":"jnl_genetics_tab_pdf"}],"panel_ajax_tab":{"path":"sites\/all\/modules\/contrib\/panels_ajax_tab"},"urlIsAjaxTrusted":{"\/content\/6\/10\/3169.abstract":true,"\/content\/6\/10\/3169":true,"\/system\/ajax":true},"color":{"logo":"https:\/\/www.g3journal.org\/sites\/default\/files\/G3logo_1.png"},"panels_ajax_pane":{"new-37":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022q9yEneyXFuh9JAu2T81uj41X6qjkaNziIwZ2u8DR2q7Y5dzI7+UQmkz91hh\\\\\\\/G+t5TZlAmjTv9qyvXtOm0EClHYRZz0gFw0Iq+I9BVq+dp0Oe624L0JJWl6D1s4EC+8pyr+Sj25zDckQyousc4ba8v8nOzbGTmhM\\\\\\\/0vk0IQAlV8GWZYEvq651sZR9IGp\\\\\\\/HOcvCz32+WZFpPkfOHdgkubjNby6V2lZTrSRYI0xYzgM3DeTkGFus9Ef80zFE\\\\\\\/M8uuU1swGapGrxP7hDaGdH4CzWK+9jg7ogvjrvvPNnfle4+PGykn5P0zMtMQMcuL1ilBIkk0Yj0aUlip83KXRqTyvVZS9MKUvQYgA9eg6qYD8yZzagCyxmnhPqCb\\\\\\\/8aPgd2+kkwr8a4nv33c+abQBue64win5OobA6yYXX0uRecuZf2yyT6EvzJZzNO4I9YJdQ3LoSvWOeIHy6WKeTCe\\\\\\\/2Qi6\\\\\\\/cbM2yx\\\\\\\/b6QrcYj7MsOP41XYpRk4me82jGuMPb2n4+aPkZQ4z43PPGraqS6mQJMiqgFhOCJ1zqz4oCR1TJW2A0qfctpORtL38kVVgJngcLAjbHj7jHNOnWSQQP21DdTqSEphcCqds2bxmZ9qe99FPJ0Gk\\\\\\\/AOZABEw2W1OdOcnUn6lBkYNi5vEoWxSfwR4Kpnj7Yjv+Vn8tvWI7+NqTB87oviRV+TJgd\\\\\\\/YKd2uTzkI9z0qew9yJdqn9W+luQ3Bp0flWwNEvpZqR1YXZfCWlJtkboAxf7xpWfp5K1lUfDtk\\\\\\\/4zzQjq2gkHq40XPtoJQNTdTFXuQKWhHaX\\\\\\\/TezrYXwtKh1MXAz\\\\\\\/ZMhUTtK0irsxjOdv5h8088fcTbD8EStQEtLtWjZdCr7okSo17ZX56V7c74FaQNMtKTGSKXtCo1IYRGY+yko4hpSIuWAYSf\\\\\\\/v8pqLzf3xZshMawsCC2Xio1KSxiP9JKFvaBKUsxBh7Yzs6fUPZ7HsuBI4osDKjnGo\\\\\\\/2k8OSqwYYX35h9YqmwjogU0uUct24gQbMzed3AxP1lw+1WX6yTu5uOBFxTM3YTDAEzY91bwan1QYo1RiEfcxdqAnS5CRR8UpyENtxH9KuFi6cw7wkhLu5RBfGozjiyu3rXRZxMJjkaLYKky4HO\\\\\\\/Mkaclke1ntaKJia\\\\\\\/fuptS+1MRQTmdu6myn9nS86e6Qxpa50+b7fe92NKfXbZdZFZe5kFroLsIz76ZrzqqwWxMvsUTvaMZVIxpNv\\\\\\\/iEb+EIo8tw\\\\\\\/2wG3DW5gbrjAPWJSwnwMeyJel0KE5as1w4Op9jrKHVVs+P\\\u0022,\\\u0022iv\\\u0022:\\\u002229bTAXpYxPMsFZ4RjsviBw==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002232a4875ef4e003abb3986a3442d85c815fc237bf7f15b8386658de20ceade22b\u0022}","new-5":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022B03qaiYg79KjNM38HhiSmmSbBJOjx894WOvrUU\\\\\\\/amzqRckmWYw1pQCPAe7pOaCBznmFLlYxftQOemiaEdFQiyxEk+QkwGwG\\\\\\\/qkAxF5H9AC9cvQqyjg2pcm0GPJdAhDeZ6uTG\\\\\\\/q8OqezBnU9cBrO6iiO3bcU+L1GcxnDs2+GZ4ruptz1tXk0HFsNfMIUdnobXULMR0p2qBkOKdhAG2xsxetb3T13HRt47+nZL51mcqtuGlNyKUEkqhvZKbL44QIOmXo5tHtQH\\\\\\\/Vd1wJB4W1ovu1++a\\\\\\\/wwE5L2NHFWLE+x90REcJ0Xxq5eunV0IStk\\\\\\\/\\\\\\\/fULtWADxkszFIbXiYhVArv+PC4JFeLTPLVw+F725aQJTY\\\\\\\/NJtz9h3U9rAnJMLnlEGtGzlN9ge+BUmul+\\\\\\\/7yURWByD39wiokJ0aiGUaBA0UNR8N69WiOsCMGtVJjUZuz8xWmFr8rzD3IJhucZ6jS+S5huh8n3T3jh0YwVBWc1wDm8OsuZqhbqCwA9IKlCL87gpZqjp34wCi+YfDZcuOjhK5DO8ql3QX+mqSfKfYjt3oS4pky3ouhtdkMHSf8RJrpAgOMDShR9H1wSVNiWRUI1TzKVTFQXP6FlKonchqyv7LHGA1H2hO9ZV0o+N7kgrlwBd2cLpOWuC3hcA54FgIlfJRwNZHORerrvk4XHF2KgB6IrfWH0aXknaS\\\\\\\/aNNTUJ8yW9Yf5ZM++j58Z+cCdLE07QWA40g4s0xcqEbrcfYoPv0ou3wjiQ2TiXY5rv74RcVSJMulxTO+CQjA5R3r\\\\\\\/bw6aV9DO4Mn03WNQJ6hdtcSb2yB51uk5cwbsuFkHXfL6\\\\\\\/XObNrXqtfpEAzQZrRoCrkQBq+3kjo0tIPxD5EUqmtad4H8YUV\\\\\\\/cXK21+ik4ME71V+F34m1wLZ3r3TvZFHvLrvuBtYOFs4I5vbsMPeCG\\\\\\\/xXNpEeBf4rzezRy3DlCuM3fQdPuxy9gslPAwGZzRNVQHj87tHtByBl7ZIGt3RRk+yYNdtDY5AY+9Xwc8yJClyoDi6xpO2dG6OD7GDuiTiZOSLAS5mNY2OOzmTSOPP3HzMAuJ+sHv0Q6\\\\\\\/fWkZJXdjbz5Ynes2NbNRxedxhxKCWDXH42bu8tPaDohtjYIeVj+\\\\\\\/IHhR2B8sFlsrPsNS2+rclSxTmf7X\\\\\\\/EsL922R9ipq9eqhoPFnYxgjrUCug59pwkQGW44zLesk7S959SO1mEAjGsFsNL93mHe4N1ld7MlKZZfzt09HZbM\\\\\\\/xlGKm8g4TWHKN01lHX33dWqVxo\\\\\\\/vjC9K4FhLImKKLRHlQ0BjLTJp1aHJwpW04V\\\\\\\/nJPSf2fyiJnLpv+kkSaORP3iBIP3mWX7K+y7iYvZ72LdUgSNqAk+j6q1At5A5qe8sZKJli\\\\\\\/dRL+Iqo5+1As5ggldzq5ngAqmaoic\\\\\\\/RQFU5HCF3PiTA1miAkzmI+XqpwFbt9T3PNSBBoiK9PoylOLWWNRhwwEME0C1FWM+sBntVDjg0ETz2tydTqFeFyaDkDTKZ3YWYZKvFV+9FwIkz6CeW+epJEK20dfohNrbMVvQ0Lti5c7uPBRuujkYsyyv2mQqB1Cd9kHu89wQQDAiOZuJ44BO3LwMpAUOY4j+UUYUt3HPqi16iPJSb0kZh+4PvAEBweLmiuX+5NXaj+KTmfodQMCL\\\\\\\/AND8D+yaRKuC7QCB\\\\\\\/TDTf3\\\\\\\/avu0vCGG5n7d+7W3+\\\\\\\/PPlAlE3HAfWkABHL5Hbr1mByQ6bG3bMvJ3Tm7b1+8Yha2CN\\\\\\\/uwl6qiAQxKlfJ8VpLt8IgpI9u+aceR9pSzI5x8GBobzFhHFgayyU\\\\\\\/tHRTKViK9v0XXXoACykibch+VEnF4P0poscSxpmtp8cHzPlqpypt6tIPPulYavVBrtlAhBWrwGbS0tnPENgHMpPt5tGqvl\\\\\\\/D5Ycrtr\\\\\\\/Q\\\\\\\/ViJqlAeFBj9Dyhe+U2LuDA1CfP\\\\\\\/dRt034FOfCij7ZUb9UVpbRCmCLXogGkbx7jZlK5t65bq2Ye4dR30972zsfCUg4pw25dqHhgRtNRXUG7\\\\\\\/un6i4nm4IsfwBVQ8vePpi0KiFIR8Oro4awKdNw\\\\\\\/u1TunYs3Lj1ebWXMGa66gO4EfNfdi9J8zJf\\\\\\\/DI\\\\\\\/u8x\\\\\\\/yXEpkMkm0W51PaFALzfBAT+1z2VDDdQCYLgALqSdNztu50kRmMj0aGUwaBxFoupZy5dG3b4xEaBQYKdgr0ky7He\\\\\\\/QytuFAGqZPQo\\\\\\\/68JHHn3D8TmCO1aS4mONJlTlFcQJwObLJFFnCJhtI4ToiYUSvcCyK8lMzBbOpc79UP8z06bb\\\\\\\/BcDYcS1lEFbUlWbmIMKlBM\\\\\\\/l2b9dV7e1EoqDc1NK8HEyP\\\\\\\/W\\\\\\\/nFHPgN0nZFNK3lW3B4W2uGOOFmGA6bGsbnwkn5omf2pigLhBHHmM0gF5ZTaENkf76qYmPeGrNMjUi8wMXg5DtqnVKiS4lZipUCeBDOMRM\\\\\\\/BVD7Pj1BerbgtLYQh7W4k104Zv9tli\\\\\\\/gn6tbH8T+FDItXGQ7cnvGeR5GgSMQBIsG+pZNxWiwv1euhuKb8vMs3sjS457ZF5m43gUGx5pF\\\\\\\/sMPwoAHzVfJP7Ivg5M\\\\\\\/jZhsAvB1rTnR88eiSsh+S5N1FUbbgm5CTdZdY2XhhkMd36z267ORVw9tL38nwy6WW63CsxtfUtxnWQh0oHUB4FAYSZRo77s\\\\\\\/9jdMwS3Y51YpgeuOabMdpvpjKV0EJWbIQHxSVPffcUacDzwlKCZyITnEP0GLgDr9ilcvGymkhmL9zLCE2G2J8nq\\\\\\\/Cahn0e8gi02qfg86kn7RrJ2YSIQyaGiISW\\\\\\\/3SMN3dubdtvQKTLZoU5yxql719LoEDsAecz1\\\\\\\/eVBIkphOJt5QIeF8W6+T7O+nfL+C4y+dDUYDtc7yLIHMpNZiHzEziVWG8Gz9hfr2FEAaff27xdTdooxx8ajyxwBOEqPTripUyITobtG33BSMbKGlDvDWIsrZhyllKLnf8SpV7AfRRBqtUxsufK\\\\\\\/RoMEeSzuMOfxNLdANteqKTc3GKgvCBWnW3pyWu21p3KokkvofBrp6BC1jMSM1PAonbMs8Ca1b8qjSC\\\\\\\/jmBbP2ubsoweLx9xBLgn2V0o02koxJI6bXc4ePlq9a7Xz8fAbjmgmg8c\\\\\\\/dcp9cP4axc8lJFlTgqDsneIl7xvxEffOXrojJjnqIW4\\\\\\\/rVn0bwFXSvb36fDMdiDfHZVIvI9ZNZCHQ0\\\\\\\/rNCTjh8tqq5pJkLmXdrSl6PkqeXx8U37q34LU3pkzVE61Et3Kq7PYSrQmfg\\\\\\\/gqA6yJX8A8th96mvxVeQCZd+o5PhDmEIyyN8eQmRlUhphcK\\\\\\\/JuXQUZIqBFGrRfDvK7PiO9+RfrOTw3kToC9chEbssVAqPGgZU9fVfDOlslgtpuMJ7rjwr7cl8X\\\\\\\/\\\\\\\/u6oTGbQrYwtnZ5NQE2W+u4R1xri1I2pcFI3svL8+bo9r0Tx3AbmB75C1l9puDDqBu8PpjveRlq\\\\\\\/N9\\\\\\\/vBGd8F9nfu1bdePEeLQfBCkUTLqKz0XVscRrlfxXtbfGzs5wtfZ+tmN4lP91cAvfPsPVWZUmJ9e+0o5gkTLGYaekGaju+OvpU2n64\\\\\\\/DgXIwUU8grwQSNffnplgxQz9BBlx2eV1x\\\\\\\/N4n2aIrVJ1uKYi28D9Nt63RDx0JudwOXG5fVTUhaZH5rzllMhdGvUDiWZEp28NVAJyEuuRsE0+F8rzzqjisRP31BueBkwgaZvBgYQC01JauUSGJO0PP+2icQ+vrYGdUxv02GzFx6MSdAtAf6b9bBBnQr3ULcojj2XNgey+76Sn\\\\\\\/UVp+j9UBrO3+3kUx5+uHlVoVwj5hr1gsVdBrF1eDNbX4zSrqB7w7Smrr\\\\\\\/hSFou6Iig8OkFAGArg2F6DPggTbFQ310y3NQs+WVM49Ov4W11TTQ8XiaGgqPsHvsBQqjE8HfCJZiiEx650pBhboMJ1dxPumlQTOynPoksK1KYNBBNBQ1dS8OqV2NvWn4b7kMaVlV3FT6DcodGhUSGFizlQc7Ijtk0b94fVK95ziKirrytYYSO3iHwo0QkkBEvg9RtqvLKSaRZIMCbqKnUz7M1n\\\\\\\/3G\\\\\\\/zaA0dSIaUAAatoSFOXWt3LXwhSbye7bdqfuMS+N6o3U0zltgSf7OQE3C2eZA\\\\\\\/5DG8L0Kg4WWm\\\\\\\/l8ZyxrhUCMX6CSaEkZ9HwLSThIVq1xW9d7wAijxN\\\\\\\/iKR5LDp8GvMAXi4KpPrCLDVl3xboRPeq6ES053RW8tEpY2eIlY8Lork98gKUASskXIC67\\\\\\\/pbEo4fVMIZ66AzvIhPoRmmoHpL8c15YvRaJI5uCNNoabLM39cJ7P7lR80gRzoC\\\\\\\/2+dBMdxE2Dt+geJUNuiUjMZoKrJsE2elQBLglMmylQ6IrQ9lkiuZ++Vo+6yx0hnc2jheMP3wkQy3BCGo+C3X4pvfy1ut3ta11U\\\\\\\/dzC8QjHBnCWwToVHUfeB2f+jSgwxWkoUTZxB1Y0M4KpWP1WqBkmQo2t+uZtaRgoofrLZXW5eDgH+JEcVND8FyUH\\\\\\\/cM9kSNC8WZpUko8spnJcOMWDFIj9JNm1kgeAMjNHhl+b9ttfZQyXoOh6Wq0LTFcRGECd\\\\\\\/vVtXRFZeEle5hJVbfbF5C7jcZDYlIV\\\\\\\/UiJqo3nwDgVPyG1HxY1AMHhoq7KOhSnqM4Cc9XpAvaRgzneQ1k+e62RJNPZI2YBeLjbNsFQh\\\\\\\/D7ygSr5k50PcNeCaexgLzSihIhy6iAw5FhmAGyksrp6N8oHx2nFJ5tEFnmEh9uaGksIg1rCMtmbQ7Y8D7NEL+j5j8fneo9N0zA+MVmnYHo\\\\\\\/U2hYYPWn\\\\\\\/gbkaqq4tGkndHdssfpu5Fn98HkN\\\\\\\/5\\\\\\\/8zt4bkeiSQDcpKQjhtgcTdSrTaixRb4D7muw6WpnAuzdtYczAbtjMk4kzECPiWN0QU8UI8nJsYmwK0VAzt3wZM163HmoTHh0QpedWlphLGR74SazgeaKVoiYXvtJ2w3L0prfbOYlMu2PL+ObitM76jCc\\\\\\\/V2fNn2CODKYBCfEeGL0jPXyQNIjhVwpeUuILTdqp9rKq\\\\\\\/WbbgljI5IIbdCksIiqtg6wBAZbDM6vPNj5F+EemMVj7OK+WqRIZNYppK8fWYFG71b8SytqL+gV\\\\\\\/aaIuQAqwwUBKEgWmrryBJ5idTIdLJ6onjWnsLNlg6wXtdWtRwqdwWqiA4DAhSU0T9b3C8ecN2h0o5\\\\\\\/zDH2sQmX5tNx0TwECFKi5Q4hjX0MJvwP5WXl+\\\\\\\/ANgc6fLfPzTcbyAL5b\\\\\\\/bcxRgdsSuZessRvPvIlraCJvaNj5lPoP4OGViI3G2LUFKZnT5Lz8hdy9k0F9x7YZOyJmCDyz4qsRoQYwrnfTyk3shLX2jbHZVbmbP3zwcikrkcYZPFBhmpbYDorpfO0ddwTcuMLQwnrIHkVhRpSZxhy8pKiiCQidtvL83sDNrQPAZdlLzvF\\\\\\\/\\\\\\\/ybumDcApvMY1LEz\\\\\\\/toXJYHuBKvUBKdXnJBP12rLIf03EFlOV5o0LpjOVkU2MyCNMTLc\\\\\\\/POpciHbHH4vt1vxhZjdGGt1I+sftUwEojb5uPiV\\\\\\\/0ZHatUftORs7n9G23Y7AaUr2N1qLcSQ9leUlUinP78WGqcmN5AS2EkYaOQO4sHCpdhBsC6i4wA1hTps8OomHEg+KI+ZFIPkybmXAEqs+Nrt1JrhvPVvz+0dhmS6s892gpS90VisrerBOCfo0hnCOMjnTdxSYLh8zBuATT3U+87iFnyrtbVtCjpyWor2mxpxvTyJ4rW2GIxV8IwEsG8RfFSZYS0PGVj1OQPP6SjaQ6xc9nxDNDMbJcPT5ekQZhElfCCEeA9YW6NGqMX8zB9RzSsXNxKTdO5+sGmredJBkqSvBFAXodz63IWTPvDiVwtaqsvyqowK2i9KPfijAy+WwRhjfUck5PECV94HGq5F\\\\\\\/CsKUERtn34cWEdiS0p4sstAqvSuPP3oUgb4a\\\\\\\/LPpLqfhtLe\\\\\\\/a2R+Xc4DSpSlia042P7Kix4fZSIo2KCz3Rm2xRvbW8jwR091GsULLQCwMdhH8MgAsVzBiBgJxF8yczyDSRRTUjXTyDVhPr6htcOE91ElNbIzAz1lf\\\\\\\/JtSoj+jGYRszoHbG+him5oTZbVS54lnXZ5DPROp4KJo2CNX7edhn0xWtFFxxbVkiewO2FuKoVlkwe\\\\\\\/zu1xrG9IB3O\\\\\\\/jJGmh3rC3Ga6j3pal2KKy9Teocvp3kORkfYS99Yf\\\\\\\/w09tX1lgzwAQaVby4rIBrAFmzJlhVbx0pRrFlrlt\\\\\\\/RKY\\\\\\\/Ft1d6BMY2QjMh6bAkt49a\\\\\\\/QGQ8zA4A8NesP3BGpULcMBVNe0g4GN7MxvUov9nbgVUa8sACYGRYkf0FixRfwMo27lJRe4x1KbRI9j\\\\\\\/3Mt4XfwnoG6a89u6uMF81hdO5Q+IyvuRzkkiV18OzwDaA1iFWJBc7iqrOqyTDTrEyRB17Fnzq3V6QOJC\\\\\\\/SYoBQfNjfKCHyeHEnMzRUv0b7KZmuy\\\\\\\/dK1RM1LnqX\\\\\\\/jN3KjOMEtT9ESBvvsgBrgDf9s6jz78u1F3sxUX1w0J0u1wJFsvCyL\\\\\\\/NC5Dc7M6E0xwDGzuq9F+Ew\\\\\\\/55oeenApXb+yOfl8aBBhsQ9WwtULv8XzHNHpe65a42UqktvlHpLDymKnu+nVymoMyd71rgdC2JmuHaLDc18OdA1KzD7iVvgXNP45ijWdC4d5DhGZIhUpd1ih0go3x5Sj9srG\\\\\\\/p7evyAHTmz8BzKpG90xHRrTknheBHN6mJF4cImrqQ5dCX+7PNt7AnRsfLDaMIZOR0l8dYS8qSFw0jsXZ8phMGAGW+ot6l2AeKmn0fre695AVsbXmcxcOD00Ciln8fQLSI5JTtlNqH7\\\\\\\/GE8ees4\\\\\\\/MUzb623Ua5iBks5FA5JiH7m8tCli4Ui2\\\\\\\/q9JcQBeHPld2TE2TRAhjKTSEE4RvFiRtGhtIguR7dqXuTDmbvbYta8abwokBPkuFBiaYvPbXKo1yveB166Y+qOcb\\\\\\\/piFi1HPc8o3UbvnR8fZKfF5ljkZRMhlea6V3yLpm8Y0uweLFnaUZRrDiFnstPwxSDw\\\\\\\/g4DqsHF5sHCQIEZrtamLtgseR8+RO+y3rQq9D68fwaq\\\\\\\/laY5\\\\\\\/JTThZ+GYUdld2NoOW916Jax4YBCCbi\\\\\\\/xl12aeT\\\\\\\/1bNIP\\\\\\\/QJmYHptVFGO1RBmt0hOMppULIZmRxbqFdRq\\\\\\\/4gYp6Jb2VUexjALQNitnlN28yu9SadrqSxbXbMHjCnPKa5QlHhDgP4tr8Q9pJJZ4iUxRuhJP9LanXs5xz7pkRSv0KT6F\\\\\\\/1Ltm8pJVIRrxH5CdZqV8QuEvGUnycN\\\\\\\/CE2LKbNzhuyyYDNfkwg8zNTcityFalHKEIp0pvrFmw60EM9NK4Sx8FgpKgUsC7EBhLF0xFyAg09KKSY9hb\\\\\\\/Tpcawop1MctzGMRb\\\\\\\/s4VuhQ2UzqNTUjzTacn8zMO4SJMpn5PrpOMPLabmY3l\\\\\\\/62NiVhQkwcNHsJGUtXzt6ABcDXlz0x1cumur2jAXuzwvejScQY8SRCep2xdL6G14mDr7veimIbgvxqG+TWK8NZxN61m9BpHYgqJRXAMRBNoOzSvcSI4Gk7CaJBfTIC4\\\\\\\/lXZc9Y1wVdy4YR\\\\\\\/hT4WCNTVhCZo0HhNZ3ZUzGh5U2oinArjGb7WFE\\\\\\\/sMX3uB5t6rwZtPpng4Cu4xr1BSmjpEAQSLjmk+oHORmf7YTDwnAFdDUefNQ\\\\\\\/+yoZzKhheoZaNFn3zn1Vgrgq7eFg9YtcARx\\\\\\\/BmISL\\\\\\\/3fFg1ocOy8D1+Q0ipr8kwt+2aznhY0Sds2ogI1IpcdBRVrnsZugyFE5MZNGMqUFGnuY3SEuEbcbYoKNKQeudvppBOccl\\\\\\\/5\\\\\\\/Ty8qeRRytjBHqIB3MIudtZJRYm9QGa5e926WX+ZKXfFEFxqqYn4jp7FblLjDOeZCDc3Cp7qQCaFlb7+fH4eKU3+8En\\\\\\\/9VYrX85sO6L5CFkCQqZhmdusSVlPSAKoDOmGbABRATDur2fmAAUv6KleaV6MeoIrVk01SLZM1uMnyf4pLu4nFu9rkByMyADAjAJkkV0VfHUUIjN0a+BoaqImcZK\\\\\\\/iPkNL2Z4co4HwjsmLAlC2AJOBQ3rFS3szL4u2\\\\\\\/18PeoEFesNA1kJySHJ6\\\\\\\/78LDULNvvugLp2uIDzBpsKRQgW4eloIcxSFu5odESn\\\\\\\/oMQbmbW+1KJXnaZxEQ5yl7\\\\\\\/ONXktXXWYStkIOOpt9cm6pqUE2uBB5yj6j1y50f+Z7xyQ2rsza2tPwYr+eJ81rtVDhJhdyv2wXpCGDrQq8goNDh1mjhnnDBGe+9HzMERwt5tkT2UMZtp8rU+4EHBe4nJi+7KEufiBbHY0nDVsl\\\\\\\/NnU+A\\\\\\\/v2VnJTy6X38cLkzvCxmS2AAf4hnPNK0Wz6FyXNbwFKZy6JKFo\\\\\\\/QFtQmarcpgbDQNmB5sZuX8fl9Nk3XEOeEYsX+KP1F+NVQzjEwJx4vhkxBwQeRhtSVyc05VQAhQOHeSkoERukoyFJA\\\\\\\/4hiaUFxBqL1kE7s4EmaDTLHQnEsP5yvT9qPCAAb9oCc+n9db7eRP\\\\\\\/\\\\\\\/y\\\\\\\/J2FZ0tPkhnsgpGxvPQ9dtWpitnwgLzWWC0cZW6hmR+A\\\\\\\/H3IDq812raWfEYWCMr3Rz+ZEa0NJQ1vpmTAD+kf8nFQQQZyG39C0WKK1LNsoULajl+FP0K+QXGDjd7F2hx+4M9egh4THD22e6n9FNi+mrqPbrZhToAhWl2aawIBOusafK+Y0C1ISam54GzY9PsOLrbtKv0Dr0ogZvt4Hrfh6mnZkhmUcXh4apt0Ysdf1P6+e23YN\\\\\\\/+n8IXvGddNcclL+TEjRgbUcPg2QfQg5a3c32Hgi+R2hC4w7UNYpzCLS4Nlwti3WKCSaLYamxaqMwEFC69lHs6vGVGfdF1pAEgJznv9yC\\\\\\\/xhVzAKvGtMIectBMIz0xBD9ZJW65UQ9ot+bobmmpoCOkyoa4UJnA5u2Ny+5DwDFwjFNNwPhGlhWadKchgC\\\\\\\/fCV8KNooSjkmGVhMRNfPKRFoXDyrDkkNoJVOXRDqp84UUt4IDhJt3R6VZrsmtCpb+Uvr2jJ3a\\\\\\\/X8u0\\\\\\\/idD7WrSo0jM09DkQYApphLJZUlgbuA0wx2Bqmo\\\\\\\/7tUMEUXvDpiwi\\\\\\\/cbCxV5uZVLht5qIcKjVB16E1BA565g1vgJfhfKhkXzbubOWlHPTbSW1ij3SnvVEkHh6VzCn3COkx1oIFjFbRkO5cpJRyMZMVdY2KeekA9uHs9D0EWxHKZNA2ouJfTDxUviZWNcPxEbI8Eg4GwREW9wgj0xhViGkN+Q316iEC7msY5Kl1JE+gLyJKYP\\\\\\\/EzMq4kCUZOVFxVlPHGMoyiWwakfeQm2oaKe3biCNzDOfQ00Tcc69eIqQdU8Py8QSSpoU0HS8XH49Qwe2lTPK4iJOXDA6KNfHSpF6Gq\\\\\\\/v2ta18IZh7w28CNn+zjBWPwKTXEgpEGsv\\\\\\\/lIXkV+A9T57JISK2jbdeBaxM9tiYFOeFweSP9gAnTG6b5VihbXhk+XFyOX6T+6\\\\\\\/1dZV1ffxKQXUQDhL7RX93iP2Go0KrVZqbkBABZXylE0KZ13psi4xrcj8qTCwq9zTcuTSJ83tYtah1PMMBTUUKTFThTO9jImmaUxQQ58HoRj6uoOZEbbmunHQc8wwELUxWYk1qV54ReLMGXLQSk\\\\\\\/ORbyAjtEd+5gZdROpy2IcOwJmu3Aljy+4+HiO99XQIGDgcKDGaAypiiCxXEZZ15WybikXKrQrpSFG7wSKF3nNIme6n8\\\\\\\/4dcDuYw8J1pt+ZGz2BqrADGMy\\\\\\\/S7vaM++UMFpAHZnOtnLZFqcWJQKmo+XfMt3cDFqNSTt8ssOY8VJGjreWdlRwMR0048mVd5\\\\\\\/EuRL3tRxmuJ6XJ2m2Mc1o+DVqvjcdl00NpaOR5bBB7vccHdREXgwO6kyBcMrexPqE+PXWAxaxXcLZDnXsKgP6vBc1vwYEJ5PL+X3OqdkEA\\\\\\\/eYagwRh9eKkRE8JvuftfvUf3CHfkGJ\\\\\\\/CjdFhZ\\\\\\\/Sp7vP+WGr6XlM\\\\\\\/5hVCpSYedeiXlEXCe8Tqhy0ClRF+rUlgwWhjvJLF0IFELUwBbYjOZcd4ssgoMvq64Ozvrc8ExvNH2y3lTg6DlDwbge197915fxdq7mURmbtRiPzR9mE0+\\\\\\\/c\\\\\\\/cvpKdRJAPiav\\\\\\\/zQWCuidb6utamj2RyrqLyZHbvHQlpDewwlrIrAJxulmZa8FvrvkfYAGrs+IcpthtxaLCxURxhusZ0PPh2OhTZ5o4i1vFO6qWdosEQVLpd++LtSxVhFf3lYn7aBbR0Z2IBCOZWxMtAEEmjCjA4GXv2KsSBK4rOqILBR9W6lYRY3dklcoejPkb\\\\\\\/M64QlPfGg4A7Wz4uE5dJETpXmJCCNjfndZVBg9fHfIQyDJG5xHOJ6sav2w53oAG8O+QEOgwkZKKmQuE7htj75unqi2Kd\\\\\\\/nrDN5E9vGlvax7AifEanU07RYC1fxm8ZQ9a94IYl9MsLapGowmKfdP5hwcSsxLTGxJychFZnvaijFWOKNSFjcjnDVb7zmcHRysVz4LS92R07\\\\\\\/W5UCCbh4O3L83t3luE3ByisOeTha8SmkgxchGgwekcx441PjyJyb72mU\\\\\\\/fVHOV\\\\\\\/GCNxlTBgX3ApUhJtsrnEaO2wZu1dus1dGrocIULw1B1qgfqXIEn8uiOwy00\\\\\\\/QDJQ7faDKU+8T31CNHY+9ifCpfcVwT86bwJNdWtrJxsdfIIlAFuK+NsWt7aj33hWXKDWFd62UcI\\\\\\\/HSYMlzAVtnWU\\\\\\\/NgGRN\\\\\\\/h\\\\\\\/bORu+tzdtkKDgBCOcf1mDkTO00QHKVLZnr9v9vB3xOUsbGXCsfr91\\\\\\\/5RNCZ1VsUt1jYv561NuwLL6Lo769QnaBLSK0+epfFs6GgwVNXNEJmNiQkor9t8pa\\\\\\\/19TJSATIDfyuqxH151k6l47AAD4kHb9z5fJXG448xAUfCk7dyDxLxtM5VBTgGHCyeE+ILrQyqTaU7noKh5Ctde7Nwxzjb\\\\\\\/Oyz2fR81qafqfNHlJ3tw4vEVkmF5hO3Wpny2FtbDyGsXQSQkxUnDt3z+hdwj9s5PLQe7E1MEDPtpp9lZm0vgokfuQHrzxs6ajMdeuxR22iyJ\\\\\\\/zSI8RIawOLPDlj\\\\\\\/ySuBp9gmv0qaxZOgGYy7BJhDGwbwcbBEJjFHxzkaGK+Kt1xnXNId+TfEmiBJIiRyCAckJySM3WPSVHd61sRi1O2hYXkaXyu4oHQu78gWdE7xECv08wZwuvoXRYACckEZ9OXRmk9XxtsuE9vNsjIkilDuPBW+e7sgeLbeDcIolRbwXRwGan+qCQPIgA6R2I3xlrN\\\\\\\/LGYGjGlFmQDGs7xt5lm6iOpU9gQWPICQSd0HEqYOfdxeQGnZMz2T7X980ErSEODJerpL2npvT9BqKI5zmH7vwUx+Tqsmly8mSlzHDzreMfxlvUeLsywVT3au3WhnfBE3WZFUz7X0ORSuSRGsDogLllHns59CCYQ9FaiTV3QWgB4nKXmGUAcA6zH5RwfR9tKvMFYJEJeys\\\\\\\/jTnuSgVNunV57K\\\\\\\/uFxpcfowUm2zcGA23WVNrp28gDHbcSbYGLobfUEkTuu3o0dM7FOJeBZNZjsbn02sZM9MVk20pesOOipcnKMvFLb8gaCS2p6CD1cPieHp8O+acntO\\\\\\\/SsnQedF1wZtUqCPDgEel2NHYRPPS2JtoFddq+niSqKAaGtCk438pHsYQrr5nrk4EGYb829W8evBJOvRXRnj8vFSOpeseXBX2RlIwCmYiRVlFKcrJelzw9KbpzQYfacjJmj1kdz3i8gw36Myv7hPzMZxbbisaqo6p8dzI1FP+ozCQMH5z9oo0l2MAse0z3iF2Ez64hxfxK9gAQtuTW7Y2qZhRoEgN6cz\\\\\\\/hzhBz8LOxCt8jftDyy\\\\\\\/GpyHa+CHnVirpNA9l45d1RGjI5b4kvFyVDNTUU+x9legVVJEZGoqy4QTR1vfYj8innpPoViJ\\\\\\\/XX9etuBanylW23GLvFYzp6hoOAcnwfbtYR+utggBFuovBXeooAk6xLckEJIDmmb8AHFeM5eZJj1zGjouubw37BA+7hqsTbBQOKlSdSXPsrrugIKAWaAzuAA7eZwMoRb7FyIh9seRkUE7G1J08yWHmu42n8OxPmYo9p9tjc1bcnMMu5NiXSj+tifOVTf2E7jUGXkwXJoSQyN\\\\\\\/9PQ2IQj2BBVbQvc4U8O3j8C4hYg3M9b9JATg+WoMimNaOYP\\\\\\\/wGpCYxynrK21nxhbEkZZlncy9dY\\\\\\\/kC+f+0DrddfaohEFkoi56NEDQTSP3ZotZtZGGfu9sAuDhfLwbuPVUH9yHs8H+RvlrYC\\\\\\\/m5akqWCOXmRaJwyw4Var023AZyqyT5OGVkW\\\\\\\/Y8fDCC5cAJMgyH\\\\\\\/w19FL3USrOrL4foXfs8sf0YUK\\\\\\\/cXyirB1SA26T4ATeNGCVQzgpknSAIJKe8+hGRpGCGZB\\\\\\\/shn1COt59ytBbXDtci8qMWhCS8jZGKKr4T+x7ReI\\\\\\\/AUD8\\\\\\\/4AVrOSrR2ZJvNU2hIHzjoVpqMh2BOk9FT7q7y7VoF9YCYCxLl3VWXgtRjjjYB8QRHkkmXtA7s65MojJRpsYN38TnAP0Kz\\\\\\\/S9Qy9MHTIR9hYS0y\\\\\\\/wCc3Pkxg3AKjTvH1c3Vj3qyl1cJm7nOQWXhJhDKWAZyKMKEP3+IYnMqy6kzTj9B0Mjam5PVa9ih41OXpYJhuarwjGAefOtjW\\\\\\\/57bwPY+3KLL1dQ8Da\\\\\\\/aE56xH6+H2Rcs1wfP8NkAPTihJt70GxWe51ecj2udlsC+ziV4+SXiXu44iltTdRCin2H4L8R2DVlgDz5P3EXw4ArzSZnmOE\\\\\\\/ZvzFihPB5OVBGrstrq0Jn3myNkknBFFuBSW3C2v3qbKf8a\\\\\\\/eJweMGbKUSB7igh2ybUCCgxVcGH0lmIWRQ4FZM1KYwV1wD5ik7LKLffKateAEajN7Dxv10pBfyn6qrBRivaI9mQlspmXOyEti4Ruenr2kk6LMPTqAWVzsfEEh\\\\\\\/qhae3v820HGNZPxUrll\\\\\\\/OHKIw\\\\\\\/pLztzRREcqWSo6plhLJeDSjT83XPg\\\\\\\/a1Aed2ptgvNqEHBGHI+yv5+XOSqyVcYehjRG+yxZET67Y3VfjEQwt5lcyl\\\\\\\/bH78p0H16RJbWrBCbvLv1MaSAPdU23V78r6xxdbbp29uW5gXJFg4g5OZCUTqajLCScSBBhh8lwarOzoBKzq\\\\\\\/OaSsbB9Dj9innJ9X8M6jtt2xJUimONNRuJDDxXXfzhgq3lyAK2FbqEzpXTNGu6iqHsRWUK10FgV1e91Q8WmvEPSIJRb8KKY3nhhLxthbOoWOyBeY5njLTpGYncZX0K8108ZkfkwVoUhiHvi4iUYfSmLd18x5azfH6Pm5AAF7YKC\\\\\\\/VMqviVMADJLyEGYeTpXEnwxqYY8sY8zFg4rQdO8yRvnA5kxJi5UbvcT96vXUwltO0DOdfqbA9mb8Hn8\\\\\\\/UbTtRaYpsOK9Te1ZpzLo6R6\\\\\\\/dj597eHNWgmXaPSUCJvzkmY48aHMGga0HLuOfQPkGJ8\\\\\\\/0bawo7ebebtUDpDuUsK8qEHB0DP7Y7UGvHoU3WAmV\\\\\\\/ctat1e2NNQXOccattkx8VpVWb78najvRkA4k6hmBoaANCIEcOWEwyrHQT9bhNvtKfULi\\\\\\\/nYSQxOfWkCmy70QDZKDX55NaO0ZGDy4b+AqZyUmTcnLaMMwBzLHcG2EYzwQvZlYDpF10nMUzvfLXV+fQ6lRT+DNHGq8SIfHSr6PrkEcm1f2tuCb6knFSaBBatcB\\\\\\\/MLLJbnjJxbMN7BTMY0V3PSJ3jayK5nNsFIb3Y3N0aPGl2BhKd20GvOwH463HAaEYWZOR1ssPJUY5INtKk+VkB3wiblr9RQTPJjc34l5CFSr+5FUSS8DPoO1JeVKgabgGqd3dQyxT6A4nLKFw1HLHQ6R02Llz+qsgQq4oA4A34efjf\\\\\\\/sMncGPypfR6jgUWtwj+ecu0AcxQzdE2M3eeBL2wDi7ZoOPDn\\\\\\\/Ygbfs\\\\\\\/5rfiYLi\\\\\\\/vElhU\\\\\\\/itxr5ST8C\\\\\\\/GJ1PiI1BEtFzEaCktz\\\\\\\/YPx71rMhYeREEJ\\\\\\\/j2zhqJaSx4mjRp23+eAOPfD4ZoyycVcDV8rZrFHlrtr9cQ0ffBO2gt7Km8\\\\\\\/VlIa+84i6BhUAYv1aUuKSL4\\\\\\\/bEssO380JrX3OQno7PXL5KrdVkrxLhlFJRmy5i1T7U2fkkvBIr\\\\\\\/IqAI+8LP1Rps\\\\\\\/pDxdP3GYOLj\\\\\\\/31A1eY8+UZ\\\\\\\/YkOdwqmv3okCgoZEMOWz4rtVTe\\\\\\\/+yE3O8HAvCvLwWtRnoBwRHDu+ZKZj9OkzSfsc5jLdm8xNpZhON9a69eeSUTfgjYeaez6oxFSIZIgpnisoc0bvT70U2st1g7jL7QUhx8xCTuOvFkX+X1Uyn3nuqITTDeuSrfCI2DCZ+SMYrWxLQ+Y\\\\\\\/dqWFYWXOxYYTcEdy4Z0O0fTf+VJSToysHpg6zu9RmRvydVc6FbgY4xucHlkN+eiKu965ncgePjd5I\\\\\\\/aB20xWBXC239v9ikkey0FsESb4ywRwBVfmrOvolYGaZHa415Sfu1hbngG6Eha51k9AaJfmajktBPHaNZBwHqrqbUF+EvtKaV0dOhCXuqhiWxdCdBj1XSkIrFZfDNU6nsjttmmM3K\\\\\\\/VYmeCnlSLkKRuzIFzY4GNigN9corxSHQhBrqzSYsq6hBfl7S6Bun0Esp1ZGbAvpVTMqg9saXSpXK7bgLSYWpnimTGrMaLH92psxelH5+X+skemTeFIotsKwN3cgByyirlFu7cPBUCIzVEH3fE1FprMSmydFHcy1h4htTdjPH9jeX\\\\\\\/7jjx9GdbSBL+WnZRPn9WfZHSCRfDn3NuL\\\\\\\/cYrCJt246VpRaoRKsU1fVoCrRlMpnum4F\\\\\\\/kWaeme+YvaiYCQeRcYZCrCgJKXTM8Y+JxXDXPtIMgeifDtzZUKm0bAHRd5z9cL7p0V3m5wLipjLs8OgN9eWk7ZZ9YmKem7mDGN1H+sMr2E4XX2AQ\\\\\\\/2YiY3T8S1SduBMbkT0l\\\\\\\/dBgSn70Xz1d+v\\\\\\\/NZVe\\\\\\\/d80U59ESALmLSdtU4h6WI+ntDrDqPggboPrAmoVkYXmXC5uOkBpkdoE0iJ9cFQreJNKRJhA1hzZgbNn\\\\\\\/oNlkga5hVL9CKhTKYBMPZIYIG6j3TBpPKwaYCA8hEn+D0UQd5W3xQAVbBMMbrDr4jru73BZa+5d39iwCCU2tZRJxliMot+c7uw4SCUSe8prxTne275yy\\\\\\\/LvPktgFFh5CWBuBytdUAH2zPCdD0O5Rib\\\\\\\/Pc+q1POiEUxzr3BIRUVC572IMUJUmLUklPhtQzwqZCCO7OgnNj8gW00seN+HINu6dGEwcHYvdWaz0vxPQnjlSPf3OPT3qh0P+uEBOEPd2Dh91gwt3IVZ\\\\\\\/M9OginkNknOrS33OpLJJyddAicLOWaioqYF07DPt7XMnxch3xbBrgonSX+ijYi1ki7MwSsvC5DHxWM8g1wcZkVRcy7kXLMe0nXOOAMG6ObviG6YReM6bh0u4UfGHNup1MX9PQU6L1QzXNLTu\\\\\\\/zt8DbJAPukOtgAOrt5JhtQE0H2CUVgmtOuXxdvRcUatlypYHXpCZFdao9JZ9i7ehPJcZlHreraFIvqYqoUGuXCtTZgfGNs7n0KAaw2TyY+b\\\\\\\/XqnQjQOZDlDYkwc+eWIj1cBj4oXacsmlnuvcLmaWzVZkNFxEZ1o4ooGiMA0RoXB\\\\\\\/MEkgodeK\\\\\\\/SRCIa2HXC81afzVxsc5\\\\\\\/B1Cju4RrQyjZAmTEc++MsAQN7NJ+I17OyoJACCX7YSZMPurX9fHj0V9hHhW55Y64hEE5JbcQUvNCFWr4LXFoEgWqhNDkfuE\\\\\\\/tLhFOyNjuf37kelfMwO8N6nlSfLdfD\\\\\\\/mgvBrka0o\\\\\\\/IVYou3VaCxQHelq9kj1E2l3rXDX8AZrAyDxAhUPACmbK77ZTcv2lDiOvgKK9sZ3q70pAYgVVWeC\\\\\\\/W5vPuykAVVyMNKUfIAMss3MkxauNI60Us\\\\\\\/0WCA73kUldOGY64A58qMCp58B2zLIo1qDKItZsjqaBa32ixI8W2mPWAnVm7s5DHKBdcH\\\\\\\/o+PAlpFA4K0OXWLhztd91R6vGk7Mel4S+W4QllBwsny+o19hLee1sog5jci1977u0Eegvw0H6Zs+jByRHK8S+taqltDAQNNZZ0PRH78l4DCkwYTKESN+w3VUT3z2KOscdWsI8PeJsIvShybHBmpiYRdbZhlvahRhMkrojvVd1MrMhjlQUX9oZhLJiF+xdRy9lnBRjNHCqvPSOUa85pj\\\\\\\/HePD5WQhuXsZ\\\\\\\/+2Tn7nwD0MqVibWQwN9eYgcHwVGRu16Sh31iNXdhYr8OKM2hTu\\\\\\\/9HFj28O3BZGCoZRkcetT8LDhUoULeE2u417o8n6M6Qe3XFIAvA\\\\\\\/zENlj93ruqPZg5IhVDOr6F+0eSNVzTumj0cPj6Rk4oRjLBIV5gbBS+a8DzFvovwnJAwBB5tFjeLhdAtLjn8oWJh6Cp\\\\\\\/t4UJ4sPWDPTuZKhEnK1+KWkyJWNd+wDxg19qmJO7986b+WKnxWeG7mwT44iwTPdX\\\\\\\/rSbzmgWdq1a3QjERah9NH50z9DXhPvkiBS9zWGvNnyjUo2tbOEBlQlwzVt17dx8YmmBvx4iTIVgCgbcNab2JmNwQyJ\\\\\\\/iIUy0kVW3WBP7VTTjl5jA9P\\\\\\\/ARvjFmNHwj9ybRBj3VO31jWo4c+EeJnxZbXn3oIKmvRapums3iQt+++CZjw\\\\\\\/QQjOiPT55M98qUvzAJ9Ob\\\\\\\/4Mg\\\\\\\/chr8HBIwkOWkydF578xuEl+8ztqnPKrx2sD+OTFYtSW8D8N36BGvTx0ASIS15G+vEMMbafSgv+Byd5zSYWmwykjedUBBUVBqNN\\\\\\\/US7WQ0tgA8fPhEq+fJf7nNqer9rYPwscS7CigW378U3n11PONs5dZmmmZX8tzHTMBNuqxM3shK\\\\\\\/8EnrWxod56m84kWM00TAFjF+dqVPPjDlUPD5GiTXm8JDkpbySj8H\\\\\\\/kMU4YDmDl3v9swrvVetfCPhRp5Q5AfoMf82Ty0L7uKzCVplU5azAt15gAR5HllawyfHt0wo0OOolsN5icJI8Md4XNEJ47tMmqqVeQvOMPBdCFVu52M3lPbxMgG5Yydh8ElW2wDQDVZBnZoQ6MnCb2ky9uS0mJIT0E5mnbzhP3N9UCqD2D3HzSPXB9QwFnmXvDv4btPunhpOJhBn4ThQeB6qer4dZtCLwJ813jxgcpbqC0VJpKSADuBJWobS9c\\\\\\\/ULt1Wam3ZtvC9ycP7T7soYVP2juDFeJ4t\\\\\\\/O70Ll1SiNK6fNfCrYV8RNs1sDI9RXmrybHctiMmvw72Pey3\\\\\\\/ULdG86NZ19a0P5nwOf5veylIpgI2ZdGT8IP\\\\\\\/DJQRQLnbvUCKDbfkgG61hqJeytsEnbWqlxh7FpzoP3TombHcnO8CyOcp+VrFAIXU2eP9IQenoTi6dow6pJLNzf2ikI77pFGn9oqy7R3JnC5\\\\\\\/QX06ED6Q368gsDxezFnwAifbcp2mlrpxK4NFkSoo7J96WXTNqIjwGqsMuLwPwpuHGuqpYJUgGKj4mnPKsasXvmCVa758zx\\\\\\\/490IuZaNz9lIX8dFaeowXr2VZPK6iCXw83mMbBtx9wiCTrBUUJFNoM767z2AhFd3wouLLWO8Tr79Bu4LwKP0ATf4J1NBvCd3KMJ72+6s8kCA0Ey7Rx8vQHiT9ADocGjkd2VD9Zf+kSzFxSjzvdDQVPnaz6pXwXxIbVOEUjof\\\\\\\/0FD5ONCM1gKlaqRYDP\\\\\\\/YQtzW6a9OXBbcxlyHSNlsEf1kEA9OLHOYnmWcyYnqbmynPBa9i72iwhZ7KQB6dHT8FT4TQHUhyhShd6TmqE1znnYZQJLeIVZ1\\\\\\\/utw0eEheHNaxuqtypQ2Vjv6FiIrxZawjl8DOolQx\\\\\\\/lYkDjDqEWH5sFLjIMCBjQkfew1fSRvb6KOtCCn2ZooaC\\\\\\\/75CLAPpGPWy10fYwu0LjsLNGQbFCmuzbx5UU+XcTy62cfHBjCT0Lzb4j6j1AzuRnNC6u++ig0DdXAol16BtSQUmiFi74UE3qdNPaYfXQN8oc8AQ87PLrRA7fZyswOD8XW5o4ftXaBu7TAvJQddZoAp6b0CZCuoG5bU0tQ2oC8j2uTBYjTuCFNcxDTtUj3YIVgn6v4KMPYbY5WbZa1OIoviX8fTaY7BImrFys40NJCQC4aHDkZhDSpZroijyrnNFwJ6IiKnIV8qSt5Igr2xJZOCO\\\\\\\/XPuBgi4r2GZ9jWlHk4q+c5T9nQB66Mr336KnuYEkWHcI2SwAlDHNr0JNA1hnfzisHk\\\\\\\/MIHwrTtoUb9Edza8FXdNliMljkmDWWG0EP4HeRIplKKPsFcJq9sYj\\\\\\\/SFtLhW5pn7c6i5AeoN800iggEIN+YdwvbIx6NsrK8pfJHpeMHEO8LBQt4YQQLqUDPrHo1+QSRqPduCbWDhBkqST3X4EfnMPXOwOtORleTdUyzpuehLN8NDUrzZRKsooCyimkzVF0ozBMx0IwGlpQg2lnQN3+JY7I4OEcQsIIbKxMfTEOX9EQLu0XdoB\\\\\\\/UW0ZrplVAriqYIkiwe4cJp7OUY+I9eunryPpstYs9lwBHcH29nI4SmjoCPGX4HhjHBt5u4X96JKp4SREp5+e9vbBiRZWUXK2tphYSxONYRSrEL2fOogbVE3i7Coj2lvZNzofMVunhRzHCfy4DrACpLpLNFHkbn+w0bS8MMesYVTWmx4ZLGdCoi0frfOukSIDGWXkiiFm3a90hjdorbkxcHAjYkg470FcA+fnfdX3MToPf23jOF5wTyT01qYrVQdnnENc4Mo7bzknZW4s0UTYxdQzEfw0EwvqKuqjFhQuqFmriyPPJ\\\\\\\/0m92riYCVFUadH1O4ouse6JL0KBZt5vlqQ5nvaZuHaE\\\\\\\/8IaO6dsMwhMrTt4kfz31XPPTDAuz3ejPOleBO8QwmTUjTMg1LXvPBhN7gL3xyaICbHbFUBVRree7zUwzKAwwkAqM\\\\\\\/z6c0gTihytBI9jqd8QrR66s958LR+HHj18uURiWKX9ZLmWipmaVlSu+FHFyLMUCkO1VGpDp\\\\\\\/DMfQFEFvyPVfIr6ZE\\\\\\\/+pXei5EcSMWuARt8GG8K\\\\\\\/4TDLRZ2h7MH9ICJYtQ8zxEiiUuK\\\\\\\/X9pv8qKGhrZGcbmMAZQtFOt8HtbOUpWj4vGx1\\\\\\\/HvnXJkk2YN4QfVD175S7I4LcB0VxOZ1Q0Od5H1C1KpA2JKm8J0V9b20f9c+3mcANSkjpN8jBxEk9LFTsMTQhwPRAh5ZJW83vlbfBkjJrFvcX8Bcabmx8TH0ktAAfFKcXiPvdAY7pZTrGKDMRxw92rdMF1YOCItQ9hgShIGHLqbMI3GRBuKbmfv9L2xP6uDprnGnzRQRoCg5zmw7fHt2h6OQ6gaKcNqIeSmdPUmJF6NkQ+j7dizh9W8rGEQYW6uhJBNQgLxuC7PSIBrpYiVLX95lbUb9OZ+\\\\\\\/PxVQh4rACmaaLBENEYLm8YcTQpjOPKCLyAzIdCm+Aft40YEYF5inX7jW3AikeR0kc5nwuQ1zYX4G79KRd4kQvup05FqGXYGg9AUahdQPYxi\\\\\\\/tEYTowwlawArmv2ubdEslVdF\\\\\\\/SRj4RVncBLmyb3NJQvBqt8ea50noOP2R4mNf2jjL46TyNmPhzDb8dUGW7g4G8ehUTGxKmcJfuq2e2spLYYk8z6sQAq3j30gSMWs6FmezE6pdolN7Ul+\\\\\\\/ip7chlKVkAOHof3l2SwLFiWEIrsfhflsjC0lr6yUGBSdFSsf9T6QU0Lz8NtA0cK0+HukxMJuc9btpqlbBK0xKDgrsNII0dn7JknbMOtowIeXULSry6ONDUlB2iufqthpLkpnbCTApEz9rm0nW3uACaNs2RbblwKPkrHgRi94Re4\\\\\\\/tEdWsJ5xgwY0mKNMCwUd8la5DK10BuYZ9xIH6qRxBaCIuFsG0Ar3mPKaoYh5rAVNgJf6NGhl0bHm3yhbCeLkflfMgMJThf6g2CbTceTHCvjuuoGIGYWPJSyLs540Iah27J6DDnCEmPFM5ELuQCNBv0N6VN4Rl4E3ZFj79\\\\\\\/q+KQVqzCXT5MtXwAHjWFcbh4yuDaQX0FjbCkSOKE1zgOfHF3QZd6f79Yd5KGoP5Z1vtOFTeLo14F\\\\\\\/5uk3abk9VkjGdhPSS9dWpATX6LboCgbT\\\\\\\/pNRehXINQs4ODDlaljtuymnBM+BS462jWoSOR\\\\\\\/spC0gKOl\\\\\\\/xtp8o9w7ORoh\\\\\\\/9rx9j5luonIlfUmTgcLyx3bChS6c1OACWplBJR5bdKH0vORIUDHAU6m6GLz25uGkH3w0\\\\\\\/95d4aK4DUOep1sOWOls3U8+FKhUu+8lIe0L8CWiQFtKvCXfHV0JovrYgOXXnMCNTeHFcoLJv7g2dBUbFV05OBT\\\\\\\/yC59NLHbXKQpy4iCQbdaAaZQwyonngYMmGleYoI4G\\\\\\\/TnuhCEqD0kQNHb76spNJoZ5T8gLNbsV9\\\\\\\/CxS\\\\\\\/cvAp2lHu\\\\\\\/QMH+BIv2mjXAnGPW0al2602UNGGoq4kayh2IOWZ8\\\\\\\/1r1CefdMRj8UD\\\\\\\/0dlMRgAqnXdTZpXNkfjOCMVy9jme7KP\\\\\\\/5LZUh28kmWVVb8dOHzDLaAkillWKeRoiCXQej+uFuqEJ1HHFLZ+vYq2vQKMyUvz6rm+qBsLAdeDmFmFqybPfkPPBRdZu1DAAu7gajr\\\\\\\/tewT0aZWwIKzLUmIeAndblUX83f77AlU7swxxATsR13h76V3u\\\\\\\/mwEshdUDs73xp1jJrSfCssg3HempHvjnKG5z6\\\\\\\/w7QAb8W8vRoYOapqrpkH+MabG5UuVyAYvZnWBd\\\\\\\/HDcnnD+0p\\\\\\\/VugZ2wvBeA0GELJVhVIvsHD+vsHCtYhRStxEihORGWKOv7vVFHdMSPnMS7gCQ8waEEOBM6dZAcDBN4bgQCab2qBcPETtOh5V0yvauJy8VgWWbCqkYVD0B5eViblKn5xmKPXqt9ek\\\\\\\/7f+sGa5ZgF4lFyUgm+tmv50CPcz4v\\\\\\\/1csA5MxAHBIjiowScKcC7mB7FfVM3yMRN9lIqB6Z4sHPEt\\\\\\\/n4Qtan\\\\\\\/rLAtPnTUFCBVmBKiQgry0k61ByjblFr9h8iJVl3F1bFo4ePvyDtHcPbNNEUk9lKmFGlHqihPc3KT3ZliquYwkQV7revgQh2C21aCb1yEu8EhyWpBBBHHDhfONgSJsZi+y1LcbSnENcp3netiiXzKe6OSH2Bdj4RtVy3cjr6KauWSk6hlKsO0zRQ1AAmTs+nejYD7YBUVjIzKpAuSHPZD5dqq51hPTa7e\\\\\\\/l4G96Aj+kkwZbb4EE9WZCmTxo8YPCE1CuWQ6gzYrTmzXFuvTsFRGEGlv6ooW+eKwpBSFckk0X3OxJKl0xJB7tkooGZ8Su+kRULWRsEJs7C33+rqp1JWSY6RV+vousCR906\\\\\\\/Lqpq4sXWQhRSGSgro1Vyb7RDqZCO6FRISS5zPX6H6havVdUQpnljBmS53sLQK6nF7W9LUhPwnaczYM\\\\\\\/1MaPXe\\\\\\\/nvXExr000iejL7vMX06sKnC8HpiibnxpyWoEaxw031SQk4MLA0yCm\\\\\\\/7LMKo3tQO\\\\\\\/DPCzsRy4unCfVQkM5XHyKd0sxsU1KDUtxBflT3ArcOi0ZqZeGZbC+auhIMlmtV98IJVUCbujFBHuvvPAV0e\\\\\\\/7ByTXt9d\\\\\\\/0bfgXU47rlAN6JKSZJLnFUh85R71O0xS44RPAyFmViQf2ycwulEd9LoT\\\\\\\/9wri6FGPG6YxOAcr6PzsT+uoJVbwms+hcGCE1fVfyIOK3OA3Lx8oy56KBuMLX4khsTgEgjCXL3Z6ZLLCzEsDKcg+jXFOgJi85zEBZpVbI98y+4pwo7zch7brz5QShgT7Rz8+OVdNml372p\\\\\\\/\\\\\\\/jA7c2KE0TrwgdPYw6qVG5lQ6GZFVkk8W9OFvEQ7NmQ553bOwC\\\\\\\/ZT+C\\\\\\\/OvgiyWP1lYxx5AJriIN1nSd1GjhC+UYBv5lB46fmTiKnyTCdFPSOlF\\\\\\\/lKvjPLuOvcSWflGZusJk+5XTnvHJ7HzVIp5LkyQQDtsrC6u9UQVAXGu2IBnsZe7M3WD3XwgfIZixXK5\\\\\\\/qXPb4GB\\\\\\\/\\\\\\\/MfDURjXAjvR1CmkRVZhiz+ryLKpCbo5T5T0aYtaZ91ULCzrvqBbkub7KO3AuqP6aOvqCy\\\\\\\/LReiYnLhphVp+FJLE1mtVgmQldVuk4VUD3jusboCg587bP9ByHCDGl5+Sk45ZX+6ubAtKs3oKIowRgtdmCoHZN+Sa0Rt\\\\\\\/RpjnfxqojWBxidY+x+4Snxy5\\\\\\\/ZpjOO2mo0cwX7DEArWFwz\\\\\\\/JMI6+Be25CA7VN0TGmYZ0zOPQA75HUtQaBKXWlzj7+Avs46YzO2vdLC+93CECwYTbEM1pa7GR0+sCFMeRn8SdTg19KbVFgMiN8G0gJbADymIpVULCI0XkZ1PaMu1RCAvZkA3Y1wahEIY+XXvJcbc1Fx7TG0IRRgx+13F77GBjUiquegT3pP0KngvMjLypwRjTjVN2JqE+avZkeNVN9xC4eqB\\\\\\\/4wbZeSwikcgFsAyj0jwMQj716VSA7bq4jncCCBbgIwFpPUK6\\\\\\\/+lled37xOl6lLYQootg5DTWxyv7XGiK4Oi0EXQxCX9e\\\\\\\/tinWk5ovJC242HxpvPbagQkVODhKzPj76WYCUCwYx1\\\\\\\/lQrcd0sJeV0Tz80Jl1A0ATc\\\\\\\/krNF5rMh22CAMZM+T8d7I2efyWkph32h+Hso3RJ70kQjhPN742d9RcjpZ6GPbZxprf0LVCoWKgi3DQYZi42NCFNMgoLn8TefkoRo9JFe7kvUwQP+JaM6wMPYUAR7sWTy39+J55vgV\\\\\\\/MY29EGGZ3MA\\\\\\\/r4ie5g1x3sb0E3mqLLwIyznnBsqMP2DXEFVeoE+l2W0HrobhJjQr9mJgMVY3EaX\\\\\\\/FpjvcpSneUuI7V2NVHaw8SeC3fV2jMlFoswD\\\\\\\/+n6f4OapDXNv\\\\\\\/KEI\\\\\\\/3sdldovq2\\\\\\\/VxfIomniheFOCiUa7xU+a5+MwQkJmfpO7Zr\\\\\\\/GYDP1+H+5NE3reJJMQmtmq\\\\\\\/Dh2obZ5BL9qpsrP5PHRFCrbh1rkh3eB962vy3teIjRUTXcYKq59wxxeLKKDRuCHflvqHrLaDWBTNuasUEe+yWSKQKwM+5ry2PK6KhsSlxqeRmYunUf1eVCrYmFD3KfMCOGSGHqg\\\\\\\/\\\\\\\/6Y6osHMUFGoAG3O9ALgTEWzDthHvbQBskl9DTXQM9OYSYUIC5TgvA4yxXWpUoegKcHNRfsrW\\\\\\\/oGuDutswxDUZLjp9YIM7zxqyBg+nh+5LIBgSK8R\\\\\\\/N4Cu86+Cr8gW+sk+jMB+mA12TurV3zuzcusNC1sHq0jC6V9U\\\\\\\/yBxhvZWG6Sc+0MjetVMqBCeIPgehMUAe3VnyQJOnN47zAw3ZwWG3NpTydo9QNQprO2c\\\\\\\/6M93XIbxTepwLWAMfVgyPz4FXFAE\\\\\\\/N23iA1V+zPEHxKUCeeGwq5lqmApejU4ZQP0fSZgSKNyOlI7IbjrO9yxpyMbQHperT3mRN8Sww\\\\\\\/acXco+j5O9FnoOkEls58rTBCpreDiXQHse6aLyDuE2bIHPFpKaOVwjCma9UPrXNf\\\\\\\/8G+GBpJM5e+JG4iwWfL6pkPb+YApIigmlmfOEKiA3ikLFBccAnf1WscWKnmZzyu8soew27lF0+s68usqp1KtUfXgVq7eAkrPt8r5gZtVVQpNjsl28Y2YUTtmAPY+3ShVcIi4rTvaKzPul5+x5BQJTGKRdiSAuT6TL4TusYGDiXuImChmakNJu3n2gwYZbuzyp4uRG3PEKtbT5Cmn3UXk2WXObxGXbHmdhXrkIBgR6O55rja\\\\\\\/csfsKXWCsT+yu+tvHAGcTmaKiF20szI+wWYS5LqmbcovN8sZ44i7\\\\\\\/51ZoD7OvitMEEQbhxRwoeijwdher5liLd+IHiwDUKQg3I+5woXMUaRLh5YtCsTKOKC2twasMg1uKqhczTheAZGMgT99m7babsiOX9dFJDfJkAeTqwYNJmeLa9zT381nRAJQ3l0WzfQLmDpx6KTYhzLV8yo\\\\\\\/qyDwJrg\\\\\\\/GDNWSZULqqbVfjjj+pmhT\\\\\\\/IWbCn54Q9RXIdUpSgIkTR7GEvpT1J+djdPiu48dvPJmeGUKaCNikt2oXpM7OnYtz0OZubs6\\\\\\\/Jl851Oif2yxHUQHbprAHsrFqTD5\\\\\\\/OSGfJH9Mz64cUN7VXI5hOiFaEEiMA8L++Rua01Ymwd6cKlMDHMnBO5EBZTIfWo0LNsZCc5AFmFfuNl3TP\\\\\\\/WYm6dWeCQeOXcCc2gIlYtkt4iu1qm7bxLysfLo6sb\\\\\\\/1LLmI5RBw2978t7WalepVRgjEDw6pHON\\\\\\\/Bj75gYnGCHSdeKpnpxMRWIQi4lupjFUJJQIomAHwAkyS+Vndt9Si45jddUN1rE9ueVA+6SlfQ59GuSIBELY+GDnce50s4Tw4q6ICWvcs8+C166tXxFkm0KuSm+KBVcpOh4BFHeGObAuTQ9SrxuPdm\\\\\\\/3lw9NHT9vGQZoBkjolwrHDOwO00ZY7XkEJx3B+u\\\\\\\/ukh66N+WAcUrAthweCEQxsfGGHxeb1rNXq44wrkRT48zVfUTJmx\\\\\\\/cFAvVZCKZmWbJIMRQu4bKO20IOWb6uZpQ6AHhsBi76htwLjVzVs4LudSeyRbTdpO53AkHsj+5OUY6rmiHz84PvEtOUVJYVShisRAONUdCfoC30QRRnkIS98OZHHwfgk49YVE5IQ0aJrHWutNCJlZjZlrlVk\\\\\\\/g2bSCPZYCrTNsJUKkJmz0JVvmG8xvuamYFrdrJqr3gKefl104ZdiFiPOZNyS\\\\\\\/JaY0wMe647M4gI5m0D5bF2NwcJUT8QUEasQOt43YAaBytZjAQZPOcRXGyBjDEE9mP55YuhvPkobFImwLD0vlU\\\\\\\/kXLtmQg3yTRgJcDCs53aZX4gCrjK9qD4ROQWqQCWkgqCuX2R0gPF22DrWghktsOqIFGNZzJboDXBSoKZdL5LEquccYR+mM2i+2qOXcybIXGc0BJGDCQrxIhSOF2vLgzOQTxnnlOnxi\\\\\\\/CHzXSCMo7vx22QZTgjrCm0JiAdTA4ngJDz+nwV4ZeFQTf9FHugDd\\\\\\\/\\\\\\\/yOSUmt\\\\\\\/w+RmGSTF1u3TamJbTBb6+\\\\\\\/SpvCpb7DP\\\\\\\/1jMvlDvCqNIgVCT+pADPhtHfHJDK5A2p+CamCm5ZDdoPHk1yfK1uizKWDX3DjjT9wIEucs9vHR8ern2eNMzXT+oEfWn90DleH31\\\\\\\/CUr6jc5Gt3XAViMrXHdPaiyKjiihRKwRb6NOrQBDANfM8ibyZe62PqSDmc62bXltLY9jjtgGIrsUin9Q7bhr2hTxyide78vTqGzcGlckiQBwOrk5jWHyz6TxIMLMWQFGqGLaFuZYn+56lzIAD7P9hgC\\\\\\\/iOa5+d3QlHLsfbSYYRyr8XDwBBvHnCh33izLw2nXpn\\\\\\\/Xj85cFKP77oxiANOLWNVzVj+08aMkgp8vGOnFUUnCK4FIvFsu4ayMEUp\\\\\\\/fOz25xPon65r56WA8OCg4HFPL0AtFiLEq8a+Tt0nA3LbgDLI\\\\\\\/jj5PPMFAtlHhKQsvU6qfiuOZC7f94rFIKeExNS+Idz85YzhM0z4n9V85fMOKiDSM2JijSzM6wD5w+k939NJDC7qJGPCDczCd\\\\\\\/3UWDEI6q9DMx6SHj+5qM0MR9tt0xVCrJZs7knfzcQwtxodHJkTjUzJrm7JGVM0ax7r8zvoCeLjDr1Q2M6LY9Sb3HKYK6rvT9w12DfgO2WkFjcrhWaq2KyWvkRG8+VurgFfrgapjn9Z7MVm5QE928lW8c2HsGPRUzIUa3Q1d8qljcb0i8ZfV7G4ELcTQrSdFfhufCX4wz4+fqs+wPHJ6wdks0uFF5myfNlJMfEcoKmvFgFLnRuNcyHMaYTOsmJm3L4ot37mfq9xK9iMAaKlBC7Ufbl6\\\\\\\/axef4vj\\\\\\\/hr3EAfcF+aUjre7CASGbhL5syAkAggw5E8m5kzpNAQh7h3FRSaULzIbMy6nLWRrYenU3LLFra5RPzQJg0LZxmE\\\\\\\/yd2WhsF4SkUafDY4WCYPyn7nsHZ2J1Os9Yt39MxP8OYActSuqqfXdKwobSM7aEg0WdIwk9FwN3D9IF5QgLSHBgHdUA8Uu\\\\\\\/uS146h4B9avtzGV0M6jsfaL0qT6MJEd3Ip13SRu0uXvRLl58Zz9yZtS4cRDH3HCxmLYhCoas9u6pSUxHxqqurUTLatY9WP1aogk6gONAB\\\\\\\/6VnzvBZsADoZHoKGu04xzbQDEG8n1WUZK2ku4seuGet6VO9UY87krb0MFAqadqMOiD\\\\\\\/geXqxbX9zzmns3pzxpGytuc0n1lVY\\\\\\\/kUcuSoxPggyh35OY+LqDdHFpR0vJ4t\\\\\\\/ATLuBI8fXZuYWnuAeMUdqhjRuTXpxWwpqnq6viBU4nk3cPVsh96zHdpcGJdZ8mUPWSuycVywsMjD93o4\\\\\\\/f7t1fy5Tx9O38LbbN5QEypcdzMuuejJBG4FtZrTEMArKjnhLWEvSYSo0VvfNS+fxYW2J7ORA1sMPbNu9uMTBmjrNj++coggIg\\\\\\\/9U3HSEfe9LnbGWTu1hWSHuUPuIR0QfKjCQKaaCTi+qk+yFKHeLhA5qqAvf\\\\\\\/Mf59vQWTN+7VtILVTnxqlNhsleUEemAWEXbdSLBbx\\\\\\\/phxPobDCVzt2vPtAZuX5iV+RyYdRH0zHgSVWb9ddlKgTAKNIyQmjeJNJjTY8Wvle9\\\\\\\/GVMnrpYyhUXRpmP+D7DdcaLBGbQF+t5VCBcX4r22WIOCke0uzxXvtlyLhR3Sc\\\\\\\/hZi\\\\\\\/TxzoG3I+TXtaQBfUzNlaOAylah9QlOZ7BVmb4iyObYEslOTwb5qw6NkM+dwnV03wn41AVZ2awfVn\\\\\\\/NTcfY\\\\\\\/VB15tRA8ARLQ8k0Yu++f10VXFS0Jb9ekNoy8aTqOY0NW8fFjCuorOVj4OAZciST75tF7+Gh+LbAvaf7YfCPeZJcALL91hvy7dWXFfm40wN28zpKmbZ0\\\\\\\/gjxSRjCM4q37Mpk+FlpQZ1Ro5WwskJu1sRMsDfBO27v4LAP6XjCGHFBgw+2TIordL00dW\\\\\\\/J+lhyEHxocE8raJgaHfgQyqzZZsoRKJoqWaGOxUkgjnu9hm5Hzd8\\\\\\\/DP3Xb+YXe6DQzjY2azXUvJ6OFNSLE4SCjcrQt4IsHUdWIMo3evnioyzUtXsIuwDVdQzAJTLt7T2LML5pi7PBrJc5vqxg26SyBLJ3wV\\\\\\\/6TCNJUNITgPYDwGrFzPJBN7KfDidaMSpA8J5ugaD\\\\\\\/5os\\\\\\\/gTyIiO3\\\\\\\/\\\\\\\/Jrr8f\\\\\\\/Wy5yoJo+gICHmLzLUH0jUg7GqGV9UbuLEDlEeFZsz9hcZ8VGjDO\\\\\\\/VerwO1ijXALvRTA\\\\\\\/EG4zH\\\\\\\/8sRwP8eiSA2Txj0RB6faqinR\\\\\\\/hkFpT1bPr5qOt8iFIKLHf+x5ugouOAENqkZ7WMlTpo0j0pDCf1z+vMkIACueyYB9HXrEDqsIZKjIPEYb5T7ChbLgEtLXsHNvH6LC4VlA+zud3G82LrqhUWKFpV9k9\\\\\\\/YoTghPI1jeVdTatUysXISYxpkXQ1duVre2QQQAQHqNXM6CNYdSwWeExOdNUZeH2ZA5NgE\\\\\\\/\\\\\\\/QT1i\\\\\\\/2WIffxJdxTd7bJQ6\\\\\\\/0ecHbQgWes7UeuiJEGl4ueJ3UOha+\\\\\\\/108iMA0eWFmsIx7GB\\\\\\\/+0cAeTOQXRLHS44Js+H814OZhCUwycY+f\\\\\\\/BKUeUSwhJLjepH\\\\\\\/33R6ULDgusFxx0dXHPRcl7slB0+YsRYs982fJkT++7i9zpl\\\\\\\/GsNhg61YLYiyemO05h1mei44XHE9Q7Gf3XL+uyzWFfXLBA39DJoxrTvo0BodUE+jO2yuMaEDUbMV4p0s948O9P5PDfqyeax8nApEcFKM9+4xCVxhUmqofn1Tl5pNsIq0Ym6jRvq1Bb2GPdCd06CKQT9rQKTExTTGqzTJ1Qmv78ci5p+xXOijhdVQpK+ZZD\\\\\\\/bxcWX35RGg6\\\\\\\/mgFr5MZW\\\\\\\/euMCK8NpTKaxLBWw\\\\\\\/liqoWs2fGtupQ3UVQAFBtPt74mE5GNeJDqV+6Mtx3vDJrUCOE2qj5Ykqhucu+EDsKPezQCB6uNnGOzyMwhMV0LbPjiB8elZVs3e\\\\\\\/hDI5cw7oda4Ob1g0l1l5M3AqA\\\\\\\/LbVX5PF6aVAlMP0JZprA8TMFi5GsqCWYT6YC4pGE5lhpDYhY5ltQrOJ9l8s5Tr+BBJHVky0HUY\\\\\\\/oS4LtGeOzKhzqpUIy8IE8Bqo+lqaAwxbr3Gzb+Hx4xu6uh0bU6Lp3zSHu4sJygPpXEhVZ6fo2GEvGTZbRDbLUUhJQd8w\\\\\\\/k7SnSrzhCS+hI+jnzBZu0reUI32S7+\\\\\\\/wERJonXMxSptaQ5i\\\\\\\/bGlvZGQZ\\\\\\\/4UYcokxi4hmA4Hz6e+20U\\\\\\\/+lTo\\\\\\\/c41fwDY0FqPOiDeAg7n3nVUgtRM0rkzJGj7EaFjXEyiewPfsQsg2nAQpAhYkzf2LYi\\\\\\\/RPg3KFHBZ\\\\\\\/PYl8RwFmlrGxRwjwrU9huySf+6iJmdFJ\\\u0022,\\\u0022iv\\\u0022:\\\u002281tuTs6H6pKZtoZkixAt1A==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u0022f8a43f697c4911f995b80baaef0d9772a5dfecead2a50da0f204674eb2d8953c\u0022}","new-6":"{\u0022encrypted\u0022:\u0022{\\\u0022encrypted\\\u0022:\\\u0022ojy96YrPk+XQNAjKnh8F+ZIXGMdNnSFm6mFMejsEE6TRXIjUl72178nsNRTKK28PD3e9YeX\\\\\\\/hCsiU5FfNxH7HMqCIWvmotcDpH2EU1UUsO+EPOKXia2c3J7OBKNwd0NE4CbLaG2GNGckgz5oc4xzDEAFdNzobbOa23fzQqBbCSxRdm\\\\\\\/KDrgcjCRBFPW1OKS3y4i5DZa4uYY9SKI3RMlgoXFpd06GL0HUT04z6NnKGhdHUXlz2tE1J7IB8PCTZN3WCTTZfBE7grG2O5Gspp2B1e7jk1nB8bbNl443tMi3xPZRrhHQknqjAW\\\\\\\/3FnJG41y3omgCHmt018Ga9uvD8D99zJ0UCl+q1HXCbkqKWPM7u8fweQIo3c3Me55H64vX4yJcqopgiTo1JdYsOFbGEV4XbmZRRIrIc\\\\\\\/yt3wqS4hP3XUwXsHNd2j+dcwzwIaGR7L63fjRzNASFTrBGyjU5HkPRcJreGS4byqYUJr+\\\\\\\/cW1fjHUBQNEOX\\\\\\\/aRB2BlEjk7sCHtHcfsSoz\\\\\\\/LAb9wDH+u72Myb\\\\\\\/Jhrp\\\\\\\/zIHUHE\\\\\\\/Df\\\\\\\/4GlGAIFH\\\\\\\/5jqmhSq+jc3ALQwy7OrBuPAoSpGrruoHgROjf6+aXwXpbAoL9YjsJmTNbai3PZWzdbgNzTUjqLk25Z8B8h0W+qCIlb+0NLX+7t1\\\\\\\/wRXZGRbuzkQbU4fvl\\\\\\\/TRb1y3uzxAplzW3G7nWN+Yr2Ww7YC1rlDFTKBkZSTnY48z8ICIy4Mc0yunKTHE+qzdyqFQLsRiAkdnvl8jMUhCcLxLAuHJ27ddrComo8pC4bv4XWui\\\\\\\/lqYvKJ26Fp\\\\\\\/2I0oPoslvg1OHbg2xCOJ\\\\\\\/xt7LLOaojLyHq7qCWox\\\\\\\/XaOyuSoVCiC0c3Bx2E6l\\\\\\\/ts\\\\\\\/j2baVxx0cD39MvPtIhInfPv0FIBs1S1pXTi5FdrCTy4IErNMm7pZV5zUcaoNcap+VaLU7OV9C6R2ExcuYsJ5V1crpn0rsYUm9tajqGnSxeTjAvuA6zNFVanxDa5e6ppLDdpgMTx0+RrQjXd0QQWqDvQneD1B3BbiBppMabVhBOxM3lIP8Sa4Qd6YQrFBckl1shoASyD540MpzmokNh1OG2FZzNzw5iicV0cUYDQTz5fHwnjjOi8lIhrTr9DqrI4qRx04zPtp9+enWWtWg6wKXfjI7E4ymZYOlIMQIIeouOJUuHtoaJaxWG6cbefjSS3yuOtm5W3QZm9H\\\\\\\/cN4aJwux\\\\\\\/181Q\\\\\\\/K3N4SN8WFtcvS+gMcTY+C\\\\\\\/fLKb8pEzHZC29JSIz4JG2aALVUJbNgYi3er2UbMaWhvndBAchz9YAX5VNbYsHXvkX\\\\\\\/BndntT9LynmKt4NtIGB9nKY5eQ\\\\\\\/NiProNKue02EfeSbUlXrbSAef1LC3PQw0lg1hts3teztD8RxlW1Rsnb5tmqh7vb68JUiEiTMjqk3EgqtloY0Lrkq5+A6FvbEhWWQdRPggx2ZLadGV3n9IPeKIBUvuLWOfE4K6K7Yw0eEQ7hgKrBug6p\\\\\\\/2yIcThdpeD01JN94G6pyVDbTLu3QebkuagU5Hw4qSAh90vjkHHcmo35GuK7U\\\\\\\/x8p2YWeiOKdkXEKIOObB0VpWdh7oJs95pFzHVoxzykaMSEk\\\\\\\/3QfpHRSc+MDzRmRiqNPOxRKhUwHpu1rhk9sgLwdTPT8vJx2qRdZpL6OGbc1z\\\\\\\/pk398o9AItmXD92Eg1NK4tSNV2ptrknmeE7esAcoNW4mNpZWHK0H0gmP+aEYEN5Rjxp+mXlKNSvCAKQ\\\\\\\/InxhM7yiFbCKekOFjheIfwTpt9+199CDKTjEOa0FFw7h+7JAEnBfebB4mzPrIlGn6fuDU2XiP5+7lriQKDfRQJvmJjO\\\\\\\/vzlIkLbGEuD\\\\\\\/tK4phYbq6NPw9k7Xm0jgkTMlE7VYiRpvS1loyatloqslsorzpseVPqRHON4Px8LiQXaqsP9fMT60s\\\\\\\/ABP7XJ9on\\\\\\\/L15nNymDT4uB8Ucs3wD2hX1YfrV0Zmu416mt9n9CY5QOvT+nfAOUgLHjA8wpbq\\\\\\\/\\\\\\\/67nAuJDRZvJdJglWPPKux1aHsnPS8HakhIKY6cE=\\\u0022,\\\u0022iv\\\u0022:\\\u0022HGTwr5q9Q6giHPXn85UBUA==\\\u0022,\\\u0022salt\\\u0022:\\\u0022eafb7996b1fcf3b52eea931fa115389d\\\u0022}\u0022,\u0022hmac\u0022:\u002275676e019729533c56241a35297922111852c3b141c0318ab9bf4aacdcef4353\u0022}"},"ajax":{"edit-submit--4":{"callback":"highwire_alerts_login_form_submit","wrapper":"alerts-form-wrapper","event":"click","url":"\/system\/ajax","submit":{"_triggering_element_name":"op","_triggering_element_value":"Submit"}}},"panels_accordion":{"highwire_article_accordion_container":{"heightStyle":"content","autoHeight":false,"collapsible":0,"region_accordion_id":"highwire_article_accordion_container","active":0,"animated":"slide"}},"highwireResponsive":{"enquire_enabled":1,"breakpoints_configured":1,"breakpoints":{"zero":"all and (min-width: 0px)","xsmall":"all and (min-width: 380px)","narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}},"googleanalytics":{"trackOutbound":1,"trackMailto":1,"trackDownload":1,"trackDownloadExtensions":"7z|aac|arc|arj|asf|asx|avi|bin|csv|doc(x|m)?|dot(x|m)?|exe|flv|gif|gz|gzip|hqx|jar|jpe?g|js|mp(2|3|4|e?g)|mov(ie)?|msi|msp|pdf|phps|png|ppt(x|m)?|pot(x|m)?|pps(x|m)?|ppam|sld(x|m)?|thmx|qtm?|ra(m|r)?|sea|sit|tar|tgz|torrent|txt|wav|wma|wmv|wpd|xls(x|m|b)?|xlt(x|m)|xlam|xml|z|zip","trackColorbox":1},"omega":{"layouts":{"primary":"normal","order":["narrow","normal","wide"],"queries":{"narrow":"all and (min-width: 768px) and (min-device-width: 768px), (max-device-width: 800px) and (min-width: 768px) and (orientation:landscape)","normal":"all and (min-width: 980px) and (min-device-width: 980px), all and (max-device-width: 1024px) and (min-width: 1024px) and (orientation:landscape)","wide":"all and (min-width: 1220px)"}}}});
//--><!]]>
</script>
    <!--[if lt IE 9]><script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script><![endif]-->
  </head>
  <body class="html not-front not-logged-in page-node page-node- page-node-460344 node-type-highwire-article context-content hw-default-jcode-ggg hw-article-type-research-article hw-article-category-investigations">
<!-- Google Tag Manager -->
<noscript><iframe src="//www.googletagmanager.com/ns.html?id=GTM-PZXJ6LS" height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
<script type="text/javascript">(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0];var j=d.createElement(s);var dl=l!='dataLayer'?'&l='+l:'';j.src='//www.googletagmanager.com/gtm.js?id='+i+dl;j.type='text/javascript';j.async=true;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-PZXJ6LS');</script>
<!-- End Google Tag Manager -->
    <div id="skip-link">
      <a href="#main-content" class="element-invisible element-focusable">Skip to main content</a>
    </div>
    <div class="region region-page-top" id="region-page-top">
  <div class="region-inner region-page-top-inner">
      </div>
</div>    <div class="page clearfix page-box-shadows footer-borders panels-page panels-layout-jcore_2col" id="page">
      <header id="section-header" class="section section-header">
  <div id="zone-user-wrapper" class="zone-wrapper zone-user-wrapper clearfix print-hidden">  
  <div id="zone-user" class="zone zone-user clearfix equal-height-container mobile-hidden print-hidden container-30">    
  	    <div class="grid-11 prefix-1 region region-user-first equal-height-element" id="region-user-first">
  <div class="region-inner region-user-first-inner">
    <div class="block block-menu social-media-menu block-menu-social-media block-menu-menu-social-media odd block-without-title" id="block-menu-menu-social-media">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-17 suffix-1 region region-user-second equal-height-element" id="region-user-second">
  <div class="region-inner region-user-second-inner">
    <div class="block block-nice-menus block-2 block-nice-menus-2 odd block-without-title" id="block-nice-menus-2">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-umbrella-menu" id="nice-menu-2" role="menu"><li class="menu-653 menuparent  menu-path-front first odd last" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><ul role="menu"><li class="menu-654 menu-path-genetics-gsaorg first odd " role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="menu-744 menu-path-geneticsorg  even " role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="menu-743 menu-path-genestogenomesorg  odd " role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="menu-745 menu-path-genetics-gsaorg-conferences-  even " role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="menu-746 menu-path-geneticscareersorg  odd last" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div><div class="block block-panels-mini block-jnl-genetics-uid-strng block-panels-mini-jnl-genetics-uid-strng even block-without-title" id="block-panels-mini-jnl-genetics-uid-strng">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_uid_strng">
  <div class="panel-panel panel-col">
    <div></div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-genetics-user-header block-panels-mini-genetics-user-header odd block-without-title" id="block-panels-mini-genetics-user-header">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_user_header">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-panels-mini pane-genetics-logged-out-links" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-genetics_logged_out_links">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-menu-user-menu-unathenticated" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-menu-user-menu-unathenticated-1 menu-name-menu-user-menu-unathenticated parent-mlid-menu-user-menu-unathenticated:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last leaf menu-mlid-748" role="menuitem"><a href="/user/login?destination=front" data-hide-link-title="0" class="" data-icon-position="">Log in</a></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</div>  
  <div id="zone-superheader" class="zone zone-superheader clearfix mobile-only print-hidden container-30">
    <div class="grid-30 region region-superheader" id="region-superheader">
  <div class="region-inner region-superheader-inner">
    <div class="block block-panels-mini block-hw-small-logo block-panels-mini-hw-small-logo odd block-without-title" id="block-panels-mini-hw-small-logo">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-hw_small_logo">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-small-logo" >
  
      
  
  <div class="pane-content">
    <a href="/" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/G3logo.png" alt="G3: Genes | Genomes | Genetics" title="G3: Genes | Genomes | Genetics" /></a>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div><div class="block block-panels-mini block-responsive-menu block-panels-mini-responsive-menu even block-without-title" id="block-panels-mini-responsive-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu">
  <div class="panel-panel panel-col">
    <div><div id="unique-id3" class="highwire-responsive-toggle-group"><div class="panel-pane pane-panels-mini pane-responsive-menu-main-menu" >
  
        <h2 class="pane-title"><i class="icon-reorder"></i><span class="element-invisible">Main menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_main_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-menu-tree pane-main-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-main-menu-1 menu-name-main-menu parent-mlid-main-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-661" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="expanded active-trail menu-mlid-662" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active active-trail" data-icon-position="">ISSUES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first expanded menu-mlid-663" role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="leaf menu-mlid-664" role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="last leaf menu-mlid-665" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-666" role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-674" role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="leaf menu-mlid-996" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="leaf menu-mlid-676" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="last leaf menu-mlid-672" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-673" role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1417" role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="leaf menu-mlid-1278" role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="last leaf menu-mlid-1279" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-1288" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-1289" role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="leaf menu-mlid-1251" role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="leaf menu-mlid-803" role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="leaf menu-mlid-1277" role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="last leaf menu-mlid-1380" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></nav></li>
<li class="expanded menu-mlid-677" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-680" role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="leaf menu-mlid-793" role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="leaf menu-mlid-790" role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="leaf menu-mlid-789" role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="leaf menu-mlid-681" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="leaf menu-mlid-791" role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="last leaf menu-mlid-792" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></nav></li>
<li class="last expanded menu-mlid-682" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-679" role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="last leaf menu-mlid-796" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-menu-tree pane-umbrella-menu" >
  
      
  
  <div class="pane-content">
    <div class="menu-block-wrapper menu-block-ctools-umbrella-menu-1 menu-name-umbrella-menu parent-mlid-umbrella-menu:0 menu-level-1">
  <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first last expanded menu-mlid-653" role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">Other GSA Resources</a><nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf menu-mlid-654" role="menuitem"><a href="http://www.genetics-gsa.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics Society of America  </a></li>
<li class="leaf menu-mlid-744" role="menuitem"><a href="http://www.genetics.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genetics</a></li>
<li class="leaf menu-mlid-743" role="menuitem"><a href="http://genestogenomes.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">Genes to Genomes: The GSA Blog</a></li>
<li class="leaf menu-mlid-745" role="menuitem"><a href="http://www.genetics-gsa.org/conferences/" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GSA Conferences</a></li>
<li class="last leaf menu-mlid-746" role="menuitem"><a href="http://geneticscareers.org" data-hide-link-title="0" target="_blank" class="" data-icon-position="">GeneticsCareers.org</a></li>
</ul></nav></li>
</ul></nav></div>
  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-user-menu" >
  
        <h2 class="pane-title"><i class="icon-gear"></i><span class="element-invisible">User menu</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_user_menu">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-user-identities" >
  
      
  
  <div class="pane-content">
    <div class="highwire-user-message" data-separator="|"><span class="highwire-user-institution" data-id-type="institution" data-identity="a%3A2%3A%7Bs%3A4%3A%22show%22%3Bi%3A1%3Bs%3A7%3A%22message%22%3Bs%3A50%3A%22Institution%3A%20%5Bidentity%3Ainstitutional_display_name%5D%22%3B%7D"></span></div>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-pane pane-panels-mini pane-responsive-menu-search" >
  
        <h2 class="pane-title"><i class="icon-search"></i><span class="element-invisible">Search</span></h2>
    
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-responsive_menu_search">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-qsearch-nocontext" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/10/3169" method="post" id="highwire-search-qsearch-nocontext-form" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search..." type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--5" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-AYhY0wCqcjmqVoMfePmRVwTKjk-G1Fg1l3gwA4PHPEo" />
<input type="hidden" name="form_id" value="highwire_search_qsearch_nocontext_form" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-1 highwire-responsive-advanced-search-link pane-menu-tree" >
  
      
  
  <div class="pane-content">
    <ul class="menu">
  <li><a href="/search" title="Advanced search">Advanced search</a></li>
  </ul>  </div>

  
  </div>
</div>
  </div>
</div>
  </div>

  
  </div>
</div></div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-branding" class="zone zone-branding clearfix mobile-hidden print-display-block container-30">
    <div class="grid-19 prefix-1 region region-branding print-display-block" id="region-branding">
  <div class="region-inner region-branding-inner">
        <div class="branding-data clearfix">
            <div class="logo-img">
        <a href="/" rel="home" class="" data-icon-position="" data-hide-link-title="0"><img alt="G3: Genes | Genomes | Genetics" src="https://www.g3journal.org/sites/default/files/G3logo_1.png" /></a>      </div>
                </div>
          </div>
</div><div class="grid-9 suffix-1 region region-branding-second print-hidden" id="region-branding-second">
  <div class="region-inner region-branding-second-inner">
    <div class="block block-panels-mini block-jnl-genetics-search-box block-panels-mini-jnl-genetics-search-box odd block-without-title" id="block-panels-mini-jnl-genetics-search-box">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_search_box">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-seach-quicksearch no-margin-bottom" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/10/3169" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords--2">Search for this keyword </label>
 <input placeholder="Search" type="text" id="edit-keywords--2" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit--6" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-Y85tpZhP_RAXQNN8LXEOnNxTlDEo9qmY2zdmCJU37pA" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-custom pane-2" >
  
      
  
  <div class="pane-content">
    <p><a href="/search">Advanced Search</a></p>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-header" class="zone zone-header clearfix mobile-hidden container-30">
  	      </div>
  
  <div id="zone-menu" class="zone zone-menu clearfix mobile-hidden container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-menu main-menu-active-caret" id="region-menu">
  <div class="region-inner region-menu-inner">
        <div class="block block-nice-menus block-1 block-nice-menus-1 odd block-without-title" id="block-nice-menus-1">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="nice-menu nice-menu-down nice-menu-main-menu" id="nice-menu-1" role="menu"><li class="menu-661 menu-path-front first odd " role="menuitem"><a href="/" data-hide-link-title="0" class="" data-icon-position="">HOME</a></li>
<li class="menu-662 menuparent  menu-path-content-current active-trail  even " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="active" data-icon-position="">ISSUES</a><ul role="menu"><li class="menu-663 menu-path-content-current first odd " role="menuitem"><a href="/content/current" data-hide-link-title="0" class="" data-icon-position="">Current Issue</a></li>
<li class="menu-664 menu-path-content-early-recent  even " role="menuitem"><a href="/content/early/recent" data-hide-link-title="0" class="" data-icon-position="">Early Online</a></li>
<li class="menu-665 menu-path-content-by-year  odd last" role="menuitem"><a href="/content/by/year" class="" data-icon-position="" data-hide-link-title="0">Archive</a></li>
</ul></li>
<li class="menu-666 menuparent  menu-path-node-1  odd " role="menuitem"><a href="/content/permissions" data-hide-link-title="0" class="" data-icon-position="">ABOUT </a><ul role="menu"><li class="menu-674 menu-path-node-7 first odd " role="menuitem"><a href="/content/about-journal" data-hide-link-title="0" class="" data-icon-position="">About the journal</a></li>
<li class="menu-996 menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">Why publish with us?</a></li>
<li class="menu-676 menu-path-node-8  odd " role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial board</a></li>
<li class="menu-672 menu-path-node-10  even last" role="menuitem"><a href="/feedback" data-hide-link-title="0" class="" data-icon-position="">Contact us</a></li>
</ul></li>
<li class="menu-673 menuparent  menu-path-collection  even " role="menuitem"><a href="/collection" data-hide-link-title="0" class="" data-icon-position="">SERIES</a><ul role="menu"><li class="menu-1417 menu-path-g3journalorg-content-series first odd " role="menuitem"><a href="http://www.g3journal.org/content/series" data-hide-link-title="0" class="" data-icon-position="">All Series</a></li>
<li class="menu-1278 menu-path-node-483320  even " role="menuitem"><a href="/content/genomic-prediction" data-hide-link-title="0" class="" data-icon-position="">Genomic Prediction</a></li>
<li class="menu-1279 menu-path-node-483321  odd last" role="menuitem"><a href="/content/multiparental-populations" data-hide-link-title="0" class="" data-icon-position="">Multiparental Populations</a></li>
</ul></li>
<li class="menu-1288 menuparent  menu-path-g3journalorg-content-article-types  odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">ARTICLE TYPES</a><ul role="menu"><li class="menu-1289 menu-path-g3journalorg-content-article-types first odd " role="menuitem"><a href="http://www.g3journal.org/content/article-types" data-hide-link-title="0" class="" data-icon-position="">About Article Types</a></li>
<li class="menu-1251 menu-path-node-458471  even " role="menuitem"><a href="/content/genome-reports" data-hide-link-title="0" class="" data-icon-position="">Genome Reports</a></li>
<li class="menu-803 menu-path-node-457473  odd " role="menuitem"><a href="/content/meeting-reports" data-hide-link-title="0" class="" data-icon-position="">Meeting Reports</a></li>
<li class="menu-1277 menu-path-node-483319  even " role="menuitem"><a href="/content/mutant-screen-reports" data-hide-link-title="0" class="" data-icon-position="">Mutant Screen Reports</a></li>
<li class="menu-1380 menu-path-g3journalorg-content-software-and-data-resources-0  odd last" role="menuitem"><a href="http://www.g3journal.org/content/software-and-data-resources-0" data-hide-link-title="0" class="" data-icon-position="">Software and Data Resources</a></li>
</ul></li>
<li class="menu-677 menuparent  menu-path-node-1037  even " role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">PUBLISH &amp; REVIEW</a><ul role="menu"><li class="menu-680 menu-path-node-11 first odd " role="menuitem"><a href="/content/scope-and-publication-policies" data-hide-link-title="0" class="" data-icon-position="">Scope &amp; publication policies</a></li>
<li class="menu-793 menu-path-node-4  even " role="menuitem"><a href="/content/submission-and-review-process" data-hide-link-title="0" class="" data-icon-position="">Submission &amp; review process</a></li>
<li class="menu-790 menu-path-node-111849  odd " role="menuitem"><a href="/content/article-types" data-hide-link-title="0" class="" data-icon-position="">Article types</a></li>
<li class="menu-789 menu-path-node-111850  even " role="menuitem"><a href="/content/prep-manuscript" data-hide-link-title="0" class="" data-icon-position="">Prepare your manuscript</a></li>
<li class="menu-681 menu-path-node-1038  odd " role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit your manuscript</a></li>
<li class="menu-791 menu-path-node-6  even " role="menuitem"><a href="/content/after-acceptance" data-hide-link-title="0" class="" data-icon-position="">After acceptance</a></li>
<li class="menu-792 menu-path-node-5  odd last" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">Guidelines for reviewers</a></li>
</ul></li>
<li class="menu-682 menuparent  menu-path-node-3  odd last" role="menuitem"><a href="/content/institutional" data-hide-link-title="0" class="" data-icon-position="">SUBSCRIBE</a><ul role="menu"><li class="menu-679 menu-path-alerts first odd " role="menuitem"><a href="/alerts" data-hide-link-title="0" class="" data-icon-position="">Email alerts</a></li>
<li class="menu-796 menu-path-node-1039  even last" role="menuitem"><a href="/content/rss" data-hide-link-title="0" class="" data-icon-position="">RSS feeds</a></li>
</ul></li>
</ul></nav>
    </div>
  </div>
</div>  </div>
</div>
  </div>
</header>    
      <section id="section-content" class="section section-content">
    
  <div id="zone-content" class="zone zone-content clearfix container-30">    
        
    <div class="grid-28 suffix-1 prefix-1 region region-content" id="region-content">
  <div class="region-inner region-content-inner">
    <a id="main-content"></a>
                        <div class="block block-system block-main block-system-main odd block-without-title" id="block-system-main">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panels-960-layout jcore-2col-layout" >
	  
  <div class="panel-row-wrapper clearfix">
		
		<div class="main-content-wrapper grid-17 suffix-1 alpha">
			<div class="panel-panel panel-region-content">
			  <div class="inside"><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/10/3161" title="Efficient CRISPR-Mediated Post-Transcriptional Gene Silencing in a Hyperthermophilic Archaeon Using Multiplexed crRNA Expression" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/10/3185" title="An Eye on Trafficking Genes: Identification of Four Eye Color Mutations in Drosophila" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460344" id="node460344" data-pisa="ggg;6/10/3169" data-pisa-master="ggg;g3.116.030536" data-apath="/ggg/6/10/3169.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A New Advanced Backcross Tomato Population Enables High Resolution Leaf QTL Mapping and Gene Identification</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3778-4107" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel Fulop</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-3447-4442" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aashish Ranjan</span>, <span class="highwire-citation-author" data-delta="2">Itai Ofner</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-0555-6409" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michael F. Covington</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-4875-1447" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel H. Chitwood</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-0624-8034" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Donelly West</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-1935-4397" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yasunori Ichihashi</span>, <span class="highwire-citation-author" data-delta="7">Lauren Headland</span>, <span class="highwire-citation-author" data-delta="8">Daniel Zamir</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-9623-2599" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Julin N. Maloof</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0002-1494-7065" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Neelima R. Sinha</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3169-3184;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.030536 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460344" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel Fulop </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BFulop%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fulop%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BFulop%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3778-4107" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel Fulop</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Aashish Ranjan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aashish%2BRanjan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ranjan%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAashish%2BRanjan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3447-4442" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aashish Ranjan</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Itai Ofner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Plant Sciences and Genetics in Agriculture, The Hebrew University of Jerusalem, Rehovot, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Itai%2BOfner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ofner%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AItai%2BOfner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Michael F. Covington </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BF.%2BCovington%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Covington%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BF.%2BCovington%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0555-6409" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael F. Covington</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Daniel H. Chitwood </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BH.%2BChitwood%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chitwood%20DH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BH.%2BChitwood%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4875-1447" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel H. Chitwood</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Donelly West </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Donelly%2BWest%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=West%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADonelly%2BWest%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0624-8034" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Donelly West</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Yasunori Ichihashi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yasunori%2BIchihashi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ichihashi%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYasunori%2BIchihashi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1935-4397" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yasunori Ichihashi</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Lauren Headland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lauren%2BHeadland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Headland%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALauren%2BHeadland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Daniel Zamir </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Plant Sciences and Genetics in Agriculture, The Hebrew University of Jerusalem, Rehovot, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BZamir%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zamir%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BZamir%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Julin N. Maloof </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julin%2BN.%2BMaloof%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maloof%20JN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJulin%2BN.%2BMaloof%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9623-2599" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Julin N. Maloof</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nrsinha@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">nrsinha@ucdavis.edu</a>
<a href="mailto:jnmaloof@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">jnmaloof@ucdavis.edu</a></span></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Neelima R. Sinha </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Neelima%2BR.%2BSinha%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sinha%20NR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANeelima%2BR.%2BSinha%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1494-7065" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Neelima R. Sinha</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nrsinha@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">nrsinha@ucdavis.edu</a>
<a href="mailto:jnmaloof@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">jnmaloof@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs pane-panels-ajax-tab-tabs  article-page-ajax-tab bordered-tab" >
  
      
  
  <div class="pane-content">
    <div class="item-list"><ul class="tabs inline panels-ajax-tab"><li class="first"><a href="/content/6/10/3169" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_art" data-target-id="highwire_article_tabs" data-entity-context="node:460344" data-trigger="" data-url-enabled="1">Article</a><a href="/panels_ajax_tab/jnl_genetics_tab_art/node:460344/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/10/3169.figures-only" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_data" data-target-id="highwire_article_tabs" data-entity-context="node:460344" data-trigger="figures-only" data-url-enabled="1">Figures & Data</a><a href="/panels_ajax_tab/jnl_genetics_tab_data/node:460344/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/10/3169.supplemental" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_supp" data-target-id="highwire_article_tabs" data-entity-context="node:460344" data-trigger="supplemental" data-url-enabled="1">Supplemental</a><a href="/panels_ajax_tab/jnl_genetics_tab_supp/node:460344/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li><a href="/content/6/10/3169.article-info" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_info" data-target-id="highwire_article_tabs" data-entity-context="node:460344" data-trigger="article-info" data-url-enabled="1">Info & Metrics</a><a href="/panels_ajax_tab/jnl_genetics_tab_info/node:460344/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li><li class="last"><a href="/content/ggg/6/10/3169.full.pdf" class="panels-ajax-tab-tab" data-panel-name="jnl_genetics_tab_pdf" data-target-id="highwire_article_tabs" data-entity-context="node:460344" data-trigger="full.pdf+html" data-url-enabled="1"><i class="icon-file-alt"></i></a><a href="/panels_ajax_tab/jnl_genetics_tab_pdf/node:460344/1" rel="nofollow" style="display:none" class="js-crawler-link"></a></li></ul></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-panel-tabs-container article-page-ajax-tab" >
  
      
  
  <div class="pane-content">
    <div data-panels-ajax-tab-preloaded="jnl_genetics_tab_art" id="panels-ajax-tab-container-highwire_article_tabs" class="panels-ajax-tab-container"><div class="panels-ajax-tab-loading" style ="display:none"><img class="loading" src="https://www.g3journal.org/sites/all/modules/contrib/panels_ajax_tab/images/loading.gif" alt="Loading" title="Loading" /></div><div class="panels-ajax-tab-wrap-jnl_genetics_tab_art"><div class="panel-display panel-1col clearfix" >
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-markup article-page-highwire" >
  
      
  
  <div class="pane-content">
    <div class="highwire-markup"><div xmlns="http://www.w3.org/1999/xhtml" id="content-block-markup" data-highwire-cite-ref-tooltip-instance="highwire_reflinks_tooltip" xmlns:xhtml="http://www.w3.org/1999/xhtml"><div class="article fulltext-view "><span class="highwire-journal-article-marker-start"></span><div class="section abstract" id="abstract-1"><h2>Abstract</h2><p id="p-7">Quantitative Trait Loci (QTL) mapping is a powerful technique for dissecting the genetic basis of traits and species differences. Established tomato mapping populations between domesticated tomato (<em>Solanum lycopersicum</em>) and its more distant interfertile relatives typically follow a near isogenic line (NIL) design, such as the <em>S. pennellii</em> Introgression Line (IL) population, with a single wild introgression per line in an otherwise domesticated genetic background. Here, we report on a new advanced backcross QTL mapping resource for tomato, derived from a cross between the M82 tomato cultivar and <em>S. pennellii</em>. This so-called Backcrossed Inbred Line (BIL) population is comprised of a mix of <em>BC<sub>2</sub></em> and <em>BC<sub>3</sub></em> lines, with domesticated tomato as the recurrent parent. The BIL population is complementary to the existing <em>S. pennellii</em> IL population, with which it shares parents. Using the BILs, we mapped traits for leaf complexity, leaflet shape, and flowering time. We demonstrate the utility of the BILs for fine-mapping QTL, particularly QTL initially mapped in the ILs, by fine-mapping several QTL to single or few candidate genes. Moreover, we confirm the value of a backcrossed population with multiple introgressions per line, such as the BILs, for epistatic QTL mapping. Our work was further enabled by the development of our own statistical inference and visualization tools, namely a heterogeneous hidden Markov model for genotyping the lines, and by using state-of-the-art sparse regression techniques for QTL mapping.</p></div><ul class="kwd-group"><li class="kwd"><a href="/keyword/heterogeneous-hidden-markov-model" class="hw-term hw-article-keyword hw-article-keyword-heterogeneous-hidden-markov-model" rel="nofollow">heterogeneous Hidden Markov Model</a></li><li class="kwd"><a href="/keyword/regularized-regression" class="hw-term hw-article-keyword hw-article-keyword-regularized-regression" rel="nofollow">regularized regression</a></li><li class="kwd"><a href="/keyword/variable-selection" class="hw-term hw-article-keyword hw-article-keyword-variable-selection" rel="nofollow">variable selection</a></li><li class="kwd"><a href="/keyword/epistatic-qtl" class="hw-term hw-article-keyword hw-article-keyword-epistatic-qtl" rel="nofollow">epistatic QTL</a></li><li class="kwd"><a href="/keyword/fine-mapping-qtl" class="hw-term hw-article-keyword hw-article-keyword-fine-mapping-qtl" rel="nofollow">fine-mapping QTL</a></li></ul><p id="p-8">Leaves are the primary sites for capturing light energy, for gas exchange, and for synthesizing metabolic compounds through photosynthesis. Leaf morphology can significantly affect growth and overall performance of the plant through the interception of environmental signals (<a id="xref-ref-64-1" class="xref-bibr" href="#ref-64">Nicotra <em>et al.</em> 2011</a>), and is constrained by developmental, environmental, and phylogenetic contexts. Therefore, understanding the genetic and gene regulatory mechanisms underlying leaf development is of fundamental importance for increasing photosynthetic efficiency and optimizing crop performance. The basic genetic mechanisms underlying leaf development, such as leaf initiation (<a id="xref-ref-88-1" class="xref-bibr" href="#ref-88">Waites <em>et al.</em> 1998</a>; <a id="xref-ref-83-1" class="xref-bibr" href="#ref-83">Timmermans <em>et al.</em> 1999</a>; <a id="xref-ref-86-1" class="xref-bibr" href="#ref-86">Tsiantis <em>et al.</em> 1999</a>), establishment of leaf polarity (<a id="xref-ref-38-1" class="xref-bibr" href="#ref-38">Kerstetter <em>et al.</em> 2001</a>; <a id="xref-ref-56-1" class="xref-bibr" href="#ref-56">McConnell <em>et al.</em> 2001</a>), and leaf differentiation (<a id="xref-ref-62-1" class="xref-bibr" href="#ref-62">Nath <em>et al.</em> 2003</a>), have been investigated in detail in species with simple leaves (<a id="xref-ref-87-1" class="xref-bibr" href="#ref-87">Tsukaya 2013</a>). In contrast to the simple leaves of <em>Antirrhinum</em>, maize, and <em>Arabidopsis</em>, leaf architecture in many plants is complex with correspondingly protracted development and leaflet specification. Tomato is a classic example of a species with a complex or compound leaf. Tomato leaf lamina are subdivided into two or three orders of leaflets produced by repeated generation of auxin maxima (<a id="xref-ref-45-1" class="xref-bibr" href="#ref-45">Koenig <em>et al.</em> 2009</a>) that pattern leaflets along the marginal blastozone (<a id="xref-ref-40-1" class="xref-bibr" href="#ref-40">Kim <em>et al.</em> 2003a</a>,<a id="xref-ref-41-1" class="xref-bibr" href="#ref-41">b</a>), which is competent to respond to the activity of class I KNOX genes necessary for leaflet initiation (<a id="xref-ref-6-1" class="xref-bibr" href="#ref-6">Bharathan <em>et al.</em> 2002</a>; <a id="xref-ref-19-1" class="xref-bibr" href="#ref-19">Efroni <em>et al.</em> 2010</a>). Most of our current understanding of simple and compound leaf development derives from mutant analyses, expression studies, and gene expression modulation in transgenic systems (<a id="xref-ref-28-1" class="xref-bibr" href="#ref-28">Goliber <em>et al.</em> 1999</a>; <a id="xref-ref-85-1" class="xref-bibr" href="#ref-85">Tsiantis and Hay 2003</a>; <a id="xref-ref-42-1" class="xref-bibr" href="#ref-42">Kimura <em>et al.</em> 2008</a>). Although these studies have provided a basic understanding of leaf development, the genetic regulation of leaf development at the level of entire genomes and incorporating epistatic interactions among many different loci has only just begun to be deciphered.</p><p id="p-9">Quantitative genetics, through the identification of QTL (Quantitative Trait Loci), provides an effective means to identify genetic loci regulating complex traits, such as leaf morphology, by capitalizing on existing phenotypic diversity and genetic variation. Considering the relationship of leaf traits with water use efficiency and crop yield, as well as with developmental mechanisms common to all lateral organs in plants, identification of genetic loci regulating leaf traits is of critical importance. Surprisingly, only a few systematic studies have focused on investigating the quantitative basis of leaf traits to identify genetic loci underlying yield, physiological, or morphological traits (<a id="xref-ref-35-1" class="xref-bibr" href="#ref-35">Jiang <em>et al.</em> 2000</a>; <a id="xref-ref-68-1" class="xref-bibr" href="#ref-68">Pérez-Pérez <em>et al.</em> 2002</a>; <a id="xref-ref-49-1" class="xref-bibr" href="#ref-49">Langlade <em>et al.</em> 2005</a>; <a id="xref-ref-82-1" class="xref-bibr" href="#ref-82">Tian <em>et al.</em> 2011</a>; <a id="xref-ref-48-1" class="xref-bibr" href="#ref-48">Ku <em>et al.</em> 2012</a>; <a id="xref-ref-14-1" class="xref-bibr" href="#ref-14">Chitwood <em>et al.</em> 2014b</a>). Genome-wide association on a nested association mapping population identified <em>liguleless</em>, a gene regulating upright leaf angles in maize, as a factor underlying yield (<a id="xref-ref-82-2" class="xref-bibr" href="#ref-82">Tian <em>et al.</em> 2011</a>). However, this maize study and one recent study in <em>Mimulus</em> suggest that leaf shape is generally highly heritable and polygenic with many additive effects, yet without much evidence for epistasis (<a id="xref-ref-23-1" class="xref-bibr" href="#ref-23">Ferris <em>et al.</em> 2015</a>). Nonetheless, there is a need for a comprehensive examination of the role of epistasis in regulating leaf shape and complexity in other species, particularly using statistical techniques that facilitate the inference of epistatic QTL.</p><p id="p-10">Leaf traits, including leaf shape and complexity, changed repeatedly during plant evolution (<a id="xref-ref-6-2" class="xref-bibr" href="#ref-6">Bharathan <em>et al.</em> 2002</a>; <a id="xref-ref-40-2" class="xref-bibr" href="#ref-40">Kim <em>et al.</em> 2003a</a>; <a id="xref-ref-44-1" class="xref-bibr" href="#ref-44">Koenig and Sinha 2010</a>; <a id="xref-ref-84-1" class="xref-bibr" href="#ref-84">Townsley and Sinha 2012</a>). Elucidation of the genetic basis of morphological changes that occurred during the evolution and divergence of lineages has been a major challenge in biology (<a id="xref-ref-31-1" class="xref-bibr" href="#ref-31">Hoekstra and Coyne 2007</a>; <a id="xref-ref-9-1" class="xref-bibr" href="#ref-9">Carroll 2008</a>). Phenotypic differences between populations and species can be ascribed to QTL inferred from experimental crosses, and thus analysis of QTL can address significant questions about the process of evolutionary divergence. A classic example of the use of QTL in an evolutionary context is the identification of genetic loci that are crucial to the evolution of modern maize from teosinte (<a id="xref-ref-18-1" class="xref-bibr" href="#ref-18">Doebley and Wang 1997</a>; <a id="xref-ref-89-1" class="xref-bibr" href="#ref-89">Westerbergh and Doebley 2002</a>).</p><p id="p-11">The tomato clade offers an excellent system to investigate the natural variation in many plant traits, as its species have similar genetic constitution but divergent morphological and anatomical features and disparate ecologies (<a id="xref-ref-80-1" class="xref-bibr" href="#ref-80">Stevens and Rick 1986</a>; <a id="xref-ref-59-1" class="xref-bibr" href="#ref-59">Moyle 2008</a>; <a id="xref-ref-70-1" class="xref-bibr" href="#ref-70">Ranjan <em>et al.</em> 2012</a>). The evolution of many traits of modern domesticated tomato have been deciphered through QTL studies. Since tomato is one of the most extensively used vegetable crops in the world, most studies have focused on investigating variation in fruit size, shape, and sugar content (<a id="xref-ref-25-1" class="xref-bibr" href="#ref-25">Frary <em>et al.</em> 2000</a>; <a id="xref-ref-27-1" class="xref-bibr" href="#ref-27">Fridman <em>et al.</em> 2004</a>; <a id="xref-ref-90-1" class="xref-bibr" href="#ref-90">Xiao <em>et al.</em> 2008</a>). Tomato is also well-established as a model system for the study of compound leaf development in Angiosperms, and the tomato clade offers a unique opportunity for investigating natural variation in leaf complexity and other leaf traits. <em>Solanum pennellii</em>, one of the most distant interfertile relatives of domesticated tomato, <em>S. lycopersicum</em>, possesses distinct leaf size, complexity, and morphology. In addition, <em>S. pennellii</em> shows striking differences in fruit characteristics, drought resistance, disease resistance, and water use efficiency as compared to <em>S. lycopersicum</em> (<a id="xref-ref-59-2" class="xref-bibr" href="#ref-59">Moyle 2008</a>; <a id="xref-ref-12-1" class="xref-bibr" href="#ref-12">Chitwood <em>et al.</em> 2013</a>; <a id="xref-ref-46-1" class="xref-bibr" href="#ref-46">Koenig <em>et al.</em> 2013</a>). All aspects of leaf complexity are reduced in <em>S. pennellii</em> relative to domesticated tomato (<a id="xref-ref-32-1" class="xref-bibr" href="#ref-32">Holtan and Hake 2003</a>). <em>S. pennellii</em> leaves are rounder and less serrated, and leaf initiation is delayed while leaf development is accelerated relative to tomato (<a id="xref-ref-33-1" class="xref-bibr" href="#ref-33">Ichihashi <em>et al.</em> 2014</a>). In addition, the leaves of <em>S. pennellii</em> are more densely covered with epidermal hairs, and show a characteristic high accumulation of anthocyanin at the base of trichomes that gives the appearance of purple spots on these leaves. In spite of these dramatic differences in leaf form, only a few studies have been undertaken to identify genetic loci, and their network of downstream and epistatic interactions, that regulate the various leaf traits at the entire genome level.</p><p id="p-12">As a key trait affecting crop yield, flowering time has also been investigated in tomato (<a id="xref-ref-8-1" class="xref-bibr" href="#ref-8">Carmel-Goren <em>et al.</em> 2003</a>; <a id="xref-ref-51-1" class="xref-bibr" href="#ref-51">Lifschitz and Eshed 2006</a>). The genetic control of flowering time was at first primarily elucidated in <em>Arabidopsis</em>, where the signals from several pathways (<em>e.g.</em>, photoperiodic, autonomous, and vernalization, among others) are combined in a set of pathway integrator genes (<em>CO</em>, <em>FLC</em>, <em>SOC1</em>, and <em>FT</em>) that activate the floral phase transition genes <em>AP1</em> and <em>LFY</em> (<a id="xref-ref-58-1" class="xref-bibr" href="#ref-58">Mouradov <em>et al.</em> 2002</a>; <a id="xref-ref-78-1" class="xref-bibr" href="#ref-78">Simpson and Dean 2002</a>). The general manner in which these pathways are integrated to control flowering time is broadly conserved, although crop systems have been instrumental in the discovery of deviations from the <em>Arabidopsis</em> paradigm (<a id="xref-ref-7-1" class="xref-bibr" href="#ref-7">Blümel <em>et al.</em> 2015</a>). Because flowering is a major developmental phase transition, flowering time genes can strongly influence plant architecture. Tomato has played a central role in investigating how the interplay of the antagonistic roles of <em>SFT</em> and <em>SP</em> (homologs of <em>Arabidopsis</em> <em>FT</em> and <em>TFL1</em>, respectively) shape plant architecture in sympodial species (<a id="xref-ref-51-2" class="xref-bibr" href="#ref-51">Lifschitz and Eshed 2006</a>; <a id="xref-ref-20-1" class="xref-bibr" href="#ref-20">Elitzur <em>et al.</em> 2009</a>; <a id="xref-ref-52-1" class="xref-bibr" href="#ref-52">Lifschitz <em>et al.</em> 2014</a>). For instance, modulating the balance between <em>SFT</em> and <em>SP</em> significantly influences yield in tomato by controlling the number of inflorescences and their developmental timing, whereas processing tomato varieties have determinate growth as a result of being homozygous recessive sp/sp (<a id="xref-ref-69-1" class="xref-bibr" href="#ref-69">Pnueli <em>et al.</em> 1998</a>; <a id="xref-ref-52-2" class="xref-bibr" href="#ref-52">Lifschitz <em>et al.</em> 2014</a>). In contrast to its wild relatives, domesticated tomato flowers rapidly and is day-neutral or photoperiod insensitive, likely as a result of artificial selection during domestication (<a id="xref-ref-4-1" class="xref-bibr" href="#ref-4">Aung L.H. 1976</a>; <a id="xref-ref-43-1" class="xref-bibr" href="#ref-43">Kinet 1977</a>; <a id="xref-ref-3-1" class="xref-bibr" href="#ref-3">Atherton and Harris 1986</a>; <a id="xref-ref-36-1" class="xref-bibr" href="#ref-36">Jiménez-Gómez <em>et al.</em> 2007</a>; <a id="xref-ref-61-1" class="xref-bibr" href="#ref-61">Müller <em>et al.</em> 2016</a>). For example, accession LA0716 of the wild relative <em>S. pennellii</em> flowers roughly 11 d later than tomato cultivar TM2a (<a id="xref-ref-16-1" class="xref-bibr" href="#ref-16">deVicente and Tanksley 1993</a>).</p><p id="p-13"><em>S. lycopersicum</em> and <em>S. pennellii</em> are interfertile, enabling genetic dissection of trait differences such as those seen for leaf complexity. The interfertility of these two species was exploited to develop a genetic resource consisting of 76 introgressions lines (IL) (<a id="xref-ref-21-1" class="xref-bibr" href="#ref-21">Eshed and Zamir 1994</a>). Most of these introgression lines contain a single introgression from the wild species in an otherwise domesticated background. These ILs, which introgress the complete <em>S. pennellii</em> genome in the cultivated tomato (M82) background, have been extensively phenotyped for numerous traits, such as morphology, yield, fruit quality, and fruit primary and secondary metabolites for identification of QTL (<a id="xref-ref-72-1" class="xref-bibr" href="#ref-72">Rousseaux <em>et al.</em> 2005</a>; <a id="xref-ref-73-1" class="xref-bibr" href="#ref-73">Schauer <em>et al.</em> 2006</a>, <a id="xref-ref-74-1" class="xref-bibr" href="#ref-74">2008</a>; <a id="xref-ref-75-1" class="xref-bibr" href="#ref-75">Semel <em>et al.</em> 2006</a>; <a id="xref-ref-81-1" class="xref-bibr" href="#ref-81">Stevens <em>et al.</em> 2007</a>; <a id="xref-ref-79-1" class="xref-bibr" href="#ref-79">Steinhauser <em>et al.</em> 2011</a>). The striking differences in leaf morphology between <em>S. lycopersicum</em> and <em>S. pennellii</em> were capitalized on to identify QTL associated with leaf dissection (<a id="xref-ref-32-2" class="xref-bibr" href="#ref-32">Holtan and Hake 2003</a>). The IL population was also used to investigate the quantitative basis of adaptive evolution of leaf traits, indicating that leaf traits are an important component of climatic niche adaptation in wild tomatoes (<a id="xref-ref-10-1" class="xref-bibr" href="#ref-10">Chitwood <em>et al.</em> 2012a</a>; <a id="xref-ref-60-1" class="xref-bibr" href="#ref-60">Muir <em>et al.</em> 2014</a>). The diverse leaf morphologies of wild tomato species (<a id="xref-ref-11-1" class="xref-bibr" href="#ref-11">Chitwood <em>et al.</em> 2012b</a>) have been compared and contextualized within the shape range of the tomato IL lines and parents (<a id="xref-ref-13-1" class="xref-bibr" href="#ref-13">Chitwood <em>et al.</em> 2014a</a>).</p><p id="p-14">A few genes underlying QTL in the tomato IL population have been fine-mapped and cloned. High resolution mapping applied to the <em>S. pennellii</em> ILs led to the map-based cloning of the first two QTL: the fruit weight QTL, <em>fw2.2</em>, and the sugar yield QTL, <em>Brix9-2-5</em> (<a id="xref-ref-26-1" class="xref-bibr" href="#ref-26">Fridman <em>et al.</em> 2000</a>; <a id="xref-ref-25-2" class="xref-bibr" href="#ref-25">Frary <em>et al.</em> 2000</a>). Fine genetic mapping using the same introgression population identified the <em>RXopJ4</em> locus, a 190 kb segment on chromosome 6 of <em>S. pennellii</em>, which confers resistance to bacterial spot disease (<a id="xref-ref-77-1" class="xref-bibr" href="#ref-77">Sharlach <em>et al.</em> 2012</a>). Advances in next-generation sequencing and bioinformatic approaches allowed ultrahigh-density genotyping of the IL population, thus increasing the resolution of QTL mapping within this population (<a id="xref-ref-12-2" class="xref-bibr" href="#ref-12">Chitwood <em>et al.</em> 2013</a>).</p><p id="p-15">Recently, we used the IL population to identify more than a thousand QTL relating not only to leaf morphological features such as leaf shape, size, complexity, and serration, but also to cellular features, such as epidermal cell morphology and stomatal density and patterning (<a id="xref-ref-12-3" class="xref-bibr" href="#ref-12">Chitwood <em>et al.</em> 2013</a>). Though numerous QTL for various traits have been identified using the IL population, there has been limited success in fine-mapping or narrowing the QTL interval to enable the identification of strong candidate genes that could then be tested for causality, in part because of the large size of most of the introgressions. Further, since almost all of the ILs contain a single introgression on a specific chromosome, the epistatic interactions regulating a trait cannot be explored using this population. Therefore, additional introgression lines, where the large introgression regions of the ILs are subdivided into smaller regions, would facilitate the identification of strong candidate genes. Moreover, lines with multiple introgressions would greatly enable the inference of the genetic interactions regulating a trait.</p><p id="p-16">Here, we report on a new genetic mapping resource, an advanced backcross experimental population with ∼500 Backcrossed Inbred Lines (BILs). This population was generated by repeated backcrossing (two or three times) of a F1 hybrid of <em>S. lycopersicum</em> cv. M82 and <em>S. pennellii</em> and its progeny to M82, followed by selfing. For this work, we sampled progeny after four or five generations of selfing (<a id="xref-fig-1-1" class="xref-fig" href="#F1">Figure 1</a>). Ultrahigh-density genotyping using reduced representation libraries revealed that the BILs provide a much higher resolution of introgressions in terms of size compared to the ILs. In addition, most of these BILs have more than one introgression, which allows the investigation of genetic interactions underlying traits. We identified QTL regulating various leaf traits and flowering time, and also characterized potential epistatic interactions regulating these traits. Furthermore, the BIL population was also investigated for its potential to fine-map the genes likely responsible for the purple spot (Punctate) phenotype of <em>S. pennellii</em> leaves, as well as to fine-map leaf traits previously mapped in the ILs. Similarly, several recent studies have used a subset of the BILs with introgressions in a region of interest to further fine-map a QTL found in the ILs (<a id="xref-ref-65-1" class="xref-bibr" href="#ref-65">Ning <em>et al.</em> 2015</a>; <a id="xref-ref-61-2" class="xref-bibr" href="#ref-61">Müller <em>et al.</em> 2016</a>; <a id="xref-ref-22-1" class="xref-bibr" href="#ref-22">Fan <em>et al.</em> 2016</a>). The precise genotypes of these lines, their utility in fine-mapping traits, the greater phenotypic spectrum seen in the lines, and their utility for mapping single locus and epistatic QTL suggests that this population will be an extremely useful resource for basic and applied research in tomato. In this study, we also strove to overcome some of the limitations of QTL mapping, such as large QTL intervals encompassing many genes and methodological obstacles to revealing the genetic architecture of traits, by developing our own statistical inference and visualization tools for high precision genotyping, and by using state-of-the-art sparse regression techniques to reveal the genetic architecture of traits through the inference of epistatic interactions without preconditioning on main effect QTL.</p><div id="F1" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/10/3169/F1.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Crossing scheme for the S. pennellii backcrossed introgression lines. BIL, backcrossed inbred line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2118868713" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Crossing scheme for the &lt;em&gt;S. pennellii&lt;/em&gt; backcrossed introgression lines. BIL, backcrossed inbred line.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 1" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/10/3169/F1.medium.gif" width="440" height="296"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 1" src="https://www.g3journal.org/content/ggg/6/10/3169/F1.medium.gif" width="440" height="296"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/10/3169/F1.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 1" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/10/3169/F1.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469292" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption" xmlns:xhtml="http://www.w3.org/1999/xhtml"><span class="fig-label">Figure 1</span> <p id="p-17" class="first-child">Crossing scheme for the <em>S. pennellii</em> backcrossed introgression lines. BIL, backcrossed inbred line.</p><div class="sb-div caption-clear"></div></div></div><div class="section materials-methods" id="sec-1"><h2 class="">Materials and Methods</h2><div id="sec-2" class="subsection"><h3>Plant materials, growth conditions, experimental design, and phenotyping</h3><p id="p-18">BIL seeds were generously provided by Itai Ofner and Dani Zamir (Hebrew University, Rehovot, Israel). These BILs were generated by crossing the wild species <em>S. pennellii</em> LA0716 with the tomato cultivar M82 and backcrossing the resulting hybrid to M82 two to three times. Five and four rounds of consecutive selfing (respectively for BC<sub>2</sub> and BC<sub>3</sub> lines) gave rise to the BIL population assayed in this study, which consisted of approximately 545 lines.</p><p id="p-19">Seeds were washed in 50% bleach for ∼2 min, rinsed, and placed on water-soaked paper towels in Phytatrays (Sigma) to germinate. Seeds were stratified in darkness at room temperature for 3 d before moving to a 16:8 light cycle in growth chambers for 4 d. Seedlings were then transplanted into 5 × 10 subdivided trays (11’’ × 22’’ in dimension) with Sunshine Mix soil (Sun Gro) and grown in a greenhouse. Approximately 3 wk after transplantation, trays were moved to a lath house where seedlings were hardened by vigorous top watering and allowing the soil to dry between waterings.</p><p id="p-20">In early May 2011, hardened seedlings were transplanted into a field in Davis, CA. The hypocotyl of the seedlings was buried in soil. Seedlings were initially sprinkler-watered, but later transitioned to ditch irrigation. Three randomized blocks were planted adjacent to each other, each of which contained all 545 BIL genotypes once and each of the <em>S. lycopersicum</em> cv. M82 and <em>S. pennellii</em> LA0716 parents five times.</p><p id="p-21">Phenotyping for the <em>Punctate</em> locus was performed on seedlings (when the phenotype is most penetrant) at the two-to-three leaf stage when the plants were still in the greenhouse. Seedlings were scored for the punctate phenotype as a discrete, presence/absence trait. Most genotypes were scored as absent or present in all three replicates, but a few were not, and this information was noted and used during map-based cloning of the trait.</p><p id="p-22">Leaf complexity was measured in the field by pairs of researchers each counting leaflets on four leaves per plant, a pair of leaves per person. Primary, intercalary, secondary, and total leaflet counts were noted. Leaflet shape was measured as previously described (<a id="xref-ref-12-4" class="xref-bibr" href="#ref-12">Chitwood <em>et al.</em> 2013</a>). Briefly, for each plant, the terminal and two distal lateral leaflets for five leaves per plant were placed into plastic Ziploc bags and transported back to the lab. The three leaflets for each of five leaves per plant were dissected and arranged under nonreflective glass. A copy stand (Adorama, 36’’ Deluxe Copy Stand) with mounted camera (Olympus SP-500 UZ) controlled remotely using Cam2Com software (Sabsik) was used to photograph images. ImageJ (<a id="xref-ref-1-1" class="xref-bibr" href="#ref-1">Abramoff <em>et al.</em> 2004</a>) was used to isolate and threshold individual leaflets, which were saved as individual files. Leaflet outlines were then batch measured in ImageJ for area, aspect ratio, circularity, roundness, and solidity values. Leaflet outlines were processed using the software package SHAPE (<a id="xref-ref-34-1" class="xref-bibr" href="#ref-34">Iwata and Ukai 2002</a>) where outlines were converted to chain code, normalized Elliptical Fourier Descriptors (EFDs) calculated relative to the orientation of the proximal-distal axis of the leaflet, and a Principal Component Analysis (PCA) performed on coefficients from the first 20 harmonics.</p></div><div id="sec-3" class="subsection"><h3>DNA library preparation and sequencing</h3><p id="p-23">After phenotyping, developing leaves and shoot apices were collected for DNA isolation and library preparation for genotyping. DNA was isolated using the DNeasy Plant Mini Kit (QIAGEN) and reduced representation libraries were prepared using the restriction enzyme-based RESCAN method (<a id="xref-ref-57-1" class="xref-bibr" href="#ref-57">Monson-Miller <em>et al.</em> 2012</a>). RESCAN library preparation involved restriction digestion of genomic DNA with the restriction enzyme <em>Nla</em>III, followed by adapter ligation and PCR enrichment. The PCR-enriched libraries were sequenced at the Vincent J. Coates Genomics Sequencing Laboratory at UC Berkeley on the Illumina HiSeq2000 platform (Illumina, Inc., San Diego, CA) to generate 100 bp single-end reads.</p></div><div id="sec-4" class="subsection"><h3>Read processing and mapping</h3><p id="p-24">Reads were preprocessed, quality filtered, and barcode sorted using the Fastx_toolkit as described in <a id="xref-ref-12-5" class="xref-bibr" href="#ref-12">Chitwood <em>et al.</em> (2013)</a>. To remove reads originating from repeat-rich genomic regions, RESCAN sequencing reads were initially mapped to the Sol Genomic Network’s tomato repeat database using BWA (BWA parameters: -e 15 -i10 -k 1 -l 25 -n 0.05) (we created the fasta file for this from the gff3 file available at <a href="ftp://ftp.sgn.cornell.edu/genomes/Solanum_lycopersicum/annotation/ITAG2.3_release/ITAG2.3_repeats.gff3">ftp://ftp.sgn.cornell.edu/genomes/Solanum_lycopersicum/annotation/ITAG2.3_release/ITAG2.3_repeats.gff3</a>). Reads not mapped to the repeat database were extracted using the bam2fastq program (<a href="http://www.hudsonalpha.org/gsl/software/bam2fastq.php">http://www.hudsonalpha.org/gsl/software/bam2fastq.php</a>). Subsequently, these repeat-filtered reads were mapped to the Heinz reference genome using the same BWA parameters. Samtools (with the ’–bq 1’ option) was used to retain the reads that mapped uniquely to the reference genome.</p></div><div id="sec-5" class="subsection"><h3>Polymorphism identification</h3><p id="p-25">We used SNPTools (<a href="https://github.com/mfcovington/SNPtools">https://github.com/mfcovington/SNPtools</a>) v0.2.4 to identify SNPs and INDELs between <em>S. lycopersicum</em> cv. M82 and <em>S. pennellii</em>. SNPTools is a variant-detection, genotype-scoring, and visualization tool composed of Perl modules, Perl scripts, and R scripts. The use of an earlier version of SNPTools, v0.1.5, has been described elsewhere (<a id="xref-ref-17-1" class="xref-bibr" href="#ref-17">Devisetty <em>et al.</em> 2014</a>).</p><p id="p-26">In brief, the SNPTools polymorphism identification pipeline consists of the following steps. Putative Heinz:cv M82 and Heinz:<em>S. pennellii</em> polymorphisms were identified using a pileup-comparison-based approach. Positions where the M82 and <em>S. pennellii</em> alleles differ were retained and the Heinz-specific polymorphisms were discarded. SNPTools also has a polymorphism noise reduction step. The cv M82 and <em>S. pennellii</em> sequence alignments were genotyped at each putative polymorphism. For a true cv M82:<em>S. pennellii</em> polymorphism, the large majority of reads (&gt;90%) should match the appropriate allele for both alignments. Putative polymorphisms that contradict this expectation were discarded.</p><p id="p-27">We performed an additional, non-SNPTools, polymorphism-filtering step to remove outliers in regards to allele ratios and coverage within the BIL population. The sequence alignment files from all of the BILs were merged together and genotyped using the postnoise reduction polymorphism database.</p><p id="p-28">Polymorphisms were kept if the <em>S. pennellii</em> allele ratio was between the 25th and 75th percentile (relative to the current chromosome) and the nonparental allele ratio was less than 0.025. Since a small number of positions had excessive coverage relative the rest of the genome, we kept data only for polymorphisms with a maximum coverage of 2000 reads.</p><p id="p-29">Detailed descriptions of the polymorphism identification pipeline, including the code that was executed and genotype plots at different steps in the pipeline, are available at: <a href="https://github.com/mfcovington/bil-paper/blob/develop/docs/polymorphism-identification.md">https://github.com/mfcovington/bil-paper/blob/develop/docs/polymorphism-identification.md</a>.</p></div><div id="sec-6" class="subsection"><h3>Heterogeneous hidden Markov model (HMM)</h3><p id="p-30">M82, heterozygous, and <em>S. pennellii</em> genotype blocks in the BILs were inferred by means of a HMM implemented using the StochHMM C library plus our own C++ code for the heterogeneous transition matrix (<a id="xref-ref-54-1" class="xref-bibr" href="#ref-54">Lott and Korf 2014</a>). The HMM may be eventually released as standalone software. Our HMM uses a heterogeneous transition matrix in which the transition probability between states (<em>i.e.</em>, M82, heterozygous, and <em>S. pennellii</em>) depends on the genetic distance between adjacent SNPs. With the transition matrix of our HMM we, in essence, modeled recombination between pairs of SNPs. Given the small number of crosses and the short distance between SNPs, we made the simplifying assumption that recombination between any two adjacent SNPs in one line of the population occurred only once (<em>i.e.</em>, in a single generation) or not at all. We tested recombination probability equations derived from classic genetic map functions and found them to be adequate, but nevertheless unstable in a number of lines with high noise and/or low sequence coverage. Hence, we explored alternative recombination probability equations with simple parameterizations that would approximate Haldane’s and Kosambi’s equations at large genetic distances yet be more stable at short distances (<em>i.e.</em>, less prone to switch states due local noise). The cells of the transition matrix are calculated according to the following equation for the probability of changing states:<span class="disp-formula" id="disp-formula-1"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10/3169/embed/mml-math-1.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10/3169/embed/mml-math-1.gif"/></noscript></span><span class="disp-formula-label">(1)</span></span>and its reciprocal for the probability of not changing states:<span class="disp-formula" id="disp-formula-2"><span class="highwire-responsive-lazyload"><img src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" class="highwire-embed lazyload" alt="Embedded Image" data-src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10/3169/embed/mml-math-2.gif"/><noscript><img class="highwire-embed" alt="Embedded Image" src="https://www.g3journal.org/sites/default/files/highwire/ggg/6/10/3169/embed/mml-math-2.gif"/></noscript></span> <span class="disp-formula-label">(2),</span></span>where d is genetic distance in cM between a pair of adjacent SNPs and σ is a free parameter (Supplemental Material, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S73</a>). We empirically chose a value of 0.5 for σ for the present study. We plan to make the σ parameter tunable, by means of the Baum–Welch algorithm, in the training phase of the HMM in future versions of our method. Equation 1 has a low probability of transition at small genetic distances because, at such distances, it has an opposite curvature to Haldane’s and Kosambi’s recombination probability equations.</p><p id="p-31">The prior probabilities for the hidden states are 0.88: 0.10: 0.02 for M82: <em>S. pennellii</em>: heterozygous, which are roughly the expected proportion of the states in the BILs given their crossing scheme. The input SNP genotype probabilities were derived by proportionally scaling the SNP likelihoods from bcftools so that they sum to one, which implicitly assumes a uniform prior. Following previous work on genotyping HMMs (<a id="xref-ref-2-1" class="xref-bibr" href="#ref-2">Andolfatto <em>et al.</em> 2011</a>), an emission error rate of 1% was assumed, which captures several potential genotype error sources. The posterior decoding algorithm was used to infer posterior probabilities for each SNP sampled in a given BIL. Following convention, a posterior probability of 0.95 was used as the threshold for inferring a change in state (<em>i.e.</em>, genotype).</p><p id="p-32">In order to estimate the posterior probability of ancestry of each genomic region (<em>i.e.</em>, which parent it came from), our HMM used the genotype probabilities at the parentally ascertained and preselected SNPs and a heterogeneous transition model, in which the transition probability between hidden states (<em>i.e.</em>, genotypes) is a function of the genetic distance between SNPs (<a id="xref-fig-2-1" class="xref-fig" href="#F2">Figure 2</a>). We did so in order to achieve high resolution and precision of genotype boundary calls. Common work-arounds for dealing with variable distances between SNPs within a homogeneous HMM are to bin SNPs into equidistant windows (<em>e.g.</em>, 1 kb in size) that are assumed to be homogenous in genotype, or to simply ignore the differences in distance and treat SNPs as if equidistant. Both of these workarounds would have degraded the resolution and precision of the inferred genotype boundaries.</p><div id="F2" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/10/3169/F2.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Examples of hard to genotype lines. (A) High noise. (B) Low sequence coverage. (C) Heterozygous region with high noise. (D) Narrow introgression (chromosome 10). Individual dots represent SNPs and their color represents the proportion of S. pennelli vs. M82 alleles at a given SNP. BILs 015 and 110 were resequenced to obtain cleaner data. The boundaries obtained with the cleaner resequenced data did not differ from those shown above, demonstrating the performance of the HMM. Plots of resequenced BILs 015 and 110 are at github.com/mfcovington/bil-paper. BIL, backcrossed inbred line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2118868713" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Examples of hard to genotype lines. (A) High noise. (B) Low sequence coverage. (C) Heterozygous region with high noise. (D) Narrow introgression (chromosome 10). Individual dots represent SNPs and their color represents the proportion of &lt;em&gt;S. pennelli&lt;/em&gt; vs. M82 alleles at a given SNP. BILs 015 and 110 were resequenced to obtain cleaner data. The boundaries obtained with the cleaner resequenced data did not differ from those shown above, demonstrating the performance of the HMM. Plots of resequenced BILs 015 and 110 are at github.com/mfcovington/bil-paper. BIL, backcrossed inbred line.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 2" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/10/3169/F2.medium.gif" width="440" height="394"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 2" src="https://www.g3journal.org/content/ggg/6/10/3169/F2.medium.gif" width="440" height="394"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/10/3169/F2.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 2" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/10/3169/F2.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469324" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 2</span> <p id="p-33" class="first-child">Examples of hard to genotype lines. (A) High noise. (B) Low sequence coverage. (C) Heterozygous region with high noise. (D) Narrow introgression (chromosome 10). Individual dots represent SNPs and their color represents the proportion of <em>S. pennelli</em> vs. M82 alleles at a given SNP. BILs 015 and 110 were resequenced to obtain cleaner data. The boundaries obtained with the cleaner resequenced data did not differ from those shown above, demonstrating the performance of the HMM. Plots of resequenced BILs 015 and 110 are at github.com/mfcovington/bil-paper. BIL, backcrossed inbred line.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-7" class="subsection"><h3>BIL genotyping, plotting, and definition of bins</h3><p id="p-34">The BILs were genotyped using the Perl script ’extract+genotype_pileups.pl’ from SNPtools v0.2.4 (<a href="https://github.com/mfcovington/SNPtools/">https://github.com/mfcovington/SNPtools/</a>). BIL sequence alignments were interrogated at the position of each cv M82: <em>S. pennellii</em> polymorphism to create a set of genotype files per BIL. Recorded within a genotype file is the number of reads matching each allele for every polymorphism with read coverage on a chromosome.</p><p id="p-35">To detect and fine-tune genotype bins (<em>i.e.</em>, regions of a single parental genotype within a BIL) and the boundaries between them, we used a set of Perl scripts from detect-boundaries v0.6.1 (<a href="https://github.com/mfcovington/detect-boundaries/">https://github.com/mfcovington/detect-boundaries/</a>). The ’filter-snps.pl’ script identifies bins and boundaries on a per BIL scale. The ’merge-boundaries.pl’ script takes the bin and boundary data for all of the individual BILs and merges them to identify every subbin within the population and calculate related boundary and bin statistics. The ’fine-tune-boundaries.pl’ script provides an automated approach for human verification of each boundary between bins of opposite parental genotype for a BIL. This command-line tool displays color-coded genotype data together with the currently-defined bin boundaries. Using shortcut keys, the operator can quickly and easily approve or fine-tune a boundary (at which point, the next boundary is instantly displayed for approval).</p><p id="p-36">BIL genotype plots were created with the R script ’plot-genotype-and-bin.R’, which uses a BIL’s genotype and bin boundaries files.</p></div><div id="sec-8" class="subsection"><h3>Phenotypic estimates used for mapping</h3><p id="p-37">BIL phenotypic estimates for plotting and QTL mapping were derived from Gaussian linear mixed effect models fitted using the lme4 R package (<a id="xref-ref-5-1" class="xref-bibr" href="#ref-5">Bates <em>et al.</em> 2015</a>). For all traits a block random effect was used. The model used for flowering time had no additional random effects. For leaf complexity traits and asymmetric leaflet EFD principal components an additional plant level random effect was used to accommodate pseudoreplicate measurements. For shape attribute traits and symmetric leaflet EFD principal components the models included a leaflet type random effect nested within the plant level effect. For asymmetric leaflet EFD principal components, the terminal leaflet data were discarded (as these should be symmetrical or near so), and the absolute values for the left and right lateral leaflets were treated equally as they are mirror images of each other.</p><p id="p-38">Model-fitted trait mean estimates were used for BIL trait plots (<a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S4</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S5</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S6</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S7</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S8</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S9</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S10</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S11</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S12</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S13</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S14</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S15</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S16</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S17</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S18</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S19</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S20</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S21</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S22</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S23</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S24</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S25</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S26</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S27</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S28</a>, and <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S29</a>) and for marginal regression QTL mapping. For QTL mapping by SparseNet (see below) we generated predictions from the linear mixed model (LMM) fits such that pseudoreplication was eliminated (<em>i.e.</em>, generated a single value per plant), and then the pseudoreplicate averaged residuals were added to the predictions.</p></div><div id="sec-9" class="subsection"><h3>Heritability and repeatability estimates</h3><p id="p-39">Broad sense heritability and repeatability estimates for the leaf traits were inferred by frequentist and bayesian random effect regressions using the R packages lme4 and MCMCglmm (<a id="xref-ref-30-1" class="xref-bibr" href="#ref-30">Hadfield 2010</a>; <a id="xref-ref-5-2" class="xref-bibr" href="#ref-5">Bates <em>et al.</em> 2015</a>), respectively. The experimental design random effects (<em>e.g.</em>, block, plant, etc.) used in this case were the same as for the phenotypic estimates.</p></div><div id="sec-10" class="subsection"><h3>QTL mapping</h3><div id="sec-11" class="subsection"><h4>Marginal regression:</h4><p id="p-40">Each individual BIL was factored for the presence or absence of <em>S. pennellii</em> introgressed DNA at a particular bin, and this process was repeated for each bin. For marginal regression, a linear model was fitted between each trait as a function of the presence for each bin. The multiple test adjustment for the significance values for each bin with respect to a given trait was performed using the Benjamini–Hochberg method. The threshold for adjusted significance value (q-value) was set at 0.001 to detect bins significantly associated with a trait.</p></div><div id="sec-12" class="subsection"><h4>SparseNet:</h4><p id="p-41">We mapped QTL by regressing bin genotypes on phenotypic traits using the regularized regression method SparseNet (<a id="xref-ref-55-1" class="xref-bibr" href="#ref-55">Mazumder <em>et al.</em> 2011</a>), which uses the MC+ penalty family (an L0 to L1 family of penalties) to impose shrinkage and sparsity on the number and magnitude of regression coefficients. For each trait, we performed two QTL mapping analyses with SparseNet: A) one in which we only considered nonepistatic or monogenic (single bin) QTL (which we denote as the Additive Model), B) and a second analysis in which we considered single locus QTL and all potential epistatic two-locus QTL whose bin combinations were observed in the BILs (which we denote as the Epistatic Model). The Additive Model had 1049 possible effects, <em>i.e.</em>, equal to the number of BINs. The Epistatic Model had 307,340 possible effects, 1049 single loci plus the 306,291 two-locus combinations observed in the population; out of the 549,676 possible two BIN combinations among the 1049 BINs, 243,385 were not observed in the BILs.</p><p id="p-42">By using a penalized method such as SparseNet, we were able to map each trait on all bins and all bins plus all observed two bin combinations simultaneously. In order to choose the tuning parameter values for SparseNet, for each trait we averaged the α and γ values from 10 sixfold cross-validations; the averaging was done to achieve reproducible tuning of the model. Briefly, α controls the severity of the penalty, and γ controls the mixture between L0 and L1 penalties. Following cross-validation, the SparseNet model was fitted to all the data of a given trait using the chosen values for α and γ. Because methods such as LASSO and SparseNet select a single coefficient among a set of highly correlated coefficients, we took the set of BINs with a correlation of 0.90 to a SparseNet selected BIN as a sort of empirical QTL interval within which to search for candidate genes.</p></div></div><div id="sec-13" class="subsection"><h3>PROVEAN analysis</h3><p id="p-43">PROVEAN (PRotein Variation Effect ANalyzer) is a command-line tool to predict the functional impact that an amino acid substitution or indel has on a protein (<a id="xref-ref-15-1" class="xref-bibr" href="#ref-15">Choi <em>et al.</em> 2012</a>). PROVEAN-Assembly-Line v0.3.2 (<a href="https://github.com/mfcovington/PROVEAN-Assembly-Line">https://github.com/mfcovington/PROVEAN-Assembly-Line</a>) was used to run a set of Perl scripts to automate high-throughput bulk PROVEAN analysis.</p><p id="p-44">The ’find-aa-substitutions.pl’ script was used to generate the amino acid substitution files required to run PROVEAN. The input files for this script are SNP files, a genomic FASTA reference file (S_lycopersicum_chromosomes.2.40.fa), and the corresponding GFF3 annotation file (ITAG2.3_gene_models.gff3).</p><p id="p-45">The ’run-provean.pl’ script was used to run the PROVEAN tool in parallel for every missense mutation, every nonsense mutation, and every mutation that caused the loss of a STOP codon. The input files for this script are the amino acid substitution files generated in the previous step and a CDS FASTA reference file (ITAG2.3_cds_SHORTNAMES.fasta).</p><p id="p-46">The ’filter-provean-results.pl’ was used to filter PROVEAN results for a single gene, a file containing a list of genes, or all genes within a genomic range.</p></div><div id="sec-14" class="subsection"><h3>Candidate gene identification</h3><p id="p-47">A set of literature-curated leaf developmental genes was used as a reference for the finding of candidate genes within QTL bins for different leaf traits (<a id="xref-ref-33-2" class="xref-bibr" href="#ref-33">Ichihashi <em>et al.</em> 2014</a>). We selected only those genes from the set that were either differentially expressed between the two parents based on expression-QTL analysis of the IL population or that showed amino acid changes between the two parents that were predicted to be functionally significant by PROVEAN. The QTL bins were then scanned for the presence of these genes. Following this, we performed permutation analyses in the R statistical environment to check whether bins associated with leaf traits were enriched for the selected literature-curated leaf developmental genes. The data were permuted 10,000 times, and the results were considered significant if &lt; 5% of the permuted datasets equaled or exceeded the real data set.</p></div><div id="sec-15" class="subsection"><h3>IL eQTL enrichment among BIL epistatic QTL</h3><p id="p-48">We assessed whether IL <em>trans</em>-eQTL were enriched among the set of paired regions that make up the BIL epistatic QTL using two eQTL clusters. One cluster was enriched for leaf development GO (gene ontology) terms and the other for photosynthesis GO terms. Together, these two clusters contained the majority of the literature-curated leaf development genes used for the candidate gene search (<a id="xref-ref-33-3" class="xref-bibr" href="#ref-33">Ichihashi <em>et al.</em> 2014</a>; <a id="xref-ref-71-1" class="xref-bibr" href="#ref-71">Ranjan <em>et al.</em> 2016</a>). The IRanges R/Bioconductor package (<a id="xref-ref-50-1" class="xref-bibr" href="#ref-50">Lawrence <em>et al.</em> 2013</a>) was used to find the overlaps between the IL <em>trans</em>-eQTL and the BIL epistatic QTL. We tested for IL <em>trans</em>-eQTL enrichment among BIL epistatic QTL by permutation using the leaf development cluster (set 1), as well as this cluster plus the photosynthesis cluster (set 2). The data were permuted 10,000 times, and the results were considered significant if &lt; 5% of the permuted datasets equaled or exceeded the real data set.</p></div><div id="sec-16" class="subsection"><h3>QTL confirmation and fine-mapping</h3><p id="p-49">The IRanges R/Bioconductor package (<a id="xref-ref-50-2" class="xref-bibr" href="#ref-50">Lawrence <em>et al.</em> 2013</a>) was used to find overlaps between the IL phenotypic QTL and the BIL single locus QTL, and the results were tabulated and plotted.</p></div><div id="sec-17" class="subsection"><h3>Data availability</h3><p id="p-50">The quality filtered, barcode-sorted, and trimmed short read dataset was deposited in the NCBI Short Read Archive under accessions SRR3188298–SRR3188738 (Bioproject accession SRP070833). </p></div></div><div class="section results-discussion" id="sec-18"><h2 class="">Results and Discussion</h2><div id="sec-19" class="subsection"><h3>High precision genotyping by means of a novel HMM</h3><p id="p-51">The BILs were genotyped at high resolution using Restriction Enzyme Sequence Comparative ANalysis (RESCAN), a reduced representation genomic sequencing method (<a id="xref-ref-57-2" class="xref-bibr" href="#ref-57">Monson-Miller <em>et al.</em> 2012</a>; <a id="xref-ref-76-1" class="xref-bibr" href="#ref-76">Seymour <em>et al.</em> 2012</a>). A polymorphism database was created for sequence variants that differentiate the two parents, the domesticated tomato variety <em>S. lycopersicum</em> cv. M82 and its wild relative <em>S. pennellii</em> LA0716. The database consisted of 346,645 polymorphisms between M82 and <em>S. pennellii</em> (295,571 SNPs and 51,074 inserted or deleted nucleotides), which was used to identify parental SNPs and INDELs for each BIL across the entire genome.</p><p id="p-52">Because of the generally low and sparse sequence coverage of RESCAN, we used a novel heterogeneous HMM to define the introgressed regions for each BIL. The HMM proved indispensable for accurately genotyping about 15% of the lines, which were problematic due to a variety of issues (<a id="xref-fig-2-2" class="xref-fig" href="#F2">Figure 2</a>). Moreover, the HMM also enabled us to infer sharper recombination boundaries (github.com/mfcovington/bil-paper). The HMM inferred narrow transition boundaries in spite of noise and/or low coverage in some of the lines (<a id="xref-fig-2-3" class="xref-fig" href="#F2">Figure 2</a>).</p><p id="p-53">We were able to establish the genotype of 493 out of a total 545 BILs with confidence. However, 52 lines were found to be genotypically identical to the M82 parent, and two out of the remaining 441 were identical to two other lines. This could be due to seed or pollen contamination, and/or random effects during backcrossing and selfing. We were not able to establish the genotype of 52 other lines, either due to lack of germination or excessive noise in the raw genotype data. The 439 uniquely genotyped BILs covered the entire tomato genome except for a small gap of ∼400 kb on chromosome 1. We detected heterozygous introgressions on almost all chromosomes (<a id="xref-fig-2-4" class="xref-fig" href="#F2">Figure 2C</a>, <a id="xref-fig-3-1" class="xref-fig" href="#F3">Figure 3A</a>, <a id="xref-fig-4-1" class="xref-fig" href="#F4">Figure 4</a>, and <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S1</a>). However, most of these heterozygous introgressions were covered by another homozygous introgression in the same region in a different BIL. Nevertheless, although the BIL lines were not fully inbred after four or five generations of selfing, overall we found little residual heterozygosity (<a id="xref-fig-3-2" class="xref-fig" href="#F3">Figure 3A</a> and <a id="xref-fig-4-2" class="xref-fig" href="#F4">Figure 4</a>). The comprehensive genotype of the 439 BILs along with the chromosome-wise distribution of introgressions for the population is presented in <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/FileS1.xlsx">File S1</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/FileS2.xlsx">File S2</a>. In addition, the plots of the genotyped BILs are presented in an online resource for this paper (github.com/mfcovington/bil-paper).</p><div id="F3" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/10/3169/F3.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Introgression and bin statistics. (A) Distribution of introgressions across bins; green = homozygous S. pennellii, black = heterozygous. (B) Introgression frequency per BIL. (C) Introgression percentage per BIL. (D) Number of bins per chromosome. Panel (A) is plotted in genetic distance as Figure S1. BIL, backcrossed inbred line." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2118868713" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Introgression and bin statistics. (A) Distribution of introgressions across bins; green = homozygous &lt;em&gt;S. pennellii&lt;/em&gt;, black = heterozygous. (B) Introgression frequency per BIL. (C) Introgression percentage per BIL. (D) Number of bins per chromosome. Panel (A) is plotted in genetic distance as Figure S1. BIL, backcrossed inbred line.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 3" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/10/3169/F3.medium.gif" width="330" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 3" src="https://www.g3journal.org/content/ggg/6/10/3169/F3.medium.gif" width="330" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/10/3169/F3.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 3" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/10/3169/F3.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469367" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 3</span> <p id="p-54" class="first-child">Introgression and bin statistics. (A) Distribution of introgressions across bins; green = homozygous <em>S. pennellii</em>, black = heterozygous. (B) Introgression frequency per BIL. (C) Introgression percentage per BIL. (D) Number of bins per chromosome. Panel (A) is plotted in genetic distance as <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S1</a>. BIL, backcrossed inbred line.</p><div class="sb-div caption-clear"></div></div></div><div id="F4" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/10/3169/F4.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Composite genotype map of the entire BIL population. See Figure S2 for a version of this figure plotted by physical distance, and Figure S3 for a version plotted with BILs clustered by each chromosome after removing samples without an introgression. BIL, backcrossed inbred line; ch, chromosome." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2118868713" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;Composite genotype map of the entire BIL population. See Figure S2 for a version of this figure plotted by physical distance, and Figure S3 for a version plotted with BILs clustered by each chromosome after removing samples without an introgression. BIL, backcrossed inbred line; ch, chromosome.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 4" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/10/3169/F4.medium.gif" width="440" height="330"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 4" src="https://www.g3journal.org/content/ggg/6/10/3169/F4.medium.gif" width="440" height="330"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/10/3169/F4.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 4" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/10/3169/F4.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469340" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 4</span> <p id="p-55" class="first-child">Composite genotype map of the entire BIL population. See <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S2</a> for a version of this figure plotted by physical distance, and <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S3</a> for a version plotted with BILs clustered by each chromosome after removing samples without an introgression. BIL, backcrossed inbred line; ch, chromosome.</p><div class="sb-div caption-clear"></div></div></div><p id="p-56">The number of introgressions for each BIL varied from 1 to 12 with a mean of 3.125 introgressions per BIL (<a id="xref-fig-3-3" class="xref-fig" href="#F3">Figure 3B</a>). Approximately 80% of the lines had from one to four introgressions (<a id="xref-fig-3-4" class="xref-fig" href="#F3">Figure 3B</a>). As opposed to the single introgressions per line of the IL population, the presence of multiple introgressions in a single BIL enables the study of the genetic interactions regulating complex traits. The BIL population provides a synergistic complement to the IL population (<a id="xref-ref-21-2" class="xref-bibr" href="#ref-21">Eshed and Zamir 1994</a>), an additional resource for linking QTL identified in the ILs to genes as multiple BILs possess smaller overlapping introgressions encompassed within the larger introgression regions of the IL population. The mean and median numbers of genes per introgression are approximately 677 and 485 for the ILs, and 479 and 293 for the BILs. For example, the introgression in IL 4–3 has been identified as having the largest contribution to tomato leaf traits in terms of the overall number of QTL mapped to this region (<a id="xref-ref-32-3" class="xref-bibr" href="#ref-32">Holtan and Hake 2003</a>; <a id="xref-ref-12-6" class="xref-bibr" href="#ref-12">Chitwood <em>et al.</em> 2013</a>). We identified at least 50 BILs with breakpoints overlapping the introgressed region of IL 4–3, allowing fine-scale dissection of QTL in this region. We also explored the BIL population for the number of introgressions on specific chromosomes (<a id="xref-table-wrap-1-1" class="xref-table" href="#T1">Table 1</a>). A relatively small number of BILs (54) have an introgression on chromosome 1, the largest chromosome by physical distance, whereas a surprisingly high number of BILs (236) have introgressions on chromosome 11 (<a id="xref-fig-3-5" class="xref-fig" href="#F3">Figure 3</a>, <a id="xref-fig-4-3" class="xref-fig" href="#F4">Figure 4</a>, and <a id="xref-table-wrap-1-2" class="xref-table" href="#T1">Table 1</a>). Chromosome 5 also harbors a large number of introgressions (171).</p><div id="T1" class="table pos-float"><div class="table-inline table-callout-links"><div class="callout"><span>View this table:</span><ul class="callout-links"><li class="view-inline first"><a href="##" class="table-expand-inline" data-table-url="/highwire/markup/469262/expansion?postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed&amp;table-expand-inline=1" data-icon-position="" data-hide-link-title="0">View inline</a></li><li class="view-popup last"><a href="/highwire/markup/469262/expansion?width=1000&amp;height=500&amp;iframe=true&amp;postprocessors=highwire_tables%2Chighwire_reclass%2Chighwire_figures%2Chighwire_math%2Chighwire_inline_linked_media%2Chighwire_embed" class="colorbox colorbox-load table-expand-popup" rel="gallery-fragment-tables" data-icon-position="" data-hide-link-title="0">View popup</a></li></ul></div></div><div class="table-caption"><span class="table-label">Table 1</span> <span class="caption-title">Per chromosome BIL introgression and bin counts</span><div class="sb-div caption-clear"></div></div></div><p id="p-58">The BILs showed remarkable variation in terms of the percentage of <em>S. pennellii</em> genome in each line (<a id="xref-fig-3-6" class="xref-fig" href="#F3">Figure 3C</a>), ranging from &lt; 1% to &gt; 10%. The introgression size ranged from 16.7 kb on chromosome 10 in BIL-025, containing only four genes, to the largest introgressions on chromosome 12 in BIL-083 and BIL-444, which span almost the entire chromosome and contain more than 2400 genes. Taken together, the remarkable variation in the number of introgressions, size of introgressions, and the level of genome contribution makes the BILs a highly useful resource for fine-mapping of complex traits and identification of the underlying genes.</p><p id="p-59">The precise boundary information for the genotyped BILs was used to define noncontiguous “bins:” regions defined by the pattern of overlap among introgressions in the population and within which no line changes genotype (Liu and Zamir 1999; <a id="xref-ref-12-7" class="xref-bibr" href="#ref-12">Chitwood <em>et al.</em> 2013</a>). A total of 1049 unique bins (87 per chromosome on average) were detected across the entire genome with the maximum number of bins on chromosomes 5 and 11, with 156 and 128 bins, respectively (<a id="xref-fig-3-7" class="xref-fig" href="#F3">Figure 3D</a>, <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/FileS3.zip">File S3</a>, and <a id="xref-table-wrap-1-3" class="xref-table" href="#T1">Table 1</a>). Thus, the BILs provide an increase of nearly an order of magnitude in the number of bins as compared to the 112 bins defined in the IL population (<a id="xref-ref-12-8" class="xref-bibr" href="#ref-12">Chitwood <em>et al.</em> 2013</a>). We used our precision genotyping and introgression boundary calls to define the gene content of bins with near certainty. A list of annotated genes within the BIL bins is provided as <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/FileS4.xlsx">File S4</a>. Interestingly, almost half of the bins defined for BILs (455 out of a total 1049 bins) had 10 or fewer genes, which will greatly facilitate the identification of candidate genes in future studies. Conversely, only 9 out of 126 IL bins (at the subbin level) have 10 or fewer genes. For the ILs, the summary statistics for the number of genes per bin (at the subbin level) are: range = 1–1890, mean = 262.06, and median = 150. In contrast, for the BILs these values are: range = 1–984, mean = 23.15, and median = 7.</p><p id="p-60">Although <a id="xref-ref-66-1" class="xref-bibr" href="#ref-66">Ofner <em>et al.</em> (2016</a>) used the same tomato BIL population, we achieved a finer resolution of recombination breakpoints and defined 66% more bins thanks to higher density genotyping (roughly an 80 times greater number of SNPs). While merging the BIL introgression and bin maps presented in these two publications would be worthwhile, it is hampered by the fact that the population was genotyped at different stages in the two studies and that different versions of the tomato reference genome were used.</p></div><div id="sec-20" class="subsection"><h3>Analysis of leaf trait variation in the population</h3><p id="p-61">As described above, the parents of this population show striking differences in leaf morphology and complexity. Not surprisingly, there is substantial variation for these same traits segregating in the BILs. In order to account for replication and pseudoreplication of trait measurements, LMM were used to estimate model-fitted BIL trait means for QTL mapping. As expected due to the backcrossed population design, the distribution of many of the traits is roughly centered around the recurrent parent, <em>S. lycopersicum</em> cv. M82, yet often present a skew toward the wild parent, <em>S. pennellii</em> (<a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S4</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S5</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S6</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S7</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S8</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S9</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S10</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S11</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S12</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S13</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S14</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S15</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S16</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S17</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S18</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S19</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S20</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S21</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S22</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S23</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S24</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S25</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S26</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S27</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S28</a>, and <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S29</a>). This is most evident in traits such as flowering time and roundness, for which the only BILs significantly different from M82 are in the <em>S. pennellii</em> direction (<a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S13</a> and <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S11</a>).</p><p id="p-62">Overall, we found positive correlation among leaf complexity traits (<em>i.e.</em>, the number of different types of leaflets and the total number of leaflets). There is stronger correlation between intercalary and secondary leaflet counts, than between either of these and primary complexity. Likewise, intercalary and secondary leaflets are also more strongly correlated than primary leaflets to the total number of leaflets (<a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S30</a>). We found no correlation between leaf complexity and leaflet shape traits. Aspect ratio and roundness are very strongly negatively correlated, which is to be expected. Circularity and solidity are strongly correlated. In the context of tomato leaflet shape, dentation and lobing of the leaflet margin similarly affects circularity and solidity. As in our previous study of tomato leaflet shape in the ILs (<a id="xref-ref-12-9" class="xref-bibr" href="#ref-12">Chitwood <em>et al.</em> 2013</a>), we used an outline morphometrics technique, elliptical fourier descriptors decomposed into their principal components (EFD-PC), to examine leaflet shape in the BILs (<a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S14</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S15</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S16</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S17</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S18</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S19</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S20</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S21</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S22</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S23</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S24</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S25</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S26</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S27</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S28</a>, and <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S29</a>). However, the EFD-PC traits showed low heritability and repeatability in the BILs, perhaps owing to the lower replication in this dataset (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/TableS1.pdf">Table S1</a>), so we excluded these traits from further discussion.</p></div><div id="sec-21" class="subsection"><h3>QTL regulating leaf traits</h3><p id="p-63">We used the precisely defined BIL bins to identify QTL regulating various leaf developmental traits, such as complexity, aspect ratio, circularity, and solidity. Two methods, marginal regression (<a id="xref-ref-12-10" class="xref-bibr" href="#ref-12">Chitwood <em>et al.</em> 2013</a>) and SparseNet (<a id="xref-ref-55-2" class="xref-bibr" href="#ref-55">Mazumder <em>et al.</em> 2011</a>), were used to identify BINs associated with different traits. Hundreds of bins were significantly associated with complexity and leaf shape traits by marginal regression at a q-value of &lt; 0.01 (<a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S31</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S32</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S33</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S34</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S35</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S36</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S37</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S38</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S39</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S40</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S41</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S42</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S43</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S44</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S45</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/FileS5.zip">File S5</a>). Corroborating what we found in the ILs, where ILs 4–3 and 5–4 harbored the largest number of bins associated with leaf traits (<a id="xref-ref-12-11" class="xref-bibr" href="#ref-12">Chitwood <em>et al.</em> 2013</a>), numerous BIL bins in chromosomes 4 and 5 were associated with leaf complexity and leaflet shape traits.</p><p id="p-64">To increase mapping resolution beyond what is possible with marginal regression we used the regularized regression method SparseNet, which performs variable selection and shrinkage of the regression coefficients (<em>i.e.</em>, inferred association between bins and traits). SparseNet reduced the number of trait associated bins by around an order of magnitude (<a id="xref-fig-5-1" class="xref-fig" href="#F5">Figure 5</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S46</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S47</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S48</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S49</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S50</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S51</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S52</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S53</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S54</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S55</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S56</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S57</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S58</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S59</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S60</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S61</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S62</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S63</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S64</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S65</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S66</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/FileS6.zip">File S6</a>). Only 28, 36, 26, and 7 bins were found to be associated with primary, intercalary, secondary, and total complexity, respectively. This method also identified a remarkably reduced number of bins and sharp QTL peaks for leaf shape traits. The bins associated with various leaf traits were distributed on multiple chromosomes across the tomato genome, emphasizing the polygenic regulation of leaf developmental traits.</p><div id="F5" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/10/3169/F5.large.jpg?width=800&amp;height=600&amp;carousel=1" title="QTL mapping results for additive and epistatic models. (A) Primary complexity. (B) Total complexity. (C) Aspect ratio. (D) Circularity. The bin selected by SparseNet as a QTL is colored opaquely, and the nearby bins correlated to it at or greater than 0.9 are taken as an approximate QTL interval and colored translucently. QTL, quantitative trait loci." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2118868713" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;QTL mapping results for additive and epistatic models. (A) Primary complexity. (B) Total complexity. (C) Aspect ratio. (D) Circularity. The bin selected by SparseNet as a QTL is colored opaquely, and the nearby bins correlated to it at or greater than 0.9 are taken as an approximate QTL interval and colored translucently. QTL, quantitative trait loci.&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 5" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/10/3169/F5.medium.gif" width="440" height="424"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 5" src="https://www.g3journal.org/content/ggg/6/10/3169/F5.medium.gif" width="440" height="424"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/10/3169/F5.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 5" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/10/3169/F5.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469430" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 5</span> <p id="p-65" class="first-child">QTL mapping results for additive and epistatic models. (A) Primary complexity. (B) Total complexity. (C) Aspect ratio. (D) Circularity. The bin selected by SparseNet as a QTL is colored opaquely, and the nearby bins correlated to it at or greater than 0.9 are taken as an approximate QTL interval and colored translucently. QTL, quantitative trait loci.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-22" class="subsection"><h3>Epistatic interactions for the leaf traits</h3><p id="p-66">Our previous work showed that tomato leaf complexity and leaflet shape is polygenic (<a id="xref-ref-12-12" class="xref-bibr" href="#ref-12">Chitwood <em>et al.</em> 2013</a>). This is also evident in the current study, where we detected numerous loci that contribute to leaf traits distributed over the entire tomato genome. Although functional genetic studies have reported evidence for epistasis in tomato leaf development (<a id="xref-ref-47-1" class="xref-bibr" href="#ref-47">Koltai and Bird 2000</a>; <a id="xref-ref-67-1" class="xref-bibr" href="#ref-67">Ori <em>et al.</em> 2007</a>; <a id="xref-ref-63-1" class="xref-bibr" href="#ref-63">Naz <em>et al.</em> 2013</a>), the genome-wide pattern of epistasis regulating tomato leaf traits has yet to be investigated.</p><p id="p-67">The variable selection capability of SparseNet enabled us to search the full epistatic QTL model space without preconditioning on QTL with a main (<em>i.e.</em>, single locus) effect. The 1049 BIL bins have 549,676 possible two-locus combinations, which define the set of potential epistatic QTL. We observed 306,291 (or roughly 56%) of said two-locus combinations in the BIL lines. Thus, the search space for the epistatic QTL model comprised 307,340 variables, 1049 single locus plus 306,291 two-locus variables. We inferred 25, 3, 33, and 23 epistatic interactions regulating primary, secondary, intercalary, and total leaf complexity (<a id="xref-fig-5-2" class="xref-fig" href="#F5">Figure 5, A and B</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S46</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S47</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/FileS6.zip">File S6</a>). We further uncovered 42, 45, 53, and 61 possible epistatic interactions regulating aspect ratio, circularity, roundness, and solidity of leaflets (<a id="xref-fig-5-3" class="xref-fig" href="#F5">Figure 5, C and D</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S48</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S49</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/FileS6.zip">File S6</a>). Although we did not do so, inclusion of a per-line effect in the regression model would help guard against false positive epistatic QTL in a backcrossed population, such as the BILs with uneven number of lines covering different genomic regions. Similarly, higher replication per line would also lower the chance for false positives.</p></div><div id="sec-23" class="subsection"><h3>Candidate genes underlying the detected QTL</h3><p id="p-68">In order to define candidate genes underlying the identified single locus and epistatic QTL, we used a previously established set of literature-curated leaf developmental genes (<a id="xref-ref-33-4" class="xref-bibr" href="#ref-33">Ichihashi <em>et al.</em> 2014</a>). We reasoned that any causal gene would have amino acid and/or <em>cis</em>-driven mRNA abundance differences segregating in the population. To identify genes with <em>cis</em>-driven mRNA abundance differences, we filtered the literature-curated leaf developmental genes to retain those exhibiting <em>cis</em>-eQTL using transcriptome profiling and expression-QTL analysis of the IL population (<a id="xref-ref-12-13" class="xref-bibr" href="#ref-12">Chitwood <em>et al.</em> 2013</a>; <a id="xref-ref-71-2" class="xref-bibr" href="#ref-71">Ranjan <em>et al.</em> 2016</a>). Out of 384 leaf developmental genes, 62 were detected with an underlying eQTL. Using these 62 genes, we identified roughly 10 candidate genes for each of the leaf complexity traits, and 10–20 genes for each leaf shape phenotype. <em>GROWTH-REGULATING FACTOR</em>, <em>ASYMMETRIC LEAVES 2</em>, <em>AINTEGUMENTA</em>, <em>ARGONAUTE</em>, <em>LEUNIG</em>, <em>LONELY GUY</em>, <em>STM</em>, <em>BEL1</em>, <em>TCP</em>, <em>YUCCA</em>, and <em>IAA</em> were some of the salient candidate genes for both leaf complexity and shape traits (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/FileS7.zip">File S7</a>). We used permutation analyses to determine the significance of enrichment of leaf developmental genes among the QTL identified. Significant enrichment of the literature-curated leaf developmental gene set was observed for leaf shape QTL, such as aspect ratio, circularity, solidity, and roundness, but the enrichment was not significant for leaf complexity traits (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/TableS2.pdf">Table S2</a>). One possible explanation for this is that, since the literature-curated leaf gene reference list was primarily based on <em>Arabidopsis</em> (simple) leaf development, it is likely biased for leaf shape and not representative of leaf complexity in tomato.</p><p id="p-69">We also detected candidate genes underlying the epistatic QTL. One possible interacting pair of genes underlying an epistatic QTL for primary complexity is <em>GRF2</em> on chromosome 2 and <em>ANT</em> on chromosome 4 (<a id="xref-fig-5-4" class="xref-fig" href="#F5">Figure 5A</a> and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/FileS8.zip">File S8</a>). Surprisingly, our transcriptomic study of tomato, <em>S. pennellii</em>, and <em>S. habrochaites</em> compound leaf development found <em>GRF2</em> and <em>ANT</em> to be highly-connected hub genes at the core of the gene coexpression network (<a id="xref-ref-33-5" class="xref-bibr" href="#ref-33">Ichihashi <em>et al.</em> 2014</a>). That these independent analyses using different methodologies identified the same genes supports their importance for compound leaf development. This finding also underscores the importance of the regulation of cell proliferation for compound leaf development.</p><p id="p-70">Another epistatic QTL for primary complexity with candidate genes for both loci was that between chromosomes 7 and 12, which contains <em>LSH10</em> and <em>TCP4</em> in one locus and <em>IAA CARBOXYMETHYL TRANSFERASE 1</em> in the other. <em>BEL1</em>, <em>ARGONAUTE 5</em>, and <em>GRF4</em> were other strong candidate genes for epistatic QTL, but without obvious interacting candidates from the reference gene set on their paired interval. It should be noted that some of the epistatic QTL may in fact be broad-peaked single locus QTL, as the inferred pairs of interacting bins are on the same chromosome and possibly too close to have been separated by recombination in a few generations of crossing. Therefore, for the purpose of candidate gene identification for epistatic QTL, we removed QTL whose bins are less than 20 cM apart. Following this filtering, we focused on the top 10 epistatic QTL by effect magnitude (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/FileS8.zip">File S8</a>).</p><p id="p-71">The above analysis only considered the candidate genes showing gene expression differences between the two parents and does not account for amino acid changes that could underlie the leaf shape and complexity differences between the two parents. Therefore, we used PROVEAN (<a id="xref-ref-15-2" class="xref-bibr" href="#ref-15">Choi <em>et al.</em> 2012</a>) to predict amino acid changes of functional significance between the two parents for the curated leaf developmental genes. This added 42 genes to the candidate genes reference list. Redoing the candidate gene search with an updated gene list, including both eQTL and PROVEAN data, increased the number of candidate genes for each trait but at the cost of significance in the permutation analysis (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/TableS2.pdf">Table S2</a>). Noteworthy PROVEAN candidate gene additions were <em>PIN-FORMED 1</em>, <em>GIBBERELLIN OXIDASE</em>s, <em>CLAVATA 1-like</em>, <em>IAA4</em>, <em>YUUCA8</em>, and <em>GRF5</em>.</p><p id="p-72">Lastly, in order to explore if gene regulatory interactions underlie some of the inferred epistatic QTL, we assessed whether IL <em>trans</em>-eQTL were enriched among the set of paired regions that make up the BIL epistatic QTL. We did this with two sets of <em>trans</em>-eQTL derived from eQTL clustering (<a id="xref-ref-71-3" class="xref-bibr" href="#ref-71">Ranjan <em>et al.</em> 2016</a>). The above set of literature-curated leaf development genes were highly represented in two eQTL clusters, one enriched for leaf development GO terms and another enriched for photosynthesis GO terms. Therefore, we tested for IL <em>trans</em>-eQTL enrichment among BIL epistatic QTL by permutation using the leaf development cluster (set 1), as well as this cluster plus the photosynthesis cluster (set 2). Both sets yielded highly significant results with p-values of 0.0028 and &lt; 1.0E-04, respectively for set 1 and 2 (<a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/TableS3.pdf">Table S3</a>).</p></div><div id="sec-24" class="subsection"><h3>Punctate: a fine-mapping test case</h3><p id="p-73">To experimentally test the usefulness of the BILs in resolving genetic loci associated with a trait, we conducted a case study for the <em>Punctate</em> phenotype, which enhances the accumulation of anthocyanins in trichome bases creating a “purple spot” phenotype. Among ILs, IL 10–3 shows the <em>Punctate</em> phenotype in leaves, a phenotype observed in <em>S. pennellii</em> but not M82. Notably, the Tomato Genetics Resource Center lists the <em>Punctate</em> locus as residing on chromosome 8 (see accessions LA0812, LA0998, and LA3089; tgrc.ucdavis.edu), suggesting a previously unknown locus on chromosome 10 is responsible for the trait variation segregating in the ILs and BILs. The <em>S. pennellii</em> introgression in IL 10–3 spans 2.65 Mbp and 385 genes (<a id="xref-ref-12-14" class="xref-bibr" href="#ref-12">Chitwood <em>et al.</em> 2013</a>), making candidate gene identification difficult. To determine the extent to which the BILs could improve the mapping resolution, we screened the BILs and identified 54 lines with the phenotype. Of the 54 BILs exhibiting the “purple spot” phenotype, 46 possessed an introgression on chromosome 10 that overlapped the IL 10–3 introgression. The remaining 8 BILs did not show the <em>Punctate</em> phenotype consistently. Among BILs with a chromosome 10 introgression and the “purple spot” phenotype, BIL-228 had the smallest introgression, covering 310 kbp and containing only 51 genes, reducing the number of candidate genes from 385. By examining BILs with chromosome 10 introgressions yet without the <em>Punctate</em> phenotype, we further narrowed the associated genomic region to a 114 kbp segment with only 18 genes (<a id="xref-fig-6-1" class="xref-fig" href="#F6">Figure 6</a>). Thus, the BILs enabled us to dramatically reduce the region and the number of candidate genes for the “purple spot” phenotype by 96% and 95%, respectively.</p><div id="F6" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/10/3169/F6.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Punctate fine-mapping. (A) Adaxial leaf images showing presence and absence of the &#x201C;purple spot&#x201D; phenotype. (B) Comparison of the chromosome 10 introgressions of IL 10&#x2013;3 and BIL-228. (C) Fine-mapping the Punctate locus using BILs; purple-labeled BILs possess the &#x201C;purple spot&#x201D; phenotype, whereas black-labeled BILs do not. (D) Gene list of the fine-mapped region with candidate MYB tandem expansion highlighted in purple. BIL, backcrossed inbred line; IL, introgression line; QTL, quantitative trait loci." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2118868713" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&lt;em&gt;Punctate&lt;/em&gt; fine-mapping. (A) Adaxial leaf images showing presence and absence of the &#x201C;purple spot&#x201D; phenotype. (B) Comparison of the chromosome 10 introgressions of IL 10&#x2013;3 and BIL-228. (C) Fine-mapping the &lt;em&gt;Punctate&lt;/em&gt; locus using BILs; purple-labeled BILs possess the &#x201C;purple spot&#x201D; phenotype, whereas black-labeled BILs do not. (D) Gene list of the fine-mapped region with candidate MYB tandem expansion highlighted in purple. BIL, backcrossed inbred line; IL, introgression line; QTL, quantitative trait loci.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 6" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/10/3169/F6.medium.gif" width="299" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 6" src="https://www.g3journal.org/content/ggg/6/10/3169/F6.medium.gif" width="299" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/10/3169/F6.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 6" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/10/3169/F6.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469299" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 6</span> <p id="p-74" class="first-child"><em>Punctate</em> fine-mapping. (A) Adaxial leaf images showing presence and absence of the “purple spot” phenotype. (B) Comparison of the chromosome 10 introgressions of IL 10–3 and BIL-228. (C) Fine-mapping the <em>Punctate</em> locus using BILs; purple-labeled BILs possess the “purple spot” phenotype, whereas black-labeled BILs do not. (D) Gene list of the fine-mapped region with candidate MYB tandem expansion highlighted in purple. BIL, backcrossed inbred line; IL, introgression line; QTL, quantitative trait loci.</p><div class="sb-div caption-clear"></div></div></div><p id="p-75">Among the 18 genes in the <em>Punctate</em> fine-mapped region, there were four tandemly-duplicated genes encoding MYB transcription factors. Overexpression of an <em>Arabidopsis</em> homolog of these MYB genes (MYB114, AT1G66380), part of an independently-derived tandem duplication of MYB genes that regulates anthocyanin biosynthesis, results in a <em>Punctate</em>-like phenotype with accumulation of anthocyanins in trichome bases (<a id="xref-ref-29-1" class="xref-bibr" href="#ref-29">Gonzalez <em>et al.</em> 2008</a>). Remarkably, a similar tandem duplication of four MYB homologs in grape (<em>Vitis vinifera</em>) (<em>VvMybA1</em>, <em>VvMybA2</em>, and <em>VvMybA3</em>) explains 62% of the variation of anthocyanin accumulation in berries in a Syrah × Grenache F1 QTL study (<a id="xref-ref-24-1" class="xref-bibr" href="#ref-24">Fournier-Level <em>et al.</em> 2009</a>). This reinforces the involvement of MYB transcription factors in the regulation of the <em>Punctate</em> phenotype in <em>S. pennellii</em> leaves, and the importance of tandemly duplicated homologs found in lineages as diverse as <em>Arabidopsis</em> and <em>V. vinifera</em> in regulating anthocyanin accumulation in a tissue-specific manner. Similarly, BILs could facilitate the identification and cloning of genes regulating various other developmental and metabolic traits.</p></div><div id="sec-25" class="subsection"><h3>Flowering time QTL and candidate genes</h3><p id="p-76">We also used the BIL population to infer QTL for flowering time and identify their candidate genes. Given the well-characterized pathways and genes for flowering time, this provides a good opportunity to further test the utility of the BIL population for candidate identification. Marginal regression identified 69 significant bins associated with flowering time in tomato. Interestingly, 49 of those were contiguous bins on chromosome 5 (<a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S39</a>). The remaining bins were localized on chromosomes 8, 9, and 12. Three narrow QTL, one each on chromosomes 5, 8, and 12, were found to be associated with flowering time using the more stringent SparseNet method (<a id="xref-fig-6-2" class="xref-fig" href="#F6">Figure 6B</a>, ring A). <em>SELF-PRUNING 5G (SP5G)</em>, a <em>FLOWERING LOCUS T</em> homolog, is present in the QTL on chromosome 5 and is the strongest candidate gene, by virtue of its function, that we identified for flowering time in the BILs (<a id="xref-ref-8-2" class="xref-bibr" href="#ref-8">Carmel-Goren <em>et al.</em> 2003</a>). The QTL on chromosome 12 also has some important developmental genes within it, such as <em>AP2</em>, <em>ULT</em>, and <em>YABBY</em>, that are plausible candidates for the regulation of flowering time in tomato.</p><p id="p-77">The QTL on chromosome 5 is the major flowering time QTL in the BILs by effect size and is coincident with a large effect QTL in the ILs (<a id="xref-fig-7-1" class="xref-fig" href="#F7">Figure 7A</a>, ring C). The overlap between the BIL and IL QTL on chromosome 5 defines a 297 kbp region containing 37 genes, among which <em>SP5G</em> is the only flowering time gene (<a id="xref-fig-6-3" class="xref-fig" href="#F6">Figure 6B</a>, inset E and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/TableS4.pdf">Table S4</a>; <a id="xref-ref-37-1" class="xref-bibr" href="#ref-37">Jones <em>et al.</em> 2007</a>). Thus, the <em>SP5G</em> example not only highlights the utility of the BILs for fine-mapping, but also the advantage of using the BILs in conjunction with the ILs for QTL confirmation and fine-mapping.</p><div id="F7" class="fig pos-float type-figure  odd"><div class="highwire-figure"><div class="fig-inline-img-wrapper"><div class="fig-inline-img"><a href="https://www.g3journal.org/content/ggg/6/10/3169/F7.large.jpg?width=800&amp;height=600&amp;carousel=1" title="Fine-mapping and confirmation of IL QTL using BILs. (A) Flowering time, showing the candidate gene SP5G in inset (E). (B) Total complexity. (C) Circularity. The BIL bin selected by SparseNet as a QTL is colored opaquely, and the nearby bins correlated to it at or greater than 0.9 are taken as an approximate QTL interval and colored translucently. BIL, backcrossed inbred line; IL, introgression line; QTL, quantitative trait loci." class="highwire-fragment fragment-images colorbox-load" rel="gallery-fragment-images-2118868713" data-figure-caption="&lt;div class=&quot;highwire-markup&quot;&gt;&lt;div xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;Fine-mapping and confirmation of IL QTL using BILs. (A) Flowering time, showing the candidate gene &lt;em&gt;SP5G&lt;/em&gt; in inset (E). (B) Total complexity. (C) Circularity. The BIL bin selected by SparseNet as a QTL is colored opaquely, and the nearby bins correlated to it at or greater than 0.9 are taken as an approximate QTL interval and colored translucently. BIL, backcrossed inbred line; IL, introgression line; QTL, quantitative trait loci.&lt;/div&gt;&lt;/div&gt;" data-icon-position="" data-hide-link-title="0"><span class="hw-responsive-img"><img class="highwire-fragment fragment-image lazyload" alt="Figure 7" src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7" data-src="https://www.g3journal.org/content/ggg/6/10/3169/F7.medium.gif" width="157" height="440"/><noscript><img class="highwire-fragment fragment-image" alt="Figure 7" src="https://www.g3journal.org/content/ggg/6/10/3169/F7.medium.gif" width="157" height="440"/></noscript></span></a></div></div><ul class="highwire-figure-links inline"><li class="download-fig first"><a href="https://www.g3journal.org/content/ggg/6/10/3169/F7.large.jpg?download=true" class="highwire-figure-link highwire-figure-link-download" title="Download Figure 7" data-icon-position="" data-hide-link-title="0">Download figure</a></li><li class="new-tab"><a href="https://www.g3journal.org/content/ggg/6/10/3169/F7.large.jpg" class="highwire-figure-link highwire-figure-link-newtab" target="_blank" data-icon-position="" data-hide-link-title="0">Open in new tab</a></li><li class="download-ppt last"><a href="/highwire/powerpoint/469473" class="highwire-figure-link highwire-figure-link-ppt" data-icon-position="" data-hide-link-title="0">Download powerpoint</a></li></ul></div><div class="fig-caption"><span class="fig-label">Figure 7</span> <p id="p-78" class="first-child">Fine-mapping and confirmation of IL QTL using BILs. (A) Flowering time, showing the candidate gene <em>SP5G</em> in inset (E). (B) Total complexity. (C) Circularity. The BIL bin selected by SparseNet as a QTL is colored opaquely, and the nearby bins correlated to it at or greater than 0.9 are taken as an approximate QTL interval and colored translucently. BIL, backcrossed inbred line; IL, introgression line; QTL, quantitative trait loci.</p><div class="sb-div caption-clear"></div></div></div></div><div id="sec-26" class="subsection"><h3>QTL confirmation and fine-mapping</h3><p id="p-79">Beyond the above two test cases, we explored the benefit of combining results from the IL and BIL populations for the purposes of QTL confirmation and fine-mapping since we measured the same traits in both populations. As mentioned above, the IL and BIL populations share the same parents, but have different crossing schemes and broadly disparate numbers of lines. The ILs have a near isogenic line (NIL) design, whereas the BILs have a mixed <em>BC<sub>2</sub></em> / <em>BC<sub>3</sub></em> advanced backcross design.</p><p id="p-80">One advantage of utilizing both of these populations is that QTL can be validated in an independent experiment using a different set of lines. In our case, we found broad congruence between the IL and BIL QTL mapping results (<a id="xref-fig-7-2" class="xref-fig" href="#F7">Figure 7</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S67</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S68</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S69</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S70</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S71</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S72</a>, and <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/FileS9.zip">File S9</a>). Nevertheless, there are discrepancies in the QTL inferred from these two populations, which could be ascribed to their different crossing schemes and patterns of wild introgression, statistical power differences due to disparate sample sizes, and false positives. Apart from the <em>Punctate</em> example, several IL QTL were significantly narrowed in the BILs, such as QTL for total leaf complexity on chromosomes 8 and 9 (<a id="xref-fig-7-3" class="xref-fig" href="#F7">Figure 7B</a>), circularity QTL on chromosomes 8 and 11 (<a id="xref-fig-7-4" class="xref-fig" href="#F7">Figure 7C</a>), and several QTL for primary, secondary, and intercalary complexity (<a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S67</a>, <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S68</a>, and <a href="http://www.g3journal.org/SupplementalFigures_/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1/SupplementalFigures.pdf">Figure S69</a>). Likewise, several recent studies have used a subset of the BILs with introgressions in a region of interest to further fine-map a QTL found in the ILs (<a id="xref-ref-65-2" class="xref-bibr" href="#ref-65">Ning <em>et al.</em> 2015</a>; <a id="xref-ref-61-3" class="xref-bibr" href="#ref-61">Müller <em>et al.</em> 2016</a>; <a id="xref-ref-22-2" class="xref-bibr" href="#ref-22">Fan <em>et al.</em> 2016</a>).</p><p id="p-81">Furthermore, the use of these two populations is quite complementary. The ILs allow for higher replication of trait measurements due to the lower number of lines. NIL populations, such as the ILs, have also been shown to enable the detection of smaller effect QTL (<a id="xref-ref-39-1" class="xref-bibr" href="#ref-39">Keurentjes <em>et al.</em> 2007</a>). In contrast, the BILs can greatly facilitate higher resolution mapping (<em>i.e.</em>, narrower QTL intervals) due to their generally narrower bins. Moreover, the BILs enable the mapping of epistatic QTL, which is not possible with the ILs. Our ability to infer epistatic QTL was further empowered by our high precision genotyping HMM and use of the regularized regression method SparseNet.</p></div><div id="sec-27" class="subsection"><h3>Conclusions</h3><p id="p-82">The BILs provide a valuable new resource for the tomato research community, one that is complementary to the long-established IL population yet also extends the research toolkit considerably. We demonstrated the utility of this new population for corroborating and fine-mapping IL QTL. Furthermore, we established the capacity of the BIL population for epistatic QTL mapping and for revealing the genetic architectures of tomato traits. The present study also highlights the dramatic gains that can be reaped from the use of advanced statistical techniques, such as our novel heterogeneous genotyping HMM and the application of regularized regression for QTL mapping.</p></div></div><div class="section" id="sec-28"></div><div class="section ack" id="ack-1"><h2 class="">Acknowledgments</h2><p id="p-83">We thank Yaniv Brandvain and Paul Lott for discussions and assistance with early versions of the HMM. We thank the Vincent J. Coates Genomics Sequencing Laboratory at University of California Berkeley (supported by National Institutes of Health S10 Instrumentation grants S10RR029668 and S10RR027303), and computational resources/cyber infrastructure provided by the iPlant Collaborative (<a href="http://www.iplantcollaborative.org">www.iplantcollaborative.org</a>), funded by the National Science Foundation (grant DBI-0735191). This work was supported through a National Science Foundation grant (IOS-0820854) awarded to N.R.S. and J.N.M. D.H.C. was a fellow of the Life Sciences Research Foundation funded through the Gordon and Betty Moore Foundation.</p><p id="p-84">Author contributions: D.H.C., J.N.M., and N.R.S. conceived and designed the experiments. D.F., A.R., D.H.C., D.W., Y.S., and L.H. performed the experiments. D.F., A.R., M.F.C., and D.H.C. analyzed the data. D.F. developed new statistical inference tools. D.F., A.R., I.F., M.F.C., D.H.C., D.Z., and J.N.M. contributed reagents, materials, and analysis tools. D.F., A.R., M.F.C., D.H.C., J.N.M., and N.R.S. wrote the paper. All authors read and approved the final manuscript.</p></div><div class="section fn-group" id="fn-group-1"><h2>Footnotes</h2><ul><li class="fn" id="fn-7"><p id="p-85">Supplemental material is available online at <a href="http://www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1">www.g3journal.org/lookup/suppl/doi:10.1534/g3.116.030536/-/DC1</a></p></li><li class="fn" id="fn-8"><p id="p-86"><em>Communicating editor: J. Wendel</em></p></li></ul></div><ul class="history-list"><li xmlns:hwp="http://schema.highwire.org/Journal" class="received" hwp:start="2016-04-27"><span class="received-label">Received </span>April 27, 2016.</li><li xmlns:hwp="http://schema.highwire.org/Journal" class="accepted" hwp:start="2016-08-01"><span class="accepted-label">Accepted </span>August 1, 2016.</li></ul><ul class="copyright-statement"><li class="fn" id="copyright-statement-1">Copyright © 2016 Fulop <em>et al.</em></li></ul><div class="license" id="license-1"><p id="p-6">This is an open-access article distributed under the terms of the Creative Commons Attribution 4.0 International License (<a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><div class="section ref-list" id="ref-list-1"><h2 class="">Literature Cited</h2><ol class="cit-list ref-use-labels"><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-1-1" title="View reference  in text" id="ref-1">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Abramoff</span>  <span class="cit-name-given-names">M. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Magalhães</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ram</span>  <span class="cit-name-given-names">S. J.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Image processing with ImageJ.</span> <abbr class="cit-jnl-abbrev">Biophotonics international</abbr> <span class="cit-vol">11</span>: <span class="cit-fpage">36</span>–<span class="cit-lpage">42</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBiophotonics%2Binternational%26rft.volume%253D11%26rft.spage%253D36%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-2-1" title="View reference  in text" id="ref-2">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.2" data-doi="10.1101/gr.115402.110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Andolfatto</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Davison</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Erezyilmaz</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hu</span>  <span class="cit-name-given-names">T. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mast</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Multiplexed shotgun genotyping for rapid and efficient genetic mapping.</span> <abbr class="cit-jnl-abbrev">Genome Res.</abbr> <span class="cit-vol">21</span>: <span class="cit-fpage">610</span>–<span class="cit-lpage">617</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BRes.%26rft_id%253Dinfo%253Adoi%252F10.1101%252Fgr.115402.110%26rft_id%253Dinfo%253Apmid%252F21233398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiZ2Vub21lIjtzOjU6InJlc2lkIjtzOjg6IjIxLzQvNjEwIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-3-1" title="View reference  in text" id="ref-3">↵</a><div class="cit ref-cit ref-book" id="cit-6.10.3169.3"><div class="cit-metadata unstructured">Atherton, J. G., and G. P. Harris, 1986 Flowering, pp. 167–200 in <em>The Tomato Crop</em>, edited by J. G. Atherton and J. Rudich. Springer, Dordrecht.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-4-1" title="View reference  in text" id="ref-4">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.4"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Aung</span>  <span class="cit-name-given-names">L. H.</span></span></li></ol><cite>, <span class="cit-pub-date">1976</span> <span class="cit-article-title">Effects of photoperiod and temperature on vegetative and reproductive responses of Lycopersicon esculentum Mill.</span> <abbr class="cit-jnl-abbrev">J. Am. Soc. Hortic. Sci.</abbr> <span class="cit-vol">101</span>: <span class="cit-fpage">358</span>–<span class="cit-lpage">360</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BSoc.%2BHortic.%2BSci.%26rft.volume%253D101%26rft.spage%253D358%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-5-1" title="View reference  in text" id="ref-5">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.5"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bates</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Mächler</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolker</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Walker</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Fitting Linear Mixed-Effects Models Using lme4.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">67</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">48</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BStat.%2BSoftw.%26rft.volume%253D67%26rft.spage%253D1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-6-1" title="View reference  in text" id="ref-6">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.6"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Bharathan</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Goliber</span>  <span class="cit-name-given-names">T. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moore</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kessler</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pham</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Homologies in Leaf Form Inferred from KNOXI Gene Expression During</span>. <abbr class="cit-jnl-abbrev">Dev. Sci.</abbr> <span class="cit-vol">296</span>: <span class="cit-fpage">1858</span>–<span class="cit-lpage">1860</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDev.%2BSci.%26rft.volume%253D296%26rft.spage%253D1858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-7-1" title="View reference  in text" id="ref-7">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.7" data-doi="10.1016/j.copbio.2014.11.023"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Blümel</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dally</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jung</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Flowering time regulation in crops — what did we learn from Arabidopsis?</span> <abbr class="cit-jnl-abbrev">Curr. Opin. Biotechnol.</abbr> <span class="cit-vol">32</span>: <span class="cit-fpage">121</span>–<span class="cit-lpage">129</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurr.%2BOpin.%2BBiotechnol.%26rft.volume%253D32%26rft.spage%253D121%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.copbio.2014.11.023%26rft_id%253Dinfo%253Apmid%252F25553537%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.copbio.2014.11.023&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25553537&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-8-1" title="View reference  in text" id="ref-8">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.8" data-doi="10.1023/B:PLAN.0000004333.96451.11"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carmel-Goren</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lifschitz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zamir</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">The SELF-PRUNING gene family in tomato.</span> <abbr class="cit-jnl-abbrev">Plant Mol. Biol.</abbr> <span class="cit-vol">52</span>: <span class="cit-fpage">1215</span>–<span class="cit-lpage">1222</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BMol%2BBiol%26rft.aulast%253DCarmel-Goren%26rft.auinit1%253DL.%26rft.volume%253D52%26rft.issue%253D6%26rft.spage%253D1215%26rft.epage%253D1222%26rft.atitle%253DThe%2BSELF-PRUNING%2Bgene%2Bfamily%2Bin%2Btomato.%26rft_id%253Dinfo%253Adoi%252F10.1023%252FB%253APLAN.0000004333.96451.11%26rft_id%253Dinfo%253Apmid%252F14682620%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1023/B:PLAN.0000004333.96451.11&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=14682620&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000186646100009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-9-1" title="View reference  in text" id="ref-9">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.9" data-doi="10.1016/j.cell.2008.06.030"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Carroll</span>  <span class="cit-name-given-names">S. B.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Evo-Devo and an Expanding Evolutionary Synthesis: A Genetic Theory of Morphological Evolution.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">25</span>–<span class="cit-lpage">36</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DCarroll%26rft.auinit1%253DS.%2BB.%26rft.volume%253D134%26rft.issue%253D1%26rft.spage%253D25%26rft.epage%253D36%26rft.atitle%253DEvo-devo%2Band%2Ban%2Bexpanding%2Bevolutionary%2Bsynthesis%253A%2Ba%2Bgenetic%2Btheory%2Bof%2Bmorphological%2Bevolution.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cell.2008.06.030%26rft_id%253Dinfo%253Apmid%252F18614008%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cell.2008.06.030&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18614008&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000257513100011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-10-1" title="View reference  in text" id="ref-10">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.10" data-doi="10.1371/journal.pone.0029570"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chitwood</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Headland</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Filiault</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiménez-Gómez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">a</span> <span class="cit-article-title">Native Environment Modulates Leaf Size and Response to Simulated Foliar Shade across Wild Tomato Species.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e29570</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DChitwood%26rft.auinit1%253DD.%2BH.%26rft.volume%253D7%26rft.issue%253D1%26rft.spage%253De29570%26rft.epage%253De29570%26rft.atitle%253DNative%2Benvironment%2Bmodulates%2Bleaf%2Bsize%2Band%2Bresponse%2Bto%2Bsimulated%2Bfoliar%2Bshade%2Bacross%2Bwild%2Btomato%2Bspecies.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0029570%26rft_id%253Dinfo%253Apmid%252F22253737%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0029570&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22253737&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-11-1" title="View reference  in text" id="ref-11">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.11" data-doi="10.1104/pp.111.192518"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chitwood</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Headland</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Peng</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maloof</span>  <span class="cit-name-given-names">J. N.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span><span class="cit-comment">b</span> <span class="cit-article-title">The Developmental Trajectory of Leaflet Morphology in Wild Tomato Species.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">158</span>: <span class="cit-fpage">1230</span>–<span class="cit-lpage">1240</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.111.192518%26rft_id%253Dinfo%253Apmid%252F22247269%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTU4LzMvMTIzMCI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTAvMzE2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-12-1" title="View reference  in text" id="ref-12">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.12" data-doi="10.1105/tpc.113.112391"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chitwood</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Headland</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ranjan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Covington</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">A Quantitative Genetic Basis for Leaf Morphology in a Set of Precisely Defined Tomato Introgression Lines.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">25</span>: <span class="cit-fpage">2465</span>–<span class="cit-lpage">2481</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.113.112391%26rft_id%253Dinfo%253Apmid%252F23872539%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI1LzcvMjQ2NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTAvMzE2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-13-1" title="View reference  in text" id="ref-13">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.13" data-doi="10.1105/tpc.114.130112"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chitwood</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ranjan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ichihashi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zumstein</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">a</span> <span class="cit-article-title">Resolving Distinct Genetic Regulators of Tomato Leaf Shape within a Heteroblastic and Ontogenetic Context.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">26</span>: <span class="cit-fpage">3616</span>–<span class="cit-lpage">3629</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.114.130112%26rft_id%253Dinfo%253Apmid%252F25271240%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjI2LzkvMzYxNiI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTAvMzE2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-14-1" title="View reference  in text" id="ref-14">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.14" data-doi="10.1104/pp.113.229708"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Chitwood</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ranjan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinez</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Headland</span>  <span class="cit-name-given-names">L. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Thiem</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span><span class="cit-comment">b</span> <span class="cit-article-title">A Modern Ampelography: A Genetic Basis for Leaf Shape and Venation Patterning in Grape.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">164</span>: <span class="cit-fpage">259</span>–<span class="cit-lpage">272</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.113.229708%26rft_id%253Dinfo%253Apmid%252F24285849%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNjQvMS8yNTkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMxNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-15-1" title="View reference  in text" id="ref-15">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.15" data-doi="10.1371/journal.pone.0046688"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Choi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sims</span>  <span class="cit-name-given-names">G. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Murphy</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chan</span>  <span class="cit-name-given-names">A. P.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Predicting the Functional Effect of Amino Acid Substitutions and Indels.</span> <abbr class="cit-jnl-abbrev">PLoS One</abbr> <span class="cit-vol">7</span>: <span class="cit-fpage">e46688</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DPLoS%2BONE%26rft.aulast%253DChoi%26rft.auinit1%253DY.%26rft.volume%253D7%26rft.issue%253D10%26rft.spage%253De46688%26rft.epage%253De46688%26rft.atitle%253DPredicting%2Bthe%2Bfunctional%2Beffect%2Bof%2Bamino%2BAcid%2Bsubstitutions%2Band%2Bindels.%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pone.0046688%26rft_id%253Dinfo%253Apmid%252F23056405%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pone.0046688&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23056405&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-16-1" title="View reference  in text" id="ref-16">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.16"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">deVicente</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tanksley</span>  <span class="cit-name-given-names">S. D.</span></span></li></ol><cite>, <span class="cit-pub-date">1993</span> <span class="cit-article-title">QTL analysis of transgressive segregation in an interspecific tomato cross.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">134</span>: <span class="cit-fpage">585</span>–<span class="cit-lpage">596</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DdeVicente%26rft.auinit1%253DM.%2BC.%26rft.volume%253D134%26rft.issue%253D2%26rft.spage%253D585%26rft.epage%253D596%26rft.atitle%253DQTL%2Banalysis%2Bof%2Btransgressive%2Bsegregation%2Bin%2Ban%2Binterspecific%2Btomato%2Bcross.%26rft_id%253Dinfo%253Apmid%252F8100788%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTM0LzIvNTg1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-17-1" title="View reference  in text" id="ref-17">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.17"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Devisetty</span>  <span class="cit-name-given-names">U. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Covington</span>  <span class="cit-name-given-names">M. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tat</span>  <span class="cit-name-given-names">A. V.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lekkala</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Maloof</span>  <span class="cit-name-given-names">J. N.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Polymorphism Identification and Improved Genome Annotation of Brassica rapa Through Deep RNA Sequencing.</span> <abbr class="cit-jnl-abbrev">G3 (Bethesda)</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">2065</span>–<span class="cit-lpage">2078</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DG3%2B%2528Bethesda%2529%26rft.volume%253D4%26rft.spage%253D2065%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-18-1" title="View reference  in text" id="ref-18">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.18" data-doi="10.1101/SQB.1997.062.01.043"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Doebley</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wang</span>  <span class="cit-name-given-names">R.-L.</span></span></li></ol><cite>, <span class="cit-pub-date">1997</span> <span class="cit-article-title">Genetics and the Evolution of Plant Form: An Example from Maize.</span> <abbr class="cit-jnl-abbrev">Cold Spring Harb. Symp. Quant. Biol.</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">361</span>–<span class="cit-lpage">367</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCold%2BSpring%2BHarbor%2BSymposia%2Bon%2BQuantitative%2BBiology%26rft.stitle%253DCold%2BSpring%2BHarb%2BSymp%2BQuant%2BBiol%26rft.aulast%253DDoebley%26rft.auinit1%253DJ.%26rft.volume%253D62%26rft.issue%253D0%26rft.spage%253D361%26rft.epage%253D367%26rft.atitle%253DGenetics%2Band%2Bthe%2BEvolution%2Bof%2BPlant%2BForm%253A%2BAn%2BExample%2Bfrom%2BMaize%26rft_id%253Dinfo%253Adoi%252F10.1101%252FSQB.1997.062.01.043%26rft_id%253Dinfo%253Apmid%252F9598370%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic3FiIjtzOjU6InJlc2lkIjtzOjg6IjYyLzAvMzYxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-19-1" title="View reference  in text" id="ref-19">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.19" data-doi="10.1105/tpc.109.073601"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Efroni</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eshed</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lifschitz</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">Morphogenesis of Simple and Compound Leaves: A Critical Review.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">1019</span>–<span class="cit-lpage">1032</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.109.073601%26rft_id%253Dinfo%253Apmid%252F20435903%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjk6IjIyLzQvMTAxOSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTAvMzE2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-20-1" title="View reference  in text" id="ref-20">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.20" data-doi="10.1093/jxb/ern334"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Elitzur</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nahum</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borovsky</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pekker</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eshed</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Co-ordinated regulation of flowering time, plant architecture and growth by FASCICULATE: the pepper orthologue of SELF PRUNING.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">60</span>: <span class="cit-fpage">869</span>–<span class="cit-lpage">880</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Fern334%26rft_id%253Dinfo%253Apmid%252F19174461%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4Ym90IjtzOjU6InJlc2lkIjtzOjg6IjYwLzMvODY5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-21-1" title="View reference  in text" id="ref-21">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.21" data-doi="10.1007/BF00022516"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Eshed</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zamir</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">1994</span> <span class="cit-article-title">A genomic library of Lycopersicon pennellii in L. esculentum: A tool for fine mapping of genes.</span> <abbr class="cit-jnl-abbrev">Euphytica</abbr> <span class="cit-vol">79</span>: <span class="cit-fpage">175</span>–<span class="cit-lpage">179</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEuphytica%26rft.volume%253D79%26rft.spage%253D175%26rft_id%253Dinfo%253Adoi%252F10.1007%252FBF00022516%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/BF00022516&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=A1994QC56400004&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-22-1" title="View reference  in text" id="ref-22">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.22" data-doi="10.1073/pnas.1517930113"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fan</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Miller</span>  <span class="cit-name-given-names">A. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schilmiller</span>  <span class="cit-name-given-names">A. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ofner</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">In vitro reconstruction and analysis of evolutionary variation of the tomato acylsucrose metabolic network.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">113</span>: <span class="cit-fpage">E239</span>–<span class="cit-lpage">E248</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1517930113%26rft_id%253Dinfo%253Apmid%252F26715757%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTEzLzIvRTIzOSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTAvMzE2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-23-1" title="View reference  in text" id="ref-23">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.23" data-doi="10.1093/aob/mcv080"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ferris</span>  <span class="cit-name-given-names">K. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rushton</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Greenlee</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Toll</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Blackman</span>  <span class="cit-name-given-names">B. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">Leaf shape evolution has a similar genetic architecture in three edaphic specialists within the Mimulus guttatus species complex.</span> <abbr class="cit-jnl-abbrev">Ann. Bot. (Lond.)</abbr> <span class="cit-vol">116</span>: <span class="cit-fpage">213</span>–<span class="cit-lpage">223</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnn.%2BBot.%2B%2528Lond.%2529%26rft_id%253Dinfo%253Adoi%252F10.1093%252Faob%252Fmcv080%26rft_id%253Dinfo%253Apmid%252F26070644%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW5uYm90IjtzOjU6InJlc2lkIjtzOjk6IjExNi8yLzIxMyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTAvMzE2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-24-1" title="View reference  in text" id="ref-24">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.24" data-doi="10.1534/genetics.109.103929"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fournier-Level</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cunff</span>  <span class="cit-name-given-names">L. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gomez</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doligez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ageorges</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Quantitative Genetic Bases of Anthocyanin Variation in Grape (Vitis vinifera L. ssp. sativa) Berry: A Quantitative Trait Locus to Quantitative Trait Nucleotide Integrated Study.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">183</span>: <span class="cit-fpage">1127</span>–<span class="cit-lpage">1139</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.109.103929%26rft_id%253Dinfo%253Apmid%252F19720862%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE4My8zLzExMjciO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMxNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-25-1" title="View reference  in text" id="ref-25">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.25" data-doi="10.1126/science.289.5476.85"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Frary</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nesbitt</span>  <span class="cit-name-given-names">T. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Frary</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Grandillo</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Knaap</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">fw2.2: A Quantitative Trait Locus Key to the Evolution of Tomato Fruit Size.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">289</span>: <span class="cit-fpage">85</span>–<span class="cit-lpage">88</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DFrary%26rft.auinit1%253DA.%26rft.volume%253D289%26rft.issue%253D5476%26rft.spage%253D85%26rft.epage%253D88%26rft.atitle%253Dfw2.2%253A%2BA%2BQuantitative%2BTrait%2BLocus%2BKey%2Bto%2Bthe%2BEvolution%2Bof%2BTomato%2BFruit%2BSize%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.289.5476.85%26rft_id%253Dinfo%253Apmid%252F10884229%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjExOiIyODkvNTQ3Ni84NSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTAvMzE2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-26-1" title="View reference  in text" id="ref-26">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.26" data-doi="10.1073/pnas.97.9.4718"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fridman</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pleban</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zamir</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">A recombination hotspot delimits a wild-species quantitative trait locus for tomato sugar content to 484 bp within an invertase gene.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">97</span>: <span class="cit-fpage">4718</span>–<span class="cit-lpage">4723</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPNAS%26rft.stitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft.aulast%253DFridman%26rft.auinit1%253DE.%26rft.volume%253D97%26rft.issue%253D9%26rft.spage%253D4718%26rft.epage%253D4723%26rft.atitle%253DA%2Brecombination%2Bhotspot%2Bdelimits%2Ba%2Bwild-species%2Bquantitative%2Btrait%2Blocus%2Bfor%2Btomato%2Bsugar%2Bcontent%2Bto%2B484%2Bbp%2Bwithin%2Ban%2Binvertase%2Bgene%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.97.9.4718%26rft_id%253Dinfo%253Apmid%252F10781077%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czo5OiI5Ny85LzQ3MTgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMxNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-27-1" title="View reference  in text" id="ref-27">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.27" data-doi="10.1126/science.1101666"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Fridman</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carrari</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y.-S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fernie</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zamir</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2004</span> <span class="cit-article-title">Zooming In on a Quantitative Trait for Tomato Yield Using Interspecific Introgressions.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">305</span>: <span class="cit-fpage">1786</span>–<span class="cit-lpage">1789</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DFridman%26rft.auinit1%253DE.%26rft.volume%253D305%26rft.issue%253D5691%26rft.spage%253D1786%26rft.epage%253D1789%26rft.atitle%253DZooming%2BIn%2Bon%2Ba%2BQuantitative%2BTrait%2Bfor%2BTomato%2BYield%2BUsing%2BInterspecific%2BIntrogressions%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1101666%26rft_id%253Dinfo%253Apmid%252F15375271%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMDUvNTY5MS8xNzg2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-28-1" title="View reference  in text" id="ref-28">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.28"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Goliber</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kessler</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chen</span>  <span class="cit-name-given-names">J. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bharathan</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinha</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">Genetic, molecular, and morphological analysis of compound leaf development.</span> <abbr class="cit-jnl-abbrev">Curr. Top. Dev. Biol.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">259</span>–<span class="cit-lpage">290</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Btopics%2Bin%2Bdevelopmental%2Bbiology%26rft.stitle%253DCurr%2BTop%2BDev%2BBiol%26rft.aulast%253DGoliber%26rft.auinit1%253DT.%26rft.volume%253D43%26rft.spage%253D259%26rft.epage%253D290%26rft.atitle%253DGenetic%252C%2Bmolecular%252C%2Band%2Bmorphological%2Banalysis%2Bof%2Bcompound%2Bleaf%2Bdevelopment.%26rft_id%253Dinfo%253Apmid%252F9891889%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=9891889&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087256300008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-29-1" title="View reference  in text" id="ref-29">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.29" data-doi="10.1111/j.1365-313X.2007.03373.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Gonzalez</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leavitt</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lloyd</span>  <span class="cit-name-given-names">A. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Regulation of the anthocyanin biosynthetic pathway by the TTG1/bHLH/Myb transcriptional complex in Arabidopsis seedlings.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">53</span>: <span class="cit-fpage">814</span>–<span class="cit-lpage">827</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DGonzalez%26rft.auinit1%253DA.%26rft.volume%253D53%26rft.issue%253D5%26rft.spage%253D814%26rft.epage%253D827%26rft.atitle%253DRegulation%2Bof%2Bthe%2Banthocyanin%2Bbiosynthetic%2Bpathway%2Bby%2Bthe%2BTTG1%252FbHLH%252FMyb%2Btranscriptional%2Bcomplex%2Bin%2BArabidopsis%2Bseedlings.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1365-313X.2007.03373.x%26rft_id%253Dinfo%253Apmid%252F18036197%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1365-313X.2007.03373.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18036197&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000253484600010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-30-1" title="View reference  in text" id="ref-30">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.30" data-doi="10.1111/j.1467-9868.2005.00503.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hadfield</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2010</span> <span class="cit-article-title">MCMC Methods for Multi-Response Generalized Linear Mixed Models: The MCMCglmm R Package.</span> <abbr class="cit-jnl-abbrev">J. Stat. Softw.</abbr> <span class="cit-vol">33</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">22</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.stitle%253DJ%2BStat%2BSoftw%26rft.aulast%253DFriedman%26rft.auinit1%253DJ.%26rft.volume%253D33%26rft.issue%253D1%26rft.spage%253D1%26rft.epage%253D22%26rft.atitle%253DRegularization%2BPaths%2Bfor%2BGeneralized%2BLinear%2BModels%2Bvia%2BCoordinate%2BDescent.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1467-9868.2005.00503.x%26rft_id%253Dinfo%253Apmid%252F20808728%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1467-9868.2005.00503.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=20808728&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000275203200001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-31-1" title="View reference  in text" id="ref-31">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.31" data-doi="10.1111/j.1558-5646.2007.00105.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Hoekstra</span>  <span class="cit-name-given-names">H. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coyne</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">The Locus of Evolution: Evo Devo and the Genetics of Adaptation.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">61</span>: <span class="cit-fpage">995</span>–<span class="cit-lpage">1016</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DHoekstra%26rft.auinit1%253DH.%2BE.%26rft.volume%253D61%26rft.issue%253D5%26rft.spage%253D995%26rft.epage%253D1016%26rft.atitle%253DThe%2Blocus%2Bof%2Bevolution%253A%2Bevo%2Bdevo%2Band%2Bthe%2Bgenetics%2Bof%2Badaptation.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2007.00105.x%26rft_id%253Dinfo%253Apmid%252F17492956%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2007.00105.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17492956&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246200800001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-32-1" title="View reference  in text" id="ref-32">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.32"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Holtan</span>  <span class="cit-name-given-names">H. E. E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hake</span>  <span class="cit-name-given-names">S.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Quantitative Trait Locus Analysis of Leaf Dissection in Tomato Using Lycopersicon pennellii Segmental Introgression Lines.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">165</span>: <span class="cit-fpage">1541</span>–<span class="cit-lpage">1550</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DHoltan%26rft.auinit1%253DH.%2BE.%2BE.%26rft.volume%253D165%26rft.issue%253D3%26rft.spage%253D1541%26rft.epage%253D1550%26rft.atitle%253DQuantitative%2BTrait%2BLocus%2BAnalysis%2Bof%2BLeaf%2BDissection%2Bin%2BTomato%2BUsing%2BLycopersicon%2Bpennellii%2BSegmental%2BIntrogression%2BLines%26rft_id%253Dinfo%253Apmid%252F14668401%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE2NS8zLzE1NDEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMxNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-33-1" title="View reference  in text" id="ref-33">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.33" data-doi="10.1073/pnas.1402835111"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ichihashi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aguilar-Martínez</span>  <span class="cit-name-given-names">J. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Farhi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chitwood</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kumar</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Evolutionary developmental transcriptomics reveals a gene network module regulating interspecific diversity in plant leaf shape.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">E2616</span>–<span class="cit-lpage">E2621</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1402835111%26rft_id%253Dinfo%253Apmid%252F24927584%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTExLzI1L0UyNjE2IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-34-1" title="View reference  in text" id="ref-34">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.34" data-doi="10.1093/jhered/93.5.384"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Iwata</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ukai</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">SHAPE: A Computer Program Package for Quantitative Evaluation of Biological Shapes Based on Elliptic Fourier Descriptors.</span> <abbr class="cit-jnl-abbrev">J. Hered.</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">384</span>–<span class="cit-lpage">385</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJournal%2Bof%2BHeredity%26rft.stitle%253DJ%2BHered%26rft.aulast%253DIwata%26rft.auinit1%253DH.%26rft.volume%253D93%26rft.issue%253D5%26rft.spage%253D384%26rft.epage%253D385%26rft.atitle%253DSHAPE%253A%2BA%2BComputer%2BProgram%2BPackage%2Bfor%2BQuantitative%2BEvaluation%2Bof%2BBiological%2BShapes%2BBased%2Bon%2BElliptic%2BFourier%2BDescriptors%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjhered%252F93.5.384%26rft_id%253Dinfo%253Apmid%252F12547931%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiRlVMTCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamhlcmVkIjtzOjU6InJlc2lkIjtzOjg6IjkzLzUvMzg0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-35-1" title="View reference  in text" id="ref-35">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.35" data-doi="10.1007/s001220050054"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wright</span>  <span class="cit-name-given-names">R. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Woo</span>  <span class="cit-name-given-names">S. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">DelMonte</span>  <span class="cit-name-given-names">T. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Paterson</span>  <span class="cit-name-given-names">A. H.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">QTL analysis of leaf morphology in tetraploid Gossypium (cotton).</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">100</span>: <span class="cit-fpage">409</span>–<span class="cit-lpage">418</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D100%26rft.spage%253D409%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs001220050054%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s001220050054&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000086145800011&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-36-1" title="View reference  in text" id="ref-36">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.36"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jiménez-Gómez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alonso-Blanco</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Borja</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Anastasio</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Angosto</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Quantitative genetic analysis of flowering time in tomato.</span> <abbr class="cit-jnl-abbrev">Genome</abbr> <span class="cit-vol">50</span>: <span class="cit-fpage">303</span>–<span class="cit-lpage">315</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.stitle%253DGenome%2B%2528Ottawa.%2BPrint%2529%26rft.aulast%253DJimenez-Gomez%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D50%26rft.issue%253D3%26rft.spage%253D303%26rft.epage%253D315%26rft.atitle%253DQuantitative%2Bgenetic%2Banalysis%2Bof%2Bflowering%2Btime%2Bin%2Btomato.%26rft_id%253Dinfo%253Apmid%252F17502904%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=17502904&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-37-1" title="View reference  in text" id="ref-37">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.37" data-doi="10.3732/ajb.94.6.935"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Rick</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jernstedt</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chetelat</span>  <span class="cit-name-given-names">R. T.</span></span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Genealogy and fine mapping of obscuravenosa, a gene affecting the distribution of chloroplasts in leaf veins, and evidence of selection during breeding of tomatoes (Lycopersicon esculentum; Solanaceae).</span> <abbr class="cit-jnl-abbrev">Am. J. Bot.</abbr> <span class="cit-vol">94</span>: <span class="cit-fpage">935</span>–<span class="cit-lpage">947</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAmerican%2BJournal%2Bof%2BBotany%26rft.stitle%253DAm.%2BJ.%2BBot.%26rft.aulast%253DJones%26rft.auinit1%253DC.%2BM.%26rft.volume%253D94%26rft.issue%253D6%26rft.spage%253D935%26rft.epage%253D947%26rft.atitle%253DGenealogy%2Band%2Bfine%2Bmapping%2Bof%2Bobscuravenosa%252C%2Ba%2Bgene%2Baffecting%2Bthe%2Bdistribution%2Bof%2Bchloroplasts%2Bin%2Bleaf%2Bveins%252C%2Band%2Bevidence%2Bof%2Bselection%2Bduring%2Bbreeding%2Bof%2Btomatoes%2B%2528Lycopersicon%2Besculentum%253B%2BSolanaceae%2529%26rft_id%253Dinfo%253Adoi%252F10.3732%252Fajb.94.6.935%26rft_id%253Dinfo%253Apmid%252F21636462%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiYW1qYm90IjtzOjU6InJlc2lkIjtzOjg6Ijk0LzYvOTM1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-38-1" title="View reference  in text" id="ref-38">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.38" data-doi="10.1038/35079629"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kerstetter</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bollman</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Taylor</span>  <span class="cit-name-given-names">R. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bomblies</span>  <span class="cit-name-given-names">K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Poethig</span>  <span class="cit-name-given-names">R. S.</span></span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">KANADI regulates organ polarity in Arabidopsis.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">411</span>: <span class="cit-fpage">706</span>–<span class="cit-lpage">709</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKerstetter%26rft.auinit1%253DR.%2BA.%26rft.volume%253D411%26rft.issue%253D6838%26rft.spage%253D706%26rft.epage%253D709%26rft.atitle%253DKANADI%2Bregulates%2Borgan%2Bpolarity%2Bin%2BArabidopsis.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35079629%26rft_id%253Dinfo%253Apmid%252F11395775%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35079629&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11395775&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169112500049&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-39-1" title="View reference  in text" id="ref-39">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.39" data-doi="10.1534/genetics.106.066423"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Keurentjes</span>  <span class="cit-name-given-names">J. J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bentsink</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alonso-Blanco</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanhart</span>  <span class="cit-name-given-names">C. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Vries</span>  <span class="cit-name-given-names">H. B.-D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Development of a Near-Isogenic Line Population of Arabidopsis thaliana and Comparison of Mapping Power With a Recombinant Inbred Line Population.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">175</span>: <span class="cit-fpage">891</span>–<span class="cit-lpage">905</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.106.066423%26rft_id%253Dinfo%253Apmid%252F17179089%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTc1LzIvODkxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-40-1" title="View reference  in text" id="ref-40">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.40" data-doi="10.1038/nature01820"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCormick</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Timmermans</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinha</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span><span class="cit-comment">a</span> <span class="cit-article-title">The expression domain of PHANTASTICA determines leaflet placement in compound leaves.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">424</span>: <span class="cit-fpage">438</span>–<span class="cit-lpage">443</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DKim%26rft.auinit1%253DM.%26rft.volume%253D424%26rft.issue%253D6947%26rft.spage%253D438%26rft.epage%253D443%26rft.atitle%253DThe%2Bexpression%2Bdomain%2Bof%2BPHANTASTICA%2Bdetermines%2Bleaflet%2Bplacement%2Bin%2Bcompound%2Bleaves.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnature01820%26rft_id%253Dinfo%253Apmid%252F12879073%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nature01820&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=12879073&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000184318400046&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-41-1" title="View reference  in text" id="ref-41">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.41" data-doi="10.1242/dev.00655"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pham</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hamidi</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">McCormick</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuzoff</span>  <span class="cit-name-given-names">R. K.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2003</span><span class="cit-comment">b</span> <span class="cit-article-title">Reduced leaf complexity in tomato wiry mutants suggests a role for PHAN and KNOX genes in generating compound leaves.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">130</span>: <span class="cit-fpage">4405</span>–<span class="cit-lpage">4415</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DKim%26rft.auinit1%253DM.%26rft.volume%253D130%26rft.issue%253D18%26rft.spage%253D4405%26rft.epage%253D4415%26rft.atitle%253DReduced%2Bleaf%2Bcomplexity%2Bin%2Btomato%2Bwiry%2Bmutants%2Bsuggests%2Ba%2Brole%2Bfor%2BPHAN%2Band%2BKNOX%2Bgenes%2Bin%2Bgenerating%2Bcompound%2Bleaves%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.00655%26rft_id%253Dinfo%253Apmid%252F12900456%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTMwLzE4LzQ0MDUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMxNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-42-1" title="View reference  in text" id="ref-42">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.42" data-doi="10.1016/j.cub.2008.04.008"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Koenig</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yoong</span>  <span class="cit-name-given-names">F. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinha</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Natural Variation in Leaf Morphology Results from Mutation of a Novel KNOX Gene.</span> <abbr class="cit-jnl-abbrev">Curr. Biol.</abbr> <span class="cit-vol">18</span>: <span class="cit-fpage">672</span>–<span class="cit-lpage">677</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCurrent%2Bbiology%2B%253A%2B%2BCB%26rft.stitle%253DCurr%2BBiol%26rft.aulast%253DKimura%26rft.auinit1%253DS.%26rft.volume%253D18%26rft.issue%253D9%26rft.spage%253D672%26rft.epage%253D677%26rft.atitle%253DNatural%2Bvariation%2Bin%2Bleaf%2Bmorphology%2Bresults%2Bfrom%2Bmutation%2Bof%2Ba%2Bnovel%2BKNOX%2Bgene.%26rft_id%253Dinfo%253Adoi%252F10.1016%252Fj.cub.2008.04.008%26rft_id%253Dinfo%253Apmid%252F18424140%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/j.cub.2008.04.008&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18424140&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000255668400028&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-43-1" title="View reference  in text" id="ref-43">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.43" data-doi="10.1016/0304-4238(77)90074-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Kinet</span>  <span class="cit-name-given-names">J. M.</span></span></li></ol><cite>, <span class="cit-pub-date">1977</span> <span class="cit-article-title">Effect of light conditions on the development of the inflorescence in tomato.</span> <abbr class="cit-jnl-abbrev">Sci. Hortic. (Amsterdam)</abbr> <span class="cit-vol">6</span>: <span class="cit-fpage">15</span>–<span class="cit-lpage">26</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DSci.%2BHortic.%2B%2528Amsterdam%2529%26rft.volume%253D6%26rft.spage%253D15%26rft_id%253Dinfo%253Adoi%252F10.1016%252F0304-4238%252877%252990074-7%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/0304-4238(77)90074-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-44-1" title="View reference  in text" id="ref-44">↵</a><div class="cit ref-cit ref-book" id="cit-6.10.3169.44"><div class="cit-metadata unstructured">Koenig, D., and N. Sinha, 2010 Chapter Six. Evolution Of Leaf Shape: A Pattern Emerges, pp. 169–183 in <em>Current Topics in Developmental Biology</em>, edited by M. C. P. Timmermans. Elsevier Academic Press, London.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-45-1" title="View reference  in text" id="ref-45">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.45" data-doi="10.1242/dev.033811"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koenig</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bayer</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kuhlemeier</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinha</span>  <span class="cit-name-given-names">N.</span></span></li></ol><cite>, <span class="cit-pub-date">2009</span> <span class="cit-article-title">Auxin patterns Solanum lycopersicum leaf morphogenesis.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">136</span>: <span class="cit-fpage">2997</span>–<span class="cit-lpage">3006</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DKoenig%26rft.auinit1%253DD.%26rft.volume%253D136%26rft.issue%253D17%26rft.spage%253D2997%26rft.epage%253D3006%26rft.atitle%253DAuxin%2Bpatterns%2BSolanum%2Blycopersicum%2Bleaf%2Bmorphogenesis%26rft_id%253Dinfo%253Adoi%252F10.1242%252Fdev.033811%26rft_id%253Dinfo%253Apmid%252F19666826%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTM2LzE3LzI5OTciO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMxNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-46-1" title="View reference  in text" id="ref-46">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.46" data-doi="10.1073/pnas.1309606110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koenig</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiménez-Gómez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kimura</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fulop</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chitwood</span>  <span class="cit-name-given-names">D. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Comparative transcriptomics reveals patterns of selection in domesticated and wild tomato.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">E2655</span>–<span class="cit-lpage">E2662</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1309606110%26rft_id%253Dinfo%253Apmid%252F23803858%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTEwLzI4L0UyNjU1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-47-1" title="View reference  in text" id="ref-47">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.47" data-doi="10.1046/j.1365-313X.2000.00754.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Koltai</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bird</span>  <span class="cit-name-given-names">D. M.</span></span></li></ol><cite>, <span class="cit-pub-date">2000</span> <span class="cit-article-title">Epistatic repression of PHANTASTICA and class 1 KNOTTED genes is uncoupled in tomato.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> <span class="cit-vol">22</span>: <span class="cit-fpage">455</span>–<span class="cit-lpage">459</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2Bjournal%2B%253A%2B%2Bfor%2Bcell%2Band%2Bmolecular%2Bbiology%26rft.stitle%253DPlant%2BJ%26rft.aulast%253DKoltai%26rft.auinit1%253DH.%26rft.volume%253D22%26rft.issue%253D5%26rft.spage%253D455%26rft.epage%253D459%26rft.atitle%253DEpistatic%2Brepression%2Bof%2BPHANTASTICA%2Band%2Bclass%2B1%2BKNOTTED%2Bgenes%2Bis%2Buncoupled%2Bin%2Btomato.%26rft_id%253Dinfo%253Adoi%252F10.1046%252Fj.1365-313X.2000.00754.x%26rft_id%253Dinfo%253Apmid%252F10849361%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1046/j.1365-313X.2000.00754.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=10849361&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000087312200008&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-48-1" title="View reference  in text" id="ref-48">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.48" data-doi="10.1093/jxb/err277"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ku</span>  <span class="cit-name-given-names">L. X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhang</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Guo</span>  <span class="cit-name-given-names">S. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">H. Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zhao</span>  <span class="cit-name-given-names">R. F.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Integrated multiple population analysis of leaf architecture traits in maize (Zea mays L.).</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">261</span>–<span class="cit-lpage">274</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferr277%26rft_id%253Dinfo%253Apmid%252F21984652%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4Ym90IjtzOjU6InJlc2lkIjtzOjg6IjYzLzEvMjYxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-49-1" title="View reference  in text" id="ref-49">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.49" data-doi="10.1073/pnas.0504210102"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Langlade</span>  <span class="cit-name-given-names">N. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Feng</span>  <span class="cit-name-given-names">X.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dransfield</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Copsey</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hanna</span>  <span class="cit-name-given-names">A. I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">Evolution through genetically controlled allometry space.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">102</span>: <span class="cit-fpage">10221</span>–<span class="cit-lpage">10226</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0504210102%26rft_id%253Dinfo%253Apmid%252F16009935%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAyLzI5LzEwMjIxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-50-1" title="View reference  in text" id="ref-50">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.50" data-doi="10.1371/journal.pcbi.1003118"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lawrence</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Huber</span>  <span class="cit-name-given-names">W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pagès</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aboyoun</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carlson</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Software for Computing and Annotating Genomic Ranges.</span> <abbr class="cit-jnl-abbrev">PLOS Comput. Biol.</abbr> <span class="cit-vol">9</span>: <span class="cit-fpage">e1003118</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPLOS%2BComput.%2BBiol.%26rft.volume%253D9%26rft.spage%253De1003118%26rft_id%253Dinfo%253Adoi%252F10.1371%252Fjournal.pcbi.1003118%26rft_id%253Dinfo%253Apmid%252F23950696%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1371/journal.pcbi.1003118&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=23950696&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-51-1" title="View reference  in text" id="ref-51">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.51" data-doi="10.1093/jxb/erl106"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lifschitz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eshed</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Universal florigenic signals triggered by FT homologues regulate growth and flowering cycles in perennial day-neutral tomato.</span> <abbr class="cit-jnl-abbrev">J. Exp. Bot.</abbr> <span class="cit-vol">57</span>: <span class="cit-fpage">3405</span>–<span class="cit-lpage">3414</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BExp.%2BBot.%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fjxb%252Ferl106%26rft_id%253Dinfo%253Apmid%252F17005925%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NjoiamV4Ym90IjtzOjU6InJlc2lkIjtzOjEwOiI1Ny8xMy8zNDA1IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-52-1" title="View reference  in text" id="ref-52">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.52" data-doi="10.3389/fpls.2014.00465"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lifschitz</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ayre</span>  <span class="cit-name-given-names">B. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eshed</span>  <span class="cit-name-given-names">Y.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Florigen and anti-florigen – a systemic mechanism for coordinating growth and termination in flowering plants.</span> <abbr class="cit-jnl-abbrev">Front. Plant Sci.</abbr> <span class="cit-vol">5</span>: <span class="cit-fpage">465</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFront.%2BPlant%2BSci.%26rft.volume%253D5%26rft.spage%253D465%26rft_id%253Dinfo%253Adoi%252F10.3389%252Ffpls.2014.00465%26rft_id%253Dinfo%253Apmid%252F25278944%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.3389/fpls.2014.00465&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25278944&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><div class="cit ref-cit ref-journal no-rev-xref" id="cit-6.10.3169.53"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">Y. S.</span></span></li></ol><cite>, and D. Zamir, <span class="cit-pub-date">1999</span> Second generation L. pennellii introgression lines and the concept of bin mapping. Tomato Genet. Coop. <span class="cit-vol">49</span>: <span class="cit-fpage">26</span>–<span class="cit-lpage">30</span>.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-54-1" title="View reference  in text" id="ref-54">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.54" data-doi="10.1093/bioinformatics/btu057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Lott</span>  <span class="cit-name-given-names">P. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Korf</span>  <span class="cit-name-given-names">I.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">StochHMM: a flexible hidden Markov model tool and C++ library.</span> <abbr class="cit-jnl-abbrev">Bioinformatics</abbr> <span class="cit-vol">30</span>: <span class="cit-fpage">1625</span>–<span class="cit-lpage">1626</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBioinformatics%26rft_id%253Dinfo%253Adoi%252F10.1093%252Fbioinformatics%252Fbtu057%26rft_id%253Dinfo%253Apmid%252F24489371%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiYmlvaW5mbyI7czo1OiJyZXNpZCI7czoxMDoiMzAvMTEvMTYyNSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTAvMzE2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-55-1" title="View reference  in text" id="ref-55">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.55" data-doi="10.1198/jasa.2011.tm09738"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mazumder</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Friedman</span>  <span class="cit-name-given-names">J. H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hastie</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">SparseNet: Coordinate Descent With Nonconvex Penalties.</span> <abbr class="cit-jnl-abbrev">J. Am. Stat. Assoc.</abbr> <span class="cit-vol">106</span>: <span class="cit-fpage">1125</span>–<span class="cit-lpage">1138</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DJ.%2BAm.%2BStat.%2BAssoc.%26rft.volume%253D106%26rft.spage%253D1125%26rft_id%253Dinfo%253Adoi%252F10.1198%252Fjasa.2011.tm09738%26rft_id%253Dinfo%253Apmid%252F25580042%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1198/jasa.2011.tm09738&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=25580042&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-56-1" title="View reference  in text" id="ref-56">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.56" data-doi="10.1038/35079635"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">McConnell</span>  <span class="cit-name-given-names">J. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Emery</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Eshed</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bao</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bowman</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2001</span> <span class="cit-article-title">Role of PHABULOSA and PHAVOLUTA in determining radial patterning in shoots.</span> <abbr class="cit-jnl-abbrev">Nature</abbr> <span class="cit-vol">411</span>: <span class="cit-fpage">709</span>–<span class="cit-lpage">713</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%26rft.stitle%253DNature%26rft.aulast%253DMcConnell%26rft.auinit1%253DJ.%2BR.%26rft.volume%253D411%26rft.issue%253D6838%26rft.spage%253D709%26rft.epage%253D713%26rft.atitle%253DRole%2Bof%2BPHABULOSA%2Band%2BPHAVOLUTA%2Bin%2Bdetermining%2Bradial%2Bpatterning%2Bin%2Bshoots.%26rft_id%253Dinfo%253Adoi%252F10.1038%252F35079635%26rft_id%253Dinfo%253Apmid%252F11395776%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/35079635&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11395776&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000169112500050&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-57-1" title="View reference  in text" id="ref-57">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.57" data-doi="10.1186/1471-2164-13-72"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Monson-Miller</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sanchez-Mendez</span>  <span class="cit-name-given-names">D. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Fass</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henry</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Tai</span>  <span class="cit-name-given-names">T. H.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Reference genome-independent assessment of mutation density using restriction enzyme-phased sequencing.</span> <abbr class="cit-jnl-abbrev">BMC Genomics</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">72</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DBMC%2Bgenomics%2B%255Belectronic%2Bresource%255D%26rft.stitle%253DBMC%2BGenomics%26rft.aulast%253DMonson-Miller%26rft.auinit1%253DJ.%26rft.volume%253D13%26rft.spage%253D72%26rft.epage%253D72%26rft.atitle%253DReference%2Bgenome-independent%2Bassessment%2Bof%2Bmutation%2Bdensity%2Busing%2Brestriction%2Benzyme-phased%2Bsequencing.%26rft_id%253Dinfo%253Adoi%252F10.1186%252F1471-2164-13-72%26rft_id%253Dinfo%253Apmid%252F22333298%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/1471-2164-13-72&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22333298&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-58-1" title="View reference  in text" id="ref-58">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.58"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Mouradov</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cremer</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coupland</span>  <span class="cit-name-given-names">G.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Control of Flowering Time.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">14</span>: <span class="cit-fpage">s111</span>–<span class="cit-lpage">s130</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DThe%2BPlant%2BCell%2BOnline%26rft.stitle%253DPlant%2BCell%26rft.volume%253D14%26rft.issue%253D90001%26rft.spage%253Ds111%26rft.epage%253D130%26rft.atitle%253DControl%2Bof%2BFlowering%2BTime%253A%2BInteracting%2BPathways%2Bas%2Ba%2BBasis%2Bfor%2BDiversity%26rft_id%253Dinfo%253Apmid%252F12045273%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12045273&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000176187500009&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-59-1" title="View reference  in text" id="ref-59">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.59" data-doi="10.1111/j.1558-5646.2008.00487.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Moyle</span>  <span class="cit-name-given-names">L. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Ecological and Evolutionary Genomics in the Wild Tomatoes (solanum Sect. Lycopersicon).</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">62</span>: <span class="cit-fpage">2995</span>–<span class="cit-lpage">3013</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DMoyle%26rft.auinit1%253DL.%2BC.%26rft.volume%253D62%26rft.issue%253D12%26rft.spage%253D2995%26rft.epage%253D3013%26rft.atitle%253DEcological%2Band%2Bevolutionary%2Bgenomics%2Bin%2Bthe%2Bwild%2Btomatoes%2B%2528Solanum%2Bsect.%2BLycopersicon%2529.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.1558-5646.2008.00487.x%26rft_id%253Dinfo%253Apmid%252F18752600%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.1558-5646.2008.00487.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=18752600&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000261442900005&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-60-1" title="View reference  in text" id="ref-60">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.60" data-doi="10.1534/genetics.114.169276"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Muir</span>  <span class="cit-name-given-names">C. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pease</span>  <span class="cit-name-given-names">J. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moyle</span>  <span class="cit-name-given-names">L. C.</span></span></li></ol><cite>, <span class="cit-pub-date">2014</span> <span class="cit-article-title">Quantitative Genetic Analysis Indicates Natural Selection on Leaf Phenotypes Across Wild Tomato Species (Solanum sect. Lycopersicon; Solanaceae).</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">198</span>: <span class="cit-fpage">1629</span>–<span class="cit-lpage">1643</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft_id%253Dinfo%253Adoi%252F10.1534%252Fgenetics.114.169276%26rft_id%253Dinfo%253Apmid%252F25298519%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6MTA6IjE5OC80LzE2MjkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMxNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-61-1" title="View reference  in text" id="ref-61">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.61"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Müller</span>  <span class="cit-name-given-names">N. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Wijnen</span>  <span class="cit-name-given-names">C. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Srinivasan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ryngajllo</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ofner</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Domestication selected for deceleration of the circadian clock in cultivated tomato.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">48</span>: <span class="cit-fpage">89</span>–<span class="cit-lpage">93</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNat.%2BGenet.%26rft.volume%253D48%26rft.spage%253D89%26rft_id%253Dinfo%253Apmid%252F26569124%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=26569124&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-62-1" title="View reference  in text" id="ref-62">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.62" data-doi="10.1126/science.1079354"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nath</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Crawford</span>  <span class="cit-name-given-names">B. C. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carpenter</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Coen</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Genetic Control of Surface Curvature.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">299</span>: <span class="cit-fpage">1404</span>–<span class="cit-lpage">1407</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DNath%26rft.auinit1%253DU.%26rft.volume%253D299%26rft.issue%253D5611%26rft.spage%253D1404%26rft.epage%253D1407%26rft.atitle%253DGenetic%2BControl%2Bof%2BSurface%2BCurvature%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1079354%26rft_id%253Dinfo%253Apmid%252F12610308%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIyOTkvNTYxMS8xNDA0IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-63-1" title="View reference  in text" id="ref-63">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.63" data-doi="10.1073/pnas.1214300110"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Naz</span>  <span class="cit-name-given-names">A. A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Raman</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Martinez</span>  <span class="cit-name-given-names">C. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinha</span>  <span class="cit-name-given-names">N. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schmitz</span>  <span class="cit-name-given-names">G.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2013</span> <span class="cit-article-title">Trifoliate encodes an MYB transcription factor that modulates leaf and shoot architecture in tomato.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">110</span>: <span class="cit-fpage">2401</span>–<span class="cit-lpage">2406</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1214300110%26rft_id%253Dinfo%253Apmid%252F23341595%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMDoiMTEwLzYvMjQwMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTAvMzE2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-64-1" title="View reference  in text" id="ref-64">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.64" data-doi="10.1071/FP11057"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Nicotra</span>  <span class="cit-name-given-names">A. B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leigh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Boyce</span>  <span class="cit-name-given-names">C. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">C. S.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Niklas</span>  <span class="cit-name-given-names">K. J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">The evolution and functional significance of leaf shape in the angiosperms.</span> <abbr class="cit-jnl-abbrev">Funct. Plant Biol.</abbr> <span class="cit-vol">38</span>: <span class="cit-fpage">535</span>–<span class="cit-lpage">552</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DFunct.%2BPlant%2BBiol.%26rft.volume%253D38%26rft.spage%253D535%26rft_id%253Dinfo%253Adoi%252F10.1071%252FFP11057%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1071/FP11057&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=000292603900001&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-65-1" title="View reference  in text" id="ref-65">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.65" data-doi="10.1104/pp.15.00474"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ning</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Moghe</span>  <span class="cit-name-given-names">G. D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Leong</span>  <span class="cit-name-given-names">B.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Kim</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ofner</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2015</span> <span class="cit-article-title">A Feedback-Insensitive Isopropylmalate Synthase Affects Acylsugar Composition in Cultivated and Wild Tomato.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">169</span>: <span class="cit-fpage">1821</span>–<span class="cit-lpage">1835</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.15.00474%26rft_id%253Dinfo%253Apmid%252F25986128%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTY5LzMvMTgyMSI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTAvMzE2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-66-1" title="View reference  in text" id="ref-66">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.66"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ofner</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Lashbrooke</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Pleban</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Aharoni</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zamir</span>  <span class="cit-name-given-names">D.</span></span></li></ol><cite>, <span class="cit-pub-date">2016</span> <span class="cit-article-title">Solanum pennellii backcross inbred lines (BILs) link small genomic bins with tomato traits.</span> <abbr class="cit-jnl-abbrev">Plant J.</abbr> DOI: 10.1111/tpj.13194.</cite></div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-67-1" title="View reference  in text" id="ref-67">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.67" data-doi="10.1038/ng2036"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ori</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Cohen</span>  <span class="cit-name-given-names">A. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Etzioni</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brand</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Yanai</span>  <span class="cit-name-given-names">O.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Regulation of LANCEOLATE by miR319 is required for compound-leaf development in tomato.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">39</span>: <span class="cit-fpage">787</span>–<span class="cit-lpage">791</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DOri%26rft.auinit1%253DN.%26rft.volume%253D39%26rft.issue%253D6%26rft.spage%253D787%26rft.epage%253D791%26rft.atitle%253DRegulation%2Bof%2BLANCEOLATE%2Bby%2BmiR319%2Bis%2Brequired%2Bfor%2Bcompound-leaf%2Bdevelopment%2Bin%2Btomato.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng2036%26rft_id%253Dinfo%253Apmid%252F17486095%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng2036&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=17486095&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000246859100024&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-68-1" title="View reference  in text" id="ref-68">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.68"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pérez-Pérez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Serrano-Cartagena</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Micol</span>  <span class="cit-name-given-names">J. L.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Genetic Analysis of Natural Variations in the Architecture of Arabidopsis thaliana Vegetative Leaves.</span> <abbr class="cit-jnl-abbrev">Genetics</abbr> <span class="cit-vol">162</span>: <span class="cit-fpage">893</span>–<span class="cit-lpage">915</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenetics%26rft.stitle%253DGenetics%26rft.aulast%253DPerez-Perez%26rft.auinit1%253DJ.%2BM.%26rft.volume%253D162%26rft.issue%253D2%26rft.spage%253D893%26rft.epage%253D915%26rft.atitle%253DGenetic%2BAnalysis%2Bof%2BNatural%2BVariations%2Bin%2Bthe%2BArchitecture%2Bof%2BArabidopsis%2Bthaliana%2BVegetative%2BLeaves%26rft_id%253Dinfo%253Apmid%252F12399398%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6ODoiZ2VuZXRpY3MiO3M6NToicmVzaWQiO3M6OToiMTYyLzIvODkzIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-69-1" title="View reference  in text" id="ref-69">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.69"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Pnueli</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Carmel-Goren</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hareven</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gutfinger</span>  <span class="cit-name-given-names">T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Alvarez</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The SELF-PRUNING gene of tomato regulates vegetative to reproductive switching of sympodial meristems and is the ortholog of CEN and TFL1.</span> <abbr class="cit-jnl-abbrev">Development</abbr> <span class="cit-vol">125</span>: <span class="cit-fpage">1979</span>–<span class="cit-lpage">1989</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DDevelopment%26rft.stitle%253DDevelopment%26rft.aulast%253DPnueli%26rft.auinit1%253DL%26rft.volume%253D125%26rft.issue%253D11%26rft.spage%253D1979%26rft.epage%253D1989%26rft.atitle%253DThe%2BSELF-PRUNING%2Bgene%2Bof%2Btomato%2Bregulates%2Bvegetative%2Bto%2Breproductive%2Bswitching%2Bof%2Bsympodial%2Bmeristems%2Band%2Bis%2Bthe%2Bortholog%2Bof%2BCEN%2Band%2BTFL1%26rft_id%253Dinfo%253Apmid%252F9570763%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NzoiZGV2ZWxvcCI7czo1OiJyZXNpZCI7czoxMToiMTI1LzExLzE5NzkiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMxNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-70-1" title="View reference  in text" id="ref-70">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.70" data-doi="10.1186/gb-2012-13-1-r1"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Ranjan</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ichihashi</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinha</span>  <span class="cit-name-given-names">N. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">The tomato genome: implications for plant breeding, genomics and evolution.</span> <abbr class="cit-jnl-abbrev">Genome Biol.</abbr> <span class="cit-vol">13</span>: <span class="cit-fpage">1</span>–<span class="cit-lpage">8</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DGenome%2BBiol.%26rft.volume%253D13%26rft.spage%253D1%26rft_id%253Dinfo%253Adoi%252F10.1186%252Fgb-2012-13-1-r1%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1186/gb-2012-13-1-r1&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-71-1" title="View reference  in text" id="ref-71">↵</a><div class="cit ref-cit ref-book" id="cit-6.10.3169.71"><div class="cit-metadata unstructured">Ranjan, A., J. Budke, S. D. Rowland, D. H. Chitwood, R. Kumar <em>et al.</em>, 2016 eQTL regulating Transcript Levels Associated with Diverse Biological Processes in Tomato. Plant Physiol. DOI: 10.1104/pp.16.00289.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-72-1" title="View reference  in text" id="ref-72">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.72" data-doi="10.1007/s00122-005-0071-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Rousseaux</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jones</span>  <span class="cit-name-given-names">C. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Adams</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chetelat</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bennett</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2005</span> <span class="cit-article-title">QTL analysis of fruit antioxidants in tomato using Lycopersicon pennellii introgression lines.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">111</span>: <span class="cit-fpage">1396</span>–<span class="cit-lpage">1408</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTAG.%2BTheoretical%2Band%2Bapplied%2Bgenetics.%2B%2BTheoretische%2Bund%2Bangewandte%2BGenetik%26rft.stitle%253DTheor%2BAppl%2BGenet%26rft.aulast%253DRousseaux%26rft.auinit1%253DM.%2BC.%26rft.volume%253D111%26rft.issue%253D7%26rft.spage%253D1396%26rft.epage%253D1408%26rft.atitle%253DQTL%2Banalysis%2Bof%2Bfruit%2Bantioxidants%2Bin%2Btomato%2Busing%2BLycopersicon%2Bpennellii%2Bintrogression%2Blines.%26rft_id%253Dinfo%253Adoi%252F10.1007%252Fs00122-005-0071-7%26rft_id%253Dinfo%253Apmid%252F16177901%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1007/s00122-005-0071-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16177901&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000233323900019&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-73-1" title="View reference  in text" id="ref-73">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.73" data-doi="10.1038/nbt1192"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schauer</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Semel</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Roessner</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gur</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balbo</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Comprehensive metabolic profiling and phenotyping of interspecific introgression lines for tomato improvement.</span> <abbr class="cit-jnl-abbrev">Nat. Biotechnol.</abbr> <span class="cit-vol">24</span>: <span class="cit-fpage">447</span>–<span class="cit-lpage">454</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bbiotechnology%26rft.stitle%253DNat%2BBiotechnol%26rft.aulast%253DSchauer%26rft.auinit1%253DN.%26rft.volume%253D24%26rft.issue%253D4%26rft.spage%253D447%26rft.epage%253D454%26rft.atitle%253DComprehensive%2Bmetabolic%2Bprofiling%2Band%2Bphenotyping%2Bof%2Binterspecific%2Bintrogression%2Blines%2Bfor%2Btomato%2Bimprovement.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fnbt1192%26rft_id%253Dinfo%253Apmid%252F16531992%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/nbt1192&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=16531992&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000236766300029&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-74-1" title="View reference  in text" id="ref-74">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.74" data-doi="10.1105/tpc.107.056523"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Schauer</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Semel</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Balbo</span>  <span class="cit-name-given-names">I.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinfath</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Repsilber</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">Mode of Inheritance of Primary Metabolic Traits in Tomato.</span> <abbr class="cit-jnl-abbrev">Plant Cell</abbr> <span class="cit-vol">20</span>: <span class="cit-fpage">509</span>–<span class="cit-lpage">523</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BCell%26rft_id%253Dinfo%253Adoi%252F10.1105%252Ftpc.107.056523%26rft_id%253Dinfo%253Apmid%252F18364465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6OToicGxhbnRjZWxsIjtzOjU6InJlc2lkIjtzOjg6IjIwLzMvNTA5IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-75-1" title="View reference  in text" id="ref-75">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.75" data-doi="10.1073/pnas.0604635103"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Semel</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nissenbaum</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Menda</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Zinder</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Krieger</span>  <span class="cit-name-given-names">U.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2006</span> <span class="cit-article-title">Overdominant quantitative trait loci for yield and fitness in tomato.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">103</span>: <span class="cit-fpage">12981</span>–<span class="cit-lpage">12986</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.0604635103%26rft_id%253Dinfo%253Apmid%252F16938842%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMjoiMTAzLzM1LzEyOTgxIjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-76-1" title="View reference  in text" id="ref-76">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.76" data-doi="10.1073/pnas.1117277109"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Seymour</span>  <span class="cit-name-given-names">D. K.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Filiault</span>  <span class="cit-name-given-names">D. L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Henry</span>  <span class="cit-name-given-names">I. M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Monson-Miller</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Ravi</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Rapid creation of Arabidopsis doubled haploid lines for quantitative trait locus mapping.</span> <abbr class="cit-jnl-abbrev">Proc. Natl. Acad. Sci. USA</abbr> <span class="cit-vol">109</span>: <span class="cit-fpage">4227</span>–<span class="cit-lpage">4232</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DProc.%2BNatl.%2BAcad.%2BSci.%2BUSA%26rft_id%253Dinfo%253Adoi%252F10.1073%252Fpnas.1117277109%26rft_id%253Dinfo%253Apmid%252F22371599%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6NDoicG5hcyI7czo1OiJyZXNpZCI7czoxMToiMTA5LzExLzQyMjciO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMxNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-77-1" title="View reference  in text" id="ref-77">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.77"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Sharlach</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dahlbeck</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Liu</span>  <span class="cit-name-given-names">L.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Chiu</span>  <span class="cit-name-given-names">J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiménez-Gómez</span>  <span class="cit-name-given-names">J. M.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">Fine genetic mapping of RXopJ4, a bacterial spot disease resistance locus from Solanum pennellii LA716.</span> <abbr class="cit-jnl-abbrev">Theor. Appl. Genet.</abbr> <span class="cit-vol">126</span>: <span class="cit-fpage">601</span>–<span class="cit-lpage">609</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DTheor.%2BAppl.%2BGenet.%26rft.volume%253D126%26rft.spage%253D601%26rft_id%253Dinfo%253Apmid%252F23117718%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=23117718&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-78-1" title="View reference  in text" id="ref-78">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.78" data-doi="10.1126/science.296.5566.285"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Simpson</span>  <span class="cit-name-given-names">G. G.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Dean</span>  <span class="cit-name-given-names">C.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Arabidopsis, the Rosetta Stone of Flowering Time?</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">296</span>: <span class="cit-fpage">285</span>–<span class="cit-lpage">289</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DSimpson%26rft.auinit1%253DG.%2BG.%26rft.volume%253D296%26rft.issue%253D5566%26rft.spage%253D285%26rft.epage%253D289%26rft.atitle%253DArabidopsis%252C%2Bthe%2BRosetta%2BStone%2Bof%2BFlowering%2BTime%253F%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.296.5566.285%26rft_id%253Dinfo%253Apmid%252F11951029%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyOTYvNTU2Ni8yODUiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMxNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-79-1" title="View reference  in text" id="ref-79">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.79" data-doi="10.1104/pp.111.181594"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Steinhauser</span>  <span class="cit-name-given-names">M.-C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Steinhauser</span>  <span class="cit-name-given-names">D.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Gibon</span>  <span class="cit-name-given-names">Y.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bolger</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Arrivault</span>  <span class="cit-name-given-names">S.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Identification of Enzyme Activity Quantitative Trait Loci in a Solanum lycopersicum × Solanum pennellii Introgression Line Population.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">157</span>: <span class="cit-fpage">998</span>–<span class="cit-lpage">1014</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.111.181594%26rft_id%253Dinfo%253Apmid%252F21890649%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czo5OiIxNTcvMy85OTgiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMxNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-80-1" title="View reference  in text" id="ref-80">↵</a><div class="cit ref-cit ref-book" id="cit-6.10.3169.80"><div class="cit-metadata unstructured">Stevens, M. A., and C. M. Rick, 1986 Genetics and breeding, pp. 35–109 in <em>The Tomato Crop</em>, edited by J. G. Atherton and J. Rudich. Springer, Dordrecht.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-81-1" title="View reference  in text" id="ref-81">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.81" data-doi="10.1104/pp.106.091413"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Stevens</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Buret</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Duffé</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Garchery</span>  <span class="cit-name-given-names">C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Baldet</span>  <span class="cit-name-given-names">P.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2007</span> <span class="cit-article-title">Candidate Genes and Quantitative Trait Loci Affecting Fruit Ascorbic Acid Content in Three Tomato Populations.</span> <abbr class="cit-jnl-abbrev">Plant Physiol.</abbr> <span class="cit-vol">143</span>: <span class="cit-fpage">1943</span>–<span class="cit-lpage">1953</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DPlant%2BPhysiol.%26rft_id%253Dinfo%253Adoi%252F10.1104%252Fpp.106.091413%26rft_id%253Dinfo%253Apmid%252F17277090%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6MTI6InBsYW50cGh5c2lvbCI7czo1OiJyZXNpZCI7czoxMDoiMTQzLzQvMTk0MyI7czo0OiJhdG9tIjtzOjE5OiIvZ2dnLzYvMTAvMzE2OS5hdG9tIjt9czo4OiJmcmFnbWVudCI7czowOiIiO30=" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-82-1" title="View reference  in text" id="ref-82">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.82" data-doi="10.1038/ng.746"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tian</span>  <span class="cit-name-given-names">F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Bradbury</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Brown</span>  <span class="cit-name-given-names">P. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hung</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sun</span>  <span class="cit-name-given-names">Q.</span></span>, </li><li><span class="cit-etal">et al.</span></li></ol><cite>, <span class="cit-pub-date">2011</span> <span class="cit-article-title">Genome-wide association study of leaf architecture in the maize nested association mapping population.</span> <abbr class="cit-jnl-abbrev">Nat. Genet.</abbr> <span class="cit-vol">43</span>: <span class="cit-fpage">159</span>–<span class="cit-lpage">162</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Bgenetics%26rft.stitle%253DNat%2BGenet%26rft.aulast%253DTian%26rft.auinit1%253DF.%26rft.volume%253D43%26rft.issue%253D2%26rft.spage%253D159%26rft.epage%253D162%26rft.atitle%253DGenome-wide%2Bassociation%2Bstudy%2Bof%2Bleaf%2Barchitecture%2Bin%2Bthe%2Bmaize%2Bnested%2Bassociation%2Bmapping%2Bpopulation.%26rft_id%253Dinfo%253Adoi%252F10.1038%252Fng.746%26rft_id%253Dinfo%253Apmid%252F21217756%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1038/ng.746&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=21217756&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000286623800016&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-83-1" title="View reference  in text" id="ref-83">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.83" data-doi="10.1126/science.284.5411.151"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Timmermans</span>  <span class="cit-name-given-names">M. C.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Becraft</span>  <span class="cit-name-given-names">P. W.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Nelson</span>  <span class="cit-name-given-names">T.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">ROUGH SHEATH2: A Myb Protein That Represses knox Homeobox Genes in Maize Lateral Organ Primordia.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">284</span>: <span class="cit-fpage">151</span>–<span class="cit-lpage">153</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTimmermans%26rft.auinit1%253DM.%2BC.%2BP.%26rft.volume%253D284%26rft.issue%253D5411%26rft.spage%253D151%26rft.epage%253D153%26rft.atitle%253DROUGH%2BSHEATH2%253A%2BA%2BMyb%2BProtein%2BThat%2BRepresses%2Bknox%2BHomeobox%2BGenes%2Bin%2BMaize%2BLateral%2BOrgan%2BPrimordia%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.284.5411.151%26rft_id%253Dinfo%253Apmid%252F10102816%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODQvNTQxMS8xNTEiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMxNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-84-1" title="View reference  in text" id="ref-84">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.84" data-doi="10.1146/annurev-arplant-042811-105524"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Townsley</span>  <span class="cit-name-given-names">B. T.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Sinha</span>  <span class="cit-name-given-names">N. R.</span></span></li></ol><cite>, <span class="cit-pub-date">2012</span> <span class="cit-article-title">A New Development: Evolving Concepts in Leaf Ontogeny.</span> <abbr class="cit-jnl-abbrev">Annu. Rev. Plant Biol.</abbr> <span class="cit-vol">63</span>: <span class="cit-fpage">535</span>–<span class="cit-lpage">562</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DAnnual%2Breview%2Bof%2Bplant%2Bbiology%26rft.stitle%253DAnnu%2BRev%2BPlant%2BBiol%26rft.aulast%253DTownsley%26rft.auinit1%253DB.%2BT.%26rft.volume%253D63%26rft.spage%253D535%26rft.epage%253D562%26rft.atitle%253DA%2Bnew%2Bdevelopment%253A%2Bevolving%2Bconcepts%2Bin%2Bleaf%2Bontogeny.%26rft_id%253Dinfo%253Adoi%252F10.1146%252Fannurev-arplant-042811-105524%26rft_id%253Dinfo%253Apmid%252F22404465%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1146/annurev-arplant-042811-105524&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=22404465&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000307953100022&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-85-1" title="View reference  in text" id="ref-85">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.85"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsiantis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hay</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">2003</span> <span class="cit-article-title">Comparative plant development: the time of the leaf?</span> <abbr class="cit-jnl-abbrev">Nat. Rev. Genet.</abbr> <span class="cit-vol">4</span>: <span class="cit-fpage">169</span>–<span class="cit-lpage">180</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DNature%2Breviews.%2BGenetics%26rft.stitle%253DNat%2BRev%2BGenet%26rft.aulast%253DTsiantis%26rft.auinit1%253DM.%26rft.volume%253D4%26rft.issue%253D3%26rft.spage%253D169%26rft.epage%253D180%26rft.atitle%253DComparative%2Bplant%2Bdevelopment%253A%2Bthe%2Btime%2Bof%2Bthe%2Bleaf%253F%26rft_id%253Dinfo%253Apmid%252F12610522%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=12610522&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000181511900010&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-86-1" title="View reference  in text" id="ref-86">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.86" data-doi="10.1126/science.284.5411.154"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Tsiantis</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schneeberger</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Golz</span>  <span class="cit-name-given-names">J. F.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Freeling</span>  <span class="cit-name-given-names">M.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Langdale</span>  <span class="cit-name-given-names">J. A.</span></span></li></ol><cite>, <span class="cit-pub-date">1999</span> <span class="cit-article-title">The Maize rough sheath2 Gene and Leaf Development Programs in Monocot and Dicot Plants.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">284</span>: <span class="cit-fpage">154</span>–<span class="cit-lpage">156</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DTsiantis%26rft.auinit1%253DM.%26rft.volume%253D284%26rft.issue%253D5411%26rft.spage%253D154%26rft.epage%253D156%26rft.atitle%253DThe%2BMaize%2Brough%2Bsheath2%2BGene%2Band%2BLeaf%2BDevelopment%2BPrograms%2Bin%2BMonocot%2Band%2BDicot%2BPlants%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.284.5411.154%26rft_id%253Dinfo%253Apmid%252F10102817%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEyOiIyODQvNTQxMS8xNTQiO3M6NDoiYXRvbSI7czoxOToiL2dnZy82LzEwLzMxNjkuYXRvbSI7fXM6ODoiZnJhZ21lbnQiO3M6MDoiIjt9" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-87-1" title="View reference  in text" id="ref-87">↵</a><div class="cit ref-cit ref-book" id="cit-6.10.3169.87"><div class="cit-metadata unstructured">Tsukaya, H., 2013 Leaf Development. Arabidopsis Book 11: e0163.</div><div class="cit-extra"></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-88-1" title="View reference  in text" id="ref-88">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.88" data-doi="10.1016/S0092-8674(00)81439-7"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Waites</span>  <span class="cit-name-given-names">R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Selvadurai</span>  <span class="cit-name-given-names">H. R. N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Oliver</span>  <span class="cit-name-given-names">I. R.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Hudson</span>  <span class="cit-name-given-names">A.</span></span></li></ol><cite>, <span class="cit-pub-date">1998</span> <span class="cit-article-title">The PHANTASTICA Gene Encodes a MYB Transcription Factor Involved in Growth and Dorsoventrality of Lateral Organs in Antirrhinum.</span> <abbr class="cit-jnl-abbrev">Cell</abbr> <span class="cit-vol">93</span>: <span class="cit-fpage">779</span>–<span class="cit-lpage">789</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DCell%26rft.stitle%253DCell%26rft.aulast%253DWaites%26rft.auinit1%253DR.%26rft.volume%253D93%26rft.issue%253D5%26rft.spage%253D779%26rft.epage%253D789%26rft.atitle%253DThe%2BPHANTASTICA%2Bgene%2Bencodes%2Ba%2BMYB%2Btranscription%2Bfactor%2Binvolved%2Bin%2Bgrowth%2Band%2Bdorsoventrality%2Bof%2Blateral%2Borgans%2Bin%2BAntirrhinum.%26rft_id%253Dinfo%253Adoi%252F10.1016%252FS0092-8674%252800%252981439-7%26rft_id%253Dinfo%253Apmid%252F9630222%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1016/S0092-8674(00)81439-7&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=9630222&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000073956700014&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-89-1" title="View reference  in text" id="ref-89">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.89" data-doi="10.1111/j.0014-3820.2002.tb01337.x"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Westerbergh</span>  <span class="cit-name-given-names">A.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Doebley</span>  <span class="cit-name-given-names">J.</span></span></li></ol><cite>, <span class="cit-pub-date">2002</span> <span class="cit-article-title">Morphological Traits Defining Species Differences in Wild Relatives of Maize Are Controlled by Multiple Quantitative Trait Loci.</span> <abbr class="cit-jnl-abbrev">Evolution</abbr> <span class="cit-vol">56</span>: <span class="cit-fpage">273</span>–<span class="cit-lpage">283</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DEvolution%26rft.stitle%253DEvolution%26rft.aulast%253DWesterbergh%26rft.auinit1%253DA.%26rft.volume%253D56%26rft.issue%253D2%26rft.spage%253D273%26rft.epage%253D283%26rft.atitle%253DMorphological%2Btraits%2Bdefining%2Bspecies%2Bdifferences%2Bin%2Bwild%2Brelatives%2Bof%2Bmaize%2Bare%2Bcontrolled%2Bby%2Bmultiple%2Bquantitative%2Btrait%2Bloci.%26rft_id%253Dinfo%253Adoi%252F10.1111%252Fj.0014-3820.2002.tb01337.x%26rft_id%253Dinfo%253Apmid%252F11926495%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/external-ref?access_num=10.1111/j.0014-3820.2002.tb01337.x&amp;link_type=DOI" class="cit-ref-sprinkles cit-ref-sprinkles-doi cit-ref-sprinkles-crossref"><span>CrossRef</span></a><a href="/lookup/external-ref?access_num=11926495&amp;link_type=MED&amp;atom=%2Fggg%2F6%2F10%2F3169.atom" class="cit-ref-sprinkles cit-ref-sprinkles-medline"><span>PubMed</span></a><a href="/lookup/external-ref?access_num=000174486200006&amp;link_type=ISI" class="cit-ref-sprinkles cit-ref-sprinkles-newisilink cit-ref-sprinkles-webofscience"><span>Web of Science</span></a></div></div></li><li><span class="ref-label ref-label-empty"></span><a class="rev-xref-ref" href="#xref-ref-90-1" title="View reference  in text" id="ref-90">↵</a><div class="cit ref-cit ref-journal" id="cit-6.10.3169.90" data-doi="10.1126/science.1153040"><div class="cit-metadata"><ol class="cit-auth-list"><li><span class="cit-auth"><span class="cit-name-surname">Xiao</span>  <span class="cit-name-given-names">H.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Jiang</span>  <span class="cit-name-given-names">N.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Schaffner</span>  <span class="cit-name-given-names">E.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">Stockinger</span>  <span class="cit-name-given-names">E. J.</span></span>, </li><li><span class="cit-auth"><span class="cit-name-surname">van der Knaap</span>  <span class="cit-name-given-names">E.</span></span></li></ol><cite>, <span class="cit-pub-date">2008</span> <span class="cit-article-title">A Retrotransposon-Mediated Gene Duplication Underlies Morphological Variation of Tomato Fruit.</span> <abbr class="cit-jnl-abbrev">Science</abbr> <span class="cit-vol">319</span>: <span class="cit-fpage">1527</span>–<span class="cit-lpage">1530</span>.</cite></div><div class="cit-extra"><a href="{openurl}?query=rft.jtitle%253DScience%26rft.stitle%253DScience%26rft.aulast%253DXiao%26rft.auinit1%253DH.%26rft.volume%253D319%26rft.issue%253D5869%26rft.spage%253D1527%26rft.epage%253D1530%26rft.atitle%253DA%2BRetrotransposon-Mediated%2BGene%2BDuplication%2BUnderlies%2BMorphological%2BVariation%2Bof%2BTomato%2BFruit%26rft_id%253Dinfo%253Adoi%252F10.1126%252Fscience.1153040%26rft_id%253Dinfo%253Apmid%252F18339939%26rft.genre%253Darticle%26rft_val_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Ajournal%26ctx_ver%253DZ39.88-2004%26url_ver%253DZ39.88-2004%26url_ctx_fmt%253Dinfo%253Aofi%252Ffmt%253Akev%253Amtx%253Actx" class="cit-ref-sprinkles cit-ref-sprinkles-openurl cit-ref-sprinkles-open-url"><span>OpenUrl</span></a><a href="/lookup/ijlink/YTozOntzOjQ6InBhdGgiO3M6MTQ6Ii9sb29rdXAvaWpsaW5rIjtzOjU6InF1ZXJ5IjthOjQ6e3M6ODoibGlua1R5cGUiO3M6NDoiQUJTVCI7czoxMToiam91cm5hbENvZGUiO3M6Mzoic2NpIjtzOjU6InJlc2lkIjtzOjEzOiIzMTkvNTg2OS8xNTI3IjtzOjQ6ImF0b20iO3M6MTk6Ii9nZ2cvNi8xMC8zMTY5LmF0b20iO31zOjg6ImZyYWdtZW50IjtzOjA6IiI7fQ==" class="cit-ref-sprinkles cit-ref-sprinkles-ijlink"><span><span class="cit-reflinks-abstract">Abstract</span><span class="cit-sep cit-reflinks-variant-name-sep">/</span><span class="cit-reflinks-full-text"><span class="free-full-text">FREE </span>Full Text</span></span></a></div></div></li></ol></div><span class="highwire-journal-article-marker-end"></span></div><span id="related-urls"></span></div><a href="https://www.g3journal.org/content/6/10/3169.abstract" class="hw-link hw-link-article-abstract" data-icon-position="" data-hide-link-title="0">View Abstract</a></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-node-pager" >
  
      
  
  <div class="pane-content">
    <div class="pager highwire-pager pager-mini clearfix highwire-node-pager highwire-article-pager"><span class="pager-prev"><a href="/content/6/10/3161" title="Efficient CRISPR-Mediated Post-Transcriptional Gene Silencing in a Hyperthermophilic Archaeon Using Multiplexed crRNA Expression" rel="prev" class="pager-link-prev link-icon"><i class="icon-circle-arrow-left"></i> <span class="title">Previous Article</span></a></span><span class="pager-next"><a href="/content/6/10/3185" title="An Eye on Trafficking Genes: Identification of Four Eye Color Mutations in Drosophila" rel="next" class="pager-link-next link-icon-right link-icon"><span class="title">Next Article</span> <i class="icon-circle-arrow-right"></i></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-back-to-top" >
  
      
  
  <div class="pane-content">
    <a href="#page" class="back-to-top" data-icon-position="" data-hide-link-title="0"><i class="icon-chevron-up"></i> Back to top</a>  </div>

  
  </div>
</div>
			</div>
		</div>
		
		<div class="sidebar-right-wrapper grid-10 omega">
			<div class="panel-panel panel-region-sidebar-right">
			  <div class="inside"><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-issue article-volume-issue-cover top-border top-padding increased-bottom-margin" >
  
        <h2 class="pane-title">PUBLICATION INFORMATION</h2>
    
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_issue" class="highwire-2col-stacked panel-display">
	  <div class="panel-row-wrapper panel-row-first clearfix">
		
		<div class="top-wrapper">
			<div class="panel-panel panel-region-top">
			  <div class="inside"><div class="panel-pane pane-custom pane-1" >
  
      
  
  <div class="pane-content">
    <p><a href="/content/6/10"><span>Volume 6 Issue 10, October 2016</span></a></p>
  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-highwire-issue-cover" >
  
      
  
  <div class="pane-content">
    <div class = "cover-issue"><div class = "cover-issue-image"><a href="/content/6/10.toc" class="" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/default/files/styles/issue_cover_248x400_/public/highwire/ggg/6/10.cover-source.jpg?itok=eRzcKCYf" width="248" height="330" alt="G3: Genes|Genomes|Genetics: 6 (10)" /></a></div></div>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"></div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
		<div class="panel-row-wrapper panel-row-last clearfix">
		
		<div class="bottom-wrapper">
			<div class="panel-panel panel-region-bottom">
			  <div class="inside"><div class="panel-pane pane-highwire-seach-quicksearch" >
  
      
  
  <div class="pane-content">
    <form class="highwire-quicksearch button-style-mini button-style-mini" action="/content/6/10/3169.abstract" method="post" id="highwire-search-quicksearch-form-0" accept-charset="UTF-8"><div><div class="form-item form-item-label-invisible form-type-textfield form-item-keywords">
  <label class="element-invisible" for="edit-keywords">Search for this keyword </label>
 <input placeholder="Search this issue" type="text" id="edit-keywords" name="keywords" value="" size="60" maxlength="128" class="form-text" />
</div>
<div class="button-wrapper button-mini"><i class="icon-search"></i><input data-icon-only="1" data-font-icon="icon-search" data-icon-position="after" type="submit" id="edit-submit" name="op" value="Search" class="form-submit" /></div><input type="hidden" name="form_build_id" value="form-VhwLqeT2W5Sz-FnsdTbLN7bsstt1YxWZqf06PvQJsUI" />
<input type="hidden" name="form_id" value="highwire_search_quicksearch_form_0" />
</div></form>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-entity-field pane-node-field-highwire-article-category vert-padding vert-borders increased-bottom-margin" >
  
        <h2 class="pane-title">ARTICLE CLASSIFICATION</h2>
    
  
  <div class="pane-content">
    <div class="field field-name-field-highwire-article-category field-type-taxonomy-term-reference field-label-hidden"><div class="field-items"><div class="field-item even"><a href="/category/investigations-0" class="" data-icon-position="" data-hide-link-title="0">Investigations</a></div></div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-lens increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="highwire-lens-button"><span class="inner"><a href="/lens/ggg/6/10/3169" target="_blank" class="" data-icon-position="" data-hide-link-title="0"><span>View this article with <em><strong>LENS</strong></em></span></a></span></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-art-tools vert-borders vert-padding increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div id="mini-panel-jnl_genetics_art_tools" class="highwire-2col-stacked panel-display">
	  
  <div class="panel-row-wrapper clearfix">
		<div class="content-left-wrapper content-column">
          <div class="panel-panel panel-region-content-left">
            <div class="inside"><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-email" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger" title="" data-icon-position="" data-hide-link-title="0"><i class = 'icon-envelope'></i> Email</a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_email">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-block pane-forward-form pane-forward" >
  
      
  
  <div class="pane-content">
    <form action="/content/6/10/3169" method="post" id="forward-form" accept-charset="UTF-8"><div><div id="edit-instructions" class="form-item form-item-label-before form-type-item">
 <p>Thank you for sharing this G3: Genes | Genomes | Genetics article.</p><p>NOTE: We request your email address only to inform the recipient that it was you who recommended this article, and that it is not junk mail. We do not retain these email addresses.</p>
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-email">
  <label for="edit-email">Your Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-email" name="email" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name">Your Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name" name="name" value="" size="58" maxlength="256" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-recipients">
  <label for="edit-recipients">Send To <span class="form-required" title="This field is required.">*</span></label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-recipients" name="recipients" cols="50" rows="5" class="form-textarea required"></textarea></div>
<div class="description">Enter multiple addresses on separate lines or separate them with commas.</div>
</div>
<div id="edit-page" class="form-item form-item-label-before form-type-item">
  <label for="edit-page">You are going to email the following </label>
 <a href="/content/6/10/3169" class="active" data-icon-position="" data-hide-link-title="0">A New Advanced Backcross Tomato Population Enables High Resolution Leaf QTL Mapping and Gene Identification</a>
</div>
<div id="edit-subject" class="form-item form-item-label-before form-type-item">
  <label for="edit-subject">Message Subject </label>
 (Your Name) has forwarded a page to you from G3: Genes | Genomes | Genetics
</div>
<div id="edit-body" class="form-item form-item-label-before form-type-item">
  <label for="edit-body">Message Body </label>
 (Your Name) thought you would be interested in this article in G3: Genes | Genomes | Genetics.
</div>
<div class="form-item form-item-label-before form-type-textarea form-item-message">
  <label for="edit-message--2">Your Personal Message </label>
 <div class="form-textarea-wrapper resizable"><textarea id="edit-message--2" name="message" cols="50" rows="10" class="form-textarea"></textarea></div>
</div>
<input type="hidden" name="path" value="node/460344" />
<input type="hidden" name="path_cid" value="" />
<input type="hidden" name="forward_footer" value=" " />
<input type="hidden" name="form_build_id" value="form-OyC8g-uDiQfeJSzZ25u317J4xVZVYHtr6SJpuDF34HY" />
<input type="hidden" name="form_id" value="forward_form" />
<div class="form-actions form-wrapper" id="edit-actions"><input type="submit" id="edit-submit" name="op" value="Send Message" class="form-submit" /></div></div></form>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/10/3169.full-text.print" target="_blank" rel="nofollow" class="link-icon"><i class="icon-print"></i> <span class="title">Print</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-art-alert" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-bullhorn"></i> <span class="title">Alerts</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_art_alert">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-article-add-form" >
  
      
  
  <div class="pane-content">
    <div id = "alerts-form-wrapper"><form action="/content/6/10/3169" method="post" id="highwire-user-opportunity-login" accept-charset="UTF-8"><div><div class="form-item form-item-label-before form-type-textfield form-item-name">
  <label for="edit-name--2">User Name <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-name--2" name="name" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div class="form-item form-item-label-before form-type-password form-item-pass">
  <label for="edit-pass">Password <span class="form-required" title="This field is required.">*</span></label>
 <input type="password" id="edit-pass" name="pass" size="60" maxlength="128" class="form-text required" />
</div>
Enter your email below to set up alert notifications for new article, or to manage your existing alerts.<div class="form-item form-item-label-invisible form-type-textfield form-item-highwire-alerts-email">
  <label class="element-invisible" for="edit-highwire-alerts-email">Email <span class="form-required" title="This field is required.">*</span></label>
 <input type="text" id="edit-highwire-alerts-email" name="highwire_alerts_email" value="" size="60" maxlength="128" class="form-text required" />
</div>
<div id="alerts_email"><b>SIGN UP OR SIGN IN WITH YOUR EMAIL</b></div><input type="submit" id="edit-submit--4" name="op" value="Submit" class="form-submit" /><input type="hidden" name="form_build_id" value="form-kWFXkAYsO_ala_ig1JvtoxkkPbuqBlKfn4EbqvJ7Xek" />
<input type="hidden" name="form_id" value="highwire_alerts_article_sign_up_form" />
<input type="hidden" name="current_path" value="content/6/10/3169" />
</div></form></div>  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-variant-link" >
  
      
  
  <div class="pane-content">
    <a href="/content/ggg/6/10/3169.full-text.pdf" target="_blank" class="link-icon"><i class="icon-download"></i> <span class="title">View PDF</span></a>  </div>

  
  </div>
</div>
          </div>
        </div>
        
        <div class="content-right-wrapper content-column">
          <div class="panel-panel panel-region-content-right">
            <div class="inside"><div class="panel-pane pane-highwire-share-link highwire_clipboard_link_ajax"  id="shareit">
  
      
  
  <div class="pane-content">
    <a href="/" class="link-icon"><i class="icon-share-alt"></i> <span class="title">Share</span></a>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-share highwire_clipboard_form_ajax_shareit" >
  
      
  
  <div class="pane-content">
    
<div class="panel-display omega-12-onecol" id="mini-panel-jnl_genetics_share">

		  <div class="panel-panel grid-12 panel-region-preface">
		  <div class="inside"><div class="panel-pane pane-highwire-article-citation" >
  
      
  
  <div class="pane-content">
    <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460344" id="node460344--2" data-pisa="ggg;6/10/3169" data-pisa-master="ggg;g3.116.030536" data-apath="/ggg/6/10/3169.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A New Advanced Backcross Tomato Population Enables High Resolution Leaf QTL Mapping and Gene Identification</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3778-4107" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel Fulop</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-3447-4442" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aashish Ranjan</span>, <span class="highwire-citation-author" data-delta="2">Itai Ofner</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-0555-6409" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michael F. Covington</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-4875-1447" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel H. Chitwood</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-0624-8034" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Donelly West</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-1935-4397" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yasunori Ichihashi</span>, <span class="highwire-citation-author" data-delta="7">Lauren Headland</span>, <span class="highwire-citation-author" data-delta="8">Daniel Zamir</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-9623-2599" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Julin N. Maloof</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0002-1494-7065" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Neelima R. Sinha</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3169-3184;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.030536 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460344--2" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel Fulop </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BFulop%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fulop%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BFulop%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3778-4107" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel Fulop</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Aashish Ranjan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aashish%2BRanjan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ranjan%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAashish%2BRanjan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3447-4442" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aashish Ranjan</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Itai Ofner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Plant Sciences and Genetics in Agriculture, The Hebrew University of Jerusalem, Rehovot, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Itai%2BOfner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ofner%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AItai%2BOfner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Michael F. Covington </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BF.%2BCovington%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Covington%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BF.%2BCovington%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0555-6409" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael F. Covington</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Daniel H. Chitwood </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BH.%2BChitwood%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chitwood%20DH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BH.%2BChitwood%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4875-1447" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel H. Chitwood</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Donelly West </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Donelly%2BWest%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=West%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADonelly%2BWest%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0624-8034" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Donelly West</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Yasunori Ichihashi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yasunori%2BIchihashi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ichihashi%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYasunori%2BIchihashi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1935-4397" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yasunori Ichihashi</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Lauren Headland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lauren%2BHeadland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Headland%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALauren%2BHeadland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Daniel Zamir </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Plant Sciences and Genetics in Agriculture, The Hebrew University of Jerusalem, Rehovot, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BZamir%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zamir%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BZamir%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Julin N. Maloof </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julin%2BN.%2BMaloof%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maloof%20JN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJulin%2BN.%2BMaloof%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9623-2599" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Julin N. Maloof</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nrsinha@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">nrsinha@ucdavis.edu</a>
<a href="mailto:jnmaloof@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">jnmaloof@ucdavis.edu</a></span></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Neelima R. Sinha </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Neelima%2BR.%2BSinha%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sinha%20NR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANeelima%2BR.%2BSinha%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1494-7065" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Neelima R. Sinha</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nrsinha@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">nrsinha@ucdavis.edu</a>
<a href="mailto:jnmaloof@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">jnmaloof@ucdavis.edu</a></span></li></ul></div></div></div>  </div>

  
  </div>
</div>
	  </div>
	
  <div class="panel-panel grid-12 panel-region-content">
  	<div class="inside"><div class="panel-pane pane-highwire-article-clipboard-copy" >
  
      
  
  <div class="pane-content">
    <div class = "clipboard-copy">
  <span class="label-url">
    <label for="dynamic">Share This Article:</label>
  </span>
  <span class="input-text-url">
    <input type="text" id="dynamic" value="https://www.g3journal.org/content/6/10/3169" size="50"/>
  </span>
  <span class="copy-button button">
    <button id="copy-dynamic" class="clipboardjs-button" data-clipboard-target="#dynamic" data-clipboard-alert-style="tooltip" data-clipboard-alert-text="Copied!">Copy</button>
  </span>
</div>
  </div>

  
  </div>
</div>
  </div>
	
		  <div class="panel-panel grid-12 panel-region-postscript">
	    <div class="inside"><div class="panel-pane pane-service-links" >
  
      
  
  <div class="pane-content">
    <div class="service-links"><a href="http://del.icio.us/post?url=https%3A//www.g3journal.org/content/6/10/3169&amp;title=A%20New%20Advanced%20Backcross%20Tomato%20Population%20Enables%20High%20Resolution%20Leaf%20QTL%20Mapping%20and%20Gene%20Identification" id="delicious" title="Bookmark this post on del.icio.us" class="service-links-delicious" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/delicious-32.png" alt="del.icio.us logo" /></a> <a href="http://digg.com/submit?phase=2&amp;url=https%3A//www.g3journal.org/content/6/10/3169&amp;title=A%20New%20Advanced%20Backcross%20Tomato%20Population%20Enables%20High%20Resolution%20Leaf%20QTL%20Mapping%20and%20Gene%20Identification" id="digg" title="Digg this post on digg.com" class="service-links-digg" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/digg.png" alt="Digg logo" /></a> <a href="http://reddit.com/submit?url=https%3A//www.g3journal.org/content/6/10/3169&amp;title=A%20New%20Advanced%20Backcross%20Tomato%20Population%20Enables%20High%20Resolution%20Leaf%20QTL%20Mapping%20and%20Gene%20Identification" id="reddit" title="Submit this post on reddit.com" class="service-links-reddit" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/reddit.png" alt="Reddit logo" /></a> <a href="http://twitter.com/share?url=https%3A//www.g3journal.org/content/6/10/3169&amp;text=A%20New%20Advanced%20Backcross%20Tomato%20Population%20Enables%20High%20Resolution%20Leaf%20QTL%20Mapping%20and%20Gene%20Identification" id="twitter" title="Share this on Twitter" class="service-links-twitter" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/twitter.png" alt="Twitter logo" /></a> <a href="http://www.citeulike.org/posturl?url=https%3A//www.g3journal.org/content/6/10/3169&amp;title=A%20New%20Advanced%20Backcross%20Tomato%20Population%20Enables%20High%20Resolution%20Leaf%20QTL%20Mapping%20and%20Gene%20Identification" id="citeulike" title="Share on CiteULike" class="service-links-citeulike" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/citeyoulike.png" alt="CiteULike logo" /></a> <a href="http://www.facebook.com/sharer.php?u=https%3A//www.g3journal.org/content/6/10/3169&amp;t=A%20New%20Advanced%20Backcross%20Tomato%20Population%20Enables%20High%20Resolution%20Leaf%20QTL%20Mapping%20and%20Gene%20Identification" id="facebook" title="Share on Facebook" class="service-links-facebook" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/fb-blue.png" alt="Facebook logo" /></a> <a href="http://www.google.com/bookmarks/mark?op=add&amp;bkmk=https%3A//www.g3journal.org/content/6/10/3169&amp;title=A%20New%20Advanced%20Backcross%20Tomato%20Population%20Enables%20High%20Resolution%20Leaf%20QTL%20Mapping%20and%20Gene%20Identification" id="google" title="Bookmark this post on Google" class="service-links-google" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/google-32.png" alt="Google logo" /></a> <a href="http://www.mendeley.com/import/?url=https%3A//www.g3journal.org/content/6/10/3169&amp;title=A%20New%20Advanced%20Backcross%20Tomato%20Population%20Enables%20High%20Resolution%20Leaf%20QTL%20Mapping%20and%20Gene%20Identification" id="mendeley" title="Share on Mendeley" class="service-links-mendeley" rel="nofollow" data-icon-position="" data-hide-link-title="0"><img src="https://www.g3journal.org/sites/all/modules/highwire/highwire/images/mendeley.png" alt="Mendeley logo" /></a></div>  </div>

  
  </div>
</div>
	  </div>
	
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-minipanel-dialog-link pane-jnl-genetics-cite-tool" >
  
      
  
  <div class="pane-content">
    <div class='minipanel-dialog-wrapper'><div class='minipanel-dialog-link-link'><a href="/highwire/citation/460344/download" oncontextmenu="javascript: return false;" class="minipanel-dialog-link-trigger link-icon" title=""><i class="icon-globe"></i> <span class="title">Citation</span></a></div><div class='minipanel-dialog-link-mini' style='display:none'><div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_cite_tool">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-highwire-citation-export" >
  
      
  
  <div class="pane-content">
    <div class="highwire-citation-export">  
  <div class="highwire-citation-info">
      <div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="460344" id="node460344--3" data-pisa="ggg;6/10/3169" data-pisa-master="ggg;g3.116.030536" data-apath="/ggg/6/10/3169.atom" data-hw-author-tooltip-instance="highwire_author_tooltip"><div class="highwire-cite highwire-cite-highwire-article highwire-citation-genetics-article-title-complete clearfix has-author-tooltip highwire-citation-genetics-article-title-complete">
  
  
  
      <h1 class="highwire-cite-title" id="page-title">A New Advanced Backcross Tomato Population Enables High Resolution Leaf QTL Mapping and Gene Identification</h1>  
      <div class="highwire-cite-authors"><span  class="highwire-citation-authors"><span class="highwire-citation-author first hw-author-orcid-logo-wrapper" data-delta="0"><a href="http://orcid.org/0000-0002-3778-4107" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel Fulop</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="1"><a href="http://orcid.org/0000-0003-3447-4442" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Aashish Ranjan</span>, <span class="highwire-citation-author" data-delta="2">Itai Ofner</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="3"><a href="http://orcid.org/0000-0002-0555-6409" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Michael F. Covington</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="4"><a href="http://orcid.org/0000-0003-4875-1447" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Daniel H. Chitwood</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="5"><a href="http://orcid.org/0000-0003-0624-8034" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Donelly West</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="6"><a href="http://orcid.org/0000-0003-1935-4397" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Yasunori Ichihashi</span>, <span class="highwire-citation-author" data-delta="7">Lauren Headland</span>, <span class="highwire-citation-author" data-delta="8">Daniel Zamir</span>, <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="9"><a href="http://orcid.org/0000-0002-9623-2599" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Julin N. Maloof</span> and <span class="highwire-citation-author hw-author-orcid-logo-wrapper" data-delta="10"><a href="http://orcid.org/0000-0002-1494-7065" target="_blank" class="hw-author-orcid-logo link-icon-only link-icon"><i class="hw-icon-orcid hw-icon-color-orcid"></i> <span class="title element-invisible">View ORCID Profile</span></a>Neelima R. Sinha</span></span></div>
  
      <div class="highwire-cite-metadata"><span  class="highwire-cite-metadata-journal highwire-cite-metadata">G3: Genes, Genomes, Genetics </span><span  class="highwire-cite-metadata-date highwire-cite-metadata">October 1, 2016 </span><span  class="highwire-cite-metadata-volume highwire-cite-metadata">vol. 6 </span><span  class="highwire-cite-metadata-issue highwire-cite-metadata">no. 10 </span><span  class="highwire-cite-metadata-pages highwire-cite-metadata">3169-3184;  </span><span  class="highwire-cite-metadata-doi highwire-cite-metadata">https://doi.org/10.1534/g3.116.030536 </span></div>
  
      <div class="highwire-cite-access"><span class="highwire-citation-access"><i class="highwire-access-icon highwire-access-icon-open-access open-access icon-unlock-alt" title="Open Access" aria-hidden="true"></i><span  class="element-invisible highwire-access-icon highwire-access-icon-open-access" style="">Open Access</span></span></div>
  
  
  
</div>
<div id="hw-article-author-popups-node460344--3" style="display: none;"><div class="author-tooltip-0"><div class="author-tooltip-name">Daniel Fulop </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BFulop%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Fulop%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BFulop%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-3778-4107" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel Fulop</a></li></ul></div><div class="author-tooltip-1"><div class="author-tooltip-name">Aashish Ranjan </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Aashish%2BRanjan%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ranjan%20A&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AAashish%2BRanjan%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-3447-4442" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Aashish Ranjan</a></li></ul></div><div class="author-tooltip-2"><div class="author-tooltip-name">Itai Ofner </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Plant Sciences and Genetics in Agriculture, The Hebrew University of Jerusalem, Rehovot, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Itai%2BOfner%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ofner%20I&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3AItai%2BOfner%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-3"><div class="author-tooltip-name">Michael F. Covington </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Michael%2BF.%2BCovington%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Covington%20MF&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AMichael%2BF.%2BCovington%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0002-0555-6409" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Michael F. Covington</a></li></ul></div><div class="author-tooltip-4"><div class="author-tooltip-name">Daniel H. Chitwood </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BH.%2BChitwood%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Chitwood%20DH&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADaniel%2BH.%2BChitwood%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-4875-1447" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Daniel H. Chitwood</a></li></ul></div><div class="author-tooltip-5"><div class="author-tooltip-name">Donelly West </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Donelly%2BWest%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=West%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ADonelly%2BWest%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-0624-8034" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Donelly West</a></li></ul></div><div class="author-tooltip-6"><div class="author-tooltip-name">Yasunori Ichihashi </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Yasunori%2BIchihashi%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Ichihashi%20Y&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AYasunori%2BIchihashi%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link last"><a href="http://orcid.org/0000-0003-1935-4397" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Yasunori Ichihashi</a></li></ul></div><div class="author-tooltip-7"><div class="author-tooltip-name">Lauren Headland </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Lauren%2BHeadland%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Headland%20L&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ALauren%2BHeadland%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-8"><div class="author-tooltip-name">Daniel Zamir </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>†</span>Institute of Plant Sciences and Genetics in Agriculture, The Hebrew University of Jerusalem, Rehovot, Israel</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Daniel%2BZamir%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Zamir%20D&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link last"><a href="/search/author1%3ADaniel%2BZamir%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li></ul></div><div class="author-tooltip-9"><div class="author-tooltip-name">Julin N. Maloof </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Julin%2BN.%2BMaloof%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Maloof%20JN&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3AJulin%2BN.%2BMaloof%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-9623-2599" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Julin N. Maloof</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nrsinha@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">nrsinha@ucdavis.edu</a>
<a href="mailto:jnmaloof@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">jnmaloof@ucdavis.edu</a></span></li></ul></div><div class="author-tooltip-10"><div class="author-tooltip-name">Neelima R. Sinha </div><div class="author-tooltip-affiliation"><span class="author-tooltip-text"><div class='author-affiliation'><span class='nlm-sup'>*</span>Department of Plant Biology, University of California at Davis, California 95616</div></span></div><ul class="author-tooltip-find-more"><li class="author-tooltip-gs-link first"><a href="/lookup/google-scholar?link_type=googlescholar&amp;gs_type=author&amp;author%5B0%5D=Neelima%2BR.%2BSinha%2B" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on Google Scholar</a></li><li class="author-tooltip-pubmed-link"><a href="/lookup/external-ref?access_num=Sinha%20NR&amp;link_type=AUTHORSEARCH" target="_blank" class="" data-icon-position="" data-hide-link-title="0">Find this author on PubMed</a></li><li class="author-site-search-link"><a href="/search/author1%3ANeelima%2BR.%2BSinha%2B" rel="nofollow" class="" data-icon-position="" data-hide-link-title="0">Search for this author on this site</a></li><li class="author-orcid-link"><a href="http://orcid.org/0000-0002-1494-7065" target="_blank" class="" data-icon-position="" data-hide-link-title="0">ORCID record for Neelima R. Sinha</a></li><li class="author-corresp-email-link last"><span>For correspondence: 
<a href="mailto:nrsinha@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">nrsinha@ucdavis.edu</a>
<a href="mailto:jnmaloof@ucdavis.edu" class="" data-icon-position="" data-hide-link-title="0">jnmaloof@ucdavis.edu</a></span></li></ul></div></div></div>  </div>
  <div class="highwire-citation-formats">
  	      <h2>Citation Manager Formats</h2>
        <div class="highwire-citation-formats-links">
      <span class="Z3988" title="ctx_ver=Z39.88-2004&amp;rft_val_fmt=info%3Aofi%2Ffmt%3Akev%3Amtx%3Ajournal&amp;rft.spage=3169&amp;rft.epage=3184&amp;rft.atitle=A%20New%20Advanced%20Backcross%20Tomato%20Population%20Enables%20High%20Resolution%20Leaf%20QTL%20Mapping%20and%20Gene%20Identification&amp;rft.volume=6&amp;rft.issue=10&amp;rft.date=2016-10-01%2000%3A00%3A00&amp;rft.stitle&amp;rft.jtitle=G3%3A%20Genes%2C%20Genomes%2C%20Genetics&amp;rft.au=Fulop%2C+Daniel&amp;rft.au=Ranjan%2C+Aashish&amp;rft.au=Ofner%2C+Itai&amp;rft.au=Covington%2C+Michael+F.&amp;rft.au=Chitwood%2C+Daniel+H.&amp;rft.au=West%2C+Donelly&amp;rft.au=Ichihashi%2C+Yasunori&amp;rft.au=Headland%2C+Lauren&amp;rft.au=Zamir%2C+Daniel&amp;rft.au=Maloof%2C+Julin+N.&amp;rft.au=Sinha%2C+Neelima+R."></span><ul class="hw-citation-links inline button button-alt button-grid clearfix"><li class="bibtext first"><a href="/highwire/citation/460344/bibtext" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">BibTeX</a></li><li class="bookends"><a href="/highwire/citation/460344/bookends" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Bookends</a></li><li class="easybib"><a href="/highwire/citation/460344/easybib" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EasyBib</a></li><li class="endnote-tagged"><a href="/highwire/citation/460344/endnote-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote (tagged)</a></li><li class="endnote-8-xml"><a href="/highwire/citation/460344/endnote-8-xml" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">EndNote 8 (xml)</a></li><li class="medlars"><a href="/highwire/citation/460344/medlars" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Medlars</a></li><li class="mendeley"><a href="/highwire/citation/460344/mendeley" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Mendeley</a></li><li class="papers"><a href="/highwire/citation/460344/papers" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Papers</a></li><li class="refworks-tagged"><a href="/highwire/citation/460344/refworks-tagged" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RefWorks Tagged</a></li><li class="reference-manager"><a href="/highwire/citation/460344/reference-manager" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Ref Manager</a></li><li class="ris"><a href="/highwire/citation/460344/ris" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">RIS</a></li><li class="zotero last"><a href="/highwire/citation/460344/zotero" class="hw-download-citation-link" data-icon-position="" data-hide-link-title="0">Zotero</a></li></ul>    </div>
  </div>
</div>
  </div>

  
  </div>
</div>
  </div>
</div>
</div></div>  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-snippet pane-genetics-cc-by-logo" >
  
      
  
  <div class="pane-content">
    <div class="snippet genetics-cc-by-logo" id="genetics-cc-by-logo">
  
      
  <div class="snippet-content">
    <p><a href="https://creativecommons.org/licenses/by/4.0/"><img alt="" src="/sites/default/files/additional-assets/cc-by-logo.png" style="width: 80px; height: 15px;" /></a></p>
  </div>

</div>
  </div>

  
  </div>
</div>
          </div>
        </div>
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-panels-mini pane-jnl-genetics-accordion increased-bottom-margin" >
  
      
  
  <div class="pane-content">
    <div class="panel-display panel-1col clearfix" id="mini-panel-jnl_genetics_accordion">
  <div class="panel-panel panel-col">
    <div><div id="highwire_article_accordion_container"><h3 >Related Articles</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-5' data-pid='new-5'></div></div><h3 >Cited By</h3><div><div class='panels-ajax-pane panels-ajax-pane-new-6' data-pid='new-6'></div></div><h3 >More in this TOC Section</h3><div><div class="highwire-list highwire-article-citation-list"><ul><li class="first odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495867" id="node495867" data-pisa="ggg;10/3/1019" data-pisa-master="ggg;g3.119.400770" data-apath="/ggg/10/3/1019.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1019" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Experimental DNA Demethylation Associates with Changes in Growth and Gene Expression of Oak Tree Seedlings</span></a></div>
</cite>
</div></li><li class="even"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495866" id="node495866" data-pisa="ggg;10/3/1029" data-pisa-master="ggg;g3.119.401035" data-apath="/ggg/10/3/1029.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1029" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">Generating Stable Knockout Zebrafish Lines by Deleting Large Chromosomal Fragments Using Multiple gRNAs</span></a></div>
</cite>
</div></li><li class="last odd"><div class="highwire-article-citation highwire-citation-type-highwire-article" data-node-nid="495863" id="node495863" data-pisa="ggg;10/3/1051" data-pisa-master="ggg;g3.119.401006" data-apath="/ggg/10/3/1051.atom"><cite class='highwire-cite highwire-citation-jcore-title-only'>
  <div class='highwire-cite-title'><a href="/content/10/3/1051" class="highwire-cite-linked-title" data-icon-position="" data-hide-link-title="0"><span class="highwire-cite-title">The Developmental Transcriptome of <em>Aedes albopictus</em>, a Major Worldwide Human Disease Vector</span></a></div>
</cite>
</div></li></ul></div><div class='highwire-list-footer'><a href="/content/by/section/Investigations" class="" data-icon-position="" data-hide-link-title="0">Show more <span class="wrapper">Investigations</span></a></div></div></div></div>
  </div>
</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-highwire-article-nav mobile-hidden floating-article" >
  
      
  
  <div class="pane-content">
    <div class="highwire-list-wrapper highwire-article-nav highwire-nav-float"><div class="highwire-list"><ul data-highwire-float="1" data-highwire-float-class="grid-10 alpha omega"><li class="first odd"><a href="#skip-link" class="" data-icon-position="" data-hide-link-title="0">Top</a></li><li class="parent even"><a href="/content/6/10/3169" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Article</a><div class="highwire-list"><ul><li class="first odd"><a href="#abstract-1" class="highwire-article-nav-jumplink first" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Abstract</a></li><li class="even"><a href="#sec-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Materials and Methods</a></li><li class="odd"><a href="#sec-18" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Results and Discussion</a></li><li class="even"><a href="#ack-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Acknowledgments</a></li><li class="odd"><a href="#fn-group-1" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Footnotes</a></li><li class="last even"><a href="#ref-list-1" class="highwire-article-nav-jumplink last" data-panel-ajax-tab="jnl_genetics_tab_art" data-icon-position="" data-hide-link-title="0">Literature Cited</a></li></ul></div></li><li class="odd"><a href="/content/6/10/3169.figures-only" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_data" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Figures & Data</a></li><li class="even"><a href="/content/6/10/3169.supplemental" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_supp" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Supplemental</a></li><li class="odd"><a href="/content/6/10/3169.article-info" class="highwire-article-nav-jumplink" data-panel-ajax-tab="jnl_genetics_tab_info" data-is-tab-link="true" data-icon-position="" data-hide-link-title="0">Info & Metrics</a></li><li class="last even"><a href="/content/6/10/3169.full.pdf" class="highwire-article-nav-jumplink" data-icon-position="" data-hide-link-title="0"><i class="icon-file-alt"></i></a></li></ul></div></div>  </div>

  
  </div>
</div>
			</div>
		</div>
	
	</div> <!-- /.panel-row-wrapper -->	
	
	</div>

    </div>
  </div>
</div>      </div>
</div>  </div>
</section>    
  
      <footer id="section-footer" class="section section-footer">
    
  <div id="zone-postscript" class="zone zone-postscript clearfix container-30">
    <div class="grid-7 prefix-1 region region-postscript-first" id="region-postscript-first">
  <div class="region-inner region-postscript-first-inner">
    <div class="block block-panels-mini block-snippet-about-gsa block-panels-mini-snippet-about-gsa odd block-without-title" id="block-panels-mini-snippet-about-gsa">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_about_gsa">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-about-gsa" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-about-gsa" id="snippet-about-gsa">
  
      
  <div class="snippet-content">
    <p><a href="http://www.genetics-gsa.org/" target="_blank"><img alt="GSA" src="/sites/default/files/additional-assets/GSALogoWhite_lg.png" style="width: 225px;" /></a></p>

<p>The Genetics Society of America (GSA), founded in 1931, is the professional membership organization for scientific researchers and educators in the field of genetics. Our members work to advance knowledge in the basic mechanisms of inheritance, from the molecular to the population level.</p>

<p>Online ISSN: <span class="issn">2160-1836</span></p>
  </div>

</div>
  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div><div class="grid-5 region region-postscript-second" id="region-postscript-second">
  <div class="region-inner region-postscript-second-inner">
    <div class="block block-menu block-menu-for-authors-menu block-menu-menu-for-authors-menu odd block-without-title" id="block-menu-menu-for-authors-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/why" data-hide-link-title="0" class="" data-icon-position="">For Authors</a></li>
<li class="leaf" role="menuitem"><a href="/content/guidelines-reviewers" data-hide-link-title="0" class="" data-icon-position="">For Reviewers</a></li>
<li class="last leaf" role="menuitem"><a href="/content/advertising" data-hide-link-title="0" class="" data-icon-position="">For Advertisers</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-7 region region-postscript-third" id="region-postscript-third">
  <div class="region-inner region-postscript-third-inner">
    <div class="block block-menu block-menu-why-publish-menu block-menu-menu-why-publish-menu odd block-without-title" id="block-menu-menu-why-publish-menu">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/submit" data-hide-link-title="0" class="" data-icon-position="">Submit a Manuscript</a></li>
<li class="leaf" role="menuitem"><a href="/content/editorial-board" data-hide-link-title="0" class="" data-icon-position="">Editorial Board</a></li>
<li class="last leaf" role="menuitem"><a href="/content/press-releases" data-hide-link-title="0" class="" data-icon-position="">Press Releases</a></li>
</ul></nav>    </div>
  </div>
</div>  </div>
</div><div class="grid-9 suffix-1 region region-postscript-fourth" id="region-postscript-fourth">
  <div class="region-inner region-postscript-fourth-inner">
    <div class="block block-panels-mini block-snippet-for-get-connected block-panels-mini-snippet-for-get-connected odd block-without-title" id="block-panels-mini-snippet-for-get-connected">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-snippet_for_get_connected">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-snippet-for-get-connected" >
  
      
  
  <div class="pane-content">
    <div class="snippet snippet-for-get-connected" id="snippet-for-get-connected">
  
      
  <div class="snippet-content">
    <script type="text/javascript">
setTimeout(function(){var a=document.createElement("script");
var b=document.getElementsByTagName("script")[0];
a.src=document.location.protocol+"//script.crazyegg.com/pages/scripts/0042/1390.js?"+Math.floor(new Date().getTime()/3600000);
a.async=true;a.type="text/javascript";b.parentNode.insertBefore(a,b)}, 1);
</script>
<style type="text/css"><!-- p.sniphang { padding-left: 30px; text-indent: -30px; } -->
</style>
<p><a href="http://byscientistsforscience.org" target="_blank"><img style="max-width: 150px;" alt="SPPA Logo" width="150" src="/sites/default/files/additional-assets/gen-sspa-logo@2x.png" /></a></p>

<p style="border-top: 1px solid #FFFFFF; padding-top: 6px;">GET CONNECTED</p>

<p class="sniphang"><a href="/content/rss"><img alt="RSS" src="/sites/default/files/additional-assets/rss-square.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe with RSS.</strong></a></p>

<p class="sniphang" style="padding-top:10px;"><a href="/alerts"><img alt="email" src="/sites/default/files/additional-assets/emailsubscribe.png" style="width: 24px; height: auto;" /> &nbsp;<strong>Subscribe via email.</strong></a> Sign up to receive alert notifications of new articles.</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-social-media social-media-menu inverted pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="https://www.facebook.com/GeneticsGSA" target="_blank" class="social-menu-facebook link-icon-only link-icon"><i class="icon-facebook"></i> <span class="title element-invisible">Facebook</span></a></li>
<li class="leaf" role="menuitem"><a href="https://twitter.com/GeneticsGSA" target="_blank" class="social-menu-twitter link-icon-only link-icon"><i class="icon-twitter"></i> <span class="title element-invisible">Twitter</span></a></li>
<li class="leaf" role="menuitem"><a href="https://www.youtube.com/user/GeneticsSociety/featured" target="_blank" class="social-menu-youtube link-icon-only link-icon"><i class="icon-youtube-play"></i> <span class="title element-invisible">YouTube</span></a></li>
<li class="leaf" role="menuitem"><a href="http://www.linkedin.com/grp/home?gid=4533930" target="_blank" class="social-menu-linkedin link-icon-only link-icon"><i class="icon-linkedin"></i> <span class="title element-invisible">LinkedIn</span></a></li>
<li class="last leaf" role="menuitem"><a href="http://plus.google.com/+Genetics-gsaOrg" target="_blank" class="social-menu-gplus link-icon-only link-icon"><i class="icon-google-plus"></i> <span class="title element-invisible">Google Plus</span></a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
  
  <div id="zone-footer" class="zone zone-footer clearfix container-30">
    <div class="grid-28 suffix-1 prefix-1 region region-footer-second" id="region-footer-second">
  <div class="region-inner region-footer-second-inner">
    <div class="block block-panels-mini block-footer-bottom block-panels-mini-footer-bottom odd block-without-title" id="block-panels-mini-footer-bottom">
  <div class="block-inner clearfix">
                
    <div class="content clearfix">
      <div class="panel-display panel-1col clearfix" id="mini-panel-footer_bottom">
  <div class="panel-panel panel-col">
    <div><div class="panel-pane pane-snippet pane-footer-copyright-text float-me-right" >
  
      
  
  <div class="pane-content">
    <div class="snippet footer-copyright-text" id="footer-copyright-text">
  
      
  <div class="snippet-content">
    <p>Copyright &copy; 2020 by the Genetics Society of America</p>  </div>

</div>
  </div>

  
  </div>
<div class="panel-separator"></div><div class="panel-pane pane-block pane-menu-menu-footer-bottom float-me-right inline pipe-separated pane-menu" >
  
      
  
  <div class="pane-content">
    <nav class="menubar-nav"><ul class="menu" role="menu"><li class="first leaf" role="menuitem"><a href="/content/about-journal" class="" data-icon-position="" data-hide-link-title="0">About G3</a></li>
<li class="leaf" role="menuitem"><a href="/content/terms-of-use" class="" data-icon-position="" data-hide-link-title="0">Terms of use</a></li>
<li class="leaf" role="menuitem"><a href="/content/permissions" class="" data-icon-position="" data-hide-link-title="0">Permissions</a></li>
<li class="leaf" role="menuitem"><a href="/feedback" class="" data-icon-position="" data-hide-link-title="0">Contact us</a></li>
<li class="last leaf" role="menuitem"><a href="http://intl.genetics.org" class="" data-icon-position="" data-hide-link-title="0">International access</a></li>
</ul></nav>  </div>

  
  </div>
</div>
  </div>
</div>
    </div>
  </div>
</div>  </div>
</div>  </div>
</footer>  </div>    <div class="region region-page-bottom" id="region-page-bottom">
  <div class="region-inner region-page-bottom-inner">
      </div>
</div><script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__bnwyEQSuZLC-kZnFqCi5iE02nOCzM7DjY7zYxcN9nsQ__VBiBT_qq_eHMMO2Q1rXK1u-hHff31XLUkmi9dgjuPeY__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__GM3GJPR36rRIz0TRkjC5OQwrioSyN9aoYRivDhCO_AM__qAl84FcCv2jyN22yFGS5Oc85cjd9zKX6p_cFNLGhe-M__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" async="async"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__yAtUDeu0LvdtdwYQ24Hq_lBQByZG7b3Bu-K9EufvaYU__EfdUTnHINMVHwamrF3_hpSyZ11KtE_Lz6AqQyj_JdBE__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__2WRbxlwOW0MEUc_hSWU5MBepQg6Lch6O5SZwefpJ6IE__HCL0YQJqLkOhrLPZZYGqosGvtFsEHMGghHIkSx4y9vA__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js" defer="defer"></script>
<script type="text/javascript" src="https://www.g3journal.org/sites/default/files/advagg_js/js__RV38jjglfSuUdY8cQOUwGWkL7LQm2m5SQGkTmInHvDs__6nRHB8HNYX1sS33y-ol7jpPLA2sY3h4Jxrk49x62fDc__UExr2Q2E8OXNvQQoz03ZVqjsKhi6mZlP4mmQlO5qsXQ.js"></script>
  </body>
</html>
